

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001207"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001207" />

    <meta name="citation_title" content="Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast"/>
    <meta itemprop="name" content="Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast"/>

      <meta name="citation_author" content="Guangxu Jin"/>
            <meta name="citation_author_institution" content="Academy of Mathematics and Systems Science, Chinese Academy of Science, Beijing, China"/>
            <meta name="citation_author_institution" content="Graduate University of Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Shihua Zhang"/>
            <meta name="citation_author_institution" content="Academy of Mathematics and Systems Science, Chinese Academy of Science, Beijing, China"/>
            <meta name="citation_author_institution" content="Graduate University of Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Xiang-Sun Zhang"/>
            <meta name="citation_author_institution" content="Academy of Mathematics and Systems Science, Chinese Academy of Science, Beijing, China"/>
      <meta name="citation_author" content="Luonan Chen"/>
            <meta name="citation_author_institution" content="Institute of Systems Biology, Shanghai University, Shanghai, China"/>
            <meta name="citation_author_institution" content="Department of Electrical Engineering and Electronics, Osaka Sangyo University, Osaka, Japan"/>
            <meta name="citation_author_institution" content="Exploratory Research for Advanced Technology (ERATO) Aihara Complexity Modelling Project, Japan Science and Technology Corporation (JST), Tokyo, Japan"/>
            <meta name="citation_author_institution" content="Institute of Industrial Science, The University of Tokyo, Tokyo, Japan"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001207.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1207"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Lethality and centrality in protein networks.; citation_author=H Jeong; citation_author=SP Mason; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nature; citation_volume=411; citation_number=1; citation_pages=41-42; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Error and attack tolerance of complex networks.; citation_author=R Albert; citation_author=H Jeong; citation_author=AL Barabasi; citation_journal_title=Nature; citation_volume=406; citation_number=2; citation_pages=378-382; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Emergence of scaling in random networks.; citation_author=AL Barabasi; citation_author=R Albert; citation_journal_title=Science; citation_volume=286; citation_number=3; citation_pages=509; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Network biology: understanding the cell's functional organization.; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nat Rev Genet; citation_volume=5(2); citation_number=4; citation_pages=101-13; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Linked: the new science of networks.; citation_author=AL Barabasi; citation_number=5; citation_date=2002; citation_publisher=Perseus Pub; " />
      <meta name="citation_reference" content="citation_title=Functional topology in a network of protein interactions.; citation_author=N Przulj; citation_author=DA Wigle; citation_author=I Jurisica; citation_journal_title=Bioinformatics.; citation_volume=12;20(3); citation_number=6; citation_pages=340-8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Subnets of scale-free networks are not scale-free: sampling properties of networks.; citation_author=MP Stumpf; citation_author=C Wiuf; citation_author=RM May; citation_journal_title=Proc Natl Acad Sci U S A.; citation_volume=22;102(12); citation_number=7; citation_pages=4221-4; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the structure of protein-protein interaction networks.; citation_author=A Thomas; citation_author=R Cannings; citation_author=NA Monk; citation_author=C Cannings; citation_journal_title=Biochem Soc Trans.; citation_volume=31(Pt 6); citation_number=8; citation_pages=1491-6; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Effect of sampling on topology predictions of protein-protein interaction networks.; citation_author=JD Han; citation_author=D Dupuy; citation_author=N Bertin; citation_author=ME Cusick; citation_author=M Vidal; citation_journal_title=Nat Biotechnol.; citation_volume=23(7); citation_number=9; citation_pages=839-44; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for dynamically organized modularity in the yeast protein-protein interaction network.; citation_author=JD Han; citation_author=N Bertin; citation_author=T Hao; citation_author=DS Goldberg; citation_author=GF Berriz; citation_journal_title=Nature; citation_volume=430; citation_number=10; citation_pages=88-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stratus not altocumulus: A new view of the yeast protein interaction network.; citation_author=NN Batada; citation_author=T Reguly; citation_author=A Breitkreutz; citation_author=L Boucher; citation_author=BJ Breitkreutz; citation_journal_title=PLoS Biol; citation_volume=4(10); citation_number=11; citation_pages=e317; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=What properties characterize the hub proteins of the protein-protein interaction network of Saccharomyces cerevisiae?; citation_author=D Ekman; citation_author=S Light; citation_author=AK Bjorklund; citation_author=A Elofsson; citation_journal_title=Genome Biol; citation_volume=7; citation_number=12; citation_pages=R45; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Why do hubs tend to be essential in protein networks?; citation_author=X He; citation_author=J Zhang; citation_journal_title=PLoS Genet; citation_volume=2(6); citation_number=13; citation_pages=0868-0834; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary and physiological importance of hub proteins.; citation_author=NN Batada; citation_author=LD Hurst; citation_author=M Tyers; citation_journal_title=PLoS Comput Biol; citation_volume=2(7); citation_number=14; citation_pages=e88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Specificity and stability in topology of protein networks.; citation_author=S Maslov; citation_author=K Sneppen; citation_journal_title=Science; citation_volume=296; citation_number=15; citation_pages=910-913; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From molecular to modular cell biology.; citation_author=LH Hartwell; citation_author=JJ Hopfield; citation_author=S Leibler; citation_author=AW Murray; citation_journal_title=Nature; citation_volume=402; citation_number=16; citation_pages=C47-C52; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Network Motifs: Simple Building Blocks of Complex Networks.; citation_author=R Milo; citation_author=S Shen-Orr; citation_author=S Itzkovitz; citation_author=N Kashtan; citation_author=D Chklovskii; citation_journal_title=Science; citation_volume=298; citation_number=17; citation_pages=824-827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Superfamilies of designed and evolved networks.; citation_author=R Milo; citation_author=S Itzkovitz; citation_author=N Kashtan; citation_author=R Levitt; citation_author=S Shen-Orr; citation_journal_title=Science; citation_volume=303; citation_number=18; citation_pages=1538-42; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Network motifs in the transcriptional regulation network of Escherichia coli.; citation_author=S Shen-Orr; citation_author=R Milo; citation_author=S Mangan; citation_author=U Alon; citation_journal_title=Nat Genet; citation_volume=31; citation_number=19; citation_pages=64-68; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Incoherent Feed-forward Loop Accelerates the Response-time of the gal System of Escherichia coli.; citation_author=S Mangan; citation_author=S Itzkovitz; citation_author=A Zaslaver; citation_author=U Alon; citation_journal_title=JMB; citation_volume=356; citation_number=20; citation_pages=1073-81; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Coherent Feedforward Loop Serves as a Sign-sensitive Delay Element in Transcription Networks.; citation_author=S Mangan; citation_author=A Zaslaver; citation_author=U Alon; citation_journal_title=JMB 334/; citation_volume=2; citation_number=21; citation_pages=197-204; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A systems biology perspective on signal processing in genetic motifs.; citation_author=C Li; citation_author=L Chen; citation_author=K Aihara; citation_journal_title=IEEE Signal Proc Mag,; citation_volume=24; citation_number=22; citation_pages=136-147; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Are network motifs the spandrels of cellular complexity?; citation_author=RV Sole; citation_author=S Valverde; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=23; citation_pages=419-422; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Topological units of environmental signal processing in the transcriptional regulatory network of Escherichia coli.; citation_author=G Balazsi; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Proc Natl Acad Sci U S A.; citation_volume=31;102(22); citation_number=24; citation_pages=7841-6; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A comprehensive two-hybrid analysis to explore the yeast protein interactome.; citation_author=T Ito; citation_author=T Chiba; citation_author=R Ozawa; citation_author=M Yoshida; citation_author=M Hattori; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=25; citation_pages=4569-4574; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.; citation_author=P Uetz; citation_author=L Giot; citation_author=G Cagney; citation_author=TA Mansfield; citation_author=RS Judson; citation_journal_title=Nature; citation_volume=403; citation_number=26; citation_pages=623-627; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Proteome survey reveals modularity of the yeast cell machinery.; citation_author=AC Gavin; citation_author=P Aloy; citation_author=P Grandi; citation_author=R Krause; citation_author=M Boesche; citation_journal_title=Nature; citation_volume=440; citation_number=27; citation_pages=631-637; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.; citation_author=NJ Krogan; citation_author=G Cagney; citation_author=H Yu; citation_author=G Zhong; citation_author=X Guo; citation_journal_title=Nature; citation_volume=440; citation_number=28; citation_pages=637-644; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MAP kinase pathways in the yeast Saccharomyces cerevisiae.; citation_author=MC Gustin; citation_author=J Albertyn; citation_author=M Alexander; citation_author=K Davenport; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=62(4); citation_number=29; citation_pages=1264-300; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MAP kinase pathways in yeast: for mating and more.; citation_author=I Herskowitz; citation_journal_title=Cell; citation_volume=80(2); citation_number=30; citation_pages=187-97; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Building with a scaffold: emerging strategies for high- to low-level cellular modeling.; citation_author=T Ideker; citation_author=D Lauffenburger; citation_journal_title=Trends Biotechnol; citation_volume=21; citation_number=31; citation_pages=255-262; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide protein interaction screens reveal functional networks involving Sm-like proteins.; citation_author=M Fromont-Racine; citation_author=AE Mayes; citation_author=A Brunet-Simon; citation_author=JC Rain; citation_author=A Colley; citation_journal_title=Yeast; citation_volume=17; citation_number=32; citation_pages=95-110; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens.; citation_author=M Fromont-Racine; citation_author=JC Rain; citation_author=P Legrain; citation_journal_title=Nature Genet; citation_volume=16; citation_number=33; citation_pages=277-282; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry.; citation_author=Y Ho; citation_author=A Gruhler; citation_author=A Heilbut; citation_author=GD Bader; citation_author=L Moore; citation_journal_title=Nature; citation_volume=415; citation_number=34; citation_pages=180-183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional organization of the yeast proteome by systematic analysis of protein complexes.; citation_author=AC Gavin; citation_author=M Bosche; citation_author=R Krause; citation_author=P Grandi; citation_author=M Marzioch; citation_journal_title=Nature; citation_volume=415; citation_number=35; citation_pages=141-147; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=BIND: the Biomolecular Interaction Network Database.; citation_author=GD Bader; citation_author=D Betel; citation_author=CW Hogue; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=36; citation_pages=248-250; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative assessment of large-scale data sets of proteinCprotein interactions.; citation_author=C von Mering; citation_author=R Krause; citation_author=B Snel; citation_author=M Cornell; citation_author=SG Oliver; citation_journal_title=Nature; citation_volume=417; citation_number=37; citation_pages=399-403; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A combined algorithm for genome-wide prediction of protein function.; citation_author=EM Marcotte; citation_author=M Pellegrini; citation_author=MJ Thompson; citation_author=TO Yeates; citation_author=D Eisenberg; citation_journal_title=Nature; citation_volume=402; citation_number=38; citation_pages=83-86; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Conservation of gene order: a fingerprint of proteins that physically interact.; citation_author=T Dandekar; citation_author=B Snel; citation_author=M Huynen; citation_author=P Bork; citation_journal_title=Trends Biochem Sci; citation_volume=23; citation_number=39; citation_pages=324-328; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=MIPS: a database for genomes and protein sequences.; citation_author=HW Mewes; citation_author=D Frishman; citation_author=U Guldener; citation_author=G Mannhaupt; citation_author=K Mayer; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=40; citation_pages=31-34; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comprehensive curation and analysis of global interaction networks in Saccharomyces cerevisiae.; citation_author=T Reguly; citation_author=A Breitkreutz; citation_author=L Boucher; citation_author=BJ Breitkreutz; citation_author=GC Hon; citation_journal_title=J Biol; citation_volume=5; citation_number=41; citation_pages=11; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DIP: The database of interacting proteins.; citation_author=I Xenarios; citation_author=DW Rice; citation_author=L Salwinski; citation_author=MK Baron; citation_author=EM Marcotte; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=42; citation_pages=289-291; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=MINT: a Molecular INTeraction database.; citation_author=A Zanzoni; citation_author=L Montecchi-Palazzi; citation_author=M Quondam; citation_author=G Ausiello; citation_author=M Helmer-Citterich; citation_journal_title=FEBS Lett.; citation_volume=20;513(1); citation_number=43; citation_pages=135-40; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From genomics to chemical genomics: new devel-opments in KEGG.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=M Hattori; citation_author=KF Aoki-Kinoshita; citation_author=M Itoh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=44; citation_pages=D354-7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global analysis of protein localization in budding yeast.; citation_author=WK Huh; citation_author=JV Falvo; citation_author=LC Gerke; citation_author=AS Carroll; citation_author=RW Howson; citation_journal_title=Nature; citation_volume=425; citation_number=45; citation_pages=686-691; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Protein interaction verification and functional annotation by integrated analysis of genome-scale data.; citation_author=P Kemmeren; citation_author=NL van Berkum; citation_author=J Vilo; citation_author=T Bijma; citation_author=R Donders; citation_journal_title=Mol Cell; citation_volume=9; citation_number=46; citation_pages=1133-1143; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genomic expression programs in the response of yeast cells to environmental changes.; citation_author=AP Gasch; citation_author=PT Spellman; citation_author=CM Kao; citation_author=O Carmel-Harel; citation_author=MB Eisen; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=47; citation_pages=4241-4257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comprehensive identification of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization.; citation_author=PT Spellman; citation_author=G Sherlock; citation_author=MQ Zhang; citation_author=VR Iyer; citation_author=K Anders; citation_journal_title=Mol Biol Evol; citation_volume=9; citation_number=48; citation_pages=3273-3297; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Signaling and circuitry of multiple MAPK pathways revealed by a matrix of global gene expression profiles.; citation_author=CJ Roberts; citation_author=B Nelson; citation_author=MJ Marton; citation_author=R Stoughton; citation_author=MR Meyer; citation_journal_title=Science; citation_volume=287; citation_number=49; citation_pages=873-880; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The transcriptional program of sporulation in budding yeast.; citation_author=S Chu; citation_author=J DeRisi; citation_author=M Eisen; citation_author=J Mulholland; citation_author=D Botstein; citation_journal_title=Science; citation_volume=282; citation_number=50; citation_pages=699-705; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional and genomic analyses reveal an essential coordination between the unfolded protein response and ERassociated degradation.; citation_author=KJ Travers; citation_author=CK Patil; citation_author=L Wodicka; citation_author=DJ Lockhart; citation_author=JS Weissman; citation_journal_title=Cell; citation_volume=101; citation_number=51; citation_pages=249-258; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microarray data normalization and transformation.; citation_author=J Quackenbush; citation_journal_title=Nat Genet; citation_volume=32; citation_number=52; citation_pages=496-501; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=BioLayout-an automatic graph layout algorithm for similarity visualization.; citation_author=AJ Enright; citation_author=CA Ouzounis; citation_journal_title=Bioinformatics; citation_volume=17(9); citation_number=53; citation_pages=853-854; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Fundamentals of Biostatistic.; citation_author=R Bernard; citation_number=54; citation_date=1986; citation_publisher=Duxbury Press; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast"/>
    <meta name="twitter:description" content="BackgroundIt has been recognized that modular organization pervades biological complexity. Based on network analysis, &lsquo;party hubs&rsquo; and &lsquo;date hubs&rsquo; were proposed to understand the basic principle of module organization of biomolecular networks. However, recent study on hubs has suggested that there is no clear evidence for coexistence of &lsquo;party hubs&rsquo; and &lsquo;date hubs&rsquo;. Thus, an open question has been raised as to whether or not &lsquo;party hubs&rsquo; and &lsquo;date hubs&rsquo; truly exist in yeast interactome.MethodologyIn contrast to previous studies focusing on the partners of a hub or the individual proteins around the hub, our work aims to study the network motifs of a hub or interactions among individual proteins including the hub and its neighbors. Depending on the relationship between a hub's network motifs and protein complexes, we define two new types of hubs, &lsquo;motif party hubs&rsquo; and &lsquo;motif date hubs&rsquo;, which have the same characteristics as the original &lsquo;party hubs&rsquo; and &lsquo;date hubs&rsquo; respectively. The network motifs of these two types of hubs display significantly different features in spatial distribution (or cellular localizations), co-expression in microarray data, controlling topological structure of network, and organizing modularity.ConclusionBy virtue of network motifs, we basically solved the open question about &lsquo;party hubs&rsquo; and &lsquo;date hubs&rsquo; which was raised by previous studies. Specifically, at the level of network motifs instead of individual proteins, we found two types of hubs, motif party hubs (mPHs) and motif date hubs (mDHs), whose network motifs display distinct characteristics on biological functions. In addition, in this paper we studied network motifs from a different viewpoint. That is, we show that a network motif should not be merely considered as an interaction pattern but be considered as an essential function unit in organizing modules of networks."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001207.g006"/>

  <meta property="og:title" content="Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9296'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9718'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1140&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001207">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001207" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001207&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guangxu Jin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Academy of Mathematics and Systems Science, Chinese Academy of Science, Beijing, China, 
                  Graduate University of Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shihua Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Academy of Mathematics and Systems Science, Chinese Academy of Science, Beijing, China, 
                  Graduate University of Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiang-Sun Zhang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zxs@amt.ac.cn">zxs@amt.ac.cn</a> (XZ); <a href="mailto:chen@eic.osaka-sandai.ac.jp">chen@eic.osaka-sandai.ac.jp</a> (LC)</p>

                <p>Affiliation:
                  Academy of Mathematics and Systems Science, Chinese Academy of Science, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Luonan Chen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:zxs@amt.ac.cn">zxs@amt.ac.cn</a> (XZ); <a href="mailto:chen@eic.osaka-sandai.ac.jp">chen@eic.osaka-sandai.ac.jp</a> (LC)</p>

                <p>Affiliations:
                  Institute of Systems Biology, Shanghai University, Shanghai, China, 
                  Department of Electrical Engineering and Electronics, Osaka Sangyo University, Osaka, Japan, 
                  Exploratory Research for Advanced Technology (ERATO) Aihara Complexity Modelling Project, Japan Science and Technology Corporation (JST), Tokyo, Japan, 
                  Institute of Industrial Science, The University of Tokyo, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001207</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001207-g001" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-g002" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-g003" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-g004" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-t001" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-g005" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-t002" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-t003" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001207-g006" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001207">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001207" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>It has been recognized that modular organization pervades biological complexity. Based on network analysis, ‘party hubs’ and ‘date hubs’ were proposed to understand the basic principle of module organization of biomolecular networks. However, recent study on hubs has suggested that there is no clear evidence for coexistence of ‘party hubs’ and ‘date hubs’. Thus, an open question has been raised as to whether or not ‘party hubs’ and ‘date hubs’ truly exist in yeast interactome.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In contrast to previous studies focusing on the partners of a hub or the individual proteins around the hub, our work aims to study the network motifs of a hub or interactions among individual proteins including the hub and its neighbors. Depending on the relationship between a hub's network motifs and protein complexes, we define two new types of hubs, ‘motif party hubs’ and ‘motif date hubs’, which have the same characteristics as the original ‘party hubs’ and ‘date hubs’ respectively. The network motifs of these two types of hubs display significantly different features in spatial distribution (or cellular localizations), co-expression in microarray data, controlling topological structure of network, and organizing modularity.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>By virtue of network motifs, we basically solved the open question about ‘party hubs’ and ‘date hubs’ which was raised by previous studies. Specifically, at the level of network motifs instead of individual proteins, we found two types of hubs, motif party hubs (mPHs) and motif date hubs (mDHs), whose network motifs display distinct characteristics on biological functions. In addition, in this paper we studied network motifs from a different viewpoint. That is, we show that a network motif should not be merely considered as an interaction pattern but be considered as an essential function unit in organizing modules of networks.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Jin G, Zhang S, Zhang X-S, Chen L (2007) Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast. PLoS ONE 2(11):
          e1207.
            doi:10.1371/journal.pone.0001207</p><p><strong>Academic Editor: </strong>Nick Monk, University of Nottingham, United Kingdom</p><p><strong>Received:</strong> April 26, 2007; <strong>Accepted:</strong> October 30, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Jin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was partially supported by the National Natural Science Foundation of China under grant No.10631070, and the Ministry of Science and Technology, China, under grant No.2006CB503905.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Many types of molecular networks display scale-free topologies which are characterized by the power-law degree distribution <a href="#pone.0001207-Jeong1">[1]</a>–<a href="#pone.0001207-Barabasi3">[5]</a>. In spite of some negative remarks <a href="#pone.0001207-Przulj1">[6]</a>–<a href="#pone.0001207-Han1">[9]</a> on the studies of network structures, a small fraction of proteins generally interacting with many partners, i.e. so-called hubs, have attracted great interests <a href="#pone.0001207-Han2">[10]</a>–<a href="#pone.0001207-Maslov1">[15]</a> from the communities of both engineering and biology. To identify whether hubs vary their biological roles with the timing and location of the interactions, Han <em>et al.</em> proposed two types of hubs, i.e. ‘party hubs’ and ‘date hubs’, based on whether or not the hubs are co-expressed with their partners by using yeast microarray data <a href="#pone.0001207-Han2">[10]</a>. The two distinct types of hubs not only display diverse spatial distribution for their partners but also organize the modules in different manners, where a module is referred as a group of physically or functionally linked molecules that work together to achieve a relatively distinct function <a href="#pone.0001207-Han2">[10]</a>, <a href="#pone.0001207-Hartwell1">[16]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>It should be noticed that the result of Han <em>et al</em>. on ‘party hub’ and ‘date hub’ was drawn from a filtered yeast interactome data (FYI). Recently, Batada <em>et al.</em> derived different results, in contrast to those of Han <em>et al</em>., based on another filtered yeast interactome data (<em>HC<sup>fyi</sup></em>) manually curated from online publications <a href="#pone.0001207-Batada1">[11]</a> (see <a href="#s4">Materials and Methods</a>). Due to the topological difference between FYI and <em>HC<sup>fyi</sup></em>, Batada <em>et al.</em> found that there is no evidence for coexistence of party hubs and date hubs, and the results about ‘party hubs’ and ‘date hubs’ are totally not correct. Thus, the most striking question raised by them is whether or not the ‘party hubs’ and ‘date hubs’ truly exist in the networks.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this paper, we aim to solve the contradiction between the two previous works. In virtue of network motifs, we define two new types of hubs in <em>HC<sup>fyi</sup></em>, i.e. ‘motif party hub’ and ‘motif date hub’, which have the same characteristics as ‘party hub’ and ‘date hub’ respectively. Network motifs are the subgraphs that occur significantly more frequently in original network than random ones <a href="#pone.0001207-Milo1">[17]</a>–<a href="#pone.0001207-Li1">[22]</a>. They have been revealed as the functional building blocks of biology networks <a href="#pone.0001207-Milo1">[17]</a>, <a href="#pone.0001207-Milo2">[18]</a> and the spandrels of cellular complexity <a href="#pone.0001207-Sole1">[23]</a>. Similar to a previous research work on the role of network motifs in information processing <a href="#pone.0001207-Balazsi1">[24]</a>, we focus on their important roles in acting as functional units in organizing modules. Moreover, in contrast to the previous studies on hubs, our work emphasizes on interactions of hubs and network motifs instead of individual proteins around hubs.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Specifically, we divide hubs into ‘motif party hubs’ (mPHs) and ‘motif date hubs’ (mDHs) based on the relationship between a hub's network motifs and protein complexes. In this paper, we demonstrate that the network motifs of an mPH (i.e. network motifs take the mPH as one of their nodes) are more likely to stay inside a protein complex with their mPH, control the local topological structure, locate in the same cellular localizations as the mPH, and co-express in microarray data. On the other hand, we reveal that the network motifs of an mDH tend to spread into different complexes, and act as the connectors among signal pathways, control the global topological structure, locate in different cellular localizations, and express differently in microarray data.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Motif party hub (mPH) and motif date hub (mDH)</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>About 20% proteins, i.e. 197 proteins, in <em>HC<sup>fyi</sup></em> were defined as hubs whose partners are not less than 12. There are 196 motif hubs with at least one network motif and only one hub without any network motif (see <a href="#s4">Methods and Materials</a> and Table S1). Based on the relationship between a hub's network motifs and protein complexes, we divided the 196 motif hubs into 98 mPHs and 98 mDHs (see <a href="#s4">Methods and Materials</a>). The quantitative criterion <em>Complex<sub>ratio-same</sub></em> was defined to identify the relationship between a hub's network motifs and protein complexes. A relatively high <em>Complex<sub>ratio-same</sub></em> implies that more network motifs of a hub (a network motif takes a hub as one of its nodes, and a hub may be used by multiple network motifs) belong to the same protein complex as the hub, e.g. the four proteins or three proteins in such a network motif are more likely to be in just one protein complex. Otherwise, it indicates that less network motifs of the hub belong to the same protein complex, e.g. the four proteins or three proteins in such a network motif are more likely to be parts of different protein complexes. From the definition of mPHs and mDHs, we can see that the network motifs of an mPH more likely stay together in the same protein complex as the mPH while those of an mDH spread outside the protein complex of the mDH.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In this paper, mPHs and mDHs defined by network motifs and protein complexes were introduced to study hubs at the level of network motifs instead of individual proteins (or nodes), so as to solve the open question whether or not <em>HC<sup>fyi</sup></em> contains date hubs and party hubs. Due to the topological distinction between <em>HC<sup>fyi</sup></em> and <em>FYI</em> (<em>HC<sup>fyi</sup></em> looks like stratus while <em>FYI</em> looks like altocumulus <a href="#pone.0001207-Batada1">[11]</a>), 103 hubs were found in the overlap between 199 hubs in <em>FYI</em> and 197 hubs in <em>HC<sup>fyi</sup></em>. We found that more than 60% of ‘party hubs’ and ‘date hubs’ defined by Han <em>et al.</em> have been correctly divided into mPHs and mDHs respectively by our method among the overlapped 103 hubs between FYI (proposed by Han <em>et al.</em>) and <em>HC<sup>fyi</sup></em> (proposed by Batada <em>et al.</em>) (see <a href="#pone-0001207-g001">Figure 1</a>). Furthermore, we can see that the motifs of mPHs and mDHs have the same characteristics as the partners of party hubs and date hubs respectively in cellular localization or spatial distribution, controlling topological structure, linking with signal pathways and co-expression in microarray data. Therefore, we call them as motif ‘party’ and motif ‘date’ hubs due to the facts that they are similar to ‘party’ and ‘date’ hubs mentioned in Han <em>et al</em>. <a href="#pone.0001207-Han2">[10]</a>.</p>
<div class="figure" id="pone-0001207-g001"><div class="img"><a name="pone-0001207-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The hub overlap of FYI and <em>HC<sup>fyi</sup></em> and the correctly divided mPHs and mDHs in the overlap.</span></strong></p><span>doi:10.1371/journal.pone.0001207.g001</span></div>

<h4>Distinct cellular localizations for mPHs and mDHs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>One of main distinctions between party hubs and date hubs is their different spatial distributions (partners of date hubs are significantly more diverse in spatial distribution than those of party hubs) <a href="#pone.0001207-Han2">[10]</a>. In virtue of another criterion, i.e. <em>Localization<sub>ratio-same</sub></em> (see <a href="#s4">Materials and Methods</a>), we found that <em>Localization<sub>ratio-same</sub></em>s of mPHs are relatively high and those of mDHs are relatively low. The criterion <em>Localization<sub>ratio-same</sub></em> shows localization relationship of network motifs of a hub. A higher one implies that three proteins or four proteins in each of network motifs of a hub are likely to locate in the same cellular localization as the hub. On the other hand, a lower one shows that the proteins in each of network motifs of a hub more likely locate in different cellular localizations. In <a href="#pone-0001207-g002">Figure 2</a>, mPHs have significantly higher <em>Localization<sub>ratio-same</sub></em>s than mDHs (mean of mPHs: 0.7926; mean of mDHs: 0.4826; <em>P</em>&lt;10<sup>−11</sup> for Mann-Whitney U test.). Thus we can say that the network motifs of mPHs and mDHs have significantly different spatial distributions.</p>
<div class="figure" id="pone-0001207-g002"><div class="img"><a name="pone-0001207-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The spatial distribution of hubs.</span></strong></p><span>doi:10.1371/journal.pone.0001207.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In our analysis, ‘nucleus’ and ‘cytoplasm’ were not excluded from the cellular localization data (see <a href="#s4">Materials and Methods</a>). In this respect, our method is also different from one of Han <em>et al</em>. who excluded the ‘nucleus’ and ‘cytoplasm’ from the cellular localization data. Moreover, we found that mPHs and mDHs have a significant difference in these two cellular localizations. Most mPHs (about 65%) are located in the nucleus. However, most mDHs (about 63%) are localized in subcellular compartments other than the nucleus (see <a href="#pone-0001207-g003">Figure 3</a>). It is clear that there is a statistically significant localization difference between mPHs and mDHs (<em>x</em><sup>2</sup> = 15.69, <em>P</em>≤0.001 for chi-square test). Therefore, mPHs prefer to ‘nucleus’ of a cell while mDHs are likely outside ‘nucleus’.</p>
<div class="figure" id="pone-0001207-g003"><div class="img"><a name="pone-0001207-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The localizations for mDHs and mPHs.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>(A) Cellular localizations of mDHs. (B) Cellular localizations of mPHs.</p>
<span>doi:10.1371/journal.pone.0001207.g003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The cellular localization distribution of hubs and their network motifs implies that, mPHs with their network motifs tend to locate in nucleus while mDHs are more likely to locate outside nucleus, and their network motifs have a scattered spatial distribution.</p>


<h4>mPHs and mDHs control network architecture differently</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In early work, it has been shown that the <em>HC<sup>fyi</sup></em> network is tolerant to hubs' deletion, which means that the key components of the <em>HC<sup>fyi</sup></em> network still remain after removal of date hubs or party hubs, or even all the hubs <a href="#pone.0001207-Batada1">[11]</a>. One of direct reasons is that the protein-protein interactions in the network are too dense to be broken into fragments by only removing ‘date hubs’ or ‘party hubs’. Therefore, hubs rarely have effect on the structure of the network in such a case. However, hubs may affect the network topological structure in a different manner. In this paper, a new approach for breaking down both the hubs and their motifs from the network was introduced based on mPHs and mDHs. In <a href="#pone-0001207-g004">Figure 4A</a>, it appears that deleting the mPHs and their motifs has little influence on the main network structure, whereas deleting the mDHs and their motifs makes the network broken into many fragments in <a href="#pone-0001207-g004">Figure 4B</a>. In other words, mDHs and their motifs clearly have a global effect on the network structure. In addition, we also evaluated the p-values for the cases by removing mPHs with their motifs and by removing mDHs with their motifs in <a href="#pone-0001207-g004">Figure 4D</a> ( both of them are less than 0.001). Moreover, in <a href="#pone-0001207-g004">Figure 4C</a>, about 50% of proteins are still connected in the largest component after removal of mPHs (<a href="#pone-0001207-g004">Figure 4A</a>) and their network motifs while only less than 10% of proteins are connected in the largest component after removal of mDHs and their network motifs (<a href="#pone-0001207-g004">Figure 4B</a>). In <a href="#pone-0001207-t001">Table 1</a>, we can see that the largest component after deletion of mPHs and their network motifs (<a href="#pone-0001207-g004">Figure 4A</a>) contains 533 protein-protein interactions while it contains only 47 protein-protein interactions after deletion of mDHs and their network motifs (<a href="#pone-0001207-g004">Figure 4B</a>). Clearly, those results demonstrate that the mPHs mainly control the local structure by their motifs while the mDHs control the global structure by their motifs.</p>
<div class="figure" id="pone-0001207-g004"><div class="img"><a name="pone-0001207-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Deleting mPHs with their motifs and mDHs with their motifs respectively.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) The case of deleting mPHs and their motifs. In this case, the main components remain (the points in the same color are in the same component). (B) The case of deleting mDHs and their network motifs. Deleting the mDHs and their motif proteins causes the main components to disappear (the points in the same color are in the same component). (C) Deleting hubs one by one. The <em>HC<sup>fyi</sup></em> network is tolerant for the deletion of mPHs with their motifs. However, it is not tolerant for the deletion of mDHs with their motifs. (D) Deleting a randomly chosen set of 98 hubs with their motifs. We repeat the removal of 98 hubs with their motifs randomly for 1000 times. The sizes of the largest remaining components are all less than 555 that are the size of the largest component after removing the mPHs with their motifs. The sizes of the largest remaining components are all larger than 47 that are the size of the largest component after removing the mDHs with their motifs. Empirical P values are both less than 10<sup>−3</sup>. Biolayout <a href="#pone.0001207-Enright1">[53]</a> has been used to produce the figures in (A) and (B).</p>
<span>doi:10.1371/journal.pone.0001207.g004</span></div><div class="figure" id="pone-0001207-t001"><div class="img"><a name="pone-0001207-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sizes and Numbers of the network components after removal of mPHs and mDHs respectively</span></strong></p><span>doi:10.1371/journal.pone.0001207.t001</span></div>

<h4>mPHs and mDHs link with signal pathways in different ways</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The definitions of mPHs and mDHs imply that most network motifs of an mPH stay together with the mPH in a protein complex while the network motifs of an mDH are not restricted in a protein complex. Furthermore, we built up a network composed of hubs and signal pathways shown in <a href="#pone-0001207-g005">Figure 5</a>, by which we found that mDHs are more likely to be the connectors among signal pathways. If at least one of a hub's network motifs takes one or more proteins in some signal pathway as its node, there is one link between the hub and the signal pathway in the network as shown in <a href="#pone-0001207-g005">Figure 5</a>. It is not difficult to see that mDHs link with more signal pathways than mPHs (see <a href="#pone-0001207-t002">Table 2</a>, <em>x</em><sup>2</sup> = 5.02, <em>P</em>&lt;0.05 for Chi-square test). In addition, each mDH links with several signal pathways, i.e. the degrees of about 90% (86/98) of mDHs are all larger than 2, and their mean value is 4.3 (see <a href="#pone.0001207.s003">Figure S1</a>). As a result, mPHs and their network motifs mostly stay inside protein complexes, whereas mDHs and their motifs act as connectors among signal pathways.</p>
<div class="figure" id="pone-0001207-g005"><div class="img"><a name="pone-0001207-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Network motifs linking up pathways with mDHs and mPHs.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>In the middle of the panel, the red circles are abstract representations of pathways, where 14 smaller ones only connect with 6 mDHs (circles in light blue), and 26 larger ones connect with not only mDHs in blue circles (except the light blue one at the top of the panel) but also mPHs in orange circles. If more than one motif proteins of an mDH or an mPH are involved in one pathway, we connect the mDH or mPH with the pathway by an edge. Biolayout <a href="#pone.0001207-Enright1">[53]</a> has been used to produce the figure.</p>
<span>doi:10.1371/journal.pone.0001207.g005</span></div><div class="figure" id="pone-0001207-t002"><div class="img"><a name="pone-0001207-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Linking with signal pathways by network motifs</span></strong></p><span>doi:10.1371/journal.pone.0001207.t002</span></div>

<h4>Network motifs of mPHs are more co-expressed than those of mDHs</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Another of main distinctions between party hubs and date hubs is whether or not the hubs are co-expressed (or are expressed simultaneously) with their partners <a href="#pone.0001207-Han2">[10]</a>. Han <em>et al</em>. took the average PCC (or APCC) of the hubs as a measure to distinguish party hubs (with relatively high APCC) from date hubs (with relatively low APCC), where PCC is a Pearson correlation coefficient between a hub and one of its partners in microarray data. In this paper, however, we take another criterion to measure whether or not the network motifs of hubs are co-expressed. Specifically, we considered the standard deviation of average motif correlations (SAMC) of a hub (see <a href="#s4">Materials and Methods</a>), in which average motif correlation (AMC) of a network motif is the average value of Pearson correlation coefficients between two proteins connected in the network motif. If SAMC is relatively low, the network motifs of the hub are more likely expressed at the same time (or say, the expression difference among the hub's network motifs is small). Moreover, besides the difference between APCC and SAMC in considering individual partners and network motifs respectively, it seems that SAMC is not equivalent to APCC while another one, i.e. mean of average motif correlations (MAMC), is similar to APCC. Such facts were actually confirmed by the numerical experiments. That is, by numerical experiments, we found that the MAMCs of mPHs have no significant difference from those of mDHs but the SAMCs of mPHs are significantly lower than those of mDHs (see <a href="#pone-0001207-t003">Table 3</a>). Therefore, in this paper, if there is no significantly difference in the average expressions of network motifs (MAMCs) between mPHs and mDHs, it is more confident for us to conclude that the network motifs of mPHs are more likely to be co-expressed than those of mDHs depending on the significant SAMC difference between mPHs and mDHs.</p>
<div class="figure" id="pone-0001207-t003"><div class="img"><a name="pone-0001207-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Statistic significance for the differences of SAMCs between mDHs and mPHs in Microarray data</span></strong></p><span>doi:10.1371/journal.pone.0001207.t003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>At the level of network motifs instead of individual proteins, we found two types of hubs, motif party hubs (mPHs) and motif date hubs (mDHs), whose network motifs display distinct characteristics in organizing modules, cellular localizations, controlling network architecture, and co-expression in microarray data. More importantly, such a result answered the open question whether or not <em>HC<sup>fyi</sup></em> contains ‘date hubs’ and ‘party hubs’, i.e. the contradiction on ‘date hubs’ and ‘party hubs’ between works of Han <em>et al</em>. <a href="#pone.0001207-Han2">[10]</a> and Batada <em>et al</em>. <a href="#pone.0001207-Batada1">[11]</a>, at the level of network motifs. Moreover, more results on degree and cluster coefficient differences (see <a href="#pone.0001207.s004">Figure S2</a>) and GO function difference between mPHs and mDHs (see <a href="#pone.0001207.s005">Figure S3</a>) were also found (see <a href="#pone.0001207.s001">Text S1</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Although our results support the observation on ‘party hub’ and ‘date hub’ conducted by Han <em>et al</em>. <a href="#pone.0001207-Han2">[10]</a>, our analysis methods are totally different from theirs. That is, their study focus remains at the level of individual proteins (or nodes), however, our work is at the level of network motifs. The main procedures of the proposed method based on network motifs can be summarized as follows:</p>


<ol class="order">

<li>First, dividing hubs by considering how many network motifs stay together with their hubs in protein complexes;</li>

<li>Second, constructing spatial distribution of hubs by considering how many hubs' network motifs locate in the same cellular localizations as the hubs;</li>

<li>Third, controlling the network architecture of hubs by considering the important roles of network motifs in breaking down network topological structure;</li>

<li>Fourth, organizing the modules by considering the roles of network motifs in linking hubs and signal pathways;</li>

<li>Lastly, analyzing co-expression in microarray data of hubs by considering the difference among network motifs' expressions.</li>

</ol><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Why were network motifs adopted to distinguish ‘party hubs’ from ‘date hubs’ in <em>HC<sup>fyi</sup></em> in this work? One of main reasons is that we surprisingly found that all network motifs in <em>HC<sup>fyi</sup></em> occupy about 74% proteins and involve 85% interactions of <em>HC<sup>fyi</sup></em> (see <a href="#pone-0001207-g006">Figure 6</a>). In other words, from the viewpoint of network motifs, the main structure of <em>HC<sup>fyi</sup></em> is composed of network motifs rather than individual proteins. Thus, it is natural that we adopt network motifs as main elements while studying hubs. Another reason is that the appropriate size of the chosen network motifs, i.e. 3 or 4, determines their important role in characterizing both small size elements, i.e. molecules, at the ‘low level’ of a network, and large size elements, i.e. modules such as protein complexes <a href="#pone.0001207-Ito1">[25]</a>–<a href="#pone.0001207-Krogan1">[28]</a> and signal pathways <a href="#pone.0001207-Gustin1">[29]</a>, <a href="#pone.0001207-Herskowitz1">[30]</a>, at the ‘high level’ of a network. In our analysis, we did study the network from different levels <a href="#pone.0001207-Ideker1">[31]</a>. For example, in our analysis, mPHs' network motifs control a local topological structures and stay together inside protein complexes, which represents a ‘lower level’ of the network. On the other hand, mDHs' network motifs control the global topological structure and act as the connectors among signal pathways, which represent a ‘high level’ of the network. At either ‘low level’ or ‘high level’ of the network, the network motif is a suitable and essential building block or functional unit to characterize both relatively small elements, i.e. molecules, at a ‘low level’, and relatively large ones, i.e. modules, at a ‘high level’.</p>
<div class="figure" id="pone-0001207-g006"><div class="img"><a name="pone-0001207-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001207" data-uri="info:doi/10.1371/journal.pone.0001207.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001207.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001207.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001207.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The proteins and interactions in <em>HC<sup>fyi</sup></em> and in all network motifs of <em>HC<sup>fyi</sup></em> respectively.</span></strong></p><span>doi:10.1371/journal.pone.0001207.g006</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We studied the biological network of yeast from the viewpoint of network motifs in the paper, in particular stressing on their biological roles in biological networks rather than the topological structures of network motifs. Both theoretical and numerical analysis show that network motifs should not be merely considered as a connection pattern from topological structures but be considered as essential function units in organizing the modules from biological processes<a href="#pone.0001207-Balazsi1">[24]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Protein interaction data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The FYI dataset of 2491 interactions among 1375 proteins was obtained from Han <em>et al</em>. <a href="#pone.0001207-Han2">[10]</a>. The <em>HC<sup>fyi</sup></em> dataset of 3976 interactions among 1291 proteins was obtained from Batada <em>et al</em>. <a href="#pone.0001207-Batada1">[11]</a>. The methodologies to construct FYI and <em>HC<sup>fyi</sup></em> are similar. They were both based on an intersection method in which only the interactions observed at least twice are retained from various datasets. Datasets of the FYI were derived from HTP <a href="#pone.0001207-Ito1">[25]</a>, <a href="#pone.0001207-Uetz1">[26]</a>, <a href="#pone.0001207-FromontRacine1">[32]</a>–<a href="#pone.0001207-Ho1">[34]</a>, APT <a href="#pone.0001207-Ho1">[34]</a>–<a href="#pone.0001207-Bader1">[36]</a>, in silico-predicted dataset <a href="#pone.0001207-vonMering1">[37]</a>–<a href="#pone.0001207-Dandekar1">[39]</a> and MIPS <a href="#pone.0001207-Mewes1">[40]</a>. Datasets of the <em>HC<sup>fyi</sup></em> were derived from all extent protein interaction datasets, which include all LC interaction data (BioGRID <a href="#pone.0001207-Reguly1">[41]</a>, BIND <a href="#pone.0001207-Bader1">[36]</a>, DIP <a href="#pone.0001207-Xenarios1">[42]</a>, MINT <a href="#pone.0001207-Zanzoni1">[43]</a>, and MIPS <a href="#pone.0001207-Mewes1">[40]</a>), and all HTP interaction data <a href="#pone.0001207-Ito1">[25]</a>–<a href="#pone.0001207-Krogan1">[28]</a>, <a href="#pone.0001207-Ho1">[34]</a>, <a href="#pone.0001207-Gavin2">[35]</a>, <a href="#pone.0001207-Xenarios1">[42]</a>. Especially, the LC data were manually curated from over 31,793 abstracts and online publications <a href="#pone.0001207-Reguly1">[41]</a>, and there is no interaction derived from standard large-scale experiments in both FYI and <em>HC<sup>fyi</sup></em>.</p>


<h4>Protein complex, signal pathway and cellular localization data</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The protein complex data were derived from MIPS <a href="#pone.0001207-Mewes1">[40]</a> in September of 2006 and the signal pathway data were derived from KEGG <a href="#pone.0001207-Kanehisa1">[44]</a> in November of 2006. Cellular localization data were derived from Huh <em>et al</em>. <a href="#pone.0001207-Huh1">[45]</a>.</p>


<h4>Gene expression data</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The 6 microarray datasets <a href="#pone.0001207-Han2">[10]</a>, <a href="#pone.0001207-Batada1">[11]</a>, <a href="#pone.0001207-Kemmeren1">[46]</a> (Stress response <a href="#pone.0001207-Gasch1">[47]</a>, cell cycle <a href="#pone.0001207-Spellman1">[48]</a>, pheromone treatment <a href="#pone.0001207-Roberts1">[49]</a>, unfolded protein response <a href="#pone.0001207-Chu1">[50]</a>, sporulation <a href="#pone.0001207-Travers1">[51]</a> and compendium <a href="#pone.0001207-Kemmeren1">[46]</a>) were normalized with Z score normalization <a href="#pone.0001207-Quackenbush1">[52]</a> (i.e. the expression measurement for each gene was adjusted to have a mean of 0 and a standard deviation of 1) using the original log<sub>2</sub> fold change values. Compendium gene expression data are an expression-profiling compendium of 315 data points for most yeast genes across other five different experimental conditions. The PCC (Pearson correlation coefficient) of motifs were calculated for the five conditions and the combined set of all conditions (compendium).</p>


<h4>Hub</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We selected about top 20% proteins with relatively more partners, i.e. 197 proteins, in the <em>HC<sup>fyi</sup></em> network, which are defined as hubs. All of their partners are not less than 12. There are 103 hubs in the overlap of hubs of <em>HC<sup>fyi</sup></em> and FYI (199 hubs).</p>


<h4>Network Motifs detected by mfinder1.2</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In consideration that the protein-protein interaction networks are undirected, the network motifs appearing in these undirected networks are undoubtedly undirected. For three-node substructures, only one network motif, i.e. triangle or ID: 238, has been found, whose Z-score is 317.43. For four-node substructures, one network motif, i.e. square or ID: 13260, has been found and been chosen as the representative four-node network motif in our study, whose Z-score is 12.90. Indeed, for four-node network motifs, others have also been found. The reason why we chose the square four-node network motif is that the other four-node network motifs can be composed of the triangle and the square. The network motifs were found by mfinder1.2 <a href="#pone.0001207-Milo1">[17]</a>, <a href="#pone.0001207-Milo2">[18]</a>.</p>


<h4>Division of hubs into mPHs and mDHs</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>We propose a quantitative criterion to divide hubs into mPHs and mDHs in this paper. According to the quantitative criterion <em>Complex<sub>ratio-same</sub></em>, we can have a partition of hubs based on the relationship between a hub's network motifs and protein complexes. If a protein <em>H</em> is a hub in FYI, and <em>M</em> is a set of the hub's network motifs that are composed of <em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>,…, <em>M</em><sub>|<em>M</em>|</sub> (for every network motif <em>M<sub>i</sub></em>, <em>i</em>Є{1,2,…,|<em>M</em>|}, <em>H</em>Є<em>M<sub>i</sub></em> must be satisfied) , then we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.e001&amp;representation=PNG" class="inline-graphic"></span><br>where set <em>M<sub>same</sub></em> is composed of those network motifs whose three proteins or four proteins all belong to just the same protein complex. In other words, for the protein <em>P<sub>j</sub></em> in some network motif <em>M<sub>k</sub></em>, <em>j</em>Є{1,2,…,|<em>M<sub>k</sub></em>|}, the protein complex set of <em>P<sub>j</sub></em> is <em>Complex<sub>j</sub></em> that is composed of those protein complexes containing the protein <em>P<sub>j</sub></em>. If <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.e002&amp;representation=PNG" class="inline-graphic"></span>, then <em>M<sub>k</sub></em>Є<em>M<sub>same</sub></em>. |•| is the number of elements in some set. Thus, 98 hubs with relatively high <em>Complex<sub>ratio-same</sub></em> (larger than or equal to 0.5) are called mPHs, and 98 hubs with relatively low <em>Complex<sub>ratio-same</sub></em> (less than 0.5) are called mDHs (see <a href="#pone.0001207.s002">Table S1</a>).</p>


<h4>A measure for cellular localizations</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>According to the quantitative criterion <em>Localization<sub>ratio-same</sub></em> proposed in this paper, we can measure the spatial distribution of a hub's network motifs. If a protein <em>H</em> is a hub in FYI, and <em>M</em> is a set of the hub's network motifs that are composed of <em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>,…, <em>M</em><sub>|<em>M</em>|</sub>, then we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.e003&amp;representation=PNG" class="inline-graphic"></span><br> <span class="inline-formula"></span>where set <em>M<sub>same</sub></em> is composed of those network motifs whose three proteins or four proteins all locate in just the same cellular localization. In other words, for the protein <em>P<sub>j</sub></em> in some network motif <em>M<sub>k</sub></em>, <em>j</em>Є{1,2,…,|<em>M<sub>k</sub></em>|}, the cellular localization set of <em>P<sub>j</sub></em> is <em>Localization<sub>j</sub></em> that is composed of those cellular localizations of the protein <em>P<sub>j</sub></em>. If <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.e004&amp;representation=PNG" class="inline-graphic"></span> <span class="inline-formula"></span>, then <em>M<sub>k</sub></em>Є<em>M<sub>same</sub></em>. |•| is the number of elements in some set.</p>


<h4>Standard deviation of Average Motif Correlation (SAMC)</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>To analyze co-expression or study the difference in gene expression of a hub's network motifs, we define Standard deviation of Average Motif Correlation (SAMC). Average Motif Correlation (AMC) for one of a hub's network motifs is to measure the average gene expression level for the network motif. If a protein <em>H</em> is a hub in FYI, and <em>M</em> is a set of the hub's network motifs that are composed of <em>M</em><sub>1</sub>, <em>M</em><sub>2</sub>,…, <em>M</em><sub>|<em>M</em>|</sub>. For each network motif <em>M<sub>i</sub></em> (<em>i</em>Є{1,2,…,|<em>M</em>|}) that contains three proteins or four proteins as mentioned in the section about network motifs, then we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001207.e005&amp;representation=PNG" class="inline-graphic"></span><br> <span class="inline-formula"></span>where |<em>M<sub>i</sub></em>| is the number of interactions in network motif <em>M<sub>i</sub></em>, i.e. 3 or 4, and <em>P<sub>j</sub></em>, <em>P<sub>k</sub></em> are any two proteins in <em>M<sub>i</sub></em>. <em>PCC</em>(<em>P<sub>j</sub></em>, <em>P<sub>k</sub></em>) is the Pearson correlation coefficient between proteins <em>P<sub>j</sub></em> and <em>P<sub>k</sub></em>. <em>I</em> is a function defined as equal to 1 if <em>P<sub>j</sub></em>, <em>P<sub>k</sub></em> are linked in the network motif <em>M<sub>i</sub></em>, and equal to 0 if <em>P<sub>j</sub></em>, <em>P<sub>k</sub></em> are not linked in the network motif <em>M<sub>i</sub></em>.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>Thus, SAMC is the Standard deviation of all AMCs for all network motifs of a hub.</p>


<h4>Mean of Average Motif Correlation (MAMC)</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Mean of Average Motif Correlation (MAMC) is the Mean of all AMCs for all network motifs of a hub.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001207"></div><a name="pone.0001207.s001" id="pone.0001207.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001207.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Additional results on mPHs and mDHs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001207.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0001207.s002" id="pone.0001207.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001207.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">mPHs and mDHs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001207.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0001207.s003" id="pone.0001207.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001207.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The degrees of mDHs in <a href="#pone-0001207-g005">Figure 5</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001207.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.08 MB TIF)</p>
<a name="pone.0001207.s004" id="pone.0001207.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001207.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">The degree and cluster coefficient differences between mPHs and mDHs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001207.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.07 MB TIF)</p>
<a name="pone.0001207.s005" id="pone.0001207.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001207.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Function_ratio-sames of mDHs is significantly different from those of mPHs</p>
<p class="siDoi">doi:10.1371/journal.pone.0001207.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are very grateful to Jing-Dong J. Han and Nizar N Batada for helpful suggestions on collecting Microarray data, and Nizar N Batada for providing us the data of <em>HC<sup>fyi</sup></em> network.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GJ. Performed the experiments: GJ. Analyzed the data: SZ GJ. Contributed reagents/materials/analysis tools: SZ GJ. Wrote the paper: XZ SZ LC GJ. Other: Supervised whole project: LC XZ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001207-Jeong1" id="pone.0001207-Jeong1"></a>Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality and centrality in protein networks. Nature  411: 41–42.  <ul class="find" data-citedArticleID="1020410" data-doi="10.1038/35075138"><li><a href="http://dx.doi.org/10.1038/35075138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethality+and+centrality+in+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethality+and+centrality+in+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001207-Albert1" id="pone.0001207-Albert1"></a>Albert R, Jeong H, Barabasi AL (2000) Error and attack tolerance of complex networks. Nature  406: 378–382.  <ul class="find" data-citedArticleID="1020354" data-doi="10.1038/35019019"><li><a href="http://dx.doi.org/10.1038/35019019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Error+and+attack+tolerance+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Error+and+attack+tolerance+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001207-Barabasi1" id="pone.0001207-Barabasi1"></a>Barabasi AL, Albert R (1999) Emergence of scaling in random networks. Science  286: 509.  <ul class="find" data-citedArticleID="1020360" data-doi="10.1126/science.286.5439.509"><li><a href="http://dx.doi.org/10.1126/science.286.5439.509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+scaling+in+random+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+scaling+in+random+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001207-Barabasi2" id="pone.0001207-Barabasi2"></a>Barabasi AL, Oltvai ZN (2004) Network biology: understanding the cell's functional organization. Nat Rev Genet  5(2): 101–13.  <ul class="find" data-citedArticleID="1020362" data-doi="10.1038/nrg1272"><li><a href="http://dx.doi.org/10.1038/nrg1272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+biology%3A+understanding+the+cell%27s+functional+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+biology%3A+understanding+the+cell%27s+functional+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001207-Barabasi3" id="pone.0001207-Barabasi3"></a>Barabasi AL (2002) Linked: the new science of networks. Cambridge Mass: Perseus Pub.   <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001207-Przulj1" id="pone.0001207-Przulj1"></a>Przulj N, Wigle DA, Jurisica I (2004) Functional topology in a network of protein interactions. Bioinformatics.  12;20(3): 340–8.  <ul class="find" data-citedArticleID="1020434" data-doi="10.1093/bioinformatics/btg415"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+topology+in+a+network+of+protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+topology+in+a+network+of+protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001207-Stumpf1" id="pone.0001207-Stumpf1"></a>Stumpf MP, Wiuf C, May RM (2005) Subnets of scale-free networks are not scale-free: sampling properties of networks. Proc Natl Acad Sci U S A.  22;102(12): 4221–4.  <ul class="find" data-citedArticleID="1020448" data-doi="10.1073/pnas.0501179102"><li><a href="http://dx.doi.org/10.1073/pnas.0501179102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subnets+of+scale-free+networks+are+not+scale-free%3A+sampling+properties+of+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subnets+of+scale-free+networks+are+not+scale-free%3A+sampling+properties+of+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001207-Thomas1" id="pone.0001207-Thomas1"></a>Thomas A, Cannings R, Monk NA, Cannings C (2003) On the structure of protein-protein interaction networks. Biochem Soc Trans.  31(Pt 6): 1491–6.  <ul class="find" data-citedArticleID="1020450" data-doi="10.1042/bst0311491"><li><a href="http://dx.doi.org/10.1042/bst0311491" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+structure+of+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+structure+of+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001207-Han1" id="pone.0001207-Han1"></a>Han JD, Dupuy D, Bertin N, Cusick ME, Vidal M (2005) Effect of sampling on topology predictions of protein-protein interaction networks. Nat Biotechnol.  23(7): 839–44.  <ul class="find" data-citedArticleID="1020392" data-doi="10.1038/nbt1116"><li><a href="http://dx.doi.org/10.1038/nbt1116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+sampling+on+topology+predictions+of+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+sampling+on+topology+predictions+of+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001207-Han2" id="pone.0001207-Han2"></a>Han JD, Bertin N, Hao T, Goldberg DS, Berriz GF, et al.  (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. Nature  430: 88–93.  <ul class="find" data-citedArticleID="1020394" data-doi="10.1038/nature02555"><li><a href="http://dx.doi.org/10.1038/nature02555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001207-Batada1" id="pone.0001207-Batada1"></a>Batada NN, Reguly T, Breitkreutz A, Boucher L, Breitkreutz BJ, et al.  (2006) Stratus not altocumulus: A new view of the yeast protein interaction network. PLoS Biol  4(10): e317.  <ul class="find" data-citedArticleID="1020366" data-doi="10.1371/journal.pbio.0040317"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stratus+not+altocumulus%3A+A+new+view+of+the+yeast+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stratus+not+altocumulus%3A+A+new+view+of+the+yeast+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001207-Ekman1" id="pone.0001207-Ekman1"></a>Ekman D, Light S, Bjorklund AK, Elofsson A (2006) What properties characterize the hub proteins of the protein-protein interaction network of <em>Saccharomyces cerevisiae</em>? Genome Biol  7: R45.  <ul class="find" data-citedArticleID="1020376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=What+properties+characterize+the+hub+proteins+of+the+protein-protein+interaction+network+of+Saccharomyces+cerevisiae%3F&amp;auth=&amp;atitle=What+properties+characterize+the+hub+proteins+of+the+protein-protein+interaction+network+of+Saccharomyces+cerevisiae%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=What+properties+characterize+the+hub+proteins+of+the+protein-protein+interaction+network+of+Saccharomyces+cerevisiae%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22What+properties+characterize+the+hub+proteins+of+the+protein-protein+interaction+network+of+Saccharomyces+cerevisiae%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001207-He1" id="pone.0001207-He1"></a>He X, Zhang J (2006) Why do hubs tend to be essential in protein networks? PLoS Genet  2(6): 0868–0834.  <ul class="find" data-citedArticleID="1020398" data-doi="10.1371/journal.pgen.0020088.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020088.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+do+hubs+tend+to+be+essential+in+protein+networks%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+do+hubs+tend+to+be+essential+in+protein+networks%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001207-Batada2" id="pone.0001207-Batada2"></a>Batada NN, Hurst LD, Tyers M (2006) Evolutionary and physiological importance of hub proteins. PLoS Comput Biol  2(7): e88.  <ul class="find" data-citedArticleID="1020368" data-doi="10.1371/journal.pcbi.0020088.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020088.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+and+physiological+importance+of+hub+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+and+physiological+importance+of+hub+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001207-Maslov1" id="pone.0001207-Maslov1"></a>Maslov S, Sneppen K (2002) Specificity and stability in topology of protein networks. Science  296: 910–913.  <ul class="find" data-citedArticleID="1020426" data-doi="10.1126/science.1065103"><li><a href="http://dx.doi.org/10.1126/science.1065103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+and+stability+in+topology+of+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+and+stability+in+topology+of+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001207-Hartwell1" id="pone.0001207-Hartwell1"></a>Hartwell LH, Hopfield JJ, Leibler S, Murray AW (1999) From molecular to modular cell biology. Nature  402: C47–C52.  <ul class="find" data-citedArticleID="1020396" data-doi="10.1038/35011540"><li><a href="http://dx.doi.org/10.1038/35011540" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+molecular+to+modular+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+molecular+to+modular+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001207-Milo1" id="pone.0001207-Milo1"></a>Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, et al.  (2002) Network Motifs: Simple Building Blocks of Complex Networks. Science  298: 824–827.  <ul class="find" data-citedArticleID="1020430" data-doi="10.1126/science.298.5594.824"><li><a href="http://dx.doi.org/10.1126/science.298.5594.824" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+Motifs%3A+Simple+Building+Blocks+of+Complex+Networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+Motifs%3A+Simple+Building+Blocks+of+Complex+Networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001207-Milo2" id="pone.0001207-Milo2"></a>Milo R, Itzkovitz S, Kashtan N, Levitt R, Shen-Orr S, et al.  (2004) Superfamilies of designed and evolved networks. Science  303: 1538–42.  <ul class="find" data-citedArticleID="1020432" data-doi="10.1126/science.1089167"><li><a href="http://dx.doi.org/10.1126/science.1089167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Superfamilies+of+designed+and+evolved+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Superfamilies+of+designed+and+evolved+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001207-ShenOrr1" id="pone.0001207-ShenOrr1"></a>Shen-Orr S, Milo R, Mangan S, Alon U (2002) Network motifs in the transcriptional regulation network of Escherichia coli. Nat Genet  31: 64–68.  <ul class="find" data-citedArticleID="1020442" data-doi="10.1038/ng881"><li><a href="http://dx.doi.org/10.1038/ng881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+motifs+in+the+transcriptional+regulation+network+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+motifs+in+the+transcriptional+regulation+network+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001207-Mangan1" id="pone.0001207-Mangan1"></a>Mangan S, Itzkovitz S, Zaslaver A, Alon U (2006) The Incoherent Feed-forward Loop Accelerates the Response-time of the gal System of Escherichia coli. JMB  356: 1073–81.  <ul class="find" data-citedArticleID="1020420" data-doi="10.1016/j.jmb.2005.12.003"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.12.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Incoherent+Feed-forward+Loop+Accelerates+the+Response-time+of+the+gal+System+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Incoherent+Feed-forward+Loop+Accelerates+the+Response-time+of+the+gal+System+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001207-Mangan2" id="pone.0001207-Mangan2"></a>Mangan S, Zaslaver A, Alon U (2003) The Coherent Feedforward Loop Serves as a Sign-sensitive Delay Element in Transcription Networks. JMB 334/  2: 197–204.  <ul class="find" data-citedArticleID="1020422" data-doi="10.1016/j.jmb.2003.09.049"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.09.049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Coherent+Feedforward+Loop+Serves+as+a+Sign-sensitive+Delay+Element+in+Transcription+Networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Coherent+Feedforward+Loop+Serves+as+a+Sign-sensitive+Delay+Element+in+Transcription+Networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001207-Li1" id="pone.0001207-Li1"></a>Li C, Chen L, Aihara K (2007) A systems biology perspective on signal processing in genetic motifs. IEEE Signal Proc Mag,  24: 136–147.  <ul class="find" data-citedArticleID="1020418" data-doi="10.1109/msp.2007.323279"><li><a href="http://dx.doi.org/10.1109/msp.2007.323279" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systems+biology+perspective+on+signal+processing+in+genetic+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systems+biology+perspective+on+signal+processing+in+genetic+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001207-Sole1" id="pone.0001207-Sole1"></a>Sole RV, Valverde S (2006) Are network motifs the spandrels of cellular complexity? Trends Ecol Evol  21: 419–422.  <ul class="find" data-citedArticleID="1020444" data-doi="10.1016/j.tree.2006.05.013"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.05.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+network+motifs+the+spandrels+of+cellular+complexity%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+network+motifs+the+spandrels+of+cellular+complexity%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001207-Balazsi1" id="pone.0001207-Balazsi1"></a>Balazsi G, Barabasi AL, Oltvai ZN (2005) Topological units of environmental signal processing in the transcriptional regulatory network of Escherichia coli. Proc Natl Acad Sci U S A.  31;102(22): 7841–6.  <ul class="find" data-citedArticleID="1020358" data-doi="10.1073/pnas.0500365102"><li><a href="http://dx.doi.org/10.1073/pnas.0500365102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topological+units+of+environmental+signal+processing+in+the+transcriptional+regulatory+network+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topological+units+of+environmental+signal+processing+in+the+transcriptional+regulatory+network+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001207-Ito1" id="pone.0001207-Ito1"></a>Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, et al.  (2001) A comprehensive two-hybrid analysis to explore the yeast protein interactome. Proc Natl Acad Sci U S A  98: 4569–4574.  <ul class="find" data-citedArticleID="1020408" data-doi="10.1073/pnas.061034498"><li><a href="http://dx.doi.org/10.1073/pnas.061034498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+two-hybrid+analysis+to+explore+the+yeast+protein+interactome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+two-hybrid+analysis+to+explore+the+yeast+protein+interactome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001207-Uetz1" id="pone.0001207-Uetz1"></a>Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, et al.  (2000) A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae. Nature  403: 623–627.  <ul class="find" data-citedArticleID="1020454" data-doi="10.1038/35001009"><li><a href="http://dx.doi.org/10.1038/35001009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001207-Gavin1" id="pone.0001207-Gavin1"></a>Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, et al.  (2006) Proteome survey reveals modularity of the yeast cell machinery. Nature  440: 631–637.  <ul class="find" data-citedArticleID="1020386" data-doi="10.1038/nature04532"><li><a href="http://dx.doi.org/10.1038/nature04532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome+survey+reveals+modularity+of+the+yeast+cell+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome+survey+reveals+modularity+of+the+yeast+cell+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001207-Krogan1" id="pone.0001207-Krogan1"></a>Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, et al.  (2006) Global landscape of protein complexes in the yeast Saccharomyces cerevisiae. Nature  440: 637–644.  <ul class="find" data-citedArticleID="1020416" data-doi="10.1038/nature04670"><li><a href="http://dx.doi.org/10.1038/nature04670" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+landscape+of+protein+complexes+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+landscape+of+protein+complexes+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001207-Gustin1" id="pone.0001207-Gustin1"></a>Gustin MC, Albertyn J, Alexander M, Davenport K (1998) MAP kinase pathways in the yeast Saccharomyces cerevisiae. Microbiol Mol Biol Rev  62(4): 1264–300.  <ul class="find" data-citedArticleID="1020390"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MAP+kinase+pathways+in+the+yeast+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=MAP+kinase+pathways+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAP+kinase+pathways+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAP+kinase+pathways+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001207-Herskowitz1" id="pone.0001207-Herskowitz1"></a>Herskowitz I (1995) MAP kinase pathways in yeast: for mating and more. Cell  80(2): 187–97.  <ul class="find" data-citedArticleID="1020400" data-doi="10.1016/0092-8674(95)90402-6"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90402-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAP+kinase+pathways+in+yeast%3A+for+mating+and+more." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAP+kinase+pathways+in+yeast%3A+for+mating+and+more.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001207-Ideker1" id="pone.0001207-Ideker1"></a>Ideker T, Lauffenburger D (2003) Building with a scaffold: emerging strategies for high- to low-level cellular modeling. Trends Biotechnol  21: 255–262.  <ul class="find" data-citedArticleID="1020406" data-doi="10.1016/s0167-7799(03)00115-x"><li><a href="http://dx.doi.org/10.1016/s0167-7799(03)00115-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Building+with+a+scaffold%3A+emerging+strategies+for+high-+to+low-level+cellular+modeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Building+with+a+scaffold%3A+emerging+strategies+for+high-+to+low-level+cellular+modeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001207-FromontRacine1" id="pone.0001207-FromontRacine1"></a>Fromont-Racine M, Mayes AE, Brunet-Simon A, Rain JC, Colley A, et al.  (2000) Genome-wide protein interaction screens reveal functional networks involving Sm-like proteins. Yeast  17: 95–110.  <ul class="find" data-citedArticleID="1020380"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome-wide+protein+interaction+screens+reveal+functional+networks+involving+Sm-like+proteins.&amp;auth=&amp;atitle=Genome-wide+protein+interaction+screens+reveal+functional+networks+involving+Sm-like+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+protein+interaction+screens+reveal+functional+networks+involving+Sm-like+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+protein+interaction+screens+reveal+functional+networks+involving+Sm-like+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001207-FromontRacine2" id="pone.0001207-FromontRacine2"></a>Fromont-Racine M, Rain JC, Legrain P (1997) Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens. Nature Genet  16: 277–282.  <ul class="find" data-citedArticleID="1020382" data-doi="10.1038/ng0797-277"><li><a href="http://dx.doi.org/10.1038/ng0797-277" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+functional+analysis+of+the+yeast+genome+through+exhaustive+two-hybrid+screens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+functional+analysis+of+the+yeast+genome+through+exhaustive+two-hybrid+screens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001207-Ho1" id="pone.0001207-Ho1"></a>Ho Y, Gruhler A, Heilbut A, Bader GD, Moore L, et al.  (2002) Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry. Nature  415: 180–183.  <ul class="find" data-citedArticleID="1020402" data-doi="10.1038/415180a"><li><a href="http://dx.doi.org/10.1038/415180a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+identification+of+protein+complexes+in+Saccharomyces+cerevisiae+by+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+identification+of+protein+complexes+in+Saccharomyces+cerevisiae+by+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001207-Gavin2" id="pone.0001207-Gavin2"></a>Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, et al.  (2002) Functional organization of the yeast proteome by systematic analysis of protein complexes. Nature  415: 141–147.  <ul class="find" data-citedArticleID="1020388" data-doi="10.1038/415141a"><li><a href="http://dx.doi.org/10.1038/415141a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+organization+of+the+yeast+proteome+by+systematic+analysis+of+protein+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+organization+of+the+yeast+proteome+by+systematic+analysis+of+protein+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001207-Bader1" id="pone.0001207-Bader1"></a>Bader GD, Betel D, Hogue CW (2003) BIND: the Biomolecular Interaction Network Database. Nucleic Acids Res  31: 248–250.  <ul class="find" data-citedArticleID="1020356" data-doi="10.1093/nar/gkg056"><li><a href="http://dx.doi.org/10.1093/nar/gkg056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BIND%3A+the+Biomolecular+Interaction+Network+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BIND%3A+the+Biomolecular+Interaction+Network+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001207-vonMering1" id="pone.0001207-vonMering1"></a>von Mering C, Krause R, Snel B, Cornell M, Oliver SG, et al.  (2002) Comparative assessment of large-scale data sets of proteinCprotein interactions. Nature  417: 399–403.  <ul class="find" data-citedArticleID="1020460" data-doi="10.1038/nature750"><li><a href="http://dx.doi.org/10.1038/nature750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+assessment+of+large-scale+data+sets+of+proteinCprotein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+assessment+of+large-scale+data+sets+of+proteinCprotein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001207-Marcotte1" id="pone.0001207-Marcotte1"></a>Marcotte EM, Pellegrini M, Thompson MJ, Yeates TO, Eisenberg D (1999) A combined algorithm for genome-wide prediction of protein function. Nature  402: 83–86.  <ul class="find" data-citedArticleID="1020424" data-doi="10.1038/47048"><li><a href="http://dx.doi.org/10.1038/47048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+combined+algorithm+for+genome-wide+prediction+of+protein+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+combined+algorithm+for+genome-wide+prediction+of+protein+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001207-Dandekar1" id="pone.0001207-Dandekar1"></a>Dandekar T, Snel B, Huynen M, Bork P (1998) Conservation of gene order: a fingerprint of proteins that physically interact. Trends Biochem Sci  23: 324–328.  <ul class="find" data-citedArticleID="1020374" data-doi="10.1016/s0968-0004(98)01274-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01274-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+of+gene+order%3A+a+fingerprint+of+proteins+that+physically+interact." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+of+gene+order%3A+a+fingerprint+of+proteins+that+physically+interact.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001207-Mewes1" id="pone.0001207-Mewes1"></a>Mewes HW, Frishman D, Guldener U, Mannhaupt G, Mayer K, et al.  (2002) MIPS: a database for genomes and protein sequences. Nucleic Acids Res  30: 31–34.  <ul class="find" data-citedArticleID="1020428" data-doi="10.1093/nar/30.1.31"><li><a href="http://dx.doi.org/10.1093/nar/30.1.31" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MIPS%3A+a+database+for+genomes+and+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MIPS%3A+a+database+for+genomes+and+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001207-Reguly1" id="pone.0001207-Reguly1"></a>Reguly T, Breitkreutz A, Boucher L, Breitkreutz BJ, Hon GC, et al.  (2006) Comprehensive curation and analysis of global interaction networks in <em>Saccharomyces cerevisiae</em>. J Biol  5: 11.  <ul class="find" data-citedArticleID="1020438"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+curation+and+analysis+of+global+interaction+networks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001207-Xenarios1" id="pone.0001207-Xenarios1"></a>Xenarios I, Rice DW, Salwinski L, Baron MK, Marcotte EM, et al.  (2000) DIP: The database of interacting proteins. Nucleic Acids Res  28: 289–291.  <ul class="find" data-citedArticleID="1020456" data-doi="10.1093/nar/28.1.289"><li><a href="http://dx.doi.org/10.1093/nar/28.1.289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DIP%3A+The+database+of+interacting+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DIP%3A+The+database+of+interacting+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001207-Zanzoni1" id="pone.0001207-Zanzoni1"></a>Zanzoni A, Montecchi-Palazzi L, Quondam M, Ausiello G, Helmer-Citterich M, et al.  (2002) MINT: a Molecular INTeraction database. FEBS Lett.  20;513(1): 135–40.  <ul class="find" data-citedArticleID="1020458" data-doi="10.1016/s0014-5793(01)03293-8"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)03293-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MINT%3A+a+Molecular+INTeraction+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MINT%3A+a+Molecular+INTeraction+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001207-Kanehisa1" id="pone.0001207-Kanehisa1"></a>Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, et al.  (2006) From genomics to chemical genomics: new devel-opments in KEGG. Nucleic Acids Res  34: D354–7.  <ul class="find" data-citedArticleID="1020412" data-doi="10.1093/nar/gkj102"><li><a href="http://dx.doi.org/10.1093/nar/gkj102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomics+to+chemical+genomics%3A+new+devel-opments+in+KEGG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomics+to+chemical+genomics%3A+new+devel-opments+in+KEGG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001207-Huh1" id="pone.0001207-Huh1"></a>Huh WK, Falvo JV, Gerke LC, Carroll AS, Howson RW, et al.  (2003) Global analysis of protein localization in budding yeast. Nature  425: 686–691.  <ul class="find" data-citedArticleID="1020404" data-doi="10.1038/nature02026"><li><a href="http://dx.doi.org/10.1038/nature02026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+protein+localization+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+protein+localization+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001207-Kemmeren1" id="pone.0001207-Kemmeren1"></a>Kemmeren P, van Berkum NL, Vilo J, Bijma T, Donders R, et al.  (2002) Protein interaction verification and functional annotation by integrated analysis of genome-scale data. Mol Cell  9: 1133–1143.  <ul class="find" data-citedArticleID="1020414" data-doi="10.1016/s1097-2765(02)00531-2"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00531-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+interaction+verification+and+functional+annotation+by+integrated+analysis+of+genome-scale+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+interaction+verification+and+functional+annotation+by+integrated+analysis+of+genome-scale+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001207-Gasch1" id="pone.0001207-Gasch1"></a>Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al.  (2000) Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Evol  11: 4241–4257.  <ul class="find" data-citedArticleID="1020384" data-doi="10.1091/mbc.11.12.4241"><li><a href="http://dx.doi.org/10.1091/mbc.11.12.4241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001207-Spellman1" id="pone.0001207-Spellman1"></a>Spellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, et al.  (1998) Comprehensive identification of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization. Mol Biol Evol  9: 3273–3297.  <ul class="find" data-citedArticleID="1020446" data-doi="10.1091/mbc.9.12.3273"><li><a href="http://dx.doi.org/10.1091/mbc.9.12.3273" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+identification+of+cell+cycle-regulated+genes+of+the+yeast+Saccharomyces+cerevisiae+by+microarray+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+identification+of+cell+cycle-regulated+genes+of+the+yeast+Saccharomyces+cerevisiae+by+microarray+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001207-Roberts1" id="pone.0001207-Roberts1"></a>Roberts CJ, Nelson B, Marton MJ, Stoughton R, Meyer MR, et al.  (2000) Signaling and circuitry of multiple MAPK pathways revealed by a matrix of global gene expression profiles. Science  287: 873–880.  <ul class="find" data-citedArticleID="1020440" data-doi="10.1126/science.287.5454.873"><li><a href="http://dx.doi.org/10.1126/science.287.5454.873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+and+circuitry+of+multiple+MAPK+pathways+revealed+by+a+matrix+of+global+gene+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+and+circuitry+of+multiple+MAPK+pathways+revealed+by+a+matrix+of+global+gene+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001207-Chu1" id="pone.0001207-Chu1"></a>Chu S, DeRisi J, Eisen M, Mulholland J, Botstein D, et al.  (1998) The transcriptional program of sporulation in budding yeast. Science  282: 699–705.  <ul class="find" data-citedArticleID="1020372" data-doi="10.1126/science.282.5389.699"><li><a href="http://dx.doi.org/10.1126/science.282.5389.699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+program+of+sporulation+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+program+of+sporulation+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001207-Travers1" id="pone.0001207-Travers1"></a>Travers KJ, Patil CK, Wodicka L, Lockhart DJ, Weissman JS, et al.  (2000) Functional and genomic analyses reveal an essential coordination between the unfolded protein response and ERassociated degradation. Cell  101: 249–258.  <ul class="find" data-citedArticleID="1020452" data-doi="10.1016/s0092-8674(00)80835-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80835-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+and+genomic+analyses+reveal+an+essential+coordination+between+the+unfolded+protein+response+and+ERassociated+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+and+genomic+analyses+reveal+an+essential+coordination+between+the+unfolded+protein+response+and+ERassociated+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001207-Quackenbush1" id="pone.0001207-Quackenbush1"></a>Quackenbush J (2002) Microarray data normalization and transformation. Nat Genet  32: Suppl496–501.  <ul class="find" data-citedArticleID="1020436" data-doi="10.1038/ng1032"><li><a href="http://dx.doi.org/10.1038/ng1032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+data+normalization+and+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+data+normalization+and+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001207-Enright1" id="pone.0001207-Enright1"></a>Enright AJ, Ouzounis CA (2001) BioLayout-an automatic graph layout algorithm for similarity visualization. Bioinformatics  17(9): 853–854.  <ul class="find" data-citedArticleID="1020378" data-doi="10.1093/bioinformatics/17.9.853"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.853" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BioLayout-an+automatic+graph+layout+algorithm+for+similarity+visualization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BioLayout-an+automatic+graph+layout+algorithm+for+similarity+visualization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001207-Bernard1" id="pone.0001207-Bernard1"></a>Bernard R (1986) Fundamentals of Biostatistic. Boston Mass: Duxbury Press.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.XML" value="101820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.PDF" value="1400376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g001.PNG_L" value="14466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g001.PNG_M" value="30304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g001.PNG_S" value="7870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g001.TIF" value="63928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g001.PNG_I" value="10099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g002.PNG_L" value="25878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g002.PNG_M" value="67239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g002.PNG_S" value="11501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g002.TIF" value="99600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g002.PNG_I" value="23518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g003.PNG_L" value="279536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g003.PNG_M" value="128520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g003.PNG_S" value="15636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g003.TIF" value="678494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g003.PNG_I" value="78138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g004.PNG_L" value="394292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g004.PNG_M" value="224493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g004.PNG_S" value="17356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g004.TIF" value="604742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g004.PNG_I" value="76831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t001.PNG_L" value="48479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t001.PNG_M" value="67730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t001.PNG_S" value="12126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t001.TIF" value="214176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t001.PNG_I" value="21626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g005.PNG_L" value="1652055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g005.PNG_M" value="517904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g005.PNG_S" value="16429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g005.TIF" value="2034836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g005.PNG_I" value="154657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t002.PNG_L" value="33127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t002.PNG_M" value="40886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t002.PNG_S" value="9115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t002.TIF" value="143150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t002.PNG_I" value="13957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t003.PNG_L" value="89185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t003.PNG_M" value="53865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t003.PNG_S" value="9328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t003.TIF" value="415882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.t003.PNG_I" value="16418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g006.PNG_L" value="21803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g006.PNG_M" value="27562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g006.PNG_S" value="7708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g006.TIF" value="134936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.g006.PNG_I" value="9233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e001.PNG" value="9486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e001.TIF" value="25424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e002.PNG" value="8909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e002.TIF" value="25076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e003.PNG" value="9786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e003.TIF" value="25724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e004.PNG" value="9401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e004.TIF" value="25648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e005.PNG" value="13448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.e005.TIF" value="29748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.s001.PDF" value="108745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.s002.PDF" value="16333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.s003.TIF" value="82424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.s004.TIF" value="79146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001207.s005.TIF" value="38290"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001207&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001207" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001207&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001207&volume=&issue=&title=Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast&author_name=Guangxu%20Jin%2C%20Shihua%20Zhang%2C%20Xiang-Sun%20Zhang%2C%20Luonan%20Chen&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001207" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207&amp;t=Hubs%20with%20Network%20Motifs%20Organize%20Modularity%20Dynamically%20in%20the%20Protein-Protein%20Interaction%20Network%20of%20Yeast" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207&title=Hubs%20with%20Network%20Motifs%20Organize%20Modularity%20Dynamically%20in%20the%20Protein-Protein%20Interaction%20Network%20of%20Yeast&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207&amp;title=Hubs%20with%20Network%20Motifs%20Organize%20Modularity%20Dynamically%20in%20the%20Protein-Protein%20Interaction%20Network%20of%20Yeast" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001207&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Hubs with Network Motifs Organize Modularity Dynamically in the Protein-Protein Interaction Network of Yeast';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001207';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Hubs%20with%20Network%20Motifs%20Organize%20Modularity%20Dynamically%20in%20the%20Protein-Protein%20Interaction%20Network%20of%20Yeast http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001207" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001207" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Han+Chinese%22" title="Search for articles in the subject area:'Han Chinese'"><div class="flagText">Han Chinese</div></a>
              <div data-categoryid="40115" data-articleid="25246"
                   data-categoryname="Han Chinese"
                   class="flagImage" title="Flag 'Han Chinese' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Network+motifs%22" title="Search for articles in the subject area:'Network motifs'"><div class="flagText">Network motifs</div></a>
              <div data-categoryid="18085" data-articleid="25246"
                   data-categoryname="Network motifs"
                   class="flagImage" title="Flag 'Network motifs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+complexes%22" title="Search for articles in the subject area:'Protein complexes'"><div class="flagText">Protein complexes</div></a>
              <div data-categoryid="17297" data-articleid="25246"
                   data-categoryname="Protein complexes"
                   class="flagImage" title="Flag 'Protein complexes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="31803" data-articleid="25246"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+networks%22" title="Search for articles in the subject area:'Protein structure networks'"><div class="flagText">Protein structure networks</div></a>
              <div data-categoryid="32559" data-articleid="25246"
                   data-categoryname="Protein structure networks"
                   class="flagImage" title="Flag 'Protein structure networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signaling+networks%22" title="Search for articles in the subject area:'Signaling networks'"><div class="flagText">Signaling networks</div></a>
              <div data-categoryid="19723" data-articleid="25246"
                   data-categoryname="Signaling networks"
                   class="flagImage" title="Flag 'Signaling networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Spatial+distribution%22" title="Search for articles in the subject area:'Spatial distribution'"><div class="flagText">Spatial distribution</div></a>
              <div data-categoryid="31855" data-articleid="25246"
                   data-categoryname="Spatial distribution"
                   class="flagImage" title="Flag 'Spatial distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="25246"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5045'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3517'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1967&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=22285">Referee Comments: Referee 1 (G&aacute;bor Bal&aacute;zsi) </a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
