

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001980"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001980" />

    <meta name="citation_title" content="The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms"/>
    <meta itemprop="name" content="The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms"/>

      <meta name="citation_author" content="Ron M. Kerkhoven"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Daoud Sie"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Marja Nieuwland"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Mike Heimerikx"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Jorma De Ronde"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Wim Brugman"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Arno Velds"/>
            <meta name="citation_author_institution" content="Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands"/>

    <meta name="citation_date" content="2008/4/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001980.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1980"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Amplified RNA synthesized from limited quantities of heterogeneous cDNA.; citation_author=RN Van Gelder; citation_author=ME von Zastrow; citation_author=A Yool; citation_author=WC Dement; citation_author=JD Barchas; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=87(5); citation_number=1; citation_pages=1663-1667; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Global amplification of mRNA by template-switching PCR: linearity and application to microarray analysis.; citation_author=L Petalidis; citation_author=S Bhattacharyya; citation_author=GA Morris; citation_author=VP Collins; citation_author=TC Freeman; citation_journal_title=Nucleic Acids Res; citation_volume=31(22); citation_number=2; citation_pages=e142; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-fidelity mRNA amplification for gene profiling.; citation_author=E Wang; citation_author=LD Miller; citation_author=GA Ohnmacht; citation_author=ET Liu; citation_author=FM Marincola; citation_journal_title=Nat Biotechnol; citation_volume=18(4); citation_number=3; citation_pages=457-459; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Three microarray platforms: an analysis of their concordance in profiling gene expression.; citation_author=D Petersen; citation_author=GV Chandramouli; citation_author=J Geoghegan; citation_author=J Hilburn; citation_author=J Paarlberg; citation_journal_title=BMC Genomics; citation_volume=6(1); citation_number=4; citation_pages=63; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A sequence-oriented comparison of gene expression measurements across different hybridization-based technologies.; citation_author=WP Kuo; citation_author=F Liu; citation_author=J Trimarchi; citation_author=C Punzo; citation_author=M Lombardi; citation_journal_title=Nat Biotechnol; citation_volume=24(7); citation_number=5; citation_pages=832-840; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The MicroArray Quality Control (MAQC) project shows inter- and intraplatform reproducibility of gene expression measurements.; citation_author=L Shi; citation_author=LH Reid; citation_author=WD Jones; citation_author=R Shippy; citation_journal_title=Nat Biotechnol; citation_volume=24(9); citation_number=6; citation_pages=1151-1161; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Quantitative analysis of mRNA amplification by in vitro transcription.; citation_author=LR Baugh; citation_author=AA Hill; citation_author=EL Brown; citation_author=CP Hunter; citation_journal_title=Nucleic Acids Res; citation_volume=29(5); citation_number=7; citation_pages=E29; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiles of laser-captured adjacent neuronal subtypes.; citation_author=L Luo; citation_author=RC Salunga; citation_author=H Guo; citation_author=A Bittner; citation_author=KC Joy; citation_journal_title=Nat Med; citation_volume=5(1); citation_number=8; citation_pages=117-122; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Preventing nondesired RNA-primed RNA extension catalyzed by T7 RNA polymerase.; citation_author=GA Nacheva; citation_author=A Berzal-Herranz; citation_journal_title=Eur J Biochem; citation_volume=270(7); citation_number=9; citation_pages=1458-1465; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=R R_Development_Core_Team; citation_title=A Language and Environment for Statistical Computing, R.F.f.S.; citation_number=10; citation_date=2007; citation_publisher=Computing.; " />
      <meta name="citation_reference" content="citation_title=A gene-expression signature as a predictor of survival in breast cancer.; citation_author=MJ van de Vijver; citation_author=YD He; citation_author=LJ van't Veer; citation_author=H Dai; citation_author=AA Hart; citation_journal_title=N Engl J Med; citation_volume=347(25); citation_number=11; citation_pages=1999-2009; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Empirical establishment of oligonucleotide probe design criteria.; citation_author=Z He; citation_author=L Wu; citation_author=X Li; citation_author=MW Fields; citation_author=J Zhou; citation_journal_title=Appl Environ Microbiol; citation_volume=71(7); citation_number=12; citation_pages=3753-3760; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms"/>
    <meta name="twitter:description" content="Eberwine(-like) amplification of mRNA adds distinct 6&ndash;10 bp nucleotide stretches to the 5&prime; end of amplified RNA transcripts. Analysis of over six thousand microarrays reveals that probes containing motifs complementary to these stretches are associated with aberrantly high signals up to a hundred fold the signal observed in unaffected probes. This is not observed when total RNA is used as target source. Different T7 primer sequences are used in different laboratories and platforms and consequently different T7 primer bias is observed in different datasets. This will hamper efforts to compare data sets across platforms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001980.g005"/>

  <meta property="og:title" content="The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1274'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2803'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4621&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001980">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001980" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001980&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ron M. Kerkhoven
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:r.kerkhoven@nki.nl">r.kerkhoven@nki.nl</a></p>

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daoud Sie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marja Nieuwland, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mike Heimerikx, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jorma De Ronde, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wim Brugman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arno Velds
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Central Microarray Facility, The Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001980</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001980-g001" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001980-g002" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001980-t001" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001980-g003" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001980-g004" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001980-g005" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001980">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001980" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Eberwine(-like) amplification of mRNA adds distinct 6–10 bp nucleotide stretches to the 5′ end of amplified RNA transcripts. Analysis of over six thousand microarrays reveals that probes containing motifs complementary to these stretches are associated with aberrantly high signals up to a hundred fold the signal observed in unaffected probes. This is not observed when total RNA is used as target source. Different T7 primer sequences are used in different laboratories and platforms and consequently different T7 primer bias is observed in different datasets. This will hamper efforts to compare data sets across platforms.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kerkhoven RM, Sie D, Nieuwland M, Heimerikx M, De Ronde J, et al.  (2008) The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms. PLoS ONE 3(4):
          e1980.
            doi:10.1371/journal.pone.0001980</p><p><strong>Editor: </strong>Janet Kelso, Max Planck Institute for Evolutionary Anthropology, Germany</p><p><strong>Received:</strong> November 27, 2007; <strong>Accepted:</strong> February 28, 2008; <strong>Published:</strong> April 23, 2008</p><p><strong>Copyright:</strong> © 2008 Kerkhoven et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Amplification of messenger RNA is widely accepted as the method of choice to acquire sufficient copies of mRNA transcripts to enable microarray expression analysis of experimental samples <a href="#pone.0001980-VanGelder1">[1]</a>, <a href="#pone.0001980-Petalidis1">[2]</a>. The method is robust, very reproducible and it yields high quality antisense RNA material which can be labeled with fluorescent dyes and hybridized to sense oligonucleotide microarrays. The differences in microarray data introduced by amplification are minimal <a href="#pone.0001980-Wang1">[3]</a>, and are outweighed by the benefits of the amplification technique. When RNA sources are sufficiently available some laboratories prefer to hybridize un-amplified labeled RNA to microarrays <a href="#pone.0001980-Petersen1">[4]</a>. Besides the mentioned discrepancies between total RNA and amplified RNA, additional variation in results is observed when the same amplified RNA sample is analyzed on different microarray platforms <a href="#pone.0001980-Kuo1">[5]</a>, <a href="#pone.0001980-MAQC1">[6]</a>. Although the results are largely overlapping they are not completely identical and the reasons for this remain unclear. We have focused on resolving experimental bias associated with the T7 amplification protocol and the T7-primer sequence in particular. We assumed that the precise sequence of the T7 primer is important in this respect because the complete T7-primer is incorporated in the first strand cDNA generated by an RT enzyme upon annealing to the poly-A tail of mRNA extracted from samples.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Many versions of original “Eberwine” T7-primer sequence have been reported over the last decade <a href="#pone.0001980-Baugh1">[7]</a>, <a href="#pone.0001980-Luo1">[8]</a>. Some of these variations concern the length of the T stretch (ranging from 11–24 bases) or the use of one or more “anchor” nucleotides to direct the primer to the start of the poly-A tail of the mRNA as was applied by Agilent. A common feature of all of the T7 primers is the core T7-binding domain which is essential for the first interaction with the RNA polymerase and the start of transcription. The bases flanking the core T7 site are spacer sequences (<a href="#pone-0001980-g001">fig 1A</a>) that have been modified in attempts to improve the binding of the enzyme and the efficiency of the enzymatic reactions <a href="#pone.0001980-Nacheva1">[9]</a>. These spacer sequences have been introduced into the cDNA for different reasons. The 5′ spacer is introduced to avoid the T7 site to be distally located on the primer whereas the 3′ spacer in the original Eberwine primer is a reflection of the viral T7 sequence at that position which is thought to be important in the polymerase binding <a href="#pone.0001980-Baugh1">[7]</a>. The general opinion today is that this spacer constitutes no more than a separator between the T7 site and the oligo-d(T) stretch.</p>
<div class="figure" id="pone-0001980-g001"><div class="img"><a name="pone-0001980-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic representation of the amplification of mRNA.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>A. Annealing of the oligo d(T) stretch of the Affymetrix T7 primer (blue) to the poly A tail of a mRNA molecule (black). Indicated are the location of the core T7 RNA polymerase binding site, the 5′ and 3′ spacer sequences flanking it and the start of reverse transcription (red) by the RT enzyme that produces the first stand cDNA incorporating the complete T7 primer sequence. B. Transcription start (blue arrow) directly 3′ from the RNA polymerase binding domain generates antisense RNA copies that have the 3′ spacer sequence of the T7 primer as a leader sequence at their 5′ side. Between this leader sequence and the gene sequences the copies have a stretch of uridine nucleotides that originate from the poly A tail of the mRNA.</p>
<span>doi:10.1371/journal.pone.0001980.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We set out to investigate the consequence of the incorporation of the 3′ spacer sequence since in the final step in the amplification procedure it is transcribed by T7 RNA polymerase. As a consequence it becomes a leader sequence in all copies of amplified RNA (<a href="#pone-0001980-g001">fig 1B</a>). The 5′ spacer sequence is not present in amplification reaction products. Interestingly, different platforms have adopted different 3′ spacer sequences in their T7 primers. It is our hypothesis that these T7 spacer sequences may indeed cause hybridization artifacts on those probes containing complementary sequences to the spacer sequence (T7 3′ spacer motifs). We moreover suggest that variations in T7 spacer sequences introduce platform specific bias that may hamper the data comparison between platforms.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Statistical software</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Statistics tests were run using the R-project software package <a href="#pone.0001980-RDevelopmentCoreTeam1">[10]</a>.</p>


<h4>Datasets</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To study the effect of the presence of motifs in a probe sequence we analyzed three data sources all using amplified RNA (I,II and III) and one data source using total RNA (IV):</p>


<ol class="alpha-upper">

<li><strong><span class="underline">“Operon (a)”</span></strong> The compendium of experiments present in the database of the Central Microarray Facility of the Netherlands Cancer Institute (data table can be found in <a href="#pone.0001980.s001">Table S1</a> and here: <a href="http://microarrays.nki.nl/research/T7/">http://microarrays.nki.nl/research/T7/</a>) comprising a total of 6657 human full genome Operon (version 3.0) oligonucleotide microarrays (<a href="http://microarrays.nki.nl/services/oligo.html">http://microarrays.nki.nl/services/oligo​.html</a>).</li>

<li><strong><span class="underline">“Agilent (a)”</span></strong> Data obtained from a set of 295 human tumors analysed on 25 K oligonucleotide microarrays from Agilent/Rosetta <a href="#pone.0001980-vandeVijver1">[11]</a>.</li>

<li><strong><span class="underline">“Affymetrix”, “Agilent (b)”, “Illumina” and “Operon (b)”</span></strong> Datasets studied by the Microarray Quality Control project <a href="#pone.0001980-MAQC1">[6]</a> containing microarray data sets obtained with amplified human RNA on the Affymetrix U133 Plus 2.0 arrays, the Agilent whole genome oligo microarrays (G4112A), Illumina Human-6 whole genome microarrays (Illumina; catalog # BD-25-101) and Operon V3.0 oligonucleotide platforms.</li>

<li><strong><span class="underline">“Operon (c)” and “Operon (d)”</span></strong> Studies on the Operon V3.0 oligonucleotide platform (from the NCI and NMC respectively) that were done <span class="underline">without RNA amplification</span> as described in the Microarray Quality Control project <a href="#pone.0001980-MAQC1">[6]</a>.</li>

</ol>

<h4>T7 primers</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We set out to collect T7 primer sequences used in the field for which datasets are publicly available for analyses. We limited the study to the following T7-primers and looked in detail to their 3′spacer sequences relative to the original Eberwine primer sequence (<a href="#pone-0001980-g002">fig 2</a>) : The Affymetrix primer <strong>(primer 1)</strong>, also described as the “LinAmp1 primer” <a href="#pone.0001980-Baugh1">[7]</a>. The Invitrogen primer <strong>(primer 2)</strong> - with partly disclosed sequence (W. Price, R.J. Rooney, personal communications) - is very similar (possibly identical) to the Affymetrix T7 primer. With respect to the 3′ spacer sequence primers <strong>(1)</strong> and <strong>(2)</strong> are identical. The Ambion primer <strong>(primer 3)</strong> having a 10 bp 3′spacer sequence that is different from the first two primers (B. Setterquist, personal communication). And finally the Agilent primer <strong>(primer 4)</strong> having a 6 bp long 3′ spacer sequence that is fully overlapping with the Ambion primer.</p>
<div class="figure" id="pone-0001980-g002"><div class="img"><a name="pone-0001980-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Primers studied. The core T7 polymerase binding site (in blue) and the 3′ spacer sequence (in red) are highlighted.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Note that the 3′ spacer sequence in the T7 primers is not identical. In all cases the 3′ spacer is different from the Eberwine T7 primer sequence listed on top.</p>
<span>doi:10.1371/journal.pone.0001980.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The 3′ spacer sequence from primers 1 and 2 is transcribed as a 9-mer leader sequence <span class="gene">5′GGGAGGCGG 3′</span>, whereas the 3′ spacer of primer 3 is transcribed as a 10-mer sequence <span class="gene">5′GGGAGAAGAG 3′</span>. Primer 4 introduces the short 6-mer leader sequence <span class="gene">5′GGGAGA 3′</span> into all copies of amplified RNA (<a href="#pone-0001980-g001">fig 1</a>). The motifs complementary to these leader sequences are <span class="gene">5′ CCGCCTCCC 3′</span>, <span class="gene">5′ CTCTTCTCCC 3′</span> and <span class="gene">5′ TCTCCC 3′</span> respectively. Sequences that appear to be present in several probe designs of microarray platforms (<a href="#pone-0001980-t001">table 1</a>) indicate that no design rules exist today to preclude them. As a consequence these probes can hybridize to the leader sequence of every amplified RNA molecule in the hybridization mixture. A possible cause for cross reaction and biased measurements.</p>
<div class="figure" id="pone-0001980-t001"><div class="img"><a name="pone-0001980-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001980.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001980.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Occurrence of sequence motifs in gene expression designs of different microarray platforms.</span></strong></p><span>doi:10.1371/journal.pone.0001980.t001</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p><a href="#pone-0001980-t001">Table 1</a> presents an overview of the numbers of probes on different microarray platforms that share sequence overlap with the primer motif sequences that are used in combination with these platforms. Since primers sequences are transcribed into leader sequences of variable length we determined the abundance of complementary 6–10 bp long sequences in probes of the microarray platforms studied. Data in the table is given for the full length motifs and for the 3′ fragments thereof. Data on the abundance of all possible (internal) parts of the motifs and the percentages with respect to the total amounts of probes per platform can be found in the <a href="#pone.0001980.s002">Table S2</a> and here: <a href="http://microarray.nki.nl/research/T7/">http://microarray.nki.nl/research/T7/</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Compared to the other platforms, the “Agilent (a)” platform has the lowest number of motif containing probes in their probe set: 1.3% (319 sequences) contain the relatively short 6-mer primer 4 motif <span class="gene">TCTCCC</span>. For “Agilent (b)” this is fraction has increased slightly to 1.6%. For “Operon” we find that 8.5% (2966 sequences) of the probes contain either the full 9-mer primer 1 motif or a 6-mer fragment of it. For the “Affymetrix” platform we find that 2,5% (3076 sequences) of all probe sequences and 37% of the 8204 probe sets contain a primer 1 motif sequence. On the Illumina platform 2% of the probe sequences contain the terminal 6-mer sequence <span class="gene">TCTCCC</span> from the primer 3 motif .</p>


<h4>Algorithms</h4>
<h5>Mean intensity of probes.</h5><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1"></a><p>Raw data (absolute intensities) was collected from 35K Operon arrays, printed in-house, and from 25K Agilent arrays and 22K Affymetrix U133 arrays from published literature. Using normalized hybridization data we calculated the mean probe intensity for each probe <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e001&amp;representation=PNG" class="inline-graphic"></span> according to:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>Where n is the number of arrays and <em>A<sub>pi</sub></em> the intensity of probe p in measurement i.</p>

<h5>Mean intensity of motifs.</h5><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1"></a><p>The mean intensity per motif <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e003&amp;representation=PNG" class="inline-graphic"></span> is calculated by averaging the mean probe intensities <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e004&amp;representation=PNG" class="inline-graphic"></span> of the probes that contain the motif in question:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Where S<sub>p</sub> is the set of probes, S<sub>mp</sub> the subset of S<sub>p</sub> that contain motif m, s the size of S<sub>mp</sub>, n the number of measurements and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e006&amp;representation=PNG" class="inline-graphic"></span> the mean intensity for probe p.</p>

<h5>Intensity related motif bias.</h5><a id="article1.body1.sec2.sec4.sec3.p1" name="article1.body1.sec2.sec4.sec3.p1"></a><p>The intensity related motif bias is defined as the mean motif intensity divided by the standard deviation of the motif intensity:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br></p>

<h5>Rank related motif bias.</h5><a id="article1.body1.sec2.sec4.sec4.p1" name="article1.body1.sec2.sec4.sec4.p1"></a><p>Rank related motif bias is defined as the rank related motif intensity divided by the standard deviation of the rank related motif intensity:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>Sampling the ranks limits the effect of outliers and enhances small differences that are observed with intensities.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>The T7 primer 3′ spacer motif causes an increase in spot intensity in a subset of spots</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To estimate the influence of the presence of T7 motif sequences in probes on the amount of amplified RNA that is observed to bind to these probes we calculated average intensity levels for subsets of probes. First, we selected those probes that do contain (parts of) the T7 motifs, and secondly we took those probes that do not contain such motifs (the ‘no-motif’ probes). For control purposes we also used subsets of probes either based on random selection of probes or on random motifs in probes. Finally, we compared the results obtained for primer motifs that were actually used in the studies under investigation (indicated with an arrow in <a href="#pone-0001980-g003">fig 3 A,B</a>) with motifs present in the primers that were not used in the studies but that closely resemble the primers used (overlap &gt;80%).</p>
<div class="figure" id="pone-0001980-g003"><div class="img"><a name="pone-0001980-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Density plots of oligonucleotide probes sets hybridized with RNA amplified with indicated T7 primers (listed in <a href="#pone-0001980-g002">fig 2</a>).</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Horizontal axis is the log intensity scale. A and B, platforms that show T7 primer 3′spacer sequence bias, C and D, platforms that show little or no T7 primer 3′spacer sequence bias. Density analyses were performed on oligonucleotides containing 6 mer sequences (A and C), and on 9-mer sequences (B and D) based on the 3′spacer sequences from primer 1 and 2 (blue) or primer 3 and 4 (red). Arrows indicate the right-shifted density lines for the probes having the 3′spacer sequence present in the primer that was used in the study (indicated at the top of the graphs). Control lines are plotted in black, grey or pink lines. The black line represents density data for all probes lacking T7 motifs; the pink lines are individual random motifs for 6-mer or 9-mer sequences (n = 50), and the grey lines represents data from individual random probes (n = equal to the size of the subset of probes containing the T7 motif used in the study).</p>
<span>doi:10.1371/journal.pone.0001980.g003</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The probes present in a ‘platform – T7 primer’ dataset were subdivided into subsets based on the motifs they contain. The discerning motifs used were the T7- 3′ spacer motifs listed in <a href="#pone-0001980-g002">fig 2</a>. For each probe an average intensity was calculated over all microarrays within the study (<strong>mean probe intensity </strong><span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e012&amp;representation=PNG" class="inline-graphic"></span>, see Algorithms). Possible effects of 3′ spacer motifs are expected to be present in all hybridizations to a similar extent because all amplified RNAs of the set share the same leader sequence derived from the 3′ spacer of the T7 primer. For each subset of probes the average intensities were plotted as a density plot over the <sup>2</sup>log intensity scale (<a href="#pone-0001980-g003">fig 3</a>). Where possible we performed these measurements for both the 6-mer and the 9-mer motifs. In all plots in <a href="#pone-0001980-g003">figure 3</a>, the control measurements for randomly selected probes (grey lines) or random sets of motifs (pink lines) consistently gave distributions that were close to the distribution of the no-motif probes (black lines).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>For the probes of the “Operon (a)” dataset (<a href="#pone-0001980-g003">fig. 3A, bottom left panel</a>) with the 6-mer primer 2 motif <span class="gene">CCTCCC</span> a mean intensity of 9.13 is observed (blue line, arrow). This mean is clearly shifted up relative to the mean of the no-motif probes (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e013&amp;representation=PNG" class="inline-graphic"></span>, black line). This difference is even more pronounced if the same dataset is analyzed for the full 9-mer primer 2 motif (<a href="#pone-0001980-g003">fig. 3B bottom left, blue line</a>). A control subset of probes (<a href="#pone-0001980-g003">fig 3A and B, bottom left panels, red line</a>) selected from “Operon (a)” dataset containing the primer 4 motif <span class="gene">TCTCCC</span> (note that primer 4 was <span class="underline">not</span> used in these experiments) displays a smaller mean intensity shift about half the shift observed for the primer 2 motif probes. These curves clearly show a difference in mean intensity but since the underlying distributions are not normal, a different metric was needed to calculate a statistic for this (see ranked motif bias below).</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Interestingly, shifts in mean probe intensities for data obtained from the same Operon V 3.0 probeset vary depending on the T7 primer used. For instance in the “Operon (b)” study (<a href="#pone-0001980-g003">fig 3A and B bottom right panels</a>), where primer 3 is used to amplify RNA we observe that primer 3 motifs shift the mean intensities furthest to the right, most pronounced in the 9-mer data (<a href="#pone-0001980-g003">fig 3B bottom right panel</a>). The shift observed for the 9-mer primer 2 motif probes in “Operon(a)” (<a href="#pone-0001980-g003">fig 3B, bottom left, blue line</a>) is almost absent in the “Operon(b)” data (<a href="#pone-0001980-g003">fig 3B bottom right panel, blue line</a>). Apparently the same Operon oligos behave differently in combination with a different primer indicating that the choice of the primer is causing these differences. If the comparison is carried out for datasets obtained again from the same Operon V3.0 platform but now using un-amplified RNA samples instead of amplified RNA, the differences between motif containing probes and their no-motif counterparts are minimal or absent (<a href="#pone-0001980-g003">fig 3C,D upper right and bottom left panels</a>), again stressing the idea that the T7 primer motif sequences are influencing intensities observed on affected probes.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>On the somewhat older “Agilent (a)” platform we find similar but less pronounced shifts in distribution curves, which still boils down to a higher intensity for primer 4 motif containing probes. Note that the 9-mer motif <span class="gene">CCGCCTCCC</span> is absent in the “Agilent (a)” platform and therefore not plotted in <a href="#pone-0001980-g003">figure 3B upper right panel</a>. In the more recent G4112A Agilent arrays from the “Agilent (b)” study the shift is much less pronounced or absent (<a href="#pone-0001980-g003">fig 3 C,D upper left panels</a>).</p>
<a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>The “Affymetrix” study uses primer 1, and again we observe a shift of 3.6 for the corresponding motif containing probes (<a href="#pone-0001980-g003">fig 3A,B upper left panels, blue lines</a>). If we study the primer 4 motif containing probes in the same dataset the shift is smaller (<a href="#pone-0001980-g003">fig 3 A,B upper left panels, red lines</a>). The Illumina platform uses primer 3 based amplification kits. From the distribution in <a href="#pone-0001980-g003">fig 3</a> C and D (bottom right panels) we can see that there is no intensity shift in intensity for probes having a primer 3 motif. Also the other probe sets selected from the Illumina dataset show no differences in intensity profiles.</p>
<a id="article1.body1.sec3.sec1.p7" name="article1.body1.sec3.sec1.p7"></a><p>To investigate if the shift in mean intensities observed in amplified RNA studies affects the ratios observed for probes with T7 spacer sequences we compared two MAQC platforms, the “Operon (b)” and “Operon (d)” platform. Both types of microarrays use the same Operon oligonucleotide set and both were hybridized with identical samples. The only difference consists of using unamplified or amplified RNA. We selected probes from both platforms that were affected by T7 bias and compared the ratio measurements obtained for these probes. We observed that ratios for these probes are markedly closer to zero in the amplified RNA study compared to the unamplified RNA study (<a href="#pone-0001980-g004">fig 4</a>). As a result of this shift 22 out of the 34 (65%) gene expression measurements are no longer considered significant outliers under the Rosetta error model. Interestingly, this shift in ratios contrasts the general trend showing more extreme ratios for the probes not affected by the T7 bias in amplified RNA data. This is illustrated in <a href="#pone.0001980.s006">Figure S3</a> that compares the spread of ratios between the unamplified RNA study (A and B) versus the amplified RNA study (C and D) in a box plot figure. When we focus on the unamplified RNA we observe that unaffected (A) and T7 bias affected probes (B) have very similar distributions. If we compare this to the amplified RNA distributions in C and D, we see that affected probes (D) show compressed distribution centering around 0. By comparing (A) and (C) one can furthermore conclude that amplification of RNA leads to increased ratios.</p>
<div class="figure" id="pone-0001980-g004"><div class="img"><a name="pone-0001980-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Significant ratios are lost when reporters are affected by T7 bias phenomenon.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>For this plot we used data from hybridizations that belong either to amplified RNA (closed circles) or unamplified RNA (open circles) in two different MAQC studies, “Operon (b)” and “Operon (d)” respectively. The “Operon (b)” study used primer 3 to amplify the total RNA. Plotted is the M versus A value for four probes that were selected from 61 probe sequences (all containing the 8-mer motif <span class="gene">CTTCTCCC</span> present in the primer used) because these four report a significant up or down regulation in the Operon(d) study. Indicated are the normalized and averaged MA values for microarrays (n = 10) hybridized with the same sample on the two platforms (error bars for standard deviation). The ratios observed for amplified RNA are situated closer to the zero level. From the individual microarray measurements 34 out of the 40 spots were a significant outlier in the unamplified RNA study. 22 (or 65%) of them lost the status of being a significant outlier after the amplification procedure. Operon Ids for the probes listed: H300004675 (black), H300006007 (red), H300008441 (green) and H300014662 (blue).</p>
<span>doi:10.1371/journal.pone.0001980.g004</span></div><a id="article1.body1.sec3.sec1.p8" name="article1.body1.sec3.sec1.p8"></a><p>To better compare subsets of probes in a normalized manner and to allow for statistical evaluation we converted the intensity axis into a rank-axis. We ranked the mean probe intensities <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e014&amp;representation=PNG" class="inline-graphic"></span> and assigned a rank number to each individual probe. We listed all possible 6-, 7-, 8- and 9-mer motifs present in the collection of probes of a platform. Each listed motif has an associated probe set, comprised of all probes containing that motif. We assigned to every motif the average rank of all probes in the corresponding probe set (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e015&amp;representation=PNG" class="inline-graphic"></span>). Motif <span class="gene">CCTCCC</span> (primer 1&amp;2) appeared to be a significant outlier in the Affymetrix study (p = 1.4×10<sup>−10</sup>), the “Operon (a)” study (p = 9.4×10<sup>−10</sup>), the “Agilent (a)” study (p = 2.4×10<sup>−7</sup>) and the “Operon (b)” study (p = 4.7×10<sup>−6</sup>). Surprisingly, this motif also was an outlier in the “Operon(c)” study (p = 3.9×10<sup>−3</sup>). Motif <span class="gene">TCTCCC</span> however, (primer 3&amp;4) was a significant outlier in the Affymetrix study (p = 5.2×10<sup>−3</sup>), the “Agilent (a)” study (p = 9.9×10<sup>−7</sup>), the “Operon (a)” study (p = 1.8×10<sup>−4</sup> ) and the “Operon (b)” study (p = 3.9×10<sup>−9</sup>). The 9-mer motifs showed a very similar trend. An overview of all density plots and the associated p-values is shown in <a href="#pone.0001980.s003">Table S3</a> and <a href="#pone.0001980.s005">Figure S2</a>. Except for the previously mentioned <span class="gene">CCTCCC</span> motif in the “Operon(c)” study, none of the motifs was a significant outlier in the Illumina study, the “Agilent (b)” study and the studies using unamplified RNA, “Operon (c)” and “Operon (d)”.</p>


<h4>Motifs that are associated with intensity bias can be identified in any probe within any platform by measuring the Rank related motif bias</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The T7 primer 3′ spacer sequence motifs are expected to have high mean intensities and therefore have a high mean probe rank. However, motifs that are exclusively part of abundantly expressed genes will also have high mean probe intensities and ranks. To discriminate motifs associated with highly abundant genes from those associated with bias motifs we reasoned that variations in observed <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e016&amp;representation=PNG" class="inline-graphic"></span> will be high for the first and low for the latter motifs. Motifs present in highly abundant genes are likely to also be present in low-abundant genes. It is unlikely that motifs in highly abundant genes are the reason for the high abundance of that mRNA. We expect therefore that the standard deviation over the observed motif intensity (σ<em><sub>Am</sub></em>) is high for abundant mRNAs and low for bias motifs (like the T7 motifs). If the motif really is the reason for the high intensity bias, the motif will invariably be associated with high <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e017&amp;representation=PNG" class="inline-graphic"></span> values and subsequently have a low σ<em><sub>Am</sub></em>. By calculating the <strong>intensity related motif bias</strong> (<em>M<sub>A</sub></em>) by dividing the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e018&amp;representation=PNG" class="inline-graphic"></span> by the σ<em><sub>Am</sub></em>, and the <strong>rank related motif bias</strong> (<em>M<sub>R</sub></em>) by dividing the <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.e019&amp;representation=PNG" class="inline-graphic"></span> by σ<em><sub>Rm</sub></em> we introduced two methods for detecting motifs with associated intensity bias. The <strong>rank related motif bias</strong> (<em>M<sub>R</sub></em>) showed to be superior in separating bias motifs from background motifs. An example of this is displayed in <a href="#pone-0001980-g005">fig 5</a> where a random selection of 10 primer-motif containing probes is taken from the “Operon (a)” study and analyzed for the <em>M<sub>R</sub></em> value. All primer related 3′ spacer sequence motifs could be traced back to peaks of rank related motif bias. Some additional peaks were also observed. These represent biases other than T7 primer related biases. To rule out that the peaks found in <a href="#pone-0001980-g005">fig 5</a> are the result of motifs present in highly abundant genes, we plotted our <em>M<sub>R</sub></em> data against SAGE data of highly abundant genes. Out of the full repository of SAGE data from NCBI (<a href="http://ftp://ftp.ncbi.nlm.nih.gov/pub/sage/extr/tag_lib_freq.zip">ftp://ftp.ncbi.nlm.nih.gov/pub/sage/extr​/tag_lib_freq.zip</a>) and (<a href="http://www.ncbi.nlm.nih.gov/SAGE">http://www.ncbi.nlm.nih.gov/SAGE</a>) the top 100 highest abundant gene products present in 4090 SAGE datasets were selected. For each motif we determined the fraction of abundant SAGE genes that are targeted by the corresponding subset of probes (F<sub>SAGE</sub>). There is no significant correlation between F<sub>SAGE</sub> versus the <em>M<sub>R</sub></em> of the motifs (see <a href="#pone.0001980.s004">Figure S1</a> and <a href="http://microarray.nki.nl/research/T7/">http://microarray.nki.nl/research/T7/</a>). We can thus conclude that the <em>M<sub>R</sub></em> calculation is a good method for finding motifs linked to aberrant high signals. Moreover, the method is not hampered by motifs belonging to highly expressed genes.</p>
<div class="figure" id="pone-0001980-g005"><div class="img"><a name="pone-0001980-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001980" data-uri="info:doi/10.1371/journal.pone.0001980.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001980.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001980.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001980.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001980.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Examples of oligonucleotide probes from the “Operon (a)” probe set where T7 bias motifs are identified by plotting the rank related motif bias (Mr).</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Column A and B: Ten examples plots for probe sequences that contain the T7primer 1 3′ spacer motif <span class="gene">CCTCCC</span>. The 70-mer probe sequence is split in 6-mer fragments over the x-axis and the intensity rank related motif bias (Mr) is plotted on the y-axis. The red dot represents the position of the <span class="gene">CCTCCC</span> motif in the probe sequence. The blue line is the average over all the motifs from a probe sequence, the red lines represent the standard deviation up and down for the same measurement over all 6-mers in the whole probe. Operon Ids for the probes listed A1 to B5 and C1 to D5: H200018976, H300020020, H200016156, H300007790, H300013997, H300022725, H200010013, H300019787, H200001204, H200004220.</p>
<span>doi:10.1371/journal.pone.0001980.g005</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The T7 primer used in amplification of mRNA for microarray applications is a potential source for bias in microarray data. The bias appears to be common to most microarray platforms that use amplification and mean intensities of one to nine percent of the probes increase aberrantly to levels up to a hundred fold the average level of unaffected probes. It is also dependent on the sequence of the T7 primer and it is absent from hybridizations where unamplified (total) RNA is used.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>We find that the older Agilent, the Operon V3.0 and the Affymetrix data sets are most affected and that the newer Agilent and Illumina platforms are virtually free from the phenomenon.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The T7 bias can be pinned down to a precise sequence domain of the T7 primer used in the amplification, the 3′ spacer sequence. This sequence is introduced by the amplification procedure as a leader sequence at the 3′ terminus of <em>all</em> amplified (and subsequently labeled) molecules. As a consequence, in microarray hybridizations using this amplified RNA, these motifs are abundantly present. Apparently this high concentration drives probes with complementary sequences to bind amplified RNA, irrespective of the gene sequence present in the amplified RNA. This leads to a change in ratio for these probes more reflecting the total amounts of red and green in the hybridization mixture which, under normal circumstances, are present in equal amounts (1:1). True gene expression differences therefore become masked by the T7 bias (see <a href="#pone-0001980-g004">fig 4</a> and <a href="#pone.0001980.s006">Figure S3</a>).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The bias has gone unnoticed until now, most likely because it is restricted to a subset of the features on the microarray, namely only those that contain motifs complementary to the 3′ spacer sequence of the T7 primer. Melting temperatures of the short motif sequences fall below the hybridization temperature so in theory these motifs are unable to hybridize. In addition, previous findings indicated that short (linker) sequences do not introduce hybridization artifacts on microarrays <a href="#pone.0001980-He1">[12]</a>. The apparent contradiction with our results might be explained by the differences in the complexities of the linker pools. Complex linker mixtures may react differently to a single 6–9 mer T7 primer motif sequence. This may also have contributed to their concealment.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The intensity measured from a feature consists of two components: the gene expression and the bias. To which extent the T7 bias influences the measured ratio, assuming that the bias signal is equal for every affected feature, therefore depends on the signal coming from gene expression. As mentioned, high bias signals shift the log-ratio measured in affected features on dual color platforms towards 0 (no difference measured) and the spot is therefore easily lost from the analysis because it will not be selected as outlier anymore. In single channel measurements, on the contrary, T7 biased motifs features become over-estimations of gene expression because of the fact that the aberrant motif bias signals add up to the true gene expression signals. Here, the spots might have been noticed as consistently up regulated. Still, as in the case of Affymetrix, the spot is likely to become an outlier in the probe-set used to detect a single transcript and will therefore be discarded from the estimation of the averaged probe intensity for the transcript. Looking at actual probe data versus gene data confirms this assumption (data not shown).</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Our results clearly show that motifs present in the 3′ spacer sequences of T7 primers correlate strongly with the intensity bias observed on probes with complementary sequences and can induce intensity shifts up to a hundred fold the average level of unaffected probes. It is remarkable that the shift in mean probe intensity for the selected probes in <a href="#pone-0001980-g003">fig 3</a> is so specific to the platform-primer combination studied. Relatively small variations in the 3′ spacer sequences of the T7 primers immediately reduce the phenomenon dramatically. This can for instance be seen in <a href="#pone-0001980-g003">fig 3A</a> where a much smaller shift in mean probe intensity is observed for those populations of probes that carry motifs from related, nearly identical T7 primers. These curves are based on probes carrying motifs that vary only by a single base compared to the primer sequences that were actually used in the studies (marked by an arrow). A possible cause for the fact that we still measure a shift for a probe sequence not used in the study is the overlap among the motifs (<span class="gene">CCTCCC</span> and <span class="gene">TCTCCC</span>) taken from the 3′ spacer sequences of the primers (see <a href="#pone-0001980-g002">fig 2</a>).</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>The absence of an apparent intensity shift for subsets of probes containing complementary primer motifs in the “Agilent (b)” and “Illumina” platforms (see <a href="#pone-0001980-g003">fig 3</a>) is remarkable since these platforms do use amplified RNA and primers with motifs that might induce T7 bias in the data. Apparently these platforms have found ways to reduce the bias effects. For example, additives in the hybridization mixtures can be used to prevent the T7 bias (see below).</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Several oligonucleotide designs for gene expression analysis have been generated with probes containing motifs complementary to 3′ spacer sequences of the T7 primers. Improvements to prevent the occurrence of T7 motif bias can be found along two lines. One option is to eliminate sequence overlap from the T7 primers. In that case the 3′ spacer sequence could be completely removed or replaced by an uncomplimentary sequence. It is unclear at the moment if this is feasible in the sense that we don't know if a T7 enzyme will be functional using such a drastically modified primer. On the other hand, probes can be designed that lack complementary sequences to these T7 spacer motifs and thus eliminate the artifact. In the mean time, one could make use of specific blocking oligos that react with the motif sequences on the targets or on the probe sequences. Preliminary results (M.N., R.M.K. and M.H) where single stranded oligos with complementary sequences to T7 spacer domains or to T7 motif domains from probe sequences were added to hybridization mixtures indicate that blocking oligos indeed reduce the T7 bias phenomenon. In fact, the approach is similar to the one taken to minimize side-effects of the stretch of T-nucleotides in the amplified RNA (a remnant of the poly(A) tail of the mRNA) by adding poly-d(A) in the hybridization buffer.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>The aberrant high signals we observe on features displaying T7-bias motifs are the result of excess binding of labeled target molecules. Our data show that this is the result of binding of the 3′spacer of these molecules to probes on the array. We hypothesize that the binding we observe is a reflection of the molar excess of the amplified material available for binding. Since <em>all</em> target molecules carry the 3′ spacer sequence, the hybridization kinetics on the features containing a complementary T7-bias domain (1 to 5% of the probe sequences) may be forced into the binding state. A secondary feature of the T7-bias domains is that they are GC-rich and distally located on the molecule which may increase binding affinity.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>The method we describe for the calculation of motif bias is capable of detecting probes that are affected by T7 based intensity artifacts. Clearly, the 3′ spacer sequence in the T7 primer is associated with a higher than average rank related motif bias. It is however not the only motif that has a high value in this respect (see <a href="#pone-0001980-g005">fig 5</a>). Until now, the T7 spacer motifs are the only motifs for which we discuss an explanation because we can directly link the observed bias sequence back to the T7 primer sequence. There are however also other motifs, unrelated to the T7 primer sequence, that score high in the <em>M<sub>R</sub></em>-value calculation. We are currently investigating the nature and experimental causes for these other high <em>M<sub>R</sub></em>-value associated motifs observed in the datasets under study.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001980"></div><a name="pone.0001980.s001" id="pone.0001980.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001980.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Condensed ratio table. Data table where we indicate the Operon Identifier (oligoID), the Central Microarray Identifier (reporterID), the mean of intensities observed for this probe over the data in the CMFdatabase (mean) and the statistical values associated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001980.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(5.66 MB TXT)</p>
<a name="pone.0001980.s002" id="pone.0001980.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001980.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Table of motif containing probes per platform.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001980.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001980.s003" id="pone.0001980.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001980.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Statistical values (p-values) reflecting the chance that a particular motif belongs to a distribution of ranked motifs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001980.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001980.s004" id="pone.0001980.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001980.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">M(R) score versus SAGE scores Figure showing that there is no correlation between the presence of a motif in a probe that binds abundant mRNAs and the T7-bias M(R) score of that motif. Dotplot of the SAGE tag-score versus the M<sub>R</sub> of the motifs. SAGE data from NCBI (<a href="http://ftp://ftp.ncbi.nlm.nih.gov/pub/sage/extr/tag_lib_freq.zip">ftp://ftp.ncbi.nlm.nih.gov/pub/sage/extr​/tag_lib_freq.zip</a>) and (<a href="http://www.ncbi.nlm.nih.gov/SAGE)">http://www.ncbi.nlm.nih.gov/SAGE)</a>. A selection made of the top 100 highest abundant (and therefore best characterized signals in SAGE) gene products present in 4090 SAGE datasets. Their sequence was split in motifs of different lengths and plotted against the M<sub>R</sub> of the same motifs. Panels A to D: plots for motifs that are 6, 7, 8 and 9-mers in length. Notice that there is no correlation between the two aspects of the same motif.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001980.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.27 MB DOC)</p>
<a name="pone.0001980.s005" id="pone.0001980.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001980.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Density plots of the motif mean ranked intensity for all 6-mer (A,C) and 9-mer (B,D) motifs in the platforms studied. Indicated is the position of the T7 bias motifs <span class="gene">CCTCCC</span> and <span class="gene">TCTCCC</span> that are significant outliers (p-values in supplemental data 3) in the studies that used the respective T7 primers (1&amp;2 and 3&amp;4, panel A). In the studies listed in panel C, these motifs aren't found as outliers (except for <span class="gene">CCTCCC</span> in “Operon (c)”. The corresponding 9-mer motifs show a very similar trend (B,D).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001980.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(2.63 MB TIF)</p>
<a name="pone.0001980.s006" id="pone.0001980.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001980.s006">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Boxplot representation of the ratio distribution observed from an unamplified RNA study “Operon (d)” (A and B) versus an amplified RNA study “Operon (b)” (C and D). The spread of the ratios is shown for the T7 bias unaffected probes (A and C) versus the affected probes (B and D). For unamplified RNA an equal distribution is present for unaffected (A) and T7 bias affected probes (B). For amplified RNA however, the T7 bias affected probes (D) display a clear reduction in ratio distribution compared to the unaffected probes (C). Comparing A and C indicate that amplification in itself leads to increased ratio levels.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001980.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(3.66 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank prof. C.P Hunter from Harvard University for T7 primer spacer information, and W. Price from Invitrogen, E. Bjeldanus from Agilent, R.J. Rooney from Genome Explorations Inc. and B. Setterquist from Ambion for T7-primer sequence information. The critical reading of the manuscript by dr J. Jonkers, C. Lieftink, dr B. van Steensel and prof dr R. Bernards led to useful suggestions that improved the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RK. Performed the experiments: DS MN MH WB. Analyzed the data: DS Jd AV. Contributed reagents/materials/analysis tools: DS MN MH Jd WB AV. Wrote the paper: RK DS Jd AV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001980-VanGelder1" id="pone.0001980-VanGelder1"></a>Van Gelder RN, von Zastrow ME, Yool A, Dement WC, Barchas JD, et al.  (1990) Amplified RNA synthesized from limited quantities of heterogeneous cDNA. Proc Natl Acad Sci U S A  87(5): 1663–1667.  <ul class="find" data-citedArticleID="1090386" data-doi="10.1073/pnas.87.5.1663"><li><a href="http://dx.doi.org/10.1073/pnas.87.5.1663" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amplified+RNA+synthesized+from+limited+quantities+of+heterogeneous+cDNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amplified+RNA+synthesized+from+limited+quantities+of+heterogeneous+cDNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001980-Petalidis1" id="pone.0001980-Petalidis1"></a>Petalidis L, Bhattacharyya S, Morris GA, Collins VP, Freeman TC, et al.  (2003) Global amplification of mRNA by template-switching PCR: linearity and application to microarray analysis. Nucleic Acids Res  31(22): e142.  <ul class="find" data-citedArticleID="1090380" data-doi="10.1093/nar/gng142"><li><a href="http://dx.doi.org/10.1093/nar/gng142" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+amplification+of+mRNA+by+template-switching+PCR%3A+linearity+and+application+to+microarray+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+amplification+of+mRNA+by+template-switching+PCR%3A+linearity+and+application+to+microarray+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001980-Wang1" id="pone.0001980-Wang1"></a>Wang E, Miller LD, Ohnmacht GA, Liu ET, Marincola FM (2000) High-fidelity mRNA amplification for gene profiling. Nat Biotechnol  18(4): 457–459.  <ul class="find" data-citedArticleID="1090388" data-doi="10.1038/74546"><li><a href="http://dx.doi.org/10.1038/74546" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-fidelity+mRNA+amplification+for+gene+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-fidelity+mRNA+amplification+for+gene+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001980-Petersen1" id="pone.0001980-Petersen1"></a>Petersen D, Chandramouli GV, Geoghegan J, Hilburn J, Paarlberg J, et al.  (2005) Three microarray platforms: an analysis of their concordance in profiling gene expression. BMC Genomics  6(1): 63.  <ul class="find" data-citedArticleID="1090382"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Three+microarray+platforms%3A+an+analysis+of+their+concordance+in+profiling+gene+expression.&amp;auth=&amp;atitle=Three+microarray+platforms%3A+an+analysis+of+their+concordance+in+profiling+gene+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+microarray+platforms%3A+an+analysis+of+their+concordance+in+profiling+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+microarray+platforms%3A+an+analysis+of+their+concordance+in+profiling+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001980-Kuo1" id="pone.0001980-Kuo1"></a>Kuo WP, Liu F, Trimarchi J, Punzo C, Lombardi M, et al.  (2006) A sequence-oriented comparison of gene expression measurements across different hybridization-based technologies. Nat Biotechnol  24(7): 832–840.  <ul class="find" data-citedArticleID="1090372" data-doi="10.1038/nbt1217"><li><a href="http://dx.doi.org/10.1038/nbt1217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+sequence-oriented+comparison+of+gene+expression+measurements+across+different+hybridization-based+technologies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+sequence-oriented+comparison+of+gene+expression+measurements+across+different+hybridization-based+technologies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001980-MAQC1" id="pone.0001980-MAQC1"></a>Shi L, Reid LH, Jones WD, Shippy R, et al. MAQC Consortium (2006) The MicroArray Quality Control (MAQC) project shows inter- and intraplatform reproducibility of gene expression measurements. Nat Biotechnol  24(9): 1151–1161.  <ul class="find" data-citedArticleID="1090376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+MicroArray+Quality+Control+%28MAQC%29+project+shows+inter-+and+intraplatform+reproducibility+of+gene+expression+measurements.&amp;auth=&amp;atitle=The+MicroArray+Quality+Control+%28MAQC%29+project+shows+inter-+and+intraplatform+reproducibility+of+gene+expression+measurements." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+MicroArray+Quality+Control+%28MAQC%29+project+shows+inter-+and+intraplatform+reproducibility+of+gene+expression+measurements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+MicroArray+Quality+Control+%28MAQC%29+project+shows+inter-+and+intraplatform+reproducibility+of+gene+expression+measurements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001980-Baugh1" id="pone.0001980-Baugh1"></a>Baugh LR, Hill AA, Brown EL, Hunter CP (2001) Quantitative analysis of mRNA amplification by in vitro transcription. Nucleic Acids Res  29(5): E29.  <ul class="find" data-citedArticleID="1090368" data-doi="10.1093/nar/29.5.e29"><li><a href="http://dx.doi.org/10.1093/nar/29.5.e29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+analysis+of+mRNA+amplification+by+in+vitro+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+analysis+of+mRNA+amplification+by+in+vitro+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001980-Luo1" id="pone.0001980-Luo1"></a>Luo L, Salunga RC, Guo H, Bittner A, Joy KC, et al.  (1999) Gene expression profiles of laser-captured adjacent neuronal subtypes. Nat Med  5(1): 117–122.  <ul class="find" data-citedArticleID="1090374"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+expression+profiles+of+laser-captured+adjacent+neuronal+subtypes.&amp;auth=&amp;atitle=Gene+expression+profiles+of+laser-captured+adjacent+neuronal+subtypes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiles+of+laser-captured+adjacent+neuronal+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiles+of+laser-captured+adjacent+neuronal+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001980-Nacheva1" id="pone.0001980-Nacheva1"></a>Nacheva GA, Berzal-Herranz A (2003) Preventing nondesired RNA-primed RNA extension catalyzed by T7 RNA polymerase. Eur J Biochem  270(7): 1458–1465.  <ul class="find" data-citedArticleID="1090378" data-doi="10.1046/j.1432-1033.2003.03510.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1033.2003.03510.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preventing+nondesired+RNA-primed+RNA+extension+catalyzed+by+T7+RNA+polymerase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preventing+nondesired+RNA-primed+RNA+extension+catalyzed+by+T7+RNA+polymerase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001980-RDevelopmentCoreTeam1" id="pone.0001980-RDevelopmentCoreTeam1"></a>R_Development_Core_Team R (2007) A Language and Environment for Statistical Computing, R.F.f.S. Vienna, Austria: Computing..   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001980-vandeVijver1" id="pone.0001980-vandeVijver1"></a>van de Vijver MJ, He YD, van't Veer LJ, Dai H, Hart AA, et al.  (2002) A gene-expression signature as a predictor of survival in breast cancer. N Engl J Med  347(25): 1999–2009.  <ul class="find" data-citedArticleID="1090390" data-doi="10.1056/nejmoa021967"><li><a href="http://dx.doi.org/10.1056/nejmoa021967" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene-expression+signature+as+a+predictor+of+survival+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene-expression+signature+as+a+predictor+of+survival+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001980-He1" id="pone.0001980-He1"></a>He Z, Wu L, Li X, Fields MW, Zhou J (2005) Empirical establishment of oligonucleotide probe design criteria. Appl Environ Microbiol  71(7): 3753–3760.  <ul class="find" data-citedArticleID="1090370" data-doi="10.1128/aem.71.7.3753-3760.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.7.3753-3760.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+establishment+of+oligonucleotide+probe+design+criteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+establishment+of+oligonucleotide+probe+design+criteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.XML" value="69811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.PDF" value="422063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g001.PNG_L" value="440482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g001.PNG_M" value="80819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g001.PNG_S" value="6643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g001.TIF" value="981708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g001.PNG_I" value="30553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g002.PNG_L" value="245790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g002.PNG_M" value="50762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g002.PNG_S" value="4628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g002.TIF" value="339074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g002.PNG_I" value="20095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.t001.PNG_L" value="71027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.t001.PNG_M" value="98641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.t001.PNG_S" value="17106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.t001.TIF" value="355760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.t001.PNG_I" value="34042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e001.PNG" value="5198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e001.TIF" value="21140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e002.PNG" value="7872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e002.TIF" value="23708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e003.PNG" value="5611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e003.TIF" value="21512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e004.PNG" value="5580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e004.TIF" value="21508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e005.PNG" value="8246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e005.TIF" value="23968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e006.PNG" value="5180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e006.TIF" value="21148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e007.PNG" value="7121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e007.TIF" value="23036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e008.PNG" value="10690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e008.TIF" value="26396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e009.PNG" value="7247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e009.TIF" value="23120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e010.PNG" value="8232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e010.TIF" value="23992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e011.PNG" value="10728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e011.TIF" value="26420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g003.PNG_L" value="3432119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g003.PNG_M" value="183809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g003.PNG_S" value="15425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g003.TIF" value="5425362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g003.PNG_I" value="72219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e012.PNG" value="5197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e012.TIF" value="21152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e013.PNG" value="6255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e013.TIF" value="22180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g004.PNG_L" value="164200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g004.PNG_M" value="41517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g004.PNG_S" value="7101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g004.TIF" value="506638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g004.PNG_I" value="16127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e014.PNG" value="5203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e014.TIF" value="21148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e015.PNG" value="5202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e015.TIF" value="21164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e016.PNG" value="5233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e016.TIF" value="21160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e017.PNG" value="5216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e017.TIF" value="21144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e018.PNG" value="5217"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e018.TIF" value="21152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e019.PNG" value="5238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.e019.TIF" value="21180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g005.PNG_L" value="726048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g005.PNG_M" value="73735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g005.PNG_S" value="14128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g005.TIF" value="1615474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.g005.PNG_I" value="76468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.s001.TXT" value="5658708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.s002.DOC" value="27136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.s003.DOC" value="30720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.s004.DOC" value="271360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.s005.TIF" value="2614376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001980.s006.TIF" value="1222094"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001980&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001980" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001980&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001980&volume=&issue=&title=The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms&author_name=Ron%20M.%20Kerkhoven%2C%20Daoud%20Sie%2C%20Marja%20Nieuwland%2C%20Mike%20Heimerikx%2C%20Jorma%20De%20Ronde%2C%20Wim%20Brugman%2C%20Arno%20Velds&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001980" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980&amp;t=The%20T7-Primer%20Is%20a%20Source%20of%20Experimental%20Bias%20and%20Introduces%20Variability%20between%20Microarray%20Platforms" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980&title=The%20T7-Primer%20Is%20a%20Source%20of%20Experimental%20Bias%20and%20Introduces%20Variability%20between%20Microarray%20Platforms&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980&amp;title=The%20T7-Primer%20Is%20a%20Source%20of%20Experimental%20Bias%20and%20Introduces%20Variability%20between%20Microarray%20Platforms" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001980&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The T7-Primer Is a Source of Experimental Bias and Introduces Variability between Microarray Platforms';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001980';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20T7-Primer%20Is%20a%20Source%20of%20Experimental%20Bias%20and%20Introduces%20Variability%20between%20Microarray%20Platforms http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001980" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001980" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26784"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Operons%22" title="Search for articles in the subject area:'Operons'"><div class="flagText">Operons</div></a>
              <div data-categoryid="21291" data-articleid="26784"
                   data-categoryname="Operons"
                   class="flagImage" title="Flag 'Operons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="48355" data-articleid="26784"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+amplification%22" title="Search for articles in the subject area:'RNA amplification'"><div class="flagText">RNA amplification</div></a>
              <div data-categoryid="21647" data-articleid="26784"
                   data-categoryname="RNA amplification"
                   class="flagImage" title="Flag 'RNA amplification' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+hybridization%22" title="Search for articles in the subject area:'RNA hybridization'"><div class="flagText">RNA hybridization</div></a>
              <div data-categoryid="48689" data-articleid="26784"
                   data-categoryname="RNA hybridization"
                   class="flagImage" title="Flag 'RNA hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="26784"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26784"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Serial+analysis+of+gene+expression%22" title="Search for articles in the subject area:'Serial analysis of gene expression'"><div class="flagText">Serial analysis of gene expression</div></a>
              <div data-categoryid="48515" data-articleid="26784"
                   data-categoryname="Serial analysis of gene expression"
                   class="flagImage" title="Flag 'Serial analysis of gene expression' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8352'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4090'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6165&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6485">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
