

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000197"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000197" />

    <meta name="citation_title" content="The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing"/>
    <meta itemprop="name" content="The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing"/>

      <meta name="citation_author" content="Jonas Binladen"/>
            <meta name="citation_author_institution" content="Center for Ancient Genetics, Institute of Biology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="M. Thomas P. Gilbert"/>
            <meta name="citation_author_institution" content="Center for Ancient Genetics, Institute of Biology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Jonathan P. Bollback"/>
            <meta name="citation_author_institution" content="Center for Bioinformatics and Institute of Biology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Frank Panitz"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Danish Institute of Agricultural Sciences Research Centre Foulum, Tjele, Denmark"/>
      <meta name="citation_author" content="Christian Bendixen"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Danish Institute of Agricultural Sciences Research Centre Foulum, Tjele, Denmark"/>
      <meta name="citation_author" content="Rasmus Nielsen"/>
            <meta name="citation_author_institution" content="Center for Bioinformatics and Institute of Biology, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Eske Willerslev"/>
            <meta name="citation_author_institution" content="Center for Ancient Genetics, Institute of Biology, University of Copenhagen, Copenhagen, Denmark"/>

    <meta name="citation_date" content="2007/2/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000197.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e197"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=1; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Unique features of a highly pathogenic Campylobacter jejuni strain.; citation_author=D Hofreuter; citation_author=J Tsai; citation_author=RO Watson; citation_author=V Novik; citation_author=B Altman; citation_journal_title=Infect Immun; citation_volume=74; citation_number=2; citation_pages=4694-4707; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sensitive mutation detection in heterogeneous cancer specimens by massively parallel picoliter reactor sequencing.; citation_author=RK Thomas; citation_author=E Nickerson; citation_author=JF Simons; citation_author=PA Janne; citation_author=T Tengs; citation_journal_title=Nat Med; citation_volume=12; citation_number=3; citation_pages=852-855; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reproducibility of ancient DNA sequences from extinct Pleistocene fauna.; citation_author=PG Taylor; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=4; citation_pages=283-285; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucl Acids Res; citation_volume=22; citation_number=5; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Recharacterization of ancient DNA miscoding lesions: insights in the era of sequencing-by-synthesis.; citation_author=MTP Gilbert; citation_author=J Binladen; citation_author=W Miller; citation_author=C Wiuf; citation_author=E Willerslev; citation_journal_title=Nucleic Acids Res.; citation_number=6; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J. Mol Biol; citation_volume=215; citation_number=7; citation_pages=403-410; citation_date=1990; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing"/>
    <meta name="twitter:description" content="BackgroundThe invention of the Genome Sequence 20&trade; DNA Sequencing System (454 parallel sequencing platform) has enabled the rapid and high-volume production of sequence data. Until now, however, individual emulsion PCR (emPCR) reactions and subsequent sequencing runs have been unable to combine template DNA from multiple individuals, as homologous sequences cannot be subsequently assigned to their original sources.MethodologyWe use conventional PCR with 5&prime;-nucleotide tagged primers to generate homologous DNA amplification products from multiple specimens, followed by sequencing through the high-throughput Genome Sequence 20&trade; DNA Sequencing System (GS20, Roche/454 Life Sciences). Each DNA sequence is subsequently traced back to its individual source through 5&prime;tag-analysis.ConclusionsWe demonstrate that this new approach enables the assignment of virtually all the generated DNA sequences to the correct source once sequencing anomalies are accounted for (miss-assignment rate&lt;0.4%). Therefore, the method enables accurate sequencing and assignment of homologous DNA sequences from multiple sources in single high-throughput GS20 run. We observe a bias in the distribution of the differently tagged primers that is dependent on the 5&prime; nucleotide of the tag. In particular, primers 5&prime; labelled with a cytosine are heavily overrepresented among the final sequences, while those 5&prime; labelled with a thymine are strongly underrepresented. A weaker bias also exists with regards to the distribution of the sequences as sorted by the second nucleotide of the dinucleotide tags. As the results are based on a single GS20 run, the general applicability of the approach requires confirmation. However, our experiments demonstrate that 5&prime;primer tagging is a useful method in which the sequencing power of the GS20 can be applied to PCR-based assays of multiple homologous PCR products. The new approach will be of value to a broad range of research areas, such as those of comparative genomics, complete mitochondrial analyses, population genetics, and phylogenetics."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000197.g001"/>

  <meta property="og:title" content="The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2801'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7228'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6637&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000197">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000197" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000197&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonas Binladen
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jonas Binladen, M. Thomas P. Gilbert</p>

              
              

                <p>Affiliation:
                  Center for Ancient Genetics, Institute of Biology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            M. Thomas P. Gilbert
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jonas Binladen, M. Thomas P. Gilbert</p>

              
              

                <p>Affiliation:
                  Center for Ancient Genetics, Institute of Biology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jonathan P. Bollback, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bioinformatics and Institute of Biology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank Panitz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Danish Institute of Agricultural Sciences Research Centre Foulum, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Bendixen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Danish Institute of Agricultural Sciences Research Centre Foulum, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rasmus Nielsen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Bioinformatics and Institute of Biology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eske Willerslev
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ewillerslev@bi.ku.dk">ewillerslev@bi.ku.dk</a>
            </p>

                <p>Affiliation:
                  Center for Ancient Genetics, Institute of Biology, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000197</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000197-g001" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000197-t001" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000197-t002" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000197-t003" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000197">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000197" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The invention of the Genome Sequence 20™ DNA Sequencing System (454 parallel sequencing platform) has enabled the rapid and high-volume production of sequence data. Until now, however, individual emulsion PCR (emPCR) reactions and subsequent sequencing runs have been unable to combine template DNA from multiple individuals, as homologous sequences cannot be subsequently assigned to their original sources.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We use conventional PCR with 5′-nucleotide tagged primers to generate homologous DNA amplification products from multiple specimens, followed by sequencing through the high-throughput Genome Sequence 20™ DNA Sequencing System (GS20, Roche/454 Life Sciences). Each DNA sequence is subsequently traced back to its individual source through 5′tag-analysis.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We demonstrate that this new approach enables the assignment of virtually all the generated DNA sequences to the correct source once sequencing anomalies are accounted for (miss-assignment rate&lt;0.4%). Therefore, the method enables accurate sequencing and assignment of homologous DNA sequences from multiple sources in single high-throughput GS20 run. We observe a bias in the distribution of the differently tagged primers that is dependent on the 5′ nucleotide of the tag. In particular, primers 5′ labelled with a cytosine are heavily overrepresented among the final sequences, while those 5′ labelled with a thymine are strongly underrepresented. A weaker bias also exists with regards to the distribution of the sequences as sorted by the second nucleotide of the dinucleotide tags. As the results are based on a single GS20 run, the general applicability of the approach requires confirmation. However, our experiments demonstrate that 5′primer tagging is a useful method in which the sequencing power of the GS20 can be applied to PCR-based assays of multiple homologous PCR products. The new approach will be of value to a broad range of research areas, such as those of comparative genomics, complete mitochondrial analyses, population genetics, and phylogenetics.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Binladen J, Gilbert MTP, Bollback JP, Panitz F, Bendixen C, et al.  (2007) The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing. PLoS ONE 2(2):
          e197.
            doi:10.1371/journal.pone.0000197</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> November 27, 2006; <strong>Accepted:</strong> January 16, 2007; <strong>Published:</strong> February 14, 2007</p><p><strong>Copyright:</strong> © 2007 Binladen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>JB and EW were supported by the Wellcome Trust, UK, the Carlsberg Foundation, DK, and the National Science Foundation, DK. MTPG acknowledges the Marie Curie Actions FP6-MEIF-CT-2005-025002 ‘FORMAPLEX’ grant for funding his research. JPB and RN were funded by the Danish FSS and the National Science Foundation, DK. None of the sponsors or funders have had any influence on the data or manuscript presented here.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The arrival of the Genome Sequence 20™ DNA Sequencing System (GS20, Roche/454 Life Sciences) and associated ‘Sequencing-by-Synthesis’ protocol has enabled pyrosequencing of up to 25 million nucleotides in a single four-hour reaction <a href="#pone.0000197-Margulies1">[1]</a>. The method employs single molecule amplification prior to sequencing and therefore eliminates the need for prior cloning. In initial implementations of the technology random fragments from DNA extracts have been sequenced without <em>a priori</em> selection of specific genetic regions. As such, all DNA that is present in the sample has a chance of being amplified and sequenced that approximately correspond to its frequency within the DNA extract. The method has proven an efficient tool for use in a number of specific cases, such as the rapid sequencing of relatively small genomes <a href="#pone.0000197-Margulies1">[1]</a>, <a href="#pone.0000197-Hofreuter1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For purposes such as comparative genomics, mitochondrial sequencing, and population genetics, it is of interest to combine the selectivity of primer-based PCR, with the sequencing power of the GS20 platform. The simplest way to achieve this is the use of the GS20 to emulsion PCR (emPCR) then pyrosequence the products of individual PCR reactions. Due to the sequencing power of the GS20 this approach results in hundreds of thousands of individual sequences from a single PCR reaction, each derived directly from a single original template within the reaction <a href="#pone.0000197-Margulies1">[1]</a>. As such, this result is similar to the generation of sequence data through conventional cloning. We henceforth term the GS20 derived sequences as single molecule sequences. Obviously, in many studies the amount of single molecule sequences produced by single GS20 runs is unnecessary and economically unfeasible, unless several PCR products can be processed simultaneously and correctly assigned.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Thomas and co-authors <a href="#pone.0000197-Thomas1">[3]</a> recently took the first step in making this possible by pooling together eleven PCR products, each targeting different regions of the genome, into single sequencing-by-synthesis reactions using the GS20. In this case, the authors could easily sort the sequence data due to the unique genetic sequence of each original target. Furthermore, by sequencing the combined PCR products from separate individual specimens on specially partitioned fragments (1/8 sections) of the GS20 PicoTitrePlate™, they were rapidly able to generate large numbers of sequences from each of the eleven PCR products (≈1,000 per product) <a href="#pone.0000197-Thomas1">[3]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>While this represents an excellent advance in the exploitation of the GS20, in theory the combined “primer specific PCR/GS20” approach can be enhanced even further. For example, the number of sequences generated in even an 1/8<sup>th</sup> run of the GS20 using a 40×75 PicoTitrePlate™ (currently the smallest commercially available subdivision of a single GS20 reaction) is large; in our experience such a run routinely generates at least 6,000, and more commonly over 10,000 sequences per run. With an estimated 10-fold coverage, using the method of Thomas et al <a href="#pone.0000197-Thomas1">[3]</a> this could enable the pooling of 600 PCR products in a single reaction. However, the subsequent identification of the sequence reads would require either the pooling of 600 PCR products targeting unique genetic regions, or, if multiple homologous PCR products were to be co-sequenced (i.e. multiple different products amplified using a single identical primer pair), an <em>a prior</em> knowledge about the exact sequence of each target.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this paper we have overcome this problem, presenting a method where initial PCR primers are 5′-tagged with short nucleotide sequences (tags) in such a way that a unique tagged primer combination can be applied to each specific DNA template source. As sequences generated by the GS20 commence at the very first position of the source DNA fragment, the tags are observed in the generated sequences. Therefore sequences can rapidly be sorted into their original template source using the tags (<a href="#pone-0000197-g001">Figure 1</a>). Currently, the method provides a means for the simultaneous sequencing, generation of single molecule sequences, and assignment of short (~120 bp) from homologous PCR products obtained from multiple individuals. However, as the GS20 sequencing-by-synthesis technologies are developed to increase both the number, and length of the sequences generated, the power of this technique will likewise increase.</p>
<div class="figure" id="pone-0000197-g001"><div class="img"><a name="pone-0000197-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000197.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000197.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The application of 5′ primer tags to the GS20 sequencing-by-synthesis process.</p>
<span>doi:10.1371/journal.pone.0000197.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>In theory, a GS20 reaction that has been performed on a pool of different PCR products at equimolar concentration should generate an equal number of sequences from each PCR product. However, in practice it can be expected that random processes occurring during the procedure will result in a Poisson distributed relative frequency of the final products. In addition to this, the different 5′ tags used on the primers for the initial PCR might potentially bias the final sequence distribution. As a result, the incorporation of too many different PCR products in a single GS20 reaction could result in some of them not being sequenced. In contrast, the incorporation of too few PCR products in a single 454 parallel sequencing run minimises the efficiency, and cost efficacy of the method. Furthermore, as one advantage of the approach is the generation of single molecule sequences, it is useful to empirically determine how many sequences can be expected from each of a set of PCR products that are pooled in equimolar concentration.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>We performed a test involving the analysis of a single genetic marker in DNA extracts from multiple different individuals to investigate the effectiveness of this method (i.e. how many individual PCR products can be expected to be represented among a set number of sequences). This was achieved using a single conventional pair of mammalian mitochondrial DNA (mtDNA) 16S primers <a href="#pone.0000197-Taylor1">[4]</a>. The primers were originally designed as mammalian generic, and amplify an 89–97 bp fragment (133–141 bp including primers) that is discriminatory between mammalian species. The study is thus an analogue to a likely use of the technique - the PCR amplification and sequencing of specific genetic regions from multiple individuals of a single species.</p>

<h4>5′ primer tagging</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The original primers were modified into sixteen unique forward, and sixteen reverse primers through the addition of 5′ dinucleotide tags (<a href="#pone-0000197-t001">Table 1</a>). In contrast to most conventional sequencing platforms, pyrosequencing methods (such as that used by the GS20) generate data from the first base of the fragment sequenced. Thus, the 5′ tags on each primer will be apparent in the final sequence. The sixteen unique forward and reverse primers can be combined to make 16*16 = 256 unique sequence tags. In this way, an investment of thirty-two initial primers could in theory enable the subsequent discrimination of 256 different products. However, under the current status of the sequencing technology, GS20 sequencing reads are limited to approximately 120 bases, thus in this experiment the full sequence (133–141 bp including primer, species dependent) was not returned and our analyses were limited to simply discriminating using the primer at the sequence end of the product. Furthermore, during the GS20 process, single DNA fragments are mobilised to beads in either orientation (c.f. <a href="#pone.0000197-Margulies1">[1]</a> for details). The implication of this is that approximately 50% of each PCR product will be sequenced from the orientation of the forward primer, and 50% from the orientation of the reverse primer. Hence, this made it necessary to label both the forward and reverse end of each PCR product.</p>
<div class="figure" id="pone-0000197-t001"><div class="img"><a name="pone-0000197-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000197.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000197.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>5′ tagged PCR primers</span></strong></p><span>doi:10.1371/journal.pone.0000197.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In addition to the above experiments, three further unique primer pairs were designed and used for PCR, that contain tetranucleotide tails (<a href="#pone-0000197-t001">Table 1</a>) in order to investigate whether an increased tail length affects the efficiency of the method. Increasing the tag sequence would exponentially increase the number of possible unique primer combinations and thus PCR reactions that can be incorporated into a single GS20 sequencing run.</p>


<h4>DNA samples analysed</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>DNA from thirteen species was used as PCR template (<a href="#pone-0000197-t002">Table 2</a>). The target species and size of the PCR insert (excluding primers) were as follows: impala (<em>Aepyceros melampus</em>) 92 bp; grey wolf (<em>Canis lupus</em>) 91 bp; cheetah (<em>Acinonyx jubatus</em>) 91 bp; hippopotamus (<em>Hippopotamus amphibious</em>) 91 bp; lion (<em>Panthera leo</em>) 95 bp; saiga antelope (<em>Saiga tartarica</em>) 93 bp, Mueller's Bornean gibbon (<em>Hylobates muelleri</em>) 94 bp, narwhal (<em>Monodon monoceros</em>) 90 bp; domestic mouse (<em>Mus domesticus</em>) 97 bp; musk ox (<em>Ovibos moschatus</em>) 93 bp; human 94 bp; Burchell's zebra (<em>Equus burchelli</em>) 89 bp; and African buffalo (<em>Syncerus caffer</em>) 94 bp. The DNA was extracted from frozen specimens using the DNEasy tissue extraction kit (Qiagen) following the manufacturer's protocol. To increase the number of different PCR products that we could pool into the GS20-reaction beyond a single product from each of available thirteen extractions, we used individual primer pairs on several different extractions each (<a href="#pone-0000197-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000197-t002"><div class="img"><a name="pone-0000197-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000197.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000197.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Assigned sequence distribution</span></strong></p><span>doi:10.1371/journal.pone.0000197.t002</span></div>

<h4>PCR conditions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We generated 64 differently labelled 16S mtDNA PCR fragments (<a href="#pone-0000197-t002">Table 2</a>). PCRs were performed in 25 µl PCR reactions containing 1× PCR Buffer, 2.5 mM MgCl<sub>2</sub> solution, 0.2 mM dNTP Mix, 1 U Taq DNA Polymerase, 1 µM each primer and 1 µl DNA extract. Cycling was performed using a Mastercycler Gradient Thermal Cycler (Eppendorf) with the following cycle program: Initial denaturation at 94°C for 2 minutes followed by 25 cycles of 94°C for 30 seconds, 56°C for 30 seconds and 72°C for 30 seconds, followed by a final extension of 8 minutes at 72°C. Five µl of the PCR products were visualised on 2% agarose gels using ethidium bromide staining and UV light transillumination. Positive PCR products were purified using the Invisorb Spin PCRapid kit (Invitek) and quantified using a Nanodrop ND-1000 (Nanodrop Technologies). Quantification was performed directly on the purified PCR products (that is, without dilution). Several duplicate measurements indicated that intrasample measurement variation was negligible. Purified yields were between 3.8–26.1 ng/µl (<a href="#pone.0000197.s001">Supplementary Table S1</a>). Subsequently the PCR products were pooled together. The PCR products were at equimolar concentrations (26.1 ng each) with two exceptions; amplification products from the buffalo were added at double concentration (52.2 ng), and PCR products generated from the zebra template used twice the number of different tags. The pooled PCR products were subsequently analysed on the GS20 platform using the complete sample preparation and analytical process, as recommended by the manufacturer (Roche). The initial sample concentration was 9.33 ng/µl and 21 ng (23 µl) was used for the reaction. No nebulization was performed and the average concentration of single stranded library was 75 pg/µl. The calculated dilution factor was 5.25 and sequencing was performed as a full titration run without bead enrichment, i.e. the run was performed on a 40×75 plate, divided into 8 sectors (a titration run uses 4 of these sectors with different numbers of DNA molecules per bead i.e. 1,4,16, and 64 respectively.).</p>


<h4>Conventional sequencing of the targets</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Although the complete 16S mtDNA sequences for most of the species analysed is available in the public domain, we regenerated the target sequences for the thirteen mammal species used using conventional dye-labelled sequencing (Sequencing reactions and analyses performed on Applied Biosystems platforms by Macrogen, Korea). This was to ensure that subsequent analyses did not mistake natural sequence variation with sequencing errors. The thirteen individual 16S mtDNA sequences are deposited in GenBank under the accession numbers EF152485–EF152497.</p>


<h4>Initial assignment of the sequence data</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>As the correct association of tags and sequences is crucial to the approach, we followed very conservative criteria post sequencing in regards to acceptance of the sequence data. Initially, we discarded all sequence reads without an exact match to any of the primers used in the studies (Primer Mismatched Sequences). Subsequently, the identity of the remaining sequences were globally aligned to the thirteen reference sequences (Sanger-sequencing generated) using direct and reverse complementation. The global alignment was performed using ClustalW <a href="#pone.0000197-Thompson1">[5]</a> used the following scoring scheme: matches (+5), mismatches (−4), gap penalty (−10), and a gap extension penalty (−10). The latter penalties were not applied to end gaps. For each alignment a percent identity score was calculated to determine the best match in the following way: excluding end gaps, ambiguous states (Ns) in the 454 sequence, and gaps introduced in the reference sequence during alignment the number of mismatches was calculated.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>If a sequence differed at more than one nucleotide from the highest scoring alignment, then the sequences were discarded into a separate dataset. We refer to these sequences as Non-Assigned sequences, and the remaining sequences are referred to as Assigned sequences. The per nucleotide error rate estimated from this type of data is 7.5×10<sup>−4</sup> 
               <a href="#pone.0000197-Gilbert1">[6]</a>. With reads of a length of ≈100 bp excluding primers, and primers of length 22 bp, the expected proportion of non-assigned sequences is then 2.7×10<sup>−3</sup> and the expected proportion of primer mismatched sequences should be 1.6×10<sup>−2</sup>. Any excess of Non-Assigned or Primer Mismatched Sequences above this level is then due to experimental errors other than sequencing errors, such as contamination.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The identity of the Non-Assigned sequences are of some interest as they may provide information regarding these other sources of experimental error. Thus the Non-assigned sequences were subsequently subjected to BLAST <a href="#pone.0000197-Altschul1">[7]</a> analyses against the NCBI GenBank DNA database in order to determine their identity. During this (and other) BLAST analyses performed, when two or more hits with identical E-score were reported, we prioritised any that matched our 13 target sequences over others.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>GS20 sequences generated</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>6765 DNA sequences were generated using the GS20 platform (Sequence Data S1). The sequence data is provided in the supplemental information. The sequencing was performed as a titration run with no bead enrichment and different DNA/bead rations, therefore the number of sequences is lower than what is previously reported for PCR products (8,000–12,000, <a href="#pone.0000197-Thomas1">[3]</a>). As such the calculations of the sequencing efficiency in this study provides a conservative estimate of the potential power of the technique.</p>


<h4>Sequence analysis</h4>
<h5>Primer Mismatch Sequences.</h5><a id="article1.body1.sec3.sec2.sec1.p1" name="article1.body1.sec3.sec2.sec1.p1"></a><p>Due to the stringent screening criteria applied in this study, 458 (6.8%) of the 6765 initial sequences generated from a 1/8<sup>th</sup> of a plate run on the GS20 were identified as Primer Mismatch Sequences (see above). These grouped as follows: 377 sequences or 5.6% have sequencing errors in the primer sequence, 54 sequences or 0.8% have the primer sequence starting one position off, 3 sequences or 0.04% have the primer sequence starting two positions off, and 24 sequences or 0.4% have the primer sequence starting more than two positions off. As the theoretically expected number of mismatches based on the sequencing error rate is 1.6%, other sources of error (such as damage to the original DNA template, sequencing errors or mtDNA heteroplasmy) may be affecting the results.</p>
<a id="article1.body1.sec3.sec2.sec1.p2" name="article1.body1.sec3.sec2.sec1.p2"></a><p>The 458 Primer Mismatch Sequences were identified using BLAST, revealing that 395 of the sequences (86.2%) match the reference sequences of the study. This includes 81 sequences where the primers are as expected, but positioned one or more base pair positions off the 5′ end of the sequence. Among these, 80 sequences match DNA sequences from species used in this study (<a href="#pone.0000197.s002">Supplementary Table S2</a>). Of sequences containing errors in the primers 313 of 377 (83.0%) matched species used in this study (<a href="#pone.0000197.s003">Supplementary Table S3</a>).</p>
<a id="article1.body1.sec3.sec2.sec1.p3" name="article1.body1.sec3.sec2.sec1.p3"></a><p>That so many sequences contained sequencing errors in the primers (n = 377) was surprising, and warranted further investigation. The sequences could be distinguished into four broad categories as follows: Those that failed show any match to the primer sequences in general (n = 2); Those that were exact match to the core primer but lacked the 5′ tag sequence (n = 121); Those that contained at least one mismatch and no indels (insertion/deletions) (n = 53); and those that contained at least one indel (n = 201) (21 of which also contained a mismatch). We subsequently investigated whether the errors may have arisen during the primer synthesis itself, and not during the sequencing-by-synthesis process. This was tested under the assumptions that a) errors arising during the primer synthesis process would be randomly distributed along the primer sequence, and that b) primers containing errors in the 3′ four nucleotides would bind poorly to the template DNA, thus not enable PCR amplification. If this was the case, then although prior to PCR a random distribution of sequence errors should be observed across the primer sequences, post PCR significantly fewer errors should be observed at the 3′ end of the primer. A χ<sup>2</sup> test on the distribution of the sequencing errors between the five 3′ terminal nucleotides, the next five (middle) nucleotides, and the remaining 5′ nucleotides confirms that there are significantly fewer sequencing errors in the five terminal 3′ nucleotides of the primers (Pearson's χ<sup>2</sup> test, χ<sup>2</sup> = 17.506, p = 0.00001). Therefore the data suggests that at least some of the primer-related errors can be explained by errors during the primer synthesis itself. (We note however that this test was only performed on the primers that contained mismatches without indels, due to the difficulty of accurately aligning the primers that contained indels).</p>

<h5>Assigned Sequences.</h5><a id="article1.body1.sec3.sec2.sec2.p1" name="article1.body1.sec3.sec2.sec2.p1"></a><p>The remaining 6307 sequences were identified through a global alignment to the 13 reference sequences. Of these, 5642 sequences (89%) diverged by no more than 1bp from one of the reference sequences, and could thus be assigned to one of the taxa analysed in the study (<a href="#pone-0000197-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec2.sec2.p2" name="article1.body1.sec3.sec2.sec2.p2"></a><p>Twenty sequences (0.4%) were miss-assigned to an incorrect identity. Strikingly, more than half of the miss-assigned sequences (n = 11) are of human origin. Based on the omnipresent nature of human DNA in most laboratory settings, this bias is likely due to contamination during extractions and/or PCR setup. Ignoring all human sequences (n = 138), only 9 sequences could be miss-assigned out of a total of 5504 GS20 non-human sequences (0.00163 percent miss-assignment). Based on a GS20 sequencing error rate of 7×10<sup>−4</sup> 
                  <a href="#pone.0000197-Gilbert1">[6]</a>, the expected number of miss-assignments due to sequencing errors in the dinucleotide tag is 2×5504× (7×10<sup>−4</sup>)≈7.4 mismatches. Thus, the obtained result is are remarkably close to the expected and miss-assignments of non-human sequences can be explained by sequencing errors alone. This result shows that despite the possibility of sequencing (and other) errors, the assignment based on 5′ tagging is remarkably reliable.</p>

<h5>Non-Assigned Sequences.</h5><a id="article1.body1.sec3.sec2.sec3.p1" name="article1.body1.sec3.sec2.sec3.p1"></a><p>Of the 6307 sequences that did not contain a primer error, 665 sequences diverge from the reference sequences by more than 1 bp. However, the expected number of such sequences based on the known sequencing error rate is only 6307×(2.7×10<sup>−3</sup>)≈17, suggesting a significant impact of other factors. Obvious candidates include the amplification of non-targeted genomic sequences (for example laboratory contamination), DNA damage or heteroplasmy in the original template, and errors introduced into the DNA during the initial PCR stage (where a non-proof reading polymerase was used). Of these, 491 sequences (73.8%) match DNA sequences from one of the 13 original taxa amplified by the highest BLAST hit (<a href="#pone.0000197.s004">Supplementary Table S4</a>).</p>

<h5>Sequence distribution.</h5><a id="article1.body1.sec3.sec2.sec4.p1" name="article1.body1.sec3.sec2.sec4.p1"></a><p>On average each of the 64 amplicons (grouping forward and reverse reads) had 85× coverage with a standard deviation of 32 (<a href="#pone-0000197-t002">Table 2</a>). The coverage variation is very large. At the extremes we observe that the zebra DNA amplified with a TA tag generating a single forward read and no reverse read, while the zebra amplified with the CC tag resulting in more than 100 forward and reverse reads. There is no evidence that forward or reverse strands are unequally distributed within the sequence dataset (Pearson's χ<sup>2</sup> test, χ<sup>2</sup> = 27.2793, df = 18, p = 0.0739).</p>

<h5>5′ tag distribution.</h5><a id="article1.body1.sec3.sec2.sec5.p1" name="article1.body1.sec3.sec2.sec5.p1"></a><p>A Pearson's χ<sup>2</sup> test strongly rejects an equal distribution among the different tags (χ<sup>2</sup> = 1725.28, df = 18, p = 0.0). The divergence from the expected numbers are primarily caused by an excess of 5′CN (N representing A,T,G,C) tagged amplicons, and a depletion of 5′TN tags (<a href="#pone-0000197-t003">Table 3</a>), with a small surplus of 5′GN and small depletion of 5′AN tags. We also investigated whether the identity of the second nucleotide within each tag led to non-random distribution of the final sequences. This was achieved using χ<sup>2</sup> analysis on the 4 independent datasets constituted by the 5′ nucleotide A, C, G and T respectively (i.e. the 4 primer groups AN, CN, GN and TN). The results indicate that with the exception of the 5′ T labelled tags, the sequences were non randomly distributed (AN, χ<sup>2</sup> = 60.0, d.f. = 3, p = 0.0; CN, χ<sup>2</sup> = 10.0, d.f. = 3, p = 0.0186; GN, χ<sup>2</sup> = 16.3, d.f. = 3, p = 0.0009; TN, χ<sup>2</sup> = 2.35, d.f. = 3, p = 0.5039). Due to the limited number of tetranucleotide tags analysed, we were unable to investigate the effect of the identity of the 3<sup>rd</sup> and 4<sup>th</sup> position nucleotides.</p>
<div class="figure" id="pone-0000197-t003"><div class="img"><a name="pone-0000197-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000197" data-uri="info:doi/10.1371/journal.pone.0000197.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000197.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000197.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000197.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000197.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000197.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Observed and Expected sequence distributions sorted by 5′ tag composition</span></strong></p><span>doi:10.1371/journal.pone.0000197.t003</span></div>


<h4>Effect of 5′ nucleotide on PCR products</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In light of the finding that the identity of the dinucleotide tag has an important effect on final sequence distribution, we also performed several statistical tests to investigate whether the 5′ nucleotide of the dinucleotide tag might also affect the initially generated PCR products. Specifically we investigated whether an association exists between the 5′ nucleotide of the tag, and either the final concentrations of the amplified products (ng/µl) or the percent of sequence discrepancies/incorrect templates among the sequences. To test the former, Student's t-tests were performed on the difference in PCR yield (ng/µl) (<a href="#pone.0000197.s005">supplementary data</a>) between primers starting with A, C, G, and T. After Bonferroni correction, only the direct comparison between 5′C and 5′G labeled primers showed any significant difference (A–C p = 0.942, A–G p = 0.397, A–T p = 0.752, C–G 0.003, C–T 0.640, G–T 0.067). To test the latter we investigated whether the number of errors in the sequences are homogeneously distributed among primers starting with A, C, G, and T. A χ2 RxC test indicates that the errors are not distributed homogenously among the primers (χ<sup>2</sup> = 102.25, p&lt;&lt;0.0001).</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>To summarise therefore, there is evidence that the identity of the first and second position with the dinucleotide tags affects the final sequence distribution, and the 5′ nucleotide of the tag affects the levels of sequence errors.</p>


<h4>Dinucleotide vs tetranucleotide tag performance</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>In comparison to the dinucleotide tags, the performance of the tetranucleotide tagged primers was less efficient than predicted (<a href="#pone-0000197-t003">Table 3</a>). Although the small number of tetranucleotide tagged primers tested makes statistically supported comparisons difficult, our observations on the data indicate that overall the rate of sequence miss-assignment for these primers was lower than for the dinucleotide tags.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3>
<h4>Caveats</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>In this study we present data describing the viability and limitations of a pooled-PCR based approach to GS20 sequencing. Naturally the specific results of this study may be dependent on the genetic region targeted and the PCR primers/target details. As such we caution that while we demonstrate the overall efficiency of this method, future studies are required to confirm the global extent of our observations on the primer efficiencies. In addition, the results of this study are clearly dependent on the quantification method used. In the study we chose the ND-1000 Nanodrop due to its commonplace availability, plus ease and rapidity of use. While more accurate quantification methods such as real-time quantitative PCR may act to reduce the variation between the numbers of single molecule sequenced PCR sequences, the efficiency of using this easily applied technique is in our eyes acceptable, and a great improvement over the costs (both financially and time) that conventional cloning plus sequencing requires.</p>


<h4>Variation in single molecule sequence numbers</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Despite pooling the PCR products together in equal concentrations, we find significant variation in the coverage level of the different amplicons, at the extremes varying from a single read to more than 200 reads. This variation is greater than would be expected due to random processes and although there is weak evidence that in some cases the 5′ nucleotide of the tag may play a role, it is more likely an artefact of natural errors. Firstly, some of the variation is likely a result of the ND-1000 measurement accuracy, whether due to the fact that the measurement accuracy of the spectrophotometer (≈±1 ng) is vastly higher than the mass of a single PCR product, or due to the fact that original genomic DNA within the final PCR product lead to overestimation of the amplicon content. Some evidence for this second hypothesis is offered by the finding that although twice as much buffalo DNA was added to the GS20 process, this did not generate twice the number of sequences. Further support for this hypothesis is the observation of different DNA sources among the GS20-sequences (see Supplemental Tables <a href="#pone.0000197.s002">S2</a>, <a href="#pone.0000197.s003">S3</a> and<a href="#pone.0000197.s004">S4</a>) and human DNA in 7 non-human PCRs. As the majority of the sequences derive from in-study amplicons the carry over of any residual template DNA is unlikely to be a problem with regards to replacing PCR amplicons during the GS20 process. A more likely explanation for the single molecule sequence variations is the effect of 5′-tag composition.</p>


<h4>5′-tag efficiency</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Although we observe all of the originally used 5′-tagged PCR primers among our data, there is clear evidence that some of the tag sequences appear preferable over others during the DNA manipulation processes that occur prior to GS20 sequencing. Most obviously, we note that dinucleotide tags containing the 5′-CN motif are significantly overrepresented in the final data. In contrast, those with the 5′-TN motif appear significantly underrepresented. Furthermore, there is evidence that the second base in the tag is also plays an affect on the sequencing efficiency. As mentioned previously, we caution however that this analysis is based on a single primer set and on one GS20 analysis, and as such, may be specific to this analysis. However, we speculate that a potential cause of the bias may be due to the GS20 DNA preparation processes. During the first stages of DNA preparation for the GS20 run, DNA is blunt ligated to adaptor sequences using the T4 Ligase enzyme Quick Ligase (New England Biolabs), followed by 3′ nick repair using <em>Bst</em> DNA polymerase, Large Fragment (New England Biolabs). Following this process all DNA is ligated to similar ligator sequences, thus this is most likely the stage where the selection occurs. As such, it would seem that the ligase has some propensity for 5′ C-labelled DNA. Under this hypothesis, it would seem logical that a similar bias should exist in DNA sequences generated from unamplified DNA using the GS20. This does not seem to be the case however. In an examination of the frequency distribution of the 5′ base of a dataset of 1214 recently published GS20 generated mammoth mtDNA sequences <a href="#pone.0000197-Gilbert1">[6]</a>) we find no significant difference between the observed and expected base frequencies (Calculated factoring in the total sequence base composition, Chi Squared Test p = 0.01). As such, in the absence of further studies to confirm the pattern we remain unable to explain our observations.</p>


<h4>Sequence Errors</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Overall we found that 89% of the total sequences generated in this dataset were identical or very close (1 bp divergence) from the target sequences. This finding was based upon our conservative data assignment criteria, and without the use of a PCR enzyme with proof-reading capacity for the initial PCRs (the use of which is likely to increase the sequence accuracy). Furthermore, in an applied experimental situation the single molecule sequences produced from each PCR product could be used to further aid the assignation of target sequences. From a practical point of view, this error rate will have ramifications on study design, with regards to the numbers of single molecule sequences required per individual. This will be exceedingly important in situations where variation is expected in the target PCR products, for example studies on allelic variation (whether due to natural or artificial genetic variation).</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The 11% of the sequences that we allocated to the primer mismatch or non-assigned data sets were broadly comprised of 3 groups of sequence. Of these, two (rejections due to primer or amplicon sequencing errors) are likely artefacts of the current GS20 pyrosequencing chemistry, or due to errors in the original primer synthesis process. The third group are sequences that did not BLAST against our 13 target species as the closest hit. This might be explained due to several reasons. First is contamination arising during the extraction or PCR setup. This is clearly evident in the sporadic human DNA presence, although also supported in BLAST results that clearly match other species, for example the occurrence of waterbuck (<em>Kobus ellipsiprymnus</em>) sequences (a species that has been worked on previously in our laboratories). Clearly these sequences could be minimised through the adoption of stricter PCR setup protocols. An additional cause for the results may also be the fact that many species are closely related in the targeted 16S mtDNA region, and as such small modifications in the sequences due to DNA damage, heteroplasmy, or PCR enzyme error might lead to erroneous sequence identification.</p>


<h4>Applied use of the technique and future prospects</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In response to our observations on how the specific tag sequence affects sequence errors and final sequence distribution, and as to the apparent reduced miss-assignment rates of the tetranucleotide tags, we suggest the following primer design guidelines. In the absence of specific information as to the behaviour or individual tags, it is difficult to pool the products at specific predetermined ratios in order to account for the sequencing behaviours. However, as there appears to be a strong effect that the final 5′ nucleotide has on the sequencing efficiency, we suggest that the ultimate 5′ nucleotide should be conserved among the different tag to give a more uniform distribution of the GS20 sequences. Although C provides the highest levels of sequences in our data, our data indicates that 5′-TN tags show little or no effect of the second nucleotide on final sequence distribution, thus may ultimately prove more useful. With regards to tag length, our limited observations are that increased length reduces the competitive binding efficiency of the PCR product (against shorter tags) indicates that all PCR products should incorporate a tag of identical length (the length will depend on the number of different tags required). The tetranucleotide tags performed well in the initial specific PCRs, yielding 7–26 ng/µl purified DNA. With a fixed 5′ nucleotide, these would still enable 4^4 (64) different forward and reverse primers to be used, which in combination enable the generation of 64*64 = 4069 identifiable PCR products (assuming that the reverse primer sequence is returned by the GS20). If tagged PCR products targeting multiple different genetic regions are also pooled (either generated through single- or multiplex PCR assays), then clearly the number of products in a single reaction increases dramatically.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>In this study we demonstrate the principal of the application of 5′ tagged PCR primers in the sequencing of homologous PCR products on the GS20 platform. As we have noted several times, our observations on the method kinetics are preliminary, and more detailed follow up studies will be required to clarify the power of the method. Furthermore, as improvements are made in sequencing-by-synthesis methods, the efficiency and power of the 5′tagged PCR method is predicted to increase. Based on our data however, it is possible to provide a more quantitative assessment of the efficacy of the method as a sequencing platform. A single run of the GS20 platform can generate 25 million nucleotides <a href="#pone.0000197-Margulies1">[1]</a>. Taking into account an 89% efficacy, the average alignment depth for 50 mitochondrial chromosomes each of 16 kbp length, will be 27.8×. Based on our distribution of number of sequences obtained from each species, we would expect the specimen with the most shallow coverage among the 50 specimens to have a coverage of 11×. If the mitochondrial genome has been sequenced in units of 100 bs, the chance that any part of the chromosome has not a coverage of at least 1 is then 1−(1−e<sup>−11</sup>)<sup>160</sup> = 2.7×10<sup>−3</sup>. These numbers may improve as the experimental techniques also improve, for example by pooling DNA considering the dinucleotide motif.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>At the moment, it is clear that the GS20 platform should be preferable in most studies where cloning otherwise would be required. Our results also suggest that even when taking into account problems relating to unequal pools of DNA or differences due to dinucleotide motif, the GS20 platform provides a time effective alternative to Sanger sequencing. As the pricing of the method decreases, it may also become more cost effective than Sanger sequencing. In conclusion, we believe that this new approach combining 5′tagged PCR with GS20 sequencing will be of importance to a broad range of research areas where large-scale comparisons of homologous DNA sequences from multiple sources are needed such as is the case in comparative genomics, population genetics, and phylogenetics.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000197"></div><a name="pone.0000197.s001" id="pone.0000197.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000197.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">DNA concentrations of original PCR products</p>
<p class="siDoi">doi:10.1371/journal.pone.0000197.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0000197.s002" id="pone.0000197.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000197.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Blast Results for the 81 sequences that have an exact primer match but are &gt;1 nucleotide offset from 5′ end of the sequence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000197.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
<a name="pone.0000197.s003" id="pone.0000197.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000197.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Blast Results for the 377 sequences that did not have an exact primer match.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000197.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.12 MB XLS)</p>
<a name="pone.0000197.s004" id="pone.0000197.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000197.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Blast Results for the 665 sequences that had an exact primer match but were excluded because of more than 1 mismatch with reference sequences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000197.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.17 MB XLS)</p>
<a name="pone.0000197.s005" id="pone.0000197.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000197.s005">Sequence Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Fasta file</p>
<p class="siDoi">doi:10.1371/journal.pone.0000197.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(1.18 MB RTF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank H. Siegismund, P. Arctander, T. Kristensen, P. F. Campos, E. Garde, P. Gravlund, The Jacobo Weinstock, and Copenhagen Zoo (Genebank Copenhagen Zoo) for providing tissue samples, T. Brand for helping out with practical work, and A. Götherström for providing inspiring settings in which the initial ideas and discussions where fostered. We also acknowledge the anonymous reviewer, who provided many useful suggestions that improved this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RN EW MG JB. Performed the experiments: MG JB CB FP. Analyzed the data: RN MG JB JB. Contributed reagents/materials/analysis tools: EW MG JB JB CB. Wrote the paper: EW MG JB. Other: Conceived initial idea to apply 5′ tagged primers to the GS20 during discussion on the topic with MTPG: JB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000197-Margulies1" id="pone.0000197-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="929156"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000197-Hofreuter1" id="pone.0000197-Hofreuter1"></a>Hofreuter D, Tsai J, Watson RO, Novik V, Altman B, et al.  (2006) Unique features of a highly pathogenic Campylobacter jejuni strain. Infect Immun  74: 4694–4707.  <ul class="find" data-citedArticleID="929154" data-doi="10.1128/iai.00210-06"><li><a href="http://dx.doi.org/10.1128/iai.00210-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unique+features+of+a+highly+pathogenic+Campylobacter+jejuni+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unique+features+of+a+highly+pathogenic+Campylobacter+jejuni+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000197-Thomas1" id="pone.0000197-Thomas1"></a>Thomas RK, Nickerson E, Simons JF, Janne PA, Tengs T, et al.  (2006) Sensitive mutation detection in heterogeneous cancer specimens by massively parallel picoliter reactor sequencing. Nat Med  12: 852–855.  <ul class="find" data-citedArticleID="929160" data-doi="10.1038/nm1437"><li><a href="http://dx.doi.org/10.1038/nm1437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitive+mutation+detection+in+heterogeneous+cancer+specimens+by+massively+parallel+picoliter+reactor+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitive+mutation+detection+in+heterogeneous+cancer+specimens+by+massively+parallel+picoliter+reactor+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000197-Taylor1" id="pone.0000197-Taylor1"></a>Taylor PG (1996) Reproducibility of ancient DNA sequences from extinct Pleistocene fauna. Mol Biol Evol  13: 283–285.  <ul class="find" data-citedArticleID="929158" data-doi="10.1093/oxfordjournals.molbev.a025566"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025566" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reproducibility+of+ancient+DNA+sequences+from+extinct+Pleistocene+fauna." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reproducibility+of+ancient+DNA+sequences+from+extinct+Pleistocene+fauna.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000197-Thompson1" id="pone.0000197-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucl Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="929162" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000197-Gilbert1" id="pone.0000197-Gilbert1"></a>Gilbert MTP, Binladen J, Miller W, Wiuf C, Willerslev E, et al. Recharacterization of ancient DNA miscoding lesions: insights in the era of sequencing-by-synthesis. Nucleic Acids Res..   In press.  <ul class="find" data-citedArticleID="929152" data-doi="10.1093/nar/gkl483"><li><a href="http://dx.doi.org/10.1093/nar/gkl483" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recharacterization+of+ancient+DNA+miscoding+lesions%3A+insights+in+the+era+of+sequencing-by-synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recharacterization+of+ancient+DNA+miscoding+lesions%3A+insights+in+the+era+of+sequencing-by-synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000197-Altschul1" id="pone.0000197-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J. Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="929150" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.XML" value="152747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.PDF" value="237312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.g001.PNG_L" value="1797539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.g001.PNG_M" value="136468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.g001.PNG_S" value="4032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.g001.TIF" value="2569344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.g001.PNG_I" value="50347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t001.PNG_L" value="150498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t001.PNG_M" value="136861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t001.PNG_S" value="8109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t001.TIF" value="795992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t001.PNG_I" value="96503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t002.PNG_L" value="346344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t002.PNG_M" value="75141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t002.PNG_S" value="3793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t002.TIF" value="2027400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t002.PNG_I" value="30928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t003.PNG_L" value="138257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t003.PNG_M" value="92404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t003.PNG_S" value="6886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t003.TIF" value="701762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.t003.PNG_I" value="71952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.s001.XLS" value="37888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.s002.XLS" value="51200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.s003.XLS" value="117760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.s004.XLS" value="165888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000197.s005.RTF" value="1180307"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000197&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000197" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000197&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000197&volume=&issue=&title=The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing&author_name=Jonas%20Binladen%2C%20M.%20Thomas%20P.%20Gilbert%2C%20Jonathan%20P.%20Bollback%2C%20Frank%20Panitz%2C%20Christian%20Bendixen%2C%20Rasmus%20Nielsen%2C%20Eske%20Willerslev&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000197" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197&amp;t=The%20Use%20of%20Coded%20PCR%20Primers%20Enables%20High-Throughput%20Sequencing%20of%20Multiple%20Homolog%20Amplification%20Products%20by%20454%20Parallel%20Sequencing" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197&title=The%20Use%20of%20Coded%20PCR%20Primers%20Enables%20High-Throughput%20Sequencing%20of%20Multiple%20Homolog%20Amplification%20Products%20by%20454%20Parallel%20Sequencing&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197&amp;title=The%20Use%20of%20Coded%20PCR%20Primers%20Enables%20High-Throughput%20Sequencing%20of%20Multiple%20Homolog%20Amplification%20Products%20by%20454%20Parallel%20Sequencing" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000197&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Use of Coded PCR Primers Enables High-Throughput Sequencing of Multiple Homolog Amplification Products by 454 Parallel Sequencing';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000197';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Use%20of%20Coded%20PCR%20Primers%20Enables%20High-Throughput%20Sequencing%20of%20Multiple%20Homolog%20Amplification%20Products%20by%20454%20Parallel%20Sequencing http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000197" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000197" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="23232"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23232"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequencing%22" title="Search for articles in the subject area:'DNA sequencing'"><div class="flagText">DNA sequencing</div></a>
              <div data-categoryid="47891" data-articleid="23232"
                   data-categoryname="DNA sequencing"
                   class="flagImage" title="Flag 'DNA sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="23232"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23232"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23232"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="23232"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23232"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5024'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5547'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2792&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7935">Use of coded primers requires minimum difference to be effective</a><br>Posted by shuse</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
