

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001322"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001322" />

    <meta name="citation_title" content="A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates"/>
    <meta itemprop="name" content="A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates"/>

      <meta name="citation_author" content="Ida Elken SÃ¸nderby"/>
            <meta name="citation_author_institution" content="Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Bjarne Gram Hansen"/>
            <meta name="citation_author_institution" content="Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Nanna Bjarnholt"/>
            <meta name="citation_author_institution" content="Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Carla Ticconi"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Barbara Ann Halkier"/>
            <meta name="citation_author_institution" content="Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Daniel J. Kliebenstein"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of California at Davis, Davis, California, United States of America"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001322.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1322"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Glucosinolate metabolism and its control.; citation_author=CD Grubb; citation_author=S Abel; citation_journal_title=Trends Plant Sci; citation_volume=11; citation_number=1; citation_pages=89-100; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biology and Biochemistry of Glucosinolates.; citation_author=BA Halkier; citation_author=J Gershenzon; citation_journal_title=Annu Rev Plant Biol; citation_volume=57; citation_number=2; citation_pages=303-333; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The glucosinolate-myrosinase system in an ecological and evolutionary context.; citation_author=DJ Kliebenstein; citation_author=J Kroymann; citation_author=T Mitchell-Olds; citation_journal_title=Curr Opin Plant Biol; citation_volume=8; citation_number=3; citation_pages=264-271; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development of isothiocyanate-enriched broccoli, and its enhanced ability to induce phase 2 detoxification enzymes in mammalian cells.; citation_author=R Mithen; citation_author=K Faulkner; citation_author=R Magrath; citation_author=P Rose; citation_author=G Williamson; citation_journal_title=Theor Appl Genet; citation_volume=106; citation_number=4; citation_pages=727-734; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of quantitative trait loci controlling glucosinolates, myrosinase and insect resistance in Arabidopsis thaliana.; citation_author=D Kliebenstein; citation_author=D Pedersen; citation_author=B Barker; citation_author=T Mitchell-Olds; citation_journal_title=Genetics; citation_volume=161; citation_number=5; citation_pages=325-332; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Major signaling pathways modulate Arabidopsis glucosinolate accumulation and response to both phloem-feeding and chewing insects.; citation_author=I Mewis; citation_author=HM Appel; citation_author=A Hom; citation_author=R Raina; citation_author=JC Schultz; citation_journal_title=Plant Physiol; citation_volume=138; citation_number=6; citation_pages=1149-1162; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Modulation of CYP79 genes and glucosinolate profiles in Arabidopsis by defense signaling pathways.; citation_author=MD Mikkelsen; citation_author=BL Petersen; citation_author=E Glawischnig; citation_author=AB Jensen; citation_author=E Andreasson; citation_journal_title=Plant Physiol; citation_volume=131; citation_number=7; citation_pages=298-308; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A Myb homologue, ATR1, activates tryptophan gene expression in Arabidopsis.; citation_author=J Bender; citation_author=GR Fink; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=8; citation_pages=5655-5660; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis ATR1 Myb transcription factor controls indolic glucosinolate homeostasis.; citation_author=JL Celenza; citation_author=JA Quiel; citation_author=GA Smolen; citation_author=H Merrikh; citation_author=AR Silvestro; citation_journal_title=Plant Physiol; citation_volume=137; citation_number=9; citation_pages=253-262; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The R2R3-MYB transcription factor HAG1/MYB28 is a regulator of methionine-derived glucosinolate biosynthesis in Arabidopsis thaliana.; citation_author=T Gigolashvili; citation_author=R Yatusevich; citation_author=B Berger; citation_author=C Muller; citation_author=UI Flugge; citation_journal_title=Plant J; citation_volume=51; citation_number=10; citation_pages=247-261; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characterization of the Arabidopsis TU8 glucosinolate mutation, an allele of TERMINAL FLOWER2.; citation_author=JH Kim; citation_author=TP Durrett; citation_author=RL Last; citation_author=G Jander; citation_journal_title=Plant Mol Biol; citation_volume=54; citation_number=11; citation_pages=671-682; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The tu8 mutation of Arabidopsis thaliana encoding a heterochromatin protein 1 homolog causes defects in the induction of secondary metabolite biosynthesis.; citation_author=RN Bennett; citation_author=T Wenke; citation_author=B Freudenberg; citation_author=FA Mellon; citation_author=J Ludwig-Muller; citation_journal_title=Plant Biol (Stuttg); citation_volume=7; citation_number=12; citation_pages=348-357; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis IQD1, a novel calmodulin-binding nuclear protein, stimulates glucosinolate accumulation and plant defense.; citation_author=M Levy; citation_author=Q Wang; citation_author=R Kaspi; citation_author=MP Parrella; citation_author=S Abel; citation_journal_title=Plant J; citation_volume=43; citation_number=13; citation_pages=79-96; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DOF transcription factor AtDof1.1 (OBP2) is part of a regulatory network controlling glucosinolate biosynthesis in Arabidopsis.; citation_author=A Skirycz; citation_author=M Reichelt; citation_author=M Burow; citation_author=C Birkemeyer; citation_author=J Rolcik; citation_journal_title=Plant J; citation_volume=47; citation_number=14; citation_pages=10-24; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Jasmonate-dependent induction of indole glucosinolates in Arabidopsis by culture filtrates of the nonspecific pathogen Erwinia carotovora.; citation_author=G Brader; citation_author=E Tas; citation_author=ET Palva; citation_journal_title=Plant Physiol; citation_volume=126; citation_number=15; citation_pages=849-860; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative quantitative trait loci mapping of aliphatic, indolic and benzylic glucosinolate production in Arabidopsis thaliana leaves and seeds.; citation_author=DJ Kliebenstein; citation_author=J Gershenzon; citation_author=T Mitchell-Olds; citation_journal_title=Genetics; citation_volume=159; citation_number=16; citation_pages=359-370; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genomic survey of gene expression diversity in Arabidopsis thaliana.; citation_author=DJ Kliebenstein; citation_author=MA West; citation_author=LH van; citation_author=K Kim; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=172; citation_number=17; citation_pages=1179-1189; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mapping and analysis of quantitative trait loci in experimental populations.; citation_author=RW Doerge; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=18; citation_pages=43-52; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetics of gene expression surveyed in maize, mouse and man.; citation_author=EE Schadt; citation_author=SA Monks; citation_author=TA Drake; citation_author=AJ Lusis; citation_author=N Che; citation_journal_title=Nature; citation_volume=422; citation_number=19; citation_pages=297-302; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetical genomics: the added value from segregation.; citation_author=RC Jansen; citation_author=JP Nap; citation_journal_title=Trends in Genetics; citation_volume=17; citation_number=20; citation_pages=388-391; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic dissection of transcriptional regulation in budding yeast.; citation_author=RB Brem; citation_author=G Yvert; citation_author=R Clinton; citation_author=L Kruglyak; citation_journal_title=Science; citation_volume=296; citation_number=21; citation_pages=752-755; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis.; citation_author=MA West; citation_author=K Kim; citation_author=DJ Kliebenstein; citation_author=LH van; citation_author=RW Michelmore; citation_journal_title=Genetics; citation_volume=175; citation_number=22; citation_pages=1441-1450; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Regulating gene expression: surprises still in store.; citation_author=RC Jansen; citation_author=JP Nap; citation_journal_title=Trends Genet; citation_volume=20; citation_number=23; citation_pages=223-225; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of QTLs controlling gene expression networks defined a priori.; citation_author=DJ Kliebenstein; citation_author=MA West; citation_author=LH van; citation_author=O Loudet; citation_author=RW Doerge; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=24; citation_pages=308; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis.; citation_author=MA West; citation_author=LH van; citation_author=A Kozik; citation_author=DJ Kliebenstein; citation_author=RW Doerge; citation_journal_title=Genome Res; citation_volume=16; citation_number=25; citation_pages=787-795; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The R2R3-MYB gene family in Arabidopsis thaliana.; citation_author=R Stracke; citation_author=M Werber; citation_author=B Weisshaar; citation_journal_title=Curr Opin Plant Biol; citation_volume=4; citation_number=26; citation_pages=447-456; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MYB-bHLH-WD40 protein complex and the evolution of cellular diversity.; citation_author=NA Ramsay; citation_author=BJ Glover; citation_journal_title=Trends Plant Sci; citation_volume=10; citation_number=27; citation_pages=63-70; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The mechanics of cell fate determination in petals.; citation_author=C Martin; citation_author=K Bhatt; citation_author=K Baumann; citation_author=H Jin; citation_author=S Zachgo; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=357; citation_number=28; citation_pages=809-813; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A R2R3 type MYB transcription factor is involved in the cold regulation of CBF genes and in acquired freezing tolerance.; citation_author=M Agarwal; citation_author=Y Hao; citation_author=A Kapoor; citation_author=CH Dong; citation_author=H Fujii; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=29; citation_pages=37636-37645; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HOS10 encodes an R2R3-type MYB transcription factor essential for cold acclimation in plants.; citation_author=J Zhu; citation_author=PE Verslues; citation_author=X Zheng; citation_author=BH Lee; citation_author=X Zhan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=30; citation_pages=9966-9971; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Guard cells: transcription factors regulate stomatal movements.; citation_author=J Gray; citation_journal_title=Curr Biol; citation_volume=15; citation_number=31; citation_pages=R593-R595; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The transcription factor HIG1/MYB51 regulates indolic glucosinolate biosynthesis in Arabidopsis thaliana.; citation_author=T Gigolashvili; citation_author=B Berger; citation_author=HP Mock; citation_author=C Muller; citation_author=B Weisshaar; citation_journal_title=Plant J; citation_volume=50; citation_number=32; citation_pages=886-901; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Identification of a flavin monoxygenase as the S-oxygenating enzyme in aliphatic glucosinolate biosynthesis in Arabidopsis.; citation_author=BG Hansen; citation_author=DJ Kliebenstein; citation_author=BA Halkier; citation_journal_title=Plant J; citation_volume=902-910; citation_number=33; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis.; citation_author=MY Hirai; citation_author=K Sugiyama; citation_author=Y Sawada; citation_author=T Tohge; citation_author=T Obayashi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=34; citation_pages=6478-6483; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characterization of seed-specific benzoyloxyglucosinolate mutations in Arabidopsis thaliana.; citation_author=DJ Kliebenstein; citation_author=JC D'Auria; citation_author=AS Behere; citation_author=JH Kim; citation_author=KL Gunderson; citation_journal_title=Plant J; citation_volume=51; citation_number=35; citation_pages=1062-1076; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Composition and content of glucosinolates in developing Arabidopsis thaliana.; citation_author=BL Petersen; citation_author=S Chen; citation_author=CH Hansen; citation_author=CE Olsen; citation_author=BA Halkier; citation_journal_title=Planta; citation_volume=214; citation_number=36; citation_pages=562-571; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Variation of glucosinolate accumulation among different organs and developmental stages of Arabidopsis thaliana.; citation_author=PD Brown; citation_author=JG Tokuhisa; citation_author=M Reichelt; citation_author=J Gershenzon; citation_journal_title=Phytochemistry; citation_volume=62; citation_number=37; citation_pages=471-481; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis ref2 mutant is defective in the gene encoding CYP83A1 and shows both phenylpropanoid and glucosinolate phenotypes.; citation_author=MR Hemm; citation_author=MO Ruegger; citation_author=C Chapple; citation_journal_title=Plant Cell; citation_volume=15; citation_number=38; citation_pages=179-194; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Secondary metabolites influence Arabidopsis/Botrytis interactions: variation in host production and pathogen sensitivity.; citation_author=DJ Kliebenstein; citation_author=HC Rowe; citation_author=KJ Denby; citation_journal_title=Plant J; citation_volume=44; citation_number=39; citation_pages=25-36; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptional repression by AtMYB4 controls production of UV-protecting sunscreens in Arabidopsis.; citation_author=H Jin; citation_author=E Cominelli; citation_author=P Bailey; citation_author=A Parr; citation_author=F Mehrtens; citation_journal_title=EMBO J; citation_volume=19; citation_number=40; citation_pages=6150-6161; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cloning of the SNG1 gene of Arabidopsis reveals a role for a serine carboxypeptidase-like protein as an acyltransferase in secondary metabolism.; citation_author=C Lehfeldt; citation_author=AM Shirley; citation_author=K Meyer; citation_author=MO Ruegger; citation_author=JC Cusumano; citation_journal_title=Plant Cell; citation_volume=12; citation_number=41; citation_pages=1295-1306; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The hyper-fluorescent trichome phenotype of the brt1 mutant of Arabidopsis is the result of a defect in a sinapic acid: UDPG glucosyltransferase.; citation_author=T Sinlapadech; citation_author=J Stout; citation_author=MO Ruegger; citation_author=M Deak; citation_author=C Chapple; citation_journal_title=Plant J; citation_volume=49; citation_number=42; citation_pages=655-668; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-Wide Insertional Mutagenesis of Arabidopsis thaliana.; citation_author=JM Alonso; citation_author=AN Stepanova; citation_author=TJ Leisse; citation_author=CJ Kim; citation_author=H Chen; citation_journal_title=Science; citation_volume=301; citation_number=43; citation_pages=653-657; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An Arabidopsis thaliana T-DNA mutagenized population (GABI-Kat) for flanking sequence tag-based reverse genetics.; citation_author=MG Rosso; citation_author=Y Li; citation_author=N Strizhov; citation_author=B Reiss; citation_author=K Dekker; citation_journal_title=Plant Mol Biol; citation_volume=53; citation_number=44; citation_pages=247-259; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multiple Independent Defective Suppressor-mutator Transposon Insertions in Arabidopsis: A Tool for Functional Genomics.; citation_author=AF Tissier; citation_author=S Marillonnet; citation_author=V Klimyuk; citation_author=K Patel; citation_author=MA Torres; citation_journal_title=Plant Cell; citation_volume=11; citation_number=45; citation_pages=1841-1852; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Linking Metabolic QTL with Network and cis-eQTL Controlling Biosynthetic Pathways.; citation_author=A Wentzell; citation_author=H Rowe; citation_author=B Hansen; citation_author=C Ticconi; citation_author=B Halkier; citation_journal_title=PLoS Genetics; citation_volume=3; citation_number=46; citation_pages=1687-1701; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene duplication in the diversification of secondary metabolism: tandem 2-oxoglutarate-dependent dioxygenases control glucosinolate biosynthesis in Arabidopsis.; citation_author=DJ Kliebenstein; citation_author=VM Lambrix; citation_author=M Reichelt; citation_author=J Gershenzon; citation_author=T Mitchell-Olds; citation_journal_title=Plant Cell; citation_volume=13; citation_number=47; citation_pages=681-693; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MAM3 Catalyzes the Formation of All Aliphatic Glucosinolate Chain Lengths in Arabidopsis.; citation_author=S Textor; citation_author=JW de Kraker; citation_author=B Hause; citation_author=J Gershenzon; citation_author=JG Tokuhisa; citation_journal_title=Plant Physiol; citation_volume=144; citation_number=48; citation_pages=60-71; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic control of natural variation in Arabidopsis glucosinolate accumulation.; citation_author=DJ Kliebenstein; citation_author=J Kroymann; citation_author=P Brown; citation_author=A Figuth; citation_author=D Pedersen; citation_journal_title=Plant Physiol; citation_volume=126; citation_number=49; citation_pages=811-825; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Biosynthesis of methionine-derived glucosinolates in Arabidopsis thaliana: recombinant expression and characterization of methylthioalkylmalate synthase, the condensing enzyme of the chain-elongation cycle.; citation_author=S Textor; citation_author=S Bartram; citation_author=J Kroymann; citation_author=KL Falk; citation_author=A Hick; citation_journal_title=Planta; citation_volume=218; citation_number=50; citation_pages=1026-1035; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Glucosinolate and amino acid biosynthesis in Arabidopsis.; citation_author=B Field; citation_author=G Cardon; citation_author=M Traka; citation_author=J Botterman; citation_author=G Vancanneyt; citation_journal_title=Plant Physiol; citation_volume=135; citation_number=51; citation_pages=828-839; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bay-0ÃShahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis.; citation_author=O Loudet; citation_author=S Chaillou; citation_author=C Camilleri; citation_author=D Bouchez; citation_author=F Niel-Vedele; citation_journal_title=Theor Appl Genet; citation_volume=104; citation_number=52; citation_pages=1173-1184; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Windows QTL Cartographer 2.5.; citation_author=S Wang; citation_author=CJ Basten; citation_author=Z-B Zeng; citation_number=53; citation_date=2006; citation_publisher=Department of Statistics, North Carolina State University; " />
      <meta name="citation_reference" content="citation_title=QTL Cartographer, version 1.13.; citation_author=CJ Basten; citation_author=BS Weir; citation_author=Z-B Zeng; citation_number=54; citation_date=1999; citation_publisher=Department of Statistics, North Carolina State University; " />
      <meta name="citation_reference" content="citation_title=Estimating the genetic architecture of quantitative traits.; citation_author=ZB Zeng; citation_author=CH Kao; citation_author=CJ Basten; citation_journal_title=Genet Res; citation_volume=74; citation_number=55; citation_pages=279-289; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Empirical threshold values for quantitative trait mapping.; citation_author=GA Churchill; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=138; citation_number=56; citation_pages=963-971; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Permutation tests for multiple loci affecting a quantitative character.; citation_author=RW Doerge; citation_author=GA Churchill; citation_journal_title=Genetics; citation_volume=142; citation_number=57; citation_pages=285-294; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Advancing uracil-excision based cloning towards an ideal technique for cloning PCR fragments.; citation_author=HH Nour-Eldin; citation_author=BG Hansen; citation_author=MH Norholm; citation_author=JK Jensen; citation_author=BA Halkier; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=58; citation_pages=e122; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ti plasmid vector for the introduction of DNA into plant cells without alteration of their normal regeneration capacity.; citation_author=P Zambryski; citation_author=H Joos; citation_author=C Genetello; citation_author=J Leemans; citation_author=MV Montagu; citation_journal_title=EMBO J; citation_volume=2; citation_number=59; citation_pages=2143-2150; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana.; citation_author=SJ Clough; citation_author=AF Bent; citation_journal_title=Plant J; citation_volume=16; citation_number=60; citation_pages=735-743; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Positional Cloning in Arabidopsis. Why It Feels Good to Have a Genome Initiative Working for You.; citation_author=W Lukowitz; citation_author=CS Gillmor; citation_author=WR Scheible; citation_journal_title=Plant Physiol; citation_volume=123; citation_number=61; citation_pages=795-806; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data.; citation_author=RA Irizarry; citation_author=B Hobbs; citation_author=F Collin; citation_author=YD Beazer-Barclay; citation_author=KJ Antonellis; citation_journal_title=Biostatistics; citation_volume=4; citation_number=62; citation_pages=249-264; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Controlling The False Discovery Rate-A Practical and Powerful Approach to Multiple Testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=J R Statist Soc B; citation_volume=57; citation_number=63; citation_pages=289-300; citation_date=1995; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates"/>
    <meta name="twitter:description" content="BackgroundGlucosinolates are natural metabolites in the order Brassicales that defend plants against both herbivores and pathogens and can attract specialized insects. Knowledge about the genes controlling glucosinolate regulation is limited. Here, we identify three R2R3 MYB transcription factors regulating aliphatic glucosinolate biosynthesis in Arabidopsis by combining several systems biology tools.Methodology/Principal FindingsMYB28 was identified as a candidate regulator of aliphatic glucosinolates based on its co-localization within a genomic region controlling variation both in aliphatic glucosinolate content (metabolite QTL) and in transcript level for genes involved in the biosynthesis of aliphatic glucosinolates (expression QTL), as well as its co-expression with genes in aliphatic glucosinolate biosynthesis. A phylogenetic analysis with the R2R3 motif of MYB28 showed that it and two homologues, MYB29 and MYB76, were members of an Arabidopsis-specific clade that included three characterized regulators of indole glucosinolates. Over-expression of the individual MYB genes showed that they all had the capacity to increase the production of aliphatic glucosinolates in leaves and seeds and induce gene expression of aliphatic biosynthetic genes within leaves. Analysis of leaves and seeds of single knockout mutants showed that mutants of MYB29 and MYB76 have reductions in only short-chained aliphatic glucosinolates whereas a mutant in MYB28 has reductions in both short- and long-chained aliphatic glucosinolates. Furthermore, analysis of a double knockout in MYB28 and MYB29 identified an emergent property of the system since the absence of aliphatic glucosinolates in these plants could not be predicted by the chemotype of the single knockouts.Conclusions/SignificanceIt seems that these cruciferous-specific MYB regulatory genes have evolved both overlapping and specific regulatory capacities. This provides a unique system within which to study the evolution of MYB regulatory factors and their downstream targets."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001322.g008"/>

  <meta property="og:title" content="A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5237'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=197'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7682&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001322">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001322" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001322&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ida Elken SÃ¸nderby
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ida Elken SÃ¸nderby, Bjarne Gram Hansen</p>

              
              

                <p>Affiliation:
                  Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bjarne Gram Hansen
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ida Elken SÃ¸nderby, Bjarne Gram Hansen</p>

              
              

                <p>Affiliation:
                  Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nanna Bjarnholt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carla Ticconi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Sciences, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Barbara Ann Halkier, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plant Biochemistry Laboratory, Department of Plant Biology, Center for Molecular Plant Physiology (PlaCe), Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel J. Kliebenstein
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kliebenstein@ucdavis.edu">kliebenstein@ucdavis.edu</a></p>

                <p>Affiliation:
                  Department of Plant Sciences, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001322</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001322-g001" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g002" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g003" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-t001" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-t002" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-t003" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g004" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g005" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g006" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g007" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-t004" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-t005" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001322-g008" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001322">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001322" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Glucosinolates are natural metabolites in the order Brassicales that defend plants against both herbivores and pathogens and can attract specialized insects. Knowledge about the genes controlling glucosinolate regulation is limited. Here, we identify three R2R3 MYB transcription factors regulating aliphatic glucosinolate biosynthesis in <em>Arabidopsis</em> by combining several systems biology tools.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p><em>MYB28</em> was identified as a candidate regulator of aliphatic glucosinolates based on its co-localization within a genomic region controlling variation both in aliphatic glucosinolate content (metabolite QTL) and in transcript level for genes involved in the biosynthesis of aliphatic glucosinolates (expression QTL), as well as its co-expression with genes in aliphatic glucosinolate biosynthesis. A phylogenetic analysis with the R2R3 motif of <em>MYB28</em> showed that it and two homologues, <em>MYB29</em> and <em>MYB76</em>, were members of an <em>Arabidopsis</em>-specific clade that included three characterized regulators of indole glucosinolates. Over-expression of the individual <em>MYB</em> genes showed that they all had the capacity to increase the production of aliphatic glucosinolates in leaves and seeds and induce gene expression of aliphatic biosynthetic genes within leaves. Analysis of leaves and seeds of single knockout mutants showed that mutants of <em>MYB29</em> and <em>MYB76</em> have reductions in only short-chained aliphatic glucosinolates whereas a mutant in <em>MYB28</em> has reductions in both short- and long-chained aliphatic glucosinolates. Furthermore, analysis of a double knockout in <em>MYB28</em> and <em>MYB29</em> identified an emergent property of the system since the absence of aliphatic glucosinolates in these plants could not be predicted by the chemotype of the single knockouts.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>It seems that these cruciferous-specific <em>MYB</em> regulatory genes have evolved both overlapping and specific regulatory capacities. This provides a unique system within which to study the evolution of <em>MYB</em> regulatory factors and their downstream targets.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>SÃ¸nderby IE, Hansen BG, Bjarnholt N, Ticconi C, Halkier BA, et al.  (2007) A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates. PLoS ONE 2(12):
          e1322.
            doi:10.1371/journal.pone.0001322</p><p><strong>Academic Editor: </strong>Ivan Baxter, Purdue University, United States of America</p><p><strong>Received:</strong> November 6, 2007; <strong>Accepted:</strong> November 19, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> Â© 2007 SÃ¸nderby et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The presented work was supported by The Danish National Research Foundation through its support to PlaCe (Center for Molecular Plant Physiology) (Barbara Ann Halkier) and by NSF grants DBI#0642481 to Dan J. Kliebenstein. The PhD stipends of Bjarne Gram Hansen and Ida Elken SÃ¸nderby were funded by FOBI (Graduate School in Biotechnology) and Faculty of Life Sciences (Copenhagen University), respectively. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Glucosinolates are amino acid-derived natural plant products characteristic of the order Brassicales, including the agriculturally important oilseed rape, the Brassica vegetables and the model plant <em>Arabidopsis</em> <a href="#pone.0001322-Grubb1">[1]</a>, <a href="#pone.0001322-Halkier1">[2]</a>. Glucosinolates are major defensive metabolites against both herbivores and pathogens and function as attractants for specialized insects <a href="#pone.0001322-Halkier1">[2]</a>, <a href="#pone.0001322-Kliebenstein1">[3]</a>. In addition, glucosinolates have gained increasing importance due to their function as flavour compounds and cancer-preventive agents in the human diet, as well as their potential for use as biopesticides <a href="#pone.0001322-Mithen1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Glucosinolates are divided into aliphatic, aromatic and indole glucosinolates dependent on the nature of the amino acid from which they are derived. The three subgroups have a pathway for the synthesis of the core structure, where distinct, but homologous genes catalyze the conversion of amino acids to the corresponding glucosinolate. Methionine-derived glucosinolates are modified via elongation of the side chain prior to entering the pathway for the synthesis of the core glucosinolate structure. Short-chained, aliphatic glucosinolates have been through one to four cycles of the chain-elongation machinery, whereas the long-chained have been through five or six cycles (<a href="#pone-0001322-g001">Figure 1</a>). Furthermore, glucosinolate structures are often secondarily modified on their side chains with e.g. hydroxylations and alkenylations. The completion of the <em>Arabidopsis</em> genomic sequence has resulted in identification of most of the biosynthetic genes involved in the formation of the core structure, the chain-elongation machinery and secondary modifications (<a href="#pone-0001322-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001322-g001"><div class="img"><a name="pone-0001322-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Biosynthetic pathway of aliphatic glucosinolates in <em>Arabidopsis</em> Col-0.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>A. The chain-elongation machinery. Methionine enters the chain-elongation cycle via deamination by BCAT4 and is subsequently condensed with acetyl-CoA in a reaction catalyzed by MAM1 and MAM3. MAM1 can catalyze one to four condensation cycles, whereas MAM3 catalyzes one to six cycles. Subsequently, an isomerization and oxidation-decarboxylation step occurs and the molecule can re-enter the cycle or enter the core pathway following a transamination step. B. Synthesis of the core methylthio glucosinolate structure. The first enzymatic step has side chain specificity with CYP79F1 that converts both short- and long- chained methionine derivatives (n = 1â6) to the corresponding aldoxime whereas CYP79F2 only takes the long-chained methionine derivatives (n = 5â6). C. Secondary modifications. Short- and long-chained methylthio glucosinolates can be secondarily modified to methylsulfinyl glucosinolates in Col-0 leaves. In the seeds, the short-chained methylsulfinyl glucosinolates can be further modified to hydroxy and benzoyloxy forms. Characterized enzymes in the pathway are noted next to the reaction arrows.</p>
<span>doi:10.1371/journal.pone.0001322.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Our knowledge about the genes controlling glucosinolate regulation is limited to a few regulatory genes. This is in contrast to the complex regulation known for glucosinolate content which includes herbivore, pathogen, biotic and abiotic stress responses <a href="#pone.0001322-Kliebenstein2">[5]</a>â<a href="#pone.0001322-Mikkelsen1">[7]</a>. Two genes are known to specifically regulate indole glucosinolates. <em>ATR1</em>/<em>MYB34</em> was identified as a gain-of-function mutant, <em>atr1D,</em> in a screen for mutants with <strong><span class="underline">a</span></strong>ltered <strong><span class="underline">t</span></strong>ryptophan <strong><span class="underline">r</span></strong>egulation <a href="#pone.0001322-Bender1">[8]</a>, <a href="#pone.0001322-Celenza1">[9]</a>. The transcript levels of <em>CYP79B2, CYP79B3</em> and <em>CYP83B1,</em> which encodes for enzymes in the biosynthesis of indole glucosinolates, are up- and down-regulated in <em>atr1D</em> and a knockout mutant, respectively <a href="#pone.0001322-Celenza1">[9]</a>. <em>MYB51/HIG1</em>, a close homologue of <em>ATR1</em>/<em>MYB34</em>, was identified as an activation tagged line with high indole glucosinolates <a href="#pone.0001322-Gigolashvili1">[10]</a>. Complementation studies suggest that <em>MYB34</em> and <em>MYB51</em> have overlapping, but distinct functions in regulating biosynthetic genes for indole glucosinolates.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Other regulators have been shown to regulate both indole and aliphatic glucosinolates. <em>TFL2</em>, a plant homologue of the animal heterochromation protein (HP1) was shown to affect both aliphatic and indole glucosinolates, probably through indirect developmental effects <a href="#pone.0001322-Kim1">[11]</a>, <a href="#pone.0001322-Bennett1">[12]</a>. A screen for high-glucosinolate mutants identified <em>IQD1</em>, a calmodulin-binding nuclear protein, as a potential regulator <a href="#pone.0001322-Levy1">[13]</a>. The <em>iqd1</em>mutant has elevated levels of both aliphatic and indole glucosinolates, although transcript levels of genes in the indole glucosinolate pathway are up-regulated and transcripts in the aliphatic pathway are repressed <a href="#pone.0001322-Levy1">[13]</a>. Over-expression of a third transcription factor, <em>AtDof1.1</em>, results in an approximately two-fold increase in the levels of both indole and short-chained aliphatic glucosinolates. Accordingly, these <em>35S:Dof1.1</em> lines up-regulate transcript levels for <em>ATR1/MYB34</em> and the genes involved in indole glucosinolate biosynthesis, <em>CYP79B2, CYP79B3</em> and <em>CYP83B1,</em> as well as of <em>MAM1</em> involved in chain-elongation machinery for aliphatic glucosinolates <a href="#pone.0001322-Skirycz1">[14]</a>. Although these three regulatory genes, <em>TFL2, IQD1</em> and <em>AtDof1.1</em>, affect both aliphatic and indole glucosinolates, many studies indicate that aliphatic and indole glucosinolates are typically not co-ordinately regulated in response to pathogens, hormones or insect herbivores <a href="#pone.0001322-Kliebenstein2">[5]</a>â<a href="#pone.0001322-Mikkelsen1">[7]</a>, <a href="#pone.0001322-Brader1">[15]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>When we initiated our study, the aim was to identify regulators specific for biosynthesis of the aliphatic glucosinolates, which had not yet been identified. Our approach was to exploit the natural variation in production of glucosinolates among <em>Arabidopsis</em> ecotypes using QTL mapping to identify candidate regulators. We have used a combination of metabolite QTL and expression QTL (eQTL) analysis within the Bay-0ÃSha Recombinant Inbred Lines (RILs) to show that a region of the genome controls variation in both the accumulation of aliphatic glucosinolates and transcript levels for the aliphatic biosynthetic genes. Combining this data with information from co-expression databases and phylogenetic analysis identified three candidate transcription factors for aliphatic glucosinolates, <em>MYB28, MYB29</em>, and <em>MYB76</em>, belonging to an <em>Arabidopsis</em> specific MYB group. Glucosinolate and transcript profiling of over-expression lines indicated that <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> positively regulate accumulation of aliphatic glucosinolates and the corresponding transcripts. This was further supported by a decrease in levels of aliphatic glucosinolates in T-DNA insertion mutants of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em>, where the mutants reveal differential specificity for chain lengths. Further, a double knockout mutant in <em>MYB28</em> and <em>MYB29</em> was devoid of aliphatic glucosinolates and displayed very low levels of transcripts not predicted from the single knockout mutants. During our study, Hirai <em>et al.</em> (2007) provided evidence that <em>MYB28</em> is a regulator of aliphatic glucosinolates using an omics-based approach and Gigolashvili <em>et al.</em> (2007) has used activation tag approach to identify <em>MYB28</em> as regulator of aliphatic glucosinolates. Their data is discussed in relation to our data showing that all three MYBs are regulators of aliphatic glucosinolates independent of biotic stresses.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A QTL on Chromosome V Appears to be a Regulatory QTL</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Previous work using the Cvi-1ÃCol-0 and Cvi-1ÃL<em>er Arabidopsis</em> RIL populations identified a metabolite QTL controlling the accumulation of total aliphatic glucosinolates at the bottom of Chromosome V <a href="#pone.0001322-Kliebenstein2">[5]</a>, <a href="#pone.0001322-Kliebenstein3">[16]</a>, <a href="#pone.0001322-Kliebenstein4">[17]</a>. An analysis of the Bay-0ÃSha RILs for total content of aliphatic glucosinolates showed the presence of a similar QTL within this population (<a href="#pone-0001322-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001322-g002"><div class="img"><a name="pone-0001322-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Bay-0ÃSha QTL for Total Content of Aliphatic Glucosinolates on Chromosome V.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The total content of aliphatic glucosinolates was measured in two independent experiments on 403 lines from the Bay-0ÃSha RIL population. The LOD plot from QTL Cartographer CIM module for QTL on chromosome V is shown. The significance threshold was obtained by 1,000 random permutations within QTL Cartographer. A 4-LOD confidence interval around the QTL is depicted (vertical dotted lines) and the arrows show the position of the genes marking the 4 LOD interval.</p>
<span>doi:10.1371/journal.pone.0001322.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Similar to QTLs controlling metabolite accumulation, regions in the genome that control variation in the transcript level for a given gene (eQTL) can be identified <a href="#pone.0001322-Doerge1">[18]</a>â<a href="#pone.0001322-Brem1">[21]</a>. Using microarray technology, genome-wide expression profiling of 211 individuals from the Bay-0ÃSha RIL population was performed to map global eQTLs <a href="#pone.0001322-West1">[22]</a>. eQTLs can be separated into <em>cis-</em> and <em>trans-</em>acting eQTLs. The <em>cis</em>-eQTLs is likely caused by sequence variation around the region of the gene itself, e.g. a promoter polymorphism that gives rise to differential expression of the gene in the two parents. <em>trans</em>-eQTLs, on the other hand, are due to polymorphisms at another location on the genome than the actual physical position of the gene <a href="#pone.0001322-Jansen2">[23]</a>. Such a region could potentially contain variation within a regulatory factor, potentially a transcription factor. Rather than looking for regions with <em>trans</em>-eQTLs for one gene, one can look for regions with <em>trans</em>-eQTLs for a set of genes (a gene network analysis) that share a common biological function, e.g. a metabolic pathway <a href="#pone.0001322-Kliebenstein5">[24]</a>. In this way, eQTL analysis may potentially identify master regulators of a network.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>A previously conducted gene network analysis based on the genes of both the aliphatic and indole glucosinolate biosynthetic pathways suggested that the QTL on Chromosome V (<a href="#pone-0001322-g002">Figure 2</a>) may affect glucosinolate content via altered transcript levels for the aliphatic glucosinolate biosynthetic genes <a href="#pone.0001322-Kliebenstein5">[24]</a>. We utilized the transcript data set from 211 RILs from the Bay-0ÃSha population <a href="#pone.0001322-West1">[22]</a> to test if the individual biosynthetic genes in the aliphatic pathway had eQTLs controlling variation in their transcript levels that also co-localized to the metabolite QTL on chromosome V. 15 of the 25 genes characterized or predicted to be involved in the biosynthesis or regulation of aliphatic glucosinolates had a statistically significant <em>trans-</em>eQTL that co-localized with the metabolite QTL at the bottom of chromosome V (<a href="#pone-0001322-g002">Figure 2</a> and <a href="#pone.0001322.s002">Table S1</a>). These data suggest that variation in a regulatory factor within this region may control the metabolite QTL and eQTL for aliphatic glucosinolates. Supporting this idea is the observation that for 13 of the 15 significant genes, the Sha allele at this <em>trans-</em>eQTL led to higher transcript levels which correlates with the same Sha allele having elevated aliphatic glucosinolate content. Thus, this locus may enhance the total amount of aliphatic glucosinolates by increasing transcript level for some of the biosynthetic genes.</p>


<h4>The candidate regulator of aliphatic glucosinolates belongs to the R2R3 transcription factor family</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To identify candidate regulatory genes within this QTL, we scanned all the genes physically located within the 4-LOD interval of the QTL for the presence of a <em>cis</em>-eQTL controlling their expression (<a href="#pone-0001322-g002">Figure 2</a>). Regulatory genes with a <em>cis</em>-eQTL in this 4-LOD interval have the potential to result in <em>trans</em>-eQTL for genes localized in other parts of the genome (<a href="#pone.0001322.s003">Table S2</a>). The 4-LOD interval corresponded to a region spanning approximately 400 genes from <em>At5g59140</em> to <em>At5g63020</em> (<a href="#pone-0001322-g002">Figure 2</a>). Among these 400 genes, we identified 63 genes whose difference in transcript levels between Bay-0 and Sha identified a <em>cis</em>-eQTL (<a href="#pone.0001322.s003">Table S2</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>This list of 63 candidate genes was next filtered to identify genes whose expression correlated with characterized genes of the aliphatic glucosinolate biosynthetic using the public co-expression database ATTED-II (<a href="http://www.atted.bio.titech.ac.jp/">http://www.atted.bio.titech.ac.jp/</a>) (<a href="#pone.0001322.s003">Table S2</a>). The filtering step left only one candidate gene, <em>At5g61420</em>, that co-expressed with biosynthetic genes for aliphatic glucosinolates and that had a <em>cis</em>-eQTL which led to the Sha allele having approximately 3-fold more transcript than Bay. Further, this gene is physically located within 1 cM of the QTL peak as defined using both the metabolite and eQTL data with a dense marker map within this region (<a href="#pone-0001322-g002">Figure 2</a>) <a href="#pone.0001322-West2">[25]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p><em>At5g61420</em> is annotated as <em>MYB28</em>, a member of the R2R3 MYB transcription factor family. The R2R3 MYB family of transcription factors is plant-specific and holds ~125 members in <em>Arabidopsis</em> <a href="#pone.0001322-Stracke1">[26]</a>. It is characterized by having two repeats of DNA binding domains termed R2 and R3 in the N-terminal end, each containing three regularly spaced conserved tryptophan residues <a href="#pone.0001322-Stracke1">[26]</a>, while the C-terminal end is believed to confer DNA activation or repression. R2R3 MYB transcription factors are involved in many plant-specific processes such as the phenylpropanoid pathway <a href="#pone.0001322-Ramsay1">[27]</a>, control of cell shape and differentiation <a href="#pone.0001322-Martin1">[28]</a>, cold regulation <a href="#pone.0001322-Agarwal1">[29]</a>, <a href="#pone.0001322-Zhu1">[30]</a>, stomatal aperture <a href="#pone.0001322-Gray1">[31]</a> as well as regulation of indole glucosinolates <a href="#pone.0001322-Celenza1">[9]</a>, <a href="#pone.0001322-Gigolashvili2">[32]</a>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Since glucosinolates are limited to the order Brassicales, we hypothesized that an aliphatic glucosinolate regulatory factor, similar to the <em>S</em>-oxygenating FMO of aliphatic glucosinolates <a href="#pone.0001322-Hansen1">[33]</a>, would not have homologues within the published rice and poplar genome sequences because both species do not contain glucosinolates. Using the <em>MYB28</em> R2R3 motif, we identified all related protein sequences in the genome of <em>Arabidopsis</em>, rice and poplar, and used the R2R3 domains from these proteins to create a phylogenetic tree (<a href="#pone-0001322-g003">Figure 3</a>, <a href="#pone.0001322.s001">Figure S1</a>, <a href="#pone.0001322.s004">Table S3</a>). The tree was similar using both neighbor joining and maximum parsimony analysis (data not shown). The unrooted tree shows that the closest homologues of <em>MYB28</em> are <em>MYB29</em>, <em>MYB76</em>, <em>MYB122</em>, <em>MYB51</em> and <em>MYB34</em>. The absence of annotated proteins from the dicot poplar and rice genomes within this clade suggests that the MYBs in this clade are specific to at least <em>Arabidopsis</em>. Additionally, no proteins that cluster within this clade were identified in sorghum, maize or any EST library. In support of the potential function of this MYB clade in glucosinolate production is the fact that <em>MYB51</em>, <em>MYB34</em> and <em>MYB122</em> have been shown to regulate indole glucosinolates levels in plants <a href="#pone.0001322-Celenza1">[9]</a>, <a href="#pone.0001322-Gigolashvili2">[32]</a>. Additionally, during the preparation of this manuscript, <em>MYB28</em> was shown to be a regulator of aliphatic glucosinolates <a href="#pone.0001322-Gigolashvili1">[10]</a>, <a href="#pone.0001322-Hirai1">[34]</a>. This generates the hypothesis that this clade arose to regulate glucosinolates and that the <em>MYB28/29/76</em> sub-clade may constitute regulators of aliphatic glucosinolates.</p>
<div class="figure" id="pone-0001322-g003"><div class="img"><a name="pone-0001322-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phylogenetic tree of plant MYBs related to <em>MYB28</em>.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The entire MYB complement from the genomic sequence for rice, <em>Arabidopsis</em> and poplar with a BLAST <em>P</em> value of 10e<sup>â60</sup> or lower when tested against the <em>MYB28, MYB29, MYB76</em> or <em>ATR1/MYB34</em> R2R3 domain, were obtained and translated into their corresponding amino acid sequence. A complete neighbor joining tree with 1000 bootstraps was conducted in ClustalX and the unrooted cladogram is shown. All branches were supported by, at minimum, 630 bootstraps. A distance scale is shown to the bottom left. <a href="#pone.0001322.s004">Table S3</a> contains the identification numbers for the sequences and <a href="#pone.0001322.s001">Figure S1</a> the alignments.</p>
<span>doi:10.1371/journal.pone.0001322.g003</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p><em>MYB29(At5g07690)</em> and <em>MYB76(At5g07700)</em> co-localize with a QTL at the top of chromosome V for aliphatic glucosinolate content in two published QTL populations, Cvi-1ÃL<em>er</em> and Cvi-1ÃCol-0 <a href="#pone.0001322-Kliebenstein2">[5]</a>, <a href="#pone.0001322-Kliebenstein3">[16]</a>. Analysis of gene expression data in the AtGenExpress database (<a href="http://www.weigelworld.org/resources/microarray/AtGenExpress/">http://www.weigelworld.org/resources/micâroarray/AtGenExpress/</a>) and the Expression Level Polymorphism project database (<a href="http://elp.ucdavis.edu/database.htm">http://elp.ucdavis.edu/database.htm</a>) revealed that <em>MYB29</em> and <em>MYB76</em> are differentially expressed in Cvi and L<em>er</em> as well as in Cvi and Col-0. Further, <em>MYB29</em> and <em>MYB76</em> show co-expression with characterized and predicted aliphatic glucosinolate genes in the ATTED-II public database (<a href="#pone.0001322.s005">Table S4</a>). Given the phylogenetic relationship, association with multiple aliphatic glucosinolate QTLs and co-expression relationships, we proceeded to test the ability of all three MYBs, 28, 29 and 76, to regulate aliphatic glucosinolate accumulation and gene expression.</p>


<h4>Accumulation of aliphatic glucosinolates in 35S:MYB over-expression lines</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Transgenic plants with the three candidate MYB genes driven by the Cauliflower Mosaic Virus 35S promoter were generated to test if the genes can regulate production of aliphatic glucosinolates in <em>Arabidopsis</em>. Ten <em>35S:MYB28, 35S:MYB29</em> and <em>35S:MYB76</em> lines were chosen for foliar glucosinolate analysis. Four, three and four lines of <em>35S:MYB28</em>, <em>35S:MYB29</em> and <em>35S:MYB76</em>, respectively, exhibited elevated levels of aliphatic glucosinolates. For further analysis, we chose two independent transgenic lines for each <em>35S:MYB</em> construct showing a significant increase in aliphatic glucosinolates.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Glucosinolate analyses on three-week-old rosette leaves showed that over-expression of all three <em>MYB</em> genes led to increased total content of aliphatic glucosinolates in comparison to wild-type Col-0 (<a href="#pone-0001322-t001">Table 1</a> and <a href="#pone-0001322-t002">2</a>, <a href="#pone.0001322.s006">Table S5</a>). In contrast, the level of total indole glucosinolates was not significantly affected (<a href="#pone-0001322-t001">Table 1</a> and <a href="#pone-0001322-t002">2</a>, <a href="#pone.0001322.s006">Table S5</a>). All plant experiments were conducted independently at two locations (University of Copenhagen and UC Davis) and the data combined into the presented ANOVA (<a href="#pone-0001322-t001">Table 1</a> and <a href="#pone-0001322-t002">2</a>, <a href="#pone.0001322.s006">Table S5</a>). The ANOVA showed that glucosinolate levels between the two selected transgenic lines for each <em>35S:MYB</em> transgene were not significantly different (<a href="#pone.0001322.s006">Table S5</a>). This allows us to combine the mean of the two independent lines to test for the effect of introducing <em>35S:MYB</em> transgenes into wildtype.</p>
<div class="figure" id="pone-0001322-t001"><div class="img"><a name="pone-0001322-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Average Foliar Glucosinolate Content in plants containing the <em>35S:MYB28</em> transgene.</span></strong></p><span>doi:10.1371/journal.pone.0001322.t001</span></div><div class="figure" id="pone-0001322-t002"><div class="img"><a name="pone-0001322-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Average Foliar Glucosinolate Content in plants containing the <em>35S:MYB29</em> or <em>35S:MYB76</em> transgene.</span></strong></p><span>doi:10.1371/journal.pone.0001322.t002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In addition to affecting total content of aliphatic glucosinolates, the three MYB genes altered the composition of the aliphatic glucosinolates present in the leaves. All three <em>35S:MYB</em> transgenes resulted in significantly elevated levels of the short-chained 4MSB and 5MSP whereas the level of 4MTB was significantly lowered (<a href="#pone-0001322-t001">Table 1</a> and <a href="#pone-0001322-t002">2</a>, <a href="#pone.0001322.s006">Table S5</a>). A list of the glucosinolate abbreviations used is provided in <a href="#pone.0001322.s007">Table S6</a>. In contrast, the different <em>35S:MYB</em> transgenes exhibited divergent effects on long-chained aliphatic glucosinolates, e.g. methylsulfinyloctyl glucosinolate, 8MSO. Over-expression of <em>MYB29</em> and <em>MYB76</em> conferred a significant increase in 8MSO levels, whereas over-expression of <em>MYB28</em> did not alter the content of 8MSO (<a href="#pone-0001322-t001">Tables 1</a> and <a href="#pone-0001322-t002">2</a>). In a separate experiment, we tested for potential influences from maternal effects by selecting progeny segregating for the presence and absence of transgene from heterozygous T<sub>2</sub> plant for each selected transgenic line. Previous analysis has shown that maternal effects can be a significant determinant of glucosinolate accumulation <a href="#pone.0001322-Kliebenstein6">[35]</a>. A comparison of segregants with the transgene to those without the transgene showed that in all instances, the presence of the <em>35S:MYB</em> transgene led to increased content of aliphatic glucosinolates, irrespective of the mother plant (data not shown).</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Glucosinolate contents and profiles vary between different tissues <a href="#pone.0001322-Petersen1">[36]</a>, <a href="#pone.0001322-Brown1">[37]</a>. To investigate whether over-expression of the three MYB transcription factors conferred changes to the levels and composition of aliphatic glucosinolates in seeds as well, glucosinolates were extracted and analyzed from seeds originating from the plants used for glucosinolate analysis in leaves. All <em>35S:MYB28</em>, <em>35S:MYB29</em> and <em>35S:MYB76</em> lines showed elevated levels of aliphatic glucosinolates in seeds (<a href="#pone-0001322-t003">Table 3</a>, <a href="#pone.0001322.s008">Table S7</a>). Similar to what was observed in foliar tissue, the increase of aliphatic glucosinolates in seeds within the <em>35S:MYB28</em> lines was entirely due to a rise in short-chained aliphatic glucosinolates. In fact, a significant decrease in long-chained aliphatic glucosinolates was observed (<a href="#pone-0001322-t003">Table 3</a>). Seeds of <em>35S:MYB29</em> and <em>35S:MYB76</em> lines had an overall increase in total aliphatic glucosinolates with the most significant effects being on the longer of the short-chained glucosinolates, 5MSP, 6MTH and 6MSH (<a href="#pone-0001322-t003">Table 3</a>). Thus, all three MYB genes can specifically alter accumulation of aliphatic glucosinolates when over-expressed using a 35S promoter, and the compositional differences between the <em>35S:MYB</em> transgenes suggest that they are not operating via identical mechanisms.</p>
<div class="figure" id="pone-0001322-t003"><div class="img"><a name="pone-0001322-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Average Seed Glucosinolate Content in transgenic <em>35S:MYB</em> lines.</span></strong></p><span>doi:10.1371/journal.pone.0001322.t003</span></div>

<h4>Gene expression analysis in <em>35S:MYB</em> over-expression lines</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The observation that over-expression of the three MYB genes resulted in elevated content of aliphatic glucosinolates led us to test if the transcript levels of genes in the biosynthetic pathway were concurrently elevated in the different genotypes. Affymetrix ATH1 genechip microarrays were used to measure transcript accumulation in wild-type and the two selected transgenic lines for each <em>35S:MYB</em> transgene. As for the glucosinolate analysis described above, an ANOVA analysis of transcript levels between the selected transgenic lines showed no significant difference, which allowed us to combine the mean of the two independent lines to test the effect of the introducing the MYB transgene rather than the effect of one single transgenic line versus wild-type.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Elevated accumulation of aliphatic glucosinolates might be expected to affect the entire sulphur metabolism of the plant due to the pull on the methionine pool. Consequently, we utilized a pathway ANOVA <a href="#pone.0001322-Kliebenstein5">[24]</a> approach to test the impact of the MYB over-expression on the major sulfur-utilization pathways, i.e. sulfate assimilation, cysteine production, methionine production, aliphatic glucosinolates, indole glucosinolates, homocysteine conversion and SAM production, as well as on the characterized transcription factors for indole and aliphatic glucosinolates (<a href="#pone.0001322.s009">Table S8</a>) . The predefined gene set for each individual pathway used in the pathway ANOVA are listed in <a href="#pone.0001322.s009">Table S8</a>.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The pathway ANOVA revealed that the primary effect of the three <em>35S:MYB</em> transgenes within these pathways is to induce the biosynthesis of aliphatic glucosinolates since this was the pathway showing the largest effect in both magnitude and statistical support (<a href="#pone-0001322-g004">Figure 4</a> and <a href="#pone.0001322.s010">Table S9</a>). Another common effect of over-expression of the three MYB genes was to lower transcript levels for genes required to convert methionine into SAM. This could potentially increase the pool of methionine available for production of aliphatic glucosinolates (<a href="#pone-0001322-g004">Figure 4</a> and <a href="#pone.0001322.s010">Table S9</a>). In addition, all three MYB genes altered transcript level for genes in the biosynthesis of PAPS (3â²-phosphoadenosyl-5â²-phosphosulfate) (sulfate assimilation genes, <a href="#pone.0001322.s009">Table S8</a>) which is the substrate required for the sulfotransferases catalyzing the final step of glucosinolate core synthesis (<a href="#pone-0001322-g001">Figures 1</a> and <a href="#pone-0001322-g004">4</a>). Interestingly, <em>MYB28</em> and <em>MYB29</em> induced the genes required for PAPS production whereas <em>MYB76</em> appeared to repress their transcript levels. As all plants for this experiment were grown side-by-side in a randomized design and all RNA extraction and microarray analysis conducted simultaneously, this difference is likely a valid biological observation rather than due to experimental error.</p>
<div class="figure" id="pone-0001322-g004"><div class="img"><a name="pone-0001322-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Expression of Sulfur Utilization Biosynthetic Pathways in <em>35S:MYB</em> lines.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Nested ANOVAs were utilized on microarray data to test for altered expression of the major sulfur utilization biosynthetic pathways as described. The pathways linking one major metabolite to another with statistically significant altered expression are shown as colored arrows. Red shows that the <em>35S:MYB</em> lines led to increased transcript levels for the biosynthetic pathway in comparison to wild-type, while blue shows decreased transcript levels. Dark color represents a change of 50 percent or more while the lighter color shows a change of less than 50 percent. The genes used for each pathway are as listed in <a href="#pone.0001322.s009">Table S8</a>. MYB28 illustrates the comparison of transcript levels in <em>35S:MYB28</em> lines versus Col-0. MYB29 illustrates the comparison of transcript levels in <em>35S:MYB29</em> lines versus Col-0. MYB76 illustrates the comparison transcript levels in <em>35S:MYB76</em> lines versus Col-0.</p>
<span>doi:10.1371/journal.pone.0001322.g004</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Within the pathway ANOVAs, we next utilized F-tests to test if the transcripts for the individual genes in the biosynthesis of aliphatic glucosinolates were altered in comparison to wild-type Col-0. As expected, the individual <em>35S:MYB</em> lines led to elevated accumulation of the specific MYB gene that was over-expressed (<a href="#pone-0001322-g005">Figure 5</a> and <a href="#pone.0001322.s011">Table S10</a>). However, even though the transcript levels of <em>MYB28</em> was significantly elevated, its modest increase of approximately 40% was in contrast to the more dramatic elevation of approximately 200 and 500% in transcript level of <em>MYB29</em> and <em>MYB76</em> within the <em>35S:MYB29</em> and <em>35S:MYB76</em> lines, respectively. The modest increase in <em>MYB28</em> expression is, however, sufficient to result in an increased accumulation of glucosinolates in the <em>35S:MYB28</em> lines (<a href="#pone-0001322-t001">Tables 1</a> and <a href="#pone-0001322-t003">3</a>). Further, <em>MYB29</em> transcript accumulated in response to over-expressing <em>MYB28</em> and <em>MYB76</em> (<a href="#pone-0001322-g005">Figure 5</a> and <a href="#pone.0001322.s011">Table S10</a>) suggesting the presence of some interplay between the MYB genes.</p>
<div class="figure" id="pone-0001322-g005"><div class="img"><a name="pone-0001322-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Altered Transcript Levels for Genes in the Biosynthetic Pathway of Aliphatic Glucosinolates in the <em>35S:MYB</em> lines.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>A. Altered transcript accumulation for the biosynthetic genes. B. Altered transcript accumulation for the MYB transcription factors by the different 35S transgenes. Nested ANOVAs were utilized on microarray data to test for altered transcript levels for biosynthetic genes in the aliphatic glucosinolate pathway. Each arrow represents a specific biosynthetic process with the transcript alteration for each of the different enzymes indicated as separate rows of boxes. From left to right, the boxes in each row illustrates the comparison of the transcript levels in, respectively, the <em>35S:MYB28</em>, <em>35S:MYB29</em> and <em>35S:MYB76</em> transgenes versus Col-0. The enzymes are listed in <a href="#pone.0001322.s011">Table S10</a>. Genes with a statistically significant altered transcript increase in the given <em>35S:MYB</em> line are shown as red while those with a decrease are in blue. Dark color represents a change of 50 percent or more while the lighter color shows a change of less than 50 percent.</p>
<span>doi:10.1371/journal.pone.0001322.g005</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Over-expression of <em>MYB28</em> and <em>MYB29</em> led to statistically significant increases in transcript levels for, respectively, 17 and 13 of the predicted and characterized genes in aliphatic glucosinolate biosynthesis or regulation (<a href="#pone.0001322.s011">Table S10</a>). In contrast to the overall induction of the aliphatic biosynthetic genes in the <em>35S:MYB</em> lines, a reduction in <em>MAM3</em> transcript levels was observed in <em>35S:MYB28</em> and <em>35S:MYB29</em> lines with the biggest difference occurring in <em>35S:MYB28</em> lines (<a href="#pone-0001322-g005">Figure 5</a> and <a href="#pone.0001322.s011">Table S10</a>). The <em>35S:MYB76</em> lines upregulated relatively fewer transcripts of aliphatic biosynthetic genes as it showed altered transcript levels for only ten of the predicted and characterized genes in aliphatic biosynthesis and regulation (<a href="#pone-0001322-g005">Figure 5</a> and <a href="#pone.0001322.s011">Table S10</a>). The ANOVA results obtained from pathways as well as the individual genes in aliphatic glucosinolate biosynthesis suggest that even though <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> have significant functional overlap, they differ in their regulatory capacities or targets.</p>


<h4>Genome-wide transcript effects of <em>35S:MYB</em> over-expression lines</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To better assess the overlap of transcripts altered in <em>35S:MYB</em> lines, we conducted separate ANOVAs to test every transcript for a significant difference between wild-type and each selected <em>35S:MYB</em> line. Using a false discovery rate (FDR) of 0.05, data indicated that the <em>35S:MYB28</em> lines altered the accumulation of 1097 transcripts, the <em>35S:MYB29</em> lines 522 transcripts and the <em>35S:MYB76</em> lines 1087 transcripts (<a href="#pone-0001322-g006">Figure 6</a> and <a href="#pone.0001322.s012">Table S11</a>). The effects were nearly equally divided between transcripts induced and those repressed (<a href="#pone.0001322.s012">Table S11</a>). In agreement with the hypothesis that all three <em>MYB</em> genes share regulatory targets such as aliphatic glucosinolates, there was a significant bias for overlap in transcripts regulated by all three <em>MYB</em> genes (<em>P</em>&lt;0.001 by Chi-square, <a href="#pone-0001322-g006">Figure 6</a>). This overlap included approximately 53% of all transcripts regulated by <em>MYB29</em>. In contrast to the overlap, there are a number of genes altered by specific subsets of the MYBs further suggesting that they are not operating via a single mechanism (<a href="#pone-0001322-g006">Figure 6</a>). Decreasing the FDR rate to 0.10 did not alter the ratio of transcripts populating the regions of the Venn diagram suggesting that the indication of specificity amongst the factors is not merely a statistical artifact of the microarray analysis (data not shown). An analysis of GO annotations in all quadrants of the Venn diagram did not show any informative bias with regards to function of the genes whose transcripts were affected (data not shown).</p>
<div class="figure" id="pone-0001322-g006"><div class="img"><a name="pone-0001322-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Overlap in Altered Gene Regulation Between the <em>35S:MYB</em> over-expressor lines.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Each ring of the Venn diagram shows the number of genes whose transcript level was statistically significantly altered by the given <em>35S:MYB</em> transgene. Statistical significance was determined by individual gene ANOVAs using a FDR of 0.05. The bottom diagram shows the predicted number of genes in each intersection under the assumption that the MYB genes have independent regulatory functions.</p>
<span>doi:10.1371/journal.pone.0001322.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Previous work has suggested a link between aliphatic glucosinolates and the accumulation of sinapate esters whereby the glucosinolate biosynthetic mutants altered sinapate accumulation and vice-versa <a href="#pone.0001322-Hemm1">[38]</a>, <a href="#pone.0001322-Kliebenstein7">[39]</a>. Interestingly, all three <em>35S:MYB</em> genotypes led to decreased transcript levels of <em>MYB4âAt4g38620</em>, a MYB transcription factor which suppresses the accumulation of sinapate esters <a href="#pone.0001322-Jin1">[40]</a>. Accordingly, transcript levels of <em>SNG1</em>-<em>At2g22990</em>, the gene responsible for conversion of sinapoyl glucose to sinapoyl malate <a href="#pone.0001322-Lehfeldt1">[41]</a> (<a href="#pone.0001322.s012">Table S11</a>) are increased in all three genotypes. Curiously, however, <em>BRT1âAt3g21560</em>, responsible for the conversion of sinapate to sinapoyl glucose <a href="#pone.0001322-Sinlapadech1">[42]</a>, on the other hand, are down-regulated in all three. This suggests that the MYBs may be involved in the suggested cross-talk between sinapate and aliphatic glucosinolate metabolism.</p>


<h4><em>myb28-1</em>, <em>myb29-1</em> and <em>-2</em> and <em>myb76-1</em> and <em>-2</em> T-DNA mutants display reduction in different aliphatic glucosinolates</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The 35S over-expression analysis provides a guide to a gene's potential activity but has the capacity to generate false-positive impressions. To validate that the MYB candidate genes play a role in biosynthesis of aliphatic glucosinolates <em>in planta</em>, loss-of-function alleles in <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> were obtained <a href="#pone.0001322-Alonso1">[43]</a>â<a href="#pone.0001322-Tissier1">[45]</a> and the borders of the T-DNA insertions sequenced to validate the insertion sites (<a href="#pone-0001322-g007">Figure 7A</a>). The transcript levels for the MYB genes were measured in all five lines, <em>myb28-1</em>, <em>myb29-1</em>, <em>myb29-2, myb76-1</em> and <em>myb76-2</em>, to determine whether the T-DNA insertions resulted in a loss of transcript. RT-PCR was conducted on RNA purified from at least two independent wild-type plants and homozygous single-mutant plants and with at least two different cycle numbers to better quantify changes. The analysis revealed that <em>myb28-1, myb29-1</em> and <em>myb29-2</em> were indeed knockout mutants whereas <em>myb76-1</em> and <em>myb76-2</em> still had residual but much reduced <em>MYB76</em> transcript levels (<a href="#pone-0001322-g007">Figure 7B</a>). Given that the T-DNA insert is in an exon, it should be noted that <em>myb76-1</em> may be a functional knockout. The knockout or knockdown of one MYB transcription factor did not lead to changes in levels of any of the other MYB transcription factors. No apparent visual phenotype was observed in any of the single knockout mutants under the conditions tested. The impact on the transcript level for the individual biosynthetic genes in the knockouts was minimal with only a slight reduction in <em>BCAT4, MAM1</em> and <em>CYP79F1</em> transcripts in <em>myb29-1</em> and <em>myb29-2</em> and in <em>MAM1</em>, <em>CYP79F2</em> and <em>CYP83A1</em> transcripts in <em>myb28-1</em> (data not shown). It is possible that levels of aliphatic glucosinolates are sensitive to changes in aliphatic glucosinolate transcript accumulation that are not easily detectable using quantitative RT-PCR. This could be enhanced as the level of a transcripts is a single time-point measure while the level of aliphatic glucosinolates likely integrates over a length of time, thereby amplifying any change.</p>
<div class="figure" id="pone-0001322-g007"><div class="img"><a name="pone-0001322-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Characterization by RT-PCR of transcript levels in <em>myb28-1</em>, <em>myb29-1</em>, <em>myb29-2</em>, <em>myb76-1</em>, <em>myb76-2</em> and <em>myb28-1 myb29-1</em> mutants.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>A. Diagram of the <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> genes with exons given as black boxes and 5â²UTR, 3â²UTR and introns given as black lines. The T-DNA insertion site in <em>myb28-1</em> is located in the 5â²UTR, the T-DNA insertion sites in <em>myb29-1</em> and <em>myb29-2</em> are located in the third exon and 5â²UTR, respectively, and the T-DNA insertions of <em>myb76-1</em> and <em>myb76-2</em> are located in the first exon and first intron, respectively. Arrows marked F and R show the approximate positions of the primers used for RT-PCR. B. Steady state foliar mRNA transcript levels of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> and various aliphatic biosynthetic genes in wild-type Col-0, <em>myb28-1</em>, <em>myb29-1, myb29-2</em>, <em>myb76-1</em> and <em>myb76-2</em> and <em>myb28-1 myb29-1</em> mutants as measured by RT-PCR in 23-25 days old plants. Each mutant is displayed with its corresponding wild-type. A PCR for actin was used as a loading control. Amplification was shown to be in the logarithmic phase. C. Steady state foliar mRNA transcript levels of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> and various aliphatic biosynthetic genes in wild-type Col-0 and the <em>myb28-1 myb29-1</em> mutant as measured by RT-PCR in 25 days old plants. A PCR for actin was used as a loading control. Amplification was shown to be in the logarithmic phase.</p>
<span>doi:10.1371/journal.pone.0001322.g007</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>To minimize maternal effects and to show that any observed chemotype segregated with the T-DNA insertion, we genotyped germinating offspring from segregating heterozygous mutants and measured glucosinolates on the homozygous knockout and wild-type sibling progeny for each mutant. The combined mean from <em>myb29-1</em> and <em>myb29-2</em> and that of <em>myb76-1</em> and <em>myb76-2</em> are presented since ANOVA showed that there was no difference in the relative levels of glucosinolates between the different mutant alleles (<a href="#pone.0001322.s013">Table S12</a>). Leaves from the <em>myb29-1</em>, <em>myb29-2</em>, <em>myb76-1</em> and <em>myb76-2</em> mutants had significantly reduced levels of short-chained aliphatic glucosinolates content with no change in the amounts of the long-chained aliphatic glucosinolates (<a href="#pone-0001322-t004">Table 4</a>). In contrast, the <em>myb28-1</em> mutant showed a dramatic reduction in long-chained and a decrease in short-chained aliphatic glucosinolates (<a href="#pone-0001322-t004">Table 4</a>). These results indicate that <em>MYB29</em> and <em>MYB76</em> play a role for regulation of short-chained aliphatic glucosinolates, whereas <em>MYB28</em> plays a role in the control of both short- and long-chained aliphatic glucosinolates in leaves. The mutations in <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> did not affect indole glucosinolate levels (<a href="#pone-0001322-t004">Table 4</a>).</p>
<div class="figure" id="pone-0001322-t004"><div class="img"><a name="pone-0001322-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Foliar glucosinolate content in T-DNA insertion mutants of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001322.t004</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The impact of the <em>myb28-1</em>, <em>myb29-1</em>, <em>myb29-2</em>, <em>myb76-1</em> and <em>myb76-2</em> insertions on the glucosinolate pool in seeds was measured on seeds derived from the plants on which glucosinolates in leaves had been measured. This showed that <em>myb76</em> only displayed a small decrease in 4MSB and 7MSH, indicating that this gene only plays a minor role in aliphatic glucosinolate regulation in seeds. However, both the levels of short and long-chained aliphatic glucosinolates were significantly reduced in homozygous <em>myb28-1</em> seeds (<a href="#pone-0001322-t005">Table 5</a>) leading to a substantial reduction in total amounts of aliphatic glucosinolates (<a href="#pone-0001322-t005">Table 5</a>). For <em>myb29</em>, a significant reduction was observed in the levels of the short-chained 4MTB, with no impact on long-chained aliphatic glucosinolates (<a href="#pone-0001322-t005">Table 5</a>). Curiously, 4BZOP levels were increased in both genotypes. The decrease of aliphatic glucosinolates in seeds further supports the observation that both <em>MYB28</em> and <em>MYB29</em> play a role in controlling the accumulation of aliphatic glucosinolates in leaves and seeds, but via different chain-length specificities. The level of indole glucosinolates was not affected in either line.</p>
<div class="figure" id="pone-0001322-t005"><div class="img"><a name="pone-0001322-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Seed glucosinolate content in T-DNA insertion mutants of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001322.t005</span></div>

<h4>A <em>myb28-1 myb29-1</em> double knockout mutant displays no detectable aliphatic glucosinolates and reduced transcript level of biosynthetic enzymes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To further investigate the roles of <em>MYB28</em> and <em>MYB29</em> on production of aliphatic glucosinolates, we crossed <em>myb28-1</em> and <em>myb29-1</em> to obtain a double knockout mutant. Homozygous double knockouts were obtained. Analysis of transcript accumulation within the homozygous double knockout in comparison to the WT Col-0 showed that transcripts of <em>BCAT4</em>, <em>MAM1</em>, <em>CYP79F1</em> and <em>CYP79F2</em> were undetectable in the double knockout leaves under the cycle numbers tested. Furthermore, a substantial reduction was observed in <em>CYP83A1</em> and <em>C-S-LYASE</em> transcripts (<a href="#pone-0001322-g007">Figure 7C</a>). Finally, the absence of <em>MYB28</em> and <em>MYB29</em> transcripts resulted in a small decrease in the transcript level of <em>MYB76</em> (<a href="#pone-0001322-g007">Figure 7C</a>). We were unable to detect aliphatic glucosinolates in either the leaves or seeds of the homozygous double <em>myb28-1 myb29-1</em> mutant (<a href="#pone-0001322-g008">Figure 8</a>). In comparison to the WT and homozygous single knockouts, this loss of glucosinolates showed a statistically significant epistatic interaction between MYB28 and MYB29. A merely additive interaction would have led to foliar total aliphatic glucosinolates having a level of 25% of wildtype in the double mutants (<a href="#pone-0001322-g008">Figure 8</a> and <a href="#pone.0001322.s015">Tables S14</a>, <a href="#pone.0001322.s016">S15</a>, <a href="#pone.0001322.s017">S16</a> and <a href="#pone.0001322.s018">S17</a>). This data confirm that in addition to having specific activities, <em>MYB28</em> and <em>MYB29</em> also have synergistic functionalities. Indole glucosinolate levels were not affected in the <em>myb28-1 myb29-1</em> double knockout mutant in comparison to the WT or homozygous single knockout lines (<a href="#pone.0001322.s015">Table S14</a>, <a href="#pone.0001322.s016">S15</a>, <a href="#pone.0001322.s017">S16</a> and <a href="#pone.0001322.s018">S17</a>). The loss of aliphatic glucosinolates in the double knockout plants could not have been predicted by the chemotype of the single knockout mutants and as such reveals an emergent property of glucosinolate regulation. Additionally, this double knockout phenotype suggests that MYB76 requires a functional MYB28 or MYB29 to control aliphatic glucosinolates.</p>
<div class="figure" id="pone-0001322-g008"><div class="img"><a name="pone-0001322-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001322" data-uri="info:doi/10.1371/journal.pone.0001322.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001322.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001322.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001322.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001322.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>The effects of <em>myb28-1 myb29-1</em> Double Mutant on Glucosinolate Accumulation.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Homozygous wild-type, homozygous single mutant or homozygous double mutant progeny were measured for foliar and seed glucosinolates by HPLC. 12 independent plants were separately measured per line for the four lines and the data analyzed via ANOVA (<a href="#pone.0001322.s015">Table S14</a>, <a href="#pone.0001322.s016">S15</a>, <a href="#pone.0001322.s017">S16</a> and <a href="#pone.0001322.s018">S17</a>). Data for 4MSB, 8MSO, total aliphatic and total indolic glucosinolate content are shown. Genotypes with different letters show statistically different glucosinolate levels for the given glucosinolate.</p>
<span>doi:10.1371/journal.pone.0001322.g008</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Identification of aliphatic glucosinolate regulators</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We have utilized diverse systems biology tools to identify <em>MYB28, MYB29</em> and <em>MYB76</em> as candidate regulators of aliphatic glucosinolates. This involved a combination of natural variation in both metabolite and transcript networks, phylogenetic analysis and bioinformatics of gene expression databases. When the three MYB genes were individually over-expressed in a wildtype Col-0 background, all lines accumulated more aliphatic glucosinolates in leaves and seeds than the corresponding wildtype (<a href="#pone-0001322-t001">Tables 1</a>â<a href="#pone-0001322-t002"></a><a href="#pone-0001322-t003">3</a>). Microarray analysis showed that transcript levels for genes involved in biosynthesis of aliphatic glucosinolates in foliar tissues were concurrently up-regulated (<a href="#pone-0001322-g005">Figure 5</a>). This showed that all three MYB genes have the potential to up-regulate the accumulation of aliphatic glucosinolates via increasing the biosynthetic transcripts. Analysis of knockout mutants of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> further established a role of the three MYB genes in regulation of aliphatic glucosinolates in Col-0 since absent expression of the genes led to reduced contents of aliphatic glucosinolates, as evidenced by altered profiles in both leaves and seeds (<a href="#pone-0001322-t004">Table 4</a>â<a href="#pone-0001322-t005">5</a>). This is in contrast to the hypothesis proposed by Hirai <em>et al.</em> (2007) that <em>MYB29</em> has no major role in regulation of aliphatic glucosinolates, except in response to jasmonic acid.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The identification of these three MYBs within a single clade and their overlapping phenotypes in the <em>35S:MYB</em> over-expressor lines suggested that they may be a redundant gene family. However, analysis of the knockout lines at the metabolic level showed that these functions were not redundant as <em>MYB29</em> and <em>MYB76</em> controlled short-chained aliphatic glucosinolates and <em>MYB28</em> controlled both short- and long-chained aliphatic glucosinolates. The impact of <em>MYB28</em> on both chain lengths is likely mediated through an impact on <em>CYP79F</em> expression as the <em>MYB28</em>/<em>MYB29</em> double knockout does not impact MAM3 expression. Similar results were obtained with the <em>myb28-1</em> knockout by Hirai <em>et al.</em> (2007). The individual knockout mutants had only minimal effects on transcripts for the individual genes involved in biosynthesis of aliphatic glucosinolates (data not shown). In contrast, the <em>myb28-1 myb29-1</em> double knockout mutant dramatically diminished most transcripts for the biosynthetic genes and abolished the contents of all aliphatic glucosinolates (<a href="#pone-0001322-g007">Figure 7C</a> and <a href="#pone-0001322-g008">Figure 8</a>). This shows that this family of MYBs functions to regulate the aliphatic glucosinolate biosynthetic pathway in Col-0 and that they have evolved specific and overlapping functions that show complex interconnectivity.</p>


<h4>The <em>myb28-1 myb29-1</em> double knockout mutant reveals an emergent interaction of <em>MYB28</em> and <em>MYB29</em> on aliphatic glucosinolates in Col-0</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The glucosinolate profiles of the single knockout mutants suggest that <em>MYB28</em> and <em>MYB29</em> play significant, but distinct roles in regulation of the biosynthetic genes for aliphatic glucosinolates as both lead to lower levels of specific aliphatic glucosinolates (<a href="#pone-0001322-t004">Table 4</a> and <a href="#pone-0001322-t005">5</a>). However, transcript levels were only minimally affected by mutations in the individual genes (data not shown). A <em>myb28-1 myb29-1</em> double knockout mutant showed that both genes apparently positively interact to control both transcript levels and metabolite accumulation for the majority of the pathway. The total level of aliphatic glucosinolates of the double knockout mutant were dramatically lower than either single knockout mutant in the leaves, and below the level of detection for all aliphatic glucosinolates in both leaves and seeds (<a href="#pone-0001322-g008">Figure 8</a> and <a href="#pone.0001322.s015">Table S14</a>, <a href="#pone.0001322.s016">S15</a>, <a href="#pone.0001322.s017">S16</a> and <a href="#pone.0001322.s018">S17</a>). In concordance, the transcripts of most characterized aliphatic biosynthetic genes were undetectable in the leaves of the double knockout mutant (<a href="#pone-0001322-g007">Figure 7C</a>). None of the phenotypes of the single mutants hinted at the striking phenotype of the double knockout mutant and, as such, the analysis of the latter identifies an emergent property of the glucosinolate regulation system not readily predictable from the phenotypes of the single knockout mutants.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The double knockout analysis points to an interplay between <em>MYB28</em> and <em>MYB29</em> whereby they interact to activate the aliphatic glucosinolate pathway. One source of possible <em>MYB</em> interplay was observed in the <em>35S:MYB</em> lines, where over-expression of <em>MYB28</em> and <em>MYB76</em> led to increased levels of <em>MYB29</em> transcript. This suggests a different role for <em>MYB29,</em> in which it integrates signals from <em>MYB28</em> and <em>MYB76</em> in regulating the aliphatic glucosinolates. However, this is not a strictly linear pathway where <em>MYB28</em> and <em>MYB76</em> would regulate <em>MYB29</em> to regulate the glucosinolates since <em>MYB29</em> transcript seems unchanged in the <em>myb28-1</em>, <em>myb76-1</em> and <em>myb76-2</em> mutants (<a href="#pone-0001322-g007">Figure 7B</a>). The observation that the <em>myb28-1 myb29-1</em> double knockout mutant altered transcripts more dramatically than either individual knockout suggests that <em>MYB28</em> has regulatory functionalities independent of <em>MYB29</em>. Future work is required to understand how the three <em>MYB</em> genes co-operatively stimulate the aliphatic glucosinolates.</p>


<h4>QTL co-localization</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>MYB28 co-localizes with one of the three major QTLs for aliphatic glucosinolate content and gene expression within the <em>Arabidopsis</em> BayÃSha population (<a href="#pone-0001322-g002">Figure 2</a> and <a href="#pone.0001322-Wentzell1">[46]</a>). Three lines of evidence strongly argue for <em>MYB28</em> being the basis for this QTL. First, <em>MYB28</em> has differential expression between Bay and Sha (<a href="#pone.0001322.s002">Table S1</a>). Second, altering <em>MYB28's</em> expression causes changes in aliphatic glucosinolate content and transcripts (<a href="#pone-0001322-t001">Table 1</a>, <a href="#pone-0001322-t004">4</a> and <a href="#pone-0001322-t005">5</a>, <a href="#pone-0001322-g005">Figure 5</a>) . Third, <em>MYB28</em> is epistatic to both the <em>MAM</em> and <em>AOP</em> QTL <a href="#pone.0001322-Wentzell1">[46]</a> and has the ability to regulate expression of these genes. An additional QTL in the L<em>er</em>ÃCvi population is likely caused by natural variation in <em>MYB76</em> expression <a href="#pone.0001322-Kliebenstein4">[17]</a>. Again, the gene has differential expression between L<em>er</em> and Cvi (<a href="http://www.weigelworld.org/resources/microarray/AtGenExpress/">http://www.weigelworld.org/resources/micâroarray/AtGenExpress/</a>), this differential expression can lead to altered glucosinolate content and the QTL is epistatic to <em>MAM</em> and <em>AOP</em> whose genes are regulated by <em>MYB76.</em> Perlegen sequence data shows that there are polymorphisms between Bay/Sha for MYB28 and Ler/Cvi for MYB76 (Clark et al. 2007). However, none of these polymorphisms cause major effect polymorphisms. Additional crosses and verification are required to conclusively prove that MYB28 and MYB76 are indeed the genes underlying the total aliphatic QTLs in the two populations and to investigate what polymorphisms might be the underlying cause. Interestingly, the <em>AOP2</em> gene within the <em>AOP</em> QTL has recently been shown to lead to altered expression of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em>, thus, establishing a molecular basis for their epistatic interactions. This sets up a highly interconnected regulatory network whereby variation in gene expression for the transcript regulators <em>MYB28</em>, <em>MYB76</em>, <em>MYB29</em> and <em>AOP2</em> can interact to generate a wide range of aliphatic glucosinolate content <a href="#pone.0001322-Kliebenstein3">[16]</a>, <a href="#pone.0001322-Wentzell1">[46]</a>. As Col-0 does not have functional <em>AOP2</em>, it remains to be seen how this feed-forward loop actually functions within <em>Arabidopsis</em> <a href="#pone.0001322-Kliebenstein8">[47]</a>. The identification of these MYBs as the likely basis for two additional QTLs within <em>Arabidopsis</em> glucosinolate accumulation raises the possibility of using a similar systems biology approach to identify the complete molecular basis for quantitative variation in aliphatic glucosinolates.</p>


<h4>Regulation of substrate availability for aliphatic glucosinolates</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>An important factor in production of a given compound is the availability of precursor substrates. The similar level of total aliphatic glucosinolates (although with a different composition) in knockout mutants of either <em>MAM1</em> and <em>MAM3</em> <a href="#pone.0001322-Textor1">[48]</a> suggests that under normal conditions in Col-0, a predetermined amount of substrate is destined to go into the glucosinolate pathway. Over-expression of the <em>MYB</em> regulators allows more substrate to enter the glucosinolate pathway as evidenced by the observed increase of up to 110% in total aliphatic glucosinolate content (<a href="#pone-0001322-t001">Table 1</a> and <a href="#pone-0001322-t002">2</a>). This is reflected in the altered levels of transcripts for both the biosynthetic as well as for the substrate pathways (<a href="#pone-0001322-g004">Figure 4</a>), although we cannot conclude if the latter is a direct or indirect effect of the <em>MYB</em> over-expression. Compared to the variation in aliphatic glucosinolate content among the <em>Arabidopsis</em> accessions <a href="#pone.0001322-Kliebenstein9">[49]</a>, the increase in aliphatic glucosinolate content modulated by the individual <em>MYB</em> genes can be regarded as rather modest. This suggests a putative restraint, when modifying a single gene, possibly due to a limitation in the substrate availability or in other components of the regulatory machinery. Interestingly, Gigolashvili <em>et al.</em> (2007) describe a line with a seven fold increase in 4MSB when over-expressing <em>MYB28</em>. However, this line shows a strong phenotype which could be due to a strong pull on the methionine pool. This suggests that when the production of aliphatic glucosinolates reaches a certain level due to, for instance, the manipulation of a single regulatory gene, plant growth is hampered by e.g. a shortage of methionine for protein biosynthesis.The alteration of expression levels of multiple genes within the natural accessions may allow for this bottleneck to be bypassed.</p>


<h4><em>MAM3</em> might be differentially regulated from the other glucosinolate genes</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The <em>MAM1</em> and <em>MAM3</em> genes have apparently arisen via a gene duplication and divergence in the past <a href="#pone.0001322-Textor2">[50]</a>, <a href="#pone.0001322-Field1">[51]</a>. Our data indicates that during this process, these two genes have obtained different regulatory patterns with <em>MAM1</em> acting similar to the other glucosinolate genes and <em>MAM3</em> being differently regulated. All three <em>35S:MYB</em> transgenes lead to an increase in <em>MAM1</em> and decrease in <em>MAM3</em> expression (<a href="#pone.0001322.s011">Table S10</a>) which suggests a reciprocal relationship. This is further substantiated by the observation that abolishment of one lead to upregulation of the other <a href="#pone.0001322-Textor1">[48]</a>. The <em>myb28-1 myb29-1</em> double knockout abolished most aliphatic glucosinolate related transcripts including <em>MAM1</em> but left <em>MAM3</em> unaffected or even slightly increased. The loss of long-chain aliphatic glucosinolates in this background is likely dependent upon the complete loss of expression for CYP79F1 and F2, removing the ability to convert the elongated homomethionines into glucosinolates. This suggests that the two genes have different regulatory networks rather than a relationship whereby the repression of one activates the other. Validating this hypothesis will require more analysis.</p>


<h4>Future Avenues</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The rapidly increasing number of identified glucosinolate regulators, provides important tools to understand how the plant controls the production of these critical herbivore and pathogen defense compounds. Transcriptional analysis of double and triple knockout lines of the MYB genes may identify how they interact to control glucosinolate production as well as the metabolic networks required to provide substrates for the production.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>The multiple duplications and divergences leading to the indole and aliphatic glucosinolate MYBs as well as in the genes which they regulate (e.g. <em>CYP79F1</em>/<em>F2</em> and <em>CYP79B2/B3</em>) provides means to address evolutionary questions, as these genes have evolved specific and overlapping functions that show complex interconnectivity over time. The glucosinolate biosynthetic genes and their regulators may be an excellent system for studies on how transcription factors have evolved simultaneously with their target genes to give rise to new biosynthetic pathways, such as the aliphatic and indole glucosinolates ones, with different regulation. Identification and validation of the regulatory role of <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> as having the same general regulatory targets will greatly facilitate such evolutionary studies in the future.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>QTL Mapping</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The Bay-0ÃSha population of 403 <em>A. thaliana</em> recombinant inbred lines <a href="#pone.0001322-Loudet1">[52]</a> was used to map QTL controlling total aliphatic glucosinolate content. Seeds were imbibed and cold stratified at 4Â°C for three days to break dormancy. A single plant per line was grown in flats containing 96 cells per flat, and maintained under short day conditions in controlled environment growth chambers. Four flats contained a single replicate of each of the 403 lines. Two independent randomized replicate blocks were grown. At 35 days post germination, a fully-expanded mature leaf was harvested, digitally photographed and analyzed for total aliphatic glucosinolate content via HPLC. The Bay-0ÃSha RIL population has previously been genotyped <a href="#pone.0001322-West2">[25]</a>, <a href="#pone.0001322-Loudet1">[52]</a>. The average total aliphatic glucosinolate content for the RILs was used for QTL mapping within Windows QTL Cartographer v2.5 <a href="#pone.0001322-Wang1">[53]</a>â<a href="#pone.0001322-Zeng1">[55]</a>. Composite interval mapping (CIM) was implemented using Zmap (Model 6) with a 10 cM window and an interval mapping increment of 2 cM. Forward regression was used to identify five cofactors per quantitative trait. The declaration of statistically significant QTL is based on permutation derived empirical thresholds using 1,000 permutations for each trait mapped <a href="#pone.0001322-Churchill1">[56]</a>, <a href="#pone.0001322-Doerge2">[57]</a>. Other QTL identified within this analysis will be presented elsewhere.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Further, 211 of these lines have been analyzed for variation in gene expression <a href="#pone.0001322-West1">[22]</a>, which provided data that has been used to map eQTL controlling transcript levels for glucosinolate biosynthetic genes <a href="#pone.0001322-Kliebenstein4">[17]</a>. This allows comparison of QTL controlling metabolite accumulation with eQTL for transcript levels. The eQTLs for glucosinolate biosynthetic transcripts identified within the previous analysis were queried for the presence of an eQTL at the same position as found for the total aliphatic glucosinolate content QTL (<a href="#pone.0001322.s003">Table S2</a>).</p>


<h4>Plant cultivation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p><em>Arabidopsis thaliana</em> ecotype <em>Colombia</em>, transgenics and mutants were grown in a growth chamber (HEMZ 20/240/S, Heraeus) at a photosynthetic flux of 100 ÂµEi at 20Â°C and 70% relative humidity at a 16 h photoperiod in a soil:vermiculite (10:1) mixture.</p>


<h4>Sequences</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Different annotations of the size of mRNA and coding sequences of <em>MYB28</em> and <em>MYB29</em> are found in the databases. The <em>MYB28</em> mRNA sequence is found in two different versions in the NCBI database (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgidbNucleotide">http://www.ncbi.nlm.nih.gov/entrez/queryâ.fcgidbNucleotide</a>). One (NM_125535.) encodes a transcript of 1425 bp long and encodes for a protein of 367 amino acids. Another mRNA (NM_180910) is 1805 bp long and is predicted to encode a 288 amino acid protein. Similarly, two different <em>MYB29</em> mRNA sequences existâone of 1595 bp (NM_120851) and one of 1292 bp (AF062872)âboth are predicted to encode a 337 aa protein. The <em>MYB76</em> mRNA (NM_120852 , DQ446930 , AF175992 ) of 1017 bp is predicted to encode a protein of 339 aa. The two different coding sequences of <em>MYB28</em> were aligned with the annotated coding sequences of <em>MYB29</em> and <em>MYB76</em>. It was found that the 288 amino acid protein lacked the crucial R2 and most of the R3 DNA binding domain. Consequently, the 367 amino acid encoding region was regarded as the correct coding sequence and subsequently used in the clonings.</p>


<h4>Creation of transgenic plants</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The over-expression constructs driven by the CaMV 35S promoter were created by cloning, respectively, <em>MYB28</em>, <em>MYB29</em> and <em>MYB76</em> coding regions from <em>Arabidopsis thaliana</em> ecotype <em>Columbia</em> cDNA into the pCAMBIA230035Su using the USERâ¢ method as described <a href="#pone.0001322-NourEldin1">[58]</a>. Primer sequences used for the PCRs are summarized in <a href="#pone.0001322.s019">Table S18</a>. Binary plasmids were transferred to <em>Agrobacterium tumefaciens</em> strain C58 <a href="#pone.0001322-Zambryski1">[59]</a> and transformed into <em>Arabidopsis</em> plants according to the floral dip method <a href="#pone.0001322-Clough1">[60]</a>. Transgenic plants were selected on Â½ MS with 50 mg L<sup>â1</sup> kanamycin.</p>


<h4>DNA extraction and genotyping of T-DNA insertion mutants</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Total DNA was extracted essentially as described in <a href="#pone.0001322-Lukowitz1">[61]</a>. T-DNA insertions in <em>At5g61420</em> (line SALK_136312 = <em>myb28-1</em>), <em>At5g07690</em> (lines GABI_868E02 = <em>myb29-1</em> and SM.34316 = <em>myb29-2</em>) and <em>At5g07700</em> (lines SALK_096949 = <em>myb76-1</em> and SALK_055242 = <em>myb76-2</em>) were confirmed by PCR. <em>myb28-1</em> had a T-DNA insertion in the 5â²UTR region of the gene, 182 bp upstream of the start codon. The T-DNAs of <em>myb29-1</em>and <em>myb29-2</em> are positioned in the third exon 730 bp upstream of the stop codon and in the 5â²UTR 40 bp upstream the start codon, respectively. The T-DNAs of <em>myb76-1</em> and <em>myb76-2</em> are situated respectively, 99 bp downstream the ATG in the first exon and 194 downstream the ATG in the first intron. Two separate PCR reactions were carried out to identify the position of the insertion site and the zygosity of the plants. Forward and reverse primers were designed according to the SIGnAL T-DNA verification primer design tool (<a href="http://signal.salk.edu/tdnaprimers.2.html">http://signal.salk.edu/tdnaprimers.2.htmâl</a>) for the SALK lines and with Primer3 (<a href="http://frodo.wi.mit.edu/cgi-bin/primer3/primer3_www.cgi">http://frodo.wi.mit.edu/cgi-bin/primer3/âprimer3_www.cgi</a>) for the GABI line. The gene specific primers were used in combination with left border primers (LBa1 for SALK lines or 8409 LB for GABI lines and Spm32 for the SM-line) to verify the presence and orientation of the T-DNAs. Primer sequences used for genotyping are summarized in <a href="#pone.0001322.s019">Table S18</a>. Eppendorf HotMaster Taq DNA Polymerase (Hotmaster) (Eppendorf, AG, Hamburg, Germany) was used in a 20 Âµl reaction using 1 unit enzyme, 187.5 ÂµM dNTP, buffer 1:10 and 187.5 ÂµM of each primer and DNA as template. The PCR program was as follows: Denaturation at 94Â°C for 3 min, 35 cycles of denaturation at 94Â°C for 30 s, annealing at 56Â°C for 30 s, extension at 65Â°C for 1.15 min and finally extension at 65Â°C for 3 min.</p>


<h4><em>myb28-1 myb29-1</em> double mutant construction</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To construct the double mutant, <em>myb28-1 myb29-1</em>, the homozygous <em>myb28-1</em> and <em>myb29-1</em> were crossed with each other. The F<sub>1</sub> plant was self-fertilized and progeny in the F<sub>2</sub> generation was genotyped by PCR (see above).</p>


<h4>RT-PCR on knockouts and wild-type</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Leaves from homozygous <em>myb28-1</em> single knockout, homozygous <em>myb28-1 myb29-1</em> double knockout and <em>Arabidopsis</em> Columbia wild-type plants (all derived from a segregating <em>myb28-1 myb29-1</em> F2 plants) were harvested 25 days after germination. Leaves from plants homozygous for the absence or presence of the <em>myb29-1</em>, <em>myb29-2</em>, <em>myb76-1</em> and <em>myb76-2</em> allele were harvested 23 days after germination. RNA was extracted with Trizol reagent (Invitrogen, Carlsbad, CA) according to the manufacturer's instructions. The samples were DNAse treated with 2 units DNA freeâ¢ (Ambion, Cambridgeshire, Great Britain) according to the manufacturer's instructions. One Âµg of total RNA was reverse transcribed using the iScript cDNA Synthesis Kit (Biorad, Hercules, CA). The primers used for RT-PCR are listed in <a href="#pone.0001322.s019">Table S18</a>. PCR was performed with Eppendorf HotMaster Taq DNA Polymerase (Hotmaster) (Eppendorf, AG, Hamburg, Germany) in a 20 Âµl reaction using 1 unit enzyme, 187.5 ÂµM dNTP, buffer 1:10 and 187.5 ÂµM of each primer and cDNA as template. The PCR program was as follows: Denaturation at 94Â°C for 3 min, 22â35 cycles of denaturation at 94Â°C for 30 s, annealing at 53â56Â°C for 30 s, extension at 65Â°C for 0.45â1.15 min and finally extension at 65Â°C for 3 min.</p>


<h4>Generation of plant material for glucosinolate analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>T2 progeny of <em>35S:MYB28</em>, <em>35S:MYB29</em> and <em>35S:MYB76</em> lines was germinated on Â½ MS with 50 mg/L<sup>â1</sup> kanamycin. Resistant plants were used for the analysis along with <em>Arabidopsis thaliana</em> ecotype <em>Colombia</em> wild-type plants germinated on Â½ MS. Plants were transferred to soil 10â12 days after germination. For the microarray experiment, transgenic plants were identified by PCR with primers for the <em>nptII</em> gene (<a href="#pone.0001322.s019">Table S18</a>). In the knockout experiments, seeds from a segregating heterozygous knockout mutants or heterozygous/heterozygous double knockout were germinated on soil and transferred to flats containing 48 or 96 cells per flat 10â12 days after germination. Leaves were harvested for glucosinolates analysis just before bolting 22â25 days after germination.</p>


<h4>Glucosinolate extraction and analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Glucosinolate extraction was performed as previously described <a href="#pone.0001322-Kliebenstein9">[49]</a>. For analysis of leaves, 30â100 mg leaves were used. For analysis of seeds, 5â10 mg or 10 seeds were used for the extraction. HPLC analysis was performed as previously described <a href="#pone.0001322-Hansen1">[33]</a>.</p>


<h4>LC-MS analysis</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>20 Âµl sample was injected by ASI-100 Automated Sample injector (Dionex, Denmark) and separated on a Zorbax SB-AQ RPC18 column (4.6 mmÃ250 mm, 5 um) (Agilent Technologies, USA) at a flow rate of 1 ml/min delivered by a P680 HPLC pump (Dionex). The program was as described previously for HPLC <a href="#pone.0001322-Hansen1">[33]</a>. A STH585 column thermostate (Dionex) kept the column temperature at the set 25Â°C. The mobile phase was split using a T-piece and delivered 20% of the total flow (1 ml/min) to the mass spectrometer and 80% to an UV-detector equipped with a micro flow cell (UVD340S, Dionex) and the desulfoglucosinolates were detected at 229 nm. Mass spectrometry was carried out on a single quadrupole Thermo Finnigan Surveyor MSQ equipped with electrospray injection. The electrospray capillary voltage was set at 3 kV, the cone voltage at a constant 75 V and the temperature was 365Â°C. For ionization 50 Âµl/min of 250 ÂµM NaCl was added to the flow (after split) using an AXP-MS high pressure pump (Dionex) and the desulfoglucosinolates were detected as [M+Na]<sup>+</sup> adduct ions. Desulfoglucosinolates were identified according to <em>m/z</em> values and retention times and quantified by the A<sub>229 nm</sub> and published response factors <a href="#pone.0001322-Brown1">[37]</a>.</p>


<h4>Glucosinolate statistical analysis</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>The glucosinolate content was analyzed via ANOVA utilizing SAS proc glm. Each <em>35S:MYB</em> line was tested for altered glucosinolate content in an individual ANOVA against the wild-type Col-0 genotype. For each <em>35S:MYB</em> transgene, two independent transgenic lines were obtained and both transgenic lines grown in two independent experiments. Each transgenic line and Col-0 were measured from at least four individual plants within each experiment resulting in a minimal number of eight plants per line. The plants were measured for both seed and leaf glucosinolates as described above. For the ANOVA, the Genotype term tested for a difference between wild-type and the <em>35S:MYB</em> lines while the impact of the two independent lines per <em>35S:MYB</em> transgene were simultaneously tested by the nested factor, Transgene(Genotype). The above ANOVAs included factors to account for variation between the different experiments.</p>
<a id="article1.body1.sec4.sec11.p2" name="article1.body1.sec4.sec11.p2"></a><p>ANOVA was also utilized to test for the effect of the different knockout mutants. For the knockouts, segregating F2 progeny from heterozygous F1 individuals were screened to identify progeny homozygous for the presence of the T-DNA knockout mutant or homozygous for its absence. The homozygous lines were then measured for glucosinolate content in the seeds and leaves. For each T-DNA knockout mutant line, the experiment was conducted in two fully independent rounds with at least three plants per genotype per experiment for a minimal N of six. The Genotype term within the ANOVA was used to test for a difference between the progeny homozygous for the knockout mutant and those siblings missing the T-DNA. For <em>MYB29</em> and <em>MYB76</em>, there were two independent T-DNA mutant lines. Both lines were tested and the difference between the independent mutants tested as a nested factor using the factor Knockout(Genotype). This allowed us to test if the effects on glucosinolate accumulation were due to the presence of any T-DNA within <em>MYB29</em> and <em>MYB76</em> (Genotype) or if there was a difference between the separate T-DNA alleles [Knockout(Genotype)]. Sums-of Squares, F-values and P-values are presented (<a href="#pone.0001322.s013">Table S12</a>).</p>


<h4>Microarray Analysis of MYB Over-expression</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>Plants for the various genotypes were grown as previously described. At 25 days post germination, a fully-expanded mature leaf was harvested, weighed and analyzed for total aliphatic glucosinolate content via HPLC. The remaining plant material was collected, flash frozen and total RNA extracted via RNeasy columns (Qiagen, Valencia, CA, USA). Two independent plants were combined to provide sufficient starting material for a single RNA extraction. Two independent samples were obtained per transgenic line with two different transgenic lines per <em>35S:MYB</em> transgene, thus providing four-fold replication. Six wild-type Col-0 RNA samples were obtained. This provided a total of 18 independent microarrays. Labeled cRNA was prepared and hybridized, according to the manufacturer's guidelines (Affymetrix, Santa Clara, CA, USA), to whole genome Affymetrix ATH1 GeneChip microarrays, containing 22,746 <em>Arabidopsis</em> transcripts. The GeneChips were scanned with an Affymetrix GeneArray 2500 Scanner and data acquired via the Microarray Suite software MAS 5.0 at the Functional Genomics Laboratory (University of California Berkeley). RMA normalization was used to obtain gene expression levels for all data analyses <a href="#pone.0001322-Irizarry1">[62]</a>.</p>


<h4>Microarray statistical analysis</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>The gene expression data was first analyzed via a network/biosynthetic pathway ANOVA approach utilizing the general linear model within SAS <a href="#pone.0001322-Kliebenstein5">[24]</a>. Sulfur utilization biosynthetic pathways were obtained from AraCyc v3.4 (<a href="http://www.arabidopsis.org/biocyc/">http://www.arabidopsis.org/biocyc/</a>) and modified to better organize the pathways based on metabolites of importance for glucosinolate synthesis (<a href="#pone.0001322.s009">Table S8</a>). Transcription factor networks for aliphatic and indole glucosinolates were added based on this research and previously published research <a href="#pone.0001322-Celenza1">[9]</a>, <a href="#pone.0001322-Levy1">[13]</a>, <a href="#pone.0001322-Skirycz1">[14]</a>. Each selected, independent <em>35S:MYB</em> line was tested against the wild-type data in an independent ANOVA. For the ANOVA, the genes were nested factors within the higher order pathway. Additionally, the two independent lines per <em>35S:MYB</em> transgene were nested factors [Transgene(Genotype)] within the Genotype term (wild-type versus <em>35S:MYB</em>). This allowed us to test for effects due to the transgene versus the independent transgenic line. Each pathway was then tested within the model for a difference between the wild-type and <em>35S:MYB</em> lines using an F-test (<a href="#pone.0001322.s010">Table S9</a>). Additionally, we tested each aliphatic glucosinolate biosynthetic and transcription factor gene for altered transcript accumulation. These individual gene tests were also done within the confines of the model using an F-test to test for a mean separation between wild-type and the <em>35S:MYB</em> line (<a href="#pone.0001322.s011">Table S10</a>). All <em>P</em> values for these genes were significant after a FDR adjustment within the confines of this pathway ANOVA utilizing a pre-defined subset of genes meant to address a specific question about sulfur-utilization and glucosinolate biosynthetic pathways.</p>
<a id="article1.body1.sec4.sec13.p2" name="article1.body1.sec4.sec13.p2"></a><p>Next, the gene expression data was analyzed via individual gene ANOVA for each transcript. This was done by conducting ANOVA on each gene using the two independent transgenic lines per <em>35S:MYB</em> transgene as nested factors [Transgene(Genotype)] within the Genotype (WT versus <em>35S:MYB</em> transgene) effect. The ANOVA calculations were programmed into Microsoft Excel to obtain all appropriate Sums-of-Squares and to obtain the F values for the effect of the Genotype (wild-type versus <em>35S:MYB</em> transgene) and Transgene(Genotype) effects for each gene (<a href="#pone.0001322.s012">Table S11</a>). The nominal <em>P</em> values for both terms are presented as well as the <em>P</em> values for the Genotype (wild-type versus <em>35S:MYB</em> transgene) effect that are significant after a FDR adjustment to the 0.05 level (<a href="#pone.0001322.s012">Table S11</a>) <a href="#pone.0001322-Benjamini1">[63]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001322"></div><a name="pone.0001322.s001" id="pone.0001322.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Multiple Sequence Alignment Alignment of genes used for Phylogenetic Reconstruction.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.21 MB TIF)</p>
<a name="pone.0001322.s002" id="pone.0001322.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">trans eQTL</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s003" id="pone.0001322.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">cis eQTL</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s004" id="pone.0001322.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Gene Lists, tree</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s005" id="pone.0001322.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s005">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Myb coexpression list</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.42 MB XLS)</p>
<a name="pone.0001322.s006" id="pone.0001322.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s006">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">35SMYB Foliar Stats</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s007" id="pone.0001322.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s007">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Abbreviations</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0001322.s008" id="pone.0001322.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s008">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">35SMYB Seed Stats</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s009" id="pone.0001322.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s009">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Pathway Lists</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s010" id="pone.0001322.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s010">Table S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Pathway Results</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001322.s011" id="pone.0001322.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s011">Table S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Aliphatic GLS Array</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s012" id="pone.0001322.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s012">Table S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Microarray data on 35S:MYB overexpressors</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(8.79 MB XLS)</p>
<a name="pone.0001322.s013" id="pone.0001322.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s013">Table S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Foliar MYB KO Stats</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s014" id="pone.0001322.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s014">Table S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Seed MYB KO Stat</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001322.s015" id="pone.0001322.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s015">Table S14. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">Foliar DoubleKO Stats</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001322.s016" id="pone.0001322.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s016">Table S15. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Foliar means doubleKO</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001322.s017" id="pone.0001322.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s017">Table S16. </a></strong></p><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Seed DoubleKO stats</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s017</p><a id="article1.body1.sec5.supplementary-material17.caption1.p2" name="article1.body1.sec5.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001322.s018" id="pone.0001322.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s018">Table S17. </a></strong></p><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Seed means doubleKO</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s018</p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001322.s019" id="pone.0001322.s019"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001322.s019">Table S18. </a></strong></p><a id="article1.body1.sec5.supplementary-material19.caption1.p1" name="article1.body1.sec5.supplementary-material19.caption1.p1"></a><p class="preSiDOI">Primers</p>
<p class="siDoi">doi:10.1371/journal.pone.0001322.s019</p><a id="article1.body1.sec5.supplementary-material19.caption1.p2" name="article1.body1.sec5.supplementary-material19.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Lis Byrsting MÃ¸ller for excellent technical assistance and endurance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DK BGH IS BAH NB. Performed the experiments: CT IS BGH NB. Analyzed the data: DK IS BGH. Wrote the paper: DK BGH IS BAH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001322-Grubb1" id="pone.0001322-Grubb1"></a>Grubb CD, Abel S (2006) Glucosinolate metabolism and its control. Trends Plant Sci  11: 89â100.  <ul class="find" data-citedArticleID="1030732" data-doi="10.1016/j.tplants.2005.12.006"><li><a href="http://dx.doi.org/10.1016/j.tplants.2005.12.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glucosinolate+metabolism+and+its+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glucosinolate+metabolism+and+its+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001322-Halkier1" id="pone.0001322-Halkier1"></a>Halkier BA, Gershenzon J (2006) Biology and Biochemistry of Glucosinolates. Annu Rev Plant Biol  57: 303â333.  <ul class="find" data-citedArticleID="1030734" data-doi="10.1146/annurev.arplant.57.032905.105228"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biology+and+Biochemistry+of+Glucosinolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biology+and+Biochemistry+of+Glucosinolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001322-Kliebenstein1" id="pone.0001322-Kliebenstein1"></a>Kliebenstein DJ, Kroymann J, Mitchell-Olds T (2005) The glucosinolate-myrosinase system in an ecological and evolutionary context. Curr Opin Plant Biol  8: 264â271.  <ul class="find" data-citedArticleID="1030752" data-doi="10.1016/j.pbi.2005.03.002"><li><a href="http://dx.doi.org/10.1016/j.pbi.2005.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+glucosinolate-myrosinase+system+in+an+ecological+and+evolutionary+context." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+glucosinolate-myrosinase+system+in+an+ecological+and+evolutionary+context.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001322-Mithen1" id="pone.0001322-Mithen1"></a>Mithen R, Faulkner K, Magrath R, Rose P, Williamson G, et al.  (2003) Development of isothiocyanate-enriched broccoli, and its enhanced ability to induce phase 2 detoxification enzymes in mammalian cells. Theor Appl Genet  106: 727â734.  <ul class="find" data-citedArticleID="1030784"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Development+of+isothiocyanate-enriched+broccoli%2C+and+its+enhanced+ability+to+induce+phase+2+detoxification+enzymes+in+mammalian+cells.&amp;auth=&amp;atitle=Development+of+isothiocyanate-enriched+broccoli%2C+and+its+enhanced+ability+to+induce+phase+2+detoxification+enzymes+in+mammalian+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+isothiocyanate-enriched+broccoli%2C+and+its+enhanced+ability+to+induce+phase+2+detoxification+enzymes+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+isothiocyanate-enriched+broccoli%2C+and+its+enhanced+ability+to+induce+phase+2+detoxification+enzymes+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001322-Kliebenstein2" id="pone.0001322-Kliebenstein2"></a>Kliebenstein D, Pedersen D, Barker B, Mitchell-Olds T (2002) Comparative analysis of quantitative trait loci controlling glucosinolates, myrosinase and insect resistance in Arabidopsis thaliana. Genetics  161: 325â332.  <ul class="find" data-citedArticleID="1030754"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana.&amp;auth=&amp;atitle=Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+quantitative+trait+loci+controlling+glucosinolates%2C+myrosinase+and+insect+resistance+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001322-Mewis1" id="pone.0001322-Mewis1"></a>Mewis I, Appel HM, Hom A, Raina R, Schultz JC (2005) Major signaling pathways modulate Arabidopsis glucosinolate accumulation and response to both phloem-feeding and chewing insects. Plant Physiol  138: 1149â1162.  <ul class="find" data-citedArticleID="1030780" data-doi="10.1104/pp.104.053389"><li><a href="http://dx.doi.org/10.1104/pp.104.053389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+signaling+pathways+modulate+Arabidopsis+glucosinolate+accumulation+and+response+to+both+phloem-feeding+and+chewing+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+signaling+pathways+modulate+Arabidopsis+glucosinolate+accumulation+and+response+to+both+phloem-feeding+and+chewing+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001322-Mikkelsen1" id="pone.0001322-Mikkelsen1"></a>Mikkelsen MD, Petersen BL, Glawischnig E, Jensen AB, Andreasson E, et al.  (2003) Modulation of CYP79 genes and glucosinolate profiles in Arabidopsis by defense signaling pathways. Plant Physiol  131: 298â308.  <ul class="find" data-citedArticleID="1030782" data-doi="10.1104/pp.011015"><li><a href="http://dx.doi.org/10.1104/pp.011015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+CYP79+genes+and+glucosinolate+profiles+in+Arabidopsis+by+defense+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+CYP79+genes+and+glucosinolate+profiles+in+Arabidopsis+by+defense+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001322-Bender1" id="pone.0001322-Bender1"></a>Bender J, Fink GR (1998) A Myb homologue, ATR1, activates tryptophan gene expression in Arabidopsis. Proc Natl Acad Sci U S A  95: 5655â5660.  <ul class="find" data-citedArticleID="1030702" data-doi="10.1073/pnas.95.10.5655"><li><a href="http://dx.doi.org/10.1073/pnas.95.10.5655" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Myb+homologue%2C+ATR1%2C+activates+tryptophan+gene+expression+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Myb+homologue%2C+ATR1%2C+activates+tryptophan+gene+expression+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001322-Celenza1" id="pone.0001322-Celenza1"></a>Celenza JL, Quiel JA, Smolen GA, Merrikh H, Silvestro AR, et al.  (2005) The Arabidopsis ATR1 Myb transcription factor controls indolic glucosinolate homeostasis. Plant Physiol  137: 253â262.  <ul class="find" data-citedArticleID="1030714" data-doi="10.1104/pp.104.054395"><li><a href="http://dx.doi.org/10.1104/pp.104.054395" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+ATR1+Myb+transcription+factor+controls+indolic+glucosinolate+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+ATR1+Myb+transcription+factor+controls+indolic+glucosinolate+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001322-Gigolashvili1" id="pone.0001322-Gigolashvili1"></a>Gigolashvili T, Yatusevich R, Berger B, Muller C, Flugge UI (2007) The R2R3-MYB transcription factor HAG1/MYB28 is a regulator of methionine-derived glucosinolate biosynthesis in Arabidopsis thaliana. Plant J  51: 247â261.  <ul class="find" data-citedArticleID="1030726" data-doi="10.1111/j.1365-313x.2007.03133.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03133.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+R2R3-MYB+transcription+factor+HAG1%2FMYB28+is+a+regulator+of+methionine-derived+glucosinolate+biosynthesis+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+R2R3-MYB+transcription+factor+HAG1%2FMYB28+is+a+regulator+of+methionine-derived+glucosinolate+biosynthesis+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001322-Kim1" id="pone.0001322-Kim1"></a>Kim JH, Durrett TP, Last RL, Jander G (2004) Characterization of the Arabidopsis TU8 glucosinolate mutation, an allele of TERMINAL FLOWER2. Plant Mol Biol  54: 671â682.  <ul class="find" data-citedArticleID="1030750" data-doi="10.1023/b:plan.0000040897.49151.98"><li><a href="http://dx.doi.org/10.1023/b:plan.0000040897.49151.98" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+Arabidopsis+TU8+glucosinolate+mutation%2C+an+allele+of+TERMINAL+FLOWER2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+Arabidopsis+TU8+glucosinolate+mutation%2C+an+allele+of+TERMINAL+FLOWER2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001322-Bennett1" id="pone.0001322-Bennett1"></a>Bennett RN, Wenke T, Freudenberg B, Mellon FA, Ludwig-Muller J (2005) The tu8 mutation of Arabidopsis thaliana encoding a heterochromatin protein 1 homolog causes defects in the induction of secondary metabolite biosynthesis. Plant Biol (Stuttg)  7: 348â357.  <ul class="find" data-citedArticleID="1030706" data-doi="10.1055/s-2005-837634"><li><a href="http://dx.doi.org/10.1055/s-2005-837634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tu8+mutation+of+Arabidopsis+thaliana+encoding+a+heterochromatin+protein+1+homolog+causes+defects+in+the+induction+of+secondary+metabolite+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tu8+mutation+of+Arabidopsis+thaliana+encoding+a+heterochromatin+protein+1+homolog+causes+defects+in+the+induction+of+secondary+metabolite+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001322-Levy1" id="pone.0001322-Levy1"></a>Levy M, Wang Q, Kaspi R, Parrella MP, Abel S (2005) Arabidopsis IQD1, a novel calmodulin-binding nuclear protein, stimulates glucosinolate accumulation and plant defense. Plant J  43: 79â96.  <ul class="find" data-citedArticleID="1030772" data-doi="10.1111/j.1365-313x.2005.02435.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02435.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+IQD1%2C+a+novel+calmodulin-binding+nuclear+protein%2C+stimulates+glucosinolate+accumulation+and+plant+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+IQD1%2C+a+novel+calmodulin-binding+nuclear+protein%2C+stimulates+glucosinolate+accumulation+and+plant+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001322-Skirycz1" id="pone.0001322-Skirycz1"></a>Skirycz A, Reichelt M, Burow M, Birkemeyer C, Rolcik J, et al.  (2006) DOF transcription factor AtDof1.1 (OBP2) is part of a regulatory network controlling glucosinolate biosynthesis in Arabidopsis. Plant J  47: 10â24.  <ul class="find" data-citedArticleID="1030798" data-doi="10.1111/j.1365-313x.2006.02767.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02767.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DOF+transcription+factor+AtDof1.1+%28OBP2%29+is+part+of+a+regulatory+network+controlling+glucosinolate+biosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DOF+transcription+factor+AtDof1.1+%28OBP2%29+is+part+of+a+regulatory+network+controlling+glucosinolate+biosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001322-Brader1" id="pone.0001322-Brader1"></a>Brader G, Tas E, Palva ET (2001) Jasmonate-dependent induction of indole glucosinolates in Arabidopsis by culture filtrates of the nonspecific pathogen Erwinia carotovora. Plant Physiol  126: 849â860.  <ul class="find" data-citedArticleID="1030708" data-doi="10.1104/pp.126.2.849"><li><a href="http://dx.doi.org/10.1104/pp.126.2.849" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Jasmonate-dependent+induction+of+indole+glucosinolates+in+Arabidopsis+by+culture+filtrates+of+the+nonspecific+pathogen+Erwinia+carotovora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Jasmonate-dependent+induction+of+indole+glucosinolates+in+Arabidopsis+by+culture+filtrates+of+the+nonspecific+pathogen+Erwinia+carotovora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001322-Kliebenstein3" id="pone.0001322-Kliebenstein3"></a>Kliebenstein DJ, Gershenzon J, Mitchell-Olds T (2001) Comparative quantitative trait loci mapping of aliphatic, indolic and benzylic glucosinolate production in Arabidopsis thaliana leaves and seeds. Genetics  159: 359â370.  <ul class="find" data-citedArticleID="1030756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds.&amp;auth=&amp;atitle=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001322-Kliebenstein4" id="pone.0001322-Kliebenstein4"></a>Kliebenstein DJ, West MA, van LH, Kim K, Doerge RW, et al.  (2006) Genomic survey of gene expression diversity in Arabidopsis thaliana. Genetics  172: 1179â1189.  <ul class="find" data-citedArticleID="1030758" data-doi="10.1534/genetics.105.049353"><li><a href="http://dx.doi.org/10.1534/genetics.105.049353" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+survey+of+gene+expression+diversity+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+survey+of+gene+expression+diversity+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001322-Doerge1" id="pone.0001322-Doerge1"></a>Doerge RW (2002) Mapping and analysis of quantitative trait loci in experimental populations. Nat Rev Genet  3: 43â52.  <ul class="find" data-citedArticleID="1030720" data-doi="10.1038/nrg703"><li><a href="http://dx.doi.org/10.1038/nrg703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+and+analysis+of+quantitative+trait+loci+in+experimental+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+and+analysis+of+quantitative+trait+loci+in+experimental+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001322-Schadt1" id="pone.0001322-Schadt1"></a>Schadt EE, Monks SA, Drake TA, Lusis AJ, Che N, et al.  (2003) Genetics of gene expression surveyed in maize, mouse and man. Nature  422: 297â302.  <ul class="find" data-citedArticleID="1030794" data-doi="10.1038/nature01434"><li><a href="http://dx.doi.org/10.1038/nature01434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+gene+expression+surveyed+in+maize%2C+mouse+and+man." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+gene+expression+surveyed+in+maize%2C+mouse+and+man.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001322-Jansen1" id="pone.0001322-Jansen1"></a>Jansen RC, Nap JP (2001) Genetical genomics: the added value from segregation. Trends in Genetics  17: 388â391.  <ul class="find" data-citedArticleID="1030744" data-doi="10.1016/s0168-9525(01)02310-1"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02310-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetical+genomics%3A+the+added+value+from+segregation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetical+genomics%3A+the+added+value+from+segregation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001322-Brem1" id="pone.0001322-Brem1"></a>Brem RB, Yvert G, Clinton R, Kruglyak L (2002) Genetic dissection of transcriptional regulation in budding yeast. Science  296: 752â755.  <ul class="find" data-citedArticleID="1030710" data-doi="10.1126/science.1069516"><li><a href="http://dx.doi.org/10.1126/science.1069516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+dissection+of+transcriptional+regulation+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+dissection+of+transcriptional+regulation+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001322-West1" id="pone.0001322-West1"></a>West MA, Kim K, Kliebenstein DJ, van LH, Michelmore RW, et al.  (2007) Global eQTL mapping reveals the complex genetic architecture of transcript-level variation in Arabidopsis. Genetics  175: 1441â1450.  <ul class="find" data-citedArticleID="1030812" data-doi="10.1534/genetics.106.064972"><li><a href="http://dx.doi.org/10.1534/genetics.106.064972" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+eQTL+mapping+reveals+the+complex+genetic+architecture+of+transcript-level+variation+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+eQTL+mapping+reveals+the+complex+genetic+architecture+of+transcript-level+variation+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001322-Jansen2" id="pone.0001322-Jansen2"></a>Jansen RC, Nap JP (2004) Regulating gene expression: surprises still in store. Trends Genet  20: 223â225.  <ul class="find" data-citedArticleID="1030746" data-doi="10.1016/j.tig.2004.03.002"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulating+gene+expression%3A+surprises+still+in+store." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulating+gene+expression%3A+surprises+still+in+store.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001322-Kliebenstein5" id="pone.0001322-Kliebenstein5"></a>Kliebenstein DJ, West MA, van LH, Loudet O, Doerge RW, et al.  (2006) Identification of QTLs controlling gene expression networks defined a priori. BMC Bioinformatics  7: 308.  <ul class="find" data-citedArticleID="1030760" data-doi="10.1186/1471-2105-7-308"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-308" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+QTLs+controlling+gene+expression+networks+defined+a+priori." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+QTLs+controlling+gene+expression+networks+defined+a+priori.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001322-West2" id="pone.0001322-West2"></a>West MA, van LH, Kozik A, Kliebenstein DJ, Doerge RW, et al.  (2006) High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis. Genome Res  16: 787â795.  <ul class="find" data-citedArticleID="1030814" data-doi="10.1101/gr.5011206"><li><a href="http://dx.doi.org/10.1101/gr.5011206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-density+haplotyping+with+microarray-based+expression+and+single+feature+polymorphism+markers+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-density+haplotyping+with+microarray-based+expression+and+single+feature+polymorphism+markers+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001322-Stracke1" id="pone.0001322-Stracke1"></a>Stracke R, Werber M, Weisshaar B (2001) The R2R3-MYB gene family in Arabidopsis thaliana. Curr Opin Plant Biol  4: 447â456.  <ul class="find" data-citedArticleID="1030800" data-doi="10.1016/s1369-5266(00)00199-0"><li><a href="http://dx.doi.org/10.1016/s1369-5266(00)00199-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+R2R3-MYB+gene+family+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+R2R3-MYB+gene+family+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001322-Ramsay1" id="pone.0001322-Ramsay1"></a>Ramsay NA, Glover BJ (2005) MYB-bHLH-WD40 protein complex and the evolution of cellular diversity. Trends Plant Sci  10: 63â70.  <ul class="find" data-citedArticleID="1030790" data-doi="10.1016/j.tplants.2004.12.011"><li><a href="http://dx.doi.org/10.1016/j.tplants.2004.12.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MYB-bHLH-WD40+protein+complex+and+the+evolution+of+cellular+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MYB-bHLH-WD40+protein+complex+and+the+evolution+of+cellular+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001322-Martin1" id="pone.0001322-Martin1"></a>Martin C, Bhatt K, Baumann K, Jin H, Zachgo S, et al.  (2002) The mechanics of cell fate determination in petals. Philos Trans R Soc Lond B Biol Sci  357: 809â813.  <ul class="find" data-citedArticleID="1030778" data-doi="10.1098/rstb.2002.1089"><li><a href="http://dx.doi.org/10.1098/rstb.2002.1089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mechanics+of+cell+fate+determination+in+petals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mechanics+of+cell+fate+determination+in+petals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001322-Agarwal1" id="pone.0001322-Agarwal1"></a>Agarwal M, Hao Y, Kapoor A, Dong CH, Fujii H, et al.  (2006) A R2R3 type MYB transcription factor is involved in the cold regulation of CBF genes and in acquired freezing tolerance. J Biol Chem  281: 37636â37645.  <ul class="find" data-citedArticleID="1030696" data-doi="10.1074/jbc.m605895200"><li><a href="http://dx.doi.org/10.1074/jbc.m605895200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+R2R3+type+MYB+transcription+factor+is+involved+in+the+cold+regulation+of+CBF+genes+and+in+acquired+freezing+tolerance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+R2R3+type+MYB+transcription+factor+is+involved+in+the+cold+regulation+of+CBF+genes+and+in+acquired+freezing+tolerance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001322-Zhu1" id="pone.0001322-Zhu1"></a>Zhu J, Verslues PE, Zheng X, Lee BH, Zhan X, et al.  (2005) HOS10 encodes an R2R3-type MYB transcription factor essential for cold acclimation in plants. Proc Natl Acad Sci U S A  102: 9966â9971.  <ul class="find" data-citedArticleID="1030820" data-doi="10.1073/pnas.0503960102"><li><a href="http://dx.doi.org/10.1073/pnas.0503960102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HOS10+encodes+an+R2R3-type+MYB+transcription+factor+essential+for+cold+acclimation+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HOS10+encodes+an+R2R3-type+MYB+transcription+factor+essential+for+cold+acclimation+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001322-Gray1" id="pone.0001322-Gray1"></a>Gray J (2005) Guard cells: transcription factors regulate stomatal movements. Curr Biol  15: R593âR595.  <ul class="find" data-citedArticleID="1030730" data-doi="10.1016/j.cub.2005.07.039"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.07.039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guard+cells%3A+transcription+factors+regulate+stomatal+movements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Guard+cells%3A+transcription+factors+regulate+stomatal+movements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001322-Gigolashvili2" id="pone.0001322-Gigolashvili2"></a>Gigolashvili T, Berger B, Mock HP, Muller C, Weisshaar B, et al.  (2007) The transcription factor HIG1/MYB51 regulates indolic glucosinolate biosynthesis in Arabidopsis thaliana. Plant J  50: 886â901.  <ul class="find" data-citedArticleID="1030728" data-doi="10.1111/j.1365-313x.2007.03099.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03099.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcription+factor+HIG1%2FMYB51+regulates+indolic+glucosinolate+biosynthesis+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcription+factor+HIG1%2FMYB51+regulates+indolic+glucosinolate+biosynthesis+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001322-Hansen1" id="pone.0001322-Hansen1"></a>Hansen BG, Kliebenstein DJ, Halkier BA (2007) Identification of a flavin monoxygenase as the S-oxygenating enzyme in aliphatic glucosinolate biosynthesis in Arabidopsis. Plant J  902-910:   <ul class="find" data-citedArticleID="1030736" data-doi="10.1111/j.1365-313x.2007.03101.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03101.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+flavin+monoxygenase+as+the+S-oxygenating+enzyme+in+aliphatic+glucosinolate+biosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+flavin+monoxygenase+as+the+S-oxygenating+enzyme+in+aliphatic+glucosinolate+biosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001322-Hirai1" id="pone.0001322-Hirai1"></a>Hirai MY, Sugiyama K, Sawada Y, Tohge T, Obayashi T, et al.  (2007) Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis. Proc Natl Acad Sci U S A  104: 6478â6483.  <ul class="find" data-citedArticleID="1030740" data-doi="10.1073/pnas.0611629104"><li><a href="http://dx.doi.org/10.1073/pnas.0611629104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Omics-based+identification+of+Arabidopsis+Myb+transcription+factors+regulating+aliphatic+glucosinolate+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Omics-based+identification+of+Arabidopsis+Myb+transcription+factors+regulating+aliphatic+glucosinolate+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001322-Kliebenstein6" id="pone.0001322-Kliebenstein6"></a>Kliebenstein DJ, D'Auria JC, Behere AS, Kim JH, Gunderson KL, et al.  (2007) Characterization of seed-specific benzoyloxyglucosinolate mutations in Arabidopsis thaliana. Plant J  51: 1062â1076.  <ul class="find" data-citedArticleID="1030762" data-doi="10.1111/j.1365-313x.2007.03205.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03205.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+seed-specific+benzoyloxyglucosinolate+mutations+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+seed-specific+benzoyloxyglucosinolate+mutations+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001322-Petersen1" id="pone.0001322-Petersen1"></a>Petersen BL, Chen S, Hansen CH, Olsen CE, Halkier BA (2002) Composition and content of glucosinolates in developing Arabidopsis thaliana. Planta  214: 562â571.  <ul class="find" data-citedArticleID="1030788" data-doi="10.1007/s004250100659"><li><a href="http://dx.doi.org/10.1007/s004250100659" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Composition+and+content+of+glucosinolates+in+developing+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Composition+and+content+of+glucosinolates+in+developing+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001322-Brown1" id="pone.0001322-Brown1"></a>Brown PD, Tokuhisa JG, Reichelt M, Gershenzon J (2003) Variation of glucosinolate accumulation among different organs and developmental stages of Arabidopsis thaliana. Phytochemistry  62: 471â481.  <ul class="find" data-citedArticleID="1030712" data-doi="10.1016/s0031-9422(02)00549-6"><li><a href="http://dx.doi.org/10.1016/s0031-9422(02)00549-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+of+glucosinolate+accumulation+among+different+organs+and+developmental+stages+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+of+glucosinolate+accumulation+among+different+organs+and+developmental+stages+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001322-Hemm1" id="pone.0001322-Hemm1"></a>Hemm MR, Ruegger MO, Chapple C (2003) The Arabidopsis ref2 mutant is defective in the gene encoding CYP83A1 and shows both phenylpropanoid and glucosinolate phenotypes. Plant Cell  15: 179â194.  <ul class="find" data-citedArticleID="1030738" data-doi="10.1105/tpc.006544"><li><a href="http://dx.doi.org/10.1105/tpc.006544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+ref2+mutant+is+defective+in+the+gene+encoding+CYP83A1+and+shows+both+phenylpropanoid+and+glucosinolate+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+ref2+mutant+is+defective+in+the+gene+encoding+CYP83A1+and+shows+both+phenylpropanoid+and+glucosinolate+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001322-Kliebenstein7" id="pone.0001322-Kliebenstein7"></a>Kliebenstein DJ, Rowe HC, Denby KJ (2005) Secondary metabolites influence Arabidopsis/Botrytis interactions: variation in host production and pathogen sensitivity. Plant J  44: 25â36.  <ul class="find" data-citedArticleID="1030764" data-doi="10.1111/j.1365-313x.2005.02508.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02508.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secondary+metabolites+influence+Arabidopsis%2FBotrytis+interactions%3A+variation+in+host+production+and+pathogen+sensitivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secondary+metabolites+influence+Arabidopsis%2FBotrytis+interactions%3A+variation+in+host+production+and+pathogen+sensitivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001322-Jin1" id="pone.0001322-Jin1"></a>Jin H, Cominelli E, Bailey P, Parr A, Mehrtens F, et al.  (2000) Transcriptional repression by AtMYB4 controls production of UV-protecting sunscreens in Arabidopsis. EMBO J  19: 6150â6161.  <ul class="find" data-citedArticleID="1030748" data-doi="10.1093/emboj/19.22.6150"><li><a href="http://dx.doi.org/10.1093/emboj/19.22.6150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+repression+by+AtMYB4+controls+production+of+UV-protecting+sunscreens+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+repression+by+AtMYB4+controls+production+of+UV-protecting+sunscreens+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001322-Lehfeldt1" id="pone.0001322-Lehfeldt1"></a>Lehfeldt C, Shirley AM, Meyer K, Ruegger MO, Cusumano JC, et al.  (2000) Cloning of the SNG1 gene of Arabidopsis reveals a role for a serine carboxypeptidase-like protein as an acyltransferase in secondary metabolism. Plant Cell  12: 1295â1306.  <ul class="find" data-citedArticleID="1030770" data-doi="10.2307/3871130"><li><a href="http://dx.doi.org/10.2307/3871130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+the+SNG1+gene+of+Arabidopsis+reveals+a+role+for+a+serine+carboxypeptidase-like+protein+as+an+acyltransferase+in+secondary+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+the+SNG1+gene+of+Arabidopsis+reveals+a+role+for+a+serine+carboxypeptidase-like+protein+as+an+acyltransferase+in+secondary+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001322-Sinlapadech1" id="pone.0001322-Sinlapadech1"></a>Sinlapadech T, Stout J, Ruegger MO, Deak M, Chapple C (2007) The hyper-fluorescent trichome phenotype of the brt1 mutant of Arabidopsis is the result of a defect in a sinapic acid: UDPG glucosyltransferase. Plant J  49: 655â668.  <ul class="find" data-citedArticleID="1030796" data-doi="10.1111/j.1365-313x.2006.02984.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02984.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hyper-fluorescent+trichome+phenotype+of+the+brt1+mutant+of+Arabidopsis+is+the+result+of+a+defect+in+a+sinapic+acid%3A+UDPG+glucosyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hyper-fluorescent+trichome+phenotype+of+the+brt1+mutant+of+Arabidopsis+is+the+result+of+a+defect+in+a+sinapic+acid%3A+UDPG+glucosyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001322-Alonso1" id="pone.0001322-Alonso1"></a>Alonso JM, Stepanova AN, Leisse TJ, Kim CJ, Chen H, et al.  (2003) Genome-Wide Insertional Mutagenesis of Arabidopsis thaliana. Science  301: 653â657.  <ul class="find" data-citedArticleID="1030698" data-doi="10.1126/science.1086391"><li><a href="http://dx.doi.org/10.1126/science.1086391" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-Wide+Insertional+Mutagenesis+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-Wide+Insertional+Mutagenesis+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001322-Rosso1" id="pone.0001322-Rosso1"></a>Rosso MG, Li Y, Strizhov N, Reiss B, Dekker K, et al.  (2003) An Arabidopsis thaliana T-DNA mutagenized population (GABI-Kat) for flanking sequence tag-based reverse genetics. Plant Mol Biol  53: 247â259.  <ul class="find" data-citedArticleID="1030792" data-doi="10.1023/b:plan.0000009297.37235.4a"><li><a href="http://dx.doi.org/10.1023/b:plan.0000009297.37235.4a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Arabidopsis+thaliana+T-DNA+mutagenized+population+%28GABI-Kat%29+for+flanking+sequence+tag-based+reverse+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Arabidopsis+thaliana+T-DNA+mutagenized+population+%28GABI-Kat%29+for+flanking+sequence+tag-based+reverse+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001322-Tissier1" id="pone.0001322-Tissier1"></a>Tissier AF, Marillonnet S, Klimyuk V, Patel K, Torres MA, et al.  (1999) Multiple Independent Defective Suppressor-mutator Transposon Insertions in Arabidopsis: A Tool for Functional Genomics. Plant Cell  11: 1841â1852.  <ul class="find" data-citedArticleID="1030806" data-doi="10.2307/3871081"><li><a href="http://dx.doi.org/10.2307/3871081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+Independent+Defective+Suppressor-mutator+Transposon+Insertions+in+Arabidopsis%3A+A+Tool+for+Functional+Genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+Independent+Defective+Suppressor-mutator+Transposon+Insertions+in+Arabidopsis%3A+A+Tool+for+Functional+Genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001322-Wentzell1" id="pone.0001322-Wentzell1"></a>Wentzell A, Rowe H, Hansen B, Ticconi C, Halkier B, et al.  (2007) Linking Metabolic QTL with Network and <em>cis</em>-eQTL Controlling Biosynthetic Pathways. PLoS Genetics  3: 1687â1701.  <ul class="find" data-citedArticleID="1030810" data-doi="10.1371/journal.pgen.0030162.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030162.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linking+Metabolic+QTL+with+Network+and+cis-eQTL+Controlling+Biosynthetic+Pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linking+Metabolic+QTL+with+Network+and+cis-eQTL+Controlling+Biosynthetic+Pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001322-Kliebenstein8" id="pone.0001322-Kliebenstein8"></a>Kliebenstein DJ, Lambrix VM, Reichelt M, Gershenzon J, Mitchell-Olds T (2001) Gene duplication in the diversification of secondary metabolism: tandem 2-oxoglutarate-dependent dioxygenases control glucosinolate biosynthesis in Arabidopsis. Plant Cell  13: 681â693.  <ul class="find" data-citedArticleID="1030766" data-doi="10.2307/3871415"><li><a href="http://dx.doi.org/10.2307/3871415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+in+the+diversification+of+secondary+metabolism%3A+tandem+2-oxoglutarate-dependent+dioxygenases+control+glucosinolate+biosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+in+the+diversification+of+secondary+metabolism%3A+tandem+2-oxoglutarate-dependent+dioxygenases+control+glucosinolate+biosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001322-Textor1" id="pone.0001322-Textor1"></a>Textor S, de Kraker JW, Hause B, Gershenzon J, Tokuhisa JG (2007) MAM3 Catalyzes the Formation of All Aliphatic Glucosinolate Chain Lengths in Arabidopsis. Plant Physiol  144: 60â71.  <ul class="find" data-citedArticleID="1030802" data-doi="10.1104/pp.106.091579"><li><a href="http://dx.doi.org/10.1104/pp.106.091579" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAM3+Catalyzes+the+Formation+of+All+Aliphatic+Glucosinolate+Chain+Lengths+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAM3+Catalyzes+the+Formation+of+All+Aliphatic+Glucosinolate+Chain+Lengths+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001322-Kliebenstein9" id="pone.0001322-Kliebenstein9"></a>Kliebenstein DJ, Kroymann J, Brown P, Figuth A, Pedersen D, et al.  (2001) Genetic control of natural variation in Arabidopsis glucosinolate accumulation. Plant Physiol  126: 811â825.  <ul class="find" data-citedArticleID="1030768" data-doi="10.1104/pp.126.2.811"><li><a href="http://dx.doi.org/10.1104/pp.126.2.811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+control+of+natural+variation+in+Arabidopsis+glucosinolate+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+control+of+natural+variation+in+Arabidopsis+glucosinolate+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001322-Textor2" id="pone.0001322-Textor2"></a>Textor S, Bartram S, Kroymann J, Falk KL, Hick A, et al.  (2004) Biosynthesis of methionine-derived glucosinolates in Arabidopsis thaliana: recombinant expression and characterization of methylthioalkylmalate synthase, the condensing enzyme of the chain-elongation cycle. Planta  218: 1026â1035.  <ul class="find" data-citedArticleID="1030804" data-doi="10.1007/s00425-003-1184-3"><li><a href="http://dx.doi.org/10.1007/s00425-003-1184-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+of+methionine-derived+glucosinolates+in+Arabidopsis+thaliana%3A+recombinant+expression+and+characterization+of+methylthioalkylmalate+synthase%2C+the+condensing+enzyme+of+the+chain-elongation+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+of+methionine-derived+glucosinolates+in+Arabidopsis+thaliana%3A+recombinant+expression+and+characterization+of+methylthioalkylmalate+synthase%2C+the+condensing+enzyme+of+the+chain-elongation+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001322-Field1" id="pone.0001322-Field1"></a>Field B, Cardon G, Traka M, Botterman J, Vancanneyt G, et al.  (2004) Glucosinolate and amino acid biosynthesis in Arabidopsis. Plant Physiol  135: 828â839.  <ul class="find" data-citedArticleID="1030724" data-doi="10.1104/pp.104.039347"><li><a href="http://dx.doi.org/10.1104/pp.104.039347" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glucosinolate+and+amino+acid+biosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glucosinolate+and+amino+acid+biosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001322-Loudet1" id="pone.0001322-Loudet1"></a>Loudet O, Chaillou S, Camilleri C, Bouchez D, Niel-Vedele F (2002) Bay-0ÃShahdara recombinant inbred line population: a powerful tool for the genetic dissection of complex traits in Arabidopsis. Theor Appl Genet  104: 1173â1184.  <ul class="find" data-citedArticleID="1030774" data-doi="10.1007/s00122-001-0825-9"><li><a href="http://dx.doi.org/10.1007/s00122-001-0825-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bay-0%C3%97Shahdara+recombinant+inbred+line+population%3A+a+powerful+tool+for+the+genetic+dissection+of+complex+traits+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bay-0%C3%97Shahdara+recombinant+inbred+line+population%3A+a+powerful+tool+for+the+genetic+dissection+of+complex+traits+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001322-Wang1" id="pone.0001322-Wang1"></a>Wang S, Basten CJ, Zeng Z-B (2006) Windows QTL Cartographer 2.5. Raleigh, NC: Department of Statistics, North Carolina State University.   <ul class="find-nolinks"></ul></li><li><span class="label">54.
              </span><a name="pone.0001322-Basten1" id="pone.0001322-Basten1"></a>Basten CJ, Weir BS, Zeng Z-B (1999) QTL Cartographer, version 1.13. Raleigh, NC: Department of Statistics, North Carolina State University.   <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0001322-Zeng1" id="pone.0001322-Zeng1"></a>Zeng ZB, Kao CH, Basten CJ (1999) Estimating the genetic architecture of quantitative traits. Genet Res  74: 279â289.  <ul class="find" data-citedArticleID="1030818" data-doi="10.1017/s0016672399004255"><li><a href="http://dx.doi.org/10.1017/s0016672399004255" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+genetic+architecture+of+quantitative+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+genetic+architecture+of+quantitative+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001322-Churchill1" id="pone.0001322-Churchill1"></a>Churchill GA, Doerge RW (1994) Empirical threshold values for quantitative trait mapping. Genetics  138: 963â971.  <ul class="find" data-citedArticleID="1030716"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+threshold+values+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+threshold+values+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001322-Doerge2" id="pone.0001322-Doerge2"></a>Doerge RW, Churchill GA (1996) Permutation tests for multiple loci affecting a quantitative character. Genetics  142: 285â294.  <ul class="find" data-citedArticleID="1030722"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Permutation+tests+for+multiple+loci+affecting+a+quantitative+character.&amp;auth=&amp;atitle=Permutation+tests+for+multiple+loci+affecting+a+quantitative+character." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Permutation+tests+for+multiple+loci+affecting+a+quantitative+character." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Permutation+tests+for+multiple+loci+affecting+a+quantitative+character.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001322-NourEldin1" id="pone.0001322-NourEldin1"></a>Nour-Eldin HH, Hansen BG, Norholm MH, Jensen JK, Halkier BA (2006) Advancing uracil-excision based cloning towards an ideal technique for cloning PCR fragments. Nucleic Acids Res  34: e122.  <ul class="find" data-citedArticleID="1030786" data-doi="10.1093/nar/gkl635"><li><a href="http://dx.doi.org/10.1093/nar/gkl635" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advancing+uracil-excision+based+cloning+towards+an+ideal+technique+for+cloning+PCR+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advancing+uracil-excision+based+cloning+towards+an+ideal+technique+for+cloning+PCR+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001322-Zambryski1" id="pone.0001322-Zambryski1"></a>Zambryski P, Joos H, Genetello C, Leemans J, Montagu MV, et al.  (1983) Ti plasmid vector for the introduction of DNA into plant cells without alteration of their normal regeneration capacity. EMBO J  2: 2143â2150.  <ul class="find" data-citedArticleID="1030816"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Ti+plasmid+vector+for+the+introduction+of+DNA+into+plant+cells+without+alteration+of+their+normal+regeneration+capacity.&amp;auth=&amp;atitle=Ti+plasmid+vector+for+the+introduction+of+DNA+into+plant+cells+without+alteration+of+their+normal+regeneration+capacity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ti+plasmid+vector+for+the+introduction+of+DNA+into+plant+cells+without+alteration+of+their+normal+regeneration+capacity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ti+plasmid+vector+for+the+introduction+of+DNA+into+plant+cells+without+alteration+of+their+normal+regeneration+capacity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001322-Clough1" id="pone.0001322-Clough1"></a>Clough SJ, Bent AF (1998) Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana. Plant J  16: 735â743.  <ul class="find" data-citedArticleID="1030718" data-doi="10.1046/j.1365-313x.1998.00343.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00343.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Floral+dip%3A+a+simplified+method+for+Agrobacterium-mediated+transformation+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Floral+dip%3A+a+simplified+method+for+Agrobacterium-mediated+transformation+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001322-Lukowitz1" id="pone.0001322-Lukowitz1"></a>Lukowitz W, Gillmor CS, Scheible WR (2000) Positional Cloning in Arabidopsis. Why It Feels Good to Have a Genome Initiative Working for You. Plant Physiol  123: 795â806.  <ul class="find" data-citedArticleID="1030776" data-doi="10.1104/pp.123.3.795"><li><a href="http://dx.doi.org/10.1104/pp.123.3.795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positional+Cloning+in+Arabidopsis.+Why+It+Feels+Good+to+Have+a+Genome+Initiative+Working+for+You." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positional+Cloning+in+Arabidopsis.+Why+It+Feels+Good+to+Have+a+Genome+Initiative+Working+for+You.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001322-Irizarry1" id="pone.0001322-Irizarry1"></a>Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, et al.  (2003) Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics  4: 249â264.  <ul class="find" data-citedArticleID="1030742" data-doi="10.1093/biostatistics/4.2.249"><li><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001322-Benjamini1" id="pone.0001322-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling The False Discovery Rate-A Practical and Powerful Approach to Multiple Testing. J R Statist Soc B  57: 289â300.  <ul class="find" data-citedArticleID="1030704"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+The+False+Discovery+Rate-A+Practical+and+Powerful+Approach+to+Multiple+Testing.&amp;auth=&amp;atitle=Controlling+The+False+Discovery+Rate-A+Practical+and+Powerful+Approach+to+Multiple+Testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+The+False+Discovery+Rate-A+Practical+and+Powerful+Approach+to+Multiple+Testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+The+False+Discovery+Rate-A+Practical+and+Powerful+Approach+to+Multiple+Testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.XML" value="216144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.PDF" value="551026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g001.PNG_L" value="100072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g001.PNG_M" value="70647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g001.PNG_S" value="15622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g001.TIF" value="1095704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g001.PNG_I" value="131017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g002.PNG_L" value="365693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g002.PNG_M" value="21311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g002.PNG_S" value="2717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g002.TIF" value="921742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g002.PNG_I" value="8731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g003.PNG_L" value="343944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g003.PNG_M" value="53251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g003.PNG_S" value="4069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g003.TIF" value="1447128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g003.PNG_I" value="22210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t001.PNG_L" value="163968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t001.PNG_M" value="175510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t001.PNG_S" value="21002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t001.TIF" value="709954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t001.PNG_I" value="86347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t002.PNG_L" value="198127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t002.PNG_M" value="201410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t002.PNG_S" value="23235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t002.TIF" value="811148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t002.PNG_I" value="109791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t003.PNG_L" value="269490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t003.PNG_M" value="185166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t003.PNG_S" value="25683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t003.TIF" value="1171132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t003.PNG_I" value="174663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g004.PNG_L" value="90194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g004.PNG_M" value="93046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g004.PNG_S" value="6170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g004.TIF" value="809488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g004.PNG_I" value="35892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g005.PNG_L" value="186846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g005.PNG_M" value="75345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g005.PNG_S" value="9886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g005.TIF" value="2382034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g005.PNG_I" value="71566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g006.PNG_L" value="212594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g006.PNG_M" value="30927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g006.PNG_S" value="3231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g006.TIF" value="986816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g006.PNG_I" value="12429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g007.PNG_L" value="1840195"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g007.PNG_M" value="49219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g007.PNG_S" value="3807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g007.TIF" value="2189048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g007.PNG_I" value="18404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t004.PNG_L" value="211246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t004.PNG_M" value="128233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t004.PNG_S" value="14593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t004.TIF" value="955530"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t004.PNG_I" value="40865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t005.PNG_L" value="237352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t005.PNG_M" value="152504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t005.PNG_S" value="15237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t005.TIF" value="1057490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.t005.PNG_I" value="48405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g008.PNG_L" value="2373850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g008.PNG_M" value="44773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g008.PNG_S" value="4904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g008.TIF" value="3937478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.g008.PNG_I" value="24882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s001.TIF" value="2214190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s002.XLS" value="34304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s003.XLS" value="28672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s004.XLS" value="26112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s005.XLS" value="420352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s006.XLS" value="25600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s007.XLS" value="91136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s008.XLS" value="29184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s009.XLS" value="25088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s010.XLS" value="22016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s011.XLS" value="32256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s012.XLS" value="8793088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s013.XLS" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s014.XLS" value="28160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s015.XLS" value="17920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s016.XLS" value="18432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s017.XLS" value="17920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s018.XLS" value="17920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001322.s019.XLS" value="18944"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001322&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001322" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001322&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001322&volume=&issue=&title=A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates&author_name=Ida%20Elken%20S%C3%B8nderby%2C%20Bjarne%20Gram%20Hansen%2C%20Nanna%20Bjarnholt%2C%20Carla%20Ticconi%2C%20Barbara%20Ann%20Halkier%2C%20Daniel%20J.%20Kliebenstein&start_page=1&end_page=16" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001322" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322&amp;t=A%20Systems%20Biology%20Approach%20Identifies%20a%20R2R3%20MYB%20Gene%20Subfamily%20with%20Distinct%20and%20Overlapping%20Functions%20in%20Regulation%20of%20Aliphatic%20Glucosinolates" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322&title=A%20Systems%20Biology%20Approach%20Identifies%20a%20R2R3%20MYB%20Gene%20Subfamily%20with%20Distinct%20and%20Overlapping%20Functions%20in%20Regulation%20of%20Aliphatic%20Glucosinolates&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322&amp;title=A%20Systems%20Biology%20Approach%20Identifies%20a%20R2R3%20MYB%20Gene%20Subfamily%20with%20Distinct%20and%20Overlapping%20Functions%20in%20Regulation%20of%20Aliphatic%20Glucosinolates" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001322&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Systems Biology Approach Identifies a R2R3 MYB Gene Subfamily with Distinct and Overlapping Functions in Regulation of Aliphatic Glucosinolates';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001322';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Systems%20Biology%20Approach%20Identifies%20a%20R2R3%20MYB%20Gene%20Subfamily%20with%20Distinct%20and%20Overlapping%20Functions%20in%20Regulation%20of%20Aliphatic%20Glucosinolates http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001322" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001322" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Analysis+of+variance%22" title="Search for articles in the subject area:'Analysis of variance'"><div class="flagText">Analysis of variance</div></a>
              <div data-categoryid="42193" data-articleid="25474"
                   data-categoryname="Analysis of variance"
                   class="flagImage" title="Flag 'Analysis of variance' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="25474"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25474"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="25474"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Indoles%22" title="Search for articles in the subject area:'Indoles'"><div class="flagText">Indoles</div></a>
              <div data-categoryid="20495" data-articleid="25474"
                   data-categoryname="Indoles"
                   class="flagImage" title="Flag 'Indoles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Regulator+genes%22" title="Search for articles in the subject area:'Regulator genes'"><div class="flagText">Regulator genes</div></a>
              <div data-categoryid="18579" data-articleid="25474"
                   data-categoryname="Regulator genes"
                   class="flagImage" title="Flag 'Regulator genes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="25474"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transgenic+plants%22" title="Search for articles in the subject area:'Transgenic plants'"><div class="flagText">Transgenic plants</div></a>
              <div data-categoryid="20465" data-articleid="25474"
                   data-categoryname="Transgenic plants"
                   class="flagImage" title="Flag 'Transgenic plants' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4822'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6019'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3014&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
