

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000386"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000386" />

    <meta name="citation_title" content="Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1"/>
    <meta itemprop="name" content="Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1"/>

      <meta name="citation_author" content="Michael Opel"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="David Lando"/>
            <meta name="citation_author_institution" content="Gurdon Institute and Department of Pathology, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Carolina Bonilla"/>
            <meta name="citation_author_institution" content="Department of Biosciences/School of Life Science, Karolinska Institutet, University College Sodertorn, Huddinge, Sweden"/>
      <meta name="citation_author" content="Sarah C. Trewick"/>
            <meta name="citation_author_institution" content="Wellcome Trust Centre for Cell Biology, The University of Edinburgh, Edinburgh, United Kingdom"/>
      <meta name="citation_author" content="Abdelhalim Boukaba"/>
            <meta name="citation_author_institution" content="Wellcome Trust Centre for Cell Biology, The University of Edinburgh, Edinburgh, United Kingdom"/>
      <meta name="citation_author" content="Julian Walfridsson"/>
            <meta name="citation_author_institution" content="Department of Biosciences/School of Life Science, Karolinska Institutet, University College Sodertorn, Huddinge, Sweden"/>
      <meta name="citation_author" content="James Cauwood"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Petra J.H. Werler"/>
            <meta name="citation_author_institution" content="Genome Damage and Stability Centre, School of Biological Sciences, University of Sussex, Falmer, Sussex, United Kingdom"/>
      <meta name="citation_author" content="Antony M. Carr"/>
            <meta name="citation_author_institution" content="Genome Damage and Stability Centre, School of Biological Sciences, University of Sussex, Falmer, Sussex, United Kingdom"/>
      <meta name="citation_author" content="Tony Kouzarides"/>
            <meta name="citation_author_institution" content="Gurdon Institute and Department of Pathology, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Natalia V. Murzina"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Robin C. Allshire"/>
            <meta name="citation_author_institution" content="Wellcome Trust Centre for Cell Biology, The University of Edinburgh, Edinburgh, United Kingdom"/>
      <meta name="citation_author" content="Karl Ekwall"/>
            <meta name="citation_author_institution" content="Department of Biosciences/School of Life Science, Karolinska Institutet, University College Sodertorn, Huddinge, Sweden"/>
      <meta name="citation_author" content="Ernest D. Laue"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"/>

    <meta name="citation_date" content="2007/4/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000386.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e386"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The diverse functions of histone lysine methylation.; citation_author=C Martin; citation_author=Y Zhang; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=6; citation_number=1; citation_pages=838-849; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Histone demethylation mediated by the nuclear amine oxidase homolog LSD1.; citation_author=Y Shi; citation_author=F Lan; citation_author=C Matson; citation_author=P Mulligan; citation_author=JR Whetstine; citation_journal_title=Cell; citation_volume=119; citation_number=2; citation_pages=941-953; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The putative oncogene GASC1 demethylates tri- and dimethylated lysine 9 on histone H3.; citation_author=PA Cloos; citation_author=J Christensen; citation_author=K Agger; citation_author=A Maiolica; citation_author=J Rappsilber; citation_journal_title=Nature; citation_volume=442; citation_number=3; citation_pages=307-311; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The transcriptional repressor JHDM3A demethylates trimethyl histone H3 lysine 9 and lysine 36.; citation_author=RJ Klose; citation_author=K Yamane; citation_author=Y Bae; citation_author=D Zhang; citation_author=H Erdjument-Bromage; citation_journal_title=Nature; citation_volume=442; citation_number=4; citation_pages=312-316; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Histone demethylation by a family of JmjC domain-containing proteins.; citation_author=Y Tsukada; citation_author=J Fang; citation_author=H Erdjument-Bromage; citation_author=ME Warren; citation_author=CH Borchers; citation_journal_title=Nature; citation_volume=439; citation_number=5; citation_pages=811-816; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=JHDM2A, a JmjC-containing H3K9 demethylase, facilitates transcription activation by androgen receptor.; citation_author=K Yamane; citation_author=C Toumazou; citation_author=Y Tsukada; citation_author=H Erdjument-Bromage; citation_author=P Tempst; citation_journal_title=Cell; citation_volume=125; citation_number=6; citation_pages=483-495; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stable histone deacetylase complexes distinguished by the presence of SANT domain proteins CoREST/kiaa0071 and Mta-L1.; citation_author=GW Humphrey; citation_author=Y Wang; citation_author=VR Russanova; citation_author=T Hirai; citation_author=J Qin; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=7; citation_pages=6817-6824; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A core-BRAF35 complex containing histone deacetylase mediates repression of neuronal-specific genes.; citation_author=MA Hakimi; citation_author=DA Bochar; citation_author=J Chenoweth; citation_author=WS Lane; citation_author=G Mandel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=8; citation_pages=7420-7425; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Coordinated histone modifications mediated by a CtBP co-repressor complex.; citation_author=Y Shi; citation_author=J Sawada; citation_author=G Sui; citation_author=B Affar el; citation_author=JR Whetstine; citation_journal_title=Nature; citation_volume=422; citation_number=9; citation_pages=735-738; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Functional interplay between histone demethylase and deacetylase enzymes.; citation_author=MG Lee; citation_author=C Wynder; citation_author=DA Bochar; citation_author=MA Hakimi; citation_author=N Cooch; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=10; citation_pages=6395-6402; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation.; citation_author=MG Lee; citation_author=C Wynder; citation_author=N Cooch; citation_author=R Shiekhattar; citation_journal_title=Nature; citation_volume=437; citation_number=11; citation_pages=432-435; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of LSD1 histone demethylase activity by its associated factors.; citation_author=YJ Shi; citation_author=C Matson; citation_author=F Lan; citation_author=S Iwase; citation_author=T Baba; citation_journal_title=Mol Cell; citation_volume=19; citation_number=12; citation_pages=857-864; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription.; citation_author=E Metzger; citation_author=M Wissmann; citation_author=N Yin; citation_author=JM Muller; citation_author=R Schneider; citation_journal_title=Nature; citation_volume=437; citation_number=13; citation_pages=436-439; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Fission yeast homologs of human histone H3 lysine 4 demethylase regulate a common set of genes with diverse functions.; citation_author=E Nicolas; citation_author=MG Lee; citation_author=MA Hakimi; citation_author=H Cam; citation_author=SS Grewal; citation_journal_title=J Biol Chem; citation_volume=47; citation_number=14; citation_pages=35983-8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein N-methyltransferase assays in the study of gene transcription.; citation_author=R Schneider; citation_author=AJ Bannister; citation_journal_title=Methods; citation_volume=26; citation_number=15; citation_pages=226-232; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional divergence between histone deacetylases in fission yeast by distinct cellular localization and in vivo specificity.; citation_author=P Bjerling; citation_author=RA Silverstein; citation_author=G Thon; citation_author=A Caudy; citation_author=S Grewal; citation_journal_title=Mol Cell Biol; citation_volume=22; citation_number=16; citation_pages=2170-2181; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Histone deacetylase homologs regulate epigenetic inheritance of transcriptional silencing and chromosome segregation in fission yeast.; citation_author=SI Grewal; citation_author=MJ Bonaduce; citation_author=AJ Klar; citation_journal_title=Genetics; citation_volume=150; citation_number=17; citation_pages=563-576; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genomewide analysis of nucleosome density histone acetylation and HDAC function in fission yeast.; citation_author=M Wiren; citation_author=RA Silverstein; citation_author=I Sinha; citation_author=J Walfridsson; citation_author=HM Lee; citation_journal_title=EMBO J; citation_volume=24; citation_number=18; citation_pages=2906-2918; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transitions in distinct histone H3 methylation patterns at the heterochromatin domain boundaries.; citation_author=K Noma; citation_author=CD Allis; citation_author=SI Grewal; citation_journal_title=Science; citation_volume=293; citation_number=19; citation_pages=1150-1155; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Establishment and maintenance of a heterochromatin domain.; citation_author=IM Hall; citation_author=GD Shankaranarayana; citation_author=K Noma; citation_author=N Ayoub; citation_author=A Cohen; citation_journal_title=Science; citation_volume=297; citation_number=20; citation_pages=2232-2237; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Centromere silencing and function in fission yeast is governed by the amino terminus of histone H3.; citation_author=BG Mellone; citation_author=L Ball; citation_author=N Suka; citation_author=MR Grunstein; citation_author=JF Partridge; citation_journal_title=Curr Biol; citation_volume=13; citation_number=21; citation_pages=1748-1757; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide patterns of histone modifications in fission yeast.; citation_author=I Sinha; citation_author=I Wiren; citation_author=K Ekwall; citation_journal_title=Chromatin Res; citation_volume=14; citation_number=22; citation_pages=95-105; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Alp13, an MRG family protein, is a component of fission yeast Clr6 histone deacetylase required for genomic integrity.; citation_author=J Nakayama; citation_author=G Xiao; citation_author=K Noma; citation_author=A Malikzay; citation_author=P Bjerling; citation_journal_title=Embo J; citation_volume=22; citation_number=23; citation_pages=2776-2787; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Double chromodomains cooperate to recognize the methylated histone H3 tail.; citation_author=JF Flanagan; citation_author=LZ Mi; citation_author=M Chruszcz; citation_author=M Cymborowski; citation_author=KL Clines; citation_journal_title=Nature; citation_volume=438; citation_number=24; citation_pages=1181-1185; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human but not yeast CHD1 binds directly and selectively to histone H3 methylated at lysine 4 via its tandem chromodomains.; citation_author=RJ Sims; citation_author=CF Chen; citation_author=H Santos-Rosa; citation_author=T Kouzarides; citation_author=SS Patel; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=25; citation_pages=41789-92; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Heterologous modules for efficient and versatile PCR-based gene targeting in Schizosaccharomyces pombe.; citation_author=J Bahler; citation_author=JQ Wu; citation_author=MS Longtine; citation_author=NG Shah; citation_author=A McKenzie; citation_journal_title=Yeast; citation_volume=14; citation_number=26; citation_pages=943-951; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Vectors and gene targeting modules for tandem affinity purification in Schizosaccharomyces pombe.; citation_author=JJ Tasto; citation_author=RH Carnahan; citation_author=WH McDonald; citation_author=KL Gould; citation_journal_title=Yeast; citation_volume=18; citation_number=27; citation_pages=657-662; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A simple Cre-loxP method for chromosomal N-terminal tagging of essential and non-essential Schizosaccharomyces pombe genes.; citation_author=PJ Werler; citation_author=E Hartsuiker; citation_author=AM Carr; citation_journal_title=Gene; citation_volume=304; citation_number=28; citation_pages=133-141; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A generic protein purification method for protein complex characterization and proteome exploration.; citation_author=G Rigaut; citation_author=A Shevchenko; citation_author=B Rutz; citation_author=M Wilm; citation_author=M Mann; citation_journal_title=Nat Biotechnol; citation_volume=17; citation_number=29; citation_pages=1030-1032; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Sensitive formaldehyde determination with Nash's reagent and a ‘tryptophan reaction’.; citation_author=U Kleeberg; citation_author=W Klinger; citation_journal_title=J Pharmacol Methods; citation_volume=8; citation_number=30; citation_pages=19-31; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=A DNA microarray for fission yeast: minimal changes in global gene expression after temperature shift.; citation_author=Y Xue; citation_author=SA Haas; citation_author=L Brino; citation_author=A Gusnanto; citation_author=M Reimers; citation_journal_title=Yeast; citation_volume=21; citation_number=31; citation_pages=25-39; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomewide histone acetylation microarrays.; citation_author=D Robyr; citation_author=M Grunstein; citation_journal_title=Methods; citation_volume=31; citation_number=32; citation_pages=83-89; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide binding map of the histone deacetylase Rpd3 in yeast.; citation_author=SK Kurdistani; citation_author=D Robyr; citation_author=S Tavazoie; citation_author=M Grunstein; citation_journal_title=Nat Genet; citation_volume=31; citation_number=33; citation_pages=248-254; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ChIP-chip: considerations for the design, analysis, and application of genome-wide chromatin immunoprecipitation experiments.; citation_author=MJ Buck; citation_author=JD Lieb; citation_journal_title=Genomics; citation_volume=83; citation_number=34; citation_pages=349-360; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1"/>
    <meta name="twitter:description" content="In order to gain a more global view of the activity of histone demethylases, we report here genome-wide studies of the fission yeast SWIRM and polyamine oxidase (PAO) domain homologues of mammalian LSD1. Consistent with previous work we find that the two S. pombe proteins, which we name Swm1 and Swm2 (after SWIRM1 and SWIRM2), associate together in a complex. However, we find that this complex specifically demethylates lysine 9 in histone H3 (H3K9) and both up- and down-regulates expression of different groups of genes. Using chromatin-immunoprecipitation, to isolate fragments of chromatin containing either H3K4me2 or H3K9me2, and DNA microarray analysis (ChIP-chip), we have studied genome-wide changes in patterns of histone methylation, and their correlation with gene expression, upon deletion of the swm1+ gene. Using hyper-geometric probability comparisons we uncover genetic links between lysine-specific demethylases, the histone deacetylase Clr6, and the chromatin remodeller Hrp1. The data presented here demonstrate that in fission yeast the SWIRM/PAO domain proteins Swm1 and Swm2 are associated in complexes that can remove methyl groups from lysine 9 methylated histone H3. In vitro, we show that bacterially expressed Swm1 also possesses lysine 9 demethylase activity. In vivo, loss of Swm1 increases the global levels of both H3K9me2 and H3K4me2. A significant accumulation of H3K4me2 is observed at genes that are up-regulated in a swm1 deletion strain. In addition, H3K9me2 accumulates at some genes known to be direct Swm1/2 targets that are down-regulated in the swm1&Delta; strain. The in vivo data indicate that Swm1 acts in concert with the HDAC Clr6 and the chromatin remodeller Hrp1 to repress gene expression. In addition, our in vitro analyses suggest that the H3K9 demethylase activity requires an unidentified post-translational modification to allow it to act. Thus, our results highlight complex interactions between histone demethylase, deacetylase and chromatin remodelling activities in the regulation of gene expression."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000386.g004"/>

  <meta property="og:title" content="Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3886'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6558'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6605&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000386">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000386" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000386&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Opel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Lando, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gurdon Institute and Department of Pathology, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carolina Bonilla, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biosciences/School of Life Science, Karolinska Institutet, University College Sodertorn, Huddinge, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sarah C. Trewick, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Wellcome Trust Centre for Cell Biology, The University of Edinburgh, Edinburgh, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Abdelhalim Boukaba, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Wellcome Trust Centre for Cell Biology, The University of Edinburgh, Edinburgh, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julian Walfridsson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biosciences/School of Life Science, Karolinska Institutet, University College Sodertorn, Huddinge, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James Cauwood, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Petra J.H. Werler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Damage and Stability Centre, School of Biological Sciences, University of Sussex, Falmer, Sussex, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Antony M. Carr, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Damage and Stability Centre, School of Biological Sciences, University of Sussex, Falmer, Sussex, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tony Kouzarides, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Gurdon Institute and Department of Pathology, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Natalia V. Murzina, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robin C. Allshire
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:robin.allshire@ed.ac.uk">robin.allshire@ed.ac.uk</a> (RCA); <a href="mailto:karl.ekwall@sh.se">karl.ekwall@sh.se</a> (KE); <a href="mailto:e.d.laue@bioc.cam.ac.uk">e.d.laue@bioc.cam.ac.uk</a> (EL)</p>

                <p>Affiliation:
                  Wellcome Trust Centre for Cell Biology, The University of Edinburgh, Edinburgh, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karl Ekwall
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:robin.allshire@ed.ac.uk">robin.allshire@ed.ac.uk</a> (RCA); <a href="mailto:karl.ekwall@sh.se">karl.ekwall@sh.se</a> (KE); <a href="mailto:e.d.laue@bioc.cam.ac.uk">e.d.laue@bioc.cam.ac.uk</a> (EL)</p>

                <p>Affiliation:
                  Department of Biosciences/School of Life Science, Karolinska Institutet, University College Sodertorn, Huddinge, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ernest D. Laue
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:robin.allshire@ed.ac.uk">robin.allshire@ed.ac.uk</a> (RCA); <a href="mailto:karl.ekwall@sh.se">karl.ekwall@sh.se</a> (KE); <a href="mailto:e.d.laue@bioc.cam.ac.uk">e.d.laue@bioc.cam.ac.uk</a> (EL)</p>

                <p>Affiliation:
                  Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000386</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000386-g001" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000386-g002" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000386-g003" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000386-g004" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000386-t001" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000386-t002" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000386">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000386" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>In order to gain a more global view of the activity of histone demethylases, we report here genome-wide studies of the fission yeast SWIRM and polyamine oxidase (PAO) domain homologues of mammalian LSD1. Consistent with previous work we find that the two <em>S. pombe</em> proteins, which we name Swm1 and Swm2 (after SWIRM1 and SWIRM2), associate together in a complex. However, we find that this complex specifically demethylates lysine 9 in histone H3 (H3K9) and both up- and down-regulates expression of different groups of genes. Using chromatin-immunoprecipitation, to isolate fragments of chromatin containing either H3K4me2 or H3K9me2, and DNA microarray analysis (ChIP-chip), we have studied genome-wide changes in patterns of histone methylation, and their correlation with gene expression, upon deletion of the <em>swm1<sup>+</sup></em> gene. Using hyper-geometric probability comparisons we uncover genetic links between lysine-specific demethylases, the histone deacetylase Clr6, and the chromatin remodeller Hrp1. The data presented here demonstrate that in fission yeast the SWIRM/PAO domain proteins Swm1 and Swm2 are associated in complexes that can remove methyl groups from lysine 9 methylated histone H3. <em>In vitro</em>, we show that bacterially expressed Swm1 also possesses lysine 9 demethylase activity. <em>In vivo</em>, loss of Swm1 increases the global levels of both H3K9me2 and H3K4me2. A significant accumulation of H3K4me2 is observed at genes that are up-regulated in a <em>swm1</em> deletion strain. In addition, H3K9me2 accumulates at some genes known to be direct Swm1/2 targets that are down-regulated in the <em>swm1</em>Δ strain. The <em>in vivo</em> data indicate that Swm1 acts in concert with the HDAC Clr6 and the chromatin remodeller Hrp1 to repress gene expression. In addition, our <em>in vitro</em> analyses suggest that the H3K9 demethylase activity requires an unidentified post-translational modification to allow it to act. Thus, our results highlight complex interactions between histone demethylase, deacetylase and chromatin remodelling activities in the regulation of gene expression.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Opel M, Lando D, Bonilla C, Trewick SC, Boukaba A, et al.  (2007) Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1. PLoS ONE 2(4):
          e386.
            doi:10.1371/journal.pone.0000386</p><p><strong>Academic Editor: </strong>Juan Valcarcel, Centre de Regulació Genòmica, Spain</p><p><strong>Received:</strong> February 6, 2007; <strong>Accepted:</strong> March 22, 2007; <strong>Published:</strong> April 18, 2007</p><p><strong>Copyright:</strong> © 2007 Opel et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>R.C.A. and E.D.L. thank the Wellcome Trust for financial support. K.E. is a Royal Swedish Academy of Sciences Research Fellow supported by grants from the Knut and Alice Wallenberg Foundation, Swedish Cancer Society, Swedish Research Council (VR) M. Bergvalls stiftelse and the EU The Epigenome NoE network. R.C.A is a Wellcome Trust Principal Research Fellow; S.C.T is a Dorothy Hodgkin Fellow of the Royal Society; D.L. is a CJ Martin Fellow of the National Health and Medical Research Council of Australia.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Post-translational modifications of histones regulate gene transcription either by recruiting other proteins/complexes or by altering the underlying chromatin structure. Until recently one such modification, lysine methylation, which can either activate or repress gene transcription [for a review see ref. 1], was thought to be irreversible. However, two classes of protein demethylase, that specifically remove methyl groups from lysine, have now been identified <a href="#pone.0000386-Shi1">[2]</a>–<a href="#pone.0000386-Yamane1">[6]</a>. One of these, represented by lysine-specific demethylase 1 (LSD1), also known as BHC110, is a flavin adenine nucleotide-dependent (FAD) amine oxidase that removes methyl-groups from mono- and di-methylated lysine 4 of histone H3 (H3K4) <a href="#pone.0000386-Shi1">[2]</a>. LSD1 is a component of various complexes that repress transcription and which often contain HDAC1/2 and CoREST <a href="#pone.0000386-Humphrey1">[7]</a>–<a href="#pone.0000386-Lee1">[10]</a>. Recent studies show that the specificity and activity of the enzyme is modulated by its association with different proteins <a href="#pone.0000386-Lee2">[11]</a>–<a href="#pone.0000386-Metzger1">[13]</a>. Metzger et al., (2005) <a href="#pone.0000386-Metzger1">[13]</a>, have interestingly shown that LSD1 when associated in a complex with the androgen receptor specifically demethylates H3K9 (instead of H3K4). The activity of LSD1 is also modulated by association with a SANT domain from the CoREST protein, which recruits the demethylase to nucleosomal substrates <a href="#pone.0000386-Lee2">[11]</a>–<a href="#pone.0000386-Shi3">[12]</a>. In addition, it has also been suggested that demethylation of nucleosomes by the LSD1-CoREST complex is inhibited by BHC80, a PHD domain protein <a href="#pone.0000386-Lee2">[11]</a>, as well as by histone acetylation <a href="#pone.0000386-Shi3">[12]</a>. These results suggest a model whereby demethylase activity can be targeted in alternative ways to different sites and that it is regulated by other modifications, e.g. acetylation, to coordinate different activities.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Identification of the members of the Swm complexes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Strains expressing C-terminally TAP-tagged Swm1 and Swm2 (from their endogenous promoters) were used to affinity purify complexes of the two proteins. The associated proteins were subsequently identified by mass spectrometry (MS). The results of the MS analysis are presented in <a href="#pone.0000386.s001">Table S1</a> in the Supplementary Information and are summarized in <a href="#pone-0000386-g001">Figure 1</a>. In brief, our data confirm the results of Nicolas et al., (2006) <a href="#pone.0000386-Nicolas1">[14]</a> and show that the Swm1 complex contains Swm2 and two new PHD domain containing proteins (<em>S. pombe</em> DB CDS: SPCC4G3.07 and SPAC30D11.08c), hereafter referred to as Swp1 (<strong>Sw</strong>m associated <strong>P</strong>HD1) and Swp2 (<strong>Sw</strong>m associated <strong>P</strong>HD2). Surprisingly, the purified Swm2 complex contained only Swm1 and Swp1, but not Swp2, suggesting that the Swm proteins may exist in more than one complex. However, in contrast to the results of Nicolas et al., (2006) <a href="#pone.0000386-Nicolas1">[14]</a> we did not detect either Hrp1 or SPBPJ758.01 (an RNA recognition motif protein), suggesting that these proteins have weaker affinities or are more transiently associated with the complexes. Alternatively, the use of different tags in the two studies may also explain the discreprency.</p>
<div class="figure" id="pone-0000386-g001"><div class="img"><a name="pone-0000386-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Schematic representation of Swm1/2 complex members: Swm1 (SPBC146.09c), Swm2 (SPAC23E2.02), Swp1 (SPCC4G3.07) and Swp2 (SPAC30D11.08c) are annotated with domain borders as in the SMART sequence analysis database (<a href="http://smart.embl-heidelberg.de">http://smart.embl-heidelberg.de</a>).</p>
<span>doi:10.1371/journal.pone.0000386.g001</span></div>

<h4>The S. pombe Swm complex demethylates lysine-9 of histone H3</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Previous studies have shown that human LSD1 can demethylate either lysine residues 4 or 9 in histone H3, depending on the presence or absence of associated proteins <a href="#pone.0000386-Metzger1">[13]</a>. To determine whether the <em>S. pombe</em> Swm complex can also demethylate histones, we carried out a series of demethylation assays using purified TAP-tagged complexes.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Initial assays for histone demethylase activity were attempted using similar methods to those described by Shi et al., (2004) <a href="#pone.0000386-Shi1">[2]</a>, involving MS and synthetic peptide substrates. However, when comparing the detection limits (assuming a similar level of activity to the <em>E. coli</em> expressed human LSD1 as a control) we found that we were not able to purify sufficient quantities of the TAP-tagged Swm1 and Swm2 complexes from <em>S. pombe</em> to reliably detect demethylase activity in MS assays. We therefore turned to a more sensitive histone demethylase assay, in which purified histone methylases are used to radiolabel histone substrates <a href="#pone.0000386-Tsukada1">[5]</a>, <a href="#pone.0000386-Schneider1">[15]</a>. In this work we used human Set7, <em>S. pombe</em> Clr4 and <em>S. cerevisiae</em> Set2 to specifically methylate K4, K9 and K36 in histone H3, either in bulk histones or in polynucleosomes. Because FAD-dependent amine oxidases should demethylate both mono- and di-methylated substrates <a href="#pone.0000386-Shi1">[2]</a>, we used Set7 to monomethylate H3K4 (and not the Set1 complex which leads to trimethylation). We also tried to ensure that the Clr4 reaction (which can also lead to trimethylation of H3K9) was partial – by carrying out the labelling reaction for short periods of time.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Interestingly, only methylated H3K9 was found to be a substrate for the <em>S. pombe</em> Swm1 and Swm2 complexes (<a href="#pone-0000386-g002">Figure 2a</a>). Similar levels of activity were detected regardless of whether the complex was purified using TAP-tagged Swm1 or Swm2. Moreover, similar levels of activity were found for histone H3 substrates in the form of either purified bulk histones or nucleosomes. By contrast, in the human LSD1 control we observed specific demethylation of H3K4 as previously reported (<a href="#pone-0000386-g002">Figure 2b</a>) <a href="#pone.0000386-Shi1">[2]</a>. We next tested whether the <em>S. pombe</em> Swm1 and Swm2 complexes were able to demethylate K9 in isolated histone H3. While both TAP-tagged complexes were found to have similar activity towards histone H3 purified from calf thymus (in comparison to nucleosomal substrates), neither complex was active on recombinant histone H3 (<a href="#pone-0000386-g002">Figure 2c</a>). It is currently unclear why the <em>S. pombe</em> demethylase complex is inactive on recombinant methylated histone H3, but one possibility is that an additional post-translational modification(s) is needed for the complex to effectively recognise its substrate. Finally, to rule out the possibility that a contaminant was responsible for the H3K9 demethylase activity, we demonstrated that recombinant <em>E. coli</em> expressed Swm1 has the same pattern of activity/specificity as that of the intact complex (<a href="#pone-0000386-g002">Figure 2d</a>). This experiment demonstrates that Swm1 is a catalytically active subunit. However, because we have not so far been able to express/purify recombinant Swm2, this PAO domain protein may also be catalytically active. Structure-based sequence analysis suggests that Swm2 also has the appropriate FAD-binding and catalytic residues required for demethylase activity (data not shown).</p>
<div class="figure" id="pone-0000386-g002"><div class="img"><a name="pone-0000386-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Histone demethylase activity of TAP tagged Swm1 and Swm2 complexes (A&amp;C), recombinant human LSD1 (B), and GST-Swm1 (D), with various methylated histone substrates. The substrates: calf thymus bulk histones (BH), chicken polynucleosomes (Nuc), calf thymus histone H3 (H3) and recombinant H3 (rH3) along with their sites of lysine (K) methylation are indicated below the panels. Control indicates a mock TAP-tag purification from the wild type strain. hLSD1 was recombinant <em>E. coli</em> expressed human LSD1 protein and GST-Swm1 was recombinant <em>E. coli</em> expressed Swm1.</p>
<span>doi:10.1371/journal.pone.0000386.g002</span></div>

<h4>Swm1 both up- and down-regulates gene expression</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To determine if the Swm1 and Swm2 proteins are involved in regulation of gene expression, we constructed strains of <em>S. pombe</em> in which either the Swm1 or Swm2 genes were deleted. Consistent with the results obtained by Nicolas et al., (2006) <a href="#pone.0000386-Nicolas1">[14]</a>, we found that <em>swm2</em> is an essential gene and that deletion of <em>swm1</em> markedly increased the cell doubling time (data not shown). Thus, gene expression profiling could only be carried out in cells lacking Swm1.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Our global analysis of gene expression shows that somewhat more genes are up-regulated in the <em>swm1</em> deletion (<em>swm1</em>Δ) strain (265 genes) than are down-regulated (173 genes) (using a 1.5 fold cut-off; see <a href="#pone.0000386.s002">Table S2</a> in the Supplementary information). In <a href="#pone-0000386-g003">Figure 3</a>, the moving average (calculated over 150 genes) of the <em>swm1</em>Δ/wild-type expression ratio <em>vs</em> the level of wild-type gene expression is plotted. If the <em>swm1</em> deletion had no effect on gene expression one would expect a global moving average ratio of 1.0. As can be seen, however, the <em>swm1</em>Δ/wild-type expression ratio is generally high for non-expressed and weakly expressed genes, suggesting that (globally) Swm1 represses these non-expressed and weakly expressed genes.</p>
<div class="figure" id="pone-0000386-g003"><div class="img"><a name="pone-0000386-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Global analysis of the gene expression profile in <em>swm1</em> <em>vs</em> wild type cells. A moving average plot (window size = 150 genes, step size = 1 gene) of the median gene expression ratio, <em>swm1</em>Δ/wt, plotted as a function of the average transcription levels of 11 wild-type cell cultures in mid-logarithmic growth.</p>
<span>doi:10.1371/journal.pone.0000386.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We emphasise, however, that Swm1 also has an activating role at 173 genes. The effect of <em>swm1</em> deletion on these genes appears to be direct. In hyper-geometric distribution tests, we found a statistically significant overlap when comparing the published genome-wide Swm1/2 localisation data <a href="#pone.0000386-Nicolas1">[14]</a> with the list of genes that are down-regulated in <em>swm1</em>Δ cells (P = 3.18×10-7), but not those that are up-regulated (compare <a href="#pone-0000386-g004">Figure 4c bottom, left and right</a>).</p>
<div class="figure" id="pone-0000386-g004"><div class="img"><a name="pone-0000386-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000386.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Genome wide analysis of histone methylation and gene expression in <em>swm1</em> cells, and binding of Swm1/2. A) A comparison of increased H3K9me2 and altered gene expression in <em>swm1</em> deletion cells. The Venn diagrams illustrate the degree of overlap between lists of IGR and ORF regions having high H3K9me2 in <em>swm1</em> deletion cells (using a cutoff value of 2.0 ), and a list of genes, which showed altered expression in <em>swm1</em>Δ (using a cutoff value of 1.5). Left: 8.2% of total <em>S. pombe</em> genes showed high IGR or ORF H3K9me2 levels in <em>swm1</em>Δ. Middle: The Venn diagram shows the fraction of genes up-regulated in <em>swm1</em>Δ having high IGR or ORF H3K9me2. Right: The Venn diagram shows the fraction of genes down-regulated in <em>swm1</em>Δ having high IGR or ORF H3K9me2. The inserted table shows a list of 10 genes down-regulated in <em>swm1</em> deletion cells, which also show high H3K9me2 ORF levels in <em>swm1</em>Δ (Swm1/2 binding targets are indicated in bold). B) A comparison of increased H3K4me2 and altered gene expression in <em>swm1</em> deletion cells. The Venn diagrams illustrate the degree of overlap between lists of IGR and ORF regions having high H3K4me2 in <em>swm1</em> deletion cells (using a cutoff value of 2.0 ), and a list of genes, which showed altered expression in <em>swm1</em> (using a cutoff value of 1.5). Left: 3.8% of total <em>S. pombe</em> genes showed high IGR or ORF H3K4me2 levels in <em>swm1</em>Δ. Middle: The Venn diagram shows the fraction of genes up-regulated in <em>swm1</em>Δ having high IGR or ORF H3K4me2. 9.8% of <em>swm1</em> up-regulated genes showed high IGR or ORF H3K4me2 levels, which is significantly more than expected from the genome average (CHI square, P&lt;0.01; indicated). Right: The Venn diagram shows the fraction of genes down-regulated in <em>swm1</em>Δ having high IGR or ORF H3K4me2. C) Comparison of Swm1/2 binding targets (determined by Nicolas et al., (2006)), H3K4me2 and H3K9me2 levels, as well as gene expression changes in <em>swm1</em> deletion cells. Top: The Venn diagrams illustrate the degree of overlap between lists of IGR and ORF regions having high H3K4me2 and H3K9me2 levels (as indicated) in <em>swm1</em> deletion cells, and a list of Swm1/2-binding targets. 13.4% of Swm1/2 binding targets showed high IGR or ORF H3K4me2 levels, which is significantly more than expected from the genome average (CHI square, P&gt;0.001; indicated). Bottom: The Venn diagrams illustrate the degree of overlap between the list of Swm1/2-binding targets and gene expression changes in <em>swm1</em> deletion cells. Bottom right: A significant proportion of s<em>wm1</em> down-regulated genes (hypergeometric P value indicated) were defined as Swm1/2-binding targets. (Note that our microarray contains only 143 of the 175 targets published by Nicolas et al., (2006).)</p>
<span>doi:10.1371/journal.pone.0000386.g004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Further analysis, using hyper-geometric distribution tests, with a database of genes affected by histone modifications, showed that there is a very significant similarity of up-regulated genes in the <em>swm1</em>Δ strain to up-regulated genes in the <em>clr6-1</em> strain (see <a href="#pone-0000386-t001">Table 1</a>). <em>clr6-1</em> is a loss of function mutant of Clr6, a histone deacetylase (HDAC) and known transcriptional repressor, which is similar to HDAC1/2 in higher organisms <a href="#pone.0000386-Bjerling1">[16]</a>–<a href="#pone.0000386-Grewal1">[17]</a>. Consistent with Swm1 acting in concert with Clr6, there is also a significant overlap of genes that are up-regulated in the <em>swm1</em>Δ strain with genes whose promoters lack acetylation in wild-type cells <a href="#pone.0000386-Wiren1">[18]</a> (see <a href="#pone-0000386-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000386-t001"><div class="img"><a name="pone-0000386-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000386.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000386.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Hyper-geometric probability comparisons of genes either down- or up-regulated in <em>swm1</em> deletion cells.</span></strong></p><span>doi:10.1371/journal.pone.0000386.t001</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Our analyses extend the observations of Nicolas et al., (2006) <a href="#pone.0000386-Nicolas1">[14]</a>. Thus, it is now clear that the <em>S. pombe</em> Swm1/2 complex has dual roles in gene regulation, up-regulating some 173 genes and down-regulating some 265 others. The correlation between Swm1/2 localisation and down-regulated genes in <em>swm1</em>Δ cells, suggests that the complex might have a more direct role in stimulating gene expression than it does in repression. Moreover, the imposition of a repressed state by Swm1/2 involves one of the HDACs – Clr6.</p>


<h4>The effects of Swm1 deletion on K4 and K9-methylation of histone H3</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We next measured levels of H3K4 and H3K9 dimethylation using ChIP-chip experiments in both inter-genic (IGR) (including all pol II promoters) and open-reading frame (ORF) regions in the <em>swm1</em>Δ strain [see Ref. 18 for details of the microarray platform]. (Our studies focussed on the levels of dimethylation, as opposed to monomethylation, because of the availability of suitable antibodies. In addition, levels of trimethylation are known to be low in <em>S. pombe</em>.) In agreement with our finding that the fission yeast Swm complex demethylates H3K9 (<a href="#pone-0000386-g002">Figure 2</a>), deletion of <em>swm1</em> resulted in increased levels of H3K9me2 in a large part of the genome – 8.2% of genes showed increased H3K9me2 in either the ORF or IGR regions. H3K4me2 levels, however, were also increased in a smaller part of the genome (3.8%). Interestingly, there appears to be a bias towards increased levels of H3K4me2 in ORF regions (as compared to IGR regions), whereas H3K9me2 levels were increased in both the ORF and IGR regions (compare <a href="#pone-0000386-g004">Figures 4a, 4b</a>). (See <a href="#pone.0000386.s003">Table S3</a> in the Supplementary information for lists of the IGR and ORF regions with high levels of H3K4me2 and H3K9me2 in the <em>swm1</em>Δ strain.) (Note that the <em>S. pombe</em> genome is composed of roughly equal amounts of IGR and ORF regions.)</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Globally, overall levels of H3K9me2 were not significantly changed in genes whose expression is either up- or down-regulated in the absence of Swm1 (<a href="#pone-0000386-g004">Figure 4a</a>). However, on closer examination of the genes that are activated by Swm1, a few have increased H3K9me2 levels in the absence of Swm1. Within this group of 10 genes, <em>sti1</em>+ and SPCC1620.06c were previously identified as Swm1/2 binding targets <a href="#pone.0000386-Nicolas1">[14]</a>. This data suggests that in wild-type cells Swm1 may have an activating function at some genes, which involves demethylation of H3K9me2 in ORF regions, consistent with the general role of H3K9me2 in gene repression and silencing in <em>S. pombe</em> <a href="#pone.0000386-Noma1">[19]</a>–<a href="#pone.0000386-Mellone1">[21]</a>. However, due to the small number of genes involved, this does not appear to be the major role of the Swm1/2 complex.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>In genes that are repressed by Swm1, and therefore up-regulated in the <em>swm1</em>Δ strain, levels of H3K4me2 were seen significantly more frequently (CHI square, P&lt;0.01). The bias towards increased levels of H3K4me2 in ORF regions (as compared to the IGR regions) was again apparent (<a href="#pone-0000386-g004">Figure 4b</a>). However, no difference in H3K4me2 levels was observed in genes that are activated by Swm1, and therefore repressed in the <em>swm1</em>Δ strain.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>The bias towards increased H3K4me2 levels in ORF regions of genes that are up-regulated in <em>swm1</em>Δ cells, when taken together with the lack of <em>in-vitro</em> H3K4 demethylase activity of the Swm1/2 complex (see above), suggests that the increased H3K4me2 levels results from either increased H3K4 methylation via Set1, or the incorporation of H3K4 methylated histones during transcription.</p>


<h4>Correlation of H3K9me2 or H3K4me2 levels with other post-translational modifications</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The changes in H3K9me2 or H3K4me2 levels observed at genes in the <em>swm1</em>Δ cells may correlate with other post-translational modifications associated with the same genes. To test this we compared the lists of IGR and ORF regions showing either high H3K9me2 or H3K4me2 in <em>swm1</em>Δ cells with our database of genes affected by histone modifications using hyper-geometric distribution tests.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The regions which show increased H3K4me2 in the <em>swm1</em>Δ cells were very significantly similar to regions where histone acetylation is low in wild-type cells. They were also significantly similar to regions with increased acetylation in the <em>clr6-1</em> mutant <a href="#pone.0000386-Wiren1">[18]</a> (<a href="#pone-0000386-t002">Table 2</a>). These findings are consistent with the observation that there is a very significant similarity of up-regulated genes in the <em>swm1</em>Δ strain to up-regulated genes in the <em>clr6-1</em> strain (see above and <a href="#pone-0000386-t001">Table 1</a>). The functional link to Clr6 is also interesting given that LSD1 (a human homologue of Swm1) physically and functionally interacts with HDAC1/2 <a href="#pone.0000386-Humphrey1">[7]</a>–<a href="#pone.0000386-Lee1">[10]</a>. Moreover, the regions associated with increased H3K4me2 in <em>swm1</em>Δ cells tend to be the 3′ regions of longer genes (&gt;1000 bp), which in wild type cells normally have low levels of both histone acetylation and H3K4me2, and which are hyperacetylated in <em>clr6-1</em> cells <a href="#pone.0000386-Sinha1">[22]</a>. These findings again suggest that Swm1 and Clr6 may collaborate to maintain repressive chromatin in both IGR and ORF regions, to influence both transcriptional initiation and elongation. Furthermore, the affected IGR regions in the <em>swm1</em>Δ cells also overlap significantly with Clr3 and Sir2 HDAC binding data (<a href="#pone-0000386-t002">Table 2</a>).</p>
<div class="figure" id="pone-0000386-t002"><div class="img"><a name="pone-0000386-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000386" data-uri="info:doi/10.1371/journal.pone.0000386.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000386.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000386.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000386.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000386.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Hyper-geometric probability comparisons of IGR and ORF lists with high H3K4me2 in <em>swm1</em> deletion cells.</span></strong></p><span>doi:10.1371/journal.pone.0000386.t002</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The lists of IGR regions which show increased levels of H3K4me2 in the <em>swm1</em>Δ strain also overlapped significantly with lists of genes representing IGR binding of the Hrp1 chromatin remodelling factor and its closely related (64% identical) paralogue Hrp3 (P = 9.01×10-6 and 5.64×10-4, respectively; see <a href="#pone-0000386-t002">Table 2</a>). Moreover, the lists of up-regulated genes in the <em>swm1</em>Δ strain showed significant similarity to lists of IGR regions bound by Hrp1 and Hrp3 (P = 5.48×10-6 and 1.33×10-5, respectively; See <a href="#pone-0000386-t001">Table 1</a>).</p>


<h4>Possible targeting of the Swm complex via Clr6?</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Given the finding that loss of function of either the Swm1 histone demethylase or Clr6 HDAC, results in up-regulation of genes and altered levels of histone modifications in the same regions of the genome, and the knowledge that human LSD1 physically and functionally interacts with HDAC1/2 <a href="#pone.0000386-Humphrey1">[7]</a>–<a href="#pone.0000386-Lee1">[10]</a>, we wondered whether there was a physical interaction between the <em>S. pombe</em> Swm1 and Clr6 complexes? Because we had not identified any of the Clr6 complex components in TAP-tagging/MS studies of Swm1 and Swm2, we also TAP-tagged components of the Clr6 complex, to see if we could identify any of the Swm partner proteins. These studies, however, only identified the same proteins found in previous work <a href="#pone.0000386-Nakayama1">[23]</a>, namely: Clr6, Alp13, Prw1 and Pst2 (data not shown). In preliminary experiments we have found that Hrp1 and Hrp3 co-purify in affinity purification experiments (Walfridsson, Khorosjutina, Gustafsson, Ekwall <em>et. al.</em> manuscript in preparation). Our results, therefore, indicate that there is a conserved functional interaction between the Swm1 histone demethylase and the Clr6 histone deacetylase that is also found in mammalian cells. They also provide further support for a model in which deacetylation of nucleosomes sets the stage for demethylation <a href="#pone.0000386-Shi3">[12]</a>. In <em>S. pombe</em>, however, despite the suggested functional association, and unlike the situation in higher organisms, any interaction between the two complexes appears to be of a more transient character.</p>


<h4>Possible targeting of the Swm complex via Hrp1/3?</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Previously, Nicolas et al (2006) <a href="#pone.0000386-Nicolas1">[14]</a> showed that the Swm1/2 complex physically interacts with Hrp1. The putative functional links to the Hrp1 and Hrp3 chromatin remodelling factors that we identify here suggest that this interaction may provide a mechanism for targeting of the Swm demethylase. The double chromo domains of human CHD1, but not those of <em>S. cerevisiae</em> Chd1p, have been shown to bind directly to methylated H3K4 <a href="#pone.0000386-Flanagan1">[24]</a>–<a href="#pone.0000386-Sims1">[25]</a>. Although the <em>S. pombe</em> Hrp1, does not have all the consensus residues required for methyl-lysine binding, Hrp3 does (data not shown). Because Hrp1 and Hrp3 co-purify we speculate that Hrp1/Swm complex interactions might target some Swm complexes to perform demethylation through the interaction with Hrp3.</p>


<h4>Concluding remarks</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>In conclusion, the data presented here demonstrate that in fission yeast the SWIRM/PAO domain proteins Swm1 and Swm2 are associated in complexes that can remove methyl groups from lysine 9 methylated histone H3. Interestingly, recognition of the H3K9 substrate appears to require an as yet unidentified modification. <em>In vivo</em>, loss of Swm1 increases the global levels of H3K9me2 and H3K4me2, and it results in a significant accumulation of H3K4me2 at genes that are up-regulated in the <em>swm1</em>Δ strain. The bias towards increased H3K4me2 levels in ORF regions of genes that are up-regulated (as compared to IGR regions), suggests that this increase results from either increased methylation by Set1, or incorporation of H3K4 methylated histones, during transcription.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>An alternative explanation for the increased levels of H3K4me2 is that Hrp1/3 might act as coregulators, and influence the specificity of the Swm1 complex. In a similar manner to that of human LSD1, which when complexed with the androgen receptor switches from a H3K4 to a H3K9 demethylase <a href="#pone.0000386-Metzger1">[13]</a>, it is possible that K4 demethylation is favored over K9 in the functional context of the chromatin remodelling factors Hrpl and Hrp3. However, the bias to increased levels of H3K4me2 in ORF regions, suggests (as discussed above) that the increased levels at these genes results in some way from increased transcription.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>Aside from at a few genes, where in the <em>swm1</em> deletion increased H3K9me2 levels are correlated with reduced levels of gene expression, the functional role of the Swm1/2 H3K9 demethylase activity is not yet clear. Our results, however, highlight complex interactions between histone demethylase, histone deacetylase and chromatin remodelling activities in the regulation of gene expression. The <em>in vivo</em> data indicate that Swm1 acts in concert with the HDAC Clr6 and the chromatin remodeller Hrp1 to repress gene expression, but further work is necessary to uncover the nature of these functional interactions.</p>
<a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4"></a><p>Supplementary information is available on line.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Reagents</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>IgG Sepharose was from Amersham, Ni-magnabeads were from Promega and the Dynabeads M280 were from Dynal Biotech. Rabbit anti-mouse immunoglobulin protein used for coating the beads was from DAKO, A/S Denmark. Calf thymus bulk histones and histone H3 were purchased from Roche. Chicken polynucleosomes were purchased from Abcam and recombinant Xenopus laevis H3 was from Upstate.</p>


<h4>Construction of strains</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Strains expressing TAP-tagged Swm1 and Swm2, and the <em>swm1</em> knockout strain, were constructed using previously described protocols <a href="#pone.0000386-Bahler1">[26]</a>–<a href="#pone.0000386-Werler1">[28]</a>. In brief, the TAP-tagging constructs for homologous recombination were made using fusion PCR of 500 base pair genomic DNA fragments and the C-terminal TAP-tagging cassette. The resulting DNA fragment was transfected into S. pombe strain 501 <em>(leu1-32; ura4-D18; ade6-704;h<sup>−</sup>).</em> Homologous recombination was confirmed by PCR with primers inside and outside the incorporated TAP-tag. Expression of the construct and fusion protein solubility was checked by Western blots with anti-protein A antibodies. Strain viability was compared to that of the wild type strain in normal growth and under stress conditions (37°C). The <em>swm1</em>Δ knockout strain was constructed by homologous integration of Clonat, replacing the ORF. The integration was confirmed by PCR using primers 5′ and 3′ to the recombined region together with primers internal to the Clonat gene.</p>


<h4>Cells growth and affinity purification of the complexes</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The published TAP-purification procedure <a href="#pone.0000386-Rigaut1">[29]</a> was followed with minor changes. The wild type and TAP-tagged strain cells were harvested and washed in ice-cold water, resuspended in 20 mM HEPES pH 8.0, 150 mM NaCl, 0.1% Tween 20 and broken in liquid nitrogen using a SPEX CertiPrep 6850 Freezer Mill. The soluble fraction was bound to either IgG coated Sepharose or IgG coated Dynabeads, washed extensively, cut with His-tagged TEV protease and eluted from the beads. Excess IgG and TEV were removed by incubation with protein A and Ni-agarose beads. The eluate was bound to calmodulin-binding beads and eluted with EGTA containing buffer. To avoid loss of protein the complex was not further purified by SDS-PAGE. Instead, the eluted complex was directly digested with trypsin and the resulting peptides identified by mass- spectrometry.</p>


<h4>Purification of hLSD1</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Human LSD1 (residues 72-852) was subcloned into the pET30 vector (Novagen). Protein expression was carried out in the <em>E. coli</em> Rossetta 2 strain (Novagen). To induce expression 0.5 mM IPTG was added to cultures at OD<sub>600</sub> of 0.6 and incubated at 25°C for 4 hours. Harvested cells were disrupted in lysis buffer (40 mM Tris pH 8.0, NaCl 300 mM, 0.5% NP40, protease inhibitors (Sigma)) using an Emulsiflex-05 (Avestin) at up to 8000 psi. hLSD1 was purified from the clarified lysate with Ni-NTA agarose (Qiagen) and further purified by anion exchange chromatography (MonoQ column (Pharmacia Biotech)) and gel filtration (Superdex 200 (Pharmacia Biotech)) using standard protocols.</p>


<h4>Purification of GST-Swm1</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Full length GST-Swm1 was subcloned into pGEX-2T. Protein expression was carried out in the <em>E.coli</em> Rossetta strain. To induce expression 0.5 mM IPTG was added to cultures at an OD<sub>600</sub> of 0.5 and incubated at 15°C for 16 hours. Harvested cells were disrupted in lysis buffer (25 mM Tris-HCL pH 8.0, 150 mM NaCl, 0.1% Triton-X-100, protease inhibitors (Roche)) by sonication. Glutathione sepharose resin (Pharmacia) was added to the clarified lysate and incubated with rotation for 2 hrs at 4°C. The resin was then extensively washed with lysis buffer without Triton-X-100. Bounds proteins were finally eluted in 50 mM Tris pH 8.0, 10 mM glutathione.</p>


<h4>In-vitro demethylase assays</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p><sup>3</sup>H-labelled methyl histone substrates were prepared using the following histone methyltransferases (HMT): GST-Set7, MBP-Clr4 and GST-Set2 as previously described <a href="#pone.0000386-Schneider1">[15]</a>. Briefly, 100 ul labelling reactions were carried out in methylase buffer (50 mM Tris pH 8.0, 1 mM DTT, 10% glycerol and 10 µM ZnCl<sub>2</sub>) containing 1–5 ug HMTs, 5 µl S-adenosyl-[<sup>3</sup>H]methyl-methionine (70 Ci/mmol, NEN) and substrate (50 µg calf thymus bulk histones, 10 µg chicken poly-nucleosomes, 10 µg calf thymus histone H3 or 10 µg recombinant Xenopus laevis H3). After a 5 minute incubation at 37°C, reactions were immediately dialysed into demethylase buffer (50 mM Tris pH 8.8, 0.5 mM DTT and 5% glycerol) at 4°C.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>For demethylase assays labelled histone substrates containing 5,000–50,000 cpm were incubated with 25 ul of either TAP-tagged Swm complex, mock purifications from wild-type cells, or 5 ug hLSD1 or GST-Swm1 in a final volume of 100 ul with demethylase buffer at 37°C for 1 hr. The Nash method was then used to detect the formation of <sup>3</sup>H-labelled formaldehyde <a href="#pone.0000386-Kleeberg1">[30]</a>. After precipitation with 10% TCA an equal volume of Nash reagent (3.89 M ammonium acetate, 0.1 M acetic acid and 0.2% 2,4-pentanedione) was added to the supernatant and incubated at 37°C for 50 minutes, followed by extraction with an equal volume of 1-pentanol. The extracted supernatant was then subjected to scintillation counting.</p>


<h4>Genome wide analysis</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>The microarray analysis in this study was performed essentially as outlined in <a href="#pone.0000386-Wiren1">[18]</a>. cDNA expression profiling was carried out according to Xue et al., (2004) <a href="#pone.0000386-Xue1">[31]</a>. We used <em>S. pombe</em> ORF and combined IGR+ORF spotted microarrays from Eurogentec custom DNA microarray services (Belgium). For histone methylation maps, ChIP-chip experiments were carried according to Robyr and Grunstein (2003) <a href="#pone.0000386-Robyr1">[32]</a>. Antibodies recognising specific methylation marks were employed: H3K9me2 and H3K4me2 (a kind gift from Prof. David Allis). For Hrp1 and Hrp3 binding studies we used the ChIP-chip procedure described by Kurdistani et al. (2002) <a href="#pone.0000386-Kurdistani1">[33]</a>. For expression profiling and analysis of histone methylation levels, two microarray experiments were performed using two independent ChIP samples and including Cy3/Cy5 dye swops. Because each Eurogentec microarray yields two data points we thus measured four data points for each experiment. Expression profiling and histone methylation in mutant <em>vs</em>. wild-type data sets were normalized using the GeneSpring software and Lowess (per spot, per chip) intensity-dependent normalization, which corrects for nonlinear rates of dye incorporation. Cut off values of 1.5 (for gene expression) and 2.0 (for histone methylation levels) in at least 3 out of 4 data points were used to generate the ‘high’ and ‘low’ gene lists. For Hrp1 and Hrp3-binding experiments three microarrays were used, yielding six data points. Here we first used the GeneSpring software for a ‘per chip’ normalization and then employed a statistically determined cut-off for binding according to the median percentile ranking method <a href="#pone.0000386-Buck1">[34]</a>. Similar gene lists were identified using the automatic hyper-geometric distribution tests in the Gene List inspector function of Gene Spring. The hyper-geometric distribution test calculates the probability of overlap corresponding to <em>k</em> or more IGR or ORF fragments between an IGR or ORF list of <em>n</em> fragments compared against another gene list of <em>m</em> fragments when randomly sampled from a universe of <em>u</em> genes:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000386.e001&amp;representation=PNG" class="inline-graphic"></span><br>The significantly overlapping gene lists were illustrated using Venn diagrams.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000386"></div><a name="pone.0000386.s001" id="pone.0000386.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000386.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">a) Identification of peptides in the Swm complex purified using TAP-tagged Swm1 (SPBC146.09C) b) Identification of peptides in the Swm complex purified using TAP-tagged Swm2 (SPAC23E2.02)</p>
<p class="siDoi">doi:10.1371/journal.pone.0000386.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.16 MB DOC)</p>
<a name="pone.0000386.s002" id="pone.0000386.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000386.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">A list of genes either down- or up-regulated in swm1 deletion cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0000386.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.17 MB DOC)</p>
<a name="pone.0000386.s003" id="pone.0000386.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000386.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Genes that have either increased levels of H3K4me2 or H3K9me2 in swm1 deletion cells</p>
<p class="siDoi">doi:10.1371/journal.pone.0000386.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.31 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Prof. David Allis for providing the antibody against H3K4me2, Indranil Sinha and Mickaël Durand-Dubief for help with microarray experiments, and Sarah Maslen for the peptide mass spectrometry analysis.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p>The microarray data has been deposited in the GEO database (accession number GSE6224).</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RA EL KE DL. Performed the experiments: KE MO DL CB. Analyzed the data: EL KE MO DL CB. Wrote the paper: RA EL KE DL. Other: Contributed to the experimental work and analysis of data: TK JW ST AB JC NM. Contributed to the writing of the paper: NM. Provided assistance with the TAP-tagging experiments: AC PW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000386-Martin1" id="pone.0000386-Martin1"></a>Martin C, Zhang Y (2005) The diverse functions of histone lysine methylation. Nat Rev Mol Cell Biol  6: 838–849.  <ul class="find" data-citedArticleID="945834" data-doi="10.1038/nrm1761"><li><a href="http://dx.doi.org/10.1038/nrm1761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+diverse+functions+of+histone+lysine+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+diverse+functions+of+histone+lysine+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000386-Shi1" id="pone.0000386-Shi1"></a>Shi Y, Lan F, Matson C, Mulligan P, Whetstine JR, et al.  (2004) Histone demethylation mediated by the nuclear amine oxidase homolog LSD1. Cell  119: 941–953.  <ul class="find" data-citedArticleID="945852" data-doi="10.1016/j.cell.2004.12.012"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.12.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000386-Cloos1" id="pone.0000386-Cloos1"></a>Cloos PA, Christensen J, Agger K, Maiolica A, Rappsilber J, et al.  (2006) The putative oncogene GASC1 demethylates tri- and dimethylated lysine 9 on histone H3. Nature  442: 307–311.  <ul class="find" data-citedArticleID="945812" data-doi="10.1038/nature04837"><li><a href="http://dx.doi.org/10.1038/nature04837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+putative+oncogene+GASC1+demethylates+tri-+and+dimethylated+lysine+9+on+histone+H3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+putative+oncogene+GASC1+demethylates+tri-+and+dimethylated+lysine+9+on+histone+H3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000386-Klose1" id="pone.0000386-Klose1"></a>Klose RJ, Yamane K, Bae Y, Zhang D, Erdjument-Bromage H, et al.  (2006) The transcriptional repressor JHDM3A demethylates trimethyl histone H3 lysine 9 and lysine 36. Nature  442: 312–316.  <ul class="find" data-citedArticleID="945826" data-doi="10.1038/nature04853"><li><a href="http://dx.doi.org/10.1038/nature04853" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+repressor+JHDM3A+demethylates+trimethyl+histone+H3+lysine+9+and+lysine+36." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+repressor+JHDM3A+demethylates+trimethyl+histone+H3+lysine+9+and+lysine+36.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000386-Tsukada1" id="pone.0000386-Tsukada1"></a>Tsukada Y, Fang J, Erdjument-Bromage H, Warren ME, Borchers CH, et al.  (2006) Histone demethylation by a family of JmjC domain-containing proteins. Nature  439: 811–816.  <ul class="find" data-citedArticleID="945864" data-doi="10.1038/nature04433"><li><a href="http://dx.doi.org/10.1038/nature04433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+demethylation+by+a+family+of+JmjC+domain-containing+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+demethylation+by+a+family+of+JmjC+domain-containing+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000386-Yamane1" id="pone.0000386-Yamane1"></a>Yamane K, Toumazou C, Tsukada Y, Erdjument-Bromage H, Tempst P, et al.  (2006) JHDM2A, a JmjC-containing H3K9 demethylase, facilitates transcription activation by androgen receptor. Cell  125: 483–495.  <ul class="find" data-citedArticleID="945872" data-doi="10.1016/j.cell.2006.03.027"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.03.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=JHDM2A%2C+a+JmjC-containing+H3K9+demethylase%2C+facilitates+transcription+activation+by+androgen+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22JHDM2A%2C+a+JmjC-containing+H3K9+demethylase%2C+facilitates+transcription+activation+by+androgen+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000386-Humphrey1" id="pone.0000386-Humphrey1"></a>Humphrey GW, Wang Y, Russanova VR, Hirai T, Qin J, et al.  (2001) Stable histone deacetylase complexes distinguished by the presence of SANT domain proteins CoREST/kiaa0071 and Mta-L1. J Biol Chem  276: 6817–6824.  <ul class="find" data-citedArticleID="945822" data-doi="10.1074/jbc.m007372200"><li><a href="http://dx.doi.org/10.1074/jbc.m007372200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stable+histone+deacetylase+complexes+distinguished+by+the+presence+of+SANT+domain+proteins+CoREST%2Fkiaa0071+and+Mta-L1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stable+histone+deacetylase+complexes+distinguished+by+the+presence+of+SANT+domain+proteins+CoREST%2Fkiaa0071+and+Mta-L1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000386-Hakimi1" id="pone.0000386-Hakimi1"></a>Hakimi MA, Bochar DA, Chenoweth J, Lane WS, Mandel G, et al.  (2002) A core-BRAF35 complex containing histone deacetylase mediates repression of neuronal-specific genes. Proc Natl Acad Sci U S A  99: 7420–7425.  <ul class="find" data-citedArticleID="945818" data-doi="10.1073/pnas.112008599"><li><a href="http://dx.doi.org/10.1073/pnas.112008599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+core-BRAF35+complex+containing+histone+deacetylase+mediates+repression+of+neuronal-specific+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+core-BRAF35+complex+containing+histone+deacetylase+mediates+repression+of+neuronal-specific+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000386-Shi2" id="pone.0000386-Shi2"></a>Shi Y, Sawada J, Sui G, Affar el B, Whetstine JR, et al.  (2003) Coordinated histone modifications mediated by a CtBP co-repressor complex. Nature  422: 735–738.  <ul class="find" data-citedArticleID="945854" data-doi="10.1038/nature01550"><li><a href="http://dx.doi.org/10.1038/nature01550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coordinated+histone+modifications+mediated+by+a+CtBP+co-repressor+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coordinated+histone+modifications+mediated+by+a+CtBP+co-repressor+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000386-Lee1" id="pone.0000386-Lee1"></a>Lee MG, Wynder C, Bochar DA, Hakimi MA, Cooch N, et al.  (2006) Functional interplay between histone demethylase and deacetylase enzymes. Mol Cell Biol  26: 6395–6402.  <ul class="find" data-citedArticleID="945830" data-doi="10.1128/mcb.00723-06"><li><a href="http://dx.doi.org/10.1128/mcb.00723-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+interplay+between+histone+demethylase+and+deacetylase+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+interplay+between+histone+demethylase+and+deacetylase+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000386-Lee2" id="pone.0000386-Lee2"></a>Lee MG, Wynder C, Cooch N, Shiekhattar R (2005) An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation. Nature  437: 432–435.  <ul class="find" data-citedArticleID="945832" data-doi="10.1038/nature04021"><li><a href="http://dx.doi.org/10.1038/nature04021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000386-Shi3" id="pone.0000386-Shi3"></a>Shi YJ, Matson C, Lan F, Iwase S, Baba T, et al.  (2005) Regulation of LSD1 histone demethylase activity by its associated factors. Mol Cell  19: 857–864.  <ul class="find" data-citedArticleID="945856" data-doi="10.1016/j.molcel.2005.08.027"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.08.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+LSD1+histone+demethylase+activity+by+its+associated+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+LSD1+histone+demethylase+activity+by+its+associated+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000386-Metzger1" id="pone.0000386-Metzger1"></a>Metzger E, Wissmann M, Yin N, Muller JM, Schneider R, et al.  (2005) LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription. Nature  437: 436–439.  <ul class="find" data-citedArticleID="945838" data-doi="10.1038/nature04020"><li><a href="http://dx.doi.org/10.1038/nature04020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000386-Nicolas1" id="pone.0000386-Nicolas1"></a>Nicolas E, Lee MG, Hakimi MA, Cam H, Grewal SS, et al.  (2006) Fission yeast homologs of human histone H3 lysine 4 demethylase regulate a common set of genes with diverse functions. J Biol Chem  47: 35983–8.  <ul class="find" data-citedArticleID="945842" data-doi="10.1074/jbc.m606349200"><li><a href="http://dx.doi.org/10.1074/jbc.m606349200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fission+yeast+homologs+of+human+histone+H3+lysine+4+demethylase+regulate+a+common+set+of+genes+with+diverse+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fission+yeast+homologs+of+human+histone+H3+lysine+4+demethylase+regulate+a+common+set+of+genes+with+diverse+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000386-Schneider1" id="pone.0000386-Schneider1"></a>Schneider R, Bannister AJ (2002) Protein N-methyltransferase assays in the study of gene transcription. Methods  26: 226–232.  <ul class="find" data-citedArticleID="945850" data-doi="10.1016/s1046-2023(02)00026-9"><li><a href="http://dx.doi.org/10.1016/s1046-2023(02)00026-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+N-methyltransferase+assays+in+the+study+of+gene+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+N-methyltransferase+assays+in+the+study+of+gene+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000386-Bjerling1" id="pone.0000386-Bjerling1"></a>Bjerling P, Silverstein RA, Thon G, Caudy A, Grewal S, et al.  (2002) Functional divergence between histone deacetylases in fission yeast by distinct cellular localization and in vivo specificity. Mol Cell Biol  22: 2170–2181.  <ul class="find" data-citedArticleID="945808" data-doi="10.1128/mcb.22.7.2170-2181.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.7.2170-2181.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+divergence+between+histone+deacetylases+in+fission+yeast+by+distinct+cellular+localization+and+in+vivo+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+divergence+between+histone+deacetylases+in+fission+yeast+by+distinct+cellular+localization+and+in+vivo+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000386-Grewal1" id="pone.0000386-Grewal1"></a>Grewal SI, Bonaduce MJ, Klar AJ (1998) Histone deacetylase homologs regulate epigenetic inheritance of transcriptional silencing and chromosome segregation in fission yeast. Genetics  150: 563–576.  <ul class="find" data-citedArticleID="945816"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Histone+deacetylase+homologs+regulate+epigenetic+inheritance+of+transcriptional+silencing+and+chromosome+segregation+in+fission+yeast.&amp;auth=&amp;atitle=Histone+deacetylase+homologs+regulate+epigenetic+inheritance+of+transcriptional+silencing+and+chromosome+segregation+in+fission+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+deacetylase+homologs+regulate+epigenetic+inheritance+of+transcriptional+silencing+and+chromosome+segregation+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+deacetylase+homologs+regulate+epigenetic+inheritance+of+transcriptional+silencing+and+chromosome+segregation+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000386-Wiren1" id="pone.0000386-Wiren1"></a>Wiren M, Silverstein RA, Sinha I, Walfridsson J, Lee HM, et al.  (2005) Genomewide analysis of nucleosome density histone acetylation and HDAC function in fission yeast. EMBO J  24: 2906–2918.  <ul class="find" data-citedArticleID="945868" data-doi="10.1038/sj.emboj.7600758"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600758" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomewide+analysis+of+nucleosome+density+histone+acetylation+and+HDAC+function+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomewide+analysis+of+nucleosome+density+histone+acetylation+and+HDAC+function+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000386-Noma1" id="pone.0000386-Noma1"></a>Noma K, Allis CD, Grewal SI (2001) Transitions in distinct histone H3 methylation patterns at the heterochromatin domain boundaries. Science  293: 1150–1155.  <ul class="find" data-citedArticleID="945844" data-doi="10.1126/science.1064150"><li><a href="http://dx.doi.org/10.1126/science.1064150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transitions+in+distinct+histone+H3+methylation+patterns+at+the+heterochromatin+domain+boundaries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transitions+in+distinct+histone+H3+methylation+patterns+at+the+heterochromatin+domain+boundaries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000386-Hall1" id="pone.0000386-Hall1"></a>Hall IM, Shankaranarayana GD, Noma K, Ayoub N, Cohen A, et al.  (2002) Establishment and maintenance of a heterochromatin domain. Science  297: 2232–2237.  <ul class="find" data-citedArticleID="945820" data-doi="10.1126/science.1076466"><li><a href="http://dx.doi.org/10.1126/science.1076466" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+and+maintenance+of+a+heterochromatin+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+and+maintenance+of+a+heterochromatin+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000386-Mellone1" id="pone.0000386-Mellone1"></a>Mellone BG, Ball L, Suka N, Grunstein MR, Partridge JF, et al.  (2003) Centromere silencing and function in fission yeast is governed by the amino terminus of histone H3. Curr Biol  13: 1748–1757.  <ul class="find" data-citedArticleID="945836" data-doi="10.1016/j.cub.2003.09.031"><li><a href="http://dx.doi.org/10.1016/j.cub.2003.09.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Centromere+silencing+and+function+in+fission+yeast+is+governed+by+the+amino+terminus+of+histone+H3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Centromere+silencing+and+function+in+fission+yeast+is+governed+by+the+amino+terminus+of+histone+H3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000386-Sinha1" id="pone.0000386-Sinha1"></a>Sinha I, Wiren I, Ekwall K (2006) Genome-wide patterns of histone modifications in fission yeast. Chromatin Res  14: 95–105.  <ul class="find" data-citedArticleID="945860" data-doi="10.1007/s10577-005-1023-4"><li><a href="http://dx.doi.org/10.1007/s10577-005-1023-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+patterns+of+histone+modifications+in+fission+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+patterns+of+histone+modifications+in+fission+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000386-Nakayama1" id="pone.0000386-Nakayama1"></a>Nakayama J, Xiao G, Noma K, Malikzay A, Bjerling P, et al.  (2003) Alp13, an MRG family protein, is a component of fission yeast Clr6 histone deacetylase required for genomic integrity. Embo J  22: 2776–2787.  <ul class="find" data-citedArticleID="945840" data-doi="10.1093/emboj/cdg248"><li><a href="http://dx.doi.org/10.1093/emboj/cdg248" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alp13%2C+an+MRG+family+protein%2C+is+a+component+of+fission+yeast+Clr6+histone+deacetylase+required+for+genomic+integrity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alp13%2C+an+MRG+family+protein%2C+is+a+component+of+fission+yeast+Clr6+histone+deacetylase+required+for+genomic+integrity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000386-Flanagan1" id="pone.0000386-Flanagan1"></a>Flanagan JF, Mi LZ, Chruszcz M, Cymborowski M, Clines KL, et al.  (2005) Double chromodomains cooperate to recognize the methylated histone H3 tail. Nature  438: 1181–1185.  <ul class="find" data-citedArticleID="945814" data-doi="10.1038/nature04290"><li><a href="http://dx.doi.org/10.1038/nature04290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Double+chromodomains+cooperate+to+recognize+the+methylated+histone+H3+tail." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Double+chromodomains+cooperate+to+recognize+the+methylated+histone+H3+tail.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000386-Sims1" id="pone.0000386-Sims1"></a>Sims RJ 3rd, Chen CF, Santos-Rosa H, Kouzarides T, Patel SS, et al.  (2005) Human but not yeast CHD1 binds directly and selectively to histone H3 methylated at lysine 4 via its tandem chromodomains. J Biol Chem  280: 41789–92.  <ul class="find" data-citedArticleID="945858" data-doi="10.1074/jbc.c500395200"><li><a href="http://dx.doi.org/10.1074/jbc.c500395200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+but+not+yeast+CHD1+binds+directly+and+selectively+to+histone+H3+methylated+at+lysine+4+via+its+tandem+chromodomains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+but+not+yeast+CHD1+binds+directly+and+selectively+to+histone+H3+methylated+at+lysine+4+via+its+tandem+chromodomains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000386-Bahler1" id="pone.0000386-Bahler1"></a>Bahler J, Wu JQ, Longtine MS, Shah NG, McKenzie A 3rd, et al.  (1998) Heterologous modules for efficient and versatile PCR-based gene targeting in Schizosaccharomyces pombe. Yeast  14: 943–951.  <ul class="find" data-citedArticleID="945806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Heterologous+modules+for+efficient+and+versatile+PCR-based+gene+targeting+in+Schizosaccharomyces+pombe.&amp;auth=&amp;atitle=Heterologous+modules+for+efficient+and+versatile+PCR-based+gene+targeting+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterologous+modules+for+efficient+and+versatile+PCR-based+gene+targeting+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterologous+modules+for+efficient+and+versatile+PCR-based+gene+targeting+in+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000386-Tasto1" id="pone.0000386-Tasto1"></a>Tasto JJ, Carnahan RH, McDonald WH, Gould KL (2001) Vectors and gene targeting modules for tandem affinity purification in Schizosaccharomyces pombe. Yeast  18: 657–662.  <ul class="find" data-citedArticleID="945862"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Vectors+and+gene+targeting+modules+for+tandem+affinity+purification+in+Schizosaccharomyces+pombe.&amp;auth=&amp;atitle=Vectors+and+gene+targeting+modules+for+tandem+affinity+purification+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vectors+and+gene+targeting+modules+for+tandem+affinity+purification+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vectors+and+gene+targeting+modules+for+tandem+affinity+purification+in+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000386-Werler1" id="pone.0000386-Werler1"></a>Werler PJ, Hartsuiker E, Carr AM (2003) A simple Cre-loxP method for chromosomal N-terminal tagging of essential and non-essential Schizosaccharomyces pombe genes. Gene  304: 133–141.  <ul class="find" data-citedArticleID="945866" data-doi="10.1016/s0378-1119(03)00402-5"><li><a href="http://dx.doi.org/10.1016/s0378-1119(03)00402-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+Cre-loxP+method+for+chromosomal+N-terminal+tagging+of+essential+and+non-essential+Schizosaccharomyces+pombe+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+Cre-loxP+method+for+chromosomal+N-terminal+tagging+of+essential+and+non-essential+Schizosaccharomyces+pombe+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000386-Rigaut1" id="pone.0000386-Rigaut1"></a>Rigaut G, Shevchenko A, Rutz B, Wilm M, Mann M, et al.  (1999) A generic protein purification method for protein complex characterization and proteome exploration. Nat Biotechnol  17: 1030–1032.  <ul class="find" data-citedArticleID="945846" data-doi="10.1038/13732"><li><a href="http://dx.doi.org/10.1038/13732" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+generic+protein+purification+method+for+protein+complex+characterization+and+proteome+exploration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+generic+protein+purification+method+for+protein+complex+characterization+and+proteome+exploration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000386-Kleeberg1" id="pone.0000386-Kleeberg1"></a>Kleeberg U, Klinger W (1982) Sensitive formaldehyde determination with Nash's reagent and a ‘tryptophan reaction’. J Pharmacol Methods  8: 19–31.  <ul class="find" data-citedArticleID="945824" data-doi="10.1016/0160-5402(82)90004-3"><li><a href="http://dx.doi.org/10.1016/0160-5402(82)90004-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitive+formaldehyde+determination+with+Nash%27s+reagent+and+a+%26%23x2018%3Btryptophan+reaction%26%23x2019%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitive+formaldehyde+determination+with+Nash%27s+reagent+and+a+%26%23x2018%3Btryptophan+reaction%26%23x2019%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000386-Xue1" id="pone.0000386-Xue1"></a>Xue Y, Haas SA, Brino L, Gusnanto A, Reimers M, et al.  (2004) A DNA microarray for fission yeast: minimal changes in global gene expression after temperature shift. Yeast  21: 25–39.  <ul class="find" data-citedArticleID="945870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+DNA+microarray+for+fission+yeast%3A+minimal+changes+in+global+gene+expression+after+temperature+shift.&amp;auth=&amp;atitle=A+DNA+microarray+for+fission+yeast%3A+minimal+changes+in+global+gene+expression+after+temperature+shift." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+DNA+microarray+for+fission+yeast%3A+minimal+changes+in+global+gene+expression+after+temperature+shift." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+DNA+microarray+for+fission+yeast%3A+minimal+changes+in+global+gene+expression+after+temperature+shift.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000386-Robyr1" id="pone.0000386-Robyr1"></a>Robyr D, Grunstein M (2003) Genomewide histone acetylation microarrays. Methods  31: 83–89.  <ul class="find" data-citedArticleID="945848" data-doi="10.1016/s1046-2023(03)00091-4"><li><a href="http://dx.doi.org/10.1016/s1046-2023(03)00091-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomewide+histone+acetylation+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomewide+histone+acetylation+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000386-Kurdistani1" id="pone.0000386-Kurdistani1"></a>Kurdistani SK, Robyr D, Tavazoie S, Grunstein M (2002) Genome-wide binding map of the histone deacetylase Rpd3 in yeast. Nat Genet  31: 248–254.  <ul class="find" data-citedArticleID="945828" data-doi="10.1038/ng907"><li><a href="http://dx.doi.org/10.1038/ng907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+binding+map+of+the+histone+deacetylase+Rpd3+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+binding+map+of+the+histone+deacetylase+Rpd3+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000386-Buck1" id="pone.0000386-Buck1"></a>Buck MJ, Lieb JD (2004) ChIP-chip: considerations for the design, analysis, and application of genome-wide chromatin immunoprecipitation experiments. Genomics  83: 349–360.  <ul class="find" data-citedArticleID="945810" data-doi="10.1016/j.ygeno.2003.11.004"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2003.11.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ChIP-chip%3A+considerations+for+the+design%2C+analysis%2C+and+application+of+genome-wide+chromatin+immunoprecipitation+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ChIP-chip%3A+considerations+for+the+design%2C+analysis%2C+and+application+of+genome-wide+chromatin+immunoprecipitation+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.XML" value="91272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.PDF" value="352225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g001.PNG_L" value="37527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g001.PNG_M" value="49793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g001.PNG_S" value="2672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g001.TIF" value="78060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g001.PNG_I" value="20105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g002.PNG_L" value="304768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g002.PNG_M" value="58260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g002.PNG_S" value="2627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g002.TIF" value="831132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g002.PNG_I" value="25285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g003.PNG_L" value="45258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g003.PNG_M" value="11643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g003.PNG_S" value="1626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g003.TIF" value="178970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g003.PNG_I" value="6319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g004.PNG_L" value="957260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g004.PNG_M" value="83662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g004.PNG_S" value="3562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g004.TIF" value="2768260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.g004.PNG_I" value="42310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t001.PNG_L" value="168575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t001.PNG_M" value="106302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t001.PNG_S" value="7837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t001.TIF" value="817024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t001.PNG_I" value="86330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t002.PNG_L" value="292234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t002.PNG_M" value="94587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t002.PNG_S" value="12131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t002.TIF" value="1348336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.t002.PNG_I" value="173012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.e001.PNG" value="5280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.e001.TIF" value="27724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.s001.DOC" value="156160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.s002.DOC" value="166912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000386.s003.DOC" value="307200"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000386&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000386" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000386&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000386&volume=&issue=&title=Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1&author_name=Michael%20Opel%2C%20David%20Lando%2C%20Carolina%20Bonilla%2C%20Sarah%20C.%20Trewick%2C%20Abdelhalim%20Boukaba%2C%20Julian%20Walfridsson%2C%20James%20Cauwood%2C%20Petra%20J.H.%20Werler%2C%20Antony%20M.%20Carr%2C%20Tony%20Kouzarides%2C%20Natalia%20V.%20Murzina%2C%20Robin%20C.%20Allshire%2C%20Karl%20Ekwall%2C%20Ernest%20D.%20Laue&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000386" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386&amp;t=Genome-Wide%20Studies%20of%20Histone%20Demethylation%20Catalysed%20by%20the%20Fission%20Yeast%20Homologues%20of%20Mammalian%20LSD1" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386&title=Genome-Wide%20Studies%20of%20Histone%20Demethylation%20Catalysed%20by%20the%20Fission%20Yeast%20Homologues%20of%20Mammalian%20LSD1&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386&amp;title=Genome-Wide%20Studies%20of%20Histone%20Demethylation%20Catalysed%20by%20the%20Fission%20Yeast%20Homologues%20of%20Mammalian%20LSD1" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000386&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome-Wide Studies of Histone Demethylation Catalysed by the Fission Yeast Homologues of Mammalian LSD1';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000386';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome-Wide%20Studies%20of%20Histone%20Demethylation%20Catalysed%20by%20the%20Fission%20Yeast%20Homologues%20of%20Mammalian%20LSD1 http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000386" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000386" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+binding%22" title="Search for articles in the subject area:'Cell binding'"><div class="flagText">Cell binding</div></a>
              <div data-categoryid="19609" data-articleid="23610"
                   data-categoryname="Cell binding"
                   class="flagImage" title="Flag 'Cell binding' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="23610"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23610"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="23610"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="23610"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="23610"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lysine%22" title="Search for articles in the subject area:'Lysine'"><div class="flagText">Lysine</div></a>
              <div data-categoryid="40335" data-articleid="23610"
                   data-categoryname="Lysine"
                   class="flagImage" title="Flag 'Lysine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Schizosaccharomyces+pombe%22" title="Search for articles in the subject area:'Schizosaccharomyces pombe'"><div class="flagText">Schizosaccharomyces pombe</div></a>
              <div data-categoryid="33499" data-articleid="23610"
                   data-categoryname="Schizosaccharomyces pombe"
                   class="flagImage" title="Flag 'Schizosaccharomyces pombe' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=781'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9827'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6109&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
