

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000904"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000904" />

    <meta name="citation_title" content="Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains"/>
    <meta itemprop="name" content="Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains"/>

      <meta name="citation_author" content="Eriikka Heikkinen"/>
            <meta name="citation_author_institution" content="Pertussis Reference Laboratory, National Public Health Institute, Turku, Finland"/>
      <meta name="citation_author" content="Teemu Kallonen"/>
            <meta name="citation_author_institution" content="Pertussis Reference Laboratory, National Public Health Institute, Turku, Finland"/>
            <meta name="citation_author_institution" content="Turku Graduate School of Biomedical Sciences, University of Turku, Turku, Finland"/>
      <meta name="citation_author" content="Lilli Saarinen"/>
            <meta name="citation_author_institution" content="Finnish DNA Microarray Centre, Turku Centre for Biotechnology, University of Turku, Åbo Akademi University, Turku, Finland"/>
      <meta name="citation_author" content="Rolf Sara"/>
            <meta name="citation_author_institution" content="Finnish DNA Microarray Centre, Turku Centre for Biotechnology, University of Turku, Åbo Akademi University, Turku, Finland"/>
      <meta name="citation_author" content="Audrey J. King"/>
            <meta name="citation_author_institution" content="Laboratory for Infectious Diseases and Screening, National Institute of Public Health and the Environment, Bilthoven, The Netherlands"/>
      <meta name="citation_author" content="Frits R. Mooi"/>
            <meta name="citation_author_institution" content="Laboratory for Infectious Diseases and Screening, National Institute of Public Health and the Environment, Bilthoven, The Netherlands"/>
      <meta name="citation_author" content="Juhani T. Soini"/>
            <meta name="citation_author_institution" content="Finnish DNA Microarray Centre, Turku Centre for Biotechnology, University of Turku, Åbo Akademi University, Turku, Finland"/>
      <meta name="citation_author" content="Jussi Mertsola"/>
            <meta name="citation_author_institution" content="Department of Pediatrics, Turku University Hospital, Turku, Finland"/>
      <meta name="citation_author" content="Qiushui He"/>
            <meta name="citation_author_institution" content="Pertussis Reference Laboratory, National Public Health Institute, Turku, Finland"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000904.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e904"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Comparative analysis of the genome sequences of Bordetella pertussis, Bordetella parapertussis and Bordetella bronchiseptica.; citation_author=J Parkhill; citation_author=M Sebaihia; citation_author=A Preston; citation_author=LD Murphy; citation_author=N Thomson; citation_journal_title=Nat Genet; citation_volume=35; citation_number=1; citation_pages=32-40; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Control of Pertussis–2007 and Beyond.; citation_author=SA Halperin; citation_journal_title=N Engl J Med; citation_volume=356; citation_number=2; citation_pages=110-113; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Resurgence of Pertussis in Europe.; citation_author=LP Celentano; citation_author=M Massari; citation_author=D Paramatti; citation_author=S Salmaso; citation_author=AE Tozzi; citation_journal_title=Pediatr Infect Dis J; citation_volume=24; citation_number=3; citation_pages=761-765; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adaptation of Bordetella pertussis to Vaccination: A Cause for Its Reemergence?; citation_author=FR Mooi; citation_author=IHM van Loo; citation_author=A King; citation_journal_title=Emerg Infect Dis; citation_volume=7; citation_number=4; citation_pages=526-528; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pertussis in Poland.; citation_author=A Gzyl; citation_author=E Augustynowicz; citation_author=D Rabczenko; citation_author=G Gniadek; citation_author=J Slusarczyk; citation_journal_title=Int J Epidemiol; citation_volume=33; citation_number=5; citation_pages=358-365; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Strain Variation among Bordetella pertussis Isolates in Finland, Where the Whole-Cell Pertussis Vaccine Has Been Used for 50 Years.; citation_author=A Elomaa; citation_author=A Advani; citation_author=D Donnelly; citation_author=M Antila; citation_author=J Mertsola; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=6; citation_pages=3681-3687; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pertussis resurgence in Canada largely caused by cohort effect.; citation_author=B Ntezayabo; citation_author=G De Serres; citation_author=B Duval; citation_journal_title=Pediatr Infect Dis J; citation_volume=22; citation_number=7; citation_pages=22-27; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Vaccine preventable diseases and vaccination coverage in Australia, 1999 to 2000.; citation_author=P McIntyre; citation_author=H Gidding; citation_author=R Gilmour; citation_author=G Lawrence; citation_author=B Hull; citation_journal_title=Commun Dis Intell; citation_number=8; citation_pages=1-111; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular epidemiology of Bordetella pertussis isolated in Taiwan, 1992–1997.; citation_author=YS Lee; citation_author=CY Yang; citation_author=CH Lu; citation_author=YH Tseng; citation_journal_title=Microbiol and Immunol; citation_volume=47; citation_number=9; citation_pages=903-912; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The re-emergence of pertussis in Israel.; citation_author=L Moerman; citation_author=A Leventhal; citation_author=P Slater; citation_author=E Anis; citation_author=R Yishai; citation_journal_title=Isr Med Assoc J; citation_volume=8; citation_number=10; citation_pages=308-311; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Epidemiological Typing of Bordetella pertussis Isolates: Recommendations for a Standard Methodology.; citation_author=FR Mooi; citation_author=H Hallander; citation_author=CHWv König; citation_author=B Hoet; citation_author=N Guiso; citation_journal_title=Euro J Clin Microbiol Infect Dis; citation_volume=19; citation_number=11; citation_pages=174-181; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Polymorphism in the Bordetella pertussis Virulence Factors P.69/Pertactin and Pertussis Toxin in The Netherlands: Temporal Trends and Evidence for Vaccine-Driven Evolution.; citation_author=FR Mooi; citation_author=H van Oirschot; citation_author=K Heuvelman; citation_author=HGJ van der Heide; citation_author=W Gaastra; citation_journal_title=Infect Immun; citation_volume=66; citation_number=12; citation_pages=670-675; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Variation in the Bordetella pertussis Virulence Factors Pertussis Toxin and Pertactin in Vaccine Strains and Clinical Isolates in Finland.; citation_author=FR Mooi; citation_author=Q He; citation_author=H van Oirschot; citation_author=J Mertsola; citation_journal_title=Infect Immun; citation_volume=67; citation_number=13; citation_pages=3133-3134; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Polymorphism in Bordetella pertussis Pertactin and Pertussis Toxin Virulence Factors in the United States, 1935–1999.; citation_author=P Cassiday; citation_author=G Sanden; citation_author=K Heuvelman; citation_author=F Mooi; citation_author=KM Bisgard; citation_journal_title=Journal Infect Dis; citation_volume=182; citation_number=14; citation_pages=1402-1408; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Polymorphisms of the Fimbria fim3 Gene of Bordetella pertussis Strains Isolated in Canada.; citation_author=RSW Tsang; citation_author=AKH Lau; citation_author=ML Sill; citation_author=SA Halperin; citation_author=P Van Caeseele; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=15; citation_pages=5364-5367; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Polymorphism of Bordetella pertussis Isolates Circulating for the Last 10 Years in France, Where a Single Effective Whole-Cell Vaccine Has Been Used for More than 30 Years.; citation_author=C Weber; citation_author=C Boursaux-Eude; citation_author=G Coralie; citation_author=V Caro; citation_author=N Guiso; citation_journal_title=J Clin Microbiol; citation_volume=39; citation_number=16; citation_pages=4396-4403; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Shifts of Bordetella pertussis Variants in Sweden from 1970 to 2003, during Three Periods Marked by Different Vaccination Programs.; citation_author=HO Hallander; citation_author=A Advani; citation_author=D Donnelly; citation_author=L Gustafsson; citation_author=R-M Carlsson; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=17; citation_pages=2856-2865; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pulsed-field gel electrophoresis analysis of Bordetella pertussis populations in various European countries with different vaccine policies.; citation_author=V Caro; citation_author=E Njamkepo; citation_author=SCM Van Amersfoorth; citation_author=FR Mooi; citation_author=A Advani; citation_journal_title=Microbes Infect; citation_volume=7; citation_number=18; citation_pages=976-982; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple-Locus Variable-Number Tandem Repeat Analysis of Dutch Bordetella pertussis Strains Reveals Rapid Genetic Changes with Clonal Expansion during the Late 1990s.; citation_author=LM Schouls; citation_author=HGJvd Heide; citation_author=L Vauterin; citation_author=P Vauterin; citation_author=FR Mooi; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=19; citation_pages=5496-5505; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of Bordetella pertussis Populations in European Countries with Different Vaccination Policies.; citation_author=SCM van Amersfoorth; citation_author=LM Schouls; citation_author=HGJ van der Heide; citation_author=A Advani; citation_author=HO Hallander; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=20; citation_pages=2837-2843; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Population dynamics of Bordetella pertussis in Finland and Sweden, neighbouring countries with different vaccination histories.; citation_author=A Elomaa; citation_author=A Advani; citation_author=D Donnelly; citation_author=M Antila; citation_author=J Mertsola; citation_journal_title=Vaccine; citation_volume=25; citation_number=21; citation_pages=918-926; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparison of the Genome Sequence of the Poultry Pathogen Bordetella avium with Those of B. bronchiseptica, B. pertussis, and B. parapertussis Reveals Extensive Diversity in Surface Structures Associated with Host Interaction.; citation_author=M Sebaihia; citation_author=A Preston; citation_author=DJ Maskell; citation_author=H Kuzmiak; citation_author=TD Connell; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=22; citation_pages=6002-6015; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=ATP-Binding Cassette Transporters Are Targets for the Development of Antibacterial Vaccines and Therapies.; citation_author=HS Garmory; citation_author=RW Titball; citation_journal_title=Infect Immun; citation_volume=72; citation_number=23; citation_pages=6757-6763; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Significant Gene Order and Expression Differences in Bordetella pertussis Despite Limited Gene Content Variation.; citation_author=MM Brinig; citation_author=CA Cummings; citation_author=GN Sanden; citation_author=P Stefanelli; citation_author=A Lawrence; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=24; citation_pages=2375-2382; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Temporal analysis of French Bordetella pertussis isolates by comparative whole-genome hybridization.; citation_author=V Caro; citation_author=D Hot; citation_author=G Guigon; citation_author=C Hubans; citation_author=M Arrive; citation_journal_title=Microbes Infect; citation_volume=8; citation_number=25; citation_pages=2228-2235; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Yersinia pestis, the causative agent of plague.; citation_author=J Parkhill; citation_author=BW Wren; citation_author=NR Thomson; citation_author=RW Titball; citation_author=MTG Holden; citation_journal_title=Nature; citation_volume=413; citation_number=26; citation_pages=523-527; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Complete Genome Sequence of Yersinia pestis Strains Antiqua and Nepal516: Evidence of Gene Reduction in an Emerging Pathogen.; citation_author=PSG Chain; citation_author=P Hu; citation_author=SA Malfatti; citation_author=L Radnedge; citation_author=F Larimer; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=27; citation_pages=4453-4463; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pseudogenes, Are They “Junk” or Functional DNA?; citation_author=ES Balakirev; citation_author=FJ Ayala; citation_journal_title=Annu Rev of Genet; citation_volume=37; citation_number=28; citation_pages=123-151; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reductive Genome Evolution from the Mother of Rickettsia.; citation_author=G Blanc; citation_author=H Ogata; citation_author=C Robert; citation_author=S Audic; citation_author= phane; citation_journal_title=PLoS Genetics; citation_volume=3; citation_number=29; citation_pages=e14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Insertion Sequence -Driven Diversification Creates a Globally Dispersed Emerging Multiresistant Subspecies of E. faecium.; citation_author=HL Leavis; citation_author=RJL Willems; citation_author=WJB van Wamel; citation_author=FH Schuren; citation_author=MPM Caspers; citation_journal_title=PLoS Pathogens; citation_volume=3; citation_number=30; citation_pages=e7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Bordetella Species Are Distinguished by Patterns of Substantial Gene Loss and Host Adaptation.; citation_author=CA Cummings; citation_author=MM Brinig; citation_author=PW Lepp; citation_author=S van de Pas; citation_author=DA Relman; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=31; citation_pages=1484-1492; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The significance of iron in infection.; citation_author=JJ Bullen; citation_journal_title=Rev Infect Dis; citation_volume=3; citation_number=32; citation_pages=1127-1165; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Identification of alcaligin as the siderophore produced by Bordetella pertussis and B. bronchiseptica.; citation_author=CH Moore; citation_author=LA Foster; citation_author=DG Gerbig; citation_author=DW Dyer; citation_author=BW Gibson; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=33; citation_pages=1116-1118; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Bordetella iron transport and virulence.; citation_author=TJ Brickman; citation_author=MT Anderson; citation_author=SJ Armstrong; citation_journal_title=Biometals; citation_volume=20; citation_number=34; citation_pages=303-322; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains"/>
    <meta name="twitter:description" content="BackgroundBordetella pertussis is a Gram-negative bacterium that infects the human respiratory tract and causes pertussis or whooping cough. The disease has resurged in many countries including Finland where the whole-cell pertussis vaccine has been used for more than 50 years. Antigenic divergence has been observed between vaccine strains and clinical isolates in Finland. To better understand genome evolution in B. pertussis circulating in the immunized population, we developed an oligonucleotide-based microarray for comparative genomic analysis of Finnish strains isolated during the period of 50 years.Methodology/Principal FindingsThe microarray consisted of 3,582 oligonucleotides (70-mer) and covered 94% of 3,816 ORFs of Tohama I, the strain of which the genome has been sequenced [1]. Twenty isolates from 1953 to 2004 were studied together with two Finnish vaccine strains and two international reference strains. The isolates were selected according to their characteristics, e.g. the year and place of isolation and pulsed-field gel electrophoresis profiles. Genomic DNA of the tested strains, along with reference DNA of Tohama I strain, was labelled and hybridized. The absence of genes as established with microarrays, was confirmed by PCR. Compared with the Tohama I strain, Finnish isolates lost 7 (8.6 kb) to 49 (55.3 kb) genes, clustered in one to four distinct loci. The number of lost genes increased with time, and one third of lost genes had functions related to inorganic ion transport and metabolism, or energy production and conversion. All four loci of lost genes were flanked by the insertion sequence element IS481.Conclusion/SignificanceOur results showed that the progressive gene loss occurred in Finnish B. pertussis strains isolated during a period of 50 years and confirmed that B. pertussis is dynamic and is continuously evolving, suggesting that the bacterium may use gene loss as one strategy to adapt to highly immunized populations."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000904.g003"/>

  <meta property="og:title" content="Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6208'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6878'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4458&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000904">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000904" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000904&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Comparative Genomics of <i>Bordetella pertussis</i> Reveals Progressive Gene Loss in Finnish Strains
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eriikka Heikkinen
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Eriikka Heikkinen, Teemu Kallonen</p>

              
              

                <p>Affiliation:
                  Pertussis Reference Laboratory, National Public Health Institute, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Teemu Kallonen
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Eriikka Heikkinen, Teemu Kallonen</p>

              
              

                <p>Affiliations:
                  Pertussis Reference Laboratory, National Public Health Institute, Turku, Finland, 
                  Turku Graduate School of Biomedical Sciences, University of Turku, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lilli Saarinen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Finnish DNA Microarray Centre, Turku Centre for Biotechnology, University of Turku, Åbo Akademi University, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rolf Sara, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Finnish DNA Microarray Centre, Turku Centre for Biotechnology, University of Turku, Åbo Akademi University, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Audrey J. King, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Infectious Diseases and Screening, National Institute of Public Health and the Environment, Bilthoven, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frits R. Mooi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Infectious Diseases and Screening, National Institute of Public Health and the Environment, Bilthoven, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juhani T. Soini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Finnish DNA Microarray Centre, Turku Centre for Biotechnology, University of Turku, Åbo Akademi University, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jussi Mertsola, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pediatrics, Turku University Hospital, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qiushui He
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:qiushui.he@ktl.fi">qiushui.he@ktl.fi</a></p>

                <p>Affiliation:
                  Pertussis Reference Laboratory, National Public Health Institute, Turku, Finland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000904</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000904-t001" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000904-g001" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000904-g002" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000904-g003" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000904-t002" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000904">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000904" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Bordetella pertussis</em> is a Gram-negative bacterium that infects the human respiratory tract and causes pertussis or whooping cough. The disease has resurged in many countries including Finland where the whole-cell pertussis vaccine has been used for more than 50 years. Antigenic divergence has been observed between vaccine strains and clinical isolates in Finland. To better understand genome evolution in <em>B. pertussis</em> circulating in the immunized population, we developed an oligonucleotide-based microarray for comparative genomic analysis of Finnish strains isolated during the period of 50 years.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The microarray consisted of 3,582 oligonucleotides (70-mer) and covered 94% of 3,816 ORFs of Tohama I, the strain of which the genome has been sequenced <a href="#pone.0000904-Parkhill1">[1]</a>. Twenty isolates from 1953 to 2004 were studied together with two Finnish vaccine strains and two international reference strains. The isolates were selected according to their characteristics, <em>e.g.</em> the year and place of isolation and pulsed-field gel electrophoresis profiles. Genomic DNA of the tested strains, along with reference DNA of Tohama I strain, was labelled and hybridized. The absence of genes as established with microarrays, was confirmed by PCR. Compared with the Tohama I strain, Finnish isolates lost 7 (8.6 kb) to 49 (55.3 kb) genes, clustered in one to four distinct loci. The number of lost genes increased with time, and one third of lost genes had functions related to inorganic ion transport and metabolism, or energy production and conversion. All four loci of lost genes were flanked by the insertion sequence element IS<em>481</em>.</p>


<h3>Conclusion/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results showed that the progressive gene loss occurred in Finnish <em>B. pertussis</em> strains isolated during a period of 50 years and confirmed that <em>B. pertussis</em> is dynamic and is continuously evolving, suggesting that the bacterium may use gene loss as one strategy to adapt to highly immunized populations.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Heikkinen E, Kallonen T, Saarinen L, Sara R, King AJ, et al.  (2007) Comparative Genomics of <em>Bordetella pertussis</em> Reveals Progressive Gene Loss in Finnish Strains. PLoS ONE 2(9):
          e904.
            doi:10.1371/journal.pone.0000904</p><p><strong>Academic Editor: </strong>Yong-Sun Bahn, Soongsil University, Republic of Korea</p><p><strong>Received:</strong> June 8, 2007; <strong>Accepted:</strong> August 23, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Heikkinen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The research was supported by a grant from the Turku University Foundation, the Turku Graduate School of Biomedical Sciences, and the Special Governmental Fund for University Hospitals (EVO), Turku, Finland.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Bordetella pertussis</em> is a Gram-negative bacterium that causes a highly contagious respiratory disease, pertussis or whooping cough. Despite extensive immunization, the disease has remained endemic causing epidemics every 3 to 5 years. A resurgence of pertussis has been observed in USA, Europe, Canada, Australia and Asia <a href="#pone.0000904-Halperin1">[2]</a>–<a href="#pone.0000904-Moerman1">[10]</a>. In Finland, a whole-cell pertussis vaccine was introduced in 1952. Although the vaccination coverage of four doses has been high, pertussis remains endemic and the incidence of pertussis has increased since last decade <a href="#pone.0000904-Elomaa1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For more effective vaccination programs to prevent pertussis, it is important to understand pathogenesis and transmission of <em>B. pertussis</em>, to monitor changes in bacterial populations and to study impact of the changes on the prevention and incidence of disease. Different typing methods have been developed and applied for studying molecular epidemiology of <em>B. pertussis</em>, such as serotyping of fimbriae (Fim), genotyping of virulence genes, pulsed-field gel electrophoresis (PFGE) and multi-locus variable number of tandem repeats (MLVA) analysis <a href="#pone.0000904-Mooi1">[4]</a>, <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Lee1">[9]</a>, <a href="#pone.0000904-Mooi2">[11]</a>–<a href="#pone.0000904-Elomaa2">[21]</a>. These studies revealed that the <em>B. pertussis</em> population is dynamic and that antigenic divergence has occurred with respect to pertussis toxin (Ptx), pertactin (Prn), Fim2 and Fim3 between the vaccine strains and circulating isolates. Further, it was observed that epidemics in many countries were associated with clonal expansion of strains.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Genome sequencing offers a global view of the genetic content of pathogenic bacteria. Of the nine <em>Bordetella</em> species, annotated genome sequences of <em>B. pertussis, B. parapertussis, B. bronchiseptica</em> and <em>B. avium</em> have been recently published <a href="#pone.0000904-Parkhill1">[1]</a>, <a href="#pone.0000904-Sebaihia1">[22]</a>. The sequenced <em>B. pertussis</em> strain Tohama I, originally isolated in Japan in 1954, is considered a reference strain and used for the production of acellular pertussis vaccines. The genome of <em>B. pertussis</em> has 3,816 ORFs <a href="#pone.0000904-Parkhill1">[1]</a>. A notable feature is the high load of insertion sequence (IS) elements and relatively high content (9.4%) of pseudogenes <a href="#pone.0000904-Parkhill1">[1]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Based on the available sequence, we developed an oligonucleotide (70-mer) microarray for comparative genomic analysis of <em>B. pertussis</em> strains. Our aim was to study the evolution and molecular epidemiology of <em>B. pertussis</em> strains isolated in Finland where the whole-cell vaccine has been used for more than 50 years. Insight into <em>B. pertussis</em> capacity to adapt to vaccine-induced immunity is important for understanding the current epidemiology of pertussis in vaccinated populations. In addition to the historical perspective, the emphasis of strain selection was on recent isolates causing nationwide epidemics. The microarray results were compared with those obtained by standardized typing methods.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bacterial strains and culture conditions</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Bacterial isolates were selected from the <em>B. pertussis</em> strain collection from the Pertussis Reference Laboratory of the National Public Health Institute, Turku, Finland. The selection criteria included the time and geographic location of isolation, and their serotypes, genotypes and PFGE profiles <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Mooi4">[13]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. A total of 20 clinical isolates from 1953 to 2004 together with two Finnish vaccine strains and two international reference strains, Tohama I, isolated in Japan in 1954 and 18323, isolated in USA in 1946 were included (<a href="#pone-0000904-t001">Table 1</a>). Finnish vaccine strain 18530 was obtained from USA and has been used since 1962, and strain 1772 was obtained from UK and was added to the vaccine in 1976. The 50-year period was covered in a way that from 1950's to 1980's one to two strains represented each decade and from 1991 onwards strains were selected according to incidence peaks observed in the whole country (<a href="#pone-0000904-t001">Table 1</a>). We have previously shown that Finnish <em>B. pertussis</em> population was dynamic and has continuously evolved <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Mooi4">[13]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. Therefore, the strains selected for the present study were representative of the most prevalent PFGE profiles from 1953 to 2004. To avoid the influence of local outbreaks, isolates from at least two geographic locations were selected. All strains were isolated from children younger than 12 years.</p>
<div class="figure" id="pone-0000904-t001"><div class="img"><a name="pone-0000904-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000904.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000904.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Characteristics of the bacterial strains and the distribution of absent genes detected by microarray.</span></strong></p><span>doi:10.1371/journal.pone.0000904.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Bacteria stored in −70°C were recultured on Charcoal agar (Oxoid Ltd, Basingstoke, England) with 10% defibrinated sheep blood. The plates were incubated at 35–36°C for 3 days. Bacteria were harvested and suspended in distilled water. Prior to DNA isolation, the concentration of bacterial suspension was adjusted to 2×10<sup>9</sup> cells/ml.</p>


<h4>DNA isolation, digestion and hybridization</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p><em>B. pertussis</em> DNA was isolated with GenElute™ Bacterial Genomic DNA Kit (Sigma-Aldrich Inc., St. Louis, USA) and digested with Sma I (10 U/µg DNA, New England Biolabs, Ipswich, USA) according to the instructions of the labeling kit BioPrime®Array CGH Genomic Labeling System (Invitrogen Life Technologies, Carlsbad, USA). The quality of digested DNA was examined by gel electrophoresis. Digested and undigested DNA was compared as the starting material for hybridization and no effect of the DNA treatment on identification of absent genes was found. The existence of restriction site (CCCGGG) of SmaI was searched into DNA sequences of the 3,582 70-mer oligonucleotides used in the microarray (see section of the oligonucleotide-based microarray). Only 55 (1.5%) oligonucleotides were found to possess the restriction site. Of them, 48 belonged to the genes identified as present, five as duplicated, and two as absent, indicating that the existence of restriction sites on oligonucleotides did not affect the identification of absent or duplicated genes in the microarray. To avoid the possible effect of length and conformation of long chromosomal DNA on labelling, the digested DNA was used for all experiments.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We next compared the amount of DNA (range: 2 to 8 µg) used as the starting material and found no effect on identification of absent genes. The amount of 4 µg was thus chosen for all experiments. The 4 µg of digested DNA of the Tohama I strain used as reference strain were labeled with Cy 3 and that of testing strain with Cy 5 (Amersham Biosciences UK Limited, Buckinghamshire, England) according to the protocol of BioPrime®Array CGH Genomic Labeling System (Invitrogen Life Technologies, Carlsbad, USA). The labeled DNA was purified with the purification kit of the same manufacturer and then combined. The concentrations of Cy3 and Cy5 were measured with ND-1000 Spectrophotometer (NanoDrop Technologies, Wilmington, USA).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The volume of labeled DNA was reduced below 15.7 µl in vacuum centrifuge, and 2.5 µl of yeast tRNA (10 mg/ml, Invitrogen Life Technologies, Carlsbad, USA), 4.3 µl of 20×SSC (1×SSC was 8.8 g/l NaCl and 4.4 g/l trisodiumcitrate) and 0.8 µl of 10% SDS were added. After denaturation at 100°C for 3 min, 1.7 µl of 10×Blocking Solution from DIG-Wash and Block Buffer set–kit (Roche, Mannheim, Germany) was added. The final volume for hybridization was adjusted with water to 25 µl. Before hybridization, slides were fixed by UV-cross linking with 90 mJ cm<sup>−2</sup>, washed for 1 min in 0.1% of SDS and 2 min in water to remove residual salt, and incubated at 50°C for 30 min in BSA solution containing 1% of BSA fraction V, 2×SSC and 0.1% of SDS to block nonspecific binding. The slides were further washed in 2×SSC for 3 min and then in 0.2×SSC for 3 min.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Hybridization was conducted under 22×25 mm LifterSlip ™ coverslip (Erie Scientific Company, Portsmouth, USA) in hybridization chamber at 65°C for 16 h. To control humidity inside the hybridization chamber, a strip of thin paper, humidified with approximately 100 µl of 3.4×SSC, was placed under the slide. After hybridization slides were washed at 65°C for 10 min with 1×SSC and 0.1% of SDS. The following washes were conducted at room temperature for 10 min with 0.5×SSC and 0.01% SDS, and for 3 min and 1 min with 0.1×SSC. Slides were dried in slide centrifuge and scanned with ScanArray ®5000 (PerkinElmer, Waltham, USA).</p>


<h4>Oligonucleotide-based microarray</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The oligonucleotides used in the microarray were designed in collaboration with Operon Biotechnologies GmbH (Cologne, Germany), and an Array-Ready Oligo Set™ was purchased from the Operon. The set of 3,582 70-mer oligonucleotides, where one oligonucleotide corresponded to one gene, covered 94% of 3,816 ORFs of <em>B. pertussis</em> Tohama I <a href="#pone.0000904-Parkhill1">[1]</a>. Lyophilized oligonucleotides were dissolved in 50% of DMSO and 2×SSC to the concentration of 20µM. The oligonucleotides were spotted on UltraGaps™ coated slides (Corning Life Sciences, Acton, USA) with “R-ray”, an in-house spotter in the Finnish DNA Microarray Centre, the Centre for Biotechnology, Turku, Finland. Each oligonucleotide was spotted twice. A total of 20 randomized negative controls with no match to DNA of <em>B. pertussis</em> Tohama I were spotted in 8 separate spots to control non-specific binding. Contamination of printing tips and non-specific binding of DNA to slide surface was controlled by spotting water and buffer in 244 spots. Oligonucleotides representing 12 house-keeping genes of <em>B. pertussis</em> were spotted 10 times and used as positive controls to monitor the intra-array variation of the hybridization.</p>


<h4>Image processing, data analysis and statistical testing</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Image processing was made using ScanAlyze program from Michael B. Eisen, (Univerity of California at Berkeley). MA-plot, where mean of intensity (A) is plotted against log-transformed intensity ratio (M) of each spot, was created for classifying the genes present or absent in the testing strain. In the MA-plot, majority of the M-values was centered to zero, and a clearly separated group of genes was formed with low M-values (<a href="#pone-0000904-g001">Figure 1</a>). As the group was completely separated from the majority, genes included in the group were considered as absent and filtered from the raw data by defining M-value&lt;−2. Student's t-test was used to determine the difference in log-ratios between present and absent genes and p-value&lt;0.05 considered the statistically significant.</p>
<div class="figure" id="pone-0000904-g001"><div class="img"><a name="pone-0000904-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000904.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000904.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>MA-plot analysis of microarray results of a <em>B. pertussis</em> clinical strain (PRCB179).</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The mean of intensity (A) is plotted against log-transformed intensity ratio (M) of individual spots. Two distinct clusters of the genes are shown. The genes present in both reference and testing strains were centered on zero, and the genes lost in the testing strain formed a separate cluster defined by M-value&lt;−2.</p>
<span>doi:10.1371/journal.pone.0000904.g001</span></div>

<h4>Confirmation of microarray results with PCR</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>PCR was used to confirm absent genes of the testing strains identified by microarray. The primers were designed with Primer3 program (Steve Rozen&amp;Helen J. Skaletsky, 2000) on the basis of the two genes which flanked the lost locus or of the two genes located at each end of the lost locus. The sequences of the primers and their target genes were shown in <a href="#pone.0000904.s001">Table S1</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>PCR reaction contained 5–50 ng of purified DNA, 200 µM of each dNTP (Promega, Madison, WI, USA), 20 pmol of primers (Sigma Aldrich, Haverhill, UK), 5% DMSO (Merck, Darmstadt, Germany), 1×Buffer II, 1.5–3 mM of MgCl<sub>2</sub>, 0.625–2.5 U of AmpliTaq Gold® DNA polymerase (Applied Biosystems, Foster City, USA). The reaction volume was adjusted to 50 µl with Molecular Biology grade water (Eppendorf, Hamburg, Germany). PCR was run in MJ Research PTC 200 Thermal Cycler (Global Medical Instrumentation, Ramsey, USA) with the following protocol: initial denaturation at 95°C for 8 min followed by 30–38 cycles of 30 sec at 94°C, 30 sec to 1 min at 60–62°C and 1 to 3 min at 72°C, and with a final elongation of 5 min at 72°C. The expected size of PCR products was confirmed by gel electrophoresis.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Population of <em>B. pertussis</em> in Finland</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>More than 400 clinical isolates collected since 1953 have been analyzed by serotyping of Fim, genotyping of pertussis toxin S1 subunit (<em>ptxA</em>) and pertactin (<em>prn</em>), and PFGE <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Mooi4">[13]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. The strains isolated in 1953–1965 had the same <em>ptxA</em> as the vaccine strains (<em>ptxA2</em>), but a change to <em>ptxA1</em> occurred in the 1970's (<a href="#pone-0000904-t001">Table 1</a>). During the 1980's, a change from <em>prn1</em> (the vaccine type) to <em>prn2</em> (non-vaccine type) was observed. We have previously shown that Finnish <em>B. pertussis</em> population was dynamic and has continuously evolved <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. The most prevalent PFGE profiles found during the period of 50 years were BpFinR1 in 1953–1965, BpSR23 in 1970s and BpSR18 in 1980's (<a href="#pone-0000904-g002">Figure 2</a>). From 1991 to 2004 the predominant profiles were BpSR1, BpSR147 and BpSR11 (<a href="#pone-0000904-g002">Figure 2</a>, <a href="#pone-0000904-t001">Table 1</a>). BpSR11 strains started to emerge in 1999, and its frequency reached 55% in 2003 and 42% in 2004, when the nationwide epidemic of pertussis occurred (unpublished data). Along with BpSR11, BpSR12 emerged in 2000, and the highest frequency (16%) for the profile was detected in 2003. Another profile that co-emerged with BpSR11 was BpSR5, for which the frequency in 1999 was 7%. PFGE profiles of all clinical isolates analyzed were clearly different from that of Tohama I strain, strain 18323 and the Finnish vaccine strain 18530 (<a href="#pone-0000904-g002">Figure 2</a>) <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. The other Finnish vaccine strain 1772 represented BpSR23, the predominant PFGE profile in 1970s. The relatedness between Tohama I and strain 18323 was less than 50% (<a href="#pone-0000904-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000904-g002"><div class="img"><a name="pone-0000904-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000904.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000904.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Dendrogram analysis of PFGE profiles of <em>B. pertussis</em> clinical and reference strains.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The DNA profiles obtained after restriction with XbaI are shown, and only different PFGE profiles identified among the strains tested by microarray were included. The unweighted pair group method using arithmetic averages (UPGMA) with 1% band tolerance and 1% optimization settings was used as the clustering method.</p>
<span>doi:10.1371/journal.pone.0000904.g002</span></div>

<h4>Performance of the microarray</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The oligonucleotide-based microarray was proved to be specific for <em>B. pertussis</em> because no hybridization was observed to randomized negative control spots or water spots. Only marginal variation in the log-ratios was observed when self to self hybridization of DNA of the reference strain and dye swap of DNA of the testing strain were tested.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The 12 housekeeping genes with multiple spots (ten per array) were used for the determination of intra-array variation. The mean of signal intensity ratio (reference strain <em>vs</em> testing strain) ranged from 1.03 to 1.29 and SD from 0.04 to 0.08. The coefficient of variation (CV) was thus 3.37 to 6.76%. Number of absent genes detected was proved to be constant among all the replicate spots. For the determination of inter-array variation, 12 virulence genes present in the testing strain were chosen. Six individual results from three separate arrays were calculated. The mean of signal intensity ratio (reference strain <em>vs</em> testing strain) ranged from 0.99 to 1.40 and SD from 0.04 to 0.09. The CV was thus 3.52 to 8.38%.</p>


<h4>Gene loss in <em>B. pertussis</em> strains</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Compared with the strain Tohama I, four distinct loci were identified to be absent among the Finnish clinical isolates (<a href="#pone-0000904-t001">Table 1</a>). Relative to the Tohama I chromosome, the number of genes absent was 24 (size: 24 kb) for locus 1 (range: BP0910A-BP0934), seven (8.6 kb) for locus 2 (BP1135-BP1141), 18 (22.7 kb) for locus 3 (BP1948-BP1966), and 16 (17.7 kb) for locus 4 (BP2088-BP2103). However, the four loci were present in the international reference strain 18323 and in the Finnish strain 18530 that has been used for vaccine production since 1962. The other strain, 1772 that was added to the vaccine in 1976, missed the loci 1 and 2. Microarray data has been deposited in Gene Expression Omnibus database (<a href="http://www.ncbi.nlm.nih.gov/projects/geo/">http://www.ncbi.nlm.nih.gov/projects/geo​/</a>) and is accessible through accession number GSE8092.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Twenty clinical isolates from the period 1953 to 2004 were analyzed with the microarrays. All three strains analyzed from 1953–1965 missed only one locus (Locus 2). All nine strains analyzed from 1977 to 1996 missed two loci (Loci 1 and 2). Finally, all eight strains analyzed from 1999 to 2004 missed the Loci 1 and 2 in addition to Locus 3. Altogether, the eight strains had 44 absent genes (55.3 kb) in the three different loci. Only the strain PRCB223 which was isolated in 1996 was found to miss Locus 4 (<a href="#pone-0000904-t001">Table 1</a>). The absence of the four lost loci was confirmed by PCR (<a href="#pone.0000904.s002">Table S2</a>).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Based on genome sequence of the strain Tohama I, all the four absent loci were flanked by IS<em>481</em>.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Although the four absent loci identified among the 20 clinical isolates were present in the reference strain 18323 and the Finnish vaccine strain 18530, additional absent loci were found in the two strains (<a href="#pone.0000904.s003">Dataset S1</a>). Strain 18323 had 196 absent genes identified in 23 loci and the number of genes in each locus ranged from 1 to 33. Of the 23 absent loci, nine were bordered by IS<em>481</em> and 14 had IS<em>481</em> adjacent to one side. The Finnish vaccine strain 18530 had 22 absent genes in five different loci and the number of genes in each locus ranged from 1 to 9 (<a href="#pone.0000904.s003">Dataset S1</a>). Of the five loci, four were bordered by IS<em>481</em> and one had IS<em>481</em> adjacent to one side. Two loci (BP2627-BP2629 and BP3104-BP3110) were absent in both strains 18323 and 18530. A single gene (BP1225), flanked on one side by IS<em>481</em>, was absent in two clinical strains KKK1277 and 1330, isolated in 1964 and 1965, respectively.</p>


<h4>Gene duplication in <em>B. pertussis</em> strain</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Of the 20 clinical isolates, one strain KKK1330 was found to have both gene loss and duplication. This strain was isolated in 1965 and had a PFGE profile BpFINR14 distinct from the prevalent profile BpFinR1 found in 1950s and 1960s (<a href="#pone-0000904-g002">Figure 2</a>, <a href="#pone-0000904-t001">Table 1</a>). The strain missed locus 2 (BP1135-BP1141), whereas two loci were duplicated (i.e. BP1288-BP1442 and BP1481-BP1487). The mean of log-transformed intensity (mean±SD: 0.48±0.22) of all the duplicated genes was significantly higher than that of all other present genes (−0.36±0.26, p&lt;0.001). When the mean of log-transformed intensity was compared between the duplicated genes and all other present genes of the strain KKK1277 isolated in 1964, no significant difference was observed (p = 0.181). IS<em>481</em> was found adjacent to one side of the locus BP1288-BP1442.</p>


<h4>Characteristics of the absent and duplicated genes</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>In the genome of the strain Tohama I, the percentage of the pseudogenes was 9.4% and average of the GC contents was 67.6% <a href="#pone.0000904-Parkhill1">[1]</a>. Of the 65 lost genes identified in 4 different loci, 11 (16.9%) were pseudogenes. The average percentage of pseudogenes is significantly higher than that in the genome of Tohama I strain (chi-squared test, p = 0.047). The corresponding percentages in the loci were 28.6%, 16.7%, 30.8% and 12.5%, respectively. The GC content of the lost genes in each locus was 69.6%, 66.0%, 70.4% and 68.9%, respectively.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>A total of 23 functional categories of genes were defined in the Tohama I strain according to the Clusters of Orthologous Groups (COG) <a href="#pone.0000904-Parkhill1">[1]</a>. On the basis of the COG, functions of the absent and duplicated genes were compared with that of the strain Tohama I. The number of genes which could be classified into the COG categories are shown in <a href="#pone-0000904-t002">Table 2</a> and included in the comparison (<a href="#pone-0000904-g003">Figure 3</a>). Of the 23 functional categories of genes, 20 were included in the comparison (<a href="#pone-0000904-g003">Figure 3</a>), because none of the absent or duplicated genes belonged to the categories of RNA processing and modification, cell cycle control and chromatin structure and dynamics.</p>
<div class="figure" id="pone-0000904-g003"><div class="img"><a name="pone-0000904-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000904.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000904.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Distributions of lost and duplicated genes defined in the 20 functions based on the Clusters of Orthologous Groups.</span></strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>Number of the genes is shown in <a href="#pone-0000904-t002">Table 2</a>.</p>
<span>doi:10.1371/journal.pone.0000904.g003</span></div><div class="figure" id="pone-0000904-t002"><div class="img"><a name="pone-0000904-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000904" data-uri="info:doi/10.1371/journal.pone.0000904.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000904.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000904.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000904.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000904.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000904.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Number of absent and duplicated genes detected in Finnish clinical strains and vaccine strain and the reference strain relative to the genome of <em>B. pertussis</em> Tohama I.</span></strong></p><span>doi:10.1371/journal.pone.0000904.t002</span></div><a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>For the clinical isolates, most notable were high frequencies of absent genes identified in the two functions: inorganic ion transport and metabolism (N = 8) and energy production and conversion (N = 7), representing one third (15/44) of total number of absent genes classified in the COG (<a href="#pone-0000904-g003">Figure 3</a>). The eight genes defined in the function of inorganic ion transport and metabolism included two coding for putative ferric siderophore receptors (BP1138 and BP1962), two for probable inner membrane components of binding-protein-dependent transport system (BP0914 and BP0915), putative exported protein (BP0913), putative signal transduction protein (BP1137), putative flavocytochrome (BP1961), and putative dioxygenase hydroxylase component (BP2091). The seven genes defined in the function of energy production and conversion were putative succinate-semialdehyde dehydrogenase [NADP+] (BP0919), putative exported protein (BP0920), citrate utilization protein B (BP0921), probable oxidoreductase (BP1953), putative monooxygenase (BP1954), putative ferredoxin (BP2095) and putative fatty-acyl-CoA racemase (BP2096).</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>There were seven absent genes identified in the function of amino acid transport and metabolism. Of the seven genes, five (BP1948-BP1951 and BP1957) were in locus 3 and two (BP2089 and BP2090) in locus 4. Four genes (BP1950, BP1951, BP2089 and BP2090) were associated with ATP binding cassette (ABC) systems which are responsible for the import and export of a wide variety of molecules across cell membranes of bacteria <a href="#pone.0000904-Garmory1">[23]</a>.</p>
<a id="article1.body1.sec3.sec5.p5" name="article1.body1.sec3.sec5.p5"></a><p>Of the five absent genes defined as function unknown, three (BP0923, BP0927 and BP0931) were in locus 1, one (BP1965) in locus 3 and one (BP2103) in locus 4. Four genes coded for putative exported proteins.</p>
<a id="article1.body1.sec3.sec5.p6" name="article1.body1.sec3.sec5.p6"></a><p>We have not observed any absent or duplicated genes that encoded known virulence factors of <em>B. pertussis</em>.</p>
<a id="article1.body1.sec3.sec5.p7" name="article1.body1.sec3.sec5.p7"></a><p>For the vaccine strain 18530, 12 of 22 absent genes were classified in the COG. Of the 12 genes, three belonged to the function for inorganic ion transport and metabolism (<a href="#pone-0000904-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.sec5.p8" name="article1.body1.sec3.sec5.p8"></a><p>For the reference strain 18323, the missing genes were found in 17 of 20 functional categories. The three categories without missing genes were replication, recombination and repair, cell cycle control, mitosis and meiosis, and nucleotide transport and metabolism (<a href="#pone-0000904-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.sec5.p9" name="article1.body1.sec3.sec5.p9"></a><p>For the strain KKK1330, the duplicated genes were found in the following functional categories: cell motility, intracellular trafficking and secretion, cell wall/membrane biogenesis, and posttranslational modification, protein turnover, chaperones (<a href="#pone-0000904-g003">Figure 3</a>). A number of duplicated genes were related to flagella.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>A resurgence of pertussis has been observed in many countries, despite high vaccine coverage <a href="#pone.0000904-Halperin1">[2]</a>–<a href="#pone.0000904-Moerman1">[10]</a>. The causes for the resurgence are multiple, such as increased awareness of the disease, use of better microbiological diagnosis and different effectiveness of the vaccines used, waning vaccine-induced immunity and pathogen adaptation. Indeed, antigenic divergence has been found between <em>B. pertussis</em> vaccine strains and circulating isolates in the countries with the re-emergence of the disease <a href="#pone.0000904-Mooi1">[4]</a>–<a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-McIntyre1">[8]</a>, <a href="#pone.0000904-Lee1">[9]</a>, <a href="#pone.0000904-Mooi3">[12]</a>–<a href="#pone.0000904-Weber1">[16]</a>. Analyses of <em>B. pertussis</em> clinical isolates using PFGE or MLVA further revealed the clonal expansion of certain strains which caused recent epidemics in the countries <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Weber1">[16]</a>–<a href="#pone.0000904-Elomaa2">[21]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Vaccination against pertussis has been introduced in Finland since 1952. The vaccine is produced at the National Public Health Institute, Helsinki, Finland. Strain 18530 has been used since 1962, and strain 1772 was added to the vaccine in 1976. The vaccine has not changed since then, and the vaccination coverage for four doses has been 95%. To better understand the evolution of <em>B. pertussis</em> in immunized populations, an oligonucleotide-based microarray was developed and used for comparative genomic analysis of Finnish strains isolated during a period of 50 years.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Compared with the genome of strain Tohama I, a progressive gene loss was observed among the Finnish clinical strains. Strains isolated in 1950's and 1960's had almost the same gene content as Tohama I, with only seven genes absent. A marked change with the deletion of 24 genes occurred in strains isolated in 1970's, more than 20 years after the vaccination was introduced. The gene content of the strains isolated from the following 20 years remained basically the same. The next major deletion of 18 genes was first detected in strains isolated in 1999, and the gene content of strains isolated throughout the following period remained unchanged. Our data confirmed that <em>B. pertussis</em> population is dynamic and is continuously evolving <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Mooi4">[13]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>, suggesting that the bacterium uses gene loss as one strategy for its better adaptation to the highly vaccinated populations. For the future, it is of particular interest to compare our results with those obtained from strains isolated from the same period and from a non-vaccinated population.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Our results obtained from the oligonucleotide-based microarray are consistent with two recent studies of <em>B. pertussis</em> strains using cDNA microarray <a href="#pone.0000904-Brinig1">[24]</a>, <a href="#pone.0000904-Caro2">[25]</a>. In addition, we detected a new lost locus that contained 16 genes. However, the lost locus was only found in one among the 20 clinical strains studied.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>In agreement with two recent studies, none of genes coding for known virulence factors were found to be lost in the Finnish strains <a href="#pone.0000904-Brinig1">[24]</a>, <a href="#pone.0000904-Caro2">[25]</a> , suggesting that virulence factors are critical for <em>B. pertussis</em> to maintain its capability in the transmission and pathogenesis among the human populations. However, changes in known virulence factors pertussis toxin, pertactin and fimbriae coincided with the loss of loci 1 and 3. Since 1977, when the locus 1 was lost, the predominant allele for the gene encoding subunit 1 of pertussis toxin had changed from <em>ptxA2</em> to <em>ptxA1</em>. Five years after the change in <em>ptxA</em>, pertactin allele shifted from <em>prn1</em> to <em>prn2</em>. Both alleles are different from the ones in the strains used for Finnish whole cell vaccine. From 1976 to 1998, about 90% of the Finnish isolates expressed Fim2. Since 1999, two nationwide epidemics occurred. In addition to the gene loss of locus 3, the common feature between the strains mainly responsible for the epidemics is the expression of Fim3.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Pseudogenes are usually formed due to frameshift mutation, mutations leading to the insertion of a stop codon or insertion of IS elements <a href="#pone.0000904-Parkhill2">[26]</a>. The percentage of the pseudogenes identified among the four lost loci was significantly higher than that in the whole genome, suggesting that the pseudogenes are prone to deletion during the evolution of <em>B. pertussis</em>. In addition, we have observed that the deleted regions have a slightly higher GC content compared with the whole genome (respectively, 68.7% and 67.6%).</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Gene loss or genome reduction has been observed in several mammalian pathogens, and IS elements are found to play an important role in this process, in addition to causing chromosomal rearrangements and insertion mutations <a href="#pone.0000904-Chain1">[27]</a>–<a href="#pone.0000904-Leavis1">[30]</a>. A recent study has shown that IS-mediated diversification in <em>Enterococcus faecium</em> strongly contributed to its adaptation to hospital environment <a href="#pone.0000904-Leavis1">[30]</a>.During its evolution from <em>B. bronchiseptica</em> like ancestor, <em>B. pertussis</em> has undergone significant gene loss, most likely due to IS elements <a href="#pone.0000904-Parkhill1">[1]</a>, <a href="#pone.0000904-Cummings1">[31]</a>. Three distinct IS elements were found in the genome of <em>B. pertussis</em> Tohama I strain, with the copy number of 238 for IS<em>481</em>, 6 for IS<em>1002</em> and 17 for IS<em>1663</em> <a href="#pone.0000904-Parkhill1">[1]</a>. Interestingly, all the four loci of lost genes identified among Finnish isolates were flanked by IS<em>481</em>. Furthermore, IS<em>481</em> was located at least on one side of all loci of lost or duplicated genes detected in this study. Taken together, the results clearly showed that IS<em>481</em> plays a critical role in the <em>B. pertussis</em> evolution.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>It is of particular interest that one third of the lost genes had functions related to inorganic ion transport and metabolism and energy production and conversion. The acquisition of iron, which is an inorganic ion, is critical for pathogenic micro-organisms, because concentration of free iron is limited in the human host <a href="#pone.0000904-Bullen1">[32]</a>. In order to acquire iron, bacterial pathogens produce iron-chelating compounds (siderophores). Like <em>B. bronchiseptica, B. pertussis</em> contains an operon encoding the production, export and uptake of the siderophore alcaligin <a href="#pone.0000904-Moore1">[33]</a>, <a href="#pone.0000904-Brickman1">[34]</a>. Iron obtained from siderophores is internalized through TonB-dependent outer-membrane ferric complex receptors. <em>B. bronchiseptica</em> has 16 genes coding for these receptors, whereas <em>B. pertussis</em> has lost 3 of the genes during its evolution from <em>B. bronchiseptica</em> like ancestor, while one of the remaining was pseudogene <a href="#pone.0000904-Parkhill1">[1]</a>. Two of the 12 functional genes coding for the receptors were deleted among the Finnish strains isolated from 1999. These strains were responsible for the nationwide epidemics observed in the country <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. It remains to be shown if these deletions confer a selective advantage of the epidemic-causing strains.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>In this study, seven lost genes had functions in amino acid transport and metabolism, four of which encoded ATP binding cassette (ABC) transporter components and were located in two distinct loci. ABC transporters are a large group of proteins which have roles in important cellular functions such as import and export of various substances including iron chelators (siderophores) <a href="#pone.0000904-Garmory1">[23]</a>. According to the Tohama I chromosome, 80 ABC transporter components have been identified <a href="#pone.0000904-Parkhill1">[1]</a>. Many ABC transporter components are surface associated and are involved in virulence of pathogenic bacteria <a href="#pone.0000904-Garmory1">[23]</a>. It is known that the surface associated and secreted proteins usually interact with host cells, and therefore are potential immune targets. Deletion of such genes might confer a selective advantage during infection or transmission in vaccinated population. In line with this observation, we also found several lost genes which coded for putative exported proteins.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>PFGE has been shown to have best discriminatory power for the study of the epidemiology of <em>B. pertussis</em>. More than 400 Finnish clinical isolates collected from 1953 have been analyzed by PFGE, and altogether about 60 PFGE profiles have been identified <a href="#pone.0000904-Elomaa1">[6]</a>, <a href="#pone.0000904-Elomaa2">[21]</a>. One PFGE profile BpSR11 has emerged since 1999 and become predominant. The strains with BpSR11 were associated with the recent nationwide epidemic in the country. Interestingly, all the strains with BpSR11 analyzed in the present study had a lost locus (Locus 3) of 18 genes with the length of 22,700 base pairs.</p>
<a id="article1.body1.sec4.p11" name="article1.body1.sec4.p11"></a><p>Among the 20 clinical strains studied, only one was found to have both gene loss and duplication. The duplicated genes were associated with the functions of cell motility, intracellular trafficking and secretion, cell wall/membrane biogenesis, and posttranslational modification, protein turnoverand chaperones. Most interesting was that a number of duplicated genes were related to flagella. Flagella are usually expressed in the Bvg<sup>−</sup>-phase and involved in the survival of the bacteria in the environment. In <em>B. pertussis</em>, the flagellar operon was considered to be inactivated due to multiple pseudogenes and IS insertions <a href="#pone.0000904-Parkhill1">[1]</a>. It is of interest to study if the <em>B. pertussis</em> strain in which flagellar genes were duplicated can express the flagella and is motile.</p>
<a id="article1.body1.sec4.p12" name="article1.body1.sec4.p12"></a><p>In this study, the oligonucleotides used for microarray were designed from the genome of the Tohama I strain <a href="#pone.0000904-Parkhill1">[1]</a>. We were able to clearly distinguish between the presence, absence or duplication of genes among the clinical strains studied. A drawback of microarray-based comparative genomic analysis is that it cannot detect genes that are not present in the Tohama I strain. Although the acquisition of “new” genetic material does not seem to be a significant source of genetic variation in <em>B. pertussis</em> <a href="#pone.0000904-Parkhill1">[1]</a>, <a href="#pone.0000904-Cummings1">[31]</a>, it is worthwhile to sequence the genome of recent <em>B. pertussis</em> isolate(s) and to show how representative the strain Tohama I, which is used for the production of a number of pertussis vaccines, is. Furthermore, sequencing of recent isolate(s) may identify genes acquired by <em>B. pertussis</em> which play a role in the persistence and resurgence of pertussis despite vaccination.</p>
<a id="article1.body1.sec4.p13" name="article1.body1.sec4.p13"></a><p>In conclusion, this microarray based on 70-mer oligonucleotides proves to be a powerful tool for comparative genomic analysis of <em>B. pertussis</em> strains. Our results showed that the progressive gene loss mediated by the homologous recombination between IS elements occurred in <em>B. pertussis</em> strains in Finland, where the vaccination against pertussis has been used for more than 50 years. The recent epidemic-causing strains showed the largest degree of gene loss compared with the vaccine strains. The high consistency in gene loss could imply the presence of antigenic determinants among lost genes and gene loss may be one strategy for <em>B. pertussis</em> to adapt to highly vaccinated populations.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000904"></div><a name="pone.0000904.s001" id="pone.0000904.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000904.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Primers used in the PCR for the confirmation of lost loci detected by microarray</p>
<p class="siDoi">doi:10.1371/journal.pone.0000904.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000904.s002" id="pone.0000904.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000904.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">PCR results confirming the absence of four lost loci.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000904.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000904.s003" id="pone.0000904.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000904.s003">Dataset S1. </a>Absent genes detected in strains 18323 and 18530</strong></p><p class="siDoi">doi:10.1371/journal.pone.0000904.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.01 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to thank Päivi Haaranen, Eveliina Virtanen and Arja Reinikainen for the technical help with design and validation of the microarray, Annika Elomaa for providing the picture of PFGE profiles of the studied strains, and Asta Laiho for the help in the data analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: QH. Performed the experiments: EH TK. Analyzed the data: EH TK. Wrote the paper: EH TK. Other: Initiated and supervised the project: QH JM. Contributed to data analysis: LS. Initiated and contributed significantly to the development of the oligonucleotides used for the microarray: AK QH. Set up the oligonucleotide microarray: EH. Contributed to the development of the microarray: FM. Commented on the manuscript: FM JM. Participated in array and hybridisation planning: JS RS. Participated in array and hybridisation design: JS RS. Supervised the array production: JS RS. Wrote the manuscript with helpful discussions and comments from all co-authors: QH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000904-Parkhill1" id="pone.0000904-Parkhill1"></a>Parkhill J, Sebaihia M, Preston A, Murphy LD, Thomson N, et al.  (2003) Comparative analysis of the genome sequences of <em>Bordetella pertussis, Bordetella parapertussis</em> and <em>Bordetella bronchiseptica</em>. Nat Genet  35: 32–40.  <ul class="find" data-citedArticleID="993180" data-doi="10.1038/ng1227"><li><a href="http://dx.doi.org/10.1038/ng1227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+the+genome+sequences+of+Bordetella+pertussis%2C+Bordetella+parapertussis+and+Bordetella+bronchiseptica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+the+genome+sequences+of+Bordetella+pertussis%2C+Bordetella+parapertussis+and+Bordetella+bronchiseptica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000904-Halperin1" id="pone.0000904-Halperin1"></a>Halperin SA (2007) The Control of Pertussis–2007 and Beyond. N Engl J Med  356: 110–113.  <ul class="find" data-citedArticleID="993158" data-doi="10.1056/nejmp068288"><li><a href="http://dx.doi.org/10.1056/nejmp068288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Control+of+Pertussis%26%23x2013%3B2007+and+Beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Control+of+Pertussis%26%23x2013%3B2007+and+Beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000904-Celentano1" id="pone.0000904-Celentano1"></a>Celentano LP, Massari M, Paramatti D, Salmaso S, Tozzi AE (2005) Resurgence of Pertussis in Europe. Pediatr Infect Dis J  24: 761–765.  <ul class="find" data-citedArticleID="993142" data-doi="10.1097/01.inf.0000177282.53500.77"><li><a href="http://dx.doi.org/10.1097/01.inf.0000177282.53500.77" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resurgence+of+Pertussis+in+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resurgence+of+Pertussis+in+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000904-Mooi1" id="pone.0000904-Mooi1"></a>Mooi FR, van Loo IHM, King A (2001) Adaptation of <em>Bordetella pertussis</em> to Vaccination: A Cause for Its Reemergence? Emerg Infect Dis  7: 526–528.  <ul class="find" data-citedArticleID="993168" data-doi="10.3201/eid0707.017708"><li><a href="http://dx.doi.org/10.3201/eid0707.017708" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptation+of+Bordetella+pertussis+to+Vaccination%3A+A+Cause+for+Its+Reemergence%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptation+of+Bordetella+pertussis+to+Vaccination%3A+A+Cause+for+Its+Reemergence%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000904-Gzyl1" id="pone.0000904-Gzyl1"></a>Gzyl A, Augustynowicz E, Rabczenko D, Gniadek G, Slusarczyk J (2004) Pertussis in Poland. Int J Epidemiol  33: 358–365.  <ul class="find" data-citedArticleID="993154" data-doi="10.1016/j.biologicals.2004.07.002"><li><a href="http://dx.doi.org/10.1016/j.biologicals.2004.07.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pertussis+in+Poland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pertussis+in+Poland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000904-Elomaa1" id="pone.0000904-Elomaa1"></a>Elomaa A, Advani A, Donnelly D, Antila M, Mertsola J, et al.  (2005) Strain Variation among <em>Bordetella pertussis</em> Isolates in Finland, Where the Whole-Cell Pertussis Vaccine Has Been Used for 50 Years. J Clin Microbiol  43: 3681–3687.  <ul class="find" data-citedArticleID="993148" data-doi="10.1128/jcm.43.8.3681-3687.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.8.3681-3687.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strain+Variation+among+Bordetella+pertussis+Isolates+in+Finland%2C+Where+the+Whole-Cell+Pertussis+Vaccine+Has+Been+Used+for+50+Years." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strain+Variation+among+Bordetella+pertussis+Isolates+in+Finland%2C+Where+the+Whole-Cell+Pertussis+Vaccine+Has+Been+Used+for+50+Years.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000904-Ntezayabo1" id="pone.0000904-Ntezayabo1"></a>Ntezayabo B, De Serres G, Duval B (2003) Pertussis resurgence in Canada largely caused by cohort effect. Pediatr Infect Dis J  22: 22–27.  <ul class="find" data-citedArticleID="993178" data-doi="10.1097/00006454-200301000-00009"><li><a href="http://dx.doi.org/10.1097/00006454-200301000-00009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pertussis+resurgence+in+Canada+largely+caused+by+cohort+effect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pertussis+resurgence+in+Canada+largely+caused+by+cohort+effect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000904-McIntyre1" id="pone.0000904-McIntyre1"></a>McIntyre P, Gidding H, Gilmour R, Lawrence G, Hull B, et al.  (2002) Vaccine preventable diseases and vaccination coverage in Australia, 1999 to 2000. Commun Dis Intell Suppl: i–xi1–111.  <ul class="find" data-citedArticleID="993164"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Vaccine+preventable+diseases+and+vaccination+coverage+in+Australia%2C+1999+to+2000.&amp;auth=&amp;atitle=Vaccine+preventable+diseases+and+vaccination+coverage+in+Australia%2C+1999+to+2000." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vaccine+preventable+diseases+and+vaccination+coverage+in+Australia%2C+1999+to+2000." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vaccine+preventable+diseases+and+vaccination+coverage+in+Australia%2C+1999+to+2000.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000904-Lee1" id="pone.0000904-Lee1"></a>Lee YS, Yang CY, Lu CH, Tseng YH (2003) Molecular epidemiology of <em>Bordetella pertussis</em> isolated in Taiwan, 1992–1997. Microbiol and Immunol  47: 903–912.  <ul class="find" data-citedArticleID="993162" data-doi="10.1111/j.1348-0421.2003.tb03463.x"><li><a href="http://dx.doi.org/10.1111/j.1348-0421.2003.tb03463.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+epidemiology+of+Bordetella+pertussis+isolated+in+Taiwan%2C+1992%26%23x2013%3B1997." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+epidemiology+of+Bordetella+pertussis+isolated+in+Taiwan%2C+1992%26%23x2013%3B1997.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000904-Moerman1" id="pone.0000904-Moerman1"></a>Moerman L, Leventhal A, Slater P, Anis E, Yishai R, et al.  (2006) The re-emergence of pertussis in Israel. Isr Med Assoc J  8: 308–311.  <ul class="find" data-citedArticleID="993166"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+re-emergence+of+pertussis+in+Israel.&amp;auth=&amp;atitle=The+re-emergence+of+pertussis+in+Israel." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+re-emergence+of+pertussis+in+Israel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+re-emergence+of+pertussis+in+Israel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000904-Mooi2" id="pone.0000904-Mooi2"></a>Mooi FR, Hallander H, König CHWv, Hoet B, Guiso N (2000) Epidemiological Typing of <em>Bordetella pertussis</em> Isolates: Recommendations for a Standard Methodology. Euro J Clin Microbiol Infect Dis  19: 174–181.  <ul class="find" data-citedArticleID="993170" data-doi="10.1007/s100960050455"><li><a href="http://dx.doi.org/10.1007/s100960050455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epidemiological+Typing+of+Bordetella+pertussis+Isolates%3A+Recommendations+for+a+Standard+Methodology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epidemiological+Typing+of+Bordetella+pertussis+Isolates%3A+Recommendations+for+a+Standard+Methodology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000904-Mooi3" id="pone.0000904-Mooi3"></a>Mooi FR, van Oirschot H, Heuvelman K, van der Heide HGJ, Gaastra W, et al.  (1998) Polymorphism in the <em>Bordetella pertussis</em> Virulence Factors P.69/Pertactin and Pertussis Toxin in The Netherlands: Temporal Trends and Evidence for Vaccine-Driven Evolution. Infect Immun  66: 670–675.  <ul class="find" data-citedArticleID="993172"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Polymorphism+in+the+Bordetella+pertussis+Virulence+Factors+P.69%2FPertactin+and+Pertussis+Toxin+in+The+Netherlands%3A+Temporal+Trends+and+Evidence+for+Vaccine-Driven+Evolution.&amp;auth=&amp;atitle=Polymorphism+in+the+Bordetella+pertussis+Virulence+Factors+P.69%2FPertactin+and+Pertussis+Toxin+in+The+Netherlands%3A+Temporal+Trends+and+Evidence+for+Vaccine-Driven+Evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+in+the+Bordetella+pertussis+Virulence+Factors+P.69%2FPertactin+and+Pertussis+Toxin+in+The+Netherlands%3A+Temporal+Trends+and+Evidence+for+Vaccine-Driven+Evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+in+the+Bordetella+pertussis+Virulence+Factors+P.69%2FPertactin+and+Pertussis+Toxin+in+The+Netherlands%3A+Temporal+Trends+and+Evidence+for+Vaccine-Driven+Evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000904-Mooi4" id="pone.0000904-Mooi4"></a>Mooi FR, He Q, van Oirschot H, Mertsola J (1999) Variation in the <em>Bordetella pertussis</em> Virulence Factors Pertussis Toxin and Pertactin in Vaccine Strains and Clinical Isolates in Finland. Infect Immun  67: 3133–3134.  <ul class="find" data-citedArticleID="993174"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Variation+in+the+Bordetella+pertussis+Virulence+Factors+Pertussis+Toxin+and+Pertactin+in+Vaccine+Strains+and+Clinical+Isolates+in+Finland.&amp;auth=&amp;atitle=Variation+in+the+Bordetella+pertussis+Virulence+Factors+Pertussis+Toxin+and+Pertactin+in+Vaccine+Strains+and+Clinical+Isolates+in+Finland." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+the+Bordetella+pertussis+Virulence+Factors+Pertussis+Toxin+and+Pertactin+in+Vaccine+Strains+and+Clinical+Isolates+in+Finland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+the+Bordetella+pertussis+Virulence+Factors+Pertussis+Toxin+and+Pertactin+in+Vaccine+Strains+and+Clinical+Isolates+in+Finland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000904-Cassiday1" id="pone.0000904-Cassiday1"></a>Cassiday P, Sanden G, Heuvelman K, Mooi F, Bisgard KM, et al.  (2000) Polymorphism in <em>Bordetella pertussis</em> Pertactin and Pertussis Toxin Virulence Factors in the United States, 1935–1999. Journal Infect Dis  182: 1402–1408.  <ul class="find" data-citedArticleID="993140" data-doi="10.1086/315881"><li><a href="http://dx.doi.org/10.1086/315881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+in+Bordetella+pertussis+Pertactin+and+Pertussis+Toxin+Virulence+Factors+in+the+United+States%2C+1935%26%23x2013%3B1999." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+in+Bordetella+pertussis+Pertactin+and+Pertussis+Toxin+Virulence+Factors+in+the+United+States%2C+1935%26%23x2013%3B1999.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000904-Tsang1" id="pone.0000904-Tsang1"></a>Tsang RSW, Lau AKH, Sill ML, Halperin SA, Van Caeseele P, et al.  (2004) Polymorphisms of the Fimbria <em>fim3</em> Gene of <em>Bordetella pertussis</em> Strains Isolated in Canada. J Clin Microbiol  42: 5364–5367.  <ul class="find" data-citedArticleID="993188" data-doi="10.1128/jcm.42.11.5364-5367.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.11.5364-5367.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphisms+of+the+Fimbria+fim3+Gene+of+Bordetella+pertussis+Strains+Isolated+in+Canada." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphisms+of+the+Fimbria+fim3+Gene+of+Bordetella+pertussis+Strains+Isolated+in+Canada.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000904-Weber1" id="pone.0000904-Weber1"></a>Weber C, Boursaux-Eude C, Coralie G, Caro V, Guiso N (2001) Polymorphism of <em>Bordetella pertussis</em> Isolates Circulating for the Last 10 Years in France, Where a Single Effective Whole-Cell Vaccine Has Been Used for More than 30 Years. J Clin Microbiol  39: 4396–4403.  <ul class="find" data-citedArticleID="993190" data-doi="10.1128/jcm.39.12.4396-4403.2001"><li><a href="http://dx.doi.org/10.1128/jcm.39.12.4396-4403.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+of+Bordetella+pertussis+Isolates+Circulating+for+the+Last+10+Years+in+France%2C+Where+a+Single+Effective+Whole-Cell+Vaccine+Has+Been+Used+for+More+than+30+Years." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+of+Bordetella+pertussis+Isolates+Circulating+for+the+Last+10+Years+in+France%2C+Where+a+Single+Effective+Whole-Cell+Vaccine+Has+Been+Used+for+More+than+30+Years.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000904-Hallander1" id="pone.0000904-Hallander1"></a>Hallander HO, Advani A, Donnelly D, Gustafsson L, Carlsson R-M (2005) Shifts of <em>Bordetella pertussis</em> Variants in Sweden from 1970 to 2003, during Three Periods Marked by Different Vaccination Programs. J Clin Microbiol  43: 2856–2865.  <ul class="find" data-citedArticleID="993156" data-doi="10.1128/jcm.43.6.2856-2865.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.6.2856-2865.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shifts+of+Bordetella+pertussis+Variants+in+Sweden+from+1970+to+2003%2C+during+Three+Periods+Marked+by+Different+Vaccination+Programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shifts+of+Bordetella+pertussis+Variants+in+Sweden+from+1970+to+2003%2C+during+Three+Periods+Marked+by+Different+Vaccination+Programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000904-Caro1" id="pone.0000904-Caro1"></a>Caro V, Njamkepo E, Van Amersfoorth SCM, Mooi FR, Advani A, et al.  (2005) Pulsed-field gel electrophoresis analysis of <em>Bordetella pertussis</em> populations in various European countries with different vaccine policies. Microbes Infect  7: 976–982.  <ul class="find" data-citedArticleID="993136" data-doi="10.1016/j.micinf.2005.04.005"><li><a href="http://dx.doi.org/10.1016/j.micinf.2005.04.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pulsed-field+gel+electrophoresis+analysis+of+Bordetella+pertussis+populations+in+various+European+countries+with+different+vaccine+policies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pulsed-field+gel+electrophoresis+analysis+of+Bordetella+pertussis+populations+in+various+European+countries+with+different+vaccine+policies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000904-Schouls1" id="pone.0000904-Schouls1"></a>Schouls LM, Heide HGJvd, Vauterin L, Vauterin P, Mooi FR (2004) Multiple-Locus Variable-Number Tandem Repeat Analysis of Dutch <em>Bordetella pertussis</em> Strains Reveals Rapid Genetic Changes with Clonal Expansion during the Late 1990s. J Bacteriol  186: 5496–5505.  <ul class="find" data-citedArticleID="993184" data-doi="10.1128/jb.186.16.5496-5505.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.16.5496-5505.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple-Locus+Variable-Number+Tandem+Repeat+Analysis+of+Dutch+Bordetella+pertussis+Strains+Reveals+Rapid+Genetic+Changes+with+Clonal+Expansion+during+the+Late+1990s." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple-Locus+Variable-Number+Tandem+Repeat+Analysis+of+Dutch+Bordetella+pertussis+Strains+Reveals+Rapid+Genetic+Changes+with+Clonal+Expansion+during+the+Late+1990s.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000904-vanAmersfoorth1" id="pone.0000904-vanAmersfoorth1"></a>van Amersfoorth SCM, Schouls LM, van der Heide HGJ, Advani A, Hallander HO, et al.  (2005) Analysis of Bordetella pertussis Populations in European Countries with Different Vaccination Policies. J Clin Microbiol  43: 2837–2843.  <ul class="find" data-citedArticleID="993192" data-doi="10.1128/jcm.43.6.2837-2843.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.6.2837-2843.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Bordetella+pertussis+Populations+in+European+Countries+with+Different+Vaccination+Policies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Bordetella+pertussis+Populations+in+European+Countries+with+Different+Vaccination+Policies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000904-Elomaa2" id="pone.0000904-Elomaa2"></a>Elomaa A, Advani A, Donnelly D, Antila M, Mertsola J, et al.  (2007) Population dynamics of <em>Bordetella pertussis</em> in Finland and Sweden, neighbouring countries with different vaccination histories. Vaccine  25: 918–926.  <ul class="find" data-citedArticleID="993150" data-doi="10.1016/j.vaccine.2006.09.012"><li><a href="http://dx.doi.org/10.1016/j.vaccine.2006.09.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+dynamics+of+Bordetella+pertussis+in+Finland+and+Sweden%2C+neighbouring+countries+with+different+vaccination+histories." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+dynamics+of+Bordetella+pertussis+in+Finland+and+Sweden%2C+neighbouring+countries+with+different+vaccination+histories.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000904-Sebaihia1" id="pone.0000904-Sebaihia1"></a>Sebaihia M, Preston A, Maskell DJ, Kuzmiak H, Connell TD, et al.  (2006) Comparison of the Genome Sequence of the Poultry Pathogen <em>Bordetella avium</em> with Those of <em>B. bronchiseptica, B. pertussis</em>, and <em>B. parapertussis</em> Reveals Extensive Diversity in Surface Structures Associated with Host Interaction. J Bacteriol  188: 6002–6015.  <ul class="find" data-citedArticleID="993186" data-doi="10.1128/jb.01927-05"><li><a href="http://dx.doi.org/10.1128/jb.01927-05" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+Genome+Sequence+of+the+Poultry+Pathogen+Bordetella+avium+with+Those+of+B.+bronchiseptica%2C+B.+pertussis%2C+and+B.+parapertussis+Reveals+Extensive+Diversity+in+Surface+Structures+Associated+with+Host+Interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+Genome+Sequence+of+the+Poultry+Pathogen+Bordetella+avium+with+Those+of+B.+bronchiseptica%2C+B.+pertussis%2C+and+B.+parapertussis+Reveals+Extensive+Diversity+in+Surface+Structures+Associated+with+Host+Interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000904-Garmory1" id="pone.0000904-Garmory1"></a>Garmory HS, Titball RW (2004) ATP-Binding Cassette Transporters Are Targets for the Development of Antibacterial Vaccines and Therapies. Infect Immun  72: 6757–6763.  <ul class="find" data-citedArticleID="993152" data-doi="10.1128/iai.72.12.6757-6763.2004"><li><a href="http://dx.doi.org/10.1128/iai.72.12.6757-6763.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP-Binding+Cassette+Transporters+Are+Targets+for+the+Development+of+Antibacterial+Vaccines+and+Therapies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP-Binding+Cassette+Transporters+Are+Targets+for+the+Development+of+Antibacterial+Vaccines+and+Therapies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000904-Brinig1" id="pone.0000904-Brinig1"></a>Brinig MM, Cummings CA, Sanden GN, Stefanelli P, Lawrence A, et al.  (2006) Significant Gene Order and Expression Differences in <em>Bordetella pertussis</em> Despite Limited Gene Content Variation. J Bacteriol  188: 2375–2382.  <ul class="find" data-citedArticleID="993132" data-doi="10.1128/jb.188.7.2375-2382.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.7.2375-2382.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significant+Gene+Order+and+Expression+Differences+in+Bordetella+pertussis+Despite+Limited+Gene+Content+Variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significant+Gene+Order+and+Expression+Differences+in+Bordetella+pertussis+Despite+Limited+Gene+Content+Variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000904-Caro2" id="pone.0000904-Caro2"></a>Caro V, Hot D, Guigon G, Hubans C, Arrive M, et al.  (2006) Temporal analysis of French <em>Bordetella pertussis</em> isolates by comparative whole-genome hybridization. Microbes Infect  8: 2228–2235.  <ul class="find" data-citedArticleID="993138" data-doi="10.1016/j.micinf.2006.04.014"><li><a href="http://dx.doi.org/10.1016/j.micinf.2006.04.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Temporal+analysis+of+French+Bordetella+pertussis+isolates+by+comparative+whole-genome+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Temporal+analysis+of+French+Bordetella+pertussis+isolates+by+comparative+whole-genome+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000904-Parkhill2" id="pone.0000904-Parkhill2"></a>Parkhill J, Wren BW, Thomson NR, Titball RW, Holden MTG, et al.  (2001) Genome sequence of <em>Yersinia pestis</em>, the causative agent of plague. Nature  413: 523–527.  <ul class="find" data-citedArticleID="993182" data-doi="10.1038/35097083"><li><a href="http://dx.doi.org/10.1038/35097083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000904-Chain1" id="pone.0000904-Chain1"></a>Chain PSG, Hu P, Malfatti SA, Radnedge L, Larimer F, et al.  (2006) Complete Genome Sequence of <em>Yersinia pestis</em> Strains Antiqua and Nepal516: Evidence of Gene Reduction in an Emerging Pathogen. J Bacteriol  188: 4453–4463.  <ul class="find" data-citedArticleID="993144" data-doi="10.1128/jb.00124-06"><li><a href="http://dx.doi.org/10.1128/jb.00124-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+Genome+Sequence+of+Yersinia+pestis+Strains+Antiqua+and+Nepal516%3A+Evidence+of+Gene+Reduction+in+an+Emerging+Pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+Genome+Sequence+of+Yersinia+pestis+Strains+Antiqua+and+Nepal516%3A+Evidence+of+Gene+Reduction+in+an+Emerging+Pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000904-Balakirev1" id="pone.0000904-Balakirev1"></a>Balakirev ES, Ayala FJ (2003) Pseudogenes, Are They “Junk” or Functional DNA? Annu Rev of Genet  37: 123–151.  <ul class="find" data-citedArticleID="993126" data-doi="10.1146/annurev.genet.37.040103.103949"><li><a href="http://dx.doi.org/10.1146/annurev.genet.37.040103.103949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudogenes%2C+Are+They+%26%23x201c%3BJunk%26%23x201d%3B+or+Functional+DNA%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudogenes%2C+Are+They+%26%23x201c%3BJunk%26%23x201d%3B+or+Functional+DNA%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000904-Blanc1" id="pone.0000904-Blanc1"></a>Blanc G, Ogata H, Robert C, Audic S, phane , et al.  (2007) Reductive Genome Evolution from the Mother of Rickettsia. PLoS Genetics  3: e14.  <ul class="find" data-citedArticleID="993128" data-doi="10.1371/journal.pgen.0030014"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reductive+Genome+Evolution+from+the+Mother+of+Rickettsia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reductive+Genome+Evolution+from+the+Mother+of+Rickettsia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000904-Leavis1" id="pone.0000904-Leavis1"></a>Leavis HL, Willems RJL, van Wamel WJB, Schuren FH, Caspers MPM, et al.  (2007) Insertion Sequence -Driven Diversification Creates a Globally Dispersed Emerging Multiresistant Subspecies of <em>E. faecium</em>. PLoS Pathogens  3: e7.  <ul class="find" data-citedArticleID="993160" data-doi="10.1371/journal.ppat.0030007"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0030007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insertion+Sequence+-Driven+Diversification+Creates+a+Globally+Dispersed+Emerging+Multiresistant+Subspecies+of+E.+faecium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insertion+Sequence+-Driven+Diversification+Creates+a+Globally+Dispersed+Emerging+Multiresistant+Subspecies+of+E.+faecium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000904-Cummings1" id="pone.0000904-Cummings1"></a>Cummings CA, Brinig MM, Lepp PW, van de Pas S, Relman DA (2004) <em>Bordetella</em> Species Are Distinguished by Patterns of Substantial Gene Loss and Host Adaptation. J Bacteriol  186: 1484–1492.  <ul class="find" data-citedArticleID="993146" data-doi="10.1128/jb.186.5.1484-1492.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.5.1484-1492.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bordetella+Species+Are+Distinguished+by+Patterns+of+Substantial+Gene+Loss+and+Host+Adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bordetella+Species+Are+Distinguished+by+Patterns+of+Substantial+Gene+Loss+and+Host+Adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000904-Bullen1" id="pone.0000904-Bullen1"></a>Bullen JJ (1981) The significance of iron in infection. Rev Infect Dis  3: 1127–1165.  <ul class="find" data-citedArticleID="993134" data-doi="10.1093/clinids/3.6.1127"><li><a href="http://dx.doi.org/10.1093/clinids/3.6.1127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+significance+of+iron+in+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+significance+of+iron+in+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000904-Moore1" id="pone.0000904-Moore1"></a>Moore CH, Foster LA, Gerbig DG Jr, Dyer DW, Gibson BW (1995) Identification of alcaligin as the siderophore produced by <em>Bordetella pertussis</em> and <em>B. bronchiseptica</em>. J Bacteriol  177: 1116–1118.  <ul class="find" data-citedArticleID="993176"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+alcaligin+as+the+siderophore+produced+by+Bordetella+pertussis+and+B.+bronchiseptica.&amp;auth=&amp;atitle=Identification+of+alcaligin+as+the+siderophore+produced+by+Bordetella+pertussis+and+B.+bronchiseptica." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+alcaligin+as+the+siderophore+produced+by+Bordetella+pertussis+and+B.+bronchiseptica." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+alcaligin+as+the+siderophore+produced+by+Bordetella+pertussis+and+B.+bronchiseptica.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000904-Brickman1" id="pone.0000904-Brickman1"></a>Brickman TJ, Anderson MT, Armstrong SJ (2007) <em>Bordetella</em> iron transport and virulence. Biometals  20: 303–322.  <ul class="find" data-citedArticleID="993130" data-doi="10.1007/s10534-006-9031-1"><li><a href="http://dx.doi.org/10.1007/s10534-006-9031-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bordetella+iron+transport+and+virulence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bordetella+iron+transport+and+virulence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.XML" value="100242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.PDF" value="269488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t001.PNG_L" value="241900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t001.PNG_M" value="136607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t001.PNG_S" value="15710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t001.TIF" value="1186534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t001.PNG_I" value="42917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g001.PNG_L" value="21076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g001.PNG_M" value="23856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g001.PNG_S" value="5748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g001.TIF" value="29806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g001.PNG_I" value="9035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g002.PNG_L" value="1357192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g002.PNG_M" value="108878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g002.PNG_S" value="6694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g002.TIF" value="2207228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g002.PNG_I" value="37397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g003.PNG_L" value="504101"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g003.PNG_M" value="116309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g003.PNG_S" value="8547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g003.TIF" value="1443630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.g003.PNG_I" value="39694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t002.PNG_L" value="60087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t002.PNG_M" value="35045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t002.PNG_S" value="7689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t002.TIF" value="276472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.t002.PNG_I" value="10453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.s001.DOC" value="38400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.s002.DOC" value="60416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000904.s003.XLS" value="14848"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000904&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000904" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000904&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000904&volume=&issue=&title=Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains&author_name=Eriikka%20Heikkinen%2C%20Teemu%20Kallonen%2C%20Lilli%20Saarinen%2C%20Rolf%20Sara%2C%20Audrey%20J.%20King%2C%20Frits%20R.%20Mooi%2C%20Juhani%20T.%20Soini%2C%20Jussi%20Mertsola%2C%20Qiushui%20He&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000904" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904&amp;t=Comparative%20Genomics%20of%20Bordetella%20pertussis%20Reveals%20Progressive%20Gene%20Loss%20in%20Finnish%20Strains" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904&title=Comparative%20Genomics%20of%20Bordetella%20pertussis%20Reveals%20Progressive%20Gene%20Loss%20in%20Finnish%20Strains&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904&amp;title=Comparative%20Genomics%20of%20Bordetella%20pertussis%20Reveals%20Progressive%20Gene%20Loss%20in%20Finnish%20Strains" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000904&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Comparative Genomics of Bordetella pertussis Reveals Progressive Gene Loss in Finnish Strains';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000904';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comparative%20Genomics%20of%20Bordetella%20pertussis%20Reveals%20Progressive%20Gene%20Loss%20in%20Finnish%20Strains http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000904" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000904" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="24646"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bordetella+pertussis%22" title="Search for articles in the subject area:'Bordetella pertussis'"><div class="flagText">Bordetella pertussis</div></a>
              <div data-categoryid="43905" data-articleid="24646"
                   data-categoryname="Bordetella pertussis"
                   class="flagImage" title="Flag 'Bordetella pertussis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Duplicated+genes%22" title="Search for articles in the subject area:'Duplicated genes'"><div class="flagText">Duplicated genes</div></a>
              <div data-categoryid="35271" data-articleid="24646"
                   data-categoryname="Duplicated genes"
                   class="flagImage" title="Flag 'Duplicated genes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="24646"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24646"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24646"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="24646"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vaccines%22" title="Search for articles in the subject area:'Vaccines'"><div class="flagText">Vaccines</div></a>
              <div data-categoryid="31861" data-articleid="24646"
                   data-categoryname="Vaccines"
                   class="flagImage" title="Flag 'Vaccines' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1167'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3222'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2018&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
