

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Comparison of Phylogenetic Network Methods Using Computer Simulation</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001913"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001913" />

    <meta name="citation_title" content="A Comparison of Phylogenetic Network Methods Using Computer Simulation"/>
    <meta itemprop="name" content="A Comparison of Phylogenetic Network Methods Using Computer Simulation"/>

      <meta name="citation_author" content="Steven M. Woolley"/>
            <meta name="citation_author_institution" content="Computational Biology Program, Washington University School of Medicine, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="David Posada"/>
            <meta name="citation_author_institution" content="Departamento de Bioquímica, Genética e Inmunología, Facultad de Ciencias, Universidad de Vigo, Vigo, Spain"/>
      <meta name="citation_author" content="Keith A. Crandall"/>
            <meta name="citation_author_institution" content="Department of Biology, Brigham Young University, Provo, Utah, United States of America"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001913.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1913"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Inferring the historical patterns of biological evolution.; citation_author=M Pagel; citation_journal_title=Nature; citation_volume=401; citation_number=1; citation_pages=877-884; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The effect of recombination on the accuracy of phylogeny estimation.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Journal of Molecular Evolution; citation_volume=54; citation_number=2; citation_pages=396-402; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Consequences of recombination on traditional phylogenetic analysis.; citation_author=MH Schierup; citation_author=J Hein; citation_journal_title=Genetics; citation_volume=156; citation_number=3; citation_pages=879-891; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reconstructing patterns of reticulate evolution in angiosperms: what can we do?; citation_author=FT Bakker; citation_journal_title=Taxon; citation_volume=54; citation_number=4; citation_pages=593-604(512); citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Intraspecific gene genealogies: trees grafting into networks.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Trends in Ecology and Evolution; citation_volume=16; citation_number=5; citation_pages=37-45; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene genealogies and population variation in plants.; citation_author=BA Schaal; citation_author=KM Olsen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=6; citation_pages=7024-7029; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Network analysis of human and simian immunodeficiency virus sequence sets reveals massive recombination resulting in shorter pathways.; citation_author=S Wain-Hobson; citation_author=C Renoux-Elbe; citation_author=JP Vartanian; citation_author=A Meyerhans; citation_journal_title=J Gen Virol; citation_volume=84; citation_number=7; citation_pages=885-895; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Tree scanning: a method for using haplotype trees in phenotype/genotype association studies.; citation_author=AR Templeton; citation_author=T Maxwell; citation_author=D Posada; citation_author=JH Stengard; citation_author=E Boerwinkle; citation_journal_title=Genetics; citation_volume=169; citation_number=8; citation_pages=441-453; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Approaches for assessing phylogenetic accuracy.; citation_author=DM Hillis; citation_journal_title=Systematic Biology; citation_volume=44; citation_number=9; citation_pages=3-16; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Intraspecific cladogram estimation: Accuracy at higher levels of divergence.; citation_author=KA Crandall; citation_journal_title=Systematic Biology; citation_volume=43; citation_number=10; citation_pages=222-235; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Experimental phylogenetics: Generation of a known phylogeny.; citation_author=DM Hillis; citation_author=JJ Bull; citation_author=ME White; citation_author=MR Badgett; citation_author=IJ Molineux; citation_journal_title=Science; citation_volume=255; citation_number=11; citation_pages=589-591; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Evaluating intraspecific “network” construction methods using simulated sequence data: do existing algorithms outperform the global maximum parsimony approach?; citation_author=I Cassens; citation_author=P Mardulyn; citation_author=MC Milinkovitch; citation_journal_title=Systematic Biology; citation_volume=54; citation_number=12; citation_pages=363-372; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The phylogeography of dusky dolphins (Lagenorhynchus obscurus): A critical examination of network methods and rooting procedures.; citation_author=I Cassens; citation_author=K Van Waerebeek; citation_author=PB Best; citation_author=EA Crespo; citation_author=JC Reyes; citation_journal_title=Molecular Ecology; citation_volume=12; citation_number=13; citation_pages=1781-1792; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene genealogies and the coalescent process.; citation_author=RR Hudson; citation_journal_title=Oxford Surveys Evol Biol; citation_volume=7; citation_number=14; citation_pages=1-44; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Evolution of protein molecules.; citation_author=TH Jukes; citation_author=CR Cantor; citation_number=15; citation_pages=21-132; citation_date=1969; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Among-site rate variation and its impact on phylogenetic analyses.; citation_author=Z Yang; citation_journal_title=Trends In Ecology & Evolution; citation_volume=11; citation_number=16; citation_pages=367-372; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular Evolution.; citation_author=W-H Li; citation_number=17; citation_date=1997; citation_publisher=Sinauer Associates, Inc.; " />
      <meta name="citation_reference" content="citation_title=Population genetics of microbial pathogens estimated from multilocus sequence typing (MLST) data.; citation_author=M Perez-Losada; citation_author=EB Browne; citation_author=A Madsen; citation_author=T Wirth; citation_author=RP Viscidi; citation_journal_title=Infection, Genetics and Evolution; citation_volume=6; citation_number=18; citation_pages=97-112; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Neutral two-locus multiple allele models with recombination.; citation_author=RC Griffiths; citation_journal_title=Theoretical Population Biology; citation_volume=19; citation_number=19; citation_pages=169-186; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Properties of a neutral allele model with intragenic recombination.; citation_author=RR Hudson; citation_journal_title=Theor Popul Biol; citation_volume=23; citation_number=20; citation_pages=183-201; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Fast algorithms for computing the tripartition-based distance between phylogenetic networks.; citation_author=N Nguyen; citation_author=C Nguyen; citation_author=W-K Sung; citation_journal_title=Journal of Combinatorial Optimization; citation_volume=13; citation_number=21; citation_pages=223-242; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparison of Tree-Child Phylogenetic Networks.; citation_author=G Cardona; citation_author=F Rossello; citation_author=G Valiente; citation_number=22; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Imputing supertrees and supernetworks from quartets.; citation_author=B Holland; citation_author=G Conner; citation_author=K Huber; citation_author=V Moulton; citation_journal_title=Syst Biol; citation_volume=56; citation_number=23; citation_pages=57-67; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=An optimal algorithm for scanning all spanning trees of undirected graphs.; citation_author=A Shioura; citation_author=A Tamura; citation_author=T Uno; citation_journal_title=SIAM Journal on Computing; citation_volume=26; citation_number=24; citation_pages=678-692; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Comparison of phylogenetic trees.; citation_author=DF Robinson; citation_author=LR Foulds; citation_journal_title=Mathematical Biosciences; citation_volume=53; citation_number=25; citation_pages=131-147; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=A simulation comparison of phylogeny algorithms under equal and unequal evolutionary rates.; citation_author=MK Kuhner; citation_author=J Felsenstein; citation_journal_title=Molecular Biology and Evolution; citation_volume=11; citation_number=26; citation_pages=459-468; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood of phylogenetic networks.; citation_author=G Jin; citation_author=L Nakhleh; citation_author=S Snir; citation_author=T Tuller; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=27; citation_pages=2604-2611; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic networks: modeling, reconstructibility, and accuracy.; citation_author=BME Moret; citation_author=L Nakhleh; citation_author=T Warnow; citation_author=CR Linder; citation_author=A Tholse; citation_journal_title=Computational Biology and Bioinformatics, IEEE/ACM Transactions on; citation_volume=1; citation_number=28; citation_pages=13-23; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Taxon sampling and the accuracy of large phylogenies.; citation_author=B Rannala; citation_author=JP Huelsenbeck; citation_author=Z Yang; citation_author=R Nielsen; citation_journal_title=Syst Biol; citation_volume=47; citation_number=29; citation_pages=702-710; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis: models and estimation procedures.; citation_author=LL Cavalli-Sforza; citation_author=AWF Edwards; citation_journal_title=Evolution; citation_volume=32; citation_number=30; citation_pages=550-570; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Molecular Biology and Evolution; citation_volume=4; citation_number=31; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogenetic Analysis Using Parsimony (*and other methods). 4.0 beta 10 ed.; citation_author=DL Swofford; citation_number=32; citation_date=2002; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation.; citation_author=AR Templeton; citation_author=KA Crandall; citation_author=CF Sing; citation_journal_title=Genetics; citation_volume=132; citation_number=33; citation_pages=619-633; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=TCS: a computer program to estimate gene genealogies.; citation_author=M Clement; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Molecular Ecology; citation_volume=9; citation_number=34; citation_pages=1657-1659; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SplitsTree: analyzing and visualizing evolutionary data.; citation_author=DH Huson; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=35; citation_pages=68-73; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Application of phylogenetic networks in evolutionary studies.; citation_author=DH Huson; citation_author=D Bryant; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=36; citation_pages=254-267; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial portraits of human populations using median networks.; citation_author=HJ Bandelt; citation_author=P Forster; citation_author=BC Sykes; citation_author=MB Richards; citation_journal_title=Genetics; citation_volume=141; citation_number=37; citation_pages=743-753; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Network version 4.2.0.1: A software for population genetics data analysis.; citation_author=M Forster; citation_author=P Forster; citation_author=J Watson; citation_journal_title=4.2.0.1 ed: Fluxus Technology Ltd 1999–2007.; citation_number=38; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Using allele frequencies and geographic subdivision to reconstruct gene trees within a species: Molecular variance parsimony.; citation_author=L Excoffier; citation_author=PE Smouse; citation_journal_title=Genetics; citation_volume=136; citation_number=39; citation_pages=343-359; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=ARLEQUIN version 2.000: A software for population genetics data analysis. 2.0 ed.; citation_author=S Schneider; citation_author=D Roessli; citation_author=L Excoffier; citation_number=40; citation_date=2000; citation_publisher=Genetics and Biometry Laboratory, University of Geneva; " />
      <meta name="citation_reference" content="citation_title=Algorithms to Distinguish the Role of Gene-Conversion from Single-Crossover Recombination in the Derivation of SNP Sequences in Populations: Springer Berlin/Heidelberg.; citation_author=JS Song; citation_author=Z Ding; citation_author=D Gusfield; citation_author=CH Langley; citation_author=Y Wu; citation_number=41; citation_pages=231-245; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Theoretical foundation of the minimum-evolution method of phylogenetic inference.; citation_author=A Rzhetsky; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=42; citation_pages=1073-1095; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation.; citation_author=AR Templeton; citation_author=KA Crandall; citation_author=CF Sing; citation_journal_title=Genetics; citation_volume=132; citation_number=43; citation_pages=619-633; citation_date=1992; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Comparison of Phylogenetic Network Methods Using Computer Simulation"/>
    <meta name="twitter:description" content="BackgroundWe present a series of simulation studies that explore the relative performance of several phylogenetic network approaches (statistical parsimony, split decomposition, union of maximum parsimony trees, neighbor-net, simulated history recombination upper bound, median-joining, reduced median joining and minimum spanning network) compared to standard tree approaches, (neighbor-joining and maximum parsimony) in the presence and absence of recombination.Principal FindingsIn the absence of recombination, all methods recovered the correct topology and branch lengths nearly all of the time when the substitution rate was low, except for minimum spanning networks, which did considerably worse. At a higher substitution rate, maximum parsimony and union of maximum parsimony trees were the most accurate. With recombination, the ability to infer the correct topology was halved for all methods and no method could accurately estimate branch lengths.ConclusionsOur results highlight the need for more accurate phylogenetic network methods and the importance of detecting and accounting for recombination in phylogenetic studies. Furthermore, we provide useful information for choosing a network algorithm and a framework in which to evaluate improvements to existing methods and novel algorithms developed in the future."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001913.g006"/>

  <meta property="og:title" content="A Comparison of Phylogenetic Network Methods Using Computer Simulation" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1080'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2341'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7052&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001913">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001913" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001913&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Comparison of Phylogenetic Network Methods Using Computer Simulation
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Steven M. Woolley
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:stevenwoolley@wustl.edu">stevenwoolley@wustl.edu</a></p>

                <p>Affiliation:
                  Computational Biology Program, Washington University School of Medicine, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Posada, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Departamento de Bioquímica, Genética e Inmunología, Facultad de Ciencias, Universidad de Vigo, Vigo, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keith A. Crandall
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Brigham Young University, Provo, Utah, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001913</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001913-g001" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-t001" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-g002" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-t002" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-g003" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-g004" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-g005" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001913-g006" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001913">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001913" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>We present a series of simulation studies that explore the relative performance of several phylogenetic network approaches (statistical parsimony, split decomposition, union of maximum parsimony trees, neighbor-net, simulated history recombination upper bound, median-joining, reduced median joining and minimum spanning network) compared to standard tree approaches, (neighbor-joining and maximum parsimony) in the presence and absence of recombination.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In the absence of recombination, all methods recovered the correct topology and branch lengths nearly all of the time when the substitution rate was low, except for minimum spanning networks, which did considerably worse. At a higher substitution rate, maximum parsimony and union of maximum parsimony trees were the most accurate. With recombination, the ability to infer the correct topology was halved for all methods and no method could accurately estimate branch lengths.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results highlight the need for more accurate phylogenetic network methods and the importance of detecting and accounting for recombination in phylogenetic studies. Furthermore, we provide useful information for choosing a network algorithm and a framework in which to evaluate improvements to existing methods and novel algorithms developed in the future.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Woolley SM, Posada D, Crandall KA (2008) A Comparison of Phylogenetic Network Methods Using Computer Simulation. PLoS ONE 3(4):
          e1913.
            doi:10.1371/journal.pone.0001913</p><p><strong>Editor: </strong>Jason E. Stajich, University of California, Berkeley, United States of America</p><p><strong>Received:</strong> December 3, 2007; <strong>Accepted:</strong> February 24, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Woolley et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIH GM66276 (KAC, DP) and the Brigham Young University Office of Research and Creative Activities (SMW). SMW is also supported by an NSF Graduate Research Fellowship. DP is supported by grant BFU2004-02700 of the Spanish Ministry of Education and Science, and by the “Ramón y Cajal” program of the Spanish government.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Phylogenies are of central importance in testing comparative hypotheses in a wide variety of fields <a href="#pone.0001913-Pagel1">[1]</a>. Yet, at the population level, there are biological phenomena, such as recombination and hybridization that lead to reticulated relationships. Furthermore, at the population genetic level, lower levels of diversity sometimes lead to a lack of phylogenetic resolution and representing this uncertainty is important. Ignoring these issues may lead to erroneous estimation of evolutionary relationships <a href="#pone.0001913-Posada1">[2]</a> and/or poor estimates of parameters based on those phylogenies <a href="#pone.0001913-Schierup1">[3]</a>. Therefore, a number of approaches have been developed to represent genealogical relationships as reticulating networks, either by explicitly modeling reticulate events or non-explicitly by representing phylogenetic ambiguity or incompatibility <a href="#pone.0001913-Bakker1">[reviewed in 4]</a>, <a href="#pone.0001913-Posada2">[5]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A wide range of network methods are now available and heavily used by researchers in fields as disparate as phylogeography <a href="#pone.0001913-Schaal1">[6]</a>, virology <a href="#pone.0001913-WainHobson1">[7]</a>, and human quantitative genetics <a href="#pone.0001913-Templeton1">[8]</a>. Nevertheless, the ability of these methods to accurately <a href="#pone.0001913-Hillis1">[sensu 9]</a> estimate the true underlying genealogical relationships has not been thoroughly tested (i.e., assessing consistency, efficiency, and robustness). Indeed, we know of only a few such studies. First, Crandall <a href="#pone.0001913-Crandall1">[10]</a> explored a single method (statistical parsimony) relative to maximum parsimony using an empirically generated data set from a known phylogeny of the bacteriophage T7 <a href="#pone.0001913-Hillis2">[11]</a>. He found that the statistical parsimony approach outperformed maximum parsimony when levels of variation were low. More recently, Cassens et al. <a href="#pone.0001913-Cassens1">[12]</a> provided a more extensive evaluation of three different network algorithms (statistical parsimony, median-joining network, and minimum-spanning network) compared to a newly developed union of maximum parsimony trees approach using simulated data over four known tree topologies. They showed that maximum parsimony performed as well or better than the network approaches under all four tree topologies, that the minimum spanning network algorithm often performed significantly worse with a greatly increased number of errors in the estimation, and that the statistical parsimony and minimum spanning network approaches performed significantly worse when the evolutionary history had many missing intermediates. However, neither study investigated the performance of methods under conditions where reticulating relationships would be an expected evolutionary outcome (e.g., under recombination). It is presumably under such conditions where network approaches have an advantage in terms of estimating genealogical relationships.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Indeed, these notable differences among network approaches, coupled with the report of conflicting inferred histories from empirical data <a href="#pone.0001913-Cassens2">[13]</a>, invite a more thorough evaluation of network methods. Previous studies were limited by the number of topologies tested and the number of methods compared, due to difficulty in automating the comparisons (as well as the lack of recombination as indicated above). Therefore, we have embarked on a more extensive study that uses computer simulation to evaluate the performance of seven “non-explicit” network approaches: statistical parsimony [SP], minimum spanning network [MSN], split decomposition [SD], NeighborNet [NN], median network [MED], reduced median joining [RMD], and union of maximum parsimony trees [UMP], as well as one discrete method (shrub-gc) [SHB] which explicitly models recombination and gene-conversion. We also included two standard bifurcating phylogenetic approaches (maximum parsimony [MP] and neighbor-joining [NJ]). We explored the effect of a variety of substitution and recombination rates, sequence lengths, numbers of taxa, and models of substitution on the relative accuracy of these network and standard phylogenetic approaches.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Our basic approach was to simulate DNA sequences using the neutral coalescent with and without recombination <a href="#pone.0001913-Hudson1">[see 14]</a> and then run the resulting alignments through a variety of algorithms for estimating network relationships among sequences (<a href="#pone-0001913-g001">Figure 1</a>). We then compared the resulting subtrees within these networks to those under the true history and tabulated the frequency of correct subtrees for different approaches. Each of these phases, data simulation, tree processing, performance measures, and estimation approaches explored are detailed below. While simulations can provide general predictions about the behavior of the methods studied, as well as some sense of their robustness (insofar as differing models are explored in the simulations), it is rarely possible to simulate the entire universe of relevant models and the models simulated may represent real data only to a given extent.</p>
<div class="figure" id="pone-0001913-g001"><div class="img"><a name="pone-0001913-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Flowchart outlining our simulation and comparison method.</span></strong></p><span>doi:10.1371/journal.pone.0001913.g001</span></div>
<h4></h4>
<h5>Data Simulation.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>DNA sequence alignments were simulated under 18 different sets of conditions (“sets”) selected to represent a range of intraspecific data sets, including some extreme cases. We explored different sequence lengths (500 and 1000 base pairs), numbers of taxa (10, 20 and 50), substitution rates (6.25×10<sup>−6</sup>, 6.25×10<sup>−7</sup> expected substitutions per site per generation), recombination rates (0, 2.5×10<sup>−5</sup>, 1×10 <sup>−6</sup>, 4×10<sup>−6</sup> recombination events per site per generation) under a simple Jukes Cantor nucleotide substitution model <a href="#pone.0001913-Jukes1">[15]</a> with and without a gamma distributed site-rate heterogeneity <a href="#pone.0001913-Yang1">[16]</a> (<a href="#pone-0001913-t001">Table 1</a>). The substitution rates modeled here are typical for nuclear genes across a diversity of organisms <a href="#pone.0001913-Li1">[17]</a>. Recombination rates, on the other hand, have not been widely estimated across loci or organisms. However, in one extensive study examining recombination rates across a diversity of studies involving multi-locus sequence typing from a variety of organisms (and therefore across a range of loci), Pérez-Losada et al. <a href="#pone.0001913-PerezLosada1">[18]</a> estimated a wide range of recombination rates similar to those modeled here. The effective population size was always 1000. One thousand histories and alignments were simulated under each one of these scenarios to afford reasonable statistical comparisons among the different methods.</p>
<div class="figure" id="pone-0001913-t001"><div class="img"><a name="pone-0001913-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Simulation parameters for the neutral coalescent simulations with and without recombination.</span></strong></p><span>doi:10.1371/journal.pone.0001913.t001</span></div>
<h5>Tree Processing.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>In order to make appropriate comparisons between the simulated and inferred trees, branch lengths from the simulated trees were expressed as the number of <em>realized</em> changes rather than as the number of <em>expected</em> changes. This is because at low substitution rates it is very common that no changes occur along short branches in the simulated trees, and therefore they would be impossible to infer. Moreover, datasets with no realized changes at all were discarded from the analysis. To compare simulated and inferred trees, branches with zero length were collapsed. This was necessary since the tree comparisons described below would consider a zero length branch present in one tree but absent in another as a topological difference.</p>

<h5>Measures of Performance.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>Comparing the estimated relationships to the simulated (“true”) underlying relationships is simple when there is no recombination because the simulated evolutionary history is a single tree (sets 1–12, <a href="#pone-0001913-t001">Table 1</a>). However, when recombination is present (sets 13–18, <a href="#pone-0001913-t001">Table 1</a>), the simulated history cannot be represented by a single tree anymore, but by multiple trees that correspond to each of the recombinant fragments. In fact, this set of trees conforms to an ancestral recombination graph <a href="#pone.0001913-Griffiths1">[19]</a>, <a href="#pone.0001913-Hudson2">[20]</a>. This presents a certain difficulty in that the standard phylogenetic methods will be incorrect, by definition, as they will only return a single tree (or set of bifurcating trees in the case of maximum parsimony).</p>
<a id="article1.body1.sec2.sec1.sec3.p2" name="article1.body1.sec2.sec1.sec3.p2"></a><p>In order to compare the inferred trees or networks with the simulated trees or networks, we first needed to devise a method for comparing both single trees and sets of trees to single trees and networks. While several metrics have been proposed to compare “idealized” networks (i.e., galled <a href="#pone.0001913-Nguyen1">[21]</a>, tree-child or tree-sibling <a href="#pone.0001913-Cardona1">[22]</a>), networks estimated from real data (or networks simulated under meaningful models like the coalescent) seldom conform to the restrictions imposed by such representations. For example, in the most general of these, the tree-sibling networks, every hybrid node has at least one sibling that is a tree node; an assumption often violated by empirical data and lacking any evolutionary meaning. Moreover, even under these rather mathematical restrictions, none of these metrics can assure that they only take a value of 0 when two networks are isomorphic, i.e., they are “imperfect” <a href="#pone.0001913-Cardona1">[22]</a>. Another method that has been employed to compare networks with networks or networks with trees is comparison of their list of splits <a href="#pone.0001913-Holland1">[e.g. 23]</a>, which in principle, does not take into account branch lengths, but provides a simple calculation of type I and type II errors. We used related measures, which compare the splits of each tree embedded within a network or tree while accounting for branch lengths associated with each split (see below). We chose to work with tree enumerations or tree lists, regardless of whether the size of these enumerations was just one (for single trees) or more (for networks). For the simulated networks, the coalescent with recombination automatically provides the enumeration of the trees within the simulated network (or ancestral recombination graph). We wanted to measure how often the underlying tree(s) from the simulation (from now on, the “model trees”) were contained somewhere within the inferred tree(s) or network. For this, we used the optimal spanning tree algorithm of Shioura et al. <a href="#pone.0001913-Shioura1">[24]</a>, which enumerates all the trees contained in an undirected graph (‘network’) efficiently in terms of computational time and memory. Since the spanning trees from the inferred network may contain inferred internal nodes as tips, they were further processed to remove superfluous internal nodes (unobserved internal nodes with less than three connected branches). Also note that duplicate trees can arise from networks when reticulations can be broken at multiple edges, potentially leaving internal nodes as leaves, which are later pruned such that only input sequences are represented as leaves in the trees. If the number of trees contained within a single network exceeded 5,000,000, that network was excluded from the analysis due to time/resource constraints. Once the set of all trees contained within a network was created, all duplicate trees were removed, leaving only one copy of each tree with a given topology and branch lengths. Furthermore, duplicates were also removed from each set of MP trees (see below). For those methods that give a visual representation of splits (SD, NN, MED and RMD), we used the representation provided, even though that representation may not necessarily be unique. It is not clear whether different representations of the same split system must always embed the same trees.</p>
<a id="article1.body1.sec2.sec1.sec3.p3" name="article1.body1.sec2.sec1.sec3.p3"></a><p>Once we enumerate the trees contained within the estimated networks, or within the set of trees estimated by the traditional phylogenetic approaches, we need to compare these trees to the model tree(s). We used two related measures for tree comparison, the Robinson-Foulds (RF) score <a href="#pone.0001913-Robinson1">[25]</a> and the branch score (BS) <a href="#pone.0001913-Kuhner1">[26]</a>. The RF score is the total number of clades present in one tree but absent in another and does not take into account branch lengths. Therefore, the higher the score, the worse the topological match between the compared trees. The BS, on the other hand, computes the sum of the squares of the differences between each branch's length in each tree. Branches that appear in one tree but not in the other are scored as if compared to a branch of length zero. Two trees are equal when the BS between them is zero (and hence the RF is also zero). Two topologies are the same when the RF is zero. Thus, a tree <em>t</em> “exists” in a set of trees <em>T</em> if and only if there is a tree <em>s</em> in <em>T</em> such that the BS between <em>t</em> and <em>s</em> is zero. Similarly, a topology <em>t</em> “exists” in a set of trees <em>T</em> if and only if there is a tree <em>s</em> in <em>T</em> such that the RF distance between <em>t</em> and <em>s</em> is zero. In addition, we calculated a number of statistics to characterize different aspects of the relative performance of the different approaches. If we let <em>T</em> be the enumeration of model trees and <em>N</em> be the enumeration of inferred trees, then for each replicate we calculated the:</p>


<ol class="order">

<li>NTR - size of <em>N</em> (0, 5×10<sup>6</sup>);</li>

<li>FP<sub>TOP</sub> – fraction of topologies in <em>N</em> that do not exist in <em>T</em> (0, 1);</li>

<li>FN<sub>TOP</sub> – fraction of topologies in <em>T</em> that do not exist in <em>N</em> (0, 1);</li>

<li>FP – fraction of trees in <em>N</em> that do not exist in <em>T</em> (0, 1);</li>

<li>FN – fraction of trees in <em>T</em> that do not exist in <em>N</em> (0, 1);</li>

<li>Mean branch length difference between matching branches.</li>

</ol><a id="article1.body1.sec2.sec1.sec3.p4" name="article1.body1.sec2.sec1.sec3.p4"></a><p>where (#,#) indicates the range of each statistic and FP are false positives (type I error) and FN are false negatives (type II error). Additionally, we calculated several other statistics (see <a href="#pone.0001913.s001">Data S1</a>):</p>


<ol class="alpha-lower">

<li>Mean RF score between each tree in <em>N</em> and each tree in <em>T</em> (0, 1);</li>

<li>Mean BS distance between each tree in <em>N</em> and each tree in <em>T</em> (0, ∞);</li>

<li>Mean RF for false positives and false negatives;</li>

<li>Mean BS for false positives and false negatives;</li>

</ol><a id="article1.body1.sec2.sec1.sec3.p5" name="article1.body1.sec2.sec1.sec3.p5"></a><p>For measure 1 above, we calculated the median across all 1000 replicates for each method and simulation scenario. For measures 2–5, we plotted the mean (with Standard Error) across all replicates. Measure 6 is calculated as follows: for each tree <em>t</em> in <em>T</em> compared with each tree <em>n</em> in <em>N</em>, for each branch that exists in both <em>n</em> (call it <em>b<sub>n</sub></em>) and <em>t</em> (call it <em>b<sub>t</sub></em>) (meaning that <em>b<sub>n</sub></em> splits the terminal nodes of <em>n</em> into the same two disjoint subsets as <em>b<sub>t</sub></em> in <em>t</em>) we compute the mean of <em>l</em>(<em>b<sub>t</sub></em>) - <em>l</em>(<em>b</em><sub>n</sub>) where <em>l</em>(<em>b</em>) is the length of branch <em>b</em>. However, when interpreting the results of this measure one should remember that this is the average of all branches that were actual matches (meaning that if the method found very few matches, with very similar branch lengths between true and inferred trees, it will appear to do better than if it inferred many of the same branches with larger differences in branch lengths. For measures 6 and <em>a</em>–<em>d</em> above, we plotted the distribution for the 1000 replicates using box-and-whisker plots displaying the median, first and second quartiles, and outliers (points further than 3/2 times the inter-quartile range of the first and third quartiles). Additionally, we performed a set of paired Mann-Whitney tests to determine whether the results from each method were significantly different. We used an experiment-wise error rate of 0.05 with the Dunn-Sidak multiple test correction.</p>
<a id="article1.body1.sec2.sec1.sec3.p6" name="article1.body1.sec2.sec1.sec3.p6"></a><p>We also considered several measures designed specifically for networks (both maximum likelihood measures <a href="#pone.0001913-Jin1">[27]</a> and extensions of a bipartite measure to networks <a href="#pone.0001913-Moret1">[28]</a>), but each of these requires a rooted phylogenetic network, which only one of the methods we tested (SHB) provides, therefore precluding its use here. Our choice of metric does have some deficiencies (RF tends to reward lack of resolution, since a star tree will receive no penalty for false positives, only penalties for false negatives, see <a href="#pone.0001913-Rannala1">[29]</a>). Unresolved trees are penalized by their false negatives (and thus will have lower accuracy) but highly reticulated networks (i.e., those that imbed numerous trees) will have increased false negatives (due to inferring many incorrect branches). Furthermore, it is perhaps not ideal to give all possible trees imbedded within a network equal weight when comparing two phylogenetic networks (since some trees will be much more likely than others); however, we feel using all imbedded trees captures the essence of both accuracy (whether the simulated history was represented) and precision (how many inferred trees do we have to look at to find the true history), even though the exact number of trees and averages across all imbedded trees should be interpreted carefully with this in mind. Also, due to the poor performance of all methods on the medium and high recombination sets, particularly in inferring correct branch lengths, we included measure 6 above to help distinguish how well the individual branch length estimates compared to the true branch lengths.</p>
<a id="article1.body1.sec2.sec1.sec3.p7" name="article1.body1.sec2.sec1.sec3.p7"></a><p>Finally, we were also interested in the broad scale effect of the type of data on the performance of each method. We measured the relationship between characteristics of the simulated data and the inferred trees using the Spearman correlation coefficient (ρ). We considered the relationship between the number of inferred trees and the number of unique simulated haplotypes used to infer those trees. In the sets with recombination, we also measured the relationship of the number of simulated trees in <em>T</em> with the number of model topologies found. In addition, we measured the relationship between the number of inferred trees and the number of trees simulated in <em>T</em> in the recombination sets.</p>

<h5>Network Methods Evaluated.</h5><a id="article1.body1.sec2.sec1.sec4.p1" name="article1.body1.sec2.sec1.sec4.p1"></a><p>We evaluated ten different approaches commonly used to infer evolutionary relationships at the intraspecific level, including two traditional bifurcating tree-building approaches and eight network building approaches. The bifurcating tree approaches employed in this study were maximum parsimony (MP) <a href="#pone.0001913-CavalliSforza1">[30]</a> and neighbor-joining (NJ) <a href="#pone.0001913-Saitou1">[31]</a> as implemented in PAUP* v4.b10 <a href="#pone.0001913-Swofford1">[32]</a>. MP was run with “maxtrees” set to 5,000,000 and 1000 random sequence additions and NJ trees were built using uncorrected sequence distances. The implicit network building approaches tested were the union of maximum parsimony trees (UMP) <a href="#pone.0001913-Cassens1">[12]</a> as implemented in the software CombineTrees, statistical parsimony (SP) <a href="#pone.0001913-Templeton2">[33]</a> as implemented in the software TCS v1.17 <a href="#pone.0001913-Clement1">[34]</a>, split decomposition (SD) as implemented in SplitsTree (also known as Jsplits) 4 beta 4 <a href="#pone.0001913-Huson1">[35]</a>, Neighbor-Net (NN) and unreduced median networks (MED) as implemented in SplitsTree4 version 4.7 <a href="#pone.0001913-Huson2">[36]</a>, reduced median-joining (RMD) <a href="#pone.0001913-Bandelt1">[37]</a> as implemented in Network version 4.2.0.1 <a href="#pone.0001913-Forster1">[38]</a> and minimum-spanning network (MSN) <a href="#pone.0001913-Excoffier1">[39]</a> as implemented in the software Arlequin v2.001 <a href="#pone.0001913-Schneider1">[40]</a>.</p>
<a id="article1.body1.sec2.sec1.sec4.p2" name="article1.body1.sec2.sec1.sec4.p2"></a><p>The explicit network building method tested seeks to calculate the upper bound on the minimum number of recombination events (and gene conversions) while simultaneously computing the most parsimonious tree, as implemented in the shrub-gc software (SHB) <a href="#pone.0001913-Song1">[41]</a>. The CombineTrees software takes as input all inferred MP trees and combines them into a single reticulated network merging branches, tip haplotypes, and interior haplotypes that are identical among all trees. CombineTrees was run as in <a href="#pone.0001913-Cassens1">[12]</a>, by randomizing the order of the input trees 10 times and picking the smallest network (<em>i.e.</em>, with the least number of branches). (Note this does not necessarily mean the smallest number of loops.) On some datasets, CombineTrees was unable to find a network for some orderings of the input trees. In these instances, we still used the smallest network, although in such instances there were less than 10 from which to choose. SD networks were built using default settings. NN was run with two configurations: first, using all defaults and then using a weight threshold set to the inverse of the input sequence length. In the latter case, splits with low support were not included, resulting in a more refined network, and branch lengths greater than or equal to one. We only included results from the former case, since they were much more accurate, even when accounting for the increase in FN. MED was run with default settings. For SP, the maximum connection limit was ignored, forcing all sequences to be connected in a single network. RMD and MSN were run with default settings. Shrub-gc was also run with default settings, but using as input the set of sites where only one or two nucleotide states was observed, since it requires biallelic site data. The resulting ancestral recombination graphs were converted to a list of trees in two manners. For the zero recombination sets, recombination edges were treated as branches defined by the sites derived from that edge's parent that differed from the other parent. For the simulations with recombination, the recombination edges were treated as defining alternative trees such that any tree could only contain one of the two edges associated with each inferred recombination event, resulting in 2<sup>n</sup> trees for a network with <em>n</em> recombination events (note, some of these trees may not be unique and only one copy of each tree was used in further analyses). For the SD, NN, and NJ methods, branch lengths were multiplied by the number of sites and rounded to the nearest integer.</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>No Recombination</h4>
<h5>Topological Type I Error.</h5><a id="article1.body1.sec3.sec1.sec1.p1" name="article1.body1.sec3.sec1.sec1.p1"></a><p>Topological false positive rate was measured as described above, and the mean over the 1000 replicates per dataset were plotted (<a href="#pone-0001913-g002">Figure 2</a>). All of the methods (except MSN which did noticeably worse) had roughly the same mean topological FP (less than 0.05) with low substitution rates. When the substitution rate was higher, MP had lowest mean topological FP (0.12–0.29), followed by UMP and NJ (0.14–0.39). MSN performed worst, with a mean topological FP always above 0.93. With low substitution rates, the methods performed slightly worse with an increasing number of unique sequences (or haplotypes). This decrease was much greater with higher substitution rates.</p>
<div class="figure" id="pone-0001913-g002"><div class="img"><a name="pone-0001913-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Mean fraction of false positive topologies (FP<sub>TOP</sub>) inferred (those topologies inferred but which did not match the simulated topology) and false positive trees (FP) without recombination.</span></strong></p><a id="article1.body1.sec3.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.sec1.fig1.caption1.p1"></a><p>The left margin shows the number of nucleotides in each simulated sequences. The top margin shows the number of sequences simulated. The right margin shows the substitution rate of the sequences.</p>
<span>doi:10.1371/journal.pone.0001913.g002</span></div>
<h5>Tree Type I Error.</h5><a id="article1.body1.sec3.sec1.sec2.p1" name="article1.body1.sec3.sec1.sec2.p1"></a><p>The mean tree false positive rate (over the 1000 replicates) is shown for each method on each set of data (<a href="#pone-0001913-g002">Figure 2</a>). With a low substitution rate, the mean branch score accuracy of all methods was roughly the same (except for MSN, which once again was significantly higher). All methods but MSN had a false positive rate of less than ~0.07 with a low substitution rate. The FP rates with higher substitution rates however were much worse. The lowest error rates in this case were achieved by MP (0.35–0.63). UMP had the next lowest mean FP (0.36–0.65), with SP and NJ following (0.40–0.76). The FP rate of MSN was again the worst, always above 0.94. Increasing the number of sequences substantially increased the mean false positives in all methods.</p>

<h5>Number of trees inferred.</h5><a id="article1.body1.sec3.sec1.sec3.p1" name="article1.body1.sec3.sec1.sec3.p1"></a><p>The previous two measures give us a sense of how many incorrect trees are inferred by a given method. We also measured the total number of unique trees (NTR) contained within each inferred network and the median of these totals for each method and simulation set (<a href="#pone-0001913-t002">Table 2</a>). At first glance, we can see that some methods generated many more trees than others. This means that while an inference may in fact include the true tree, it may do so by inferring thousands of potential trees. When the substitution rate was low, all methods had a median NTR of 1, while MP inferred fewer trees on average when the substitution rate was higher (except NJ which always infers a single tree). SP and UMP had lower NTR on average on all simulations. When the substitution rate was high, SD, NN and MSN had median NTR above 100 in most simulations, but the remaining methods typically had NTR below 10, except SHB and MED in sets 11 and 12 where they too had significantly higher median NTR. We also found that at the higher substitution rate, several methods (SD, MSN, NN, SHB, MED, RMD) inferred highly reticulated networks (containing more than 5,000,000 trees) on one or more of the simulated histories (<a href="#pone-0001913-t002">Table 2</a>). The number of networks containing this large NTR ranges from a few to over 672 for NN in set 12 (<a href="#pone-0001913-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001913-t002"><div class="img"><a name="pone-0001913-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>The median number of trees (NTR) inferred by each method for each set of simulated sequences (number of simulations out of 1000 that were used).</span></strong></p><span>doi:10.1371/journal.pone.0001913.t002</span></div><a id="article1.body1.sec3.sec1.sec3.p2" name="article1.body1.sec3.sec1.sec3.p2"></a><p>All methods (except NJ, which infers a single tree in all cases) showed a highly significant positive correlation between the NTR and the number of unique sequences. This was especially true for SD and MSN, with Spearman's ρ = 0.664 and 0.637 respectively. NN, SHB, and MED were slightly less correlated, with ρ = 0.59, 0.527 and 0.509 respectively. For SP, MP, RMD and UMP, the correlation was lower with ρ = 0.356 for SP and ρ = 0.316 for MP, RMD and UMP.</p>

<h5>Topological Type II Error.</h5><a id="article1.body1.sec3.sec1.sec4.p1" name="article1.body1.sec3.sec1.sec4.p1"></a><p>We also computed the mean topological FN rate and the mean tree FN rate (<a href="#pone-0001913-g003">Figure 3</a>). (Note, the fraction of true positives is simply 1-FN). The mean topological FN with a low substitution rate was not qualitatively different from the topological FP (<a href="#pone-0001913-g002">Figure 2</a>). However, a higher substitution rate resulted in FN patterns different from the FP rates. The first conspicuous but expected result was that all methods (except NJ) had lower FN when compared to FP since there was a single model tree, but all methods (except NJ) could potentially infer more than a single tree, which may not match the model topology (increasing the FP rate). This effect was most dramatic on the SD method, whose mean FP was sometimes double its FN rate. NN also had much larger FP than FN rates with a higher substitution rate.</p>
<div class="figure" id="pone-0001913-g003"><div class="img"><a name="pone-0001913-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Mean fraction of false negative topologies (FN<sub>TOP</sub>) inferred (fraction of times that the simulated topology was not correctly inferred) and false positive trees (FN) without recombination.</span></strong></p><a id="article1.body1.sec3.sec1.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec1.sec4.fig1.caption1.p1"></a><p>The left margin shows the number of nucleotides in each simulated sequences. The top margin shows the number of sequences simulated. The right margin shows the substitution rate of the sequences. Note, the fraction of true positives is 1 - FN.</p>
<span>doi:10.1371/journal.pone.0001913.g003</span></div>
<h5>Tree Type II Error.</h5><a id="article1.body1.sec3.sec1.sec5.p1" name="article1.body1.sec3.sec1.sec5.p1"></a><p>We also calculated the mean tree FN (<a href="#pone-0001913-g003">Figure 3</a>). When the substitution rate was low, the results were again very similar to the tree FP. The most pronounced difference was again the lower mean FN of SD and NN, compared to FP. Smaller increases in FN, relative to FP occurred in all other methods (except NJ, in which FP and FN are by definition equal with only one simulated tree). The tree FN of UMP and MP were the lowest (0.31–0.57), followed by SP (0.36–0.70).</p>



<h4>Recombination</h4>
<h5>Topological Type I Error.</h5><a id="article1.body1.sec3.sec2.sec1.p1" name="article1.body1.sec3.sec2.sec1.p1"></a><p>With the simulated history potentially containing multiple distinct trees for different sites, we now can potentially recover more than one model tree or topology. In order to evaluate how well the simulated topologies were inferred, we again calculated the mean topological FP rate (<a href="#pone-0001913-g004">Figure 4</a>). Thus, if all inferred topologies were found in the list of true simulated trees for a given method in all 1000 simulations for a particular set of parameters, this value would be 0.0. In the simulations with no rate heterogeneity, MP and UMP exhibited the lowest mean topological FP at 0.31 and 0.34, respectively, for low recombination, but MP, UMP and NN had the lowest topological FP (0.87) with the medium recombination rate. With site rate heterogeneity, the relative topological FP among methods was similar to the constant rate simulations, but all methods had much higher mean topological FP. MP had the lowest FP with both low and medium recombination (0.69 and 0.94 respectively). Notably, all of the methods had mean topological FP rates of 1.0 at high recombination.</p>
<div class="figure" id="pone-0001913-g004"><div class="img"><a name="pone-0001913-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Mean fraction of false positive topologies (FP<sub>TOP</sub>) inferred (those topologies inferred but which did not match the simulated topology) and false positive trees (FP) with recombination.</span></strong></p><a id="article1.body1.sec3.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec1.fig1.caption1.p1"></a><p>The top row was simulated with a constant substitution rate among sites, while the bottom row was simulated with gamma distributed site-rate heterogeneity. The top margin shows the recombination rate.</p>
<span>doi:10.1371/journal.pone.0001913.g004</span></div>
<h5>Tree Type I Error.</h5><a id="article1.body1.sec3.sec2.sec2.p1" name="article1.body1.sec3.sec2.sec2.p1"></a><p>We also computed the mean tree FP rate in the presence of recombination (<a href="#pone-0001913-g004">Figure 4</a>). It is noteworthy that none of the methods achieved a mean tree FP less than 0.95. In fact, for medium and high recombination rates, on average all methods had tree FP of 1.0. With low recombination, NN had the lowest mean tree FP (0.95 and 0.97 with homogenous and heterogeneous rate variation, respectively).</p>

<h5>Number of trees inferred.</h5><a id="article1.body1.sec3.sec2.sec3.p1" name="article1.body1.sec3.sec2.sec3.p1"></a><p>In order to compare the number of trees inferred by each method, <a href="#pone-0001913-t002">Table 2</a> shows the median NTR for each method on each recombination simulation (sets 13–18, <a href="#pone-0001913-t001">Table 1</a>). As expected, NJ was the lowest, since it always infers one and only one tree. MP and SP on average inferred the second and third lowest NTR respectively, except in set 16 (<a href="#pone-0001913-t001">Table 1</a>), when SHB inferred fewer trees than SP on average. The median NTR for MP and SP was always less than five. The difference between the mean number of trees inferred with MP and SP was not significant in sets 14, 15 or 18 (<a href="#pone-0001913-t001">Table 1</a>). Some of the median values for NTR for NN and MED could not be accurately calculated due to the inability to enumerate all trees within their output (see <a href="#pone-0001913-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec2.sec3.p2" name="article1.body1.sec3.sec2.sec3.p2"></a><p>All methods (except NJ which infers a single tree in all cases, and NN probably due to our inability to enumerate all trees in many of the simulations with recombination) showed a highly significant positive correlation of the number of trees inferred with the number of unique sequences simulated when all recombination sets were analyzed together. The smallest spearman correlation was with NN (ρ = 0.042) followed by SD, with ρ = 0.051 and SHB had the greatest correlation with ρ = 0.351. RMD, MP, MSN, UMP, SP and MED had ρ = 0.218, 0.195, 0.176, 0.155, 0.147, 0.117, respectively. The number of trees inferred by a method when the sequences have undergone recombination should ideally be positively correlated with the number of simulated trees. The association between the number of trees inferred and the number of trees simulated with recombination were statistically significant for all methods tested. SHB had the largest correlation with ρ = 0.829. RMD, MED, UMP, MP, NN, SD and SP had ρ = 0.612, 0.368, 0.304, 0.299, 0.296, 0.167 and 0.123, respectively. Surprisingly, MSN had a ρ = −0.076 (meaning that as the number of trees simulated increases, the number of trees inferred by MSN tends to decrease).</p>

<h5>Topological Type II Error.</h5><a id="article1.body1.sec3.sec2.sec4.p1" name="article1.body1.sec3.sec2.sec4.p1"></a><p>In order to assess the fraction of false negative inferences (FN) of each method in finding the simulated topology in the presence of recombination, we calculated the mean topological FN for each method on each simulation set (<a href="#pone-0001913-g005">Figure 5</a>). (Note, the fraction of true positives is simply 1-FN). NN had the lowest mean topological FN in all the low and medium recombination simulations. In the constant site substitution rate simulations, NN had mean topological FN of 0.56 and 0.90 for low and medium recombination respectively and 0.65 and 0.94 with heterogeneous substitution rates among sites. For the highest recombination rate, all methods had mean topological FN of 1.0.</p>
<div class="figure" id="pone-0001913-g005"><div class="img"><a name="pone-0001913-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Mean fraction of false negative topologies (FN<sub>TOP</sub>) inferred (fraction of times that the simulated topology was not correctly inferred) and false positive trees (FN) with recombination.</span></strong></p><a id="article1.body1.sec3.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec4.fig1.caption1.p1"></a><p>The top row was simulated with a constant substitution rate among sites, while the bottom row was simulated with gamma distributed site-rate heterogeneity. The top margin shows the recombination rate. Note, the fraction of true positives is 1 - FN.</p>
<span>doi:10.1371/journal.pone.0001913.g005</span></div>
<h5>Tree Type II Error.</h5><a id="article1.body1.sec3.sec2.sec5.p1" name="article1.body1.sec3.sec2.sec5.p1"></a><p>Similarly, the fraction of false negative inferences of the simulated trees was calculated. (Again, the fraction of true positives is simply 1-FN). The mean tree FN across each set of 1000 simulations is shown in <a href="#pone-0001913-g005">Figure 5</a>. Mean tree FN was very near 1.0 for all methods tested (over 0.93), and like the topological FN, NN was the lowest with low and medium recombination.</p>

<h5>Comparison of branch lengths.</h5><a id="article1.body1.sec3.sec2.sec6.p1" name="article1.body1.sec3.sec2.sec6.p1"></a><p>Since the error rates for inferring true trees (both FN and FP) with recombination were so high (see mean tree FN and FP in <a href="#pone-0001913-g004">Figure 4</a> and <a href="#pone-0001913-g005">Figure 5</a>), we present another view of the accuracy of estimating the branch lengths with recombination. For each branch in an inferred tree that was found to be a match (e.g., induced the same split) in a simulated tree, we calculated the difference between the true branch length and the inferred branch length. We then averaged this value over all matching branches between compared trees and over all 1000 simulations for each set (<a href="#pone-0001913-g006">Figure 6</a>). Thus, if the method consistently overestimates branch lengths, we will have a negative mean, and if it underestimates branch lengths, we have a positive mean. All methods overestimated branch lengths with recombination (<a href="#pone-0001913-g006">Figure 6</a>) and without (see <a href="#pone.0001913.s001">Data S1</a>). The branch lengths estimated by SHB were the closest to the true matching branch's length on average, and had the smallest variance. However, one should be careful interpreting this result, since we can only compare two matching branch's lengths if they actually exist in both model and inference. Thus, a method that had very few matching branches would not necessarily perform poorly in this respect (i.e., if it only inferred one correct branch, with the correct branch length, it would have zero variance and zero mean, even though it would likely have large FP and FN).</p>
<div class="figure" id="pone-0001913-g006"><div class="img"><a name="pone-0001913-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001913" data-uri="info:doi/10.1371/journal.pone.0001913.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001913.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001913.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001913.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001913.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Mean branch length difference between matching branches with recombination.</span></strong></p><a id="article1.body1.sec3.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec2.sec6.fig1.caption1.p1"></a><p>The top row was simulated with a constant substitution rate among sites, while the bottom row was simulated with gamma distributed site-rate heterogeneity. The top margin shows the recombination rate. Vertical lines separate those methods that were significantly different in a paired Mann-Whitney test (see Measures of Performance).</p>
<span>doi:10.1371/journal.pone.0001913.g006</span></div>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3>
<h4>No Recombination</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The common use of phylogenetic inference in population studies warrants a thorough analysis of the strengths and weaknesses of network methods. This study was designed to assess the relative performance of ten commonly used network methods on data simulated in a variety of biologically meaningful scenarios. Our analyses have shown that not all methods fare equally well in many circumstances. One important but expected finding is that increasing substitution rate resulted in a significant increase in error (both topologically and in terms of inferring the correct branch lengths) in all methods. Increasing the number of sites also resulted in an increase in mean topological error rates for all methods except MP and UMP when the number of sequences was 20 or 50. When taking into account branch lengths, all methods had increased error as the number of sites increased. We speculate that this decrease in accuracy with an increasing number of sites is a result of the increasing number of unique haplotypes that result from longer sequences. Since we found that an increase in the number of unique haplotypes correlated with an increase in the number of inferred trees, which increases the type II error, we also speculate that with a larger number of sequences to connect there is more uncertainty as to how they are related (and more internal nodes), and thus the error rates are higher. Increasing the number of sequences also resulted in an increase in error for all methods. Overall, MP had at least as low, if not lower error rates than the other methods tested under all circumstances. With low substitution rates, however, the difference in accuracy of MP over UMP, NJ, SP, and SD in general faded away. At higher substitution rates, MP was always significantly less erroneous than all other methods.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>One major advantage, however, of the network approaches, is the ability to display ambiguity in the inference in a single graphical representation. MP does not provide such a view, beyond the total number of equally parsimonious trees. However, the method of UMP was designed specifically to facilitate visualization of the set of MP trees in a single graphical representation. The UMP method, by definition, will always result in the same or lower FN as MP with one caveat: increasing the number of trees imbedded in the network may increase the FP rate. This minor limitation is apparent with higher substitution rates when the FP rate is increased in UMP as compared to MP. The accuracy of SD, NN and MSN suffered, although the overall accuracy of the other methods (except NJ) also decreased somewhat due to ambiguity (higher FP rates). It is apparent from the results that MP and/or UMP should be preferred when lower error rates (e.g., higher reconstruction accuracy) is the goal, particularly with relatively divergent sequences. The relative level of topological error as compared to overall tree error was slightly different between all methods, but again, MP and UMP generally had lower error than the rest.</p>


<h4>Recombination</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>As only one of the inference methods tested (SHB) explicitly accounts for recombination, it is not surprising that the results on the sets simulated with recombination were quite poor. However, even SHB performed poorly in the presence of recombination. Furthermore, as the recombination rate increased, error rates increased to 100% in all methods. When the branch length accuracy was considered (tree FP and FN), no method had mean error below 0.94 (see <a href="#pone-0001913-g004">Figure 4</a> and <a href="#pone-0001913-g005">Figure 5</a>). This was attributable in part to the difficulty in estimating branch lengths for the entire length of a set of sequences when only certain sites within those sequences actually share the same history, as is the case with recombination. When we only measured each method's topological error, it was much lower, though still not as low as in the simulations without recombination. NN, MP, UMP and NJ were the least erroneous in inferring topology with recombination (see <a href="#pone-0001913-g004">Figure 4</a> and <a href="#pone-0001913-g005">Figure 5</a>). It was much less clear which method did best when estimated branch lengths in the presence of recombination. In order to better judge the branch length estimates for the methods tested, we calculated the mean difference between branches that matched between the inferred and simulated trees (<a href="#pone-0001913-g006">Figure 6</a> with recombination and <a href="#pone.0001913.s001">Data S1</a> for no recombination). This gives us some sense of whether a method consistently over or under estimated the branch lengths. SHB had the smallest mean difference in branch lengths and the smallest variance, which may imply that it does quite well in estimating certain branches accurately, but still can not globally infer all branches in the simulated histories.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Another important consideration in the recombination inferences is the proportion of sites that support a given tree. One might value accuracy in inferring the tree or trees that underlie a large number of sites over one that is only representing a few sites. Trees could be weighted based on this to achieve a more useful measure of accuracy, penalizing a method more for not finding those trees that are supported by a majority of the sites, for example. This should be an area of additional focus in future comparisons and benchmarking of new methods. However, our results indicate that estimates of branch lengths from data with recombination should not be relied upon, at least at the level of the full tree. In addition, rate variation increased the error of all methods significantly. While these results do not look promising for inferring histories from sequences that have undergone recombination in their history, they certainly highlight the importance of detecting recombination within a sample of sequences before confidence is placed on any histories inferred using these methods. Alternatively, methods that explicitly account for recombination during inference could be used, although SHB as tested here showed no general advantage over the other methods (although the strong correlation between the number of trees inferred by SHB with the number of unique simulated histories in the ARG and lower individual branch length inference error, do give some hope for better characterizing its sources of error).</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The method that was consistently the least erroneous in our simulations was MP and the related UMP method. While, nearly all methods exhibited similar performance on sequences with low substitution rates, MP and UMP outperformed the other methods in terms of both lower topological and overall tree error in nearly every case. The development of the UMP method to combine maximum parsimony trees into a single network appears to be quite appropriate. Particularly, if the UMP method can be refined in such a way as to 1) not depend on the order of the input trees, 2) not choke on particular sets of trees ordered in a particular manner, 3) reduce the ambiguity to only that ambiguity existing in the input trees and 4) express the confidence of particular branches within the combined network, it looks very promising for the accurate estimation and visualization of intraspecific phylogenies. While there were some instances where UMP inferred highly reticulated networks on the simulations with recombination, it was not as common as with NN or SD (see <a href="#pone-0001913-t002">Table 2</a>).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>As for the other methods tested, the biggest drawback for SD and NN was their highly reticulated representations and their less accurate estimation of branch lengths. However, NN did have slightly lower FN in the recombination sets, indicating that it may still have some potential to capture correct relationships. Since both SD and NN aim to represent the compatible splits in the sequence data, resolution is not necessarily their primary goal, but our results indicate that quite frequently the model tree is not included within their representations, a finding that needs closer inspection.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>RMD, MED, and SHB performed fine with low substitution rates, but were significantly less accurate than the best methods when the substitution rate was higher. SHB, in spite of being designed to deal with recombined sequences, performed poorly, even in sets with recombination, although the number of trees it inferred was highly correlated with the number of unique trees simulated in the ancestral recombination graph and its average branch length estimation with recombination was promising. SHB's increased error rates might be due in part to its requirement for binary state alleles as input, reducing the amount of information available for reconstruction.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>NJ performed marginally well, although its obvious drawback is its inability to represent ambiguity, either by reticulations, or by inferring multiple trees. This could possibly be addressed by building NJ trees from various partitions of the alignment, and combining the results in a manner similar to UMP, or by including ties or suboptimal NJ trees <a href="#pone.0001913-Rzhetsky1">[e.g. 42]</a>, although NN also uses an agglomerative approach similar to NJ, but appears to do worse than NJ in most of our simulations.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>SP's performance was not as good as the tree approaches (MP, UMP and NJ) under higher substitution rates, but in most of our simulations, it had lower error than the other network methods (except for topological FN). This gives us hope for improvement, particularly with these benchmarks on which to assess its deficiencies. One possible reason for the method's lower accuracy could be the effect of ignoring the parsimony limit and forcing the software to connect all sequences. This act violates the theoretical advantage of SP over MP, but was necessary in order to compare the performance of all methods on equal ground. One potential improvement of SP (or more accurately, the TCS software) would be the ability to use the statistical parsimony connection limit to connect the less divergent sequences, followed by use of MP to complete the disconnected networks, as was originally proposed by Templeton <em>et al.</em> <a href="#pone.0001913-Templeton3">[43]</a>.</p>
<a id="article1.body1.sec4.sec3.p6" name="article1.body1.sec4.sec3.p6"></a><p>Finally, the performance of MSN was by far the worst on all simulated data sets. This finding, as pointed out by Cassens <em>et al.</em> <a href="#pone.0001913-Cassens1">[12]</a>, is likely due to the inability of the minimum spanning network method to infer unsampled historical individuals. The Median Joining Network (RMD) reconstruction method had much better performance than MSN, due to its ability to infer ancestral haplotypes. We strongly discourage the use of MSN for any analyses that rely on the topology of the inferred relationships. Furthermore, when the phylogenetic relationship of any set of sequences is being inferred, it is important that several methods be used and their inferences inspected and compared for discrepancies.</p>
<a id="article1.body1.sec4.sec3.p7" name="article1.body1.sec4.sec3.p7"></a><p>It is clear that there is much room for improvement in the development of methods that infer the historical relationship of intraspecific sequences, particularly when the sequences might have undergone some level of recombination. We look forward to experimenting to increase the accuracy of the existing methods and developing novel methods to more accurately deal with such data.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001913"></div><a name="pone.0001913.s001" id="pone.0001913.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001913.s001">Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Document describing and displaying additional information (Summary statistics of RF and BS from all simulations).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001913.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.87 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Thomas Mailund and Barbara Holland for their helpful comments and suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KC DP SW. Performed the experiments: SW. Analyzed the data: SW. Contributed reagents/materials/analysis tools: SW. Wrote the paper: KC DP SW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001913-Pagel1" id="pone.0001913-Pagel1"></a>Pagel M (1999) Inferring the historical patterns of biological evolution. Nature  401: 877–884.  <ul class="find" data-citedArticleID="1084902" data-doi="10.1038/44766"><li><a href="http://dx.doi.org/10.1038/44766" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+the+historical+patterns+of+biological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+the+historical+patterns+of+biological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001913-Posada1" id="pone.0001913-Posada1"></a>Posada D, Crandall KA (2002) The effect of recombination on the accuracy of phylogeny estimation. Journal of Molecular Evolution  54: 396–402.  <ul class="find" data-citedArticleID="1084906" data-doi="10.1007/s00239-001-0034-9"><li><a href="http://dx.doi.org/10.1007/s00239-001-0034-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+recombination+on+the+accuracy+of+phylogeny+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+recombination+on+the+accuracy+of+phylogeny+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001913-Schierup1" id="pone.0001913-Schierup1"></a>Schierup MH, Hein J (2000) Consequences of recombination on traditional phylogenetic analysis. Genetics  156: 879–891.  <ul class="find" data-citedArticleID="1084920"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Consequences+of+recombination+on+traditional+phylogenetic+analysis.&amp;auth=&amp;atitle=Consequences+of+recombination+on+traditional+phylogenetic+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consequences+of+recombination+on+traditional+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consequences+of+recombination+on+traditional+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001913-Bakker1" id="pone.0001913-Bakker1"></a>Bakker FTAugust2005 Reconstructing patterns of reticulate evolution in angiosperms: what can we do? Taxon  54: 593–604(512).  <ul class="find" data-citedArticleID="1084854" data-doi="10.2307/25065417"><li><a href="http://dx.doi.org/10.2307/25065417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+patterns+of+reticulate+evolution+in+angiosperms%3A+what+can+we+do%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+patterns+of+reticulate+evolution+in+angiosperms%3A+what+can+we+do%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001913-Posada2" id="pone.0001913-Posada2"></a>Posada D, Crandall KA (2001) Intraspecific gene genealogies: trees grafting into networks. Trends in Ecology and Evolution  16: 37–45.  <ul class="find" data-citedArticleID="1084908" data-doi="10.1016/s0169-5347(00)02026-7"><li><a href="http://dx.doi.org/10.1016/s0169-5347(00)02026-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intraspecific+gene+genealogies%3A+trees+grafting+into+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intraspecific+gene+genealogies%3A+trees+grafting+into+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001913-Schaal1" id="pone.0001913-Schaal1"></a>Schaal BA, Olsen KM (2000) Gene genealogies and population variation in plants. Proc Natl Acad Sci U S A  97: 7024–7029.  <ul class="find" data-citedArticleID="1084918" data-doi="10.1073/pnas.97.13.7024"><li><a href="http://dx.doi.org/10.1073/pnas.97.13.7024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+genealogies+and+population+variation+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+genealogies+and+population+variation+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001913-WainHobson1" id="pone.0001913-WainHobson1"></a>Wain-Hobson S, Renoux-Elbe C, Vartanian JP, Meyerhans A (2003) Network analysis of human and simian immunodeficiency virus sequence sets reveals massive recombination resulting in shorter pathways. J Gen Virol  84: 885–895.  <ul class="find" data-citedArticleID="1084936" data-doi="10.1099/vir.0.18894-0"><li><a href="http://dx.doi.org/10.1099/vir.0.18894-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+analysis+of+human+and+simian+immunodeficiency+virus+sequence+sets+reveals+massive+recombination+resulting+in+shorter+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+analysis+of+human+and+simian+immunodeficiency+virus+sequence+sets+reveals+massive+recombination+resulting+in+shorter+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001913-Templeton1" id="pone.0001913-Templeton1"></a>Templeton AR, Maxwell T, Posada D, Stengard JH, Boerwinkle E, et al.  (2005) Tree scanning: a method for using haplotype trees in phenotype/genotype association studies. Genetics  169: 441–453.  <ul class="find" data-citedArticleID="1084930" data-doi="10.1534/genetics.104.030080"><li><a href="http://dx.doi.org/10.1534/genetics.104.030080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tree+scanning%3A+a+method+for+using+haplotype+trees+in+phenotype%2Fgenotype+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tree+scanning%3A+a+method+for+using+haplotype+trees+in+phenotype%2Fgenotype+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001913-Hillis1" id="pone.0001913-Hillis1"></a>Hillis DM (1995) Approaches for assessing phylogenetic accuracy. Systematic Biology  44: 3–16.  <ul class="find" data-citedArticleID="1084876"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Approaches+for+assessing+phylogenetic+accuracy.&amp;auth=&amp;atitle=Approaches+for+assessing+phylogenetic+accuracy." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approaches+for+assessing+phylogenetic+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approaches+for+assessing+phylogenetic+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001913-Crandall1" id="pone.0001913-Crandall1"></a>Crandall KA (1994) Intraspecific cladogram estimation: Accuracy at higher levels of divergence. Systematic Biology  43: 222–235.  <ul class="find" data-citedArticleID="1084868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Intraspecific+cladogram+estimation%3A+Accuracy+at+higher+levels+of+divergence.&amp;auth=&amp;atitle=Intraspecific+cladogram+estimation%3A+Accuracy+at+higher+levels+of+divergence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intraspecific+cladogram+estimation%3A+Accuracy+at+higher+levels+of+divergence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intraspecific+cladogram+estimation%3A+Accuracy+at+higher+levels+of+divergence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001913-Hillis2" id="pone.0001913-Hillis2"></a>Hillis DM, Bull JJ, White ME, Badgett MR, Molineux IJ (1992) Experimental phylogenetics: Generation of a known phylogeny. Science  255: 589–591.  <ul class="find" data-citedArticleID="1084878" data-doi="10.1126/science.1736360"><li><a href="http://dx.doi.org/10.1126/science.1736360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+phylogenetics%3A+Generation+of+a+known+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+phylogenetics%3A+Generation+of+a+known+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001913-Cassens1" id="pone.0001913-Cassens1"></a>Cassens I, Mardulyn P, Milinkovitch MC (2005) Evaluating intraspecific “network” construction methods using simulated sequence data: do existing algorithms outperform the global maximum parsimony approach? Systematic Biology  54: 363–372.  <ul class="find" data-citedArticleID="1084860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evaluating+intraspecific+%3Fnetwork%3F+construction+methods+using+simulated+sequence+data%3A+do+existing+algorithms+outperform+the+global+maximum+parsimony+approach%3F&amp;auth=&amp;atitle=Evaluating+intraspecific+%3Fnetwork%3F+construction+methods+using+simulated+sequence+data%3A+do+existing+algorithms+outperform+the+global+maximum+parsimony+approach%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+intraspecific+%E2%80%9Cnetwork%E2%80%9D+construction+methods+using+simulated+sequence+data%3A+do+existing+algorithms+outperform+the+global+maximum+parsimony+approach%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+intraspecific+%E2%80%9Cnetwork%E2%80%9D+construction+methods+using+simulated+sequence+data%3A+do+existing+algorithms+outperform+the+global+maximum+parsimony+approach%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001913-Cassens2" id="pone.0001913-Cassens2"></a>Cassens I, Van Waerebeek K, Best PB, Crespo EA, Reyes JC, et al.  (2003) The phylogeography of dusky dolphins (Lagenorhynchus obscurus): A critical examination of network methods and rooting procedures. Molecular Ecology  12: 1781–1792.  <ul class="find" data-citedArticleID="1084862" data-doi="10.1046/j.1365-294x.2003.01876.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2003.01876.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phylogeography+of+dusky+dolphins+%28Lagenorhynchus+obscurus%29%3A+A+critical+examination+of+network+methods+and+rooting+procedures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phylogeography+of+dusky+dolphins+%28Lagenorhynchus+obscurus%29%3A+A+critical+examination+of+network+methods+and+rooting+procedures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001913-Hudson1" id="pone.0001913-Hudson1"></a>Hudson RR (1990) Gene genealogies and the coalescent process. Oxford Surveys Evol Biol  7: 1–44.  <ul class="find" data-citedArticleID="1084882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+genealogies+and+the+coalescent+process.&amp;auth=&amp;atitle=Gene+genealogies+and+the+coalescent+process." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+genealogies+and+the+coalescent+process." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+genealogies+and+the+coalescent+process.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001913-Jukes1" id="pone.0001913-Jukes1"></a>Jukes TH, Cantor CR (1969)  Evolution of protein molecules. In: Munro HM, editor. Mammalian Protein Metabolism. New York, NY.: Academic Press. pp. 21–132.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0001913-Yang1" id="pone.0001913-Yang1"></a>Yang Z (1996) Among-site rate variation and its impact on phylogenetic analyses. Trends In Ecology &amp; Evolution  11: 367–372.  <ul class="find" data-citedArticleID="1084938" data-doi="10.1016/0169-5347(96)10041-0"><li><a href="http://dx.doi.org/10.1016/0169-5347(96)10041-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Among-site+rate+variation+and+its+impact+on+phylogenetic+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Among-site+rate+variation+and+its+impact+on+phylogenetic+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001913-Li1" id="pone.0001913-Li1"></a>Li W-H (1997) Molecular Evolution. Sunderland, MA.: Sinauer Associates, Inc..   <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0001913-PerezLosada1" id="pone.0001913-PerezLosada1"></a>Perez-Losada M, Browne EB, Madsen A, Wirth T, Viscidi RP, et al.  (2006) Population genetics of microbial pathogens estimated from multilocus sequence typing (MLST) data. Infection, Genetics and Evolution  6: 97–112.  <ul class="find" data-citedArticleID="1084904" data-doi="10.1016/j.meegid.2005.02.003"><li><a href="http://dx.doi.org/10.1016/j.meegid.2005.02.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genetics+of+microbial+pathogens+estimated+from+multilocus+sequence+typing+%28MLST%29+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genetics+of+microbial+pathogens+estimated+from+multilocus+sequence+typing+%28MLST%29+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001913-Griffiths1" id="pone.0001913-Griffiths1"></a>Griffiths RC (1981) Neutral two-locus multiple allele models with recombination. Theoretical Population Biology  19: 169–186.  <ul class="find" data-citedArticleID="1084874" data-doi="10.1016/0040-5809(81)90016-2"><li><a href="http://dx.doi.org/10.1016/0040-5809(81)90016-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutral+two-locus+multiple+allele+models+with+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutral+two-locus+multiple+allele+models+with+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001913-Hudson2" id="pone.0001913-Hudson2"></a>Hudson RR (1983) Properties of a neutral allele model with intragenic recombination. Theor Popul Biol  23: 183–201.  <ul class="find" data-citedArticleID="1084884" data-doi="10.1016/0040-5809(83)90013-8"><li><a href="http://dx.doi.org/10.1016/0040-5809(83)90013-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properties+of+a+neutral+allele+model+with+intragenic+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properties+of+a+neutral+allele+model+with+intragenic+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001913-Nguyen1" id="pone.0001913-Nguyen1"></a>Nguyen N, Nguyen C, Sung W-K (2007) Fast algorithms for computing the tripartition-based distance between phylogenetic networks. Journal of Combinatorial Optimization  13: 223–242.  <ul class="find" data-citedArticleID="1084900" data-doi="10.1007/s10878-006-9025-5"><li><a href="http://dx.doi.org/10.1007/s10878-006-9025-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+algorithms+for+computing+the+tripartition-based+distance+between+phylogenetic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+algorithms+for+computing+the+tripartition-based+distance+between+phylogenetic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001913-Cardona1" id="pone.0001913-Cardona1"></a>Cardona G, Rossello F, Valiente G (2007) Comparison of Tree-Child Phylogenetic Networks.  <ul class="find" data-citedArticleID="1084858" data-doi="10.1109/tcbb.2007.70270"><li><a href="http://dx.doi.org/10.1109/tcbb.2007.70270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+Tree-Child+Phylogenetic+Networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+Tree-Child+Phylogenetic+Networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001913-Holland1" id="pone.0001913-Holland1"></a>Holland B, Conner G, Huber K, Moulton V (2007) Imputing supertrees and supernetworks from quartets. Syst Biol  56: 57–67.  <ul class="find" data-citedArticleID="1084880" data-doi="10.1007/11851561_15"><li><a href="http://dx.doi.org/10.1007/11851561_15" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Imputing+supertrees+and+supernetworks+from+quartets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Imputing+supertrees+and+supernetworks+from+quartets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001913-Shioura1" id="pone.0001913-Shioura1"></a>Shioura A, Tamura A, Uno T (1997) An optimal algorithm for scanning all spanning trees of undirected graphs. SIAM Journal on Computing  26: 678–692.  <ul class="find" data-citedArticleID="1084924" data-doi="10.1137/s0097539794270881"><li><a href="http://dx.doi.org/10.1137/s0097539794270881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+optimal+algorithm+for+scanning+all+spanning+trees+of+undirected+graphs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+optimal+algorithm+for+scanning+all+spanning+trees+of+undirected+graphs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001913-Robinson1" id="pone.0001913-Robinson1"></a>Robinson DF, Foulds LR (1981) Comparison of phylogenetic trees. Mathematical Biosciences  53: 131–147.  <ul class="find" data-citedArticleID="1084912" data-doi="10.1016/0025-5564(81)90043-2"><li><a href="http://dx.doi.org/10.1016/0025-5564(81)90043-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001913-Kuhner1" id="pone.0001913-Kuhner1"></a>Kuhner MK, Felsenstein J (1994) A simulation comparison of phylogeny algorithms under equal and unequal evolutionary rates. Molecular Biology and Evolution  11: 459–468.  <ul class="find" data-citedArticleID="1084894"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simulation+comparison+of+phylogeny+algorithms+under+equal+and+unequal+evolutionary+rates.&amp;auth=&amp;atitle=A+simulation+comparison+of+phylogeny+algorithms+under+equal+and+unequal+evolutionary+rates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simulation+comparison+of+phylogeny+algorithms+under+equal+and+unequal+evolutionary+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simulation+comparison+of+phylogeny+algorithms+under+equal+and+unequal+evolutionary+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001913-Jin1" id="pone.0001913-Jin1"></a>Jin G, Nakhleh L, Snir S, Tuller T (2006) Maximum likelihood of phylogenetic networks. Bioinformatics  22: 2604–2611.  <ul class="find" data-citedArticleID="1084890" data-doi="10.1093/bioinformatics/btl452"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+of+phylogenetic+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+of+phylogenetic+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001913-Moret1" id="pone.0001913-Moret1"></a>Moret BME, Nakhleh L, Warnow T, Linder CR, Tholse A, et al.  (2004) Phylogenetic networks: modeling, reconstructibility, and accuracy. Computational Biology and Bioinformatics, IEEE/ACM Transactions on  1: 13–23.  <ul class="find" data-citedArticleID="1084898" data-doi="10.1109/tcbb.2004.10"><li><a href="http://dx.doi.org/10.1109/tcbb.2004.10" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+networks%3A+modeling%2C+reconstructibility%2C+and+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+networks%3A+modeling%2C+reconstructibility%2C+and+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001913-Rannala1" id="pone.0001913-Rannala1"></a>Rannala B, Huelsenbeck JP, Yang Z, Nielsen R (1998) Taxon sampling and the accuracy of large phylogenies. Syst Biol  47: 702–710.  <ul class="find" data-citedArticleID="1084910"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Taxon+sampling+and+the+accuracy+of+large+phylogenies.&amp;auth=&amp;atitle=Taxon+sampling+and+the+accuracy+of+large+phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Taxon+sampling+and+the+accuracy+of+large+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Taxon+sampling+and+the+accuracy+of+large+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001913-CavalliSforza1" id="pone.0001913-CavalliSforza1"></a>Cavalli-Sforza LL, Edwards AWF (1967) Phylogenetic analysis: models and estimation procedures. Evolution  32: 550–570.  <ul class="find" data-citedArticleID="1084864" data-doi="10.2307/2406616"><li><a href="http://dx.doi.org/10.2307/2406616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis%3A+models+and+estimation+procedures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis%3A+models+and+estimation+procedures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001913-Saitou1" id="pone.0001913-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Molecular Biology and Evolution  4: 406–425.  <ul class="find" data-citedArticleID="1084916"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001913-Swofford1" id="pone.0001913-Swofford1"></a>Swofford DL (2002) PAUP*: Phylogenetic Analysis Using Parsimony (*and other methods). 4.0 beta 10 ed. Sunderland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0001913-Templeton2" id="pone.0001913-Templeton2"></a>Templeton AR, Crandall KA, Sing CF (1992) A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation. Genetics  132: 619–633.  <ul class="find" data-citedArticleID="1084932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation.&amp;auth=&amp;atitle=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001913-Clement1" id="pone.0001913-Clement1"></a>Clement M, Posada D, Crandall KA (2000) TCS: a computer program to estimate gene genealogies. Molecular Ecology  9: 1657–1659.  <ul class="find" data-citedArticleID="1084866" data-doi="10.1046/j.1365-294x.2000.01020.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.01020.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TCS%3A+a+computer+program+to+estimate+gene+genealogies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TCS%3A+a+computer+program+to+estimate+gene+genealogies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001913-Huson1" id="pone.0001913-Huson1"></a>Huson DH (1998) SplitsTree: analyzing and visualizing evolutionary data. Bioinformatics  14: 68–73.  <ul class="find" data-citedArticleID="1084886" data-doi="10.1093/bioinformatics/14.1.68"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.1.68" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SplitsTree%3A+analyzing+and+visualizing+evolutionary+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SplitsTree%3A+analyzing+and+visualizing+evolutionary+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001913-Huson2" id="pone.0001913-Huson2"></a>Huson DH, Bryant D (2006) Application of phylogenetic networks in evolutionary studies. Mol Biol Evol  23: 254–267.  <ul class="find" data-citedArticleID="1084888" data-doi="10.1093/molbev/msj030"><li><a href="http://dx.doi.org/10.1093/molbev/msj030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+phylogenetic+networks+in+evolutionary+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+phylogenetic+networks+in+evolutionary+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001913-Bandelt1" id="pone.0001913-Bandelt1"></a>Bandelt HJ, Forster P, Sykes BC, Richards MB (1995) Mitochondrial portraits of human populations using median networks. Genetics  141: 743–753.  <ul class="find" data-citedArticleID="1084856"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mitochondrial+portraits+of+human+populations+using+median+networks.&amp;auth=&amp;atitle=Mitochondrial+portraits+of+human+populations+using+median+networks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+portraits+of+human+populations+using+median+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+portraits+of+human+populations+using+median+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001913-Forster1" id="pone.0001913-Forster1"></a>Forster M, Forster P, Watson J (2007) Network version 4.2.0.1: A software for population genetics data analysis. 4.2.0.1 ed: Fluxus Technology Ltd 1999–2007..   <ul class="find" data-citedArticleID="1084872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Network+version+4.2.0.1%3A+A+software+for+population+genetics+data+analysis.&amp;auth=&amp;atitle=Network+version+4.2.0.1%3A+A+software+for+population+genetics+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+version+4.2.0.1%3A+A+software+for+population+genetics+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+version+4.2.0.1%3A+A+software+for+population+genetics+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001913-Excoffier1" id="pone.0001913-Excoffier1"></a>Excoffier L, Smouse PE (1994) Using allele frequencies and geographic subdivision to reconstruct gene trees within a species: Molecular variance parsimony. Genetics  136: 343–359.  <ul class="find" data-citedArticleID="1084870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Using+allele+frequencies+and+geographic+subdivision+to+reconstruct+gene+trees+within+a+species%3A+Molecular+variance+parsimony.&amp;auth=&amp;atitle=Using+allele+frequencies+and+geographic+subdivision+to+reconstruct+gene+trees+within+a+species%3A+Molecular+variance+parsimony." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+allele+frequencies+and+geographic+subdivision+to+reconstruct+gene+trees+within+a+species%3A+Molecular+variance+parsimony." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+allele+frequencies+and+geographic+subdivision+to+reconstruct+gene+trees+within+a+species%3A+Molecular+variance+parsimony.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001913-Schneider1" id="pone.0001913-Schneider1"></a>Schneider S, Roessli D, Excoffier L (2000) ARLEQUIN version 2.000: A software for population genetics data analysis. 2.0 ed. Geneva: Genetics and Biometry Laboratory, University of Geneva.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001913-Song1" id="pone.0001913-Song1"></a>Song JS, Ding Z, Gusfield D, Langley CH, Wu Y (2006) Algorithms to Distinguish the Role of Gene-Conversion from Single-Crossover Recombination in the Derivation of SNP Sequences in Populations: Springer Berlin/Heidelberg.231–245.  <ul class="find" data-citedArticleID="1084926" data-doi="10.1007/11732990_20"><li><a href="http://dx.doi.org/10.1007/11732990_20" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Algorithms+to+Distinguish+the+Role+of+Gene-Conversion+from+Single-Crossover+Recombination+in+the+Derivation+of+SNP+Sequences+in+Populations%3A+Springer+Berlin%2FHeidelberg." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Algorithms+to+Distinguish+the+Role+of+Gene-Conversion+from+Single-Crossover+Recombination+in+the+Derivation+of+SNP+Sequences+in+Populations%3A+Springer+Berlin%2FHeidelberg.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001913-Rzhetsky1" id="pone.0001913-Rzhetsky1"></a>Rzhetsky A, Nei M (1993) Theoretical foundation of the minimum-evolution method of phylogenetic inference. Mol Biol Evol  10: 1073–1095.  <ul class="find" data-citedArticleID="1084914"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Theoretical+foundation+of+the+minimum-evolution+method+of+phylogenetic+inference.&amp;auth=&amp;atitle=Theoretical+foundation+of+the+minimum-evolution+method+of+phylogenetic+inference." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theoretical+foundation+of+the+minimum-evolution+method+of+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theoretical+foundation+of+the+minimum-evolution+method+of+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001913-Templeton3" id="pone.0001913-Templeton3"></a>Templeton AR, Crandall KA, Sing CF (1992) A cladistic analysis of phenotypic associations with haplotypes inferred from restriction endonuclease mapping and DNA sequence data. III. Cladogram estimation. Genetics  132: 619–633.  <ul class="find" data-citedArticleID="1084934"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation.&amp;auth=&amp;atitle=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cladistic+analysis+of+phenotypic+associations+with+haplotypes+inferred+from+restriction+endonuclease+mapping+and+DNA+sequence+data.+III.+Cladogram+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.XML" value="124400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.PDF" value="441060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g001.PNG_L" value="401580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g001.PNG_M" value="35185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g001.PNG_S" value="7406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g001.TIF" value="1096864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g001.PNG_I" value="17078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t001.PNG_L" value="168700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t001.PNG_M" value="98178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t001.PNG_S" value="13291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t001.TIF" value="883998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t001.PNG_I" value="30310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g002.PNG_L" value="256400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g002.PNG_M" value="39128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g002.PNG_S" value="6650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g002.TIF" value="1024738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g002.PNG_I" value="13430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t002.PNG_L" value="267946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t002.PNG_M" value="155234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t002.PNG_S" value="20975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t002.TIF" value="1526006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.t002.PNG_I" value="53178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g003.PNG_L" value="247760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g003.PNG_M" value="37704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g003.PNG_S" value="6590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g003.TIF" value="967866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g003.PNG_I" value="13044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g004.PNG_L" value="181655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g004.PNG_M" value="23747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g004.PNG_S" value="5413"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g004.TIF" value="1189458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g004.PNG_I" value="8596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g005.PNG_L" value="180776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g005.PNG_M" value="22968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g005.PNG_S" value="5435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g005.TIF" value="1218148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g005.PNG_I" value="8189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g006.PNG_L" value="294498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g006.PNG_M" value="48010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g006.PNG_S" value="6133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g006.TIF" value="872338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.g006.PNG_I" value="18142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001913.s001.DOC" value="1872896"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001913&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001913" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001913&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001913&volume=&issue=&title=A Comparison of Phylogenetic Network Methods Using Computer Simulation&author_name=Steven%20M.%20Woolley%2C%20David%20Posada%2C%20Keith%20A.%20Crandall&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001913" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913&amp;t=A%20Comparison%20of%20Phylogenetic%20Network%20Methods%20Using%20Computer%20Simulation" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913&title=A%20Comparison%20of%20Phylogenetic%20Network%20Methods%20Using%20Computer%20Simulation&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913&amp;title=A%20Comparison%20of%20Phylogenetic%20Network%20Methods%20Using%20Computer%20Simulation" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001913&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Comparison of Phylogenetic Network Methods Using Computer Simulation';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001913';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Comparison%20of%20Phylogenetic%20Network%20Methods%20Using%20Computer%20Simulation http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001913" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001913" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+networks%22" title="Search for articles in the subject area:'Computer networks'"><div class="flagText">Computer networks</div></a>
              <div data-categoryid="8643" data-articleid="26650"
                   data-categoryname="Computer networks"
                   class="flagImage" title="Flag 'Computer networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26650"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="31807" data-articleid="26650"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="26650"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="26650"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="26650"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Simulation+and+modeling%22" title="Search for articles in the subject area:'Simulation and modeling'"><div class="flagText">Simulation and modeling</div></a>
              <div data-categoryid="8375" data-articleid="26650"
                   data-categoryname="Simulation and modeling"
                   class="flagImage" title="Flag 'Simulation and modeling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="26650"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=971'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=909'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9853&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=8781">Link to more data</a><br>Posted by wooo</p>
      <p><a href="/annotation/listThread.action?root=14887">The simulated sequences and trees for with recombination are available here</a><br>Posted by wooo</p>
      <p><a href="/annotation/listThread.action?root=1987">Referee comments: Referee 3 (Thomas Mailund)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
