

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Ethnic Related Selection for an ADH Class I Variant within East Asia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001881"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001881" />

    <meta name="citation_title" content="Ethnic Related Selection for an ADH Class I Variant within East Asia"/>
    <meta itemprop="name" content="Ethnic Related Selection for an ADH Class I Variant within East Asia"/>

      <meta name="citation_author" content="Hui Li"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>
            <meta name="citation_author_institution" content="MOE Key Laboratory of Contemporary Anthropology and Center for Evolutionary Biology, School of Life Sciences and Institutes of Biomedical Sciences, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="Sheng Gu"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Xiaoyun Cai"/>
            <meta name="citation_author_institution" content="MOE Key Laboratory of Contemporary Anthropology and Center for Evolutionary Biology, School of Life Sciences and Institutes of Biomedical Sciences, Fudan University, Shanghai, China"/>
      <meta name="citation_author" content="William C. Speed"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Andrew J. Pakstis"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Efim I. Golub"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Judith R. Kidd"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Kenneth K. Kidd"/>
            <meta name="citation_author_institution" content="Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001881.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1881"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Pharmacogenetics of alcohol metabolism and alcoholism.; citation_author=DP Agarwal; citation_author=HW Goedde; citation_journal_title=Pharmacogenetics; citation_volume=2; citation_number=1; citation_pages=48-62; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Research advances in ethanol metabolism.; citation_author=VA Ramchandani; citation_author=WF Bosron; citation_author=TK Li; citation_journal_title=Pathol Biol (Paris); citation_volume=49; citation_number=2; citation_pages=676-682; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetics of human alcohol-metabolizing enzymes.; citation_author=A Yoshida; citation_author=LC Hsu; citation_author=M Yasunami; citation_journal_title=Prog Nucleic Acid Res Mol Biol; citation_volume=40; citation_number=3; citation_pages=255-287; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Class I Alcohol Dehydorgenase is highly expressed in normal human mammary epithelium but not in invasive breast cancer: Implications for breast carcinogenesis.; citation_author=EA Triano; citation_author=LB Slusher; citation_author=TA Atkins; citation_author=JT Beneski; citation_author=SA Gestl; citation_journal_title=Cancer Research; citation_volume=63; citation_number=4; citation_pages=3092-3100; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genes and gene expression in the brain of the alcoholic.; citation_author=PR Dodd; citation_author=PF Foley; citation_author=ST Buckley; citation_author=AL Eckert; citation_author=DJ Innes; citation_journal_title=Addict Behav; citation_volume=29; citation_number=5; citation_pages=1295-1309; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Expression and localization of human alcohol and aldehyde dehydrogenase enzymes in skin.; citation_author=C Cheung; citation_author=CK Smith; citation_author=JO Hoog; citation_author=SA Hotchkiss; citation_journal_title=Biochem Biophys Res Commun.; citation_volume=261; citation_number=6; citation_pages=100-107; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Human hepatic alcohol and aldehyde dehydrogenases: genetic polymorphism and activities.; citation_author=CT Yao; citation_author=CS Liao; citation_author=SJ Yin; citation_journal_title=Proc Natl Sci Counc Repub China B.; citation_volume=21(3); citation_number=7; citation_pages=106-111; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Human stomach alcohol and aldehyde dehydrogenases: comparison of expression pattern and activities in alimentary tract.; citation_author=SJ Yin; citation_author=CS Liao; citation_author=CW Wu; citation_author=TT Li; citation_author=LL Chen; citation_journal_title=Gastroenterology; citation_volume=112; citation_number=8; citation_pages=766-775; citation_date=1997; " />
      <meta name="citation_reference" content="citation_number=9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A global perspective on genetic variation at the ADH genes reveals unusual patterns of linkage disequilibrium and diversity.; citation_author=MV Osier; citation_author=AJ Pakstis; citation_author=H Soodyall; citation_author=D Comas; citation_author=D Goldman; citation_journal_title=Am J Hum Genet; citation_volume=71; citation_number=10; citation_pages=84-99; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Association of alcohol dehydrogenase genes with alcohol dependence: a comprehensive analysis.; citation_author=HJ Edenberg; citation_author=X Xuei; citation_author=HJ Chen; citation_author=H Tian; citation_author=LF Wetherill; citation_journal_title=Hum Mol Genet; citation_volume=15; citation_number=11; citation_pages=1539-1549; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence of Positive Selection on a Class I ADH Locus.; citation_author=Y Han; citation_author=S Gu; citation_author=H Oota; citation_author=MV Osier; citation_author=AJ Pakstis; citation_journal_title=Am J Hum Genet; citation_volume=80; citation_number=12; citation_pages=441-456; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional relevance of human adh polymorphism.; citation_author=CJ Eriksson; citation_author=T Fukunaga; citation_author=T Sarkola; citation_author=WJ Chen; citation_author=CC Chen; citation_journal_title=Alcohol Clin Exp Res; citation_number=13; citation_pages=157S-163S; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Impact of the Alcohol-Dehydrogenase (ADH) 1C and ADH1B polymorphisms on drinking behavior in nonalcoholic Japanese.; citation_author=K Matsuo; citation_author=A Hiraki; citation_author=K Hirose; citation_author=H Ito; citation_author=T Suzuki; citation_journal_title=Hum Mutat; citation_volume=28; citation_number=14; citation_pages=506-510; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Allelic variation at alcohol metabolism genes (ADH1B, ADH1C, ALDH2) and alcohol dependence in an American Indian population.; citation_author=CJ Mulligan; citation_author=RW Robin; citation_author=MV Osier; citation_author=N Sambuughin; citation_author=LG Goldfarb; citation_journal_title=Hum Genet; citation_volume=113; citation_number=15; citation_pages=325-336; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Interaction between the functional polymorphisms of the alcohol-metabolism genes in protection against alcoholism.; citation_author=CC Chen; citation_author=RB Lu; citation_author=YC Chen; citation_author=MF Wang; citation_author=YC Chang; citation_journal_title=Am J Hum Genet; citation_volume=65; citation_number=16; citation_pages=795-807; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genome-wide search for genes affecting the risk for alcohol dependence.; citation_author=T Reich; citation_author=HJ Edenberg; citation_author=A Goate; citation_author=JT Williams; citation_author=JP Rice; citation_journal_title=Am J Med Genet; citation_volume=81; citation_number=17; citation_pages=207-215; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A genome screen of maximum number of drinks as an alcoholism phenotype.; citation_author=NL Saccone; citation_author=JM Kwon; citation_author=J Corbett; citation_author=A Goate; citation_author=N Rochberg; citation_journal_title=Am J Med Genet B Neuropsychiatr Genet; citation_volume=96; citation_number=18; citation_pages=632-637; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evidence for genetic linkage to alcohol dependence on chromosomes 4 and 11 from an autosome-wide scan in an American Indian population.; citation_author=JC Long; citation_author=WC Knowler; citation_author=RL Hanson; citation_author=RW Robin; citation_author=M Urbanek; citation_journal_title=Am J Med Genet; citation_volume=81; citation_number=19; citation_pages=216-221; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular genetic underpinnings of human substance abuse vulnerability: likely contributions to understanding addiction as a mnemonic process.; citation_author=GR Uhl; citation_journal_title=Neuropharmacology; citation_volume=47; citation_number=20; citation_pages=140-147; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Low frequency of the ADH2*2 allele among Atayal natives of Taiwan with alcohol use disorders.; citation_author=HR Thomasson; citation_author=DW Crabb; citation_author=HJ Edenberg; citation_author=TK Li; citation_author=HG Hwu; citation_journal_title=Alcohol Clin Exp Res; citation_volume=18; citation_number=21; citation_pages=640-643; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Alcohol and aldehyde dehydrogenase genotypes and drinking behavior of Chinese living in Shanghai.; citation_author=T Muramatsu; citation_author=ZC Wang; citation_author=YR Fang; citation_author=KB Hu; citation_author=H Yan; citation_journal_title=Hum Genet; citation_volume=96; citation_number=22; citation_pages=151-154; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Alcohol and aldehyde dehydrogenase polymorphisms and the risk for alcoholism.; citation_author=S Higuchi; citation_author=S Matsushita; citation_author=M Murayama; citation_author=S Takagi; citation_author=M Hayashida; citation_journal_title=Am J Psychiatry; citation_volume=152; citation_number=23; citation_pages=1219-1221; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Polymorphism of ADH and ALDH genes among four ethnic groups in China and effects upon the risk for alcoholism.; citation_author=YC Shen; citation_author=JH Fan; citation_author=HJ Edenberg; citation_author=TK Li; citation_author=YH Cui; citation_journal_title=Alcohol Clin Exp Res; citation_volume=21; citation_number=24; citation_pages=1272-1277; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium at the ADH2 and ADH3 loci and risk of alcoholism.; citation_author=M Osier; citation_author=AJ Pakstis; citation_author=JR Kidd; citation_author=JF Lee; citation_author=SJ Yin; citation_journal_title=Am J Hum Genet; citation_volume=64; citation_number=25; citation_pages=1147-1157; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genetic polymorphism of alcohol dehydrogenase in europeans: the ADH2*2 allele decreases the risk for alcoholism and is associated with ADH3*1.; citation_author=E Borras; citation_author=C Coutelle; citation_author=A Rosell; citation_author=F Fernandez-Muixi; citation_author=M Broch; citation_journal_title=Hepatology; citation_volume=31; citation_number=26; citation_pages=984-989; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Association of the ADH2*2 allele with reduced ethanol consumption in Jewish men in Israel: a pilot study.; citation_author=YD Neumark; citation_author=Y Friedlander; citation_author=HR Thomasson; citation_author=TK Li; citation_journal_title=J Stud Alcohol; citation_volume=59; citation_number=27; citation_pages=133-139; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Alcohol and ADH2 in Israel: Ashkenazis, Sephardics, and Recent Russian Immigrants.; citation_author=D Hasin; citation_author=E Aharonovich; citation_author=XH Liu; citation_author=Z Mamman; citation_author=K Matseoane; citation_journal_title=Am J Psychiatry; citation_volume=159; citation_number=28; citation_pages=1432-1434; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ADH genotypes and alcohol use and dependence in Europeans.; citation_author=JB Whitfield; citation_author=BN Nightingale; citation_author=KK Bucholz; citation_author=PA Madden; citation_author=AC Heath; citation_journal_title=Alcohol Clin Exp Res; citation_volume=22; citation_number=29; citation_pages=1463-1469; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Geographically Separate Increases in the Frequency of the Derived ADH1B*47His Allele in Eastern and Western Asia.; citation_author=H Li; citation_author=N Mukherjee; citation_author=U Soundararajan; citation_author=Z Tárnok; citation_author=C Barta; citation_journal_title=Am J Hum Genet; citation_volume=81; citation_number=30; citation_pages=842-846; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Distribution of ADH2 and ALDH2 genotypes in different populations.; citation_author=HW Goedde; citation_author=DP Agarwal; citation_author=G Fritze; citation_author=D Meier-Tackmann; citation_author=S Singh; citation_journal_title=Hum Genet; citation_volume=88; citation_number=31; citation_pages=344-346; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Alcohol-metabolising genes and alcoholism among Taiwanese Han men: independent effect of ADH2, ADH3 and ALDH2.; citation_author=WJ Chen; citation_author=EW Loh; citation_author=YP Hsu; citation_author=CC Chen; citation_author=JM Yu; citation_journal_title=Br J Psychiatry; citation_volume=168; citation_number=32; citation_pages=762-767; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Alcohol dehydrogenase and aldehyde dehydrogenase genotypes and alcoholism among Taiwanese aborigines.; citation_author=WJ Chen; citation_author=EW Loh; citation_author=YP Hsu; citation_author=AT Cheng; citation_journal_title=Biol Psychiatry; citation_volume=41; citation_number=33; citation_pages=703-709; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Polymorphism study of seven SNPs at ADH genes in 15 Chinese populations.; citation_author=LL Ma; citation_author=YL Xue; citation_author=Y Liu; citation_author=Z Wang; citation_author=XB Cui; citation_journal_title=Hereditas; citation_volume=142; citation_number=34; citation_pages=103-111; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=RG Gordon; citation_number=35; citation_date=2005; citation_publisher=SIL International; " />
      <meta name="citation_reference" content="citation_title=The Cambridge History of Early Inner Asia.; citation_author=D Sinor; citation_number=36; citation_date=1990; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=History of the Turkic Peoples in the Pre-Islamic Period.; citation_author=HR Roemer; citation_number=37; citation_date=2000; citation_publisher=Klaus Schwarz Verlag; " />
      <meta name="citation_reference" content="citation_title=The Uighur Empire According to the T'ang Dynastic Histories.; citation_author=C Mackerras; citation_number=38; citation_date=1972; citation_publisher=Australian National University Press; " />
      <meta name="citation_reference" content="citation_title=Weiwuerzu lishi:shang bian (History of the Uighur: Vol. 1).; citation_author=ZX Liu; citation_number=39; citation_date=1985; citation_publisher=Minzu chubanshe; " />
      <meta name="citation_reference" content="citation_title=A discussion of rice-growing origin in the Nanling Mountains, South China.; citation_author=QH Weng; citation_journal_title=Tropical Geography; citation_volume=18(1); citation_number=40; citation_pages=72-80; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Agricultural Archeology in China its research development and main results.; citation_author=WN Gu; citation_journal_title=Agricultural Archaeology; citation_volume=21(1); citation_number=41; citation_pages=1-16; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Outlines of some topics on the origin of agriculture in Southeast Asia and South China.; citation_author=EZ Tong; citation_journal_title=Agricultural Archaeology; citation_volume=4(2); citation_number=42; citation_pages=29-38; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=A History of Thailand.; citation_author=C Baker; citation_author=P Phongpaichit; citation_number=43; citation_date=2005; citation_publisher=Chulalongkorn University; " />
      <meta name="citation_reference" content="citation_title=Differential Regulation of the Alcohol Dehydrogenase 1B (ADH1B) and ADH1C Genes by DNA Methylation and Histone Deacetylation.; citation_author=LO Dannenberg; citation_author=HJ Chen; citation_author=HJ Tian; citation_author=HJ Edenberg; citation_journal_title=Alcohol Clin Exp Res; citation_volume=30; citation_number=44; citation_pages=928-937; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Implications of biogeography of human populations for ‘race’ and medicine.; citation_author=SA Tishkoff; citation_author=KK Kidd; citation_journal_title=Nat Genet; citation_volume=36; citation_number=45; citation_pages=S21-7; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of East Asian mitochondrial DNA lineages inferred from complete sequences.; citation_author=QP Kong; citation_author=YG Yao; citation_author=C Sun; citation_author=HJ Bandelt; citation_author=CL Zhu; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=46; citation_pages=671-676; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Natives or immigrants: origin and migrations of modern humans in East Asia.; citation_author=L Jin; citation_author=B Su; citation_journal_title=Nat Rev Genet; citation_volume=1; citation_number=47; citation_pages=126-133; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Quaternary glacier in Eastern China and the climate fluctuation.; citation_author=YF Shi; citation_author=ZJ Cui; citation_author=JJ Li; citation_number=48; citation_date=1989; citation_publisher=Science Press; " />
      <meta name="citation_reference" content="citation_title=An exact test for population differentiation.; citation_author=M Raymond; citation_author=F Rousset; citation_journal_title=Evolution; citation_volume=49; citation_number=49; citation_pages=1280-1283; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Ethnic-related differences in the frequency distribution of genetic polymorphisms in the CYP1A1 and CYP1B1 genes in Japanese and Caucasian populations.; citation_author=K Inoue; citation_author=T Asao; citation_author=T Shimada; citation_journal_title=Xenobiotica; citation_volume=30; citation_number=50; citation_pages=285-295; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ethnic differences in intake and excretion of sodium, potassium, calcium and magnesium in South Africans.; citation_author=KE Charlton; citation_author=K Steyn; citation_author=NS Levitt; citation_author=JV Zulu; citation_author=D Jonathan; citation_journal_title=Eur J Cardiovasc Prev Rehabil; citation_volume=12; citation_number=51; citation_pages=355-362; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Breast cancer treatment among women of different ethnicity in Hawaii.; citation_author=BF Issell; citation_author=G Maskarinec; citation_author=I Pagano; citation_author=CC Gotay; citation_journal_title=Cancer Invest; citation_volume=23; citation_number=52; citation_pages=497-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Y chromosome haplotypes reveal prehistorical migrations to the Himalayas.; citation_author=B Su; citation_author=C Xiao; citation_author=R Deka; citation_author=MT Seielstad; citation_author=D Kangwanpong; citation_journal_title=Human Genetics; citation_volume=107; citation_number=53; citation_pages=582-590; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence supports demic diffusion of Han culture.; citation_author=B Wen; citation_author=H Li; citation_author=DR Lu; citation_author=XF Song; citation_author=F Zhang; citation_journal_title=Nature; citation_volume=431; citation_number=54; citation_pages=302-305; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Three major lineages of Asian Y chromosomes: implications for the peopling of east and southeast Asia.; citation_author=A Tajima; citation_author=IH Pan; citation_author=G Fucharoen; citation_author=S Fucharoen; citation_author=M Matsuo; citation_journal_title=Hum Genet; citation_volume=110; citation_number=55; citation_pages=80-88; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Use of autosomal loci for clustering individuals and populations of East Asian origin.; citation_author=J Kim; citation_author=P Verdu; citation_author=AJ Pakstis; citation_author=WC Speed; citation_author=JR Kidd; citation_journal_title=Hum Genet; citation_volume=117; citation_number=56; citation_pages=511-519; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Y chromosomes of Prehistoric People along the Yangtze River.; citation_author=H Li; citation_author=Y Huang; citation_author=LF Mustavich; citation_author=F Zhang; citation_author=JZ Tan; citation_journal_title=Hum Genet; citation_volume=122; citation_number=57; citation_pages=383-388; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=On the Origin of Ethnic Groups Speaking Austronesian Languages and their Historical Relationship with the Ethnic Groups in South China.; citation_author=P He; citation_journal_title=J Yunnan University for Nationalities; citation_volume=20; citation_number=58; citation_pages=45-48; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Primordial History: Brothers Stories of the Qiang. Time, Memory and History.; citation_author=MK Wang; citation_number=59; citation_date=1999; citation_publisher=The Institute of Ethnology, Academia Sinica; " />
      <meta name="citation_reference" content="citation_title=Joint effects of eNOS gene T-786C and ADH2 Arg47His polymorphisms on the risk of premature coronary artery disease.; citation_author=C Jia; citation_author=T Liu; citation_author=Z Liu; citation_author=M Li; citation_author=M Hu; citation_journal_title=Thromb Res; citation_volume=120; citation_number=60; citation_pages=679-684; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Alcohol consumption and cancer of the gastrointestinal tract.; citation_author=HK Seitz; citation_author=B Maurer; citation_author=F Stickel; citation_journal_title=Dig Dis; citation_volume=23; citation_number=61; citation_pages=297-303; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic polymorphisms of alcohol and aldehyde dehydrogenases and risk for esophageal and head and neck cancers.; citation_author=A Yokoyama; citation_author=T Omori; citation_journal_title=Jpn J Clin Oncol; citation_volume=33; citation_number=62; citation_pages=111-121; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Alcohol dehydrogenase 1C*1 allele is a genetic marker for alcohol-associated cancer in heavy drinkers.; citation_author=N Homann; citation_author=F Stickel; citation_author=IR Konig; citation_author=A Jacobs; citation_author=K Junghanns; citation_journal_title=Int J Cancer; citation_volume=118; citation_number=63; citation_pages=1998-2002; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The ADH1C polymorphism modifies the risk of squamous cell carcinoma of the head and neck associated with alcohol and tobacco use.; citation_author=ES Peters; citation_author=MD McClean; citation_author=M Liu; citation_author=EA Eisen; citation_author=N Mueller; citation_journal_title=Cancer Epidemiol Biomarkers Prev; citation_volume=14; citation_number=64; citation_pages=476-482; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Alcohol consumption, alcohol dehydrogenase 3 polymorphism, and colorectal adenomas.; citation_author=EW Tiemersma; citation_author=PA Wark; citation_author=MC Ocke; citation_author=A Bunschoten; citation_author=MH Otten; citation_journal_title=Cancer Epidemiol Biomarkers Prev; citation_volume=12; citation_number=65; citation_pages=419-425; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic polymorphism of alcohol dehydrogenase 3 in alcohol liver cirrhosis and in alcohol chronic pancreatitis.; citation_author=H Cichoz-Lach; citation_author=J Partycka; citation_author=I Nesina; citation_author=K Celinski; citation_author=M Slomka; citation_journal_title=Alcohol Alcohol; citation_volume=41; citation_number=66; citation_pages=14-17; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic epidemiology of ethanol metabolic enzymes: a role for selection.; citation_author=D Goldman; citation_author=MA Enoch; citation_journal_title=World Rev Nutr Diet; citation_volume=63; citation_number=67; citation_pages=143-160; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Trends in the incidence of cancer in Qidong, China, 1978-2002.; citation_author=JG Chen; citation_author=J Zhu; citation_author=DM Parkin; citation_author=YH Zhang; citation_author=JH Lu; citation_journal_title=Int J Cancer.; citation_volume=119; citation_number=68; citation_pages=1447-1454; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Accuracy of haplotype frequency estimation for biallelic loci, via the expectation-maximization algorithm for unphased diploid genotype data.; citation_author=D Fallin; citation_author=NJ Schork; citation_journal_title=Am J Hum Genet; citation_volume=67; citation_number=69; citation_pages=947-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Comprehensive human genome amplification using multiple displacement amplification.; citation_author=FB Dean; citation_author=S Hosono; citation_author=L Fang; citation_author=X Wu; citation_author=AF Faruqi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=70; citation_pages=5261-5266; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The International HapMap Project.; citation_journal_title=Nature; citation_volume=426; citation_number=71; citation_pages=789-796; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Towards fully automated genome-wide polymorphism screening.; citation_author=KJ Livak; citation_author=J Marmaro; citation_author=JA Todd; citation_journal_title=Nat Genet; citation_volume=9; citation_number=72; citation_pages=341-342; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Evolution in Mendelian populations.; citation_author=S Wright; citation_journal_title=Genetics; citation_volume=16; citation_number=73; citation_pages=97-159; citation_date=1931; " />
      <meta name="citation_reference" content="citation_title=A new statistical method for haplotype reconstruction from population data.; citation_author=M Stephens; citation_author=NJ Smith; citation_author=P Donnelly; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=74; citation_pages=978-989; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A comparison of Bayesian methods for haplotype reconstruction.; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=75; citation_pages=1162-1169; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=HAPLOT: a graphical comparison of haplotype blocks, tagSNP sets and SNP variation for multiple populations.; citation_author=S Gu; citation_author=AJ Pakstis; citation_author=KK Kidd; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=76; citation_pages=3938-3939; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Factor analysis by least-square and maximum-likelihood method.; citation_author=KG Jöreskog; citation_number=77; citation_date=1977; citation_publisher=John Wiley & Sons, Inc; " />
      <meta name="citation_reference" content="citation_title=Large-scale search for genes on which positive selection may operate.; citation_author=T Endo; citation_author=K Ikeo; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=78; citation_pages=685-690; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genome-wide detection and characterization of positive selection in human populations.; citation_author=PC Sabeti; citation_author=P Varilly; citation_author=B Fry; citation_author=J Lohmueller; citation_author=E Hostetter; citation_journal_title=Nature; citation_volume=449; citation_number=79; citation_pages=913-918; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A map of recent positive selection in the human genome.; citation_author=BF Voight; citation_author=S Kudaravalli; citation_author=X Wen; citation_author=JK Pritchard; citation_journal_title=PLoS Biol; citation_volume=4(3); citation_number=80; citation_pages=e72; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Detecting recent positive selection in the human genome from haplotype structure.; citation_author=PC Sabeti; citation_author=DE Reich; citation_author=JM Higgins; citation_author=HZ Levine; citation_author=DJ Richter; citation_journal_title=Nature; citation_volume=419; citation_number=81; citation_pages=832-837; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Median-joining networks for inferring intraspecific phylogenies.; citation_author=HJ Bandelt; citation_author=P Forster; citation_author=A Rohl; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=82; citation_pages=37-48; citation_date=1999; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Ethnic Related Selection for an ADH Class I Variant within East Asia"/>
    <meta name="twitter:description" content="BackgroundThe alcohol dehydrogenases (ADH) are widely studied enzymes and the evolution of the mammalian gene cluster encoding these enzymes is also well studied. Previous studies have shown that the ADH1B*47His allele at one of the seven genes in humans is associated with a decrease in the risk of alcoholism and the core molecular region with this allele has been selected for in some East Asian populations. As the frequency of ADH1B*47His is highest in East Asia, and very low in most of the rest of the world, we have undertaken more detailed investigation in this geographic region.Methodology/Principal FindingsHere we report new data on 30 SNPs in the ADH7 and Class I ADH region in samples of 24 populations from China and Laos. These populations cover a wide geographic region and diverse ethnicities. Combined with our previously published East Asian data for these SNPs in 8 populations, we have typed populations from all of the 6 major linguistic phyla (Altaic including Korean-Japanese and inland Altaic, Sino-Tibetan, Hmong-Mien, Austro-Asiatic, Daic, and Austronesian). The ADH1B genotyping data are strongly related to ethnicity. Only some eastern ethnic phyla or subphyla (Korean-Japanese, Han Chinese, Hmong-Mien, Daic, and Austronesian) have a high frequency of ADH1B*47His. ADH1B haplotype data clustered the populations into linguistic subphyla, and divided the subphyla into eastern and western parts. In the Hmong-Mien and Altaic populations, the extended haplotype homozygosity (EHH) and relative EHH (REHH) tests for the ADH1B core were consistent with selection for the haplotype with derived SNP alleles. In the other ethnic phyla, the core showed only a weak signal of selection at best.Conclusions/SignificanceThe selection distribution is more significantly correlated with the frequency of the derived ADH1B regulatory region polymorphism than the derived amino-acid altering allele ADH1B*47His. Thus, the real focus of selection may be the regulatory region. The obvious ethnicity-related distributions of ADH1B diversities suggest the existence of some culture-related selective forces that have acted on the ADH1B region."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001881.g008"/>

  <meta property="og:title" content="Ethnic Related Selection for an ADH Class I Variant within East Asia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5998'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1501'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2414&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001881">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001881" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001881&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Ethnic Related Selection for an <i>ADH</i> Class I Variant within East Asia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hui Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America, 
                  MOE Key Laboratory of Contemporary Anthropology and Center for Evolutionary Biology, School of Life Sciences and Institutes of Biomedical Sciences, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sheng Gu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaoyun Cai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  MOE Key Laboratory of Contemporary Anthropology and Center for Evolutionary Biology, School of Life Sciences and Institutes of Biomedical Sciences, Fudan University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William C. Speed, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew J. Pakstis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Efim I. Golub, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Judith R. Kidd, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth K. Kidd
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:Kenneth.Kidd@Yale.edu">Kenneth.Kidd@Yale.edu</a></p>

                <p>Affiliation:
                  Lab for Human Polymorphism Studies, Department of Genetics, School of Medicine, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001881</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001881-g001" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-t001" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g002" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g003" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g004" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g005" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g006" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g007" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001881-g008" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001881">Reader Comments (5)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001881" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The alcohol dehydrogenases (ADH) are widely studied enzymes and the evolution of the mammalian gene cluster encoding these enzymes is also well studied. Previous studies have shown that the <em>ADH1B*47His</em> allele at one of the seven genes in humans is associated with a decrease in the risk of alcoholism and the core molecular region with this allele has been selected for in some East Asian populations. As the frequency of <em>ADH1B*47His</em> is highest in East Asia, and very low in most of the rest of the world, we have undertaken more detailed investigation in this geographic region.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we report new data on 30 SNPs in the <em>ADH7</em> and Class I <em>ADH</em> region in samples of 24 populations from China and Laos. These populations cover a wide geographic region and diverse ethnicities. Combined with our previously published East Asian data for these SNPs in 8 populations, we have typed populations from all of the 6 major linguistic phyla (Altaic including Korean-Japanese and inland Altaic, Sino-Tibetan, Hmong-Mien, Austro-Asiatic, Daic, and Austronesian). The <em>ADH1B</em> genotyping data are strongly related to ethnicity. Only some eastern ethnic phyla or subphyla (Korean-Japanese, Han Chinese, Hmong-Mien, Daic, and Austronesian) have a high frequency of <em>ADH1B*47His</em>. <em>ADH1B</em> haplotype data clustered the populations into linguistic subphyla, and divided the subphyla into eastern and western parts. In the Hmong-Mien and Altaic populations, the extended haplotype homozygosity (EHH) and relative EHH (REHH) tests for the <em>ADH1B</em> core were consistent with selection for the haplotype with derived SNP alleles. In the other ethnic phyla, the core showed only a weak signal of selection at best.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The selection distribution is more significantly correlated with the frequency of the derived <em>ADH1B</em> regulatory region polymorphism than the derived amino-acid altering allele <em>ADH1B*47His</em>. Thus, the real focus of selection may be the regulatory region. The obvious ethnicity-related distributions of <em>ADH1B</em> diversities suggest the existence of some culture-related selective forces that have acted on the <em>ADH1B</em> region.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Li H, Gu S, Cai X, Speed WC, Pakstis AJ, et al.  (2008) Ethnic Related Selection for an <em>ADH</em> Class I Variant within East Asia. PLoS ONE 3(4):
          e1881.
            doi:10.1371/journal.pone.0001881</p><p><strong>Editor: </strong>Henry Harpending, University of Utah, United States of America</p><p><strong>Received:</strong> December 5, 2007; <strong>Accepted:</strong> February 25, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Li et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was funded primarily by US NIH grant AA009379 to KKK, and in part by USPHS grant GM057672 and NSF grant BCS0096588.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Historically, the alcohol dehydrogenases (ADH) have been among the most widely studied sets of enzymes along with their genes. Alcoholism, a complex genetic disorder that affects a large proportion of people, has been known for some time to be strongly associated with variants of alcohol dehydrogenase <a href="#pone.0001881-Agarwal1">[1]</a>–<a href="#pone.0001881-Yoshida1">[3]</a>. Alcohol dehydrogenase plays a role not only in alcohol metabolism but also in many other metabolic pathways, and thus forms of the enzyme exist in many organs <a href="#pone.0001881-Triano1">[4]</a>–<a href="#pone.0001881-Yin1">[8]</a>. The human <em>ADH</em> gene cluster is located on chromosome 4q23-24, and the several genes are clearly related evolutionarily. Sequentially this <em>ADH</em> cluster contains <em>ADH7, Class I ADH (1C, 1B, 1A), ADH6, ADH4, ADH5</em> <a href="#pone.0001881-HUGO1">[9]</a>, <a href="#pone.0001881-Osier1">[10]</a>. Hundreds of polymorphic sites have already been studied within the <em>ADH</em> cluster <a href="#pone.0001881-Edenberg1">[11]</a>, <a href="#pone.0001881-Han1">[12]</a>. Some polymorphic sites alter amino acids or lie in other functional regions; these can cause functional changes in the enzyme and result in different phenotypes <a href="#pone.0001881-Edenberg1">[11]</a>, <a href="#pone.0001881-Eriksson1">[13]</a>–<a href="#pone.0001881-Uhl1">[20]</a>. The Arg47His (rs1229984) polymorphism at <em>ADH1B</em> (previously <em>ADH2</em>) is a typical function-related polymorphism <a href="#pone.0001881-Thomasson1">[21]</a>, <a href="#pone.0001881-Muramatsu1">[22]</a>. The derived allele, coding for histidine, provides a well confirmed protection against alcoholism <a href="#pone.0001881-Higuchi1">[23]</a>–<a href="#pone.0001881-Whitfield1">[29]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Global investigation of <em>ADH1B</em> diversity shows a strong geographic distribution <a href="#pone.0001881-Osier1">[10]</a>, <a href="#pone.0001881-Li1">[30]</a>. Some variants of <em>ADH1B</em> appear specifically in some geographic regions <a href="#pone.0001881-Osier1">[10]</a>, <a href="#pone.0001881-Shen1">[24]</a>, <a href="#pone.0001881-Goedde1">[31]</a>–<a href="#pone.0001881-Chen3">[33]</a>. For instance, <em>ADH1B*47His</em> reaches very high frequencies almost exclusively in East Asian populations <a href="#pone.0001881-Osier1">[10]</a>, <a href="#pone.0001881-Han1">[12]</a>, <a href="#pone.0001881-Ma1">[34]</a> while fairly high frequencies occur in West Asia and North Africa; in contrast the allele is rare to absent in the rest of the world <a href="#pone.0001881-Li1">[30]</a>. This quite unusual geographic pattern argues for more detailed research. Linkage disequilibrium (LD) studies revealed evidence that the upstream region of <em>ADH1B</em> has been under positive selection in several East Asian populations such as Chinese, Koreans and Japanese, but the selective force remains unknown <a href="#pone.0001881-Han1">[12]</a>. There are still unsolved issues in relating the high frequency of the functional variant to positive selection in East Asian populations. For example, in some Austronesian (AU) populations, the derived allele frequency of the functional polymorphism is very high, but there is no evidence by the REHH test showing that the functional allele <em>ADH1B*47His</em> underwent selection <a href="#pone.0001881-Han1">[12]</a>. Thus, not only is the selective force unknown, but selection cannot explain the frequency of <em>ADH1B*47His</em> in all populations in which it is high. While absence of evidence in some populations may have been a power issue, further investigation is clearly needed.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Our previously studied populations belong to several ethnic phyla (same as linguistic phyla): the Korean-Japanese (KJ) subphylum of the Altaic phylum (Japanese, Koreans), the Chinese (also called Sinitic Han: SN) subphylum of the Sino-Tibetan phylum (Cantonese, Hakka and Minnam Taiwanese), the Taiwan subphylum of the AU phylum (Atayal, Amis), and the East Mon-Khmer subphylum of the Austro-Asiatic (AA) phylum (a small sample of Cambodians) <a href="#pone.0001881-Han1">[12]</a>. These eight populations are all located in the coastal region. According to the complicated landforms, climates, ethnic distributions, and population histories in East Asia, this set of population samples is not sufficiently representative of East Asia, especially not for the populations in the western region of East Asia such as the Tibetan, Uigur, Mongol, etc., populations. Based on the geographic distribution of the <em>ADH1B*47His</em> allele <a href="#pone.0001881-Li1">[30]</a>, we do not believe that selection happened everywhere in East Asia, certainly not to the same effect. Therefore, we are working to improve coverage of other East Asian ethnic groups and geographic regions to reveal the true histories of ADH genes <a href="#pone.0001881-Li1">[30]</a>. More detailed distributions of allele and haplotype frequencies will be essential for attempting to address the questions of when, where, and even how selection occurred, although the distributions alone will not allow definitive answers.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The cultural and ethnic diversity in East Asia is noticeable. For example, Daic (also called Tai-Kadai: TK) is the major phylum in the peninsula of Southeast Asia and southern East Asia. In northern East Asia, Altaic is the major phylum, but the KJ subphylum is an atypical branch. Inland Altaic (AT) subphyla such as Mongolian and Turkic are much more representative of Altaic. In the western side of East Asia, the Tibeto-Burman (TB) subphylum and the Hmong-Mien (HM) phylum are both dominant <a href="#pone.0001881-Gordon1">[35]</a>, <a href="#pone.0001881-Sinor1">[36]</a>. Different ethnic phyla not only have different languages and religions but also completely different life styles <a href="#pone.0001881-Sinor1">[36]</a>, <a href="#pone.0001881-Roemer1">[37]</a>. Almost all of those AT populations are nomandic tribes except Uigur, which switched to farming 1000 years ago <a href="#pone.0001881-Mackerras1">[38]</a>, <a href="#pone.0001881-Liu1">[39]</a>. The northern tribes of TB are all mainly pastoralists. TK, HM, SN, KJ, and AU populations have very long histories of farming <a href="#pone.0001881-Weng1">[40]</a>–<a href="#pone.0001881-Tong1">[42]</a>. Most of the AA (mainly Mon-Khmer) groups were hunter-gatherers historically, and began farming very recently <a href="#pone.0001881-Baker1">[43]</a>. Different life styles might have led to differences in the impacts of any selective forces, and might have influenced the distribution of <em>ADH</em> allele and haplotype frequencies.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>To determine the allele frequency distributions with greater geographic and ethnic precision, we have studied 24 more populations from different geographic areas of East Asia. These populations belong to different ethnic phyla and their major subphyla. The most important populations of TK, AT, HM, TB, and AA phyla are included (<a href="#pone-0001881-g001">Figure 1A, 1B</a> and <a href="#pone-0001881-t001">Table 1</a>). This type of sampling can minimize the bias of the geographic and ethnic distribution of the samples and the possible misreading of the results. With such data we may be able to identify selection on the relevant allele and to find relationships between haplotype pattern and selection, as much more genetic diversity may be found during this more comprehensive investigation in the focused East Asia region.</p>
<div class="figure" id="pone-0001881-g001"><div class="img"><a name="pone-0001881-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Locations of the populations and distributions of the <em>ADH1B</em> rs1229984 and rs3811801 derived allele frequencies.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Note: The map of part A showed the ethnic phyla in East Asia, and part B displayed the locations of the populations. Populations marked with stars were cited from literature <a href="#pone.0001881-Goedde1">[31]</a>, <a href="#pone.0001881-Ma1">[34]</a>. The codes of the star-marked populations are ISO639-3 codes. Populations shown by gray spots are previously published by our team <a href="#pone.0001881-Han1">[12]</a>. The colorful spots are the populations collected in this study. Part C is the distribution of the derived allele frequency of rs1229984 (<em>ADH1B*47His</em>). Part D is the distribution of the regulatory region polymorphism rs3811801 derived allele frequency.</p>
<span>doi:10.1371/journal.pone.0001881.g001</span></div><div class="figure" id="pone-0001881-t001"><div class="img"><a name="pone-0001881-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>General Information for the East Asian populations included in the analyses</span></strong></p><span>doi:10.1371/journal.pone.0001881.t001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Distribution of ADH1B variants</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We obtained the allele frequencies of 30 single nucleotide polymorphisms (SNPs) on the 24 new populations. The haplotype frequencies for all the 30 SNPs are given in <a href="#pone.0001881.s001">Table S1</a> and allele frequencies are in ALFRED under the UIDs in <a href="#pone-0001881-g002">Figure 2</a>. The frequencies of the <em>ADH1B</em> functional variant, <em>ADH1B*47His</em>, and the variant at the <em>ADH1B</em> regulatory polymorphism rs3811801 (SNPs 9 and 11) were transformed into contour maps in <a href="#pone-0001881-g001">Figure 1</a>. The sharp borders across which the frequencies changed quickly are marked by bold lines in the map. The distributions of both variants show very clear clinal geographic patterns of east-west division. The sharp border for <em>ADH1B*47His</em> lies between the eastern part and western part of both the continental East and Southeast Asia, from the south end in Cambodia to the north end in Inner Mongol. The frequencies of <em>ADH1B*47His</em> are quite high to the east of this border. The rs3811801 derived allele shows a somewhat different distribution within the range of <em>ADH1B*47His</em> with a less sharp border and a similar higher frequency to the east.</p>
<div class="figure" id="pone-0001881-g002"><div class="img"><a name="pone-0001881-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Pattern of regions of high LD using HAPLOT and the default r<sup>2</sup> algorithm.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Note: The codes of the non-East Asian populations are shown as NAS(Nasioi), KTY(Khanty), KMZ(Komi), FIN(Finns), SAM(Samaritans), DRU(Druze), ASH(Ashkenazi Jews). CN: New collected samples from China. LA: Newly collected samples from Laos. Both dbSNP numbers and ALFRED UID numbers are presented for the SNPs in the ADH region we typed.</p>
<span>doi:10.1371/journal.pone.0001881.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Ethnically, the distributions of these two variants are also quite regular. The populations with high frequency of <em>ADH1B*47His</em> belong to TK, AU, SN, KJ, and HM. Those subphyla all have a long history of agriculture <a href="#pone.0001881-Weng1">[40]</a>–<a href="#pone.0001881-Tong1">[42]</a>, while those populations with a low frequency of <em>ADH1B*47His</em> are all pastoral or hunting populations or began to farm recently. The TK and AU populations are excluded from among the populations with a high frequency of the rs3811801 (SNP 11) derived allele.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>This non-synonymous SNP and the <em>ADH1B</em> promoter <a href="#pone.0001881-Dannenberg1">[44]</a> polymorphism (SNP 11) have very high F<sub>st</sub> values on a global scale <a href="#pone.0001881-Han1">[12]</a>. The fixation index F<sub>st</sub>, originally designed as the most inclusive measure of population substructure, is used here as a measure of allelic difference among the populations <a href="#pone.0001881-Osier1">[10]</a>. As the populations in East Asia are fairly similar to each other <a href="#pone.0001881-Tishkoff1">[45]</a>–<a href="#pone.0001881-Jin1">[47]</a>, the F<sub>st</sub> values within the region are expected to be lower than those of the same SNPs globally. Using data on the East Asian populations listed in <a href="#pone-0001881-t001">Table 1</a>, we calculated the F<sub>st</sub> values of all the SNPs as follows, numbered as in <a href="#pone-0001881-g002">Figure 2:</a> 1(.056), 2(.196), 3(.185), 4(.153), 5(.182), 6(.177), 7(.182), 8(.051), <strong>9(.240)</strong>, 10(.098), <strong>11(.218)</strong>, 12(.145), 13(.145), 14(.198), 15(.236), 16(.145), 17(.166), 18(.166), 19(.166), 20(.094), 21(.203), 22(.168), 23(.084), 24(.101), 25(.089), 26(.101), 27(.057), 28(.053), 29(.154), 30(.225). The F<sub>st</sub> values among East Asian populations are much lower than the global values of the same SNPs, though the F<sub>st</sub> of our focused SNPs 9 and 11 are still much higher than the global F<sub>st</sub> mean of general SNPs (0.14). The F<sub>st</sub> values of the Arg47His variant and the promoter variant of <em>ADH1B</em> are among the highest values, but we also see high values in the upstream region at <em>ADH1C</em> and even upstream of <em>ADH1C</em>.</p>


<h4>Haplotypes and regions of high LD</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Six-SNP haplotypes of the <em>ADH1B</em> upstream region (SNPs 6–11) were estimated and the haplotype frequencies of the Asian populations are displayed in <a href="#pone-0001881-g003">Figure 3</a>. The patterns of the East Asian populations are obviously different from the non-East Asian populations (Uralic and Afro-Asiatic). Haplotypes 1 and 2 have high frequencies in most of the East Asian populations, while they occurred at much lower frequency in ASH (Jews). The populations in the most southwest region of East Asia (WBM: Ava, KHG: Tibetan, and PHO: Phunoi Lolo) have the lowest frequencies of these two haplotypes.</p>
<div class="figure" id="pone-0001881-g003"><div class="img"><a name="pone-0001881-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Haplotype frequencies of the <em>ADH1B</em> gene region including the regulatory region.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Note: the SNPs in the haplotypes are rs2066701-rs2075633-rs4147536-rs1229984-​6810842-rs3811801,corresponding to SNPs 6–11 in <a href="#pone-0001881-g002">Figure 2</a>. Phyla, PN: Papuan-New Guinean, AU: Austronesian, TK: Daic, AA: Austro-Asiatic, TB: Tibeto-Burman, HM: Hmong-Mien, SN: Sinitic Han, KJ: Korean-Japanese, AT: Altaic (inland), UR: Uralic, AF: Afro-Asiatic. The patterns of the non-East Asian phyla (UR, AF, PN) are quite different from those of East Asian phyla. The patterns of the phyla in East Asia can be classified into four groups as the colors shown in the left side bar. The frequency data for all haplotypes are in <a href="#pone.0001881.s001">Table S1</a>.</p>
<span>doi:10.1371/journal.pone.0001881.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Among the East Asian populations, there are four types of haplotype patterns, and the classification shows an obvious ethnic correlation. The first group (Southeast) contains AU and TK, and haplotype 2 is the major haplotype of this group. The second group (Southwest) contains AA and TB, with the characteristic haplotype 3. Haplotype 1 has a frequency greater than 50% in the third group (Northeast) that contains HM, SN, and KJ. AT populations form the fourth group (Northwest). The haplotypes in group 4 are most diverse. Only haplotype 7 is a little richer than it is in the other groups. For those non-East Asian populations, the patterns also seem regular. Haplotypes 5 and 8 are frequent in Afro-Asiatic populations but absent in Uralic populations.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Regions of high LD across the whole region were displayed in <a href="#pone-0001881-g002">Figure 2</a>. The patterns of the <em>ADH7</em> region and the region between <em>Class I ADH</em> and <em>ADH7</em> (SNPs 17–30) are quite similar among the populations in East Asia, except for TB (BQH, QMR, KHG) with fewer regions of high LD and AU (ATL, AMI) with larger high LD regions. In contrast, the LD patterns of <em>Class I ADH</em> (SNPs 1–16) are quite different among the populations. It is interesting that only the Mongol (MVF) and Hmong (HMQ, MWW) have high LD extending to the region upstream of <em>ADH1B</em> (SNPs 9–11). In the two AU populations from Taiwan, high levels of LD encompass more SNPs than seen for the other populations; this pattern suggests they are quite young and/or have undergone considerable random genetic drift recently.</p>


<h4>Population Comparison</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A principal component analysis based on the <em>ADH1B</em> haplotype frequencies in <a href="#pone-0001881-g003">Figure 3</a> was used to examine the data distribution among populations (<a href="#pone-0001881-g004">Figure 4A</a>). The first plot was constructed by principal components (PC) 1 and 2. In this plot, PC1 divides East Asian populations from non-East Asian populations. This confirms the overall genetic unity and distinctiveness of East Asia. In the second map PC2 divides the western part of East Asia from the eastern part with a sharp border. The distribution of PC2 is very similar to the distribution of <em>ADH1B*47His</em>, with the TK, HM, SN and KJ phyla in the east and the others in the west. The correlation between PC2 value and <em>ADH1B*47His</em> frequency is −0.971 (P&lt;0.001), while the correlation between PC2 and longitude is −0.491 (P = 0.005). The distribution of PC3 is also quite geography-related. It divides southern phyla (TK, AA, AU) from the northern phyla. The correlation coefficient between PC3 and the latitude is −0.808 (P&lt;0.001). Though PC2 and PC3 are significantly correlated with geography, they are more ethnic-related, as is shown in the second plot of <a href="#pone-0001881-g004">Figure 4A</a>. It is noticeable that those eastern populations such as LAO and MWW that have moved into the western area are still clustered with the eastern phyla. That suggests the distribution of this <em>ADH1B</em> haplotype (encompassing the 5′ half of the gene and the 5′ flanking region) must be related to the history of the ethnic phyla.</p>
<div class="figure" id="pone-0001881-g004"><div class="img"><a name="pone-0001881-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Principal Component Analysis plots.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Notes: The plots show the relationships among populations estimated by PCA. Plots in part A were based on the <em>ADH1B</em> haplotypes frequency data in <a href="#pone-0001881-g003">Figure 3</a>. Plots in part B used the haplotype frequency data of the whole <em>ADH</em> region in <a href="#pone-0001881-g002">Figure 2</a>. In part A, an ethnic related distribution is obvious, while in part B the distribution shows no strong distinct clusters corresponding to ethnicity.</p>
<span>doi:10.1371/journal.pone.0001881.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To test if the ethnic-related distribution of <em>ADH1B</em> is common in the human genome, we applied principal component analysis (PCA) to the haplotype frequency data of the whole <em>ADH</em> region we typed. Results are shown in <a href="#pone-0001881-g004">Figure 4B</a>. The distribution of the PCs is much closer to the general genetic relationship (a supposed genetic relationship measured by the whole genome diversity) among populations than the <em>ADH1B</em> distribution. PC1 still divides East Asia from the rest of the world. Neither PC2 nor PC3 is related to ethnicities. The PCA result of the whole region is very different from that of the smaller <em>ADH1B</em> region. Therefore the ethnic-related distribution of the first PCA of ADH1B is uncommon. The distribution of the <em>ADH1B</em> upstream region diversity is related to ethnicity, but different from the distribution of the whole Class I <em>ADH</em> and <em>ADH7</em> region diversity.</p>


<h4>Positive Selection Test of ADH1B Core Haplotypes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>As SNPs 9 to 11 are candidates for being related to function or having been positively selected, we chose this short region as the core for a test for selection. The allele frequencies of the core haplotypes are shown graphically in <a href="#pone-0001881-g005">Figure 5A</a>. The four-group classification of <a href="#pone-0001881-g002">Figure 2</a> can still be observed in <a href="#pone-0001881-g005">Figure 5A</a>. There are eight haplotypes but only four are common in East Asia. Haplotype (2) is most common in the AU-TK group. Haplotype (3) reaches highest frequency in the HM-SN-KJ group. Uralic and Afro-Asiatic are obviously different. Though AT and Uralic phyla are neighbors and share much of their culture and customs, their haplotype patterns are quite different.</p>
<div class="figure" id="pone-0001881-g005"><div class="img"><a name="pone-0001881-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Frequencies and network of the core haplotypes (rs1229984-rs6810842-rs3811801).</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Notes: Haplotype codes (1) to (8) are in the same system for both part A and part B of the figure. The sizes of the balls in the network of part B represent the rough relative frequencies of the haplotypes. The arrows are the most likely mutational relationships. The broken lines indicate possible historical recombinations. Haplotype (3) is more derived and presumably younger than haplotypes (1), (2), (4), and (6). Its high frequency in some populations suggests selection may have operated.</p>
<span>doi:10.1371/journal.pone.0001881.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In <a href="#pone-0001881-g005">Figure 5B</a>, the relationships among the haplotypes are plotted in a network. Haplotype (1) is the ancestral haplotype based on sequence comparison with four other ape species. The functional mutation <em>ADH1B*47His</em> occurred in the step from (1) to (2). The promoter mutation occurred in the step from (2) to (3). The sizes of the circles in <a href="#pone-0001881-g005">Figure 5B</a> are roughly proportional to the haplotype frequencies in the different ethnic phyla. In the network, the frequencies of the younger haplotypes are expected to be lower. Actually, young haplotypes (5) to (8) are obviously less frequent than those old ones (1), (2) and (4). The only exception is haplotype (3) which is young but still reaches very high frequency in East Asia. Outside of East Asia, this haplotype is very rare. This indicates this haplotype has undergone strong genetic drift or positive selection in East Asia. This also focuses attention on the derived allele in the promoter region.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>To test whether the high frequency of the young haplotype was caused by positive selection, we applied the EHH and REHH tests. The results are presented in <a href="#pone-0001881-g006">Figure 6</a> and <a href="#pone.0001881.s002">Figure S1</a> to scale along the chromosomal segment studied. If the EHH value decayed quickly or the REHH value did not rise to high levels with increasing distance from the core, there would be no evidence for selection. The EHH values of core haplotype (3), AGA, decayed more slowly than the other core haplotypes, indicating haplotype (3) may have been under positive selection. The REHH values of haplotype (3) also increased in the upstream direction. The REHH values of the Altaic populations including KJ were the highest among the ethnic phyla. The REHH values of two HM populations, BQH, CBD and SKB also rose to fairly high values. The high REHH values suggest the possible existence of positive selection; however, because of limitations of our sample sizes (~50 individuals per population) only those core haplotypes of moderate frequency convey meaningful results. Some populations with high REHH values failed to have moderate haplotype frequencies, such as YAK, KAZ, CBD and SKB. Thus, we cannot draw a definite conclusion for these populations.</p>
<div class="figure" id="pone-0001881-g006"><div class="img"><a name="pone-0001881-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Extended Haplotype Homozygosity (EHH) and Relative Extended Haplotype Homozygosity(REHH) of Altaic, Han and Hmong populations.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Note: Colorful lines are data of core haplotype (3)AGA, and gray lines are data of other haplotypes. The data following the population codes are frequencies of the core haplotype in the populations. EHH and REHH of the other populations are in <a href="#pone.0001881.s002">Figure S1</a>.</p>
<span>doi:10.1371/journal.pone.0001881.g006</span></div><a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>We plotted all the REHH values for all core haplotypes at SNP 26 (around 90 kb from the core) in <a href="#pone-0001881-g007">Figure 7A</a>. In Altaic and some other populations, the REHH values of haplotype (3), AGA, were much higher than those of the other haplotypes. To test whether this haplotype might have been under positive selection in these populations, we simulated the population demography (<a href="#pone-0001881-g007">Figure 7B</a>). The population history model used in the simulation was based on the migration history reconstructed by other genetic studies (mtDNA <a href="#pone.0001881-Kong1">[46]</a> and Y-chromosome <a href="#pone.0001881-Jin1">[47]</a>) and the tremendous changes of the environment or the society in East Asia. The population size was at its lowest level at the peak of the ice age <a href="#pone.0001881-Shi1">[48]</a>, and increased quickly when agriculture started. Archaeological evidence <a href="#pone.0001881-Gu1">[41]</a> confirms that agriculture started more than 8000 years ago in East Asia. If we assume a generation is about 20 years, agriculture started 400 generations ago. With this model, 10,000 populations were simulated, and the results of simulated REHH values were shown in <a href="#pone-0001881-g007">Figure 7C</a> together with the observed REHH values of haplotype AGA. Four lines (50, 75, 95 and 99 percentile) were drawn for visual comparison. All of the Altaic and HM populations are above the 95 percentile line. We also calculated the significance by a <em>t</em>-test after transforming our original result to achieve normality. The P values were transformed into the contour map in <a href="#pone-0001881-g008">Figure 8A</a>. There are 15 populations with a P value less than 0.05. The regions covered by these populations are marked in <a href="#pone-0001881-g008">Figure 8B</a> by a purple background. All of the Altaic populations and HM populations are included. Among four SN populations only HKA (Hakka) is excluded. BQH, SKB, CBD and KUF are also included. The sample size of CBD is rather small (25), and the core haplotype AGA frequency is low; therefore the REHH of CBD is not reliable. Fisher's Exact Test <a href="#pone.0001881-Raymond1">[49]</a> on haplotype composition shows CBD can be combined with LBO, a nearby and similar population (P = 0.492). REHH of the combined population is 4.467, which suggests that the combined population shows no evidence of selection. Therefore the populations with significant evidence of selection are ethnic-specific with only four exceptions. The conclusion is that strong evidence of selection on the core haplotype AGA was found in Altaic and HM populations, and weak evidence of selection in SN.</p>
<div class="figure" id="pone-0001881-g007"><div class="img"><a name="pone-0001881-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>REHH of observed and simulated populations.</span></strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>Note: The colored dots are observed REHH data of core haplotype AGA both in chart A and C. In chart A, the observed REHH data shows that most of the REHH values of haplotype AGA are higher than those of the other haplotypes. Part B is the East Asian population history model determined by complicated factors. Six phases were defined with the effective population numbers and the generation numbers to present. Chart C indicates the REHH data simulated by the model in part B along with the data in Chart A. The lines in chart C are comparison borders of the simulated data. The observed REHH of haplotype AGA of all the Altaic and Hmong populations are above the 95% border, which is the evidence of positive selection.</p>
<span>doi:10.1371/journal.pone.0001881.g007</span></div><div class="figure" id="pone-0001881-g008"><div class="img"><a name="pone-0001881-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001881" data-uri="info:doi/10.1371/journal.pone.0001881.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001881.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001881.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001881.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Significance P values of the positive selection on the <em>ADH1B</em> gene in East Asia.</span></strong></p><a id="article1.body1.sec2.sec4.fig4.caption1.p1" name="article1.body1.sec2.sec4.fig4.caption1.p1"></a><p>Note: The map of part A displayed the distribution of the significance P value of the positive selection on the <em>ADH1B</em> gene. Populations from most areas of East Asia have been significantly selected for except those in the southwest. In part B, the selection area was compared with the high frequency areas of two <em>ADH1B</em> SNPs. The dotted line encloses the region in which the <em>ADH1B*47His</em> frequency is &gt;40%, and the broken line encloses the region in which the <em>ADH1B rs3811801</em> frequency is &gt;40%. The distributions of <em>ADH1B*47His</em> and selection area differ from each other, which indicates that they are unrelated. The high frequency area of rs3811801 derived allele is included in the selection area, indicating the possible effect of this allele in the selection on the <em>ADH1B</em> gene.</p>
<span>doi:10.1371/journal.pone.0001881.g008</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>In the map of <a href="#pone-0001881-g008">Figure 8</a>, we can see that the geographic area with evidence of selection is quite different from the <em>ADH1B*47His</em> high frequency area. Correlation analyses were computed between the P values of the selection significance test and the derived allele frequencies of the SNPs in the core region. The results were SNP 9 (r = 0.100, P = 0.581), SNP 10 (r = 0.097, P = 0.590), <strong>SNP 11 (r = 0.522, P = 0.002)</strong>. Only SNP 11 has significant P values for selection. SNP 11 (rs3811801) is located at −1761 bp, the promoter region of <em>ADH1B</em> <a href="#pone.0001881-Dannenberg1">[44]</a>. Therefore the promoter region may be the actual region that has undergone selection. The alleles at this SNP may quantitatively alter <em>ADH1B</em> expression. However, since the derived allele at SNP 11 (rs3811801) occurs only on chromosomes with the <em>ADH1B*47His</em> allele, the cis-acting combination of the two alleles may have been the focus of selection.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Population History and Ethnic-related Distribution of ADH1B</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In all of the analyses based on the <em>ADH1B</em> region in this study, we saw ethnic relationships with allele and haplotype frequencies. Especially in <a href="#pone-0001881-g004">Figure 4A</a> of the PCA analysis, genetic clustering of populations matched very well with the ethnic phyla or subphyla. That is rather unusual, as we rarely see the distribution of alleles at a single autosomal locus matching finer structure of ethnic classification. There are many papers describing ethnic differences for certain loci, but those ethnic differences are among populations from different geographic regions of the world <a href="#pone.0001881-Inoue1">[50]</a>–<a href="#pone.0001881-Issell1">[52]</a>. In our study on <em>ADH1B</em>, the eight ethnic phyla or subphyla included are all in East Asia. In the PCA, these eight groups can be distinguished very well by <em>ADH1B</em> data, but not by the whole Class I <em>ADH</em> and <em>ADH7</em> region data. The ethnic-related patterns are only seen for the <em>ADH1B</em> region.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In East Asia, one hypothesis argues that most of the homologous populations in one ethnic phylum share a distinct common ancestor, which has some evidence from Y chromosome DNA <a href="#pone.0001881-Su1">[53]</a>–<a href="#pone.0001881-Tajima1">[55]</a>. As the paternal societies have lasted for a very long time in most of the East Asian populations, the Y chromosome had less chance to flow among populations than other chromosomes. Therefore the distribution of Y chromosome diversity can represent the original relationships among the ethnic phyla to a certain extent. On the other hand, two factors can change the ethnic-related genetic structure. Interactions among different ethnic phyla in East Asia have frequently occurred. Historical facts of interactions were often used to explain the different genetic structures revealed by autosomal <a href="#pone.0001881-Kim1">[56]</a> and mitochondrial <a href="#pone.0001881-Kong1">[46]</a> DNA diversity. At the same time, random genetic drift will change the frequency of a single locus in a certain population. Therefore most of the distributions of any single locus are not as ethnically related as the Y chromosome is. The ethnically related distribution of <em>ADH1B</em> cannot be explained only by random genetic drift. There can be two possible explanations. One is that some culture related incidence has maintained the special distribution. Another possible explanation is that different <em>ADH1B</em> haplotype patterns occurred at the founding of the different ethnic phyla and remained largely unchanged during population expansions and interactions.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>A number of Y chromosome diversity studies showed that TB, SN, TK and AU phyla could be recognized by some ethnicity-specific haplogroups <a href="#pone.0001881-Su1">[53]</a>, <a href="#pone.0001881-Wen1">[54]</a>, <a href="#pone.0001881-Li2">[57]</a>. TB and SN subphyla belong to Sino-Tibetan phyla, and they both originated from the upper Yellow River area and shared a common ancestor <a href="#pone.0001881-Su1">[53]</a>. TK and AU are two different ethnic phyla, but a great deal of evidence showed that they are very similar to each other <a href="#pone.0001881-He1">[58]</a>. Y-chromosome data also supports a common origin of TK and AU <a href="#pone.0001881-Li2">[57]</a>. In our <em>ADH1B</em> data, TB and SN departed from each other, while TK and AU clustered closely. <em>ADH1B</em> data do not always match the hypothesized original relationships. Therefore the ethnic-related structure of <em>ADH1B</em> might not always be caused by the original differentiations of the ethnic phyla. Some cultural factors such as life styles may have influenced the special distribution of <em>ADH1B</em>, especially in the Sino-Tibetan phylum. Weak evidence of selection was seen in the SN subphylum. That implies the <em>ADH1B</em> haplotype pattern in SN may be due to selection, and the selective forces on <em>ADH1B</em> may be something like ethnic culture. Therefore, the high frequency of <em>ADH1B*47His</em> in TK and AU may have resulted from the original population differentiation, while that in SN may have resulted from the positive selection.</p>


<h4>Cultural or Natural Force in the Selection of ADH1B Region</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The frequencies of <em>ADH1B*47His</em> are high in AU, TK, HM, SN, and KJ, but <em>ADH1B</em> haplotype patterns of the AU-TK group are different from those of the other three subphyla. According to <a href="#pone-0001881-g005">Figure 5</a>, the haplotype AGG is responsible for the high frequency of <em>ADH1B*47His</em> in AU-TK group, while it was haplotype AGA in HM-SN-KJ group. And from the selection analyses only AGA showed evidence of having been selected. Therefore, we conclude that the high frequency of haplotype AGG in the AU-TK group most probably did not result from positive selection, but from the random drift that occurred in their common ancestral population. The populations in AU-TK group may have maintained the high frequency of haplotype AGG of their common ancestor since they diverged.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>If we check the haplotype patterns of TB and SN, two members of the Sino-Tibetan phylum, in <a href="#pone-0001881-g005">Figure 5</a>, some similarities between them can be found. The AGA haplotype frequency with respect to the AGG haplotype frequency is high in each population. According to the history of Han Chinese, they derived from ancient Qiang population several thousand years ago <a href="#pone.0001881-Wang1">[59]</a>. QMR (modern Qiang) may be a very old population. There are even several individuals in QMR with the ATA haplotype that would need a quite long time to appear by rare recombination events. Therefore the haplotype pattern of QMR may be most similar to that of the Sino-Tibetan ancestor. The lower haplotype frequencies of AGG and AGA in the other TB populations might result from genetic drift, and the increase in SN might result from positive selection for which we found weak evidence. The TB and SN diverged more than 5000 years ago <a href="#pone.0001881-Su1">[53]</a>. The SN populations moved to the east and started agriculture on the plain. Both the culture and environment changed. It is not easy to determine what might have been the selective force on the Sino-Tibetan populations.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Weakly or strongly, all of the Altaic (including AT and KJ) populations show evidence of selection. The ratio of the haplotype AGA frequency with respect to AGG in Altaic is highest among the ethnic phyla. This might be a characteristic of the original Altaic population. The highest significance level for the selection significance tests appeared in three eastern populations, MVF, KOR and JPN, but we are unaware of any social or environmental similarities among these populations. KOR and JPN are agriculture populations living by the sea, while MVF is a pastoral population on the highlands. As for the other populations with evidence of positive selection, HM phylum, KUF and SKB in the Southeast Asia, the environments and cultures are even more different from the Altaic phylum. Judging from the distribution of the “selected” populations, it is clear that climate and other environmental aspects differ. Therefore it is difficult to determine how nature has influenced the allele frequency. As the distribution is ethnic-related in some extent, some cultural force of selection is understandable, though we cannot determine the common cultural factors among the populations showing the strong evidence of selection. Perhaps cultural anthropologists will shed light on this problem.</p>


<h4>The Selective Region and Allele</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The distribution of populations showing evidence of selection at <em>ADH1B</em> is quite different from the high frequency distribution of the <em>ADH1B*47His</em> allele. The high significance levels for selection are only correlated with allele frequencies of the promoter SNP rs3811801, indicating that the <em>ADH1B*47His</em> allele has not been selected for in the absence of the derived promoter allele. Evolutionarily, the derived promoter allele at SNP 11, rs3811801, occurred on a chromosome with the <em>ADH1B*47His</em> allele; thus, the derived promoter allele appears together with the <em>ADH1B*47His</em> allele in most cases. Because the selected core haplotype AGA includes two SNPs with derived alleles, both with likely functional consequences, we cannot be sure if <em>ADH1B*47His</em> has been important in the positive selection on the <em>ADH1B</em> region. The core haplotypes with the promoter derived allele but without <em>ADH1B*47His</em> are very rare; therefore the sample is too small to test for positive selection. We can speculate that it is the combined effect of an increased activity of the enzyme caused by the <em>ADH1B*47His</em> allele and an increased quantity of the enzyme caused by the derived promoter allele. However, it is possible that the promoter region variant has no effect but is simply associated by chance with that chromosome that underwent selection in specific ethnic populations.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In previous case-control studies, it was found that the <em>ADH1B*47His</em> allele decreased the risk for alcoholism both in Asian and European populations. In the Asian populations <a href="#pone.0001881-Thomasson1">[21]</a>–<a href="#pone.0001881-Osier2">[25]</a> included in the alcoholism correlation studies, both derived allele frequencies of <em>ADH1B*47His</em> and the promoter polymorphism are high. But in the European populations <a href="#pone.0001881-Borras1">[26]</a>, <a href="#pone.0001881-Neumark1">[27]</a>, <a href="#pone.0001881-Whitfield1">[29]</a>, the promoter derived allele does not appear together with <em>ADH1B*47His</em>, implying that <em>ADH1B*47His</em> is solely responsible for the decrease in the risk for alcoholism. Others report that <em>ADH1B*47His</em> was not always related to alcoholism, especially in Taiwanese populations <a href="#pone.0001881-Chen2">[32]</a>, <a href="#pone.0001881-Chen3">[33]</a>, in which the frequency of the <em>ADH1B</em> promoter derived allele is very low as we revealed in our study. Some papers <a href="#pone.0001881-Hasin1">[28]</a> also doubted the significance of the association between <em>ADH1B*47His</em> and alcoholism in European populations as the frequency of the <em>ADH1B*47His</em> is also low. Therefore the <em>ADH1B*47His</em> allele alone may not be important in changing the genetic structure of the populations.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>However the decrease in the risk for alcoholism has not been argued to be the selective force, and our results argue that selection is not solely related to <em>ADH1B*47His</em>. The derived promoter allele may have led to the increase of the ADH enzyme expression, and then enhance the protection against some deleterious effects. The <em>ADH</em> variants are also related to some types of cancers and other serious diseases <a href="#pone.0001881-Triano1">[4]</a>, <a href="#pone.0001881-Jia1">[60]</a>–<a href="#pone.0001881-CichozLach1">[66]</a>. Infectious disease is one of the plausible selective forces suggested by Goldman &amp; Enoch <a href="#pone.0001881-Goldman1">[67]</a>. Other diseases such as food poisoning can also have similar effects, and the populations will be susceptible if they happen to be partial to certain foods. This kind of pathology was reported in some populations in China <a href="#pone.0001881-Chen4">[68]</a>. In the case of <em>ADH</em> selection, Altaic and Hmong populations may have special food or other customs that increase the risk of a certain disease. The enhancement of the ADH enzyme caused by the <em>ADH1B*47His</em> may not be enough to protect the individuals from that disease, and an increased enzyme activity caused by the derived allele of the promoter variant would then be helpful. Our hypothesis is that <em>ADH1B*47His</em> can enhance the activity of ADH enzyme and the derived allele of the promoter variant of <em>ADH1B</em> can increase the quantity of the enzyme in the body; thus the protection against some related diseases will be stronger and the <em>ADH1B</em> haplotype will be selected. However, more case-control studies including both of the derived alleles are required. And we suggest these studies be conducted in the Daic or Austronesian populations, because there are more types of <em>ADH1B</em> core haplotypes and the linkage disequilibrium is weaker. Therefore, a false signal caused by the strong linkage disequilibrium can be avoided.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Populations</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We collected samples of 24 populations (populations with stars behind their code in <a href="#pone-0001881-t001">Table 1</a> and marked by colorful icons in <a href="#pone-0001881-g001">Figure 1B</a>) from six ethnic phyla. Among these populations, nine were collected from Laos, and 15 were from China. This is the first population genetics study in Laos. Laos is an inland country surrounded by all the other nations in the Peninsula. All types of populations in Peninsular Southeast Asia can be found in Laos. All individuals we collected are healthy adults without alcoholism or related disorders. Everyone signed the informed consent. Our study was approved by the Ethics Committee of the Chinese National Human Genome Center at Shanghai and the Human Investigations Committee at Yale University School of Medicine, and approved by the Laos government. The sample sizes are all large enough for meaningful frequency and haplotype estimates <a href="#pone.0001881-Fallin1">[69]</a>. The population locations are also indicated in <a href="#pone-0001881-t001">Table 1</a> by both county names and geographic coordinates, and also shown in the map of <a href="#pone-0001881-g001">Figure 1B</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p><a href="#pone-0001881-t001">Table 1</a> also lists the information of populations previously studied <a href="#pone.0001881-Han1">[12]</a> to give an overview of the <em>ADH</em> study in East Asia. These population samples are categorized both ethnically and geographically. Each phylum contains 2~7 populations. HM is the smallest phylum, represented in our data by only two populations.</p>


<h4>Sample Preparation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For the new populations blood samples were collected from finger tips of participants and kept dry on filter paper. A whole genome amplification (WGA) <a href="#pone.0001881-Dean1">[70]</a> was applied for each sample. A 3 mm<sup>2</sup> piece of paper with blood was cut off and dipped into 5 µl water, lysed with 5 µl of alkaline lysis solution (400 mM KOH, 100 mM DTT, 10 mM EDTA) and incubated for 10 min on ice. The lysed liquid was neutralized with 5 µl neutralization solution(1 M HCl:1 M Tris-HCl = 2:3, pH 0.6). The neutralized mixture was used directly as template in WGA. The WGA cocktail is water 31.5 µl, 10×MDA buffer [Tris-HCl pH7.5 375 mM, KCl 500 mM, MgCl<sub>2</sub> 100 mM, (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> 50 mM] 5 µl, 10 mM dNTP 5 µl, 2 mM thiophosphate-modified random hexamer (NNNN*N*N, where* denotes a phosphorothioate) 1.25 µl, 2 mg/ml Rnase A 0.25 µl, template mixture 5 µl, 200 ng/µl φ29 polymerase 2 µl. Cocktail was mixed well and incubated at 31°C for 15 hours. After the reaction, the polymerase activity was inactivated at 75°C for 5 min. The product was spun down to get rid of the sediment, and the produced DNA was purified by alcohol precipitation before TaqMan reactions.</p>


<h4>Markers</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We chose 30 SNPs from the ADH cluster, covering <em>Class I ADH</em> (encompassing the <em>ADH1B</em> Arg47His polymorphism) and <em>ADH7</em>. These 30 SNPs have high heterozygosity in East Asia <a href="#pone.0001881-The1">[71]</a>. The dbSNP and ALFRED numbers of each SNP from the ADH clusters are listed in <a href="#pone-0001881-g002">Figure 2</a>. All SNPs were genotyped by the TaqMan method <a href="#pone.0001881-Livak1">[72]</a> using commercial assays and reagents (<a href="http://products.appliedbiosystems.com">http://products.appliedbiosystems.com</a>) except for the assay for <em>ADH1B</em> Arg47His, which we designed <a href="#pone.0001881-Li1">[30]</a>.</p>


<h4>Analysis Methods</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Allele frequencies were calculated directly by gene counting assuming two-allele codominant inheritance. F<sub>st</sub> values <a href="#pone.0001881-Wright1">[73]</a> are calculated as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.e001&amp;representation=PNG" class="inline-graphic"></span>. Our F<sub>st</sub> values were calculated across all 32 East Asian populations. Haplotypes of the 30 SNPs in <em>ADH</em> genes were estimated by PHASE2.1 <a href="#pone.0001881-Stephens1">[74]</a>, <a href="#pone.0001881-Stephens2">[75]</a>. The population-specific patterns of regions of high LD were calculated and graphed (<a href="#pone-0001881-g002">Figure 2</a>) using HAPLOT <a href="#pone.0001881-Gu2">[76]</a> and the default Kidd r<sup>2</sup> partition algorithm. The PCA <a href="#pone.0001881-Jreskog1">[77]</a> was initially applied based on the <em>ADH1B</em> haplotype frequency of 6 SNPs (rs2066701-rs2075633-rs4147536-rs1229984​-rs6810842-rs3811801)centered on <em>ADH1B Arg47His</em> by SPSS13.0. Another PCA was based on frequencies of all available haplotypes of all the 30 SNPs we typed. The second PCA was expected to show the general population relationships with less functional effects, while the first one was supposed to be affected by the functional variant. SNP allele frequency and PCA maps were presented by Surfer8.0 .</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Analyses of empirical data suggest positive selection has operated on the ADH Class I cluster in East Asian populations <a href="#pone.0001881-Han1">[12]</a> . Therefore, the Long Range Haplotype analysis <a href="#pone.0001881-Endo1">[78]</a> has been applied to test for a potential positive selection effect on these new populations. Pilot studies suggested that SNPs (9~11: rs1229984-rs6810842-rs3811801) showed a signature of positive selection and therefore these three SNPs were selected as the core region. We used two values of Long Range Haplotype analysis to measure the selection, EHH and REHH <a href="#pone.0001881-Sabeti1">[79]</a>.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>EHH is defined as the probability that two randomly chosen chromosomes carrying a tested core haplotype are homozygous at all SNPs for the entire interval from the core region to the distance <em>x.</em> REHH is defined as the ratio of the EHH of the tested core haplotype to the EHH of the grouped set of core haplotypes at the region not including the tested core haplotype. In implementation, EHH is calculated as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where N is the total number of chromosomes/haplotype sequences, and G is the number of homozygous groups, with each group <em>i</em> having <em>n<sub>i</sub></em> elements. If there are M chromosome groups, each with C<sub>i</sub> chromosomes and an EHH value of EHH<sub>i</sub>, REHH is calculated as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001881.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>Generally speaking, if a certain core haplotype with moderate to high frequency shows high EHH and REHH over a long distance, it will be considered as an effect of positive selection <a href="#pone.0001881-Endo1">[78]</a>–<a href="#pone.0001881-Voight1">[80]</a>. The REHH value of this core haplotype was further compared with numerous data points generated by simulation that assumes neutral evolution <a href="#pone.0001881-Sabeti2">[81]</a>. A pilot and cursory population history mode in our simulation was designed according to the migration history (out of Africa around 50,000 years ago and arrived in East Asia around 20,000 years ago), archaeological discoveries (agriculture stared in East Asia around 8,000 ago, and iron tools were used around 2,500 years ago) and the environmental changes (the Last Ice Age ended around 12,000 years ago) in East Asia <a href="#pone.0001881-Gu1">[41]</a>, <a href="#pone.0001881-Issell1">[52]</a>. The simulated data were logarithmicly transformed to achieve normality for a T-test with REHH values of our core haplotype.</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>The network of the core region haplotypes was drawn by NETWORK4.201 <a href="#pone.0001881-Bandelt1">[82]</a>. The evolutionary relationships among the haplotypes were determined from the network given the identity of the ancestral allele.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001881"></div><a name="pone.0001881.s001" id="pone.0001881.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001881.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Haplotype frequencies of the ADH region in Asian populations</p>
<p class="siDoi">doi:10.1371/journal.pone.0001881.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.73 MB XLS)</p>
<a name="pone.0001881.s002" id="pone.0001881.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001881.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Extended Haplotype Homozygosity (EHH) and Relative Extended Haplotype Homozygosity(REHH) of southwest populations in East Asia. Note: Colorful lines are data of core haplotype (3)AGA, and gray lines are data of other haplotypes. The data following the population codes are frequencies of the core haplotype in the populations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001881.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.31 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We especially thank the many individuals who volunteered to provide samples for this study. We also thank the many individuals who assisted in assembling the collection of population samples.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KK JK HL SG. Performed the experiments: HL XC. Analyzed the data: HL SG WS AP. Contributed reagents/materials/analysis tools: EG. Wrote the paper: KK HL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001881-Agarwal1" id="pone.0001881-Agarwal1"></a>Agarwal DP, Goedde HW (1992) Pharmacogenetics of alcohol metabolism and alcoholism. Pharmacogenetics  2: 48–62.  <ul class="find" data-citedArticleID="1081898" data-doi="10.1097/00008571-199204000-00002"><li><a href="http://dx.doi.org/10.1097/00008571-199204000-00002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pharmacogenetics+of+alcohol+metabolism+and+alcoholism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pharmacogenetics+of+alcohol+metabolism+and+alcoholism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001881-Ramchandani1" id="pone.0001881-Ramchandani1"></a>Ramchandani VA, Bosron WF, Li TK (2001) Research advances in ethanol metabolism. Pathol Biol (Paris)  49: 676–682.  <ul class="find" data-citedArticleID="1081998" data-doi="10.1016/s0369-8114(01)00232-2"><li><a href="http://dx.doi.org/10.1016/s0369-8114(01)00232-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Research+advances+in+ethanol+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Research+advances+in+ethanol+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001881-Yoshida1" id="pone.0001881-Yoshida1"></a>Yoshida A, Hsu LC, Yasunami M (1991) Genetics of human alcohol-metabolizing enzymes. Prog Nucleic Acid Res Mol Biol  40: 255–287.  <ul class="find" data-citedArticleID="1082060" data-doi="10.1016/s0079-6603(08)60844-2"><li><a href="http://dx.doi.org/10.1016/s0079-6603(08)60844-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+human+alcohol-metabolizing+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+human+alcohol-metabolizing+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001881-Triano1" id="pone.0001881-Triano1"></a>Triano EA, Slusher LB, Atkins TA, Beneski JT, Gestl SA, et al.  (2003) Class I Alcohol Dehydorgenase is highly expressed in normal human mammary epithelium but not in invasive breast cancer: Implications for breast carcinogenesis. Cancer Research  63: 3092–3100.  <ul class="find" data-citedArticleID="1082038"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Class+I+Alcohol+Dehydorgenase+is+highly+expressed+in+normal+human+mammary+epithelium+but+not+in+invasive+breast+cancer%3A+Implications+for+breast+carcinogenesis.&amp;auth=&amp;atitle=Class+I+Alcohol+Dehydorgenase+is+highly+expressed+in+normal+human+mammary+epithelium+but+not+in+invasive+breast+cancer%3A+Implications+for+breast+carcinogenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Class+I+Alcohol+Dehydorgenase+is+highly+expressed+in+normal+human+mammary+epithelium+but+not+in+invasive+breast+cancer%3A+Implications+for+breast+carcinogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Class+I+Alcohol+Dehydorgenase+is+highly+expressed+in+normal+human+mammary+epithelium+but+not+in+invasive+breast+cancer%3A+Implications+for+breast+carcinogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001881-Dodd1" id="pone.0001881-Dodd1"></a>Dodd PR, Foley PF, Buckley ST, Eckert AL, Innes DJ (2004) Genes and gene expression in the brain of the alcoholic. Addict Behav  29: 1295–1309.  <ul class="find" data-citedArticleID="1081924" data-doi="10.1016/j.addbeh.2004.06.010"><li><a href="http://dx.doi.org/10.1016/j.addbeh.2004.06.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+and+gene+expression+in+the+brain+of+the+alcoholic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+and+gene+expression+in+the+brain+of+the+alcoholic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001881-Cheung1" id="pone.0001881-Cheung1"></a>Cheung C, Smith CK, Hoog JO, Hotchkiss SA (1999) Expression and localization of human alcohol and aldehyde dehydrogenase enzymes in skin. Biochem Biophys Res Commun.  261: 100–107.  <ul class="find" data-citedArticleID="1081916" data-doi="10.1006/bbrc.1999.0943"><li><a href="http://dx.doi.org/10.1006/bbrc.1999.0943" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+and+localization+of+human+alcohol+and+aldehyde+dehydrogenase+enzymes+in+skin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+and+localization+of+human+alcohol+and+aldehyde+dehydrogenase+enzymes+in+skin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001881-Yao1" id="pone.0001881-Yao1"></a>Yao CT, Liao CS, Yin SJ (1997) Human hepatic alcohol and aldehyde dehydrogenases: genetic polymorphism and activities. Proc Natl Sci Counc Repub China B.  21(3): 106–111.  <ul class="find" data-citedArticleID="1082054"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+hepatic+alcohol+and+aldehyde+dehydrogenases%3A+genetic+polymorphism+and+activities.&amp;auth=&amp;atitle=Human+hepatic+alcohol+and+aldehyde+dehydrogenases%3A+genetic+polymorphism+and+activities." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+hepatic+alcohol+and+aldehyde+dehydrogenases%3A+genetic+polymorphism+and+activities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+hepatic+alcohol+and+aldehyde+dehydrogenases%3A+genetic+polymorphism+and+activities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001881-Yin1" id="pone.0001881-Yin1"></a>Yin SJ, Liao CS, Wu CW, Li TT, Chen LL, et al.  (1997) Human stomach alcohol and aldehyde dehydrogenases: comparison of expression pattern and activities in alimentary tract. Gastroenterology  112: 766–775.  <ul class="find" data-citedArticleID="1082056" data-doi="10.1053/gast.1997.v112.pm9041238"><li><a href="http://dx.doi.org/10.1053/gast.1997.v112.pm9041238" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+stomach+alcohol+and+aldehyde+dehydrogenases%3A+comparison+of+expression+pattern+and+activities+in+alimentary+tract." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+stomach+alcohol+and+aldehyde+dehydrogenases%3A+comparison+of+expression+pattern+and+activities+in+alimentary+tract.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001881-HUGO1" id="pone.0001881-HUGO1"></a>HUGO Gene Nomenclature Committee (2001)   <a href="http://www.gene.ucl.ac.uk/nomenclature/">http://www.gene.ucl.ac.uk/nomenclature/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001881-Osier1" id="pone.0001881-Osier1"></a>Osier MV, Pakstis AJ, Soodyall H, Comas D, Goldman D, et al.  (2002) A global perspective on genetic variation at the ADH genes reveals unusual patterns of linkage disequilibrium and diversity. Am J Hum Genet  71: 84–99.  <ul class="find" data-citedArticleID="1081992" data-doi="10.1086/341290"><li><a href="http://dx.doi.org/10.1086/341290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+perspective+on+genetic+variation+at+the+ADH+genes+reveals+unusual+patterns+of+linkage+disequilibrium+and+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+perspective+on+genetic+variation+at+the+ADH+genes+reveals+unusual+patterns+of+linkage+disequilibrium+and+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001881-Edenberg1" id="pone.0001881-Edenberg1"></a>Edenberg HJ, Xuei X, Chen HJ, Tian H, Wetherill LF, et al.  (2006) Association of alcohol dehydrogenase genes with alcohol dependence: a comprehensive analysis. Hum Mol Genet  15: 1539–1549.  <ul class="find" data-citedArticleID="1081926" data-doi="10.1093/hmg/ddl073"><li><a href="http://dx.doi.org/10.1093/hmg/ddl073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+alcohol+dehydrogenase+genes+with+alcohol+dependence%3A+a+comprehensive+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+alcohol+dehydrogenase+genes+with+alcohol+dependence%3A+a+comprehensive+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001881-Han1" id="pone.0001881-Han1"></a>Han Y, Gu S, Oota H, Osier MV, Pakstis AJ, et al.  (2007) Evidence of Positive Selection on a Class I <em>ADH</em> Locus. Am J Hum Genet  80: 441–456.  <ul class="find" data-citedArticleID="1081946" data-doi="10.1086/512485"><li><a href="http://dx.doi.org/10.1086/512485" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+of+Positive+Selection+on+a+Class+I+ADH+Locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+of+Positive+Selection+on+a+Class+I+ADH+Locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001881-Eriksson1" id="pone.0001881-Eriksson1"></a>Eriksson CJ, Fukunaga T, Sarkola T, Chen WJ, Chen CC, et al.  (2001) Functional relevance of human adh polymorphism. Alcohol Clin Exp Res 5 Suppl ISBRA157S–163S.  <ul class="find" data-citedArticleID="1081930" data-doi="10.1111/j.1530-0277.2001.tb02391.x"><li><a href="http://dx.doi.org/10.1111/j.1530-0277.2001.tb02391.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+relevance+of+human+adh+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+relevance+of+human+adh+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001881-Matsuo1" id="pone.0001881-Matsuo1"></a>Matsuo K, Hiraki A, Hirose K, Ito H, Suzuki T, et al.  (2007) Impact of the Alcohol-Dehydrogenase (<em>ADH</em>) 1C and <em>ADH1B</em> polymorphisms on drinking behavior in nonalcoholic Japanese. Hum Mutat  28: 506–510.  <ul class="find" data-citedArticleID="1081984" data-doi="10.1002/humu.20477"><li><a href="http://dx.doi.org/10.1002/humu.20477" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+the+Alcohol-Dehydrogenase+%28ADH%29+1C+and+ADH1B+polymorphisms+on+drinking+behavior+in+nonalcoholic+Japanese." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+the+Alcohol-Dehydrogenase+%28ADH%29+1C+and+ADH1B+polymorphisms+on+drinking+behavior+in+nonalcoholic+Japanese.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001881-Mulligan1" id="pone.0001881-Mulligan1"></a>Mulligan CJ, Robin RW, Osier MV, Sambuughin N, Goldfarb LG, et al.  (2003) Allelic variation at alcohol metabolism genes (<em>ADH1B, ADH1C, ALDH2</em>) and alcohol dependence in an American Indian population. Hum Genet  113: 325–336.  <ul class="find" data-citedArticleID="1081986" data-doi="10.1007/s00439-003-0971-z"><li><a href="http://dx.doi.org/10.1007/s00439-003-0971-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allelic+variation+at+alcohol+metabolism+genes+%28ADH1B%2C+ADH1C%2C+ALDH2%29+and+alcohol+dependence+in+an+American+Indian+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allelic+variation+at+alcohol+metabolism+genes+%28ADH1B%2C+ADH1C%2C+ALDH2%29+and+alcohol+dependence+in+an+American+Indian+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001881-Chen1" id="pone.0001881-Chen1"></a>Chen CC, Lu RB, Chen YC, Wang MF, Chang YC, et al.  (1999) Interaction between the functional polymorphisms of the alcohol-metabolism genes in protection against alcoholism. Am J Hum Genet  65: 795–807.  <ul class="find" data-citedArticleID="1081908" data-doi="10.1086/302540"><li><a href="http://dx.doi.org/10.1086/302540" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+between+the+functional+polymorphisms+of+the+alcohol-metabolism+genes+in+protection+against+alcoholism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+between+the+functional+polymorphisms+of+the+alcohol-metabolism+genes+in+protection+against+alcoholism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001881-Reich1" id="pone.0001881-Reich1"></a>Reich T, Edenberg HJ, Goate A, Williams JT, Rice JP, et al.  (1998) Genome-wide search for genes affecting the risk for alcohol dependence. Am J Med Genet  81: 207–215.  <ul class="find" data-citedArticleID="1082002" data-doi="10.1002/(sici)1096-8628(19980508)81:3<207::aid-ajmg1&gt;3.0.co;2-t"><li><a href="http://dx.doi.org/10.1002/(sici)1096-8628(19980508)81:3<207::aid-ajmg1&gt;3.0.co;2-t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+search+for+genes+affecting+the+risk+for+alcohol+dependence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+search+for+genes+affecting+the+risk+for+alcohol+dependence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001881-Saccone1" id="pone.0001881-Saccone1"></a>Saccone NL, Kwon JM, Corbett J, Goate A, Rochberg N, et al.  (2001) A genome screen of maximum number of drinks as an alcoholism phenotype. Am J Med Genet B Neuropsychiatr Genet  96: 632–637.  <ul class="find" data-citedArticleID="1082010" data-doi="10.1002/1096-8628(20001009)96:5<632::aid-ajmg8&gt;3.0.co;2-#"><li><a href="http://dx.doi.org/10.1002/1096-8628(20001009)96:5<632::aid-ajmg8&gt;3.0.co;2-#" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome+screen+of+maximum+number+of+drinks+as+an+alcoholism+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome+screen+of+maximum+number+of+drinks+as+an+alcoholism+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001881-Long1" id="pone.0001881-Long1"></a>Long JC, Knowler WC, Hanson RL, Robin RW, Urbanek M, et al.  (1998) Evidence for genetic linkage to alcohol dependence on chromosomes 4 and 11 from an autosome-wide scan in an American Indian population. Am J Med Genet  81: 216–221.  <ul class="find" data-citedArticleID="1081978" data-doi="10.1002/(sici)1096-8628(19980508)81:3<216::aid-ajmg2&gt;3.0.co;2-u"><li><a href="http://dx.doi.org/10.1002/(sici)1096-8628(19980508)81:3<216::aid-ajmg2&gt;3.0.co;2-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+genetic+linkage+to+alcohol+dependence+on+chromosomes+4+and+11+from+an+autosome-wide+scan+in+an+American+Indian+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+genetic+linkage+to+alcohol+dependence+on+chromosomes+4+and+11+from+an+autosome-wide+scan+in+an+American+Indian+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001881-Uhl1" id="pone.0001881-Uhl1"></a>Uhl GR (2004) Molecular genetic underpinnings of human substance abuse vulnerability: likely contributions to understanding addiction as a mnemonic process. Neuropharmacology  47: Suppl 1140–147.  <ul class="find" data-citedArticleID="1082040" data-doi="10.1016/j.neuropharm.2004.07.029"><li><a href="http://dx.doi.org/10.1016/j.neuropharm.2004.07.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetic+underpinnings+of+human+substance+abuse+vulnerability%3A+likely+contributions+to+understanding+addiction+as+a+mnemonic+process." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetic+underpinnings+of+human+substance+abuse+vulnerability%3A+likely+contributions+to+understanding+addiction+as+a+mnemonic+process.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001881-Thomasson1" id="pone.0001881-Thomasson1"></a>Thomasson HR, Crabb DW, Edenberg HJ, Li TK, Hwu HG, et al.  (1994) Low frequency of the ADH2*2 allele among Atayal natives of Taiwan with alcohol use disorders. Alcohol Clin Exp Res  18: 640–643.  <ul class="find" data-citedArticleID="1082030" data-doi="10.1111/j.1530-0277.1994.tb00923.x"><li><a href="http://dx.doi.org/10.1111/j.1530-0277.1994.tb00923.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Low+frequency+of+the+ADH2*2+allele+among+Atayal+natives+of+Taiwan+with+alcohol+use+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Low+frequency+of+the+ADH2*2+allele+among+Atayal+natives+of+Taiwan+with+alcohol+use+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001881-Muramatsu1" id="pone.0001881-Muramatsu1"></a>Muramatsu T, Wang ZC, Fang YR, Hu KB, Yan H, et al.  (1995) Alcohol and aldehyde dehydrogenase genotypes and drinking behavior of Chinese living in Shanghai. Hum Genet  96: 151–154.  <ul class="find" data-citedArticleID="1081988" data-doi="10.1007/bf00207371"><li><a href="http://dx.doi.org/10.1007/bf00207371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+and+aldehyde+dehydrogenase+genotypes+and+drinking+behavior+of+Chinese+living+in+Shanghai." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+and+aldehyde+dehydrogenase+genotypes+and+drinking+behavior+of+Chinese+living+in+Shanghai.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001881-Higuchi1" id="pone.0001881-Higuchi1"></a>Higuchi S, Matsushita S, Murayama M, Takagi S, Hayashida M (1995) Alcohol and aldehyde dehydrogenase polymorphisms and the risk for alcoholism. Am J Psychiatry  152: 1219–1221.  <ul class="find" data-citedArticleID="1081952"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alcohol+and+aldehyde+dehydrogenase+polymorphisms+and+the+risk+for+alcoholism.&amp;auth=&amp;atitle=Alcohol+and+aldehyde+dehydrogenase+polymorphisms+and+the+risk+for+alcoholism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+and+aldehyde+dehydrogenase+polymorphisms+and+the+risk+for+alcoholism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+and+aldehyde+dehydrogenase+polymorphisms+and+the+risk+for+alcoholism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001881-Shen1" id="pone.0001881-Shen1"></a>Shen YC, Fan JH, Edenberg HJ, Li TK, Cui YH, et al.  (1997) Polymorphism of <em>ADH</em> and <em>ALDH</em> genes among four ethnic groups in China and effects upon the risk for alcoholism. Alcohol Clin Exp Res  21: 1272–1277.  <ul class="find" data-citedArticleID="1082014" data-doi="10.1111/j.1530-0277.1997.tb04448.x"><li><a href="http://dx.doi.org/10.1111/j.1530-0277.1997.tb04448.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+of+ADH+and+ALDH+genes+among+four+ethnic+groups+in+China+and+effects+upon+the+risk+for+alcoholism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+of+ADH+and+ALDH+genes+among+four+ethnic+groups+in+China+and+effects+upon+the+risk+for+alcoholism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001881-Osier2" id="pone.0001881-Osier2"></a>Osier M, Pakstis AJ, Kidd JR, Lee JF, Yin SJ, et al.  (1999) Linkage disequilibrium at the <em>ADH2</em> and <em>ADH3</em> loci and risk of alcoholism. Am J Hum Genet  64: 1147–1157.  <ul class="find" data-citedArticleID="1081994" data-doi="10.1086/302317"><li><a href="http://dx.doi.org/10.1086/302317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+at+the+ADH2+and+ADH3+loci+and+risk+of+alcoholism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+at+the+ADH2+and+ADH3+loci+and+risk+of+alcoholism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001881-Borras1" id="pone.0001881-Borras1"></a>Borras E, Coutelle C, Rosell A, Fernandez-Muixi F, Broch M, et al.  (2000) Genetic polymorphism of alcohol dehydrogenase in europeans: the ADH2*2 allele decreases the risk for alcoholism and is associated with ADH3*1. Hepatology  31: 984–989.  <ul class="find" data-citedArticleID="1081904" data-doi="10.1053/he.2000.5978"><li><a href="http://dx.doi.org/10.1053/he.2000.5978" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+polymorphism+of+alcohol+dehydrogenase+in+europeans%3A+the+ADH2*2+allele+decreases+the+risk+for+alcoholism+and+is+associated+with+ADH3*1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+polymorphism+of+alcohol+dehydrogenase+in+europeans%3A+the+ADH2*2+allele+decreases+the+risk+for+alcoholism+and+is+associated+with+ADH3*1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001881-Neumark1" id="pone.0001881-Neumark1"></a>Neumark YD, Friedlander Y, Thomasson HR, Li TK (1998) Association of the ADH2*2 allele with reduced ethanol consumption in Jewish men in Israel: a pilot study. J Stud Alcohol  59: 133–139.  <ul class="find" data-citedArticleID="1081990"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Association+of+the+ADH2*2+allele+with+reduced+ethanol+consumption+in+Jewish+men+in+Israel%3A+a+pilot+study.&amp;auth=&amp;atitle=Association+of+the+ADH2*2+allele+with+reduced+ethanol+consumption+in+Jewish+men+in+Israel%3A+a+pilot+study." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+of+the+ADH2*2+allele+with+reduced+ethanol+consumption+in+Jewish+men+in+Israel%3A+a+pilot+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+of+the+ADH2*2+allele+with+reduced+ethanol+consumption+in+Jewish+men+in+Israel%3A+a+pilot+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001881-Hasin1" id="pone.0001881-Hasin1"></a>Hasin D, Aharonovich E, Liu XH, Mamman Z, Matseoane K, et al.  (2002) Alcohol and ADH2 in Israel: Ashkenazis, Sephardics, and Recent Russian Immigrants. Am J Psychiatry  159: 1432–1434.  <ul class="find" data-citedArticleID="1081948" data-doi="10.1176/appi.ajp.159.8.1432"><li><a href="http://dx.doi.org/10.1176/appi.ajp.159.8.1432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+and+ADH2+in+Israel%3A+Ashkenazis%2C+Sephardics%2C+and+Recent+Russian+Immigrants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+and+ADH2+in+Israel%3A+Ashkenazis%2C+Sephardics%2C+and+Recent+Russian+Immigrants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001881-Whitfield1" id="pone.0001881-Whitfield1"></a>Whitfield JB, Nightingale BN, Bucholz KK, Madden PA, Heath AC, et al.  (1998) ADH genotypes and alcohol use and dependence in Europeans. Alcohol Clin Exp Res  22: 1463–1469.  <ul class="find" data-citedArticleID="1082050" data-doi="10.1111/j.1530-0277.1998.tb03936.x"><li><a href="http://dx.doi.org/10.1111/j.1530-0277.1998.tb03936.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ADH+genotypes+and+alcohol+use+and+dependence+in+Europeans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ADH+genotypes+and+alcohol+use+and+dependence+in+Europeans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001881-Li1" id="pone.0001881-Li1"></a>Li H, Mukherjee N, Soundararajan U, Tárnok Z, Barta C, et al.  (2007) Geographically Separate Increases in the Frequency of the Derived <em>ADH1B*47His</em> Allele in Eastern and Western Asia. Am J Hum Genet  81: 842–846.  <ul class="find" data-citedArticleID="1081970" data-doi="10.1086/521201"><li><a href="http://dx.doi.org/10.1086/521201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geographically+Separate+Increases+in+the+Frequency+of+the+Derived+ADH1B*47His+Allele+in+Eastern+and+Western+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geographically+Separate+Increases+in+the+Frequency+of+the+Derived+ADH1B*47His+Allele+in+Eastern+and+Western+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001881-Goedde1" id="pone.0001881-Goedde1"></a>Goedde HW, Agarwal DP, Fritze G, Meier-Tackmann D, Singh S, et al.  (1992) Distribution of ADH2 and ALDH2 genotypes in different populations. Hum Genet  88: 344–346.  <ul class="find" data-citedArticleID="1081934" data-doi="10.1007/bf00197271"><li><a href="http://dx.doi.org/10.1007/bf00197271" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+ADH2+and+ALDH2+genotypes+in+different+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+ADH2+and+ALDH2+genotypes+in+different+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001881-Chen2" id="pone.0001881-Chen2"></a>Chen WJ, Loh EW, Hsu YP, Chen CC, Yu JM, et al.  (1996) Alcohol-metabolising genes and alcoholism among Taiwanese Han men: independent effect of ADH2, ADH3 and ALDH2. Br J Psychiatry  168: 762–767.  <ul class="find" data-citedArticleID="1081910" data-doi="10.1192/bjp.168.6.762"><li><a href="http://dx.doi.org/10.1192/bjp.168.6.762" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol-metabolising+genes+and+alcoholism+among+Taiwanese+Han+men%3A+independent+effect+of+ADH2%2C+ADH3+and+ALDH2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol-metabolising+genes+and+alcoholism+among+Taiwanese+Han+men%3A+independent+effect+of+ADH2%2C+ADH3+and+ALDH2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001881-Chen3" id="pone.0001881-Chen3"></a>Chen WJ, Loh EW, Hsu YP, Cheng AT (1997) Alcohol dehydrogenase and aldehyde dehydrogenase genotypes and alcoholism among Taiwanese aborigines. Biol Psychiatry  41: 703–709.  <ul class="find" data-citedArticleID="1081912" data-doi="10.1016/s0006-3223(96)00072-8"><li><a href="http://dx.doi.org/10.1016/s0006-3223(96)00072-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+dehydrogenase+and+aldehyde+dehydrogenase+genotypes+and+alcoholism+among+Taiwanese+aborigines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+dehydrogenase+and+aldehyde+dehydrogenase+genotypes+and+alcoholism+among+Taiwanese+aborigines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001881-Ma1" id="pone.0001881-Ma1"></a>Ma LL, Xue YL, Liu Y, Wang Z, Cui XB, et al.  (2005) Polymorphism study of seven SNPs at ADH genes in 15 Chinese populations. Hereditas  142: 103–111.  <ul class="find" data-citedArticleID="1081980" data-doi="10.1111/j.1601-5223.2005.01910.x"><li><a href="http://dx.doi.org/10.1111/j.1601-5223.2005.01910.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphism+study+of+seven+SNPs+at+ADH+genes+in+15+Chinese+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphism+study+of+seven+SNPs+at+ADH+genes+in+15+Chinese+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001881-Gordon1" id="pone.0001881-Gordon1"></a>Gordon RG Jr (2005) Ethnologue: Languages of the World, Fifteenth edition. Dallas, Tex.: SIL International.   Web version: <a href="http://www.ethnologue.com/">http://www.ethnologue.com/</a>. <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001881-Sinor1" id="pone.0001881-Sinor1"></a>Sinor D (1990) The Cambridge History of Early Inner Asia. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0001881-Roemer1" id="pone.0001881-Roemer1"></a>Roemer HR (2000) History of the Turkic Peoples in the Pre-Islamic Period. Berlin: Klaus Schwarz Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0001881-Mackerras1" id="pone.0001881-Mackerras1"></a>Mackerras C (1972) The Uighur Empire According to the T'ang Dynastic Histories. Canberra: Australian National University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001881-Liu1" id="pone.0001881-Liu1"></a>Liu ZX (1985) Weiwuerzu lishi:shang bian (History of the Uighur: Vol. 1). Beijing: Minzu chubanshe.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001881-Weng1" id="pone.0001881-Weng1"></a>Weng QH (1998) A discussion of rice-growing origin in the Nanling Mountains, South China. Tropical Geography  18(1): 72–80.  <ul class="find" data-citedArticleID="1082048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+discussion+of+rice-growing+origin+in+the+Nanling+Mountains%2C+South+China.&amp;auth=&amp;atitle=A+discussion+of+rice-growing+origin+in+the+Nanling+Mountains%2C+South+China." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+discussion+of+rice-growing+origin+in+the+Nanling+Mountains%2C+South+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+discussion+of+rice-growing+origin+in+the+Nanling+Mountains%2C+South+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001881-Gu1" id="pone.0001881-Gu1"></a>Gu WN (2001) Agricultural Archeology in China its research development and main results. Agricultural Archaeology  21(1): 1–16.  <ul class="find" data-citedArticleID="1081940"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Agricultural+Archeology+in+China+its+research+development+and+main+results.&amp;auth=&amp;atitle=Agricultural+Archeology+in+China+its+research+development+and+main+results." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Agricultural+Archeology+in+China+its+research+development+and+main+results." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Agricultural+Archeology+in+China+its+research+development+and+main+results.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001881-Tong1" id="pone.0001881-Tong1"></a>Tong EZ (1984) Outlines of some topics on the origin of agriculture in Southeast Asia and South China. Agricultural Archaeology  4(2): 29–38.  <ul class="find" data-citedArticleID="1082036"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Outlines+of+some+topics+on+the+origin+of+agriculture+in+Southeast+Asia+and+South+China.&amp;auth=&amp;atitle=Outlines+of+some+topics+on+the+origin+of+agriculture+in+Southeast+Asia+and+South+China." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outlines+of+some+topics+on+the+origin+of+agriculture+in+Southeast+Asia+and+South+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outlines+of+some+topics+on+the+origin+of+agriculture+in+Southeast+Asia+and+South+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001881-Baker1" id="pone.0001881-Baker1"></a>Baker C, Phongpaichit P (2005) A History of Thailand. Bangkok: Chulalongkorn University.   <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0001881-Dannenberg1" id="pone.0001881-Dannenberg1"></a>Dannenberg LO, Chen HJ, Tian HJ, Edenberg HJ (2006) Differential Regulation of the Alcohol Dehydrogenase 1B (<em>ADH1B</em>) and <em>ADH1C</em> Genes by DNA Methylation and Histone Deacetylation. Alcohol Clin Exp Res  30: 928–937.  <ul class="find" data-citedArticleID="1081920" data-doi="10.1111/j.1530-0277.2006.00107.x"><li><a href="http://dx.doi.org/10.1111/j.1530-0277.2006.00107.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+Regulation+of+the+Alcohol+Dehydrogenase+1B+%28ADH1B%29+and+ADH1C+Genes+by+DNA+Methylation+and+Histone+Deacetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+Regulation+of+the+Alcohol+Dehydrogenase+1B+%28ADH1B%29+and+ADH1C+Genes+by+DNA+Methylation+and+Histone+Deacetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001881-Tishkoff1" id="pone.0001881-Tishkoff1"></a>Tishkoff SA, Kidd KK (2004) Implications of biogeography of human populations for ‘race’ and medicine. Nat Genet  36: 11 SupplS21–7.  <ul class="find" data-citedArticleID="1082034" data-doi="10.1038/ng1438"><li><a href="http://dx.doi.org/10.1038/ng1438" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Implications+of+biogeography+of+human+populations+for+%E2%80%98race%E2%80%99+and+medicine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Implications+of+biogeography+of+human+populations+for+%E2%80%98race%E2%80%99+and+medicine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001881-Kong1" id="pone.0001881-Kong1"></a>Kong QP, Yao YG, Sun C, Bandelt HJ, Zhu CL, et al.  (2003) Phylogeny of East Asian mitochondrial DNA lineages inferred from complete sequences. Am J Hum Genet  73: 671–676.  <ul class="find" data-citedArticleID="1081968" data-doi="10.1086/377718"><li><a href="http://dx.doi.org/10.1086/377718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+East+Asian+mitochondrial+DNA+lineages+inferred+from+complete+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+East+Asian+mitochondrial+DNA+lineages+inferred+from+complete+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001881-Jin1" id="pone.0001881-Jin1"></a>Jin L, Su B (2000) Natives or immigrants: origin and migrations of modern humans in East Asia. Nat Rev Genet  1: 126–133.  <ul class="find" data-citedArticleID="1081962"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Natives+or+immigrants%3A+origin+and+migrations+of+modern+humans+in+East+Asia.&amp;auth=&amp;atitle=Natives+or+immigrants%3A+origin+and+migrations+of+modern+humans+in+East+Asia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natives+or+immigrants%3A+origin+and+migrations+of+modern+humans+in+East+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natives+or+immigrants%3A+origin+and+migrations+of+modern+humans+in+East+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001881-Shi1" id="pone.0001881-Shi1"></a>Shi YF, Cui ZJ, Li JJ (1989) Quaternary glacier in Eastern China and the climate fluctuation. Beijing: Science Press.   <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0001881-Raymond1" id="pone.0001881-Raymond1"></a>Raymond M, Rousset F (1995) An exact test for population differentiation. Evolution  49: 1280–1283.  <ul class="find" data-citedArticleID="1082000" data-doi="10.2307/2410454"><li><a href="http://dx.doi.org/10.2307/2410454" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+exact+test+for+population+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+exact+test+for+population+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001881-Inoue1" id="pone.0001881-Inoue1"></a>Inoue K, Asao T, Shimada T (2000) Ethnic-related differences in the frequency distribution of genetic polymorphisms in the CYP1A1 and CYP1B1 genes in Japanese and Caucasian populations. Xenobiotica  30: 285–295.  <ul class="find" data-citedArticleID="1081956" data-doi="10.1080/004982500237677"><li><a href="http://dx.doi.org/10.1080/004982500237677" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ethnic-related+differences+in+the+frequency+distribution+of+genetic+polymorphisms+in+the+CYP1A1+and+CYP1B1+genes+in+Japanese+and+Caucasian+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ethnic-related+differences+in+the+frequency+distribution+of+genetic+polymorphisms+in+the+CYP1A1+and+CYP1B1+genes+in+Japanese+and+Caucasian+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001881-Charlton1" id="pone.0001881-Charlton1"></a>Charlton KE, Steyn K, Levitt NS, Zulu JV, Jonathan D, et al.  (2005) Ethnic differences in intake and excretion of sodium, potassium, calcium and magnesium in South Africans. Eur J Cardiovasc Prev Rehabil  12: 355–362.  <ul class="find" data-citedArticleID="1081906" data-doi="10.1097/01.hjr.0000170265.22938.d1"><li><a href="http://dx.doi.org/10.1097/01.hjr.0000170265.22938.d1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ethnic+differences+in+intake+and+excretion+of+sodium%2C+potassium%2C+calcium+and+magnesium+in+South+Africans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ethnic+differences+in+intake+and+excretion+of+sodium%2C+potassium%2C+calcium+and+magnesium+in+South+Africans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001881-Issell1" id="pone.0001881-Issell1"></a>Issell BF, Maskarinec G, Pagano I, Gotay CC (2005) Breast cancer treatment among women of different ethnicity in Hawaii. Cancer Invest  23: 497–504.  <ul class="find" data-citedArticleID="1081958" data-doi="10.1080/07357900500201442"><li><a href="http://dx.doi.org/10.1080/07357900500201442" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breast+cancer+treatment+among+women+of+different+ethnicity+in+Hawaii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breast+cancer+treatment+among+women+of+different+ethnicity+in+Hawaii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001881-Su1" id="pone.0001881-Su1"></a>Su B, Xiao C, Deka R, Seielstad MT, Kangwanpong D, et al.  (2000) Y chromosome haplotypes reveal prehistorical migrations to the Himalayas. Human Genetics  107: 582–590.  <ul class="find" data-citedArticleID="1082024" data-doi="10.1007/s004390000406"><li><a href="http://dx.doi.org/10.1007/s004390000406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y+chromosome+haplotypes+reveal+prehistorical+migrations+to+the+Himalayas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y+chromosome+haplotypes+reveal+prehistorical+migrations+to+the+Himalayas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001881-Wen1" id="pone.0001881-Wen1"></a>Wen B, Li H, Lu DR, Song XF, Zhang F, et al.  (2004) Genetic evidence supports demic diffusion of Han culture. Nature  431: 302–305.  <ul class="find" data-citedArticleID="1082046" data-doi="10.1038/nature02878"><li><a href="http://dx.doi.org/10.1038/nature02878" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+supports+demic+diffusion+of+Han+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+supports+demic+diffusion+of+Han+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001881-Tajima1" id="pone.0001881-Tajima1"></a>Tajima A, Pan IH, Fucharoen G, Fucharoen S, Matsuo M, et al.  (2002) Three major lineages of Asian Y chromosomes: implications for the peopling of east and southeast Asia. Hum Genet  110: 80–88.  <ul class="find" data-citedArticleID="1082026" data-doi="10.1007/s00439-001-0651-9"><li><a href="http://dx.doi.org/10.1007/s00439-001-0651-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+major+lineages+of+Asian+Y+chromosomes%3A+implications+for+the+peopling+of+east+and+southeast+Asia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+major+lineages+of+Asian+Y+chromosomes%3A+implications+for+the+peopling+of+east+and+southeast+Asia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001881-Kim1" id="pone.0001881-Kim1"></a>Kim J, Verdu P, Pakstis AJ, Speed WC, Kidd JR, et al.  (2005) Use of autosomal loci for clustering individuals and populations of East Asian origin. Hum Genet  117: 511–519.  <ul class="find" data-citedArticleID="1081966" data-doi="10.1007/s00439-005-1334-8"><li><a href="http://dx.doi.org/10.1007/s00439-005-1334-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+autosomal+loci+for+clustering+individuals+and+populations+of+East+Asian+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+autosomal+loci+for+clustering+individuals+and+populations+of+East+Asian+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001881-Li2" id="pone.0001881-Li2"></a>Li H, Huang Y, Mustavich LF, Zhang F, Tan JZ, et al.  (2007) Y chromosomes of Prehistoric People along the Yangtze River. Hum Genet  122: 383–388.  <ul class="find" data-citedArticleID="1081972" data-doi="10.1007/s00439-007-0407-2"><li><a href="http://dx.doi.org/10.1007/s00439-007-0407-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y+chromosomes+of+Prehistoric+People+along+the+Yangtze+River." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y+chromosomes+of+Prehistoric+People+along+the+Yangtze+River.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001881-He1" id="pone.0001881-He1"></a>He P (2003) On the Origin of Ethnic Groups Speaking Austronesian Languages and their Historical Relationship with the Ethnic Groups in South China. J Yunnan University for Nationalities  20: 45–48.  <ul class="find" data-citedArticleID="1081950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+Origin+of+Ethnic+Groups+Speaking+Austronesian+Languages+and+their+Historical+Relationship+with+the+Ethnic+Groups+in+South+China.&amp;auth=&amp;atitle=On+the+Origin+of+Ethnic+Groups+Speaking+Austronesian+Languages+and+their+Historical+Relationship+with+the+Ethnic+Groups+in+South+China." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+Origin+of+Ethnic+Groups+Speaking+Austronesian+Languages+and+their+Historical+Relationship+with+the+Ethnic+Groups+in+South+China." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+Origin+of+Ethnic+Groups+Speaking+Austronesian+Languages+and+their+Historical+Relationship+with+the+Ethnic+Groups+in+South+China.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001881-Wang1" id="pone.0001881-Wang1"></a>Wang MK (1999) Primordial History: Brothers Stories of the Qiang. Time, Memory and History. Taipei: The Institute of Ethnology, Academia Sinica.   <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0001881-Jia1" id="pone.0001881-Jia1"></a>Jia C, Liu T, Liu Z, Li M, Hu M (2007) Joint effects of eNOS gene T-786C and ADH2 Arg47His polymorphisms on the risk of premature coronary artery disease. Thromb Res  120: 679–684.  <ul class="find" data-citedArticleID="1081960" data-doi="10.1016/j.thromres.2006.12.023"><li><a href="http://dx.doi.org/10.1016/j.thromres.2006.12.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Joint+effects+of+eNOS+gene+T-786C+and+ADH2+Arg47His+polymorphisms+on+the+risk+of+premature+coronary+artery+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Joint+effects+of+eNOS+gene+T-786C+and+ADH2+Arg47His+polymorphisms+on+the+risk+of+premature+coronary+artery+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001881-Seitz1" id="pone.0001881-Seitz1"></a>Seitz HK, Maurer B, Stickel F (2005) Alcohol consumption and cancer of the gastrointestinal tract. Dig Dis  23: 297–303.  <ul class="find" data-citedArticleID="1082012" data-doi="10.1159/000090177"><li><a href="http://dx.doi.org/10.1159/000090177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+consumption+and+cancer+of+the+gastrointestinal+tract." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+consumption+and+cancer+of+the+gastrointestinal+tract.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001881-Yokoyama1" id="pone.0001881-Yokoyama1"></a>Yokoyama A, Omori T (2003) Genetic polymorphisms of alcohol and aldehyde dehydrogenases and risk for esophageal and head and neck cancers. Jpn J Clin Oncol  33: 111–121.  <ul class="find" data-citedArticleID="1082058" data-doi="10.1093/jjco/hyg026"><li><a href="http://dx.doi.org/10.1093/jjco/hyg026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+polymorphisms+of+alcohol+and+aldehyde+dehydrogenases+and+risk+for+esophageal+and+head+and+neck+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+polymorphisms+of+alcohol+and+aldehyde+dehydrogenases+and+risk+for+esophageal+and+head+and+neck+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001881-Homann1" id="pone.0001881-Homann1"></a>Homann N, Stickel F, Konig IR, Jacobs A, Junghanns K, et al.  (2006) Alcohol dehydrogenase 1C*1 allele is a genetic marker for alcohol-associated cancer in heavy drinkers. Int J Cancer  118: 1998–2002.  <ul class="find" data-citedArticleID="1081954" data-doi="10.1002/ijc.21583"><li><a href="http://dx.doi.org/10.1002/ijc.21583" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+dehydrogenase+1C*1+allele+is+a+genetic+marker+for+alcohol-associated+cancer+in+heavy+drinkers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+dehydrogenase+1C*1+allele+is+a+genetic+marker+for+alcohol-associated+cancer+in+heavy+drinkers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001881-Peters1" id="pone.0001881-Peters1"></a>Peters ES, McClean MD, Liu M, Eisen EA, Mueller N, et al.  (2005) The ADH1C polymorphism modifies the risk of squamous cell carcinoma of the head and neck associated with alcohol and tobacco use. Cancer Epidemiol Biomarkers Prev  14: 476–482.  <ul class="find" data-citedArticleID="1081996" data-doi="10.1158/1055-9965.epi-04-0431"><li><a href="http://dx.doi.org/10.1158/1055-9965.epi-04-0431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ADH1C+polymorphism+modifies+the+risk+of+squamous+cell+carcinoma+of+the+head+and+neck+associated+with+alcohol+and+tobacco+use." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ADH1C+polymorphism+modifies+the+risk+of+squamous+cell+carcinoma+of+the+head+and+neck+associated+with+alcohol+and+tobacco+use.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001881-Tiemersma1" id="pone.0001881-Tiemersma1"></a>Tiemersma EW, Wark PA, Ocke MC, Bunschoten A, Otten MH, et al.  (2003) Alcohol consumption, alcohol dehydrogenase 3 polymorphism, and colorectal adenomas. Cancer Epidemiol Biomarkers Prev  12: 419–425.  <ul class="find" data-citedArticleID="1082032"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alcohol+consumption%2C+alcohol+dehydrogenase+3+polymorphism%2C+and+colorectal+adenomas.&amp;auth=&amp;atitle=Alcohol+consumption%2C+alcohol+dehydrogenase+3+polymorphism%2C+and+colorectal+adenomas." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alcohol+consumption%2C+alcohol+dehydrogenase+3+polymorphism%2C+and+colorectal+adenomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alcohol+consumption%2C+alcohol+dehydrogenase+3+polymorphism%2C+and+colorectal+adenomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001881-CichozLach1" id="pone.0001881-CichozLach1"></a>Cichoz-Lach H, Partycka J, Nesina I, Celinski K, Slomka M, et al.  (2006) Genetic polymorphism of alcohol dehydrogenase 3 in alcohol liver cirrhosis and in alcohol chronic pancreatitis. Alcohol Alcohol  41: 14–17.  <ul class="find" data-citedArticleID="1081918" data-doi="10.1080/00365520600965723"><li><a href="http://dx.doi.org/10.1080/00365520600965723" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+polymorphism+of+alcohol+dehydrogenase+3+in+alcohol+liver+cirrhosis+and+in+alcohol+chronic+pancreatitis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+polymorphism+of+alcohol+dehydrogenase+3+in+alcohol+liver+cirrhosis+and+in+alcohol+chronic+pancreatitis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001881-Goldman1" id="pone.0001881-Goldman1"></a>Goldman D, Enoch MA (1990) Genetic epidemiology of ethanol metabolic enzymes: a role for selection. World Rev Nutr Diet  63: 143–160.  <ul class="find" data-citedArticleID="1081936"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+epidemiology+of+ethanol+metabolic+enzymes%3A+a+role+for+selection.&amp;auth=&amp;atitle=Genetic+epidemiology+of+ethanol+metabolic+enzymes%3A+a+role+for+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+epidemiology+of+ethanol+metabolic+enzymes%3A+a+role+for+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+epidemiology+of+ethanol+metabolic+enzymes%3A+a+role+for+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001881-Chen4" id="pone.0001881-Chen4"></a>Chen JG, Zhu J, Parkin DM, Zhang YH, Lu JH, et al.  (2006) Trends in the incidence of cancer in Qidong, China, 1978-2002. Int J Cancer.  119: 1447–1454.  <ul class="find" data-citedArticleID="1081914" data-doi="10.1002/ijc.21952"><li><a href="http://dx.doi.org/10.1002/ijc.21952" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trends+in+the+incidence+of+cancer+in+Qidong%2C+China%2C+1978-2002." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trends+in+the+incidence+of+cancer+in+Qidong%2C+China%2C+1978-2002.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001881-Fallin1" id="pone.0001881-Fallin1"></a>Fallin D, Schork NJ (2000) Accuracy of haplotype frequency estimation for biallelic loci, via the expectation-maximization algorithm for unphased diploid genotype data. Am J Hum Genet  67: 947–959.  <ul class="find" data-citedArticleID="1081932" data-doi="10.1086/303069"><li><a href="http://dx.doi.org/10.1086/303069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accuracy+of+haplotype+frequency+estimation+for+biallelic+loci%2C+via+the+expectation-maximization+algorithm+for+unphased+diploid+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accuracy+of+haplotype+frequency+estimation+for+biallelic+loci%2C+via+the+expectation-maximization+algorithm+for+unphased+diploid+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001881-Dean1" id="pone.0001881-Dean1"></a>Dean FB, Hosono S, Fang L, Wu X, Faruqi AF, et al.  (2002) Comprehensive human genome amplification using multiple displacement amplification. Proc Natl Acad Sci U S A  99: 5261–5266.  <ul class="find" data-citedArticleID="1081922" data-doi="10.1073/pnas.082089499"><li><a href="http://dx.doi.org/10.1073/pnas.082089499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+human+genome+amplification+using+multiple+displacement+amplification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+human+genome+amplification+using+multiple+displacement+amplification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001881-The1" id="pone.0001881-The1"></a>The International HapMap Consortium (2003) The International HapMap Project. Nature  426: 789–796.  <ul class="find" data-citedArticleID="1082028" data-doi="10.1038/nrg1351"><li><a href="http://dx.doi.org/10.1038/nrg1351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+International+HapMap+Project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+International+HapMap+Project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001881-Livak1" id="pone.0001881-Livak1"></a>Livak KJ, Marmaro J, Todd JA (1995) Towards fully automated genome-wide polymorphism screening. Nat Genet  9: 341–342.  <ul class="find" data-citedArticleID="1081976" data-doi="10.1038/ng0495-341"><li><a href="http://dx.doi.org/10.1038/ng0495-341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+fully+automated+genome-wide+polymorphism+screening." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+fully+automated+genome-wide+polymorphism+screening.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001881-Wright1" id="pone.0001881-Wright1"></a>Wright S (1931) Evolution in Mendelian populations. Genetics  16: 97–159.  <ul class="find" data-citedArticleID="1082052"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+in+Mendelian+populations.&amp;auth=&amp;atitle=Evolution+in+Mendelian+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+in+Mendelian+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+in+Mendelian+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001881-Stephens1" id="pone.0001881-Stephens1"></a>Stephens M, Smith NJ, Donnelly P (2001) A new statistical method for haplotype reconstruction from population data. Am J Hum Genet  68: 978–989.  <ul class="find" data-citedArticleID="1082020" data-doi="10.1086/319501"><li><a href="http://dx.doi.org/10.1086/319501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+statistical+method+for+haplotype+reconstruction+from+population+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+statistical+method+for+haplotype+reconstruction+from+population+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001881-Stephens2" id="pone.0001881-Stephens2"></a>Stephens M, Donnelly P (2003) A comparison of Bayesian methods for haplotype reconstruction. Am J Hum Genet  73: 1162–1169.  <ul class="find" data-citedArticleID="1082022" data-doi="10.1086/379378"><li><a href="http://dx.doi.org/10.1086/379378" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+Bayesian+methods+for+haplotype+reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+Bayesian+methods+for+haplotype+reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001881-Gu2" id="pone.0001881-Gu2"></a>Gu S, Pakstis AJ, Kidd KK (2005) HAPLOT: a graphical comparison of haplotype blocks, tagSNP sets and SNP variation for multiple populations. Bioinformatics  21: 3938–3939.  <ul class="find" data-citedArticleID="1081942" data-doi="10.1093/bioinformatics/bti649"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti649" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HAPLOT%3A+a+graphical+comparison+of+haplotype+blocks%2C+tagSNP+sets+and+SNP+variation+for+multiple+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HAPLOT%3A+a+graphical+comparison+of+haplotype+blocks%2C+tagSNP+sets+and+SNP+variation+for+multiple+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001881-Jreskog1" id="pone.0001881-Jreskog1"></a>Jöreskog KG (1977)  Factor analysis by least-square and maximum-likelihood method. In: Enslein K, Ralston A, Wilf RS, editors. Statistical Methods for Digital Computers, volume 3. New York: John Wiley &amp; Sons, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">78.
              </span><a name="pone.0001881-Endo1" id="pone.0001881-Endo1"></a>Endo T, Ikeo K, Gojobori T (1996) Large-scale search for genes on which positive selection may operate. Mol Biol Evol  13: 685–690.  <ul class="find" data-citedArticleID="1081928" data-doi="10.1093/oxfordjournals.molbev.a025629"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025629" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+search+for+genes+on+which+positive+selection+may+operate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+search+for+genes+on+which+positive+selection+may+operate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001881-Sabeti1" id="pone.0001881-Sabeti1"></a>Sabeti PC, Varilly P, Fry B, Lohmueller J, Hostetter E, et al.  (2007) Genome-wide detection and characterization of positive selection in human populations. Nature  449: 913–918.  <ul class="find" data-citedArticleID="1082006"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome-wide+detection+and+characterization+of+positive+selection+in+human+populations.&amp;auth=&amp;atitle=Genome-wide+detection+and+characterization+of+positive+selection+in+human+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+detection+and+characterization+of+positive+selection+in+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+detection+and+characterization+of+positive+selection+in+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001881-Voight1" id="pone.0001881-Voight1"></a>Voight BF, Kudaravalli S, Wen X, Pritchard JK (2006) A map of recent positive selection in the human genome. PLoS Biol  4(3): e72.  <ul class="find" data-citedArticleID="1082042" data-doi="10.1371/journal.pbio.0040072"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+map+of+recent+positive+selection+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+map+of+recent+positive+selection+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001881-Sabeti2" id="pone.0001881-Sabeti2"></a>Sabeti PC, Reich DE, Higgins JM, Levine HZ, Richter DJ, et al.  (2002) Detecting recent positive selection in the human genome from haplotype structure. Nature  419: 832–837.  <ul class="find" data-citedArticleID="1082008" data-doi="10.1038/nature01140"><li><a href="http://dx.doi.org/10.1038/nature01140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+recent+positive+selection+in+the+human+genome+from+haplotype+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+recent+positive+selection+in+the+human+genome+from+haplotype+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001881-Bandelt1" id="pone.0001881-Bandelt1"></a>Bandelt HJ, Forster P, Rohl A (1999) Median-joining networks for inferring intraspecific phylogenies. Mol Biol Evol  16: 37–48.  <ul class="find" data-citedArticleID="1081902" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Median-joining+networks+for+inferring+intraspecific+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Median-joining+networks+for+inferring+intraspecific+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.XML" value="153902"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.PDF" value="1209903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g001.PNG_L" value="7262480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g001.PNG_M" value="359401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g001.PNG_S" value="18612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g001.TIF" value="7675212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g001.PNG_I" value="111615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.t001.PNG_L" value="439929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.t001.PNG_M" value="247878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.t001.PNG_S" value="22182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.t001.TIF" value="1856476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.t001.PNG_I" value="78591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g002.PNG_L" value="785983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g002.PNG_M" value="95865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g002.PNG_S" value="9724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g002.TIF" value="1163388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g002.PNG_I" value="34055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g003.PNG_L" value="1410799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g003.PNG_M" value="126192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g003.PNG_S" value="8924"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g003.TIF" value="2380962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g003.PNG_I" value="43193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g004.PNG_L" value="1634685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g004.PNG_M" value="146957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g004.PNG_S" value="15576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g004.TIF" value="2683576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g004.PNG_I" value="48357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g005.PNG_L" value="615290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g005.PNG_M" value="151884"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g005.PNG_S" value="9731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g005.TIF" value="892716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g005.PNG_I" value="52251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g006.PNG_L" value="496256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g006.PNG_M" value="146784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g006.PNG_S" value="8914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g006.TIF" value="746710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g006.PNG_I" value="51356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g007.PNG_L" value="741876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g007.PNG_M" value="196006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g007.PNG_S" value="18611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g007.TIF" value="935078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g007.PNG_I" value="82550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g008.PNG_L" value="746501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g008.PNG_M" value="136899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g008.PNG_S" value="8861"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g008.TIF" value="1053172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.g008.PNG_I" value="45988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.e001.PNG" value="5467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.e001.TIF" value="21412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.e002.PNG" value="11437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.e002.TIF" value="26936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.e003.PNG" value="15912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.e003.TIF" value="32100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.s001.XLS" value="726528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001881.s002.TIF" value="691856"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001881&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001881" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001881&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001881&volume=&issue=&title=Ethnic Related Selection for an ADH Class I Variant within East Asia&author_name=Hui%20Li%2C%20Sheng%20Gu%2C%20Xiaoyun%20Cai%2C%20William%20C.%20Speed%2C%20Andrew%20J.%20Pakstis%2C%20Efim%20I.%20Golub%2C%20Judith%20R.%20Kidd%2C%20Kenneth%20K.%20Kidd&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001881" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881&amp;t=Ethnic%20Related%20Selection%20for%20an%20ADH%20Class%20I%20Variant%20within%20East%20Asia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881&title=Ethnic%20Related%20Selection%20for%20an%20ADH%20Class%20I%20Variant%20within%20East%20Asia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881&amp;title=Ethnic%20Related%20Selection%20for%20an%20ADH%20Class%20I%20Variant%20within%20East%20Asia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001881&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Ethnic Related Selection for an ADH Class I Variant within East Asia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001881';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Ethnic%20Related%20Selection%20for%20an%20ADH%20Class%20I%20Variant%20within%20East%20Asia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001881" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001881" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Agriculture%22" title="Search for articles in the subject area:'Agriculture'"><div class="flagText">Agriculture</div></a>
              <div data-categoryid="3263" data-articleid="26586"
                   data-categoryname="Agriculture"
                   class="flagImage" title="Flag 'Agriculture' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alcoholism%22" title="Search for articles in the subject area:'Alcoholism'"><div class="flagText">Alcoholism</div></a>
              <div data-categoryid="37705" data-articleid="26586"
                   data-categoryname="Alcoholism"
                   class="flagImage" title="Flag 'Alcoholism' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="26586"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Asia%22" title="Search for articles in the subject area:'Asia'"><div class="flagText">Asia</div></a>
              <div data-categoryid="40353" data-articleid="26586"
                   data-categoryname="Asia"
                   class="flagImage" title="Flag 'Asia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Geographic+distribution%22" title="Search for articles in the subject area:'Geographic distribution'"><div class="flagText">Geographic distribution</div></a>
              <div data-categoryid="19113" data-articleid="26586"
                   data-categoryname="Geographic distribution"
                   class="flagImage" title="Flag 'Geographic distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="26586"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="26586"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="26586"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9035'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1342'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2357&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15377">Yale study suggests evolutionary source of alcoholism's accidental enemy</a><br>Posted by HuiLiYale</p>
      <p><a href="/annotation/listThread.action?root=11343">discussion on Anthropology.com</a><br>Posted by HuiLiYale</p>
      <p><a href="/annotation/listThread.action?root=14417">Genetics is then handling the baton to anthropology</a><br>Posted by HuiLiYale</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
