

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000553"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000553" />

    <meta name="citation_title" content="Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function"/>
    <meta itemprop="name" content="Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function"/>

      <meta name="citation_author" content="Pearl A. Campbell"/>
            <meta name="citation_author_institution" content="Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="University of Ottawa, Department of Cellular and Molecular Medicine, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Carolina Perez-Iratxeta"/>
            <meta name="citation_author_institution" content="Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Miguel A. Andrade-Navarro"/>
            <meta name="citation_author_institution" content="Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="University of Ottawa, Department of Cellular and Molecular Medicine, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Michael A. Rudnicki"/>
            <meta name="citation_author_institution" content="Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="University of Ottawa, Department of Cellular and Molecular Medicine, Ottawa, Ontario, Canada"/>

    <meta name="citation_date" content="2007/6/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000553.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e553"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The derivation and potential use of human embryonic stem cells.; citation_author=AO Trounson; citation_journal_title=Reprod Fertil Dev; citation_volume=13; citation_number=1; citation_pages=523-532; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Formation of Pluripotent Stem Cells in the Mammalian Embryo Depends on the POU Transcription Factor Oct4.; citation_author=J Nichols; citation_author=B Zevnik; citation_author=K Anastassiadis; citation_author=H Niwa; citation_author=D Klewe-Nebenius; citation_journal_title=Cell; citation_volume=95; citation_number=2; citation_pages=379-391; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Involvement of Oct3/4 in the enhancement of neuronal differentiation of ES cells in neurogenesis-inducing cultures.; citation_author=K Shimozaki; citation_author=K Nakashima; citation_author=H Niwa; citation_author=T Taga; citation_journal_title=Development; citation_volume=130; citation_number=3; citation_pages=2505-2512; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The POU Domain Protein Spg (Pou2/Oct4) Is Essential for Endoderm Formation in Cooperation with the HMG Domain Protein Casanova.; citation_author=G Reim; citation_author=T Mizoguchi; citation_author=DY Stainier; citation_author=Y Kikuchi; citation_author=M Brand; citation_journal_title=Developmental Cell; citation_volume=6; citation_number=4; citation_pages=91-101; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative expression of Oct3/4 defines differentiation, dedifferentiation, or self-renewal of ES Cells.; citation_author=H Niwa; citation_author=J Miyazake; citation_author=A Smith; citation_journal_title=Nature Genetics; citation_volume=24; citation_number=5; citation_pages=372-376; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Oct-3/4 is a dose-dependent fate determinant.; citation_author=S Gidekel; citation_author=G Pizov; citation_author=Y Bergman; citation_author=E Pikarsky; citation_journal_title=Cancer Cell; citation_volume=4; citation_number=6; citation_pages=361-371; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ectopic expression of Oct4 blocks progenitor-cell differentiation and causes dysplasia in epithelial tissue.; citation_author=K Hochediinger; citation_author=Y Yamada; citation_author=C Beard; citation_author=R Jaenisch; citation_journal_title=Cell; citation_volume=121; citation_number=7; citation_pages=465-477; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human Embryonic Genes Re-expressed in Cancer Cells.; citation_author=M Monk; citation_author= Holding; citation_author= Cathy; citation_journal_title=Oncogene; citation_volume=20; citation_number=8; citation_pages=8085-8091; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Examination of POU Homeobox Gene Expression in Human Breast Cancer Cells.; citation_author=T Jin; citation_author= Branch; citation_author=R Donald; citation_author=Xiaoyun Zhang; citation_author=Shangle Qi; citation_author= Youngson; citation_author= Bruce; citation_author= Goss; citation_author=E Paul; citation_journal_title=International Journal of Cancer; citation_volume=81; citation_number=9; citation_pages=104-112; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Oct4 expression in adult human stem cells: evidence in support of the stem cell theory of carcinogenesis.; citation_author=M-H Tai; citation_author=C-C Chang; citation_author=LK Olson; citation_author=JE Trosko; citation_journal_title=Carcinogenesis; citation_volume=26; citation_number=10; citation_pages=495-502; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Core Transcriptional Regulatory Circuitry in Human Embryonic Stem Cells.; citation_author=LA Boyer; citation_author=TI Lee; citation_author=MF Cole; citation_author=SE Johnstone; citation_author=SS Levine; citation_journal_title=Cell; citation_volume=122; citation_number=11; citation_pages=1-10; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Oct4 and Nanog transcription network regulates pluripotency in mouse embryonic stem cells.; citation_author=Y-H Loh; citation_author=Q Wu; citation_author=J-L Chew; citation_author=VB Vega; citation_author=W Zhang; citation_volume=38; citation_number=12; citation_pages=431-440; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=B-catenin and TCF mediate stem cell positioning in the intestinal epitheleum by controlling the expression of EphB/ephrinB.; citation_author=HJ Batlle E; citation_author=H Beghtel; citation_author=MM van den Born; citation_author=E Sancho; citation_author=G Huls; citation_author=J Meeldijk; citation_author=J Robertson; citation_author=M van de Wetering; citation_author=T Pawson; citation_author=H Clevers; citation_journal_title=Cell; citation_volume=111; citation_number=13; citation_pages=251-263; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Regulatory networks in embryo-derived pluripotent stem cells.; citation_author=M Boiani; citation_author=H Scholer; citation_journal_title=Nature Reviews Molecular and Cellular Biology; citation_volume=6; citation_number=14; citation_pages=872-884; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptional Dynamics of the Embryonic Stem Cell Switch.; citation_author=V Chickarmane; citation_author=C Troein; citation_author=UA Nuber; citation_author=HM Sauro; citation_author=C Peterson; citation_journal_title=PLOS Computational Bioloby; citation_volume=2; citation_number=15; citation_pages=1080-1092; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Signalling, cell cycle, and pluripotency in embryonic stem cells.; citation_author=T Burdon; citation_author=A Smith; citation_author=P Savatier; citation_journal_title=Trends in Cell Biology; citation_volume=12; citation_number=16; citation_pages=432-438; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The functional landscape ot mouse gene expression.; citation_author=W Zhang; citation_author=QD Morris; citation_author=R Chang; citation_author=O Shai; citation_author=MA Bakowski; citation_journal_title=Journal of Biology; citation_volume=3; citation_number=17; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Study of stem cell function using microarray experiments.; citation_author=C Perez-Iratxeta; citation_author=G Palidwor; citation_author=CJ Porter; citation_author=NA Sanche; citation_author=MR Huska; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=18; citation_pages=1795-1801; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A stem cell molecular signature.; citation_author=N Ivanova; citation_author=J Dimos; citation_author=C Schaniel; citation_author=J Hackney; citation_author=K Moore; citation_journal_title=Science; citation_volume=298; citation_number=19; citation_pages=601-604; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=“Stemness”: transcriptional profiling of embryonic and adult stem cells.; citation_author=M Ramalho-Santos; citation_author=S Yoon; citation_author=Y Matsuzake; citation_author=RC Mulligan; citation_author=DA Melton; citation_journal_title=Science; citation_volume=298; citation_number=20; citation_pages=597-600; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Deconstructing stemnes.; citation_author=H Mikkers; citation_author=J Frisen; citation_journal_title=European Molecular Biology Organization; citation_volume=24; citation_number=21; citation_pages=2715-2719; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Filtering transcriptional noise during development: concepts and mechanisms.; citation_author=AM Arias; citation_author=P Hayward; citation_journal_title=Nature; citation_volume=7; citation_number=22; citation_pages=34-44; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in Saccaromyces cerevisiae.; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=F Robert; citation_author=DT Odom; citation_author=Z Bar-Joseph; citation_journal_title=Science; citation_volume=298; citation_number=23; citation_pages=799-894; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=GOstat: find statistically overrepresented Gene Ontologies within a group of genes.; citation_author=T Beissbarth; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=24; citation_pages=1464-1465; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of Oct4-dependent transcriptional networks regulating self-renewal and pluripotency in human embryonic stem cells.; citation_author=Y Babaie; citation_author=R Herwig; citation_author=B Greber; citation_author=TC Brink; citation_author=W Wruck; citation_journal_title=Stem Cells; citation_volume=25; citation_number=25; citation_pages=500-510; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Dissecting self-renewal in stem cells with RNA interference.; citation_author=N Ivanova; citation_author=R Dobrin; citation_author=R Lu; citation_author=I Kotenko; citation_author=J Levorse; citation_journal_title=Nature; citation_volume=442; citation_number=26; citation_pages=533-538; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DissectingOct3/4-regulated gene networks in embryonic stem cells by expression profiling.; citation_author=R Matoba; citation_author=H Niwa; citation_author=S Masui; citation_author=S Ohtsuka; citation_author=MG Carter; citation_journal_title=PlOS One; citation_volume=1; citation_number=27; citation_pages=e26; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chromatin remodeling complexes: strength in diversity, precision through specialization.; citation_author=BR Cairns; citation_journal_title=Current Opinion in Genetics&Development; citation_volume=15; citation_number=28; citation_pages=185-190; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=JmjC-domain-containing proteins and histone demethylation.; citation_author=RJ Klose; citation_author=EM Kallin; citation_author=Y Zhang; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=29; citation_pages=715-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptional repression by REST: recruitment of Sin3a and histone deacetylase to neuronal genes.; citation_author=Y Huang; citation_author=SJ Meyers; citation_author=R Dingledine; citation_journal_title=Nature Neuroscience; citation_volume=2; citation_number=30; citation_pages=867-872; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=REST and Its Corepressors Mediate Plasticity of Neuronal Gene Chromatin throughout Neurogenesis.; citation_author=N Ballas; citation_author=C Grunseich; citation_author=DD Lu; citation_author=JC Speh; citation_author=G Mandel; citation_journal_title=Cell; citation_volume=121; citation_number=31; citation_pages=645-657; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Genetic Screen for Candidate Tumor Suppressors Identifies REST.; citation_author=TF Westbrook; citation_author=ES Martin; citation_author=MR Schlabach; citation_author=Y Leng; citation_author=AC Liang; citation_journal_title=Cell; citation_volume=121; citation_number=32; citation_pages=837-848; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The neuronal repressor REST/NRSF is an essential regulator in medulloblastoma cells.; citation_author=P Lawinger; citation_author=R Venugopal; citation_author=ZS Guo; citation_author=A Immaneni; citation_author=D Sengupta; citation_journal_title=Nature Medicine; citation_volume=6; citation_number=33; citation_pages=826-831; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Distinct Profiles of REST Interactions with Its Target Genes at Different Stages of Neuronal Development.; citation_author=Y-M Sun; citation_author=DJ Greenway; citation_author=R Johnson; citation_author=M Street; citation_author=ND Belyaev; citation_journal_title=Mol Biol Cell; citation_volume=16; citation_number=34; citation_pages=5630-5638; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Stem Cells and Cancer: the Polycomb Connection.; citation_author=ME Valk-Lingbeek; citation_author=SWM Bruggeman; citation_author=M van Lohuizen; citation_journal_title=Cell; citation_volume=118; citation_number=35; citation_pages=409-418; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bmi-1 determines the proliferative capacity of normal and leukaemic stem cells.; citation_author=J Lessard; citation_author=G Sauvageau; citation_volume=423; citation_number=36; citation_pages=255-260; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Control of Developmental Regulators by Polycomb in Human Embryonic Stem Cells.; citation_author=TI Lee; citation_author=RG Jenner; citation_author=LA Boyer; citation_author=MG Guenther; citation_author=SS Levine; citation_journal_title=Cell; citation_volume=125; citation_number=37; citation_pages=301-313; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Suz12 binds to silenced regions of the genome in a cell-type-specific manner/gr.5306606.; citation_author=SL Squazzo; citation_author=H O'Geen; citation_author=VM Komashko; citation_author=SR Krig; citation_author=VX Jin; citation_journal_title=Genome Res; citation_volume=16; citation_number=38; citation_pages=890-900; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Suz12 binds to silenced regions of the genome in a cell-type-specific manner.; citation_author=SL Squazzo; citation_author=H O'Geen; citation_author=VM Komashko; citation_author=SR Krig; citation_author=VX Jin; citation_journal_title=Genome Research; citation_volume=16; citation_number=39; citation_pages=890-900; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chromatin compaction by a polycomb group protein complex.; citation_author=NJ Francis; citation_author=RE Kingston; citation_author=CL Woodcock; citation_journal_title=Science; citation_volume=306; citation_number=40; citation_pages=1574-1577; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Composition and histone substrates of polycomb repressive group complexes change during cellular differentiation.; citation_author=A Kuzmichev; citation_author=R Margueron; citation_author=A Vaquero; citation_author=TS Preissner; citation_author=M Scher; citation_journal_title=Proceedings of the National Academy of Sciences; citation_volume=102; citation_number=41; citation_pages=1859-1864; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Defective long-term repopulating ability in hematopoietic stem cells lacking the Polycomb-group gene rae28.; citation_author=JY Kim; citation_author=A Sawada; citation_author=S Tokimasa; citation_author=H Endo; citation_author=K Ozono; citation_journal_title=European Journal of Haematology; citation_volume=73; citation_number=42; citation_pages=75-84; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Binding of pRB to the PHD Protein RBP2 Promotes Cellular Differentiation.; citation_author=EV Benevolenskaya; citation_author=HL Murray; citation_author=P Branton; citation_author=RA Young; citation_author=J Kaelin; citation_author=G William; citation_journal_title=Molecular Cell; citation_volume=18; citation_number=43; citation_pages=623-635; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Acanthamoeba induces cell-cycle arrest in host cells.; citation_author=J Sissons; citation_author=S Alsam; citation_author=S Jayasekera; citation_author=KS Kim; citation_author=M Stins; citation_journal_title=J Med Microbiol; citation_volume=53; citation_number=44; citation_pages=711-717; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Role of pRB Dephosphorylation In Cell Cycle Regulation.; citation_author=S Tamrakar; citation_author=E Rubin; citation_author=JW Ludlow; citation_journal_title=Fronteirs in Bioscience; citation_volume=5; citation_number=45; citation_pages=d121-137; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The Nuclear Scaffold Protein NIPP1 Is Essential for Early Embryonic Development and Cell Proliferation.; citation_author=A Van Eynde; citation_author=M Nuytten; citation_author=M Dewerchin; citation_author=L Schoonjans; citation_author=S Keppens; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=46; citation_pages=5863-5874; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RB activation defect in tumor cell lines.; citation_author=C Broceno; citation_author=S Wilkie; citation_author=S Mittnacht; citation_journal_title=Proceedings of the National Academy of Sciences; citation_volume=99; citation_number=47; citation_pages=14200-14205; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Pathways governing G1/S transition and their response to DNA damage.; citation_author=J Bartek; citation_author=J Lukas; citation_journal_title=Federation of Biochemical Societies; citation_volume=490; citation_number=48; citation_pages=117-122; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Tudor domains track down DNA breaks.; citation_author=M Stucki; citation_author=SP Jackson; citation_journal_title=Nature Cell Biology; citation_volume=6; citation_number=49; citation_pages=1150-1152; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Caspase 3 activity is required for skeletal muscle differentiation.; citation_author=P Fernando; citation_author=JF Kelly; citation_author=K Balazsi; citation_author=RS Slack; citation_author=LA Megeney; citation_journal_title=PNAS; citation_volume=99; citation_number=50; citation_pages=11025-11030; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Parp1-deficiency induces differentiation of ES cells into trophoblast derivatives.; citation_author=M Hemberger; citation_author=T Nozaki; citation_author=E Winterhager; citation_author=H Yamamoto; citation_author=H Nakagama; citation_journal_title=Developmental Biology; citation_volume=257; citation_number=51; citation_pages=371-381; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=p53 induces differentiation of mouse embryonic stem cells by suppressing Nanog expression.; citation_author=T Lin; citation_author=C Chao; citation_author=Si Saito; citation_author=SJ Mazur; citation_author=ME Murphy; citation_journal_title=Nature Cell Biology; citation_volume=7; citation_number=52; citation_pages=165-171; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Depletion of BRCA1 impairs differentiation but enhances proliferation of mammary epithelial cells.; citation_author=S Furuta; citation_author=X Jiang; citation_author=B Gu; citation_author=E Cheng; citation_author=P-L Chen; citation_journal_title=PNAS; citation_volume=102; citation_number=53; citation_pages=9176-9181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=INO80 and [gamma]-H2AX Interaction Links ATP-Dependent Chromatin Remodeling to DNA Damage Repair.; citation_author=AJ Morrison; citation_author=J Highland; citation_author=NJ Krogan; citation_author=A Arbel-Eden; citation_author=JF Greenblatt; citation_journal_title=Cell; citation_volume=119; citation_number=54; citation_pages=767-775; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Linking histone deacetylation with the repair of DNA breaks.; citation_author=O Fernandez-Capetillo; citation_author=A Nussenzweig; citation_journal_title=PNAS; citation_volume=101; citation_number=55; citation_pages=1427-1428; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nuclear bodies and compartiments: functional roles and cellular signalling in health and disease.; citation_author=A Zimber; citation_author=Q-D Nguyen; citation_author=C Gespach; citation_journal_title=Cellular Signalling; citation_volume=16; citation_number=56; citation_pages=1085-1104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chromatin decondensation and nuclear reorganization of the HoxB locus upon induction of transcription.; citation_author=S Chambeyron; citation_author=WA Bickmore; citation_journal_title=Genes Dev; citation_volume=18; citation_number=57; citation_pages=1119-1130; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Maturation arrest of stem cell differentiation is a common pathway for the cellular origin of teratocarcinomas and epithelial cancers.; citation_author=S Sell; citation_author=G Pierce; citation_journal_title=Laboratory Investigations; citation_volume=70; citation_number=58; citation_pages=6-22; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Crystal Structure of a POU/HMG/DNA ternary complex suggests differential assembly of Oct4 and Sox2 on two enhancers.; citation_author=A Remenyi; citation_author=K Lins; citation_author=LJ Nissen; citation_author=R Reinbold; citation_author=H Scholer; citation_journal_title=Genes and Development; citation_volume=17; citation_number=59; citation_pages=2048-2059; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Now POU dimer configuration mediates antagonistic control of an osteopontin preimplantation enhancer by Oct-4 and Sox-2.; citation_author=V Botquin; citation_author=H Hess; citation_author=G Fuhrnamm; citation_author=C Anastassiadis; citation_author=MK Gross; citation_journal_title=Genes and Development; citation_volume=12; citation_number=60; citation_pages=2073-2090; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function"/>
    <meta name="twitter:description" content="Stem cells are characterized by two defining features, the ability to self-renew and to differentiate into highly specialized cell types. The POU homeodomain transcription factor Oct4 (Pou5f1) is an essential mediator of the embryonic stem cell state and has been implicated in lineage specific differentiation, adult stem cell identity, and cancer. Recent description of the regulatory networks which maintain &lsquo;ES&rsquo; have highlighted a dual role for Oct4 in the transcriptional activation of genes required to maintain self-renewal and pluripotency while concomitantly repressing genes which facilitate lineage specific differentiation. However, the molecular mechanism by which Oct4 mediates differential activation or repression at these loci to either maintain stem cell identity or facilitate the emergence of alternate transcriptional programs required for the realization of lineage remains to be elucidated. To further investigate Oct4 function, we employed gene expression profiling together with a robust statistical analysis to identify genes highly correlated to Oct4. Gene Ontology analysis to categorize overrepresented genes has led to the identification of themes which may prove essential to stem cell identity, including chromatin structure, nuclear architecture, cell cycle control, DNA repair, and apoptosis. Our experiments have identified previously unappreciated roles for Oct4 for firstly, regulating chromatin structure in a state consistent with self-renewal and pluripotency, and secondly, facilitating the expression of genes that keeps the cell poised to respond to cues that lead to differentiation. Together, these data define the mechanism by which Oct4 orchestrates cellular regulatory pathways to enforce the stem cell state and provides important insight into stem cell function and cancer."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000553.g004"/>

  <meta property="og:title" content="Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9375'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5174'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8069&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000553">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000553" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000553&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pearl A. Campbell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada, 
                  University of Ottawa, Department of Cellular and Molecular Medicine, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carolina Perez-Iratxeta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Miguel A. Andrade-Navarro, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada, 
                  University of Ottawa, Department of Cellular and Molecular Medicine, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael A. Rudnicki
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mrudnicki@ohri.ca">mrudnicki@ohri.ca</a></p>

                <p>Affiliations:
                  Sprott Centre for Stem Cell Research, Ottawa Health Research Institute, Ottawa, Ontario, Canada, 
                  University of Ottawa, Department of Cellular and Molecular Medicine, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 20, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000553</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000553-g001" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000553-g002" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000553-t001" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000553-g003" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000553-t002" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000553-g004" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000553">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000553" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Stem cells are characterized by two defining features, the ability to self-renew and to differentiate into highly specialized cell types. The POU homeodomain transcription factor <em>Oct4</em> (<em>Pou5f1</em>) is an essential mediator of the embryonic stem cell state and has been implicated in lineage specific differentiation, adult stem cell identity, and cancer. Recent description of the regulatory networks which maintain ‘ES’ have highlighted a dual role for <em>Oct4</em> in the transcriptional activation of genes required to maintain self-renewal and pluripotency while concomitantly repressing genes which facilitate lineage specific differentiation. However, the molecular mechanism by which <em>Oct4</em> mediates differential activation or repression at these loci to either maintain stem cell identity or facilitate the emergence of alternate transcriptional programs required for the realization of lineage remains to be elucidated. To further investigate <em>Oct4</em> function, we employed gene expression profiling together with a robust statistical analysis to identify genes highly correlated to <em>Oct4</em>. Gene Ontology analysis to categorize overrepresented genes has led to the identification of themes which may prove essential to stem cell identity, including chromatin structure, nuclear architecture, cell cycle control, DNA repair, and apoptosis. Our experiments have identified previously unappreciated roles for <em>Oct4</em> for firstly, regulating chromatin structure in a state consistent with self-renewal and pluripotency, and secondly, facilitating the expression of genes that keeps the cell poised to respond to cues that lead to differentiation. Together, these data define the mechanism by which <em>Oct4</em> orchestrates cellular regulatory pathways to enforce the stem cell state and provides important insight into stem cell function and cancer.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Campbell PA, Perez-Iratxeta C, Andrade-Navarro MA, Rudnicki MA (2007) Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function. PLoS ONE 2(6):
          e553.
            doi:10.1371/journal.pone.0000553</p><p><strong>Academic Editor: </strong>Janet Kelso, Max Planck Institute for Evolutionary Anthropology, Germany</p><p><strong>Received:</strong> January 17, 2007; <strong>Accepted:</strong> May 24, 2007; <strong>Published:</strong> June 20, 2007</p><p><strong>Copyright:</strong> © 2007 Campbell et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Embryonic Stem Cells (ESCs) are derived from the inner cell mass of the pre-implantation embryo and are characterized by their unlimited capacity for self-renewal and their ability to contribute to all cell lineages. The successful derivation and culture of human ESCs (hESCs) <a href="#pone.0000553-Trounson1">[1]</a> has opened the possibility of their use for generating cells for transplant, for tissue engineering or for drug development and testing. Importantly, full exploitation of the potential of hESCs will require the complete understanding of the function of the genetic factors that specify stem cell identity and regulate their commitment towards specific differentiated cell lineages. However, the transcriptional networks and molecular mechanisms that regulate the formation, self-renewal, and differentiation of hESC and mouse ESC (mESC) remain at best poorly understood.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p><em>Oct4 (Pou5f1)</em>, a POU-homeodomain transcription factor, plays a central role in self-renewal, pluripotency, and lineage commitment. Initially expressed as a maternal transcript, <em>Oct4</em> is required for the formation of a pluripotent inner cell mass <a href="#pone.0000553-Nichols1">[2]</a>. Moreover, strict control of <em>Oct4</em> expression is necessary to maintain ESC identity. Alterations in Oct4 expression promote differentiation and leads to the specification of ectodermal <a href="#pone.0000553-Shimozaki1">[3]</a>, endodermal <a href="#pone.0000553-Reim1">[4]</a>, or mesodermal <a href="#pone.0000553-Niwa1">[5]</a> primitive progenitors. Furthermore, <em>Oct4</em> has been shown to promote tumor growth in a dose dependent manner <a href="#pone.0000553-Gidekel1">[6]</a> and epithelial dysplasia by interfering with progenitor cell differentiation <a href="#pone.0000553-Hochediinger1">[7]</a>, is expressed in various human tumors <a href="#pone.0000553-Monk1">[8]</a>, <a href="#pone.0000553-Jin1">[9]</a> and adult stem cells <a href="#pone.0000553-Tai1">[10]</a> thus extending the role of <em>Oct4</em> from embryo to adult.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Recent identification of <em>Oct4</em> transcriptional targets in ESCs has revealed an unanticipated collaboration between <em>Oct4</em>, Sox2, and Nanog and provides a starting framework of the core transcriptional circuitry which maintains ‘ES’ through coordination of a series of feedback and feedforward loops <a href="#pone.0000553-Boyer1">[11]</a>, <a href="#pone.0000553-Loh1">[12]</a>. Furthermore, several signaling pathways including LIF/JAK/STAT, BMP, WNT, PI3K, MAPK/ERK, TGFβ and Notch <a href="#pone.0000553-BatlleE1">[13]</a>, <a href="#pone.0000553-Boiani1">[14]</a>, <a href="#pone.0000553-Chickarmane1">[15]</a>, <a href="#pone.0000553-Burdon1">[16]</a> have been shown to modulate stem cell function. Several key questions however still remain unresolved as a result of these studies. Firstly, what are the regulatory mechanisms that maintain self-renewal and pluripotency? Conversely, what are the molecular inputs that drive differentiation? Finally, and most importantly, can we deduce the essential themes that characterize stem cell function and thereby utilize this knowledge to gain insight into normal developmental processes to predict the consequences of aberrations to these processes that ultimately lead to human disease?</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To address these questions and further elucidate the factors that mediate stem cell function, we undertook an analysis to identify genes whose expression is correlated to Oct4. With the understanding that coexpression of genes may imply coregulation and participation in similar biological processes <a href="#pone.0000553-Zhang1">[17]</a>, we sought to identify genes which were correlated to Oct4 transcript expression in a wide variety of stem/progenitor populations which were analyzed by Affymetrix GeneChip technology as part of the Stem Cell Genomics Project <a href="#pone.0000553-PerezIratxeta1">[18]</a>. We hypothesized that by using Oct4 as a marker gene for self-renewal, pluripotency, and early lineage commitment, this analysis would lead to the identification of 1) Genes that are central to stem cell identity; 2) <em>Oct4</em> target genes; and 3) Genes that modulate <em>Oct4</em> function. Although several previous studies have sought to harmonize our understanding of ‘stemness’ <a href="#pone.0000553-Ivanova1">[19]</a>, <a href="#pone.0000553-RamalhoSantos1">[20]</a> it has been suggested that rather than the capacity for self-renewal and differentiation, the unique defining feature of a stem cell is that it represents a lasting steady-state of gene expression suspended in its differentiation pathway, yet maintaining the ability to respond to niche induced signals to carry out the indicated program of cellular specialization <a href="#pone.0000553-Mikkers1">[21]</a>. Insight into the juncture between cell extrinsic and intrinsic factors described above will provide an enhanced understanding of the molecular mechanisms which confer stem cells with this ability.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Lineage commitment can be described as a process whereby the unlimited ability for self-renewal and potency are gradually restricted as a cell progresses from one steady state of gene expression to the next. Recently attributed to stochastic events which increase the likelihood of a specific developmental outcome <a href="#pone.0000553-Arias1">[22]</a>, this view is in direct opposition to determinism, which precludes the processing of molecular cues emanating from the cellular niche. In juxtaposition to both the stochastic and deterministic models of development is the view that cellular commitment is facilitated by a hierarchy of transcriptional regulatory networks <a href="#pone.0000553-Lee1">[23]</a> which exert precise biological control by combinatorial interactions at the protein-protein, and protein-DNA level. The function of these networks is highly responsive to molecular inputs, allowing the rapid processing and relay of information required for either maintenance of a specific cellular state, or progression to an altered steady state. Importantly, our data suggests that Oct4 maintains stem cell identity by targeting key regulatory genes which play critical roles in determining cell fate.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Oct4 Correlation Analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A set of 45 murine samples collected as part of the <em>Stem Cell Genomics Project</em> and deposited in <em>StemBase</em> <strong>(</strong><a href="http://www.StemBase.ca/"><strong>http://www.StemBase.ca/</strong></a><strong>)</strong> <a href="#pone.0000553-PerezIratxeta1">[18]</a> were selected to form the basis of this analysis (<a href="#pone.0000553.s001">Supplemental Table S1</a>). A wide variety of samples comprising adult and embryonic stem cells and their differentiated derivatives were collected in biological triplicate and hybridized to the Affymetrix MOE430 GeneChip Set for a total of 270 GeneChips. Following normalization, scaling, and filtering of the data the standard Pearson correlation coefficient (rho) between every probeset which passed the filter, to the <em>Oct4</em> probeset was computed. A probeset was considered correlated to <em>Oct4</em> if |rho|≥0.75. This computation was repeated 10,000 times with random subsets consisting of 65% to 70% of the data. Probesets that were correlated to the Oct4 associated probe in at least 40% of the trials were retained for further analysis (<a href="#pone.0000553.s002">Supplemental Table S2</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The stringency of our correlation analysis is set by two parameters; |rho|≥0.75 and the percentage of trials in which this value for rho is met or exceeded. In setting these parameters our aim was to prioritize genes for analysis which may have either represented <em>Oct4</em> targets or genes which were implicated in self-renewal, pluripotency, or early lineage commitment. The values were pragmatic in nature; chosen as such to produce a reasonable number of genes which could be analyzed in a coherent fashion, possibly being able to provide a snapshot as it were of ‘stemness’. The use of more or less stringent parameters would result in the identification of fewer or more genes. Of note, cursory examination of the cutoffs used reveals that should we have increased the percentage of trials for which |rho|≥0.75 from 40% to 50% we would not have identified at least two previously identified Oct4 targets; Sox2 (49%) and Cdyl (40%) <a href="#pone.0000553-Boyer1">[11]</a>, <a href="#pone.0000553-Loh1">[12]</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>As a result of this analysis 1299 probesets (1155 unique transcripts) were found to be correlated to <em>Oct4</em>. Seventy-five probesets (69 transcripts) were negatively correlated, while 1224 probesets (1086 transcripts) were positively correlated. The validity of this method for the identification of genes related to stem cell identity is assured by the presence of genes which have previously defined roles in ESCs such as <em>Utf1, Fgf4, Nanog</em>, and <em>Sox2</em> which were correlated to Oct4 in 100%, 99%, 97% and 49% of the trials respectively. Comparison of the transcript expression levels of <em>Oct4</em> and correlated <em>Nanog, Sox2, Tdrd7, Mef2a</em>, and uncorrelated <em>Myog</em> across all samples utilized in this analysis demonstrates the range of <em>Oct4</em> expression in these samples and also lends meaning at a biological level to the statistical analysis performed (<a href="#pone-0000553-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0000553-g001"><div class="img"><a name="pone-0000553-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Oct4 Correlation Analysis.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>ESC, EC, myogenic, neuronal, retinal, and hematopoietic stem cells and their differentiated derivatives underwent Affymetrix gene expression profiling as part of the Stem Cell Genomics Project. A set of 45 samples were profiled in biological triplicate and hybridized to the MOE430 GeneChip set. Mean intensity values for each biological triplicate are plotted in log scale on the Y-axis, with an approximate cutoff of 1000 demarcating detection status of each gene represented. Transcript expression levels of genes positively (<em>Nanog, Sox2</em>), negatively (<em>Tdrd7</em> and <em>Mef2a</em>), and not (<em>Myog</em>) correlated to <em>Oct4</em> are displayed. Detection calls of ‘Present’ for <em>Oct4</em> are depicted by solid black squares. ‘Absent’ calls are represented by open black squares.</p>
<span>doi:10.1371/journal.pone.0000553.g001</span></div>

<h4>GO Categorization of Oct4 Correlated Genes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In order to gain insight into the functions of <em>Oct4</em> correlated genes, GOstat analysis <a href="#pone.0000553-Beissbarth1">[24]</a> was performed. As a result of this analysis a number of gene ontology (GO) categories were found to be correlated to Oct4 expression. Many over-represented terms were related to transcription and DNA replication (nucleic acid binding, DNA helicase, nucleolus), RNA processing (rRNA processing, splicesome complex, and RNA splicing), and cellular localization (nucleolus and Cajal body). Many under-represented terms were related to inter-cellular communication (cell communication, receptor activity, signal transduction). A complete output from GOstat is provided (<a href="#pone.0000553.s003">Supplemental Table S3</a>). Because this method of analysis is highly dependent upon the GO categories associated with a specific gene, the use of alternate GO databases can result in divergent findings. Moreover, such analyses are limited by the availability of databases which possess accurate annotations that keep pace with current research.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To overcome these limitations, further refinement of GO classifications for the <em>Oct4</em> correlated genes was performed by manual curation of a wide variety of databases such as <em>NetAffx, GeneCards, Ensembl, Stanford Source, Bioinformatics Harvester</em>, and <em>PubMed</em> (<a href="#pone.0000553.s002">Supplemental Table S2</a>). This analysis revealed that the categories transcriptional regulation, intracellular signaling, mRNA splicing, cell cycle, DNA repair, and chromatin were highly represented within the positively <em>Oct4</em> correlated genes. Categories highly represented within the negatively correlated genes included transcriptional regulation, protein modification, transport, intracellular signaling, and apoptosis. A summary of these findings is provided in <a href="#pone-0000553-g002">Figure 2</a> with representative genes in highly enriched categories provided in <a href="#pone-0000553-t001">Table 1</a>. Of note, these findings are highly consistent with a previously published GO analysis performed following <em>Oct4</em> knockdown in hESC <a href="#pone.0000553-Babaie1">[25]</a>.</p>
<div class="figure" id="pone-0000553-g002"><div class="img"><a name="pone-0000553-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Functional classification of <em>Oct4</em> correlated genes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Manual curation of databases reveals highly enriched Gene Ontology (GO) categories for <em>Oct4</em> positively (A) and negatively (B) correlated genes. Numbers displayed represent percentage of unique transcripts attributed to each category with only the most abundant categories listed individually.</p>
<span>doi:10.1371/journal.pone.0000553.g002</span></div><div class="figure" id="pone-0000553-t001"><div class="img"><a name="pone-0000553-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000553.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000553.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Categories of Genes Identified as Oct4 Correlated.</span></strong></p><span>doi:10.1371/journal.pone.0000553.t001</span></div>

<h4>Target Gene Validation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To validate our premise that this analysis would lead to the identification of Oct4 direct transcriptional targets, we performed a screen scanning the genomic region from 2 kb upstream of the transcriptional start site to 2 kb downstream from the 3-prime end of the transcribed region of the correlated genes for the presence of neighboring <em>Oct4</em> and dimerization partner <em>Sox2</em> binding sites (Supplemental <a href="#s3">Material and Methods</a>). As a result of this analysis 392 genes were found to possess at least one putative composite binding site (<a href="#pone.0000553.s004">Supplemental Table S4</a>) with several genes such as <em>Oct1/Pou2f1, Smyd3</em>, and <em>Ranbp17</em> containing multiple (17, 16, and 14 respectively) putative sites, which may reflect a requirement for strict regulatory control of these genes throughout development. Although one might predict that genes containing multiple binding sites would show a higher degree of correlation to <em>Oct4</em>, a very cursory analysis of the data reveals that this is in fact not the case. Genes containing 1 <em>Oct4</em>/<em>Sox2</em> binding site (and % correlation) are: <em>Lig3</em> (+62), <em>Kctd3</em> (+91), <em>Bin1</em> (−41), <em>Bmi1</em> (−55), <em>Nasp</em> (99 and 79-two probesets). Genes containing from 5 to 10 sites include: <em>Insig2</em> (−61), <em>Ipo11</em> (+92), <em>Myst4</em> (+94), <em>Nr6a1</em> (52 and 53) and <em>Strbp</em> (52). Genes with greater than 10 sites are: <em>Pou2f1</em> (+50), <em>Ranbp17</em> (+99), and <em>Smyd3</em> (+45).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Validation of 28 of these loci by chromatin immunoprecipitation (ChIP) followed by quantitative real-time PCR (QRT-PCR) confirmed the identification of 26 <em>Oct4</em> direct transcriptional targets (<a href="#pone-0000553-g003">Figure 3</a>; <a href="#pone.0000553.s006">Supplemental Table S6</a>). Notably, since the completion of our studies, these findings have been confirmed by several groups <a href="#pone.0000553-Boyer1">[11]</a>, <a href="#pone.0000553-Loh1">[12]</a>, <a href="#pone.0000553-Babaie1">[25]</a>, <a href="#pone.0000553-Ivanova2">[26]</a>, <a href="#pone.0000553-Matoba1">[27]</a>.</p>
<div class="figure" id="pone-0000553-g003"><div class="img"><a name="pone-0000553-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Validation of Oct4 targets.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Chromatin immunoprecipitation (ChIP) assays were performed with <em>Oct4</em> and <em>IgG</em> antibody and no antibody as a negative control followed by Quantitative Real-time PCR analysis (ChIP/QRT-PCR) for putative positively regulated <em>Oct4</em> target (A), negatively regulated Oct4 target (B), and non-validated (C) genes. *8L16Rik represents <em>1110008L16Rik.</em> Results are from two independent ChIP assays, with duplicate QRT-PCR assessment for each. Error bars denote standard error of the mean (SEM).</p>
<span>doi:10.1371/journal.pone.0000553.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Further examination of these directly regulated target genes in the context of the correlated gene-list reveals important insights into how <em>Oct4</em> regulates pivotal pathways involved in controlling pluripotency, self-renewal and early lineage commitment.</p>


<h4>Oct4 Correlated Genes are Implicated in Chromatin Regulation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Recent experiments indicate that chromatin organization is dynamic and is subject to regulatory mechanisms that enforce the transcriptional potential of the genome during cellular commitment and differentiation. Chromatin is remodeled into transcriptionally permissive or repressive conformations by complexes that covalently modify histones, act in an ATP-dependent manner to reposition nucleosomes along DNA, or facilitate histone exchange. Several complexes have been identified including SWI/SNF, ISWI, INO80, and M1-2/CHD, and Trithorax group (TrxG), and Polycomb group (PcG) proteins which mediate chromatin remodeling by facilitating epigenetic modification of histone tails to activate or repress gene expression, respectively <a href="#pone.0000553-Cairns1">[28]</a>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Thirty five genes implicated in chromatin remodeling were correlated to Oct4. Putative positive target genes include SWI/SNF members <em>Smarcc1</em>, AT rich interactive domain (Swi1 like) containing proteins (ARID domains) <em>Arid1a, Arid5b, Jarid1b</em> and <em>Jarid2</em>, which was confirmed as a direct <em>Oct4</em> target. Notably, these ARID domain containing proteins, a subset of the Jumonji C family, have recently been associated with histone demethylase activity <a href="#pone.0000553-Klose1">[29]</a>. Several other genes containing MYST, SET, and CHROMO, and BROMO domains, which facilitate or recognize specific histone modifications, were also identified.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p><em>Rest</em> has been implicated in the repression of neuronal specific genes via its ability to recruit cofactors such as histone deacetylases (HDACs), Corest, Sin3, and Mecp2 <a href="#pone.0000553-Huang1">[30]</a>. The identification of <em>Rest</em> as a direct <em>Oct4</em> target, in light of its role in maintaining chromatin plasticity throughout neurogenesis, <a href="#pone.0000553-Ballas1">[31]</a> provides a mechanistic understanding of <em>Oct4's</em> role in promoting neural differentiation <a href="#pone.0000553-Shimozaki1">[3]</a>. Ironically, <em>Rest</em> has recently been described as both a tumor suppressor <a href="#pone.0000553-Westbrook1">[32]</a> and an oncogene <a href="#pone.0000553-Lawinger1">[33]</a>. The identification of <em>Corest</em> and <em>Mecp2</em> as respectively positively and negatively correlated to <em>Oct4</em> may provide insight into the dynamic nature of <em>Rest</em> co-repressor complexes throughout development that could explain these seeming incongruities. Furthermore, this hypothesis is supported by the recent description of the changing Rest-regulon in the progression from embryonic stem cells to neural stem cells (NSC) to differentiated neurons <a href="#pone.0000553-Sun1">[34]</a></p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Importantly, several members of the TrxG and PcG of transcription factors such as <em>Ash1l, Suz12, Ash2l, Phc1</em>, and <em>Rnf134, Bmi1,</em> and <em>Phc3</em> were correlated to Oct4, with the five latter genes validated as <em>Oct4</em> targets. Diverse functions for PcG and TrxG genes in cancer, cell cycle control, and stem cell function have recently been described <a href="#pone.0000553-ValkLingbeek1">[35]</a>, <a href="#pone.0000553-Lessard1">[36]</a>, <a href="#pone.0000553-Lee2">[37]</a>, <a href="#pone.0000553-Squazzo1">[38]</a>. The direct transcriptional regulation of several members of these complexes places Oct4 central to the coordination of these activities. The localization of Suz12, a member of Polycomb Repressor Complex 2 (PRC2) at many Oct4 repressed loci in ESC <a href="#pone.0000553-Lee2">[37]</a>, <a href="#pone.0000553-Squazzo2">[39]</a> provides indication that Oct4-Polycomb interaction may play a significant role in the active repression of lineage. Furthermore, knock-down or overexpression of Oct4 has been shown to result in perturbed expression of several members of PcG and TrxG that we have identified as Oct4 targets and has led to loss of the pluripotent state <a href="#pone.0000553-Ivanova2">[26]</a>, <a href="#pone.0000553-Matoba1">[27]</a>. A comparison of the results of this study to the previous studies can be found in <a href="#pone-0000553-t002">Table 2</a>. Taken together, these data provide strong support for Oct4's role in maintaining chromatin structure in mESC via regulation of and interaction with a unique constellation of PcG and TrxG complexes.</p>
<div class="figure" id="pone-0000553-t002"><div class="img"><a name="pone-0000553-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000553.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000553.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Cross-Study Comparison of Oct4 Target Genes</span></strong></p><span>doi:10.1371/journal.pone.0000553.t002</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>The negative correlation between <em>Bmi1</em> and <em>Oct4</em> was surprising in light of its role in maintaining hematopoietic and neuronal stem cells (HSCs, NSCs). Although necessary for self-renewal of HSCs and NSCs, expression of <em>Bmi1,</em> which leads to chromatin condensation and stable gene silencing <a href="#pone.0000553-Francis1">[40]</a> may be inconsistent with self-renewal in pluripotent cells. Pluripotency involves the ability to repress genes whose expression would result in a loss of potential while retaining the ability to reawaken these transcriptional programs upon differentiation. Therefore, while transcriptional repression is necessary in both pluripotent cells and their differentiated progeny, the means to accomplish it may, of necessity, be entirely different.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>PcGs exist as developmentally regulated multi-subunit complexes <a href="#pone.0000553-Kuzmichev1">[41]</a>. Therefore it is predicted that alterations in the balance of PcG members would have profound implications for maintenance of the stem cell state. If, as anticipated above, inappropriate upregulation of <em>Bmi1</em> (and/or <em>Phc3</em>) leads to the repression of genes that are required for pluripotency, this may ultimately be manifested in a cell's inability to differentiate and may provide a partial explanation for the oncogenic roles of these proteins. Conversely, it is postulated that downregulation of other PcG members such as <em>Phc1</em> would result in the de-repression of genes required for differentiation which would compromise self-renewal <a href="#pone.0000553-Kim1">[42]</a>.</p>


<h4>Cell Cycle Control in Stem Cells Requires Inactivation of pRb for Self-Renewal, Activation for Differentiation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Carefully regulated execution of cell cycle progression is accomplished in stem cells by a unique constellation of genes which impact self-renewal and lineage commitment. Activation of intracellular signaling pathways such as <em>PI3K, Ras/Raf</em>, and <em>Jak</em>/<em>Stat</em> by molecular cues emanating from the stem cell niche mediate phosphorylation events which control the activity of cyclin/CDK complexes and culminate in the modulation of genes (such as <em>pRb</em> and <em>Trp53</em>) that are implicated in cell cycle checkpoint, cell cycle exit, and differentiation <a href="#pone.0000553-Benevolenskaya1">[43]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Assessment of GO terms revealed that 38 cell-cycle related genes were positively correlated to <em>Oct4</em> including <em>Cdc25a, Gspt1, Ppp1r8, Ccnb2, Ccne1, Ccna2</em>, Ccnb1, and <em>Ccnf</em>. Validated <em>Oct4</em> target <em>Ccnf</em> is implicated in cell cycle control at the G1/S and G2/M checkpoints and has recently been associated with the maintenance of <em>pRb</em> in a hyperphosphorylated, inactive state <a href="#pone.0000553-Sissons1">[44]</a>. The role of <em>Ccnf</em> in this process may in part be due to the E3 ubiquitin ligase domain of <em>Ccnf</em> to mediate the degradation of phosphatases such as <em>Pp1</em> involved in the sequential activation of <em>pRb</em> through G1/S and G2/M <a href="#pone.0000553-Tamrakar1">[45]</a><em>.</em> Conversely, the significantly Oct4 correlated (92%) Pp1 negative regulatory subunit Nipp1 (Ppp1r8) may facilitate the functional inactivation of <em>pRb.</em> This hypothesis is consistent with the requirement of <em>Nipp1</em> in early embryonic development <a href="#pone.0000553-VanEynde1">[46]</a> and points toward a potential role for <em>Nipp1</em> in tumorigenesis <a href="#pone.0000553-Broceno1">[47]</a>. In addition to its role in cell cycle control is also involved in mRNA splicing, and transcriptional repression through interactions with the PcG complexes making it an important putative <em>Oct4</em> target, capable of integrating the diverse functions of cell cycle control, alternate splicing, chromatin structure, and transcriptional regulation <a href="#pone.0000553-VanEynde1">[46]</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Based upon this analysis it is predicted that alterations in the expression of Oct4 correlated genes such as <em>Ccnf</em> or <em>Nipp1</em> that impact the functional status of <em>pRb</em> (or <em>pRb</em> family member <em>p107</em>) would have profound consequences. Inactivation of <em>pRb</em> is required for self-renewal; activation of <em>pRb</em> is obligatory for cell cycle exit and differentiation. An imbalance in either of these processes, possibly emanating from deregulated signaling from the stem cell niche or mutations in the key regulators would lead to unrestrained cellular proliferation.</p>


<h4>Genes Involved in Apoptosis and DNA Repair are Correlated to Oct4 and are Implicated in Stem Cell Differentiation</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Prevailing thought holds that the initial stages of apoptosis involve the caspase mediated induction of DNA strand breaks and either the recruitment of DNA repair genes that act in concert to halt cell cycle progression and restore genomic stability or, if the damage is not able to be repaired, in further cleavage of DNA, nuclear blebbing, and other processes which have been elegantly and thoroughly described elsewhere that culminate in programmed cell death. Both apoptosis and DNA repair are regulated by several multi-component complexes with the roles of <em>Trp53, Brca1</em>, and <em>pRb</em> being central to their coordination <a href="#pone.0000553-Bartek1">[48]</a>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Analysis of the <em>Oct4</em> gene list revealed an important emerging theme; mechanisms to actively repress apoptotic pathways are involved in maintaining the stem cell state. Twenty-five apoptotic genes were positively correlated to <em>Oct4</em>, the majority of which, including <em>Aatf, Api5, Aven, Bag4, Commd10, Nipa</em>, and <em>Opa1,</em> function to inhibit apoptosis. In addition, <em>Bin1, Blp1, Serpinb9, Sh3glb1</em>, and <em>Casp6</em>, all apoptosis inducing genes, were found to be negatively correlated to Oc<em>t4</em>, with <em>Sh3glb1</em> and <em>Casp6</em> confirmed as targets.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Thirty genes implicated in DNA damage and repair, were positively correlated to <em>Oct4</em>. Members of the <em>Brca1</em> associated surveillance complex (BASC) including <em>Brca1, Msh2, Mre11a, Rad51, Blm, Chek1</em>, as well as <em>Parp1, Trp53, Fancd2, Tdrd7, and Xrcc5</em>, were included. The validation of <em>Trp53, Tdrd7, Brca1</em>, and <em>Parp1</em> as direct <em>Oct4</em> targets strengthens the importance of this group of genes in stem cell function.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>The high frequency at which apoptotic genes were negatively correlated to <em>Oct4</em> and anti-apoptotic genes were positively correlated to <em>Oct4</em> implies that ‘anti-apoptosis’ is an important theme for maintaining the stem cell state. Conversely, this may also suggest that genes which modulate the initial response to aberrant chromatin structure, apoptosis, and DNA repair, may play important roles in lineage commitment. This notion is consistent with the role of tudor domain containing proteins (such as <em>Tdrd7</em>) in DNA damage response <a href="#pone.0000553-Stucki1">[49]</a>, <em>Casp3</em> in skeletal muscle differentiation <a href="#pone.0000553-Fernando1">[50]</a>, and the roles of <em>Parp1</em> <a href="#pone.0000553-Hemberger1">[51]</a>, <em>Trp53</em> <a href="#pone.0000553-Lin1">[52]</a>, and <em>Brca1</em><a href="#pone.0000553-Furuta1">[53]</a> to modulate differentiation. Interestingly, a relationship between DNA damage repair, chromatin remodeling <a href="#pone.0000553-Morrison1">[54]</a>, and histone deacetylation <a href="#pone.0000553-FernandezCapetillo1">[55]</a>, all previously implicated in cellular differentiation, has recently been described. Moreover, knowledge of the normal developmental functions of these genes in cellular differentiation provides mechanistic insight into how these genes, when mutated, lead to cancer.</p>


<h4>Nuclear Architecture in Stem Cells Reinforces Their Defining Characteristics</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The nucleus is the site of many processes that profoundly impact cellular phenotype including transcription, mRNA splicing, and DNA replication and repair. Research has revealed that in fact control of these activities is coordinated in a dynamic, spatio-temporal manner. The presence of specific nuclear structures (nuclear bodies; NBs), whose function is to concentrate key regulatory molecules, mainly to loci of actively transcribed genes, facilitates this coordination <a href="#pone.0000553-Zimber1">[56]</a>.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>As a result of this analysis several key molecules whose presence is indicative of NBs were observed. <em>Pml</em> and <em>Coil</em> (Cajal Bodies and PML Bodies), <em>Gemin4</em> and <em>Gemin5</em> (Gems), <em>Nup35, 43, 54, 98, 133, 160, 188</em> (Nuclear Pore Complex), <em>Ncl</em> and <em>Nolc1</em> (Nucleolus) and 46 genes implicated in RNA metabolism (Splicing Speckles, Spliceosome, Exosome, and Cajal Bodies) were positively correlated to Oct4. Several genes implicated in nuclear transport such as <em>Ipo11, Kpna1, Tnpo2</em> and <em>3, Xpot, Gle1l, Xpo5</em> and <em>6</em> and direct Oct4 targets <em>Igf2bp1</em> and <em>Phb</em> were also positively correlated.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>The incidence of nuclear bodies is incremental with cellular proliferative capacity and their localization is predominately to transcriptionally active regions of chromatin, although the mechanisms that direct their localization are largely unknown. Based upon the high degree of correlation of <em>Oct4</em> to constituents of NBs, it is conceivable that <em>Oct4</em> target binding may function to modulate the accessibility of local chromatin to these structures and thereby enforce the transcriptional potential of specific genetic loci in early development. The identification of Hoxb1 as a negatively regulated <em>Oct4</em> target is consistent with this hypothesis in light of the recent finding that in ESCs <em>Hoxb1</em>, although not expressed, is poised at the surface of its chromosome territory. In the initial stages of differentiation <em>Hoxb1</em> is transcriptionally activated which results in chromatin decondensation and reorientation of this locus to the nuclear centre <a href="#pone.0000553-Chambeyron1">[57]</a>. Together, these findings lead us to predict that <em>Oct4</em> binding functions not only in the transcriptional repression of genes that would otherwise facilitate lineage commitment, but also presents a means whereby these loci are organized spatially within the nucleus so as to be poised for activation given the appropriate cue.</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>In addition to the normal physiological roles for NBs described above, they also play key roles in the response to DNA damage, DNA repair, apoptosis, and senescence. Loss of regulation in the recruitment and coordination of key genes contained in these structures (<em>Trp53, Pml, Brca1, Blm</em>, etc.) would be predicted to have profound implications in the ability of a cell to respond to signals that would lead to differentiation. Such dysregulation is associated with the accumulation of NBs at sites of DNA damage (DNA damage induced foci) and is implicated in several types of cancer such as acute promyelocytic leukemia (<em>Pml-Rara</em> translocation) and Bloom's Syndrome <a href="#pone.0000553-Zimber1">[56]</a>.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Through the use of gene expression data compiled from a vast collection of adult and embryonic stem cells and their differentiated derivatives we have performed a robust statistical analytic method to identify genes that are correlated to <em>Oct4</em>. Although several previous studies have mapped transcriptional targets of <em>Oct4</em>, we believe that this study provides further insight into the transcriptional regulatory networks, factors, and cofactors that modulate stem cell function. Importantly, our experiments have revealed hitherto unappreciated roles for <em>Oct4</em> for firstly, regulating chromatin structure in a state consistent with self-renewal and pluripotency, and secondly, facilitating the expression of genes that keeps the cell poised to respond to cues that lead to differentiation. Furthermore, our analyses has led to the elucidation of themes that are essential for maintaining ‘ES’ including permissive chromatin structure, nuclear architecture, cell cycle control, apoptosis, and DNA repair. Finally, we have identified 26 direct Oct4 transcriptional targets which may represent candidate regulatory nodes by which cell fate decisions could be directed to facilitate the use of hESCs in therapeutic and regenerative medicine (<a href="#pone-0000553-g004">Figure 4A</a> and <a href="#pone.0000553.s002">Table S2</a>).</p>
<div class="figure" id="pone-0000553-g004"><div class="img"><a name="pone-0000553-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000553" data-uri="info:doi/10.1371/journal.pone.0000553.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000553.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000553.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000553.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000553.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The <em>Oct4</em> transcriptional regulatory network.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>Validated Oct4 targets (A) are indicated by solid red or green lines. Red and green indicate negative and positive regulation, respectively for all cases. Dashed lines emanating from Oct4 indicate putatively regulated genes. Solid black lines represent potential regulatory nodes that could facilitate the directed differentiation of ESCs. The pressures that preserve stem cell function and modulate early lineage commitment are diametrically opposed. While Oct4 acts to maintain self-renewal and pluripotency in the undifferentiated ‘ES’ state by its modulation of genes that act to maintain permissive chromatin structure, DNA repair, anti-apoptosis, and inactive <em>pRb</em> (B), in differentiation the balance of these forces is altered to favour repressive chromatin structure, DNA checkpoint control, apoptosis, and active pRB which facilitate cellular commitment (C).</p>
<span>doi:10.1371/journal.pone.0000553.g004</span></div>

<h4>The Oct4 Transcriptional Regulatory Network</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>The expression of <em>Oct4</em> in various forms of human cancer <a href="#pone.0000553-Monk1">[8]</a>, <a href="#pone.0000553-Jin1">[9]</a> and a recently described role for <em>Oct4</em> in adult stem cells <a href="#pone.0000553-Tai1">[10]</a> and the expansion of epithelial progenitor cells <a href="#pone.0000553-Hochediinger1">[7]</a> supports the theory that cancer is a disease of stem cells. This theory postulates that cancers arise in stem cells or early committed progenitors <a href="#pone.0000553-Sell1">[58]</a> due to their inability to differentiate in a regulated fashion. <em>Oct4</em> directly regulates the transcription of genes such as <em>Trp53, Brca1, Parp1</em>, and <em>Bmi1</em> which play a central role in a cell's proclivity to undergo transformation, apoptosis, senescence, and now differentiation.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>The process of development and the commitment to differentiate is guided by the ordered expression and repression of genes required to enforce specific transcriptional programs. Knowledge of the emerging Oct4 transcriptional regulatory network provides a means whereby we can begin to understand the molecular mechanisms that guide these processes and gain insight into aberrations that lead to disease. While the stem cell state is guarded by highly dynamic, complex, and interrelated mechanisms which impact the repertoire, location, and functional state of expressed genes, lineage commitment can be described as a process whereby the unlimited ability for self-renewal and potency are gradually restricted as a cell progresses from one steady state of gene expression to the next. These diametrically opposed states are mediated by a contrasted balance of forces that impact chromatin structure, nuclear architecture, cell cycle, DNA repair, and apoptosis (<a href="#pone-0000553-g004">Figure 4B and C</a>). Further examination of the interactions among the genes identified as a result of this study will provide a more thorough understanding of the pressures that guide cell fate. Critically, only by understanding the normal developmental function of a gene can we begin to understand the role that it may play in disease. Importantly, our experiments have defined how <em>Oct4</em>, as the master regulator of embryonic stem cell function, plays a central role in regulating key genes in pivotal pathways involved in controlling pluripotency, self-renewal and differentiation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Stem Cell Culture and Isolation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The samples included in this study were obtained from various members of the Stem Cell Network in support of The Stem Cell Genomics Project. Full descriptions of the origin and experimental conditions used to derive each sample can be obtained from StemBase; <strong>(</strong><a href="http://www.scgp.ca:8080/StemBase"><strong>http://www.scgp.ca:8080/StemBase</strong></a><strong>).</strong></p>


<h4>Target Labeling and Hybridization</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Total RNA (10 ug or 10–50 ng) was labeled as per manufacturer's suggested methods (Afymetrix, Santa Clara, California, USA). Briefly, following first strand and second strand cDNA synthesis, samples underwent a single round (10 ug starting material) or two rounds (10–50 ng starting material) of linear amplification using a T7 based <em>in vitro</em> transcription (IVT) kit (MegascriptT7, Ambion). During the final round of IVT, biotinylated nucleotides were incorporated into the nascent strand (Enzo Biotech, Farmington, Connecticut, USA) to produce the labeled target cRNA. Ten micrograms of cRNA were fragmented to reduce complexity and hybridized overnight to the MOE 430 GeneChip Set, according to standard protocol. The GeneChips were then washed and stained with Streptavidin R-Phycoerythrin (SAPE). Signal amplification was accomplished by subsequent staining with biotinylated anti-streptavidin, followed by an additional incubation with SAPE. Scanning and absolute analysis was performed in MAS 5.0 to generate the experiment (.exp), raw image (.dat), intensity (.cel) and absolute analysis (.chp) files. All samples were scaled to a target intensity of 1500 during analysis.</p>


<h4>Correlation Analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Normalized expression values for each probeset were obtained from MAS 5.0 (<a href="http://www.affymetrix.com/products/software/specific/mas.affx">http://www.affymetrix.com/products/softw​are/specific/mas.affx</a>) and the mean expression value for each set of biological triplicates was calculated. The data were scaled by normalizing to the trimmed mean for all probesets in the chips (98%). Probesets that had a consensus detection call of present (P) in more than 7% and less than 93% of the samples were included in the analysis. The standard Pearson correlation coefficient (rho) between every probeset which passed the filter, to the <em>Oct4</em> probeset (1417945_at) was computed. A probeset is considered correlated to <em>Oct4</em> if the absolute value of rho is greater than or equal to 0.75. This computation was repeated 10,000 times with random subsets consisting of 65% to 70% of the data. Probesets that were correlated in at least 40% of the trials were retained for further analysis.</p>


<h4>GOStat Analysis</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>GOstat (<a href="http://gostat.wehi.edu.au/">http://gostat.wehi.edu.au/</a>) was used to examine selected sets of probesets for over- and under-representation of GO terms, using MGI (<a href="http://www.informatics.jax.org/mgihome/">http://www.informatics.jax.org/mgihome/</a>) as GO to gene association database, and using false discovery rate correction. This method is sensitive to the GO annotations attached to the genes related to the probes, thus the result might change if another database (e.g. GOA) is used.</p>


<h4>Binding Site Analysis</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The genomic region from 2 kb upstream of the transcriptional start site to 2 kb downstream from the 3-prime end of the transcribed region of the correlated genes was scanned for the presence of neighboring <em>Oct4</em> (ATGCAAAT) and <em>Sox2</em> (AACAAAG) binding sites. Global analysis of the <em>Oct4</em> correlated gene-list was performed in a conservative fashion based upon POU/HMG/DNA ternary complex assembly as determined by crystal structure assessment of <em>Fgf4</em> and <em>Utf1</em> <a href="#pone.0000553-Remenyi1">[59]</a>. First, the two components of the <em>Oct4</em> binding site, namely the POU specific domain (POU<sub>S</sub>) and the POU homeodomain (POU<sub>H</sub>) were forced to be consecutive in the sequence while independently in any direction, and in any of the two strands. A perfect match was required for POU<sub>S</sub> (ATGC), and one mismatch was allowed at any of the four positions of POU<sub>H</sub> (AAAT). Second, we defined the <em>Sox2</em> binding site as either AACAAAG, which corresponds to the predominant pattern, or the observed variations AACAAAT, or AACAATG, in any direction or strand. The maximum distance between <em>Oct4</em> and <em>Sox2</em> binding was constrained to 3 nucleotides.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Manual assessment of binding sites for a subset of the <em>Oct4</em> correlated genes as well as developmentally important regulators <em>Hoxb1</em> and <em>Tcf4</em> was performed in a less restrictive fashion. POU<sub>S</sub> was held invariant while the POU<sub>H</sub> (AAAT) was allowed to vary by one mismatch in any of the four nucleotide positions. Target sequence identification for the two POU domains relative to each other and to the <em>Sox2</em> site were not restricted in order, orientation, or strand. Finally, as has been observed for Oc<em>t</em>4/<em>Sox2</em> cooperative binding on <em>Opn</em> <a href="#pone.0000553-Botquin1">[60]</a>, the distances between the <em>Oct4</em> and <em>Sox2</em> binding sites was relaxed and allowed to span up to 100 nucleotides.</p>


<h4>Chromatin Immunoprecipitation</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Chromatin Immunoprecipitation (ChIP) assays were performed using the Chromatin Immunoprecipitation Assay Kit (Upstate Biotechnology, Lake Placid, NJ, USA). Briefly, 5×10<sup>6</sup> J1 ESCs were cross-linked with 1% formaldehyde for 15 minutes at room temperature. Cells were washed three times in ice-cold PBS with protease inhibitors and lysed in buffer provided to which protease inhibitors were also added. The cells were sonicated to an average size of 1500 bp and 250 ug of input chromatin was used for each assay. Immunoprecipitation was performed overnight at 4°C with <em>Oct4</em> antibody (Santa Cruz Biotechnology, Inc., Santa Cruz, California) and no antibody as a negative control.</p>


<h4>Quantitative Real-time PCR</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>Quantitative PCR was performed using primers that flanked the regions containing putative <em>Oct4</em> and <em>Sox2</em> binding sites with the MX4000 (Stratagene, La Jolla, California, USA) using iQ SYBR Green Supermix (BioRad, Hercules, California.). The following cycling parameters were employed: 96° 10 minutes, followed by 40 cycles of 96°C for 30 seconds, 57°C for 1 minute, and 72°C for 45 seconds. Primer sequences for each amplicon are described in <a href="#pone.0000553.s002">Supplemental Table S2</a>. Each result represents two independent ChIP assays with duplicate QRT-PCR analyses performed on each target gene for each assay. 100% amplification efficiency is assumed based on ΔΔC<sub>t</sub> values of ~3.3 between each point of a 10-fold serial dilution curve performed for a subset of the amplicons. A 2-fold enrichment therefore represents the minimum threshold for confirmation as an <em>Oct4</em> target. Error bars denotes the Standard Error of the Mean. Subsequent to QRT-PCR analysis, each amplicon underwent DNA sequence analysis on and ABI 3730 to confirm identity.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000553"></div><a name="pone.0000553.s001" id="pone.0000553.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000553.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Samples used for Oct4 correlation analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0000553.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0000553.s002" id="pone.0000553.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000553.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Summary of Oct4 Correlated Genes with Probeset ID, gene symbol, gene name, direction and percentage of correlation, chromosomal location, summary GO category used for <a href="#pone-0000553-g002">Figure 2</a> and GO biological process were listed when known.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000553.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.28 MB XLS)</p>
<a name="pone.0000553.s003" id="pone.0000553.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000553.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">GoStat Analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0000553.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.33 MB XLS)</p>
<a name="pone.0000553.s004" id="pone.0000553.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000553.s004">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Oct4/Sox2 putative binding site analysis with Gene symbol, RefSeq or Ensembl ID, putative binding sequence, and location in transcript enumerated</p>
<p class="siDoi">doi:10.1371/journal.pone.0000553.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.11 MB XLS)</p>
<a name="pone.0000553.s005" id="pone.0000553.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000553.s005">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Primer sequences for Oct4 target validation by ChIP/QRT-PCR</p>
<p class="siDoi">doi:10.1371/journal.pone.0000553.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000553.s006" id="pone.0000553.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000553.s006">Table S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Annotation of Oct4 targets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000553.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.09 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The Authors would like to acknowledge the Stem Cell Network for their support of the Stem Cell Genomics Project and the technical staff of the Ontario Genomics Innovation Centre for their expert assistance. Kind thanks to Dave Picketts, Marjorie Brand, and Jeff Dilworth for insightful discussions. MAR is an International Scholar of the Howard Hughes Medical Institute and holds the Canada Research Chair in Molecular Genetics.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MR PC. Performed the experiments: PC. Analyzed the data: CP MA PC. Contributed reagents/materials/analysis tools: CP. Wrote the paper: MR MA PC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000553-Trounson1" id="pone.0000553-Trounson1"></a>Trounson AO (2001) The derivation and potential use of human embryonic stem cells. Reprod Fertil Dev  13: 523–532.  <ul class="find" data-citedArticleID="960998"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+derivation+and+potential+use+of+human+embryonic+stem+cells.&amp;auth=&amp;atitle=The+derivation+and+potential+use+of+human+embryonic+stem+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+derivation+and+potential+use+of+human+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+derivation+and+potential+use+of+human+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000553-Nichols1" id="pone.0000553-Nichols1"></a>Nichols J, Zevnik B, Anastassiadis K, Niwa H, Klewe-Nebenius D, et al.  (1998) Formation of Pluripotent Stem Cells in the Mammalian Embryo Depends on the POU Transcription Factor Oct4. Cell  95: 379–391.  <ul class="find" data-citedArticleID="960968" data-doi="10.1016/s0092-8674(00)81769-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81769-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Formation+of+Pluripotent+Stem+Cells+in+the+Mammalian+Embryo+Depends+on+the+POU+Transcription+Factor+Oct4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Formation+of+Pluripotent+Stem+Cells+in+the+Mammalian+Embryo+Depends+on+the+POU+Transcription+Factor+Oct4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000553-Shimozaki1" id="pone.0000553-Shimozaki1"></a>Shimozaki K, Nakashima K, Niwa H, Taga T (2003) Involvement of Oct3/4 in the enhancement of neuronal differentiation of ES cells in neurogenesis-inducing cultures. Development  130: 2505–2512.  <ul class="find" data-citedArticleID="960982" data-doi="10.1242/dev.00476"><li><a href="http://dx.doi.org/10.1242/dev.00476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Involvement+of+Oct3%2F4+in+the+enhancement+of+neuronal+differentiation+of+ES+cells+in+neurogenesis-inducing+cultures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Involvement+of+Oct3%2F4+in+the+enhancement+of+neuronal+differentiation+of+ES+cells+in+neurogenesis-inducing+cultures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000553-Reim1" id="pone.0000553-Reim1"></a>Reim G, Mizoguchi T, Stainier DY, Kikuchi Y, Brand M (2004) The POU Domain Protein Spg (Pou2/Oct4) Is Essential for Endoderm Formation in Cooperation with the HMG Domain Protein Casanova. Developmental Cell  6: 91–101.  <ul class="find" data-citedArticleID="960976" data-doi="10.1016/s1534-5807(03)00396-4"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00396-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+POU+Domain+Protein+Spg+%28Pou2%2FOct4%29+Is+Essential+for+Endoderm+Formation+in+Cooperation+with+the+HMG+Domain+Protein+Casanova." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+POU+Domain+Protein+Spg+%28Pou2%2FOct4%29+Is+Essential+for+Endoderm+Formation+in+Cooperation+with+the+HMG+Domain+Protein+Casanova.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000553-Niwa1" id="pone.0000553-Niwa1"></a>Niwa H, Miyazake J, Smith A (2000) Quantitative expression of Oct3/4 defines differentiation, dedifferentiation, or self-renewal of ES Cells. Nature Genetics  24: 372–376.  <ul class="find" data-citedArticleID="960970"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantitative+expression+of+Oct3%2F4+defines+differentiation%2C+dedifferentiation%2C+or+self-renewal+of+ES+Cells.&amp;auth=&amp;atitle=Quantitative+expression+of+Oct3%2F4+defines+differentiation%2C+dedifferentiation%2C+or+self-renewal+of+ES+Cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+expression+of+Oct3%2F4+defines+differentiation%2C+dedifferentiation%2C+or+self-renewal+of+ES+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+expression+of+Oct3%2F4+defines+differentiation%2C+dedifferentiation%2C+or+self-renewal+of+ES+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000553-Gidekel1" id="pone.0000553-Gidekel1"></a>Gidekel S, Pizov G, Bergman Y, Pikarsky E (2003) Oct-3/4 is a dose-dependent fate determinant. Cancer Cell  4: 361–371.  <ul class="find" data-citedArticleID="960928" data-doi="10.1016/s1535-6108(03)00270-8"><li><a href="http://dx.doi.org/10.1016/s1535-6108(03)00270-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oct-3%2F4+is+a+dose-dependent+fate+determinant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oct-3%2F4+is+a+dose-dependent+fate+determinant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000553-Hochediinger1" id="pone.0000553-Hochediinger1"></a>Hochediinger K, Yamada Y, Beard C, Jaenisch R (2005) Ectopic expression of Oct4 blocks progenitor-cell differentiation and causes dysplasia in epithelial tissue. Cell  121: 465–477.  <ul class="find" data-citedArticleID="960932" data-doi="10.1016/j.cell.2005.02.018"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.02.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ectopic+expression+of+Oct4+blocks+progenitor-cell+differentiation+and+causes+dysplasia+in+epithelial+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ectopic+expression+of+Oct4+blocks+progenitor-cell+differentiation+and+causes+dysplasia+in+epithelial+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000553-Monk1" id="pone.0000553-Monk1"></a>Monk M, Holding , Cathy  (2001) Human Embryonic Genes Re-expressed in Cancer Cells. Oncogene  20: 8085–8091.  <ul class="find" data-citedArticleID="960964" data-doi="10.1038/sj.onc.1205088"><li><a href="http://dx.doi.org/10.1038/sj.onc.1205088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Embryonic+Genes+Re-expressed+in+Cancer+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Embryonic+Genes+Re-expressed+in+Cancer+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000553-Jin1" id="pone.0000553-Jin1"></a>Jin T, Branch , Donald R, Zhang Xiaoyun, Qi Shangle, Youngson , Bruce , Goss , Paul E (1999) Examination of POU Homeobox Gene Expression in Human Breast Cancer Cells. International Journal of Cancer  81: 104–112.  <ul class="find" data-citedArticleID="960940" data-doi="10.1002/(sici)1097-0215(19990331)81:1<104::aid-ijc18&gt;3.0.co;2-q"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0215(19990331)81:1<104::aid-ijc18&gt;3.0.co;2-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Examination+of+POU+Homeobox+Gene+Expression+in+Human+Breast+Cancer+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Examination+of+POU+Homeobox+Gene+Expression+in+Human+Breast+Cancer+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000553-Tai1" id="pone.0000553-Tai1"></a>Tai M-H, Chang C-C, Olson LK, Trosko JE (2005) Oct4 expression in adult human stem cells: evidence in support of the stem cell theory of carcinogenesis. Carcinogenesis  26: 495–502.  <ul class="find" data-citedArticleID="960994" data-doi="10.1093/carcin/bgh321"><li><a href="http://dx.doi.org/10.1093/carcin/bgh321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oct4+expression+in+adult+human+stem+cells%3A+evidence+in+support+of+the+stem+cell+theory+of+carcinogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oct4+expression+in+adult+human+stem+cells%3A+evidence+in+support+of+the+stem+cell+theory+of+carcinogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000553-Boyer1" id="pone.0000553-Boyer1"></a>Boyer LA, Lee TI, Cole MF, Johnstone SE, Levine SS, et al.  (2005) Core Transcriptional Regulatory Circuitry in Human Embryonic Stem Cells. Cell  122: 1–10.  <ul class="find" data-citedArticleID="960908" data-doi="10.1016/j.cell.2005.08.020"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.08.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Core+Transcriptional+Regulatory+Circuitry+in+Human+Embryonic+Stem+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Core+Transcriptional+Regulatory+Circuitry+in+Human+Embryonic+Stem+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000553-Loh1" id="pone.0000553-Loh1"></a>Loh Y-H, Wu Q, Chew J-L, Vega VB, Zhang W, et al.  (2006) The Oct4 and Nanog transcription network regulates pluripotency in mouse embryonic stem cells. 38: 431–440.  <ul class="find" data-citedArticleID="960958" data-doi="10.1038/ng1760"><li><a href="http://dx.doi.org/10.1038/ng1760" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Oct4+and+Nanog+transcription+network+regulates+pluripotency+in+mouse+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Oct4+and+Nanog+transcription+network+regulates+pluripotency+in+mouse+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000553-BatlleE1" id="pone.0000553-BatlleE1"></a>Batlle E HJ, Beghtel H, van den Born MM, Sancho E, Huls G, Meeldijk J, Robertson J, van de Wetering M, Pawson T, Clevers H (2002) B-catenin and TCF mediate stem cell positioning in the intestinal epitheleum by controlling the expression of EphB/ephrinB. Cell  111: 251–263.  <ul class="find" data-citedArticleID="960898" data-doi="10.1016/s0092-8674(02)01015-2"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)01015-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=B-catenin+and+TCF+mediate+stem+cell+positioning+in+the+intestinal+epitheleum+by+controlling+the+expression+of+EphB%2FephrinB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22B-catenin+and+TCF+mediate+stem+cell+positioning+in+the+intestinal+epitheleum+by+controlling+the+expression+of+EphB%2FephrinB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000553-Boiani1" id="pone.0000553-Boiani1"></a>Boiani M, Scholer H (2005) Regulatory networks in embryo-derived pluripotent stem cells. Nature Reviews Molecular and Cellular Biology  6: 872–884.  <ul class="find" data-citedArticleID="960904" data-doi="10.1038/nrm1744"><li><a href="http://dx.doi.org/10.1038/nrm1744" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+networks+in+embryo-derived+pluripotent+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+networks+in+embryo-derived+pluripotent+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000553-Chickarmane1" id="pone.0000553-Chickarmane1"></a>Chickarmane V, Troein C, Nuber UA, Sauro HM, Peterson C (2006) Transcriptional Dynamics of the Embryonic Stem Cell Switch. PLOS Computational Bioloby  2: 1080–1092.  <ul class="find" data-citedArticleID="960918" data-doi="10.1371/journal.pcbi.0020123.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020123.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+Dynamics+of+the+Embryonic+Stem+Cell+Switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+Dynamics+of+the+Embryonic+Stem+Cell+Switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000553-Burdon1" id="pone.0000553-Burdon1"></a>Burdon T, Smith A, Savatier P (2002) Signalling, cell cycle, and pluripotency in embryonic stem cells. Trends in Cell Biology  12: 432–438.  <ul class="find" data-citedArticleID="960912" data-doi="10.1016/s0962-8924(02)02352-8"><li><a href="http://dx.doi.org/10.1016/s0962-8924(02)02352-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signalling%2C+cell+cycle%2C+and+pluripotency+in+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signalling%2C+cell+cycle%2C+and+pluripotency+in+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000553-Zhang1" id="pone.0000553-Zhang1"></a>Zhang W, Morris QD, Chang R, Shai O, Bakowski MA, et al.  (2004) The functional landscape ot mouse gene expression. Journal of Biology  3:   <ul class="find" data-citedArticleID="961006"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+functional+landscape+ot+mouse+gene+expression.&amp;auth=&amp;atitle=The+functional+landscape+ot+mouse+gene+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+functional+landscape+ot+mouse+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+functional+landscape+ot+mouse+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000553-PerezIratxeta1" id="pone.0000553-PerezIratxeta1"></a>Perez-Iratxeta C, Palidwor G, Porter CJ, Sanche NA, Huska MR, et al.  (2005) Study of stem cell function using microarray experiments. FEBS Lett  579: 1795–1801.  <ul class="find" data-citedArticleID="960972" data-doi="10.1016/j.febslet.2005.02.020"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.02.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+of+stem+cell+function+using+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+of+stem+cell+function+using+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000553-Ivanova1" id="pone.0000553-Ivanova1"></a>Ivanova N, Dimos J, Schaniel C, Hackney J, Moore K, et al.  (2002) A stem cell molecular signature. Science  298: 601–604.  <ul class="find" data-citedArticleID="960936" data-doi="10.1126/science.1073823"><li><a href="http://dx.doi.org/10.1126/science.1073823" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+stem+cell+molecular+signature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+stem+cell+molecular+signature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000553-RamalhoSantos1" id="pone.0000553-RamalhoSantos1"></a>Ramalho-Santos M, Yoon S, Matsuzake Y, Mulligan RC, Melton DA (2002) “Stemness”: transcriptional profiling of embryonic and adult stem cells. Science  298: 597–600.  <ul class="find" data-citedArticleID="960974" data-doi="10.1126/science.1072530"><li><a href="http://dx.doi.org/10.1126/science.1072530" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%26%23x201c%3BStemness%26%23x201d%3B%3A+transcriptional+profiling+of+embryonic+and+adult+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%26%23x201c%3BStemness%26%23x201d%3B%3A+transcriptional+profiling+of+embryonic+and+adult+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000553-Mikkers1" id="pone.0000553-Mikkers1"></a>Mikkers H, Frisen J (2005) Deconstructing stemnes. European Molecular Biology Organization  24: 2715–2719.  <ul class="find" data-citedArticleID="960962" data-doi="10.1038/sj.emboj.7600749"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600749" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deconstructing+stemnes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deconstructing+stemnes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000553-Arias1" id="pone.0000553-Arias1"></a>Arias AM, Hayward P (2006) Filtering transcriptional noise during development: concepts and mechanisms. Nature  7: 34–44.  <ul class="find" data-citedArticleID="960890" data-doi="10.1038/nrg1750"><li><a href="http://dx.doi.org/10.1038/nrg1750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filtering+transcriptional+noise+during+development%3A+concepts+and+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Filtering+transcriptional+noise+during+development%3A+concepts+and+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000553-Lee1" id="pone.0000553-Lee1"></a>Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, et al.  (2002) Transcriptional regulatory networks in Saccaromyces cerevisiae. Science  298: 799–894.  <ul class="find" data-citedArticleID="960950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptional+regulatory+networks+in+Saccaromyces+cerevisiae.&amp;auth=&amp;atitle=Transcriptional+regulatory+networks+in+Saccaromyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+networks+in+Saccaromyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+networks+in+Saccaromyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000553-Beissbarth1" id="pone.0000553-Beissbarth1"></a>Beissbarth T, Speed TP (2004) GOstat: find statistically overrepresented Gene Ontologies within a group of genes. Bioinformatics  20: 1464–1465.  <ul class="find" data-citedArticleID="960900" data-doi="10.1093/bioinformatics/bth088"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GOstat%3A+find+statistically+overrepresented+Gene+Ontologies+within+a+group+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GOstat%3A+find+statistically+overrepresented+Gene+Ontologies+within+a+group+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000553-Babaie1" id="pone.0000553-Babaie1"></a>Babaie Y, Herwig R, Greber B, Brink TC, Wruck W, et al.  (2007) Analysis of Oct4-dependent transcriptional networks regulating self-renewal and pluripotency in human embryonic stem cells. Stem Cells  25: 500–510.  <ul class="find" data-citedArticleID="960892" data-doi="10.1634/stemcells.2006-0426"><li><a href="http://dx.doi.org/10.1634/stemcells.2006-0426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Oct4-dependent+transcriptional+networks+regulating+self-renewal+and+pluripotency+in+human+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Oct4-dependent+transcriptional+networks+regulating+self-renewal+and+pluripotency+in+human+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000553-Ivanova2" id="pone.0000553-Ivanova2"></a>Ivanova N, Dobrin R, Lu R, Kotenko I, Levorse J, et al.  (2006) Dissecting self-renewal in stem cells with RNA interference. Nature  442: 533–538.  <ul class="find" data-citedArticleID="960938" data-doi="10.1038/nature04915"><li><a href="http://dx.doi.org/10.1038/nature04915" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissecting+self-renewal+in+stem+cells+with+RNA+interference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissecting+self-renewal+in+stem+cells+with+RNA+interference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000553-Matoba1" id="pone.0000553-Matoba1"></a>Matoba R, Niwa H, Masui S, Ohtsuka S, Carter MG, et al.  (2006) DissectingOct3/4-regulated gene networks in embryonic stem cells by expression profiling. PlOS One  1: e26.  <ul class="find" data-citedArticleID="960960" data-doi="10.1371/journal.pone.0000026"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DissectingOct3%2F4-regulated+gene+networks+in+embryonic+stem+cells+by+expression+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DissectingOct3%2F4-regulated+gene+networks+in+embryonic+stem+cells+by+expression+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000553-Cairns1" id="pone.0000553-Cairns1"></a>Cairns BR (2005) Chromatin remodeling complexes: strength in diversity, precision through specialization. Current Opinion in Genetics&amp;Development  15: 185–190.  <ul class="find" data-citedArticleID="960914" data-doi="10.1016/j.gde.2005.01.003"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.01.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+remodeling+complexes%3A+strength+in+diversity%2C+precision+through+specialization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+remodeling+complexes%3A+strength+in+diversity%2C+precision+through+specialization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000553-Klose1" id="pone.0000553-Klose1"></a>Klose RJ, Kallin EM, Zhang Y (2006) JmjC-domain-containing proteins and histone demethylation. Nat Rev Genet  7: 715–727.  <ul class="find" data-citedArticleID="960944" data-doi="10.1038/nrg1945"><li><a href="http://dx.doi.org/10.1038/nrg1945" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=JmjC-domain-containing+proteins+and+histone+demethylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22JmjC-domain-containing+proteins+and+histone+demethylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000553-Huang1" id="pone.0000553-Huang1"></a>Huang Y, Meyers SJ, Dingledine R (1999) Transcriptional repression by REST: recruitment of Sin3a and histone deacetylase to neuronal genes. Nature Neuroscience  2: 867–872.  <ul class="find" data-citedArticleID="960934"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptional+repression+by+REST%3A+recruitment+of+Sin3a+and+histone+deacetylase+to+neuronal+genes.&amp;auth=&amp;atitle=Transcriptional+repression+by+REST%3A+recruitment+of+Sin3a+and+histone+deacetylase+to+neuronal+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+repression+by+REST%3A+recruitment+of+Sin3a+and+histone+deacetylase+to+neuronal+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+repression+by+REST%3A+recruitment+of+Sin3a+and+histone+deacetylase+to+neuronal+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000553-Ballas1" id="pone.0000553-Ballas1"></a>Ballas N, Grunseich C, Lu DD, Speh JC, Mandel G (2005) REST and Its Corepressors Mediate Plasticity of Neuronal Gene Chromatin throughout Neurogenesis. Cell  121: 645–657.  <ul class="find" data-citedArticleID="960894" data-doi="10.1016/j.cell.2005.03.013"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.03.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=REST+and+Its+Corepressors+Mediate+Plasticity+of+Neuronal+Gene+Chromatin+throughout+Neurogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22REST+and+Its+Corepressors+Mediate+Plasticity+of+Neuronal+Gene+Chromatin+throughout+Neurogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000553-Westbrook1" id="pone.0000553-Westbrook1"></a>Westbrook TF, Martin ES, Schlabach MR, Leng Y, Liang AC, et al.  (2005) A Genetic Screen for Candidate Tumor Suppressors Identifies REST. Cell  121: 837–848.  <ul class="find" data-citedArticleID="961004" data-doi="10.1016/j.cell.2005.03.033"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.03.033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Genetic+Screen+for+Candidate+Tumor+Suppressors+Identifies+REST." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Genetic+Screen+for+Candidate+Tumor+Suppressors+Identifies+REST.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000553-Lawinger1" id="pone.0000553-Lawinger1"></a>Lawinger P, Venugopal R, Guo ZS, Immaneni A, Sengupta D, et al.  (2000) The neuronal repressor REST/NRSF is an essential regulator in medulloblastoma cells. Nature Medicine  6: 826–831.  <ul class="find" data-citedArticleID="960948" data-doi="10.1038/77565"><li><a href="http://dx.doi.org/10.1038/77565" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neuronal+repressor+REST%2FNRSF+is+an+essential+regulator+in+medulloblastoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neuronal+repressor+REST%2FNRSF+is+an+essential+regulator+in+medulloblastoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000553-Sun1" id="pone.0000553-Sun1"></a>Sun Y-M, Greenway DJ, Johnson R, Street M, Belyaev ND, et al.  (2005) Distinct Profiles of REST Interactions with Its Target Genes at Different Stages of Neuronal Development. Mol Biol Cell  16: 5630–5638.  <ul class="find" data-citedArticleID="960992" data-doi="10.1091/mbc.e05-07-0687"><li><a href="http://dx.doi.org/10.1091/mbc.e05-07-0687" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+Profiles+of+REST+Interactions+with+Its+Target+Genes+at+Different+Stages+of+Neuronal+Development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+Profiles+of+REST+Interactions+with+Its+Target+Genes+at+Different+Stages+of+Neuronal+Development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000553-ValkLingbeek1" id="pone.0000553-ValkLingbeek1"></a>Valk-Lingbeek ME, Bruggeman SWM, van Lohuizen M (2004) Stem Cells and Cancer: the Polycomb Connection. Cell  118: 409–418.  <ul class="find" data-citedArticleID="961000" data-doi="10.1016/j.cell.2004.08.005"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.08.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stem+Cells+and+Cancer%3A+the+Polycomb+Connection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stem+Cells+and+Cancer%3A+the+Polycomb+Connection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000553-Lessard1" id="pone.0000553-Lessard1"></a>Lessard J, Sauvageau G (2003) Bmi-1 determines the proliferative capacity of normal and leukaemic stem cells. 423: 255–260.  <ul class="find" data-citedArticleID="960954" data-doi="10.1038/nature01572"><li><a href="http://dx.doi.org/10.1038/nature01572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bmi-1+determines+the+proliferative+capacity+of+normal+and+leukaemic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bmi-1+determines+the+proliferative+capacity+of+normal+and+leukaemic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000553-Lee2" id="pone.0000553-Lee2"></a>Lee TI, Jenner RG, Boyer LA, Guenther MG, Levine SS, et al.  (2006) Control of Developmental Regulators by Polycomb in Human Embryonic Stem Cells. Cell  125: 301–313.  <ul class="find" data-citedArticleID="960952" data-doi="10.1016/j.cell.2006.02.043"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.02.043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+Developmental+Regulators+by+Polycomb+in+Human+Embryonic+Stem+Cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+Developmental+Regulators+by+Polycomb+in+Human+Embryonic+Stem+Cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000553-Squazzo1" id="pone.0000553-Squazzo1"></a>Squazzo SL, O'Geen H, Komashko VM, Krig SR, Jin VX, et al.  (2006) Suz12 binds to silenced regions of the genome in a cell-type-specific manner/gr.5306606. Genome Res  16: 890–900.  <ul class="find" data-citedArticleID="960986" data-doi="10.1101/gr.5306606"><li><a href="http://dx.doi.org/10.1101/gr.5306606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suz12+binds+to+silenced+regions+of+the+genome+in+a+cell-type-specific+manner%2Fgr.5306606." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suz12+binds+to+silenced+regions+of+the+genome+in+a+cell-type-specific+manner%2Fgr.5306606.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000553-Squazzo2" id="pone.0000553-Squazzo2"></a>Squazzo SL, O'Geen H, Komashko VM, Krig SR, Jin VX, et al.  (2006) Suz12 binds to silenced regions of the genome in a cell-type-specific manner. Genome Research  16: 890–900.  <ul class="find" data-citedArticleID="960988" data-doi="10.1101/gr.5306606"><li><a href="http://dx.doi.org/10.1101/gr.5306606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suz12+binds+to+silenced+regions+of+the+genome+in+a+cell-type-specific+manner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suz12+binds+to+silenced+regions+of+the+genome+in+a+cell-type-specific+manner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000553-Francis1" id="pone.0000553-Francis1"></a>Francis NJ, Kingston RE, Woodcock CL (2004) Chromatin compaction by a polycomb group protein complex. Science  306: 1574–1577.  <ul class="find" data-citedArticleID="960924" data-doi="10.1126/science.1100576"><li><a href="http://dx.doi.org/10.1126/science.1100576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+compaction+by+a+polycomb+group+protein+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+compaction+by+a+polycomb+group+protein+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000553-Kuzmichev1" id="pone.0000553-Kuzmichev1"></a>Kuzmichev A, Margueron R, Vaquero A, Preissner TS, Scher M, et al.  (2004) Composition and histone substrates of polycomb repressive group complexes change during cellular differentiation. Proceedings of the National Academy of Sciences  102: 1859–1864.  <ul class="find" data-citedArticleID="960946" data-doi="10.1073/pnas.0409875102"><li><a href="http://dx.doi.org/10.1073/pnas.0409875102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Composition+and+histone+substrates+of+polycomb+repressive+group+complexes+change+during+cellular+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Composition+and+histone+substrates+of+polycomb+repressive+group+complexes+change+during+cellular+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000553-Kim1" id="pone.0000553-Kim1"></a>Kim JY, Sawada A, Tokimasa S, Endo H, Ozono K, et al.  (2004) Defective long-term repopulating ability in hematopoietic stem cells lacking the Polycomb-group gene rae28. European Journal of Haematology  73: 75–84.  <ul class="find" data-citedArticleID="960942" data-doi="10.1111/j.1600-0609.2004.00268.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0609.2004.00268.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defective+long-term+repopulating+ability+in+hematopoietic+stem+cells+lacking+the+Polycomb-group+gene+rae28." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defective+long-term+repopulating+ability+in+hematopoietic+stem+cells+lacking+the+Polycomb-group+gene+rae28.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000553-Benevolenskaya1" id="pone.0000553-Benevolenskaya1"></a>Benevolenskaya EV, Murray HL, Branton P, Young RA, Kaelin J, William G (2005) Binding of pRB to the PHD Protein RBP2 Promotes Cellular Differentiation. Molecular Cell  18: 623–635.  <ul class="find" data-citedArticleID="960902" data-doi="10.1016/j.molcel.2005.05.012"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.05.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+of+pRB+to+the+PHD+Protein+RBP2+Promotes+Cellular+Differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+of+pRB+to+the+PHD+Protein+RBP2+Promotes+Cellular+Differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000553-Sissons1" id="pone.0000553-Sissons1"></a>Sissons J, Alsam S, Jayasekera S, Kim KS, Stins M, et al.  (2004) Acanthamoeba induces cell-cycle arrest in host cells. J Med Microbiol  53: 711–717.  <ul class="find" data-citedArticleID="960984" data-doi="10.1099/jmm.0.45604-0"><li><a href="http://dx.doi.org/10.1099/jmm.0.45604-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acanthamoeba+induces+cell-cycle+arrest+in+host+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acanthamoeba+induces+cell-cycle+arrest+in+host+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000553-Tamrakar1" id="pone.0000553-Tamrakar1"></a>Tamrakar S, Rubin E, Ludlow JW (2000) Role of pRB Dephosphorylation In Cell Cycle Regulation. Fronteirs in Bioscience  5: d121–137.  <ul class="find" data-citedArticleID="960996" data-doi="10.2741/tamrakar"><li><a href="http://dx.doi.org/10.2741/tamrakar" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+pRB+Dephosphorylation+In+Cell+Cycle+Regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+pRB+Dephosphorylation+In+Cell+Cycle+Regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000553-VanEynde1" id="pone.0000553-VanEynde1"></a>Van Eynde A, Nuytten M, Dewerchin M, Schoonjans L, Keppens S, et al.  (2004) The Nuclear Scaffold Protein NIPP1 Is Essential for Early Embryonic Development and Cell Proliferation. Mol Cell Biol  24: 5863–5874.  <ul class="find" data-citedArticleID="961002" data-doi="10.1128/mcb.24.13.5863-5874.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.13.5863-5874.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Nuclear+Scaffold+Protein+NIPP1+Is+Essential+for+Early+Embryonic+Development+and+Cell+Proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Nuclear+Scaffold+Protein+NIPP1+Is+Essential+for+Early+Embryonic+Development+and+Cell+Proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000553-Broceno1" id="pone.0000553-Broceno1"></a>Broceno C, Wilkie S, Mittnacht S (2002) RB activation defect in tumor cell lines. Proceedings of the National Academy of Sciences  99: 14200–14205.  <ul class="find" data-citedArticleID="960910" data-doi="10.1073/pnas.212519499"><li><a href="http://dx.doi.org/10.1073/pnas.212519499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RB+activation+defect+in+tumor+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RB+activation+defect+in+tumor+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000553-Bartek1" id="pone.0000553-Bartek1"></a>Bartek J, Lukas J (2001) Pathways governing G1/S transition and their response to DNA damage. Federation of Biochemical Societies  490: 117–122.  <ul class="find" data-citedArticleID="960896" data-doi="10.1016/s0014-5793(01)02114-7"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)02114-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathways+governing+G1%2FS+transition+and+their+response+to+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathways+governing+G1%2FS+transition+and+their+response+to+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000553-Stucki1" id="pone.0000553-Stucki1"></a>Stucki M, Jackson SP (2004) Tudor domains track down DNA breaks. Nature Cell Biology  6: 1150–1152.  <ul class="find" data-citedArticleID="960990" data-doi="10.1038/ncb1204-1150"><li><a href="http://dx.doi.org/10.1038/ncb1204-1150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tudor+domains+track+down+DNA+breaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tudor+domains+track+down+DNA+breaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000553-Fernando1" id="pone.0000553-Fernando1"></a>Fernando P, Kelly JF, Balazsi K, Slack RS, Megeney LA (2002) Caspase 3 activity is required for skeletal muscle differentiation. PNAS  99: 11025–11030.  <ul class="find" data-citedArticleID="960922" data-doi="10.1073/pnas.162172899"><li><a href="http://dx.doi.org/10.1073/pnas.162172899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caspase+3+activity+is+required+for+skeletal+muscle+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caspase+3+activity+is+required+for+skeletal+muscle+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000553-Hemberger1" id="pone.0000553-Hemberger1"></a>Hemberger M, Nozaki T, Winterhager E, Yamamoto H, Nakagama H, et al.  (2003) Parp1-deficiency induces differentiation of ES cells into trophoblast derivatives. Developmental Biology  257: 371–381.  <ul class="find" data-citedArticleID="960930" data-doi="10.1016/s0012-1606(03)00097-6"><li><a href="http://dx.doi.org/10.1016/s0012-1606(03)00097-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parp1-deficiency+induces+differentiation+of+ES+cells+into+trophoblast+derivatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parp1-deficiency+induces+differentiation+of+ES+cells+into+trophoblast+derivatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000553-Lin1" id="pone.0000553-Lin1"></a>Lin T, Chao C, Saito Si, Mazur SJ, Murphy ME, et al.  (2005) p53 induces differentiation of mouse embryonic stem cells by suppressing Nanog expression. Nature Cell Biology  7: 165–171.  <ul class="find" data-citedArticleID="960956" data-doi="10.1038/ncb1211"><li><a href="http://dx.doi.org/10.1038/ncb1211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p53+induces+differentiation+of+mouse+embryonic+stem+cells+by+suppressing+Nanog+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p53+induces+differentiation+of+mouse+embryonic+stem+cells+by+suppressing+Nanog+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000553-Furuta1" id="pone.0000553-Furuta1"></a>Furuta S, Jiang X, Gu B, Cheng E, Chen P-L, et al.  (2005) Depletion of BRCA1 impairs differentiation but enhances proliferation of mammary epithelial cells. PNAS  102: 9176–9181.  <ul class="find" data-citedArticleID="960926" data-doi="10.1073/pnas.0503793102"><li><a href="http://dx.doi.org/10.1073/pnas.0503793102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Depletion+of+BRCA1+impairs+differentiation+but+enhances+proliferation+of+mammary+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Depletion+of+BRCA1+impairs+differentiation+but+enhances+proliferation+of+mammary+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000553-Morrison1" id="pone.0000553-Morrison1"></a>Morrison AJ, Highland J, Krogan NJ, Arbel-Eden A, Greenblatt JF, et al.  (2004) INO80 and [gamma]-H2AX Interaction Links ATP-Dependent Chromatin Remodeling to DNA Damage Repair. Cell  119: 767–775.  <ul class="find" data-citedArticleID="960966" data-doi="10.1016/j.cell.2004.11.037"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=INO80+and+%5Bgamma%5D-H2AX+Interaction+Links+ATP-Dependent+Chromatin+Remodeling+to+DNA+Damage+Repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22INO80+and+%5Bgamma%5D-H2AX+Interaction+Links+ATP-Dependent+Chromatin+Remodeling+to+DNA+Damage+Repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000553-FernandezCapetillo1" id="pone.0000553-FernandezCapetillo1"></a>Fernandez-Capetillo O, Nussenzweig A (2004) Linking histone deacetylation with the repair of DNA breaks. PNAS  101: 1427–1428.  <ul class="find" data-citedArticleID="960920" data-doi="10.1073/pnas.0307342101"><li><a href="http://dx.doi.org/10.1073/pnas.0307342101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linking+histone+deacetylation+with+the+repair+of+DNA+breaks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linking+histone+deacetylation+with+the+repair+of+DNA+breaks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000553-Zimber1" id="pone.0000553-Zimber1"></a>Zimber A, Nguyen Q-D, Gespach C (2004) Nuclear bodies and compartiments: functional roles and cellular signalling in health and disease. Cellular Signalling  16: 1085–1104.  <ul class="find" data-citedArticleID="961008" data-doi="10.1016/j.cellsig.2004.03.020"><li><a href="http://dx.doi.org/10.1016/j.cellsig.2004.03.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+bodies+and+compartiments%3A+functional+roles+and+cellular+signalling+in+health+and+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+bodies+and+compartiments%3A+functional+roles+and+cellular+signalling+in+health+and+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000553-Chambeyron1" id="pone.0000553-Chambeyron1"></a>Chambeyron S, Bickmore WA (2004) Chromatin decondensation and nuclear reorganization of the HoxB locus upon induction of transcription. Genes Dev  18: 1119–1130.  <ul class="find" data-citedArticleID="960916" data-doi="10.1101/gad.292104"><li><a href="http://dx.doi.org/10.1101/gad.292104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+decondensation+and+nuclear+reorganization+of+the+HoxB+locus+upon+induction+of+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+decondensation+and+nuclear+reorganization+of+the+HoxB+locus+upon+induction+of+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000553-Sell1" id="pone.0000553-Sell1"></a>Sell S, Pierce G (1994) Maturation arrest of stem cell differentiation is a common pathway for the cellular origin of teratocarcinomas and epithelial cancers. Laboratory Investigations  70: 6–22.  <ul class="find" data-citedArticleID="960980"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maturation+arrest+of+stem+cell+differentiation+is+a+common+pathway+for+the+cellular+origin+of+teratocarcinomas+and+epithelial+cancers.&amp;auth=&amp;atitle=Maturation+arrest+of+stem+cell+differentiation+is+a+common+pathway+for+the+cellular+origin+of+teratocarcinomas+and+epithelial+cancers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maturation+arrest+of+stem+cell+differentiation+is+a+common+pathway+for+the+cellular+origin+of+teratocarcinomas+and+epithelial+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maturation+arrest+of+stem+cell+differentiation+is+a+common+pathway+for+the+cellular+origin+of+teratocarcinomas+and+epithelial+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000553-Remenyi1" id="pone.0000553-Remenyi1"></a>Remenyi A, Lins K, Nissen LJ, Reinbold R, Scholer H, et al.  (2003) Crystal Structure of a POU/HMG/DNA ternary complex suggests differential assembly of Oct4 and Sox2 on two enhancers. Genes and Development  17: 2048–2059.  <ul class="find" data-citedArticleID="960978" data-doi="10.1101/gad.269303"><li><a href="http://dx.doi.org/10.1101/gad.269303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+Structure+of+a+POU%2FHMG%2FDNA+ternary+complex+suggests+differential+assembly+of+Oct4+and+Sox2+on+two+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+Structure+of+a+POU%2FHMG%2FDNA+ternary+complex+suggests+differential+assembly+of+Oct4+and+Sox2+on+two+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000553-Botquin1" id="pone.0000553-Botquin1"></a>Botquin V, Hess H, Fuhrnamm G, Anastassiadis C, Gross MK, et al.  (1998) Now POU dimer configuration mediates antagonistic control of an osteopontin preimplantation enhancer by Oct-4 and Sox-2. Genes and Development  12: 2073–2090.  <ul class="find" data-citedArticleID="960906" data-doi="10.1101/gad.12.13.2073"><li><a href="http://dx.doi.org/10.1101/gad.12.13.2073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Now+POU+dimer+configuration+mediates+antagonistic+control+of+an+osteopontin+preimplantation+enhancer+by+Oct-4+and+Sox-2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Now+POU+dimer+configuration+mediates+antagonistic+control+of+an+osteopontin+preimplantation+enhancer+by+Oct-4+and+Sox-2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.XML" value="131110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.PDF" value="678991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g001.PNG_L" value="161240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g001.PNG_M" value="143365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g001.PNG_S" value="4868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g001.TIF" value="274794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g001.PNG_I" value="60967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g002.PNG_L" value="187260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g002.PNG_M" value="111696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g002.PNG_S" value="4804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g002.TIF" value="248164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g002.PNG_I" value="51655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t001.PNG_L" value="146853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t001.PNG_M" value="50829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t001.PNG_S" value="2752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t001.TIF" value="740656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t001.PNG_I" value="21859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g003.PNG_L" value="114826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g003.PNG_M" value="86270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g003.PNG_S" value="3405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g003.TIF" value="257380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g003.PNG_I" value="36271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t002.PNG_L" value="204812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t002.PNG_M" value="76575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t002.PNG_S" value="3272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t002.TIF" value="1265392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.t002.PNG_I" value="33167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g004.PNG_L" value="164710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g004.PNG_M" value="158080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g004.PNG_S" value="7281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g004.TIF" value="215076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.g004.PNG_I" value="95751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.s001.DOC" value="75776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.s002.XLS" value="278016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.s003.XLS" value="333312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.s004.XLS" value="108544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.s005.DOC" value="58880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000553.s006.DOC" value="92160"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000553&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000553" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000553&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000553&volume=&issue=&title=Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function&author_name=Pearl%20A.%20Campbell%2C%20Carolina%20Perez-Iratxeta%2C%20Miguel%20A.%20Andrade-Navarro%2C%20Michael%20A.%20Rudnicki&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000553" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553&amp;t=Oct4%20Targets%20Regulatory%20Nodes%20to%20Modulate%20Stem%20Cell%20Function" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553&title=Oct4%20Targets%20Regulatory%20Nodes%20to%20Modulate%20Stem%20Cell%20Function&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553&amp;title=Oct4%20Targets%20Regulatory%20Nodes%20to%20Modulate%20Stem%20Cell%20Function" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000553&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Oct4 Targets Regulatory Nodes to Modulate Stem Cell Function';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000553';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Oct4%20Targets%20Regulatory%20Nodes%20to%20Modulate%20Stem%20Cell%20Function http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000553" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000553" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Apoptosis%22" title="Search for articles in the subject area:'Apoptosis'"><div class="flagText">Apoptosis</div></a>
              <div data-categoryid="17189" data-articleid="23944"
                   data-categoryname="Apoptosis"
                   class="flagImage" title="Flag 'Apoptosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+differentiation%22" title="Search for articles in the subject area:'Cell differentiation'"><div class="flagText">Cell differentiation</div></a>
              <div data-categoryid="17923" data-articleid="23944"
                   data-categoryname="Cell differentiation"
                   class="flagImage" title="Flag 'Cell differentiation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="23944"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="23944"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23944"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="23944"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="23944"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Stem+cells%22" title="Search for articles in the subject area:'Stem cells'"><div class="flagText">Stem cells</div></a>
              <div data-categoryid="17585" data-articleid="23944"
                   data-categoryname="Stem cells"
                   class="flagImage" title="Flag 'Stem cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7803'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5735'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2706&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
