

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001324"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001324" />

    <meta name="citation_title" content="Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies"/>
    <meta itemprop="name" content="Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies"/>

      <meta name="citation_author" content="Pascale Quignon"/>
            <meta name="citation_author_institution" content="Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Laetitia Herbin"/>
            <meta name="citation_author_institution" content="CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France"/>
      <meta name="citation_author" content="Edouard Cadieu"/>
            <meta name="citation_author_institution" content="Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Ewen F. Kirkness"/>
            <meta name="citation_author_institution" content="The J Craig Venter Institute, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="Benoit Hédan"/>
            <meta name="citation_author_institution" content="CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France"/>
      <meta name="citation_author" content="Dana S. Mosher"/>
            <meta name="citation_author_institution" content="Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Francis Galibert"/>
            <meta name="citation_author_institution" content="CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France"/>
      <meta name="citation_author" content="Catherine André"/>
            <meta name="citation_author_institution" content="CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France"/>
      <meta name="citation_author" content="Elaine A. Ostrander"/>
            <meta name="citation_author_institution" content="Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America"/>
      <meta name="citation_author" content="Christophe Hitte"/>
            <meta name="citation_author_institution" content="CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001324.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1324"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genetic structure of the purebred domestic dog.; citation_author=HG Parker; citation_author=LV Kim; citation_author=NB Sutter; citation_author=S Carlson; citation_author=TD Lorentzen; citation_journal_title=Science; citation_volume=304; citation_number=1; citation_pages=1160-1164; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Breed relationships facilitate fine mapping studies: A 7.8 Kb deletion cosegregates with Collie eye anomaly across multiple dog breeds.; citation_author=HG Parker; citation_author=AV Kukekova; citation_author=DT Akey; citation_author=O Goldstein; citation_author=EF Kirkness; citation_journal_title=Genome Res; citation_volume=17; citation_number=2; citation_pages=1562-1571; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The canine genome.; citation_author=EA Ostrander; citation_author=RK Wayne; citation_journal_title=Genome Res; citation_volume=15; citation_number=3; citation_pages=1706-1716; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Canine genomics and genetics: Running with the pack.; citation_author=HG Parker; citation_author=EA Ostrander; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=4; citation_pages=e58; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=[The dog and its genome].; citation_author=F Galibert; citation_author=C André; citation_journal_title=Med Sci (Paris); citation_volume=22; citation_number=5; citation_pages=806-808; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The dog: a powerful model for studying genotype-phenotype relationships. Comparative Biochemistry and Physiology ParD: Genomics and Proteomics.; citation_author=F Galibert; citation_author=C André; citation_number=6; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Lessons learned from the dog genome.; citation_author=RK Wayne; citation_author=EA Ostrander; citation_journal_title=Trends Genet; citation_number=7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Facilitating genome navigation: survey sequencing and dense radiation-hybrid gene mapping.; citation_author=C Hitte; citation_author=J Madeoy; citation_author=EF Kirkness; citation_author=C Priat; citation_author=TD Lorentzen; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=8; citation_pages=643-648; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A mutation in the myostatin gene increases muscle mass and enhances racing performance in heterozygote dogs.; citation_author=DS Mosher; citation_author=P Quignon; citation_author=CD Bustamante; citation_author=NB Sutter; citation_author=CS Mellersh; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=9; citation_pages=e79; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The dog genome: survey sequencing and comparative analysis.; citation_author=EF Kirkness; citation_author=V Bafna; citation_author=AL Halpern; citation_author=S Levy; citation_author=K Remington; citation_journal_title=Science; citation_volume=301; citation_number=10; citation_pages=1898-1903; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome sequence, comparative analysis and haplotype structure of the domestic dog.; citation_author=K Lindblad-Toh; citation_author=CM Wade; citation_author=TS Mikkelsen; citation_author=EK Karlsson; citation_author=DB Jaffe; citation_journal_title=Nature; citation_volume=438; citation_number=11; citation_pages=803-819; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic basis for systems of skeletal quantitative traits: Principal component analysis of the canid skeleton.; citation_author=K Chase; citation_author=DR Carrier; citation_author=FR Adler; citation_author=T Jarvik; citation_author=EA Ostrander; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=12; citation_pages=9930-9935; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A single IGF1 allele is a major determinant of small size in dogs.; citation_author=NB Sutter; citation_author=CD Bustamante; citation_author=K Chase; citation_author=MM Gray; citation_author=K Zhao; citation_journal_title=Science; citation_volume=316; citation_number=13; citation_pages=112-115; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium mapping in domestic dog breeds narrows the progressive rod-cone degeneration interval and identifies ancestral disease-transmitting chromosome.; citation_author=O Goldstein; citation_author=B Zangerl; citation_author=S Pearce-Kelling; citation_author=DJ Sidjanin; citation_author=JW Kijas; citation_journal_title=Genomics; citation_volume=18; citation_number=14; citation_pages=18; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Extensive and breed-specific linkage disequilibrium in Canis familiaris.; citation_author=NB Sutter; citation_author=MA Eberle; citation_author=HG Parker; citation_author=BJ Pullar; citation_author=EF Kirkness; citation_journal_title=Genome Res; citation_volume=14; citation_number=15; citation_pages=2388-2396; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Linkage mapping of the primary disease locus for collie eye anomaly.; citation_author=JK Lowe; citation_author=AV Kukekova; citation_author=EF Kirkness; citation_author=MC Langlois; citation_author=GD Aguirre; citation_journal_title=Genomics; citation_volume=82; citation_number=16; citation_pages=86-95; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The sleep disorder canine narcolepsy is caused by a mutation in the hypocretin (orexin) receptor 2 gene.; citation_author=L Lin; citation_author=J Faraco; citation_author=R Li; citation_author=H Kadotani; citation_author=W Rogers; citation_journal_title=Cell; citation_volume=98; citation_number=17; citation_pages=365-376; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Linkage analysis and comparative mapping of canine progressive rod-cone degeneration (prcd) establishes potential locus homology with retinitis pigmentosa (RP17) in humans.; citation_author=GM Acland; citation_author=K Ray; citation_author=CS Mellersh; citation_author=W Gu; citation_author=AA Langston; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=18; citation_pages=3048-3053; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A novel retinal degeneration locus identified by linkage and comparative mapping of canine early retinal degeneration.; citation_author=GM Acland; citation_author=K Ray; citation_author=CS Mellersh; citation_author=W Gu; citation_author=AA Langston; citation_journal_title=Genomics; citation_volume=59; citation_number=19; citation_pages=134-142; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Understanding the genetics of autoimmune disease: two loci that regulate late onset Addison's disease in Portuguese Water Dogs.; citation_author=K Chase; citation_author=D Sargan; citation_author=K Miller; citation_author=EA Ostrander; citation_author=KG Lark; citation_journal_title=Int J Immunogenet; citation_volume=33; citation_number=20; citation_pages=179-184; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Coat colour in dogs: identification of the Merle locus in the Australian shepherd breed.; citation_author=B Hédan; citation_author=S Corre; citation_author=C Hitte; citation_author=S Dréano; citation_author=T Vilboux; citation_journal_title=BMC Vet Res; citation_volume=27; citation_number=21; citation_pages=9; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Localization of White Spotting Locus in Boxer Dogs on CFA20 by Genome-Wide Linkage Analysis with 1500 SNPs.; citation_author=P Leegwater; citation_author=M van Hagen; citation_author=B van Oost; citation_journal_title=J Hered; citation_volume=98; citation_number=22; citation_pages=549-552; citation_date=2007; " />
      <meta name="citation_reference" content="citation_number=23; citation_pages=790; citation_date=1998; citation_publisher=Howell Book House; " />
      <meta name="citation_reference" content="citation_title=Clinical and radiographic manifestations of canine malignant histiocytosis.; citation_author=ML Schmidt; citation_author=GR Rutteman; citation_author=MH van Niel; citation_author=PT Wolvekamp; citation_journal_title=Vet Q; citation_volume=15; citation_number=24; citation_pages=117-120; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Localized and disseminated histiocytic sarcoma of dendritic cell origin in dogs.; citation_author=VK Affolter; citation_author=PF Moore; citation_journal_title=Vet Pathol; citation_volume=39; citation_number=25; citation_pages=74-83; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Malignant histiocytosis in three Bernese mountain dogs.; citation_author=IK Ramsey; citation_author=JS McKay; citation_author=H Rudorf; citation_author=JM Dobson; citation_journal_title=Vet Rec; citation_volume=138; citation_number=26; citation_pages=440-444; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Malignant histiocytosis of Bernese mountain dogs.; citation_author=PF Moore; citation_author=A Rosin; citation_journal_title=Vet Pathol; citation_volume=23; citation_number=27; citation_pages=1-10; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Malignant histiocytosis in Bernese Mountain dogs.; citation_author=A Rosin; citation_author=P Moore; citation_author=R Dubielzig; citation_journal_title=J Am Vet Med Assoc; citation_volume=188; citation_number=28; citation_pages=1041-1045; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Association mapping in structured populations.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=NA Rosenberg; citation_author=P Donnelly; citation_journal_title=Am J Hum Genet; citation_volume=67; citation_number=29; citation_pages=170-181; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Common Kibra alleles are associated with human memory performance.; citation_author=A Papassotiropoulos; citation_author=DA Stephan; citation_author=MJ Huentelman; citation_author=FJ Hoerndli; citation_author=DW Craig; citation_journal_title=Science; citation_volume=314; citation_number=30; citation_pages=475-478; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=31; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=BLAT – the BLAST-like alignment tool.; citation_author=W Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=32; citation_pages=656-664; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Haploview: analysis and visualization of LD and haplotype maps.; citation_author=JC Barrett; citation_author=B Fry; citation_author=J Maller; citation_author=MJ Daly; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=33; citation_pages=263-265; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=POPDIST, version 1.1.1: a program to calculate population genetic distance and identity measures.; citation_author=B Guldbrandtsen; citation_author=J Tomiuk; citation_author=V Loeschcke; citation_journal_title=J Hered; citation_volume=91; citation_number=34; citation_pages=178-179; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=35; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Population structure and eigenanalysis.; citation_author=N Patterson; citation_author=A Price; citation_author=D Reich; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=36; citation_pages=e190; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.; citation_author=P Scheet; citation_author=M Stephens; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=37; citation_pages=629-644; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HaploBlockFinder: haplotype block analyses.; citation_author=K Zhang; citation_author=L Jin; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=38; citation_pages=1300-1301; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies"/>
    <meta name="twitter:description" content="BackgroundIn canine genetics, the impact of population structure on whole genome association studies is typically addressed by sampling approximately equal numbers of cases and controls from dogs of a single breed, usually from the same country or geographic area. However one way to increase the power of genetic studies is to sample individuals of the same breed but from different geographic areas, with the expectation that independent meiotic events will have shortened the presumed ancestral haplotype around the mutation differently. Little is known, however, about genetic variation among dogs of the same breed collected from different geographic regions.Methodology/Principal FindingsIn this report, we address the magnitude and impact of genetic diversity among common breeds sampled in the U.S. and Europe. The breeds selected, including the Rottweiler, Bernese mountain dog, flat-coated retriever, and golden retriever, share susceptibility to a class of soft tissue cancers typified by malignant histiocytosis in the Bernese mountain dog. We genotyped 722 SNPs at four unlinked loci (between 95 and 271 per locus) on canine chromosome 1 (CFA1). We showed that each population is characterized by distinct genetic diversity that can be correlated with breed history. When the breed studied has a reduced intra-breed diversity, the combination of dogs from international locations does not increase the rate of false positives and potentially increases the power of association studies. However, over-sampling cases from one geographic location is more likely to lead to false positive results in breeds with significant genetic diversity.ConclusionsThese data provide new guidelines for association studies using purebred dogs that take into account population structure."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001324.g002"/>

  <meta property="og:title" content="Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2305'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5923'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1592&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001324">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001324" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001324&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pascale Quignon, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laetitia Herbin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edouard Cadieu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ewen F. Kirkness, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The J Craig Venter Institute, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Benoit Hédan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dana S. Mosher, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francis Galibert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine André, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elaine A. Ostrander
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:eostrand@mail.nih.gov">eostrand@mail.nih.gov</a> (EO); <a href="mailto:hitte@univ-rennes1.fr">hitte@univ-rennes1.fr</a> (CH)</p>

                <p>Affiliation:
                  Cancer Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christophe Hitte
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:eostrand@mail.nih.gov">eostrand@mail.nih.gov</a> (EO); <a href="mailto:hitte@univ-rennes1.fr">hitte@univ-rennes1.fr</a> (CH)</p>

                <p>Affiliation:
                  CNRS UMR6061 Génétique et Développement, Université de Rennes 1, IFR140, CS 34317, Rennes, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001324</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001324-t001" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001324-t002" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001324-g001" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001324-t003" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001324-g002" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001324-t004" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001324">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001324" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>In canine genetics, the impact of population structure on whole genome association studies is typically addressed by sampling approximately equal numbers of cases and controls from dogs of a single breed, usually from the same country or geographic area. However one way to increase the power of genetic studies is to sample individuals of the same breed but from different geographic areas, with the expectation that independent meiotic events will have shortened the presumed ancestral haplotype around the mutation differently. Little is known, however, about genetic variation among dogs of the same breed collected from different geographic regions.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In this report, we address the magnitude and impact of genetic diversity among common breeds sampled in the U.S. and Europe. The breeds selected, including the Rottweiler, Bernese mountain dog, flat-coated retriever, and golden retriever, share susceptibility to a class of soft tissue cancers typified by malignant histiocytosis in the Bernese mountain dog. We genotyped 722 SNPs at four unlinked loci (between 95 and 271 per locus) on canine chromosome 1 (CFA1). We showed that each population is characterized by distinct genetic diversity that can be correlated with breed history. When the breed studied has a reduced intra-breed diversity, the combination of dogs from international locations does not increase the rate of false positives and potentially increases the power of association studies. However, over-sampling cases from one geographic location is more likely to lead to false positive results in breeds with significant genetic diversity.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These data provide new guidelines for association studies using purebred dogs that take into account population structure.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Quignon P, Herbin L, Cadieu E, Kirkness EF, Hédan B, et al.  (2007) Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies. PLoS ONE 2(12):
          e1324.
            doi:10.1371/journal.pone.0001324</p><p><strong>Academic Editor: </strong>Philip Awadalla, University of Montreal, Canada</p><p><strong>Received:</strong> August 23, 2007; <strong>Accepted:</strong> November 20, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> © 2007 Quignon et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We would like to thank the American Kennel Club Canine Health Foundation, the French Centre National de la Recherche Scientifique (CNRS) and the intramural program of the National Human Genome Research Institute of the National Institutes of Health for supporting this work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The domestic dog species (<em>Canis familiaris</em>) is divided into over 300 pure breeding populations known as breeds. Many breeds are characterized by reduced genetic diversity related to small numbers of founders, popular sires whose allelic pool is over represented in subsequent generations, and changes in breed popularity over time. In addition, no dog can become a registered member of a breed unless both its parents are registered members of the same breed. As a result, while phenotypic variation across breeds is large, within breed variation at the DNA level is considerably more limited than in humans <a href="#pone.0001324-Parker1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In recent years, researchers have taken advantage of growing knowledge about the population structure of dog breeds <a href="#pone.0001324-Parker1">[1]</a>, <a href="#pone.0001324-Parker2">[2]</a> to map genes associated with monogenetic traits (reviewed in <a href="#pone.0001324-Ostrander1">[3]</a>–<a href="#pone.0001324-Wayne1">[7]</a>). Relying on comparative maps generated between dog and human <a href="#pone.0001324-Hitte1">[8]</a>, <a href="#pone.0001324-Mosher1">[9]</a>, the 1.5× poodle sequence <a href="#pone.0001324-Kirkness1">[10]</a>, and, most recently, the whole genome assembly of the boxer <a href="#pone.0001324-LindbladToh1">[11]</a>, researchers have utilized families of dogs from one or a limited number of related breeds to identify loci responsible for variable phenotypes, typically associated with disease susceptibility (reviewed in <a href="#pone.0001324-Ostrander1">[3]</a>, <a href="#pone.0001324-Parker3">[4]</a>, <a href="#pone.0001324-Galibert2">[6]</a>) and morphology<a href="#pone.0001324-Mosher1">[9]</a>, <a href="#pone.0001324-Chase1">[12]</a>, <a href="#pone.0001324-Sutter1">[13]</a>. As recent results demonstrate, the power to both map genes and identify causative variants is improved by comparing data from related breeds which likely share a single ancestral mutation <a href="#pone.0001324-Parker2">[2]</a>, <a href="#pone.0001324-Goldstein1">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Studies suggest that linkage disequilibrium (LD) in dogs extends, on average, for megabases and varies both along the genome and between breeds <a href="#pone.0001324-LindbladToh1">[11]</a>, <a href="#pone.0001324-Sutter2">[15]</a>. While this facilitates the initial mapping stages of locus identification, as only 10–30 thousand informative SNPs are needed for locus identification, it greatly expands the problem of moving from linked marker to gene, as a given LD block can extend for megabases and span over 100 genes (eg: <a href="#pone.0001324-Lowe1">[16]</a>–<a href="#pone.0001324-Leegwater1">[22]</a>).</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>While some dog breeds exist in only one country or geographic region, many, such as a majority of those recognized by the American Kennel Club (AKC) <a href="#pone.0001324-American1">[23]</a>, are distributed worldwide. Power for both meiotic linkage mapping and whole genome association studies is optimized when independent breeding populations sharing a common ancestral mutation can be considered simultaneously. Independent meiotic recombination events optimize the chance that a small region of common haplotype spanning the disease allele can be identified with strong statistical significance.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Many study designs in current use by canine geneticists address the problem of population stratification by sampling cases and controls from dogs of a single breed with little regard for the problem of population sub-structure. Worldwide sampling of pure-breed dogs sharing the same disease may increase power for both finding a locus as well as reducing LD. However it may also increase the likelihood of false positive results as population sub-structure can exist in some breeds. Indeed, the accuracy of association studies will be jeopardized if they are based on the erroneous assumption that all dogs of a single breed share the same level and type of genetic variation. Over representation of a rare allele can lead to the conclusion that a linked marker has been found when, in fact, the frequency of the allele reflects the relatedness of cases in the population. It is therefore important to determine the extent to which dog breeds may be sub-divided into smaller genetically differentiated entities, especially for cohorts sampled from different countries.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Towards that end, we have evaluated the genetic relatedness of independently bred lines of European and American dogs from the Bernese mountain dog (BMD), flat-coated retriever (FCR), golden retriever (GR) and Rottweiler (ROT) breeds. These breeds were selected based on their common susceptibility to malignant histiocytosis, a rare but highly lethal histiocytic cancer <a href="#pone.0001324-Schmidt1">[24]</a>, <a href="#pone.0001324-Affolter1">[25]</a> for which no cure or causative locus has been identified to date and for which genetic studies combining dogs from different geographic locations could be reasonably proposed. The disease is particularly common in BMD and FCR with 20% of the former ultimately succumbing to the disease <a href="#pone.0001324-Ramsey1">[26]</a>–<a href="#pone.0001324-Rosin1">[28]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Our study design utilized a set of 722 SNPs from four loci (95, 96, 260 and 271 SNPs per locus) on CFA1 each of which was genotyped in 120 dogs collected in equal proportions from the US and Europe for each of the four breeds. We analyzed the level and magnitude of diversity within each breed using allelic distribution, haplotype analysis, F<sub>ST</sub> measure and clustering studies. In addition, we determined if within-breed variation was large enough to confound association studies. We studied the joint behavior of chi-square and p-value statistics for SNPs spanning a candidate region centered on a simulated causative SNP using either equal allele frequencies or allele frequencies that differ among subpopulations. Our results indicate that current sampling strategies need to take into account population substructure, particularly in breeds with high genetic diversity. When appropriately designed, the combination of samples from multiple countries or locations can offer better results and reduced false-positive rates.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>SNP genotyping analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Of the 120 dogs initially sampled, data was collected on 119. DNA from one US collected ROT failed to amplify for all reactions. Of the 722 SNPs selected for analysis, 43 failed to generate data in the SNPlex assay. Approximately 80% of genotypes were done in either duplicate or triplicate. The error rate, evaluated as inconstancies between duplicated genotypes, was &lt;1%. The overall genotyping rate success was 86%. To be included in all subsequent analyses a given SNP must have had genotyping success rate greater or equal to 50% and be polymorphic, as defined by a minor allele frequency (MAF) greater than 0.05. Considering the four breeds together and the four genomic regions, 556 SNPs met these criteria.</p>


<h4>Breed population analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We calculated allelic polymorphism rate, heterozygosity rate, genetic distances, linkage disequilibrium (LD), and ultimately performed clustering methods to test the hypothesis that the four individual breeds could be separated using the polymorphism pattern of the genotyping data collected.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The GR had the greatest percentage of polymorphic SNPs (MAF&gt;0.05) with 66.6%, whereas the BMD had the lowest with 49.0%. The ROT and FCR had intermediate values of 54.4% and 57.7%, respectively (<a href="#pone-0001324-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001324-t001"><div class="img"><a name="pone-0001324-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Allele frequencies of the 722 SNPs for the four breeds.</span></strong></p><span>doi:10.1371/journal.pone.0001324.t001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The observed nucleotide heterozygosity (H<sub>o</sub>) rate varied between breeds, with the highest rate observed for the GR (23.7%), and the lowest for the BMD (15.1%). The H<sub>o</sub> rates were close to the H<sub>e</sub> (expected heterozygosity) (p&lt;0.01), indicating no deviation from Hardy-Weinberg equilibrium for any breed (<a href="#pone-0001324-t001">Table 1</a>). Pair-wise genetic distances (F<sub>ST</sub>) between breeds were also calculated, with a mean value of 0.33. Among the six pairs analyzed, the smallest distance was observed between the two retriever breeds FCR and GR (F<sub>ST</sub> = 0.237) while the BMD showed the greatest distance from any of the other breeds (F<sub>ST</sub>&gt;0.34) (<a href="#pone-0001324-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001324-t002"><div class="img"><a name="pone-0001324-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Genetic distances (F<sub>ST</sub>) inter and intra-breed.</span></strong></p><span>doi:10.1371/journal.pone.0001324.t002</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The extent of LD was determined using the D′ statistic. For D′ = 0.5, LD values of 1.2, 1.8, 1.8 and 1.9 Mb were obtained, respectively, for GR, FCR, BMD and ROT. The extent of LD was large and not unexpected based on previous reports <a href="#pone.0001324-LindbladToh1">[11]</a>, <a href="#pone.0001324-Sutter2">[15]</a>. In our study, the GR population demonstrated the shortest average LD, approximately 30% less than the three other breeds.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>To determine the breed structure of the 119 samples we utilized the STRUCTURE <a href="#pone.0001324-Pritchard1">[29]</a> PLINK (<a href="http://pngu.mgh.harvard.edu/purcell/plink">http://pngu.mgh.harvard.edu/purcell/plin​k</a>) and HCLUST (<a href="http://cran.r-project.org/">http://cran.r-project.org/</a>) programs. STRUCTURE will group individuals into populations using either maximum likelihood methods, or by assignment of a preset number of groups (K). This program correctly divided 118 samples into four distinct breeds at the preset value of K = 4. Only one FCR clustered incorrectly and was assigned as a GR, but with a low confidence level of 0.53.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>PLINK perfectly clustered all 119 dogs, with each dog assigned to its correct breed. HCLUST divided 118 dogs into four clusters with one GR remaining as an outlier (<a href="#pone-0001324-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001324-g001"><div class="img"><a name="pone-0001324-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001324.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001324.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Hierarchical clustering between the four breeds.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Clustering of the 119 dogs (k = 4) from the four different breeds. In the dendogram each vertical line represents a single dog and the color reflects the four clusters obtained by PLINK analysis. A grey dashed line indicates an outlier dog. Below the dendogram, dogs are named by their breed origin, and names are colored upon their cluster assignment. At the bottom of the figure, colored squares are drawn on four lines, each line representing the different breeds. The scale on the right axis represents the genetics distances calculated by PLINK software. Each breed separates from the others as the four colors correspond exactly to the four breeds, red for BMD, blue for FCR, black for GR and green for ROT.</p>
<span>doi:10.1371/journal.pone.0001324.g001</span></div>

<h4>Within breed population analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To explore each breed for substructure, we considered the hypothesis that a breed sampled from two geographical areas will correspond to two subsets. For each of the four breeds, the number of polymorphic SNP is very similar for both the US and European sample sets (data not shown). However, we calculated the number of SNPs for which the minor allele in one population becomes the major allele in the other population. For the GR and ROT breeds 4.1% of SNPs have allelic frequencies less than 40% in one population and more than 60% in the other population (<a href="#pone.0001324.s001">Table S1</a>). Only 0.9% and 0.7% of SNPs display this feature in the BMD and FCR, respectively. In addition, we examined allelic frequencies that vary more than 20% between the two populations. For the GR, 22% of SNPs fell into this category, while only 6.1%, 10% and 11% of SNPs for FCR, BMD and ROT, respectively, met this criteria (<a href="#pone.0001324.s002">Table S2</a>). By comparison, an average of 41.5% of SNPs demonstrated an allelic frequency difference ≥20% when considering pairs of breeds (minimum value of 39% when comparing BMD with FCR and a maximum of 43% for BMD and GR) (<a href="#pone.0001324.s003">Table S3</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>F<sub>ST</sub> values were determined for each breed between the European and US populations. Although the F<sub>ST</sub> values were small (mean = 0.044), the distance between US and European populations for the GR (F<sub>ST</sub> = 0.07) and BMD (F<sub>ST</sub> = 0.048) reflects increased genetic variation compared to the FCR and ROT, (F<sub>ST</sub> = 0.03) (<a href="#pone-0001324-t002">Table 2</a>). As a baseline, the F<sub>ST</sub> value between each breed was &gt;0.23 (Range = 0.23–0.39).</p>


<h4>Haplotype analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To further explore the intra-breed analysis, we analyzed haplotype diversity for each locus in each population, for each of the four breeds. We first determined the haplotype blocks using the HaploBlockFinder program. The GR breed had the greatest number of haplotype blocks (n = 62, mean size = 0.61 Mb) while the BMD had the fewest (n = 36, mean size = 1.0 Mb). Within each haplotype block we then studied haplotype diversity (<a href="#pone-0001324-t003">Table 3</a>). The rate of shared haplotypes between US and European dogs varies only slightly for the four breeds (mean = 72.9%, range: 70.1–76.2) with the GR being the lowest (<a href="#pone-0001324-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001324-t003"><div class="img"><a name="pone-0001324-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Number of shared and specific haplotypes between the US and European subpopulations for each breed.</span></strong></p><span>doi:10.1371/journal.pone.0001324.t003</span></div>

<h4>Linkage disequilibrium within breed populations</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We next assessed the extent of LD for each US and European population and compared it to the complete breed data. The extent of LD between the ROTs from US and Europe were very similar (D′ = 0.5, LD of 2.1 and 2.0 Mb, respectively). When dogs from the two ROT subpopulations were combined, the results were similar with only a 5% decrease in LD (LD = 1.96 Mb). By comparison, the extent of LD for the GR between the two subpopulations is 1.4 Mb for the US subpopulation and 1.8 Mb for the European. But when GR subpopulations are combined, the extent of LD is reduced to 1.2 Mb. This 25% decrease in LD is comparable to that observed when combining two distinct breeds. This is expected as analysis of the two populations produces a shared haplotype that is, expectedly, much smaller than that observed for either population individually (<a href="#pone.0001324.s004">Figure S1</a>). For the BMD and FCR, when the US and European subpopulations are combined the LD decreased by only 13% and 11%, respectively.</p>


<h4>Within breed clustering analysis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We used the programs PLINK, HCLUST and smartpca to evaluate population substructure in each of the four breeds. No relevant clustering could be obtained for the FCR or ROT with either approach (<a href="#pone-0001324-g002">Figure 2</a>). However, for the GR, clustering with PLINK separated the two populations cleanly (k = 2, using SNPs with MAF≥0.10). Validity assessment of clusters is eased in our study since we possess <em>a priori</em> knowledge of the geographical origin of the dogs. In each of the two clusters (k = 2), 13 out of 15 dogs from the same geographical origin were assigned to their correct intra-breed cluster. To determine if the validity of the results depends on the clustering method used by PLINK, we used two other methods: HCLUST a hierarchical clustering method and smpartpca, a principal component analysis (PCA). FCR and ROT breeds do not show any separation of the US and the European individuals using either method. Substructure was observed for the BMD only with PCA and did not reflect exactly the country of origin: all the BMD from Europe and four US BMD have negative PC1 values, the remaining 11 US BMD have positive PC1 values (<a href="#pone.0001324.s005">Figure S2</a>). Substructure was also observed for the GR. With HCLUST, 13 of 15 US dogs grouped together into a cluster and 12/14 Europeans dogs separate into a second cluster with one GR remaining as an outlier (<a href="#pone-0001324-g002">Figure 2</a>). With PCA, 14 out of the 15 US dogs have a positive PC1 value and 14 out of the 15 European dogs have a negative PC1 value.</p>
<div class="figure" id="pone-0001324-g002"><div class="img"><a name="pone-0001324-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001324.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001324.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Hierarchical clustering within breeds.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The figure shows the clustering analysis (k = 2) within each breed for BMD, FCR, ROT and GR (n = 30 for each breed). For each breed, a dendogram is drawn in which each vertical line represents a single dog. The two clusters are represented in blue and red, and grey dashed lines indicate outlier dogs. Below the dendogram, dogs are named by geographical origin (US or EU), and names are colored upon their cluster assignment. The scale on the right axis represents the genetics distances calculated by PLINK software.</p>
<span>doi:10.1371/journal.pone.0001324.g002</span></div>

<h4>Whole genome association study simulations</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We have observed that a large number of SNPs have allele frequencies that vary significantly within the GR population between the US and Europe, while the same SNPs demonstrate less variation in the BMD breed. Such diversity is irrespective of disease or traits status, and within–breed population structure could influence even carefully designed studies and affect the validity of association results. To investigate the impact of within-breed genetic diversity in SNP-based whole genome association studies, we randomly selected a single SNP out of a well distributed set (n = 25 for each breed) as the causal mutation of a monogenic and recessive trait, i.e. 100% penetrance and 0% phenocopy rate. The simulated disease-causing SNPs met the following criteria: (1) dogs homozygous for the two alleles had to be found in the European and American subpopulations in order to have equal or similar proportion of the ‘causal’ genotype, i.e. four to six ‘cases’ in each subpopulation; (2) the proportion of European and American dogs for the simulated SNP had to be in Hardy-Weinberg equilibrium. In addition, the 25 chosen SNPs has to belong to different LD blocks to order avoid an artificial increase of significant p-values.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>We studied the joint behavior of chi-square and p-value statistics for SNPs in the 50 Mb covered by our study. All SNPs having a MAF ≥5% were considered in the association analyses. For each breed, p-values were calculated both for the two geographical groups (n = 15 individuals each) and for the entire breed population (n = 30 individuals).</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>We first determined the number of SNPs with a significant p-value (&lt;0.05) in intervals of 1, 1.5 and 2 Mb surrounding the ‘causative’ SNP. The results were similar for all four breeds. Within 1 Mb of the causative SNP, 20 to 35% of SNPs have significant p-values for the European or US subpopulation, while 32 to 44% of SNP showed significant p-values in the combined (US+European) population. We determined that p-values were one log better, from 10e-03 to 10e-04 on average, for the combined population. To assess the potential impact of allelic diversity that arises from the US and European subpopulations, we determined the number of significant p-value for SNPs located beyond 5 Mb of the causative SNP suggesting false-positives (type 1 errors) leading to spurious association. When sampled separately by continent, 5.1, 5.3, 4.1 and 3.3% of SNPs located 5 Mb beyond the simulated causative SNP had significant p-values for BMD, FCR, ROT and GR, respectively (<a href="#pone-0001324-t004">Table 4</a>). When the US and European datasets are combined, no inflation of false positives is observed. Indeed, we observed very similar rates of 5.9, 4.7, 3.5 and 3.7% (<a href="#pone-0001324-t004">Table 4</a>).</p>
<div class="figure" id="pone-0001324-t004"><div class="img"><a name="pone-0001324-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001324" data-uri="info:doi/10.1371/journal.pone.0001324.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001324.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001324.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001324.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001324.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Percentage of SNPs with p-value &lt;0.05 located beyond 5 Mb of a causative SNP.</span></strong></p><span>doi:10.1371/journal.pone.0001324.t004</span></div><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>Concerns regarding population stratification are particularly relevant when considering samples in which the cases and controls were not collected from the same geographic regions. We generated artificial sets of SNPs with extreme allelic differentiation bias based on geographical sampling with cases from only one continent (BMD: n = 9, FCR: n = 8, GR: n = 25 and ROT: n = 18). Causative SNPs were picked as homozygous for samples in the European or the US population in order to have 4 to 6 cases with ‘causal’ genotypes. As was done previously, we calculated p-values for all SNPs spanning the 50 Mb surrounding the simulated SNP ‘variant’. When the number of disease genotypes was unequal in the two populations, we found 7.1, 6.2, 7.2 and 4.3% for BMD, FCR, GR and ROT, respectively. The most significant change was observed for the GR, where we noted a 2-fold increase (7.2% vs 3.5%) of false positives (type I error) compared to the previous analysis, in which the number of cases and controls where equally divided in the two subpopulations. No significant increases were found for BMD, FCR and ROT.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our results demonstrate that, as expected, dog breeds do not constitute homogenous entities and population stratification needs to be systematically assessed in association studies. Even within breeds with large populations, assumptions of random mating and independent allele distribution may be incorrect. Indeed, allele frequencies may vary widely both within and between breed populations irrespective of trait status with the disparities reflecting the unique genetic and social history, ancestral patterns of geographical establishment, mating practices, reproductive expansions and bottlenecks, as well as stochastic events that characterize each breed. We assessed the impact of population stratification on association studies using realistic scenarios considering purebred dogs sampled from Europe and the US. We measured the genetic variability and performed association studies with a simulated disease to assess the extent of the spurious association that can occur if unobserved population stratification exists within canine breeds or if the population structure has not been taken into account.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We used bi-allelic markers located on four unlinked genomic regions on the largest autosome (CFA1) which spans 120 Mb. Anonymous genetic markers spanning large genomic regions, such as SNPs, are considered a good indicator of the level of background diversity in cases and controls. We genotyped SNPs in four different breeds that are prone to malignant histiocystosis, including the Bernese mountain dog, flat-coated retriever, golden retriever and rottweiler.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our results show that the GR has the highest rate of polymorphic SNPs (76.5%) and also has the greatest number of haplotypes within an LD block. These results correlate with the popularity of the breed in both Europe and the US. In the US, the GR rose in popularity in the 1930s (<a href="http://www.akc.org/breeds/golden_retriever/history.cfm">http://www.akc.org/breeds/golden_retriev​er/history.cfm</a>) and was ranked 4<sup>th</sup> among nearly 155 breeds in 2006 with more than 42,000 American Kennel Club (AKC) registrations. In France, GR was ranked 2<sup>nd</sup> in popularity in 2006. Genetic diversity measures such as haplotype sharing or genetic distances (F<sub>ST</sub>) show that the GR has the greatest genetic variability among the four breeds studied. This breed also has the most SNPs with allelic frequency differences; 22% of SNPs have a MAF that varies by more than 20% between the two populations. Those results were supported by the fact that the GRs analyzed in this study separate significantly into two populations corresponding to their geographical origins in clustering analyses. While the US and the European GR populations have common ancestors, the popularity of this breed creates a large breeding pool on both continents and mixing between US and European populations is rare. Thus, clear population differences and associated differences in allele frequency and distribution exist in the GR that can be traced to sampling location.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The BMD has the lowest rate of polymorphic SNPs (61.7%) and the lowest number of haplotypes within an LD block. However, in contrast, they have a significant F<sub>ST</sub> of 0.048 between the European and the US populations. By comparison, the F<sub>ST</sub> values were negative when comparing two populations containing randomly mixed US and European samples. In addition, the PC analysis showed a substructure with all the European samples and 4 US samples in a first cluster, and a second cluster composed of the 11 remaining US samples. This result is predicted by breed history. BMD was originally developed in Switzerland in the region of Berne in 1892. Dogs were initially exported to other European countries in the 1920s and brought in the US in 1926 (<a href="http://www.akc.org/breeds/bernese_mountain_dog/history.cfm">http://www.akc.org/breeds/bernese_mounta​in_dog/history.cfm</a>). The breed has since gained in popularity in several countries and was ranked 41<sup>st</sup> in 2006 with nearly 4,000 new AKC registrations, and ranked 15<sup>th</sup> in 2006 with 2900 new registrations in France (<a href="http://www.scc.asso.fr">http://www.scc.asso.fr</a>). Despite the significant increase in registrations, particularly since the 1970s, BMD are still regularly imported from Europe to the US to enhance breeding programs. The demographic history of BMD may explain why we observed a significant F<sub>ST</sub> value, sub-structure was not always observed between the US and European subpopulations in the clustering analyses.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The FCR had both low allelic diversity and a low F<sub>ST</sub> (F<sub>ST</sub> = 0.03) and of the four breeds studied, the FCR has the lowest number of registrations in the US (551 in 2006 and ranked 100<sup>th</sup>by the AKC). The overall small population size of the breed severely limits the observed genetic variation within the breed, regardless of geographic origin of a particular individual.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The last breed considered in this study was the ROT, which had an intermediate allelic diversity comparable to results observed with the FCR. The ROT went through a drastic decrease of popularity in Europe at the end of the 19<sup>th</sup> century, which was exacerbated during and following World War II (<a href="http://www.akc.org/breeds/rottweiler/history.cfm">http://www.akc.org/breeds/rottweiler/his​tory.cfm</a>). In the US, the ROT is ranked 17<sup>th</sup> in popularity, with 14,700 new AKC registrations in 2006. The popularity of the breed has been in steady decline since the mid 1990s, when there were more than 100,000 new registrations per year (83% decrease in the past decade) (<a href="http://www.akc.org/reg/dogreg_stats.cfm">http://www.akc.org/reg/dogreg_stats.cfm</a>). The drastic population bottleneck that occurred after World War II and to a lesser extent the decline in popularity have limited the gene pool of ROT breed today, and may explain why its observed genetic diversity is not as high as the GR, despite the striking popularity of the ROT in the 1990s.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our study was developed with the aim of assessing inflation of false positive results within individual breeds that could result from population stratification. By using disease simulated genotypes we have shown that type I error rate correlates with both the genetic diversity level and the sampling proportion of cases and controls between sub-populations. These results have important consequences for association studies using a population of purebred dogs sampled from multiple locations. Indeed, we did not observe a significant inflation in the false positive rate for three of the breeds in this study (BMD, FCR and ROT). Those breeds have (1) a small to medium size population and (2) a low allelic diversity between different geographical origins. In such situations, it would be advantageous to sample purebred dogs from multiple international locations to generate a larger sample size, thus increasing the overall power of the study. Indeed, we observed that p-values are one-log better, from 10e-03 to 10e-04 using a two-fold larger dataset. In contrast, the GR data demonstrated that association studies are prone to inflation of type I errors when cases are over sampled from one country. The GR have a high level of genetic diversity between sub-populations and for breeds like the GR, sampling designs need to be conservative.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Since we used a small sample size (30 dogs of each breed) to analyze simulated disease-causative SNPs, one concern is that the power and the reliability of our dataset to detect stratification may be limiting. In the GR, the sample size of 30 was sufficient to allow identification of a clear sub-structure. Although we did not detect stratification for the FCR and ROT, we cannot rule out the possibility that subtle population structure effects would have been evident had we analyzed a larger dataset or additional lineages within each breed. However if such discrete population structure exists, it would likely have little impact on association studies.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In addition to population stratification effects due to geographical origin, non random breeding based on phenotypic traits such as coat color, performance, or breed-associated behaviors such as herding, drafting and hunting may also play a role. In addition, traits such as olfaction, memory and cognitive abilities, which are partially heritable <a href="#pone.0001324-Papassotiropoulos1">[30]</a>, may have been selected in breeds used as service dogs, leading to undetected population structure in studies involving those breeds.</p>

<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We have addressed the magnitude and impact of genetic diversity between four breeds sampled in the U.S and Europe and assessed the impact of within-breed population structure on SNP-based association studies. Based on this study, we recommend that researchers assess population structure through metrics such as F<sub>ST</sub> values, allelic distribution, LD extent and clustering analysis. Knowing the extent of intra-breed sub-structure will increase the likelihood that results from whole genome association studies carried out on collections of dogs from single breeds are accurate. It is not possible to know all the factors that contribute to differences in allele frequency and distribution within a breed. But as these studies show, it is critical to consider such confounding effects to reduce the likelihood of erroneous conclusions.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>DNA samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Fifteen DNA samples from each of four breeds were collected in both Europe and the United States (120 samples total). Blood samples were collected in ACD or EDTA tubes from BMD, FCR, GR and ROT. Efforts were made to sample dogs from different breeders to minimize the chance they were themselves related to one another. For US-collected BMD, FCR, ROT and European-collected BMD dogs, pedigree data was used to verify that the dogs were unrelated to one another to at least the grandparent level. DNA from samples collected in the US was isolated using a proteinase-K, phenol-chloroform protocol <a href="#pone.0001324-Sambrook1">[31]</a>. DNA from samples collected in Europe was isolated using the Pharmacia-Biotech kit (BACC2), as per the manufacturer's instructions.</p>


<h4>SNP selection and genotyping</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Samples were evaluated using single nucleotide polymorphisms (SNPs). A total of 2.5 million distinct SNPs were discovered by the canine whole genome sequencing project (<a href="http://www.broad.mit.edu/mammals/dog">http://www.broad.mit.edu/mammals/dog</a>) by comparing reads of a boxer (7.5× whole genome assembly <a href="#pone.0001324-LindbladToh1">[11]</a>) with a poodle (low-pass sequencing 1.5× coverage <a href="#pone.0001324-Kirkness1">[10]</a>,) and 9 other breeds (German Shepherd, Rottweiler, Bedlington terrier, Beagle, Labrador retriever, English shepherd, Italian greyhound, Alaskan malamute and Portuguese water dog with 100,000 reads for each breed, i.e. 0.02× coverage <a href="#pone.0001324-LindbladToh1">[11]</a>). The 722 SNPs selected from this study were mostly from within the boxer sequence or between the boxer and the 1.5× Poodle. Thus, we expect no bias in the heterozygosity statistics or other analyses. We selected SNPs from four unlinked genomic regions (termed 1, 2 3, and 4) on CFA1 at positions 7–17 Mb, 20–45 Mb, 70–85 Mb and 106–117 Mb. SNPs were selected for which both alleles were present in the boxer sequence, or alternatively, only one allele was present in the boxer sequence, with the other allele present in either the standard poodle sequence or one of the additional nine breeds. A subset of those initially selected was discarded, as they were not suitable for genotyping using the Applied Biosystems SNPlex assay system. This system was used for all subsequent genotyping as it allows simultaneous genotyping of up to 48 SNPs in one assay. Prior to SNPlex assembly sequences surrounding each variant were submitted for Blat analysis <a href="#pone.0001324-Kent1">[32]</a> to ensure that 30 base pairs flanking the SNP were unique in the canine genome. The overall SNP density was 1 SNP/50 kilobases for regions 1 and 4 and 1 SNP/100 kilobases for regions 2 and 3. Genotypes were assigned using GeneMapper software v4.0 (Applied Biosystems).</p>


<h4>Data analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Raw data statistics including the number of scored genotypes, number of polymorphic SNPs as well as number of SNPs with MAF&gt;0.05, average observed and expected heterozygosity were calculated using Haploview software, v4.0 <a href="#pone.0001324-Barrett1">[33]</a>. Population genetic distances (F<sub>ST</sub>) were assessed with genepop (<a href="http://genepop.curtin.edu.au/">http://genepop.curtin.edu.au/</a>), popdist v1.1.2, using the Cavalli-Sforza and Edwards distance reconstructing measure <a href="#pone.0001324-Guldbrandtsen1">[34]</a> and Genetic Data Analysis (GDA) (<a href="http://www.eeb.uconn.edu/people/plewis/software.php">http://www.eeb.uconn.edu/people/plewis/s​oftware.php</a>). Clustering analyses were done using the statistical software package PLINK v 0.99q (<a href="http://pngu.mgh.harvard.edu/purcell/plink">http://pngu.mgh.harvard.edu/purcell/plin​k</a>), the program STRUCTURE <a href="#pone.0001324-Pritchard2">[35]</a>, and hierarchical clustering using the R function Hclust (<a href="http://cran.r-project.org/">http://cran.r-project.org/</a>). Both PLINK and HCLUST use distance matrix to perform clustering. The clustering method used by PLINK will not merge clusters that contain significantly different individuals based on identity-by-state distance. For HCLUST, we used a method that merges clusters having the lowest average distances. Principal component analyses were performed using the smpartpca program included in the EIGENSOFT software <a href="#pone.0001324-Patterson1">[36]</a>. Haplotypes were inferred using fastPHASE software v 1.0.1 <a href="#pone.0001324-Scheet1">[37]</a> with the default settings. This software estimates the missing genotypes and reconstructs haplotypes using unphased genotype data from unrelated individuals. Haplotype blocks were determined with HaploBlockFinder program, v0.7, using SNP with MAF&gt;0.05 <a href="#pone.0001324-Zhang1">[38]</a>. Association studies simulations were conducted with Haploview_4.0beta14 and PLINK_0.99q packages.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001324"></div><a name="pone.0001324.s001" id="pone.0001324.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001324.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Percentage of SNPs with variation of allele frequencies for each breed. 1: percentage of SNP (and number in parenthesis) for which allelic frequency varies more than 10% between US and EU population. 2: difference in allelic frequency more than 20%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001324.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001324.s002" id="pone.0001324.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001324.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Percentage of SNPs with allele frequencies that vary more than 20% between breeds 1: Actual number of SNPs is in parenthesis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001324.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001324.s003" id="pone.0001324.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001324.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Percentage of SNPs with switch from major allele to minor allele in each breed 1: Bernese mountain dog, 2: Flat-coated retriever, 3: Golden retriever, 4: Rottweiler, 6: Number of SNP for which there is a change from minor allele (MAF&lt;0.5) to major allele (MAF&gt;0.5) in percentage and in number of SNP in parenthesis, 7: change from minor allele with MAF&lt;0.4 to major allele with MAF&gt;0.6 and vice-versa in percentage and in number of SNP in parenthesis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001324.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001324.s004" id="pone.0001324.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001324.s004">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Using subtle population sub-structure to shape shorter-range patterns of haplotype and LD. Individuals in isolated dog populations (US or EU) with few or no exchange between them lead to a specific haplotype pattern and extent of LD. Population merged from both US and EU continent generated shorter-range patterns of haplotype, and thereby create shorter-range LD.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001324.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB JPG)</p>
<a name="pone.0001324.s005" id="pone.0001324.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001324.s005">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Principal component analyses were performed with the smartpca program from the Eigenstrat software. Each breed were analyzed separately and the PC1 and PC2 scores of each dog were plotted. Red squares represent European dogs and blue diamond US dogs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001324.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.18 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the American Kennel Club Canine Health Foundation, the French <em>Centre National de la Recherche Scientifique</em> (CNRS) and the intramural program of the National Human Genome Research Institute of the National Institutes of Health for supporting this work. We also thank Stéphane Dréano, Kristelle Lourgant and Adrien Herledan for technical help. Finally, we thank Drs. Gilles Chaudieu, Philippe Pilorge, and David Sargan for sampling European dogs, and the many dog owners and breeders who provided samples for this study.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EO BH FG CA CH. Performed the experiments: PQ EC LH. Analyzed the data: PQ CH. Contributed reagents/materials/analysis tools: EO DM EK FG CA. Wrote the paper: EO PQ FG CH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001324-Parker1" id="pone.0001324-Parker1"></a>Parker HG, Kim LV, Sutter NB, Carlson S, Lorentzen TD, et al.  (2004) Genetic structure of the purebred domestic dog. Science  304: 1160–1164.  <ul class="find" data-citedArticleID="1030996" data-doi="10.1126/science.1097406"><li><a href="http://dx.doi.org/10.1126/science.1097406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+of+the+purebred+domestic+dog." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+of+the+purebred+domestic+dog.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001324-Parker2" id="pone.0001324-Parker2"></a>Parker HG, Kukekova AV, Akey DT, Goldstein O, Kirkness EF, et al.  (2007) Breed relationships facilitate fine mapping studies: A 7.8 Kb deletion cosegregates with Collie eye anomaly across multiple dog breeds. Genome Res  17: 1562–1571.  <ul class="find" data-citedArticleID="1030998" data-doi="10.1101/gr.6772807"><li><a href="http://dx.doi.org/10.1101/gr.6772807" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breed+relationships+facilitate+fine+mapping+studies%3A+A+7.8+Kb+deletion+cosegregates+with+Collie+eye+anomaly+across+multiple+dog+breeds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breed+relationships+facilitate+fine+mapping+studies%3A+A+7.8+Kb+deletion+cosegregates+with+Collie+eye+anomaly+across+multiple+dog+breeds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001324-Ostrander1" id="pone.0001324-Ostrander1"></a>Ostrander EA, Wayne RK (2005) The canine genome. Genome Res  15: 1706–1716.  <ul class="find" data-citedArticleID="1030992" data-doi="10.1101/gr.3736605"><li><a href="http://dx.doi.org/10.1101/gr.3736605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+canine+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+canine+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001324-Parker3" id="pone.0001324-Parker3"></a>Parker HG, Ostrander EA (2005) Canine genomics and genetics: Running with the pack. PLoS Genet  1: e58.  <ul class="find" data-citedArticleID="1031000" data-doi="10.1371/journal.pgen.0010058"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Canine+genomics+and+genetics%3A+Running+with+the+pack." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Canine+genomics+and+genetics%3A+Running+with+the+pack.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001324-Galibert1" id="pone.0001324-Galibert1"></a>Galibert F, André C (2006) [The dog and its genome]. Med Sci (Paris)  22: 806–808.  <ul class="find" data-citedArticleID="1030964" data-doi="10.1051/medsci/20062210806"><li><a href="http://dx.doi.org/10.1051/medsci/20062210806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BThe+dog+and+its+genome%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BThe+dog+and+its+genome%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001324-Galibert2" id="pone.0001324-Galibert2"></a>Galibert F, André C (2007) The dog: a powerful model for studying genotype-phenotype relationships. Comparative Biochemistry and Physiology ParD: Genomics and Proteomics.  In Press.  <ul class="find" data-citedArticleID="1030966" data-doi="10.1016/j.cbd.2007.06.001"><li><a href="http://dx.doi.org/10.1016/j.cbd.2007.06.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dog%3A+a+powerful+model+for+studying+genotype-phenotype+relationships.+Comparative+Biochemistry+and+Physiology+ParD%3A+Genomics+and+Proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dog%3A+a+powerful+model+for+studying+genotype-phenotype+relationships.+Comparative+Biochemistry+and+Physiology+ParD%3A+Genomics+and+Proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001324-Wayne1" id="pone.0001324-Wayne1"></a>Wayne RK, Ostrander EA (2007) Lessons learned from the dog genome. Trends Genet.   <ul class="find" data-citedArticleID="1031022" data-doi="10.1016/j.tig.2007.08.013"><li><a href="http://dx.doi.org/10.1016/j.tig.2007.08.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lessons+learned+from+the+dog+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lessons+learned+from+the+dog+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001324-Hitte1" id="pone.0001324-Hitte1"></a>Hitte C, Madeoy J, Kirkness EF, Priat C, Lorentzen TD, et al.  (2005) Facilitating genome navigation: survey sequencing and dense radiation-hybrid gene mapping. Nat Rev Genet  6: 643–648.  <ul class="find" data-citedArticleID="1030974" data-doi="10.1038/nrg1658"><li><a href="http://dx.doi.org/10.1038/nrg1658" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Facilitating+genome+navigation%3A+survey+sequencing+and+dense+radiation-hybrid+gene+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Facilitating+genome+navigation%3A+survey+sequencing+and+dense+radiation-hybrid+gene+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001324-Mosher1" id="pone.0001324-Mosher1"></a>Mosher DS, Quignon P, Bustamante CD, Sutter NB, Mellersh CS, et al.  (2007) A mutation in the myostatin gene increases muscle mass and enhances racing performance in heterozygote dogs. PLoS Genet  3: e79.  <ul class="find" data-citedArticleID="1030990" data-doi="10.1371/journal.pgen.0030079.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030079.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mutation+in+the+myostatin+gene+increases+muscle+mass+and+enhances+racing+performance+in+heterozygote+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mutation+in+the+myostatin+gene+increases+muscle+mass+and+enhances+racing+performance+in+heterozygote+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001324-Kirkness1" id="pone.0001324-Kirkness1"></a>Kirkness EF, Bafna V, Halpern AL, Levy S, Remington K, et al.  (2003) The dog genome: survey sequencing and comparative analysis. Science  301: 1898–1903.  <ul class="find" data-citedArticleID="1030978" data-doi="10.1126/science.1086432"><li><a href="http://dx.doi.org/10.1126/science.1086432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dog+genome%3A+survey+sequencing+and+comparative+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dog+genome%3A+survey+sequencing+and+comparative+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001324-LindbladToh1" id="pone.0001324-LindbladToh1"></a>Lindblad-Toh K, Wade CM, Mikkelsen TS, Karlsson EK, Jaffe DB, et al.  (2005) Genome sequence, comparative analysis and haplotype structure of the domestic dog. Nature  438: 803–819.  <ul class="find" data-citedArticleID="1030984"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog.&amp;auth=&amp;atitle=Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence%2C+comparative+analysis+and+haplotype+structure+of+the+domestic+dog.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001324-Chase1" id="pone.0001324-Chase1"></a>Chase K, Carrier DR, Adler FR, Jarvik T, Ostrander EA, et al.  (2002) Genetic basis for systems of skeletal quantitative traits: Principal component analysis of the canid skeleton. Proc Natl Acad Sci U S A  99: 9930–9935.  <ul class="find" data-citedArticleID="1030960" data-doi="10.1073/pnas.152333099"><li><a href="http://dx.doi.org/10.1073/pnas.152333099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+basis+for+systems+of+skeletal+quantitative+traits%3A+Principal+component+analysis+of+the+canid+skeleton." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+basis+for+systems+of+skeletal+quantitative+traits%3A+Principal+component+analysis+of+the+canid+skeleton.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001324-Sutter1" id="pone.0001324-Sutter1"></a>Sutter NB, Bustamante CD, Chase K, Gray MM, Zhao K, et al.  (2007) A single IGF1 allele is a major determinant of small size in dogs. Science  316: 112–115.  <ul class="find" data-citedArticleID="1031018" data-doi="10.1126/science.1137045"><li><a href="http://dx.doi.org/10.1126/science.1137045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+IGF1+allele+is+a+major+determinant+of+small+size+in+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+IGF1+allele+is+a+major+determinant+of+small+size+in+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001324-Goldstein1" id="pone.0001324-Goldstein1"></a>Goldstein O, Zangerl B, Pearce-Kelling S, Sidjanin DJ, Kijas JW, et al.  (2006) Linkage disequilibrium mapping in domestic dog breeds narrows the progressive rod-cone degeneration interval and identifies ancestral disease-transmitting chromosome. Genomics  18: 18.  <ul class="find" data-citedArticleID="1030968" data-doi="10.1016/j.ygeno.2006.05.013"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2006.05.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+mapping+in+domestic+dog+breeds+narrows+the+progressive+rod-cone+degeneration+interval+and+identifies+ancestral+disease-transmitting+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+mapping+in+domestic+dog+breeds+narrows+the+progressive+rod-cone+degeneration+interval+and+identifies+ancestral+disease-transmitting+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001324-Sutter2" id="pone.0001324-Sutter2"></a>Sutter NB, Eberle MA, Parker HG, Pullar BJ, Kirkness EF, et al.  (2004) Extensive and breed-specific linkage disequilibrium in Canis familiaris. Genome Res  14: 2388–2396.  <ul class="find" data-citedArticleID="1031020" data-doi="10.1101/gr.3147604"><li><a href="http://dx.doi.org/10.1101/gr.3147604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+and+breed-specific+linkage+disequilibrium+in+Canis+familiaris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+and+breed-specific+linkage+disequilibrium+in+Canis+familiaris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001324-Lowe1" id="pone.0001324-Lowe1"></a>Lowe JK, Kukekova AV, Kirkness EF, Langlois MC, Aguirre GD, et al.  (2003) Linkage mapping of the primary disease locus for collie eye anomaly. Genomics  82: 86–95.  <ul class="find" data-citedArticleID="1030986" data-doi="10.1016/s0888-7543(03)00078-8"><li><a href="http://dx.doi.org/10.1016/s0888-7543(03)00078-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+mapping+of+the+primary+disease+locus+for+collie+eye+anomaly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+mapping+of+the+primary+disease+locus+for+collie+eye+anomaly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001324-Lin1" id="pone.0001324-Lin1"></a>Lin L, Faraco J, Li R, Kadotani H, Rogers W, et al.  (1999) The sleep disorder canine narcolepsy is caused by a mutation in the hypocretin (orexin) receptor 2 gene. Cell  98: 365–376.  <ul class="find" data-citedArticleID="1030982" data-doi="10.1016/s0092-8674(00)81965-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81965-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sleep+disorder+canine+narcolepsy+is+caused+by+a+mutation+in+the+hypocretin+%28orexin%29+receptor+2+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sleep+disorder+canine+narcolepsy+is+caused+by+a+mutation+in+the+hypocretin+%28orexin%29+receptor+2+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001324-Acland1" id="pone.0001324-Acland1"></a>Acland GM, Ray K, Mellersh CS, Gu W, Langston AA, et al.  (1998) Linkage analysis and comparative mapping of canine progressive rod-cone degeneration (prcd) establishes potential locus homology with retinitis pigmentosa (RP17) in humans. Proc Natl Acad Sci U S A  95: 3048–3053.  <ul class="find" data-citedArticleID="1030950" data-doi="10.1073/pnas.95.6.3048"><li><a href="http://dx.doi.org/10.1073/pnas.95.6.3048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+analysis+and+comparative+mapping+of+canine+progressive+rod-cone+degeneration+%28prcd%29+establishes+potential+locus+homology+with+retinitis+pigmentosa+%28RP17%29+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+analysis+and+comparative+mapping+of+canine+progressive+rod-cone+degeneration+%28prcd%29+establishes+potential+locus+homology+with+retinitis+pigmentosa+%28RP17%29+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001324-Acland2" id="pone.0001324-Acland2"></a>Acland GM, Ray K, Mellersh CS, Gu W, Langston AA, et al.  (1999) A novel retinal degeneration locus identified by linkage and comparative mapping of canine early retinal degeneration. Genomics  59: 134–142.  <ul class="find" data-citedArticleID="1030952" data-doi="10.1006/geno.1999.5842"><li><a href="http://dx.doi.org/10.1006/geno.1999.5842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+retinal+degeneration+locus+identified+by+linkage+and+comparative+mapping+of+canine+early+retinal+degeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+retinal+degeneration+locus+identified+by+linkage+and+comparative+mapping+of+canine+early+retinal+degeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001324-Chase2" id="pone.0001324-Chase2"></a>Chase K, Sargan D, Miller K, Ostrander EA, Lark KG (2006) Understanding the genetics of autoimmune disease: two loci that regulate late onset Addison's disease in Portuguese Water Dogs. Int J Immunogenet  33: 179–184.  <ul class="find" data-citedArticleID="1030962" data-doi="10.1111/j.1744-313x.2006.00593.x"><li><a href="http://dx.doi.org/10.1111/j.1744-313x.2006.00593.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+the+genetics+of+autoimmune+disease%3A+two+loci+that+regulate+late+onset+Addison%27s+disease+in+Portuguese+Water+Dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+the+genetics+of+autoimmune+disease%3A+two+loci+that+regulate+late+onset+Addison%27s+disease+in+Portuguese+Water+Dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001324-Hdan1" id="pone.0001324-Hdan1"></a>Hédan B, Corre S, Hitte C, Dréano S, Vilboux T, et al.  (2006) Coat colour in dogs: identification of the Merle locus in the Australian shepherd breed. BMC Vet Res  27: 9.  <ul class="find" data-citedArticleID="1030972" data-doi="10.1186/1746-6148-2-9"><li><a href="http://dx.doi.org/10.1186/1746-6148-2-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coat+colour+in+dogs%3A+identification+of+the+Merle+locus+in+the+Australian+shepherd+breed." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coat+colour+in+dogs%3A+identification+of+the+Merle+locus+in+the+Australian+shepherd+breed.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001324-Leegwater1" id="pone.0001324-Leegwater1"></a>Leegwater P, van Hagen M, van Oost B (2007) Localization of White Spotting Locus in Boxer Dogs on CFA20 by Genome-Wide Linkage Analysis with 1500 SNPs. J Hered  98: 549–552.  <ul class="find" data-citedArticleID="1030980" data-doi="10.1093/jhered/esm022"><li><a href="http://dx.doi.org/10.1093/jhered/esm022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localization+of+White+Spotting+Locus+in+Boxer+Dogs+on+CFA20+by+Genome-Wide+Linkage+Analysis+with+1500+SNPs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localization+of+White+Spotting+Locus+in+Boxer+Dogs+on+CFA20+by+Genome-Wide+Linkage+Analysis+with+1500+SNPs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001324-American1" id="pone.0001324-American1"></a> krug , editor. American KC (1998) The Complete Dog Book. New York, NY: Howell Book House.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001324-Schmidt1" id="pone.0001324-Schmidt1"></a>Schmidt ML, Rutteman GR, van Niel MH, Wolvekamp PT (1993) Clinical and radiographic manifestations of canine malignant histiocytosis. Vet Q  15: 117–120.  <ul class="find" data-citedArticleID="1031016" data-doi="10.1080/01652176.1993.9694387"><li><a href="http://dx.doi.org/10.1080/01652176.1993.9694387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clinical+and+radiographic+manifestations+of+canine+malignant+histiocytosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clinical+and+radiographic+manifestations+of+canine+malignant+histiocytosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001324-Affolter1" id="pone.0001324-Affolter1"></a>Affolter VK, Moore PF (2002) Localized and disseminated histiocytic sarcoma of dendritic cell origin in dogs. Vet Pathol  39: 74–83.  <ul class="find" data-citedArticleID="1030954" data-doi="10.1354/vp.39-1-74"><li><a href="http://dx.doi.org/10.1354/vp.39-1-74" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localized+and+disseminated+histiocytic+sarcoma+of+dendritic+cell+origin+in+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localized+and+disseminated+histiocytic+sarcoma+of+dendritic+cell+origin+in+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001324-Ramsey1" id="pone.0001324-Ramsey1"></a>Ramsey IK, McKay JS, Rudorf H, Dobson JM (1996) Malignant histiocytosis in three Bernese mountain dogs. Vet Rec  138: 440–444.  <ul class="find" data-citedArticleID="1031008" data-doi="10.1136/vr.138.18.440"><li><a href="http://dx.doi.org/10.1136/vr.138.18.440" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Malignant+histiocytosis+in+three+Bernese+mountain+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Malignant+histiocytosis+in+three+Bernese+mountain+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001324-Moore1" id="pone.0001324-Moore1"></a>Moore PF, Rosin A (1986) Malignant histiocytosis of Bernese mountain dogs. Vet Pathol  23: 1–10.  <ul class="find" data-citedArticleID="1030988"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Malignant+histiocytosis+of+Bernese+mountain+dogs.&amp;auth=&amp;atitle=Malignant+histiocytosis+of+Bernese+mountain+dogs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Malignant+histiocytosis+of+Bernese+mountain+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Malignant+histiocytosis+of+Bernese+mountain+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001324-Rosin1" id="pone.0001324-Rosin1"></a>Rosin A, Moore P, Dubielzig R (1986) Malignant histiocytosis in Bernese Mountain dogs. J Am Vet Med Assoc  188: 1041–1045.  <ul class="find" data-citedArticleID="1031010"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Malignant+histiocytosis+in+Bernese+Mountain+dogs.&amp;auth=&amp;atitle=Malignant+histiocytosis+in+Bernese+Mountain+dogs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Malignant+histiocytosis+in+Bernese+Mountain+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Malignant+histiocytosis+in+Bernese+Mountain+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001324-Pritchard1" id="pone.0001324-Pritchard1"></a>Pritchard JK, Stephens M, Rosenberg NA, Donnelly P (2000) Association mapping in structured populations. Am J Hum Genet  67: 170–181.  <ul class="find" data-citedArticleID="1031004" data-doi="10.1086/302959"><li><a href="http://dx.doi.org/10.1086/302959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+mapping+in+structured+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+mapping+in+structured+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001324-Papassotiropoulos1" id="pone.0001324-Papassotiropoulos1"></a>Papassotiropoulos A, Stephan DA, Huentelman MJ, Hoerndli FJ, Craig DW, et al.  (2006) Common Kibra alleles are associated with human memory performance. Science  314: 475–478.  <ul class="find" data-citedArticleID="1030994" data-doi="10.1126/science.1129837"><li><a href="http://dx.doi.org/10.1126/science.1129837" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+Kibra+alleles+are+associated+with+human+memory+performance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+Kibra+alleles+are+associated+with+human+memory+performance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001324-Sambrook1" id="pone.0001324-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning: A laboratory manual. N.Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001324-Kent1" id="pone.0001324-Kent1"></a>Kent W (2002) BLAT – the BLAST-like alignment tool. Genome Res  12: 656–664.  <ul class="find" data-citedArticleID="1030976" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT+%E2%80%93+the+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT+%E2%80%93+the+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001324-Barrett1" id="pone.0001324-Barrett1"></a>Barrett JC, Fry B, Maller J, Daly MJ (2005) Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics  21: 263–265.  <ul class="find" data-citedArticleID="1030958" data-doi="10.1093/bioinformatics/bth457"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haploview%3A+analysis+and+visualization+of+LD+and+haplotype+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haploview%3A+analysis+and+visualization+of+LD+and+haplotype+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001324-Guldbrandtsen1" id="pone.0001324-Guldbrandtsen1"></a>Guldbrandtsen B, Tomiuk J, Loeschcke V (2000) POPDIST, version 1.1.1: a program to calculate population genetic distance and identity measures. J Hered  91: 178–179.  <ul class="find" data-citedArticleID="1030970" data-doi="10.1093/jhered/91.2.178"><li><a href="http://dx.doi.org/10.1093/jhered/91.2.178" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=POPDIST%2C+version+1.1.1%3A+a+program+to+calculate+population+genetic+distance+and+identity+measures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22POPDIST%2C+version+1.1.1%3A+a+program+to+calculate+population+genetic+distance+and+identity+measures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001324-Pritchard2" id="pone.0001324-Pritchard2"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="1031006"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001324-Patterson1" id="pone.0001324-Patterson1"></a>Patterson N, Price A, Reich D (2006) Population structure and eigenanalysis. PLoS Genet  2: e190.  <ul class="find" data-citedArticleID="1031002" data-doi="10.1371/journal.pgen.0020190"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020190" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure+and+eigenanalysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure+and+eigenanalysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001324-Scheet1" id="pone.0001324-Scheet1"></a>Scheet P, Stephens M (2006) A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase. Am J Hum Genet  78: 629–644.  <ul class="find" data-citedArticleID="1031014" data-doi="10.1086/502802"><li><a href="http://dx.doi.org/10.1086/502802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+fast+and+flexible+statistical+model+for+large-scale+population+genotype+data%3A+applications+to+inferring+missing+genotypes+and+haplotypic+phase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+fast+and+flexible+statistical+model+for+large-scale+population+genotype+data%3A+applications+to+inferring+missing+genotypes+and+haplotypic+phase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001324-Zhang1" id="pone.0001324-Zhang1"></a>Zhang K, Jin L (2003) HaploBlockFinder: haplotype block analyses. Bioinformatics  19: 1300–1301.  <ul class="find" data-citedArticleID="1031024" data-doi="10.1093/bioinformatics/btg142"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg142" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HaploBlockFinder%3A+haplotype+block+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HaploBlockFinder%3A+haplotype+block+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.XML" value="91781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.PDF" value="2246969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t001.PNG_L" value="100274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t001.PNG_M" value="53964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t001.PNG_S" value="8520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t001.TIF" value="440250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t001.PNG_I" value="16371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t002.PNG_L" value="75828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t002.PNG_M" value="46745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t002.PNG_S" value="8541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t002.TIF" value="357666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t002.PNG_I" value="14257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g001.PNG_L" value="437407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g001.PNG_M" value="123256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g001.PNG_S" value="8723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g001.TIF" value="1504790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g001.PNG_I" value="40917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t003.PNG_L" value="59659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t003.PNG_M" value="78292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t003.PNG_S" value="12696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t003.TIF" value="250554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t003.PNG_I" value="27314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g002.PNG_L" value="558235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g002.PNG_M" value="141640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g002.PNG_S" value="9678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g002.TIF" value="1374600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.g002.PNG_I" value="46826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t004.PNG_L" value="72396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t004.PNG_M" value="96849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t004.PNG_S" value="14167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t004.TIF" value="315314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.t004.PNG_I" value="34760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.s001.DOC" value="22016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.s002.DOC" value="22528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.s003.DOC" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.s004.JPG" value="37985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001324.s005.TIF" value="208324"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001324&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001324" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001324&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001324&volume=&issue=&title=Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies&author_name=Pascale%20Quignon%2C%20Laetitia%20Herbin%2C%20Edouard%20Cadieu%2C%20Ewen%20F.%20Kirkness%2C%20Benoit%20H%C3%A9dan%2C%20Dana%20S.%20Mosher%2C%20Francis%20Galibert%2C%20Catherine%20Andr%C3%A9%2C%20Elaine%20A.%20Ostrander%2C%20Christophe%20Hitte&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001324" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324&amp;t=Canine%20Population%20Structure%3A%20Assessment%20and%20Impact%20of%20Intra-Breed%20Stratification%20on%20SNP-Based%20Association%20Studies" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324&title=Canine%20Population%20Structure%3A%20Assessment%20and%20Impact%20of%20Intra-Breed%20Stratification%20on%20SNP-Based%20Association%20Studies&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324&amp;title=Canine%20Population%20Structure%3A%20Assessment%20and%20Impact%20of%20Intra-Breed%20Stratification%20on%20SNP-Based%20Association%20Studies" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001324&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Canine Population Structure: Assessment and Impact of Intra-Breed Stratification on SNP-Based Association Studies';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001324';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Canine%20Population%20Structure%3A%20Assessment%20and%20Impact%20of%20Intra-Breed%20Stratification%20on%20SNP-Based%20Association%20Studies http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001324" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001324" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="25478"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Dogs%22" title="Search for articles in the subject area:'Dogs'"><div class="flagText">Dogs</div></a>
              <div data-categoryid="42823" data-articleid="25478"
                   data-categoryname="Dogs"
                   class="flagImage" title="Flag 'Dogs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="25478"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="25478"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pets+and+companion+animals%22" title="Search for articles in the subject area:'Pets and companion animals'"><div class="flagText">Pets and companion animals</div></a>
              <div data-categoryid="22935" data-articleid="25478"
                   data-categoryname="Pets and companion animals"
                   class="flagImage" title="Flag 'Pets and companion animals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25478"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Stratigraphy%22" title="Search for articles in the subject area:'Stratigraphy'"><div class="flagText">Stratigraphy</div></a>
              <div data-categoryid="20919" data-articleid="25478"
                   data-categoryname="Stratigraphy"
                   class="flagImage" title="Flag 'Stratigraphy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="25478"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6279'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=555'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3192&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=15749">How would this work across different 'kinds' of breeds?</a><br>Posted by GregLaden</p>
      <p><a href="/annotation/listThread.action?root=8773">Why would this be?</a><br>Posted by asolove</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
