

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001963"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001963" />

    <meta name="citation_title" content="Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices"/>
    <meta itemprop="name" content="Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices"/>

      <meta name="citation_author" content="Akira R. Kinjo"/>
            <meta name="citation_author_institution" content="Institute for Protein Research, Osaka University, Suita, Osaka, Japan"/>
      <meta name="citation_author" content="Haruki Nakamura"/>
            <meta name="citation_author_institution" content="Institute for Protein Research, Osaka University, Suita, Osaka, Japan"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001963.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1963"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Identification of protein sequence homology by consensus template alignment.; citation_author=WR Taylor; citation_journal_title=J Mol Biol; citation_volume=188; citation_number=1; citation_pages=233-258; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Profile Analysis: Detection of Distantly Related Proteins.; citation_author=M Gribskov; citation_author=AD McLachlan; citation_author=D Eisenberg; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=84; citation_number=2; citation_pages=4355-4358; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Hidden Markov Models.; citation_author=SR Eddy; citation_journal_title=Curr Opin Struct Biol; citation_volume=6; citation_number=3; citation_pages=361-365; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Gapped Blast and PSI-Blast: A new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=4; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A Model of Evolutionary Change in Proteins.; citation_author=MO Dayhoff; citation_author=RM Schwartz; citation_author=BC Orcutt; citation_number=5; citation_pages=345-352; citation_date=1978; citation_publisher=National Biomedical Research Foundation; " />
      <meta name="citation_reference" content="citation_title=Amino Acid Substitution Matrices from Protein Blocks.; citation_author=S Henikoff; citation_author=JG Henikoff; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=89; citation_number=6; citation_pages=10915-10919; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Amino Acid Substitution Matrices from an Information Theoretic Perspective.; citation_author=SF Altschul; citation_journal_title=J Mol Biol; citation_volume=219; citation_number=7; citation_pages=555-565; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Analysis of Amino Acid Indices and Mutation Matrices for Sequence Comparison and Structure Prediction of Proteins.; citation_author=K Tomii; citation_author=M Kanehisa; citation_journal_title=Protein Eng; citation_volume=9; citation_number=8; citation_pages=27-36; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Eigenvalue analysis of amino acid substitution matrices reveals a sharp transition of the mode of sequence conservation in proteins.; citation_author=AR Kinjo; citation_author=K Nishikawa; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=9; citation_pages=2504-2508; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ideal amino acid exchange forms for approximating substitution matrices.; citation_author=P Pokarowski; citation_author=A Kloczkowski; citation_author=S Nowakowski; citation_author=M Pokarowska; citation_author=RL Jernigan; citation_journal_title=Proteins; citation_volume=69; citation_number=10; citation_pages=379-393; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=RF Doolittle; citation_number=11; citation_date=1986; citation_publisher=University Science Books; " />
      <meta name="citation_reference" content="citation_title=Twilight Zone of Protein Sequence Alignments.; citation_author=B Rost; citation_journal_title=Protein Eng; citation_volume=12; citation_number=12; citation_pages=85-94; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Optimization of a New Score Function for the Detection of Remote Homologs.; citation_author=M Kann; citation_author=B Qian; citation_author=RA Goldstein; citation_journal_title=Proteins; citation_volume=41; citation_number=13; citation_pages=498-503; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Optimization of a New Score Function for the Generation of Accurate Alignments.; citation_author=B Qian; citation_author=RA Goldstein; citation_journal_title=Proteins; citation_volume=48; citation_number=14; citation_pages=605-610; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=R Durbin; citation_author=R Eddy; citation_author=A Krogh; citation_author=G Mitchison; citation_number=15; citation_date=1999; citation_publisher=Cambridge Univ. Press; " />
      <meta name="citation_reference" content="citation_title=Statistical Analysis of the Physical Properties of the 20 Naturally Occurring Amino Acids.; citation_author=A Kidera; citation_author=Y Konishi; citation_author=M Oka; citation_author=T Ooi; citation_author=HA Scheraga; citation_journal_title=J Protein Chem; citation_volume=4; citation_number=16; citation_pages=23-55; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis of amino acid indices for prediction of protein structure and function.; citation_author=K Nakai; citation_author=A Kidera; citation_author=M Kanehisa; citation_journal_title=Protein Eng; citation_volume=2; citation_number=17; citation_pages=93-100; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=AAindex: amino acid index database.; citation_author=S Kawashima; citation_author=H Ogata; citation_author=M Kanehisa; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=18; citation_pages=368-369; citation_date=1999; " />
      <meta name="citation_reference" content="citation_author=RA Horn; citation_author=CR Johnson; citation_number=19; citation_date=1985; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=20; citation_pages=235-242; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services.; citation_author=RD Finn; citation_author=J Mistry; citation_author=B Schuster-Böckler; citation_author=S Griffiths-Jones; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=21; citation_pages=D247-D251; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PISCES: a protein sequence culling server.; citation_author=G Wang; citation_author=RL Dunbrack; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=22; citation_pages=1589-1591; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=SCOP: A structural classification of proteins database for the investigation of sequences and structures.; citation_author=AG Murzin; citation_author=SE Brenner; citation_author=T Hubbard; citation_author=C Chothia; citation_journal_title=J Mol Biol; citation_volume=247; citation_number=23; citation_pages=536-540; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=UniRef: comprehensive and non-redundant UniProt reference clusters.; citation_author=BE Suzek; citation_author=H Huang; citation_author=P McGarvey; citation_author=R Mazumder; citation_author=CH Wu; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=24; citation_pages=1282-1288; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Rapid Generation of Mutation Data Matrices from Protein Sequences.; citation_author=DT Jones; citation_author=WR Taylor; citation_author=JM Thornton; citation_journal_title=Comput Appl Biosci; citation_volume=8; citation_number=25; citation_pages=275-282; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Protein surface amino acid compositions distinctively differ between thermophilic and mesophilic bacteria.; citation_author=S Fukuchi; citation_author=K Nishikawa; citation_journal_title=J Mol Biol; citation_volume=309; citation_number=26; citation_pages=835-843; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Une méthode statistique simple de prédiction des conformations protéiques.; citation_author=F Beghin; citation_author=J Dirkx; citation_journal_title=Arch Int Physiol Biochim; citation_volume=83; citation_number=27; citation_pages=167-168; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=A simple method for displaying the hydropathic character of a protein.; citation_author=J Kyte; citation_author=RF Doolittle; citation_journal_title=J Mol Biol; citation_volume=157; citation_number=28; citation_pages=105-132; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Relations between chemical structure and biological activity in peptides.; citation_author=PHA Sneath; citation_journal_title=J Theor Biol; citation_volume=12; citation_number=29; citation_pages=157-195; citation_date=1966; " />
      <meta name="citation_reference" content="citation_author=TM Cover; citation_author=JA Thomas; citation_number=30; citation_date=2006; citation_publisher=John Wiley and Sons; " />
      <meta name="citation_reference" content="citation_title=Self-consistent estimation of inter-residue protein contact energies based on an equilibrium mixture approximation of residues.; citation_author=S Miyazawa; citation_author=RL Jernigan; citation_journal_title=Proteins; citation_volume=34; citation_number=31; citation_pages=49-68; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Principal eigenvector of contact matrices and hydrophobicity profiles in proteins.; citation_author=U Bastolla; citation_author=M Porto; citation_author=HE Roman; citation_author=M Vendruscolo; citation_journal_title=Proteins; citation_volume=58; citation_number=32; citation_pages=22-30; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Amino acid difference formula to help explain protein evolution.; citation_author=R Grantham; citation_journal_title=Science; citation_volume=185; citation_number=33; citation_pages=862-864; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Correlation of sequence hydrophobicities measures similarity in three-dimensional protein structure.; citation_author=RM Sweet; citation_author=D Eisenberg; citation_journal_title=J Mol Biol; citation_volume=171; citation_number=34; citation_pages=479-488; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Prediction of the surface-interior diagram of globular proteins by an empirical method.; citation_author=K Nishikawa; citation_author=T Ooi; citation_journal_title=Int J Peptide Protein Res; citation_volume=16; citation_number=35; citation_pages=19-32; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Radial locations of amino acid residues in a globular protein: Correlation with the sequence.; citation_author=K Nishikawa; citation_author=T Ooi; citation_journal_title=J Biochem; citation_volume=100; citation_number=36; citation_pages=1043-1047; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Predicting absolute contact numbers of native protein structure from amino acid sequence.; citation_author=AR Kinjo; citation_author=K Horimoto; citation_author=K Nishikawa; citation_journal_title=Proteins; citation_volume=58; citation_number=37; citation_pages=158-165; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure-based conformational preferences of amino acids.; citation_author=P Koehl; citation_author=M Levitt; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=96; citation_number=38; citation_pages=12524-12529; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Areas, volumes, packing, and protein structure.; citation_author=FM Richards; citation_journal_title=Ann Rev Biophys Bioeng; citation_volume=6; citation_number=39; citation_pages=151-176; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=The Quickhull algorithm for convex hulls.; citation_author=C Barber; citation_author=D Dobkin; citation_author=H Huhdanpaa; citation_journal_title=ACM Trans on Math Software; citation_volume=22; citation_number=40; citation_pages=469-483; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Helix capping.; citation_author=R Aurora; citation_author=GD Rose; citation_journal_title=Protein Sci; citation_volume=7; citation_number=41; citation_pages=21-38; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Prediction of protein function from sequence properties: Discriminant analysis of a data base.; citation_author=P Klein; citation_author=M Kanehisa; citation_author=C DeLisi; citation_journal_title=Biochim Biophys Acta; citation_volume=787; citation_number=42; citation_pages=221-226; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=1H-nmr parameters of the common amino acid residues measured in aqueous solutions of the linear tetrapeptides H-Gly-Gly-X-L-Ala-OH.; citation_author=A Bundi; citation_author=K Wuthrich; citation_journal_title=Biopolymers; citation_volume=18; citation_number=43; citation_pages=285-297; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=High Resolution Crystallographic Analysis of a Co-operative Dimeric Hemoglobin.; citation_author=WE Royer; citation_journal_title=J Mol Biol; citation_volume=235; citation_number=44; citation_pages=657-681; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Determinants of a protein fold.; citation_author=D Bashford; citation_author=C Chothia; citation_author=AM Lesk; citation_journal_title=J Mol Biol; citation_volume=196; citation_number=45; citation_pages=199-216; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Structural divergence and distant relationships in proteins: evolution of the globins.; citation_author=JT Lecomte; citation_author=DA Vuletich; citation_author=AM Lesk; citation_journal_title=Curr Opin Struct Biol; citation_volume=15; citation_number=46; citation_pages=290-301; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural requirement of highly-conserved residues in globins.; citation_author=M Ota; citation_author=Y Isogai; citation_author=K Nishikawa; citation_journal_title=FEBS Lett; citation_volume=415; citation_number=47; citation_pages=129-133; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A method to identify protein sequences that fold into a known three-dimensional structure.; citation_author=JU Bowie; citation_author=R Lüthy; citation_author=D Eisenberg; citation_journal_title=Science; citation_volume=253; citation_number=48; citation_pages=164-170; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Assessment of pseudo-energy potentials by the best-five test: A new use of the three-dimensional profiles of proteins.; citation_author=M Ota; citation_author=K Nishikawa; citation_journal_title=Protein Eng; citation_volume=10; citation_number=49; citation_pages=339-351; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Grouping of amino acid types and extraction of amino acid properties from multiple sequence alignments using variance maximization.; citation_author=JO Wrabl; citation_author=NV Grishin; citation_journal_title=Proteins; citation_volume=61; citation_number=50; citation_pages=523-534; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reconstruction of protein structures from a vectorial representation.; citation_author=M Porto; citation_author=U Bastolla; citation_author=HE Roman; citation_author=M Vendruscolo; citation_journal_title=Phys Rev Lett; citation_volume=92; citation_number=51; citation_pages=218101; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recoverable one-dimensional encoding of three-dimensional protein structures.; citation_author=AR Kinjo; citation_author=K Nishikawa; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=52; citation_pages=2167-2170; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Better prediction of protein contact number using a support vector regression analysis of amino acid sequence.; citation_author=Z Yuan; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=53; citation_pages=248; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Predicting secondary structures, contact numbers, and residue-wise contact orders of native protein structure from amino acid sequence using critical random networks.; citation_author=AR Kinjo; citation_author=K Nishikawa; citation_journal_title=BIOPHYSICS; citation_volume=1; citation_number=54; citation_pages=67-74; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Potential for assessing quality of protein structure based on contact number prediction.; citation_author=T Ishida; citation_author=S Nakamura; citation_author=K Shimizu; citation_journal_title=Proteins; citation_volume=64; citation_number=55; citation_pages=940-947; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CRNPRED: Highly accurate prediction of one-dimensional protein structures by large-scale critical random networks.; citation_author=AR Kinjo; citation_author=K Nishikawa; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=56; citation_pages=401; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=On the optimal contact potential of proteins.; citation_author=AR Knjo; citation_author=S Miyazawa; citation_journal_title=Chem Phys Lett; citation_volume=451; citation_number=57; citation_pages=132-135; citation_date=2008; " />
      <meta name="citation_reference" content="citation_title=Identification of protein folds: matching hydrophobicity patterns of sequence sets with solvent accessibility patterns of known structures.; citation_author=JU Bowie; citation_author=ND Clarke; citation_author=CO Pabo; citation_author=RT Sauer; citation_journal_title=Proteins; citation_volume=7; citation_number=58; citation_pages=257-264; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A new approach to protein fold recognition.; citation_author=DT Jones; citation_author=WR Taylor; citation_author=JM Thornton; citation_journal_title=Nature; citation_volume=358; citation_number=59; citation_pages=86-89; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Probing the catalytic mechanism of GDP-4-keto-6-deoxy-d-mannose Epimerase/Reductase by kinetic and crystallographic characterization of site-specific mutants.; citation_author=C Rosano; citation_author=A Bisso; citation_author=G Izzo; citation_author=M Tonetti; citation_author=L Sturla; citation_journal_title=J Mol Biol; citation_volume=303; citation_number=60; citation_pages=77-91; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Use of multiple anomalous dispersion to phase highly merohedrally twinned crystals of interleukin-1beta.; citation_author=MG Rudolph; citation_author=MS Kelker; citation_author=TR Schneider; citation_author=TO Yeates; citation_author=V Oseroff; citation_journal_title=Acta Crystallogr D; citation_volume=59; citation_number=61; citation_pages=290-298; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The dependence of the Chou-Fasman parameters on amino acid side chain structure.; citation_author=M Charton; citation_author=B Charton; citation_journal_title=J Theor Biol; citation_volume=111; citation_number=62; citation_pages=447-450; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Amino acid side chain parameters for correlation studies in biology and pharmacology.; citation_author=JL Fauchère; citation_author=M Charton; citation_author=LB Kier; citation_author=A Verloop; citation_author=V Pliska; citation_journal_title=Int J Pept Protein Res; citation_volume=32; citation_number=63; citation_pages=269-278; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=A simplified representation of protein conformations for rapid simulation of protein folfing.; citation_author=M Levitt; citation_journal_title=J Mol Biol; citation_volume=104; citation_number=64; citation_pages=59-107; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=An analysis of non-bonded energy of proteins.; citation_author=M Oobatake; citation_author=T Ooi; citation_journal_title=J Theor Biol; citation_volume=67; citation_number=65; citation_pages=567-584; citation_date=1977; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices"/>
    <meta name="twitter:description" content="Position-specific scoring matrices (PSSMs) are useful for detecting weak homology in protein sequence analysis, and they are thought to contain some essential signatures of the protein families. In order to elucidate what kind of ingredients constitute such family-specific signatures, we apply singular value decomposition to a set of PSSMs and examine the properties of dominant right and left singular vectors. The first right singular vectors were correlated with various amino acid indices including relative mutability, amino acid composition in protein interior, hydropathy, or turn propensity, depending on proteins. A significant correlation between the first left singular vector and a measure of site conservation was observed. It is shown that the contribution of the first singular component to the PSSMs act to disfavor potentially but falsely functionally important residues at conserved sites. The second right singular vectors were highly correlated with hydrophobicity scales, and the corresponding left singular vectors with contact numbers of protein structures. It is suggested that sequence alignment with a PSSM is essentially equivalent to threading supplemented with functional information. In addition, singular vectors may be useful for analyzing and annotating the characteristics of conserved sites in protein families."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001963.g007"/>

  <meta property="og:title" content="Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8155'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5527'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8733&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001963">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001963" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001963&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Akira R. Kinjo
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:akinjo@protein.osaka-u.ac.jp">akinjo@protein.osaka-u.ac.jp</a></p>

                <p>Affiliation:
                  Institute for Protein Research, Osaka University, Suita, Osaka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Haruki Nakamura
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute for Protein Research, Osaka University, Suita, Osaka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001963</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001963-g001" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-t001" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-g002" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-g003" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-t002" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-g004" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-g005" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-t003" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-g006" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-t004" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-g007" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001963-t005" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001963">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001963" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Position-specific scoring matrices (PSSMs) are useful for detecting weak homology in protein sequence analysis, and they are thought to contain some essential signatures of the protein families. In order to elucidate what kind of ingredients constitute such family-specific signatures, we apply singular value decomposition to a set of PSSMs and examine the properties of dominant right and left singular vectors. The first right singular vectors were correlated with various amino acid indices including relative mutability, amino acid composition in protein interior, hydropathy, or turn propensity, depending on proteins. A significant correlation between the first left singular vector and a measure of site conservation was observed. It is shown that the contribution of the first singular component to the PSSMs act to disfavor potentially but falsely functionally important residues at conserved sites. The second right singular vectors were highly correlated with hydrophobicity scales, and the corresponding left singular vectors with contact numbers of protein structures. It is suggested that sequence alignment with a PSSM is essentially equivalent to threading supplemented with functional information. In addition, singular vectors may be useful for analyzing and annotating the characteristics of conserved sites in protein families.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kinjo AR, Nakamura H (2008) Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices. PLoS ONE 3(4):
          e1963.
            doi:10.1371/journal.pone.0001963</p><p><strong>Editor: </strong>David Jones, University College London, United Kingdom</p><p><strong>Received:</strong> January 17, 2008; <strong>Accepted:</strong> March 5, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Kinjo, Nakamura. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Supported by Japan Science and Technology Agency, which played no role in the study but financial support.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Protein sequence alignment using a position-specific scoring matrix (PSSM) or sequence profile <a href="#pone.0001963-Taylor1">[1]</a>, <a href="#pone.0001963-Gribskov1">[2]</a> is now a standard tool for sequence analysis<a href="#pone.0001963-Eddy1">[3]</a>, <a href="#pone.0001963-Altschul1">[4]</a>. Using a PSSM, it is often possible to detect very distantly related proteins which cannot be detected by the standard pairwise alignment based on a position-independent amino acid substitution matrix (AASM).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>An AASM is a 20×20 real (usually symmetric) matrix each element of which reflects the tendency of substitution between amino acid residues. There have been many kinds of AASMs developed to date among which the most popular ones include the PAM <a href="#pone.0001963-Dayhoff1">[5]</a> and the BLOSUM series <a href="#pone.0001963-Henikoff1">[6]</a>. General properties of AASMs are now well clarified<a href="#pone.0001963-Altschul2">[7]</a>, <a href="#pone.0001963-Tomii1">[8]</a>, <a href="#pone.0001963-Kinjo1">[9]</a>, <a href="#pone.0001963-Pokarowski1">[10]</a>. Tomii and Kanehisa found that the PAM matrices can be well approximated by the volume and hydrophobicity of amino acid residues<a href="#pone.0001963-Tomii1">[8]</a>. A similar result was obtained by Pokarowski et al.<a href="#pone.0001963-Pokarowski1">[10]</a>, but they also pointed out the importance of the coil preferences of amino acids residues. Using eigenvalue decomposition, Kinjo and Nishikawa<a href="#pone.0001963-Kinjo1">[9]</a> showed that the most dominant component of AASMs is the relative mutability<a href="#pone.0001963-Dayhoff1">[5]</a> for closely related homologs, but it changes to hydrophobicity below the sequence identity of 30%, and this transition of dominant modes was related to the so-called twilight zone of sequence comparison<a href="#pone.0001963-Doolittle1">[11]</a>, <a href="#pone.0001963-Rost1">[12]</a>. There are also AASMs specifically optimized to overcome the twilight zone <a href="#pone.0001963-Kann1">[13]</a>, <a href="#pone.0001963-Qian1">[14]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Detection of very distant homologs is often possible by using PSSM-based sequence alignment methods such as PSI-BLAST<a href="#pone.0001963-Altschul1">[4]</a> or hidden Markov models<a href="#pone.0001963-Eddy1">[3]</a>, <a href="#pone.0001963-Durbin1">[15]</a> because a PSSM is specific to a particular protein family so that some family-specific features can be exploited. In a PSSM, family-specific features are expressed as position-dependent substitution scores, and hence a PSSM is an <em>N</em>×20 matrix where <em>N</em> is the length of the protein or protein family it represents. Since PSSMs can be regarded as an extension of sequence motifs<a href="#pone.0001963-Durbin1">[15]</a>, family-specific features are, to the first approximation, a pattern of amino acid residues around functionally or structurally important sites expressed in a probabilistic manner. In order to further understand the mechanism by which the effectiveness of PSSMs is realized, however, it is necessary to elucidate more general characteristics of PSSMs that are shared across different protein families.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To delineate the general properties of PSSMs, we analyze them by using singular value decomposition (SVD). By applying SVD, a PSSM can be decomposed into 20 orthogonal components of varying importance. Each singular component consists of a singular value (a scalar), right singular vector (r-SV) and left singular vector (l-SV). A singular value represents the relative importance of the component whereas the corresponding r-SV (a 20-vector) represents a property of 20 amino acid types and the l-SV may be regarded as a one-dimensional (1D) numerical representation of the amino acid sequence that is “dual” to the property represented by the r-SV. Since r-SVs can be regarded as amino acid indices<a href="#pone.0001963-Kidera1">[16]</a>, <a href="#pone.0001963-Nakai1">[17]</a>, <a href="#pone.0001963-Tomii1">[8]</a>, we can infer their meaning by comparing them with the entries of the AAindex database<a href="#pone.0001963-Kawashima1">[18]</a> which compiles many amino acid indices published to date. This is a natural generalization of a previous work where AASMs were analyzed by using eigenvalue decomposition <a href="#pone.0001963-Kinjo1">[9]</a>. The present analysis revealed a tendency of PSSMs that is analogous to the AASMs for close homologs. That is, the first principal component disfavors any substitutions and potentially functionally important residues are more severely penalized, and the second component is highly correlated with sequence and structural properties related to hydrophobicity. These features are expected to contribute to the effectiveness of sequence alignment based on PSSMs.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Singular value decomposition of position-specific scoring matrix</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A position-specific scoring matrix (PSSM) is a real rectangular matrix of size <em>N</em>×20 where <em>N</em> is the length of the amino acid sequence of a protein (or protein family). We assume <em>N</em>&gt;20 although this condition is not strictly necessary. Each column of a PSSM corresponds to an amino acid type, whereas each row corresponds to a site in the amino acid sequence. Let <em>M</em> = (<em>M<sub>ij</sub></em>) be a PSSM. The element <em>M<sub>ij</sub></em> represents the score for the amino acid <em>j</em> at the site <em>i</em> (<a href="#pone-0001963-g001">Fig. 1A</a>). By applying singular value decomposition <a href="#pone.0001963-Horn1">[19]</a> (SVD), we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>U</em> = (<strong>u</strong><sub>1</sub>,…,<strong>u</strong><sub>20</sub>) and <em>V</em> = (<strong>v</strong><sub>1</sub>,…,<strong>v</strong><sub>20</sub>) are <em>N</em>×20 and 20×20 orthogonal matrices, respectively, that is, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e002&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e003&amp;representation=PNG" class="inline-graphic"></span> (δ<sub>αβ</sub> is Kronecker's delta). An example of SVD of a PSSM is given in <a href="#pone-0001963-g001">Fig. 1</a>. The 20-vectors <strong>v</strong><sub>α</sub>'s are called right singular vectors (r-SV, <a href="#pone-0001963-g001">Fig. 1C</a>). Since each element of a right singular vector numerically represents some property of an amino acid type, we can regard a right singular vector of a PSSM as an amino acid index <a href="#pone.0001963-Kidera1">[16]</a>, <a href="#pone.0001963-Nakai1">[17]</a>, <a href="#pone.0001963-Tomii1">[8]</a> (possibly specific to the parent PSSM). The <em>N</em>-vectors <strong>u</strong><sub>α</sub>'s are called left singular vectors (l-SV, <a href="#pone-0001963-g001">Fig. 1C</a>). Since each element of a left singular vector numerically represents some property of the corresponding site in the sequence, we can regard a left singular vector of a PSSM as a generalized 1D structure. Σ = <em>diag</em>(σ<sub>1</sub>,…,σ<sub>20</sub>) is a diagonal matrix whose elements are the singular values of the PSSM, sorted in the decreasing order (<a href="#pone-0001963-g001">Fig. 1B</a>). Singular values are always non-negative and their magnitudes represent relative importance of the corresponding singular components (i.e., the pair of right and left singular vectors).</p>
<div class="figure" id="pone-0001963-g001"><div class="img"><a name="pone-0001963-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Example of singular value decomposition<a href="#pone.0001963-Horn1">[19]</a> of a PSSM (c.f. Eq. 1).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A: The original PSSM (based on the PDB entry 3sdhA <a href="#pone.0001963-Royer1">[44]</a>); B: Singular values; C: Pairs of left singular vector (l-SV) u<sub>α</sub> of <em>N</em> dimensions and right singular vector (r-SV) v<sub>α</sub> of 20 dimensions (α = 1,…,20). The abscissa indicates residue number for the left singular vectors (l-SV), and amino acid type for the right singular vectors (r-SV). The ordinate shows the vector elements relative to zero (note that only the relative values, not absolute ones, are meaningful).</p>
<span>doi:10.1371/journal.pone.0001963.g001</span></div>

<h4>Data sets</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We analyze two sets of PSSMs. One is a representative set derived from the Protein Data Bank (PDB)<a href="#pone.0001963-Berman1">[20]</a> and the other is the Pfam database<a href="#pone.0001963-Finn1">[21]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The representative protein chains in the PDB were obtained from the PISCES server <a href="#pone.0001963-Wang1">[22]</a> with cutoffs of 25% sequence identity, 20% R-factor, 2.0Å resolution and sequence length ranging from 40 to 500. Only the structures determined by X-ray crystallography were used. Those proteins which were classified as all-α, all-β, α/β, α+β, multi-domain, or small proteins according to the SCOP (version 1.71) <a href="#pone.0001963-Murzin1">[23]</a> database were retained. As a result, we obtained 1096 protein chains. For each of these proteins, a PSSM was created by running PSI-BLAST against the UniRef100 protein sequence database (release 12.1) <a href="#pone.0001963-Suzek1">[24]</a> with e-value cutoff of 0.0005 and 3 iterations.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Although Pfam is a database of hidden Markov models of protein families<a href="#pone.0001963-Durbin1">[15]</a>, we can regard its entries as PSSMs by using only the scores for matching states. We extracted from Pfam release 22.0 (July 2007) those proteins whose sequence lengths were at least 40 residues, resulting in 8869 protein families.</p>


<h4>Searching AAindex</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As mentioned above, each right singular vector (r-SV) can be regarded as an amino acid index, a set of numerical values reflecting some property of amino acid residues. In order to clarify the meaning of each r-SV, we scanned the AAindex database <a href="#pone.0001963-Tomii1">[8]</a>, <a href="#pone.0001963-Kawashima1">[18]</a> (Release 9.1, August, 2006) which compiles many amino acid indices published to date. For a given α ( = 1, 2, …, 20), the amino acid index that showed the highest correlation to the α-th r-SV of each PSSM were identified. If the absolute value of the correlation coefficient between the index and the r-SV is greater than or equal to 0.6, then the index is counted as significant. Identified indices are sorted according to the number of times they are counted as significant. In <a href="#pone-0001963-t001">Table 1</a>, we summarize the descriptions of the AAindex entries that will be mentioned in the Results section.</p>
<div class="figure" id="pone-0001963-t001"><div class="img"><a name="pone-0001963-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>AAindex entries mentioned in the text.</span></strong></p><span>doi:10.1371/journal.pone.0001963.t001</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Overview</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In order to check to what extent a subset of singular components can explain the original PSSM, we calculated the accumulative contribution ratio of each PSSM. The accumulative contribution ratio up to <em>k</em>-th singular value is defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br></p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The averages of <em>S<sub>k</sub></em> for <em>k</em> = 1,…,20 are shown in <a href="#pone-0001963-g002">Fig. 2</a>. We observe that the first singular value contributes 17% of the total singular values in the PDB set, and 24% in the Pfam set. Thus, the contribution of the first singular component is relatively larger in the Pfam PSSMs than in the PSI-BLAST-generated PSSMs of PDB entries. This tendency may be related to the higher specificity of the Pfam hidden Markov models. 50% contributions are made by first 4 or 5 components in the PDB or Pfam sets, respectively, whereas 90% contributions are made by the first 15 components in the both sets. Compared to the case with AASMs where 50% and 90% contributions are made by first 3 and 10 singular values (or eigenvalues) <a href="#pone.0001963-Kinjo1">[9]</a>, the “compressibility” of PSSMs is lower in the sense that more components are needed to explain the same fraction (50% or 90%) of the total components. This is a reasonable result since each PSSM should contain some detailed information specific to the family to which the protein sequence belongs, whereas AASMs should contain more general information regarding the patterns of amino acid substitutions shared by many protein families.</p>
<div class="figure" id="pone-0001963-g002"><div class="img"><a name="pone-0001963-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Accumulative contribution ratio (<em>S<sub>k</sub></em>%, <em>k</em> = 1,…,20) averaged over the PDB and Pfam sets.</span></strong></p><span>doi:10.1371/journal.pone.0001963.g002</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>In order to glance at the overall characteristics of decomposed PSSMs, we constructed a partial matrix <em>M<sub>k</sub></em> for each PSSM by summing the first <em>k</em> components, that is,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>and calculated the fraction of positive elements (<em>M</em><sub>20</sub> is identical to the original PSSM). In both the PDB and Pfam sets, there are usually more negative elements than positive ones (<a href="#pone-0001963-g003">Fig. 3</a>). This is an expected behavior for log-odds matrices <a href="#pone.0001963-Altschul2">[7]</a>. However, this skewed distribution is greatly pronounced for the <em>M</em><sub>1</sub> matrices. In fact, most substitutions are disfavored by the first singular component of a PSSM. A typical example is shown in <a href="#pone-0001963-g001">Fig. 1 C</a> where the contribution of the first component (i.e., <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e006&amp;representation=PNG" class="inline-graphic"></span>) is purely negative. Compared to <em>M</em><sub>1</sub>, other partial matrices (<em>M<sub>k</sub></em> with <em>k</em>&gt;1) have more positive elements. This indicates that positive values in the final PSSM must originate from components other than the first one.</p>
<div class="figure" id="pone-0001963-g003"><div class="img"><a name="pone-0001963-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Fraction of positive elements in partial matrices <em>M<sub>k</sub></em> averaged over the PDB and Pfam sets.</span></strong></p><span>doi:10.1371/journal.pone.0001963.g003</span></div>

<h4>Characteristics of first singular components</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In order to interpret the physicochemical or biochemical meaning of the first r-SVs (<strong>v</strong><sub>1</sub> in Eq. 1), we scanned the AAindex database and identified amino acid indices that frequently show significant correlations (<a href="#pone-0001963-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001963-t002"><div class="img"><a name="pone-0001963-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Amino acid indices most correlated to the first right singular vectors [frequency (%) in the parentheses].</span></strong></p><span>doi:10.1371/journal.pone.0001963.t002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In the PDB representative set, the most frequently correlated index was the relative mutability<a href="#pone.0001963-Jones1">[25]</a> (AAindex: JOND920102) which is also the fifth most frequent index for the Pfam set. The relative mutabilities<a href="#pone.0001963-Dayhoff1">[5]</a> represent the tendency of amino acid residues to be mutated during molecular evolution, and are not highly correlated with any other indices <a href="#pone.0001963-Tomii1">[8]</a>. It is thus expected that some intrinsic characteristics of protein evolution is embedded in their values. The relative mutability is the most dominant component in the ordinary (position-independent) AASMs targeted at closely related proteins<a href="#pone.0001963-Kinjo1">[9]</a>. As in the case of AASMs, the first r-SVs are negatively correlated with the relative mutability (Recall that all the elements of the first r-SVs are of the same sign in most cases so that we can make them all positive without losing generality). An example is shown in <a href="#pone-0001963-g004">Fig. 4 A</a>. Thus, noting that the first singular components (i.e., partial matrix <em>M</em><sub>1</sub> in <a href="#pone-0001963-g003">Fig. 3</a>) are mostly negative, we can see that substitutions of those residues with low mutabilities are more severely penalized.</p>
<div class="figure" id="pone-0001963-g004"><div class="img"><a name="pone-0001963-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Examples of correlations between the first r-SV and amino acid indices.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>The abscissa of each panel indicates the value of elements in the first right singular vector of a PSSM where the signs are determined by making the value for cysteine positive. The ordinates are (A) the relative mutability <a href="#pone.0001963-Jones1">[25]</a>, (B) the interior composition of amino acids in intracellular proteins of mesophiles<a href="#pone.0001963-Fukuchi1">[26]</a>, (C) the hydropathy scale<a href="#pone.0001963-Kyte1">[28]</a>, and (D) the conformational parameter of beta-turn<a href="#pone.0001963-Beghin1">[27]</a>. The labels on the ordinates indicate the identifiers of the AAindex database (<a href="#pone-0001963-t001">Table 1</a>).</p>
<span>doi:10.1371/journal.pone.0001963.g004</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The interior composition of amino acids in intracellular proteins of mesophiles<a href="#pone.0001963-Fukuchi1">[26]</a> (FUKS010106) is another frequently correlated index, ranked second and sixth in the PDB and Pfam sets, respectively. As we can see in the example shown in <a href="#pone-0001963-g004">Fig. 4 B</a>, those residues that are less abundant in protein interior are more severely penalized. This seems to contradict our intuition that residues in the protein interior are more conservative than those on the protein surface. However, many functionally important residues exist on the surface (ligand binding sites and catalytic sites, etc.). Thus, these r-SVs should be regarded as representing potentially functionally important residues. Note, however, although these residues share some properties common to conserved residues, most of them are not actually important (otherwise they should not be penalized).</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Other frequently correlated indices shared by both PDB and Pfam sets are the conformational parameter of β-turn<a href="#pone.0001963-Beghin1">[27]</a> (BEGF750103) and the hydropathy index of Kyte and Doolittle<a href="#pone.0001963-Kyte1">[28]</a> (KYTJ820101). The most frequently correlated index for the Pfam set was “principal component I” of Sneath (SNEP660101) <a href="#pone.0001963-Sneath1">[29]</a>. The name of this index is rather cryptic, but it is weakly negatively correlated with turn or coil propensities (data not shown). These indices can be readily related to interior-surface propensities: β-turns, coils, and hydrophilic residues tend to be on the surface of a protein, and so on. The general trend is that substitutions of those residues that tend to be on the surface are more severely penalized (<a href="#pone-0001963-g004">Fig. 4C, D</a>). Again, this may be due to the fact that many (potentially) functionally important residues are on the protein surface.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>It is noted that no single index is overwhelmingly dominant in the first r-SVs so that different PSSMs are characterized by different properties. This is a reasonable result since each PSSM is specific to a particular protein family which is under the influence of specific evolutionary pressures and biological constraints. Nevertheless, relative mutability, hydrophobicity, and turn/coil propensity are the relatively more dominant characteristics of the first r-SVs.</p>
<a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>If the first r-SV of a PSSM represents a property of amino acid residues that is well-conserved, then the first l-SV is expected to represent the pattern or extent of conservation of that property along the amino acid sequence. One such measure is the information content (also referred to as Kullback-Leibler divergence or relative entropy <a href="#pone.0001963-Cover1">[30]</a>) which is a kind of distance of the distribution of amino acid residues at a given site of the sequence from the background distribution. The information content <em>D<sub>i</sub></em> of site <em>i</em> is defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <em>P<sub>i</sub></em>(<em>a</em>) is the frequency of amino acid type <em>a</em> at the site <em>i</em> and <em>Q</em>(<em>a</em>) is the background frequency of amino acid type <em>a</em>. In general, information content tends to be larger at more conserved sites. This information is available in the PSSMs created with PSI-BLAST. A significant correlation was found between the first l-SVs and information content of PSSMs of the PDB set with correlation coefficient of 0.543 on average with standard deviation of 0.218 (<em>P</em>&lt;10<sup>−17</sup>, assuming the average sequence length of 217 residues). The median of the correlation coefficient was 0.601 indicating that the correlation is even higher for many of the PSSMs. When calculating the correlation coefficient, we converted the signs of the elements of the l-SV so that most elements become positive. Thus, a positive correlation implies that a site with a large value of the first l-SV element usually has high information content, indicating that substitutions at those sites with more information content are more severely penalized. An example of such correlation is shown in <a href="#pone-0001963-g005">Fig. 5</a>. l-SVs other than the first one did not show high correlations with information content (data not shown). For those PSSMs whose first r-SVs are highly correlated with JOND920102 (110 entries), FUKS010106 (74), BEGF750103 (56), KYTJ820101 (49), and SNEP660101 (24) (<a href="#pone-0001963-t002">Table 2</a>), the average correlation coefficients were 0.646, 0.703, 0.654, 0.536, and 0.593, respectively. Thus, the high correlation between the first l-SV and information content is not limited to specific PSSMs whose first r-SVs are correlated to some particular indices.</p>
<div class="figure" id="pone-0001963-g005"><div class="img"><a name="pone-0001963-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Example of the first left singular vector and information content.</span></strong></p><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1"></a><p>Shown are the first l-SV and information content of the PDB entry 1e6uA<a href="#pone.0001963-Rosano1">[60]</a>. The values of the l-SV elements are scaled by 20 times to match the information content. The correlation coefficient is 0.76.</p>
<span>doi:10.1371/journal.pone.0001963.g005</span></div>

<h4>Characteristics of second singular components</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>In the same manner as the first r-SVs, we searched for indices that are highly correlated with the second r-SVs of the PSSMs (<a href="#pone-0001963-t003">Table 3</a>). In this case, relative partition energies derived by the Bethe approximation of Miyazawa and Jernigan <a href="#pone.0001963-Miyazawa1">[31]</a> (AAindex: MIYS990101) is the most correlated index: 33% of the PDB set and 54% of the Pfam set. This index is a kind of hydrophobicity scale. Furthermore, other frequently correlated indices, such as interactivity scales of Bastolla et al. <a href="#pone.0001963-Bastolla1">[32]</a> (BASU050101, BASU050103), polarity <a href="#pone.0001963-Grantham1">[33]</a> (GRAR740102), optimal matching hydrophobicity <a href="#pone.0001963-Sweet1">[34]</a> (SWER830101), and all other indices in <a href="#pone-0001963-t003">Table 3</a>, are all related to hydrophobicity scales. The ten most frequently correlated indices alone match 85% and 94% of the second r-SVs of the PSSMs in the PDB and Pfam sets, respectively. Therefore, while the first r-SVs are of diverse characteristics, the second r-SVs are almost exclusively determined by hydrophobic properties. It is interesting to note that the hydropathy index of Kyte and Doolittle <a href="#pone.0001963-Kyte1">[28]</a> which was found to be correlated to some first r-SVs (<a href="#pone-0001963-t002">Table 2</a>) was not found to be the the index most correlated with the second r-SVs in most cases. Although the hydropathy index is highly correlated with the partition energy of Miyazawa and Jernigan (correlation coefficient of −0.84), there seems to be a meaningful difference between them.</p>
<div class="figure" id="pone-0001963-t003"><div class="img"><a name="pone-0001963-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Amino acid indices most correlated to the second right singular vectors [frequency (%) in the parentheses].</span></strong></p><span>doi:10.1371/journal.pone.0001963.t003</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The correlation between the second r-SVs and hydrophobicity scales is striking. Therefore, it is expected that the second left singular vectors (l-SVs) are correlated with some structural property that is dual to the hydrophobicity. One such structural property is the contact number <a href="#pone.0001963-Nishikawa1">[35]</a>, <a href="#pone.0001963-Nishikawa2">[36]</a>, <a href="#pone.0001963-Kinjo2">[37]</a>, which is the number of residues in contact with a given residue in a native protein structure. We calculated contact numbers of the PDB set (based on the definition by Kinjo et al. <a href="#pone.0001963-Kinjo2">[37]</a>) and their correlations with the second l-SVs. The average correlation coefficient was 0.511 (standard deviation 0.113) which is highly significant (<em>P</em>&lt;10<sup>−15</sup>) for the average protein length of 217 residues in the PDB set. <a href="#pone-0001963-g006">Fig. 6</a> shows an example of the highly correlated second l-SV and contact numbers.</p>
<div class="figure" id="pone-0001963-g006"><div class="img"><a name="pone-0001963-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Example of the second left singular vector and contact numbers.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Shown are the second l-SV and contact numbers of the PDB entry 1l2hA<a href="#pone.0001963-Rudolph1">[61]</a>. The values of the l-SV elements are shifted and scaled to match the contact numbers. The correlation coefficient is 0.71.</p>
<span>doi:10.1371/journal.pone.0001963.g006</span></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Recall that the elements of the first r-SVs were of the same sign in most cases (<a href="#pone-0001963-g003">Fig. 3</a>). Thus, by the orthogonality of singular vectors, the elements of the second r-SVs should necessarily contain values of both signs in most cases. The same argument also applies to l-SVs. Therefore, the contribution from the second component of a PSSM, namely <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e008&amp;representation=PNG" class="inline-graphic"></span>, contains both positive and negative elements corresponding to favorable and unfavorable substitutions, respectively. Now let <strong>w</strong> represent the relative partition energy of Miyazawa and Jernigan <a href="#pone.0001963-Miyazawa1">[31]</a> (MIYS990101), and <strong>n</strong> represent the contact number vector of a protein standardized by subtracting the average value from each element. We calculated the correlation coefficient between the two matrices <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e009&amp;representation=PNG" class="inline-graphic"></span> and <strong>wn</strong><em><sup>T</sup></em> for those 361 proteins whose second r-SVs are most correlated with <strong>w</strong>. We obtained the average correlation of −0.45 which is highly significant (<em>P</em>&lt;10<sup>−220</sup>) taking into account the average number of elements (217×20). Since hydrophilic and hydrophobic residues have positive and negative partition energies, respectively, the negative correlation means that hydrophobic residues with high contact numbers (buried) and hydrophilic residues with low contact numbers (exposed) are more favored compared to hydrophobic residues with low contact numbers and hydrophilic residues with high contact numbers. Thus, within the framework developed here, we can consider the second singular component represents the structural stability of the protein.</p>


<h4>Characteristics of third and other singular components</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The indices that are most frequently correlated with the third r-SVs of the PSSMs are listed in <a href="#pone-0001963-t004">Table 4</a>. In general, the third r-SVs are correlated with those indices related to the volume or bulkiness of amino acid residues such as CHAM830106, SNEP660103, LEVM760102, LEVM760105 and OOBM770105 (see <a href="#pone-0001963-t001">Table 1</a> for descriptions). Another kind of index common to the PDB and Pfam sets is the α-helix propensity derived from designed sequences<a href="#pone.0001963-Koehl1">[38]</a> (KOEP990101) which is actually correlated with coil propensity (data not shown). This index was also found to be frequently correlated with the fourth r-SVs. A structural quantity that may be associated with bulkiness of amino acid residues is the volume of the “territory” of residues as defined by the Voronoi tessellation<a href="#pone.0001963-Richards1">[39]</a>, <a href="#pone.0001963-Barber1">[40]</a>. When we compared the Voronoi volumes calculated from protein structures with the third l-SV, we observed a significant but weak correlation of 0.345 (<em>P</em>&lt;0.0003). (The Voronoi volume of a residue was calculated by summing the Voronoi volumes of the atoms that belong to the residue; only half of the residues with smaller volumes are used for comparison as surface residues with [sometimes infinitely] large volumes are not meaningful.) If we limit the comparison to those proteins whose third r-SVs are most correlated with CHAM830106 (214 entries), SNEP660103 (201) or LEVM760102 (137), the correlations were 0.366, 0.251, or 0.479, respectively. Therefore, the correlation of the third l-SV to the Voronoi volume is significant, but not as consistent as those of the first and second l-SVs to information content and contact numbers, respectively.</p>
<div class="figure" id="pone-0001963-t004"><div class="img"><a name="pone-0001963-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Amino acid indices most correlated to the third right singular vectors [frequency (%) in the parentheses].</span></strong></p><span>doi:10.1371/journal.pone.0001963.t004</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The propensity of the fourth and fifth r-SVs are not so clearly characterized as the first three r-SVs, but helix (KOEP990101) and helix cap propensities <a href="#pone.0001963-Aurora1">[41]</a> as well as some bulkiness parameters are relatively highly correlated with the fourth r-SVs, while the net charge (KLEP840101) <a href="#pone.0001963-Klein1">[42]</a> and α-NH chemical shifts (BUNA790101) <a href="#pone.0001963-Bundi1">[43]</a> were the indices most correlated with the fifth r-SVs of more than 30% of the PSSMs in both the PDB and Pfam sets.</p>


<h4>Example: Conserved sites in the globin family</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>To illustrate the points made above, we now examine the PSI-BLAST PSSM of a globin (PDB 3sdhA<a href="#pone.0001963-Royer1">[44]</a>, hemoglobin I from <em>Scapharca inaequivalvis</em>). The globin family is one of the most extensively studied protein families<a href="#pone.0001963-Bashford1">[45]</a>, <a href="#pone.0001963-Lecomte1">[46]</a>. Ota et al.<a href="#pone.0001963-Ota1">[47]</a> examined in detail seven highly conserved residues in globins identified by Bashford et al.<a href="#pone.0001963-Bashford1">[45]</a> (namely, the sites B10, C2, CD1, CD4, E7, F4, and F8, according to the numbering scheme of Bashford et al.<a href="#pone.0001963-Bashford1">[45]</a>), and succeeded in separating structurally important sites from functionally important sites. <a href="#pone-0001963-g007">Fig. 7</a> shows the contributions of various components to the seven highly conserved sites studied in Ota et al. <a href="#pone.0001963-Ota1">[47]</a>. The most correlated amino acid indices for the first 5 r-SVs are BEGF750103, MIYS990101, FAUJ880106, KOEP990101, and AURR980119 (see <a href="#pone-0001963-t001">Table 1</a> for their descriptions).</p>
<div class="figure" id="pone-0001963-g007"><div class="img"><a name="pone-0001963-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Decomposed PSSM scores of conserved sites in the globin family.</span></strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>The labels B10, C2, CD1, CD4, E7, F4, and F8 on the top of panels are site identifiers of the globin family defined by Bashford et al.<a href="#pone.0001963-Bashford1">[45]</a> (in the parentheses is the most conserved residue at each site). Rows of the partial matrices <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.e010&amp;representation=PNG" class="inline-graphic"></span> (α = 1, …, 5) as well as <em>M</em>−<em>m</em><sub>1</sub> are plotted for the selected sites. The PSSM is based on the PDB entry 3sdhA<a href="#pone.0001963-Royer1">[44]</a>.</p>
<span>doi:10.1371/journal.pone.0001963.g007</span></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>The contributions to those conserved residues that were identified as functionally important by Ota et al. <a href="#pone.0001963-Ota1">[47]</a> (namely, E7 and F8) are mainly from the third and fifth components which are correlated to bulkiness and helix capping propensity, respectively. Other conserved residues were identified as structurally important, and their scores consist mainly of the second singular component which is related to the hydrophobicity, except for the proline residue at the C2 site to which the helix capping propensity is the main contributor. These observations are consistent with the analysis of Ota et al. <a href="#pone.0001963-Ota1">[47]</a> which was based on three-dimensional profiles<a href="#pone.0001963-Bowie1">[48]</a>, <a href="#pone.0001963-Ota2">[49]</a>.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>The contributions of the first singular component to these sites are all negative for all residues (<a href="#pone-0001963-g007">Fig. 7</a>) which is consistent with the general argument provided above. We now consider the meaning of the negative contribution of the first singular component. For simplicity, we first consider the site F8 where the histidine residue is perfectly conserved. At this site, only the score of histidine should be positive and all others be negative. Positive contributions to the score of histidine is made from the third, fifth and other singular components so that the total contributions from second to twentieth components are as large as 14. Without the contribution from the first singular component, the scores of some other residues such as asparagine and tyrosine are also positive although not as large as that of histidine. Thus, we can see that the large positive score of a conserved residue (histidine) is made by coherent contributions from multiple singular components whereas the scores of residues that are not conserved may be positive but small due to incoherent contributions. Nevertheless, positive scores of non-conserved residues degrades the specificity of a PSSM. Thus, they should be somehow made negative. Similar arguments apply to other conserved sites except that different residues may be conserved at different sites for different reasons. The score of potentially but falsely functionally important residues at all conserved sites can be made negative at once by simply subtracting the scores according to the common properties of amino acid residues at these sites, and this is the role of the first singular component. In the present example, the common property happened to be related to the β-turn propensity.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>Note that the shapes of the corresponding components (e.g., <em>m</em><sub>1</sub> for different sites) are similar among different sites. This is because they are all scalar multiple of the same r-SVs. What distinguishes different sites is the relative contributions due to the l-SVs.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Kinjo and Nishikawa<a href="#pone.0001963-Kinjo1">[9]</a> analyzed a set of amino acid substitution matrices constructed from multiple alignments of protein families of varying percent sequence identities (%ID). It was found that, at high %IDs (&gt;35%), the first and second most dominant components were correlated with relative mutability and hydrophobicity, respectively, while at low %IDs (&lt;30%), the order was opposite (hydrophobicity first, and then the relative mutability). It was suggested that the dominance of the relative mutability over hydrophobicity patterns is the prerequisite for reliable detection of homologs. In the case of PSSMs, the characteristics of the first singular component may vary depending on the protein (family). Nevertheless, the first singular components seem to represent some functional constraints which disfavor any substitutions, and the second (and third) singular components are predominantly determined by such structural requirements as hydrophobicity (and packing). Although both functional and structural constraints are important for distant homolog detection, the dominance of the former over the latter may be more influential for the high specificity of sequence alignment methods based on PSSMs. Noting again that the Pfam PSSMs have larger first singular values (<a href="#pone-0001963-g002">Fig. 2</a>) and their first components contain more negative elements (<a href="#pone-0001963-g003">Fig. 3</a>) compared to PSI-BLAST-generated PSSMs of the PDB set, this view of the first singular component is consistent with a general observation that Pfam PSSMs exhibit, on average, higher specificity than those generated by PSI-BLAST.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>As pointed out by Tomii and Kanehisa <a href="#pone.0001963-Tomii1">[8]</a>, side-chain volume and hydrophobicity are the main ingredients of AASMs. In addition to these two properties, Pokarowski et al.<a href="#pone.0001963-Pokarowski1">[10]</a> also noted the importance of the coil propensity. Wrabl and Grishin<a href="#pone.0001963-Wrabl1">[50]</a> also found similar preferences in the study of properties extracted from multiple sequence alignments. These properties are also found to be the main ingredients of PSSMs in the present study. Some of the first r-SVs showed significant correlation to indices related to coil propensity such as BEGF750103 and SNEP660101 (<a href="#pone-0001963-t002">Table 2</a>); hydrophobicity is predominant in the second r-SVs; and side-chain volumes often show high correlation with the third r-SVs.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Bastolla et al.<a href="#pone.0001963-Bastolla1">[32]</a> have studied the correlation between the “interactivity” scale of amino acid residues and the principal eigenvectors of the native contact maps<a href="#pone.0001963-Porto1">[51]</a>. Their interactivity scale is a kind of hydrophobicity scale, obtained by eigenvalue decomposition of a contact potential and subsequent optimizations. The principal eigenvector of a contact map is known to contain almost sufficient information for recovering the native structure itself<a href="#pone.0001963-Porto1">[51]</a>, and is highly correlated with contact number vector <a href="#pone.0001963-Kinjo3">[52]</a>. Bastolla et al.<a href="#pone.0001963-Bastolla1">[32]</a> showed that the interactivity scales aligned along the amino acid sequence of a protein, then averaged over homologs, were significantly correlated with the principal eigenvector with the average correlation coefficient of 0.47. Note that the interactivity scales of Bastolla et al. are found among those indices that are most correlated with the second r-SVs in <a href="#pone-0001963-t003">Table 3</a> (BASU050101 and BASU050103), and that the second l-SVs are correlated with contact number vectors. Thus, the present result is not only consistent with that of Bastolla et al. <a href="#pone.0001963-Bastolla1">[32]</a>, but also demonstrates that some structural information is already embedded in a PSSM, which also explains why contact numbers can be predicted at high accuracy by using PSSMs<a href="#pone.0001963-Kinjo2">[37]</a>, <a href="#pone.0001963-Yuan1">[53]</a>, <a href="#pone.0001963-Kinjo4">[54]</a>, <a href="#pone.0001963-Ishida1">[55]</a>, <a href="#pone.0001963-Kinjo5">[56]</a>. The present finding may be useful for deriving optimized contact potentials<a href="#pone.0001963-Knjo1">[57]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>To summarize, we analyzed PSSMs by decomposing them into singular components (<a href="#pone-0001963-t005">Table 5</a>). The characteristics of the first right singular vectors was found to vary depending on protein families, but the corresponding left singular vectors showed high correlation with information content. The contributions of the first singular components to the original PSSMs are usually negative so that the substitutions of potentially but falsely functionally important residues at conserved sites are more severely penalized. The second right singular vectors were almost always related to hydrophobicity of amino acid residues, and the left singular vectors are significantly correlated with contact number vectors, thus demonstrating that the structural information is directly embedded in the PSSMs. Other structural information seem to be also included in the PSSMs, although not as significantly as hydrophobicity and contact numbers. Therefore, sequence alignment using PSSMs may be regarded as threading <a href="#pone.0001963-Bowie2">[58]</a>, <a href="#pone.0001963-Bowie1">[48]</a>, <a href="#pone.0001963-Jones2">[59]</a> supplemented with some functional information. Based on the present analysis, it may be possible to define <em>a priori</em> measure of the quality of PSSMs which may lead to a rational strategy for constructing more effective PSSMs by mixing various functionally/structurally relevant contributions with appropriate singular values.</p>
<div class="figure" id="pone-0001963-t005"><div class="img"><a name="pone-0001963-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001963" data-uri="info:doi/10.1371/journal.pone.0001963.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001963.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001963.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001963.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001963.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Summary of characteristics of first three singular vectors of PSSMs.</span></strong></p><span>doi:10.1371/journal.pone.0001963.t005</span></div></div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Daron M. Standley for critically reading the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AK. Performed the experiments: AK. Analyzed the data: AK. Contributed reagents/materials/analysis tools: AK. Wrote the paper: AK HN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001963-Taylor1" id="pone.0001963-Taylor1"></a>Taylor WR (1986) Identification of protein sequence homology by consensus template alignment. J Mol Biol  188: 233–258.  <ul class="find" data-citedArticleID="1088986" data-doi="10.1016/0022-2836(86)90308-6"><li><a href="http://dx.doi.org/10.1016/0022-2836(86)90308-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+protein+sequence+homology+by+consensus+template+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+protein+sequence+homology+by+consensus+template+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001963-Gribskov1" id="pone.0001963-Gribskov1"></a>Gribskov M, McLachlan AD, Eisenberg D (1987) Profile Analysis: Detection of Distantly Related Proteins. Proc Natl Acad Sci USA  84: 4355–4358.  <ul class="find" data-citedArticleID="1088908" data-doi="10.1073/pnas.84.13.4355"><li><a href="http://dx.doi.org/10.1073/pnas.84.13.4355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profile+Analysis%3A+Detection+of+Distantly+Related+Proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profile+Analysis%3A+Detection+of+Distantly+Related+Proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001963-Eddy1" id="pone.0001963-Eddy1"></a>Eddy SR (1996) Hidden Markov Models. Curr Opin Struct Biol  6: 361–365.  <ul class="find" data-citedArticleID="1088898"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hidden+Markov+Models.&amp;auth=&amp;atitle=Hidden+Markov+Models." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hidden+Markov+Models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hidden+Markov+Models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001963-Altschul1" id="pone.0001963-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped Blast and PSI-Blast: A new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1088866" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+Blast+and+PSI-Blast%3A+A+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+Blast+and+PSI-Blast%3A+A+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001963-Dayhoff1" id="pone.0001963-Dayhoff1"></a>Dayhoff MO, Schwartz RM, Orcutt BC (1978)  A Model of Evolutionary Change in Proteins. In: Dayhoff MO, editor. Atlas of Protein Sequence and Structure, vol. 5 (Suppl.3). Washington, D.C.: National Biomedical Research Foundation. pp. 345–352.  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001963-Henikoff1" id="pone.0001963-Henikoff1"></a>Henikoff S, Henikoff JG (1992) Amino Acid Substitution Matrices from Protein Blocks. Proc Natl Acad Sci USA  89: 10915–10919.  <ul class="find" data-citedArticleID="1088910" data-doi="10.1073/pnas.89.22.10915"><li><a href="http://dx.doi.org/10.1073/pnas.89.22.10915" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+Acid+Substitution+Matrices+from+Protein+Blocks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+Acid+Substitution+Matrices+from+Protein+Blocks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001963-Altschul2" id="pone.0001963-Altschul2"></a>Altschul SF (1991) Amino Acid Substitution Matrices from an Information Theoretic Perspective. J Mol Biol  219: 555–565.  <ul class="find" data-citedArticleID="1088868" data-doi="10.1016/0022-2836(91)90193-a"><li><a href="http://dx.doi.org/10.1016/0022-2836(91)90193-a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+Acid+Substitution+Matrices+from+an+Information+Theoretic+Perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+Acid+Substitution+Matrices+from+an+Information+Theoretic+Perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001963-Tomii1" id="pone.0001963-Tomii1"></a>Tomii K, Kanehisa M (1996) Analysis of Amino Acid Indices and Mutation Matrices for Sequence Comparison and Structure Prediction of Proteins. Protein Eng  9: 27–36.  <ul class="find" data-citedArticleID="1088988" data-doi="10.1093/protein/9.1.27"><li><a href="http://dx.doi.org/10.1093/protein/9.1.27" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Amino+Acid+Indices+and+Mutation+Matrices+for+Sequence+Comparison+and+Structure+Prediction+of+Proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Amino+Acid+Indices+and+Mutation+Matrices+for+Sequence+Comparison+and+Structure+Prediction+of+Proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001963-Kinjo1" id="pone.0001963-Kinjo1"></a>Kinjo AR, Nishikawa K (2004) Eigenvalue analysis of amino acid substitution matrices reveals a sharp transition of the mode of sequence conservation in proteins. Bioinformatics  20: 2504–2508.  <ul class="find" data-citedArticleID="1088926" data-doi="10.1093/bioinformatics/bth297"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eigenvalue+analysis+of+amino+acid+substitution+matrices+reveals+a+sharp+transition+of+the+mode+of+sequence+conservation+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eigenvalue+analysis+of+amino+acid+substitution+matrices+reveals+a+sharp+transition+of+the+mode+of+sequence+conservation+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001963-Pokarowski1" id="pone.0001963-Pokarowski1"></a>Pokarowski P, Kloczkowski A, Nowakowski S, Pokarowska M, Jernigan RL, et al.  (2007) Ideal amino acid exchange forms for approximating substitution matrices. Proteins  69: 379–393.  <ul class="find" data-citedArticleID="1088964" data-doi="10.1002/prot.21509"><li><a href="http://dx.doi.org/10.1002/prot.21509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ideal+amino+acid+exchange+forms+for+approximating+substitution+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ideal+amino+acid+exchange+forms+for+approximating+substitution+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001963-Doolittle1" id="pone.0001963-Doolittle1"></a>Doolittle RF (1986) Of URFs and ORFs: A Primer on How to Analyze Derived Amino Acid Sequences. Mill Valley, CA: University Science Books.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001963-Rost1" id="pone.0001963-Rost1"></a>Rost B (1999) Twilight Zone of Protein Sequence Alignments. Protein Eng  12: 85–94.  <ul class="find" data-citedArticleID="1088974" data-doi="10.1093/protein/12.2.85"><li><a href="http://dx.doi.org/10.1093/protein/12.2.85" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Twilight+Zone+of+Protein+Sequence+Alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Twilight+Zone+of+Protein+Sequence+Alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001963-Kann1" id="pone.0001963-Kann1"></a>Kann M, Qian B, Goldstein RA (2000) Optimization of a New Score Function for the Detection of Remote Homologs. Proteins  41: 498–503.  <ul class="find" data-citedArticleID="1088920" data-doi="10.1002/1097-0134(20001201)41:4<498::aid-prot70&gt;3.0.co;2-3"><li><a href="http://dx.doi.org/10.1002/1097-0134(20001201)41:4<498::aid-prot70&gt;3.0.co;2-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimization+of+a+New+Score+Function+for+the+Detection+of+Remote+Homologs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimization+of+a+New+Score+Function+for+the+Detection+of+Remote+Homologs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001963-Qian1" id="pone.0001963-Qian1"></a>Qian B, Goldstein RA (2002) Optimization of a New Score Function for the Generation of Accurate Alignments. Proteins  48: 605–610.  <ul class="find" data-citedArticleID="1088968" data-doi="10.1002/prot.10132"><li><a href="http://dx.doi.org/10.1002/prot.10132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimization+of+a+New+Score+Function+for+the+Generation+of+Accurate+Alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimization+of+a+New+Score+Function+for+the+Generation+of+Accurate+Alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001963-Durbin1" id="pone.0001963-Durbin1"></a>Durbin R, Eddy R, Krogh A, Mitchison G (1999) Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids. Cambridge, U. K.: Cambridge Univ. Press.   <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0001963-Kidera1" id="pone.0001963-Kidera1"></a>Kidera A, Konishi Y, Oka M, Ooi T, Scheraga HA (1985) Statistical Analysis of the Physical Properties of the 20 Naturally Occurring Amino Acids. J Protein Chem  4: 23–55.  <ul class="find" data-citedArticleID="1088924" data-doi="10.1007/bf01025492"><li><a href="http://dx.doi.org/10.1007/bf01025492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+Analysis+of+the+Physical+Properties+of+the+20+Naturally+Occurring+Amino+Acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+Analysis+of+the+Physical+Properties+of+the+20+Naturally+Occurring+Amino+Acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001963-Nakai1" id="pone.0001963-Nakai1"></a>Nakai K, Kidera A, Kanehisa M (1988) Cluster analysis of amino acid indices for prediction of protein structure and function. Protein Eng  2: 93–100.  <ul class="find" data-citedArticleID="1088952" data-doi="10.1093/protein/2.2.93"><li><a href="http://dx.doi.org/10.1093/protein/2.2.93" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+of+amino+acid+indices+for+prediction+of+protein+structure+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+of+amino+acid+indices+for+prediction+of+protein+structure+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001963-Kawashima1" id="pone.0001963-Kawashima1"></a>Kawashima S, Ogata H, Kanehisa M (1999) AAindex: amino acid index database. Nucleic Acids Res  27: 368–369.  <ul class="find" data-citedArticleID="1088922" data-doi="10.1093/nar/27.1.368"><li><a href="http://dx.doi.org/10.1093/nar/27.1.368" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AAindex%3A+amino+acid+index+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AAindex%3A+amino+acid+index+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001963-Horn1" id="pone.0001963-Horn1"></a>Horn RA, Johnson CR (1985) Matrix analysis. Cambridge, U. K.: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0001963-Berman1" id="pone.0001963-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al.  (2000) The Protein Data Bank. Nucleic Acids Res  28: 235–242.  <ul class="find" data-citedArticleID="1088880" data-doi="10.1093/nar/28.1.235"><li><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001963-Finn1" id="pone.0001963-Finn1"></a>Finn RD, Mistry J, Schuster-Böckler B, Griffiths-Jones S, Hollich V, et al.  (2006) Pfam: clans, web tools and services. Nucleic Acids Res  34: D247–D251.  <ul class="find" data-citedArticleID="1088902" data-doi="10.1093/nar/gkj149"><li><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pfam%3A+clans%2C+web+tools+and+services." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pfam%3A+clans%2C+web+tools+and+services.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001963-Wang1" id="pone.0001963-Wang1"></a>Wang G, Dunbrack RL (2003) PISCES: a protein sequence culling server. Bioinformatics  19: 1589–1591.  <ul class="find" data-citedArticleID="1088990" data-doi="10.1093/bioinformatics/btg224"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PISCES%3A+a+protein+sequence+culling+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PISCES%3A+a+protein+sequence+culling+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001963-Murzin1" id="pone.0001963-Murzin1"></a>Murzin AG, Brenner SE, Hubbard T, Chothia C (1995) SCOP: A structural classification of proteins database for the investigation of sequences and structures. J Mol Biol  247: 536–540.  <ul class="find" data-citedArticleID="1088950" data-doi="10.1016/s0022-2836(05)80134-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80134-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCOP%3A+A+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCOP%3A+A+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001963-Suzek1" id="pone.0001963-Suzek1"></a>Suzek BE, Huang H, McGarvey P, Mazumder R, Wu CH (2007) UniRef: comprehensive and non-redundant UniProt reference clusters. Bioinformatics  23: 1282–1288.  <ul class="find" data-citedArticleID="1088982" data-doi="10.1093/bioinformatics/btm098"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UniRef%3A+comprehensive+and+non-redundant+UniProt+reference+clusters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UniRef%3A+comprehensive+and+non-redundant+UniProt+reference+clusters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001963-Jones1" id="pone.0001963-Jones1"></a>Jones DT, Taylor WR, Thornton JM (1992) The Rapid Generation of Mutation Data Matrices from Protein Sequences. Comput Appl Biosci  8: 275–282.  <ul class="find" data-citedArticleID="1088916" data-doi="10.1093/bioinformatics/8.3.275"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.3.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Rapid+Generation+of+Mutation+Data+Matrices+from+Protein+Sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Rapid+Generation+of+Mutation+Data+Matrices+from+Protein+Sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001963-Fukuchi1" id="pone.0001963-Fukuchi1"></a>Fukuchi S, Nishikawa K (2001) Protein surface amino acid compositions distinctively differ between thermophilic and mesophilic bacteria. J Mol Biol  309: 835–843.  <ul class="find" data-citedArticleID="1088904" data-doi="10.1006/jmbi.2001.4718"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.4718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+surface+amino+acid+compositions+distinctively+differ+between+thermophilic+and+mesophilic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+surface+amino+acid+compositions+distinctively+differ+between+thermophilic+and+mesophilic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001963-Beghin1" id="pone.0001963-Beghin1"></a>Beghin F, Dirkx J (1975) Une méthode statistique simple de prédiction des conformations protéiques. Arch Int Physiol Biochim  83: 167–168.  <ul class="find" data-citedArticleID="1088878"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Une+m%E9thode+statistique+simple+de+pr%E9diction+des+conformations+prot%E9iques.&amp;auth=&amp;atitle=Une+m%E9thode+statistique+simple+de+pr%E9diction+des+conformations+prot%E9iques." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Une+m%C3%A9thode+statistique+simple+de+pr%C3%A9diction+des+conformations+prot%C3%A9iques." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Une+m%C3%A9thode+statistique+simple+de+pr%C3%A9diction+des+conformations+prot%C3%A9iques.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001963-Kyte1" id="pone.0001963-Kyte1"></a>Kyte J, Doolittle RF (1982) A simple method for displaying the hydropathic character of a protein. J Mol Biol  157: 105–132.  <ul class="find" data-citedArticleID="1088942" data-doi="10.1016/0022-2836(82)90515-0"><li><a href="http://dx.doi.org/10.1016/0022-2836(82)90515-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+method+for+displaying+the+hydropathic+character+of+a+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+method+for+displaying+the+hydropathic+character+of+a+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001963-Sneath1" id="pone.0001963-Sneath1"></a>Sneath PHA (1966) Relations between chemical structure and biological activity in peptides. J Theor Biol  12: 157–195.  <ul class="find" data-citedArticleID="1088980" data-doi="10.1016/0022-5193(66)90112-3"><li><a href="http://dx.doi.org/10.1016/0022-5193(66)90112-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relations+between+chemical+structure+and+biological+activity+in+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relations+between+chemical+structure+and+biological+activity+in+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001963-Cover1" id="pone.0001963-Cover1"></a>Cover TM, Thomas JA (2006) Elements of Information Theory. 2nd ed. Hoboken, NJ, U. S. A.: John Wiley and Sons.   <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0001963-Miyazawa1" id="pone.0001963-Miyazawa1"></a>Miyazawa S, Jernigan RL (1999) Self-consistent estimation of inter-residue protein contact energies based on an equilibrium mixture approximation of residues. Proteins  34: 49–68.  <ul class="find" data-citedArticleID="1088948" data-doi="10.1002/(sici)1097-0134(19990101)34:1<49::aid-prot5&gt;3.3.co;2-c"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(19990101)34:1<49::aid-prot5&gt;3.3.co;2-c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-consistent+estimation+of+inter-residue+protein+contact+energies+based+on+an+equilibrium+mixture+approximation+of+residues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-consistent+estimation+of+inter-residue+protein+contact+energies+based+on+an+equilibrium+mixture+approximation+of+residues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001963-Bastolla1" id="pone.0001963-Bastolla1"></a>Bastolla U, Porto M, Roman HE, Vendruscolo M (2005) Principal eigenvector of contact matrices and hydrophobicity profiles in proteins. Proteins  58: 22–30.  <ul class="find" data-citedArticleID="1088876" data-doi="10.1002/prot.20240"><li><a href="http://dx.doi.org/10.1002/prot.20240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principal+eigenvector+of+contact+matrices+and+hydrophobicity+profiles+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principal+eigenvector+of+contact+matrices+and+hydrophobicity+profiles+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001963-Grantham1" id="pone.0001963-Grantham1"></a>Grantham R (1974) Amino acid difference formula to help explain protein evolution. Science  185: 862–864.  <ul class="find" data-citedArticleID="1088906" data-doi="10.1126/science.185.4154.862"><li><a href="http://dx.doi.org/10.1126/science.185.4154.862" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+difference+formula+to+help+explain+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+difference+formula+to+help+explain+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001963-Sweet1" id="pone.0001963-Sweet1"></a>Sweet RM, Eisenberg D (1983) Correlation of sequence hydrophobicities measures similarity in three-dimensional protein structure. J Mol Biol  171: 479–488.  <ul class="find" data-citedArticleID="1088984" data-doi="10.1016/0022-2836(83)90041-4"><li><a href="http://dx.doi.org/10.1016/0022-2836(83)90041-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+of+sequence+hydrophobicities+measures+similarity+in+three-dimensional+protein+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+of+sequence+hydrophobicities+measures+similarity+in+three-dimensional+protein+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001963-Nishikawa1" id="pone.0001963-Nishikawa1"></a>Nishikawa K, Ooi T (1980) Prediction of the surface-interior diagram of globular proteins by an empirical method. Int J Peptide Protein Res  16: 19–32.  <ul class="find" data-citedArticleID="1088954" data-doi="10.1111/j.1399-3011.1980.tb02931.x"><li><a href="http://dx.doi.org/10.1111/j.1399-3011.1980.tb02931.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+the+surface-interior+diagram+of+globular+proteins+by+an+empirical+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+the+surface-interior+diagram+of+globular+proteins+by+an+empirical+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001963-Nishikawa2" id="pone.0001963-Nishikawa2"></a>Nishikawa K, Ooi T (1986) Radial locations of amino acid residues in a globular protein: Correlation with the sequence. J Biochem  100: 1043–1047.  <ul class="find" data-citedArticleID="1088956"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Radial+locations+of+amino+acid+residues+in+a+globular+protein%3A+Correlation+with+the+sequence.&amp;auth=&amp;atitle=Radial+locations+of+amino+acid+residues+in+a+globular+protein%3A+Correlation+with+the+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Radial+locations+of+amino+acid+residues+in+a+globular+protein%3A+Correlation+with+the+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Radial+locations+of+amino+acid+residues+in+a+globular+protein%3A+Correlation+with+the+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001963-Kinjo2" id="pone.0001963-Kinjo2"></a>Kinjo AR, Horimoto K, Nishikawa K (2005) Predicting absolute contact numbers of native protein structure from amino acid sequence. Proteins  58: 158–165.  doi:10.1002/prot.20300.  <ul class="find" data-citedArticleID="1088928" data-doi="10.1002/prot.20300"><li><a href="http://dx.doi.org/10.1002/prot.20300" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+absolute+contact+numbers+of+native+protein+structure+from+amino+acid+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+absolute+contact+numbers+of+native+protein+structure+from+amino+acid+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001963-Koehl1" id="pone.0001963-Koehl1"></a>Koehl P, Levitt M (1999) Structure-based conformational preferences of amino acids. Proc Natl Acad Sci USA  96: 12524–12529.  <ul class="find" data-citedArticleID="1088940" data-doi="10.1073/pnas.96.22.12524"><li><a href="http://dx.doi.org/10.1073/pnas.96.22.12524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-based+conformational+preferences+of+amino+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-based+conformational+preferences+of+amino+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001963-Richards1" id="pone.0001963-Richards1"></a>Richards FM (1977) Areas, volumes, packing, and protein structure. Ann Rev Biophys Bioeng  6: 151–176.  <ul class="find" data-citedArticleID="1088970" data-doi="10.1146/annurev.bb.06.060177.001055"><li><a href="http://dx.doi.org/10.1146/annurev.bb.06.060177.001055" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Areas%2C+volumes%2C+packing%2C+and+protein+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Areas%2C+volumes%2C+packing%2C+and+protein+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001963-Barber1" id="pone.0001963-Barber1"></a>Barber C, Dobkin D, Huhdanpaa H (1996) The Quickhull algorithm for convex hulls. ACM Trans on Math Software  22: 469–483.  <a href="http://www.qhull.org">http://www.qhull.org</a>.  <ul class="find" data-citedArticleID="1088872" data-doi="10.1145/235815.235821"><li><a href="http://dx.doi.org/10.1145/235815.235821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Quickhull+algorithm+for+convex+hulls." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Quickhull+algorithm+for+convex+hulls.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001963-Aurora1" id="pone.0001963-Aurora1"></a>Aurora R, Rose GD (1998) Helix capping. Protein Sci  7: 21–38.  <ul class="find" data-citedArticleID="1088870" data-doi="10.1002/pro.5560070103"><li><a href="http://dx.doi.org/10.1002/pro.5560070103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Helix+capping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Helix+capping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001963-Klein1" id="pone.0001963-Klein1"></a>Klein P, Kanehisa M, DeLisi C (1984) Prediction of protein function from sequence properties: Discriminant analysis of a data base. Biochim Biophys Acta  787: 221–226.  <ul class="find" data-citedArticleID="1088936" data-doi="10.1016/0167-4838(84)90312-1"><li><a href="http://dx.doi.org/10.1016/0167-4838(84)90312-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+protein+function+from+sequence+properties%3A+Discriminant+analysis+of+a+data+base." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+protein+function+from+sequence+properties%3A+Discriminant+analysis+of+a+data+base.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001963-Bundi1" id="pone.0001963-Bundi1"></a>Bundi A, Wuthrich K (1979) 1H-nmr parameters of the common amino acid residues measured in aqueous solutions of the linear tetrapeptides H-Gly-Gly-X-L-Ala-OH. Biopolymers  18: 285–297.  <ul class="find" data-citedArticleID="1088886" data-doi="10.1002/bip.1979.360180206"><li><a href="http://dx.doi.org/10.1002/bip.1979.360180206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=1H-nmr+parameters+of+the+common+amino+acid+residues+measured+in+aqueous+solutions+of+the+linear+tetrapeptides+H-Gly-Gly-X-L-Ala-OH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%221H-nmr+parameters+of+the+common+amino+acid+residues+measured+in+aqueous+solutions+of+the+linear+tetrapeptides+H-Gly-Gly-X-L-Ala-OH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001963-Royer1" id="pone.0001963-Royer1"></a>Royer WE Jr (1994) High Resolution Crystallographic Analysis of a Co-operative Dimeric Hemoglobin. J Mol Biol  235: 657–681.  <ul class="find" data-citedArticleID="1088976" data-doi="10.1006/jmbi.1994.1019"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.1019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+Resolution+Crystallographic+Analysis+of+a+Co-operative+Dimeric+Hemoglobin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+Resolution+Crystallographic+Analysis+of+a+Co-operative+Dimeric+Hemoglobin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001963-Bashford1" id="pone.0001963-Bashford1"></a>Bashford D, Chothia C, Lesk AM (1987) Determinants of a protein fold. J Mol Biol  196: 199–216.  <ul class="find" data-citedArticleID="1088874" data-doi="10.1016/0022-2836(87)90521-3"><li><a href="http://dx.doi.org/10.1016/0022-2836(87)90521-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determinants+of+a+protein+fold." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determinants+of+a+protein+fold.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001963-Lecomte1" id="pone.0001963-Lecomte1"></a>Lecomte JT, Vuletich DA, Lesk AM (2005) Structural divergence and distant relationships in proteins: evolution of the globins. Curr Opin Struct Biol  15: 290–301.  <ul class="find" data-citedArticleID="1088944" data-doi="10.1016/j.sbi.2005.05.008"><li><a href="http://dx.doi.org/10.1016/j.sbi.2005.05.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+divergence+and+distant+relationships+in+proteins%3A+evolution+of+the+globins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+divergence+and+distant+relationships+in+proteins%3A+evolution+of+the+globins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001963-Ota1" id="pone.0001963-Ota1"></a>Ota M, Isogai Y, Nishikawa K (1997) Structural requirement of highly-conserved residues in globins. FEBS Lett  415: 129–133.  <ul class="find" data-citedArticleID="1088960" data-doi="10.1016/s0014-5793(97)01110-1"><li><a href="http://dx.doi.org/10.1016/s0014-5793(97)01110-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+requirement+of+highly-conserved+residues+in+globins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+requirement+of+highly-conserved+residues+in+globins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001963-Bowie1" id="pone.0001963-Bowie1"></a>Bowie JU, Lüthy R, Eisenberg D (1991) A method to identify protein sequences that fold into a known three-dimensional structure. Science  253: 164–170.  <ul class="find" data-citedArticleID="1088882" data-doi="10.1126/science.1853201"><li><a href="http://dx.doi.org/10.1126/science.1853201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+to+identify+protein+sequences+that+fold+into+a+known+three-dimensional+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+to+identify+protein+sequences+that+fold+into+a+known+three-dimensional+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001963-Ota2" id="pone.0001963-Ota2"></a>Ota M, Nishikawa K (1997) Assessment of pseudo-energy potentials by the best-five test: A new use of the three-dimensional profiles of proteins. Protein Eng  10: 339–351.  <ul class="find" data-citedArticleID="1088962" data-doi="10.1093/protein/10.4.339"><li><a href="http://dx.doi.org/10.1093/protein/10.4.339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+pseudo-energy+potentials+by+the+best-five+test%3A+A+new+use+of+the+three-dimensional+profiles+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+pseudo-energy+potentials+by+the+best-five+test%3A+A+new+use+of+the+three-dimensional+profiles+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001963-Wrabl1" id="pone.0001963-Wrabl1"></a>Wrabl JO, Grishin NV (2005) Grouping of amino acid types and extraction of amino acid properties from multiple sequence alignments using variance maximization. Proteins  61: 523–534.  <ul class="find" data-citedArticleID="1088992" data-doi="10.1002/prot.20648"><li><a href="http://dx.doi.org/10.1002/prot.20648" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Grouping+of+amino+acid+types+and+extraction+of+amino+acid+properties+from+multiple+sequence+alignments+using+variance+maximization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Grouping+of+amino+acid+types+and+extraction+of+amino+acid+properties+from+multiple+sequence+alignments+using+variance+maximization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001963-Porto1" id="pone.0001963-Porto1"></a>Porto M, Bastolla U, Roman HE, Vendruscolo M (2004) Reconstruction of protein structures from a vectorial representation. Phys Rev Lett  92: 218101.  <ul class="find" data-citedArticleID="1088966" data-doi="10.1103/physrevlett.92.218101"><li><a href="http://dx.doi.org/10.1103/physrevlett.92.218101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstruction+of+protein+structures+from+a+vectorial+representation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstruction+of+protein+structures+from+a+vectorial+representation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001963-Kinjo3" id="pone.0001963-Kinjo3"></a>Kinjo AR, Nishikawa K (2005) Recoverable one-dimensional encoding of three-dimensional protein structures. Bioinformatics  21: 2167–2170.  doi:10.1093/bioinformatics/bti330.  <ul class="find" data-citedArticleID="1088930" data-doi="10.1093/bioinformatics/bti330"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti330" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recoverable+one-dimensional+encoding+of+three-dimensional+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recoverable+one-dimensional+encoding+of+three-dimensional+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001963-Yuan1" id="pone.0001963-Yuan1"></a>Yuan Z (2005) Better prediction of protein contact number using a support vector regression analysis of amino acid sequence. BMC Bioinformatics  6: 248.  <ul class="find" data-citedArticleID="1088994"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Better+prediction+of+protein+contact+number+using+a+support+vector+regression+analysis+of+amino+acid+sequence.&amp;auth=&amp;atitle=Better+prediction+of+protein+contact+number+using+a+support+vector+regression+analysis+of+amino+acid+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Better+prediction+of+protein+contact+number+using+a+support+vector+regression+analysis+of+amino+acid+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Better+prediction+of+protein+contact+number+using+a+support+vector+regression+analysis+of+amino+acid+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001963-Kinjo4" id="pone.0001963-Kinjo4"></a>Kinjo AR, Nishikawa K (2005) Predicting secondary structures, contact numbers, and residue-wise contact orders of native protein structure from amino acid sequence using critical random networks. BIOPHYSICS  1: 67–74.  doi:10.2142/biophysics.1.67.  <ul class="find" data-citedArticleID="1088932" data-doi="10.2142/biophysics.1.67"><li><a href="http://dx.doi.org/10.2142/biophysics.1.67" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+secondary+structures%2C+contact+numbers%2C+and+residue-wise+contact+orders+of+native+protein+structure+from+amino+acid+sequence+using+critical+random+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+secondary+structures%2C+contact+numbers%2C+and+residue-wise+contact+orders+of+native+protein+structure+from+amino+acid+sequence+using+critical+random+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001963-Ishida1" id="pone.0001963-Ishida1"></a>Ishida T, Nakamura S, Shimizu K (2006) Potential for assessing quality of protein structure based on contact number prediction. Proteins  64: 940–947.  <ul class="find" data-citedArticleID="1088914" data-doi="10.1002/prot.21047"><li><a href="http://dx.doi.org/10.1002/prot.21047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potential+for+assessing+quality+of+protein+structure+based+on+contact+number+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potential+for+assessing+quality+of+protein+structure+based+on+contact+number+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001963-Kinjo5" id="pone.0001963-Kinjo5"></a>Kinjo AR, Nishikawa K (2006) CRNPRED: Highly accurate prediction of one-dimensional protein structures by large-scale critical random networks. BMC Bioinformatics  7: 401.  <ul class="find" data-citedArticleID="1088934"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CRNPRED%3A+Highly+accurate+prediction+of+one-dimensional+protein+structures+by+large-scale+critical+random+networks.&amp;auth=&amp;atitle=CRNPRED%3A+Highly+accurate+prediction+of+one-dimensional+protein+structures+by+large-scale+critical+random+networks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CRNPRED%3A+Highly+accurate+prediction+of+one-dimensional+protein+structures+by+large-scale+critical+random+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CRNPRED%3A+Highly+accurate+prediction+of+one-dimensional+protein+structures+by+large-scale+critical+random+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001963-Knjo1" id="pone.0001963-Knjo1"></a>Knjo AR, Miyazawa S (2008) On the optimal contact potential of proteins. Chem Phys Lett  451: 132–135.  <ul class="find" data-citedArticleID="1088938" data-doi="10.1016/j.cplett.2007.12.005"><li><a href="http://dx.doi.org/10.1016/j.cplett.2007.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+optimal+contact+potential+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+optimal+contact+potential+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001963-Bowie2" id="pone.0001963-Bowie2"></a>Bowie JU, Clarke ND, Pabo CO, Sauer RT (1990) Identification of protein folds: matching hydrophobicity patterns of sequence sets with solvent accessibility patterns of known structures. Proteins  7: 257–264.  <ul class="find" data-citedArticleID="1088884" data-doi="10.1002/prot.340070307"><li><a href="http://dx.doi.org/10.1002/prot.340070307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+protein+folds%3A+matching+hydrophobicity+patterns+of+sequence+sets+with+solvent+accessibility+patterns+of+known+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+protein+folds%3A+matching+hydrophobicity+patterns+of+sequence+sets+with+solvent+accessibility+patterns+of+known+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001963-Jones2" id="pone.0001963-Jones2"></a>Jones DT, Taylor WR, Thornton JM (1992) A new approach to protein fold recognition. Nature  358: 86–89.  <ul class="find" data-citedArticleID="1088918" data-doi="10.1038/358086a0"><li><a href="http://dx.doi.org/10.1038/358086a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+approach+to+protein+fold+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+approach+to+protein+fold+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001963-Rosano1" id="pone.0001963-Rosano1"></a>Rosano C, Bisso A, Izzo G, Tonetti M, Sturla L, et al.  (2000) Probing the catalytic mechanism of GDP-4-keto-6-deoxy-d-mannose Epimerase/Reductase by kinetic and crystallographic characterization of site-specific mutants. J Mol Biol  303: 77–91.  <ul class="find" data-citedArticleID="1088972" data-doi="10.1006/jmbi.2000.4106"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+the+catalytic+mechanism+of+GDP-4-keto-6-deoxy-d-mannose+Epimerase%2FReductase+by+kinetic+and+crystallographic+characterization+of+site-specific+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+the+catalytic+mechanism+of+GDP-4-keto-6-deoxy-d-mannose+Epimerase%2FReductase+by+kinetic+and+crystallographic+characterization+of+site-specific+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001963-Rudolph1" id="pone.0001963-Rudolph1"></a>Rudolph MG, Kelker MS, Schneider TR, Yeates TO, Oseroff V, et al.  (2003) Use of multiple anomalous dispersion to phase highly merohedrally twinned crystals of interleukin-1beta. Acta Crystallogr D  59: 290–298.  <ul class="find" data-citedArticleID="1088978" data-doi="10.1107/s0907444902021704"><li><a href="http://dx.doi.org/10.1107/s0907444902021704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+multiple+anomalous+dispersion+to+phase+highly+merohedrally+twinned+crystals+of+interleukin-1beta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+multiple+anomalous+dispersion+to+phase+highly+merohedrally+twinned+crystals+of+interleukin-1beta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001963-Charton1" id="pone.0001963-Charton1"></a>Charton M, Charton B (1983) The dependence of the Chou-Fasman parameters on amino acid side chain structure. J Theor Biol  111: 447–450.  <ul class="find" data-citedArticleID="1088888" data-doi="10.1016/0022-5193(83)90265-5"><li><a href="http://dx.doi.org/10.1016/0022-5193(83)90265-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dependence+of+the+Chou-Fasman+parameters+on+amino+acid+side+chain+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dependence+of+the+Chou-Fasman+parameters+on+amino+acid+side+chain+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001963-Fauchre1" id="pone.0001963-Fauchre1"></a>Fauchère JL, Charton M, Kier LB, Verloop A, Pliska V (1988) Amino acid side chain parameters for correlation studies in biology and pharmacology. Int J Pept Protein Res  32: 269–278.  <ul class="find" data-citedArticleID="1088900" data-doi="10.1111/j.1399-3011.1988.tb01261.x"><li><a href="http://dx.doi.org/10.1111/j.1399-3011.1988.tb01261.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+side+chain+parameters+for+correlation+studies+in+biology+and+pharmacology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+side+chain+parameters+for+correlation+studies+in+biology+and+pharmacology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001963-Levitt1" id="pone.0001963-Levitt1"></a>Levitt M (1976) A simplified representation of protein conformations for rapid simulation of protein folfing. J Mol Biol  104: 59–107.  <ul class="find" data-citedArticleID="1088946" data-doi="10.1016/0022-2836(76)90004-8"><li><a href="http://dx.doi.org/10.1016/0022-2836(76)90004-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simplified+representation+of+protein+conformations+for+rapid+simulation+of+protein+folfing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simplified+representation+of+protein+conformations+for+rapid+simulation+of+protein+folfing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001963-Oobatake1" id="pone.0001963-Oobatake1"></a>Oobatake M, Ooi T (1977) An analysis of non-bonded energy of proteins. J Theor Biol  67: 567–584.  <ul class="find" data-citedArticleID="1088958" data-doi="10.1016/0022-5193(77)90058-3"><li><a href="http://dx.doi.org/10.1016/0022-5193(77)90058-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+analysis+of+non-bonded+energy+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+analysis+of+non-bonded+energy+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.XML" value="115692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.PDF" value="501351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e001.PNG" value="10182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e001.TIF" value="26708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e002.PNG" value="6630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e002.TIF" value="22832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e003.PNG" value="6520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e003.TIF" value="22716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g001.PNG_L" value="640153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g001.PNG_M" value="257221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g001.PNG_S" value="19116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g001.TIF" value="1905394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g001.PNG_I" value="141122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t001.PNG_L" value="260336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t001.PNG_M" value="146445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t001.PNG_S" value="12899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t001.TIF" value="1147894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t001.PNG_I" value="44498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e004.PNG" value="8930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e004.TIF" value="25416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g002.PNG_L" value="34778"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g002.PNG_M" value="82220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g002.PNG_S" value="10233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g002.TIF" value="161410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g002.PNG_I" value="30079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e005.PNG" value="8756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e005.TIF" value="25164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e006.PNG" value="5921"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e006.TIF" value="22072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g003.PNG_L" value="38254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g003.PNG_M" value="100118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g003.PNG_S" value="12237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g003.TIF" value="184156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g003.PNG_I" value="36914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t002.PNG_L" value="92771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t002.PNG_M" value="132561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t002.PNG_S" value="15800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t002.TIF" value="454796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t002.PNG_I" value="42850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g004.PNG_L" value="91139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g004.PNG_M" value="73416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g004.PNG_S" value="9140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g004.TIF" value="458390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g004.PNG_I" value="25491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e007.PNG" value="10532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e007.TIF" value="27492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g005.PNG_L" value="75502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g005.PNG_M" value="126236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g005.PNG_S" value="13042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g005.TIF" value="357844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g005.PNG_I" value="46525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t003.PNG_L" value="78938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t003.PNG_M" value="109758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t003.PNG_S" value="14351"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t003.TIF" value="394392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t003.PNG_I" value="35439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g006.PNG_L" value="84233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g006.PNG_M" value="149277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g006.PNG_S" value="14160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g006.TIF" value="357002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g006.PNG_I" value="55460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e008.PNG" value="6099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e008.TIF" value="22284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e009.PNG" value="5690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e009.TIF" value="21756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t004.PNG_L" value="80596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t004.PNG_M" value="113244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t004.PNG_S" value="14438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t004.TIF" value="406100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t004.PNG_I" value="36033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e010.PNG" value="6473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.e010.TIF" value="22660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g007.PNG_L" value="987949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g007.PNG_M" value="175939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g007.PNG_S" value="15676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g007.TIF" value="1708854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.g007.PNG_I" value="112097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t005.PNG_L" value="43164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t005.PNG_M" value="63607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t005.PNG_S" value="10753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t005.TIF" value="205756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001963.t005.PNG_I" value="19810"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001963&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001963" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001963&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001963&volume=&issue=&title=Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices&author_name=Akira%20R.%20Kinjo%2C%20Haruki%20Nakamura&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001963" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963&amp;t=Nature%20of%20Protein%20Family%20Signatures%3A%20Insights%20from%20Singular%20Value%20Analysis%20of%20Position-Specific%20Scoring%20Matrices" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963&title=Nature%20of%20Protein%20Family%20Signatures%3A%20Insights%20from%20Singular%20Value%20Analysis%20of%20Position-Specific%20Scoring%20Matrices&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963&amp;title=Nature%20of%20Protein%20Family%20Signatures%3A%20Insights%20from%20Singular%20Value%20Analysis%20of%20Position-Specific%20Scoring%20Matrices" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001963&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Nature of Protein Family Signatures: Insights from Singular Value Analysis of Position-Specific Scoring Matrices';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001963';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Nature%20of%20Protein%20Family%20Signatures%3A%20Insights%20from%20Singular%20Value%20Analysis%20of%20Position-Specific%20Scoring%20Matrices http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001963" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001963" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+sequence+analysis%22" title="Search for articles in the subject area:'Amino acid sequence analysis'"><div class="flagText">Amino acid sequence analysis</div></a>
              <div data-categoryid="20173" data-articleid="26750"
                   data-categoryname="Amino acid sequence analysis"
                   class="flagImage" title="Flag 'Amino acid sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+substitution%22" title="Search for articles in the subject area:'Amino acid substitution'"><div class="flagText">Amino acid substitution</div></a>
              <div data-categoryid="32897" data-articleid="26750"
                   data-categoryname="Amino acid substitution"
                   class="flagImage" title="Flag 'Amino acid substitution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Globins%22" title="Search for articles in the subject area:'Globins'"><div class="flagText">Globins</div></a>
              <div data-categoryid="24873" data-articleid="26750"
                   data-categoryname="Globins"
                   class="flagImage" title="Flag 'Globins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+sequencing%22" title="Search for articles in the subject area:'Protein sequencing'"><div class="flagText">Protein sequencing</div></a>
              <div data-categoryid="47503" data-articleid="26750"
                   data-categoryname="Protein sequencing"
                   class="flagImage" title="Flag 'Protein sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomic+databases%22" title="Search for articles in the subject area:'Proteomic databases'"><div class="flagText">Proteomic databases</div></a>
              <div data-categoryid="43483" data-articleid="26750"
                   data-categoryname="Proteomic databases"
                   class="flagImage" title="Flag 'Proteomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26750"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26750"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26750"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1383'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7706'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6169&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4583">Referee Comments: Referee 1 (Martin Madera)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
