

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000335"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000335" />

    <meta name="citation_title" content="Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?"/>
    <meta itemprop="name" content="Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?"/>

      <meta name="citation_author" content="Stefan Wuchty"/>
            <meta name="citation_author_institution" content="Northwestern Institute on Complexity, Northwestern University, Evanston, Illinois, United States of America"/>

    <meta name="citation_date" content="2007/3/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000335.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e335"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A protein interaction network of the malaria parasite plasmodium falciparum.; citation_author=D LaCount; citation_author=M Vignali; citation_author=R Chettier; citation_author=A Phansalkar; citation_author=R Bell; citation_journal_title=Nature; citation_volume=438; citation_number=1; citation_pages=103-107; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lethality and centrality in protein networks.; citation_author=H Jeong; citation_author=S Mason; citation_author=A-L Barabasi; citation_author=Z Oltvai; citation_journal_title=Nature; citation_volume=411; citation_number=2; citation_pages=41-42; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Network biology: Understanding the cell's functional organization.; citation_author=A-L Barabasi; citation_author=Z Oltvai; citation_journal_title=Nat Rev Gen; citation_volume=4; citation_number=3; citation_pages=101-113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary rate in the protein interaction network.; citation_author=H Fraser; citation_author=A Hirsh; citation_author=L Steinmetz; citation_author=C Scharfe; citation_author=M Feldman; citation_journal_title=Science; citation_volume=296; citation_number=4; citation_pages=750-752; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolutionary conservation of motif constituents within the yeast protein interaction network.; citation_author=S Wuchty; citation_author=Z Oltvai; citation_author=A-L Barabasi; citation_journal_title=Nat Genet; citation_volume=35; citation_number=5; citation_pages=176-179; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Topology and evolution in yeast interaction networks.; citation_author=S Wuchty; citation_journal_title=Genome Res; citation_volume=14; citation_number=6; citation_pages=1310-1314; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The plasmodium protein network diverges from those of other eukaryotes.; citation_author=S Suthram; citation_author=T Sittler; citation_author=T Ideker; citation_journal_title=Nature; citation_volume=438; citation_number=7; citation_pages=108-112; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for dynamically organized modularity in the yeast protein-protein interaction network.; citation_author=J Han; citation_author=N Bertin; citation_author=T Hao; citation_author=DS Goldberg; citation_author=G Berriz; citation_journal_title=Nature; citation_volume=430; citation_number=8; citation_pages=88-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detecting rich-club ordering in complex networks.; citation_author=V Colizza; citation_author=A Flamini; citation_author=M Serano; citation_author=A Vespignani; citation_journal_title=Nat Physics; citation_volume=2; citation_number=9; citation_pages=110-115; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Specificity and stability in topology of protein networks.; citation_author=S Maslov; citation_author=K Sneppen; citation_journal_title=Science; citation_volume=296; citation_number=10; citation_pages=910-913; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions.; citation_author=I Xenarios; citation_author=L Salwinski; citation_author=X Duan; citation_author=P Higney; citation_author=SM Kim; citation_journal_title=Nucl Acids Res; citation_volume=30; citation_number=11; citation_pages=303-305; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A protein interaction map of drosophila melanogaster.; citation_author=L Giot; citation_author=J Bader; citation_author=C Brouwer; citation_author=A Chaudhuri; citation_author=B Kuang; citation_journal_title=Science; citation_volume=302; citation_number=12; citation_pages=1727-1736; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons.; citation_author=M Remm; citation_author=C Storm; citation_author=E Sonnhammer; citation_journal_title=J Mol Biol; citation_volume=314; citation_number=13; citation_pages=1041-1052; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Plasmodium biology: Genomic gleanings.; citation_author=L Aravind; citation_author=L Iyer; citation_author=T Wellems; citation_author=L Miller; citation_journal_title=Cell; citation_volume=115; citation_number=14; citation_pages=771-785; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Targeting malaria virulence and remodeling proteins to the host erythrocyte.; citation_author=M Marti; citation_author=R Good; citation_author=M Rug; citation_author=E Knuepfer; citation_author=A Cowman; citation_journal_title=Science; citation_volume=306; citation_number=15; citation_pages=1930-1934; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A host-targeting signal in virulence proteins reveals a secretome in malarial infection.; citation_author=N Hiller; citation_author=S Bhattacharjee; citation_author=C van Ooij; citation_author=K Liolios; citation_author=T Harrison; citation_journal_title=Science; citation_volume=306; citation_number=16; citation_pages=1934-1937; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lineage-specific expansion of proteins exported to erythrocytes in malaria parasites.; citation_author=T Sargeant; citation_author=M Marti; citation_author=E Caler; citation_author=J Carlton; citation_author=K Simpson; citation_journal_title=Genome Biol; citation_volume=7; citation_number=17; citation_pages=R12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cytoscape: A software environment for integrated models of biomolecular interaction networks.; citation_author=P Shannon; citation_author=A Markiel; citation_author=O Ozier; citation_author=N Baliga; citation_author=J Wang; citation_journal_title=Genome Res; citation_volume=13; citation_number=18; citation_pages=2498-2504; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The transcriptome of the intraerythrocytic developmental cycle of plasmodium falciparum.; citation_author=Z Bozdech; citation_author=M Llinas; citation_author=B Pulliam; citation_author=E Wong; citation_author=J Zhu; citation_journal_title=PLoS Biology; citation_volume=1; citation_number=19; citation_pages=1-16; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Applied systems biology and malaria.; citation_author=E Winzeler; citation_journal_title=Nat Rev Micro; citation_volume=4; citation_number=20; citation_pages=145-151; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global analysis of transcript and protein levels across the plasmodium falciparum life cycle.; citation_author=KLJ Johnson; citation_author=L Florens; citation_author=Y Zhou; citation_author=A Santrosyan; citation_author=M Grainger; citation_journal_title=Genome Res; citation_volume=14; citation_number=21; citation_pages=2308-2318; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Plasmodb: the plasmodium genome resource. a database integrating experimental and computational data.; citation_author=A Bahl; citation_author=B Brunk; citation_author=J Crabtree; citation_author=M Fraunholz; citation_author=B Gajria; citation_journal_title=Nucl Acids Res; citation_volume=31; citation_number=22; citation_pages=212-215; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Effect of sampling on topology predictions of protein-protein interaction networks.; citation_author=J Han; citation_author=D Dupuy; citation_author=N Bertin; citation_author=M Cusick; citation_author=M Vidal; citation_journal_title=Nat Biotech; citation_volume=23; citation_number=23; citation_pages=839-844; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Subnets of scale-free networks are not scale-free: Sampling properties of networks.; citation_author=M Stumpf; citation_author=C Wiuf; citation_author=R May; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=24; citation_pages=4221-4224; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparative assessment of large-scale data sets of protein-protein interactions.; citation_author=C Von Mering; citation_author=R Krause; citation_author=B Snel; citation_author=M Cornell; citation_author=S Oliver; citation_journal_title=Nature; citation_volume=417; citation_number=25; citation_pages=399-403; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Stable evolutionary signal in a protein interaction network of S. cerevisiae.; citation_author=S Wuchty; citation_author=A-L Barabasi; citation_author=M Ferdig; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=26; citation_pages=8; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?"/>
    <meta name="twitter:description" content="Recent advances have provided a first experimental protein interaction map of the human malaria parasite P. falciparum, which appears to be remotely related to interactomes of other eukaryotes. Here, we present a comparative topological analysis of this experimentally determined web with a network of conserved interactions between proteins in S. cerevisiae, C. elegans and D. melanogaster that have an ortholog in Plasmodium. Focusing on experimental interactions, we find a significant presence of a &ldquo;rich-club,&rdquo; a topological characteristic that features an &ldquo;oligarchy&rdquo; of highly connected proteins being intertwined with one another. In complete contrast, the network of interologs and particularly the web of evolutionary-conserved interactions in P. falciparum lack this feature. This observation prompts the question of whether this result points to a topological signature of the parasite's biology, since experimentally obtained interactions widely cover parasite-specific functions. Significantly, hub proteins that appear in such an oligarchy revolve around invasion functions, shaping an island of parasite-specific activities in a sea of evolutionary inherited interactions. This presence of a biologically unprecedented network feature in the human malaria parasite might be an artifact of the quality and the methods to obtain interaction data in this organism. Yet, the observation that rich-club proteins have distinctive and statistically significant functions that revolve around parasite-specific activities point to a topological signature of a parasitic life style."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000335.g003"/>

  <meta property="og:title" content="Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5982'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5885'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2860&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000335">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000335" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000335&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Rich-Club Phenomenon in the Interactome of <i>P. falciparum</i>—Artifact or Signature of a Parasitic Life Style?
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stefan Wuchty
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:s-wuchty@northwestern.edu">s-wuchty@northwestern.edu</a></p>

                <p>Affiliation:
                  Northwestern Institute on Complexity, Northwestern University, Evanston, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000335</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000335-g001" data-doi="info:doi/10.1371/journal.pone.0000335" data-uri="info:doi/10.1371/journal.pone.0000335.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000335-g002" data-doi="info:doi/10.1371/journal.pone.0000335" data-uri="info:doi/10.1371/journal.pone.0000335.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000335-g003" data-doi="info:doi/10.1371/journal.pone.0000335" data-uri="info:doi/10.1371/journal.pone.0000335.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000335">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000335" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Recent advances have provided a first experimental protein interaction map of the human malaria parasite <em>P. falciparum</em>, which appears to be remotely related to interactomes of other eukaryotes. Here, we present a comparative topological analysis of this experimentally determined web with a network of conserved interactions between proteins in <em>S. cerevisiae, C. elegans</em> and <em>D. melanogaster</em> that have an ortholog in <em>Plasmodium</em>. Focusing on experimental interactions, we find a significant presence of a “rich-club,” a topological characteristic that features an “oligarchy” of highly connected proteins being intertwined with one another. In complete contrast, the network of interologs and particularly the web of evolutionary-conserved interactions in <em>P. falciparum</em> lack this feature. This observation prompts the question of whether this result points to a topological signature of the parasite's biology, since experimentally obtained interactions widely cover parasite-specific functions. Significantly, hub proteins that appear in such an oligarchy revolve around invasion functions, shaping an island of parasite-specific activities in a sea of evolutionary inherited interactions. This presence of a biologically unprecedented network feature in the human malaria parasite might be an artifact of the quality and the methods to obtain interaction data in this organism. Yet, the observation that rich-club proteins have distinctive and statistically significant functions that revolve around parasite-specific activities point to a topological signature of a parasitic life style.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wuchty S (2007) Rich-Club Phenomenon in the Interactome of <em>P. falciparum</em>—Artifact or Signature of a Parasitic Life Style? PLoS ONE 2(3):
          e335.
            doi:10.1371/journal.pone.0000335</p><p><strong>Academic Editor: </strong>Neil Hall, Institute for Genomic Research, United States of America</p><p><strong>Received:</strong> December 13, 2006; <strong>Accepted:</strong> February 8, 2007; <strong>Published:</strong> March 28, 2007</p><p><strong>Copyright:</strong> © 2007 Stefan Wuchty. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The Northwestern Institute of Complexity supported this study.</p><p><strong>Competing interests:</strong> The author has declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The application of an adapted Yeast-two-hybrid technique allowed for the determination of a core set of protein interaction in the human malaria parasite <em>P. falciparum</em>, roughly covering 25% of the predicted proteome <a href="#pone.0000335-LaCount1">[1]</a>. Statistical analysis of the network structure resulted in the presence of scale-free behavior. This well established inhomogeneity of biological networks highlights the role of hubs, a small minority of strongly connected proteins <a href="#pone.0000335-Jeong1">[2]</a>, <a href="#pone.0000335-Barabasi1">[3]</a>, which govern a networks integrity. Another special, yet general feature of biological networks is their tendency to shape densely connected and well pronounced modules that largely share similar functions. An evolutionary corollary to modularity comes from the observation, that tightly connected modules not only show a high degree of functional homogeneity but are largely evolutionary conserved as orthologs in other organisms <a href="#pone.0000335-Fraser1">[4]</a>, <a href="#pone.0000335-Wuchty1">[5]</a>, <a href="#pone.0000335-Wuchty2">[6]</a>. In particular, a network comparison of the currently available interactome of <em>P. falciparum</em> revealed that functional modules which are largely conserved in the comparative set of other eukaryotic organisms are rudimentary present in the parasites interactome <a href="#pone.0000335-Suthram1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The modern picture of networks as being governed by highly connected nodes connecting and/or maintaining the integrity of a single module <a href="#pone.0000335-Han1">[8]</a> has been recently nurtured, yet challenged by the determination of the so-called ‘rich-club’ phenomenon <a href="#pone.0000335-Colizza1">[9]</a>. This property refers to the tendency of hubs to be well connected to each other. Essentially, highly interacting nodes are more likely to form tight and intertwined sub-graphs (clubs) than their less connected counterparts. In a social context, a strong rich-club phenomenon indicates the dominance of an “oligarchy” of highly connected and mutually communicating individuals. In biological systems, many well defined functional sub-communities are governed and loosely connected by highly interacting proteins <a href="#pone.0000335-Han1">[8]</a>; however, such hubs lack strong inter-connectivity among each other. Protein-protein interaction networks, however, do not show rich-club behavior <a href="#pone.0000335-Colizza1">[9]</a>. In fact, protein hubs appear to be organized in remotely placed, yet functionally homogeneous sub-communities and are largely linked to less well conjoined proteins.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In the light of these results, we present an analysis of the experimentally determined interactome of <em>P. falciparum</em>, pointing to the surprising result that the underlying network features an ‘oligarchy’ of highly interacting proteins. In total contrast, we find that evolutionary conserved interactions between <em>Plasmodium</em> proteins that we derived from <em>S. cerevisiae, C. elegans</em> and <em>D. melanogaster</em> preserve the absence of any oligarchy of dominating hubs. Combining the interactions sets from different origin, we observe not only a minimal overlap, but also see that interactions appear to be downright spatially separated from each other. Highlighting the network vicinity around the oligarchy of proteins in the experimentally obtained interactions, we find that the rich club revolves around parasite specific functions and is embedded in a sea of interologs. In particular, we find an enrichment of secreted proteins while orthologous proteins are considerably diluted. This particular observations as quantified by the detection of the rich club nodes which appear to be parasite specific leaves us with questions about the parasite in general and the current interaction data of its proteins in particular.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Determination of Rich Club Phenomenon:</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The so-called rich-club phenomenon is quantitatively defined by the rich-club coefficient <em>Φ(k)</em> <a href="#pone.0000335-Colizza1">[9]</a>. Denoting by <em>E<sub>k</sub></em> the number of edges among the <em>N<sub>≥k</sub></em> nodes which have more than <em>k</em> interaction partners, the rich-club coefficient is expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>N<sub>≥k</sub></em>(<em>N<sub>≥k</sub></em>−1)/2 represents the maximally possible number of edges among <em>N<sub>≥k</sub></em> nodes. An appropriate choice for the normalization of the rich-club coefficient is provided by the ratio<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.e002&amp;representation=PNG" class="inline-graphic"></span><br>where Φ<em><sub>ran</sub></em>(<em>k</em>) is the rich-club coefficient of a random network with the same degree distribution <em>P(k)</em>. The choice of pairs of links, whose end nodes are exchanged, allows us to obtain a maximally random network while the degree distribution is preserved <a href="#pone.0000335-Colizza1">[9]</a>, <a href="#pone.0000335-Maslov1">[10]</a>. In order to have a reasonably large ensemble, we repeat the randomization process <em>10,000</em> times. Binning nodes according to their degrees k we obtain a degree dependent mean value of the rich-club coefficient by averaging over all <em>ρ</em>'s in each bin. A ratio larger than one, <em>ρ (k)&gt;1</em>, is the actual evidence for the presence of a rich-club phenomenon, leading to an increase in the inter-connectivity of large degree nodes in a more pronounced way than in the random case. This process is well displayed by the presence of an oligarchy of highly interacting nodes that are well connected among each other. On the contrary, a ratio <em>ρ (k)&lt;1</em> points to a lack of inter-connectivity among large degree nodes which are separated in distinguishable modules.</p>


<h4>Protein Interaction Data:</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The application of an adapted Yeast-two-hybrid technique allowed for the determination of a core set of <em>2, 811</em> interactions among <em>1, 308</em> proteins in the human malaria parasite <em>P. falciparum</em> <a href="#pone.0000335-LaCount1">[1]</a>. As for yeast and worm specific protein interaction data, we utilized the DIP database <a href="#pone.0000335-Xenarios1">[11]</a>, providing <em>17, 346</em> interactions among <em>4, 928</em> proteins of <em>S. cerevisiae</em> and <em>3, 926</em> interactions among <em>2, 718</em> proteins of <em>C. elegans</em>. As a reliable source of interactions among proteins of <em>D. melanogaster</em>, we utilized <em>6, 222</em> proteins and <em>16, 914</em> links <a href="#pone.0000335-Giot1">[12]</a> from a two-hybrid study.</p>


<h4>Orthologous Protein Data:</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As a source of reliable orthologous protein information we utilized the InParanoid database <a href="#pone.0000335-Remm1">[13]</a>. The algorithm for detecting orthologous relationships is based on pairwise similarity scores which are by default calculated with BLASTP. Mutually best hits between two sequences from different species serve as main orthologs that form an orthologous group. Other sequences are added to this group if they are closely related to one of the main orthologs and are known as “in-paralogs.” A confidence value for each group member is provided by a standard bootstrap procedure and shows how closely related a protein sequence is to the main ortholog. Because of the difficulties in the detection of similarities arising from the parasites specific genome composition <a href="#pone.0000335-Aravind1">[14]</a>, we only selected the main sequence pairs of each orthologous group allowing us to obtain <em>1, 024</em> proteins in S. cerevisae with putative orthologs in <em>P. falciparum.</em> Analogously, we obtain <em>1, 333</em> orthologous pairs in <em>C. elegans</em> and <em>1, 351</em> in <em>D. melanogaster</em>.</p>


<h4>Secretome:</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To establish infection in the host, malaria parasites export remodeling and virulence proteins into the erythrocyte. Recent studies independently uncovered a host cell targeting (HCT) signal that allows proteins to cross into the human erythrocyte cell by passing several membranes <a href="#pone.0000335-Marti1">[15]</a>, <a href="#pone.0000335-Hiller1">[16]</a>, <a href="#pone.0000335-Sargeant1">[17]</a>. Combining these data sets, we compile a list of <em>525</em> proteins in <em>P. falciparum</em>.</p>


<h4>Enrichment:</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>According to their placement in layers around the rich-club proteins, we pool proteins or interactions in groups from the rich-club (layer #0) up to a layer #<em>l</em>, in question. In each group #<em>l</em>, we determine the fraction of proteins or links <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.e003&amp;representation=PNG" class="inline-graphic"></span>, that have a certain feature <em>A</em>, where <em>n<sup>A</sup><sub>l</sub></em> is the respective number of proteins or interactions with feature <em>A</em> and <em>n<sub>l</sub></em> is the total number of proteins or interaction in group <em>l</em>. As an appropriate null-model, we randomly distribute feature <em>A</em> among proteins or interactions, and analogously determine their respective fraction in group <em>l</em>, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.e004&amp;representation=PNG" class="inline-graphic"></span>. Thus, for each group <em>l</em> we define the enrichment of a feature <em>A</em> as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.e005&amp;representation=PNG" class="inline-graphic"></span>, which has the <em>l</em>-independent value <em>E<sub>l</sub> = 1</em> if the underlying distribution of <em>A</em> was random. According to the definition of groups <em>E<sub>l</sub></em> converges to 1 while reaching the outer layers. In order to have a reasonably large ensemble, we repeat the randomization process <em>1,000</em> times and average over all <em>E</em>'s in each group. An enrichment of larger than one, <em>E<sup>A</sup><sub>l</sub></em>&gt;1, is a signal for a significant over-representation of feature <em>A</em> in group <em>l</em> and <em>vice versa</em>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Quantitatively, the rich-club phenomenon is defined by the normalized rich-club coefficient <em>ρ (k)</em>. Comparing fractions of actually existing links between nodes that have at least <em>k</em> interaction partners with the maximally possible number of edges in both the underlying and randomized networks <em>ρ</em> allows the determination of significant network patterns. In particular, a ratio larger than one, <em>ρ</em>&gt;1, points to the presence of a rich-club, reflecting an increase in the inter-connectivity of hubs and <em>vice versa</em> <a href="#pone.0000335-Colizza1">[9]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>As representative examples of well established and investigated protein interaction networks, we analyzed a network of <em>17, 346</em> interactions among <em>4, 928</em> proteins of <em>S. cerevisiae</em> and observe a distribution of <em>ρ</em> reaching a minimum at k = 70, a strong signal that points to the absence of any rich clubs (<a href="#pone-0000335-g001">Fig. 1a</a>). Analogously, we find similar minima around <em>k = 58</em> for the web of <em>3, 926</em> interactions among <em>2, 718</em> proteins in <em>C. elegans</em> and <em>k = 70</em> for <em>16, 914</em> interactions between <em>6, 222</em> proteins in <em>D. melanogaster</em>. In order to verify the putative presence of remotely placed modules we extract sub-networks of proteins that have a degree ≥<em>k</em> in the underlying interactomes. Indeed, we find that sub-networks composed of nodes that have at least the number of interactions where minimas occur in the distributions of <em>ρ (k = 70)</em>, break into the largest amount of disconnected parts, validating our assumption that the underlying organism-specific networks indeed are composed of a small number of remotely placed sub-communities (<a href="#pone-0000335-g001">Fig. 1b</a>). Changing our focus on a core set of <em>2, 811</em> experimentally determined interactions among <em>1, 308</em> proteins in <em>P. falciparum</em> <a href="#pone.0000335-LaCount1">[1]</a>, we observe that this network shows a slightly mixed rich-club signal encountering a minimum at <em>k = 19</em> and reaching a maximum at <em>k = 45</em> (<a href="#pone-0000335-g001">Fig. 1b</a>). Noteworthy, such a signature has been observed in pure scale-free networks <a href="#pone.0000335-Colizza1">[9]</a>, that lack any clustering and degree correlations. The observation that sub-networks where each protein has at least a certain degree in the underlying full web are organized in one connected component validates our assumption that the underlying network indeed features an oligarchy of highly connected proteins (<a href="#pone-0000335-g001">Fig. 1c</a>). Even at the minimum in the distribution of the rich-club coefficient <em>ρ</em> at <em>k</em>≥19 the network keeps its integrity.</p>
<div class="figure" id="pone-0000335-g001"><div class="img"><a name="pone-0000335-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000335" data-uri="info:doi/10.1371/journal.pone.0000335.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000335.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000335.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Determination and features of rich-clubs.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(a) The distributions of the mean rich-club coefficient <em>ρ</em> in protein interaction networks of <em>S. cerevisiae, D. melanogaster</em> and <em>C. elegans</em> run through significant minimas, <em>ρ (k)&lt;1</em>, indicating the absence of any rich club phenomenon. In particular, the corresponding distributions reach a minimum at <em>k <sub>yeast,fly</sub> = 70</em> and <em>k <sub>worm</sub> = 58</em>. (b) Focusing on a network of experimentally determined protein interactions in the human malaria parasite <em>P. falciparum</em>, we obtain a fundamentally different picture. A considerable increase of <em>ρ</em> at elevated levels of connectivity peaking at <em>k = 45</em> suggests the presence of an oligarchy of highly interacting and well intertwined proteins. We also observe a minimum at <em>k = 19</em>. Assuming that interactions between proteins in the eukaryotic networks in (a) that have an ortholog in <em>Plasmodium</em> are conserved as well, we observe that the network of interologs lacks this feature. Combining both interaction sets, the rich-club signal we encountered in the experimental interaction network weakly prevails at the same degree cut-off. (c) Since the absence of any rich-club phenomenon points to the presence of well defined sub-networks, we calculate the number of disconnected sub-networks emerging from interactions between proteins with increasing degree cut-offs. We observe that the sub-networks of yeast, fly and worm expectedly break into parts at cut-offs <em>k</em> where <em>ρ (k)&lt;1</em>. Noteworthy, we obtain the largest number of disconnected components at degree cut-offs, where the corresponding distributions of <em>ρ</em> reach a minimum. In turn, sub-networks of experimentally determined protein interactions in <em>P. falciparum</em> keep their integrity. Inheriting this feature from its model organisms, evolutionary conserved protein interactions not only break into two disconnected parts but also keep this characteristic upon combination with experimental interactions in <em>Plasmodium</em>. (d) Although similar in their sizes, interactions in both networks only overlap to a very small but significant extent (<em>P&lt;10<sup>−4</sup></em> by assuming a hypergeometric distribution), while a considerable and significant amount of proteins (in parentheses) appear in both the experimental and evolutionary conserved protein interactions network (<em>P&lt;10<sup>−4</sup></em>).</p>
<span>doi:10.1371/journal.pone.0000335.g001</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In the light of this anomaly, we wonder if this effect is limited to the experimentally obtained interactions only, or prevails in evolutionary conserved links that we obtain from interologs in the eukaryotic interactomes. Utilizing orthologous protein information from the InParanoid database <a href="#pone.0000335-Remm1">[13]</a>, we obtain a network of <em>1,006</em> proteins embedded in <em>2, 654</em> conserved interactions in <em>Plasmodium</em> that we derived from interactions in <em>S. cerevisiae, C. elegans</em> and <em>D. melanogaster</em>. Comparing the experimentally and evolutionary derived sets of interactions, we observe a very small but statistically significant overlap of links (<a href="#pone-0000335-g001">Fig. 1d</a>). In the distribution of the normalized rich-club coefficient of conserved protein interactions we find (<a href="#pone-0000335-g001">Fig. 1b</a>) a minimum at <em>k</em>≥21, pointing to the absence of any rich club phenomenon, supported by the observations that sub-networks of different degree cut-offs split into disconnected parts (<a href="#pone-0000335-g001">Fig. 1c</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>However, combining the two data sets in a network of <em>5,458</em> interactions between <em>1, 986</em> proteins and repeating our analysis we observe that the initially found mixed rich-club signal weakly prevails (All protein interactions can be found in <a href="#pone.0000335.s001">Table S1</a>).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>As for a qualitative inspection of these networks we show a graphical depiction in <a href="#pone-0000335-g002">Fig. 2a</a>. Coloring each interaction by its origin, we observe that the different data sets are almost spatially separated from each other when applying a standard graph layout algorithm provided by the Cytoscape program <a href="#pone.0000335-Shannon1">[18]</a>. In <a href="#pone-0000335-g002">Fig. 2b</a>, we focus on the sub-network that is spanned by proteins that have at least <em>19</em> neighbors, a degree cut-off that corresponds to the minimum of the rich-club coefficient in the network of experimentally determined interactions. Confirming our previous observation, we find a significant separation of areas that largely correspond to either experimental or evolutionary conserved interactions. Experimentally determined interactions appear to embed proteins that are part of the rich club (inset), mostly carrying parasite specific functions. In particular, the <em>9</em> proteins being embedded in an oligarchy of highly connected nodes proteins are predominately important for the invasion of the human host in the merozoite stage <a href="#pone.0000335-LaCount1">[1]</a>, <a href="#pone.0000335-Bozdech1">[19]</a>, <a href="#pone.0000335-Winzeler1">[20]</a>, <a href="#pone.0000335-Johnson1">[21]</a>, <a href="#pone.0000335-Bahl1">[22]</a>, where the surface proteins PFL1385c, PFI1475w and PFE0040c play a pivotal role. Especially the latter protein causes attention since it is the only one in the rich club that carries an export signal, allowing it to enter the human host cell <a href="#pone.0000335-Marti1">[15]</a>, <a href="#pone.0000335-Hiller1">[16]</a>, <a href="#pone.0000335-Sargeant1">[17]</a>. Utilizing human orthologs as a common denominator, we find that one third of the rich club proteins have a human ortholog. Clarifying, we labeled each protein of <em>Plasmodium</em> with its human ortholog in <a href="#pone-0000335-g002">Fig. 2b</a>, allowing us to see that a relatively small number of proteins in <em>Plasmodium</em> have a human ortholog in the immediate vicinity of the rich-club proteins. However, in a small number of steps away from the rich-club, we encounter a lot of proteins with human orthologs, which are largely organized in evolutionary conserved interactions.</p>
<div class="figure" id="pone-0000335-g002"><div class="img"><a name="pone-0000335-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000335" data-uri="info:doi/10.1371/journal.pone.0000335.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000335.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000335.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Structure of the rich-club.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>(a) Combining experimentally and evolutionary conserved interactions in <em>P. falciparum</em> we obtain a network of <em>5, 458</em> interactions among <em>1, 986</em> proteins. Recalling that overlaps between these two data sets is minimal we obtain a visually striking image by coloring interactions according to their origin, allowing us to observe that both types of interactions appear to be largely separated. (b) Here, we focus on the sub-network that is spanned by proteins that have at least <em>19</em> neighbors, a degree cut-off that corresponds to the minimum of the rich-club coefficient in the network of experimentally determined interactions. Confirming our previous observation, we find a strong separation of areas that largely correspond to either experimental or evolutionary interactions. Experimentally determined interactions appear to embed proteins that are part of the rich club (inset). Notably, in the immediate vicinity of the oligarchy of nodes that are strongly connected among each other, we find a relatively small number of proteins in <em>Plasmodium</em> that have a human ortholog. Clarifying, we labeled each protein of <em>Plasmodium</em> with its human ortholog. However, in a small number of steps away from the rich-club, we encounter a lot of proteins with human orthologs, which are largely organized in evolutionary conserved interactions.</p>
<span>doi:10.1371/journal.pone.0000335.g002</span></div><a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Quantifying these observations, we constructed layers around the sample of nine rich-club proteins. In particular, proteins that are a certain number of steps away in the combined network of evolutionary conserved and experimental interactions constitute a new layer, allowing us to find a total of six layers. In <a href="#pone-0000335-g003">Fig. 3a</a>, we show the cumulative frequency distribution of the number of proteins we find running toward the outer layers, suggesting that reaching the second layer already accounts for more than half of all proteins in the network. Analyzing the composition of layers, we define enrichment as the fraction of orthologs obtained in the pool of proteins up to this layer over the total number of orthologs in the whole network compared to a random null model. If the initial placement of orthologs was random, the enrichment value would always be <em>1</em>. In <a href="#pone-0000335-g003">Fig. 3b</a>, we observe that orthologs seem to be randomly distributed in experimental interactions. While the concentration of human orthologs in a network that combines both experiments and interologs drops around the rich club, it ascends to <em>1</em> throughout the following layers, suggesting that the layers around the rich-club are largely unaffected by evolution and therefore constitute an area of parasite specific activity. Similarly, we determined the enrichment of proteins with an export signal in both the experimental and combined network of interactions in layers around the rich-club proteins. In particular, these proteins are the major constituents that allow the parasite to invade and gain control over the host cell by secreting them into the lumen of the human red blood cell. Compared to experimental interactions, we find that the addition of interologs enhances the enrichment of secreted proteins, confirming the particular role of the rich-club proteins for parasitic functions (<a href="#pone-0000335-g003">Fig. 3c</a>). Support for our conclusion that rich-club proteins arrange parasite specific functions also comes from the observation that especially in the inner layers experimentally obtained interactions dominate. Experimental observations largely cover links between parasite specific proteins <a href="#pone.0000335-LaCount1">[1]</a> (<a href="#pone-0000335-g003">Fig. 3d</a>), suggesting that the area around the rich club of proteins not only is predominately a matter of parasite specific activity. The rich-club of proteins also appears to be the center of these activities, shaping a ‘spear-tip’ of the parasitic invasion (Detailed information about the different layers can be found in <a href="#pone.0000335.s002">Table S2</a>).</p>
<div class="figure" id="pone-0000335-g003"><div class="img"><a name="pone-0000335-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000335" data-uri="info:doi/10.1371/journal.pone.0000335.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000335.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000335.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000335.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000335.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000335.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Statistics of layers around the rich-club.</span></strong></p><a id="article1.body1.sec3.fig3.caption1.p1" name="article1.body1.sec3.fig3.caption1.p1"></a><p>(a) Here, we show the cumulative frequency distribution of the layers around the rich-club (layer #0) toward the periphery of the network (layer #6). In particular, layer <em>#n</em> refers to all the nodes that can be reached <em>n</em> steps away from the rich club of proteins. Reaching the second layer we already accumulate half of all the proteins in the network. These growing pools of proteins obtained by consecutive layers are the references for the following observations. (b) Analyzing the composition of consecutive layers we find no significant enrichment of human orthologs in experimental interactions, pointing to a random distribution of orthologs in the experimental data set. In contrast, the concentration of human orthologs in a network that combines both experiments and interologs is considerably diluted around the rich club and converges to <em>1</em> by definition throughout the following layers (dotted line). (c) Similarly, we determined the enrichment of proteins participating in the secretome in both the experimental and combined network of interactions in layers around the rich-club proteins. Compared to experimental interactions, we find that the addition of interologs enhances the enrichment of proteins that are secreted into the human host cell upon parasitic invasion. (d) Focusing on experimental interactions alone, we find that such links are predominately enriched in layers around the rich club, all together suggesting that the area around the rich club of proteins is predominately a matter of parasite specific activity.</p>
<span>doi:10.1371/journal.pone.0000335.g003</span></div></div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>These considerable differences between experimentally determined and computationally derived interaction networks of the same species incurs interesting questions. While protein hubs in the conserved network widely are members of well known evolutionary conserved house hold functions, protein hubs in the experimental interaction set are widely found in more parasite specific features. In particular, the interesting topology revolves around such functions that are responsible for the parasites ability to invade the human host. Although such a signal might be potentially an artifact of the experimental procedures, further support for its biological significance comes from the observation that the topological vicinity around the rich club shows strong enrichment signals of parasite specific entities. This observation combined with an unprecedented topology, could putatively point to a signature of a parasitic lifestyle. Yet, the absence of any noteworthy overlaps and accordingly the rudimentary conservation of prominent cell functions <a href="#pone.0000335-Suthram1">[7]</a> in <em>Plasmodium</em> remains noteworthy, especially since about 25% of proteins appear in both data sets. On the one hand, this observation might be the consequence of technical difficulties to express <em>Plasmodium</em> proteins in yeast and sampling issues that arise from choosing random fragments to screen. The latter aspect that leads to a random sample of interactions might be a reason for obtaining such a remarkable topology since random sampling methods tend to inaccurately reflect the underlying topology of an interaction network <a href="#pone.0000335-Han2">[23]</a>, <a href="#pone.0000335-Stumpf1">[24]</a>. On a different note, a recent assessment of protein interactions in Yeast revealed startling error rates <a href="#pone.0000335-VonMering1">[25]</a>. Although such an assessment is not available for <em>Plasmodium</em> we assume that this data set is error prone as well, especially since interactions have been determined by an adapted <em>Y2H</em> approach. Therefore, the experimentally determined interaction set might carry erroneous signals. However, considering that we inferred conserved interactions predominately from Yeast, we compare experimentally obtained and evolutionary conserved interactions on a reasonably similar level of error. Alternatively, the presence of other, evolutionary conserved interactions provides computational hints that <em>Plasmodium</em> most likely contains conserved protein complexes which have been missed by the experimental approach. In the presence of a large amount of orthologous proteins in <em>Plasmodium</em> that interact in other organisms, there is no reason to assume that interactions do not occur. A reason for the absence of any evolutionary conserved interactions may also be based on limitations of sequence alignment algorithms to accurately detect homologies between eukaryotic and <em>Plasmodium</em> specific proteins. Although sharing significant similarities sequence disruptions caused by repeats and other inserts can aggravate the proper detection of evolutionary relationships of <em>Plasmodium</em> genes and proteins with other organisms <a href="#pone.0000335-Aravind1">[14]</a>. Yet, the pronounced placement of orthologous proteins in interaction networks mitigates this effect since the relevant signals can still be detected even in the face of tremendous noise in interaction data as well as orthologus data <a href="#pone.0000335-Wuchty3">[26]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Although there are considerable difficulties in the experimental and computational detection of protein interactions in <em>Plasmodium</em> that can lead to artefactual observations, we nevertheless find strong biological significance in the obtained rich-club phenomenon in the interactome of <em>Plasmodium</em>, results that potentially can point to a topological signature of a parasite specific life style. Even though results appear encouraging, the current level of the determination of the parasites interactome is in its early phases, pointing to the necessity to complete the set of interactions as well as assess their quality to further support these results.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000335"></div><a name="pone.0000335.s001" id="pone.0000335.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000335.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Protein interactions in P. falciparum. This file contains all the protein interactions in P. falciparum used in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000335.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.84 MB XLS)</p>
<a name="pone.0000335.s002" id="pone.0000335.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000335.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Layers in the interactome of P. falciparum. This file contains information about the proteins in the different layers of the interactome of P. falciparum.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000335.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.31 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The author thanks V. Colizza for many discussions and A. Atkinson for helping to write the paper. The Northwestern Institute of Complexity (NICO) supported this study.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SW. Performed the experiments: SW. Analyzed the data: SW. Contributed reagents/materials/analysis tools: SW. Wrote the paper: SW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000335-LaCount1" id="pone.0000335-LaCount1"></a>LaCount D, Vignali M, Chettier R, Phansalkar A, Bell R, et al.  (2005) A protein interaction network of the malaria parasite plasmodium falciparum. Nature  438: 103–107.  <ul class="find" data-citedArticleID="941134" data-doi="10.1038/nature04104"><li><a href="http://dx.doi.org/10.1038/nature04104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein+interaction+network+of+the+malaria+parasite+plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein+interaction+network+of+the+malaria+parasite+plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000335-Jeong1" id="pone.0000335-Jeong1"></a>Jeong H, Mason S, Barabasi A-L, Oltvai Z (2001) Lethality and centrality in protein networks. Nature  411: 41–42.  <ul class="find" data-citedArticleID="941130" data-doi="10.1038/35075138"><li><a href="http://dx.doi.org/10.1038/35075138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethality+and+centrality+in+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethality+and+centrality+in+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000335-Barabasi1" id="pone.0000335-Barabasi1"></a>Barabasi A-L, Oltvai Z (2004) Network biology: Understanding the cell's functional organization. Nat Rev Gen  4: 101–113.  <ul class="find" data-citedArticleID="941114" data-doi="10.1038/nrg1272"><li><a href="http://dx.doi.org/10.1038/nrg1272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+biology%3A+Understanding+the+cell%27s+functional+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+biology%3A+Understanding+the+cell%27s+functional+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000335-Fraser1" id="pone.0000335-Fraser1"></a>Fraser H, Hirsh A, Steinmetz L, Scharfe C, Feldman M (2002) Evolutionary rate in the protein interaction network. Science  296: 750–752.  <ul class="find" data-citedArticleID="941120" data-doi="10.1126/science.1068696"><li><a href="http://dx.doi.org/10.1126/science.1068696" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+rate+in+the+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+rate+in+the+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000335-Wuchty1" id="pone.0000335-Wuchty1"></a>Wuchty S, Oltvai Z, Barabasi A-L (2003) Evolutionary conservation of motif constituents within the yeast protein interaction network. Nat Genet  35: 176–179.  <ul class="find" data-citedArticleID="941154" data-doi="10.1038/ng1242"><li><a href="http://dx.doi.org/10.1038/ng1242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+conservation+of+motif+constituents+within+the+yeast+protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+conservation+of+motif+constituents+within+the+yeast+protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000335-Wuchty2" id="pone.0000335-Wuchty2"></a>Wuchty S (2004) Topology and evolution in yeast interaction networks. Genome Res  14: 1310–1314.  <ul class="find" data-citedArticleID="941156" data-doi="10.1101/gr.2300204"><li><a href="http://dx.doi.org/10.1101/gr.2300204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topology+and+evolution+in+yeast+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topology+and+evolution+in+yeast+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000335-Suthram1" id="pone.0000335-Suthram1"></a>Suthram S, Sittler T, Ideker T (2006) The plasmodium protein network diverges from those of other eukaryotes. Nature  438: 108–112.  <ul class="find" data-citedArticleID="941148" data-doi="10.1038/nature04135"><li><a href="http://dx.doi.org/10.1038/nature04135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+plasmodium+protein+network+diverges+from+those+of+other+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+plasmodium+protein+network+diverges+from+those+of+other+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000335-Han1" id="pone.0000335-Han1"></a>Han J, Bertin N, Hao T, Goldberg DS, Berriz G, et al.  (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. Nature  430: 88–93.  <ul class="find" data-citedArticleID="941124" data-doi="10.1038/nature02555"><li><a href="http://dx.doi.org/10.1038/nature02555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+dynamically+organized+modularity+in+the+yeast+protein-protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000335-Colizza1" id="pone.0000335-Colizza1"></a>Colizza V, Flamini A, Serano M, Vespignani A (2006) Detecting rich-club ordering in complex networks. Nat Physics  2: 110–115.  <ul class="find" data-citedArticleID="941118" data-doi="10.1038/nphys209"><li><a href="http://dx.doi.org/10.1038/nphys209" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+rich-club+ordering+in+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+rich-club+ordering+in+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000335-Maslov1" id="pone.0000335-Maslov1"></a>Maslov S, Sneppen K (2002) Specificity and stability in topology of protein networks. Science  296: 910–913.  <ul class="find" data-citedArticleID="941138" data-doi="10.1126/science.1065103"><li><a href="http://dx.doi.org/10.1126/science.1065103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+and+stability+in+topology+of+protein+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+and+stability+in+topology+of+protein+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000335-Xenarios1" id="pone.0000335-Xenarios1"></a>Xenarios I, Salwinski L, Duan X, Higney P, Kim SM, et al.  (2002) DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions. Nucl Acids Res  30: 303–305.  <ul class="find" data-citedArticleID="941160" data-doi="10.1093/nar/30.1.303"><li><a href="http://dx.doi.org/10.1093/nar/30.1.303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DIP%2C+the+Database+of+Interacting+Proteins%3A+a+research+tool+for+studying+cellular+networks+of+protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DIP%2C+the+Database+of+Interacting+Proteins%3A+a+research+tool+for+studying+cellular+networks+of+protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000335-Giot1" id="pone.0000335-Giot1"></a>Giot L, Bader J, Brouwer C, Chaudhuri A, Kuang B, et al.  (2004) A protein interaction map of drosophila melanogaster. Science  302: 1727–1736.  <ul class="find" data-citedArticleID="941122" data-doi="10.1126/science.1090289"><li><a href="http://dx.doi.org/10.1126/science.1090289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein+interaction+map+of+drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein+interaction+map+of+drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000335-Remm1" id="pone.0000335-Remm1"></a>Remm M, Storm C, Sonnhammer E (2001) Automatic clustering of orthologs and in-paralogs from pairwise species comparisons. J Mol Biol  314: 1041–1052.  <ul class="find" data-citedArticleID="941140" data-doi="10.1006/jmbi.2000.5197"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.5197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+clustering+of+orthologs+and+in-paralogs+from+pairwise+species+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+clustering+of+orthologs+and+in-paralogs+from+pairwise+species+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000335-Aravind1" id="pone.0000335-Aravind1"></a>Aravind L, Iyer L, Wellems T, Miller L (2003) Plasmodium biology: Genomic gleanings. Cell  115: 771–785.  <ul class="find" data-citedArticleID="941110" data-doi="10.1016/s0092-8674(03)01023-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)01023-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+biology%3A+Genomic+gleanings." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+biology%3A+Genomic+gleanings.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000335-Marti1" id="pone.0000335-Marti1"></a>Marti M, Good R, Rug M, Knuepfer E, Cowman A (2004) Targeting malaria virulence and remodeling proteins to the host erythrocyte. Science  306: 1930–1934.  <ul class="find" data-citedArticleID="941136" data-doi="10.1126/science.1102452"><li><a href="http://dx.doi.org/10.1126/science.1102452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeting+malaria+virulence+and+remodeling+proteins+to+the+host+erythrocyte." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeting+malaria+virulence+and+remodeling+proteins+to+the+host+erythrocyte.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000335-Hiller1" id="pone.0000335-Hiller1"></a>Hiller N, Bhattacharjee S, van Ooij C, Liolios K, Harrison T, et al.  (2004) A host-targeting signal in virulence proteins reveals a secretome in malarial infection. Science  306: 1934–1937.  <ul class="find" data-citedArticleID="941128" data-doi="10.1126/science.1102737"><li><a href="http://dx.doi.org/10.1126/science.1102737" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+host-targeting+signal+in+virulence+proteins+reveals+a+secretome+in+malarial+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+host-targeting+signal+in+virulence+proteins+reveals+a+secretome+in+malarial+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000335-Sargeant1" id="pone.0000335-Sargeant1"></a>Sargeant T, Marti M, Caler E, Carlton J, Simpson K, et al.  (2006) Lineage-specific expansion of proteins exported to erythrocytes in malaria parasites. Genome Biol  7: R12.  <ul class="find" data-citedArticleID="941142" data-doi="10.1186/gb-2006-7-2-r12"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-2-r12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lineage-specific+expansion+of+proteins+exported+to+erythrocytes+in+malaria+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lineage-specific+expansion+of+proteins+exported+to+erythrocytes+in+malaria+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000335-Shannon1" id="pone.0000335-Shannon1"></a>Shannon P, Markiel A, Ozier O, Baliga N, Wang J, et al.  (2003) Cytoscape: A software environment for integrated models of biomolecular interaction networks. Genome Res  13: 2498–2504.  <ul class="find" data-citedArticleID="941144" data-doi="10.1101/gr.1239303"><li><a href="http://dx.doi.org/10.1101/gr.1239303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoscape%3A+A+software+environment+for+integrated+models+of+biomolecular+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoscape%3A+A+software+environment+for+integrated+models+of+biomolecular+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000335-Bozdech1" id="pone.0000335-Bozdech1"></a>Bozdech Z, Llinas M, Pulliam B, Wong E, Zhu J, et al.  (2003) The transcriptome of the intraerythrocytic developmental cycle of plasmodium falciparum. PLoS Biology  1: 1–16.  <ul class="find" data-citedArticleID="941116" data-doi="10.1371/journal.pbio.0000005"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptome+of+the+intraerythrocytic+developmental+cycle+of+plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptome+of+the+intraerythrocytic+developmental+cycle+of+plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000335-Winzeler1" id="pone.0000335-Winzeler1"></a>Winzeler E (2006) Applied systems biology and malaria. Nat Rev Micro  4: 145–151.  <ul class="find" data-citedArticleID="941152" data-doi="10.1038/nrmicro1327"><li><a href="http://dx.doi.org/10.1038/nrmicro1327" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applied+systems+biology+and+malaria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applied+systems+biology+and+malaria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000335-Johnson1" id="pone.0000335-Johnson1"></a>Johnson KLJ, Florens L, Zhou Y, Santrosyan A, Grainger M, et al.  (2004) Global analysis of transcript and protein levels across the plasmodium falciparum life cycle. Genome Res  14: 2308–2318.  <ul class="find" data-citedArticleID="941132" data-doi="10.1101/gr.2523904"><li><a href="http://dx.doi.org/10.1101/gr.2523904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+transcript+and+protein+levels+across+the+plasmodium+falciparum+life+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+transcript+and+protein+levels+across+the+plasmodium+falciparum+life+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000335-Bahl1" id="pone.0000335-Bahl1"></a>Bahl A, Brunk B, Crabtree J, Fraunholz M, Gajria B, et al.  (2003) Plasmodb: the plasmodium genome resource. a database integrating experimental and computational data. Nucl Acids Res  31: 212–215.  <ul class="find" data-citedArticleID="941112" data-doi="10.1093/nar/gkg081"><li><a href="http://dx.doi.org/10.1093/nar/gkg081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodb%3A+the+plasmodium+genome+resource.+a+database+integrating+experimental+and+computational+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodb%3A+the+plasmodium+genome+resource.+a+database+integrating+experimental+and+computational+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000335-Han2" id="pone.0000335-Han2"></a>Han J, Dupuy D, Bertin N, Cusick M, Vidal M (2005) Effect of sampling on topology predictions of protein-protein interaction networks. Nat Biotech  23: 839–844.  <ul class="find" data-citedArticleID="941126" data-doi="10.1038/nbt1116"><li><a href="http://dx.doi.org/10.1038/nbt1116" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+sampling+on+topology+predictions+of+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+sampling+on+topology+predictions+of+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000335-Stumpf1" id="pone.0000335-Stumpf1"></a>Stumpf M, Wiuf C, May R (2005) Subnets of scale-free networks are not scale-free: Sampling properties of networks. Proc Natl Acad Sci USA  101: 4221–4224.  <ul class="find" data-citedArticleID="941146" data-doi="10.1073/pnas.0501179102"><li><a href="http://dx.doi.org/10.1073/pnas.0501179102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subnets+of+scale-free+networks+are+not+scale-free%3A+Sampling+properties+of+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subnets+of+scale-free+networks+are+not+scale-free%3A+Sampling+properties+of+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000335-VonMering1" id="pone.0000335-VonMering1"></a>Von Mering C, Krause R, Snel B, Cornell M, Oliver S, et al.  (2002) Comparative assessment of large-scale data sets of protein-protein interactions. Nature  417: 399–403.  <ul class="find" data-citedArticleID="941150" data-doi="10.1038/nature750"><li><a href="http://dx.doi.org/10.1038/nature750" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+assessment+of+large-scale+data+sets+of+protein-protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+assessment+of+large-scale+data+sets+of+protein-protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000335-Wuchty3" id="pone.0000335-Wuchty3"></a>Wuchty S, Barabasi A-L, Ferdig M (2006) Stable evolutionary signal in a protein interaction network of S. cerevisiae. BMC Evol Biol  6: 8.  <ul class="find" data-citedArticleID="941158" data-doi="10.1186/1471-2148-6-8"><li><a href="http://dx.doi.org/10.1186/1471-2148-6-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stable+evolutionary+signal+in+a+protein+interaction+network+of+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stable+evolutionary+signal+in+a+protein+interaction+network+of+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.XML" value="61692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.PDF" value="341836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e001.PNG" value="3459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e001.TIF" value="25248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e002.PNG" value="2843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e002.TIF" value="24356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e003.PNG" value="1459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e003.TIF" value="22480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e004.PNG" value="1751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e004.TIF" value="22936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e005.PNG" value="1809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.e005.TIF" value="23028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g001.PNG_L" value="131013"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g001.PNG_M" value="154931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g001.PNG_S" value="6282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g001.TIF" value="167984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g001.PNG_I" value="60796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g002.PNG_L" value="408495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g002.PNG_M" value="305761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g002.PNG_S" value="12512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g002.TIF" value="499266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g002.PNG_I" value="169645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g003.PNG_L" value="35640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g003.PNG_M" value="86147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g003.PNG_S" value="4328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g003.TIF" value="85480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.g003.PNG_I" value="36369"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.s001.XLS" value="838656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000335.s002.XLS" value="308224"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000335&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000335" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000335&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000335&volume=&issue=&title=Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?&author_name=Stefan%20Wuchty&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000335" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335&amp;t=Rich-Club%20Phenomenon%20in%20the%20Interactome%20of%20P.%20falciparum%E2%80%94Artifact%20or%20Signature%20of%20a%20Parasitic%20Life%20Style%3F" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335&title=Rich-Club%20Phenomenon%20in%20the%20Interactome%20of%20P.%20falciparum%E2%80%94Artifact%20or%20Signature%20of%20a%20Parasitic%20Life%20Style%3F&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335&amp;title=Rich-Club%20Phenomenon%20in%20the%20Interactome%20of%20P.%20falciparum%E2%80%94Artifact%20or%20Signature%20of%20a%20Parasitic%20Life%20Style%3F" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000335&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Rich-Club Phenomenon in the Interactome of P. falciparum—Artifact or Signature of a Parasitic Life Style?';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000335';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Rich-Club%20Phenomenon%20in%20the%20Interactome%20of%20P.%20falciparum%E2%80%94Artifact%20or%20Signature%20of%20a%20Parasitic%20Life%20Style%3F http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000335" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000335" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fungal+evolution%22" title="Search for articles in the subject area:'Fungal evolution'"><div class="flagText">Fungal evolution</div></a>
              <div data-categoryid="17551" data-articleid="23508"
                   data-categoryname="Fungal evolution"
                   class="flagImage" title="Flag 'Fungal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Malarial+parasites%22" title="Search for articles in the subject area:'Malarial parasites'"><div class="flagText">Malarial parasites</div></a>
              <div data-categoryid="42885" data-articleid="23508"
                   data-categoryname="Malarial parasites"
                   class="flagImage" title="Flag 'Malarial parasites' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="23508"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Parasite+evolution%22" title="Search for articles in the subject area:'Parasite evolution'"><div class="flagText">Parasite evolution</div></a>
              <div data-categoryid="19291" data-articleid="23508"
                   data-categoryname="Parasite evolution"
                   class="flagImage" title="Flag 'Parasite evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmodium%22" title="Search for articles in the subject area:'Plasmodium'"><div class="flagText">Plasmodium</div></a>
              <div data-categoryid="18205" data-articleid="23508"
                   data-categoryname="Plasmodium"
                   class="flagImage" title="Flag 'Plasmodium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="31803" data-articleid="23508"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="23508"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="23508"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9801'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2742'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4519&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
