

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001113"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001113" />

    <meta name="citation_title" content="Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans"/>
    <meta itemprop="name" content="Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans"/>

      <meta name="citation_author" content="Alisha K. Holloway"/>
            <meta name="citation_author_institution" content="Section of Evolution and Ecology, University of California at Davis, Davis, California, United States of America"/>
            <meta name="citation_author_institution" content="Center for Population Biology, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="David J. Begun"/>
            <meta name="citation_author_institution" content="Section of Evolution and Ecology, University of California at Davis, Davis, California, United States of America"/>
            <meta name="citation_author_institution" content="Center for Population Biology, University of California at Davis, Davis, California, United States of America"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001113.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1113"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Population genomics: whole-genome analysis of polymorphism and divergence in Drosophila simulans.; citation_author=DJ Begun; citation_author=AK Holloway; citation_author=KS Stevens; citation_author=LW Hillier; citation_author=Y Poh; citation_journal_title=PLoS Biol.; citation_volume=5; citation_number=1; citation_pages=e310; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The genomic rate of adaptive evolution.; citation_author=A Eyre-Walker; citation_journal_title=Trends Ecol Evol; citation_volume=21; citation_number=2; citation_pages=569-575; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The genomic rate of adaptive amino-acid substitution in Drosophila.; citation_author=N Bierne; citation_author=A Eyre-Walker; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=3; citation_pages=1350-1360; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Estimating the genomewide rate of adaptive protein evolution in Drosophila.; citation_author=JJ Welch; citation_journal_title=Genetics; citation_volume=173; citation_number=4; citation_pages=821-837; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of relish, a Drosophila NF-kappaB/IkappaB protein.; citation_author=DJ Begun; citation_author=P Whitley; citation_journal_title=Genetics; citation_volume=154; citation_number=5; citation_pages=1231-1238; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of the histone fold domain in centromeric histones.; citation_author=JL Cooper; citation_author=S Henikoff; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=6; citation_pages=1712-1718; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of a candidate gene for aging in Drosophila.; citation_author=PS Schmidt; citation_author=DD Duvernell; citation_author=WF Eanes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=7; citation_pages=10861-10865; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The PROSITE database.; citation_author=N Hulo; citation_author=A Bairoch; citation_author=V Bulliard; citation_author=L Cerutti; citation_author=E De Castro; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=8; citation_pages=D227-D230; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adaptive protein evolution at the Adh locus in Drosophila.; citation_author=JH McDonald; citation_author=M Kreitman; citation_journal_title=Nature; citation_volume=351; citation_number=9; citation_pages=652-654; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Excess amino acid polymorphism in mitochondrial DNA: contrasts among genes from Drosophila, mice, and humans.; citation_author=DM Rand; citation_author=LM Kann; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=10; citation_pages=735-748; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=PROSITE: a documented database using patterns and profiles as motif descriptors.; citation_author=CJA Sigrist; citation_author=L Cerutti; citation_author=N Hulo; citation_author=A Gattiker; citation_author=L Falquet; citation_journal_title=Brief Bioinform; citation_volume=3; citation_number=11; citation_pages=265-274; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions.; citation_author=M Nei; citation_author=T Gojobori; citation_journal_title=Mol Biol Evol; citation_volume=3; citation_number=12; citation_pages=418-426; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Comparison of models for nucleotide substitution used in maximum-likelihood phylogenetic estimation.; citation_author=Z Yang; citation_author=N Goldman; citation_author=A Friday; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=13; citation_pages=316-324; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The Bioperl toolkit: Perl modules for the life sciences.; citation_author=JE Stajich; citation_author=D Block; citation_author=K Boulez; citation_author=SE Brenner; citation_author=SA Chervitz; citation_journal_title=Genome Res; citation_volume=12; citation_number=14; citation_pages=1611-1618; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans"/>
    <meta name="twitter:description" content="Adaptive protein evolution is pervasive in Drosophila. Genomic studies, thus far, have analyzed each protein as a single entity. However, the targets of adaptive events may be localized to particular parts of proteins, such as protein domains or regions involved in protein folding. We compared the population genetic mechanisms driving sequence polymorphism and divergence in defined protein domains and non-domain regions. Interestingly, we find that non-domain regions of proteins are more frequent targets of directional selection. Protein domains are also evolving under directional selection, but appear to be under stronger purifying selection than non-domain regions. Non-domain regions of proteins clearly play a major role in adaptive protein evolution on a genomic scale and merit future investigations of their functional properties."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001113.g001"/>

  <meta property="og:title" content="Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5466'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6831'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2344&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001113">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001113" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001113&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in <i>Drosophila simulans</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alisha K. Holloway
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:akholloway@ucdavis.edu">akholloway@ucdavis.edu</a></p>

                <p>Affiliations:
                  Section of Evolution and Ecology, University of California at Davis, Davis, California, United States of America, 
                  Center for Population Biology, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David J. Begun
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Section of Evolution and Ecology, University of California at Davis, Davis, California, United States of America, 
                  Center for Population Biology, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001113</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001113-g001" data-doi="info:doi/10.1371/journal.pone.0001113" data-uri="info:doi/10.1371/journal.pone.0001113.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001113.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001113-t001" data-doi="info:doi/10.1371/journal.pone.0001113" data-uri="info:doi/10.1371/journal.pone.0001113.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001113.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001113">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001113" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Adaptive protein evolution is pervasive in <em>Drosophila</em>. Genomic studies, thus far, have analyzed each protein as a single entity. However, the targets of adaptive events may be localized to particular parts of proteins, such as protein domains or regions involved in protein folding. We compared the population genetic mechanisms driving sequence polymorphism and divergence in defined protein domains and non-domain regions. Interestingly, we find that non-domain regions of proteins are more frequent targets of directional selection. Protein domains are also evolving under directional selection, but appear to be under stronger purifying selection than non-domain regions. Non-domain regions of proteins clearly play a major role in adaptive protein evolution on a genomic scale and merit future investigations of their functional properties.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Holloway AK, Begun DJ (2007) Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in <em>Drosophila simulans</em>. PLoS ONE 2(10):
          e1113.
            doi:10.1371/journal.pone.0001113</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> October 4, 2007; <strong>Accepted:</strong> October 12, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Holloway, Begun. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This material is based upon work supported by the National Science Foundation to AKH (0434670) and by the National Institutes of Health to DJB (R01 GM071926).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Population genetics analyses indicate that protein divergence in Drosophila, unlike in humans and Arabidopsis, is frequently adaptive <a href="#pone.0001113-Begun1">[1]</a> (see review <a href="#pone.0001113-EyreWalker1">[2]</a>). In flies, the proportion of amino acid substitutions that are adaptive has been estimated to be about 50% <a href="#pone.0001113-Begun1">[1]</a>, <a href="#pone.0001113-EyreWalker1">[2]</a> and is largely consistent across genes <a href="#pone.0001113-Bierne1">[3]</a>, <a href="#pone.0001113-Welch1">[4]</a>. Though most population genetics analyses of adaptive protein divergence treat entire proteins as single units, some analyses have addressed the question of the functional units within proteins that are the primary targets of directional selection (e.g. <a href="#pone.0001113-Begun2">[5]</a>–<a href="#pone.0001113-Schmidt1">[7]</a>). However, there are no genome-scale analyses addressing how population genetic processes may differ between functionally annotated regions of proteins versus those regions with no known function.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Protein domains serve a diversity of specialized functions relating to biochemical activity, binding affinity, subcellular location, or other aspects of protein biology. Regions of proteins that are not annotated as belonging to a domain may still have critical, yet unknown roles in protein function. This parsing of proteins raises the question as to which portion of proteins, domain vs. non-domain is more often subject to directional selection. In one world-view, if adaptive evolution implies functional divergence, such divergence might be more likely to occur in a known, functional domain. Alternatively, if most adaptive protein evolution resulted from fine scale tuning of function relating to, for example, protein folding, then adaptation might tend to occur in non-domain regions. Importantly, rates of divergence in annotated versus unannotated regions of proteins do not resolve these issues because variation in functional constraint cannot be distinguished from variation in the frequency of directional selection. We set out to investigate these issues on a whole-genome scale using population genetic data from the <em>Drosophila simulans</em> genome project.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>A syntenic assembly of partial genome sequences from six <em>D. simulans</em> lines was aligned to the reference sequence from the closely related species, <em>D. melanogaster</em> <a href="#pone.0001113-Begun1">[1]</a>. An alignment of the outgroup, <em>D. yakuba</em>, to <em>D. melanogaster</em> was used to infer substitutions specifically along the <em>D. simulans</em> lineage. Thus, the rich annotation of <em>D. melanogaster</em> was used directly to investigate polymorphism and divergence in <em>D. simulans</em>. <em>Drosophila melanogaster</em> annotations define the locations of many functional protein domains. We directly superimposed PROSITE domain coordinates (v4.3 <em>D. melanogaster</em> annotation; <a href="#pone.0001113.s001">Table S1</a>; <a href="#pone.0001113-Hulo1">[8]</a>) onto the <em>D. simulans</em> population genomic data. Any codons that overlapped multiple domains were counted a single time. Overall, in these analyses we have data for 5,838 genes with defined domains that are comprised of 17,935 total domains, 1,013 of which are unique domain types.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>We used contrasts of polymorphic and fixed, synonymous and nonsynonymous variants to compare the population genetics of domains to non-domain regions. Within genes, domain regions were concatenated and non-domain regions were concatenated for comparisons. These data can be found in <a href="#pone.0001113.s002">Table S2</a> and data for polymorphism and divergence of each gene can be found in <a href="#pone.0001113.s003">Table S3</a>. Levels of synonymous polymorphism were similar between domains and non-domain regions (π<sub>Sdom</sub> = 0.0338, π<sub>Sout</sub> = 0.0333, for domains and non-domains, respectively; Mann-Whitney U [MWU] p = 0.0965; <a href="#pone-0001113-g001">Figure 1</a>). Rates of synonymous site divergence were also comparable (dS<sub>dom</sub> = 0.0496, dS<sub>out</sub> = 0.0502; MWU p = 0.0605; <a href="#pone-0001113-g001">Figure 1</a>). Amino acid polymorphism is quite similar, but is significantly lower in domains compared to non-domain regions (π<sub>Ndom</sub> = 0.0020, π<sub>Nout</sub> = 0.0022; MWU p&lt;0.0001; <a href="#pone-0001113-g001">Figure 1</a>). The rate of protein evolution in domains was significantly lower than in non-domain regions (dN<sub>dom</sub> = 0.0046, dN<sub>out</sub> = 0.0055; MWU p&lt;0.0001; <a href="#pone-0001113-g001">Figure 1</a>). Lower levels of protein polymorphism and divergence in domains are consistent with higher functional constraint. However, slower protein evolution could also result from less frequent adaptive evolution.</p>
<div class="figure" id="pone-0001113-g001"><div class="img"><a name="pone-0001113-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001113.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001113" data-uri="info:doi/10.1371/journal.pone.0001113.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001113.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001113.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001113.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001113.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001113.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001113.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001113.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001113.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001113.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distribution of polymorphism and divergence in domain and non-domain regions of proteins.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Synonymous (top left panel) and nonsynonymous (bottom left) polymorphism in <em>D. simulans</em>. Lineage-specific divergence for synonymous (top right panel) and nonsynonymous (bottom right panel) sites in <em>D. simulans</em> and <em>D. melanogaster</em>.</p>
<span>doi:10.1371/journal.pone.0001113.g001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>To distinguish between these alternatives we used the McDonald-Kreitman test <a href="#pone.0001113-McDonald1">[9]</a>, which tests the neutral theory prediction that the ratio of synonymous-to-nonsynonymous polymorphism should be the same as the ratio of synonymous-to-nonsynonymous divergence. <a href="#pone-0001113-t001">Table 1</a> shows synonymous and nonsynonymous counts for codons in domains and non-domain regions (n = 4,969 genes). Domain and non-domain regions both reject the neutral model (Fisher's Exact Test [FET], p≪10<sup>−6</sup>). In both cases, the ratio of synonymous to nonsynonymous fixations is smaller than the corresponding ratio for polymorphism, which is consistent with adaptive protein divergence. Polymorphic and fixed synonymous variants in non-domain vs. domain sites are not significantly heterogeneous (1.82 vs. 1.81, FET p = 0.538; <a href="#pone-0001113-t001">Table 1</a>). However, the ratio of polymorphic-to-fixed nonsynonymous variants is significantly smaller for non-domain vs. domain codons (0.88 vs. 0.94, FET p = 0.008; <a href="#pone-0001113-t001">Table 1</a>). This suggests that although both classes of sites experience frequent adaptive fixation, non-domain codons may experience more adaptive evolution than domain codons.</p>
<div class="figure" id="pone-0001113-t001"><div class="img"><a name="pone-0001113-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001113.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001113" data-uri="info:doi/10.1371/journal.pone.0001113.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001113.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001113.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001113.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001113.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001113.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001113.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001113.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001113.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001113.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Sum of nonsynonymous and synonymous polymorphisms and fixations over domains and over non-domain regions.</span></strong></p><span>doi:10.1371/journal.pone.0001113.t001</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>To investigate the distribution of variation on an individual gene basis, we used the neutrality index (NI), which is simply a different arrangement of McDonald-Kreitman 2×2 contingency tables <a href="#pone.0001113-Rand1">[10]</a>. Excess nonsynonymous fixation, one signature of adaptive protein evolution, causes NI to be less than 1. We retained 504 domain regions and 1,658 non-domain regions of genes that met our criteria of having at least five nonsynonymous and 5 synonymous variants for further analysis. One count was added to each cell in the 2x2 matrix in order to calculate NI in case any cell contained a zero. This procedure makes the test more conservative as adding one to each cell reduces the power to reject neutrality. <a href="#pone.0001113.s002">Table S2</a> contains all counts of polymorphic and fixed variants used in analyses. We calculated NI for (1) codons within domains and (2) codons in non-domain regions (see <a href="#s3">Methods</a>). The mean neutrality index in protein domains was significantly higher than non-domain regions (analysis of variance: p = 0.0030; both distributions were normally distributed after log<sub>2</sub> transformation) indicating more frequent adaptive evolution in non-domain regions, which is consistent with the interpretation of the MK tests on pooled domain and non-domain codons. However, the proportion of codons in domains is much lower than in non-domain regions (35.7% vs. 64.3%; p&lt;0.0001 MWU) and rates of amino acid divergence are slower. These two factors lead to many fewer counts being recorded in protein domains. Additionally, the method used to calculate NI (see <a href="#s3">Methods</a>) is particularly conservative when counts are low. Given these limitations, we removed domain and non-domain regions with cell counts of zero for synonymous polymorphisms or fixations. We then recalculated NI without adding one to each cell. NI in non-domain regions is still lower than in domains, but not significantly so (analysis of variance: p = 0.0691).</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>In summary, both protein domains and amino acids in non-domain regions have experienced a high proportion of adaptive substitutions. Interestingly, non-domain regions appear to experience more frequent bouts of directional selection. This suggests that although non-domain regions may be less attractive targets of functional analysis in the laboratory, they are extremely important in terms of functional divergence under selection in nature. Future investigations of the mechanistic explanation of frequent adaptive evolution in non-domain regions, whether it is due to fine-tuning of folding patterns or yet to be discovered functions of non-domain regions, are clearly warranted.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>PROSITE protein domain coordinates from the <em>D. melanogaster</em> v4.3 annotation were retrieved by querying the ensembl database <a href="#pone.0001113-Hulo1">[8]</a>. PROSITE domains were identified by the conservation of particular amino acid residues <a href="#pone.0001113-Sigrist1">[11]</a>. All domain coordinates for genes used in the analysis are listed in <a href="#pone.0001113.s001">Table S1</a>. Any codons that overlapped multiple domains were counted a single time.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Syntenic alignments of <em>D. simulans</em> and <em>D. yakuba</em> to the <em>D. melanogaster</em> reference are from <a href="#pone.0001113-Begun1">[1]</a>. Features were defined in the <em>D. melanogaster</em> v4.3 annotation from Flybase (<a href="http://ftp://ftp.flybase.net/genomes/Drosophila_melanogaster/dmel_r4.3_20060303/fasta">ftp://ftp.flybase.net/genomes/Drosophila​_melanogaster/dmel_r4.3_20060303/fasta</a>). A single isoform from each gene (i.e. the isoform with the greatest number of codons) was used for analyses. We used a conservative set of genes that preserved the gene model of <em>D. melanogaster</em> in both <em>D. simulans</em> and <em>D. yakuba</em>. More specifically, the start codon and splice junction locations and sequence and the termination codon location agreed with the <em>D. melanogaster</em> reference sequence.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Polymorphism, as measured by nucleotide diversity (π), was estimated as in <a href="#pone.0001113-Begun1">[1]</a>. The numbers of silent and replacement sites were counted using the method of Nei and Gojobori <a href="#pone.0001113-Nei1">[12]</a>. The pathway between two codons was calculated as the average number of silent and replacement changes from all possible paths between the pair. Estimates of π on the X chromosome were corrected for sample size [π<sub> w</sub> = π * (4/3)] under the assumption that males and females have equal population sizes. Lineage-specific divergence was estimated by maximum likelihood using PAML v3.14 <a href="#pone.0001113-Yang1">[13]</a> and was reported as a weighted average over each <em>D. simulans</em> line with greater than 20 codons in the segment being analyzed. PAML was run in batch mode using a BioPerl wrapper <a href="#pone.0001113-Stajich1">[14]</a> using codeml with codon frequencies estimated from the data. <a href="#pone.0001113.s003">Table S3</a> contains all polymorphism and divergence estimates used in analyses.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>For counts of polymorphic and fixed differences, we only analyzed codons where <em>D. melanogaster</em> and <em>D. yakuba</em> were identical. This allowed us to attribute fixed differences to the <em>D. simulans</em> lineage. Counts of nonsynonymous and synonymous polymorphisms and diverged sites took the path that minimized the number of nonsynonymous substitutions. All data were included in genomic comparisons of domains vs. non-domains. To be included in gene-by-gene domain vs. non-domain NI analyses, we required that there be at least 5 nonsynonymous variants and 5 synonymous variants for each domain/non-domain region. The neutrality index was calculated as the ratio of nonsynonymous polymorphisms to fixations divided by the ratio of synonymous polymorphisms to fixations <a href="#pone.0001113-Rand1">[10]</a>. One count was added to each cell in the 2x2 matrix in order to calculate NI in case any cell contained a zero. This procedure makes the test more conservative as adding one to each cell reduces the power to reject neutrality. <a href="#pone.0001113.s002">Table S2</a> contains all counts of polymorphic and fixed variants used in analyses.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001113"></div><a name="pone.0001113.s001" id="pone.0001113.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001113.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">PROSITE domain coordinates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001113.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.47 MB TXT)</p>
<a name="pone.0001113.s002" id="pone.0001113.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001113.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Counts of polymorphic and fixed sites for (1) the portion of each gene in protein domains and (2) the remainder of the protein for each gene.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001113.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.39 MB TXT)</p>
<a name="pone.0001113.s003" id="pone.0001113.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001113.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Estimates of polymorphism and divergence for (1) the portion of each gene in protein domains and (2) the remainder of the protein for each gene.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001113.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.10 MB TXT)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DB AH. Analyzed the data: AH. Contributed reagents/materials/analysis tools: AH. Wrote the paper: AH. Other: Edited the paper: DB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001113-Begun1" id="pone.0001113-Begun1"></a>Begun DJ, Holloway AK, Stevens KS, Hillier LW, Poh Y, et al.  (2007) Population genomics: whole-genome analysis of polymorphism and divergence in <em>Drosophila simulans</em>. PLoS Biol.  5: e310.  <ul class="find" data-citedArticleID="1012286" data-doi="10.1371/journal.pbio.0050310"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0050310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+genomics%3A+whole-genome+analysis+of+polymorphism+and+divergence+in+Drosophila+simulans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+genomics%3A+whole-genome+analysis+of+polymorphism+and+divergence+in+Drosophila+simulans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001113-EyreWalker1" id="pone.0001113-EyreWalker1"></a>Eyre-Walker A (2006) The genomic rate of adaptive evolution. Trends Ecol Evol  21: 569–575.  <ul class="find" data-citedArticleID="1012294" data-doi="10.1016/j.tree.2006.06.015"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.06.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomic+rate+of+adaptive+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomic+rate+of+adaptive+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001113-Bierne1" id="pone.0001113-Bierne1"></a>Bierne N, Eyre-Walker A (2004) The genomic rate of adaptive amino-acid substitution in Drosophila. Mol Biol Evol  21: 1350–1360.  <ul class="find" data-citedArticleID="1012290" data-doi="10.1093/molbev/msh134"><li><a href="http://dx.doi.org/10.1093/molbev/msh134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomic+rate+of+adaptive+amino-acid+substitution+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomic+rate+of+adaptive+amino-acid+substitution+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001113-Welch1" id="pone.0001113-Welch1"></a>Welch JJ (2006) Estimating the genomewide rate of adaptive protein evolution in Drosophila. Genetics  173: 821–837.  <ul class="find" data-citedArticleID="1012310" data-doi="10.1534/genetics.106.056911"><li><a href="http://dx.doi.org/10.1534/genetics.106.056911" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+genomewide+rate+of+adaptive+protein+evolution+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+genomewide+rate+of+adaptive+protein+evolution+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001113-Begun2" id="pone.0001113-Begun2"></a>Begun DJ, Whitley P (2000) Adaptive evolution of relish, a Drosophila NF-kappaB/IkappaB protein. Genetics  154: 1231–1238.  <ul class="find" data-citedArticleID="1012288"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Adaptive+evolution+of+relish%2C+a+Drosophila+NF-kappaB%2FIkappaB+protein.&amp;auth=&amp;atitle=Adaptive+evolution+of+relish%2C+a+Drosophila+NF-kappaB%2FIkappaB+protein." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+relish%2C+a+Drosophila+NF-kappaB%2FIkappaB+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+relish%2C+a+Drosophila+NF-kappaB%2FIkappaB+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001113-Cooper1" id="pone.0001113-Cooper1"></a>Cooper JL, Henikoff S (2004) Adaptive evolution of the histone fold domain in centromeric histones. Mol Biol Evol  21: 1712–1718.  <ul class="find" data-citedArticleID="1012292" data-doi="10.1093/molbev/msh179"><li><a href="http://dx.doi.org/10.1093/molbev/msh179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+the+histone+fold+domain+in+centromeric+histones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+the+histone+fold+domain+in+centromeric+histones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001113-Schmidt1" id="pone.0001113-Schmidt1"></a>Schmidt PS, Duvernell DD, Eanes WF (2000) Adaptive evolution of a candidate gene for aging in Drosophila. Proc Natl Acad Sci U S A  97: 10861–10865.  <ul class="find" data-citedArticleID="1012304" data-doi="10.1073/pnas.190338897"><li><a href="http://dx.doi.org/10.1073/pnas.190338897" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+a+candidate+gene+for+aging+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+a+candidate+gene+for+aging+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001113-Hulo1" id="pone.0001113-Hulo1"></a>Hulo N, Bairoch A, Bulliard V, Cerutti L, De Castro E, et al.  (2006) The PROSITE database. Nucleic Acids Res  34: D227–D230.  <ul class="find" data-citedArticleID="1012296" data-doi="10.1093/nar/gkj063"><li><a href="http://dx.doi.org/10.1093/nar/gkj063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PROSITE+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PROSITE+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001113-McDonald1" id="pone.0001113-McDonald1"></a>McDonald JH, Kreitman M (1991) Adaptive protein evolution at the Adh locus in <em>Drosophila</em>. Nature  351: 652–654.  <ul class="find" data-citedArticleID="1012298" data-doi="10.1038/351652a0"><li><a href="http://dx.doi.org/10.1038/351652a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+protein+evolution+at+the+Adh+locus+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+protein+evolution+at+the+Adh+locus+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001113-Rand1" id="pone.0001113-Rand1"></a>Rand DM, Kann LM (1996) Excess amino acid polymorphism in mitochondrial DNA: contrasts among genes from Drosophila, mice, and humans. Mol Biol Evol  13: 735–748.  <ul class="find" data-citedArticleID="1012302" data-doi="10.1093/oxfordjournals.molbev.a025634"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excess+amino+acid+polymorphism+in+mitochondrial+DNA%3A+contrasts+among+genes+from+Drosophila%2C+mice%2C+and+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Excess+amino+acid+polymorphism+in+mitochondrial+DNA%3A+contrasts+among+genes+from+Drosophila%2C+mice%2C+and+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001113-Sigrist1" id="pone.0001113-Sigrist1"></a>Sigrist CJA, Cerutti L, Hulo N, Gattiker A, Falquet L, et al.  (2002) PROSITE: a documented database using patterns and profiles as motif descriptors. Brief Bioinform  3: 265–274.  <ul class="find" data-citedArticleID="1012306" data-doi="10.1093/bib/3.3.265"><li><a href="http://dx.doi.org/10.1093/bib/3.3.265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PROSITE%3A+a+documented+database+using+patterns+and+profiles+as+motif+descriptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PROSITE%3A+a+documented+database+using+patterns+and+profiles+as+motif+descriptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001113-Nei1" id="pone.0001113-Nei1"></a>Nei M, Gojobori T (1986) Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Mol Biol Evol  3: 418–426.  <ul class="find" data-citedArticleID="1012300"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.&amp;auth=&amp;atitle=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simple+methods+for+estimating+the+numbers+of+synonymous+and+nonsynonymous+nucleotide+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001113-Yang1" id="pone.0001113-Yang1"></a>Yang Z, Goldman N, Friday A (1994) Comparison of models for nucleotide substitution used in maximum-likelihood phylogenetic estimation. Mol Biol Evol  11: 316–324.  <ul class="find" data-citedArticleID="1012312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation.&amp;auth=&amp;atitle=Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+models+for+nucleotide+substitution+used+in+maximum-likelihood+phylogenetic+estimation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001113-Stajich1" id="pone.0001113-Stajich1"></a>Stajich JE, Block D, Boulez K, Brenner SE, Chervitz SA, et al.  (2002) The Bioperl toolkit: Perl modules for the life sciences. Genome Res  12: 1611–1618.  <ul class="find" data-citedArticleID="1012308" data-doi="10.1101/gr.361602"><li><a href="http://dx.doi.org/10.1101/gr.361602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bioperl+toolkit%3A+Perl+modules+for+the+life+sciences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bioperl+toolkit%3A+Perl+modules+for+the+life+sciences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.XML" value="34378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.PDF" value="155384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.g001.PNG_L" value="167958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.g001.PNG_M" value="137076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.g001.PNG_S" value="11407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.g001.TIF" value="980626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.g001.PNG_I" value="48863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.t001.PNG_L" value="59297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.t001.PNG_M" value="79336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.t001.PNG_S" value="12541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.t001.TIF" value="235556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.t001.PNG_I" value="27879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.s001.TXT" value="471587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.s002.TXT" value="390330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001113.s003.TXT" value="1096617"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001113&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001113" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001113&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001113&volume=&issue=&title=Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans&author_name=Alisha%20K.%20Holloway%2C%20David%20J.%20Begun&start_page=1&end_page=4" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001113" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113&amp;t=Rampant%20Adaptive%20Evolution%20in%20Regions%20of%20Proteins%20with%20Unknown%20Function%20in%20Drosophila%20simulans" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113&title=Rampant%20Adaptive%20Evolution%20in%20Regions%20of%20Proteins%20with%20Unknown%20Function%20in%20Drosophila%20simulans&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113&amp;title=Rampant%20Adaptive%20Evolution%20in%20Regions%20of%20Proteins%20with%20Unknown%20Function%20in%20Drosophila%20simulans" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001113&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Rampant Adaptive Evolution in Regions of Proteins with Unknown Function in Drosophila simulans';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001113';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Rampant%20Adaptive%20Evolution%20in%20Regions%20of%20Proteins%20with%20Unknown%20Function%20in%20Drosophila%20simulans http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001113" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001113" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="25060"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila%22" title="Search for articles in the subject area:'Drosophila'"><div class="flagText">Drosophila</div></a>
              <div data-categoryid="42801" data-articleid="25060"
                   data-categoryname="Drosophila"
                   class="flagImage" title="Flag 'Drosophila' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drosophila+melanogaster%22" title="Search for articles in the subject area:'Drosophila melanogaster'"><div class="flagText">Drosophila melanogaster</div></a>
              <div data-categoryid="32461" data-articleid="25060"
                   data-categoryname="Drosophila melanogaster"
                   class="flagImage" title="Flag 'Drosophila melanogaster' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+adaptation%22" title="Search for articles in the subject area:'Evolutionary adaptation'"><div class="flagText">Evolutionary adaptation</div></a>
              <div data-categoryid="17795" data-articleid="25060"
                   data-categoryname="Evolutionary adaptation"
                   class="flagImage" title="Flag 'Evolutionary adaptation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25060"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="25060"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="25060"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+domains%22" title="Search for articles in the subject area:'Protein domains'"><div class="flagText">Protein domains</div></a>
              <div data-categoryid="40153" data-articleid="25060"
                   data-categoryname="Protein domains"
                   class="flagImage" title="Flag 'Protein domains' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6229'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3289'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9795&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
