

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Model Structure of Human APOBEC3G</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000378"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000378" />

    <meta name="citation_title" content="Model Structure of Human APOBEC3G"/>
    <meta itemprop="name" content="Model Structure of Human APOBEC3G"/>

      <meta name="citation_author" content="Kun-Lin Zhang"/>
            <meta name="citation_author_institution" content="Institute of Microbiology, University Hospital Center, University of Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Bastien Mangeat"/>
            <meta name="citation_author_institution" content="Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Millan Ortiz"/>
            <meta name="citation_author_institution" content="Institute of Microbiology, University Hospital Center, University of Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Vincent Zoete"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Didier Trono"/>
            <meta name="citation_author_institution" content="Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Amalio Telenti"/>
            <meta name="citation_author_institution" content="Institute of Microbiology, University Hospital Center, University of Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Olivier Michielin"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland"/>

    <meta name="citation_date" content="2007/4/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000378.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e378"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Role and mechanism of action of the APOBEC3 family of antiretroviral resistance factors.; citation_author=BR Cullen; citation_journal_title=J Virol; citation_volume=80; citation_number=1; citation_pages=1067-1076; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Innate cellular defenses of APOBEC3 cytidine deaminases and viral counter-defenses.; citation_author=X-F Yu; citation_journal_title=Curr Opin HIV/AIDS; citation_volume=1; citation_number=2; citation_pages=187-193; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dancin' deaminase.; citation_author=RS Harris; citation_author=H Matsuo; citation_journal_title=Nat Struct Mol Biol; citation_volume=13; citation_number=3; citation_pages=380-381; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation of a human gene that inhibits HIV-1 infection and is suppressed by the viral Vif protein.; citation_author=AM Sheehy; citation_author=NC Gaddis; citation_author=JD Choi; citation_author=MH Malim; citation_journal_title=Nature; citation_volume=418; citation_number=4; citation_pages=646-650; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A single amino acid determinant governs the species-specific sensitivity of APOBEC3G to Vif action.; citation_author=B Mangeat; citation_author=P Turelli; citation_author=S Liao; citation_author=D Trono; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=5; citation_pages=14481-14483; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A single amino acid difference in the host APOBEC3G protein controls the primate species specificity of HIV type 1 virion infectivity factor.; citation_author=HP Bogerd; citation_author=BP Doehle; citation_author=HL Wiegand; citation_author=BR Cullen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=6; citation_pages=3770-3774; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A single amino acid of APOBEC3G controls its species-specific interaction with virion infectivity factor (Vif).; citation_author=B Schrofelbauer; citation_author=D Chen; citation_author=NR Landau; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=7; citation_pages=3927-3932; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A single amino acid substitution in human APOBEC3G antiretroviral enzyme confers resistance to HIV-1 virion infectivity factor-induced depletion.; citation_author=H Xu; citation_author=ES Svarovskaia; citation_author=R Barr; citation_author=Y Zhang; citation_author=MA Khan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=8; citation_pages=5652-5657; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complementary function of the two catalytic domains of APOBEC3G.; citation_author=F Navarro; citation_author=B Bollman; citation_author=H Chen; citation_author=R Konig; citation_author=Q Yu; citation_journal_title=Virology; citation_volume=333; citation_number=9; citation_pages=374-386; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Amino-terminal region of the human immunodeficiency virus type 1 nucleocapsid is required for human APOBEC3G packaging.; citation_author=K Luo; citation_author=B Liu; citation_author=Z Xiao; citation_author=Y Yu; citation_author=X Yu; citation_journal_title=J Virol; citation_volume=78; citation_number=10; citation_pages=11841-11852; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Antiviral function of APOBEC3G can be dissociated from cytidine deaminase activity.; citation_author=EN Newman; citation_author=RK Holmes; citation_author=HM Craig; citation_author=KC Klein; citation_author=JR Lingappa; citation_journal_title=Curr Biol; citation_volume=15; citation_number=11; citation_pages=166-170; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular modelling and the biosynthesis of apolipoprotein B containing lipoproteins.; citation_author=J Scott; citation_author=N Navaratnam; citation_author=C Carter; citation_journal_title=Atherosclerosis; citation_volume=141 Suppl 1; citation_number=12; citation_pages=S17-S24; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli cytidine deaminase provides a molecular model for ApoB RNA editing and a mechanism for RNA substrate recognition.; citation_author=N Navaratnam; citation_author=T Fujino; citation_author=J Bayliss; citation_author=A Jarmuz; citation_author=A How; citation_journal_title=J Mol Biol; citation_volume=275; citation_number=13; citation_pages=695-714; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The structure of a yeast RNA-editing deaminase provides insight into the fold and function of activation-induced deaminase and APOBEC-1.; citation_author=K Xie; citation_author=MP Sowden; citation_author=GS Dance; citation_author=AT Torelli; citation_author=HC Smith; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=14; citation_pages=8114-8119; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Domain organization of activation-induced cytidine deaminase.; citation_author=J Zaim; citation_author=AM Kierzek; citation_journal_title=Nat Immunol; citation_volume=4; citation_number=15; citation_pages=1153; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=AID mutant analyses indicate requirement for class-switch-specific cofactors.; citation_author=VT Ta; citation_author=H Nagaoka; citation_author=N Catalan; citation_author=A Durandy; citation_author=A Fischer; citation_journal_title=Nat Immunol; citation_volume=4; citation_number=16; citation_pages=843-848; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Messenger RNA editing in mammals: new members of the APOBEC family seeking roles in the family business.; citation_author=JE Wedekind; citation_author=GS Dance; citation_author=MP Sowden; citation_author=HC Smith; citation_journal_title=Trends Genet; citation_volume=19; citation_number=17; citation_pages=207-216; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytidine deamination and resistance to retroviral infection: towards a structural understanding of the APOBEC proteins.; citation_author=H Huthoff; citation_author=MH Malim; citation_journal_title=Virology; citation_volume=334; citation_number=18; citation_pages=147-153; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The APOBEC-2 crystal structure and functional implications for the deaminase AID.; citation_author=C Prochnow; citation_author=R Bransteitter; citation_author=MG Klein; citation_author=MF Goodman; citation_author=XS Chen; citation_journal_title=Nature; citation_number=19; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of amino acid residues in APOBEC3G required for regulation by HIV-1 Vif and virion encapsidation.; citation_author=H Huthoff; citation_author=MH Malim; citation_journal_title=J Virol PMID; citation_number=20; citation_pages=17267497; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative protein modelling by satisfaction of spatial restraints.; citation_author=A Sali; citation_author=TL Blundell; citation_journal_title=J Mol Biol; citation_volume=234; citation_number=21; citation_pages=779-815; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Comparative protein structure modeling of genes and genomes.; citation_author=MA Marti-Renom; citation_author=AC Stuart; citation_author=A Fiser; citation_author=R Sanchez; citation_author=F Melo; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=29; citation_number=22; citation_pages=291-325; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Modeling of loops in protein structures.; citation_author=A Fiser; citation_author=RK Do; citation_author=A Sali; citation_journal_title=Protein Sci; citation_volume=9; citation_number=23; citation_pages=1753-1773; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features.; citation_author=W Kabsch; citation_author=C Sander; citation_journal_title=Biopolymers; citation_volume=22; citation_number=24; citation_pages=2577-2637; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=The PSIPRED protein structure prediction server.; citation_author=LJ McGuffin; citation_author=K Bryson; citation_author=DT Jones; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=25; citation_pages=404-405; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Protein structure prediction servers at University College London.; citation_author=K Bryson; citation_author=LJ McGuffin; citation_author=RL Marsden; citation_author=JJ Ward; citation_author=JS Sodhi; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=26; citation_pages=W36-W38; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein secondary structure prediction based on position-specific scoring matrices.; citation_author=DT Jones; citation_journal_title=J Mol Biol; citation_volume=292; citation_number=27; citation_pages=195-202; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The Jalview Java alignment editor.; citation_author=M Clamp; citation_author=J Cuff; citation_author=SM Searle; citation_author=GJ Barton; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=28; citation_pages=426-427; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Assessing protein structures with a non-local atomic interaction energy.; citation_author=F Melo; citation_author=E Feytmans; citation_journal_title=J Mol Biol; citation_volume=277; citation_number=29; citation_pages=1141-1152; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Novel knowledge-based mean force potential at atomic level.; citation_author=F Melo; citation_author=E Feytmans; citation_journal_title=J Mol Biol; citation_volume=267; citation_number=30; citation_pages=207-222; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=ANOLEA: a www server to assess protein structures.; citation_author=F Melo; citation_author=D Devos; citation_author=E Depiereux; citation_author=E Feytmans; citation_journal_title=Proc Int Conf Intell Syst Mol Biol; citation_volume=5; citation_number=31; citation_pages=187-190; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=CHARMM: A Program for Macromolecular Energy, Minimization, and Dynamics Calculations.; citation_author=BR Brooks; citation_author=RE Bruccoleri; citation_author=BD Olafson; citation_author=DJ States; citation_author=S Swaminathan; citation_journal_title=J Comp Chem; citation_volume=4; citation_number=32; citation_pages=187-217; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=All-atom empirical potential for molecular modeling and dynamics studies of proteins.; citation_author=AD MacKerell; citation_author=D Bashford; citation_author=M Bellott; citation_author=RL Dunbrack; citation_author=JD Evaseck; citation_journal_title=J Phys Chem B; citation_volume=102; citation_number=33; citation_pages=3586-3616; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Predicting changes in the stability of proteins and protein complexes: a study of more than 1000 mutations.; citation_author=R Guerois; citation_author=JE Nielsen; citation_author=L Serrano; citation_journal_title=J Mol Biol; citation_volume=320; citation_number=34; citation_pages=369-387; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Reduced surface: an efficient way to compute molecular surfaces.; citation_author=MF Sanner; citation_author=AJ Olson; citation_author=JC Spehner; citation_journal_title=Biopolymers; citation_volume=38; citation_number=35; citation_pages=305-320; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=UCSF Chimera–a visualization system for exploratory research and analysis.; citation_author=EF Pettersen; citation_author=TD Goddard; citation_author=CC Huang; citation_author=GS Couch; citation_author=DM Greenblatt; citation_journal_title=J Comput Chem; citation_volume=25; citation_number=36; citation_pages=1605-1612; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Electrostatics and diffusion of molecules in solution: simulations with the University of Houston Brownian Dynamics program.; citation_author=ME Davis; citation_author=JD Madura; citation_author=BA Luty; citation_author=JA McCammon; citation_journal_title=Comp Phys Comm; citation_volume=62; citation_number=37; citation_pages=187-197; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evolutionary origins of apoB mRNA editing: catalysis by a cytidine deaminase that has acquired a novel RNA-binding motif at its active site.; citation_author=N Navaratnam; citation_author=S Bhattacharya; citation_author=T Fujino; citation_author=D Patel; citation_author=AL Jarmuz; citation_journal_title=Cell; citation_volume=81; citation_number=38; citation_pages=187-195; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Patterns of evolution of host proteins involved in retroviral pathogenesis.; citation_author=M Ortiz; citation_author=G Bleiber; citation_author=R Martinez; citation_author=H Kaessmann; citation_author=A Telenti; citation_journal_title=Retrovirology; citation_volume=3; citation_number=39; citation_pages=11; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The interaction between HIV-1 Gag and APOBEC3G.; citation_author=S Cen; citation_author=F Guo; citation_author=M Niu; citation_author=J Saadatmand; citation_author=J Deflassieux; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=40; citation_pages=33177-33184; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Specific packaging of APOBEC3G into HIV-1 virions is mediated by the nucleocapsid domain of the gag polyprotein precursor.; citation_author=A Schafer; citation_author=HP Bogerd; citation_author=BR Cullen; citation_journal_title=Virology; citation_volume=328; citation_number=41; citation_pages=163-168; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human apolipoprotein B mRNA-editing enzyme-catalytic polypeptide-like 3G (APOBEC3G) is incorporated into HIV-1 virions through interactions with viral and nonviral RNAs.; citation_author=ES Svarovskaia; citation_author=H Xu; citation_author=JL Mbisa; citation_author=R Barr; citation_author=RJ Gorelick; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=42; citation_pages=35822-35828; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=APOBEC3G incorporation into human immunodeficiency virus type 1 particles.; citation_author=V Zennou; citation_author=D Perez-Caballero; citation_author=H Gottlinger; citation_author=PD Bieniasz; citation_journal_title=J Virol; citation_volume=78; citation_number=43; citation_pages=12058-12061; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Vif protein of HIV triggers degradation of the human antiretroviral DNA deaminase APOBEC3G.; citation_author=SG Conticello; citation_author=RS Harris; citation_author=MS Neuberger; citation_journal_title=Curr Biol; citation_volume=13; citation_number=44; citation_pages=2009-2013; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=All three variable regions of the TRIM5 alpha B30.2 domain can contribute to the specificity of the retrovirus restriction.; citation_author=S Ohkura; citation_author=MW Yap; citation_author=T Sheldon; citation_author=JP Stoye; citation_journal_title=J Virol; citation_volume=80; citation_number=45; citation_pages=8554-8565; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Model Structure of Human APOBEC3G"/>
    <meta name="twitter:description" content="BackgroundAPOBEC3G (apolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like 3G) has antiretroviral activity associated with the hypermutation of viral DNA through cytosine deamination. APOBEC3G has two cytosine deaminase (CDA) domains; the catalytically inactive amino-terminal domain of APOBEC3G (N-CDA) carries the Vif interaction domain. There is no 3-D structure of APOBEC3G solved by X-ray or nuclear magnetic resonance.Methodology/Principal FindingsWe predicted the structure of human APOBEC3G based on the crystal structure of APOBEC2. To assess the model structure, we evaluated 48 mutants of APOBEC3G N-CDA that identify novel variants altering &Delta;Vif HIV-1 infectivity and packaging of APOBEC3G. Results indicated that the key residue D128 is exposed at the surface of the model, with a negative local electrostatic potential. Mutation D128K changes the sign of that local potential. In addition, two novel functionally relevant residues that result in defective APOBEC3G encapsidation, R122 and W127, cluster at the surface.Conclusions/SignificanceThe structure model identifies a cluster of residues important for packaging of APOBEC3G into virions, and may serve to guide functional analysis of APOBEC3G."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000378.g004"/>

  <meta property="og:title" content="Model Structure of Human APOBEC3G" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=367'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3227'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9964&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000378">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000378" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000378&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Model Structure of Human APOBEC3G
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kun-Lin Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Microbiology, University Hospital Center, University of Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bastien Mangeat, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Millan Ortiz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Microbiology, University Hospital Center, University of Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vincent Zoete, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Swiss Institute of Bioinformatics, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Didier Trono, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amalio Telenti
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:amalio.telenti@chuv.ch">amalio.telenti@chuv.ch</a> (AT); <a href="mailto:olivier.michielin@unil.ch">olivier.michielin@unil.ch</a> (OM)</p>

                <p>Affiliation:
                  Institute of Microbiology, University Hospital Center, University of Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olivier Michielin
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:amalio.telenti@chuv.ch">amalio.telenti@chuv.ch</a> (AT); <a href="mailto:olivier.michielin@unil.ch">olivier.michielin@unil.ch</a> (OM)</p>

                <p>Affiliation:
                  Swiss Institute of Bioinformatics, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000378</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000378-g001" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000378-t001" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000378-g002" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000378-g003" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000378-g004" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000378">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000378" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>APOBEC3G (apolipoprotein B mRNA-editing enzyme, catalytic polypeptide-like 3G) has antiretroviral activity associated with the hypermutation of viral DNA through cytosine deamination. APOBEC3G has two cytosine deaminase (CDA) domains; the catalytically inactive amino-terminal domain of APOBEC3G (N-CDA) carries the Vif interaction domain. There is no 3-D structure of APOBEC3G solved by X-ray or nuclear magnetic resonance.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We predicted the structure of human APOBEC3G based on the crystal structure of APOBEC2. To assess the model structure, we evaluated 48 mutants of APOBEC3G N-CDA that identify novel variants altering ΔVif HIV-1 infectivity and packaging of APOBEC3G. Results indicated that the key residue D128 is exposed at the surface of the model, with a negative local electrostatic potential. Mutation D128K changes the sign of that local potential. In addition, two novel functionally relevant residues that result in defective APOBEC3G encapsidation, R122 and W127, cluster at the surface.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The structure model identifies a cluster of residues important for packaging of APOBEC3G into virions, and may serve to guide functional analysis of APOBEC3G.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhang K-L, Mangeat B, Ortiz M, Zoete V, Trono D, et al.  (2007) Model Structure of Human APOBEC3G. PLoS ONE 2(4):
          e378.
            doi:10.1371/journal.pone.0000378</p><p><strong>Academic Editor: </strong>Alejandro Aballay, Duke University Medical Center, United States of America</p><p><strong>Received:</strong> February 9, 2007; <strong>Accepted:</strong> March 26, 2007; <strong>Published:</strong> April 18, 2007</p><p><strong>Copyright:</strong> © 2007 Zhang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Supported by Swiss National Science Foundation grant no. 310000-110012/1 (to A.T.), SCORE funds 3232B0-103172 and 3200B0-103173 (to O.M.) and a grant for interdisciplinary research from the Faculty of Biology and Medicine of the University of Lausanne (to A.T. and O.M.). KZ received a postdoctoral award from the Foundation for advancement in Microbiology and Infectious Diseases. The funding agencies had no role in the design or interpretation of the study.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Primate APOBEC3G has antiretroviral activity associated with the hypermutation of viral DNA through cytosine deamination (for recent review see <a href="#pone.0000378-Cullen1">[1]</a>–<a href="#pone.0000378-Harris1">[3]</a>). Human APOBEC3G (huAPOBEC3G) fails to restrict HIV-1 due to the degradation imposed by the HIV-1 Vif <a href="#pone.0000378-Sheehy1">[4]</a>. In contrast, a number of primate APOBEC3G orthologs display activity against HIV-1 <a href="#pone.0000378-Mangeat1">[5]</a>–<a href="#pone.0000378-Xu1">[8]</a>. APOBEC3G has a duplicated catalytic deaminase domain (CDA); the amino-terminal domain (N-CDA) of APOBEC3G is required for viral encapsidation but not cytosine deamination <a href="#pone.0000378-Navarro1">[9]</a>–<a href="#pone.0000378-Newman1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There is no 3-D structure solved by X-ray or NMR nor an accurate model of APOBEC3G available. APOBEC3G relates to APOBEC family and AID (activation-induced deaminase) at the sequence level. Recent comparative modeling work for APOBEC-1 and AID <a href="#pone.0000378-Scott1">[12]</a>–<a href="#pone.0000378-Wedekind1">[17]</a> led to the proposition of a secondary structure alignment between APOBEC3G and cytidine deaminase <a href="#pone.0000378-Huthoff1">[18]</a>. The recent publishing of the crystal structure of APOBEC2 <a href="#pone.0000378-Prochnow1">[19]</a> provides the template to build a reliable model structure of APOBEC3G by theoretical methods.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In the present work, we model human APOBEC3G, with particular emphasis on the N-terminal domain. Using mutant data of the N-CDA, we mapped critical residues for packaging of APOBEC3G into viral particles on the new structure model of the huAPOBEC3G N-CDA. We completed the analysis by mapping N-CDA residues that are under positive evolutionary pressure in primate APOBEC3G.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>While this work was concluded, Huthoff and Malim provided a detailed molecular genetic analysis of the N-CDA region spanning amino acids residues 119 to 146 (Ref <a href="#pone.0000378-Huthoff2">[20]</a>). This analysis defined residues 124 to 127 as having a role in APOBEC3G packaging into HIV-1 virions, and residues 128 to 130 as crucial for the interaction with HIV-1 Vif. Our current results confirm and extend this work, and place the findings in a detailed structural model that may serve to advance rational drug design.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Target sequence and template for structural model</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The huAPOBEC3G sequence (residues 1-384; NCBI accession: NP_068594, see <a href="http://www.ncbi.nih.gov/">http://www.ncbi.nih.gov/</a>) was defined as target sequence. The newly crystallized Human APOBEC2 dimer <a href="#pone.0000378-Prochnow1">[19]</a> (PDB ID 2NYT obtained from Xiaojiang S. Chen, see <a href="http://www.rcsb.org/pdb/Welcome.do">http://www.rcsb.org/pdb/Welcome.do</a>) served as template.</p>


<h4>Target-template alignment</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We used the align2d function of MODELLER program (<a href="http://salilab.org/modeller/">http://salilab.org/modeller/</a>) <a href="#pone.0000378-Sali1">[21]</a>–<a href="#pone.0000378-Fiser1">[23]</a>, to align huAPOBEC3G N-CDA sequence (residues 1-194) and huAPOBEC3G C-CDA sequence (residues 195-384) to the huAPOBEC2 dimer. Additional assessment of the target-template alignment compared structurally determined (DSSP, <a href="http://bioweb.pasteur.fr/seqanal/interfaces/dssp-simple.html">http://bioweb.pasteur.fr/seqanal/interfa​ces/dssp-simple.html</a>) <a href="#pone.0000378-Kabsch1">[24]</a> and predicted (PSIPRED, <a href="http://bioinf.cs.ucl.ac.uk/psipred/">http://bioinf.cs.ucl.ac.uk/psipred/</a>) <a href="#pone.0000378-McGuffin1">[25]</a>–<a href="#pone.0000378-Jones1">[27]</a> secondary structures. The alignment was analyzed and viewed by Jalview (<a href="http://www.jalview.org/">http://www.jalview.org/</a>) <a href="#pone.0000378-Clamp1">[28]</a>.</p>


<h4>Model building, evaluation and mapping of key residues</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The target-template alignment was used to build the model by satisfaction of spatial restraints. The ANOLEA program (<a href="http://swissmodel.expasy.org/anolea/">http://swissmodel.expasy.org/anolea/</a>) <a href="#pone.0000378-Melo1">[29]</a>–<a href="#pone.0000378-Melo3">[31]</a>, that estimates the folding free energy of each residue of a protein chain to assess the quality of the predicted structure, was used to score all the models, using the default 5 residue window averaging.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The geometry of the active site/pseudo-active site was based on the corresponding homologous active site regions of the huAPOBEC2 template. The main interaction of anti-parallel β2-β2′, between huAPOBEC3G N-CDA and C-CDA was defined by restraining segments 48-56 and 235-243 as anti-parallel β-sheet, based on the huAPOBEC2 dimer. For the chain connection between N-CDA and C-CDA, the first 20 amino acid residues of C-CDA were used as a linker and modeled ab initio using the loop routine of the MODELER program.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Generation of 100 models allowed selection of the best model candidate based on the global ANOLEA score. Final refinement for alignment of gap regions was performed by generating 100 additional models, followed by selection of the best model on the basis of the ANOLEA score. The final model was energy-minimized using the CHARMM program (<a href="http://www.charmm.org/">http://www.charmm.org/</a>) <a href="#pone.0000378-Brooks1">[32]</a> and the CHARMM22 all atom force field <a href="#pone.0000378-MacKerell1">[33]</a>. The minimization consisted of 200 steps of steepest descent using a dielectric constant of 1 and the Generalized Born GB-MV2 implicit solvent model without cutoff for the solvation free energy. Model evaluation was based on ANOLEA with window 5 averaging, as described above.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The CHARMM program was used to calculate the solvent accessible surface area of the final model. Additionally, using the FoldX program (<a href="http://foldx.embl.de/">http://foldx.embl.de/</a>) <a href="#pone.0000378-Guerois1">[34]</a>, in silico alanine-scanning was performed by mutating each residue and calculated the change of folding energy. Molecular surface visualization was done using Chimera (<a href="http://www.cgl.ucsf.edu/chimera/">http://www.cgl.ucsf.edu/chimera/</a>) <a href="#pone.0000378-Sanner1">[35]</a>, <a href="#pone.0000378-Pettersen1">[36]</a>. The surface was color-coded according to the Poisson-Boltzmann potential calculated by the UHBD program <a href="#pone.0000378-Davis1">[37]</a>.</p>


<h4>Mutation analysis, constructs, viral production and titration</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The plasmid expressing a hemagglutinin (HA)-tagged form of APOBEC3G was a kind gift from M. Malim. A series of APOBEC3G alanine and specific mutants was constructed with the QuickChange Mutagenesis kit (Stratagene). The collection was already constituted, and it was not defined on the basis of the new structural data. HIV-1 particles were produced by transient transfection of 293T cells with Fugene (Roche) of a wild-type or of a Vif-defective HIV-1 proviral clone. Viral titers were determined in single-round infectivity assays by applying filtered supernatant from producer cells on HeLa-CD4-LTRLacZ indicator cells. Virion infectivity was derived by dividing the infectious titer by the amount of physical particles.</p>


<h4>Packaging assay and Western Blots</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>HIV-1 particles were produced by transient transfection of 293T cells with Fugene (Roche). 1 ml of virus was then spun in Eppendorf tubes at 13'000rpm in a microfuge at 4° for 90 minutes, without sucrose cushion. Pellets were resuspended in PBS 1% Triton, and the virion amount was measured by a standard RT assay. Normalized amount of virions were then loaded on standard Laemmli protein gels to perform Western Blots. Cell extracts were obtained through a standard RIPA extraction procedure. The HA tag was detected with the mouse hrp-coupled anti-HA 3F10 antibody (Roche). PCNA (proliferating cell nuclear antigen) was detected with the mouse Ab-1 antibody (Oncogene Science). The HIV-1 capsid was detected with the murine anti-p24 antibody produced from the AIDS Research and Reference Reagent Program #183-H12-5C.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The huAPOBEC2 tetramer is composed of two outer monomers and two middle monomers. The outer and middle monomers share the same sequence but a slightly different structure in the region of residues 57-68. For an outer monomer, this region is a hairpin-loop; but for a middle monomer this region is a loop with residue E60 coordinating with Zn<sup>2+</sup> <a href="#pone.0000378-Prochnow1">[19]</a>, <a href="#pone-0000378-g001"><strong>Fig. 1A</strong></a>. We modeled the huAPOBEC3G N-CDA based on the outer monomer of APOBEC2 to resolve the structure between residues 22 and 33 of huAPOBEC3G. The outer monomer of APOBEC2 is more suitable as template because the middle monomer coordinates the Zn<sup>2+</sup> with E60, a residue not present at the corresponding position in huAPOBEC3G N-CDA. The corresponding region of huAPOBEC3G C-CDA was modeled ab initio. The target-template alignment generated by MODELLER agreed with the secondary structure alignment (<a href="#pone.0000378.s001">Supplementary Fig. S1A</a>). The resulting huAPOBEC3G model is shown in <a href="#pone-0000378-g001"><strong>Fig. 1B</strong></a>. The structural pattern of this model is very similar to that of the huAPOBEC2 dimer. The final model has a good ANOLEA score profile (indicating reliability of the structure prediction) with a pattern comparable to that of huAPOBEC2 dimer despite the low sequence identity (27%), <a href="#pone.0000378.s001"><strong>Supplementary Fig. S1</strong></a>.</p>
<div class="figure" id="pone-0000378-g001"><div class="img"><a name="pone-0000378-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Ribbon view of the huAPOBEC2 dimer and the model of huAPOBEC3G.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Panel A. The APOBEC2 “homo”-dimer. Panel B. The huAPOBEC3G model underscoring the six active and pseudo-active site residues in blue, the two zinc ions, and the position of the residue D128, key in the interaction with HIV-1 Vif.</p>
<span>doi:10.1371/journal.pone.0000378.g001</span></div>
<h4>Mutation analysis and mapping of key functional residues</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Mutation analysis interrogated 48 of 194 (25%) residues of the huAPOBEC3G N-CDA <a href="#pone-0000378-t001"><strong>(Table 1)</strong></a>. Mutation of the conserved residues constituting the N-CDA Zn<sup>2+</sup> coordination domain H65, C97 and C100 prevented or reduced encapsidation as previously reported <a href="#pone.0000378-Navarro1">[9]</a>. Mutation of residues F70 and Y91, shown to mediate RNA binding <a href="#pone.0000378-Navaratnam2">[38]</a>, was associated with reduced APOBEC3G encapsidation, as previously reported <a href="#pone.0000378-Navarro1">[9]</a>. Mutation of the pseudo-catalytic site residue E67 (E67Q), resulted in poor protein expression, which limited functional evaluation; E67A was previously reported to reduce the rates of APOBEC3G encapsidation into HIV1 virions <a href="#pone.0000378-Navarro1">[9]</a>. E67 and F70 are buried inside of the model; E67A and F70A may affect folding stability (in silico alanine-scanning). Y91 is a partially exposed at the surface.</p>
<div class="figure" id="pone-0000378-t001"><div class="img"><a name="pone-0000378-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000378.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000378.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mutation, antiviral activity, and surface exposure of residues of the huAPOBEC3G N-CDA.</span></strong></p><span>doi:10.1371/journal.pone.0000378.t001</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Amino acids substitutions at positions Y13, F17, F21, R29, T32, Y37, K40, S45, L49, L62, E85, Y86, W90, I92, S93, P96, M104, F107, L108, L116, T117, I118, L123, Y125, F126, D130, Y131, E133, L135, L138, M152, C160 and F164 did not result in changes in anti-viral activity. Mutations S28A, F74L, W94L, Y124A, Y154A resulted in reduced inhibition of ΔVif HIV1, while mutations R122A and W127L completely abolished this activity (<a href="#pone-0000378-t001"><strong>Table 1</strong></a>). The functionally defective phenotype of these mutants correlated with a failure to become encapsidated into ΔVif HIV-1 particles (<a href="#pone-0000378-g002"><strong>Fig. 2</strong></a>). R122, W127 and D128 form a cluster at the surface (<a href="#pone-0000378-g003"><strong>Fig. 3</strong></a>) and contribute to changes in surface charge or structure (<a href="#pone-0000378-g004"><strong>Fig. 4</strong></a>).</p>
<div class="figure" id="pone-0000378-g002"><div class="img"><a name="pone-0000378-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Infectivity, and packaging assay of novel defective huAPOBEC3G mutants.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Panel A. Δ Vif HIV-1 particles were produced in presence of different doses of wt APOBEC3G-HA (A3G), or with the highest dose of the R122A and W127L mutants of APOBEC3G-HA. The infectivity of these particles was determined by titration on P4.2 indicator cells. The doses d1 to d4 of APOBEC3G correspond respectively to a molar ratio of APOBEC3G-HA plasmid to virus plasmid of 0.6:1, 1.3:1, 2.7:1 and 5.5:1. The graph is made from one duplicate experiment, and is representative of a total of at least 4 independent duplicate experiments. Panel B. The presence of wt APOBEC3G-HA or the R122A and W127L mutants in ΔVif HIV-1 virions was assessed by western blot. The PCNA and capsid western blots serve to check for consistent loading of cellular and virion extracts, respectively. Representative of at least two independent experiments.</p>
<span>doi:10.1371/journal.pone.0000378.g002</span></div><div class="figure" id="pone-0000378-g003"><div class="img"><a name="pone-0000378-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Mapping of functional and evolutionary informative residues at the molecular surface of N-CDA.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Panel A, Mapping of functional residues. The color gradient (dark red&gt;red&gt;orange&gt;yellow) reflects the role of various mutations in abolishing anti-HIV activity (see <a href="#pone-0000378-t001">Table 1</a>) ranging from no antiviral activity (dark red) to normal activity (yellow). Panel B. Mapping of evolutionary informative residues at the molecular surface. Green color identifies amino acids under positive selective pressure in primate APOBEC3G. The positive selection cluster includes T98, K99, R102, D128, and P129. Blue color identifies the pseudo–active site.</p>
<span>doi:10.1371/journal.pone.0000378.g003</span></div><div class="figure" id="pone-0000378-g004"><div class="img"><a name="pone-0000378-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000378" data-uri="info:doi/10.1371/journal.pone.0000378.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000378.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000378.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000378.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000378.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Mapping potential to molecular surface.</span></strong></p><a id="article1.body1.sec3.sec1.fig3.caption1.p1" name="article1.body1.sec3.sec1.fig3.caption1.p1"></a><p>Zoomed in view on the functional surface region within the N-CDA. Panel A, wild-type huAPOBEC3G N-CDA. Panel B-D, effect of mutations R122A, W127L, and D128K on surface charge and shape. The red color represents negative potential, the blue color expresses positive potential, and the white color expresses zero potential.</p>
<span>doi:10.1371/journal.pone.0000378.g004</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Prior evolutionary analysis of primate APOBEC3G identified a number of residues under diversifying (positive) selection <a href="#pone.0000378-Ortiz1">[39]</a>. Most of the residues under positive selection pressure are exposed at the surface of the model. A cluster of residues under positive selection that includes T98, K99, R102, D128, and P129 overlaps with the cluster of functionally important residues around D128 (<a href="#pone-0000378-g003"><strong>Fig. 3</strong></a>).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>This work presents a model structure of huAPOBEC3G that captures information from the recently published APOBEC2 structure <a href="#pone.0000378-Prochnow1">[19]</a>. Critically, the APOBEC2 template provides a structural reference for the predicted extra helix α4, that carries residue D128–a residue that governs the virus-specific sensitivity of APOBEC3G to Vif-mediated inhibition <a href="#pone.0000378-Mangeat1">[5]</a>–<a href="#pone.0000378-Xu1">[8]</a>. Previously available CDA structures provided suboptimal templates for APOBEC3G N-CDA. This is not only due to the low sequence identity, but also because of the presence in APOBEC3G N-CDA of an extra α helix. In contrast, APOBEC2 has an extra α helix, resulting in a α1-β1-β2-α2-β3-α3-β4-<span class="underline">α4-β5-α5-α6</span> configuration, while CDA structures such as human cytidine deaminase have the pattern α1-β1-β2-α2-β3-α3-β4-<span class="underline">β5-α4-α5</span>. In addition, the direction of β4 and β5 are different: parallel in APOBEC2 but anti-parallel in cytidine deaminase. We had previously used the yeast Cytosine deaminase (PDB ID 1P6O, chain A) to address the position of the extra helix and the correct direction of β4 and β5.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The model emerging from this analysis allows speculation on various functionally relevant structural details of interest for the understanding of the Vif-APOBEC3G interaction and the process of APOBEC3G encapsidation into HIV-1 virions. In contrast with previous secondary and tertiary structure models, the current model locates the distinctive extra alpha helix on the same planar surface as the pseudo-active site. We evaluated extensive mutation data on this surface, that characterized two functionally relevant residues R122 and W127 that resulted in failure to inhibit infection by HIV-1/Δ<em>vif</em> due to lack of packaging of APOBEC3G into viral particles. Interaction of APOBEC3G with the NC-domain of HIV-1 Gag and non-specific RNA binding leads to its encapsidation into progeny virions <a href="#pone.0000378-Luo1">[10]</a>, <a href="#pone.0000378-Cen1">[40]</a>–<a href="#pone.0000378-Zennou1">[43]</a>. The structural model proposes a surface hot-spot domain that includes residues R122, W127 in proximity to the active site, and overlapping with a cluster of residues under positive selective pressure that includes D128 and P129.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Our results are consistent with the work of Huthoff and Malim <a href="#pone.0000378-Huthoff2">[20]</a>. While their work concentrated on the molecular genetic analysis of the 28-amino acid region between residues 119 and 146, our analysis interrogated 48 of the 194 residues of the N-CDA domain. This reflects the interest to identify additional motifs capable of interacting with HIV-1 Vif, or participating in packaging <a href="#pone.0000378-Huthoff2">[20]</a>, <a href="#pone.0000378-Conticello1">[44]</a>. Minor differences were observed between the two studies. We observed a defect in the packaging efficiency of R122A mutant protein–that extends the relevant motif to include R122-Y124-Y125-F126-W127. Regarding the motif affecting regulation by Vif, our analysis of D130 mutants (D130K and D130N) did not result in a Vif-resistant phenotype. This difference may reflect a greater dependence of the N-CDA and Vif for the negative charge in position 128 (Ref <a href="#pone.0000378-Huthoff2">[20]</a>). While not formally tested in the present study, the biological relevance of P129 is highlighted by its inclusion–together with D128-in a patch of residues under positive selective pressure in primates.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Inspection of surface modifications conferred by various mutations highlights the structural and/or charge differences in this region as the molecular basis for disruption of the APOBEC3G packaging into HIV-1 virions, and modification in the interaction with Vif. The detailed model structure presented here could serve to advance rational drug design. The quality of a homology model is strongly related to the sequence identity with the structural template. The current model, based on a sequence identity of 27%, should be satisfactory in its global fold, as supported by the good ANOLEA energy score profile (<a href="#pone.0000378.s001"><strong>Supplementary Fig. S1</strong></a>). From such a model, correct positioning of the protein backbone and orientation of the side chains can be expected, allowing reliable conclusion to be made in the design and interpretation of experimental mutation data. Whether or not the accuracy of such a model is sufficient to perform docking simulations is still an open question. However, the fact that critical residues obtained experimentally by mutation analysis do cluster in well-defined patches at the surface of the model argue that side chain packing is correct. In the latter case, docking simulations could be envisioned based on the current model. As recently shown for TRIM5α <a href="#pone.0000378-Ohkura1">[45]</a>, the model proposed may serve to guide further functional analysis of APOBEC3G.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000378"></div><a name="pone.0000378.s001" id="pone.0000378.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000378.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Panel A, Target-template alignment. Alpha helix are shown in red and beta sheets in green. The huAPOBEC3G N-CDA is aligned to the outer (O) monomer of APOBEC2, and the C-CDA to the middle (M) monomer. Panel B, ANOLEA scores. The three ANOLEA profiles and the secondary structure were aligned according to the target-template alignment. The ANOLEA profile excludes the uncharacterized linker (residues 195-214)</p>
<p class="siDoi">doi:10.1371/journal.pone.0000378.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.86 MB EPS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank X.S. Chen for APOBEC2 PDB data.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AT DT OM. Performed the experiments: KZ BM. Analyzed the data: AT DT MO KZ BM OM. Contributed reagents/materials/analysis tools: VZ OM. Wrote the paper: AT KZ OM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000378-Cullen1" id="pone.0000378-Cullen1"></a>Cullen BR (2006) Role and mechanism of action of the APOBEC3 family of antiretroviral resistance factors. J Virol  80: 1067–1076.  <ul class="find" data-citedArticleID="945146" data-doi="10.1128/jvi.80.3.1067-1076.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.3.1067-1076.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+and+mechanism+of+action+of+the+APOBEC3+family+of+antiretroviral+resistance+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+and+mechanism+of+action+of+the+APOBEC3+family+of+antiretroviral+resistance+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000378-Yu1" id="pone.0000378-Yu1"></a>Yu X-F (2006) Innate cellular defenses of APOBEC3 cytidine deaminases and viral counter-defenses. Curr Opin HIV/AIDS  1: 187–193.  <ul class="find" data-citedArticleID="945218" data-doi="10.1097/01.coh.0000221590.03670.32"><li><a href="http://dx.doi.org/10.1097/01.coh.0000221590.03670.32" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Innate+cellular+defenses+of+APOBEC3+cytidine+deaminases+and+viral+counter-defenses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Innate+cellular+defenses+of+APOBEC3+cytidine+deaminases+and+viral+counter-defenses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000378-Harris1" id="pone.0000378-Harris1"></a>Harris RS, Matsuo H (2006) Dancin' deaminase. Nat Struct Mol Biol  13: 380–381.  <ul class="find" data-citedArticleID="945154" data-doi="10.1038/nsmb0506-380"><li><a href="http://dx.doi.org/10.1038/nsmb0506-380" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dancin%27+deaminase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dancin%27+deaminase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000378-Sheehy1" id="pone.0000378-Sheehy1"></a>Sheehy AM, Gaddis NC, Choi JD, Malim MH (2002) Isolation of a human gene that inhibits HIV-1 infection and is suppressed by the viral Vif protein. Nature  418: 646–650.  <ul class="find" data-citedArticleID="945206" data-doi="10.1038/nature00939"><li><a href="http://dx.doi.org/10.1038/nature00939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+a+human+gene+that+inhibits+HIV-1+infection+and+is+suppressed+by+the+viral+Vif+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+a+human+gene+that+inhibits+HIV-1+infection+and+is+suppressed+by+the+viral+Vif+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000378-Mangeat1" id="pone.0000378-Mangeat1"></a>Mangeat B, Turelli P, Liao S, Trono D (2004) A single amino acid determinant governs the species-specific sensitivity of APOBEC3G to Vif action. J Biol Chem  279: 14481–14483.  <ul class="find" data-citedArticleID="945168" data-doi="10.1074/jbc.c400060200"><li><a href="http://dx.doi.org/10.1074/jbc.c400060200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+amino+acid+determinant+governs+the+species-specific+sensitivity+of+APOBEC3G+to+Vif+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+amino+acid+determinant+governs+the+species-specific+sensitivity+of+APOBEC3G+to+Vif+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000378-Bogerd1" id="pone.0000378-Bogerd1"></a>Bogerd HP, Doehle BP, Wiegand HL, Cullen BR (2004) A single amino acid difference in the host APOBEC3G protein controls the primate species specificity of HIV type 1 virion infectivity factor. Proc Natl Acad Sci U S A  101: 3770–3774.  <ul class="find" data-citedArticleID="945134" data-doi="10.1073/pnas.0307713101"><li><a href="http://dx.doi.org/10.1073/pnas.0307713101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+amino+acid+difference+in+the+host+APOBEC3G+protein+controls+the+primate+species+specificity+of+HIV+type+1+virion+infectivity+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+amino+acid+difference+in+the+host+APOBEC3G+protein+controls+the+primate+species+specificity+of+HIV+type+1+virion+infectivity+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000378-Schrofelbauer1" id="pone.0000378-Schrofelbauer1"></a>Schrofelbauer B, Chen D, Landau NR (2004) A single amino acid of APOBEC3G controls its species-specific interaction with virion infectivity factor (Vif). Proc Natl Acad Sci U S A  101: 3927–3932.  <ul class="find" data-citedArticleID="945202" data-doi="10.1073/pnas.0307132101"><li><a href="http://dx.doi.org/10.1073/pnas.0307132101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+amino+acid+of+APOBEC3G+controls+its+species-specific+interaction+with+virion+infectivity+factor+%28Vif%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+amino+acid+of+APOBEC3G+controls+its+species-specific+interaction+with+virion+infectivity+factor+%28Vif%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000378-Xu1" id="pone.0000378-Xu1"></a>Xu H, Svarovskaia ES, Barr R, Zhang Y, Khan MA, et al.  (2004) A single amino acid substitution in human APOBEC3G antiretroviral enzyme confers resistance to HIV-1 virion infectivity factor-induced depletion. Proc Natl Acad Sci U S A  101: 5652–5657.  <ul class="find" data-citedArticleID="945216" data-doi="10.1073/pnas.0400830101"><li><a href="http://dx.doi.org/10.1073/pnas.0400830101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+amino+acid+substitution+in+human+APOBEC3G+antiretroviral+enzyme+confers+resistance+to+HIV-1+virion+infectivity+factor-induced+depletion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+amino+acid+substitution+in+human+APOBEC3G+antiretroviral+enzyme+confers+resistance+to+HIV-1+virion+infectivity+factor-induced+depletion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000378-Navarro1" id="pone.0000378-Navarro1"></a>Navarro F, Bollman B, Chen H, Konig R, Yu Q, et al.  (2005) Complementary function of the two catalytic domains of APOBEC3G. Virology  333: 374–386.  <ul class="find" data-citedArticleID="945184" data-doi="10.1016/j.virol.2005.01.011"><li><a href="http://dx.doi.org/10.1016/j.virol.2005.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementary+function+of+the+two+catalytic+domains+of+APOBEC3G." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementary+function+of+the+two+catalytic+domains+of+APOBEC3G.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000378-Luo1" id="pone.0000378-Luo1"></a>Luo K, Liu B, Xiao Z, Yu Y, Yu X, et al.  (2004) Amino-terminal region of the human immunodeficiency virus type 1 nucleocapsid is required for human APOBEC3G packaging. J Virol  78: 11841–11852.  <ul class="find" data-citedArticleID="945164" data-doi="10.1128/jvi.78.21.11841-11852.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.21.11841-11852.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino-terminal+region+of+the+human+immunodeficiency+virus+type+1+nucleocapsid+is+required+for+human+APOBEC3G+packaging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino-terminal+region+of+the+human+immunodeficiency+virus+type+1+nucleocapsid+is+required+for+human+APOBEC3G+packaging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000378-Newman1" id="pone.0000378-Newman1"></a>Newman EN, Holmes RK, Craig HM, Klein KC, Lingappa JR, et al.  (2005) Antiviral function of APOBEC3G can be dissociated from cytidine deaminase activity. Curr Biol  15: 166–170.  <ul class="find" data-citedArticleID="945186" data-doi="10.1016/j.cub.2004.12.068"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.12.068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antiviral+function+of+APOBEC3G+can+be+dissociated+from+cytidine+deaminase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antiviral+function+of+APOBEC3G+can+be+dissociated+from+cytidine+deaminase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000378-Scott1" id="pone.0000378-Scott1"></a>Scott J, Navaratnam N, Carter C (1998) Molecular modelling and the biosynthesis of apolipoprotein B containing lipoproteins. Atherosclerosis  141 Suppl 1: S17–S24.  <ul class="find" data-citedArticleID="945204" data-doi="10.1016/s0021-9150(98)00213-5"><li><a href="http://dx.doi.org/10.1016/s0021-9150(98)00213-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+modelling+and+the+biosynthesis+of+apolipoprotein+B+containing+lipoproteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+modelling+and+the+biosynthesis+of+apolipoprotein+B+containing+lipoproteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000378-Navaratnam1" id="pone.0000378-Navaratnam1"></a>Navaratnam N, Fujino T, Bayliss J, Jarmuz A, How A, et al.  (1998) Escherichia coli cytidine deaminase provides a molecular model for ApoB RNA editing and a mechanism for RNA substrate recognition. J Mol Biol  275: 695–714.  <ul class="find" data-citedArticleID="945180" data-doi="10.1006/jmbi.1997.1506"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.1506" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+cytidine+deaminase+provides+a+molecular+model+for+ApoB+RNA+editing+and+a+mechanism+for+RNA+substrate+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+cytidine+deaminase+provides+a+molecular+model+for+ApoB+RNA+editing+and+a+mechanism+for+RNA+substrate+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000378-Xie1" id="pone.0000378-Xie1"></a>Xie K, Sowden MP, Dance GS, Torelli AT, Smith HC, et al.  (2004) The structure of a yeast RNA-editing deaminase provides insight into the fold and function of activation-induced deaminase and APOBEC-1. Proc Natl Acad Sci U S A  101: 8114–8119.  <ul class="find" data-citedArticleID="945214" data-doi="10.1073/pnas.0400493101"><li><a href="http://dx.doi.org/10.1073/pnas.0400493101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+a+yeast+RNA-editing+deaminase+provides+insight+into+the+fold+and+function+of+activation-induced+deaminase+and+APOBEC-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+a+yeast+RNA-editing+deaminase+provides+insight+into+the+fold+and+function+of+activation-induced+deaminase+and+APOBEC-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000378-Zaim1" id="pone.0000378-Zaim1"></a>Zaim J, Kierzek AM (2003) Domain organization of activation-induced cytidine deaminase. Nat Immunol  4: 1153.  <ul class="find" data-citedArticleID="945220" data-doi="10.1038/ni1203-1153"><li><a href="http://dx.doi.org/10.1038/ni1203-1153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+organization+of+activation-induced+cytidine+deaminase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+organization+of+activation-induced+cytidine+deaminase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000378-Ta1" id="pone.0000378-Ta1"></a>Ta VT, Nagaoka H, Catalan N, Durandy A, Fischer A, et al.  (2003) AID mutant analyses indicate requirement for class-switch-specific cofactors. Nat Immunol  4: 843–848.  <ul class="find" data-citedArticleID="945210" data-doi="10.1038/ni964"><li><a href="http://dx.doi.org/10.1038/ni964" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AID+mutant+analyses+indicate+requirement+for+class-switch-specific+cofactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AID+mutant+analyses+indicate+requirement+for+class-switch-specific+cofactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000378-Wedekind1" id="pone.0000378-Wedekind1"></a>Wedekind JE, Dance GS, Sowden MP, Smith HC (2003) Messenger RNA editing in mammals: new members of the APOBEC family seeking roles in the family business. Trends Genet  19: 207–216.  <ul class="find" data-citedArticleID="945212" data-doi="10.1016/s0168-9525(03)00054-4"><li><a href="http://dx.doi.org/10.1016/s0168-9525(03)00054-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Messenger+RNA+editing+in+mammals%3A+new+members+of+the+APOBEC+family+seeking+roles+in+the+family+business." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Messenger+RNA+editing+in+mammals%3A+new+members+of+the+APOBEC+family+seeking+roles+in+the+family+business.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000378-Huthoff1" id="pone.0000378-Huthoff1"></a>Huthoff H, Malim MH (2005) Cytidine deamination and resistance to retroviral infection: towards a structural understanding of the APOBEC proteins. Virology  334: 147–153.  <ul class="find" data-citedArticleID="945156" data-doi="10.1016/j.virol.2005.01.038"><li><a href="http://dx.doi.org/10.1016/j.virol.2005.01.038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytidine+deamination+and+resistance+to+retroviral+infection%3A+towards+a+structural+understanding+of+the+APOBEC+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytidine+deamination+and+resistance+to+retroviral+infection%3A+towards+a+structural+understanding+of+the+APOBEC+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000378-Prochnow1" id="pone.0000378-Prochnow1"></a>Prochnow C, Bransteitter R, Klein MG, Goodman MF, Chen XS (2006) The APOBEC-2 crystal structure and functional implications for the deaminase AID. Nature.   <ul class="find" data-citedArticleID="945194" data-doi="10.1038/nature05492"><li><a href="http://dx.doi.org/10.1038/nature05492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+APOBEC-2+crystal+structure+and+functional+implications+for+the+deaminase+AID." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+APOBEC-2+crystal+structure+and+functional+implications+for+the+deaminase+AID.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000378-Huthoff2" id="pone.0000378-Huthoff2"></a>Huthoff H, Malim MH (2007) Identification of amino acid residues in APOBEC3G required for regulation by HIV-1 Vif and virion encapsidation. J Virol PMID 17267497.  <ul class="find" data-citedArticleID="945158" data-doi="10.1128/jvi.02795-06"><li><a href="http://dx.doi.org/10.1128/jvi.02795-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+amino+acid+residues+in+APOBEC3G+required+for+regulation+by+HIV-1+Vif+and+virion+encapsidation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+amino+acid+residues+in+APOBEC3G+required+for+regulation+by+HIV-1+Vif+and+virion+encapsidation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000378-Sali1" id="pone.0000378-Sali1"></a>Sali A, Blundell TL (1993) Comparative protein modelling by satisfaction of spatial restraints. J Mol Biol  234: 779–815.  <ul class="find" data-citedArticleID="945196" data-doi="10.1006/jmbi.1993.1626"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1626" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+protein+modelling+by+satisfaction+of+spatial+restraints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000378-MartiRenom1" id="pone.0000378-MartiRenom1"></a>Marti-Renom MA, Stuart AC, Fiser A, Sanchez R, Melo F, et al.  (2000) Comparative protein structure modeling of genes and genomes. Annu Rev Biophys Biomol Struct  29: 291–325.  <ul class="find" data-citedArticleID="945170" data-doi="10.1146/annurev.biophys.29.1.291"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.29.1.291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+protein+structure+modeling+of+genes+and+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+protein+structure+modeling+of+genes+and+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000378-Fiser1" id="pone.0000378-Fiser1"></a>Fiser A, Do RK, Sali A (2000) Modeling of loops in protein structures. Protein Sci  9: 1753–1773.  <ul class="find" data-citedArticleID="945150" data-doi="10.1110/ps.9.9.1753"><li><a href="http://dx.doi.org/10.1110/ps.9.9.1753" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+of+loops+in+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+of+loops+in+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000378-Kabsch1" id="pone.0000378-Kabsch1"></a>Kabsch W, Sander C (1983) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. Biopolymers  22: 2577–2637.  <ul class="find" data-citedArticleID="945162" data-doi="10.1002/bip.360221211"><li><a href="http://dx.doi.org/10.1002/bip.360221211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dictionary+of+protein+secondary+structure%3A+pattern+recognition+of+hydrogen-bonded+and+geometrical+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dictionary+of+protein+secondary+structure%3A+pattern+recognition+of+hydrogen-bonded+and+geometrical+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000378-McGuffin1" id="pone.0000378-McGuffin1"></a>McGuffin LJ, Bryson K, Jones DT (2000) The PSIPRED protein structure prediction server. Bioinformatics  16: 404–405.  <ul class="find" data-citedArticleID="945172" data-doi="10.1093/bioinformatics/16.4.404"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.4.404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PSIPRED+protein+structure+prediction+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PSIPRED+protein+structure+prediction+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000378-Bryson1" id="pone.0000378-Bryson1"></a>Bryson K, McGuffin LJ, Marsden RL, Ward JJ, Sodhi JS, et al.  (2005) Protein structure prediction servers at University College London. Nucleic Acids Res  33: W36–W38.  <ul class="find" data-citedArticleID="945138" data-doi="10.1093/nar/gki410"><li><a href="http://dx.doi.org/10.1093/nar/gki410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+structure+prediction+servers+at+University+College+London." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+structure+prediction+servers+at+University+College+London.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000378-Jones1" id="pone.0000378-Jones1"></a>Jones DT (1999) Protein secondary structure prediction based on position-specific scoring matrices. J Mol Biol  292: 195–202.  <ul class="find" data-citedArticleID="945160" data-doi="10.1006/jmbi.1999.3091"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+secondary+structure+prediction+based+on+position-specific+scoring+matrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+secondary+structure+prediction+based+on+position-specific+scoring+matrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000378-Clamp1" id="pone.0000378-Clamp1"></a>Clamp M, Cuff J, Searle SM, Barton GJ (2004) The Jalview Java alignment editor. Bioinformatics  20: 426–427.  <ul class="find" data-citedArticleID="945142" data-doi="10.1093/bioinformatics/btg430"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg430" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Jalview+Java+alignment+editor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Jalview+Java+alignment+editor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000378-Melo1" id="pone.0000378-Melo1"></a>Melo F, Feytmans E (1998) Assessing protein structures with a non-local atomic interaction energy. J Mol Biol  277: 1141–1152.  <ul class="find" data-citedArticleID="945174" data-doi="10.1006/jmbi.1998.1665"><li><a href="http://dx.doi.org/10.1006/jmbi.1998.1665" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+protein+structures+with+a+non-local+atomic+interaction+energy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+protein+structures+with+a+non-local+atomic+interaction+energy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000378-Melo2" id="pone.0000378-Melo2"></a>Melo F, Feytmans E (1997) Novel knowledge-based mean force potential at atomic level. J Mol Biol  267: 207–222.  <ul class="find" data-citedArticleID="945176" data-doi="10.1006/jmbi.1996.0868"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0868" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+knowledge-based+mean+force+potential+at+atomic+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+knowledge-based+mean+force+potential+at+atomic+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000378-Melo3" id="pone.0000378-Melo3"></a>Melo F, Devos D, Depiereux E, Feytmans E (1997) ANOLEA: a www server to assess protein structures. Proc Int Conf Intell Syst Mol Biol  5: 187–190.  <ul class="find" data-citedArticleID="945178"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=ANOLEA%3A+a+www+server+to+assess+protein+structures.&amp;auth=&amp;atitle=ANOLEA%3A+a+www+server+to+assess+protein+structures." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ANOLEA%3A+a+www+server+to+assess+protein+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ANOLEA%3A+a+www+server+to+assess+protein+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000378-Brooks1" id="pone.0000378-Brooks1"></a>Brooks BR, Bruccoleri RE, Olafson BD, States DJ, Swaminathan S, et al.  (1983) CHARMM: A Program for Macromolecular Energy, Minimization, and Dynamics Calculations. J Comp Chem  4: 187–217.  <ul class="find" data-citedArticleID="945136" data-doi="10.1002/jcc.540040211"><li><a href="http://dx.doi.org/10.1002/jcc.540040211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CHARMM%3A+A+Program+for+Macromolecular+Energy%2C+Minimization%2C+and+Dynamics+Calculations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CHARMM%3A+A+Program+for+Macromolecular+Energy%2C+Minimization%2C+and+Dynamics+Calculations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000378-MacKerell1" id="pone.0000378-MacKerell1"></a>MacKerell AD, Bashford D, Bellott M, Dunbrack RL, Evaseck JD, et al.  (1998) All-atom empirical potential for molecular modeling and dynamics studies of proteins. J Phys Chem B  102: 3586–3616.  <ul class="find" data-citedArticleID="945166" data-doi="10.1021/jp973084f"><li><a href="http://dx.doi.org/10.1021/jp973084f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22All-atom+empirical+potential+for+molecular+modeling+and+dynamics+studies+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000378-Guerois1" id="pone.0000378-Guerois1"></a>Guerois R, Nielsen JE, Serrano L (2002) Predicting changes in the stability of proteins and protein complexes: a study of more than 1000 mutations. J Mol Biol  320: 369–387.  <ul class="find" data-citedArticleID="945152" data-doi="10.1016/s0022-2836(02)00442-4"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00442-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+changes+in+the+stability+of+proteins+and+protein+complexes%3A+a+study+of+more+than+1000+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+changes+in+the+stability+of+proteins+and+protein+complexes%3A+a+study+of+more+than+1000+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000378-Sanner1" id="pone.0000378-Sanner1"></a>Sanner MF, Olson AJ, Spehner JC (1996) Reduced surface: an efficient way to compute molecular surfaces. Biopolymers  38: 305–320.  <ul class="find" data-citedArticleID="945198" data-doi="10.1002/(sici)1097-0282(199603)38:3<305::aid-bip4&gt;3.3.co;2-8"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0282(199603)38:3<305::aid-bip4&gt;3.3.co;2-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+surface%3A+an+efficient+way+to+compute+molecular+surfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+surface%3A+an+efficient+way+to+compute+molecular+surfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000378-Pettersen1" id="pone.0000378-Pettersen1"></a>Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, et al.  (2004) UCSF Chimera–a visualization system for exploratory research and analysis. J Comput Chem  25: 1605–1612.  <ul class="find" data-citedArticleID="945192" data-doi="10.1002/jcc.20084"><li><a href="http://dx.doi.org/10.1002/jcc.20084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UCSF+Chimera%26%23x2013%3Ba+visualization+system+for+exploratory+research+and+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UCSF+Chimera%26%23x2013%3Ba+visualization+system+for+exploratory+research+and+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000378-Davis1" id="pone.0000378-Davis1"></a>Davis ME, Madura JD, Luty BA, McCammon JA (1991) Electrostatics and diffusion of molecules in solution: simulations with the University of Houston Brownian Dynamics program. Comp Phys Comm  62: 187–197.  <ul class="find" data-citedArticleID="945148" data-doi="10.1016/0010-4655(91)90094-2"><li><a href="http://dx.doi.org/10.1016/0010-4655(91)90094-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electrostatics+and+diffusion+of+molecules+in+solution%3A+simulations+with+the+University+of+Houston+Brownian+Dynamics+program." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electrostatics+and+diffusion+of+molecules+in+solution%3A+simulations+with+the+University+of+Houston+Brownian+Dynamics+program.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000378-Navaratnam2" id="pone.0000378-Navaratnam2"></a>Navaratnam N, Bhattacharya S, Fujino T, Patel D, Jarmuz AL, et al.  (1995) Evolutionary origins of apoB mRNA editing: catalysis by a cytidine deaminase that has acquired a novel RNA-binding motif at its active site. Cell  81: 187–195.  <ul class="find" data-citedArticleID="945182" data-doi="10.1016/0092-8674(95)90328-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90328-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+origins+of+apoB+mRNA+editing%3A+catalysis+by+a+cytidine+deaminase+that+has+acquired+a+novel+RNA-binding+motif+at+its+active+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+origins+of+apoB+mRNA+editing%3A+catalysis+by+a+cytidine+deaminase+that+has+acquired+a+novel+RNA-binding+motif+at+its+active+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000378-Ortiz1" id="pone.0000378-Ortiz1"></a>Ortiz M, Bleiber G, Martinez R, Kaessmann H, Telenti A (2006) Patterns of evolution of host proteins involved in retroviral pathogenesis. Retrovirology  3: 11.  <ul class="find" data-citedArticleID="945190" data-doi="10.1186/1742-4690-3-11"><li><a href="http://dx.doi.org/10.1186/1742-4690-3-11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+evolution+of+host+proteins+involved+in+retroviral+pathogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+evolution+of+host+proteins+involved+in+retroviral+pathogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000378-Cen1" id="pone.0000378-Cen1"></a>Cen S, Guo F, Niu M, Saadatmand J, Deflassieux J, et al.  (2004) The interaction between HIV-1 Gag and APOBEC3G. J Biol Chem  279: 33177–33184.  <ul class="find" data-citedArticleID="945140" data-doi="10.1074/jbc.m402062200"><li><a href="http://dx.doi.org/10.1074/jbc.m402062200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+interaction+between+HIV-1+Gag+and+APOBEC3G." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+interaction+between+HIV-1+Gag+and+APOBEC3G.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000378-Schafer1" id="pone.0000378-Schafer1"></a>Schafer A, Bogerd HP, Cullen BR (2004) Specific packaging of APOBEC3G into HIV-1 virions is mediated by the nucleocapsid domain of the gag polyprotein precursor. Virology  328: 163–168.  <ul class="find" data-citedArticleID="945200" data-doi="10.1016/j.virol.2004.08.006"><li><a href="http://dx.doi.org/10.1016/j.virol.2004.08.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+packaging+of+APOBEC3G+into+HIV-1+virions+is+mediated+by+the+nucleocapsid+domain+of+the+gag+polyprotein+precursor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+packaging+of+APOBEC3G+into+HIV-1+virions+is+mediated+by+the+nucleocapsid+domain+of+the+gag+polyprotein+precursor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000378-Svarovskaia1" id="pone.0000378-Svarovskaia1"></a>Svarovskaia ES, Xu H, Mbisa JL, Barr R, Gorelick RJ, et al.  (2004) Human apolipoprotein B mRNA-editing enzyme-catalytic polypeptide-like 3G (APOBEC3G) is incorporated into HIV-1 virions through interactions with viral and nonviral RNAs. J Biol Chem  279: 35822–35828.  <ul class="find" data-citedArticleID="945208" data-doi="10.1074/jbc.m405761200"><li><a href="http://dx.doi.org/10.1074/jbc.m405761200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+apolipoprotein+B+mRNA-editing+enzyme-catalytic+polypeptide-like+3G+%28APOBEC3G%29+is+incorporated+into+HIV-1+virions+through+interactions+with+viral+and+nonviral+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+apolipoprotein+B+mRNA-editing+enzyme-catalytic+polypeptide-like+3G+%28APOBEC3G%29+is+incorporated+into+HIV-1+virions+through+interactions+with+viral+and+nonviral+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000378-Zennou1" id="pone.0000378-Zennou1"></a>Zennou V, Perez-Caballero D, Gottlinger H, Bieniasz PD (2004) APOBEC3G incorporation into human immunodeficiency virus type 1 particles. J Virol  78: 12058–12061.  <ul class="find" data-citedArticleID="945222" data-doi="10.1128/jvi.78.21.12058-12061.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.21.12058-12061.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=APOBEC3G+incorporation+into+human+immunodeficiency+virus+type+1+particles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22APOBEC3G+incorporation+into+human+immunodeficiency+virus+type+1+particles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000378-Conticello1" id="pone.0000378-Conticello1"></a>Conticello SG, Harris RS, Neuberger MS (2003) The Vif protein of HIV triggers degradation of the human antiretroviral DNA deaminase APOBEC3G. Curr Biol  13: 2009–2013.  <ul class="find" data-citedArticleID="945144" data-doi="10.1016/j.cub.2003.10.034"><li><a href="http://dx.doi.org/10.1016/j.cub.2003.10.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Vif+protein+of+HIV+triggers+degradation+of+the+human+antiretroviral+DNA+deaminase+APOBEC3G." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Vif+protein+of+HIV+triggers+degradation+of+the+human+antiretroviral+DNA+deaminase+APOBEC3G.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000378-Ohkura1" id="pone.0000378-Ohkura1"></a>Ohkura S, Yap MW, Sheldon T, Stoye JP (2006) All three variable regions of the TRIM5 alpha B30.2 domain can contribute to the specificity of the retrovirus restriction. J Virol  80: 8554–8565.  <ul class="find" data-citedArticleID="945188" data-doi="10.1128/jvi.00688-06"><li><a href="http://dx.doi.org/10.1128/jvi.00688-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=All+three+variable+regions+of+the+TRIM5+alpha+B30.2+domain+can+contribute+to+the+specificity+of+the+retrovirus+restriction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22All+three+variable+regions+of+the+TRIM5+alpha+B30.2+domain+can+contribute+to+the+specificity+of+the+retrovirus+restriction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.XML" value="79927"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.PDF" value="421381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g001.PNG_L" value="1041050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g001.PNG_M" value="139846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g001.PNG_S" value="7362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g001.TIF" value="1792168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g001.PNG_I" value="96062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.t001.PNG_L" value="167206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.t001.PNG_M" value="41189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.t001.PNG_S" value="7834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.t001.TIF" value="1128100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.t001.PNG_I" value="94269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g002.PNG_L" value="520460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g002.PNG_M" value="59666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g002.PNG_S" value="3133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g002.TIF" value="578750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g002.PNG_I" value="25330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g003.PNG_L" value="3794212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g003.PNG_M" value="201109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g003.PNG_S" value="10937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g003.TIF" value="6664458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g003.PNG_I" value="122653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g004.PNG_L" value="4142456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g004.PNG_M" value="273411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g004.PNG_S" value="6769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g004.TIF" value="6482516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.g004.PNG_I" value="93322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000378.s001.EPS" value="2864919"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000378&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000378" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000378&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000378&volume=&issue=&title=Model Structure of Human APOBEC3G&author_name=Kun-Lin%20Zhang%2C%20Bastien%20Mangeat%2C%20Millan%20Ortiz%2C%20Vincent%20Zoete%2C%20Didier%20Trono%2C%20Amalio%20Telenti%2C%20Olivier%20Michielin&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000378" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378&amp;t=Model%20Structure%20of%20Human%20APOBEC3G" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378&title=Model%20Structure%20of%20Human%20APOBEC3G&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378&amp;title=Model%20Structure%20of%20Human%20APOBEC3G" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000378&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Model Structure of Human APOBEC3G';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000378';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Model%20Structure%20of%20Human%20APOBEC3G http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000378" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000378" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="23594"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytosine%22" title="Search for articles in the subject area:'Cytosine'"><div class="flagText">Cytosine</div></a>
              <div data-categoryid="18467" data-articleid="23594"
                   data-categoryname="Cytosine"
                   class="flagImage" title="Flag 'Cytosine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Encapsidation%22" title="Search for articles in the subject area:'Encapsidation'"><div class="flagText">Encapsidation</div></a>
              <div data-categoryid="29931" data-articleid="23594"
                   data-categoryname="Encapsidation"
                   class="flagImage" title="Flag 'Encapsidation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="23594"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Primates%22" title="Search for articles in the subject area:'Primates'"><div class="flagText">Primates</div></a>
              <div data-categoryid="42883" data-articleid="23594"
                   data-categoryname="Primates"
                   class="flagImage" title="Flag 'Primates' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23594"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+structure%22" title="Search for articles in the subject area:'Viral structure'"><div class="flagText">Viral structure</div></a>
              <div data-categoryid="18769" data-articleid="23594"
                   data-categoryname="Viral structure"
                   class="flagImage" title="Flag 'Viral structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Virions%22" title="Search for articles in the subject area:'Virions'"><div class="flagText">Virions</div></a>
              <div data-categoryid="19387" data-articleid="23594"
                   data-categoryname="Virions"
                   class="flagImage" title="Flag 'Virions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4595'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7065'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5845&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
