

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000514"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000514" />

    <meta name="citation_title" content="Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes"/>
    <meta itemprop="name" content="Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes"/>

      <meta name="citation_author" content="Nadia A. Ayoub"/>
            <meta name="citation_author_institution" content="Department of Biology, University of California Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Jessica E. Garb"/>
            <meta name="citation_author_institution" content="Department of Biology, University of California Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Robin M. Tinghitella"/>
            <meta name="citation_author_institution" content="Department of Biology, University of California Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Matthew A. Collin"/>
            <meta name="citation_author_institution" content="Department of Biology, University of California Riverside, Riverside, California, United States of America"/>
      <meta name="citation_author" content="Cheryl Y. Hayashi"/>
            <meta name="citation_author_institution" content="Department of Biology, University of California Riverside, Riverside, California, United States of America"/>

    <meta name="citation_date" content="2007/6/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000514.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e514"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Liquid crystalline spinning of spider silk.; citation_author=F Vollrath; citation_author=DP Knight; citation_journal_title=Nature; citation_volume=410; citation_number=1; citation_pages=541-548; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Elastic proteins: biological roles and mechanical properties.; citation_author=J Gosline; citation_author=M Lillie; citation_author=E Carrington; citation_author=P Guerrette; citation_author=C Ortlepp; citation_journal_title=Phil Trans R Soc Lond B Biol Sci; citation_volume=357; citation_number=2; citation_pages=121-132; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetic engineering of fibrous proteins: spider dragline silk and collagen.; citation_author=CWP Foo; citation_author=DL Kaplan; citation_journal_title=Adv Drug Deliv Rev; citation_volume=54; citation_number=3; citation_pages=1131-1143; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The structure and properties of spider silk.; citation_author=JM Gosline; citation_author=ME DeMont; citation_author=MW Denny; citation_journal_title=Endeavour; citation_volume=10; citation_number=4; citation_pages=37-43; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Spider silk: a mystery starting to unravel.; citation_author=M Hinman; citation_author=Z Dong; citation_author=M Xu; citation_author=RV Lewis; citation_number=5; citation_pages=227-254; citation_date=1992; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=The mechanical design of spider silks: from fibroin sequence to mechanical function.; citation_author=JM Gosline; citation_author=PA Guerette; citation_author=CS Ortlepp; citation_author=KN Savage; citation_journal_title=J Exp Biol; citation_volume=202; citation_number=6; citation_pages=3295-3303; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Biology of Spiders.; citation_author=R Foelix; citation_number=7; citation_pages=336; citation_date=1996; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Silken toolkits: biomechanics of silk fibers spun by the orb web spider Argiope argentata (Fabricius 1775).; citation_author=TA Blackledge; citation_author=CY Hayashi; citation_journal_title=J Exp Biol; citation_volume=209; citation_number=8; citation_pages=2452-2461; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Silk properties determined by gland-specific expression of a spider fibroin gene family.; citation_author=PA Guerette; citation_author=DG Ginzinger; citation_author=BHF Weber; citation_author=JM Gosline; citation_journal_title=Science; citation_volume=272; citation_number=9; citation_pages=112-115; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Extreme diversity, conservation, and convergence of spider silk fibroin sequences.; citation_author=J Gatesy; citation_author=C Hayashi; citation_author=D Motriuk; citation_author=J Woods; citation_author=R Lewis; citation_journal_title=Science; citation_volume=291; citation_number=10; citation_pages=2603-2605; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization and evolutionary study of spider tubuliform (eggcase) silk protein.; citation_author=M Tian; citation_author=RV Lewis; citation_journal_title=Biochemistry; citation_volume=44; citation_number=11; citation_pages=8006-8012; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Silk genes support the single origin of orb webs.; citation_author=JE Garb; citation_author=T DiMauro; citation_author=V Vo; citation_author=CY Hayashi; citation_journal_title=Science; citation_volume=312; citation_number=12; citation_pages=1762; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hypotheses that correlate the sequence, structure, and mechanical properties of spider silk proteins.; citation_author=CY Hayashi; citation_author=NH Shipley; citation_author=RV Lewis; citation_journal_title=Int J Biol Macromol; citation_volume=24; citation_number=13; citation_pages=271-275; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular and mechanical characterization of aciniform silk: uniformity of iterated sequence modules in a novel member of the spider silk fibroin gene family.; citation_author=CY Hayashi; citation_author=TA Blackledge; citation_author=RV Lewis; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=14; citation_pages=1950-1959; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of the protein components of Nephila clavipes dragline silk.; citation_author=A Sponner; citation_author=B Schlott; citation_author=F Vollrath; citation_author=E Unger; citation_author=F Grosse; citation_journal_title=Biochemistry; citation_volume=44; citation_number=15; citation_pages=4727-4736; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Novel molecular and mechanical properties of egg case silk from wasp spider, Argiope bruennichi.; citation_author=A Zhao; citation_author=T Zhao; citation_author=K Nakagaki; citation_author=Y-S Zhang; citation_author=Y SiMa; citation_journal_title=Biochemistry-US; citation_volume=45; citation_number=16; citation_pages=3348-3356; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure of a protein superfiber: spider dragline silk.; citation_author=M Xu; citation_author=RV Lewis; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=87; citation_number=17; citation_pages=7120-7124; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Isolation of a clone encoding a second dragline silk fibroin. Nephila clavipes dragline silk is a two-protein fiber.; citation_author=MB Hinman; citation_author=RV Lewis; citation_journal_title=J Biol Chem; citation_volume=267; citation_number=18; citation_pages=19320-19324; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Differential polymerization of the two main protein components of dragline silk during fibre spinning.; citation_author=A Sponner; citation_author=E Unger; citation_author=F Grosse; citation_author=K Weisshart; citation_journal_title=Nat Mater; citation_volume=4; citation_number=19; citation_pages=772-775; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Solid-state 13C NMR of Nephila clavipes dragline silk establishes structure and identity of crystalline regions.; citation_author=A Simmons; citation_author=E Ray; citation_author=LW Jelinski; citation_journal_title=Macromolecules; citation_volume=27; citation_number=20; citation_pages=5235-5237; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Local structure in spider dragline silk investigated by two dimensional spin-diffusion NMR.; citation_author=J Kümmerlen; citation_author=JD van Beek; citation_author=F Vollrath; citation_author=BH Meier; citation_journal_title=Macromolecules; citation_volume=29; citation_number=21; citation_pages=2920-2928; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular orientation and two-component nature of the crystalline fraction of spider dragline silk.; citation_author=AH Simmons; citation_author=CA Michal; citation_author=LW Jelinski; citation_journal_title=Science; citation_volume=271; citation_number=22; citation_pages=84-87; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=X-ray diffraction from single fibers of spider silk.; citation_author=A Bram; citation_author=CI Bränden; citation_author=C Craig; citation_author=I Snigireva; citation_author=C Riekel; citation_journal_title=J Appl Cryst; citation_volume=30; citation_number=23; citation_pages=390-392; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Structural studies of spider silk proteins in the fiber.; citation_author=AD Parkhe; citation_author=SK Seeley; citation_author=K Gardner; citation_author=L Thompson; citation_author=RV Lewis; citation_journal_title=J Mol Recogn; citation_volume=10; citation_number=24; citation_pages=1-6; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanisms of spider silk elasticity.; citation_author=Z Dong; citation_author=RV Lewis; citation_author=CR Middaugh; citation_journal_title=Arch Biochem Biophys; citation_volume=284; citation_number=25; citation_pages=53-57; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Evidence from flagelliform silk cDNA for the structural basis of elasticity and modular nature of spider silks.; citation_author=CY Hayashi; citation_author=RV Lewis; citation_journal_title=J Mol Biol; citation_volume=275; citation_number=26; citation_pages=773-784; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The molecular structure of spider dragline silk: folding and orientation of the protein backbone; citation_author=JD van Beek; citation_author=S Hess; citation_author=F Vollrath; citation_author=BH Meier; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=27; citation_pages=10266-10271; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Nylons from nature: synthetic analogs to spider silk.; citation_author=JP O'Brien; citation_author=SR Fahnestock; citation_author=Y Termonia; citation_author=KH Gardner; citation_journal_title=Adv Mater; citation_volume=10; citation_number=28; citation_pages=1185-1195; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Synthetic spider silk: a modular fiber.; citation_author=MB Hinman; citation_author=JA Jones; citation_author=RV Lewis; citation_journal_title=Trends Biotechnol; citation_volume=18; citation_number=29; citation_pages=374-379; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Spider silk fibers spun from soluble recombinant silk produced in mammalian cells.; citation_author=A Lazaris; citation_author=S Arcidiacono; citation_author=Y Huang; citation_author=J-F Zhou; citation_author=F Duguay; citation_journal_title=Science; citation_volume=295; citation_number=30; citation_pages=472-476; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Purification of spider silk-elastin from transgenic plants and application for human chondrocyte proliferation.; citation_author=J Scheller; citation_author=D Henggeler; citation_author=A Viviani; citation_author=U Conrad; citation_journal_title=Transgenic Res; citation_volume=13; citation_number=31; citation_pages=51-57; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RGD-functionalized bioengineered spider dragline silk biomaterial; citation_author=E Bini; citation_author=CWP Foo; citation_author=J Huang; citation_author=V Karageorgiou; citation_author=B Kitchel; citation_journal_title=Biomacromolecules; citation_volume=7; citation_number=32; citation_pages=3139-3145; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Production of spider silk proteins in tobacco and potato.; citation_author=J Scheller; citation_author=K-H Gührs; citation_author=F Grosse; citation_author=U Conrad; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=33; citation_pages=573-577; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Solution behavior of synthetic silk peptides and modified recombinant silk proteins.; citation_author=CWP Foo; citation_author=E Bini; citation_author=J Huang; citation_author=SY Lee; citation_author=DL Kaplan; citation_journal_title=Appl Phys A; citation_volume=82; citation_number=34; citation_pages=193-203; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An essential role for the C-terminal domain of a dragline spider silk protein in directing fiber formation.; citation_author=S Ittah; citation_author=S Cohen; citation_author=S Garty; citation_author=D Cohn; citation_author=U Gat; citation_journal_title=Biomacromolecules; citation_volume=7; citation_number=35; citation_pages=1790-1795; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Bombyx mori silk proteins: characterization of large polypeptides.; citation_author=KU Sprague; citation_journal_title=Biochemistry; citation_volume=14; citation_number=36; citation_pages=925-931; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=The conserved C-termini contribute to the properties of spider silk fibroins.; citation_author=A Sponner; citation_author=W Vater; citation_author=W Rommerskirch; citation_author=F Vollrath; citation_author=E Unger; citation_journal_title=Biochem Biophys Res Commun; citation_volume=338; citation_number=37; citation_pages=897-902; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of the conserved N-terminal domains in major ampullate spider silk proteins.; citation_author=D Motriuk-Smith; citation_author=A Smith; citation_author=CY Hayashi; citation_author=RV Lewis; citation_journal_title=Biomacromolecules; citation_volume=6; citation_number=38; citation_pages=3152-3159; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=N-terminal nonrepetitive domain common to dragline, flagelliform, and cylindriform spider silk proteins.; citation_author=A Rising; citation_author=G Hjälm; citation_author=W Engström; citation_author=J Johansson; citation_journal_title=Biomacromolecules; citation_volume=7; citation_number=39; citation_pages=3120-3124; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modular evolution of egg case silk genes across orb-weaving spider superfamilies.; citation_author=JE Garb; citation_author=CY Hayashi; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=40; citation_pages=11379-11384; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular architecture and evolution of a modular spider silk protein gene.; citation_author=CY Hayashi; citation_author=RV Lewis; citation_journal_title=Science; citation_volume=287; citation_number=41; citation_pages=1477-1479; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of the orb-web building spiders (Araneae, Orbiculariae: Deinopoidea, Araneoidea).; citation_author=CE Griswold; citation_author=JA Coddington; citation_author=G Hormiga; citation_author=N Scharff; citation_journal_title=Zool J Linn Soc–Lond; citation_volume=123; citation_number=42; citation_pages=1-99; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular and mechanical properties of major ampullate silk of the black widow spider, Latrodectus hesperus.; citation_author=BA Lawrence; citation_author=CA Vierra; citation_author=AF Moore; citation_journal_title=Biomacromolecules; citation_volume=5; citation_number=43; citation_pages=689-695; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gumfooted lines in black widow cobwebs and the mechanical properties of spider capture silk; citation_author=TA Blackledge; citation_author=AP Summers; citation_author=CY Hayashi; citation_journal_title=Zoology; citation_volume=108; citation_number=44; citation_pages=41-46; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Variation in the material properties of spider dragline silk across species; citation_author=BO Swanson; citation_author=TA Blackledge; citation_author=J Beltrán; citation_author=CY Hayashi; citation_journal_title=Appl Phys A; citation_volume=82; citation_number=45; citation_pages=213-218; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein and amino acid composition of silks from the cob weaver, Latrodectus hesperus (black widow).; citation_author=ML Casem; citation_author=D Turner; citation_author=K Houchin; citation_journal_title=Int J Biol Macromol; citation_volume=24; citation_number=46; citation_pages=103-108; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Amino acid composition of spider silks.; citation_author=SO Anderson; citation_journal_title=Comp Biochem Physiol; citation_volume=35; citation_number=47; citation_pages=705-711; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=The amino acid composition of major ampullate gland silk (dragline) of Nephila clavipes (Araneae, Tetragnathidae).; citation_author=SJ Lombardi; citation_author=DL Kaplan; citation_journal_title=J Arachnol; citation_volume=18; citation_number=48; citation_pages=297-306; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Evolution of repetitive proteins: spider silks from Nephila clavipes (Tetragnathidae) and Araneus bicentenarius (Araneidae).; citation_author=R Beckwitt; citation_author=S Arcidiacono; citation_author=R Stote; citation_journal_title=Insect Biochem Mol Biol; citation_volume=28; citation_number=49; citation_pages=121-130; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A simple method for displaying the hydropathic character of a protein.; citation_author=J Kyte; citation_author=R Doolittle; citation_journal_title=J Mol Biol; citation_volume=157; citation_number=50; citation_pages=105-132; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Comparative genomics.; citation_author=RC Hardison; citation_journal_title=PLoS Biology; citation_volume=1; citation_number=51; citation_pages=156-160; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational approaches to identify promoters and cis-regulatory elements in plant genomes.; citation_author=S Rombauts; citation_author=K Florquin; citation_author=M Lescot; citation_author=K Marchal; citation_author=P Rouzé; citation_journal_title=Plant Physiol; citation_volume=132; citation_number=52; citation_pages=1162-1176; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PipMaker–A web server for aligning two genomic DNA sequences.; citation_author=S Schwartz; citation_author=Z Zhang; citation_author=KA Frazer; citation_author=A Smit; citation_author=C Riemer; citation_journal_title=Genome Res; citation_volume=10; citation_number=53; citation_pages=577-586; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human-mouse alignments with BLASTZ.; citation_author=S Schwartz; citation_author=WJ Kent; citation_author=A Smit; citation_author=Z Zhang; citation_author=R Baertsch; citation_journal_title=Genome Res; citation_volume=13; citation_number=54; citation_pages=103-107; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Detecting the limits of regulatory element conservation and divergence estimation using pairwise and multiple alignments.; citation_author=DA Pollard; citation_author=AM Moses; citation_author=VN Iyer; citation_author=MB Eisen; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=55; citation_pages=376; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of multiple-species conserved sequences.; citation_author=EH Margulies; citation_author=M Blanchette; citation_author=D Haussler; citation_author=ED Green; citation_journal_title=Genome Res; citation_volume=13; citation_number=56; citation_pages=2507-2518; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=TRANSFAC® and its module TRANSCompel®: transcriptional gene regulation in eukaryotes.; citation_author=V Matys; citation_author=OV Kel-Margoulis; citation_author=E Fricke; citation_author=I Liebich; citation_author=S Land; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=57; citation_pages=D108-D110; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fundamentals of Molecular Evolution, second edition.; citation_author=D Graur; citation_author=WH Li; citation_number=58; citation_pages=481; citation_date=2000; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Detecting selection in noncoding regions of nucleotide sequences.; citation_author=WSW Wong; citation_author=R Nielsen; citation_journal_title=Genetics; citation_volume=167; citation_number=59; citation_pages=949-958; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=AVID: A Global Alignment Program.; citation_author=N Bray; citation_author=I Dubchak; citation_author=L Pachter; citation_journal_title=Genome Res; citation_volume=13; citation_number=60; citation_pages=97-102; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=61; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=SEGE: A database on ‘intron less/single exonic’ genes from eukaryotes.; citation_author=MK Sakharkar; citation_author=P Kangueane; citation_author=DA Petrov; citation_author=AS Kolaskar; citation_author=S Subbiah; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=62; citation_pages=1266-1267; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome SEGE: A database for ‘intronless’ genes in eukaryotic genomes.; citation_author=MK Sakharkar; citation_author=P Kangueane; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=63; citation_pages=67; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution by gene duplication: an update.; citation_author=J Zhang; citation_journal_title=Trends Ecol Evol; citation_volume=18; citation_number=64; citation_pages=292-298; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Selection for short introns in highly expressed genes.; citation_author=CI Castillo-Davis; citation_author=SL Mekhedov; citation_author=DL Hartl; citation_author=EV Koonin; citation_author=FA Kondrashov; citation_journal_title=Nat Genet; citation_volume=31; citation_number=65; citation_pages=415-418; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The signature of selection mediated by expression on human genes.; citation_author=AO Urrutia; citation_author=LD Hurst; citation_journal_title=Genome Res; citation_volume=13; citation_number=66; citation_pages=2260-2264; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Intron size and exon evolution in Drosophila.; citation_author=G Marais; citation_author=P Nouvellet; citation_author=PD Keightley; citation_author=B Charlesworth; citation_journal_title=Genetics; citation_volume=170; citation_number=67; citation_pages=481-485; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for diet effects on the composition of silk proteins produced by spiders.; citation_author=CL Craig; citation_author=C Riekel; citation_author=ME Herberstein; citation_author=RS Weber; citation_author=D Kaplan; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=68; citation_pages=1904-1913; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sequence conservation in the C-terminal region of spider silk proteins (spidroin) from Nephila clavipes (Tetragnathidae) and Araneus bicentenarius (Araneidae).; citation_author=R Beckwitt; citation_author=S Arcidiacono; citation_journal_title=J Biol Chem; citation_volume=269; citation_number=69; citation_pages=6661-6663; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Spider silk protein refolding is controlled by changing pH.; citation_author=C Dicko; citation_author=F Vollrath; citation_author=JM Kenney; citation_journal_title=Biomacromolecules; citation_volume=5; citation_number=70; citation_pages=704-710; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Lower Cretaceous spiders from the Sierra-de-Montsech, north-east Spain.; citation_author=PA Selden; citation_journal_title=Palaeontology; citation_volume=33; citation_number=71; citation_pages=257-285; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Utility of the nuclear protein-coding gene, elongation factor-1 gamma (EF-1γ), for spider systematics, emphasizing family level relationships of tarantulas and their kin (Araneae: Mygalomorphae).; citation_author=NA Ayoub; citation_author=JE Garb; citation_author=M Hedin; citation_author=CY Hayashi; citation_journal_title=Mol Phylogenet Evol; citation_volume=42; citation_number=72; citation_pages=394-409; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sequence turnover and tandem repeats in cis-regulatory modules in Drosophila.; citation_author=S Sinha; citation_author=ED Siggia; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=73; citation_pages=874-885; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Finding functional elements in Saccharomyces genomes by phylogenetic footprinting.; citation_author=P Cliften; citation_author=P Sudarsanam; citation_author=A Desikan; citation_author=L Fulton; citation_author=B Fulton; citation_journal_title=Science; citation_volume=301; citation_number=74; citation_pages=71-76; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=75; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Conserved noncoding sequences in the grasses.; citation_author=DC Inada; citation_author=A Bashir; citation_author=C Lee; citation_author=BC Thomas; citation_author=C Ko; citation_journal_title=Genome Res; citation_volume=13; citation_number=76; citation_pages=2030-2041; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic shadowing of primate sequences to find functional regions of the human genome.; citation_author=D Boffelli; citation_author=J McAuliffe; citation_author=D Ovcharenko; citation_author=KD Lewis; citation_author=I Ovcharenko; citation_journal_title=Science; citation_volume=299; citation_number=77; citation_pages=1391-1394; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular estimates of primate divergences and new hypotheses for primate dispersal and the origin of modern humans.; citation_author=U Arnason; citation_author=A Gullberg; citation_author=AS Burguete; citation_author=A Janice; citation_journal_title=Hereditas; citation_volume=133; citation_number=78; citation_pages=217-228; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogeny of the major tetrapod groups: morphological data and divergence dates.; citation_author=MJ Benton; citation_journal_title=J Mol Evol; citation_volume=30; citation_number=79; citation_pages=409-424; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Molecular clock calibration and metazoan divergence dates.; citation_author=MSY Lee; citation_journal_title=J Mol Evol; citation_volume=49; citation_number=80; citation_pages=385-391; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Comparative analyses of multi-species sequences from targeted genomic regions.; citation_author=JW Thomas; citation_author=JW Touchman; citation_author=RW Blakesley; citation_author=GG Bouffard; citation_author=SM Beckstrom-Sternberg; citation_journal_title=Nature; citation_volume=424; citation_number=81; citation_pages=788-793; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Discovery of regulatory elements in vertebrates through comparative genomics.; citation_author=A Prakash; citation_author=M Tompa; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=82; citation_pages=1249-1256; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Promoter analysis of MADS-box genes in eudicots through phylogenetic footprinting.; citation_author=S De Bodt; citation_author=G Theissen; citation_author=Y Van de Peer; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=83; citation_pages=1293-1303; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Dating the monocot-dicot divergence and the origin of core eudicots using whole chloroplast genomes.; citation_author=S-M Chaw; citation_author=C-C Chang; citation_author=H-L Chen; citation_author=W-H Li; citation_journal_title=J Mol Evol; citation_volume=58; citation_number=84; citation_pages=424-441; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of the angiosperms: calibrating the family tree.; citation_author=N Wikström; citation_author=V Savolainen; citation_author=MW Chase; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=268; citation_number=85; citation_pages=2211-2220; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Transcriptional activation by heterodimers of the achaete-scute and daughterless gene products of Drosophila.; citation_author=CV Cabrera; citation_author=MC Alonso; citation_journal_title=EMBO J; citation_volume=10; citation_number=86; citation_pages=2965-2973; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The expression and role of a proneural gene, achaete, in the development of the larval nervous system of Drosophila.; citation_author=M Ruiz-Gómez; citation_author=A Ghysen; citation_journal_title=EMBO J; citation_volume=12; citation_number=87; citation_pages=1121-1130; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Characterization of a novel class II bHLH transcription factor from the black widow spider, Latrodectus hesperus, with silk-gland restricted patterns of expression.; citation_author=K Kohler; citation_author=W Thayer; citation_author=T Le; citation_author=A Sembhi; citation_author=K Vasanthavada; citation_journal_title=DNA Cell Biol.; citation_volume=24; citation_number=88; citation_pages=371-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Drosophila gene Bearded encodes a novel small protein and shares 3′ UTR sequence motifs with multiple Enhancer of split Complex genes.; citation_author=MW Leviten; citation_author=EC Lai; citation_author=JW Posakony; citation_journal_title=Development; citation_volume=124; citation_number=89; citation_pages=4039-4051; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A complex pattern of evolutionary conservation and alternative polyadenylation within the long 3′-untranslated region of the methyl-CpG-binding protein 2 gene (MeCP2) suggests a regulatory role in gene expression.; citation_author=JF Coy; citation_author=Z Sedlacek; citation_author=D Bächner; citation_author=H Delius; citation_author=A Poustka; citation_journal_title=Hum Mol Genet; citation_volume=8; citation_number=90; citation_pages=1253-1262; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals.; citation_author=X Xie; citation_author=J Lu; citation_author=EJ Kulbokas; citation_author=TR Golub; citation_author=V Mootha; citation_journal_title=Nature; citation_volume=434; citation_number=91; citation_pages=338-345; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A genetic pathway for regulation of tra-2 translation.; citation_author=EB Goodwin; citation_author=K Hofstra; citation_author=CA Hurney; citation_author=S Mango; citation_author=J Kimble; citation_journal_title=Development; citation_volume=124; citation_number=92; citation_pages=749-758; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A PUF family portrait: 3′ UTR regulation as a way of life.; citation_author=M Wickens; citation_author=DS Bernstein; citation_author=J Kimble; citation_author=R Parker; citation_journal_title=Trends Genet; citation_volume=18; citation_number=93; citation_pages=150-157; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Enhancer of split [E(spl)] locus of Drosophila encodes seven independent helix-loop-helix proteins.; citation_author=C Delidakis; citation_author=S Artavanis-Tsakonas; citation_journal_title=Proc Natl Acad USA; citation_volume=89; citation_number=94; citation_pages=8731-8735; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The Bearded box, a novel 3′ UTR sequence motif, mediates negative post-transcriptional regulation of Bearded and Enhancer of split Complex gene expression.; citation_author=EC Lai; citation_author=JW Posakony; citation_journal_title=Development; citation_volume=124; citation_number=95; citation_pages=4847-4856; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The K box, a conserved 3′ UTR sequence motif, negatively regulates accumulation of Enhancer of split Complex transcripts.; citation_author=EC Lai; citation_author=C Burks; citation_author=JW Posakony; citation_journal_title=Development; citation_volume=125; citation_number=96; citation_pages=4077-4088; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Purification and characterization of recombinant spider silk expressed in Escherichia coli.; citation_author=S Arcidiacono; citation_author=C Mello; citation_author=D Kaplan; citation_author=S Cheley; citation_author=H Bayley; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=49; citation_number=97; citation_pages=31-38; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Primary structure elements of spider dragline silks and their contribution to protein solubility.; citation_author=D Huemmerich; citation_author=CW Helsen; citation_author=S Quedzuweit; citation_author=J Oschmann; citation_author=R Rudolph; citation_journal_title=Biochemistry; citation_volume=43; citation_number=98; citation_pages=13604-13612; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome sizes of spiders.; citation_author=TR Gregory; citation_author=DP Shorthouse; citation_journal_title=J Hered; citation_volume=94; citation_number=99; citation_pages=285-290; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual, third edition. Cold Spring Harbor Laboratory. 999 p.; citation_author=J Sambrook; citation_author=DW Russell; citation_number=100; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sánchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=101; citation_pages=2496-2497; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). Version 4.; citation_author=DL Swofford; citation_number=102; citation_date=2002; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=103; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.; citation_author=M Hasegawa; citation_author=H Kishino; citation_author=T Yano; citation_journal_title=J Mol Evol; citation_volume=22; citation_number=104; citation_pages=160-174; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees.; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=105; citation_pages=512-526; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models DOI: 10.1093/bioinformatics/btg180.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=106; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=VISTA: Visualizing global DNA sequence alignments of arbitrary length.; citation_author=C Mayor; citation_author=M Brudno; citation_author=JR Schwartz; citation_author=A Poliakov; citation_author=EM Rubin; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=107; citation_pages=1046-1047; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=VISTA: computational tools for comparative genomics.; citation_author=KA Frazer; citation_author=L Pachter; citation_author=A Poliakov; citation_author=EM Rubin; citation_author=I Dubchak; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=108; citation_pages=W273-W279; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes"/>
    <meta name="twitter:description" content="Spider dragline (major ampullate) silk outperforms virtually all other natural and manmade materials in terms of tensile strength and toughness. For this reason, the mass-production of artificial spider silks through transgenic technologies has been a major goal of biomimetics research. Although all known arthropod silk proteins are extremely large (&gt;200 kiloDaltons), recombinant spider silks have been designed from short and incomplete cDNAs, the only available sequences. Here we describe the first full-length spider silk gene sequences and their flanking regions. These genes encode the MaSp1 and MaSp2 proteins that compose the black widow's high-performance dragline silk. Each gene includes a single enormous exon (&gt;9000 base pairs) that translates into a highly repetitive polypeptide. Patterns of variation among sequence repeats at the amino acid and nucleotide levels indicate that the interaction of selection, intergenic recombination, and intragenic recombination governs the evolution of these highly unusual, modular proteins. Phylogenetic footprinting revealed putative regulatory elements in non-coding flanking sequences. Conservation of both upstream and downstream flanking sequences was especially striking between the two paralogous black widow major ampullate silk genes. Because these genes are co-expressed within the same silk gland, there may have been selection for similarity in regulatory regions. Our new data provide complete templates for synthesis of recombinant silk proteins that significantly improve the degree to which artificial silks mimic natural spider dragline fibers."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000514.g006"/>

  <meta property="og:title" content="Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6844'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5555'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6765&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000514">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000514" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000514&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nadia A. Ayoub
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:nadiaa@ucr.edu">nadiaa@ucr.edu</a></p>

                <p>Affiliation:
                  Department of Biology, University of California Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jessica E. Garb, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of California Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robin M. Tinghitella, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of California Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew A. Collin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of California Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cheryl Y. Hayashi
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, University of California Riverside, Riverside, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 13, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000514</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000514-g001" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-g002" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-t001" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-t002" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-g003" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-g004" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-g005" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000514-g006" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000514">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000514" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Spider dragline (major ampullate) silk outperforms virtually all other natural and manmade materials in terms of tensile strength and toughness. For this reason, the mass-production of artificial spider silks through transgenic technologies has been a major goal of biomimetics research. Although all known arthropod silk proteins are extremely large (&gt;200 kiloDaltons), recombinant spider silks have been designed from short and incomplete cDNAs, the only available sequences. Here we describe the first full-length spider silk gene sequences and their flanking regions. These genes encode the MaSp1 and MaSp2 proteins that compose the black widow's high-performance dragline silk. Each gene includes a single enormous exon (&gt;9000 base pairs) that translates into a highly repetitive polypeptide. Patterns of variation among sequence repeats at the amino acid and nucleotide levels indicate that the interaction of selection, intergenic recombination, and intragenic recombination governs the evolution of these highly unusual, modular proteins. Phylogenetic footprinting revealed putative regulatory elements in non-coding flanking sequences. Conservation of both upstream and downstream flanking sequences was especially striking between the two paralogous black widow major ampullate silk genes. Because these genes are co-expressed within the same silk gland, there may have been selection for similarity in regulatory regions. Our new data provide complete templates for synthesis of recombinant silk proteins that significantly improve the degree to which artificial silks mimic natural spider dragline fibers.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ayoub NA, Garb JE, Tinghitella RM, Collin MA, Hayashi CY (2007) Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes. PLoS ONE 2(6):
          e514.
            doi:10.1371/journal.pone.0000514</p><p><strong>Academic Editor: </strong>Robert DeSalle, American Museum of Natural History, United States of America</p><p><strong>Received:</strong> January 9, 2007; <strong>Accepted:</strong> May 12, 2007; <strong>Published:</strong> June 13, 2007</p><p><strong>Copyright:</strong> © 2007 Ayoub et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The research was funded by the Army Research Office (DAAD19-02-1-0358 and W911NF-06-1-0455) and the National Science Foundation (DEB-0236020).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Spider silks have received much economic and biomedical attention because of their outstanding mechanical properties <a href="#pone.0000514-Vollrath1">[e.g. 1]</a>–<a href="#pone.0000514-Foo1">[3]</a>. For example, the dragline silk of araneoids (ecribellate orb-weaving spiders and their relatives) displays both high tensile strength and extensibility, making it tougher than nearly all other natural or synthetic materials <a href="#pone.0000514-Gosline2">[4]</a>–<a href="#pone.0000514-Gosline3">[6]</a>. Spider silks are primarily composed of proteins that are synthesized in specialized abdominal glands. An individual orb-weaving spider spins up to five different types of silk fibers, each serving critical ecological functions, including prey capture, shelter, predator avoidance, egg protection, and dispersal <a href="#pone.0000514-Foelix1">[7]</a>–<a href="#pone.0000514-Blackledge1">[8]</a>. Each distinct fiber is made from one or two unique types of silk structural proteins (fibroins), almost all of which are encoded by members of a single gene family <a href="#pone.0000514-Guerette1">[9]</a>–<a href="#pone.0000514-Garb1">[12]</a>. Thus, the spectacular diversity of spider silk proteins evolved through successive rounds of gene duplication and divergence.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Spider fibroins have very high molecular weights, estimated at 200–350 kiloDaltons <a href="#pone.0000514-Hayashi1">[13]</a> with transcript sizes of approximately 10,000 base pairs (bp) or larger. Such considerable size is conserved over a diverse range of spider species and fibroin types <a href="#pone.0000514-Hayashi1">[13]</a>–<a href="#pone.0000514-Zhao1">[16]</a>. Partial-length complementary DNA (cDNA) sequences indicate that silk proteins are highly modular; each polypeptide is primarily composed of an uninterrupted block of repetitive sequence that is flanked on both sides by ~100 amino acids (aa) of non-repetitive amino- (N-) and carboxy- (C-) termini. The sequence attributes of the repetitive region vary according to silk protein type, with some fibroins containing short, simple repeat units, and others composed of longer, more complex repeats <a href="#pone.0000514-Gatesy1">[10]</a>, <a href="#pone.0000514-Hayashi2">[14]</a>. Because of the difficulty associated with cloning long stretches of repetitive DNA, only two full-length cDNA silk sequences have been characterized <a href="#pone.0000514-Zhao1">[16]</a>. These cDNAs encode the silk proteins that form the egg case fibers of the orb-weaving spider, <em>Argiope bruennichi</em>. Egg case fibers, however, have substantially lower tensile strength and toughness than dragline silk <a href="#pone.0000514-Blackledge1">[8]</a>, <a href="#pone.0000514-Zhao1">[16]</a>. Complete gene sequences are still unknown for any spider silk.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Because of its extremely high tensile strength and toughness, dragline (major ampullate) silk has received the most attention of the spider silks. This silk is composed of two types of fibroins, MaSp1 <a href="#pone.0000514-Xu1">[17]</a> and MaSp2 <a href="#pone.0000514-Hinman2">[18]</a>. The genes encoding these proteins are co-expressed in the major ampullate silk glands, and both proteins are found throughout the fiber <a href="#pone.0000514-Sponner1">[15]</a>, <a href="#pone.0000514-Sponner2">[19]</a>. Short glycine-rich regions (GGX, where X represents a subset of aa) followed by a stretch of multiple alanines (poly-A) characterize both proteins. The ubiquitous poly-A stretches are hypothesized to form hydrophobic crystalline domains that are responsible for the high tensile strength of the fiber <a href="#pone.0000514-Simmons1">[20]</a>–<a href="#pone.0000514-Parkhe1">[24]</a>. In contrast, the glycine-rich regions are hydrophilic with runs of the peptide motif GGX conforming to a 3<sub>1</sub>-helix <a href="#pone.0000514-Kmmerlen1">[21]</a>, <a href="#pone.0000514-Dong1">[25]</a>. While poly-A and GGX motifs describe almost all of MaSp1, MaSp2 also has a large proportion of GPG motifs <a href="#pone.0000514-Hinman2">[18]</a>. These proline-containing repeats likely form type II beta-turns, and such kinks in part explain the reversible extensibility of dragline fiber <a href="#pone.0000514-Hayashi1">[13]</a>, <a href="#pone.0000514-Sponner2">[19]</a>, <a href="#pone.0000514-Hayashi3">[26]</a>–<a href="#pone.0000514-vanBeek1">[27]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Much of the applied research on spider silks has focused on mass-producing silk fibers for industrial use <a href="#pone.0000514-OBrien1">[e.g. 28]</a>. However, unlike domesticated silkworm caterpillars, spiders cannot be readily farmed for silk because they are predatory and cannibalistic. Instead, researchers have created biomimetic silks through transgenic technologies using partial-length silk cDNA sequences <a href="#pone.0000514-Hinman3">[e.g. 29]</a>–<a href="#pone.0000514-Bini1">[32]</a>. These manmade silks have, thus far, fallen short of native dragline silk in both sequence and mechanical properties <a href="#pone.0000514-Lazaris1">[e.g. 30]</a>, <a href="#pone.0000514-Scheller2">[33]</a>–<a href="#pone.0000514-Ittah1">[35]</a>. All known arthropod fibroins are extremely large, including the convergently evolved heavy-chain fibroin of lepidopterans (~370 kiloDaltons <a href="#pone.0000514-Sprague1">[36]</a>). Such evolutionary convergence among distantly related silk spinning species suggests that large size is a critical molecular feature for silk fiber mechanical performance. For example, larger fibroins possess more repeat units (such as poly-A motifs that crosslink to form crystalline domains) than shorter fibroins, thereby increasing the number of interactions among monomers. However, large size is probably not the only key functional attribute of spider fibroins. The evolutionarily conserved C-termini of MaSp1 and MaSp2 aid in conversion of the liquid silk dope into a solid fiber <a href="#pone.0000514-Sponner3">[37]</a>, and facilitate assembly of the fiber's characteristic crystalline domains <a href="#pone.0000514-Ittah1">[35]</a>. The few known N-termini are even more conserved than the C-termini <a href="#pone.0000514-MotriukSmith1">[38]</a>–<a href="#pone.0000514-Rising1">[39]</a>, which suggests that N-termini are also necessary for proper fiber assembly and may influence the mechanical properties of spider silk. Thus, determining the entire coding sequences for MaSp1 and MaSp2 is a key step in the generation of recombinant silks that closely mimic natural spider dragline silk.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Full-length silk sequences are also crucial for understanding the molecular evolutionary dynamics of long, repetitive genes. To date, it has been assumed that the modular organization of fibroins is maintained throughout the entirety of the amino acid sequence. Yet, some molecular evidence is consistent with past recombination events between <em>MaSp1</em> and <em>MaSp2</em> <a href="#pone.0000514-Gatesy1">[10]</a>, <a href="#pone.0000514-Garb1">[12]</a>, <a href="#pone.0000514-Garb2">[40]</a>, and thus repeat units may have transferred from one gene to the other. Moreover, there is sparse information about the exon-intron structure of spider silk genes. <em>Flag</em> (the gene encoding the capture spiral filament) from <em>Nephila clavipes</em> <a href="#pone.0000514-Hayashi4">[41]</a> and <em>MaSp2</em> from <em>Argiope trifasciata</em> <a href="#pone.0000514-MotriukSmith1">[38]</a> both show highly repetitive exon-intron gene organizations in which sequential introns <span class="underline">within</span> a gene have nearly identical nucleotide sequences. The presence of iterated introns and exons is evidence that intragenic recombination can swiftly homogenize (eliminate or spread new variants throughout) the sequence of an entire silk gene. It is unknown if this atypical gene architecture is a common feature of spider silk genes.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>We have constructed a genomic library for the black widow spider, <em>Latrodectus hesperus</em> (Theridiidae), in order to identify full-length silk genes and their associated regulatory regions. Black widows, notorious for their neurotoxic venom, are members of the Araneoidea, a superfamily of orb-weaving spiders and their close relatives. Black widows are descended from orb-web weaving ancestors, but they build three-dimensional cobwebs rather than the symmetrical, wagon-wheel shaped orb-web <a href="#pone.0000514-Griswold1">[42]</a>. Despite this difference in web architecture, the breaking strength and extensibility of <em>Latrodectus</em> dragline silk are equal to or higher than those of true orb-weaving spiders <a href="#pone.0000514-Lawrence1">[43]</a>–<a href="#pone.0000514-Swanson1">[45]</a>. Here we report complete gene sequences of <em>MaSp1</em> and <em>MaSp2</em> as well as adjacent non-coding regions. We document the existence of higher-order repeat units that range from ~70 to over 2,000 bp, and show that the repetitive sequences of <em>MaSp1</em> are more homogenized than those of <em>MaSp2</em>. We also demonstrate marked evolutionary conservation of N-terminal and upstream non-coding regions between paralogs within a species and across orthologs from divergent species. Based on these multi-gene comparisons, we identify putative regulatory sequences that may be involved in co-expression of the two major ampullate silk genes. Collectively, our data provide the first templates for complete recombinant major ampullate fibroins and illustrate the dramatic effects of intragenic and intergenic recombination in the evolution of these extraordinarily modular genes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Large, single exon gene organizations</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We sequenced two fosmid clones each containing ~37,000 bp of the black widow genome. One clone (GenBank accession EF595246) encompassed the complete coding sequence for the dragline silk gene <em>MaSp1</em> as well as 9,928 bp upstream of its start codon and 14,728 bp downstream of its stop codon. The <em>MaSp1</em> gene is composed of a single exon with 9,390 bp encoding 3,129 aa (<a href="#pone-0000514-g001">Figure 1</a>). The second clone (EF595245) includes the entire coding sequence for <em>MaSp2</em> plus 17,205 bp of upstream and 8,546 bp of downstream flanking sequence. Like <em>MaSp1</em>, the <em>MaSp2</em> gene contains one enormous exon with 11,340 bp encoding 3,779 aa (<a href="#pone-0000514-g002">Figure 2</a>). Both <em>MaSp1</em> and <em>MaSp2</em> genes contain sequences that match partial-length cDNAs from <em>L. hesperus</em> silk gland expression libraries <a href="#pone.0000514-Garb2">[40]</a>, <a href="#pone.0000514-Lawrence1">[43]</a>, indicating that these genes are transcribed. The C-terminal coding region (~300 bp) of the <em>MaSp1</em> gene is 97% identical to the corresponding 3′ partial <em>MaSp1</em> cDNA clones (AY953074, DQ409057) and the N-terminal coding region (~450 bp) is 99.8% identical to our 5′ partial cDNA clone (EF595247). Both the C-terminal coding region and the 3′ untranslated region (UTR) of the <em>MaSp2</em> gene share 99% sequence identity with 3′ partial <em>MaSp2</em> cDNA clones (AY953075, DQ409058). Similarly, the N-terminal coding regions of the <em>MaSp2</em> gene and our 5′ partial cDNA (EF595248) are 95.5% identical.</p>
<div class="figure" id="pone-0000514-g001"><div class="img"><a name="pone-0000514-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Complete amino acid sequence for <em>L. hesperus</em> major ampullate spidroin 1 (MaSp1).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The sequence is read from left to right and then top to bottom. The diamond marks the start position and the asterisk denotes the stop position. The protein is dominated by poly-A (red) and GGX (green) motifs. The majority of the sequence can be categorized into four types of ensemble repeat units. Repeats of each type are aligned within a box. Gaps (-) have been inserted in order to align repeat units within a type.</p>
<span>doi:10.1371/journal.pone.0000514.g001</span></div><div class="figure" id="pone-0000514-g002"><div class="img"><a name="pone-0000514-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Complete amino acid sequence for <em>L. hesperus</em> major ampullate spidroin 2 (MaSp2).</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The sequence is read from left to right and then top to bottom. Start position, stop position, and alignment gaps are indicated as for MaSp1 (<a href="#pone-0000514-g001">Figure 1</a>). MaSp2 is characterized by poly-A (red), GGX (green), GPX (blue), and QQ (purple) motifs. There are four types of ensemble repeats. Repeats of each type are aligned within a box, except for Type 1, which is separated into two boxes because it is approximately twice as prevalent as any other repeat type. Right and left pointing arrows mark beginnings and ends of two near-perfect repeats of 778 aa.</p>
<span>doi:10.1371/journal.pone.0000514.g002</span></div>

<h4>Extreme sequence modularity</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Glycine and alanine are by far the most abundant amino acids in our predicted <em>L. hesperus</em> MaSp1 and MaSp2 fibroins. These two amino acids constitute greater than 64% of both sequences, followed by glutamine in MaSp1 and proline in MaSp2 (<a href="#pone-0000514-t001">Table 1</a>). These values closely match published amino acid compositions of major ampullate silk from black widows <a href="#pone.0000514-Casem1">[46]</a> and other araneoid spiders <a href="#pone.0000514-Anderson1">[47]</a>–<a href="#pone.0000514-Lombardi1">[48]</a>, further confirming that our genes encode the two dominant protein components of major ampullate silk. Because the first two codon positions for alanine, glycine, and proline are guanine or cytosine, the base compositions of these genes are guanine/cytosine-rich (<em>MaSp1</em>–61%; <em>MaSp2</em>–59%). However, overall base compositions are not highly skewed because the third positions for these codons in the <em>L. hesperus MaSp1</em> and <em>MaSp2</em> are extremely biased towards adenine and also strongly biased, but less dramatically, towards thymine (86% of <em>MaSp1</em>, 91% of <em>MaSp2</em> glycine, alanine, and proline codons end with adenine or thymine; <a href="#pone-0000514-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000514-t001"><div class="img"><a name="pone-0000514-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Amino acid content and codon usage for the most common amino acids of black widow MaSp1 and MaSp2.</span></strong></p><span>doi:10.1371/journal.pone.0000514.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The repetitive region of the <em>L. hesperus MaSp1</em> translation is dominated by amino acid sequence motifs commonly found in MaSp1 of other spider species: GGX (X = A, Q, or Y), GX (X = Q, A, or R), and poly-A (4–10 consecutive alanines, mean number = 7.7) <a href="#pone.0000514-Gatesy1">[10]</a>, <a href="#pone.0000514-Xu1">[17]</a>, <a href="#pone.0000514-Beckwitt1">[49]</a>. These motifs are organized into four types of ensemble (higher order) repeat units, with each ensemble consisting of a glycine-rich region followed by a poly-A region (<a href="#pone-0000514-g001">Figure 1</a>). Starting at residue 542, the different ensemble types are tandemly arrayed in a consistent pattern, and this aggregate of four ensembles is iterated 20 times with near perfect fidelity. Pairwise amino acid differences between aggregates are extremely low, ranging from 0.0 to 4.3% and averaging 1.9%. This remarkable sequence homogeneity is also maintained at the nucleotide level with average uncorrected pairwise differences of only 2.5% (range = 0.3–6.3%).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The repetitive region of the <em>L. hesperus</em> MaSp2 amino acid sequence is characterized by a larger suite of motifs than MaSp1. The common MaSp2 motifs include GPX (X = G or S), QQ, GGX (X is usually A), GSG, and poly-A (3–9 consecutive alanines, mean number = 6.7). Similar to MaSp1, these motifs are organized into four types of ensemble repeat units that each contain one poly-A motif (<a href="#pone-0000514-g002">Figure 2</a>). However, the four types of MaSp2 ensembles are more variable than those of MaSp1, with pairwise amino acid differences between ensembles of the same type as high as 36% (<a href="#pone-0000514-t002">Table 2</a>). In addition, the MaSp2 ensemble types are not always strung together in the same order and do not form clearly discernible higher-level aggregates (<a href="#pone-0000514-g002">Figure 2</a>). Nevertheless, there is a pair of 778 amino acid long tandem repeats that differ by a scant five aa (<a href="#pone-0000514-g002">Figure 2</a>). The 2,334 nucleotides encoding each repeat vary at only six positions (&gt;99.7% identity).</p>
<div class="figure" id="pone-0000514-t002"><div class="img"><a name="pone-0000514-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Prevalence (#) and average pairwise amino acid differences between MaSp2 ensemble repeats of the same type.</span></strong></p><span>doi:10.1371/journal.pone.0000514.t002</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Using the method of Kyte and Doolittle <a href="#pone.0000514-Kyte1">[50]</a>, we predicted the hydrophilicity of <em>L. hesperus</em> MaSp1 and MaSp2. Both fibroins show regions of hydrophobicity (corresponding to the poly-A motifs) and hydrophilicity (corresponding to the glycine-rich regions) that alternate throughout the entire repetitive portions of the two proteins (<a href="#pone-0000514-g003">Figure 3A</a>). Both MaSp1 and MaSp2 repetitive regions are slightly hydrophilic when averaged across all residues (MaSp1 average = 0.13 on the Kyte-Doolittle scale; MaSp2 average = 0.14) but MaSp2 displays higher amplitude of hydrophilicity (MaSp1 max = 2.0; MaSp2 max = 2.6) (<a href="#pone-0000514-g003">Figure 3A</a>). The N- and C-terminal domains show a similar pattern of alternating hydrophobicity and hydrophilicity but are generally more hydrophobic than the repetitive regions (average hydrophilicity across residues: MaSp1 N-terminus = -0.29, C-terminus = −0.44; MaSp2 N-terminus = −0.34; C-terminus = −0.31; negative values indicate degree of hydrophobicity). The most hydrophobic region of both fibroins is found at the beginning of the N-terminus (<a href="#pone-0000514-g003">Figure 3B</a>).</p>
<div class="figure" id="pone-0000514-g003"><div class="img"><a name="pone-0000514-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Kyte and Doolittle <a href="#pone.0000514-Kyte1">[50]</a> hydrophilicity plots for <em>L. hesperus</em> MaSp1 and MaSp2.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Scan window size = 7. Negative values indicate hydrophobicity. (A) Complete proteins. (B) Non-repetitive terminal regions.</p>
<span>doi:10.1371/journal.pone.0000514.g003</span></div>

<h4>Congruence between silk N- and C-termini evolutionary relationships</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We aligned the N-terminal regions of <em>L. hesperus</em> MaSp1 and MaSp2 with N-termini from other spider fibroins (<a href="#pone-0000514-g004">Figure 4A</a>). These proteins are constituents of three spider silk fiber types: the dragline silk composed of MaSp1 and MaSp2, the capture spiral filament of flagelliform silk protein (Flag), and the eggcase fibers produced from tubuliform (also called cylindrical) gland proteins (TuSp1, CySp1 and CySp2). For each of these N-termini, we also aligned the corresponding C-termini, if available (<a href="#pone-0000514-g004">Figure 4B</a>). However, only in the case of the <em>L. hesperus MaSp1</em> and <em>MaSp2</em> that we report here, and the full-length <em>CySp1</em> and <em>CySp2</em> cDNAs from <em>Argiope bruennichi</em>, is it certain that the N- and C-termini coding regions belong to the same gene. All others were partial 5′ or 3′ sequences that were assumed to represent the ends of the same gene.</p>
<div class="figure" id="pone-0000514-g004"><div class="img"><a name="pone-0000514-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of N-termini, C-termini and repeat units of spider silk proteins.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Alignment of published N-terminal amino acid sequences. Amino acids shared by ≥50% of proteins are highlighted in grey. Gaps are represented by dashes and missing characters by question marks. (B) Alignment of corresponding C-terminal amino acid sequences. Taxa with an asterisk result from partial sequencing and are presumed to belong to the same locus as the N-terminal sequences. (C) MP trees of N and C-terminal encoding sequences treating gaps as a fifth state and employing midpoint rooting. Left tree length = 1449 (N-terminus); Right tree length = 838 (C-terminus). Dots represent nodes with &gt;75% bootstrap support in all MP and ML analyses and &gt;95% Bayesian posterior probability. (D) Exemplar repeat units for each of the major ampullate fibroins and representative TuSp1 and Flag repeats. Amino acid motifs are colored as in <a href="#pone-0000514-g002">Figure 2</a>. Abbreviations: LhMaSp2, <em>Latrodectus hesperus MaSp2</em> (EF595245); LhMaSp1, <em>L. hesperus MaSp1</em> (EF595246); LgMaSp1, <em>L. geometricus MaSp1</em> (5′ sequence: DQ059133S1, 3′ sequence: DQ059133S2); NiMaSp2, <em>Nephila inaurata madagascariensis MaSp2</em> (5′ sequence: DQ059135, 3′ sequence: AF350278); AtMaSp2, <em>Argiope trifasciata MaSp2</em> (5′ sequence: DQ059136, 3′ sequence: AF350266); EaMaSp1; <em>Euprosthenops australis MaSp1</em> (AM259067); LhTuSp1, <em>L. hesperus TuSp1</em> (5′ sequence: DQ379383, 3′ sequence: AY953070); AbCySp1, <em>A. bruennichi CySp1</em> AB242144; AbCySp2, <em>A. bruennichi CySp2</em> (AB242145); NcaCySp1, <em>N. clavata CySp1</em> (5′ sequence: AB218974, 3′ sequence: AB218973); NiFlag, <em>N. i. madagascariensis Flag</em> (5′ sequence: AF218623S1, 3′ sequence: AF218623S2); NclFlag, <em>N. clavipes Flag</em> (5′ sequence: AF027972, 3′ sequence: AF027973).</p>
<span>doi:10.1371/journal.pone.0000514.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We assessed evolutionary relationships among the N-termini and C-termini encoding sequences using maximum likelihood (ML), maximum parsimony (MP), and Bayesian phylogenetic methods. All methods produced similar relationships among N-terminal sequences (<a href="#pone-0000514-g004">Figure 4C</a>). A clade of eggcase silks (TuSp1, CySp1 and CySp2) was always well-supported. A major ampullate silk clade (MaSp1 and MaSp2) was consistently recovered with greater than 90% bootstrap support and 100% posterior probability. Despite the distinct differences between the repetitive portions of MaSp1 and MaSp2 (<a href="#pone-0000514-g001">Figures 1</a>, <a href="#pone-0000514-g002">2</a>, <a href="#pone-0000514-g004">4D</a>), all N-termini analyses strongly supported a grouping of <em>Latrodectus</em> MaSp1 and MaSp2, rather than a multi-species MaSp1 clade that is distinct from a MaSp2 clade. Within <em>Latrodectus</em>, however, <em>L. hesperus</em> MaSp1 grouped with <em>L. geometricus</em> MaSp1.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Relationships among the corresponding C-terminal encoding sequences typically mirrored those of the N-terminal encoding sequences (<a href="#pone-0000514-g004">Figure 4C</a>). However, placement of the <em>L. hesperus</em> TuSp1 C-terminus was unstable. Depending on the type of analysis, it grouped with Flag, CySp1 and CySp2, or MaSp1 and MaSp2, but always with low support (less than 75% bootstrap support or 95% posterior probability). A MaSp1 and MaSp2 C-terminal clade was consistently recovered with high support values, and nested within it, a <em>Latrodectus</em> MaSp1 and MaSp2 sub-clade. The only difference among analyses was that <em>L. hesperus</em> MaSp1 grouped with either <em>L. geometricus</em> MaSp1 (ML and Bayesian trees) or <em>L. hesperus</em> MaSp2 (in the MP trees).</p>


<h4>Multi-species comparisons identify conserved non-coding sequences</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Phylogenetic footprinting is a powerful approach for discovering putative gene regulatory regions. This method generally relies on alignments of orthologous, non-coding sequences from multiple species <a href="#pone.0000514-Hardison1">[51]</a>. The presence of conserved non-coding nucleotide stretches implies that a region is under selective constraint and therefore is likely to perform an important function. A similar approach can be applied to the non-coding sequences of co-regulated genes <a href="#pone.0000514-Rombauts1">[52]</a>. We thus compared flanking sequences of <em>L. hesperus MaSp1</em> and <em>MaSp2</em>, paralogous genes which are simultaneously expressed <a href="#pone.0000514-Sponner1">[15]</a>, <a href="#pone.0000514-Sponner2">[19]</a>. We also analyzed these sequences with available flanking sequences of <em>MaSp1</em> and <em>MaSp2</em> from other spider species. Because the <em>L. hesperus MaSp2</em> clone contained another open reading frame (ORF) 2,611 bp upstream of the <em>MaSp2</em> start codon, we limited comparisons to ~2,500 bp of upstream sequence. Using MultiPipMaker <a href="#pone.0000514-Schwartz1">[53]</a>, we identified regions that could be reliably aligned among <em>L. hesperus MaSp1</em> and <em>MaSp2, L. geometricus MaSp1</em> (5′: DQ059133S1, 3′: DQ059133S2), <em>Argiope trifasciata MaSp2</em> (DQ059136), and <em>Nephila inaurata madagascariensis MaSp2</em> (DQ059135; only 700 bp upstream sequence available). Downstream genomic sequences were not available for <em>Argiope</em> and <em>Nephila MaSp2</em>. MultiPipMaker generates local alignments using the BLASTZ algorithm and only produces an alignment if identity among sequences exceeds a threshold, below which alignments are considered random <a href="#pone.0000514-Schwartz2"></a>,<a href="#pone.0000514-Pollard1">[54−55]</a>. Margulies et al. <a href="#pone.0000514-Margulies1">[56]</a> argued that pairwise alignments are unreliable for detecting regulatory elements. Thus, we focused on conserved regions found in at least three sequences. When attempting to align only upstream non-coding sequence, MultiPipMaker produced alignments among <em>Latrodectus</em> sequences but not between <em>Latrodectus</em> and <em>Argiope</em> or <em>Nephila</em>. When the coding sequences were included as an anchor, a span of ~90 bp directly upstream of the start codon could be aligned among all 5 genes. This region included the conserved motif CACG and the TATA box, which were also identified by Motriuk-Smith et al. <a href="#pone.0000514-MotriukSmith1">[38]</a>. While the TATA box is thought to guide RNA polymerase II to the transcription initiation site in many eukaryotic genes, the motif CACG represents a potentially novel regulatory element for spider silk genes. Approximately 150 bp of sequence upstream from the start codon could be aligned among the three <em>Latrodectus</em> genes and ~300 bp upstream sequence between <em>L. hesperus MaSp1</em> and <em>MaSp2</em>. Additionally, ~180 bp of sequence downstream of the stop codon could be aligned among all three <em>Latrodectus</em> genes.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We further investigated the regions of similarity identified among the <em>Latrodectus</em> non-coding sequences by creating global alignments of the ~300 bp region upstream of the start codon and of the ~180 bp segment downstream of the stop codon. In addition to the CACG motif and TATA box found among all sequences examined, the three <em>Latrodectus</em> upstream sequences share a 15 bp motif found ~110 bp upstream of the start codon that has only 2 variable positions. When scanned against the TRANSFAC database <a href="#pone.0000514-Matys1">[57]</a>, this conserved region perfectly matches a 6 bp binding site for the Achaete-Scute family of transcription factors.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>We also compared nucleotide substitution rates for various regions of the <em>Latrodectus</em> sequences (<a href="#pone-0000514-g005">Figure 5</a>). To detect selection on protein coding sequences, we calculated the ratio of the number of nonsynonymous substitutions per nonsynonymous site (K<sub>n</sub>) to the number of synonymous substitutions per synonymous site (K<sub>s</sub>) <a href="#pone.0000514-Graur1">[58]</a>. As expected for evolutionarily conserved proteins, we found K<sub>n</sub>/K<sub>s</sub> was very low, ranging from 0.05 to 0.20 for <em>Latrodectus MaSp1</em> and <em>MaSp2</em> terminal coding regions, suggesting strong purifying selection (<a href="#pone-0000514-g005">Figure 5</a>). We applied an analogous approach (as in Wong&amp;Nielsen <a href="#pone.0000514-Wong1">[59]</a>) to estimate selective pressures in non-coding sequences by calculating the ratio of the number of substitutions per site (K) to K<sub>s</sub> for the adjacent coding sequence. We found K<sub>(150 bp upstream)</sub>/K<sub>s(N-terminus)</sub> ranged from 0.26 to 0.63, which is higher than for coding sequence but still considerably less than 1. In contrast, K<sub>(300−150 bp upstream)</sub>/K<sub>s(N-terminus)</sub> ranged from 0.82 to 1.45 (<a href="#pone-0000514-g005">Figure 5</a>), suggesting that the 150 bp directly upstream of coding sequence are under selective constraints while regions farther upstream are not. We also found K<sub>(3′ UTR)</sub>/K<sub>s(C-terminus)</sub> = 0.27 for <em>L. hesperus MaSp1</em> and <em>MaSp2</em>, consistent with strong purifying selection on the 3′ UTR.</p>
<div class="figure" id="pone-0000514-g005"><div class="img"><a name="pone-0000514-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>K/K<sub>s</sub> or K<sub>n</sub>/K<sub>s</sub> for flanking and terminal regions of <em>Latrodectus</em> major ampullate silk genes. K<sub>s(N-terminus)</sub> is the denominator for upstream ratios; K<sub>s(C-terminus)</sub> is the denominator for downstream ratios.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Actual K values shown above bars. Gene abbreviations are the same as for <a href="#pone-0000514-g004">Figure 4</a>.</p>
<span>doi:10.1371/journal.pone.0000514.g005</span></div>

<h4>Global comparisons of genomic clones</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We compared the entire clones containing <em>MaSp1</em> (34,046 bp) and <em>MaSp2</em> (37,092 bp) using MultiPipMaker and the global alignment program AVID <a href="#pone.0000514-Bray1">[60]</a>. We also compared the flanking sequences of the genes using BLASTN <a href="#pone.0000514-Altschul1">[61]</a> to search for repetitive elements in the <em>L. hesperus</em> genome. As expected, the N- and C-terminal coding regions are significantly conserved between the two genes (<a href="#pone-0000514-g006">Figure 6</a>). Within the genes themselves, there are also multiple regions of significant similarity at the DNA level. These regions correspond to the poly-A, GG, GGXG and GQ motifs found in both proteins. Additionally, there were numerous significant matches between regions of non-silk-protein-coding sequence. Each of these regions, when translated, was similar to transposable elements in the NCBI nr protein database (based on BLASTX <a href="#pone.0000514-Altschul1">[61]</a> scores: E&lt;e<sup>−10</sup>). Most notably, there is a significantly conserved region spanning ~700 bp that is found ~10,000 bp downstream of the <em>MaSp1</em> and <em>MaSp2</em> ORFs (<a href="#pone-0000514-g006">Figure 6</a>). The translated sequence of this region from the <em>MaSp2</em> clone significantly matched TCb1-transposase. The translated sequence from the <em>MaSp1</em> clone significantly matched gag-pol polyprotein, which contains a retrotransposon. Although both clones contain ORFs in this region, they do not encode full-length proteins. Thus, these genomic regions appear to be inactive transposable elements.</p>
<div class="figure" id="pone-0000514-g006"><div class="img"><a name="pone-0000514-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000514" data-uri="info:doi/10.1371/journal.pone.0000514.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000514.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000514.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000514.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000514.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Global AVID alignment of <em>L. hesperus</em> genomic clones containing <em>MaSp1</em> and <em>MaSp2</em> visualized with VISTA.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The <em>MaSp1</em>-containing clone was used as the reference sequence. Peak height corresponds to level of identity between the two clones. Regions exceeding 70% conservation over a window length of 100 bp are colored (blue for exons, red for non-coding sequence). The red peak corresponds to a putative transposable element found in both clones. Arrows mark open reading frames (ORFs) on the clone containing <em>MaSp1</em>.</p>
<span>doi:10.1371/journal.pone.0000514.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Gene structure</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Black widow dragline silk is an exceptionally tough biomaterial, even compared to the high-performance draglines spun by other spiders <a href="#pone.0000514-Lawrence1">[43]</a>, <a href="#pone.0000514-Swanson1">[45]</a>. Here we report the complete gene sequences for the MaSp1 and MaSp2 fibroins that form this silk. We found that both genes lack introns and thus <em>MaSp1</em> and <em>MaSp2</em> each possess only one enormous exon containing either 9,390 bp (<em>MaSp1</em>) or 11,340 bp (<em>MaSp2</em>) of coding sequence. No other full-length spider silk genes have been characterized, but the few known partial-length gene sequences fit into two categories of exon-intron structure. First, based on <em>L. geometricus MaSp1</em> and <em>Nephila MaSp2</em> fragments <a href="#pone.0000514-MotriukSmith1">[38]</a>, and the full-length genes described here, some silk genes are composed of single exons. Second, <em>Nephila Flag</em> and <em>Argiope MaSp2</em> have introns that are peculiar because successive introns <span class="underline">within</span> the same gene are nearly identical in sequence <a href="#pone.0000514-MotriukSmith1">[38]</a>, <a href="#pone.0000514-Hayashi4">[41]</a>. Thus, all known spider silk genes have unusual architectures.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In eukaryotes, proteins encoded by single exons are rare and strongly biased towards sizes much smaller (&lt;1,000 aa, <a href="#pone.0000514-Sakharkar1">[62]</a>–<a href="#pone.0000514-Sakharkar2">[63]</a>) than the spider silk proteins (&gt;3,000 aa). Intronless genes may reflect one type of gene duplication process that led to the diversification of the spider silk gene family; retroposition of mRNA transcripts (inherently intronless) into the genome can give rise to functional gene duplicates <a href="#pone.0000514-Zhang1">[64]</a>. Alternatively, intronless genes may be selectively favored. Intron length is negatively correlated with expression level <a href="#pone.0000514-CastilloDavis1">[65]</a>–<a href="#pone.0000514-Marais1">[67]</a> and major ampullate silk genes must be highly expressed throughout the lifetime of a spider. However, once an intron invades a silk gene, the intron can be rapidly propagated throughout the gene due to unequal crossing over, which appears to be common in silk genes (see <a href="#pone-0000514-g002">Figure 2</a>, <a href="#pone.0000514-Garb2">[40]</a>–<a href="#pone.0000514-Hayashi4">[41]</a>, <a href="#pone.0000514-Beckwitt1">[49]</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>MaSp1 and MaSp2 are almost entirely composed of a small suite of amino acid sequence motifs, such as GGX and poly-A, which are repeated many times throughout both fibroins (<a href="#pone-0000514-g001">Figures 1</a>, <a href="#pone-0000514-g002">2</a>). In each fibroin, the frequency and arrangement of these motifs occur in four distinct types of repeat units, termed ensemble repeats. Although the ensemble repeats of both MaSp1 and MaSp2 are similar in length (~30 aa) and composition (glycine-rich regions interspersed with alanine-rich regions), no repeat units from one protein are found in the other (<a href="#pone-0000514-g001">Figures 1</a>, <a href="#pone-0000514-g002">2</a>). These results confirm that distinct genes encode each silk protein <a href="#pone.0000514-Guerette1">[e.g. 9]</a>, <a href="#pone.0000514-Xu1">[17]</a>–<a href="#pone.0000514-Hinman2">[18]</a>, rather than posttranscriptional processing of a single gene leading to silk protein diversity as previously suggested by Craig et al. <a href="#pone.0000514-Craig1">[68]</a>.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Both <em>L. hesperus</em> MaSp1 and MaSp2 have glycine and alanine-rich motifs that occur in ensemble repeats, but the fibroins differ in their higher-level repeat organization (repetitiveness) and similarity of repeat copies (homogenization). In MaSp1, the four types of ensemble repeats are strung together to form an ~120 amino acid long, higher-level repeat unit. This large aggregate is tandemly arrayed twenty times and the iterations share high identity at both the amino acid and nucleotide level (98.1% and 97.5% mean pairwise identity, respectively). In contrast, MaSp2 does not have clearly discernible higher-level repeats and has more sequence and length variation among its ensemble repeats than in MaSp1 (<a href="#pone-0000514-g002">Figure 2</a>, <a href="#pone-0000514-t002">Table 2</a>). MaSp2, however, has a tandem repetition of 778 aa that is &gt;99.7% identical over the 2,334 encoding nucleotides (<a href="#pone-0000514-g002">Figure 2</a>). The modular architectures of MaSp1 and MaSp2 likely reflect concerted evolution within a single gene, as has been implicated in maintaining similarity among Flag (~440 aa) ensemble repeats and the long repeats of TuSp1 (~200 aa) and AcSp1 (aciniform silk; 200 aa <a href="#pone.0000514-Hayashi2">[14]</a>).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Modular architecture is also hypothesized to facilitate replication slippage in silk genes that have tandem arrays of codons for simple amino acid sequence motifs (e.g., poly-A, GGX, GA). Replication slippage would result in length variation among the ensemble repeats within a gene, as has been observed in <em>MaSp1, MaSp2</em>, and <em>Flag</em> <a href="#pone.0000514-Hayashi3">[26]</a>, <a href="#pone.0000514-Hayashi4">[41]</a>, <a href="#pone.0000514-Beckwitt1">[49]</a>. Because previously described <em>MaSp1</em> and <em>MaSp2</em> gene or cDNA fragments are substantially incomplete (typically&lt;2000 bp) and represent the least homogenized parts of the genes (5′ or 3′ ends), it is unknown if these genes are composed of higher level aggregates of ensemble repeats. Thus, it remains to be seen whether the extreme repetitiveness and homogenization of <em>L. hesperus MaSp1</em> compared to <em>L. hesperus MaSp2</em> is a general feature of spider dragline fibroins, or whether this pattern is peculiar to black widows.</p>


<h4>Relationship to other silk proteins</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Attempts to reconstruct evolutionary relationships among members of the spider silk gene family have relied exclusively on the non-repetitive C-terminus <a href="#pone.0000514-Tian1">[11]</a>–<a href="#pone.0000514-Garb1">[12]</a>, <a href="#pone.0000514-Hayashi2">[14]</a>, <a href="#pone.0000514-Hayashi3">[26]</a>, <a href="#pone.0000514-Garb2">[40]</a>, <a href="#pone.0000514-Beckwitt2">[69]</a>, but the N-terminus has great promise for phylogenetic reconstruction <a href="#pone.0000514-MotriukSmith1">[38]</a>–<a href="#pone.0000514-Rising1">[39]</a>. In our analyses, there was extensive congruence between trees based on N- and C-termini of silk gene family members (<a href="#pone-0000514-g004">Figure 4C</a>). A curious relationship found in both the N- and C-terminal phylogenetic trees is the grouping of <em>Latrodectus</em> major ampullate silk genes rather than a clade of <em>MaSp1</em> from all species separate from a <em>MaSp2</em> clade (<a href="#pone-0000514-g004">Figure 4C</a>). A similar sister relationship between <em>MaSp1</em> and <em>MaSp2</em> C-termini has been found for other species <a href="#pone.0000514-Gatesy1">[10]</a>, <a href="#pone.0000514-Garb1">[12]</a>, <a href="#pone.0000514-Garb2">[40]</a>. Given the striking conservation of repetitive amino acid motifs for each fibroin across divergent species, it seems unlikely that this pattern could result from independent duplication and convergence events. To explain the similarities in the repetitive regions by these means requires the convergence of thousands of nucleotides within a silk gene to encode either entirely MaSp1 or MaSp2 motifs, and for such convergences to have occurred multiple times in different spider lineages. Instead, recombination, selection, or the interaction of these two processes more likely explains the similarity of MaSp1 and MaSp2 N- and C-termini within species. Intergenic pairing during meiosis could be facilitated by the stretches of DNA coding for similar amino acid motifs, such as poly-A and GGX, in both <em>MaSp1</em> and <em>MaSp2</em>. For example, pairwise comparisons of the <em>L. hesperus MaSp1</em> and <em>MaSp2</em> genes show that they contain multiple regions of significant similarity spanning at least 100 bp (<a href="#pone-0000514-g006">Figure 6</a>). If recombination occurs between these two genes, it is less frequent than speciation events; <em>MaSp1</em> of <em>L. geometricus</em> and <em>L. hesperus</em> were clustered in the N-terminal trees and the C-terminal ML tree. Thus far, no single gene has been described that contains repeat units typical of both genes, which would provide the most convincing evidence for intergenic recombination. We did not find any clones in the <em>L. hesperus</em> genomic library that were positive for both <em>MaSp1</em> and <em>MaSp2,</em> nor did Sponner et al. <a href="#pone.0000514-Sponner1">[15]</a> find double positive clones in a <em>Nephila clavipes</em> genomic library. However, there could be strong selection against proteins with a mixture of repeat units, while terminal recombinants may be tolerated. Convergent evolution could alternatively explain the grouping of <em>MaSp1</em> and <em>MaSp2</em> paralogs by their N- and C-termini. Selection could drive convergence of terminal amino acid sequences within species if similarity in these regions is necessary for accurate assembly of the two fibroins into a single fiber. Both proteins are exposed to the same environments, such as salt and pH gradients in the silk gland and duct <a href="#pone.0000514-Dicko1">[70]</a>, which could also favor evolutionary convergence of terminal domains.</p>


<h4>Non-coding sequence</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Non-coding sequences upstream of major ampullate silk genes from different genera were too divergent to reliably align or identify regulatory elements other than the conserved motif CACG and the TATA-box identified by Motriuk-Smith et al. <a href="#pone.0000514-MotriukSmith1">[38]</a>. Although phylogenetic footprinting is a powerful tool for identifying novel regulatory elements, the appropriate level of divergence among species is critical <a href="#pone.0000514-Hardison1">[51]</a>. The genera examined here, <em>Latrodectus, Nephila</em>, and <em>Argiope</em>, belong to three different families that shared a common ancestor ~135–160 million years ago (MYA) <a href="#pone.0000514-Selden1">[71]</a>–<a href="#pone.0000514-Ayoub1">[72]</a>. In contrast, some of the most successful examples of phylogenetic footprinting involve more recent divergences (e.g. <em>Drosophila</em> spp. <a href="#pone.0000514-Sinha1">[73]</a>, <em>Saccharomyces</em> spp. <a href="#pone.0000514-Cliften1">[74]</a>–<a href="#pone.0000514-Kellis1">[75]</a>, grasses <a href="#pone.0000514-Inada1">[76]</a>, and primates <a href="#pone.0000514-Boffelli1">[77]</a>). Comparisons of human and rodent genomes, thought to have split ~100 MYA <a href="#pone.0000514-Arnason1">[78]</a>, yield many novel regulatory elements, while extending divergence to mammals and birds (~310 MYA <a href="#pone.0000514-Benton1">[79]</a>–<a href="#pone.0000514-Lee1">[80]</a>) causes a precipitous drop off in the ability to detect motifs <a href="#pone.0000514-Thomas1">[81]</a>–<a href="#pone.0000514-Prakash1">[82]</a>. In plants, the limit of motif detection is gene specific but appears to be reached when comparing poplars and <em>Arabidopsis</em> <a href="#pone.0000514-DeBodt1">[83]</a>, which diverged ~110 MYA <a href="#pone.0000514-Chaw1">[84]</a>–<a href="#pone.0000514-Wikstrm1">[85]</a>. Thus, given the divergence times of the spider taxa examined here, the fact that the promoter regions of their major ampullate silk genes retain any significant sequence similarity is notable.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In <em>Latrodectus</em>, ~300 bp of upstream sequence could be reliably aligned. However, the ~150 bp directly upstream of the start codon are more conserved than the adjacent, upstream non-coding sequence or synonymous sites in coding regions of the genes (<a href="#pone-0000514-g005">Figure 5</a>). We found a conserved motif in this region that matches the binding site for the Achaete-Scute family of transcription factors, which regulate neurogenesis and sensory mother cell development in <em>Drosophila</em> <a href="#pone.0000514-Cabrera1">[86]</a>–<a href="#pone.0000514-RuizGmez1">[87]</a>. A homolog of this transcription factor family, called SGSF, shows a silk gland-restricted pattern of expression in <em>L. hesperus</em>, specifically to the tubuliform and major ampullate silk glands <a href="#pone.0000514-Kohler1">[88]</a>. These are the only glands that appear to express <em>MaSp1</em> and <em>MaSp2</em> in appreciable quantities <a href="#pone.0000514-Garb2">[40]</a>. Experimental manipulation is needed to elucidate if SGSF or a related protein is, in fact, involved with regulating major ampullate silk gene expression in black widows and other spider species.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>The conserved, upstream non-coding regions and the 3′ UTRs of <em>L. hesperus MaSp1</em> and <em>L. hesperus MaSp2</em> show evidence for stronger selective constraints than do <em>L. hesperus MaSp1</em> and <em>L. geometricus MaSp1</em> (lower K/K<sub>s</sub>, <a href="#pone-0000514-g005">Figure 5</a>). Although regulatory element evolution in the 3′ UTR has received less attention than in promoter regions, many genes display significantly conserved sequence motifs in the 3′ UTR <a href="#pone.0000514-Leviten1">[e.g. 89]</a>–<a href="#pone.0000514-Xie1">[91]</a>. Additionally, experimental evidence has shown that elements in the 3′ UTR bind factors involved in posttranscriptional regulation <a href="#pone.0000514-Goodwin1">[e.g. 92]</a>–<a href="#pone.0000514-Wickens1">[93]</a>. A striking example of 3′ regulation is in <em>Drosophila's Enhancer of split</em> Complex, which belongs to the same class of genes (beta helix-loop-helix) as <em>achaete</em> and <em>scute</em> <a href="#pone.0000514-Leviten1">[89]</a>, <a href="#pone.0000514-Delidakis1">[94]</a>–<a href="#pone.0000514-Lai2">[96]</a>. Taken together, our findings suggest selection on non-coding sequences for co-regulation of the paralogous dragline silk genes, <em>MaSp1</em> and <em>MaSp2</em>.</p>


<h4>Recipe for a high-performance biomaterial</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The production of synthetic spider dragline silk is a major goal of biomimetics research <a href="#pone.0000514-Vollrath1">[1]</a>, <a href="#pone.0000514-Hinman3">[29]</a>. Though promising advances have been made with a variety of transgene constructs and host organisms, an exact mimic of a native dragline silk fiber has yet to be produced <a href="#pone.0000514-Lazaris1">[e.g. 30]</a>, <a href="#pone.0000514-Scheller2">[33]</a>–<a href="#pone.0000514-Ittah1">[35]</a>. While artificial spinning is certainly an important consideration, a significant challenge to the efforts to create synthetic silk proteins has been the incomplete knowledge of spider silk genes. Thus far, all transgene constructs for recombinant silk proteins have relied on partial cDNA sequences from two orb-weaving species, <em>Nephila clavipes</em> and <em>Araneus diadematus</em> <a href="#pone.0000514-Hinman3">[e.g. 29]</a>–<a href="#pone.0000514-Lazaris1">[30]</a>, <a href="#pone.0000514-Bini1">[32]</a>–<a href="#pone.0000514-Ittah1">[35]</a>, <a href="#pone.0000514-Sponner3">[37]</a>, <a href="#pone.0000514-Arcidiacono1">[97]</a>. These truncated cDNAs encode only a fraction (typically 20% or less) of the repetitive region and the C-terminal domain. Experiments on recombinant silks made with and without the C-terminal region showed that the C-terminus was required for fibroins to form aggregates. Protein aggregation is an essential step in the precipitation of liquid spinning dope into a solid silk fiber <a href="#pone.0000514-Sponner3">[37]</a>, <a href="#pone.0000514-Huemmerich1">[98]</a>. The C-terminus is not only necessary for aggregation of recombinant fibroins, but also for the formation of the characteristic crystalline structures that impart strength to dragline silk fibers <a href="#pone.0000514-Ittah1">[35]</a>. As has been proposed for the C-terminus <a href="#pone.0000514-Sponner3">[37]</a>, the evolutionary conservation of the N-terminus suggests that this region is also functionally significant. For example, N-termini may play a central role in the proper transport of fibroins from secretory cells to silk gland lumen, aid in fiber formation, and contribute to the structural properties of silk fibers. In both <em>L. hesperus</em> MaSp1 and MaSp2, the N-terminal domain contains the most hydrophobic region of the entire fibroin (<a href="#pone-0000514-g003">Figure 3</a>). The next most hydrophobic region is the C-terminus. Sponner et al. <a href="#pone.0000514-Sponner3">[37]</a> hypothesized that the hydrophobicity of the C-terminus was a key characteristic for its role in fibroin aggregation. The hydrophobic N-terminal region could thus similarly enhance silk fiber formation and mechanical properties. Another evolutionarily conserved aspect of spider fibroins is their extremely large size, which is also a feature of independently evolved insect fibroins. Thus, large size has been repeatedly selected for in the evolution of fibroin genes. Therefore, a complete silk gene, with full representation of the N- and C-terminal regions, the intervening repetitive sequence, and the transitions among these domains, should dramatically improve recombinant silk performance.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The complete gene sequences described here highlight the extraordinary molecular characteristics of spider silks. Black widow major ampullate silk genes are highly modular, exhibiting a hierarchical organization of iterated short motifs and ensemble repeats (groups of motifs). By characterizing full-length <em>MaSp1</em> and <em>MaSp2</em> genes, we were able to detect even higher-level repeats (aggregates of ensemble repeats) and uncover a striking difference in the degree of repeat homogenization between <em>MaSp1</em> and <em>MaSp2</em>. The extreme modularity of <em>MaSp1</em> (<a href="#pone-0000514-g001">Figure 1</a>) may reflect selection on the MaSp1 fibroin for perfect repeats, perhaps important for rapid and consistent spinning of high quality silk fibers. Sequence homogenization, however, is also due to molecular mechanisms, such as unequal crossing over (e.g., two large tandem repeats in <a href="#pone-0000514-g002">Figure 2</a>), and the interaction between selection and concerted evolution is a subject for further investigation. We have additionally identified putative regulatory elements that may enhance expression of transgenic silks. Thus, the clones sequenced here provide the complete genetic blueprints for the primary protein components of the major ampullate silk fiber. These designs hold critical information for the mass production of artificial fibers that accurately mimic the spectacular high-performance properties of native spider silk.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Genomic Library Construction and Screening</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We targeted black widow silk genes because in addition to the exemplary properties of their silk, <em>Latrodectus hesperus</em> has one of the smallest known genome sizes for a spider (C-value of 1.29 picograms <a href="#pone.0000514-Gregory1">[99]</a>), meaning that fewer genomic clones must be screened to find a gene of interest. Individuals were collected from a single locality in Riverside, California (USA), live frozen in liquid nitrogen, and stored at −80°C. High-molecular-weight DNA was isolated from the cephalothoraxes of eight individuals using a modified method of Sambrook and Russell <a href="#pone.0000514-Sambrook1">[100]</a>. Following isolation, DNA was mechanically sheared through a pipette tip and subsequently treated with End-Repair Enzyme Mix (Epicentre) to produce blunt 5′ phosphorylated ends. Fragments ranging from 38–50 kilobases were gel excised, purified, and ligated into pCC1FOS™ vector (Epicentre). Resulting fosmids were packaged using MaxPlax™ Lambda Packaging Extracts and transfected into Epi300-T1R <em>E. coli</em> cells following protocols for the CopyControl™ Fosmid Library Production kit (Epicentre). Approximately 100,000 recombinant <em>E. coli</em> colonies were picked and arrayed into 276 culture plates each containing 384 wells using a QPIX robotic picker (Genetix). Each culture plate was replicated and original stock plates containing 7.5% glycerol were stored at −80°C.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>To efficiently screen the genomic library, fosmid DNA was extracted from cell cultures combined from a single 384-well plate, and such extractions were done for every plate in the library. Polymerase chain reaction (PCR) experiments targeting genes of interest were used to identify which plate contained one or more positive clones. Once the plate was identified, that plate was replicated twice, and cell cultures from the rows were combined to form 16 templates, while cell cultures from the columns were combined to form 24 templates. Templates were then PCR screened to identify individual clones containing the gene of interest. Primers targeting <em>MaSp1</em> and <em>MaSp2</em> were designed from <em>L. hesperus</em> cDNA clones <a href="#pone.0000514-Garb2">[40]</a> (<em>MaSp1</em>–N-terminal clone, EF595247; <em>MaSp2</em>–C-terminal clone, AY953075). The primers, LhMaSp1NF254, 5′-TGGCTTTCGCATCATCTGTAGC-3′ and LhMaSp1NR607, 5′-CTCCTTGACCATAACTAACTGGCTG-3′ amplified a 350 bp portion of the <em>MaSp1</em> 5′ region. Primers LhMaSp2_1086F, 5′-CATCAGCAGCAGGACCAAGTG-3′, and LhMaSp2_1337R, 5′-GCGTTGTCGGTGAAGATAAAGC-3′, amplified a 250 bp portion of the <em>MaSp2</em> 3′ region.</p>


<h4>Sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Seven <em>MaSp1</em>-positive clones and three <em>MaSp2</em>-positive clones were found after screening half of the library. One positive clone for each gene was shotgun sequenced and assembled by Qiagen (Hilden, Germany) to 6× coverage for the <em>MaSp2</em>-positive clone and 8× coverage for the <em>MaSp1</em>-positive clone. This resulted in three contiguous sequences (contigs) for the <em>MaSp2</em>-positive clone with two gaps within the coding sequence and one directly after the stop codon. The 707 bp gap between the stop codon and the downstream contig was closed by sequencing directly off the fosmid clone using primers designed from the C-terminal coding region of <em>MaSp2</em> and for the beginning of the downstream contig (all primer sequences used in this study are available upon request). Primer walking to close the two gaps within the <em>MaSp2</em> coding sequence was not possible due to its repetitive nature. Instead the clone was digested with NotI and BamHI (New England Biolabs) and a 9 kb restriction fragment containing almost the entire repetitive portion of <em>MaSp2</em> was subcloned into pZErO™-2 plasmids (Invitrogen) and electroporated into Epi-400 <em>E. coli</em> (Epicentre). The subclone was partially digested with PstI (New England Biolabs) and 2000-3000 bp fragments were gel excised and ligated into PstI digested and dephosphorylated pZErO™-2. Ligation products were electroporated into TOP10 <em>E. coli</em> (Invitrogen). A library of 96 PstI partial-digest clones were arrayed and sequenced in one direction. Sequences were assembled independently and using the fosmid contigs as a backbone in SEQUENCHER v4.5 (Gene Codes Corp.), requiring 100% identity for high-quality bases. Ten clones spanned the first gap (111 bp) and 18 clones spanned the second gap (632 bp) with no less than 5× sequence coverage of any base along the length of the NotI-BamHI subclone. No disagreement between the sequences of the subclone and the fosmid contigs was found.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Shotgun sequencing of a <em>MaSp1</em>-postive clone resulted in a single contig containing the entire coding sequence of <em>MaSp1</em> and the vector. However, this contig was ~7000 bp smaller than expected based on restriction digests. This missing sequence was determined by PCR amplifying with AccuPrime™ Taq DNA Polymerase High Fidelity (Invitrogen) and primers designed from both ends of the contig. The 7890 bp PCR product was sequenced with at least 2× coverage by primer walking. Additionally, the fosmid was directly sequenced at intervals along the gap to ensure that no mutations had been introduced by the PCR amplifcation. Experimental restriction digests of the <em>MaSp1</em>-positive and <em>MaSp2</em>-positive clones matched predicted restriction sites in the final sequences, verifying that assembly had not erroneously excluded repetitive sequence.</p>


<h4>Sequence analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Nucleotide sequences were conceptually translated using the standard genetic code. Base composition, amino acid content, codon usage, and Kyte and Doolittle <a href="#pone.0000514-Kyte1">[50]</a> hydrophilicity predictions were calculated in MacVector™ (Oxford Molecular Group). Amino acid sequences were considered to start at the first methionine in frame. The first M on the MaSp1 sequence corresponded to the conserved start position identified by Rising et al. <a href="#pone.0000514-Rising1">[39]</a> (see also <a href="#pone-0000514-g004">Figure 4A</a>). The MaSp2 sequence also displayed an M at this position, but the first in frame M codon was 9 bp upstream (<a href="#pone-0000514-g004">Figure 4A</a>). Pairwise K, K<sub>s</sub>, and K<sub>n</sub> were calculated using DnaSp v4.0 <a href="#pone.0000514-Rozas1">[101]</a> excluding gaps and missing data.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Predicted amino acid sequences of all currently published N-termini were aligned (<a href="#pone-0000514-g004">Figure 4A</a>), making corrections to the nucleotide sequences of <em>L. geometricus MaSp1, A. bruennichi CySp2</em>, and <em>N. clavipes Flag</em> according to the modifications described in Rising et al. <a href="#pone.0000514-Rising1">[39]</a>. Alignments of N- and C-terminal amino acid sequences were made separately using default parameters in ClustalW (MacVector™). The C-terminal alignment was modified slightly such that the first position of the C-terminal Flag sequences aligned with the first position of the other sequences (<a href="#pone-0000514-g004">Figure 4B</a>). Amino acid alignments were used to guide nucleotide alignments, which formed the basis for phylogenetic analyses. Heuristic ML and MP searches were performed in PAUP* <a href="#pone.0000514-Swofford1">[102]</a> using TBR (tree bisection reconnection) branch swapping and 10,000 (MP) or 100 (ML) random stepwise addition replicates. Support for clades was evaluated with 1000 (MP) or 100 (ML) bootstrap pseudoreplicates (of all characters), and 100 (MP) or 1 (ML) random stepwise addition replicates per pseudoreplicate. ML analyses treated gaps as missing data. MP analyses were performed treating gaps as missing data and as a 5<sup>th</sup> state. Optimal model parameters for ML analyses were calculated with MODELTEST <a href="#pone.0000514-Posada1">[103]</a>. The N-termini fit the HKY+G <a href="#pone.0000514-Hasegawa1">[104]</a> model of evolution (transitions/transversions = 1.24; gamma = 0.9058). The C-termini fit the TrN+G <a href="#pone.0000514-Tamura1">[105]</a> model of evolution (A&lt;&gt;G = 2.34; C&lt;&gt;T = 1.27; transversions = 1; gamma = 1.34). To further evaluate tree structure and clade support in a model-based framework, Bayesian analyses were carried out using MRBAYES v.3.1.2 <a href="#pone.0000514-Ronquist1">[106]</a>. The same model of evolution determined by MODELTEST was used but parameter values were evaluated during the Bayesian analysis. Default priors and Metropolis-coupled, Markov-chain, Monte Carlo (MCMC) sampling procedures were executed for two independent runs, sampled every 100<sup>th</sup> generation, carried out simultaneously. Convergence was assessed every 1000<sup>th</sup> generation and the posterior distribution was considered adequately sampled when the standard deviation of split frequencies of these two runs dropped below 0.01 (&lt;1 million generations). A second analysis was run for 10 million generations (sampling every 500) to ensure that a longer sampling time did not change the results. For each run, the first 50% of sampled trees were discarded as burnin prior to calculating the majority rule consensus tree.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Comparisons of genes with MultiPipMaker were done using the “high sensitivity low time limit” option. Each major ampullate silk gene with upstream sequence was sequentially input as the reference to obtain maximal pairwise alignments. AVID alignments were made using default parameters and viewed on the VISTA browser &lt;<a href="http://www-gsd.lbl.gov/vista/">www-gsd.lbl.gov/vista/</a>&gt; <a href="#pone.0000514-Mayor1">[107]</a>–<a href="#pone.0000514-Frazer1">[108]</a>. Global alignments of conserved non-coding sequence identified by MultiPipMaker were made using default parameters in ClustalW and modified manually. Approximately 300 bp of upstream sequence were scanned against insect transcription factor binding sites in the TRANSFAC 6.0 database using the program PATCH™ v1.0 <a href="#pone.0000514-Matys1">[57]</a> with a minimum match of 6 and a maximum mismatch of 2.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>Open reading frames on the black widow genomic clones were identified using the ORFFinder program on the NCBI website &lt;<a href="http://www.ncbi.nlm.nih.gov/gorf/gorf.html">http://www.ncbi.nlm.nih.gov/gorf/gorf.ht​ml</a>&gt;, with a minimum cutoff of 300 nucleotides.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Richard Baker, Clay Clark, Dayan Colon-Sanchez, Teresa DiMauro, Tim Kingan, and Lobna Shahatto for help constructing, arraying, and screening the genomic library. We also thank Richard Baker, Laura Baldo, and John Gatesy for comments on this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RT NA JG MC CH. Performed the experiments: RT NA JG MC CH. Analyzed the data: NA CH. Contributed reagents/materials/analysis tools: CH. Wrote the paper: NA JG CH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000514-Vollrath1" id="pone.0000514-Vollrath1"></a>Vollrath F, Knight DP (2001) Liquid crystalline spinning of spider silk. Nature  410: 541–548.  DOI: 10.1038/35069000.  <ul class="find" data-citedArticleID="957640" data-doi="10.1038/35069000"><li><a href="http://dx.doi.org/10.1038/35069000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Liquid+crystalline+spinning+of+spider+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Liquid+crystalline+spinning+of+spider+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000514-Gosline1" id="pone.0000514-Gosline1"></a>Gosline J, Lillie M, Carrington E, Guerrette P, Ortlepp C, et al.  (2002) Elastic proteins: biological roles and mechanical properties. Phil Trans R Soc Lond B Biol Sci  357: 121–132.  DOI: 10.1098/rstb.2001.1022.  <ul class="find" data-citedArticleID="957508" data-doi="10.1098/rstb.2001.1022"><li><a href="http://dx.doi.org/10.1098/rstb.2001.1022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elastic+proteins%3A+biological+roles+and+mechanical+properties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elastic+proteins%3A+biological+roles+and+mechanical+properties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000514-Foo1" id="pone.0000514-Foo1"></a>Foo CWP, Kaplan DL (2002) Genetic engineering of fibrous proteins: spider dragline silk and collagen. Adv Drug Deliv Rev  54: 1131–1143.  DOI: 10.1016/S0169-409X(02)00061-3.  <ul class="find" data-citedArticleID="957494" data-doi="10.1016/s0169-409x(02)00061-3"><li><a href="http://dx.doi.org/10.1016/s0169-409x(02)00061-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+engineering+of+fibrous+proteins%3A+spider+dragline+silk+and+collagen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+engineering+of+fibrous+proteins%3A+spider+dragline+silk+and+collagen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000514-Gosline2" id="pone.0000514-Gosline2"></a>Gosline JM, DeMont ME, Denny MW (1986) The structure and properties of spider silk. Endeavour  10: 37–43.  DOI: 10.1016/0160-9327(86)90049-9.  <ul class="find" data-citedArticleID="957510" data-doi="10.1016/0160-9327(86)90049-9"><li><a href="http://dx.doi.org/10.1016/0160-9327(86)90049-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+and+properties+of+spider+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+and+properties+of+spider+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000514-Hinman1" id="pone.0000514-Hinman1"></a>Hinman M, Dong Z, Xu M, Lewis RV (1992)  Spider silk: a mystery starting to unravel. In: Case ST, editor. Biopolymers. Berlin: Springer. pp. 227–254.  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0000514-Gosline3" id="pone.0000514-Gosline3"></a>Gosline JM, Guerette PA, Ortlepp CS, Savage KN (1999) The mechanical design of spider silks: from fibroin sequence to mechanical function. J Exp Biol  202: 3295–3303.  <ul class="find" data-citedArticleID="957512"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+mechanical+design+of+spider+silks%3A+from+fibroin+sequence+to+mechanical+function.&amp;auth=&amp;atitle=The+mechanical+design+of+spider+silks%3A+from+fibroin+sequence+to+mechanical+function." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mechanical+design+of+spider+silks%3A+from+fibroin+sequence+to+mechanical+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mechanical+design+of+spider+silks%3A+from+fibroin+sequence+to+mechanical+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000514-Foelix1" id="pone.0000514-Foelix1"></a>Foelix R (1996) Biology of Spiders. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0000514-Blackledge1" id="pone.0000514-Blackledge1"></a>Blackledge TA, Hayashi CY (2006) Silken toolkits: biomechanics of silk fibers spun by the orb web spider <em>Argiope argentata</em> (Fabricius 1775). J Exp Biol  209: 2452–2461.  DOI: 10.1242/jeb.02275.  <ul class="find" data-citedArticleID="957460" data-doi="10.1242/jeb.02275"><li><a href="http://dx.doi.org/10.1242/jeb.02275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silken+toolkits%3A+biomechanics+of+silk+fibers+spun+by+the+orb+web+spider+Argiope+argentata+%28Fabricius+1775%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silken+toolkits%3A+biomechanics+of+silk+fibers+spun+by+the+orb+web+spider+Argiope+argentata+%28Fabricius+1775%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000514-Guerette1" id="pone.0000514-Guerette1"></a>Guerette PA, Ginzinger DG, Weber BHF, Gosline JM (1996) Silk properties determined by gland-specific expression of a spider fibroin gene family. Science  272: 112–115.  DOI: 10.1126/science.272.5258.112.  <ul class="find" data-citedArticleID="957520" data-doi="10.1126/science.272.5258.112"><li><a href="http://dx.doi.org/10.1126/science.272.5258.112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silk+properties+determined+by+gland-specific+expression+of+a+spider+fibroin+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silk+properties+determined+by+gland-specific+expression+of+a+spider+fibroin+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000514-Gatesy1" id="pone.0000514-Gatesy1"></a>Gatesy J, Hayashi C, Motriuk D, Woods J, Lewis R (2001) Extreme diversity, conservation, and convergence of spider silk fibroin sequences. Science  291: 2603–2605.  DOI: 10.1126/science.1057561.  <ul class="find" data-citedArticleID="957504" data-doi="10.1126/science.1057561"><li><a href="http://dx.doi.org/10.1126/science.1057561" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extreme+diversity%2C+conservation%2C+and+convergence+of+spider+silk+fibroin+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extreme+diversity%2C+conservation%2C+and+convergence+of+spider+silk+fibroin+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000514-Tian1" id="pone.0000514-Tian1"></a>Tian M, Lewis RV (2005) Molecular characterization and evolutionary study of spider tubuliform (eggcase) silk protein. Biochemistry  44: 8006–8012.  DOI: 10.1021/bi050366u.  <ul class="find" data-citedArticleID="957636" data-doi="10.1021/bi050366u"><li><a href="http://dx.doi.org/10.1021/bi050366u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+and+evolutionary+study+of+spider+tubuliform+%28eggcase%29+silk+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+and+evolutionary+study+of+spider+tubuliform+%28eggcase%29+silk+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000514-Garb1" id="pone.0000514-Garb1"></a>Garb JE, DiMauro T, Vo V, Hayashi CY (2006) Silk genes support the single origin of orb webs. Science  312: 1762.  DOI: 10.1126/science.1127946.  <ul class="find" data-citedArticleID="957500" data-doi="10.1126/science.1127946"><li><a href="http://dx.doi.org/10.1126/science.1127946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silk+genes+support+the+single+origin+of+orb+webs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silk+genes+support+the+single+origin+of+orb+webs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000514-Hayashi1" id="pone.0000514-Hayashi1"></a>Hayashi CY, Shipley NH, Lewis RV (1999) Hypotheses that correlate the sequence, structure, and mechanical properties of spider silk proteins. Int J Biol Macromol  24: 271–275.  DOI: 10.1016/S0141-8130(98)00089-0.  <ul class="find" data-citedArticleID="957526" data-doi="10.1016/s0141-8130(98)00089-0"><li><a href="http://dx.doi.org/10.1016/s0141-8130(98)00089-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hypotheses+that+correlate+the+sequence%2C+structure%2C+and+mechanical+properties+of+spider+silk+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hypotheses+that+correlate+the+sequence%2C+structure%2C+and+mechanical+properties+of+spider+silk+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000514-Hayashi2" id="pone.0000514-Hayashi2"></a>Hayashi CY, Blackledge TA, Lewis RV (2004) Molecular and mechanical characterization of aciniform silk: uniformity of iterated sequence modules in a novel member of the spider silk fibroin gene family. Mol Biol Evol  21: 1950–1959.  DOI: 10.1093/molbev/msh204.  <ul class="find" data-citedArticleID="957528" data-doi="10.1093/molbev/msh204"><li><a href="http://dx.doi.org/10.1093/molbev/msh204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+mechanical+characterization+of+aciniform+silk%3A+uniformity+of+iterated+sequence+modules+in+a+novel+member+of+the+spider+silk+fibroin+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+mechanical+characterization+of+aciniform+silk%3A+uniformity+of+iterated+sequence+modules+in+a+novel+member+of+the+spider+silk+fibroin+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000514-Sponner1" id="pone.0000514-Sponner1"></a>Sponner A, Schlott B, Vollrath F, Unger E, Grosse F, et al.  (2005) Characterization of the protein components of <em>Nephila clavipes</em> dragline silk. Biochemistry  44: 4727–4736.  DOI: 10.1021/bi047671k.  <ul class="find" data-citedArticleID="957620" data-doi="10.1021/bi047671k"><li><a href="http://dx.doi.org/10.1021/bi047671k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+protein+components+of+Nephila+clavipes+dragline+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+protein+components+of+Nephila+clavipes+dragline+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000514-Zhao1" id="pone.0000514-Zhao1"></a>Zhao A, Zhao T, Nakagaki K, Zhang Y-S, SiMa Y, et al.  (2006) Novel molecular and mechanical properties of egg case silk from wasp spider, <em>Argiope bruennichi</em>. Biochemistry-US  45: 3348–3356.  DOI: 10.1021/bi052414g.  <ul class="find" data-citedArticleID="957654" data-doi="10.1021/bi052414g"><li><a href="http://dx.doi.org/10.1021/bi052414g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+molecular+and+mechanical+properties+of+egg+case+silk+from+wasp+spider%2C+Argiope+bruennichi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+molecular+and+mechanical+properties+of+egg+case+silk+from+wasp+spider%2C+Argiope+bruennichi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000514-Xu1" id="pone.0000514-Xu1"></a>Xu M, Lewis RV (1990) Structure of a protein superfiber: spider dragline silk. Proc Natl Acad Sci USA  87: 7120–7124.  DOI: 10.1073/pnas.87.18.7120.  <ul class="find" data-citedArticleID="957650" data-doi="10.1073/pnas.87.18.7120"><li><a href="http://dx.doi.org/10.1073/pnas.87.18.7120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+a+protein+superfiber%3A+spider+dragline+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+a+protein+superfiber%3A+spider+dragline+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000514-Hinman2" id="pone.0000514-Hinman2"></a>Hinman MB, Lewis RV (1992) Isolation of a clone encoding a second dragline silk fibroin. <em>Nephila clavipes</em> dragline silk is a two-protein fiber. J Biol Chem  267: 19320–19324.  <ul class="find" data-citedArticleID="957536"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+of+a+clone+encoding+a+second+dragline+silk+fibroin.+Nephila+clavipes+dragline+silk+is+a+two-protein+fiber.&amp;auth=&amp;atitle=Isolation+of+a+clone+encoding+a+second+dragline+silk+fibroin.+Nephila+clavipes+dragline+silk+is+a+two-protein+fiber." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+a+clone+encoding+a+second+dragline+silk+fibroin.+Nephila+clavipes+dragline+silk+is+a+two-protein+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+a+clone+encoding+a+second+dragline+silk+fibroin.+Nephila+clavipes+dragline+silk+is+a+two-protein+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000514-Sponner2" id="pone.0000514-Sponner2"></a>Sponner A, Unger E, Grosse F, Weisshart K (2005) Differential polymerization of the two main protein components of dragline silk during fibre spinning. Nat Mater  4: 772–775.  DOI: 10.1038/nmat1493.  <ul class="find" data-citedArticleID="957622" data-doi="10.1038/nmat1493"><li><a href="http://dx.doi.org/10.1038/nmat1493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+polymerization+of+the+two+main+protein+components+of+dragline+silk+during+fibre+spinning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+polymerization+of+the+two+main+protein+components+of+dragline+silk+during+fibre+spinning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000514-Simmons1" id="pone.0000514-Simmons1"></a>Simmons A, Ray E, Jelinski LW (1994) Solid-state <sup>13</sup>C NMR of <em>Nephila clavipes</em> dragline silk establishes structure and identity of crystalline regions. Macromolecules  27: 5235–5237.  DOI: 10.1021/ma00096a060.  <ul class="find" data-citedArticleID="957614" data-doi="10.1021/ma00096a060"><li><a href="http://dx.doi.org/10.1021/ma00096a060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solid-state+13C+NMR+of+Nephila+clavipes+dragline+silk+establishes+structure+and+identity+of+crystalline+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solid-state+13C+NMR+of+Nephila+clavipes+dragline+silk+establishes+structure+and+identity+of+crystalline+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000514-Kmmerlen1" id="pone.0000514-Kmmerlen1"></a>Kümmerlen J, van Beek JD, Vollrath F, Meier BH (1996) Local structure in spider dragline silk investigated by two dimensional spin-diffusion NMR. Macromolecules  29: 2920–2928.  DOI: 10.1021/ma951098i.  <ul class="find" data-citedArticleID="957548" data-doi="10.1021/ma951098i"><li><a href="http://dx.doi.org/10.1021/ma951098i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Local+structure+in+spider+dragline+silk+investigated+by+two+dimensional+spin-diffusion+NMR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Local+structure+in+spider+dragline+silk+investigated+by+two+dimensional+spin-diffusion+NMR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000514-Simmons2" id="pone.0000514-Simmons2"></a>Simmons AH, Michal CA, Jelinski LW (1996) Molecular orientation and two-component nature of the crystalline fraction of spider dragline silk. Science  271: 84–87.  DOI: 10.1126/science.271.5245.84.  <ul class="find" data-citedArticleID="957616" data-doi="10.1126/science.271.5245.84"><li><a href="http://dx.doi.org/10.1126/science.271.5245.84" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+orientation+and+two-component+nature+of+the+crystalline+fraction+of+spider+dragline+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+orientation+and+two-component+nature+of+the+crystalline+fraction+of+spider+dragline+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000514-Bram1" id="pone.0000514-Bram1"></a>Bram A, Bränden CI, Craig C, Snigireva I, Riekel C (1997) X-ray diffraction from single fibers of spider silk. J Appl Cryst  30: 390–392.  DOI: 10.1107/S0021889896012344.  <ul class="find" data-citedArticleID="957466" data-doi="10.1107/s0021889896012344"><li><a href="http://dx.doi.org/10.1107/s0021889896012344" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X-ray+diffraction+from+single+fibers+of+spider+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X-ray+diffraction+from+single+fibers+of+spider+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000514-Parkhe1" id="pone.0000514-Parkhe1"></a>Parkhe AD, Seeley SK, Gardner K, Thompson L, Lewis RV (1997) Structural studies of spider silk proteins in the fiber. J Mol Recogn  10: 1–6.  DOI: 10.1002/(SICI)1099-1352(199701/02)10:1&lt;1​::AID-JMR338&gt;3.0.CO;2-7.  <ul class="find" data-citedArticleID="957580" data-doi="10.1002/(sici)1099-1352(199701/02)10:1<1::aid-jmr338&gt;3.0.co;2-7"><li><a href="http://dx.doi.org/10.1002/(sici)1099-1352(199701/02)10:1<1::aid-jmr338&gt;3.0.co;2-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+studies+of+spider+silk+proteins+in+the+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+studies+of+spider+silk+proteins+in+the+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000514-Dong1" id="pone.0000514-Dong1"></a>Dong Z, Lewis RV, Middaugh CR (1991) Molecular mechanisms of spider silk elasticity. Arch Biochem Biophys  284: 53–57.  <ul class="find" data-citedArticleID="957490" data-doi="10.1016/0003-9861(91)90262-h"><li><a href="http://dx.doi.org/10.1016/0003-9861(91)90262-h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanisms+of+spider+silk+elasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanisms+of+spider+silk+elasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000514-Hayashi3" id="pone.0000514-Hayashi3"></a>Hayashi CY, Lewis RV (1998) Evidence from flagelliform silk cDNA for the structural basis of elasticity and modular nature of spider silks. J Mol Biol  275: 773–784.  DOI: 10.1006/jmbi.1997.1478.  <ul class="find" data-citedArticleID="957530" data-doi="10.1006/jmbi.1997.1478"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.1478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+from+flagelliform+silk+cDNA+for+the+structural+basis+of+elasticity+and+modular+nature+of+spider+silks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+from+flagelliform+silk+cDNA+for+the+structural+basis+of+elasticity+and+modular+nature+of+spider+silks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000514-vanBeek1" id="pone.0000514-vanBeek1"></a>van Beek JD, Hess S, Vollrath F, Meier BH (2002) The molecular structure of spider dragline silk: folding and orientation of the protein backbone Proc Natl Acad Sci USA  99: 10266–10271.  DOI: 10.1073/pnas.152162299.  <ul class="find" data-citedArticleID="957656" data-doi="10.1073/pnas.152162299"><li><a href="http://dx.doi.org/10.1073/pnas.152162299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+structure+of+spider+dragline+silk%3A+folding+and+orientation+of+the+protein+backbone" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+structure+of+spider+dragline+silk%3A+folding+and+orientation+of+the+protein+backbone%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000514-OBrien1" id="pone.0000514-OBrien1"></a>O'Brien JP, Fahnestock SR, Termonia Y, Gardner KH (1998) Nylons from nature: synthetic analogs to spider silk. Adv Mater  10: 1185–1195.  DOI: 10.1002/(SICI)1521-4095(199810)10:15&lt;118​5::AID-ADMA1185&gt;3.0.CO;2-T.  <ul class="find" data-citedArticleID="957578" data-doi="10.1002/(sici)1521-4095(199810)10:15<1185::aid-adma1185&gt;3.0.co;2-t"><li><a href="http://dx.doi.org/10.1002/(sici)1521-4095(199810)10:15<1185::aid-adma1185&gt;3.0.co;2-t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nylons+from+nature%3A+synthetic+analogs+to+spider+silk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nylons+from+nature%3A+synthetic+analogs+to+spider+silk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000514-Hinman3" id="pone.0000514-Hinman3"></a>Hinman MB, Jones JA, Lewis RV (2000) Synthetic spider silk: a modular fiber. Trends Biotechnol  18: 374–379.  DOI: 10.1016/S0167-7799(00)01481-5.  <ul class="find" data-citedArticleID="957538" data-doi="10.1016/s0167-7799(00)01481-5"><li><a href="http://dx.doi.org/10.1016/s0167-7799(00)01481-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthetic+spider+silk%3A+a+modular+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthetic+spider+silk%3A+a+modular+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000514-Lazaris1" id="pone.0000514-Lazaris1"></a>Lazaris A, Arcidiacono S, Huang Y, Zhou J-F, Duguay F, et al.  (2002) Spider silk fibers spun from soluble recombinant silk produced in mammalian cells. Science  295: 472–476.  DOI: 10.1126/science.1065780.  <ul class="find" data-citedArticleID="957560" data-doi="10.1126/science.1065780"><li><a href="http://dx.doi.org/10.1126/science.1065780" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spider+silk+fibers+spun+from+soluble+recombinant+silk+produced+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spider+silk+fibers+spun+from+soluble+recombinant+silk+produced+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000514-Scheller1" id="pone.0000514-Scheller1"></a>Scheller J, Henggeler D, Viviani A, Conrad U (2004) Purification of spider silk-elastin from transgenic plants and application for human chondrocyte proliferation. Transgenic Res  13: 51–57.  DOI: 10.1023/B:TRAG.0000017175.78809.7a.  <ul class="find" data-citedArticleID="957604" data-doi="10.1023/b:trag.0000017175.78809.7a"><li><a href="http://dx.doi.org/10.1023/b:trag.0000017175.78809.7a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+of+spider+silk-elastin+from+transgenic+plants+and+application+for+human+chondrocyte+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+of+spider+silk-elastin+from+transgenic+plants+and+application+for+human+chondrocyte+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000514-Bini1" id="pone.0000514-Bini1"></a>Bini E, Foo CWP, Huang J, Karageorgiou V, Kitchel B, et al.  (2006) RGD-functionalized bioengineered spider dragline silk biomaterial Biomacromolecules  7: 3139–3145.  DOI: 10.1021/bm0607877.  <ul class="find" data-citedArticleID="957458" data-doi="10.1021/bm0607877"><li><a href="http://dx.doi.org/10.1021/bm0607877" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RGD-functionalized+bioengineered+spider+dragline+silk+biomaterial" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RGD-functionalized+bioengineered+spider+dragline+silk+biomaterial%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000514-Scheller2" id="pone.0000514-Scheller2"></a>Scheller J, Gührs K-H, Grosse F, Conrad U (2001) Production of spider silk proteins in tobacco and potato. Nat Biotechnol  19: 573–577.  DOI:10.1038/89335.  <ul class="find" data-citedArticleID="957606" data-doi="10.1038/89335"><li><a href="http://dx.doi.org/10.1038/89335" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+of+spider+silk+proteins+in+tobacco+and+potato." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+of+spider+silk+proteins+in+tobacco+and+potato.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000514-Foo2" id="pone.0000514-Foo2"></a>Foo CWP, Bini E, Huang J, Lee SY, Kaplan DL (2006) Solution behavior of synthetic silk peptides and modified recombinant silk proteins. Appl Phys A  82: 193–203.  DOI: 10.1007/s00339-005-3425-8.  <ul class="find" data-citedArticleID="957496" data-doi="10.1007/s00339-005-3425-8"><li><a href="http://dx.doi.org/10.1007/s00339-005-3425-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solution+behavior+of+synthetic+silk+peptides+and+modified+recombinant+silk+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solution+behavior+of+synthetic+silk+peptides+and+modified+recombinant+silk+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000514-Ittah1" id="pone.0000514-Ittah1"></a>Ittah S, Cohen S, Garty S, Cohn D, Gat U (2006) An essential role for the C-terminal domain of a dragline spider silk protein in directing fiber formation. Biomacromolecules  7: 1790–1795.  DOI: 10.1021/bm060120k.  <ul class="find" data-citedArticleID="957544" data-doi="10.1021/bm060120k"><li><a href="http://dx.doi.org/10.1021/bm060120k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+essential+role+for+the+C-terminal+domain+of+a+dragline+spider+silk+protein+in+directing+fiber+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+essential+role+for+the+C-terminal+domain+of+a+dragline+spider+silk+protein+in+directing+fiber+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000514-Sprague1" id="pone.0000514-Sprague1"></a>Sprague KU (1975) The <em>Bombyx mori</em> silk proteins: characterization of large polypeptides. Biochemistry  14: 925–931.  DOI: 10.1021/bi00676a008.  <ul class="find" data-citedArticleID="957626" data-doi="10.1021/bi00676a008"><li><a href="http://dx.doi.org/10.1021/bi00676a008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bombyx+mori+silk+proteins%3A+characterization+of+large+polypeptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bombyx+mori+silk+proteins%3A+characterization+of+large+polypeptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000514-Sponner3" id="pone.0000514-Sponner3"></a>Sponner A, Vater W, Rommerskirch W, Vollrath F, Unger E, et al.  (2005) The conserved C-termini contribute to the properties of spider silk fibroins. Biochem Biophys Res Commun  338: 897–902.  DOI: 10.1016/j.bbrc.2005.10.048.  <ul class="find" data-citedArticleID="957624" data-doi="10.1016/j.bbrc.2005.10.048"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.10.048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+conserved+C-termini+contribute+to+the+properties+of+spider+silk+fibroins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+conserved+C-termini+contribute+to+the+properties+of+spider+silk+fibroins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000514-MotriukSmith1" id="pone.0000514-MotriukSmith1"></a>Motriuk-Smith D, Smith A, Hayashi CY, Lewis RV (2005) Analysis of the conserved N-terminal domains in major ampullate spider silk proteins. Biomacromolecules  6: 3152–3159.  DOI: 10.1021/bm050472b.  <ul class="find" data-citedArticleID="957576" data-doi="10.1021/bm050472b"><li><a href="http://dx.doi.org/10.1021/bm050472b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+conserved+N-terminal+domains+in+major+ampullate+spider+silk+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+conserved+N-terminal+domains+in+major+ampullate+spider+silk+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000514-Rising1" id="pone.0000514-Rising1"></a>Rising A, Hjälm G, Engström W, Johansson J (2006) N-terminal nonrepetitive domain common to dragline, flagelliform, and cylindriform spider silk proteins. Biomacromolecules  7: 3120–3124.  DOI: 10.1021/bm060693x.  <ul class="find" data-citedArticleID="957588" data-doi="10.1021/bm060693x"><li><a href="http://dx.doi.org/10.1021/bm060693x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=N-terminal+nonrepetitive+domain+common+to+dragline%2C+flagelliform%2C+and+cylindriform+spider+silk+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22N-terminal+nonrepetitive+domain+common+to+dragline%2C+flagelliform%2C+and+cylindriform+spider+silk+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000514-Garb2" id="pone.0000514-Garb2"></a>Garb JE, Hayashi CY (2005) Modular evolution of egg case silk genes across orb-weaving spider superfamilies. Proc Natl Acad Sci USA  102: 11379–11384.  DOI: 10.1073/pnas.0502473102.  <ul class="find" data-citedArticleID="957502" data-doi="10.1073/pnas.0502473102"><li><a href="http://dx.doi.org/10.1073/pnas.0502473102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modular+evolution+of+egg+case+silk+genes+across+orb-weaving+spider+superfamilies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modular+evolution+of+egg+case+silk+genes+across+orb-weaving+spider+superfamilies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000514-Hayashi4" id="pone.0000514-Hayashi4"></a>Hayashi CY, Lewis RV (2000) Molecular architecture and evolution of a modular spider silk protein gene. Science  287: 1477–1479.  DOI: 10.1126/science.287.5457.1477.  <ul class="find" data-citedArticleID="957532" data-doi="10.1126/science.287.5457.1477"><li><a href="http://dx.doi.org/10.1126/science.287.5457.1477" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+architecture+and+evolution+of+a+modular+spider+silk+protein+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+architecture+and+evolution+of+a+modular+spider+silk+protein+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000514-Griswold1" id="pone.0000514-Griswold1"></a>Griswold CE, Coddington JA, Hormiga G, Scharff N (1998) Phylogeny of the orb-web building spiders (Araneae, Orbiculariae: Deinopoidea, Araneoidea). Zool J Linn Soc–Lond  123: 1–99.  DOI: 10.1006/zjls.1997.0125.  <ul class="find" data-citedArticleID="957518" data-doi="10.1111/j.1096-3642.1998.tb01290.x"><li><a href="http://dx.doi.org/10.1111/j.1096-3642.1998.tb01290.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+the+orb-web+building+spiders+%28Araneae%2C+Orbiculariae%3A+Deinopoidea%2C+Araneoidea%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+the+orb-web+building+spiders+%28Araneae%2C+Orbiculariae%3A+Deinopoidea%2C+Araneoidea%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000514-Lawrence1" id="pone.0000514-Lawrence1"></a>Lawrence BA, Vierra CA, Moore AF (2004) Molecular and mechanical properties of major ampullate silk of the black widow spider, <em>Latrodectus hesperus</em>. Biomacromolecules  5: 689–695.  DOI: 10.1021/bm0342640.  <ul class="find" data-citedArticleID="957558" data-doi="10.1021/bm0342640"><li><a href="http://dx.doi.org/10.1021/bm0342640" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+mechanical+properties+of+major+ampullate+silk+of+the+black+widow+spider%2C+Latrodectus+hesperus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+mechanical+properties+of+major+ampullate+silk+of+the+black+widow+spider%2C+Latrodectus+hesperus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000514-Blackledge2" id="pone.0000514-Blackledge2"></a>Blackledge TA, Summers AP, Hayashi CY (2005) Gumfooted lines in black widow cobwebs and the mechanical properties of spider capture silk Zoology  108: 41–46.  DOI: 10.1016/j.zool.2004.11.001.  <ul class="find" data-citedArticleID="957462" data-doi="10.1016/j.zool.2004.11.001"><li><a href="http://dx.doi.org/10.1016/j.zool.2004.11.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gumfooted+lines+in+black+widow+cobwebs+and+the+mechanical+properties+of+spider+capture+silk" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gumfooted+lines+in+black+widow+cobwebs+and+the+mechanical+properties+of+spider+capture+silk%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000514-Swanson1" id="pone.0000514-Swanson1"></a>Swanson BO, Blackledge TA, Beltrán J, Hayashi CY (2006) Variation in the material properties of spider dragline silk across species Appl Phys A  82: 213–218.  DOI: 10.1007/s00339-005-3427-6.  <ul class="find" data-citedArticleID="957628" data-doi="10.1007/s00339-005-3427-6"><li><a href="http://dx.doi.org/10.1007/s00339-005-3427-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+the+material+properties+of+spider+dragline+silk+across+species" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+the+material+properties+of+spider+dragline+silk+across+species%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000514-Casem1" id="pone.0000514-Casem1"></a>Casem ML, Turner D, Houchin K (1999) Protein and amino acid composition of silks from the cob weaver, <em>Latrodectus hesperus</em> (black widow). Int J Biol Macromol  24: 103–108.  DOI: 10.1016/S0141-8130(98)00078-6.  <ul class="find" data-citedArticleID="957472" data-doi="10.1016/s0141-8130(98)00078-6"><li><a href="http://dx.doi.org/10.1016/s0141-8130(98)00078-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+and+amino+acid+composition+of+silks+from+the+cob+weaver%2C+Latrodectus+hesperus+%28black+widow%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+and+amino+acid+composition+of+silks+from+the+cob+weaver%2C+Latrodectus+hesperus+%28black+widow%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000514-Anderson1" id="pone.0000514-Anderson1"></a>Anderson SO (1970) Amino acid composition of spider silks. Comp Biochem Physiol  35: 705–711.  <ul class="find" data-citedArticleID="957444" data-doi="10.1016/0010-406x(70)90988-6"><li><a href="http://dx.doi.org/10.1016/0010-406x(70)90988-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+composition+of+spider+silks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+composition+of+spider+silks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000514-Lombardi1" id="pone.0000514-Lombardi1"></a>Lombardi SJ, Kaplan DL (1990) The amino acid composition of major ampullate gland silk (dragline) of <em>Nephila clavipes</em> (Araneae, Tetragnathidae). J Arachnol  18: 297–306.  <ul class="find" data-citedArticleID="957566"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+amino+acid+composition+of+major+ampullate+gland+silk+%28dragline%29+of+Nephila+clavipes+%28Araneae%2C+Tetragnathidae%29.&amp;auth=&amp;atitle=The+amino+acid+composition+of+major+ampullate+gland+silk+%28dragline%29+of+Nephila+clavipes+%28Araneae%2C+Tetragnathidae%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+amino+acid+composition+of+major+ampullate+gland+silk+%28dragline%29+of+Nephila+clavipes+%28Araneae%2C+Tetragnathidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+amino+acid+composition+of+major+ampullate+gland+silk+%28dragline%29+of+Nephila+clavipes+%28Araneae%2C+Tetragnathidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000514-Beckwitt1" id="pone.0000514-Beckwitt1"></a>Beckwitt R, Arcidiacono S, Stote R (1998) Evolution of repetitive proteins: spider silks from <em>Nephila clavipes</em> (Tetragnathidae) and <em>Araneus bicentenarius</em> (Araneidae). Insect Biochem Mol Biol  28: 121–130.  DOI: 10.1016/S0965-1748(97)00083-0.  <ul class="find" data-citedArticleID="957452" data-doi="10.1016/s0965-1748(97)00083-0"><li><a href="http://dx.doi.org/10.1016/s0965-1748(97)00083-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+repetitive+proteins%3A+spider+silks+from+Nephila+clavipes+%28Tetragnathidae%29+and+Araneus+bicentenarius+%28Araneidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+repetitive+proteins%3A+spider+silks+from+Nephila+clavipes+%28Tetragnathidae%29+and+Araneus+bicentenarius+%28Araneidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000514-Kyte1" id="pone.0000514-Kyte1"></a>Kyte J, Doolittle R (1982) A simple method for displaying the hydropathic character of a protein. J Mol Biol  157: 105–132.  <ul class="find" data-citedArticleID="957552" data-doi="10.1016/0022-2836(82)90515-0"><li><a href="http://dx.doi.org/10.1016/0022-2836(82)90515-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+method+for+displaying+the+hydropathic+character+of+a+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+method+for+displaying+the+hydropathic+character+of+a+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000514-Hardison1" id="pone.0000514-Hardison1"></a>Hardison RC (2003) Comparative genomics. PLoS Biology  1: 156–160.  DOI: 10.1371/journal.pbio.0000058.  <ul class="find" data-citedArticleID="957522" data-doi="10.1371/journal.pbio.0000058"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000514-Rombauts1" id="pone.0000514-Rombauts1"></a>Rombauts S, Florquin K, Lescot M, Marchal K, Rouzé P, et al.  (2003) Computational approaches to identify promoters and cis-regulatory elements in plant genomes. Plant Physiol  132: 1162–1176.  DOI: 10.1104/pp.102.017715.  <ul class="find" data-citedArticleID="957590" data-doi="10.1104/pp.102.017715"><li><a href="http://dx.doi.org/10.1104/pp.102.017715" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+approaches+to+identify+promoters+and+cis-regulatory+elements+in+plant+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+approaches+to+identify+promoters+and+cis-regulatory+elements+in+plant+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000514-Schwartz1" id="pone.0000514-Schwartz1"></a>Schwartz S, Zhang Z, Frazer KA, Smit A, Riemer C, et al.  (2000) PipMaker–A web server for aligning two genomic DNA sequences. Genome Res  10: 577–586.  <ul class="find" data-citedArticleID="957608" data-doi="10.1101/gr.10.4.577"><li><a href="http://dx.doi.org/10.1101/gr.10.4.577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PipMaker%26%23x2013%3BA+web+server+for+aligning+two+genomic+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PipMaker%26%23x2013%3BA+web+server+for+aligning+two+genomic+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000514-Schwartz2" id="pone.0000514-Schwartz2"></a>Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, et al.  (2003) Human-mouse alignments with BLASTZ. Genome Res  13: 103–107.  DOI: 10.1101/gr.809403.  <ul class="find" data-citedArticleID="957610" data-doi="10.1101/gr.809403"><li><a href="http://dx.doi.org/10.1101/gr.809403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human-mouse+alignments+with+BLASTZ." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human-mouse+alignments+with+BLASTZ.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000514-Pollard1" id="pone.0000514-Pollard1"></a>Pollard DA, Moses AM, Iyer VN, Eisen MB (2006) Detecting the limits of regulatory element conservation and divergence estimation using pairwise and multiple alignments. BMC Bioinformatics  7: 376.  DOI: 10.1186/1471-2105-7-376.  <ul class="find" data-citedArticleID="957582"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Detecting+the+limits+of+regulatory+element+conservation+and+divergence+estimation+using+pairwise+and+multiple+alignments.&amp;auth=&amp;atitle=Detecting+the+limits+of+regulatory+element+conservation+and+divergence+estimation+using+pairwise+and+multiple+alignments." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+the+limits+of+regulatory+element+conservation+and+divergence+estimation+using+pairwise+and+multiple+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+the+limits+of+regulatory+element+conservation+and+divergence+estimation+using+pairwise+and+multiple+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000514-Margulies1" id="pone.0000514-Margulies1"></a>Margulies EH, Blanchette M, Haussler D, Green ED, NISC Comparative Sequencing Program (2003) Identification and characterization of multiple-species conserved sequences. Genome Res  13: 2507–2518.  DOI: 10.1101/gr.1602203.  <ul class="find" data-citedArticleID="957570" data-doi="10.1101/gr.1602203"><li><a href="http://dx.doi.org/10.1101/gr.1602203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+multiple-species+conserved+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+multiple-species+conserved+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000514-Matys1" id="pone.0000514-Matys1"></a>Matys V, Kel-Margoulis OV, Fricke E, Liebich I, Land S, et al.  (2006) TRANSFAC® and its module TRANSCompel®: transcriptional gene regulation in eukaryotes. Nucleic Acids Res  34: D108–D110.  &lt;<a href="http://www.gene-regulation.com/pub/databases.html">http://www.gene-regulation.com/pub/datab​ases.html</a>&gt; DOI: 10.1093/nar/gkj143.  <ul class="find" data-citedArticleID="957572" data-doi="10.1093/nar/gkj143"><li><a href="http://dx.doi.org/10.1093/nar/gkj143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRANSFAC%26%23xae%3B+and+its+module+TRANSCompel%26%23xae%3B%3A+transcriptional+gene+regulation+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRANSFAC%26%23xae%3B+and+its+module+TRANSCompel%26%23xae%3B%3A+transcriptional+gene+regulation+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000514-Graur1" id="pone.0000514-Graur1"></a>Graur D, Li WH (2000) Fundamentals of Molecular Evolution, second edition. Sunderland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0000514-Wong1" id="pone.0000514-Wong1"></a>Wong WSW, Nielsen R (2004) Detecting selection in noncoding regions of nucleotide sequences. Genetics  167: 949–958.  DOI: 10.1534/genetics.102.010959.  <ul class="find" data-citedArticleID="957646" data-doi="10.1534/genetics.102.010959"><li><a href="http://dx.doi.org/10.1534/genetics.102.010959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+selection+in+noncoding+regions+of+nucleotide+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+selection+in+noncoding+regions+of+nucleotide+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000514-Bray1" id="pone.0000514-Bray1"></a>Bray N, Dubchak I, Pachter L (2003) AVID: A Global Alignment Program. Genome Res  13: 97–102.  DOI: 10.1101/gr.789803.  <ul class="find" data-citedArticleID="957468" data-doi="10.1101/gr.789803"><li><a href="http://dx.doi.org/10.1101/gr.789803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AVID%3A+A+Global+Alignment+Program." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AVID%3A+A+Global+Alignment+Program.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000514-Altschul1" id="pone.0000514-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="957442" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000514-Sakharkar1" id="pone.0000514-Sakharkar1"></a>Sakharkar MK, Kangueane P, Petrov DA, Kolaskar AS, Subbiah S (2002) SEGE: A database on ‘intron less/single exonic’ genes from eukaryotes. Bioinformatics  18: 1266–1267.  <ul class="find" data-citedArticleID="957598" data-doi="10.1093/bioinformatics/18.9.1266"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.9.1266" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SEGE%3A+A+database+on+%26%23x2018%3Bintron+less%2Fsingle+exonic%26%23x2019%3B+genes+from+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SEGE%3A+A+database+on+%26%23x2018%3Bintron+less%2Fsingle+exonic%26%23x2019%3B+genes+from+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000514-Sakharkar2" id="pone.0000514-Sakharkar2"></a>Sakharkar MK, Kangueane P (2004) Genome SEGE: A database for ‘intronless’ genes in eukaryotic genomes. BMC Bioinformatics  5: 67.  DOI: 10.1186/1471-2105-5-67.  <ul class="find" data-citedArticleID="957600" data-doi="10.1186/1471-2105-5-67"><li><a href="http://dx.doi.org/10.1186/1471-2105-5-67" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+SEGE%3A+A+database+for+%26%23x2018%3Bintronless%26%23x2019%3B+genes+in+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+SEGE%3A+A+database+for+%26%23x2018%3Bintronless%26%23x2019%3B+genes+in+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000514-Zhang1" id="pone.0000514-Zhang1"></a>Zhang J (2003) Evolution by gene duplication: an update. Trends Ecol Evol  18: 292–298.  DOI: 10.1016/S0169-5347(03)00033-8.  <ul class="find" data-citedArticleID="957652" data-doi="10.1016/s0169-5347(03)00033-8"><li><a href="http://dx.doi.org/10.1016/s0169-5347(03)00033-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+by+gene+duplication%3A+an+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+by+gene+duplication%3A+an+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000514-CastilloDavis1" id="pone.0000514-CastilloDavis1"></a>Castillo-Davis CI, Mekhedov SL, Hartl DL, Koonin EV, Kondrashov FA (2002) Selection for short introns in highly expressed genes. Nat Genet  31: 415–418.  DOI: 10.1038/ng940.  <ul class="find" data-citedArticleID="957474" data-doi="10.1038/ng940"><li><a href="http://dx.doi.org/10.1038/ng940" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+for+short+introns+in+highly+expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+for+short+introns+in+highly+expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000514-Urrutia1" id="pone.0000514-Urrutia1"></a>Urrutia AO, Hurst LD (2003) The signature of selection mediated by expression on human genes. Genome Res  13: 2260–2264.  DOI: 10.1101/gr.641103.  <ul class="find" data-citedArticleID="957638" data-doi="10.1101/gr.641103"><li><a href="http://dx.doi.org/10.1101/gr.641103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+signature+of+selection+mediated+by+expression+on+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+signature+of+selection+mediated+by+expression+on+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000514-Marais1" id="pone.0000514-Marais1"></a>Marais G, Nouvellet P, Keightley PD, Charlesworth B (2005) Intron size and exon evolution in <em>Drosophila</em>. Genetics  170: 481–485.  DOI: 10.1534/genetics.104.037333.  <ul class="find" data-citedArticleID="957568" data-doi="10.1534/genetics.104.037333"><li><a href="http://dx.doi.org/10.1534/genetics.104.037333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intron+size+and+exon+evolution+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intron+size+and+exon+evolution+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000514-Craig1" id="pone.0000514-Craig1"></a>Craig CL, Riekel C, Herberstein ME, Weber RS, Kaplan D, et al.  (2000) Evidence for diet effects on the composition of silk proteins produced by spiders. Mol Biol Evol  17: 1904–1913.  <ul class="find" data-citedArticleID="957482" data-doi="10.1093/oxfordjournals.molbev.a026292"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+diet+effects+on+the+composition+of+silk+proteins+produced+by+spiders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+diet+effects+on+the+composition+of+silk+proteins+produced+by+spiders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000514-Beckwitt2" id="pone.0000514-Beckwitt2"></a>Beckwitt R, Arcidiacono S (1994) Sequence conservation in the C-terminal region of spider silk proteins (spidroin) from <em>Nephila clavipes</em> (Tetragnathidae) and <em>Araneus bicentenarius</em> (Araneidae). J Biol Chem  269: 6661–6663.  <ul class="find" data-citedArticleID="957454" data-doi="10.1016/s0965-1748(97)00083-0"><li><a href="http://dx.doi.org/10.1016/s0965-1748(97)00083-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+conservation+in+the+C-terminal+region+of+spider+silk+proteins+%28spidroin%29+from+Nephila+clavipes+%28Tetragnathidae%29+and+Araneus+bicentenarius+%28Araneidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+conservation+in+the+C-terminal+region+of+spider+silk+proteins+%28spidroin%29+from+Nephila+clavipes+%28Tetragnathidae%29+and+Araneus+bicentenarius+%28Araneidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000514-Dicko1" id="pone.0000514-Dicko1"></a>Dicko C, Vollrath F, Kenney JM (2004) Spider silk protein refolding is controlled by changing pH. Biomacromolecules  5: 704–710.  DOI: 10.1021/bm034307c.  <ul class="find" data-citedArticleID="957488" data-doi="10.1021/bm034307c"><li><a href="http://dx.doi.org/10.1021/bm034307c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spider+silk+protein+refolding+is+controlled+by+changing+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spider+silk+protein+refolding+is+controlled+by+changing+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000514-Selden1" id="pone.0000514-Selden1"></a>Selden PA (1990) Lower Cretaceous spiders from the Sierra-de-Montsech, north-east Spain. Palaeontology  33: 257–285.  <ul class="find" data-citedArticleID="957612"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lower+Cretaceous+spiders+from+the+Sierra-de-Montsech%2C+north-east+Spain.&amp;auth=&amp;atitle=Lower+Cretaceous+spiders+from+the+Sierra-de-Montsech%2C+north-east+Spain." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lower+Cretaceous+spiders+from+the+Sierra-de-Montsech%2C+north-east+Spain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lower+Cretaceous+spiders+from+the+Sierra-de-Montsech%2C+north-east+Spain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000514-Ayoub1" id="pone.0000514-Ayoub1"></a>Ayoub NA, Garb JE, Hedin M, Hayashi CY (2007) Utility of the nuclear protein-coding gene, elongation factor-1 gamma (<em>EF-1γ</em>), for spider systematics, emphasizing family level relationships of tarantulas and their kin (Araneae: Mygalomorphae). Mol Phylogenet Evol  42: 394–409.  DOI: 10.1016/j.ympev.2006.07.018.  <ul class="find" data-citedArticleID="957450" data-doi="10.1016/j.ympev.2006.07.018"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.07.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Utility+of+the+nuclear+protein-coding+gene%2C+elongation+factor-1+gamma+%28EF-1%26%23x3b3%3B%29%2C+for+spider+systematics%2C+emphasizing+family+level+relationships+of+tarantulas+and+their+kin+%28Araneae%3A+Mygalomorphae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Utility+of+the+nuclear+protein-coding+gene%2C+elongation+factor-1+gamma+%28EF-1%26%23x3b3%3B%29%2C+for+spider+systematics%2C+emphasizing+family+level+relationships+of+tarantulas+and+their+kin+%28Araneae%3A+Mygalomorphae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000514-Sinha1" id="pone.0000514-Sinha1"></a>Sinha S, Siggia ED (2005) Sequence turnover and tandem repeats in <em>cis</em>-regulatory modules in <em>Drosophila</em>. Mol Biol Evol  22: 874–885.  DOI: 10.1093/molbev.msi090.  <ul class="find" data-citedArticleID="957618" data-doi="10.1093/molbev/msi090"><li><a href="http://dx.doi.org/10.1093/molbev/msi090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+turnover+and+tandem+repeats+in+cis-regulatory+modules+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+turnover+and+tandem+repeats+in+cis-regulatory+modules+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000514-Cliften1" id="pone.0000514-Cliften1"></a>Cliften P, Sudarsanam P, Desikan A, Fulton L, Fulton B, et al.  (2003) Finding functional elements in <em>Saccharomyces</em> genomes by phylogenetic footprinting. Science  301: 71–76.  DOI: 10.1126/science.1084337.  <ul class="find" data-citedArticleID="957478" data-doi="10.1126/science.1084337"><li><a href="http://dx.doi.org/10.1126/science.1084337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+functional+elements+in+Saccharomyces+genomes+by+phylogenetic+footprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+functional+elements+in+Saccharomyces+genomes+by+phylogenetic+footprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000514-Kellis1" id="pone.0000514-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  DOI: 10.1038/nature01644.  <ul class="find" data-citedArticleID="957546" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000514-Inada1" id="pone.0000514-Inada1"></a>Inada DC, Bashir A, Lee C, Thomas BC, Ko C, et al.  (2003) Conserved noncoding sequences in the grasses. Genome Res  13: 2030–2041.  DOI: 10.1101/gr.1280703.  <ul class="find" data-citedArticleID="957542" data-doi="10.1101/gr.1280703"><li><a href="http://dx.doi.org/10.1101/gr.1280703" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+noncoding+sequences+in+the+grasses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+noncoding+sequences+in+the+grasses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000514-Boffelli1" id="pone.0000514-Boffelli1"></a>Boffelli D, McAuliffe J, Ovcharenko D, Lewis KD, Ovcharenko I, et al.  (2003) Phylogenetic shadowing of primate sequences to find functional regions of the human genome. Science  299: 1391–1394.  DOI: 10.1126/science.1081331.  <ul class="find" data-citedArticleID="957464" data-doi="10.1126/science.1081331"><li><a href="http://dx.doi.org/10.1126/science.1081331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+shadowing+of+primate+sequences+to+find+functional+regions+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+shadowing+of+primate+sequences+to+find+functional+regions+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000514-Arnason1" id="pone.0000514-Arnason1"></a>Arnason U, Gullberg A, Burguete AS, Janice A (2000) Molecular estimates of primate divergences and new hypotheses for primate dispersal and the origin of modern humans. Hereditas  133: 217–228.  DOI: 10.1111/j.1601-5223.2000.00217.x.  <ul class="find" data-citedArticleID="957448" data-doi="10.1111/j.1601-5223.2000.00217.x"><li><a href="http://dx.doi.org/10.1111/j.1601-5223.2000.00217.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+estimates+of+primate+divergences+and+new+hypotheses+for+primate+dispersal+and+the+origin+of+modern+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+estimates+of+primate+divergences+and+new+hypotheses+for+primate+dispersal+and+the+origin+of+modern+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000514-Benton1" id="pone.0000514-Benton1"></a>Benton MJ (1990) Phylogeny of the major tetrapod groups: morphological data and divergence dates. J Mol Evol  30: 409–424.  DOI: 10.1007/BF02101113.  <ul class="find" data-citedArticleID="957456" data-doi="10.1007/bf02101113"><li><a href="http://dx.doi.org/10.1007/bf02101113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+of+the+major+tetrapod+groups%3A+morphological+data+and+divergence+dates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+of+the+major+tetrapod+groups%3A+morphological+data+and+divergence+dates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000514-Lee1" id="pone.0000514-Lee1"></a>Lee MSY (1999) Molecular clock calibration and metazoan divergence dates. J Mol Evol  49: 385–391.  DOI: 10.1007/PL00006562.  <ul class="find" data-citedArticleID="957562" data-doi="10.1007/pl00006562"><li><a href="http://dx.doi.org/10.1007/pl00006562" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+clock+calibration+and+metazoan+divergence+dates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+clock+calibration+and+metazoan+divergence+dates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000514-Thomas1" id="pone.0000514-Thomas1"></a>Thomas JW, Touchman JW, Blakesley RW, Bouffard GG, Beckstrom-Sternberg SM, et al.  (2003) Comparative analyses of multi-species sequences from targeted genomic regions. Nature  424: 788–793.  DOI: 10.1038/nature01858.  <ul class="find" data-citedArticleID="957634" data-doi="10.1038/nature01858"><li><a href="http://dx.doi.org/10.1038/nature01858" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analyses+of+multi-species+sequences+from+targeted+genomic+regions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analyses+of+multi-species+sequences+from+targeted+genomic+regions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000514-Prakash1" id="pone.0000514-Prakash1"></a>Prakash A, Tompa M (2005) Discovery of regulatory elements in vertebrates through comparative genomics. Nat Biotechnol  23: 1249–1256.  DOI: 10.1038/nbt1140.  <ul class="find" data-citedArticleID="957586" data-doi="10.1038/nbt1140"><li><a href="http://dx.doi.org/10.1038/nbt1140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+regulatory+elements+in+vertebrates+through+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+regulatory+elements+in+vertebrates+through+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000514-DeBodt1" id="pone.0000514-DeBodt1"></a>De Bodt S, Theissen G, Van de Peer Y (2006) Promoter analysis of MADS-box genes in eudicots through phylogenetic footprinting. Mol Biol Evol  23: 1293–1303.  DOI: 10.1093/molbev/msk016.  <ul class="find" data-citedArticleID="957484" data-doi="10.1093/molbev/msk016"><li><a href="http://dx.doi.org/10.1093/molbev/msk016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promoter+analysis+of+MADS-box+genes+in+eudicots+through+phylogenetic+footprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promoter+analysis+of+MADS-box+genes+in+eudicots+through+phylogenetic+footprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000514-Chaw1" id="pone.0000514-Chaw1"></a>Chaw S-M, Chang C-C, Chen H-L, Li W-H (2001) Dating the monocot-dicot divergence and the origin of core eudicots using whole chloroplast genomes. J Mol Evol  58: 424–441.  DOI: 10.1007/S00239-003-2564-9.  <ul class="find" data-citedArticleID="957476" data-doi="10.1007/s00239-003-2564-9"><li><a href="http://dx.doi.org/10.1007/s00239-003-2564-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+the+monocot-dicot+divergence+and+the+origin+of+core+eudicots+using+whole+chloroplast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+the+monocot-dicot+divergence+and+the+origin+of+core+eudicots+using+whole+chloroplast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000514-Wikstrm1" id="pone.0000514-Wikstrm1"></a>Wikström N, Savolainen V, Chase MW (2001) Evolution of the angiosperms: calibrating the family tree. Proc R Soc Lond B Biol Sci  268: 2211–2220.  DOI: 10.1098/rspb.2001.1782.  <ul class="find" data-citedArticleID="957644" data-doi="10.1098/rspb.2001.1782"><li><a href="http://dx.doi.org/10.1098/rspb.2001.1782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+angiosperms%3A+calibrating+the+family+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+angiosperms%3A+calibrating+the+family+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000514-Cabrera1" id="pone.0000514-Cabrera1"></a>Cabrera CV, Alonso MC (1991) Transcriptional activation by heterodimers of the <em>achaete-scute</em> and <em>daughterless</em> gene products of <em>Drosophila</em>. EMBO J  10: 2965–2973.  <ul class="find" data-citedArticleID="957470"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptional+activation+by+heterodimers+of+the+achaete-scute+and+daughterless+gene+products+of+Drosophila.&amp;auth=&amp;atitle=Transcriptional+activation+by+heterodimers+of+the+achaete-scute+and+daughterless+gene+products+of+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+activation+by+heterodimers+of+the+achaete-scute+and+daughterless+gene+products+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+activation+by+heterodimers+of+the+achaete-scute+and+daughterless+gene+products+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0000514-RuizGmez1" id="pone.0000514-RuizGmez1"></a>Ruiz-Gómez M, Ghysen A (1993) The expression and role of a proneural gene, <em>achaete</em>, in the development of the larval nervous system of <em>Drosophila</em>. EMBO J  12: 1121–1130.  <ul class="find" data-citedArticleID="957596"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+expression+and+role+of+a+proneural+gene%2C+achaete%2C+in+the+development+of+the+larval+nervous+system+of+Drosophila.&amp;auth=&amp;atitle=The+expression+and+role+of+a+proneural+gene%2C+achaete%2C+in+the+development+of+the+larval+nervous+system+of+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+expression+and+role+of+a+proneural+gene%2C+achaete%2C+in+the+development+of+the+larval+nervous+system+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+expression+and+role+of+a+proneural+gene%2C+achaete%2C+in+the+development+of+the+larval+nervous+system+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000514-Kohler1" id="pone.0000514-Kohler1"></a>Kohler K, Thayer W, Le T, Sembhi A, Vasanthavada K, et al.  (2005) Characterization of a novel class II bHLH transcription factor from the black widow spider, <em>Latrodectus hesperus</em>, with silk-gland restricted patterns of expression. DNA Cell Biol.  24: 371–380.  <ul class="find" data-citedArticleID="957550" data-doi="10.1089/dna.2005.24.371"><li><a href="http://dx.doi.org/10.1089/dna.2005.24.371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+novel+class+II+bHLH+transcription+factor+from+the+black+widow+spider%2C+Latrodectus+hesperus%2C+with+silk-gland+restricted+patterns+of+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+novel+class+II+bHLH+transcription+factor+from+the+black+widow+spider%2C+Latrodectus+hesperus%2C+with+silk-gland+restricted+patterns+of+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000514-Leviten1" id="pone.0000514-Leviten1"></a>Leviten MW, Lai EC, Posakony JW (1997) The <em>Drosophila</em> gene <em>Bearded</em> encodes a novel small protein and shares 3′ UTR sequence motifs with multiple <em>Enhancer of split</em> Complex genes. Development  124: 4039–4051.  <ul class="find" data-citedArticleID="957564"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Drosophila+gene+Bearded+encodes+a+novel+small+protein+and+shares+3%26%23x2032%3B+UTR+sequence+motifs+with+multiple+Enhancer+of+split+Complex+genes.&amp;auth=&amp;atitle=The+Drosophila+gene+Bearded+encodes+a+novel+small+protein+and+shares+3%26%23x2032%3B+UTR+sequence+motifs+with+multiple+Enhancer+of+split+Complex+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Drosophila+gene+Bearded+encodes+a+novel+small+protein+and+shares+3%26%23x2032%3B+UTR+sequence+motifs+with+multiple+Enhancer+of+split+Complex+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Drosophila+gene+Bearded+encodes+a+novel+small+protein+and+shares+3%26%23x2032%3B+UTR+sequence+motifs+with+multiple+Enhancer+of+split+Complex+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000514-Coy1" id="pone.0000514-Coy1"></a>Coy JF, Sedlacek Z, Bächner D, Delius H, Poustka A (1999) A complex pattern of evolutionary conservation and alternative polyadenylation within the long 3′-untranslated region of the methyl-CpG-binding protein 2 gene (<em>MeCP2</em>) suggests a regulatory role in gene expression. Hum Mol Genet  8: 1253–1262.  <ul class="find" data-citedArticleID="957480" data-doi="10.1093/hmg/8.7.1253"><li><a href="http://dx.doi.org/10.1093/hmg/8.7.1253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complex+pattern+of+evolutionary+conservation+and+alternative+polyadenylation+within+the+long+3%26%23x2032%3B-untranslated+region+of+the+methyl-CpG-binding+protein+2+gene+%28MeCP2%29+suggests+a+regulatory+role+in+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complex+pattern+of+evolutionary+conservation+and+alternative+polyadenylation+within+the+long+3%26%23x2032%3B-untranslated+region+of+the+methyl-CpG-binding+protein+2+gene+%28MeCP2%29+suggests+a+regulatory+role+in+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000514-Xie1" id="pone.0000514-Xie1"></a>Xie X, Lu J, Kulbokas EJ, Golub TR, Mootha V, et al.  (2005) Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals. Nature  434: 338–345.  DOI: 10.1038/nature03441.  <ul class="find" data-citedArticleID="957648" data-doi="10.1038/nature03441"><li><a href="http://dx.doi.org/10.1038/nature03441" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+discovery+of+regulatory+motifs+in+human+promoters+and+3%26%23x2032%3B+UTRs+by+comparison+of+several+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+discovery+of+regulatory+motifs+in+human+promoters+and+3%26%23x2032%3B+UTRs+by+comparison+of+several+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0000514-Goodwin1" id="pone.0000514-Goodwin1"></a>Goodwin EB, Hofstra K, Hurney CA, Mango S, Kimble J (1997) A genetic pathway for regulation of <em>tra-2</em> translation. Development  124: 749–758.  <ul class="find" data-citedArticleID="957506"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+genetic+pathway+for+regulation+of+tra-2+translation.&amp;auth=&amp;atitle=A+genetic+pathway+for+regulation+of+tra-2+translation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+pathway+for+regulation+of+tra-2+translation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+pathway+for+regulation+of+tra-2+translation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000514-Wickens1" id="pone.0000514-Wickens1"></a>Wickens M, Bernstein DS, Kimble J, Parker R (2002) A PUF family portrait: 3′ UTR regulation as a way of life. Trends Genet  18: 150–157.  DOI: 10.1016/S0168-9525(01)02616-6.  <ul class="find" data-citedArticleID="957642" data-doi="10.1016/s0168-9525(01)02616-6"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02616-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+PUF+family+portrait%3A+3%26%23x2032%3B+UTR+regulation+as+a+way+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+PUF+family+portrait%3A+3%26%23x2032%3B+UTR+regulation+as+a+way+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000514-Delidakis1" id="pone.0000514-Delidakis1"></a>Delidakis C, Artavanis-Tsakonas S (1992) The Enhancer of split [<em>E(spl)</em>] locus of <em>Drosophila</em> encodes seven independent helix-loop-helix proteins. Proc Natl Acad USA  89: 8731–8735.  <ul class="find" data-citedArticleID="957486" data-doi="10.1073/pnas.89.18.8731"><li><a href="http://dx.doi.org/10.1073/pnas.89.18.8731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Enhancer+of+split+%5BE%28spl%29%5D+locus+of+Drosophila+encodes+seven+independent+helix-loop-helix+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Enhancer+of+split+%5BE%28spl%29%5D+locus+of+Drosophila+encodes+seven+independent+helix-loop-helix+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000514-Lai1" id="pone.0000514-Lai1"></a>Lai EC, Posakony JW (1997) The Bearded box, a novel 3′ UTR sequence motif, mediates negative post-transcriptional regulation of <em>Bearded</em> and <em>Enhancer of split</em> Complex gene expression. Development  124: 4847–4856.  <ul class="find" data-citedArticleID="957554"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Bearded+box%2C+a+novel+3%26%23x2032%3B+UTR+sequence+motif%2C+mediates+negative+post-transcriptional+regulation+of+Bearded+and+Enhancer+of+split+Complex+gene+expression.&amp;auth=&amp;atitle=The+Bearded+box%2C+a+novel+3%26%23x2032%3B+UTR+sequence+motif%2C+mediates+negative+post-transcriptional+regulation+of+Bearded+and+Enhancer+of+split+Complex+gene+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bearded+box%2C+a+novel+3%26%23x2032%3B+UTR+sequence+motif%2C+mediates+negative+post-transcriptional+regulation+of+Bearded+and+Enhancer+of+split+Complex+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bearded+box%2C+a+novel+3%26%23x2032%3B+UTR+sequence+motif%2C+mediates+negative+post-transcriptional+regulation+of+Bearded+and+Enhancer+of+split+Complex+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000514-Lai2" id="pone.0000514-Lai2"></a>Lai EC, Burks C, Posakony JW (1998) The K box, a conserved 3′ UTR sequence motif, negatively regulates accumulation of Enhancer of split Complex transcripts. Development  125: 4077–4088.  <ul class="find" data-citedArticleID="957556"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+K+box%2C+a+conserved+3%26%23x2032%3B+UTR+sequence+motif%2C+negatively+regulates+accumulation+of+Enhancer+of+split+Complex+transcripts.&amp;auth=&amp;atitle=The+K+box%2C+a+conserved+3%26%23x2032%3B+UTR+sequence+motif%2C+negatively+regulates+accumulation+of+Enhancer+of+split+Complex+transcripts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+K+box%2C+a+conserved+3%26%23x2032%3B+UTR+sequence+motif%2C+negatively+regulates+accumulation+of+Enhancer+of+split+Complex+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+K+box%2C+a+conserved+3%26%23x2032%3B+UTR+sequence+motif%2C+negatively+regulates+accumulation+of+Enhancer+of+split+Complex+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0000514-Arcidiacono1" id="pone.0000514-Arcidiacono1"></a>Arcidiacono S, Mello C, Kaplan D, Cheley S, Bayley H (1998) Purification and characterization of recombinant spider silk expressed in <em>Escherichia coli</em>. Appl Microbiol Biotechnol  49: 31–38.  DOI: 10.1007/S002530051133.  <ul class="find" data-citedArticleID="957446" data-doi="10.1007/s002530051133"><li><a href="http://dx.doi.org/10.1007/s002530051133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+characterization+of+recombinant+spider+silk+expressed+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+characterization+of+recombinant+spider+silk+expressed+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0000514-Huemmerich1" id="pone.0000514-Huemmerich1"></a>Huemmerich D, Helsen CW, Quedzuweit S, Oschmann J, Rudolph R, et al.  (2004) Primary structure elements of spider dragline silks and their contribution to protein solubility. Biochemistry  43: 13604–13612.  DOI: 10.1021/bi048983q.  <ul class="find" data-citedArticleID="957540" data-doi="10.1021/bi048983q"><li><a href="http://dx.doi.org/10.1021/bi048983q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primary+structure+elements+of+spider+dragline+silks+and+their+contribution+to+protein+solubility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primary+structure+elements+of+spider+dragline+silks+and+their+contribution+to+protein+solubility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0000514-Gregory1" id="pone.0000514-Gregory1"></a>Gregory TR, Shorthouse DP (2003) Genome sizes of spiders. J Hered  94: 285–290.  DOI: 10.1093/jhered/esg070.  <ul class="find" data-citedArticleID="957516" data-doi="10.1093/jhered/esg070"><li><a href="http://dx.doi.org/10.1093/jhered/esg070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sizes+of+spiders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sizes+of+spiders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0000514-Sambrook1" id="pone.0000514-Sambrook1"></a>Sambrook J, Russell DW (2001) Molecular Cloning: A Laboratory Manual, third edition. Cold Spring Harbor Laboratory. 999 p.  <ul class="find" data-citedArticleID="957602" data-doi="10.1002/abio.370050118"><li><a href="http://dx.doi.org/10.1002/abio.370050118" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+Cloning%3A+A+Laboratory+Manual%2C+third+edition.+Cold+Spring+Harbor+Laboratory.+999+p." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+Cloning%3A+A+Laboratory+Manual%2C+third+edition.+Cold+Spring+Harbor+Laboratory.+999+p.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0000514-Rozas1" id="pone.0000514-Rozas1"></a>Rozas J, Sánchez-DelBarrio JC, Messeguer X, Rozas R (2003) DnaSP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  DOI: 10.1093/bioinformatics/btg359.  <ul class="find" data-citedArticleID="957594" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">102.
              </span><a name="pone.0000514-Swofford1" id="pone.0000514-Swofford1"></a>Swofford DL (2002) PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods). Version 4. Sunderland, Massachusetts: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">103.
              </span><a name="pone.0000514-Posada1" id="pone.0000514-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="957584" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">104.
              </span><a name="pone.0000514-Hasegawa1" id="pone.0000514-Hasegawa1"></a>Hasegawa M, Kishino H, Yano T (1985) Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol  22: 160–174.  DOI: 10.1007/BF02101694.  <ul class="find" data-citedArticleID="957524" data-doi="10.1007/bf02101694"><li><a href="http://dx.doi.org/10.1007/bf02101694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+of+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+of+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">105.
              </span><a name="pone.0000514-Tamura1" id="pone.0000514-Tamura1"></a>Tamura K, Nei M (1993) Estimation of the number of nucleotide substitutions in the control region of mitochondrial DNA in humans and chimpanzees. Mol Biol Evol  10: 512–526.  <ul class="find" data-citedArticleID="957632"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees.&amp;auth=&amp;atitle=Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+the+number+of+nucleotide+substitutions+in+the+control+region+of+mitochondrial+DNA+in+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">106.
              </span><a name="pone.0000514-Ronquist1" id="pone.0000514-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models DOI: 10.1093/bioinformatics/btg180. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="957592" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models+DOI%3A+10.1093%2Fbioinformatics%2Fbtg180." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models+DOI%3A+10.1093%2Fbioinformatics%2Fbtg180.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">107.
              </span><a name="pone.0000514-Mayor1" id="pone.0000514-Mayor1"></a>Mayor C, Brudno M, Schwartz JR, Poliakov A, Rubin EM, et al.  (2000) VISTA: Visualizing global DNA sequence alignments of arbitrary length. Bioinformatics  16: 1046–1047.  &lt;<a href="http://genome.lbl.gov/vista/index.shtml">http://genome.lbl.gov/vista/index.shtml</a>&gt;.  <ul class="find" data-citedArticleID="957574" data-doi="10.1093/bioinformatics/16.11.1046"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.11.1046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VISTA%3A+Visualizing+global+DNA+sequence+alignments+of+arbitrary+length." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VISTA%3A+Visualizing+global+DNA+sequence+alignments+of+arbitrary+length.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">108.
              </span><a name="pone.0000514-Frazer1" id="pone.0000514-Frazer1"></a>Frazer KA, Pachter L, Poliakov A, Rubin EM, Dubchak I (2004) VISTA: computational tools for comparative genomics. Nucleic Acids Res  32: W273–W279.  <ul class="find" data-citedArticleID="957498" data-doi="10.1093/nar/gkh458"><li><a href="http://dx.doi.org/10.1093/nar/gkh458" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VISTA%3A+computational+tools+for+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VISTA%3A+computational+tools+for+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.XML" value="175324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.PDF" value="1123941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g001.PNG_L" value="2934859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g001.PNG_M" value="259405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g001.PNG_S" value="5191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g001.TIF" value="4112146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g001.PNG_I" value="102430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g002.PNG_L" value="4638303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g002.PNG_M" value="214917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g002.PNG_S" value="4627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g002.TIF" value="6538466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g002.PNG_I" value="98772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t001.PNG_L" value="55022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t001.PNG_M" value="52498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t001.PNG_S" value="2959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t001.TIF" value="378154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t001.PNG_I" value="23368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t002.PNG_L" value="34434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t002.PNG_M" value="33871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t002.PNG_S" value="2309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t002.TIF" value="188776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.t002.PNG_I" value="15339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g003.PNG_L" value="1172662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g003.PNG_M" value="80555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g003.PNG_S" value="3758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g003.TIF" value="3930388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g003.PNG_I" value="54747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g004.PNG_L" value="4997976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g004.PNG_M" value="199795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g004.PNG_S" value="7998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g004.TIF" value="7051062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g004.PNG_I" value="159075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g005.PNG_L" value="191085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g005.PNG_M" value="18182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g005.PNG_S" value="1876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g005.TIF" value="354554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g005.PNG_I" value="8916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g006.PNG_L" value="169877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g006.PNG_M" value="14558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g006.PNG_S" value="1819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g006.TIF" value="410498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000514.g006.PNG_I" value="8318"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000514&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000514" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000514&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000514&volume=&issue=&title=Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes&author_name=Nadia%20A.%20Ayoub%2C%20Jessica%20E.%20Garb%2C%20Robin%20M.%20Tinghitella%2C%20Matthew%20A.%20Collin%2C%20Cheryl%20Y.%20Hayashi&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000514" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514&amp;t=Blueprint%20for%20a%20High-Performance%20Biomaterial%3A%20Full-Length%20Spider%20Dragline%20Silk%20Genes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514&title=Blueprint%20for%20a%20High-Performance%20Biomaterial%3A%20Full-Length%20Spider%20Dragline%20Silk%20Genes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514&amp;title=Blueprint%20for%20a%20High-Performance%20Biomaterial%3A%20Full-Length%20Spider%20Dragline%20Silk%20Genes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000514&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Blueprint for a High-Performance Biomaterial: Full-Length Spider Dragline Silk Genes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000514';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Blueprint%20for%20a%20High-Performance%20Biomaterial%3A%20Full-Length%20Spider%20Dragline%20Silk%20Genes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000514" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000514" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="23866"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23866"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+sequencing%22" title="Search for articles in the subject area:'Protein sequencing'"><div class="flagText">Protein sequencing</div></a>
              <div data-categoryid="47503" data-articleid="23866"
                   data-categoryname="Protein sequencing"
                   class="flagImage" title="Flag 'Protein sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="23866"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="23866"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23866"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Silk%22" title="Search for articles in the subject area:'Silk'"><div class="flagText">Silk</div></a>
              <div data-categoryid="26631" data-articleid="23866"
                   data-categoryname="Silk"
                   class="flagImage" title="Flag 'Silk' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Spiders%22" title="Search for articles in the subject area:'Spiders'"><div class="flagText">Spiders</div></a>
              <div data-categoryid="43977" data-articleid="23866"
                   data-categoryname="Spiders"
                   class="flagImage" title="Flag 'Spiders' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2217'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6016'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8705&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
