

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000584"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000584" />

    <meta name="citation_title" content="A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients"/>
    <meta itemprop="name" content="A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients"/>

      <meta name="citation_author" content="Leonid I. Brodsky"/>
            <meta name="citation_author_institution" content="Institute of Evolution, University of Haifa, Haifa, Israel"/>
      <meta name="citation_author" content="Abdus S. Wahed"/>
            <meta name="citation_author_institution" content="Epidemiology Data Center, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Jia Li"/>
            <meta name="citation_author_institution" content="Epidemiology Data Center, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="John E. Tavis"/>
            <meta name="citation_author_institution" content="Molecular Microbiology and Immunology, St. Louis University, St Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Takuma Tsukahara"/>
            <meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana, United States of America"/>
      <meta name="citation_author" content="Milton W. Taylor"/>
            <meta name="citation_author_institution" content="Department of Biology, Indiana University, Bloomington, Indiana, United States of America"/>

    <meta name="citation_date" content="2007/7/4"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000584.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e584"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A multicenter, randomized trial of daily high-dose interferon-alfa 2b for the treatment of chronic hepatitis c: pretreatment stratification by viral burden and genotype,; citation_author=MW Fried; citation_author=M Shiffman; citation_author=RK Sterling; citation_author=J Weinstein; citation_author=J Crippin; citation_journal_title=Am J Gastroenterol; citation_volume=95; citation_number=1; citation_pages=3225-3229; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Peginterferon alfa-2a plus ribavirin for chronic hepatitis C virus infection,; citation_author=MW Fried; citation_author=ML Shiffman; citation_author=KR Reddy; citation_author=C Smith; citation_author=G Marinos; citation_journal_title=N Engl J Med; citation_volume=347; citation_number=2; citation_pages=975-982; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Peginterferon alfa-2b plus ribavirin compared with interferon alfa-2b plus ribavirin for initial treatment of chronic hepatitis C: a randomised trial,; citation_author=MP Manns; citation_author=JG McHutchison; citation_author=SC Gordon; citation_author=VK Rustgi; citation_author=M Shiffman; citation_journal_title=Lancet; citation_volume=358; citation_number=3; citation_pages=958-965; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose,; citation_author=SJ Hadziyannis; citation_author=H Sette; citation_author=TR Morgan; citation_author=V Balan; citation_author=M Diago; citation_journal_title=Ann Intern Med; citation_volume=140; citation_number=4; citation_pages=346-355; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hepatitis C viral dynamics in vivo and the antiviral efficacy of interferon-alpha therapy,; citation_author=AU Neumann; citation_author=NP Lam; citation_author=H Dahari; citation_author=DR Gretch; citation_author=TE Wiley; citation_journal_title=Science; citation_volume=282; citation_number=5; citation_pages=103-107; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Viral kinetics in patients with chronic hepatitis C treated with standard or peginterferon alpha2a,; citation_author=S Zeuzem; citation_author=E Herrmann; citation_author=JH Lee; citation_author=J Fricke; citation_author=AU Neumann; citation_journal_title=Gastroenterology; citation_volume=120; citation_number=6; citation_pages=1438-1447; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global effect of PEG-IFN-alpha and ribavirin on gene expression in PBMC in vitro,; citation_author=MW Taylor; citation_author=WM Grosse; citation_author=JE Schaley; citation_author=C Sanda; citation_author=X Wu; citation_journal_title=J Interferon Cytokine Res; citation_volume=24; citation_number=7; citation_pages=107-118; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of genes differentially regulated by interferon alpha, beta, or gamma using oligonucleotide arrays,; citation_author=SD Der; citation_author=A Zhou; citation_author=BR Williams; citation_author=RH Silverman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=8; citation_pages=15623-15628; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional classification of interferon-stimulated genes identified using microarrays,; citation_author=MJ de Veer; citation_author=M Holko; citation_author=M Frevel; citation_author=E Walker; citation_author=S Der; citation_journal_title=J Leukoc Biol; citation_volume=69; citation_number=9; citation_pages=912-920; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global transcriptional profiling combination of type I and type II demonstrates the interferon enhances antiviral and immune responses at clinically relevant doses,; citation_author=H Tan; citation_author=J Derrick; citation_author=J Hong; citation_author=C Sanda; citation_author=WM Grosse; citation_journal_title=J Interf Cytok Res; citation_volume=25; citation_number=10; citation_pages=632-649; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Changes in gene expression during pegylated interferon and ribavirin therapy of chronic hepatitis C virus distinguish responders from nonresponders to antiviral therapy,; citation_author=MW Taylor; citation_author=T Tsukahara; citation_author=L Brodsky; citation_author=J Schaley; citation_author=C Sanda; citation_journal_title=J Virol; citation_volume=81; citation_number=11; citation_pages=3391-3401; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Singular value decomposition for genome-wide expression data processing and modeling,; citation_author=O Alter; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=12; citation_pages=10101-10106; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Algorithms for Clustering Data,; citation_author=A Jail; citation_author=R Dubes; citation_number=13; citation_date=1988; citation_publisher=Prentice Hall; " />
      <meta name="citation_reference" content="citation_title=Similarity of phylogenetic trees as indicator of protein-protein interaction,; citation_author=F Pazos; citation_author=A Valencia; citation_journal_title=Protein engineering; citation_volume=14; citation_number=14; citation_pages=609-614; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Inferring protein interactions from phylogenetic distance matrices,; citation_author=J Gertz; citation_author=G Elfond; citation_author=A Shustrova; citation_author=M Weisinger; citation_author=M Pellegrini; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=15; citation_pages=2039-2045; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Statistical Methods.; citation_author=GW Snedecor; citation_author=WG Cochran; citation_number=16; citation_date=1989; citation_publisher=Iowa University Press; " />
      <meta name="citation_reference" content="citation_title=Multidimensional Scaling.; citation_author=T Cox; citation_author=M Cox; citation_number=17; citation_date=2003; citation_publisher=Chapman and Hall; " />
      <meta name="citation_reference" content="citation_title=Differential Gene Induction by Type I and Type II Interferons and Their Combination,; citation_author=C Sanda; citation_author=P Weitzel; citation_author=T Tsukahara; citation_author=J Schaley; citation_author=HJ Edenberg; citation_journal_title=J Interferon Cytokine Res; citation_volume=26; citation_number=18; citation_pages=462-472; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Interferon alfa regulated gene expression in patients initiating interferon treatment for chronic hepatitis C,; citation_author=X Ji; citation_author=R Cheung; citation_author=S Cooper; citation_author=Q Li; citation_author=HB Greenberg; citation_journal_title=Hepatology; citation_volume=37; citation_number=19; citation_pages=610-621; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Antiviral actions of interferons,; citation_author=CE Samuel; citation_journal_title=Clin Microbiol Rev; citation_volume=14; citation_number=20; citation_pages=778-809, table of contents; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global and distinct targets of IRF-5 and IRF-7 during innate response to viral infection,; citation_author=BJ Barnes; citation_author=J Richards; citation_author=M Mancl; citation_author=S Hanash; citation_author=L Beretta; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=21; citation_pages=45194-45207; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The growing family of interferon regulatory factors,; citation_author=H Nguyen; citation_author=J Hiscott; citation_author=PM Pitha; citation_journal_title=Cytokine Growth Factor Rev; citation_volume=8; citation_number=22; citation_pages=293-312; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=IRF-7 is the master regulator of type-I interferon-dependent immune responses,; citation_author=K Honda; citation_author=H Yanai; citation_author=H Negishi; citation_author=M Asagiri; citation_author=M Sato; citation_journal_title=Nature; citation_volume=434; citation_number=23; citation_pages=772-777; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=LPS-TLR4 signaling to IRF-3/7 and NF-kappaB involves the toll adapters TRAM and TRIF,; citation_author=KA Fitzgerald; citation_author=DC Rowe; citation_author=BJ Barnes; citation_author=DR Caffrey; citation_author=A Visintin; citation_journal_title=J Exp Med; citation_volume=198; citation_number=24; citation_pages=1043-1055; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Regulation of B7.1 costimulatory molecule is mediated by the IFN regulatory factor-7 through the activation of JNK in lipopolysaccharide-stimulated human monocytic cells,; citation_author=W Lim; citation_author=K Gee; citation_author=S Mishra; citation_author=A Kumar; citation_journal_title=J Immunol; citation_volume=175; citation_number=25; citation_pages=5690-5700; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The interferon regulatory factor, IRF5, is a central mediator of toll-like receptor 7 signaling,; citation_author=A Schoenemeyer; citation_author=BJ Barnes; citation_author=ME Mancl; citation_author=E Latz; citation_author=N Goutagny; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=26; citation_pages=17005-17012; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Immune evasion by hepatitis C virus NS3/4A protease-mediated cleavage of the Toll-like receptor 3 adaptor protein TRIF,; citation_author=K Li; citation_author=E Foy; citation_author=JC Ferreon; citation_author=M Nakamura; citation_author=AC Ferreon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=27; citation_pages=2992-2997; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulating intracellular antiviral defense and permissiveness to hepatitis C virus RNA replication through a cellular RNA helicase,; citation_author=R Sumpter; citation_author=YM Loo; citation_author=E Foy; citation_author=K Li; citation_author=M Yoneyama; citation_journal_title=RIG-I, J Virol; citation_volume=79; citation_number=28; citation_pages=2689-2699; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CARD games between virus and host get a new player,; citation_author=CL Johnson; citation_author=MJ Gale; citation_journal_title=Trends Immunol; citation_volume=27; citation_number=29; citation_pages=1-4; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis of the host response to hepatitis C virus infection,; citation_author=AI Su; citation_author=JP Pezacki; citation_author=L Wodicka; citation_author=AD Brideau; citation_author=L Supekova; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=30; citation_pages=15669-15674; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of ISG expression in chronic hepatitis C identifies viperin as a potential antiviral effector,; citation_author=KJ Helbig; citation_author=DT Lau; citation_author=L Semendric; citation_author=HA Harley; citation_author=MR Beard; citation_journal_title=Hepatology; citation_volume=42; citation_number=31; citation_pages=702-710; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phospholipid scramblase 1 potentiates the antiviral activity of interferon,; citation_author=B Dong; citation_author=Q Zhou; citation_author=J Zhao; citation_author=A Zhou; citation_author=RN Harty; citation_journal_title=J Virol; citation_volume=78; citation_number=32; citation_pages=8983-8993; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=G1P3, an interferon inducible gene 6–16, is expressed in gastric cancers and inhibits mitochondrial-mediated apoptosis in gastric cancer cell line TMK-1 cell,; citation_author=E Tahara; citation_author=H Tahara; citation_author=M Kanno; citation_author=K Naka; citation_author=Y Takeda; citation_journal_title=Cancer Immunol Immunother; citation_volume=54; citation_number=33; citation_pages=729-740; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=iPABP, an inducible poly(A)-binding protein detected in activated human T cells,; citation_author=H Yang; citation_author=CS Duckett; citation_author=T Lindsten; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=34; citation_pages=6770-6776; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Interaction of anti-proliferative protein Tob with poly(A)-binding protein and inducible poly(A)-binding protein: implication of Tob in translational control,; citation_author=K Okochi; citation_author=T Suzuki; citation_author=J Inoue; citation_author=S Matsuda; citation_author=T Yamamoto; citation_journal_title=Genes Cells; citation_volume=10; citation_number=35; citation_pages=151-163; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Elevated expression of ISG15 in tumor cells interferes with the ubiquitin/26S proteasome pathway,; citation_author=SD Desai; citation_author=AL Haas; citation_author=LM Wood; citation_author=YC Tsai; citation_author=S Pestka; citation_journal_title=Cancer research; citation_volume=66; citation_number=36; citation_pages=921-928; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of human sialoadhesin, a sialic acid binding receptor expressed by resident and inflammatory macrophage populations,; citation_author=A Hartnell; citation_author=J Steel; citation_author=H Turley; citation_author=M Jones; citation_author=DG Jackson; citation_journal_title=Blood; citation_volume=97; citation_number=37; citation_pages=288-296; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cutting edge: CD43 functions as a T cell counterreceptor for the macrophage adhesion receptor sialoadhesin (Siglec-1),; citation_author=TK van den Berg; citation_author=D Nath; citation_author=HJ Ziltener; citation_author=D Vestweber; citation_author=M Fukuda; citation_journal_title=J Immunol; citation_volume=166; citation_number=38; citation_pages=3637-3640; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sialoadhesin-deficient mice exhibit subtle changes in B- and T-cell populations and reduced immunoglobulin M levels,; citation_author=C Oetke; citation_author=MC Vinson; citation_author=C Jones; citation_author=PR Crocker; citation_journal_title=Mol Cell Biol; citation_volume=26; citation_number=39; citation_pages=1549-1557; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients"/>
    <meta name="twitter:description" content="
         
            Background

            Treating hepatitis C with interferon/ribavirin results in a varied response in terms of decrease in viral titer and ultimate outcome. Marked responders have a sharp decline in viral titer within a few days of treatment initiation, whereas in other patients there is no effect on the virus (poor responders). Previous studies have shown that combination therapy modifies expression of hundreds of genes in vitro and in vivo. However, identifying which, if any, of these genes have a role in viral clearance remains challenging.
         
         
            Aims

            The goal of this paper is to link viral levels with gene expression and thereby identify genes that may be responsible for early decrease in viral titer.
         
         
            Methods

            Microarrays were performed on RNA isolated from PBMC of patients undergoing interferon/ribavirin therapy. Samples were collected at pre-treatment (day 0), and 1, 2, 7, 14 and 28 days after initiating treatment. A novel method was applied to identify genes that are linked to a decrease in viral titer during interferon/ribavirin treatment. The method uses the relationship between inter-patient gene expression based proximities and inter-patient viral titer based proximities to define the association between microarray gene expression measurements of each gene and viral-titer measurements.
         
         
            Results

            We detected 36 unique genes whose expressions provide a clustering of patients that resembles viral titer based clustering of patients. These genes include IRF7, MX1, OASL and OAS2, viperin and many ISG's of unknown function.
         
         
            Conclusion

            The genes identified by this method appear to play a major role in the reduction of hepatitis C virus during the early phase of treatment. The method has broad utility and can be used to analyze response to any group of factors influencing biological outcome such as antiviral drugs or anti-cancer agents where microarray data are available.
         
      "/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000584.g006"/>

  <meta property="og:title" content="A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5271'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6138'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1250&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000584">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000584" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000584&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leonid I. Brodsky, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Evolution, University of Haifa, Haifa, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Abdus S. Wahed, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Epidemiology Data Center, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jia Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Epidemiology Data Center, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John E. Tavis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular Microbiology and Immunology, St. Louis University, St Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takuma Tsukahara, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Indiana University, Bloomington, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Milton W. Taylor
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:taylor@indiana.edu">taylor@indiana.edu</a></p>

                <p>Affiliation:
                  Department of Biology, Indiana University, Bloomington, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 04, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000584</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000584-g001" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000584-g002" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000584-g003" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000584-g004" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000584-t001" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000584-g005" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000584-g006" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000584">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000584" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Treating hepatitis C with interferon/ribavirin results in a varied response in terms of decrease in viral titer and ultimate outcome. Marked responders have a sharp decline in viral titer within a few days of treatment initiation, whereas in other patients there is no effect on the virus (poor responders). Previous studies have shown that combination therapy modifies expression of hundreds of genes in vitro and in vivo. However, identifying which, if any, of these genes have a role in viral clearance remains challenging.</p>


<h3>Aims</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The goal of this paper is to link viral levels with gene expression and thereby identify genes that may be responsible for early decrease in viral titer.</p>


<h3>Methods</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Microarrays were performed on RNA isolated from PBMC of patients undergoing interferon/ribavirin therapy. Samples were collected at pre-treatment (day 0), and 1, 2, 7, 14 and 28 days after initiating treatment. A novel method was applied to identify genes that are linked to a decrease in viral titer during interferon/ribavirin treatment. The method uses the relationship between inter-patient gene expression based proximities and inter-patient viral titer based proximities to define the association between microarray gene expression measurements of each gene and viral-titer measurements.</p>


<h3>Results</h3>
<a id="article1.front1.article-meta1.abstract1.sec4.p1" name="article1.front1.article-meta1.abstract1.sec4.p1"></a><p>We detected 36 unique genes whose expressions provide a clustering of patients that resembles viral titer based clustering of patients. These genes include IRF7, MX1, OASL and OAS2, viperin and many ISG's of unknown function.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec5.p1" name="article1.front1.article-meta1.abstract1.sec5.p1"></a><p>The genes identified by this method appear to play a major role in the reduction of hepatitis C virus during the early phase of treatment. The method has broad utility and can be used to analyze response to any group of factors influencing biological outcome such as antiviral drugs or anti-cancer agents where microarray data are available.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Brodsky LI, Wahed AS, Li J, Tavis JE, Tsukahara T, et al.  (2007) A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients. PLoS ONE 2(7):
          e584.
            doi:10.1371/journal.pone.0000584</p><p><strong>Academic Editor: </strong>Sebastian Fugmann, National Institute on Aging, United States of America</p><p><strong>Received:</strong> May 8, 2007; <strong>Accepted:</strong> June 3, 2007; <strong>Published:</strong> July 4, 2007</p><p><strong>Copyright:</strong> © 2007 Brodsky et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded as a cooperative agreement by the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK) with co-support from the Intramural Research Program of the National Cancer Institute (NCI) with further support under a Cooperative Research and Development Agreement (CRADA) with Roche Laboratories, Grant numbers: U01 DK60329, U01 DK60340, U01 DK60324, U01 DK60344, U01 DK60327, U01 DK60335, U01 DK60352, U01 DK60342, U01 DK60345, U01 DK60309, U01 DK60346, U01 DK60349, U01 DK60341. Other support: National Center for Research Resources (NCRR) General Clinical Research Centers Program grants: M01 RR00645 (New York Presbyterian), M02 RR000079 (University of California, San Francisco), M01 RR16500 (University of Maryland), M01 RR000042 (University of Michigan), M01 RR00046 (University of North Carolina).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Treating with peginterferon/ribavirin combination therapy patients who have chronic hepatitis C virus (HCV) infection results in a varied response in terms of outcome and decrease in viral titer <a href="#pone.0000584-Fried1">[1]</a>–<a href="#pone.0000584-Hadziyannis1">[4]</a>. For patients who respond well there is a sharp decrease in viral titer within 24–48 hours after treatment initiation whereas in other patients there is little or no effect on the viral titer and only temporary, or no, clearance of the virus over a long period <a href="#pone.0000584-Neumann1">[5]</a>, <a href="#pone.0000584-Zeuzem1">[6]</a>. Previous in vitro studies have shown that combination interferon treatment induces or decreases expression of hundreds of genes <a href="#pone.0000584-Taylor1">[7]</a>–<a href="#pone.0000584-Tan1">[10]</a>. One of the major problems, however, is to identify which of these genes are linked to viral clearance in vivo.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In this paper we report a novel mathematical method to explore the association between decrease in viral titer and changes in gene expression in hepatitis C patients following combination treatment with pegylated interferon and ribavirin. The viral clearance time course profile will not necessarily directly correlate with the gene expression time course profile even if the gene is an active participant of the interferon treatment response because the decrease of the viral levels depends on the interplay of many genes and gene products. Therefore, an indirect approach was used in which the relationship between gene expression across days and viral decrease was examined using inter-patient distances (proximity) according to both characteristics.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Using this approach we selected thirty seven gene probes that were linked with the anti-HCV response during the first 28 days of treatment. A visual demonstration of the association of detected genes with the viral decrease is demonstrated by a comparison of patient clusterings. Indeed, the inter-patient proximities according to the pattern of decrease in virus titer provide an unsupervised clustering of patients based on changes in viral levels. Similarly, the inter-patient proximities according to expressions of the specified genes across time provide another unsupervised clustering of patients. A visual inspection of viral-titer based and selected genes expression based clusterings of patients indicates their close relationship. Since the unsupervised clustering of patients according to the pattern of viral clearance is in good correspondence with an a priori biological categorization of patients into marked, slow and poor response at day 28 <a href="#pone.0000584-Taylor2">[11]</a>,the selected subset of genes can be considered as genes that are involved specifically in treatment response. Such genes products may have anti-viral activity or be related to the immune response against virus-infected cells.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Patient population</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The Virahep-C study included a cohort of 401 participants who provided written consent at 8 U.S. clinical centers between September 9, 2002 and January 7, 2004. Per protocol, all participants were treated for up to 48 weeks with pegylated interferon alfa-2a (PEGASYS, Roche Inc. Nutley, NJ) at 180 mcg weekly by self-administered subcutaneous injection and ribavirin (COPEGUS, Roche, Inc. Nutley N.J.) by mouth at 1000 mg/day for those who weighed less than 75 kg or 1200 mg/day for those who weighed at least 75 kg. Treatment was discontinued at week 24 in participants who had a detectable serum HCV RNA virus in duplicate qualitative assays using the Roche Cobas Amplicor HCV Test, v2.0 (sensitivity 50 IU/ml). The primary endpoint of the study was the sustained virologic response, defined as an undetectable serum HCV RNA at week 72 (at least 24 weeks after completion of treatment). The clinical trial was #-NCT00038974.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Fifty two patients for whom gene expression and viral level data were available at all time points were selected for this analysis. Based on the log-decline in serum HCV RNA on day 28 of treatment relative to day 0, as measured by the quantitative Roche Cobas Monitor HCV Test, v2.0 (sensitivity 600 IU/ml) these patients were divided into three groups <a href="#pone.0000584-Taylor2">[11]</a> Patients were identified as poor responders if they had less than a 1.4 log<sub>10</sub> IU/ml decrease in serum HCV RNA between day 0 and day 28, as marked responders if they experienced more than a 3.5 log<sub>10</sub> IU/ml decrease or if their viral titers dropped to undetectable by day 28, and as intermediate responders if the log<sub>10</sub> drop in HCV RNA was between 1.4 and 3.5 IU/ml. For the purpose of unsupervised clustering of viral titers, patients with undetected viral levels by the quantitative assay were assigned a viral level of 599 IU/ml and those whose viral levels were undetected by the qualitative assay were assigned a viral level of 49 IU/ml</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>This study met all necessary approvals of Institutional Review Boards of each institution participating in the Virahep-C consortium (Beth Isreal Deaconess Medical Center, New York Presbyterian Medical Center, Rush University, University of California at San Francisco, University of Maryland, University of Miami, University of Michigan, and University of North Carolina).</p>


<h4>Cell preparation and RNA extraction</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Peripheral Blood Mononuclear Cells (PBMC) were collected in sodium heparin-CPT tubes at day 0, 1, 2, 7, 14 and 28. Samples were shipped overnight from each clinical center to a central repository by express courier at 4°C. Whole blood was diluted with an equal volume (8 ml) of phosphate buffered saline, carefully layered over a 10 ml Ficoll-Hypaque gradient (Amersham/Pharmacia) and centrifuged at 800 rpm for 20 minutes at room temperature. The buffy coat layer was transferred to a 15 ml RNAse-free tube and further diluted with PBS. Tubes were centrifuged at 100-× g for 15 minutes at room temperature. The supernatants were discarded and the PBMC were retained.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The isolation of RNA, quality control, the labeling and hybridization on to the micro-arrays have been previously described <a href="#pone.0000584-Taylor1">[7]</a>, <a href="#pone.0000584-Tan1">[10]</a>.</p>


<h4>Array Analysis and Data Processing:</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The microarrays were scanned using a dedicated Model 3000 scanner controlled by Affymetrix Microarray Suite 5 software (MAS5). The average intensity on each array was normalized by global scaling to a target intensity of 1000. Data were exported from MAS5 into a custom-designed database (MicroArray Data Portal) in the Center for Medical Genomics (IUPUI, Indianapolis). The data from the microarrays has been deposited with NCBI, GEODATA# GSF7123</p>


<h4>Visualization of patient grouping</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Visualizing the patient data was simplified through Principal Component Analysis (PCA) <a href="#pone.0000584-Alter1">[12]</a> which reduces the dimensionality of the data into a small number of independent components. An alternative method to present data grouping is hierarchical clustering. We used the hierarchical clustering (UPGMA) <a href="#pone.0000584-Jail1">[13]</a> algorithm for the clustering.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Both PCA and the clustering of patient's groupings according to either viral titer or gene expression data were performed to visualize the correspondence between predefined grouping of patients (marked, intermediate, and poor, highlighted by color) and either gene expression or viral titer based grouping of patients.</p>


<h4>Viral titer based proximity of patients and “candidate” genes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Detection of genes that provided clustering similar to viral titer based clustering of patients was performed by a mathematical method similar to the mirror tree method for inferring protein interactions from phylogenetic distance matrices <a href="#pone.0000584-Pazos1">[14]</a>, <a href="#pone.0000584-Gertz1">[15]</a>. This method was applied to gene expression and linked the gene-specific variability of patients with their viral titer variability. The link is a correlation between the matrix of inter-patient distances that is based on viral titer and the individual gene expression based matrix. If a gene expression based matrix correlates with a viral titer matrix, then it can be concluded that there is a link between expression of that specific gene and the viral levels in patients. Such a link considers both the direct correlation between the expression profile of a gene across patients and the viral titer pattern of patients, and the indirect involvement of the gene in the viral level change process. A detailed description of the method follows.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The distance between patients' viral titer changes from day 0 was used to estimate the inter-patient proximity with respect to viral change. Similarly, the gene expression based inter-patient proximity was measured for each gene. The correlations between each gene based matrix proximity and viral titer based matrix-proximity were estimated and genes with the highest correlations were selected.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>As metrics of inter-patient proximity two measures were used: Euclidean distance and the coefficient of covariation. The larger the coefficient of covariation, the closer the patients are whereas the smaller the Euclidean distance the closer the patients are. Thus, the inter-patient distances based on viral titer time-course measurements created a grouping of patients that reflected the response of patients to antiviral treatment. On the other hand, the inter-patient proximities based on gene expression measurements across days (gene expression profile) for a specific gene reflected the variability of patients either according to response of this gene to treatment, or according to genetic heterogeneity of patients, or both.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>The Euclidean distance between the natural logarithms (ln) of changes in viral levels from baseline (day 0) for patient <em>i</em> and patient <em>j</em> was defined as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>v<sub>id</sub></em> is the viral titer value for <em>i</em><sup>th</sup> patient on day <em>d</em>. Denote <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e002&amp;representation=PNG" class="inline-graphic"></span> as the vector of Euclidean distances between each pair of patients with dimension <em>k</em>(<em>k</em>−1)/2, where <em>k</em> is the number of patients.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Similarly, the inter-patient Euclidean proximities with respect to the natural logarithm (ln) of the gene expression for the <em>r</em><sup>th</sup> gene on day <em>d</em> were estimated by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e003&amp;representation=PNG" class="inline-graphic"></span><br>where <em>g<sup>r</sup><sub>id</sub></em> is the expression of r<sup>th</sup> gene for <em>i</em><sup>th</sup> patient on day <em>d</em>. Let <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e004&amp;representation=PNG" class="inline-graphic"></span> denote the vector of inter-patient proximity according to expression of r<sup>th</sup> gene at day d. Corresponding to <em>V<sub>ij</sub></em>, define <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e005&amp;representation=PNG" class="inline-graphic"></span> to be the inter-patient Euclidean proximities with respect to the gene expression for the <em>r</em><sup>th</sup> gene. Denote vectors <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e006&amp;representation=PNG" class="inline-graphic"></span> with dimension <em>k</em>(<em>k</em>−1)/2.</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>The inter-patient proximities according to expression of <span class="underline">all genes</span> was then estimated by the vector of the same <em>k</em>(<em>k</em>−1)/2 dimension <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e007&amp;representation=PNG" class="inline-graphic"></span>, where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e008&amp;representation=PNG" class="inline-graphic"></span>, where <em>N</em> is the number of genes on the array. Note that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e009&amp;representation=PNG" class="inline-graphic"></span></p>
<a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7"></a><p>We will refer to the vectors <em>G̅</em><em><sup>rd</sup></em> and <em>G̅</em><em><sup>r</sup></em> as GDeu and Geu vectors respectively. Vector <em>G̅</em> will be referred to as the main vector (MVeu) of the method and has equal biological importance to the virus titer related vector <em>V̅</em> (VTeu) . Indeed, MVeu is the vector of distances between each pair of patients across all genes and all days. These distances reflect differences between patients both due to individual variability and due to the response of patients to the combination treatment. Thus, inter-patient distances across all genes and all days are the superimposition of patient's individual and response differences. The norm of MV is a cumulative measure of how patients are divergent according to their gene expression. Genes that are not co-regulated (the majority of them) have random distances with other genes thus making a minimal impact in the length of the MV. Indeed, gene vectors G with random signals are distributed symmetrically in <em>k</em>(<em>k</em>−1)/2 space. Under summation they will compensate each other, reducing the norm of the MV vector. Therefore, a large norm for MV is defined by a relatively small number of co-regulated groups of genes, and these groups are related to the biological divergence of patients (including the divergence of their response). From a biological point of view it is important to understand whether the input of genes in MV is correlated with the virus titer vector VT in <em>k</em>(<em>k</em>−1)/2 space. If they are, then the variability of patients according to gene expression is closely related to their treatment response; if not, then gene expression variability of patients is defined by their other characteristics.</p>
<a id="article1.body1.sec2.sec5.p8" name="article1.body1.sec2.sec5.p8"></a><p>Similarly to Euclidean distance based vectors <em>G̅</em><em><sup>rd</sup></em>, <em>G̅</em><em><sup>r</sup></em>, <em>G̅</em> and <em>V̅</em> of the <em>k</em>(<em>k</em>−1)/2 space, the covariance based vectors <em>G̅</em><em><sup>rd</sup></em><sup>,cov</sup>, <em>G̅</em><em><sup>r</sup></em><sup>,cov</sup>, MVcov (<em>G̅</em><sup>cov</sup>) and VTcov (<em>V̅</em><sup>cov</sup>) of the same space <em>k</em>(<em>k</em>−1)/2 were defined as follows. The element of VTcov vector is thus <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e010&amp;representation=PNG" class="inline-graphic"></span>, where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e011&amp;representation=PNG" class="inline-graphic"></span>, and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e012&amp;representation=PNG" class="inline-graphic"></span> is the average of <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e013&amp;representation=PNG" class="inline-graphic"></span> across days 1, 2, 7, 14, and 28. Similarly, for the <em>r</em><sup>th</sup> gene, <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e014&amp;representation=PNG" class="inline-graphic"></span>, where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.e015&amp;representation=PNG" class="inline-graphic"></span> and <em>β̅</em><em><sub>i</sub></em> is the average of <em>β<sup>r</sup><sub>id</sub></em> across all genes and all days.</p>
<a id="article1.body1.sec2.sec5.p9" name="article1.body1.sec2.sec5.p9"></a><p>The associations between vectors G, GD and vectors MV and VT both for Euclidian and Covariance metrics were estimated by the Pearson correlation coefficients. We assume that (i) the expression of the gene log-signal in a patient is normally distributed, (ii) expression values of a gene across patients are independent, and (iii) for all genes there are no dependences of between-patient distances according to gene expression and between-patient distances according to viral titer values. The method detects linkage of genes with viral titer as statistically significant deviations against the point (iii), i.e., differences in correlation coefficients from 0. The statistical significance of correlation coefficients may be calculated using Fisher's Z transformation<a href="#pone.0000584-Snedecor1">[16]</a> for correlation coefficients which is distributed normally with mean zero and variance 1/[{<em>k</em>(<em>k</em>−1)/2}−3]. We can alternatively estimate the significance of <em>R</em><sup>2</sup> (the square of the correlation coefficient) assuming a beta distribution with 0.5 and <em>k</em>(<em>k</em>−1)/2 degrees of freedom. If we take into consideration that there is a weak mutual dependence between each pair of patient distances with the same patient is in both pairs, the significance estimation according to beta distribution with 0.5 and <em>k</em>(<em>k</em>−1)/4 degrees of freedom will compensate this weak mutual dependence of inter-patient distances.</p>


<h4>MV and VT relationship and selection of VT-linked genes</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In order to take into account both Euclidean and Covariance measures of proximity in the selection of VT-linked genes, the analysis was done as follows. A multidimensional scaling<a href="#pone.0000584-Cox1">[17]</a> was applied to reduce the dimensionality of the <em>k</em>(<em>k</em>−1)/2 space. Namely, the mutual proximities of gene (G) and gene-day (GD) vectors estimated through their correlations with MV and VT vectors (vectors are prepared according to Euclidean and covariance measures) were represented by two-dimensional Euclidean distances in the principal component analysis (PCA) <a href="#pone.0000584-Alter1">[12]</a> space. Two principal components (PC1 and PC2) were detected according to 8 initial variables, these variables being correlation coefficients for vector pairs (Gcov-MVcov), (GDcov-MVcov), (Gcov-VTcov), (GDcov-VTcov), (Geu-MVeu), (GDeu-MVeu), (Geu-VTeu), (GDeu-VTeu) across all gene-days. The first principal component appears to be defined by correlations of G and GD vectors with MVcov and MVeu vectors, and the second one by correlations of G and GD vectors with VTcov and VTeu vectors. This implies that gene (G and GD) correlations with two MV vectors are independent from gene correlations with two VT vectors, and therefore the general variability of patients according to gene expression is not related to the response to the combination therapy of patients.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The deviation of some GD-vectors from the core of their distribution in the multi-scaling space in the direction of VT vectors (i.e. at PC2 direction) indicates a link of these gene-days (and genes) with viral titers. As the initial criterion for gene-day detection the PC2&gt;threshold was used. The statistical significance of such gene detection was checked by Fisher's Z-test and via beta distribution for correlation coefficients of G(GD) vectors regarding VTcov vector. Both tests used an adjusted number of degrees of freedom to compensate for a weak inter-dependence of vector coordinates.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The more accurate check of the gene detection significance and the estimation of False Discovery Rate (FDR) were performed through permutations. In the first step the distribution of “random” gene-days in the multiscaling space was prepared through permutation of gene-days log-signals over 52 patients of the study. After that correlations of permutated gene-day based inter-patient matrices with VT and MV vectors in k(k−1)/2 space were calculated, and the previously found transformation of the initial 8-dimension space into the multiscaling space was applied to these correlations.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>The FDR estimation for the detected set of real genes was done as follows. The sub-space that the gene set is occupied in the multiscaling space was defined as the sphere around the central position of this set. The ratio of the number of permutated gene-days inside the sphere (normalized to the size of real gene set) to the number of the real gene-days inside the sphere is the False Discovery Rate estimation for a sphere of the given radius.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>As a first step, the correlation between patient classification based on decrease in virus titer by day 28 and the unsupervised viral titer based clustering of patients was tested. Namely, natural log-transformed viral titers of the 52 patients were normalized using the baseline viral level (i.e. day 0) [i.e. ln(v<sub>i</sub> )−ln(v<sub>0</sub>) ], where v<sub>i</sub> is the viral titer value at day i. The clustering of patients was done by the hierarchical UPGMA method <a href="#pone.0000584-Jail1">[13]</a> using a Euclidean measure of similarity. The branches of the clustering tree (patients) were labeled by type of response according to the initial categorization of the early response (marked, intermediate, and poor) <a href="#pone.0000584-Taylor2">[11]</a> (<a href="#pone-0000584-g001">Fig 1</a>). This hypothesis-free clustering based on inter-patient Euclidean distances according to viral titer demonstrated that patients could be placed into three groups corresponding to the response classification of patients. Indeed, the biggest and most compact cluster consists of poor response patients (the left branch of the tree). The right branch of the cluster contains just marked response patients. The middle branch of the cluster contains a mix of all three classes of patients. If only poor and marked response patients are considered, then there is no misclassification for extreme right and left clusters of the hierarchical classification.</p>
<div class="figure" id="pone-0000584-g001"><div class="img"><a name="pone-0000584-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The clustering of 52 patients according to viral titer measurements across days.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>All viral titer values of each patient are normalized by the same patient day 0 viral titer measurement. The Euclidean measure based hierarchical (UPGM) clustering indicates a good separation of the early response Marked (pink) and Poor response (green) patients. The Slow (yellow) patients are mostly concentrated at the intermediate branch of the clustering tree.</p>
<span>doi:10.1371/journal.pone.0000584.g001</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Another compact visualization of patient grouping according to the same baseline-normalized viral titer data was performed using PCA. The first two principal components covered 93% of total data variability. The distribution of patients using the classification given above demonstrated clear separation of poor response patients from marked response patients (<a href="#pone-0000584-g002">Fig 2, see separation line</a>).</p>
<div class="figure" id="pone-0000584-g002"><div class="img"><a name="pone-0000584-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The PCA visualization of inter-patient proximities according to day 0 normalized per day viral titer values.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>First two Principal Components cover 93% of data variability and thus gives the good presentation of the patient distribution in the 5 dimensional space. There is the clear divergence of Marked (dark blue points) and Poor (green points) response patients. The separation line (dotted blue line) indicates the virtual border between these two populations. Slow patients (pink points) are mostly concentrated along the borderline.</p>
<span>doi:10.1371/journal.pone.0000584.g002</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In the second step, the viral titer linked genes were determined. We defined VT-linked genes as the ones that produced a clustering of patients similar to viral titer based clustering of patients. The procedure for their detection was as follows: Gene expression for each day was normalized with regard to day 0 expression, as was done for viral titer. The determination of genes was based on the hypothesis that the between-patient proximities according to gene and gene-day expression pattern of specific genes are correlated with the viral titer based inter-patient proximities. Thus we looked for genes (gene-days) that “associated” with the viral titer. The selection was done through multidimensional scaling PCA representation for the space of correlations between inter-patient distance matrices (<a href="#pone-0000584-g003">Fig 3</a>). The deviations of some GD-vectors from their uniform distribution in the PCA space in the direction of VT vectors (<a href="#pone-0000584-g004">Fig.4, pink dots</a>) indicated a link of these gene-days (and genes) with viral titer behavior of patients. Thus, gene-days with the second principal component (PC2) value more than 5 were selected. Genes having less than 3 gene-days with PC2 greater than 5 were filtered out.</p>
<div class="figure" id="pone-0000584-g003"><div class="img"><a name="pone-0000584-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The schematic representation of the algorithm's main idea.</span></strong></p><a id="article1.body1.sec3.fig3.caption1.p1" name="article1.body1.sec3.fig3.caption1.p1"></a><p>Any matrix of inter-patient distances for k patients could be presented as a point (vector) in k (k−1)/2 dimensions space. There are two main vectors in this space; the inter-patient matrix according to viral titer (blue dotted vector) and inter-patient matrix according to expression of all genes (black vector). The expression of every gene is presented as a point (vector) in the same space: the inter-patient matrix according to the expression values of this gene. Some genes could be close to viral-titer vector VT or/and to all gene expression vector MV. Inter-patient matrices according to expression of individual genes (G) at specific days (GD) are dots of the figure. Points of high correlation with VT vector (pink dots) are VT-linked genes. MV linked genes (green dots) define individual variability of patients according to their gene expression. The genes not linked with the two main vectors genes are in red.</p>
<span>doi:10.1371/journal.pone.0000584.g003</span></div><div class="figure" id="pone-0000584-g004"><div class="img"><a name="pone-0000584-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The two-dimensional PCA representation (cover 63% of data variability) of inter-vector correlations in k (k−1)/2 space – the space of distances between inter-patient matrices.</span></strong></p><a id="article1.body1.sec3.fig4.caption1.p1" name="article1.body1.sec3.fig4.caption1.p1"></a><p>One may interpret this figure as the view from above on <a href="#pone-0000584-g003">Fig. 3</a> schema. Follow to multidimensional-scaling method the proximities between gene based inter-patient matrices as estimated through their correlations with MV and VT matrices were represented by inter-gene (gene-days) Euclidian distances in the PCA two-dimensional space. Positions of VTeu, MVeu (Euclidean metrics) and VTcov, MVcov (covariance metrics) matrices in the PCA space are seen on the insert. VT linked genes are denoted by pink dots. MV-linked genes are in the frame on the figure right.</p>
<span>doi:10.1371/journal.pone.0000584.g004</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The correlation coefficients of all detected genes with VTcov (as representing VT group of inter-patient matrices) are more that 0.24 (<a href="#pone.0000584.s001">Table S1</a>) which corresponds to a p-value of less than 1.0E-08 for 1326 (52*51/2) degrees of freedom.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The permutation analysis (see M&amp;M) was based on 1000 permutations of gene-day signals across all 22000 genes at 5 days. The minimal radius of the sphere that covers 80% of gene-days of the selected 37 genes in the 4-mer multi-scaling space is 2.6. This radius corresponds to FDR 1% of the detection. The four dimensional multi-scaling space was taken because such a number of PCA components cover more than 80% of gene expression data variability.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Most of detected genes (<a href="#pone-0000584-t001">Table 1</a>) have been identified previously as interferon induced in PBMC or A549 cells in vitro <a href="#pone.0000584-Taylor1">[7]</a>, <a href="#pone.0000584-Sanda1">[18]</a> supporting the conclusion that these are meaningful genes although they were identified in an unsupervised manner. <a href="#pone.0000584.s001">Table S1</a> lists the top 37 probe sets with details of each probe set and with the day-by-day correlations between gene expression and viral titer.</p>
<div class="figure" id="pone-0000584-t001"><div class="img"><a name="pone-0000584-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Genes hypothesized to be important in the anti-hepatitis C response.</span></strong></p><span>doi:10.1371/journal.pone.0000584.t001</span></div><a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Visualization of the sources of variation of patients according to gene expressions of genes- classifiers was simplified through PCA, which reduced the dimensionality of the data into a relatively small number of components. PCA presentation is illustrated in <a href="#pone-0000584-g005">Figure 5</a>. The baseline-normalized expression of these 37 genes (probe sets) appears to be a good discriminator of Marked vs. Poor. Indeed, the PCA presentation of the distribution for 52 patients according to expression profiles across all days of these 37 probe sets demonstrates the obvious shift of poor response patients to negative values of the component 1 (<a href="#pone-0000584-g005">Fig 5</a>). The PCA presentation of the data distribution is relevant because the first two components of PCA analysis cover a major portion (63%) of total data variability. One can see the same basic separation of marked vs. poor response patients in clustering in <a href="#pone-0000584-g006">Fig. 6</a>.</p>
<div class="figure" id="pone-0000584-g005"><div class="img"><a name="pone-0000584-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The PCA presentation of the divergence between Rapid and poor response patients according to day 0 normalized expression of 35 VT-linked genes across other days.</span></strong></p><a id="article1.body1.sec3.fig5.caption1.p1" name="article1.body1.sec3.fig5.caption1.p1"></a><p>Two first principal components cover 63% of the data variability. The virtual border line between distributions of Marked and Poor response patients gives three misclassified Marked patients and four misclassified Poor response patients.</p>
<span>doi:10.1371/journal.pone.0000584.g005</span></div><div class="figure" id="pone-0000584-g006"><div class="img"><a name="pone-0000584-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000584" data-uri="info:doi/10.1371/journal.pone.0000584.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000584.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000584.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000584.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000584.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The Euclidean measure based hierarchical (UPGM) clustering of patients according to day 0 normalized expression of 35 VT-linked genes across other days.</span></strong></p><a id="article1.body1.sec3.fig6.caption1.p1" name="article1.body1.sec3.fig6.caption1.p1"></a><p>Marked (pink) and Poor response (green) patients are mostly separated in two branches of the tree. There are three Marked and six Poor response patients misclassified.</p>
<span>doi:10.1371/journal.pone.0000584.g006</span></div></div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The goal of this paper is to link heterogeneous sets of observations (gene expressions and viral levels) without an a priori hypothesis. We developed a mathematical model that can be applied to any situation using gene expression and viral titer or any other attributes. Application of this approach to patients treated with interferon/ribavirin is based on the assumption that distances between sets of patient's attributes reflects a biological demarcation of patients. As patient attributes we applied the following measurements: (i) viral titer profile during the first four weeks of treatment (viral levels on days 0, 1, 2, 7, 14, and 28), and (ii) the expression of all 22,000 genes of the Affymetrix array across the time course of the treatment.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The difference among patients according to all 22,000 genes on the array reflects the overall gene expression heterogeneity of hepatitis C patients undergoing interferon/ribavirin treatment. This could be due to differences in response, to genetic heterogeneity, or to difference in arrays and handling of RNA. The differences between patients according to any single gene reflect the gene-specific variability of the hepatitis C patients. The gene-specific variability that correlates with the viral titer variability is what is analyzed in this paper and which may be independent of the above. No differences were found in gene expression in an analysis of RNA isolated from PBMC before treatment, when patients were divided into responders and non-responders. Thus the differences are reflection of treatment rather than a reflection of the course of hepatitis C infection.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The gene-specific divergence of patients is checked against the overall patient divergence according to all genes. It appears that the process of virus clearance by interferon/ribavirin is not the major part of the overall gene expression pattern. Indeed, VT-linked genes (genes identified as viral clearance related) make a rather small input into the MV vector, which represents the pattern of overall gene expression variability of patients (<a href="#pone-0000584-g003">Fig.3</a> and <a href="#pone-0000584-g004">4</a>).</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>We examined the variability of patients according to changes in viral titer with time. This analysis demonstrates that the clustering of patients [<a href="#pone-0000584-g001">Fig. 1</a>] according to viral titer largely matches the a priori definition for classification of patients (marked, intermediate, poor) used in this and earlier papers <a href="#pone.0000584-Taylor2">[11]</a>. Although there are a few outliers, in general our initial division using day 28 viral titer appears to have biological as well as mathematical relevance. Even though the a priori classification of patients was based solely on the viral decline at day 28 relative to day 0, the unsupervised clustering of patients based on viral titers at all 6 days resulted in similar clusters, indicating that patterns of viral declines for marked, intermediate and poor response patients are different in-between day 0 and day 28. Since the clustering of patients according to the viral titer profiles is in good correspondence with patient classification (<a href="#pone-0000584-g001">Fig. 1</a>,) the selected 37 probe sets (<a href="#pone-0000584-t001">table 1</a> and <a href="#pone.0000584.s001">table S1</a>) could be considered as genes that are important in the anti-viral and immune response to interferon/ribavirin therapy.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>A very large number of genes are modified by the treatment <em>in vitro</em> of PBMC with interferon and ribavirin <a href="#pone.0000584-Taylor1">[7]</a>, <a href="#pone.0000584-Ji1">[19]</a>. Similar results have been found recently following the treatment of A549 cells and Huh 7 cells with IFN-αcon1 <a href="#pone.0000584-Tan1">[10]</a>. Many of the genes listed in <a href="#pone-0000584-t001">table 1</a> are known to be involved in the interferon response from <em>in vitro</em> studies <a href="#pone.0000584-Taylor1">[7]</a>, <a href="#pone.0000584-Tan1">[10]</a>, <a href="#pone.0000584-Sanda1">[18]</a>. In most cases the level of induction for these genes was much lower in poor response patients than in marked patients, as would be expected from the relationship to viral titer.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Among the genes we identified as important is IRF-7. This gene is required for the induction of type I interferons <a href="#pone.0000584-Samuel1">[20]</a>–<a href="#pone.0000584-Honda1">[23]</a> and is an important component of the Toll signaling pathway <a href="#pone.0000584-Fitzgerald1">[24]</a>. Studies using LPS have indicated that IRF-7 plays an important role in LPS induced B7.1 activation through the JNK system <a href="#pone.0000584-Lim1">[25]</a>. B7-1 is an important co-stimulatory factor required for T-cell activation. Binding of TLR7 and activation of the NFkB pathway via IRF-7 <a href="#pone.0000584-Schoenemeyer1">[26]</a> is required for the endogenous production of interferon-alpha and beta. Thus the levels of IRF-7 may be important in the overall patient response <a href="#pone.0000584-Honda1">[23]</a>, and the endogenous production of IFN-α or IFN-β would certainly lead to further anti-viral effects. Another gene recently reported to be involved in the signaling of the interferon response, is RIG-I, a key ds-RNA sensor protein. Activation of RIG-I (DDX58) leads to the induction of IRF-3, IRF-7, and NFκB <a href="#pone.0000584-Li1">[27]</a>–<a href="#pone.0000584-Johnson1">[29]</a>. This pathway is blocked <span class="underline">in vitro</span> in hepatitis C infection by the NS3/4A protein of the virus <a href="#pone.0000584-Johnson1">[29]</a>. However it is unlikely that this occurs in PBMC, since there is no evidence that PBMC are widely infected with the virus. IFIH1 is also a DEAD protein with a helicase domain. Its relationship to RIG-I is unknown. Among the other genes listed in <a href="#pone-0000584-t001">Table 1</a>, many have been describes as part of the initial response in vivo (in Chimpanzees) to hepatitis C infection, and are the results of endogenous interferon production. These include cig 5, (RSAD2), IFI44, MX1, and the OAS genes <a href="#pone.0000584-Su1">[30]</a>. Viperin (cig5) has also been reported as being induced in the liver in human patients during hepatitis C infection, presumably by endogenous interferon or double-stranded RNA <a href="#pone.0000584-Helbig1">[31]</a>. It has been proposed that this gene product has anti –hepatitis C activity <a href="#pone.0000584-Helbig1">[31]</a>. PLSCR1 has also been shown to potentiate the anti-viral activity of interferon . When PLSCR1 expression was decreased by siRNA, higher titers of VSV and EMC were obtained <a href="#pone.0000584-Dong1">[32]</a>. G1P3 (IFI-16-6) may function as a cell survival protein by inhibiting mitochondrial-mediated apoptosis through the inhibition of caspase 3 activity <a href="#pone.0000584-Tahara1">[33]</a>. Its role in the anti-viral response or anti-hepatitis response is unknown. .. Many of the genes on this list have unknown functions (IFI27, IFIT1, IFIT5, , IFI44, IFI44L, LAMP 3, , FLJ20035, IFRG28, DNAPTP6). Other genes are involved in transcription regulation including RGL1, and TBX3 or translation repression (SAMD4).). PABPC4 is also known as inducible poly(A) binding protein and is upregulated in activated T-cells, but not in resting T-cells <a href="#pone.0000584-Yang1">[34]</a>. However in this study it was slightly down regulated. It also appears to be involved in the regulation of IL-2 <a href="#pone.0000584-Okochi1">[35]</a>. G1P2 is identical to ISG15. This protein is a ubiquitin-like protein which is conjugated to many cellular proteins <a href="#pone.0000584-Desai1">[36]</a>. It appears to interfere with protein poly-ubiquitination and protein degradation. Its role in the interferon response is unknown, although it is highly induced. USP18, a member of the de-ubiquitinating protease family of enzymes, removes ubiquitin adducts from a broad range of protein substrates. Herc 5 and Herc 6 belong to a family of ubiquitin ligases. Thus many genes involved in protein modification are essential to the anti-viral response. SN (Sialoadhesion) very highly induced both in vitro and in vivo is a member of the sialic acid binding immunoglobulin ( Ig)-like lectins. And is primarily expressed in resident and inflammatory macrophage populations <a href="#pone.0000584-Hartnell1">[37]</a>, <a href="#pone.0000584-vandenBerg1">[38]</a>. SN deficient mice exhibit changes in B and T-cell populations and it is proposed that this molecule regulates the immune system <a href="#pone.0000584-Oetke1">[39]</a>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Two genes that are down regulated correlate with the viral response: RPL22 (ribosomal protein L22), a component of the 60S ribosome, and eukaryotic translation initiation factor 3, subunit 6 interacting protein. Whether this decrease is involved in virus inhibition through modifying IRES-dependent translation of the HCV genome is speculative.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>It is of interest that not only inducible genes appear to be a major component of the interferon response, but also down regulation (repression) of a translation factor and ribosomal protein.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>Since this was an unsupervised analysis and did not take into account A/P ( absence/present) filtering, some of the genes are possibly not involved in the anti-viral response, since they were not present in specific classes of patients as analysed with MAS5 soft ware. These include BLZF1, EPHB2, PCTK3 and SNF7DC2</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>In summary we have identified key genes in the response to interferon/ribavirin in hepatitis C patients using a novel method of analysis. This is based on correlation with decrease in virus titer. This method has broad utility and can be used to analyze response to any group of factors influencing biological outcome.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000584"></div><a name="pone.0000584.s001" id="pone.0000584.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000584.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Contains 166 gene-days of 37 viral titer linked genes (probe sets). Not less than 3 gene-days of the gene have component PC2 values more than 5 (these gene-days are pink dots of the <a href="#pone-0000584-g004">Fig. 4</a>). The component PC2 is PCA second component after multidimensional scaling of 8 variables: correlation coefficients in the space of (52*51)/2 dimensions between Gcov_MVcov, GDcov_MVcov, Gcov_VTcov, GDcov_VTcov, Geu_MVeu, GDeu_MVeu, Geu_VTeu, GDeu_VTeu for all gene-days. Columns of the <a href="#pone.0000584.s001">Table S1</a> are as follows: SID - Affymetrix probe set ID; DAY - day of the treatment; Static_CA_1013 - log-transformed and day0 normalized gene expression values [(Ln(d<sub>i</sub>)- Ln(d<sub>0</sub>))] for a patient 1013. The response based classification of the patient and her/his race are identified in the patient's name; Slow_CA_1016; Rapid_AA_1018; .................. - 52 patients of the study; VTeu_Geu - the following 8 columns are correlations between corresponding vectors in the space of (52*51)/2 dimensions. These correlations are 8 initial variables for the multidimensional scaling procedure; VTeu_GDeu; MVeu_Geu; MVeu_GDeu; VTcov_Gcov; VTcov_GDcov; MVcov_Gcov; MVcov_GDcov; GENE_SYMBOL - the following are gene names and descriptions; LOCUSLINK; UNIGENE; GENBANK; GENE; SOURCE_ID; DESCRIPTION; PC1 - the multidimensional scaling first component; PC2 - the multidimensional scaling second component.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000584.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.23 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Mary Ferris for the excellent record-keeping, and entering of information into the portal at the center for Medical Genetics. We thank Ron Jerome and Chunxiao Zhu for expert assistance with the microarray studies, which were carried out using the facilities of the Center for Medical Genomics at Indiana University School of Medicine. The Center for Medical Genomics is supported in part by grants from the Indiana 21st Century Research and Technology Fund and the Indiana Genomics Initiative (supported in part by the Lilly Endowment, Inc.). We also wish to thank Song Zhang from the data coordinating center, Pittsburgh for statistical support and Jay H. Hoofnagle for help in editing the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MT LB. Performed the experiments: MT. Analyzed the data: MT JT AW JL LB TT. Contributed reagents/materials/analysis tools: TT. Wrote the paper: MT JT LB. Other: Confirmed statistical analysis: JL AW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000584-Fried1" id="pone.0000584-Fried1"></a>Fried MW, Shiffman M, Sterling RK, Weinstein J, Crippin J, et al.  (2000) A multicenter, randomized trial of daily high-dose interferon-alfa 2b for the treatment of chronic hepatitis c: pretreatment stratification by viral burden and genotype, Am J Gastroenterol  95: 3225–3229.  <ul class="find" data-citedArticleID="963722" data-doi="10.1016/s0016-5085(98)85040-3"><li><a href="http://dx.doi.org/10.1016/s0016-5085(98)85040-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+multicenter%2C+randomized+trial+of+daily+high-dose+interferon-alfa+2b+for+the+treatment+of+chronic+hepatitis+c%3A+pretreatment+stratification+by+viral+burden+and+genotype%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+multicenter%2C+randomized+trial+of+daily+high-dose+interferon-alfa+2b+for+the+treatment+of+chronic+hepatitis+c%3A+pretreatment+stratification+by+viral+burden+and+genotype%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000584-Fried2" id="pone.0000584-Fried2"></a>Fried MW, Shiffman ML, Reddy KR, Smith C, Marinos G, et al.  (2002) Peginterferon alfa-2a plus ribavirin for chronic hepatitis C virus infection, N Engl J Med  347: 975–982.  <ul class="find" data-citedArticleID="963754" data-doi="10.1056/nejmoa020047"><li><a href="http://dx.doi.org/10.1056/nejmoa020047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peginterferon+alfa-2a+plus+ribavirin+for+chronic+hepatitis+C+virus+infection%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peginterferon+alfa-2a+plus+ribavirin+for+chronic+hepatitis+C+virus+infection%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000584-Manns1" id="pone.0000584-Manns1"></a>Manns MP, McHutchison JG, Gordon SC, Rustgi VK, Shiffman M, et al.  (2001) Peginterferon alfa-2b plus ribavirin compared with interferon alfa-2b plus ribavirin for initial treatment of chronic hepatitis C: a randomised trial, Lancet  358: 958–965.  <ul class="find" data-citedArticleID="963774" data-doi="10.1016/s0140-6736(01)06102-5"><li><a href="http://dx.doi.org/10.1016/s0140-6736(01)06102-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peginterferon+alfa-2b+plus+ribavirin+compared+with+interferon+alfa-2b+plus+ribavirin+for+initial+treatment+of+chronic+hepatitis+C%3A+a+randomised+trial%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peginterferon+alfa-2b+plus+ribavirin+compared+with+interferon+alfa-2b+plus+ribavirin+for+initial+treatment+of+chronic+hepatitis+C%3A+a+randomised+trial%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000584-Hadziyannis1" id="pone.0000584-Hadziyannis1"></a>Hadziyannis SJ, Sette H Jr, Morgan TR, Balan V, Diago M, et al.  (2004) Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose, Ann Intern Med  140: 346–355.  <ul class="find" data-citedArticleID="963752" data-doi="10.7326/0003-4819-140-5-200403020-00010"><li><a href="http://dx.doi.org/10.7326/0003-4819-140-5-200403020-00010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peginterferon-alpha2a+and+ribavirin+combination+therapy+in+chronic+hepatitis+C%3A+a+randomized+study+of+treatment+duration+and+ribavirin+dose%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peginterferon-alpha2a+and+ribavirin+combination+therapy+in+chronic+hepatitis+C%3A+a+randomized+study+of+treatment+duration+and+ribavirin+dose%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000584-Neumann1" id="pone.0000584-Neumann1"></a>Neumann AU, Lam NP, Dahari H, Gretch DR, Wiley TE, et al.  (1998) Hepatitis C viral dynamics in vivo and the antiviral efficacy of interferon-alpha therapy, Science  282: 103–107.  <ul class="find" data-citedArticleID="963742" data-doi="10.1126/science.282.5386.103"><li><a href="http://dx.doi.org/10.1126/science.282.5386.103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hepatitis+C+viral+dynamics+in+vivo+and+the+antiviral+efficacy+of+interferon-alpha+therapy%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hepatitis+C+viral+dynamics+in+vivo+and+the+antiviral+efficacy+of+interferon-alpha+therapy%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000584-Zeuzem1" id="pone.0000584-Zeuzem1"></a>Zeuzem S, Herrmann E, Lee JH, Fricke J, Neumann AU, et al.  (2001) Viral kinetics in patients with chronic hepatitis C treated with standard or peginterferon alpha2a, Gastroenterology  120: 1438–1447.  <ul class="find" data-citedArticleID="963784" data-doi="10.1053/gast.2001.24006"><li><a href="http://dx.doi.org/10.1053/gast.2001.24006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+kinetics+in+patients+with+chronic+hepatitis+C+treated+with+standard+or+peginterferon+alpha2a%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+kinetics+in+patients+with+chronic+hepatitis+C+treated+with+standard+or+peginterferon+alpha2a%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000584-Taylor1" id="pone.0000584-Taylor1"></a>Taylor MW, Grosse WM, Schaley JE, Sanda C, Wu X, et al.  (2004) Global effect of PEG-IFN-alpha and ribavirin on gene expression in PBMC in vitro, J Interferon Cytokine Res  24: 107–118.  <ul class="find" data-citedArticleID="963720" data-doi="10.1089/107999004322813354"><li><a href="http://dx.doi.org/10.1089/107999004322813354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+effect+of+PEG-IFN-alpha+and+ribavirin+on+gene+expression+in+PBMC+in+vitro%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+effect+of+PEG-IFN-alpha+and+ribavirin+on+gene+expression+in+PBMC+in+vitro%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000584-Der1" id="pone.0000584-Der1"></a>Der SD, Zhou A, Williams BR, Silverman RH (1998) Identification of genes differentially regulated by interferon alpha, beta, or gamma using oligonucleotide arrays, Proc Natl Acad Sci U S A  95: 15623–15628.  <ul class="find" data-citedArticleID="963772" data-doi="10.1073/pnas.95.26.15623"><li><a href="http://dx.doi.org/10.1073/pnas.95.26.15623" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genes+differentially+regulated+by+interferon+alpha%2C+beta%2C+or+gamma+using+oligonucleotide+arrays%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genes+differentially+regulated+by+interferon+alpha%2C+beta%2C+or+gamma+using+oligonucleotide+arrays%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000584-deVeer1" id="pone.0000584-deVeer1"></a>de Veer MJ, Holko M, Frevel M, Walker E, Der S, et al.  (2001) Functional classification of interferon-stimulated genes identified using microarrays, J Leukoc Biol  69: 912–920.  <ul class="find" data-citedArticleID="963734"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+classification+of+interferon-stimulated+genes+identified+using+microarrays%2C&amp;auth=&amp;atitle=Functional+classification+of+interferon-stimulated+genes+identified+using+microarrays%2C" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+classification+of+interferon-stimulated+genes+identified+using+microarrays%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+classification+of+interferon-stimulated+genes+identified+using+microarrays%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000584-Tan1" id="pone.0000584-Tan1"></a>Tan H, Derrick J, Hong J, Sanda C, Grosse WM, et al.  (2005) Global transcriptional profiling combination of type I and type II demonstrates the interferon enhances antiviral and immune responses at clinically relevant doses, J Interf Cytok Res  25: 632–649.  <ul class="find" data-citedArticleID="963750" data-doi="10.1089/jir.2005.25.632"><li><a href="http://dx.doi.org/10.1089/jir.2005.25.632" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transcriptional+profiling+combination+of+type+I+and+type+II+demonstrates+the+interferon+enhances+antiviral+and+immune+responses+at+clinically+relevant+doses%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transcriptional+profiling+combination+of+type+I+and+type+II+demonstrates+the+interferon+enhances+antiviral+and+immune+responses+at+clinically+relevant+doses%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000584-Taylor2" id="pone.0000584-Taylor2"></a>Taylor MW, Tsukahara T, Brodsky L, Schaley J, Sanda C, et al.  (2007) Changes in gene expression during pegylated interferon and ribavirin therapy of chronic hepatitis C virus distinguish responders from nonresponders to antiviral therapy, J Virol  81: 3391–3401.  <ul class="find" data-citedArticleID="963724" data-doi="10.1128/jvi.02640-06"><li><a href="http://dx.doi.org/10.1128/jvi.02640-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+gene+expression+during+pegylated+interferon+and+ribavirin+therapy+of+chronic+hepatitis+C+virus+distinguish+responders+from+nonresponders+to+antiviral+therapy%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+gene+expression+during+pegylated+interferon+and+ribavirin+therapy+of+chronic+hepatitis+C+virus+distinguish+responders+from+nonresponders+to+antiviral+therapy%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000584-Alter1" id="pone.0000584-Alter1"></a>Alter O, Brown PO, Botstein D (2000) Singular value decomposition for genome-wide expression data processing and modeling, Proc Natl Acad Sci U S A  97: 10101–10106.  <ul class="find" data-citedArticleID="963760" data-doi="10.1073/pnas.97.18.10101"><li><a href="http://dx.doi.org/10.1073/pnas.97.18.10101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Singular+value+decomposition+for+genome-wide+expression+data+processing+and+modeling%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Singular+value+decomposition+for+genome-wide+expression+data+processing+and+modeling%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000584-Jail1" id="pone.0000584-Jail1"></a>Jail A, Dubes R (1988) Algorithms for Clustering Data,. Englewood Cliffs, NJ.USA: Prentice Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000584-Pazos1" id="pone.0000584-Pazos1"></a>Pazos F, Valencia A (2001) Similarity of phylogenetic trees as indicator of protein-protein interaction, Protein engineering  14: 609–614.  <ul class="find" data-citedArticleID="963792" data-doi="10.1093/protein/14.9.609"><li><a href="http://dx.doi.org/10.1093/protein/14.9.609" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similarity+of+phylogenetic+trees+as+indicator+of+protein-protein+interaction%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similarity+of+phylogenetic+trees+as+indicator+of+protein-protein+interaction%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000584-Gertz1" id="pone.0000584-Gertz1"></a>Gertz J, Elfond G, Shustrova A, Weisinger M, Pellegrini M, et al.  (2003) Inferring protein interactions from phylogenetic distance matrices, Bioinformatics  19: 2039–2045.  <ul class="find" data-citedArticleID="963726" data-doi="10.1093/bioinformatics/btg278"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg278" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+protein+interactions+from+phylogenetic+distance+matrices%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+protein+interactions+from+phylogenetic+distance+matrices%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000584-Snedecor1" id="pone.0000584-Snedecor1"></a>Snedecor GW, Cochran WG (1989) Statistical Methods. Ames: Iowa University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000584-Cox1" id="pone.0000584-Cox1"></a>Cox T, Cox M (2003) Multidimensional Scaling. Chapman and Hall.   <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000584-Sanda1" id="pone.0000584-Sanda1"></a>Sanda C, Weitzel P, Tsukahara T, Schaley J, Edenberg HJ, et al.  (2006) Differential Gene Induction by Type I and Type II Interferons and Their Combination, J Interferon Cytokine Res  26: 462–472.  <ul class="find" data-citedArticleID="963778" data-doi="10.1089/jir.2006.26.462"><li><a href="http://dx.doi.org/10.1089/jir.2006.26.462" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+Gene+Induction+by+Type+I+and+Type+II+Interferons+and+Their+Combination%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+Gene+Induction+by+Type+I+and+Type+II+Interferons+and+Their+Combination%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000584-Ji1" id="pone.0000584-Ji1"></a>Ji X, Cheung R, Cooper S, Li Q, Greenberg HB, et al.  (2003) Interferon alfa regulated gene expression in patients initiating interferon treatment for chronic hepatitis C, Hepatology  37: 610–621.  <ul class="find" data-citedArticleID="963746" data-doi="10.1053/jhep.2003.50105"><li><a href="http://dx.doi.org/10.1053/jhep.2003.50105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interferon+alfa+regulated+gene+expression+in+patients+initiating+interferon+treatment+for+chronic+hepatitis+C%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interferon+alfa+regulated+gene+expression+in+patients+initiating+interferon+treatment+for+chronic+hepatitis+C%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000584-Samuel1" id="pone.0000584-Samuel1"></a>Samuel CE (2001) Antiviral actions of interferons, Clin Microbiol Rev  14: 778–809, table of contents.  <ul class="find" data-citedArticleID="963776" data-doi="10.1128/cmr.14.4.778-809.2001"><li><a href="http://dx.doi.org/10.1128/cmr.14.4.778-809.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antiviral+actions+of+interferons%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antiviral+actions+of+interferons%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000584-Barnes1" id="pone.0000584-Barnes1"></a>Barnes BJ, Richards J, Mancl M, Hanash S, Beretta L, et al.  (2004) Global and distinct targets of IRF-5 and IRF-7 during innate response to viral infection, J Biol Chem  279: 45194–45207.  <ul class="find" data-citedArticleID="963780" data-doi="10.1074/jbc.m400726200"><li><a href="http://dx.doi.org/10.1074/jbc.m400726200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+and+distinct+targets+of+IRF-5+and+IRF-7+during+innate+response+to+viral+infection%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+and+distinct+targets+of+IRF-5+and+IRF-7+during+innate+response+to+viral+infection%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000584-Nguyen1" id="pone.0000584-Nguyen1"></a>Nguyen H, Hiscott J, Pitha PM (1997) The growing family of interferon regulatory factors, Cytokine Growth Factor Rev  8: 293–312.  <ul class="find" data-citedArticleID="963730" data-doi="10.1016/s1359-6101(97)00019-1"><li><a href="http://dx.doi.org/10.1016/s1359-6101(97)00019-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+growing+family+of+interferon+regulatory+factors%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+growing+family+of+interferon+regulatory+factors%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000584-Honda1" id="pone.0000584-Honda1"></a>Honda K, Yanai H, Negishi H, Asagiri M, Sato M, et al.  (2005) IRF-7 is the master regulator of type-I interferon-dependent immune responses, Nature  434: 772–777.  <ul class="find" data-citedArticleID="963762" data-doi="10.1038/nature03464"><li><a href="http://dx.doi.org/10.1038/nature03464" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=IRF-7+is+the+master+regulator+of+type-I+interferon-dependent+immune+responses%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22IRF-7+is+the+master+regulator+of+type-I+interferon-dependent+immune+responses%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000584-Fitzgerald1" id="pone.0000584-Fitzgerald1"></a>Fitzgerald KA, Rowe DC, Barnes BJ, Caffrey DR, Visintin A, et al.  (2003) LPS-TLR4 signaling to IRF-3/7 and NF-kappaB involves the toll adapters TRAM and TRIF, J Exp Med  198: 1043–1055.  <ul class="find" data-citedArticleID="963788" data-doi="10.1084/jem.20031023"><li><a href="http://dx.doi.org/10.1084/jem.20031023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LPS-TLR4+signaling+to+IRF-3%2F7+and+NF-kappaB+involves+the+toll+adapters+TRAM+and+TRIF%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LPS-TLR4+signaling+to+IRF-3%2F7+and+NF-kappaB+involves+the+toll+adapters+TRAM+and+TRIF%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000584-Lim1" id="pone.0000584-Lim1"></a>Lim W, Gee K, Mishra S, Kumar A (2005) Regulation of B7.1 costimulatory molecule is mediated by the IFN regulatory factor-7 through the activation of JNK in lipopolysaccharide-stimulated human monocytic cells, J Immunol  175: 5690–5700.  <ul class="find" data-citedArticleID="963756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+B7.1+costimulatory+molecule+is+mediated+by+the+IFN+regulatory+factor-7+through+the+activation+of+JNK+in+lipopolysaccharide-stimulated+human+monocytic+cells%2C&amp;auth=&amp;atitle=Regulation+of+B7.1+costimulatory+molecule+is+mediated+by+the+IFN+regulatory+factor-7+through+the+activation+of+JNK+in+lipopolysaccharide-stimulated+human+monocytic+cells%2C" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+B7.1+costimulatory+molecule+is+mediated+by+the+IFN+regulatory+factor-7+through+the+activation+of+JNK+in+lipopolysaccharide-stimulated+human+monocytic+cells%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+B7.1+costimulatory+molecule+is+mediated+by+the+IFN+regulatory+factor-7+through+the+activation+of+JNK+in+lipopolysaccharide-stimulated+human+monocytic+cells%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000584-Schoenemeyer1" id="pone.0000584-Schoenemeyer1"></a>Schoenemeyer A, Barnes BJ, Mancl ME, Latz E, Goutagny N, et al.  (2005) The interferon regulatory factor, IRF5, is a central mediator of toll-like receptor 7 signaling, J Biol Chem  280: 17005–17012.  <ul class="find" data-citedArticleID="963738" data-doi="10.1074/jbc.m412584200"><li><a href="http://dx.doi.org/10.1074/jbc.m412584200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+interferon+regulatory+factor%2C+IRF5%2C+is+a+central+mediator+of+toll-like+receptor+7+signaling%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+interferon+regulatory+factor%2C+IRF5%2C+is+a+central+mediator+of+toll-like+receptor+7+signaling%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000584-Li1" id="pone.0000584-Li1"></a>Li K, Foy E, Ferreon JC, Nakamura M, Ferreon AC, et al.  (2005) Immune evasion by hepatitis C virus NS3/4A protease-mediated cleavage of the Toll-like receptor 3 adaptor protein TRIF, Proc Natl Acad Sci U S A  102: 2992–2997.  <ul class="find" data-citedArticleID="963744" data-doi="10.1073/pnas.0408824102"><li><a href="http://dx.doi.org/10.1073/pnas.0408824102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immune+evasion+by+hepatitis+C+virus+NS3%2F4A+protease-mediated+cleavage+of+the+Toll-like+receptor+3+adaptor+protein+TRIF%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immune+evasion+by+hepatitis+C+virus+NS3%2F4A+protease-mediated+cleavage+of+the+Toll-like+receptor+3+adaptor+protein+TRIF%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000584-Sumpter1" id="pone.0000584-Sumpter1"></a>Sumpter R Jr, Loo YM, Foy E, Li K, Yoneyama M, et al.  (2005) Regulating intracellular antiviral defense and permissiveness to hepatitis C virus RNA replication through a cellular RNA helicase, RIG-I, J Virol  79: 2689–2699.  <ul class="find" data-citedArticleID="963766" data-doi="10.1128/jvi.79.5.2689-2699.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.5.2689-2699.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulating+intracellular+antiviral+defense+and+permissiveness+to+hepatitis+C+virus+RNA+replication+through+a+cellular+RNA+helicase%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulating+intracellular+antiviral+defense+and+permissiveness+to+hepatitis+C+virus+RNA+replication+through+a+cellular+RNA+helicase%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000584-Johnson1" id="pone.0000584-Johnson1"></a>Johnson CL, Gale MJ Jr (2006) CARD games between virus and host get a new player, Trends Immunol  27: 1–4.  <ul class="find" data-citedArticleID="963740" data-doi="10.1016/j.it.2005.11.004"><li><a href="http://dx.doi.org/10.1016/j.it.2005.11.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CARD+games+between+virus+and+host+get+a+new+player%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CARD+games+between+virus+and+host+get+a+new+player%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000584-Su1" id="pone.0000584-Su1"></a>Su AI, Pezacki JP, Wodicka L, Brideau AD, Supekova L, et al.  (2002) Genomic analysis of the host response to hepatitis C virus infection, Proc Natl Acad Sci U S A  99: 15669–15674.  <ul class="find" data-citedArticleID="963764" data-doi="10.1073/pnas.202608199"><li><a href="http://dx.doi.org/10.1073/pnas.202608199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+of+the+host+response+to+hepatitis+C+virus+infection%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+of+the+host+response+to+hepatitis+C+virus+infection%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000584-Helbig1" id="pone.0000584-Helbig1"></a>Helbig KJ, Lau DT, Semendric L, Harley HA, Beard MR (2005) Analysis of ISG expression in chronic hepatitis C identifies viperin as a potential antiviral effector, Hepatology  42: 702–710.  <ul class="find" data-citedArticleID="963782" data-doi="10.1002/hep.20844"><li><a href="http://dx.doi.org/10.1002/hep.20844" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+ISG+expression+in+chronic+hepatitis+C+identifies+viperin+as+a+potential+antiviral+effector%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+ISG+expression+in+chronic+hepatitis+C+identifies+viperin+as+a+potential+antiviral+effector%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000584-Dong1" id="pone.0000584-Dong1"></a>Dong B, Zhou Q, Zhao J, Zhou A, Harty RN, et al.  (2004) Phospholipid scramblase 1 potentiates the antiviral activity of interferon, J Virol  78: 8983–8993.  <ul class="find" data-citedArticleID="963790" data-doi="10.1128/jvi.78.17.8983-8993.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.17.8983-8993.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phospholipid+scramblase+1+potentiates+the+antiviral+activity+of+interferon%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phospholipid+scramblase+1+potentiates+the+antiviral+activity+of+interferon%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000584-Tahara1" id="pone.0000584-Tahara1"></a>Tahara E Jr, Tahara H, Kanno M, Naka K, Takeda Y, et al.  (2005) G1P3, an interferon inducible gene 6–16, is expressed in gastric cancers and inhibits mitochondrial-mediated apoptosis in gastric cancer cell line TMK-1 cell, Cancer Immunol Immunother  54: 729–740.  <ul class="find" data-citedArticleID="963768" data-doi="10.1007/s00262-004-0645-2"><li><a href="http://dx.doi.org/10.1007/s00262-004-0645-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G1P3%2C+an+interferon+inducible+gene+6%E2%80%9316%2C+is+expressed+in+gastric+cancers+and+inhibits+mitochondrial-mediated+apoptosis+in+gastric+cancer+cell+line+TMK-1+cell%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G1P3%2C+an+interferon+inducible+gene+6%E2%80%9316%2C+is+expressed+in+gastric+cancers+and+inhibits+mitochondrial-mediated+apoptosis+in+gastric+cancer+cell+line+TMK-1+cell%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000584-Yang1" id="pone.0000584-Yang1"></a>Yang H, Duckett CS, Lindsten T (1995) iPABP, an inducible poly(A)-binding protein detected in activated human T cells, Mol Cell Biol  15: 6770–6776.  <ul class="find" data-citedArticleID="963758"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=iPABP%2C+an+inducible+poly%28A%29-binding+protein+detected+in+activated+human+T+cells%2C&amp;auth=&amp;atitle=iPABP%2C+an+inducible+poly%28A%29-binding+protein+detected+in+activated+human+T+cells%2C" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=iPABP%2C+an+inducible+poly%28A%29-binding+protein+detected+in+activated+human+T+cells%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22iPABP%2C+an+inducible+poly%28A%29-binding+protein+detected+in+activated+human+T+cells%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000584-Okochi1" id="pone.0000584-Okochi1"></a>Okochi K, Suzuki T, Inoue J, Matsuda S, Yamamoto T (2005) Interaction of anti-proliferative protein Tob with poly(A)-binding protein and inducible poly(A)-binding protein: implication of Tob in translational control, Genes Cells  10: 151–163.  <ul class="find" data-citedArticleID="963786" data-doi="10.1111/j.1365-2443.2005.00826.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2443.2005.00826.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+anti-proliferative+protein+Tob+with+poly%28A%29-binding+protein+and+inducible+poly%28A%29-binding+protein%3A+implication+of+Tob+in+translational+control%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+anti-proliferative+protein+Tob+with+poly%28A%29-binding+protein+and+inducible+poly%28A%29-binding+protein%3A+implication+of+Tob+in+translational+control%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000584-Desai1" id="pone.0000584-Desai1"></a>Desai SD, Haas AL, Wood LM, Tsai YC, Pestka S, et al.  (2006) Elevated expression of ISG15 in tumor cells interferes with the ubiquitin/26S proteasome pathway, Cancer research  66: 921–928.  <ul class="find" data-citedArticleID="963732" data-doi="10.1158/0008-5472.can-05-1123"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-1123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elevated+expression+of+ISG15+in+tumor+cells+interferes+with+the+ubiquitin%2F26S+proteasome+pathway%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elevated+expression+of+ISG15+in+tumor+cells+interferes+with+the+ubiquitin%2F26S+proteasome+pathway%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000584-Hartnell1" id="pone.0000584-Hartnell1"></a>Hartnell A, Steel J, Turley H, Jones M, Jackson DG, et al.  (2001) Characterization of human sialoadhesin, a sialic acid binding receptor expressed by resident and inflammatory macrophage populations, Blood  97: 288–296.  <ul class="find" data-citedArticleID="963718" data-doi="10.1182/blood.v97.1.288"><li><a href="http://dx.doi.org/10.1182/blood.v97.1.288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+human+sialoadhesin%2C+a+sialic+acid+binding+receptor+expressed+by+resident+and+inflammatory+macrophage+populations%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+human+sialoadhesin%2C+a+sialic+acid+binding+receptor+expressed+by+resident+and+inflammatory+macrophage+populations%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000584-vandenBerg1" id="pone.0000584-vandenBerg1"></a>van den Berg TK, Nath D, Ziltener HJ, Vestweber D, Fukuda M, et al.  (2001) Cutting edge: CD43 functions as a T cell counterreceptor for the macrophage adhesion receptor sialoadhesin (Siglec-1), J Immunol  166: 3637–3640.  <ul class="find" data-citedArticleID="963728"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cutting+edge%3A+CD43+functions+as+a+T+cell+counterreceptor+for+the+macrophage+adhesion+receptor+sialoadhesin+%28Siglec-1%29%2C&amp;auth=&amp;atitle=Cutting+edge%3A+CD43+functions+as+a+T+cell+counterreceptor+for+the+macrophage+adhesion+receptor+sialoadhesin+%28Siglec-1%29%2C" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cutting+edge%3A+CD43+functions+as+a+T+cell+counterreceptor+for+the+macrophage+adhesion+receptor+sialoadhesin+%28Siglec-1%29%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cutting+edge%3A+CD43+functions+as+a+T+cell+counterreceptor+for+the+macrophage+adhesion+receptor+sialoadhesin+%28Siglec-1%29%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000584-Oetke1" id="pone.0000584-Oetke1"></a>Oetke C, Vinson MC, Jones C, Crocker PR (2006) Sialoadhesin-deficient mice exhibit subtle changes in B- and T-cell populations and reduced immunoglobulin M levels, Mol Cell Biol  26: 1549–1557.  <ul class="find" data-citedArticleID="963770" data-doi="10.1128/mcb.26.4.1549-1557.2006"><li><a href="http://dx.doi.org/10.1128/mcb.26.4.1549-1557.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sialoadhesin-deficient+mice+exhibit+subtle+changes+in+B-+and+T-cell+populations+and+reduced+immunoglobulin+M+levels%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sialoadhesin-deficient+mice+exhibit+subtle+changes+in+B-+and+T-cell+populations+and+reduced+immunoglobulin+M+levels%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.XML" value="101373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.PDF" value="725182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e001.PNG" value="12976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e001.TIF" value="27724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e002.PNG" value="9404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e002.TIF" value="23940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e003.PNG" value="12184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e003.TIF" value="26952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e004.PNG" value="10267"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e004.TIF" value="24688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e005.PNG" value="10458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e005.TIF" value="24756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e006.PNG" value="9778"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e006.TIF" value="24244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e007.PNG" value="9590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e007.TIF" value="24128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e008.PNG" value="9040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e008.TIF" value="23504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e009.PNG" value="12516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e009.TIF" value="26580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e010.PNG" value="11685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e010.TIF" value="25928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e011.PNG" value="9196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e011.TIF" value="23756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e012.PNG" value="6460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e012.TIF" value="20980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e013.PNG" value="6503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e013.TIF" value="21044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e014.PNG" value="12139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e014.TIF" value="26340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e015.PNG" value="9945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.e015.TIF" value="24424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g001.PNG_L" value="137845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g001.PNG_M" value="312938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g001.PNG_S" value="14341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g001.TIF" value="202020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g001.PNG_I" value="40669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g002.PNG_L" value="83314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g002.PNG_M" value="83314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g002.PNG_S" value="16732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g002.TIF" value="114748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g002.PNG_I" value="45965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g003.PNG_L" value="159288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g003.PNG_M" value="391703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g003.PNG_S" value="14257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g003.TIF" value="209596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g003.PNG_I" value="53056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g004.PNG_L" value="225191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g004.PNG_M" value="550465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g004.PNG_S" value="19202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g004.TIF" value="306286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g004.PNG_I" value="74750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.t001.PNG_L" value="355666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.t001.PNG_M" value="503794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.t001.PNG_S" value="31673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.t001.TIF" value="1580280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.t001.PNG_I" value="64971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g005.PNG_L" value="75489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g005.PNG_M" value="75489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g005.PNG_S" value="14356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g005.TIF" value="105944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g005.PNG_I" value="34766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g006.PNG_L" value="140167"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g006.PNG_M" value="318624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g006.PNG_S" value="14599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g006.TIF" value="213852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.g006.PNG_I" value="41293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000584.s001.XLS" value="232448"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000584&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000584" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000584&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000584&volume=&issue=&title=A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients&author_name=Leonid%20I.%20Brodsky%2C%20Abdus%20S.%20Wahed%2C%20Jia%20Li%2C%20John%20E.%20Tavis%2C%20Takuma%20Tsukahara%2C%20Milton%20W.%20Taylor&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000584" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584&amp;t=A%20Novel%20Unsupervised%20Method%20to%20Identify%20Genes%20Important%20in%20the%20Anti-viral%20Response%3A%20Application%20to%20Interferon%2FRibavirin%20in%20Hepatitis%20C%20Patients" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584&title=A%20Novel%20Unsupervised%20Method%20to%20Identify%20Genes%20Important%20in%20the%20Anti-viral%20Response%3A%20Application%20to%20Interferon%2FRibavirin%20in%20Hepatitis%20C%20Patients&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584&amp;title=A%20Novel%20Unsupervised%20Method%20to%20Identify%20Genes%20Important%20in%20the%20Anti-viral%20Response%3A%20Application%20to%20Interferon%2FRibavirin%20in%20Hepatitis%20C%20Patients" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000584&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Novel Unsupervised Method to Identify Genes Important in the Anti-viral Response: Application to Interferon/Ribavirin in Hepatitis C Patients';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000584';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Novel%20Unsupervised%20Method%20to%20Identify%20Genes%20Important%20in%20the%20Anti-viral%20Response%3A%20Application%20to%20Interferon%2FRibavirin%20in%20Hepatitis%20C%20Patients http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000584" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000584" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antivirals%22" title="Search for articles in the subject area:'Antivirals'"><div class="flagText">Antivirals</div></a>
              <div data-categoryid="43107" data-articleid="24006"
                   data-categoryname="Antivirals"
                   class="flagImage" title="Flag 'Antivirals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24006"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hepatitis+C%22" title="Search for articles in the subject area:'Hepatitis C'"><div class="flagText">Hepatitis C</div></a>
              <div data-categoryid="32271" data-articleid="24006"
                   data-categoryname="Hepatitis C"
                   class="flagImage" title="Flag 'Hepatitis C' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immune+response%22" title="Search for articles in the subject area:'Immune response'"><div class="flagText">Immune response</div></a>
              <div data-categoryid="17773" data-articleid="24006"
                   data-categoryname="Immune response"
                   class="flagImage" title="Flag 'Immune response' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Interferons%22" title="Search for articles in the subject area:'Interferons'"><div class="flagText">Interferons</div></a>
              <div data-categoryid="20929" data-articleid="24006"
                   data-categoryname="Interferons"
                   class="flagImage" title="Flag 'Interferons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+proteins%22" title="Search for articles in the subject area:'Membrane proteins'"><div class="flagText">Membrane proteins</div></a>
              <div data-categoryid="17569" data-articleid="24006"
                   data-categoryname="Membrane proteins"
                   class="flagImage" title="Flag 'Membrane proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vector+spaces%22" title="Search for articles in the subject area:'Vector spaces'"><div class="flagText">Vector spaces</div></a>
              <div data-categoryid="33091" data-articleid="24006"
                   data-categoryname="Vector spaces"
                   class="flagImage" title="Flag 'Vector spaces' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+vectors%22" title="Search for articles in the subject area:'Viral vectors'"><div class="flagText">Viral vectors</div></a>
              <div data-categoryid="17699" data-articleid="24006"
                   data-categoryname="Viral vectors"
                   class="flagImage" title="Flag 'Viral vectors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=486'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7725'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7067&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
