

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001546"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001546" />

    <meta name="citation_title" content="Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays"/>
    <meta itemprop="name" content="Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays"/>

      <meta name="citation_author" content="Michael A. Cook"/>
            <meta name="citation_author_institution" content="Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada"/>
      <meta name="citation_author" content="Chi-Kin Chan"/>
            <meta name="citation_author_institution" content="Microarray Laboratory, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada"/>
      <meta name="citation_author" content="Paul Jorgensen"/>
            <meta name="citation_author_institution" content="Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada"/>
      <meta name="citation_author" content="Troy Ketela"/>
            <meta name="citation_author_institution" content="Terrence Donnelly Center for Cellular and Biomolecular Research, University of Toronto, 1Toronto, Ontario, Canada"/>
      <meta name="citation_author" content="Daniel So"/>
            <meta name="citation_author_institution" content="Scenterra Inc., Bowie, Maryland, United States of America"/>
      <meta name="citation_author" content="Mike Tyers"/>
            <meta name="citation_author_institution" content="Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada"/>
      <meta name="citation_author" content="Chi-Yip Ho"/>
            <meta name="citation_author_institution" content="Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Microarray Laboratory, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada"/>

    <meta name="citation_date" content="2008/2/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001546.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1546"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae.; citation_author=EF Glynn; citation_author=PC Megee; citation_author=HG Yu; citation_author=C Mistrot; citation_author=E Unal; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=1; citation_pages=E259; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microarray technology: beyond transcript profiling and genotype analysis.; citation_author=JD Hoheisel; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=2; citation_pages=200-210; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Replication dynamics of the yeast genome.; citation_author=MK Raghuraman; citation_author=EA Winzeler; citation_author=D Collingwood; citation_author=S Hunt; citation_author=L Wodicka; citation_journal_title=Science; citation_volume=294; citation_number=3; citation_pages=115-121; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of chromatin status using tiling microarrays.; citation_author=S Shivaswamy; citation_author=VR Iyer; citation_journal_title=Methods; citation_volume=41; citation_number=4; citation_pages=304-311; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=ChIP-chip comes of age for genome-wide functional analysis.; citation_author=J Wu; citation_author=LT Smith; citation_author=C Plass; citation_author=TH Huang; citation_journal_title=Cancer Res; citation_volume=66; citation_number=5; citation_pages=6899-6902; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global analysis of gene function in yeast by quantitative phenotypic profiling.; citation_author=JA Brown; citation_author=G Sherlock; citation_author=CL Myers; citation_author=NM Burrows; citation_author=C Deng; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=6; citation_pages=2006 0001; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide requirements for resistance to functionally distinct DNA-damaging agents.; citation_author=W Lee; citation_author=RP St Onge; citation_author=M Proctor; citation_author=P Flaherty; citation_author=MI Jordan; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=7; citation_pages=e24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Discovering modes of action for therapeutic compounds using a genome-wide screen of yeast heterozygotes.; citation_author=PY Lum; citation_author=CD Armour; citation_author=SB Stepaniants; citation_author=G Cavet; citation_author=MK Wolf; citation_journal_title=Cell; citation_volume=116; citation_number=8; citation_pages=121-137; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global synthetic-lethality analysis and yeast functional profiling.; citation_author=SL Ooi; citation_author=X Pan; citation_author=BD Peyser; citation_author=P Ye; citation_author=PB Meluh; citation_journal_title=Trends Genet; citation_volume=22; citation_number=9; citation_pages=56-63; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A resource for large-scale RNA-interference-based screens in mammals.; citation_author=PJ Paddison; citation_author=JM Silva; citation_author=DS Conklin; citation_author=M Schlabach; citation_author=M Li; citation_journal_title=Nature; citation_volume=428; citation_number=10; citation_pages=427-431; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=dSLAM analysis of genome-wide genetic interactions in Saccharomyces cerevisiae.; citation_author=X Pan; citation_author=DS Yuan; citation_author=SL Ooi; citation_author=X Wang; citation_author=S Sookhai-Mahadeo; citation_journal_title=Methods; citation_volume=41; citation_number=11; citation_pages=206-221; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions.; citation_author=RP St Onge; citation_author=R Mani; citation_author=J Oh; citation_author=M Proctor; citation_author=E Fung; citation_journal_title=Nat Genet; citation_volume=39; citation_number=12; citation_pages=199-206; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Fabrication of DNA microarrays using unmodified oligonucleotide probes.; citation_author=DR Call; citation_author=DP Chandler; citation_author=F Brockman; citation_journal_title=Biotechniques; citation_volume=30; citation_number=13; citation_pages=368-372, 374, 376 passim; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Making and reading microarrays.; citation_author=VG Cheung; citation_author=M Morley; citation_author=F Aguilar; citation_author=A Massimi; citation_author=R Kucherlapati; citation_journal_title=Nat Genet; citation_volume=21; citation_number=14; citation_pages=15-19; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Manufacturing DNA microarrays of high spot homogeneity and reduced background signal.; citation_author=F Diehl; citation_author=S Grahlmann; citation_author=M Beier; citation_author=JD Hoheisel; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=15; citation_pages=E38; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA microarray technology for the microbiologist: an overview.; citation_author=A Ehrenreich; citation_journal_title=Appl Microbiol Biotechnol; citation_volume=73; citation_number=16; citation_pages=255-273; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Photoreaction of thymidine with primary amines. Application to specific modification of DNA.; citation_author=I Saito; citation_author=H Sugiyama; citation_author=N Furukawa; citation_author=T Matsuura; citation_journal_title=Nucleic Acids Symp Ser; citation_number=17; citation_pages=61-64; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Covalent attachment of synthetic DNA to self-assembled monolayer films.; citation_author=LA Chrisey; citation_author=GU Lee; citation_author=CE O'Ferrall; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=18; citation_pages=3031-3039; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer.; citation_author=TR Hughes; citation_author=M Mao; citation_author=AR Jones; citation_author=J Burchard; citation_author=MJ Marton; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=19; citation_pages=342-347; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Photolithographic synthesis of high-density oligonucleotide arrays.; citation_author=GH McGall; citation_author=JA Fidanza; citation_journal_title=Methods Mol Biol; citation_volume=170; citation_number=20; citation_pages=71-101; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Determinants of sensitivity and specificity in spotted DNA microarrays with unmodified oligonucleotides.; citation_author=K Kucho; citation_author=H Yoneda; citation_author=M Harada; citation_author=M Ishiura; citation_journal_title=Genes Genet Syst; citation_volume=79; citation_number=21; citation_pages=189-197; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Assessing unmodified 70-mer oligonucleotide probe performance on glass-slide microarrays.; citation_author=HY Wang; citation_author=RL Malek; citation_author=AE Kwitek; citation_author=AS Greene; citation_author=TV Luu; citation_journal_title=Genome Biol; citation_volume=4; citation_number=22; citation_pages=R5; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fabrication of DNA microarrays on nanoengineered polymeric ultrathin film prepared by self-assembly of polyelectrolyte multilayers.; citation_author=X Zhou; citation_author=L Wu; citation_author=J Zhou; citation_journal_title=Langmuir; citation_volume=20; citation_number=23; citation_pages=8877-8885; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Construction of unmodified oligonucleotide-based microarrays in the thermophilic cyanobacterium Thermosynechococcus elongatus BP-1: screening of the candidates for circadianly expressed genes.; citation_author=K Kucho; citation_author=Y Tsuchiya; citation_author=Y Okumoto; citation_author=M Harada; citation_author=M Yamada; citation_journal_title=Genes Genet Syst; citation_volume=79; citation_number=24; citation_pages=319-329; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Methylation profiles of genes utilizing newly developed CpG island methylation microarray on colorectal cancer patients.; citation_author=N Kimura; citation_author=T Nagasaka; citation_author=J Murakami; citation_author=H Sasamoto; citation_author=M Murakami; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=25; citation_pages=e46; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Attachment of oligonucleotide probes to poly carbodiimide-coated glass for microarray applications.; citation_author=N Kimura; citation_author=R Oda; citation_author=Y Inaki; citation_author=O Suzuki; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=26; citation_pages=e68; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional profiling of the Saccharomyces cerevisiae genome.; citation_author=G Giaever; citation_author=AM Chu; citation_author=L Ni; citation_author=C Connelly; citation_author=L Riles; citation_journal_title=Nature; citation_volume=418; citation_number=27; citation_pages=387-391; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Systematic identification of pathways that couple cell growth and division in yeast.; citation_author=P Jorgensen; citation_author=JL Nishikawa; citation_author=BJ Breitkreutz; citation_author=M Tyers; citation_journal_title=Science; citation_volume=297; citation_number=28; citation_pages=395-400; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Characterization of synthetic DNA bar codes in Saccharomyces cerevisiae gene-deletion strains.; citation_author=RG Eason; citation_author=N Pourmand; citation_author=W Tongprasit; citation_author=ZS Herman; citation_author=K Anthony; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=29; citation_pages=11046-11051; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A DNA microarray-based genetic screen for nonhomologous end-joining mutants in Saccharomyces cerevisiae.; citation_author=SL Ooi; citation_author=DD Shoemaker; citation_author=JD Boeke; citation_journal_title=Science; citation_volume=294; citation_number=30; citation_pages=2552-2556; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=How cells coordinate growth and division.; citation_author=P Jorgensen; citation_author=M Tyers; citation_journal_title=Curr Biol; citation_volume=14; citation_number=31; citation_pages=R1014-1027; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Systematic screen for human disease genes in yeast.; citation_author=LM Steinmetz; citation_author=C Scharfe; citation_author=AM Deutschbauer; citation_author=D Mokranjac; citation_author=ZS Herman; citation_journal_title=Nat Genet; citation_volume=31; citation_number=32; citation_pages=400-404; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A genome-wide screen in Saccharomyces cerevisiae for genes affecting UV radiation sensitivity.; citation_author=GW Birrell; citation_author=G Giaever; citation_author=AM Chu; citation_author=RW Davis; citation_author=JM Brown; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=33; citation_pages=12608-12613; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Parallel phenotypic analysis of sporulation and postgermination growth in Saccharomyces cerevisiae.; citation_author=AM Deutschbauer; citation_author=RM Williams; citation_author=AM Chu; citation_author=RW Davis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=34; citation_pages=15530-15535; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A global view of pleiotropy and phenotypically derived gene function in yeast.; citation_author=AM Dudley; citation_author=DM Janse; citation_author=A Tanay; citation_author=R Shamir; citation_author=GM Church; citation_journal_title=Mol Syst Biol; citation_volume=1; citation_number=35; citation_pages=2005 0001; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Assessment of the sensitivity and specificity of oligonucleotide (50mer) microarrays.; citation_author=MD Kane; citation_author=TA Jatkoe; citation_author=CR Stumpf; citation_author=J Lu; citation_author=JD Thomas; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=36; citation_pages=4552-4557; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Detection and localization of single molecular recognition events using atomic force microscopy.; citation_author=P Hinterdorfer; citation_author=YF Dufrene; citation_journal_title=Nat Methods; citation_volume=3; citation_number=37; citation_pages=347-355; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Atomic force microscopy of long and short double-stranded, single-stranded and triple-stranded nucleic acids.; citation_author=HG Hansma; citation_author=I Revenko; citation_author=K Kim; citation_author=DE Laney; citation_journal_title=Nucleic Acids Res; citation_volume=24; citation_number=38; citation_pages=713-720; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Immobilization of oligonucleotides onto a glass support via disulfide bonds: A method for preparation of DNA microarrays.; citation_author=YH Rogers; citation_author=P Jiang-Baucom; citation_author=ZJ Huang; citation_author=V Bogdanov; citation_author=S Anderson; citation_journal_title=Anal Biochem; citation_volume=266; citation_number=39; citation_pages=23-30; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Improved statistical analysis of budding yeast TAG microarrays revealed by defined spike-in pools.; citation_author=BD Peyser; citation_author=RA Irizarry; citation_author=CW Tiffany; citation_author=O Chen; citation_author=DS Yuan; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=40; citation_pages=e140; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Preferred analysis methods for Affymetrix GeneChips revealed by a wholly defined control dataset.; citation_author=SE Choe; citation_author=M Boutros; citation_author=AM Michelson; citation_author=GM Church; citation_author=MS Halfon; citation_journal_title=Genome Biol; citation_volume=6; citation_number=41; citation_pages=R16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic scale mutant hunt identifies cell size homeostasis genes in S. cerevisiae.; citation_author=J Zhang; citation_author=C Schneider; citation_author=L Ottmers; citation_author=R Rodriguez; citation_author=A Day; citation_journal_title=Curr Biol; citation_volume=12; citation_number=42; citation_pages=1992-2001; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=RNA expression profiling at the single molecule level.; citation_author=J Hesse; citation_author=J Jacak; citation_author=M Kasper; citation_author=G Regl; citation_author=T Eichberger; citation_journal_title=Genome Res; citation_volume=16; citation_number=43; citation_pages=1041-1045; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays"/>
    <meta name="twitter:description" content="BackgroundMolecular barcode arrays provide a powerful means to analyze cellular phenotypes in parallel through detection of short (20&ndash;60 base) unique sequence tags, or &ldquo;barcodes&rdquo;, associated with each strain or clone in a collection. However, costs of current methods for microarray construction, whether by in situ oligonucleotide synthesis or ex situ coupling of modified oligonucleotides to the slide surface are often prohibitive to large-scale analyses.Methodology/Principal FindingsHere we demonstrate that unmodified 20mer oligonucleotide probes printed on conventional surfaces show comparable hybridization signals to covalently linked 5&prime;-amino-modified probes. As a test case, we undertook systematic cell size analysis of the budding yeast Saccharomyces cerevisiae genome-wide deletion collection by size separation of the deletion pool followed by determination of strain abundance in size fractions by barcode arrays. We demonstrate that the properties of a 13K unique feature spotted 20 mer oligonucleotide barcode microarray compare favorably with an analogous covalently-linked oligonucleotide array. Further, cell size profiles obtained with the size selection/barcode array approach recapitulate previous cell size measurements of individual deletion strains. Finally, through atomic force microscopy (AFM), we characterize the mechanism of hybridization to unmodified barcode probes on the slide surface.Conclusions/SignificanceThese studies push the lower limit of probe size in genome-scale unmodified oligonucleotide microarray construction and demonstrate a versatile, cost-effective and reliable method for molecular barcode analysis."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001546.g005"/>

  <meta property="og:title" content="Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4303'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5791'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3694&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001546">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001546" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001546&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael A. Cook, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada, 
                  Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chi-Kin Chan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Microarray Laboratory, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul Jorgensen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada, 
                  Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada
                </p>

                <p>
                    Current address: Department of Systems Biology, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Troy Ketela, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Terrence Donnelly Center for Cellular and Biomolecular Research, University of Toronto, 1Toronto, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel So, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Scenterra Inc., Bowie, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mike Tyers
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:tyers@mshri.on.ca">tyers@mshri.on.ca</a> (MT), <a href="mailto:cyho@mshri.on.ca">cyho@mshri.on.ca</a> (CYH)</p>

                <p>Affiliations:
                  Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada, 
                  Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chi-Yip Ho
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:tyers@mshri.on.ca">tyers@mshri.on.ca</a> (MT), <a href="mailto:cyho@mshri.on.ca">cyho@mshri.on.ca</a> (CYH)</p>

                <p>Affiliations:
                  Centre for Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada, 
                  Microarray Laboratory, Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 06, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001546</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001546-g001" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001546-g002" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001546-t001" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001546-g003" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001546-g004" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001546-g005" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001546-t002" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001546">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001546" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Molecular barcode arrays provide a powerful means to analyze cellular phenotypes in parallel through detection of short (20–60 base) unique sequence tags, or “barcodes”, associated with each strain or clone in a collection. However, costs of current methods for microarray construction, whether by <em>in situ</em> oligonucleotide synthesis or <em>ex situ</em> coupling of modified oligonucleotides to the slide surface are often prohibitive to large-scale analyses.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we demonstrate that unmodified 20mer oligonucleotide probes printed on conventional surfaces show comparable hybridization signals to covalently linked 5′-amino-modified probes. As a test case, we undertook systematic cell size analysis of the budding yeast <em>Saccharomyces cerevisiae</em> genome-wide deletion collection by size separation of the deletion pool followed by determination of strain abundance in size fractions by barcode arrays. We demonstrate that the properties of a 13K unique feature spotted 20 mer oligonucleotide barcode microarray compare favorably with an analogous covalently-linked oligonucleotide array. Further, cell size profiles obtained with the size selection/barcode array approach recapitulate previous cell size measurements of individual deletion strains. Finally, through atomic force microscopy (AFM), we characterize the mechanism of hybridization to unmodified barcode probes on the slide surface.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These studies push the lower limit of probe size in genome-scale unmodified oligonucleotide microarray construction and demonstrate a versatile, cost-effective and reliable method for molecular barcode analysis.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Cook MA, Chan C-K, Jorgensen P, Ketela T, So D, et al.  (2008) Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays. PLoS ONE 3(2):
          e1546.
            doi:10.1371/journal.pone.0001546</p><p><strong>Academic Editor: </strong>Hany El-Shemy, Cairo University, Egypt</p><p><strong>Received:</strong> August 10, 2007; <strong>Accepted:</strong> January 2, 2008; <strong>Published:</strong> February 6, 2008</p><p><strong>Copyright:</strong> © 2008 Cook et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants to Michael Tyers from the Canadian Institutes of Health Research (CIHR) and the Canada Foundation for Innovation. Mike Cook is the recipient of CIHR Doctoral Training Award and Michael Tyers holds a Canada Research Chair in Bioinformatics and Functional Genomics.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>DNA microarray technology has become a standard component in the toolbox of molecular biology. Microarrays have been applied to genome-wide analysis of gene expression, location of transcription factor binding sites (chromatin immunoprecipitation on microarray chip, ChIP-chip), DNA replication fork progression, sister chromatid cohesion, and nucleosome phasing <a href="#pone.0001546-Glynn1">[1]</a>–<a href="#pone.0001546-Wu1">[5]</a>. More recently, molecular barcode arrays have been used for phenotypic profiling, drug sensitivity and systematic synthetic lethal analysis <a href="#pone.0001546-Brown1">[6]</a>–<a href="#pone.0001546-StOnge1">[12]</a>. These microarray-based methods facilitate the prediction and definition of gene function, and have broad application in drug discovery and development.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Microarray technology relies on the hybridization of a labeled target sequence to a complementary cDNA or oligonucleotide probe immobilized on a glass surface. The method of deposition and immobilization varies depending upon the average length of the probe. For cDNA and long oligonucleotide sequences, probes produced <em>ex situ</em> are typically spotted onto a positively charged surface, such as poly-lysine or amino-silane, and are immobilized through UV cross-linking <a href="#pone.0001546-Call1">[13]</a>–<a href="#pone.0001546-Diehl1">[15]</a>. Covalent bond formation is thought to occur primarily through thymine bases in the DNA probes <a href="#pone.0001546-Ehrenreich1">[16]</a>, <a href="#pone.0001546-Saito1">[17]</a>. However, in the case of shorter oligonucleotides (15–60 mer), which possess a smaller complementary sequence over which to bind their cognate targets, probes are commonly synthesized with a 5′-chemically reactive linker <a href="#pone.0001546-Chrisey1">[18]</a>. The linker serves to introduce physical distance between the probe and the glass surface, thereby reducing steric hindrance during hybridization, and to allow covalent coupling of the probe to the derivatized surface of a slide via its reactive thiol or amino group, rather than an internal nucleotide base. In a more sophisticated approach, probes can be synthesized <em>in situ</em> on the array surface using ink-jet or light-directed oligonucleotide synthesizers, thereby bypassing the need for a secondary linkage reaction <a href="#pone.0001546-Hughes1">[19]</a>, <a href="#pone.0001546-McGall1">[20]</a>. The complications of <em>in situ</em> synthesis or <em>ex situ</em> derivatization of oligonucleotides add considerable expense to the fabrication process, particularly when hundreds of high-density microarrays are required.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>To ameliorate the cost of array fabrication, small unmodified oligonucleotides have been successfully spotted on conventional surfaces <a href="#pone.0001546-Call1">[13]</a>, <a href="#pone.0001546-Kucho1">[21]</a>, <a href="#pone.0001546-Wang1">[22]</a> or on surfaces modified for better adsorption of molecules <a href="#pone.0001546-Zhou1">[23]</a> on both trial and genome-wide scales <a href="#pone.0001546-Zhou1">[23]</a>, <a href="#pone.0001546-Kucho2">[24]</a>. However, under commonly used hybridization conditions, as probe size is reduced below ~40 bases, hybridization efficiencies have been shown to drop precipitously <a href="#pone.0001546-Call1">[13]</a>, <a href="#pone.0001546-Kucho1">[21]</a>. Recent small-scale application of reactive poly-carbodiimide surface substrates has enabled use of the smallest yet unmodified oligonucleotide probes (10–12 mer) <a href="#pone.0001546-Kimura1">[25]</a>, <a href="#pone.0001546-Kimura2">[26]</a>; however, the performance of this system on a genome-wide scale, with the corresponding large dynamic range of target abundances, hybridization efficiencies, and probe sequence compositions has yet to be ascertained.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Rather than applying newly introduced microarray surface substrates, we optimized a method to spot and hybridize unmodified short 20 mer oligonucleotide probes on conventional amino-silane based microarray surfaces. We applied the method to construct a 12,683 unique feature array that is complementary to the barcode tags of the budding yeast deletion strain collection. This collection, constructed by an international consortium, is composed of ~6000 individual yeast strains that bear precise null deletions of each known or predicted open reading frame (ORFs) <a href="#pone.0001546-Giaever1">[27]</a>. Each deletion construct in the collection is flanked by two 56 bp cassettes, which are comprised of universal primer sequences flanking a unique 20 mer DNA sequence identifier referred to as a barcode. The barcode tags enable genome-wide profiles of pooled populations to be assessed in a single experiment. In a typical experiment, DNA is extracted from the pooled population before and after selection, barcode sequences are amplified and differentially labeled, and the degree of enrichment or depletion of each strain in the selected population is determined by barcode microarray analysis.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Here, we demonstrate that short unmodified oligonucleotide probes spotted on Corning GAPS™II slides yield comparable signal intensities and signal-to-noise ratios (SNRs) to 5′-amino-modified covalently linked oligonucleotides. In a proof-of-concept application, we use a 13K feature barcode microarray to analyze the size distribution of the entire yeast deletion collection in a single experiment. By direct comparison of these experiments with previously obtained genome-scale cell size data <a href="#pone.0001546-Jorgensen1">[28]</a>, we provide rigorous biological validation of our unmodified oligonucleotide arrays and of the barcode approach to cell size determination. Further, as part of our analyses, we outline methodology to minimize false discovery and to define significant enrichment. We demonstrate that data obtained from short unmodified oligonucleotide arrays, while not equally precise as that obtained using inkjet-synthesized Agilent barcode arrays, are nonetheless specific, and as biologically accurate and comprehensive. Lastly, we use atomic force microscopy (AFM) to examine the arrangement of spotted 20 mer oligonucleotides, both before and after binding of their cognate target sequences. Through this we provide evidence as to the mechanism of target hybridization to unmodified oligonucleotide probes at a molecular level. The properties of short unmodified oligonucleotide arrays provide a considerable cost-saving alternative in barcode or short oligonucleotide DNA microarray fabrication, particularly amenable to large-scale in-house synthesis efforts (in our case, forgoing oligonucleotide modification confers an approximately 75% savings in synthesis costs; Supplementary <a href="#pone.0001546.s012">Table S1</a>). The savings imparted can, in turn, be passed on to other laboratories, providing greater access to barcode microarray technology.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Performance of amino-modified short oligonucleotide 20 mer probes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Prior to constructing whole genome yeast barcode arrays, we compared performance of arrays prepared by different methods. We designed a pilot array containing 5′-amino modified 20 mer probes for both UP and DOWN (hereafter “DN”) tags of 92 non-essential open reading frames (ORFs) from chromosome 2 (total of 184 unique probe sequences). To assess the effect of the 5′ modification reaction, we compared the hybridization properties of eight control probes lacking the 5′-modification versus their modified counterparts on two substrates that did or did not supply reactive groups for covalent bonding, namely SuperAldehyde® (Telechem) and GAPS™II (Corning), respectively. We prepared barcode PCR products from two pools of yeast deletion strains: barcode tags from the first pool (EUROSCARF Chr 2_1, comprised of 75 strains and their 150 universal barcodes tags), were amplified and labeled with Cy5 fluorescent primers, while barcode tags from the second pool (EUROSCARF Chr2_2, comprised of 76 deletion strains, of which only 15 ORFs and a total of 30 tags were represented on the array), were amplified and labeled with Cy3. The remaining four barcode probes of the pilot array served as negative controls that had no labeled targets. On both Superaldehyde® and GAPS™II substrates, the barcode features displayed either red (Cy5) or green (Cy3) but not yellow (mixed) colors, thus representing their cognate fluorescent signals without detectable cross-hybridization (<a href="#pone-0001546-g001">Figure 1A</a>). Overall signal intensities were comparable regardless of the substrate used. Negative controls showed only background intensities (dashed boxes). The differences in signal intensity between 5′-amino-modified (circled features) and unmodified control spots (boxed features) on the Superaldehyde® surface were indistinguishable.</p>
<div class="figure" id="pone-0001546-g001"><div class="img"><a name="pone-0001546-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Sensitivity and specificity of spotted 20mer barcode oligonucleotides.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A. Fluorescent images of hybridized pilot barcode arrays fabricated on Superaldehyde® (top; substrate permissive to covalent linkage of probes) and GAPS™II (bottom; substrate non-permissive to covalent linkage of probes) slides. 184 barcode oligonucleotides were printed as two sets of quadruplet features in each of two identical arrays (or “super-grids”; only the top super-grid is shown for each substrate). Pairs of amino-modified (white circles) and unmodified control probes (white squares) are shown linked by dotted lines. A defective modified barcode, excluded from further analysis, is noted (white asters). Negative controls are shown (white dashed box). All other unmarked features hybridized with barcodes from sub-populations of non-essential deletion strains of chromosome 2, labeled with either Cy5 (Chr2_1) or Cy3 (Chr2_2). B. Effect of 5′-amino-modification on oligonucleotide performance. Performance of each of seven different barcode oligonucleotide pairs (sequences are provided in a Supplemental GeneList file; List <a href="#pone.0001546.s015">Data S1</a>; the probe pairs correspond to YBL090W-DN, YBL091C-DN, YBL091C-UP, YBL093C-DN, YBL093C-UP, YBL094C-DN, YBL094C-UP, from left to right) is measured by log2 ratio of SNR(unmodified)/SNR(modified) on Superaldehyde® substrate, corrected by subtracting the log2 ratio of that on GAPS™II substrate; where SNR denotes signal to noise ratio of the probe [(median intensity Cy5 [or Cy3] - median background Cy5 [or Cy3])/standard deviation background Cy5 (or Cy3)]. Performance was determined based on hybridization to the cognate barcode sequences (barcode, from <a href="#pone-0001546-g001">Figure 1A</a>) or to a Cy3-labeled randomized 9mer probe. C. Specificity and signal intensity of hybridized barcodes in the absence of covalent linkage to substrate. The log2 ratio of the background subtracted Cy5/Cy3 channels (M) is plotted versus the average log2 value of the signal intensity in each channel (A) for 8 replicates of each barcode. Barcodes from sub-populations of non-essential deletion strains from chromosome 2 are labeled with Cy5 (Chr2_1; black [amino-modified] and gold [unmodified]) and Cy3 (Chr2_2; magenta). Negative control sequences are shown (red). Spots with background-subtracted intensities below zero for one channel fall along straight lines. Negative values were assigned an arbitrary floor value of 1 (log2 = 0) in calculations of A and M. (The fluorescent image of 4 replicates for each probe is shown in the bottom panel of <a href="#pone-0001546-g001">Figure 1A</a>.)</p>
<span>doi:10.1371/journal.pone.0001546.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To determine the effect of covalent linkage on hybridization efficiency, we compared the performance of unmodified versus modified 20 mer probes, as the ratio of the two fractions, namely [(signal of unmodified probe)/(signal of modified probe)] on Superaldehyde® over that of GAPS™II substrate. The latter fraction would reflect (and adjust for) differences due to idiosyncrasies in the printing and synthesis, including the quality and quantity of oligonucleotide synthesized and any positional effects during printing, as opposed to direct effects of covalent linkage. One probe pair, marked with white asters (<a href="#pone-0001546-g001">Figure 1A</a>), showed substantially lower hybridization signal for the modified probe even on the GAPS™II substrate, suggesting a problem during synthesis. Consistently, mass spectrometric (MS) analysis showed an aberrant mass in the case of the modified probe, but not of its unmodified counterpart (not shown). As such, we excluded this probe pair from our analysis. All other seven pairs of oligonucleotides had expected masses. To compare hybridization signals of different probes on different substrates, we used SNR instead of raw fluorescent intensities in calculations of their performances. To eliminate any bias caused by target sequence length, we determined the performance of probes in hybridization to both labeled 56 mer barcode amplicons and Cy3-labeled random 9 mers. The results of both of these analyses revealed that the unmodified probes yielded a signal on average of 0.7 fold of the corresponding modified probes (<a href="#pone-0001546-g001">Figure 1B</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Given the unexpectedly modest advantage conferred by 5′-modification and covalent linkage, we further characterized the performance of modified probes on GAPS™II slides. As expected, SNRs for modified and unmodified probes were approximately equal for analogous probe sequences (not shown). In a plot of log2 ratio (M) of the Cy5 and Cy3 signals versus the average log2 intensity (A) of the two channels (<a href="#pone-0001546-g001">Figure 1C</a>), we observed no significant cross-hybridization between barcode signals from the two sub-populations of deletion strains. Both sub-populations showed specific (|log2 ratio| &gt;2) signals for their respective labels, with the exception of 1 (of 8) replicate spots for each of two barcode probes; unmodified probes (gold circles) showed comparable specificity to their amino-modified counterparts. 83% of barcode tags displayed 3-fold higher signal than background; moreover, only one gene (~1% of probes) failed for both barcodes. This detection rate is unexpectedly high, given that re-sequencing has revealed that ~31% of all barcode tags are different from those originally designed, and approximately 25% have anomalous hybridization properties for at least one barcode <a href="#pone.0001546-Eason1">[29]</a>. When we examined the barcode probes listed as anomalous, we found that the majority actually had signal intensities comparable to correct barcodes, suggesting that performance may be improved by non-covalent array construction (<a href="#pone.0001546.s002">Figure S1</a>). In summary, from this pilot comparison, we concluded that non-covalently linked 20mer barcode probes spotted on a GAPS™II substrate perform at least as well as covalently coupled probes.</p>


<h4>A 13K genome-wide yeast barcode array</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Given the above results with a pilot barcode microarray, we constructed a <span class="underline">s</span>hort <span class="underline">u</span>nmodified <span class="underline">b</span>arcode probe <span class="underline">array</span> with 13K unique features (which hereafter is referred to as the “SUBarray”). The complete set of barcodes in the yeast deletion set was synthesized commercially at low cost (Illumina) and spotted in duplicate on GAPS™II slides (<a href="#pone-0001546-g002">Figure 2A</a>). In addition to the barcodes initially described in the yeast deletion project <a href="#pone.0001546-Giaever1">[27]</a>, we included 896 updated probe sequences to correct errors in the deletion cassettes themselves <a href="#pone.0001546-Eason1">[29]</a>. We also included 4 control pairs (2 UP tags and 2 DN tags) within each of the 48 printed blocks of probes (<a href="#pone-0001546-g002">Figure 2B, 2C</a>). These control probes were chosen for their consistent detection across a broad range of experimental conditions (not shown) and serve as a measure of consistent hybridization efficiency across the surface of the array.</p>
<div class="figure" id="pone-0001546-g002"><div class="img"><a name="pone-0001546-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Features of the 13K unique feature SUBarray.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A. A representative microarray from an experiment with the haploid yeast deletion set. Arrays are constructed of 48 blocks. B. Enlargement of a region overlapping two of the blocks. In white boxes are the four oligonucleotide controls that are present within every block. C. Logarithmic scale scatter plot of background subtracted intensity for Cy5 versus Cy3-dye for a representative barcode elutriation cell size experiment. Barcodes of strains not present within the experimental pool (black) and the four sets of positive controls (magenta) are overlayed on the barcodes of strains represented in the pool (blue). D. Overlap of strains represented by at least one significant barcode signal between Agilent covalently linked 20mer arrays and SUBarrays. Significant barcode signals are as defined in the text.</p>
<span>doi:10.1371/journal.pone.0001546.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Not surprisingly, given the novel construction of our array, we found that optimal performance required very different hybridization conditions than previously published for other covalent barcode microarray constructions <a href="#pone.0001546-Ooi2">[30]</a>. In particular, lowering the temperature to 25°C, changing the hybridization buffer (DIG Easy Hybe, Roche), and decreasing the hybridization volume by applying sample under a raised coverslip (LifterSlip, Erie Scientific) drastically increased signal intensities, lowered background, and reduced false signals (not shown).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To initially characterize the SUBarray, we performed an analysis of the complete haploid <em>MAT</em><strong><em>a</em></strong> yeast deletion pool. As approximately 20% of yeast genes are essential and absent from the haploid collection, the cohort of barcodes from essential genes allowed estimation of both false positive rate (all barcodes absent from pools) and true positive rate (all barcodes present in pools). Greater than 95% of barcode signals for essential gene deletions, and other negative controls, clustered in the low intensity end of a Cy5- versus Cy3-dye intensity scatter plot (<a href="#pone-0001546-g002">Figure 2C</a>, signal intensity for both channels &lt;200) of a representative barcode elutriation cell size experiment (detailed in the next subsection); elutriated sample was labeled with Cy5 versus control or pre-elutriated sample labeled with Cy3. We then used essential gene barcodes to develop a two-step filter for signal significance. In the first step, we applied an SNR threshold on an individual probe basis, choosing the threshold such that false positives were minimized and true positives were maximized. This point is represented by the 45° tangent to a receiver operating characteristic (ROC) curve, at which the rates of loss of false positive and true positive spots are equal (<a href="#pone.0001546.s003">Figure S2A</a>). Comparable thresholds were obtained when different measures of microarray quality were used (<a href="#pone.0001546.s003">Figure S2B</a>). In a second step, we assessed the number of times a given barcode was significant across multiple replicates and multiple arrays, again using the criteria of equal rates of false positive and true positive loss (<a href="#pone.0001546.s003">Figure S2C</a>). We reasoned that true positive barcode signals at near-noise levels should occur above background more frequently than random false positive signals. Using this filter, we recovered only 4.0% (89) of all possible false positive barcodes but captured 83.3% (7835) of all possible true positive barcodes (<a href="#pone-0001546-t001">Table 1</a>). Filtering according to the total number of barcode replicate spots detected across both UP and DN tags further improved resolution between false positive and true positive data (<a href="#pone.0001546.s003">Figure S2D</a>). By this criterion, we identified 5.0% (57) of all possible false positive deletion strains and 94.4% (4518) of all possible true positive deletion strains (<a href="#pone-0001546-t001">Table 1</a>). A detailed description of these methods is provided in the Supplementary material (Supplemental <a href="#pone.0001546.s001">Text S1</a>, Supplemental <a href="#pone.0001546.s013">Charts S1</a>). Finally, gene coverage on the SUBarray was comparable to and overlapped significantly with an Agilent array of covalently coupled barcode DNA oligonucleotides that represented the entire haploid deletion collection (<a href="#pone-0001546-t001">Table 1</a> and <a href="#pone-0001546-g002">Figure 2D</a>). A detailed description of this covalent barcode array is provided in the <a href="#s4">Materials and Methods</a> section. Across both technical and biological replicates, the SUBarray experiments yielded a Pearson's correlation coefficient of 0.7 (<a href="#pone-0001546-t001">Table 1</a>). The Agilent arrays we used displayed better correlation between replicate experiments for most barcodes than displayed by the SUBarrays (Supplemental <a href="#pone.0001546.s001">text S1</a>; <a href="#pone.0001546.s006">Figure S5</a>); however, due to the presence of a number of anti-correlated barcodes in these Agilent experiments, the correlation coefficient between dye-swap replicates had a similar value of 0.7 (<a href="#pone-0001546-t001">Table 1</a>). The average log2 transformed Cy5/Cy3 ratios for these anti-correlated barcodes across replicate arrays were approximately zero. Comparison of the average Z scores, meaning the standardized results of the log2 transformed Cy5/Cy3 ratios, across all replicate experiments (similar to the aforementioned cell size elutriation) between the SUBarray (n = 6) and the Agilent array (n = 4), yielded a correlation coefficient of 0.92. Characteristic scatter plots for SUBarrays and Agilent arrays, as well as a comparison between the two platforms, are provided in the Supplementary materials (<a href="#pone.0001546.s005">Figures S4</a>, <a href="#pone.0001546.s006">S5</a>, <a href="#pone.0001546.s007">S6</a>).</p>
<div class="figure" id="pone-0001546-t001"><div class="img"><a name="pone-0001546-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Coverage, specificity, and reproducibility of unmodified 20mer arrays.</span></strong></p><span>doi:10.1371/journal.pone.0001546.t001</span></div>

<h4>Application of short oligonucleotide barcode arrays to cell size control</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To prove SUBarrays in a biologically demanding application, we undertook comprehensive analysis of cell size across all viable haploid gene deletion strains. Our previous systematic strain-by-strain analysis of the entire cell size phenome <a href="#pone.0001546-Jorgensen1">[28]</a> provided a rigorous benchmark with which to assess the SUBarray microarray platform. To measure cell size in parallel on a genome-wide scale, we subjected pools of the haploid deletion collection grown in rich media to centrifugal elutriation. This technique physically separates cells on the basis of size; progressively increasing the flow rate of liquid through a continuous flow rotor in a direction opposite to the centrifugal force expels yeast cells of increasingly larger size from the chamber (<a href="#pone-0001546-g003">Figure 3B</a>). A series of elutriated yeast fractions were harvested and the barcodes derived from cultures obtained immediately before and after elutriation were differentially labeled and hybridized to barcode microarrays (<a href="#pone-0001546-g003">Figure 3A</a>). Deletion strains enriched after elutriation represent small (<em>whi</em>) mutants, while those depleted from the small elutriated populations represent large (<em>lge</em>) mutants (microarray data are deposited at ArrayExpress; E-MEXP-1200).</p>
<div class="figure" id="pone-0001546-g003"><div class="img"><a name="pone-0001546-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Barcode elutriation cell size experiment.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A. Schematic of the experiment. Elutriation of the haploid pool enriches for small cells. Genomic DNA is isolated from cell populations immediately before and immediately after elutriation, differentially labeled with either Cy5 or Cy3, and applied to a barcode microarray. B. Cell size distributions of a characteristic elutriation as determined by use of a Coulter Z2 particle analyzer (Beckman). Increasing the rate of flow through the elutriation rotor increases the median cell size of the elutriated fraction. C. Scatter plot of the average barcode Z score. Agilent (Y axis). SUBarray (X axis). Left. Overlay of barcodes of strains confirmed as either <em>lge</em> (black), <em>whi</em> (magenta), or wild-type (WT; red) by systematic experiments <a href="#pone.0001546-Jorgensen1">[28]</a> on all other barcodes (blue). Right. Overlay of barcodes of structural components of the cytosolic or mitochondrial ribosome (magenta) or respiration defective strains <a href="#pone.0001546-Steinmetz1">[32]</a> (black) on all other barcodes (blue). D. Venn diagrams of overlap between systematic and barcode data using SUBarrays or Agilent arrays. Cell size mutants are as defined in the text. E. Overlap between systematic and barcode data. Clustered cell size distributions are represented by horizontal bars color coded by intensity to reflect the shape of the distributions, with deletion strains on the vertical, and cell size on the horizontal axis. Barcodes, divided into UP (U) and DN (D) tags, with Z scores greater than 1 are represented as Lge (green bars) or Whi (red bars) adjacent to their cognate deletion strain for all experiments (E1, E2, E3) and all elutriation cuts (16, 21, 24 mL/min). SUBarrays and Agilent arrays are shown. The density of <em>lge</em> or <em>whi</em> barcodes (Z score&gt;1) over a 3 gene window is represented by horizontal bars color coded by intensity (&gt;10% brown; &gt;30% yellow; &gt;70% orange). Only barcodes with significant intensity/SNR were used in density calculations. Gene deletion strains with systematic mean cell sizes &gt;1 SD larger (green) or smaller (red) than average are shown. Systematic <em>lge</em> strains with broad distributions and unexpected cell size by barcode are indicated (dotted red box and asterisk).</p>
<span>doi:10.1371/journal.pone.0001546.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To compare the results of the original systematic cell size screen to our elutriation/barcode array experiments, we overlaid our set of known and high confidence <em>lge</em> and <em>whi</em> deletion strains (mean and median systematic cell size differing by &gt;1 standard deviation [SD] from genome-wide average), as well as our confirmed wild-type deletion strains (mean, median, and mode cell size differing by &lt;0.2 SD from genome-wide average) on a correlation plot of average barcode Z scores using SUBarray versus the Agilent arrays (<a href="#pone-0001546-g003">Figure 3C</a>, left). Approximately 92% of known <em>whi</em> strains fell in the upper right quadrant of the plot, i.e., these strains were enriched in both elutriated populations. Correspondingly, 76% of known <em>lge</em> strains fell in the lower left quadrant, i.e., these strains were depleted from the elutriated populations. As expected, barcodes that corresponded to strains of wild type size showed neither enrichment nor depletion.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Cell size is a complex biological readout that reflects contributions from cell morphology, bud size and distribution and the balance of growth and division <a href="#pone.0001546-Jorgensen2">[31]</a>. In particular, any mutation that compromises growth but not division will lead to a small cell size. For example, deletion of genes involved in respiration or in ribosome function confers a small cell size <a href="#pone.0001546-Jorgensen1">[28]</a>, <a href="#pone.0001546-Steinmetz1">[32]</a>. As expected, respiration defective strains and strains lacking structural components of the mitochondrial and cytosolic ribosome also clustered in the upper right quadrant of the scatter plot that reports small cell size (<a href="#pone-0001546-g003">Figure 3C</a>, right).</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To compare the individual cell size deletion strains identified by each method, we first defined two-step filters for significant enrichment or depletion by barcode. In the first step, Z scores for each array (typically in the range of 1.0–1.3) were chosen to exclude &gt;95% of the high confidence wild-type gene set. In the second step, we filtered putative <em>whi</em> or <em>lge</em> gene deletions according to the total number of times a replicate for either barcode tag was detected above the array-specific Z score thresholds. Using ROC curves, we again found that incorporating data from both tags and from all replicate spots gave the best results (<a href="#pone.0001546.s004">Figure S3A</a>). We further adjusted the threshold of the second step filter to limit dye-swap artifacts. A detailed description of this analysis is provided in the Supplementary material (Supplemental <a href="#pone.0001546.s001">Text S1</a> and Supplemental <a href="#pone.0001546.s013">Charts S1</a>). From ROC plots, we found that SUBarrays and Agilent arrays were comparable in their ability to distinguish size mutants from wild-type populations (<a href="#pone.0001546.s004">Figure S3B</a>), though we did observe a larger number of high intensity dye swap artifacts with Agilent arrays (<a href="#pone.0001546.s004">Figure S3C,D</a>).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Despite employing entirely different measures of cell size over hundreds of size mutants, i.e., size profiles of individual cultures measured by electrolyte displacement on a Coulter channelizer versus a population continuum of physical sizes separated by centrifugal elutriation, we observed a very substantial overlap between the high confidence systematic data and both sets of barcode data: ~52% of systematic phenotypes were reported in both sets of elutriation/barcode data and ~65% in at least one set. This is comparable to overlap of 60–90% previously reported between barcode analyses and other systematic phenotypic datasets, which typically cover 5- to 10-fold fewer mutant genes <a href="#pone.0001546-Ooi2">[30]</a>, <a href="#pone.0001546-Steinmetz1">[32]</a>–<a href="#pone.0001546-Dudley1">[35]</a>. Of the 189 cell size mutants identified only in the systematic data, 38 (~20%) had insufficient signals for both barcode tags, and 96 (~52%) showed the expected enrichment or depletion by barcode (mean and median Z score for either platform greater or less than zero, respectively), but were below the filtering thresholds employed. An even greater overlap in phenotypic profiles was recovered between the two different barcode array formats (~76% overlap; <a href="#pone-0001546-g003">Figure 3D</a>). 57% of deletion strains with cell size phenotypes detected by both elutriation/barcode array formats were absent from the high confidence systematic data set. These strains have moderate systematic size phenotypes, with average mean cell sizes 0.25 SD smaller and 0.3 SD larger than the population average, and thus represent <em>bona fide whi</em> and <em>lge</em> mutants, respectively (not shown).</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>We noted substantially greater overlap between the three methods for <em>whi</em> mutant strains as compared to <em>lge</em> mutants. This effect is likely due to the difference in the methods of cell size determination. Because elutriation selects for the smallest cells in the population, it is thus biased towards the positive identification of <em>whi</em> strains, whereas direct size analysis based on both mean and median of the size distribution is not biased in this fashion. Moreover, a <em>lge</em> deletion strain with a broad size distribution that includes small G1 daughter cells, even if most cells are large, would not be classified in the same manner by elutriation and systematic size analysis. This trend can be observed when the systematic cell size data is displayed as a heat map of cell size distributions and compared to <em>lge</em> and <em>whi</em> strains identified by barcode experiments (<a href="#pone-0001546-g003">Figure 3E</a>). While <em>whi</em> strains identified systematically (i.e., shifted left on the distribution; red bars in Syst. Mean column) consistently co-occurred with a high density of enriched barcodes (red bars), the <em>lge</em> phenotype was less consistent. In most cases, <em>lge</em> strains identified systematically (i.e., shifted right on the distribution; green bars in Syst. Mean column) co-occurred with a high density of depleted barcodes (green bars). However, in the case of the systematically defined <em>lge</em> deletion strains with broad distributions (boxed and marked with an aster), the density of <em>lge</em> barcode mutants detected was much lower, and many strains were actually detected as <em>whi</em>. Aside from this difference imparted by the method of enrichment, the barcode experiments appear remarkably consistent with systematic size determinations given the differences in methodology. The complete list of significant barcode values for Agilent and SUBarray replicate arrays is provided in the Supplementary materials (<a href="#pone.0001546.s015">Data S1</a>) as is supporting data for <a href="#pone-0001546-g003">Figure 3E</a> (<a href="#pone.0001546.s016">Data S2</a>).</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>The consistency between methods is also evident across Gene Ontology (GO; <a href="http://www.geneontology.org">http://www.geneontology.org</a>) component and process annotations (<a href="#pone-0001546-g004">Figure 4</a>). <em>Whi</em> genes were enriched (p&lt;0.01) for GO functions in growth related processes, including cytosolic or mitochondrial ribosome assembly and function, mitochondrial function, and metabolism. <em>Lge</em> genes were enriched (p&lt;0.01) for GO functions in cell cycle processes, including DNA replication and repair, chromatin structure, and the cytoskeleton. These observations are readily explained by shifts in the balance between growth and division: a decrease in growth rate without a compensating decrease in cell division rate leads to smaller size, whereas a converse delay in division in the absence of any attenuation in growth rate leads to larger size. Despite the large number of additional mutants recovered in the elutriation/barcode experiments, most GO categories were proportionately represented in each dataset.</p>
<div class="figure" id="pone-0001546-g004"><div class="img"><a name="pone-0001546-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Barcode elutriation and systematic cell size enrichment of gene ontology (GO) annotations.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Statistical enrichment from the haploid deletion set was calculated using cumulative hypergeometric probability functions (CDFs) with Bonferroni correction for either the high confidence systematic cell size data; SUBarray or Agilent array data; or the combined set of all <em>whi</em> or <em>lge</em> deletion strains identified. Significant enrichment for <em>whi</em> (red bars) or <em>lge</em> (green bars) strains is defined by [p*N&lt;0.01, where N is the number of categories per subset)]. GO component (N = 330); GO process (N = 1380); Morphology (N = 11) <a href="#pone.0001546-Giaever1">[27]</a>. GO categories are arranged by related function.</p>
<span>doi:10.1371/journal.pone.0001546.g004</span></div>

<h4>Atomic force microscopy (AFM) images of spotted probes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Given that short oligonucleotides are typically linked covalently to the glass surface of a slide, and as previous studies have shown loss of significant specific hybridization with less than 13 contiguous bases of homology <a href="#pone.0001546-Kane1">[36]</a>, we wondered how 20 mers on the surface (hereafter probe) were able to specifically hybridize with labeled barcode amplicons (hereafter target). To address this, we used AFM to image the arrangement of both complementary and non-complementary spotted probe sequences, with or without hybridization to a labeled target sequence, at the single molecule level (<a href="#pone-0001546-g005">Figure 5A</a>). AFM affords a 2 nano meter resolution on the horizontal axis and a better than 1 Angstrom (or 100 pico meter) resolution in the vertical axis <a href="#pone.0001546-Hinterdorfer1">[37]</a>. We scanned hybridized or mock-hybridized arrays for fluorescence prior to AFM analysis in order to choose an area of maximum hybridization. As expected, signal was significantly above background only in the presence of a complementary pair of target and probe sequences (<a href="#pone-0001546-g005">Figure 5A</a>; arrows mark the 1 µm×1 µm areas chosen for AFM scans).</p>
<div class="figure" id="pone-0001546-g005"><div class="img"><a name="pone-0001546-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Atomic force microscopy (AFM) of hybridized and mock hybridized spotted 20mer barcodes.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A. Fluorescent imaging of hybridized and mock hybridized slides. Each slide contained two spots of oligonucleotides complementary or non-complementary to the labeled target sequence. The location of the AFM scans is indicated by a red arrow. B. Mock hybridized barcode oligonucleotide. 1 µm×1 µm scan of the surface arrangement. Pixels are ~2 nm on the x and y axis, and &lt;100 pm on the z axis. C. High density non-complementary barcode oligonucleotide hybridized in the presence of its non-cognate target sequence. The location of a region of aberrant signal omitted from the analysis is indicated by a red arrow. D. Complementary barcode oligonucleotide hybridized in the presence of its cognate target sequence. E. Cumulative frequency of peak number with increasing height cut-off for high (solid blue) and low density (dashed blue) non-complementary and complementary barcode oligonucleotides minus (green) or plus (red) hybridized target. F. Cumulative frequency of peak number with increasing diameter cut-off for high density (solid blue) and low density (dashed blue) non-complementary and complementary barcode oligonucleotides minus (green) or plus (red) hybridized target. The method of estimation of peak dimensions is described in the Supplementary material (Supplemental <a href="#pone.0001546.s001">text S1</a>). The height and diameter at which 99% of mock hybridized peaks are excluded is indicated by a red dotted line.</p>
<span>doi:10.1371/journal.pone.0001546.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>When we examined a complementary probe-containing region, mock hybridized in the absence of target barcodes (<a href="#pone-0001546-g005">Figure 5B</a>), we observed peaks with mean diameters of approximately 8 nm and heights of 0.7 nm (<a href="#pone-0001546-t002">Table 2</a>). These results are similar to the arrangement and measurements previously reported for spotted single-stranded 50mer and 25mer DNA oligonucleotides <a href="#pone.0001546-Hansma1">[38]</a>. The characteristics of a low density (L.D.) region of spotted probe hybridized against a non-complementary target sequence were similar (<a href="#pone-0001546-t002">Table 2</a>; not shown). A high density (H.D.) region within the same probe spot also appeared similar (<a href="#pone-0001546-g005">Figure 5C</a>), though with a modest increase in mean peak heights and diameters (<a href="#pone-0001546-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001546-t002"><div class="img"><a name="pone-0001546-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001546" data-uri="info:doi/10.1371/journal.pone.0001546.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001546.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001546.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001546.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001546.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001546.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Peak attributes of atomic force microscopy measurements.</span></strong></p><span>doi:10.1371/journal.pone.0001546.t002</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>When we scanned a probe region hybridized in the presence of its complementary target sequence (<a href="#pone-0001546-g005">Figure 5D</a>), there was a marked increase in the number of larger height pixels relative to controls. However, despite an increase in peak dimensions, the majority of peaks still displayed similar properties to the mock-hybridized region, as observed in cumulative frequency plots for peak height (<a href="#pone-0001546-g005">Figure 5E</a>) and diameter (<a href="#pone-0001546-g005">Figure 5F</a>). This result implies that a large portion of the spotted probe sequence remained unbound by target. Peak diameter appeared to be the defining characteristic of hybridized regions: for thresholds that excluded 95% of peaks in each condition, all negative conditions displayed diameters within 2 nm of one another (16.6–17.9 nm), whereas hybridized complementary regions had a much larger threshold, at 21.0nm (<a href="#pone-0001546-t002">Table 2</a>, <a href="#pone.0001546.s008">Figure S7</a>). Applying an exclusion limit of 99% of the mock-hybridized condition (&gt;1.3 nm average height and &gt;20.2 nm diameter), which was chosen because it was the only condition where there was no possibility of target-probe hybridization, we estimated the percentage of bound probe at 3.1% in the hybridized condition versus &lt;0.77% in the negative control conditions (<a href="#pone-0001546-t002">Table 2</a>). We suspect that background signal in the absence of specific hybridization was in fact far lower than estimated, as when we examined defined probe-target peaks in the high density spotted probe regions, we found a high incidence of shouldered peaks that likely result from juxtaposed spotted oligonucleotide (<a href="#pone.0001546.s009">Figure S8</a>).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>To determine if our estimates of hybridized probe-target peaks were consistent with the observed fluorescent signals in the region scanned by AFM, we spotted a range of known concentrations of the Cy5-labeled primer used in target amplification on GAPS™II slides. From this, we generated a standard curve relating fluorescent signal and target molecule number per 1 µm<sup>2</sup> (<a href="#pone.0001546.s010">Figure S9</a>). Given the location of the 1 µm×1 µm AFM scan, we defined a range of possible signal intensities for this region, from 3000–6000 under the scan settings used. This corresponds to an expected 6-30 target molecules per 1 µm<sup>2</sup>, consistent with the 24 probe-target peaks estimated by AFM (<a href="#pone-0001546-t002">Table 2</a>, <a href="#pone.0001546.s010">Figure S9</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Terminal modification of short oligonucleotide probes for covalent coupling to slide surfaces has long been the accepted standard for fabrication of short oligonucleotide arrays <a href="#pone.0001546-Rogers1">[39]</a>. Yet surprisingly, we found that the absence of a 5′-amino linker did not overtly affect hybridization performance. Comparisons of unmodified and amino-linker containing probes, even on an uncharged and non-optimal surface for spotting of unmodified probes, SuperAldehyde®, showed only a moderate defect in performance. This defect was noted using previously published hybridization conditions <a href="#pone.0001546-Ooi2">[30]</a>, which we have improved substantially for our arrays. While published hybridization conditions were sufficient for our pilot study, application of our modified protocol is essential for optimal SUBarray performance. As a caveat to our approach, while these conditions did not negatively impact on barcode microarray specificity, the suitability of this approach to other microarray applications utilizing longer DNA or RNA targets would need to be assessed.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the context of a genome-wide experiment, SUBarrays showed comprehensive performance that was on par with Agilent covalently-linked 20mer arrays. Further, results from SUBarrays were sufficiently reproducible to ensure biological accuracy, though, likely due to higher background and the use of background subtracted intensities, displayed less precision than the Agilent platform, particularly for higher Z score values, where one signal was close to background (Supplementary <a href="#pone.0001546.s001">text S1</a>, <a href="#pone.0001546.s005">Figure S4</a>, <a href="#pone.0001546.s006">S5</a>). The cell size distribution across the haploid deletion pool represents a robust test case for barcode array analysis, as the results are largely invariant, even when one compares different elutriations performed on different days, and different elutriation fractions within the same experiment. Even though our comparison with arrays from Agilent was based on entirely different elutriations, we obtained a high degree of correlation between these two platforms (r = 0.92). Interestingly, we observed an increased frequency of dye-swap artifacts in the Agilent array experiments, even at high signal intensity and high log ratios, which were not observed with our SUBarray platform. These artifacts occurred in a common set of barcodes in two independent dye-swap experiments using different biological samples. Whether this observation represents a defect in a small subset of arrays, an issue with sub-ideal hybridization conditions, or a general trend in similar arrays, cannot be assessed without further analysis.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Because we analyzed the haploid deletion set, which excludes a significant fraction of barcodes from essential genes, we could define a large set of both false and true barcodes, thereby allowing us to optimize our filtering strategy. We found that a two step filtering approach could be applied to different biological replicates, and even different types of experiments (not shown), as long as the pool contained the same cohort of deletion strains. We found this method to be much better at reducing the number of false positives than a simple static intensity or SNR threshold. Similarly, by exploiting deletion strains known to possess wild-type characteristics during our size selection experiment, it allowed us to define appropriate Z score thresholds on the basis of exclusion of these strains from the list of cell size mutants. This method of excluding false negative signals is a viable alternative to applying static thresholds, trying to maximize a small number of true positive hits, or applying mixed variance models that derive their power from extensive replication of both control and experimental samples. Our strategy of false negative exclusion is particularly compatible with the use of strain or target/barcode controls that are spiked-in at known ratios prior to amplification or hybridization <a href="#pone.0001546-Peyser1">[40]</a>, <a href="#pone.0001546-Choe1">[41]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We applied the SUBarray platform to the problem of cell size control, and demonstrated a very substantial overlap between systematically determined sizes <a href="#pone.0001546-Jorgensen1">[28]</a> and those obtained by barcode, as well as a strong concordance in the types of genes isolated by each method. Since the systematic strain-by-strain approach to cell size determination on a genome-wide scale is very time consuming, initial studies of the cell size phenome were very limited in the number of conditions tested <a href="#pone.0001546-Jorgensen1">[28]</a>, <a href="#pone.0001546-Zhang1">[42]</a>. Population level analysis by barcode array profile enables many different conditions to be surveyed in rapid succession. We are currently applying the barcode approach to interrogation of cell size phenotypes in the context of different nutrient sources, ploidy conditions, genetic backgrounds and in the presence of various chemical reagents as well as ‘perturbagens’ to systematically decipher the global network that coordinates cell growth and division.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>An unexpected feature of the unmodified 20mer oligonucleotides is the ability to specifically hybridize to complementary sequences even after UV cross-linking to the slide surface. The UV cross-link is thought to occur predominantly through free radical generation on thymine bases and subsequent covalent bond formation <a href="#pone.0001546-Ehrenreich1">[16]</a>, <a href="#pone.0001546-Saito1">[17]</a>. If significant hybridization requires at least 13 contiguous bases of homology <a href="#pone.0001546-Kane1">[36]</a>, then one would predict an enrichment of thymidine-containing sequences in non-functional barcodes. However, we observed no significant difference in thymidine content between functional and non-functional barcodes, across the sequence, or at any internal position (not shown), similar to previous reports <a href="#pone.0001546-Kimura2">[26]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>While it is possible that hybridization occurs cooperatively in large DNA clusters, since the DNA peaks we observed by AFM are of comparable size to those of single stranded 25mer and 50mer sequences observed in previous AFM studies <a href="#pone.0001546-Hansma1">[38]</a>, it seems unlikely that each peak contains more than one target or probe sequence. Moreover, since each peak in the complementary hybridized condition is separated by an average of 35–40 nm (and each peak has an average diameter of 8 nm) compared to a peak diameter of ~20–25 nm in the case of bound probe, interactions between a single target molecule and multiple spotted probes would seem physically impossible. Further, our predictions of 6-30 bound target molecules based on correlation of fluorescent signals and target number closely match our estimates of bound peaks in our hybridized condition (24), suggesting that a single target sequence binds to each probe peak. That the peak diameter increases by two-fold upon hybridization is unsurprising given that the PCR products include 30 bases of primer sequence in addition to the barcode complement. However, while the height of each unhybridized peak is consistent with a single strand/base of DNA extending from the surface (0.7–0.8 nm), the peak diameter is larger than would hypothetically be required to contain a compact 20mer oligonucleotide with base-stacking interactions (7–9 nm observed versus 3–4 nm length predicted for an extended oligonucleotide). Similarly, hybridization results in a doubling of the diameter of each peak, which is more than would be necessary to contain a single probe and target molecule. This may be an issue of over-estimation of the peak diameters due to the insufficient resolution of AFM. However, we cannot eliminate the possibility that each peak contains more than one probe, and that either hybridization occurs through partial hybrids between a single target and multiple probes, or that multiple probe-target interactions occur within each peak.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>It could be that a substantial proportion of the spotted oligonucleotide is non-functional due to thymidine cross-linking, especially as only an estimated ~3% binding of target to the probe was observed by AFM. However, if the probe densities achieved on our microarrays are similar to those observed by AFM, based on our correlations of fluorescent intensity and target molecule number, we estimate that the most intense spots on our array achieve near saturation of target-probe binding, suggesting that the majority of spotted probe is functional.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In summary, arrays of unmodified 20mer oligonucleotide barcode arrays exhibit specific and reproducible hybridization behavior that enables the systematic dissection of complex phenotypes such as cell size. Given the excessively greater cost of modifying oligonucleotides with reactive linkers, and the relative time and cost of ink-jet synthesis, our results document a cost-saving alternative in array fabrication. The substantial savings in initial array construction are likely to be especially advantageous for microarray applications in which oligonucleotide quality control is the primary consideration. Given current constraints in single molecule detection <a href="#pone.0001546-Hesse1">[43]</a>, which can only be achieved under low density conditions, unmodified 20mer oligonucleotide arrays may also enable cost effective surveys of weakly expressed genes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Microarray fabrication</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Barcode probe sequences were as published <a href="#pone.0001546-Giaever1">[27]</a>, <a href="#pone.0001546-Eason1">[29]</a>. Array design and probe sequence information of the 13K v.2 Universal Barcode has been submitted to the European Bioinformatics Institute (EBI); array information can be accessed at ArrayExpress (E-MEXP-1200). Pilot barcode modified or unmodified 20 mer oligonucleotides were constructed according to standard 25 nmol scale protocols on a PolyPlex 96-well Oligonucleotide Synthesizer (GeneMachines/Genomic Solutions, Ann Arbor, MI). 5′-amino modification with C3 and C6 linker was used in synthesis of UP and DN barcodes, respectively. Probes used in pair-wise comparisons consist of the UP and DN barcodes for YBL090W, YBL091C, YBL093C, and YBL094C. Sequences are included in a GeneList file in the Supplemental material (List <a href="#pone.0001546.s015">Data S1</a>). Probes were characterized using mass spectrometry to ensure correct sequence and incorporation of the modified linker. Except for in-house oligonucleotides used in the pilot experiments, all oligonucleotides were synthesized by Illumina Inc (San Diego, CA).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Prior to printing, oligonucleotides were dissolved in Micro Spotting Solution Plus printing buffer (Telechem, Sunnyvale, CA) at 40 µM. Pilot microarrays were printed in quadruplet on all slides using BioRad VersArray ChipWriter Pro and SMP3 stealth pins (Telechem). Oligonucleotides spotted on SuperAldehyde® (Telechem) or GAPS™II (Corning, Corning, NY) were UV cross-linked at 200 mJ/cm<sup>2</sup>. Pilot arrays were sequentially washed twice in 0.1% SDS, three times in double deionized water (ddH<sub>2</sub>O, Millipore UltraPure), in boiling ddH<sub>2</sub>O for 2–3 minutes; and were dipped 5 times in 95% ethanol and spun dry. Arrays were stored in vacuum desiccators before use. The post–printing processing procedure for 13K feature barcode arrays on GAPS™II slides was carried out as described above, with the exception that slides were first immersed and washed in 1% (w/v) BSA (fraction V), 0.1% SDS, 3× SSC for 2 minutes with mild agitation. Quality control of printed DNA was performed by hybridizing arrays with a 7.5 µM Cy3-labeled random 9-mer (Operon, Huntsville, AL) in Hybridization Solution [4× SSC, 1 mg/ml poly-dA, 50 mM HEPES pH 7 (or Tris pH 7.5), 0.2% SDS]. Prior to hybridization, Cy3 9mer hybridization mix was heated to 85°C, cooled, loaded between the array surface and a LifterSlip (Erie Scientific, Portsmouth, NH), incubated at 25°C for 3–5 minutes, and washed sequentially in 2× SSC, 0.2% SDS; 2× SSC; and 0.2× SSC before spinning dry.</p>


<h4>Agilent covalently coupled barcode microarrays</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>“Agilent Custom Yeast Barcode, Version 1.0” (ArrayExpress accession: A-MEXP-842) was designed by Tim Hughes (Centre for Cellular and Biomolecular Research, University of Toronto) and was a gift from Charlie Boone (Centre for Cellular and Biomolecular Research, University of Toronto). This array was custom-made using Agilent proprietary inkjet technology and consists of one grid of 215×105 features. The array consists of 22575 features made up of 4771 unique UP and 4609 unique DOWN tag pairs in duplicate and triplicate for all non-essential yeast deletion strains according to Giaever et al. <a href="#pone.0001546-Giaever1">[27]</a> and Agilent proprietary positive and negative controls.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Essentially the Agilent Custom Yeast Barcode arrays differ from the in-house “SLRI_Yeast_Barcode_13k, Version 2.0” arrays by probe sequences and their surface substrates. The former consists of barcode probes with an additional stretch of 10 of T's (served as a spacer) at the 3′ end of the <em>in situ</em> synthesized covalently linked probes.</p>


<h4>Growth and cell-size selection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Haploid yeast deletion strains from the <em>MAT</em><strong><em>a</em></strong> deletion collection were grown as individual colonies on XY glucose solid medium (YEPD+100 mg/L adenine+200 mg/L tryptophan) containing 200 µg/ml G418, pooled, aliquoted, and frozen in XY containing 15% glycerol. This pool was used to inoculate all elutriation experiments. Approximately 1.5×10<sup>7</sup> cells from the pool were used to inoculate each of two 1L cultures in XY+2% glucose containing 100 µg/ml G418 for cell-size selection experiments. Log phase cultures were harvested at a cell density of 1-3×10<sup>7</sup> cells/mL and were loaded into a 40 mL JE-5.0 elutriation rotor in a J6-Mi centrifuge (Beckman, Fullerton, CA) at 16°C. Successive elutriation size fractions were obtained at the indicated flow rates and at a rotor speed of 2400 rpm. Genomic DNA was extracted from elutriated fractions and from samples of the pool culture taken immediately before elutriation.</p>


<h4>Barcode amplification, hybridization, and image analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>UP and DN universal barcode tags were amplified and fluorescently labeled in PCR reactions using the UP-tag [Primers U1 (<span class="gene">5′-GATGTCCACGAGGTCTCT</span>) and U2-Cy3 (<span class="gene">5′-Cy3-GTCGACCTGCAGCGTACG</span>)] and DOWN-tag [Primers D1 (<span class="gene">5′-CGGTGTCGGTCTCGTAG</span>) and D2-Cy3 (<span class="gene">5′-Cy3-CGAGCTCGAATTCATCGAT</span>)] for the control and UP-tag [Primers U1 and U2-Cy5 (<span class="gene">5′-Cy5-GTCGACCTGCAGCGTACG</span>)] and DOWN-tag [Primers D1 and D2-Cy5 (<span class="gene">5′-Cy5-CGAGCTCGAATTCATCGAT</span>)] for the experimental samples. Dye-swap experiments were performed with reciprocal labeling. All amplification primers were from Operon.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Briefly, 50 µL PCR reaction mixtures containing 1.5 mM MgCl<sub>2</sub>, 0.2 mM dNTPs, 100 ng yeast genomic DNA, 1 µM of the primer pair and 5 units of Taq polymerase (Invitrogen, Burlington, Ontario) was brought to 94°C for 3 minutes and subjected to 38 cycles [94°C, 30 s; 50°C, 30 s; 72°C, 30 s] and terminated at 72°C for 5 min. PCR reaction products were ethanol precipitated in 0.3 M sodium acetate (pH 5.2) in the presence of 5 µg of linear acrylamide (Ambion, Austin, TX) and a ten fold excess of blocking primers (U1, D1, U2block: <span class="gene">5′-CGTACGCTGCAGGTCGAC</span>, D2block: <span class="gene">5′-ATCGATGAATTCGAGCTCG</span>). Precipitated PCR products were dissolved in 5 µl of ddH<sub>2</sub>O and mixed into 60 µL of DIG Easy Hybridization (Roche, Laval, Quebec) solution. Hybridization targets were heated to 95°C, quick chilled on ice, and kept at 50°C covered from light until applied to arrays. Hybridization was performed at 25°C overnight (&gt;12 hours) under a LifterSlip (Erie Scientific). Hybridized arrays were washed sequentially at 30°C with 6× SSPE, 0.05% Triton X-100; 25°C with 2× SSPE, 0.05% Triton X-100; and 0.2× SSPE, 0.05% Triton X-100; and 0.2× SSPE before spinning dry. Hybridized unmodified oligonucleotide pilot and 13K arrays were imaged using a GenePix 4000B Array Scanner (Axon Instruments/Molecular Devices, Sunnyvale, CA). Samples for Agilent arrays and for the pilot barcode array were prepared and hybridized as described previously <a href="#pone.0001546-Ooi2">[30]</a>. Briefly, PCR was performed with Platinum PCR SuperMix (Invitrogen) as above for 35 cycles. PCR probes were heat denatured at 100°C for 1 minute. Hybridization was performed in 3.5 mL of 1× SSTE (1 M NaCl, 10 mM Tris.Cl, pH 7.5, 0.5% Triton X-100), containing 70 µl each of Cy3 and Cy5 labeled probes, and 1.3 µM of each blocking primer. Hybridization was performed at 40°C for 3 hours in a rotator hybridization oven in a heat sealable bag (Kapak Corporation, Minneapolis, MN). Hybridized arrays were washed sequentially at 42°C with 6× SSPE, 0.005% Triton X-100; 25°C with 2× SSPE, 0.005% Triton X-100; and 0.2× SSPE, 0.005% Triton X-100; and 0.2× SSPE before spinning dry. Agilent microarrays were scanned with a GSI Lumonics machine (Moorpark, CA). Initial scanning was used to assess print quality. All images were processed with GenePix Pro v.6 (Axon Instruments/Molecular Devices). Data were LOWESS normalized using Vector Xpression 3 (Invitrogen), and were subsequently analyzed in Excel (Microsoft, Redmond, WA) as described in the Supplementary material (Supplemental <a href="#pone.0001546.s001">text S1</a>).</p>


<h4>Atomic Force Microscopy (AFM)</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Complementary and non-complementary oligonucleotide probe sequences were <span class="gene">5′-TACTGAGCGGCATGTCACTG</span> (WHI5/YOR083W-UP) and <span class="gene">5′-CCAGTTCGGGAATGTGCTTC</span> (MBP1/YDL056W-UP). Probes were brought to 40 µM in 10 µl of Micro Spotting Solution Plus (Telechem), spotted on GAPS™II slides, air dried, and hybridized as above for 13K feature unmodified oligonucleotide arrays, with the exception that precipitation was performed in the absence of linear acrylamide and blocking primers. The Cy5 labeled targets (<span class="gene">5′-Cy5-GTCGACCTGCAGCGTACG-CAGTGACATGCCGC​TCAGTA-AGAGACCTCGTGGACATC-3′</span>) were generated by a PCR reaction using genomic DNA from the yeast deletion strain <em>whi5/yor083wΔ</em>. AFM experiments were carried out on a Digital Instruments Dimension 3000 (Model MPP-11100) in tapping mode, using the etched <em>Si</em> probes with tip diameter at 17 nm, and pyramidal shape and front, back, and both side angles of 15°, 25° and 17.5°, respectively (RTESP, NanoDevices/Veeco Probes, Camarillo, CA). Voltage output files were first processed with the software NanoScope III for initial roughness, grain size, and density analyses. Subsequent statistics of the peak heights and peak areas of hybridized and non-hybridized DNA oligonucleotide probes on the GAPS™II slides were calculated using Excel (Microsoft) and according to the header information of the voltage output files. The voltage to height (in nanometer) conversion was: HEIGHT = (Data_point * Full_data_range)/(2ˆ(8*Bytes/Pixel)), where Full_data_range = Z_scale * Z_scan_sensitivity. A custom PERL script was used to adjust local background height. Three-dimensional rendering and color surface presentations of AFM results (presented in the Supplemental <a href="#pone.0001546.s009">Figure S8</a>) were produced in MATLAB (Version R2006a, MathWorks, Natick, MA). A complete description of the analysis is provided in the Supplemental material (Supplemental <a href="#pone.0001546.s001">text S1</a>, <a href="#pone.0001546.s011">Figure S10</a>).</p>


<h4>Fluorescence calibration curves</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The concentration and percent Cy5-incorporation of each of two aliquots of the primer U2-Cy5 (5′-Cy5-GTCGACCTGCAGCGTACG) used in amplification of the target barcode for AFM were determined by absorbance at 260 nm and 650 nm, respectively. These primers were each used to create 5-fold dilution series in ddH<sub>2</sub>0. 1 µL of each dilution was spotted on a GAPS™II slide. Independent dilution series were generated and spotted on a duplicate slide. Both slides were scanned with comparable settings to the slides used in AFM.</p>


<h4>URLs</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p><a href="http://www.mshri.on.ca/microarray/">http://www.mshri.on.ca/microarray/</a> and <a href="http://www.mshri.on.ca/tyers/">http://www.mshri.on.ca/tyers/</a></p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001546"></div><a name="pone.0001546.s001" id="pone.0001546.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s001">Supplemental Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Detailed description of a) supplementary scatter plots of SUBarray and Agilent replicate arrays, and the methods for analysis of b) barcode microarray data and c) atomic force microscopy data are included in this supplemental text.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001546.s002" id="pone.0001546.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Analysis of previously defined anomalous barcodes <a href="#pone.0001546-Glynn1">[1]</a>. A. The log2 ratio of the Cy5/Cy3 (M) channels is plotted versus the average log2 value of the signal intensity in each channel (A) for 8 replicates of each barcode. Barcodes from sub-populations of non-essential deletion strains from chromosome 2 are labeled with Cy5 (Chr2_1; black) and Cy3 (Chr2_2; magenta). Negative control sequences are shown (red). Previously defined anomalous barcodes are plotted in gold. B. Frequency of the average log2 value of the signal intensity from each channel for anomalous and all other barcodes. Barcode values are an average of all replicates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.89 MB TIF)</p>
<a name="pone.0001546.s003" id="pone.0001546.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Use of ROC curves in defining intensity thresholds. A. True and false barcodes or genes are defined based on their presence or absence from the experimental pools, respectively. The 45{degree sign} tangent (dashed black) to the characteristic ROC plots (red, blue) is the point at which the rate of loss of false positives equals the rate of loss of true positives. B. Comparison of SNR thresholds defined by different measures of microarray data quality. Dotted line indicates the maximum obtained using true positive and false positive data (green) to set thresholds. Plots obtained from the measures of the average standard deviation (STDEV) between analogous spots on different arrays (blue) or the Pearson's correlation coefficient between arrays (red) begin to plateau at a similar threshold. A representative comparison between two dye-swap replicates is shown for all methods. C. Filtering according to the total number of barcodes with significant signal across multiple arrays (solid), rather than by the SNR from individual arrays (dashed) increases the ability to distinguish false and true data. Data is plotted for UP tags, but DN tag ROC plots are analogous. D. Filtering according to the total number of barcodes with significant data (Total; blue) yields slightly better data than using the fraction of total barcodes (% Total; green) or the maximum number of significant replicates for the best of the UP or DN barcode only (Max; red), or either the UP (black) or DN (black dashed line) tag data alone.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.49 MB TIF)</p>
<a name="pone.0001546.s004" id="pone.0001546.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Use of ROC plots in defining Z score thresholds for significant hits. A. True positive (either <em>whi</em> or <em>lge</em>) and false positive (wild-type) genes are defined based on systematically confirmed size characteristics. Individual array Z score thresholds (dashed black) are defined at the point that removes at least 95% of the false positive genes. Filtering by the total number of significant values across all arrays for both barcodes (green) yields better data than the fraction of possible hits (red), or the average (blue) or maximum (dashed black line) Z score for both UP (black) or DN (not shown) barcodes. ROC plots are shown for <em>whi</em> strains only. Data are analogous for <em>lge</em> strains. B. Comparison of the performance of optimized filtered data from Agilent (blue) or SUBarrays (red) for both <em>whi</em> (solid) and <em>lge</em> (dashed) strains. C. Dye-swap analysis from Agilent (left) and SUBarrays (right). Dye swap and technical replicate spots with consistent enrichment or depletion by elutriation (magenta) or those with any inconsistent values (blue) are plotted on a graph of the average absolute value of the log2 ratios (M) versus the average log2 value of the signal intensities (A). Barcodes with high intensities and high log ratios are the most consistent. Inconsistent high intensity, high log ratio barcode replicates represent dye swap artifacts and are more frequent in these Agilent arrays. Comparisons of Agilent and SUBarrays are between two and four replicate experiments, respectively. Consistent values agree for all experiments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(2.60 MB TIF)</p>
<a name="pone.0001546.s005" id="pone.0001546.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">(A–D) Scatter plots for the best (left) and worst (right) correlated SUBarray replicate arrays (as defined by Pearson's correlation coefficients) for technical (A), dye-swap (B), and intra- (C) or inter-experimental (D) replicate arrays. Z scores (as defined in the text) from on-chip replicate spots were averaged prior to generation of the scatter plot. Each experiment represents an independent set of PCR reactions. Dye-swap Z score values were multiplied by a factor of -1. All array data are derived from a log2 ratio of elutriated/pre-elutriated samples. Arrays are defined as: 592 (E2-21ml/min, log2[Cy5/Cy3]); 593 (E2-24ml/min, log2[Cy5/Cy3]); <em>597 (E2-21ml/min, log2[Cy3/Cy5])</em>; 806 (E2-21ml/min, log2[Cy5/Cy3]); <em>807 (E2-21ml/min, log2[Cy3/Cy5])</em>; 808 (E1-24ml/min, log2[Cy5/Cy3]). Dye-swap replicates are italicized.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(2.03 MB TIF)</p>
<a name="pone.0001546.s006" id="pone.0001546.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s006">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">(A–B) Scatter plots for the best (left) and worst (right) correlated Agilent replicate arrays (as defined by Pearson's correlation coefficients) for dye-swap (A) and intra-experimental (B) replicate arrays. An intra-experimental comparison between two arrays with the same labeling scheme is indicated by an asterisk and represents an estimate of technical replication; the average Pearson's correlation coefficient is listed in <a href="#pone-0001546-t001">Table 1</a> (Average r = 0.95). Comparisons were executed as in <a href="#pone.0001546.s005">Figure S4</a>. Arrays are defined as: FR16 (E3-16ml/min, log2[Cy5/Cy3]); <em>FF16 (E3-16ml/min, log2[Cy3/Cy5])</em>; FR24 (E3-24ml/min, log2[Cy5/Cy3]); <em>FF24 (E3-24ml/min, log2[Cy3/Cy5])</em>. Dye-swap replicates are italicized.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.38 MB TIF)</p>
<a name="pone.0001546.s007" id="pone.0001546.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s007">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Scatter plot of the average Z scores from Agilent (y-axis) and SUBarrays (x-axis). Z scores were averaged from all arrays (Agilent, n = 4; SUBarray, n = 6). Comparisons were executed as in <a href="#pone.0001546.s005">Figure S4</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0001546.s008" id="pone.0001546.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s008">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Scatter plot of the dimensions of all peaks (squares) identified by atomic force microscopy. 95% exclusion limits for each scan area are shown as colored diamonds. Peaks from a region lacking any probe (yellow) are less numerous (n = 103) and have significantly different properties than all other peaks.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.16 MB TIF)</p>
<a name="pone.0001546.s009" id="pone.0001546.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s009">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Background adjusted plots of the AFM scan regions. Data from original scans was adjusted to reduce background variation in the height of the glass surface. Peak height for each region is color coded to height. Both complementary probe minus target (A) and low density non-complementary probe plus target (B) show few large height peaks, while high density non-complementary probe plus target (C) and complementary probe plus target (D) show many more. However, the majority of the large height peaks in the high density non-complementary condition appear to have shoulders (E, white arrows), while the those in the complementary condition do not (F), suggesting that these peaks may be an artifact of high density, and represent two juxtaposed peaks. The region boxed in E has very different surface arrangement than any other peaks observed, is marked in <a href="#pone-0001546-g004">Figure 4C</a> as aberrant, and was omitted in all analyses.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(7.58 MB TIF)</p>
<a name="pone.0001546.s010" id="pone.0001546.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s010">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Fluorescence calibration curve for Cy5-target abundance. Replicate slides with known dilution series of one of two Cy5-labelled UP primers were scanned under comparable PMT and power settings to the AFM run. Mean signal intensities for each spot were plotted versus the total number of target molecules per total spot area (μm<sup>2</sup>). The potential range of signal intensities (3000–6000) for the scanned AFM region was used to estimate the theoretical target concentration (6-30; equivalent to hybridized probe-target peak number) in the scanned AFM region.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0001546.s011" id="pone.0001546.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s011">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Pixel and adjusted analysis of AFM data. A. Pixel distribution for a scanned region containing no probe (red). Shown for comparison is the left side of this distribution reflected across its modal point (blue dashed) B. Residual distribution of scan regions after removal of reflected background distributions. To remove an aberrant region containing abnormal peaks, histogram represents only half of the high density non-complementary probe containing region. C. Vertical strip of AFM scan before (blue) and after (green) removal of estimated background (brown). D. Pixel distribution before (red) and after (blue) background correction.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(1.53 MB TIF)</p>
<a name="pone.0001546.s012" id="pone.0001546.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s012">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Breakdown of microarray construction cost by item.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.12 MB TIF)</p>
<a name="pone.0001546.s013" id="pone.0001546.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s013">Supplemental Charts S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Flow charts describe the algorithms for 2-step filtering of ‘Intensity’(to generate present and absent calls) and ‘Z score’(to define cell size mutants).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001546.s014" id="pone.0001546.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s014">List Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">GeneList file of the GenePix ArrayList v1.0 format describes the microarray features of <a href="#pone-0001546-g001">Figure 1A</a>. The sequence of the barcode probe is used as the unique “ID” and the yeast gene name is used as the “Name” of the probe. The letter ‘X’ at the 5′ of the probe sequence denotes the C3 or C6 amino-linker; the ‘_UP’ suffix attached to the yeast gene name indicates the ‘UP’ barcode tag, where the absence of suffix indicates the ‘DN’ tag.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.07 MB TXT)</p>
<a name="pone.0001546.s015" id="pone.0001546.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s015">Data S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">All on-chip replicate averaged Z scores for Agilent and barcode experiments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(1.26 MB XLS)</p>
<a name="pone.0001546.s016" id="pone.0001546.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001546.s016">Data S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Numerical data used to generate <a href="#pone-0001546-g003">Figure 3E</a>. The yeast cell size cluster was derived from reference <a href="#pone.0001546-Jorgensen1">[28]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001546.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(6.45 MB ZIP)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Pavel Metalnikov for mass spectral analysis, Material Analysis Inc. (California) for technical assistance with Atomic Force Microscopy and Charlie Boone for providing Agilent barcode arrays.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MT CH MC PJ. Performed the experiments: MC CC PJ TK DS. Analyzed the data: CH MC CC DS. Contributed reagents/materials/analysis tools: MC CC TK DS. Wrote the paper: MT CH MC. Other: Oversaw design, execution and analyses of all experiments and wrote the manuscript: MT. Oversaw design, execution and analyses of all experiments and wrote the manuscript; Wrote custom PERL scripts used in this study: CH. Undertook microarray and AFM experiment design, yeast barcode cell-size selection experiments with our SUBarrays, pilot barcode experiment, all barcode and AFM data analyses, and wrote the manuscript: MC. Designed and fabricated all arrays, synthesized oligonucleotides used in the pilot barcode experiments, participated in data processing, including array data normalization: CC. Designed and executed a pilot yeast barcode cell-size selection experiment with Agilent arrays: PJ. Executed a pilot yeast barcode cell-size selection experiment with Agilent arrays: TK. Executed AFM scanning and interpreted initial AFM data and contributed to algorithm development for AFM analysis: DS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001546-Glynn1" id="pone.0001546-Glynn1"></a>Glynn EF, Megee PC, Yu HG, Mistrot C, Unal E, et al.  (2004) Genome-wide mapping of the cohesin complex in the yeast Saccharomyces cerevisiae. PLoS Biol  2: E259.  <ul class="find" data-citedArticleID="1050872" data-doi="10.1371/journal.pbio.0020259"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+mapping+of+the+cohesin+complex+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+mapping+of+the+cohesin+complex+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001546-Hoheisel1" id="pone.0001546-Hoheisel1"></a>Hoheisel JD (2006) Microarray technology: beyond transcript profiling and genotype analysis. Nat Rev Genet  7: 200–210.  <ul class="find" data-citedArticleID="1050880" data-doi="10.1038/nrg1809"><li><a href="http://dx.doi.org/10.1038/nrg1809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+technology%3A+beyond+transcript+profiling+and+genotype+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+technology%3A+beyond+transcript+profiling+and+genotype+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001546-Raghuraman1" id="pone.0001546-Raghuraman1"></a>Raghuraman MK, Winzeler EA, Collingwood D, Hunt S, Wodicka L, et al.  (2001) Replication dynamics of the yeast genome. Science  294: 115–121.  <ul class="find" data-citedArticleID="1050914" data-doi="10.1126/science.294.5540.115"><li><a href="http://dx.doi.org/10.1126/science.294.5540.115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Replication+dynamics+of+the+yeast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Replication+dynamics+of+the+yeast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001546-Shivaswamy1" id="pone.0001546-Shivaswamy1"></a>Shivaswamy S, Iyer VR (2007) Genome-wide analysis of chromatin status using tiling microarrays. Methods  41: 304–311.  <ul class="find" data-citedArticleID="1050920" data-doi="10.1016/j.ymeth.2006.11.002"><li><a href="http://dx.doi.org/10.1016/j.ymeth.2006.11.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+chromatin+status+using+tiling+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+chromatin+status+using+tiling+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001546-Wu1" id="pone.0001546-Wu1"></a>Wu J, Smith LT, Plass C, Huang TH (2006) ChIP-chip comes of age for genome-wide functional analysis. Cancer Res  66: 6899–6902.  <ul class="find" data-citedArticleID="1050928" data-doi="10.1158/0008-5472.can-06-0276"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-0276" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ChIP-chip+comes+of+age+for+genome-wide+functional+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ChIP-chip+comes+of+age+for+genome-wide+functional+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001546-Brown1" id="pone.0001546-Brown1"></a>Brown JA, Sherlock G, Myers CL, Burrows NM, Deng C, et al.  (2006) Global analysis of gene function in yeast by quantitative phenotypic profiling. Mol Syst Biol  2: 2006 0001.  <ul class="find" data-citedArticleID="1050850" data-doi="10.1038/msb4100043"><li><a href="http://dx.doi.org/10.1038/msb4100043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+gene+function+in+yeast+by+quantitative+phenotypic+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+gene+function+in+yeast+by+quantitative+phenotypic+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001546-Lee1" id="pone.0001546-Lee1"></a>Lee W, St Onge RP, Proctor M, Flaherty P, Jordan MI, et al.  (2005) Genome-wide requirements for resistance to functionally distinct DNA-damaging agents. PLoS Genet  1: e24.  <ul class="find" data-citedArticleID="1050898" data-doi="10.1371/journal.pgen.0010024"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+requirements+for+resistance+to+functionally+distinct+DNA-damaging+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+requirements+for+resistance+to+functionally+distinct+DNA-damaging+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001546-Lum1" id="pone.0001546-Lum1"></a>Lum PY, Armour CD, Stepaniants SB, Cavet G, Wolf MK, et al.  (2004) Discovering modes of action for therapeutic compounds using a genome-wide screen of yeast heterozygotes. Cell  116: 121–137.  <ul class="find" data-citedArticleID="1050900" data-doi="10.1016/s0092-8674(03)01035-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)01035-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovering+modes+of+action+for+therapeutic+compounds+using+a+genome-wide+screen+of+yeast+heterozygotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovering+modes+of+action+for+therapeutic+compounds+using+a+genome-wide+screen+of+yeast+heterozygotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001546-Ooi1" id="pone.0001546-Ooi1"></a>Ooi SL, Pan X, Peyser BD, Ye P, Meluh PB, et al.  (2006) Global synthetic-lethality analysis and yeast functional profiling. Trends Genet  22: 56–63.  <ul class="find" data-citedArticleID="1050904" data-doi="10.1016/j.tig.2005.11.003"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.11.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+synthetic-lethality+analysis+and+yeast+functional+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+synthetic-lethality+analysis+and+yeast+functional+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001546-Paddison1" id="pone.0001546-Paddison1"></a>Paddison PJ, Silva JM, Conklin DS, Schlabach M, Li M, et al.  (2004) A resource for large-scale RNA-interference-based screens in mammals. Nature  428: 427–431.  <ul class="find" data-citedArticleID="1050908" data-doi="10.1038/nature02370"><li><a href="http://dx.doi.org/10.1038/nature02370" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+resource+for+large-scale+RNA-interference-based+screens+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+resource+for+large-scale+RNA-interference-based+screens+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001546-Pan1" id="pone.0001546-Pan1"></a>Pan X, Yuan DS, Ooi SL, Wang X, Sookhai-Mahadeo S, et al.  (2007) dSLAM analysis of genome-wide genetic interactions in Saccharomyces cerevisiae. Methods  41: 206–221.  <ul class="find" data-citedArticleID="1050910" data-doi="10.1016/j.ymeth.2006.07.033"><li><a href="http://dx.doi.org/10.1016/j.ymeth.2006.07.033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=dSLAM+analysis+of+genome-wide+genetic+interactions+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22dSLAM+analysis+of+genome-wide+genetic+interactions+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001546-StOnge1" id="pone.0001546-StOnge1"></a>St Onge RP, Mani R, Oh J, Proctor M, Fung E, et al.  (2007) Systematic pathway analysis using high-resolution fitness profiling of combinatorial gene deletions. Nat Genet  39: 199–206.  <ul class="find" data-citedArticleID="1050922" data-doi="10.1038/ng1948"><li><a href="http://dx.doi.org/10.1038/ng1948" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+pathway+analysis+using+high-resolution+fitness+profiling+of+combinatorial+gene+deletions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+pathway+analysis+using+high-resolution+fitness+profiling+of+combinatorial+gene+deletions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001546-Call1" id="pone.0001546-Call1"></a>Call DR, Chandler DP, Brockman F (2001) Fabrication of DNA microarrays using unmodified oligonucleotide probes. Biotechniques  30: 368–372, 374, 376 passim.  <ul class="find" data-citedArticleID="1050852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fabrication+of+DNA+microarrays+using+unmodified+oligonucleotide+probes.&amp;auth=&amp;atitle=Fabrication+of+DNA+microarrays+using+unmodified+oligonucleotide+probes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fabrication+of+DNA+microarrays+using+unmodified+oligonucleotide+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fabrication+of+DNA+microarrays+using+unmodified+oligonucleotide+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001546-Cheung1" id="pone.0001546-Cheung1"></a>Cheung VG, Morley M, Aguilar F, Massimi A, Kucherlapati R, et al.  (1999) Making and reading microarrays. Nat Genet  21: 15–19.  <ul class="find" data-citedArticleID="1050854" data-doi="10.1038/4439"><li><a href="http://dx.doi.org/10.1038/4439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Making+and+reading+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Making+and+reading+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001546-Diehl1" id="pone.0001546-Diehl1"></a>Diehl F, Grahlmann S, Beier M, Hoheisel JD (2001) Manufacturing DNA microarrays of high spot homogeneity and reduced background signal. Nucleic Acids Res  29: E38.  <ul class="find" data-citedArticleID="1050862" data-doi="10.1093/nar/29.7.e38"><li><a href="http://dx.doi.org/10.1093/nar/29.7.e38" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Manufacturing+DNA+microarrays+of+high+spot+homogeneity+and+reduced+background+signal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Manufacturing+DNA+microarrays+of+high+spot+homogeneity+and+reduced+background+signal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001546-Ehrenreich1" id="pone.0001546-Ehrenreich1"></a>Ehrenreich A (2006) DNA microarray technology for the microbiologist: an overview. Appl Microbiol Biotechnol  73: 255–273.  <ul class="find" data-citedArticleID="1050868" data-doi="10.1007/s00253-006-0584-2"><li><a href="http://dx.doi.org/10.1007/s00253-006-0584-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarray+technology+for+the+microbiologist%3A+an+overview." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarray+technology+for+the+microbiologist%3A+an+overview.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001546-Saito1" id="pone.0001546-Saito1"></a>Saito I, Sugiyama H, Furukawa N, Matsuura T (1981) Photoreaction of thymidine with primary amines. Application to specific modification of DNA. Nucleic Acids Symp Ser 61–64.  <ul class="find" data-citedArticleID="1050918"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Photoreaction+of+thymidine+with+primary+amines.+Application+to+specific+modification+of+DNA.&amp;auth=&amp;atitle=Photoreaction+of+thymidine+with+primary+amines.+Application+to+specific+modification+of+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Photoreaction+of+thymidine+with+primary+amines.+Application+to+specific+modification+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Photoreaction+of+thymidine+with+primary+amines.+Application+to+specific+modification+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001546-Chrisey1" id="pone.0001546-Chrisey1"></a>Chrisey LA, Lee GU, O'Ferrall CE (1996) Covalent attachment of synthetic DNA to self-assembled monolayer films. Nucleic Acids Res  24: 3031–3039.  <ul class="find" data-citedArticleID="1050858" data-doi="10.1093/nar/24.15.3031"><li><a href="http://dx.doi.org/10.1093/nar/24.15.3031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covalent+attachment+of+synthetic+DNA+to+self-assembled+monolayer+films." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covalent+attachment+of+synthetic+DNA+to+self-assembled+monolayer+films.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001546-Hughes1" id="pone.0001546-Hughes1"></a>Hughes TR, Mao M, Jones AR, Burchard J, Marton MJ, et al.  (2001) Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer. Nat Biotechnol  19: 342–347.  <ul class="find" data-citedArticleID="1050882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer.&amp;auth=&amp;atitle=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001546-McGall1" id="pone.0001546-McGall1"></a>McGall GH, Fidanza JA (2001) Photolithographic synthesis of high-density oligonucleotide arrays. Methods Mol Biol  170: 71–101.  <ul class="find" data-citedArticleID="1050902" data-doi="10.1385/1-59259-234-1:71"><li><a href="http://dx.doi.org/10.1385/1-59259-234-1:71" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Photolithographic+synthesis+of+high-density+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Photolithographic+synthesis+of+high-density+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001546-Kucho1" id="pone.0001546-Kucho1"></a>Kucho K, Yoneda H, Harada M, Ishiura M (2004) Determinants of sensitivity and specificity in spotted DNA microarrays with unmodified oligonucleotides. Genes Genet Syst  79: 189–197.  <ul class="find" data-citedArticleID="1050894" data-doi="10.1266/ggs.79.189"><li><a href="http://dx.doi.org/10.1266/ggs.79.189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determinants+of+sensitivity+and+specificity+in+spotted+DNA+microarrays+with+unmodified+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determinants+of+sensitivity+and+specificity+in+spotted+DNA+microarrays+with+unmodified+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001546-Wang1" id="pone.0001546-Wang1"></a>Wang HY, Malek RL, Kwitek AE, Greene AS, Luu TV, et al.  (2003) Assessing unmodified 70-mer oligonucleotide probe performance on glass-slide microarrays. Genome Biol  4: R5.  <ul class="find" data-citedArticleID="1050926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Assessing+unmodified+70-mer+oligonucleotide+probe+performance+on+glass-slide+microarrays.&amp;auth=&amp;atitle=Assessing+unmodified+70-mer+oligonucleotide+probe+performance+on+glass-slide+microarrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+unmodified+70-mer+oligonucleotide+probe+performance+on+glass-slide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+unmodified+70-mer+oligonucleotide+probe+performance+on+glass-slide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001546-Zhou1" id="pone.0001546-Zhou1"></a>Zhou X, Wu L, Zhou J (2004) Fabrication of DNA microarrays on nanoengineered polymeric ultrathin film prepared by self-assembly of polyelectrolyte multilayers. Langmuir  20: 8877–8885.  <ul class="find" data-citedArticleID="1050932" data-doi="10.1021/la048950b"><li><a href="http://dx.doi.org/10.1021/la048950b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fabrication+of+DNA+microarrays+on+nanoengineered+polymeric+ultrathin+film+prepared+by+self-assembly+of+polyelectrolyte+multilayers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fabrication+of+DNA+microarrays+on+nanoengineered+polymeric+ultrathin+film+prepared+by+self-assembly+of+polyelectrolyte+multilayers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001546-Kucho2" id="pone.0001546-Kucho2"></a>Kucho K, Tsuchiya Y, Okumoto Y, Harada M, Yamada M, et al.  (2004) Construction of unmodified oligonucleotide-based microarrays in the thermophilic cyanobacterium Thermosynechococcus elongatus BP-1: screening of the candidates for circadianly expressed genes. Genes Genet Syst  79: 319–329.  <ul class="find" data-citedArticleID="1050896" data-doi="10.1266/ggs.79.319"><li><a href="http://dx.doi.org/10.1266/ggs.79.319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+unmodified+oligonucleotide-based+microarrays+in+the+thermophilic+cyanobacterium+Thermosynechococcus+elongatus+BP-1%3A+screening+of+the+candidates+for+circadianly+expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+unmodified+oligonucleotide-based+microarrays+in+the+thermophilic+cyanobacterium+Thermosynechococcus+elongatus+BP-1%3A+screening+of+the+candidates+for+circadianly+expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001546-Kimura1" id="pone.0001546-Kimura1"></a>Kimura N, Nagasaka T, Murakami J, Sasamoto H, Murakami M, et al.  (2005) Methylation profiles of genes utilizing newly developed CpG island methylation microarray on colorectal cancer patients. Nucleic Acids Res  33: e46.  <ul class="find" data-citedArticleID="1050890" data-doi="10.1093/nar/gni046"><li><a href="http://dx.doi.org/10.1093/nar/gni046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation+profiles+of+genes+utilizing+newly+developed+CpG+island+methylation+microarray+on+colorectal+cancer+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation+profiles+of+genes+utilizing+newly+developed+CpG+island+methylation+microarray+on+colorectal+cancer+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001546-Kimura2" id="pone.0001546-Kimura2"></a>Kimura N, Oda R, Inaki Y, Suzuki O (2004) Attachment of oligonucleotide probes to poly carbodiimide-coated glass for microarray applications. Nucleic Acids Res  32: e68.  <ul class="find" data-citedArticleID="1050892" data-doi="10.1093/nar/gnh057"><li><a href="http://dx.doi.org/10.1093/nar/gnh057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Attachment+of+oligonucleotide+probes+to+poly+carbodiimide-coated+glass+for+microarray+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Attachment+of+oligonucleotide+probes+to+poly+carbodiimide-coated+glass+for+microarray+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001546-Giaever1" id="pone.0001546-Giaever1"></a>Giaever G, Chu AM, Ni L, Connelly C, Riles L, et al.  (2002) Functional profiling of the Saccharomyces cerevisiae genome. Nature  418: 387–391.  <ul class="find" data-citedArticleID="1050870" data-doi="10.1038/nature00935"><li><a href="http://dx.doi.org/10.1038/nature00935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+profiling+of+the+Saccharomyces+cerevisiae+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+profiling+of+the+Saccharomyces+cerevisiae+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001546-Jorgensen1" id="pone.0001546-Jorgensen1"></a>Jorgensen P, Nishikawa JL, Breitkreutz BJ, Tyers M (2002) Systematic identification of pathways that couple cell growth and division in yeast. Science  297: 395–400.  <ul class="find" data-citedArticleID="1050884" data-doi="10.1126/science.1070850"><li><a href="http://dx.doi.org/10.1126/science.1070850" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+identification+of+pathways+that+couple+cell+growth+and+division+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+identification+of+pathways+that+couple+cell+growth+and+division+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001546-Eason1" id="pone.0001546-Eason1"></a>Eason RG, Pourmand N, Tongprasit W, Herman ZS, Anthony K, et al.  (2004) Characterization of synthetic DNA bar codes in Saccharomyces cerevisiae gene-deletion strains. Proc Natl Acad Sci U S A  101: 11046–11051.  <ul class="find" data-citedArticleID="1050866" data-doi="10.1073/pnas.0403672101"><li><a href="http://dx.doi.org/10.1073/pnas.0403672101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+synthetic+DNA+bar+codes+in+Saccharomyces+cerevisiae+gene-deletion+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+synthetic+DNA+bar+codes+in+Saccharomyces+cerevisiae+gene-deletion+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001546-Ooi2" id="pone.0001546-Ooi2"></a>Ooi SL, Shoemaker DD, Boeke JD (2001) A DNA microarray-based genetic screen for nonhomologous end-joining mutants in Saccharomyces cerevisiae. Science  294: 2552–2556.  <ul class="find" data-citedArticleID="1050906" data-doi="10.1126/science.1065672"><li><a href="http://dx.doi.org/10.1126/science.1065672" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+DNA+microarray-based+genetic+screen+for+nonhomologous+end-joining+mutants+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+DNA+microarray-based+genetic+screen+for+nonhomologous+end-joining+mutants+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001546-Jorgensen2" id="pone.0001546-Jorgensen2"></a>Jorgensen P, Tyers M (2004) How cells coordinate growth and division. Curr Biol  14: R1014–1027.  <ul class="find" data-citedArticleID="1050886" data-doi="10.1016/j.cub.2004.11.027"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.11.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+cells+coordinate+growth+and+division." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+cells+coordinate+growth+and+division.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001546-Steinmetz1" id="pone.0001546-Steinmetz1"></a>Steinmetz LM, Scharfe C, Deutschbauer AM, Mokranjac D, Herman ZS, et al.  (2002) Systematic screen for human disease genes in yeast. Nat Genet  31: 400–404.  <ul class="find" data-citedArticleID="1050924" data-doi="10.1038/ng929"><li><a href="http://dx.doi.org/10.1038/ng929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+screen+for+human+disease+genes+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+screen+for+human+disease+genes+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001546-Birrell1" id="pone.0001546-Birrell1"></a>Birrell GW, Giaever G, Chu AM, Davis RW, Brown JM (2001) A genome-wide screen in Saccharomyces cerevisiae for genes affecting UV radiation sensitivity. Proc Natl Acad Sci U S A  98: 12608–12613.  <ul class="find" data-citedArticleID="1050848" data-doi="10.1073/pnas.231366398"><li><a href="http://dx.doi.org/10.1073/pnas.231366398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+screen+in+Saccharomyces+cerevisiae+for+genes+affecting+UV+radiation+sensitivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+screen+in+Saccharomyces+cerevisiae+for+genes+affecting+UV+radiation+sensitivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001546-Deutschbauer1" id="pone.0001546-Deutschbauer1"></a>Deutschbauer AM, Williams RM, Chu AM, Davis RW (2002) Parallel phenotypic analysis of sporulation and postgermination growth in Saccharomyces cerevisiae. Proc Natl Acad Sci U S A  99: 15530–15535.  <ul class="find" data-citedArticleID="1050860" data-doi="10.1073/pnas.202604399"><li><a href="http://dx.doi.org/10.1073/pnas.202604399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+phenotypic+analysis+of+sporulation+and+postgermination+growth+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+phenotypic+analysis+of+sporulation+and+postgermination+growth+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001546-Dudley1" id="pone.0001546-Dudley1"></a>Dudley AM, Janse DM, Tanay A, Shamir R, Church GM (2005) A global view of pleiotropy and phenotypically derived gene function in yeast. Mol Syst Biol  1: 2005 0001.  <ul class="find" data-citedArticleID="1050864" data-doi="10.1038/msb4100004"><li><a href="http://dx.doi.org/10.1038/msb4100004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+view+of+pleiotropy+and+phenotypically+derived+gene+function+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+view+of+pleiotropy+and+phenotypically+derived+gene+function+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001546-Kane1" id="pone.0001546-Kane1"></a>Kane MD, Jatkoe TA, Stumpf CR, Lu J, Thomas JD, et al.  (2000) Assessment of the sensitivity and specificity of oligonucleotide (50mer) microarrays. Nucleic Acids Res  28: 4552–4557.  <ul class="find" data-citedArticleID="1050888" data-doi="10.1093/nar/28.22.4552"><li><a href="http://dx.doi.org/10.1093/nar/28.22.4552" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+the+sensitivity+and+specificity+of+oligonucleotide+%2850mer%29+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+the+sensitivity+and+specificity+of+oligonucleotide+%2850mer%29+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001546-Hinterdorfer1" id="pone.0001546-Hinterdorfer1"></a>Hinterdorfer P, Dufrene YF (2006) Detection and localization of single molecular recognition events using atomic force microscopy. Nat Methods  3: 347–355.  <ul class="find" data-citedArticleID="1050878" data-doi="10.1038/nmeth871"><li><a href="http://dx.doi.org/10.1038/nmeth871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+localization+of+single+molecular+recognition+events+using+atomic+force+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+localization+of+single+molecular+recognition+events+using+atomic+force+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001546-Hansma1" id="pone.0001546-Hansma1"></a>Hansma HG, Revenko I, Kim K, Laney DE (1996) Atomic force microscopy of long and short double-stranded, single-stranded and triple-stranded nucleic acids. Nucleic Acids Res  24: 713–720.  <ul class="find" data-citedArticleID="1050874" data-doi="10.1093/nar/24.4.713"><li><a href="http://dx.doi.org/10.1093/nar/24.4.713" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atomic+force+microscopy+of+long+and+short+double-stranded%2C+single-stranded+and+triple-stranded+nucleic+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atomic+force+microscopy+of+long+and+short+double-stranded%2C+single-stranded+and+triple-stranded+nucleic+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001546-Rogers1" id="pone.0001546-Rogers1"></a>Rogers YH, Jiang-Baucom P, Huang ZJ, Bogdanov V, Anderson S, et al.  (1999) Immobilization of oligonucleotides onto a glass support via disulfide bonds: A method for preparation of DNA microarrays. Anal Biochem  266: 23–30.  <ul class="find" data-citedArticleID="1050916" data-doi="10.1006/abio.1998.2857"><li><a href="http://dx.doi.org/10.1006/abio.1998.2857" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immobilization+of+oligonucleotides+onto+a+glass+support+via+disulfide+bonds%3A+A+method+for+preparation+of+DNA+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immobilization+of+oligonucleotides+onto+a+glass+support+via+disulfide+bonds%3A+A+method+for+preparation+of+DNA+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001546-Peyser1" id="pone.0001546-Peyser1"></a>Peyser BD, Irizarry RA, Tiffany CW, Chen O, Yuan DS, et al.  (2005) Improved statistical analysis of budding yeast TAG microarrays revealed by defined spike-in pools. Nucleic Acids Res  33: e140.  <ul class="find" data-citedArticleID="1050912" data-doi="10.1093/nar/gni138"><li><a href="http://dx.doi.org/10.1093/nar/gni138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+statistical+analysis+of+budding+yeast+TAG+microarrays+revealed+by+defined+spike-in+pools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+statistical+analysis+of+budding+yeast+TAG+microarrays+revealed+by+defined+spike-in+pools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001546-Choe1" id="pone.0001546-Choe1"></a>Choe SE, Boutros M, Michelson AM, Church GM, Halfon MS (2005) Preferred analysis methods for Affymetrix GeneChips revealed by a wholly defined control dataset. Genome Biol  6: R16.  <ul class="find" data-citedArticleID="1050856" data-doi="10.1186/gb-2005-6-2-r16"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-2-r16" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preferred+analysis+methods+for+Affymetrix+GeneChips+revealed+by+a+wholly+defined+control+dataset." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preferred+analysis+methods+for+Affymetrix+GeneChips+revealed+by+a+wholly+defined+control+dataset.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001546-Zhang1" id="pone.0001546-Zhang1"></a>Zhang J, Schneider C, Ottmers L, Rodriguez R, Day A, et al.  (2002) Genomic scale mutant hunt identifies cell size homeostasis genes in S. cerevisiae. Curr Biol  12: 1992–2001.  <ul class="find" data-citedArticleID="1050930" data-doi="10.1016/s0960-9822(02)01305-2"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)01305-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+scale+mutant+hunt+identifies+cell+size+homeostasis+genes+in+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+scale+mutant+hunt+identifies+cell+size+homeostasis+genes+in+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001546-Hesse1" id="pone.0001546-Hesse1"></a>Hesse J, Jacak J, Kasper M, Regl G, Eichberger T, et al.  (2006) RNA expression profiling at the single molecule level. Genome Res  16: 1041–1045.  <ul class="find" data-citedArticleID="1050876" data-doi="10.1101/gr.4999906"><li><a href="http://dx.doi.org/10.1101/gr.4999906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+expression+profiling+at+the+single+molecule+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+expression+profiling+at+the+single+molecule+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.XML" value="138928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.PDF" value="737142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g001.PNG_L" value="4943890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g001.PNG_M" value="223699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g001.PNG_S" value="14296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g001.TIF" value="7002144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g001.PNG_I" value="71899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g002.PNG_L" value="7226462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g002.PNG_M" value="299642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g002.PNG_S" value="11941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g002.TIF" value="9964620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g002.PNG_I" value="94619"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t001.PNG_L" value="193140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t001.PNG_M" value="201663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t001.PNG_S" value="21232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t001.TIF" value="791068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t001.PNG_I" value="102984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g003.PNG_L" value="1915326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g003.PNG_M" value="190481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g003.PNG_S" value="12916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g003.TIF" value="3579558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g003.PNG_I" value="105348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g004.PNG_L" value="316876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g004.PNG_M" value="73964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g004.PNG_S" value="8824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g004.TIF" value="973438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g004.PNG_I" value="28814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g005.PNG_L" value="3197812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g005.PNG_M" value="188053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g005.PNG_S" value="15668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g005.TIF" value="4659044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.g005.PNG_I" value="100495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t002.PNG_L" value="161946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t002.PNG_M" value="91449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t002.PNG_S" value="11242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t002.TIF" value="751996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.t002.PNG_I" value="28968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s001.DOC" value="57344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s002.TIF" value="884574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s003.TIF" value="1485960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s004.TIF" value="2591994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s005.TIF" value="2028604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s006.TIF" value="1373338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s007.TIF" value="628782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s008.TIF" value="505012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s009.TIF" value="7819610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s010.TIF" value="389381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s011.TIF" value="2384644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s012.TIF" value="678261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s013.DOC" value="64000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s014.TXT" value="69273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s015.XLS" value="1264640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001546.s016.ZIP" value="6454710"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001546&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001546" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001546&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001546&volume=&issue=&title=Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays&author_name=Michael%20A.%20Cook%2C%20Chi-Kin%20Chan%2C%20Paul%20Jorgensen%2C%20Troy%20Ketela%2C%20Daniel%20So%2C%20Mike%20Tyers%2C%20Chi-Yip%20Ho&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001546" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546&amp;t=Systematic%20Validation%20and%20Atomic%20Force%20Microscopy%20of%20Non-Covalent%20Short%20Oligonucleotide%20Barcode%20Microarrays" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546&title=Systematic%20Validation%20and%20Atomic%20Force%20Microscopy%20of%20Non-Covalent%20Short%20Oligonucleotide%20Barcode%20Microarrays&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546&amp;title=Systematic%20Validation%20and%20Atomic%20Force%20Microscopy%20of%20Non-Covalent%20Short%20Oligonucleotide%20Barcode%20Microarrays" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001546&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Systematic Validation and Atomic Force Microscopy of Non-Covalent Short Oligonucleotide Barcode Microarrays';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001546';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Systematic%20Validation%20and%20Atomic%20Force%20Microscopy%20of%20Non-Covalent%20Short%20Oligonucleotide%20Barcode%20Microarrays http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001546" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001546" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Atomic+force+microscopy%22" title="Search for articles in the subject area:'Atomic force microscopy'"><div class="flagText">Atomic force microscopy</div></a>
              <div data-categoryid="28769" data-articleid="25918"
                   data-categoryname="Atomic force microscopy"
                   class="flagImage" title="Flag 'Atomic force microscopy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25918"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+pool%22" title="Search for articles in the subject area:'Gene pool'"><div class="flagText">Gene pool</div></a>
              <div data-categoryid="35161" data-articleid="25918"
                   data-categoryname="Gene pool"
                   class="flagImage" title="Flag 'Gene pool' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25918"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="25918"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="19653" data-articleid="25918"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signal+filtering%22" title="Search for articles in the subject area:'Signal filtering'"><div class="flagText">Signal filtering</div></a>
              <div data-categoryid="20157" data-articleid="25918"
                   data-categoryname="Signal filtering"
                   class="flagImage" title="Flag 'Signal filtering' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="25918"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4697'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5151'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6626&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=3317">Referee comments: Referee 3 (Patrick Brown)</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=1899">Referee comments: Referee 3 (Patrick Brown)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
