

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Gene Characterization Index: Assessing the Depth of Gene Annotation</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001440"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001440" />

    <meta name="citation_title" content="Gene Characterization Index: Assessing the Depth of Gene Annotation"/>
    <meta itemprop="name" content="Gene Characterization Index: Assessing the Depth of Gene Annotation"/>

      <meta name="citation_author" content="Danielle Kemmer"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden"/>
            <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of British Columbia, Vancouver, Canada"/>
      <meta name="citation_author" content="Raf M. Podowski"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden"/>
      <meta name="citation_author" content="Dimas Yusuf"/>
            <meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada"/>
      <meta name="citation_author" content="Jochen Brumm"/>
            <meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of British Columbia, Vancouver, Canada"/>
      <meta name="citation_author" content="Warren Cheung"/>
            <meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada"/>
      <meta name="citation_author" content="Claes Wahlestedt"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden"/>
            <meta name="citation_author_institution" content="Molecular and Integrative Neurosciences Department, The Scripps Research Institute, Jupiter, Florida, United States of America"/>
      <meta name="citation_author" content="Boris Lenhard"/>
            <meta name="citation_author_institution" content="Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden"/>
            <meta name="citation_author_institution" content="Computational Biology Unit, Bergen Center for Computational Science, Sars International Centre for Marine Molecular Biology, Unifob AS, University of Bergen, Bergen, Norway"/>
      <meta name="citation_author" content="Wyeth W. Wasserman"/>
            <meta name="citation_author_institution" content="Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada"/>

    <meta name="citation_date" content="2008/1/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001440.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1440"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project.; citation_author=E Birney; citation_author=JA Stamatoyannopoulos; citation_author=A Dutta; citation_author=R Guigo; citation_author=TR Gingeras; citation_journal_title=Nature; citation_volume=447; citation_number=1; citation_pages=799-816; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Mouse Phenome Project.; citation_author=MA Bogue; citation_author=SC Grubb; citation_journal_title=Genetica; citation_volume=122; citation_number=2; citation_pages=71-74; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rat Phenome Project: the untapped potential of existing rat strains.; citation_author=T Mashimo; citation_author=B Voigt; citation_author=T Kuramoto; citation_author=T Serikawa; citation_journal_title=J Appl Physiol; citation_volume=98; citation_number=3; citation_pages=371-379; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Toward improving Caenorhabditis elegans phenome mapping with an ORFeome-based RNAi library.; citation_author=JF Rual; citation_author=J Ceron; citation_author=J Koreth; citation_author=T Hao; citation_author=AS Nicot; citation_journal_title=Genome Res; citation_volume=14; citation_number=4; citation_pages=2162-2168; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A golden age of brain exploration.; citation_author=V Gewin; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=5; citation_pages=e24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Facial attractiveness: beauty and the machine.; citation_author=Y Eisenthal; citation_author=G Dror; citation_author=E Ruppin; citation_journal_title=Neural Comput; citation_volume=18; citation_number=6; citation_pages=119-142; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Automated classification of female facial beauty by image analysis and supervised learning; 2004; San Jose, CA, USA.; citation_author=H Gunes; citation_author=M Piccardi; citation_author=T Jan; citation_number=7; citation_pages=968-978; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=8; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Entrez Gene: gene-centered information at NCBI.; citation_author=D Maglott; citation_author=J Ostell; citation_author=KD Pruitt; citation_author=T Tatusova; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=9; citation_pages=D26-31; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=GeneLynx: a gene-centric portal to the human genome.; citation_author=B Lenhard; citation_author=WS Hayes; citation_author=WW Wasserman; citation_journal_title=Genome Res; citation_volume=11; citation_number=10; citation_pages=2151-2157; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=11; citation_pages=D21-25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Nature of Statistical Learning Theory: NY Springer.; citation_author=V Vapnik; citation_number=12; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Multivariate adaptive regression splines.; citation_author=JH Friedman; citation_journal_title=The Annals of Statistics; citation_volume=19; citation_number=13; citation_pages=1-141; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=New developments in the InterPro database.; citation_author=NJ Mulder; citation_author=R Apweiler; citation_author=TK Attwood; citation_author=A Bairoch; citation_author=A Bateman; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=14; citation_pages=D224-228; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=PRINTS and its automatic supplement, prePRINTS.; citation_author=TK Attwood; citation_author=P Bradley; citation_author=DR Flower; citation_author=A Gaulton; citation_author=N Maudling; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=15; citation_pages=400-402; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The PROSITE database.; citation_author=N Hulo; citation_author=A Bairoch; citation_author=V Bulliard; citation_author=L Cerutti; citation_author=E De Castro; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=16; citation_pages=D227-230; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=From genomics to chemical genomics: new developments in KEGG.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=M Hattori; citation_author=KF Aoki-Kinoshita; citation_author=M Itoh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=17; citation_pages=D354-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=18; citation_pages=D61-65; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt): an expanding universe of protein information.; citation_author=CH Wu; citation_author=R Apweiler; citation_author=A Bairoch; citation_author=DA Natale; citation_author=WC Barker; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=19; citation_pages=D187-191; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=20; citation_pages=235-242; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The Gene Ontology Annotation (GOA) Database: sharing knowledge in Uniprot with Gene Ontology.; citation_author=E Camon; citation_author=M Magrane; citation_author=D Barrell; citation_author=V Lee; citation_author=E Dimmer; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=21; citation_pages=D262-266; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Using Information Content to Evaluate Semantic Similarity in a Taxonomy; 1995.; citation_author=P Resnik; citation_number=22; citation_pages=448-453; " />
      <meta name="citation_reference" content="citation_title=How many drug targets are there?; citation_author=JP Overington; citation_author=B Al-Lazikani; citation_author=AL Hopkins; citation_journal_title=Nat Rev Drug Discov; citation_volume=5; citation_number=23; citation_pages=993-996; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DrugBank: a comprehensive resource for in silico drug discovery and exploration.; citation_author=DS Wishart; citation_author=C Knox; citation_author=AC Guo; citation_author=S Shrivastava; citation_author=M Hassanali; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=24; citation_pages=D668-672; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=GPCRDB information system for G protein-coupled receptors.; citation_author=F Horn; citation_author=E Bettler; citation_author=L Oliveira; citation_author=F Campagne; citation_author=FE Cohen; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=25; citation_pages=294-297; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transcriptome analysis of human gastric cancer.; citation_author=JH Oh; citation_author=JO Yang; citation_author=Y Hahn; citation_author=MR Kim; citation_author=SS Byun; citation_journal_title=Mamm Genome; citation_volume=16; citation_number=26; citation_pages=942-954; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Collecting and harvesting biological data: the GPCRDB and NucleaRDB information systems.; citation_author=F Horn; citation_author=G Vriend; citation_author=FE Cohen; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=27; citation_pages=346-349; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The flip side: Identifying small molecule regulators of nuclear receptors.; citation_author=IG Schulman; citation_author=RA Heyman; citation_journal_title=Chem Biol; citation_volume=11; citation_number=28; citation_pages=639-646; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Cys-loop superfamily of ligand-gated ion channels: the impact of receptor structure on function.; citation_author=CN Connolly; citation_author=KA Wafford; citation_journal_title=Biochem Soc Trans; citation_volume=32; citation_number=29; citation_pages=529-534; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=LGICdb: a manually curated sequence database after the genomes.; citation_author=M Donizelli; citation_author=MA Djite; citation_author=N Le Novere; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=30; citation_pages=D267-269; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The role of GABAA receptor biogenesis, structure and function in epilepsy.; citation_author=S Mizielinska; citation_author=S Greenwood; citation_author=CN Connolly; citation_journal_title=Biochem Soc Trans; citation_volume=34; citation_number=31; citation_pages=863-867; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Potassium ion channels and human disease: phenotypes to drug targets?; citation_author=ME Curran; citation_journal_title=Curr Opin Biotechnol; citation_volume=9; citation_number=32; citation_pages=565-572; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genome research: fulfilling the public's expectations for knowledge and commercialization.; citation_author=RG Adler; citation_journal_title=Science; citation_volume=257; citation_number=33; citation_pages=908-914; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Are DNA patents bad for medicine?; citation_author=DB Resnik; citation_journal_title=Health Policy; citation_volume=65; citation_number=34; citation_pages=181-197; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Can patents deter innovation? The anticommons in biomedical research.; citation_author=MA Heller; citation_author=RS Eisenberg; citation_journal_title=Science; citation_volume=280; citation_number=35; citation_pages=698-701; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Human genome. Reaching their goal early, sequencing labs celebrate.; citation_author=E Pennisi; citation_journal_title=Science; citation_volume=300; citation_number=36; citation_pages=409; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Human genome. A low number wins the GeneSweep Pool.; citation_author=E Pennisi; citation_journal_title=Science; citation_volume=300; citation_number=37; citation_pages=1484; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Gene Ontology (GO) project in 2006.; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=38; citation_pages=D322-326; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=GOSim–an R-package for computation of information theoretic GO similarities between terms and gene products.; citation_author=H Frohlich; citation_author=N Speer; citation_author=A Poustka; citation_author=T Beissbarth; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=39; citation_pages=166; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2007.; citation_author=TJ Hubbard; citation_author=BL Aken; citation_author=K Beal; citation_author=B Ballester; citation_author=M Caccamo; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=40; citation_pages=D610-617; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Gene Characterization Index: Assessing the Depth of Gene Annotation"/>
    <meta name="twitter:description" content="BackgroundWe introduce the Gene Characterization Index, a bioinformatics method for scoring the extent to which a protein-encoding gene is functionally described. Inherently a reflection of human perception, the Gene Characterization Index is applied for assessing the characterization status of individual genes, thus serving the advancement of both genome annotation and applied genomics research by rapid and unbiased identification of groups of uncharacterized genes for diverse applications such as directed functional studies and delineation of novel drug targets.Methodology/Principal FindingsThe scoring procedure is based on a global survey of researchers, who assigned characterization scores from 1 (poor) to 10 (extensive) for a sample of genes based on major online resources. By evaluating the survey as training data, we developed a bioinformatics procedure to assign gene characterization scores to all genes in the human genome. We analyzed snapshots of functional genome annotation over a period of 6 years to assess temporal changes reflected by the increase of the average Gene Characterization Index. Applying the Gene Characterization Index to genes within pharmaceutically relevant classes, we confirmed known drug targets as high-scoring genes and revealed potentially interesting novel targets with low characterization indexes. Removing known drug targets and genes linked to sequence-related patent filings from the entirety of indexed genes, we identified sets of low-scoring genes particularly suited for further experimental investigation.Conclusions/SignificanceThe Gene Characterization Index is intended to serve as a tool to the scientific community and granting agencies for focusing resources and efforts on unexplored areas of the genome. The Gene Characterization Index is available from http://cisreg.ca/gci/."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001440.g006"/>

  <meta property="og:title" content="Gene Characterization Index: Assessing the Depth of Gene Annotation" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1004'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4562'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5968&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001440">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001440" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001440&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Gene Characterization Index: Assessing the Depth of Gene Annotation
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Danielle Kemmer
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Danielle Kemmer, Raf M. Podowski</p>

              
              

                <p>Affiliations:
                  Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden, 
                  Department of Biochemistry and Molecular Biology, University of British Columbia, Vancouver, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Raf M. Podowski
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Danielle Kemmer, Raf M. Podowski</p>

              
              

                <p>Affiliation:
                  Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dimas Yusuf, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jochen Brumm, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada, 
                  Department of Statistics, University of British Columbia, Vancouver, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Warren Cheung, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claes Wahlestedt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden, 
                  Molecular and Integrative Neurosciences Department, The Scripps Research Institute, Jupiter, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Boris Lenhard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Genomics and Bioinformatics, Karolinska Institute, Stockholm, Sweden, 
                  Computational Biology Unit, Bergen Center for Computational Science, Sars International Centre for Marine Molecular Biology, Unifob AS, University of Bergen, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wyeth W. Wasserman
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:wyeth@cmmt.ubc.ca">wyeth@cmmt.ubc.ca</a></p>

                <p>Affiliation:
                  Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Department of Medical Genetics, University of British Columbia, Vancouver, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001440</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001440-t001" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-t002" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-g001" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-g002" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-t003" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-t004" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-t005" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-g003" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-g004" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-g005" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-t006" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001440-g006" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001440">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001440" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>We introduce the Gene Characterization Index, a bioinformatics method for scoring the extent to which a protein-encoding gene is functionally described. Inherently a reflection of human perception, the Gene Characterization Index is applied for assessing the characterization status of individual genes, thus serving the advancement of both genome annotation and applied genomics research by rapid and unbiased identification of groups of uncharacterized genes for diverse applications such as directed functional studies and delineation of novel drug targets.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The scoring procedure is based on a global survey of researchers, who assigned characterization scores from 1 (poor) to 10 (extensive) for a sample of genes based on major online resources. By evaluating the survey as training data, we developed a bioinformatics procedure to assign gene characterization scores to all genes in the human genome. We analyzed snapshots of functional genome annotation over a period of 6 years to assess temporal changes reflected by the increase of the average Gene Characterization Index. Applying the Gene Characterization Index to genes within pharmaceutically relevant classes, we confirmed known drug targets as high-scoring genes and revealed potentially interesting novel targets with low characterization indexes. Removing known drug targets and genes linked to sequence-related patent filings from the entirety of indexed genes, we identified sets of low-scoring genes particularly suited for further experimental investigation.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The Gene Characterization Index is intended to serve as a tool to the scientific community and granting agencies for focusing resources and efforts on unexplored areas of the genome. The Gene Characterization Index is available from <a href="http://cisreg.ca/gci/">http://cisreg.ca/gci/</a>.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kemmer D, Podowski RM, Yusuf D, Brumm J, Cheung W, et al.  (2008) Gene Characterization Index: Assessing the Depth of Gene Annotation. PLoS ONE 3(1):
          e1440.
            doi:10.1371/journal.pone.0001440</p><p><strong>Academic Editor: </strong>Juan Valcarcel, Centre de Regulació Genòmica, Spain</p><p><strong>Received:</strong> November 9, 2007; <strong>Accepted:</strong> December 16, 2007; <strong>Published:</strong> January 23, 2008</p><p><strong>Copyright:</strong> © 2008 Kemmer et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding from the Canadian Institutes of Health Research and the Michael Smith Foundation for Health Research. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Elucidation of the function(s) for each human protein-encoding gene has been a prominent challenge in biomedical research after the completed deciphering of the sequence of the human genome. Systematic characterization projects have been launched, ranging from the ENCODE project for detailed genome annotation <a href="#pone.0001440-Birney1">[1]</a> to the phenome projects to identify phenotypes generated by mutations of human gene orthologs in model organisms <a href="#pone.0001440-Bogue1">[2]</a>–<a href="#pone.0001440-Rual1">[4]</a>. These efforts were undertaken, in part, to evoke new insights into the functions of uncharacterized genes revealed through the successful sequencing of the human genome. At the level of basic human curiosity, scientists are drawn to these uncharacterized genes, for it is the deciphering of the functions of these genes which offers the greatest potential to gain fundamental insights into novel biological processes; to peer into the unknown. The therapeutic and financial benefits associated with successful identification of genes that are suitable targets for pharmaceutical research and informative biomarkers for treatment selection stands as another strong motivator.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The arsenal of the modern molecular researcher, when directed at specific genes, can elucidate properties that offer glimpses of underlying functions. In the laboratory we can determine specific phenotypic effects of a gene when disrupted in model organisms, where the encoded protein localizes within the cell, the spatio-temporal coordinates of gene activity, the function in cells or model organisms through biological assays, and further techniques <em>ad infinitum</em>. To unleash these often expensive and time-consuming studies, researchers (and funding agencies) are usually motivated by preliminary glimmers of functional knowledge. However, with the goal of comprehensiveness, attempts to explore genome function in an unbiased manner have been made. In the ENCODE project, undertaken by a portion of the global research community to systematically annotate functions for 1% of the human genome, a portion of the genome was selected for study for the glaring absence of knowledge about the genes in the region <a href="#pone.0001440-Birney1">[1]</a>. The Allan Brain Atlas <a href="#pone.0001440-Gewin1">[5]</a> places a premium on the systematic study of expression in the mouse brain of uncharacterized genes. In the pharmaceutical industries, gaining insights into the functions of uncharacterized genes can offer a direct and meaningful path to successful drug target identification.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>However, based on available methodologies, it has been challenging to divide genes into classes depending on functional knowledge and to extract sets of uncharacterized genes from the genome. The characterization state of each gene exists in the eye of the beholder; each scientist brings a distinct perspective to the interpretation of the characterization status of a gene. While scientists can select individual genes, judged as being scarcely annotated based on available information in multiple data repositories, no quantitative measurement of annotation status exists widely applicable to sets of genes of particular interest to further experimental study. However, previous studies have shown that human opinion, based on specific sets of predictors, can be quantified and predicted. In fact, researchers have surveyed the perception of human beauty and developed computational methods capable of accurately predicting consensus opinions by collecting human ratings and developing machine learning procedures based on those ratings <a href="#pone.0001440-Eisenthal1">[6]</a>, <a href="#pone.0001440-Gunes1">[7]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>With the goal of establishing a quantitative and universal system for measuring the annotation status of human genes, we developed the Gene Characterization Index (GCI), a bioinformatics procedure to quantitatively assign a characterization score to each human gene relying on collected opinions from the global research community. In this report, based on training data derived from a reference collection of genes with assigned characterization scores from over 50 scientists worldwide, we constructed a classification function that successfully predicts the characterization of human protein-encoding genes. By applying the method to well-studied classes of genes followed by comparison to the depth of annotation in the Gene Ontology (GO) system <a href="#pone.0001440-Ashburner1">[8]</a>, we confirmed the accurate prediction of the level of functional characterization by the GCI. At the genome scale, we integrated GCI values across all human protein-encoding genes to determine the characterization status of the human genome. We found that the progress made by the research community to assign functions to human genes after the release of the first draft of the human genome in 2000 was well-reflected by an increase of the average GCI score across the genome over time.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>At a finer grain, analyzing classes of pharmaceutically relevant gene families such as G protein-coupled receptors, nuclear receptors and ion channels, we revealed specific characteristics of the different groups and highlighted opportunities for the identification of hitherto overlooked novel drug targets within those therapeutically relevant protein families with potentially important roles in the treatment of various diseases.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The GCI is the first automated method for quantitatively assessing the extent to which each gene is annotated. By applying the GCI scoring system on a genome scale, we identified a large portion of the human genome, likely including groups of genes with potentially interesting applications, essentially neglected by the scientific community. By drawing attention to these groups of weakly annotated genes, the GCI could help advance genome characterization in an unbiased manner. Translated to the single gene level, our scoring system is intended to help direct funding agencies to these neglected areas and to guide the level of analysis that should be funded. It serves as a resource for focusing experimental efforts and can provide both computational and laboratory scientists with opportunities to demonstrate the novelty of current findings and the utility of new methods.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Implementation of the Gene Characterization Index</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To create a quantitative method for assigning characterization scores to each human gene, a representative reference collection of scores for a subset of genes was required. This collection served both as training data for determining predictive characteristics for a perceived characterization state, as well as test data for determining the reliability of the predictive methods generated. To create a broadly representative method, the reference collection of gene characterization scores was created through a global survey of life sciences researchers with diverse scientific backgrounds. We asked 52 scientists worldwide to assess a sample of genes and assign a score within a 10-point scale, with 1 indicating a completely uncharacterized gene and 10 a gene that is fully described. As the GCI directly reflects the scores obtained in the survey, the survey is described in detail.</p>


<h4>Reference gene definition and annotation source</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Both for the survey and the ultimate production of the GCI, it was necessary to define a reference set of human genes. Such lists could be obtained from a variety of sources, each with unique characteristics. In addition to a list of genes, we desired a system that provided diverse functional annotations, as these gene characteristics constituted the variables that could be evaluated and quantitated by a scoring function. We selected the Entrez Gene database <a href="#pone.0001440-Maglott1">[9]</a> as an appropriate data source, as it met the above-mentioned requirements and was expected to maintain data quality with regular updates. To retroactively measure the progress of human genome annotation, we further required a gene annotation source providing access to releases over a multi-year period. For this purpose, we selected the GeneLynx database <a href="#pone.0001440-Lenhard1">[10]</a>, as it was the most accessible system with resources comparable to Entrez Gene for which annotations could be obtained from an extended time period. Finally, we selected the set of “training” genes from release 1.2 of the GeneLynx database (June 2003) in a 2-stage process to insure a balanced representation of genes with diverse characteristics. We randomly chose an initial set of 90 genes from the subset of genes for which cDNA sequences were available in GenBank <a href="#pone.0001440-Benson1">[11]</a>. In addition, to insure the inclusion of genes with minimal functional characterization, we included 10 genes represented only by expressed sequence tags (ESTs).</p>


<h4>Evaluator ratings</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To gather scores for the reference genes we developed a web-based survey system. Each participating scientist was assigned 10 genes (including 1 EST-only gene) and asked to provide a characterization value. An optional guide to scoring was made available to the evaluators to assist in determining scores (see online Supplementary Material). Each gene was assessed by multiple biologists to allow the determination of an average score. We sought at least 3 independent scores for each gene in the reference collection. As genes were randomly assigned, the actual number of responses per gene varied, with a minimum of 3 and an average of 4.6. Supplementary <a href="#pone.0001440.s001">Table S1</a> describes the complete list of reference genes that was used in the survey together with average evaluator ratings for each gene. Supplementary <a href="#pone.0001440.s003">Figure S1</a> shows the distribution of evaluator ratings.</p>


<h4>GCI classifier selection</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Once we had collected evaluator ratings for the reference genes, we set out to train a predictor from those ratings through the application of machine learning procedures including linear models (LM), regression trees (RT), neural nets (NN), support vector machines (SVM) <a href="#pone.0001440-Vapnik1">[12]</a> and multivariate additive regression splines (MARS) <a href="#pone.0001440-Friedman1">[13]</a>. For the implementation of universal predictors of gene annotation status, we first selected sets of gene characteristics used as classifiers from the training data. As indicated previously, we selected historical releases of the GeneLynx database <a href="#pone.0001440-Lenhard1">[10]</a> as the source for annotation over an extended time period. Similar to the more recent Entrez Gene <a href="#pone.0001440-Maglott1">[9]</a>, GeneLynx provided links to a diverse array of online resources with gene-specific information; these links were automatically compiled from numerous systems. We treated the compilation of resources for each gene as gene characteristics that could be quantified. As those resources have changed over time, we restricted the GCI training data to those resources that had been recorded in GeneLynx since 2001. In addition to these core GeneLynx annotations, we also considered gene-related articles published prior to the survey; the article counts were based on references in the Entrez Gene and SwissProt databases.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Initially, we compiled a total of 40 gene characteristics for the reference genes (see Supplementary <a href="#pone.0001440.s002">Table S2</a>). We then created each predictor vector by determining the total number of unique attributes for each gene, followed by post-processing of each field, including z-score normalization (see online Supplementary Material). As a consequence, the list of 40 attributes was subsequently reduced to 16 by removal of gene characteristics not represented in all GeneLynx releases, and further by selection of a single representative from highly correlated groups (<a href="#pone-0001440-t001">Table 1</a> and <a href="#pone-0001440-t002">2</a>). We considered attributes with a correlation of 0.7 or higher similar, thus becoming candidates for removal.</p>
<div class="figure" id="pone-0001440-t001"><div class="img"><a name="pone-0001440-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Classification model attributes used by the MARS model.</span></strong></p><span>doi:10.1371/journal.pone.0001440.t001</span></div><div class="figure" id="pone-0001440-t002"><div class="img"><a name="pone-0001440-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Additional available classification model attributes.</span></strong></p><span>doi:10.1371/journal.pone.0001440.t002</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Available attributes for use in the statistical model for defining gene annotation state include single nucleotide polymorphisms (SNPs) describing DNA sequence variations often linked to disease, genomic sequences derived from GenBank <a href="#pone.0001440-Benson1">[11]</a>, InterPro <a href="#pone.0001440-Mulder1">[14]</a>, PRINTS <a href="#pone.0001440-Attwood1">[15]</a> and PROSITE <a href="#pone.0001440-Hulo1">[16]</a> domains assigning genes to families, Gene Ontology (GO) annotations <a href="#pone.0001440-Ashburner1">[8]</a> using consistent descriptions for gene function, reports in KEGG <a href="#pone.0001440-Kanehisa1">[17]</a> integrating genes into cellular pathways, associations in OMIM linking genes to known diseases with a genetic component (<a href="http://www.ncbi.nlm.nih.gov/omim/">http://www.ncbi.nlm.nih.gov/omim/</a>), RefSeq accessions <a href="#pone.0001440-Pruitt1">[18]</a> pointing to a gene's curated annotation level, SwissProt entries <a href="#pone.0001440-Wu1">[19]</a> providing manually curated information on a gene's protein product, PubMed (<a href="http://www.pubmed.gov">http://www.pubmed.gov</a>) literature references, PDB <a href="#pone.0001440-Berman1">[20]</a> protein structure information, HomoloGene (<a href="http://www.ncbi.nlm.nih.gov/HomoloGene/">http://www.ncbi.nlm.nih.gov/HomoloGene/</a>) linking genes to homologous sequences in other species and descriptive annotations such as gene name, symbol and functional description.</p>


<h4>Model selection</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Based on the performance observed and the clarity of the underlying procedure, we selected a MARS model for the final function. It is important to note that the selection of the statistical approach had limited importance, as several methods performed comparably well in the model validation step. The ultimate model produced scores based on 6 attributes (<a href="#pone-0001440-t001">Table 1</a>). We explored models using less attributes, but found that 6 attributes performed best. Full documentation of the performance review and validation can be found in the online Supplementary Material.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The MARS method assigned scores within +/−1 of the average reviewer score for 57% of reference genes and 81% of assigned scores were within +/−2 of the reviewer assigned scores (<a href="#pone-0001440-g001">Figure 1</a>). However, for a few genes that scored greater than +/−2 from the mean, we conducted a manual review to assess the overall annotation levels by enumerating available gene characteristics possibly explaining the discrepancy between evaluator ratings and computational predictions. For example, SOCS box-containing WD protein SWiP-1 (<em>WSB1</em>) (Entrez Gene ID 26118), rated at 2.5 but was predicted at 5.3. This gene presented a moderate number of associated publications in PubMed, contained mainly protein domain-specific annotations and therefore, was scored low by human evaluators. However, inclusion of <em>WSB1</em> in the KEGG database of cellular pathways, the OMIM database of disease associations, the SwissProt protein database, as well as the presence of a number of SNPs suggested a curated level of functional annotation. As a consequence, we judged that the predicted score more accurately reflected the functional understanding of the gene.</p>
<div class="figure" id="pone-0001440-g001"><div class="img"><a name="pone-0001440-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>GCI Model Cross-validation Performance.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>GCI Predictor Performance–Leave-One-Out cross-validation results for the final GCI predictor model utilizing the MARS method on z-score normalized data. The X-axis displays average evaluator assigned scores, while the Y-axis displays the predicted scores for each gene in the leave-one-out cross validation analysis (the score assigned when the gene was not included in the training data). As observed, the MARS model can assign scores greater than 10 (in all further analysis such scores are rounded down to 10).</p>
<span>doi:10.1371/journal.pone.0001440.g001</span></div>

<h4>Performance of the Gene Characterization Index</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To assess the capacity of the GCI scoring model to successfully predict gene annotation levels, we integrated characterization scores across all predicted human genes and further compared those scores to information content from the Gene Ontology (GO) gene annotation system <a href="#pone.0001440-Ashburner1">[8]</a>.</p>


<h4>Genome-scale scoring and temporal annotation changes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Since the release of the first draft of the human genome sequence in 2000, concerted efforts of the scientific community have resulted in the successful delineation and functional annotation of a large body of predicted human genes. We reasoned that, following genome evolution over the years since its release, we could follow and quantify the expansion of functional knowledge via an increase in the average GCI score over time. To compare successive versions of the human genome, we applied the final GCI scoring model to all genes present in either the Entrez Gene database <a href="#pone.0001440-Maglott1">[9]</a> for recent genome releases, or in the GeneLynx database <a href="#pone.0001440-Lenhard1">[10]</a> for historical genome versions.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The histogram in <a href="#pone-0001440-g002">Figure 2</a> shows the distribution of GCI scores across all human genes taken at 3 different time points. In the earliest version, 64% of all genes (20475 of 31987 genes) clustered at the bottom of the scale with scores lower than 2.5, and less than 3% (885 genes) scored at 7.5 or higher, reflecting the scarce overall annotation level of the human genome. Successive genome releases showed an important decrease in the low-scoring group of genes with 30.9% of scores (10308 of 33410 genes) under 2.5 for the latest release in September 2007. Also apparent from the histogram was the steady increase in high-scoring genes over time. The portion of genes scoring above 7.5 rose from merely 2.8% (885 of 31987 genes) in May 2001 to 8.3% (2588 of 31096 genes) in April 2004 reaching 15.8% (5286 of 33410 genes) in the latest release. Overall gene numbers fluctuated between successive genome releases due to changing genome annotations and transcript-to-gene mappings. To provide some perspective to the reader, we assembled a subset of the highest and lowest scoring genes from the September 2007 release presented in <a href="#pone-0001440-t003">Tables 3</a> and <a href="#pone-0001440-t004">4</a>.</p>
<div class="figure" id="pone-0001440-g002"><div class="img"><a name="pone-0001440-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genome-wide GCI Score Distribution.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Histogram displaying the frequency of scores observed in the analysis of genes at 3 different time points after the release of the first draft of the human genome sequence. Genes based only on predictions and/or EST sequences have been removed (~3000 genes in 2007 data).</p>
<span>doi:10.1371/journal.pone.0001440.g002</span></div><div class="figure" id="pone-0001440-t003"><div class="img"><a name="pone-0001440-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Sets of genes with extreme characterization scores: sampling of well-characterized genes.</span></strong></p><span>doi:10.1371/journal.pone.0001440.t003</span></div><div class="figure" id="pone-0001440-t004"><div class="img"><a name="pone-0001440-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Sets of genes with extreme characterization scores: sampling of scarcely characterized genes.</span></strong></p><span>doi:10.1371/journal.pone.0001440.t004</span></div>

<h4>Qualitative adaptation of scoring model</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>To qualitatively assess the increase in gene characterization over time, we analyzed contributions of different annotation sources assembled in Entrez Gene and GeneLynx to rising gene annotation levels. We counted entries for each attribute selected by the MARS model. Analyzing annotation changes between July 2006 and September 2007, it became apparent that the increase in the number of PubMed references largely dominated all other attributes, and thus, publications could be considered the most important contributor to the recent advancement of gene annotation levels (<a href="#pone-0001440-t005">Table 5</a>). In addition to publication in peer-reviewed journals, recent expansion of the KEGG database of cellular pathways contributed moderately to improved annotation. While new disease associations recorded in the OMIM database and new SwissProt entries contributed to increased annotation levels, there was a decrease in the number of protein domain family associations reported in InterPro likely reflecting a change in genome annotation procedures.</p>
<div class="figure" id="pone-0001440-t005"><div class="img"><a name="pone-0001440-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Individual contribution of attributes to rising annotation levels.</span></strong></p><span>doi:10.1371/journal.pone.0001440.t005</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>After investigating characteristics of the more recent genome annotation, we analyzed earlier developments directly following the release of the first genome sequence drafts. Historical annotation changes were reflected by changing attributes selected by the MARS model in the GeneLynx database. While there had been a steady increase in the number of PubMed references across all time points, earlier releases were essentially marked by an increased association of cDNAs-to-genes and the designation of official gene names and symbols. Other characteristics of functional gene annotation were the inclusion of SNPs, increases in Gene Ontology annotations, InterPro domains, KEGG pathways, and OMIM disease associations.</p>


<h4>Comparison to Gene Ontology annotation levels</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>The Gene Ontology (GO) project is an effort to standardize gene descriptions using a predefined vocabulary of functional terms <a href="#pone.0001440-Ashburner1">[8]</a>, and GO terms are now widely used to functionally annotate genes and their protein products <a href="#pone.0001440-Camon1">[21]</a>. GO is a highly curated system that uses 3 structured ontologies to describe genes in terms of their associated <em>molecular function</em>, <em>biological process</em> and <em>cellular component</em>. As gene annotation levels deepen, the hierarchy of specialized GO terms extends, describing the gene in greater detail. Functional comparison of sequences annotated with GO terms can be performed based on semantic similarity measures like the ones developed by Resnik <a href="#pone.0001440-Resnik1">[22]</a>.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>For genes drawn from the latest genome release (September 2007) with GO annotations, we extracted the maximum information content for each gene by using Resnik's similarity measure reflecting the granularity of the GO terms assigned to a given gene. We then compared these raw information content scores to the GCI scores obtained with our scoring model using Pearson correlations. For the comparisons, we selected equal numbers of genes from each bin of GCI scores (e.g. 1–1.4, 1.5–2.4, etc.; see <a href="#pone-0001440-g002">Figure 2</a> for bins) and calculated overall correlations. Comparing all GCI scores to the GO <em>molecular function</em> category using Resnik scores, we observed a Pearson correlation coefficient of 0.59. We also combined bins into larger groups representing increasing annotation levels (low &lt;2.5, medium low 2.5–4.9, medium high 5.0–7.4 and high 7.5–10.0) and examined differences between GCI and GO (<a href="#pone-0001440-g003">Figure 3</a>). Genes with higher scores agreed best with the Resnik scores for GO <em>molecular functions</em>, while genes with low scores showed weaker correspondence. No strong correlations were observed for GO <em>biological process</em> and <em>cellular compartment</em> categories (data not shown). The positive correlation between Resnik scores of GO <em>molecular function</em> and GCI scores further validated our automated scoring model as an accurate reflection of the depth of gene annotation levels.</p>
<div class="figure" id="pone-0001440-g003"><div class="img"><a name="pone-0001440-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Resnik Scores for Depth of GO Gene Annotation Correspond with GCI Scores.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>The Resnik score describes the granularity of annotations attached to each gene. There is an overall Pearson correlation of 0.6 between GCI and Resnik scores. The distribution plot shows the distribution of Resnik scores for ranges of GCI scores.</p>
<span>doi:10.1371/journal.pone.0001440.g003</span></div>

<h4>Application of the Gene Characterization Index</h4>
<h5>Analysis of drug target gene families.</h5><a id="article1.body1.sec2.sec10.sec1.p1" name="article1.body1.sec2.sec10.sec1.p1"></a><p>Using the predictions of functional gene annotation levels from the GCI scoring system, we investigated the properties of annotation for a variety of gene sets relevant in medicine and drug development. Over the past few decades, 3 gene families have stood out as the most common drug targets: G protein-coupled receptors (GPCRs), nuclear receptors (NRs), and ion channel proteins (ICs) <a href="#pone.0001440-Overington1">[23]</a>. We extracted all gene members of these families from appropriate resources and, using DrugBank as a resource for drug data and protein target information <a href="#pone.0001440-Wishart1">[24]</a>, divided them into known drug targets and non-drug targets. By analyzing each family for its annotation status, we uncovered family-specific characteristics and revealed potential opportunities to study neglected members of these biologically relevant gene classes, as well as to discover new drug targets.</p>

<h5>G protein-coupled receptors.</h5><a id="article1.body1.sec2.sec10.sec2.p1" name="article1.body1.sec2.sec10.sec2.p1"></a><p>GPCRs, targeted by nearly one third of currently marketed drugs <a href="#pone.0001440-Overington1">[23]</a>, are diverse in structure and function, and, with their broad evolutionary conservation, are considered the oldest cellular machineries devoted to signal transduction. From the G Protein-Coupled Receptor Data Base (<a href="http://www.gpcr.org/">http://www.gpcr.org/</a>) <a href="#pone.0001440-Horn1">[25]</a> and GO annotations, we extracted a set of 750 GPCRs for further analysis. Applying annotations from DrugBank, we divided the set into GPCRs of FDA-approved drug targets (79) and receptors previously not targeted by pharmaceuticals (671) and applied the GCI scoring system. As apparent from <a href="#pone-0001440-g004">Figure 4A</a>, over 80% of drug target GPCRs presented deep annotation levels and scored between 8.5 and 10.0. This group included highly relevant drug targets such as the angiotensin receptor (<em>AGTR1</em>), serotonin receptor (<em>HTR2A</em>), and endothelin receptor (<em>EDNRB</em>). However, analyzing GPCRs not targeted by small molecule drugs revealed that the majority had remained weakly characterized with GCI scores for over 60% of genes ranging from 1.5 to 5.4 highlighting opportunity for further study and development of new targets within this highly relevant gene family. For example, G protein-coupled receptor 137C (<em>GPR137C</em>, Entrez Gene ID 283554, GCI = 3.23), a weakly annotated GPCR, is linked to a single PubMed publication associating the gene's transcription to a gastric cancer gene expression profile <a href="#pone.0001440-Oh1">[26]</a>. Another example is the 7 transmembrane helix receptor <em>LOC440683</em> (Entrez Gene ID 440683) scoring at 2.16 with reported rhodopsin-like receptor activity and no further functional information associated. It should be noted, however, that the group of low-scoring GPCRs could be partly accounted for by the inclusion of olfactory receptors, a subclass with limited therapeutic potential. Comparing the non-drug target group of GPCRs with the GenBank patented sequence repository showed that 537 genes (80%) had patents associated with them.</p>
<div class="figure" id="pone-0001440-g004"><div class="img"><a name="pone-0001440-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>A. Distribution of GCI Scores for Genes in Selected Protein Families and Classes.</span></strong></p><a id="article1.body1.sec2.sec10.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec10.sec2.fig1.caption1.p1"></a><p>750 G Protein-Coupled Recptors: 79 DTG, 671 NDTG; 50 Nuclear Receptors: 23 DTG, 27 NDTG; 66 Ligand-Gated Ion Channels: 21 DTG, 45 NDTG; 111 Potassium Ion Channels: 14 DTG, 97 NDTG. B. Genome-wide GCI Score Distribution for Drug Targets, Patented and All Other Genes. Based on genome release July 2006: 1095 drug targets, 14237 patented, 14913 non-target, non-patented genes. 10867 non-targeted, non-patented genes were highly uncharacterized with GCI scores &lt;3.5.</p>
<span>doi:10.1371/journal.pone.0001440.g004</span></div>
<h5>Nuclear receptors.</h5><a id="article1.body1.sec2.sec10.sec3.p1" name="article1.body1.sec2.sec10.sec3.p1"></a><p>Based on GO annotations and the Nuclear Receptor Database (<a href="http://www.receptors.org/NR/">http://www.receptors.org/NR/</a>) <a href="#pone.0001440-Horn2">[27]</a>, we extracted 50 NRs for further analysis. This second largest group of current drug targets functions as ligand-activated transcription factors and regulate core cellular processes such as cell growth and differentiation, inflammatory responses and metabolism. Their important role in physiology and the ability to regulate their functional activity with synthetic small molecules has rendered this gene family a favorite target for drug discovery <a href="#pone.0001440-Schulman1">[28]</a>. Subjecting NRs to the same analysis as GPCRs, we found that 22 of 23 drug target NRs presented GCI scores of 7.5 and above (<a href="#pone-0001440-g004">Figure 4A</a>). Similarly, the distribution of GCI scores for all 27 non-targeted NRs was high (average GCI = 7.8) reflecting the intense scrutiny given to these proteins. Across all NRs, a single gene obtained a low GCI score at 2.9. This gene (Entrez Gene ID 55566), coding for the estrogen receptor-like p65 protein and linked to colorectal cancer through 2 PubMed publications, lacked additional functional annotation making it a particularly interesting candidate for further study. As shown for GPCRs, the majority of the non-drug target NRs (24) had patents associated with them.</p>

<h5>Ion channels.</h5><a id="article1.body1.sec2.sec10.sec4.p1" name="article1.body1.sec2.sec10.sec4.p1"></a><p>ICs constitute the third major class of current drug targets including ligand- and voltage-gated ion channels. The family of ligand-gated ion channels (LGICs) comprises several superfamilies and their physiological activity controls information flow in the brain, thus becoming a relevant class of targets for drugs treating disorders such as epilepsy and anxiety <a href="#pone.0001440-Connolly1">[29]</a>. From the Ligand-Gated Ion Channel database (<a href="http://www.ebi.ac.uk/compneur-srv/LGICdb/LGICdb.php">http://www.ebi.ac.uk/compneur-srv/LGICdb​/LGICdb.php</a>) <a href="#pone.0001440-Donizelli1">[30]</a> we extracted 66 genes, of which 21 coded for FDA-approved drug targets. Comparable to NRs, 86% of drug target LGICs scored above 7.5, whereas the distribution of GCI scores for non-drug target LGICs was more widely distributed with 38% of genes scoring below 7.5 (<a href="#pone-0001440-g004">Figure 4A</a>). Several low-scoring, non-drug target LGICs coded for weakly annotated subunits of well-known receptor complexes: the alpha 4 subunit of the glycine receptor (<em>GLRA4</em>, Entrez Gene ID 441509, GCI = 1.5), the rho3 subunit of the gamma-aminobutyric acid (GABA) receptor (<em>GABRR3</em>, Entrez Gene ID 200959, GCI = 3.3), and several subunits of the type 3 receptor for 5-hydroxytryptamine (serotonin). It should be noted that the pool of non-drug target LGICs may include subunits of receptor complexes already targeted by small molecule compounds, as illustrated by the GABA<sub>A</sub> receptor gamma 2 subunit (<em>GABRG2</em>), with a GCI score of 10.0 and no current records in DrugBank. The gene's mutated isoforms have been implicated in epilepsy <a href="#pone.0001440-Mizielinska1">[31]</a> and, as a subunit of the type A GABA receptor, this gene is part of a protein complex heavily targeted by pharmaceuticals. However, weakly annotated subunits of already targeted protein complexes may present opportunities for the development of novel drugs targeting new sets of targets within protein complexes with known therapeutic potential. As for GPCRs and NRs, the large majority of non-drug target genes (89%) were associated with patents.</p>
<a id="article1.body1.sec2.sec10.sec4.p2" name="article1.body1.sec2.sec10.sec4.p2"></a><p>Another important class of ICs, Potassium Ion Channels (KICs), associated with action potentials and intercellular signaling, perform a wide variety of functions in both excitable and non-excitable cells and thus, have been recognized as potential drug targets <a href="#pone.0001440-Curran1">[32]</a>. From the KChannelDB (<a href="http://www.receptors.org/KCN/">http://www.receptors.org/KCN/</a>), we extracted 111 KIC genes, 14 of which were targeted by small molecule drugs according to DrugBank. Applying the GCI scoring system to the drug targets, we observed that 9 of 14 KICs obtained scores at 7.5 and above, and that no gene scored below 5.5 (<a href="#pone-0001440-g004">Figure 4A</a>). The analysis of the non-drug target KICs yielded a somewhat different distribution with almost half of the genes (47%) scoring at medium high levels (5.5–7.4), and an important portion (22%) in the range of moderate to weak annotation levels (&lt;5.5). Several of these moderately annotated genes were members of a group of potassium channels containing tetramerisation domains. Non-targeted KICs, and thus interesting for further study, included the potassium channel subfamily T member 1 (<em>KCNT1</em>, Entrez Gene ID 57582, GCI = 4.8) and member 2 (<em>KCNT2</em>, Entrez Gene ID 343450, GCI = 5.0), the potassium channel subfamily K member 18 (<em>KCNK18</em>, Entrez Gene ID 338567, GCI = 4.2), the potassium voltage-gated channel shaker-related subfamily member 7 (<em>KCNA7</em>, Entrez Gene ID 3743, GCI = 5.7), and the KCNE1-like membrane protein (<em>KCNE1L</em>, Entrez Gene ID 23630, GCI = 6.4). Similar to the rates for other drug target gene families examined, we observed that 88 of 97 (91%) non-drug target KICs were associated to patents in GenBank.</p>

<h5>Genome-wide analysis of drug target and patented genes.</h5><a id="article1.body1.sec2.sec10.sec5.p1" name="article1.body1.sec2.sec10.sec5.p1"></a><p>To expand the analysis of GCI distributions to the whole genome, we extracted the entire set of drug targets from DrugBank (1095 proteins) and all patented genes reported in NCBI's patent nucleotide sequence database (14295 genes). <a href="#pone-0001440-g004">Figure 4B</a> illustrates the genome-wide distribution of GCI scores for the different gene classes. Similar to the distribution pattern observed for specific drug target gene families, drug target genes in the genome clustered at the high end of the scoring scale with over 75% having GCI scores above 7.5. Patented genes were scattered across the range of scores with a smaller fractions at the low end (i.e., 1.9% receiving scores &lt;2.4). Removing all drug target and patented genes from the genome, we applied the GCI scoring model to the remaining 14913 genes. Tellingly, over 70% of the non-patented non-target genes were highly unexplored with GCI scores &lt;3.5. The remainder of genes was distributed across the entire scale with decreasing numbers for increasing scores.</p>

<h5>Effect of patent filing on gene characterization.</h5><a id="article1.body1.sec2.sec10.sec6.p1" name="article1.body1.sec2.sec10.sec6.p1"></a><p>Since applications for patents on DNA sequences became common practice in the early 1990's <a href="#pone.0001440-Adler1">[33]</a>, critics have raised a number of objections against the patenting of biological material. The debate has remained vivid to the current day <a href="#pone.0001440-Resnik2">[34]</a>, <a href="#pone.0001440-Heller1">[35]</a>. The question whether, and to what extent, patenting of genes impacts academic science has remained a topic of controversy. To follow functional annotation progress of patented versus non-patented genes since 2001, we applied the GCI scoring system to all patented and non-patented genes and followed the fate of the different groups up to September 2007, 6 years after the release of the first draft of the human genome sequence. From NCBI's patented sequence database, we extracted 11278 patented genes, applied the GCI scoring system based on both GeneLynx release 0.9 (May 2001) and Entrez Gene release September 2007 and compared the GCI score distribution for the 2 time points. In the same way, we extracted 20867 non-patented genes from GeneLynx 0.9 and 19331 non-patented genes from Entrez Gene September 2007 and compared the score distributions between the 2 genome releases. It should be noted that there have been fluctuations in the mappings of cDNAs to genes in successive genome releases accounting for the slight variations in the number of genes in the different groups between 2001 and 2007. For patented genes in 2001, 93% of GCI scores spanned across the low to medium part of the scale (GCI&lt;7.5), with merely 7% clustering at the high end (GCI≥7.5) (<a href="#pone-0001440-g005">Figure 5</a>). In 2007, however, the centre of the distribution was shifted from low (GCI = 1.5–2.4) to medium high scores (GCI = 6.5–7.4), with over 36% of genes ranking above 7.4. In 2007, there were only 1.3% of patented genes left scoring below 3.5. By dividing the genes into bins, it became apparent that over 40% of patented genes had climbed up the scale from the low GCI bin, with the highest increase for the high GCI bin (<a href="#pone-0001440-t006">Table 6</a>).</p>
<div class="figure" id="pone-0001440-g005"><div class="img"><a name="pone-0001440-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Evolution of Patented versus Non-Patented Genes between 2001 and 2007.</span></strong></p><a id="article1.body1.sec2.sec10.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec10.sec6.fig1.caption1.p1"></a><p>Histogram presenting substantial differences in annotation progress between patented and non-patented genes. Fluctuating gene numbers due to changes in genome annotations and transcript mappings.</p>
<span>doi:10.1371/journal.pone.0001440.g005</span></div><div class="figure" id="pone-0001440-t006"><div class="img"><a name="pone-0001440-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Change in GCI score distribution of patented versus non-patented genes between May 2001 and September 2007.</span></strong></p><span>doi:10.1371/journal.pone.0001440.t006</span></div><a id="article1.body1.sec2.sec10.sec6.p2" name="article1.body1.sec2.sec10.sec6.p2"></a><p>The evolution for the GCI distribution for non-patented genes was markedly different. As illustrated in <a href="#pone-0001440-g005">Figure 5</a>, in 2001 over 90% of all non-patented genes clustered at scores below 3.5, with less than 1% reaching the high end of the scale (GCI≥7.5). In September 2007, this distribution had not notably changed with over 75% of genes remaining at the low end of the scale (GCI&lt;3.5). The majority of genes that had left the low GCI bin had moved to the medium GCI bin, and not to the high GCI bin as observed for patented genes (<a href="#pone-0001440-t006">Table 6</a>). Also, a fraction of the genes originally in the low score bin had been removed all together (1535 genes), likely due to genome reannotations.</p>



<h4>Accessing the Gene Characterization Index</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>Users may access the GCI at <a href="http://www.cisreg.ca/gci/">http://www.cisreg.ca/gci/</a>, where they can search for the functional characterization level of their favorite gene, browse for genes with specific annotation levels, choose random genes and further explore the system. After searching with the gene name, Entrez Gene ID or free text, the GCI scoring system returns the GCI score for the searched gene, as well as links to various data sources underlying the computed score (<a href="#pone-0001440-g006">Figure 6</a>). It is also possible to search for genes in bulk, where the GCI scoring system accepts lists of genes with Entrez Gene IDs and returns scores for all genes from the list. For scientists interested in comparing their own perception to the automated scores, they may perform a blinded test. A randomly selected set of 10 genes will be displayed and the user may assign their own scores. After assignments are submitted, a comparison report is generated. Scores submitted through this process will be used as training data for future releases of GCI (to reflect temporal shifts in human perception of what constitutes a well-characterized gene). Finally, users may download the entire set of scores, as well as access the data and Supplementary Material described in this manuscript.</p>
<div class="figure" id="pone-0001440-g006"><div class="img"><a name="pone-0001440-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001440" data-uri="info:doi/10.1371/journal.pone.0001440.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001440.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001440.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001440.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001440.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Screenshot of GCI Web Page.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Example of Calmodulin-like protein 6 returned by GCI search engine with gene-specific GCI score and links to data sources.</p>
<span>doi:10.1371/journal.pone.0001440.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have introduced a novel bioinformatics procedure to quantitate the functional characterization of each protein-encoding human gene. Based on a reference collection of characterization scores assigned by diverse life scientists, we trained a classification function to predict scores depending on functional annotations in the Entrez Gene database. Using a MARS classifier that performed well in cross-validation, we were able to assign characterization scores to all human genes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>As a first assessment of the performance of the GCI scoring system, we assigned scores to all accessible genes in consecutive releases of the human genome and followed GCI score distribution and attribute selection of the MARS model. We could show that the GCI scoring function mirrored an increase in genome annotation performed by the research community through a shift of score distributions to deeper annotation levels. The changing weight assigned by the GCI scoring system to different attributes over time reflected befittingly the evolution of the human genome. While early releases were marked by predictions of fluctuating numbers of genes, whose active transcription could only be confirmed for a fraction, the association of cDNAs with predicted genes was an important factor in the genome characterization effort. Also, the rate of gene discovery was high, and the classification of novel genes according to sequence motifs and domains into protein families was an important step in inferring functional information. Unlike this early phase of the genome era, the sequence quality of later releases improved dramatically with the official closure of the human genome project in April 2003 <a href="#pone.0001440-Pennisi1">[36]</a>. As a consequence, gene numbers became more stable leveling off around 25000 human genes <a href="#pone.0001440-Pennisi2">[37]</a>. Most of these genes were confirmed by RNA transcripts and basic functional information existed for many of them. Later advancements in gene annotations captured by the GCI scoring model, therefore, were defined by deeper levels of functional characterization, especially in the form of scientific publications describing individual genes and curated functional knowledge in databases such as KEGG, OMIM and SwissProt.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In a second assessment of model performance, we showed that GCI annotation levels correlated well with Gene Ontology terms of <em>molecular function</em>, with no correlations observed for the <em>biological process</em> and <em>cellular component</em> categories. This observation revealed that GCI scores, which are based on specific gene attributes, referred to unambiguous cellular functions such as binding, receptor, transporter or enzyme activities rather than higher-ranking biological processes or cellular components of a gene's activity. It also became apparent that the level of functional annotation captured by the GCI scoring model increasingly paralleled GO <em>molecular function</em> hierarchies the deeper a gene's functional characterization reached.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Applying the GCI to 3 major protein families of current drug targets, we presented each family's overall annotation status and pointed to potential candidates for further investigation within these highly “druggable” gene classes. Most likely due to their great sequence diversity rendering homology searches across species more difficult, the sequencing of the human genome had revealed many new GPCRs. In our survey, this gene class produced the largest fraction of weakly characterized genes leaving ample opportunity for further investigations. The small protein family of nuclear receptors, to the contrary, displayed less room for development with the majority of genes deeply characterized, possibly due to their small number and easy accessibility compared to membrane-bound receptors. Even though both families of ion-channel genes lacked genes remaining at the very low end of the characterization scale, there still was prospect of novel therapeutic applications owing to hitherto untargeted subunits of target protein complexes and family members regulating protein structure. Given that over 50% of current drugs target G protein-coupled receptors, nuclear receptors and ion channels and that the majority of new drugs target precedent domains, to tap the full potential of these families by highlighting any neglect may be important for the advancement of therapeutic approaches by small molecule drugs.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>As opposed to the common belief that corporate interest had a negative impact on scientific progress in particular through the filing of patents on genes and gene-derived sequences, we observed that patenting did not hamper gene characterization. As already apparent from the analysis of “druggable” genes both within specific gene classes (<a href="#pone-0001440-g004">Figure 4A</a>) and the whole genome (<a href="#pone-0001440-g004">Figure 4B</a>) and based on the fact that the large majority of drug target genes (86%) was protected by patents, drug targets mostly clustered at the high end of the characterization scale. Analyzing annotation progress of patented versus non-patented genes across an extended time period unveiled an association of functional characterization and gene patenting. While it is probable that patenting was biased towards proteins with detectable protein domains now recorded by genome annotation engines, the increase in the patented gene scores was heavily driven by new publications. The scientific community seemed to have been focusing on patented genes in their efforts to study the molecular function of genes.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Another capacity of the GCI scoring system is to identify areas of “neglect” within either groups of genes or the whole genome. Following the annotation status of all human genes since the release of the first draft of the human genome, we identified a large pool (~14500) of genes essentially uncharacterized to the present day. These genes have minimal functional annotation with scores below 3.5 and represent over 75% of genes neither protected by patents nor targeted by small molecule drugs. These findings clearly illustrate the inestimable potential still hidden within the human genome in that a considerable portion of genes are shrouded in darkness, awaiting attention and functional elucidation.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Using the GCI scoring system to delineate groups of scarcely annotated genes could aide decision-making for the allocation of research funds. Current research funds are distributed in large part on the basis of extensive preliminary data; there is great reluctance to fund projects deemed as “high-risk” which would include the study of uncharacterized genes. As indicated in this report, vast genome areas have remained unexplored. The availability of GCI scores could direct resources to these neglected areas. During grant reviews, GCI could help identify proposals of greater novelty (a weighting factor for many funding groups) by serving as a novelty measure.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>The GCI produced by this study represents a novel direction in bioinformatics research. By classifying genes based on a qualitative score assigned by humans, it represents opinion research. The existing GCI scores reflect the state of opinion at the time of the survey. For instance, in 19 cases evaluators assigned a score of 10 to a gene. While this perceived high characterization reflects the most deeply studied genes, it is unlikely that future scientists would view our present state of gene knowledge as complete. As research advances, the perspectives of scientists will without doubt change and our expectations for the required properties of genes to be considered characterized will become more stringent. In addition, over time, the available types and sources of data change. Therefore, the GCI scoring function will require periodic updating to reflect available annotation resources and changing opinions of researchers. Besides, it would be desirable in the future to develop GCI scores for model organism genes to determine how the characterization of orthologs of human genes influences the progression of the GCI. The GCI scoring system could serve as an important tool for focusing efforts and resources on the study of weakly characterized genes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Analysis of results from survey</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>As described, the survey of 52 scientists produced an initial collection of scores for the 100 reference genes with an average of 4.6 scores per gene. We implemented quality control procedures to identify unusual results. For each respondent, we compared the set of assigned gene scores to the mean score for each gene using a Pearson correlation coefficient. Scores from 3 reviewers with correlation scores more than 2 standard deviations from the average correlation score were excluded from the reference collection (Supplementary <a href="#pone.0001440.s004">Figure S2</a>). As an additional quality control procedure, we reviewed individual gene scores for unaccounted deviations from the average. No individual gene or reviewer appeared particularly unusual after this final review, although we identified 3 scores as outside the observed range of scores and thus excluded them. After removal of outlying scores, an average of 4.2 scores per gene remained in the reference set.</p>


<h4>Model selection</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We used the R Statistical Language (<a href="http://www.r-project.org/">http://www.r-project.org/</a>) to implement all gene characterization score prediction methods. Initially, we conducted a performance review of linear models (LM), regression trees (RT), neural nets (NN), support vector machines (SVM) and multivariate additive regression splines (MARS). All of the approaches exhibited similar and adequate accuracy for GCI scoring, as assessed by cross-validation.</p>


<h4>Model validation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We verified the quality of the model by a Leave-One-Out (LOO) cross-validation method. For each of the 100 genes, we constructed a model with the other 99 genes and compared the score assigned by the model to the excluded gene to the reference score. The Root Mean Square (RMS) error provided an overall performance measure. We tested a number of data transformations including binary values, binned, log, maximum value normalization, and z-score normalization. Of the 5 model frameworks tested, SVM and MARS performed best. The SVM model failed to assign scores across the entire scale and was therefore set aside. For the MARS model a range of values for the parameters “degree“ (range 1 to 3) and “penalty“ (range 0 to 6) was tested. We presented the best model with degree 1 and penalty of 1. The MARS procedure selected a subset of available attributes to optimize the fit to training data. The “degree” controlled the maximum number of splines that could be used for each attribute across the range of the expected predictions, and the “penalty” was used to decide if an additional attribute should be utilized in constructing the model. Increasing the “penalty” reduced the number of attributes used by the final model. Attributes utilized in the best MARS model are shown in <a href="#pone-0001440-t001">Table 1</a>. A complete documentation of the model validation is provided in the online Supplementary Material.</p>


<h4>Calculation of Resnik scores</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The usage of GO terms varies considerably–a rather general GO term may be associated with many genes and is therefore not very explicit, whereas a very detailed term will only be used for few genes for which the specific function may be applied. Resnik self-similarity scores for the GO <em>molecular function</em> taxonomy provide a numerical measure for the depth of GO annotations for individual genes. The Resnik self-similarity score is based on the lowest (most detailed) parent node in the GO annotation hierarchy for a specific gene by extracting the maximum information content of the node <a href="#pone.0001440-Resnik1">[22]</a>. The information content is linked to the probability of observing the GO term based on counting how many times the GO term appears in annotated gene products which is reported regularly by the GO Consortium <a href="#pone.0001440-GO1">[38]</a>. The GOSim 1.0.2 package <a href="#pone.0001440-Frohlich1">[39]</a> was used to compute the scores in the R 2.5.1 statistics package (<a href="http://www.r-project.org/">http://www.r-project.org/</a>).</p>


<h4>Analysis of patented genes</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We screened NCBI's patented nucleotide sequence database for human transcripts and matched a total of 14295 GenBank accessions to patent records. To assess GCI score distributions across several years, we mapped the patented genes to either GeneLynx or Entrez Gene identifiers and applied the GCI scoring model based on successive GeneLynx <a href="#pone.0001440-Lenhard1">[10]</a> and Entrez Gene <a href="#pone.0001440-Maglott1">[9]</a> releases.</p>


<h4>Gene mappings between GeneLynx and Entrez Gene</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>For all analyses of datasets comparing data from the latest GeneLynx release (April 2004) and earlier with datasets from more recent Entrez Gene releases, mappings between GeneLynx and Entrez Gene identifiers were performed via EnsEMBL <a href="#pone.0001440-Hubbard1">[40]</a> gene and transcript intermediates. Due to substantial changes in gene mappings between May 2001 and September 2007, only 16494 individual genes could be directly mapped between GeneLynx and Entrez Gene with a bias towards genes with more extensive functional annotation (see Supplementary Material).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001440"></div><a name="pone.0001440.s001" id="pone.0001440.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001440.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Collection of reference genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001440.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB PDF)</p>
<a name="pone.0001440.s002" id="pone.0001440.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001440.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Initial list of 40 gene attributes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001440.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0001440.s003" id="pone.0001440.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001440.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Evaluator-assigned GCI score distribution</p>
<p class="siDoi">doi:10.1371/journal.pone.0001440.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0001440.s004" id="pone.0001440.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001440.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Outlier evaluator</p>
<p class="siDoi">doi:10.1371/journal.pone.0001440.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Dr. Robert Cassidy for insightful discussions and critical reading of the manuscript and Drs. Christer Höög and Mark Gurney for the vision to study uncharacterized proteins. Dr. James Fickett suggested that one could quantitatively assign a characterization score to genes.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: WW BL DK RP. Performed the experiments: RP. Analyzed the data: JB DK DY RP WC. Contributed reagents/materials/analysis tools: CW. Wrote the paper: WW DK. Other: Infrastructure: BL. Funding: CW WW. Statistical models: JB. Supervision: WW. Web site: DY RP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001440-Birney1" id="pone.0001440-Birney1"></a>Birney E, Stamatoyannopoulos JA, Dutta A, Guigo R, Gingeras TR, et al.  (2007) Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. Nature  447: 799–816.  <ul class="find" data-citedArticleID="1042158"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+and+analysis+of+functional+elements+in+1%25+of+the+human+genome+by+the+ENCODE+pilot+project.&amp;auth=&amp;atitle=Identification+and+analysis+of+functional+elements+in+1%25+of+the+human+genome+by+the+ENCODE+pilot+project." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+analysis+of+functional+elements+in+1%25+of+the+human+genome+by+the+ENCODE+pilot+project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+analysis+of+functional+elements+in+1%25+of+the+human+genome+by+the+ENCODE+pilot+project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001440-Bogue1" id="pone.0001440-Bogue1"></a>Bogue MA, Grubb SC (2004) The Mouse Phenome Project. Genetica  122: 71–74.  <ul class="find" data-citedArticleID="1042160" data-doi="10.1007/s10709-004-1438-4"><li><a href="http://dx.doi.org/10.1007/s10709-004-1438-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mouse+Phenome+Project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mouse+Phenome+Project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001440-Mashimo1" id="pone.0001440-Mashimo1"></a>Mashimo T, Voigt B, Kuramoto T, Serikawa T (2005) Rat Phenome Project: the untapped potential of existing rat strains. J Appl Physiol  98: 371–379.  <ul class="find" data-citedArticleID="1042198" data-doi="10.1152/japplphysiol.01006.2004"><li><a href="http://dx.doi.org/10.1152/japplphysiol.01006.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rat+Phenome+Project%3A+the+untapped+potential+of+existing+rat+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rat+Phenome+Project%3A+the+untapped+potential+of+existing+rat+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001440-Rual1" id="pone.0001440-Rual1"></a>Rual JF, Ceron J, Koreth J, Hao T, Nicot AS, et al.  (2004) Toward improving Caenorhabditis elegans phenome mapping with an ORFeome-based RNAi library. Genome Res  14: 2162–2168.  <ul class="find" data-citedArticleID="1042218" data-doi="10.1101/gr.2505604"><li><a href="http://dx.doi.org/10.1101/gr.2505604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+improving+Caenorhabditis+elegans+phenome+mapping+with+an+ORFeome-based+RNAi+library." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+improving+Caenorhabditis+elegans+phenome+mapping+with+an+ORFeome-based+RNAi+library.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001440-Gewin1" id="pone.0001440-Gewin1"></a>Gewin V (2005) A golden age of brain exploration. PLoS Biol  3: e24.  <ul class="find" data-citedArticleID="1042178" data-doi="10.1371/journal.pbio.0030024"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+golden+age+of+brain+exploration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+golden+age+of+brain+exploration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001440-Eisenthal1" id="pone.0001440-Eisenthal1"></a>Eisenthal Y, Dror G, Ruppin E (2006) Facial attractiveness: beauty and the machine. Neural Comput  18: 119–142.  <ul class="find" data-citedArticleID="1042170" data-doi="10.1162/089976606774841602"><li><a href="http://dx.doi.org/10.1162/089976606774841602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Facial+attractiveness%3A+beauty+and+the+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Facial+attractiveness%3A+beauty+and+the+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001440-Gunes1" id="pone.0001440-Gunes1"></a>Gunes H, Piccardi M, Jan TAutomated classification of female facial beauty by image analysis and supervised learning; 2004; San Jose, CA, USA. Proc. of the SPIE. pp. 968–978.  <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0001440-Ashburner1" id="pone.0001440-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="1042150"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001440-Maglott1" id="pone.0001440-Maglott1"></a>Maglott D, Ostell J, Pruitt KD, Tatusova T (2007) Entrez Gene: gene-centered information at NCBI. Nucleic Acids Res  35: D26–31.  <ul class="find" data-citedArticleID="1042196" data-doi="10.1093/nar/gkl993"><li><a href="http://dx.doi.org/10.1093/nar/gkl993" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Entrez+Gene%3A+gene-centered+information+at+NCBI." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Entrez+Gene%3A+gene-centered+information+at+NCBI.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001440-Lenhard1" id="pone.0001440-Lenhard1"></a>Lenhard B, Hayes WS, Wasserman WW (2001) GeneLynx: a gene-centric portal to the human genome. Genome Res  11: 2151–2157.  <ul class="find" data-citedArticleID="1042194" data-doi="10.1101/gr.199801"><li><a href="http://dx.doi.org/10.1101/gr.199801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneLynx%3A+a+gene-centric+portal+to+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneLynx%3A+a+gene-centric+portal+to+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001440-Benson1" id="pone.0001440-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2007) GenBank. Nucleic Acids Res  35: D21–25.  <ul class="find" data-citedArticleID="1042154" data-doi="10.1093/nar/gkl986"><li><a href="http://dx.doi.org/10.1093/nar/gkl986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001440-Vapnik1" id="pone.0001440-Vapnik1"></a>Vapnik V (1995) The Nature of Statistical Learning Theory: NY Springer.  <ul class="find" data-citedArticleID="1042222" data-doi="10.1007/978-1-4757-2440-0"><li><a href="http://dx.doi.org/10.1007/978-1-4757-2440-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Nature+of+Statistical+Learning+Theory%3A+NY+Springer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Nature+of+Statistical+Learning+Theory%3A+NY+Springer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001440-Friedman1" id="pone.0001440-Friedman1"></a>Friedman JH (1991) Multivariate adaptive regression splines. The Annals of Statistics  19: 1–141.  <ul class="find" data-citedArticleID="1042172" data-doi="10.1214/aos/1176347973"><li><a href="http://dx.doi.org/10.1214/aos/1176347973" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multivariate+adaptive+regression+splines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multivariate+adaptive+regression+splines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001440-Mulder1" id="pone.0001440-Mulder1"></a>Mulder NJ, Apweiler R, Attwood TK, Bairoch A, Bateman A, et al.  (2007) New developments in the InterPro database. Nucleic Acids Res  35: D224–228.  <ul class="find" data-citedArticleID="1042202" data-doi="10.1093/nar/gkl841"><li><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+developments+in+the+InterPro+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+developments+in+the+InterPro+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001440-Attwood1" id="pone.0001440-Attwood1"></a>Attwood TK, Bradley P, Flower DR, Gaulton A, Maudling N, et al.  (2003) PRINTS and its automatic supplement, prePRINTS. Nucleic Acids Res  31: 400–402.  <ul class="find" data-citedArticleID="1042152" data-doi="10.1093/nar/gkg030"><li><a href="http://dx.doi.org/10.1093/nar/gkg030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PRINTS+and+its+automatic+supplement%2C+prePRINTS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PRINTS+and+its+automatic+supplement%2C+prePRINTS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001440-Hulo1" id="pone.0001440-Hulo1"></a>Hulo N, Bairoch A, Bulliard V, Cerutti L, De Castro E, et al.  (2006) The PROSITE database. Nucleic Acids Res  34: D227–230.  <ul class="find" data-citedArticleID="1042190" data-doi="10.1093/nar/gkp885"><li><a href="http://dx.doi.org/10.1093/nar/gkp885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PROSITE+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PROSITE+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001440-Kanehisa1" id="pone.0001440-Kanehisa1"></a>Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, et al.  (2006) From genomics to chemical genomics: new developments in KEGG. Nucleic Acids Res  34: D354–357.  <ul class="find" data-citedArticleID="1042192" data-doi="10.1093/nar/gkj102"><li><a href="http://dx.doi.org/10.1093/nar/gkj102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001440-Pruitt1" id="pone.0001440-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2007) NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res  35: D61–65.  <ul class="find" data-citedArticleID="1042212" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+reference+sequences+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001440-Wu1" id="pone.0001440-Wu1"></a>Wu CH, Apweiler R, Bairoch A, Natale DA, Barker WC, et al.  (2006) The Universal Protein Resource (UniProt): an expanding universe of protein information. Nucleic Acids Res  34: D187–191.  <ul class="find" data-citedArticleID="1042226" data-doi="10.1093/nar/gkj161"><li><a href="http://dx.doi.org/10.1093/nar/gkj161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Universal+Protein+Resource+%28UniProt%29%3A+an+expanding+universe+of+protein+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Universal+Protein+Resource+%28UniProt%29%3A+an+expanding+universe+of+protein+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001440-Berman1" id="pone.0001440-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al.  (2000) The Protein Data Bank. Nucleic Acids Res  28: 235–242.  <ul class="find" data-citedArticleID="1042156" data-doi="10.1093/nar/28.1.235"><li><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001440-Camon1" id="pone.0001440-Camon1"></a>Camon E, Magrane M, Barrell D, Lee V, Dimmer E, et al.  (2004) The Gene Ontology Annotation (GOA) Database: sharing knowledge in Uniprot with Gene Ontology. Nucleic Acids Res  32: D262–266.  <ul class="find" data-citedArticleID="1042162" data-doi="10.1093/nar/gkh021"><li><a href="http://dx.doi.org/10.1093/nar/gkh021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Gene+Ontology+Annotation+%28GOA%29+Database%3A+sharing+knowledge+in+Uniprot+with+Gene+Ontology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Gene+Ontology+Annotation+%28GOA%29+Database%3A+sharing+knowledge+in+Uniprot+with+Gene+Ontology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001440-Resnik1" id="pone.0001440-Resnik1"></a>Resnik PUsing Information Content to Evaluate Semantic Similarity in a Taxonomy; 1995. pp. 448–453.  <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001440-Overington1" id="pone.0001440-Overington1"></a>Overington JP, Al-Lazikani B, Hopkins AL (2006) How many drug targets are there? Nat Rev Drug Discov  5: 993–996.  <ul class="find" data-citedArticleID="1042206" data-doi="10.1038/nrd2199"><li><a href="http://dx.doi.org/10.1038/nrd2199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+many+drug+targets+are+there%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+many+drug+targets+are+there%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001440-Wishart1" id="pone.0001440-Wishart1"></a>Wishart DS, Knox C, Guo AC, Shrivastava S, Hassanali M, et al.  (2006) DrugBank: a comprehensive resource for in silico drug discovery and exploration. Nucleic Acids Res  34: D668–672.  <ul class="find" data-citedArticleID="1042224" data-doi="10.1093/nar/gkj067"><li><a href="http://dx.doi.org/10.1093/nar/gkj067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DrugBank%3A+a+comprehensive+resource+for+in+silico+drug+discovery+and+exploration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DrugBank%3A+a+comprehensive+resource+for+in+silico+drug+discovery+and+exploration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001440-Horn1" id="pone.0001440-Horn1"></a>Horn F, Bettler E, Oliveira L, Campagne F, Cohen FE, et al.  (2003) GPCRDB information system for G protein-coupled receptors. Nucleic Acids Res  31: 294–297.  <ul class="find" data-citedArticleID="1042184" data-doi="10.1093/nar/gkg103"><li><a href="http://dx.doi.org/10.1093/nar/gkg103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GPCRDB+information+system+for+G+protein-coupled+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GPCRDB+information+system+for+G+protein-coupled+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001440-Oh1" id="pone.0001440-Oh1"></a>Oh JH, Yang JO, Hahn Y, Kim MR, Byun SS, et al.  (2005) Transcriptome analysis of human gastric cancer. Mamm Genome  16: 942–954.  <ul class="find" data-citedArticleID="1042204" data-doi="10.1007/s00335-005-0075-2"><li><a href="http://dx.doi.org/10.1007/s00335-005-0075-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+analysis+of+human+gastric+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+analysis+of+human+gastric+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001440-Horn2" id="pone.0001440-Horn2"></a>Horn F, Vriend G, Cohen FE (2001) Collecting and harvesting biological data: the GPCRDB and NucleaRDB information systems. Nucleic Acids Res  29: 346–349.  <ul class="find" data-citedArticleID="1042186" data-doi="10.1093/nar/29.1.346"><li><a href="http://dx.doi.org/10.1093/nar/29.1.346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collecting+and+harvesting+biological+data%3A+the+GPCRDB+and+NucleaRDB+information+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collecting+and+harvesting+biological+data%3A+the+GPCRDB+and+NucleaRDB+information+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001440-Schulman1" id="pone.0001440-Schulman1"></a>Schulman IG, Heyman RA (2004) The flip side: Identifying small molecule regulators of nuclear receptors. Chem Biol  11: 639–646.  <ul class="find" data-citedArticleID="1042220" data-doi="10.1016/s1074-5521(04)00120-6"><li><a href="http://dx.doi.org/10.1016/s1074-5521(04)00120-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+flip+side%3A+Identifying+small+molecule+regulators+of+nuclear+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+flip+side%3A+Identifying+small+molecule+regulators+of+nuclear+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001440-Connolly1" id="pone.0001440-Connolly1"></a>Connolly CN, Wafford KA (2004) The Cys-loop superfamily of ligand-gated ion channels: the impact of receptor structure on function. Biochem Soc Trans  32: 529–534.  <ul class="find" data-citedArticleID="1042164" data-doi="10.1042/bst0320529"><li><a href="http://dx.doi.org/10.1042/bst0320529" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Cys-loop+superfamily+of+ligand-gated+ion+channels%3A+the+impact+of+receptor+structure+on+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Cys-loop+superfamily+of+ligand-gated+ion+channels%3A+the+impact+of+receptor+structure+on+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001440-Donizelli1" id="pone.0001440-Donizelli1"></a>Donizelli M, Djite MA, Le Novere N (2006) LGICdb: a manually curated sequence database after the genomes. Nucleic Acids Res  34: D267–269.  <ul class="find" data-citedArticleID="1042168" data-doi="10.1093/nar/gkj104"><li><a href="http://dx.doi.org/10.1093/nar/gkj104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LGICdb%3A+a+manually+curated+sequence+database+after+the+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LGICdb%3A+a+manually+curated+sequence+database+after+the+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001440-Mizielinska1" id="pone.0001440-Mizielinska1"></a>Mizielinska S, Greenwood S, Connolly CN (2006) The role of GABAA receptor biogenesis, structure and function in epilepsy. Biochem Soc Trans  34: 863–867.  <ul class="find" data-citedArticleID="1042200" data-doi="10.1042/bst0340863"><li><a href="http://dx.doi.org/10.1042/bst0340863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+GABAA+receptor+biogenesis%2C+structure+and+function+in+epilepsy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+GABAA+receptor+biogenesis%2C+structure+and+function+in+epilepsy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001440-Curran1" id="pone.0001440-Curran1"></a>Curran ME (1998) Potassium ion channels and human disease: phenotypes to drug targets? Curr Opin Biotechnol  9: 565–572.  <ul class="find" data-citedArticleID="1042166" data-doi="10.1016/s0958-1669(98)80133-x"><li><a href="http://dx.doi.org/10.1016/s0958-1669(98)80133-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Potassium+ion+channels+and+human+disease%3A+phenotypes+to+drug+targets%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Potassium+ion+channels+and+human+disease%3A+phenotypes+to+drug+targets%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001440-Adler1" id="pone.0001440-Adler1"></a>Adler RG (1992) Genome research: fulfilling the public's expectations for knowledge and commercialization. Science  257: 908–914.  <ul class="find" data-citedArticleID="1042148" data-doi="10.1126/science.1502557"><li><a href="http://dx.doi.org/10.1126/science.1502557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+research%3A+fulfilling+the+public%27s+expectations+for+knowledge+and+commercialization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+research%3A+fulfilling+the+public%27s+expectations+for+knowledge+and+commercialization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001440-Resnik2" id="pone.0001440-Resnik2"></a>Resnik DB (2003) Are DNA patents bad for medicine? Health Policy  65: 181–197.  <ul class="find" data-citedArticleID="1042216" data-doi="10.1016/s0168-8510(02)00206-3"><li><a href="http://dx.doi.org/10.1016/s0168-8510(02)00206-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+DNA+patents+bad+for+medicine%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+DNA+patents+bad+for+medicine%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001440-Heller1" id="pone.0001440-Heller1"></a>Heller MA, Eisenberg RS (1998) Can patents deter innovation? The anticommons in biomedical research. Science  280: 698–701.  <ul class="find" data-citedArticleID="1042182" data-doi="10.1126/science.280.5364.698"><li><a href="http://dx.doi.org/10.1126/science.280.5364.698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+patents+deter+innovation%3F+The+anticommons+in+biomedical+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+patents+deter+innovation%3F+The+anticommons+in+biomedical+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001440-Pennisi1" id="pone.0001440-Pennisi1"></a>Pennisi E (2003) Human genome. Reaching their goal early, sequencing labs celebrate. Science  300: 409.  <ul class="find" data-citedArticleID="1042208" data-doi="10.1126/science.300.5618.409"><li><a href="http://dx.doi.org/10.1126/science.300.5618.409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+genome.+Reaching+their+goal+early%2C+sequencing+labs+celebrate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+genome.+Reaching+their+goal+early%2C+sequencing+labs+celebrate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001440-Pennisi2" id="pone.0001440-Pennisi2"></a>Pennisi E (2003) Human genome. A low number wins the GeneSweep Pool. Science  300: 1484.  <ul class="find" data-citedArticleID="1042210" data-doi="10.1126/science.300.5625.1484b"><li><a href="http://dx.doi.org/10.1126/science.300.5625.1484b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+genome.+A+low+number+wins+the+GeneSweep+Pool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+genome.+A+low+number+wins+the+GeneSweep+Pool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001440-GO1" id="pone.0001440-GO1"></a>GO Consortium (2006) The Gene Ontology (GO) project in 2006. Nucleic Acids Res  34: D322–326.  <ul class="find" data-citedArticleID="1042176" data-doi="10.1093/nar/gkj021"><li><a href="http://dx.doi.org/10.1093/nar/gkj021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Gene+Ontology+%28GO%29+project+in+2006." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Gene+Ontology+%28GO%29+project+in+2006.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001440-Frohlich1" id="pone.0001440-Frohlich1"></a>Frohlich H, Speer N, Poustka A, Beissbarth T (2007) GOSim–an R-package for computation of information theoretic GO similarities between terms and gene products. BMC Bioinformatics  8: 166.  <ul class="find" data-citedArticleID="1042174" data-doi="10.1186/1471-2105-8-166"><li><a href="http://dx.doi.org/10.1186/1471-2105-8-166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GOSim%E2%80%93an+R-package+for+computation+of+information+theoretic+GO+similarities+between+terms+and+gene+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GOSim%E2%80%93an+R-package+for+computation+of+information+theoretic+GO+similarities+between+terms+and+gene+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001440-Hubbard1" id="pone.0001440-Hubbard1"></a>Hubbard TJ, Aken BL, Beal K, Ballester B, Caccamo M, et al.  (2007) Ensembl 2007. Nucleic Acids Res  35: D610–617.  <ul class="find" data-citedArticleID="1042188" data-doi="10.1093/nar/gkl996"><li><a href="http://dx.doi.org/10.1093/nar/gkl996" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2007." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2007.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.XML" value="114472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.PDF" value="404160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t001.PNG_L" value="46660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t001.PNG_M" value="59838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t001.PNG_S" value="10578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t001.TIF" value="223780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t001.PNG_I" value="19686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t002.PNG_L" value="69518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t002.PNG_M" value="96330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t002.PNG_S" value="13441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t002.TIF" value="338274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t002.PNG_I" value="31053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g001.PNG_L" value="1360680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g001.PNG_M" value="248225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g001.PNG_S" value="10086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g001.TIF" value="2232642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g001.PNG_I" value="83021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g002.PNG_L" value="533765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g002.PNG_M" value="50335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g002.PNG_S" value="3036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g002.TIF" value="667496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g002.PNG_I" value="17146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t003.PNG_L" value="203319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t003.PNG_M" value="116630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t003.PNG_S" value="12439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t003.TIF" value="917838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t003.PNG_I" value="35474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t004.PNG_L" value="131212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t004.PNG_M" value="78041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t004.PNG_S" value="10244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t004.TIF" value="601812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t004.PNG_I" value="23477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t005.PNG_L" value="57025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t005.PNG_M" value="73381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t005.PNG_S" value="12237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t005.TIF" value="251834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t005.PNG_I" value="25353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g003.PNG_L" value="183597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g003.PNG_M" value="27095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g003.PNG_S" value="5320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g003.TIF" value="781818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g003.PNG_I" value="9990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g004.PNG_L" value="707020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g004.PNG_M" value="74282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g004.PNG_S" value="9604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g004.TIF" value="2231346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g004.PNG_I" value="36964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g005.PNG_L" value="478562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g005.PNG_M" value="126019"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g005.PNG_S" value="8928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g005.TIF" value="952370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g005.PNG_I" value="47015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t006.PNG_L" value="49194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t006.PNG_M" value="63792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t006.PNG_S" value="11306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t006.TIF" value="212402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.t006.PNG_I" value="22150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g006.PNG_L" value="1825500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g006.PNG_M" value="105525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g006.PNG_S" value="12347"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g006.TIF" value="3788624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.g006.PNG_I" value="45667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.s001.PDF" value="79586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.s002.PDF" value="23123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.s003.PDF" value="17159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001440.s004.PDF" value="13764"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001440&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001440" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001440&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001440&volume=&issue=&title=Gene Characterization Index: Assessing the Depth of Gene Annotation&author_name=Danielle%20Kemmer%2C%20Raf%20M.%20Podowski%2C%20Dimas%20Yusuf%2C%20Jochen%20Brumm%2C%20Warren%20Cheung%2C%20Claes%20Wahlestedt%2C%20Boris%20Lenhard%2C%20Wyeth%20W.%20Wasserman&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001440" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440&amp;t=Gene%20Characterization%20Index%3A%20Assessing%20the%20Depth%20of%20Gene%20Annotation" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440&title=Gene%20Characterization%20Index%3A%20Assessing%20the%20Depth%20of%20Gene%20Annotation&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440&amp;title=Gene%20Characterization%20Index%3A%20Assessing%20the%20Depth%20of%20Gene%20Annotation" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001440&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Gene Characterization Index: Assessing the Depth of Gene Annotation';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001440';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Gene%20Characterization%20Index%3A%20Assessing%20the%20Depth%20of%20Gene%20Annotation http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001440" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001440" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drug+discovery%22" title="Search for articles in the subject area:'Drug discovery'"><div class="flagText">Drug discovery</div></a>
              <div data-categoryid="19989" data-articleid="25708"
                   data-categoryname="Drug discovery"
                   class="flagImage" title="Flag 'Drug discovery' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Functional+genomics%22" title="Search for articles in the subject area:'Functional genomics'"><div class="flagText">Functional genomics</div></a>
              <div data-categoryid="17853" data-articleid="25708"
                   data-categoryname="Functional genomics"
                   class="flagImage" title="Flag 'Functional genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="47601" data-articleid="25708"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+annotation%22" title="Search for articles in the subject area:'Genome annotation'"><div class="flagText">Genome annotation</div></a>
              <div data-categoryid="40359" data-articleid="25708"
                   data-categoryname="Genome annotation"
                   class="flagImage" title="Flag 'Genome annotation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25708"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25708"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="25708"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Scientists%22" title="Search for articles in the subject area:'Scientists'"><div class="flagText">Scientists</div></a>
              <div data-categoryid="46109" data-articleid="25708"
                   data-categoryname="Scientists"
                   class="flagImage" title="Flag 'Scientists' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7727'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9638'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3118&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13877">A few links to comments about the paper</a><br>Posted by Wyeth_Wasserman</p>
      <p><a href="/annotation/listThread.action?root=4903">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
