

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000770"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000770" />

    <meta name="citation_title" content="Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America"/>
    <meta itemprop="name" content="Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America"/>

      <meta name="citation_author" content="Raymond K. Auerbach"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Apichai Tuanyok"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="William S. Probert"/>
            <meta name="citation_author_institution" content="Microbial Diseases Laboratory, California Department of Public Health, Richmond, California, United States of America"/>
      <meta name="citation_author" content="Leo Kenefic"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Amy J. Vogler"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="David C. Bruce"/>
            <meta name="citation_author_institution" content="Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Christine Munk"/>
            <meta name="citation_author_institution" content="Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Thomas S. Brettin"/>
            <meta name="citation_author_institution" content="Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America"/>
      <meta name="citation_author" content="Mark Eppinger"/>
            <meta name="citation_author_institution" content="J. Craig Venter Institute, The Institute for Genomic Research, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="Jacques Ravel"/>
            <meta name="citation_author_institution" content="J. Craig Venter Institute, The Institute for Genomic Research, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="David M. Wagner"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
      <meta name="citation_author" content="Paul Keim"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America"/>
            <meta name="citation_author_institution" content="Pathogen Genomics Division, Translational Genomics Research Institute, Phoenix, Arizona, United States of America"/>

    <meta name="citation_date" content="2007/8/22"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000770.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e770"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Yersinia pestis–etiologic agent of plague.; citation_author=RD Perry; citation_author=JD Fetherston; citation_journal_title=Clin Microbiol Rev; citation_volume=10; citation_number=1; citation_pages=35-66; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Plague as a biological weapon: medical and public health management. Working Group on Civilian Biodefense.; citation_author=TV Inglesby; citation_author=DT Dennis; citation_author=DA Henderson; citation_author=JG Bartlett; citation_author=MS Ascher; citation_journal_title=JAMA; citation_volume=283; citation_number=2; citation_pages=2281-2290; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiple antimicrobial resistance in plague: an emerging public health risk.; citation_author=TJ Welch; citation_author=WF Fricke; citation_author=PF McDermott; citation_author=DG White; citation_author=ML Rosso; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=3; citation_pages=e309; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Age, descent, and genetic diversity within Yersinia pestis.; citation_author=M Achtman; citation_number=4; citation_pages=432; citation_date=2004; citation_publisher=Horizon Bioscience; " />
      <meta name="citation_reference" content="citation_title=Microevolution and history of the plague bacillus, Yersinia pestis.; citation_author=M Achtman; citation_author=G Morelli; citation_author=P Zhu; citation_author=T Wirth; citation_author=I Diehl; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=5; citation_pages=17837-17842; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=[Varieties of Pasteurella pestis; new hypothesis.].; citation_author=R Devignat; citation_journal_title=Bull World Health Organ; citation_volume=4; citation_number=6; citation_pages=247-263; citation_date=1951; " />
      <meta name="citation_reference" content="citation_title=A history of plague in the United States of America.; citation_author=VB Link; citation_journal_title=Public Health Monogr; citation_volume=26; citation_number=7; citation_pages=1-120; citation_date=1955; " />
      <meta name="citation_reference" content="citation_title=Interspecific comparisons of sylvatic plague in prairie dogs.; citation_author=JF Cully; citation_author=ES Williams; citation_journal_title=Journal of Mammalogy; citation_volume=82; citation_number=8; citation_pages=894-905; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Yersinia pestis, the causative agent of plague.; citation_author=J Parkhill; citation_author=BW Wren; citation_author=NR Thomson; citation_author=RW Titball; citation_author=MT Holden; citation_journal_title=Nature; citation_volume=413; citation_number=9; citation_pages=523-527; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Differential plague-transmission dynamics determine Yersinia pestis population genetic structure on local, regional, and global scales.; citation_author=JM Girard; citation_author=DM Wagner; citation_author=AJ Vogler; citation_author=C Keys; citation_author=CJ Allender; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=10; citation_pages=8408-8413; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A North American Yersinia pestis draft genome sequence: SNPs and phylogenetic analysis.; citation_author=JW Touchman; citation_author=DM Wagner; citation_author=J Hao; citation_author=SD Mastrian; citation_author=MK Shah; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=11; citation_pages=e220; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Yersinia pestis KIM.; citation_author=W Deng; citation_author=V Burland; citation_author=G Plunkett; citation_author=A Boutin; citation_author=GF Mayhew; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=12; citation_pages=4601-4611; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Yersinia pestis strains Antiqua and Nepal516: evidence of gene reduction in an emerging pathogen.; citation_author=PS Chain; citation_author=P Hu; citation_author=SA Malfatti; citation_author=L Radnedge; citation_author=F Larimer; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=13; citation_pages=4453-4463; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome rearrangement distances and gene order phylogeny in gamma-Proteobacteria.; citation_author=E Belda; citation_author=A Moya; citation_author=FJ Silva; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=14; citation_pages=1456-1467; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Yersinia pestis strain 91001, an isolate avirulent to humans.; citation_author=Y Song; citation_author=Z Tong; citation_author=J Wang; citation_author=L Wang; citation_author=Z Guo; citation_journal_title=DNA Res; citation_volume=11; citation_number=15; citation_pages=179-197; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of the variable-number tandem repeats in vrrA from different Bacillus anthracis isolates.; citation_author=PJ Jackson; citation_author=EA Walthers; citation_author=AS Kalif; citation_author=KL Richmond; citation_author=DM Adair; citation_journal_title=Appl Environ Microbiol; citation_volume=63; citation_number=16; citation_pages=1400-1405; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular evolution and diversity in Bacillus anthracis as detected by amplified fragment length polymorphism markers.; citation_author=P Keim; citation_author=A Kalif; citation_author=J Schupp; citation_author=K Hill; citation_author=SE Travis; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=17; citation_pages=818-824; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing.; citation_author=D Gordon; citation_author=C Abajian; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=18; citation_pages=195-202; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Mauve: multiple alignment of conserved genomic sequence with rearrangements.; citation_author=AC Darling; citation_author=B Mau; citation_author=FR Blattner; citation_author=NT Perna; citation_journal_title=Genome Res; citation_volume=14; citation_number=19; citation_pages=1394-1403; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=20; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes.; citation_author=S Kurtz; citation_author=A Phillippy; citation_author=AL Delcher; citation_author=M Smoot; citation_author=M Shumway; citation_journal_title=Genome Biol; citation_volume=5; citation_number=21; citation_pages=R12; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=GRIMM: genome rearrangements web server.; citation_author=G Tesler; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=22; citation_pages=492-493; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America"/>
    <meta name="twitter:description" content="BackgroundYersinia pestis, the etiologic agent of plague, was responsible for several devastating epidemics throughout history and is currently of global importance to current public heath and biodefense efforts. Y. pestis is widespread in the Western United States. Because Y. pestis was first introduced to this region just over 100 years ago, there has been little time for genetic diversity to accumulate. Recent studies based upon single nucleotide polymorphisms have begun to quantify the genetic diversity of Y. pestis in North America.Methodology/Principal FindingsTo examine the evolution of Y. pestis in North America, a gapped genome sequence of CA88-4125 was generated. Sequence comparison with another North American Y. pestis strain, CO92, identified seven regions of difference (six inversions, one rearrangement), differing IS element copy numbers, and several SNPs.Conclusions/SignificanceThe relatively large number of inverted/rearranged segments suggests that North American Y. pestis strains may be undergoing inversion fixation at high rates over a short time span, contributing to higher-than-expected diversity in this region. These findings will hopefully encourage the scientific community to sequence additional Y. pestis strains from North America and abroad, leading to a greater understanding of the evolutionary history of this pathogen."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000770.g001"/>

  <meta property="og:title" content="Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3165'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7327'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8156&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000770">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000770" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000770&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    <i>Yersinia pestis</i> Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Raymond K. Auerbach, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>

                <p>
                    Current address: Program in Computational Biology and Bioinformatics, Yale University, New Haven, Connecticut, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Apichai Tuanyok, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            William S. Probert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Microbial Diseases Laboratory, California Department of Public Health, Richmond, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leo Kenefic, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amy J. Vogler, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David C. Bruce, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christine Munk, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas S. Brettin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioscience Division, Los Alamos National Laboratory, Los Alamos, New Mexico, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Eppinger, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  J. Craig Venter Institute, The Institute for Genomic Research, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacques Ravel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  J. Craig Venter Institute, The Institute for Genomic Research, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David M. Wagner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul Keim
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Paul.Keim@nau.edu">Paul.Keim@nau.edu</a></p>

                <p>Affiliations:
                  Department of Biological Sciences, Northern Arizona University, Flagstaff, Arizona, United States of America, 
                  Pathogen Genomics Division, Translational Genomics Research Institute, Phoenix, Arizona, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 22, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000770</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000770-g001" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000770-t001" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000770-t002" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000770-t003" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000770-t004" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000770-t005" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000770-t006" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000770">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000770" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Yersinia pestis</em>, the etiologic agent of plague, was responsible for several devastating epidemics throughout history and is currently of global importance to current public heath and biodefense efforts. <em>Y. pestis</em> is widespread in the Western United States. Because <em>Y. pestis</em> was first introduced to this region just over 100 years ago, there has been little time for genetic diversity to accumulate. Recent studies based upon single nucleotide polymorphisms have begun to quantify the genetic diversity of <em>Y. pestis</em> in North America.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To examine the evolution of <em>Y. pestis</em> in North America, a gapped genome sequence of CA88-4125 was generated. Sequence comparison with another North American <em>Y. pestis</em> strain, CO92, identified seven regions of difference (six inversions, one rearrangement), differing IS element copy numbers, and several SNPs.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The relatively large number of inverted/rearranged segments suggests that North American <em>Y. pestis</em> strains may be undergoing inversion fixation at high rates over a short time span, contributing to higher-than-expected diversity in this region. These findings will hopefully encourage the scientific community to sequence additional <em>Y. pestis</em> strains from North America and abroad, leading to a greater understanding of the evolutionary history of this pathogen.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Auerbach RK, Tuanyok A, Probert WS, Kenefic L, Vogler AJ, et al.  (2007) <em>Yersinia pestis</em> Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America. PLoS ONE 2(8):
          e770.
            doi:10.1371/journal.pone.0000770</p><p><strong>Academic Editor: </strong>Steven Salzberg, University of Maryland, United States of America</p><p><strong>Received:</strong> July 4, 2007; <strong>Accepted:</strong> July 25, 2007; <strong>Published:</strong> August 22, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>Funding for Y. pestis CA88-4125 sequencing was provided by the Intelligence Technology Innovation Center. JR and ME were supported with federal funds from the National Institute of Allergy and Infectious Diseases, National Institutes of Health, Department of Health and Human Services under NIAID contract NO1-AI-30071.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Yersinia pestis</em> is a Gram-negative bacterium and the causative agent of plague, a disease with global importance to public health and to biodefense efforts. <em>Y. pestis</em> is thought to have been responsible for three pandemics throughout history. The first pandemic, or Justinian's plague, occurred in Europe during the 6th century. The second pandemic lasted from the 14th to the 17th centuries and includes the Black Death that reduced Europe's population by 30–40% <a href="#pone.0000770-Perry1">[1]</a>. We are currently living within the third pandemic, or modern plague, which began in the 19th century when the disease emerged from Eastern China and was spread throughout the world via steamships. All continents except Australia and Antarctica currently possess active plague foci <a href="#pone.0000770-Perry1">[1]</a>. <em>Y. pestis</em> is an obligate pathogen that is found exclusively in arthropod vectors or mammalian hosts. The bacterium was developed as a biological weapon by the United States, the former Soviet Union, and Japan during the 20th century. It is currently classified as a Category A Select Agent by the US Centers for Disease Control and Prevention <a href="#pone.0000770-Inglesby1">[2]</a>. Although most <em>Y. pestis</em> infections are easily treated with antibiotics, an antibiotic-resistant strain has been discovered recently in Madagascar and fixation of such strains could pose a significant public health risk <a href="#pone.0000770-Welch1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p><em>Y. pestis</em> is a recently emerged clone of <em>Yersinia pseudotuberculosis</em>, evolving within the last 9,000–40,000 years <a href="#pone.0000770-Achtman1">[4]</a>, <a href="#pone.0000770-Achtman2">[5]</a>. <em>Y. pestis</em> nomenclature was originally based upon differing biochemical characteristics, dividing strains into four biovars: Orientalis, Medievalis, Antiqua, and Microtus <a href="#pone.0000770-Devignat1">[6]</a>. As new analysis methods emerged, these biovar groupings were found to inadequately reflect molecular relatedness among <em>Y. pestis</em> strains. As a result, a new nomenclature based on molecular relatedness was developed that incorporated the traditional biovar designations. The <em>Y. pestis</em> phylogeny currently has three major branches. Branch 0 contains almost all pestoides isolates and the Microtus isolate 91001 (groups 0.PE1, 0.PE2, 0.PE3, 0.PE4). Branch 1 contains all of Orientalis (1.ORI) and African Antiqua (1.ANT). Branch 2 contains all of Medievalis (2.MED) and Asian Antiqua (2.ANT) <a href="#pone.0000770-Achtman2">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>1.ORI spread throughout the world during the Third Pandemic and is the only <em>Y. pestis</em> type found in North America, having been introduced to this region within the last 125 years. A single synapomorphic SNP has been found for North American <em>Y. pestis</em>, supporting the hypothesis that North American plague is the result of a single introduction [Vogler et al, unpublished data]. In North America, plague was first documented in non-native rat populations in the 1890s and 1900s in Los Angeles, San Francisco, and Galveston <a href="#pone.0000770-Link1">[7]</a>. Plague then largely disappeared from these cities due to improved hygiene and efforts to control the non-native rat population; however it reemerged in native rat populations and was responsible for an epidemic near San Francisco in 1908 <a href="#pone.0000770-Link1">[7]</a>. After this initial introduction into native fauna, plague spread rapidly eastward through rodent populations. By 1950, <em>Y. pestis</em> reached its current North American distribution which includes the 17 westernmost states and a boundary of approximately 100°W longitude <a href="#pone.0000770-Cully1">[8]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>This note will address our early findings regarding <em>Y. pestis</em> CA88-4125 (GenBank accession: ABCD00000000), a strain isolated from a human case at Fort Hunter Liggett in Monterey County, California, in 1988. The California Department of Health ID is 88A-4125. Annotation of CA88 is currently in progress at the Enteropathogen Resource Integration Center (ERIC) and the contig sequences will soon be released into GenBank. Because plague was first introduced into North America through San Francisco, comparing CA88 to other North American Y. pestis strains may offer a glimpse into how plague has evolved as it spread eastward in this region. The CO92 genome sequence was closed and released in 2001 <a href="#pone.0000770-Parkhill1">[9]</a>. CO92 is a clinical isolate from Chafee County, Colorado, which was isolated in 1992. FV-1, a strain isolated from a natural outbreak near Flagstaff, Arizona, in 2001 <a href="#pone.0000770-Girard1">[10]</a>, is also currently available in GenBank as 400 contig sequences <a href="#pone.0000770-Touchman1">[11]</a>. CO92, FV-1, and CA88 are all members of biovar Orientalis and the 1.ORI branch. This note examines large-scale genomic differences between CA88 and CO92, two strains that are potentially very divergent among North American <em>Y. pestis</em>. Analyses were conducted <em>in silico</em> and indicate that North American <em>Y. pestis</em> have undergone rapid evolution in a very short time period.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>CA88 contigs</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The current CA88 sequence consists of ten contigs, three of which are from the three <em>Y. pestis</em> plasmids. The seven contigs from the chromosome total 4,650,262 bp, range from 43,407 to 1,676,077 bp in length, and have an average contig size of 664,323 bp.</p>


<h4>Chromosomal inversions</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Fifteen shared local collinear blocks (LCBs) were determined, six of which were inverted and one of which was rearranged between CO92 and CA88 (<a href="#pone-0000770-g001">Figure 1</a>). These seven large regions of difference (RDs) are annotated in <a href="#pone-0000770-t001">Table 1</a>. Each of these regions is flanked by transposases on each side of the LCB, indicating a probable mechanism by which these rearrangements occurred <a href="#pone.0000770-Deng1">[12]</a>. None of these regions represent a contig in its entirety, reducing the chance that these rearrangements are the result of incorrect pseudomolecule assembly.</p>
<div class="figure" id="pone-0000770-g001"><div class="img"><a name="pone-0000770-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A whole-genome comparison between <em>Y. pestis</em> strains CO92 (top) and CA88 (bottom) using MAUVE.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>MAUVE found 15 LCBs shared between CO92 and CA88, six of which are inverted and one of which is rearranged. The seven large regions of difference are indicated. Some LCBs are too small to display on this figure and are shown by long diagonal lines connecting the genomes.</p>
<span>doi:10.1371/journal.pone.0000770.g001</span></div><div class="figure" id="pone-0000770-t001"><div class="img"><a name="pone-0000770-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Characteristics of the six inversions and one large rearranged LCB found via MAUVE.</span></strong></p><span>doi:10.1371/journal.pone.0000770.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In addition to flanking transposases, several insertion (IS) elements of varying types and families were found within each LCB. These insertion elements could promote further rearrangement within these shared LCBs as time progresses and as the strains continue to diverge. This finding is consistent with a previous comparison between <em>Y. pestis</em> CO92 and <em>Y. pestis</em> KIM in which a large amount of genome rearrangement was observed. <em>Y. pestis</em> KIM is a member of biovar Medievalis and the 2.MED branch <a href="#pone.0000770-Achtman2">[5]</a>. The rearrangements between CO92 and KIM were primarily due to multiple inversions of genome segments at insertion sequences <a href="#pone.0000770-Deng1">[12]</a>. When comparing CA88 and CO92, IS elements identified in CA88 inside and flanking the inversions included IS<em>100</em>, IS<em>1541A</em>, IS<em>200G</em>, IS<em>1661</em>, and IS<em>285</em>. All nucleotide BLAST hits possessed greater than 99.6% identity over the full IS element lengths. Further information regarding the IS elements flanking each large rearrangement and the number of IS elements found internal to these LCBs is provided in <a href="#pone-0000770-t002">Table 2</a>.</p>
<div class="figure" id="pone-0000770-t002"><div class="img"><a name="pone-0000770-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>IS element analysis of the seven large regions of difference (RDs).</span></strong></p><span>doi:10.1371/journal.pone.0000770.t002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We attempted to determine the ancestral states for each of the six inversions by comparing CO92 and CA88 to all sequenced <em>Y. pestis</em> genomes in GenBank. Unfortunately the large LCBs were severely rearranged in all other GenBank whole genome sequences. This is not unexpected, as <em>Y. pestis</em> 1.ORI and 2.MED last shared a common ancestor ~7,000 years ago <a href="#pone.0000770-Achtman1">[4]</a>. Even a more closely-related genome, <em>Y. pestis</em> Antiqua, shows significant rearrangement among the inverted CO92-CA88 LCBs <a href="#pone.0000770-Chain1">[13]</a>. To determine the ancestral state of these inversions, sequence from another member of <em>Y. pestis</em> 1.ORI is needed. Although sequences from three strains meeting this requirement are available as unfinished sequences in GenBank, they are not yet complete enough to obtain an accurate assembly solely from the available contigs. We expect to be able to identify possible homoplasy, determine ancestral state, and gain further resolution within the 1.ORI group and North American <em>Y. pestis</em> if these sequences ever get to a completed stage.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The magnitude of rearrangements observed between CO92/CA88 and other completed <em>Y. pestis</em> strains is not surprising given the number of IS elements detected inside each LCB. These IS elements provide a powerful mechanism for subsections of each LCB to translocate and/or invert independent of the entire block. Inversions are recognized as one of the most frequent rearrangement types in γ-proteobacteria <a href="#pone.0000770-Belda1">[14]</a>. The LCB profile of CA88 can be transformed to match that of CO92 in only ten inversion steps and zero translocations. Although there are many ways CA88 and CO92 could have evolved after diverging from a common ancestor, the ten inversion steps shown in <a href="#pone-0000770-t003">Table 3</a> represent the most parsimonious solution. These results are in line with expectations, as more inversion events than translocation events are expected in recently-diverged strains of γ-proteobacteria such as <em>Y. pestis</em> <a href="#pone.0000770-Belda1">[14]</a>.</p>
<div class="figure" id="pone-0000770-t003"><div class="img"><a name="pone-0000770-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Output from GRIMM showing the most parsimonious reversal scenario.</span></strong></p><span>doi:10.1371/journal.pone.0000770.t003</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Plague was introduced into North America within the past 125 years, so any rearrangements between CO92 and CA88 have occurred within a very short time period. Ten inversions separate <em>Y. pestis</em> CO92 from CA88 but only fourteen inversions separate <em>Y. pestis</em> CO92 from KIM <a href="#pone.0000770-Belda1">[14]</a>. This suggests that the fixation rate of inversion events could be particularly high in North American <em>Y. pestis</em> strains, although any confirmation would require additional whole-genome sequences from North American strains. Fixation of inversion events in bacterial genomes during evolution is an irregular phenomenon with periods of stasis and others of acceleration <a href="#pone.0000770-Belda1">[14]</a>, so our findings indicate a possible acceleration of inversion fixation rate in a species prone to inversion events and already believed to be undergoing rapid evolution.</p>


<h4>IS element comparison</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A comparison of total IS elements in CA88 and CO92 is shown in <a href="#pone-0000770-t004">Table 4</a>. These totals appear to be essentially unchanged between the two genomes with many of the same interruptions and truncations observed by Parkhill et al in CO92 <a href="#pone.0000770-Parkhill1">[9]</a>. <em>In silico</em> analysis indicates CA88 has an additional complete copy of IS<em>285</em> as well as an additional partial copy that is missing the first 338 bases and located 69 bases downstream of an intact IS<em>285</em> element. An IS<em>285</em> segment containing only the first 338 bases was not found using BLAST.</p>
<div class="figure" id="pone-0000770-t004"><div class="img"><a name="pone-0000770-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>IS element counts in CO92 and CA88.</span></strong></p><span>doi:10.1371/journal.pone.0000770.t004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We also <em>in silico</em>-typed CA88 based on IS elements using the system described in Achtman et al <a href="#pone.0000770-Achtman2">[5]</a>. Amplicon sequences were extracted and the presence/absence of IS<em>100</em> was determined for each loci. The CA88 profile matched that of CO92 using these methods and the results are shown in <a href="#pone-0000770-t005">Table 5</a>.</p>
<div class="figure" id="pone-0000770-t005"><div class="img"><a name="pone-0000770-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Presence/absence of IS<em>100</em> loci described in Achtman et al (2004).</span></strong></p><span>doi:10.1371/journal.pone.0000770.t005</span></div>

<h4>SNPs</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>15 non-synonymous and 5 synonymous SNPs were discovered when comparing the CO92 and CA88 genomes. Details for each SNP are presented in <a href="#pone-0000770-t006">Table 6</a>. Five of the SNPs were previously discovered between CO92 and <em>Y. pestis</em> FV-1 and these SNPs have been verified in a laboratory setting. All other SNPs have been identified <em>in silico</em> but have not been confirmed via wet bench techniques. Of the 20 discovered SNPs, the CA88 and FV-1 states matched in 15 cases.</p>
<div class="figure" id="pone-0000770-t006"><div class="img"><a name="pone-0000770-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000770" data-uri="info:doi/10.1371/journal.pone.0000770.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000770.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000770.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000770.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000770.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Putative non-synonymous and synonymous SNPs found between CO92 and CA88.</span></strong></p><span>doi:10.1371/journal.pone.0000770.t006</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>None of the non-synonymous SNPs result in nonsense mutations, indicating that no pseudogenes were created via this method. Gene gain/loss does not appear to be occurring between CO92 and CA88, but only approximately 125 years have passed since CO92 and CA88 shared a common ancestor. As these two strains were both found in Western North America, one can hypothesize that essential genes in CO92 are also essential in CA88 and that evolutionary pressure to diverge due to environmental conditions would be minimal in such a short time period.</p>


<h4>Whole gene differences</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>No obvious whole-gene differences were found when comparing CO92 and CA88. Several small regions of non-shared sequence were detected by MAUVE, but in many cases these regions matched transposases. Because the BLAST method showed only a difference of one IS element between CO92 and CA88, MAUVE likely found IS elements that have shifted positions and was unable to match them to their counterparts in the other strain.</p>


<h4>Plasmid comparisons</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>In addition to the chromosome, we also examined differences between the plasmids of CO92 and CA88. LCB profiles matched exactly for both pMT1 and pCD1 but a 999-bp alignment gap was discovered in pPCP1 of CA88. This gap corresponds to nucleotide positions 3124-4122 in the CO92 pPCP1 plasmid sequence (GenBank accession: AL109969). No annotated features appear in the corresponding CO92 pPCP1 region but this region is flanked by the <em>rop</em> and <em>pim</em> genes (YPPCP1.03 and YPPCP1.04). Both <em>rop</em> and <em>pim</em> are intact in the CA88 pPCP1 sequence and the location of the gap corresponds to the ColE1 site in the CO92 plasmid sequence. ColE1 is the origin of replication for the pPCP1 plasmid in <em>Y. pestis</em>. This alignment gap was also present when comparing CA88 to Nepal516 (2.ANT, <a href="#pone.0000770-Chain1">[13]</a>), Antiqua (1.ANT, <a href="#pone.0000770-Chain1">[13]</a>), KIM (2.MED, <a href="#pone.0000770-Deng1">[12]</a>), 91001 (0.PE4, <a href="#pone.0000770-Song1">[15]</a>) and CO92 (1.ORI, <a href="#pone.0000770-Parkhill1">[9]</a>). Confirmation via PCR shows that the alignment gap in CA88 is not real. This region may have been missed during sequencing and was not included as part of the draft CA88 pPCP1 contig sequence.</p>


<h4>Summary</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Initial findings suggest that North American <em>Y. pestis</em> strains may be undergoing inversion fixation at relatively high rates considering the short time span separating the CA88 and the CO92 isolates. Differences in IS element copy number were observed, as well as several SNPs between CO92 and CA88. No whole-gene differences were detected using the CA88 contig sequences We hope these initial findings will encourage the scientific community to fully-sequence more <em>Y. pestis</em> strains from North America and abroad, as it would further our understanding of the evolutionary history of this important pathogen.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>DNA Preparation and Sequencing CA88</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>DNA preparation was performed via chloroform extractions <a href="#pone.0000770-Jackson1">[16]</a>, <a href="#pone.0000770-Keim1">[17]</a>. The genome was sequenced at the Joint Genome Institute (JGI) using small (2–3kb) and medium (6–8kb) insert plasmid libraries. Draft assemblies were based on 15× coverage. The Phred/Phrap/Consed software package (<a href="http://www.phrap.com">http://www.phrap.com</a>) was used for sequence assembly and quality assessment <a href="#pone.0000770-Gordon1">[18]</a>. After shotgun sequencing, reads were assembled with parallel phrap (High Performance Software, LLC). Two rounds of finishing were performed resulting in 10 contigs and 6 scaffolds. During finishing, possible mis-assemblies were corrected by transposon bombing (Epicentre Biotechnologies) of bridging clones. Gaps between contigs were closed by editing in Consed, by custom primer walks, or by PCR amplification.</p>


<h4>Assembling the contigs into a pseudomolecule</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Ten contigs were obtained, of which three contigs contained plasmid sequences. The seven contigs comprising the <em>Y. pestis</em> CA88 chromosome were aligned to the complete genome sequence of <em>Y. pestis</em> CO92 (GenBank accession: NC_003143) using MAUVE <a href="#pone.0000770-Darling1">[19]</a>. After determining proper contig order using MAUVE, the seven contigs were concatenated into one pseudomolecule representing the CA88 chromosome.</p>


<h4>Identifying flanking transposases</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Once the CA88 pseudomolecule was produced, it was aligned against the whole-genome sequence of CO92 and the positions of the major LCBs were determined relative to CO92. The CO92 GenBank annotation was queried to identify annotated coding sequences (CDS) immediately flanking LCB boundaries.</p>


<h4>Locating additional IS elements inside rearranged LCBs and throughout the genome</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Nucleotide sequences for IS elements were obtained from IS Finder (<a href="http://www-is.biotoul.fr/">http://www-is.biotoul.fr/</a>) by searching for “pestis” in all fields. This returned a list of IS elements documented in <em>Y. pestis</em> and <em>Y. pseudotuberculosis</em>. Sequences for each IS element were saved and compared to a sequence database containing CA88 nucleotide sequences for the six inversions and the largest rearrangement. The comparison was performed using BLAST <a href="#pone.0000770-Altschul1">[20]</a>.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The same procedure was used to locate IS elements on a whole-genome scale. IS element sequences were queried against the CO92 genome and the CA88 pseudomolecule using blastn and the results were compared.</p>


<h4>SNP discovery</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>A high-throughput automated bioinformatic pipeline was used to discover and classify SNPs. This pipeline integrates the whole-genome alignment tools MUMmer to map contigs of the draft genome to the reference genome sequence and to identify putative polymorphic sites <a href="#pone.0000770-Kurtz1">[21]</a>. Base-calling software assigns an error probability to each base pair in a sequenced read. The probability of each underlying sequence is used to compute the accuracy of any base pair in the assembled genome. High quality SNPs were selected based on a combination of these statistics and coverage information. To limit the number of false positives due to sequencing errors, the comparative analysis in this study used only regions with at least 3× coverage where the chromatograms agreed with each other and the median quality was more than 30. For each high quality SNP that was located within a gene, the effect of the nucleotide change on the encoded protein was reported, which allowed us to differentiate between synonymous and non-synonymous SNPs. All duplicated regions were removed from consideration. In this comparative analysis, it was assumed that each base pair of CO92 was of high quality, as the underlying chromatograms for the consensus sequence was not available and this genome is closed.</p>


<h4>Whole gene differences</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The islands file created by MAUVE was used to determine regions potentially unique to each genome. For regions identified as being solely present in CA88, sequence was extracted from the appropriate positions within the pseudomolecule and queried against NCBI's nr database using the blastx algorithm. For regions identified as being solely present in CO92, positions from the MAUVE islands file were compared to the CO92 GenBank annotation.</p>


<h4>Determining the most parsimonious rearrangement scheme</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>The GRIMM website (<a href="http://nbcr.sdsc.edu/GRIMM/mgr.cgi">http://nbcr.sdsc.edu/GRIMM/mgr.cgi</a>) was used to analyze the LCB patterns produced in MAUVE. A signed analysis using circular chromosomes was selected. GRIMM analyzes LCB-order and orientation to determine the most parsimonious method to transform one strain into another <a href="#pone.0000770-Tesler1">[22]</a>.</p>


<h4>Plasmid comparisons</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>All plasmids were compared to their CO92 counterparts using the MAUVE Aligner. In the event of gaps between detected LCBs, sequence for this region was extracted from the appropriate strain and compared to its counterpart via BLAST to confirm the findings from MAUVE. PCR was run on CA88 and CO92 to confirm the existence/absence of a sequence gap in the pPCP1 plasmid of CA88. Primers were designed on each side of the putative gap. The following primer sequences were used: CA88-for 5′-AAG CCA GAG CCT GAT ACT GCT TGA-3′ and CA88-rev 5′AAG TAA CAT GGG TGT TAC CGC AGC-3′.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors acknowledge the Joint Genome Institute's Production Genomics Facility for providing the draft sequencing. The authors also thank Zeina Ziade for her technical assistance with the pPCP1 gap verification.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PK DW DB AT RA. Performed the experiments: AV LK TB CM AT RA. Analyzed the data: JR ME AT RA. Contributed reagents/materials/analysis tools: JR ME WP. Wrote the paper: PK DW TB RA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000770-Perry1" id="pone.0000770-Perry1"></a>Perry RD, Fetherston JD (1997) <em>Yersinia pestis</em>–etiologic agent of plague. Clin Microbiol Rev  10: 35–66.  <ul class="find" data-citedArticleID="980426"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Yersinia+pestis%26%23x2013%3Betiologic+agent+of+plague.&amp;auth=&amp;atitle=Yersinia+pestis%26%23x2013%3Betiologic+agent+of+plague." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yersinia+pestis%26%23x2013%3Betiologic+agent+of+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yersinia+pestis%26%23x2013%3Betiologic+agent+of+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000770-Inglesby1" id="pone.0000770-Inglesby1"></a>Inglesby TV, Dennis DT, Henderson DA, Bartlett JG, Ascher MS, et al.  (2000) Plague as a biological weapon: medical and public health management. Working Group on Civilian Biodefense. JAMA  283: 2281–2290.  <ul class="find" data-citedArticleID="980414" data-doi="10.1001/jama.283.17.2281"><li><a href="http://dx.doi.org/10.1001/jama.283.17.2281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plague+as+a+biological+weapon%3A+medical+and+public+health+management.+Working+Group+on+Civilian+Biodefense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plague+as+a+biological+weapon%3A+medical+and+public+health+management.+Working+Group+on+Civilian+Biodefense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000770-Welch1" id="pone.0000770-Welch1"></a>Welch TJ, Fricke WF, McDermott PF, White DG, Rosso ML, et al.  (2007) Multiple antimicrobial resistance in plague: an emerging public health risk. PLoS ONE  2: e309.  <ul class="find" data-citedArticleID="980434" data-doi="10.1371/journal.pone.0000309"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+antimicrobial+resistance+in+plague%3A+an+emerging+public+health+risk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+antimicrobial+resistance+in+plague%3A+an+emerging+public+health+risk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000770-Achtman1" id="pone.0000770-Achtman1"></a>Achtman M (2004)  Age, descent, and genetic diversity within <em>Yersinia pestis</em>. In: BJ. H EC, editor. Yersinia: Molecular and Cellular Biology. Norwich, UK: Horizon Bioscience.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000770-Achtman2" id="pone.0000770-Achtman2"></a>Achtman M, Morelli G, Zhu P, Wirth T, Diehl I, et al.  (2004) Microevolution and history of the plague bacillus, <em>Yersinia pestis</em>. Proc Natl Acad Sci U S A  101: 17837–17842.  <ul class="find" data-citedArticleID="980394" data-doi="10.1073/pnas.0408026101"><li><a href="http://dx.doi.org/10.1073/pnas.0408026101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microevolution+and+history+of+the+plague+bacillus%2C+Yersinia+pestis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microevolution+and+history+of+the+plague+bacillus%2C+Yersinia+pestis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000770-Devignat1" id="pone.0000770-Devignat1"></a>Devignat R (1951) [Varieties of <em>Pasteurella pestis</em>; new hypothesis.]. Bull World Health Organ  4: 247–263.  <ul class="find" data-citedArticleID="980408"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=%5BVarieties+of+Pasteurella+pestis%3B+new+hypothesis.%5D.&amp;auth=&amp;atitle=%5BVarieties+of+Pasteurella+pestis%3B+new+hypothesis.%5D." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%5BVarieties+of+Pasteurella+pestis%3B+new+hypothesis.%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%5BVarieties+of+Pasteurella+pestis%3B+new+hypothesis.%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000770-Link1" id="pone.0000770-Link1"></a>Link VB (1955) A history of plague in the United States of America. Public Health Monogr  26: 1–120.  <ul class="find" data-citedArticleID="980422"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+history+of+plague+in+the+United+States+of+America.&amp;auth=&amp;atitle=A+history+of+plague+in+the+United+States+of+America." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+history+of+plague+in+the+United+States+of+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+history+of+plague+in+the+United+States+of+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000770-Cully1" id="pone.0000770-Cully1"></a>Cully JF, Williams ES (2001) Interspecific comparisons of sylvatic plague in prairie dogs. Journal of Mammalogy  82: 894–905.  <ul class="find" data-citedArticleID="980402" data-doi="10.1644/1545-1542(2001)082<0894:icospi&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1644/1545-1542(2001)082<0894:icospi&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interspecific+comparisons+of+sylvatic+plague+in+prairie+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interspecific+comparisons+of+sylvatic+plague+in+prairie+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000770-Parkhill1" id="pone.0000770-Parkhill1"></a>Parkhill J, Wren BW, Thomson NR, Titball RW, Holden MT, et al.  (2001) Genome sequence of <em>Yersinia pestis</em>, the causative agent of plague. Nature  413: 523–527.  <ul class="find" data-citedArticleID="980424" data-doi="10.1038/35097083"><li><a href="http://dx.doi.org/10.1038/35097083" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Yersinia+pestis%2C+the+causative+agent+of+plague.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000770-Girard1" id="pone.0000770-Girard1"></a>Girard JM, Wagner DM, Vogler AJ, Keys C, Allender CJ, et al.  (2004) Differential plague-transmission dynamics determine Yersinia pestis population genetic structure on local, regional, and global scales. Proc Natl Acad Sci U S A  101: 8408–8413.  <ul class="find" data-citedArticleID="980410" data-doi="10.1073/pnas.0401561101"><li><a href="http://dx.doi.org/10.1073/pnas.0401561101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+plague-transmission+dynamics+determine+Yersinia+pestis+population+genetic+structure+on+local%2C+regional%2C+and+global+scales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+plague-transmission+dynamics+determine+Yersinia+pestis+population+genetic+structure+on+local%2C+regional%2C+and+global+scales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000770-Touchman1" id="pone.0000770-Touchman1"></a>Touchman JW, Wagner DM, Hao J, Mastrian SD, Shah MK, et al.  (2007) A North American <em>Yersinia pestis</em> draft genome sequence: SNPs and phylogenetic analysis. PLoS ONE  2: e220.  <ul class="find" data-citedArticleID="980432" data-doi="10.1371/journal.pone.0000220"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000220" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+North+American+Yersinia+pestis+draft+genome+sequence%3A+SNPs+and+phylogenetic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+North+American+Yersinia+pestis+draft+genome+sequence%3A+SNPs+and+phylogenetic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000770-Deng1" id="pone.0000770-Deng1"></a>Deng W, Burland V, Plunkett G 3rd, Boutin A, Mayhew GF, et al.  (2002) Genome sequence of <em>Yersinia pestis</em> KIM. J Bacteriol  184: 4601–4611.  <ul class="find" data-citedArticleID="980406" data-doi="10.1128/jb.184.16.4601-4611.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.16.4601-4611.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Yersinia+pestis+KIM." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Yersinia+pestis+KIM.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000770-Chain1" id="pone.0000770-Chain1"></a>Chain PS, Hu P, Malfatti SA, Radnedge L, Larimer F, et al.  (2006) Complete genome sequence of <em>Yersinia pestis</em> strains Antiqua and Nepal516: evidence of gene reduction in an emerging pathogen. J Bacteriol  188: 4453–4463.  <ul class="find" data-citedArticleID="980400" data-doi="10.1128/jb.00124-06"><li><a href="http://dx.doi.org/10.1128/jb.00124-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Yersinia+pestis+strains+Antiqua+and+Nepal516%3A+evidence+of+gene+reduction+in+an+emerging+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Yersinia+pestis+strains+Antiqua+and+Nepal516%3A+evidence+of+gene+reduction+in+an+emerging+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000770-Belda1" id="pone.0000770-Belda1"></a>Belda E, Moya A, Silva FJ (2005) Genome rearrangement distances and gene order phylogeny in gamma-Proteobacteria. Mol Biol Evol  22: 1456–1467.  <ul class="find" data-citedArticleID="980398" data-doi="10.1093/molbev/msi134"><li><a href="http://dx.doi.org/10.1093/molbev/msi134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+rearrangement+distances+and+gene+order+phylogeny+in+gamma-Proteobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+rearrangement+distances+and+gene+order+phylogeny+in+gamma-Proteobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000770-Song1" id="pone.0000770-Song1"></a>Song Y, Tong Z, Wang J, Wang L, Guo Z, et al.  (2004) Complete genome sequence of Yersinia pestis strain 91001, an isolate avirulent to humans. DNA Res  11: 179–197.  <ul class="find" data-citedArticleID="980428" data-doi="10.1093/dnares/11.3.179"><li><a href="http://dx.doi.org/10.1093/dnares/11.3.179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Yersinia+pestis+strain+91001%2C+an+isolate+avirulent+to+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Yersinia+pestis+strain+91001%2C+an+isolate+avirulent+to+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000770-Jackson1" id="pone.0000770-Jackson1"></a>Jackson PJ, Walthers EA, Kalif AS, Richmond KL, Adair DM, et al.  (1997) Characterization of the variable-number tandem repeats in vrrA from different Bacillus anthracis isolates. Appl Environ Microbiol  63: 1400–1405.  <ul class="find" data-citedArticleID="980416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+the+variable-number+tandem+repeats+in+vrrA+from+different+Bacillus+anthracis+isolates.&amp;auth=&amp;atitle=Characterization+of+the+variable-number+tandem+repeats+in+vrrA+from+different+Bacillus+anthracis+isolates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+variable-number+tandem+repeats+in+vrrA+from+different+Bacillus+anthracis+isolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+variable-number+tandem+repeats+in+vrrA+from+different+Bacillus+anthracis+isolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000770-Keim1" id="pone.0000770-Keim1"></a>Keim P, Kalif A, Schupp J, Hill K, Travis SE, et al.  (1997) Molecular evolution and diversity in Bacillus anthracis as detected by amplified fragment length polymorphism markers. J Bacteriol  179: 818–824.  <ul class="find" data-citedArticleID="980418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+evolution+and+diversity+in+Bacillus+anthracis+as+detected+by+amplified+fragment+length+polymorphism+markers.&amp;auth=&amp;atitle=Molecular+evolution+and+diversity+in+Bacillus+anthracis+as+detected+by+amplified+fragment+length+polymorphism+markers." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+and+diversity+in+Bacillus+anthracis+as+detected+by+amplified+fragment+length+polymorphism+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+and+diversity+in+Bacillus+anthracis+as+detected+by+amplified+fragment+length+polymorphism+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000770-Gordon1" id="pone.0000770-Gordon1"></a>Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence finishing. Genome Res  8: 195–202.  <ul class="find" data-citedArticleID="980412" data-doi="10.1101/gr.8.3.195"><li><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consed%3A+a+graphical+tool+for+sequence+finishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consed%3A+a+graphical+tool+for+sequence+finishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000770-Darling1" id="pone.0000770-Darling1"></a>Darling AC, Mau B, Blattner FR, Perna NT (2004) Mauve: multiple alignment of conserved genomic sequence with rearrangements. Genome Res  14: 1394–1403.  <ul class="find" data-citedArticleID="980404" data-doi="10.1101/gr.2289704"><li><a href="http://dx.doi.org/10.1101/gr.2289704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mauve%3A+multiple+alignment+of+conserved+genomic+sequence+with+rearrangements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mauve%3A+multiple+alignment+of+conserved+genomic+sequence+with+rearrangements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000770-Altschul1" id="pone.0000770-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="980396" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000770-Kurtz1" id="pone.0000770-Kurtz1"></a>Kurtz S, Phillippy A, Delcher AL, Smoot M, Shumway M, et al.  (2004) Versatile and open software for comparing large genomes. Genome Biol  5: R12.  <ul class="find" data-citedArticleID="980420" data-doi="10.1186/gb-2004-5-2-r12"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Versatile+and+open+software+for+comparing+large+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Versatile+and+open+software+for+comparing+large+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000770-Tesler1" id="pone.0000770-Tesler1"></a>Tesler G (2002) GRIMM: genome rearrangements web server. Bioinformatics  18: 492–493.  <ul class="find" data-citedArticleID="980430" data-doi="10.1093/bioinformatics/18.3.492"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GRIMM%3A+genome+rearrangements+web+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GRIMM%3A+genome+rearrangements+web+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.XML" value="75000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.PDF" value="159161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.g001.PNG_L" value="1932330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.g001.PNG_M" value="121870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.g001.PNG_S" value="8572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.g001.TIF" value="5539218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.g001.PNG_I" value="39707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t001.PNG_L" value="87262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t001.PNG_M" value="48900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t001.PNG_S" value="8571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t001.TIF" value="379348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t001.PNG_I" value="14782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t002.PNG_L" value="63311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t002.PNG_M" value="81100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t002.PNG_S" value="12848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t002.TIF" value="280652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t002.PNG_I" value="26909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t003.PNG_L" value="113503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t003.PNG_M" value="156955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t003.PNG_S" value="20416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t003.TIF" value="461440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t003.PNG_I" value="52561"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t004.PNG_L" value="51629"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t004.PNG_M" value="62402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t004.PNG_S" value="10956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t004.TIF" value="230078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t004.PNG_I" value="22095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t005.PNG_L" value="33726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t005.PNG_M" value="41511"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t005.PNG_S" value="9876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t005.TIF" value="142704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t005.PNG_I" value="14888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t006.PNG_L" value="373322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t006.PNG_M" value="104402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t006.PNG_S" value="12474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t006.TIF" value="900058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000770.t006.PNG_I" value="31926"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000770&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000770" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000770&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000770&volume=&issue=&title=Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America&author_name=Raymond%20K.%20Auerbach%2C%20Apichai%20Tuanyok%2C%20William%20S.%20Probert%2C%20Leo%20Kenefic%2C%20Amy%20J.%20Vogler%2C%20David%20C.%20Bruce%2C%20Christine%20Munk%2C%20Thomas%20S.%20Brettin%2C%20Mark%20Eppinger%2C%20Jacques%20Ravel%2C%20David%20M.%20Wagner%2C%20Paul%20Keim&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000770" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770&amp;t=Yersinia%20pestis%20Evolution%20on%20a%20Small%20Timescale%3A%20Comparison%20of%20Whole%20Genome%20Sequences%20from%20North%20America" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770&title=Yersinia%20pestis%20Evolution%20on%20a%20Small%20Timescale%3A%20Comparison%20of%20Whole%20Genome%20Sequences%20from%20North%20America&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770&amp;title=Yersinia%20pestis%20Evolution%20on%20a%20Small%20Timescale%3A%20Comparison%20of%20Whole%20Genome%20Sequences%20from%20North%20America" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000770&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Yersinia pestis Evolution on a Small Timescale: Comparison of Whole Genome Sequences from North America';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000770';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Yersinia%20pestis%20Evolution%20on%20a%20Small%20Timescale%3A%20Comparison%20of%20Whole%20Genome%20Sequences%20from%20North%20America http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000770" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000770" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24378"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24378"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24378"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plagues%22" title="Search for articles in the subject area:'Plagues'"><div class="flagText">Plagues</div></a>
              <div data-categoryid="50361" data-articleid="24378"
                   data-categoryname="Plagues"
                   class="flagImage" title="Flag 'Plagues' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24378"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="24378"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="24378"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yersinia+pestis%22" title="Search for articles in the subject area:'Yersinia pestis'"><div class="flagText">Yersinia pestis</div></a>
              <div data-categoryid="44253" data-articleid="24378"
                   data-categoryname="Yersinia pestis"
                   class="flagImage" title="Flag 'Yersinia pestis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1424'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4675'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4678&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
