

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Using Ribosomal Protein Genes as Reference: A Tale of Caution</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001854"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001854" />

    <meta name="citation_title" content="Using Ribosomal Protein Genes as Reference: A Tale of Caution"/>
    <meta itemprop="name" content="Using Ribosomal Protein Genes as Reference: A Tale of Caution"/>

      <meta name="citation_author" content="Lieven Thorrez"/>
            <meta name="citation_author_institution" content="Gene Expression Unit, Department of Molecular Cell Biology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="Department of Electrical Engineering, ESAT-SCD, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Katrijn Van Deun"/>
            <meta name="citation_author_institution" content="Department of Psychology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Léon-Charles Tranchevent"/>
            <meta name="citation_author_institution" content="Department of Electrical Engineering, ESAT-SCD, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Leentje Van Lommel"/>
            <meta name="citation_author_institution" content="Gene Expression Unit, Department of Molecular Cell Biology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Kristof Engelen"/>
            <meta name="citation_author_institution" content="Department of Microbial and Molecular Systems, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Kathleen Marchal"/>
            <meta name="citation_author_institution" content="Department of Microbial and Molecular Systems, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Yves Moreau"/>
            <meta name="citation_author_institution" content="Department of Electrical Engineering, ESAT-SCD, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Iven Van Mechelen"/>
            <meta name="citation_author_institution" content="Department of Psychology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>
      <meta name="citation_author" content="Frans Schuit"/>
            <meta name="citation_author_institution" content="Gene Expression Unit, Department of Molecular Cell Biology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
            <meta name="citation_author_institution" content="SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium"/>

    <meta name="citation_date" content="2008/3/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001854.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1854"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Further defining housekeeping, or “maintenance,” genes Focus on “A compendium of gene expression in normal human tissues”.; citation_author=AJ Butte; citation_author=VJ Dzau; citation_author=SB Glueck; citation_journal_title=Physiol Genomics; citation_volume=7; citation_number=1; citation_pages=95-96; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=beta-Actin and GAPDH housekeeping gene expression in asthmatic airways is variable and not suitable for normalising mRNA levels.; citation_author=EM Glare; citation_author=M Divjak; citation_author=MJ Bailey; citation_author=EH Walters; citation_journal_title=Thorax; citation_volume=57; citation_number=2; citation_pages=765-770; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Control genes and variability: absence of ubiquitous reference transcripts in diverse mammalian expression studies.; citation_author=PD Lee; citation_author=R Sladek; citation_author=CM Greenwood; citation_author=TJ Hudson; citation_journal_title=Genome Res; citation_volume=12; citation_number=3; citation_pages=292-297; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Control selection for RNA quantitation.; citation_author=T Suzuki; citation_author=PJ Higgins; citation_author=DR Crawford; citation_journal_title=Biotechniques; citation_volume=29; citation_number=4; citation_pages=332-337; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Housekeeping genes as internal standards: use and limits.; citation_author=O Thellin; citation_author=W Zorzi; citation_author=B Lakaye; citation_author=BB De; citation_author=B Coumans; citation_journal_title=J Biotechnol; citation_volume=75; citation_number=5; citation_pages=291-295; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evidence based selection of housekeeping genes.; citation_author=HJ de Jonge; citation_author=RS Fehrmann; citation_author=ES de Bont; citation_author=RM Hofstra; citation_author=F Gerbens; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=6; citation_pages=e898; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A compendium of gene expression in normal human tissues.; citation_author=LL Hsiao; citation_author=F Dangond; citation_author=T Yoshida; citation_author=R Hong; citation_author=RV Jensen; citation_journal_title=Physiol Genomics; citation_volume=7; citation_number=7; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Joint mapping of genes and conditions via multidimensional unfolding analysis.; citation_author=K Van Deun; citation_author=K Marchal; citation_author=WJ Heiser; citation_author=K Engelen; citation_author=I Van Mechelen; citation_journal_title=BMC Bioinformatics; citation_volume=8; citation_number=8; citation_pages=181; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Modern multidimensional scaling: Theory and Applications.; citation_author=I Borg; citation_author=PJF Groenen; citation_number=9; citation_date=2005; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Housekeeping and tissue-specific genes in mouse tissues.; citation_author=KE Kouadjo; citation_author=Y Nishida; citation_author=JF Cadrin-Girard; citation_author=M Yoshioka; citation_author=J St-Amand; citation_journal_title=BMC Genomics; citation_volume=8; citation_number=10; citation_pages=127; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Identification of novel universal housekeeping genes by statistical analysis of microarray data.; citation_author=S Lee; citation_author=M Jo; citation_author=J Lee; citation_author=SS Koh; citation_author=S Kim; citation_journal_title=J Biochem Mol Biol; citation_volume=40; citation_number=11; citation_pages=226-231; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The cell: a molecular approach.; citation_author=GM Cooper; citation_number=12; citation_date=2000; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Possible control of lymphocyte growth at the level of ribosome assembly.; citation_author=AD Rubin; citation_journal_title=Nature; citation_volume=220; citation_number=13; citation_pages=196-197; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Ribosome assembly in eukaryotes.; citation_author=M Fromont-Racine; citation_author=B Senger; citation_author=C Saveanu; citation_author=F Fasiolo; citation_journal_title=Gene; citation_volume=313; citation_number=14; citation_pages=17-42; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A novel ribosomal protein L3-like gene (RPL3L) maps to the autosomal dominant polycystic kidney disease gene region.; citation_author=TJ Van Raay; citation_author=TD Connors; citation_author=KW Klinger; citation_author=GM Landes; citation_author=TC Burn; citation_journal_title=Genomics; citation_volume=37; citation_number=15; citation_pages=172-176; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Alternative splicing and nonsense-mediated mRNA decay regulate mammalian ribosomal gene expression.; citation_author=M Cuccurese; citation_author=G Russo; citation_author=A Russo; citation_author=C Pietropaolo; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=16; citation_pages=5965-5977; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structure of the human gene (COX6A2) for the heart/muscle isoform of cytochrome c oxidase subunit VIa and its chromosomal location in humans, mice, and cattle.; citation_author=NJ Bachman; citation_author=PK Riggs; citation_author=N Siddiqui; citation_author=GJ Makris; citation_author=JE Womack; citation_journal_title=Genomics; citation_volume=42; citation_number=17; citation_pages=146-151; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Large-scale analysis of human alternative protein isoforms: pattern classification and correlation with subcellular localization signals.; citation_author=M Nakao; citation_author=RA Barrero; citation_author=Y Mukai; citation_author=C Motono; citation_author=M Suwa; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=18; citation_pages=2355-2363; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development and evaluation of canine reference genes for accurate quantification of gene expression.; citation_author=B Brinkhof; citation_author=B Spee; citation_author=J Rothuizen; citation_author=LC Penning; citation_journal_title=Anal Biochem; citation_volume=356; citation_number=19; citation_pages=36-43; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Normalizing genes for quantitative RT-PCR in differentiating human intestinal epithelial cells and adenocarcinomas of the colon.; citation_author=AB Dydensborg; citation_author=E Herring; citation_author=J Auclair; citation_author=E Tremblay; citation_author=JF Beaulieu; citation_journal_title=Am J Physiol Gastrointest Liver Physiol; citation_volume=290; citation_number=20; citation_pages=G1067-G1074; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of new reference genes for the normalisation of canine osteoarthritic joint tissue transcripts from microarray data.; citation_author=LJ Maccoux; citation_author=DN Clements; citation_author=F Salway; citation_author=PJ Day; citation_journal_title=BMC Mol Biol; citation_volume=8; citation_number=21; citation_pages=62; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Development and application of multiple internal reference (housekeeper) gene assays for accurate normalisation of canine gene expression studies.; citation_author=IR Peters; citation_author=D Peeters; citation_author=CR Helps; citation_author=MJ Day; citation_journal_title=Vet Immunol Immunopathol; citation_volume=117; citation_number=22; citation_pages=55-66; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Improved generation of germline-competent embryonic stem cell lines from inbred mouse strains.; citation_author=L Schoonjans; citation_author=V Kreemers; citation_author=S Danloy; citation_author=RW Moreadith; citation_author=Y Laroche; citation_journal_title=Stem Cells; citation_volume=21; citation_number=23; citation_pages=90-97; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=affy–analysis of Affymetrix GeneChip data at the probe level.; citation_author=L Gautier; citation_author=L Cope; citation_author=BM Bolstad; citation_author=RA Irizarry; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=24; citation_pages=307-315; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics.; citation_author=RC Gentleman; citation_author=VJ Carey; citation_author=DM Bates; citation_author=B Bolstad; citation_author=M Dettling; citation_journal_title=Genome Biol; citation_volume=5; citation_number=25; citation_pages=R80; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Using Ribosomal Protein Genes as Reference: A Tale of Caution"/>
    <meta name="twitter:description" content="BackgroundHousekeeping genes are needed in every tissue as their expression is required for survival, integrity or duplication of every cell. Housekeeping genes commonly have been used as reference genes to normalize gene expression data, the underlying assumption being that they are expressed in every cell type at approximately the same level. Often, the terms &ldquo;reference genes&rdquo; and &ldquo;housekeeping genes&rdquo; are used interchangeably. In this paper, we would like to distinguish between these terms. Consensus is growing that housekeeping genes which have traditionally been used to normalize gene expression data are not good reference genes. Recently, ribosomal protein genes have been suggested as reference genes based on a meta-analysis of publicly available microarray data.Methodology/Principal FindingsWe have applied several statistical tools on a dataset of 70 microarrays representing 22 different tissues, to assess and visualize expression stability of ribosomal protein genes. We confirmed the housekeeping status of these genes, but further estimated expression stability across tissues in order to assess their potential as reference genes. One- and two-way ANOVA revealed that all ribosomal protein genes have significant expression variation across tissues and exhibit tissue-dependent expression behavior as a group. Via multidimensional unfolding analysis, we visualized this tissue-dependency. In addition, we explored mechanisms that may cause tissue dependent effects of individual ribosomal protein genes.Conclusions/SignificanceHere we provide statistical and biological evidence that ribosomal protein genes exhibit important tissue-dependent variation in mRNA expression. Though these genes are most stably expressed of all investigated genes in a meta-analysis they cannot be considered true reference genes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001854.g004"/>

  <meta property="og:title" content="Using Ribosomal Protein Genes as Reference: A Tale of Caution" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7731'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3582'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1374&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001854">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001854" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001854&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Using Ribosomal Protein Genes as Reference: A Tale of Caution
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lieven Thorrez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gene Expression Unit, Department of Molecular Cell Biology, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  Department of Electrical Engineering, ESAT-SCD, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Katrijn Van Deun, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Psychology, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Léon-Charles Tranchevent, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Electrical Engineering, ESAT-SCD, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leentje Van Lommel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Gene Expression Unit, Department of Molecular Cell Biology, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kristof Engelen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Microbial and Molecular Systems, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kathleen Marchal, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Microbial and Molecular Systems, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yves Moreau, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Electrical Engineering, ESAT-SCD, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Iven Van Mechelen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Psychology, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frans Schuit
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:frans.schuit@med.kuleuven.be">frans.schuit@med.kuleuven.be</a></p>

                <p>Affiliations:
                  Gene Expression Unit, Department of Molecular Cell Biology, Katholieke Universiteit Leuven, Leuven, Belgium, 
                  SymBioSys, K.U.Leuven Center for Computational Systems Biology, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 26, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001854</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001854-g001" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001854-g002" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001854-g003" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001854-g004" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001854">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001854" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Housekeeping genes are needed in every tissue as their expression is required for survival, integrity or duplication of every cell. Housekeeping genes commonly have been used as reference genes to normalize gene expression data, the underlying assumption being that they are expressed in every cell type at approximately the same level. Often, the terms “reference genes” and “housekeeping genes” are used interchangeably. In this paper, we would like to distinguish between these terms. Consensus is growing that housekeeping genes which have traditionally been used to normalize gene expression data are not good reference genes. Recently, ribosomal protein genes have been suggested as reference genes based on a meta-analysis of publicly available microarray data.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have applied several statistical tools on a dataset of 70 microarrays representing 22 different tissues, to assess and visualize expression stability of ribosomal protein genes. We confirmed the housekeeping status of these genes, but further estimated expression stability across tissues in order to assess their potential as reference genes. One- and two-way ANOVA revealed that all ribosomal protein genes have significant expression variation across tissues and exhibit tissue-dependent expression behavior as a group. Via multidimensional unfolding analysis, we visualized this tissue-dependency. In addition, we explored mechanisms that may cause tissue dependent effects of individual ribosomal protein genes.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Here we provide statistical and biological evidence that ribosomal protein genes exhibit important tissue-dependent variation in mRNA expression. Though these genes are most stably expressed of all investigated genes in a meta-analysis they cannot be considered true reference genes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Thorrez L, Van Deun K, Tranchevent L-C, Van Lommel L, Engelen K, et al.  (2008) Using Ribosomal Protein Genes as Reference: A Tale of Caution. PLoS ONE 3(3):
          e1854.
            doi:10.1371/journal.pone.0001854</p><p><strong>Editor: </strong>Sui Huang, Children's Hospital Boston, United States of America</p><p><strong>Received:</strong> December 6, 2007; <strong>Accepted:</strong> February 20, 2008; <strong>Published:</strong> March 26, 2008</p><p><strong>Copyright:</strong> © 2008 Thorrez et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported by research grants from the Katholieke Universiteit Leuven (GOA/2004/11 to the FS laboratory, GOA AMBioRICS (2004–2009) and SymBioSys: CoE EF/05/007) and the Belgian Federal Science Policy Office (IUAP P6/25:BioMaGNet, 2007–2011). L. Tranchevent is funded by EU-RTD: ERNSI: FP6-MC-EST Bioptrain. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A challenge for the accurate quantification of differences in gene expression level across biological conditions is to normalize for potential artifacts caused by sample preparation or gene expression detection. A common technique in RT-PCR, northern blots or western blots is to normalize data for such artifacts by measuring in the same samples the expression of a reference gene in parallel. The reference gene(s) are assumed to be expressed at constant levels across all the experimental conditions, tissues or cell lines. When only one tissue or cell line is studied, it suffices to look at genes that are constantly expressed in that particular tissue, but need not be expressed in other tissues. In the study of the relative levels of gene expression in various tissues, such as in the study of tissue-specific regulatory elements, a gene that is expressed at constant levels in many tissues is needed. The choice for such reference gene(s) has been a subject of debate for many years. Typical choices were beta-actin, <em>GAPDH, HPRT</em>, or <em>18S RNA</em>. These genes were thought to be stably expressed since they are considered as “housekeeping genes”. Housekeeping genes have been defined functionally as “constitutively expressed to maintain cellular function” <a href="#pone.0001854-Butte1">[1]</a>. Being constitutively expressed however does not necessarily meet the prerequisites for a good reference gene, which also needs to display a sufficiently small variation in expression among different tissues. Many of the commonly used reference genes exhibit considerable variability in expression over different tissues and/or experimental conditions <a href="#pone.0001854-Glare1">[2]</a>–<a href="#pone.0001854-Thellin1">[5]</a> and therefore are not a good choice as reference. More recently, several attempts were performed based on microarray or large-scale sequencing technologies to find more stably expressed reference genes. A meta-analysis of 13629 human Affymetrix arrays was conducted to identify the most stably expressed genes <a href="#pone.0001854-deJonge1">[6]</a>. As a result, a list of 15 genes was suggested with the most constant expression level, based on a coefficient of variation smaller than 4%, a maximum fold change smaller than 2 and a mean expression level lower than the maximum expression level minus 2 times the standard deviation. Thirteen of these fifteen genes are coding for ribosomal proteins. We assessed the expression stability of ribosomal proteins across 22 tissues and suggest that, from a biological and statistical perspective, one should be careful to use these genes as reference genes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>1. Ribosomal proteins genes: housekeeping but also reference?</h4>
<h5>1.1 Ribosomal genes are constitutively expressed.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>It was suggested before <a href="#pone.0001854-Hsiao1">[7]</a> that ribosomal genes are good housekeeping genes as they are expressed in all cell types to direct biogenesis of new ribosomes. To validate this, we analyzed the probesets for 81 different ribosomal protein genes represented on the Affymetrix mouse 430 2.0 expression arrays in a set of 22 different mouse tissues. In total, 6951 probesets were called present over all individual arrays which was 15% of all probesets on the 430 2.0 array and corresponded to 4845 unique genes (20% of estimated number of mouse genes). With a few exceptions, all ribosomal protein mRNAs were called present over all individual arrays and two were exceptionally called absent (<a href="#pone-0001854-g001">Figure 1A</a>). Transcripts from these two genes, <em>Rpl39</em>-like and <em>Rpl3</em>-like, were only expressed in a few tissues. Therefore, based on their ubiquitous expression profiles, we could endorse the status of most ribosomal protein genes as housekeeping genes. The next question arising was whether these genes would be suitable as reference genes.</p>
<div class="figure" id="pone-0001854-g001"><div class="img"><a name="pone-0001854-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Housekeeping versus reference gene status of 81 ribosomal protein genes.</span></strong></p><a id="article1.body1.sec2.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec1.fig1.caption1.p1"></a><p>(A) Percentage of present expression calls tested in 22 different mouse tissues (3–5 replicates per tissues, in total 70 arrays). Most ribosomal protein genes were present in all tissues examined and thus can be called housekeeping genes. Exceptions were <em>Rpl39l</em> and <em>Rpl3l</em>. (B) Variance of expression levels within replicates of tissues, representing biological variation between animals and technical error on measurements (triangles) compared to variance of expression between tissues (circles). Variance within replicated measurements was significantly smaller than variance of expression between different conditions (Wilcoxon's rank sum statistic: p&lt;0.0001).</p>
<span>doi:10.1371/journal.pone.0001854.g001</span></div>
<h5>1.2 Variation analysis within versus between different tissues.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>For all 81 probes representing the ribosomal protein genes, we found with one-way analysis of variance (ANOVA) that the expression levels differed significantly between tissues at a simultaneous significance level of .01 (using a Bonferroni correction to account for multiple testing). To get a clearer picture of the different sources of variation, we performed a two-way ANOVA with the 22 tissues and the 81 probesets representing ribosomal protein genes considered as the factors of variation. The gene effect was most significant, reflecting different average expression levels of ribosomal protein genes over all tissues (F<sub>80,5669</sub> = 5598.61, p&lt;0.0001). After correcting for gene variation, the tissue effect was also highly significant (F<sub>21,5669</sub> = 4094.43, p&lt;0.0001), reflecting that in general ribosomal proteins as a group were more highly expressed in certain tissues. In addition to these main effects (genes as a group or tissues as a group), significant variation could be attributed to the gene-tissue interaction effect (F<sub>1680,5669</sub> = 38.55, p&lt;0.0001), reflecting gene specific deviations in expression across various tissues.</p>
<a id="article1.body1.sec2.sec1.sec2.p2" name="article1.body1.sec2.sec1.sec2.p2"></a><p>In order to assess the importance of expression variation of the ribosomal protein genes among different mouse tissues, we first compared the variance amongst replicates of the same tissue from different animals (representing technical variation plus inter-individual variation) with tissue variance (<a href="#pone-0001854-g001">Figure 1B</a>). We performed this analysis for 81 ribosomal protein gene probesets for the 22 different tissues, with 3–5 replicates per tissue. Frequency distribution of the variance on the 1782 (81×22) sets of biologically replicated data is shown in <a href="#pone-0001854-g001">Figure 1B</a>. Median variance was 0.0055 and 95% of all data had a variance of 0.0463 or less. In contrast, differences in expression between tissues were much larger as median variance was 0.3011 and all variances were larger than 0.0463 (<a href="#pone-0001854-g001">Figure 1B</a>). The variance between tissues was significantly larger than that between replicates (Wilcoxon's rank sum statistic: p&lt;0.0001). These data indicated that the variance of expression of ribosomal protein genes caused by replication within one tissue (both of biological and technical origin) was much smaller than the variation between tissues. We repeated this analysis with a smaller set of 15 genes described to be the most stably expressed <a href="#pone.0001854-deJonge1">[6]</a>. Wilcoxon's rank sum statistic again resulted in a significant value (p&lt;0.0001) meaning that also for these genes the variance between tissues was larger than between replicates. Similar significant data were obtained invariable of the normalization method used. These data indicated that the variation of expression of ribosomal protein genes caused by replication within one tissue was much smaller than the variation between tissues. Therefore, although most ribosomal protein genes could be called housekeeping genes, there was significant variation in expression across multiple tissues which indicated that these genes cannot be used in all conditions as reference genes.</p>
<a id="article1.body1.sec2.sec1.sec2.p3" name="article1.body1.sec2.sec1.sec2.p3"></a><p>In contrast to our data, a subset of ribosomal protein genes was described as stable over a large set of publicly available arrays <a href="#pone.0001854-deJonge1">[6]</a>. The criterion for genes to be considered stable was their expression profiles showing a coefficient of variation (CV) &lt;4% across all tissues (the CV is calculated as the standard deviation divided by the mean). The use of CV as a measure implies that the variance increases with higher expression levels. Our results however showed that the biological variation did not increase with higher expression levels, but rather the opposite (data not shown), invalidating the use of a CV cutoff as a stability criterion. Moreover, such a global criterion was incapable of accounting for smaller numbers of differing tissues. To illustrate, we calculated the following hypothetical example. Out of a total of 13629 samples (the same number as in the meta analysis), 100 samples were taken from a tissue in which a certain gene was only marginally expressed (log2 expression normally distributed with mean 8 and standard deviation 0.3) whereas in the other 13529 samples this gene was more abundant (log2 expression normally distributed with mean 12 and standard deviation 0.3). We generated 10000 random expression profiles that followed this scheme and calculated the CV. Distribution of the CV is shown in <a href="#pone.0001854.s001">Figure S1</a>. The mean±standard deviation was 3.80±0.02 and all randomly generated profiles had a CV lower than 4%. This example illustrates that the CV&lt;4% criterion applied on a large dataset would still include genes which were expressed significantly lower in a subset of tissues and thus would still retain them as candidate reference genes.</p>



<h4>2. Ribosomal protein genes are co-expressed in a tissue-specific manner</h4>
<h5>2.1 Tissue-specific variation.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>We analyzed the origins of this variation across different tissues in our dataset. For <em>Rps13</em>, the most stably expressed gene <a href="#pone.0001854-deJonge1">[6]</a>, we obtained the graph displayed in <a href="#pone-0001854-g002">Figure 2A</a>. Large expression differences can be observed between different tissues, e.g. embryonic stem cells (ES cells) contained nearly 6-fold as much <em>Rps13</em> as compared to the brain cortex. One-way analysis of variance confirmed that these differences in expression were significant (F<sub>21,48</sub> = 63.40, p&lt;0.0001). This does not necessarily reflect the absolute levels of transcripts, since one tissue type might contain more RNA per cell as compared to another. Applying other normalization methods also revealed significant differences between tissues (data not shown). In addition, we analyzed expression of the human <em>Rps13</em> expression within the GDS596 record in Gene Expression Omnibus (GEO) which is a subset of the data used for the meta-analysis <a href="#pone.0001854-deJonge1">[6]</a>. The expression profiles across the 79 physiologically normal human tissues in this dataset also displayed significant variation (F<sub>78,79</sub> = 4.54, p&lt;0.0001).</p>
<div class="figure" id="pone-0001854-g002"><div class="img"><a name="pone-0001854-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Tissue-specific co-expression of ribosomal protein genes.</span></strong></p><a id="article1.body1.sec2.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec1.fig1.caption1.p1"></a><p>(A) Expression of <em>Rps13</em> in 22 different tissues. Individual measurements are displayed as blue dots and average corresponds to the top of the grey bars. Expression values are displayed in log2 scale. (B) Marginal means (estimated under the two-way ANOVA) of all ribosomal protein genes in each of the tissues, together with the 95 percent confidence interval.</p>
<span>doi:10.1371/journal.pone.0001854.g002</span></div><a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>This difference in expression between tissues of <em>Rps13</em> was representative for most of the ribosomal proteins. Moreover, tissues in which <em>Rps13</em> was highly expressed had consistently higher expression levels for all ribosomal proteins. The occurrence of significant differences in expression between the tissues was already established in general by the tissue main effect in the two-way ANOVA. Next, we investigated between which specific tissues the expression levels differed, by means of pairwise comparisons of all possible tissue pairs using Tukey's multiple comparisons procedure, maintaining the overall significance level at .05. In <a href="#pone-0001854-g002">Figure 2B</a>, we display the estimated marginal means (estimated under the two-way ANOVA) together with the 95 percent simultaneous confidence intervals. It can be observed that many intervals do not overlap and, specifically for thymus, ES cells, ovary, and fetus there was no overlap with intervals of the other tissues.</p>
<a id="article1.body1.sec2.sec2.sec1.p3" name="article1.body1.sec2.sec2.sec1.p3"></a><p>In favor of a biological explanation underlying these differences between tissues, we noted that certain tissues consistently had a higher expression for the whole set of mRNA's encoding large and small subunit ribosomal proteins. Interestingly, such tissues contain either a high percentage of proliferating cells (ES cells, fetus, lymphoid tissues) and/or are specialized in exocrine protein secretion (salivary gland, seminal vesicle). Alternative to a biological explanation, one might argue that the high variance amongst tissues and tissue-specific co-regulated expression of all transcripts encoding ribosomal proteins reflects tissue-dependent artifacts of the normalization procedure or a microarray batch effect. Therefore, using the same microarray data, we performed a similar ANOVA analysis on another family of housekeeping genes, those encoding for the mitochondrial respiratory chain proteins. These data are displayed in <a href="#pone.0001854.s002">Figure S2A</a>. Similar to the ribosomal protein genes, significant tissue effects can be observed, but tissues with a high expression in respiratory chain proteins were not the same as those with a high expression of ribosomal proteins. Especially striated muscle tissues such as heart, gastrocnemius muscle and diaphragm, displayed the highest tissue-effect on respiratory chain gene expression. This distinct behavior of a group of genes represented on the same arrays excluded batch or normalization artifacts. Comparable to the data set obtained with the probes hybridizing to the ribosomal protein-encoding transcripts, the 69 respiratory chain proteins exhibited significantly more inter-tissue variance (median = 0.55) than technical variance (median = 0.01) (p&lt;.0001 using Wilcoxon's rank sum test) (<a href="#pone.0001854.s002">Figure S2B</a>).</p>

<h5>2.2 Unfolding analysis.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>The tissue-specific expression of the ribosomal protein genes and the respiratory chain genes can also be supported by a purely exploratory (unsupervised, distribution free) analysis, being the multidimensional unfolding representation <a href="#pone.0001854-VanDeun1">[8]</a> depicted in <a href="#pone-0001854-g003">Figure 3</a>. The multidimensional unfolding model, which is an extension of multidimensional scaling to rectangular data <a href="#pone.0001854-Borg1">[9]</a>, represents both the genes and the tissues as points in a low-dimensional space such that the (Euclidean) distances from a gene point to the tissue points reflect the expression profile. In other words, genes are located close to those tissues in which they are highly expressed (and far from those tissues in which they are barely expressed). Clearly, there were two distinct groups with the respiratory chain genes being located close to diaphragm, heart and muscle, and the ribosomal protein genes being located close to thymus, ES cells, ovary and fetus. Based on this consistent tissue-specific grouping of sets of housekeeping genes participating in separate pathways, a general normalization artifact to explain inter-tissue differences can be excluded.</p>
<div class="figure" id="pone-0001854-g003"><div class="img"><a name="pone-0001854-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Multidimensional unfolding analysis of mRNA expression signals encoding 81 ribosomal proteins (purple symbols) versus 69 mitochondrial respiratory chain subunits (blue symbols).</span></strong></p><a id="article1.body1.sec2.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec2.fig1.caption1.p1"></a><p>This analysis gives a graphical overview based on expression profiles; genes with a high expression in a certain tissue will be represented close to that tissue. Note that both groups of transcripts formed separate clusters. These clusters indicated high co-expression of ribosomal protein mRNA's in tissues that are active in exocrine protein secretion and/or cell division. Respiratory chain mRNA's were also co-expressed but were particularly high in striated muscle. Arrowheads indicate 2 ribosomal protein genes outside of the cluster. The upper arrowhead is <em>Rpl3l</em>, which clustered closely to the contractile tissues. The lower arrowhead is <em>Rpl39l</em>, located close to testis.</p>
<span>doi:10.1371/journal.pone.0001854.g003</span></div><a id="article1.body1.sec2.sec2.sec2.p2" name="article1.body1.sec2.sec2.sec2.p2"></a><p>An advantage of the unfolding representation (<a href="#pone-0001854-g003">Figure 3</a>) is that probe-specific behavior can be easily grasped and therefore a number of particularities can be observed, indicated by arrowheads. Two ribosomal protein genes were distant from their cluster, <em>Rpl3l</em> (ribosomal protein L3-like) and <em>Rpl39l</em> (ribosomal protein L39-like). Interestingly, <em>Rpl3l</em> could be found close to muscle, heart and diaphragm, exactly the group of tissues which had a very high expression of respiratory chain genes. An explanation for this exception to the clustered unfolding of ribosomal protein transcripts is proposed in section 3.1. <em>Rpl39l</em> was even more distinct on the unfolding plot and was in close proximity to testis; in multiple databases for different species (e.g. Genecards, Mouse Genome Informatics) this gene was indeed described to be testis-specific.</p>



<h4>3. Origins of exceptionally deviating ribosomal protein mRNA expression profiles</h4>
<h5>3.1 Tissue-specific isoforms: Rpl3 versus Rpl3l.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>On top of the gene and tissue effect contributing to expression variation, we described a significant gene-tissue interaction effect. This originated from individual genes displaying an aberrant expression in one or a few tissues. To illustrate, we further examined <em>Rpl3</em>, which has an isoform <em>Rpl3l</em> (<em>Rpl3</em>-like). The expression of <em>Rpl3</em> was significantly lower in skeletal muscle and diaphragm muscle –and to a lesser extent the heart- than in other tissues. However, exactly in these tissues, expression of <em>Rpl3</em>-like mRNA was specifically and abundantly detected. A similar result was observed independent of the way microarray data were processed (data not shown).</p>
<a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>We also confirmed tissue-specific expression of <em>Rpl3l</em> in another species (rat), based on a more limited set of tissue microarrays (7 tissues, with 3 replicates for each tissue). Similar to mouse tissues, <em>Rpl3l</em> was exclusively expressed in muscle, the tissue with the lowest <em>Rpl3</em> expression (data not shown). In addition, inspection of publicly available microarray data from human tissues revealed a similar tissue specific expression. The GDS596 record in Gene Expression Omnibus (GEO) is based on data from 79 physiologically normal human tissues. In <a href="#pone.0001854.s003">Figure S3A and S3B</a>, the expression profiles of probesets 211073_x_at and 206768_at representing <em>Rpl3</em> and <em>Rpl3l</em> respectively is shown. Expression in heart and skeletal muscle was low for <em>Rpl3</em> and high for <em>Rpl3l</em> and vice versa for the other tissues. These additional data emphasize that –in addition to a general biological variation among tissues in ribosomal protein mRNA expression- evolutionary conserved more profound differences exist for <em>Rpl3</em>, related to expression of a tissue specific isoform, <em>Rpl3l</em> .</p>

<h5>3.2 Splice variants and alternative termination.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>In addition to tissue-specific isoforms, profound differences in mRNA signal between tissues may also be the result of probe design and tissue-dependent alternative splicing or alternative termination. This was exemplified by ribosomal protein L9 (<em>Rpl9</em>), a gene ranked number 6 in the list of most stable genes <a href="#pone.0001854-deJonge1">[6]</a>. Two Affymetrix mouse 430 2.0 probesets are targeted to exons of the mouse <em>Rpl9</em> gene, as shown on <a href="#pone-0001854-g004">Figure 4B</a>. Probeset 1416420_a_at is targeted to exon 4 (out of 7 exons) and probeset 1443843_x_at in part to an intron and exon 7 (3′ UTR). We plotted the tissue profile of hybridization signals for both probesets side by side (<a href="#pone-0001854-g004">Figure 4C</a>), with individual data points displayed as dots. Both probesets detected similar expression levels in all tissues, however, there was one strong exception as a very low signal was detected by probeset 1443843_x_at (blue) in seminal vesicles. For the same tissue, probeset 1416420_a_at (red) produced the highest signal (10-fold higher than 1443843_x_at). Again, this expression pattern could be observed independent of the way microarray data were processed (data not shown). Therefore, dependent on the region in the gene where the probes are binding to measure expression of a reference gene, profound differences among conditions can be the result of alternatively spliced or terminated transcript variants.</p>
<div class="figure" id="pone-0001854-g004"><div class="img"><a name="pone-0001854-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001854" data-uri="info:doi/10.1371/journal.pone.0001854.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001854.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001854.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001854.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001854.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Two biological mechanisms underlying more pronounced variation in gene expression.</span></strong></p><a id="article1.body1.sec2.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec3.sec2.fig1.caption1.p1"></a><p>(A) Expression of <em>Rpl3</em> (blue) and <em>Rpl3l</em> (red) in 22 different tissues. Note that tissues where <em>Rpl3l</em> was expressed (heart, diaphragm and muscle) had the lowest expression of <em>Rpl3</em>. (B) Schematic of Affymetrix probeset binding sites on the <em>Rpl9</em> transcript. Probeset 1443843_x_at (blue) binds most 3′ and probeset 1416420_a_at (red) binds more upstream. (C) Expression levels of <em>Rpl9</em> as detected by each of the probesets shown above. Both probesets yield similar signals, except in one tissue: seminal vesicles. This indicates that in seminal vesicles, an <em>Rpl9</em> transcript variant exists.</p>
<span>doi:10.1371/journal.pone.0001854.g004</span></div>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The selection of good reference genes is an ongoing debate. A confounding issue is the use of the terms “housekeeping” and “reference” genes; housekeeping genes are often used as reference genes, although for many of these individually it was shown they are not good reference genes. Some groups claimed good reference genes do not exist <a href="#pone.0001854-Lee1">[3]</a>, whereas publications regularly appear in which new reference genes are proposed <a href="#pone.0001854-Kouadjo1">[10]</a>, <a href="#pone.0001854-Lee2">[11]</a>. Hsiao <em>et al.</em> studied gene expression in 19 different human tissue types <a href="#pone.0001854-Hsiao1">[7]</a> and defined a subset of 451 genes as housekeeping/maintenance genes based on a present call by the Affymetrix algorithm in all tissues. Most of the ribosomal protein encoding genes and some respiratory chain protein genes were included in this set. They noted that housekeeping genes define basic cellular processes and could be used as a reference standard but paradoxically mentioned that maintenance/housekeeping genes exhibit unique patterns for each specific tissue type. So far, there is still a discussion whether certain housekeeping genes can be used as reference genes over a broad range of tissues or conditions.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Recently, a large scale meta-analysis revealed a set of genes with an enhanced stability, of which the majority were ribosomal protein genes <a href="#pone.0001854-deJonge1">[6]</a>. Interestingly, no ribosomal protein genes appeared in the list of stable housekeeping genes provided by Hsiao <em>et al</em>. In the present study, we further investigated the stability of ribosomal protein genes and identified some biological phenomena affecting this stability. By means of two-way ANOVA, we still observed a very significant difference in gene expression among different tissues, on top of any possible probe effect, which reflects the specific (e.g., metabolic, protein synthetic) needs of the different tissues. The variation which was attributable to technical replication errors and biological differences between animals was much smaller than the expression variation of these candidate reference genes between tissues. This indicated that ribosomal protein genes cannot be used as housekeeping reference genes when comparing different tissues. When we investigated the most stable gene, <em>Rps13</em>, we observed up to 6-fold expression differences across tissues in our dataset. These differences were representative for larger publicly available datasets. <em>Rps13</em> was highly expressed in exocrine protein secreting glands and tissues containing a high proportion of proliferating cells. In fact, we observed that almost all ribosomal protein genes were co-expressed at high levels in these same tissues. The most likely explanation for this observation is biological as a high need for ribosomes during protein secretion and cell division is expected. Actively growing mammalian cells contain 5 to 10 million ribosomes that must be synthesized each time the cell divides <a href="#pone.0001854-Cooper1">[12]</a>. Proliferating lymphocytes, of which many are present in spleen and thymus, continuously produce cytoplasmic ribosomes <a href="#pone.0001854-Rubin1">[13]</a>. This idea was further supported by molecular insights in the coordinated expression of genes required for ribosome biogenesis <a href="#pone.0001854-FromontRacine1">[14]</a>. However, a second explanation for our data could be a series of tissue-specific artifacts, which resulted in tissue-dependent variations in hybridization signals, while the probed mRNA were -in fact- stably expressed. We provided strong evidence against this possibility by investigating in the same samples the expression of respiratory chain proteins, which are responsible for nutrient-induced ATP production and therefore essential for virtually all cells. When we quantified the nuclear encoded mRNAs for 69 of these subunits, we found again important variation between tissues in the expression level; importantly, the tissue pattern of expression levels was different as compared to ribosomal protein genes, which makes artifacts in sample preparation, hybridization or normalization unlikely. The tissue-specific behavior of gene groups was visualized by multidimensional unfolding analysis <a href="#pone.0001854-VanDeun1">[8]</a>. This method graphically plots genes close to the conditions (tissues) in which they have the highest expression. Strikingly, all ribosomal protein genes cluster in a group which is distinct from another housekeeping gene family, the respiratory chain genes. Two ribosomal protein genes were observed outside their cluster, <em>Rpl3l</em> and <em>Rpl39l</em>, which were the genes which had no present calls in certain tissues and were already known to be tissue-specific.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In further support of a biological explanation, we observed that these differences between tissues were not limited to one specific way of processing the microarray data. Gene expression measures are only obtained after a number of pre-processing steps which can be performed by different normalization procedures. We evaluated the effect of background corrected data, global scaling instead of quantile normalization and median polish summarization, but found all described effects present invariant of the processing method.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In addition to this general biological variation for all of these housekeeping genes, a more profound degree of variation seems based upon the existence of isoforms which are present only in a subset of tissues. This tissue specific expression of the isoform is often accompanied by a lowered expression of the other isoform(s). An example given here is ribosomal protein L3 (<em>Rpl3</em>), which has an isoform called ribosomal protein L3-like (<em>Rpl3l</em>). <em>Rpl3</em> was described to be ubiquitously expressed in all tissues, whereas <em>Rpl3l</em> was strongly expressed in skeletal muscle and heart tissue <a href="#pone.0001854-VanRaay1">[15]</a>, exactly the tissues which had the lowest <em>Rpl3</em> expression. <em>Rpl3</em> expression is autoregulated by alternative splicing of overexpressed transcripts, followed by degradation of these transcripts through nonsense-mediated RNA decay <a href="#pone.0001854-Cuccurese1">[16]</a>. Expression of <em>Rpl3l</em> in certain tissues might also favor alternative splicing of <em>Rpl3</em>, which would explain the lower expression in these tissues. Also for respiratory chain protein genes, tissue-specific isoforms have been described. The gene for the heart isoform of cytochrome c oxidase subunit VIa (<em>Cox6ah</em>) is expressed only in striated muscle, whereas the gene for the liver form (<em>Cox6al</em>) is expressed in all tissues, albeit at low levels in contractile muscle <a href="#pone.0001854-Bachman1">[17]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Another origin of variation in gene expression data relates to the probe position relative to the gene they are designed to interrogate. Since Affymetrix 3′ expression arrays contain probes designed at the 3′ end of genes, they may falsely not detect any transcript when the transcript is alternatively terminated before the binding site or when alternative splicing occurs. We showed evidence for alternative splicing or termination specifically in one tissue, being the seminal vesicle, where a shortened transcript was present. Seminal vesicle was not included in any of the 13629 arrays used for the data analysis by de Jonge <em>et al</em>. Even if this tissue would be represented, then still the alternative splicing or termination could be obscured by the averaging over multiple probesets. More importantly, when a single QPCR probe would be designed in this region, this effect might unexpectedly appear. Terminal changes may affect regulation by skipping (or introducing) microRNA binding sites or even cause differential subcellular localization. It is estimated that 40–79% of human genes with multiple exons produce transcript variants <a href="#pone.0001854-Nakao1">[18]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>How can these differences in gene expression across many tissues, both in our own dataset and in public datasets be reconciled with the finding of stable housekeeping genes in a large meta-analysis <a href="#pone.0001854-deJonge1">[6]</a>? One reason that de Jonge <em>et al.</em> still retain 16 stable genes (of which 15 are listed in the paper) is based on their selection criteria. A CV&lt;4% is being used. In our example with the random expression profiles (data displayed in <a href="#pone.0001854.s002">Fig S2</a>), we calculated the effect of 100 tissues having a significantly lower expression of a gene in a set of 13629 arrays. We show that none of these examples had a CV higher than 4%. Obviously, when a large number of arrays are used, outliers will not easily influence this CV. Moreover, all of these examples also satisfy the two other criteria used by de Jonge <em>et al.</em>, showing that these criteria do not exclude potential reference genes which are significantly lower in a subset of tissues.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>A more important reason that none of these examples would be excluded by the criteria used by de Jonge <em>et al.</em> is their interpretation of the maximum fold change (MFC). A ratio of 2 was applied to the log2 transformed expression values instead of the absolute expression values (see Table S1 in <a href="#pone.0001854-deJonge1">[6]</a>). This results in a high range of expression values that can meet the criterion of de Jonge <em>et al</em>: the maximum expression that can be measured using Affymetrix chips is a log2 value of 16, which would result in log2 values above 8 still included as a potential reference gene, representing a 256 fold change in expression values.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In this paper, the reference gene status of ribosomal protein genes was questioned by showing that as a group they were more highly expressed in tissues with faster cell division and by showing more profound differences between conditions on the basis of tissue-specific isoforms or transcript variants. This supports the idea that even for housekeeping genes, whose products are indispensable for every living cell and which are relatively stably expressed, there are tissue-specific differences based upon extra demands in the required rate at which new housekeeping proteins need to be produced to maintain cell function. For a replicating cell, this means the extra synthesis of a new set of ribosomes, and for skeletal muscle, the maintenance of the mitochondrial respiratory chain to sustain ATP production for mechanical work. The selection of good reference genes will be thus be dependent on the subset of tissues used in a particular experiment and the experimental variables. As previously discussed, it seems unlikely to find genes which are expressed at the same level across all tissues of an organism. Therefore, we caution against using so-called stable genes identified by meta-analyses when designing an experiment. However, we support the use of microarrays to select reference genes, since this permits the selection of the most stable genes within the limited subset of tissues/conditions present in the particular experiment. The optimal set of reference genes depends on the tissue and should be selected and evaluated for each series of experiments <a href="#pone.0001854-Brinkhof1">[19]</a>. This has already been successfully described by means of microarray <a href="#pone.0001854-Dydensborg1">[20]</a>, <a href="#pone.0001854-Maccoux1">[21]</a> or QPCR screens <a href="#pone.0001854-Brinkhof1">[19]</a>, <a href="#pone.0001854-Peters1">[22]</a>. The genes found through these screens may very well be ribosomal protein genes, but need to be verified for stable expression before use as a reference gene.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Preparation of tissues and purified cells</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All experiments based upon laboratory animals were approved by committees for animal welfare at the Katholieke Universiteit Leuven. The following tissues were hand dissected from 10–12 week old C57Bl6 mice: liver, gastrocnemius muscle, brain, heart, adrenal gland, eye, small intestine, thymus, epidydimal adipose tissue, pituitary gland, kidney, parotis gland, spleen, lung, diaphragma, bone marrow, testis, and seminal vesicles (males); ovary and placenta (females). Fetal tissue was isolated at day 16. Embryonic stem cells were isolated as described in <a href="#pone.0001854-Schoonjans1">[23]</a>. Tissues were rinsed in phosphate-buffered saline, frozen in liquid nitrogen and stored at −80°C.</p>


<h4>RNA extraction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Total RNA was extracted using TRIzol Reagent according to the manufacturer's protocol (Gibco BRL, Carlsbad, CA), followed by a cleanup procedure with RNeasy columns (Qiagen, Cologne, Germany). Total RNA from pituitary gland, adrenal gland and embryonic stem cells was extracted using the Absolutely RNA microprep from Stratagene (CA). The total RNA quantity and quality was determined using the NanoDrop ND-1000 spectrophotometer (NanoDrop Technologies, DW) and the 2100 Bioanalyzer (Agilent, Waldbronn, Germany), respectively. Total RNA profiles of all tested samples were similar with sharp 18S and 28S rRNA peaks on a flat baseline.</p>


<h4>mRNA expression analysis via microarray</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Cellular mRNA was reverse transcribed into cDNA (SuperScript Choice System Invitrogen, Carlsbad, CA) using oligo-dT primers and a T7 RNA polymerase promoter site. Two µg of total RNA was used to prepare biotinylated cRNA with IVT labeling kit (Affymetrix, Santa Clara, CA) according to the Genechip expression analysis technical manual 701025 Rev.5, except for adrenal gland and pituitary gland where 1 µg of total RNA was used. The concentration of labeled cRNA was measured using the NanoDrop ND-1000 spectrophotometer. Labeled cRNA was fragmented in a fragmentation buffer during 35 min at 94°C. The quality of labeled and fragmented cRNA was analyzed using the Agilent bioanalyzer 2100. Fragmented cRNA was hybridised to mouse 430 2.0 arrays (Affymetrix) during 16 h at 45°C. The arrays were washed and stained in a fluidics station (Affymetrix) and scanned using the Affymetrix 3000 GeneScanner.</p>


<h4>Data Analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We used a microarray dataset consisting of 22 different murine tissues, with 3–5 replicates for each tissue (in total 70 microarrays). Quality controls of the arrays were according to manufacturer's criteria. All CEL files were analyzed using GCOS (Affymetrix GeneChip Operating Software) and the affy library <a href="#pone.0001854-Gautier1">[24]</a> of the BioConductor project <a href="#pone.0001854-Gentleman1">[25]</a>. We independently applied the MAS5 algorithm using global scaling to 150 (to assess the present calls) and RMA. We performed RMA with and without convolution background adjustment and with either median polish or average difference summarization. Results shown are based on RMA data uncorrected for background, with probe-level quantile normalization, and average difference summarization. All data were log2 transformed for normalization and all further data analysis was performed on log2 transformed data. For each of the genes discussed where more than one probeset referred to the same gene annotation, only the best performing probeset with the highest expression values was used in order to avoid bias towards transcripts with more than one probeset. The data files have been deposited in the NCBI Gene Expression Omnibus (GEO, <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) and are accessible through GEO series accession number GSE9954.</p>


<h4>ANOVA</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The analysis of variance was carried out using the generalized linear model (GLM) procedure in SAS. For the 81 one-way analyses of variance, significance was set equal to .01/81 = 0.00012 to keep the overall type I error at .01. The assumption of homoscedasticity using the Brown-Forsythe test, was met for all probes at the .01 level of significance and for all probes except one (<em>Rpl37a</em>) at the .05 level. For the two-way ANOVA, both the tissue and probe factor were treated as fixed (leading to fixed model) and options were specified to obtain sequential sum of squares. Since this was an unbalanced design, we used the type I (sequential) sum of squares considering the probe effect as the first effect. Note that in this analysis, independence at probe level was assumed. The pairwise comparisons, using Tukey's multiple comparison procedure were obtained on the estimated marginal means by use of the LSMEANS statement and appropriate options in this GLM procedure.</p>


<h4>Rank sum test</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The Wilcoxon rank sum statistic was obtained from S-PLUS and used to test the null hypothesis of equal means versus the one-sided alternative that the mean variance between tissues was larger than the mean variance between replicates.</p>


<h4>Multidimensional unfolding</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The average (over replicates) expression values, obtained from the log2 transformed data of 69 probesets for nuclear encoded respiratory chain genes and 81 probesets for ribosomal protein mRNA's, were submitted to the publicly available GENEFOLD toolbox <a href="#pone.0001854-VanDeun1">[8]</a>, using a multistart procedure based on 100 semi-rational starts and convergence set to 1000 iterations or a difference in loss with the previous iteration smaller than 1e-5.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001854"></div><a name="pone.0001854.s001" id="pone.0001854.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001854.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Calculated CV when out of a total of 13629 samples (the same number as in the meta analysis), 100 samples were taken from a tissue in which a certain gene was only marginally expressed (log2 expression normally distributed with mean 8 and standard deviation 0.3) whereas in the other 13529 samples this gene was abundant (log2 expression normally distributed with mean 12 and standard deviation 0.3). We generated 10000 random expression profiles that follow this scheme and calculated the CV. The mean±standard deviation was 3.80±0.02 and all randomly generated profiles had a CV lower than 4%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001854.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(6.73 MB TIF)</p>
<a name="pone.0001854.s002" id="pone.0001854.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001854.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">(A) Marginal means (estimated under the two-way ANOVA) of expression values for mRNAs encoding 69 respiratory chain proteins in each of the tissues, together with the 95 percent confidence interval. (B) Variance of expression levels for respiratory chain genes within replicates of tissues, representing biological variation between animals and technical error on measurements (triangles) compared to variance of expression between tissues (circles). Variance within replicated measurements was significantly smaller than variance of expression between different conditions (p&lt;.0001 using Wilcoxon's rank sum test).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001854.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.74 MB TIF)</p>
<a name="pone.0001854.s003" id="pone.0001854.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001854.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Conservation in mammals of tissue specific expression of Rpl3 isoforms (A) GDS596 record for probeset 211073_x_at in GEO, showing Rpl3 expression across 79 physiologically normal human tissues. (B) GDS596 record for probeset 206768_at in GEO, showing Rpl3l expression across 79 physiologically normal human tissues. Expression in heart and skeletal muscle was low for Rpl3 and high for Rpl3l and vice versa for the other tissues.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001854.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(2.75 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank Katleen Lemaire for mouse tissue preparation, Katrien Venken for mouse bone marrow, Luc Schoonjans for mouse embryonic stem cells, Sonia Leach and Peter Van Loo for helpful comments.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FS. Performed the experiments: LV. Analyzed the data: FS LT LT KE KV. Contributed reagents/materials/analysis tools: FS KV. Wrote the paper: FS KM YM LT LT KE KV IV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001854-Butte1" id="pone.0001854-Butte1"></a>Butte AJ, Dzau VJ, Glueck SB (2001) Further defining housekeeping, or “maintenance,” genes Focus on “A compendium of gene expression in normal human tissues”. Physiol Genomics  7: 95–96.  <ul class="find" data-citedArticleID="1079670"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Further+defining+housekeeping%2C+or+%3Fmaintenance%2C%3F+genes+Focus+on+%3FA+compendium+of+gene+expression+in+normal+human+tissues%3F.&amp;auth=&amp;atitle=Further+defining+housekeeping%2C+or+%3Fmaintenance%2C%3F+genes+Focus+on+%3FA+compendium+of+gene+expression+in+normal+human+tissues%3F." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+defining+housekeeping%2C+or+%E2%80%9Cmaintenance%2C%E2%80%9D+genes+Focus+on+%E2%80%9CA+compendium+of+gene+expression+in+normal+human+tissues%E2%80%9D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+defining+housekeeping%2C+or+%E2%80%9Cmaintenance%2C%E2%80%9D+genes+Focus+on+%E2%80%9CA+compendium+of+gene+expression+in+normal+human+tissues%E2%80%9D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001854-Glare1" id="pone.0001854-Glare1"></a>Glare EM, Divjak M, Bailey MJ, Walters EH (2002) beta-Actin and GAPDH housekeeping gene expression in asthmatic airways is variable and not suitable for normalising mRNA levels. Thorax  57: 765–770.  <ul class="find" data-citedArticleID="1079684" data-doi="10.1136/thorax.57.9.765"><li><a href="http://dx.doi.org/10.1136/thorax.57.9.765" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=beta-Actin+and+GAPDH+housekeeping+gene+expression+in+asthmatic+airways+is+variable+and+not+suitable+for+normalising+mRNA+levels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22beta-Actin+and+GAPDH+housekeeping+gene+expression+in+asthmatic+airways+is+variable+and+not+suitable+for+normalising+mRNA+levels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001854-Lee1" id="pone.0001854-Lee1"></a>Lee PD, Sladek R, Greenwood CM, Hudson TJ (2002) Control genes and variability: absence of ubiquitous reference transcripts in diverse mammalian expression studies. Genome Res  12: 292–297.  <ul class="find" data-citedArticleID="1079690" data-doi="10.1101/gr.217802"><li><a href="http://dx.doi.org/10.1101/gr.217802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+genes+and+variability%3A+absence+of+ubiquitous+reference+transcripts+in+diverse+mammalian+expression+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+genes+and+variability%3A+absence+of+ubiquitous+reference+transcripts+in+diverse+mammalian+expression+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001854-Suzuki1" id="pone.0001854-Suzuki1"></a>Suzuki T, Higgins PJ, Crawford DR (2000) Control selection for RNA quantitation. Biotechniques  29: 332–337.  <ul class="find" data-citedArticleID="1079704"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Control+selection+for+RNA+quantitation.&amp;auth=&amp;atitle=Control+selection+for+RNA+quantitation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+selection+for+RNA+quantitation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+selection+for+RNA+quantitation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001854-Thellin1" id="pone.0001854-Thellin1"></a>Thellin O, Zorzi W, Lakaye B, De BB, Coumans B, et al.  (1999) Housekeeping genes as internal standards: use and limits. J Biotechnol  75: 291–295.  <ul class="find" data-citedArticleID="1079706" data-doi="10.1016/s0168-1656(99)00163-7"><li><a href="http://dx.doi.org/10.1016/s0168-1656(99)00163-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Housekeeping+genes+as+internal+standards%3A+use+and+limits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Housekeeping+genes+as+internal+standards%3A+use+and+limits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001854-deJonge1" id="pone.0001854-deJonge1"></a>de Jonge HJ, Fehrmann RS, de Bont ES, Hofstra RM, Gerbens F, et al.  (2007) Evidence based selection of housekeeping genes. PLoS ONE  2: e898.  <ul class="find" data-citedArticleID="1079712" data-doi="10.1371/journal.pone.0000898"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+based+selection+of+housekeeping+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+based+selection+of+housekeeping+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001854-Hsiao1" id="pone.0001854-Hsiao1"></a>Hsiao LL, Dangond F, Yoshida T, Hong R, Jensen RV, et al.  (2001) A compendium of gene expression in normal human tissues. Physiol Genomics  7: 97–104.  <ul class="find" data-citedArticleID="1079686"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+compendium+of+gene+expression+in+normal+human+tissues.&amp;auth=&amp;atitle=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+compendium+of+gene+expression+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001854-VanDeun1" id="pone.0001854-VanDeun1"></a>Van Deun K, Marchal K, Heiser WJ, Engelen K, Van Mechelen I (2007) Joint mapping of genes and conditions via multidimensional unfolding analysis. BMC Bioinformatics  8: 181.  <ul class="find" data-citedArticleID="1079708" data-doi="10.1186/1471-2105-8-181"><li><a href="http://dx.doi.org/10.1186/1471-2105-8-181" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Joint+mapping+of+genes+and+conditions+via+multidimensional+unfolding+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Joint+mapping+of+genes+and+conditions+via+multidimensional+unfolding+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001854-Borg1" id="pone.0001854-Borg1"></a>Borg I, Groenen PJF (2005) Modern multidimensional scaling: Theory and Applications. New-York: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001854-Kouadjo1" id="pone.0001854-Kouadjo1"></a>Kouadjo KE, Nishida Y, Cadrin-Girard JF, Yoshioka M, St-Amand J (2007) Housekeeping and tissue-specific genes in mouse tissues. BMC Genomics  8: 127.  <ul class="find" data-citedArticleID="1079688" data-doi="10.1186/1471-2164-8-127"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Housekeeping+and+tissue-specific+genes+in+mouse+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Housekeeping+and+tissue-specific+genes+in+mouse+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001854-Lee2" id="pone.0001854-Lee2"></a>Lee S, Jo M, Lee J, Koh SS, Kim S (2007) Identification of novel universal housekeeping genes by statistical analysis of microarray data. J Biochem Mol Biol  40: 226–231.  <ul class="find" data-citedArticleID="1079692" data-doi="10.5483/bmbrep.2007.40.2.226"><li><a href="http://dx.doi.org/10.5483/bmbrep.2007.40.2.226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+novel+universal+housekeeping+genes+by+statistical+analysis+of+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+novel+universal+housekeeping+genes+by+statistical+analysis+of+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001854-Cooper1" id="pone.0001854-Cooper1"></a>Cooper GM (2000) The cell: a molecular approach. Washington, D.C: ASM Press.   <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0001854-Rubin1" id="pone.0001854-Rubin1"></a>Rubin AD (1968) Possible control of lymphocyte growth at the level of ribosome assembly. Nature  220: 196–197.  <ul class="find" data-citedArticleID="1079700" data-doi="10.1038/220196a0"><li><a href="http://dx.doi.org/10.1038/220196a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Possible+control+of+lymphocyte+growth+at+the+level+of+ribosome+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Possible+control+of+lymphocyte+growth+at+the+level+of+ribosome+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001854-FromontRacine1" id="pone.0001854-FromontRacine1"></a>Fromont-Racine M, Senger B, Saveanu C, Fasiolo F (2003) Ribosome assembly in eukaryotes. Gene  313: 17–42.  <ul class="find" data-citedArticleID="1079678" data-doi="10.1016/s0378-1119(03)00629-2"><li><a href="http://dx.doi.org/10.1016/s0378-1119(03)00629-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosome+assembly+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosome+assembly+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001854-VanRaay1" id="pone.0001854-VanRaay1"></a>Van Raay TJ, Connors TD, Klinger KW, Landes GM, Burn TC (1996) A novel ribosomal protein L3-like gene (RPL3L) maps to the autosomal dominant polycystic kidney disease gene region. Genomics  37: 172–176.  <ul class="find" data-citedArticleID="1079710" data-doi="10.1006/geno.1996.0538"><li><a href="http://dx.doi.org/10.1006/geno.1996.0538" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+ribosomal+protein+L3-like+gene+%28RPL3L%29+maps+to+the+autosomal+dominant+polycystic+kidney+disease+gene+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+ribosomal+protein+L3-like+gene+%28RPL3L%29+maps+to+the+autosomal+dominant+polycystic+kidney+disease+gene+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001854-Cuccurese1" id="pone.0001854-Cuccurese1"></a>Cuccurese M, Russo G, Russo A, Pietropaolo C (2005) Alternative splicing and nonsense-mediated mRNA decay regulate mammalian ribosomal gene expression. Nucleic Acids Res  33: 5965–5977.  <ul class="find" data-citedArticleID="1079674" data-doi="10.1093/nar/gki905"><li><a href="http://dx.doi.org/10.1093/nar/gki905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+splicing+and+nonsense-mediated+mRNA+decay+regulate+mammalian+ribosomal+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+splicing+and+nonsense-mediated+mRNA+decay+regulate+mammalian+ribosomal+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001854-Bachman1" id="pone.0001854-Bachman1"></a>Bachman NJ, Riggs PK, Siddiqui N, Makris GJ, Womack JE, et al.  (1997) Structure of the human gene (COX6A2) for the heart/muscle isoform of cytochrome c oxidase subunit VIa and its chromosomal location in humans, mice, and cattle. Genomics  42: 146–151.  <ul class="find" data-citedArticleID="1079664" data-doi="10.1006/geno.1997.4687"><li><a href="http://dx.doi.org/10.1006/geno.1997.4687" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+human+gene+%28COX6A2%29+for+the+heart%2Fmuscle+isoform+of+cytochrome+c+oxidase+subunit+VIa+and+its+chromosomal+location+in+humans%2C+mice%2C+and+cattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+human+gene+%28COX6A2%29+for+the+heart%2Fmuscle+isoform+of+cytochrome+c+oxidase+subunit+VIa+and+its+chromosomal+location+in+humans%2C+mice%2C+and+cattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001854-Nakao1" id="pone.0001854-Nakao1"></a>Nakao M, Barrero RA, Mukai Y, Motono C, Suwa M, et al.  (2005) Large-scale analysis of human alternative protein isoforms: pattern classification and correlation with subcellular localization signals. Nucleic Acids Res  33: 2355–2363.  <ul class="find" data-citedArticleID="1079696" data-doi="10.1093/nar/gki520"><li><a href="http://dx.doi.org/10.1093/nar/gki520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+analysis+of+human+alternative+protein+isoforms%3A+pattern+classification+and+correlation+with+subcellular+localization+signals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+analysis+of+human+alternative+protein+isoforms%3A+pattern+classification+and+correlation+with+subcellular+localization+signals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001854-Brinkhof1" id="pone.0001854-Brinkhof1"></a>Brinkhof B, Spee B, Rothuizen J, Penning LC (2006) Development and evaluation of canine reference genes for accurate quantification of gene expression. Anal Biochem  356: 36–43.  <ul class="find" data-citedArticleID="1079668" data-doi="10.1016/j.ab.2006.06.001"><li><a href="http://dx.doi.org/10.1016/j.ab.2006.06.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+evaluation+of+canine+reference+genes+for+accurate+quantification+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+evaluation+of+canine+reference+genes+for+accurate+quantification+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001854-Dydensborg1" id="pone.0001854-Dydensborg1"></a>Dydensborg AB, Herring E, Auclair J, Tremblay E, Beaulieu JF (2006) Normalizing genes for quantitative RT-PCR in differentiating human intestinal epithelial cells and adenocarcinomas of the colon. Am J Physiol Gastrointest Liver Physiol  290: G1067–G1074.  <ul class="find" data-citedArticleID="1079676" data-doi="10.1152/ajpgi.00234.2005"><li><a href="http://dx.doi.org/10.1152/ajpgi.00234.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Normalizing+genes+for+quantitative+RT-PCR+in+differentiating+human+intestinal+epithelial+cells+and+adenocarcinomas+of+the+colon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Normalizing+genes+for+quantitative+RT-PCR+in+differentiating+human+intestinal+epithelial+cells+and+adenocarcinomas+of+the+colon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001854-Maccoux1" id="pone.0001854-Maccoux1"></a>Maccoux LJ, Clements DN, Salway F, Day PJ (2007) Identification of new reference genes for the normalisation of canine osteoarthritic joint tissue transcripts from microarray data. BMC Mol Biol  8: 62.  <ul class="find" data-citedArticleID="1079694" data-doi="10.1186/1471-2199-8-62"><li><a href="http://dx.doi.org/10.1186/1471-2199-8-62" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+new+reference+genes+for+the+normalisation+of+canine+osteoarthritic+joint+tissue+transcripts+from+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+new+reference+genes+for+the+normalisation+of+canine+osteoarthritic+joint+tissue+transcripts+from+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001854-Peters1" id="pone.0001854-Peters1"></a>Peters IR, Peeters D, Helps CR, Day MJ (2007) Development and application of multiple internal reference (housekeeper) gene assays for accurate normalisation of canine gene expression studies. Vet Immunol Immunopathol  117: 55–66.  <ul class="find" data-citedArticleID="1079698" data-doi="10.1016/j.vetimm.2007.01.011"><li><a href="http://dx.doi.org/10.1016/j.vetimm.2007.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+application+of+multiple+internal+reference+%28housekeeper%29+gene+assays+for+accurate+normalisation+of+canine+gene+expression+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+application+of+multiple+internal+reference+%28housekeeper%29+gene+assays+for+accurate+normalisation+of+canine+gene+expression+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001854-Schoonjans1" id="pone.0001854-Schoonjans1"></a>Schoonjans L, Kreemers V, Danloy S, Moreadith RW, Laroche Y, et al.  (2003) Improved generation of germline-competent embryonic stem cell lines from inbred mouse strains. Stem Cells  21: 90–97.  <ul class="find" data-citedArticleID="1079702" data-doi="10.1634/stemcells.21-1-90"><li><a href="http://dx.doi.org/10.1634/stemcells.21-1-90" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+generation+of+germline-competent+embryonic+stem+cell+lines+from+inbred+mouse+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+generation+of+germline-competent+embryonic+stem+cell+lines+from+inbred+mouse+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001854-Gautier1" id="pone.0001854-Gautier1"></a>Gautier L, Cope L, Bolstad BM, Irizarry RA (2004) affy–analysis of Affymetrix GeneChip data at the probe level. Bioinformatics  20: 307–315.  <ul class="find" data-citedArticleID="1079680" data-doi="10.1093/bioinformatics/btg405"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=affy%E2%80%93analysis+of+Affymetrix+GeneChip+data+at+the+probe+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22affy%E2%80%93analysis+of+Affymetrix+GeneChip+data+at+the+probe+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001854-Gentleman1" id="pone.0001854-Gentleman1"></a>Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, et al.  (2004) Bioconductor: open software development for computational biology and bioinformatics. Genome Biol  5: R80.  <ul class="find" data-citedArticleID="1079682" data-doi="10.1186/gb-2004-5-10-r80"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.XML" value="76404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.PDF" value="370075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g001.PNG_L" value="417500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g001.PNG_M" value="83620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g001.PNG_S" value="10238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g001.TIF" value="1606452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g001.PNG_I" value="29845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g002.PNG_L" value="428037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g002.PNG_M" value="86954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g002.PNG_S" value="14369"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g002.TIF" value="1174028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g002.PNG_I" value="55355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g003.PNG_L" value="405366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g003.PNG_M" value="82295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g003.PNG_S" value="11554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g003.TIF" value="921374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g003.PNG_I" value="27158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g004.PNG_L" value="606612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g004.PNG_M" value="89598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g004.PNG_S" value="14388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g004.TIF" value="2431822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.g004.PNG_I" value="96793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.s001.TIF" value="6701318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.s002.TIF" value="2053566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001854.s003.TIF" value="3840376"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001854&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001854" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001854&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001854&volume=&issue=&title=Using Ribosomal Protein Genes as Reference: A Tale of Caution&author_name=Lieven%20Thorrez%2C%20Katrijn%20Van%20Deun%2C%20L%C3%A9on-Charles%20Tranchevent%2C%20Leentje%20Van%20Lommel%2C%20Kristof%20Engelen%2C%20Kathleen%20Marchal%2C%20Yves%20Moreau%2C%20Iven%20Van%20Mechelen%2C%20Frans%20Schuit&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001854" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854&amp;t=Using%20Ribosomal%20Protein%20Genes%20as%20Reference%3A%20A%20Tale%20of%20Caution" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854&title=Using%20Ribosomal%20Protein%20Genes%20as%20Reference%3A%20A%20Tale%20of%20Caution&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854&amp;title=Using%20Ribosomal%20Protein%20Genes%20as%20Reference%3A%20A%20Tale%20of%20Caution" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001854&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Using Ribosomal Protein Genes as Reference: A Tale of Caution';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001854';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Using%20Ribosomal%20Protein%20Genes%20as%20Reference%3A%20A%20Tale%20of%20Caution http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001854" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001854" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Analysis+of+variance%22" title="Search for articles in the subject area:'Analysis of variance'"><div class="flagText">Analysis of variance</div></a>
              <div data-categoryid="42193" data-articleid="26532"
                   data-categoryname="Analysis of variance"
                   class="flagImage" title="Flag 'Analysis of variance' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26532"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heart%22" title="Search for articles in the subject area:'Heart'"><div class="flagText">Heart</div></a>
              <div data-categoryid="47373" data-articleid="26532"
                   data-categoryname="Heart"
                   class="flagImage" title="Flag 'Heart' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="26532"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Meta-analysis%22" title="Search for articles in the subject area:'Meta-analysis'"><div class="flagText">Meta-analysis</div></a>
              <div data-categoryid="32065" data-articleid="26532"
                   data-categoryname="Meta-analysis"
                   class="flagImage" title="Flag 'Meta-analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26532"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Muscle+tissue%22" title="Search for articles in the subject area:'Muscle tissue'"><div class="flagText">Muscle tissue</div></a>
              <div data-categoryid="49177" data-articleid="26532"
                   data-categoryname="Muscle tissue"
                   class="flagImage" title="Flag 'Muscle tissue' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="26532"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=836'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6944'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7391&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=4689">maybe the true and only (=universal) control gene does simply not exist</a><br>Posted by m_ralser</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
