

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Origins of Novel Protein Interactions during Animal Opsin Evolution</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001054"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001054" />

    <meta name="citation_title" content="The Origins of Novel Protein Interactions during Animal Opsin Evolution"/>
    <meta itemprop="name" content="The Origins of Novel Protein Interactions during Animal Opsin Evolution"/>

      <meta name="citation_author" content="David C. Plachetzki"/>
            <meta name="citation_author_institution" content="Ecology, Evolution and Marine Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>
      <meta name="citation_author" content="Bernard M. Degnan"/>
            <meta name="citation_author_institution" content="School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Todd H. Oakley"/>
            <meta name="citation_author_institution" content="Ecology, Evolution and Marine Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America"/>

    <meta name="citation_date" content="2007/10/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001054.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1054"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular evolution of vertebrate visual pigments.; citation_author=S Yokoyama; citation_journal_title=Prog Retin Eye Res; citation_volume=19; citation_number=1; citation_pages=385-419; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization and expression of the UV opsin in bumblebees: three ommatidial subtypes in the retina and a new photoreceptor organ in the lamina.; citation_author=J Spaethe; citation_author=AD Briscoe; citation_journal_title=J Exp Biol; citation_volume=208; citation_number=2; citation_pages=2347-2361; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of a duplicated pancreatic ribonuclease gene in a leaf-eating monkey.; citation_author=J Zhang; citation_author=YP Zhang; citation_author=HF Rosenberg; citation_journal_title=Nat Genet; citation_volume=30; citation_number=3; citation_pages=411-415; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolution of hormone-receptor complexity by molecular exploitation.; citation_author=JT Bridgham; citation_author=SM Carroll; citation_author=JW Thornton; citation_journal_title=Science; citation_volume=312; citation_number=4; citation_pages=97-101; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A single amino acid mutation contributes to adaptive beach mouse color pattern.; citation_author=HE Hoekstra; citation_author=RJ Hirschmann; citation_author=RA Bundey; citation_author=PA Insel; citation_author=JP Crossland; citation_journal_title=Science; citation_volume=313; citation_number=5; citation_pages=101-104; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic and developmental basis of evolutionary pelvic reduction in threespine sticklebacks.; citation_author=MD Shapiro; citation_author=ME Marks; citation_author=CL Peichel; citation_author=BK Blackman; citation_author=KS Nereng; citation_journal_title=Nature; citation_volume=428; citation_number=6; citation_pages=717-723; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=AVPR1a and SLC6A4 gene polymorphisms are associated with creative dance performance.; citation_author=R Bachner-Melman; citation_author=C Dina; citation_author=AH Zohar; citation_author=N Constantini; citation_author=E Lerer; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=7; citation_pages=e42; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ancient and recent positive selection transformed opioid cis-regulation in humans.; citation_author=MV Rockman; citation_author=MW Hahn; citation_author=N Soranzo; citation_author=F Zimprich; citation_author=DB Goldstein; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=8; citation_pages=e387; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Convergent adaptation of human lactase persistence in Africa and Europe.; citation_author=SA Tishkoff; citation_author=FA Reed; citation_author=A Ranciaro; citation_author=BF Voight; citation_author=CC Babbitt; citation_journal_title=Nat Genet; citation_volume=39; citation_number=9; citation_pages=31-40; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The evolution of genetic networks by non-adaptive processes.; citation_author=M Lynch; citation_journal_title=Nat Rev Genet; citation_volume=8; citation_number=10; citation_pages=803-813; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of HoxA-11 and HoxA-13 at the origin of the uterus in mammals.; citation_author=VJ Lynch; citation_author=JJ Roth; citation_author=K Takahashi; citation_author=CW Dunn; citation_author=DF Nonaka; citation_journal_title=Proc Biol Sci; citation_volume=271; citation_number=11; citation_pages=2201-2207; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of eyes and photoreceptor cell types.; citation_author=D Arendt; citation_journal_title=Int J Dev Biol; citation_volume=47; citation_number=12; citation_pages=563-571; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The locus of evolution: evo devo and the genetics of adaptation.; citation_author=HE Hoekstra; citation_author=JA Coyne; citation_journal_title=Evolution Int J Org Evolution; citation_volume=61; citation_number=13; citation_pages=995-1016; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Reconstructing the eyes of Urbilateria.; citation_author=D Arendt; citation_author=J Wittbrodt; citation_journal_title=Transactions of the Royal Society of London B; citation_number=14; citation_pages=1545-1563; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The opsins.; citation_author=A Terakita; citation_journal_title=Genome Biol; citation_volume=6; citation_number=15; citation_pages=213; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Opsins and clusters of sensory G-protein-coupled receptors in the sea urchin genome.; citation_author=F Raible; citation_author=K Tessmar-Raible; citation_author=E Arboleda; citation_author=T Kaller; citation_author=P Bork; citation_journal_title=Dev Biol; citation_volume=300; citation_number=16; citation_pages=461-475; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Visual transduction in Drosophila.; citation_author=RC Hardie; citation_author=P Raghu; citation_journal_title=Nature; citation_volume=413; citation_number=17; citation_pages=186-193; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A novel Go-mediated phototransduction cascade in scallop visual cells.; citation_author=D Kojima; citation_author=A Terakita; citation_author=T Ishikawa; citation_author=Y Tsukahara; citation_author=A Maeda; citation_journal_title=J Biol Chem; citation_volume=272; citation_number=18; citation_pages=22979-22982; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Amphioxus homologs of Go-coupled rhodopsin and peropsin having 11-cis- and all-trans-retinals as their chromophores.; citation_author=M Koyanagi; citation_author=A Terakita; citation_author=K Kubokawa; citation_author=Y Shichida; citation_journal_title=FEBS Lett; citation_volume=531; citation_number=19; citation_pages=525-528; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cephalochordate melanopsin: evolutionary linkage between invertebrate visual cells and vertebrate photosensitive retinal ganglion cells.; citation_author=M Koyanagi; citation_author=K Kubokawa; citation_author=H Tsukamoto; citation_author=Y Shichida; citation_author=A Terakita; citation_journal_title=Curr Biol; citation_volume=15; citation_number=20; citation_pages=1065-1069; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Neurospora Photoreceptors.; citation_author=JC Dunlap; citation_author=JJ Loros; citation_number=21; citation_pages=473; citation_date=2005; citation_publisher=Wiley-VCH Verlag GmbH & Co.; " />
      <meta name="citation_reference" content="citation_title=Of clades and clans: terms for phylogenetic relationships in unrooted trees.; citation_author=M Wilkinson; citation_author=JO McInerney; citation_author=RP Hirt; citation_author=PG Foster; citation_author=TM Embley; citation_journal_title=Trends in Ecology and Evolution; citation_volume=22; citation_number=22; citation_pages=114-115; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Estimating metazoan divergence times with a molecular clock.; citation_author=KJ Peterson; citation_author=JB Lyons; citation_author=KS Nowak; citation_author=CM Takacs; citation_author=MJ Wargo; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=101; citation_number=23; citation_pages=6536-6541; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record.; citation_author=KJ Peterson; citation_author=NJ Butterfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=24; citation_pages=9547-9552; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ciliary photoreceptors with a vertebrate-type opsin in an invertebrate brain.; citation_author=D Arendt; citation_author=K Tessmar-Raible; citation_author=H Snyman; citation_author=AW Dorresteijn; citation_author=J Wittbrodt; citation_journal_title=Science; citation_volume=306; citation_number=25; citation_pages=869-871; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Pteropsin: a vertebrate-like non-visual opsin expressed in the honey bee brain.; citation_author=RA Velarde; citation_author=CD Sauer; citation_author=KK Walden; citation_author=SE Fahrbach; citation_author=HM Robertson; citation_journal_title=Insect Biochem Mol Biol; citation_volume=35; citation_number=26; citation_pages=1367-1377; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The amino terminus of the fourth cytoplasmic loop of rhodopsin modulates rhodopsin-transducin interaction.; citation_author=EP Marin; citation_author=AG Krishna; citation_author=TA Zvyaga; citation_author=J Isele; citation_author=F Siebert; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=27; citation_pages=1930-1936; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Trichomonas hydrogenosomes contain the NADH dehydrogenase module of mitochondrial complex I.; citation_author=I Hrdy; citation_author=RP Hirt; citation_author=P Dolezal; citation_author=L Bardonova; citation_author=PG Foster; citation_journal_title=Nature; citation_volume=432; citation_number=28; citation_pages=618-622; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional divergence after gene duplication and sequence-structure relationship: a case study of G-protein alpha subunits.; citation_author=Y Zheng; citation_author=D Xu; citation_author=X Gu; citation_journal_title=J Exp Zoolog B Mol Dev Evol; citation_volume=308; citation_number=29; citation_pages=85-96; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Parietal-eye phototransduction components and their potential evolutionary implications.; citation_author=CY Su; citation_author=DG Luo; citation_author=A Terakita; citation_author=Y Shichida; citation_author=HW Liao; citation_journal_title=Science; citation_volume=311; citation_number=30; citation_pages=1617-1621; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Spectral sensitivity in a sponge larva.; citation_author=SP Leys; citation_author=TW Cronin; citation_author=BM Degnan; citation_author=JN Marshall; citation_journal_title=J Comp Physiol A Neuroethol Sens Neural Behav Physiol; citation_volume=188; citation_number=31; citation_pages=199-202; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Retinylidene proteins: structures and functions from archaea to humans.; citation_author=JL Spudich; citation_author=CS Yang; citation_author=KH Jung; citation_author=EN Spudich; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=16; citation_number=32; citation_pages=365-392; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fine Structure of Photoreceptors in the Hydromedusan, Polyorchis Penicillatus.; citation_author=RM Eakin; citation_author=JA Westfall; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=48; citation_number=33; citation_pages=826-833; citation_date=1962; " />
      <meta name="citation_reference" content="citation_title=Evolutionary Significance of Photoreceptors: In Retrospect.; citation_author=RM Eakin; citation_journal_title=American Zoologist; citation_volume=19; citation_number=34; citation_pages=647-653; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Extensive gene duplication in the early evolution of animals before the parazoan-eumetazoan split demonstrated by G proteins and protein tyrosine kinases from sponge and hydra.; citation_author=H Suga; citation_author=M Koyanagi; citation_author=D Hoshiyama; citation_author=K Ono; citation_author=N Iwabe; citation_journal_title=J Mol Evol; citation_volume=48; citation_number=35; citation_pages=646-653; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The molecular phylogeny of a nematode-specific clade of heterotrimeric G-protein alpha-subunit genes.; citation_author=DM O'Halloran; citation_author=DA Fitzpatrick; citation_author=GP McCormack; citation_author=JO McInerney; citation_author=AM Burnell; citation_journal_title=J Mol Evol; citation_volume=63; citation_number=36; citation_pages=87-94; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Light transduction in invertebrate hyperpolarizing photoreceptors: possible involvement of a Go-regulated guanylate cyclase.; citation_author=MP Gomez; citation_author=E Nasi; citation_journal_title=J Neurosci; citation_volume=20; citation_number=37; citation_pages=5254-5263; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The coevolution of blue-light photoreception and circadian rhythms.; citation_author=W Gehring; citation_author=M Rosbash; citation_journal_title=J Mol Evol; citation_volume=57; citation_number=38; citation_pages=S286-289; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Duplication and divergence: the evolution of new genes and old ideas.; citation_author=JS Taylor; citation_author=J Raes; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=39; citation_pages=615-643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution by gene duplication.; citation_author=S Ohno; citation_number=40; citation_date=1970; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=41; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=42; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Computational inference of homologous gene structures in the human genome.; citation_author=RF Yeh; citation_author=LP Lim; citation_author=CB Burge; citation_journal_title=Genome Res; citation_volume=11; citation_number=43; citation_pages=803-816; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of rhodopsin: A G protein-coupled receptor.; citation_author=K Palczewski; citation_author=T Kumasaka; citation_author=T Hori; citation_author=CA Behnke; citation_author=H Motoshima; citation_journal_title=Science; citation_volume=289; citation_number=44; citation_pages=739-745; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=CnNK-2, an NK-2 homeobox gene, has a role in patterning the basal end of the axis in hydra.; citation_author=A Grens; citation_author=L Gee; citation_author=DA Fisher; citation_author=HR Bode; citation_journal_title=Dev Biol; citation_volume=180; citation_number=45; citation_pages=473-488; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=T-Coffee: A novel method for fast and accurate multiple sequence alignment.; citation_author=C Notredame; citation_author=DG Higgins; citation_author=J Heringa; citation_journal_title=J Mol Biol; citation_volume=302; citation_number=46; citation_pages=205-217; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SEAVIEW and PHYLO_WIN: two graphic tools for sequence alignment and molecular phylogeny.; citation_author=N Galtier; citation_author=M Gouy; citation_author=C Gautier; citation_journal_title=Comput Appl Biosci; citation_volume=12; citation_number=47; citation_pages=543-548; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=PAUP*. 4.1b10 ed.; citation_author=D Swofford; citation_number=48; citation_date=2002; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=49; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=50; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ProtTest: selection of best-fit models of protein evolution.; citation_author=F Abascal; citation_author=R Zardoya; citation_author=D Posada; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=51; citation_pages=2104-2105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Tracer. MCMC Trace Analysis Tool. 1.0 ed.; citation_author=A Rambaut; citation_author=A Drummond; citation_number=52; citation_date=2003; citation_publisher=University of Oxford; " />
      <meta name="citation_reference" content="citation_title=TREEFINDER: a powerful graphical analysis environment for molecular phylogenetics.; citation_author=G Jobb; citation_author=A von Haeseler; citation_author=K Strimmer; citation_journal_title=BMC Evol Biol; citation_volume=4; citation_number=53; citation_pages=18; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=54; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=CONSEL: for assessing the confidence of phylogenetic tree selection.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=55; citation_pages=1246-1247; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cases in which parsimony or compatibility methods will be positively misleading.; citation_author=J Felsenstein; citation_journal_title=Systematic Zoology; citation_volume=27; citation_number=56; citation_pages=401-410; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Is the Felsenstein zone a fly trap?; citation_author=JP Huelsenbeck; citation_journal_title=Systematic Biology; citation_volume=46; citation_number=57; citation_pages=69-74; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Seq-Gen: An application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees.; citation_author=A Rambaut; citation_author=NC Grassly; citation_journal_title=Computer Applications in the Biosciences in press; citation_number=58; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Gene family evolution and homology: genomics meets phylogenetics.; citation_author=JW Thornton; citation_author=R DeSalle; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=1; citation_number=59; citation_pages=41-73; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=NOTUNG: a program for dating gene duplications and optimizing gene family trees.; citation_author=K Chen; citation_author=D Durand; citation_author=M Farach-Colton; citation_journal_title=J Comput Biol; citation_volume=7; citation_number=60; citation_pages=429-447; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A hybrid micro-macroevolutionary approach to gene tree reconstruction.; citation_author=D Durand; citation_author=BV Halldorsson; citation_author=B Vernot; citation_journal_title=J Comput Biol; citation_volume=13; citation_number=61; citation_pages=320-335; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Animal phylogeny.; citation_author=MJ Telford; citation_journal_title=Curr Biol; citation_volume=16; citation_number=62; citation_pages=R981-985; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Detecting positively selected amino acid sites using posterior predictive p-values. Pacific Symposium on Biocomputing, Proceedings: World Scientific.; citation_author=RH, JP Nielsen; citation_number=63; citation_pages=576-588; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=SIMMAP: stochastic character mapping of discrete traits on phylogenies.; citation_author=JP Bollback; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=64; citation_pages=88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure of bovine rhodopsin in a trigonal crystal form.; citation_author=J Li; citation_author=PC Edwards; citation_author=M Burghammer; citation_author=C Villa; citation_author=GF Schertler; citation_journal_title=J Mol Biol; citation_volume=343; citation_number=65; citation_pages=1409-1438; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An approximately unbiased test of phylogenetic tree selection.; citation_author=H Shimodaira; citation_journal_title=Syst Biol; citation_volume=51; citation_number=66; citation_pages=492-508; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the maximum likelihood estimate of the evolutionary tree topologies from DNA sequence data, and the branching order in hominoidea.; citation_author=H Kishino; citation_author=M Hasegawa; citation_journal_title=J Mol Evol; citation_volume=29; citation_number=67; citation_pages=170-179; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Bayesian phylogenetic inference using DNA sequences: a Markov Chain Monte Carlo Method.; citation_author=Z Yang; citation_author=B Rannala; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=68; citation_pages=717-724; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Origins of Novel Protein Interactions during Animal Opsin Evolution"/>
    <meta name="twitter:description" content="BackgroundBiologists are gaining an increased understanding of the genetic bases of phenotypic change during evolution. Nevertheless, the origins of phenotypes mediated by novel protein-protein interactions remain largely undocumented.Methodology/Principle FindingsHere we analyze the evolution of opsin visual pigment proteins from the genomes of early branching animals, including a new class of opsins from Cnidaria. We combine these data with existing knowledge of the molecular basis of opsin function in a rigorous phylogenetic framework. We identify adaptive amino acid substitutions in duplicated opsin genes that correlate with a diversification of physiological pathways mediated by different protein-protein interactions.Conclusions/SignificanceThis study documents how gene duplication events early in the history of animals followed by adaptive structural mutations increased organismal complexity by adding novel protein-protein interactions that underlie different physiological pathways. These pathways are central to vision and other photo-reactive phenotypes in most extant animals. Similar evolutionary processes may have been at work in generating other metazoan sensory systems and other physiological processes mediated by signal transduction."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001054.g007"/>

  <meta property="og:title" content="The Origins of Novel Protein Interactions during Animal Opsin Evolution" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8741'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3946'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8624&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001054">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001054" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001054&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Origins of Novel Protein Interactions during Animal Opsin Evolution
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David C. Plachetzki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Ecology, Evolution and Marine Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard M. Degnan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Integrative Biology, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Todd H. Oakley
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:oakley@lifesci.ucsb.edu">oakley@lifesci.ucsb.edu</a></p>

                <p>Affiliation:
                  Ecology, Evolution and Marine Biology, University of California at Santa Barbara, Santa Barbara, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 17, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001054</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001054-g001" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-g002" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-g003" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-g004" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-t001" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-g005" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-g006" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001054-g007" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001054">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001054" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Biologists are gaining an increased understanding of the genetic bases of phenotypic change during evolution. Nevertheless, the origins of phenotypes mediated by novel protein-protein interactions remain largely undocumented.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we analyze the evolution of opsin visual pigment proteins from the genomes of early branching animals, including a new class of opsins from Cnidaria. We combine these data with existing knowledge of the molecular basis of opsin function in a rigorous phylogenetic framework. We identify adaptive amino acid substitutions in duplicated opsin genes that correlate with a diversification of physiological pathways mediated by different protein-protein interactions.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study documents how gene duplication events early in the history of animals followed by adaptive structural mutations increased organismal complexity by adding novel protein-protein interactions that underlie different physiological pathways. These pathways are central to vision and other photo-reactive phenotypes in most extant animals. Similar evolutionary processes may have been at work in generating other metazoan sensory systems and other physiological processes mediated by signal transduction.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Plachetzki DC, Degnan BM, Oakley TH (2007) The Origins of Novel Protein Interactions during Animal Opsin Evolution. PLoS ONE 2(10):
          e1054.
            doi:10.1371/journal.pone.0001054</p><p><strong>Academic Editor: </strong>Berend Snel, Utrecht University, Netherlands</p><p><strong>Received:</strong> July 12, 2007; <strong>Accepted:</strong> September 17, 2007; <strong>Published:</strong> October 17, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This work was supported by US National Science Foundation grants DEB-0710406 to DCP and THO and DEB-0643840 to THO, and Australian Research Council grants to BMD. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Documenting the specific genetic changes driving phenotypic evolution is a fundamental goal of current biology. Genetic changes are known to modify phenotype during evolution by altering the interactions between a protein and its ecological or biochemical environment <a href="#pone.0001054-Yokoyama1">[1]</a>–<a href="#pone.0001054-Bridgham1">[4]</a>, by modulating existing protein-protein interactions <a href="#pone.0001054-Hoekstra1">[5]</a>, or by changing protein-DNA interactions through regulatory mutations <a href="#pone.0001054-Shapiro1">[6]</a>–<a href="#pone.0001054-Tishkoff1">[9]</a>. However, the specific genetic changes that give rise to the evolutionary origins of novel protein-protein interactions have rarely been documented in detail <a href="#pone.0001054-Lynch1">[10]</a>, but see <a href="#pone.0001054-Lynch2">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Animal phototransduction pathways offer great opportunity for elucidating the genetic basis for evolutionary novelty for a number of reasons. First, a diversity of presumably ancient phototransduction pathways exists in animals <a href="#pone.0001054-Arendt1">[12]</a>. Second, the composition of these cascades has been the subject of numerous functional biochemical studies. This is especially important because experimental demonstration that specific mutations cause phenotypic changes is often the most difficult aspect of a full documentation of the causal genetic changes driving phenotypic evolution <a href="#pone.0001054-Hoekstra2">[13]</a>. Third, the proteins of animal phototransduction are amenable to phylogenetic study.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Fundamental to animal phototransduction pathways are the opsin visual pigment proteins, which bind to light reactive chromophores. As members of the G protein-coupled-receptor (GPCR) family <a href="#pone.0001054-Shapiro1">[6]</a>, the various clades of animal opsins activate alternative G-proteins, resulting in three major phototransduction networks: ciliary, rhabdomeric, and G<sub>o</sub>–coupled <a href="#pone.0001054-Arendt2">[14]</a>–<a href="#pone.0001054-Raible1">[16]</a>. Ciliary opsins initiate signaling through binding of a G<sub>i/t</sub> α subunit of the G-protein <a href="#pone.0001054-Terakita1">[15]</a>, rhabdomeric opsins utilize a G<sub>q</sub> α subunit in signaling <a href="#pone.0001054-Hardie1">[17]</a> and class-specific G<sub>o</sub> α has been identified in the G<sub>o</sub>-coupled opsin signaling pathway <a href="#pone.0001054-Kojima1">[18]</a>–<a href="#pone.0001054-Koyanagi2">[20]</a>. Another class of opsins, including <span class="underline">R</span>etinal <span class="underline">G</span>-protein-coupled <span class="underline">R</span>eceptor (RGR) and retinochromes, probably do not signal through any G-protein. Instead they are involved in the re-activation of the light reactive chromophore.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here, we couple specific functional knowledge about opsin's role in signal transduction with new phylogenetic analyses. These analyses elucidate specific genetic changes that were likely involved in the origins of the different animal phototransduction networks, which mediate various light responses of animals. Our phylogenetically based analyses indicate a significant correlation between opsins' G-protein binding phenotypes and amino acid positions in the fourth cytoplasmic loop of opsin, especially positions homologous to 310 and 312 of bovine rhodopsin. Previous biochemical analyses demonstrate that these same amino acid positions are involved in G-protein binding function. Our additional analyses indicate that G-protein binding phenotypes likely diversified at the time of opsin gene duplication events before the origin of bilaterians, and the specific amino acid changes involved retain a pattern consistent with purifying selection.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Multiple opsin genes are present in cnidarians but absent in a demosponge</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Screens for opsin genes in the genome trace data from the cnidarians <em>Hydra magnipapillata</em> and <em>Nematostella vectensis</em> produced multiple unique opsins which all lack introns. Six of these cnidarian opsins were found in public Expressed Sequence Tag (EST) databases (<a href="#pone.0001054.s001">Table S1</a>). Consistent with opsin status, Hydra2 is expressed in the nerve net of <em>Hydra</em> based on in situ hybridization and includes sequence motif hallmarks of opsin (<a href="#pone-0001054-g001">figure 1</a>).</p>
<div class="figure" id="pone-0001054-g001"><div class="img"><a name="pone-0001054-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Sequence motifs and expression of cnidarian opsin in the nerve net of <em>Hydra magnipallata</em>.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Sequence alignment of 4<sup>th</sup> cytoplasmic loop region of animal opsins used in this study indicating the Lys 296 chromophore binding site (arrowhead) and the G protein-binding tripeptide (asterisks). (B) In situ hybridization with Hm2 cnidops probe. Asterisk denotes the hypostome. Opsin is expressed most strongly in a ring of sensory neurons that surround the mouth. Inset, oral view.</p>
<span>doi:10.1371/journal.pone.0001054.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>While multiple opsins were found in the cnidarian genomes, our screens for opsins in trace data from the poriferan <em>Amphimedon queenslandica</em> did not produce any putative opsins. These screens did produce several non-opsin, rhodopsin-class GPCR genes from <em>Amphimedon</em> (data not shown). We were also unable to obtain opsin sequences from the trace genome data of the placozoan <em>Trichoplax</em> or the choanoflagellate <em>Monosiga</em>. Animal (type II) opsins are also unknown from numerous fungal genomes <a href="#pone.0001054-Dunlap1">[21]</a>.</p>


<h4>A new class of cnidarian opsins helps resolve phylogenetic relationships</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In our discussion of unrooted phylogenies, we refrain from using the common terms “sister group” and “clade” in favor of the terms “adjacent group” and “clan” as the former terms imply a rooting hypothesis <em>a priori</em> <a href="#pone.0001054-Wilkinson1">[22]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Our phylogenetic analyses reveal a new clan of opsins known only in cnidarians, which we have named “cnidops”. Together, metazoan opsins form two major clans in unrooted analyses. One clan unites rhabdomeric and RGR/G<sub>o</sub>–coupled opsins in an adjacent group and the second consists of the newly identified cnidops family plus the ciliary opsins, which we here confirm by phylogenetic analysis to include a non-bilaterian representative (Nematostella4; <a href="#pone-0001054-g002">figure 2</a>).</p>
<div class="figure" id="pone-0001054-g002"><div class="img"><a name="pone-0001054-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Unrooted metazoan-wide phylogeny of opsins, new cnidarian genes in bold, branches proportional to substitutions per site.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Circles at nodes indicate Bayesian posterior probabilities (White = 1.0, Red&gt;0.90, Blue&gt;0.80, Green&gt;0.70, Yellow&gt;0.60, Black&gt;0.50). cil = ciliary, rh = rhabdomeric.</p>
<span>doi:10.1371/journal.pone.0001054.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Without non-opsin outgroups, our analyses of metazoan opsin phylogeny yielded well supported topologies. In order to root our tree, we used a combination of likelihood comparisons, reconciled tree analyses and parametric bootstrapping (summarized in <a href="#pone-0001054-g003">figure 3</a> and described in <a href="#s4">Methods</a>). The majority of available evidence indicates that opsin phylogeny is best rooted between the ciliary and non-ciliary clades (<a href="#pone-0001054-g004">figure 4A</a>).</p>
<div class="figure" id="pone-0001054-g003"><div class="img"><a name="pone-0001054-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Summary of analyses to determine and root opsin phylogeny.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Illustrated is each possible unrooted and rooted hypothesis, assuming monophyly of four major opsin clades. Trees in (A) 1–3 correspond to possible unrooted topologyies and those in (B) 1–15 represent all possible rooted trees, see <a href="#pone-0001054-t001">Table 1</a>. Orange X indicates that tree had significantly lower likelihood in opsin-only dataset. Red X indicates tree had significantly lower likelihood in opsin+outgroup dataset. Yellow X indicates that tree implies additional gene duplication/loss events compared to minimum (tree 1 is minimum with 2 duplications 0 losses implied). Grey X indicates tree with cnidops as earliest branching opsin group–a result inferred in parametric bootstrap analyses, which incorrectly grouped cnidops+outgroup opsins because of long branch-attraction (see supplementary <a href="#s4">methods</a>). Cil = ciliary; Rh = rhabdomeric; RG = Go/RGR.</p>
<span>doi:10.1371/journal.pone.0001054.g003</span></div><div class="figure" id="pone-0001054-g004"><div class="img"><a name="pone-0001054-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Three hypotheses for metazoan opsin relationships.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>These are statistically indistinguishable when considering only the likelihood of observed amino acid sequences. (A) Our preferred phylogenetic hypothesis where ciliary opsins are an outgroup to other opsin clades and cnidops is the sister to the rhabdomeric+G<sub>o</sub> clade. This hypothesis minimizes the number of gene duplication and loss events required to explain the evolutionary history of metazoan opsins and is consistent with morphological data <a href="#pone.0001054-Eakin1">[33]</a>. (B and C) Phylogenetic hypotheses in which cnidops represents the outgroup to other opsins. Given our finding of a ciliary opsin in <em>Nematostella</em>, these hypotheses require the additional loss of ciliary opsin in the <em>Hydra</em> lineage and an additional loss of cnidops in bilaterian animals.</p>
<span>doi:10.1371/journal.pone.0001054.g004</span></div><div class="figure" id="pone-0001054-t001"><div class="img"><a name="pone-0001054-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Likelihood comparisons for alternative hypotheses of opsin evolution</span></strong></p><span>doi:10.1371/journal.pone.0001054.t001</span></div>

<h4>Timing of origin of major opsin clades</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Using our rooted opsin phylogeny, reconciled tree analyses (RTA) provide new information on the timing of origin of the various opsin clades (<a href="#pone-0001054-g005">figure 5</a>). RTA indicates that the ciliary and cnidops lineages originated by gene duplication at or before the cnidarian-bilaterian common ancestor (Eumetazoa), approximately 600 million years ago <a href="#pone.0001054-Peterson1">[23]</a>, <a href="#pone.0001054-Peterson2">[24]</a>.</p>
<div class="figure" id="pone-0001054-g005"><div class="img"><a name="pone-0001054-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Our preferred hypothesis for opsin phylogeny (<a href="#pone-0001054-g004">figure 4A</a>) reconciled to a conservative view of metazoan phylogeny.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Black circles indicate gene duplication events. Tan ovals indicate the opsin complement at key nodes in metazoan phylogeny. By this hypothesis, both ciliary and cnidops opsins were present in the eumetazoan ancestor of Cnidaria+Bilateria while rhabdomeric and G<sub>o</sub> opsins evolved by gene duplication prior to the evolution of bilaterian animals, but not before.</p>
<span>doi:10.1371/journal.pone.0001054.g005</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Of the two ancient metazoan opsin lineages, one is represented in extant taxa by the newly described cnidops opsins of cnidaria, RGR/G<sub>o</sub> and rhabdomeric opsins. Our results concur with previous minima for the origins of RGR/G<sub>o</sub> and rhabdomeric opsins at the protostome-deuterostome common ancestor (Urbilateria) <a href="#pone.0001054-Arendt2">[14]</a>, <a href="#pone.0001054-Arendt3">[25]</a> and provide a new maximum for the origin of these clades by a gene duplication event younger than the eumetazoan ancestor (<a href="#pone-0001054-g005">figure 5</a>). A second ancient opsin lineage survives as the ciliary opsins and includes one cnidarian opsin (Nematostella4) and those of the vertebrate rods, cones, pineal and parapineal, and of invertebrate extra-ocular cells <a href="#pone.0001054-Arendt3">[25]</a>, <a href="#pone.0001054-Velarde1">[26]</a>. Despite finding the <em>Nematostella</em> gene, we did not find any <em>Hydra</em> genes from this ciliary opsin clade.</p>


<h4>The early evolution of G protein binding partners</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our phylogeny provides a framework for understanding key transitions in the evolution of animal phototransduction pathways. Opsins interact with their corresponding G proteins, in part through binding at conserved sequence motifs located in opsin's fourth cytoplasmic loop <a href="#pone.0001054-Marin1">[27]</a>. We obtained clear reconstructed ancestral states for most of the residues in a conserved tri-peptide motif for the ciliary, rhabdomeric and RGR/G<sub>o</sub> nodes. For the most part, the remainder of the residues in the fourth cytoplasmic loop can be unequivocally reconstructed to the level of Dayhoff classes (i.e., C/HRK/FYW/DENQ/LIVM/GATSP) <a href="#pone.0001054-Hrdy1">[28]</a> (<a href="#pone-0001054-g006">figure 6</a>). In addition, many of the residues of this region significantly co-vary with G protein α subunit interaction on our phylogeny. Two of the highest scoring residues in the co-variation analysis, residues 310 and 312 corresponding to positions 1 and 3 of the G protein-binding tripeptide motif <a href="#pone.0001054-Marin1">[27]</a>, possessed a signature of selection using algorithms in DIVERGE2 <a href="#pone.0001054-Zheng1">[29]</a> (<a href="#pone-0001054-g007">figure 7</a>).</p>
<div class="figure" id="pone-0001054-g006"><div class="img"><a name="pone-0001054-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Ancestral state reconstruction of G protein-binding interactions for each metazoan opsin-mediated phototransduction cascade obtained by simulated mutational mapping <a href="#pone.0001054-Bollback1">[64]</a> (see <a href="#s4">methods</a>).</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>For each class of opsin, the P value of the reconstructed ancestral G α interactions is represented in pie graphs. Ancestral G protein interactions in phototransduction cascades mediated by ciliary, rhabdomeric and G<sub>o</sub> opsins can be significantly resolved (P&gt;0.95) but the ancestral states of the rhabdomeric+G<sub>o</sub>, and cnidops clades are equivocal. ML state reconstructions shown for each node as colored branches. Red, G<sub>i/t</sub>; Blue, G<sub>q</sub>; Green, G<sub>o</sub>; Black, no G protein interaction (as is the case for RGR/Retinochrome opsins); Grey, equivocal reconstruction from ML. Reconstructed ancestral amino acid motifs of the 4th cytoplasmic loop region of opsin are shown along branches in logos. Maximum vertical height scales to P  = 1.0. We obtained clear reconstructed states for most of the residues in a conserved tripeptide motif (residues 310, 311 and 312, horizontal bar) for the ciliary, rhabdomeric and G<sub>o</sub> /RGR nodes. For the most part, the remainder of the residues in can be unequivocally reconstructed to the level of Dayhoff classes. B = HRK, X = LIVM, J = GATSP, Z = DENQ.</p>
<span>doi:10.1371/journal.pone.0001054.g006</span></div><div class="figure" id="pone-0001054-g007"><div class="img"><a name="pone-0001054-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001054" data-uri="info:doi/10.1371/journal.pone.0001054.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001054.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001054.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001054.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001054.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Summary of co-variation analyses and tests for selection.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Previously described G protein binding states, together with amino acid sequences of the 4<sup>th</sup> cytoplasmic loop regions from our opsin dataset, were used to test the hypothesis that the two character sets co-vary across our opsin phylogeny. Co-variation was assessed using mutual information content (MIC). Shown here is a representation of these results. Predictive P-values (Bottom axis) based on MIC for co-variation between each residue in the 4<sup>th</sup> cytoplasimic loop and their respective G protein interactions (Right axis) are shown. These analyses were conducted using Bayesian mutational simulation mapping in SIMMAP <a href="#pone.0001054-Bollback1">[64]</a>. We also tested possibility that these residues had evolved under a selective regime using the criterion of site specific rate heterogeneity as implemented in DIVERGE2 <a href="#pone.0001054-Zheng1">[29]</a>. The two highest scoring residues from co-variation analyses (310 and 312) also retain the signature of selection (asterisk). See <a href="#pone.0001054.s001">table S1</a> for citations. <em>P</em> = predictive P-value, <em>M</em> = the M statistic given by MIC.</p>
<span>doi:10.1371/journal.pone.0001054.g007</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We also explored the character history of G protein binding interactions by reconstructing their ancestral states across our phylogeny (<a href="#pone-0001054-g006">figure 6</a>). While we cannot conclusively resolve the G protein interaction in the ancestor of rhabdomeric+G<sub>o</sub> opsins or the ancestor of ciliary+rhabdomeric+G<sub>o </sub>clades, our reconstructed ancestral G protein interaction states for ciliary, rhabdomeric and G<sub>o</sub> are highly significant (P&gt;0.95 for each) and are congruent with previous empirical biochemical studies <a href="#pone.0001054-Hardie1">[17]</a>, <a href="#pone.0001054-Koyanagi1">[19]</a>, <a href="#pone.0001054-Su1">[30]</a> (<a href="#pone-0001054-g006">figure 6</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our findings provide several new insights into the evolution of animal phototransduction cascades. First, we identify a new class of opsins that appears to be unique to cnidarians. Second, we find no evidence of any opsins in genomes of a demosponge, a parazoan, a choanoflagellate or fungi. We conclude that opsin-mediated phototransduction cascades originated early in eumetazoan evolution, after this lineage diverged from the demosponge lineage. The rhabdomeric and G<sub>o</sub>-coupled cascades appear to be bilaterian innovations. Third, the origins of novel phototransduction cascades, as defined by the evolution of alternative opsin-G protein interactions, co-occur on our phylogeny with the gene duplications that have given rise to the major opsin classes. Fourth, the specific mutational events that have lead to the functional diversification of the major opsin lineages are suggested by co-variation analyses and tests for selection. These results demonstrate how gene duplications, together with structural mutations that lead to novel protein-protein interactions, can contribute to the evolution of novel physiological traits. Each of these points is discussed further below.</p>

<h4>The timing and relationships of animal opsins</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The inter-relationships of animal opsins reported here are based on thorough phylogenetic analyses that included standard phylogenetic tree estimation, comprehensive likelihood comparisons, RTA, and parametric bootstrapping (<a href="#pone-0001054-g003">figure 3</a>). Taken together, the majority of evidence favors the hypothesis depicted in <a href="#pone-0001054-g004">figure 4A</a>, although the trees in <a href="#pone-0001054-g004">figures 4B and C</a> remain viable alternative hypotheses. However, because these latter hypotheses require additional gene duplication and loss events to explain the rooted topology, and because parametric bootstrapping simulations indicate that long branch lengths can contribute to the erroneous resolution of specifically these topologies, we favor the former hypothesis (<a href="#pone-0001054-g004">figure 4A</a>). Given present sampling and the assumption of monophyly of major opsin clades, any other phylogenetic hypotheses are significantly rejected by the data. Even if <a href="#pone-0001054-g004">figure 4A</a> does not represent the true history of opsin, the two viable alternative hypotheses (<a href="#pone-0001054-g004">figures 4B and C</a>) do not drastically alter our interpretation of the timing of origins of animal opsin in RTA analysis. Irrespective of which hypothesis we consider, animal opsins originate at the same node (Eumetazoa). If we assume the hypothesis depicted in <a href="#pone-0001054-g004">figure 4B</a> our conclusion that RGR/G<sub>o</sub>-coupled and rhabdomeric opsins are bilaterian synapomorphies also remains intact, but requires an additional loss of cnidops prior to Bilateria. The latter is also true of the topology shown in <a href="#pone-0001054-g004">figure 4C</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>A maximum date for the origin of animal opsins (when a GPCR first bound a light reactive chromophore) is equivocal because we did not obtain any opsins in the genome sequence of an earlier branching animal, the demosponge <em>Amphimedon queenslandica</em>, or from any non-animal genomes. While the larvae of <em>Amphimedon</em> are capable of phototaxis, studies of spectral sensitivity have suggested the activities of a flavin-based photoreceptor in this behavior <a href="#pone.0001054-Leys1">[31]</a>, as opposed to retinal as used by opsins. Further, we uncovered no animal (type II) opsins from bioinformatic screens of the placozoan <em>Trichoplax</em> or the choanoflagelate <em>Monosiga</em> genomes and no true homolog of type II opsins have been described outside of Metazoa <a href="#pone.0001054-Spudich1">[32]</a>. While we cannot rule out an earlier opsin origin coupled with loss in lineages leading to the species with genome sequences, we strongly favor the hypothesis that animal opsins were present in the eumetazoan ancestor of cnidarians and bilaterians, but not at earlier nodes in eukaryote phylogeny. Since porifera, <em>Monosiga</em>, and fungi seem to lack opsins, yet possess rhodopsin-class GPCR proteins without the Lysine residue to bind a light reactive chromophore, one possibility is that a rhodopsin-class GPCR gained the ability to bind a chromophore by substitution to a Lysine residue during animal evolution.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Our results contrast with previous studies on opsin relationships that placed ciliary and G<sub>o</sub>-coupled opsins as sister groups <a href="#pone.0001054-Arendt2">[14]</a>, or adjacent groups <a href="#pone.0001054-Terakita1">[15]</a>, and those that placed ciliary and rhabdomeric opsins as sister groups <a href="#pone.0001054-Arendt1">[12]</a>. However, in lacking data from non-bilaterian animals, these previous works were able to include only three of the four animal opsin lineages discussed here. The new clade of opsins from the Cnidarian, “cnidops”, has aided in the resolution of the overall opsin phylogeny, especially because non-opsin outgroups seem to destabilize ingroup topology.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The presence of an opsin from the cnidarian <em>Nematostella</em> that in our analyses groups within the ciliary clade has been foreshadowed by previous morphological descriptions of ciliary photoreceptors <a href="#pone.0001054-Eakin1">[33]</a>, <a href="#pone.0001054-Eakin2">[34]</a>, but no previous opsin data from the group have confirmed this hypothesis. Our ability to find only cnidops and not ciliary opsins in <em>Hydra</em> probably indicates that ciliary clade opsins were lost during this species' evolutionary history.</p>


<h4>The genetic basis for opsin-G protein interactions</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Multiple lines of evidence support the hypothesis that amino acid substitutions in the 4th cytoplasmic loop of duplicated opsins were involved in the origins of novel opsin-G protein interactions. First, specific amino acid states significantly co-vary with G-protein activation phenotypes during opsin phylogeny (<a href="#pone-0001054-g007">figure 7</a>). Because we used a Bayesian approach that estimated the posterior probability of all possible trees, the estimated correlation accounts for uncertainty in opsin phylogeny. The highest p-values in these correlation analyses correspond to residues formerly demonstrated by site-directed mutagenesis to mediate opsin-G protein interaction in ciliary opsins (i.e., residues 310 and 312 <a href="#pone.0001054-Marin1">[27]</a>).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>These amino acid changes retain the signature of selection, as given by site-specific shifts in evolutionary rates, a statistical test that uncovers significant conservation within gene clades compared to diversification between gene clades <a href="#pone.0001054-Zheng1">[29]</a> (<a href="#pone-0001054-g007">figure 7</a>). Although this pattern cannot be used to determine if the mutations were originally fixed by natural selection, conservation after an ancient diversification indicates a long period of stasis, consistent with purifying selection on adaptive, alternative amino acid states. The combination of correlation, functional tests, and a pattern consistent with purifying selection strongly suggests that substitutions in residues homologous to bovine opsin 310 and 312 contributed to the origin of novel phototransduction pathways with alternative G-protein binding partners.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The pattern of early divergence followed by conservation is also evident in our estimates of ancestral opsin-G protein binding phenotypes. Using available data, we were able to unequivocally resolve the ancestral G-protein usage phenotypes for the ciliary, rhabdomeric and RGR/G<sub>o</sub>-coupled opsin clades (<a href="#pone-0001054-g006">figure 6</a>). That we can resolve the G protein binding specificities at the base of each of these clades, but not at deeper nodes, suggests an early diversification of G protein binding specificities, coupled with relative stasis of protein interactions once the major opsin clades were established.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Because we find that opsins were present in extant eumetazoans but not in animals representing earlier branching lineages (i.e. demosponges), the evolution of novel phototransduction cascades in animals probably proceeded by evolving specialized protein-protein interactions between new opsin paralogs and existing G protein signaling intermediaries, an evolutionary process previously described as molecular exploitation <a href="#pone.0001054-Bridgham1">[4]</a>. Each of the G α subunit paralogs known to interact with opsins in bilaterians has also been described from plants, fungi, protists, cnidarians and sponges indicating a long pre-animal history for the different G-proteins <a href="#pone.0001054-Suga1">[35]</a>, <a href="#pone.0001054-OHalloran1">[36]</a>.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>While our inclusion of new data from the Cnidaria has greatly enhanced our ability to reconstruct opsin phylogeny, functional or phenotypic data from the newly reported cnidops clade do not exist. Further studies on the biochemistry of signaling by cnidops are required if we are to better understand the history of opsin-G protein interactions in this important family of opsins. Despite this missing data, P-values for our ancestral state reconstructions for the remaining opsin classes suggest that our results are robust.</p>


<h4>Implications for early animal sensory systems</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Our finding of an adaptive, early partitioning of opsin-mediated signaling into discrete functional classes in the course of their evolution provides insight into the organismal properties of the ancestor of cnidarian and bilaterian animals. The activities of ciliary, rhabdomeric and G<sub>o</sub>-coupled opsins are known to exist in close proximity <a href="#pone.0001054-Gomez1">[37]</a>, if not the same cell <a href="#pone.0001054-Su1">[30]</a>, in numerous taxa. The early, adaptive evolution of the different photosensory pathways suggested by our analyses may be an indication of the functional necessity to transduse these signaling cascades through non-overlapping channels in the eumetazoan ancestor, an animal that by our analysis possessed ancestral ciliary and ancestral cnidops+rhabdomeric+RGR/G<sub>o</sub>-clade opsins. Such early functional requirements could have included the need to delineate light information used for vision from that used to entrain the circadian clock cycle <a href="#pone.0001054-Gehring1">[38]</a> or to differentiate between light of differing wavelengths, two core processes accomplished by specific cell types in bilaterian animals <a href="#pone.0001054-Arendt1">[12]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The proposed relationship between gene duplication and biological complexity has been invoked and re-invoked at various points throughout the past century of biological investigation <a href="#pone.0001054-Taylor1">[39]</a>, and was most famously articulated by Ohno <a href="#pone.0001054-Ohno1">[40]</a>. Despite the great level of interest in models for the evolution of complexity based on gene duplication, data demonstrating how gene duplication can be implicated in the origins of novel protein-protein interactions are not common. Here we begin to fill this gap and show that a group of familiar signaling pathways, animal phototransduction cascades, provides a useful model for the study of the origins of novel phenotypes.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data mining</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Publicly available trace genome sequence data from <em>Hydra magnipapillata</em>, <em>Nematostella vectensis</em> and <em>Amphimedon queenslandica</em> (<a href="http://www.ncbi.nlm.nih.gov/Traces">http://www.ncbi.nlm.nih.gov/Traces</a>) were subjected to tblastn <a href="#pone.0001054-Altschul1">[41]</a> searches using a wide range of homologous opsin proteins from bilaterian animals (<a href="#pone.0001054.s001">Table S1</a>). We implemented an in-house assembly pipeline, where BLAST hits were extended to their largest possible contigs under stringent parameters, using only clean trace data as determined using PHRAP <a href="#pone.0001054-Ewing1">[42]</a> and an 85% identity cut off for joining trace fragments. Genes were predicted from genomic contigs using GenomeScan <a href="#pone.0001054-Yeh1">[43]</a>. A combination of BLAST searches, phylogenetic analyses and sequence motif analyses were used to establish the authenticity of opsin sequences recovered from genome analyses. In particular, the residue homologous to the bovine lysine 296, which binds a light reactive chromophore <a href="#pone.0001054-Palczewski1">[44]</a>, has been used to validate opsin identity (<a href="#pone-0001054-g001">figure 1</a>). We corroborated the existence of our predicted proteins by using EST databases at either NCBI (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>), for <em>Hydra</em>, or the Joint Genome Institute, for <em>Nematostella</em> (<a href="http://www.jgi.doe.gov/">http://www.jgi.doe.gov/</a>). Homologous proteins and outgroup data from bilaterian animals used in phylogenetic analyses were also obtained from NCBI (<a href="#pone.0001054.s001">Table S1</a>).</p>


<h4>Animal culture and in situ hybridization</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p><em>Hydra magnipapillata</em> (UC Irvine strain) were reared in the laboratory under standard conditions at 18°C. The expression of one opsin, HM2, was investigated in <em>Hydra magnipapillata</em> using in situ hybridization following Grens et al. <a href="#pone.0001054-Grens1">[45]</a>. The following modifications were made: A pCRII vector (Invitrogen) containing HM2 opsin cDNA (~750 bp) was used as template for probe synthesis. DIG-labeled probes were synthesized using T3 and T7 polymerase (Roche). Both sense and anti-sense probes (0.3 µg/ml) were hybridized at 55°C for 48 to 60 hours. Sense probes produced no detectable signal.</p>


<h4>Phylogenetic approaches</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For phylogenetic analyses, only the 7-transmembrane region including intervening inter- and extra-cellular domains was included (330 amino acids), as it was difficult to ascertain homology of N- and C- termini due to sequence length variation and lack of conservation across genes. Taxon/gene selection was done in a manner that enhanced both the taxonomic representation across the Metazoa and the inclusion of well-studied subfamilies (i.e., arthropod Rh opsins and vertebrate visual opsins), but also allowed for thorough computational analyses. Protein sequences were aligned using T-COFFEE under default parameters <a href="#pone.0001054-Notredame1">[46]</a> and alignment manipulations were done using Seaview <a href="#pone.0001054-Galtier1">[47]</a>. Phylogenetic analyses were conducted using unweighted Maximum Parsimony (MP) implemented in PAUP* 4.0b10 <a href="#pone.0001054-Swofford1">[48]</a>, Maximum Likelihood (ML) implemented in PHYML v2.4.4 <a href="#pone.0001054-Guindon1">[49]</a>, and Bayesian Markov Chain Monte Carlo (BMCMC) implemented in MrBayes 3.1 <a href="#pone.0001054-Ronquist1">[50]</a>. Support for internal nodes was assessed with 1000 bootstrap replicates for MP and ML, and posterior probability for BMCMC analyses. ML and BMCMC approaches assumed best-fit models of protein evolution, as determined using ProtTest <a href="#pone.0001054-Abascal1">[51]</a>. For ML tree calculations, both the proportion of invariant sites and the α parameter of the gamma distribution were estimated and both tree topology and branch lengths were optimized in PHYML. Bayesian analyses were conducted for 20 million generations using default priors and heating parameters for each of four chains. For the monitoring of progress in BMCMC runs, we assumed convergence of the Markov Chains when standard deviation of split frequencies (SDSF) fell below 0.01, indicating that the two independent runs resulted in similar phylogenies <a href="#pone.0001054-Ronquist1">[50]</a>. Following the BMCMC runs, burnin was assessed using Tracer <a href="#pone.0001054-Rambaut1">[52]</a> and by the SDSF of the two independent BMCMC runs. For un-rooted analyses, the initial 11 million generations with an SDSF of 0.0015 or greater were discarded. For rooted analyses the first 12 million generations with an SDSF of 0.0024 or greater were removed prior to consensus tree calculation.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The evolutionary relationships between members of the larger GPCR class of proteins remain poorly understood. Because of this, our outgroup selection in rooted analyses was empirical. We selected the shortest-branched sequences from a pool of rhodopsin class GPCRs <a href="#pone.0001054-Spudich1">[32]</a> from a variety of taxa (<a href="#pone.0001054.s001">Table S1</a>) after non-bootstrapped ML and shorter (i.e., 3-5 million steps) BMCMC analyses.</p>


<h4>Tests of phylogenetic hypotheses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In order to address the lack of resolution among the major opsin clades encountered in phylogenetic analyses that included outgroups (<a href="#pone.0001054.s003">figure S2</a>), we tested the significance of a wide range of competing phylogenetic hypotheses for metazoan opsin. First, we assumed monophyly of each of the four major opsin clades (cnidarian opsin, ciliary, RGR/G<sub>o</sub> and rhabdomeric). This assumption is supported by the results of unrooted MP, ML and BMCMC analyses (<a href="#pone.0001054.s003">figure S2</a>) and by previous studies on opsin phylogeny <a href="#pone.0001054-Arendt1">[12]</a>, <a href="#pone.0001054-Terakita1">[15]</a>. Our topology contains four opsin clades that can be rearranged in 15 possible rooted binary trees. Second, we calculated the likelihood for each possible tree using the “resolve multifurcations” function in TREEFINDER <a href="#pone.0001054-Jobb1">[53]</a>. Finally, we used CODEML, as included in PAML <a href="#pone.0001054-Yang1">[54]</a>, to assess the significance of each of these trees. These results were further analyzed using CONSEL <a href="#pone.0001054-Shimodaira1">[55]</a>. A similar approach was used to assess alternative topologies in unrooted trees where having four major clades involves three possible unrooted topologies (<a href="#pone-0001054-g004">figure 4</a>).</p>


<h4>Parametric bootstrapping</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Given the ambiguous results obtained when including non-opsin outgroups, we suspected that long branch-attraction (LBA) artifacts <a href="#pone.0001054-Felsenstein1">[56]</a>, where the cnidops sequences were pulled to the base of the tree by long branch outgroups, were confounding our analyses. We tested this hypothesis by using Huelsenbeck's <a href="#pone.0001054-Huelsenbeck1">[57]</a> method of parametric bootstrapping. Here, a tree including a non-opsin outgroup was constrained to reflect the best topology consistent with Tree 1B in <a href="#pone-0001054-g003">figure 3</a>. Assuming this tree and the best-fit model of opsin molecular evolution from prot-test analysis <a href="#pone.0001054-Abascal1">[51]</a> , we simulated 100 data sets using SeqGen <a href="#pone.0001054-Rambaut2">[58]</a>. Each of these simulated data sets was subjected to ML and MP analyses. Since the data were simulated, we expected to recover the assumed phylogeny unless the topology/model combination is sensitive to artifacts like long branch-attraction <a href="#pone.0001054-Huelsenbeck1">[57]</a>. For ML analyses we found that the simulated cnidarian genes are incorrectly attracted to the base of the tree in 45% of replicates, whereas the correct, simulated topology was recovered in only 35% of the replicates. In MP analyses, the cnidarian opsin genes were incorrectly attracted to the outgroup in 49% of the replicates, while the simulated phylogeny was again recovered in only 35% of cases. These simulations show that analyses including opsin outgroups are sensitive to artifacts that induce erroneous topology estimates. Our results are similar to those reported for analyses of strepsipteran relationships, a well-known case of LBA <a href="#pone.0001054-Huelsenbeck1">[57]</a>.</p>


<h4>Reconciled tree analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The number of gene duplication and loss events that are implied by a given rooting hypothesis can be used to assess the chances that a given hypothesis is tenable <a href="#pone.0001054-Thornton1">[59]</a>. We compared the number of duplications and losses implied by each possible root position of our unrooted phylogeny using NOTUNG <a href="#pone.0001054-Chen1">[60]</a>, <a href="#pone.0001054-Durand1">[61]</a>. For these analyses we assumed a conservative species-level phylogeny for the major taxa included in our analyses: Hydrozoa, Anthozoa, Cephalochordata, Urochordata, Vertebrata, Annelida, Mollusca, and Arthropoda <a href="#pone.0001054-Telford1">[62]</a> .</p>


<h4>Character mapping and tests of co-variance</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>We scored G-protein interaction phenotypes for the opsins in our phylogenetic analysis as a discrete state character. Although these phenotypes are unknown for many opsins, especially those from non-model organisms, our analysis includes interaction data for 48% of the opsin sequences represented on our phylogeny (<a href="#pone-0001054-g006">figures 6</a>, <a href="#pone.0001054.s002">S1</a>).</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>Ancestral state reconstructions and tests for correlated character evolution were conducted using Bayesian mutational mapping <a href="#pone.0001054-Nielsen1">[63]</a> as implemented in SIMMAP <a href="#pone.0001054-Bollback1">[64]</a>. Reconstructions were integrated over 18,000 trees that were sampled from our unrooted BMCMC analyses. Ancestral state reconstructions were conducted with 10 realizations sampled from the prior and 10 realizations sampled for each tree. We set an equal prior on the bias parameter but did not abstract a prior for the rate parameter, instead using branch lengths from BMCMC trees as relative rates. As there is no <em>a priori</em> reason to assume different prior parameters, we chose this model, as it is the least parameterized model for morphological evolution possible in SIMMAP <a href="#pone.0001054-Bollback1">[64]</a>. We also tested a range of alternative prior settings and obtained similar P-values in these analyses (results not shown). Tests for selection were conducted in DIVERGE2 <a href="#pone.0001054-Zheng1">[29]</a> using our amino acid alignment (see supplemental materials), rooted phylogeny and the structure of bovine opsin previously calculated by x-ray crystallography <a href="#pone.0001054-Li1">[65]</a>. Our estimate for site specific rate shift (functional divergence) between ciliary and cnidops+RGR/G<sub>o</sub>+Rhabdomeric was significantly larger than 0: θ<sub>1</sub> = 5.82±0.05. Site-specific posterior probabilities for residues 310 and 312 were 0.45 and 0.37 respectively.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001054"></div><a name="pone.0001054.s001" id="pone.0001054.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001054.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Sequences used in opsin phylogenetic analyses</p>
<p class="siDoi">doi:10.1371/journal.pone.0001054.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0001054.s002" id="pone.0001054.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001054.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Phylogenetic analyses of opsins without an outgroup A. ML analyses were conducted assuming the WAG +I+G+F model. B. Unweighted MP. Node numbers represent bootstrap proportions out of 1000 replicates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001054.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.31 MB TIF)</p>
<a name="pone.0001054.s003" id="pone.0001054.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001054.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Results of analyses when including non-opsin outgroup. A. BMCMC, B. MP, and C. ML. BMCMC and ML analyses were conducted under the WAG+I+G+F model. Similar analyses with other outgroups produced qualitatively similar results (not shown)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001054.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(6.19 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank members of the Oakley lab for comments and ideas and H. Bode for providing <em>Hydra</em>. Thanks also to T. Cronin, M. Porter and M. S. Pankey for valuable comments on an earlier draft.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TO DP. Performed the experiments: TO DP. Analyzed the data: TO DP. Contributed reagents/materials/analysis tools: TO BD DP. Wrote the paper: TO BD DP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001054-Yokoyama1" id="pone.0001054-Yokoyama1"></a>Yokoyama S (2000) Molecular evolution of vertebrate visual pigments. Prog Retin Eye Res  19: 385–419.  <ul class="find" data-citedArticleID="1006882" data-doi="10.1016/s1350-9462(00)00002-1"><li><a href="http://dx.doi.org/10.1016/s1350-9462(00)00002-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evolution+of+vertebrate+visual+pigments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evolution+of+vertebrate+visual+pigments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001054-Spaethe1" id="pone.0001054-Spaethe1"></a>Spaethe J, Briscoe AD (2005) Molecular characterization and expression of the UV opsin in bumblebees: three ommatidial subtypes in the retina and a new photoreceptor organ in the lamina. J Exp Biol  208: 2347–2361.  <ul class="find" data-citedArticleID="1006852" data-doi="10.1242/jeb.01634"><li><a href="http://dx.doi.org/10.1242/jeb.01634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+and+expression+of+the+UV+opsin+in+bumblebees%3A+three+ommatidial+subtypes+in+the+retina+and+a+new+photoreceptor+organ+in+the+lamina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+and+expression+of+the+UV+opsin+in+bumblebees%3A+three+ommatidial+subtypes+in+the+retina+and+a+new+photoreceptor+organ+in+the+lamina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001054-Zhang1" id="pone.0001054-Zhang1"></a>Zhang J, Zhang YP, Rosenberg HF (2002) Adaptive evolution of a duplicated pancreatic ribonuclease gene in a leaf-eating monkey. Nat Genet  30: 411–415.  <ul class="find" data-citedArticleID="1006884" data-doi="10.1038/ng852"><li><a href="http://dx.doi.org/10.1038/ng852" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+a+duplicated+pancreatic+ribonuclease+gene+in+a+leaf-eating+monkey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+a+duplicated+pancreatic+ribonuclease+gene+in+a+leaf-eating+monkey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001054-Bridgham1" id="pone.0001054-Bridgham1"></a>Bridgham JT, Carroll SM, Thornton JW (2006) Evolution of hormone-receptor complexity by molecular exploitation. Science  312: 97–101.  <ul class="find" data-citedArticleID="1006766" data-doi="10.1126/science.1123348"><li><a href="http://dx.doi.org/10.1126/science.1123348" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+hormone-receptor+complexity+by+molecular+exploitation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+hormone-receptor+complexity+by+molecular+exploitation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001054-Hoekstra1" id="pone.0001054-Hoekstra1"></a>Hoekstra HE, Hirschmann RJ, Bundey RA, Insel PA, Crossland JP (2006) A single amino acid mutation contributes to adaptive beach mouse color pattern. Science  313: 101–104.  <ul class="find" data-citedArticleID="1006794" data-doi="10.1126/science.1126121"><li><a href="http://dx.doi.org/10.1126/science.1126121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+amino+acid+mutation+contributes+to+adaptive+beach+mouse+color+pattern." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+amino+acid+mutation+contributes+to+adaptive+beach+mouse+color+pattern.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001054-Shapiro1" id="pone.0001054-Shapiro1"></a>Shapiro MD, Marks ME, Peichel CL, Blackman BK, Nereng KS, et al.  (2004) Genetic and developmental basis of evolutionary pelvic reduction in threespine sticklebacks. Nature  428: 717–723.  <ul class="find" data-citedArticleID="1006846" data-doi="10.1038/nature02415"><li><a href="http://dx.doi.org/10.1038/nature02415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+and+developmental+basis+of+evolutionary+pelvic+reduction+in+threespine+sticklebacks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+and+developmental+basis+of+evolutionary+pelvic+reduction+in+threespine+sticklebacks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001054-BachnerMelman1" id="pone.0001054-BachnerMelman1"></a>Bachner-Melman R, Dina C, Zohar AH, Constantini N, Lerer E, et al.  (2005) AVPR1a and SLC6A4 gene polymorphisms are associated with creative dance performance. PLoS Genet  1: e42.  <ul class="find" data-citedArticleID="1006762" data-doi="10.1371/journal.pgen.0010042"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AVPR1a+and+SLC6A4+gene+polymorphisms+are+associated+with+creative+dance+performance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AVPR1a+and+SLC6A4+gene+polymorphisms+are+associated+with+creative+dance+performance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001054-Rockman1" id="pone.0001054-Rockman1"></a>Rockman MV, Hahn MW, Soranzo N, Zimprich F, Goldstein DB, et al.  (2005) Ancient and recent positive selection transformed opioid cis-regulation in humans. PLoS Biol  3: e387.  <ul class="find" data-citedArticleID="1006842" data-doi="10.1371/journal.pbio.0030387"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+and+recent+positive+selection+transformed+opioid+cis-regulation+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+and+recent+positive+selection+transformed+opioid+cis-regulation+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001054-Tishkoff1" id="pone.0001054-Tishkoff1"></a>Tishkoff SA, Reed FA, Ranciaro A, Voight BF, Babbitt CC, et al.  (2007) Convergent adaptation of human lactase persistence in Africa and Europe. Nat Genet  39: 31–40.  <ul class="find" data-citedArticleID="1006870" data-doi="10.1038/ng1946"><li><a href="http://dx.doi.org/10.1038/ng1946" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Convergent+adaptation+of+human+lactase+persistence+in+Africa+and+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Convergent+adaptation+of+human+lactase+persistence+in+Africa+and+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001054-Lynch1" id="pone.0001054-Lynch1"></a>Lynch M (2007) The evolution of genetic networks by non-adaptive processes. Nat Rev Genet  8: 803–813.  <ul class="find" data-citedArticleID="1006816" data-doi="10.1038/nrg2192"><li><a href="http://dx.doi.org/10.1038/nrg2192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+genetic+networks+by+non-adaptive+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+genetic+networks+by+non-adaptive+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001054-Lynch2" id="pone.0001054-Lynch2"></a>Lynch VJ, Roth JJ, Takahashi K, Dunn CW, Nonaka DF, et al.  (2004) Adaptive evolution of HoxA-11 and HoxA-13 at the origin of the uterus in mammals. Proc Biol Sci  271: 2201–2207.  <ul class="find" data-citedArticleID="1006818" data-doi="10.1098/rspb.2004.2848"><li><a href="http://dx.doi.org/10.1098/rspb.2004.2848" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+HoxA-11+and+HoxA-13+at+the+origin+of+the+uterus+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+HoxA-11+and+HoxA-13+at+the+origin+of+the+uterus+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001054-Arendt1" id="pone.0001054-Arendt1"></a>Arendt D (2003) Evolution of eyes and photoreceptor cell types. Int J Dev Biol  47: 563–571.  <ul class="find" data-citedArticleID="1006756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+eyes+and+photoreceptor+cell+types.&amp;auth=&amp;atitle=Evolution+of+eyes+and+photoreceptor+cell+types." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+eyes+and+photoreceptor+cell+types." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+eyes+and+photoreceptor+cell+types.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001054-Hoekstra2" id="pone.0001054-Hoekstra2"></a>Hoekstra HE, Coyne JA (2007) The locus of evolution: evo devo and the genetics of adaptation. Evolution Int J Org Evolution  61: 995–1016.  <ul class="find" data-citedArticleID="1006796" data-doi="10.1111/j.1558-5646.2007.00105.x"><li><a href="http://dx.doi.org/10.1111/j.1558-5646.2007.00105.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+locus+of+evolution%3A+evo+devo+and+the+genetics+of+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+locus+of+evolution%3A+evo+devo+and+the+genetics+of+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001054-Arendt2" id="pone.0001054-Arendt2"></a>Arendt D, Wittbrodt J (2001) Reconstructing the eyes of Urbilateria. Transactions of the Royal Society of London B 1545–1563.  <ul class="find" data-citedArticleID="1006758" data-doi="10.1098/rstb.2001.0971"><li><a href="http://dx.doi.org/10.1098/rstb.2001.0971" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+the+eyes+of+Urbilateria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+the+eyes+of+Urbilateria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001054-Terakita1" id="pone.0001054-Terakita1"></a>Terakita A (2005) The opsins. Genome Biol  6: 213.  <ul class="find" data-citedArticleID="1006866" data-doi="10.1186/gb-2005-6-3-213"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-3-213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+opsins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+opsins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001054-Raible1" id="pone.0001054-Raible1"></a>Raible F, Tessmar-Raible K, Arboleda E, Kaller T, Bork P, et al.  (2006) Opsins and clusters of sensory G-protein-coupled receptors in the sea urchin genome. Dev Biol  300: 461–475.  <ul class="find" data-citedArticleID="1006836" data-doi="10.1016/j.ydbio.2006.08.070"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2006.08.070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Opsins+and+clusters+of+sensory+G-protein-coupled+receptors+in+the+sea+urchin+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Opsins+and+clusters+of+sensory+G-protein-coupled+receptors+in+the+sea+urchin+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001054-Hardie1" id="pone.0001054-Hardie1"></a>Hardie RC, Raghu P (2001) Visual transduction in Drosophila. Nature  413: 186–193.  <ul class="find" data-citedArticleID="1006792" data-doi="10.1038/35093002"><li><a href="http://dx.doi.org/10.1038/35093002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Visual+transduction+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Visual+transduction+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001054-Kojima1" id="pone.0001054-Kojima1"></a>Kojima D, Terakita A, Ishikawa T, Tsukahara Y, Maeda A, et al.  (1997) A novel Go-mediated phototransduction cascade in scallop visual cells. J Biol Chem  272: 22979–22982.  <ul class="find" data-citedArticleID="1006806" data-doi="10.1074/jbc.272.37.22979"><li><a href="http://dx.doi.org/10.1074/jbc.272.37.22979" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+Go-mediated+phototransduction+cascade+in+scallop+visual+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+Go-mediated+phototransduction+cascade+in+scallop+visual+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001054-Koyanagi1" id="pone.0001054-Koyanagi1"></a>Koyanagi M, Terakita A, Kubokawa K, Shichida Y (2002) Amphioxus homologs of Go-coupled rhodopsin and peropsin having 11-cis- and all-trans-retinals as their chromophores. FEBS Lett  531: 525–528.  <ul class="find" data-citedArticleID="1006808" data-doi="10.1016/s0014-5793(02)03616-5"><li><a href="http://dx.doi.org/10.1016/s0014-5793(02)03616-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amphioxus+homologs+of+Go-coupled+rhodopsin+and+peropsin+having+11-cis-+and+all-trans-retinals+as+their+chromophores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amphioxus+homologs+of+Go-coupled+rhodopsin+and+peropsin+having+11-cis-+and+all-trans-retinals+as+their+chromophores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001054-Koyanagi2" id="pone.0001054-Koyanagi2"></a>Koyanagi M, Kubokawa K, Tsukamoto H, Shichida Y, Terakita A (2005) Cephalochordate melanopsin: evolutionary linkage between invertebrate visual cells and vertebrate photosensitive retinal ganglion cells. Curr Biol  15: 1065–1069.  <ul class="find" data-citedArticleID="1006810" data-doi="10.1016/j.cub.2005.04.063"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.04.063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cephalochordate+melanopsin%3A+evolutionary+linkage+between+invertebrate+visual+cells+and+vertebrate+photosensitive+retinal+ganglion+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cephalochordate+melanopsin%3A+evolutionary+linkage+between+invertebrate+visual+cells+and+vertebrate+photosensitive+retinal+ganglion+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001054-Dunlap1" id="pone.0001054-Dunlap1"></a>Dunlap JC, Loros JJ (2005)  Neurospora Photoreceptors. In: Briggs WR, Spudich JL, editors. Handbook of Photosensory Receptors. Weinheim: Wiley-VCH Verlag GmbH &amp; Co..   <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001054-Wilkinson1" id="pone.0001054-Wilkinson1"></a>Wilkinson M, McInerney JO, Hirt RP, Foster PG, Embley TM (2007) Of clades and clans: terms for phylogenetic relationships in unrooted trees. Trends in Ecology and Evolution  22: 114–115.  <ul class="find" data-citedArticleID="1006874" data-doi="10.1016/j.tree.2007.01.002"><li><a href="http://dx.doi.org/10.1016/j.tree.2007.01.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Of+clades+and+clans%3A+terms+for+phylogenetic+relationships+in+unrooted+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Of+clades+and+clans%3A+terms+for+phylogenetic+relationships+in+unrooted+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001054-Peterson1" id="pone.0001054-Peterson1"></a>Peterson KJ, Lyons JB, Nowak KS, Takacs CM, Wargo MJ, et al.  (2004) Estimating metazoan divergence times with a molecular clock. Proceedings of the National Academy of Sciences of the United States of America  101: 6536–6541.  <ul class="find" data-citedArticleID="1006832" data-doi="10.1073/pnas.0401670101"><li><a href="http://dx.doi.org/10.1073/pnas.0401670101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+metazoan+divergence+times+with+a+molecular+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+metazoan+divergence+times+with+a+molecular+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001054-Peterson2" id="pone.0001054-Peterson2"></a>Peterson KJ, Butterfield NJ (2005) Origin of the Eumetazoa: testing ecological predictions of molecular clocks against the Proterozoic fossil record. Proc Natl Acad Sci U S A  102: 9547–9552.  <ul class="find" data-citedArticleID="1006834" data-doi="10.1073/pnas.0503660102"><li><a href="http://dx.doi.org/10.1073/pnas.0503660102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+the+Eumetazoa%3A+testing+ecological+predictions+of+molecular+clocks+against+the+Proterozoic+fossil+record.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001054-Arendt3" id="pone.0001054-Arendt3"></a>Arendt D, Tessmar-Raible K, Snyman H, Dorresteijn AW, Wittbrodt J (2004) Ciliary photoreceptors with a vertebrate-type opsin in an invertebrate brain. Science  306: 869–871.  <ul class="find" data-citedArticleID="1006760" data-doi="10.1126/science.1099955"><li><a href="http://dx.doi.org/10.1126/science.1099955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ciliary+photoreceptors+with+a+vertebrate-type+opsin+in+an+invertebrate+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ciliary+photoreceptors+with+a+vertebrate-type+opsin+in+an+invertebrate+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001054-Velarde1" id="pone.0001054-Velarde1"></a>Velarde RA, Sauer CD, Walden KK, Fahrbach SE, Robertson HM (2005) Pteropsin: a vertebrate-like non-visual opsin expressed in the honey bee brain. Insect Biochem Mol Biol  35: 1367–1377.  <ul class="find" data-citedArticleID="1006872" data-doi="10.1016/j.ibmb.2005.09.001"><li><a href="http://dx.doi.org/10.1016/j.ibmb.2005.09.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pteropsin%3A+a+vertebrate-like+non-visual+opsin+expressed+in+the+honey+bee+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pteropsin%3A+a+vertebrate-like+non-visual+opsin+expressed+in+the+honey+bee+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001054-Marin1" id="pone.0001054-Marin1"></a>Marin EP, Krishna AG, Zvyaga TA, Isele J, Siebert F, et al.  (2000) The amino terminus of the fourth cytoplasmic loop of rhodopsin modulates rhodopsin-transducin interaction. J Biol Chem  275: 1930–1936.  <ul class="find" data-citedArticleID="1006820" data-doi="10.1074/jbc.275.3.1930"><li><a href="http://dx.doi.org/10.1074/jbc.275.3.1930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+amino+terminus+of+the+fourth+cytoplasmic+loop+of+rhodopsin+modulates+rhodopsin-transducin+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+amino+terminus+of+the+fourth+cytoplasmic+loop+of+rhodopsin+modulates+rhodopsin-transducin+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001054-Hrdy1" id="pone.0001054-Hrdy1"></a>Hrdy I, Hirt RP, Dolezal P, Bardonova L, Foster PG, et al.  (2004) Trichomonas hydrogenosomes contain the NADH dehydrogenase module of mitochondrial complex I. Nature  432: 618–622.  <ul class="find" data-citedArticleID="1006798" data-doi="10.1038/nature03149"><li><a href="http://dx.doi.org/10.1038/nature03149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trichomonas+hydrogenosomes+contain+the+NADH+dehydrogenase+module+of+mitochondrial+complex+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trichomonas+hydrogenosomes+contain+the+NADH+dehydrogenase+module+of+mitochondrial+complex+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001054-Zheng1" id="pone.0001054-Zheng1"></a>Zheng Y, Xu D, Gu X (2007) Functional divergence after gene duplication and sequence-structure relationship: a case study of G-protein alpha subunits. J Exp Zoolog B Mol Dev Evol  308: 85–96.  <ul class="find" data-citedArticleID="1006886" data-doi="10.1002/jez.b.21140"><li><a href="http://dx.doi.org/10.1002/jez.b.21140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+divergence+after+gene+duplication+and+sequence-structure+relationship%3A+a+case+study+of+G-protein+alpha+subunits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+divergence+after+gene+duplication+and+sequence-structure+relationship%3A+a+case+study+of+G-protein+alpha+subunits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001054-Su1" id="pone.0001054-Su1"></a>Su CY, Luo DG, Terakita A, Shichida Y, Liao HW, et al.  (2006) Parietal-eye phototransduction components and their potential evolutionary implications. Science  311: 1617–1621.  <ul class="find" data-citedArticleID="1006856" data-doi="10.1126/science.1123802"><li><a href="http://dx.doi.org/10.1126/science.1123802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parietal-eye+phototransduction+components+and+their+potential+evolutionary+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parietal-eye+phototransduction+components+and+their+potential+evolutionary+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001054-Leys1" id="pone.0001054-Leys1"></a>Leys SP, Cronin TW, Degnan BM, Marshall JN (2002) Spectral sensitivity in a sponge larva. J Comp Physiol A Neuroethol Sens Neural Behav Physiol  188: 199–202.  <ul class="find" data-citedArticleID="1006812" data-doi="10.1007/s00359-002-0293-y"><li><a href="http://dx.doi.org/10.1007/s00359-002-0293-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spectral+sensitivity+in+a+sponge+larva." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spectral+sensitivity+in+a+sponge+larva.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001054-Spudich1" id="pone.0001054-Spudich1"></a>Spudich JL, Yang CS, Jung KH, Spudich EN (2000) Retinylidene proteins: structures and functions from archaea to humans. Annu Rev Cell Dev Biol  16: 365–392.  <ul class="find" data-citedArticleID="1006854" data-doi="10.1146/annurev.cellbio.16.1.365"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.16.1.365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retinylidene+proteins%3A+structures+and+functions+from+archaea+to+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retinylidene+proteins%3A+structures+and+functions+from+archaea+to+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001054-Eakin1" id="pone.0001054-Eakin1"></a>Eakin RM, Westfall JA (1962) Fine Structure of Photoreceptors in the Hydromedusan, Polyorchis Penicillatus. Proc Natl Acad Sci U S A  48: 826–833.  <ul class="find" data-citedArticleID="1006774" data-doi="10.1073/pnas.48.5.826"><li><a href="http://dx.doi.org/10.1073/pnas.48.5.826" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fine+Structure+of+Photoreceptors+in+the+Hydromedusan%2C+Polyorchis+Penicillatus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fine+Structure+of+Photoreceptors+in+the+Hydromedusan%2C+Polyorchis+Penicillatus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001054-Eakin2" id="pone.0001054-Eakin2"></a>Eakin RM (1979) Evolutionary Significance of Photoreceptors: In Retrospect. American Zoologist  19: 647–653.  <ul class="find" data-citedArticleID="1006776" data-doi="10.1093/icb/19.2.647"><li><a href="http://dx.doi.org/10.1093/icb/19.2.647" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+Significance+of+Photoreceptors%3A+In+Retrospect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+Significance+of+Photoreceptors%3A+In+Retrospect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001054-Suga1" id="pone.0001054-Suga1"></a>Suga H, Koyanagi M, Hoshiyama D, Ono K, Iwabe N, et al.  (1999) Extensive gene duplication in the early evolution of animals before the parazoan-eumetazoan split demonstrated by G proteins and protein tyrosine kinases from sponge and hydra. J Mol Evol  48: 646–653.  <ul class="find" data-citedArticleID="1006858" data-doi="10.1007/pl00006508"><li><a href="http://dx.doi.org/10.1007/pl00006508" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+gene+duplication+in+the+early+evolution+of+animals+before+the+parazoan-eumetazoan+split+demonstrated+by+G+proteins+and+protein+tyrosine+kinases+from+sponge+and+hydra." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+gene+duplication+in+the+early+evolution+of+animals+before+the+parazoan-eumetazoan+split+demonstrated+by+G+proteins+and+protein+tyrosine+kinases+from+sponge+and+hydra.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001054-OHalloran1" id="pone.0001054-OHalloran1"></a>O'Halloran DM, Fitzpatrick DA, McCormack GP, McInerney JO, Burnell AM (2006) The molecular phylogeny of a nematode-specific clade of heterotrimeric G-protein alpha-subunit genes. J Mol Evol  63: 87–94.  <ul class="find" data-citedArticleID="1006826" data-doi="10.1007/s00239-005-0215-z"><li><a href="http://dx.doi.org/10.1007/s00239-005-0215-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+phylogeny+of+a+nematode-specific+clade+of+heterotrimeric+G-protein+alpha-subunit+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+phylogeny+of+a+nematode-specific+clade+of+heterotrimeric+G-protein+alpha-subunit+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001054-Gomez1" id="pone.0001054-Gomez1"></a>Gomez MP, Nasi E (2000) Light transduction in invertebrate hyperpolarizing photoreceptors: possible involvement of a Go-regulated guanylate cyclase. J Neurosci  20: 5254–5263.  <ul class="find" data-citedArticleID="1006786"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Light+transduction+in+invertebrate+hyperpolarizing+photoreceptors%3A+possible+involvement+of+a+Go-regulated+guanylate+cyclase.&amp;auth=&amp;atitle=Light+transduction+in+invertebrate+hyperpolarizing+photoreceptors%3A+possible+involvement+of+a+Go-regulated+guanylate+cyclase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Light+transduction+in+invertebrate+hyperpolarizing+photoreceptors%3A+possible+involvement+of+a+Go-regulated+guanylate+cyclase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Light+transduction+in+invertebrate+hyperpolarizing+photoreceptors%3A+possible+involvement+of+a+Go-regulated+guanylate+cyclase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001054-Gehring1" id="pone.0001054-Gehring1"></a>Gehring W, Rosbash M (2003) The coevolution of blue-light photoreception and circadian rhythms. J Mol Evol  57: Suppl 1S286–289.  <ul class="find" data-citedArticleID="1006784" data-doi="10.1007/s00239-003-0038-8"><li><a href="http://dx.doi.org/10.1007/s00239-003-0038-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+coevolution+of+blue-light+photoreception+and+circadian+rhythms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+coevolution+of+blue-light+photoreception+and+circadian+rhythms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001054-Taylor1" id="pone.0001054-Taylor1"></a>Taylor JS, Raes J (2004) Duplication and divergence: the evolution of new genes and old ideas. Annu Rev Genet  38: 615–643.  <ul class="find" data-citedArticleID="1006862" data-doi="10.1146/annurev.genet.38.072902.092831"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.092831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001054-Ohno1" id="pone.0001054-Ohno1"></a>Ohno S (1970) Evolution by gene duplication. New York: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001054-Altschul1" id="pone.0001054-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Research  25: 3389–3402.  <ul class="find" data-citedArticleID="1006754" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001054-Ewing1" id="pone.0001054-Ewing1"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="1006778" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001054-Yeh1" id="pone.0001054-Yeh1"></a>Yeh RF, Lim LP, Burge CB (2001) Computational inference of homologous gene structures in the human genome. Genome Res  11: 803–816.  <ul class="find" data-citedArticleID="1006880" data-doi="10.1101/gr.175701"><li><a href="http://dx.doi.org/10.1101/gr.175701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+inference+of+homologous+gene+structures+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+inference+of+homologous+gene+structures+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001054-Palczewski1" id="pone.0001054-Palczewski1"></a>Palczewski K, Kumasaka T, Hori T, Behnke CA, Motoshima H, et al.  (2000) Crystal structure of rhodopsin: A G protein-coupled receptor. Science  289: 739–745.  <ul class="find" data-citedArticleID="1006830" data-doi="10.1016/s0002-9394(00)00841-2"><li><a href="http://dx.doi.org/10.1016/s0002-9394(00)00841-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+rhodopsin%3A+A+G+protein-coupled+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+rhodopsin%3A+A+G+protein-coupled+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001054-Grens1" id="pone.0001054-Grens1"></a>Grens A, Gee L, Fisher DA, Bode HR (1996) CnNK-2, an NK-2 homeobox gene, has a role in patterning the basal end of the axis in hydra. Dev Biol  180: 473–488.  <ul class="find" data-citedArticleID="1006788" data-doi="10.1006/dbio.1996.0321"><li><a href="http://dx.doi.org/10.1006/dbio.1996.0321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CnNK-2%2C+an+NK-2+homeobox+gene%2C+has+a+role+in+patterning+the+basal+end+of+the+axis+in+hydra." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CnNK-2%2C+an+NK-2+homeobox+gene%2C+has+a+role+in+patterning+the+basal+end+of+the+axis+in+hydra.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001054-Notredame1" id="pone.0001054-Notredame1"></a>Notredame C, Higgins DG, Heringa J (2000) T-Coffee: A novel method for fast and accurate multiple sequence alignment. J Mol Biol  302: 205–217.  <ul class="find" data-citedArticleID="1006824" data-doi="10.1006/jmbi.2000.4042"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001054-Galtier1" id="pone.0001054-Galtier1"></a>Galtier N, Gouy M, Gautier C (1996) SEAVIEW and PHYLO_WIN: two graphic tools for sequence alignment and molecular phylogeny. Comput Appl Biosci  12: 543–548.  <ul class="find" data-citedArticleID="1006782" data-doi="10.1093/bioinformatics/12.6.543"><li><a href="http://dx.doi.org/10.1093/bioinformatics/12.6.543" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SEAVIEW+and+PHYLO_WIN%3A+two+graphic+tools+for+sequence+alignment+and+molecular+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SEAVIEW+and+PHYLO_WIN%3A+two+graphic+tools+for+sequence+alignment+and+molecular+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001054-Swofford1" id="pone.0001054-Swofford1"></a>Swofford D (2002) PAUP*. 4.1b10 ed. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0001054-Guindon1" id="pone.0001054-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="1006790"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001054-Ronquist1" id="pone.0001054-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="1006844" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001054-Abascal1" id="pone.0001054-Abascal1"></a>Abascal F, Zardoya R, Posada D (2005) ProtTest: selection of best-fit models of protein evolution. Bioinformatics  21: 2104–2105.  <ul class="find" data-citedArticleID="1006752" data-doi="10.1093/bioinformatics/bti263"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProtTest%3A+selection+of+best-fit+models+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProtTest%3A+selection+of+best-fit+models+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001054-Rambaut1" id="pone.0001054-Rambaut1"></a>Rambaut A, Drummond A (2003) Tracer. MCMC Trace Analysis Tool. 1.0 ed. Oxford: University of Oxford.   <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0001054-Jobb1" id="pone.0001054-Jobb1"></a>Jobb G, von Haeseler A, Strimmer K (2004) TREEFINDER: a powerful graphical analysis environment for molecular phylogenetics. BMC Evol Biol  4: 18.  <ul class="find" data-citedArticleID="1006802" data-doi="10.1186/1471-2148-4-18"><li><a href="http://dx.doi.org/10.1186/1471-2148-4-18" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREEFINDER%3A+a+powerful+graphical+analysis+environment+for+molecular+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREEFINDER%3A+a+powerful+graphical+analysis+environment+for+molecular+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001054-Yang1" id="pone.0001054-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="1006876" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001054-Shimodaira1" id="pone.0001054-Shimodaira1"></a>Shimodaira H, Hasegawa M (2001) CONSEL: for assessing the confidence of phylogenetic tree selection. Bioinformatics  17: 1246–1247.  <ul class="find" data-citedArticleID="1006848" data-doi="10.1093/bioinformatics/17.12.1246"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CONSEL%3A+for+assessing+the+confidence+of+phylogenetic+tree+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CONSEL%3A+for+assessing+the+confidence+of+phylogenetic+tree+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001054-Felsenstein1" id="pone.0001054-Felsenstein1"></a>Felsenstein J (1978) Cases in which parsimony or compatibility methods will be positively misleading. Systematic Zoology  27: 401–410.  <ul class="find" data-citedArticleID="1006780" data-doi="10.2307/2412923"><li><a href="http://dx.doi.org/10.2307/2412923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cases+in+which+parsimony+or+compatibility+methods+will+be+positively+misleading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cases+in+which+parsimony+or+compatibility+methods+will+be+positively+misleading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001054-Huelsenbeck1" id="pone.0001054-Huelsenbeck1"></a>Huelsenbeck JP (1997) Is the Felsenstein zone a fly trap? Systematic Biology  46: 69–74.  <ul class="find" data-citedArticleID="1006800"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Is+the+Felsenstein+zone+a+fly+trap%3F&amp;auth=&amp;atitle=Is+the+Felsenstein+zone+a+fly+trap%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+the+Felsenstein+zone+a+fly+trap%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+the+Felsenstein+zone+a+fly+trap%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001054-Rambaut2" id="pone.0001054-Rambaut2"></a>Rambaut A, Grassly NC (1997) Seq-Gen: An application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees. Computer Applications in the Biosciences in press.   <ul class="find" data-citedArticleID="1006840" data-doi="10.1093/bioinformatics/13.3.235"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.3.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seq-Gen%3A+An+application+for+the+Monte+Carlo+simulation+of+DNA+sequence+evolution+along+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seq-Gen%3A+An+application+for+the+Monte+Carlo+simulation+of+DNA+sequence+evolution+along+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001054-Thornton1" id="pone.0001054-Thornton1"></a>Thornton JW, DeSalle R (2000) Gene family evolution and homology: genomics meets phylogenetics. Annu Rev Genomics Hum Genet  1: 41–73.  <ul class="find" data-citedArticleID="1006868" data-doi="10.1146/annurev.genom.1.1.41"><li><a href="http://dx.doi.org/10.1146/annurev.genom.1.1.41" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+family+evolution+and+homology%3A+genomics+meets+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+family+evolution+and+homology%3A+genomics+meets+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001054-Chen1" id="pone.0001054-Chen1"></a>Chen K, Durand D, Farach-Colton M (2000) NOTUNG: a program for dating gene duplications and optimizing gene family trees. J Comput Biol  7: 429–447.  <ul class="find" data-citedArticleID="1006768" data-doi="10.1089/106652700750050871"><li><a href="http://dx.doi.org/10.1089/106652700750050871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NOTUNG%3A+a+program+for+dating+gene+duplications+and+optimizing+gene+family+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NOTUNG%3A+a+program+for+dating+gene+duplications+and+optimizing+gene+family+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001054-Durand1" id="pone.0001054-Durand1"></a>Durand D, Halldorsson BV, Vernot B (2006) A hybrid micro-macroevolutionary approach to gene tree reconstruction. J Comput Biol  13: 320–335.  <ul class="find" data-citedArticleID="1006772" data-doi="10.1089/cmb.2006.13.320"><li><a href="http://dx.doi.org/10.1089/cmb.2006.13.320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hybrid+micro-macroevolutionary+approach+to+gene+tree+reconstruction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hybrid+micro-macroevolutionary+approach+to+gene+tree+reconstruction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001054-Telford1" id="pone.0001054-Telford1"></a>Telford MJ (2006) Animal phylogeny. Curr Biol  16: R981–985.  <ul class="find" data-citedArticleID="1006864" data-doi="10.1016/j.cub.2006.10.048"><li><a href="http://dx.doi.org/10.1016/j.cub.2006.10.048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Animal+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Animal+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001054-Nielsen1" id="pone.0001054-Nielsen1"></a>Nielsen RH, JP (2001) Detecting positively selected amino acid sites using posterior predictive p-values. Pacific Symposium on Biocomputing, Proceedings: World Scientific. pp. 576–588.  <ul class="find-nolinks"></ul></li><li><span class="label">64.
              </span><a name="pone.0001054-Bollback1" id="pone.0001054-Bollback1"></a>Bollback JP (2006) SIMMAP: stochastic character mapping of discrete traits on phylogenies. BMC Bioinformatics  7: 88.  <ul class="find" data-citedArticleID="1006764"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SIMMAP%3A+stochastic+character+mapping+of+discrete+traits+on+phylogenies.&amp;auth=&amp;atitle=SIMMAP%3A+stochastic+character+mapping+of+discrete+traits+on+phylogenies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SIMMAP%3A+stochastic+character+mapping+of+discrete+traits+on+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SIMMAP%3A+stochastic+character+mapping+of+discrete+traits+on+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001054-Li1" id="pone.0001054-Li1"></a>Li J, Edwards PC, Burghammer M, Villa C, Schertler GF (2004) Structure of bovine rhodopsin in a trigonal crystal form. J Mol Biol  343: 1409–1438.  <ul class="find" data-citedArticleID="1006814" data-doi="10.1016/j.jmb.2004.08.090"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.08.090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+bovine+rhodopsin+in+a+trigonal+crystal+form." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+bovine+rhodopsin+in+a+trigonal+crystal+form.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001054-Shimodaira2" id="pone.0001054-Shimodaira2"></a>Shimodaira H (2002) An approximately unbiased test of phylogenetic tree selection. Syst Biol  51: 492–508.  <ul class="find" data-citedArticleID="1006850"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+approximately+unbiased+test+of+phylogenetic+tree+selection.&amp;auth=&amp;atitle=An+approximately+unbiased+test+of+phylogenetic+tree+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+approximately+unbiased+test+of+phylogenetic+tree+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+approximately+unbiased+test+of+phylogenetic+tree+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001054-Kishino1" id="pone.0001054-Kishino1"></a>Kishino H, Hasegawa M (1989) Evaluation of the maximum likelihood estimate of the evolutionary tree topologies from DNA sequence data, and the branching order in hominoidea. J Mol Evol  29: 170–179.  <ul class="find" data-citedArticleID="1006804" data-doi="10.1007/bf02100115"><li><a href="http://dx.doi.org/10.1007/bf02100115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+maximum+likelihood+estimate+of+the+evolutionary+tree+topologies+from+DNA+sequence+data%2C+and+the+branching+order+in+hominoidea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+maximum+likelihood+estimate+of+the+evolutionary+tree+topologies+from+DNA+sequence+data%2C+and+the+branching+order+in+hominoidea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001054-Yang2" id="pone.0001054-Yang2"></a>Yang Z, Rannala B (1997) Bayesian phylogenetic inference using DNA sequences: a Markov Chain Monte Carlo Method. Mol Biol Evol  14: 717–724.  <ul class="find" data-citedArticleID="1006878" data-doi="10.1093/oxfordjournals.molbev.a025811"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+phylogenetic+inference+using+DNA+sequences%3A+a+Markov+Chain+Monte+Carlo+Method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+phylogenetic+inference+using+DNA+sequences%3A+a+Markov+Chain+Monte+Carlo+Method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.XML" value="117820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.PDF" value="548064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g001.PNG_L" value="5809764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g001.PNG_M" value="247366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g001.PNG_S" value="24985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g001.TIF" value="8885974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g001.PNG_I" value="323309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g002.PNG_L" value="977913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g002.PNG_M" value="144510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g002.PNG_S" value="10841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g002.TIF" value="1564506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g002.PNG_I" value="51487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g003.PNG_L" value="719645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g003.PNG_M" value="154193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g003.PNG_S" value="21216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g003.TIF" value="1341436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g003.PNG_I" value="98165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g004.PNG_L" value="581765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g004.PNG_M" value="73778"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g004.PNG_S" value="7571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g004.TIF" value="866860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g004.PNG_I" value="26920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.t001.PNG_L" value="216738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.t001.PNG_M" value="130463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.t001.PNG_S" value="14650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.t001.TIF" value="1012266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.t001.PNG_I" value="40371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g005.PNG_L" value="87383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g005.PNG_M" value="147636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g005.PNG_S" value="12756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g005.TIF" value="642566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g005.PNG_I" value="54054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g006.PNG_L" value="1914002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g006.PNG_M" value="127308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g006.PNG_S" value="10621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g006.TIF" value="2755934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g006.PNG_I" value="50496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g007.PNG_L" value="656384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g007.PNG_M" value="189751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g007.PNG_S" value="15724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g007.TIF" value="2194674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.g007.PNG_I" value="73781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.s001.DOC" value="114176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.s002.TIF" value="594566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001054.s003.DOC" value="6194484"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001054&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001054" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001054&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001054&volume=&issue=&title=The Origins of Novel Protein Interactions during Animal Opsin Evolution&author_name=David%20C.%20Plachetzki%2C%20Bernard%20M.%20Degnan%2C%20Todd%20H.%20Oakley&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001054" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054&amp;t=The%20Origins%20of%20Novel%20Protein%20Interactions%20during%20Animal%20Opsin%20Evolution" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054&title=The%20Origins%20of%20Novel%20Protein%20Interactions%20during%20Animal%20Opsin%20Evolution&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054&amp;title=The%20Origins%20of%20Novel%20Protein%20Interactions%20during%20Animal%20Opsin%20Evolution" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001054&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Origins of Novel Protein Interactions during Animal Opsin Evolution';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001054';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Origins%20of%20Novel%20Protein%20Interactions%20during%20Animal%20Opsin%20Evolution http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001054" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001054" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+evolution%22" title="Search for articles in the subject area:'Animal evolution'"><div class="flagText">Animal evolution</div></a>
              <div data-categoryid="18779" data-articleid="24942"
                   data-categoryname="Animal evolution"
                   class="flagImage" title="Flag 'Animal evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="24942"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="24942"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+duplication%22" title="Search for articles in the subject area:'Gene duplication'"><div class="flagText">Gene duplication</div></a>
              <div data-categoryid="17929" data-articleid="24942"
                   data-categoryname="Gene duplication"
                   class="flagImage" title="Flag 'Gene duplication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24942"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Molecular+evolution%22" title="Search for articles in the subject area:'Molecular evolution'"><div class="flagText">Molecular evolution</div></a>
              <div data-categoryid="17533" data-articleid="24942"
                   data-categoryname="Molecular evolution"
                   class="flagImage" title="Flag 'Molecular evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24942"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="24942"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7485'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7990'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4299&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=21777">Updated Table S1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
