

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Single Feature Polymorphism Discovery in Rice</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000284"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000284" />

    <meta name="citation_title" content="Single Feature Polymorphism Discovery in Rice"/>
    <meta itemprop="name" content="Single Feature Polymorphism Discovery in Rice"/>

      <meta name="citation_author" content="Rajesh Kumar"/>
            <meta name="citation_author_institution" content="Division of Plant Sciences, University of Missouri-Columbia, Columbia, Missouri, United States of America"/>
      <meta name="citation_author" content="Jing Qiu"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Missouri-Columbia, Columbia, Missouri, United States of America"/>
      <meta name="citation_author" content="Trupti Joshi"/>
            <meta name="citation_author_institution" content="Computer Science Department and Christopher S. Bond Life Sciences Center, University of Missouri-Columbia, Columbia, Missouri, United States of America"/>
      <meta name="citation_author" content="Babu Valliyodan"/>
            <meta name="citation_author_institution" content="Division of Plant Sciences, University of Missouri-Columbia, Columbia, Missouri, United States of America"/>
      <meta name="citation_author" content="Dong Xu"/>
            <meta name="citation_author_institution" content="Computer Science Department and Christopher S. Bond Life Sciences Center, University of Missouri-Columbia, Columbia, Missouri, United States of America"/>
      <meta name="citation_author" content="Henry T. Nguyen"/>
            <meta name="citation_author_institution" content="Division of Plant Sciences, University of Missouri-Columbia, Columbia, Missouri, United States of America"/>

    <meta name="citation_date" content="2007/3/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000284.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e284"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome.; citation_journal_title=Nature; citation_volume=436; citation_number=1; citation_pages=793-800; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The small, the large and the wild: the value of comparison in plant genomics.; citation_author=M Bevan; citation_author=G Murphy; citation_journal_title=Trends Genet; citation_volume=15; citation_number=2; citation_pages=211-214; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An SNP resource for rice genetics and breeding based on subspecies indica and japonica genome alignments.; citation_author=FA Feltus; citation_author=J Wan; citation_author=SR Schulze; citation_author=JC Estill; citation_author=N Jiang; citation_journal_title=Genome Res; citation_volume=14; citation_number=3; citation_pages=1812-1819; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Development of genome-wide DNA polymorphism database for map-based cloning of rice genes.; citation_author=YJ Shen; citation_author=H Jiang; citation_author=JP Jin; citation_author=ZB Zhang; citation_author=B Xi; citation_journal_title=Plant Physiol; citation_volume=135; citation_number=4; citation_pages=1198-1205; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity in yeast assessed with whole-genome oligonucleotide arrays.; citation_author=EA Winzeler; citation_author=CI Castillo-Davis; citation_author=G Oshiro; citation_author=D Liang; citation_author=DR Richards; citation_journal_title=Genetics; citation_volume=163; citation_number=5; citation_pages=79-89; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Large-scale identification of single-feature polymorphisms in complex genomes.; citation_author=JO Borevitz; citation_author=D Liang; citation_author=D Plouffe; citation_author=HS Chang; citation_author=T Zhu; citation_journal_title=Genome Res; citation_volume=13; citation_number=6; citation_pages=513-523; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Single-feature polymorphism discovery in the barley transcriptome.; citation_author=N Rostoks; citation_author=JO Borevitz; citation_author=PE Hedley; citation_author=J Russell; citation_author=S Mudie; citation_journal_title=Genome Biol; citation_volume=6; citation_number=7; citation_pages=R54; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detecting single-feature polymorphisms using oligonucleotide arrays and robustified projection pursuit.; citation_author=X Cui; citation_author=J Xu; citation_author=R Asghar; citation_author=P Condamine; citation_author=JT Svensson; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=8; citation_pages=3852-3858; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Population Structure and Breeding Patterns of 145 U.S. Rice Cultivars Based on SSR Marker Analysis.; citation_author=H Lu; citation_author=MA Redus; citation_author=JR Coburn; citation_author=JN Rutger; citation_author=SR McCouch; citation_author=TH Tai; citation_journal_title=Crop Sci; citation_volume=45; citation_number=9; citation_pages=66-76; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RiceCAP project.; citation_number=10; " />
      <meta name="citation_reference" content="citation_title=QTL mapping of grain quality traits from the interspecific cross Oryza sativa x O. glaberrima.; citation_author=G Aluko; citation_author=C Martinez; citation_author=J Tohme; citation_author=C Castano; citation_author=C Bergman; citation_journal_title=Theor Appl Genet; citation_volume=109; citation_number=11; citation_pages=630-639; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Array Genotyping and Mapping.; citation_author=J Borevitz; citation_number=12; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=13; " />
      <meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response.; citation_author=VG Tusher; citation_author=R Tibshirani; citation_author=G Chu; citation_journal_title=Proc Natl Acad Sci; citation_volume=98; citation_number=14; citation_pages=5116-21; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic structure and diversity in Oryza sativa L.; citation_author=AJ Garris; citation_author=TH Tai; citation_author=J Coburn; citation_author=S Kresovich; citation_author=S McCouch; citation_journal_title=Genetics; citation_volume=169; citation_number=15; citation_pages=1631-1638; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluation of Genetic Diversity in Rice Subspecies Using Microsatellite Markers.; citation_author=J Ni; citation_author=PM Colowit; citation_author=DJ Mackill; citation_journal_title=Crop Sci; citation_volume=42; citation_number=16; citation_pages=601-607; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=17; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=18; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene arrays are not just for measuring gene expression.; citation_author=SP Hazen; citation_author=SA Kay; citation_journal_title=Trends Plant Sci; citation_volume=8; citation_number=19; citation_pages=413-416; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FRIGIDA-independent variation in flowering time of natural Arabidopsis thaliana accessions.; citation_author=JD Werner; citation_author=JO Borevitz; citation_author=NH Uhlenhaut; citation_author=JR Ecker; citation_author=J Chory; citation_author=D Weigel; citation_journal_title=Genetics; citation_volume=170; citation_number=20; citation_pages=1197-1207; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Light-response quantitative trait loci identified with composite interval and eXtreme array mapping in Arabidopsis thaliana.; citation_author=DJ Wolyn; citation_author=JO Borevitz; citation_author=O Loudet; citation_author=C Schwartz; citation_author=J Maloof; citation_journal_title=Genetics; citation_volume=167; citation_number=21; citation_pages=907-917; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rapid array mapping of circadian clock and developmental mutations in Arabidopsis.; citation_author=SP Hazen; citation_author=JO Borevitz; citation_author=FG Harmon; citation_author=JL Pruneda-Paz; citation_author=TF Schultz; citation_journal_title=Plant Physiol; citation_volume=138; citation_number=22; citation_pages=990-997; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Plant molecular diversity and applications to genomics.; citation_author=ES Buckler; citation_author=JM Thornsberry; citation_journal_title=Curr Opin Plant Biol; citation_volume=5; citation_number=23; citation_pages=107-111; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Whole-genome patterns of common DNA variation in three human populations.; citation_author=DA Hinds; citation_author=LL Stuve; citation_author=GB Nilsen; citation_author=E Halperin; citation_author=E Eskin; citation_journal_title=Science; citation_volume=307; citation_number=24; citation_pages=1072-1079; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Association mapping with Single-Feature Polymorphisms.; citation_author=S Kim; citation_author=K Zhao; citation_author=R Jiang; citation_author=J Molitor; citation_author=J Borevitz; citation_journal_title=Genetics; citation_volume=173; citation_number=25; citation_pages=1125-1133; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Positional cloning in Arabidopsis. Why it feels good to have a genome initiative working for you.; citation_author=W Lukowitz; citation_author=CS Gillmor; citation_author=WR Scheible; citation_journal_title=Plant Physiol; citation_volume=123; citation_number=26; citation_pages=795-806; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Single Feature Polymorphism Discovery in Rice"/>
    <meta name="twitter:description" content="The discovery of nucleotide diversity captured as single feature polymorphism (SFP) by using the expression array is a high-throughput and effective method in detecting genome-wide polymorphism. The efficacy of such method was tested in rice, and the results presented in the paper indicate high sensitivity in predicting SFP. The sensitivity of polymorphism detection was further demonstrated by the fact that no biasness was observed in detecting SFP with either single or multiple nucleotide polymorphisms. The high density SFP data that can be generated quite effectively by the current method has promise for high resolution genetic mapping studies, as physical location of features are well-defined on rice genome."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000284.g004"/>

  <meta property="og:title" content="Single Feature Polymorphism Discovery in Rice" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8557'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9762'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1282&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000284">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000284" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000284&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Single Feature Polymorphism Discovery in Rice
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rajesh Kumar
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kumarr@missouri.edu">kumarr@missouri.edu</a> (RK); <a href="mailto:nguyenhenry@missouri.edu">nguyenhenry@missouri.edu</a> (HN)</p>

                <p>Affiliation:
                  Division of Plant Sciences, University of Missouri-Columbia, Columbia, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jing Qiu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Missouri-Columbia, Columbia, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Trupti Joshi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computer Science Department and Christopher S. Bond Life Sciences Center, University of Missouri-Columbia, Columbia, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Babu Valliyodan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Plant Sciences, University of Missouri-Columbia, Columbia, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dong Xu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computer Science Department and Christopher S. Bond Life Sciences Center, University of Missouri-Columbia, Columbia, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Henry T. Nguyen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kumarr@missouri.edu">kumarr@missouri.edu</a> (RK); <a href="mailto:nguyenhenry@missouri.edu">nguyenhenry@missouri.edu</a> (HN)</p>

                <p>Affiliation:
                  Division of Plant Sciences, University of Missouri-Columbia, Columbia, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000284</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000284-g001" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-g002" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-t001" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-g003" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-t002" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-t003" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-t004" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-t005" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-t006" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000284-g004" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000284">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000284" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The discovery of nucleotide diversity captured as single feature polymorphism (SFP) by using the expression array is a high-throughput and effective method in detecting genome-wide polymorphism. The efficacy of such method was tested in rice, and the results presented in the paper indicate high sensitivity in predicting SFP. The sensitivity of polymorphism detection was further demonstrated by the fact that no biasness was observed in detecting SFP with either single or multiple nucleotide polymorphisms. The high density SFP data that can be generated quite effectively by the current method has promise for high resolution genetic mapping studies, as physical location of features are well-defined on rice genome.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kumar R, Qiu J, Joshi T, Valliyodan B, Xu D, et al.  (2007) Single Feature Polymorphism Discovery in Rice. PLoS ONE 2(3):
          e284.
            doi:10.1371/journal.pone.0000284</p><p><strong>Academic Editor: </strong>Justin Borevitz, University of Chicago, United States of America</p><p><strong>Received:</strong> October 10, 2006; <strong>Accepted:</strong> February 16, 2007; <strong>Published:</strong> March 14, 2007</p><p><strong>Copyright:</strong> © 2007 Kumar et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was conducted under RiceCAP project (<a href="http://www.uark.edu/ua/ricecap/">www.uark.edu/ua/ricecap/</a>) entitled “A coordinated research, education, and extension project for the application of genomics discoveries to improve rice in the United States.” The multi-institutional RiceCAP project was supported by the USDA-NRI Competitive Grants Program and funding was from the USDA-CSREES (2004-35317-14867) to HTN.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The publicly available genome sequence information of rice <a href="#pone.0000284-International1">[1]</a> opens a great opportunity for facilitating and integrating various genomics studies, for example, isolation of genetic determinants associated with traits of economic importance. Such discovery has not only promises to complement the molecular breeding efforts but also speeding up the process of crop improvements in general by incorporating useful genes into agronomically suitable varieties through genetic engineering. Additionally, information obtained with rice can also be translated into other crops as well because of high conservation of synteny observed among related cereal crop species <a href="#pone.0000284-Bevan1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The nucleotide diversity across a genome is the source of most of the phenotypic variation. Such DNA polymorphism is the basis for development of molecular markers, an indispensable tool in genetic mapping studies. In general, the high resolution fine mapping of genes is often limited by lack of sufficient number of polymorphic molecular markers. This problem is compounded with traits controlled by multi-genes because in several such studies QTL locus can't be resolved to a workable resolution that could be feasible for predicting the candidate gene(s) associated with traits of interests. The sequence comparison of Nipponbare and 93-11 genome has shown high degree of polymorphisms ranging from a SNP/~300 bp to an indel/kbp <a href="#pone.0000284-Feltus1">[3]</a>–<a href="#pone.0000284-Shen1">[4]</a> that can potentially be exploited as molecular markers between these two genetically diverged sub-species. In genetic mapping studies PCR-based SSR and CAPS markers are used routinely and generation of such molecular markers becomes easier if one of the two parents of mapping population has sequence information available. Many varieties of rice endowed with different traits of economic interests are grown worldwide. In situations where both parents of mapping population lack sequence information and are in similar genetic background often requires multiple steps in the process of identification and generation of such PCR based molecular markers.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>There are several ways through which polymorphisms are identified across the genome. The most direct method is to re-sequence the PCR amplicons, the DNA fragments containing alleles from the inbred parents. Similarly EST sequences also provide the direct way to compare the sequence and provide useful information about polymorphism. These methods although straight forward in approach but are quite labor intensive and lack high-throughput. The high density oligonucleotide expression arrays, designed for transcript profiling, have been used successfully as an effective tool for DNA genotyping to measure numerous polymorphic loci in yeast <a href="#pone.0000284-Winzeler1">[5]</a> and Arabidopsis <a href="#pone.0000284-Borevitz1">[6]</a>. Application of such DNA-based technique in complex genome like barley however were not as sensitive as it was reported in Arabidopsis but when RNA was used as surrogate for DNA the efficiency in predicting polymorphism increased significantly <a href="#pone.0000284-Rostoks1">[7]</a>–<a href="#pone.0000284-Cui1">[8]</a>. The basis of genome wide polymorphism discovery by the above approach is dependent on the principle that a sequence which is perfect match to a feature/probe sequence present on gene-chip or array may hybridize with greater affinity than one with a mismatch sequence. The polymorphism of the two sequences, originating from two different varieties or genotype, results in differential hybridization intensity and this property associated with sequence characteristics functions as a molecular marker popularly known as <strong>s</strong>ingle <strong>f</strong>eature <strong>p</strong>olymorphism (SFP) <a href="#pone.0000284-Winzeler1">[5]</a>–<a href="#pone.0000284-Borevitz1">[6]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The objective of present study was to test the efficacy of such DNA-based gene-chip approach to identify polymorphism in rice. Here we demonstrate that hybridization of probes, generated from labeling of g-DNA, to rice whole genome expression array (Affymetrix) is quite sensitive in predicting SFP <em>a priori</em> of their sequence information. The rice varieties used in this study were Cypress (CP), LaGrue (LG) and RT0034 (RT); first two belong to japonica <a href="#pone.0000284-Lu1">[9]</a> whereas RT0034 to indica background <a href="#pone.0000284-1">[10]</a> respectively. These three varieties constitutes parents of two population developed by RiceCAP <a href="#pone.0000284-1">[10]</a> for mapping QTLs/genes associated with economically important milling yield trait. Milling yield is a complex quantitative trait <a href="#pone.0000284-Aluko1">[11]</a> and considered as products of numerous loci with varying degrees of effect upon the observed phenotypes. Since SFPs can reliably be predicted in rice, the methods presented in the paper can be applied to any rice varieties irrespective of their sequence information for not only polymorphism discovery but also as a tool for functional genotyping of natural varieties.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Hybridization and data quality</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The biotin labeled probes generated from labeling of g-DNA was hybridized to Affymetrix rice expression array (see experimental procedure). Following hybridization the preliminary data quality was assessed from GCOS1.3 software (Affymetrix) generated expression report according to guidelines (see Affymetrix manual) set for such experiments. The average background, noise (RawQ) and the call rate was comparable among all the three rice varieties viz. Cypress (CP), LaGrue (LG) and RT0034 (RT) and also among their biological replicates (data not shown). However to get better assessment of data quality the raw intensity data of only perfect match (PM) probes/features of rice varieties viz. CP, LG and RT were log2 transformed and studied by density plots (<a href="#pone-0000284-g001">Figure 1</a>) and pair-wise scatter plots (<a href="#pone-0000284-g002">Figure 2</a>) respectively. The results obtained from density plot indicated no major deviations as replicates of rice varieties were correlated to each other. For scatter plot study 12000 randomly chosen features were plotted against each other for all pair-wise combinations (<a href="#pone-0000284-g002">Figure 2</a>) as suggested in Borevitz's methods paper <a href="#pone.0000284-Borevitz2">[12]</a>. No major variation was observed among biological replicates of each variety as most of the features were falling along the diagonal. The features falling above or below diagonal lines indicate their differential hybridization intensity and thus qualify for SFPs. The number of such features showing differential hybridization in CP&amp;LG (blue box) was much less than those in CP&amp;RT (red box) or LG&amp;RT (green box) pairs respectively as one would expect between varieties of same than to different genetic background.</p>
<div class="figure" id="pone-0000284-g001"><div class="img"><a name="pone-0000284-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Density plots for the raw PM probes intensity data.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The data are in log2 scale and the biological replicate arrays for rice varieties CP, LG and RT are shown in black, red and green color respectively. CP = Cypress, LG = LaGrue, RT = RT0034 and PM = perfect match.</p>
<span>doi:10.1371/journal.pone.0000284.g001</span></div><div class="figure" id="pone-0000284-g002"><div class="img"><a name="pone-0000284-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Pair-wise scatter plots for the raw PM probes intensity data across all arrays.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The intersection of 12000 randomly chosen features/probes data for CP and LG is boxed in blue, for CP and RT in red and for LG and RT in green respectively. The biological replicates of each variety are highly correlated as features are falling along diagonal line. CP = Cypress, LG = LaGrue, RT = RT0034 and PM = perfect match.</p>
<span>doi:10.1371/journal.pone.0000284.g002</span></div>

<h4>SFP prediction</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The background corrected and quantile-normalized log2 intensity values of all PM features of triplicate data of each rice variety were subjected to SFPs call by using the siggene package (<a href="http://www.boiconductor.org">www.boiconductor.org</a>) and SAM procedure in R language software essentially as described <a href="#pone.0000284-Rostoks1">[7]</a>, <a href="#pone.0000284-Borevitz2">[12]</a>–<a href="#pone.0000284-Supplementary1">[13]</a>. SAM procedure allows the users to choose the delta value, a threshold for the SAM d-statistics, so as to get a balanced number of significant genes or SFPs as in present study with a tolerable FDR, which is estimated by permutation <a href="#pone.0000284-Tusher1">[14]</a>. In this paper, the FDR that is determined by permutation according to the SAM procedure is referred to the <em>estimated FDR</em> and the FDR that is determined after sequencing is referred to the <em>observed FDR</em>. The SFPs called at different threshold (delta) in the three datasets viz. CP&amp;LG (a), CP&amp;RT (b) and LG&amp;RT (c) is presented in <a href="#pone-0000284-t001">Table 1</a>. Since the estimated FDR was stable for a wide range of delta values (data not shown), meaning a larger number of significant SFP would imply a larger number of false positives although the proportion of false positive doesn't change; we elected to choose the largest number of SFPs at the given stable estimated FDR for their verification by sequence analyses. Based on above consideration we selected 5376 SFPs that were called with an estimated 9.5% FDR in CP&amp;LG pair (<a href="#pone-0000284-t001">Table 1a</a>) and 25325 SFPs for CP&amp;RT pair at an estimated 9% FDR (<a href="#pone-0000284-t001">Table 1b</a>).</p>
<div class="figure" id="pone-0000284-t001"><div class="img"><a name="pone-0000284-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Number of gene-chip predicted SFPs called at different threshold (delta) in datasets.</span></strong></p><span>doi:10.1371/journal.pone.0000284.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The distribution of gene-chip predicted SFPs (at ≥10% estimated FDR) among polymorphic probesets in the three datasets is shown in <a href="#pone-0000284-t002">Table 2</a>. The observation of ~6-7xs polymorphism in CP&amp;RT or LG&amp;RT datasets compared to CP&amp;LG in the present study is therefore in agreement with significantly higher genetic divergence between japonica and indica variety of rice <a href="#pone.0000284-Feltus1">[3]</a>–<a href="#pone.0000284-Shen1">[4]</a>, <a href="#pone.0000284-Garris1">[15]</a> than those observed between varieties of similar genetic background <a href="#pone.0000284-Ni1">[16]</a>. The SAM plot of normalized data of all the PM probes on array for all the three datasets is shown in <a href="#pone-0000284-g003">Figure 3</a>. The probes exceeding the threshold, shown in green color, signify SFPs, sign (+/−) associated with SFP indicates direction of polymorphism and the values as its SAM d-stat value.</p>
<div class="figure" id="pone-0000284-g003"><div class="img"><a name="pone-0000284-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>SAM plot of normalized data for CP&amp;LG (a) and CP&amp;RT (b) and LG&amp;RT(c) pairs.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Observed d-statistics (y-axis) is plotted against the expected d-statistics (x-axis) as determined by permutations and SFPs exceeding the threshold are shown in green. The sign (+/−) with SFPs indicates direction of polymorphism. In (a) the (-) sign (i.e. CP-SFP) indicates polymorphism in LG (i.e. CP&gt;LG) and (+) sign (i.e. LG-SFP) polymorphism in CP (i.e. LG&gt;CP); in (b) the (-) sign (i.e. CP-SFP) signifies polymorphism in RT(i.e. CP&gt;RT) and the (+) sign (i.e. RT-SFP) polymorphism in CP(i.e. RT&gt;CP) and in (c) the (-) sign (i.e. LG-SFP) indicates polymorphism in RT (i.e. LG&gt;RT) and (+) sign (i.e. RT-SFP) polymorphism in LG (i.e. RT&gt;LG). CP = Cypress, LG = LaGrue and RT = RT0034.</p>
<span>doi:10.1371/journal.pone.0000284.g003</span></div><div class="figure" id="pone-0000284-t002"><div class="img"><a name="pone-0000284-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Distribution of gene-chip predicted SFPs among polymorphic probesets.</span></strong></p><span>doi:10.1371/journal.pone.0000284.t002</span></div>

<h4>SFP verification</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To test the sensitivity of gene chip predicted SFPs data we verified them by sequence comparison. To the best of our knowledge no sequence information was available for rice varieties Cypress (CP), LaGrue (LG) and RT0034 (RT) and due to this limitation fragments flanking the SFPs were amplified from their respective genome (see experimental procedure) and their sequences was compared to verify polymorphism, if any. To simplify and maintain uniformity we validated SFPs whose corresponding 25mer probe/feature sequence had unique location, 100% identity and distributed randomly on all 12 chromosomes (supplementary <a href="#pone.0000284.s001">Table S1</a> &amp; <a href="#pone.0000284.s002">S2</a>) of Nipponbare genome (TIGRv3) sequence available during the course of experiment. We generated sequence information of altogether 186 probes having unique location for CP&amp;LG pair. The 104 probes were having known sequence polymorphism and 77 were predicted correctly at 9.5% estimated FDR by SAM procedure suggesting 74% sensitivity of SFP detection (<a href="#pone-0000284-t003">Table 3a</a>). Similarly for CP&amp;RT pair we generated sequence information of 603 probes having unique location and among the 245 probes having known sequence polymorphism 180 were predicted correctly at 9% estimated FDR by SAM procedure suggesting 73% sensitivity (<a href="#pone-0000284-t003">Table 3b</a>). The direction of polymorphism was correct in all of the sequence verified SFPs in both the pairs/datasets except four in CP&amp;RT pair. Such rare reversal of polymorphism has also been observed with SFP study in barley <a href="#pone.0000284-Rostoks1">[7]</a>.</p>
<div class="figure" id="pone-0000284-t003"><div class="img"><a name="pone-0000284-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Verification of gene-chip predicted SFPs by sequence information.</span></strong></p><span>doi:10.1371/journal.pone.0000284.t003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Since a significant proportion (~25%) of probes having known sequence variation in both the data set escaped from being predicted as SFP at the given analyzed FDR, we explored the possibility of finding such SFPs at less stringent estimated FDR. In CP&amp;LG dataset at 11.9% estimated FDR the number of predicted SFP nearly doubled to 9093 compared to those predicted at 9.5% estimated FDR (<a href="#pone-0000284-t001">Table 1a</a>). From the available sequence information, we found that out of 27 SFPs that escaped detection at 9.5% FDR, five were predicted correctly at 11.9% FDR and thus increasing the sensitivity further to 79% (<a href="#pone-0000284-t003">Table 3a</a>). Similarly at estimated FDR of 10% the number of SFPs predicted in CP&amp;RT dataset nearly doubled to 61055 (<a href="#pone-0000284-t001">Table 1b</a>) including 35730 unique SFPs. Based on available probe/feature sequence information, we verified sequences of 257 SFPs predicted at 10% FDR. Among the 245 probes having known sequence polymorphism, 207 were predicted correctly increasing the sensitivity to 84% (<a href="#pone-0000284-t003">Table 3b</a>) and the 27 additional SFP were those that escaped detection at 9% estimated FDR. Although by lowering the stringency of estimated FDR, the sensitivity of SFP detection was increased substantially in CP&amp;RT dataset but simultaneously the observed FDR determined after sequencing also doubled (20%) compared to the estimated value by permutation (<a href="#pone-0000284-t003">Table 3b</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The higher the value of d-stat of SFP, the greater is the likelihood of being predicted true <a href="#pone.0000284-Rostoks1">[7]</a>. In CP&amp;RT dataset, among the 204 sequence verified SFPs, 24 turned out to be false positive indicating marginally higher FDR (11.7%) compared to the estimated value (9%) by permutation. Since we couldn't find SFPs with higher d-stat value being false positive in both the datasets and all the false positives in CP&amp;RT dataset had d-stat values close to permuted cut-off value (data not shown) suggesting that most false positives, if not all, would be among the SFPs that has d-stat value close to permuted cut-off level. In CP&amp;LG dataset most of the SFPs, which were verified by sequencing, had higher d-stat value (data not shown) and that could possibly explain why we couldn't find number of false positives determined after sequencing comparable to that of permuted value.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>In genetic mapping studies theoretically a probeset/gene having either one or multiple SFP will provide the same information if such probeset/gene having their defined position on genome is being exploited as molecular marker. According to result presented in <a href="#pone-0000284-t002">Table 2</a>, the number of SFPs per probeset and the number of polymorphic probesets were quite variable in all the three datasets. Since ~55% of the total polymorphic probesets were having only one SFP each in all three datasets, we estimated how many of such polymorphic probesets were true positive. Among the sequence verified SFP predicted at 9.5% estimated FDR in CP&amp;LG dataset (<a href="#pone-0000284-t003">Table 3a</a>), 11 probesets had one SFP each and none escaped detection (data not shown). On the other hand among the sequence validated SFPs predicted at 9% estimated FDR in CP&amp;RT dataset (<a href="#pone-0000284-t003">Table 3b</a>), 47 probesets were having one SFP each and all were true positive except five and the d-stat values of SFPs associated with such false positive probesets were close to permuted cut-off level (data not shown). The absence of such false positive probesets in CP&amp;LG dataset might be due to either higher d-stat value of sequence verified SFPs as mentioned earlier and/or less number of such probesets were analyzed compared to CP&amp;RT dataset.</p>


<h4>Nature of polymorphism and detection sensitivity</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>SNPs (single nucleotide polymorphisms) are the most frequent form of polymorphism observed in any organism. Given their wide application in genetic fine mapping studies, it was of interest to analyze how many of probes/features having known SNP was detected by this method. From available sequence information in CP&amp;LG dataset 46 probes were having SNP but only 29 were predicted correctly indicating 63% detection efficiency (<a href="#pone-0000284-t004">Table 4a</a>). On the other hand in CP&amp;RT dataset 144 probes were having SNP and 108 were predicted correctly suggesting 75% efficiency (<a href="#pone-0000284-t004">Table 4b</a>). The less detection efficiency of SFP having SNP in CP&amp;LG dataset could be due to less number of such SFPs were analyzed compared to SFPs with multiple nucleotide polymorphisms (NP). Based on similar nature of previous study <a href="#pone.0000284-Cui1">[8]</a> SFPs containing SNP were classified into two category viz. SNP residing either at margin (flanking1–5 bases) or in the middle (6–20 bases) of 25mer probe/feature (<a href="#pone-0000284-t004">Table 4</a>). Although the number of sequence verified SFPs having SNP at the flanking 1–5 bases were comparatively less than those present in the middle of features, the present study reconfirms the poor detection of SFPs having SNP situated in the flanking 1–5 bases than those present in middle of features/probes <a href="#pone.0000284-Rostoks1">[7]</a>–<a href="#pone.0000284-Cui1">[8]</a>. The above observation is better explained at least by CP&amp;RT dataset where both the number of SFPs with SNP and also ratio of SFPs with SNP to SFPs with multiple NP were comparatively higher than that of CP&amp;LG dataset. We further compared the detection sensitivity of SFPs containing SNP versus SFPs with multiple NP in order to test the biasness of detection, if any. Given the numbers of SFPs as verified by sequencing we observed that SFPs with SNP were detected as efficiently as SFPs with multiple NP (<a href="#pone-0000284-t004">Table 4</a>), an observation contrary to an earlier report <a href="#pone.0000284-Cui1">[8]</a>.</p>
<div class="figure" id="pone-0000284-t004"><div class="img"><a name="pone-0000284-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Nature of polymorphisms and detection sensitivity of SFPs.</span></strong></p><span>doi:10.1371/journal.pone.0000284.t004</span></div>

<h4>SFP comparison</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The objective of present study was to find SFPs in CP&amp;LG and CP&amp;RT datasets as varieties in these two pairs constitute parents of two different mapping populations created to map QTLs/genes associated with milling yields trait <a href="#pone.0000284-1">[10]</a>. The CP and LG belong to japonica <a href="#pone.0000284-Lu1">[9]</a> and RT to indica <a href="#pone.0000284-1">[10]</a> background respectively. However with the available results in the three gene-chip predicted SFPs datasets we estimated frequency of overlapping SFPs. As expected we found significantly higher number of common SFPs between two japonica&amp;indica datasets combinations viz. CP&amp;RT and LG&amp;RT (<a href="#pone-0000284-t005">Table 5b</a>). The above finding indicated not only the occurrence of common variations in the two japonica varieties (CP&amp;LG) against indica variety (RT) but also efficacy of gene-chip method for predicting SFPs in the present study. Although common SFPs were also observed between diverse CP&amp;LG and CP&amp;RT (<a href="#pone-0000284-t005">Table 5a</a>) and also between CP&amp;LG and LG&amp;RT (<a href="#pone-0000284-t005">Table 5c</a>) datasets combinations respectively and expectedly the number of overlapping SFPs were much less. The chi-square test for independence for all three datasets combinations (<a href="#pone-0000284-t005">Table 5</a>) were highly significant showing strong association among the three pairwise comparison</p>
<div class="figure" id="pone-0000284-t005"><div class="img"><a name="pone-0000284-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Overlapping SFPs among gene-chip experiments datasets.</span></strong></p><span>doi:10.1371/journal.pone.0000284.t005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The availability of genome sequence information of Nipponbare <a href="#pone.0000284-IRGSP1">[17]</a> and 93-11 <a href="#pone.0000284-SuperScaffold1">[18]</a> allowed us to predict in-silico SFP candidates between above genetically diverged japonica and indica sub-species of rice respectively. Among the three gene-chips predicted SFPs datasets in the present study, the CP&amp;RT and LG&amp;RT pairs belongs to japonica and indica sub-species combination. The in-silico or computationally predicted SFPs in Nipponbare&amp;93-11 (supplementary <a href="#pone.0000284.s003">Table S3</a>) were compared with gene-chip predicted SFPs, particularly with respect to japonica&amp;indica datasets, in order to estimate overlapping SFPs among them. The significantly higher and comparable number of common polymorphism in two different japonica&amp;indica datasets combination (<a href="#pone-0000284-t006">Table 6b&amp;c</a>) reconfirms our earlier observation of occurrence of common variation between japonica&amp;indica subspecies of rice (<a href="#pone-0000284-t005">Table 5b</a>). In gene-chip predicted SFPs comparison study in japonica&amp;indica datasets combination ~&gt;70% of the total polymorphic SFP were common (<a href="#pone-0000284-t005">Table 5b</a>). This contrast with ~30% of common SFPs observed in gene-chip vs. in-silico studies in two different japonica&amp;indica datasets combination (<a href="#pone-0000284-t006">Table 6b&amp;c</a>). The above discrepancy could be because (a) both japonica &amp; indica varieties were different (b) in-silico predicted SFPs has significantly less number of +SFP compared to −SFP (supplementary <a href="#pone.0000284.s003">Table S3</a>) possibly because of the criteria used to predict +SFP (see <a href="#s4">methods</a>; In silico SFP analyses) and (c) only 384998 features shared by in-silico and gene-chip were considered instead of ~630000 features considered in only gene-chip predicted SFP comparison study. The higher number of common SFPs in gene-chip predicted SFPs of japonica&amp;indica datasets combination (<a href="#pone-0000284-t005">Table 5b</a>) may be due to indica variety (RT) was common in both the datasets and secondly genetic divergence between two japonica varieties viz. CP and LG were low as evidenced by their frequency of polymorphisms (<a href="#pone-0000284-t002">Table 2</a>). From common polymorphisms data of gene-chip vs. in-silico SFPs comparison study of japonica&amp;indica datasets combinations (<a href="#pone-0000284-t006">Table 6b&amp;c</a>), we estimated further the frequency of overlapping SFPs by comparing their common SFPs. The analyses showed that 80–85% of SFPs were still common (<a href="#pone-0000284-t006">Table 6d</a>) in the three different japonica&amp;indica datasets and also there wasn't any overlap with regards to polymorphism directions. This finding also supports the view that irrespective of varieties and the methods (gene-chip or in-silico) adopted for predicting SFPs, there will always be some common variation between japonica&amp;indica varieties as evidenced by significantly higher number of common SFPs when two independently analyzed common polymorphism (<a href="#pone-0000284-t006">Table 6b&amp;c</a>) were compared (<a href="#pone-0000284-t006">Table 6d</a>).</p>
<div class="figure" id="pone-0000284-t006"><div class="img"><a name="pone-0000284-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Overlapping SFPs between datasets of gene-chip experiments and in-silico analyses.</span></strong></p><span>doi:10.1371/journal.pone.0000284.t006</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>All the computationally predicted SFPs (supplementary <a href="#pone.0000284.s003">Table S3</a>), irrespective of number associated with polymorphism direction (+/−), were considered as an example of approximate number of SFPs that can be predicted when the methods presented in the paper is used to derive SFP call between japonica and indica sub-species. Since SFPs in CP&amp;RT pair was validated by sequence information, we compared the numbers of gene-chip predicted SFPs to those predicted by in-silico analyses. From gene-chip method at 9% estimated FDR, 4% of the total perfect match (PM) features present on the array (see <a href="#s4">methods</a>) were polymorphic in CP&amp;RT compared to 10.8% predicted by in-silico analyses in Nipponbare &amp; 93-11 (<a href="#pone-0000284-g004">Figure 4</a>). However by lowering the stringency of estimated FDR to 10% the number of predicted SFPs in CP&amp;RT dataset nearly doubled to 61055 that account 9.6% of the total PM features (<a href="#pone-0000284-t001">Table 1b</a>, <a href="#pone-0000284-g004">Figure 4</a>). Although validation of SFPs predicted at 10% estimated FDR increased the sensitivity of SFP detection considerably but simultaneously the observed FDR determined after sequencing also doubled to 20% compared to the estimated value by permutation (<a href="#pone-0000284-t003">Table 3b</a>). Considering the above results if we exclude ~20% of the total SFP predicted at 10% estimated FDR in CP&amp;RT dataset still ~8% of total features present on the array are polymorphic compared to 10.8% predicted by in-silico analyses in Nipponbare &amp; 93-11. The discrepancy in number of polymorphic features between the CP&amp;RT and Nipponbare&amp;93-11 datasets may be because of expected inherent diversity in their genome.</p>
<div class="figure" id="pone-0000284-g004"><div class="img"><a name="pone-0000284-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000284" data-uri="info:doi/10.1371/journal.pone.0000284.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000284.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000284.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000284.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000284.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Polymorphic features predicted by in-silico analyses and gene-chip experiments. NIP = Nipponbare; CP = Cypress; RT = RT0034 and PM = perfect match.</p>
<span>doi:10.1371/journal.pone.0000284.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The phenotypic variations associated within organisms are products of underlying DNA diversity. Such variations in the nucleotides are great resources for development of molecular markers for mapping genes associated with either qualitative or quantitative traits. The genome wide polymorphism discovery captured as single feature polymorphism (SFP) resulting from differential hybridization of probes is a unique high-throughput approach for both genotyping and polymorphism discovery in a single assay <a href="#pone.0000284-Hazen1">[19]</a>. Such strategy was highly successful in identifying polymorphism in yeast <a href="#pone.0000284-Winzeler1">[5]</a> and Arabidopsis <a href="#pone.0000284-Borevitz1">[6]</a> when probes generated from labeling of g-DNA of two varieties were hybridized to high density oligos expression arrays.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the present study we tested the feasibility of such gene-chip based approach for polymorphism discovery in rice by hybridizing probes, generated from labeling of g-DNA, to rice whole genome expression array (Affymetrix). From verification of sequence information of predicted SFP conducted in two independent datasets viz. CP&amp;RT and CP&amp;LG, we found that SFPs can reliably be predicted in rice with ~75% detection sensitivity (<a href="#pone-0000284-t003">Table 3</a>). Such a high rate of sensitivity is comparable to those reported in Arabidopsis <a href="#pone.0000284-Borevitz1">[6]</a>, <a href="#pone.0000284-Werner1">[20]</a> but certainly more than barley <a href="#pone.0000284-Rostoks1">[7]</a> when similar DNA based method was used to predict SFP. The rice genome (389 Mb) is three times bigger than Arabidopsis (125 Mb) but much smaller than barley (5200 Mb) in size and comparatively higher efficiency of SFP detection in rice may be due greater representation of gene regions in probes as genome is less complex in size than barley. Although we observed considerable increase in detection sensitivity of SFPs at less stringent estimated FDR (<a href="#pone-0000284-t003">Table 3</a>), it was also accompanied by a significant increase in the observed FDR determined after sequencing compared to the estimated value by permutation. Since we couldn't find SFPs with higher d-stat values as false positive in both the datasets and the d-stat value of most of the false positive was close to the permuted cut-off level (data not shown), one can increase the likelihood of getting true SFPs <em>a priori</em> of their sequence analyses by deselecting SFPs having d-stat value close to permuted cut-off. Although a caution must be exercised in such approach as one may loose a considerable number of true SFPs also.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The rice varieties CP and LG are in japonica (tropical) <a href="#pone.0000284-Lu1">[9]</a> while RT in indica <a href="#pone.0000284-1">[10]</a> genetic background respectively. The genetic differences between tropical and temperate japonica are very small <a href="#pone.0000284-Ni1">[16]</a> as compared to high degree of polymorphism observed between indica and japonica sub-species in rice <a href="#pone.0000284-Feltus1">[3]</a>–<a href="#pone.0000284-Shen1">[4]</a>, <a href="#pone.0000284-Garris1">[15]</a>. The gene-chip prediction of ~6-7xs polymorphism in CP&amp;RT or LG&amp;RT compared to CP&amp;LG dataset (<a href="#pone-0000284-t002">Table 2</a>) therefore mirrors the fact that varieties with similar genetic background are less polymorphic than to diverse genetic backgrounds. Additionally the pairs plot study (<a href="#pone-0000284-g002">Figure 2</a>) also supports the above observation as the number of features showing differential hybridization intensity was significantly more in CP&amp;RT or LG&amp;RT than in CP&amp;LG dataset. The sensitivity of SFP discovery is further evidenced by the observation that SFP with SNP was detected as efficiently as SFP with multiple NP (<a href="#pone-0000284-t004">Table 4</a>); an observation in contrast to similar studies in barley <a href="#pone.0000284-Cui1">[8]</a>. Although variations ranging from single to multiple nucleotides were captured with nearly similar efficiency, SFP with SNP at flanking 1–5 bases of 25mer feature was detected poorly than those present in the middle (<a href="#pone-0000284-t004">Table 4</a>), a phenomenon similar to those reported earlier <a href="#pone.0000284-Rostoks1">[7]</a>. The interesting observation of significant number of common SFPs/polymorphism found among different japonica&amp;indica datasets comparison studies (<a href="#pone-0000284-t005">Table 5b</a> and <a href="#pone-0000284-t006">Table 6b,c&amp;d</a>) supports not only effectiveness of gene-chip approach for genome-wide polymorphism discovery but also provides a useful information regarding natural occurrence of common variations between japonica &amp; indica subspecies.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Based on predicted SFPs, traditionally one can generate molecular markers once the fragment flanking SFP of interest is amplified and sequenced. Such type of approach can be useful to further narrow down the genetic interval of already identified QTLs. However given their higher sensitivity of detection (<a href="#pone-0000284-t003">Table 3</a>) together with dense coverage on genome, the SFPs data in rice can be used directly as molecular markers thus obviating the cumbersome process of marker development. With the defined physical location of SFPs on the chromosomes and the ease of generation of high density SFP data as demonstrated in the present study; their direct application as “molecular markers” will help substantially to constrain the genetic intervals containing “favorite genes” to high resolution thus making the prediction of candidate genes feasible provided genes present in the regions are annotated. The direct use of SFPs as “molecular markers” have been demonstrated in mapping genes associated with either qualitative <a href="#pone.0000284-Borevitz1">[6]</a> or quantitative traits <a href="#pone.0000284-Werner1">[20]</a>–<a href="#pone.0000284-Hazen2">[22]</a> in Arabidopsis. In such QTL studies the probes generated from pooled DNA of RILs (recombinant inbred lines) showing extreme of phenotypes were used for hybridization and prediction of SFPs. Based on allelic frequency differences in both extreme pools, QTLs containing candidate genes were mapped with high resolution by extreme array mapping (XAM) in above studies.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Many of the plant traits of economic importance are generally controlled by numerous loci and to fine map genes associated with such traits is not trivial in terms of both time and resources. An alternative approach to speed up the process of gene discovery associated with quantitative traits that complements map-based cloning is association studies where candidate gene diversity is evaluated across natural populations and polymorphisms that correlate with phenotypic variation are identified <a href="#pone.0000284-Buckler1">[23]</a>. The application of such approaches has been well demonstrated in humans by using gene-chip based SNP panels <a href="#pone.0000284-Hinds1">[24]</a>. Since generation of SNP panels require prior sequence information and are quite expensive; the ease of generation of high density polymorphism (SFP) resulting from hybridization of probes to publicly available inexpensive expression array together with similar detection efficiency of SFP having either SNP or multiple NPs has promises that such SFPs data of rice can equally be used for associating functional variations with phenotypes similar to those suggested in Arabidopsis <a href="#pone.0000284-Kim1">[25]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and methods"></a><h3>Materials and methods</h3>
<h4>Plant materials</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The three rice varieties used in this study include Cypress, LaGrue and RT0034; first two belong to japonica while later to indica subspecies respectively. These rice varieties are the parents of two mapping population developed by RiceCAP <a href="#pone.0000284-1">[10]</a> to map QTLs/genes associated with milling yield traits. The RT0034 &amp; Cypress are parents of milling yield 1 (MY1) and Cypress &amp; LaGrue of milling yield 2 (MY2) populations.</p>


<h4>Rice genome array</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The rice genome array (<a href="http://www.affymetrix.com">www.affymetrix.com</a>), designed for gene expression analyses, and contains probes to query 49,824 transcripts representing two rice cultivars with 48,564 from japonica and 1,260 from indica cultivar. The array is believed to represent about 46,000 distinct rice genes however the probesets from japonica is 54,168 and 1347 from indica subspecies. The arrays were designed using NCBI UniGene Build #52, (May7, 2004) incorporating predicted genes from GenBank® and the TIGR Os1 v2 data set (Affymetrix). Each probeset is represented by 11 perfect matches (PM) and an equal number of mismatch (MM) probes/features and each probe is 25 bp long. The array contains ~630,000 PM probes/features from rice. The PM and MM probes positioned next to each other constitute a probe pair and each probe pair is distributed randomly but with defined position on the array or gene-chip.</p>


<h4>Probe generation and hybridization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>DNA was isolated from leaves by CTAB method <a href="#pone.0000284-Lukowitz1">[26]</a>. RNase treated DNA was phenol purified and dissolved in nuclease free water. The purified DNA was subjected to generation of biotin (biotin-dCTP) labeled probes by bio-prime labeling kit (Invitrogen). The reaction condition and parameters were same as described in Borevitz's methods paper <a href="#pone.0000284-Borevitz2">[12]</a> except that 400ng of purified DNA was used as template per reaction. The reaction was allowed to proceed for 16 hours at 20°C after which the product was ethanol purified and dissolved in 100 µl of nuclease free water. The amplified products were approximately ~80–100 bp in size as verified on 1% agarose gel. From each labeling reaction 28–30 µg of products were obtained as quantified by nano-drop methods using default parameter set for DNA measurements. For each sample approximately 28 µg of reaction products generated from single reaction were used for hybridization to Affymetrix rice expression array at DNA core facility (<a href="http://www.biotech.missouri.edu/dnacore/">http://www.biotech.missouri.edu/dnacore/</a>). The hybridization and washing was performed according to standard RNA protocol as described in Affymetrix manual.</p>


<h4>Data analyses</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The data quality was assessed from expression report generated by GCOS1.3 software using the default parameter set for rice genome array. The raw intensity values of all the probes present on the chip was transferred from .cel files into .txt file by using the tools available on GCOS1.3 software. The intensity values of only PM (perfect match) probes/features (628,551) were subsequently extracted and subjected to data analyses as described <a href="#pone.0000284-Borevitz2">[12]</a>–<a href="#pone.0000284-Supplementary1">[13]</a>. Briefly, intensity data was preprocessed by RMA2 and quantile normalized using Affy package (<a href="http://www.bioconductor.org">www.bioconductor.org</a>). The log2 transformed intensity value of each feature was subsequently used to derive SFP call by publicly available <em>siggene</em> package (<a href="http://www.bioconductor.org">www.bioconductor.org</a>) and scripts and codes <a href="#pone.0000284-Borevitz2">[12]</a> for such statistical analyses.</p>


<h4>PCR and sequencing</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>DNA fragments of 200–300 bp flanking the SFP regions were amplified from genomic DNA. The sequence information of features/probes was obtained from Affymetrix (<a href="http://www.affymetrix.com">www.affymetrix.com</a>). Primers were designed on the basis of known sequence information of Nipponbare genome (TIGRv3) by using the blast tool of <em>Gramene</em> (<a href="http://www.gramene.org">www.gramene.org</a>). The thermal cycle program for PCR were 95°C for 2 min, 28 cycles of 95°C for 15 sec, 58°C for 15 sec, 72°C for 30 sec and with final extension at 72°C for 5 min. The PCR conditions were; 1× of 10× Ex Taq buffer (Mg<sup>+</sup>), 125 µM of each dNTPs, 0.5 units of DNA polymerase, 0.5 µM each primer, 1 µl of genomic DNA to a final volume of 40 µl. In order to get DNA fragments without any error during amplification, PCR was performed with hot-start Ex Taq DNA polymerase (Panvera US, Madision WI) having proof reading activity. PCR products showing single bands were purified (PCR purification kit, Qiagen) and subjected to sequencing by Applied Biosystems 3730 DNA Analyzer using Prism Big Dye Terminator cycle sequencing chemistry at DNA core facility.</p>


<h4>SFP confirmation</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>For sequence validation only those SFPs were considered whose corresponding probe/feature sequence had unique location and perfect match (PM) on Nipponbare genome (TIGRv3). In the present study any variation in sequence ranging from substitution or indel involving single (SNPs) to multiple bases were considered while determining the nature of polymorphism associated with SFPs. High quality single reads, as obtained from ABI chromatograms, were used for sequence comparison with publicly available vectorNTI tools (Invitrogen).</p>


<h4>In silico SFP analyses</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The Affymetrix rice whole genome expression array has been designed mainly on the sequence from japonica sub-species cv. Nipponbare. The sequences of all ~630,000 25mer PM probes/features sequences present on the array were compared (megablast and E-value: 1e-4) against Nipponbare genome build 4<a href="#pone.0000284-IRGSP1">[17]</a> to remove features that were repetitive and not perfect matches. From above analyses we identified 382205 features that were perfect match (100% identity, 25/25) and each feature was having unique location on genome (supplementary <a href="#pone.0000284.s003">Table S3</a>). When sequences of these unique features were compared (megablast and E-value: 1e-4) against 93-11 genome <a href="#pone.0000284-SuperScaffold1">[18]</a>, altogether 64883 features showing changes ranging from single to multiple nucleotides and also no matches were identified and were considered together as in-silico predicted −SFP candidates (i.e. Nip&gt;93-11) and rest as non-SFPs (Nip = 93-11). The sequences of remaining features (25mer) from above analyses that didn't show 100% identity on Nipponbare genome, were compared (megablast and E-value: 1e-4) against 93-11 genome and information of unique features showing 100% identity (25/25) and also single location on genome were extracted. When sequences of these unique features were compared (megablast and E-value: 1e-4) against Nipponbare genome, information of 3344 features showing not only variations in their sequences but also having unique location were extracted and were considered as in-silico predicted +SFP candidates (i.e. 93-11&gt;Nip). In determining SFPs (+/−), we considered their unique position with reference to Nipponbare genome only.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000284"></div><a name="pone.0000284.s001" id="pone.0000284.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000284.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Sequence verified SFPs in CP&amp; LG dataset. Changes in sequences are highlighted in blue and black; - indicates deletion; CP = Cypress and LG = LaGrue.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000284.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0000284.s002" id="pone.0000284.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000284.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Sequence verified SFPs in CP&amp;RT dataset. Changes in sequences are highlighted in blue and black; - indicates deletion; CP = Cypress and RT = RT0034.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000284.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
<a name="pone.0000284.s003" id="pone.0000284.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000284.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">In-silico predicted SFPs and their distribution. Nip = Nipponbare, −SFP(Nip&gt;93-11), +SFP (93-11&gt;Nip), non SFP (Nip = 93-11)</p>
<p class="siDoi">doi:10.1371/journal.pone.0000284.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are thankful to DNA Core facility; University of Missouri-Columbia for performing microarray and sequencing. The help from Tonya Mueller, Division of Plant Science, University of Missouri-Columbia in preparing figures is highly appreciated. We thank to staff of Sears Green house facility at University of Missouri-Columbia for growing rice plants.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RK HN. Performed the experiments: RK. Analyzed the data: RK JQ TJ. Wrote the paper: RK HN. Other: Conducted computational analyses: TJ DX. Conducted statistical analyses and provided critical comments on paper: JQ. Provided technical help in microarray work: BV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000284-International1" id="pone.0000284-International1"></a>International Rice Genome Sequencing Project (2005) The map-based sequence of the rice genome. Nature  436: 793–800.  <ul class="find" data-citedArticleID="936568" data-doi="10.1038/nature03895"><li><a href="http://dx.doi.org/10.1038/nature03895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+map-based+sequence+of+the+rice+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+map-based+sequence+of+the+rice+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000284-Bevan1" id="pone.0000284-Bevan1"></a>Bevan M, Murphy G (1999) The small, the large and the wild: the value of comparison in plant genomics. Trends Genet  15: 211–214.  <ul class="find" data-citedArticleID="936546" data-doi="10.1016/s0168-9525(99)01744-8"><li><a href="http://dx.doi.org/10.1016/s0168-9525(99)01744-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+small%2C+the+large+and+the+wild%3A+the+value+of+comparison+in+plant+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+small%2C+the+large+and+the+wild%3A+the+value+of+comparison+in+plant+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000284-Feltus1" id="pone.0000284-Feltus1"></a>Feltus FA, Wan J, Schulze SR, Estill JC, Jiang N, et al.  (2004) An SNP resource for rice genetics and breeding based on subspecies indica and japonica genome alignments. Genome Res  14: 1812–1819.  <ul class="find" data-citedArticleID="936556" data-doi="10.1101/gr.2479404"><li><a href="http://dx.doi.org/10.1101/gr.2479404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+SNP+resource+for+rice+genetics+and+breeding+based+on+subspecies+indica+and+japonica+genome+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+SNP+resource+for+rice+genetics+and+breeding+based+on+subspecies+indica+and+japonica+genome+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000284-Shen1" id="pone.0000284-Shen1"></a>Shen YJ, Jiang H, Jin JP, Zhang ZB, Xi B, et al.  (2004) Development of genome-wide DNA polymorphism database for map-based cloning of rice genes. Plant Physiol  135: 1198–1205.  <ul class="find" data-citedArticleID="936580" data-doi="10.1104/pp.103.038463"><li><a href="http://dx.doi.org/10.1104/pp.103.038463" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+genome-wide+DNA+polymorphism+database+for+map-based+cloning+of+rice+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+genome-wide+DNA+polymorphism+database+for+map-based+cloning+of+rice+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000284-Winzeler1" id="pone.0000284-Winzeler1"></a>Winzeler EA, Castillo-Davis CI, Oshiro G, Liang D, Richards DR, et al.  (2003) Genetic diversity in yeast assessed with whole-genome oligonucleotide arrays. Genetics  163: 79–89.  <ul class="find" data-citedArticleID="936590"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays.&amp;auth=&amp;atitle=Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000284-Borevitz1" id="pone.0000284-Borevitz1"></a>Borevitz JO, Liang D, Plouffe D, Chang HS, Zhu T, et al.  (2003) Large-scale identification of single-feature polymorphisms in complex genomes. Genome Res  13: 513–523.  <ul class="find" data-citedArticleID="936548" data-doi="10.1101/gr.541303"><li><a href="http://dx.doi.org/10.1101/gr.541303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+identification+of+single-feature+polymorphisms+in+complex+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+identification+of+single-feature+polymorphisms+in+complex+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000284-Rostoks1" id="pone.0000284-Rostoks1"></a>Rostoks N, Borevitz JO, Hedley PE, Russell J, Mudie S, et al.  (2005) Single-feature polymorphism discovery in the barley transcriptome. Genome Biol  6: R54.  <ul class="find" data-citedArticleID="936578" data-doi="10.1186/gb-2005-6-6-r54"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-6-r54" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-feature+polymorphism+discovery+in+the+barley+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-feature+polymorphism+discovery+in+the+barley+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000284-Cui1" id="pone.0000284-Cui1"></a>Cui X, Xu J, Asghar R, Condamine P, Svensson JT, et al.  (2005) Detecting single-feature polymorphisms using oligonucleotide arrays and robustified projection pursuit. Bioinformatics  21: 3852–3858.  <ul class="find" data-citedArticleID="936554" data-doi="10.1093/bioinformatics/bti640"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti640" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+single-feature+polymorphisms+using+oligonucleotide+arrays+and+robustified+projection+pursuit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+single-feature+polymorphisms+using+oligonucleotide+arrays+and+robustified+projection+pursuit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000284-Lu1" id="pone.0000284-Lu1"></a>Lu H, Redus MA, Coburn JR, Rutger JN, McCouch SR, Tai TH (2005) Population Structure and Breeding Patterns of 145 U.S. Rice Cultivars Based on SSR Marker Analysis. Crop Sci  45: 66–76.  <ul class="find" data-citedArticleID="936572" data-doi="10.2135/cropsci2005.0066"><li><a href="http://dx.doi.org/10.2135/cropsci2005.0066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+Structure+and+Breeding+Patterns+of+145+U.S.+Rice+Cultivars+Based+on+SSR+Marker+Analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+Structure+and+Breeding+Patterns+of+145+U.S.+Rice+Cultivars+Based+on+SSR+Marker+Analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000284-1" id="pone.0000284-1"></a>RiceCAP project.  Available: <a href="http://www.ricecap.uark.edu/">http://www.ricecap.uark.edu/</a>.  <ul class="find" data-citedArticleID="936542"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RiceCAP+project.&amp;auth=&amp;atitle=RiceCAP+project." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RiceCAP+project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RiceCAP+project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000284-Aluko1" id="pone.0000284-Aluko1"></a>Aluko G, Martinez C, Tohme J, Castano C, Bergman C, et al.  (2004) QTL mapping of grain quality traits from the interspecific cross <em>Oryza sativa</em> x <em>O. glaberrima</em>. Theor Appl Genet  109: 630–639.  <ul class="find" data-citedArticleID="936544" data-doi="10.1007/s00122-004-1668-y"><li><a href="http://dx.doi.org/10.1007/s00122-004-1668-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=QTL+mapping+of+grain+quality+traits+from+the+interspecific+cross+Oryza+sativa+x+O.+glaberrima." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22QTL+mapping+of+grain+quality+traits+from+the+interspecific+cross+Oryza+sativa+x+O.+glaberrima.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000284-Borevitz2" id="pone.0000284-Borevitz2"></a>Borevitz J (2005) Array Genotyping and Mapping.  Available: <a href="http://naturalvariation.org/">http://naturalvariation.org/</a>.  <ul class="find" data-citedArticleID="936550" data-doi="10.1385/1-59745-003-0:137"><li><a href="http://dx.doi.org/10.1385/1-59745-003-0:137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Array+Genotyping+and+Mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Array+Genotyping+and+Mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000284-Supplementary1" id="pone.0000284-Supplementary1"></a>  Supplementary data for Single Feature Polymorphism discovery in the barley transcriptome. Available: <a href="http://naturalvariation.org/barley">http://naturalvariation.org/barley</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000284-Tusher1" id="pone.0000284-Tusher1"></a>Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci  98: 5116–21.  <ul class="find" data-citedArticleID="936586" data-doi="10.1073/pnas.091062498"><li><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000284-Garris1" id="pone.0000284-Garris1"></a>Garris AJ, Tai TH, Coburn J, Kresovich S, McCouch S (2005) Genetic structure and diversity in <em>Oryza sativa</em> L. Genetics  169: 1631–1638.  <ul class="find" data-citedArticleID="936558" data-doi="10.1534/genetics.104.035642"><li><a href="http://dx.doi.org/10.1534/genetics.104.035642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+and+diversity+in+Oryza+sativa+L." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+and+diversity+in+Oryza+sativa+L.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000284-Ni1" id="pone.0000284-Ni1"></a>Ni J, Colowit PM, Mackill DJ (2002) Evaluation of Genetic Diversity in Rice Subspecies Using Microsatellite Markers. Crop Sci  42: 601–607.  <ul class="find" data-citedArticleID="936576" data-doi="10.2135/cropsci2002.0601"><li><a href="http://dx.doi.org/10.2135/cropsci2002.0601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+Genetic+Diversity+in+Rice+Subspecies+Using+Microsatellite+Markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+Genetic+Diversity+in+Rice+Subspecies+Using+Microsatellite+Markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000284-IRGSP1" id="pone.0000284-IRGSP1"></a>IRGSP Releases Build 4.0 Pseudomolecules of the Rice Genome (2005)   Available: <a href="http://rgp.dna.affrc.go.jp/E/IRGSP/Build4/build4.html">http://rgp.dna.affrc.go.jp/E/IRGSP/Build​4/build4.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000284-SuperScaffold1" id="pone.0000284-SuperScaffold1"></a>SuperScaffold Statistics of <em>Oryza sativa</em> spp. indica (93-11) (2003)   Available: <a href="http://rice.genomics.org.cn/statistics/jsp/SuperScaffold/SuperScaffoldTotal.jsp?organism%E2%80%8A=%E2%80%8A9311">http://rice.genomics.org.cn/statistics/j​sp/SuperScaffold/SuperScaffoldTotal.jsp?​organism= 9311</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0000284-Hazen1" id="pone.0000284-Hazen1"></a>Hazen SP, Kay SA (2003) Gene arrays are not just for measuring gene expression. Trends Plant Sci  8: 413–416.  <ul class="find" data-citedArticleID="936560" data-doi="10.1016/s1360-1385(03)00186-9"><li><a href="http://dx.doi.org/10.1016/s1360-1385(03)00186-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+arrays+are+not+just+for+measuring+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+arrays+are+not+just+for+measuring+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000284-Werner1" id="pone.0000284-Werner1"></a>Werner JD, Borevitz JO, Uhlenhaut NH, Ecker JR, Chory J, Weigel D (2005) FRIGIDA-independent variation in flowering time of natural <em>Arabidopsis thaliana</em> accessions. Genetics  170: 1197–1207.  <ul class="find" data-citedArticleID="936588" data-doi="10.1534/genetics.104.036533"><li><a href="http://dx.doi.org/10.1534/genetics.104.036533" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FRIGIDA-independent+variation+in+flowering+time+of+natural+Arabidopsis+thaliana+accessions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FRIGIDA-independent+variation+in+flowering+time+of+natural+Arabidopsis+thaliana+accessions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000284-Wolyn1" id="pone.0000284-Wolyn1"></a>Wolyn DJ, Borevitz JO, Loudet O, Schwartz C, Maloof J, et al.  (2004) Light-response quantitative trait loci identified with composite interval and eXtreme array mapping in <em>Arabidopsis thaliana</em>. Genetics  167: 907–917.  <ul class="find" data-citedArticleID="936592" data-doi="10.1534/genetics.103.024810"><li><a href="http://dx.doi.org/10.1534/genetics.103.024810" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Light-response+quantitative+trait+loci+identified+with+composite+interval+and+eXtreme+array+mapping+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Light-response+quantitative+trait+loci+identified+with+composite+interval+and+eXtreme+array+mapping+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000284-Hazen2" id="pone.0000284-Hazen2"></a>Hazen SP, Borevitz JO, Harmon FG, Pruneda-Paz JL, Schultz TF, et al.  (2005) Rapid array mapping of circadian clock and developmental mutations in Arabidopsis. Plant Physiol  138: 990–997.  <ul class="find" data-citedArticleID="936562" data-doi="10.1104/pp.105.061408"><li><a href="http://dx.doi.org/10.1104/pp.105.061408" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+array+mapping+of+circadian+clock+and+developmental+mutations+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+array+mapping+of+circadian+clock+and+developmental+mutations+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000284-Buckler1" id="pone.0000284-Buckler1"></a>Buckler ES 4th, Thornsberry JM (2002) Plant molecular diversity and applications to genomics. Curr Opin Plant Biol  5: 107–111.  <ul class="find" data-citedArticleID="936552" data-doi="10.1016/s1369-5266(02)00238-8"><li><a href="http://dx.doi.org/10.1016/s1369-5266(02)00238-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+molecular+diversity+and+applications+to+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+molecular+diversity+and+applications+to+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000284-Hinds1" id="pone.0000284-Hinds1"></a>Hinds DA, Stuve LL, Nilsen GB, Halperin E, Eskin E, et al.  (2005) Whole-genome patterns of common DNA variation in three human populations. Science  307: 1072–1079.  <ul class="find" data-citedArticleID="936564" data-doi="10.1126/science.1105436"><li><a href="http://dx.doi.org/10.1126/science.1105436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+patterns+of+common+DNA+variation+in+three+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+patterns+of+common+DNA+variation+in+three+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000284-Kim1" id="pone.0000284-Kim1"></a>Kim S, Zhao K, Jiang R, Molitor J, Borevitz J, et al.  (2006) Association mapping with Single-Feature Polymorphisms. Genetics  173: 1125–1133.  <ul class="find" data-citedArticleID="936570" data-doi="10.1534/genetics.105.052720"><li><a href="http://dx.doi.org/10.1534/genetics.105.052720" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+mapping+with+Single-Feature+Polymorphisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+mapping+with+Single-Feature+Polymorphisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000284-Lukowitz1" id="pone.0000284-Lukowitz1"></a>Lukowitz W, Gillmor CS, Scheible WR (2000) Positional cloning in Arabidopsis. Why it feels good to have a genome initiative working for you. Plant Physiol  123: 795–806.  <ul class="find" data-citedArticleID="936574" data-doi="10.1104/pp.123.3.795"><li><a href="http://dx.doi.org/10.1104/pp.123.3.795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positional+cloning+in+Arabidopsis.+Why+it+feels+good+to+have+a+genome+initiative+working+for+you." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positional+cloning+in+Arabidopsis.+Why+it+feels+good+to+have+a+genome+initiative+working+for+you.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.XML" value="110026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.PDF" value="309637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g001.PNG_L" value="304054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g001.PNG_M" value="98042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g001.PNG_S" value="3949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g001.TIF" value="505446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g001.PNG_I" value="46700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g002.PNG_L" value="696239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g002.PNG_M" value="138838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g002.PNG_S" value="5515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g002.TIF" value="896440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g002.PNG_I" value="71822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t001.PNG_L" value="126287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t001.PNG_M" value="47342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t001.PNG_S" value="2854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t001.TIF" value="548244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t001.PNG_I" value="20567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g003.PNG_L" value="1378801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g003.PNG_M" value="55300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g003.PNG_S" value="2513"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g003.TIF" value="1950142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g003.PNG_I" value="26739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t002.PNG_L" value="125621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t002.PNG_M" value="135263"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t002.PNG_S" value="6470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t002.TIF" value="576386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t002.PNG_I" value="61263"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t003.PNG_L" value="94410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t003.PNG_M" value="79328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t003.PNG_S" value="4675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t003.TIF" value="527380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t003.PNG_I" value="42361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t004.PNG_L" value="100713"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t004.PNG_M" value="93903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t004.PNG_S" value="5056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t004.TIF" value="530654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t004.PNG_I" value="45988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t005.PNG_L" value="82208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t005.PNG_M" value="36301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t005.PNG_S" value="2375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t005.TIF" value="384670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t005.PNG_I" value="15240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t006.PNG_L" value="200360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t006.PNG_M" value="72228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t006.PNG_S" value="3361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t006.TIF" value="933642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.t006.PNG_I" value="31717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g004.PNG_L" value="734452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g004.PNG_M" value="109797"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g004.PNG_S" value="3169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g004.TIF" value="1227056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.g004.PNG_I" value="47086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.s001.XLS" value="31232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.s002.XLS" value="54784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000284.s003.XLS" value="15872"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000284&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000284" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000284&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000284&volume=&issue=&title=Single Feature Polymorphism Discovery in Rice&author_name=Rajesh%20Kumar%2C%20Jing%20Qiu%2C%20Trupti%20Joshi%2C%20Babu%20Valliyodan%2C%20Dong%20Xu%2C%20Henry%20T.%20Nguyen&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000284" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284&amp;t=Single%20Feature%20Polymorphism%20Discovery%20in%20Rice" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284&title=Single%20Feature%20Polymorphism%20Discovery%20in%20Rice&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284&amp;title=Single%20Feature%20Polymorphism%20Discovery%20in%20Rice" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000284&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Single Feature Polymorphism Discovery in Rice';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000284';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Single%20Feature%20Polymorphism%20Discovery%20in%20Rice http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000284" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000284" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="23406"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="23406"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23406"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="23406"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40539" data-articleid="23406"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="48355" data-articleid="23406"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rice%22" title="Search for articles in the subject area:'Rice'"><div class="flagText">Rice</div></a>
              <div data-categoryid="31915" data-articleid="23406"
                   data-categoryname="Rice"
                   class="flagImage" title="Flag 'Rice' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="23406"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6259'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2352'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3858&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
