

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Modern Mode of Activation for Nucleic Acid Enzymes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000673"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000673" />

    <meta name="citation_title" content="A Modern Mode of Activation for Nucleic Acid Enzymes"/>
    <meta itemprop="name" content="A Modern Mode of Activation for Nucleic Acid Enzymes"/>

      <meta name="citation_author" content="Dominique Lévesque"/>
            <meta name="citation_author_institution" content="RNA Group/Groupe ARN, Département de Biochimie, Faculté de médecine et des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada"/>
      <meta name="citation_author" content="Francis P. Brière"/>
            <meta name="citation_author_institution" content="RNA Group/Groupe ARN, Département de Biochimie, Faculté de médecine et des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada"/>
      <meta name="citation_author" content="Jean-Pierre Perreault"/>
            <meta name="citation_author_institution" content="RNA Group/Groupe ARN, Département de Biochimie, Faculté de médecine et des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada"/>

    <meta name="citation_date" content="2007/7/25"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000673.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e673"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Structure and mechanism in protein science.; citation_author=A Fersht; citation_number=1; citation_pages=289-318; citation_date=1999; citation_publisher=WH Freeman and Company; " />
      <meta name="citation_reference" content="citation_title=Control of gene expression by a natural metabolite-responsive ribozyme.; citation_author=WC Winkler; citation_author=A Nahvi; citation_author=A Roth; citation_author=JA Collins; citation_author=RR Breaker; citation_journal_title=Nature; citation_volume=428; citation_number=2; citation_pages=281-286; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Backbone and nucleobase contacts to glucosamine-6-phosphate in the glmS ribozyme.; citation_author=JA Jansen; citation_author=TJ McCarthy; citation_author=GA Soukup; citation_author=JK Soukup; citation_journal_title=Nature Struct Mol Biol; citation_volume=13; citation_number=3; citation_pages=517-523; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Signalling specificity of Ser/Thr protein kinases through docking-site-mediated interactions.; citation_author=RM Biondi; citation_author=AR Nebreda; citation_journal_title=Biochem J; citation_volume=372; citation_number=4; citation_pages=1-13; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular recognitions in the MAP kinase cascades.; citation_author=T Tanoue; citation_author=E Nishida; citation_journal_title=Cell Signalling; citation_volume=15; citation_number=5; citation_pages=455-462; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Target-dependent on/off switch increases ribozyme fidelity.; citation_author=L Bergeron; citation_author=JP Perreault; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=6; citation_pages=1240-1248; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Functional characterization of the SOFA delta ribozyme.; citation_author=L Bergeron; citation_author=C Reymond; citation_author=JP Perreault; citation_journal_title=RNA; citation_volume=11; citation_number=7; citation_pages=1858-1868; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Delta ribozyme benefits from a good stability in vitro that becomes outstanding in vivo.; citation_author=D Lévesque; citation_author=S Choufani; citation_author=JP Perreault; citation_journal_title=RNA; citation_volume=8; citation_number=8; citation_pages=464-477; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The structure-function dilemma of the hammerhead ribozyme.; citation_author=KF Blount; citation_author=OC Uhlenbeck; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=34; citation_number=9; citation_pages=415-440; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In vitro selection, characterization, and application of deoxyribozymes that cleave RNA.; citation_author=SK Silverman; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=10; citation_pages=6151-6163; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Folding and catalysis of the hairpin ribozyme.; citation_author=TJ Wilson; citation_author=M Nahas; citation_author=T Ha; citation_author=DM Lilley; citation_journal_title=Biochem Soc Trans; citation_volume=33; citation_number=11; citation_pages=461-465; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Intracellular cleavage of hepatitis C virus RNA and inhibition of viral protein translation by hammerhead ribozymes.; citation_author=N Sakamoto; citation_author=CH Wu; citation_author=GY Wu; citation_journal_title=J Clin Invest; citation_volume=98; citation_number=12; citation_pages=2720-2728; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Gaining target access for deoxyribozymes.; citation_author=S Schubert; citation_author=JP Furste; citation_author=D Werk; citation_author=HP Grunert; citation_author=H Zeichhardt; citation_journal_title=J Mol Biol; citation_volume=339; citation_number=13; citation_pages=335-363; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Construction of hairpin ribozymes with a three-way junction.; citation_author=Y Komatsu; citation_author=M Shirai; citation_author=S Yamashita; citation_author=E Ohtsuka; citation_journal_title=Bioorg Med Chem; citation_volume=5; citation_number=14; citation_pages=1063-1069; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=In vitro selection of second site revertants analysis of the hairpin ribozyme active site.; citation_author=B Sargueil; citation_author=KJ Hampel; citation_author=D Lambert; citation_author=JM Burke; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=15; citation_pages=52783-52791; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Two independently selected capping ribozymes share similar substrate requirements.; citation_author=HS Zaher; citation_author=RA Watkins; citation_author=PJ Unrau; citation_journal_title=RNA; citation_volume=12; citation_number=16; citation_pages=1949-1958; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A deoxyribozyme-based molecular automaton.; citation_author=MN Stojanovic; citation_author=D Stefanovic; citation_journal_title=Nature Biotech; citation_volume=21; citation_number=17; citation_pages=1069-1074; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational design and experimental validation of oligonucleotide-sensing allosteric ribozymes.; citation_author=R Penchovsky; citation_author=RR Breaker; citation_journal_title=Nature Biotech; citation_volume=23; citation_number=18; citation_pages=1424-1433; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Synthetic RNA circuits.; citation_author=EA Davidson; citation_author=AD Ellington; citation_journal_title=Nature Chem Biol; citation_volume=3; citation_number=19; citation_pages=23-28; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The RNA moiety of ribonuclease P is the catalytic subunit of the enzyme.; citation_author=C Guerrier-Takada; citation_author=K Gardiner; citation_author=T Marsh; citation_author=N Pace; citation_author=S Altman; citation_journal_title=Cell; citation_volume=35; citation_number=20; citation_pages=849-857; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Unusual resistance of peptidyl transferase to protein extraction procedures.; citation_author=HF Noller; citation_author=V Hoffarth; citation_author=L Zimniak; citation_journal_title=Science; citation_volume=256; citation_number=21; citation_pages=1416-1419; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Splicing-related catalysis by protein-free snRNAs.; citation_author=S Valadkhan; citation_author=JL Manley; citation_journal_title=Nature; citation_volume=413; citation_number=22; citation_pages=701-707; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The RNA of both polarities of the peach latent mosaic viroid self-cleaves in vitro solely by single hammerhead structures.; citation_author=D Beaudry; citation_author=F Bussière; citation_author=F Lareau; citation_author=C Lessard; citation_author=JP Perreault; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=23; citation_pages=745-752; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A novel high throughput screening assay for HCV NS3 helicase activity.; citation_author=MH Alaoui-Ismaili; citation_author=C Gervais; citation_author=S Brunette; citation_author=G Gouin; citation_author=M Hamel; citation_journal_title=Antiviral Res; citation_volume=46; citation_number=24; citation_pages=181-193; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Modern Mode of Activation for Nucleic Acid Enzymes"/>
    <meta name="twitter:description" content="Through evolution, enzymes have developed subtle modes of activation in order to ensure the sufficiently high substrate specificity required by modern cellular metabolism. One of these modes is the use of a target-dependent module (i.e. a docking domain) such as those found in signalling kinases. Upon the binding of the target to a docking domain, the substrate is positioned within the catalytic site. The prodomain acts as a target-dependent module switching the kinase from an off state to an on state. As compared to the allosteric mode of activation, there is no need for the presence of a third partner. None of the ribozymes discovered to date have such a mode of activation, nor does any other known RNA. Starting from a specific on/off adaptor for the hepatitis delta virus ribozyme, that differs but has a mechanism reminiscent of this signalling kinase, we have adapted this mode of activation, using the techniques of molecular engineering, to both catalytic RNAs and DNAs exhibiting various activities. Specifically, we adapted three cleaving ribozymes (hepatitis delta virus, hammerhead and hairpin ribozymes), a cleaving 10-23 deoxyribozyme, a ligating hairpin ribozyme and an artificially selected capping ribozyme. In each case, there was a significant gain in terms of substrate specificity. Even if this mode of control is unreported for natural catalytic nucleic acids, its use needs not be limited to proteinous enzymes. We suggest that the complexity of the modern cellular metabolism might have been an important selective pressure in this evolutionary process."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000673.g003"/>

  <meta property="og:title" content="A Modern Mode of Activation for Nucleic Acid Enzymes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2322'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3140'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2653&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000673">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000673" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000673&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Modern Mode of Activation for Nucleic Acid Enzymes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dominique Lévesque
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Dominique Lévesque, Francis P. Brière</p>

              
              

                <p>Affiliation:
                  RNA Group/Groupe ARN, Département de Biochimie, Faculté de médecine et des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francis P. Brière
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Dominique Lévesque, Francis P. Brière</p>

              
              

                <p>Affiliation:
                  RNA Group/Groupe ARN, Département de Biochimie, Faculté de médecine et des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Pierre Perreault
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Jean-Pierre.Perreault@usherbrooke.ca">Jean-Pierre.Perreault@usherbrooke.ca</a></p>

                <p>Affiliation:
                  RNA Group/Groupe ARN, Département de Biochimie, Faculté de médecine et des sciences de la santé, Université de Sherbrooke, Sherbrooke, Québec, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 25, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000673</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000673-g001" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000673-t001" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000673-g002" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000673-g003" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000673">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000673" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Through evolution, enzymes have developed subtle modes of activation in order to ensure the sufficiently high substrate specificity required by modern cellular metabolism. One of these modes is the use of a target-dependent module (i.e. a docking domain) such as those found in signalling kinases. Upon the binding of the target to a docking domain, the substrate is positioned within the catalytic site. The prodomain acts as a target-dependent module switching the kinase from an <em>off</em> state to an <em>on</em> state. As compared to the allosteric mode of activation, there is no need for the presence of a third partner. None of the ribozymes discovered to date have such a mode of activation, nor does any other known RNA. Starting from a specific on/off adaptor for the hepatitis delta virus ribozyme, that differs but has a mechanism reminiscent of this signalling kinase, we have adapted this mode of activation, using the techniques of molecular engineering, to both catalytic RNAs and DNAs exhibiting various activities. Specifically, we adapted three cleaving ribozymes (hepatitis delta virus, hammerhead and hairpin ribozymes), a cleaving 10-23 deoxyribozyme, a ligating hairpin ribozyme and an artificially selected capping ribozyme. In each case, there was a significant gain in terms of substrate specificity. Even if this mode of control is unreported for natural catalytic nucleic acids, its use needs not be limited to proteinous enzymes. We suggest that the complexity of the modern cellular metabolism might have been an important selective pressure in this evolutionary process.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lévesque D, Brière FP, Perreault J-P (2007) A Modern Mode of Activation for Nucleic Acid Enzymes. PLoS ONE 2(7):
          e673.
            doi:10.1371/journal.pone.0000673</p><p><strong>Academic Editor: </strong>Sebastian Fugmann, National Institute on Aging, United States of America</p><p><strong>Received:</strong> June 5, 2007; <strong>Accepted:</strong> June 17, 2007; <strong>Published:</strong> July 25, 2007</p><p><strong>Copyright:</strong> © 2007 Lévesque et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the Canadian Institute of Health Research (CIHR; MOP-44002 and EOP-38322) to J.P.P. The RNA group is supported by grants from the Université de Sherbrooke and the CIHR (PRG-80169).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Enzymes have evolved various versions of a “molecular switch” with which they control their catalytic activities. For example, regulatory proteins bind effectors that induce conformational changes that control catalytic activity <a href="#pone.0000673-Fersht1">[1]</a>. Similar modes of activation have been adopted by catalytic RNA motifs such as the <em>glm</em>S riboswitch, a ribozyme that regulates gene expression in a unique fashion through autocatalytic cleavage upon its binding to glucosamine-6-phosphate <a href="#pone.0000673-Winkler1">[2]</a>, <a href="#pone.0000673-Jansen1">[3]</a>. Mitogen-activated protein kinases (MAP kinases) have evolved a similar, but more subtle, mode of activation <a href="#pone.0000673-Biondi1">[4]</a>, <a href="#pone.0000673-Tanoue1">[5]</a>. Upon the binding of the target to a docking domain, the substrate is positioned within the catalytic site. The prodomain acts as a target-dependent module switching the kinase from an <em>off</em> state to an <em>on</em> state, a property which considerably improves the substrate specificity of the enzyme. None of the ribozymes discovered to date exhibit such a mode of activation, nor does any other known RNA. However, a Specific <em>On</em>/<em>ofF</em> Adaptor (SOFA) module, which includes both a blocker and a biosensor domain, has been developed for the hepatitis delta virus (HDV) ribozyme <a href="#pone.0000673-Bergeron1">[6]</a>, <a href="#pone.0000673-Bergeron2">[7]</a>. The rationale behind this design was that in the absence of the target, the ribozyme should be turned <em>off</em> by the addition of a blocker domain (bl) that forms an intramolecular stem with a portion of the substrate recognition domain (stem P1; <a href="#pone-0000673-g001">Figure 1A</a>). Conversely, a biosensor domain (BS) must bind its complementary sequence on the substrate in order to unlock the catalytic core (turning <em>on</em>). In the SOFA module the blocker and biosensor are juxtaposed, which is not the case for the domain involved in the activation of MAP kinases. In the present work, we have adapted this mode of activation, using the techniques of molecular biology, to a wide range of catalysis types exhibited by nucleic acids.</p>
<div class="figure" id="pone-0000673-g001"><div class="img"><a name="pone-0000673-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000673.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000673.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>SOFA module on the HDV ribozyme.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Secondary structure of both the <em>off</em> and <em>on</em> conformation of the SOFA module (in grey) engineered on the P2 stem of the HDV ribozyme. The biosensor (BS, in green) and the blocker (bl, in red) are juxtaposed. (B) Schematic representation of separated blocker and biosensor domains modulating the activity of a ribozyme in a substrate-dependent manner. (C) Autoradiogram of cleavage assays performed with various HDV ribozymes. Their structures, with both the biosensor (green) and the blocker (red), are illustrated above the lanes of the gel. The control (-) was performed in the absence of ribozyme (lane 1), while lane 2 was performed in the presence of the original version. Lane 3 is the version extended by the blocker sequence only. Lanes 4 and 5 are the versions extended by a biosensor domain either complementary, or not, to the substrate, respectively. Finally, lanes 6 and 7 are the <em>on</em> and <em>off</em> versions including separated blocker and biosensor domains, respectively. BPB and XC indicate the positions of the bromophenol blue and xylene cyanol dyes, respectively. The nucleotide sequences of each of the engineered ribozyme are shown in the <a href="#pone.0000673.s001">Figure S1</a>. (D) Verification of the gain in substrate specificity of the <em>trans</em>-acting HDV ribozymes. The upper portion shows a schematic representation of the substrates in which the P1 binding domain is identical (grey), but the sequences of the biosensors vary (different colors). The lower portion shows the autoradiogram of a cleavage assay performed on a pool of substrates either by a specific HDV <em>on</em> ribozyme (a to e correspond to the cleavage produced by specific ribozymes, lanes 2 to 6, respectively), or by the original ribozyme (lane 7). Lane 1 is the negative control performed in the absence of ribozyme. The nucleotide sequences of each of ribozyme are shown in the <a href="#pone.0000673.s002">Figure S2</a>.</p>
<span>doi:10.1371/journal.pone.0000673.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Development of a new on/off adaptor version for HDV ribozyme</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Initially, a new design that considers the blocker and biosensor domains as distinct entities was performed by including them at each end of the P2 stem of the HDV ribozyme in a process that involved several steps (<a href="#pone-0000673-g001">Figure 1B</a>). First, a blocker sequence that base-pairs with the binding domain of the ribozyme was added to the 5′-end. The resulting ribozyme exhibited no cleavage activity with a model substrate derived from the internal ribosome entry site (IRES) region of the hepatitis C virus (HCV) (<a href="#pone-0000673-g001">Figure 1C</a>, lane 3). The blocker acts as a “safety lock” placing the ribozyme in an inactive conformation (<em>off</em> state). When the ribozyme possesses a 12 nucleotide (nt) biosensor sequence complementary to the substrate at its 3′-end, it exhibited an improved cleavage activity (<a href="#pone-0000673-g001">Figure 1C</a>, lane 4). The presence of an inappropriate biosensor domain (i.e. a sequence not complementary to the substrate) resulted in a cleavage level comparable to that of the original ribozyme (<a href="#pone-0000673-g001">Figure 1C</a>, lane 5). When an appropriate biosensor was added to a ribozyme that already possessed a blocker, a dramatic increase in the cleavage activity, as compared to that observed in the presence of an inappropriate biosensor, was observed (<a href="#pone-0000673-g001">Figure 1C</a>, lanes 6 and 7, respectively). Thus, the ribozyme can be switched <em>off</em> and <em>on</em> by independent domains in a way reminiscent of the MAP kinases.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Kinetic analyses were performed in order to determine the second order rate constants (k<sub>cat</sub>/K<sub>M</sub>') as an indication of the substrate specificities. The HDV <em>on</em> (i.e. the one including both a blocker and an appropriate biosensor) exhibited a cleavage activity characterized by a relative k<sub>cat</sub>/K<sub>M</sub>' 6-fold higher than that of the original ribozyme, which in turn had a value 12-fold higher than that of the HDV <em>off</em> ribozyme (i.e the one including a blocker and an inappropriate biosensor) (see <a href="#pone-0000673-t001">Table 1</a>). The difference between the k<sub>cat</sub>/K<sub>M</sub>' values of the <em>off</em> and <em>on</em> ribozymes is therefore greater than 60 times. This difference is smaller than the determined 1500 to 2500 times observed between the <em>on</em> and <em>off</em> forms of the original configuration of the SOFA module on other HDV ribozymes <a href="#pone.0000673-Bergeron1">[6]</a>, <a href="#pone.0000673-Bergeron2">[7]</a>. The larger difference in the latter case is likely due to the fact that its configuration is probably better adapted to the architecture of this particular ribozyme.</p>
<div class="figure" id="pone-0000673-t001"><div class="img"><a name="pone-0000673-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000673.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000673.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Kinetic parameters of four cleaving nucleic acid enzymes in single turnover conditions</span></strong></p><span>doi:10.1371/journal.pone.0000673.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>As the specificity of a ribozyme is commonly defined as its ability to discriminate between similar substrates <a href="#pone.0000673-Fersht1">[1]</a>, a collection of five substrates possessing identical ribozyme binding domains, but different biosensor sequences, was synthesized (<a href="#pone-0000673-g001">Figure 1D</a>). Because these substrates harbor different 5′-ends, they can be distinguished by their electrophoretic migration (<a href="#pone-0000673-g001">Figure 1D</a>, lane 1). When the 5 substrates were incubated together with only one HDV <em>on</em> ribozyme, solely the substrate possessing the sequence perfectly complementary to the biosensor was cleaved (<a href="#pone-0000673-g001">Figure 1D</a>, lanes 2-6). Conversely, an original ribozyme (i.e. one without both a biosensor and a blocker) had the ability to cleave all of the substrates, although at different levels (<a href="#pone-0000673-g001">Figure 1D</a>, lane 7). Together, these data demonstrate the improvement, in terms of substrate specificity, resulting from the addition of both a blocker and a biosensor. Moreover, it shows that a ribozyme, when activated by the proper substrate, does not cleave other substrates via a <em>trans</em>-cleavage mechanism. Finally, the potential to cleave HCV-derived transcripts was tested in cell culture (data not shown). Only small amounts of transcripts were cleaved. Determination of the half-life of the new conformation of the SOFA-HDV ribozymes according to a procedure reported previously <a href="#pone.0000673-Lvesque1">[8]</a>, revealed a relatively low molecular stability of these ribozymes (a likely by-product of the single-stranded extremities that these ribozymes possess). More important is that it has been possible to adapt the target-dependent module of activation from the MAP kinase to the HDV ribozyme.</p>


<h4>Adaptation of this mode of activation to various cleaving nucleic acid enzymes</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Subsequently, we tried to adapt this mode of activation to other catalytic nucleic acids. As a first attempt, we chose three cleaving molecules: the hammerhead ribozyme, the 10-23 deoxyribozyme and the hairpin ribozyme (<a href="#pone-0000673-g002">Figure 2</a>) <a href="#pone.0000673-Blount1">[9]</a>–<a href="#pone.0000673-Wilson1">[11]</a>. All of these nucleic acid enzymes share a relatively similar architecture. The cleavage site of the substrates is located in the middle of the two stems formed between the catalytic center. The sequences of both of these ribozymes and of the deoxyribozyme have been reported to cleave various substrates that were used <a href="#pone.0000673-Sakamoto1">[12]</a>–<a href="#pone.0000673-Sargueil1">[15]</a>. Each of these catalytic nucleic acids showed significant cleavage of their 5′-end labelled substrates, although at different levels (<a href="#pone-0000673-g002">Figure 2A–C</a>, lane 2). The addition of a blocker domain that “zips off” the binding domain dramatically decreased the cleavage activity (i.e. <a href="#pone-0000673-g002">Figure 2A–C</a>, lane 3). As is observed with the biosensor-blocker HDV ribozyme, the specificity of these engineered nucleic acid molecules was dramatically increased through blocker-mediated inactivation followed by reactivation via an extended complementarity with the appropriate biosensor sequence (<a href="#pone-0000673-g002">Figure 2A–C</a>, lanes 4 to 7). With both the biosensor and blocker domains, the presence of an inappropriate biosensor locked the catalytic core into an <em>off</em> state, while the presence of an appropriate biosensor permitted the switch into the active mode, by displacing the blocker, solely in presence of its specific substrate. The biosensors interact through more base pairs than do the blockers, thereby favoring the <em>on</em> state over the <em>off</em> state. When comparing the k<sub>cat</sub>/K<sub>M'</sub> values for the <em>off</em> and <em>on</em> versions of these catalytic molecules, the overall improvements were of a minimum of three orders of magnitude (<a href="#pone-0000673-t001">Table 1</a>). Specifically, the k<sub>cat</sub>/K<sub>M'</sub> values vary from 2588-, 2762- and 9167-folds between the <em>off</em> and <em>on</em> version of hammerhead ribozyme, hairpin ribozyme and 10-23 deoxyribozyme, respectively. This illustrates the important gain in terms of specificity. These important differences were mainly the result of dramatic decreases in the k<sub>cat</sub> values between those of the original structures and those of their respective <em>off</em> states, suggesting that they lost their abilities to perform non-specific cleavage. Importantly, the addition of both a blocker and a biosensor permits the generation of cleaving nucleic acid enzymes that operate in a target-dependent fashion.</p>
<div class="figure" id="pone-0000673-g002"><div class="img"><a name="pone-0000673-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000673.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000673.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Autoradiograms of cleavage assays performed with controlled cleaving nucleic acid enzymes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) (B) and (C) are the reactions performed with the hammerhead ribozyme, the 10-23 deoxyribozyme and the hairpin ribozyme, respectively. In each case the structures of the nucleic acid enzyme with the blocker (red) and biosensor (green) are illustrated above the appropriate lanes of the gels, and the control (-) was performed in the absence of any nucleic acid enzyme (lane 1), and, lane 2, in the presence of the unmodified version of the ribozyme. Lane 3 is the version extended by the blocker sequence. Lanes 4 and 5 are the versions extended by a biosensor that is either complementary, or not, to the substrate. Finally, lanes 6 and 7 are the <em>on</em> and <em>off</em> versions, respectively. The nucleotide sequences of each nucleic acid enzyme are depicted in the <a href="#pone.0000673.s003">Figures S3</a><a href="#pone.0000673.s004">pone.0000673.s004</a> to <a href="#pone.0000673.s005">S5</a>.</p>
<span>doi:10.1371/journal.pone.0000673.g002</span></div>

<h4>Engineering ribozymes catalyzing various reactions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Next, we investigated other ribozyme's catalyses. As a first attempt, the ligation catalyzed by the hairpin ribozyme was studied. Extending the 3′-end of the original hairpin ribozyme by 6 nt resulted in the retention of the cleavage product, and consequently favored the reverse reaction (<a href="#pone-0000673-g003">Figure 3A</a>) <a href="#pone.0000673-Komatsu1">[14]</a>. Using a 5′-<sup>32</sup>P-labelled RNA strand possessing a terminal 2′-3′-cyclic phosphate and a second RNA strand possessing a 5′-hydroxyl resulted in the detection of ligation products (<a href="#pone-0000673-g003">Figure 3B</a>). The addition of a biosensor with a sequence complementary to that of the substrate increased the amount of ligation observed. In contrast, the addition of the blocker module almost completely abolished the ligation. Finally, an <em>on</em> version, which includes both the blocker and an appropriate biosensor, exhibited a ligation activity of up to 65%; while an <em>off</em> version (i.e. one with an inappropriate biosensor) showed less than 5% activity.</p>
<div class="figure" id="pone-0000673-g003"><div class="img"><a name="pone-0000673-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000673" data-uri="info:doi/10.1371/journal.pone.0000673.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000673.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000673.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000673.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000673.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000673.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Catalytic activity of the engineered dependent ligating and capping ribozymes.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) and (C) are the nucleotide sequences and secondary structures of the ribozyme-substrate complexes of the ligating hairpin and capping ribozymes, respectively. (B) and (D) are autoradiograms of the PAGE gels used to analyse the ligation and capping reactions, respectively. The schematic structures of the nucleic acid enzymes with both the blocker (red) and biosensor (green) are illustrated above the appropriate lanes of the gels. The controls (-) was performed in the absence of ribozyme (lane 1), and, lane 2, in the presence of the unmodified versions. Lane 3 is the ribozyme extended by the blocker sequence. Lanes 4 and 5 are the versions extended by a biosensor that is either complementary, or not, to the substrate, respectively. Finally, lanes 6 and 7 are the <em>on</em> and <em>off</em> ribozymes, respectively. The nucleotide sequences of each engineered nucleic acid enzyme are depicted in <a href="#pone.0000673.s006">Figures S6</a> and <a href="#pone.0000673.s007">S7</a>.</p>
<span>doi:10.1371/journal.pone.0000673.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Subsequently, the concept was adapted to an artificial ribozyme that catalyzed the attack of a 5′-terminal nucleotide phosphate on the α-phosphate of the terminal triphosphate of an RNA substrate, forming a 5′-5′ phosphodiester linkage (i.e. a capping ribozyme; <a href="#pone-0000673-g003">Figure 3C–D</a>) <a href="#pone.0000673-Zaher1">[16]</a>. A version harboring a blocker alone exhibited only residual activity for the capping of a 33-nucleotide substrate (&lt;3%), while one possessing only an appropriate biosensor sequence was slightly more efficient than the original. The <em>on</em> version has the ability to switch from its <em>off</em> to <em>on</em> state, while the <em>off</em> version remained trapped in the inactive conformation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Together, these experiments demonstrate the possibility of adapting the target-dependent activation concept to various types of nucleic acid enzymes. The limiting factor with such an adaptation is the fact that the suitable candidates need to have a recognition mechanism based on base pair formation in order to permit a rational design. As compared to the allosteric mode of activation that has been engineered for synthetic nucleic acid circuits based on oligonucleotide-sensing allosteric ribozymes and deoxyribozyme <a href="#pone.0000673-Stojanovic1">[17]</a>–<a href="#pone.0000673-Davidson1">[19]</a>, in this case there is no need for the presence of a third partner.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>As first glance it might appear somewhat surprising that any known natural catalytic RNA would resort to this mode of regulation in order to provide the appropriate substrate specificity. This might be a mode of regulation for processes that require additional selective pressures in order to ensure a greater fidelity of action, for example as it is required for the MAP kinases. The complexity of the modern cellular metabolism might have been an important selective pressure in this evolutionary process. Catalytic RNAs might not have needed to evolve such a regulation mechanism due to the fact that their recognition strategy, which is based on base pair formation, provided sufficient specificity. Moreover, these ribozymes act in <em>cis</em>, therefore there is no requirement for such mechanism. Other natural ribozymes, such as the ribonuclease P <a href="#pone.0000673-GuerrierTakada1">[20]</a>, ribosome <a href="#pone.0000673-Noller1">[21]</a> and spliceosome <a href="#pone.0000673-Valadkhan1">[22]</a>, catalyze their reactions with many substrates rendering high substrate specificity detrimental. Alternatively, the ability of <em>cis</em>-acting ribozymes to adopt different conformations may have contributed to their regulation. For example, viroids self-cleave their nascent transcripts through a hammerhead structure, but when the polymerase progresses further, a sequence partially complementary to the self-cleavage motif is synthesized, and, consequently, a rod-like structure is folded <a href="#pone.0000673-Beaudry1">[23]</a>. In this structure, the hammerhead motif is in an <em>off</em> state that prevents cleavage after viroid circularization. Thus, examples of RNA evolution that have already reached the blocker step of this mode of activation do indeed exist. Here, we show that a target-dependent activation, analogous to that of the MAP kinases, is possible for a wide range of nucleic acid catalysis types. It is therefore tempting to speculate that it already exists in nature, and that it simply remains to be found.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>DNA templates and constructs</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Deoxyribozymes were purchased as oligonucleotides from Invitrogen. All ribozymes and small RNA substrates used were produced by PCR using oligonucleotides purchased from either Invitrogen or IDT Inc. Briefly, DNA oligonucleotides corresponding to the sequences complementary to the different templates, and possessing a T7 promoter complementary sequence, were annealed to T7 promoter primer and the primers extended in PCR reactions. The reaction products were then used as templates for run-off transcription (see below). Alternatively, some constructs were prepared by annealing of two overlapping oligonucleotides that were then filled in by PCR using Vent DNA polymerase (New England Biolab), producing a double-stranded DNA fragment. Each of these fragments was provided with a T7 promoter at the beginning of the sense strand for subsequent use as a template for run-off transcription. Hepatitis C virus (HCV) derived substrate (a 575 nt long template containing the internal ribosome entry site (IRES) and the first part of the ORF) was produced by <em>Kpn</em>I digestion of the pHCVA plasmid <a href="#pone.0000673-Bergeron1">[6]</a>, which contains 1348 nt of the 5′ sequence of hepatitis C virus (genotype 1b) <a href="#pone.0000673-AlaouiIsmaili1">[24]</a>. PCR products and linearized plasmid templates were purified by phenol/chloroform extraction, precipitated with ethanol and dissolved in water.</p>


<h4>RNA synthesis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Run-off transcriptions were performed as described previously <a href="#pone.0000673-Bergeron1">[6]</a>. Briefly, transcriptions were performed using purified T7 RNA polymerase (10 µg) in the presence of RNA Guard (24 U, Amersham Biosciences) and pyrophosphatase (0.01 U, Roche Diagnostics) in a buffer containing 80 mM HEPES-KOH, pH 7.5, 24 mM MgCl<sub>2</sub>, 2 mM spermidine, 40 mM DTT and either linearized plasmid DNA (5 µg) or PCR product (100 pmol) as template. Each NTP (5 mM) was added to the mixture either with or without 50 µCi of [α-<sup>32</sup>P]UTP (3 000 Ci/mmol, New England Nuclear) in a final volume of 100 µL, and the reaction incubated at 37°C for 2 to 4 h. Upon completion, the reaction mixtures were treated with DNase RQ1 (Promega) at 37°C for 20 min, and the RNA then purified by phenol:chloroform extraction and precipitation with ethanol. RNA products were fractionated by a denaturing (7 M urea) 6 to 20% polyacrylamide gel electrophoresis (PAGE; 19:1 ratio of acrylamide to bisacrylamide) using 45 mM Tris-borate, pH 7.5/1 mM EDTA solution as running buffer. The reaction products were visualized either by UV shadowing, or by autoradiography. The bands corresponding to the correct sizes for both the ribozymes and the substrates were excised from the gel and the transcripts eluted overnight at room temperature in a buffer containing 1 mM EDTA, 0.1% SDS and 0.5 M ammonium acetate. The transcripts were ethanol precipitated at −80°C, washed, dried and dissolved in water. The amounts retrieved were determined either by <sup>32</sup>P counting, or by absorbance at 260 nm.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>In order to produce 5′-labelled RNA substrates, transcripts were dephosphorylated by adding 1 U of antartic phosphatase (New England Biolab) to 20 pmol of RNA and incubating for 30 min at 37°C in a final volume of 10 µL containing 50 mM Tris-HCl, pH 8.5, 0.1 mM EDTA and 40 U of RNAGuard (Amersham Biosciences). The enzyme was inactivated by incubating at 65°C for 5 min. Dephosphorylated RNA (5 pmol) was then radiolabelled at its 5′-end using 3 U of T4 polynucleotide kinase in the presence of 3.2 pmol of [γ-<sup>32</sup>P]ATP (6000 Ci/mmol, New England Nuclear). Radiolabelled transcripts were purified by electrophoresis on a denaturing gel and recovered as described above.</p>


<h4>Cleavage assays and kinetics</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>All reactions with the cleaving HDV, hammerhead and hairpin ribozymes, as well as those with the deoxyribozyme, were performed under single-turnover conditions ([Rz]&gt;&gt;[S]) using 100 nM of nucleic acid enzyme and trace amounts of 5′-<sup>32</sup>P-labelled substrates at 37°C in a final volume of 10 µL containing 50 mM Tris-HCl, pH 7.5 and 10 mM MgCl<sub>2</sub>. After an incubation of 1 h, the reactions were quenched by the addition of ice-cold formamide dye buffer (1 vol of 97% formamide, 10 mM EDTA, pH 8.0, 0.25% xylene cyanol, 0.25% bromophenol blue) and fractionated on 6% to 20% PAGE gels. The gels were then analyzed with a radioanalytic scanner (Storm, Molecular Dynamics). For time-course experiments, aliquots of 0.8 µL were taken at various times up to 3 h and treated as described above. Kinetic analyses were performed under single turnover conditions. Trace amounts of 5′-<sup>32</sup>P-labelled substrate (&lt;1 nM) were cleaved by various concentrations of either ribozyme or deoxyribozyme (5–400 nM). The fraction cleaved was determined, and the rate of cleavage (k<sub>obs</sub>) obtained from fitting the data to the equation A<sub>t</sub> = A<sub>□</sub>(1-e<sup>−kt</sup>) where A<sub>t</sub> is the percentage of cleavage at time t, A<sub>□</sub> is the maximum percent cleavage (or the end-point of cleavage), and k is the rate constant (k<sub>obs</sub>). Each rate constant was calculated from at least two independent measurements. The values of k<sub>obs</sub> obtained were then plotted as a function of nucleic acid enzyme concentration for the determination of k<sub>cat</sub>, K<sub>M</sub>' and k<sub>cat</sub>/K<sub>M</sub>'. The values obtained from independent experiments varied by less than 15%. The experiments of substrate specificity using several substrates were performed under multiple turnover conditions using 100 nM HDV ribozyme and 1 µM substrate.</p>


<h4>Ligation reactions</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Ligation reactions were performed as described previously <a href="#pone.0000673-Komatsu1">[14]</a>, <a href="#pone.0000673-Sargueil1">[15]</a> with minor modifications. Full length 5′-<sup>32</sup>P-labelled substrate (5 pmol) was cleaved in the presence of 10 µM of hairpin ribozyme in a volume of 50 µl containing 50 mM Tris-HCl, pH 7.5 and 10 mM MgCl<sub>2</sub>. The mixture was incubated 2 h at 37°C, quenched by the addition of ice-cold formamide dye buffer, purified on a 20% denaturing PAGE gel and the radiolabelled 20 nt 5′-product recovered as described. This RNA strand was then used as substrate for the ligation reaction. The 3′-product used in this ligation reaction was purchased from IDT Inc. This deprotected 5′-hydroxyl RNA was purified as described above. Ligation reactions were performed using 10 µM of the hairpin ribozyme in the presence of 15 µM of the 3′-end substrate and trace amounts of 5′-<sup>32</sup>P-labelled substrate mixed in a total volume of 9 µl containing 50 mM HEPES-KOH, pH 7.0. The reactions were preheated at 37°C for 10 min so as to permit the annealing of the substrate to the ribozyme, and the ligation reactions started (time = 0) by adding 1 µL of 150 mM MgCl<sub>2</sub>. Samples were incubated at 37°C for 1 h, fractionated on gels and visualized by autoradiography.</p>


<h4>Capping enzyme reaction</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The capping reaction was performed as described previously <a href="#pone.0000673-Zaher1">[16]</a>. Substrate was 3′-end labelled by incubating, at 37°C, 10 pmol of transcripts with 3.2 pmoles of [<sup>32</sup>P]Cp (3000 ci/mmol, New England Nuclear) and 1 U T4 RNA ligase (New England Biolab) in the buffer supplied by the manufacturer supplemented with 10% DMSO. After 3 h, the reaction was stopped and the transcripts purified by denaturing 10% denaturing PAGE and recovered as described above. Capping reactions were carried out under single turnover conditions. Both the ribozyme (2 µM) and substrate (0.2 µM) were annealed together for 5 min at 22°C in a reaction buffer containing 50 mM Tris-HCl, pH 7.5, 25 mM MgCl<sub>2</sub> and 150 mM KCl. The reactions were initiated by the addition of ATP to a final concentration of 1 mM. After 16 h of incubation at 22°C, the reactions were quenched by addition of ice-cold formamide dye buffer and analyzed on denaturing 15% PAGE gels.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000673"></div><a name="pone.0000673.s001" id="pone.0000673.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structure of the various versions of the HDV ribozyme cleaving a 28 nt substrate derived from the hepatitis C virus. (A) The original version. (B) The version including a 4 nt blocker (bl). (C) The version with an appropriate 12 nt biosensor (BS). (D) The version with an inappropriate 12 nt biosensor (iBS). (E) The on version with both a blocker and an appropriate biosensor. (F) The off version with both a blocker and an inappropriate biosensor (i.e. a biosensor of sequence not complementary to the substrate). The blocker, appropriate biosensor and inappropriate biosensor are in red, green and grey, respectively. The arrows indicate the cleavage sites.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.37 MB TIF)</p>
<a name="pone.0000673.s002" id="pone.0000673.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structures of the HDV ribozyme-substrate complexes for the substrate specificity experiment. All of the ribozymes and substrates possess the same P1 binding domain. (A) The original version of the HDV ribozyme cleaving a 28 nt substrate derived from the hepatitis C virus. (B) to (F) The HDV on ribozymes with the different biosensor modules (in different colors). (G) The original ribozyme that can bind all of the substrates, regardless of their binding domains to the biosensor.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.60 MB TIF)</p>
<a name="pone.0000673.s003" id="pone.0000673.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structure of the hammerhead ribozyme cleaving a 575 nt substrate derived from the HCV. (A) The original version as reported previously (12). (B) The version including an 8 nt blocker (bl). (C) The version with an appropriate 15 nt biosensor (BS). (D) The version with an inappropriate biosensor (iBS). (E) The on version with both a blocker and appropriate biosensor. (F) The off version with both a blocker and an inappropriate biosensor. The blocker, appropriate biosensor and inappropriate biosensor are in red, green and grey, respectively. The arrows indicate the cleavage sites.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.59 MB TIF)</p>
<a name="pone.0000673.s004" id="pone.0000673.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structure of the 10-23 deoxyribozyme cleaving a 46 nt substrate derived from the 5′ UTR of the human rhinovirus 14. (A) The original version as reported previously (13). (B) The version including an 8 nt blocker (bl). (C) The version with an appropriate 14 nt biosensor (BS). (D) The version with an inappropriate biosensor (iBS). (E) The on version with both a blocker and an appropriate biosensor. (F) The off version with both a blocker and an inappropriate biosensor. The blocker, appropriate biosensor and inappropriate biosensor are in red, green and grey, respectively. The arrows indicate the cleavage sites.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.18 MB TIF)</p>
<a name="pone.0000673.s005" id="pone.0000673.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structure of the hairpin ribozyme cleaving a 44 nt substrate. (A) The original version as reported previously (14). (B) The version including an 8 nt blocker (bl). (C) The version with an appropriate 11 nt biosensor (BS). (D) The version with an inappropriate biosensor (iBS). (E) The on version with both a blocker and appropriate biosensor. (F) The off version with both a blocker and an inappropriate biosensor. The blocker, appropriate biosensor and inappropriate biosensor are in red, green and grey, respectively. The arrows indicate the cleavage sites.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(1.69 MB TIF)</p>
<a name="pone.0000673.s006" id="pone.0000673.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structure of the hairpin ribozyme ligating two RNA strands 20 and 24 nt in length. (A) The original version. (B) The version including a 6 nt blocker (bl). (C) The version with an appropriate 11 nt biosensor (BS). (D) The version with an inappropriate biosensor (iBS). (E) The on version with both a blocker and appropriate biosensor. (F) The off version with both a blocker and an inappropriate biosensor. The blocker, appropriate biosensor and inappropriate biosensor are in red, green and grey, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.73 MB TIF)</p>
<a name="pone.0000673.s007" id="pone.0000673.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000673.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Nucleotide sequence and secondary structure of the ribozyme capping a 33 nt RNA strand by an ATP molecule as reported previously (15). (A) The original version. (B) The version including a 5 nt blocker (bl). (C) The version with an appropriate 10 nt biosensor (BS). (D) The version with an inappropriate biosensor (iBS). (E) The on version with both a blocker and an appropriate biosensor. (F) The off version with both a blocker and an inappropriate biosensor. The blocker, appropriate biosensor and inappropriate biosensor are in red, green and grey, respectively.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000673.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.93 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors gratefully acknowledge L. J. Bergeron and their colleagues for stimulating discussions. J.P.P. holds the Canada Research Chair of Genomics and Catalytic RNA.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JP DL. Performed the experiments: DL FB. Analyzed the data: JP DL FB. Wrote the paper: JP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000673-Fersht1" id="pone.0000673-Fersht1"></a>Fersht A (1999) Structure and mechanism in protein science. New York: WH Freeman and Company. pp. 289–318.  ed. 3.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000673-Winkler1" id="pone.0000673-Winkler1"></a>Winkler WC, Nahvi A, Roth A, Collins JA, Breaker RR (2004) Control of gene expression by a natural metabolite-responsive ribozyme. Nature  428: 281–286.  <ul class="find" data-citedArticleID="971468" data-doi="10.1038/nature02362"><li><a href="http://dx.doi.org/10.1038/nature02362" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+gene+expression+by+a+natural+metabolite-responsive+ribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+gene+expression+by+a+natural+metabolite-responsive+ribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000673-Jansen1" id="pone.0000673-Jansen1"></a>Jansen JA, McCarthy TJ, Soukup GA, Soukup JK (2006) Backbone and nucleobase contacts to glucosamine-6-phosphate in the glmS ribozyme. Nature Struct Mol Biol  13: 517–523.  <ul class="find" data-citedArticleID="971442" data-doi="10.1038/nsmb1094"><li><a href="http://dx.doi.org/10.1038/nsmb1094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Backbone+and+nucleobase+contacts+to+glucosamine-6-phosphate+in+the+glmS+ribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Backbone+and+nucleobase+contacts+to+glucosamine-6-phosphate+in+the+glmS+ribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000673-Biondi1" id="pone.0000673-Biondi1"></a>Biondi RM, Nebreda AR (2003) Signalling specificity of Ser/Thr protein kinases through docking-site-mediated interactions. Biochem J  372: 1–13.  <ul class="find" data-citedArticleID="971432" data-doi="10.1042/bj20021641"><li><a href="http://dx.doi.org/10.1042/bj20021641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signalling+specificity+of+Ser%2FThr+protein+kinases+through+docking-site-mediated+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signalling+specificity+of+Ser%2FThr+protein+kinases+through+docking-site-mediated+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000673-Tanoue1" id="pone.0000673-Tanoue1"></a>Tanoue T, Nishida E (2003) Molecular recognitions in the MAP kinase cascades. Cell Signalling  15: 455–462.  <ul class="find" data-citedArticleID="971462" data-doi="10.1016/s0898-6568(02)00112-2"><li><a href="http://dx.doi.org/10.1016/s0898-6568(02)00112-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+recognitions+in+the+MAP+kinase+cascades." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+recognitions+in+the+MAP+kinase+cascades.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000673-Bergeron1" id="pone.0000673-Bergeron1"></a>Bergeron L Jr, Perreault JP (2005) Target-dependent on/off switch increases ribozyme fidelity. Nucleic Acids Res  33: 1240–1248.  <ul class="find" data-citedArticleID="971428" data-doi="10.1093/nar/gki278"><li><a href="http://dx.doi.org/10.1093/nar/gki278" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Target-dependent+on%2Foff+switch+increases+ribozyme+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Target-dependent+on%2Foff+switch+increases+ribozyme+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000673-Bergeron2" id="pone.0000673-Bergeron2"></a>Bergeron L Jr, Reymond C, Perreault JP (2005) Functional characterization of the SOFA delta ribozyme. RNA  11: 1858–1868.  <ul class="find" data-citedArticleID="971430" data-doi="10.1261/rna.2112705"><li><a href="http://dx.doi.org/10.1261/rna.2112705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+characterization+of+the+SOFA+delta+ribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+characterization+of+the+SOFA+delta+ribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000673-Lvesque1" id="pone.0000673-Lvesque1"></a>Lévesque D, Choufani S, Perreault JP (2002) Delta ribozyme benefits from a good stability in vitro that becomes outstanding in vivo. RNA  8: 464–477.  <ul class="find" data-citedArticleID="971446" data-doi="10.1017/s1355838202020289"><li><a href="http://dx.doi.org/10.1017/s1355838202020289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Delta+ribozyme+benefits+from+a+good+stability+in+vitro+that+becomes+outstanding+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Delta+ribozyme+benefits+from+a+good+stability+in+vitro+that+becomes+outstanding+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000673-Blount1" id="pone.0000673-Blount1"></a>Blount KF, Uhlenbeck OC (2005) The structure-function dilemma of the hammerhead ribozyme. Annu Rev Biophys Biomol Struct  34: 415–440.  <ul class="find" data-citedArticleID="971434" data-doi="10.1146/annurev.biophys.34.122004.184428"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.34.122004.184428" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure-function+dilemma+of+the+hammerhead+ribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure-function+dilemma+of+the+hammerhead+ribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000673-Silverman1" id="pone.0000673-Silverman1"></a>Silverman SK (2005) In vitro selection, characterization, and application of deoxyribozymes that cleave RNA. Nucleic Acids Res  33: 6151–6163.  <ul class="find" data-citedArticleID="971458" data-doi="10.1093/nar/gki930"><li><a href="http://dx.doi.org/10.1093/nar/gki930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection%2C+characterization%2C+and+application+of+deoxyribozymes+that+cleave+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection%2C+characterization%2C+and+application+of+deoxyribozymes+that+cleave+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000673-Wilson1" id="pone.0000673-Wilson1"></a>Wilson TJ, Nahas M, Ha T, Lilley DM (2005) Folding and catalysis of the hairpin ribozyme. Biochem Soc Trans  33: 461–465.  <ul class="find" data-citedArticleID="971466" data-doi="10.1042/bst0330461"><li><a href="http://dx.doi.org/10.1042/bst0330461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Folding+and+catalysis+of+the+hairpin+ribozyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Folding+and+catalysis+of+the+hairpin+ribozyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000673-Sakamoto1" id="pone.0000673-Sakamoto1"></a>Sakamoto N, Wu CH, Wu GY (1996) Intracellular cleavage of hepatitis C virus RNA and inhibition of viral protein translation by hammerhead ribozymes. J Clin Invest  98: 2720–2728.  <ul class="find" data-citedArticleID="971452" data-doi="10.1172/jci119097"><li><a href="http://dx.doi.org/10.1172/jci119097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intracellular+cleavage+of+hepatitis+C+virus+RNA+and+inhibition+of+viral+protein+translation+by+hammerhead+ribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intracellular+cleavage+of+hepatitis+C+virus+RNA+and+inhibition+of+viral+protein+translation+by+hammerhead+ribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000673-Schubert1" id="pone.0000673-Schubert1"></a>Schubert S, Furste JP, Werk D, Grunert HP, Zeichhardt H, et al.  (2004) Gaining target access for deoxyribozymes. J Mol Biol  339: 335–363.  <ul class="find" data-citedArticleID="971456" data-doi="10.1016/j.jmb.2004.03.064"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.03.064" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gaining+target+access+for+deoxyribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gaining+target+access+for+deoxyribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000673-Komatsu1" id="pone.0000673-Komatsu1"></a>Komatsu Y, Shirai M, Yamashita S, Ohtsuka E (1997) Construction of hairpin ribozymes with a three-way junction. Bioorg Med Chem  5: 1063–1069.  <ul class="find" data-citedArticleID="971444" data-doi="10.1016/s0968-0896(97)00042-4"><li><a href="http://dx.doi.org/10.1016/s0968-0896(97)00042-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+hairpin+ribozymes+with+a+three-way+junction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+hairpin+ribozymes+with+a+three-way+junction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000673-Sargueil1" id="pone.0000673-Sargueil1"></a>Sargueil B, Hampel KJ, Lambert D, Burke JM (2003) In vitro selection of second site revertants analysis of the hairpin ribozyme active site. J Biol Chem  278: 52783–52791.  <ul class="find" data-citedArticleID="971454" data-doi="10.1074/jbc.m306703200"><li><a href="http://dx.doi.org/10.1074/jbc.m306703200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+of+second+site+revertants+analysis+of+the+hairpin+ribozyme+active+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+of+second+site+revertants+analysis+of+the+hairpin+ribozyme+active+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000673-Zaher1" id="pone.0000673-Zaher1"></a>Zaher HS, Watkins RA, Unrau PJ (2006) Two independently selected capping ribozymes share similar substrate requirements. RNA  12: 1949–1958.  <ul class="find" data-citedArticleID="971470" data-doi="10.1261/rna.131306"><li><a href="http://dx.doi.org/10.1261/rna.131306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+independently+selected+capping+ribozymes+share+similar+substrate+requirements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+independently+selected+capping+ribozymes+share+similar+substrate+requirements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000673-Stojanovic1" id="pone.0000673-Stojanovic1"></a>Stojanovic MN, Stefanovic D (2003) A deoxyribozyme-based molecular automaton. Nature Biotech  21: 1069–1074.  <ul class="find" data-citedArticleID="971460" data-doi="10.1038/nbt862"><li><a href="http://dx.doi.org/10.1038/nbt862" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+deoxyribozyme-based+molecular+automaton." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+deoxyribozyme-based+molecular+automaton.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000673-Penchovsky1" id="pone.0000673-Penchovsky1"></a>Penchovsky R, Breaker RR (2005) Computational design and experimental validation of oligonucleotide-sensing allosteric ribozymes. Nature Biotech  23: 1424–1433.  <ul class="find" data-citedArticleID="971450" data-doi="10.1038/nbt1155"><li><a href="http://dx.doi.org/10.1038/nbt1155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+design+and+experimental+validation+of+oligonucleotide-sensing+allosteric+ribozymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+design+and+experimental+validation+of+oligonucleotide-sensing+allosteric+ribozymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000673-Davidson1" id="pone.0000673-Davidson1"></a>Davidson EA, Ellington AD (2007) Synthetic RNA circuits. Nature Chem Biol  3: 23–28.  <ul class="find" data-citedArticleID="971436" data-doi="10.1038/nchembio846"><li><a href="http://dx.doi.org/10.1038/nchembio846" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthetic+RNA+circuits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthetic+RNA+circuits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000673-GuerrierTakada1" id="pone.0000673-GuerrierTakada1"></a>Guerrier-Takada C, Gardiner K, Marsh T, Pace N, Altman S (1983) The RNA moiety of ribonuclease P is the catalytic subunit of the enzyme. Cell  35: 849–857.  <ul class="find" data-citedArticleID="971440" data-doi="10.1016/0092-8674(83)90117-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(83)90117-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RNA+moiety+of+ribonuclease+P+is+the+catalytic+subunit+of+the+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RNA+moiety+of+ribonuclease+P+is+the+catalytic+subunit+of+the+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000673-Noller1" id="pone.0000673-Noller1"></a>Noller HF, Hoffarth V, Zimniak L (1992) Unusual resistance of peptidyl transferase to protein extraction procedures. Science  256: 1416–1419.  <ul class="find" data-citedArticleID="971448" data-doi="10.1126/science.1604315"><li><a href="http://dx.doi.org/10.1126/science.1604315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unusual+resistance+of+peptidyl+transferase+to+protein+extraction+procedures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unusual+resistance+of+peptidyl+transferase+to+protein+extraction+procedures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000673-Valadkhan1" id="pone.0000673-Valadkhan1"></a>Valadkhan S, Manley JL (2001) Splicing-related catalysis by protein-free snRNAs. Nature  413: 701–707.  <ul class="find" data-citedArticleID="971464" data-doi="10.1038/35099500"><li><a href="http://dx.doi.org/10.1038/35099500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splicing-related+catalysis+by+protein-free+snRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splicing-related+catalysis+by+protein-free+snRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000673-Beaudry1" id="pone.0000673-Beaudry1"></a>Beaudry D, Bussière F, Lareau F, Lessard C, Perreault JP (1995) The RNA of both polarities of the peach latent mosaic viroid self-cleaves in vitro solely by single hammerhead structures. Nucleic Acids Res  23: 745–752.  <ul class="find" data-citedArticleID="971426" data-doi="10.1093/nar/23.5.745"><li><a href="http://dx.doi.org/10.1093/nar/23.5.745" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RNA+of+both+polarities+of+the+peach+latent+mosaic+viroid+self-cleaves+in+vitro+solely+by+single+hammerhead+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RNA+of+both+polarities+of+the+peach+latent+mosaic+viroid+self-cleaves+in+vitro+solely+by+single+hammerhead+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000673-AlaouiIsmaili1" id="pone.0000673-AlaouiIsmaili1"></a>Alaoui-Ismaili MH, Gervais C, Brunette S, Gouin G, Hamel M, et al.  (2000) A novel high throughput screening assay for HCV NS3 helicase activity. Antiviral Res  46: 181–193.  <ul class="find" data-citedArticleID="971424" data-doi="10.1016/s0166-3542(00)00085-1"><li><a href="http://dx.doi.org/10.1016/s0166-3542(00)00085-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+high+throughput+screening+assay+for+HCV+NS3+helicase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+high+throughput+screening+assay+for+HCV+NS3+helicase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.XML" value="65826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.PDF" value="246450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g001.PNG_L" value="1045831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g001.PNG_M" value="77845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g001.PNG_S" value="20047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g001.TIF" value="2121576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g001.PNG_I" value="141935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.t001.PNG_L" value="116892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.t001.PNG_M" value="65122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.t001.PNG_S" value="9750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.t001.TIF" value="558848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.t001.PNG_I" value="19826"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g002.PNG_L" value="917736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g002.PNG_M" value="97118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g002.PNG_S" value="15149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g002.TIF" value="1494596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g002.PNG_I" value="88994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g003.PNG_L" value="935069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g003.PNG_M" value="84547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g003.PNG_S" value="13608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g003.TIF" value="1508158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.g003.PNG_I" value="99291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s001.TIF" value="1603142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s002.TIF" value="1865655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s003.TIF" value="1596548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s004.TIF" value="1443800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s005.TIF" value="1693465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s006.TIF" value="1736557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000673.s007.TIF" value="1932546"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000673&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000673" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000673&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000673&volume=&issue=&title=A Modern Mode of Activation for Nucleic Acid Enzymes&author_name=Dominique%20L%C3%A9vesque%2C%20Francis%20P.%20Bri%C3%A8re%2C%20Jean-Pierre%20Perreault&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000673" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673&amp;t=A%20Modern%20Mode%20of%20Activation%20for%20Nucleic%20Acid%20Enzymes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673&title=A%20Modern%20Mode%20of%20Activation%20for%20Nucleic%20Acid%20Enzymes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673&amp;title=A%20Modern%20Mode%20of%20Activation%20for%20Nucleic%20Acid%20Enzymes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000673&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Modern Mode of Activation for Nucleic Acid Enzymes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000673';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Modern%20Mode%20of%20Activation%20for%20Nucleic%20Acid%20Enzymes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000673" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000673" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biosensors%22" title="Search for articles in the subject area:'Biosensors'"><div class="flagText">Biosensors</div></a>
              <div data-categoryid="19817" data-articleid="24184"
                   data-categoryname="Biosensors"
                   class="flagImage" title="Flag 'Biosensors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24184"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="24184"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hammerhead+ribozymes%22" title="Search for articles in the subject area:'Hammerhead ribozymes'"><div class="flagText">Hammerhead ribozymes</div></a>
              <div data-categoryid="36105" data-articleid="24184"
                   data-categoryname="Hammerhead ribozymes"
                   class="flagImage" title="Flag 'Hammerhead ribozymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleic+acids%22" title="Search for articles in the subject area:'Nucleic acids'"><div class="flagText">Nucleic acids</div></a>
              <div data-categoryid="19373" data-articleid="24184"
                   data-categoryname="Nucleic acids"
                   class="flagImage" title="Flag 'Nucleic acids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="24184"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+denaturation%22" title="Search for articles in the subject area:'RNA denaturation'"><div class="flagText">RNA denaturation</div></a>
              <div data-categoryid="52041" data-articleid="24184"
                   data-categoryname="RNA denaturation"
                   class="flagImage" title="Flag 'RNA denaturation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribozymes%22" title="Search for articles in the subject area:'Ribozymes'"><div class="flagText">Ribozymes</div></a>
              <div data-categoryid="17535" data-articleid="24184"
                   data-categoryname="Ribozymes"
                   class="flagImage" title="Flag 'Ribozymes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=992'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=84'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5585&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44117">Improvement of specificity of nucleic acid enzymes</a><br>Posted by JPP23</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
