

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000872"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000872" />

    <meta name="citation_title" content="Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle"/>
    <meta itemprop="name" content="Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle"/>

      <meta name="citation_author" content="Dov Dori"/>
            <meta name="citation_author_institution" content="Faculty of Industrial Engineering and Management, Technion, Israel Institute of Technology, Haifa, Israel"/>
      <meta name="citation_author" content="Mordechai Choder"/>
            <meta name="citation_author_institution" content="Department of Molecular Biotechnology, Rappaport Faculty of Medicine, Technion, Israel Institute of Technology, Haifa, Israel"/>

    <meta name="citation_date" content="2007/9/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000872.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e872"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Systems Biology: A Brief Overview.; citation_author=H Kitano; citation_journal_title=Science,; citation_volume=295(5560); citation_number=1; citation_pages=1662-1664; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Editorial to “Systems Biology: A User's Guide”. Joint online resource by Nature Reviews Molecular Cell Biology and Nature Cell Biology, 2006.; citation_author=E Kritikou; citation_author=B Pulverer; citation_author=A Heinrichs; citation_number=2; " />
      <meta name="citation_reference" content="citation_title=A Genomic Regulatory Network for Development.; citation_author=EH Davidson; citation_journal_title=Science,; citation_volume=295(5560); citation_number=3; citation_pages=1669-1678; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Time scale and dimension analysis of a budding yeast cell cycle model.; citation_author=A Lovrics; citation_journal_title=BMC Bioinformatics,; citation_volume=7(494); citation_number=4; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Whole-cell simulation: a grand challenge of the 21st century.; citation_author=M Tomita; citation_journal_title=Trends Biotechnol,; citation_volume=19(6); citation_number=5; citation_pages=205-10; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=E-Cell 2: multi-platform E-Cell simulation system.; citation_author=K Takahashi; citation_journal_title=Bioinformatics,; citation_volume=19(13); citation_number=6; citation_pages=1727-9; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=7; " />
      <meta name="citation_reference" content="citation_title=Quantitative cell biology with the Virtual Cell.; citation_author=BM Slepchenko; citation_journal_title=Trends Cell Biol,; citation_volume=13(11); citation_number=8; citation_pages=570-6; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A Grand Challenge for Computing: Full Reactive Modeling of a Multi-Cellular Animal.; citation_author=D Harel; citation_journal_title=LNCS,; citation_volume=2623; citation_number=9; citation_pages=2-2; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A Turing-like test for biological modeling.; citation_author=D Harel; citation_journal_title=Nat Biotechnol,; citation_volume=23(4); citation_number=10; citation_pages=495-6; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Formal modeling of C. elegans development: a scenario-based approach. in Modelling in Molecular Biology; citation_author=N Kam; citation_number=11; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Enhancing biological information systems with granularity. in KnowledgeWeb PhD Symposium (KWEPSY06).; citation_author=C Keet; citation_number=12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cell Modeling using Agent-based Formalisms,. in Proc. 3rd International Joint Conference on Autonomous Agents and Multiagent Systems (AAMAS 2004), 19-23 August 2004, New York, NY, USA. (AAMAS 2004).; citation_author=K Webb; citation_author=T White; citation_number=13; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=14; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The UML User Guide.; citation_author=G Booch; citation_author=J Rumbaugh; citation_author=I Jacobson; citation_number=15; citation_date=1999; citation_publisher=Addison-Wesley; " />
      <meta name="citation_reference" content="citation_number=16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Object-Management-Group, Introduction to OMG's Unified Modeling Language™ (UML®).; citation_number=17; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Unified Modeling Language: A Complexity Analysis.; citation_author=K.a.CQ Siau; citation_journal_title=ournal of Database Management,; citation_volume=12(1); citation_number=18; citation_pages=26-34; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Model Multiplicity Problem: Experimenting with Real-Time Specification Methods.; citation_author=MDD Peleg; citation_journal_title=IEEE Transaction on Software Engineering,; citation_volume=26(8); citation_number=19; citation_pages=742-759; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=An Empirical Investigation in Quantifying Inconsistency and. Incompleteness of UML Designs. in Consistency Problems in UML-based Software Development II Workshop within the Sixth International Conference on the Unified Modelling Language.; citation_author=C Lange; citation_number=20; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The systems biology markup language (SBML): a medium for representation and exchange of biochemical network models.; citation_author=M Hucka; citation_journal_title=Bioinformatics,; citation_volume=19(4); citation_number=21; citation_pages=524-31; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A short introduction to CellML.; citation_author=W Hedley; citation_journal_title=Philosophical Transactions-Mathematical, Physical and Engineering Sciences,; citation_volume=359; citation_number=22; citation_pages=1073-1089; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=SysML.; citation_number=23; " />
      <meta name="citation_reference" content="citation_title=Molecular interaction maps as information organizers and simulation guides.; citation_author=KW Kohn; citation_journal_title=Chaos,; citation_volume=11; citation_number=24; citation_pages=84-97; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Using process diagrams for the graphical representation of biological networks.; citation_author=H Kitano; citation_journal_title=Nature Biotechnology,; citation_volume=23; citation_number=25; citation_pages=961-966; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CellDesigner.; citation_author=A Funahashi; citation_number=26; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Depicting signaling cascades.; citation_author=ML Blinov; citation_journal_title=Nature Biotechnology,; citation_volume=24; citation_number=27; citation_pages=137-138; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The dynamics of cell cycle regulation.; citation_author=JJ Tyson; citation_author=A Csikasz-Nagy; citation_author=B Novak; citation_journal_title=BioEssays Special Issue: Modelling complex biological systems,; citation_volume=24(12); citation_number=28; citation_pages=1095-1109; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=BioTapestry. Davidson Lab, California Institute of Technology, 2007: p.; citation_author=WJR Longabaugh; citation_author=EH Davidson; citation_author=H Bolouri; citation_number=29; " />
      <meta name="citation_reference" content="citation_title=Object-Process Methodology-A Holistic Systems Paradigm.; citation_author=D Dori; citation_number=30; citation_date=2002; citation_publisher=Springer Verlag; " />
      <meta name="citation_reference" content="citation_title=Gene-Ontology, 2006: p.; citation_number=31; " />
      <meta name="citation_reference" content="citation_title=Developing Complex Systems with Object-Process Methodology using OPCAT. Conceptual Modeling-ER 2003. Lecture Notes in Computer Science (2813).; citation_author=D Dori; citation_author=I Reinhartz-Berger; citation_author=A Sturm; citation_number=32; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multimedia Learning. C,.; citation_author=RE Mayer; citation_number=33; citation_date=2001; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of the gene.; citation_author=J Watson; citation_number=34; citation_date=2004; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Systems-level analyses identify extensive coupling among gene expression machines.; citation_author=K Maciag; citation_journal_title=Mol Syst Biol,; citation_volume=2; citation_number=35; citation_pages=E1-E14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Eukaryotic mRNA decapping.; citation_author=J Coller; citation_author=R Parker; citation_journal_title=Annu Rev Biochem,; citation_volume=73; citation_number=36; citation_pages=861-90; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bringing the role of mRNA decay in the control of gene expression into focus.; citation_author=CJ Wilusz; citation_author=J Wilusz; citation_journal_title=Trends Genet,; citation_volume=20(10); citation_number=37; citation_pages=491-7; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Decapping and decay of messenger RNA occur in cytoplasmic processing bodies.; citation_author=U Sheth; citation_author=R Parker; citation_journal_title=Science,; citation_volume=300(5620); citation_number=38; citation_pages=805-8; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A mouse cytoplasmic exoribonuclease (mXRN1p) with preference for G4 tetraplex substrates.; citation_author=VI Bashkirov; citation_journal_title=J Cell Biol,; citation_volume=136(4); citation_number=39; citation_pages=761-73; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The human LSm1-7 proteins colocalize with the mRNA-degrading enzymes Dcp1/2 and Xrnl in distinct cytoplasmic foci.; citation_author=D Ingelfinger; citation_journal_title=Rna,; citation_volume=8(12); citation_number=40; citation_pages=1489-501; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of a human decapping complex associated with hUpf proteins in nonsense-mediated decay.; citation_author=J Lykke-Andersen; citation_journal_title=Mol Cell Biol,; citation_volume=22(23); citation_number=41; citation_pages=8114-21; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Human Dcp2: a catalytically active mRNA decapping enzyme located in specific cytoplasmic structures.; citation_author=E van Dijk; citation_journal_title=Embo J,; citation_volume=21(24); citation_number=42; citation_pages=6915-24; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Processing bodies require RNA for assembly and contain nontranslating mRNAs.; citation_author=D Teixeira; citation_journal_title=Rna; citation_number=43; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The GW182 protein colocalizes with mRNA degradation associated proteins hDcp1 and hLSm4 in cytoplasmic GW bodies.; citation_author=T Eystathioy; citation_journal_title=Rna,; citation_volume=9(10); citation_number=44; citation_pages=1171-3; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The translational regulator CPEB1 provides a link between dcp1 bodies and stress granules.; citation_author=A Wilczynska; citation_journal_title=J Cell Sci,; citation_volume=118(Pt 5); citation_number=45; citation_pages=981-92; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RNA decapping inside and outside of processing bodies.; citation_author=C Fillman; citation_author=J Lykke-Andersen; citation_journal_title=Curr Opin Cell Biol,; citation_volume=17(3); citation_number=46; citation_pages=326-31; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=P bodies: at the crossroads of post-transcriptional pathways.; citation_author=A Eulalio; citation_author=I Behm-Ansmant; citation_author=E Izaurralde; citation_journal_title=Nat Rev Mol Cell Biol,; citation_volume=8(1); citation_number=47; citation_pages=9-22; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=RNA granules.; citation_author=P Anderson; citation_author=N Kedersha; citation_journal_title=J Cell Biol,; citation_volume=172(6); citation_number=48; citation_pages=803-8; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=P bodies and the control of mRNA translation and degradation. ; citation_author=R Parker; citation_author=U Sheth; citation_journal_title=Mol. Cell; citation_volume=25; citation_number=49; citation_pages=635-646; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Dynamic shuttling of TIA-1 accompanies the recruitment of mRNA to mammalian stress granules.; citation_author=N Kedersha; citation_journal_title=J Cell Biol,; citation_volume=151(6); citation_number=50; citation_pages=1257-68; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A role for the eIF4E-binding protein 4E-T in P-body formation and mRNA decay.; citation_author=MA Ferraiuolo; citation_journal_title=J Cell Biol,; citation_volume=170(6); citation_number=51; citation_pages=913-24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence that ternary complex (eIF2-GTP-tRNA(i)(Met))-deficient preinitiation complexes are core constituents of mammalian stress granules.; citation_author=N Kedersha; citation_journal_title=Mol Biol Cell,; citation_volume=13(1); citation_number=52; citation_pages=195-210; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mammalian stress granules represent sites of accumulation of stalled translation initiation complexes.; citation_author=SR Kimball; citation_journal_title=Am J Physiol Cell Physiol,; citation_volume=284(2); citation_number=53; citation_pages=C273-84; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Stressful initiations.; citation_author=P Anderson; citation_author=N Kedersha; citation_journal_title=J Cell Sci,; citation_volume=115(Pt 16); citation_number=54; citation_pages=3227-34; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Stress granules and processing bodies are dynamically linked sites of mRNP remodeling.; citation_author=N Kedersha; citation_journal_title=J Cell Biol,; citation_volume=169(6); citation_number=55; citation_pages=871-84; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Movement of eukaryotic mRNAs between polysomes and cytoplasmic processing bodies.; citation_author=M Brengues; citation_author=D Teixeira; citation_author=R Parker; citation_journal_title=Science,; citation_volume=310(5747); citation_number=56; citation_pages=486-9; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Release of extraction-resistant mRNA in stationary-phase S. cerevisiae produces a massive increase in transcript abundance in response to stress.; citation_author=AD Aragon; citation_author=GA Quiñones; citation_author=EV Thomas; citation_author=S Roy; citation_author=M Werner-Washburne; citation_journal_title=Genome Biology 7; citation_number=57; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The RNA polymerase II subunit Rpb4p mediates decay of a specific class of mRNAs.; citation_author=R Lotan; citation_journal_title=Genes Dev,; citation_volume=19(24); citation_number=58; citation_pages=3004-16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Look at Messenger RNP Moving through the Nuclear Pore.; citation_author=B Daneholt; citation_journal_title=Cell,; citation_volume=88; citation_number=59; citation_pages=585-588; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Molecular biology. P-bodies mark the spot for controlling protein production.; citation_author=J Marx; citation_journal_title=Science,; citation_volume=310(5749); citation_number=60; citation_pages=764-5; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In vitro reconstitution of eukaryotic translation reveals cooperativity between release factors eRF1 and eRF3.; citation_author=EZ Alkalaeva; citation_journal_title=Cell,; citation_volume=125(6); citation_number=61; citation_pages=1125-36; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A faux 3′-UTR promotes aberrant termination and triggers nonsense-mediated mRNA decay.; citation_author=N Amrani; citation_journal_title=Nature,; citation_volume=432(7013); citation_number=62; citation_pages=112-8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Translation termination factor eRF3 mediates mRNA decay through the regulation of deadenylation.; citation_author=N Hosoda; citation_journal_title=J Biol Chem,; citation_volume=278(40); citation_number=63; citation_pages=38287-91; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The decapping enzyme Dcp1 participates in translation termination through its interaction with the release factor eRF3 in budding yeast.; citation_author=S Kofuji; citation_journal_title=Biochem Biophys Res Commun,; citation_volume=344(2); citation_number=64; citation_pages=547-53; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Postsynaptic recruitment of Dendrin depends on both dendritic mRNA transport and synaptic anchoring.; citation_author=J Kremerskothen; citation_journal_title=J Neurochem,; citation_volume=96(6); citation_number=65; citation_pages=1659-66; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle"/>
    <meta name="twitter:description" content="One of the main obstacles to understanding complex biological systems is the extent and rapid evolution of information, way beyond the capacity individuals to manage and comprehend. Current modeling approaches and tools lack adequate capacity to model concurrently structure and behavior of biological systems. Here we propose Object-Process Methodology (OPM), a holistic conceptual modeling paradigm, as a means to model both diagrammatically and textually biological systems formally and intuitively at any desired number of levels of detail. OPM combines objects, e.g., proteins, and processes, e.g., transcription, in a way that is simple and easily comprehensible to researchers and scholars. As a case in point, we modeled the yeast mRNA lifecycle. The mRNA lifecycle involves mRNA synthesis in the nucleus, mRNA transport to the cytoplasm, and its subsequent translation and degradation therein. Recent studies have identified specific cytoplasmic foci, termed processing bodies that contain large complexes of mRNAs and decay factors. Our OPM model of this cellular subsystem, presented here, led to the discovery of a new constituent of these complexes, the translation termination factor eRF3. Association of eRF3 with processing bodies is observed after a long-term starvation period. We suggest that OPM can eventually serve as a comprehensive evolvable model of the entire living cell system. The model would serve as a research and communication platform, highlighting unknown and uncertain aspects that can be addressed empirically and updated consequently while maintaining consistency."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000872.g007"/>

  <meta property="og:title" content="Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5755'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7213'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5655&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000872">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000872" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000872&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dov Dori
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:dori@ie.technion.ac.il">dori@ie.technion.ac.il</a></p>

                <p>Affiliation:
                  Faculty of Industrial Engineering and Management, Technion, Israel Institute of Technology, Haifa, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mordechai Choder
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biotechnology, Rappaport Faculty of Medicine, Technion, Israel Institute of Technology, Haifa, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000872</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000872-g001" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000872-g002" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000872-g003" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000872-g004" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000872-g005" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000872-g006" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000872-g007" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000872">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000872" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>One of the main obstacles to understanding complex biological systems is the extent and rapid evolution of information, way beyond the capacity individuals to manage and comprehend. Current modeling approaches and tools lack adequate capacity to model concurrently structure and behavior of biological systems. Here we propose Object-Process Methodology (OPM), a holistic conceptual modeling paradigm, as a means to model both diagrammatically and textually biological systems formally and intuitively at any desired number of levels of detail. OPM combines objects, e.g., proteins, and processes, e.g., transcription, in a way that is simple and easily comprehensible to researchers and scholars. As a case in point, we modeled the yeast mRNA lifecycle. The mRNA lifecycle involves mRNA synthesis in the nucleus, mRNA transport to the cytoplasm, and its subsequent translation and degradation therein. Recent studies have identified specific cytoplasmic foci, termed processing bodies that contain large complexes of mRNAs and decay factors. Our OPM model of this cellular subsystem, presented here, led to the discovery of a new constituent of these complexes, the translation termination factor eRF3. Association of eRF3 with processing bodies is observed after a long-term starvation period. We suggest that OPM can eventually serve as a comprehensive evolvable model of the entire living cell system. The model would serve as a research and communication platform, highlighting unknown and uncertain aspects that can be addressed empirically and updated consequently while maintaining consistency.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Dori D, Choder M (2007) Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle. PLoS ONE 2(9):
          e872.
            doi:10.1371/journal.pone.0000872</p><p><strong>Academic Editor: </strong>Peter Fraser, The Babraham Institute, United Kingdom</p><p><strong>Received:</strong> April 18, 2007; <strong>Accepted:</strong> July 30, 2007; <strong>Published:</strong> September 12, 2007</p><p><strong>Copyright:</strong> © 2007 Dori, Choder. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Israel Science Foundation, founded by the Israel Academy of Sciences and Humanities, by the Rappaport Foundation, by a grant from Paamei Tikva, by a grant from the Bernard M. Gordon Center for Systems Engineering at the Technion, and by a grant from the Elias Fund for Medical Research at the Technion.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Recent years have witnessed unprecedented increases in the number, variety and complexity of information resources available to researchers in the life sciences. We are at a turning point in biological research, where emphasis is shifting from the study of a single molecular process to studying complete cellular pathways and the entire cell as a system. This pivotal time for the life sciences is captured by the words of Kitano: “a <em>transition is occurring in biology from the molecular level to the system level that promises to revolutionize our understanding of complex biological regulatory systems and to provide major new opportunities for practical application of such knowledge</em>” <a href="#pone.0000872-Kitano1">[1]</a>. There is now a drive to acquire system-level comprehension of the countless pieces of information that have been gathered thanks to decades of meticulous laboratory research by thousands of scientists. These efforts, many of which are currently considered as contributions to Systems Biology, are aimed at understanding the underlying structure and behavior of biological systems at the molecular, cellular, organism, and habitat levels. Kitano <a href="#pone.0000872-Kitano1">[1]</a> also noted that new tools, ranging from experimental devices to software and analytical methods, are required if we are to meet the challenges of systems biology.</p>

<h4>Overview of the emerging systems biology field</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>In November 2006, Nature Cell Biology and Nature Reviews Molecular Cell Biology published jointly on the Web (<a href="http://www.nature.com/focus/systemsbiologyuserguide">http://www.nature.com/focus/systemsbiolo​gyuserguide</a>) Systems Biology: a User's Guide <a href="#pone.0000872-Kritikou1">[2]</a>. This is a collection of Review-type articles concerning the most important approaches and challenges in systems biology. The editors of the guide echo the words of Davidson et al. <a href="#pone.0000872-Davidson1">[3]</a> “<em>systems biology is essential if we are ever to make sense of biological complexity, as intuitive ‘conceptual’ models quickly reach their limits beyond simple linear pathways</em>”. Moreover, they affirm: “<em>The time has come for molecular cell biologists, computer scientists and mathematicians to embrace each other's approaches, as is commonplace in the physical sciences. … More importantly, there is an urgent need to make the next generation of molecular cell biologists ‘systems savvy’. The traditional segregation in higher education of biology from mathematics and physics presents challenges and requires an integration of these subjects for the biologists of the future.</em>” In the Editorial, systems biology is defined broadly as the integration of complex and highly diverse biological information into a holistic, quantitative and predictive conceptual framework <a href="#pone.0000872-Kritikou1">[2]</a>. A key notion here, as in the work of Davidson et al. <a href="#pone.0000872-Davidson1">[3]</a>, is the ability of system modeling to be compatible with empirical research, but more specifically, it is suggested that the model must foster empirical predictions. Since systems biology is based on quantitative empirical data and marries informal cartoon-like static flowchart-type models with formal mathematical and computational modeling, it has indeed the promise of generating biological predictions accessible to experimental verification. Moreover, the requirement for modeling and model friendly experimentation to work hand-in-hand, they claim, creates a dynamic interplay that has the potential to result in cyclically better mechanistic understanding and more rigorous interpretation of the system under study. In line with this call to integrate modeling and experimentation, we present a new systems biology modeling approach that has already enabled a significant model-driven experimental finding, which we present as part of this work.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>Also in the editorial, after noting how molecular cell biology is emancipating itself from an informal, reductive, hypothesis-driven approach by embracing high-throughput data acquisition, rigorous quantification and mathematical modeling, it is forecast by Kritikou et al. <a href="#pone.0000872-Kritikou1">[2]</a> that ultimately a “virtual cell” will be developed. With such an aspiration in mind, we have designed a predictive, conceptual object-process-based model of the mRNA lifecycle. As we demonstrate here, this model facilitates novel insights into this integral cellular subsystem. Moreover, our model potentially represents a new tool for modeling other biological systems.</p>
<a id="article1.body1.sec1.sec1.p3" name="article1.body1.sec1.sec1.p3"></a><p>Adopting a mechanistic view, Davidson et al. <a href="#pone.0000872-Davidson1">[3]</a> contended that traditional biological approaches, which focus on determining the functions of one or a few genes at a time, are not adequate for analyzing large regulatory control systems organized as networks. The need for formal modeling is manifested, among other requirements, by the need to express logical expressions even when describing the expression of a single gene. For example, cis-regulatory elements active within defined spatial limits during development often use AND logic, in that two different transcription factors, each present in a given spatial domain, must be bound to the cis-regulatory DNA at once in order for transcription to be activated. Davidson et al. <a href="#pone.0000872-Davidson1">[3]</a> propose that understanding why a given developmental process occurs requires learning the critical inputs and outputs and their key target sites throughout the genomic regulatory system that controls the dynamic process, and moreover relies on experimental determination of the functional significance of each parameter. In summary, Davidson et al. <a href="#pone.0000872-Davidson1">[3]</a> argue that biological complexity dictates the need for formal modeling, but specifically for models that can work hand-in-hand with empirical research.</p>


<h4>Current approaches to biological modeling</h4>
<a id="article1.body1.sec1.sec2.p1" name="article1.body1.sec1.sec2.p1"></a><p>The number of interactions, processes, and transport activities in the living cell is enormous. Therefore, often preceding a quantitative problem of how much or to what extent is the qualitative one of figuring out how and what. Thus, a combined qualitative then quantitative conceptual modeling approach, such as the one adopted in this work, plays a crucial role in facilitating human comprehension of complex cellular mechanisms. Conceptual models advocate the construction of primarily qualitative models, in which biological concepts are put in context with each other in an attempt to gain insights into the function, structure, and dynamics of the biological systems under study. Once a particular, relatively small subsystem in a specific cell location is understood well enough, mathematical tools, such as differential equations, can successfully describe time varying changes. In what follows we survey briefly current approaches and software environments for modeling biological systems, highlighting their advantages and disadvantages.</p>
<a id="article1.body1.sec1.sec2.p2" name="article1.body1.sec1.sec2.p2"></a><p>Modeling efforts in biology are sometimes classified according to their focus on quantitative vs. qualitative aspects. However, oftentimes the two approaches cannot be separated; for example understanding a qualitative process such as the mechanism regulating cell division requires quantitative understanding of this system's dynamics. It must be appreciated that a complex network of protein interactions that influence the activities of cyclin-dependent kinases control major events of the cell cycle, including DNA synthesis, mitosis and cell division. <a href="#pone.0000872-Lovrics1">[4]</a> modeled this network using a set of nonlinear differential equations and by numerical simulation predicted its behavior. However, like other researchers before them, they realized that these computer simulations, despite enabling detailed quantitative comparisons between theory and experiment, give little insight into the qualitative dynamics of the control system and do not reveal how molecular interactions determine the fundamental physiological properties of cell replication. To that end, they used bifurcation diagrams as an analytical tool to obtain new views of the dynamic organization of the cell cycle, the role of checkpoints in assuring the integrity of the genome, and the abnormal regulation of cell cycle events in mutants. They validated these insights by analyzing cell cycle regulation in fission yeast. Here, a combined quantitative and qualitative modeling approach is what ultimately provided genuine insights, but this combination is likely paradigmatic.</p>
<h5>Quantitative Models.</h5><a id="article1.body1.sec1.sec2.sec1.p1" name="article1.body1.sec1.sec2.sec1.p1"></a><p>One quantitative modeling environment is E-Cell <a href="#pone.0000872-Tomita1">[5]</a>, <a href="#pone.0000872-Takahashi1">[6]</a>, which uses general technologies and theoretical supports for computational biology with the grand aim to allow for precise whole cell simulation at the molecular level. E-cell simulates cell behavior by integrating numerically the differential equations described implicitly by reaction rules. It includes numerical simulations and mathematical analysis technologies to predict, obtain or estimate parameters such as reaction rates and concentrations of molecules in the cell. In spite of all its capabilities, E-cell lacks the ability to specify highly complex systems, based on qualitative data, with multiple components.</p>
<a id="article1.body1.sec1.sec2.sec1.p2" name="article1.body1.sec1.sec2.sec1.p2"></a><p>Another example of a mathematics-oriented software modeling environment is the Virtual Cell <a href="#pone.0000872-httpwww.vcell.org.1">[7]</a>, developed for quantitative cell biological research by the National Resource for Cell Analysis and Modeling. Slepchenko <a href="#pone.0000872-Slepchenko1">[8]</a> described applications of this tool to nucleocytoplasmic transport and intracellular calcium dynamics. The Virtual Cell software environment enables sophisticated quantitative dynamics modeling, such as the one described in <a href="#pone.0000872-Slepchenko1">[8]</a>. The biological to mathematical mapping allows for separate use of biological and mathematical components, and includes automatic mathematical simplification using pseudo-steady approximations and mass conservation relationships. This mapping allows for direct specification of mathematical problems, performing simulations and analysis on those systems. However, like E-cell, the Virtual Cell software environment lacks the ability to faithfully describe complex systems that are based on qualitative results. It also lacks the capacity to describe multiple interconnected components that need to be modeled at various levels of detail.</p>
<a id="article1.body1.sec1.sec2.sec1.p3" name="article1.body1.sec1.sec2.sec1.p3"></a><p>Investigating multi-cellular organisms by constructing their conceptual models has been promoted by Harel <a href="#pone.0000872-Harel1">[9]</a>, who also suggested a Turing-like test for biological modeling <a href="#pone.0000872-Harel2">[10]</a>. Formal modeling of C. elegans development has been carried out by Kam et al. <a href="#pone.0000872-Kam1">[11]</a> using a scenario-based approach. They have presented preliminary results of a new approach to the formal modeling of biological phenomena based on the language of live sequence charts with the play-in/play-out process. Keet <a href="#pone.0000872-Keet1">[12]</a> has suggested exploiting existing data better and bringing more structure to the “biological data anarchy” on the Web by enhancing biological information systems with granularity and harnessing Semantic Web technologies.</p>
<a id="article1.body1.sec1.sec2.sec1.p4" name="article1.body1.sec1.sec2.sec1.p4"></a><p>Some quantitative approaches combine the concept of intelligent computer programs, commonly known as software agents, with mathematical models. Applying an OO and agent-based approach, Webb and White <a href="#pone.0000872-Webb1">[13]</a> modeled and simulated metabolic and genetic pathways. Due to limitations of the OO paradigm that stem from its origins in the software domain, this model includes such non-biological artifacts as capsules, ports, and connectors that exchange messages, making it less than intuitive.</p>

<h5>Object-Oriented and UML-based modeling approaches.</h5><a id="article1.body1.sec1.sec2.sec2.p1" name="article1.body1.sec1.sec2.sec2.p1"></a><p>Conceptual modeling originated with efforts to streamline software development some three decades ago. Therefore, the object-oriented approach, which is the currently accepted paradigm in the software engineering community, has been very popular in recent years for modeling systems in general and biological systems in particular. The object-oriented (OO) approach advocates that objects are the prime entities or building blocks of software systems, and this notion has been recently extended via SysML (<a href="http://www.sysml.org">www.sysml.org</a>) to systems in general. A basic tenet of OO modeling is the encapsulation principle, which states that objects, the basic building blocks of the model, encapsulate (own) processes, known as methods or operations. The latter do not have their own right of existence as stand-alone things, making it awkward to try to model biological processes. Cell-level biological processes usually involve a host of input, output, and facilitating molecules of all kinds, which are the objects. Since the OO modeling paradigm advocates that each process be a subordinate of some object, an arbitrary choice must be made as to which object is the owner of the process being modeled. This enforced subordination inevitably leads to a counter-intuitive model right from the outset.</p>
<a id="article1.body1.sec1.sec2.sec2.p2" name="article1.body1.sec1.sec2.sec2.p2"></a><p>BioUML, <a href="#pone.0000872-BioUML1">[14]</a> is an open source software framework for systems biology, which is based on Unified Modeling Language, UML <a href="#pone.0000872-Booch1">[15]</a>, <a href="#pone.0000872-UML21">[16]</a>, the industry standard in software development. UML caters to the OO paradigm in that its terminology and notation closely follow the notions and capabilities of current OO programming languages.</p>
<a id="article1.body1.sec1.sec2.sec2.p3" name="article1.body1.sec1.sec2.sec2.p3"></a><p>UML is built on the premise that “Modeling is the designing of software applications before coding” <a href="#pone.0000872-1">[17]</a>. UML-based modeling approaches like BioUML are object-oriented, meaning that their main building blocks are objects, which are primary static entities that own processes. The inherent orientation of UML towards software, its unnecessary complexity <a href="#pone.0000872-Siau1">[18]</a> and its model multiplicity problems <a href="#pone.0000872-Peleg1">[19]</a> cause intra- and inter-model consistency problems <a href="#pone.0000872-Lange1">[20]</a>.</p>
<a id="article1.body1.sec1.sec2.sec2.p4" name="article1.body1.sec1.sec2.sec2.p4"></a><p>In general, the current Object-Oriented approach to modeling and developing software systems is not suitable for representing effectively biological concepts, because, as argued, it cannot model concurrently in a single type of diagram both the objects, e.g., a protein, and the processes, e.g., transcription, that transform (create, consume, or change the state of) these objects. UML 2.0 <a href="#pone.0000872-UML21">[16]</a>, for example, includes 13 different types of models, each with its own diagram type, separate set of symbols and concepts. Moreover, the lack of the process as a stand-alone concept in the OO modeling approach is a major hindrance for modeling biological systems, which are mostly process-intensive. Finally, many software engineers find it difficult to master the UML modeling framework, making it unrealistic to expect biologists to employ it in a valuable way to model biological systems.</p>
<a id="article1.body1.sec1.sec2.sec2.p5" name="article1.body1.sec1.sec2.sec2.p5"></a><p>Systems Biology Markup Language, SBML <a href="#pone.0000872-Hucka1">[21]</a> is an open, XML-based format for representing biochemical reaction networks and describing models common to research in many areas of computational biology, including cell signaling pathways, metabolic pathways, and gene regulation. SBML data objects use a graphical notation based upon UML, which in turn is translated into XML. CellML <a href="#pone.0000872-Hedley1">[22]</a> is another XML-based language for storage, sharing, exchange, and reuse of computer-based mathematical models. It includes information about model structure, equations describing processes, and metadata to search for model components.</p>
<a id="article1.body1.sec1.sec2.sec2.p6" name="article1.body1.sec1.sec2.sec2.p6"></a><p>The recent Systems Modeling Language, SysML <a href="#pone.0000872-2">[23]</a> initiative offers no solutions to the problems of current OO approaches to modeling, as it is based on UML and therefore suffers from most of UML's deficiencies, namely multiple diagram types and segregation between structure and behavior.</p>

<h5>Specialized modeling frameworks.</h5><a id="article1.body1.sec1.sec2.sec3.p1" name="article1.body1.sec1.sec2.sec3.p1"></a><p>Kohn <a href="#pone.0000872-Kohn1">[24]</a> has proposed a graphical method for mapping bioregulatory networks and representing multimolecular complexes, protein modifications, and actions at cell membranes and between protein domains. The symbol conventions, defined for these molecular interaction maps, accommodate multiprotein assemblies and protein modifications and thus can generate combinatorially large numbers of molecular species. However, most of the 20 or so pictogram symbols are highly specialized. For example, one of them is defined as “Transport of Protein A from cytosol to nucleus”. Clearly, this method is limited to modeling very specific systems.</p>
<a id="article1.body1.sec1.sec2.sec3.p2" name="article1.body1.sec1.sec2.sec3.p2"></a><p>In general, when considering human-readable diagrammatic representations, it is notable that the current informal ways most biologists draw diagrams means that correct biological interpretation depends entirely on the reader's knowledge <a href="#pone.0000872-Kitano2">[25]</a>. Kitano et al. <a href="#pone.0000872-Funahashi1">[26]</a> recount examples where an arrow symbol has four different potential interpretations and indicate rightly that such ambiguities become a major problem as the size and complexity of the system increases, highlighting the need for formality to avoid ambiguity. Process diagrams proposed by <a href="#pone.0000872-Kitano2">[25]</a> that make use of CellDesigner <a href="#pone.0000872-Funahashi1">[26]</a> are state transition-based.</p>
<a id="article1.body1.sec1.sec2.sec3.p3" name="article1.body1.sec1.sec2.sec3.p3"></a><p>Using different arrowhead shapes CellDesigner diagrams focus on conveying the semantics of several process types prevalent in signaling, such as translocation, catalysis, splitting, phosphorylation, or state transition. Formalized process diagrams have been used to describe signal-transduction cascades and pathway maps, and are readable and precise as long as the network is not too large. However, scalability is an issue. There is no way to refine mechanisms and designating new pictograms for each new reaction type is problematic. Moreover, as Blinov et al. <a href="#pone.0000872-Blinov1">[27]</a> observe, because process diagrams require explicit representation of all the species (which in our ontology are referred to as objects) at some level, they omit the vast majority of species and reactions, which are processes in our ontology, that could potentially be generated during signaling.</p>
<a id="article1.body1.sec1.sec2.sec3.p4" name="article1.body1.sec1.sec2.sec3.p4"></a><p>In an attempt to solve this problem, Blivnov et al. have introduced graphical rules to allow the connectivity of proteins in a complex to be represented explicitly. These rules provide a means to visualize comprehensibly protein-protein interactions. Nevertheless, for more general biological modeling, process maps are likely to be of unmanageable size due to combinatorial complexity.</p>
<a id="article1.body1.sec1.sec2.sec3.p5" name="article1.body1.sec1.sec2.sec3.p5"></a><p>An example of a combined quantitative-qualitative model is the work of Tyson et al. <a href="#pone.0000872-Tyson1">[28]</a>, who modeled the dynamics of cell cycle regulation. They applied a systems dynamics-based approach and bifurcation diagrams to provide a new perspective on cell cycle checkpoints and mutant phenotypes in fission yeast. In this model, qualitative changes can occur, for example, when a stable steady state loses its stability or even ceases to exist and is replaced by an oscillatory solution. Such an event is in the nature of the recurrent solutions of a dynamic system and is necessary for a model attempting to characterize the cell cycle. These qualitative changes, called bifurcations, happen at specific values of the parameters termed bifurcation points and are described by a one-parameter bifurcation diagram. These are two-dimensional graphs, where each axis shows some quantity and the quantitative analysis yields certain meaningful qualitative results, such as the G1, G2, or metaphase checkpoints in the fission yeast cell cycle.</p>
<a id="article1.body1.sec1.sec2.sec3.p6" name="article1.body1.sec1.sec2.sec3.p6"></a><p>BioTapestry <a href="#pone.0000872-Longabaugh1">[29]</a> is the latest interactive tool for building, visualizing, and simulating genetic regulatory networks. It is designed around the concept of a developmental network model, intended to handle large scale models and represent systems that exhibit increasing complexity over time. The system supports data generated by perturbing the expression of specific genes, portrays views of the network during development, and lays out network models.</p>

<h5>Problems with current models.</h5><a id="article1.body1.sec1.sec2.sec4.p1" name="article1.body1.sec1.sec2.sec4.p1"></a><p>The current state of affairs, reflected in this survey, is that there are quite a number of modeling approaches and software environments for modeling biological systems, but many of them are object-oriented, hindering direct and explicit process modeling, which is at the heart of systems biology. Moreover, since most approaches are non-scalable and specialized for specific types of cellular reactions or subsystems, they cannot be extended naturally to modeling the entire cell, not to mention organisms, societies, habitats and ecologies.</p>
<a id="article1.body1.sec1.sec2.sec4.p2" name="article1.body1.sec1.sec2.sec4.p2"></a><p>Perhaps most importantly, according to the definition of systems biology, models are supposed to advance research, yet none of the existing modeling approaches or systems have been shown to promote innovative questions that trigger experiments to confirm or refute assumptions emerging from the model. Such a disappointing situation indicates that a totally different modeling approach is in order. This situation was a major stimulus for the work presented here, a non-traditional conceptual modeling approach that has already stimulated a new empirical finding concerning the mRNA lifecycle.</p>

<h5>Our approach to biological modeling.</h5><a id="article1.body1.sec1.sec2.sec5.p1" name="article1.body1.sec1.sec2.sec5.p1"></a><p>Representing the vast amount of ever increasing knowledge formally, yet accessibly, can be compared to putting the pieces of a gigantic puzzle together, mandating adoption of a common evolving grand model. The model needs to be founded on a compact generic set of the most basic ontological building blocks in order for it to be general enough to serve as a basis for modeling the gamut of biological systems, from molecules to ecosystems.</p>
<a id="article1.body1.sec1.sec2.sec5.p2" name="article1.body1.sec1.sec2.sec5.p2"></a><p>We submit that stateful objects and processes that transform them, along with several types of links, as advocated by OPM—Object-Process Methodology <a href="#pone.0000872-Dori1">[30]</a>, constitute a mandatory and sufficient set of ontological building blocks to enable conceptual modeling of biological systems with various scales and complexities. Moreover, such building blocks allow reasoning that links theory with empiricism. We envision an OPM-based comprehensive shared Web-accessible modeling framework of the entire cell, which, if and when created, would enable the evolution of state-of-the-art knowledge in biology. This model will keep pace with the rapid evolution of knowledge, and will be revised constantly and updated with new findings and conjectures.</p>
<a id="article1.body1.sec1.sec2.sec5.p3" name="article1.body1.sec1.sec2.sec5.p3"></a><p>In the present study we aimed to carry out a modest first step toward this admittedly ambitious goal. As the yeast mRNA life cycle is a key cellular system, we chose it to be our case in point for conceptual modeling that employs Object-Process Methodology. In the next section we explain in more detail why we chose Object-Process Methodology, OPM.</p>

<h5>Object-Process Methodology.</h5><a id="article1.body1.sec1.sec2.sec6.p1" name="article1.body1.sec1.sec2.sec6.p1"></a><p>Object-Process Methodology, OPM <a href="#pone.0000872-Dori1">[30]</a> is a holistic approach to the study and development of complex systems that caters to human intuition while maintaining a formal framework. The living cell is a prime example of a highly complex system, in which the two main system aspects—structure and behavior—are highly intertwined and hard to separate. Motivated by the requirement of a single model to represent these two major system aspects, OPM is founded upon two elementary building blocks—objects and processes—which represent concurrently the system's structure, i.e., the objects, or components, that comprise the system, and behavior, i.e., the processes that transform the system's objects by creating them, consuming them, or changing their states, in a balanced way without highlighting one at the expense of the other.</p>
<a id="article1.body1.sec1.sec2.sec6.p2" name="article1.body1.sec1.sec2.sec6.p2"></a><p>The elements of OPM ontology are entities and links. A complete list of OPM elements with their symbols and definitions is provided in <a href="#pone.0000872.s001">Table S1</a>. Entities, the basic building blocks of any system modeled using OPM, are of three types: <em>objects</em>, possibly with <em>states</em> (stateful objects), and <em>processes</em>. An object is a thing that exists, possibly in some state, while a process is a thing that can transform objects. More specifically, a process is a thing that transforms objects, namely creates one or more objects, consumes one or more objects, or changes the state of one or more objects. Examples of biological objects are Protein, Cell, and Organism, and examples of biological processes are Cleavage, Mitosis, and Birth.</p>
<a id="article1.body1.sec1.sec2.sec6.p3" name="article1.body1.sec1.sec2.sec6.p3"></a><p>A link can be structural or procedural. A structural link expresses a static, time-independent relation between pairs of entities. The four fundamental structural relations are: aggregation-participation, generalization-specialization, exhibition-characterization, and classification-instantiation. An example of using the aggregation-participation structural relation is derived from the phrase “The eukaryotic cytoskeleton is composed of microfilaments, intermediate filaments and microtubules.” (<a href="http://en.wikipedia.org/wiki/Cell_(biology)#Subcellular_components">http://en.wikipedia.org/wiki/Cell_(biolo​gy)#Subcellular_components</a>) In OPM, this statement is interpreted such that the Eukaryotic Cytoskeleton is the aggregating object, the whole, which consists of the three objects which are parts of the Eukaryotic Cytoskeleton, each being a set of objects: the Microfilaments Set, the Intermediate Filaments Set, and the Microtubules Set. Unidirectional and bidirectional tagged structural links enable creation of additional user-defined links with specified semantics. A procedural link connects entities (objects, processes, and states) to describe the behavior of a system. The behavior is manifested in three major ways: (1) a process can <em>transform</em> (generate, consume, or change the state of) one or more objects; (2) an object can <em>enable</em> one or more processes without being transformed by them, in which case it acts as an <em>enabler</em>, i.e., a human <em>agent</em> or an inanimate <em>instrument</em>; and (3) entities can <em>trigger events</em> that invoke processes if some conditions are met. Accordingly, a procedural link can be a transformation link, an enabling link, or an event link. A <em>transformation link</em> expresses object transformation, i.e., object consumption, generation, or state change. An <em>enabling link</em> expresses the need for a (possibly state-specified) object to be present in order for the enabled process to occur. The enabled process does not transform the enabling object. An <em>event link</em> connects a triggering entity (object, process, or state) with a process that it invokes.</p>
<a id="article1.body1.sec1.sec2.sec6.p4" name="article1.body1.sec1.sec2.sec6.p4"></a><p>The Gene-Ontology (GO) <a href="#pone.0000872-3">[31]</a> project sets out to provide a defined, universal vocabulary for describing gene and gene product attributes in any organism. The three organizing principles of GO are cellular component, biological process, and molecular function. A comparison between GO principles and OPM entities is useful as it emphasizes the advantages of OPM ontology. A cellular component corresponds to an OPM object and a biological process corresponds to an OPM process. However, molecular function does not have a clear OPM equivalent. According to the definition of process in <a href="#pone.0000872-3">[31]</a>, a biological process is a “series of events accomplished by one or more ordered assemblies of molecular functions.” Examples include signal transduction and alpha-glucoside transport. A GO molecular function “describes activities, such as catalytic or binding activities, that occur at the molecular level.” Examples include catalytic activity, binding, or adenylate cyclase activity. Problems with this definition for molecular function are admitted in <a href="#pone.0000872-3">[31]</a>, “It can be difficult to distinguish between a biological process and a molecular function, but the general rule is that a process must have more than one distinct step.” We contend that any ontology that lacks precise, clear-cut definitions, and relies on examples as part of the definition, is problematic. What is the meaning of step? Is it identical to function, and if so, is function in turn identical to activity? If so–why use so many terms, and if not, how do they differ? Indeed, it is unclear how to differentiate between GO functions and GO processes. OPM takes another approach where it does not distinguish between simple and complex processes, just as it does not distinguish between simple and complex objects. For example the GO molecular function “pre-mRNA 3′-splice site binding” (taken from the actual GO file) in OPM ontology is the OPM process of binding of the cellular component (OPM object) “pre-mRNA 3′-splice site”, which changes that object from state unbound to state bound. We advocate that the OPM ontology is more precise and intuitive.</p>
<a id="article1.body1.sec1.sec2.sec6.p5" name="article1.body1.sec1.sec2.sec6.p5"></a><p>Two semantically equivalent modalities, one graphic and the other textual, are used to describe each OPM model. A set of inter-related Object-Process Diagrams (OPDs), showing portions of the system at various levels of detail, constitute the graphical, visual OPM formalism. Each OPM element is denoted by a symbol in an OPD, and the OPD syntax specifies correct and consistent ways by which entities can be connected via structural and procedural links, such that each legal entity-link-entity combination bears specific, unambiguous semantics. OPCAT <a href="#pone.0000872-Dori2">[32]</a> is a software environment that supports OPM-based system modeling and evolution.</p>
<a id="article1.body1.sec1.sec2.sec6.p6" name="article1.body1.sec1.sec2.sec6.p6"></a><p>The Object-Process Language (OPL), which is the textual counterpart of the graphical OPD, is a dual-purpose language, oriented towards humans as well as machines. Catering to human needs, OPL is designed as a subset of English, which serves domain experts (e.g., biologists) and system architects, engaged jointly in modeling a complex system. Every OPD construct is expressed by a semantically equivalent OPL sentence or phrase. According to the modality principle of the cognitive theory of multimodal learning <a href="#pone.0000872-Mayer1">[33]</a>, this dual graphic/textual representation of the OPM model increases the human processing capability. Indeed, it has been our experience that human understanding of the OPM model is enhanced by the convenient opportunity of reflecting upon both the graphic and textual model representations, whereby what is missed in one modality can be grasped when considering the other one.</p>
<a id="article1.body1.sec1.sec2.sec6.p7" name="article1.body1.sec1.sec2.sec6.p7"></a><p><a href="#pone-0000872-g001">Figure 1(A)</a> is an Object-Process Diagram (OPD), showing the top level, bird's eye view of the process mRNA Lifecycle, which generates the object Protein from the object Amino Acid Set. <a href="#pone-0000872-g001">Figure 1(B)</a> lists the corresponding two OPL sentences that were generated automatically by OPCAT:</p>
<a id="article1.body1.sec1.sec2.sec6.p8" name="article1.body1.sec1.sec2.sec6.p8"></a><p>mRNA Lifecycle consumes Amino Acid Set, Ribonucleotide Set, and mRNA.</p>
<a id="article1.body1.sec1.sec2.sec6.p9" name="article1.body1.sec1.sec2.sec6.p9"></a><p>mRNA Lifecycle yields mRNA, Ribonucleotide Set, and Protein.</p>
<div class="figure" id="pone-0000872-g001"><div class="img"><a name="pone-0000872-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A top-level view of the mRNA Lifecycle OPM model.</span></strong></p><a id="article1.body1.sec1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec1.sec2.sec6.fig1.caption1.p1"></a><p>(A) An Object-Process Diagram (OPD) showing the top-level, bird's eye view of the process mRNA Lifecycle, which generates the object Protein from the object Amino Acid Set. The process, marked as the blue ellipse, shows the generation of mRNA from Ribonucleotide Set and the degradation of mRNA back to Ribonucleotide Set. (B) The corresponding Object-Process Language (OPL) text that was generated automatically by OPCAT, the software that supports OPM modeling.</p>
<span>doi:10.1371/journal.pone.0000872.g001</span></div><a id="article1.body1.sec1.sec2.sec6.p10" name="article1.body1.sec1.sec2.sec6.p10"></a><p>Examining these sentences, we see that each arrow from an object to the process (mRNA Lifecycle) has the semantics of consumption (degradation, or destruction), while each arrow from the process to an object has the semantics of result (creation, or generation). During the process mRNA Lifecycle, Ribonucleotide Set and mRNA are both consumed and generated, while Amino Acid Set is consumed to create Protein.</p>

<h5>Complexity Management.</h5><a id="article1.body1.sec1.sec2.sec7.p1" name="article1.body1.sec1.sec2.sec7.p1"></a><p>A major problem with most graphic modeling approaches is their scalability. As the system's complexity increases, the graphic model becomes cluttered with symbols and their connecting links. The limited channel capacity <a href="#pone.0000872-Mayer1">[33]</a> is a cognitive principle which states that there is an upper limit on the amount of detail a human can process before being overwhelmed. This principle is addressed by OPM and implemented by OPCAT via three abstraction/refinement mechanisms. These enable complexity management by providing for the creation of a set of interrelated OPDs (along with their corresponding OPL paragraphs) that are limited in size, thereby avoiding information overload and enabling comfortable human cognitive processing. The three refinement/abstraction mechanisms are: (1) <em>unfolding/folding</em>, which is used for refining/abstracting the structural hierarchy of a thing and is applied by default to objects; (2) <em>in-zooming/out-zooming</em>, which exposes/hides the inner details of a thing within its frame and is applied primarily to processes; and (3) <em>state expressing/suppressing</em>, which exposes/hides the states of an object.</p>
<a id="article1.body1.sec1.sec2.sec7.p2" name="article1.body1.sec1.sec2.sec7.p2"></a><p><a href="#pone-0000872-g002">Figure 2</a> provides examples of these three abstraction/refinement mechanisms. The mRNA Lifecycle process from <a href="#pone-0000872-g001">Figure 1</a> is in-zoomed, exposing three main subprocesses: Transcription, Translation, and Post-Ribosomal Processing. Two examples of unfolding are shown in <a href="#pone-0000872-g002">Figure 2</a>. One is the unfolding of mRNA Polymerase II as <em>part</em> of Nucleus. The other is the unfolding of Localization as an <em>attribute</em> of mRNA. Finally, state expression is exemplified when the possible states of Localization—nucleus, P body, and ribosome—are expressed. Using flexible combinations of these abstraction/refinement mechanisms, OPM enables a system to be specified to any desired level of detail, without losing legibility or comprehension of the specification. The complete OPM system specification is expressed graphically by the resulting set of consistent, inter-related OPDs, and textually by the corresponding OPL paragraphs.</p>
<div class="figure" id="pone-0000872-g002"><div class="img"><a name="pone-0000872-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Zooming into the mRNA Lifecycle process from <a href="#pone-0000872-g001">Figure 1</a>.</span></strong></p><a id="article1.body1.sec1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec1.sec2.sec7.fig1.caption1.p1"></a><p>(A) An OPD in which the mRNA Lifecycle process from <a href="#pone-0000872-g001">Figure 1</a> is in-zoomed into the three main subprocesses Transcription, Translation, and Post-ribosomal Processing. Time in an OPD flows from top to bottom, so Transcription is the initial process, Translation comes next, and finally Post-ribosomal Processing. While more detailed, the view of the system in this figure is fully consistent with its more abstract ancestor, shown in <a href="#pone-0000872-g001">Figure 1</a>. For the sake of simplicity transport-related processes are not depicted here yet. mRNA exhibits the attribute Localization, which can be at the states ribosome, nucleus, or P body. This relation between mRNA and its Localization attribute is denoted by the black-in-white triangle connecting these two objects. A thick process contour indicates that this process is in-zoomed in a different, more detailed OPD to show the inner content of that process. For example, the Translation process is zoomed in <a href="#pone-0000872-g004">Figure 4</a>. (B) The OPL text that explains the graphical model. For example, the OPL sentence Transcription occurs if Localization is nucleus. is denoted in the OPD by the link from the nucleus Localization to the Transcription process. In OPM, this link is called condition link, and it is denoted by the letter ‘c’ inside the link's circle end. In free text this means that transcription takes place in the nucleus.</p>
<span>doi:10.1371/journal.pone.0000872.g002</span></div>
<h5>The mRNA Lifecycle.</h5><a id="article1.body1.sec1.sec2.sec8.p1" name="article1.body1.sec1.sec2.sec8.p1"></a><p>The expression of protein encoding genes is a complex process that determines which genes are expressed as proteins at any given time, as well as the relative levels of these proteins <a href="#pone.0000872-Watson1">[34]</a>. This process involves several distinct stages, (i) RNA synthesis, or transcription, (ii) RNA processing (after processing is completed, the RNA is considered mRNA), (iii) mRNA transport from the nucleus to the cytoplasm (in eukaryotes), (iv) protein synthesis, or translation, (v) mRNA degradation, and (vi) posttranslational modifications (including degradation) of the proteins. In eukaryotes, transcription is carried out by three different RNA polymerases, each responsible for transcribing a distinct class of RNAs. RNA polymerase II (pol II) is responsible for transcribing protein-encoding RNAs, namely mRNAs, which are the focus of this paper. Work from many laboratories has established that regulation of the aforementioned stages of pol II-mediated gene expression is coordinated <a href="#pone.0000872-Maciag1">[35]</a>. Thus, in order to understand the expression of protein-encoding genes we need to consider the entire multi-stage process, as each stage can be regarded as a subdivision of a continuous gene expression process. Correctly and accurately specifying this complex process is a formidable task, beyond the realms of free text. Clearly, it calls for the use of an appropriate modeling language and methodology.</p>
<a id="article1.body1.sec1.sec2.sec8.p2" name="article1.body1.sec1.sec2.sec8.p2"></a><p>Transcription by pol II, the first stage in the expression of protein-encoding genes, produces RNA–the primary transcript. This primary transcript is processed to yield an mRNA (usually shorter than the primary transcript) that contains a 5′ cap (m(7)GpppN) and 3′ poly(A) tail. These two tags are critical for the appropriate function, localization and stability of the mRNA <a href="#pone.0000872-Watson1">[34]</a>. Following its synthesis in the nucleus, the mRNA is transported to the cytoplasm, where it is recognized by ribosome(s)-the protein synthesis machinery. The mRNA is then used as a template for translation into protein, the amino acid sequence of which is related to the nucleotide sequence of the mRNA <a href="#pone.0000872-Watson1">[34]</a>. The last stage of the mRNA lifecycle in the cytoplasm is its decay, which is carried out by an array of decay factors <a href="#pone.0000872-Coller1">[36]</a>, <a href="#pone.0000872-Wilusz1">[37]</a>. Each one of the stages described above is tightly regulated. Once produced, the mRNA is the key target in the regulation of the gene expression, referred to as post-transcriptional regulation. One aspect of post-transcriptional regulation is at the level of mRNA localization within the cell, for example nuclear vs. cytoplasmic localization.</p>
<a id="article1.body1.sec1.sec2.sec8.p3" name="article1.body1.sec1.sec2.sec8.p3"></a><p>Recently, a new venue for mRNA localization was uncovered that revolutionized our view of how gene expression is regulated post-transcriptionally. Specifically, yeast mRNA can be localized in discrete cytoplasmic foci together with a number of mRNA decay factors and limited repertoire of translation factors, mostly translation repressors. These foci, termed processing bodies (P bodies), represent complexes where mRNA degradation can occur, since mRNA decay intermediates <a href="#pone.0000872-Sheth1">[38]</a> and many factors of the major mRNA decay pathway reside in P bodies <a href="#pone.0000872-Sheth1">[38]</a>–<a href="#pone.0000872-Parker1">[49]</a>. The discovery of P bodies in yeast and related bodies in higher eukaryotes, e.g., dcp bodies, GW bodies, brings the spatial control of macromolecule to the focus of our attention.</p>
<a id="article1.body1.sec1.sec2.sec8.p4" name="article1.body1.sec1.sec2.sec8.p4"></a><p>A different kind of cytoplasmic foci, referred to as stress granules, have been discovered in higher eukaryotes under various stress conditions (recently reviewed in <a href="#pone.0000872-Parker1">[49]</a>, <a href="#pone.0000872-Kedersha1">[50]</a>). Unlike P bodies, these foci contain several translation factors, but not the ribosomal large subunit <a href="#pone.0000872-Ferraiuolo1">[51]</a>, <a href="#pone.0000872-Kedersha2">[52]</a>. Stress granules are believed to be sites where non-translating mRNA resides during stress conditions <a href="#pone.0000872-Kimball1">[53]</a>. Association of mRNAs with stress granules was proposed to represent a mechanism for translational repression. This kind of repression can be reversible, as the mRNA can be transported back to the ribosome when conditions favor translation. Anderson and Kedersha <a href="#pone.0000872-Anderson2">[54]</a> proposed that mRNAs in stress granules are subjected to triage: first they are monitored for integrity and composition, and then they are sorted for productive translational initiation or targeted to degradation. In addition, it has been suggested that stress granules may communicate with P bodies when sorting the mRNA for degradation <a href="#pone.0000872-Kedersha3">[55]</a>. In yeast, the organism under study here, no stress granules have been identified and the bulk of known P bodies do not contain translation factors <a href="#pone.0000872-Teixeira1">[43]</a>. Recently, work from Parker's group has revealed that mRNAs in the yeast P bodies are not necessarily degraded, but rather can be transported to the ribosome for translation <a href="#pone.0000872-Brengues1">[56]</a>. Thus, yeast P bodies may have dual function, carrying out the functions of both mammalian P bodies and stress granules.</p>
<a id="article1.body1.sec1.sec2.sec8.p5" name="article1.body1.sec1.sec2.sec8.p5"></a><p>Most published works to date have used fluorescent microscopy to study P bodies or stress granules. This technology allows the detection of large complexes whose fluorescence is above that of the background, but small P bodies might escape detection. Interestingly, though, Aragon et al. <a href="#pone.0000872-Aragon1">[57]</a> have recently reported that mRNAs in starved yeast cells are sequestered in proteinatious complexes and therefore resist standard extraction procedures. These mRNAs could be recovered by disrupting the complexes with proteases. The released mRNA can then be analyzed using whole genome technology. The authors proposed a plausible model in which the protecting complexes are P bodies <a href="#pone.0000872-Aragon1">[57]</a>. If proved correct, the differential mRNA extraction technique will allow analyzing even small P bodies and also obtain, for the first time, quantitative results at the organism level.</p>
<a id="article1.body1.sec1.sec2.sec8.p6" name="article1.body1.sec1.sec2.sec8.p6"></a><p>It has been contended both for yeast <a href="#pone.0000872-Sheth1">[38]</a>, <a href="#pone.0000872-Anderson2">[54]</a>, <a href="#pone.0000872-Kedersha3">[55]</a> and higher eukaryotes <a href="#pone.0000872-Anderson1">[48]</a>, <a href="#pone.0000872-Brengues1">[56]</a> that there is frequent shuttling of mRNA between the ribosome (or poly-ribosome) and these cytoplasmic bodies. Moreover, it has been suggested that two of the cytoplasmic structures, i.e., the ribosome, which activates translation, and the P body, which represses it, compete for mRNA. The outcome of this competition determines mRNA translatability and hence protein synthesis <a href="#pone.0000872-Lotan1">[58]</a>. To demonstrate shuttling of mRNA between these two complexes, investigators utilized specific drugs. Drugs that block mRNAs within the ribosomes cause P bodies to disassemble, whereas drugs or mutations that compromise mRNA loading onto the ribosomes enhance the assembly of P bodies <a href="#pone.0000872-Sheth1">[38]</a>, <a href="#pone.0000872-Anderson1">[48]</a>, <a href="#pone.0000872-Anderson2">[54]</a>, <a href="#pone.0000872-Kedersha1">[50]</a>, <a href="#pone.0000872-Daneholt1">[59]</a>. Such studies reveal that P bodies are dynamic structures, for their mass varies as the mRNA is transported back and forth between P bodies and ribosomes. It has also become clear that the balance between these two sub-cytoplasmic compartments, and hence the P bodies' size, is responsive to environmental signals <a href="#pone.0000872-Teixeira1">[43]</a>, <a href="#pone.0000872-Anderson1">[48]</a>, <a href="#pone.0000872-Anderson2">[54]</a>, <a href="#pone.0000872-Lotan1">[58]</a>. Thus, external signals, such as starvation, UV irradiation, or changes in osmolarity, can trigger mRNA redistribution between the two compartments. It was the complexity and intricacy of processes constituting the mRNA lifecycle, outlined above, which triggered our realization that conceptual modeling might likely contribute to understanding this particular cellular sub-system. Moreover, we anticipated that the modeling should raise empirically addressable questions, the answers to which would help researchers comprehend better mRNA biology. Indeed, the modeling activity has raised at least one research question, the localization of eRF3 in P bodies, which we have confirmed experimentally.</p>


</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>OPM allows us to model the system under study—the mRNA lifecycle—at various hierarchically arranged levels of detail. We started modeling only established knowledge concerning the mRNA lifecycle. <a href="#pone-0000872-g001">Figure 1A</a> is the System Diagram (SD). It is the top level Object-Process Diagram (OPD), which provides a bird's eye view of the system, illustrating graphically the mRNA Lifecycle system in a nutshell. In the mRNA Lifecycle process, Ribonucletides are consumed to generate mRNA. mRNA is produced by the Synthesis process, which, as the model's next levels of detail reveals, includes Transcription and Nucleo-cytoplasmic Transport. The main product of the mRNA Lifecycle process is Protein, which is synthesized during the Translation process. Stages that follow Translation take place in the P bodies and might produce Ribonucletides, as detailed below. To complete the lifecycle, one option that occurs following translation (see below) is that the mRNA is decomposed back into its constituent Ribonucletides.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p><a href="#pone-0000872-g001">Figure 1(B)</a> is the automatically generated textual description, called Object-Process Language (OPL), expressing what the Object-Process Diagram (OPD) shown in <a href="#pone-0000872-g001">Figure 1(A)</a> tells us graphically. This OPL paragraph, a collection of OPL sentences, is equivalent in its informational content to its corresponding OPD.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><a href="#pone-0000872-g002">Figure 2(A)</a> shows an OPD in which the mRNA Lifecycle process from <a href="#pone-0000872-g001">Figure 1</a> is zoomed in to expose its three main sub-processes, Transcription, Translation, and Post-ribosomal Processing. This enables us to visualize the details of the mRNA Lifecycle process. The convention is that within an in-zoomed process, there is a timeline—the Y axis of the diagram—that flows from the top of the in-zoomed process ellipse to its bottom. Accordingly, Transcription happens first, followed by Translation, and finally the Post-ribosomal Processing takes place. Parallel or alternative processes are depicted at the same height. Cycles and loops are easily expressed, and the mRNA cycle as a whole is an example. As discussed in the introduction, an important feature of the mRNA Lifecycle is the Localization of the mRNA, which we model as an attribute of mRNA (indicated by the black-inside-white triangle link), with each Localization state being a concrete localization. It is common knowledge that mRNA is produced in the nucleus and then transported to the cytoplasm, where it is translated and degraded <a href="#pone.0000872-Watson1">[34]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the cytoplasm, mRNA can be located in various complexes, such as the ribosome or P body. It is quite possible that the yeast cytoplasm contains other large bodies that accommodate mRNA. However, as such bodies have not been reported, the only two cytoplasmic mRNA locations in our model are the ribosome (including also poly-ribosome) and P bodies. It has not been established whether mRNA can move in the cytoplasmic matrix, unattached to any complex. Trying to include this option in our initial model rendered the model more complex with no tangible benefit. Following Occum's Razor, we therefore assumed the simplest option, i.e., that mRNA does not reside in the cytoplasmic matrix as an unbound molecule. Thus, <a href="#pone-0000872-g001">Figure 1(A)</a> shows only three mRNA Localization states: nucleus, P body, and ribosome. Interestingly, when only these three mRNA Localization states are considered, a question arises regarding what is the first cytoplasmic location of mRNA after it is transported from the nucleus.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The model leaves only two options: either ribosome or P body. Although ribosomes have been implicated as the mRNA acceptor in the cytoplasm <a href="#pone.0000872-Daneholt1">[59]</a>, an as yet unconsidered possibility raised by this model is that in some cases a P body can be the first cytoplasmic location for mRNA. Indeed, Jean Marx has noted that P bodies are found around the nucleus <a href="#pone.0000872-Marx1">[60]</a>. The conjecture that P bodies may be the first cytoplasmic location for mRNA illustrates the utility and the value of a formal, expressive conceptual modeling approach in provoking new ideas and viewpoints. As we show below, our modeling framework indeed triggered an experiment that provided new, significant information related to the mRNA lifecycle.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p><a href="#pone-0000872-g003">Figure 3</a> shows a refinement of the mRNA lifecycle OPM model of <a href="#pone-0000872-g002">Figure 2</a>, in which three mRNA Transport processes, marked in cyan, have been added: Nucleo-cytoplasmic Transport, P Body-Ribosome Transport, and Ribosome-P body Transport. According to our model, the cytoplasmic mRNA is localized either in the Ribosome or in the P body. The Nucleo-cytoplasmic Transport, which requires three factors, eIF Set, Dhh1p, and Pat1p, changes the Localization of the mRNA from nucleus to P body. A pair of input/output links, the cyan and purple arrows, denote this transport. The two other transport processes are the inverse of each other: P Body-Ribosome Transport changes the Localization of the mRNA from P body to ribosome and induces translation initiation, while Ribosome-P body Transport does the opposite, repressing translation. As in the Nucleo-cytoplasmic Transport, for each of these two transport processes there is a pair of input/output links, the cyan and purple arrows. If we follow the sequence of these alternating input/output links (cyan and purple arrows), we get that the Localization of mRNA, which starts at the nucleus, changes to P body and then to ribosome. This can be followed by alternating between ribosome and P body, possibly many times, until the mRNA is consumed (degraded) by the Post-ribosomal Processing, as indicated by the consumption link—the arrow in <a href="#pone-0000872-g003">Figure 3</a> from mRNA to that process. As discussed in the Introduction, dynamic transport of mRNAs between the two compartments has been proposed, but little is known about the transport mechanism, in particular, whether it is direct or involves the cytoplasmic matrix. This black box is yet another example of a model-triggered potential area of research. Two mRNA Decay Factors, Pat1p and Dhh1p, are implicated in the transport of mRNA from the ribosome to the P bodies <a href="#pone.0000872-Anderson2">[54]</a>. These two factors are therefore depicted as instruments that enable the Ribosome-P body Transport process.</p>
<div class="figure" id="pone-0000872-g003"><div class="img"><a name="pone-0000872-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Adding the mRNA transport processes.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) Three mRNA transport processes, marked in cyan, have been added to the OPD of <a href="#pone-0000872-g002">Figure 2:</a> Nucleo-cytoplasmic Transport, P Body-Ribosome Transport, and Ribosome-P body Transport. According to our model, the cytoplasmic mRNA is localized either in the ribosome or in the P bodies. For simplicity, possible transport from the nucleus to the ribosome is not modeled here. Therefore, the Nucleo-cytoplasmic Transport changes the Localization of the mRNA from nucleus to P body. The two other transport processes are the inverse of each other: P Body-Ribosome Transport, which changes the Localization of the mRNA from P body to ribosome, while Ribosome-P body Transport, which does the opposite. (B) The sentences of the OPL paragraph listing the processes and objects into which mRNA Lifecycle zooms and how each transport process changes the value of the Localization attribute of mRNA between nucleus, P body, and ribosome.</p>
<span>doi:10.1371/journal.pone.0000872.g003</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In <a href="#pone-0000872-g004">Figure 4</a>, the Translation process is in-zoomed, exposing its three subprocesses: Elongation, Termination, and Protein Cleavage&amp;Releasing, which are executed in this order. Translation initiation is depicted in <a href="#pone-0000872-g003">Figure 3</a> as P Body-Ribosome Transport. The corresponding factors involved as instruments in these processes are also shown linked to the processes with an instrument link (a line ending with a circle at the process end). The object eEF Set is the instrument for the Elongation process, while eRF Set with its members, the factors eRF1 and eRF3, is the instrument for the Protein Cleavage&amp;Releasing process.</p>
<div class="figure" id="pone-0000872-g004"><div class="img"><a name="pone-0000872-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Zooming into the Translation process.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>(A) Zooming into the Translation process exposes its three subprocesses that follow P Body-Ribosome Transport (initiation): Elongation, Termination, and Protein Cleavage&amp;Releasing. The corresponding factors involved as instruments in these processes are also shown linked to the processes with an instrument link (a line ending with a circle at the process end). The object eEF Set is the instrument for the Elongation process, while eRF Set with its members, the factors eRF1 and eRF3, is the instrument for the Protein Cleavage&amp;Releasing process. Our conjecture is that eRF3 is the factor which is also involved as instrument for the Ribosome-P body Transport process. Since there is no proof for this as yet, the instrument link from eRF3 to Ribosome-P body Transport is colored red, denoting uncertainty. (B) The OPL text of the OPD in (A). Note that the word requires in the OPL sentence Ribosome-P body Transport requires eRF3. denotes the same uncertainty regarding the role of eRF3 as instrument to the Ribosome-P body Transport process, analogous to the instrument link from eRF3 to Ribosome-P body Transport in (A).</p>
<span>doi:10.1371/journal.pone.0000872.g004</span></div><a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>eRF3 is an instrument for translation termination (see <a href="#pone.0000872-Alkalaeva1">[61]</a> and references therein), which has also been shown to be involved in mRNA decay <a href="#pone.0000872-Amrani1">[62]</a>–<a href="#pone.0000872-Kofuji1">[64]</a>. These observations provoked us to propose that eRF3 helps coupling between translation termination and P body assembly by serving as an instrument for the Ribosome-P body Transport process. Our model predicts that eRF3 is transported together with the mRNA to P bodies. Indeed, eRF3 is associated with mRNP <a href="#pone.0000872-Hosoda1">[63]</a>, <a href="#pone.0000872-Kofuji1">[64]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>This prediction prompted us to examine whether eRF3 is a component of P bodies. <a href="#pone-0000872-g005">Figure 5</a> demonstrates that eRF3 can indeed be found in cytoplasmic foci of starved cells, together with two known P body constituents. It should be emphasized that association of eRF3 with P bodies was observed after long-term starvation in a stationary phase. We surmise that localization of eRF3 with P bodies is a special case that is subject to specific regulation. This finding is a direct consequence of the conceptual modeling process we engaged in and a conjecture triggered by the OPM model of the mRNA lifecycle that was confirmed experimentally.</p>
<div class="figure" id="pone-0000872-g005"><div class="img"><a name="pone-0000872-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Colocalization of eRF3-GFP with P bodies markers Dcp2p and Rpb4p.</span></strong></p><a id="article1.body1.sec3.fig3.caption1.p1" name="article1.body1.sec3.fig3.caption1.p1"></a><p>Some P bodies are indicated by arrows (different kinds of arrows point at different P bodies). P bodies were examined after 7 days starvation in stationary phase as described in (Lotan et al., 2005). Merging of the green (eRF3-GFP) and the red (Dcp2p-RFP or Rpb4p-RFP, as indicated) channels was done using PhotoShop software. Note that if the green and red foci colocalize the resulting foci are yellow.</p>
<span>doi:10.1371/journal.pone.0000872.g005</span></div><a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Continuing our OPM-based conceptual modeling process, Post-ribosomal Processing from <a href="#pone-0000872-g003">Figure 3</a>, which is the final process of the mRNA Lifecycle, is in-zoomed in <a href="#pone-0000872-g006">Figure 6</a>, showing the subprocesses that the mRNA undergoes within the P body. As indicated in the introduction, the fate of the mRNA in the P body is complex, as it can be degraded, stored, or unloaded. Our model therefore includes a regulated decision-making process, called Checkpoint, which controls execution of one of the following options: (1) degrade (which includes decapping as a first subprocess), (2) store (sequestrate), and (3) unload (a condition for the P body-Ribosome Transport process, which releases mRNA back to the Ribosome). Accordingly, the Object-Process Diagram in <a href="#pone-0000872-g006">Figure 6</a> specifies that Decision can be in one of the three states: degrade, which activates the Degradation process, store, which sequestrates the mRNA, or unload, which activates the P body-Ribosome Transport, the process that initiates translation. The red color of Checkpoint indicates that it is a hypothetical process. We propose that the object D-factor, as yet a hypothetical factor, is the instrument that governs the Checkpoint process. This process generates the logical object Decision, whose state determines which one of the above three options will be executed. D-Factor, also colored red, as its identity remains to be determined, could be any one of the known P body constituents. Other options include kinase, phosphatase, ubiquitin E3 ligase, acetylase, protease, RNase, chaperon, etc. It is likely, although not necessary, that it resides in the P body.</p>
<div class="figure" id="pone-0000872-g006"><div class="img"><a name="pone-0000872-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Zooming into Post-ribosomal Processing.</span></strong></p><a id="article1.body1.sec3.fig4.caption1.p1" name="article1.body1.sec3.fig4.caption1.p1"></a><p>(A) Zooming into Post-ribosomal Processing exposes its subprocesses Checkpoint, Storing, and Degradation, as well as the objects D-factor and Decision. As before, red indicates uncertainty or hypothesis: We propose that D-factor is the instrument for the process we call Checkpoint, which in turn, determines whether to store, degrade, or unload the mRNA for reuse. Green links denote an uncertain conjecture that was confirmed in this work by our experiments. Here, the structural link from P Body to eRF3 and the tag contains along it are green, denoting that we demonstrated experimentally our model-based conjecture that P-body contains eRF3. (B) The OPL text of the OPD in (A). Note the red color of the words in the sentences Checkpoint requires D-factor. and in Checkpoint yields Decision. The red denotes that we are not certain whether Checkpoint and D-factor exist, and if so whether Checkpoint requires D-factor. On the other hand, the green color of the word contains in the sentence P-body contains eRF3 indicates our success at experimentally proving our model-based conjecture that P-body contains eRF3.</p>
<span>doi:10.1371/journal.pone.0000872.g006</span></div>
<h4>Generalizing the opm-based modeling process</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Having presented the modeling process for specific portions of the mRNA lifecycle in some detail, we now outline general guidelines for OPM-based conceptual modeling of a biological system. At the molecular level, all biological systems obey relatively few common rules. Molecules can interact with other molecules, change their molecular environment if they have enzymatic activity, or affect localization within the cell. All these activities are included in our mRNA lifecycle model, which can therefore be potentially used as a case in point for modeling other subsystems of increasing complexity, and ultimately, the entire cell.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Using the OPCAT modeling environment (downloadable from <a href="http://www.opcat.com/downloads/restricted">www.opcat.com/downloads/restricted</a>), the first step in an OPM-based modeling process is to determine and phrase the name of the main process being modeled. For example, for our case study, the main process was mRNA Lifecycle. For an entire organism, this top-level process would be Living. In an OPM-based model, the major process, such as mRNA Lifecycle, is depicted as an ellipse at the center of the System Diagram, the top level, most abstract OPD. Around this process we arrange the objects that take part in the major process, both the systemic and the environmental ones. For Living, one might put the Organism or Eukaryotic Cell as the main systemic object and substances that are needed to support life, like Oxygen, Water and Food, as the environmental objects. The Organism would be linked to Living with an effect link, denoting that Living affects Organism, while substances would be consumed and generated by the Living process. As the model is constructed graphically, sentences start showing up in the OPL text window, which the modelers in the modeling teams should read frequently to make sure that OPCAT “understands” the modelers' intentions, and correct the model in case of misinterpretations. Having obtained this top-level view of the system's structure and behavior, the Living process can be in-zoomed in a new diagram, to reveal a small number of major subprocesses, such as Breathing, Digesting, Moving, and Multiplying. In parallel, one or more objects from the System Diagram can be unfolded to reveal their parts, which can be linked to one or more of the Living subprocesses. This practice of zooming into processes while unfolding related, same level objects, can be repeated to create a hierarchical tree of Object-Process Diagrams that are fully aware of each other, since they were all created from the same System Diagram root, and each one of them focuses on some portion of combined structure and behavior of the same system at various levels of granularity. OPCAT, the OPM-based conceptual modeling environment, ensures that all the OPDs are consistent with each other in spite of the fact that they are at different detail levels. This hierarchical model structure balances the need for clarity, i.e., the need to provide comprehensible diagrams, on one hand and completeness, i.e., the need to provide as many details as possible about the objects in the system and the processes that transform them, on the other hand. Once an initial model has been designed, it can be animated to visualize the timings of process occurrences and associated object transformations.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>In this work, we have applied Object-Process Methodology to create a conceptual model of the mRNA lifecycle. This relatively simple model demonstrates the usefulness of conceptual modeling not just for understanding and communicating the structure and behavior of cell-level systems, but also for provoking conjectures and triggering ideas for experiments that can confirm or refute such conjectures. The findings update the model, which keeps evolving by repeating this cycle. Our model includes basic, well-known aspects of the mRNA lifecycle as well as recently discovered features. We modeled deliberately both established and less established knowledge in order to demonstrate that in both cases the model generates useful predictions. The relatively new feature of the mRNA lifecycle that we focused on here is the capacity of mRNA-protein (RNP) molecules to bundle together and form large cytoplasmic complexes, termed P bodies. We hypothesized that the purported regulation of P body biology is governed by D-Factor (which may be composed of several distinct components). This hypothetical D-Factor controls the fate of mRNA by “deciding” whether each mRNA is degraded, transported to the ribosome for reuse, or sequestered in the P body.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Recently, we proposed that P bodies are heterogeneous complexes <a href="#pone.0000872-Lotan1">[58]</a> and that specific P bodies interact with a particular class of mRNAs, which encode proteins sharing a common biological function (e.g., the protein biosynthetic machinery). We proposed that P bodies specialize to coordinate the regulation (storage, translation, or degradation) of classes of mRNAs. We also speculated that, in addition to their being hubs, P bodies may contribute also to the transport of mRNAs to specific locations within the cytoplasm. Such transport might be vital for large cells, e.g., nerve or dendrite cells. A recent description of mRNA localization in dendritic cells can be found in <a href="#pone.0000872-Kremerskothen1">[65]</a>. In particular, we demonstrated here that modeling recent data helps pinpoint uncertainties, raise new questions, and experiment to get answers to these questions. This is how we obtained the new result we report in this work, which establishes the localization of eRF3 in P bodies.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>OPM has rich and flexible modeling capabilities with high expressive power. For example, as <a href="#pone-0000872-g007">Figure 7</a> shows, variants, such as splicing variants, can be modeled explicitly and clearly.</p>
<div class="figure" id="pone-0000872-g007"><div class="img"><a name="pone-0000872-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000872" data-uri="info:doi/10.1371/journal.pone.0000872.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000872.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000872.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000872.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000872.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>An example of modeling variants with OPM: A primary transcript is spliced to yield two splicing variants, A and B, each containing a different exon combination, as expressed in both the OPD at the top and the OPL text at the bottom.</span></strong></p><a id="article1.body1.sec4.fig1.caption1.p1" name="article1.body1.sec4.fig1.caption1.p1"></a><p>Modeling other types of variants, e.g., protein phosphorylation, protein ubiquitination, RNA editing, can be done in a similar manner.</p>
<span>doi:10.1371/journal.pone.0000872.g007</span></div><a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>We emphasize that the OPM conceptual model of the mRNA lifecycle presented in this work is by no means complete. It illustrates superficially certain parts of this extremely complex cycle (which can be referred to as a pathway, if nucleotide recycling is ignored) as we conceive it today. Still, our model was detailed enough to provoke research questions, and a solution for one of those questions was found experimentally. The mRNA lifecycle is studied extensively, so any sections of this model can be in-zoomed further to provide ever more detailed descriptions.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>In general, systems biology can benefit from using OPM as a generic framework for knowledge capture and representation, particularly since it enables balanced and unified representation of the system's structure and behavior using both objects and processes in the same diagram. Using the refinement-abstraction mechanisms that are built into OPM, the system under study can be clearly understood and communicated at various detail (granularity) levels. Moreover, as demonstrated here, OPM-based modeling provokes consideration of links missing from the process chain and stimulates ideas for experiments to prove or disprove new theories. Without the intellectual activity underlying conceptual modeling, such gaps or inconsistencies in the model can easily go unnoticed, evading the researchers' attention. Indeed, while engaged in modeling, we encountered portions of the system which we were uncertain how to model. The knowledge gaps become more apparent as we tried to further zoom (drill down) into specific subprocesses of the mRNA lifecycle. Graphically, this was manifested by increasing red color in the diagrams. Based on our positive experience, we propose that the friendly, yet formal, OPM modeling framework is a tool for modeling biological systems, whose adoption would benefit the emerging domain of systems biology.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Due to the ability to forge a holistic conceptual model of a biological system, we maintain that this research should be valuable to biologists and computer scientists who work on developing a Systems Biology understanding of the living organisms. The OPM-based conceptual modeling framework provides a clear, unambiguous way to describe our current knowledge of the state of the cell. It allows incremental resolution of different parts of the cellular machinery, helps pinpoint areas where our current understanding of the model is lacking, and finally, what type of experiments we may conduct to improve our understanding.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Following intensive future research and development, we envision an evolving model that would be developed, maintained, and updated constantly by the research community at large. When incorporating new data into the existing model, it would be possible to determine if it is consistent with previous knowledge. Unproven conjectures could also be incorporated into this evolving model, tagged as uncertain. Their inclusion in the model would help point towards evidence required to support these hypotheses. Such a modeling framework would help researchers tackling the huge challenge of understanding holistically the intricacies of the living cell.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000872"></div><a name="pone.0000872.s001" id="pone.0000872.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000872.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">A Quick Guide to the Syntax and Semantics of the Object-Process Methodology (OPM) Language.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000872.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.06 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Roy Parker for RFP DCP2-RFP plasmid.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DD MC. Performed the experiments: DD MC. Analyzed the data: DD MC. Contributed reagents/materials/analysis tools: DD MC. Wrote the paper: DD MC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000872-Kitano1" id="pone.0000872-Kitano1"></a>Kitano H (2002) Systems Biology: A Brief Overview. Science,  295(5560): 1662–1664.  <ul class="find" data-citedArticleID="990212" data-doi="10.1126/science.1069492"><li><a href="http://dx.doi.org/10.1126/science.1069492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems+Biology%3A+A+Brief+Overview." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems+Biology%3A+A+Brief+Overview.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000872-Kritikou1" id="pone.0000872-Kritikou1"></a>Kritikou E, Pulverer B, Heinrichs AEditorial to “Systems Biology: A User's Guide”. Joint online resource by Nature Reviews Molecular Cell Biology and Nature Cell Biology, 2006.   <a href="http://www.nature.com/focus/systemsbiologyuserguide/editorial/sysbiol-s3.html">http://www.nature.com/focus/systemsbiolo​gyuserguide/editorial/sysbiol-s3.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000872-Davidson1" id="pone.0000872-Davidson1"></a>Davidson EH, et al.  (2002) A Genomic Regulatory Network for Development. Science,  295(5560): 1669–1678.  <ul class="find" data-citedArticleID="990172" data-doi="10.1126/science.1069883"><li><a href="http://dx.doi.org/10.1126/science.1069883" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Genomic+Regulatory+Network+for+Development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Genomic+Regulatory+Network+for+Development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000872-Lovrics1" id="pone.0000872-Lovrics1"></a>Lovrics A, et al.  (2006) Time scale and dimension analysis of a budding yeast cell cycle model. BMC Bioinformatics,  7(494):   <ul class="find" data-citedArticleID="990230"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Time+scale+and+dimension+analysis+of+a+budding+yeast+cell+cycle+model.&amp;auth=&amp;atitle=Time+scale+and+dimension+analysis+of+a+budding+yeast+cell+cycle+model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Time+scale+and+dimension+analysis+of+a+budding+yeast+cell+cycle+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Time+scale+and+dimension+analysis+of+a+budding+yeast+cell+cycle+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000872-Tomita1" id="pone.0000872-Tomita1"></a>Tomita M (2001) Whole-cell simulation: a grand challenge of the 21st century. Trends Biotechnol,  19(6): 205–10.  <ul class="find" data-citedArticleID="990254" data-doi="10.1016/s0167-7799(01)01636-5"><li><a href="http://dx.doi.org/10.1016/s0167-7799(01)01636-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-cell+simulation%3A+a+grand+challenge+of+the+21st+century." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-cell+simulation%3A+a+grand+challenge+of+the+21st+century.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000872-Takahashi1" id="pone.0000872-Takahashi1"></a>Takahashi K, et al.  (2003) E-Cell 2: multi-platform E-Cell simulation system. Bioinformatics,  19(13): 1727–9.  <ul class="find" data-citedArticleID="990250" data-doi="10.1093/bioinformatics/btg221"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=E-Cell+2%3A+multi-platform+E-Cell+simulation+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22E-Cell+2%3A+multi-platform+E-Cell+simulation+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000872-httpwww.vcell.org.1" id="pone.0000872-httpwww.vcell.org.1"></a>  <a href="http://www.vcell.org">http://www.vcell.org</a>. [cited; Available from: <a href="http://www.vcell.org">http://www.vcell.org</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0000872-Slepchenko1" id="pone.0000872-Slepchenko1"></a>Slepchenko BM, et al.  (2003) Quantitative cell biology with the Virtual Cell. Trends Cell Biol,  13(11): 570–6.  <ul class="find" data-citedArticleID="990248" data-doi="10.1016/j.tcb.2003.09.002"><li><a href="http://dx.doi.org/10.1016/j.tcb.2003.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+cell+biology+with+the+Virtual+Cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+cell+biology+with+the+Virtual+Cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000872-Harel1" id="pone.0000872-Harel1"></a>Harel D (2003) A Grand Challenge for Computing: Full Reactive Modeling of a Multi-Cellular Animal. LNCS,  2623: 2–2.  <ul class="find" data-citedArticleID="990188" data-doi="10.1007/3-540-36580-x_2"><li><a href="http://dx.doi.org/10.1007/3-540-36580-x_2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Grand+Challenge+for+Computing%3A+Full+Reactive+Modeling+of+a+Multi-Cellular+Animal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Grand+Challenge+for+Computing%3A+Full+Reactive+Modeling+of+a+Multi-Cellular+Animal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000872-Harel2" id="pone.0000872-Harel2"></a>Harel D (2005) A Turing-like test for biological modeling. Nat Biotechnol,  23(4): 495–6.  <ul class="find" data-citedArticleID="990190" data-doi="10.1038/nbt0405-495"><li><a href="http://dx.doi.org/10.1038/nbt0405-495" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Turing-like+test+for+biological+modeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Turing-like+test+for+biological+modeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000872-Kam1" id="pone.0000872-Kam1"></a>Kam N, et al.  (2004) Formal modeling of C. elegans development: a scenario-based approach. in Modelling in Molecular Biology.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0000872-Keet1" id="pone.0000872-Keet1"></a>Keet C (2006) Enhancing biological information systems with granularity. in KnowledgeWeb PhD Symposium (KWEPSY06). Budva, Montenegro.  <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0000872-Webb1" id="pone.0000872-Webb1"></a>Webb K, White T (2004) Cell Modeling using Agent-based Formalisms,. in Proc. 3rd International Joint Conference on Autonomous Agents and Multiagent Systems (AAMAS 2004), 19-23 August 2004, New York, NY, USA. (AAMAS 2004).   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000872-BioUML1" id="pone.0000872-BioUML1"></a> (2003)   BioUML, BioUML: Framework for Systems Biology <a href="http://www.it-careernet.com/itc/BioUML/bioUML.htm">http://www.it-careernet.com/itc/BioUML/b​ioUML.htm</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000872-Booch1" id="pone.0000872-Booch1"></a>Booch G, Rumbaugh J, Jacobson I (1999) The UML User Guide. Addison-Wesley.   <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0000872-UML21" id="pone.0000872-UML21"></a> (2005)   UML-2, Unified Modeling Language (UML), version 2.1.1. <a href="http://www.omg.org/technology/documents/formal/uml.htm">http://www.omg.org/technology/documents/​formal/uml.htm</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000872-1" id="pone.0000872-1"></a> (2005) Object-Management-Group, Introduction to OMG's Unified Modeling Language™ (UML®).   Available: <a href="http://www.omg.org/gettingstarted/what_is_uml.htm">http://www.omg.org/gettingstarted/what_i​s_uml.htm</a>. Accessed Jan. 11 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000872-Siau1" id="pone.0000872-Siau1"></a>Siau K.a.CQ (2001) Unified Modeling Language: A Complexity Analysis. ournal of Database Management,  12(1): 26–34.  <ul class="find" data-citedArticleID="990246" data-doi="10.4018/jdm.2001010103"><li><a href="http://dx.doi.org/10.4018/jdm.2001010103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unified+Modeling+Language%3A+A+Complexity+Analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unified+Modeling+Language%3A+A+Complexity+Analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000872-Peleg1" id="pone.0000872-Peleg1"></a>Peleg MDD (2000) The Model Multiplicity Problem: Experimenting with Real-Time Specification Methods. IEEE Transaction on Software Engineering,  26(8): 742–759.  <ul class="find" data-citedArticleID="990242" data-doi="10.1109/32.879812"><li><a href="http://dx.doi.org/10.1109/32.879812" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Model+Multiplicity+Problem%3A+Experimenting+with+Real-Time+Specification+Methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Model+Multiplicity+Problem%3A+Experimenting+with+Real-Time+Specification+Methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000872-Lange1" id="pone.0000872-Lange1"></a>Lange C, et al.  (2003) An Empirical Investigation in Quantifying Inconsistency and. Incompleteness of UML Designs. in Consistency Problems in UML-based Software Development II Workshop within the Sixth International Conference on the Unified Modelling Language. San Francisco, CA, USA.  <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0000872-Hucka1" id="pone.0000872-Hucka1"></a>Hucka M, et al.  (2003) The systems biology markup language (SBML): a medium for representation and exchange of biochemical network models. Bioinformatics,  19(4): 524–31.  <ul class="find" data-citedArticleID="990196" data-doi="10.1093/bioinformatics/btg015"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+systems+biology+markup+language+%28SBML%29%3A+a+medium+for+representation+and+exchange+of+biochemical+network+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+systems+biology+markup+language+%28SBML%29%3A+a+medium+for+representation+and+exchange+of+biochemical+network+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000872-Hedley1" id="pone.0000872-Hedley1"></a>Hedley W, et al.  (2001) A short introduction to CellML. Philosophical Transactions-Mathematical, Physical and Engineering Sciences,  359: 1073–1089.  <ul class="find" data-citedArticleID="990192" data-doi="10.1098/rsta.2001.0817"><li><a href="http://dx.doi.org/10.1098/rsta.2001.0817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+short+introduction+to+CellML." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+short+introduction+to+CellML.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000872-2" id="pone.0000872-2"></a>SysML.   SysML-Open Source Specification Project <a href="http://www.sysml.org/">http://www.sysml.org/</a>. 2007 [cited; Available from: <a href="http://www.sysml.org/">http://www.sysml.org/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000872-Kohn1" id="pone.0000872-Kohn1"></a>Kohn KW (2001) Molecular interaction maps as information organizers and simulation guides. Chaos,  11: 84–97.  <ul class="find" data-citedArticleID="990218" data-doi="10.1063/1.1338126"><li><a href="http://dx.doi.org/10.1063/1.1338126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+interaction+maps+as+information+organizers+and+simulation+guides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+interaction+maps+as+information+organizers+and+simulation+guides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000872-Kitano2" id="pone.0000872-Kitano2"></a>Kitano H, et al.  (2005) Using process diagrams for the graphical representation of biological networks. Nature Biotechnology,  23: 961–966.  <ul class="find" data-citedArticleID="990214" data-doi="10.1038/nbt1111"><li><a href="http://dx.doi.org/10.1038/nbt1111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+process+diagrams+for+the+graphical+representation+of+biological+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+process+diagrams+for+the+graphical+representation+of+biological+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000872-Funahashi1" id="pone.0000872-Funahashi1"></a>Funahashi A (2007) CellDesigner.   Systems Biology Institute <a href="http://www.systems-biology.org/002/">http://www.systems-biology.org/002/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000872-Blinov1" id="pone.0000872-Blinov1"></a>Blinov ML, et al.  (2006) Depicting signaling cascades. Nature Biotechnology,  24: 137–138.  <ul class="find" data-citedArticleID="990162" data-doi="10.1038/nbt0206-137"><li><a href="http://dx.doi.org/10.1038/nbt0206-137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Depicting+signaling+cascades." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Depicting+signaling+cascades.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000872-Tyson1" id="pone.0000872-Tyson1"></a>Tyson JJ, Csikasz-Nagy A, Novak B (2002) The dynamics of cell cycle regulation. BioEssays Special Issue: Modelling complex biological systems,  24(12): 1095–1109.  <ul class="find" data-citedArticleID="990256" data-doi="10.1002/bies.10191"><li><a href="http://dx.doi.org/10.1002/bies.10191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dynamics+of+cell+cycle+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dynamics+of+cell+cycle+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000872-Longabaugh1" id="pone.0000872-Longabaugh1"></a>Longabaugh WJR, Davidson EH, Bolouri HBioTapestry. Davidson Lab, California Institute of Technology, 2007: p.   Available <a href="http://labs.systemsbiology.net/bolouri/software/BioTapestry/Jan">http://labs.systemsbiology.net/bolouri/s​oftware/BioTapestry/Jan</a>. 12, 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0000872-Dori1" id="pone.0000872-Dori1"></a>Dori D (2002) Object-Process Methodology-A Holistic Systems Paradigm. Heidelberg, New York: Springer Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0000872-3" id="pone.0000872-3"></a>Gene-Ontology, 2006: p.   Available: <a href="http://www.geneontology.org/index.shtml">http://www.geneontology.org/index.shtml</a>. Accessed: Feb. 7, 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0000872-Dori2" id="pone.0000872-Dori2"></a>Dori D, Reinhartz-Berger I, Sturm A (2003) Developing Complex Systems with Object-Process Methodology using OPCAT. Conceptual Modeling-ER 2003. Lecture Notes in Computer Science (2813).  [cited; 570-572].  <ul class="find" data-citedArticleID="990176" data-doi="10.1007/978-3-540-39648-2_46"><li><a href="http://dx.doi.org/10.1007/978-3-540-39648-2_46" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developing+Complex+Systems+with+Object-Process+Methodology+using+OPCAT.+Conceptual+Modeling-ER+2003.+Lecture+Notes+in+Computer+Science+%282813%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developing+Complex+Systems+with+Object-Process+Methodology+using+OPCAT.+Conceptual+Modeling-ER+2003.+Lecture+Notes+in+Computer+Science+%282813%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000872-Mayer1" id="pone.0000872-Mayer1"></a>Mayer RE (2001) Multimedia Learning. C,. NewYork, NY: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0000872-Watson1" id="pone.0000872-Watson1"></a>Watson J, et al.  (2004) Molecular biology of the gene. Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000872-Maciag1" id="pone.0000872-Maciag1"></a>Maciag K, et al.  (2006) Systems-level analyses identify extensive coupling among gene expression machines. Mol Syst Biol,  2: E1–E14.  <ul class="find" data-citedArticleID="990234" data-doi="10.1038/msb4100045"><li><a href="http://dx.doi.org/10.1038/msb4100045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems-level+analyses+identify+extensive+coupling+among+gene+expression+machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems-level+analyses+identify+extensive+coupling+among+gene+expression+machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000872-Coller1" id="pone.0000872-Coller1"></a>Coller J, Parker R (2004) Eukaryotic mRNA decapping. Annu Rev Biochem,  73: 861–90.  <ul class="find" data-citedArticleID="990168" data-doi="10.1146/annurev.biochem.73.011303.074032"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.73.011303.074032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eukaryotic+mRNA+decapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eukaryotic+mRNA+decapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000872-Wilusz1" id="pone.0000872-Wilusz1"></a>Wilusz CJ, Wilusz J (2004) Bringing the role of mRNA decay in the control of gene expression into focus. Trends Genet,  20(10): 491–7.  <ul class="find" data-citedArticleID="990266" data-doi="10.1016/j.tig.2004.07.011"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.07.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bringing+the+role+of+mRNA+decay+in+the+control+of+gene+expression+into+focus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bringing+the+role+of+mRNA+decay+in+the+control+of+gene+expression+into+focus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000872-Sheth1" id="pone.0000872-Sheth1"></a>Sheth U, Parker R (2003) Decapping and decay of messenger RNA occur in cytoplasmic processing bodies. Science,  300(5620): 805–8.  <ul class="find" data-citedArticleID="990244" data-doi="10.1126/science.1082320"><li><a href="http://dx.doi.org/10.1126/science.1082320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Decapping+and+decay+of+messenger+RNA+occur+in+cytoplasmic+processing+bodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Decapping+and+decay+of+messenger+RNA+occur+in+cytoplasmic+processing+bodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000872-Bashkirov1" id="pone.0000872-Bashkirov1"></a>Bashkirov VI, et al.  (1997) A mouse cytoplasmic exoribonuclease (mXRN1p) with preference for G4 tetraplex substrates. J Cell Biol,  136(4): 761–73.  <ul class="find" data-citedArticleID="990158" data-doi="10.1083/jcb.136.4.761"><li><a href="http://dx.doi.org/10.1083/jcb.136.4.761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mouse+cytoplasmic+exoribonuclease+%28mXRN1p%29+with+preference+for+G4+tetraplex+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mouse+cytoplasmic+exoribonuclease+%28mXRN1p%29+with+preference+for+G4+tetraplex+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000872-Ingelfinger1" id="pone.0000872-Ingelfinger1"></a>Ingelfinger D, et al.  (2002) The human LSm1-7 proteins colocalize with the mRNA-degrading enzymes Dcp1/2 and Xrnl in distinct cytoplasmic foci. Rna,  8(12): 1489–501.  <ul class="find" data-citedArticleID="990198"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+human+LSm1-7+proteins+colocalize+with+the+mRNA-degrading+enzymes+Dcp1%2F2+and+Xrnl+in+distinct+cytoplasmic+foci.&amp;auth=&amp;atitle=The+human+LSm1-7+proteins+colocalize+with+the+mRNA-degrading+enzymes+Dcp1%2F2+and+Xrnl+in+distinct+cytoplasmic+foci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+LSm1-7+proteins+colocalize+with+the+mRNA-degrading+enzymes+Dcp1%2F2+and+Xrnl+in+distinct+cytoplasmic+foci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+LSm1-7+proteins+colocalize+with+the+mRNA-degrading+enzymes+Dcp1%2F2+and+Xrnl+in+distinct+cytoplasmic+foci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000872-LykkeAndersen1" id="pone.0000872-LykkeAndersen1"></a>Lykke-Andersen J (2002) Identification of a human decapping complex associated with hUpf proteins in nonsense-mediated decay. Mol Cell Biol,  22(23): 8114–21.  <ul class="find" data-citedArticleID="990232" data-doi="10.1128/mcb.22.23.8114-8121.2002"><li><a href="http://dx.doi.org/10.1128/mcb.22.23.8114-8121.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+human+decapping+complex+associated+with+hUpf+proteins+in+nonsense-mediated+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+human+decapping+complex+associated+with+hUpf+proteins+in+nonsense-mediated+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000872-vanDijk1" id="pone.0000872-vanDijk1"></a>van Dijk E, et al.  (2002) Human Dcp2: a catalytically active mRNA decapping enzyme located in specific cytoplasmic structures. Embo J,  21(24): 6915–24.  <ul class="find" data-citedArticleID="990270" data-doi="10.1093/emboj/cdf678"><li><a href="http://dx.doi.org/10.1093/emboj/cdf678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Dcp2%3A+a+catalytically+active+mRNA+decapping+enzyme+located+in+specific+cytoplasmic+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Dcp2%3A+a+catalytically+active+mRNA+decapping+enzyme+located+in+specific+cytoplasmic+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000872-Teixeira1" id="pone.0000872-Teixeira1"></a>Teixeira D, et al.  (2005) Processing bodies require RNA for assembly and contain nontranslating mRNAs. Rna.   <ul class="find" data-citedArticleID="990252" data-doi="10.1261/rna.7258505"><li><a href="http://dx.doi.org/10.1261/rna.7258505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+bodies+require+RNA+for+assembly+and+contain+nontranslating+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+bodies+require+RNA+for+assembly+and+contain+nontranslating+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000872-Eystathioy1" id="pone.0000872-Eystathioy1"></a>Eystathioy T, et al.  (2003) The GW182 protein colocalizes with mRNA degradation associated proteins hDcp1 and hLSm4 in cytoplasmic GW bodies. Rna,  9(10): 1171–3.  <ul class="find" data-citedArticleID="990180" data-doi="10.1261/rna.5810203"><li><a href="http://dx.doi.org/10.1261/rna.5810203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+GW182+protein+colocalizes+with+mRNA+degradation+associated+proteins+hDcp1+and+hLSm4+in+cytoplasmic+GW+bodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+GW182+protein+colocalizes+with+mRNA+degradation+associated+proteins+hDcp1+and+hLSm4+in+cytoplasmic+GW+bodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000872-Wilczynska1" id="pone.0000872-Wilczynska1"></a>Wilczynska A, et al.  (2005) The translational regulator CPEB1 provides a link between dcp1 bodies and stress granules. J Cell Sci,  118(Pt 5): 981–92.  <ul class="find" data-citedArticleID="990264" data-doi="10.1242/jcs.01692"><li><a href="http://dx.doi.org/10.1242/jcs.01692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+translational+regulator+CPEB1+provides+a+link+between+dcp1+bodies+and+stress+granules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+translational+regulator+CPEB1+provides+a+link+between+dcp1+bodies+and+stress+granules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000872-Fillman1" id="pone.0000872-Fillman1"></a>Fillman C, Lykke-Andersen J (2005) RNA decapping inside and outside of processing bodies. Curr Opin Cell Biol,  17(3): 326–31.  <ul class="find" data-citedArticleID="990184" data-doi="10.1016/j.ceb.2005.04.002"><li><a href="http://dx.doi.org/10.1016/j.ceb.2005.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+decapping+inside+and+outside+of+processing+bodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+decapping+inside+and+outside+of+processing+bodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000872-Eulalio1" id="pone.0000872-Eulalio1"></a>Eulalio A, Behm-Ansmant I, Izaurralde E (2007) P bodies: at the crossroads of post-transcriptional pathways. Nat Rev Mol Cell Biol,  8(1): 9–22.  <ul class="find" data-citedArticleID="990178" data-doi="10.1038/nrm2080"><li><a href="http://dx.doi.org/10.1038/nrm2080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=P+bodies%3A+at+the+crossroads+of+post-transcriptional+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22P+bodies%3A+at+the+crossroads+of+post-transcriptional+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000872-Anderson1" id="pone.0000872-Anderson1"></a>Anderson P, Kedersha N (2006) RNA granules. J Cell Biol,  172(6): 803–8.  <ul class="find" data-citedArticleID="990152" data-doi="10.1083/jcb.200512082"><li><a href="http://dx.doi.org/10.1083/jcb.200512082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+granules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+granules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000872-Parker1" id="pone.0000872-Parker1"></a>Parker R, Sheth U (2007) P bodies and the control of mRNA translation and degradation.  Mol. Cell  25: 635–646.  <ul class="find" data-citedArticleID="990240" data-doi="10.1016/j.molcel.2007.02.011"><li><a href="http://dx.doi.org/10.1016/j.molcel.2007.02.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=P+bodies+and+the+control+of+mRNA+translation+and+degradation.+" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22P+bodies+and+the+control+of+mRNA+translation+and+degradation.+%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000872-Kedersha1" id="pone.0000872-Kedersha1"></a>Kedersha N, et al.  (2000) Dynamic shuttling of TIA-1 accompanies the recruitment of mRNA to mammalian stress granules. J Cell Biol,  151(6): 1257–68.  <ul class="find" data-citedArticleID="990202" data-doi="10.1083/jcb.151.6.1257"><li><a href="http://dx.doi.org/10.1083/jcb.151.6.1257" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+shuttling+of+TIA-1+accompanies+the+recruitment+of+mRNA+to+mammalian+stress+granules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+shuttling+of+TIA-1+accompanies+the+recruitment+of+mRNA+to+mammalian+stress+granules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000872-Ferraiuolo1" id="pone.0000872-Ferraiuolo1"></a>Ferraiuolo MA, et al.  (2005) A role for the eIF4E-binding protein 4E-T in P-body formation and mRNA decay. J Cell Biol,  170(6): 913–24.  <ul class="find" data-citedArticleID="990182" data-doi="10.1083/jcb.200504039"><li><a href="http://dx.doi.org/10.1083/jcb.200504039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+the+eIF4E-binding+protein+4E-T+in+P-body+formation+and+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+the+eIF4E-binding+protein+4E-T+in+P-body+formation+and+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000872-Kedersha2" id="pone.0000872-Kedersha2"></a>Kedersha N, et al.  (2002) Evidence that ternary complex (eIF2-GTP-tRNA(i)(Met))-deficient preinitiation complexes are core constituents of mammalian stress granules. Mol Biol Cell,  13(1): 195–210.  <ul class="find" data-citedArticleID="990204" data-doi="10.1091/mbc.01-05-0221"><li><a href="http://dx.doi.org/10.1091/mbc.01-05-0221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+ternary+complex+%28eIF2-GTP-tRNA%28i%29%28Met%29%29-deficient+preinitiation+complexes+are+core+constituents+of+mammalian+stress+granules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+ternary+complex+%28eIF2-GTP-tRNA%28i%29%28Met%29%29-deficient+preinitiation+complexes+are+core+constituents+of+mammalian+stress+granules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000872-Kimball1" id="pone.0000872-Kimball1"></a>Kimball SR, et al.  (2003) Mammalian stress granules represent sites of accumulation of stalled translation initiation complexes. Am J Physiol Cell Physiol,  284(2): C273–84.  <ul class="find" data-citedArticleID="990210" data-doi="10.1152/ajpcell.00314.2002"><li><a href="http://dx.doi.org/10.1152/ajpcell.00314.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+stress+granules+represent+sites+of+accumulation+of+stalled+translation+initiation+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+stress+granules+represent+sites+of+accumulation+of+stalled+translation+initiation+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000872-Anderson2" id="pone.0000872-Anderson2"></a>Anderson P, Kedersha N (2002) Stressful initiations. J Cell Sci,  115(Pt 16): 3227–34.  <ul class="find" data-citedArticleID="990154"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stressful+initiations.&amp;auth=&amp;atitle=Stressful+initiations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stressful+initiations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stressful+initiations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000872-Kedersha3" id="pone.0000872-Kedersha3"></a>Kedersha N, et al.  (2005) Stress granules and processing bodies are dynamically linked sites of mRNP remodeling. J Cell Biol,  169(6): 871–84.  <ul class="find" data-citedArticleID="990206" data-doi="10.1083/jcb.200502088"><li><a href="http://dx.doi.org/10.1083/jcb.200502088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stress+granules+and+processing+bodies+are+dynamically+linked+sites+of+mRNP+remodeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stress+granules+and+processing+bodies+are+dynamically+linked+sites+of+mRNP+remodeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000872-Brengues1" id="pone.0000872-Brengues1"></a>Brengues M, Teixeira D, Parker R (2005) Movement of eukaryotic mRNAs between polysomes and cytoplasmic processing bodies. Science,  310(5747): 486–9.  <ul class="find" data-citedArticleID="990166" data-doi="10.1126/science.1115791"><li><a href="http://dx.doi.org/10.1126/science.1115791" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Movement+of+eukaryotic+mRNAs+between+polysomes+and+cytoplasmic+processing+bodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Movement+of+eukaryotic+mRNAs+between+polysomes+and+cytoplasmic+processing+bodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000872-Aragon1" id="pone.0000872-Aragon1"></a>Aragon AD, Quiñones GA, Thomas EV, Roy S, Werner-Washburne M (2006) Release of extraction-resistant mRNA in stationary-phase S. cerevisiae produces a massive increase in transcript abundance in response to stress. Genome Biology 7.   R9 doi:10.1186/gb-2006-1187-1182-r1189.  <ul class="find" data-citedArticleID="990156"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Release+of+extraction-resistant+mRNA+in+stationary-phase+S.+cerevisiae+produces+a+massive+increase+in+transcript+abundance+in+response+to+stress.&amp;auth=&amp;atitle=Release+of+extraction-resistant+mRNA+in+stationary-phase+S.+cerevisiae+produces+a+massive+increase+in+transcript+abundance+in+response+to+stress." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Release+of+extraction-resistant+mRNA+in+stationary-phase+S.+cerevisiae+produces+a+massive+increase+in+transcript+abundance+in+response+to+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Release+of+extraction-resistant+mRNA+in+stationary-phase+S.+cerevisiae+produces+a+massive+increase+in+transcript+abundance+in+response+to+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000872-Lotan1" id="pone.0000872-Lotan1"></a>Lotan R, et al.  (2005) The RNA polymerase II subunit Rpb4p mediates decay of a specific class of mRNAs. Genes Dev,  19(24): 3004–16.  <ul class="find" data-citedArticleID="990228" data-doi="10.1101/gad.353205"><li><a href="http://dx.doi.org/10.1101/gad.353205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RNA+polymerase+II+subunit+Rpb4p+mediates+decay+of+a+specific+class+of+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RNA+polymerase+II+subunit+Rpb4p+mediates+decay+of+a+specific+class+of+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000872-Daneholt1" id="pone.0000872-Daneholt1"></a>Daneholt B (1997) A Look at Messenger RNP Moving through the Nuclear Pore. Cell,  88: 585–588.  <ul class="find" data-citedArticleID="990170" data-doi="10.1016/s0092-8674(00)81900-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81900-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Look+at+Messenger+RNP+Moving+through+the+Nuclear+Pore." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Look+at+Messenger+RNP+Moving+through+the+Nuclear+Pore.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000872-Marx1" id="pone.0000872-Marx1"></a>Marx J (2005) Molecular biology. P-bodies mark the spot for controlling protein production. Science,  310(5749): 764–5.  <ul class="find" data-citedArticleID="990236" data-doi="10.1126/science.310.5749.764"><li><a href="http://dx.doi.org/10.1126/science.310.5749.764" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+biology.+P-bodies+mark+the+spot+for+controlling+protein+production." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+biology.+P-bodies+mark+the+spot+for+controlling+protein+production.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000872-Alkalaeva1" id="pone.0000872-Alkalaeva1"></a>Alkalaeva EZ, et al.  (2006) In vitro reconstitution of eukaryotic translation reveals cooperativity between release factors eRF1 and eRF3. Cell,  125(6): 1125–36.  <ul class="find" data-citedArticleID="990148" data-doi="10.1016/j.cell.2006.04.035"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.04.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+reconstitution+of+eukaryotic+translation+reveals+cooperativity+between+release+factors+eRF1+and+eRF3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+reconstitution+of+eukaryotic+translation+reveals+cooperativity+between+release+factors+eRF1+and+eRF3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000872-Amrani1" id="pone.0000872-Amrani1"></a>Amrani N, et al.  (2004) A faux 3′-UTR promotes aberrant termination and triggers nonsense-mediated mRNA decay. Nature,  432(7013): 112–8.  <ul class="find" data-citedArticleID="990150" data-doi="10.1038/nature03060"><li><a href="http://dx.doi.org/10.1038/nature03060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+faux+3%26%23x2032%3B-UTR+promotes+aberrant+termination+and+triggers+nonsense-mediated+mRNA+decay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+faux+3%26%23x2032%3B-UTR+promotes+aberrant+termination+and+triggers+nonsense-mediated+mRNA+decay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000872-Hosoda1" id="pone.0000872-Hosoda1"></a>Hosoda N, et al.  (2003) Translation termination factor eRF3 mediates mRNA decay through the regulation of deadenylation. J Biol Chem,  278(40): 38287–91.  <ul class="find" data-citedArticleID="990194" data-doi="10.1074/jbc.c300300200"><li><a href="http://dx.doi.org/10.1074/jbc.c300300200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translation+termination+factor+eRF3+mediates+mRNA+decay+through+the+regulation+of+deadenylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translation+termination+factor+eRF3+mediates+mRNA+decay+through+the+regulation+of+deadenylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000872-Kofuji1" id="pone.0000872-Kofuji1"></a>Kofuji S, et al.  (2006) The decapping enzyme Dcp1 participates in translation termination through its interaction with the release factor eRF3 in budding yeast. Biochem Biophys Res Commun,  344(2): 547–53.  <ul class="find" data-citedArticleID="990216" data-doi="10.1016/j.bbrc.2006.03.174"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.03.174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+decapping+enzyme+Dcp1+participates+in+translation+termination+through+its+interaction+with+the+release+factor+eRF3+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+decapping+enzyme+Dcp1+participates+in+translation+termination+through+its+interaction+with+the+release+factor+eRF3+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000872-Kremerskothen1" id="pone.0000872-Kremerskothen1"></a>Kremerskothen J, et al.  (2006) Postsynaptic recruitment of Dendrin depends on both dendritic mRNA transport and synaptic anchoring. J Neurochem,  96(6): 1659–66.  <ul class="find" data-citedArticleID="990220" data-doi="10.1111/j.1471-4159.2006.03679.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.2006.03679.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Postsynaptic+recruitment+of+Dendrin+depends+on+both+dendritic+mRNA+transport+and+synaptic+anchoring." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Postsynaptic+recruitment+of+Dendrin+depends+on+both+dendritic+mRNA+transport+and+synaptic+anchoring.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.XML" value="121371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.PDF" value="867954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g001.PNG_L" value="866523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g001.PNG_M" value="63096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g001.PNG_S" value="6442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g001.TIF" value="1927536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g001.PNG_I" value="22211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g002.PNG_L" value="153435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g002.PNG_M" value="217433"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g002.PNG_S" value="14094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g002.TIF" value="216278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g002.PNG_I" value="73681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g003.PNG_L" value="48490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g003.PNG_M" value="218638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g003.PNG_S" value="13382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g003.TIF" value="168968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g003.PNG_I" value="75380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g004.PNG_L" value="293494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g004.PNG_M" value="274925"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g004.PNG_S" value="17500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g004.TIF" value="410802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g004.PNG_I" value="134023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g005.PNG_L" value="198441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g005.PNG_M" value="152243"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g005.PNG_S" value="12718"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g005.TIF" value="282910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g005.PNG_I" value="52492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g006.PNG_L" value="321576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g006.PNG_M" value="246387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g006.PNG_S" value="17956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g006.TIF" value="448882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g006.PNG_I" value="147420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g007.PNG_L" value="170957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g007.PNG_M" value="122261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g007.PNG_S" value="10360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g007.TIF" value="284100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.g007.PNG_I" value="39744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000872.s001.DOC" value="1058816"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000872&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000872" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000872&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000872&volume=&issue=&title=Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle&author_name=Dov%20Dori%2C%20Mordechai%20Choder&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000872" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872&amp;t=Conceptual%20Modeling%20in%20Systems%20Biology%20Fosters%20Empirical%20Findings%3A%20The%20mRNA%20Lifecycle" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872&title=Conceptual%20Modeling%20in%20Systems%20Biology%20Fosters%20Empirical%20Findings%3A%20The%20mRNA%20Lifecycle&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872&amp;title=Conceptual%20Modeling%20in%20Systems%20Biology%20Fosters%20Empirical%20Findings%3A%20The%20mRNA%20Lifecycle" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000872&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Conceptual Modeling in Systems Biology Fosters Empirical Findings: The mRNA Lifecycle';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000872';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Conceptual%20Modeling%20in%20Systems%20Biology%20Fosters%20Empirical%20Findings%3A%20The%20mRNA%20Lifecycle http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000872" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000872" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="24582"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Computer+software%22" title="Search for articles in the subject area:'Computer software'"><div class="flagText">Computer software</div></a>
              <div data-categoryid="2287" data-articleid="24582"
                   data-categoryname="Computer software"
                   class="flagImage" title="Flag 'Computer software' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cytoplasm%22" title="Search for articles in the subject area:'Cytoplasm'"><div class="flagText">Cytoplasm</div></a>
              <div data-categoryid="17277" data-articleid="24582"
                   data-categoryname="Cytoplasm"
                   class="flagImage" title="Flag 'Cytoplasm' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24582"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="24582"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+translation%22" title="Search for articles in the subject area:'Protein translation'"><div class="flagText">Protein translation</div></a>
              <div data-categoryid="24691" data-articleid="24582"
                   data-categoryname="Protein translation"
                   class="flagImage" title="Flag 'Protein translation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomes%22" title="Search for articles in the subject area:'Ribosomes'"><div class="flagText">Ribosomes</div></a>
              <div data-categoryid="32243" data-articleid="24582"
                   data-categoryname="Ribosomes"
                   class="flagImage" title="Flag 'Ribosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Systems+biology%22" title="Search for articles in the subject area:'Systems biology'"><div class="flagText">Systems biology</div></a>
              <div data-categoryid="34555" data-articleid="24582"
                   data-categoryname="Systems biology"
                   class="flagImage" title="Flag 'Systems biology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6876'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8493'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8340&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=55489">Strong statement?</a><br>Posted by systemsbiologist</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
