

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001244"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001244" />

    <meta name="citation_title" content="Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension"/>
    <meta itemprop="name" content="Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension"/>

      <meta name="citation_author" content="Xiaofeng Zhu"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Biostatistics, School of Medicine, Case Western Reserve University, Cleveland, Ohio, United States of America"/>
      <meta name="citation_author" content="Richard S. Cooper"/>
            <meta name="citation_author_institution" content="Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University, Maywood, Illinois, United States of America"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001244.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1244"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Admixture mapping for hypertension loci with genome-scan markers.; citation_author=X Zhu; citation_author=A Luke; citation_author=RS Cooper; citation_author=T Quertermous; citation_author=C Hanis; citation_journal_title=Nat Genet; citation_volume=37; citation_number=1; citation_pages=177-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Is hypertension more severe among U.S. blacks, or is severe hypertension more common?; citation_author=RS Cooper; citation_author=Y Liao; citation_author=C Rotimi; citation_journal_title=Ann Epidemiol; citation_volume=6; citation_number=2; citation_pages=173-180; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Familial aggregation and genetic epidemiology of blood pressure.; citation_author=R Ward; citation_number=3; citation_pages=81-100; citation_date=1990; citation_publisher=Raven Press; " />
      <meta name="citation_reference" content="citation_title=Genome scans for hypertension and blood pressure regulation.; citation_author=NJ Samani; citation_journal_title=Am J Hypertens; citation_volume=16; citation_number=4; citation_pages=167-171; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Searching for genetic determinants in the new millennium.; citation_author=NJ Risch; citation_journal_title=Nature; citation_volume=405; citation_number=5; citation_pages=847-856; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease.; citation_author=KE Lohmueller; citation_author=CL Pearce; citation_author=M Pike; citation_author=ES Lander; citation_author=JN Hirschhorn; citation_journal_title=Nat Genet; citation_volume=33; citation_number=6; citation_pages=177-182; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Linkage and association analysis of angiotensin I-converting enzyme (ACE)-gene polymorphisms with ACE concentration and blood pressure.; citation_author=X Zhu; citation_author=N Bouzekri; citation_author=L Southam; citation_author=RS Cooper; citation_author=A Adeyemo; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=7; citation_pages=1139-1148; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Angiotensin I-converting enzyme polymorphisms, ACE level and blood pressure among Nigerians, Jamaicans and African-Americans.; citation_author=N Bouzekri; citation_author=X Zhu; citation_author=Y Jiang; citation_author=CA McKenzie; citation_author=A Luke; citation_journal_title=Eur J Hum Genet; citation_volume=12; citation_number=8; citation_pages=460-468; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Whole-genome patterns of common DNA variation in three human populations.; citation_author=DA Hinds; citation_author=LL Stuve; citation_author=GB Nilsen; citation_author=E Halperin; citation_author=E Eskin; citation_journal_title=Science; citation_volume=307; citation_number=9; citation_pages=1072-1079; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=10; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complement factor H polymorphism in age-related macular degeneration.; citation_author=RJ Klein; citation_author=C Zeiss; citation_author=EY Chew; citation_author=JY Tsai; citation_author=RS Sackler; citation_journal_title=Science; citation_volume=308; citation_number=11; citation_pages=385-389; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A common genetic variant is associated with adult and childhood obesity.; citation_author=A Herbert; citation_author=NP Gerry; citation_author=MB McQueen; citation_author=IM Heid; citation_author=A Pfeufer; citation_journal_title=Science.; citation_volume=312; citation_number=12; citation_pages=279-283; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (IFIH1) region.; citation_author=DJ Smyth; citation_author=JD Cooper; citation_author=R Bailey; citation_author=S Field; citation_author=O Burren; citation_journal_title=Nat Genet; citation_volume=38; citation_number=13; citation_pages=617-619; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants.; citation_author=LJ Scott; citation_author=KL Mohlke; citation_author=LL Bonnycastle; citation_author=CJ Willer; citation_author=Y Li; citation_journal_title=Science; citation_volume=316; citation_number=14; citation_pages=1341-1345; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity.; citation_author=TM Frayling; citation_author=NJ Timpson; citation_author=MN Weedon; citation_author=E Zeggini; citation_author=RM Freathy; citation_journal_title=Science; citation_volume=316; citation_number=15; citation_pages=889-894; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels.; citation_author=R Saxena; citation_author=BF Voight; citation_journal_title=Science; citation_volume=316; citation_number=16; citation_pages=1331-1336; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes.; citation_author=BW Zanke; citation_author=CM Greenwood; citation_author=J Rangrej; citation_author=R Kustra; citation_author=A Tenesa; citation_journal_title=Nat Genet; citation_volume=39; citation_number=17; citation_pages=977-983; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24.; citation_author=J Gudmundsson; citation_author=P Sulem; citation_author=V Steinthorsdottir; citation_author=JT Bergthorsson; citation_author=G Thorleifsson; citation_journal_title=Nat Genet; citation_volume=39; citation_number=18; citation_pages=989-994; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Risk alleles for multiple sclerosis identified by a genomewide study.; citation_journal_title=N Engl J Med; citation_volume=357; citation_number=19; citation_pages=851-862; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Statistical tests for admixture mapping with case-control and cases-only data.; citation_author=G Montana; citation_author=JK Pritchard; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=20; citation_pages=771-789; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Linkage analysis of a complex disease through use of admixed populations.; citation_author=X Zhu; citation_author=RS Cooper; citation_author=RC Elston; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=21; citation_pages=1136-1153; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Methods for high-density admixture mapping of disease genes.; citation_author=N Patterson; citation_author=N Hattangadi; citation_author=B Lane; citation_author=KE Lohmueller; citation_author=DA Hafler; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=22; citation_pages=979-1000; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Design and analysis of admixture mapping studies.; citation_author=CJ Hoggart; citation_author=MD Shriver; citation_author=RA Kittles; citation_author=DG Clayton; citation_author=PM McKeigue; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=23; citation_pages=965-978; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mapping genes that underlie ethnic differences in disease risk: methods for detecting linkage in admixed populations, by conditioning on parental admixture.; citation_author=PM McKeigue; citation_journal_title=Am J Hum Genet; citation_volume=63; citation_number=24; citation_pages=241-251; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A Classical Likelihood Based Admixture Mapping Method and Comparison with the Bayesian Approach.; citation_author=X Zhu; citation_author=SL Zhang; citation_author=H Tang; citation_author=RS Cooper; citation_journal_title=Hum Genet; citation_volume=120; citation_number=25; citation_pages=431-445; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reconstructing Genetic Ancestry Blocks in Admixed Individuals.; citation_author=H Tang; citation_author=M Coram; citation_author=P Wang; citation_author=X Zhu; citation_author=N Risch; citation_journal_title=Am J Hum Genet; citation_volume=79; citation_number=26; citation_pages=1-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A hidden Markov Modeling approach for admixture mapping based on case-control data.; citation_author=C Zhang; citation_author=K Chen; citation_author=MF Seldin; citation_author=H Li; citation_journal_title=Genet Epidemiol; citation_volume=27; citation_number=27; citation_pages=225-239; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Measureing and using admixture to study the genetics of complex diseases.; citation_author=I Halder; citation_author=MD Shriver; citation_journal_title=Hum Genomics; citation_volume=1; citation_number=28; citation_pages=52-62; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Admixture mapping as a discovery approach for complex human traits and diseases.; citation_author=CM Nievergelt; citation_author=NJ Schork; citation_journal_title=Curr Hypertens Rep; citation_volume=7; citation_number=29; citation_pages=31-37; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The beauty of admixture.; citation_author=A Darvasi; citation_author=S Shifman; citation_journal_title=Nat Genet; citation_volume=37; citation_number=30; citation_pages=118-119; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A high-density admixture map for disease gene discovery in african americans.; citation_author=MW Smith; citation_author=N Patterson; citation_author=JA Lautenberger; citation_author=AL Truelove; citation_author=GJ McDonald; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=31; citation_pages=1001-1013; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A whole-genome admixture scan finds a candidate locus for multiple sclerosis susceptibility.; citation_author=D Reich; citation_author=N Patterson; citation_author=PL De Jager; citation_author=GJ McDonald; citation_author=A Waliszewska; citation_journal_title=Nat Genet; citation_volume=37; citation_number=32; citation_pages=1113-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Discovering genotypes underlying human phenotypes: past successes for mendelian disease, future approaches for complex disease.; citation_author=D Botstein; citation_author=N Risch; citation_journal_title=Nat Genet 33 Suppl; citation_number=33; citation_pages=228-237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A common variant associated with prostate cancer in European and African populations.; citation_author=LT Amundadottir; citation_author=P Sulem; citation_author=J Gudmundsson; citation_author=A Helgason; citation_author=A Baker; citation_journal_title=Nat Genet; citation_volume=38; citation_number=34; citation_pages=652-658; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Admixture mapping identifies 8q24 as a prostate cancer risk locus in African-American men.; citation_author=ML Freedman; citation_author=CA Haiman; citation_author=N Patterson; citation_author=GJ McDonald; citation_author=A Tandon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=35; citation_pages=14068-14073; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mapping by admixture linkage disequilibrium: advances, limitations and guidelines.; citation_author=MW Smith; citation_author=SJ O'Brien; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=36; citation_pages=623-632; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.; citation_author=D Falush; citation_author=M Stephens; citation_author=JK Pritchard; citation_journal_title=Genetics; citation_volume=164; citation_number=37; citation_pages=1567-1587; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A genomewide single-nucleotide-polymorphism panel with hihg ancestry information for African American admixture mapping.; citation_author=C Tian; citation_author=DA Hinds; citation_author=R Shigeta; citation_author=R Kittles; citation_author=DG Ballinger; citation_journal_title=Am J Hum Genet; citation_volume=79; citation_number=38; citation_pages=640-649; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Confronting ethnicity-specific disease risk.; citation_author=H Tang; citation_journal_title=Nat Genet; citation_volume=38; citation_number=39; citation_pages=13-15; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world.; citation_author=T Nakajima; citation_author=S Wooding; citation_author=T Sakagami; citation_author=M Emi; citation_author=K Tokunaga; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=40; citation_pages=898-916; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=variation and the evolution of salt-sensitivity variants.; citation_author=EE Thompson; citation_author=H Kuttab-Boulos; citation_author=D Witonsky; citation_author=L Yang; citation_author=BA Roe; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=41; citation_pages=1059-1069; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A Differential Susceptibility to Hypertension Is Due to Selection during the Out-of-Africa Expansion.; citation_author=JH Young; citation_author=YP Chang; citation_author=JD Kim; citation_author=JP Chretien; citation_author=MJ Klag; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=42; citation_pages=e82; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A map of recent positive selection in the human genome.; citation_author=BF Voight; citation_author=S Kudaravalli; citation_author=X Wen; citation_author=JK Pritchard; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=43; citation_pages=e154; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pantetheinase activity of membrane-bound vanin-1: lack of free cysteamine in tissues of vanin-1 deficient mice.; citation_author=G Pitari; citation_author=F Malergue; citation_author=F Martin; citation_author=JM Philippe; citation_author=MT Massucci; citation_journal_title=FEBS Lett.; citation_volume=483; citation_number=44; citation_pages=149-154; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Pathogenesis of hypertension.; citation_author=S Oparil; citation_author=MA Zaman; citation_author=DA Calhoun; citation_journal_title=Ann Intern Med; citation_volume=139; citation_number=45; citation_pages=761-776; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes.; citation_author=HH Goring; citation_author=JE Curran; citation_author=MP Johnson; citation_author=TD Dyer; citation_author=J Charlesworth; citation_journal_title=Nat Genet.; citation_volume=39; citation_number=46; citation_pages=1208-16; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=VNN1, A novel gene for cardiovascular disease risk.; citation_author=JE Curran; citation_author=MP Johnson; citation_author=HH Goring; citation_author=TD Dyer; citation_author=JC Charlesworth; citation_journal_title=Am J Hum Genet. 2007. Abstract.; citation_number=47; citation_pages=62; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Dallas Heart Study: a population-based probability sample for the multidisciplinary study of ethnic differences in cardiovascular health.; citation_author=RG Victor; citation_author=RW Haley; citation_author=DL Willett; citation_author=RM Peshock; citation_author=PC Vaeth; citation_journal_title=Am J Cardiol; citation_volume=93; citation_number=48; citation_pages=1473-1480; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Haploview: analysis and visualization of LD and haplotype maps.; citation_author=JC Barrett; citation_author=B Fry; citation_author=J Maller; citation_author=MJ Daly; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=49; citation_pages=263-265; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=50; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genomic control for association studies.; citation_author=B Devlin; citation_author=K Roeder; citation_journal_title=Biometrics; citation_volume=55; citation_number=51; citation_pages=997-1004; citation_date=1999; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension"/>
    <meta name="twitter:description" content="Migration patterns in modern societies have created the opportunity to use population admixture as a strategy to identify susceptibility genes. To implement this strategy, we genotyped a highly informative ancestry marker panel of 2270 single nucleotide polymorphisms in a random population sample of African Americans (N&#8202;=&#8202;1743), European Americans (N&#8202;=&#8202;1000) and Mexican Americans (N&#8202;=&#8202;581). We then examined the evidence for over-transmission of specific loci to cases from one of the two ancestral populations. Hypertension cases and controls were defined based on standard clinical criteria. Both case-only and case-control analyses were performed among African Americans. With the genome-wide markers we replicated the findings identified in our previous admixture mapping study on chromosomes 6 and 21 [1]. For case-control analysis we then genotyped 51 missense SNPs in 36 genes spaced across an 18.3 Mb region. Further analyses demonstrated that the missense SNP rs2272996 (or N131S) in the VNN1 gene was significantly associated with hypertension in African Americans and the association was replicated in Mexican Americans; a non-significant opposite association was observed in European Americans. This SNP also accounted for most of the evidence observed in the admixture analysis on chromosome 6. Despite these encouraging results, susceptibility loci for hypertension have been exceptionally difficult to localize and confirmation by independent studies will be necessary to establish these findings."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001244.g004"/>

  <meta property="og:title" content="Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1163'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8107'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9509&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001244">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001244" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001244&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaofeng Zhu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:xzhu1@darwin.case.edu">xzhu1@darwin.case.edu</a></p>

                <p>Affiliation:
                  Department of Epidemiology and Biostatistics, School of Medicine, Case Western Reserve University, Cleveland, Ohio, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard S. Cooper
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Preventive Medicine and Epidemiology, Stritch School of Medicine, Loyola University, Maywood, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001244</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001244-t001" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-g001" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-t002" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-t003" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-g002" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-g003" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-t004" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-t005" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001244-g004" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001244">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001244" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Migration patterns in modern societies have created the opportunity to use population admixture as a strategy to identify susceptibility genes. To implement this strategy, we genotyped a highly informative ancestry marker panel of 2270 single nucleotide polymorphisms in a random population sample of African Americans (N = 1743), European Americans (N = 1000) and Mexican Americans (N = 581). We then examined the evidence for over-transmission of specific loci to cases from one of the two ancestral populations. Hypertension cases and controls were defined based on standard clinical criteria. Both case-only and case-control analyses were performed among African Americans. With the genome-wide markers we replicated the findings identified in our previous admixture mapping study on chromosomes 6 and 21 <a href="#pone.0001244-Zhu1">[1]</a>. For case-control analysis we then genotyped 51 missense SNPs in 36 genes spaced across an 18.3 Mb region. Further analyses demonstrated that the missense SNP rs2272996 (or N131S) in the VNN1 gene was significantly associated with hypertension in African Americans and the association was replicated in Mexican Americans; a non-significant opposite association was observed in European Americans. This SNP also accounted for most of the evidence observed in the admixture analysis on chromosome 6. Despite these encouraging results, susceptibility loci for hypertension have been exceptionally difficult to localize and confirmation by independent studies will be necessary to establish these findings.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhu X, Cooper RS (2007) Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension. PLoS ONE 2(11):
          e1244.
            doi:10.1371/journal.pone.0001244</p><p><strong>Academic Editor: </strong>Greg Gibson, North Carolina State University, United States of America</p><p><strong>Received:</strong> November 1, 2007; <strong>Accepted:</strong> November 6, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Zhu, Cooper. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants (HG003054) from the National Human Genome Research Institute, (HL074166) from the National Heart, Lung, and Blood Institute, the Donald W. Reynolds Clinical Cardiovascular Center at UT Southwestern, Dallas, TX.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Hypertension is a consequence of common lifestyle patterns in modern society and makes an important contribution to risk of cardiovascular disease. The prevalence of hypertension varies among ethnic populations in the US from 25 to 40% and is an attributable cause for approximately 13% of deaths in whites and 24% in blacks <a href="#pone.0001244-Cooper1">[2]</a>. Blood pressure is a moderately heritable trait and results from the combined effect of a complex set of genetic and environmental influences, with genes cumulatively accounting for 30% of the population variance <a href="#pone.0001244-Ward1">[3]</a>. Genome-wide linkage analysis has been widely applied in efforts to identify genomic regions harboring genes affecting the risk of hypertension. A recent review of 20 genome scans suggested that a large number genes, each exerting a small effect, is the most likely molecular architecture underlying hypertension <a href="#pone.0001244-Samani1">[4]</a>. The observed effects are highly inconsistent, however, and it is well recognized that linkage analysis has limited power when applied to complex traits <a href="#pone.0001244-Risch1">[5]</a>; locus heterogeneity may further contribute to this observed inconsistency.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>As an alternative to linkage methods, numerous candidate genes selected on physiological and/or metabolic criteria have been examined using single nucleotide polymorphisms (SNPs) or associated haplotypes. Despite the myriad of reports, meta-analyses have identified few candidate loci with consistent effects across population samples <a href="#pone.0001244-Lohmueller1">[6]</a>. For example, the genes underlying physiologic systems that control BP, like the renin-angiotensin axis, have been extensively studied, and yielding inconsistent results <a href="#pone.0001244-Zhu2">[7]</a>, <a href="#pone.0001244-Bouzekri1">[8]</a>. Genome-wide association studies based on 100,000 or more SNPs are now technically feasible <a href="#pone.0001244-Hinds1">[9]</a>, <a href="#pone.0001244-The1">[10]</a> and initial results with macular degeneration, obesity, type 1 and type 2 diabetes, prostate cancer and multiple sclerosis, among others, suggest promise for this method <a href="#pone.0001244-Klein1">[11]</a>–<a href="#pone.0001244-International1">[19]</a>. These studies are still quite expensive, however, and can only be conducted by a limited number of laboratories.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>On the basis of recent theoretical work it has been suggested that the information generated by recent admixture of historically separated populations can help to map disease-associated genes <a href="#pone.0001244-Montana1">[20]</a>–<a href="#pone.0001244-Nievergelt1">[29]</a>. Admixture mapping can be more powerful than traditional linkage analysis when the relative risk in the parental populations is substantially different and much less genotyping is required in comparison to association analysis based on linkage disequilibrium. Admixture mapping may also be less sensitive to genetic heterogeneity <a href="#pone.0001244-Darvasi1">[30]</a>. Based on these theoretical propositions, Zhu et al. performed the first large-scale genome wide admixture mapping study in African Americans using the markers designed for a family-based linkage study <a href="#pone.0001244-Zhu1">[1]</a>. In these analyses the distribution of marker location-specific ancestry was shifted upward in hypertensive cases versus normotensives and this shift was largely due to the loci on 6q24 and 21q21, indicating that genetic variants in these two regions may influence the risk of hypertension. Since the marker information content for inferring ancestry was relatively low in this initial study, the findings require further confirmation. Consequently using a large, multi-ethnic population sample, we conducted an admixture mapping study by genotyping a marker panel that is highly informative for ancestry for the African-American population <a href="#pone.0001244-Smith1">[31]</a>. In this paper we describe the admixture linkage results for hypertension in the Dallas Heart Study, followed by an association study of all missense SNPs in the region identified by the admixture mapping.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The demographic and descriptive characteristics of the hypertensive cases and controls are presented in <a href="#pone-0001244-t001">Table 1</a> for the European-American, African-American and Mexican-American population samples. Cases were on average older than controls in all three groups, as anticipated; the percentage of treated cases was similar.</p>
<div class="figure" id="pone-0001244-t001"><div class="img"><a name="pone-0001244-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Descriptive characteristics of the study subjects from each of racial/ethnic groups (means±sd)</span></strong></p><span>doi:10.1371/journal.pone.0001244.t001</span></div>
<h4>Admixture mapping in African-Americans</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>2,270 ancestry informative SNPs located on 22 autosomes selected from the SNP panel of Smith et al.<a href="#pone.0001244-Smith1">[31]</a> were successfully genotyped. After examination of Hardy-Weinberg equilibrium (HWE) for possible genotyping errors and background linkage disequilibrium which may violate the assumption of the method <a href="#pone.0001244-Zhu4">[25]</a>, only 1,890 SNPs were used in further analyses. Our analysis used the software ADMIXPROGRAM, which is based on the hidden Markov model using the continuous gene flow model <a href="#pone.0001244-Zhu4">[25]</a>. The estimated number of generations since the occurrence of population admixture was 12 in this African-American sample. We estimated each individual's average African and European ancestries (<a href="#pone.0001244.s001">Table S1</a>)(European ancestry = 15.8±7.6% in cases, and 16.9±8.3% in controls) as well as ancestry at each marker locus in cases and controls separately (<a href="#pone.0001244.s002">Table S2</a>). We then calculated the genome-wide case-only and case-control Z scores, as presented in <a href="#pone-0001244-g001">figure 1</a>. The Z score can be used to evaluate evidence for a disease variant at each location and it asymptotically follows the standard normal distribution <a href="#pone.0001244-Zhu3">[21]</a>, <a href="#pone.0001244-Zhu4">[25]</a>. We observed 6 markers with both |Z<sub>case</sub>| and |Z<sub>case-control</sub>|&gt;2.0 distributed on 6 chromosomes (<a href="#pone-0001244-t002">Table 2</a>). Among them, the signal identified by SNP rs703193 (137cM) on chromosome 6 falls within the region identified in our previous study <a href="#pone.0001244-Zhu1">[1]</a>. To further rule out that the possibility that the evidence observed on chromosome 6 results from the bias resulting either from mis-estimation of a specific marker allele's frequency in ancestral populations or background linkage disequilibrium, we calculated both the case-only and case-control Z scores using different sets of SNPs as suggested by Reich et al. <a href="#pone.0001244-Reich1">[32]</a> (<a href="#pone-0001244-t003">Table 3</a>). Similar evidence was observed at rs703193 when we used the SNPs with an adjacent distance greater than 1 cM. The evidence remains when either odd SNPs or even SNPs were used. Gender-specific analysis suggested that women contribute more evidence, which may in part reflect the inclusion of more hypertensive women in the study (<a href="#pone-0001244-t001">Table 1</a>). The relative risk of hypertension due to two copies of African ancestry allele in the region around SNP rs703193 was 1.32 (95% CI: (1.04, 1.67), p = 0.023). We also examined the region on chromosome 21 identified in our previous study <a href="#pone.0001244-Zhu1">[1]</a>. Interestingly, the admixture mapping evidence in the region around rs380417 on chromosome 21 was weaker than on chromosome 6q, but with a stronger relative risk of African ancestry at rs380417 = 1.48 (viz, 95%CI: (1.15, 1.89), p = 0.0021).</p>
<div class="figure" id="pone-0001244-g001"><div class="img"><a name="pone-0001244-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The genome-wide Z-scores.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Top: the Z-scores calculated using hypertensive cases only ; Bottom: the Z-scores calculated based on case-control samples.</p>
<span>doi:10.1371/journal.pone.0001244.g001</span></div><div class="figure" id="pone-0001244-t002"><div class="img"><a name="pone-0001244-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Marker locations with the maximum absolute Z score larger than 2.0 for association with hypertension</span></strong></p><span>doi:10.1371/journal.pone.0001244.t002</span></div><div class="figure" id="pone-0001244-t003"><div class="img"><a name="pone-0001244-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Zcase and Zcase-control scores at rs703193 when different set of ancestral informative SNPs are used.</span></strong></p><span>doi:10.1371/journal.pone.0001244.t003</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We next empirically explored the probability of Z<sub>case</sub> and Z<sub>case-control</sub> scores greater than we observed on chromosome 6 and 21 in the African-American sample when no genetic variants contributed to a trait. We applied a one-sided test for the African-American sample because of previous evidence in this region. We then simulated 1,000 data sets for both samples with the estimated parental allele frequencies and 12 generations based on a continuous gene flow model <a href="#pone.0001244-Zhu3">[21]</a>. The numbers of analyses with Z<sub>case</sub>&gt;2.1, Z<sub>case-control</sub>&gt;2.8 for chromosome 6, and Z<sub>case</sub>&gt;2.1, Z<sub>case-control</sub>&gt;1.4 for chromosome 21, were summarized. Among 1,000 simulations, we observed 184 instances where both Z<sub>case</sub>&gt;2.1, Z<sub>case-control</sub>&gt;2.8 and 688 instances where both Z<sub>case</sub>&gt;2.1 and Z<sub>case-control</sub>&gt;1.4 respectively, indicating that our observations on both 6q and 21q do not reach genome-wide significance. However, when we restricted the analysis to the regions between 129–166 cM on chromosome 6 and between 0–30 cM on chromosome 21 identified in our previous study <a href="#pone.0001244-Zhu1">[1]</a>, we only observed 4 and 11 instances respectively (p = 0.004 and 0.011), suggesting that our results do in fact replicate the previous findings. We performed similar analyses with the other 5 regions, using a two-side test since no prior evidence was observed in these regions, and none reached genome-wide significance (<a href="#pone-0001244-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We performed similar analysis using the software STRUCTURE. Among the African Americans, European ancestry was estimated as 15.8%±8.3% in cases, and 17.0±9.1% in controls, which are the same as we obtained with ADMIXPROGRAM. The detailed European ancestry for each individual estimated by both methods is presented in Supplement <a href="#pone.0001244.s001">Table S1</a>. The correlation between the two methods is 99.96%. We also calculated the case-only and case-control Z scores by excluding the estimates of an individual's marker-location specific ancestry if the difference of the estimates between the two methods is greater than 0.1. No substantial changes in the Z scores were observed (<a href="#pone.0001244.s002">Table S2</a>).</p>


<h4>Association analysis on chromosome 6</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We next focused on the region around rs703193 defined by Z<sub>case</sub>&gt;1.0 on chromosome 6 which encompassed 18.3 Mb and 36 genes. We hypothesized that missense SNPs are more likely responsible for the evidence identified by admixture mapping <a href="#pone.0001244-Risch1">[5]</a>, <a href="#pone.0001244-Botstein1">[33]</a>. We identified 51 missense SNPs in the 36 genes from public databases and typed them in the African-American sample (<a href="#pone-0001244-g002">Figure 2</a>). Eight of these SNPs were found to have a MAF&lt;5% and an additional 4 were in strong HWD (p&lt;0.01). Admixture mapping has extremely limited power to identify rare variants so these 12 SNPs were excluded from further analyses. We performed logistic regression on each of the remaining 39 SNPs assuming an additive model with adjustment for gender, age, BMI and estimated individual ancestry (IA) based on the original 1890 marker panel. We chose to focus only on the additive model in this step because the reduction of multiple comparisons more than compensates for the small sacrifice in power associated with this strategy.</p>
<div class="figure" id="pone-0001244-g002"><div class="img"><a name="pone-0001244-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The Z-scores and 39 missense SNPs on chromosome 6.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The region represents 18.3 Mb with Z-score &gt;1,0. (in black): case-only Z score (in red) : case-control Z score; small vertical bars on the X-axis represent the location of the 39 missense SNPs.</p>
<span>doi:10.1371/journal.pone.0001244.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We also adjusted for IA to reduce the likelihood of false association that results from population stratification. Association with hypertension was identified at α = 0.05 significance level at rs2244008 (A/G) and rs2272996(C/T) in the LAMA2 and VNN1 genes, respectively (<a href="#pone-0001244-t004">Table 4</a>). We then examined whether these two SNPs captured the evidence from admixture mapping. First, we attempted to isolate the ancestry effect. After inferring each individual's ancestry at the admixture peak (137cM) we repeated the logistic regression analysis in the subgroup whose African ancestry was &gt;80% at this locus, assuming that this reduced any potential effects of population stratification. While the result for rs2244008 was unchanged, the association of rs2272996 with hypertension was strengthened (p = 0.00248) despite the reduction in sample size. Based on a permutation test this SNP remained significant (p = 0.045) after adjustment for multiple comparisons (See <a href="#s4">Methods</a>). SNP rs2272996 is located 3.6 Mb from rs703193, with virtually no linkage disequilibrium between these two SNPs (D' = 0.008, 95% CI (−0.01–0.12)). To determine if two SNPs with this relationship can potentially be in LD in an admixed population similar to African-Americans, we calculated the LD between these two SNPs in a synthetic admixed population using HapMap YRI and CEU date. We assumed that the admixture proportions were 84% African and 16% European. Under these conditions D' was estimated as 0.095, which is within the 95% CI of the finding in our data, suggesting that an AIM used for admixture mapping is not necessarily in strong LD with a SNP associated with the phenotype. We also performed logistic regression for SNP rs703193 adjusting for gender, age, BMI and IA and no evidence of association was found. To further evaluate the plausibility of the evidence for rs2244008 we compared the allele frequency between YRI and CEU samples from HapMap data <a href="#pone.0001244-The1">[10]</a>, as well as in our own European-American sample. The frequencies were almost the same, confirming that this SNP is unlikely to account for the evidence observed by the admixture mapping. In contrast, for rs2272996 the allele frequencies were substantially different between YRI and CEU and between YRI and our own European-American sample (<a href="#pone-0001244-t004">Table 4</a>). Further analysis suggested that applying a recessive model rs2272996 (TT vs TC and CC) yielded the most significant association, with hypertension risk increased in the presence of the TT genotype (p = 0.0021, <a href="#pone-0001244-t005">Table 5</a>). Further correction for multiple comparisons of 39 SNPs and additive and recessive modes of inheritance yielded a p-value of 0.06 for rs2272996. We therefore hypothesized that the excess of African ancestry at the peak location (137cM) in cases was accounted for by those who carried the TT genotype. <a href="#pone-0001244-g003">Figure 3</a> presents African ancestry at the peak location for people who carry TT compared to the remaining genotypes between cases and controls. As expected, we observed a significant excess of African ancestry between cases and controls who were TT (p = 0.00045) but not those that were TC and CC (p = 0.34). These results suggest that rs2272996 is a potential candidate to explain the admixture mapping evidence. When we added rs2272996 in an admixture mapping analysis and the Z score at rs227299 did not improve. This result is reasonable because the information content after adding rs2272996 does not increase, given the AIMs used in this region.</p>
<div class="figure" id="pone-0001244-g003"><div class="img"><a name="pone-0001244-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>African ancestry in cases and controls grouped by genotypes of SNP rs2272996.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A significant excess of African ancestry between cases and controls who were TT was observed but among those that bore the genotype TC and CC.</p>
<span>doi:10.1371/journal.pone.0001244.g003</span></div><div class="figure" id="pone-0001244-t004"><div class="img"><a name="pone-0001244-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Two SNPs identified by logistic regression analysis assuming additive model at significance level 0.05. Gender, Age, age<sup>2</sup>, BMI and African ancestry were adjusted. Minor allele was coded as the risk allele.</span></strong></p><span>doi:10.1371/journal.pone.0001244.t004</span></div><div class="figure" id="pone-0001244-t005"><div class="img"><a name="pone-0001244-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Analysis of rs2272996 in African Americans, Mexican Americans and European Americans using recessive model (TT vs TC, CC). Gender, Age, age<sup>2</sup>, BMI and African ancestry were adjusted. Allele T was coded as the risk allele.</span></strong></p><span>doi:10.1371/journal.pone.0001244.t005</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We then analyzed rs2272996 by assuming a recessive model in both European Americans and Mexican Americans using logistic regression with adjustment for gender, age, age<sup>2</sup>, and BMI. We observed significant evidence of association between rs2272996 and hypertension in Mexican Americans (p = 0.021), where the TT genotype increased the risk of hypertension (<a href="#pone-0001244-t005">Table 5</a>). Among European Americans, on the other hand, a similar analysis demonstrated the opposite–albeit non-significant–association (TT genotype protective, odds ratio = 0.79 (p = 0.089). This result is consistent with the findings in African Americans, where the association increased when analysis was limited to individuals with &gt;80% African ancestry at the peak by admixture mapping. We then estimated the population attributable risk of rs2272996 in the African-American and Mexican populations. Under the recessive model we obtained a population attributable risk of 16% in African-Americans, and 35% in Mexican-Americans; this result occurs because T-the risk allele–is relatively common in both populations.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Admixture mapping has been proposed as an alternative to traditional linkage and association studies and in theory holds great promise for selected traits <a href="#pone.0001244-Montana1">[20]</a>–<a href="#pone.0001244-Nievergelt1">[29]</a>. In the first application of this strategy to mapping complex traits, Zhu et al. performed a genome-wide admixture mapping of hypertension in an African-American sample based on a set of microsatellite markers designed for traditional linkage analysis and identified two regions on chromosome 6 and 21 that potentially harbored hypertension susceptibility genes <a href="#pone.0001244-Zhu1">[1]</a>. Reich et al. <a href="#pone.0001244-Reich1">[32]</a> also reported a locus on chromosome 1 associated with multiple sclerosis by admixture mapping. More recently, linkage analysis followed by fine mapping identified a chromosome region harboring genes for susceptibility to prostate cancer <a href="#pone.0001244-Amundadottir1">[34]</a> and this locus was subsequently replicated with admixture mapping <a href="#pone.0001244-Freedman1">[35]</a>. In this report we present additional evidence from genome-wide admixture mapping for hypertension in 1743 unrelated African Americans using a highly informative SNP panel <a href="#pone.0001244-Smith1">[31]</a>. This panel extracted on average 89% of the information of ancestry (measured by SIC) <a href="#pone.0001244-Zhu4">[25]</a> for the African-American sample and as a consequence the results from both case-only and case-control tests were reasonably robust. Our estimate of the average information for ancestry is higher than was found in the original report for this SNP panel <a href="#pone.0001244-Smith1">[31]</a>. This may be explained by the fact that we assumed a 50%–50% mixture when no SNPs are genotyped while Smith et al. assumed a 79%–21% mixture. Under the latter assumption the estimated average information was 65%, slightly less than the 71% suggested by Smith et al. <a href="#pone.0001244-Smith1">[31]</a>. The estimated ancestral allele frequencies in the parental populations are close to the corresponding observed frequencies in contemporary African and European populations, suggesting that our model fits the data well. We observed 6 regions with consistent excess of African or European ancestry in both case-only and case-control analyses, including the region on chromosome 6 identified in our previous study <a href="#pone.0001244-Zhu1">[1]</a>. However, simulation studies indicate that we did not find any region that reaches genome-wide significance in this study. Although the regions on chromosome 6 and 21 identified in our previous study are large (37cM and 30 cM, respectively), our simulations indeed suggest that the present result is unlikely to be due to chance. It is interesting that we observed stronger evidence but less significant association with the African-derived allele on chromosome 6 than on chromosome 21. Several reasons may explain this outcome. 1) The power of admixture mapping is dependent on the underlying genetic model, with the recessive mode of inheritance being more powerful than additive or dominant modes <a href="#pone.0001244-Zhu3">[21]</a>. A recessive effect for the T allele of missense SNP rs2272996 in VNN1 gene best explains the evidence, suggesting it is reasonable to observe better admixture mapping evidence on chromosome 6 than 21. 2) The power of admixture mapping is dependent on the disease allele frequency difference between two ancestral populations, while the power of association is dependent on the linkage disequilibrium between the underlying disease variant and the test marker, which is the ancestry in this case, and the disease allele frequency in the admixed population. We performed a power analysis by assuming that the missense SNP rs2272996 is the true causative variant. We further assumed that the odds ratio of the TT genotype vs others in African ancestry population is 1.57, as estimated in <a href="#pone-0001244-t005">Table 5</a>. Since we failed to observe evidence of association in the European American population, the odds ratio was simply placed at 1.0, which leads to the estimate of the relative population risk ratio of 1.38. Under these assumptions, our power analysis suggests that, post hoc, we had 38% power to detect a region reaching the genome wide significance with the sample size in the current study. Thus, our study clearly has relatively low power to find similar locus. 3) A spurious finding is still possible because of the statistical fluctuation despite our previous report. Further association analysis is necessary to rule out the possibility that this is a false positive finding.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We performed our analysis based on a method which directly maximizes the likelihood function from the hidden Markov Model using the EM algorithm, allowing for uncertainty in model parameters, such as the allele frequencies in the parental populations <a href="#pone.0001244-Zhu4">[25]</a>. Simulations suggested that this method can perform as well as the widely used Bayesian MCMC method STRUCTURE for the data generated from various population admixture models.<a href="#pone.0001244-Zhu4">[25]</a> However, we believe the results can be regarded with greater confidence if different approaches lead to similar results, as suggested by others <a href="#pone.0001244-Smith2">[36]</a>. We thus performed similar analysis using STRUCTURE, although we are also aware that other similar MCMC based software has become available <a href="#pone.0001244-Patterson1">[22]</a>–<a href="#pone.0001244-McKeigue1">[24]</a>. The difference between ADMIXPROGRAM and STRUCTURE includes the approach to estimation of a large number of parameters, eg, ADMIXPROGRAM directly maximizing the likelihood function based on EM algorithm while STRUCTURE uses a Gibbs sampling scheme. The other difference involves the transition probability in the Hidden Markov model. ADMIXPROGRAM uses the transition probability derived from a continuous-gene-flow model and STRUCTURE uses linkage model by assuming that chunks of chromosomes are derived from ancestral populations and the breakpoints between successive chunks occur randomly <a href="#pone.0001244-Zhu4">[25]</a>, <a href="#pone.0001244-Falush1">[37]</a>. Despite the substantial difference of the two approaches, the results are almost identical, providing further confidence that our results are not biased of the selection of the statistical method.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Several publications <a href="#pone.0001244-Patterson1">[22]</a>, <a href="#pone.0001244-Hoggart1">[23]</a>, <a href="#pone.0001244-Zhu4">[25]</a>, <a href="#pone.0001244-Tang1">[26]</a>, <a href="#pone.0001244-Tian1">[38]</a> have demonstrated that admixture mapping can be seriously biased by background LD between adjacent SNPs when this phenomenon is not properly considered. We thus examined the background LD in European Americans and dropped the SNPs that are in strong LD. Since the maker panel was initially selected to minimize background LD, eliminating additional selected markers based on this characteristic had a very limited effect. We further repeated the analyses using different sets of SNPs: (eg, odd or even SNPs , those with adjacent distance&gt;1 cM) and the results on chromosome 6 were essentially the same. We note, however, that more information might have been obtained by keeping those SNPs that were in LD when using the Markov Hidden Markov Model method recently proposed by Tang et al.<a href="#pone.0001244-Tang1">[26]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We next followed up the region on chromosome 6 by genotyping 51 missense SNPs in 36 genes spaced across the 18.3 Mb region. Searching for disease variants by testing all functional SNPs has been advocated by Risch <a href="#pone.0001244-Risch1">[5]</a> and Risch and Botstein <a href="#pone.0001244-Botstein1">[33]</a> and this strategy successfully identified the sixth type 1 diabetes locus by examining all functional SNPs across the genome <a href="#pone.0001244-Smyth1">[13]</a>. We adapted this strategy in the chromosome 6 region and identified a missense SNP rs2272996 (or N131S) in the VNN1 gene significantly associated with hypertension in African Americans after adjusting for multiple comparisons; this association was replicated in Mexican Americans. A non-significant association in the opposite direction was observed in European Americans. (No correction was made for multiple comparisons in Mexican Americans and European Americans since we tested only rs2272996 in these two populations.) Further analysis also indicates that this SNP accounts for most of the evidence of excess of African ancestry observed in this region for African Americans. Thus, the association of SNP rs2272996 to hypertension is unlikely to be due to chance alone, although additional replication studies in independent studies are necessary. It should also be noted that our search of causative variants was not comprehensive and other susceptibility variants may well exist in this region.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We were also puzzled by the direction of the risk estimate associated with allele T in different populations. Theory suggests that the divergent risk relationship at this locus in ancestral populations will increase the power of admixture mapping method. However, the different patterns of risk in these populations cannot be explained with the data currently available, but could reflect gene-gene or gene-environmental interactions <a href="#pone.0001244-Tang2">[39]</a>. It is possible that this locus was under different selective pressure in different populations, however, the evolutionary processes that have molded susceptibility to chronic cardiovascular disease have not been defined. Whether the at-risk alleles arose under positive selection or a neutral-equilibrium model therefore cannot be determined. The suggestion has been made that susceptibility variants for hypertension may have been under selection pressure due to the climate adaptation, however most of the loci identified under this assumption have not been replicated in association analyses <a href="#pone.0001244-Nakajima1">[40]</a>–<a href="#pone.0001244-Young1">[42]</a>. Under this model, the variants are unlikely to be deleterious and could be common. Voight et al.<a href="#pone.0001244-Voight1">[43]</a> proposed a method to detect signals of very recent positive selection in the human genome using HapMap data and the VNN1 gene is located in one of the regions with strong positive selection pressure in YRI sample. VNN1 belongs to the vanin family of proteins, including secreted and membrane-associated proteins which have been reported to participate in hematopoietic cell trafficking and to possess pantetheinase activity which may play a role in oxidative-stress response <a href="#pone.0001244-Pitari1">[44]</a>. It has been recently suggested that increased oxidative stress may antedate hypertension and contribute to its pathogenesis <a href="#pone.0001244-Oparil1">[45]</a>. More recently, VNN1 has been suggested as a novel gene for cardiovascular disease risk, strongly associated with expression levels of several lipid metabolism/CVD-risk genes, <a href="#pone.0001244-Goring1">[46]</a>, <a href="#pone.0001244-Curran1">[47]</a> although the underlying pathway of VNN1 and other CVD-risk genes remains unknown.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, we conducted a genome-wide search for susceptibility loci for hypertension using admixture mapping in African Americans. Further association studies identified a missense SNP rs2272996 in the VNN1 gene that may explain the evidence identified through admixture analysis. While requiring further confirmation, this finding demonstrates the potential for the admixture approach and suggests it could be a tool for use in studies to define genes affecting selected complex traits. In addition, our study only surveyed the non-coding SNPs on the region on chromosome 6. A more comprehensive assessment of variants in this region will be required to provide assurance that the causative variants associated with hypertension have been identified.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The design and methodology of the Dallas Heart Study have been reported elsewhere <a href="#pone.0001244-Victor1">[48]</a>. In brief, a multistep probability sample of civilian, non-institutionalized English- or Spanish-speaking Dallas County residents were recruited from households. African Americans were oversampled to ensure that they represented 50% of the final cohort. Eligible subjects were invited to participate in three stages of the project including two home visits during which a survey was administered and blood and urine specimens were obtained and a third visit at the University of Texas at Southwestern Medical Center during which imaging studies were accomplished. Three thousand three hundred and ninety eight individuals (52% black) participated in the blood drawing that led to DNA isolation. Informed consent was obtained from all participants and the Institutional Review Board of the University of Texas Southwestern Medical Center approved the study protocol.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Race-ethnicity was based on self-identification. Subjects were asked in separate questions “Are you of Mexican American origin” and “What is your primary racial or ethnic identity.” The following options were provided for the latter question: Black/African American; White/Caucasian; American Indian, Alaska Native; Asian, Pacific Islander, East Indian; Other (Specify). Self-identified whites were used as a proxy for the European founding populations in the genetic analyses.</p>


<h4>Phenotype</h4>
<h5>Blood pressure/hypertension.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p>At each of the three visits (home visits 1 and 2 and visit 3), 5 sets of blood pressure measurements were obtained using an automatic oscillometric device (Welch Allyn, Series #52,000, Arden, NC) with an appropriately sized blood pressure cuff. This device has been validated against catheter measurement of arterial pressure <a href="#pone.0001244-Victor1">[48]</a>. The BP for analysis was considered the average of measurements 3 through 5 at each visit (total 9 readings). Hypertensive cases were defined as persons with either a systolic BP&gt; = 140 or diastolic BP&gt; = 90, or current treatment with an antihypertensive medication.</p>



<h4>Genotyping</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For each subject, DNA sample was individually genotyped by <em>Perlegen Sciences</em> for the complete marker set defined by Smith et al <a href="#pone.0001244-Smith1">[31]</a>. Among them, 2,270 SNPs were successfully genotyped. Fifty one additional missense SNPs in 36 genes on chromosome 6 were identified from National Center for Biotechnology Information (NCBI) build 34, and then genotyped by Perlegen Sciences.</p>


<h4>Selection of admixture marker set</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Hardy-Weinberg equilibrium (HWE) was first examined for all the 2,270 successfully genotyped SNPs on 22 autosomes in the three populations separately using the software Haploview <a href="#pone.0001244-Barrett1">[49]</a>. Thirty two SNPs were identified with significant departure from HWE at significance level α&lt;0.001 and were excluded in future analyses. To study the LD in the ancestral populations, pairwise linkage disequilibrium defined by D' values between SNPs on the same chromosome was calculated using the software Haploview in Whites. We identified 251 sets of adjacent SNPs in strong LD (95% lower bound D'&gt;0.5). Since strong LD between adjacent markers can seriously bias the estimation of marker location-specific ancestry <a href="#pone.0001244-Tang1">[26]</a>, we retained only one SNP from each of these sets in the subsequent analyses. After these data verification procedures, we defined an analytic set of 1,890 from the original 2,270 SNPs. These SNPs were entered in the hidden Markov model (HMM) to estimate the marker locus specific ancestry in African-Americans based on EM algorithm using software ADMIXPROGRAM <a href="#pone.0001244-Zhu4">[25]</a>.</p>


<h4>Estimation of marker location-specific ancestry</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The genome-wide admixture mapping analysis was only performed in African-Americans since the SNP set of Smith et al. was specifically selected for the African-American population. We estimated marker location-specific ancestry using the HMM, in which the transmission probability was calculated based on the continuous gene flow (CGF) model <a href="#pone.0001244-Zhu3">[21]</a>, <a href="#pone.0001244-Zhu4">[25]</a>. This method directly maximizes the likelihood function through an EM iterative algorithm and allows consideration of uncertainty of marker allele frequencies in the parental populations <a href="#pone.0001244-Zhu4">[25]</a>. The number of generations since population admixture was estimated by maximizing the likelihood function for the number of generations. To estimate the marker location-specific ancestry, we assumed there were two parental populations for the African-American population. If our model fit the data well, we would expect the estimated allele frequencies in ancestral African and European populations to be close to what is observed in contemporary populations, although not the same. To allow for the uncertainty of ancestral marker allele frequencies, we compared the estimated ancestral allele frequencies and marker location-specific ancestries when no ancestral population information was used to a situation where only observed European allele frequencies were used as the initial European ancestral allele frequencies in the model. The results were essentially the same and we then reported the results using the European allele frequencies. The correlation between the observed and estimated allele frequencies in the European sample was 97.1% and the correlation between estimated African ancestral allele frequencies with that in panel described by Smith et al <a href="#pone.0001244-Smith1">[31]</a> (calculated from the weighted average allele frequencies of Ghana and Cameroon) was 84.5% (<a href="#pone-0001244-g004">figure 4 A and B</a>). The lower correlation for African allele frequencies is apparently due to inconsistent designation of the minor allele. However, this inconsistency has no effect to the results because no African ancestral information was used in our analysis.</p>
<div class="figure" id="pone-0001244-g004"><div class="img"><a name="pone-0001244-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001244" data-uri="info:doi/10.1371/journal.pone.0001244.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001244.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001244.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001244.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001244.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Ancestral allele frequencies.</span></strong></p><a id="article1.body1.sec4.sec5.fig1.caption1.p1" name="article1.body1.sec4.sec5.fig1.caption1.p1"></a><p>A. European ancestral allele frequencies estimated by Hidden Markov Model in the African-American sample vs the observed allele frequencies in European Americans. B. African ancestral allele frequencies estimated by Hidden Markov Model in the African-American sample vs the observed allele frequencies by the weighted average from Ghana and Cameroon obtained from Smith et al. (2004).</p>
<span>doi:10.1371/journal.pone.0001244.g004</span></div>

<h4>Testing for linkage by admixture mapping</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To test for linkage we calculated two test statistics: the Z score of the case-only test and the Z score of the case-control test <a href="#pone.0001244-Zhu3">[21]</a>, <a href="#pone.0001244-Zhu4">[25]</a>. We also estimated the marker location-specific ancestry using the software STRUCTURE <a href="#pone.0001244-Smith1">[31]</a>, <a href="#pone.0001244-Pritchard1">[50]</a>. STRUCTURE was run under the linkage model without haplotype phase information, with 50,000 burn-in iterations followed by an additional 50,000 iterations. We specified SITEBYSITE = 1 and STRUCTURE provided output for the joint posterior assignment probabilities of population origin for the two alleles at each marker location. We next compared the marker-specific ancestry obtained with the software STRUCTURE to the results obtained from the EM method.</p>


<h4>Simulation of admixture populations</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>We directly simulated admixed populations according to the continuous gene flow model <a href="#pone.0001244-Zhu3">[21]</a>, <a href="#pone.0001244-Zhu4">[25]</a>. The ancestral allele frequencies, the number of generations since the occurrence, and average ancestry were based on the estimates in the African American samples. To simulate African-American samples, at the first generation the marker genotypes of 10,000 unrelated people were simulated according to the African allele frequencies assuming HWE and independence of the markers. An admixed population was then formed by taking a proportion of 3.3% randomly selected from the simulated population to mate with people generated according to the marker allele frequencies in the European population, with the remaining individuals randomly mating among themselves. The number of children produced by each marriage was assumed to follow a Poisson distribution with mean size 2. The number of crossovers between two marker loci at a distance d cM was assumed to follow a Poisson distribution with mean d/100. This process was repeated in the following generations to form the current African American populations.</p>


<h4>Testing association between and marker and hypertension</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>We applied a logistic model regressing hypertension status on gender, age, age<sup>2</sup> and the genotype effect assuming additive model. For African Americans, individual ancestry estimated by HMM was included in the model. For Mexican Americans, we applied the genome-control method <a href="#pone.0001244-Devlin1">[51]</a> to control for population stratification by randomly selecting 100 unlinked SNPs. SNPs with significant association were further analyzed using different models of inheritance.</p>


<h4>Permutation test</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>To obtain the significance level of SNP rs2272996 in VNN1 accounting for multiple tests, we performed permutation tests for those SNPs with allele frequency difference between African Americans and Europeans greater than that of SNP rs2272996. We permutated the hypertension status together with gender, age, BMI and African ancestry 10,000 time and analyzed the individuals whose African ancestral&gt;80% at each permutation. Logistic regression was then performed for each SNP and the minimum P-value was recorded at each permutation. These minimum P-values were tallied to obtain the null distribution of the test adjusting for multiple tests.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001244"></div><a name="pone.0001244.s001" id="pone.0001244.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001244.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Individual European Ancestral estimated by ADMIXPROGRAM and STRUCTURE</p>
<p class="siDoi">doi:10.1371/journal.pone.0001244.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.14 MB XLS)</p>
<a name="pone.0001244.s002" id="pone.0001244.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001244.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Case-only and case-control Z-scores across the genome</p>
<p class="siDoi">doi:10.1371/journal.pone.0001244.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.46 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Helen Hobbs and Jonathan Cohen for their generous support with access to the clinical samples and in providing the genotype data. We also thank Dr. Hua Tang for helpful discussion and D. Kan for his assistance in programming.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RC XZ. Analyzed the data: XZ. Wrote the paper: RC XZ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001244-Zhu1" id="pone.0001244-Zhu1"></a>Zhu X, Luke A, Cooper RS, Quertermous T, Hanis C, et al.  (2005) Admixture mapping for hypertension loci with genome-scan markers. Nat Genet  37: 177–181.  <ul class="find" data-citedArticleID="1024228" data-doi="10.1038/ng1510"><li><a href="http://dx.doi.org/10.1038/ng1510" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Admixture+mapping+for+hypertension+loci+with+genome-scan+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Admixture+mapping+for+hypertension+loci+with+genome-scan+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001244-Cooper1" id="pone.0001244-Cooper1"></a>Cooper RS, Liao Y, Rotimi C (1996) Is hypertension more severe among U.S. blacks, or is severe hypertension more common? Ann Epidemiol  6: 173–180.  <ul class="find" data-citedArticleID="1024142" data-doi="10.1016/1047-2797(96)00009-9"><li><a href="http://dx.doi.org/10.1016/1047-2797(96)00009-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+hypertension+more+severe+among+U.S.+blacks%2C+or+is+severe+hypertension+more+common%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+hypertension+more+severe+among+U.S.+blacks%2C+or+is+severe+hypertension+more+common%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001244-Ward1" id="pone.0001244-Ward1"></a>Ward R (1990)  Familial aggregation and genetic epidemiology of blood pressure. In: Laragh JH, Brenner BM, editors. Hypertension: Pathophysiology, Diagnosis, and Management. New York, NY: Raven Press. pp. 81–100.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001244-Samani1" id="pone.0001244-Samani1"></a>Samani NJ (2003) Genome scans for hypertension and blood pressure regulation. Am J Hypertens  16: 167–171.  <ul class="find" data-citedArticleID="1024196"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+scans+for+hypertension+and+blood+pressure+regulation.&amp;auth=&amp;atitle=Genome+scans+for+hypertension+and+blood+pressure+regulation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+scans+for+hypertension+and+blood+pressure+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+scans+for+hypertension+and+blood+pressure+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001244-Risch1" id="pone.0001244-Risch1"></a>Risch NJ (2000) Searching for genetic determinants in the new millennium. Nature  405: 847–856.  <ul class="find" data-citedArticleID="1024194" data-doi="10.1038/35015718"><li><a href="http://dx.doi.org/10.1038/35015718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+for+genetic+determinants+in+the+new+millennium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+for+genetic+determinants+in+the+new+millennium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001244-Lohmueller1" id="pone.0001244-Lohmueller1"></a>Lohmueller KE, Pearce CL, Pike M, Lander ES, Hirschhorn JN (2003) Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease. Nat Genet  33: 177–182.  <ul class="find" data-citedArticleID="1024174" data-doi="10.1038/ng1071"><li><a href="http://dx.doi.org/10.1038/ng1071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001244-Zhu2" id="pone.0001244-Zhu2"></a>Zhu X, Bouzekri N, Southam L, Cooper RS, Adeyemo A, et al.  (2001) Linkage and association analysis of angiotensin I-converting enzyme (ACE)-gene polymorphisms with ACE concentration and blood pressure. Am J Hum Genet  68: 1139–1148.  <ul class="find" data-citedArticleID="1024230" data-doi="10.1086/320104"><li><a href="http://dx.doi.org/10.1086/320104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+and+association+analysis+of+angiotensin+I-converting+enzyme+%28ACE%29-gene+polymorphisms+with+ACE+concentration+and+blood+pressure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+and+association+analysis+of+angiotensin+I-converting+enzyme+%28ACE%29-gene+polymorphisms+with+ACE+concentration+and+blood+pressure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001244-Bouzekri1" id="pone.0001244-Bouzekri1"></a>Bouzekri N, Zhu X, Jiang Y, McKenzie CA, Luke A, et al.  (2004) Angiotensin I-converting enzyme polymorphisms, ACE level and blood pressure among Nigerians, Jamaicans and African-Americans. Eur J Hum Genet  12: 460–468.  <ul class="find" data-citedArticleID="1024140" data-doi="10.1038/sj.ejhg.5201166"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Angiotensin+I-converting+enzyme+polymorphisms%2C+ACE+level+and+blood+pressure+among+Nigerians%2C+Jamaicans+and+African-Americans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Angiotensin+I-converting+enzyme+polymorphisms%2C+ACE+level+and+blood+pressure+among+Nigerians%2C+Jamaicans+and+African-Americans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001244-Hinds1" id="pone.0001244-Hinds1"></a>Hinds DA, Stuve LL, Nilsen GB, Halperin E, Eskin E, et al.  (2005) Whole-genome patterns of common DNA variation in three human populations. Science  307: 1072–1079.  <ul class="find" data-citedArticleID="1024166" data-doi="10.1126/science.1105436"><li><a href="http://dx.doi.org/10.1126/science.1105436" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+patterns+of+common+DNA+variation+in+three+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+patterns+of+common+DNA+variation+in+three+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001244-The1" id="pone.0001244-The1"></a>The International HapMap Consortium (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="1024210" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001244-Klein1" id="pone.0001244-Klein1"></a>Klein RJ, Zeiss C, Chew EY, Tsai JY, Sackler RS, et al.  (2005) Complement factor H polymorphism in age-related macular degeneration. Science  308: 385–389.  <ul class="find" data-citedArticleID="1024172" data-doi="10.1126/science.1109557"><li><a href="http://dx.doi.org/10.1126/science.1109557" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complement+factor+H+polymorphism+in+age-related+macular+degeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complement+factor+H+polymorphism+in+age-related+macular+degeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001244-Herbert1" id="pone.0001244-Herbert1"></a>Herbert A, Gerry NP, McQueen MB, Heid IM, Pfeufer A, et al.  (2006) A common genetic variant is associated with adult and childhood obesity. Science.  312: 279–283.  <ul class="find" data-citedArticleID="1024164" data-doi="10.1126/science.1124779"><li><a href="http://dx.doi.org/10.1126/science.1124779" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+genetic+variant+is+associated+with+adult+and+childhood+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+genetic+variant+is+associated+with+adult+and+childhood+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001244-Smyth1" id="pone.0001244-Smyth1"></a>Smyth DJ, Cooper JD, Bailey R, Field S, Burren O, et al.  (2006) A genome-wide association study of nonsynonymous SNPs identifies a type 1 diabetes locus in the interferon-induced helicase (IFIH1) region. Nat Genet  38: 617–619.  <ul class="find" data-citedArticleID="1024204" data-doi="10.1038/ng1800"><li><a href="http://dx.doi.org/10.1038/ng1800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+association+study+of+nonsynonymous+SNPs+identifies+a+type+1+diabetes+locus+in+the+interferon-induced+helicase+%28IFIH1%29+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+association+study+of+nonsynonymous+SNPs+identifies+a+type+1+diabetes+locus+in+the+interferon-induced+helicase+%28IFIH1%29+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001244-Scott1" id="pone.0001244-Scott1"></a>Scott LJ, Mohlke KL, Bonnycastle LL, Willer CJ, Li Y, et al.  (2007) A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. Science  316: 1341–1345.  <ul class="find" data-citedArticleID="1024198" data-doi="10.1126/science.1142382"><li><a href="http://dx.doi.org/10.1126/science.1142382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001244-Frayling1" id="pone.0001244-Frayling1"></a>Frayling TM, Timpson NJ, Weedon MN, Zeggini E, Freathy RM, et al.  (2007) A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. Science  316: 889–894.  <ul class="find" data-citedArticleID="1024154" data-doi="10.1126/science.1141634"><li><a href="http://dx.doi.org/10.1126/science.1141634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001244-Diabetes1" id="pone.0001244-Diabetes1"></a>Saxena R, Voight BF, et al. Diabetes Genetics Initiative of Broad Institute of Harvard and MIT, Lund University, and Novartis Institutes of BioMedical Research (2007) Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. Science  316: 1331–1336.  <ul class="find" data-citedArticleID="1024150" data-doi="10.1126/science.1142358"><li><a href="http://dx.doi.org/10.1126/science.1142358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001244-Zanke1" id="pone.0001244-Zanke1"></a>Zanke BW, Greenwood CM, Rangrej J, Kustra R, Tenesa A, et al.  (2007) Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes. Nat Genet  39: 977–983.  <ul class="find" data-citedArticleID="1024224"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Two+variants+on+chromosome+17+confer+prostate+cancer+risk%2C+and+the+one+in+TCF2+protects+against+type+2+diabetes.&amp;auth=&amp;atitle=Two+variants+on+chromosome+17+confer+prostate+cancer+risk%2C+and+the+one+in+TCF2+protects+against+type+2+diabetes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+variants+on+chromosome+17+confer+prostate+cancer+risk%2C+and+the+one+in+TCF2+protects+against+type+2+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+variants+on+chromosome+17+confer+prostate+cancer+risk%2C+and+the+one+in+TCF2+protects+against+type+2+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001244-Gudmundsson1" id="pone.0001244-Gudmundsson1"></a>Gudmundsson J, Sulem P, Steinthorsdottir V, Bergthorsson JT, Thorleifsson G, et al.  (2007) Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24. Nat Genet  39: 989–994.  <ul class="find" data-citedArticleID="1024160"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24.&amp;auth=&amp;atitle=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001244-International1" id="pone.0001244-International1"></a>International Multiple Sclerosis Genetics Consortium (2007) Risk alleles for multiple sclerosis identified by a genomewide study. N Engl J Med  357: 851–862.  <ul class="find" data-citedArticleID="1024170" data-doi="10.1056/nejmoa073493"><li><a href="http://dx.doi.org/10.1056/nejmoa073493" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Risk+alleles+for+multiple+sclerosis+identified+by+a+genomewide+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001244-Montana1" id="pone.0001244-Montana1"></a>Montana G, Pritchard JK (2004) Statistical tests for admixture mapping with case-control and cases-only data. Am J Hum Genet  75: 771–789.  <ul class="find" data-citedArticleID="1024178" data-doi="10.1086/425281"><li><a href="http://dx.doi.org/10.1086/425281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+tests+for+admixture+mapping+with+case-control+and+cases-only+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+tests+for+admixture+mapping+with+case-control+and+cases-only+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001244-Zhu3" id="pone.0001244-Zhu3"></a>Zhu X, Cooper RS, Elston RC (2004) Linkage analysis of a complex disease through use of admixed populations. Am J Hum Genet  74: 1136–1153.  <ul class="find" data-citedArticleID="1024232" data-doi="10.1086/421329"><li><a href="http://dx.doi.org/10.1086/421329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+analysis+of+a+complex+disease+through+use+of+admixed+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+analysis+of+a+complex+disease+through+use+of+admixed+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001244-Patterson1" id="pone.0001244-Patterson1"></a>Patterson N, Hattangadi N, Lane B, Lohmueller KE, Hafler DA, et al.  (2004) Methods for high-density admixture mapping of disease genes. Am J Hum Genet  74: 979–1000.  <ul class="find" data-citedArticleID="1024186" data-doi="10.1086/420871"><li><a href="http://dx.doi.org/10.1086/420871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methods+for+high-density+admixture+mapping+of+disease+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methods+for+high-density+admixture+mapping+of+disease+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001244-Hoggart1" id="pone.0001244-Hoggart1"></a>Hoggart CJ, Shriver MD, Kittles RA, Clayton DG, McKeigue PM (2004) Design and analysis of admixture mapping studies. Am J Hum Genet  74: 965–978.  <ul class="find" data-citedArticleID="1024168" data-doi="10.1086/420855"><li><a href="http://dx.doi.org/10.1086/420855" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design+and+analysis+of+admixture+mapping+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design+and+analysis+of+admixture+mapping+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001244-McKeigue1" id="pone.0001244-McKeigue1"></a>McKeigue PM (1998) Mapping genes that underlie ethnic differences in disease risk: methods for detecting linkage in admixed populations, by conditioning on parental admixture. Am J Hum Genet  63: 241–251.  <ul class="find" data-citedArticleID="1024176" data-doi="10.1086/301908"><li><a href="http://dx.doi.org/10.1086/301908" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+genes+that+underlie+ethnic+differences+in+disease+risk%3A+methods+for+detecting+linkage+in+admixed+populations%2C+by+conditioning+on+parental+admixture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+genes+that+underlie+ethnic+differences+in+disease+risk%3A+methods+for+detecting+linkage+in+admixed+populations%2C+by+conditioning+on+parental+admixture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001244-Zhu4" id="pone.0001244-Zhu4"></a>Zhu X, Zhang SL, Tang H, Cooper RS (2006) A Classical Likelihood Based Admixture Mapping Method and Comparison with the Bayesian Approach. Hum Genet  120: 431–445.  <ul class="find" data-citedArticleID="1024234" data-doi="10.1007/s00439-006-0224-z"><li><a href="http://dx.doi.org/10.1007/s00439-006-0224-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Classical+Likelihood+Based+Admixture+Mapping+Method+and+Comparison+with+the+Bayesian+Approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Classical+Likelihood+Based+Admixture+Mapping+Method+and+Comparison+with+the+Bayesian+Approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001244-Tang1" id="pone.0001244-Tang1"></a>Tang H, Coram M, Wang P, Zhu X, Risch N (2006) Reconstructing Genetic Ancestry Blocks in Admixed Individuals. Am J Hum Genet  79: 1–12.  <ul class="find" data-citedArticleID="1024206" data-doi="10.1086/504302"><li><a href="http://dx.doi.org/10.1086/504302" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstructing+Genetic+Ancestry+Blocks+in+Admixed+Individuals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstructing+Genetic+Ancestry+Blocks+in+Admixed+Individuals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001244-Zhang1" id="pone.0001244-Zhang1"></a>Zhang C, Chen K, Seldin MF, Li H (2004) A hidden Markov Modeling approach for admixture mapping based on case-control data. Genet Epidemiol  27: 225–239.  <ul class="find" data-citedArticleID="1024226" data-doi="10.1002/gepi.20021"><li><a href="http://dx.doi.org/10.1002/gepi.20021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hidden+Markov+Modeling+approach+for+admixture+mapping+based+on+case-control+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hidden+Markov+Modeling+approach+for+admixture+mapping+based+on+case-control+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001244-Halder1" id="pone.0001244-Halder1"></a>Halder I, Shriver MD (2003) Measureing and using admixture to study the genetics of complex diseases. Hum Genomics  1: 52–62.  <ul class="find" data-citedArticleID="1024162" data-doi="10.1186/1479-7364-1-1-52"><li><a href="http://dx.doi.org/10.1186/1479-7364-1-1-52" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Measureing+and+using+admixture+to+study+the+genetics+of+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Measureing+and+using+admixture+to+study+the+genetics+of+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001244-Nievergelt1" id="pone.0001244-Nievergelt1"></a>Nievergelt CM, Schork NJ (2005) Admixture mapping as a discovery approach for complex human traits and diseases. Curr Hypertens Rep  7: 31–37.  <ul class="find" data-citedArticleID="1024182" data-doi="10.1007/s11906-005-0052-x"><li><a href="http://dx.doi.org/10.1007/s11906-005-0052-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Admixture+mapping+as+a+discovery+approach+for+complex+human+traits+and+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Admixture+mapping+as+a+discovery+approach+for+complex+human+traits+and+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001244-Darvasi1" id="pone.0001244-Darvasi1"></a>Darvasi A, Shifman S (2005) The beauty of admixture. Nat Genet  37: 118–119.  <ul class="find" data-citedArticleID="1024146" data-doi="10.1038/ng0205-118"><li><a href="http://dx.doi.org/10.1038/ng0205-118" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+beauty+of+admixture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+beauty+of+admixture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001244-Smith1" id="pone.0001244-Smith1"></a>Smith MW, Patterson N, Lautenberger JA, Truelove AL, McDonald GJ, et al.  (2004) A high-density admixture map for disease gene discovery in african americans. Am J Hum Genet  74: 1001–1013.  <ul class="find" data-citedArticleID="1024200" data-doi="10.1086/420856"><li><a href="http://dx.doi.org/10.1086/420856" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+high-density+admixture+map+for+disease+gene+discovery+in+african+americans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+high-density+admixture+map+for+disease+gene+discovery+in+african+americans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001244-Reich1" id="pone.0001244-Reich1"></a>Reich D, Patterson N, De Jager PL, McDonald GJ, Waliszewska A, et al.  (2005) A whole-genome admixture scan finds a candidate locus for multiple sclerosis susceptibility. Nat Genet  37: 1113–1118.  <ul class="find" data-citedArticleID="1024192" data-doi="10.1038/ng1646"><li><a href="http://dx.doi.org/10.1038/ng1646" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+whole-genome+admixture+scan+finds+a+candidate+locus+for+multiple+sclerosis+susceptibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+whole-genome+admixture+scan+finds+a+candidate+locus+for+multiple+sclerosis+susceptibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001244-Botstein1" id="pone.0001244-Botstein1"></a>Botstein D, Risch N (2003) Discovering genotypes underlying human phenotypes: past successes for mendelian disease, future approaches for complex disease. Nat Genet 33 Suppl 228–237.  <ul class="find" data-citedArticleID="1024138" data-doi="10.1038/ng1090"><li><a href="http://dx.doi.org/10.1038/ng1090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovering+genotypes+underlying+human+phenotypes%3A+past+successes+for+mendelian+disease%2C+future+approaches+for+complex+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovering+genotypes+underlying+human+phenotypes%3A+past+successes+for+mendelian+disease%2C+future+approaches+for+complex+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001244-Amundadottir1" id="pone.0001244-Amundadottir1"></a>Amundadottir LT, Sulem P, Gudmundsson J, Helgason A, Baker A, et al.  (2006) A common variant associated with prostate cancer in European and African populations. Nat Genet  38: 652–658.  <ul class="find" data-citedArticleID="1024134" data-doi="10.1038/ng1808"><li><a href="http://dx.doi.org/10.1038/ng1808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+variant+associated+with+prostate+cancer+in+European+and+African+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+variant+associated+with+prostate+cancer+in+European+and+African+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001244-Freedman1" id="pone.0001244-Freedman1"></a>Freedman ML, Haiman CA, Patterson N, McDonald GJ, Tandon A, et al.  (2006) Admixture mapping identifies 8q24 as a prostate cancer risk locus in African-American men. Proc Natl Acad Sci U S A  103: 14068–14073.  <ul class="find" data-citedArticleID="1024156" data-doi="10.1073/pnas.0605832103"><li><a href="http://dx.doi.org/10.1073/pnas.0605832103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Admixture+mapping+identifies+8q24+as+a+prostate+cancer+risk+locus+in+African-American+men." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Admixture+mapping+identifies+8q24+as+a+prostate+cancer+risk+locus+in+African-American+men.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001244-Smith2" id="pone.0001244-Smith2"></a>Smith MW, O'Brien SJ (2005) Mapping by admixture linkage disequilibrium: advances, limitations and guidelines. Nat Rev Genet  6: 623–632.  <ul class="find" data-citedArticleID="1024202" data-doi="10.1038/nrg1657"><li><a href="http://dx.doi.org/10.1038/nrg1657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+by+admixture+linkage+disequilibrium%3A+advances%2C+limitations+and+guidelines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+by+admixture+linkage+disequilibrium%3A+advances%2C+limitations+and+guidelines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001244-Falush1" id="pone.0001244-Falush1"></a>Falush D, Stephens M, Pritchard JK (2003) Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics  164: 1567–1587.  <ul class="find" data-citedArticleID="1024152"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data%3A+linked+loci+and+correlated+allele+frequencies.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data%3A+linked+loci+and+correlated+allele+frequencies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data%3A+linked+loci+and+correlated+allele+frequencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data%3A+linked+loci+and+correlated+allele+frequencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001244-Tian1" id="pone.0001244-Tian1"></a>Tian C, Hinds DA, Shigeta R, Kittles R, Ballinger DG, et al.  (2006) A genomewide single-nucleotide-polymorphism panel with hihg ancestry information for African American admixture mapping. Am J Hum Genet  79: 640–649.  <ul class="find" data-citedArticleID="1024214" data-doi="10.1086/507954"><li><a href="http://dx.doi.org/10.1086/507954" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomewide+single-nucleotide-polymorphism+panel+with+hihg+ancestry+information+for+African+American+admixture+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomewide+single-nucleotide-polymorphism+panel+with+hihg+ancestry+information+for+African+American+admixture+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001244-Tang2" id="pone.0001244-Tang2"></a>Tang H (2006) Confronting ethnicity-specific disease risk. Nat Genet  38: 13–15.  <ul class="find" data-citedArticleID="1024208" data-doi="10.1038/ng0106-13"><li><a href="http://dx.doi.org/10.1038/ng0106-13" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Confronting+ethnicity-specific+disease+risk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Confronting+ethnicity-specific+disease+risk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001244-Nakajima1" id="pone.0001244-Nakajima1"></a>Nakajima T, Wooding S, Sakagami T, Emi M, Tokunaga K, et al.  (2004) Natural selection and population history in the human angiotensinogen gene (AGT): 736 complete AGT sequences in chromosomes from around the world. Am J Hum Genet  74: 898–916.  <ul class="find" data-citedArticleID="1024180" data-doi="10.1086/420793"><li><a href="http://dx.doi.org/10.1086/420793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+population+history+in+the+human+angiotensinogen+gene+%28AGT%29%3A+736+complete+AGT+sequences+in+chromosomes+from+around+the+world.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001244-Thompson1" id="pone.0001244-Thompson1"></a>Thompson EE, Kuttab-Boulos H, Witonsky D, Yang L, Roe BA, et al.  (2004) variation and the evolution of salt-sensitivity variants. Am J Hum Genet  75: 1059–1069.  <ul class="find" data-citedArticleID="1024212" data-doi="10.1086/426406"><li><a href="http://dx.doi.org/10.1086/426406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=variation+and+the+evolution+of+salt-sensitivity+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22variation+and+the+evolution+of+salt-sensitivity+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001244-Young1" id="pone.0001244-Young1"></a>Young JH, Chang YP, Kim JD, Chretien JP, Klag MJ, et al.  (2005) A Differential Susceptibility to Hypertension Is Due to Selection during the Out-of-Africa Expansion. PLoS Genet  1: e82.  <ul class="find" data-citedArticleID="1024222" data-doi="10.1371/journal.pgen.0010082.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010082.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Differential+Susceptibility+to+Hypertension+Is+Due+to+Selection+during+the+Out-of-Africa+Expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Differential+Susceptibility+to+Hypertension+Is+Due+to+Selection+during+the+Out-of-Africa+Expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001244-Voight1" id="pone.0001244-Voight1"></a>Voight BF, Kudaravalli S, Wen X, Pritchard JK (2006) A map of recent positive selection in the human genome. PLoS Biol  4: e154.  <ul class="find" data-citedArticleID="1024218" data-doi="10.1371/journal.pbio.0040154"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+map+of+recent+positive+selection+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+map+of+recent+positive+selection+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001244-Pitari1" id="pone.0001244-Pitari1"></a>Pitari G, Malergue F, Martin F, Philippe JM, Massucci MT, et al.  (2000) Pantetheinase activity of membrane-bound vanin-1: lack of free cysteamine in tissues of vanin-1 deficient mice. FEBS Lett.  483: 149–154.  <ul class="find" data-citedArticleID="1024188" data-doi="10.1016/s0014-5793(00)02110-4"><li><a href="http://dx.doi.org/10.1016/s0014-5793(00)02110-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pantetheinase+activity+of+membrane-bound+vanin-1%3A+lack+of+free+cysteamine+in+tissues+of+vanin-1+deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pantetheinase+activity+of+membrane-bound+vanin-1%3A+lack+of+free+cysteamine+in+tissues+of+vanin-1+deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001244-Oparil1" id="pone.0001244-Oparil1"></a>Oparil S, Zaman MA, Calhoun DA (2003) Pathogenesis of hypertension. Ann Intern Med  139: 761–776.  <ul class="find" data-citedArticleID="1024184" data-doi="10.7326/0003-4819-139-9-200311040-00011"><li><a href="http://dx.doi.org/10.7326/0003-4819-139-9-200311040-00011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathogenesis+of+hypertension." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathogenesis+of+hypertension.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001244-Goring1" id="pone.0001244-Goring1"></a>Goring HH, Curran JE, Johnson MP, Dyer TD, Charlesworth J, et al.  (2007) Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes. Nat Genet.  39: 1208–16.  <ul class="find" data-citedArticleID="1024158" data-doi="10.1038/ng2119"><li><a href="http://dx.doi.org/10.1038/ng2119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+of+expression+QTLs+using+large-scale+transcriptional+profiling+in+human+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+of+expression+QTLs+using+large-scale+transcriptional+profiling+in+human+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001244-Curran1" id="pone.0001244-Curran1"></a>Curran JE, Johnson MP, Goring HH, Dyer TD, Charlesworth JC, et al.  (2007) VNN1, A novel gene for cardiovascular disease risk. Am J Hum Genet. 2007. Abstract. 62.  <ul class="find" data-citedArticleID="1024144"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=VNN1%2C+A+novel+gene+for+cardiovascular+disease+risk.&amp;auth=&amp;atitle=VNN1%2C+A+novel+gene+for+cardiovascular+disease+risk." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=VNN1%2C+A+novel+gene+for+cardiovascular+disease+risk." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22VNN1%2C+A+novel+gene+for+cardiovascular+disease+risk.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001244-Victor1" id="pone.0001244-Victor1"></a>Victor RG, Haley RW, Willett DL, Peshock RM, Vaeth PC, et al.  (2004) The Dallas Heart Study: a population-based probability sample for the multidisciplinary study of ethnic differences in cardiovascular health. Am J Cardiol  93: 1473–1480.  <ul class="find" data-citedArticleID="1024216" data-doi="10.1016/j.amjcard.2004.02.058"><li><a href="http://dx.doi.org/10.1016/j.amjcard.2004.02.058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Dallas+Heart+Study%3A+a+population-based+probability+sample+for+the+multidisciplinary+study+of+ethnic+differences+in+cardiovascular+health." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Dallas+Heart+Study%3A+a+population-based+probability+sample+for+the+multidisciplinary+study+of+ethnic+differences+in+cardiovascular+health.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001244-Barrett1" id="pone.0001244-Barrett1"></a>Barrett JC, Fry B, Maller J, Daly MJ (2005) Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics  21: 263–265.  <ul class="find" data-citedArticleID="1024136" data-doi="10.1093/bioinformatics/bth457"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth457" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haploview%3A+analysis+and+visualization+of+LD+and+haplotype+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haploview%3A+analysis+and+visualization+of+LD+and+haplotype+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001244-Pritchard1" id="pone.0001244-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="1024190"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001244-Devlin1" id="pone.0001244-Devlin1"></a>Devlin B, Roeder K (1999) Genomic control for association studies. Biometrics  55: 997–1004.  <ul class="find" data-citedArticleID="1024148" data-doi="10.1111/j.0006-341x.1999.00997.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.1999.00997.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+control+for+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+control+for+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.XML" value="109507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.PDF" value="437505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t001.PNG_L" value="85798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t001.PNG_M" value="51722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t001.PNG_S" value="9193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t001.TIF" value="377616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t001.PNG_I" value="15973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g001.PNG_L" value="224342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g001.PNG_M" value="79740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g001.PNG_S" value="8962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g001.TIF" value="606794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g001.PNG_I" value="37323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t002.PNG_L" value="106288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t002.PNG_M" value="59290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t002.PNG_S" value="9578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t002.TIF" value="479558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t002.PNG_I" value="18386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t003.PNG_L" value="50993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t003.PNG_M" value="63812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t003.PNG_S" value="11597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t003.TIF" value="253354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t003.PNG_I" value="20711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g002.PNG_L" value="219215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g002.PNG_M" value="46776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g002.PNG_S" value="7111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g002.TIF" value="666060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g002.PNG_I" value="16992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g003.PNG_L" value="55402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g003.PNG_M" value="40794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g003.PNG_S" value="8031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g003.TIF" value="444126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g003.PNG_I" value="14764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t004.PNG_L" value="70521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t004.PNG_M" value="41035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t004.PNG_S" value="8317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t004.TIF" value="305832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t004.PNG_I" value="12412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t005.PNG_L" value="62757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t005.PNG_M" value="35339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t005.PNG_S" value="8257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t005.TIF" value="297980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.t005.PNG_I" value="10624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g004.PNG_L" value="299859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g004.PNG_M" value="53172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g004.PNG_S" value="7118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g004.TIF" value="688976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.g004.PNG_I" value="18059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.s001.XLS" value="139264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001244.s002.XLS" value="461312"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001244&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001244" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001244&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001244&volume=&issue=&title=Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension&author_name=Xiaofeng%20Zhu%2C%20Richard%20S.%20Cooper&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001244" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244&amp;t=Admixture%20Mapping%20Provides%20Evidence%20of%20Association%20of%20the%20VNN1%20Gene%20with%20Hypertension" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244&title=Admixture%20Mapping%20Provides%20Evidence%20of%20Association%20of%20the%20VNN1%20Gene%20with%20Hypertension&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244&amp;title=Admixture%20Mapping%20Provides%20Evidence%20of%20Association%20of%20the%20VNN1%20Gene%20with%20Hypertension" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001244&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Admixture Mapping Provides Evidence of Association of the VNN1 Gene with Hypertension';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001244';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Admixture%20Mapping%20Provides%20Evidence%20of%20Association%20of%20the%20VNN1%20Gene%20with%20Hypertension http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001244" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001244" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22African+Americans%22" title="Search for articles in the subject area:'African Americans'"><div class="flagText">African Americans</div></a>
              <div data-categoryid="40667" data-articleid="25320"
                   data-categoryname="African Americans"
                   class="flagImage" title="Flag 'African Americans' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+6%22" title="Search for articles in the subject area:'Chromosome 6'"><div class="flagText">Chromosome 6</div></a>
              <div data-categoryid="20871" data-articleid="25320"
                   data-categoryname="Chromosome 6"
                   class="flagImage" title="Flag 'Chromosome 6' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+mapping%22" title="Search for articles in the subject area:'Chromosome mapping'"><div class="flagText">Chromosome mapping</div></a>
              <div data-categoryid="19841" data-articleid="25320"
                   data-categoryname="Chromosome mapping"
                   class="flagImage" title="Flag 'Chromosome mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="25320"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25320"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hidden+Markov+models%22" title="Search for articles in the subject area:'Hidden Markov models'"><div class="flagText">Hidden Markov models</div></a>
              <div data-categoryid="33299" data-articleid="25320"
                   data-categoryname="Hidden Markov models"
                   class="flagImage" title="Flag 'Hidden Markov models' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hypertension%22" title="Search for articles in the subject area:'Hypertension'"><div class="flagText">Hypertension</div></a>
              <div data-categoryid="21783" data-articleid="25320"
                   data-categoryname="Hypertension"
                   class="flagImage" title="Flag 'Hypertension' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="25320"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1835'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4839'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7273&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
