

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000915"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000915" />

    <meta name="citation_title" content="Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications"/>
    <meta itemprop="name" content="Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications"/>

      <meta name="citation_author" content="Michael S. Akhras"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden"/>
      <meta name="citation_author" content="Magnus Unemo"/>
            <meta name="citation_author_institution" content="National Reference Laboratory for Pathogenic Neisseria, Department of Clinical Microbiology, Örebro University Hospital, Örebro, Sweden"/>
      <meta name="citation_author" content="Sreedevi Thiyagarajan"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Pål Nyrén"/>
            <meta name="citation_author_institution" content="Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden"/>
      <meta name="citation_author" content="Ronald W. Davis"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Andrew Z. Fire"/>
            <meta name="citation_author_institution" content="Department of Pathology, Stanford University School of Medicine, Stanford, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, Stanford, California, United States of America"/>
      <meta name="citation_author" content="Nader Pourmand"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
            <meta name="citation_author_institution" content="Biomolecular Engineering, University of California at Santa Cruz, Santa Cruz, California, United States of America"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000915.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e915"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia.; citation_author=RK Saiki; citation_author=S Scharf; citation_author=F Faloona; citation_author=KB Mullis; citation_author=GT Horn; citation_journal_title=Science; citation_volume=230; citation_number=1; citation_pages=1350-1354; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Present and future of rapid and/or high-throughput methods for nucleic acid testing.; citation_author=G Csako; citation_journal_title=Clin Chim Acta; citation_volume=363; citation_number=2; citation_pages=6-31; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Deletion screening of the Duchenne muscular dystrophy locus via multiplex DNA amplification.; citation_author=JS Chamberlain; citation_author=RA Gibbs; citation_author=JE Ranier; citation_author=PN Nguyen; citation_author=CT Caskey; citation_journal_title=Nucleic Acids Res; citation_volume=16; citation_number=3; citation_pages=11141-11156; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Multiplex PCR: optimization and application in diagnostic virology.; citation_author=EM Elnifro; citation_author=AM Ashshi; citation_author=RJ Cooper; citation_author=PE Klapper; citation_journal_title=Clin Microbiol Rev; citation_volume=13; citation_number=4; citation_pages=559-570; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Analyzing genes using closing and replicating circles.; citation_author=M Nilsson; citation_author=F Dahl; citation_author=C Larsson; citation_author=M Gullberg; citation_author=J Stenberg; citation_journal_title=Trends Biotechnol; citation_volume=24; citation_number=5; citation_pages=83-88; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector.; citation_author=S Fredriksson; citation_author=J Baner; citation_author=F Dahl; citation_author=A Chu; citation_author=H Ji; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=6; citation_pages=e47; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Multigene amplification and massively parallel sequencing for cancer mutation discovery.; citation_author=F Dahl; citation_author=J Stenberg; citation_author=S Fredriksson; citation_author=K Welch; citation_author=M Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_number=7; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic disease detection and DNA amplification using cloned thermostable ligase.; citation_author=F Barany; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=88; citation_number=8; citation_pages=189-193; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Detection of point mutations with a modified ligase chain reaction (Gap-LCR).; citation_author=K Abravaya; citation_author=JJ Carrino; citation_author=S Muldoon; citation_author=HH Lee; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=9; citation_pages=675-682; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mechanism of DNA chain growth. I. Possible discontinuity and unusual secondary structure of newly synthesized chains.; citation_author=R Okazaki; citation_author=T Okazaki; citation_author=K Sakabe; citation_author=K Sugimoto; citation_author=A Sugino; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=59; citation_number=10; citation_pages=598-605; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Padlock probes: circularizing oligonucleotides for localized DNA detection.; citation_author=M Nilsson; citation_author=H Malmgren; citation_author=M Samiotaki; citation_author=M Kwiatkowski; citation_author=BP Chowdhary; citation_journal_title=Science; citation_volume=265; citation_number=11; citation_pages=2085-2088; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Rolling replication of short DNA circles.; citation_author=A Fire; citation_author=SQ Xu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=12; citation_pages=4641-4645; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mutation detection and single-molecule counting using isothermal rolling-circle amplification.; citation_author=PM Lizardi; citation_author=X Huang; citation_author=Z Zhu; citation_author=P Bray-Ward; citation_author=DC Thomas; citation_journal_title=Nat Genet; citation_volume=19; citation_number=13; citation_pages=225-232; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Multiplexed genotyping with sequence-tagged molecular inversion probes.; citation_author=P Hardenbol; citation_author=J Baner; citation_author=M Jain; citation_author=M Nilsson; citation_author=EA Namsaraev; citation_journal_title=Nat Biotechnol; citation_volume=21; citation_number=14; citation_pages=673-678; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Highly multiplexed molecular inversion probe genotyping: over 10,000 targeted SNPs genotyped in a single tube assay.; citation_author=P Hardenbol; citation_author=F Yu; citation_author=J Belmont; citation_author=J Mackenzie; citation_author=C Bruckner; citation_journal_title=Genome Res; citation_volume=15; citation_number=15; citation_pages=269-275; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Allele quantification using molecular inversion probes (MIP).; citation_author=Y Wang; citation_author=M Moorhead; citation_author=G Karlin-Neumann; citation_author=M Falkowski; citation_author=C Chen; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=16; citation_pages=e183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=PathogenMip Assay: A Multiplex Pathogen Detection Assay.; citation_author=MS Akhras; citation_author=S Thiyagarajan; citation_author=AC Villablanca; citation_author=RW Davis; citation_author=P Nyrén; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=17; citation_pages=e223; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=PathogenMIPer: a tool for the design of molecular inversion probes to detect multiple pathogens.; citation_author=S Thiyagarajan; citation_author=M Karhanek; citation_author=M Akhras; citation_author=RW Davis; citation_author=N Pourmand; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=18; citation_pages=500; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Classification of papillomaviruses.; citation_author=EM de Villiers; citation_author=C Fauquet; citation_author=TR Broker; citation_author=HU Bernard; citation_author=H zur Hausen; citation_journal_title=Virology; citation_volume=324; citation_number=19; citation_pages=17-27; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of human papillomavirus infection and cervical cancer.; citation_author=J Doorbar; citation_journal_title=Clin Sci (Lond); citation_volume=110; citation_number=20; citation_pages=525-541; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA gyrase gene in Neisseria gonorrhoeae as indicator for resistance to ciprofloxacin and species verification.; citation_author=E Lindbäck; citation_author=B Gharizadeh; citation_author=F Ataker; citation_author=A Airell; citation_author=S Jalal; citation_journal_title=Int J STD AIDS; citation_volume=16; citation_number=21; citation_pages=142-147; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pyrosequencing of the DNA gyrase gene in Neisseria species: effective indicator of ciprofloxacin resistance in Neisseria gonorrhoeae.; citation_author=E Lindbäck; citation_author=M Unemo; citation_author=M Akhras; citation_author=B Gharizadeh; citation_author=H Fredlund; citation_journal_title=Apmis; citation_volume=114; citation_number=22; citation_pages=837-841; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of Neisseria gonorrhoeae identifies transmission and resistance of one ciprofloxacin-resistant strain.; citation_author=M Unemo; citation_author=A Sjöstrand; citation_author=M Akhras; citation_author=B Gharizadeh; citation_author=E Lindback; citation_journal_title=Apmis; citation_volume=115; citation_number=23; citation_pages=231-241; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Detection of gyrA mutations associated with ciprofloxacin resistance in Neisseria gonorrhoeae by rapid and reliable pre-programmed short DNA sequencing.; citation_author=B Gharizadeh; citation_author=M Akhras; citation_author=M Unemo; citation_author=B Wretlind; citation_author=P Nyren; citation_journal_title=Int J Antimicrob Agents; citation_volume=26; citation_number=24; citation_pages=486-490; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sentinel-base DNA genotyping using multiple sequencing primers for high-risk human papillomaviruses.; citation_author=B Gharizadeh; citation_author=B Zheng; citation_author=M Akhras; citation_author=M Ghaderi; citation_author=O Jejelowo; citation_journal_title=Mol Cell Probes; citation_volume=20; citation_number=25; citation_pages=230-238; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Methodological improvements of pyrosequencing technology.; citation_author=B Gharizadeh; citation_author=M Akhras; citation_author=N Nourizad; citation_author=M Ghaderi; citation_author=K Yasuda; citation_journal_title=J Biotechnol; citation_volume=124; citation_number=26; citation_pages=504-511; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The use of general primers GP5 and GP6 elongated at their 3′ ends with adjacent highly conserved sequences improves human papillomavirus detection by PCR.; citation_author=AM de Roda Husman; citation_author=JM Walboomers; citation_author=AJ van den Brule; citation_author=CJ Meijer; citation_author=PJ Snijders; citation_journal_title=J Gen Virol; citation_volume=76 ( Pt 4); citation_number=27; citation_pages=1057-1062; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Improved amplification of genital human papillomaviruses.; citation_author=PE Gravitt; citation_author=CL Peyton; citation_author=TQ Alessi; citation_author=CM Wheeler; citation_author=F Coutlee; citation_journal_title=J Clin Microbiol; citation_volume=38; citation_number=28; citation_pages=357-361; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Oligonucleotide properties determination and primer designing: a critical examination of predictions.; citation_author=S Chavali; citation_author=A Mahajan; citation_author=R Tabassum; citation_author=S Maiti; citation_author=D Bharadwaj; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=29; citation_pages=3918-3925; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Advantage of using inosine at the 3′ termini of 16S rRNA gene universal primers for the study of microbial diversity.; citation_author=E Ben-Dov; citation_author=OH Shapiro; citation_author=N Siboni; citation_author=A Kushmaro; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=30; citation_pages=6902-6906; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular mechanisms of antibacterial multidrug resistance.; citation_author=MN Alekshun; citation_author=SB Levy; citation_journal_title=Cell; citation_volume=128; citation_number=31; citation_pages=1037-1050; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular inversion probe analysis of gene copy alterations reveals distinct categories of colorectal carcinoma.; citation_author=H Ji; citation_author=J Kumm; citation_author=M Zhang; citation_author=K Farnam; citation_author=K Salari; citation_journal_title=Cancer Res; citation_volume=66; citation_number=32; citation_pages=7910-7919; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA cloning using in vitro site-specific recombination.; citation_author=JL Hartley; citation_author=GF Temple; citation_author=MA Brasch; citation_journal_title=Genome Res; citation_volume=10; citation_number=33; citation_pages=1788-1795; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations.; citation_author=D Dressman; citation_author=H Yan; citation_author=G Traverso; citation_author=KW Kinzler; citation_author=B Vogelstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=34; citation_pages=8817-8822; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Linear mRNA amplification from as little as 5 ng total RNA for global gene expression analysis.; citation_author=A Dafforn; citation_author=P Chen; citation_author=G Deng; citation_author=M Herrler; citation_author=D Iglehart; citation_journal_title=Biotechniques; citation_volume=37; citation_number=35; citation_pages=854-857; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Loop-mediated isothermal amplification of DNA.; citation_author=T Notomi; citation_author=H Okayama; citation_author=H Masubuchi; citation_author=T Yonekawa; citation_author=K Watanabe; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=36; citation_pages=E63; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rapid SNP diagnostics using asymmetric isothermal amplification and a new mismatch-suppression technology.; citation_author=Y Mitani; citation_author=A Lezhava; citation_author=Y Kawai; citation_author=T Kikuchi; citation_author=A Oguchi-Katayama; citation_journal_title=Nat Methods; citation_volume=4; citation_number=37; citation_pages=257-262; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=38; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=BEAMing up for detection and quantification of rare sequence variants.; citation_author=M Li; citation_author=F Diehl; citation_author=D Dressman; citation_author=B Vogelstein; citation_author=KW Kinzler; citation_journal_title=Nat Methods; citation_volume=3; citation_number=39; citation_pages=95-97; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Toward the 1,000 dollars human genome.; citation_author=ST Bennett; citation_author=C Barnes; citation_author=A Cox; citation_author=L Davies; citation_author=C Brown; citation_journal_title=Pharmacogenomics; citation_volume=6; citation_number=40; citation_pages=373-382; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Accurate multiplex polony sequencing of an evolved bacterial genome.; citation_author=J Shendure; citation_author=GJ Porreca; citation_author=NB Reppas; citation_author=X Lin; citation_author=JP McCutcheon; citation_journal_title=Science; citation_volume=309; citation_number=41; citation_pages=1728-1732; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequence information can be obtained from single DNA molecules.; citation_author=I Braslavsky; citation_author=B Hebert; citation_author=E Kartalov; citation_author=SR Quake; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=42; citation_pages=3960-3964; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Digital quantification using amplified single-molecule detection.; citation_author=J Jarvius; citation_author=J Melin; citation_author=J Goransson; citation_author=J Stenberg; citation_author=S Fredriksson; citation_journal_title=Nat Methods; citation_volume=3; citation_number=43; citation_pages=725-727; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The use of coded PCR primers enables high-throughput sequencing of multiple homolog amplification products by 454 parallel sequencing.; citation_author=J Binladen; citation_author=MT Gilbert; citation_author=JP Bollback; citation_author=F Panitz; citation_author=C Bendixen; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=44; citation_pages=e197; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ligation-based synthesis of oligonucleotides with block structure.; citation_author=TA Borodina; citation_author=H Lehrach; citation_author=AV Soldatov; citation_journal_title=Anal Biochem; citation_volume=318; citation_number=45; citation_pages=309-313; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications"/>
    <meta name="twitter:description" content="We combined components of a previous assay referred to as Molecular Inversion Probe (MIP) with a complete gap filling strategy, creating a versatile powerful one-primer multiplex amplification system. As a proof-of-concept, this novel method, which employs a Connector Inversion Probe (CIPer), was tested as a genetic tool for pathogen diagnosis, typing, and antibiotic resistance screening with two distinct systems: i) a conserved sequence primer system for genotyping Human Papillomavirus (HPV), a cancer-associated viral agent and ii) screening for antibiotic resistance mutations in the bacterial pathogen Neisseria gonorrhoeae. We also discuss future applications and advances of the CIPer technology such as integration with digital amplification and next-generation sequencing methods. Furthermore, we introduce the concept of two-dimension informational barcodes, i.e. &ldquo;multiplex multiplexing padlocks&rdquo; (MMPs). For the readers' convenience, we also provide an on-line tutorial with user-interface software application CIP creator 1.0.1, for custom probe generation from virtually any new or established primer-pairs."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000915.g003"/>

  <meta property="og:title" content="Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5800'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1282'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=514&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000915">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000915" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000915&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael S. Akhras, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America, 
                  Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Magnus Unemo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Reference Laboratory for Pathogenic Neisseria, Department of Clinical Microbiology, Örebro University Hospital, Örebro, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sreedevi Thiyagarajan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pål Nyrén, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ronald W. Davis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew Z. Fire, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Pathology, Stanford University School of Medicine, Stanford, California, United States of America, 
                  Department of Genetics, Stanford University School of Medicine, Stanford, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nader Pourmand
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pourmand@stanford.edu">pourmand@stanford.edu</a></p>

                <p>Affiliations:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America, 
                  Biomolecular Engineering, University of California at Santa Cruz, Santa Cruz, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000915</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000915-g001" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000915-g002" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000915-t001" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000915-g003" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000915">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000915" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We combined components of a previous assay referred to as Molecular Inversion Probe (MIP) with a complete gap filling strategy, creating a versatile powerful one-primer multiplex amplification system. As a proof-of-concept, this novel method, which employs a Connector Inversion Probe (CIPer), was tested as a genetic tool for pathogen diagnosis, typing, and antibiotic resistance screening with two distinct systems: i) a conserved sequence primer system for genotyping Human Papillomavirus (HPV), a cancer-associated viral agent and ii) screening for antibiotic resistance mutations in the bacterial pathogen <em>Neisseria gonorrhoeae</em>. We also discuss future applications and advances of the CIPer technology such as integration with digital amplification and next-generation sequencing methods. Furthermore, we introduce the concept of two-dimension informational barcodes, i.e. “multiplex multiplexing padlocks” (MMPs). For the readers' convenience, we also provide an on-line tutorial with user-interface software application CIP creator 1.0.1, for custom probe generation from virtually any new or established primer-pairs.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Akhras MS, Unemo M, Thiyagarajan S, Nyrén P, Davis RW, et al.  (2007) Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications. PLoS ONE 2(9):
          e915.
            doi:10.1371/journal.pone.0000915</p><p><strong>Academic Editor: </strong>Neil Hall, University of Liverpool, United Kingdom</p><p><strong>Received:</strong> July 19, 2007; <strong>Accepted:</strong> August 20, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Akhras et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by a grant from National Institutes of Health (PO1-HG000205). Michael S. Akhras was supported by Hans Werthén Foundation; Sixten Gemzéus Scholarship Foundation; The Foundation Blanceflor Boncompagni-Ludovisi, neé Bildt; and the The Fulbright Commission.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Since the introduction of PCR <a href="#pone.0000915-Saiki1">[1]</a>, a vast variety of offspring technologies have been engineered, continuously expanding our molecular toolbox <a href="#pone.0000915-Csako1">[2]</a>. One such technique was multiplex PCR <a href="#pone.0000915-Chamberlain1">[3]</a>, which allows for multiple reactions to occur in a single-tube. Multiplex PCR offers clear benefits in clinical management with respect to cost, time, sample need and contamination risk <a href="#pone.0000915-Elnifro1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recent advances in multiplex PCR technology have been significant <a href="#pone.0000915-Nilsson1">[5]</a>–<a href="#pone.0000915-Dahl1">[7]</a>. Ligase Chain Reaction (LCR), an alternative method of DNA amplification, uses a thermostable ligase for target amplification <a href="#pone.0000915-Barany1">[8]</a>. In Gap-LCR <a href="#pone.0000915-Abravaya1">[9]</a>, a thermostable polymerase is also used, and dNTPs are included to allow for gap filling of the flanking primers, mimicking nature's Okazaki fragments <a href="#pone.0000915-Okazaki1">[10]</a>. The padlock probe <a href="#pone.0000915-Nilsson2">[11]</a> introduced a novel concept of using a one-primer system for DNA target detection, based on ligase probe circularization and Rolling Circle Amplification (RCA) <a href="#pone.0000915-Fire1">[12]</a>, <a href="#pone.0000915-Lizardi1">[13]</a>. The Molecular Inversion Probe (MIP) <a href="#pone.0000915-Hardenbol1">[14]</a>, a more advanced version of the padlock probe, contained also i) a molecular barcode for downstream chip detection, ii) a single base gap that enables single nucleotide polymorphism (SNP) detection, and iii) an exponential amplification strategy based on probe inversion and universal primer PCR amplification. MIP has since proved suitable as an ultra high-throughput method <a href="#pone.0000915-Hardenbol2">[15]</a>, showing potential for use in allele quantification <a href="#pone.0000915-Wang1">[16]</a> and pathogen diagnostics <a href="#pone.0000915-Akhras1">[17]</a>, <a href="#pone.0000915-Thiyagarajan1">[18]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We describe here a novel method, using a Connector Inversion Probe (CIPer), which combines the gap fill strategy of Gap-LCR with the structure of a padlock probe, creating a one-primer amplification with complete gap filling properties (<a href="#pone-0000915-g001">Figure 1</a>). PCR, a highly selective method, offers two degrees of selectivity, due to the use of two primers. CIPer technology uses only one primer, but retains two degrees of selectivity, owing to its dual detection; the target homologous extension site (ES) and anchor site (AS). CIPer also uses universal primer amplification in a one-protocol-fits-all fashion, which circumvents the need for primer-specific thermoprofiles. As a proof-of-concept, we tested CIPer on pathogen recognition with two distinct systems: i) conserved sequence primer genotyping of the cancer-associated viral agent Human Papillomavirus (HPV) <a href="#pone.0000915-deVilliers1">[19]</a>, <a href="#pone.0000915-Doorbar1">[20]</a>, and ii) a duplex amplification of two genes prone to mutations and strongly linked with resistance to key antibiotics (e.g. ciprofloxacin), in the bacterial pathogen <em>Neisseria gonorrhoeae</em> <a href="#pone.0000915-Lindbck1">[21]</a>–<a href="#pone.0000915-Unemo1">[23]</a>. Previously developed Pyrosequencing protocols <a href="#pone.0000915-Gharizadeh1">[24]</a>–<a href="#pone.0000915-Gharizadeh3">[26]</a> were used for downstream sequence validation.</p>
<div class="figure" id="pone-0000915-g001"><div class="img"><a name="pone-0000915-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000915.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000915.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic overview of Connector Inversion Probe (CIPer) technology.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>A) Synthetic oligonucleotide containing regions i) <em>AS and ES</em> (anchor site and extension sites): approximately 20 base pair fragments homologous to regions flanking target of interest, ii) <em>R</em>: restriction site for probe linearization, iii) <em>U1 and U2</em>: universal primer regions for inverted probe amplification. B) Target rescue through complete gap filling by a DNA polymerase and ligation-based probe circularization. C) Circular DNA enrichment through degradation of linear DNA by enzymes Exonuclease I and III, which are deactivated prior to the next step. D) Probe linearization by restriction site cleavage. E) Inverted probes are amplified with universal primers; one is biotinylated for subsequent amplicon validation. F) Amplicon validation using multiple sequencing primers.</p>
<span>doi:10.1371/journal.pone.0000915.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here we introduce the design and development of CIPer as well as apply it with a robust protocol on viral and bacterial genetic diagnostics, typing and antibiotic resistance screening. We also provide an on-line tutorial with the user-interface software application CIP creator 1.0.1, for probe generation from virtually any primer pair (<a href="http://bioel.stanford.edu/CIPer">http://bioel.stanford.edu/CIPer</a>).</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Connector Inversion Probe (CIPer) Design and Performance</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The Connector Inversion Probe (CIPer) reaction (<a href="#pone-0000915-g001">Figure 1</a>) shares some basic design features with a recently described assay, the PathogenMip assay <a href="#pone.0000915-Akhras1">[17]</a>, which includes: i) probe hybridization, single “G” base gap fill and probe circularization, ii) exonuclease based circular DNA selection, iii) probe inversion by re-linearization, and iv) universal PCR amplification. The distinguishing and key feature of the CIPer method is a modified first reaction step involving a complete filling of an extensive gap formed by the probe and target (<a href="#pone-0000915-g001">Figure 1B</a>).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The 3′end of the probe, which we call the Extension Site (ES), primes for polymerization, while the 5′end serves as point of ligation and is referred to as the Anchor Site (AS) (<a href="#pone-0000915-g001">Figure 1</a>). With all four dNTPs present, the polymerase is capable of extending past the intended point of ligation, generating a false negative. To address this problem, probes were designed to have an annealing temperature of approximately 5 degrees higher for the AS than that of the ES. The AS therefore blocks DNA extension when using a polymerase deficient in strand displacement at an appropriate intermediate temperature (50°C).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>For enhanced sensitivity and specificity a vast variety of probe concentrations and thermoprofiles were explored. Probes used in the 10–100 fmol concentration range yielded the strongest signal. Higher cycle numbers and shorter annealing times created a more efficient thermoprofile. The CIPer method successfully genotyped 1 pg of plasmid DNA in the presence of 200 ng non-reacting human genomic DNA, without generating any false positives (<a href="#pone.0000915.s001">Figure S1</a>). Assay optimizations were based on genotyping of HPV plasmid 56, validated by either SYBR GREEN assay or agarose gel electrophoresis, followed by Pyrosequencing verification (see Results, <em>The Conserved Sequence Primer Connector Inversion Probe (CSP-CIPer</em>)).</p>


<h4>The Conserved Sequence Primer Connector Inversion Probe (CSP-CIPer)</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The HPV CIPer (CSP-CIPer) was modeled on the primer-pair, GP5+/6+ <a href="#pone.0000915-deRodaHusman1">[27]</a>, and exemplifies the use of conserved sequence primer (CSP) amplification. HPV genotyping is conventionally performed with a nested PCR <a href="#pone.0000915-Gravitt1">[28]</a>. The interest here lies in the detection potential of the CIPer compared with one-step PCR, and therefore the nested PCR strategy was not a suitable comparator for the model. The purpose of the CSP-CIPer was initially to genotype HPV plasmids -16 and -18, which produced amplicons visualized as agarose gel-bands at the expected size range of 185-188 base pairs and correctly validated with Pyrosequencing (<a href="#pone.0000915.s002">Figure S2</a>). With an accessible collection of 57 plasmids, a CSP-CIPer screen was performed in parallel with conventional GP5+/6+ PCR. All amplicons were validated as described in the Sentinel-base DNA genotyping method <a href="#pone.0000915-Gharizadeh2">[25]</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Among the non-detectable HPV genotypes were plasmids encoding HPV-6, -11, -27, -28, -34, -39, -40, -44 and -73. These genotypes all required a C-base instead of a T-base in the third position from the ES 3′end (<a href="#pone-0000915-g002">Figure 2</a> and <a href="#pone.0000915.s003">Figure S3</a>). A degenerate CSP-CIPer was constructed with a mixed “Y” base (C+T) at the position of interest. The degenerate CSP-CIPer recognized and genotyped 35 plasmids, while the GP5+/6+ PCR genotyped 38 plasmids (<a href="#pone-0000915-t001">Table 1</a>). Successful target amplification schemes appear to be different for CIPers than for PCR primers; the success of a PCR primer-pair is highly dependent on the conservation of the 3'ends <a href="#pone.0000915-Chavali1">[29]</a>, while for the CIPer the anchor site seems more dependent upon annealing temperature. With the degenerate probe, detection rates improved by 15%. With proper mathematical modeling of aligned regions, higher detection success can be achieved with the use of degenerate bases or inosine residues <a href="#pone.0000915-BenDov1">[30]</a>.</p>
<div class="figure" id="pone-0000915-g002"><div class="img"><a name="pone-0000915-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000915.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000915.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Sequence alignments (WebLogos, http://weblogo.berkeley.edu/logo.cgi) of the positive strand containing conserved sequence primer regions for GP5+/6+.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The height of a DNA-base height represents the relative abundance of that particular base at the position of interest; multiple bases at the same position indicate a variable site. Alignment was performed for 40 genotypes detected with the CSP-CIPer or GP5+/6+ PCR (<a href="#pone-0000915-t001">Table 1</a> and <a href="#pone.0000915.s003">Figure S3</a>). The target region between the two primers varies between 90-100 base pairs, depending on genotype characteristics. The positive strand of the GP6+ region is the complementary sequence of the established primer, and is therefore denoted cGP6+.</p>
<span>doi:10.1371/journal.pone.0000915.g002</span></div><div class="figure" id="pone-0000915-t001"><div class="img"><a name="pone-0000915-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000915.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000915.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Genotyping data from CSP-CIPer and GP5+/6+ PCR parallel screens.</span></strong></p><span>doi:10.1371/journal.pone.0000915.t001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>To investigate the multiplexing strength of the CSP-CIPer, mixtures of 1-8 plasmids (encoding HPV-16, -18, -33, -35, -39, -45, -58 and -59) were made to mimic multiple co-infections, as these are known to occur. The GP5+/6+ primers could genotype four plasmids (encoding HPV-16, -18, -33 and -45) in the same mixture, while the CSP-CIPer could genotype all eight plasmids (<a href="#pone.0000915.s004">Figure S4</a>). The CSP-CIPer showed a 50% improved discriminatory power over that of the GP5+/6+ PCR. Differences in target amplification strategy account for this improvement. That is, PCR is based on target-dependent amplification, while the CIPer is based on target dependent probe circularization, followed by non-target dependent universal amplification (<a href="#pone-0000915-g001">Figure 1</a>).</p>


<h4>Antibiotic Resistance Mutation Screening</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>CIPer detection was also applied to screening of resistance mutations in <em>Neisseria gonorrhoeae</em>. Two CIPers were used to perform a duplex amplification in a single-tube of the genes <em>gyrA</em> and <em>parC</em> <a href="#pone.0000915-Unemo1">[23]</a>, which are known to mutate easily and by specific SNPs generate resistance to ciprofloxacin and other quinolone antibiotics. The CIPers were designed with CIP creator 1.0.1 (<a href="http://bioel.stanford.edu/CIPer">http://bioel.stanford.edu/CIPer</a>) based on previously described PCR primer-pairs not originally intended for multiplex PCR <a href="#pone.0000915-Lindbck1">[21]</a>. We intentionally inverted the <em>gyrA</em> CIPer to target the negative strand, while the <em>parC</em> CIPer targeted the positive strand, to illustrate that CIPers can be used on both DNA strands. In the case of close target proximities, it might be wise to revise such a strategy. Mutations on amino acid residues S91 and D95 for <em>gyrA</em>, and E91 for <em>parC</em> were the particular subjects of our interrogation. There are many other candidate genes for future CIPer-based epidemiological studies of antibiotic resistance <a href="#pone.0000915-Alekshun1">[31]</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Thirty-one <em>Neisseria gonorrhoeae</em> DNA extracts were screened in parallel with the CIPers in a single-tube reaction, and conventional PCR was performed in separate tubes for the two genes. Results from CIPer and PCR were in 100% agreement. In <em>gyrA</em>, eight wild type strains, three strains with double mutations resulting in amino acid alterations S91P and D95A, ten strains with double mutations resulting in S91P and D95G, eight strains with single mutation S91P, and two strains with single mutation D95N were observed. In <em>parC</em>, thirty wild type strains, and one strain with an E91G mutation was observed (<a href="#pone-0000915-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0000915-g003"><div class="img"><a name="pone-0000915-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000915" data-uri="info:doi/10.1371/journal.pone.0000915.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000915.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000915.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000915.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000915.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000915.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Pyrograms of Connector Inversion Probes (CIPer) used for screening of mutations conferring antibiotic (ciprofloxacin) resistance of <em>Neisseria gonorrhoeae</em> strains.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Amino acid codons are marked alternating blue and orange, and their positions in the coded amino acid sequence of the target enzymes (GyrA and ParC) are placed on top of the traces. Codons marked in red are known to be subject to mutations, which are strongly correlated with ciprofloxacin resistance <a href="#pone.0000915-Lindbck1">[21]</a>–<a href="#pone.0000915-Unemo1">[23]</a>. The <em>gyrA</em> and <em>parC</em> mutant sequences are compared with the respective wild type sequences. The displayed pyrograms show two common mutations (S91P and D95G) in <em>gyrA</em>, and one mutation (E91G) in <em>parC</em>.</p>
<span>doi:10.1371/journal.pone.0000915.g003</span></div>

<h4>Applications and Advances in Connector Inversion Probe (CIPer) Technology</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Connector Inversion Probe (CIPer) technology is a powerful method for multiplex amplification, and has been validated here in genetic pathogen diagnostics, typing, and antibiotic resistance detection arenas as a proof-of-concept. This technology can be designed and optimized for use in numerous other scientific areas and applications. Other areas suitable for use of the CIPer method include cancer genomics <a href="#pone.0000915-Ji1">[32]</a>, human SNP detection <a href="#pone.0000915-Hardenbol2">[15]</a>, and selective exon amplification <a href="#pone.0000915-Dahl1">[7]</a>, among numerous other promising applications. Modified CIPers may also in the future offer an alternative strategy for gateway cloning <a href="#pone.0000915-Hartley1">[33]</a>. In this paper, we focused on “lowplex” systems amplifying shorter stretches of DNA of up to 100 base pair lengths. To achieve higher degrees of multiplexing and longer clone reads, one must take into account i) clone GC content, ii) clone length variability, and iii) steric effects for longer clones.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Analog amplification and downstream validation are delimiting factors for multiplex detection methods. We believe that the future of CIPer technology will rely on digital amplification <a href="#pone.0000915-Dressman1">[34]</a> and high throughput sequencing validation (<a href="#pone.0000915.s005">Figure S5</a>). The CIPers universal primer sites can be formatted to fit any amplification method, e.g. RCA <a href="#pone.0000915-Fire1">[12]</a>, <a href="#pone.0000915-Lizardi1">[13]</a>, emulsion PCR <a href="#pone.0000915-Dressman1">[34]</a>, SPIA <a href="#pone.0000915-Dafforn1">[35]</a> LAMP <a href="#pone.0000915-Notomi1">[36]</a> and SMAP 2 <a href="#pone.0000915-Mitani1">[37]</a>; universal adaptor ligation, a tedious and sample-consuming procedure <a href="#pone.0000915-Margulies1">[38]</a>, is therefore optional. Proposed digital amplification methods include emulsion PCR, RCA, or a combination of the two <a href="#pone.0000915-Li1">[39]</a>. Suitable upcoming high throughput validation methods include next-generation sequence systems such as the 454 Life Sciences platform <a href="#pone.0000915-Margulies1">[38]</a> (<a href="http://www.454.com">http://www.454.com</a>), the Solexa platform <a href="#pone.0000915-Bennett1">[40]</a> (<a href="http://www.illumina.com">http://www.illumina.com</a>), polony sequencing <a href="#pone.0000915-Shendure1">[41]</a> (<a href="http://www.agentcourt.com">http://www.agentcourt.com</a>), Helicos Biosciences <a href="#pone.0000915-Braslavsky1">[42]</a> (<a href="http://www.helicosbio.com">http://www.helicosbio.com</a>), or use of a resequencing array (GeneChip CustomSeq, <a href="http://www.affymetrix.com">www.affymetrix.com</a>). Recent digital quantification methods <a href="#pone.0000915-Jarvius1">[43]</a> open the possibility for quantitative CIPer technology.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>With the addition of a molecular barcode to the padlocks <a href="#pone.0000915-Hardenbol1">[14]</a>, MIPs were used for ultra high-throughput SNP detection based on microarray barcode hybridization <a href="#pone.0000915-Hardenbol2">[15]</a>. Because CIPers can also include a barcode, they could be used for microarray validation, or amplicon tracing with sequencing methods <a href="#pone.0000915-Akhras1">[17]</a>, <a href="#pone.0000915-Binladen1">[44]</a> allowing for longer read lengths <a href="#pone.0000915-Margulies1">[38]</a>. With the introduction of a third universal sequence flanking the barcode and the universal primer sites, one can begin to envision a two-dimension informational barcode, a concept we call “multiplexing multiplex padlocks” (MMP) (<a href="#pone.0000915.s006">Figure S6</a>). Probe construction based on ligation of three smaller constructs <a href="#pone.0000915-Borodina1">[45]</a> would allow for variability in fragment assembly. By combining the one-genotype detection pair of AS/ES constructs with multiple unique barcodes, we can create patient and genotype specific probes with one barcode per probe. This will allow for analysis of multiple parallel samples, essentially lowering the cost of microarray chips and sequencing reactions.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>DNA Materials</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Oligonucleotides used in the assay (<a href="#pone.0000915.s007">Table S1</a>) were produced by IDT technologies (Coralville, IA) and melting temperatures were calculated using SciTools (IDT technologies). HPV plasmids were kindly provided by Dr. E. M. de Villiers (DKFZ; Heidelberg, Germany) HPV-6B, -11, -16, -18, -26, -27, -40, -45, -53, -72 and -73; HPV-9, -10, -12, -14, -15, -17, -19, -20, -21, -22, -23, -24, -28, -29, -32, -33, -34, -36, -39, -42, -49, -50, -54, -66, -70 and -74 by Dr. M. Favre (Institute Pasteur; Paris, France); HPV-35, -44 and -56 by Dr. A. Lorincz (Digene Corporation; Gaithersburg, MD); HPV-58, -59, -67, -69, -71, -81 and -82 by Dr. T. Matsukura (National Institute of Health; Tokyo, Japan); HPV-52 by Dr. W. Lancaster (Wayne State University School of Medicine; Detroit, MI); HPV-62, -84, -cand86, -cand89, -90 and -91 by Dr. R. D. Burk (Albert Einstein College of Medicine of Yeshiva University, Bronx, NY); HPV-92, -93 and -96 by Dr. O. Forslund (UMAS; Malmö University Hospital, Malmö, Sweden). All HPV plasmids were normalized at 100 ng/µl (±15%) using a ND-100 Spectrophotometer (NanoDrop, Wilmington, DE). <em>Neisseria gonorrhoeae</em> DNA had been previously extracted <a href="#pone.0000915-Gharizadeh1">[24]</a>. Commercially available human genomic DNA G3041 (Promega, Madison, WI) was used for assay evaluation.</p>


<h4>Connector Inversion Probe Assay</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>CIPer reactions were performed as follows: <strong>i)</strong> 100 ng of HPV plasmid DNA or <em>Neisseria gonorrhoeae</em> DNA extract, 10 fmol each of probe, 0.25 units Ampligase (Epicenter Biotechnologies, Madison, WI), 0.25 mM of each dNTP (Fermentas, Hanover, MD), and 0.5 units AmpliTaq DNA Polymerase Stoffel Fragment (Applied Biosystems, Foster City, CA) in 1X Ampligase buffer (Epicenter Biotechnologies, Madison, WI) were combined in a total volume of 10 µl/reaction. After pre-incubation at 95°C for 10 min, the reaction went through 99 cycles of 95°C for 2 min, 50°C for 5 min and 37°C for 2 min. <strong>ii)</strong> 10 µl of 130 mM Tris-HCl (pH 7.8), 85.75 mM KCl, 3.85 mM MgCl<sub>2</sub> and 0.1% BSA solution containing 5.6 units Exonuclease I and 112 units Exonuclease III (Epicenter Biotechnologies, Madison, WI) was added to each reaction. The mixture was incubated at 37°C for 60 min followed by enzyme deactivation at 80°C for 20 min. <strong>iii)</strong> 4 units of Uracil-DNA-Glycosylase (New England BioLabs, Ipswich, MA) was added to each reaction and incubated at 37°C for 60 min followed by enzyme deactivation at 80°C for 20 min. <strong>iv)</strong> PCR amplification was carried out in a total reaction volume of 50 µl, containing 5 µl CIPer reaction, 1X PCR Buffer II (Applied Biosystems, Foster City, CA), 2.5 mM MgCl<sub>2</sub> (Applied Biosystems, Foster City, CA), 0.12 mM dNTPs (Fermentas, Hanover, MD), 2.5 units AmpliTaq Gold DNA polymerase (Applied Biosystems, Foster City, CA) and 0.2 µM of each universal primer B-NP-F and NP-R (<a href="#pone.0000915.s007">Table S1</a>). A 10-min incubation step at 95°C was followed by 40 cycles of amplification with a thermocycler GeneAmp PCR system 9700 (Applied Biosystems, Foster City, CA). Each cycle included a denaturation step at 95°C for 45 sec, an annealing step at 52°C for 30 sec, and an extension step at 72°C for 30 sec. A final extension was done at 72°C for 5 min. Real-time PCR reactions were performed with the same thermoprofile on an Mx3005P system (Stratagene, La Jolla, CA), with a SYBR green PCR master mix (Applied Biosystems, Foster City, CA). Gel electrophoresis was performed on a Sub-Cell Gt Agarose Electrophoresis System (Bio-Rad, Hercules, CA) using a 2.5% agarose gel GenePure LE Quick Dissolve (ISC Bioexpress, Kaysville, UT) with 5 µg/ml ethidium bromide (Sigma-Aldrich, St. Louis, MO) staining.</p>


<h4>Pyrosequencing</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Single strand template preparation was performed as previously described <a href="#pone.0000915-Gharizadeh3">[26]</a>. GP5+ and Sentinel-base sequencing primers were used to screen all HPV CSP-CIPer clones and GYRA2-4 and PARC-3 sequencing primers were used for <em>Neisseria gonorrhoeae</em> CIPer clones (<a href="#pone.0000915.s007">Table S1</a>). Pyrosequencing was performed with a cyclic <em>de novo</em> sequencing dispensation (ACGT) using a PSQ™ HS96A DNA sequencing system (Biotage, Uppsala, Sweden).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000915"></div><a name="pone.0000915.s001" id="pone.0000915.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Agarose gel stained with ethidium bromide for CSP-CIPer detection limit of HPV -56 in presence of human genomic DNA. The plasmid concentration was varied, ranging from 10 ng to 100 fg per reaction at a constant background of 200 ng non-HPV-contaminated human genomic DNA. The minimum detectable amount of HPV observed was 1 pg, and the 100 fg mixture showed no significant amplification. The upper triangular graph represents a visual interpretation of CIPer amplicon intensity as a function of plasmid concentration, and the lower graph the background intensity as a function of plasmid concentration.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB PDF)</p>
<a name="pone.0000915.s002" id="pone.0000915.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Agarose gel stained with ethidium bromide for fragment size determination and pyrograms for HPV genotypes -16 and -18. Amplicon sizes appear in the expected size of 185 base pairs for HPV-16 and 188 base pairs for HPV-18. The negative controls for the CIPer reaction (“CIPer-”) and PCR (“PCR-”) show no significant amplification. Pyrograms derived from multiple sequencing primers MSP-16 and MSP-18 (<a href="#pone.0000915.s007">Table S1</a>) validate the expected sequence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.21 MB PDF)</p>
<a name="pone.0000915.s003" id="pone.0000915.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Genomic regions GP5+/6+ aligned for 40 genotypes detected by either CSP-CIPer or GP5+/6+ PCR (<a href="#pone-0000915-t001">Table 1</a>). Alignment was performed and displayed with ClustalX (<a href="http://www.biodirectory.com/biowiki/ClustalX">http://www.biodirectory.com/biowiki/Clus​talX</a>). The targeted region flanking the two primers varies between 90-100 base pairs depending on genotype characteristics, and is represented in the figure with three Ns.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.30 MB PDF)</p>
<a name="pone.0000915.s004" id="pone.0000915.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Detecting multiple HPV co-infections. In order to compare the discriminative power of the CIPer vs. PCR detection, artificial mixtures of the eight plasmids HPV-16, -18, -33, -35, -39, -45, -58 and -59 were constructed to “mimic” real-case multiple co-infections. The CIPer method could detect all eight genotypes present in the same sample, while PCR managed with the four genotypes HPV-16, -18, -33 and -45. As seen in the figure the Pyrogram intensities vary for different genotypes, indicating a lower discriminative preference for certain types. Among the four PCR detected genotypes, HPV-16 and -33 showed very weak diagrams and were barely detectable in the presence of the preferred genotypes -18 and -45. CIPer detection suffered to a lesser degree of such preferred selectivity, as all types were clearly distinguishable. We believe that differences in target amplification strategy account for this improvement, i.e. PCR is based on target dependent amplification, while the CIPer is based on target dependent probe circularization, followed by non-target dependent universal amplification.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.64 MB PDF)</p>
<a name="pone.0000915.s005" id="pone.0000915.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Digital amplification strategies prior to probes validation. A) Emulsion based PCR; adapters are optional since CIPers already contain universal segments flanking the target of interest. The technology involves the inclusion of DNA and a primed magnetic bead in mineral oil (an emulsion), which allows for single molecule amplifications. Suitable upcoming methods for downstream sequence validation include the 454 Life Sciences platform (<a href="http://www.454.com">http://www.454.com</a>), the Solexa platform (<a href="http://www.illumina.com">http://www.illumina.com</a>), polony sequencing (<a href="http://www.agentcourt.com">http://www.agentcourt.com</a>), Helicos Biosciences (<a href="http://www.helicosbio.com">http://www.helicosbio.com</a>), or use of a resequencing array (GeneChip CustomSeq, <a href="http://www.affymetrix.com">www.affymetrix.com</a>). B) Rolling circle amplification (RCA) with single-molecule detection (SMD). Digital quantification combines RCA and SMD in form of fluorescent-labeled target specific oligonucleotides. The amplified CIPers can be quantified using microfluidic analysis and visualized with a microscope for ultimate levels of quantification. The number of available fluorescent labels limits the degree of multiplexing.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.06 MB PDF)</p>
<a name="pone.0000915.s006" id="pone.0000915.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">“Multiplex multiplexing padlocks” (MMP) construct assembly for CIPer probes production. The extended CIPer carries a third universal segment and flanks the barcode together with the other universal segments. The probe is divided and synthesized in three constructs, with the middle one containing the barcode. Through a two-way ligation scheme the three fragments can be joined into a full-length probe. Following the grid-like assembly strategy displayed in the figure, one-genotype detection pair of AS/ES constructs is combined with multiple unique barcodes. The unique barcode now carries two dimensions of information, i.e. genotype and patient ID. Post-reaction CIPers can be combined into multiple patient pools and all screened simultaneously, essentially lowering costs involved with downstream validations, both for hybridization-based techniques and sequencing procedures.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0000915.s007" id="pone.0000915.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000915.s007">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Oligonucleotides used in the present study. Bold italic marked sequences in the anchor sites of CIPers -gyrA and +parC were added to the originally described primers <a href="#pone.0000915-Lindbck1">[21]</a> to obtain higher annealing temperature values for the anchor site. The number in parenthesis following the multiple sequencing primers (MSPs) denotes which primer pool the MSP belongs to.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000915.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.09 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Julianna P. Erickson for excellent technical assistance, Mike Jensen and Keith Anderson for oligo synthesis, Heather Halenbeck for critical reading of this work, Prof. Bengt Wretlind and Dr. Emma Lindbäck for their clinical expertise and support, Amina Sutherland-Stolting and Jantien van der Laan for their technical assistance, and Monika Trebo for her website support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AF NP MA. Performed the experiments: MA. Analyzed the data: NP MA. Contributed reagents/materials/analysis tools: NP MA MU ST. Wrote the paper: AF NP MA. Other: Director of the Stanford Genome Technology Center: RW. Academic supervisor of Michael S. Akhras: PN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000915-Saiki1" id="pone.0000915-Saiki1"></a>Saiki RK, Scharf S, Faloona F, Mullis KB, Horn GT, et al.  (1985) Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia. Science  230: 1350–1354.  <ul class="find" data-citedArticleID="994200" data-doi="10.1126/science.2999980"><li><a href="http://dx.doi.org/10.1126/science.2999980" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymatic+amplification+of+beta-globin+genomic+sequences+and+restriction+site+analysis+for+diagnosis+of+sickle+cell+anemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymatic+amplification+of+beta-globin+genomic+sequences+and+restriction+site+analysis+for+diagnosis+of+sickle+cell+anemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000915-Csako1" id="pone.0000915-Csako1"></a>Csako G (2006) Present and future of rapid and/or high-throughput methods for nucleic acid testing. Clin Chim Acta  363: 6–31.  <ul class="find" data-citedArticleID="994146" data-doi="10.1016/j.cccn.2005.07.009"><li><a href="http://dx.doi.org/10.1016/j.cccn.2005.07.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Present+and+future+of+rapid+and%2For+high-throughput+methods+for+nucleic+acid+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Present+and+future+of+rapid+and%2For+high-throughput+methods+for+nucleic+acid+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000915-Chamberlain1" id="pone.0000915-Chamberlain1"></a>Chamberlain JS, Gibbs RA, Ranier JE, Nguyen PN, Caskey CT (1988) Deletion screening of the Duchenne muscular dystrophy locus via multiplex DNA amplification. Nucleic Acids Res  16: 11141–11156.  <ul class="find" data-citedArticleID="994142" data-doi="10.1093/nar/16.23.11141"><li><a href="http://dx.doi.org/10.1093/nar/16.23.11141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletion+screening+of+the+Duchenne+muscular+dystrophy+locus+via+multiplex+DNA+amplification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletion+screening+of+the+Duchenne+muscular+dystrophy+locus+via+multiplex+DNA+amplification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000915-Elnifro1" id="pone.0000915-Elnifro1"></a>Elnifro EM, Ashshi AM, Cooper RJ, Klapper PE (2000) Multiplex PCR: optimization and application in diagnostic virology. Clin Microbiol Rev  13: 559–570.  <ul class="find" data-citedArticleID="994156" data-doi="10.1128/cmr.13.4.559-570.2000"><li><a href="http://dx.doi.org/10.1128/cmr.13.4.559-570.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+PCR%3A+optimization+and+application+in+diagnostic+virology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+PCR%3A+optimization+and+application+in+diagnostic+virology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000915-Nilsson1" id="pone.0000915-Nilsson1"></a>Nilsson M, Dahl F, Larsson C, Gullberg M, Stenberg J (2006) Analyzing genes using closing and replicating circles. Trends Biotechnol  24: 83–88.  <ul class="find" data-citedArticleID="994192" data-doi="10.1016/j.tibtech.2005.12.005"><li><a href="http://dx.doi.org/10.1016/j.tibtech.2005.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyzing+genes+using+closing+and+replicating+circles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyzing+genes+using+closing+and+replicating+circles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000915-Fredriksson1" id="pone.0000915-Fredriksson1"></a>Fredriksson S, Baner J, Dahl F, Chu A, Ji H, et al.  (2007) Multiplex amplification of all coding sequences within 10 cancer genes by Gene-Collector. Nucleic Acids Res  35: e47.  <ul class="find" data-citedArticleID="994160" data-doi="10.1093/nar/gkm078"><li><a href="http://dx.doi.org/10.1093/nar/gkm078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+amplification+of+all+coding+sequences+within+10+cancer+genes+by+Gene-Collector." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+amplification+of+all+coding+sequences+within+10+cancer+genes+by+Gene-Collector.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000915-Dahl1" id="pone.0000915-Dahl1"></a>Dahl F, Stenberg J, Fredriksson S, Welch K, Zhang M, et al.  (2007) Multigene amplification and massively parallel sequencing for cancer mutation discovery. Proc Natl Acad Sci U S A.   <ul class="find" data-citedArticleID="994150" data-doi="10.1073/pnas.0702165104"><li><a href="http://dx.doi.org/10.1073/pnas.0702165104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multigene+amplification+and+massively+parallel+sequencing+for+cancer+mutation+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multigene+amplification+and+massively+parallel+sequencing+for+cancer+mutation+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000915-Barany1" id="pone.0000915-Barany1"></a>Barany F (1991) Genetic disease detection and DNA amplification using cloned thermostable ligase. Proc Natl Acad Sci U S A  88: 189–193.  <ul class="find" data-citedArticleID="994130" data-doi="10.1073/pnas.88.1.189"><li><a href="http://dx.doi.org/10.1073/pnas.88.1.189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+disease+detection+and+DNA+amplification+using+cloned+thermostable+ligase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+disease+detection+and+DNA+amplification+using+cloned+thermostable+ligase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000915-Abravaya1" id="pone.0000915-Abravaya1"></a>Abravaya K, Carrino JJ, Muldoon S, Lee HH (1995) Detection of point mutations with a modified ligase chain reaction (Gap-LCR). Nucleic Acids Res  23: 675–682.  <ul class="find" data-citedArticleID="994124" data-doi="10.1093/nar/23.4.675"><li><a href="http://dx.doi.org/10.1093/nar/23.4.675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+point+mutations+with+a+modified+ligase+chain+reaction+%28Gap-LCR%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+point+mutations+with+a+modified+ligase+chain+reaction+%28Gap-LCR%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000915-Okazaki1" id="pone.0000915-Okazaki1"></a>Okazaki R, Okazaki T, Sakabe K, Sugimoto K, Sugino A (1968) Mechanism of DNA chain growth. I. Possible discontinuity and unusual secondary structure of newly synthesized chains. Proc Natl Acad Sci U S A  59: 598–605.  <ul class="find" data-citedArticleID="994198" data-doi="10.1073/pnas.59.2.598"><li><a href="http://dx.doi.org/10.1073/pnas.59.2.598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+DNA+chain+growth.+I.+Possible+discontinuity+and+unusual+secondary+structure+of+newly+synthesized+chains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+DNA+chain+growth.+I.+Possible+discontinuity+and+unusual+secondary+structure+of+newly+synthesized+chains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000915-Nilsson2" id="pone.0000915-Nilsson2"></a>Nilsson M, Malmgren H, Samiotaki M, Kwiatkowski M, Chowdhary BP, et al.  (1994) Padlock probes: circularizing oligonucleotides for localized DNA detection. Science  265: 2085–2088.  <ul class="find" data-citedArticleID="994194" data-doi="10.1126/science.7522346"><li><a href="http://dx.doi.org/10.1126/science.7522346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Padlock+probes%3A+circularizing+oligonucleotides+for+localized+DNA+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Padlock+probes%3A+circularizing+oligonucleotides+for+localized+DNA+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000915-Fire1" id="pone.0000915-Fire1"></a>Fire A, Xu SQ (1995) Rolling replication of short DNA circles. Proc Natl Acad Sci U S A  92: 4641–4645.  <ul class="find" data-citedArticleID="994158" data-doi="10.1073/pnas.92.10.4641"><li><a href="http://dx.doi.org/10.1073/pnas.92.10.4641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rolling+replication+of+short+DNA+circles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rolling+replication+of+short+DNA+circles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000915-Lizardi1" id="pone.0000915-Lizardi1"></a>Lizardi PM, Huang X, Zhu Z, Bray-Ward P, Thomas DC, et al.  (1998) Mutation detection and single-molecule counting using isothermal rolling-circle amplification. Nat Genet  19: 225–232.  <ul class="find" data-citedArticleID="994186" data-doi="10.1038/898"><li><a href="http://dx.doi.org/10.1038/898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+detection+and+single-molecule+counting+using+isothermal+rolling-circle+amplification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+detection+and+single-molecule+counting+using+isothermal+rolling-circle+amplification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000915-Hardenbol1" id="pone.0000915-Hardenbol1"></a>Hardenbol P, Baner J, Jain M, Nilsson M, Namsaraev EA, et al.  (2003) Multiplexed genotyping with sequence-tagged molecular inversion probes. Nat Biotechnol  21: 673–678.  <ul class="find" data-citedArticleID="994170" data-doi="10.1038/nbt821"><li><a href="http://dx.doi.org/10.1038/nbt821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplexed+genotyping+with+sequence-tagged+molecular+inversion+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplexed+genotyping+with+sequence-tagged+molecular+inversion+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000915-Hardenbol2" id="pone.0000915-Hardenbol2"></a>Hardenbol P, Yu F, Belmont J, Mackenzie J, Bruckner C, et al.  (2005) Highly multiplexed molecular inversion probe genotyping: over 10,000 targeted SNPs genotyped in a single tube assay. Genome Res  15: 269–275.  <ul class="find" data-citedArticleID="994172" data-doi="10.1101/gr.3185605"><li><a href="http://dx.doi.org/10.1101/gr.3185605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+multiplexed+molecular+inversion+probe+genotyping%3A+over+10%2C000+targeted+SNPs+genotyped+in+a+single+tube+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+multiplexed+molecular+inversion+probe+genotyping%3A+over+10%2C000+targeted+SNPs+genotyped+in+a+single+tube+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000915-Wang1" id="pone.0000915-Wang1"></a>Wang Y, Moorhead M, Karlin-Neumann G, Falkowski M, Chen C, et al.  (2005) Allele quantification using molecular inversion probes (MIP). Nucleic Acids Res  33: e183.  <ul class="find" data-citedArticleID="994208" data-doi="10.1093/nar/gni177"><li><a href="http://dx.doi.org/10.1093/nar/gni177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allele+quantification+using+molecular+inversion+probes+%28MIP%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allele+quantification+using+molecular+inversion+probes+%28MIP%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000915-Akhras1" id="pone.0000915-Akhras1"></a>Akhras MS, Thiyagarajan S, Villablanca AC, Davis RW, Nyrén P, et al.  (2007) PathogenMip Assay: A Multiplex Pathogen Detection Assay. PLoS ONE  2: e223.  <ul class="find" data-citedArticleID="994126" data-doi="10.1371/journal.pone.0000223"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000223" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PathogenMip+Assay%3A+A+Multiplex+Pathogen+Detection+Assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PathogenMip+Assay%3A+A+Multiplex+Pathogen+Detection+Assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000915-Thiyagarajan1" id="pone.0000915-Thiyagarajan1"></a>Thiyagarajan S, Karhanek M, Akhras M, Davis RW, Pourmand N (2006) PathogenMIPer: a tool for the design of molecular inversion probes to detect multiple pathogens. BMC Bioinformatics  7: 500.  <ul class="find" data-citedArticleID="994204" data-doi="10.1186/1471-2105-7-500"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PathogenMIPer%3A+a+tool+for+the+design+of+molecular+inversion+probes+to+detect+multiple+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PathogenMIPer%3A+a+tool+for+the+design+of+molecular+inversion+probes+to+detect+multiple+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000915-deVilliers1" id="pone.0000915-deVilliers1"></a>de Villiers EM, Fauquet C, Broker TR, Bernard HU, zur Hausen H (2004) Classification of papillomaviruses. Virology  324: 17–27.  <ul class="find" data-citedArticleID="994212" data-doi="10.1016/j.virol.2004.03.033"><li><a href="http://dx.doi.org/10.1016/j.virol.2004.03.033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000915-Doorbar1" id="pone.0000915-Doorbar1"></a>Doorbar J (2006) Molecular biology of human papillomavirus infection and cervical cancer. Clin Sci (Lond)  110: 525–541.  <ul class="find" data-citedArticleID="994152" data-doi="10.1042/cs20050369"><li><a href="http://dx.doi.org/10.1042/cs20050369" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+biology+of+human+papillomavirus+infection+and+cervical+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+biology+of+human+papillomavirus+infection+and+cervical+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000915-Lindbck1" id="pone.0000915-Lindbck1"></a>Lindbäck E, Gharizadeh B, Ataker F, Airell A, Jalal S, et al.  (2005) DNA gyrase gene in Neisseria gonorrhoeae as indicator for resistance to ciprofloxacin and species verification. Int J STD AIDS  16: 142–147.  <ul class="find" data-citedArticleID="994182" data-doi="10.1258/0956462053057675"><li><a href="http://dx.doi.org/10.1258/0956462053057675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+gyrase+gene+in+Neisseria+gonorrhoeae+as+indicator+for+resistance+to+ciprofloxacin+and+species+verification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+gyrase+gene+in+Neisseria+gonorrhoeae+as+indicator+for+resistance+to+ciprofloxacin+and+species+verification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000915-Lindbck2" id="pone.0000915-Lindbck2"></a>Lindbäck E, Unemo M, Akhras M, Gharizadeh B, Fredlund H, et al.  (2006) Pyrosequencing of the DNA gyrase gene in Neisseria species: effective indicator of ciprofloxacin resistance in Neisseria gonorrhoeae. Apmis  114: 837–841.  <ul class="find" data-citedArticleID="994184" data-doi="10.1111/j.1600-0463.2006.apm_495.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0463.2006.apm_495.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pyrosequencing+of+the+DNA+gyrase+gene+in+Neisseria+species%3A+effective+indicator+of+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pyrosequencing+of+the+DNA+gyrase+gene+in+Neisseria+species%3A+effective+indicator+of+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000915-Unemo1" id="pone.0000915-Unemo1"></a>Unemo M, Sjöstrand A, Akhras M, Gharizadeh B, Lindback E, et al.  (2007) Molecular characterization of Neisseria gonorrhoeae identifies transmission and resistance of one ciprofloxacin-resistant strain. Apmis  115: 231–241.  <ul class="find" data-citedArticleID="994206" data-doi="10.1111/j.1600-0463.2007.apm_487.x"><li><a href="http://dx.doi.org/10.1111/j.1600-0463.2007.apm_487.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+Neisseria+gonorrhoeae+identifies+transmission+and+resistance+of+one+ciprofloxacin-resistant+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+Neisseria+gonorrhoeae+identifies+transmission+and+resistance+of+one+ciprofloxacin-resistant+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000915-Gharizadeh1" id="pone.0000915-Gharizadeh1"></a>Gharizadeh B, Akhras M, Unemo M, Wretlind B, Nyren P, et al.  (2005) Detection of gyrA mutations associated with ciprofloxacin resistance in Neisseria gonorrhoeae by rapid and reliable pre-programmed short DNA sequencing. Int J Antimicrob Agents  26: 486–490.  <ul class="find" data-citedArticleID="994162" data-doi="10.1016/j.ijantimicag.2005.08.017"><li><a href="http://dx.doi.org/10.1016/j.ijantimicag.2005.08.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+gyrA+mutations+associated+with+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae+by+rapid+and+reliable+pre-programmed+short+DNA+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+gyrA+mutations+associated+with+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae+by+rapid+and+reliable+pre-programmed+short+DNA+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000915-Gharizadeh2" id="pone.0000915-Gharizadeh2"></a>Gharizadeh B, Zheng B, Akhras M, Ghaderi M, Jejelowo O, et al.  (2006) Sentinel-base DNA genotyping using multiple sequencing primers for high-risk human papillomaviruses. Mol Cell Probes  20: 230–238.  <ul class="find" data-citedArticleID="994164" data-doi="10.1016/j.mcp.2006.01.002"><li><a href="http://dx.doi.org/10.1016/j.mcp.2006.01.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sentinel-base+DNA+genotyping+using+multiple+sequencing+primers+for+high-risk+human+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sentinel-base+DNA+genotyping+using+multiple+sequencing+primers+for+high-risk+human+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000915-Gharizadeh3" id="pone.0000915-Gharizadeh3"></a>Gharizadeh B, Akhras M, Nourizad N, Ghaderi M, Yasuda K, et al.  (2006) Methodological improvements of pyrosequencing technology. J Biotechnol  124: 504–511.  <ul class="find" data-citedArticleID="994166" data-doi="10.1016/j.jbiotec.2006.01.025"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2006.01.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methodological+improvements+of+pyrosequencing+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methodological+improvements+of+pyrosequencing+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000915-deRodaHusman1" id="pone.0000915-deRodaHusman1"></a>de Roda Husman AM, Walboomers JM, van den Brule AJ, Meijer CJ, Snijders PJ (1995) The use of general primers GP5 and GP6 elongated at their 3′ ends with adjacent highly conserved sequences improves human papillomavirus detection by PCR. J Gen Virol  76 ( Pt 4): 1057–1062.  <ul class="find" data-citedArticleID="994210" data-doi="10.1099/0022-1317-76-4-1057"><li><a href="http://dx.doi.org/10.1099/0022-1317-76-4-1057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+general+primers+GP5+and+GP6+elongated+at+their+3%26%23x2032%3B+ends+with+adjacent+highly+conserved+sequences+improves+human+papillomavirus+detection+by+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+general+primers+GP5+and+GP6+elongated+at+their+3%26%23x2032%3B+ends+with+adjacent+highly+conserved+sequences+improves+human+papillomavirus+detection+by+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000915-Gravitt1" id="pone.0000915-Gravitt1"></a>Gravitt PE, Peyton CL, Alessi TQ, Wheeler CM, Coutlee F, et al.  (2000) Improved amplification of genital human papillomaviruses. J Clin Microbiol  38: 357–361.  <ul class="find" data-citedArticleID="994168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Improved+amplification+of+genital+human+papillomaviruses.&amp;auth=&amp;atitle=Improved+amplification+of+genital+human+papillomaviruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+amplification+of+genital+human+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+amplification+of+genital+human+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000915-Chavali1" id="pone.0000915-Chavali1"></a>Chavali S, Mahajan A, Tabassum R, Maiti S, Bharadwaj D (2005) Oligonucleotide properties determination and primer designing: a critical examination of predictions. Bioinformatics  21: 3918–3925.  <ul class="find" data-citedArticleID="994144" data-doi="10.1093/bioinformatics/bti633"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti633" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oligonucleotide+properties+determination+and+primer+designing%3A+a+critical+examination+of+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oligonucleotide+properties+determination+and+primer+designing%3A+a+critical+examination+of+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000915-BenDov1" id="pone.0000915-BenDov1"></a>Ben-Dov E, Shapiro OH, Siboni N, Kushmaro A (2006) Advantage of using inosine at the 3′ termini of 16S rRNA gene universal primers for the study of microbial diversity. Appl Environ Microbiol  72: 6902–6906.  <ul class="find" data-citedArticleID="994132" data-doi="10.1128/aem.00849-06"><li><a href="http://dx.doi.org/10.1128/aem.00849-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advantage+of+using+inosine+at+the+3%26%23x2032%3B+termini+of+16S+rRNA+gene+universal+primers+for+the+study+of+microbial+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advantage+of+using+inosine+at+the+3%26%23x2032%3B+termini+of+16S+rRNA+gene+universal+primers+for+the+study+of+microbial+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000915-Alekshun1" id="pone.0000915-Alekshun1"></a>Alekshun MN, Levy SB (2007) Molecular mechanisms of antibacterial multidrug resistance. Cell  128: 1037–1050.  <ul class="find" data-citedArticleID="994128" data-doi="10.1016/j.cell.2007.03.004"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.03.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+mechanisms+of+antibacterial+multidrug+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+mechanisms+of+antibacterial+multidrug+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000915-Ji1" id="pone.0000915-Ji1"></a>Ji H, Kumm J, Zhang M, Farnam K, Salari K, et al.  (2006) Molecular inversion probe analysis of gene copy alterations reveals distinct categories of colorectal carcinoma. Cancer Res  66: 7910–7919.  <ul class="find" data-citedArticleID="994178" data-doi="10.1158/0008-5472.can-06-0595"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-0595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+inversion+probe+analysis+of+gene+copy+alterations+reveals+distinct+categories+of+colorectal+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+inversion+probe+analysis+of+gene+copy+alterations+reveals+distinct+categories+of+colorectal+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000915-Hartley1" id="pone.0000915-Hartley1"></a>Hartley JL, Temple GF, Brasch MA (2000) DNA cloning using in vitro site-specific recombination. Genome Res  10: 1788–1795.  <ul class="find" data-citedArticleID="994174" data-doi="10.1101/gr.143000"><li><a href="http://dx.doi.org/10.1101/gr.143000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+cloning+using+in+vitro+site-specific+recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+cloning+using+in+vitro+site-specific+recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000915-Dressman1" id="pone.0000915-Dressman1"></a>Dressman D, Yan H, Traverso G, Kinzler KW, Vogelstein B (2003) Transforming single DNA molecules into fluorescent magnetic particles for detection and enumeration of genetic variations. Proc Natl Acad Sci U S A  100: 8817–8822.  <ul class="find" data-citedArticleID="994154" data-doi="10.1073/pnas.1133470100"><li><a href="http://dx.doi.org/10.1073/pnas.1133470100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transforming+single+DNA+molecules+into+fluorescent+magnetic+particles+for+detection+and+enumeration+of+genetic+variations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transforming+single+DNA+molecules+into+fluorescent+magnetic+particles+for+detection+and+enumeration+of+genetic+variations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000915-Dafforn1" id="pone.0000915-Dafforn1"></a>Dafforn A, Chen P, Deng G, Herrler M, Iglehart D, et al.  (2004) Linear mRNA amplification from as little as 5 ng total RNA for global gene expression analysis. Biotechniques  37: 854–857.  <ul class="find" data-citedArticleID="994148"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Linear+mRNA+amplification+from+as+little+as+5+ng+total+RNA+for+global+gene+expression+analysis.&amp;auth=&amp;atitle=Linear+mRNA+amplification+from+as+little+as+5+ng+total+RNA+for+global+gene+expression+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linear+mRNA+amplification+from+as+little+as+5+ng+total+RNA+for+global+gene+expression+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linear+mRNA+amplification+from+as+little+as+5+ng+total+RNA+for+global+gene+expression+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000915-Notomi1" id="pone.0000915-Notomi1"></a>Notomi T, Okayama H, Masubuchi H, Yonekawa T, Watanabe K, et al.  (2000) Loop-mediated isothermal amplification of DNA. Nucleic Acids Res  28: E63.  <ul class="find" data-citedArticleID="994196" data-doi="10.1093/nar/28.12.e63"><li><a href="http://dx.doi.org/10.1093/nar/28.12.e63" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loop-mediated+isothermal+amplification+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loop-mediated+isothermal+amplification+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000915-Mitani1" id="pone.0000915-Mitani1"></a>Mitani Y, Lezhava A, Kawai Y, Kikuchi T, Oguchi-Katayama A, et al.  (2007) Rapid SNP diagnostics using asymmetric isothermal amplification and a new mismatch-suppression technology. Nat Methods  4: 257–262.  <ul class="find" data-citedArticleID="994190" data-doi="10.1038/nmeth1007"><li><a href="http://dx.doi.org/10.1038/nmeth1007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+SNP+diagnostics+using+asymmetric+isothermal+amplification+and+a+new+mismatch-suppression+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+SNP+diagnostics+using+asymmetric+isothermal+amplification+and+a+new+mismatch-suppression+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000915-Margulies1" id="pone.0000915-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="994188"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000915-Li1" id="pone.0000915-Li1"></a>Li M, Diehl F, Dressman D, Vogelstein B, Kinzler KW (2006) BEAMing up for detection and quantification of rare sequence variants. Nat Methods  3: 95–97.  <ul class="find" data-citedArticleID="994180" data-doi="10.1038/nmeth850"><li><a href="http://dx.doi.org/10.1038/nmeth850" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BEAMing+up+for+detection+and+quantification+of+rare+sequence+variants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BEAMing+up+for+detection+and+quantification+of+rare+sequence+variants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000915-Bennett1" id="pone.0000915-Bennett1"></a>Bennett ST, Barnes C, Cox A, Davies L, Brown C (2005) Toward the 1,000 dollars human genome. Pharmacogenomics  6: 373–382.  <ul class="find" data-citedArticleID="994134" data-doi="10.1517/14622416.6.4.373"><li><a href="http://dx.doi.org/10.1517/14622416.6.4.373" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+the+1%2C000+dollars+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+the+1%2C000+dollars+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000915-Shendure1" id="pone.0000915-Shendure1"></a>Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, et al.  (2005) Accurate multiplex polony sequencing of an evolved bacterial genome. Science  309: 1728–1732.  <ul class="find" data-citedArticleID="994202" data-doi="10.1126/science.1117389"><li><a href="http://dx.doi.org/10.1126/science.1117389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000915-Braslavsky1" id="pone.0000915-Braslavsky1"></a>Braslavsky I, Hebert B, Kartalov E, Quake SR (2003) Sequence information can be obtained from single DNA molecules. Proc Natl Acad Sci U S A  100: 3960–3964.  <ul class="find" data-citedArticleID="994140" data-doi="10.1073/pnas.0230489100"><li><a href="http://dx.doi.org/10.1073/pnas.0230489100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+information+can+be+obtained+from+single+DNA+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+information+can+be+obtained+from+single+DNA+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000915-Jarvius1" id="pone.0000915-Jarvius1"></a>Jarvius J, Melin J, Goransson J, Stenberg J, Fredriksson S, et al.  (2006) Digital quantification using amplified single-molecule detection. Nat Methods  3: 725–727.  <ul class="find" data-citedArticleID="994176" data-doi="10.1038/nmeth916"><li><a href="http://dx.doi.org/10.1038/nmeth916" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Digital+quantification+using+amplified+single-molecule+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Digital+quantification+using+amplified+single-molecule+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000915-Binladen1" id="pone.0000915-Binladen1"></a>Binladen J, Gilbert MT, Bollback JP, Panitz F, Bendixen C, et al.  (2007) The use of coded PCR primers enables high-throughput sequencing of multiple homolog amplification products by 454 parallel sequencing. PLoS ONE  2: e197.  <ul class="find" data-citedArticleID="994136" data-doi="10.1371/journal.pone.0000197"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+coded+PCR+primers+enables+high-throughput+sequencing+of+multiple+homolog+amplification+products+by+454+parallel+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+coded+PCR+primers+enables+high-throughput+sequencing+of+multiple+homolog+amplification+products+by+454+parallel+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000915-Borodina1" id="pone.0000915-Borodina1"></a>Borodina TA, Lehrach H, Soldatov AV (2003) Ligation-based synthesis of oligonucleotides with block structure. Anal Biochem  318: 309–313.  <ul class="find" data-citedArticleID="994138" data-doi="10.1016/s0003-2697(03)00250-1"><li><a href="http://dx.doi.org/10.1016/s0003-2697(03)00250-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ligation-based+synthesis+of+oligonucleotides+with+block+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ligation-based+synthesis+of+oligonucleotides+with+block+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.XML" value="84662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.PDF" value="212811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g001.PNG_L" value="362651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g001.PNG_M" value="74864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g001.PNG_S" value="9472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g001.TIF" value="689566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g001.PNG_I" value="26208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g002.PNG_L" value="302044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g002.PNG_M" value="48590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g002.PNG_S" value="5463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g002.TIF" value="1061804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g002.PNG_I" value="18618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.t001.PNG_L" value="112203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.t001.PNG_M" value="67231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.t001.PNG_S" value="8867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.t001.TIF" value="432868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.t001.PNG_I" value="21178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g003.PNG_L" value="1109864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g003.PNG_M" value="154415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g003.PNG_S" value="11385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g003.TIF" value="2253404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.g003.PNG_I" value="52711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s001.PDF" value="55649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s002.PDF" value="206387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s003.PDF" value="298640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s004.PDF" value="644735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s005.PDF" value="55028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s006.PDF" value="54893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000915.s007.PDF" value="93807"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000915&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000915" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000915&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000915&volume=&issue=&title=Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications&author_name=Michael%20S.%20Akhras%2C%20Magnus%20Unemo%2C%20Sreedevi%20Thiyagarajan%2C%20P%C3%A5l%20Nyr%C3%A9n%2C%20Ronald%20W.%20Davis%2C%20Andrew%20Z.%20Fire%2C%20Nader%20Pourmand&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000915" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915&amp;t=Connector%20Inversion%20Probe%20Technology%3A%20A%20Powerful%20One-Primer%20Multiplex%20DNA%20Amplification%20System%20for%20Numerous%20Scientific%20Applications" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915&title=Connector%20Inversion%20Probe%20Technology%3A%20A%20Powerful%20One-Primer%20Multiplex%20DNA%20Amplification%20System%20for%20Numerous%20Scientific%20Applications&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915&amp;title=Connector%20Inversion%20Probe%20Technology%3A%20A%20Powerful%20One-Primer%20Multiplex%20DNA%20Amplification%20System%20for%20Numerous%20Scientific%20Applications" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000915&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Connector Inversion Probe Technology: A Powerful One-Primer Multiplex DNA Amplification System for Numerous Scientific Applications';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000915';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Connector%20Inversion%20Probe%20Technology%3A%20A%20Powerful%20One-Primer%20Multiplex%20DNA%20Amplification%20System%20for%20Numerous%20Scientific%20Applications http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000915" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000915" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24668"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HPV-16%22" title="Search for articles in the subject area:'HPV-16'"><div class="flagText">HPV-16</div></a>
              <div data-categoryid="45139" data-articleid="24668"
                   data-categoryname="HPV-16"
                   class="flagImage" title="Flag 'HPV-16' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+papillomavirus+infection%22" title="Search for articles in the subject area:'Human papillomavirus infection'"><div class="flagText">Human papillomavirus infection</div></a>
              <div data-categoryid="48361" data-articleid="24668"
                   data-categoryname="Human papillomavirus infection"
                   class="flagImage" title="Flag 'Human papillomavirus infection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="24668"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24668"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neisseria+gonorrhoeae%22" title="Search for articles in the subject area:'Neisseria gonorrhoeae'"><div class="flagText">Neisseria gonorrhoeae</div></a>
              <div data-categoryid="43649" data-articleid="24668"
                   data-categoryname="Neisseria gonorrhoeae"
                   class="flagImage" title="Flag 'Neisseria gonorrhoeae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24668"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24668"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4745'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2691'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6108&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
