

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000858"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000858" />

    <meta name="citation_title" content="Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein"/>
    <meta itemprop="name" content="Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein"/>

      <meta name="citation_author" content="Li-Tzu Chen"/>
            <meta name="citation_author_institution" content="Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan"/>
            <meta name="citation_author_institution" content="Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan"/>
      <meta name="citation_author" content="Tzu-Ping Ko"/>
            <meta name="citation_author_institution" content="Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan"/>
      <meta name="citation_author" content="Yu-Wei Chang"/>
            <meta name="citation_author_institution" content="Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan"/>
            <meta name="citation_author_institution" content="Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan"/>
      <meta name="citation_author" content="Kuei-An Lin"/>
            <meta name="citation_author_institution" content="Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan"/>
      <meta name="citation_author" content="Andrew H.-J. Wang"/>
            <meta name="citation_author_institution" content="Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan"/>
            <meta name="citation_author_institution" content="Department of Life Sciences, National Taiwan University, Taipei, Taiwan"/>
            <meta name="citation_author_institution" content="Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan"/>
            <meta name="citation_author_institution" content="National Core Facility of High-Throughput Protein Crystallography, Academia Sinica, Taipei, Taiwan"/>
      <meta name="citation_author" content="Ting-Fang Wang"/>
            <meta name="citation_author_institution" content="Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan"/>
            <meta name="citation_author_institution" content="Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan"/>

    <meta name="citation_date" content="2007/9/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000858.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e858"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Homologous pairing and DNA strand-exchange proteins.; citation_author=SC Kowalczykowski; citation_author=AK Eggleston; citation_journal_title=Annu Rev Biochem; citation_volume=63; citation_number=1; citation_pages=991-1043; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The bacterial RecA protein as a motor protein.; citation_author=MM Cox; citation_journal_title=Annu Rev Microbiol; citation_volume=57; citation_number=2; citation_pages=551-577; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular views of recombination proteins and their control.; citation_author=SC West; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=4; citation_number=3; citation_pages=435-445; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Rad51-deficient vertebrate cells accumulate chromosomal breaks prior to cell death.; citation_author=E Sonoda; citation_author=MS Sasaki; citation_author=JM Buerstedde; citation_author=O Bezzubova; citation_author=A Shinohara; citation_journal_title=EMBO J; citation_volume=17; citation_number=4; citation_pages=598-608; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.; citation_author=DK Bishop; citation_journal_title=Cell; citation_volume=79; citation_number=5; citation_pages=1081-1092; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Roles for two RecA homologs in promoting meiotic chromosome synapsis.; citation_author=B Rockmill; citation_author=M Sym; citation_author=H Scherthan; citation_author=GS Roeder; citation_journal_title=Genes Dev; citation_volume=9; citation_number=6; citation_pages=2684-2695; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Homologous recombination and cell cycle checkpoints: Rad51 in tumour progression and therapy resistance.; citation_author=W Henning; citation_author=HW Sturzbecher; citation_journal_title=Toxicology; citation_volume=193; citation_number=7; citation_pages=91-109; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Interactions between human BRCA2 protein and the meiosis-specific recombinase DMC1.; citation_author=T Thorslund; citation_author=F Esashi; citation_author=SC West; citation_journal_title=EMBO J; citation_volume=26; citation_number=8; citation_pages=2915-2922; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structure of the recA protein-ADP complex.; citation_author=RM Story; citation_author=TA Steitz; citation_journal_title=Nature; citation_volume=355; citation_number=9; citation_pages=374-376; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Full-length archaeal Rad51 structure and mutants: mechanisms for RAD51 assembly and control by BRCA2.; citation_author=DS Shin; citation_author=L Pellegrini; citation_author=DS Daniels; citation_author=B Yelent; citation_author=L Craig; citation_journal_title=EMBO J; citation_volume=22; citation_number=10; citation_pages=4566-4576; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis for octameric ring formation and DNA interaction of the human homologous-pairing protein Dmc1.; citation_author=T Kinebuchi; citation_author=W Kagawa; citation_author=R Enomoto; citation_author=K Tanaka; citation_author=K Miyagawa; citation_journal_title=Mol Cell; citation_volume=14; citation_number=11; citation_pages=363-374; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of a Rad51 filament.; citation_author=AB Conway; citation_author=TW Lynch; citation_author=Y Zhang; citation_author=GS Fortin; citation_author=CW Fung; citation_journal_title=Nat Struct Mol Biol; citation_volume=11; citation_number=12; citation_pages=791-796; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of archaeal recombinase RadA: a snapshot of its extended conformation.; citation_author=Y Wu; citation_author=Y He; citation_author=IA Moya; citation_author=X Qian; citation_author=Y Luo; citation_journal_title=Mol Cell; citation_volume=15; citation_number=13; citation_pages=423-435; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of Methanococcus voltae RadA in complex with ADP: hydrolysis-induced conformational change.; citation_author=X Qian; citation_author=Y Wu; citation_author=Y He; citation_author=Y Luo; citation_journal_title=Biochemistry; citation_volume=44; citation_number=14; citation_pages=13753-13761; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conformational flexibility revealed by the crystal structure of a crenarchaeal RadA.; citation_author=A Ariza; citation_author=DJ Richard; citation_author=MF White; citation_author=CS Bond; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=15; citation_pages=1465-1473; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the left-handed archaeal RadA helical filament: identification of a functional motif for controlling quaternary structures and enzymatic functions of RecA family proteins.; citation_author=LT Chen; citation_author=TP Ko; citation_author=YC Chang; citation_author=KA Lin; citation_author=CS Chang; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=16; citation_pages=1787-1801; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Insights into DNA recombination from the structure of a RAD51-BRCA2 complex.; citation_author=L Pellegrini; citation_author=DS Yu; citation_author=T Lo; citation_author=S Anand; citation_author=M Lee; citation_journal_title=Nature; citation_volume=420; citation_number=17; citation_pages=287-293; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The N-terminal domain of the human Rad51 protein binds DNA: structure and a DNA binding surface as revealed by NMR.; citation_author=H Aihara; citation_author=Y Ito; citation_author=H Kurumizaka; citation_author=S Yokoyama; citation_author=T Shibata; citation_journal_title=J Mol Biol; citation_volume=290; citation_number=18; citation_pages=495-504; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Role of the N-terminal domain of the human DMC1 protein in octamer formation and DNA binding.; citation_author=T Kinebuchi; citation_author=W Kagawa; citation_author=H Kurumizaka; citation_author=S Yokoyama; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=19; citation_pages=28382-28387; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Motoring along with the bacterial RecA protein.; citation_author=MM Cox; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=8; citation_number=20; citation_pages=127-138; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of an ATPase-active form of Rad51 homolog from Methanococcus voltae. Insights into potassium dependence.; citation_author=Y Wu; citation_author=X Qian; citation_author=Y He; citation_author=IA Moya; citation_author=Y Luo; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=21; citation_pages=722-728; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Self-polymerization of archaeal RadA protein into long and fine helical filaments.; citation_author=MH Lee; citation_author=CH Leng; citation_author=YC Chang; citation_author=CC Chou; citation_journal_title=Biochem Biophys Res Commun; citation_volume=323; citation_number=22; citation_pages=845-851; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Image-analysis reveals that Escherichia coli RecA protein consists of 2 domains.; citation_author=X Yu; citation_author=EH Egelman; citation_journal_title=Biophy J; citation_volume=57; citation_number=23; citation_pages=555-566; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=RadA protein is an archaeal RecA protein homolog that catalyzes DNA strand exchange.; citation_author=EM Seitz; citation_author=JP Brockman; citation_author=SJ Sandler; citation_author=AJ Clark; citation_author=SC Kowalczykowski; citation_journal_title=Genes Dev; citation_volume=12; citation_number=24; citation_pages=1248-1253; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Atomic force microscopic study of aggregation of RecA-DNA nucleoprotein filaments into left-handed supercoiled bundles.; citation_author=WX Shi; citation_author=RG Larson; citation_journal_title=Nano Lett; citation_volume=5; citation_number=25; citation_pages=2476-2481; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The hRad51 and RecA proteins show significant differences in cooperative binding to single-stranded DNA.; citation_author=JK De Zutter; citation_author=KL Knight; citation_journal_title=J Mol Biol; citation_volume=293; citation_number=26; citation_pages=769-780; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Common fold in helix-hairpin-helix proteins.; citation_author=X Shao; citation_author=NV Grishin; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=27; citation_pages=2643-2650; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Structures of ternary complexes of rat DNA polymerase beta, a DNA template-primer, and ddCTP.; citation_author=H Pelletier; citation_author=MR Sawaya; citation_author=A Kumar; citation_author=SH Wilson; citation_author=J Kraut; citation_journal_title=Science; citation_volume=264; citation_number=28; citation_pages=1891-1903; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=DNA bending and a flip-out mechanism for base excision by the helix-hairpin-helix DNA glycosylase, Escherichia coli AlkA.; citation_author=T Hollis; citation_author=Y Ichikawa; citation_author=T Ellenberger; citation_journal_title=EMBO J; citation_volume=19; citation_number=29; citation_pages=758-766; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Direct visualization of a DNA glycosylase searching for damage.; citation_author=L Chen; citation_author=KA Haushalter; citation_author=CM Lieber; citation_author=GL Verdine; citation_journal_title=Chem Biol; citation_volume=9; citation_number=30; citation_pages=345-350; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gly-103 in the N-terminal domain of Saccharomyces cerevisiae Rad51 protein is critical for DNA binding.; citation_author=XP Zhang; citation_author=KI Lee; citation_author=JA Solinger; citation_author=K Kiianitsa; citation_author=WD Heyer; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=31; citation_pages=26303-26311; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The ATP synthase - A splendid molecular machine.; citation_author=PD Boyer; citation_journal_title=Annu Rev Biochem; citation_volume=66; citation_number=32; citation_pages=717-749; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Processing of X-ray diffraction data collected in oscillation mode.; citation_author=Z Otwinowski; citation_author=W Minor; citation_journal_title=Macromol Crystallogr; citation_volume=276; citation_number=33; citation_pages=307-326; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Crystallography & NMR system: A new software suite for macromolecular structure determination.; citation_author=AT Brunger; citation_author=PD Adams; citation_author=GM Clore; citation_author=WL DeLano; citation_author=P Gros; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=54; citation_number=34; citation_pages=905-921; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Improved methods for building protein models in electron density maps and the location of errors in these models.; citation_author=TA Jones; citation_author=JY Zou; citation_author=SW Cowan; citation_author=M Kjeldgaard; citation_journal_title=Acta Crystallogr A; citation_volume=47; citation_number=35; citation_pages=110-119; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The CCP4 suite: programs for protein crystallography.; citation_author= CCP4; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=50; citation_number=36; citation_pages=760-763; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Heterodimeric complexes of Hop2 and Mnd1 function with Dmc1 to promote meiotic homolog juxtaposition and strand assimilation.; citation_author=YK Chen; citation_author=CH Leng; citation_author=H Olivares; citation_author=MH Lee; citation_author=YC Chang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=37; citation_pages=10572-10577; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein"/>
    <meta name="twitter:description" content="RecA family proteins engage in an ATP-dependent DNA strand exchange reaction that includes a ssDNA nucleoprotein helical filament and a homologous dsDNA sequence. In spite of more than 20 years of efforts, the molecular mechanism of homology pairing and strand exchange is still not fully understood. Here we report a crystal structure of Sulfolobus solfataricus RadA overwound right-handed filament with three monomers per helical pitch. This structure reveals conformational details of the first ssDNA binding disordered loop (denoted L1 motif) and the dsDNA binding N-terminal domain (NTD). L1 and NTD together form an outwardly open palm structure on the outer surface of the helical filament. Inside this palm structure, five conserved basic amino acid residues (K27, K60, R117, R223 and R229) surround a 25 &Aring; pocket that is wide enough to accommodate anionic ssDNA, dsDNA or both. Biochemical analyses demonstrate that these five positively charged residues are essential for DNA binding and for RadA-catalyzed D-loop formation. We suggest that the overwound right-handed RadA filament represents a functional conformation in the homology search and pairing reaction. A new structural model is proposed for the homologous interactions between a RadA-ssDNA nucleoprotein filament and its dsDNA target."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000858.g008"/>

  <meta property="og:title" content="Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6303'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3646'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2394&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000858">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000858" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000858&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Li-Tzu Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan, 
                  Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tzu-Ping Ko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yu-Wei Chang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan, 
                  Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kuei-An Lin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew H.-J. Wang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ahjwang@gate.sinica.edu.tw">ahjwang@gate.sinica.edu.tw</a> (AW); <a href="mailto:tfwang@gate.sinica.edu.tw">tfwang@gate.sinica.edu.tw</a> (TW)</p>

                <p>Affiliations:
                  Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan, 
                  Department of Life Sciences, National Taiwan University, Taipei, Taiwan, 
                  Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan, 
                  National Core Facility of High-Throughput Protein Crystallography, Academia Sinica, Taipei, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ting-Fang Wang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ahjwang@gate.sinica.edu.tw">ahjwang@gate.sinica.edu.tw</a> (AW); <a href="mailto:tfwang@gate.sinica.edu.tw">tfwang@gate.sinica.edu.tw</a> (TW)</p>

                <p>Affiliations:
                  Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan, 
                  Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000858</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000858-g001" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-t001" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g002" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g003" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g004" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g005" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g006" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g007" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000858-g008" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000858">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000858" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>RecA family proteins engage in an ATP-dependent DNA strand exchange reaction that includes a ssDNA nucleoprotein helical filament and a homologous dsDNA sequence. In spite of more than 20 years of efforts, the molecular mechanism of homology pairing and strand exchange is still not fully understood. Here we report a crystal structure of <em>Sulfolobus solfataricus</em> RadA overwound right-handed filament with three monomers per helical pitch. This structure reveals conformational details of the first ssDNA binding disordered loop (denoted L1 motif) and the dsDNA binding N-terminal domain (NTD). L1 and NTD together form an outwardly open palm structure on the outer surface of the helical filament. Inside this palm structure, five conserved basic amino acid residues (K27, K60, R117, R223 and R229) surround a 25 Å pocket that is wide enough to accommodate anionic ssDNA, dsDNA or both. Biochemical analyses demonstrate that these five positively charged residues are essential for DNA binding and for RadA-catalyzed D-loop formation. We suggest that the overwound right-handed RadA filament represents a functional conformation in the homology search and pairing reaction. A new structural model is proposed for the homologous interactions between a RadA-ssDNA nucleoprotein filament and its dsDNA target.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Chen L-T, Ko T-P, Chang Y-W, Lin K-A, Wang AH-J, et al.  (2007) Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein. PLoS ONE 2(9):
          e858.
            doi:10.1371/journal.pone.0000858</p><p><strong>Academic Editor: </strong>Martin Egli, Vanderbilt University, United States of America</p><p><strong>Received:</strong> July 20, 2007; <strong>Accepted:</strong> August 16, 2007; <strong>Published:</strong> September 12, 2007</p><p><strong>Copyright:</strong> © 2007 Chen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by an Academia Sinica Investigator Research Award (to TFW), two Research Grants from the National Science Council (NSC96-2811-B-001-018 and NSC95-2811-B-001-058 to TFW), and a National Core Facility of High-Throughput Protein Crystallography Grant from the National Science Council (NSC93-3112-B-001-011-Y to AHJW).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The RecA family of DNA strand exchange proteins exists in all three kingdoms of life. Members of the RecA protein family include prokaryotic RecA, archaeal RadA and Rad51, and eukaryotic Rad51 and Dmc1. These proteins play a central role in homologous recombination, an error-free DNA repair mechanism <a href="#pone.0000858-Kowalczykowski1">[for reviews see 1]</a>, <a href="#pone.0000858-Cox1">[2]</a>, <a href="#pone.0000858-West1">[3]</a>. While RecA-deficient <em>E. coli</em> cells or Rad51-deficient yeast cells are viable, Rad51-deficient vertebrate cells are not. The latter accumulate chromosomal breaks prior to cell death <a href="#pone.0000858-Sonoda1">[4]</a>. Rad51 and its meiosis-specific homolog, Dmc1, are indispensable for meiosis <a href="#pone.0000858-Bishop1">[5]</a>, <a href="#pone.0000858-Rockmill1">[6]</a>. Mammalian Rad51 and Dmc1 proteins interact with tumor suppressor proteins, such as BRCA2 <a href="#pone.0000858-Henning1">[7]</a>, <a href="#pone.0000858-Thorslund1">[8]</a>. Together, RecA family proteins have important roles in cell proliferation, genome maintenance, and genetic diversity, particularly in higher eukaryotes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Most of our understanding of the mechanism of RecA family proteins comes from <em>E. coli</em> studies. In all current models, RecA molecules load onto ssDNA, forming a <em>6<sub>1</sub></em> right-handed helical filament with six monomers per helical turn in the presence of ATP and scan for homologous dsDNA. Once homology is recognized, a synaptic complex, consisting of a three-stranded DNA filament, forms. Eventually, the DNA strands are exchanged and ATP is hydrolyzed, resulting in the displacement of one of the original duplex strands, and the creation of a new heteroduplex (or D-loop) is created. The molecular mechanisms underlying these processes are not fully understood.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Members of RecA family all contain a central ATPase domain that is preceded by a short β-stranded polymerization motif (PM). The central ATPase domain not only mediates a classical ATP-induced allosteric effect that causes large-scale changes in protomers and filament assembly, it also contains two disordered loops (denoted L1 and L2 motifs) for ssDNA binding <a href="#pone.0000858-Story1">[9]</a>. The PM is responsible for the assembly of helical filaments and toroidal rings <a href="#pone.0000858-Story1">[9]</a>–<a href="#pone.0000858-Chen1">[16]</a>. In RadA, Rad51, and Dmc1, a conserved phenylalanine residue in the PM docks into a hydrophobic pocket on the neighboring core ATPase domain. A similar interaction was also observed with a fusion protein containing the human Rad51 monomer and a peptide from the BRCA2 protein <a href="#pone.0000858-Shin1">[10]</a>, <a href="#pone.0000858-Pellegrini1">[17]</a>. Immediately following the PM and before the central ATPase domain, we recently identified a highly flexible region, called subunit rotation motif (SRM). Clockwise rotation of the SRM along the axes of RadA protein polymers is responsible for quaternary structural transitions from a protein ring to a <em>6<sub>1</sub></em> right-handed filament with six monomers per helical turn, then to a <em>3<sub>1</sub></em> overwound right-handed filament with three monomers per helical turn and finally to a <em>4<sub>3</sub></em> left-handed filament with four monomers per helical turn <a href="#pone.0000858-Chen1">[see Figure 5 in 16]</a>. RadA, Rad51 or Dmc1 each has an additional N-terminal domain (NTD) that interacts with dsDNA <a href="#pone.0000858-Aihara1">[18]</a>, <a href="#pone.0000858-Kinebuchi2">[19]</a>, and RecA has a small C-terminal domain (CTD), for which a similar function has been proposed. The helix-hairpin-helix (HhH) motif in the NTD mediates dsDNA binding <a href="#pone.0000858-Aihara1">[18]</a>, <a href="#pone.0000858-Kinebuchi2">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The biochemistry of the ATP-dependent strand exchange reaction was recently reviewed in detail <a href="#pone.0000858-Cox2">[20]</a>. In some models, RecA family proteins function as DNA pairing enzymes, and ATP hydrolysis facilitates RecA dissociation and/or distribution along DNA substrates. These models do not explain the bypass of non-homologous DNA sequences during the homology pairing and strand exchange reactions. Alternatively, the facilitated DNA rotation model proposes that RecA family proteins function as motor proteins that coordinate rotation between dsDNA and ssDNA. The facilitated DNA rotation model seemed compatible with all experimental results obtained to date <a href="#pone.0000858-Cox2">[20]</a>. Despite some mechanistic differences, all current models of RecA family proteins share the principle that these proteins function as <em>6<sub>1</sub></em> right-handed helical filaments throughout their catalytic cycles, including the homology pairing and strand exchange reactions. This creates a problem for the facilitated DNA rotation model. Since all known DNA binding motifs (i.e., L1, L2, NTD, CTD) are localized along or near the central axes of <em>6<sub>1</sub></em> right-handed helical filaments <a href="#pone.0000858-Story1">[9]</a>, <a href="#pone.0000858-Conway1">[ 12]</a>–<a href="#pone.0000858-Qian1">[14]</a>, <a href="#pone.0000858-Chen1">[see Figure 3D in 16]</a>, <a href="#pone.0000858-Wu2">[21]</a>, the facilitated DNA rotation model implies that novel DNA binding sites are located on the exterior of the right-handed helical filaments to facilitate DNA rotation <a href="#pone.0000858-Cox2">[20]</a>. To date, such novel DNA binding sites have not been identified in any RecA family protein, and the mechanisms underlying protein-DNA and ssDNA-dsDNA interactions are not clear.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Recent structural studies indicate that large structural variations of RecA family protein filaments can occur. For example, X-ray crystallographic analyses revealed that <em>Sulfolobus solfataricus</em> RadA (<em>Sso</em>RadA) proteins could self-polymerize into a <em>3<sub>1</sub></em> overwound right-handed filament <a href="#pone.0000858-Ariza1">[this study, 15]</a> and a <em>4<sub>3</sub></em> left-handed filament <a href="#pone.0000858-Chen1">[16]</a>. Using atomic force microscopy (AFM) with carbon nanotube tips, we showed that <em>Sso</em>RadA <a href="#pone.0000858-Lee1">[see Figure 3B in ref. 22]</a> and budding yeast <em>S. cerevisiae</em> Dmc1 <a href="#pone.0000858-Chen1">[see supporting data Figure S1 in ref. 16]</a> could each form both right- and left-handed helical filaments in solution. In addition, EM imaging analysis also revealed that RadA protein could form filaments with both right- and left-handed helical pitches <a href="#pone.0000858-Chen1">[16]</a>. These results suggest that left-handed filaments exist not only in protein crystallization conditions but also in neutral pH solutions. Moreover, the ability to form left-handed helical filaments is likely a general property of most RecA family proteins.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>It is noteworthy that other investigators might have noted left-handed RecA family protein filaments. Yu and Egelman first reported in 1990 that <em>E. coli</em> RecA filaments formed on linear dsDNA in the presence of ATP and aluminum fluoride showed both right- and left-handed helical pitches <a href="#pone.0000858-Yu1">[23]</a>. Aluminum fluoride is able to substitute for phosphate after the hydrolysis of ATP, and it was used to trap the ADP-Pi state of the RecA protein. However, after the report of the crystal structure of <em>E. coli</em> RecA <em>6<sub>1</sub></em> right-handed protein filament <a href="#pone.0000858-Story1">[9]</a>, this filament form became generally accepted as the active form of RecA protein. The left-handed helical filaments were then judged to be experimental artifacts due to the negative staining EM protocols. Specifically, it was postulated that visual impression of the left-handed filaments might result from deformation of flexible filaments during the absorption to EM grids. Such deformations may break the axial symmetry and cause the superposition of the signals from the upper and lower part of the filament, thus giving a visual impression of inclined striation. Although this interpretation seems reasonable, it also has pitfalls. First, it arose from a preconception that RecA and RecA family proteins could only form helical filaments with right-handed pitches. Second, to our knowledge, it was not verified by other experimental approaches, e.g., AFM or X-ray crystallography. As described above, we have recently reported the crystal structure of <em>Sso</em>RadA left-handed helical filament. In addition, using the AFM imaging approach, we <a href="#pone.0000858-Chen1">[16]</a>, <a href="#pone.0000858-Lee1">[22]</a> and others (see below) have observed left-handed helical filaments of various RecA family proteins. For example, the cover picture of the May 1, 1998 issue of <em>Genes and Development</em> shows an AFM image of a left-handed helical filament made by the <em>Sso</em>RadA proteins; however, the paper described only right-handed filaments <a href="#pone.0000858-Seitz1">[24]</a>. It would be interesting to know if the image was inadvertently mirrored. Moreover, another recent AFM imaging study revealed a single RecA-dsDNA helical filament with both right- and left-handed helical pitches <a href="#pone.0000858-Shi1">[Figure 1E in 25]</a>. These results together, raise the possibility that the <em>6<sub>1</sub></em> right-handed helical filaments may not be the only functional conformation for RecA family proteins.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>In this study, we present the crystal structure of <em>Sso</em>RadA <em>3<sub>1</sub></em> overwound right-handed filament at 1.93Å resolution. Although this structure is similar to that reported previously <a href="#pone.0000858-Ariza1">[15]</a>, it reveals unprecedented geometric/conformational details in the L1 motif and the NTD due to its higher resolution. Subsequent biochemical analyses indicate that five conserved basic amino acid residues (K27, K60, R217, R227, R229) in the L1 motif and the NTD are involved in DNA binding and in promoting formation of D-loop between a ssDNA and its complementary strand in the dsDNA target. We describe these results and propose a new structural model for DNA binding and homologous pairing.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Overall Structure</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In the final 1.93Å resolution crystal structure, <em>Sso</em>RadA protomers are packed into three extended helical filaments in a 98Å pitch helix with three protomers per turn (<a href="#pone-0000858-g001">Figure 1A and 1B</a>). Chain A is located at the origin of the unit cell, whereas chains B and C are located one-third and two-thirds diagonal to the unit cell, respectively (<a href="#pone-0000858-g001">Figure 1A</a>). Chains B and C run in opposite directions. This arrangement is similar to the recently reported <em>P3<sub>1</sub>21</em> form at 3.2 Å resolution <a href="#pone.0000858-Ariza1">[15]</a>. Chains B and C have average B-values of 38.6 Å<sup>2</sup> and 38.1 Å<sup>2</sup>, respectively. In contrast, chain A does not contain a visible NTD and has an average B-value of 48.2 Å<sup>2</sup> (<a href="#pone-0000858-t001">Table 1</a>). A portion of chain A filaments may run in the opposite direction or has partial occupancy.</p>
<div class="figure" id="pone-0000858-g001"><div class="img"><a name="pone-0000858-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Crystal packing and quaternary structures.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) <em>Sso</em>RadA protomers packed into three extended helical filaments. Chain A was located at the origin of the unit cell, whereas chains B and C were located one-third and two-third diagonal to the unit cell. (B) Side view of the <em>Sso</em>RadA right-handed helical filament crystal structure. The helical pitch of the filament is 98 Å. Each protomer is shown in a different color. The N-terminal domain (NTD), polymerization motif (PM), and central ATPase domain are indicated. (C) The Phe73 of the PM is buried in the hydrophobic pocket of the neighboring ATPase domain. Several hydrophobic residues that interact with the Phe73 side chain are indicated. The interactions result in the assembly of <em>Sso</em>RadA protomers into a filament. 2<em>F</em><sub>o</sub>–<em>F</em><sub>c</sub> electron density maps (contoured at 1.0 σ), corresponding to the PM are shown in orange.</p>
<span>doi:10.1371/journal.pone.0000858.g001</span></div><div class="figure" id="pone-0000858-t001"><div class="img"><a name="pone-0000858-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Data collection and refinement statistics for the RadA crystal.</span></strong></p><span>doi:10.1371/journal.pone.0000858.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>As in other filamentous structures of archaeal and eukaryotic RecA family proteins <a href="#pone.0000858-Conway1">[12]</a>–<a href="#pone.0000858-Chen1">[16]</a>, a conserved amino acid residue (Phe73) in the PM is responsible for the assembly of the <em>3<sub>1</sub></em> overwound right-handed <em>Sso</em>RadA filament. Phe73 is docked into a hydrophobic pocket (which includes Val145, Tyr177, Ile190, Asp193, Leu194, and Leu197) of the neighboring protomer (<a href="#pone-0000858-g001">Figure 1C</a>). Intriguingly, the overwound right-handed <em>Sso</em>RadA filament contains ordered polypeptide backbones and side-chains in two important structural regions, L1 motif (amino acid residues 219–226) and NTD (amino acid residues 14–69), compared to those found in the previous lower resolution <em>P3<sub>1</sub>21</em> form. These new structural details allow better modelling of protein-DNA and ssDNA-dsDNA interactions.</p>


<h4>Structure of the L1 motif</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A previous structure of the <em>Methanococcus voltae</em> RadA (<em>Mv</em>RadA) <em>6<sub>1</sub></em> right-handed helical filament at 2.4 Å resolution indicated that L1 and L2 form positively charged patches compatible for binding anionic DNA <a href="#pone.0000858-Wu2">[21]</a>. In our <em>Sso</em>RadA <em>3<sub>1</sub></em> overwound right-handed helical filament, the L1 motif (<a href="#pone-0000858-g002">Figure 2A</a>) is located between the α11 and α12 helices (<a href="#pone-0000858-g002">Figure 2B</a>). A surface charge potential analysis of the L1 motif reveals a linear basic patch on one face of the motif (<a href="#pone-0000858-g002">Figure 2C</a>). This linear basic patch of <em>Sso</em>RadA L1 motif consists of three arginine residues (i.e., Arg217, Arg223, and Arg229), with their positively charged side chains directed upward. Intriguingly, Arg217, Arg223, and Arg229 are conserved in all archaeal and eukaryotic RecA family proteins (<a href="#pone-0000858-g002">Figure 2A</a>) but not in <em>E. coli</em> RecA protein. This difference may explain the previous observation that human Rad51 and <em>E. coli</em> RecA exhibit profound mechanistic differences in ssDNA binding <a href="#pone.0000858-DeZutter1">[26]</a>.</p>
<div class="figure" id="pone-0000858-g002"><div class="img"><a name="pone-0000858-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Structure of the L1 motif.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Sequence alignment of RadA homologs from <em>S. solfataricus</em> (<em>Sso)</em> RadA, <em>M. voltae</em> (<em>Mv</em>) RadA, <em>P. furiosus</em> (<em>Pf</em>) Rad51, <em>H. sapiens</em> (<em>Hs)</em> Rad51 and Dmc1, and <em>S. cerevisiae</em> (<em>Sc</em>) Rad51 and Dmc1. Three conserved arginine residues are shown in cyan. (B) A ribbon diagram of the L1 motif showing two alpha-helices (grey). The hinge region is depicted with a ball-and-stick model (yellow). The side chains of three conserved arginine residues are shown in green. 2<em>F</em><sub>o</sub>–<em>F</em><sub>c</sub> electron density maps (contoured at 1.0σ), corresponding to the ssDNA binging site, are shown in cyan. (C) Surface charge potential of the L1 motif. The positively and negatively charged regions are indicated in blue and red, respectively. The linear basic patch is ~18 Å in length.</p>
<span>doi:10.1371/journal.pone.0000858.g002</span></div>

<h4>Structure of the NTD</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The NTD is actually an (HhH)<sub>2</sub> domain, in which a pseudo two-fold unit is composed of two HhH motifs linked by an α-helix <a href="#pone.0000858-Shao1">[27]</a>. The HhH motifs and the connector α-helix are denoted as H1'h'H2', H1hH2, and Hc (<a href="#pone-0000858-g003">Figure 3A</a>). Each HhH motif contains two helices (denoted as H1, H1', H2, or H2') and a hairpin (denoted as h or h') (<a href="#pone-0000858-g003">Figure 3B</a>). Alignment of the archaeal RadA and eukaryotic Dmc1 and Rad51 protein sequences clearly shows the conservation of a GΦ pattern in the two hairpins, where G is glycine and Φ is a hydrophobic residue (e.g., Ile, Val, or Leu) (<a href="#pone-0000858-g003">Figure 3A</a>). The GΦ pattern completes the hydrophobic core within each HhH motif. In contrast, the connector helix, Hc, links the two HhH motifs and completes the hydrophobic core of the entire (HhH)<sub>2</sub> domain (<a href="#pone-0000858-g003">Figure 3C</a>).</p>
<div class="figure" id="pone-0000858-g003"><div class="img"><a name="pone-0000858-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Structure of the N-terminal domain (NTD).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Sequence alignment of the NTD and (HhH)<sub>2</sub> domains in RadA homologs from <em>S. solfataricus</em>(<em>Sso</em>) RadA), <em>M. voltae</em> (<em>Mv</em>) RadA, <em>P. furiosus</em> (<em>Pf</em>) Rad51, <em>H. sapiens</em>(<em>Hs</em>) Rad51 and Dmc1, and <em>S. cerevisiae</em> (<em>Sc</em>) Rad51 and Dmc1. The first and second HhH motifs are denoted as H1'h'H2' and H1hH2, respectively. (B) Cartoon diagram of the (HhH)<sub>2</sub> domains. The first and second HhH motifs are connected by the connector alpha-helix Hc (yellow). (C) A ribbon diagram of the NTD. The hinge regions (h' and h) are depicted with a ball-and-stick model (green). 2<em>F</em><sub>o</sub>–<em>F</em><sub>c</sub> electron density maps (contoured at 1.0 σ) of several key amino acid residues are in purple. Oxygen and nitrogen atoms are shown in red and blue, respectively. (D) Surface charge potential of the NTD. The positively and negatively charged regions are indicated by blue and red, respectively. The two borders of this 92° arched basic patch are 15 Å and 14 Å in length, respectively. (E) Top view of the NTD surface charge potential reveals a central channel that is 18 Å long and 14 Å wide. The channel is too narrow to accommodate a B-type dsDNA substrate.</p>
<span>doi:10.1371/journal.pone.0000858.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Previous studies of other enzymes involved in DNA metabolism (e.g., rat polymerase β, <em>E. coli</em> AlkA base excision repair glycosylase, and human 8-oxoguanine DNA glycolysase) revealed that their (HhH)<sub>2</sub> domains are capable of inducing dsDNA bending and led to distortion and/or flipping of base pairs <a href="#pone.0000858-Pelletier1">[28]</a>–<a href="#pone.0000858-Chen2">[30]</a>. Surface potential analysis of the entire (HhH)<sub>2</sub> domain of <em>Sso</em>RadA protein revealed a 92° arched basic patch along the border of the second HhH motif (<a href="#pone-0000858-g003">Figure 3D</a>). Two lysine residues (i.e., Lys27 and Lys60) are located at each end of this arched basic patch. Lys27 is part of the first HhH motif, and its amino group contacts the carbonyl group of Gly52 (<a href="#pone-0000858-g003">Figure 3C</a>). In contrast, Gly52 and Lys60 are part of the second HhH motif. Importantly, the Lys27, Gly52, and Lys60 residues are conserved in other archaeal RadA proteins and eukaryotic Rad51 and Dmc1 proteins (<a href="#pone-0000858-g003">Figure 3A</a>). The corresponding residues in human Rad51 are Lys40, Gly65, and Lys73. A previous nuclear magnetic resonance study revealed that Gly65 and Lys73 of human Rad51 are directly involved in dsDNA binding <a href="#pone.0000858-Aihara1">[18]</a>. Conceivably, RadA, Rad51, and Dmc1 may utilize this arched basic patch for non sequence-specific interactions with anionic dsDNA. A top view of the arched basic patch reveals a central channel with ~18 Å in length and ~14 Å in width. The channel is likely too narrow to accommodate a B-type dsDNA substrate. We propose that dsDNA associates with the arched basic patch along its border (i.e., from Lys27 to Lys60), which is ~29 Å in length and long enough to make contact with 6–7 base pairs (<a href="#pone-0000858-g003">Figures 3D and 3E</a>). Accordingly, association with the arched basic patch may bend dsDNA, resulting into distortion and/or flipping of the base pairs. This hypothesis is consistent with previous reports that the (HhH)<sub>2</sub> domains in other DNA metabolic enzymes are responsible for DNA bending and base pair distortion or flipping <a href="#pone.0000858-Pelletier1">[28]</a>–<a href="#pone.0000858-Chen2">[30]</a>.</p>


<h4>Spatial arrangement of the L1 motif and the NTD in the <em>3<sub>1</sub></em> overwound right-handed helical filament</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>L1 and NTD localize to the exterior of <em>3<sub>1</sub></em> overwound right-handed helical filament (<a href="#pone-0000858-g004">Figure 4A</a>). A closer look at this structure revealed that the linear basic patch of the L1 motif faces the arched basic patch of the (HhH)<sub>2</sub> domains at a distance of ~25 Å (<a href="#pone-0000858-g004">Figures 4B and 4C</a>). Together, they constitute an outwardly open palm structure with its inner pocket containing five positively charged amino acid residues, i.e., R217, R223, R229 of the L1 motif and K27, K60 of the NTD. Intriguingly, these five basic resides are evolutionarily conserved in all arachael and eukaryotic RecA family proteins (<a href="#pone-0000858-g002">Figures 2</a> and <a href="#pone-0000858-g003">3</a>). It is also noteworthy that the inner pocket of this palm structure is wide enough to accommodate anionic ssDNA, dsDNA, or both via electrostatic interactions and hydrogen bonds (<a href="#pone-0000858-g004">Figure 4A</a>). Such an interesting structural arrangement does not exist in other crystal structures of archaeal RadA proteins, such as RecA and <em>Mv</em>RadA-AMPPNP <em>6<sub>1</sub></em> right-handed filament <a href="#pone.0000858-Story1">[9]</a>, <a href="#pone.0000858-Wu1">[13]</a>, <a href="#pone.0000858-Wu2">[21]</a>, <em>Pyrococcus furiosus</em> (<em>Pf</em>) Rad51 ring <a href="#pone.0000858-Shin1">[10]</a>, and <em>Sso</em>RadA <em>4<sub>3</sub></em> left-handed helical filament <a href="#pone.0000858-Chen1">[16]</a>. One intriguing possibility is that <em>Sso</em>RadA protein in this <em>3<sub>1</sub></em> overwound right-handed filament may represent or is similar to the structural intermediate or conformation during the homology pairing reaction (see <a href="#s3">Discussion</a>).</p>
<div class="figure" id="pone-0000858-g004"><div class="img"><a name="pone-0000858-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Spatial arrangement of the L1 motif and the NTD along the <em>3<sub>1</sub></em> overwound right-handed <em>Sso</em>RadA filament.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Quaternary structure. The putative dsDNA binding regions in the NTD are shown in blue. The L1 and L2 ssDNA binding motifs are shown in pink and green, respectively. ATP binding sites are shown in yellow. The polymerization motif (PM) is indicated by an arrow. (B) A local surface charge potential of the L1 motif and the NTD region. Positive and negative charges are indicated by blue and red, respectively. (C) A ribbon diagram of two neighboring protomers (grey) showing the L1 motif (pink) and the NTD (cyan). The side chains of key basic residues K27, K60, R217, R223, and R229 are depicted in ball-and-stick representations.</p>
<span>doi:10.1371/journal.pone.0000858.g004</span></div>

<h4>K27, K60, R217, R223 and R229 are all essential for <em>Sso</em>RadA promotion of D-loop formation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To elucidate the functional roles of these five evolutionarily conserved basic residues, we first expressed and purified a panel of corresponding <em>Sso</em>RadA mutant proteins, i.e., the two lysine residues in the NTD were mutated to alanine (K27A, K60A, K27A K60A) or arginine (K27R, K60R); and the three arginine residues in the L1 motif were mutated to alanine (R217A, R223A, R229A) or lysine (R217K, R223K, R229K). We analyzed their ability to promote D-loop formation (<a href="#pone-0000858-g005">Figure 5</a> and Supplementary <a href="#pone.0000858.s001">Figure S1</a>) and to bind ssDNA (<a href="#pone-0000858-g006">Figure 6</a>) and dsDNA (<a href="#pone-0000858-g007">Figure 7</a>). As described previously <a href="#pone.0000858-Chen1">[16]</a>, <a href="#pone.0000858-Lee1">[22]</a>, wild-type <em>Sso</em>RadA catalyzed homology-dependent D-loop formation between a 50 mer <sup>32</sup>P-labeled oligonucleotide, P1655, and a supercoiled plasmid, GW1. We found that all mutants examined here produced no or much less D-loop product as compared to that of the wild type <em>Sso</em>RadA protein (<a href="#pone-0000858-g005">Figure 5</a> and Supporting Information <a href="#pone.0000858.s001">Figure S1</a>). These results are in agreement with the structural results presented above, which suggest that these evolutionarily conserved lysine and arginine residues play critical roles in the catalytic function of <em>Sso</em>RadA.</p>
<div class="figure" id="pone-0000858-g005"><div class="img"><a name="pone-0000858-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>D-loop formation.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p><em>Sso</em>RadA-promoted homologous strand assimilation between a γ-<sup>32</sup>P labeled oligonucleotide and a dsDNA plasmid was carried out as described previously <a href="#pone.0000858-Chen1">[16]</a>, <a href="#pone.0000858-Lee1">[22]</a></p>
<span>doi:10.1371/journal.pone.0000858.g005</span></div><div class="figure" id="pone-0000858-g006"><div class="img"><a name="pone-0000858-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>ssDNA binding.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>The 5′-biotinylated (dT)<sub>50</sub> oligonucleotide (10 µM in nucleotides) was first injected into BIAcore SA sensor chips. Wild-type or point mutant <em>Sso</em>RadA protein (1, 5, 10 µM) was passed over the chip at 25°C. Curves represent responses with the background subtracted. Binding signal was not detected when solutions that did not contain <em>Sso</em>RadA protein were injected.</p>
<span>doi:10.1371/journal.pone.0000858.g006</span></div><div class="figure" id="pone-0000858-g007"><div class="img"><a name="pone-0000858-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>dsDNA binding.</span></strong></p><a id="article1.body1.sec2.sec5.fig3.caption1.p1" name="article1.body1.sec2.sec5.fig3.caption1.p1"></a><p>Nucleoprotein gel of reactions containing 4.3 µM (in bps) dsDNA without <em>Sso</em>RadA protein (first lane in upper panel) or with 1, 5, or 10 µM of <em>Sso</em>RadA protein. The nucleoprotein complexes were fixed with glutaraldehyde to a final concentration of 2.5%, separated from free DNA on a 0.5% agarose gel, and visualized with ethidium bromide.</p>
<span>doi:10.1371/journal.pone.0000858.g007</span></div>

<h4>R217, R223, and R229 are indispensable for ssDNA binding</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We then examined the <em>Sso</em>RadA-ssDNA interactions in real time using surface plasmon resonance (SPR) imaging by immoblizing 5′-biotinylated oligonucleotide (dT)<sub>50</sub> onto the Biacore SA sensor chips. Various concentrations (1, 5, 10 µM) of the wild-type and mutant proteins were passed over the chip at 25°C in a buffer containing 2 mM MgCl<sub>2</sub> and 2 mM ATP (see Material and Methods); buffers without <em>Sso</em>RadA protein were then passed over the chip for 10 min to measure association and dissociation of <em>Sso</em>RadA proteins. Binding and release of different <em>Sso</em>RadA proteins to and from ssDNA are depicted in <a href="#pone-0000858-g006">Figure 6</a>. Binding signals of wild-type <em>Sso</em>RadA protein gradually increased and eventually reached steady-state plateau levels (<a href="#pone-0000858-g006">Figure 6A</a>, upper left panel). Maximum <em>Sso</em>RadA-ssDNA binding was nearly proportional to the protein concentration. Moreover, upon washing, wild-type <em>Sso</em>RadA protein rapidly dissociated from the ssDNA substrate.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>SPR analyses indicated that most mutants (except R217A, R223A and R229A) exhibited association and dissociation kinetics largely identical or similar to that of the wild-type protein. The K60A mutant had a slightly higher steady-state level, and the K27R and R217K mutants exhibited slower association rate (<a href="#pone-0000858-g006">Figure 6A</a>). Therefore, K27 and K60 have little or no effect on ssDNA binding. In contrast, three arginine residues (i.e., R217, R223, and R229) in the L1 motif were all indispensable for ssDNA binding. A point mutation in any of these arginines to alanine (i.e., R217A, R223A, R229A) resulted in 90–100% reduction of the SPR binding signal. On the other hand, R217K, R223K, and R229K mutants exhibited SPR binding signals similar to that of wild-type protein (<a href="#pone-0000858-g006">Figure 6B</a>). These results indicate that the positive charge of these three arginine residues is the main determinant for association of <em>Sso</em>RadA with ssDNA. This is compatible with the fact that <em>Sso</em>RadA and other RecA family proteins exhibit no or little nucleotide sequence specificity in binding to anionic ssDNA.</p>


<h4>K27 and K60 are specifically important for dsDNA binding</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We also compared the ability of the <em>Sso</em>RadA protein to bind dsDNA. Wild-type <em>Sso</em>RadA protein exhibited very low SPR binding signals to a (dA-dT)<sub>50</sub> dsDNA substrate (data not shown). Therefore, dsDNA binding was examined in nucleoprotein gel assays with a ~1000 base pairs (bps) dsDNA substrate, according to a protocol used previously for yeast Rad51 protein <a href="#pone.0000858-Zhang1">[31]</a>. In order to be visualized by electrophoresis on agarose gels, <em>Sso</em>RadA-dsDNA nucleoprotein complexes require stabilization by glutaraldehyde cross-linking. <em>Sso</em>RadA-dsDNA nucleoprotein complexes migrated much more slowly than free DNA. dsDNA substrate was treated with glutaraldehyde as a negative control and did not show a supershifting signal (<a href="#pone-0000858-g007">Figure 7</a>, the first lane of upper panel).</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>It is of interest that dsDNA binding to the wild-type <em>Sso</em>RadA protein saturated at ~one protein per bp. In contrast, K60A, R223A, R229A, R223K and R229K mutants did not saturate even at ~2.5 proteins per bp, indicating that these mutants are defective in dsDNA binding. R217A, R217K, K27A and K27R mutants also exhibited weaker affinity to dsDNA as compared to wild-type protein. In contrast, K60R mutant bound dsDNA as well as wild-type protein did (<a href="#pone-0000858-g007">Figure 7</a>). Taken together, these results indicate that K27, K60, R217, R223, and R229 are all important for the formation of <em>Sso</em>RadA-dsDNA complexes. Although the positive charge of R217, R223 or R229 is absolutely crucial for ssDNA binding (<a href="#pone-0000858-g006">Figure 6</a>), it is not the sole determinant for dsDNA binding. The results for the K60A and K60R mutants also indicate that the positive charge of K60 has a strong effect on dsDNA binding. In contrast, substitution of Lys27 with arginine could not restore the ability of <em>Sso</em>RaA to bind dsDNA. The amino group of Lys27 contacts the carbonyl group of Gly52 (<a href="#pone-0000858-g003">Figure 3C</a>). The equivalent residues of Lys27 and Gly52 in human Rad51 are Lys40 and Gly65. These two amino acid residues of human Rad51 had been observed to directly bind dsDNA <a href="#pone.0000858-Aihara1">[18]</a>. Therefore, dsDNA is likely to make direct contacts with both Lys27 and Lys60 of <em>Sso</em>RadA protein .</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>It is of interest that the dsDNA binding deficiency of the K60A mutant was suppressed by an additional lysine-to-alanine substitution on K27. Still, the K27A K60A mutant is apparently defective in promoting D-loop formation (<a href="#pone-0000858-g005">Figure 5</a>), suggesting that binding of <em>Sso</em>RadA protein to dsDNA in the absence of ssDNA differs from that in the presence of ssDNA. Taken together, our results here support the notion that K27 and K60 have specific effects on dsDNA binding to a <em>Sso</em>RadA-ssDNA nucleoprotein helical filament (<a href="#pone-0000858-g005">Figures 5</a>–<a href="#pone-0000858-g006"></a><a href="#pone-0000858-g007">7</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Two lysines (K27, K60) in the NTD and three arginines (R217, R223, R229) in the L1 motif of <em>Sso</em>RadA are evolutionarily conserved in all archaeal and eukaryotic RecA family proteins (<a href="#pone-0000858-g002">Figures 2</a> and <a href="#pone-0000858-g003">3</a>). Structural and biochemical analyses in this study indicate that these five basic residues play key roles in DNA binding and strand exchange activities (<a href="#pone-0000858-g002">Figures 2</a>–<a href="#pone-0000858-g003"></a><a href="#pone-0000858-g004"></a><a href="#pone-0000858-g005"></a><a href="#pone-0000858-g006"></a><a href="#pone-0000858-g007">7</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the <em>3<sub>1</sub></em> overwound right-handed helical filament of <em>Sso</em>RadA proteins, these five basic residues generate a positively charged surface for a palm structure formed by the L1 motif and the NTD. Intriguingly, this palm structure is not only localized on but also opens outwardly to the exterior of the helical filament (<a href="#pone-0000858-g003">Figure 3</a>). Although we can not exclude the possibility that such a structural arrangement simply arises from protein packing or filament organization during protein crystallization, we speculate that this conformation may represent or is similar to the structural intermediate responsible for homology search and pairing between ssDNA and dsDNA. <a href="#pone-0000858-g008">Figure 8</a> is a cartoon that illustrates the function of the five positively charged amino acid residues. First of all, the three arginine residues in the L1 motif constitute a linear basic patch (<a href="#pone-0000858-g002">Figure 2</a>) for ssDNA binding via either electrostatic interactions or hydrogen bonds with the negatively charged sugar-phosphate backbone of ssDNA. This is compatible with the fact that RadA/Rad51 proteins exhibit little or no sequence specificity for DNA substrates. The ~18 Å linear basic patch is long enough to make contact with three or four contiguous nucleotides of a ssDNA substrate, in which the purine and pyrimidine bases of bound nucleotides may be outwardly exposed. Such an arrangement not only suggests a model for RadA-mediated ssDNA stretching but also provides a framework in which a RadA-ssDNA nucleoprotein filament can form base pairs with target dsDNA (see below). Second, our structural analysis also revealed that NTD forms a 92° arched basic patch along the border of the second HhH motif (<a href="#pone-0000858-g003">Figure 3D</a>). Lys27 and Lys60 localize at each end of the arched basic patch and are required for dsDNA binding (<a href="#pone-0000858-g007">Figure 7</a>) and D-loop formation (<a href="#pone-0000858-g005">Figure 5</a>). The 92° arched basic patch likely associates with dsDNA along its border, and may lead to dsDNA bending or even distortion and flipping of base pairs. Taken together, we suggest that these two modes of interaction (i.e., L1-ssDNA and NTD-dsDNA) function in unison to mediate homologous pairing and strand exchange between the RadA-ssDNA nucleoprotein filament and the bound dsDNA (<a href="#pone-0000858-g008">Figure 8</a>).</p>
<div class="figure" id="pone-0000858-g008"><div class="img"><a name="pone-0000858-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000858" data-uri="info:doi/10.1371/journal.pone.0000858.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000858.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000858.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000858.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000858.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>A new hypothesis for homology interactions mediated by RadA protein filaments.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Interactions between three arginine residues of the L1 motif and sugar-phosphate backbone of ssDNA result in the nucleotide bases of ssDNA facing the NTD (<a href="#pone-0000858-g001">Figure 1</a>). An anionic dsDNA associates with the NTD along the border of a 92° basic arch via electrostatic interactions or hydrogen bonding. Lys27 and Lys60 are located at each end of this arched basic patch (<a href="#pone-0000858-g003">Figure 3</a>). As a result, NTD-dsDNA association may lead to DNA bending or distortion or flipping of base pairs. L1-ssDNA and NTD-dsDNA interactions function in unison to mediate homologous search and pairing between a <em>3<sub>1</sub></em> overwound right-handed RadA-ssDNA nucleoprotein filament and dsDNA.</p>
<span>doi:10.1371/journal.pone.0000858.g008</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The nature of our new model differs from that of the facilitated DNA rotation model <a href="#pone.0000858-Cox2">[20]</a>, which, like other current models of RecA family proteins, is based on the assumption that these proteins are active only when they form a <em>6<sub>1</sub></em> right-handed helical nucleoprotein filament. Although the latter hypothesis has been well accepted for almost two decades, it has at least two problems. First, in the <em>6<sub>1</sub></em> right-handed helical filaments, all known DNA binding motifs (i.e., L1, L2, NTD, CTD) are located at or near the central axis. Therefore, the facilitated DNA rotation model assumes that a novel dsDNA binding motif(s), localized on the exterior of a <em>6<sub>1</sub></em> right-handed helical nucleoprotein filament, is required for homology search, pairing, and strand exchange reactions. However, such a DNA binding motif has never been identified in any RecA family protein. In contrast, here we report that both L1 and NTD are relocated to the exterior of the <em>3<sub>1</sub></em> overwound right-handed filament (<a href="#pone-0000858-g004">Figure 4</a>). Such a spatial arrangement allows the RadA protein to capture both ssDNA and dsDNA simultaneously. Moreover, biochemical analyses in this study have demonstrated that NTD is essential for both dsDNA binding and D-loop formation (<a href="#pone-0000858-g006">Figures 6</a> and <a href="#pone-0000858-g007">7</a>). Second, the facilitated DNA rotation model overlooks the fact that RecA family proteins are flexible enough to form different quaternary structures, including protein rings, <em>3<sub>1</sub></em> overwound right-handed filaments and <em>4<sub>3</sub></em> left-handed helical filaments. In our previous paper <a href="#pone.0000858-Chen1">[16]</a>, we reported that a progressive clockwise rotation along the axes of RadA protein polymers is responsible for the structural transition from a protein ring to a <em>6<sub>1</sub></em> right-handed helical filament, then to a <em>3<sub>1</sub></em> overwound right-handed filament, and finally, to a <em>4<sub>3</sub></em> left-handed helical filament <a href="#pone.0000858-Chen1">[see Figure 5 in ref. 16]</a>. The second and third rotation each involve a 120° discrete step. Importantly, this clockwise axial rotation accompanies the migration of L1, L2, and HhH DNA binding motifs from the interior to the exterior of RadA protein polymers; i.e., from the inner circle of a toroidal ring to the central axis of the <em>6<sub>1</sub></em> right-handed filament, then to the exterior of the <em>3<sub>1</sub></em> over-wound right-handed filament, and finally to the outermost surface of <em>4<sub>3</sub></em> left-handed filament <a href="#pone.0000858-Chen1">[see Figure 3 in ref. 16]</a>. Thus, the L1, L2, and HhH DNA binding motifs are capable of moving simultaneously to promote ssDNA binding, homology pairing (i.e., dsDNA capturing), and finally strand exchange. We propose that DNA and RecA family protein filaments rotate simultaneously during these processes. The energy of ATP is likely used to facilitate axial rotation of RecA protein helical filaments and then to promote DNA rotation and strand exchange <a href="#pone.0000858-Chen1">[16]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>How is the energy of ATP hydrolysis coupled to axial rotation of RadA protein helical filaments? We had identified an arginine amino acid (Arg83) in the SRM of <em>Sso</em>RadA protein. This arginine, referred to as “R<sub>0</sub>”, is evolutionarily conserved in all members of RecA protein family <a href="#pone.0000858-Chen1">[see Figure 1 in ref. 16]</a>. In the RadA-AMP-PNP <em>6<sub>1</sub></em> right-handed filament, the guanidinium group of R<sub>0</sub> forms salt bridges with the carboxyl groups of two glutamate residues: Glu96 (denoted “E<sub>1</sub>”) of the same protomer and Glu157 (denoted “E<sub>2</sub>”) of the neighboring protomer. These two salt bridges are likely to directly control opening and closure of the ATP-binding pocket between two neighboring protomers. In the closed ring, R<sub>0</sub> interacts with E<sub>2</sub> but not with E<sub>1</sub>. In the RadA-AMP-PNP <em>6<sub>1</sub></em> right-handed filament, R<sub>0</sub>-E<sub>1</sub> and R<sub>0</sub>-E<sub>2</sub> interactions (i.e., the “E<sub>1</sub>-R<sub>0</sub>-E<sub>2</sub>” triad) function as a clip to fasten the AMP-PNP or ATP binding between two protomers <a href="#pone.0000858-Chen1">[see Figure 6 in ref. 16]</a>. This model is supported by our findings that point mutations of R<sub>0</sub> to glutamate in <em>Sso</em>RadA <a href="#pone.0000858-Chen1">[16]</a>, <em>E. coli</em> RecA or <em>S. cerevisiae</em> Rad51 (Lin KA, Lee CD and Wang TF, unpublished results) all result in significant decreases in ATP-binding affinity. As the <em>6<sub>1</sub></em> RadA helical filament undergoes clockwise axial rotation in two discrete ~120° steps to the <em>3<sub>1</sub></em> overwound right-handed filament and then to the <em>4<sub>3</sub></em> left-handed filament, E<sub>1</sub> and E<sub>2</sub> break their ionic interactions with R<sub>0</sub> and gradually move away from R<sub>0 </sub><a href="#pone.0000858-Chen1">[see Figure 6 in ref. 16]</a>. Therefore, this clockwise axial rotation progressively opens up the ATP-binding pocket. We propose that the dimeric (or monomeric) conformations in the <em>6<sub>1</sub></em> right-handed, <em>3<sub>1</sub></em> overwound right-handed and <em>4<sub>3</sub></em> left-handed filaments represent the TP (ATP-bound), DP (ADP+ Pi− or ADP-bound), and E (empty) states of RadA proteins, respectively. In this scenario, the RadA helical filament is functionally similar to the F1-ATPase. The F1-ATPase is a rotary motor in which a central gamma-subunit rotates against a surrounding cylinder made of α<sub>3</sub>β<sub>3</sub>-subunits. Driven by the three beta subunits that sequentially hydrolyze ATP, the motor also carries out “clockwise” rotation in three discrete 120° steps. These steps were denoted as TP, DP, and E <a href="#pone.0000858-Boyer1">[32]</a>. Because the ATPase domains of F1-ATP and RecA family proteins are structurally and functionally conserved, we favour the possibility that the neighboring dimers or monomers along a RadA helical filament, like those of F-ATPase motor, proceed through a sequential TP-DP-E structural transition during their catalytic cycles.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>One potential problem of our model arises from the stoichiometry of RadA-DNA interaction. The <em>3<sub>1</sub></em> overwound right-handed filament (3 monomers per turn with 98Å pitch) may not be obtained by continuous transformation from a right-handed filament with 6 monomers per turn with 95 or 107 Å pitch, if we assume that continuous transformation requires conservation of stoichiometry (3 nucleotides or 3 base pairs per RadA). The axial spacing between consecutive bases would need to increase to at least 10.9 Å in the <em>3<sub>1</sub></em> overwound right-handed filament. In fact, the actual spacing would also need to be significantly bigger than 10.9 Å since the DNA binding sites are on the exterior of the filament and the DNA would have to wrap around the filament. Such interbase distances will require breaking the DNA into mononucleotide pieces and this is certainly not what happens during DNA pairing and strand exchange. However, this dilemma may be addressed in one of two ways. First, there is no experimental evidence to support that stoichiometry (3 nucleotides or 3 base pairs per RadA) is conserved throughout the catalytic cycles of RadA or RecA family proteins. Therefore, conservation of stoichiometry may not be a prerequisite for continuous transformation. Second, it assumes that all RadA monomers in a helical filament carry out axial rotation at the same time. As described above, we speculate that RadA proteins undergo sequential structural transformation of dimeric (or monomeric) conformations from that in a <em>6<sub>1</sub></em> right-handed filament (for ssDNA binding) to that in a <em>3<sub>1</sub></em> overwound right-handed filament (for homology pairing), and finally to that in a <em>4<sub>3</sub></em> left-handed filament (for strand exchange, ssDNA exclusion or protein dissociation) <a href="#pone.0000858-Chen1">[16]</a>. In this scenario, DNA-protein stoichiometry can be conserved and DNA substrates will also remain intact. Moreover, because the monomeric structure of RadA protein in the <em>3<sub>1</sub></em> overwound right-handed filament is more extended along the axis of helical filament than those in the <em>6<sub>1</sub></em> or <em>4<sub>3</sub></em> filaments, it allows a transient extension or stretch of ssDNA for homology pairing (see <a href="#pone-0000858-g008">Figure 8</a>). Therefore, we suggest that the monomeric structure of RadA protein in the <em>3<sub>1</sub></em> right-handed filament likely represents a functional form.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, structural and biochemical analyses in this study and our previous paper <a href="#pone.0000858-Chen1">[16]</a> suggest a new mechanism for RadA/Rad51 mediated DNA binding, homology pairing and strand exchange reaction. We propose that RadA proteins in a helical filament sequentially carry out their catalytic function via a clockwise or right-to-left axial rotation. This is in contrast to all current models, which assume that these proteins function exclusively as 6<sub>1</sub> right-handed helical filaments throughout their catalytic cycles. Ultimately, the validity of our model will have to be tested by determining the structures of RadA-DNA nucleoprotein filaments, or by visualizing ATP-fueled axial rotation of RecA family protein filaments during homology pairing and strand exchange reactions through single molecule studies.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Protein expression and purification</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Wild-type and mutant <em>Sso</em>RadA protein were expressed and purified as described previously <a href="#pone.0000858-Chen1">[16]</a>. The amino acid sequence of purified wild-type <em>Sso</em>RadA is identical to that encoded by the <em>Sso</em>RadA gene.</p>


<h4>Crystallization and X-ray data collection</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p><em>Sso</em>RadA protein (24 mg/ml) in 30 mM Tris-HCl (pH 8.0) was crystallized using the hanging drop vapor diffusion method. Initial screening was perfomed with Hampton Research and Emerald BioStructure crystallization kits. The protein solution:reservoir ratio was 2 µl:2 µl. Protein crystals were obtained in ~5 days, with the reservoir containing 500 µl of 100 mM KCl, 25 mM MgCl<sub>2</sub>, 15% isopropanol, and 50 mM sodium cacodylate (pH 6.0). Crystals were cryoprotected by an ~1 min wash in the reservoir solution plus 25% glycerol prior to mounting on the X-ray machine. X-ray diffraction data were collected using the National Synchrotron Radiation Research Center (NSRRC) beamline 13B1 in Taiwan (<a href="#pone-0000858-t001">Table 1</a>). All diffraction data were processed and scaled using the HKL2000 package <a href="#pone.0000858-Otwinowski1">[33]</a>. The space group is <em>P3<sub>1</sub></em>, with unit cell dimensions <em>a</em> = <em>b</em> = 99.55 Å, <em>c</em> = 99.41 Å and three protomers in the asymmetric unit. The three filaments along the <em>c</em> axis were related to each other by translational symmetry. Data were processed to a final resolution of 1.93 Å. The structure was determined by molecular replacement using the CNS program <a href="#pone.0000858-Brunger1">[34]</a>. The twin factor was 0.5, as revealed by the twin detect function of the CNS program, indicating that it was a perfect twin structure. The search model was <em>Sso</em>RadA of the space group <em>P3<sub>1</sub>21</em> (PDB = 2BKE) <a href="#pone.0000858-Ariza1">[15]</a>. The structure was manually rebuilt using the O program <a href="#pone.0000858-Jones1">[35]</a> and refined with the CNS program, with or without twin refinement. All figures were generated by PyMol (<a href="http://pymol.sourceforge.net">http://pymol.sourceforge.net</a>). Diffraction data and refinement statistics are shown in <a href="#pone-0000858-t001">Table 1</a>. Surface area accessibility calculations were performed by the CCP4 program <a href="#pone.0000858-CCP41">[36]</a>. Atomic coordinates and structure factors have been deposited in the PDB under accession code 2Z43.</p>


<h4>DNA assimilation assays</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p><em>Sso</em>RadA mediated D-loop formation assays were described previously <a href="#pone.0000858-Chen1">[16]</a>, <a href="#pone.0000858-Lee1">[22]</a>, <a href="#pone.0000858-Chen3">[37]</a>. D-loop formation efficiency was calculated according to the molar ratio of joint molecules over total dsDNA substrate. The relative efficiency of each mutant relative to that of wild-type protein (t = 15 min) is presented.</p>


<h4>Surface plasmon resonance (SPR)</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Interactions of wild-type and mutant <em>Sso</em>RadA protein with a single-stranded oligonucleotide (dT)<sub>50</sub> were determined by a Biacore X surface plasmon resonance (SPR) biosensor instrument (Biacore at Uppsala, Sweden). The 5′-biotinylated oligonucleotide (dT)<sub>50</sub> was diluted to 10 µM with HBS buffer (Biacore) and manually injected into a Biacore SA sensor chip in the channel 2 flow cell to 1300 resonance or response units. The SA sensor chip is commercially available and pre-immobilized with streptavidin (Biacore). The channel 1 flow cell was used as a reference for online background subtraction. Protein was diluted in running buffer containing HBS buffer (Biacore), 0.005% (w/v) p-20 (Biacore), 2 mM MgCl<sub>2</sub>, and 2 mM ATP (pH 8.0). To monitor DNA-protein interaction, <em>Sso</em>RadA protein solution (1, 5, or 10 µM in 50 µL) was injected onto the ssDNA surface with a flow rate of 30 µL/min at 25°C. The SA sensor chip was regenerated with a quick injection of 1 M NaCl and 50 mM NaOH (30 µL). Binding signals were not detected when the control buffers (i.e., no <em>Sso</em>RadA protein) were injected.</p>


<h4>dsDNA binding assay</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We used the difference in electrophoretic mobility of glutaraldehyde-fixed <em>Sso</em>RadA-dsDNA complexes to detect the dsDNA binding ability of wild-type and mutant <em>Sso</em>RadA proteins. The experimental procedures were modified from a yeast Rad51 protocol <a href="#pone.0000858-Zhang1">[31]</a>. A greater amount of glutaraldyde was applied for cross-linking, as all <em>Sso</em>RadA proteins were originally dissolved in a buffer containing 30 mM Tris-HCl (pH 8.0). An ~1000 bp PCR product from cDNA encoding the vaccinia virus G9 protein (sequence available on request) was used as dsDNA substrate. <em>Sso</em>RadA-dsDNA complexes were formed by mixing the indicated concentrations of RadA (1, 5, or 10 µM) and dsDNA (4.3 µM in bps) in running buffer (20 mM Hepes pH 7.5, 20 mM magnesium acetate, 2 mM DTT, and 5 mM ATP) at 65°C for 30 min. Glutaraldehyde was added to a final concentration of 2.5% and the reaction incubated at 65°C for 10 min. Reaction products were loaded directly onto 0.5% agarose gels in 1X Tris-buffered EDTA, run for 70 min at 50 V (4 V/cm), and visualized with ethidium bromide.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000858"></div><a name="pone.0000858.s001" id="pone.0000858.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000858.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000858.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.47 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. Chris S.-C. Jao for assistance in the SPR experiment, the National Synchrotron Radiation Research Center (Hsinchu, Taiwan) for beamtime, and Dr. Neal Lue (Weill Cornell Medical College, NY) for editing the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TW AW. Performed the experiments: LC. Analyzed the data: TW LC TK. Contributed reagents/materials/analysis tools: TW LC TK KL YC. Wrote the paper: TW AW. Other: Principle Investigator: AW TW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000858-Kowalczykowski1" id="pone.0000858-Kowalczykowski1"></a>Kowalczykowski SC, Eggleston AK (1994) Homologous pairing and DNA strand-exchange proteins. Annu Rev Biochem  63: 991–1043.  <ul class="find" data-citedArticleID="988978" data-doi="10.1146/annurev.bi.63.070194.005015"><li><a href="http://dx.doi.org/10.1146/annurev.bi.63.070194.005015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homologous+pairing+and+DNA+strand-exchange+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homologous+pairing+and+DNA+strand-exchange+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000858-Cox1" id="pone.0000858-Cox1"></a>Cox MM (2003) The bacterial RecA protein as a motor protein. Annu Rev Microbiol  57: 551–577.  <ul class="find" data-citedArticleID="988962"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+bacterial+RecA+protein+as+a+motor+protein.&amp;auth=&amp;atitle=The+bacterial+RecA+protein+as+a+motor+protein." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+bacterial+RecA+protein+as+a+motor+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+bacterial+RecA+protein+as+a+motor+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000858-West1" id="pone.0000858-West1"></a>West SC (2003) Molecular views of recombination proteins and their control. Nat Rev Mol Cell Biol  4: 435–445.  <ul class="find" data-citedArticleID="989006" data-doi="10.1038/nrm1127"><li><a href="http://dx.doi.org/10.1038/nrm1127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+views+of+recombination+proteins+and+their+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+views+of+recombination+proteins+and+their+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000858-Sonoda1" id="pone.0000858-Sonoda1"></a>Sonoda E, Sasaki MS, Buerstedde JM, Bezzubova O, Shinohara A, et al.  (1998) Rad51-deficient vertebrate cells accumulate chromosomal breaks prior to cell death. EMBO J  17: 598–608.  <ul class="find" data-citedArticleID="989000" data-doi="10.1093/emboj/17.2.598"><li><a href="http://dx.doi.org/10.1093/emboj/17.2.598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rad51-deficient+vertebrate+cells+accumulate+chromosomal+breaks+prior+to+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rad51-deficient+vertebrate+cells+accumulate+chromosomal+breaks+prior+to+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000858-Bishop1" id="pone.0000858-Bishop1"></a>Bishop DK (1994) RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis. Cell  79: 1081–1092.  <ul class="find" data-citedArticleID="988946" data-doi="10.1016/0092-8674(94)90038-8"><li><a href="http://dx.doi.org/10.1016/0092-8674(94)90038-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RecA+homologs+Dmc1+and+Rad51+interact+to+form+multiple+nuclear+complexes+prior+to+meiotic+chromosome+synapsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RecA+homologs+Dmc1+and+Rad51+interact+to+form+multiple+nuclear+complexes+prior+to+meiotic+chromosome+synapsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000858-Rockmill1" id="pone.0000858-Rockmill1"></a>Rockmill B, Sym M, Scherthan H, Roeder GS (1995) Roles for two RecA homologs in promoting meiotic chromosome synapsis. Genes Dev  9: 2684–2695.  <ul class="find" data-citedArticleID="988990" data-doi="10.1101/gad.9.21.2684"><li><a href="http://dx.doi.org/10.1101/gad.9.21.2684" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+for+two+RecA+homologs+in+promoting+meiotic+chromosome+synapsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+for+two+RecA+homologs+in+promoting+meiotic+chromosome+synapsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000858-Henning1" id="pone.0000858-Henning1"></a>Henning W, Sturzbecher HW (2003) Homologous recombination and cell cycle checkpoints: Rad51 in tumour progression and therapy resistance. Toxicology  193: 91–109.  <ul class="find" data-citedArticleID="988968" data-doi="10.1016/s0300-483x(03)00291-9"><li><a href="http://dx.doi.org/10.1016/s0300-483x(03)00291-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homologous+recombination+and+cell+cycle+checkpoints%3A+Rad51+in+tumour+progression+and+therapy+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homologous+recombination+and+cell+cycle+checkpoints%3A+Rad51+in+tumour+progression+and+therapy+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000858-Thorslund1" id="pone.0000858-Thorslund1"></a>Thorslund T, Esashi F, West SC (2007) Interactions between human BRCA2 protein and the meiosis-specific recombinase DMC1. EMBO J  26: 2915–2922.  <ul class="find" data-citedArticleID="989004" data-doi="10.1038/sj.emboj.7601739"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601739" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactions+between+human+BRCA2+protein+and+the+meiosis-specific+recombinase+DMC1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactions+between+human+BRCA2+protein+and+the+meiosis-specific+recombinase+DMC1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000858-Story1" id="pone.0000858-Story1"></a>Story RM, Steitz TA (1992) Structure of the recA protein-ADP complex. Nature  355: 374–376.  <ul class="find" data-citedArticleID="989002" data-doi="10.1038/355374a0"><li><a href="http://dx.doi.org/10.1038/355374a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+recA+protein-ADP+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+recA+protein-ADP+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000858-Shin1" id="pone.0000858-Shin1"></a>Shin DS, Pellegrini L, Daniels DS, Yelent B, Craig L, et al.  (2003) Full-length archaeal Rad51 structure and mutants: mechanisms for RAD51 assembly and control by BRCA2. EMBO J  22: 4566–4576.  <ul class="find" data-citedArticleID="988998" data-doi="10.1093/emboj/cdg429"><li><a href="http://dx.doi.org/10.1093/emboj/cdg429" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-length+archaeal+Rad51+structure+and+mutants%3A+mechanisms+for+RAD51+assembly+and+control+by+BRCA2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-length+archaeal+Rad51+structure+and+mutants%3A+mechanisms+for+RAD51+assembly+and+control+by+BRCA2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000858-Kinebuchi1" id="pone.0000858-Kinebuchi1"></a>Kinebuchi T, Kagawa W, Enomoto R, Tanaka K, Miyagawa K, et al.  (2004) Structural basis for octameric ring formation and DNA interaction of the human homologous-pairing protein Dmc1. Mol Cell  14: 363–374.  <ul class="find" data-citedArticleID="988974" data-doi="10.1016/s1097-2765(04)00218-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(04)00218-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+octameric+ring+formation+and+DNA+interaction+of+the+human+homologous-pairing+protein+Dmc1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+octameric+ring+formation+and+DNA+interaction+of+the+human+homologous-pairing+protein+Dmc1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000858-Conway1" id="pone.0000858-Conway1"></a>Conway AB, Lynch TW, Zhang Y, Fortin GS, Fung CW, et al.  (2004) Crystal structure of a Rad51 filament. Nat Struct Mol Biol  11: 791–796.  <ul class="find" data-citedArticleID="988960" data-doi="10.1038/nsmb795"><li><a href="http://dx.doi.org/10.1038/nsmb795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+a+Rad51+filament." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+a+Rad51+filament.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000858-Wu1" id="pone.0000858-Wu1"></a>Wu Y, He Y, Moya IA, Qian X, Luo Y (2004) Crystal structure of archaeal recombinase RadA: a snapshot of its extended conformation. Mol Cell  15: 423–435.  <ul class="find" data-citedArticleID="989008" data-doi="10.1016/j.molcel.2004.07.014"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.07.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+archaeal+recombinase+RadA%3A+a+snapshot+of+its+extended+conformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+archaeal+recombinase+RadA%3A+a+snapshot+of+its+extended+conformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000858-Qian1" id="pone.0000858-Qian1"></a>Qian X, Wu Y, He Y, Luo Y (2005) Crystal structure of <em>Methanococcus voltae</em> RadA in complex with ADP: hydrolysis-induced conformational change. Biochemistry  44: 13753–13761.  <ul class="find" data-citedArticleID="988988" data-doi="10.1021/bi051222i"><li><a href="http://dx.doi.org/10.1021/bi051222i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+Methanococcus+voltae+RadA+in+complex+with+ADP%3A+hydrolysis-induced+conformational+change." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+Methanococcus+voltae+RadA+in+complex+with+ADP%3A+hydrolysis-induced+conformational+change.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000858-Ariza1" id="pone.0000858-Ariza1"></a>Ariza A, Richard DJ, White MF, Bond CS (2005) Conformational flexibility revealed by the crystal structure of a crenarchaeal RadA. Nucleic Acids Res  33: 1465–1473.  <ul class="find" data-citedArticleID="988944" data-doi="10.1093/nar/gki288"><li><a href="http://dx.doi.org/10.1093/nar/gki288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformational+flexibility+revealed+by+the+crystal+structure+of+a+crenarchaeal+RadA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformational+flexibility+revealed+by+the+crystal+structure+of+a+crenarchaeal+RadA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000858-Chen1" id="pone.0000858-Chen1"></a>Chen LT, Ko TP, Chang YC, Lin KA, Chang CS, et al.  (2007) Crystal structure of the left-handed archaeal RadA helical filament: identification of a functional motif for controlling quaternary structures and enzymatic functions of RecA family proteins. Nucleic Acids Res  35: 1787–1801.  <ul class="find" data-citedArticleID="988954" data-doi="10.1093/nar/gkl1131"><li><a href="http://dx.doi.org/10.1093/nar/gkl1131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+left-handed+archaeal+RadA+helical+filament%3A+identification+of+a+functional+motif+for+controlling+quaternary+structures+and+enzymatic+functions+of+RecA+family+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+left-handed+archaeal+RadA+helical+filament%3A+identification+of+a+functional+motif+for+controlling+quaternary+structures+and+enzymatic+functions+of+RecA+family+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000858-Pellegrini1" id="pone.0000858-Pellegrini1"></a>Pellegrini L, Yu DS, Lo T, Anand S, Lee M, et al.  (2002) Insights into DNA recombination from the structure of a RAD51-BRCA2 complex. Nature  420: 287–293.  <ul class="find" data-citedArticleID="988984" data-doi="10.1038/nature01230"><li><a href="http://dx.doi.org/10.1038/nature01230" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+DNA+recombination+from+the+structure+of+a+RAD51-BRCA2+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+DNA+recombination+from+the+structure+of+a+RAD51-BRCA2+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000858-Aihara1" id="pone.0000858-Aihara1"></a>Aihara H, Ito Y, Kurumizaka H, Yokoyama S, Shibata T (1999) The N-terminal domain of the human Rad51 protein binds DNA: structure and a DNA binding surface as revealed by NMR. J Mol Biol  290: 495–504.  <ul class="find" data-citedArticleID="988942" data-doi="10.1006/jmbi.1999.2904"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.2904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+N-terminal+domain+of+the+human+Rad51+protein+binds+DNA%3A+structure+and+a+DNA+binding+surface+as+revealed+by+NMR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+N-terminal+domain+of+the+human+Rad51+protein+binds+DNA%3A+structure+and+a+DNA+binding+surface+as+revealed+by+NMR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000858-Kinebuchi2" id="pone.0000858-Kinebuchi2"></a>Kinebuchi T, Kagawa W, Kurumizaka H, Yokoyama S (2005) Role of the N-terminal domain of the human DMC1 protein in octamer formation and DNA binding. J Biol Chem  280: 28382–28387.  <ul class="find" data-citedArticleID="988976" data-doi="10.1074/jbc.m503372200"><li><a href="http://dx.doi.org/10.1074/jbc.m503372200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+N-terminal+domain+of+the+human+DMC1+protein+in+octamer+formation+and+DNA+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+N-terminal+domain+of+the+human+DMC1+protein+in+octamer+formation+and+DNA+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000858-Cox2" id="pone.0000858-Cox2"></a>Cox MM (2007) Motoring along with the bacterial RecA protein. Nat Rev Mol Cell Biol  8: 127–138.  <ul class="find" data-citedArticleID="988964" data-doi="10.1038/nrm2099"><li><a href="http://dx.doi.org/10.1038/nrm2099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Motoring+along+with+the+bacterial+RecA+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Motoring+along+with+the+bacterial+RecA+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000858-Wu2" id="pone.0000858-Wu2"></a>Wu Y, Qian X, He Y, Moya IA, Luo Y (2005) Crystal structure of an ATPase-active form of Rad51 homolog from <em>Methanococcus voltae</em>. Insights into potassium dependence. J Biol Chem  280: 722–728.  <ul class="find" data-citedArticleID="989010" data-doi="10.1074/jbc.m411093200"><li><a href="http://dx.doi.org/10.1074/jbc.m411093200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+an+ATPase-active+form+of+Rad51+homolog+from+Methanococcus+voltae.+Insights+into+potassium+dependence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+an+ATPase-active+form+of+Rad51+homolog+from+Methanococcus+voltae.+Insights+into+potassium+dependence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000858-Lee1" id="pone.0000858-Lee1"></a>Lee MH, Leng CH, Chang YC, Chou CC, et al.  (2004) Self-polymerization of archaeal RadA protein into long and fine helical filaments. Biochem Biophys Res Commun  323: 845–851.  <ul class="find" data-citedArticleID="988980" data-doi="10.1016/j.bbrc.2004.08.163"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2004.08.163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-polymerization+of+archaeal+RadA+protein+into+long+and+fine+helical+filaments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-polymerization+of+archaeal+RadA+protein+into+long+and+fine+helical+filaments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000858-Yu1" id="pone.0000858-Yu1"></a>Yu X, Egelman EH (1990) Image-analysis reveals that <em>Escherichia coli</em> RecA protein consists of 2 domains. Biophy J  57: 555–566.  <ul class="find" data-citedArticleID="989012" data-doi="10.1016/s0006-3495(90)82571-6"><li><a href="http://dx.doi.org/10.1016/s0006-3495(90)82571-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Image-analysis+reveals+that+Escherichia+coli+RecA+protein+consists+of+2+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Image-analysis+reveals+that+Escherichia+coli+RecA+protein+consists+of+2+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000858-Seitz1" id="pone.0000858-Seitz1"></a>Seitz EM, Brockman JP, Sandler SJ, Clark AJ, Kowalczykowski SC (1998) RadA protein is an archaeal RecA protein homolog that catalyzes DNA strand exchange. Genes Dev  12: 1248–1253.  <ul class="find" data-citedArticleID="988992" data-doi="10.1101/gad.12.9.1248"><li><a href="http://dx.doi.org/10.1101/gad.12.9.1248" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RadA+protein+is+an+archaeal+RecA+protein+homolog+that+catalyzes+DNA+strand+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RadA+protein+is+an+archaeal+RecA+protein+homolog+that+catalyzes+DNA+strand+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000858-Shi1" id="pone.0000858-Shi1"></a>Shi WX, Larson RG (2005) Atomic force microscopic study of aggregation of RecA-DNA nucleoprotein filaments into left-handed supercoiled bundles. Nano Lett  5: 2476–2481.  <ul class="find" data-citedArticleID="988996" data-doi="10.1021/nl051783v"><li><a href="http://dx.doi.org/10.1021/nl051783v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Atomic+force+microscopic+study+of+aggregation+of+RecA-DNA+nucleoprotein+filaments+into+left-handed+supercoiled+bundles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Atomic+force+microscopic+study+of+aggregation+of+RecA-DNA+nucleoprotein+filaments+into+left-handed+supercoiled+bundles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000858-DeZutter1" id="pone.0000858-DeZutter1"></a>De Zutter JK, Knight KL (1999) The hRad51 and RecA proteins show significant differences in cooperative binding to single-stranded DNA. J Mol Biol  293: 769–780.  <ul class="find" data-citedArticleID="988966" data-doi="10.1006/jmbi.1999.3200"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hRad51+and+RecA+proteins+show+significant+differences+in+cooperative+binding+to+single-stranded+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hRad51+and+RecA+proteins+show+significant+differences+in+cooperative+binding+to+single-stranded+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000858-Shao1" id="pone.0000858-Shao1"></a>Shao X, Grishin NV (2000) Common fold in helix-hairpin-helix proteins. Nucleic Acids Res  28: 2643–2650.  <ul class="find" data-citedArticleID="988994" data-doi="10.1093/nar/28.14.2643"><li><a href="http://dx.doi.org/10.1093/nar/28.14.2643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+fold+in+helix-hairpin-helix+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+fold+in+helix-hairpin-helix+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000858-Pelletier1" id="pone.0000858-Pelletier1"></a>Pelletier H, Sawaya MR, Kumar A, Wilson SH, Kraut J (1994) Structures of ternary complexes of rat DNA polymerase beta, a DNA template-primer, and ddCTP. Science  264: 1891–1903.  <ul class="find" data-citedArticleID="988986" data-doi="10.1126/science.7516580"><li><a href="http://dx.doi.org/10.1126/science.7516580" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structures+of+ternary+complexes+of+rat+DNA+polymerase+beta%2C+a+DNA+template-primer%2C+and+ddCTP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structures+of+ternary+complexes+of+rat+DNA+polymerase+beta%2C+a+DNA+template-primer%2C+and+ddCTP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000858-Hollis1" id="pone.0000858-Hollis1"></a>Hollis T, Ichikawa Y, Ellenberger T (2000) DNA bending and a flip-out mechanism for base excision by the helix-hairpin-helix DNA glycosylase, <em>Escherichia coli</em> AlkA. EMBO J  19: 758–766.  <ul class="find" data-citedArticleID="988970" data-doi="10.1093/emboj/19.4.758"><li><a href="http://dx.doi.org/10.1093/emboj/19.4.758" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+bending+and+a+flip-out+mechanism+for+base+excision+by+the+helix-hairpin-helix+DNA+glycosylase%2C+Escherichia+coli+AlkA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+bending+and+a+flip-out+mechanism+for+base+excision+by+the+helix-hairpin-helix+DNA+glycosylase%2C+Escherichia+coli+AlkA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000858-Chen2" id="pone.0000858-Chen2"></a>Chen L, Haushalter KA, Lieber CM, Verdine GL (2002) Direct visualization of a DNA glycosylase searching for damage. Chem Biol  9: 345–350.  <ul class="find" data-citedArticleID="988956" data-doi="10.1016/s1074-5521(02)00120-5"><li><a href="http://dx.doi.org/10.1016/s1074-5521(02)00120-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+visualization+of+a+DNA+glycosylase+searching+for+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+visualization+of+a+DNA+glycosylase+searching+for+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000858-Zhang1" id="pone.0000858-Zhang1"></a>Zhang XP, Lee KI, Solinger JA, Kiianitsa K, Heyer WD (2005) Gly-103 in the N-terminal domain of <em>Saccharomyces cerevisiae</em> Rad51 protein is critical for DNA binding. J Biol Chem  280: 26303–26311.  <ul class="find" data-citedArticleID="989014" data-doi="10.1074/jbc.m503244200"><li><a href="http://dx.doi.org/10.1074/jbc.m503244200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gly-103+in+the+N-terminal+domain+of+Saccharomyces+cerevisiae+Rad51+protein+is+critical+for+DNA+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gly-103+in+the+N-terminal+domain+of+Saccharomyces+cerevisiae+Rad51+protein+is+critical+for+DNA+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000858-Boyer1" id="pone.0000858-Boyer1"></a>Boyer PD (1997) The ATP synthase - A splendid molecular machine. Annu Rev Biochem  66: 717–749.  <ul class="find" data-citedArticleID="988948" data-doi="10.1146/annurev.biochem.66.1.717"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.66.1.717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ATP+synthase+-+A+splendid+molecular+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ATP+synthase+-+A+splendid+molecular+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000858-Otwinowski1" id="pone.0000858-Otwinowski1"></a>Otwinowski Z, Minor W (1997) Processing of X-ray diffraction data collected in oscillation mode. Macromol Crystallogr  276: 307–326.  <ul class="find" data-citedArticleID="988982" data-doi="10.1016/s0076-6879(97)76066-x"><li><a href="http://dx.doi.org/10.1016/s0076-6879(97)76066-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000858-Brunger1" id="pone.0000858-Brunger1"></a>Brunger AT, Adams PD, Clore GM, DeLano WL, Gros P, et al.  (1998) Crystallography &amp; NMR system: A new software suite for macromolecular structure determination. Acta Crystallogr D Biol Crystallogr  54: 905–921.  <ul class="find" data-citedArticleID="988950" data-doi="10.1107/s0907444998003254"><li><a href="http://dx.doi.org/10.1107/s0907444998003254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystallography+%26amp%3B+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystallography+%26amp%3B+NMR+system%3A+A+new+software+suite+for+macromolecular+structure+determination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000858-Jones1" id="pone.0000858-Jones1"></a>Jones TA, Zou JY, Cowan SW, Kjeldgaard M (1991) Improved methods for building protein models in electron density maps and the location of errors in these models. Acta Crystallogr A  47: 110–119.  <ul class="find" data-citedArticleID="988972" data-doi="10.1107/s0108767390010224"><li><a href="http://dx.doi.org/10.1107/s0108767390010224" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+methods+for+building+protein+models+in+electron+density+maps+and+the+location+of+errors+in+these+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+methods+for+building+protein+models+in+electron+density+maps+and+the+location+of+errors+in+these+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000858-CCP41" id="pone.0000858-CCP41"></a>CCP4  (1994) The CCP4 suite: programs for protein crystallography. Acta Crystallogr D Biol Crystallogr  50: 760–763.  <ul class="find" data-citedArticleID="988952" data-doi="10.1107/s0907444994003112"><li><a href="http://dx.doi.org/10.1107/s0907444994003112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CCP4+suite%3A+programs+for+protein+crystallography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CCP4+suite%3A+programs+for+protein+crystallography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000858-Chen3" id="pone.0000858-Chen3"></a>Chen YK, Leng CH, Olivares H, Lee MH, Chang YC, et al.  (2004) Heterodimeric complexes of Hop2 and Mnd1 function with Dmc1 to promote meiotic homolog juxtaposition and strand assimilation. Proc Natl Acad Sci U S A  101: 10572–10577.  <ul class="find" data-citedArticleID="988958" data-doi="10.1073/pnas.0404195101"><li><a href="http://dx.doi.org/10.1073/pnas.0404195101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterodimeric+complexes+of+Hop2+and+Mnd1+function+with+Dmc1+to+promote+meiotic+homolog+juxtaposition+and+strand+assimilation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterodimeric+complexes+of+Hop2+and+Mnd1+function+with+Dmc1+to+promote+meiotic+homolog+juxtaposition+and+strand+assimilation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.XML" value="100490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.PDF" value="553063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g001.PNG_L" value="3674444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g001.PNG_M" value="310051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g001.PNG_S" value="18798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g001.TIF" value="5414978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g001.PNG_I" value="152515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.t001.PNG_L" value="152073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.t001.PNG_M" value="129264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.t001.PNG_S" value="17378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.t001.TIF" value="744756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.t001.PNG_I" value="82137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g002.PNG_L" value="2757043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g002.PNG_M" value="231099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g002.PNG_S" value="11907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g002.TIF" value="4011860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g002.PNG_I" value="74716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g003.PNG_L" value="4715419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g003.PNG_M" value="328959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g003.PNG_S" value="16706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g003.TIF" value="7318616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g003.PNG_I" value="108337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g004.PNG_L" value="1498974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g004.PNG_M" value="325954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g004.PNG_S" value="18205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g004.TIF" value="2169096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g004.PNG_I" value="116129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g005.PNG_L" value="923872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g005.PNG_M" value="138937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g005.PNG_S" value="7815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g005.TIF" value="889888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g005.PNG_I" value="43224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g006.PNG_L" value="728685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g006.PNG_M" value="49744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g006.PNG_S" value="6945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g006.TIF" value="1004110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g006.PNG_I" value="16838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g007.PNG_L" value="1298278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g007.PNG_M" value="177739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g007.PNG_S" value="9361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g007.TIF" value="1253098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g007.PNG_I" value="53390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g008.PNG_L" value="735190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g008.PNG_M" value="133149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g008.PNG_S" value="12683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g008.TIF" value="1255454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.g008.PNG_I" value="58714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000858.s001.DOC" value="473088"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000858&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000858" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000858&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000858&volume=&issue=&title=Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein&author_name=Li-Tzu%20Chen%2C%20Tzu-Ping%20Ko%2C%20Yu-Wei%20Chang%2C%20Kuei-An%20Lin%2C%20Andrew%20H.-J.%20Wang%2C%20Ting-Fang%20Wang&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000858" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858&amp;t=Structural%20and%20Functional%20Analyses%20of%20Five%20Conserved%20Positively%20Charged%20Residues%20in%20the%20L1%20and%20N-Terminal%20DNA%20Binding%20Motifs%20of%20Archaeal%20RadA%20Protein" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858&title=Structural%20and%20Functional%20Analyses%20of%20Five%20Conserved%20Positively%20Charged%20Residues%20in%20the%20L1%20and%20N-Terminal%20DNA%20Binding%20Motifs%20of%20Archaeal%20RadA%20Protein&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858&amp;title=Structural%20and%20Functional%20Analyses%20of%20Five%20Conserved%20Positively%20Charged%20Residues%20in%20the%20L1%20and%20N-Terminal%20DNA%20Binding%20Motifs%20of%20Archaeal%20RadA%20Protein" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000858&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Structural and Functional Analyses of Five Conserved Positively Charged Residues in the L1 and N-Terminal DNA Binding Motifs of Archaeal RadA Protein';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000858';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Structural%20and%20Functional%20Analyses%20of%20Five%20Conserved%20Positively%20Charged%20Residues%20in%20the%20L1%20and%20N-Terminal%20DNA%20Binding%20Motifs%20of%20Archaeal%20RadA%20Protein http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000858" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000858" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arginine%22" title="Search for articles in the subject area:'Arginine'"><div class="flagText">Arginine</div></a>
              <div data-categoryid="40219" data-articleid="24554"
                   data-categoryname="Arginine"
                   class="flagImage" title="Flag 'Arginine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40167" data-articleid="24554"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24554"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="24554"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleoproteins%22" title="Search for articles in the subject area:'Nucleoproteins'"><div class="flagText">Nucleoproteins</div></a>
              <div data-categoryid="26853" data-articleid="24554"
                   data-categoryname="Nucleoproteins"
                   class="flagImage" title="Flag 'Nucleoproteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interactions%22" title="Search for articles in the subject area:'Protein interactions'"><div class="flagText">Protein interactions</div></a>
              <div data-categoryid="17661" data-articleid="24554"
                   data-categoryname="Protein interactions"
                   class="flagImage" title="Flag 'Protein interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="24554"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24554"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6934'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8551'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6301&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
