

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001424"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001424" />

    <meta name="citation_title" content="Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads"/>
    <meta itemprop="name" content="Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads"/>

      <meta name="citation_author" content="Morgane Rolland"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="David Heckerman"/>
            <meta name="citation_author_institution" content="Machine Learning and Applied Statistics Group, Microsoft Research, Redmond, Washington, United States of America"/>
      <meta name="citation_author" content="Wenjie Deng"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Christine M. Rousseau"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>
      <meta name="citation_author" content="Hoosen Coovadia"/>
            <meta name="citation_author_institution" content="HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa"/>
      <meta name="citation_author" content="Karen Bishop"/>
            <meta name="citation_author_institution" content="HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa"/>
      <meta name="citation_author" content="Philip J. R. Goulder"/>
            <meta name="citation_author_institution" content="HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa"/>
            <meta name="citation_author_institution" content="Nuffield Department of Medicine, Oxford University, Oxford, United Kingdom"/>
            <meta name="citation_author_institution" content="Partners AIDS Research Center, Massachusetts General Hospital, Harvard Medical School, Charlestown, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Bruce D. Walker"/>
            <meta name="citation_author_institution" content="HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa"/>
            <meta name="citation_author_institution" content="Partners AIDS Research Center, Massachusetts General Hospital, Harvard Medical School, Charlestown, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Christian Brander"/>
            <meta name="citation_author_institution" content="Partners AIDS Research Center, Massachusetts General Hospital, Harvard Medical School, Charlestown, Massachusetts, United States of America"/>
      <meta name="citation_author" content="James I. Mullins"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America"/>

    <meta name="citation_date" content="2008/1/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001424.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1424"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Consistent cytotoxic-T-lymphocyte targeting of immunodominant regions in human immunodeficiency virus across multiple ethnicities.; citation_author=N Frahm; citation_author=BT Korber; citation_author=CM Adams; citation_author=JJ Szinger; citation_author=R Draenert; citation_journal_title=J Virol; citation_volume=78; citation_number=1; citation_pages=2187-2200; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Relative dominance of Gag p24-specific cytotoxic T lymphocytes is associated with human immunodeficiency virus control.; citation_author=R Zuniga; citation_author=A Lucchetti; citation_author=P Galvan; citation_author=S Sanchez; citation_author=C Sanchez; citation_journal_title=J Virol; citation_volume=80; citation_number=2; citation_pages=3122-3125; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hierarchical targeting of subtype C human immunodeficiency virus type 1 proteins by CD8+ T cells: correlation with viral load.; citation_author=A Masemola; citation_author=T Mashishi; citation_author=G Khoury; citation_author=P Mohube; citation_author=P Mokgotho; citation_journal_title=J Virol; citation_volume=78; citation_number=3; citation_pages=3233-3243; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Maintenance of large numbers of virus-specific CD8+ T cells in HIV-infected progressors and long-term nonprogressors.; citation_author=JC Gea-Banacloche; citation_author=SA Migueles; citation_author=L Martino; citation_author=WL Shupert; citation_author=AC McNeil; citation_journal_title=J Immunol; citation_volume=165; citation_number=4; citation_pages=1082-1092; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Magnitude of functional CD8+ T-cell responses to the gag protein of human immunodeficiency virus type 1 correlates inversely with viral load in plasma.; citation_author=BH Edwards; citation_author=A Bansal; citation_author=S Sabbaj; citation_author=J Bakari; citation_author=MJ Mulligan; citation_journal_title=J Virol; citation_volume=76; citation_number=5; citation_pages=2298-2305; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of total human immunodeficiency virus (HIV)-specific CD4(+) and CD8(+) T-cell responses: relationship to viral load in untreated HIV infection.; citation_author=MR Betts; citation_author=DR Ambrozak; citation_author=DC Douek; citation_author=S Bonhoeffer; citation_author=JM Brenchley; citation_journal_title=J Virol; citation_volume=75; citation_number=6; citation_pages=11983-11991; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comprehensive epitope analysis of human immunodeficiency virus type 1 (HIV-1)-specific T-cell responses directed against the entire expressed HIV-1 genome demonstrate broadly directed responses, but no correlation to viral load.; citation_author=MM Addo; citation_author=XG Yu; citation_author=A Rathod; citation_author=D Cohen; citation_author=RL Eldridge; citation_journal_title=J Virol; citation_volume=77; citation_number=7; citation_pages=2081-2092; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytotoxic-T-cell responses, viral load, and disease progression in early human immunodeficiency virus type 1 infection.; citation_author=L Musey; citation_author=J Hughes; citation_author=T Schacker; citation_author=T Shea; citation_author=L Corey; citation_journal_title=N Engl J Med; citation_volume=337; citation_number=8; citation_pages=1267-1274; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evolution of HIV-1 CTL epitopes: Fitness-Balanced Escape.; citation_author=Y Liu; citation_author=J McNevin; citation_author=H Zhao; citation_author=DM Tebit; citation_author=M McSweyn; citation_journal_title=ePub; citation_number=9; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Association between virus-specific T-cell responses and plasma viral load in human immunodeficiency virus type 1 subtype C infection.; citation_author=V Novitsky; citation_author=P Gilbert; citation_author=T Peter; citation_author=MF McLane; citation_author=S Gaolekwe; citation_journal_title=J Virol; citation_volume=77; citation_number=10; citation_pages=882-890; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Quantitation of HIV-1-specific T lymphocytes and plasma viral load of viral RNA.; citation_author=GS Ogg; citation_author=X Jin; citation_author=S Bonhoeffer; citation_author=PR Dunbar; citation_author=MA Nowak; citation_journal_title=Science; citation_volume=279; citation_number=11; citation_pages=2103-2106; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Gag-specific cytotoxic responses to HIV type 1 are associated with a decreased risk of progression to AIDS-related complex or AIDS.; citation_author=Y Riviere; citation_author=MB McChesney; citation_author=F Porrot; citation_author=F Tanneau-Salvadori; citation_author=P Sansonetti; citation_journal_title=AIDS Res Hum Retroviruses; citation_volume=11; citation_number=12; citation_pages=903-907; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=CD8 T-cell recognition of multiple epitopes within specific Gag regions is associated with maintenance of a low steady-state viremia in human immunodeficiency virus type 1-seropositive patients.; citation_author=C Geldmacher; citation_author=JR Currier; citation_author=E Herrmann; citation_author=A Haule; citation_author=E Kuta; citation_journal_title=J Virol; citation_volume=81; citation_number=13; citation_pages=2440-2448; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gag-specific CD8+ T lymphocytes recognize infected cells before AIDS-virus integration and viral protein expression.; citation_author=JB Sacha; citation_author=C Chung; citation_author=EG Rakasz; citation_author=SP Spencer; citation_author=AK Jonas; citation_journal_title=J Immunol; citation_volume=178; citation_number=14; citation_pages=2746-2754; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Differential immunogenicity of HIV-1 clade C proteins in eliciting CD8+ and CD4+ cell responses.; citation_author=D Ramduth; citation_author=P Chetty; citation_author=NC Mngquandaniso; citation_author=N Nene; citation_author=JD Harlow; citation_journal_title=J Infect Dis; citation_volume=192; citation_number=15; citation_pages=1588-1596; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Compensatory Mutation Partially Restores Fitness and Delays Reversion of Escape Mutation within the Immunodominant Hla-B*5703-Restricted Gag Epitope in Chronic Hiv-1 Infection.; citation_author=H Crawford; citation_author=JG Prado; citation_author=A Leslie; citation_author=S Hue; citation_author=I Honeyborne; citation_journal_title=J Virol; citation_number=16; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Loss of HIV-1-specific T-cell responses associated with very rapid HIV-1 disease progression.; citation_author=H Streeck; citation_author=B Schweighardt; citation_author=H Jessen; citation_author=RL Allgaier; citation_author=T Wrin; citation_journal_title=Aids; citation_volume=21; citation_number=17; citation_pages=889-891; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Control of human immunodeficiency virus type 1 is associated with HLA-B*13 and targeting of multiple gag-specific CD8+ T-cell epitopes.; citation_author=I Honeyborne; citation_author=A Prendergast; citation_author=F Pereyra; citation_author=A Leslie; citation_author=H Crawford; citation_journal_title=J Virol; citation_volume=81; citation_number=18; citation_pages=3667-3672; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The influence of HLA genotype on AIDS.; citation_author=M Carrington; citation_author=SJ O'Brien; citation_journal_title=Annu Rev Med; citation_volume=54; citation_number=19; citation_pages=535-551; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=HLA B*5701 is highly associated with restriction of virus replication in a subgroup of HIV-infected long term nonprogressors.; citation_author=SA Migueles; citation_author=MS Sabbaghian; citation_author=WL Shupert; citation_author=MP Bettinotti; citation_author=FM Marincola; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=20; citation_pages=2709-2714; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Advantage of rare HLA supertype in HIV disease progression.; citation_author=E Trachtenberg; citation_author=B Korber; citation_author=C Sollars; citation_author=TB Kepler; citation_author=PT Hraber; citation_journal_title=Nat Med; citation_volume=9; citation_number=21; citation_pages=928-935; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=HLA and AIDS: a cautionary tale.; citation_author=SJ O'Brien; citation_author=X Gao; citation_author=M Carrington; citation_journal_title=Trends Mol Med; citation_volume=7; citation_number=22; citation_pages=379-381; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effect of a single amino acid change in MHC class I molecules on the rate of progression to AIDS.; citation_author=X Gao; citation_author=GW Nelson; citation_author=P Karacki; citation_author=MP Martin; citation_author=J Phair; citation_journal_title=N Engl J Med; citation_volume=344; citation_number=23; citation_pages=1668-1675; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HLA and HIV-1: heterozygote advantage and B*35-Cw*04 disadvantage [see comments].; citation_author=M Carrington; citation_author=GW Nelson; citation_author=MP Martin; citation_author=T Kissner; citation_author=D Vlahov; citation_journal_title=Science; citation_volume=283; citation_number=24; citation_pages=1748-1752; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA.; citation_author=P Kiepiela; citation_author=AJ Leslie; citation_author=I Honeyborne; citation_author=D Ramduth; citation_author=C Thobakgale; citation_journal_title=Nature; citation_volume=432; citation_number=25; citation_pages=769-775; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HLA-B63 presents HLA-B57/B58-restricted cytotoxic T-lymphocyte epitopes and is associated with low human immunodeficiency virus load.; citation_author=N Frahm; citation_author=S Adams; citation_author=P Kiepiela; citation_author=CH Linde; citation_author=HS Hewitt; citation_journal_title=J Virol; citation_volume=79; citation_number=26; citation_pages=10218-10225; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Control of human immunodeficiency virus replication by cytotoxic T lymphocytes targeting subdominant epitopes.; citation_author=N Frahm; citation_author=P Kiepiela; citation_author=S Adams; citation_author=CH Linde; citation_author=HS Hewitt; citation_journal_title=Nat Immunol; citation_volume=7; citation_number=27; citation_pages=173-178; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The HLA A2/6802 supertype is associated with reduced risk of perinatal human immunodeficiency virus type 1 transmission.; citation_author=KS MacDonald; citation_author=JE Embree; citation_author=NJ Nagelkerke; citation_author=J Castillo; citation_author=S Ramhadin; citation_journal_title=J Infect Dis; citation_volume=183; citation_number=28; citation_pages=503-506; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Influence of HLA supertypes on susceptibility and resistance to human immunodeficiency virus type 1 infection.; citation_author=KS MacDonald; citation_author=KR Fowke; citation_author=J Kimani; citation_author=VA Dunand; citation_author=NJ Nagelkerke; citation_journal_title=J Infect Dis; citation_volume=181; citation_number=29; citation_pages=1581-1589; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Determinants of human immunodeficiency virus type 1 escape from the primary CD8+ cytotoxic T lymphocyte response.; citation_author=NA Jones; citation_author=X Wei; citation_author=DR Flower; citation_author=M Wong; citation_author=F Michor; citation_journal_title=J Exp Med; citation_volume=200; citation_number=30; citation_pages=1243-1256; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Selective escape from CD8+ T-cell responses represents a major driving force of human immunodeficiency virus type 1 (HIV-1) sequence diversity and reveals constraints on HIV-1 evolution.; citation_author=TM Allen; citation_author=M Altfeld; citation_author=SC Geer; citation_author=ET Kalife; citation_author=C Moore; citation_journal_title=J Virol; citation_volume=79; citation_number=31; citation_pages=13239-13249; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selection on the human immunodeficiency virus type 1 proteome following primary infection.; citation_author=Y Liu; citation_author=J McNevin; citation_author=J Cao; citation_author=H Zhao; citation_author=I Genowati; citation_journal_title=J Virol; citation_volume=80; citation_number=32; citation_pages=9519-9529; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prediction of an HLA B8-restricted influenza epitope by motif.; citation_author=A Suhrbier; citation_author=C Schmidt; citation_author=A Fernan; citation_journal_title=Immunology; citation_volume=79; citation_number=33; citation_pages=171-173; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=An integrative approach to CTL epitope prediction: a combined algorithm integrating MHC class I binding, TAP transport efficiency, and proteasomal cleavage predictions.; citation_author=MV Larsen; citation_author=C Lundegaard; citation_author=K Lamberth; citation_author=S Buus; citation_author=S Brunak; citation_journal_title=Eur J Immunol; citation_volume=35; citation_number=34; citation_pages=2295-2303; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The validity of predicted T-cell epitopes.; citation_author=C Lundegaard; citation_author=M Nielsen; citation_author=O Lund; citation_journal_title=Trends Biotechnol; citation_volume=24; citation_number=35; citation_pages=537-538; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=T-cell epitope repertoire as predicted from human and viral genomes.; citation_author=Y Louzoun; citation_author=T Vider; citation_author=M Weigert; citation_journal_title=Mol Immunol; citation_volume=43; citation_number=36; citation_pages=559-569; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Leveraging information across HLA alleles/supertypes improves epitope prediction.; citation_author=D Heckerman; citation_author=C Kadie; citation_author=J Listgarten; citation_number=37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale amplification, cloning and sequencing of near full-length HIV-1 subtype C genomes.; citation_author=C Rousseau; citation_author=BA Birditt; citation_author=AR McKay; citation_author=JN Stoddard; citation_author=TC Lee; citation_journal_title=J Virol Methods; citation_volume=136; citation_number=38; citation_pages=118-125; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Extensive Intra-subtype Recombination in South African HIV-1 Subtype C Infections.; citation_author=CM Rousseau; citation_author=GH Learn; citation_author=T Bhattacharya; citation_author=DC Nickle; citation_author=D Heckerman; citation_journal_title=J Virol; citation_volume=81; citation_number=39; citation_pages=4492-4500; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Divergent patterns of progression to AIDS after infection from the same source: human immunodeficiency virus type 1 evolution and antiviral responses.; citation_author=SL Liu; citation_author=T Schacker; citation_author=L Musey; citation_author=D Shriner; citation_author=MJ McElrath; citation_journal_title=J Virol; citation_volume=71; citation_number=40; citation_pages=4284-4295; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Virus population homogenization following acute human immunodeficiency virus type 1 infection.; citation_author=GH Learn; citation_author=D Muthui; citation_author=SJ Brodie; citation_author=T Zhu; citation_author=K Diem; citation_journal_title=J Virol; citation_volume=76; citation_number=41; citation_pages=11953-11959; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Concordance between HIV source partner identification and molecular confirmation in acute retroviral syndrome.; citation_author=HM Truong; citation_author=MM Berrey; citation_author=T Shea; citation_author=K Diem; citation_author=L Corey; citation_journal_title=J Acquir Immune Defic Syndr; citation_volume=29; citation_number=42; citation_pages=232-243; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Clinical and epidemiologic features of primary HIV infection.; citation_author=T Schacker; citation_author=AC Collier; citation_author=J Hughes; citation_author=T Shea; citation_author=L Corey; citation_journal_title=Ann Intern Med; citation_volume=125; citation_number=43; citation_pages=257-264; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=CD8+ T-cell responses to different HIV proteins have discordant associations with viral load.; citation_author=P Kiepiela; citation_author=K Ngumbela; citation_author=C Thobakgale; citation_author=D Ramduth; citation_author=I Honeyborne; citation_journal_title=Nat Med; citation_volume=13; citation_number=44; citation_pages=46-53; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Extensive HLA class I allele promiscuity among viral CTL epitopes.; citation_author=N Frahm; citation_author=K Yusim; citation_author=TJ Suscovich; citation_author=S Adams; citation_author=J Sidney; citation_journal_title=Eur J Immunol; citation_volume=37; citation_number=45; citation_pages=2419-2433; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Differential selection pressure exerted on HIV by CTL targeting identical epitopes but restricted by distinct HLA alleles from the same HLA supertype.; citation_author=A Leslie; citation_author=DA Price; citation_author=P Mkhize; citation_author=K Bishop; citation_author=A Rathod; citation_journal_title=J Immunol; citation_volume=177; citation_number=46; citation_pages=4699-4708; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24.; citation_author=JA Borghans; citation_author=A Molgaard; citation_author=RJ de Boer; citation_author=C Kesmir; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=47; citation_pages=e920; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A new, core elements approach to vaccine immunogen design.; citation_author=M Rolland; citation_author=DC Nickle; citation_author=JI Mullins; citation_journal_title=PLoS Pathogens: In press; citation_number=48; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Hitting HIV where it hurts: an alternative approach to HIV vaccine design.; citation_author=M Altfeld; citation_author=TM Allen; citation_journal_title=Trends Immunol; citation_volume=27; citation_number=49; citation_pages=504-510; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV Molecular Immunology 2005.; citation_author=BTM Korber; citation_author=C Brander; citation_author=BF Haynes; citation_author=R Koup; citation_author=JP Moore; citation_number=50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt): an expanding universe of protein information.; citation_author=CH Wu; citation_author=R Apweiler; citation_author=A Bairoch; citation_author=DA Natale; citation_author=WC Barker; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=51; citation_pages=D187-191; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads"/>
    <meta name="twitter:description" content="BackgroundHLA class-I alleles differ in their ability to control HIV replication through cell-mediated immune responses. No consistent associations have been found between the breadth of Cytotoxic T Lymphocytes (CTL) responses and the control of HIV-1, and it is unknown whether the size or distribution of the viral proteome-wide epitope repertoire, i.e., the intrinsic ability to present fewer, more or specific viral epitopes, could affect clinical markers of disease progression.Methodology/Principal FindingsWe used an epitope prediction model to identify all epitope motifs in a set of 302 HIV-1 full-length proteomes according to each individual's HLA (Human Leukocyte Antigen) genotype. The epitope repertoire, i.e., the number of predicted epitopes per HIV-1 proteome, varied considerably between HLA alleles and thus among individual proteomes. In a subgroup of 270 chronically infected individuals, we found that lower viral loads and higher CD4 counts were associated with a larger predicted epitope repertoire. Additionally, in Gag and Rev only, more epitopes were restricted by alleles associated with low viral loads than by alleles associated with higher viral loads.Conclusions/SignificanceThis comprehensive analysis puts forth the epitope repertoire as a mechanistic component of the multi-faceted HIV-specific CTL response. The favorable impact on markers of disease status of the propensity to present more HLA binding peptides and specific proteins gives impetus to vaccine design strategies that seek to elicit responses to a broad array of HIV-1 epitopes, and suggest a particular focus on Gag."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001424.g002"/>

  <meta property="og:title" content="Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2739'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=486'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6429&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001424">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001424" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001424&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Morgane Rolland, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Heckerman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Machine Learning and Applied Statistics Group, Microsoft Research, Redmond, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wenjie Deng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christine M. Rousseau, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hoosen Coovadia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karen Bishop, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philip J. R. Goulder, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa, 
                  Nuffield Department of Medicine, Oxford University, Oxford, United Kingdom, 
                  Partners AIDS Research Center, Massachusetts General Hospital, Harvard Medical School, Charlestown, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bruce D. Walker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  HIV Pathogenesis Programme, Doris Duke Medical Research Institute, University of KwaZulu-Natal, Durban, South Africa, 
                  Partners AIDS Research Center, Massachusetts General Hospital, Harvard Medical School, Charlestown, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Brander, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Partners AIDS Research Center, Massachusetts General Hospital, Harvard Medical School, Charlestown, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James I. Mullins
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jmullins@u.washington.edu">jmullins@u.washington.edu</a></p>

                <p>Affiliation:
                  Department of Microbiology, University of Washington School of Medicine, Seattle, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001424</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001424-g001" data-doi="info:doi/10.1371/journal.pone.0001424" data-uri="info:doi/10.1371/journal.pone.0001424.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001424.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001424-g002" data-doi="info:doi/10.1371/journal.pone.0001424" data-uri="info:doi/10.1371/journal.pone.0001424.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001424.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001424">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001424" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>HLA class-I alleles differ in their ability to control HIV replication through cell-mediated immune responses. No consistent associations have been found between the breadth of Cytotoxic T Lymphocytes (CTL) responses and the control of HIV-1, and it is unknown whether the size or distribution of the viral proteome-wide epitope repertoire, i.e., the intrinsic ability to present fewer, more or specific viral epitopes, could affect clinical markers of disease progression.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We used an epitope prediction model to identify all epitope motifs in a set of 302 HIV-1 full-length proteomes according to each individual's HLA (Human Leukocyte Antigen) genotype. The epitope repertoire, i.e., the number of predicted epitopes per HIV-1 proteome, varied considerably between HLA alleles and thus among individual proteomes. In a subgroup of 270 chronically infected individuals, we found that lower viral loads and higher CD4 counts were associated with a larger predicted epitope repertoire. Additionally, in Gag and Rev only, more epitopes were restricted by alleles associated with low viral loads than by alleles associated with higher viral loads.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This comprehensive analysis puts forth the epitope repertoire as a mechanistic component of the multi-faceted HIV-specific CTL response. The favorable impact on markers of disease status of the propensity to present more HLA binding peptides and specific proteins gives impetus to vaccine design strategies that seek to elicit responses to a broad array of HIV-1 epitopes, and suggest a particular focus on Gag.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rolland M, Heckerman D, Deng W, Rousseau CM, Coovadia H, et al.  (2008) Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads. PLoS ONE 3(1):
          e1424.
            doi:10.1371/journal.pone.0001424</p><p><strong>Academic Editor: </strong>Douglas Nixon, University of California at San Francisco, United States of America</p><p><strong>Received:</strong> September 26, 2007; <strong>Accepted:</strong> October 12, 2007; <strong>Published:</strong> January 9, 2008</p><p><strong>Copyright:</strong> © 2008 Rolland et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by an unrestricted gift from the Boeing Corporation and US Public Health Service grants to JIM (AI57005, AI058894) and the University of Washington Center for AIDS Research (AI27757), and by Microsoft Research. These sponsors had no role in any aspect of the design preparation or submission of this work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Although attempts to correlate the breadth of the CTL antiviral response and control of HIV-1 infection <em>in vivo</em> have been equivocal <a href="#pone.0001424-Frahm1">[1]</a>, <a href="#pone.0001424-Zuniga1">[2]</a>, <a href="#pone.0001424-Masemola1">[3]</a>, <a href="#pone.0001424-GeaBanacloche1">[4]</a>, <a href="#pone.0001424-Edwards1">[5]</a>, <a href="#pone.0001424-Betts1">[6]</a>, <a href="#pone.0001424-Addo1">[7]</a>, <a href="#pone.0001424-Musey1">[8]</a>, <a href="#pone.0001424-Liu1">[9]</a>, accumulating evidence support the beneficial role of Gag-specific CTL responses in HIV-1 containment <a href="#pone.0001424-Zuniga1">[2]</a>, <a href="#pone.0001424-Novitsky1">[10]</a>, <a href="#pone.0001424-Ogg1">[11]</a>, <a href="#pone.0001424-Riviere1">[12]</a>, <a href="#pone.0001424-Geldmacher1">[13]</a>, <a href="#pone.0001424-Sacha1">[14]</a>, <a href="#pone.0001424-Ramduth1">[15]</a>, <a href="#pone.0001424-Crawford1">[16]</a>, <a href="#pone.0001424-Streeck1">[17]</a>, <a href="#pone.0001424-Honeyborne1">[18]</a> - alluding to differences in antiviral efficacy among specific CD8<sup>+</sup> T-cell responses. Additionally, certain HLA genes are associated with different rates of disease progression <a href="#pone.0001424-Carrington1">[19]</a>, <a href="#pone.0001424-Migueles1">[20]</a>, <a href="#pone.0001424-Trachtenberg1">[21]</a>, <a href="#pone.0001424-OBrien1">[22]</a>, <a href="#pone.0001424-Gao1">[23]</a>, <a href="#pone.0001424-Carrington2">[24]</a>, <a href="#pone.0001424-Kiepiela1">[25]</a>, <a href="#pone.0001424-Frahm2">[26]</a>, <a href="#pone.0001424-Frahm3">[27]</a>, <a href="#pone.0001424-MacDonald1">[28]</a>, <a href="#pone.0001424-MacDonald2">[29]</a>. In particular, HLA alleles B*27 and B*57 seem to confer a survival benefit whereas HLA B*35_Px confers a survival disadvantage <a href="#pone.0001424-OBrien1">[22]</a>. While HLA-B alleles appear to impact disease progression more than other alleles <a href="#pone.0001424-Kiepiela1">[25]</a>, possessing heterologous HLA alleles, and thus possibly allowing the individual to present broader arrays of epitopes, has also been associated with delayed/slower disease progression <a href="#pone.0001424-Carrington2">[24]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The extent of polymorphisms in HIV sequences and in HLA loci underscores that a vast array of HLA/viral peptide combinations can be generated during infection. HIV-1 is characterized by its extensive diversity both among and within infected individuals due to its extreme capacity to mutate, its persistent replication, and the important role of CD8<sup>+</sup> lymphocyte responses in driving viral evolution <a href="#pone.0001424-Jones1">[30]</a>, <a href="#pone.0001424-Allen1">[31]</a>, <a href="#pone.0001424-Liu2">[32]</a>. In addition, the HLA region is one of the most polymorphic loci in the human genome. HLA class I molecules are codominantly expressed on antigen-presenting cells such that all six allotypes (if the person expresses heterologous HLA-A, -B and -C proteins) can present viral epitopes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The fine-mapping of epitopes, typically 9 amino acid (AA) long but ranging from 8 to 11 AA, together with data on their binding properties to HLA molecules, has allowed definition of HLA class I allele-specific sequence motifs that are able to prime virus-specific CD8<sup>+</sup> T-cell responses. Consistent associations between HLA alleles and disease outcomes suggest an underlying mechanistic function, and prompted us to question whether the scope of the epitope repertoire contributes to the composite effectiveness of the CTL response. “Epitope repertoire” here refers to all viral peptide sequences that fulfill HLA class I allele-specific binding motifs for a specific whole HIV-1 proteome.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Epitope mapping data has been used to develop computational methods of epitope prediction, which are important for the development of diagnostic tools and the design and evaluation of vaccines. Identification of novel HIV-1 epitopes simultaneously fuels our greater understanding of the immune recognition of the HIV proteome and incremental improvements of epitope prediction algorithms <a href="#pone.0001424-Suhrbier1">[33]</a>, <a href="#pone.0001424-Larsen1">[34]</a>, <a href="#pone.0001424-Lundegaard1">[35]</a>, <a href="#pone.0001424-Louzoun1">[36]</a>. Here, we predicted HLA class I epitopes using a new method based on logistic regression and designed to leverage data across HLA alleles and/or supertypes (<a href="#pone.0001424-Heckerman1">[37]</a> available at <a href="http://atom.research.microsoft.com/hlabinding/hlabinding.aspx">http://atom.research.microsoft.com/hlabi​nding/hlabinding.aspx</a>). The prediction method produces approximately 10% false positive results when set to yield 10% false negatives.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We predicted the epitope repertoire in 302 full-length HIV-1 proteomes, isolated from 302 untreated individuals infected with HIV-1 subtype C and B, based on each subject's HLA genotype <a href="#pone.0001424-Liu2">[32]</a>, <a href="#pone.0001424-Rousseau1">[38]</a>, <a href="#pone.0001424-Rousseau2">[39]</a>, <a href="#pone.0001424-Liu3">[40]</a>, <a href="#pone.0001424-Learn1">[41]</a>, <a href="#pone.0001424-Truong1">[42]</a>, <a href="#pone.0001424-Schacker1">[43]</a>, <a href="#pone.0001424-Kiepiela2">[44]</a>. We report that a larger epitope repertoire was associated with lower levels of viremia. Furthermore, alleles associated with reduced viral loads tended to target particularly Gag when compared to alleles associated with a lack of control of viral replication.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>The size of the epitope repertoire differed between HLA alleles and thereby between autologous HIV-1 proteomes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Scanning 302 HIV-1 proteomes, corresponding to 2,718 HIV-1 protein sequences, resulted in the identification of 22,779 epitope motifs, including 8,208 experimentally defined CTL epitopes compiled in databases prior to our study (the latter two figures include redundancies from detection of the same epitopes in multiple individuals). The number of predicted epitopes varied greatly among alleles, i.e., between 1 and 47 per allele per proteome, and henceforth among individuals ranging between 14 and 186 per proteome (mean = 75; median = 72). Given that the vast majority of known epitopes were defined experimentally using peptides corresponding to subtype B, more CTL epitopes are known for subtype B than for subtype C. Thus in turn, more motifs were identified in proteomes from subtype B (mean = 113 per proteome, including 60 previously known epitopes) than in subtype C (mean = 71, including 23 previously known epitopes). To rule out this experimental bias toward HIV-1 subtype B in our study, we verified that the number of epitopes restricted by an allele was not associated with the allele frequency in the population. There were no associations between the number of epitopes restricted by an allele and its frequency in the population in a subgroup of 32 individuals from the Seattle Primary Infection Cohort (r<sup>2</sup> = 0.0280; p = 0.2307), nor in a subtype C infected South African cohort from Durban (r<sup>2</sup> = 0.0347; p = 0.1998; n = 270 individuals) or in a representative Sub-Saharan population (r<sup>2</sup> = 0.0319; p = 0.2197). However, there was a positive relationship between the number of epitopes and the allele frequency in the overall North American population (r<sup>2</sup> = 0.1153; p = 0.0129; n = 1021 individuals), likely reflecting the focus of HIV/AIDS research on this population. Interestingly, HLA B*27, an allele repeatedly associated with favorable disease outcomes <a href="#pone.0001424-OBrien1">[22]</a> and found 3 times in our dataset, presented the third largest epitope repertoire with 42 predicted epitopes per HIV-1 proteome, i.e., over 3 times the average repertoire size (Mean number of epitopes/HLA allele = 13.87; Confidence Interval (CI) with α = 0.99, Lower CI = 9.46; Upper CI = 18.27).</p>


<h4>Relationship between epitope repertoires and clinical data</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We compared numbers of predicted epitopes per proteome to viral loads and CD4 counts in subtype C infected individuals from the South African cohort. We found that the more epitopes predicted for an individual, the lower the observed viral load (r<sup>2</sup> = 0.0446, p = 0.0005; Spearman's correlation factor: Rho = −0.1751, p = 0.0039). In particular, we found a stronger negative relationship between the size of the epitope repertoire and the viral loads among the 81 individuals who had CD4 counts above 400, i.e., when we excluded from the analysis the individuals with vanishing T cell numbers, and presumably function (r<sup>2</sup> = 0.1009, p = 0.0038; Spearman's correlation factor: Rho = −0.3090, p = 0.0050) (<a href="#pone-0001424-g001">Figure 1A</a>). Additionally, larger epitope repertoires were associated with higher CD4 counts (r<sup>2</sup> = 0.0620, p = 0.0250; Spearman's Rho = 0.1549, p = 0.0395) (<a href="#pone-0001424-g001">Figure 1B</a>). A relatively weaker association was observed for CD4 than for viral loads, possibly due to CD4 counts being available for only 177 of the 270 individuals evaluated.</p>
<div class="figure" id="pone-0001424-g001"><div class="img"><a name="pone-0001424-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001424.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001424" data-uri="info:doi/10.1371/journal.pone.0001424.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001424.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001424.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001424.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001424.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001424.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001424.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001424.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001424.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001424.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>HIV-1 epitope repertoires and clinical data.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Putative epitopes were identified <em>in silico</em> within full-length autologous HIV-1 proteomes and combined with previously described optimally defined CTL epitopes found in the LANL and IEDB databases. (A) and (B) respectively show the log viral loads and the CD4 counts plotted as a function of the number of predicted HIV-1 epitopes identified per proteome for the individuals with CD4 counts above 400 (n = 81). (C) Shows the number of predicted HIV-1 epitopes for individuals belonging to the highest (mean = 65) and lowest (mean = 76) viremia quartile. (D) Shows the average viral loads of individuals presenting a specific allele as a function of the average number of HIV-1 predicted epitopes for that allele (only alleles presented by at least 3 individuals in the South-African cohort were included).</p>
<span>doi:10.1371/journal.pone.0001424.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>By grouping individuals according to their plasma viral loads, we found significantly different numbers of predicted HIV-1 epitopes in individuals within the lowest (&lt;16,437 viral RNA copies/ml; n = 67) and highest (&gt;186,250 copies; n = 67) quartiles. HIV-1 proteomes from individuals in the quartile with the lowest viral load had a mean number of 80 predicted epitopes, compared to 67 in the highest quartile (p =  0.0047) (<a href="#pone-0001424-g001">Figure 1C</a>). There was also a trend for individuals with higher CD4 counts to have more predicted epitopes, the mean number was 76 for individuals in the highest quartile (n = 45; CD4&gt;521.5), and 65 in the lowest quartile (n = 44; CD4&lt;234.5) (p = 0.0686).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The least frequent alleles in the cohort were found to be associated with lower viral loads (Spearman's Rho = 0.2880; p = 0.0448), in agreement with Trachtenberg and colleagues <a href="#pone.0001424-Trachtenberg1">[21]</a>. And, we found a trend indicating that HLA alleles restricting larger repertoires were associated with lower viral loads in HLA-matched individuals (<a href="#pone-0001424-g001">Figure 1D</a>) (Spearman's Rho = −0.2952; p = 0.0517).</p>


<h4>Distribution of epitope repertoires vary between HLA alleles associated with different viral loads</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Next, we ranked HLA alleles by the average viral loads of subjects in the Durban cohort: the quartile with the lowest viral loads (&lt;125,437 viral copies; mean = 65,384; median = 58,229) included 12 alleles, herein referred as “good” alleles; the quartile with the highest viral loads (&gt;320,643 viral copies; mean = 971,587; median = 531,20) included 12 “bad” alleles. Interestingly, the distribution of predicted epitopes among HIV-1 proteins revealed that “good” HLA alleles focused more on Gag (<a href="#pone-0001424-g002">Figure 2A</a>) and less on Nef (<a href="#pone-0001424-g002">Figure 2B</a>). For “good” HLA alleles, predicted Gag epitopes increased 1.69 fold (p = 0.036) compared to the distribution found for “bad” HLA alleles, while predicted Nef epitopes decreased 2.35 fold (p = 0.038). When analyzed by individual protein, Gag- and Rev-specific repertoires showed more epitopes restricted by “good” HLA alleles than by “bad” ones, whereas there were more epitopes restricted by “bad” HLA alleles than by “good” ones in Nef, Env, Pol, Tat, Vif, Vpu, and also Vpr (albeit marginally) (<a href="#pone-0001424-g002">Figure 2C</a>). Nef- and Gag-specific epitope repertoires showed similar percentages of epitopes restricted by “good” alleles, however, the proportion of epitopes restricted by “bad” alleles was significantly higher in Nef compared to its proportion in Gag.</p>
<div class="figure" id="pone-0001424-g002"><div class="img"><a name="pone-0001424-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001424.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001424" data-uri="info:doi/10.1371/journal.pone.0001424.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001424.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001424.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001424.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001424.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001424.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001424.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001424.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001424.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001424.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of epitopes by HLA alleles and by protein.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Distribution of epitopes among HIV-1 proteins for HLA alleles associated with lowest/highest viral loads. The ratio of predicted epitopes predicted for each protein corresponded to the number of epitope-fulfilling motifs identified in each protein over the total number of epitopes identified for the whole proteome. (A) Shows the distribution of epitopes for “good” alleles, i.e., those associated with the lowest viral loads in the cohort (lowest quartile: VL&lt;125,437; mean = 65,384; median = 58,229). (B) Shows the epitope distribution for “bad” alleles, those associated with the highest viral loads in the cohort (highest quartile: VL&gt;320,643; mean = 971,587; median = 531,208). For each allele belonging to a quartile, average values per allele were calculated based on the viral loads of HLA-matched individuals). (C) Illustrates the percentage of epitopes restricted by “good” and “bad” HLA alleles for each protein.</p>
<span>doi:10.1371/journal.pone.0001424.g002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We systematically examined the immunogenic potential of HIV-1 at the population level through <em>in silico</em> estimation of the epitope repertoire of 302 HIV-1 proteomes. The number of predicted HIV-1 epitopes per proteome varied considerably between HLA alleles and thereby among individuals. Additionally, there were more epitopes identified in subtype B viruses than in subtype C, reflecting the existing bias of databases for inclusion of data from subtype B viruses and subtype B-infected individuals. Importantly, while we demonstrated that our analysis was not confounded by this experimental bias, it also highlights the need for better characterization of CTL responses against HIV-1 subtype C in the affected population (i.e., with typical motifs and HLA allele restrictions). Nonetheless, limitations to epitope prediction analyses intrinsically include biases derived from their training datasets, the fact that certain epitopes are not optimally defined or have incorrect HLA alleles restrictions (e.g., due to linkage disequilibrium) and pervasive of HLA class I allele promiscuity <a href="#pone.0001424-Frahm4">[45]</a>. Despite those potential shortcomings, our findings corroborate those from immunological studies in this cohort. Principally, individuals with high viral loads tended to target preferentially Env and Accesssory/Regulatory proteins <a href="#pone.0001424-Honeyborne1">[18]</a>, <a href="#pone.0001424-Frahm3">[27]</a>, <a href="#pone.0001424-Kiepiela2">[44]</a>, <a href="#pone.0001424-Leslie1">[46]</a>, while individuals with low viral loads tended to make strong CTL responses against Gag <a href="#pone.0001424-Honeyborne1">[18]</a>, <a href="#pone.0001424-Frahm3">[27]</a>, <a href="#pone.0001424-Kiepiela2">[44]</a>, <a href="#pone.0001424-Leslie1">[46]</a>. Additionally, by comparison with subtype B infected individuals Frahm and colleagues showed the importance of subdominant CTL responses for the control of replication in subtype C infected individuals <a href="#pone.0001424-Honeyborne1">[18]</a>, <a href="#pone.0001424-Frahm3">[27]</a>, <a href="#pone.0001424-Kiepiela2">[44]</a>, <a href="#pone.0001424-Leslie1">[46]</a>. Collectively, those studies lend support to our <em>in silico</em> approach, especially in the context of a relatively limited knowledge of CTL responses in HIV-1 subtype C infection.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We also explored whether specificities of the epitope repertoires could affect clinical markers of disease progression. By integrating HIV-1 proteome-wide epitope mining to clinical and laboratory data in a South African cohort, our data showed a trend indicating that the number of HLA/epitope pairs was correlated both negatively with viral loads and positively with CD4 counts. Hence, HLA alleles associated with lower viral load in this cohort, referred to as “good” alleles, tended to present larger predicted epitope repertoires, than HLA alleles associated with high viremias, the “bad” alleles. This suggests that the inherent ability to present more epitopes could be a contributing factor to better clinical disease status. Alternatively, certain sets of epitopes may be needed to control the infection and thus, the more epitope motifs presented, the more likely individuals are to cover those epitopes. Our data alludes to a mechanistic paradigm in the cell-mediated immune response, supporting the intuitive assertion that control of HIV infection would capitalize on a broad repertoire while control would be stymied by a narrower epitopic pool. However, a nettlesome HIV characteristic is that despite eliciting relatively broad CTL responses, this generally does not result in the containment of the virus. Although attempts to correlate the breadth of the CTL antiviral response and control of HIV-1 infection <em>in vivo</em> have been equivocal <a href="#pone.0001424-Frahm1">[1]</a>, <a href="#pone.0001424-Zuniga1">[2]</a>, <a href="#pone.0001424-Masemola1">[3]</a>, <a href="#pone.0001424-GeaBanacloche1">[4]</a>, <a href="#pone.0001424-Edwards1">[5]</a>, <a href="#pone.0001424-Betts1">[6]</a>, <a href="#pone.0001424-Addo1">[7]</a>, <a href="#pone.0001424-Musey1">[8]</a>, <a href="#pone.0001424-Liu1">[9]</a>, <a href="#pone.0001424-Honeyborne1">[18]</a>, it could nonetheless be beneficial for the host to have a larger epitopic pool to choose from – maybe not as a means to broaden the CTL response but rather to increase the probability of producing the more limited, effective set of CTL responses, since a diverse panoply of epitopes can be available for CTL recognition simultaneously and/or successively.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>While the efficacy of the CTL response does not appear to rely solely on its breadth, it is widely believed that CTL escape has a major impact on disease outcome. As such, the limited epitope repertoire we identified for individuals/alleles associated with high viremia could reflect escape mutations that eliminated binding motifs from the autologous viral sequences.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In addition to quantitative distinctions, there were also qualitative differences between epitope repertoires restricted by specific HLA alleles: Those associated with better control of HIV replication were likely to present more Gag epitopes in their repertoire than “bad” alleles did; “bad” alleles were instead associated with a higher proportion of Nef epitopes. Interestingly, a recent study by Kiepiela and colleagues showed that Nef-specific CTL responses were associated with higher viral loads, unlike Gag-specific CTL responses, which were associated with lower viral burdens <a href="#pone.0001424-Zuniga1">[2]</a>, <a href="#pone.0001424-Kiepiela2">[44]</a>. While numerous reports have emphasized that CTL responses targeting Gag are the most tightly associated with the control of HIV replication <a href="#pone.0001424-Zuniga1">[2]</a>, <a href="#pone.0001424-Novitsky1">[10]</a>, <a href="#pone.0001424-Ogg1">[11]</a>, <a href="#pone.0001424-Riviere1">[12]</a>, <a href="#pone.0001424-Geldmacher1">[13]</a>, <a href="#pone.0001424-Sacha1">[14]</a>, <a href="#pone.0001424-Ramduth1">[15]</a>, <a href="#pone.0001424-Crawford1">[16]</a>, <a href="#pone.0001424-Streeck1">[17]</a>, little is known about the underlying mechanism. Our study indicates that “good” alleles preferentially target Gag, and that within Gag there is an over-representation of epitopes restricted by “good” alleles instead of “bad” ones, as seen for all other HIV-1 proteins (except Rev). Interestingly, our results using clinical and laboratory data from infected individuals agrees with a very recent <em>in silico</em> study showing that HLA alleles with a low Relative Hazard (RH) of disease progression preferentially presented p24 epitopes <a href="#pone.0001424-Borghans1">[47]</a>. Thus, discordant viral loads depending on specific protein targeting are apparently associated with particular HLA allele restriction sets for each protein. Nonetheless, this leaves open the question of what accounts for the beneficial effect on viremia: CTL responses focusing specifically on Gag, or CTL responses restricted by certain “good” alleles, or both.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The potential shortcomings of <em>in silico</em> epitope predictions cannot be entirely dismissed. And, notwithstanding the composite aspect of the cell-mediated immune response and the difficulty in ascertaining the relative importance of each attribute, evidence that the CTL response is in part mechanically predetermined could be significant in on-going efforts to define more palatable criteria of the immune response to assist vaccine design. Our findings are therefore relevant for vaccine design as they suggest the need to 1) maximize the number of possible epitopes to include in a vaccine candidate and to 2) direct the immune response toward Gag rather than Nef proteins <a href="#pone.0001424-Kiepiela2">[44]</a>, <a href="#pone.0001424-Rolland1">[48]</a>, <a href="#pone.0001424-Altfeld1">[49]</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Dataset</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We evaluated 302 HIV-1 full-length plasma-derived genome sequences along with the HLA genotypes of the infected individuals. 270 subjects were from Durban (South Africa) infected with HIV-1 subtype C <a href="#pone.0001424-Rousseau1">[38]</a>, <a href="#pone.0001424-Rousseau2">[39]</a> and 32 subjects from the Seattle PIC cohort (USA)<a href="#pone.0001424-Schacker1">[43]</a> infected with HIV-1 subtype B <a href="#pone.0001424-Liu2">[32]</a>, <a href="#pone.0001424-Liu3">[40]</a>, <a href="#pone.0001424-Learn1">[41]</a>, <a href="#pone.0001424-Truong1">[42]</a>(and unpublished). Immunological and clinical data (viral loads and CD4 counts) were available at the time of virus sampling for a subgroup of the Durban cohort; details were described elsewhere <a href="#pone.0001424-Kiepiela2">[44]</a>. HIV-1 amino acid sequences were derived for all recognized protein coding sequences of the 302 HIV-1 genomes. HLA allele frequencies in different ethnicities were obtained at <a href="http://www.ncbi.nlm.nih.gov/projects/mhc/ihwg.cgicmdPRJOVID9">http://www.ncbi.nlm.nih.gov/projects/mhc​/ihwg.cgicmdPRJOVID9</a>.</p>


<h4>Epitope Prediction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We employed an implementation of our previously described model <a href="#pone.0001424-Heckerman1">[37]</a> that uses logistic regression and leverages data across HLA alleles to predict CTL epitopes (<a href="http://atom.research.microsoft.com/hlabinding/hlabinding.aspx">http://atom.research.microsoft.com/hlabi​nding/hlabinding.aspx</a>). The predictor was trained on all T-cell epitope data from the LANL <a href="#pone.0001424-Korber1">[50]</a> and IEDB (<a href="http://www.immuneepitope.org/home.do">http://www.immuneepitope.org/home.do</a>) databases in July 2006. Examples of non-epitopes (nine for each positive example) were obtained by randomly sampling proteins from UniProt <a href="#pone.0001424-Wu1">[51]</a>. Eight-, nine-, ten-, and eleven-mer predictors were trained separately. The prior probability of an epitope for each allele was set to 0.1. The prior probability of an epitope for a given allele of length <em>k</em> was proportional to the number of positive examples found for that length-allele combination in the datasets. A peptide-HLA pair was deemed a potential epitope if its posterior probability according to the predictor was greater than 0.5.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Statistical analyses were done using JMP® version 5.1.2. Relationships between 2 variables were analyzed using Spearman's correlation factor Rho. Parametric Student's t tests were used to compare each pair of means.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Brian Birditt and Brandon S. Maust for help with data gathering, and Dr. Joshua T. Herbeck for valuable discussions and critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MR. Performed the experiments: HC DH WD KB. Analyzed the data: CB JM DH MR CR. Contributed reagents/materials/analysis tools: PG BW CB HC CR KB. Wrote the paper: PG BW JM DH MR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001424-Frahm1" id="pone.0001424-Frahm1"></a>Frahm N, Korber BT, Adams CM, Szinger JJ, Draenert R, et al.  (2004) Consistent cytotoxic-T-lymphocyte targeting of immunodominant regions in human immunodeficiency virus across multiple ethnicities. J Virol  78: 2187–2200.  <ul class="find" data-citedArticleID="1040480" data-doi="10.1128/jvi.78.5.2187-2200.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.5.2187-2200.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consistent+cytotoxic-T-lymphocyte+targeting+of+immunodominant+regions+in+human+immunodeficiency+virus+across+multiple+ethnicities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consistent+cytotoxic-T-lymphocyte+targeting+of+immunodominant+regions+in+human+immunodeficiency+virus+across+multiple+ethnicities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001424-Zuniga1" id="pone.0001424-Zuniga1"></a>Zuniga R, Lucchetti A, Galvan P, Sanchez S, Sanchez C, et al.  (2006) Relative dominance of Gag p24-specific cytotoxic T lymphocytes is associated with human immunodeficiency virus control. J Virol  80: 3122–3125.  <ul class="find" data-citedArticleID="1040562" data-doi="10.1128/jvi.80.6.3122-3125.2006"><li><a href="http://dx.doi.org/10.1128/jvi.80.6.3122-3125.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+dominance+of+Gag+p24-specific+cytotoxic+T+lymphocytes+is+associated+with+human+immunodeficiency+virus+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+dominance+of+Gag+p24-specific+cytotoxic+T+lymphocytes+is+associated+with+human+immunodeficiency+virus+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001424-Masemola1" id="pone.0001424-Masemola1"></a>Masemola A, Mashishi T, Khoury G, Mohube P, Mokgotho P, et al.  (2004) Hierarchical targeting of subtype C human immunodeficiency virus type 1 proteins by CD8+ T cells: correlation with viral load. J Virol  78: 3233–3243.  <ul class="find" data-citedArticleID="1040526" data-doi="10.1128/jvi.78.7.3233-3243.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.7.3233-3243.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+targeting+of+subtype+C+human+immunodeficiency+virus+type+1+proteins+by+CD8%2B+T+cells%3A+correlation+with+viral+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+targeting+of+subtype+C+human+immunodeficiency+virus+type+1+proteins+by+CD8%2B+T+cells%3A+correlation+with+viral+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001424-GeaBanacloche1" id="pone.0001424-GeaBanacloche1"></a>Gea-Banacloche JC, Migueles SA, Martino L, Shupert WL, McNeil AC, et al.  (2000) Maintenance of large numbers of virus-specific CD8+ T cells in HIV-infected progressors and long-term nonprogressors. J Immunol  165: 1082–1092.  <ul class="find" data-citedArticleID="1040490"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maintenance+of+large+numbers+of+virus-specific+CD8%2B+T+cells+in+HIV-infected+progressors+and+long-term+nonprogressors.&amp;auth=&amp;atitle=Maintenance+of+large+numbers+of+virus-specific+CD8%2B+T+cells+in+HIV-infected+progressors+and+long-term+nonprogressors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+large+numbers+of+virus-specific+CD8%2B+T+cells+in+HIV-infected+progressors+and+long-term+nonprogressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+large+numbers+of+virus-specific+CD8%2B+T+cells+in+HIV-infected+progressors+and+long-term+nonprogressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001424-Edwards1" id="pone.0001424-Edwards1"></a>Edwards BH, Bansal A, Sabbaj S, Bakari J, Mulligan MJ, et al.  (2002) Magnitude of functional CD8+ T-cell responses to the gag protein of human immunodeficiency virus type 1 correlates inversely with viral load in plasma. J Virol  76: 2298–2305.  <ul class="find" data-citedArticleID="1040478" data-doi="10.1128/jvi.76.5.2298-2305.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.5.2298-2305.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Magnitude+of+functional+CD8%2B+T-cell+responses+to+the+gag+protein+of+human+immunodeficiency+virus+type+1+correlates+inversely+with+viral+load+in+plasma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Magnitude+of+functional+CD8%2B+T-cell+responses+to+the+gag+protein+of+human+immunodeficiency+virus+type+1+correlates+inversely+with+viral+load+in+plasma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001424-Betts1" id="pone.0001424-Betts1"></a>Betts MR, Ambrozak DR, Douek DC, Bonhoeffer S, Brenchley JM, et al.  (2001) Analysis of total human immunodeficiency virus (HIV)-specific CD4(+) and CD8(+) T-cell responses: relationship to viral load in untreated HIV infection. J Virol  75: 11983–11991.  <ul class="find" data-citedArticleID="1040468" data-doi="10.1128/jvi.75.24.11983-11991.2001"><li><a href="http://dx.doi.org/10.1128/jvi.75.24.11983-11991.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+total+human+immunodeficiency+virus+%28HIV%29-specific+CD4%28%2B%29+and+CD8%28%2B%29+T-cell+responses%3A+relationship+to+viral+load+in+untreated+HIV+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+total+human+immunodeficiency+virus+%28HIV%29-specific+CD4%28%2B%29+and+CD8%28%2B%29+T-cell+responses%3A+relationship+to+viral+load+in+untreated+HIV+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001424-Addo1" id="pone.0001424-Addo1"></a>Addo MM, Yu XG, Rathod A, Cohen D, Eldridge RL, et al.  (2003) Comprehensive epitope analysis of human immunodeficiency virus type 1 (HIV-1)-specific T-cell responses directed against the entire expressed HIV-1 genome demonstrate broadly directed responses, but no correlation to viral load. J Virol  77: 2081–2092.  <ul class="find" data-citedArticleID="1040462" data-doi="10.1128/jvi.77.3.2081-2092.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.3.2081-2092.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+epitope+analysis+of+human+immunodeficiency+virus+type+1+%28HIV-1%29-specific+T-cell+responses+directed+against+the+entire+expressed+HIV-1+genome+demonstrate+broadly+directed+responses%2C+but+no+correlation+to+viral+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+epitope+analysis+of+human+immunodeficiency+virus+type+1+%28HIV-1%29-specific+T-cell+responses+directed+against+the+entire+expressed+HIV-1+genome+demonstrate+broadly+directed+responses%2C+but+no+correlation+to+viral+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001424-Musey1" id="pone.0001424-Musey1"></a>Musey L, Hughes J, Schacker T, Shea T, Corey L, et al.  (1997) Cytotoxic-T-cell responses, viral load, and disease progression in early human immunodeficiency virus type 1 infection. N Engl J Med  337: 1267–1274.  <ul class="find" data-citedArticleID="1040530" data-doi="10.1056/nejm199710303371803"><li><a href="http://dx.doi.org/10.1056/nejm199710303371803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytotoxic-T-cell+responses%2C+viral+load%2C+and+disease+progression+in+early+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytotoxic-T-cell+responses%2C+viral+load%2C+and+disease+progression+in+early+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001424-Liu1" id="pone.0001424-Liu1"></a>Liu Y, McNevin J, Zhao H, Tebit DM, McSweyn M, et al.  (2007) Evolution of HIV-1 CTL epitopes: Fitness-Balanced Escape. ePub.   <ul class="find" data-citedArticleID="1040512" data-doi="10.1128/jvi.01277-07"><li><a href="http://dx.doi.org/10.1128/jvi.01277-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+HIV-1+CTL+epitopes%3A+Fitness-Balanced+Escape." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+HIV-1+CTL+epitopes%3A+Fitness-Balanced+Escape.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001424-Novitsky1" id="pone.0001424-Novitsky1"></a>Novitsky V, Gilbert P, Peter T, McLane MF, Gaolekwe S, et al.  (2003) Association between virus-specific T-cell responses and plasma viral load in human immunodeficiency virus type 1 subtype C infection. J Virol  77: 882–890.  <ul class="find" data-citedArticleID="1040532" data-doi="10.1128/jvi.77.2.882-890.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.2.882-890.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Association+between+virus-specific+T-cell+responses+and+plasma+viral+load+in+human+immunodeficiency+virus+type+1+subtype+C+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Association+between+virus-specific+T-cell+responses+and+plasma+viral+load+in+human+immunodeficiency+virus+type+1+subtype+C+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001424-Ogg1" id="pone.0001424-Ogg1"></a>Ogg GS, Jin X, Bonhoeffer S, Dunbar PR, Nowak MA, et al.  (1998) Quantitation of HIV-1-specific T lymphocytes and plasma viral load of viral RNA. Science  279: 2103–2106.  <ul class="find" data-citedArticleID="1040536" data-doi="10.1126/science.279.5359.2103"><li><a href="http://dx.doi.org/10.1126/science.279.5359.2103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitation+of+HIV-1-specific+T+lymphocytes+and+plasma+viral+load+of+viral+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitation+of+HIV-1-specific+T+lymphocytes+and+plasma+viral+load+of+viral+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001424-Riviere1" id="pone.0001424-Riviere1"></a>Riviere Y, McChesney MB, Porrot F, Tanneau-Salvadori F, Sansonetti P, et al.  (1995) Gag-specific cytotoxic responses to HIV type 1 are associated with a decreased risk of progression to AIDS-related complex or AIDS. AIDS Res Hum Retroviruses  11: 903–907.  <ul class="find" data-citedArticleID="1040540" data-doi="10.1089/aid.1995.11.903"><li><a href="http://dx.doi.org/10.1089/aid.1995.11.903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gag-specific+cytotoxic+responses+to+HIV+type+1+are+associated+with+a+decreased+risk+of+progression+to+AIDS-related+complex+or+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gag-specific+cytotoxic+responses+to+HIV+type+1+are+associated+with+a+decreased+risk+of+progression+to+AIDS-related+complex+or+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001424-Geldmacher1" id="pone.0001424-Geldmacher1"></a>Geldmacher C, Currier JR, Herrmann E, Haule A, Kuta E, et al.  (2007) CD8 T-cell recognition of multiple epitopes within specific Gag regions is associated with maintenance of a low steady-state viremia in human immunodeficiency virus type 1-seropositive patients. J Virol  81: 2440–2448.  <ul class="find" data-citedArticleID="1040492" data-doi="10.1128/jvi.01847-06"><li><a href="http://dx.doi.org/10.1128/jvi.01847-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD8+T-cell+recognition+of+multiple+epitopes+within+specific+Gag+regions+is+associated+with+maintenance+of+a+low+steady-state+viremia+in+human+immunodeficiency+virus+type+1-seropositive+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD8+T-cell+recognition+of+multiple+epitopes+within+specific+Gag+regions+is+associated+with+maintenance+of+a+low+steady-state+viremia+in+human+immunodeficiency+virus+type+1-seropositive+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001424-Sacha1" id="pone.0001424-Sacha1"></a>Sacha JB, Chung C, Rakasz EG, Spencer SP, Jonas AK, et al.  (2007) Gag-specific CD8+ T lymphocytes recognize infected cells before AIDS-virus integration and viral protein expression. J Immunol  178: 2746–2754.  <ul class="find" data-citedArticleID="1040548"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression.&amp;auth=&amp;atitle=Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gag-specific+CD8%2B+T+lymphocytes+recognize+infected+cells+before+AIDS-virus+integration+and+viral+protein+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001424-Ramduth1" id="pone.0001424-Ramduth1"></a>Ramduth D, Chetty P, Mngquandaniso NC, Nene N, Harlow JD, et al.  (2005) Differential immunogenicity of HIV-1 clade C proteins in eliciting CD8+ and CD4+ cell responses. J Infect Dis  192: 1588–1596.  <ul class="find" data-citedArticleID="1040538" data-doi="10.1086/496894"><li><a href="http://dx.doi.org/10.1086/496894" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+immunogenicity+of+HIV-1+clade+C+proteins+in+eliciting+CD8%2B+and+CD4%2B+cell+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+immunogenicity+of+HIV-1+clade+C+proteins+in+eliciting+CD8%2B+and+CD4%2B+cell+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001424-Crawford1" id="pone.0001424-Crawford1"></a>Crawford H, Prado JG, Leslie A, Hue S, Honeyborne I, et al.  (2007) Compensatory Mutation Partially Restores Fitness and Delays Reversion of Escape Mutation within the Immunodominant Hla-B*5703-Restricted Gag Epitope in Chronic Hiv-1 Infection. J Virol.   <ul class="find" data-citedArticleID="1040476" data-doi="10.1128/jvi.00465-07"><li><a href="http://dx.doi.org/10.1128/jvi.00465-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensatory+Mutation+Partially+Restores+Fitness+and+Delays+Reversion+of+Escape+Mutation+within+the+Immunodominant+Hla-B*5703-Restricted+Gag+Epitope+in+Chronic+Hiv-1+Infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensatory+Mutation+Partially+Restores+Fitness+and+Delays+Reversion+of+Escape+Mutation+within+the+Immunodominant+Hla-B*5703-Restricted+Gag+Epitope+in+Chronic+Hiv-1+Infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001424-Streeck1" id="pone.0001424-Streeck1"></a>Streeck H, Schweighardt B, Jessen H, Allgaier RL, Wrin T, et al.  (2007) Loss of HIV-1-specific T-cell responses associated with very rapid HIV-1 disease progression. Aids  21: 889–891.  <ul class="find" data-citedArticleID="1040552" data-doi="10.1097/qad.0b013e3280f77439"><li><a href="http://dx.doi.org/10.1097/qad.0b013e3280f77439" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+HIV-1-specific+T-cell+responses+associated+with+very+rapid+HIV-1+disease+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+HIV-1-specific+T-cell+responses+associated+with+very+rapid+HIV-1+disease+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001424-Honeyborne1" id="pone.0001424-Honeyborne1"></a>Honeyborne I, Prendergast A, Pereyra F, Leslie A, Crawford H, et al.  (2007) Control of human immunodeficiency virus type 1 is associated with HLA-B*13 and targeting of multiple gag-specific CD8+ T-cell epitopes. J Virol  81: 3667–3672.  <ul class="find" data-citedArticleID="1040496" data-doi="10.1128/jvi.02689-06"><li><a href="http://dx.doi.org/10.1128/jvi.02689-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+human+immunodeficiency+virus+type+1+is+associated+with+HLA-B*13+and+targeting+of+multiple+gag-specific+CD8%2B+T-cell+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+human+immunodeficiency+virus+type+1+is+associated+with+HLA-B*13+and+targeting+of+multiple+gag-specific+CD8%2B+T-cell+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001424-Carrington1" id="pone.0001424-Carrington1"></a>Carrington M, O'Brien SJ (2003) The influence of HLA genotype on AIDS. Annu Rev Med  54: 535–551.  <ul class="find" data-citedArticleID="1040472" data-doi="10.1146/annurev.med.54.101601.152346"><li><a href="http://dx.doi.org/10.1146/annurev.med.54.101601.152346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+influence+of+HLA+genotype+on+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+influence+of+HLA+genotype+on+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001424-Migueles1" id="pone.0001424-Migueles1"></a>Migueles SA, Sabbaghian MS, Shupert WL, Bettinotti MP, Marincola FM, et al.  (2000) HLA B*5701 is highly associated with restriction of virus replication in a subgroup of HIV-infected long term nonprogressors. Proc Natl Acad Sci U S A  97: 2709–2714.  <ul class="find" data-citedArticleID="1040528" data-doi="10.1073/pnas.050567397"><li><a href="http://dx.doi.org/10.1073/pnas.050567397" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+B*5701+is+highly+associated+with+restriction+of+virus+replication+in+a+subgroup+of+HIV-infected+long+term+nonprogressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+B*5701+is+highly+associated+with+restriction+of+virus+replication+in+a+subgroup+of+HIV-infected+long+term+nonprogressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001424-Trachtenberg1" id="pone.0001424-Trachtenberg1"></a>Trachtenberg E, Korber B, Sollars C, Kepler TB, Hraber PT, et al.  (2003) Advantage of rare HLA supertype in HIV disease progression. Nat Med  9: 928–935.  <ul class="find" data-citedArticleID="1040556" data-doi="10.1038/nm893"><li><a href="http://dx.doi.org/10.1038/nm893" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advantage+of+rare+HLA+supertype+in+HIV+disease+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advantage+of+rare+HLA+supertype+in+HIV+disease+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001424-OBrien1" id="pone.0001424-OBrien1"></a>O'Brien SJ, Gao X, Carrington M (2001) HLA and AIDS: a cautionary tale. Trends Mol Med  7: 379–381.  <ul class="find" data-citedArticleID="1040534" data-doi="10.1016/s1471-4914(01)02131-1"><li><a href="http://dx.doi.org/10.1016/s1471-4914(01)02131-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+and+AIDS%3A+a+cautionary+tale." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+and+AIDS%3A+a+cautionary+tale.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001424-Gao1" id="pone.0001424-Gao1"></a>Gao X, Nelson GW, Karacki P, Martin MP, Phair J, et al.  (2001) Effect of a single amino acid change in MHC class I molecules on the rate of progression to AIDS. N Engl J Med  344: 1668–1675.  <ul class="find" data-citedArticleID="1040488" data-doi="10.1056/nejm200105313442203"><li><a href="http://dx.doi.org/10.1056/nejm200105313442203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+a+single+amino+acid+change+in+MHC+class+I+molecules+on+the+rate+of+progression+to+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+a+single+amino+acid+change+in+MHC+class+I+molecules+on+the+rate+of+progression+to+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001424-Carrington2" id="pone.0001424-Carrington2"></a>Carrington M, Nelson GW, Martin MP, Kissner T, Vlahov D, et al.  (1999) HLA and HIV-1: heterozygote advantage and B*35-Cw*04 disadvantage [see comments]. Science  283: 1748–1752.  <ul class="find" data-citedArticleID="1040474" data-doi="10.1126/science.283.5408.1748"><li><a href="http://dx.doi.org/10.1126/science.283.5408.1748" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+and+HIV-1%3A+heterozygote+advantage+and+B*35-Cw*04+disadvantage+%5Bsee+comments%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+and+HIV-1%3A+heterozygote+advantage+and+B*35-Cw*04+disadvantage+%5Bsee+comments%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001424-Kiepiela1" id="pone.0001424-Kiepiela1"></a>Kiepiela P, Leslie AJ, Honeyborne I, Ramduth D, Thobakgale C, et al.  (2004) Dominant influence of HLA-B in mediating the potential co-evolution of HIV and HLA. Nature  432: 769–775.  <ul class="find" data-citedArticleID="1040500" data-doi="10.1038/nature03113"><li><a href="http://dx.doi.org/10.1038/nature03113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominant+influence+of+HLA-B+in+mediating+the+potential+co-evolution+of+HIV+and+HLA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominant+influence+of+HLA-B+in+mediating+the+potential+co-evolution+of+HIV+and+HLA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001424-Frahm2" id="pone.0001424-Frahm2"></a>Frahm N, Adams S, Kiepiela P, Linde CH, Hewitt HS, et al.  (2005) HLA-B63 presents HLA-B57/B58-restricted cytotoxic T-lymphocyte epitopes and is associated with low human immunodeficiency virus load. J Virol  79: 10218–10225.  <ul class="find" data-citedArticleID="1040482" data-doi="10.1128/jvi.79.16.10218-10225.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.16.10218-10225.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA-B63+presents+HLA-B57%2FB58-restricted+cytotoxic+T-lymphocyte+epitopes+and+is+associated+with+low+human+immunodeficiency+virus+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA-B63+presents+HLA-B57%2FB58-restricted+cytotoxic+T-lymphocyte+epitopes+and+is+associated+with+low+human+immunodeficiency+virus+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001424-Frahm3" id="pone.0001424-Frahm3"></a>Frahm N, Kiepiela P, Adams S, Linde CH, Hewitt HS, et al.  (2006) Control of human immunodeficiency virus replication by cytotoxic T lymphocytes targeting subdominant epitopes. Nat Immunol  7: 173–178.  <ul class="find" data-citedArticleID="1040484" data-doi="10.1038/ni1281"><li><a href="http://dx.doi.org/10.1038/ni1281" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+human+immunodeficiency+virus+replication+by+cytotoxic+T+lymphocytes+targeting+subdominant+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+human+immunodeficiency+virus+replication+by+cytotoxic+T+lymphocytes+targeting+subdominant+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001424-MacDonald1" id="pone.0001424-MacDonald1"></a>MacDonald KS, Embree JE, Nagelkerke NJ, Castillo J, Ramhadin S, et al.  (2001) The HLA A2/6802 supertype is associated with reduced risk of perinatal human immunodeficiency virus type 1 transmission. J Infect Dis  183: 503–506.  <ul class="find" data-citedArticleID="1040522" data-doi="10.1086/318092"><li><a href="http://dx.doi.org/10.1086/318092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HLA+A2%2F6802+supertype+is+associated+with+reduced+risk+of+perinatal+human+immunodeficiency+virus+type+1+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HLA+A2%2F6802+supertype+is+associated+with+reduced+risk+of+perinatal+human+immunodeficiency+virus+type+1+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001424-MacDonald2" id="pone.0001424-MacDonald2"></a>MacDonald KS, Fowke KR, Kimani J, Dunand VA, Nagelkerke NJ, et al.  (2000) Influence of HLA supertypes on susceptibility and resistance to human immunodeficiency virus type 1 infection. J Infect Dis  181: 1581–1589.  <ul class="find" data-citedArticleID="1040524" data-doi="10.1086/315472"><li><a href="http://dx.doi.org/10.1086/315472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+HLA+supertypes+on+susceptibility+and+resistance+to+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+HLA+supertypes+on+susceptibility+and+resistance+to+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001424-Jones1" id="pone.0001424-Jones1"></a>Jones NA, Wei X, Flower DR, Wong M, Michor F, et al.  (2004) Determinants of human immunodeficiency virus type 1 escape from the primary CD8+ cytotoxic T lymphocyte response. J Exp Med  200: 1243–1256.  <ul class="find" data-citedArticleID="1040498" data-doi="10.1084/jem.20040511"><li><a href="http://dx.doi.org/10.1084/jem.20040511" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determinants+of+human+immunodeficiency+virus+type+1+escape+from+the+primary+CD8%2B+cytotoxic+T+lymphocyte+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determinants+of+human+immunodeficiency+virus+type+1+escape+from+the+primary+CD8%2B+cytotoxic+T+lymphocyte+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001424-Allen1" id="pone.0001424-Allen1"></a>Allen TM, Altfeld M, Geer SC, Kalife ET, Moore C, et al.  (2005) Selective escape from CD8+ T-cell responses represents a major driving force of human immunodeficiency virus type 1 (HIV-1) sequence diversity and reveals constraints on HIV-1 evolution. J Virol  79: 13239–13249.  <ul class="find" data-citedArticleID="1040464" data-doi="10.1128/jvi.79.21.13239-13249.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.21.13239-13249.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+escape+from+CD8%2B+T-cell+responses+represents+a+major+driving+force+of+human+immunodeficiency+virus+type+1+%28HIV-1%29+sequence+diversity+and+reveals+constraints+on+HIV-1+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+escape+from+CD8%2B+T-cell+responses+represents+a+major+driving+force+of+human+immunodeficiency+virus+type+1+%28HIV-1%29+sequence+diversity+and+reveals+constraints+on+HIV-1+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001424-Liu2" id="pone.0001424-Liu2"></a>Liu Y, McNevin J, Cao J, Zhao H, Genowati I, et al.  (2006) Selection on the human immunodeficiency virus type 1 proteome following primary infection. J Virol  80: 9519–9529.  <ul class="find" data-citedArticleID="1040514" data-doi="10.1128/jvi.00575-06"><li><a href="http://dx.doi.org/10.1128/jvi.00575-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+on+the+human+immunodeficiency+virus+type+1+proteome+following+primary+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+on+the+human+immunodeficiency+virus+type+1+proteome+following+primary+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001424-Suhrbier1" id="pone.0001424-Suhrbier1"></a>Suhrbier A, Schmidt C, Fernan A (1993) Prediction of an HLA B8-restricted influenza epitope by motif. Immunology  79: 171–173.  <ul class="find" data-citedArticleID="1040554"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prediction+of+an+HLA+B8-restricted+influenza+epitope+by+motif.&amp;auth=&amp;atitle=Prediction+of+an+HLA+B8-restricted+influenza+epitope+by+motif." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+an+HLA+B8-restricted+influenza+epitope+by+motif." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+an+HLA+B8-restricted+influenza+epitope+by+motif.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001424-Larsen1" id="pone.0001424-Larsen1"></a>Larsen MV, Lundegaard C, Lamberth K, Buus S, Brunak S, et al.  (2005) An integrative approach to CTL epitope prediction: a combined algorithm integrating MHC class I binding, TAP transport efficiency, and proteasomal cleavage predictions. Eur J Immunol  35: 2295–2303.  <ul class="find" data-citedArticleID="1040506" data-doi="10.1002/eji.200425811"><li><a href="http://dx.doi.org/10.1002/eji.200425811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrative+approach+to+CTL+epitope+prediction%3A+a+combined+algorithm+integrating+MHC+class+I+binding%2C+TAP+transport+efficiency%2C+and+proteasomal+cleavage+predictions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrative+approach+to+CTL+epitope+prediction%3A+a+combined+algorithm+integrating+MHC+class+I+binding%2C+TAP+transport+efficiency%2C+and+proteasomal+cleavage+predictions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001424-Lundegaard1" id="pone.0001424-Lundegaard1"></a>Lundegaard C, Nielsen M, Lund O (2006) The validity of predicted T-cell epitopes. Trends Biotechnol  24: 537–538.  <ul class="find" data-citedArticleID="1040520" data-doi="10.1016/j.tibtech.2006.10.001"><li><a href="http://dx.doi.org/10.1016/j.tibtech.2006.10.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+validity+of+predicted+T-cell+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+validity+of+predicted+T-cell+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001424-Louzoun1" id="pone.0001424-Louzoun1"></a>Louzoun Y, Vider T, Weigert M (2006) T-cell epitope repertoire as predicted from human and viral genomes. Mol Immunol  43: 559–569.  <ul class="find" data-citedArticleID="1040518" data-doi="10.1016/j.molimm.2005.04.017"><li><a href="http://dx.doi.org/10.1016/j.molimm.2005.04.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-cell+epitope+repertoire+as+predicted+from+human+and+viral+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-cell+epitope+repertoire+as+predicted+from+human+and+viral+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001424-Heckerman1" id="pone.0001424-Heckerman1"></a>Heckerman D, Kadie C, Listgarten J (2006) Leveraging information across HLA alleles/supertypes improves epitope prediction.   Tenth Annual International Conference on Research in Computational Molecular Biology (RECOMB). Venice, Italy.  <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0001424-Rousseau1" id="pone.0001424-Rousseau1"></a>Rousseau C, Birditt BA, McKay AR, Stoddard JN, Lee TC, et al.  (2006) Large-scale amplification, cloning and sequencing of near full-length HIV-1 subtype C genomes. J Virol Methods  136: 118–125.  <ul class="find" data-citedArticleID="1040544" data-doi="10.1016/j.jviromet.2006.04.009"><li><a href="http://dx.doi.org/10.1016/j.jviromet.2006.04.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+amplification%2C+cloning+and+sequencing+of+near+full-length+HIV-1+subtype+C+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+amplification%2C+cloning+and+sequencing+of+near+full-length+HIV-1+subtype+C+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001424-Rousseau2" id="pone.0001424-Rousseau2"></a>Rousseau CM, Learn GH, Bhattacharya T, Nickle DC, Heckerman D, et al.  (2007) Extensive Intra-subtype Recombination in South African HIV-1 Subtype C Infections. J Virol  81: 4492–4500.  <ul class="find" data-citedArticleID="1040546" data-doi="10.1128/jvi.02050-06"><li><a href="http://dx.doi.org/10.1128/jvi.02050-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+Intra-subtype+Recombination+in+South+African+HIV-1+Subtype+C+Infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+Intra-subtype+Recombination+in+South+African+HIV-1+Subtype+C+Infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001424-Liu3" id="pone.0001424-Liu3"></a>Liu SL, Schacker T, Musey L, Shriner D, McElrath MJ, et al.  (1997) Divergent patterns of progression to AIDS after infection from the same source: human immunodeficiency virus type 1 evolution and antiviral responses. J Virol  71: 4284–4295.  <ul class="find" data-citedArticleID="1040516"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Divergent+patterns+of+progression+to+AIDS+after+infection+from+the+same+source%3A+human+immunodeficiency+virus+type+1+evolution+and+antiviral+responses.&amp;auth=&amp;atitle=Divergent+patterns+of+progression+to+AIDS+after+infection+from+the+same+source%3A+human+immunodeficiency+virus+type+1+evolution+and+antiviral+responses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergent+patterns+of+progression+to+AIDS+after+infection+from+the+same+source%3A+human+immunodeficiency+virus+type+1+evolution+and+antiviral+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergent+patterns+of+progression+to+AIDS+after+infection+from+the+same+source%3A+human+immunodeficiency+virus+type+1+evolution+and+antiviral+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001424-Learn1" id="pone.0001424-Learn1"></a>Learn GH, Muthui D, Brodie SJ, Zhu T, Diem K, et al.  (2002) Virus population homogenization following acute human immunodeficiency virus type 1 infection. J Virol  76: 11953–11959.  <ul class="find" data-citedArticleID="1040508" data-doi="10.1128/jvi.76.23.11953-11959.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.23.11953-11959.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virus+population+homogenization+following+acute+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virus+population+homogenization+following+acute+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001424-Truong1" id="pone.0001424-Truong1"></a>Truong HM, Berrey MM, Shea T, Diem K, Corey L (2002) Concordance between HIV source partner identification and molecular confirmation in acute retroviral syndrome. J Acquir Immune Defic Syndr  29: 232–243.  <ul class="find" data-citedArticleID="1040558" data-doi="10.1097/00042560-200203010-00003"><li><a href="http://dx.doi.org/10.1097/00042560-200203010-00003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concordance+between+HIV+source+partner+identification+and+molecular+confirmation+in+acute+retroviral+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concordance+between+HIV+source+partner+identification+and+molecular+confirmation+in+acute+retroviral+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001424-Schacker1" id="pone.0001424-Schacker1"></a>Schacker T, Collier AC, Hughes J, Shea T, Corey L (1996) Clinical and epidemiologic features of primary HIV infection. Ann Intern Med  125: 257–264.  <ul class="find" data-citedArticleID="1040550" data-doi="10.7326/0003-4819-125-4-199608150-00001"><li><a href="http://dx.doi.org/10.7326/0003-4819-125-4-199608150-00001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clinical+and+epidemiologic+features+of+primary+HIV+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clinical+and+epidemiologic+features+of+primary+HIV+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001424-Kiepiela2" id="pone.0001424-Kiepiela2"></a>Kiepiela P, Ngumbela K, Thobakgale C, Ramduth D, Honeyborne I, et al.  (2007) CD8+ T-cell responses to different HIV proteins have discordant associations with viral load. Nat Med  13: 46–53.  <ul class="find" data-citedArticleID="1040502" data-doi="10.1038/nm1520"><li><a href="http://dx.doi.org/10.1038/nm1520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CD8%2B+T-cell+responses+to+different+HIV+proteins+have+discordant+associations+with+viral+load." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CD8%2B+T-cell+responses+to+different+HIV+proteins+have+discordant+associations+with+viral+load.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001424-Frahm4" id="pone.0001424-Frahm4"></a>Frahm N, Yusim K, Suscovich TJ, Adams S, Sidney J, et al.  (2007) Extensive HLA class I allele promiscuity among viral CTL epitopes. Eur J Immunol  37: 2419–2433.  <ul class="find" data-citedArticleID="1040486" data-doi="10.1002/eji.200737365"><li><a href="http://dx.doi.org/10.1002/eji.200737365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+HLA+class+I+allele+promiscuity+among+viral+CTL+epitopes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+HLA+class+I+allele+promiscuity+among+viral+CTL+epitopes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001424-Leslie1" id="pone.0001424-Leslie1"></a>Leslie A, Price DA, Mkhize P, Bishop K, Rathod A, et al.  (2006) Differential selection pressure exerted on HIV by CTL targeting identical epitopes but restricted by distinct HLA alleles from the same HLA supertype. J Immunol  177: 4699–4708.  <ul class="find" data-citedArticleID="1040510"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Differential+selection+pressure+exerted+on+HIV+by+CTL+targeting+identical+epitopes+but+restricted+by+distinct+HLA+alleles+from+the+same+HLA+supertype.&amp;auth=&amp;atitle=Differential+selection+pressure+exerted+on+HIV+by+CTL+targeting+identical+epitopes+but+restricted+by+distinct+HLA+alleles+from+the+same+HLA+supertype." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+selection+pressure+exerted+on+HIV+by+CTL+targeting+identical+epitopes+but+restricted+by+distinct+HLA+alleles+from+the+same+HLA+supertype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+selection+pressure+exerted+on+HIV+by+CTL+targeting+identical+epitopes+but+restricted+by+distinct+HLA+alleles+from+the+same+HLA+supertype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001424-Borghans1" id="pone.0001424-Borghans1"></a>Borghans JA, Molgaard A, de Boer RJ, Kesmir C (2007) HLA Alleles Associated with Slow Progression to AIDS Truly Prefer to Present HIV-1 p24. PLoS ONE  2: e920.  <ul class="find" data-citedArticleID="1040470" data-doi="10.1371/journal.pone.0000920"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000920" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+Alleles+Associated+with+Slow+Progression+to+AIDS+Truly+Prefer+to+Present+HIV-1+p24." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+Alleles+Associated+with+Slow+Progression+to+AIDS+Truly+Prefer+to+Present+HIV-1+p24.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001424-Rolland1" id="pone.0001424-Rolland1"></a>Rolland M, Nickle DC, Mullins JI (2007) A new, core elements approach to vaccine immunogen design. PLoS Pathogens: In press.   <ul class="find" data-citedArticleID="1040542" data-doi="10.1371/journal.ppat.0030157"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0030157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new%2C+core+elements+approach+to+vaccine+immunogen+design." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new%2C+core+elements+approach+to+vaccine+immunogen+design.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001424-Altfeld1" id="pone.0001424-Altfeld1"></a>Altfeld M, Allen TM (2006) Hitting HIV where it hurts: an alternative approach to HIV vaccine design. Trends Immunol  27: 504–510.  <ul class="find" data-citedArticleID="1040466" data-doi="10.1016/j.it.2006.09.007"><li><a href="http://dx.doi.org/10.1016/j.it.2006.09.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hitting+HIV+where+it+hurts%3A+an+alternative+approach+to+HIV+vaccine+design." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hitting+HIV+where+it+hurts%3A+an+alternative+approach+to+HIV+vaccine+design.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001424-Korber1" id="pone.0001424-Korber1"></a>Korber BTM, Brander C, Haynes BF, Koup R, Moore JP, et al.  (2005) HIV Molecular Immunology 2005.   Los Alamos National Laboratory, Theoretical Biology and Biophysics, Los Alamos, New Mexico. LA-UR 06-0036.  <ul class="find-nolinks"></ul></li><li><span class="label">51.
              </span><a name="pone.0001424-Wu1" id="pone.0001424-Wu1"></a>Wu CH, Apweiler R, Bairoch A, Natale DA, Barker WC, et al.  (2006) The Universal Protein Resource (UniProt): an expanding universe of protein information. Nucleic Acids Res  34: D187–191.  <ul class="find" data-citedArticleID="1040560" data-doi="10.1093/nar/gkj161"><li><a href="http://dx.doi.org/10.1093/nar/gkj161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Universal+Protein+Resource+%28UniProt%29%3A+an+expanding+universe+of+protein+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Universal+Protein+Resource+%28UniProt%29%3A+an+expanding+universe+of+protein+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.XML" value="84908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.PDF" value="220682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g001.PNG_L" value="289774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g001.PNG_M" value="48706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g001.PNG_S" value="7338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g001.TIF" value="512106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g001.PNG_I" value="17660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g002.PNG_L" value="543879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g002.PNG_M" value="100158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g002.PNG_S" value="11993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g002.TIF" value="1549244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001424.g002.PNG_I" value="37250"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001424&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001424" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001424&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001424&volume=&issue=&title=Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads&author_name=Morgane%20Rolland%2C%20David%20Heckerman%2C%20Wenjie%20Deng%2C%20Christine%20M.%20Rousseau%2C%20Hoosen%20Coovadia%2C%20Karen%20Bishop%2C%20Philip%20J.%20R.%20Goulder%2C%20Bruce%20D.%20Walker%2C%20Christian%20Brander%2C%20James%20I.%20Mullins&start_page=1&end_page=6" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001424" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424&amp;t=Broad%20and%20Gag-Biased%20HIV-1%20Epitope%20Repertoires%20Are%20Associated%20with%20Lower%20Viral%20Loads" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424&title=Broad%20and%20Gag-Biased%20HIV-1%20Epitope%20Repertoires%20Are%20Associated%20with%20Lower%20Viral%20Loads&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424&amp;title=Broad%20and%20Gag-Biased%20HIV-1%20Epitope%20Repertoires%20Are%20Associated%20with%20Lower%20Viral%20Loads" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001424&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Broad and Gag-Biased HIV-1 Epitope Repertoires Are Associated with Lower Viral Loads';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001424';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Broad%20and%20Gag-Biased%20HIV-1%20Epitope%20Repertoires%20Are%20Associated%20with%20Lower%20Viral%20Loads http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001424" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001424" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV%22" title="Search for articles in the subject area:'HIV'"><div class="flagText">HIV</div></a>
              <div data-categoryid="42663" data-articleid="25678"
                   data-categoryname="HIV"
                   class="flagImage" title="Flag 'HIV' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="25678"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immune+response%22" title="Search for articles in the subject area:'Immune response'"><div class="flagText">Immune response</div></a>
              <div data-categoryid="17773" data-articleid="25678"
                   data-categoryname="Immune response"
                   class="flagImage" title="Flag 'Immune response' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomes%22" title="Search for articles in the subject area:'Proteomes'"><div class="flagText">Proteomes</div></a>
              <div data-categoryid="18649" data-articleid="25678"
                   data-categoryname="Proteomes"
                   class="flagImage" title="Flag 'Proteomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="25678"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22T+cells%22" title="Search for articles in the subject area:'T cells'"><div class="flagText">T cells</div></a>
              <div data-categoryid="40147" data-articleid="25678"
                   data-categoryname="T cells"
                   class="flagImage" title="Flag 'T cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+load%22" title="Search for articles in the subject area:'Viral load'"><div class="flagText">Viral load</div></a>
              <div data-categoryid="21755" data-articleid="25678"
                   data-categoryname="Viral load"
                   class="flagImage" title="Flag 'Viral load' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Viral+replication%22" title="Search for articles in the subject area:'Viral replication'"><div class="flagText">Viral replication</div></a>
              <div data-categoryid="20015" data-articleid="25678"
                   data-categoryname="Viral replication"
                   class="flagImage" title="Flag 'Viral replication' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7172'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4495'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2289&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
