

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Redundancy in Genotyping Arrays</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000287"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000287" />

    <meta name="citation_title" content="Redundancy in Genotyping Arrays"/>
    <meta itemprop="name" content="Redundancy in Genotyping Arrays"/>

      <meta name="citation_author" content="Scott Smemo"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, Chicago, Illinois, United States of America"/>
      <meta name="citation_author" content="Justin O. Borevitz"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Chicago, Chicago, Illinois, United States of America"/>

    <meta name="citation_date" content="2007/3/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000287.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e287"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Highly parallel genomic assays.; citation_author=JB Fan; citation_author=MS Chee; citation_author=KL Gunderson; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=1; citation_pages=632-644; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Resequencing and mutational analysis using oligonucleotide microarrays.; citation_author=JG Hacia; citation_journal_title=Nat Genet; citation_volume=21; citation_number=2; citation_pages=42-47; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA microarray technology: devices, systems, and applications.; citation_author=MJ Heller; citation_journal_title=Annu Rev Biomed Eng; citation_volume=4; citation_number=3; citation_pages=129-153; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genotyping pooled DNA using 100K SNP microarrays: a step towards genomewide association scans.; citation_author=E Meaburn; citation_author=LM Butcher; citation_author=LC Schalkwyk; citation_author=R Plomin; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=4; citation_pages=e27; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays.; citation_author=H Matsuzaki; citation_author=S Dong; citation_author=H Loi; citation_author=X Di; citation_author=G Liu; citation_journal_title=Nat Methods; citation_volume=1; citation_number=5; citation_pages=109-111; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The International HapMap Project.; citation_author=TIH Consortium; citation_journal_title=Nature; citation_volume=426; citation_number=6; citation_pages=789-796; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A genotype calling algorithm for affymetrix SNP arrays.; citation_author=N Rabbee; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=7; citation_pages=7-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bioconductor: open software development for computational biology and bioinformatics.; citation_author=RC Gentleman; citation_author=VJ Carey; citation_author=DM Bates; citation_author=B Bolstad; citation_author=M Dettling; citation_journal_title=Genome Biol; citation_volume=5; citation_number=8; citation_pages=R80; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Exploration, Normalization, and Genotype Calls of High Density Oligonucleotide SNP Array Data.; citation_author=B Carvalho; citation_author=H Bengtsson; citation_author=TP Speed; citation_author=RA Irizarry; citation_journal_title=Biostatistics; citation_number=9; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale identification of single-feature polymorphisms in complex genomes.; citation_author=JO Borevitz; citation_author=D Liang; citation_author=D Plouffe; citation_author=HS Chang; citation_author=T Zhu; citation_journal_title=Genome Res; citation_volume=13; citation_number=10; citation_pages=513-523; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Redundancy in Genotyping Arrays"/>
    <meta name="twitter:description" content="Despite their unprecedented density, current SNP genotyping arrays contain large amounts of redundancy, with up to 40 oligonucleotide features used to query each SNP. By using publicly available reference genotype data from the International HapMap, we show that 93.6% sensitivity at &lt;5% false positive rate can be obtained with only four probes per SNP, compared with 98.3% with the full data set. Removal of this redundancy will allow for more comprehensive whole-genome association studies with increased SNP density and larger sample sizes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000287.g003"/>

  <meta property="og:title" content="Redundancy in Genotyping Arrays" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2821'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=613'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7225&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000287">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000287" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000287&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Redundancy in Genotyping Arrays
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Scott Smemo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, University of Chicago, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Justin O. Borevitz
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:borevitz@uchicago.edu">borevitz@uchicago.edu</a></p>

                <p>Affiliation:
                  Department of Ecology and Evolution, University of Chicago, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000287</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000287-g001" data-doi="info:doi/10.1371/journal.pone.0000287" data-uri="info:doi/10.1371/journal.pone.0000287.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000287-g002" data-doi="info:doi/10.1371/journal.pone.0000287" data-uri="info:doi/10.1371/journal.pone.0000287.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000287-g003" data-doi="info:doi/10.1371/journal.pone.0000287" data-uri="info:doi/10.1371/journal.pone.0000287.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000287">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000287" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Despite their unprecedented density, current SNP genotyping arrays contain large amounts of redundancy, with up to 40 oligonucleotide features used to query each SNP. By using publicly available reference genotype data from the International HapMap, we show that 93.6% sensitivity at &lt;5% false positive rate can be obtained with only four probes per SNP, compared with 98.3% with the full data set. Removal of this redundancy will allow for more comprehensive whole-genome association studies with increased SNP density and larger sample sizes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Smemo S, Borevitz JO (2007) Redundancy in Genotyping Arrays. PLoS ONE 2(3):
          e287.
            doi:10.1371/journal.pone.0000287</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> February 6, 2007; <strong>Accepted:</strong> February 13, 2007; <strong>Published:</strong> March 14, 2007</p><p><strong>Copyright:</strong> © 2007 Smemo, Borevitz. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We received no external compensation for this work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>High density genotyping arrays have been heralded as one of the technologies that will enable whole-genome association tests to make good on the promise that sequencing the human genome will reveal the genetic causes of complex diseases <a href="#pone.0000287-Fan1">[1]</a>–<a href="#pone.0000287-Meaburn1">[4]</a>. While currently-available arrays, or chips, have the capacity for assaying hundreds of thousands of Single Nucleotide Polymorphisms (SNPs) in a single experiment, data from millions of SNPs across a large number of samples is needed to fine map complex traits. Fortunately, with current technology, this is already possible because available arrays have a large amount of redundancy—multiple oligonucleotide features (probes) are used to assay the same SNP. If that redundancy were reduced or even eliminated, commensurately more data could be obtained for the same price. We therefore sought to determine the minimum number of features necessary to accurately assign genotypes to Affymetrix's GeneChip.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The physical Affymetrix GeneChip Human Mapping 100K chip contains 116,204 biallelic SNPs interrogated in two subsets that differ in which restriction enzyme was used in their preparation, either XbaI or HindIII <a href="#pone.0000287-Matsuzaki1">[5]</a>. We analyzed only the Xba subset but assume the results will be applicable to the Hind data. The Affymetrix GeneChip Human Mapping 100K data set (<a href="http://www.affymetrix.com/support/technical/sample_data/hapmap_trio_data.affx">http://www.affymetrix.com/support/techni​cal/sample_data/hapmap_trio_data.affx</a>) contains raw, feature-level data for 90 CEPH individuals genotyped across these 116,204 SNPs. Our analyses are based on the Xba subset, which contains 58,960 SNPs. Most of these (58,011, 98%) were also included in the HapMap Project <a href="#pone.0000287-Consortium1">[6]</a>. HapMap SNP genotypes were used as our gold standard because of the project's quality control measures (trios for genotyping, multi-site verification), though we and others <a href="#pone.0000287-Rabbee1">[7]</a> acknowledge errors may still be present in this data. The Affymetrix data set lacked genotype calls for 4,651 SNPs and were omitted in all subsequent analyses, leaving 53,360 SNPs in our working set.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>We used the Bioconductor <a href="#pone.0000287-Gentleman1">[8]</a> package Robust Linear Model with Mahalanobis Distance Classifier (RLMM) <a href="#pone.0000287-Rabbee1">[7]</a> for making genotype classifications or calls. By using a multi-chip, multi-SNP approach, predicated on a large training set, RLMM has been shown to be an accurate and informationally efficient method <a href="#pone.0000287-Rabbee1">[7]</a>. A variation, Bayesian RLMM, is now provided by Affymetrix for analyzing their 500K arrays as a white paper (BRLMM, <a href="http://www.affymetrix.com/support/technical/product_updates/brlmm_algorithm.affx">http://www.affymetrix.com/support/techni​cal/product_updates/brlmm_algorithm.affx</a>), and an improved method, CRLMM, was just released as an R/oligo developmental package <a href="#pone.0000287-Carvalho1">[9]</a>.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>The Affymetrix arrays contain probe sets for each SNP with up to 20 perfect match and 20 mismatch features (<a href="#pone-0000287-g001">Fig. 1</a>). RLMM, BRLMM, CRLMM do not use mismatch data, making ½ of the array immediately redundant. We modified RLMM to allow genotype assignments to be made using specific subsets of features. This includes all 5 feature positions (the default for RLMM with the 100K chip), the 3 central-most (default for the 500K chip), and a single central feature position. By default RLMM uses the information from both the sense and antisense strands. We further modified it to allow for selection of only a single strand using two different methods. One method selects the strand with the strongest average signal across features of both A and B alleles, while the other method uses the strand with the maximal allele intensity difference. These 6 parameter combinations (1, 3, or 5 features per strand; 1 or 2 strands) were used in turn to make genotype classifications, with a 100% calling rate setting in RLMM. It is also possible to use a single probe for genotyping using intensity alone to make the call. This approach called Single Feature Polymorphism <a href="#pone.0000287-Borevitz1">[10]</a> was not investigated here as both alleles are known and included on the array. Additional workflow information, including our full data sets and algorithms, are available online (<a href="http://www.naturalvariation.org/snpredundancy">http://www.naturalvariation.org/snpredun​dancy</a>).</p>
<div class="figure" id="pone-0000287-g001"><div class="img"><a name="pone-0000287-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000287" data-uri="info:doi/10.1371/journal.pone.0000287.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000287.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000287.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Probe set layout for Affymetrix 100K chip. Each SNP is assayed with 20 perfect match 25-mer oligonucleotides. The additional 20 mismatch oligonucleotides are not shown. Both sense and antisense strands are interrogated with 5 features for each allele that differ only at the SNP nucleotide, denoted A and B. The position of the SNP within the 25mer features is shifted from the central nucleotide (boxed, “0”) by 1 or 2 nucleotides in either direction. For this SNP (rs836702), and sample (NA11994) allele B has a stronger normalized hybridization signal, as indicated by the lighter color and higher hybridization values for most probes. Generally the central base and a particular strand have more discrimination specificity. The genotype for this individual was called as BB with full and all reduced probesets. This is clear when many other samples are tested such that the range of intensity categories is known.</p>
<span>doi:10.1371/journal.pone.0000287.g001</span></div></div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We first tested for internal concordance in genotype classification, comparing genotype calls made with reduced probesets to the full probeset of 20 features (5 positions, 2 strands, 2 alleles). For each SNP, the number of discordant genotypes was counted. That is, for each of the 90 individuals in the data set, their reduced probeset-derived genotype was compared to the full probeset-derived genotype. <a href="#pone.0000287.s001">Supplemental Figure 1</a> and Supplementary <a href="#pone.0000287.s002">Table S1</a> show the cumulative distribution of SNPs at different discordance thresholds. Notably, even with only 2 features (1 central position &amp; 1 strand, or 10% of the total data), half of the SNPs are called in perfect concordance and 78% have 2 or fewer discordant calls. When both strands are used, 66% of SNPs have perfect concordance and 85% have 2 or fewer discordances.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We next tested specificity and sensitivity with independent genotype classification by comparing RLMM genotype calls made with various feature subsets to HapMap genotypes. Again, for each SNP, specificity was measured as the number of discrepancies across samples (<a href="#pone-0000287-g002">Figure 2, x axis</a>). Sensitivity is shown as the proportion of SNPs with given specificity. The full model is less than perfect and achieves 98.3% sensitivity at 95% specificity (4/90 or fewer discrepancies). So, how much worse are less redundant designs? The reduced models with 1 or 3 features on 2 strands perform quite well achieving 93.6% and 96.8%, respectively. <a href="#pone-0000287-g002">Figure 2</a> shows the cumulative distribution of discrepancies across all SNPs. The single strand with maximum intensity was also quite predictive and would be important for RNA assays. 3 positions on 1 strand (6 features) was better than 1 position on 2 strands (4 features), and 5 positions on 1 strand (10 features) was worse than 3 positions on 2 strands (12 features), as would be expected given extra redundancy (<a href="#pone.0000287.s003">Supplemental table S2</a>). The point is that the improvement with extra data is minimal.</p>
<div class="figure" id="pone-0000287-g002"><div class="img"><a name="pone-0000287-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000287" data-uri="info:doi/10.1371/journal.pone.0000287.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000287.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000287.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Cumulative distribution of discrepancies between HapMap and RLMM genotypes across probe set combinations Each curve shows the cumulative proportion of discrepancies between the 90 individuals for which genotypes were called using the indicated RLMM reduced probeset compared against the HapMap project. The vertical line at 4 discrepancies corresponds to approximately 95% specificity. 93.6% of all SNPs had 4 or fewer discrepancies when only the central probe on both strands was used (red curve). Note the break at 30 discrepancies, at which point 5 of the 6 curves have all but converged.</p>
<span>doi:10.1371/journal.pone.0000287.g002</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We next considered the manner in which discrepancies occurred by comparing how genotypes should have been called (HapMap AA, AB, BB calls) to how they were actually called (RLMM AA, AB, BB calls) for each of the reduced probesets. As has been previously reported <a href="#pone.0000287-Rabbee1">[7]</a>, heterozygotes were most prone to misclassification. For example SNPs typed at 1 position across two strands, hets are miscalled 2.8% and homozygotes each at 0.4%. This bias however, was not greatly exaggerated for the reduced probesets (<a href="#pone.0000287.s004">Supplementary Table S3</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Finally, we tested if the genotyping errors were non-randomly distributed, perhaps due to bad samples or to bad SNPs. No DNA sample gave an abnormally large number of discrepant genotypes as would be the case with a poor sample or array; however some SNPs performed poorly, independent of the redundancy in the probeset. For example, we compared genotyping discrepancies for genotypes generated using a reduced probeset (1 position, 2 strands, 2 alleles) to the full probeset (5 positions, 2 strands, 2 alleles) in <a href="#pone-0000287-g003">Figure 3</a>. 33,712 SNPs (63%) lie on or above the diagonal and are not improved by the full feature set. The majority of these, 30,585 SNPs, are called perfectly in either case. 2,516 SNPs benefit from larger probesets and do pass (&lt;%5 false positives) with the full probeset. However, more data does not always help: 17 SNPs that fail (&gt;5% false positives) with the full set actually pass with the reduced set.</p>
<div class="figure" id="pone-0000287-g003"><div class="img"><a name="pone-0000287-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000287" data-uri="info:doi/10.1371/journal.pone.0000287.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000287.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000287.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000287.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000287.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000287.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>Correlation among poorly performing SNPs The number of discrepancies between HapMap calls and RLMM calls made with the reduced or full feature genotype data. Most SNPs (30,585) are perfectly called with both the full and reduced sets (peach), while another 3,030 poor SNPs fall along the diagonal and are not improved with data from additional features. Those few 2,516 SNPs (red box) that do improve with additional features do not justify redundant designs for all SNPs on the array.</p>
<span>doi:10.1371/journal.pone.0000287.g003</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In general, poorly performing SNPs reduce the overall performance of the array and should be eliminated in future array designs. For example, by removing SNPs that had more than 1 discrepancy when using the full feature set (6.04% of all SNPs), the overall proportion of SNPs with 4 or fewer discrepancies (5% false positive) when typed with 2 strands 1 and position increased by 3.15% to 96.7%.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We have shown that genotypes classified with as few as 4 features are nearly as sensitive as those classified using sets with 20 features (93.6% vs 98.3% at a 95% specificity threshold). While additional features generally improve sensitivity and specificity, they do so at a diminishing rate. The discrepancy rate for SNP genotypes was correlated between full and reduced probeset calls, indicating poorly performing SNPs are only slightly improved with additional features. By contrast, 58% of SNPs are perfectly genotyped with only 4 features improving to 87% with the full set. This improvement comes at a cost of 5–10× in SNP real estate. By thoughtfully choosing which SNPs to include, and drastically reducing the number of features to use for each SNP, large increases in total SNPs can be realized in new array designs. At the current 5 µm feature density, GeneChip arrays contain 6.55 million features. At 4 features per SNP &gt;1.5 million SNPs can be achieved. In addition, as more robust genotype calling methods are developed (BRLMM, CRLMM) <a href="#pone.0000287-Carvalho1">[9]</a>, accuracy will increase, further reducing any necessity for feature redundancy. Together this will allow a single genotyping array to have SNP densities high enough to enable true genome-wide association studies in the near future.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000287"></div><a name="pone.0000287.s001" id="pone.0000287.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000287.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Cumulative distribution of internal discordances across probeset combinations Each curve shows the cumulative proportion of discrepant genotypes obtained when genotypes are called using the full probe set compared to genotyped called using the indicated subset of probes. While the total number of discrepancies is out of a possible 90, only the first 30 are shown, which is the point at which the 5 curves converge.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000287.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.63 MB TIF)</p>
<a name="pone.0000287.s002" id="pone.0000287.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000287.s002">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Cumulative distribution showing the proportion of SNPs typed with a maximum number of discrepencies when compared to genotypes generated with all 5 probes and both strands.By default RLMM uses the information from both the sense and antisense strands, but we further modified it in two ways. Method 1 selected the strand with the strongest signal averaged across A and B probe sets. Method 2 selected the strand with the larger difference between average A and average B allele intensities. Performance for method 2 was inferior across all parameter combinations, therefore method 1 results are shown in subsequent figures/tables.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000287.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000287.s003" id="pone.0000287.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000287.s003">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Cumulative distribution showing the proportion of SNPs typed with a maximum number of discrepencies when compared to HapMap genotypes.By default RLMM uses the information from both the sense and antisense strands, but we further modified it in two ways. Method 1 selected the strand with the strongest signal averaged across A and B probe sets. Method 2 selected the strand with the larger difference between average A and average B allele intensities. Performance for method 2 was inferior across all parameter combinations, therefore method 1 results are shown in subsequent figures/tables.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000287.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000287.s004" id="pone.0000287.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000287.s004">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Misclassified genotypes accumulate in heterozygotes.The number of probes per strand and strands used is indicated in the upper left-hand corner of each box. The numbers represent counts of genotypes binned by their “true” genotypes (HapMap calls) compared to how RLMM classified them (RLMM Calls). When only one probe and one strand are used, a large excess of misclassified heterozygotes is seen. False positive rate corresponds to the proportion of misclassified genotypes in each column or row. Relative amount indicates the proportion of each type of false positive within the total number of misclassified genotypes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000287.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JB. Performed the experiments: SS. Analyzed the data: SS. Contributed reagents/materials/analysis tools: SS. Wrote the paper: JB SS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000287-Fan1" id="pone.0000287-Fan1"></a>Fan JB, Chee MS, Gunderson KL (2006) Highly parallel genomic assays. Nat Rev Genet  7: 632–644.  <ul class="find" data-citedArticleID="936784" data-doi="10.1038/nrg1901"><li><a href="http://dx.doi.org/10.1038/nrg1901" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+parallel+genomic+assays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+parallel+genomic+assays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000287-Hacia1" id="pone.0000287-Hacia1"></a>Hacia JG (1999) Resequencing and mutational analysis using oligonucleotide microarrays. Nat Genet  21: 42–47.  <ul class="find" data-citedArticleID="936788" data-doi="10.1038/4469"><li><a href="http://dx.doi.org/10.1038/4469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resequencing+and+mutational+analysis+using+oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resequencing+and+mutational+analysis+using+oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000287-Heller1" id="pone.0000287-Heller1"></a>Heller MJ (2002) DNA microarray technology: devices, systems, and applications. Annu Rev Biomed Eng  4: 129–153.  <ul class="find" data-citedArticleID="936790" data-doi="10.1146/annurev.bioeng.4.020702.153438"><li><a href="http://dx.doi.org/10.1146/annurev.bioeng.4.020702.153438" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarray+technology%3A+devices%2C+systems%2C+and+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarray+technology%3A+devices%2C+systems%2C+and+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000287-Meaburn1" id="pone.0000287-Meaburn1"></a>Meaburn E, Butcher LM, Schalkwyk LC, Plomin R (2006) Genotyping pooled DNA using 100K SNP microarrays: a step towards genomewide association scans. Nucleic Acids Res  34: e27.  <ul class="find" data-citedArticleID="936794" data-doi="10.1093/nar/gnj027"><li><a href="http://dx.doi.org/10.1093/nar/gnj027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+pooled+DNA+using+100K+SNP+microarrays%3A+a+step+towards+genomewide+association+scans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+pooled+DNA+using+100K+SNP+microarrays%3A+a+step+towards+genomewide+association+scans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000287-Matsuzaki1" id="pone.0000287-Matsuzaki1"></a>Matsuzaki H, Dong S, Loi H, Di X, Liu G, et al.  (2004) Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays. Nat Methods  1: 109–111.  <ul class="find" data-citedArticleID="936792" data-doi="10.1038/nmeth718"><li><a href="http://dx.doi.org/10.1038/nmeth718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+over+100%2C000+SNPs+on+a+pair+of+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+over+100%2C000+SNPs+on+a+pair+of+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000287-Consortium1" id="pone.0000287-Consortium1"></a>Consortium TIH (2003) The International HapMap Project. Nature  426: 789–796.  <ul class="find" data-citedArticleID="936782"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+International+HapMap+Project.&amp;auth=&amp;atitle=The+International+HapMap+Project." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+International+HapMap+Project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+International+HapMap+Project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000287-Rabbee1" id="pone.0000287-Rabbee1"></a>Rabbee N, Speed TP (2006) A genotype calling algorithm for affymetrix SNP arrays. Bioinformatics  22: 7–12.  <ul class="find" data-citedArticleID="936796" data-doi="10.1093/bioinformatics/bti741"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti741" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genotype+calling+algorithm+for+affymetrix+SNP+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genotype+calling+algorithm+for+affymetrix+SNP+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000287-Gentleman1" id="pone.0000287-Gentleman1"></a>Gentleman RC, Carey VJ, Bates DM, Bolstad B, Dettling M, et al.  (2004) Bioconductor: open software development for computational biology and bioinformatics. Genome Biol  5: R80.  <ul class="find" data-citedArticleID="936786" data-doi="10.1186/gb-2004-5-10-r80"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioconductor%3A+open+software+development+for+computational+biology+and+bioinformatics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000287-Carvalho1" id="pone.0000287-Carvalho1"></a>Carvalho B, Bengtsson H, Speed TP, Irizarry RA (2006) Exploration, Normalization, and Genotype Calls of High Density Oligonucleotide SNP Array Data. Biostatistics.   <ul class="find" data-citedArticleID="936780" data-doi="10.1093/biostatistics/kxl042"><li><a href="http://dx.doi.org/10.1093/biostatistics/kxl042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration%2C+Normalization%2C+and+Genotype+Calls+of+High+Density+Oligonucleotide+SNP+Array+Data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration%2C+Normalization%2C+and+Genotype+Calls+of+High+Density+Oligonucleotide+SNP+Array+Data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000287-Borevitz1" id="pone.0000287-Borevitz1"></a>Borevitz JO, Liang D, Plouffe D, Chang HS, Zhu T, et al.  (2003) Large-scale identification of single-feature polymorphisms in complex genomes. Genome Res  13: 513–523.  <ul class="find" data-citedArticleID="936778" data-doi="10.1101/gr.541303"><li><a href="http://dx.doi.org/10.1101/gr.541303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+identification+of+single-feature+polymorphisms+in+complex+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+identification+of+single-feature+polymorphisms+in+complex+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.XML" value="29755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.PDF" value="151445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g001.PNG_L" value="317707"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g001.PNG_M" value="54283"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g001.PNG_S" value="3020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g001.TIF" value="425974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g001.PNG_I" value="19437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g002.PNG_L" value="132148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g002.PNG_M" value="91100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g002.PNG_S" value="3698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g002.TIF" value="267316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g002.PNG_I" value="38820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g003.PNG_L" value="584200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g003.PNG_M" value="175463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g003.PNG_S" value="5990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g003.TIF" value="744294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.g003.PNG_I" value="60898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.s001.TIF" value="625936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.s002.XLS" value="16896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.s003.XLS" value="17408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000287.s004.XLS" value="17920"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000287&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000287" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000287&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000287&volume=&issue=&title=Redundancy in Genotyping Arrays&author_name=Scott%20Smemo%2C%20Justin%20O.%20Borevitz&start_page=1&end_page=3" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000287" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287&amp;t=Redundancy%20in%20Genotyping%20Arrays" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287&title=Redundancy%20in%20Genotyping%20Arrays&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287&amp;title=Redundancy%20in%20Genotyping%20Arrays" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000287&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Redundancy in Genotyping Arrays';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000287';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Redundancy%20in%20Genotyping%20Arrays http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000287" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000287" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="23412"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40169" data-articleid="23412"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Data+reduction%22" title="Search for articles in the subject area:'Data reduction'"><div class="flagText">Data reduction</div></a>
              <div data-categoryid="18411" data-articleid="23412"
                   data-categoryname="Data reduction"
                   class="flagImage" title="Flag 'Data reduction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genotyping%22" title="Search for articles in the subject area:'Genotyping'"><div class="flagText">Genotyping</div></a>
              <div data-categoryid="47691" data-articleid="23412"
                   data-categoryname="Genotyping"
                   class="flagImage" title="Flag 'Genotyping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="23412"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="19653" data-articleid="23412"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sense+strands%22" title="Search for articles in the subject area:'Sense strands'"><div class="flagText">Sense strands</div></a>
              <div data-categoryid="20151" data-articleid="23412"
                   data-categoryname="Sense strands"
                   class="flagImage" title="Flag 'Sense strands' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="23412"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9052'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=735'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7525&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
