

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001468"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001468" />

    <meta name="citation_title" content="A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells"/>
    <meta itemprop="name" content="A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells"/>

      <meta name="citation_author" content="Gilles Charvin"/>
            <meta name="citation_author_institution" content="Center For Studies in Physics and Biology, The Rockefeller University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Frederick R. Cross"/>
            <meta name="citation_author_institution" content="The Rockefeller University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Eric D. Siggia"/>
            <meta name="citation_author_institution" content="Center For Studies in Physics and Biology, The Rockefeller University, New York, New York, United States of America"/>

    <meta name="citation_date" content="2008/1/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001468.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1468"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Variability in individual cell cycles of sacchormyces cerevisiae.; citation_author=PG Lord; citation_author=AE Wheals; citation_journal_title=J. Cell. Sci.; citation_volume=50; citation_number=1; citation_pages=361-376; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Symmetric cell division in pseudohyphae of the yeast Saccharomyces Cerevisiae.; citation_author=SJ Kron; citation_author=CA Styles; citation_author=GR Fink; citation_journal_title=Mol. Biol. Cell.; citation_volume=5(9); citation_number=2; citation_pages=1003-1022; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The size control of fission yeast revisited.; citation_author=A Sveiczer; citation_author=B Novak; citation_author=JM Mitchison; citation_journal_title=J. Cell Sci.; citation_volume=109 (Pt 12); citation_number=3; citation_pages=2947-57; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Global analysis of protein localization in budding yeast.; citation_author=WK Huh; citation_author=JV Falvo; citation_author=LC Gerke; citation_author=AS Carroll; citation_author=RW Howson; citation_journal_title=Nature; citation_volume=425(6959); citation_number=4; citation_pages=686-91; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications.; citation_author=T Nagai; citation_author=K Ibata; citation_author=ES Park; citation_author=M Kubota; citation_author=K Mikoshiba; citation_journal_title=Nat. Biotech.; citation_volume=20(1); citation_number=5; citation_pages=87-90; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Improved monomeric red, orange and yellow fluorescent proteins derived from discosoma sp. red fluorescent protein.; citation_author=NC Shaner; citation_author=RE Campbell; citation_author=PA Steinbach; citation_author=BN Giepmans; citation_author=AE Palmer; citation_journal_title=Nat. Biotech.; citation_volume=22(12); citation_number=6; citation_pages=1567:1572; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Coherence and timing of cell cycle start examined at single-cell resolution.; citation_author=JM Bean; citation_author=ED Siggia; citation_author=FR Cross; citation_journal_title=Mol. Cell.; citation_volume=21(1); citation_number=7; citation_pages=3-14; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Single-cell quantification of molecules and rates using open-source microscope-based cytometry.; citation_author=A Gordon; citation_author=A Colman-Lerner; citation_author=TE Chin; citation_author=KR Benjamin; citation_author=RC Yu; citation_journal_title=Nat. Methods; citation_volume=4(2); citation_number=8; citation_pages=175-81; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MAPK-mediated bimodal gene expression and adaptive gradient sensing in yeast.; citation_author=S Paliwal; citation_author=PA Iglesias; citation_author=K Campbell; citation_author=Z Hilioti; citation_author=A Groisman; citation_journal_title=Nature; citation_volume=446(7131); citation_number=9; citation_pages=46-51; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Monitoring dynamics of single-cell gene expression over multiple cell cycles.; citation_author=S Cookson; citation_author=N Ostroff; citation_author=WL Pang; citation_author=D Volfson; citation_author=J Hasty; citation_journal_title=Mol Syst Biol; citation_volume=1; citation_number=10; citation_pages=2005:00024; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacterial persistence as a phenotypic switch.; citation_author=NQ Balaban; citation_author=J Merrin; citation_author=R Chait; citation_author=L Kowalik; citation_author=S Leibler; citation_journal_title=Science; citation_volume=300(5690); citation_number=11; citation_pages=1622-5; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The mechanical properties of saccharomyces cerevisiae.; citation_author=AE Smith; citation_author=Z Zhang; citation_author=CR Thomas; citation_author=KE Moxham; citation_author=AP Middelberg; citation_journal_title=Proc Natl Acad Sci U S A.; citation_volume=97(18); citation_number=12; citation_pages=9871-4; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microfluidic devices fabricated in poly(dimethylsiloxane) for biological studies.; citation_author=SK Sia; citation_author=GM Whitesides; citation_journal_title=Electrophoresis; citation_volume=24(21); citation_number=13; citation_pages=3563-76; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=In Methods in Yeast Genetics, 2000; citation_author=D Burke; citation_author=D Dawson; citation_author=T Stearns; citation_number=14; " />
      <meta name="citation_reference" content="citation_title=Destabilized green fluorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry.; citation_author=C Mateus; citation_author=S Avery; citation_journal_title=Yeast; citation_volume=16(14); citation_number=15; citation_pages=1313-1323; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rapid degradation of the G1 cyclin cln2 induced by cdk-dependent phosphorylation.; citation_author=S Lanker; citation_author=MH Valdivieso; citation_author=C Wittenberg; citation_journal_title=Science; citation_volume=271(5255); citation_number=16; citation_pages=1597-601; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Regulatable promoters of saccharomyces cerevisiae: comparison of transcriptional activity and their use for heterologous expression.; citation_author=D Mumberg; citation_author=R Muller; citation_author=M Funk; citation_journal_title=Nucl. Acid Res.; citation_volume=22(25); citation_number=17; citation_pages=5767-5768; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Met3 promoter: A tightly regulated promoter and its application in construction of conditional lethal strain.; citation_author=X Mao; citation_author=Y Hu; citation_author=C Liang; citation_author=C Lu; citation_journal_title=Curr. Microb.; citation_volume=45(1); citation_number=18; citation_pages=37-40; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The effects of molecular noise and size control on variability in the budding yeast cell cycle.; citation_author=S Di Talia; citation_author=JM Skotheim; citation_author=JM Bean; citation_author=ED Siggia; citation_author=FR Cross; citation_journal_title=Nature; citation_volume=448(7156); citation_number=19; citation_pages=947-951; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Closing the cell cycle circle in yeast: G2 cyclin proteolysis initiated at mitosis persists until the activation of G1 cyclins in the next cycle.; citation_author=A Amon; citation_author=S Irniger; citation_author=K Nasmyth; citation_journal_title=Cell; citation_volume=77(7); citation_number=20; citation_pages=1037-50; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Coherence and timing of cell cycle start examined at single-cell resolution.; citation_author=M Costanzo; citation_author=JL Nishikawa; citation_author=X Tang; citation_author=JS Millman; citation_author=O Schub; citation_journal_title=Cell; citation_volume=117(7); citation_number=21; citation_pages=899-913; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cln3 activates G1-specific transcription via phosphorylation of the SBF bound repressor Whi5.; citation_author=RA de Bruin; citation_author=WH McDonald; citation_author=TI Kalashnikova; citation_author=J Yates; citation_author=C Wittenberg; citation_journal_title=Cell; citation_volume=117(7); citation_number=22; citation_pages=887-898; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Coordination of growth with cell division in the yeast saccharomyces cerevisiae.; citation_author=GC Johnston; citation_author=JR Pringle; citation_author=LH Hartwell; citation_journal_title=Exp Cell Res.; citation_volume=105(1); citation_number=23; citation_pages=79-98; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Fusion of GAL4-VP16 to a steroid-binding domain provides a tool for gratuitous induction of galactose-responsive genes in yeast.; citation_author=JF Louvion; citation_author=B Havaux-Copf; citation_author=D Picard; citation_journal_title=Gene; citation_volume=131; citation_number=24; citation_pages=129-134; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Mode locking the cell cycle.; citation_author=FR Cross; citation_author=ED Siggia; citation_journal_title=PRE; citation_volume=72(2 Pt 1); citation_number=25; citation_pages=021910; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Periodic forcing of a mathematical model of the eukaryotic cell cycle.; citation_author=D Battogtokh; citation_author=JJ Tyson; citation_journal_title=PRE; citation_volume=73(1 Pt 1); citation_number=26; citation_pages=011910; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The green fluorescent protein.; citation_author=RY Tsien; citation_journal_title=Annu Rev Biochem.; citation_volume=67; citation_number=27; citation_pages=509-44; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comprehensive identification of cell cycle-regulated genes of the yeast Saccharomyces Cerevisiae by microarray hybridization.; citation_author=PT Spellman; citation_author=G Sherlock; citation_author=MQ Zhang; citation_author=VR Iyer; citation_author=K Anders; citation_journal_title=Mol. Biol. Cell; citation_volume=9(12); citation_number=28; citation_pages=3273-97; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells"/>
    <meta name="twitter:description" content="BackgroundImaging single cells with fluorescent markers over multiple cell cycles is a powerful tool for unraveling the mechanism and dynamics of the cell cycle. Over the past ten years, microfluidic techniques in cell biology have emerged that allow for good control of growth environment. Yet the control and quantification of transient gene expression in unperturbed dividing cells has received less attention.Methodology/Principal FindingsHere, we describe a microfluidic flow cell to grow Saccharomyces Cerevisiae for more than 8 generations (&asymp;12 hrs) starting with single cells, with controlled flow of the growth medium. This setup provides two important features: first, cells are tightly confined and grow in a remarkably planar array. The pedigree can thus be determined and single-cell fluorescence measured with 3 minutes resolution for all cells, as a founder cell grows to a micro-colony of more than 200 cells. Second, we can trigger and calibrate rapid and transient gene expression using reversible administration of inducers that control the GAL1 or MET3 promoters. We then show that periodic 10&ndash;20 minutes gene induction pulses can drive many cell division cycles with complete coherence across the cell cluster, with either a G1/S trigger (cln1 cln2 cln3 MET3-CLN2) or a mitotic trigger (cdc20 GALL-CDC20).Conclusions/SignificanceIn addition to evident cell cycle applications, this device can be used to directly measure the amount and duration of any fluorescently scorable signal-transduction or gene-induction response over a long time period. The system allows direct correlation of cell history (e.g., hysteresis or epigenetics) or cell cycle position with the measured response."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001468.g005"/>

  <meta property="og:title" content="A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4606'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5770'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3623&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001468">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001468" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001468&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gilles Charvin
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gilles.charvin@gmail.com">gilles.charvin@gmail.com</a></p>

                <p>Affiliation:
                  Center For Studies in Physics and Biology, The Rockefeller University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frederick R. Cross, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Rockefeller University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eric D. Siggia
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center For Studies in Physics and Biology, The Rockefeller University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001468</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001468-g001" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001468-g002" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001468-g003" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001468-g004" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001468-g005" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001468">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001468" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Imaging single cells with fluorescent markers over multiple cell cycles is a powerful tool for unraveling the mechanism and dynamics of the cell cycle. Over the past ten years, microfluidic techniques in cell biology have emerged that allow for good control of growth environment. Yet the control and quantification of transient gene expression in unperturbed dividing cells has received less attention.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here, we describe a microfluidic flow cell to grow <em>Saccharomyces Cerevisiae</em> for more than 8 generations (≈12 hrs) starting with single cells, with controlled flow of the growth medium. This setup provides two important features: first, cells are tightly confined and grow in a remarkably planar array. The pedigree can thus be determined and single-cell fluorescence measured with 3 minutes resolution for all cells, as a founder cell grows to a micro-colony of more than 200 cells. Second, we can trigger and calibrate rapid and transient gene expression using reversible administration of inducers that control the <em>GAL1</em> or <em>MET3</em> promoters. We then show that periodic 10–20 minutes gene induction pulses can drive many cell division cycles with complete coherence across the cell cluster, with either a G1/S trigger (<em>cln1 cln2 cln3 MET3-CLN2</em>) or a mitotic trigger (<em>cdc20 GALL-CDC20</em>).</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>In addition to evident cell cycle applications, this device can be used to directly measure the amount and duration of any fluorescently scorable signal-transduction or gene-induction response over a long time period. The system allows direct correlation of cell history (e.g., hysteresis or epigenetics) or cell cycle position with the measured response.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Charvin G, Cross FR, Siggia ED (2008) A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells. PLoS ONE 3(1):
          e1468.
            doi:10.1371/journal.pone.0001468</p><p><strong>Academic Editor: </strong>Daniel Lew, Duke University Medical Centre, United States of America</p><p><strong>Received:</strong> November 14, 2007; <strong>Accepted:</strong> December 18, 2007; <strong>Published:</strong> January 23, 2008</p><p><strong>Copyright:</strong> © 2008 Charvin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the National Institute of Health (EDS, FRC), the National Science Foundation DMR-0517138 (EDS), and a long-term cross-disciplinary fellowship from the Human Frontier Science Program Organization (GC).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The utility of time lapse microscopy to study time dependent processes such as the cell cycle and signal transduction has been apparent since the 70's<a href="#pone.0001468-Lord1">[1]</a>–<a href="#pone.0001468-Sveiczer1">[3]</a>. The availability of numerous fluorescently tagged proteins<a href="#pone.0001468-Huh1">[4]</a>–<a href="#pone.0001468-Shaner1">[6]</a>, and the ease of integrating digital imaging with software to control the experiments has made this technology more appealing and accessible. Most applications to yeast grow the cells under a gel pad, a quick and easy method which provides fair confinement and a source of nutrients<a href="#pone.0001468-Kron1">[2]</a>–<a href="#pone.0001468-Sveiczer1">[3]</a>, <a href="#pone.0001468-Bean1">[7]</a>. However, it has two major limitations : first, continuous tracking of single cells becomes impossible after 5 divisions since cells pile up, so that large-scale and long-term collection of data (for the study of the cell cycle, epigenetic inheritance, adaptation and signal transduction pathways) are compromised. Second, it is impossible to vary the growth media, yet the need to investigate the single cell response to time-dependent stimuli is of major importance in cell cycle research and more generally for yeast cell biology.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recently, cells attached to a substrate were imaged for long period <a href="#pone.0001468-Gordon1">[8]</a>. This method allows media exchange, but daughter cells may bud upwards and be lost. Yeast have been confined and imaged in microfluidic chambers <a href="#pone.0001468-Paliwal1">[9]</a>, <a href="#pone.0001468-Cookson1">[10]</a> but rapid control of gene expression was not required for their applications and not demonstrated. Embodying in a single device the capacity for long term imaging and the means to induce and measure transient single-cell responses such as gene expression in unperturbed dividing cells, will greatly improve the resolution of many imaging studies, in the cell cycle context in particular.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here, we report a novel microfluidic setup for yeast that uses a dialysis membrane to separate the cells from the external flow <a href="#pone.0001468-Balaban1">[11]</a>. It provides much better confinement and planar growth than with gel pads and therefore allows high resolution imaging of single cells for more than 8 division cycles. Dense fields of cells can be segmented from a phase image, leaving the fluorescent channels free. Media can be rapidly changed with no visible movement in the growing colony. We have studied the induction of the regulatable <em>GAL1</em> and <em>MET3</em> promoters with their respective metabolites, and have shown that externally controlled pulses of gene expression can be achieved with a 10 minutes resolution using the <em>MET3</em> promoter. These experiments combined with modeling have allowed a precise quantitative characterization of single-cell responses of these promoters to induction. As a demonstration of the utility of the system for cell cycle studies, we synchronized a growing field of cells to periodic external signals for more than five division cycles, using either a G1/S or a G2/M trigger. These experiments yield quantitative information on cell cycle regulation and variability. The coupling of imaging with a tight control over gene expression will provide accurate and tightly controlled data for any dynamic process with a fluorescent or morphological readout.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A microfluidic device to grow single yeast cells as a flat monolayer colony</h4>
<h5>Design of the setup.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>We used a diffusive cellulose membrane that separates the growing cells from a main chamber where flow of media occurs<a href="#pone.0001468-Balaban1">[11]</a>: yeast cells were clamped between a polydimethylsiloxane (PDMS) coated coverslip and a membrane, so that they were mechanically constrained to bud horizontally (see <a href="#pone-0001468-g001">Fig. 1a</a>).</p>
<div class="figure" id="pone-0001468-g001"><div class="img"><a name="pone-0001468-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Layout of the microfluidic device; a) sketch of the setup: yeast cells are confined betwen a small layer of PDMS coated on a glass coverslip and a diffusive cellulose membrane (dialysis tubing); the PDMS probably deforms under the cells, thus creating a space between the PDMS layer and the membrane.</span></strong></p><a id="article1.body1.sec2.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec1.fig1.caption1.p1"></a><p>On top of the membrane, a microfabricated PDMS flow chamber is clamped to control the media; b) Top: Sequence of images showing overlay of phase and YFP fluorescence of a colony of cells that carry the bud neck marker CDC10-YFP during exponential growth; Bottom : cell contours and parentage as retrieved by the annotation software; Right images show enlargments of the cell colony (top), bud neck position (middle), and cell contour and parentage (bottom); c) Growth curves of WT cells during a time-lapse experiment. Data using a standard agar setup (blue line) and the flow chamber (red line) were fit using an exponential (dashed line, discarding the first 100 minutes). The doubling time was respectively 77 and 78 minutes.</p>
<span>doi:10.1371/journal.pone.0001468.g001</span></div>
<h5>Chamber stability under flow.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>On top of the membrane, a 100 micron deep channel (lateral dimensions 40 mm×600 µm) cast in PDMS was connected to media tanks using appropriate plastic tubing. We used a peristaltic pump to drive the flow and electric control valves to switch media (see <a href="#pone.0001468.s001">Fig. S1</a>). We could quickly and reversibly change media without washing away the cells or having to stick them on the surface. A flow rate up to 1 ml/min did not displace the cells nor the imaging focus point, even though this rate corresponds to a high fluid velocity of 0.1 m/s over the membrane and correspondingly large pressures. The setup was mechanically stable over more than 12 hours.</p>

<h5>Coverslip PDMS coating.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>Coating the coverslip with a ≈40 µm thick layer of PDMS was necessary to ensure the fast and reliable growth of cells : cells did not grow well when sandwiched directly between glass and the membrane. PDMS is 100 times softer than a yeast cell <a href="#pone.0001468-Smith1">[12]</a>, <a href="#pone.0001468-Sia1">[13]</a>, and therefore cushions the cells, perhaps explaining this effect. E. coli will also grow in our apparatus (data not shown), directly sandwiched between the PDMS and the membrane. In Ref <a href="#pone.0001468-Balaban1">[11]</a> bacteria were confined to groves in the PDMS to faciliate imaging. Evidently this is not required for growth.</p>
<a id="article1.body1.sec2.sec1.sec3.p2" name="article1.body1.sec2.sec1.sec3.p2"></a><p>Whether nonadherent mammalian cells can be grown in our device has not been investigated.</p>

<h5>Media switching time.</h5><a id="article1.body1.sec2.sec1.sec4.p1" name="article1.body1.sec2.sec1.sec4.p1"></a><p>The dead volumes in the tubes were at most tens of microliters, so fluid switching times with our flow rates were a few seconds. Therefore, the effective time to switch media was likely to be limited by diffusion through the cellulose membrane (thickness ≈30 µm and molecular weight cutoff 14 kDa) . The kinetics of fluorescence washout following switch from a fluorescein solution to water was on the order of 30–40 seconds, which gives an estimate of the diffusion time (data not shown).</p>

<h5>Sample heating.</h5><a id="article1.body1.sec2.sec1.sec5.p1" name="article1.body1.sec2.sec1.sec5.p1"></a><p>The microfluidic setup was placed on a heated aluminium plate at 32°C (see <a href="#pone.0001468.s002">Fig. S2</a>). The microscope objective was heated in a similar manner in order to ensure that no heat is lost through the objective. In contrast to enclosing the microscope in a heating chamber, local Peltier-based heating-cooling regulation allows efficient temperature regulation over ≈10 degrees within a few minutes.</p>

<h5>Image quality.</h5><a id="article1.body1.sec2.sec1.sec6.p1" name="article1.body1.sec2.sec1.sec6.p1"></a><p>Phase contrast and fluorescent images were acquired using a low noise digital camera. The phase images were quite sharp, allowing accurate cell boundary segmentation (see <a href="#pone-0001468-g001">Fig. 1b</a>)<a href="#pone.0001468-Bean1">[7]</a>. Fluorescent images of subcellular structures tend to be improved over agar confinement, but at the expense of a higher background, due to wavelength-dependent membrane autofluorescence. We used a software-based background subtraction method to remove the membrane contribution to the signal (see <a href="#s4">Methods</a>).</p>

<h5>Setup automation.</h5><a id="article1.body1.sec2.sec1.sec7.p1" name="article1.body1.sec2.sec1.sec7.p1"></a><p>The whole setup was fully automated and driven by Matlab based software: image acquisition; temperature, pump, valves, microscope and stage control, allowing for a fast and versatile control of the time-lapse process. Image processing (background substraction, cell segmentation) could be integrated within the same master program. Therefore, automatic steps in image analysis were done immediately after image acquisition.</p>

<h5>Growing and tracking single cells over 8 generations.</h5><a id="article1.body1.sec2.sec1.sec8.p1" name="article1.body1.sec2.sec1.sec8.p1"></a><p>Under these conditions, we were able to monitor proliferation of a single founder yeast cell over more than 8 generations (see <a href="#pone-0001468-g001">Fig. 1b</a> and suppl. <a href="#pone.0001468.s003">Fig. S3</a> and <a href="#pone.0001468.s009">Movie S1</a>). Cell division time (about 78 min for WT cells at 32°C) was found to be in good agreement with agar pad time-lapse experiments done on our thermally controlled stage (see <a href="#pone-0001468-g001">Fig. 1c</a>) and comparable to liquid culture growth rate (data not shown).</p>
<a id="article1.body1.sec2.sec1.sec8.p2" name="article1.body1.sec2.sec1.sec8.p2"></a><p>We followed up to 10 separate colonies in an experiment, derived from 10 founder cells found in the flow chamber where about 20 cells were present at the beginning of the experiment. The typical number of cells in the chamber after 12 hours (≈9 divisions) was thus about 10∧6 cell/mL, which is 100-fold below saturation density (the volume of the chamber is few microliters). Therefore, even in the absence of the usual medium flow, no nutrient limitation was expected at the end of the experiment. There are many validated fluorescent markers for phases of the yeast cell cycle. The most useful of these, such as the budneck marker <em>CDC10</em>-YFP<a href="#pone.0001468-Bean1">[7]</a>, signal by translocation: Cdc10 localizes to the bud neck at bud emergence and then disperses into the cytoplasm upon cytokinesis. Using this marker we could score budding and cytokinesis even in a dense field of cells, with nearly single frame (3 minute) accuracy. We have further developed our MATLAB-based segmentation and fluorescence analysis software <a href="#pone.0001468-Bean1">[7]</a> to deal with larger colonies. We could thus monitor the growth and division of cells and construct pedigree trees of microcolonies containing more than 200 cells (see <a href="#pone-0001468-g001">Fig. 1b</a>, <a href="#pone.0001468.s006">Table S1</a> for timings, suppl. <a href="#pone.0001468.s008">Text S1</a> and <a href="#pone.0001468.s009">Movie S1</a>).</p>



<h4>Controlled gene expression using inducible promoters</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In budding yeast, there is a limited number of effective inducible promoters, among which are the <em>MET3</em> promoter (<em>MET3</em>pr) and <em>GAL1</em>pr. The <em>MET3</em>pr is appealing for the temporal control of gene expression since its expression level is not as high as the <em>GAL1</em>pr, which generally causes massive overexpression compared to most native promoters; also, the <em>MET3</em>pr does not require a change in carbon source (unlike <em>GAL1)</em>; the carbon source changes required for <em>GAL1</em> regulation can have highly pleiotropic effects on metabolism obscuring specific effects of gene induction. The <em>MET3</em> promoter is repressed by methionine (Met) in the media, and there is very little change in growth conditions comparing SCD+0xMet and SCD+10xMet (where 1xMet is 20 mg/L as in standard SCD media<a href="#pone.0001468-Burke1">[14]</a>)<a href="#pone.0001468-Bean1">[7]</a>.</p>


<h4>Kinetics of the <em>MET3</em> promoter</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To study induction kinetics in single cells, we integrated <em>MET3</em>pr fused to the Venus YFP protein<a href="#pone.0001468-Nagai1">[5]</a> destabilized by the <em>CLN2</em> PEST sequence <a href="#pone.0001468-Mateus1">[15]</a>, Venus-deg (<a href="#s4">Methods</a>). We grew a colony of cells in the flow cell in SCD+10xMet, and then switched to 0xMet for a defined period τ to induce expression, and then switched back to 10xMet (<a href="#pone-0001468-g002">Fig. 2</a> and <a href="#pone.0001468.s010">Movie S2</a>). To quantify both the promoter and the fluorescent reporter, we fit single cell time traces to a minimal model, <a href="#pone-0001468-g002">Fig. 2b</a> and Supplementary <a href="#pone.0001468.s008">Text S1</a>. Induction is very synchronous across all cells with a lag δ (defined from the beginning of the media shift to the onset of fluorescence). The lag could come from Met signal transduction, consumption of cell Met reserves, and the activation of transcription and translation. For each cell we fit a production rate γ for non-fluorescent Venus; a fluorescence maturation rate β (<em>p</em> and <em>p</em>* represent respectively the unmatured and fluorescent protein concentration); a protein decay rate α and the lag δ.</p>
<div class="figure" id="pone-0001468-g002"><div class="img"><a name="pone-0001468-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Transient induction of the MET3 promoter; a) Phase contrast, and fluorescence images of MET3pr-Venus-deg cells that were shifted to media lacking Met (0x Met) for 15 minutes at T = 420 min.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Blue contours indicate cells whose mean fluorescence is monitored in (b). Top images show the phase contrast overlaid with the MYO1-mCherry budneck marker. Bottom images show the Venus fluorescence in the yellow channel. b) Quantification of mean cellular fluorescence as a function of time after media switch (indicated as grey shade). Each color curve represents a single (mother) cell. Several representative fits of the bold curves to the model (inset, see text) are shown as dashed lines. Inset: protein expression model used to analyze the fluorescence signal (see suppl. model for details). c) Measurement of Venus maturation time in cells carrying stable MET3pr-Venus; Cycloheximide (2 µg/ml) is added to the 0xMet media 30 minutes after MET3pr induction (gray shade). An abrupt cessation of growth ensues 21 min later as defined by the intersection of the two exponential fits (dashed) to the colony growth size (red curve, in square pixel units) in the top panel. Bottom: Colored curves show the kinetics of single cell fluorescence. Solid black lines are representative exponential fits to determine the maturation rate b. d) Histogram of the maturation half-time &lt;Tβ&gt; = ln(2)/β measured by fitting individual cells as in (c) : &lt;Tβ&gt; = ln(2)/β = 18.5+/−0.4 min, COV = 0.16. e) Single cell histograms of Venus-deg protein degradation half-time &lt;Tα&gt; = ln(2)/α and expression rate γ; &lt;Tα&gt; = ln(2)/α = 39.2+/−1.4 min, COV = 0.23, &lt;γ&gt; = 7.3+/−0.3, COV = 0.25. f) Integrated mean fluorescence as function of pulse duration (red dots); black dots are predicted using the model, the mean model parameters, and the imposed pulse duration. g) Fraction of blocked cells in a strain that requires CLN2 expression from the MET3pr after division for cell cycle progression, as a function of the time between the switch to 10xMet media and division. This sets a stringent bound on the turnoff time of MET3pr; see text.</p>
<span>doi:10.1371/journal.pone.0001468.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>These four parameters could not be fit simultaneously from single cell traces since their effects were coupled (especially α and β). To get an independent measurement of Venus maturation time we integrated <em>MET3</em>pr expressing stable Venus protein (no degron) and then did a 30 min induction of <em>MET3</em>pr followed by the introduction of cycloheximide to block new protein synthesis (<a href="#pone-0001468-g002">Fig. 2c</a>)<a href="#pone.0001468-Gordon1">[8]</a>. Colony growth rate (in area) stopped abruptly 21 min after cycloheximide addition. Actual protein synthesis turn off is known to be faster than 21 min, so it is likely that cells keep swelling for a while after cycloheximide flows in (for example, by vacuolar expansion, see top panel in <a href="#pone-0001468-g002">Fig. 2c</a>). However, the same exponential maturation rate will be found no matter when we begin the fit, provided there is no new protein synthesis and all new fluorescence comes from maturation of previously synthesized YFP. A single exponential fit gave directly the maturation half-time &lt;Tβ&gt; = ln(2)/β = 18.5+/−0.4 min, with low cell-to-cell variability, as expected (standard-deviation/mean or coefficient of variation (COV) = 0.16), <a href="#pone-0001468-g002">Fig. 2d</a>. The illumination level was 3x lower than in <a href="#pone-0001468-g002">Fig. 2b</a> to limit photobleaching; indeed, no fluorescence decay was observed on a 2 hour timescale. We used this mean value of maturation rate in order to retrieve α, γ and δ from the curves in <a href="#pone-0001468-g002">Fig. 2b</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>δ displayed very little variability (not shown) and its mean value was 16.8±0.6 min; it did not depend on the pulse duration τ (not shown). Histograms of α and γ for each cell are displayed in <a href="#pone-0001468-g002">Fig. 2e</a>. The mean degradation half time &lt;Tα&gt; = ln(2)/α = 39.2+/−1.4 min (COV = 0.23)) is consistent with previous bulk measurements<a href="#pone.0001468-Mateus1">[15]</a>. It includes the effect of dilution by cell growth and photobleaching (which is negligible). The mean expression rate was &lt;γ&gt; = 7.3±0.3, with moderate cell-to-cell dispersion (COV = 0.25). No substantial correlation was observed between these three parameters, or between each parameter and cell size or the phase of the cell cycle, see suppl. <a href="#pone.0001468.s004">Fig. S4</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Varying the pulse duration τ changed the total amount of protein produced, as measured by the area under the fluorescence response curves, see <a href="#pone-0001468-g002">Fig. 2f</a>. Values found were in good agreement with the model using average parameters extracted for τ = 15 min. Thus, the length of time in 0xMet is a good way to control the total amount of protein induced.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>To determine an upper bound for the turn-off time of <em>MET3pr</em>, we took advantage of a <em>cln1,2,3</em> strain with <em>CLN2</em> expressed from <em>MET3pr</em>. These cells arrest in G1 unless Cln2 is present after division. Both <em>CLN2</em> message and protein are degraded in 5–10 minutes <a href="#pone.0001468-Lanker1">[16]</a> and thus Cln2 is a good readout for the current transcription rate. We grew a colony of cells in 0xMet, shifted to 10xMet and recorded the division time of each cell (as defined by <em>CDC10</em>-YFP) relative to the media shift, and their subsequent history. When the shift occured more than 9 (resp. 15) minutes prior to division, 50% (resp. 100%) of the cells arrested (<a href="#pone-0001468-g002">Fig. 2g</a>), implying that they did not have enough Cln2 to enter the next cell cycle. Putting together the various times, implies a firm bound of less than ≈10 minutes for <em>MET3</em>pr shutoff following the beginning of the media shift, that is very coherent across the population.</p>


<h4>Graded response of <em>MET3</em>pr and comparison to other promoters</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To further characterize the potential utility of the regulatable <em>MET3</em>pr in various physiological contexts, we needed to determine its response to Met level and its strength relative to other promoters. There have been prior measurements that quantify the steady-state expression of <em>MET25</em> and <em>MET3</em> promoters in batch culture as a function of Met concentration<a href="#pone.0001468-Mumberg1">[17]</a>, <a href="#pone.0001468-Mao1">[18]</a>. However, a concern with these experiments at low Met levels is the consumption of Met by the yeast, and thus gradual induction due to Met depletion (N. Buchler, pers. comm.). In the flow cell, the low density of cells (see above) and continual refreshment of medium means that cells will be unable to significantly reduce available Met even at low concentrations. Moreover, in addition to steady-state levels, the transient regimes following media switch can be precisely characterized by measuring the parameters δ and γ.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We partially induced <em>MET3</em>pr by switching from 10xMet to various Met concentrations, see <a href="#pone-0001468-g003">Fig. 3</a>. The model parameters determined for the Met pulse experiments in <a href="#pone-0001468-g002">Fig. 2</a> describe the shift to 0xMet, with no adjustment. However for shift to low but non-zero Met levels (partial induction), the model describes only the initial fluorescence increase but not the steady state, while if γ is adjusted to reproduce the steady state the initial rise is clearly wrong. Thus the Met regulatory system apparently leads to maximum induction whenever Met becomes limiting to any degree, and then decreases expression, possibly to attain a constant steady-state cellular Met level that is independent of the concentration in the media.</p>
<div class="figure" id="pone-0001468-g003"><div class="img"><a name="pone-0001468-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of expression rates among different promoters; A log scale was used to display signals with different orders of magnitudes; (a-c) Mean fluorescence vs time for various Met induction levels following a media shift.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The black curve in (a) is a prediction from the gene expression model fit to the data in <a href="#pone-0001468-g002">Fig. 2</a>, and it is repeated in (b-c) to show that it continues to match the initial increase. The lower black curves in (b-c) are the model with the expression rate γ adjusted to match the asymptotic value (and the other parameters unchanged). d) Induction of the GAL1 promoter following the switch from SCR to SCR+1.5% galactose; The black line is a fit using the same parameters determined in <a href="#pone-0001468-g002">Fig. 2</a>, except γ has been changed to 28; e) Constitutive expression of the ACT1 promoter; f) Expression from the CLN2 promoter. The curves have been shifted to overlay and the black curve is a fit of our model with γ = 10 and an effective pulse duration of 15 minutes. The maturation and degradation parameters are from <a href="#pone-0001468-g002">Fig. 2</a>. f) Comparison of expression rates and steady-state levels for the different promoters, with the MET3 promoter normalized to 100.</p>
<span>doi:10.1371/journal.pone.0001468.g003</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The <em>MET3</em>pr strength was then compared to standard promoters : <em>GAL1</em>, <em>ACT1</em>, and <em>CLN2</em>. The same induction experiment as described above was done with <em>GAL1</em>pr-Venus-deg cells. The media was shifted from Raffinose (SCR) to SCR+1.5% Galactose.We obtained a good agreement to the data by adjusting the production rate γ to 28 to the steady state level and the time lag δ to 20 minutes (the maturation and degradation rates were unchanged), see <a href="#pone-0001468-g003">Fig. 3d</a>. However, the initial fluorescence rise was not as steep as the model would predict, which may indicate that the <em>GAL1</em> promoter activates more gradually than <em>MET3</em>pr.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Comparison of the steady-state level of the <em>MET3</em>pr at 0xMet to the constitutive <em>ACT1</em>pr revealed that the latter was sligthly weaker than <em>MET3</em>pr, see <a href="#pone-0001468-g003">Fig 3e</a>. With the highly transient and burst-like <em>CLN2</em> promoter, we obtained a good fit to the model by setting γ = 10 and a <em>CLN2</em> transcription duration τ = 15 min, see <a href="#pone-0001468-g003">Fig. 3f</a> (γ sets the fluorescence slope while τ defines the peak height). The rate of protein production from the <em>MET3</em>pr is thus comparable to the <em>CLN2</em>pr, making it a very suitable way to externally control the induction of <em>CLN2</em> protein. The comparison among all the promoters is shown in <a href="#pone-0001468-g003">Fig. 3g</a>. Thus, overall, the <em>MET3</em>pr displays fast turn on and off times, has rather low cell-cell variability, and has peak expression in a physiological range at least as judged by the comparison to <em>CLN2</em>pr. These characteristics make it suitable for physiological experiments in our flow cell.</p>


<h4>Control of cell cycle dynamics using transient pulses of gene expression</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The ability to transiently express a gene of interest while simultaneously monitoring cell cycle progression in a controlled environment will facilitate and better quantify many experiments in yeast cell biology. To illustrate the power of our technique for cell cycle applications, we manually triggered the cell cycle by periodically inducing a gene in a strain background where it is required for cell cycling. Since the budding yeast cell cycle is free running and noisy <a href="#pone.0001468-Bean1">[7]</a>, <a href="#pone.0001468-DiTalia1">[19]</a>, this provided an external clock against which cell cycle transitions can be timed; it immediately generated a large data set for both gene induction and cell cycle response, in a very physiological context. We examined two standard synchronization points, G1/S and mitotic exit.</p>


<h4>The G1/S transition trigger</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To externally trigger the G1/S transition we used a <em>cln1,2,3 MET3pr-CLN2</em> strain which is dependent on <em>CLN2</em> expression from the <em>MET3</em>pr to activate Start<a href="#pone.0001468-Amon1">[20]</a>. We used the budneck marker <em>CDC10</em>-YFP to time bud emergence and cytokinesis, and added a second fluorescent marker, a <em>WHI5</em>-GFP fusion<a href="#pone.0001468-Costanzo1">[21]</a>. Whi5 represses the transcription of genes involved in the G1/S transition, <a href="#pone-0001468-g004">Fig. 4a</a>. Whi5 enters the nucleus shortly before division and its exit from the nucleus is the earliest marker available for the Start transition<a href="#pone.0001468-Costanzo1">[21]</a>, <a href="#pone.0001468-deBruin1">[22]</a>. We measured Whi5 nuclear residence by computing the covariance of the <em>WHI5</em>-GFP signal for each cell. The covariance was high when Whi5 was nuclear (i.e. when almost all of the signal was restricted to a small area in the cell), but was low when Whi5 was uniformly spread in the cytoplasm. We used a narrow bandpass GFP filter, which allows better separation of WHI5-GFP and CDC10-YFP.</p>
<div class="figure" id="pone-0001468-g004"><div class="img"><a name="pone-0001468-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>G1/S and mitotic exit trigger; a,e) Sketches of the genetic networks involved in triggering of the G1/S transition and anaphase b,f) The fluorescent markers and design of the periodic pulses c,g) Overlay of phase and fluorescence images during a pulsing experiment.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The colors correspond to the schematic in the panel above and the times are chosen to coincide with the events in the schematic. The rows correspond to three successive periods. d) Phase and fluorescence for the growth of WT cells, rows spaced by one doubling-time.</p>
<span>doi:10.1371/journal.pone.0001468.g004</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We transferred cells growing in 0xMet to 1xMet in the device to block them, and then applied sequential pulses of 0xMet of 20 minutes duration, followed by 110 minutes in 1xMet (<a href="#pone-0001468-g004">Fig. 4b</a>). Cells rapidly and coherently responded to the external trigger, by activating the G1/S transition. This sequence of events was quantitatively monitored (see <a href="#pone-0001468-g004">Fig 4c</a>, <a href="#pone.0001468.s011">Movie S3</a> and time traces in 5a) by defining appropriate times: t0, beginning of induction pulse; t1, Whi5 nuclear exit; t2, bud emergence, t3, division. Strikingly, the cells maintained perfect synchrony for Whi5 nuclear exit and budding during the whole sequence of pulses (3 cycles shown in <a href="#pone-0001468-g004">Fig. 4c</a>); compared to the well-known asynchrony of WT cells (<a href="#pone-0001468-g004">Fig. 4d</a>). The 130 minute clock period is substantially longer than the WT population doubling time of 78 minutes. Cells thus got larger each division since in yeast, cell growth is largely independent of the cell cycle<a href="#pone.0001468-Johnston1">[23]</a>. Synchrony is also evident in the cell cluster phylogeny in <a href="#pone-0001468-g005">Fig. 5c</a> compared with WT cells in <a href="#pone-0001468-g005">Fig. 5e</a>.</p>
<div class="figure" id="pone-0001468-g005"><div class="img"><a name="pone-0001468-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001468" data-uri="info:doi/10.1371/journal.pone.0001468.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001468.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001468.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001468.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001468.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Quantitation of the periodic trigger experiments.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>a) Time traces of the fluorescent markers used in the G1/S trigger: bud state (blue, 1 is budded) and WHI5 location inferred from the its covariance (red points and dashed line fit), high values are nuclear. The grey band indicates the pulses of SCD+0xMet. b) For the mitotic trigger, bud state is blue, the onset of anaphase is indicated in red, and the SCG pulse is the grey band. c-e) Phylogenies showing the budded/unbudded periods for all descendents of a founder cell, with pulsing bands overlayed where appropriate : c) the G1/S trigger; d) the mitotic exit trigger; e) WT cells.</p>
<span>doi:10.1371/journal.pone.0001468.g005</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The average length of the t0-&gt;t1 interval was 18±2.5 minutes in clocked cells. This is in good agreement with our fluorescence based assay for <em>MET3</em>pr activation, where the lag δ was similar. Thus the initial ramp up in Cln2 production was sufficient to trigger a sharp Start transition as defined by Whi5 nuclear exit. Therefore, our data place tight bounds on all sources of noise between Met in the media and <em>MET3</em>pr activity in a physiologically relevant context.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>For comparison to WT cells, we assigned t0 to be the time of the previous WT division, in which case the t0-&gt;t1 interval is 11 and 1 minute(s) in WT daughters and mothers respectively. The appreciable WT mother-daughter asymmetry in the t0-&gt;t1 interval, was abolished in our clocked cells, since both mother and daughter experienced a synchronous pulse of Cln2 (as opposed to WT cells where G1/S cyclin expression is delayed in daughters). Pooling all mothers and daughters in the clocked population, we found tight control of with a COV of 0.2, whereas the corresponding ditribution in WT daughters yielded a COV of 1.2 (Whi5 exit is too rapid in mothers to accurately measure the COV). Subsequent intervals corresponding to events such as budding (t0-&gt;t2) and division (t0-&gt;t3) had comparable variability to WT (see Suppl. <a href="#pone.0001468.s005">Fig.S5</a> nd <a href="#pone.0001468.s006">Table S1</a>), indicating that noise in the cell cycle rapidly caused loss of synchrony, even within a single cycle.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>In addition to the 130 minute period cycles described above, we carried out the same procedure with a 78 min period (still with a 20 min 0X Met pulse) that matches the WT doubling time. We found that while mothers remained reliably entrained to the 0X Met pulses, a fraction of the daughter cells (the small ones) had not completed their cycle when the next pulse was triggered : therefore they (and their daughters) failed to bud in response to the next pulse, but they caught up at the next one (data not shown). This finding suggests that small cells, even when forced to bud following a <em>CLN2</em> pulse, lengthen the budded period and slow down the progression through the cell division cycle<a href="#pone.0001468-DiTalia1">[19]</a>.</p>


<h4>The Mitotic exit trigger</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p><em>CDC20</em> is an essential gene that is required to activate the Anaphase Promoting Complex (APC) in metaphase, and therefore controls anaphase and ultimate mitotic exit. <em>cdc20</em> mutants thus arrest in metaphase, but can be rescued by <em>CDC20</em> under the control of the <em>GALL</em> promoter (a truncated weaker version of the <em>GAL1</em> promoter) when grown on Galactose (see <a href="#pone-0001468-g004">Fig. 4e</a>). We tested synchronization of a <em>cdc20</em> mutant using short periodic induction of <em>GALL-CDC20</em>, by monitoring the bud neck marker <em>CDC10</em>-YFP and a <em>TUB1</em>-GFP (beta-tubulin) fusion in order to monitor the state of the spindle during mitosis.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>We briefly grew the cells in SCD so they all arrested in metaphase, which was characterized by a short, bright microtubule spindle close to the bud, <a href="#pone-0001468-g004">Fig. 4g</a>. These arrested cells were then placed in the flow cell, and periodically pulsed for 10 minutes in SCG followed by 100 min in SCD, for a total period of 110 minutes, see <a href="#pone-0001468-g004">Fig. 4f</a>. For more than 5 cycles (of which 3 are shown in <a href="#pone-0001468-g004">Fig. 4g</a>, see also <a href="#pone.0001468.s012">Movie S4</a>), the cells divided rapidly after the galactose pulses.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>We scored the onset of anaphase occuring at by visual inspection (monitoring the elongation of the spindle) and measured division and budding timings (resp. t2 and t3) using the budneck marker. <a href="#pone-0001468-g005">Fig. 5b</a> displays typical recordings showing these sequences of events (see <a href="#pone.0001468.s005">Fig. S5</a> and <a href="#pone.0001468.s006">Table S1</a> for histograms and timings). The cells reached anaphase 15 minutes after the beginning of the pulse with good synchrony (COV = 0.2). A cluster phylogeny in <a href="#pone-0001468-g005">Fig. 5d</a> further illustrates the very good synchrony of the whole micro-colony. In addition, we observed that an SCG pulse as short as 2.5 minutes allows 80% of the cells to enter anaphase, with similar timing (data not shown), showing very sharp induction of anaphase. In contrast, the unbudded period immediately following this synchronous mitosis exhibited significant variability. Thus, these experiments confirm noise in the unbudded period<a href="#pone.0001468-DiTalia1">[19]</a>. Comparison to the G1/S trigger experiment shows that this noise can be suppressed by tight induction of <em>CLN2</em>, but not by making the preceeding mitosis very synchronous.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our flow cell design offers several advantages over other designs for long-term imaging of the cell cycle. It is simple to build and uses PDMS technology only to fabricate the flow chamber. Size tolerances are not critical and all the parts can be reused, since we assemble our cell with a mechanical clamp rather than sealing the PDMS to the glass. An unexpected benefit of the membrane confinement is a clearer phase image, and better imaging of subcellular fluorescent markers than what was possible with the agar setup. The two inpenetratable yet elastically deformable layers confining the yeast force almost all cells to bud in the plane of focus. Remarkably, these constraints have no effect on the doubling time compared to spatially unconstrained growth in liquid medium.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The fast and reliable media switching offered by microfluidic devices allows for precise control of gene expression in single cells; this advantage has not been exploited previously. This application is most favorable with promoters than turn on and off quickly. The <em>MET3</em> promoter is shown here to exhibit excellent characteristics for such experiments. Induction timing is very homogeneous across the population, although the expression rate varies somewhat from cell to cell. Partial induction is possible with no appreciable increase in cell-cell variability. The promoter turns off in less than 10 minutes. We have also shown usability of galactose in the flow cell to induce the commonly used <em>GAL1</em> promoter; in other experiments we have also demonstrated efficient induction with steroid hormone receptors (data not shown)<a href="#pone.0001468-Louvion1">[24]</a>. Thus it is likely that most small-molecule inducers will work in this system.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Control of both the expression level and the timing is essential if an inducible gene is to substitute for a natural gene. The <em>MET3</em>pr expresses in a reasonable activity range. It is comparable to <em>ACT1</em>pr and 1/4 the level of the other widely used <em>GAL1</em>pr, which is sometimes too strong. The output from the naturally very dynamic <em>CLN2</em> gene can be recapitulated by maximal induction of <em>MET3</em>pr for 15 minutes, with a cell-cell variability of about 25%. Other evidence that <em>MET3</em>pr operates in a physiological range comes from the observation that when fused to <em>CLN2</em> in a <em>cln1,2,3</em> strain, all the cells arrest when grown in 1xMet, but they all progress when induced for 20 minutes in 0xMet.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The efficacy of our device for controlling the cell cycle is manifest in the periodic block-release experiments using either a G1/S or mitotic arrest. We could then time downstream cell cycle events relative to the external clock and thereby better isolate the sources of variability. The flow cell obliviates the need for synchronizing bulk cultures whenever a fluorescent readout is available, and does so with virtually no perturbation in the bulk growth rate.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>There is a natural follow-up to the observation of nearly perfect induction of cell cycle synchrony using brief pulses of <em>CLN2</em> or <em>CDC20</em> in <em>cln1,2,3</em> or <em>cdc20</em> backgrounds. One may ask if such pulses can also entrain the cell cycle in a wild-type background, and under what conditions. This “mode-locking” experiment, analogous to jet-lag or the phase-response curve in circadian studies, has been examined theoretically using a computational model for the cell cycle <a href="#pone.0001468-Cross1">[25]</a>, <a href="#pone.0001468-Battogtokh1">[26]</a>; our device now allows an experimental realization.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Other cell cycle questions that we are addressing include the question of whether if we put a cyclin such as Cln2 under the control of the <em>MET3</em> promoter, do its targets respond to the integrated gene dosage or its maximum? Is there a hierarchy of targets, that are triggered as the Cln2-dependent kinase level increases? How sigmoidal is the response? How tight is the temporal correlation between the early transcriptional events in Start and sensitivity to mating pheromone? Do the events downstream of Start (e.g. bud emergence, spindle pole body duplication, DNA replication) display different sensitivities to cyclin dosage? If we supply more mitotic cyclin in G2 will mitosis run faster?</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Comprehensive genetic screens and genome wide expression data sets provide hints of the complex networks that underlie cellular function. Flow cell technology and time lapse imaging of single cells are essential for performing well-controlled dynamic measurements. We have demonstrated a practical way to follow multiple fluorescent markers, and construct complete cell pedigrees over many cycles of division and growth comparable to conditions in liquid culture, combined with the ability to pulse with inducers. Any pathway that can be activated by small molecules with a response that can be monitored by fluorescence will benefit from our approach. The ability to measure responses in cells for which the complete history and cell cycle positions are known allows detection of epigenetic effects (e.g., hysteresis, ‘memory’ of previous responses) and cell cycle regulation of the response, without the need to purify or synchronize populations, and all manipulations are carried out on otherwise unperturbed, exponentially growing cells.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Strains and plasmids</h4>
<h5>Strain constructions.</h5><a id="article1.body1.sec4.sec1.sec1.p1" name="article1.body1.sec4.sec1.sec1.p1"></a><p>All strains were constructed either by transformation (see below) or by mating and tetrad analysis, using our standard lab stocks (all W303 background) as starting material (See <a href="#pone.0001468.s007">Table S2</a> for the list of strain)</p>

<h5>Plasmids carrying transcriptional reporters.</h5><a id="article1.body1.sec4.sec1.sec2.p1" name="article1.body1.sec4.sec1.sec2.p1"></a><p>All the plasmids carrying the coding sequence of Venus under the control of a given promoter (<em>ACT1</em>pr, <em>GAL1</em>pr, <em>CLN2</em>pr, <em>MET3</em>pr) were made in the following way: the plasmid pJB04T <a href="#pone.0001468-Bean1">[7]</a> (made from pSVA17<a href="#pone.0001468-Mateus1">[15]</a> containing the GFP sequence fused to the PEST sequence of the <em>CLN2</em> (deg) gene and driven by the <em>ACT1</em> promoter was cut using PacI and BsrGI and the GFP insert was discarded. We swapped in the Venus insert cut from pKT90 (EUROSCARF, set 1 of optimized tagging cassettes). The resulting plasmid (containing <em>ACT1</em>pr-Venus-deg was named pGC04D. The plasmid containing the <em>CLN2</em> promoter was made by swapping the promoter of pSVA17 and the one of pGC04D (using BamHI and PacI sites), resulting in pGC08D. Plasmids carrying the <em>GAL1</em> or the <em>MET3</em> promoter were built as follows : the promoter sequences were amplified using PCR from a template (respectively p405<em>GAL1</em><a href="#pone.0001468-Mumberg1">[17]</a> and pRS426-<em>MET3</em>-HA-Hog1 (a gift from the Engelberg lab, University of Jerusalem, Israel) for <em>GAL1</em> and <em>MET3</em>.</p>

<h5><em>MYO1</em>-mCherry fusion protein.</h5><a id="article1.body1.sec4.sec1.sec3.p1" name="article1.body1.sec4.sec1.sec3.p1"></a><p>We used the mCherry tagging plasmid (pKT355) donated by the Bi lab (University of Pennsylvania, USA). We designed specific oligos to amplify the coding sequence with 40 nucleotides flanking regions (see primer sequence in <a href="#pone.0001468.s007">Table S2</a>) in order to proceed with a single step PCR gene replacement. Insertion was screened by PCR and fluorescence visual inspection.</p>



<h4>Growth conditions and media</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Before starting the time-lapse experiments, cells were pregrown overnight in standard SCD with the appropriate Met dosage. The standard 1xMet concentration was set to 0.02 g/L<a href="#pone.0001468-Burke1">[14]</a>. The temperature was set to 32°C in the microfluidic setup, which ensured a growth rate very close to the optimal one defined in liquid cultures.</p>


<h4>Microfluidic device</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The microfluidic device, as reported on <a href="#pone-0001468-g001">Fig. 1</a>, is made by assembling three components: 1) A PDMS coated coverslip; 2) A cellulose membrane; 3) a PDMS flow cell. The components were prepared as follows:</p>
<h5>PDMS Coated Coverslip.</h5><a id="article1.body1.sec4.sec3.sec1.p1" name="article1.body1.sec4.sec3.sec1.p1"></a><p>The PDMS-coated coverslip was prepared as follows: 30 g of PDMS (Sylgard 184, GE) was mixed to the appropriate curing agent ( ratio 1:10 ), and degassed on the bench for 30 minutes. Then, 50–80 microliters were deposited on the surface of a clean 4 inch silicon wafer pre-coated with trimethylchlorosilane (Sigma Aldrich). A cleaned glass coverlip (gauge 1.5, whose thickness is about 170 microns) was put on top of the PDMS droplet; we allowed the PDMS to spread evenly on the surface of the coverslip for about 30 minutes. The wafer with the coverslip was then baked for 45 minutes at 80°C. The coverslip was then peeled off slowly using a razor blade or scalpel. The typical thickness of the PDMS layer is around 30–40 microns ( coverslip size : 24 mm×50 mm). This extra layer must be smaller than the objective free working distance (which is 100 microns in our case). It did not affect the quality of the imaging.</p>
<a id="article1.body1.sec4.sec3.sec1.p2" name="article1.body1.sec4.sec3.sec1.p2"></a><p>A coated coverslip can be used for many experiments. We washed them with detergent (Triton, 20 %) and deionized water. The surface of the PDMS tends to become more and more hydrophilic, so that the natural tendency of cells to stick on the surface decreases with time (after ≈10 experiments).</p>

<h5>Cellulose membrane preparation.</h5><a id="article1.body1.sec4.sec3.sec2.p1" name="article1.body1.sec4.sec3.sec2.p1"></a><p>The cellulose membrane used is taken from dialysis tubing (Sigma Alldrich, D9527). The high cutoff size (14 kD) allows media molecules to diffuse freely through the membrane pores. The tubing was cut to an appropriate size (22 mm×50 mm) , then soaked in deionized water in order to separate the two sides of the tube of membrane. The cellulose membrane was first boiled in a 2 % w./v. Na2CO3 solution for half an hour to remove glycerin and sulfur compounds, and boiled a second time in TE at pH8 for half an hour. Membranes can be stored wet at 4°C, and reused after cleaning.</p>
<a id="article1.body1.sec4.sec3.sec2.p2" name="article1.body1.sec4.sec3.sec2.p2"></a><p>The thickness of the membrane (about 30 microns) ensures that media and inducers diffuse quickly (about 30–60 s for a nanoscale particle).</p>

<h5>PDMS Flow Cell.</h5><a id="article1.body1.sec4.sec3.sec3.p1" name="article1.body1.sec4.sec3.sec3.p1"></a><p>The flow cell was made using standard PDMS techniques. A silicon wafer mould was generated using optical lithography techniques at the Cornell Nanoscale Facility (Ithaca, NY). A 4 inch silicon wafer was coated with photoresist following a desired pattern. The wafer was then etched using a plasma machine. The length of the designed channel was set to 40 mm. Its width is 600 microns and height is 100 microns . Because of the extreme simplicity and the dimensions of the channel, the simpler SU8 lithography techniques would probably work as well. PDMS (mixed with curing agent, as described above) was then cast on top of the mould and baked at 80°C for 2 hours. Inlet and oulet were generated using blunt needles (Gauge 26). Unlike the usual PDMS flow chambers that are sealed to the glass using air plasma-induced covalent bonding, the flow chamber is not sealed so that it can be cleaned after the experiment and reused at will.</p>

<h5>Assembling the device.</h5><a id="article1.body1.sec4.sec3.sec4.p1" name="article1.body1.sec4.sec3.sec4.p1"></a><p>A droplet of log-phase cells (10 microliters at about 2. 10<sup>6</sup> cells/ml) was deposited on the coated coverslip. A clean wet membrane was put on top of it. In order for the membrane to be in contact with the PDMS, we waited about 15 minutes so that the water layer between membrane and coverslip mostly evaporates. The membrane then tends to naturally stick to the PDMS. However, excessive drying would tend to damage the cells and the membrane ultimately unsticks when completely dried out. The flow cell was then disposed on top of the membrane at the appropriate time, and a plastic plate was used to clamp the whole system. The flow cell was connected to tubing (tygon, GE) using Gauge 21 needles. Flows were driven by a peristaltic pump (Ismatec, IPC4) and media selection is controlled using an array of electric control valves (The Lee Company, LFAA1200118H) that were powered using a 12 volts power supply (MacMaster) and computer-controlled using a RS232 switch board (National Control Devices, PWM8x).</p>



<h4>Heating stage and objective heating devices</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The microfluidic device was placed on a home made aluminium stage, mounted with two thermoelectric 72W devices (Melcor, CP 1.4-127-045L). Aluminium heatsinks (McMaster) were used to dissipate heat on the cold side of the modules. A powersupply with an integrated PID regulation (Oven Industries, 5C7-195) and temperature sensors (Oven Industries, TS-104 thermistors) were used to maintain the temperature at 32°C. Heating the objective as well (using a similar thermoelectric module and regulation) ensured that no heat was lost through the contact between the objective and the sample.</p>


<h4>Data Acquisition</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The heating stage was put on the automated stage of an inverted microscope (Leica DMI6000B) with a Leica 63x N.A. 1.4 objective (HCX PL APO CS). This microscope was fully automated and computer controlled. A FireWire Hamamatsu camera (C4742-80-12AG) was used to acquire 12 bits pictures at 1344×1024 resolution. The camera was connected to a fast shutter (Uniblitz, VS35S2ZM1) using a TTL link to precisely control the synchrony with the electronic triggering of the camera. Thus we could use low exposure times (typically 200 ms). We wrote dedicated software in Matlab 7.2 to control every step of the experiment using a graphical user interface : image acquisition (using Matlab Image Acquisition Toolbox 2.10), microscope driving, temperature, pump and electrovalves control. Hardware (except the camera) were interfaced through RS232 ports. The source code is available upon request.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Images were acquired every 3 minutes. Autofocusing was achieved on phase contrast images by acquiring a stack of images and selecting the one with maximal gradient. Fluorescent images (of Venus, GFP, mCherry fluorophores) were acquired using the appropriate filter sets (Chroma, respectively : 41028, 41020, 31004) using 2×2 pixels binning. Using a narrow bandpass filter for GFP helped cutting out the naturally high cell autofluorescence in the green, thus improving the signal/noise ratio.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>We noticed that the mean cell fluorescence levels tend to decrease with time (with a timescale of several hours) 7–8 hours after the beggining of the experiment. We found that a constant aeration of the media tanks (using an aquarium pump and bubblers from Aquatic Gardens) and large flow rates ( &gt;30 µL/min ) prevented this phenomenon. Without the bubbling, the dissolved oxygen concentration in our medium reservoirs dropped to a very low level by the end of an overnight run. Oxygen is required for maturation of fluorescent proteins<a href="#pone.0001468-Tsien1">[27]</a>.</p>


<h4>Data analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Data were analyzed using a Matlab program. It was used to edit and correct cell contours generated by a segmenting program previously described<a href="#pone.0001468-Bean1">[7]</a>. The new capability to automatically track bud neck markers eased the monitoring of large cell colonies. Raw fluorescence images were retreated in order to substract the membrane autofluorescence contribution to the signal. See supplementary <a href="#pone.0001468.s008">Text S1</a> for detail.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001468"></div><a name="pone.0001468.s001" id="pone.0001468.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Overall principle of the setup. The whole setup is driven by a custom-made Matlab application run on a PC computer, through RS232 interface (except the camera which has FireWire interface). The microscope, the camera, the fluoresence shutter, the control unit for the thermo-electric devices (TE, stage and objective heaters) as well as for the electric valves can be remotely controlled. A TTL link between the fluorescence shutter and the camera ensures a better synchrony of these two devices.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0001468.s002" id="pone.0001468.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Picture and sketch of the assembly of the setup. Picture and sketch of the assembly of the setup; Top: top view of the microfluidic device mounted on its stage. The whole flow cell ( coverslip+membrane+flow cell ) is clamped onto a heated aluminium alloy stage using screws attached to an aluminium top. Heating the stage as well as the objective using thermoelectric modules ensures a constant temperature.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.88 MB PDF)</p>
<a name="pone.0001468.s003" id="pone.0001468.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Comparison of cell proliferation in a standard 2% agar setup versus the microfluidic sandwich setup. Cells carrying a copy of CDC10-YFP were grown in the microfluidic device for 12 hours in SCD. Timelapse of cell growth using the standard agar method was carried out as previously described<a href="#pone.0001468-Bean1">[7]</a>. On the left is shown a sketch of the respective setups. On the right are displayed a sequence of image taken during the experiment at indicated times. Each phase image is processed using custom software in order to identify cells and extract their contours. The very flat colonies in the microfluidic device allows one to image cells for 8–10 generations, versus about 5 using agar pads <a href="#pone.0001468-Bean1">[7]</a>. Enlargments on the right emphasize the difference between the two setups at high cell density.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.84 MB PDF)</p>
<a name="pone.0001468.s004" id="pone.0001468.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Correlation of fitted γ and α with cell size (a-b) and cell-cycle phase (c-d), scored using the budneckmarker in <a href="#pone-0001468-g003">Fig. 3</a>. Numbers indicate the value of the coefficient of correlation. To determine what factors could affect the cell-to-cell variability of γ (which sets the peak height for a given τ in <a href="#pone-0001468-g002">Fig. 2b</a>), we correlated γ obtained for individual cells with cell size and cell cycle phase (using the red budneck marker MYO1-mCherry, which works similarly as CDC10-YFP). We did not see any obvious correlation (Fig. 10 a and b). The same analysis was done with α (Fig. 15c and d). We did not see any systematic fluorescence oscillations with the cell cycle for steady growth in 0xMet or 10xMet, in contrast to the G2 peak in expression from the Met regulon in rich medium<a href="#pone.0001468-Spellman1">[28]</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0001468.s005" id="pone.0001468.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Histograms for the G1/S trigger and the mitotic trigger. Histograms of timings associated to the various markers used in the pulsing experiments in <a href="#pone-0001468-g004">Fig. 4</a>. t0 , t1 , t2 and t3 are defined in the text (different for a and b).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.31 MB PDF)</p>
<a name="pone.0001468.s006" id="pone.0001468.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s006">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Timings of cell cycle events in triggering experimentsexperiments. WT Mother and Daughter represent control</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0001468.s007" id="pone.0001468.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s007">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Strain and plasmid list</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0001468.s008" id="pone.0001468.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s008">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Methods and models description</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(2.65 MB DOC)</p>
<a name="pone.0001468.s009" id="pone.0001468.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s009">Movie S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">WT cell growth, cell contour and bud neck annotation. The movie corresponds to experiments shown in <a href="#pone-0001468-g001">figure 1</a></p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(6.58 MB AVI)</p>
<a name="pone.0001468.s010" id="pone.0001468.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s010">Movie S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Transient gene expression. 15 minutes MET3 induction upon media shift</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(12.31 MB AVI)</p>
<a name="pone.0001468.s011" id="pone.0001468.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s011">Movie S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">The G1/S trigger. Repeated pulses of CLN2 (this movie corresponds to experiments displayed in <a href="#pone-0001468-g004">Fig. 4c</a>)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(14.09 MB AVI)</p>
<a name="pone.0001468.s012" id="pone.0001468.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001468.s012">Movie S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">The mitotic exit trigger. Repeated pulse od CDC20 (this movie correspondes to experiments displayed in <a href="#pone-0001468-g004">Fig. 4g</a>).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001468.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(11.42 MB AVI)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: ES GC FC. Performed the experiments: GC. Analyzed the data: ES GC FC. Wrote the paper: ES GC FC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001468-Lord1" id="pone.0001468-Lord1"></a>Lord PG, Wheals AE (1981) Variability in individual cell cycles of sacchormyces cerevisiae. J. Cell. Sci.  50: 361–376.  <ul class="find" data-citedArticleID="1044884"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Variability+in+individual+cell+cycles+of+sacchormyces+cerevisiae.&amp;auth=&amp;atitle=Variability+in+individual+cell+cycles+of+sacchormyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variability+in+individual+cell+cycles+of+sacchormyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variability+in+individual+cell+cycles+of+sacchormyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001468-Kron1" id="pone.0001468-Kron1"></a>Kron SJ, Styles CA, Fink GR (1994) Symmetric cell division in pseudohyphae of the yeast S<em>accharomyces Cerevisiae</em>. Mol. Biol. Cell.  5(9): 1003–1022.  <ul class="find" data-citedArticleID="1044880" data-doi="10.1091/mbc.5.9.1003"><li><a href="http://dx.doi.org/10.1091/mbc.5.9.1003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symmetric+cell+division+in+pseudohyphae+of+the+yeast+Saccharomyces+Cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symmetric+cell+division+in+pseudohyphae+of+the+yeast+Saccharomyces+Cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001468-Sveiczer1" id="pone.0001468-Sveiczer1"></a>Sveiczer A, Novak B, Mitchison JM (1996) The size control of fission yeast revisited. J. Cell Sci.  109 (Pt 12): 2947–57.  <ul class="find" data-citedArticleID="1044906"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+size+control+of+fission+yeast+revisited.&amp;auth=&amp;atitle=The+size+control+of+fission+yeast+revisited." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+size+control+of+fission+yeast+revisited." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+size+control+of+fission+yeast+revisited.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001468-Huh1" id="pone.0001468-Huh1"></a>Huh WK, Falvo JV, Gerke LC, Carroll AS, Howson RW, et al.  (2003) Global analysis of protein localization in budding yeast. Nature  425(6959): 686–91.  <ul class="find" data-citedArticleID="1044876" data-doi="10.1038/nature02026"><li><a href="http://dx.doi.org/10.1038/nature02026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+analysis+of+protein+localization+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+analysis+of+protein+localization+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001468-Nagai1" id="pone.0001468-Nagai1"></a>Nagai T, Ibata K, Park ES, Kubota M, Mikoshiba K, et al.  (2002) A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications. Nat. Biotech.  20(1): 87–90.  <ul class="find" data-citedArticleID="1044894" data-doi="10.1038/nbt0102-87"><li><a href="http://dx.doi.org/10.1038/nbt0102-87" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+variant+of+yellow+fluorescent+protein+with+fast+and+efficient+maturation+for+cell-biological+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+variant+of+yellow+fluorescent+protein+with+fast+and+efficient+maturation+for+cell-biological+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001468-Shaner1" id="pone.0001468-Shaner1"></a>Shaner NC, Campbell RE, Steinbach PA, Giepmans BN, Palmer AE, et al.  (2004) Improved monomeric red, orange and yellow fluorescent proteins derived from discosoma sp. red fluorescent protein. Nat. Biotech.  22(12): 1567:1572.  <ul class="find" data-citedArticleID="1044898" data-doi="10.1038/nbt1037"><li><a href="http://dx.doi.org/10.1038/nbt1037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+monomeric+red%2C+orange+and+yellow+fluorescent+proteins+derived+from+discosoma+sp.+red+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+monomeric+red%2C+orange+and+yellow+fluorescent+proteins+derived+from+discosoma+sp.+red+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001468-Bean1" id="pone.0001468-Bean1"></a>Bean JM, Siggia ED, Cross FR (2006) Coherence and timing of cell cycle start examined at single-cell resolution. Mol. Cell.  21(1): 3–14.  <ul class="find" data-citedArticleID="1044862" data-doi="10.1016/j.molcel.2005.10.035"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.10.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coherence+and+timing+of+cell+cycle+start+examined+at+single-cell+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coherence+and+timing+of+cell+cycle+start+examined+at+single-cell+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001468-Gordon1" id="pone.0001468-Gordon1"></a>Gordon A, Colman-Lerner A, Chin TE, Benjamin KR, Yu RC, et al.  (2007) Single-cell quantification of molecules and rates using open-source microscope-based cytometry. Nat. Methods  4(2): 175–81.  <ul class="find" data-citedArticleID="1044874" data-doi="10.1038/nmeth1008"><li><a href="http://dx.doi.org/10.1038/nmeth1008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-cell+quantification+of+molecules+and+rates+using+open-source+microscope-based+cytometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-cell+quantification+of+molecules+and+rates+using+open-source+microscope-based+cytometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001468-Paliwal1" id="pone.0001468-Paliwal1"></a>Paliwal S, Iglesias PA, Campbell K, Hilioti Z, Groisman A, et al.  (2007) MAPK-mediated bimodal gene expression and adaptive gradient sensing in yeast. Nature  446(7131): 46–51.  <ul class="find" data-citedArticleID="1044896" data-doi="10.1038/nature05561"><li><a href="http://dx.doi.org/10.1038/nature05561" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAPK-mediated+bimodal+gene+expression+and+adaptive+gradient+sensing+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAPK-mediated+bimodal+gene+expression+and+adaptive+gradient+sensing+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001468-Cookson1" id="pone.0001468-Cookson1"></a>Cookson S, Ostroff N, Pang WL, Volfson D, Hasty J (2005) Monitoring dynamics of single-cell gene expression over multiple cell cycles. Mol Syst Biol  1: 2005:00024.  <ul class="find" data-citedArticleID="1044866" data-doi="10.1038/msb4100032"><li><a href="http://dx.doi.org/10.1038/msb4100032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Monitoring+dynamics+of+single-cell+gene+expression+over+multiple+cell+cycles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Monitoring+dynamics+of+single-cell+gene+expression+over+multiple+cell+cycles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001468-Balaban1" id="pone.0001468-Balaban1"></a>Balaban NQ, Merrin J, Chait R, Kowalik L, Leibler S (2004) Bacterial persistence as a phenotypic switch. Science  300(5690): 1622–5.  <ul class="find" data-citedArticleID="1044858" data-doi="10.1126/science.1099390"><li><a href="http://dx.doi.org/10.1126/science.1099390" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+persistence+as+a+phenotypic+switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+persistence+as+a+phenotypic+switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001468-Smith1" id="pone.0001468-Smith1"></a>Smith AE, Zhang Z, Thomas CR, Moxham KE, Middelberg AP (2000) The mechanical properties of saccharomyces cerevisiae. Proc Natl Acad Sci U S A.  97(18): 9871–4.  <ul class="find" data-citedArticleID="1044902" data-doi="10.1073/pnas.97.18.9871"><li><a href="http://dx.doi.org/10.1073/pnas.97.18.9871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mechanical+properties+of+saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mechanical+properties+of+saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001468-Sia1" id="pone.0001468-Sia1"></a>Sia SK, Whitesides GM (2003) Microfluidic devices fabricated in poly(dimethylsiloxane) for biological studies. Electrophoresis  24(21): 3563–76.  <ul class="find" data-citedArticleID="1044900" data-doi="10.1002/elps.200305584"><li><a href="http://dx.doi.org/10.1002/elps.200305584" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microfluidic+devices+fabricated+in+poly%28dimethylsiloxane%29+for+biological+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microfluidic+devices+fabricated+in+poly%28dimethylsiloxane%29+for+biological+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001468-Burke1" id="pone.0001468-Burke1"></a>Burke D, Dawson D, Stearns TIn <em>Methods in Yeast Genetics</em>, 2000.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0001468-Mateus1" id="pone.0001468-Mateus1"></a>Mateus C, Avery S (2000) Destabilized green fluorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry. Yeast  16(14): 1313–1323.  <ul class="find" data-citedArticleID="1044890"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Destabilized+green+fluorescent+protein+for+monitoring+dynamic+changes+in+yeast+gene+expression+with+flow+cytometry.&amp;auth=&amp;atitle=Destabilized+green+fluorescent+protein+for+monitoring+dynamic+changes+in+yeast+gene+expression+with+flow+cytometry." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Destabilized+green+fluorescent+protein+for+monitoring+dynamic+changes+in+yeast+gene+expression+with+flow+cytometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Destabilized+green+fluorescent+protein+for+monitoring+dynamic+changes+in+yeast+gene+expression+with+flow+cytometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001468-Lanker1" id="pone.0001468-Lanker1"></a>Lanker S, Valdivieso MH, Wittenberg C (1996) Rapid degradation of the G1 cyclin cln2 induced by cdk-dependent phosphorylation. Science  271(5255): 1597–601.  <ul class="find" data-citedArticleID="1044882" data-doi="10.1126/science.271.5255.1597"><li><a href="http://dx.doi.org/10.1126/science.271.5255.1597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+degradation+of+the+G1+cyclin+cln2+induced+by+cdk-dependent+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+degradation+of+the+G1+cyclin+cln2+induced+by+cdk-dependent+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001468-Mumberg1" id="pone.0001468-Mumberg1"></a>Mumberg D, Muller R, Funk M (1994) Regulatable promoters of saccharomyces cerevisiae: comparison of transcriptional activity and their use for heterologous expression. Nucl. Acid Res.  22(25): 5767–5768.  <ul class="find" data-citedArticleID="1044892" data-doi="10.1093/nar/22.25.5767"><li><a href="http://dx.doi.org/10.1093/nar/22.25.5767" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatable+promoters+of+saccharomyces+cerevisiae%3A+comparison+of+transcriptional+activity+and+their+use+for+heterologous+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatable+promoters+of+saccharomyces+cerevisiae%3A+comparison+of+transcriptional+activity+and+their+use+for+heterologous+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001468-Mao1" id="pone.0001468-Mao1"></a>Mao X, Hu Y, Liang C, Lu C (2002) Met3 promoter: A tightly regulated promoter and its application in construction of conditional lethal strain. Curr. Microb.  45(1): 37–40.  <ul class="find" data-citedArticleID="1044888" data-doi="10.1007/s00284-001-0046-0"><li><a href="http://dx.doi.org/10.1007/s00284-001-0046-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Met3+promoter%3A+A+tightly+regulated+promoter+and+its+application+in+construction+of+conditional+lethal+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Met3+promoter%3A+A+tightly+regulated+promoter+and+its+application+in+construction+of+conditional+lethal+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001468-DiTalia1" id="pone.0001468-DiTalia1"></a>Di Talia S, Skotheim JM, Bean JM, Siggia ED, Cross FR (2007) The effects of molecular noise and size control on variability in the budding yeast cell cycle. Nature  448(7156): 947–951.  <ul class="find" data-citedArticleID="1044872" data-doi="10.1038/nature06072"><li><a href="http://dx.doi.org/10.1038/nature06072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+molecular+noise+and+size+control+on+variability+in+the+budding+yeast+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+molecular+noise+and+size+control+on+variability+in+the+budding+yeast+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001468-Amon1" id="pone.0001468-Amon1"></a>Amon A, Irniger S, Nasmyth K (1994) Closing the cell cycle circle in yeast: G2 cyclin proteolysis initiated at mitosis persists until the activation of G1 cyclins in the next cycle. Cell  77(7): 1037–50.  <ul class="find" data-citedArticleID="1044856" data-doi="10.1016/0092-8674(94)90443-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(94)90443-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Closing+the+cell+cycle+circle+in+yeast%3A+G2+cyclin+proteolysis+initiated+at+mitosis+persists+until+the+activation+of+G1+cyclins+in+the+next+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Closing+the+cell+cycle+circle+in+yeast%3A+G2+cyclin+proteolysis+initiated+at+mitosis+persists+until+the+activation+of+G1+cyclins+in+the+next+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001468-Costanzo1" id="pone.0001468-Costanzo1"></a>Costanzo M, Nishikawa JL, Tang X, Millman JS, Schub O, et al.  (2004) Coherence and timing of cell cycle start examined at single-cell resolution. Cell  117(7): 899–913.  <ul class="find" data-citedArticleID="1044868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Coherence+and+timing+of+cell+cycle+start+examined+at+single-cell+resolution.&amp;auth=&amp;atitle=Coherence+and+timing+of+cell+cycle+start+examined+at+single-cell+resolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coherence+and+timing+of+cell+cycle+start+examined+at+single-cell+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coherence+and+timing+of+cell+cycle+start+examined+at+single-cell+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001468-deBruin1" id="pone.0001468-deBruin1"></a>de Bruin RA, McDonald WH, Kalashnikova TI, Yates J 3rd, Wittenberg C (2004) Cln3 activates G1-specific transcription via phosphorylation of the SBF bound repressor Whi5. Cell  117(7): 887–898.  <ul class="find" data-citedArticleID="1044910" data-doi="10.1016/j.cell.2004.05.025"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.05.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cln3+activates+G1-specific+transcription+via+phosphorylation+of+the+SBF+bound+repressor+Whi5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cln3+activates+G1-specific+transcription+via+phosphorylation+of+the+SBF+bound+repressor+Whi5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001468-Johnston1" id="pone.0001468-Johnston1"></a>Johnston GC, Pringle JR, Hartwell LH (1977) Coordination of growth with cell division in the yeast saccharomyces cerevisiae. Exp Cell Res.  105(1): 79–98.  <ul class="find" data-citedArticleID="1044878" data-doi="10.1016/0014-4827(77)90154-9"><li><a href="http://dx.doi.org/10.1016/0014-4827(77)90154-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coordination+of+growth+with+cell+division+in+the+yeast+saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coordination+of+growth+with+cell+division+in+the+yeast+saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001468-Louvion1" id="pone.0001468-Louvion1"></a>Louvion JF, Havaux-Copf B, Picard D (1993) Fusion of GAL4-VP16 to a steroid-binding domain provides a tool for gratuitous induction of galactose-responsive genes in yeast. Gene  131: 129–134.  <ul class="find" data-citedArticleID="1044886" data-doi="10.1016/0378-1119(93)90681-r"><li><a href="http://dx.doi.org/10.1016/0378-1119(93)90681-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fusion+of+GAL4-VP16+to+a+steroid-binding+domain+provides+a+tool+for+gratuitous+induction+of+galactose-responsive+genes+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fusion+of+GAL4-VP16+to+a+steroid-binding+domain+provides+a+tool+for+gratuitous+induction+of+galactose-responsive+genes+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001468-Cross1" id="pone.0001468-Cross1"></a>Cross FR, Siggia ED (2005) Mode locking the cell cycle. PRE  72(2 Pt 1): 021910.  <ul class="find" data-citedArticleID="1044870" data-doi="10.1103/physreve.72.021910"><li><a href="http://dx.doi.org/10.1103/physreve.72.021910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mode+locking+the+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mode+locking+the+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001468-Battogtokh1" id="pone.0001468-Battogtokh1"></a>Battogtokh D, Tyson JJ (2006) Periodic forcing of a mathematical model of the eukaryotic cell cycle. PRE  73(1 Pt 1): 011910.  <ul class="find" data-citedArticleID="1044860" data-doi="10.1103/physreve.73.011910"><li><a href="http://dx.doi.org/10.1103/physreve.73.011910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Periodic+forcing+of+a+mathematical+model+of+the+eukaryotic+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Periodic+forcing+of+a+mathematical+model+of+the+eukaryotic+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001468-Tsien1" id="pone.0001468-Tsien1"></a>Tsien RY (1998) The green fluorescent protein. Annu Rev Biochem.  67: 509–44.  <ul class="find" data-citedArticleID="1044908" data-doi="10.1146/annurev.biochem.67.1.509"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.67.1.509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+green+fluorescent+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+green+fluorescent+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001468-Spellman1" id="pone.0001468-Spellman1"></a>Spellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, et al.  (1998) Comprehensive identification of cell cycle-regulated genes of the yeast S<em>accharomyces Cerevisiae</em> by microarray hybridization. Mol. Biol. Cell  9(12): 3273–97.  <ul class="find" data-citedArticleID="1044904" data-doi="10.1091/mbc.9.12.3273"><li><a href="http://dx.doi.org/10.1091/mbc.9.12.3273" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+identification+of+cell+cycle-regulated+genes+of+the+yeast+Saccharomyces+Cerevisiae+by+microarray+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+identification+of+cell+cycle-regulated+genes+of+the+yeast+Saccharomyces+Cerevisiae+by+microarray+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.XML" value="91136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.PDF" value="783469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g001.PNG_L" value="3172026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g001.PNG_M" value="270284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g001.PNG_S" value="12237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g001.TIF" value="5335162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g001.PNG_I" value="93346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g002.PNG_L" value="3572407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g002.PNG_M" value="194591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g002.PNG_S" value="16524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g002.TIF" value="5635464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g002.PNG_I" value="131280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g003.PNG_L" value="2255893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g003.PNG_M" value="164054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g003.PNG_S" value="10139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g003.TIF" value="3579032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g003.PNG_I" value="54349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g004.PNG_L" value="3260202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g004.PNG_M" value="216946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g004.PNG_S" value="12963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g004.TIF" value="5029394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g004.PNG_I" value="99944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g005.PNG_L" value="1177263"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g005.PNG_M" value="146598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g005.PNG_S" value="11747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g005.TIF" value="4023434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.g005.PNG_I" value="62923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s001.PDF" value="54152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s002.PDF" value="883314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s003.PDF" value="835235"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s004.PDF" value="104679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s005.PDF" value="308700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s006.PDF" value="29664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s007.PDF" value="36898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s008.DOC" value="2650432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s009.AVI" value="6580224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s010.AVI" value="12306432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s011.AVI" value="14091264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001468.s012.AVI" value="11415552"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001468&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001468" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001468&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001468&volume=&issue=&title=A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells&author_name=Gilles%20Charvin%2C%20Frederick%20R.%20Cross%2C%20Eric%20D.%20Siggia&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001468" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468&amp;t=A%20Microfluidic%20Device%20for%20Temporally%20Controlled%20Gene%20Expression%20and%20Long-Term%20Fluorescent%20Imaging%20in%20Unperturbed%20Dividing%20Yeast%20Cells" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468&title=A%20Microfluidic%20Device%20for%20Temporally%20Controlled%20Gene%20Expression%20and%20Long-Term%20Fluorescent%20Imaging%20in%20Unperturbed%20Dividing%20Yeast%20Cells&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468&amp;title=A%20Microfluidic%20Device%20for%20Temporally%20Controlled%20Gene%20Expression%20and%20Long-Term%20Fluorescent%20Imaging%20in%20Unperturbed%20Dividing%20Yeast%20Cells" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001468&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Microfluidic Device for Temporally Controlled Gene Expression and Long-Term Fluorescent Imaging in Unperturbed Dividing Yeast Cells';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001468';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Microfluidic%20Device%20for%20Temporally%20Controlled%20Gene%20Expression%20and%20Long-Term%20Fluorescent%20Imaging%20in%20Unperturbed%20Dividing%20Yeast%20Cells http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001468" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001468" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->

<div class="block">
    <div class="header">
        <h3>Included in the Following Collection</h3>
    </div>
    <p><dd><a href="http://www.ploscollections.org/article/browse/issue/info%3Adoi%2F10.1371%2Fissue.pcol.v02.i35" title="Browse the Open-Access Collection">Instruments</a></dd></p>
</div>

<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="25764"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+membranes%22" title="Search for articles in the subject area:'Cell membranes'"><div class="flagText">Cell membranes</div></a>
              <div data-categoryid="17599" data-articleid="25764"
                   data-categoryname="Cell membranes"
                   class="flagImage" title="Flag 'Cell membranes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="25764"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fluorescence+imaging%22" title="Search for articles in the subject area:'Fluorescence imaging'"><div class="flagText">Fluorescence imaging</div></a>
              <div data-categoryid="17303" data-articleid="25764"
                   data-categoryname="Fluorescence imaging"
                   class="flagImage" title="Flag 'Fluorescence imaging' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25764"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Membrane+dialysis%22" title="Search for articles in the subject area:'Membrane dialysis'"><div class="flagText">Membrane dialysis</div></a>
              <div data-categoryid="22989" data-articleid="25764"
                   data-categoryname="Membrane dialysis"
                   class="flagImage" title="Flag 'Membrane dialysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microfluidics%22" title="Search for articles in the subject area:'Microfluidics'"><div class="flagText">Microfluidics</div></a>
              <div data-categoryid="17537" data-articleid="25764"
                   data-categoryname="Microfluidics"
                   class="flagImage" title="Flag 'Microfluidics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Organism+development%22" title="Search for articles in the subject area:'Organism development'"><div class="flagText">Organism development</div></a>
              <div data-categoryid="46207" data-articleid="25764"
                   data-categoryname="Organism development"
                   class="flagImage" title="Flag 'Organism development' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8950'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=677'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4084&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9269">added citation to Groisman et al. paper - See commentary for details</a><br>Posted by gcharvin</p>
      <p><a href="/annotation/listThread.action?root=18513">microfluidic yeast cultures</a><br>Posted by microflu</p>
      <p><a href="/annotation/listThread.action?root=5209">Text S1 file</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
