

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000454"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000454" />

    <meta name="citation_title" content="Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity"/>
    <meta itemprop="name" content="Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity"/>

      <meta name="citation_author" content="Yoshan Moodley"/>
            <meta name="citation_author_institution" content="School of Biosciences, Cardiff University, Cardiff, United Kingdom"/>
      <meta name="citation_author" content="Michael W. Bruford"/>
            <meta name="citation_author_institution" content="School of Biosciences, Cardiff University, Cardiff, United Kingdom"/>

    <meta name="citation_date" content="2007/5/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000454.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e454"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The Global 200: a representation approach to conserving the Earth's distinctive ecoregions.; citation_author=DM Olson; citation_author=E Dinerstein; citation_journal_title=Conserv Biol; citation_volume=12; citation_number=1; citation_pages=502-515; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Changing places: how the results of new field work in the Rubeho Mountains influence conservation priorities in the Eastern Arc Mountains of Tanzania.; citation_author=N Doggart; citation_author=A Perkin; citation_author=J Kiure; citation_author=J Fjeldsa; citation_author=J Poyntin; citation_journal_title=Afr J Ecol; citation_volume=44; citation_number=2; citation_pages=134-144; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=MDF Udvardy; citation_number=3; citation_date=1975; citation_publisher=IUCN Occasional Paper 18; " />
      <meta name="citation_reference" content="citation_author=HA Gleason; citation_author=A Cronquist; citation_number=4; citation_pages=420; citation_date=1964; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_author=R Good; citation_number=5; citation_pages=518; citation_date=1964; citation_publisher=Longmans; " />
      <meta name="citation_reference" content="citation_title=On the mammals and winter birds of east Florida, with an examination ofcertain assumed specific characters in birds and a sketch of the bird faunae of eastern North America.; citation_author=JA Allen; citation_journal_title=Bull Mus Comp Zool; citation_volume=2; citation_number=6; citation_pages=161-450; citation_date=1871; " />
      <meta name="citation_reference" content="citation_title=Terrestrial ecoregions of the world: a new map of life on earth.; citation_author=DM Olson; citation_author=E Dinerstein; citation_author=ED Wikramanayake; citation_author=ND Burgess; citation_author=GVN Powell; citation_journal_title=BioScience; citation_volume=51; citation_number=7; citation_pages=933-937; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Endemic ranid (Amphibia : Anura) genera in southern mountain ranges of the Indian subcontinent represent ancient frog lineages: evidence from molecular data.; citation_author=K Roelants; citation_author=JP Jiang; citation_author=F Bossuyt; citation_journal_title=Mol Phyl Evol; citation_volume=31; citation_number=8; citation_pages=730-740; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Historical climate change and speciation: neotropical seasonally dry forest plants show patterns of both tertiary and quaternary diversification.; citation_author=RT Pennington; citation_author=M Lavin; citation_author=DE Prado; citation_author=CA Pendry; citation_author=SK Pell; citation_journal_title=Philos Trans R Soc London B; citation_volume=359; citation_number=9; citation_pages=515-537; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Conservation Biogeography: assessment and prospect.; citation_author=RJ Whittaker; citation_author=MB Araujo; citation_author=J Paul; citation_author=RJ Ladle; citation_author=JEM Watson; citation_journal_title=Divers Distrib; citation_volume=11; citation_number=10; citation_pages=3-23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=J Kingdon; citation_number=11; citation_pages=476; citation_date=1997; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_author=R East; citation_number=12; citation_pages=116-122; citation_date=1999; citation_publisher=IUCN/SSC Antelope Specialist Group; " />
      <meta name="citation_reference" content="citation_author=T Haltenorth; citation_author=H Diller; citation_number=13; citation_pages=57-58; citation_date=1980; citation_publisher=Collins; " />
      <meta name="citation_reference" content="citation_author=WFH Ansell; citation_volume=15; citation_number=14; citation_pages=1-84; citation_date=1971; citation_publisher=Smithsonian Institution Press; " />
      <meta name="citation_reference" content="citation_title=Geographical variation in the bushbuck of eastern Africa (Tragelaphus scriptus; Bovidae).; citation_author=P Grubb; citation_number=15; citation_pages=11-26; citation_date=1985; citation_publisher=Museum A König; " />
      <meta name="citation_reference" content="citation_author=R Lydekker; citation_number=16; citation_pages=317-326; citation_date=1914; citation_publisher=British Museum; " />
      <meta name="citation_reference" content="citation_title=A checklist of African mammals.; citation_author=GM Allen; citation_journal_title=Bull Mus Comp Zool Harvard; citation_volume=83; citation_number=17; citation_pages=1-763; citation_date=1939; " />
      <meta name="citation_reference" content="citation_title=Phylogeography of three closely related African bovids (Tribe Alcelaphini).; citation_author=P Arctander; citation_author=C Johansen; citation_author=M-A Coutellec-Vret; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=18; citation_pages=1724-1739; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Environmental change and rates of evolution: the phylogeographic pattern within the hartebeest complex as related to climatic variation.; citation_author=Ø Flagstad; citation_author=PO Syvertsen; citation_author=NC Stenseth; citation_author=KS Jakobsen; citation_journal_title=Philos Trans R Soc London B; citation_volume=268; citation_number=19; citation_pages=667-677; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Phylogeography and population structure of the common warthog (Phacochoerus africanus) inferred from variation in mitochondrial DNA sequences and microsatellite loci.; citation_author=VB Muwanika; citation_author=S Nyakaana; citation_author=HR Siegismund; citation_author=P Arctander; citation_journal_title=Heredity; citation_volume=91; citation_number=20; citation_pages=361-372; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Status of the so-called African pygmy elephant (Loxodonta pumilio (Noack 1906)): phylogeny of cytochrome b and mitochondrial control region sequences.; citation_author=R Debruyne; citation_author=A Van Holt; citation_author=V Barriel; citation_author=P Tassy; citation_journal_title=C R Biol; citation_volume=326; citation_number=21; citation_pages=687-697; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogeography and conservation of impala and greater kudu.; citation_author=LG Nersting; citation_author=P Arctander; citation_journal_title=Mol Ecol; citation_volume=10; citation_number=22; citation_pages=711-719; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Regional genetic structuring and evolutionary history of the impala Aepyceros melampus.; citation_author=ED Lorenzen; citation_author=P Arctander; citation_author=HR Siegismund; citation_journal_title=J Hered; citation_volume=97; citation_number=23; citation_pages=119-132; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Estimating the rate of evolution of the rate of molecular evolution.; citation_author=JL Thorne; citation_author=H Kishino; citation_author=JS Painter; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=24; citation_pages=1647-1657; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Environment and Evolution: alternative causes of the temporal distribution of evolutionary events.; citation_author=ES Vrba; citation_journal_title=S Afr J Sci; citation_volume=81; citation_number=25; citation_pages=229-236; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Late Pliocene faunal turnover in the Turkana Basin, Kenya and Ethiopia.; citation_author=AK Behrensmeyer; citation_author=NE Todd; citation_author=R Potts; citation_author=GE McBrinn; citation_journal_title=Science; citation_volume=278; citation_number=26; citation_pages=1589-1594; citation_date=1997; " />
      <meta name="citation_reference" content="citation_author=GA Best; citation_number=27; citation_pages=198-208; citation_date=1962; citation_publisher=Rowland Ward Ltd; " />
      <meta name="citation_reference" content="citation_author=MG Leakey; citation_author=JM Harris; citation_number=28; citation_pages=678; citation_date=2003; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_title=Stratigraphy and paleontology of Pliocene and Pleistocene localities west of Lake Turkana, Kenya.; citation_author=JM Harris; citation_author=FH Brown; citation_author=MG Leakey; citation_journal_title=Los Angeles County Mus Nat Hist Contr Sci; citation_volume=399; citation_number=29; citation_pages=1-128; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Geology and stratigraphy of Neogene deposits, Middle Awash Valley, Ethiopia.; citation_author=JE Kalb; citation_author=EB Oswald; citation_author=S Tebedge; citation_author=A Mebrate; citation_author=E Tola; citation_journal_title=Nature; citation_volume=298; citation_number=30; citation_pages=98-106; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Uplift of the roof of Africa and its bearing on the evolution of Mankind.; citation_author=M Pickford; citation_journal_title=Hum Evol; citation_volume=5; citation_number=31; citation_pages=1-20; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=The influence of global climatic change and regional uplift on large-mammalian evolution in East and Southern Africa.; citation_author=TC Partridge; citation_author=B Wood; citation_author=PB deMenocal; citation_number=32; citation_pages=330-355; citation_date=1995; citation_publisher=Yale Univ Press; " />
      <meta name="citation_reference" content="citation_title=Early hominid evolution and ecological change through the African Plio-Pleistocene.; citation_author=KE Reed; citation_journal_title=J Hum Evol; citation_volume=32; citation_number=33; citation_pages=289-322; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Responses of African bovids to Pliocene climatic change.; citation_author=R Bobe; citation_author=GG Eck; citation_journal_title=Paleobiology; citation_volume=27; citation_number=34; citation_pages=1-47; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Plio-Pleistocene climatic change in the Turkana Basin (East Africa): Evidence from large mammal faunas.; citation_author=M Hernandez Fernandez; citation_author=ES Vrba; citation_journal_title=J Hum Evol; citation_volume=50; citation_number=35; citation_pages=595-626; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The structure of biodiversity - insights from molecular phylogeography.; citation_author=GM Hewitt; citation_journal_title=Front Zool; citation_volume=1; citation_number=36; citation_pages=4; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Klassifikation der Säugetiere: Artiodactyla 1.; citation_author=H Haltenorth T; citation_journal_title=Handbuch der Zoologie; citation_volume=8; citation_number=37; citation_pages=1-167; citation_date=1963; " />
      <meta name="citation_reference" content="citation_author=J Dorst; citation_author=P Dandelot; citation_number=38; citation_pages=252; citation_date=1970; citation_publisher=Collins; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=39; citation_pages=9.16-9.19; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Cetacean mitochondrial DNA control region: Sequences of all extant baleen whales and two sperm whale species.; citation_author=U Arnason; citation_author=A Gullberg; citation_author=B Widegren; citation_journal_title=Mol Biol Evol; citation_volume=10; citation_number=40; citation_pages=960-970; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Population structure of African buffalo inferred from mtDNA sequences and microsatellite loci: high variation but low differentiation.; citation_author=BT Simonsen; citation_author=HR Siegismund; citation_author=P Arctander; citation_journal_title=Mol Ecol; citation_volume=7; citation_number=41; citation_pages=225-237; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Large sequence divergence of mitochondrial DNA genotypes of the control region within populations of the African antelope, kob (Kobus kob).; citation_author=J Birungi; citation_author=P Arctander; citation_journal_title=Mol Ecol; citation_volume=9; citation_number=42; citation_pages=1997-2008; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Polymerase Chain Reaction reveals cloning artifacts.; citation_author=S Paabo; citation_author=AC Wilson; citation_journal_title=Nature; citation_volume=334; citation_number=43; citation_pages=387-388; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Median-joining networks for inferring intraspecific phylogenies.; citation_author=HJ Bandelt; citation_author=P Forster; citation_author=A Röhl; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=44; citation_pages=37-48; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Dating of the human-ape splitting by a molecular clock of mitochondrial DNA.; citation_author=M Hasegawa; citation_author=H Kishino; citation_author=K Yano; citation_journal_title=J Mol Evol; citation_volume=22; citation_number=45; citation_pages=160-174; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Modeltest: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=46; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=TREEFINDER: A powerful graphical analysis environment for molecular phylogenetics.; citation_author=G Jobb; citation_author=A von Haeseler; citation_author=K Strimmer; citation_journal_title=BMC Evol Biol.; citation_volume=4; citation_number=47; citation_pages=18; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bayesian phylogenetic inference using DNA sequences: a Markov chain Monte Carlo method.; citation_author=Z Yang; citation_author=B Rannala; citation_journal_title=Mol Biol Evol; citation_volume=14; citation_number=48; citation_pages=717-724; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Performance of a divergence time estimation method under a probabilistic model of rate evolution.; citation_author=H Kishino; citation_author=JL Thorne; citation_author=WJ Bruno; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=49; citation_pages=352-361; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data.; citation_author=L Excoffier; citation_author=PE Smouse; citation_author=JM Quattro; citation_journal_title=Genetics; citation_volume=131; citation_number=50; citation_pages=479-491; citation_date=1992; " />
      <meta name="citation_reference" content="citation_author=MJ Anderson; citation_number=51; citation_date=2004; citation_publisher=Department of Statistics, University of Auckland, New Zealand; " />
      <meta name="citation_reference" content="citation_title=Ecological factors influence population genetic structure of European grey wolves.; citation_author=M Pilot; citation_author=W Jedrzejewski; citation_author=W Branicki; citation_author=VE Sidorovich; citation_author=B Jedrzejewska; citation_journal_title=Mol Ecol; citation_volume=15; citation_number=52; citation_pages=4533-4553; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa.; citation_author=S Ramachandran; citation_author=O Deshpande; citation_author=CC Roseman; citation_author=NA Rosenberg; citation_author=MW Feldman; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=53; citation_pages=15942-15947; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=MJ Anderson; citation_number=54; citation_date=2003; citation_publisher=Department of Statistics, University of Auckland, New Zealand; " />
      <meta name="citation_reference" content="citation_title=Trends, rhythms, and aberrations in global climate 65 Ma to present.; citation_author=JC Zachos; citation_author=M Pagani; citation_author=L Sloan; citation_author=E Thomas; citation_author=K Billups; citation_journal_title=Science; citation_volume=292; citation_number=55; citation_pages=686-693; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity"/>
    <meta name="twitter:description" content="BackgroundBiogeographic models partition ecologically similar species assemblages into discrete ecoregions. However, the history, relationship and interactions between these regions and their assemblages have rarely been explored.Methodology/Principal FindingsHere we develop a taxon-based approach that explicitly utilises molecular information to compare ecoregion history and status, which we exemplify using a continentally distributed mammalian species: the African bushbuck (Tragelaphus scriptus). We reveal unprecedented levels of genetic diversity and structure in this species and show that ecoregion biogeographic history better explains the distribution of molecular variation than phenotypic similarity or geography. We extend these data to explore ecoregion connectivity, identify core habitats and infer ecological affinities from them.Conclusions/SignificanceThis analysis defines 28 key biogeographic regions for sub-Saharan Africa, and provides a valuable framework for the incorporation of genetic and biogeographic information into a more widely applicable model for the conservation of continental biodiversity."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000454.g004"/>

  <meta property="og:title" content="Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4526'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6675'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2406&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000454">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000454" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000454&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshan Moodley
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:moodley@mpiib-berlin.mpg.de">moodley@mpiib-berlin.mpg.de</a></p>

                <p>Affiliation:
                  School of Biosciences, Cardiff University, Cardiff, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael W. Bruford
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Biosciences, Cardiff University, Cardiff, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 23, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000454</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000454-g001" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000454-g002" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000454-g003" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000454-t001" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000454-g004" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000454">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000454" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Biogeographic models partition ecologically similar species assemblages into discrete ecoregions. However, the history, relationship and interactions between these regions and their assemblages have rarely been explored.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we develop a taxon-based approach that explicitly utilises molecular information to compare ecoregion history and status, which we exemplify using a continentally distributed mammalian species: the African bushbuck (<em>Tragelaphus scriptus</em>). We reveal unprecedented levels of genetic diversity and structure in this species and show that ecoregion biogeographic history better explains the distribution of molecular variation than phenotypic similarity or geography. We extend these data to explore ecoregion connectivity, identify core habitats and infer ecological affinities from them.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This analysis defines 28 key biogeographic regions for sub-Saharan Africa, and provides a valuable framework for the incorporation of genetic and biogeographic information into a more widely applicable model for the conservation of continental biodiversity.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Moodley Y, Bruford MW (2007) Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity. PLoS ONE 2(5):
          e454.
            doi:10.1371/journal.pone.0000454</p><p><strong>Academic Editor: </strong>Björn Brembs, Freie Universitaet Berlin, Germany</p><p><strong>Received:</strong> December 18, 2006; <strong>Accepted:</strong> April 24, 2007; <strong>Published:</strong> May 23, 2007</p><p><strong>Copyright:</strong> © 2007 Moodley, Bruford. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a postdoctoral fellowship from the National Research Foundation of the Republic of South Africa and by Cardiff University. Neither had a role in the preparation of the manuscript nor in the design, conduct or analysis of the study.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Continental-scale biogeographic models capture and incorporate the complexity of natural ecosystems by partitioning this variation into provinces or ‘ecoregions’ which can be used as manageable foci for the conservation of biodiversity <a href="#pone.0000454-Olson1">[1]</a>, <a href="#pone.0000454-Doggart1">[2]</a>. The Afrotropical Biogeographic Realm <a href="#pone.0000454-Udvardy1">[3]</a> contains one of the highest levels of biodiversity on Earth. This model divides continental Africa into a series of biogeographical provinces on the basis of ecosystematic or biotic features. Biogeographic provinces can be translated loosely into floral <a href="#pone.0000454-Gleason1">[4]</a>, <a href="#pone.0000454-Good1">[5]</a> and eco-faunal regions <a href="#pone.0000454-Allen1">[6]</a>. A recent analysis of the Afrotropical Biogeographic Realm <a href="#pone.0000454-Olson2">[7]</a> accounted for intra-province variation and defined smaller ‘ecoregions’ with greater ecological specificity, such as seasonal floodplains and mangrove swamps. Besides helping to enhance and focus the management of conservation efforts <a href="#pone.0000454-Roelants1">[eg. 8]</a> biogeographic models can hint at recent evolutionary processes <a href="#pone.0000454-Pennington1">[9]</a> that have given rise to the faunal and floral assemblages associated with each ecoregion. However, species assemblages may differ extensively between different ecoregions due to unequal taxonomic effort and scale effects <a href="#pone.0000454-Whittaker1">[10]</a> such that only contiguous and/or very similar ecoregions can be readily compared, which hampers objective estimates of alpha and gamma diversity. A further problem is that research has ignored taxa with wide-scale evolutionary histories, including adaptation within a large range of key ecoregions, in favour of specialists, which are often of greater conservation concern and serve as examples of singular threatened ecoregions.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, a number of potential species are sufficiently widespread and adapted to biogeographic provinces at a continental scale to be potential models. One example in Africa is the bushbuck (<em>Tragelaphus scriptus</em>): a mammalian generalist and herbivore that has kept pace with environmental changes by local adaptation to changing habitats, since this sedentary species appears to require only water, cover and the availability of grazing or browse <a href="#pone.0000454-Kingdon1">[11]</a>. As a consequence, <em>T. scriptus</em> is Africa's most widely distributed ungulate, occurring in every country in sub-Saharan Africa with its range limited only by the lowland closed canopy forests of the Central Congo Basin, deserts and highly xeric shrublands. Hence, bushbuck remain common in Equatorial and Guinean lowland forests <a href="#pone.0000454-East1">[12]</a> while inhabiting montane forests across the continent at elevations up to 4000 m <a href="#pone.0000454-Haltenorth1">[13]</a> and even penetrating into the xeric Sahel and Somali zones along seasonal watercourses <a href="#pone.0000454-Kingdon1">[11]</a>, <a href="#pone.0000454-Ansell1">[14]</a>. The bushbuck inhabits 17 of the 19 terrestrial Sub-Saharan biogeographical provinces in Udvardy's <a href="#pone.0000454-Udvardy1">[3]</a> model and 62 of the 91 ecoregions in the extended Olson <em>et al.</em> <a href="#pone.0000454-Olson2">[7]</a> model. This equates to approximately 73% of the total land area of Sub-Saharan Africa.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Local adaptation across this vast and heterogeneous range has resulted in marked geographic variation in body and horn size, coat length and pattern, colouration and sexual dimorphism <a href="#pone.0000454-Ansell1">[14]</a>, <a href="#pone.0000454-Grubb1">[15]</a>. Over 40 subspecies have been described but systematic studies indicate that between 24 <a href="#pone.0000454-Lydekker1">[16]</a>, <a href="#pone.0000454-Allen2">[17]</a> and six <a href="#pone.0000454-Grubb1">[15]</a> distinct forms may exist. The structure of mitochondrial genetic diversity in grassland <a href="#pone.0000454-Arctander1">[18]</a>–<a href="#pone.0000454-Muwanika1">[20]</a>, forest <a href="#pone.0000454-Debruyne1">[21]</a> and arid-adapted African species <a href="#pone.0000454-Nersting1">[22]</a>, <a href="#pone.0000454-Lorenzen1">[23]</a> has previously been analysed in a phylogeographic context. These studies provide essential information on evolutionary history with respect to the paleohistories of discrete, ecologically homogeneous regions, but do not reveal how these distinct ecological regions are linked. The bushbuck provides an opportunity for establishing such links due to its unsurpassed phenotypic diversity, comprising forest, savanna, woodland, montane and arid-adapted forms.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Using this model taxon approach to continental biogeography provides a quantitative framework for the integration of traditional approaches with ecological data, because it partitions diversity into regionally meaningful components and links biogeographic ecoregions with the evolutionary history of ubiquitous species. Novel inferences about ecoregional connectivity, core habitats, ecological affinities and adaptation then become possible and regions of core evolutionary and ecological importance can be identified. This approach requires extensive sampling to validate the resulting inferences. In this study we assessed the evolutionary history of the bushbuck from an unprecedented, continent-wide sample using mitochondrial DNA control region sequences and a complementary data set for the cytochrome <em>b</em> gene. We analyse genetic structure in the context of biogeographic history, phenotype and geography, and using a new approach to combine genetic and ecological data, we elaborated a taxon-linked model for pan-African molecular biogeography.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We examined 516 bp of sequence from the 5′ end of the <em>Tragelaphus scriptus</em> mitochondrial control region (CR) in 485 specimens covering the entire species range and accounting for all known phenotypic variation (<a href="#pone.0000454.s001">Table S1</a>). We also examined 556 bp of the mitochondrial cytochrome <em>b</em> gene (cyt <em>b</em>) from a sub-sample of 161 specimens. 259 sites (50.2%) were polymorphic for the CR and 159 sites (28.6%) for cyt <em>b</em>, yielding 320 and 90 unique haplotypes, respectively. Nucleotide diversity (<em>π</em>) for both sequences was very high (<em>π</em>CR: 11.7%, <em>π</em>cyt <em>b</em>: 7.4%).</p>

<h4>Genetic Structure</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Genetic structure was assessed by a median-joining network for CR haplotypes (<a href="#pone-0000454-g001">Fig. 1</a>) and by maximum likelihood (ML) for the cyt <em>b</em> data set (<a href="#pone-0000454-g002">Fig. 2</a>). The results show that all genetic variation was partitioned into 2 basal haplogroups which divide sub-Saharan Africa roughly along the axis of the Rift Valley into Western-Northern and Eastern-Southern sectors (<a href="#pone-0000454-g001">Fig. 1</a>); hereafter referred to as the <em>Scriptus</em> [Pallas 1766] and <em>Sylvaticus</em> [Sparrman 1780] groups respectively, in accordance with the earliest described taxon from each group. Both groups possessed high nucleotide diversity (<em>Sylvaticus</em>: <em>π</em>CR 6.2%, <em>π</em>cyt <em>b</em> 3.5%; <em>Scriptus</em>: <em>π</em>CR 5.2%, <em>π</em>cyt <em>b</em> 3.6%). Additional substructure was extremely high, with high bootstrap support (<a href="#pone-0000454-g002">Fig. 2</a>) for 23 terminal haplogroups with distinct geographic ranges (<a href="#pone-0000454-g001">Fig. 1</a>). The cyt <em>b</em> data resolved the branching order of intermediate-level nodes more efficiently, partitioning the <em>Scriptus</em> group into three intermediate and eight terminal level haplogroups and the <em>Sylvaticus</em> group into seven and 15 intermediate and terminal haplogroups, respectively (<a href="#pone-0000454-g002">Fig. 2</a>, <a href="#pone.0000454.s002">Table S2</a>). All 23 terminal haplogroups were monophyletic, however four (labelled ‘<em>bor</em>’, ‘<em>phaleratus</em>’, ‘<em>dianae</em>’ and ‘<em>massaicus</em>’) were not reciprocally monophyletic relative to haplogroups nested within them. The division of the major <em>Scriptus</em> and <em>Sylvaticus</em> groups (picture panel, <a href="#pone-0000454-g002">Fig. 2</a>) corresponded well with phenotype, showing marked (<em>scriptus</em>) and moderate (<em>ornatus</em>) white patterning in the phylogenetically basal haplogroups of both groups and a general loss of patterning in more derived forms. Evolution of darker, thicker coats and larger size within the <em>Sylvaticus</em> group appears to have arisen independently on the montane inselbergs of East Africa. Due to the high geographic structure in the data (<a href="#pone-0000454-g001">Fig. 1</a>), discrete population units could be inferred directly from the terminal haplogroups (<a href="#pone-0000454-g002">Fig. 2</a>). Previously described subspecies provide convenient (although not necessarily biologically meaningful) labels for most terminal haplogroups, however in six instances the molecular data support non-taxonomic groupings; namely in Angola, the Middle Zambezi Valley, the Luangwa Valley, Upper Volta, Lower Volta and Niger (<a href="#pone.0000454.s002">Table S2</a>). A relaxed Bayesian molecular clock <a href="#pone.0000454-Thorne1">[24]</a> dated the coalescence of the cyt <em>b</em> lineages to the late Neogene, 4.8 Mya (95% CI: 3.9–6.5 Mya). While diversification into the major <em>Sylvaticus</em> and <em>Scriptus</em> groups was dated to the late Pliocene at 2.7 and 2.5 Mya respectively (95% CL: 3.0–2.0 Mya), during a time of considerable faunal turnover in Africa <a href="#pone.0000454-Vrba1">[25]</a>, <a href="#pone.0000454-Behrensmeyer1">[26]</a>, individual haplogroups date to the Pleistocene between 1.7 and 0.067 Mya (<a href="#pone-0000454-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0000454-g001"><div class="img"><a name="pone-0000454-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Pan-African relationships among 23 bushbuck haplogroups in relation to toplogy and habitat heterogeneity.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Part A. An unrooted median joining network of 485 control region sequences, overlaid onto a topographical map of Africa. 320 haplotypes are arranged according to original sampling location, thus describing the approximate geographic range of each terminal haplogroup. The size of each circle is scaled to haplotype frequency. A dashed white line separates <em>Scriptus</em> and <em>Sylvaticus</em> haplotypes. Relationships between intermediate-level haplogroups (thick red links) were unresolved by CR data and were inferred from a cytochrome <em>b</em> network (data not shown). Thick black links join terminal haplogroups nested within intermediate haplogroups and thin black links join individual haplotypes within terminal haplogroups. Inset panels B and C show the comparitive distribution of biogeographic provinces <a href="#pone.0000454-Udvardy1">[3]</a> and ecoregions <a href="#pone.0000454-Olson2">[7]</a> respectively.</p>
<span>doi:10.1371/journal.pone.0000454.g001</span></div><div class="figure" id="pone-0000454-g002"><div class="img"><a name="pone-0000454-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Linearised phylogenetic tree for 90 cytochrome <em>b</em> haplotypes obtained by maximum likelihood assuming an HKY model for nucleotide substitution.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Nodal support for haplogroups is displayed and nodes with less than 50% support were collapsed. Haplogroups are determined by monophyly and distribution. Haplogroups are colour coded as in <a href="#pone-0000454-g001">Figure 1</a>. The picture panel shows the general phenotype of each intermediate-level haplogroup. The <em>decula</em> terminal haplogroup is depicted to show the loss of patterning in this subspecies. The bottom panel displays the timing of evolutionary events with equivalent changes in the global temperature (as measured by changes in the oxygen isotope 18) adapted from <a href="#pone.0000454-Zachos1">[55]</a>.</p>
<span>doi:10.1371/journal.pone.0000454.g002</span></div><div class="figure" id="pone-0000454-g003"><div class="img"><a name="pone-0000454-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Ecoregional habitation of 23 bushbuck haplogroups, the inference of ecological affinities and the identification of key biogeographic regions as defined by molecular data.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Ecological affinity was inferred from a haplogroup's index of specialisation (IS); Arrows (red for the <em>Scriptus</em> group and blue for the <em>Sylvaticus</em> group) summarise the core ecoregions of each haplogroup into their appropriate core biomes (shaded), which were then translated into key biogeographic regions by the inclusion of geography. Haplogroups are colour-coded as in <a href="#pone-0000454-g001">Figure 1</a>.</p>
<span>doi:10.1371/journal.pone.0000454.g003</span></div>

<h4>Phenotype models</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Partitioning all sequences into their respective terminal haplogroups showed that only 9% (AMOVA) to 14% (multivariate matrix regression, MMR) of the variation in the CR data occurs at the within-population level (<a href="#pone-0000454-t001">Table 1</a>). Of the six taxonomic hypotheses that attempt to partition bushbuck on the basis of phenotype, the 24-group Lydekker <a href="#pone.0000454-Lydekker1">[16]</a> model and the 10-group Grubb-Best <a href="#pone.0000454-Grubb1">[15]</a>, <a href="#pone.0000454-Best1">[27]</a> combined model provided the best congruence between genetic divergence and phenotype. Despite this, both models still only explained 64% and 63% of the variation in the data respectively (<a href="#pone-0000454-t001">Table 1</a>). The better performance of Lydekker's older model challenges the wisdom of later attempts to obtain a manageable overview of bushbuck variation by synonymising phenotypically variable subspecies on the basis of geographical proximity. Even the one-group subspecies comparison, which provides a measure of overall structure in the data relative to subspecies definitions, showed that between 29% (AMOVA) and 31% (MMR) of the molecular variance in the data was found within-populations, highlighting a poor molecular correlation with the present taxonomic designations in bushbuck.</p>
<div class="figure" id="pone-0000454-t001"><div class="img"><a name="pone-0000454-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000454.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000454.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Model testing by analyses of molecular variance (AMOVA) and multivariate matrix regressions (MMR).</span></strong></p><span>doi:10.1371/journal.pone.0000454.t001</span></div>

<h4>Biogeographic models</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Of the two biogeographic models tested, the Udvardy model accounted for 59% (AMOVA) to 48% (MMR) of CR variation in 17 biogeographic provinces (<a href="#pone-0000454-t001">Table 1</a>). However, the Olson model which includes 62 Afrotropical ecoregions described 76–77% of the variation in the CR data, significantly greater even than the 69–71% accounted by taxonomic designation. Of the 50 ecoregions in our data set, 27 are inhabited by only a single haplogroup (<a href="#pone.0000454.s003">Table S3</a>). The remaining 23 ecoregions were inhabited by individuals belonging to more than one haplogroup and hence comprises the 15% difference in explanatory molecular variance between the Olson model and the CR data. These 23 ‘shared ecoregions’ could be partitioned into those where haplogroup ranges were mutually exclusive and those where they overlapped. The six shared ecoregions with mutually exclusive haplogroup distributions occurred as a result of a) broadly defined ecoregions where haplogroups are separated either by distance (Guinean forest-savanna mosaic) or unfavourable habitat (Sahelian Acacia savanna, East Sudanian savanna); b) separation by geomorphological barriers such as escarpments, river or rift valleys (Zambezian and Mopane woodlands, Central Zambezian Miombo woodlands) and c) comprising of groupings of distinct insular ecosystems (East African montane forests). These results indicate that even the most detailed biogeographic models available have limitations when including isolation by distance, physical barriers to gene flow, and habitat heterogeneity. We therefore refined the Olson model by partitioning the above six ecoregions with mutually exclusive haplogroup distributions into 14 sub-ecoregions (cf. <a href="#pone.0000454.s003">Table S3</a>) where haplogroup ranges are not mutually exclusive. This revised ecoregional model divides sub-Saharan Africa into 58 ecoregions and accounts for 80–82% of the variation in the CR data set (<a href="#pone-0000454-t001">Table 1</a>). Nevertheless, a further 22 shared ecoregions (five of which were newly defined, <a href="#pone.0000454.s003">Table S3</a>) are co-inhabited by more than one haplogroup and these collectively account for the 6–9% difference in variance between the refined ecoregional model and the molecular (terminal haplogroup) model. We did not attempt to further revise the model, given that the bushbuck is an adaptable generalist, and individuals may often range outside of the limits of their core habitat.</p>


<h4>Geography</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Geographic distance accounted for 25% of the variation in the CR data set, only 4% of which was attributed to longitude. That latitudinal variation comprises the majority of the geographical variance may not be surprising, given the distribution of sampling locations from as far as 15.7°N to 34°S (<a href="#pone.0000454.s001">Table S1</a>), and the variety of habitats occurring within this range. Nevertheless, both biogeography and phenotype still explained significantly higher levels of variation over and above the conditional influence of geography (<a href="#pone-0000454-t001">Table 1</a>). Furthermore, sequential multivariate regressions always ranked geography as less important than any of the other models tested. Except in the case of the Udvardy model, geographic distance accounted for very little (&lt;2%) of the molecular variation that remained once the best model was fitted to the regression, implying that phenotypic and the more detailed biogeographic models already take most of the geographic variation into account.</p>


<h4>Core habitats</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We defined the core habitats of each haplogroup on the basis of the distribution of its members. Ecoregions were considered part of a haplogroup's core habitat if they were inhabited exclusively or if they contained more than 50% of individuals in a terminal haplogroup. Implementing these criteria, 44 of the 58 ecoregions in the model were defined as core ecoregions to the 23 terminal haplogroups (<a href="#pone.0000454.s003">Table S3</a>). According to our redefined ecoregional model, haplogroup presence ranged from the habitation of a single core ecoregion to occurrence in up to four (<em>ornatus</em>), five (<em>bor, phaleratus</em> and <em>sylvaticus</em>) and maximally six (<em>massaicus</em>; <a href="#pone-0000454-g003">Fig. 3</a>). This core habitat concept allowed the molecular inference of haplogroup ecological affinity and formed the basis for the definition of core biomes and regions of key biogeographic importance.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Origins and Evolution</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The bushbuck is the most widespread and taxonomically diverse ungulate on the African continent and we have shown that all molecular variation in our dataset may be partitioned into two divergent lineages. Fossil remains of <em>T. scriptus</em> are known from several sites in eastern and southern Africa, but its appearances in the fossil record, from locations in Kenya <a href="#pone.0000454-Leakey1">[28]</a>, <a href="#pone.0000454-Harris1">[29]</a> and Ethiopia <a href="#pone.0000454-Kalb1">[30]</a> as early as 3.9 Mya, predate the diversification of <em>Sylvaticus</em> and <em>Scriptus</em> lineages and suggest north-east Africa as the centre of origin for this species. While this region is open and xeric today, it was thickly forested until the late Pliocene <a href="#pone.0000454-Pickford1">[31]</a>, <a href="#pone.0000454-Partridge1">[32]</a>, <a href="#pone.0000454-Reed1">[33]</a>. The presence of common phenotypic characteristics of <em>scriptus</em> and <em>ornatus</em> - the two most basal haplogroups within the <em>Scriptus</em> and <em>Sylvaticus</em> groups (picture panel, <a href="#pone-0000454-g002">Fig. 2</a>), the species' preference for cover and the fact that all juvenile bushbuck are patterned <a href="#pone.0000454-Kingdon1">[11]</a>, <a href="#pone.0000454-Haltenorth1">[13]</a>, suggests that ancestral bushbuck were also strikingly patterned and adapted to the dappled light of forest habitat.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>A global climatic shift approximately 2.8 Mya (see lower panel <a href="#pone-0000454-g002">Fig. 2</a>) is thought to have resulted in a major increase in grassland habitats and, consequently, a dramatic change (or turnover pulse; <a href="#pone.0000454-Vrba1">[25]</a>) in species composition, most notably in north-east Africa <a href="#pone.0000454-Bobe1">[34]</a>, <a href="#pone.0000454-HernandezFernandez1">[35]</a>. The divergence of the <em>Sylvaticus</em> (2.7 Mya) and <em>Scriptus</em> (2.5 Mya) lineages coincides with this paeleoclimatic event, and the resulting loss of patterning in derived bushbuck haplogroups is indicative of the expansion and diversification of ancestral populations into more open habitats. However, paeleoclimatic events and ensuing diversification/specialisation do not explain the independent evolutionary trajectories of the <em>Sylvaticus</em> and <em>Scriptus</em> lineages, as extant phenotypic diversity and ecoregional habitation (<a href="#pone.0000454.s003">Table S3</a>) indicates that ecological specialisation is derived, and haplogroups within the two lineages are still capable of exploiting both forested and open niches. The major tectonic uplift events along the Gregory and Albertine Rifts also date to the late Pliocene (3–2 Mya), <a href="#pone.0000454-Pickford1">[31]</a>, <a href="#pone.0000454-Partridge1">[32]</a> and the extant distributions of <em>Sylvaticus</em> and <em>Scriptus</em> groups on either side of the Rift Valley (<a href="#pone-0000454-g001">Fig. 1</a>) suggest allopatric separation of the two lineages, initially on the basis of geomorphology. Thereafter, diversification can be shown to be influenced primarily by paeleoclimate. The dominance of the ancestral coat patterning within the <em>Scriptus</em> lineage implies a longer association with forested habitats and reflects differences in paeleoclimate on either side of the Rift Valley. Nevertheless the diversification of the first bushbuck haplogroups (<em>scriptus, ornatus</em> and Angola) between 1.7 and 1.4 Mya (<a href="#pone-0000454-g002">Fig. 2</a>), is hypothesised to have been in response the expansion of grasslands during the glacial period of the early Pleistocene, around 1.7 Mya <a href="#pone.0000454-Hewitt1">[36]</a>. The phenotypically similar, temperate-adapted haplogroups of the Ethiopian Highlands (<em>meneliki</em> and <em>powelli</em>) and the Cape (<em>sylvaticus</em>) form a sister clade, that may have been more dominant during colder periods, but now only persist in these temperate zone refuges at opposite ends of the continent. All other diversification events occurred less than 800 Kya, after the periodicity of glacial cycles changed from 41 kyr to 100 kyr (lower panel, <a href="#pone-0000454-g002">Fig. 2</a>), producing prolonged periods of cooling in Africa. The late Pleistocene (&lt;400 Kyr) was especially important for divergence in many large African mammal species <a href="#pone.0000454-Hewitt1">[36]</a> and 18 of the 23 bushbuck haplogroups defined in this study are estimated to have diversified during this period. While it may not be possible to accurately reconstruct the timing of events at this timescale, it is interesting to note the convergence of phenotypically similar montane forms in East Africa during glacial maxima 137 Kya (<em>barkeri</em>) and 69 Kya (<em>haywoodi</em>), as well as the contraction of semi-arid-adapted populations (<em>fasciatus</em>1, Zambezi Valley and Luangwa Valley) to drier refugia during the glacial minimum, 125–80 Kya.</p>


<h4>Model testing</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Despite a number of previous attempts to partition the vast array of phenotypic variation within this species <a href="#pone.0000454-Kingdon1">[11]</a>, <a href="#pone.0000454-Haltenorth1">[13]</a>–<a href="#pone.0000454-Allen2">[17]</a>, <a href="#pone.0000454-Best1">[27]</a>, <a href="#pone.0000454-HaltenorthT1">[37]</a>, <a href="#pone.0000454-Dorst1">[38]</a>, inclusion of all described phenotypes account for only 71% (AMOVA) and 69% (MMR) of the variance in the CR data (<a href="#pone-0000454-t001">Table 1</a>), and more variation can be explained using the molecular data alone. The relative failure of the phenotype models is largely due to the inability of taxonomists to recognise phenotypic variation within the subspecies <em>scriptus</em> (formerly ranging from Senegal to Congo) and <em>ornatus</em> (formerly ranging in South-Central Africa) due to consideration of only a handful of historically popular phenotypic characteristics such as coat colour, patterning, horn and body size and hair length. The use of more rigorous taxonomic methods such as geometric morphometrics may increase phenotypic resolution of these traditionally accepted subspecies. The present study (<a href="#pone-0000454-g001">Fig. 1</a>) partitions the historic <em>scriptus</em> into West Africa (<em>scriptus</em>), Upper Volta, Lower Volta, Niger, and Cameroon-Congo (<em>phaleratus</em>) haplogroups and historic <em>ornatus</em> into Central Plateau (<em>ornatus</em>), Angola, Luangwa Valley, Middle Zambezi Valley and East African Coastal (<em>massaicus</em>) haplogroups. Biogeographic ecoregions <a href="#pone.0000454-Olson2">[7]</a> provided a more solid foundation for the description of species history, especially after the incorporation of potential barriers to dispersal such as unfavourable habitat, distance and altitudinal gradients (<a href="#pone-0000454-t001">Table 1</a>). This strong association between genetic structure and biogeography strengthens our choice of the bushbuck as a model taxon for the elucidation of biogeographical processes in the Afrotropical Realm. Molecular biogeography - defining genetic structure in terms of a biogeographic model - allows the investigation of a number of ecological and evolutionary processes that would otherwise be intractable. These are discussed below.</p>


<h4>Connectivity between Ecoregions and the inference of Ecological affinities</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The number of sampled ecoregions (58) in our taxon-linked ecoregional model is more than twice the number of terminal haplogroups defined by mitochondrial DNA, which suggests considerable connectivity between ecoregions. Since only between 6 and 9% of the genetic variation in the CR data is found within haplogroups, we assume that adjacent ecoregions that are inhabited by a single haplogroup are maximally connected. This assumption only holds true for ecoregions that constitute a haplogroup's core habitat since ecoregions on the outer limits of a haplogroup's range are more likely to experience unidirectional gene flow. The nature of the ecoregions being connected by a single haplogroup therefore provides a measure of a haplogroup's ecological affinity. If the core habitat of a haplogroup comprises a single ecoregion, (<a href="#pone-0000454-g003">Fig. 3</a>) then that haplogroup is more likely to be an ecological specialist, whereas generalists would be expected to inhabit more than one ecoregion. The proportion of each haplogroup that was sampled within its core habitat provides a second, complementary measure of ecological affinity as this value will be lower for generalists. Dividing the proportion of each haplogroup sampled in its core habitat by the number of core ecoregions in which that haplogroup was sampled (columns 3–5, <a href="#pone-0000454-g003">Fig. 3</a>) describes an index of specialisation (IS). This index accounts for those haplogroups that inhabit a single core habitat, but are present at lower frequency in diverse secondary habitats (eg. <em>scriptus</em>, Niger and <em>dianae</em>). When IS was plotted against haplogroup divergence time, a continuum from generalist to specialist haplogroups became apparent, with a transition in ecological affinity evident between IS = 60–70% (shaded grey area, <a href="#pone-0000454-g004">Fig. 4</a>). Furthermore, three separate groupings are present in <a href="#pone-0000454-g004">Figure 4</a>. Early generalists <em>scriptus</em>, <em>ornatus</em> and Angola comprise the most phylogenetically basal (see <a href="#pone-0000454-g002">Fig. 2</a>) of the bushbuck haplogroups, and strongly suggest a generalist ancestral state for this species. An increase in the length and severity of glacial cycles led to an increase in the number of diversification events after 800 Kya. Not all diversification, however, resulted in specialisation: the proliferation of late generalists shows that ubiquitous haplogroups were continually able to exploit diverse habitats from the late Pleistocene to the present. Late specialists are associated with montane inselbergs, semi-arid zones and close canopy forests, and confirm that less ubiquitous haplogroups are derived. The strong association between haplogroups and their core habitats means that early lineages were able to colonise newly created ecoregions, adapt to these ecoregions and diversify within them with little inter-habitat migration. The persistence of these lineages and their close association with habitat to the present day, therefore, hints at the persistence of these ecoregions through the major climatic oscillations of the Pleistocene.</p>
<div class="figure" id="pone-0000454-g004"><div class="img"><a name="pone-0000454-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000454" data-uri="info:doi/10.1371/journal.pone.0000454.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000454.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000454.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000454.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000454.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The evolution of specialization through time in bushbuck.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>The shaded grey area marks the threshold distinguishing specialist from generalist haplogroups and the dashed red line denotes the onset of major diversification events. Boxes denote early generalists; circles, late generalists; and triangles, late specialists. Haplogroups are colour coded as in <a href="#pone-0000454-g001">Figure 1</a>.</p>
<span>doi:10.1371/journal.pone.0000454.g004</span></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>While 44 of the 58 ecoregions defined in this model were found to be core to the 23 haplogroups as a whole, the proportion of each haplogroup sampled in core habitat varied from as little as 30.9% to 100% (<a href="#pone-0000454-g003">Fig. 3</a>). In order to define ecologically meaningful biogeographic regions, we integrated the core habitat concept into our taxon-linked model. This translated to 17 core biomes, with the haplogroup biome diversity graphically described by linking each terminal haplogroup to a biome if it constituted part of its core habitat (blue and red arrows, <a href="#pone-0000454-g003">Fig. 3</a>). The <em>Scriptus</em> and <em>Sylvaticus</em> groups were colour coded for ease of inference. The <em>Scriptus</em> forms inhabit lowland forest, savanna, and coastal forest exclusively while <em>Sylvaticus</em> forms occur exclusively in a much wider range of biomes. Of the three biomes that are shared between the major groups, montane forests were the most widely inhabited by both forms, further demonstrating the adaptive nature of this species to changes in paeleoclimate and geology. It should be noted that this does not imply that <em>Scriptus</em> and <em>Sylvaticus</em> groups are ecologically distinct, but rather reflects the different ecological characteristics of habitats in the Western-Northern and Eastern-Southern halves of Africa.</p>


<h4>Key Biogeographic Regions</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The strong relationship between haplotype structure in the bushbuck and Afrotropical biogeographic structure forms the basis for a model-taxon approach, complementary to, but different from, classical species-assemblage biogeography and traditional phylogeography, because it takes ecological factors into account in testable hypotheses. Implicit in this approach is the need to include several key organisms of different dispersal ability in any similar assessment and as many molecular markers (including those known to be under diversifying selection) as possible. An essential final development to our model is the addition of a geographical component to each of the core biomes that identifies centres of ecological importance. We thus proposed 28 key biogeographic regions (<a href="#pone-0000454-g003">Fig. 3</a>) as defined by the genetic associations of our model taxon in ecologically heterogeneous habitats. This model differs from previously defined species assemblage models in that it links ecological regions on the basis of genetic similarity and in doing so provides a quantitative framework for the inclusion of evolutionary processes in the description of biodiversity.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>With respect to our model taxon, we suggest that these 28 key biogeographic regions summarise the core habitats essential for maintaining continental-scale processes within the bushbuck complex. However, the ubiquity and ecological diversity of this species emphasises the potentially widespread applicability of this model to forest, savanna, montane, woodland and arid adapted species, as well as to other African generalists and means that a molecular biogeographical approach to conservation, using the 28 key biogeographic regions of our model as foci for conservation, will invariably conserve areas of core importance to most Afrotropical mammals. A well-chosen set of organisms and markers can potentially extend the applicability of this approach to a fuller ecoregion analysis.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Samples and molecular methods</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>485 skin or tissue specimens were collected from museum collections, hunters and taxidermists covering 239 locations in 27 sub-Saharan countries and included 33 putative subspecies (<a href="#pone.0000454.s001">Table S1</a>). Genomic DNA was isolated from ≤500 mg (dry weight) of source material by SDS-proteinase K digestion and phenol-chloroform extraction <a href="#pone.0000454-Sambrook1">[39]</a> in an isolated laboratory. Primers MT4 <a href="#pone.0000454-Arnason1">[40]</a> and BT16168H <a href="#pone.0000454-Simonsen1">[41]</a> reliably amplify a 400–500 bp non-nuclear 5′ fragment of the mitochondrial control region in African bovids <a href="#pone.0000454-Nersting1">[22]</a>, <a href="#pone.0000454-Simonsen1">[41]</a>, <a href="#pone.0000454-Birungi1">[42]</a>. Control region PCR was carried out on 50 ng DNA in a total reaction volume of 25 µl containing 0.2 mM of each primer, 3.0 mM MgCl<sub>2</sub>, 0.5 mM DNTPs, 1 U Taq DNA polymerase (Invitrogen) and 1× PCR buffer. Cycling conditions were: initial denaturation for 5 min at 95°C; 35 cycles of denaturation for 30 s at 95°C, annealing for 30 s at 58°C and extension for 1 min at 72°C; followed by a final extension phase for 10 min at 72°C. A 556 bp fragment of the cytochrome (cyt) <em>b</em> gene was amplified with primers L15162 <a href="#pone.0000454-Paabo1">[43]</a> and H15761 [C. Fernandes, pers. comm.] in 161 samples in order to confirm haplogroups and for the estimation of haplogroup divergence times. The cyt <em>b</em> PCR protocol was as above except that reactions contained 0.4 mM of each primer and primer annealing was for 30 s at 55°C. PCR products were purified by digestion with 5 U/µl Exonuclease I and 0.5 U/µl Shrimp Alkaline Phosphatase (Amersham) for 60 min at 37°C, followed by denaturation at 80°C for 15 min. Direct sequencing in both directions was carried out using the BigDye Terminator Kit (Applied Biosystems) and sequencing products were analysed with an ABI 3100 sequencer. Sequences were assembled using Sequencher 4.1.2 (Gene Codes). The 646 sequences have Genbank accession numbers EF138117-EF138601 (CR) and EF137956-EF138116 (cyt <em>b</em>).</p>


<h4>Genetic structure</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The relationships among haplotypes in the CR data set were visualised in a median joining network [using NETWORK 4.1, 44; <a href="#pone-0000454-g001">Fig. 1</a>]. CR haplotypes were assigned to monophyletic haplogroups (<a href="#pone.0000454.s002">Table S2</a>). The diversity of the cyt <em>b</em> data set was examined by a maximum likelihood (ML) phylogenetic analysis (<a href="#pone-0000454-g002">Fig. 2</a>) based on the optimal HKY model of nucleotide substitution <a href="#pone.0000454-Hasegawa1">[45]</a> as indicated by ModelTest 3.7 <a href="#pone.0000454-Posada1">[46]</a>, using TREEFINDER <a href="#pone.0000454-Jobb1">[47]</a> with empirically determined nucleotide frequencies and transition-transversion ratio. Cyt <em>b</em> nodes were dated by a Bayesian relaxed molecular clock as implemented in the PAML 3.14 <a href="#pone.0000454-Yang1">[48]</a> and MULTIDISTRIBUTE <a href="#pone.0000454-Thorne1">[24]</a>, <a href="#pone.0000454-Kishino1">[49]</a>. Four calibration points were invoked: we conservatively estimated the emergence of two main lineages at an upper limit of 3 Mya, and restricted the evolution of Gregory Rift montane isolates to the onset of the last glacial cycle, approximately 110 Kya. We estimated a lower limit of the age of the phylogeny at 3.9 Mya, based on the earliest appearance of <em>T. scriptus</em> in the fossil record <a href="#pone.0000454-Kalb1">[30]</a>.</p>


<h4>Model testing</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Phenotypic and biogeographic model testing of the CR data set (<a href="#pone-0000454-t001">Table 1</a>) was carried out using two methods. Firstly the analysis of molecular variance [AMOVA, 50] framework was implemented, as it allowed the hierarchical partitioning of the data into variance components. For AMOVA analyses, we defined the basic units for each model relative to the phenotypic or biogeographic model being tested. The basic phenotypic unit was the original ‘subspecies’ assignment of each specimen based on taxonomy, which were in turn grouped (or synonymised) according to the classifications of Lydekker <a href="#pone.0000454-Lydekker1">[16]</a>, Allen <a href="#pone.0000454-Allen2">[17]</a>, Best <a href="#pone.0000454-Dorst1">[38]</a>, Haltenorth <a href="#pone.0000454-Sambrook1">[39]</a> and Grubb <a href="#pone.0000454-Grubb1">[15]</a>. Grubb's scheme partitioned the phenotypic variation in the species into four broad groups, all of which are represented in East Africa. In order to test Grubb's East African hypothesis over the entire species range, we assumed the Best <a href="#pone.0000454-Dorst1">[38]</a> classification for the individuals in our data set that were sampled outside East Africa. The relative statistical support for these groupings was assessed by the partitioning of variation among groups, among subspecies and within subspecies. To test the biogeographical models of both Udvardy <a href="#pone.0000454-Udvardy1">[3]</a> and Olson <em>et al.</em> <a href="#pone.0000454-Olson2">[7]</a> under the same framework, each individual was assigned to a biogeographic province as well as an ecoregion (cf. <a href="#pone-0000454-g001">Fig. 1B and C</a>) based on the location from which it was sampled. Seventeen of the biogeographic provinces in the Udvardy model and 50 ecoregions in the Olson model were represented by samples in our CR data set (<a href="#pone.0000454.s003">Table S3</a>), accounting for 100% and 81% of the provinces and ecoregions respectively in which bushbuck are known to occur. The remaining unsampled ecoregions are small, primarily montane or highly isolated. Overall model strength was assessed by one-group AMOVA (<a href="#pone-0000454-t001">Table 1</a>). The variance among populations (<em>F</em><sub>ST</sub>) provided a measure of the maximum genetic variation accounted for by each model. The same data set was used for each test, thus <em>F</em><sub>ST</sub> was used to assess each model relative to one based on terminal haplogroup definition (the molecular model), which is expected to return the upper limit of <em>F</em><sub>ST</sub> for the data set.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Model testing was also performed by multivariate matrix regression (MMR), with the software DISTLM <a href="#pone.0000454-Anderson1">[51]</a>. While this does not allow for the hierarchical partitioning of variance, the advantage of this method over AMOVA is that it uses an explicit linear model and does not require an <em>a priori</em> user-defined population structure. Furthermore, the geographic distance separating sampling locations of widely distributed mammal species may significantly influence genetic structure <a href="#pone.0000454-Pilot1">[52]</a>, <a href="#pone.0000454-Ramachandran1">[53]</a>. MMR allows the quantification of this influence, conditional on that of biogeography and phenotype. In addition, the forward selection method <a href="#pone.0000454-Anderson2">[DISTLM <em>forward</em>, 54]</a> sequentially determines which of two sets of variables (geography <em>versus</em> phenotype, biogeography or molecular) fit the data best, and the proportion of the remaining variance described by the secondary set of variables. Pair-wise genetic distances between all 239 sampling locations was used as the response matrix and tested against phenotypic, biogeographic, molecular and geographic predictor matrices. A matrix of latitude and longitude covariables for all sampling locations was used to assess the conditional and sequential influence of geographic distance on the models being tested.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>In a species exhibiting such marked local differentiation, many of the haplogroups defined by monophyly may be expected to exclusively inhabit the ecoregions to which they have become associated. Ecoregions inhabited by more than one haplogroup are also expected to occur, first due to inadequacies in the underlying biogeographic model and second, due to the general ubiquity of this species. We addressed the former by refining our biogeographic model and analysed the latter by defining areas of core habitation for each haplogroup. The nature of the core ecoregions linked by a single haplogroup allowed the inference of ecological affinities as well as the identification core biomes (<a href="#pone-0000454-g003">Fig. 3</a>).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000454"></div><a name="pone.0000454.s001" id="pone.0000454.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000454.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Classification and reference details of 485 bushbuck specimens</p>
<p class="siDoi">doi:10.1371/journal.pone.0000454.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.13 MB XLS)</p>
<a name="pone.0000454.s002" id="pone.0000454.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000454.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Summary statistics for higher, intermediate and terminal level haplogroups as defined in <a href="#pone-0000454-g001">Figure 1</a></p>
<p class="siDoi">doi:10.1371/journal.pone.0000454.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0000454.s003" id="pone.0000454.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000454.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Refining the Olson model and defining core ecoregions for 23 bushbuck haplogroups.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000454.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Peter Grubb, Woody Cotterill, Sabrina Locatelli, Mark Achtman, Phillippe Roumagnac, Bodo Linz and Viktória Rajnics. We wish to thank the following museums/departments for allowing us access to their collections: American Museum of Natural History, New York; Department of Evolutionary Biology, University of Copenhagen; Koninklijk Belgisch Instituut voor Natuurwetenschappen, Brussel; Livingstone Museum, Livingstone, Zambia; Museum für Naturkunde, Berlin; Nationaal Natuurhistorisch Museum, Leiden; Natural History Museum, Bulawayo; Natural History Museum, London; Naturhistoriska riksmuseet, Stockholm; Powell Cotton Museum, Birchington, Kent; Royal Museum for Central Africa, Tervuren; Staatliche Naturhistorische Sammlungen Dresden; Staatliches Museum für Naturkunde, Stuttgart; Zoologisches Forschungsmuseum Alexander Koenig, Bonn. Also for the provision of specimens, we thank the following taxidermists/hunting operators: Bangweulu Taxidermy, Lusaka, Zambia; Bromley Game Skin Tannery, Harare, Zimbabwe; Dendro Park Hunting Ranch, Nanzhila, Zambia; Derek Robinson Taxidermists, Westgate, South Africa; Life-Form Taxidermy, White River, South Africa; McDonald Pro Hunting, South Africa; Nico van Rooyen Taxidermy, Rosslyn, South Africa; Taxidermy Africa, Humansdorp, South Africa; Taxidermy Enterprises, Bulawayo, Zimbabwe; Theo Pohl Taxidermy, Northam, South Africa; Trans Africa Taxidermists, Muldersdrift, South Africa; and Travel Ethiopia, Addis Ababa, Ethiopia.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YM MB. Performed the experiments: YM. Analyzed the data: YM MB. Contributed reagents/materials/analysis tools: MB. Wrote the paper: YM MB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000454-Olson1" id="pone.0000454-Olson1"></a>Olson DM, Dinerstein E (1998) The Global 200: a representation approach to conserving the Earth's distinctive ecoregions. Conserv Biol  12: 502–515.  <ul class="find" data-citedArticleID="952578" data-doi="10.1046/j.1523-1739.1998.012003502.x"><li><a href="http://dx.doi.org/10.1046/j.1523-1739.1998.012003502.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Global+200%3A+a+representation+approach+to+conserving+the+Earth%27s+distinctive+ecoregions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Global+200%3A+a+representation+approach+to+conserving+the+Earth%27s+distinctive+ecoregions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000454-Doggart1" id="pone.0000454-Doggart1"></a>Doggart N, Perkin A, Kiure J, Fjeldsa J, Poyntin J, et al.  (2006) Changing places: how the results of new field work in the Rubeho Mountains influence conservation priorities in the Eastern Arc Mountains of Tanzania. Afr J Ecol  44: 134–144.  <ul class="find" data-citedArticleID="952532" data-doi="10.1111/j.1365-2028.2006.00572.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2028.2006.00572.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changing+places%3A+how+the+results+of+new+field+work+in+the+Rubeho+Mountains+influence+conservation+priorities+in+the+Eastern+Arc+Mountains+of+Tanzania." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changing+places%3A+how+the+results+of+new+field+work+in+the+Rubeho+Mountains+influence+conservation+priorities+in+the+Eastern+Arc+Mountains+of+Tanzania.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000454-Udvardy1" id="pone.0000454-Udvardy1"></a>Udvardy MDF (1975) A classification of the biogeographical provinces of the world. Switzerland: IUCN Occasional Paper 18.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000454-Gleason1" id="pone.0000454-Gleason1"></a>Gleason HA, Cronquist A (1964) The Natural Geography of Plants. New York: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000454-Good1" id="pone.0000454-Good1"></a>Good R (1964) The Geography of Flowering Plants. London: Longmans.   <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0000454-Allen1" id="pone.0000454-Allen1"></a>Allen JA (1871) On the mammals and winter birds of east Florida, with an examination ofcertain assumed specific characters in birds and a sketch of the bird faunae of eastern North America. Bull Mus Comp Zool  2: 161–450.  <ul class="find" data-citedArticleID="952506" data-doi="10.1086/270784"><li><a href="http://dx.doi.org/10.1086/270784" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+mammals+and+winter+birds+of+east+Florida%2C+with+an+examination+ofcertain+assumed+specific+characters+in+birds+and+a+sketch+of+the+bird+faunae+of+eastern+North+America." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+mammals+and+winter+birds+of+east+Florida%2C+with+an+examination+ofcertain+assumed+specific+characters+in+birds+and+a+sketch+of+the+bird+faunae+of+eastern+North+America.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000454-Olson2" id="pone.0000454-Olson2"></a>Olson DM, Dinerstein E, Wikramanayake ED, Burgess ND, Powell GVN, et al.  (2001) Terrestrial ecoregions of the world: a new map of life on earth. BioScience  51: 933–937.  <ul class="find" data-citedArticleID="952580" data-doi="10.1641/0006-3568(2001)051[0933:teotwa]2.0.co;2"><li><a href="http://dx.doi.org/10.1641/0006-3568(2001)051[0933:teotwa]2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Terrestrial+ecoregions+of+the+world%3A+a+new+map+of+life+on+earth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Terrestrial+ecoregions+of+the+world%3A+a+new+map+of+life+on+earth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000454-Roelants1" id="pone.0000454-Roelants1"></a>Roelants K, Jiang JP, Bossuyt F (2004) Endemic ranid (Amphibia : Anura) genera in southern mountain ranges of the Indian subcontinent represent ancient frog lineages: evidence from molecular data. Mol Phyl Evol  31: 730–740.  <ul class="find" data-citedArticleID="952598" data-doi="10.1016/j.ympev.2003.09.011"><li><a href="http://dx.doi.org/10.1016/j.ympev.2003.09.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endemic+ranid+%28Amphibia+%3A+Anura%29+genera+in+southern+mountain+ranges+of+the+Indian+subcontinent+represent+ancient+frog+lineages%3A+evidence+from+molecular+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endemic+ranid+%28Amphibia+%3A+Anura%29+genera+in+southern+mountain+ranges+of+the+Indian+subcontinent+represent+ancient+frog+lineages%3A+evidence+from+molecular+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000454-Pennington1" id="pone.0000454-Pennington1"></a>Pennington RT, Lavin M, Prado DE, Pendry CA, Pell SK, et al.  (2004) Historical climate change and speciation: neotropical seasonally dry forest plants show patterns of both tertiary and quaternary diversification. Philos Trans R Soc London B  359: 515–537.  <ul class="find" data-citedArticleID="952586" data-doi="10.1098/rstb.2003.1435"><li><a href="http://dx.doi.org/10.1098/rstb.2003.1435" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Historical+climate+change+and+speciation%3A+neotropical+seasonally+dry+forest+plants+show+patterns+of+both+tertiary+and+quaternary+diversification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Historical+climate+change+and+speciation%3A+neotropical+seasonally+dry+forest+plants+show+patterns+of+both+tertiary+and+quaternary+diversification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000454-Whittaker1" id="pone.0000454-Whittaker1"></a>Whittaker RJ, Araujo MB, Paul J, Ladle RJ, Watson JEM, et al.  (2005) Conservation Biogeography: assessment and prospect. Divers Distrib  11: 3–23.  <ul class="find" data-citedArticleID="952610" data-doi="10.1111/j.1366-9516.2005.00143.x"><li><a href="http://dx.doi.org/10.1111/j.1366-9516.2005.00143.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+Biogeography%3A+assessment+and+prospect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+Biogeography%3A+assessment+and+prospect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000454-Kingdon1" id="pone.0000454-Kingdon1"></a>Kingdon J (1997) The Kingdon field guide to African mammals. London: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0000454-East1" id="pone.0000454-East1"></a>East R (1999) African antelope database 1998. Gland, Switzerland: IUCN/SSC Antelope Specialist Group. pp. 116–122.  <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0000454-Haltenorth1" id="pone.0000454-Haltenorth1"></a>Haltenorth T, Diller H (1980) A field guide to the mammals of Africa including Madagascar. London: Collins. pp. 57–58.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000454-Ansell1" id="pone.0000454-Ansell1"></a>Ansell WFH (1971) The mammals of Africa: an identification manual.  Meester J, Setzer HW, editors. 15. Washington DC: Smithsonian Institution Press. pp. 1–84.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000454-Grubb1" id="pone.0000454-Grubb1"></a>Grubb P (1985)  Geographical variation in the bushbuck of eastern Africa (<em>Tragelaphus scriptus</em>; Bovidae). In: Schuchmann KL, editor. Proc Intern Symp African Vertebr. Bonn: Museum A König. pp. 11–26.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0000454-Lydekker1" id="pone.0000454-Lydekker1"></a>Lydekker R (1914) Catalogue of the ungulate mammals in the British Museum. Natural History) Vol. 3. London: British Museum. pp. 317–326.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000454-Allen2" id="pone.0000454-Allen2"></a>Allen GM (1939) A checklist of African mammals. Bull Mus Comp Zool Harvard  83: 1–763.  <ul class="find" data-citedArticleID="952508"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+checklist+of+African+mammals.&amp;auth=&amp;atitle=A+checklist+of+African+mammals." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+checklist+of+African+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+checklist+of+African+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000454-Arctander1" id="pone.0000454-Arctander1"></a>Arctander P, Johansen C, Coutellec-Vret M-A (1999) Phylogeography of three closely related African bovids (Tribe <em>Alcelaphini</em>). Mol Biol Evol  16: 1724–1739.  <ul class="find" data-citedArticleID="952516" data-doi="10.1093/oxfordjournals.molbev.a026085"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+of+three+closely+related+African+bovids+%28Tribe+Alcelaphini%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+of+three+closely+related+African+bovids+%28Tribe+Alcelaphini%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000454-Flagstad1" id="pone.0000454-Flagstad1"></a>Flagstad Ø, Syvertsen PO, Stenseth NC, Jakobsen KS (2001) Environmental change and rates of evolution: the phylogeographic pattern within the hartebeest complex as related to climatic variation. Philos Trans R Soc London B  268: 667–677.  <ul class="find" data-citedArticleID="952540" data-doi="10.1098/rspb.2000.1416"><li><a href="http://dx.doi.org/10.1098/rspb.2000.1416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environmental+change+and+rates+of+evolution%3A+the+phylogeographic+pattern+within+the+hartebeest+complex+as+related+to+climatic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environmental+change+and+rates+of+evolution%3A+the+phylogeographic+pattern+within+the+hartebeest+complex+as+related+to+climatic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000454-Muwanika1" id="pone.0000454-Muwanika1"></a>Muwanika VB, Nyakaana S, Siegismund HR, Arctander P (2003) Phylogeography and population structure of the common warthog (<em>Phacochoerus africanus</em>) inferred from variation in mitochondrial DNA sequences and microsatellite loci. Heredity  91: 361–372.  <ul class="find" data-citedArticleID="952574" data-doi="10.1038/sj.hdy.6800341"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+and+population+structure+of+the+common+warthog+%28Phacochoerus+africanus%29+inferred+from+variation+in+mitochondrial+DNA+sequences+and+microsatellite+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+and+population+structure+of+the+common+warthog+%28Phacochoerus+africanus%29+inferred+from+variation+in+mitochondrial+DNA+sequences+and+microsatellite+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000454-Debruyne1" id="pone.0000454-Debruyne1"></a>Debruyne R, Van Holt A, Barriel V, Tassy P (2003) Status of the so-called African pygmy elephant (<em>Loxodonta pumilio</em> (Noack 1906)): phylogeny of cytochrome b and mitochondrial control region sequences. C R Biol  326: 687–697.  <ul class="find" data-citedArticleID="952530" data-doi="10.1016/s1631-0691(03)00158-6"><li><a href="http://dx.doi.org/10.1016/s1631-0691(03)00158-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Status+of+the+so-called+African+pygmy+elephant+%28Loxodonta+pumilio+%28Noack+1906%29%29%3A+phylogeny+of+cytochrome+b+and+mitochondrial+control+region+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Status+of+the+so-called+African+pygmy+elephant+%28Loxodonta+pumilio+%28Noack+1906%29%29%3A+phylogeny+of+cytochrome+b+and+mitochondrial+control+region+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000454-Nersting1" id="pone.0000454-Nersting1"></a>Nersting LG, Arctander P (2001) Phylogeography and conservation of impala and greater kudu. Mol Ecol  10: 711–719.  <ul class="find" data-citedArticleID="952576" data-doi="10.1046/j.1365-294x.2001.01205.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2001.01205.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+and+conservation+of+impala+and+greater+kudu." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+and+conservation+of+impala+and+greater+kudu.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000454-Lorenzen1" id="pone.0000454-Lorenzen1"></a>Lorenzen ED, Arctander P, Siegismund HR (2006) Regional genetic structuring and evolutionary history of the impala <em>Aepyceros melampus</em>. J Hered  97: 119–132.  <ul class="find" data-citedArticleID="952570" data-doi="10.1093/jhered/esj012"><li><a href="http://dx.doi.org/10.1093/jhered/esj012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regional+genetic+structuring+and+evolutionary+history+of+the+impala+Aepyceros+melampus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regional+genetic+structuring+and+evolutionary+history+of+the+impala+Aepyceros+melampus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000454-Thorne1" id="pone.0000454-Thorne1"></a>Thorne JL, Kishino H, Painter JS (1998) Estimating the rate of evolution of the rate of molecular evolution. Mol Biol Evol  15: 1647–1657.  <ul class="find" data-citedArticleID="952604" data-doi="10.1093/oxfordjournals.molbev.a025892"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025892" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+rate+of+evolution+of+the+rate+of+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+rate+of+evolution+of+the+rate+of+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000454-Vrba1" id="pone.0000454-Vrba1"></a>Vrba ES (1985) Environment and Evolution: alternative causes of the temporal distribution of evolutionary events. S Afr J Sci  81: 229–236.  <ul class="find" data-citedArticleID="952608"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Environment+and+Evolution%3A+alternative+causes+of+the+temporal+distribution+of+evolutionary+events.&amp;auth=&amp;atitle=Environment+and+Evolution%3A+alternative+causes+of+the+temporal+distribution+of+evolutionary+events." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environment+and+Evolution%3A+alternative+causes+of+the+temporal+distribution+of+evolutionary+events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environment+and+Evolution%3A+alternative+causes+of+the+temporal+distribution+of+evolutionary+events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000454-Behrensmeyer1" id="pone.0000454-Behrensmeyer1"></a>Behrensmeyer AK, Todd NE, Potts R, McBrinn GE (1997) Late Pliocene faunal turnover in the Turkana Basin, Kenya and Ethiopia. Science  278: 1589–1594.  <ul class="find" data-citedArticleID="952522" data-doi="10.1126/science.278.5343.1589"><li><a href="http://dx.doi.org/10.1126/science.278.5343.1589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Late+Pliocene+faunal+turnover+in+the+Turkana+Basin%2C+Kenya+and+Ethiopia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Late+Pliocene+faunal+turnover+in+the+Turkana+Basin%2C+Kenya+and+Ethiopia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000454-Best1" id="pone.0000454-Best1"></a>Best GA (1962) Rowland Ward's records of big game. XI<sup>th</sup> Edition: Africa. London: Rowland Ward Ltd. pp. 198–208.  <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0000454-Leakey1" id="pone.0000454-Leakey1"></a>Leakey MG, Harris JM (2003) Lothagam: the dawn of humanity in eastern Africa. New York: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0000454-Harris1" id="pone.0000454-Harris1"></a>Harris JM, Brown FH, Leakey MG (1988) Stratigraphy and paleontology of Pliocene and Pleistocene localities west of Lake Turkana, Kenya. Los Angeles County Mus Nat Hist Contr Sci  399: 1–128.  <ul class="find" data-citedArticleID="952552"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stratigraphy+and+paleontology+of+Pliocene+and+Pleistocene+localities+west+of+Lake+Turkana%2C+Kenya.&amp;auth=&amp;atitle=Stratigraphy+and+paleontology+of+Pliocene+and+Pleistocene+localities+west+of+Lake+Turkana%2C+Kenya." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stratigraphy+and+paleontology+of+Pliocene+and+Pleistocene+localities+west+of+Lake+Turkana%2C+Kenya." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stratigraphy+and+paleontology+of+Pliocene+and+Pleistocene+localities+west+of+Lake+Turkana%2C+Kenya.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000454-Kalb1" id="pone.0000454-Kalb1"></a>Kalb JE, Oswald EB, Tebedge S, Mebrate A, Tola E, et al.  (1982) Geology and stratigraphy of Neogene deposits, Middle Awash Valley, Ethiopia. Nature  298: 98–106.  <ul class="find" data-citedArticleID="952562" data-doi="10.1038/298017a0"><li><a href="http://dx.doi.org/10.1038/298017a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geology+and+stratigraphy+of+Neogene+deposits%2C+Middle+Awash+Valley%2C+Ethiopia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geology+and+stratigraphy+of+Neogene+deposits%2C+Middle+Awash+Valley%2C+Ethiopia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000454-Pickford1" id="pone.0000454-Pickford1"></a>Pickford M (1990) Uplift of the roof of Africa and its bearing on the evolution of Mankind. Hum Evol  5: 1–20.  <ul class="find" data-citedArticleID="952588" data-doi="10.1007/bf02436472"><li><a href="http://dx.doi.org/10.1007/bf02436472" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uplift+of+the+roof+of+Africa+and+its+bearing+on+the+evolution+of+Mankind." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uplift+of+the+roof+of+Africa+and+its+bearing+on+the+evolution+of+Mankind.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000454-Partridge1" id="pone.0000454-Partridge1"></a>Partridge TC, Wood B, deMenocal PB (1995)  The influence of global climatic change and regional uplift on large-mammalian evolution in East and Southern Africa. In: Vrba E, Denton G, Partridge TC, Burckle L, editors. Paleoclimate and Evolution With Emphasis of Human Origins. New Haven: Yale Univ Press. pp. 330–355.  <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000454-Reed1" id="pone.0000454-Reed1"></a>Reed KE (1997) Early hominid evolution and ecological change through the African Plio-Pleistocene. J Hum Evol  32: 289–322.  <ul class="find" data-citedArticleID="952596" data-doi="10.1006/jhev.1996.0106"><li><a href="http://dx.doi.org/10.1006/jhev.1996.0106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+hominid+evolution+and+ecological+change+through+the+African+Plio-Pleistocene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+hominid+evolution+and+ecological+change+through+the+African+Plio-Pleistocene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000454-Bobe1" id="pone.0000454-Bobe1"></a>Bobe R, Eck GG (2001) Responses of African bovids to Pliocene climatic change. Paleobiology  27: 1–47.  <ul class="find" data-citedArticleID="952528" data-doi="10.1666/0094-8373(2001)027<0001:roabtp&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1666/0094-8373(2001)027<0001:roabtp&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Responses+of+African+bovids+to+Pliocene+climatic+change." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Responses+of+African+bovids+to+Pliocene+climatic+change.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000454-HernandezFernandez1" id="pone.0000454-HernandezFernandez1"></a>Hernandez Fernandez M, Vrba ES (2006) Plio-Pleistocene climatic change in the Turkana Basin (East Africa): Evidence from large mammal faunas. J Hum Evol  50: 595–626.  <ul class="find" data-citedArticleID="952556" data-doi="10.1016/j.jhevol.2005.11.004"><li><a href="http://dx.doi.org/10.1016/j.jhevol.2005.11.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plio-Pleistocene+climatic+change+in+the+Turkana+Basin+%28East+Africa%29%3A+Evidence+from+large+mammal+faunas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plio-Pleistocene+climatic+change+in+the+Turkana+Basin+%28East+Africa%29%3A+Evidence+from+large+mammal+faunas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000454-Hewitt1" id="pone.0000454-Hewitt1"></a>Hewitt GM (2004) The structure of biodiversity - insights from molecular phylogeography. Front Zool  1: 4.  <ul class="find" data-citedArticleID="952558"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+structure+of+biodiversity+-+insights+from+molecular+phylogeography.&amp;auth=&amp;atitle=The+structure+of+biodiversity+-+insights+from+molecular+phylogeography." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+biodiversity+-+insights+from+molecular+phylogeography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+biodiversity+-+insights+from+molecular+phylogeography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000454-HaltenorthT1" id="pone.0000454-HaltenorthT1"></a>Haltenorth T H (1963) Klassifikation der Säugetiere: Artiodactyla 1. Handbuch der Zoologie  8: 1–167.  <ul class="find" data-citedArticleID="952550"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Klassifikation+der+S%26%23xe4%3Bugetiere%3A+Artiodactyla+1.&amp;auth=&amp;atitle=Klassifikation+der+S%26%23xe4%3Bugetiere%3A+Artiodactyla+1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Klassifikation+der+S%26%23xe4%3Bugetiere%3A+Artiodactyla+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Klassifikation+der+S%26%23xe4%3Bugetiere%3A+Artiodactyla+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000454-Dorst1" id="pone.0000454-Dorst1"></a>Dorst J, Dandelot P (1970) A field guide to the larger mammals of Africa. London: Collins.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0000454-Sambrook1" id="pone.0000454-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory Press. pp. 9.16–9.19.  <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0000454-Arnason1" id="pone.0000454-Arnason1"></a>Arnason U, Gullberg A, Widegren B (1993) Cetacean mitochondrial DNA control region: Sequences of all extant baleen whales and two sperm whale species. Mol Biol Evol  10: 960–970.  <ul class="find" data-citedArticleID="952518"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cetacean+mitochondrial+DNA+control+region%3A+Sequences+of+all+extant+baleen+whales+and+two+sperm+whale+species.&amp;auth=&amp;atitle=Cetacean+mitochondrial+DNA+control+region%3A+Sequences+of+all+extant+baleen+whales+and+two+sperm+whale+species." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cetacean+mitochondrial+DNA+control+region%3A+Sequences+of+all+extant+baleen+whales+and+two+sperm+whale+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cetacean+mitochondrial+DNA+control+region%3A+Sequences+of+all+extant+baleen+whales+and+two+sperm+whale+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000454-Simonsen1" id="pone.0000454-Simonsen1"></a>Simonsen BT, Siegismund HR, Arctander P (1998) Population structure of African buffalo inferred from mtDNA sequences and microsatellite loci: high variation but low differentiation. Mol Ecol  7: 225–237.  <ul class="find" data-citedArticleID="952602" data-doi="10.1046/j.1365-294x.1998.00343.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.1998.00343.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure+of+African+buffalo+inferred+from+mtDNA+sequences+and+microsatellite+loci%3A+high+variation+but+low+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure+of+African+buffalo+inferred+from+mtDNA+sequences+and+microsatellite+loci%3A+high+variation+but+low+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000454-Birungi1" id="pone.0000454-Birungi1"></a>Birungi J, Arctander P (2000) Large sequence divergence of mitochondrial DNA genotypes of the control region within populations of the African antelope, kob (<em>Kobus kob</em>). Mol Ecol  9: 1997–2008.  <ul class="find" data-citedArticleID="952526" data-doi="10.1046/j.1365-294x.2000.01107.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.01107.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large+sequence+divergence+of+mitochondrial+DNA+genotypes+of+the+control+region+within+populations+of+the+African+antelope%2C+kob+%28Kobus+kob%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large+sequence+divergence+of+mitochondrial+DNA+genotypes+of+the+control+region+within+populations+of+the+African+antelope%2C+kob+%28Kobus+kob%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000454-Paabo1" id="pone.0000454-Paabo1"></a>Paabo S, Wilson AC (1988) Polymerase Chain Reaction reveals cloning artifacts. Nature  334: 387–388.  <ul class="find" data-citedArticleID="952582" data-doi="10.1038/334387b0"><li><a href="http://dx.doi.org/10.1038/334387b0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymerase+Chain+Reaction+reveals+cloning+artifacts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymerase+Chain+Reaction+reveals+cloning+artifacts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000454-Bandelt1" id="pone.0000454-Bandelt1"></a>Bandelt HJ, Forster P, Röhl A (1999) Median-joining networks for inferring intraspecific phylogenies. Mol Biol Evol  16: 37–48.  <ul class="find" data-citedArticleID="952520" data-doi="10.1093/oxfordjournals.molbev.a026036"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Median-joining+networks+for+inferring+intraspecific+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Median-joining+networks+for+inferring+intraspecific+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000454-Hasegawa1" id="pone.0000454-Hasegawa1"></a>Hasegawa M, Kishino H, Yano K (1985) Dating of the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol  22: 160–174.  <ul class="find" data-citedArticleID="952554" data-doi="10.1007/bf02101694"><li><a href="http://dx.doi.org/10.1007/bf02101694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+of+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+of+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000454-Posada1" id="pone.0000454-Posada1"></a>Posada D, Crandall KA (1998) Modeltest: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="952592" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeltest%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeltest%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000454-Jobb1" id="pone.0000454-Jobb1"></a>Jobb G, von Haeseler A, Strimmer K (2004) TREEFINDER: A powerful graphical analysis environment for molecular phylogenetics. BMC Evol Biol.  4: 18.  <ul class="find" data-citedArticleID="952560" data-doi="10.1186/1471-2148-4-18"><li><a href="http://dx.doi.org/10.1186/1471-2148-4-18" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREEFINDER%3A+A+powerful+graphical+analysis+environment+for+molecular+phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREEFINDER%3A+A+powerful+graphical+analysis+environment+for+molecular+phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000454-Yang1" id="pone.0000454-Yang1"></a>Yang Z, Rannala B (1997) Bayesian phylogenetic inference using DNA sequences: a Markov chain Monte Carlo method. Mol Biol Evol  14: 717–724.  <ul class="find" data-citedArticleID="952612" data-doi="10.1093/oxfordjournals.molbev.a025811"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+phylogenetic+inference+using+DNA+sequences%3A+a+Markov+chain+Monte+Carlo+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+phylogenetic+inference+using+DNA+sequences%3A+a+Markov+chain+Monte+Carlo+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000454-Kishino1" id="pone.0000454-Kishino1"></a>Kishino H, Thorne JL, Bruno WJ (2001) Performance of a divergence time estimation method under a probabilistic model of rate evolution. Mol Biol Evol  18: 352–361.  <ul class="find" data-citedArticleID="952566" data-doi="10.1093/oxfordjournals.molbev.a003811"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Performance+of+a+divergence+time+estimation+method+under+a+probabilistic+model+of+rate+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Performance+of+a+divergence+time+estimation+method+under+a+probabilistic+model+of+rate+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000454-Excoffier1" id="pone.0000454-Excoffier1"></a>Excoffier L, Smouse PE, Quattro JM (1992) Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data. Genetics  131: 479–491.  <ul class="find" data-citedArticleID="952538"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data.&amp;auth=&amp;atitle=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+application+to+human+mitochondrial+DNA+restriction+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000454-Anderson1" id="pone.0000454-Anderson1"></a>Anderson MJ (2004) DISTLM v.5: a FORTRAN computer program to calculate a distance-based multivariate analysis for a linear model. Department of Statistics, University of Auckland, New Zealand.   <ul class="find-nolinks"></ul></li><li><span class="label">52.
              </span><a name="pone.0000454-Pilot1" id="pone.0000454-Pilot1"></a>Pilot M, Jedrzejewski W, Branicki W, Sidorovich VE, Jedrzejewska B, et al.  (2006) Ecological factors influence population genetic structure of European grey wolves. Mol Ecol  15: 4533–4553.  <ul class="find" data-citedArticleID="952590" data-doi="10.1111/j.1365-294x.2006.03110.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.03110.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+factors+influence+population+genetic+structure+of+European+grey+wolves." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+factors+influence+population+genetic+structure+of+European+grey+wolves.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000454-Ramachandran1" id="pone.0000454-Ramachandran1"></a>Ramachandran S, Deshpande O, Roseman CC, Rosenberg NA, Feldman MW, et al.  (2005) Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa. Proc Natl Acad Sci USA  102: 15942–15947.  <ul class="find" data-citedArticleID="952594" data-doi="10.1073/pnas.0507611102"><li><a href="http://dx.doi.org/10.1073/pnas.0507611102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Support+from+the+relationship+of+genetic+and+geographic+distance+in+human+populations+for+a+serial+founder+effect+originating+in+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Support+from+the+relationship+of+genetic+and+geographic+distance+in+human+populations+for+a+serial+founder+effect+originating+in+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000454-Anderson2" id="pone.0000454-Anderson2"></a>Anderson MJ (2003) DISTLM <em>forward</em>: a FORTRAN computer program to calculate a distance-based multivariate analysis for a linear model using forward selection. Department of Statistics, University of Auckland, New Zealand.   <ul class="find-nolinks"></ul></li><li><span class="label">55.
              </span><a name="pone.0000454-Zachos1" id="pone.0000454-Zachos1"></a>Zachos JC, Pagani M, Sloan L, Thomas E, Billups K (2001) Trends, rhythms, and aberrations in global climate 65 Ma to present. Science  292: 686–693.  <ul class="find" data-citedArticleID="952614" data-doi="10.1126/science.1059412"><li><a href="http://dx.doi.org/10.1126/science.1059412" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Trends%2C+rhythms%2C+and+aberrations+in+global+climate+65+Ma+to+present." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Trends%2C+rhythms%2C+and+aberrations+in+global+climate+65+Ma+to+present.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.XML" value="104016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.PDF" value="537708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g001.PNG_L" value="2956790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g001.PNG_M" value="471970"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g001.PNG_S" value="12941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g001.TIF" value="4258602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g001.PNG_I" value="186401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g002.PNG_L" value="3112850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g002.PNG_M" value="143897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g002.PNG_S" value="7344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g002.TIF" value="5213012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g002.PNG_I" value="110254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g003.PNG_L" value="856495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g003.PNG_M" value="92966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g003.PNG_S" value="3737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g003.TIF" value="1439222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g003.PNG_I" value="45309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.t001.PNG_L" value="209427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.t001.PNG_M" value="76376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.t001.PNG_S" value="3152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.t001.TIF" value="932704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.t001.PNG_I" value="32198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g004.PNG_L" value="53289"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g004.PNG_M" value="41546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g004.PNG_S" value="2337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g004.TIF" value="218340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.g004.PNG_I" value="19819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.s001.XLS" value="126976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.s002.XLS" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000454.s003.XLS" value="31744"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000454&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000454" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000454&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000454&volume=&issue=&title=Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity&author_name=Yoshan%20Moodley%2C%20Michael%20W.%20Bruford&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000454" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454&amp;t=Molecular%20Biogeography%3A%20Towards%20an%20Integrated%20Framework%20for%20Conserving%20Pan-African%20Biodiversity" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454&title=Molecular%20Biogeography%3A%20Towards%20an%20Integrated%20Framework%20for%20Conserving%20Pan-African%20Biodiversity&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454&amp;title=Molecular%20Biogeography%3A%20Towards%20an%20Integrated%20Framework%20for%20Conserving%20Pan-African%20Biodiversity" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000454&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Molecular Biogeography: Towards an Integrated Framework for Conserving Pan-African Biodiversity';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000454';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Molecular%20Biogeography%3A%20Towards%20an%20Integrated%20Framework%20for%20Conserving%20Pan-African%20Biodiversity http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000454" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000454" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biogeography%22" title="Search for articles in the subject area:'Biogeography'"><div class="flagText">Biogeography</div></a>
              <div data-categoryid="2723" data-articleid="23746"
                   data-categoryname="Biogeography"
                   class="flagImage" title="Flag 'Biogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+ecology%22" title="Search for articles in the subject area:'Evolutionary ecology'"><div class="flagText">Evolutionary ecology</div></a>
              <div data-categoryid="48159" data-articleid="23746"
                   data-categoryname="Evolutionary ecology"
                   class="flagImage" title="Flag 'Evolutionary ecology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Habitats%22" title="Search for articles in the subject area:'Habitats'"><div class="flagText">Habitats</div></a>
              <div data-categoryid="40231" data-articleid="23746"
                   data-categoryname="Habitats"
                   class="flagImage" title="Flag 'Habitats' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplogroups%22" title="Search for articles in the subject area:'Haplogroups'"><div class="flagText">Haplogroups</div></a>
              <div data-categoryid="43141" data-articleid="23746"
                   data-categoryname="Haplogroups"
                   class="flagImage" title="Flag 'Haplogroups' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Macroecology%22" title="Search for articles in the subject area:'Macroecology'"><div class="flagText">Macroecology</div></a>
              <div data-categoryid="48687" data-articleid="23746"
                   data-categoryname="Macroecology"
                   class="flagImage" title="Flag 'Macroecology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleoecology%22" title="Search for articles in the subject area:'Paleoecology'"><div class="flagText">Paleoecology</div></a>
              <div data-categoryid="35111" data-articleid="23746"
                   data-categoryname="Paleoecology"
                   class="flagImage" title="Flag 'Paleoecology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="23746"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Theoretical+ecology%22" title="Search for articles in the subject area:'Theoretical ecology'"><div class="flagText">Theoretical ecology</div></a>
              <div data-categoryid="47903" data-articleid="23746"
                   data-categoryname="Theoretical ecology"
                   class="flagImage" title="Flag 'Theoretical ecology' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2647'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6865'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7553&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16953">African icons</a><br>Posted by BjoernBrembs</p>
      <p><a href="/annotation/listThread.action?root=9097">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
