

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Indexing Strategies for Rapid Searches of Short Words in Genome Sequences</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000579"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000579" />

    <meta name="citation_title" content="Indexing Strategies for Rapid Searches of Short Words in Genome Sequences"/>
    <meta itemprop="name" content="Indexing Strategies for Rapid Searches of Short Words in Genome Sequences"/>

      <meta name="citation_author" content="Christian Iseli"/>
            <meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland"/>
      <meta name="citation_author" content="Giovanna Ambrosini"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland"/>
            <meta name="citation_author_institution" content="Swiss Institute for Experimental Cancer Research, Epalinges, Switzerland"/>
      <meta name="citation_author" content="Philipp Bucher"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland"/>
            <meta name="citation_author_institution" content="Swiss Institute for Experimental Cancer Research, Epalinges, Switzerland"/>
      <meta name="citation_author" content="C. Victor Jongeneel"/>
            <meta name="citation_author_institution" content="Ludwig Institute for Cancer Research, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland"/>
            <meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland"/>

    <meta name="citation_date" content="2007/6/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000579.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e579"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Netaffx: Affymetrix probesets and annotations.; citation_author=G Liu; citation_author=AE Loraine; citation_author=R Shigeta; citation_author=M Cline; citation_author=J Cheng; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=1; citation_pages=82-86; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genbank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res (Database issue); citation_volume=34; citation_number=2; citation_pages=16-20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Serial analysis of gene expression: Probing transcriptomes for molecular targets.; citation_author=A Lal; citation_author=IM Sui; citation_author=G Riggins; citation_journal_title=Current Opinion in Molecular Therapeutics; citation_volume=1; citation_number=3; citation_pages=720-726; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=5′ long serial analysis of gene expression (longsage) and 3′ longsage for transcriptome characterization and genome annotation.; citation_author=CL Wei; citation_author=P Ng; citation_author=KP Chiu; citation_author=CH Wong; citation_author=CC Ang; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=32; citation_number=4; citation_pages=11701-11706; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=In vitro cloning of complex mixtures of dna on microbeads: Physical separation of differentially expressed cdnas.; citation_author=S Brenner; citation_author=SR Williams; citation_author=EH Vermaas; citation_author=T Storck; citation_author=K Moon; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=4; citation_number=5; citation_pages=1665-1670; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=An atlas of human gene expression from massively parallel signature sequencing (mpss).; citation_author=CV Jongeneel; citation_author=M Delorenzi; citation_author=C Iseli; citation_author=D Zhou; citation_author=CD Haudenschild; citation_journal_title=Genome Res; citation_volume=7; citation_number=6; citation_pages=1007-1014; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A greedy algorithm for aligning dna sequences.; citation_author=Z Zhang; citation_author=S Schwartz; citation_author=L Wagner; citation_author=W Miller; citation_journal_title=J Comput Biol; citation_volume=7; citation_number=7; citation_pages=203-214; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ssaha: A fast search method for large dna databases.; citation_author=Z Ning; citation_author=AJ Cox; citation_author=JC Mullikin; citation_journal_title=Genome Res; citation_volume=11; citation_number=8; citation_pages=1725-1729; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The ensembl web site: Mechanics of a genome browser.; citation_author=J Stalker; citation_author=B Gibbins; citation_author=P Meidl; citation_author=J Smith; citation_author=W Spooner; citation_journal_title=Genome Res; citation_volume=5; citation_number=9; citation_pages=951-955; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The UCSC genome browser database: update 2007.; citation_author=RM Kuhn; citation_journal_title=Nucleic Acid Res. (Database issue); citation_volume=35; citation_number=10; citation_pages=668-673; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center fro Biotechnology Information.; citation_author=DL Wheeler; citation_journal_title=Nucleic Acid Res. (Database issue); citation_volume=35; citation_number=11; citation_pages=5-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucleic Acid Res. (Database issue); citation_volume=33; citation_number=12; citation_pages=501-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The art of computer programming.; citation_author=D Knuth; citation_number=13; citation_pages=409-426; citation_date=1997; citation_publisher=Addison-Wesley; " />
      <meta name="citation_reference" content="citation_title=Long-range heterogeneity at the 3′ ends of human mRNAs.; citation_author=C Iseli; citation_author=BJ Stevenson; citation_author=SJ de Souza; citation_author=HB Samaia; citation_author=AA Camargo; citation_journal_title=Genome Res; citation_volume=7; citation_number=14; citation_pages=1068-1074; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Epd in its twentieth year: towards complete promoter coverage of selected model organisms.; citation_author=CD Schmid; citation_author=R Perier; citation_author=V Praz; citation_author=P Bucher; citation_journal_title=Nucleic Acids Res (Database issue); citation_volume=34; citation_number=15; citation_pages=82-85; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cleanex: a database of heterogeneous gene expression data based on a consistent gene nomenclature.; citation_author=V Praz; citation_author=V Jagannathan; citation_author=P Bucher; citation_journal_title=Nucleic Acids Res (Database issue); citation_volume=32; citation_number=16; citation_pages=542-547; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Indexing Strategies for Rapid Searches of Short Words in Genome Sequences"/>
    <meta name="twitter:description" content="
Searching for matches between large collections of short (14&ndash;30 nucleotides) words and sequence databases comprising full genomes or transcriptomes is a common task in biological sequence analysis. We investigated the performance of simple indexing strategies for handling such tasks and developed two programs, fetchGWI and tagger, that index either the database or the query set. Either strategy outperforms megablast for searches with more than 10,000 probes. FetchGWI is shown to be a versatile tool for rapidly searching multiple genomes, whose performance is limited in most cases by the speed of access to the filesystem. We have made publicly available a Web interface for searching the human, mouse, and several other genomes and transcriptomes with oligonucleotide queries.
"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000579.g005"/>

  <meta property="og:title" content="Indexing Strategies for Rapid Searches of Short Words in Genome Sequences" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9802'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8478'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5377&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000579">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000579" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000579&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Indexing Strategies for Rapid Searches of Short Words in Genome Sequences
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Iseli
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Christian.Iseli@licr.org">Christian.Iseli@licr.org</a></p>

                <p>Affiliations:
                  Ludwig Institute for Cancer Research, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland, 
                  Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Giovanna Ambrosini, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland, 
                  Swiss Institute for Experimental Cancer Research, Epalinges, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philipp Bucher, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland, 
                  Swiss Institute for Experimental Cancer Research, Epalinges, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            C. Victor Jongeneel
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Ludwig Institute for Cancer Research, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland, 
                  Swiss Institute of Bioinformatics, Bâtiment Génopode, Université de Lausanne, Lausanne, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 27, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000579</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000579-g001" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000579-g002" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000579-g003" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000579-g004" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000579-g005" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000579">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000579" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Searching for matches between large collections of short (14–30 nucleotides) words and sequence databases comprising full genomes or transcriptomes is a common task in biological sequence analysis. We investigated the performance of simple indexing strategies for handling such tasks and developed two programs, fetchGWI and tagger, that index either the database or the query set. Either strategy outperforms megablast for searches with more than 10,000 probes. FetchGWI is shown to be a versatile tool for rapidly searching multiple genomes, whose performance is limited in most cases by the speed of access to the filesystem. We have made publicly available a Web interface for searching the human, mouse, and several other genomes and transcriptomes with oligonucleotide queries.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Iseli C, Ambrosini G, Bucher P, Jongeneel CV (2007) Indexing Strategies for Rapid Searches of Short Words in Genome Sequences. PLoS ONE 2(6):
          e579.
            doi:10.1371/journal.pone.0000579</p><p><strong>Academic Editor: </strong>Xiaolin Wu, National Cancer Institute at Frederick, United States of America</p><p><strong>Received:</strong> April 30, 2007; <strong>Accepted:</strong> June 5, 2007; <strong>Published:</strong> June 27, 2007</p><p><strong>Copyright:</strong> © 2007 Iseli et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>None, except the normal salary paid by the employer of each author.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>One of the challenges of the post-genomic era is to be able to identify rapidly, in fully sequenced genomes, transcriptomes, and variants thereof, exact or near-exact matches to short words with a limited number of occurrences. Such words could be the sequences of individual probes from Affymetrix<a href="#pone.0000579-Liu1">[1]</a> or other gene chips, PCR primers for the amplification of unique genomic regions (STS<a href="#pone.0000579-Benson1">[2]</a>), tags derived from SAGE<a href="#pone.0000579-Lal1">[3]</a>, <a href="#pone.0000579-Wei1">[4]</a> or MPSS<a href="#pone.0000579-Brenner1">[5]</a>, <a href="#pone.0000579-Jongeneel1">[6]</a> experiments, or the raw data from next-generation high-throughput sequencers (using e.g. the Illumina technology). Applications include probe and PCR primer design, mapping of probe sets to genome features, interpretation of digital gene expression analysis experiments, or individual genome re-sequencing. Ideally, software for word matching should be able to accomplish the following tasks: (i) search very large databases (≫10<sup>9</sup> nucleotides) efficiently; (ii) return accurate and fully exhaustive results for short (14–30 nucleotides) query sequences; (iii) scale up gracefully to large collections of queries (10<sup>6</sup> queries); (iv) provide a mechanism for finding single- or multiple-nucleotide mismatches in alignments that are global relative to the query.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>To our knowledge, no software has yet been designed to precisely match these specifications. Megablast<a href="#pone.0000579-Zhang1">[7]</a>, a variant of the NCBI BLAST suite that uses a word index to reduce the database search space, and then a greedy algorithm to align only highly conserved regions, can be tweaked to efficiently find exact matches to short sequences (see below), although this was not part of its original design. SSAHA<a href="#pone.0000579-Ning1">[8]</a>, which creates a database hash table in memory to accelerate search functions and was thus expected to perform reasonably well, is optimized to find longer alignments and fails in practice on most of the criteria set out above. The indexing strategy used by the PCR primer analysis program of Jim Kent (<a href="http://genome.ucsc.edu/cgi-bin/hgPcr?">http://genome.ucsc.edu/cgi-bin/hgPcr?</a>command = start, <a href="http://www.soe.ucsc.edu/%E2%88%BCkent/src/">http://www.soe.ucsc.edu/~kent/src/</a>) is probably the approach that comes closest to our goals, but it is meant to work with primer pairs and not with independent queries.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>For fast performance, the implementation of SSAHA or BLAT keep the index or the hash table in the memory. They only use non overlapping words and may use repeat masked sequence. Thus for many of the short sequence searches they may not find any or all matches. In contrast, the fetchGWI and tagger programs use a complete indexing strategy for every base in the genome or other databases and thus have higher sensitivity in sacrifice of speed. This is desirable for certain applications.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In order to screen single or multiple genomes with collections of sequence tags of various sizes, we designed two programs with complementary goals and approaches, fetchGWI and tagger. Tagger builds an index in RAM from a set of query sequences, and is particularly well-suited for generating one-time mappings of large collections of tags (typically all probes on a standard Affymetrix chip set or a Nimblegen tiling array, or all tags derived from a SAGE experiment) onto a genome or transcriptome. FetchGWI relies on pre-computed genome indices stored on disk, and is best used in cases where a limited number of queries have to be mapped very rapidly, e.g. as a component of a Web service. Only fetchGWI will be described in detail here, as it performs as well or better than tagger in all tests, provided precomputed genome or transcriptome indices are available.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>For the occasional user, fetchGWI may be the fastest connection from a small piece of a gene sequence to a genome browser window displaying information about the genomic environment of the corresponding gene. To this end, we developed a web interface that provides direct HTML links to Ensembl<a href="#pone.0000579-Stalker1">[9]</a> UCSC<a href="#pone.0000579-Kuhn1">[10]</a>, and NCBI<a href="#pone.0000579-Wheeler1">[11]</a> genome browsers for each exact or near-exact matches to a query target found in a collection of large sequence databases. Sequence databases comprise entire genomes or mRNA reference sequences<a href="#pone.0000579-Pruitt1">[12]</a> (i.e. human, mouse, drosophila, dog, rat and chimpanzee). This web service can be used interactively, or via sequence tag-associated hyperlinks from within text documents.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Design Considerations</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Genome sequences are high-quality, so we need to consider only the four unambiguous nucleotides A, C, G, and T. The information necessary to represent those 4 nucleotides can be encoded on 2 bits of data. Current processors are built to easily handle 64-bit words, so it is straightforward to encode up to 32 nucleotides within a single data word.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To get maximal search speed, we want to search only within the index file. Thus we need one index entry for each nucleotide in the genome. This exhaustive index also ensures that no match can possibly be missed, as can happen when only non overlapping words are used to construct the index.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To further improve search speed within the index file, we also implemented a second format, which we call a compressed index file, where the primary index file is subdivided into 16,777,216 (2<sup>24</sup>) parts according to the first 12 nucleotides of each word. The compressed format also allows a 25% size reduction of the index file.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The method is divided into two separate steps: generation of the index files, and actual sequence search. The generation of index files needs to be performed once for each new version of a genome or transcriptome. The search step can be repeated as often as necessary once the index files have been generated.</p>


<h4>Index Files Structure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The main index file is composed of a sorted list of entries. Each entry is composed of four parts: a sequence tag, two flags, a sequence offset, and an accession number index.</p>
<h5>sequence tag.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>The first and largest part is the sequence tag, which is a binary encoding of a piece of DNA sequence. Each nucleotide is encoded using two bits. The optimal size of the sequence tag is a compromise between two opposing goals: while longer sequence tags are more specific, and less likely to occur at multiple positions in the genome, shorter sequence tags make for smaller index files and shorter search times. We analyzed the percentage of unique sequence tags, in a number of complete genomes, as a function of tag length (<a href="#pone-0000579-g001">Fig. 1</a>). The data show that all curves have an inflection point located between 15 and 20 nucleotides. For example, in the human genome most tags shorter than 16 nucleotides are not unique as expected from the fact that the size of the human genome is comprised between 4<sup>15</sup> and 4<sup>16</sup>. We also took into account the fact that many current methodologies (SAGE, MPSS, Affymetrix or Nimblegen genome arrays, PCR primers, etc.) use tags where the size ranges in length from 17 to 25 nucleotides. We thus decided to use a tag size of 25 nucleotides or 50 bits of sequence.</p>
<div class="figure" id="pone-0000579-g001"><div class="img"><a name="pone-0000579-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Word duplication in several genomes.</span></strong></p><a id="article1.body1.sec2.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec2.sec1.fig1.caption1.p1"></a><p>Analysis of the percentage of unique sequence tags in several genomes as a function of tag length.</p>
<span>doi:10.1371/journal.pone.0000579.g001</span></div>
<h5>flags.</h5><a id="article1.body1.sec2.sec2.sec2.p1" name="article1.body1.sec2.sec2.sec2.p1"></a><p>The second component of an index entry is two flags: one marking incomplete sequence tags, the other selecting short or long sequence offset size.</p>
<a id="article1.body1.sec2.sec2.sec2.p2" name="article1.body1.sec2.sec2.sec2.p2"></a><p>While each sequence tag is 25 nucleotides long, we would still like to allow shorter queries, with possible matches at the ends of database sequences. This causes a problem near gaps and sequence ends, because sub-words within the last 25 nucleotides would not be indexed unless we allow for the indexing of incomplete (i.e., shorter than 25) sequence tags. In this context, a gap refers to a single or consecutive run of ambiguous nucleotide characters, usually N's. Note that in the current genome sequence data files, short gaps of unsequenced regions are represented by runs of 50–100 N's. The flag is here to mark such shortened index entries. The smallest query allowed is 10 nucleotides long. To be more specific, such a word will be padded by a run of complemented nucleotides of the last valid nucleotide to make it a valid 25-letter word, and the incomplete flag will be set. For example, the 10-mer <span class="gene">ATGGCTGAGG</span> will be stored as <span class="gene">ATGGCTGAGGCCCCCCCCCCCCCCC</span>, the flag will be set, and it is easy to find back the original word by removing all occurrences of the repeated last nucleotide.</p>
<a id="article1.body1.sec2.sec2.sec2.p3" name="article1.body1.sec2.sec2.sec2.p3"></a><p>We have analyzed the genomic and mRNA content of the RefSeq database, release 22. The longest sequence is 748,055,161 nucleotides long (chromosome 1 from the gray short-tailed opossum), so at least 30 bits are needed to specify an offset within such a large sequence. However, most sequences are shorter. There are 1,769,766 sequences in the analyzed release, and only 5,330 are longer than 1,048,576 nucleotides. To help decrease the size of the index, we decided to use a flag to distinguish between sequences which need large offset values but are few in numbers, and sequences which can use a shorter offset but are more numerous.</p>

<h5>sequence offset.</h5><a id="article1.body1.sec2.sec2.sec3.p1" name="article1.body1.sec2.sec2.sec3.p1"></a><p>The third index component is the offset of the sequence tag within the sequence. As noted in the previous paragraph, the longest sequence currently known is roughly 750 megabases long so we need 30 bits to encode the long offsets. The short offset uses 20 bits and can thus be used for sequences up to 1,048,576 nucleotides long.</p>

<h5>accession number index.</h5><a id="article1.body1.sec2.sec2.sec4.p1" name="article1.body1.sec2.sec2.sec4.p1"></a><p>The last component is used to retrieve the accession number (or other unique identifier) of the sequence from which the sequence tag was derived. A list of all the accession numbers of the indexed sequences is kept in an auxiliary index file. This auxiliary file is a plain text file, where each sequence accession number is given a unique index number, sequentially, starting from 0 for long sequences and from 16,384 for the shorter ones. We have set aside 14 bits to encode the index number for long sequences, and 24 bits for shorter sequences. We can thus accommodate 16,384 long sequences and 16,760,832 short sequences, which is currently more than enough for all genomes and transcriptomes found in RefSeq and other (e.g. FANTOM, H-INV) collections.</p>
<a id="article1.body1.sec2.sec2.sec4.p2" name="article1.body1.sec2.sec2.sec4.p2"></a><p>The complete index entry uses 96 bits, or 12 bytes, of storage. Given that the human genome is roughly 3 billion nucleotides long, the index file for this genome will occupy about 30 gigabytes of storage space. With today's hard disks storing 500 gigabytes apiece, this is a manageable size.</p>



<h4>Compressed Index Files</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The idea behind the compressed index is to diminish the size of the index files and to improve filesystem locality when searching multiple words. Filesystem locality can improve the search speed by reducing the amount of time spent seeking between different parts of the hard disks and by taking advantage of data caching. The whole index file is logically subdivided into 16,777,216 parts, each part representing words having identical first 12 nucleotides. A secondary index, recording the position in the main index file where each part starts, is added to the end of the index file.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As an added benefit, since the first 12 nucleotides of each word are given through the secondary index, the first 3 bytes of each entry in the main index can be suppressed thus saving 25% of storage space. The space required to store the secondary index is 128 megabytes, which is negligible.</p>


<h4>Index File Generation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The generation of an index file is performed in two steps:</p>


<ol class="order">

<li>the genwin program is used to transform a set of FASTA formatted files, containing nucleotide sequences, into a set of files containing unsorted index entries, in the format explained in the previous paragraphs. The generated files are not sorted nor compressed. They must be sorted using the sortGWI program before being usable by fetchGWI. Short words down to 10 nucleotides long are indexed, using the incomplete flag explained above</li>

<li>the sortGWI program is used to sort and/or merge the files generated in step 1 into a file containing sorted index entries; the sortGWI program can generate and use both plain and compressed index files</li>

</ol>

<h4>Searching Through the Index Files</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Finding where a query matches in the genomic sequence is accomplished through the fetchGWI program. This program performs the following steps:</p>


<ol class="order">

<li>collect all the queries to be searched, either from command line arguments or from text files containing one query per line. Each query line must contain a sequence tag to be searched at the beginning of the line, optionally followed by a non-alphabetical character and arbitrary data on the rest of the line, which will be copied verbatim in the output. The sequence tag can contain degenerate nucleotides (the standard letters BDHKMNRSVWY are accepted), which will be automatically expanded into all possible matching tags</li>

<li>if the user specified a search with one or several mismatches, generate all possible sequence tags to be searched by replacing the specified number of non-degenerate nucleotides with all other possibilities</li>

<li>add all the reverse complemented queries, unless the user specified otherwise, so that the search is performed on both strands</li>

<li>split queries longer than 25 nucleotides into 25 nucleotide sub-queries, keeping proper linking with the original queries. Queries shorter than 25 nucleotides are padded on the right with the nucleotide A to 25 nucleotides and a mask is generated to allow proper comparison with the index by masking out the unused nucleotides on the end of both the query and the index</li>

<li>sort all the queries using the same sorting order as the index files, so that we can benefit from better filesystem locality when performing the search</li>

<li>map the secondary index structure in memory (when using compressed index files)</li>

<li>perform the search of each query within the index file, using a dichotomic search (also known as binary search) technique<a href="#pone.0000579-Knuth1">[13]</a> and collect all the matches for each query. When using compressed index files, a lookup in the secondary index is performed to determine the boundaries of the dichotomy search within the main index. The index is masked to the proper length during each comparison for queries shorter than 25 nucleotides</li>

<li>for queries longer than 25 nucleotides, analyze the matches of the sub-queries and keep only those compatible with the original query. This is performed by sorting all the matches by queries, and examining all the sub-queries to find those occurring in the proper arrangement</li>

<li>report the results by appending the actual sequence tag found, along with the accession number and position offset within the sequence for each matched query lines</li>

</ol><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>It is possible to search short queries down to 10 nucleotides long exhaustively, since short words down to 10 nucleotides are indexed. It is possible to force fetchGWI to seek also shorter queries, but in this case some words can be missed near gaps and sequence ends.</p>


<h4>The tagger Program</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>There are cases where constructing the index file itself is considered too time consuming, usually because it would be used only once. For these cases, we also supply a program named tagger which uses a similar indexing strategy, but in the case of tagger the index is built from the queries instead of the genome, and is only kept in RAM. The word size is 13 nucleotides and the index table uses 512 megabytes of memory. Each tag to be searched is entered in the table, and they are kept in a linked list when several tags are attached to the same index location in the table. This is a very simple hashing technique. The tagger program then parses through FASTA formatted files containing the sequences to search and reports matches as it finds them. All searched tags must be the same length and cannot contain ambiguous nucleotides, which are further restrictions compared with the fetchGWI program.</p>


<h4>Tested programs</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To test and validate our approach, seeking only exact matches for the queries, we compared the output results and execution speed of our fetchGWI and tagger tools with the results obtained by megablast.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>It should be noted that searches using megablast produced two different results:</p>


<ol class="order">

<li>the query <span class="gene">TTGTGTTGTGTTGTGTTGTGTTGTG</span> found only 58 matches whereas 94 are expected on the current assembly of the human genome. This is probably because megablast reports non-overlapping matches, which is a matter of choice – note that the above sequence is a repeat of the 5-letter word GTGTT</li>

<li>the query <span class="gene">TGAATTCGGTCTTGCCTTGAACACA</span> found a spurious perfect match on the genomic sequence NGAATTCGGTCTTGCCTTGAACACA.</li>

</ol><a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Except for these minor problems, all three tools reported the same hits. The version of megablast was 2.2.13. The parameters used for the megablast runs were the following:</p>


<ol class="order">

<li>-f T -J F -F F -W 12 -s 25 -D 0</li>

</ol><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>We also attempted to perform the same searches using SSAHA (Version 3.2), but it only would match 24 of the 25 nucleotides of the query sequences (the reason for this behavior is unknown to the authors).</p>
<a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>We then proceeded to test results when seeking possibly inexact matches for the queries. However, neither megablast, nor blast, were able to retrieve all correct hits reliably. For example, the sequence <span class="gene">ATGGCTGAAGGCCTTATGAGTCAAA</span> has one exact match on human chromosome 12 (NC_000012.10[10029321..10029345]) and 5 inexact matches (2 mismatched nucleotides: <span class="gene">ATGGCTGA<span class="underline">G</span>GGCCTTA<span class="underline">A</span>GAGTCAAA</span>) on human chromosomes 5, 10, 11, X, and Y. FetchGWI finds all matches in about 10 seconds Megablast only finds the exact match and 2 of the 5 inexact ones in roughly 6 minutes, while blast finds the exact match only (in about 10 seconds). The parameters used for megablast in this case were:</p>


<ol class="order">

<li>-f T -J F -F F -W 8 -s 21 -D 0 -q -1 -G 10 -E 4</li>

</ol><a id="article1.body1.sec2.sec7.p6" name="article1.body1.sec2.sec7.p6"></a><p>The fetchGWI and tagger results of <a href="#pone-0000579-g002">Fig. 2</a>, <a href="#pone-0000579-g003">Fig. 3</a>, and <a href="#pone-0000579-g004">Fig. 4</a> were computed on a Linux workstation with Intel Xeon processors running at 3 GHz, 4 GB RAM, and a local SATA hard disk of 250 GB. The megablast results were computed on a Linux workstation with Intel Xeon processors running at 3.4 GHz in em64t mode, 8 GB RAM, and local hard disks of 128 GB in striped RAID mode. The fetchGWI results on the SFS (a commercial implementation of LUSTRE from Cluster file systems, inc., <a href="http://www.lustre.org/">http://www.lustre.org/</a>.) filesystem (<a href="#pone-0000579-g005">Fig. 5</a>) were computed on a Linux workstation with Intel Itanium2 processors running at 1.3 GHz, 4 GB RAM, and an 8 TB SFS filesystem attached through an InfiniBand network.</p>
<div class="figure" id="pone-0000579-g002"><div class="img"><a name="pone-0000579-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Runtime comparisons on the human genome.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Runtime comparisons between fetchGWI using plain and compressed index files, tagger, and megablast. Each point is computed from the average of three runs on the human genome with different input data, except the last run done on the whole dataset. Only perfect matches are sought, except for the 2 experiments explicitly noted were 2 mismatches were allowed.</p>
<span>doi:10.1371/journal.pone.0000579.g002</span></div><div class="figure" id="pone-0000579-g003"><div class="img"><a name="pone-0000579-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Runtime comparisons on multiple genomes.</span></strong></p><a id="article1.body1.sec2.sec7.fig2.caption1.p1" name="article1.body1.sec2.sec7.fig2.caption1.p1"></a><p>Runtime comparisons between fetchGWI (using a compressed index file) and megablast. Each point is computed from the average of three runs on the combined genome of 9 species (human, mouse, honey bee, cattle, dog, drosophila, zebrafish, chimp, and rat) with different input data, except the last run done on the whole dataset. Only perfect matches were sought.</p>
<span>doi:10.1371/journal.pone.0000579.g003</span></div><div class="figure" id="pone-0000579-g004"><div class="img"><a name="pone-0000579-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Runtime comparisons on combined index files.</span></strong></p><a id="article1.body1.sec2.sec7.fig3.caption1.p1" name="article1.body1.sec2.sec7.fig3.caption1.p1"></a><p>Runtime comparisons of fetchGWI when using either multiple index files, or a single, combined, index file. Each point is computed from the average of three runs on the human and mouse genomes with different input data, except the last run done on the whole dataset Only perfect matches were sought.</p>
<span>doi:10.1371/journal.pone.0000579.g004</span></div><div class="figure" id="pone-0000579-g005"><div class="img"><a name="pone-0000579-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000579" data-uri="info:doi/10.1371/journal.pone.0000579.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000579.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000579.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000579.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000579.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Runtime comparisons on different filesystems.</span></strong></p><a id="article1.body1.sec2.sec7.fig4.caption1.p1" name="article1.body1.sec2.sec7.fig4.caption1.p1"></a><p>Runtime comparisons of fetchGWI when the index file is stored either on a filesystem on local disks, or on a SFS cluster filesystem. Each point is computed from the average of three runs with different input data, except the last run done on the whole dataset. Only perfect matches were sought.</p>
<span>doi:10.1371/journal.pone.0000579.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>To examine the behavior of the fetchGWI and the tagger programs, we ran queries with variable numbers of tag sequences on the human and mouse genomes. Only exact matches were sought, except for two experiments reported in <a href="#pone-0000579-g002">Fig. 2</a>. For building the queries, we used the probes from the Affymetrix “Human Genome U133 Plus 2.0” array. The reason for choosing this probe set is that it should offer a pretty good coverage of the whole human genome, and it contains a reasonably sized set of 604,258 tags.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The test queries were generated in the following way:</p>


<ul class="bulletlist">

<li>take the first three probes, and create three files with a single tag</li>

<li>take the next thirty probes, and create three files of ten tags</li>

<li>continue the procedure up to three files of one hundred thousand tags</li>

<li>the whole 604,258 tags file is also used as the final test</li>

</ul><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Each query set (three files each containing 1 to 100,000 tags, and one with 604,258 tags) was searched against assembly 36 of the human genome, using fetchGWI, tagger, and megablast.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The average wall clock run time for the searches is plotted in <a href="#pone-0000579-g002">Fig. 2</a>. The CPU time needed by fetchGWI is only a few percent of the wall clock time; most of the time needed is spent waiting for data to be retrieved from the filesystem. The CPU time needed by both tagger and megablast is nearly the same as the total wall clock time; most of the time needed is spent doing computations. As expected, the search time of tagger is essentially independent of the number of queries; however, it is somewhat surprising that tagger never outperformed fetchGWI, even for very large query sets. The use of compressed index speeds up the search 3- to 4-fold for small number of tags, and is still slightly faster for large number of tags. It is also worth noting that for query sets with 1000 tags, megablast performs almost as well as fetchGWI, provided the correct command-line switches are used. However, fetchGWI is over 10-fold faster than megablast for mapping all 604,258 probes of the Hu133 chip; larger query sets such as those found on genome-wide tiling arrays are expected to show an even larger advantage for fetchGWI. The speed advantage of fetchGWI is even slightly better when searching through several genomes at once, as shown in <a href="#pone-0000579-g003">Fig. 3</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>We also show in <a href="#pone-0000579-g002">Fig. 2</a> the run time of fetchGWI when allowing 2 mismatches, and note that it increases by a factor between 10 and 100. The search space is much larger, since it will grow as the binomial coefficient of n and k, where n is the number of nucleotides in the query and k is the number of mismatches allowed. For example, searching for one query of 25 nucleotides and allowing 2 mismatches is equivalent to searching for 2,776 queries with no mismatches. During the tests we performed, the maximum amount of RAM used when searching for 1000 queries and allowing 2 mismatches was 550 megabytes on a 32-bit machine and 700 megabytes on a 64-bit machine.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The run time of a dichotomic search is <em>O</em>(ln<em>n</em>), where <em>n</em> is the number of elements in the database. When the number of queries gets large, some additional time will be spent sorting the queries themselves, where the standard Unix quicksort routine is used and has a run time of <em>O</em>(<em>n</em> · ln<em>n</em>) where <em>n</em> is the number of queries. What we observe in the results is the large influence of filesystem data caching by the operating system. For example, repeating a search containing 1000 queries brings the wall clock time down from around 90 seconds to 0.15 second. On a machine with sufficient amounts of RAM, even very large searches can be performed in less than 1 second once the data have been cached. When the set of queries gets large, the same data caching effect starts to show (flattening of the curve with query sets of more than 10,000). The reason is that fetchGWI sorts the queries, and thus the denser the hits get on the genome, the higher the chance that the next query to be searched corresponds to index data already loaded in RAM.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>One of our goals was to provide a quick means to search for matches across all known genomes, and this raises the question of whether to keep each genome in its own index file, or to produce index files of combined genomes. The behavior of fetchGWI has been plotted in <a href="#pone-0000579-g004">Fig. 4</a> for the following two cases:</p>


<ol class="order">

<li>produce a combined index file for the human and mouse genomes, which has a size of 50 gigabytes and takes around 10 minutes to produce from the individual index files of the human and mouse genomes</li>

<li>search first through the human genome index file, then repeat the search for the mouse genome</li>

</ol><a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p><a href="#pone-0000579-g004">Figure 4</a> shows that the performance gain for combining the two indices is minimal. Again the influence of caching is apparent: for large query sets, the difference disappears. For small query sets, the <em>O</em>(ln<em>n</em>) behavior of the dichotomic search is the key.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Dealing with very large files is not an easy task, and it makes sense to use computer clusters to speed things up further. We therefore analyzed the behavior of fetchGWI on a server that gets its data from a distributed SFS cluster filesystem, to see how such a system would cope with multiple random accesses distributed through large files. <a href="#pone-0000579-g005">Figure 5</a> shows that the results are very similar to those measured for local hard disks. This indicates that multiple instances of fetchGWI should be able to run efficiently on a modern compute cluster with LUSTRE-based file sharing.</p>

<h4>Availability and requirements</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We have created a web portal, TagScan at <a href="http://www.isrec.isb-sib.ch/tagger">http://www.isrec.isb-sib.ch/tagger</a>, for rapid mapping of short oligo sequences to large sequence databases comprising full genomes or transcriptomes (i.e. full length mRNA sequences from human, mouse, bee, drosophila, dog, rat, cow, and chimpanzee). The web server implementation responds to the need of having a page that lets the user screen vertebrate genomes or transcriptomes with oligo sequences (on average 19–25 nucleotides long) and returns both perfect matches and one or two nucleotide mismatches. This could certainly be a very useful service for people designing PCR primers or oligos for custom arrays. Other potential uses could include tasks such as remapping large number of probe sequences from SNP or tiling arrays to new assemblies of the human or other genomes. The TagScan web server supports searches for perfect matches and for single or double nucleotide mismatches. Along with word matches, genomic or transcript coordinates are returned. For genome-wide searches, useful hyperlinks to popular genome browsers such as the Ensembl, UCSC and NCBI ones, are also provided.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Alternatively, the TagScan programs can be invoked directly by using tag-associated hyperlinks from within text documents. The hyperlink format is shown below:</p>


<ol class="order">

<li><a href="http://www.isrec.isb-sib.ch/cgi-bin/tagger/tagscan?">http://www.isrec.isb-sib.ch/cgi-bin/tagg​er/tagscan?</a>dbtype = dna&amp;dbname = HS&amp;mode = 0&amp;tag = ATGAGGTATTAGGAT</li>

</ol><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Further details on the use of inline hyperlinks are available from the web page.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The TagScan inline URL service provides an easy and elegant mechanism for specialized databases to hyper-link sequence-tagged features to genome browsers without explicitly providing chromosomal coordinates. The advantage of this on-the-fly mapping mechanism is that it doesn't require recalculation of genome coordinates for new assemblies by the client database.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>The source code is freely available on the SourceForge server:</p>


<ul class="bulletlist">

<li>Project name: Tagger</li>

<li>Project home page: <a href="http://sourceforge.net/projects/tagger">http://sourceforge.net/projects/tagger</a></li>

<li>Operating system(s): All POSIX</li>

<li>Programming language: C</li>

<li>Other requirements: none</li>

<li>License: GNU GPL</li>

</ul><a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>The distribution also contains a user manual in the form of man pages for the provided programs. There is a description of the index structures and access functions for prospective developers in the source code itself.</p>


<h4>Conclusions and Perspectives</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Tagger and fetchGWI were originally developed to support research and development programs at our parent institutions. The first application was to find sequences proximal to EST-derived polyadenylation sites in the human genome<a href="#pone.0000579-Iseli1">[14]</a>. Tagger was subsequently used to remap the eukaryotic promoters in EPD<a href="#pone.0000579-Schmid1">[15]</a> to new assemblies of the human and other genomes with the aid of unique sequence tags of length 60, and to reliably associate “historically annotated” Affymetrix probe sets with newly annotated genes and transcripts<a href="#pone.0000579-Praz1">[16]</a>. It was also essential in producing a reliable annotation for human and mouse MPSS signatures<a href="#pone.0000579-Jongeneel1">[6]</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The programs were designed for speed and robustness rather than for elegance or flexibility. They have been extensively field-tested, and shown to produce accurate results very rapidly. The results shown here indicate that their main strength lies in the matching of very large probe collections to one or more genomes. As such, they may become the sequence similarity search engines of choice for developers of complex arrays of probes while keeping track of issues of within-species and cross-species hybridization. We also found fetchGWI extremely useful for tracking the source(s) of contaminants in biological samples.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>We have for the first time presented benchmark results to assess the efficiencies of different indexing strategies for rapid exact sequence matching in realistic settings. Our results indicate that a compressed sorted word-index accessed by dichotomic search outperforms other approaches for mapping large collections of short tags to large genomes by an exact match criterion.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>The fetchGWI and tagger programs could serve as search engines for a large variety of other applications. We are currently developing a tool using a modified version of fetchGWI for rapid location of weight matrix-defined transcription factor binding sites in whole genomes. The underlying principle is to expand the user supplied weight matrix to the complete set of k-words that match the matrix with scores equal to or higher than a threshold value. A virtually infinite number of heuristic sequence similarity search algorithms could use a fetchGWI-like mechanism as a first filtering step. The index structures and search algorithms described here are straightforward to use by application programmers. The benchmarks presented will enable interested developers to judge whether a fetchGWI type index structure could provide an efficient solution to a specific problem arising in a whole genome or transcriptome scan application.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the Ludwig Institute for Cancer Research (<a href="http://www.licr.org">http://www.licr.org</a>), the Swiss Institute of Bioinformatics (<a href="http://www.isb-sib.ch">http://www.isb-sib.ch</a>) and the Swiss Institute for Experimental Cancer Research (<a href="http://www.isrec.ch">http://www.isrec.ch</a>) for their continued support. The computations were performed in part at the Vital-IT (<a href="http://www.vital-it.ch">http://www.vital-it.ch</a>) Center for high-performance computing of the Swiss Institute of Bioinformatics.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Performed the experiments: CI GA. Analyzed the data: CI. Wrote the paper: CJ CI. Other: Made the initial proposal to develop this software, contributed useful ideas regarding its implementation, provided constructive criticisms on the manuscript: PB. Designed and wrote the code for fetchGWI, sortGWI, and genwin, designed and wrote the code for tagger together with GA: CI. Designed and wrote the code for tagger together with CI, implemented the Web interface: GA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000579-Liu1" id="pone.0000579-Liu1"></a>Liu G, Loraine AE, Shigeta R, Cline M, Cheng J, et al.  (2003) Netaffx: Affymetrix probesets and annotations. Nucleic Acids Res  31: 82–86.  <ul class="find" data-citedArticleID="963174" data-doi="10.1093/nar/gkg121"><li><a href="http://dx.doi.org/10.1093/nar/gkg121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Netaffx%3A+Affymetrix+probesets+and+annotations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Netaffx%3A+Affymetrix+probesets+and+annotations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000579-Benson1" id="pone.0000579-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2006) Genbank. Nucleic Acids Res (Database issue)  34: 16–20.  <ul class="find" data-citedArticleID="963158" data-doi="10.1093/nar/gkm929"><li><a href="http://dx.doi.org/10.1093/nar/gkm929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genbank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genbank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000579-Lal1" id="pone.0000579-Lal1"></a>Lal A, Sui IM, Riggins G (1999) Serial analysis of gene expression: Probing transcriptomes for molecular targets. Current Opinion in Molecular Therapeutics  1: 720–726.  <ul class="find" data-citedArticleID="963188"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Serial+analysis+of+gene+expression%3A+Probing+transcriptomes+for+molecular+targets.&amp;auth=&amp;atitle=Serial+analysis+of+gene+expression%3A+Probing+transcriptomes+for+molecular+targets." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serial+analysis+of+gene+expression%3A+Probing+transcriptomes+for+molecular+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serial+analysis+of+gene+expression%3A+Probing+transcriptomes+for+molecular+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000579-Wei1" id="pone.0000579-Wei1"></a>Wei CL, Ng P, Chiu KP, Wong CH, Ang CC, et al.  (2004) 5′ long serial analysis of gene expression (longsage) and 3′ longsage for transcriptome characterization and genome annotation. Proc Natl Acad Sci USA  32: 11701–11706.  <ul class="find" data-citedArticleID="963166" data-doi="10.1073/pnas.0403514101"><li><a href="http://dx.doi.org/10.1073/pnas.0403514101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=5%E2%80%B2+long+serial+analysis+of+gene+expression+%28longsage%29+and+3%E2%80%B2+longsage+for+transcriptome+characterization+and+genome+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%225%E2%80%B2+long+serial+analysis+of+gene+expression+%28longsage%29+and+3%E2%80%B2+longsage+for+transcriptome+characterization+and+genome+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000579-Brenner1" id="pone.0000579-Brenner1"></a>Brenner S, Williams SR, Vermaas EH, Storck T, Moon K, et al.  (2000) In vitro cloning of complex mixtures of dna on microbeads: Physical separation of differentially expressed cdnas. Proc Natl Acad Sci USA  4: 1665–1670.  <ul class="find" data-citedArticleID="963162" data-doi="10.1073/pnas.97.4.1665"><li><a href="http://dx.doi.org/10.1073/pnas.97.4.1665" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+cloning+of+complex+mixtures+of+dna+on+microbeads%3A+Physical+separation+of+differentially+expressed+cdnas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+cloning+of+complex+mixtures+of+dna+on+microbeads%3A+Physical+separation+of+differentially+expressed+cdnas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000579-Jongeneel1" id="pone.0000579-Jongeneel1"></a>Jongeneel CV, Delorenzi M, Iseli C, Zhou D, Haudenschild CD, et al.  (2005) An atlas of human gene expression from massively parallel signature sequencing (mpss). Genome Res  7: 1007–1014.  <ul class="find" data-citedArticleID="963180" data-doi="10.1101/gr.4041005"><li><a href="http://dx.doi.org/10.1101/gr.4041005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+atlas+of+human+gene+expression+from+massively+parallel+signature+sequencing+%28mpss%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+atlas+of+human+gene+expression+from+massively+parallel+signature+sequencing+%28mpss%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000579-Zhang1" id="pone.0000579-Zhang1"></a>Zhang Z, Schwartz S, Wagner L, Miller W (2000) A greedy algorithm for aligning dna sequences. J Comput Biol  7: 203–214.  <ul class="find" data-citedArticleID="963176" data-doi="10.1089/10665270050081478"><li><a href="http://dx.doi.org/10.1089/10665270050081478" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+greedy+algorithm+for+aligning+dna+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+greedy+algorithm+for+aligning+dna+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000579-Ning1" id="pone.0000579-Ning1"></a>Ning Z, Cox AJ, Mullikin JC (2001) Ssaha: A fast search method for large dna databases. Genome Res  11: 1725–1729.  <ul class="find" data-citedArticleID="963170" data-doi="10.1101/gr.194201"><li><a href="http://dx.doi.org/10.1101/gr.194201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ssaha%3A+A+fast+search+method+for+large+dna+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ssaha%3A+A+fast+search+method+for+large+dna+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000579-Stalker1" id="pone.0000579-Stalker1"></a>Stalker J, Gibbins B, Meidl P, Smith J, Spooner W, et al.  (2004) The ensembl web site: Mechanics of a genome browser. Genome Res  5: 951–955.  <ul class="find" data-citedArticleID="963184" data-doi="10.1101/gr.1863004"><li><a href="http://dx.doi.org/10.1101/gr.1863004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ensembl+web+site%3A+Mechanics+of+a+genome+browser." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ensembl+web+site%3A+Mechanics+of+a+genome+browser.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000579-Kuhn1" id="pone.0000579-Kuhn1"></a>Kuhn RM, et al.  (2007) The UCSC genome browser database: update 2007. Nucleic Acid Res. (Database issue)  35: 668–673.  <ul class="find" data-citedArticleID="963182" data-doi="10.1093/nar/gkl928"><li><a href="http://dx.doi.org/10.1093/nar/gkl928" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+genome+browser+database%3A+update+2007." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+genome+browser+database%3A+update+2007.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000579-Wheeler1" id="pone.0000579-Wheeler1"></a>Wheeler DL, et al.  (2007) Database resources of the National Center fro Biotechnology Information. Nucleic Acid Res. (Database issue)  35: 5–12.  <ul class="find" data-citedArticleID="963164" data-doi="10.1093/nar/gki062"><li><a href="http://dx.doi.org/10.1093/nar/gki062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+fro+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+fro+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000579-Pruitt1" id="pone.0000579-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2005) NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acid Res. (Database issue)  33: 501–504.  <ul class="find" data-citedArticleID="963160" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000579-Knuth1" id="pone.0000579-Knuth1"></a>Knuth D (1997) The art of computer programming. Sorting and Searching. Boston: Addison-Wesley. pp. 409–426.  volume&nbsp;3. 3rd edition,.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000579-Iseli1" id="pone.0000579-Iseli1"></a>Iseli C, Stevenson BJ, de&nbsp;Souza SJ, Samaia HB, Camargo AA, et al.  (2002) Long-range heterogeneity at the 3′ ends of human mRNAs. Genome Res  7: 1068–1074.  <ul class="find" data-citedArticleID="963178" data-doi="10.1101/gr.62002"><li><a href="http://dx.doi.org/10.1101/gr.62002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-range+heterogeneity+at+the+3%E2%80%B2+ends+of+human+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-range+heterogeneity+at+the+3%E2%80%B2+ends+of+human+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000579-Schmid1" id="pone.0000579-Schmid1"></a>Schmid CD, Perier R, Praz V, Bucher P (2006) Epd in its twentieth year: towards complete promoter coverage of selected model organisms. Nucleic Acids Res (Database issue)  34: 82–85.  <ul class="find" data-citedArticleID="963172" data-doi="10.1093/nar/gkj146"><li><a href="http://dx.doi.org/10.1093/nar/gkj146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epd+in+its+twentieth+year%3A+towards+complete+promoter+coverage+of+selected+model+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epd+in+its+twentieth+year%3A+towards+complete+promoter+coverage+of+selected+model+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000579-Praz1" id="pone.0000579-Praz1"></a>Praz V, Jagannathan V, Bucher P (2004) Cleanex: a database of heterogeneous gene expression data based on a consistent gene nomenclature. Nucleic Acids Res (Database issue)  32: 542–547.  <ul class="find" data-citedArticleID="963186" data-doi="10.1093/nar/gkh107"><li><a href="http://dx.doi.org/10.1093/nar/gkh107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cleanex%3A+a+database+of+heterogeneous+gene+expression+data+based+on+a+consistent+gene+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cleanex%3A+a+database+of+heterogeneous+gene+expression+data+based+on+a+consistent+gene+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.XML" value="57464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.PDF" value="254376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g001.PNG_L" value="228288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g001.PNG_M" value="253006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g001.PNG_S" value="13030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g001.TIF" value="758830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g001.PNG_I" value="36597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g002.PNG_L" value="163296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g002.PNG_M" value="219398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g002.PNG_S" value="13776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g002.TIF" value="629348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g002.PNG_I" value="33130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g003.PNG_L" value="311564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g003.PNG_M" value="137406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g003.PNG_S" value="10540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g003.TIF" value="696918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g003.PNG_I" value="20578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g004.PNG_L" value="97703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g004.PNG_M" value="134357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g004.PNG_S" value="9691"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g004.TIF" value="517808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g004.PNG_I" value="20424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g005.PNG_L" value="313865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g005.PNG_M" value="128588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g005.PNG_S" value="9503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g005.TIF" value="701612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000579.g005.PNG_I" value="18904"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000579&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000579" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000579&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000579&volume=&issue=&title=Indexing Strategies for Rapid Searches of Short Words in Genome Sequences&author_name=Christian%20Iseli%2C%20Giovanna%20Ambrosini%2C%20Philipp%20Bucher%2C%20C.%20Victor%20Jongeneel&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000579" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579&amp;t=Indexing%20Strategies%20for%20Rapid%20Searches%20of%20Short%20Words%20in%20Genome%20Sequences" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579&title=Indexing%20Strategies%20for%20Rapid%20Searches%20of%20Short%20Words%20in%20Genome%20Sequences&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579&amp;title=Indexing%20Strategies%20for%20Rapid%20Searches%20of%20Short%20Words%20in%20Genome%20Sequences" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000579&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Indexing Strategies for Rapid Searches of Short Words in Genome Sequences';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000579';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Indexing%20Strategies%20for%20Rapid%20Searches%20of%20Short%20Words%20in%20Genome%20Sequences http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000579" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000579" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23996"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23996"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="23996"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="23996"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23996"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="23996"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23996"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+tagged+site+analysis%22" title="Search for articles in the subject area:'Sequence tagged site analysis'"><div class="flagText">Sequence tagged site analysis</div></a>
              <div data-categoryid="48851" data-articleid="23996"
                   data-categoryname="Sequence tagged site analysis"
                   class="flagImage" title="Flag 'Sequence tagged site analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1568'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3999'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1636&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
