

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001905"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001905" />

    <meta name="citation_title" content="Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice"/>
    <meta itemprop="name" content="Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice"/>

      <meta name="citation_author" content="Takahiro Suzuki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
            <meta name="citation_author_institution" content="Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan"/>
      <meta name="citation_author" content="Shinji Kondo"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Teruhiko Wakayama"/>
            <meta name="citation_author_institution" content="Laboratory for Genomic Reprogramming, Center for Developmental Biology, RIKEN Kobe, Kobe, Hyogo, Japan"/>
      <meta name="citation_author" content="Paul E. Cizdziel"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Yoshihide Hayashizaki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
            <meta name="citation_author_institution" content="Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan"/>
            <meta name="citation_author_institution" content="Genome Science Laboratory, Discovery and Research Institute, RIKEN Wako Main Campus, Wako, Saitama, Japan"/>
            <meta name="citation_author_institution" content="Functional RNA Research Program, RIKEN Frontier Research System, Wako, Saitama, Japan"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001905.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1905"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Full-term development of mice from enucleated oocytes injected with cumulus cell nuclei.; citation_author=T Wakayama; citation_author=AC Perry; citation_author=M Zuccotti; citation_author=KR Johnson; citation_author=R Yanagimachi; citation_journal_title=Nature; citation_volume=394; citation_number=1; citation_pages=369-374; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Mice cloned from embryonic stem cells.; citation_author=T Wakayama; citation_author=I Rodriguez; citation_author=AC Perry; citation_author=R Yanagimachi; citation_author=P Mombaerts; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=2; citation_pages=14984-14989; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Epigenetic characteristics and development of embryos cloned from donor cells treated by trichostatin A or 5-aza-2′-deoxycytidine.; citation_author=BP Enright; citation_author=C Kubota; citation_author=X Yang; citation_author=XC Tian; citation_journal_title=Biol Reprod; citation_volume=69; citation_number=3; citation_pages=896-901; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Significant improvement of mouse cloning technique by treatment with trichostatin A after somatic nuclear transfer.; citation_author=S Kishigami; citation_author=E Mizutani; citation_author=H Ohta; citation_author=T Hikichi; citation_author=NV Thuan; citation_journal_title=Biochem Biophys Res Commun; citation_volume=340; citation_number=4; citation_pages=183-189; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Production of cloned mice and ES cells from adult somatic cells by nuclear transfer: how to improve cloning efficiency?; citation_author=T Wakayama; citation_journal_title=J Reprod Dev; citation_volume=53; citation_number=5; citation_pages=13-26; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Successful mouse cloning of an outbred strain by trichostatin A treatment after somatic nuclear transfer.; citation_author=S Kishigami; citation_author=HT Bui; citation_author=S Wakayama; citation_author=K Tokunaga; citation_author=N Van Thuan; citation_journal_title=J Reprod Dev; citation_volume=53; citation_number=6; citation_pages=165-170; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cloning the laboratory mouse.; citation_author=T Wakayama; citation_author=R Yanagimachi; citation_journal_title=Semin Cell Dev Biol; citation_volume=10; citation_number=7; citation_pages=253-258; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cloned mice have an obese phenotype not transmitted to their offspring.; citation_author=KL Tamashiro; citation_author=T Wakayama; citation_author=H Akutsu; citation_author=Y Yamazaki; citation_author=JL Lachey; citation_journal_title=Nat Med; citation_volume=8; citation_number=8; citation_pages=262-267; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Placentomegaly in cloned mouse concepti caused by expansion of the spongiotrophoblast layer.; citation_author=S Tanaka; citation_author=M Oda; citation_author=Y Toyoshima; citation_author=T Wakayama; citation_author=M Tanaka; citation_journal_title=Biol Reprod; citation_volume=65; citation_number=9; citation_pages=1813-1821; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Sall3 locus is an epigenetic hotspot of aberrant DNA methylation associated with placentomegaly of cloned mice.; citation_author=J Ohgane; citation_author=T Wakayama; citation_author=S Senda; citation_author=Y Yamazaki; citation_author=K Inoue; citation_journal_title=Genes Cells; citation_volume=9; citation_number=10; citation_pages=253-260; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The origins of the developmental origins theory.; citation_author=DJ Barker; citation_journal_title=J Intern Med; citation_volume=261; citation_number=11; citation_pages=412-417; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of disease: in utero programming in the pathogenesis of hypertension.; citation_author=DJ Barker; citation_author=SP Bagby; citation_author=MA Hanson; citation_journal_title=Nat Clin Pract Nephrol; citation_volume=2; citation_number=12; citation_pages=700-707; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Piezo-assisted nuclear transfer affects cloning efficiency and may cause apoptosis.; citation_author=Y Yu; citation_author=C Ding; citation_author=E Wang; citation_author=X Chen; citation_author=X Li; citation_journal_title=Reproduction; citation_volume=133; citation_number=13; citation_pages=947-954; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Abnormal gene expression in cloned mice derived from embryonic stem cell and cumulus cell nuclei.; citation_author=D Humpherys; citation_author=K Eggan; citation_author=H Akutsu; citation_author=A Friedman; citation_author=K Hochedlinger; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=14; citation_pages=12889-12894; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Faithful expression of imprinted genes in cloned mice.; citation_author=K Inoue; citation_author=T Kohda; citation_author=J Lee; citation_author=N Ogonuki; citation_author=K Mochida; citation_journal_title=Science; citation_volume=295; citation_number=15; citation_pages=297; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Variation in gene expression and aberrantly regulated chromosome regions in cloned mice.; citation_author=T Kohda; citation_author=K Inoue; citation_author=N Ogonuki; citation_author=H Miki; citation_author=M Naruse; citation_journal_title=Biol Reprod; citation_volume=73; citation_number=16; citation_pages=1302-1311; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Aberrant profile of gene expression in cloned mouse embryos derived from donor cumulus nuclei.; citation_author=GQ Tong; citation_author=BC Heng; citation_author=LG Tan; citation_author=SC Ng; citation_journal_title=Cell Tissue Res; citation_volume=325; citation_number=17; citation_pages=231-243; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Maintenance and regulation of DNA methylation patterns in mammals.; citation_author=ZX Chen; citation_author=AD Riggs; citation_journal_title=Biochem Cell Biol; citation_volume=83; citation_number=18; citation_pages=438-448; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Theoretical Analysis of Epigenetic Cell Memoryby Nucleosome Modification.; citation_author=IB Dodd; citation_author=MA Micheelsen; citation_author=KGT Sneppen; citation_journal_title=Cell; citation_volume=129; citation_number=19; citation_pages=813-822; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Rabbit somatic cell cloning: effects of donor cell type, histone acetylation status and chimeric embryo complementation.; citation_author=F Yang; citation_author=R Hao; citation_author=B Kessler; citation_author=G Brem; citation_author=E Wolf; citation_journal_title=Reproduction; citation_volume=133; citation_number=20; citation_pages=219-230; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Epigenetic marks in cloned rhesus monkey embryos: comparison with counterparts produced in vitro.; citation_author=J Yang; citation_author=S Yang; citation_author=N Beaujean; citation_author=Y Niu; citation_author=X He; citation_journal_title=Biol Reprod; citation_volume=76; citation_number=21; citation_pages=36-42; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic maps and comparative analysis of histone modifications in human and mouse.; citation_author=BE Bernstein; citation_author=M Kamal; citation_author=K Lindblad-Toh; citation_author=S Bekiranov; citation_author=DK Bailey; citation_journal_title=Cell; citation_volume=120; citation_number=22; citation_pages=169-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Histone H3 acetylated at lysine 9 in promoter is associated with low nucleosome density in the vicinity of transcription start site in human cell.; citation_author=H Nishida; citation_author=T Suzuki; citation_author=S Kondo; citation_author=H Miura; citation_author=Y Fujimura; citation_journal_title=Chromosome Res; citation_volume=14; citation_number=23; citation_pages=203-211; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome.; citation_author=ND Heintzman; citation_author=RK Stuart; citation_author=G Hon; citation_author=Y Fu; citation_author=CW Ching; citation_journal_title=Nat Genet; citation_volume=39; citation_number=24; citation_pages=311-318; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Active chromatin domains are defined by acetylation islands revealed by genome-wide mapping.; citation_author=TY Roh; citation_author=S Cuddapah; citation_author=K Zhao; citation_journal_title=Genes Dev; citation_volume=19; citation_number=25; citation_pages=542-552; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Large-scale transcriptional activity in chromosomes 21 and 22.; citation_author=P Kapranov; citation_author=SE Cawley; citation_author=J Drenkow; citation_author=S Bekiranov; citation_author=RL Strausberg; citation_journal_title=Science; citation_volume=296; citation_number=26; citation_pages=916-919; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22.; citation_author=D Kampa; citation_author=J Cheng; citation_author=P Kapranov; citation_author=M Yamanaka; citation_author=S Brubaker; citation_journal_title=Genome Res; citation_volume=14; citation_number=27; citation_pages=331-342; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution.; citation_author=J Cheng; citation_author=P Kapranov; citation_author=J Drenkow; citation_author=S Dike; citation_author=S Brubaker; citation_journal_title=Science; citation_volume=308; citation_number=28; citation_pages=1149-1154; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A RecA-mediated exon profiling method.; citation_author=Y Hasegawa; citation_author=S Fukuda; citation_author=K Shimokawa; citation_author=S Kondo; citation_author=N Maeda; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=29; citation_pages=e97; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Unbiased mapping of transcription factor binding sites along human chromosomes 21 and 22 points to widespread regulation of noncoding RNAs.; citation_author=S Cawley; citation_author=S Bekiranov; citation_author=HH Ng; citation_author=P Kapranov; citation_author=EA Sekinger; citation_journal_title=Cell; citation_volume=116; citation_number=30; citation_pages=499-509; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs.; citation_author=Y Okazaki; citation_author=M Furuno; citation_author=T Kasukawa; citation_author=J Adachi; citation_author=H Bono; citation_journal_title=Nature; citation_volume=420; citation_number=31; citation_pages=563-573; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=32; citation_pages=656-664; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A computer program for aligning a cDNA sequence with a genomic DNA sequence.; citation_author=L Florea; citation_author=G Hartzell; citation_author=Z Zhang; citation_author=GM Rubin; citation_author=W Miller; citation_journal_title=Genome Res; citation_volume=8; citation_number=33; citation_pages=967-974; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A chromatin landmark and transcription initiation at most promoters in human cells.; citation_author=MG Guenther; citation_author=SS Levine; citation_author=LA Boyer; citation_author=R Jaenisch; citation_author=RA Young; citation_journal_title=Cell; citation_volume=130; citation_number=34; citation_pages=77-88; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Comparative analysis of expression of histone H2a genes in mouse.; citation_author=H Nishida; citation_author=T Suzuki; citation_author=H Ookawa; citation_author=Y Tomaru; citation_author=Y Hayashizaki; citation_journal_title=BMC Genomics; citation_volume=6; citation_number=35; citation_pages=108; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.; citation_author=KJ Livak; citation_author=TD Schmittgen; citation_journal_title=Methods; citation_volume=25; citation_number=36; citation_pages=402-408; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Chromosome stability differs in cloned mouse embryos and derivative ES cells.; citation_author=ST Balbach; citation_author=A Jauch; citation_author=B Bohm-Steuer; citation_author=FM Cavaleri; citation_author=YM Han; citation_journal_title=Dev Biol; citation_volume=308; citation_number=37; citation_pages=309-321; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=DNA methylation variation in cloned mice.; citation_author=J Ohgane; citation_author=T Wakayama; citation_author=Y Kogo; citation_author=S Senda; citation_author=N Hattori; citation_journal_title=Genesis; citation_volume=30; citation_number=38; citation_pages=45-50; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mammalian RNA polymerase II core promoters: insights from genome-wide studies.; citation_author=A Sandelin; citation_author=P Carninci; citation_author=B Lenhard; citation_author=J Ponjavic; citation_author=Y Hayashizaki; citation_journal_title=Nat Rev Genet; citation_number=39; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=C-Reactive Protein and the Metabolic Syndrome: Useful Addition to the Cardiovascular Risk Profile?; citation_author=PE Szmitko; citation_author=S Verma; citation_journal_title=J Cardiometab Syndr; citation_volume=1; citation_number=40; citation_pages=66-69; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Is leptin an important physiological regulator of CRP?; citation_author=IS Farooqi; citation_author=S O'Rahilly; citation_journal_title=Nat Med; citation_volume=13; citation_number=41; citation_pages=16-17; author reply 19–21; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Is leptin an important physiological regulator of CRP?; citation_author=A Gertler; citation_author=L Niv-Spector; citation_author=S Reicher; citation_journal_title=Nat Med; citation_volume=13; citation_number=42; citation_pages=18-19; author reply 19–21; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Is leptin an important physiological regulator of CRP?; citation_author=WL Hutchinson; citation_author=AP Coll; citation_author=JR Gallimore; citation_author=GA Tennent; citation_author=MB Pepys; citation_journal_title=Nat Med; citation_volume=13; citation_number=43; citation_pages=17-18; author reply 19–21; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Induction of leptin resistance through direct interaction of C-reactive protein with leptin.; citation_author=K Chen; citation_author=F Li; citation_author=J Li; citation_author=H Cai; citation_author=S Strom; citation_journal_title=Nat Med; citation_volume=12; citation_number=44; citation_pages=425-432; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Radiommunological measurement of leptin in plasma of obese and diabetic human subjects.; citation_author=GP McGregor; citation_author=JF Desaga; citation_author=K Ehlenz; citation_author=A Fischer; citation_author=F Heese; citation_journal_title=Endocrinology; citation_volume=137; citation_number=45; citation_pages=1501-1504; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Serum leptin concentration is associated with total body fat mass, but not abdominal fat distribution.; citation_author=H Shimizu; citation_author=Y Shimomura; citation_author=R Hayashi; citation_author=K Ohtani; citation_author=N Sato; citation_journal_title=Int J Obes Relat Metab Disord; citation_volume=21; citation_number=46; citation_pages=536-541; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Obesity research. Leptin not impressive in clinical trial.; citation_author=T Gura; citation_journal_title=Science; citation_volume=286; citation_number=47; citation_pages=881-882; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The role of SOCS-3 in leptin signaling and leptin resistance.; citation_author=C Bjorbaek; citation_author=K El-Haschimi; citation_author=JD Frantz; citation_author=JS Flier; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=48; citation_pages=30059-30065; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA methylation errors in cloned mice disappear with advancement of aging.; citation_author=S Senda; citation_author=T Wakayama; citation_author=Y Arai; citation_author=Y Yamazaki; citation_author=J Ohgane; citation_journal_title=Cloning Stem Cells; citation_volume=9; citation_number=49; citation_pages=293-302; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mice cloned by nuclear transfer from somatic and ntES cells derived from the same individuals.; citation_author=S Wakayama; citation_author=E Mizutani; citation_author=S Kishigami; citation_author=NV Thuan; citation_author=H Ohta; citation_journal_title=J Reprod Dev; citation_volume=51; citation_number=50; citation_pages=765-772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An improved culture medium supports development of random-bred 1-cell mouse embryos in vitro.; citation_author=CL Chatot; citation_author=CA Ziomek; citation_author=BD Bavister; citation_author=JL Lewis; citation_author=I Torres; citation_journal_title=J Reprod Fertil; citation_volume=86; citation_number=51; citation_pages=679-688; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction.; citation_author=P Chomczynski; citation_author=N Sacchi; citation_journal_title=Anal Biochem; citation_volume=162; citation_number=52; citation_pages=156-159; citation_date=1987; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice"/>
    <meta name="twitter:description" content="Somatic nuclear transfer is a cloning technique that shows great promise in the application to regenerative medicine. Although cloned animals are genetically identical to their donor counterparts, abnormalities in phenotype and gene expression are frequently observed. One hypothesis is that the cause of these abnormalities is due to epigenetic aberration. In this report, we focused our analysis on the acetylation of histone H3 at lysine9 (H3K9Ac). Through the use of whole genome tiling arrays and quantitative PCR, we examined this epigenetic event and directly compared and assessed the differences between a cloned mouse (C1) and its parental nuclear donor (D1) counterpart. We identified 4720 regions of chromosomal DNA that showed notable differences in H3K9Ac and report here many genes identified in these hyper- and hypo-acetylated regions. Analysis of a second clone (C2) and its parental donor counterpart (D2) for H3K9Ac showed a high degree of similarity to the C1/D1 pair. This conservation of aberrant acetylation is suggestive of a reproducible epigenetic phenomenon that may lead to the frequent abnormalities observed in cloned mice, such as obesity. Furthermore, we demonstrated Crp which was identified as a hyper-acetylated gene in this study is related to the body mass, suggesting that Crp is a possible candidate of a cause for the abnormal obesity in cloned mice. In this, one of the first reports describing genome-wide epigenetic abberation between parental and nuclear transfer-cloned mammals, we propose that aberrant acetylation of histones (H3K9Ac) flanking promoter regions highly correlates with gene-expression and may itself be an epigenetic change that accounts for variable expression patterns observed in cloned animals."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001905.g004"/>

  <meta property="og:title" content="Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2160'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9705'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4745&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001905">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001905" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001905&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takahiro Suzuki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan, 
                  Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shinji Kondo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Teruhiko Wakayama, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Genomic Reprogramming, Center for Developmental Biology, RIKEN Kobe, Kobe, Hyogo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul E. Cizdziel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshihide Hayashizaki
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:rgscerg@gsc.riken.jp">rgscerg@gsc.riken.jp</a></p>

                <p>Affiliations:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center (GSC), RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan, 
                  Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan, 
                  Genome Science Laboratory, Discovery and Research Institute, RIKEN Wako Main Campus, Wako, Saitama, Japan, 
                  Functional RNA Research Program, RIKEN Frontier Research System, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001905</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001905-g001" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001905-t001" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001905-t002" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001905-g002" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001905-g003" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001905-g004" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001905">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001905" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Somatic nuclear transfer is a cloning technique that shows great promise in the application to regenerative medicine. Although cloned animals are genetically identical to their donor counterparts, abnormalities in phenotype and gene expression are frequently observed. One hypothesis is that the cause of these abnormalities is due to epigenetic aberration. In this report, we focused our analysis on the acetylation of histone H3 at lysine9 (H3K9Ac). Through the use of whole genome tiling arrays and quantitative PCR, we examined this epigenetic event and directly compared and assessed the differences between a cloned mouse (C1) and its parental nuclear donor (D1) counterpart. We identified 4720 regions of chromosomal DNA that showed notable differences in H3K9Ac and report here many genes identified in these hyper- and hypo-acetylated regions. Analysis of a second clone (C2) and its parental donor counterpart (D2) for H3K9Ac showed a high degree of similarity to the C1/D1 pair. This conservation of aberrant acetylation is suggestive of a reproducible epigenetic phenomenon that may lead to the frequent abnormalities observed in cloned mice, such as obesity. Furthermore, we demonstrated Crp which was identified as a hyper-acetylated gene in this study is related to the body mass, suggesting that Crp is a possible candidate of a cause for the abnormal obesity in cloned mice. In this, one of the first reports describing genome-wide epigenetic abberation between parental and nuclear transfer-cloned mammals, we propose that aberrant acetylation of histones (H3K9Ac) flanking promoter regions highly correlates with gene-expression and may itself be an epigenetic change that accounts for variable expression patterns observed in cloned animals.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Suzuki T, Kondo S, Wakayama T, Cizdziel PE, Hayashizaki Y (2008) Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice. PLoS ONE 3(4):
          e1905.
            doi:10.1371/journal.pone.0001905</p><p><strong>Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> November 29, 2007; <strong>Accepted:</strong> February 26, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Suzuki et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by a Research Grant for the RIKEN Genome Exploration Research Project from the Ministry of Education, Culture, Sports, Science, and Technology of the Japanese Government to YH; a Grant of the Genome Network Project from the Ministry of Education, Culture, Sports, Science and Technology, Japan; The Strategic Program for R&amp;D of RIKEN; Grant for the RIKEN Frontier Research System, Functional RNA research program and Scientific Research in Priority Areas (15080211) and a project for the realization of regenerative medicine from the Ministry of Education, Culture, Sports, Science and Technology of Japanese Government to TW.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Because success in cloning techniques using somatic nuclear transfer (NT) has led to the generation of genetically identical animals, the technique is now being developed for regenerative medicine. But despite substantial improvements in mouse NT cloning during recent years <a href="#pone.0001905-Wakayama1">[1]</a>–<a href="#pone.0001905-Wakayama4">[7]</a>, the cloning efficiency remains low. The best success rate is only approximately7 percent <a href="#pone.0001905-Kishigami1">[4]</a>. After birth, cloned offspring frequently exhibit a variety of abnormal phenotypes, some of which commonly include an increased body weight (large fetus syndrome) <a href="#pone.0001905-Tamashiro1">[8]</a>, placental overgrowth <a href="#pone.0001905-Tanaka1">[9]</a>, immuno-system anomalies, and a short life span <a href="#pone.0001905-Ohgane1">[10]</a>. Since the environment for fetal development largely influences health and disease of post-natal life (so called “barker's hypothesis”) <a href="#pone.0001905-Barker1">[11]</a>, <a href="#pone.0001905-Barker2">[12]</a>, the abnormal phenotypes may be derived from aberrations in nascent stage. Therefore, these low efficiency and abnormalities may be a consequence of physical damages from a piezo-assisted NT<a href="#pone.0001905-Yu1">[13]</a>, epigenetic changes caused by <em>in vitro</em> culture prior to implantation, and failure or incomplete nuclear-reprogramming. Furthermore, different uterine environment from the donor may contribute to the abnormalities as well. In many labs, general attempts to identify abnormalities at a molecular level revealed atypical profiles of gene expression compared to naturally conceived animals. These abnormal gene expression patterns were observed not only in early development, such as in embryos but also in placentas and normally birthed cloned mice <a href="#pone.0001905-Humpherys1">[14]</a>–<a href="#pone.0001905-Tong1">[17]</a>. Kohoda <em>et al</em>. have reported that cloned mice offspring show an over-diversity of gene expression compared to normal mice, but common abnormalities are surprisingly few. They also showed that some of these genes map in a linked cluster and are commonly changed in the same manner. Taken together, this evidence suggest that although there are many abnormalities caused by random components, cloned mice indeed exhibit common abnormalities that result from non-random components.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>DNA methylation and histone modifications are known to be important epigenetic events that control cellular morphology and function <a href="#pone.0001905-Chen1">[18]</a>, <a href="#pone.0001905-Dodd1">[19]</a>. There is now a growing body of evidence suggesting that proper histone acetylation may be a critical element in successful NT cloning. Treatment with trichostatin A (TSA), a histone deacetylase inhibitor, after somatic nuclear transfer significantly improves the mouse cloning not only in the generation rate but also in reducing the abnormal phenotypes, with the exception of placenta overgrowth <a href="#pone.0001905-Kishigami1">[4]</a>, and enables mouse cloning of an outbred strain<a href="#pone.0001905-Kishigami2">[6]</a>Furthermore, the developmental potential of clones correlate with the histone acetylation levels in rabbits <a href="#pone.0001905-Yang1">[20]</a> and primates<a href="#pone.0001905-Yang2">[21]</a>. These reports clearly indicate that histone acetylation is a key element in animal cloning but the mechanism that links histone acetylation to phenotypes remains undetermined.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Acetylation of histone H3 at Lysine 9 (H3K9Ac), one of the most well known epigenetic markers, is enriched in the promoter region of activated genes <a href="#pone.0001905-Bernstein1">[22]</a>–<a href="#pone.0001905-Roh1">[25]</a>. The level of H3K9Ac in the promoter strongly correlates to the gene expression. High density oligo tiling arrays which cover entire genomes have enabled the genome wide analysis not only of transcriptome by RNA mapping <a href="#pone.0001905-Kapranov1">[26]</a>–<a href="#pone.0001905-Hasegawa1">[29]</a> but also of protein-DNA interactions or histone modifications by a combination with chromatin immunoprecipitation (ChIP) <a href="#pone.0001905-Bernstein1">[22]</a>–<a href="#pone.0001905-Heintzman1">[24]</a>, <a href="#pone.0001905-Cawley1">[30]</a>. Hence, we have obtained genome-wide data of H3K9Ac by using commercially available whole-genome tiling arrays.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Application to a regenerative medicine, a comparison of clones to their donor counterparts may provide more valuable insights than comparing clones and normal animals. However, nearly all studies of cloned mice so far, have focused on this latter comparison. In the gene expression analysis, individual cloned mice show diverse results, suggesting that combined data from multiple cloned mice will not accurately represent abnormalities or differences of significance <a href="#pone.0001905-Kohda1">[16]</a>. Therefore, our approach focused on investigating differences between a single cloned mouse and its parental (donor) counterpart. In this report we characterized one key epigenetic abnormality at the genome-wide level, analyzing the level of H3K9Ac in the liver of the clone and that of its parental (donor) counterpart by using whole-genome tiling arrays. We identified genomic regions that showed a notable difference of H3K9Ac between the mouse clone and donor and identified genes in those regions whose promoters were either significantly hypo-acetylated or hyper-acetylated. We also have shown that abnormal gene expression in a cloned mouse often correlates with abnormal H3K9Ac at their promoters. These findings suggest that, among epigenetic factors, H3K9Ac may be one of the major causative elements of abnormalities that need to be addressed for improving success rates in animal cloning or perhaps applications in regenerative medicine.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>H3K9 acetylation profiles of cloned mouse and its donor mouse by <em>ChIP on chip</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To identify the H3K9Ac enriched sites, we performed ChIP followed by tiling array analysis (<em>ChIP on chip</em>) using the Affymetrix mouse tiling 1.0 array, which spans the entire mouse genome at a 30 bp resolution with 47 million 25-mer probes (excluding repetitive sequence elements). Fixed chromatin preparations from cloned mouse (C1) and donor mouse (D1) livers were immunoprecipitated with an antibody directed against the H3K9Ac. The immunoprecipitated DNA (treatment sample) and the control DNA from a whole cell extract (WCE) were amplified by the Ligation-Mediated PCR (LM-PCR) method and hybridized to the tiling arrays as previously described <a href="#pone.0001905-Nishida1">[23]</a>. As a measure of the H3K9Ac enrichment, a P-value (P) was computed at each probe position by assessing the excess of signal intensities detected by probes located within±200 bp in the treatment relative to the control sample by wilcoxon rank sum test <a href="#pone.0001905-Cawley1">[30]</a>. Using P≤10<sup>−4</sup> as a significant cut-off <a href="#pone.0001905-Bernstein1">[22]</a>, <a href="#pone.0001905-Nishida1">[23]</a>, we have identified 30902 and 32757 H3K9Ac enriched sites in the C1 and the D1 livers, respectively. Among them, 9584 sites (31%) and 10201 sites (31%) were located in the promoter regions which are defined as being±1 kb from transcription start site (TSS). The TSSs were determined by aligning 470,133 RIKEN 5′ ESTs<a href="#pone.0001905-Okazaki1">[31]</a> to the mouse genome, by using BLAT<a href="#pone.0001905-Kent1">[32]</a> and SIM4<a href="#pone.0001905-Florea1">[33]</a>. These results clearly indicated that the H3K9Ac enriched sites were biased toward the promoter regions, which is consistent with previous studies performed on human cell lines <a href="#pone.0001905-Bernstein1">[22]</a>–<a href="#pone.0001905-Roh1">[25]</a>, <a href="#pone.0001905-Guenther1">[34]</a>, and suggest this regional bias and functions of H3K9Ac are conserved among higher eukaryotes.</p>


<h4>Validation for tiling array data</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Although <em>ChIP on chip</em> data is a useful mass screening tool, the technology is less reliable for quantitation of DNA than many other methods. Hence we selected quantitative PCR (qPCR) as a validation tool and picked 43 genomic sites to test. No consideration was given to the P-values of the selected sites but rather they were chosen for their general dispersion across the genome (<a href="#pone.0001905.s001">Table S1</a>). Of these 43 sites, 23 were in the gene promoters and 20 sites were located among intergenic sequences. Using identical immuno-precipitation techniques, we carried out ChIP followed by qPCR (ChIP-qPCR) on the 43 sites <a href="#pone.0001905-Nishida2">[35]</a>. qPCR was achieved by real-time PCR and the enrichment of the ChIP DNA as multiples of the WCE were computed for each site. The quantitative values for enrichment as determined by ChIP-qPCR were plotted against P-values measured by <em>ChIP on chip</em> (<a href="#pone-0001905-g001">Figure 1A, B</a>). Sites displaying little to no enrichment [log<sub>2</sub>] measured by ChIP-qPCR coordinately showed very low significance P-values as measured by <em>ChIP on chip</em> (values of 10 and below in -10logP). In the sites showing 2-fold enrichment and above (≥log<sub>2</sub>1), increasing enrichment clearly correlates with an increasing significance level of P-value. Out of 18 sites, Seventeen (17) sites which were detected as P≤10<sup>−4</sup> yielded an enrichment greater than 4-fold in ChIP-qPCR (11/11 in C1 liver, 6/7 in D1 liver). Based on this, we conclude that the cut-off of P≤10<sup>−4</sup> is reliable to identify the H3K9Ac enriched sites with little false positives and high certainty. Furthermore, the total Pearson correlation coefficients were 0.68 and 0.74 in C1 and D1 liver samples, respectively. In addition, from the data, we can interpret that amplification of ChIP DNA by LM-PCR prior to hybridization to the tiling array did not introduce a significant bias into the <em>ChIP on chip</em> results.</p>
<div class="figure" id="pone-0001905-g001"><div class="img"><a name="pone-0001905-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span><em> ChIP on chip</em> data Analysis.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A, B) Relative enrichment measured by ChIP-qPCR for 43 sites in C1 (A) and D1 (B) and the average of −10logP of each of 43 regions were plotted as a scatter plot. Black points represent the un-enriched amplicons, blue points represent the enriched amplicons of qPCR data where −10logP&lt;40 for <em>ChIP on chip</em>, red points represent the enriched amplicons of qPCR data where −10logP≥40 for <em>ChIP on chip</em>. (C) Averages of −10logP at each promoter (±1 kb of TSS) were plotted as a scatter plot. Vertical axis represents D1 liver and the horizontal axis represents C1 liver. Total promoter sites used in this analysis included 16203 sites. Spearman correlation coefficient between C1 and D1 was 0.898.</p>
<span>doi:10.1371/journal.pone.0001905.g001</span></div>

<h4>Promoter H3K9 acetylation differences between Cloned and Donor mouse</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Promoter regions are a major target for epigenetic studies to explain gene expression level changes between cloned mice and their parental counterparts. A comparison of the H3K9Ac <em>ChIP on chip</em> tiling array P-values from over 16,000 promoter region data points between C1 and D1 livers samples showed a high degree of similarity, with a Spearman correlation coefficient of 0.898 (<a href="#pone-0001905-g001">Figure 1C</a>). However, to specifically identify and focus on promoter regions where significant differences exist in H3K9Ac levels between mouse clones and donors, signal intensities from <em>ChIP on chip</em> tiling array data in the C1 sample relative to the D1 sample and <em>vice versa</em> were evaluated by the Wilcoxon rank sum test <a href="#pone.0001905-Cawley1">[30]</a>. In the cloned mouse (C1), 2550 hyper-acetylated sites and 2170 hypo-acetylated sites were identified with a P≤10<sup>−4</sup>. Of these sites, 113 and 147 sites respectively, were located in gene promoter regions. These results establish that abnormal H3K9Ac differences between a clone and a parental donor is not significantly biased toward promoter regions, but rather scattered across the chromosomal DNA.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In that changes in gene expression are ultimately responsible for abnormalities in clones, we next focused our analysis on the regions identified that mapped to promoters and were either hyper- or hypo-acetylated at H3K9. To confirm abnormal H3K9Ac of the promoters, we performed ChIP-qPCR using primers designed by Primer3 software from sequence information of the regions that were statistically significant at a P≤10<sup>−4</sup> or less. From 113 hyper-acetylated and 147 hypo-acetylated site possibilities, only 74 and 61 primer sets could be designed for validation due to sequence constraints (<a href="#pone.0001905.s002">Table S2</a>). As before, ChIP DNA was amplified by LM-PCR method followed by real-time PCR measurement. The relative difference in the level of H3K9Ac between the samples was computed as the change in the C1 relative to the D1 sample as log<sub>2</sub> (C1/D1). Cut-off values for this comparison were relaxed relative to the previous ChIP-qPCR analysis because the reference material was another ChIP sample (not WCE) and as a consequence, the results displayed more variability. In addition, since the difference of H3K9Ac was relatively insignificant between C1 and D1 (<a href="#pone-0001905-g001">Figure 1C</a>), cut-off value should be looser to identify imperceptible differences. Furthermore, stringency did not need to be high at this stage of selection since a first screening was already performed by tiling array data. Thus, we needed here sensitivity rather than specificity. A significant increase (&gt;0.5 [log<sub>2</sub> (C1/D1)]) of an H3K9Ac level in the C1 sample was detected on 26 hyper-acetylated sites (<a href="#pone-0001905-t001">Table 1</a>), and a significant decrease (&lt;−0.5 [log<sub>2</sub> (C1/D1)]) was detected on 22 hypo-acetylated sites (<a href="#pone-0001905-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001905-t001"><div class="img"><a name="pone-0001905-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001905.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001905.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Hyper-acetylated Genes of Mouse Clone (C1)</span></strong></p><span>doi:10.1371/journal.pone.0001905.t001</span></div><div class="figure" id="pone-0001905-t002"><div class="img"><a name="pone-0001905-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001905.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001905.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Hypo-acetylated Genes of Mouse Clone (C1)</span></strong></p><span>doi:10.1371/journal.pone.0001905.t002</span></div>

<h4>Relationship between abnormalities of H3K9 acetylation and gene expression</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Previous studies have shown that the level of H3K9Ac in promoter regions is strongly correlated with gene expression<a href="#pone.0001905-Nishida1">[23]</a>. Our hypothesis was that hyper- and hypo-acetylation in the promoter region leads to abnormal gene expression in a cloned mouse. To test this hypothesis, the expression levels of the genes whose promoters were hyper- or hypo-acetylated in the C1 and D1 mice were measured by using qRT-PCR. Total RNA was isolated from livers of the C1 and the D1 mice, and reverse-transcribed with random hexamers. Using the cDNA as a template, a real-time PCR measurement was performed using coding region primers (<a href="#pone.0001905.s003">Table S3</a>) for the 26 hyper-acetylated genes (<a href="#pone-0001905-t001">Table 1</a>) and 22 hypo-acetylated genes (<a href="#pone-0001905-t002">Table 2</a>) from the C1 and D1 mice. We then determined the differences in gene expression between the C1 liver and the D1 liver using the 2<sup>−ΔΔCt</sup> method<a href="#pone.0001905-Livak1">[36]</a> (<a href="#pone-0001905-g002">Figure 2</a>). We detected 2-fold or greater increase in the level of expression for 6 (23.1%) out of the 26 hyper-acetylated genes, and none showed a 2-fold or greater decrease in gene expression. For the 22 hypo-acetylated genes examined, we found 5 (21.7%) that showed 2-fold or greater decrease in the level of expression and none that showed 2-fold or greater increase in expression levels. These results suggest that hyper- or hypo-acetylation of H3K9Ac may contribute to a part of the abnormal gene expression in the NT cloned mice, but is likely not the only factor involved.</p>
<div class="figure" id="pone-0001905-g002"><div class="img"><a name="pone-0001905-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Expression Modulation of Genes with Promoters Hyper- and Hypo-acetylated at H3K9.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Hyper-acetylated genes (A) and Hypo-acetylated genes (B) were pre-selected by <em>ChIP on chip</em> data and validated by ChIP-qPCR for alterations in H3K9Ac, and then analyzed for changes in gene expression by qRT-PCR (n = 2). Bars are shown as the average of duplicate assay results for changes in expression levels in C1 relative to D1. Standard deviations are shown using error bars. Upper red dotted lines represent 2-fold increase and lower red dotted lines represent 2-fold decrease.</p>
<span>doi:10.1371/journal.pone.0001905.g002</span></div>

<h4>Conservation of H3K9Ac variation in other mouse clones</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>It is important to determine if the same aberrant acetylation events are reproducible and lead to the reproducible abnormal phenotypes observed in cloned mice. To investigate this, we examined another cloned mouse (C2) and donor (D2) pair that was derived using the same cumulus cell NT technique. We repeated the ChIP-qPCR experiment using the same primer sets for 26 hyper- and 22 hypo-acetylated genes (<a href="#pone.0001905.s002">Table S2</a>) on ChIP from the livers of the C2 and D2 mice. The change in acetylation for each gene for the C1/D1 pair and C2/D2 pair (<a href="#pone-0001905-g003">Figure 3A</a>) was statistically analyzed for group significance (<a href="#pone-0001905-g003">Figure 3B</a>). Among the hyper-acetylated sites, the overall tendency of changes in the C2/D2 pair was clearly biased toward the positive compared to all (hyper- and hypo-acetylated) sites, with a P&lt;0.05 (Welch's t-test), indicating statistical significance toward hyper-acetylation (<a href="#pone-0001905-g003">Figure 3B</a>). On the other hand, the overall distribution of the hypo-acetylated sites was biased toward negative change with a P&lt;0.05 (Welch's t-test), indicating statistical significance toward hypo-acetylation (<a href="#pone-0001905-g003">Figure 3B</a>). These results suggest that the abnormalities in H3K9Ac patterns among cloned mice derived from cumulus cells by NT tends to be conserved among cloned individuals, and the abnormalities may be related to common abnormal phenotypes of cloned mice.</p>
<div class="figure" id="pone-0001905-g003"><div class="img"><a name="pone-0001905-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Conservation of H3K9Ac variation in other mouse clones.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Acetylation-state changes of each promoter site in the cloned mice are indicated by color, and as a multiple of corresponding donor mice (A). No-change is represented as white, increased acetylation is depicted as intensifying red color, and decreased acetylation is depicted as intensifying yellow color. The value of each change is the average of technical replicates (n = 2). Details of the target genes are reported in <a href="#pone-0001905-t001">Table 1</a> and <a href="#pone-0001905-t002">Table 2</a>. (B) For each gene, the actual experimental values for the C2/D2 pair is graphically aligned in categories of hyper- or hypo-acetylation containing 26 and 22 genes, respectively, which is a reflection of gene-state determined in the C1/D1 mouse pair. Medians for each data set are indicated by black center lines, upper quartiles are indicated by upper edges of the box, and lower quartiles are indicated by lower edges of the box. Maximum and minimum values are marked as end of lines extending from the boxes. Values which are more than 1.5 times the inter-quartile range from the upper or lower quartile are represented as white circles and are considered outliers and not used in the calculations. The red dotted horizontal line indicates the position of median of all genes (hyper- and hypo-acetylated genes, total 48 genes).</p>
<span>doi:10.1371/journal.pone.0001905.g003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The abnormalities of cloned mice have been recognized not only by phenotype <a href="#pone.0001905-Tamashiro1">[8]</a>–<a href="#pone.0001905-Ohgane1">[10]</a> but also partially characterized by gene expression <a href="#pone.0001905-Humpherys1">[14]</a>–<a href="#pone.0001905-Tong1">[17]</a>. The root causes of these abnormalities are thought to be attributed to multiple factors. For instance, a growing body of evidence has emerged to support the general supposition that chromosomal abnormalities are one of the elements for failures in animal cloning <a href="#pone.0001905-Yu1">[13]</a>, <a href="#pone.0001905-Balbach1">[37]</a>.Among these factors, epigenetic factors are important elements for the successful animal cloning<a href="#pone.0001905-Enright1">[3]</a>, <a href="#pone.0001905-Kishigami1">[4]</a>, <a href="#pone.0001905-Kishigami2">[6]</a>, <a href="#pone.0001905-Yang1">[20]</a>, <a href="#pone.0001905-Yang2">[21]</a>. However, the epigenetic properties responsible for the aberrant cloned mouse phenotypes are poorly understood. We have chosen to analyze H3K9Ac because much evidence exists relating this epigenetic feature to active gene expression <a href="#pone.0001905-Bernstein1">[22]</a>–<a href="#pone.0001905-Roh1">[25]</a>. Previous studies have focused on normal mice as the reference control <a href="#pone.0001905-Tanaka1">[9]</a>, <a href="#pone.0001905-Ohgane1">[10]</a>, <a href="#pone.0001905-Humpherys1">[14]</a>–<a href="#pone.0001905-Tong1">[17]</a>, <a href="#pone.0001905-Ohgane2">[38]</a>, but not specifically the parental NT-donor mouse. A comparison of the clone with its donor, more accurately identifies the variation of H3K9Ac that occurs in cloning and suggests that a large number of genomic sites are involved. We also examined whether the same H3K9Ac alterations existed in two pairs of cumulus cell-derived cloned mouse and parental donor pairs. Evidence showed that the promoter region hyper- and hypo-acetylated sites detected in one cloned mouse tended to be similar and statistically significant in the other examined cloned mouse, as well. However, it is not clear if this conservation of the H3K9Ac alteration pattern among separate clones is an inheritance from the donor cell, or if it results from the aberrant nuclear-reprogramming which is based on common factors. To answer this question, further study is required.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In higher eukaryotes, transcriptional regulation is complex. Many factors and elements influence and directly regulate transcription, such as transcriptional regulatory factors, DNA methylation, and Chromatin structure. Furthermore, these elements work together and clearly influence one other, establishing a concerted functional “network” <a href="#pone.0001905-Sandelin1">[39]</a>. Although we previously reported that the degree of H3K9Ac and gene expression showed positive correlation, we also found many examples that are not consistent with this observation <a href="#pone.0001905-Nishida1">[23]</a>. Another group also reported that the histone modifications which are known to be associated with active gene promoter are enriched at both active and inactive promoters <a href="#pone.0001905-Guenther1">[34]</a>. Hence, the cause of abnormal gene expression is not solely a direct consequence of H3K9Ac, but perhaps an indirect consequence or a contributing factor influenced by other elements as well. From our experiments looking at genes with aberrant H3K9Ac, almost one-fourth of the hyper-acetylated genes we examined for expression in clone mouse (C1) livers were up-regulated by more than 2-fold, and almost one-fifth of the hypo-acetylated genes were down regulated when compared to the donor animal (D1) liver. It is possible that H3K9Ac increases the possibility of active transcription, but itself is not sufficient <a href="#pone.0001905-Heintzman1">[24]</a>, <a href="#pone.0001905-Guenther1">[34]</a>. The correlation between gene expression and H3K9Ac presented in this study is consistent with the possibility. This may explain the phenotypic variability in TSA treated cloned mice. We believe that aberrant H3K9Ac in cloned mice is a critical factor influencing abnormal gene expression, but by simply looking at one tissue (such as liver in this study), or embryonic stage, or animal age, the real overall impact cannot be fully determined. One interesting finding is that H3K9Ac enriched sites are clearly biased toward gene promoters. However, approximately two-thirds of H3K9Ac was also scattered among the chromosomes, which is consistent with previous studies in human <a href="#pone.0001905-Bernstein1">[22]</a>–<a href="#pone.0001905-Heintzman1">[24]</a>, <a href="#pone.0001905-Guenther1">[34]</a>. Many intergenic sites (&gt;2000) showed a significant difference in the H3K9Ac between cloned and donor mice in this study. Hence it is possible that H3K9Ac influences not only to proximal regulatory elements in and around promoters, but also distant regulatory elements, such as enhancers, silencers, insulators, and locus control regions. This can also have a significant impact on the expression of genes located in the vicinity, and is an important subject for future investigation.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>As a possible phenotype-related H3K9Ac aberration, we noted C-reactive protein (Crp) is a hyper-acetylated gene identified in this study (<a href="#pone-0001905-g004">Figure 4A</a>). Crp is now recognized as a potential predictor of metabolic syndrome <a href="#pone.0001905-Szmitko1">[40]</a>. Although there are several disputations <a href="#pone.0001905-Farooqi1">[41]</a>–<a href="#pone.0001905-Hutchinson1">[43]</a>, Chen <em>et al.</em> reported that Crp induces leptin resistance, inhibiting the binding of leptin to its receptor <a href="#pone.0001905-Chen2">[44]</a>. In humans and rodents, leptin concentration in the blood shows strong positive correlation with body mass <a href="#pone.0001905-McGregor1">[45]</a>, <a href="#pone.0001905-Shimizu1">[46]</a>, which may be explained by the leptin resistance <a href="#pone.0001905-Gura1">[47]</a>, <a href="#pone.0001905-Bjorbaek1">[48]</a>. Interestingly, one of the common abnormal phenotypes in cloned mice is obesity and the concentration of leptin in the blood is usually more than 10 times higher than that of normal mice <a href="#pone.0001905-Tamashiro1">[8]</a>. Hence, we speculated that the reason for the obesity in cloned mice is the leptin resistance induced by an excess of the Crp protein. To investigate a relationship between the body mass and the Crp in cloned mice, we measured mice body weight and the Crp protein level in serum by the Enzyme-Linked ImmunoSorbent Assay (ELISA) from eight (8) cloned mice treated with TSA. TSA treatment in mouse cloning partially abrogates the tendency toward obesity, generating cloned mice that have the same background but various body weights (<a href="#pone-0001905-g004">Figure 4B, D</a>). The Crp protein level in serum showed a strong correlation with the weight of these animals (<a href="#pone-0001905-g004">Figure 4C</a>). For this experiment, the Pearson correlation coefficient was 0.72. This result suggests that cloned mice obesity could be related to Crp. However, expression of the Crp mRNA was not significantly changed in the C1 liver compared to D1 liver (<a href="#pone-0001905-g002">Figure 2A</a>). H3K9 hyper-acetylation may be not sufficient to increase the active transcription <a href="#pone.0001905-Heintzman1">[24]</a>, <a href="#pone.0001905-Guenther1">[34]</a>. Hence the abnormal Crp expression may occur at another age and/or in another tissue in the obese cloned animal and is a topic that warrants further investigation. In a recent paper, Senda et al. report that abnormalities of clones fade with aging<a href="#pone.0001905-Senda1">[49]</a>. The obesity actually appears 8 weeks after birth<a href="#pone.0001905-Tamashiro1">[8]</a>. Considering their result and our data, it is plausible to assume that some abnormalities can escape the fading.</p>
<div class="figure" id="pone-0001905-g004"><div class="img"><a name="pone-0001905-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001905" data-uri="info:doi/10.1371/journal.pone.0001905.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001905.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001905.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001905.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001905.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Crp levels in the blood of adult TSA-treated cloned mice of various weights.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) Result of <em>ChIP on chip</em> analysis at the region of the Crp gene. Horizontal axis represents alignment of Chr1 and the vertical axes represent P-value of tiling array intensity data. The scale of the genome viewer is given as coordinate of Chr.1 (bp). Upper tier (magenta) represents direct comparison of C1 ChIP sample vs. D1 ChIP sample, middle tier (green) represents D1 ChIP sample vs. WCE, and lower tier (yarrow) represents C1 ChIP sample vs. WCE. Each vertical line standing on the each tire represents positions of array probes and its −10logP value. Horizontal lines in each tier represent P = 10<sup>−4</sup> and filled bars being bottom of each tier represent regions which are P&lt;10<sup>−4</sup> or less, respectively. Blue line represents the exon structure of the Crp gene. (B) Photograph of TSA-treated mice used for Crp plasma protein level measurements by ELISA. Right mouse is the leanest mouse (TSA-5) and left mouse is the most obese mouse (TSA-8). (C) Crp protein concentration in TSA-treated mouse clone serum measured by ELISA. The vertical axis represents the concentration of Crp protein in serum and the horizontal axis represents body weight. The concentration is shown as the average of four technical replicates (n = 4). Pearson correlation coefficient for the test is 0.72. (D) General information of TSA-treated mice. The information shows ID for the mice and corresponding sex, age (month), and body weight.</p>
<span>doi:10.1371/journal.pone.0001905.g004</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>To our knowledge, this is the first report describing genome-wide epigenetic aberration between donor and cloned animals. We propose that epigenetic abnormalities, such as those in H3K9Ac lead to abnormal gene expression, and that abnormal gene expression results in abnormal phenotypes. Hence, controlling the epigenetic landscape of a cell or by selecting a proper donor cell type possessing epigenetic information suitable to cloning we may be able to improve cloning success rates and the health of cloned animals.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Production of Cloned mice</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>BDF1 (C57BL/6×DBA/2) oocytes and fresh isolated BD129F1 (BDF1×129/Sv) cumulus cells were collected and used for the production of cloned mice as previously described <a href="#pone.0001905-Wakayama5">[50]</a>. A group of oocytes was transferred to a droplet of HEPES-CZB <a href="#pone.0001905-Chatot1">[51]</a>containing 5 µg/ml cytochalasin B, then metaphase II chromosome-spindle complexes were removed from the oocytes by an enucleation pipette. The donor cumulus cells were drawn in and out of the injection pipette until the cell membrane was broken and nuclei were injected one by one into an enucleated oocyte. After nuclear transfer, the reconstructed oocytes were activated by 10 mM SrCl<sub>2</sub> in Ca<sup>2+</sup>-free CZB medium in the presence of 5 g/mL cytochalasin B, and cultured for 24 h in CZB medium. When cloned embryos were developed to the 2-cell stage, it was transferred to the oviduct of foster mothers (ICR strain) which had been mated with vasectomized ICR males 1 day previously. All recipient females were euthanized at 19.5 dpc (days post coitum) and pups raised by lactating ICR foster mothers. All animals (obtained from SLC, Shizuoka, Japan, or bred in Riken) were maintained in accordance with the Animal Experiment Handbook at Riken Center of Developmental Biology.</p>


<h4>Chromatin immunoprecipitation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Livers extracted from cloned and donor mice at the age of 2.5 month-old were homogenized using the BioMasher™ extraction device (Nippi Corp.). ChIP experiments using an anti-acetylated K9 histone H3 antibody (catalog no. 07-352, Upstate Bio Corp.) were performed as described <a href="#pone.0001905-Nishida1">[23]</a> with some modification. Briefly, the homogenized samples were fixed with 1% formaldehyde, resuspended in TE buffer (pH8.0), and fragmented to a size range of 200 to 1000 bases using a Sonifier® S-450 (Branson Crop.) with a cup horn. The fragmented chromatin was pre-cleared with protein G sepharose after adding 100 mM NaCl and 0.4% NP-40. After removal of a control aliquot (whole cell extract, WCE), pre-cleared lysates were incubated at 4°C overnight adding the anti-acetylated K9 histone H3 antibody. The immune complexes were precipitated with protein G Dynabeads® (Invitrogen Dynal Corp.), washed and eluted from beads into elution buffer (100 mM NaHCO<sub>3</sub>, 1% SDS). The WCE aliquot and the eluted immune complexes were reverse cross-linked at 65°C for 3.5h and treated with RNase and Proteinase K. The DNA from WCE and the ChIP DNA was extracted using phenol/chloroform method and ethanol precipitation.</p>


<h4>Ligation Mediated-PCR (LM-PCR) and array hybridization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>LM-PCR and array hybridization were performed as previously described <a href="#pone.0001905-Nishida1">[23]</a>. Briefly, the blunted ChIP DNAs and WCE DNAs were ligated with the annealed oligonucleotides, amplified by using Blend <em>Taq™</em>-Plus (Cosmo Bio Corp.). The amplified DNA was purified, fragmented with DNase I, and end-labeled with biotin-ddATP by using terminal deoxytransferase. The labeled fragments were hybridized to the GeneChip® Mouse Tiling 1.0 Array Set (Affymetrix Corp.). The arrays were washed, and scanned using the Affymetrix GeneChip® System. Each sample was hybridized and tested as duplicate samples.</p>


<h4>Level of enrichment for acetylation of lysine 9 of histone H3 in the whole mouse genome</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The hybridization intensities (background-subtracted intensity, PM–MM where PM and MM indicate intensities detected by a 25-mer perfectly matching and another one-base-mismatching to genome, respectively.) of the probes were measured in two technical replicates for each of the treatment and control samples, and were mapped to the genome using exact 25-mer matching to mouse genome (NCBI version 32). Raw array data were quantile normalized within treatment/control replicate groups and scaled to a median feature intensity of 500. A shift of the intensities in the treatment relative to control data in a 400 bp window centered at each probe was evaluated by a Wilcoxon Rank Sum test, which assigned a P-value to the probe position <a href="#pone.0001905-Bernstein1">[22]</a>, <a href="#pone.0001905-Nishida1">[23]</a>. We used the Affymetrix software, TAS (<a href="http://www.affymetrix.com/support/developer/downloads/TilingArrayTools">http://www.affymetrix.com/support/develo​per/downloads/TilingArrayTools</a>) for the P-value calculation. All tiling array data including raw signal intensity and calculated P-value are available in Center for Information Biology gene EXpression database (CIBEX) (<a href="http://cibex.nig.ac.jp/">http://cibex.nig.ac.jp/</a>) accession number CBX38.</p>


<h4>Profile of acetylation at lysine 9 of histone H3 in promoter region of protein-coding genes in donor and cloned mice</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The profile of H3K9Ac enrichment in the promoter region (±1 kb of TSS) of protein-coding genes was compared between the donor and cloned mice. Using the 5′ ends of mRNA sequences (18,707 transcripts in 16,720 loci downloaded on Jan. 19, 2006 from RefSeq database: <a href="http://ftp://ftp.ncbi.nih.gov/refseq">ftp://ftp.ncbi.nih.gov/refseq</a>) aligned to genome as TSS, we attempted to measure the magnitude of H3K9Ac enrichment in the promoter region. The alignment was carried out by using BLAT<a href="#pone.0001905-Kent1">[32]</a> and SIM4<a href="#pone.0001905-Florea1">[33]</a>. For the comparison of the H3K9Ac enrichment between the donor and cloned mice, we averaged −10 log P detected by the probes located in the promoter region, and computed correlation coefficient (Spearman).</p>


<h4>Real-time PCR</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>PCR primers for evaluating ChIP assays (<a href="#pone.0001905.s001">Table S1</a>) were designed to amplify 70–150 base pair fragments from the genomic regions indicated for validation of <em>ChIP on chip</em> data, and for identification of the H3K9 hyper- or hypo-acetylated genes (<a href="#pone.0001905.s002">Table S2</a>). PCR amplification was performed on an ABI PRISM® 7500 Sequence Detection System (Applied Biosystems, Inc.). For amplification, SYBR Premix Ex <em>Taq™</em> (Takara Corp.) was used as instructed in the manual. The PCR conditions were an initial step of 10 sec at 95°C, followed by 40 cycles of 3 sec at 95°C and 20 sec at 62.5°C. To generate a standard curve for each amplicon, Ct values of serially diluted WCE DNA which extracted in ChIP experiment (5 ng, 2.5 ng, 1.25 ng, 0.625 ng, and 0.3125 ng) were determined and plotted against the amount of WCE DNA template. ChIPed DNA was measured by real-time PCR in duplicates and the median of the average Ct values of all the amplicons in each sample was scaled to 22. The amount of an amplicon was estimated from the standard curve. Enrichment, (fold change or relative enrichment) was determined as multiple of the amount for each amplicon.</p>


<h4>Quantitative Reverse Transcription-Polymerase Chain Reaction (qRT-PCR)</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Total RNA was extracted from the livers of the D1 and C1 mice by the acid phenol–guanidinium thiocyanate–chloroform method <a href="#pone.0001905-Chomczynski1">[52]</a>. Reverse transcription of total RNA was achieved with PrimeScript™ Reverse Transcriptase (Takara Corp.) and random hexamers in accordance with the manufacturer's protocol. Glyceraldehyde-3-phosphate dehydrogenase (GAPDH) mRNA which yielded a amplicon of 76 bp was used as a control (primers <span class="gene">5′-TGTGTCCGTCGTGGATCTGA-3′</span> and <span class="gene">5′-CCTGCTTCACCACCTTCTTGA-3′</span>) for data normalization. The PCR primers used for each gene in this analysis are given (<a href="#pone.0001905.s003">Table S3</a>). PCR amplification using the real-time PCR was performed as described above. Changes of gene expression were determined using the 2<sup>−ΔΔCt</sup> method <a href="#pone.0001905-Livak1">[36]</a>.</p>


<h4>Enzyme-Linked ImmunoSorbent Assay (ELISA)</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Blood was taken from the eyeground vein of eight TSA-treated cloned mice. After blood clotting, the serum from each sample was recovered as a supernatant by centrifugation. The C-Reactive Protein (CRP) ELISA (High-Sensitivity), Mouse kit (cat. #KT-095, Kamiya Biomedical Company) was used to measure the Crp levels in the serums in accordance with the manufacturer's protocol.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001905"></div><a name="pone.0001905.s001" id="pone.0001905.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001905.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">ChIP-qPCR Primers used for validation of the tiling array data.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001905.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001905.s002" id="pone.0001905.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001905.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Hyper- or hypo-acetylated sites (start and end position on the mouse genome NCBI version 32) and primers used for ChIP-qPCR.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001905.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
<a name="pone.0001905.s003" id="pone.0001905.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001905.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Hyper- or hypo-acetylated genes and corresponding coding region primers used for qRT-PCR.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001905.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Hiromi Nishida, Alistair Forrest, Yuki Hasegawa, Takashi Khoda, and Fumitoshi Ishino for helpful advices and useful suggestions. We also thank Yuki Tsujimura for an experimental support. We are also grateful to the members of RIKEN Genome Exploration Research Group and RIKEN Laboratory for Animal Resources and Genetic Engineering for the housing of mice.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TS YH. Performed the experiments: TW TS. Analyzed the data: TS SK. Contributed reagents/materials/analysis tools: TW. Wrote the paper: TW TS SK PC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001905-Wakayama1" id="pone.0001905-Wakayama1"></a>Wakayama T, Perry AC, Zuccotti M, Johnson KR, Yanagimachi R (1998) Full-term development of mice from enucleated oocytes injected with cumulus cell nuclei. Nature  394: 369–374.  <ul class="find" data-citedArticleID="1084290"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Full-term+development+of+mice+from+enucleated+oocytes+injected+with+cumulus+cell+nuclei.&amp;auth=&amp;atitle=Full-term+development+of+mice+from+enucleated+oocytes+injected+with+cumulus+cell+nuclei." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-term+development+of+mice+from+enucleated+oocytes+injected+with+cumulus+cell+nuclei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-term+development+of+mice+from+enucleated+oocytes+injected+with+cumulus+cell+nuclei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001905-Wakayama2" id="pone.0001905-Wakayama2"></a>Wakayama T, Rodriguez I, Perry AC, Yanagimachi R, Mombaerts P (1999) Mice cloned from embryonic stem cells. Proc Natl Acad Sci U S A  96: 14984–14989.  <ul class="find" data-citedArticleID="1084292" data-doi="10.1073/pnas.96.26.14984"><li><a href="http://dx.doi.org/10.1073/pnas.96.26.14984" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mice+cloned+from+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mice+cloned+from+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001905-Enright1" id="pone.0001905-Enright1"></a>Enright BP, Kubota C, Yang X, Tian XC (2003) Epigenetic characteristics and development of embryos cloned from donor cells treated by trichostatin A or 5-aza-2′-deoxycytidine. Biol Reprod  69: 896–901.  <ul class="find" data-citedArticleID="1084226" data-doi="10.1095/biolreprod.103.017954"><li><a href="http://dx.doi.org/10.1095/biolreprod.103.017954" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+characteristics+and+development+of+embryos+cloned+from+donor+cells+treated+by+trichostatin+A+or+5-aza-2%E2%80%B2-deoxycytidine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+characteristics+and+development+of+embryos+cloned+from+donor+cells+treated+by+trichostatin+A+or+5-aza-2%E2%80%B2-deoxycytidine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001905-Kishigami1" id="pone.0001905-Kishigami1"></a>Kishigami S, Mizutani E, Ohta H, Hikichi T, Thuan NV, et al.  (2006) Significant improvement of mouse cloning technique by treatment with trichostatin A after somatic nuclear transfer. Biochem Biophys Res Commun  340: 183–189.  <ul class="find" data-citedArticleID="1084254" data-doi="10.1016/j.bbrc.2005.11.164"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2005.11.164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significant+improvement+of+mouse+cloning+technique+by+treatment+with+trichostatin+A+after+somatic+nuclear+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significant+improvement+of+mouse+cloning+technique+by+treatment+with+trichostatin+A+after+somatic+nuclear+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001905-Wakayama3" id="pone.0001905-Wakayama3"></a>Wakayama T (2007) Production of cloned mice and ES cells from adult somatic cells by nuclear transfer: how to improve cloning efficiency? J Reprod Dev  53: 13–26.  <ul class="find" data-citedArticleID="1084294" data-doi="10.1262/jrd.18120"><li><a href="http://dx.doi.org/10.1262/jrd.18120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+of+cloned+mice+and+ES+cells+from+adult+somatic+cells+by+nuclear+transfer%3A+how+to+improve+cloning+efficiency%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+of+cloned+mice+and+ES+cells+from+adult+somatic+cells+by+nuclear+transfer%3A+how+to+improve+cloning+efficiency%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001905-Kishigami2" id="pone.0001905-Kishigami2"></a>Kishigami S, Bui HT, Wakayama S, Tokunaga K, Van Thuan N, et al.  (2007) Successful mouse cloning of an outbred strain by trichostatin A treatment after somatic nuclear transfer. J Reprod Dev  53: 165–170.  <ul class="find" data-citedArticleID="1084256" data-doi="10.1262/jrd.18098"><li><a href="http://dx.doi.org/10.1262/jrd.18098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Successful+mouse+cloning+of+an+outbred+strain+by+trichostatin+A+treatment+after+somatic+nuclear+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Successful+mouse+cloning+of+an+outbred+strain+by+trichostatin+A+treatment+after+somatic+nuclear+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001905-Wakayama4" id="pone.0001905-Wakayama4"></a>Wakayama T, Yanagimachi R (1999) Cloning the laboratory mouse. Semin Cell Dev Biol  10: 253–258.  <ul class="find" data-citedArticleID="1084296" data-doi="10.1006/scdb.1998.0267"><li><a href="http://dx.doi.org/10.1006/scdb.1998.0267" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+the+laboratory+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+the+laboratory+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001905-Tamashiro1" id="pone.0001905-Tamashiro1"></a>Tamashiro KL, Wakayama T, Akutsu H, Yamazaki Y, Lachey JL, et al.  (2002) Cloned mice have an obese phenotype not transmitted to their offspring. Nat Med  8: 262–267.  <ul class="find" data-citedArticleID="1084284" data-doi="10.1038/nm0302-262"><li><a href="http://dx.doi.org/10.1038/nm0302-262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloned+mice+have+an+obese+phenotype+not+transmitted+to+their+offspring." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloned+mice+have+an+obese+phenotype+not+transmitted+to+their+offspring.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001905-Tanaka1" id="pone.0001905-Tanaka1"></a>Tanaka S, Oda M, Toyoshima Y, Wakayama T, Tanaka M, et al.  (2001) Placentomegaly in cloned mouse concepti caused by expansion of the spongiotrophoblast layer. Biol Reprod  65: 1813–1821.  <ul class="find" data-citedArticleID="1084286" data-doi="10.1095/biolreprod65.6.1813"><li><a href="http://dx.doi.org/10.1095/biolreprod65.6.1813" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Placentomegaly+in+cloned+mouse+concepti+caused+by+expansion+of+the+spongiotrophoblast+layer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Placentomegaly+in+cloned+mouse+concepti+caused+by+expansion+of+the+spongiotrophoblast+layer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001905-Ohgane1" id="pone.0001905-Ohgane1"></a>Ohgane J, Wakayama T, Senda S, Yamazaki Y, Inoue K, et al.  (2004) The Sall3 locus is an epigenetic hotspot of aberrant DNA methylation associated with placentomegaly of cloned mice. Genes Cells  9: 253–260.  <ul class="find" data-citedArticleID="1084268" data-doi="10.1111/j.1356-9597.2004.00720.x"><li><a href="http://dx.doi.org/10.1111/j.1356-9597.2004.00720.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Sall3+locus+is+an+epigenetic+hotspot+of+aberrant+DNA+methylation+associated+with+placentomegaly+of+cloned+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Sall3+locus+is+an+epigenetic+hotspot+of+aberrant+DNA+methylation+associated+with+placentomegaly+of+cloned+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001905-Barker1" id="pone.0001905-Barker1"></a>Barker DJ (2007) The origins of the developmental origins theory. J Intern Med  261: 412–417.  <ul class="find" data-citedArticleID="1084204" data-doi="10.1111/j.1365-2796.2007.01809.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2796.2007.01809.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origins+of+the+developmental+origins+theory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origins+of+the+developmental+origins+theory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001905-Barker2" id="pone.0001905-Barker2"></a>Barker DJ, Bagby SP, Hanson MA (2006) Mechanisms of disease: in utero programming in the pathogenesis of hypertension. Nat Clin Pract Nephrol  2: 700–707.  <ul class="find" data-citedArticleID="1084206" data-doi="10.1038/ncpneph0344"><li><a href="http://dx.doi.org/10.1038/ncpneph0344" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+disease%3A+in+utero+programming+in+the+pathogenesis+of+hypertension." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+disease%3A+in+utero+programming+in+the+pathogenesis+of+hypertension.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001905-Yu1" id="pone.0001905-Yu1"></a>Yu Y, Ding C, Wang E, Chen X, Li X, et al.  (2007) Piezo-assisted nuclear transfer affects cloning efficiency and may cause apoptosis. Reproduction  133: 947–954.  <ul class="find" data-citedArticleID="1084304" data-doi="10.1530/rep-06-0358"><li><a href="http://dx.doi.org/10.1530/rep-06-0358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Piezo-assisted+nuclear+transfer+affects+cloning+efficiency+and+may+cause+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Piezo-assisted+nuclear+transfer+affects+cloning+efficiency+and+may+cause+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001905-Humpherys1" id="pone.0001905-Humpherys1"></a>Humpherys D, Eggan K, Akutsu H, Friedman A, Hochedlinger K, et al.  (2002) Abnormal gene expression in cloned mice derived from embryonic stem cell and cumulus cell nuclei. Proc Natl Acad Sci U S A  99: 12889–12894.  <ul class="find" data-citedArticleID="1084242" data-doi="10.1073/pnas.192433399"><li><a href="http://dx.doi.org/10.1073/pnas.192433399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Abnormal+gene+expression+in+cloned+mice+derived+from+embryonic+stem+cell+and+cumulus+cell+nuclei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Abnormal+gene+expression+in+cloned+mice+derived+from+embryonic+stem+cell+and+cumulus+cell+nuclei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001905-Inoue1" id="pone.0001905-Inoue1"></a>Inoue K, Kohda T, Lee J, Ogonuki N, Mochida K, et al.  (2002) Faithful expression of imprinted genes in cloned mice. Science  295: 297.  <ul class="find" data-citedArticleID="1084246" data-doi="10.1126/science.295.5553.297"><li><a href="http://dx.doi.org/10.1126/science.295.5553.297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Faithful+expression+of+imprinted+genes+in+cloned+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Faithful+expression+of+imprinted+genes+in+cloned+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001905-Kohda1" id="pone.0001905-Kohda1"></a>Kohda T, Inoue K, Ogonuki N, Miki H, Naruse M, et al.  (2005) Variation in gene expression and aberrantly regulated chromosome regions in cloned mice. Biol Reprod  73: 1302–1311.  <ul class="find" data-citedArticleID="1084258" data-doi="10.1095/biolreprod.105.044958"><li><a href="http://dx.doi.org/10.1095/biolreprod.105.044958" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+gene+expression+and+aberrantly+regulated+chromosome+regions+in+cloned+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+gene+expression+and+aberrantly+regulated+chromosome+regions+in+cloned+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001905-Tong1" id="pone.0001905-Tong1"></a>Tong GQ, Heng BC, Tan LG, Ng SC (2006) Aberrant profile of gene expression in cloned mouse embryos derived from donor cumulus nuclei. Cell Tissue Res  325: 231–243.  <ul class="find" data-citedArticleID="1084288" data-doi="10.1007/s00441-006-0189-1"><li><a href="http://dx.doi.org/10.1007/s00441-006-0189-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aberrant+profile+of+gene+expression+in+cloned+mouse+embryos+derived+from+donor+cumulus+nuclei." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aberrant+profile+of+gene+expression+in+cloned+mouse+embryos+derived+from+donor+cumulus+nuclei.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001905-Chen1" id="pone.0001905-Chen1"></a>Chen ZX, Riggs AD (2005) Maintenance and regulation of DNA methylation patterns in mammals. Biochem Cell Biol  83: 438–448.  <ul class="find" data-citedArticleID="1084216" data-doi="10.1139/o05-138"><li><a href="http://dx.doi.org/10.1139/o05-138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+and+regulation+of+DNA+methylation+patterns+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+and+regulation+of+DNA+methylation+patterns+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001905-Dodd1" id="pone.0001905-Dodd1"></a>Dodd IB, Micheelsen MA, Sneppen KGT (2007) Theoretical Analysis of Epigenetic Cell Memoryby Nucleosome Modification. Cell  129: 813–822.  <ul class="find" data-citedArticleID="1084224" data-doi="10.1016/j.cell.2007.02.053"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.02.053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theoretical+Analysis+of+Epigenetic+Cell+Memoryby+Nucleosome+Modification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theoretical+Analysis+of+Epigenetic+Cell+Memoryby+Nucleosome+Modification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001905-Yang1" id="pone.0001905-Yang1"></a>Yang F, Hao R, Kessler B, Brem G, Wolf E, et al.  (2007) Rabbit somatic cell cloning: effects of donor cell type, histone acetylation status and chimeric embryo complementation. Reproduction  133: 219–230.  <ul class="find" data-citedArticleID="1084300" data-doi="10.1530/rep.1.01206"><li><a href="http://dx.doi.org/10.1530/rep.1.01206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rabbit+somatic+cell+cloning%3A+effects+of+donor+cell+type%2C+histone+acetylation+status+and+chimeric+embryo+complementation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rabbit+somatic+cell+cloning%3A+effects+of+donor+cell+type%2C+histone+acetylation+status+and+chimeric+embryo+complementation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001905-Yang2" id="pone.0001905-Yang2"></a>Yang J, Yang S, Beaujean N, Niu Y, He X, et al.  (2007) Epigenetic marks in cloned rhesus monkey embryos: comparison with counterparts produced in vitro. Biol Reprod  76: 36–42.  <ul class="find" data-citedArticleID="1084302" data-doi="10.1095/biolreprod.106.051383"><li><a href="http://dx.doi.org/10.1095/biolreprod.106.051383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+marks+in+cloned+rhesus+monkey+embryos%3A+comparison+with+counterparts+produced+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+marks+in+cloned+rhesus+monkey+embryos%3A+comparison+with+counterparts+produced+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001905-Bernstein1" id="pone.0001905-Bernstein1"></a>Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, et al.  (2005) Genomic maps and comparative analysis of histone modifications in human and mouse. Cell  120: 169–181.  <ul class="find" data-citedArticleID="1084208" data-doi="10.1016/j.cell.2005.01.001"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001905-Nishida1" id="pone.0001905-Nishida1"></a>Nishida H, Suzuki T, Kondo S, Miura H, Fujimura Y, et al.  (2006) Histone H3 acetylated at lysine 9 in promoter is associated with low nucleosome density in the vicinity of transcription start site in human cell. Chromosome Res  14: 203–211.  <ul class="find" data-citedArticleID="1084264" data-doi="10.1007/s10577-006-1036-7"><li><a href="http://dx.doi.org/10.1007/s10577-006-1036-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H3+acetylated+at+lysine+9+in+promoter+is+associated+with+low+nucleosome+density+in+the+vicinity+of+transcription+start+site+in+human+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H3+acetylated+at+lysine+9+in+promoter+is+associated+with+low+nucleosome+density+in+the+vicinity+of+transcription+start+site+in+human+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001905-Heintzman1" id="pone.0001905-Heintzman1"></a>Heintzman ND, Stuart RK, Hon G, Fu Y, Ching CW, et al.  (2007) Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome. Nat Genet  39: 311–318.  <ul class="find" data-citedArticleID="1084240" data-doi="10.1038/ng1966"><li><a href="http://dx.doi.org/10.1038/ng1966" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+and+predictive+chromatin+signatures+of+transcriptional+promoters+and+enhancers+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+and+predictive+chromatin+signatures+of+transcriptional+promoters+and+enhancers+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001905-Roh1" id="pone.0001905-Roh1"></a>Roh TY, Cuddapah S, Zhao K (2005) Active chromatin domains are defined by acetylation islands revealed by genome-wide mapping. Genes Dev  19: 542–552.  <ul class="find" data-citedArticleID="1084274" data-doi="10.1101/gad.1272505"><li><a href="http://dx.doi.org/10.1101/gad.1272505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Active+chromatin+domains+are+defined+by+acetylation+islands+revealed+by+genome-wide+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Active+chromatin+domains+are+defined+by+acetylation+islands+revealed+by+genome-wide+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001905-Kapranov1" id="pone.0001905-Kapranov1"></a>Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, et al.  (2002) Large-scale transcriptional activity in chromosomes 21 and 22. Science  296: 916–919.  <ul class="find" data-citedArticleID="1084250" data-doi="10.1126/science.1068597"><li><a href="http://dx.doi.org/10.1126/science.1068597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+transcriptional+activity+in+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+transcriptional+activity+in+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001905-Kampa1" id="pone.0001905-Kampa1"></a>Kampa D, Cheng J, Kapranov P, Yamanaka M, Brubaker S, et al.  (2004) Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22. Genome Res  14: 331–342.  <ul class="find" data-citedArticleID="1084248" data-doi="10.1101/gr.2094104"><li><a href="http://dx.doi.org/10.1101/gr.2094104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+RNAs+identified+from+an+in-depth+analysis+of+the+transcriptome+of+human+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+RNAs+identified+from+an+in-depth+analysis+of+the+transcriptome+of+human+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001905-Cheng1" id="pone.0001905-Cheng1"></a>Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, et al.  (2005) Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution. Science  308: 1149–1154.  <ul class="find" data-citedArticleID="1084220" data-doi="10.1126/science.1108625"><li><a href="http://dx.doi.org/10.1126/science.1108625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001905-Hasegawa1" id="pone.0001905-Hasegawa1"></a>Hasegawa Y, Fukuda S, Shimokawa K, Kondo S, Maeda N, et al.  (2006) A RecA-mediated exon profiling method. Nucleic Acids Res  34: e97.  <ul class="find" data-citedArticleID="1084238" data-doi="10.1093/nar/gkl497"><li><a href="http://dx.doi.org/10.1093/nar/gkl497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+RecA-mediated+exon+profiling+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+RecA-mediated+exon+profiling+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001905-Cawley1" id="pone.0001905-Cawley1"></a>Cawley S, Bekiranov S, Ng HH, Kapranov P, Sekinger EA, et al.  (2004) Unbiased mapping of transcription factor binding sites along human chromosomes 21 and 22 points to widespread regulation of noncoding RNAs. Cell  116: 499–509.  <ul class="find" data-citedArticleID="1084212" data-doi="10.1016/s0092-8674(04)00127-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00127-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unbiased+mapping+of+transcription+factor+binding+sites+along+human+chromosomes+21+and+22+points+to+widespread+regulation+of+noncoding+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unbiased+mapping+of+transcription+factor+binding+sites+along+human+chromosomes+21+and+22+points+to+widespread+regulation+of+noncoding+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001905-Okazaki1" id="pone.0001905-Okazaki1"></a>Okazaki Y, Furuno M, Kasukawa T, Adachi J, Bono H, et al.  (2002) Analysis of the mouse transcriptome based on functional annotation of 60,770 full-length cDNAs. Nature  420: 563–573.  <ul class="find" data-citedArticleID="1084272" data-doi="10.1038/nature01266"><li><a href="http://dx.doi.org/10.1038/nature01266" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+mouse+transcriptome+based+on+functional+annotation+of+60%2C770+full-length+cDNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+mouse+transcriptome+based+on+functional+annotation+of+60%2C770+full-length+cDNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001905-Kent1" id="pone.0001905-Kent1"></a>Kent WJ (2002) BLAT–the BLAST-like alignment tool. Genome Res  12: 656–664.  <ul class="find" data-citedArticleID="1084252" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT%E2%80%93the+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT%E2%80%93the+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001905-Florea1" id="pone.0001905-Florea1"></a>Florea L, Hartzell G, Zhang Z, Rubin GM, Miller W (1998) A computer program for aligning a cDNA sequence with a genomic DNA sequence. Genome Res  8: 967–974.  <ul class="find" data-citedArticleID="1084230"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence.&amp;auth=&amp;atitle=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001905-Guenther1" id="pone.0001905-Guenther1"></a>Guenther MG, Levine SS, Boyer LA, Jaenisch R, Young RA (2007) A chromatin landmark and transcription initiation at most promoters in human cells. Cell  130: 77–88.  <ul class="find" data-citedArticleID="1084234" data-doi="10.1016/j.cell.2007.05.042"><li><a href="http://dx.doi.org/10.1016/j.cell.2007.05.042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chromatin+landmark+and+transcription+initiation+at+most+promoters+in+human+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chromatin+landmark+and+transcription+initiation+at+most+promoters+in+human+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001905-Nishida2" id="pone.0001905-Nishida2"></a>Nishida H, Suzuki T, Ookawa H, Tomaru Y, Hayashizaki Y (2005) Comparative analysis of expression of histone H2a genes in mouse. BMC Genomics  6: 108.  <ul class="find" data-citedArticleID="1084266" data-doi="10.1186/1471-2164-6-108"><li><a href="http://dx.doi.org/10.1186/1471-2164-6-108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+analysis+of+expression+of+histone+H2a+genes+in+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+analysis+of+expression+of+histone+H2a+genes+in+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001905-Livak1" id="pone.0001905-Livak1"></a>Livak KJ, Schmittgen TD (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods  25: 402–408.  <ul class="find" data-citedArticleID="1084260" data-doi="10.1006/meth.2001.1262"><li><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001905-Balbach1" id="pone.0001905-Balbach1"></a>Balbach ST, Jauch A, Bohm-Steuer B, Cavaleri FM, Han YM, et al.  (2007) Chromosome stability differs in cloned mouse embryos and derivative ES cells. Dev Biol  308: 309–321.  <ul class="find" data-citedArticleID="1084202" data-doi="10.1016/j.ydbio.2007.05.034"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2007.05.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+stability+differs+in+cloned+mouse+embryos+and+derivative+ES+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+stability+differs+in+cloned+mouse+embryos+and+derivative+ES+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001905-Ohgane2" id="pone.0001905-Ohgane2"></a>Ohgane J, Wakayama T, Kogo Y, Senda S, Hattori N, et al.  (2001) DNA methylation variation in cloned mice. Genesis  30: 45–50.  <ul class="find" data-citedArticleID="1084270" data-doi="10.1002/gene.1031"><li><a href="http://dx.doi.org/10.1002/gene.1031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+variation+in+cloned+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+variation+in+cloned+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001905-Sandelin1" id="pone.0001905-Sandelin1"></a>Sandelin A, Carninci P, Lenhard B, Ponjavic J, Hayashizaki Y, et al.  (2007) Mammalian RNA polymerase II core promoters: insights from genome-wide studies. Nat Rev Genet.   <ul class="find" data-citedArticleID="1084276" data-doi="10.1038/nrg2026"><li><a href="http://dx.doi.org/10.1038/nrg2026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+RNA+polymerase+II+core+promoters%3A+insights+from+genome-wide+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+RNA+polymerase+II+core+promoters%3A+insights+from+genome-wide+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001905-Szmitko1" id="pone.0001905-Szmitko1"></a>Szmitko PE, Verma S (2006) C-Reactive Protein and the Metabolic Syndrome: Useful Addition to the Cardiovascular Risk Profile? J Cardiometab Syndr  1: 66–69.  <ul class="find" data-citedArticleID="1084282" data-doi="10.1111/j.0197-3118.2006.05340.x"><li><a href="http://dx.doi.org/10.1111/j.0197-3118.2006.05340.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=C-Reactive+Protein+and+the+Metabolic+Syndrome%3A+Useful+Addition+to+the+Cardiovascular+Risk+Profile%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22C-Reactive+Protein+and+the+Metabolic+Syndrome%3A+Useful+Addition+to+the+Cardiovascular+Risk+Profile%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001905-Farooqi1" id="pone.0001905-Farooqi1"></a>Farooqi IS, O'Rahilly S (2007) Is leptin an important physiological regulator of CRP? Nat Med  13: 16–17; author reply 19–21.  <ul class="find" data-citedArticleID="1084228" data-doi="10.1038/nm0107-16"><li><a href="http://dx.doi.org/10.1038/nm0107-16" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+leptin+an+important+physiological+regulator+of+CRP%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+leptin+an+important+physiological+regulator+of+CRP%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001905-Gertler1" id="pone.0001905-Gertler1"></a>Gertler A, Niv-Spector L, Reicher S (2007) Is leptin an important physiological regulator of CRP? Nat Med  13: 18–19; author reply 19–21.  <ul class="find" data-citedArticleID="1084232" data-doi="10.1038/nm0107-18"><li><a href="http://dx.doi.org/10.1038/nm0107-18" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+leptin+an+important+physiological+regulator+of+CRP%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+leptin+an+important+physiological+regulator+of+CRP%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001905-Hutchinson1" id="pone.0001905-Hutchinson1"></a>Hutchinson WL, Coll AP, Gallimore JR, Tennent GA, Pepys MB (2007) Is leptin an important physiological regulator of CRP? Nat Med  13: 17–18; author reply 19–21.  <ul class="find" data-citedArticleID="1084244" data-doi="10.1038/nm0107-17"><li><a href="http://dx.doi.org/10.1038/nm0107-17" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Is+leptin+an+important+physiological+regulator+of+CRP%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Is+leptin+an+important+physiological+regulator+of+CRP%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001905-Chen2" id="pone.0001905-Chen2"></a>Chen K, Li F, Li J, Cai H, Strom S, et al.  (2006) Induction of leptin resistance through direct interaction of C-reactive protein with leptin. Nat Med  12: 425–432.  <ul class="find" data-citedArticleID="1084218" data-doi="10.1038/nm1372"><li><a href="http://dx.doi.org/10.1038/nm1372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+leptin+resistance+through+direct+interaction+of+C-reactive+protein+with+leptin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+leptin+resistance+through+direct+interaction+of+C-reactive+protein+with+leptin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001905-McGregor1" id="pone.0001905-McGregor1"></a>McGregor GP, Desaga JF, Ehlenz K, Fischer A, Heese F, et al.  (1996) Radiommunological measurement of leptin in plasma of obese and diabetic human subjects. Endocrinology  137: 1501–1504.  <ul class="find" data-citedArticleID="1084262" data-doi="10.1210/endo.137.4.8625930"><li><a href="http://dx.doi.org/10.1210/endo.137.4.8625930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Radiommunological+measurement+of+leptin+in+plasma+of+obese+and+diabetic+human+subjects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Radiommunological+measurement+of+leptin+in+plasma+of+obese+and+diabetic+human+subjects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001905-Shimizu1" id="pone.0001905-Shimizu1"></a>Shimizu H, Shimomura Y, Hayashi R, Ohtani K, Sato N, et al.  (1997) Serum leptin concentration is associated with total body fat mass, but not abdominal fat distribution. Int J Obes Relat Metab Disord  21: 536–541.  <ul class="find" data-citedArticleID="1084280" data-doi="10.1038/sj.ijo.0800437"><li><a href="http://dx.doi.org/10.1038/sj.ijo.0800437" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serum+leptin+concentration+is+associated+with+total+body+fat+mass%2C+but+not+abdominal+fat+distribution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serum+leptin+concentration+is+associated+with+total+body+fat+mass%2C+but+not+abdominal+fat+distribution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001905-Gura1" id="pone.0001905-Gura1"></a>Gura T (1999) Obesity research. Leptin not impressive in clinical trial. Science  286: 881–882.  <ul class="find" data-citedArticleID="1084236" data-doi="10.1126/science.286.5441.881a"><li><a href="http://dx.doi.org/10.1126/science.286.5441.881a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Obesity+research.+Leptin+not+impressive+in+clinical+trial." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Obesity+research.+Leptin+not+impressive+in+clinical+trial.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001905-Bjorbaek1" id="pone.0001905-Bjorbaek1"></a>Bjorbaek C, El-Haschimi K, Frantz JD, Flier JS (1999) The role of SOCS-3 in leptin signaling and leptin resistance. J Biol Chem  274: 30059–30065.  <ul class="find" data-citedArticleID="1084210" data-doi="10.1074/jbc.274.42.30059"><li><a href="http://dx.doi.org/10.1074/jbc.274.42.30059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+SOCS-3+in+leptin+signaling+and+leptin+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+SOCS-3+in+leptin+signaling+and+leptin+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001905-Senda1" id="pone.0001905-Senda1"></a>Senda S, Wakayama T, Arai Y, Yamazaki Y, Ohgane J, et al.  (2007) DNA methylation errors in cloned mice disappear with advancement of aging. Cloning Stem Cells  9: 293–302.  <ul class="find" data-citedArticleID="1084278" data-doi="10.1089/clo.2006.0050"><li><a href="http://dx.doi.org/10.1089/clo.2006.0050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+errors+in+cloned+mice+disappear+with+advancement+of+aging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+errors+in+cloned+mice+disappear+with+advancement+of+aging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001905-Wakayama5" id="pone.0001905-Wakayama5"></a>Wakayama S, Mizutani E, Kishigami S, Thuan NV, Ohta H, et al.  (2005) Mice cloned by nuclear transfer from somatic and ntES cells derived from the same individuals. J Reprod Dev  51: 765–772.  <ul class="find" data-citedArticleID="1084298" data-doi="10.1262/jrd.17061"><li><a href="http://dx.doi.org/10.1262/jrd.17061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mice+cloned+by+nuclear+transfer+from+somatic+and+ntES+cells+derived+from+the+same+individuals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mice+cloned+by+nuclear+transfer+from+somatic+and+ntES+cells+derived+from+the+same+individuals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001905-Chatot1" id="pone.0001905-Chatot1"></a>Chatot CL, Ziomek CA, Bavister BD, Lewis JL, Torres I (1989) An improved culture medium supports development of random-bred 1-cell mouse embryos in vitro. J Reprod Fertil  86: 679–688.  <ul class="find" data-citedArticleID="1084214" data-doi="10.1530/jrf.0.0860679"><li><a href="http://dx.doi.org/10.1530/jrf.0.0860679" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+culture+medium+supports+development+of+random-bred+1-cell+mouse+embryos+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+culture+medium+supports+development+of+random-bred+1-cell+mouse+embryos+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001905-Chomczynski1" id="pone.0001905-Chomczynski1"></a>Chomczynski P, Sacchi N (1987) Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Anal Biochem  162: 156–159.  <ul class="find" data-citedArticleID="1084222" data-doi="10.1016/0003-2697(87)90021-2"><li><a href="http://dx.doi.org/10.1016/0003-2697(87)90021-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-step+method+of+RNA+isolation+by+acid+guanidinium+thiocyanate-phenol-chloroform+extraction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-step+method+of+RNA+isolation+by+acid+guanidinium+thiocyanate-phenol-chloroform+extraction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.XML" value="112552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.PDF" value="311381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g001.PNG_L" value="1117845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g001.PNG_M" value="88342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g001.PNG_S" value="7883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g001.TIF" value="2321248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g001.PNG_I" value="31735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t001.PNG_L" value="344353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t001.PNG_M" value="192729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t001.PNG_S" value="16237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t001.TIF" value="1468136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t001.PNG_I" value="57479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t002.PNG_L" value="286341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t002.PNG_M" value="160181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t002.PNG_S" value="14846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t002.TIF" value="1238428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.t002.PNG_I" value="48539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g002.PNG_L" value="166151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g002.PNG_M" value="44844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g002.PNG_S" value="5310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g002.TIF" value="248016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g002.PNG_I" value="17089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g003.PNG_L" value="638062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g003.PNG_M" value="85353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g003.PNG_S" value="13221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g003.TIF" value="1204582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g003.PNG_I" value="60904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g004.PNG_L" value="1099491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g004.PNG_M" value="155576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g004.PNG_S" value="13434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g004.TIF" value="2051892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.g004.PNG_I" value="73026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.s001.XLS" value="22016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.s002.XLS" value="46080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001905.s003.XLS" value="23552"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001905&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001905" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001905&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001905&volume=&issue=&title=Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice&author_name=Takahiro%20Suzuki%2C%20Shinji%20Kondo%2C%20Teruhiko%20Wakayama%2C%20Paul%20E.%20Cizdziel%2C%20Yoshihide%20Hayashizaki&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001905" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905&amp;t=Genome-Wide%20Analysis%20of%20Abnormal%20H3K9%20Acetylation%20in%20Cloned%20Mice" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905&title=Genome-Wide%20Analysis%20of%20Abnormal%20H3K9%20Acetylation%20in%20Cloned%20Mice&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905&amp;title=Genome-Wide%20Analysis%20of%20Abnormal%20H3K9%20Acetylation%20in%20Cloned%20Mice" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001905&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome-Wide Analysis of Abnormal H3K9 Acetylation in Cloned Mice';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001905';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome-Wide%20Analysis%20of%20Abnormal%20H3K9%20Acetylation%20in%20Cloned%20Mice http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001905" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001905" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acetylation%22" title="Search for articles in the subject area:'Acetylation'"><div class="flagText">Acetylation</div></a>
              <div data-categoryid="21455" data-articleid="26634"
                   data-categoryname="Acetylation"
                   class="flagImage" title="Flag 'Acetylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40167" data-articleid="26634"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cloning%22" title="Search for articles in the subject area:'Cloning'"><div class="flagText">Cloning</div></a>
              <div data-categoryid="23551" data-articleid="26634"
                   data-categoryname="Cloning"
                   class="flagImage" title="Flag 'Cloning' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Epigenetics%22" title="Search for articles in the subject area:'Epigenetics'"><div class="flagText">Epigenetics</div></a>
              <div data-categoryid="18227" data-articleid="26634"
                   data-categoryname="Epigenetics"
                   class="flagImage" title="Flag 'Epigenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26634"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="26634"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="26634"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Promoter+regions%22" title="Search for articles in the subject area:'Promoter regions'"><div class="flagText">Promoter regions</div></a>
              <div data-categoryid="32767" data-articleid="26634"
                   data-categoryname="Promoter regions"
                   class="flagImage" title="Flag 'Promoter regions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6751'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8408'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7229&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
