

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mosaic Genome Architecture of the Anopheles gambiae Species Complex</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001249"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001249" />

    <meta name="citation_title" content="Mosaic Genome Architecture of the Anopheles gambiae Species Complex"/>
    <meta itemprop="name" content="Mosaic Genome Architecture of the Anopheles gambiae Species Complex"/>

      <meta name="citation_author" content="Rui Wang-Sattler"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>
      <meta name="citation_author" content="Stephanie Blandin"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>
      <meta name="citation_author" content="Ye Ning"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>
      <meta name="citation_author" content="Claudia Blass"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>
      <meta name="citation_author" content="Guimogo Dolo"/>
            <meta name="citation_author_institution" content="Faculté de Médicine, de Pharmacie et d'Odonto-Stomatologie, Université du Mali, Bamako, Mali"/>
      <meta name="citation_author" content="Yeya T. Touré"/>
            <meta name="citation_author_institution" content="Faculté de Médicine, de Pharmacie et d'Odonto-Stomatologie, Université du Mali, Bamako, Mali"/>
      <meta name="citation_author" content="Alessandra della Torre"/>
            <meta name="citation_author_institution" content="Dipartimento di Scienze di Sanità Pubblica, Sezione di Parassitologia, Università degli Studi di Roma-La Sapienza, Roma, Italy"/>
      <meta name="citation_author" content="Gregory C. Lanzaro"/>
            <meta name="citation_author_institution" content="Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California at Davis, Davis, California, United States of America"/>
      <meta name="citation_author" content="Lars M. Steinmetz"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory, Heidelberg, Germany"/>
      <meta name="citation_author" content="Fotis C. Kafatos"/>
            <meta name="citation_author_institution" content="Section of Infection and Immunity, Faculty of Natural Sciences, Imperial College London, London, United Kingdom"/>
      <meta name="citation_author" content="Liangbiao Zheng"/>
            <meta name="citation_author_institution" content="Shanghai Institute of Plant Physiology and Ecology, Shanghai, China"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001249.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1249"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The causes of phylogenetic conflict in a classic Drosophila species group.; citation_author=CA Machado; citation_author=J Hey; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=270; citation_number=1; citation_pages=1193-1202; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Anopheles gambiae complex and disease transmission in Africa.; citation_author=GB White; citation_journal_title=Trans R Soc Trop Med Hyg; citation_volume=68; citation_number=2; citation_pages=278-301; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Chromosomal differentiation and adaptation to human environments in the Anopheles gambiae complex.; citation_author=M Coluzzi; citation_author=A Sabatini; citation_author=V Petrarca; citation_author=MA Di Deco; citation_journal_title=Trans R Soc Trop Med Hyg; citation_volume=73; citation_number=3; citation_pages=483-497; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=The Anopheles gambiae complex: a new species from Ethiopia.; citation_author=RH Hunt; citation_author=M Coetzee; citation_author=M Fettene; citation_journal_title=Trans R Soc Trop Med Hyg; citation_volume=92; citation_number=4; citation_pages=231-235; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A polytene chromosome analysis of the Anopheles gambiae species complex.; citation_author=M Coluzzi; citation_author=A Sabatini; citation_author=A della Torre; citation_author=MA Di Deco; citation_author=V Petrarca; citation_journal_title=Science; citation_volume=298; citation_number=5; citation_pages=1415-1418; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Chromosome speciation: humans, Drosophila, and mosquitoes.; citation_author=FJ Ayala; citation_author=M Coluzzi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=6; citation_pages=6535-6542; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The five mating-types in the anopheles gambiae compelx.; citation_author=G Davidson; citation_journal_title=Riv Malariol; citation_volume=43; citation_number=7; citation_pages=167-183; citation_date=1965; " />
      <meta name="citation_reference" content="citation_title=The distribution and inversion polymorphism of chromosomally recognized taxa of the Anopheles gambiae complex in Mali, West Africa.; citation_author=YT Touré; citation_author=V Petrarca; citation_author=SF Traore; citation_author=A Coulibaly; citation_author=HM Maiga; citation_journal_title=Parassitologia; citation_volume=40; citation_number=8; citation_pages=477-511; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Inversion monophyly in African anopheline malaria vectors.; citation_author=BA Garcia; citation_author=A Caccone; citation_author=KD Mathiopoulos; citation_author=JR Powell; citation_journal_title=Genetics; citation_volume=143; citation_number=9; citation_pages=1313-1320; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Breakpoint structure reveals the unique origin of an interspecific chromosomal inversion (2La) in the Anopheles gambiae complex.; citation_author=IV Sharakhov; citation_author=BJ White; citation_author=MV Sharakhova; citation_author=J Kayondo; citation_author=NF Lobo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=10; citation_pages=6258-6262; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of the Anopheles gambiae complex suggests genetic introgression between principal malaria vectors.; citation_author=NJ Besansky; citation_author=JR Powell; citation_author=A Caccone; citation_author=DM Hamm; citation_author=JA Scott; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=11; citation_pages=6885-6888; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Selective introgression of paracentric inversions between two sibling species of the Anopheles gambiae complex.; citation_author=A della Torre; citation_author=L Merzagora; citation_author=JR Powell; citation_author=M Coluzzi; citation_journal_title=Genetics; citation_volume=146; citation_number=12; citation_pages=239-244; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Differential introgression of chromsomal regions between Anopheles gambiae and An. arabiensis.; citation_author=MA Slotman; citation_author=A Della Torre; citation_author=M Calzetta; citation_author=JR Powell; citation_journal_title=Am J Trop Med Hyg; citation_volume=73; citation_number=13; citation_pages=326-335; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple origins of cytologically identical chromosome inversions in the Anopheles gambiae complex.; citation_author=A Caccone; citation_author=GS Min; citation_author=JR Powell; citation_journal_title=Genetics; citation_volume=150; citation_number=14; citation_pages=807-814; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Semipermeable species boundaries between Anopheles gambiae and Anopheles arabiensis: evidence from multilocus DNA sequence variation.; citation_author=NJ Besansky; citation_author=J Krzywinski; citation_author=T Lehmann; citation_author=F Simard; citation_author=M Kern; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=15; citation_pages=10818-10823; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Chromosomal inversion intergradation and incipient speciation in Anopheles gambiae.; citation_author=M Coluzzi; citation_author=V Petrarca; citation_author=MA Di Deco; citation_journal_title=Boll Zool; citation_volume=52; citation_number=16; citation_pages=45-63; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Gene flow among populations of the malaria vector, Anopheles gambiae, in Mali, West Africa.; citation_author=C Taylor; citation_author=YT Toure; citation_author=J Carnahan; citation_author=DE Norris; citation_author=G Dolo; citation_journal_title=Genetics; citation_volume=157; citation_number=17; citation_pages=743-750; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence of incipient speciation within Anopheles gambiae s.s. in West Africa.; citation_author=A della Torre; citation_author=C Fanello; citation_author=M Akogbeto; citation_author=J Dossou-yovo; citation_author=G Favia; citation_journal_title=Insect Mol Biol; citation_volume=10; citation_number=18; citation_pages=9-18; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Attempts to molecularly distinguish cryptic taxa in Anopheles gambiae s.s.; citation_author=G Gentile; citation_author=M Slotman; citation_author=V Ketmaier; citation_author=JR Powell; citation_author=A Caccone; citation_journal_title=Insect Mol Biol; citation_volume=10; citation_number=19; citation_pages=25-32; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of ribosomal DNA polymorphisms discriminating among chromosomal forms of Anopheles gambiae s.s.; citation_author=G Favia; citation_author=A Lanfrancotti; citation_author=L Spanos; citation_author=I Siden-Kiamos; citation_author=C Louis; citation_journal_title=Insect Mol Biol; citation_volume=10; citation_number=20; citation_pages=19-23; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=When genetic distance matters: measuring genetic differentiation at microsatellite loci in whole-genome scans of recent and incipient mosquito species.; citation_author=R Wang; citation_author=L Zheng; citation_author=YT Toure; citation_author=T Dandekar; citation_author=FC Kafatos; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=21; citation_pages=10769-10774; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sex-linked differentiation between incipient species of Anopheles gambiae.; citation_author=AD Stump; citation_author=JA Shoener; citation_author=C Costantini; citation_author=N Sagnon; citation_author=NJ Besansky; citation_journal_title=Genetics; citation_volume=169; citation_number=22; citation_pages=1509-1519; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Centromere-proximal differentiation and speciation in Anopheles gambiae.; citation_author=AD Stump; citation_author=MC Fitzpatrick; citation_author=NF Lobo; citation_author=S Traore; citation_author=N Sagnon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=23; citation_pages=15930-15935; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic islands of speciation in Anopheles gambiae.; citation_author=TL Turner; citation_author=MW Hahn; citation_author=SV Nuzhdin; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=24; citation_pages=e285; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Population Structure of Anopheles gambiae in Africa.; citation_author=T Lehmann; citation_author=M Licht; citation_author=N Elissa; citation_author=BT Maega; citation_author=JM Chimumbwa; citation_journal_title=J Hered; citation_volume=94; citation_number=25; citation_pages=133-147; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Speciation within Anopheles gambiae–the glass is half full.; citation_author=A della Torre; citation_author=C Costantini; citation_author=NJ Besansky; citation_author=A Caccone; citation_author=V Petrarca; citation_journal_title=Science; citation_volume=298; citation_number=26; citation_pages=115-117; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Current distribution of a pyrethroid resistance gene (kdr) in Anopheles gambiae complex from west Africa and further evidence for reproductive isolation of the Mopti form.; citation_author=F Chandre; citation_author=S Manguin; citation_author=C Brengues; citation_author=J Dossou Yovo; citation_author=F Darriet; citation_journal_title=Parassitologia; citation_volume=41; citation_number=27; citation_pages=319-322; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The kdr mutation occurs in the Mopti form of Anopheles gambiae s.s. through introgression.; citation_author=M Weill; citation_author=F Chandre; citation_author=C Brengues; citation_author=S Manguin; citation_author=M Akogbeto; citation_journal_title=Insect Mol Biol; citation_volume=9; citation_number=28; citation_pages=451-455; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Locus- and population-specific selection and differentiation between incipient species of Anopheles gambiae.; citation_author=TL Turner; citation_author=MW Hahn; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=29; citation_pages=2132-2138; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Evidence for subdivision within the M molecular form of Anopheles gambiae.; citation_author=MA Slotman; citation_author=F Tripet; citation_author=AJ Cornel; citation_author=CR Meneses; citation_author=Y Lee; citation_journal_title=Mol Ecol; citation_volume=16; citation_number=30; citation_pages=639-649; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of the malaria mosquito Anopheles gambiae.; citation_author=RA Holt; citation_author=GM Subramanian; citation_author=A Halpern; citation_author=GG Sutton; citation_author=R Charlab; citation_journal_title=Science; citation_volume=298; citation_number=31; citation_pages=129-149; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=32; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Variation in recombination rate across the X chromosome of Anopheles gambiae.; citation_author=M Pombi; citation_author=AD Stump; citation_author=A Della Torre; citation_author=NJ Besansky; citation_journal_title=Am J Trop Med Hyg; citation_volume=75; citation_number=33; citation_pages=901-903; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reduced recombination rate and genetic differentiation between the M and S forms of Anopheles gambiae s.s.; citation_author=MA Slotman; citation_author=LJ Reimer; citation_author=T Thiemann; citation_author=G Dolo; citation_author=E Fondjo; citation_journal_title=Genetics; citation_volume=174; citation_number=34; citation_pages=2081-2093; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Correspondence analysis. Correspondence analysis.; citation_author=X He; citation_number=35; citation_pages=345-372; citation_date=1998; citation_publisher=RenMen Uni. Publisher; " />
      <meta name="citation_reference" content="citation_title=Microsatellite markers and genotyping procedures for Anopheles gambiae.; citation_author=R Wang; citation_author=FC Kafatos; citation_author=L Zheng; citation_journal_title=Parasitol Today; citation_volume=15; citation_number=36; citation_pages=33-37; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA analysis of transferred sperm reveals significant levels of gene flow between molecular forms of Anopheles gambiae.; citation_author=F Tripet; citation_author=YT Toure; citation_author=CE Taylor; citation_author=DE Norris; citation_author=G Dolo; citation_journal_title=Mol Ecol; citation_volume=10; citation_number=37; citation_pages=1725-1732; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Spatial and habitat distribution of Anopheles gambiae and Anopheles arabiensis (Diptera: Culicidae) in Banambani village, Mali.; citation_author=FE Edillo; citation_author=YT Toure; citation_author=GC Lanzaro; citation_author=G Dolo; citation_author=CE Taylor; citation_journal_title=J Med Entomol; citation_volume=39; citation_number=38; citation_pages=70-77; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetic differentiation in the African malaria vector, Anopheles gambiae s.s., and the problem of taxonomic status.; citation_author=G Gentile; citation_author=A Della Torre; citation_author=B Maegga; citation_author=JR Powell; citation_author=A Caccone; citation_journal_title=Genetics; citation_volume=161; citation_number=39; citation_pages=1561-1578; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Population structure, speciation, and introgression in the Anopheles gambiae complex.; citation_author=JR Powell; citation_author=V Petrarca; citation_author=A della Torre; citation_author=A Caccone; citation_author=M Coluzzi; citation_journal_title=Parassitologia; citation_volume=41; citation_number=40; citation_pages=101-113; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Major ecological transitions in wild sunflowers facilitated by hybridization.; citation_author=LH Rieseberg; citation_author=O Raymond; citation_author=DM Rosenthal; citation_author=Z Lai; citation_author=K Livingstone; citation_journal_title=Science; citation_volume=301; citation_number=41; citation_pages=1211-1216; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of single specimens of the Anopheles gambiae complex by the polymerase chain reaction.; citation_author=JA Scott; citation_author=WG Brogdon; citation_author=FH Collins; citation_journal_title=Am J Trop Med Hyg; citation_volume=49; citation_number=42; citation_pages=520-529; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Molecular identification of sympatric chromosomal forms of Anopheles gambiae and further evidence of their reproductive isolation.; citation_author=G Favia; citation_author=A della Torre; citation_author=M Bagayoko; citation_author=A Lanfrancotti; citation_author=N Sagnon; citation_journal_title=Insect Mol Biol; citation_volume=6; citation_number=43; citation_pages=377-383; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Comparative genomic analysis in the region of a major Plasmodium-refractoriness locus of Anopheles gambiae.; citation_author=D Thomasova; citation_author=LQ Ton; citation_author=RR Copley; citation_author=EM Zdobnov; citation_author=X Wang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=44; citation_pages=8179-8184; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An integrated genetic map of the African human malaria vector mosquito, Anopheles gambiae.; citation_author=L Zheng; citation_author=MQ Benedict; citation_author=AJ Cornel; citation_author=FH Collins; citation_author=FC Kafatos; citation_journal_title=Genetics; citation_volume=143; citation_number=45; citation_pages=941-952; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A technique for nucleic acid in situ hybridization to polytene chromosomes of mosquitoes in the Anopheles gambiae complex.; citation_author=V Kumar; citation_author=FH Collins; citation_journal_title=Insect Mol Biol; citation_volume=3; citation_number=46; citation_pages=41-47; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A detailed genetic map for the X chromosome of the malaria vector, Anopheles gambiae [see comments].; citation_author=L Zheng; citation_author=FH Collins; citation_author=V Kumar; citation_author=FC Kafatos; citation_journal_title=Science; citation_volume=261; citation_number=47; citation_pages=605-608; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Genetic differentiation of Anopheles gambiae populations from East and west Africa: comparison of microsatellite and allozyme loci.; citation_author=T Lehmann; citation_author=WA Hawley; citation_author=L Kamau; citation_author=D Fontenille; citation_author=F Simard; citation_journal_title=Heredity; citation_volume=77 (Pt 2); citation_number=48; citation_pages=192-200; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Integrated genetic map of Anopheles gambiae: use of RAPD polymorphisms for genetic, cytogenetic and STS landmarks.; citation_author=G Dimopoulos; citation_author=L Zheng; citation_author=V Kumar; citation_author=A della Torre; citation_author=FC Kafatos; citation_journal_title=Genetics; citation_volume=143; citation_number=49; citation_pages=953-960; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait loci in Anopheles gambiae controlling the encapsulation response against Plasmodium cynomolgi Ceylon.; citation_author=L Zheng; citation_author=S Wang; citation_author=P Romans; citation_author=H Zhao; citation_author=C Luna; citation_journal_title=BMC Genet; citation_volume=4; citation_number=50; citation_pages=16; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=InSatDb: a microsatellite database of fully sequenced insect genomes.; citation_author=S Archak; citation_author=E Meduri; citation_author=PS Kumar; citation_author=J Nagaraju; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=51; citation_pages=D36-39; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Principal component ananlysis. Principal component ananlysis.; citation_author=X He; citation_number=52; citation_pages=281-315; citation_date=1998; citation_publisher=RenMen Uni. Publisher; " />
      <meta name="citation_reference" content="citation_title=Faktorenanalyse: Springer-Verlag.; citation_author=K Ueberla; citation_number=53; citation_date=1977; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mosaic Genome Architecture of the Anopheles gambiae Species Complex"/>
    <meta name="twitter:description" content="BackgroundAttempts over the last three decades to reconstruct the phylogenetic history of the Anopheles gambiae species complex have been important for developing better strategies to control malaria transmission.MethodologyWe used fingerprint genotyping data from 414 field-collected female mosquitoes at 42 microsatellite loci to infer the evolutionary relationships of four species in the A. gambiae complex, the two major malaria vectors A. gambiae sensu stricto (A. gambiae s.s.) and A. arabiensis, as well as two minor vectors, A. merus and A. melas.Principal FindingsWe identify six taxonomic units, including a clear separation of West and East Africa A. gambiae s.s. S molecular forms. We show that the phylogenetic relationships vary widely between different genomic regions, thus demonstrating the mosaic nature of the genome of these species. The two major malaria vectors are closely related and closer to A. merus than to A. melas at the genome-wide level, which is also true if only autosomes are considered. However, within the Xag inversion region of the X chromosome, the M and two S molecular forms are most similar to A. merus. Near the X centromere, outside the Xag region, the two S forms are highly dissimilar to the other taxa. Furthermore, our data suggest that the centromeric region of chromosome 3 is a strong discriminator between the major and minor malaria vectors.ConclusionsAlthough further studies are needed to elucidate the basis of the phylogenetic variation among the different regions of the genome, the preponderance of sympatric admixtures among taxa strongly favor introgression of different genomic regions between species, rather than lineage sorting of ancestral polymorphism, as a possible mechanism."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001249.g004"/>

  <meta property="og:title" content="Mosaic Genome Architecture of the Anopheles gambiae Species Complex" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8742'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3873'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5052&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001249">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001249" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001249&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mosaic Genome Architecture of the <i>Anopheles gambiae</i> Species Complex
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rui Wang-Sattler
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rui.wang-sattler@gsf.de">rui.wang-sattler@gsf.de</a> (RW); <a href="mailto:f.kafatos@imperial.ac.uk">f.kafatos@imperial.ac.uk</a> (FK)</p>

                <p>Affiliation:
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>

                <p>
                    Current address: GSF-National Research Center for Environment and Health, Institute of Epidemiology, Neuherberg, Germany,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephanie Blandin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ye Ning, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudia Blass, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>

                <p>
                    Current address: German Cancer Research Center, Signaling and Functional Genomics, Heidelberg, Germany,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guimogo Dolo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Faculté de Médicine, de Pharmacie et d'Odonto-Stomatologie, Université du Mali, Bamako, Mali
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yeya T. Touré, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Faculté de Médicine, de Pharmacie et d'Odonto-Stomatologie, Université du Mali, Bamako, Mali
                </p>

                <p>
                    Current address: World Health Organization, Geneva, Switzerland
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alessandra della Torre, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Dipartimento di Scienze di Sanità Pubblica, Sezione di Parassitologia, Università degli Studi di Roma-La Sapienza, Roma, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gregory C. Lanzaro, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Microbiology and Immunology, School of Veterinary Medicine, University of California at Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lars M. Steinmetz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  European Molecular Biology Laboratory, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fotis C. Kafatos
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rui.wang-sattler@gsf.de">rui.wang-sattler@gsf.de</a> (RW); <a href="mailto:f.kafatos@imperial.ac.uk">f.kafatos@imperial.ac.uk</a> (FK)</p>

                <p>Affiliation:
                  Section of Infection and Immunity, Faculty of Natural Sciences, Imperial College London, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Liangbiao Zheng
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Shanghai Institute of Plant Physiology and Ecology, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001249</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001249-g001" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-t001" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-g002" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-t002" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-t003" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-g003" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-t004" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001249-g004" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001249">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001249" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Attempts over the last three decades to reconstruct the phylogenetic history of the <em>Anopheles gambiae</em> species complex have been important for developing better strategies to control malaria transmission.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We used fingerprint genotyping data from 414 field-collected female mosquitoes at 42 microsatellite loci to infer the evolutionary relationships of four species in the <em>A. gambiae</em> complex, the two major malaria vectors <em>A. gambiae sensu stricto</em> (<em>A. gambiae s.s.</em>) and <em>A. arabiensis</em>, as well as two minor vectors, <em>A. merus</em> and <em>A. melas</em>.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We identify six taxonomic units, including a clear separation of West and East Africa <em>A. gambiae s.s.</em> S molecular forms. We show that the phylogenetic relationships vary widely between different genomic regions, thus demonstrating the mosaic nature of the genome of these species. The two major malaria vectors are closely related and closer to <em>A. merus</em> than to <em>A. melas</em> at the genome-wide level, which is also true if only autosomes are considered. However, within the <em>Xag</em> inversion region of the <em>X</em> chromosome, the M and two S molecular forms are most similar to <em>A. merus</em>. Near the <em>X</em> centromere, outside the <em>Xag</em> region, the two S forms are highly dissimilar to the other taxa. Furthermore, our data suggest that the centromeric region of chromosome 3 is a strong discriminator between the major and minor malaria vectors.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec4.p1" name="article1.front1.article-meta1.abstract1.sec4.p1"></a><p>Although further studies are needed to elucidate the basis of the phylogenetic variation among the different regions of the genome, the preponderance of sympatric admixtures among taxa strongly favor introgression of different genomic regions between species, rather than lineage sorting of ancestral polymorphism, as a possible mechanism.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang-Sattler R, Blandin S, Ning Y, Blass C, Dolo G, et al.  (2007) Mosaic Genome Architecture of the <em>Anopheles gambiae</em> Species Complex. PLoS ONE 2(11):
          e1249.
            doi:10.1371/journal.pone.0001249</p><p><strong>Academic Editor: </strong>Matthew Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> July 27, 2007; <strong>Accepted:</strong> November 2, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Wang-Sattler et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Deutsche Forschungsgemeinschaft (SFB 544 to FCK and STE 1422/2–1 to LMS), the John D. and Catherine T. MacArthur Foundation (FCK), the NIH grant AI40408 (GCL and LZ), the Italian Ministry for University and Research, PRIN 2005 programme (AdT), and the EMBO long-term fellowship (SB).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A clear delineation of the divergence of closely related species is critical for understanding the process of speciation. However, reconstruction of historical relationships between closely related species is not a simple task, because these species usually have low levels of genetic divergence, and it is difficult to ascertain whether the recent divergence is the consequence of lineage sorting of ancestral polymorphisms or introgressive hybridization. In addition, the ‘gene tree versus species tree’ problem adds to the difficulty in establishing evolutionary relationships <a href="#pone.0001249-Machado1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The <em>Anopheles gambiae</em> species complex consists of at least seven morphologically indistinguishable and largely sympatric (geographically co-existing) species <a href="#pone.0001249-White1">[2]</a>, <a href="#pone.0001249-Coluzzi1">[3]</a>, <a href="#pone.0001249-Hunt1">[4]</a>, <a href="#pone.0001249-Coluzzi2">[5]</a>, <a href="#pone.0001249-Ayala1">[6]</a>. <em>A. gambiae sensu stricto</em> (<em>s.s.</em>) and <em>A. arabiensis</em> are major malaria vectors, widely distributed in sub-Saharan Africa and are freshwater species. <em>A. quadriannulatus</em> species A and B are also freshwater breeders, but are not vectors, because of their strong zoophily and exophily. <em>A. merus</em> and <em>A. melas</em> are saltwater breeders and are mostly zoophilic but also bite humans in the absence of animals. <em>A. bwambae</em> breeds only in mineral water springs in Uganda and has a role as a local vector in this area. Laboratory crosses produce fertile female and sterile male offspring among all species pairs, and interspecies hybrids are found in nature at a low rate, of about 0.01–0.2% <a href="#pone.0001249-White1">[2]</a>, <a href="#pone.0001249-Davidson1">[7]</a>, <a href="#pone.0001249-Tour1">[8]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The pioneering work of Coluzzi <em>et al</em>. <a href="#pone.0001249-Coluzzi1">[3]</a>, <a href="#pone.0001249-Ayala1">[6]</a> involved using the distribution of fixed inversions observed in polytene chromosomes to infer phylogenetic relationships within the <em>A. gambiae</em> species complex, on the assumption that cytologically identical inversions are monophyletic. <em>A. gambiae s.s.</em> and <em>A. merus</em> are both fixed for a cytogenetically identical inversion, <em>Xag</em>, and thus appear to be sister taxa, whereas <em>A. arabiensis</em> seems more distantly related as it lacks <em>Xag</em> and possesses three different inversions (<em>bcd</em>) that extensively overlap the region of the <em>Xag</em> inversion. The close phylogenetic relationship between <em>A. gambiae s.s.</em> and <em>A. merus</em> was further supported by sequence information from the <em>guanylate cyclase</em> gene on chromosome-<em>X</em> <a href="#pone.0001249-Garcia1">[9]</a> and from the breakpoint region of the <em>2La</em> inversion <a href="#pone.0001249-Sharakhov1">[10]</a>. However, contradictory results were obtained with other molecular markers. Indeed, rDNA and mtDNA sequence analyses suggested that <em>A. gambiae s.s.</em> is more closely related to <em>A. arabiensis</em> than to <em>A. merus</em> and <em>A. melas</em> <a href="#pone.0001249-Besansky1">[11]</a>. Selective introgression has been observed in laboratory crossing of <em>A. gambiae s.s.</em> and <em>A. arabiensis</em>, in which the fate of chromosomal inversions was followed <a href="#pone.0001249-dellaTorre1">[12]</a>, <a href="#pone.0001249-Slotman1">[13]</a>. Heterospecific <em>X</em>-chromosomes are lost very quickly, whereas some autosomal inversions can be introgressed quite freely under laboratory conditions <a href="#pone.0001249-dellaTorre1">[12]</a>. To reconcile these conflicting datasets, Caccone <em>et al</em>. <a href="#pone.0001249-Caccone1">[14]</a> proposed that the <em>A. gambiae</em> genome has a mosaic nature resulting from introgression between species in some chromosomal regions but not in the others. A similar architecture has been observed in species such as <em>Drosophila</em>, and can generate conflicting phylogenetic trees for markers at different regions of the genome <a href="#pone.0001249-Machado1">[1]</a>. Besansky <em>et al</em>. have provided evidence for different phylogenetic relationships in different genomic regions of <em>A. gambiae s.s.</em>, and implied that this species may have acquired traits from other sibling species, facilitating its spread and role as a main vector for human malaria in Africa <a href="#pone.0001249-Besansky2">[15]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In addition to the existence of seven sibling species of the <em>A. gambiae</em> complex, an incipient speciation process has been hypothesized within <em>A. gambiae s.s</em>. Based on non-random associations of several polymorphic chromosomal inversions, five chromosomal forms named Mopti, Savanna, Bamako, Forest and Bissau have been identified in <em>A. gambiae s.s.</em> <a href="#pone.0001249-Tour1">[8]</a>, <a href="#pone.0001249-Coluzzi3">[16]</a>. Field investigations in Mali have documented reproductive isolation of the Mopti, Savanna and Bamako chromosomal forms <a href="#pone.0001249-Tour1">[8]</a>, <a href="#pone.0001249-Taylor1">[17]</a>. However, laboratory crosses between these forms produce fertile offspring <a href="#pone.0001249-Tour1">[8]</a>. Analysis of molecular markers has revealed two molecular forms: M and S <a href="#pone.0001249-dellaTorre2">[18]</a>, <a href="#pone.0001249-Gentile1">[19]</a>, <a href="#pone.0001249-Favia1">[20]</a>. Comparison of molecular with chromosomal forms indicates that in Mali and Burkina Faso, the M-form corresponds to the Mopti chromosomal form, whereas the S-form encompasses the Savanna and Bamako chromosomal forms <a href="#pone.0001249-dellaTorre2">[18]</a>. However, this correlation between chromosomally-defined and molecularly-defined forms does not apply in more humid southern areas <a href="#pone.0001249-dellaTorre2">[18]</a>, <a href="#pone.0001249-Gentile1">[19]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Two chromosomal regions have been found to significantly distinguish the M and S forms, i.e. near the <em>X</em>- and <em>2L</em>-chromosome centromeres. By using a set of 25 microsatellite loci distributed throughout the genome, it was shown that the M and S forms are practically indistinguishable except at two loci, <em>AGXH678</em> and <em>AGXE614</em>, both of which are outside the <em>Xag</em> inversion and close to the <em>X</em>-chromosome centromere and the rDNA locus that originally defined these forms <a href="#pone.0001249-Wang1">[21]</a>. This has been further supported by studies of 17 <em>X</em>-chromosome localized microsatellites <a href="#pone.0001249-Stump1">[22]</a>, sequencing of 22 introns <a href="#pone.0001249-Stump2">[23]</a>, and hybridization to Affymetrix gene expression arrays <a href="#pone.0001249-Turner1">[24]</a>. At locus <em>AGXH678</em>, different genotypes have also been observed between the M and S forms in another genome-wide study of 11 microsatellites <a href="#pone.0001249-Lehmann1">[25]</a>. In addition to the differences near the <em>X</em> centromere, allelic variation in the gene encoding a sodium-gated channel gene (<em>kdr</em>), close to the <em>2L</em> centromere that confers resistance to pyrethroids, is not equally distributed in the M- and S-forms <a href="#pone.0001249-dellaTorre2">[18]</a>, <a href="#pone.0001249-dellaTorre3">[26]</a>, <a href="#pone.0001249-Chandre1">[27]</a>, <a href="#pone.0001249-Weill1">[28]</a>. Variations in the <em>kdr</em> gene and in another region on chromosome <em>2R</em> were also detected in a genome-wide array comparison between the M- and S-forms <a href="#pone.0001249-Turner1">[24]</a>. Recently, sequencing results further confirmed differentiation of the M and S forms in regions close to the <em>X</em>- and <em>2L</em>-centromeres, and variation was reported on the <em>2R</em> chromosome between mosquitoes collected in Cameroon and Mali <a href="#pone.0001249-Turner2">[29]</a>. Subdivision within the M-form has been reported based on 12 microsatellites on the third chromosome <a href="#pone.0001249-Slotman2">[30]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To date, the analysis of the structure of the <em>A. gambiae</em> complex has been limited largely to single genes <a href="#pone.0001249-Garcia1">[9]</a>, <a href="#pone.0001249-Sharakhov1">[10]</a>, <a href="#pone.0001249-Caccone1">[14]</a>, <a href="#pone.0001249-Chandre1">[27]</a>, <a href="#pone.0001249-Weill1">[28]</a>, or either to the <em>X</em>- <a href="#pone.0001249-Stump1">[22]</a>, <a href="#pone.0001249-Stump2">[23]</a> or to the third- chromosome <a href="#pone.0001249-Slotman2">[30]</a>, and only preliminary genome-wide studies between the M- and S-forms have been carried out on 171 mosquitoes by pairwise comparison at 25 markers <a href="#pone.0001249-Wang1">[21]</a>, and by using Affymetrix array hybridization of 14 individuals <a href="#pone.0001249-Turner1">[24]</a>. A genome-wide analysis of the <em>A. gambiae</em> complex in a large sample set is still lacking.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>In this study, 414 field collected mosquitoes, representing both sympatric and allopatric samples, were genotyped at 42 microsatellites loci distributed throughout the genome, and the data were evaluated using three statistical methods. The results provide strong evidence in support of a mosaic genome architecture in the <em>A. gambiae</em> complex. A new taxonomic unit was identified within the S molecular form. Notably, a centromeric region of chromosome <em>3</em> stands out from our data as a strong discriminator between major and minor malaria vectors.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>(a) Variation among 42 microsatellites loci</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To characterize variation at multiple loci, the 414 individuals were grouped into nine populations (M1, M2, S1, S2, S3, A1, A2, R and L), according to species and collection sites (<a href="#pone-0001249-t001">Table 1</a>), and were genotyped at 42 microsatellite loci (<a href="#pone-0001249-g001">Figure 1</a> and <a href="#pone.0001249.s007">Table S1</a>. For raw genotyping data, see <a href="#pone.0001249.s008">Table S2</a>). All 42 markers were either cytogenetically localized or assigned to chromosomal positions by comparison with the genome sequence <a href="#pone.0001249-Holt1">[31]</a>. PCRs were performed for each individual at each locus. All of the <em>A. gambiae s.s.</em> microsatellite primers gave PCR products in all nine populations (<a href="#pone.0001249.s009">Table S3</a>), except at six loci: two close to the <em>X</em> telomere (<em>AGXH145</em> amplifiable only in <em>A. gambiae s.s.</em> and <em>AGXH36</em> not amplifiable in either <em>A. arabiensis</em> or <em>A. melas</em>) and four near the centromere of chromosome <em>3</em> (<em>AG3E35B</em>, <em>AG3E37B</em>, <em>AG3E38B3</em> and <em>3L09-C1</em>). Interestingly, loci <em>AG3E35B</em> and <em>AG3E37B</em> could not be amplified in <em>A. merus</em> and <em>A. melas</em>, indicating significant sequence divergence from the major malaria vectors in these two minor vector species. In most cases the microsatellite loci were polymorphic among all species, except for one monomorphic locus in <em>A. merus</em> (<em>AG3H93</em>) and five loci in <em>A. melas</em> (<em>AG2H197</em>, <em>AG3H555, AG3E34B2, 3L09-C1</em> and <em>AG3E40A1</em>). Remarkably, four of these markers that are monomorphic in all 37 <em>A. melas</em> individuals, are located on the third chromosome, although insufficient sampling could not be ruled out as a cause of this observation.</p>
<div class="figure" id="pone-0001249-g001"><div class="img"><a name="pone-0001249-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The 42 microsatellite markers used in the current study.</span></strong></p><span>doi:10.1371/journal.pone.0001249.g001</span></div><div class="figure" id="pone-0001249-t001"><div class="img"><a name="pone-0001249-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Origin of the 414 mosquito samples.</span></strong></p><span>doi:10.1371/journal.pone.0001249.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To identify potential differences among the nine populations, polymorphisms were characterized in terms of allele size distribution, allele size range, number of alleles and degree of heterozygosity at each locus (<a href="#pone.0001249.s001">Figure S1</a> to <a href="#pone.0001249.s005">Figure S5</a>, and <a href="#pone.0001249.s009">Table S3</a>). In this way, shared or non-shared genetic variation among the nine populations could be explored at each locus. This analysis allows the identification of populations with similar patterns of genetic variation, which are thus expected to be more closely related genetically. For example, at locus <em>AGXH53</em>, the allele size ranges of <em>A. arabiensis</em>, <em>A. merus</em> and <em>A. melas</em> do not overlap, whereas those of the <em>A. gambiae</em> M- and S-forms do overlap with those of <em>A. arabiensis</em> and <em>A. merus</em>. Furthermore, at this locus the allele size distribution is relatively wide in M and S populations, as well as in <em>A. arabiensis</em>, but tightly clustered in <em>A. merus</em> and <em>A. melas</em>. The existence of non-overlapping alleles in certain populations suggests that these populations are distantly related compared with those possessing shared alleles in the regions covered by these markers.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>As reported previously <a href="#pone.0001249-Wang1">[21]</a>, the Mali M and S molecular forms (M2, S1 and S2) are quite comparable in terms of allele size distribution, differing significantly only at two loci, <em>AGXH678</em> and <em>AGXE614</em>. In this study, this was further confirmed at a third locus <em>AGXND6U3</em> (<a href="#pone.0001249.s001">Figure S1</a>), and also for the Burkina Faso M-form (M1), as well as for the Kenya S-form (S3) at these three loci. At locus <em>AGXH678</em>, for example, heterozygosities are lower in M populations (M1 = 29%and M2 = 34%) than in S populations (S1 = 86%, S2 = 92% and S3 = 65%). Heterozygosity does not correlate with sample size, indicating greater genetic variability at this locus in the S-form (<a href="#pone.0001249.s009">Table S3</a>).</p>


<h4>(b) Population Structure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Clustering analysis was performed on 36 loci spanning the genome. Of the initial 42 markers, six were excluded since they could not be amplified in all populations (<a href="#pone.0001249.s009">Table S3</a>). We used the program <em>STRUCTURE</em> <a href="#pone.0001249-Pritchard1">[32]</a>, in which each individual is assigned a probability (i.e. the mean estimated ancestry) of belonging to one of <em>K</em> populations, where <em>K</em> is predefined in the simulation. Various combinations of the number of populations <em>K</em> (1≤<em>K</em>≤10) and the simulation lengths (10<sup>4</sup>, 5×10<sup>4</sup>, 10<sup>5</sup>, 5×10<sup>5</sup>, 10<sup>6</sup> and 2×10<sup>6</sup> “Burn-ins” and “Repeats” each) were tested with three different input orders for the individuals. Consistent results were obtained for 4≤<em>K</em>≤7 with long simulation length (5×10<sup>5</sup>, 10<sup>6</sup> and 2×10<sup>6</sup>), independent of input order (<a href="#pone-0001249-g002">Figure 2A</a> and for <em>K</em>≤3 or <em>K</em>≥8, data not shown), suggesting that short simulation lengths (10<sup>4</sup>, 5×10<sup>4</sup> and 10<sup>5</sup>) are inadequate for our application (data not shown). When <em>K</em> is 4, four clusters, corresponding to the four species, i.e. <em>A. gambiae s.s.</em>, <em>A. arabiensis</em>, <em>A. merus</em> and <em>A. melas</em>, could be distinguished. When <em>K</em> is 5, the M and S molecular forms were further separated (<a href="#pone-0001249-g002">Figure 2A</a>). Interestingly, when <em>K</em> is 6 or 7, six distinct clusters were obtained. S mosquitoes were further separated into two clusters: S1 and S2, both from Mali, and S3 from Kenya, implying possible new taxonomic units within the S molecular form that correlate with spatial isolation. When we ran <em>STRUCTURE</em> with <em>K</em> set to 7 using 25 of the 42 microsatellites, i.e. those we had used in our previous study <a href="#pone.0001249-Wang1">[21]</a>, clustering of the six units was not observed (data not shown), suggesting that these 25 markers are not adequate for a fine-scale phylogenetic study of the <em>A. gambiae</em> complex, particularly when one aims to characterize the closely related <em>A. gambiae s.s.</em> M and S molecular forms.</p>
<div class="figure" id="pone-0001249-g002"><div class="img"><a name="pone-0001249-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The <em>STRUCTURE</em> results.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Consistent <em>STRUCTURE</em> results for 4≤<em>K</em>≤7 across the simulation lengths (5×10<sup>5</sup>, 10<sup>6</sup> and 2×10<sup>6</sup>) for plot A. Each of the 414 individuals in the analysis is represented by a thin vertical line, and is partitioned into <em>K</em> colored segments that represent the individual's probability of belonging to one of the <em>K</em> populations. Mosquitoes were positioned in the figure according to the collection site (Burkina Faso (BF), Mali, Kenya and Senegal) and to their species/forms (M molecular form (M1 and M2), S-form (S1, S2 and S3), <em>A. arabiensis</em> (A1 and A2), <em>A. merus</em> (R) and <em>A. melas</em> (L). In plot B, the M1 is further separated from M2 for <em>K</em> = 2 with simulation length 5×10<sup>5</sup>.</p>
<span>doi:10.1371/journal.pone.0001249.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The logarithmic value of probability distribution <em>ln(P(X/K))</em> and the probability of a given range of clusters <em>P(K/X)</em> were listed in <a href="#pone-0001249-t002">Table 2</a>, for cluster number calculation. The value of <em>ln(P(X/K))</em> was the output of the <em>STRUCTURE</em> analysis, where <em>X</em> denotes genotypes of the sampled individuals for the K cluster. <em>P(K/X)</em> was calculated on the basis of <em>ln(P(X/K))</em> value by considering <em>K</em> = 3,…,8 <a href="#pone.0001249-Pritchard1">[32]</a>. The largest value of <em>P(K/X)</em> corresponds to the smallest <em>ln(P(X/K))</em> in the given range of <em>K</em>. Consistent results were obtained across independent runs (5×10<sup>5</sup>, 10<sup>6</sup> and 2×10<sup>6</sup>), as well as all three different initial input orders (for the other two sets of calculations with different input orders, data not shown). For <em>K</em> = 6, the probability was 1 (<a href="#pone-0001249-t002">Table 2</a>). Therefore, we conclude that the most likely number of extant taxonomic units is six in the sample set, i.e., the M molecular form, the Mali and Kenya S forms, <em>A. arabiensis</em>, <em>A. merus</em> and <em>A. melas</em> (<a href="#pone-0001249-g002">Figure 2A</a>).</p>
<div class="figure" id="pone-0001249-t002"><div class="img"><a name="pone-0001249-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Inferring the number of clusters.</span></strong></p><span>doi:10.1371/journal.pone.0001249.t002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Finally, separate simulation runs using <em>STRUCTURE</em> were performed using only individuals with different geographical origins that we suspected might be subdivided. No subpopulations were identified in the Mali S-form (S1 and S2) and Mali and Kenya <em>A. arabiensis</em> (A1 and A2), but Burkina Faso (M1) and Mali M-form (M2) were further subdivided (<a href="#pone-0001249-g002">Figure 2B</a> and <a href="#pone.0001249.s010">Table S4</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The advantage of <em>STRUCTURE</em> is that it can not only identify population structure, but also detect migrants or admixed individuals <a href="#pone.0001249-Pritchard1">[32]</a>. In our analysis, any mosquito attributed to one cluster but with a higher than 5% mean estimated ancestry probability of belonging to another cluster is considered to be a migrant or admixed individual, which represents a quantified indicator of gene flow (<a href="#pone-0001249-t003">Table 3</a>). Admixed individuals between M and S forms (14 sympatric and 5 allopatric individuals, respectively) and between <em>A. gambiae s.s.</em> and <em>A. arabiensis</em> (13 sympatric and 5 allopatric mosquitoes, respectively) are the most common ones. Two sympatric and one allopatric migrant were found for <em>A. merus</em> with the other taxa, and none for <em>A. melas</em>. The preponderance of sympatric admixtures (70% when all five taxa are considered) suggests that introgression is the main contributor to the observed similarities among taxa. However, nine allopatric migrants were observed between Mali and Kenya S forms, suggesting that the detected migrants between the two S forms may be the results of ancestral polymorphisms because of the high-level geographic separation.</p>
<div class="figure" id="pone-0001249-t003"><div class="img"><a name="pone-0001249-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Migrants or admixed individuals detected between the taxonomic units.</span></strong></p><span>doi:10.1371/journal.pone.0001249.t003</span></div>

<h4>(c) Evidence for a mosaic genome in the <em>A. gambiae</em> complex</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The recombination rate varies for different regions of the genome, especially for the <em>X</em> chromosome of <em>A. gambiae</em> <a href="#pone.0001249-Pombi1">[33]</a>, <a href="#pone.0001249-Slotman3">[34]</a>, making it inappropriate to compare averages genetic distance measurements (e.g. <em>Fst</em>) across the genome when analyzing multilocus genotyping data. Therefore, we used systematical principal component analysis (PCA) and correspondence analysis (CA), where portions of the genome each comprising a few markers were analyzed to further clarify the phylogenetic relationships of the <em>A. gambiae</em> complex. At each locus, mean allele sizes for each genotyped individual were used.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>PCA reduces the dataset dimension and enables visualization of individuals and populations in 3D or 4D graphics. The first principal component (PC) represents the largest part of the variance in the whole dataset (i.e. corresponding to the most separated cluster), followed by the second largest part, the third, etc. CA involves two separated PCA analyses that are merged to represent the rows (population) and the columns (microsatellite marker) of a two-dimensional matrix <a href="#pone.0001249-He1">[35]</a>. As a result, CA can also reveal correlations between populations and microsatellite markers that are plotted on the same graphs.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Each PC is a weighted sum of microsatellites, the weights representing the strength of the microsatellite contributions to the PC. First, we analyzed the 414 individuals at 36 loci. The first principal component (PC1) separated the two major vector species (<em>A. gambiae s.s.</em> and <em>A. arabiensis</em>) from the zoophilic species (<em>A. merus</em> and <em>A. melas</em>) (<a href="#pone-0001249-g003">Figure 3A</a> and <a href="#pone.0001249.s006">Figure S6A</a>). PC1 was dominated by eight loci across the genome (two, four and two from chromosomes <em>X</em>, <em>2</em> and <em>3</em>, respectively; <a href="#pone-0001249-t004">Table 4A</a>), indicating that eight loci are mainly responsible for separating the two major and the two minor vectors in our sample set. PC2 (dominated by three, two and three loci from chromosomes <em>X</em>, <em>2</em> and <em>3</em>, respectively) and PC3 (dominated by three, two and three markers from chromosomes <em>X</em>, <em>2</em> and <em>3</em>, respectively) further distinguished the four named species, <em>A. gambiae s.s.</em>, <em>A. arabiensis</em>, <em>A. merus</em> and <em>A. melas</em>, from each other. Finally, PC4 separated the M and S molecular forms (<a href="#pone.0001249.s006">Figure S6A</a>); it was dominated by six loci: <em>AGXH678</em>, <em>AGXE614</em> (both near the <em>X</em> centromere) and four from chromosome <em>2</em>.</p>
<div class="figure" id="pone-0001249-g003"><div class="img"><a name="pone-0001249-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>PCA and CA results of the 414 individuals at 36 loci.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>PCA analysis of the 414 individuals in a genome-wide study for 36 microsatellites (A) and for seven loci on the <em>X</em> chromosome (B). CA analysis of the nine populations in the <em>Xag</em> inversion region (C), near the <em>X</em> centromere region (D), on chromosomes <em>2</em> (E) and <em>3</em> (F). Individuals and populations are color-coded as described (see legend where numbers in brackets next to the populations name indicate the number of mosquitoes in each group). The contribution of each PC or CA is indicated as a percentage in brackets on the axis. In plots A and B, each point represents a single mosquito, while in plots C to F, each point represents a population. In plots C to F, microsatellite markers are localized by black bars. The first three letters of their name (AGN, where N is chromosome <em>X</em>, <em>2</em> or <em>3</em>) were removed.</p>
<span>doi:10.1371/journal.pone.0001249.g003</span></div><div class="figure" id="pone-0001249-t004"><div class="img"><a name="pone-0001249-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The weights of the microsatellites contributing to the principal components.</span></strong></p><span>doi:10.1371/journal.pone.0001249.t004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>When we focused our analysis only on the <em>X</em> chromosome, a different phylogeny was obtained. The M and S molecular forms of <em>A. gambiae s.s.</em> are separated from each other, and the M-form is closely related to <em>A. merus</em>, and is more distant from <em>A. arabiensis</em> and <em>A. melas</em> (<a href="#pone-0001249-g003">Figure 3B</a> and <a href="#pone.0001249.s006">Figure S6B</a>).</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>PCA analyses of all 414 mosquitoes taken either as individuals or as nine populations led to generally congruent results, differences were nonetheless found in the contributions to the total variance (<a href="#pone-0001249-g003">Figure 3</a> and <a href="#pone.0001249.s006">Figure S6</a>). PCA transforms and reduces the 36 multidimensional dataset (for the 36 loci) to lower dimensions while retaining those characteristics of the dataset that contribute most to PCA variance. The PC1, PC2 and PC3 contributed 22.6%, 12% and 11.2% to the total variance in the PCA of the 414 mosquito taken individually, respectively (<a href="#pone-0001249-g003">Figure 3A</a>). However, PCA of the nine populations showed a more efficient reduction of dimensionality, with the corresponding contributions of 45.4%, 24.4% and 16.7%, respectively (<a href="#pone.0001249.s006">Figure S6C</a>). Although PCA of individuals is less efficient for the analysis of the phylogenetic relationship among populations, it reveals variations within each population.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>When studying the <em>Xag</em> inversion region in the nine populations using CA, we noticed that the M and S forms of <em>A. gambiae s.s.</em> were very much alike and closely related to <em>A. merus</em>, but were more distant from <em>A. melas</em> and <em>A. arabiensis</em> (<a href="#pone-0001249-g003">Figure 3C</a>), which is almost identical with the eigenvectorial representation based on the frequency of chromosome polymorphic inversions (see <a href="#pone.0001249.s002">Figure S2A</a> in <a href="#pone.0001249-Coluzzi2">[5]</a>). These data indicate that, in the <em>Xag</em> region, the maximum affinity is between <em>A. gambiae s.s.</em> and <em>A. merus</em>, consistent with the hypothesis of a monophyletic inversion. This conclusion was reinforced by the observation that the locus <em>AGXH36</em>, which is also located within the <em>Xag</em> inversion, is amplifiable in <em>A. gambiae s.s.</em> and <em>A. merus</em>, but not in <em>A. arabiensis</em> or <em>A. melas</em> (<a href="#pone.0001249.s001">Figure S1</a> and <a href="#pone.0001249.s009">Table S3</a>). However, the three loci near the centromere on the <em>X</em> chromosome outside the <em>Xag</em> region showed a very different pattern, which clearly separated the S-form from both the M-form and the other species (<a href="#pone-0001249-g003">Figure 3D</a>).</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>Markers on chromosome 2 (<em>2R</em>, 12 loci; <em>2L</em>, 5 loci) showed that <em>A. gambiae s.s.</em> is more closely related to <em>A. arabiensis</em> than to <em>A. merus</em> and <em>A. melas</em> (<a href="#pone-0001249-g003">Figure 3E</a>). Markers on chromosome 3 (<em>3R</em>, 6 loci; <em>3L</em>, 6 loci) indicated a separation of <em>A. melas</em> from the other species (<a href="#pone-0001249-g003">Figure 3F</a>), which might have resulted partially from the four invariant alleles in <em>A. melas</em>.</p>
<a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>Finally, PCA analysis of the 262 <em>A. gambiae s.s.</em> samples at 42 microsatellite markers, including the six loci <em>AGXH145</em>, <em>AGXH36</em>, <em>AG3E35B</em>, <em>AG3E37B</em>, <em>AG3E38B3</em> and <em>3L09-C1,</em> which could be amplified in all <em>A. gambiae s.s.</em> individuals but not in the other species, provided more detailed information about the phylogeny of <em>A. gambiae s.s.</em> (<a href="#pone-0001249-g004">Figure 4A</a> and <a href="#pone.0001249.s006">Figure S6D</a>). The M and S forms were separated by PC1 (dominated by <em>AGXH678</em>, <em>AGXND6U3</em>, <em>AGXE614</em>, <em>AG2H85</em>, <em>AG2H135</em> and <em>AG3E40A1</em>, see <a href="#pone-0001249-t004">Table 4B</a>). Mali and Kenya S forms were distinguished by PC2 and PC3, dominated by three, three and six loci on chromosomes <em>X</em>, <em>2</em> and <em>3,</em> respectively. To identify the chromosomal regions that are responsible for this separation, we applied CA to the mean genotypes of the five populations at each locus (<a href="#pone-0001249-g004">Figure 4B and 4C</a>). The Kenya S-form was shown to be markedly separated from the Mali S-forms across the whole genome, except in the region near the centromere of the <em>X</em> chromosome, outside the <em>Xag</em> inversion.</p>
<div class="figure" id="pone-0001249-g004"><div class="img"><a name="pone-0001249-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001249" data-uri="info:doi/10.1371/journal.pone.0001249.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001249.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001249.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001249.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001249.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>PCA and CA results of the 262 <em>A. gambiae s.s.</em> at 42 loci.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>PCA results of the 262 <em>A. gambiae s.s.</em> at 42 microsatellites (A), and CA results of the five populations of <em>A. gambiae s.s.</em> in the <em>Xag</em> inversion region and near the <em>X</em> centromere (B), and on autosomes (C). Each individual (in plot A) or each population (in plots B and C) is represented by a single point. See <a href="#pone-0001249-g003">Figure 3</a> for legend. Only the most significant microsatellites markers are indicated.</p>
<span>doi:10.1371/journal.pone.0001249.g004</span></div><a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9"></a><p>The correlations between taxa and microsatellite markers can be extracted from the CA graphs shown in <a href="#pone-0001249-g003">Figures 3C–F</a> and <a href="#pone-0001249-g004">Figures 4B–C</a>, which are more specific than the markers found by PCA (see above). For instance, the M-form is identified by loci <em>AG2H135</em> and <em>AG2H197</em>, the S-form by loci <em>AGXH678</em>, <em>AGXND6U3</em> and <em>AG2H187</em>, while loci <em>AG2H802</em> and <em>AG3E34B2</em> are more specific for Kenya S-form (S3). <em>A. arabiensis</em> is identified by loci <em>AGXH503</em> and <em>AG2E12D</em>, and <em>A. merus</em> by loci <em>AG2H796</em> and <em>AG3E40A1</em>. For <em>A. melas</em>, there are five most contributing loci, <em>AGXH53</em>, <em>AGXE614</em>, <em>AG2H802</em>, <em>AG3H555</em> and <em>19C20-B1</em>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>To investigate the phylogenetic relationships among species in the <em>A. gambiae</em> complex, we have used microsatellite polymorphism in 414 field-collected female mosquitoes with 42 markers covering the whole genome, and systematically analyzed the data genome-wide or regionally using three statistical methods: the program <em>STRUCTURE</em>, Principal Component Analysis and Correspondence Analysis. These three statistical methods are applicable for studying various individuals or populations, genome-wide or in parts of the genome. The <em>STRUCUTRE</em> method is more suited to whole genome phylogeny analysis (with a large number of loci) under proper simulation conditions, and has the great advantage of enabling the identification of the most likely number of extant populations, and for the detection of migrants or admixed individuals among these populations <a href="#pone.0001249-Pritchard1">[32]</a>. In contrast to <em>STRUCTURE</em>, PCA and CA are more determinate, and are not only capable of differentiating individuals or populations from each other, but also of identifying the genomic regions responsible for the observed differentiation, which makes them useful tools to detect and analyze mosaic genomes</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Our results demonstrate an evident genome mosaicism within the <em>A. gambiae</em> complex. The two major malaria vectors (<em>A. gambiae s.s.</em> and <em>A. arabiensis</em>) are closely related, further away from <em>A. merus</em> and most distantly related to <em>A. melas</em>, in agreement with the analyses on rDNA and mtDNA <a href="#pone.0001249-Besansky1">[11]</a>. This indicates that <em>A. arabiensis,</em> rather than <em>A. merus,</em> is the sister taxon of <em>A. gambiae s.s.</em>, in contrast to expectations based on chromosomal inversion patterns <a href="#pone.0001249-Coluzzi2">[5]</a>. Nevertheless, within the <em>Xag</em> inversion on the <em>X</em> chromosome, the M and S molecular forms of <em>A. gambiae s.s.</em> are indeed most similar to <em>A. merus</em>, and more distant from <em>A. melas</em> and <em>A. arabiensis</em>. These results support the inference from polytene chromosomal inversions <a href="#pone.0001249-Coluzzi1">[3]</a>, <a href="#pone.0001249-Coluzzi2">[5]</a> and agree with the single gene (guanylate cyclase) and <em>2La</em> marker studies <a href="#pone.0001249-Garcia1">[9]</a>, <a href="#pone.0001249-Sharakhov1">[10]</a>. Outside the <em>Xag</em> region and near the <em>X</em> centromere, the S form is the most distant to the others, in particular to the M form, in agreement with previous results <a href="#pone.0001249-Wang1">[21]</a>, <a href="#pone.0001249-Stump1">[22]</a>, <a href="#pone.0001249-Stump2">[23]</a>, <a href="#pone.0001249-Turner1">[24]</a>. In addition, the M form is equally close in this region to <em>A. merus</em> and to <em>A. arabiensis</em>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Within <em>A. gambiae s.s.</em>, three taxonomic units, the M and two S molecular forms, have been identified in our sample set. The two S-forms are allopatric, consistent with the separation of the southeastern vs. northwestern S forms observed previously <a href="#pone.0001249-Lehmann1">[25]</a>. Considering the high-level of premating isolation of these allopatric mosquitoes, we can assume that the observed genetic divergences between the Mali and Kenya S forms are the direct consequence of geographic isolation. Interestingly, we did not observe a similar separation in two clusters for the Mali and Kenya <em>A. arabiensis</em> populations (A1 and A2), although they were collected from the same sites as the two S-forms. Moreover, the weaker separation of the Burkina Faso and Mali M forms is consistent with the results obtained using different statistical methods and concentrating on the third chromosome <a href="#pone.0001249-Slotman2">[30]</a>. Still, as the sample sizes from the Burkina Faso M1, as well as from the Kenya S3 populations are small, the results obtained here need to be verified on larger samples of wild mosquitoes and by laboratory crosses.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In regards to the ‘gene tree versus species tree’ issue, it is generally accepted that the use of multiple loci would in principle provide a more complete picture of the divergence history for a group of closely related species <a href="#pone.0001249-Machado1">[1]</a>. The <em>A. gambiae</em> genome size is about 278 megabases (Mb) <a href="#pone.0001249-Holt1">[31]</a>, leading to a coverage of the 42 microsatellites at about 7 Mb per marker, which represents the highest resolution to date. Previous studies from us and others, with 25 or fewer microsatellites were not as robust to resolve the phylogeny of the <em>A. gambiae</em> complex at the genome-wide level, especially among the closely related molecular forms <a href="#pone.0001249-Wang1">[21]</a>, <a href="#pone.0001249-Lehmann1">[25]</a>. This might be due to the lower number of markers used compared to our study, and/or to a “biased” choice of markers, although the 25 microsatellites used in <a href="#pone.0001249-Wang2">[36]</a> were evenly distributed throughout the genome. Moreover, in contrast to previous reports that relied on the analysis of each locus separately, we used PCA and CA methods that can grasp the different contributions of multiple loci, and therefore allow a true genome-wide comparison of the different populations.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our analysis also identified genomic regions that are characteristic of certain populations or taxa. First, despite the fact that polymorphisms are shared among the nine populations overall, four loci near the centromeric region of chromosome <em>3</em> exhibit private alleles for all 37 <em>A. melas</em> individuals. This observation awaits further validation with samples from different collection sites and of larger sizes. Second, the two minor malaria vectors can be distinguished from the two major vectors at two loci in the centromeric region of chromosome <em>3</em>. This region is therefore a potential strong discriminator between major and minor malaria vectors. This differentiation might be related to ecological factors as well, such as fresh water versus salt water breeding habitats, which distinguish the major vector and minor vector species we have analyzed. Third, we confirmed that the centromeric region of chromosome <em>X</em> is critical to separate the M and S forms <a href="#pone.0001249-Wang1">[21]</a>, <a href="#pone.0001249-Stump1">[22]</a>, <a href="#pone.0001249-Stump2">[23]</a>, <a href="#pone.0001249-Turner1">[24]</a>, <a href="#pone.0001249-Turner2">[29]</a>. The centromere of chromosome <em>2L</em> was also reported to distinguish these two forms <a href="#pone.0001249-dellaTorre2">[18]</a>, <a href="#pone.0001249-Turner1">[24]</a>, <a href="#pone.0001249-dellaTorre3">[26]</a>, <a href="#pone.0001249-Chandre1">[27]</a>, <a href="#pone.0001249-Weill1">[28]</a>, <a href="#pone.0001249-Turner2">[29]</a>. Thus, taken together, our data and previous work point to the centromere regions as strong identifiers of different populations of the <em>A. gambiae</em> complex. Reduced recombination rates and a strong differentiation at the sequence level between the M and S forms have been reported near the <em>X</em> centromere region <a href="#pone.0001249-Pombi1">[33]</a>, <a href="#pone.0001249-Slotman3">[34]</a>. It would be interesting to measure recombination rates near the <em>2L</em>- and <em>3<sup>rd</sup></em> chromosome centromeres. Regions adjacent to centromeres are known to experience less recombination in several species, and it has recently been proposed that low recombination rates can facilitate the accumulation and maintenance of isolation genes in partially isolated populations.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Migrants, or admixed individuals, were found mostly between M and S molecular forms and between <em>A. gambiae s.s.</em> and <em>A. arabiensis</em>, reflecting the genetic proximity of these populations. The detected frequencies between M and S forms (0.6–3.9%) are similar to those (0.3–5.6%) estimated or found previously <a href="#pone.0001249-Taylor1">[17]</a>, <a href="#pone.0001249-Tripet1">[37]</a>, <a href="#pone.0001249-Edillo1">[38]</a>, <a href="#pone.0001249-Gentile2">[39]</a>. The migrant frequencies between <em>A. gambiae s.s.</em> and <em>A. arabiensis</em> 0.5–4.7% are much higher than those observed in the field previously (0.01–0.2%) <a href="#pone.0001249-Tour1">[8]</a>, <a href="#pone.0001249-Powell1">[40]</a>. The presence of admixed individuals between <em>A. gambiae s.s.</em> and <em>A. merus</em> (2.0%) further supports the previously reported close relationship between these two species <a href="#pone.0001249-Coluzzi2">[5]</a>, <a href="#pone.0001249-Garcia1">[9]</a>, <a href="#pone.0001249-Caccone1">[14]</a>.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In summary, about 30% of the detected admixed individuals between the M and S forms of <em>A. gambiae s.s., A. arabiensis, A. merus and A. melas</em> are allopatric mosquitoes with a high level of possible geographic isolation, thus the contribution of introgression to the variations in our sample set is estimated to be 70%. This estimation might be higher than in reality as we cannot exclude the possibility that some of the observed sympatric migrants resulted from ancestral polymorphisms.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Variation among loci is an expected consequence of recent speciation events, because, following divergence, some loci will develop monophyletic patterns more quickly than others, simply by chance (lineage sorting) <a href="#pone.0001249-Machado1">[1]</a>. We have observed in <em>A. melas</em>, five loci are monomorphic, though in most cases the microsatellite loci were polymorphic in all species. This might be an indication of lineage sorting. However, we have observed different phylogenetic relationships within the <em>Xag</em> inversion, as well as near the centromere region of the <em>X</em> chromosome, contrary to those on the autosomal regions. These data suggested that genetic divergence might have been a consequence of introgressive hybridization as has been proposed <a href="#pone.0001249-Coluzzi1">[3]</a>, <a href="#pone.0001249-Caccone1">[14]</a>, as well as experimentally demonstrated for species in the <em>A. gambiae</em> species <a href="#pone.0001249-dellaTorre1">[12]</a>.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Introgressive hybridization is a well-supported hypothesis for the <em>A. gambiae s.s.</em>/<em>A. arabiensis</em> species pair <a href="#pone.0001249-dellaTorre1">[12]</a>, <a href="#pone.0001249-Besansky2">[15]</a>, <a href="#pone.0001249-dellaTorre2">[18]</a>, which is consistent with the preponderance of sympatric admixtures among these taxa. The observed sympatric and allopatric migrants may have contributed to the expansion of <em>A. gambiae s.s.</em> across different ecological settings over Africa, in much the same way that hybrid sunflower species were found to occupy extreme environmental settings <a href="#pone.0001249-Rieseberg1">[41]</a>. As a consequence, the <em>A. gambiae</em> species complex appears to be highly dynamic: continuous speciation occurs in situations where closely related taxa (M- and S-forms) are undergoing incipient speciation <a href="#pone.0001249-dellaTorre3">[26]</a>, while introgressive hybridization besides the lineage sorting of ancestral polymorphism among the taxa may provide traits that make these mosquitoes such a menace in Africa.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Mosquito samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>A total of 414 adult female mosquitoes were collected from several locations in Africa: Burkina Faso: Goundry; Mali: Selenkenyi (Sel), Soulouba (Soul) and Kokouna (Kn); Kenya: Kilifi and in Senegal (<a href="#pone-0001249-t001">Table 1</a>). Species were identified by PCR <a href="#pone.0001249-Scott1">[42]</a> and <em>A. gambiae s.s.</em> mosquitoes were further divided into M and S forms <a href="#pone.0001249-Favia2">[43]</a>. We grouped the 414 mosquitoes into nine populations ranging in sample size between 20 and 80, according to species identified by PCR and collection sites (<a href="#pone-0001249-t001">Table 1</a>).</p>


<h4>Cytogenetic localization</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A bacterial artificial chromosome (BAC) library of <em>A. gambiae</em> (described in <a href="#pone.0001249-Thomasova1">[44]</a>) was screened by PCR with microsatellite-specific primers <a href="#pone.0001249-Wang2">[36]</a>, <a href="#pone.0001249-Zheng1">[45]</a>. Validated BAC clones were localized on squashed nurse cell polytene chromosomes of <em>A. gambiae</em> by <em>in situ</em> hybridization as described <a href="#pone.0001249-Kumar1">[46]</a>. Hybridization images and illustrations of the detailed chromosomal location of each clone are available in the <em>Anopheles</em> database AnoDB (<a href="http://www.anobase.org/cgi-bin/insitu.pl">http://www.anobase.org/cgi-bin/insitu.pl</a>).</p>


<h4>Genotyping</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Mosquitoes were collected and genomic DNA was extracted as described <a href="#pone.0001249-Tour1">[8]</a>. Of the 42 microsatellite markers used in this study (see <a href="#pone.0001249.s007">Table S1</a>), 35 were used previously <a href="#pone.0001249-Wang1">[21]</a>, <a href="#pone.0001249-Stump1">[22]</a>, <a href="#pone.0001249-Wang2">[36]</a>, <a href="#pone.0001249-Zheng2">[47]</a>, <a href="#pone.0001249-Lehmann2">[48]</a>, <a href="#pone.0001249-Dimopoulos1">[49]</a>, <a href="#pone.0001249-Zheng3">[50]</a>, and 7 were newly designed. Attempts to design microsatellite markers near the <em>2L</em> centromere failed because all flanking sequences were not unique when compared to the genome sequence. In this region, the microsatellite family does exist <a href="#pone.0001249-Archak1">[51]</a>. At least three tested markers have the same flanking regions as do markers near the <em>X</em> centromere region. Genotyping was carried out as described <a href="#pone.0001249-Wang2">[36]</a> with some modifications. PCRs were performed using all the markers. Each reaction contained 125 p<em>M</em> of each primer, 0.15 m<em>M</em> of each dNTP, 0.025 U/µl of Taq polymerase, and 0.3–0.9 ng/µl genomic DNA of a single mosquito. Cycling conditions were 94°C denaturation for 2 min, followed by 30–35 cycles of 94°C for 15 s, 56–68°C for 30s, and 72°C for 30 s, with a final 72°C extension of 10 min. Diluted (1/10) and multiplexed PCR products were detected using MegaBase capillary sequencer (Amersham Biosciences), and allele sizes for each individual at each locus were assigned and manually checked using the Genetic Profiler software (Amersham Biosciences).</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The genotyping data were evaluated using the software <em>STRUCTURE</em> <a href="#pone.0001249-Pritchard1">[32]</a>. <em>STRUCTURE</em> uses Bayesian clustering method to assign individuals probabilistically to populations based on multilocus genotype data. In this analysis, we did not use prior information about the origin of individuals. Various combinations of the number of populations <em>K</em> (1≤<em>K</em>≤10) and the simulation lengths (10<sup>4</sup>, 5×10<sup>4</sup>, 10<sup>5</sup>, 5×10<sup>5</sup>, 10<sup>6</sup> and 2×10<sup>6</sup> “Burn-ins” and “Repeats” each) were tested. The probability of each individual belonging to each of the <em>K</em> populations was plotted.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>In addition, principal component analysis <a href="#pone.0001249-He2">[52]</a> and correspondence analysis <a href="#pone.0001249-He1">[35]</a>, <a href="#pone.0001249-Ueberla1">[53]</a> were performed systematically on genotypes either for the 414 mosquitoes taken individually or for the corresponding nine populations genome-wide or on parts of the genome. The 262 <em>A. gambiae s.s.</em> mosquitoes and the corresponding five populations were further analysed by PCA and CA at all 42 microsatellite loci.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001249"></div><a name="pone.0001249.s001" id="pone.0001249.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Allele frequency vs. size distributions of the X chromosome</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.31 MB TIF)</p>
<a name="pone.0001249.s002" id="pone.0001249.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Allele frequency vs. size distributions of the 2R chromosome arm</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.09 MB TIF)</p>
<a name="pone.0001249.s003" id="pone.0001249.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Allele frequency vs. size distributions of the 2R/2L chromosome arms</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.12 MB TIF)</p>
<a name="pone.0001249.s004" id="pone.0001249.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Allele frequency vs. size distributions of the 3R chromosome arm</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.10 MB TIF)</p>
<a name="pone.0001249.s005" id="pone.0001249.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Allele frequency vs. size distributions of the 3L chromosome arm</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(1.11 MB TIF)</p>
<a name="pone.0001249.s006" id="pone.0001249.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">PCA results of 2D plots</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(2.27 MB TIF)</p>
<a name="pone.0001249.s007" id="pone.0001249.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s007">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">The 42 microsatellite markers used in this study</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.21 MB DOC)</p>
<a name="pone.0001249.s008" id="pone.0001249.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s008">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Raw genotyping data of the 414 individuals at 42 markers. The 42 markers are listed in the first row, and the first two letters in their names (AG) are omitted. The 414 individuals of the <em>A. gambiae</em> complex were grouped into nine populations: molecular forms M (M1 and M2), and S (S1, S2 and S3) of <em>A. gambiae</em> s.s., <em>A. arabiensis</em> (A1 and A2), <em>A. merus</em> (R), and <em>A. melas</em> (L). Each individual of a population is assigned an ID, which consists of the population name followed by a serial number. The sizes of the two alleles at the same locus are given for each individual (numbers with the same ID and the same locus name), indicating either heterozygosity (two alleles have different sizes) or homozygosity (two alleles have the same size). “-9” stands for missing data.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.27 MB XLS)</p>
<a name="pone.0001249.s009" id="pone.0001249.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s009">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Variation of 42 microsatellite loci in nine populations of the <em>A. gambiae</em> complex</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0001249.s010" id="pone.0001249.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001249.s010">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Inferring the number of clusters for the Burkina Faso and Mali M molecular forms (104 individuals) at 42 loci</p>
<p class="siDoi">doi:10.1371/journal.pone.0001249.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the staff of the Genomic Core Facility at EMBL, especially R. Carmouche for his technical assistance on MegaBACE. We are grateful to M. Niegls for suggesting the use of the CA method, and to C. Mbogo, S. N'Fale, C. Costantini and the entomology team of the Centre National de Recherche et Formation sur le Paludisme, CNRFP (Ouagadougou, Burkina Faso) for providing mosquito samples. We thank V. Petrarca, J. Gagneur, Y. Lee, T. Wang, G. Ding and A. Moore for helpful comments, and we are also thankful to the three anonymous reviewers for their instructive professional suggestions on our manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FK RW LZ. Performed the experiments: RW CB. Analyzed the data: RW. Contributed reagents/materials/analysis tools: RW GD YT Ad GL. Wrote the paper: FK RW LZ SB Ad GL LS YN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001249-Machado1" id="pone.0001249-Machado1"></a>Machado CA, Hey J (2003) The causes of phylogenetic conflict in a classic Drosophila species group. Proc R Soc Lond B Biol Sci  270: 1193–1202.  <ul class="find" data-citedArticleID="1024720" data-doi="10.1098/rspb.2003.2333"><li><a href="http://dx.doi.org/10.1098/rspb.2003.2333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+causes+of+phylogenetic+conflict+in+a+classic+Drosophila+species+group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+causes+of+phylogenetic+conflict+in+a+classic+Drosophila+species+group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001249-White1" id="pone.0001249-White1"></a>White GB (1974) <em>Anopheles gambiae</em> complex and disease transmission in Africa. Trans R Soc Trop Med Hyg  68: 278–301.  <ul class="find" data-citedArticleID="1024764"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Anopheles+gambiae+complex+and+disease+transmission+in+Africa.&amp;auth=&amp;atitle=Anopheles+gambiae+complex+and+disease+transmission+in+Africa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anopheles+gambiae+complex+and+disease+transmission+in+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anopheles+gambiae+complex+and+disease+transmission+in+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001249-Coluzzi1" id="pone.0001249-Coluzzi1"></a>Coluzzi M, Sabatini A, Petrarca V, Di Deco MA (1979) Chromosomal differentiation and adaptation to human environments in the <em>Anopheles gambiae</em> complex. Trans R Soc Trop Med Hyg  73: 483–497.  <ul class="find" data-citedArticleID="1024684" data-doi="10.1016/0035-9203(79)90036-1"><li><a href="http://dx.doi.org/10.1016/0035-9203(79)90036-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+differentiation+and+adaptation+to+human+environments+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+differentiation+and+adaptation+to+human+environments+in+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001249-Hunt1" id="pone.0001249-Hunt1"></a>Hunt RH, Coetzee M, Fettene M (1998) The <em>Anopheles gambiae</em> complex: a new species from Ethiopia. Trans R Soc Trop Med Hyg  92: 231–235.  <ul class="find" data-citedArticleID="1024712" data-doi="10.1016/s0035-9203(98)90761-1"><li><a href="http://dx.doi.org/10.1016/s0035-9203(98)90761-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Anopheles+gambiae+complex%3A+a+new+species+from+Ethiopia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Anopheles+gambiae+complex%3A+a+new+species+from+Ethiopia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001249-Coluzzi2" id="pone.0001249-Coluzzi2"></a>Coluzzi M, Sabatini A, della Torre A, Di Deco MA, Petrarca V (2002) A polytene chromosome analysis of the <em>Anopheles gambiae</em> species complex. Science  298: 1415–1418.  <ul class="find" data-citedArticleID="1024686" data-doi="10.1126/science.1077769"><li><a href="http://dx.doi.org/10.1126/science.1077769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+polytene+chromosome+analysis+of+the+Anopheles+gambiae+species+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+polytene+chromosome+analysis+of+the+Anopheles+gambiae+species+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001249-Ayala1" id="pone.0001249-Ayala1"></a>Ayala FJ, Coluzzi M (2005) Chromosome speciation: humans, Drosophila, and mosquitoes. Proc Natl Acad Sci U S A  102: Suppl 16535–6542.  <ul class="find" data-citedArticleID="1024674" data-doi="10.1073/pnas.0501847102"><li><a href="http://dx.doi.org/10.1073/pnas.0501847102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+speciation%3A+humans%2C+Drosophila%2C+and+mosquitoes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+speciation%3A+humans%2C+Drosophila%2C+and+mosquitoes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001249-Davidson1" id="pone.0001249-Davidson1"></a>Davidson G (1965) The five mating-types in the <em>anopheles gambiae</em> compelx. Riv Malariol  43: 167–183.  <ul class="find" data-citedArticleID="1024690"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+five+mating-types+in+the+anopheles+gambiae+compelx.&amp;auth=&amp;atitle=The+five+mating-types+in+the+anopheles+gambiae+compelx." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+five+mating-types+in+the+anopheles+gambiae+compelx." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+five+mating-types+in+the+anopheles+gambiae+compelx.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001249-Tour1" id="pone.0001249-Tour1"></a>Touré YT, Petrarca V, Traore SF, Coulibaly A, Maiga HM, et al.  (1998) The distribution and inversion polymorphism of chromosomally recognized taxa of the <em>Anopheles gambiae</em> complex in Mali, West Africa. Parassitologia  40: 477–511.  <ul class="find" data-citedArticleID="1024748"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa.&amp;auth=&amp;atitle=The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001249-Garcia1" id="pone.0001249-Garcia1"></a>Garcia BA, Caccone A, Mathiopoulos KD, Powell JR (1996) Inversion monophyly in African anopheline malaria vectors. Genetics  143: 1313–1320.  <ul class="find" data-citedArticleID="1024700"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inversion+monophyly+in+African+anopheline+malaria+vectors.&amp;auth=&amp;atitle=Inversion+monophyly+in+African+anopheline+malaria+vectors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inversion+monophyly+in+African+anopheline+malaria+vectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inversion+monophyly+in+African+anopheline+malaria+vectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001249-Sharakhov1" id="pone.0001249-Sharakhov1"></a>Sharakhov IV, White BJ, Sharakhova MV, Kayondo J, Lobo NF, et al.  (2006) Breakpoint structure reveals the unique origin of an interspecific chromosomal inversion (2La) in the <em>Anopheles gambiae</em> complex. Proc Natl Acad Sci U S A  103: 6258–6262.  <ul class="find" data-citedArticleID="1024732" data-doi="10.1073/pnas.0509683103"><li><a href="http://dx.doi.org/10.1073/pnas.0509683103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breakpoint+structure+reveals+the+unique+origin+of+an+interspecific+chromosomal+inversion+%282La%29+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breakpoint+structure+reveals+the+unique+origin+of+an+interspecific+chromosomal+inversion+%282La%29+in+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001249-Besansky1" id="pone.0001249-Besansky1"></a>Besansky NJ, Powell JR, Caccone A, Hamm DM, Scott JA, et al.  (1994) Molecular phylogeny of the <em>Anopheles gambiae</em> complex suggests genetic introgression between principal malaria vectors. Proc Natl Acad Sci U S A  91: 6885–6888.  <ul class="find" data-citedArticleID="1024676" data-doi="10.1073/pnas.91.15.6885"><li><a href="http://dx.doi.org/10.1073/pnas.91.15.6885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+the+Anopheles+gambiae+complex+suggests+genetic+introgression+between+principal+malaria+vectors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+the+Anopheles+gambiae+complex+suggests+genetic+introgression+between+principal+malaria+vectors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001249-dellaTorre1" id="pone.0001249-dellaTorre1"></a>della Torre A, Merzagora L, Powell JR, Coluzzi M (1997) Selective introgression of paracentric inversions between two sibling species of the <em>Anopheles gambiae</em> complex. Genetics  146: 239–244.  <ul class="find" data-citedArticleID="1024772"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Selective+introgression+of+paracentric+inversions+between+two+sibling+species+of+the+Anopheles+gambiae+complex.&amp;auth=&amp;atitle=Selective+introgression+of+paracentric+inversions+between+two+sibling+species+of+the+Anopheles+gambiae+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+introgression+of+paracentric+inversions+between+two+sibling+species+of+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+introgression+of+paracentric+inversions+between+two+sibling+species+of+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001249-Slotman1" id="pone.0001249-Slotman1"></a>Slotman MA, Della Torre A, Calzetta M, Powell JR (2005) Differential introgression of chromsomal regions between <em>Anopheles gambiae</em> and <em>An. arabiensis</em>. Am J Trop Med Hyg  73: 326–335.  <ul class="find" data-citedArticleID="1024734"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Differential+introgression+of+chromsomal+regions+between+Anopheles+gambiae+and+An.+arabiensis.&amp;auth=&amp;atitle=Differential+introgression+of+chromsomal+regions+between+Anopheles+gambiae+and+An.+arabiensis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differential+introgression+of+chromsomal+regions+between+Anopheles+gambiae+and+An.+arabiensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differential+introgression+of+chromsomal+regions+between+Anopheles+gambiae+and+An.+arabiensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001249-Caccone1" id="pone.0001249-Caccone1"></a>Caccone A, Min GS, Powell JR (1998) Multiple origins of cytologically identical chromosome inversions in the <em>Anopheles gambiae</em> complex. Genetics  150: 807–814.  <ul class="find" data-citedArticleID="1024680"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+origins+of+cytologically+identical+chromosome+inversions+in+the+Anopheles+gambiae+complex.&amp;auth=&amp;atitle=Multiple+origins+of+cytologically+identical+chromosome+inversions+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+origins+of+cytologically+identical+chromosome+inversions+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+origins+of+cytologically+identical+chromosome+inversions+in+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001249-Besansky2" id="pone.0001249-Besansky2"></a>Besansky NJ, Krzywinski J, Lehmann T, Simard F, Kern M, et al.  (2003) Semipermeable species boundaries between <em>Anopheles gambiae</em> and <em>Anopheles arabiensis</em>: evidence from multilocus DNA sequence variation. Proc Natl Acad Sci U S A  100: 10818–10823.  <ul class="find" data-citedArticleID="1024678" data-doi="10.1073/pnas.1434337100"><li><a href="http://dx.doi.org/10.1073/pnas.1434337100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Semipermeable+species+boundaries+between+Anopheles+gambiae+and+Anopheles+arabiensis%3A+evidence+from+multilocus+DNA+sequence+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Semipermeable+species+boundaries+between+Anopheles+gambiae+and+Anopheles+arabiensis%3A+evidence+from+multilocus+DNA+sequence+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001249-Coluzzi3" id="pone.0001249-Coluzzi3"></a>Coluzzi M, Petrarca V, Di Deco MA (1985) Chromosomal inversion intergradation and incipient speciation in <em>Anopheles gambiae</em>. Boll Zool  52: 45–63.  <ul class="find" data-citedArticleID="1024688" data-doi="10.1080/11250008509440343"><li><a href="http://dx.doi.org/10.1080/11250008509440343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+inversion+intergradation+and+incipient+speciation+in+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+inversion+intergradation+and+incipient+speciation+in+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001249-Taylor1" id="pone.0001249-Taylor1"></a>Taylor C, Toure YT, Carnahan J, Norris DE, Dolo G, et al.  (2001) Gene flow among populations of the malaria vector, <em>Anopheles gambiae</em>, in Mali, West Africa. Genetics  157: 743–750.  <ul class="find" data-citedArticleID="1024744"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+flow+among+populations+of+the+malaria+vector%2C+Anopheles+gambiae%2C+in+Mali%2C+West+Africa.&amp;auth=&amp;atitle=Gene+flow+among+populations+of+the+malaria+vector%2C+Anopheles+gambiae%2C+in+Mali%2C+West+Africa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+flow+among+populations+of+the+malaria+vector%2C+Anopheles+gambiae%2C+in+Mali%2C+West+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+flow+among+populations+of+the+malaria+vector%2C+Anopheles+gambiae%2C+in+Mali%2C+West+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001249-dellaTorre2" id="pone.0001249-dellaTorre2"></a>della Torre A, Fanello C, Akogbeto M, Dossou-yovo J, Favia G, et al.  (2001) Molecular evidence of incipient speciation within <em>Anopheles gambiae</em> s.s. in West Africa. Insect Mol Biol  10: 9–18.  <ul class="find" data-citedArticleID="1024774" data-doi="10.1046/j.1365-2583.2001.00235.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.2001.00235.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+of+incipient+speciation+within+Anopheles+gambiae+s.s.+in+West+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+of+incipient+speciation+within+Anopheles+gambiae+s.s.+in+West+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001249-Gentile1" id="pone.0001249-Gentile1"></a>Gentile G, Slotman M, Ketmaier V, Powell JR, Caccone A (2001) Attempts to molecularly distinguish cryptic taxa in <em>Anopheles gambiae</em> s.s. Insect Mol Biol  10: 25–32.  <ul class="find" data-citedArticleID="1024702" data-doi="10.1046/j.1365-2583.2001.00237.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.2001.00237.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Attempts+to+molecularly+distinguish+cryptic+taxa+in+Anopheles+gambiae+s.s." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Attempts+to+molecularly+distinguish+cryptic+taxa+in+Anopheles+gambiae+s.s.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001249-Favia1" id="pone.0001249-Favia1"></a>Favia G, Lanfrancotti A, Spanos L, Siden-Kiamos I, Louis C (2001) Molecular characterization of ribosomal DNA polymorphisms discriminating among chromosomal forms of <em>Anopheles gambiae s.s.</em> Insect Mol Biol  10: 19–23.  <ul class="find" data-citedArticleID="1024696" data-doi="10.1046/j.1365-2583.2001.00236.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.2001.00236.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+ribosomal+DNA+polymorphisms+discriminating+among+chromosomal+forms+of+Anopheles+gambiae+s.s." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+ribosomal+DNA+polymorphisms+discriminating+among+chromosomal+forms+of+Anopheles+gambiae+s.s.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001249-Wang1" id="pone.0001249-Wang1"></a>Wang R, Zheng L, Toure YT, Dandekar T, Kafatos FC (2001) When genetic distance matters: measuring genetic differentiation at microsatellite loci in whole-genome scans of recent and incipient mosquito species. Proc Natl Acad Sci U S A  98: 10769–10774.  <ul class="find" data-citedArticleID="1024758" data-doi="10.1073/pnas.191003598"><li><a href="http://dx.doi.org/10.1073/pnas.191003598" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=When+genetic+distance+matters%3A+measuring+genetic+differentiation+at+microsatellite+loci+in+whole-genome+scans+of+recent+and+incipient+mosquito+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22When+genetic+distance+matters%3A+measuring+genetic+differentiation+at+microsatellite+loci+in+whole-genome+scans+of+recent+and+incipient+mosquito+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001249-Stump1" id="pone.0001249-Stump1"></a>Stump AD, Shoener JA, Costantini C, Sagnon N, Besansky NJ (2005) Sex-linked differentiation between incipient species of <em>Anopheles gambiae</em>. Genetics  169: 1509–1519.  <ul class="find" data-citedArticleID="1024740" data-doi="10.1534/genetics.104.035303"><li><a href="http://dx.doi.org/10.1534/genetics.104.035303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sex-linked+differentiation+between+incipient+species+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sex-linked+differentiation+between+incipient+species+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001249-Stump2" id="pone.0001249-Stump2"></a>Stump AD, Fitzpatrick MC, Lobo NF, Traore S, Sagnon N, et al.  (2005) Centromere-proximal differentiation and speciation in <em>Anopheles gambiae</em>. Proc Natl Acad Sci U S A  102: 15930–15935.  <ul class="find" data-citedArticleID="1024742" data-doi="10.1073/pnas.0508161102"><li><a href="http://dx.doi.org/10.1073/pnas.0508161102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Centromere-proximal+differentiation+and+speciation+in+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Centromere-proximal+differentiation+and+speciation+in+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001249-Turner1" id="pone.0001249-Turner1"></a>Turner TL, Hahn MW, Nuzhdin SV (2005) Genomic islands of speciation in <em>Anopheles gambiae</em>. PLoS Biol  3: e285.  <ul class="find" data-citedArticleID="1024752" data-doi="10.1371/journal.pbio.0030285"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030285" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+islands+of+speciation+in+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+islands+of+speciation+in+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001249-Lehmann1" id="pone.0001249-Lehmann1"></a>Lehmann T, Licht M, Elissa N, Maega BT, Chimumbwa JM, et al.  (2003) Population Structure of <em>Anopheles gambiae</em> in Africa. J Hered  94: 133–147.  <ul class="find" data-citedArticleID="1024716" data-doi="10.1093/jhered/esg024"><li><a href="http://dx.doi.org/10.1093/jhered/esg024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+Structure+of+Anopheles+gambiae+in+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+Structure+of+Anopheles+gambiae+in+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001249-dellaTorre3" id="pone.0001249-dellaTorre3"></a>della Torre A, Costantini C, Besansky NJ, Caccone A, Petrarca V, et al.  (2002) Speciation within <em>Anopheles gambiae</em>–the glass is half full. Science  298: 115–117.  <ul class="find" data-citedArticleID="1024776" data-doi="10.1126/science.1078170"><li><a href="http://dx.doi.org/10.1126/science.1078170" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Speciation+within+Anopheles+gambiae%E2%80%93the+glass+is+half+full." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Speciation+within+Anopheles+gambiae%E2%80%93the+glass+is+half+full.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001249-Chandre1" id="pone.0001249-Chandre1"></a>Chandre F, Manguin S, Brengues C, Dossou Yovo J, Darriet F, et al.  (1999) Current distribution of a pyrethroid resistance gene (kdr) in <em>Anopheles gambiae</em> complex from west Africa and further evidence for reproductive isolation of the Mopti form. Parassitologia  41: 319–322.  <ul class="find" data-citedArticleID="1024682"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Current+distribution+of+a+pyrethroid+resistance+gene+%28kdr%29+in+Anopheles+gambiae+complex+from+west+Africa+and+further+evidence+for+reproductive+isolation+of+the+Mopti+form.&amp;auth=&amp;atitle=Current+distribution+of+a+pyrethroid+resistance+gene+%28kdr%29+in+Anopheles+gambiae+complex+from+west+Africa+and+further+evidence+for+reproductive+isolation+of+the+Mopti+form." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Current+distribution+of+a+pyrethroid+resistance+gene+%28kdr%29+in+Anopheles+gambiae+complex+from+west+Africa+and+further+evidence+for+reproductive+isolation+of+the+Mopti+form." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Current+distribution+of+a+pyrethroid+resistance+gene+%28kdr%29+in+Anopheles+gambiae+complex+from+west+Africa+and+further+evidence+for+reproductive+isolation+of+the+Mopti+form.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001249-Weill1" id="pone.0001249-Weill1"></a>Weill M, Chandre F, Brengues C, Manguin S, Akogbeto M, et al.  (2000) The kdr mutation occurs in the Mopti form of <em>Anopheles gambiae s.s.</em> through introgression. Insect Mol Biol  9: 451–455.  <ul class="find" data-citedArticleID="1024762" data-doi="10.1046/j.1365-2583.2000.00206.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.2000.00206.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+kdr+mutation+occurs+in+the+Mopti+form+of+Anopheles+gambiae+s.s.+through+introgression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+kdr+mutation+occurs+in+the+Mopti+form+of+Anopheles+gambiae+s.s.+through+introgression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001249-Turner2" id="pone.0001249-Turner2"></a>Turner TL, Hahn MW (2007) Locus- and population-specific selection and differentiation between incipient species of Anopheles gambiae. Mol Biol Evol  24: 2132–2138.  <ul class="find" data-citedArticleID="1024754" data-doi="10.1093/molbev/msm143"><li><a href="http://dx.doi.org/10.1093/molbev/msm143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locus-+and+population-specific+selection+and+differentiation+between+incipient+species+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locus-+and+population-specific+selection+and+differentiation+between+incipient+species+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001249-Slotman2" id="pone.0001249-Slotman2"></a>Slotman MA, Tripet F, Cornel AJ, Meneses CR, Lee Y, et al.  (2007) Evidence for subdivision within the M molecular form of <em>Anopheles gambiae</em>. Mol Ecol  16: 639–649.  <ul class="find" data-citedArticleID="1024736" data-doi="10.1111/j.1365-294x.2006.03172.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2006.03172.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+subdivision+within+the+M+molecular+form+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+subdivision+within+the+M+molecular+form+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001249-Holt1" id="pone.0001249-Holt1"></a>Holt RA, Subramanian GM, Halpern A, Sutton GG, Charlab R, et al.  (2002) The genome sequence of the malaria mosquito <em>Anopheles gambiae</em>. Science  298: 129–149.  <ul class="find" data-citedArticleID="1024710" data-doi="10.1126/science.1076181"><li><a href="http://dx.doi.org/10.1126/science.1076181" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+the+malaria+mosquito+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+the+malaria+mosquito+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001249-Pritchard1" id="pone.0001249-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="1024726"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001249-Pombi1" id="pone.0001249-Pombi1"></a>Pombi M, Stump AD, Della Torre A, Besansky NJ (2006) Variation in recombination rate across the X chromosome of <em>Anopheles gambiae</em>. Am J Trop Med Hyg  75: 901–903.  <ul class="find" data-citedArticleID="1024722"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Variation+in+recombination+rate+across+the+X+chromosome+of+Anopheles+gambiae.&amp;auth=&amp;atitle=Variation+in+recombination+rate+across+the+X+chromosome+of+Anopheles+gambiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+recombination+rate+across+the+X+chromosome+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+recombination+rate+across+the+X+chromosome+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001249-Slotman3" id="pone.0001249-Slotman3"></a>Slotman MA, Reimer LJ, Thiemann T, Dolo G, Fondjo E, et al.  (2006) Reduced recombination rate and genetic differentiation between the M and S forms of <em>Anopheles gambiae s.s.</em> Genetics  174: 2081–2093.  <ul class="find" data-citedArticleID="1024738" data-doi="10.1534/genetics.106.059949"><li><a href="http://dx.doi.org/10.1534/genetics.106.059949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+recombination+rate+and+genetic+differentiation+between+the+M+and+S+forms+of+Anopheles+gambiae+s.s." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+recombination+rate+and+genetic+differentiation+between+the+M+and+S+forms+of+Anopheles+gambiae+s.s.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001249-He1" id="pone.0001249-He1"></a>He X (1998) Correspondence analysis. Correspondence analysis. BeiJing: RenMen Uni. Publisher. pp. 345–372.  <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001249-Wang2" id="pone.0001249-Wang2"></a>Wang R, Kafatos FC, Zheng L (1999) Microsatellite markers and genotyping procedures for <em>Anopheles gambiae</em>. Parasitol Today  15: 33–37.  <ul class="find" data-citedArticleID="1024760" data-doi="10.1016/s0169-4758(98)01360-x"><li><a href="http://dx.doi.org/10.1016/s0169-4758(98)01360-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellite+markers+and+genotyping+procedures+for+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellite+markers+and+genotyping+procedures+for+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001249-Tripet1" id="pone.0001249-Tripet1"></a>Tripet F, Toure YT, Taylor CE, Norris DE, Dolo G, et al.  (2001) DNA analysis of transferred sperm reveals significant levels of gene flow between molecular forms of <em>Anopheles gambiae</em>. Mol Ecol  10: 1725–1732.  <ul class="find" data-citedArticleID="1024750" data-doi="10.1046/j.0962-1083.2001.01301.x"><li><a href="http://dx.doi.org/10.1046/j.0962-1083.2001.01301.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+analysis+of+transferred+sperm+reveals+significant+levels+of+gene+flow+between+molecular+forms+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+analysis+of+transferred+sperm+reveals+significant+levels+of+gene+flow+between+molecular+forms+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001249-Edillo1" id="pone.0001249-Edillo1"></a>Edillo FE, Toure YT, Lanzaro GC, Dolo G, Taylor CE (2002) Spatial and habitat distribution of <em>Anopheles gambiae</em> and <em>Anopheles arabiensis</em> (Diptera: Culicidae) in Banambani village, Mali. J Med Entomol  39: 70–77.  <ul class="find" data-citedArticleID="1024694" data-doi="10.1603/0022-2585-39.1.70"><li><a href="http://dx.doi.org/10.1603/0022-2585-39.1.70" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+and+habitat+distribution+of+Anopheles+gambiae+and+Anopheles+arabiensis+%28Diptera%3A+Culicidae%29+in+Banambani+village%2C+Mali." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+and+habitat+distribution+of+Anopheles+gambiae+and+Anopheles+arabiensis+%28Diptera%3A+Culicidae%29+in+Banambani+village%2C+Mali.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001249-Gentile2" id="pone.0001249-Gentile2"></a>Gentile G, Della Torre A, Maegga B, Powell JR, Caccone A (2002) Genetic differentiation in the African malaria vector, <em>Anopheles gambiae s.s.</em>, and the problem of taxonomic status. Genetics  161: 1561–1578.  <ul class="find" data-citedArticleID="1024704"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+differentiation+in+the+African+malaria+vector%2C+Anopheles+gambiae+s.s.%2C+and+the+problem+of+taxonomic+status.&amp;auth=&amp;atitle=Genetic+differentiation+in+the+African+malaria+vector%2C+Anopheles+gambiae+s.s.%2C+and+the+problem+of+taxonomic+status." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+differentiation+in+the+African+malaria+vector%2C+Anopheles+gambiae+s.s.%2C+and+the+problem+of+taxonomic+status." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+differentiation+in+the+African+malaria+vector%2C+Anopheles+gambiae+s.s.%2C+and+the+problem+of+taxonomic+status.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001249-Powell1" id="pone.0001249-Powell1"></a>Powell JR, Petrarca V, della Torre A, Caccone A, Coluzzi M (1999) Population structure, speciation, and introgression in the <em>Anopheles gambiae</em> complex. Parassitologia  41: 101–113.  <ul class="find" data-citedArticleID="1024724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Population+structure%2C+speciation%2C+and+introgression+in+the+Anopheles+gambiae+complex.&amp;auth=&amp;atitle=Population+structure%2C+speciation%2C+and+introgression+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+structure%2C+speciation%2C+and+introgression+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+structure%2C+speciation%2C+and+introgression+in+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001249-Rieseberg1" id="pone.0001249-Rieseberg1"></a>Rieseberg LH, Raymond O, Rosenthal DM, Lai Z, Livingstone K, et al.  (2003) Major ecological transitions in wild sunflowers facilitated by hybridization. Science  301: 1211–1216.  <ul class="find" data-citedArticleID="1024728" data-doi="10.1126/science.1086949"><li><a href="http://dx.doi.org/10.1126/science.1086949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+ecological+transitions+in+wild+sunflowers+facilitated+by+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+ecological+transitions+in+wild+sunflowers+facilitated+by+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001249-Scott1" id="pone.0001249-Scott1"></a>Scott JA, Brogdon WG, Collins FH (1993) Identification of single specimens of the <em>Anopheles gambiae</em> complex by the polymerase chain reaction. Am J Trop Med Hyg  49: 520–529.  <ul class="find" data-citedArticleID="1024730"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+single+specimens+of+the+Anopheles+gambiae+complex+by+the+polymerase+chain+reaction.&amp;auth=&amp;atitle=Identification+of+single+specimens+of+the+Anopheles+gambiae+complex+by+the+polymerase+chain+reaction." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+single+specimens+of+the+Anopheles+gambiae+complex+by+the+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+single+specimens+of+the+Anopheles+gambiae+complex+by+the+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001249-Favia2" id="pone.0001249-Favia2"></a>Favia G, della Torre A, Bagayoko M, Lanfrancotti A, Sagnon N, et al.  (1997) Molecular identification of sympatric chromosomal forms of <em>Anopheles gambiae</em> and further evidence of their reproductive isolation. Insect Mol Biol  6: 377–383.  <ul class="find" data-citedArticleID="1024698" data-doi="10.1046/j.1365-2583.1997.00189.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.1997.00189.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+identification+of+sympatric+chromosomal+forms+of+Anopheles+gambiae+and+further+evidence+of+their+reproductive+isolation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+identification+of+sympatric+chromosomal+forms+of+Anopheles+gambiae+and+further+evidence+of+their+reproductive+isolation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001249-Thomasova1" id="pone.0001249-Thomasova1"></a>Thomasova D, Ton LQ, Copley RR, Zdobnov EM, Wang X, et al.  (2002) Comparative genomic analysis in the region of a major Plasmodium-refractoriness locus of <em>Anopheles gambiae</em>. Proc Natl Acad Sci U S A  99: 8179–8184.  <ul class="find" data-citedArticleID="1024746" data-doi="10.1073/pnas.082235599"><li><a href="http://dx.doi.org/10.1073/pnas.082235599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genomic+analysis+in+the+region+of+a+major+Plasmodium-refractoriness+locus+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genomic+analysis+in+the+region+of+a+major+Plasmodium-refractoriness+locus+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001249-Zheng1" id="pone.0001249-Zheng1"></a>Zheng L, Benedict MQ, Cornel AJ, Collins FH, Kafatos FC (1996) An integrated genetic map of the African human malaria vector mosquito, <em>Anopheles gambiae</em>. Genetics  143: 941–952.  <ul class="find" data-citedArticleID="1024766"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+integrated+genetic+map+of+the+African+human+malaria+vector+mosquito%2C+Anopheles+gambiae.&amp;auth=&amp;atitle=An+integrated+genetic+map+of+the+African+human+malaria+vector+mosquito%2C+Anopheles+gambiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrated+genetic+map+of+the+African+human+malaria+vector+mosquito%2C+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrated+genetic+map+of+the+African+human+malaria+vector+mosquito%2C+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001249-Kumar1" id="pone.0001249-Kumar1"></a>Kumar V, Collins FH (1994) A technique for nucleic acid in situ hybridization to polytene chromosomes of mosquitoes in the <em>Anopheles gambiae</em> complex. Insect Mol Biol  3: 41–47.  <ul class="find" data-citedArticleID="1024714" data-doi="10.1111/j.1365-2583.1994.tb00149.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.1994.tb00149.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+technique+for+nucleic+acid+in+situ+hybridization+to+polytene+chromosomes+of+mosquitoes+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+technique+for+nucleic+acid+in+situ+hybridization+to+polytene+chromosomes+of+mosquitoes+in+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001249-Zheng2" id="pone.0001249-Zheng2"></a>Zheng L, Collins FH, Kumar V, Kafatos FC (1993) A detailed genetic map for the X chromosome of the malaria vector, <em>Anopheles gambiae</em> [see comments]. Science  261: 605–608.  <ul class="find" data-citedArticleID="1024768" data-doi="10.1126/science.8342025"><li><a href="http://dx.doi.org/10.1126/science.8342025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+detailed+genetic+map+for+the+X+chromosome+of+the+malaria+vector%2C+Anopheles+gambiae+%5Bsee+comments%5D." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+detailed+genetic+map+for+the+X+chromosome+of+the+malaria+vector%2C+Anopheles+gambiae+%5Bsee+comments%5D.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001249-Lehmann2" id="pone.0001249-Lehmann2"></a>Lehmann T, Hawley WA, Kamau L, Fontenille D, Simard F, et al.  (1996) Genetic differentiation of <em>Anopheles gambiae</em> populations from East and west Africa: comparison of microsatellite and allozyme loci. Heredity  77 (Pt 2): 192–200.  <ul class="find" data-citedArticleID="1024718" data-doi="10.1038/hdy.1996.124"><li><a href="http://dx.doi.org/10.1038/hdy.1996.124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+differentiation+of+Anopheles+gambiae+populations+from+East+and+west+Africa%3A+comparison+of+microsatellite+and+allozyme+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+differentiation+of+Anopheles+gambiae+populations+from+East+and+west+Africa%3A+comparison+of+microsatellite+and+allozyme+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001249-Dimopoulos1" id="pone.0001249-Dimopoulos1"></a>Dimopoulos G, Zheng L, Kumar V, della Torre A, Kafatos FC, et al.  (1996) Integrated genetic map of <em>Anopheles gambiae</em>: use of RAPD polymorphisms for genetic, cytogenetic and STS landmarks. Genetics  143: 953–960.  <ul class="find" data-citedArticleID="1024692"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Integrated+genetic+map+of+Anopheles+gambiae%3A+use+of+RAPD+polymorphisms+for+genetic%2C+cytogenetic+and+STS+landmarks.&amp;auth=&amp;atitle=Integrated+genetic+map+of+Anopheles+gambiae%3A+use+of+RAPD+polymorphisms+for+genetic%2C+cytogenetic+and+STS+landmarks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrated+genetic+map+of+Anopheles+gambiae%3A+use+of+RAPD+polymorphisms+for+genetic%2C+cytogenetic+and+STS+landmarks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrated+genetic+map+of+Anopheles+gambiae%3A+use+of+RAPD+polymorphisms+for+genetic%2C+cytogenetic+and+STS+landmarks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001249-Zheng3" id="pone.0001249-Zheng3"></a>Zheng L, Wang S, Romans P, Zhao H, Luna C, et al.  (2003) Quantitative trait loci in <em>Anopheles gambiae</em> controlling the encapsulation response against Plasmodium cynomolgi Ceylon. BMC Genet  4: 16.  <ul class="find" data-citedArticleID="1024770"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantitative+trait+loci+in+Anopheles+gambiae+controlling+the+encapsulation+response+against+Plasmodium+cynomolgi+Ceylon.&amp;auth=&amp;atitle=Quantitative+trait+loci+in+Anopheles+gambiae+controlling+the+encapsulation+response+against+Plasmodium+cynomolgi+Ceylon." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+loci+in+Anopheles+gambiae+controlling+the+encapsulation+response+against+Plasmodium+cynomolgi+Ceylon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+loci+in+Anopheles+gambiae+controlling+the+encapsulation+response+against+Plasmodium+cynomolgi+Ceylon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001249-Archak1" id="pone.0001249-Archak1"></a>Archak S, Meduri E, Kumar PS, Nagaraju J (2007) InSatDb: a microsatellite database of fully sequenced insect genomes. Nucleic Acids Res  35: D36–39.  <ul class="find" data-citedArticleID="1024672" data-doi="10.1093/nar/gkl778"><li><a href="http://dx.doi.org/10.1093/nar/gkl778" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=InSatDb%3A+a+microsatellite+database+of+fully+sequenced+insect+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22InSatDb%3A+a+microsatellite+database+of+fully+sequenced+insect+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001249-He2" id="pone.0001249-He2"></a>He X (1998) Principal component ananlysis. Principal component ananlysis. BeiJing: RenMen Uni. Publisher. pp. 281–315.  <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0001249-Ueberla1" id="pone.0001249-Ueberla1"></a>Ueberla K (1977) Faktorenanalyse: Springer-Verlag.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.XML" value="152961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.PDF" value="521138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g001.PNG_L" value="842048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g001.PNG_M" value="60462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g001.PNG_S" value="6896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g001.TIF" value="1161846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g001.PNG_I" value="24200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t001.PNG_L" value="105937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t001.PNG_M" value="58818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t001.PNG_S" value="10073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t001.TIF" value="518922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t001.PNG_I" value="18099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g002.PNG_L" value="276345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g002.PNG_M" value="76771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g002.PNG_S" value="11258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g002.TIF" value="1427552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g002.PNG_I" value="27460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t002.PNG_L" value="95037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t002.PNG_M" value="135996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t002.PNG_S" value="17523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t002.TIF" value="408928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t002.PNG_I" value="47330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t003.PNG_L" value="76132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t003.PNG_M" value="45502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t003.PNG_S" value="8915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t003.TIF" value="356848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t003.PNG_I" value="13663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g003.PNG_L" value="1836929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g003.PNG_M" value="176621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g003.PNG_S" value="12478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g003.TIF" value="2795400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g003.PNG_I" value="61997"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t004.PNG_L" value="269274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t004.PNG_M" value="111404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t004.PNG_S" value="27015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t004.TIF" value="1353146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.t004.PNG_I" value="196869"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g004.PNG_L" value="603524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g004.PNG_M" value="142380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g004.PNG_S" value="10674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g004.TIF" value="1392088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.g004.PNG_I" value="51450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s001.TIF" value="1383661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s002.TIF" value="1137644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s003.TIF" value="1166862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s004.TIF" value="1157090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s005.TIF" value="1154850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s006.TIF" value="2202615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s007.DOC" value="207872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s008.XLS" value="268288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s009.DOC" value="113664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001249.s010.DOC" value="29184"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001249&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001249" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001249&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001249&volume=&issue=&title=Mosaic Genome Architecture of the Anopheles gambiae Species Complex&author_name=Rui%20Wang-Sattler%2C%20Stephanie%20Blandin%2C%20Ye%20Ning%2C%20Claudia%20Blass%2C%20Guimogo%20Dolo%2C%20Yeya%20T.%20Tour%C3%A9%2C%20Alessandra%20della%20Torre%2C%20Gregory%20C.%20Lanzaro%2C%20Lars%20M.%20Steinmetz%2C%20Fotis%20C.%20Kafatos%2C%20Liangbiao%20Zheng&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001249" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249&amp;t=Mosaic%20Genome%20Architecture%20of%20the%20Anopheles%20gambiae%20Species%20Complex" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249&title=Mosaic%20Genome%20Architecture%20of%20the%20Anopheles%20gambiae%20Species%20Complex&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249&amp;title=Mosaic%20Genome%20Architecture%20of%20the%20Anopheles%20gambiae%20Species%20Complex" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001249&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mosaic Genome Architecture of the Anopheles gambiae Species Complex';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001249';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mosaic%20Genome%20Architecture%20of%20the%20Anopheles%20gambiae%20Species%20Complex http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001249" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001249" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Centromeres%22" title="Search for articles in the subject area:'Centromeres'"><div class="flagText">Centromeres</div></a>
              <div data-categoryid="43147" data-articleid="25330"
                   data-categoryname="Centromeres"
                   class="flagImage" title="Flag 'Centromeres' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="25330"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25330"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomics%22" title="Search for articles in the subject area:'Genomics'"><div class="flagText">Genomics</div></a>
              <div data-categoryid="18221" data-articleid="25330"
                   data-categoryname="Genomics"
                   class="flagImage" title="Flag 'Genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microsatellite+loci%22" title="Search for articles in the subject area:'Microsatellite loci'"><div class="flagText">Microsatellite loci</div></a>
              <div data-categoryid="41413" data-articleid="25330"
                   data-categoryname="Microsatellite loci"
                   class="flagImage" title="Flag 'Microsatellite loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+myopathy%22" title="Search for articles in the subject area:'Mitochondrial myopathy'"><div class="flagText">Mitochondrial myopathy</div></a>
              <div data-categoryid="27431" data-articleid="25330"
                   data-categoryname="Mitochondrial myopathy"
                   class="flagImage" title="Flag 'Mitochondrial myopathy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mosquitoes%22" title="Search for articles in the subject area:'Mosquitoes'"><div class="flagText">Mosquitoes</div></a>
              <div data-categoryid="31953" data-articleid="25330"
                   data-categoryname="Mosquitoes"
                   class="flagImage" title="Flag 'Mosquitoes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22X+chromosomes%22" title="Search for articles in the subject area:'X chromosomes'"><div class="flagText">X chromosomes</div></a>
              <div data-categoryid="17587" data-articleid="25330"
                   data-categoryname="X chromosomes"
                   class="flagImage" title="Flag 'X chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9436'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5373'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2166&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
