

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001012"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001012" />

    <meta name="citation_title" content="Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity"/>
    <meta itemprop="name" content="Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity"/>

      <meta name="citation_author" content="Saeid Naderi"/>
            <meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France"/>
            <meta name="citation_author_institution" content="Natural Resources Faculty, University of Guilan, Guilan, Iran"/>
      <meta name="citation_author" content="Hamid-Reza Rezaei"/>
            <meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France"/>
            <meta name="citation_author_institution" content="Environmental Sciences Department, Gorgan University of Agriculture and Natural Resources, Gorgan, Iran"/>
      <meta name="citation_author" content="Pierre Taberlet"/>
            <meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France"/>
      <meta name="citation_author" content="Stéphanie Zundel"/>
            <meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France"/>
      <meta name="citation_author" content="Seyed-Abbas Rafat"/>
            <meta name="citation_author_institution" content="Animal Science Department, Faculty of Agriculture, University of Tabriz, Tabriz, Iran"/>
      <meta name="citation_author" content="Hamid-Reza Naghash"/>
            <meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France"/>
      <meta name="citation_author" content="Mohamed A. A. El-Barody"/>
            <meta name="citation_author_institution" content="Animal Production Department, Faculty of Agriculture, Minia University, Minia, Egypt"/>
      <meta name="citation_author" content="Okan Ertugrul"/>
            <meta name="citation_author_institution" content="Department of Genetics, Faculty of Veterinary Medicine, Ankara University, Ankara, Turkey"/>
      <meta name="citation_author" content="François Pompanon"/>
            <meta name="citation_author_institution" content="Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France"/>
      <meta name="citation_author" content="for the Econogene Consortium"/>

    <meta name="citation_date" content="2007/10/10"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001012.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1012"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Domesticated animals from early times.; citation_author=J Clutton-Brock; citation_number=1; citation_pages=238; citation_date=1999; citation_publisher=The Natural History Museum: Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Goats of the world.; citation_author=V Porter; citation_number=2; citation_pages=172; citation_date=1996; citation_publisher=Farming Press; " />
      <meta name="citation_reference" content="citation_title=Reading the signs of ancient animal domestication.; citation_author=H Pringle; citation_journal_title=Science; citation_volume=282; citation_number=3; citation_pages=1448-1450; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Documenting domestication: the intersection of genetics and archaeology.; citation_author=M Zeder; citation_author=E Emshwiller; citation_author=B Smith; citation_author=D Bradley; citation_journal_title=Trends Genet; citation_volume=22; citation_number=4; citation_pages=139-155; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Livestock genetic origins: goats buck the trend.; citation_author=D MacHugh; citation_author=D Bradley; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=5; citation_pages=5382-5384; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Origins and Spread of Agriculture and Pastoralism in northwestern South Asia. In: Harris DR. The Origins and Spread of Agriculture and Pastoralism in Eurasia.; citation_author=RH Meadow; citation_number=6; citation_pages=390-412; citation_date=1996; citation_publisher=UCL Press; " />
      <meta name="citation_reference" content="citation_title=The initial domestication of goats (Capra hircus) in the Zagros Mountains 10,000 years ago.; citation_author=M Zeder; citation_author=B Hesse; citation_journal_title=Sciences; citation_volume=287; citation_number=7; citation_pages=2254-2257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A view from the Zagros: new perspectives on livestock domestication in the Fertile Crescent.; citation_author=M Zeder; citation_author=JD Vigne; citation_author=D Helmer; citation_number=8; citation_pages=125-146; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bezoar (Capra aegagrus) is a matriarchal candidate for ancestor of domestic goat (Capra hircus): evidence from the mitochondrial DNA diversity.; citation_author=T Takada; citation_author=Y Kikkawa; citation_author=H Yonekawa; citation_author=S Kawakami; citation_author=T Amano; citation_journal_title=Biochem Genet; citation_volume=35(9–10); citation_number=9; citation_pages=315-326; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Systematics of the genus Capra inferred from mitochondrial DNA sequence data.; citation_author=V Manceau; citation_author=L Despres; citation_author=J Bouvet; citation_author=P Taberlet; citation_journal_title=Mol Phylogenet Evol; citation_volume=13(3); citation_number=10; citation_pages=504-510; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evolutionary history of the genus Capra (Mammalia, Artiodactyla): discordance between mitochondrial DNA and Y-chromosome phylogenies.; citation_author=N Pidancier; citation_author=S Jordan; citation_author=G Luikart; citation_author=P Taberlet; citation_journal_title=Mol Phylogenet Evol; citation_volume=40; citation_number=11; citation_pages=739-749; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA markers reveal the complexity of livestock domestication.; citation_author=M Bruford; citation_author=D Bradley; citation_author=G Luikart; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=12; citation_pages=900-910; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Multiple maternal origins and weak phylogeographic structure in domestic goats.; citation_author=G Luikart; citation_author=L Gielly; citation_author=L Excoffier; citation_author=JD Vigne; citation_author=J Bouvet; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=13; citation_pages=5927-5932; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA diversity of Pakistani goats.; citation_author=S Sultana; citation_author=H Mannen; citation_author=S Tsuji; citation_journal_title=Anim Genet; citation_volume=34(6); citation_number=14; citation_pages=417-421; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogeography and origin of Indian domestic goats.; citation_author=MB Joshi; citation_author=PK Rout; citation_author=AK Mandal; citation_author=C Tyler-Smith; citation_author=L Singh; citation_journal_title=Mol Biol Evol; citation_volume=21(3); citation_number=15; citation_pages=454-462; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial diversity and phylogeographic structure of Chinese domestic goats.; citation_author=SY Chen; citation_author=YH Su; citation_author=SF Wu; citation_author=T Sha; citation_author=YP Zhang; citation_journal_title=Mol Phylogenet Evol; citation_volume=37(3); citation_number=16; citation_pages=804-814; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial DNA diversity of Korean native goats.; citation_author=S Odahara; citation_author=H Chung; citation_author=S Choi; citation_author=S Yu; citation_author=S Sasazaki; citation_journal_title=Asian Australas J Anim Sci; citation_volume=19; citation_number=17; citation_pages=482-485; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of Sicilian goats reveals a new mtDNA lineage.; citation_author=MT Sardina; citation_author=M Ballester; citation_author=J Marmi; citation_author=R Finocchiaro; citation_author=JB van Kaam; citation_journal_title=Anim Genet; citation_volume=37(4); citation_number=18; citation_pages=376-378; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Strong phylogeographic relationships among three goat breeds from the Canary Islands.; citation_author=M Amills; citation_author=J Capote; citation_author=A Tomas; citation_author=L Kelly; citation_author=G Obexer-Ruff; citation_journal_title=J Dairy Res; citation_volume=71(3); citation_number=19; citation_pages=257-262; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic relationships among Spanish goats breeds.; citation_author=PJ Azor; citation_author=LV Monteagudo; citation_author=M Luque; citation_author=MT Tejedor; citation_author=E Rodero; citation_journal_title=Anim Genet; citation_volume=36(5); citation_number=20; citation_pages=423-425; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The mtDNA catalogue of all Portuguese autochthonous goat (Capra hircus) breeds: high diversity of female lineages at the western fringe of European distribution.; citation_author=F Pereira; citation_author=L Pereira; citation_author=B Van Asch; citation_author=D Bradley; citation_author=A Amorim; citation_journal_title=Mol Ecol; citation_volume=14; citation_number=21; citation_pages=2313-2318; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Population growth makes waves in the distribution of pairwise genetic differences.; citation_author=A Rogers; citation_author=H Harpending; citation_journal_title=Mol Biol Evol; citation_volume=9; citation_number=22; citation_pages=552-569; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The Pedigree Rate of Sequence Divergence in the Human Mitochondrial Genome: There Is a Difference Between Phylogenetic and Pedigree Rates.; citation_author=N Howell; citation_author=CB Smejkal; citation_author=DA Mackey; citation_author=PF Chinnery; citation_author=DM Turnbull; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=23; citation_pages=659-670; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High Direct Estimate of the Mutation Rate in the Mitochondrial Genome of Caenorhabditis elegans.; citation_author=DR Denver; citation_author=K Morris; citation_author=M Lynch; citation_author=LL Vassilieva; citation_author=WK Thomas; citation_journal_title=Science; citation_volume=289; citation_number=24; citation_pages=2342; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rates of Evolution in Ancient DNA from Adélie Penguins.; citation_author=DM Lambert; citation_author=PA Ritchie; citation_author=CD Millar; citation_author=B Holland; citation_author=AJ Drummond; citation_journal_title=Science; citation_volume=295; citation_number=25; citation_pages=2270; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial diversity and the origins of African and European cattle.; citation_author=D Bradley; citation_author=D MacHugh; citation_author=P Cunningham; citation_author=R Loftus; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=93; citation_number=26; citation_pages=5131-5135; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Genetic evidence for a large scale domestication in goats; citation_author=S Naderi; citation_author=HR Rezaei; citation_author=F Pompanon; citation_author=MGB Blum; citation_author=R Negrini; citation_number=27; citation_date=In Prep.; " />
      <meta name="citation_reference" content="citation_title=Origin and evolution of homologous repeated sequences in the mitochondrial DNA control region of shrews.; citation_author=L Fumagalli; citation_author=P Taberlet; citation_author=L Favre; citation_author=J Hausser; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=28; citation_pages=31-46; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The complete mitochondrial DNA sequence of the domestic sheep (Ovis aries) and comparison with the other major ovine haplotype.; citation_author=S Hiendleder; citation_author=H Lewalski; citation_author=R Wassmuth; citation_author=A Janke; citation_journal_title=J Mol Evol; citation_volume=47; citation_number=29; citation_pages=441-448; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Mitochondrial Sequence Reveals High Levels of Gene Flow Between Breeds of Domestic Sheep from Asia and Europe.; citation_author=JRS Meadows; citation_author=K Li; citation_author=J Kantanen; citation_author=M Tapio; citation_author=W Sipos; citation_journal_title=J Hered; citation_volume=96(5); citation_number=30; citation_pages=494-501; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Five Ovine Mitochondrial Lineages Identified From Sheep Breeds of the Near East.; citation_author=JRS Meadows; citation_author=I Cemal; citation_author=O Karaca; citation_author=E Gootwine; citation_author=JW Kijas; citation_journal_title=Genetics; citation_volume=175; citation_number=31; citation_pages=1371-1379; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Independent mitochondrial origin and historical genetic differentiation in North Eastern Asian cattle.; citation_author=H Mannen; citation_author=M Kohno; citation_author=Y Nagata; citation_author=S Tsuji; citation_author=DG Bradley; citation_journal_title=Mol Phylogenet Evol; citation_volume=32; citation_number=32; citation_pages=539-544; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Admixture and diversity in West African cattle populations.; citation_author=AR Freeman; citation_author=CM Meghan; citation_author=DE Machugh; citation_author=RT Loftus; citation_author=MD Achukwi; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=33; citation_pages=3477-3487; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Divergent mtDNA lineages of goats in an Early Neolithic site, far from the initial domestication areas.; citation_author=H Fernández; citation_author=S Hugues; citation_author=JD Vigne; citation_author=D Helmer; citation_author=G Hodgins; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=34; citation_pages=15375-15379; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=J.-D Vigne; citation_number=35; citation_pages=52-75; " />
      <meta name="citation_reference" content="citation_title=Genetic relationships between 14 native spanish breeds of goat.; citation_author=M Tuñon; citation_author=JP Gonzalez; citation_author=M Vallejo; citation_journal_title=Anim Genet; citation_volume=20; citation_number=36; citation_pages=205-212; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Geographical partitioning of goat diversity in Europe and the Middle East.; citation_author=J Cañon; citation_author=D García; citation_author=M García-Atance; citation_author=AG Obexer-Ruff; citation_author=J Lenstra; citation_journal_title=Anim Genet; citation_volume=37; citation_number=37; citation_pages=327-334; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Assessment of population structure by single nucleotide polymorphisms (SNPs) in goat breeds.; citation_author=L Pariset; citation_author=I Cappuccio; citation_author=P Ajmone Marsan; citation_author=S Dunner; citation_author=G Luikart; citation_journal_title=J Chromatography B; citation_number=38; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Are cattle, sheep, and goats endangered species?; citation_author=P Taberlet; citation_author=P Ajmone-Marsan; citation_author=A Valentini; citation_author=HR Rezaei; citation_author=S Naderi; citation_journal_title=Mol Ecol; citation_number=39; citation_date=In Press; " />
      <meta name="citation_reference" content="citation_title=The power and promise of population genomics: from genotyping to genome typing.; citation_author=G Luikart; citation_author=P England; citation_author=D Tallmon; citation_author=S Jordan; citation_author=P Taberlet; citation_journal_title=Nat Rev Genet; citation_volume=4; citation_number=40; citation_pages=981-994; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Wild sheep and goats and their relatives: Status survey and conservation action plan for Caprinae.; citation_author=DM Shackleton; citation_number=41; citation_pages=390; citation_date=1997; citation_publisher=IUCN; " />
      <meta name="citation_reference" content="citation_title=The complete nucleotide sequence of goat (Capra hircus) mitochondrial genome. Goat mitochondrial genome.; citation_author=P Parma; citation_author=M Feligini; citation_author=G Greeppi; citation_author=G Enne; citation_journal_title=DNA Seq; citation_volume=14(3); citation_number=42; citation_pages=199-203; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignement.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=43; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=CABIOS; citation_volume=13; citation_number=44; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=PAUP*: Phylogenetic Analysis Using Parsimony (*and Other Methods), Version 4.; citation_author=DL Swofford; citation_number=45; citation_date=2003; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Arlequin ver. 3.0: An integrated software package for population genetics data analysis.; citation_author=L Excoffier; citation_author=G Laval; citation_author=S Schneider; citation_journal_title=Evolutionary Bioinformatics Online; citation_volume=1; citation_number=46; citation_pages=47-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA.; citation_author=S Schneider; citation_author=L Excoffier; citation_journal_title=Genetics; citation_volume=152; citation_number=47; citation_pages=1079-1089; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=LAMARC 2.0: maximum likelihood and Bayesian estimation of population parameters.; citation_author=M Kuhner; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=48; citation_pages=768-770; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity"/>
    <meta name="twitter:description" content="BackgroundFrom the beginning of domestication, the transportation of domestic animals resulted in genetic and demographic processes that explain their present distribution and genetic structure. Thus studying the present genetic diversity helps to better understand the history of domestic species.Methodology/Principal FindingsThe genetic diversity of domestic goats has been characterized with 2430 individuals from all over the old world, including 946 new individuals from regions poorly studied until now (mainly the Fertile Crescent). These individuals represented 1540 haplotypes for the HVI segment of the mitochondrial DNA (mtDNA) control region. This large-scale study allowed the establishment of a clear nomenclature of the goat maternal haplogroups. Only five of the six previously defined groups of haplotypes were divergent enough to be considered as different haplogroups. Moreover a new mitochondrial group has been localized around the Fertile Crescent. All groups showed very high haplotype diversity. Most of this diversity was distributed among groups and within geographic regions. The weak geographic structure may result from the worldwide distribution of the dominant A haplogroup (more than 90% of the individuals). The large-scale distribution of other haplogroups (except one), may be related to human migration. The recent fragmentation of local goat populations into discrete breeds is not detectable with mitochondrial markers. The estimation of demographic parameters from mismatch analyses showed that all groups had a recent demographic expansion corresponding roughly to the period when domestication took place. But even with a large data set it remains difficult to give relative dates of expansion for different haplogroups because of large confidence intervals.Conclusions/SignificanceWe propose standard criteria for the definition of the different haplogroups based on the result of mismatch analysis and on the use of sequences of reference. Such a method could be also applied for clarifying the nomenclature of mitochondrial haplogroups in other domestic species."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001012.g003"/>

  <meta property="og:title" content="Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6938'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3107'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7408&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001012">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001012" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001012&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Saeid Naderi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France, 
                  Natural Resources Faculty, University of Guilan, Guilan, Iran
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hamid-Reza Rezaei, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France, 
                  Environmental Sciences Department, Gorgan University of Agriculture and Natural Resources, Gorgan, Iran
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pierre Taberlet, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stéphanie Zundel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Seyed-Abbas Rafat, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Animal Science Department, Faculty of Agriculture, University of Tabriz, Tabriz, Iran
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hamid-Reza Naghash, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mohamed A. A. El-Barody, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Animal Production Department, Faculty of Agriculture, Minia University, Minia, Egypt
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Okan Ertugrul, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Faculty of Veterinary Medicine, Ankara University, Ankara, Turkey
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            François Pompanon
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:francois.pompanon@ujf-grenoble.fr">francois.pompanon@ujf-grenoble.fr</a></p>

                <p>Affiliation:
                  Laboratoire d'Ecologie Alpine, CNRS-UMR 5553, Université Joseph Fourier, Grenoble, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author no_author_meta">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            for the Econogene Consortium
          </span>
        </span>

      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 10, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001012</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001012-g001" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-g002" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-t001" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-t002" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-t003" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-g003" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-t004" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001012-t005" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001012">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001012" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>From the beginning of domestication, the transportation of domestic animals resulted in genetic and demographic processes that explain their present distribution and genetic structure. Thus studying the present genetic diversity helps to better understand the history of domestic species.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The genetic diversity of domestic goats has been characterized with 2430 individuals from all over the old world, including 946 new individuals from regions poorly studied until now (mainly the Fertile Crescent). These individuals represented 1540 haplotypes for the HVI segment of the mitochondrial DNA (mtDNA) control region. This large-scale study allowed the establishment of a clear nomenclature of the goat maternal haplogroups. Only five of the six previously defined groups of haplotypes were divergent enough to be considered as different haplogroups. Moreover a new mitochondrial group has been localized around the Fertile Crescent. All groups showed very high haplotype diversity. Most of this diversity was distributed among groups and within geographic regions. The weak geographic structure may result from the worldwide distribution of the dominant A haplogroup (more than 90% of the individuals). The large-scale distribution of other haplogroups (except one), may be related to human migration. The recent fragmentation of local goat populations into discrete breeds is not detectable with mitochondrial markers. The estimation of demographic parameters from mismatch analyses showed that all groups had a recent demographic expansion corresponding roughly to the period when domestication took place. But even with a large data set it remains difficult to give relative dates of expansion for different haplogroups because of large confidence intervals.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We propose standard criteria for the definition of the different haplogroups based on the result of mismatch analysis and on the use of sequences of reference. Such a method could be also applied for clarifying the nomenclature of mitochondrial haplogroups in other domestic species.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Naderi S, Rezaei H-R, Taberlet P, Zundel S, Rafat S-A, et al.  (2007) Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity. PLoS ONE 2(10):
          e1012.
            doi:10.1371/journal.pone.0001012</p><p><strong>Academic Editor: </strong>Henry Harpending, University of Utah, United States of America</p><p><strong>Received:</strong> May 21, 2007; <strong>Accepted:</strong> September 17, 2007; <strong>Published:</strong> October 10, 2007</p><p><strong>Copyright:</strong> © 2007 Naderi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>S. Naderi and HR. Rezaei were supported by PhD scholarships from the Iranian Ministry of Science, Research, and Technology (number 800125 and, 791135 respectively). F. Pompanon was supported by the Institut National de la Recherche Agronomique. This work has been partially supported by the Chronobos project (ANR 05-GANI-004-02) and by the EU ECONOGENE contract QLK5-CT-2001-02461. The content of the publication does not represent necessarily the views of the European Commission or its services.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>More than 10,000 years ago, the transition of humans from hunting to the manipulation of the behavior of certain animals lead to the process of domestication <a href="#pone.0001012-CluttonBrock1">[1]</a>. This process contributed to the rise of human civilization by enabling people to settle into a sedentary lifestyle. The goat was one of the first domesticated animals <a href="#pone.0001012-Porter1">[2]</a>–<a href="#pone.0001012-Zeder1">[4]</a>. It was a source of milk, meat, dung for fuel and materials for clothing and building such as hair, bone and skin <a href="#pone.0001012-CluttonBrock1">[1]</a>, <a href="#pone.0001012-MacHugh1">[5]</a>. Archaeological studies suggested that the domestic goat <em>Capra hircus</em> was domesticated from the bezoar <em>Capra aegagrus</em> in the Fertile Crescent <a href="#pone.0001012-Meadow1">[e.g. 6]</a>–<a href="#pone.0001012-Zeder3">[8]</a>. This origin was confirmed by genetic studies based on mitochondrial <a href="#pone.0001012-Takada1">[e.g. 9]</a>, <a href="#pone.0001012-Manceau1">[10]</a> and nuclear DNA <a href="#pone.0001012-Pidancier1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>From the beginning of the domestication process, the exchange and transportation of domestic animals has been related to human migration and trade. This resulted in genetic (e.g., selection, gene flow) and demographic processes that explain the present worldwide distribution of more than 300 different breeds of <em>Capra hircus</em> and their genetic structure <a href="#pone.0001012-Porter1">[2]</a>. Thus, the present genetic diversity bears the molecular signature of past events, such as rapid demographic expansions. Therefore, the study of this diversity helps to reconstitute the evolutionary history of the goat <a href="#pone.0001012-Bruford1">[12]</a> and could bring new facts that help to understand the history of domestication.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Mitochondrial DNA is commonly used for the study of domesticated species. The control region has been especially used for describing the genetic polymorphism of goats <a href="#pone.0001012-Luikart1">[13]</a>, because it is variable and structured enough across the geographical range of the species, and evolves at a constant rate <a href="#pone.0001012-Bruford1">[12]</a>. Moreover, it allows maternal lineages to be followed and is less sensitive to introgression from wild species than nuclear DNA <a href="#pone.0001012-Luikart1">[13]</a>. However, studies on nuclear genes are needed because they give information on gene flow and selection processes that had a great influence on the evolution of livestock species <a href="#pone.0001012-Bruford1">[12]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Luikart et al. <a href="#pone.0001012-Luikart1">[13]</a> conducted the first study of the overall genetic structure of domestic goats at the worldwide scale. They analyzed 406 individuals representing 88 breeds from the old world. They found three mitochondrial haplogroups (A, B and C) that diverged more than 200,000 years ago and have undergone demographic expansion at different times. This would suggest multiple maternal origins of domestic goats or introgression of other haplotypes after the first domestication event. Moreover, they showed that most of genetic diversity occurred within breeds, and interpreted the very weak geographic structure as the result of the extensive transportation of goats among continents.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The initial global survey by Luikart et al. <a href="#pone.0001012-Luikart1">[13]</a> has been followed by regional studies describing more precisely the genetic diversity of goat breeds. However, these studies were always realized in restricted geographic regions corresponding to different countries such as Pakistan <a href="#pone.0001012-Sultana1">[14]</a>, India <a href="#pone.0001012-Joshi1">[15]</a>, China <a href="#pone.0001012-Chen1">[16]</a>, South Korea <a href="#pone.0001012-Odahara1">[17]</a>, Sicily <a href="#pone.0001012-Sardina1">[18]</a>, Spain <a href="#pone.0001012-Amills1">[19]</a>, <a href="#pone.0001012-Azor1">[20]</a> and Portugal <a href="#pone.0001012-Pereira1">[21]</a>. The existence of three new haplogroups has been suggested <a href="#pone.0001012-Sultana1">[14]</a>, <a href="#pone.0001012-Joshi1">[15]</a>, <a href="#pone.0001012-Sardina1">[18]</a>. However, this has sometimes been based only on a few individuals, and without comparing the new divergent haplotypes to a sample representative of the worldwide haplotype diversity. In general, the identification of a new haplogroup might be controversial in the absence of standardized criteria. All previous studies describing the mitochondrial polymorphism of domestic animals use the term of “maternal lineage” for characterizing a group of closely related haplotypes. However, this term is ambiguous as it usually corresponds to many haplotypes, and thus to many maternal lineages <em>sensu stricto</em>. As a consequence, we propose to use “mitochondrial haplogroup” instead of “maternal lineage”, a term that is already in common use in genetic studies.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In this context, the goals of the present study are (i) to characterize the domestic goat mtDNA diversity based on a worldwide sampling and make a global synthesis including previous studies, (ii) to establish the relationships between mitochondrial haplogroups and to propose a clear nomenclature, and (iii) to give standard criteria for the definition of mitochondrial haplogroups. For this purpose we used data from previous studies (1484 sequences retrieved from GeneBank), and we analyzed 946 new samples from all over the old world. New samples were especially taken from localities that have not been adequately sampled before, and that may have played an important role in the history of goat domestication (i.e., Middle East and especially the Fertile Crescent).</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Sequence polymorphism</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The HVI fragment of the control region shows a high polymorphism with 336 variable sites over the 558 bp of the alignment. We observed 285 substitutions (226 transitions and 59 transversions) and 110 insertions/deletions (from 1 to 76 bp). The 2430 individuals correspond to 1540 different haplotypes.</p>


<h4>Phylogenetic analysis and genetic structure of domestic goats</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The Neighbor-joining tree of the 2430 domestic goats (<a href="#pone-0001012-g001">Figure 1</a>) shows 6 highly divergent groups corresponding to different mitochondrial haplogroups called A, B, C, D, F (according to previous studies) and G. Each group has high haplotype diversity (<a href="#pone-0001012-t001">Table 1</a>), and has been defined by high bootstrap values (except a bootstrap of 53 % for A ; <a href="#pone-0001012-g001">Figure 1A</a>), and by high mean pairwise distance with all other groups (see below). The A haplogroup is the most represented when considering either the number of individuals or the number of haplotypes and is highly dominant all over the old world (<a href="#pone-0001012-t001">Table 1</a> and <a href="#pone-0001012-g002">Figure 2</a>). Except for two individuals situated at the base of the B group, this clade is composed of two sub-groups, B1 (35 haplotypes) and B2 (9 haplotypes), as previously defined by Chen et al. <a href="#pone.0001012-Chen1">[16]</a>. The B group is mostly found in whole Asia, with a few individuals from the Sub-Saharan Africa and one European goat from Greece. The B2 individuals are restricted to China and Mongolia. Goats from the C group are from whole Asia and Europe and the D group is present in the whole Asia and Northern Europe. The three goats from the F group are from Sicily. The G group has not been reported until now, and is present in Middle East and Northern Africa, near the Fertile Crescent.</p>
<div class="figure" id="pone-0001012-g001"><div class="img"><a name="pone-0001012-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Neighbor-joining trees of domestic goat based on 1540 mtDNA haplotypes (A) and on the 22 reference mtDNA haplotypes (B).</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Distances were calculated using the Kimura 2-Parameter model with gamma correction (alpha = 0.28). On the (A) tree, the numbers on the branches represent bootstrap values out of 1000 replications, and the stars point out the position of reference individuals for each haplogroup used to construct the (B) tree (see <a href="#pone-0001012-t005">Table 5</a>).</p>
<span>doi:10.1371/journal.pone.0001012.g001</span></div><div class="figure" id="pone-0001012-g002"><div class="img"><a name="pone-0001012-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Geographic distribution of domestic goat mtDNA haplogroups.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The size of each circle is proportional to the sample size and each specific haplotype is represented by a different colour.</p>
<span>doi:10.1371/journal.pone.0001012.g002</span></div><div class="figure" id="pone-0001012-t001"><div class="img"><a name="pone-0001012-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Genetic diversity of goat mtDNA haplogroups</span></strong></p><span>doi:10.1371/journal.pone.0001012.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The haplotype diversity is very high all over the Eurasia and Africa with a value above 0.97 in 39 of the 54 studied countries (<a href="#pone-0001012-t002">Table 2</a>). More than 77% of the mtDNA variation is distributed within breeds while about 11% is found among breeds within geographic regions and 12 % among geographic regions (<a href="#pone-0001012-t003">Table 3</a>). Nevertheless this low but significant geographic structure is coherent with the fact that all breeds are composed of individuals from the A group, with eventually a lower percentage of individuals from other haplogroups (for about 25% of the breeds). This low geographic structure is also confirmed by the distribution of all haplogroups that are present in several regions (except for F). Most of the mtDNA diversity is distributed among groups and within geographic regions, while less than 4% of this variability is found among regions within groups (<a href="#pone-0001012-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001012-t002"><div class="img"><a name="pone-0001012-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Geographic origin and characteristics of the studied domestic goat</span></strong></p><span>doi:10.1371/journal.pone.0001012.t002</span></div><div class="figure" id="pone-0001012-t003"><div class="img"><a name="pone-0001012-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Partition of the genetic variance among haplogroups, breeds and continental regions revealed by hierarchical AMOVAs</span></strong></p><span>doi:10.1371/journal.pone.0001012.t003</span></div>

<h4>Demography of mitochondrial haplogroups</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Because of the low number of goats in the F group, demographic parameters were not estimated for this group. The overall mismatch distribution shows a multi-modal distribution (<a href="#pone-0001012-g003">Figure 3</a>). The first peak with a maximum of 10 pairwise differences corresponds to the differences between haplotypes from the same group. Two other peaks with maxima at 27 and 39 pairwise differences correspond to differences between haplotypes from different groups. The distributions of within-groups and between-groups pairwise differences have a very thin overlap around 20 mismatches. The mismatch distribution analysis reveals a unimodal bell-shaped distribution of pairwise sequence differences for all haplogroups (<a href="#pone-0001012-g003">Figure 3</a>), except for B that is bimodal (data not shown). B1 and B2 are unimodal, and individuals from these sub- groups generally differ by 8 or 9 mismatches (always less than 14 mismatches). This unimodal pattern that is less clear for the D group, perhaps because of the low sample size (n = 13), would be coherent with recent demographic expansions. The time of expansion would differ according to the group, as suggested by the different means of pairwise distribution (<a href="#pone-0001012-g003">Figure 3</a>) and the estimations made under a model of pure demographic expansion <a href="#pone.0001012-Rogers1">[22]</a> (<a href="#pone-0001012-t004">Table 4</a>). However, the validity of the expansion model used for estimating the expansion time is only accepted for the A, C groups (SSD <em>P</em>-Values &lt;0.00001 and ≤0.05 respectively see <a href="#pone-0001012-t004">Table 4</a>). All groups have high growth rates indicating high demographic expansion (<a href="#pone-0001012-t004">Table 4</a>). The estimates differ according to the groups, but the overlapping of confidence intervals, as well as the different sample sizes, preclude further interpretation.</p>
<div class="figure" id="pone-0001012-g003"><div class="img"><a name="pone-0001012-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Mismatch distributions for mtDNA haplogroups of domestic goats.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>For the overall dataset, the distribution of pairwise differences were realized separately for comparisons between and within haplogroups.</p>
<span>doi:10.1371/journal.pone.0001012.g003</span></div><div class="figure" id="pone-0001012-t004"><div class="img"><a name="pone-0001012-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Estimation of demographic parameters from genetic data</span></strong></p><span>doi:10.1371/journal.pone.0001012.t004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>High mtDNA diversity in domestic goat</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The very high mt DNA diversity may partly result from a high mutation rate of the control region. Higher pedigree divergence rates than phylogenetic divergence rates have been shown for the control region in human <a href="#pone.0001012-Howell1">[23]</a> and other animals (e.g.,<a href="#pone.0001012-Denver1">[24]</a>, <a href="#pone.0001012-Lambert1">[25]</a>). This could explain that we observe a higher diversity than the one expected with the phylogenetic mutation rate estimated for Bovidae (i.e., 30.1 % of divergence per Myr on the total control region sequence based on the <em>Bos</em>/<em>Bison</em> divergence <a href="#pone.0001012-Bradley1">[26]</a>). Such high variability could also result from the selection of polymorphism but, to our knowledge, this has never been shown for the control region. Another explanation would be the capture of a large part of the diversity of the wild ancestor (i.e., the bezoar) during the domestication, with a large maternal effective population size. Testing this last hypothesis requires comparing the diversity of goats to that of the bezoar <a href="#pone.0001012-Naderi1">[27]</a>.</p>


<h4>Characteristics and nomenclature of mitochondrial haplogroups</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Five reliable mitochondrial haplogroups have previously been described in domestic goats <a href="#pone.0001012-Luikart1">[13]</a>–<a href="#pone.0001012-Joshi1">[15]</a>, <a href="#pone.0001012-Sardina1">[18]</a>. However, most of the previous studies were based on local samples and thus only considered a part of the whole haplotype variability. Therefore, it may be difficult to assess the pertinence of defining a new group on the base of few haplotypes. It would also be difficult to make the correspondence between several studies analyzing samples from different geographic origins. Our study can lead to a clear nomenclature of goat mitochondrial haplogroups, because we analyzed 2430 goats representing 1540 different haplotypes from all over Africa, Asia and Europe (946 new sequences mainly from the region of domestication and 1484 sequences from previous studies). We revealed the existence of 6 highly divergent groups. Five of them (A, B, C, D and F) have already been described, and one (G) is a new group. The two sequences that have been previously used to define the E group <a href="#pone.0001012-Joshi1">[15]</a> now fall within the A haplogroup. This is partly due to the finding of new haplotypes, which are intermediate between those from A and E used by Joshi et al. <a href="#pone.0001012-Joshi1">[15]</a>. Therefore, the E group cannot be considered as a mitochondrial haplogroup anymore. The B clade is composed of two groups (B1 and B2) that have previously been described as “sub-lineages” by Chen et al. <a href="#pone.0001012-Chen1">[16]</a>. We agree that the B1 and B2 are part of the same haplogroup because the genetic divergence between them (pairwise differences always lower than 14 mismatches) is lower than the divergence between all pairs of haplogroups (more than 20 mismatches). They must be considered as two subgroups because even with a low divergence they are supported by robust bootstrap values.</p>


<h4>Standard criteria for defining goat mitochondrial haplogroups</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>More generally, previous considerations point out the problem of defining groups and sub-groups. A new haplogroup is defined when it highly diverges from all other haplotypes. However, the haplogroups may change over time, as more and more haplotypes will be available. We faced this situation for the E haplogroup that is no valid any more. There is therefore a need for standard and easy-to-use criteria in order to assign new goat haplotypes to existing haplogroups or to define new haplogroups. A haplotype can be related to an existing group if it presents a moderate genetic divergence from this group. The difficulty may be to define what is a “moderate” divergence. It can be deduced from the distributions of pairwise sequence differences within and between haplogroups. For goats, almost all haplotypes from the same group differ by less than 20 mismatches (whatever the group) while haplotypes from different groups usually present more than 20 mismatches (<a href="#pone-0001012-g003">Figure 3</a>). This threshold value would give a quick and easy indication for almost all studied haplotypes. However, it may be inadequate for some haplotypes (about 1% in our study) because the two mismatch distributions overlap.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Given the increasing number of sequences available, analyzing new haplotypes together with all previously published sequences will be time consuming and will require huge computational resources. Moreover several programs cannot be used because the algorithm complexity does not allow managing such datasets. Especially when a few haplotypes from restricted localities are studied, their assignation to haplogroups should be quick and easy. For a first approach, an accurate solution would be to place the new different haplotypes in a phylogenetic tree containing sequences of reference representative of the diversity of <em>C. hircus</em> mitochondrial DNA. For this purpose we have selected 22 haplotypes representing the variability of the 6 present goat mitochondrial haplogroups (<a href="#pone-0001012-t005">Table 5</a> and <a href="#pone-0001012-g001">Figure 1B</a>).</p>
<div class="figure" id="pone-0001012-t005"><div class="img"><a name="pone-0001012-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001012" data-uri="info:doi/10.1371/journal.pone.0001012.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001012.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001012.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001012.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001012.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>The 22 reference individuals of the 6 domestic goat haplogroups</span></strong></p><span>doi:10.1371/journal.pone.0001012.t005</span></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Four of the 1540 haplotypes present a tandemly repeated sequence of 76 bp. Three individuals are from the A group (from Iran, Morocco and India) and one from the B1 sub-group (Malaysia). Such tandem repeats are common in vertebrate species <a href="#pone.0001012-Fumagalli1">[28]</a> and have already been found in the Bovidae family <a href="#pone.0001012-Hiendleder1">[29]</a>. They are attributed to slippage-mispairing events that are more likely to appear in regions where the polymerase activity is interrupted <a href="#pone.0001012-Fumagalli1">[28]</a>. This phenomenon corresponding to a single duplication event is found in a few individuals from different haplogroups, and has occurred more than once in the history of goats.</p>


<h4>Genetic structure of domestic goats</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Our results show that most of the genetic variation is found among goat haplogroups, with a weak phylogeographic structure. The strongly dominant A group (91 % of the goats) is distributed worldwide, and even if the other groups have more restricted distributions they still occupy large geographic areas (<a href="#pone-0001012-g002">Figure 2</a>). The F group is the exception, with three haplotypes restricted to a single locality (Sicily) that could have been brought along from recently captured wild goats. However, the sampling effort may still be insufficient to see the whole distribution of haplogroups other than A, because of their low frequency. The differences among geographic regions at the worldwide scale are low (about 12%) but significant. This is concordant with previous results showing a very weak phylogeographic structure of goats <a href="#pone.0001012-Luikart1">[13]</a> and sheep <a href="#pone.0001012-Meadows1">[30]</a>, <a href="#pone.0001012-Meadows2">[31]</a> compared to cattle <a href="#pone.0001012-Mannen1">[32]</a>, <a href="#pone.0001012-Freeman1">[33]</a>. The genetic differences among continental regions could partly result from the differential geographic distribution of mitochondrial haplogroups. However, there is still a low but significant genetic variation (3.5%) among region within groups, indicating regional differentiations of haplotypes. At the regional scale, the lack of geographic structure has also been reported in several places <a href="#pone.0001012-Chen1">[16]</a>, <a href="#pone.0001012-Amills1">[19]</a>, <a href="#pone.0001012-Pereira1">[21]</a> while a structure has been found in India <a href="#pone.0001012-Joshi1">[15]</a>. The weak phylogeographic structure found today in goats has been explained by a high mobility of this species in relation to human migration and commercial trade <a href="#pone.0001012-Bruford1">[12]</a>, <a href="#pone.0001012-Luikart1">[13]</a>, <a href="#pone.0001012-Fernndez1">[34]</a>. This mobility would have been higher than those of cattle due to their versatility in feeding habits and ability to live under extreme conditions <a href="#pone.0001012-CluttonBrock1">[1]</a>. However, the mixing of goat haplogroups could have existed before the worldwide translocation of goats. The presence of goats in Cyprus 10,000 years ago <a href="#pone.0001012-Vigne1">[35]</a> suggests that goats could have been translocated within the domestication area since the first domestication events. Moreover, we cannot exclude that the mt-haplogroups were already mixed in the wild ancestor before domestication. When considering the local scale, the genetic pattern of domestic goats also seems related to human history. For instance, the geographic structure found in Indian goats would have a common historical basis in the sequential migrations of human populations with different cultural and linguistic characteristics <a href="#pone.0001012-Joshi1">[15]</a>.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>However, the information given by mitochondrial markers is limited because it does not detect male-mediated gene flow and does not predict the nuclear genomic diversity <a href="#pone.0001012-Bruford1">[12]</a>. In particular, the breeds cannot be distinguished on the base of mtDNA <a href="#pone.0001012-Chen1">[16]</a>, <a href="#pone.0001012-Amills1">[19]</a>, <a href="#pone.0001012-Azor1">[20]</a> while nuclear markers show a genetic structure <a href="#pone.0001012-Tuon1">[36]</a>–<a href="#pone.0001012-Pariset1">[38]</a>. Our study confirmed that more than 77 % of the mtDNA variation is found within breeds and that nearly 25% of the breeds are composed of at least 2 haplogroups. This is in accordance with the recent fragmentation of local goat populations into discrete breeds about 200 years ago, under strong selection pressures on a few phenotypic traits <a href="#pone.0001012-Taberlet1">[39]</a>. This structure can be seen on nuclear markers linked to selected parts of the genome, but not on mitochondrial markers. Then, looking at the evolutionary history of breeds using mtDNA markers could lead to misinterpretation. For example, a breed composed of two mitochondrial haplogroups would have a bimodal mismatch distribution due to within- and between-breeds pairwise differences, and should not be interpreted in term of demographic history of the breed. Thus, fully understanding the evolutionary history of domestic goats would also require the use of nuclear markers.</p>


<h4>Demography of mitochondrial haplogroups</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The present structure of the genetic diversity retains the signature of past demographic events and helps reconstitute the evolutionary history <a href="#pone.0001012-Luikart2">[40]</a>. The estimation of demographic parameters remains difficult because of the difficulties of verifying the hypothesis of the models used, of estimating accurate initial parameters (e.g., absolute date of domestication) and sometimes because of low sample sizes. However, rough estimations from the present work and previous studies <a href="#pone.0001012-Luikart1">[13]</a>, <a href="#pone.0001012-Joshi1">[15]</a>, <a href="#pone.0001012-Chen1">[16]</a> are concordant and agree on the same scenario. All haplogroups had a recent demographic expansion corresponding roughly to the period when domestication took place about 10,000 years ago. It is difficult to give relative dates of expansion because of large confidence intervals, especially for D and G groups, but our results confirm that the expansions of B and C groups were more recent than that of A <a href="#pone.0001012-Luikart1">[13]</a>. Also, our results show that all groups had high growth rates, with a tendency for slower growth in B2 sub-group and C and G. A faster growth of A relative to C is in accordance with archaeozoological data: the genotyping of fossil goats showed that about 7000 years ago A and C were equally represented in Southern France <a href="#pone.0001012-Fernndez1">[34]</a> while A is strongly dominant in Southern Europe now.</p>


<h4>Limits of genetic data from domestic goats for reconstituting the history of domestication</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Divergence time between haplogroups has been estimated on adequate molecular markers (mainly cytochrome <em>b</em>) between 103,000 and 597,800 years <a href="#pone.0001012-Luikart1">[13]</a>–<a href="#pone.0001012-Chen1">[16]</a>. All these values are far greater than the domestication time, showing that most of goat genetic diversity existed before domestication, and that several haplogroups were domesticated in one or several events. However, the genetic data available for domestic goats does not permit furthering our understanding of the domestication process and identifying potential domestication centre(s). A higher genetic diversity would have been expected near the Fertile Crescent where the goat domestication took place according to archaeological data, and where extensive sampling has been done. But the haplotype diversity is similar all over the world (more than 80% of the countries with a haplotype diversity greater than 0.9), because of the high migration rates in domestic goats due to human migration and commercial trade.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Moreover, the presence of a possible ancestral haplotype in a particular area does not prove that this is a domestication centre, since many events could have occurred to mask the real history (e.g., coalescence or founder effects). For instance the domestication of a B sub-group in China supported by genetic data <a href="#pone.0001012-Chen1">[16]</a> is doubtful since the wild ancestor of the domestic goat (i.e. the bezoar <em>Capra aegagrus</em>) has credibly never been present in this area <a href="#pone.0001012-Pidancier1">[11]</a>, <a href="#pone.0001012-Shackleton1">[41]</a>. Overall, in order to fully understand the domestication of goats it is necessary to characterize the genetic diversity of wild goat species, and to establish the evolutionary relationships between wild and domesticated haplotypes.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sampling and DNA extraction</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Samples were collected from 946 individuals from 42 countries of the old world (See <a href="#pone-0001012-t002">Table 2</a>) from which 569 individuals were studied within the Econogene project (<a href="http://www.econogene.eu">www.econogene.eu</a>). Samples consisted of ear tissue preserved in ethanol 95% until extraction, or of blood collection. DNA was extracted from tissue using the Qiagen DNeasy tissue kit following the manufacturer's instructions, and from blood samples using QIAamp DNA blood kit.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>To have a good coverage of the goat breeds, the dataset was completed with 1484 sequences containing the <em>Capra hircus</em> HVI control region (450 to 1200 bp long) retrieved from GenBank (<a href="#pone-0001012-t002">Table 2</a>).</p>


<h4>DNA amplification and sequencing</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The HVI segment of the control region was sequenced for all blood and tissue DNA extracts. Using the primers CAP-F (5′-CGTGTATGCAAGTACATTAC-3′) and CAP-R (5′-CTGATTAGTCATTAGTCCATC-3′), we amplified a fragment of 598 bp (without primers) that corresponds to the positions 15,653 to 16,250 on the complete goat mitochondrial sequence of reference (<a href="#pone.0001012-Parma1">[42]</a>; accession number AF533441). PCR amplifications were conducted in a 25 µl volume with 2 mM MgCl<sub>2</sub>, 200 µM of each dNTP, 1 µM of each primer and 1 unit of AmpliTaq Gold Polymerase (Applied Biosystems). After a 10 min period at 95°C for polymerase activation, 35 cycles were run with the following steps: 95°C: 30 s, 55°C: 30 s, 72°C: 1 min. PCR products were purified using the Qiaquick PCR purification kit (Qiagen). 35 ng of purified DNA from this PCR product was used for sequencing with the CAP-F or CAP-R primer. Sequence reactions were performed for both DNA strands by using the ABI PRISM Dye Terminator Cycle Sequencing Reaction Kit (Applied Biosystems) in a 20 µl volume with 2 µM of each primer. 25 cycles were run with the following steps 96°C: 30 s, 55°C: 30 s, 60°C: 4 min. Excess dye terminators were removed by spin-column purification and the products were electrophorezed on an ΑΒΙ 3700 PRISM DNA sequencer (Applied Biosystems) using the POP 7 polymer.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Sequences were edited for correction with the SeqScape v2.5 software (Applied Biosystems). All sequences were deposited in GenBank (Accession Numbers EF617601- EF618546, <a href="#pone-0001012-t002">Table 2</a> and <a href="#pone.0001012.s001">Table S1</a>).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Sequences from GenBank and from our dataset were aligned with Mega v3.1 <a href="#pone.0001012-Kumar1">[43]</a>, and then adjusted by eye. For further analyses, we only kept the region used by Luikart et al. <a href="#pone.0001012-Luikart1">[13]</a> because this is the part of the sequence available for most of the GenBank records, and also the most informative one. This region is 481 bp long and corresponds to the positions 15,707 to 16,187 on the <em>Capra hircus</em> reference sequence (mtDNA complete sequence of <em>C. hircus</em>, Accession number AF533441 <a href="#pone.0001012-Parma1">[42]</a>). According to the insertion/deletion events, the analyzed sequences ranged from 481 to 558 bp. For Indian goats a shorter fragment of 453 bp has been sequenced <a href="#pone.0001012-Joshi1">[15]</a> and the 28 missing nucleotides were treated as missing data. The alignment of the 2430 sequences used in this study is provided as supplementary information (<a href="#pone.0001012.s001">Table S1</a>).</p>


<h4>Data analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The substitution model used for the HVI region was the Kimura 2-parameters model, as previously used on several subsets of the present dataset (e.g., <a href="#pone.0001012-Luikart1">[13]</a>, <a href="#pone.0001012-Joshi1">[15]</a>). The heterogeneity in substitution rates among nucleotide sites was modelled by a gamma distribution. The alpha parameter was estimated as the mean of 10 estimations by a maximum-likelihood method under the Kimura 2-parameters model using PAML v 2.0.2 <a href="#pone.0001012-Yang1">[44]</a>. Each estimation was based on the analysis of 1000 individuals randomly chosen in the dataset of 2430 individuals. The estimated value (alpha = 0.28) was similar to that estimated for the same region on a smaller sample of domestic and wild goats by Luikart et al. <a href="#pone.0001012-Luikart1">[13]</a>. These settings were used for further phylogenetic reconstruction and analysis of genetic diversity. We used 1484 published sequences for checking the validity of the halpogroups previously defined (see <a href="#pone-0001012-t002">Table 2</a> and <a href="#pone.0001012.s001">Table S1</a> for references and GenBank accession numbers).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Given the very high number of sequences analyzed, the phylogenetic tree was constructed using the Neighbor-joining method using PAUP* v 4.0 <a href="#pone.0001012-Swofford1">[45]</a>, with 1000 bootstraps for measuring branch robustness. The ARLEQUIN v 3.0 software <a href="#pone.0001012-Excoffier1">[46]</a> was used for estimating haplotype and nucleotide diversity, for analyzing mismatch distribution within mitochondrial haplogroups, and for estimating the parameters of demographic expansion. Four individuals that showed a 76 bp insertion were discarded for mismatch analyses and the analyses were thus performed on 481 bp long sequences. The expansion time was estimated under a model of pure demographic expansion <a href="#pone.0001012-Rogers1">[22]</a> with parameters set to default values in ARLEQUIN 3.0. The parameter of demographic expansion τ was estimated according to the method of Schneider and Excoffier <a href="#pone.0001012-Schneider1">[47]</a>. The validity of the expansion model was tested using the sum of square deviations (SSD) between the observed and expected mismatches <a href="#pone.0001012-Schneider1">[47]</a>. Growth rates of mitochondrial haplogroups were estimated with Lamarc v2.2 <a href="#pone.0001012-Kuhner1">[48]</a> using a bayesian framework allowing migrations across haplogroups (with a maximum of 10000 migration events, default priors used for migration rates estimation). The estimation of growth rates was done with linear prior (upper bound of 1000 and lower bound of −500), 10 initial chains (500 samples, sampling interval of 20 and burn-in period of 1000) and 2 final chains (10000 samples, sampling interval of 20 and burn-in period of 1000).</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>In order to test the geographic structure of the mtDNA haplotype diversity, the goat distribution has been partitioned in 7 geographic regions (Northern Europe, Southern Europe, Northern Africa, Sub-Saharan Africa, Middle East, Western Asia and Eastern Asia, see <a href="#pone-0001012-t002">Table 2</a>). Two hierarchical AMOVA were performed using ARLEQUIN v3.0 to test the partition of the genetic variance among haplogroups and among continents within haplogroups, as well as among continents and among breeds within continents. This second AMOVA was performed on the 1602 goats for which the breeds were known.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001012"></div><a name="pone.0001012.s001" id="pone.0001012.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001012.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Alignements of the 2430 control region sequences of domestic goat. The code for geographic regions are defined in <a href="#pone-0001012-t002">Table 2</a>. Missing data are coded as ‘?’.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001012.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.24 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Prof. Oscar Gaggiotti for fruitful discussions and constructive comments, Cécile Albert for help with figures, and Jessica Scriven and Alison Cleary for help with the English. We are grateful to Ebrahim Ghaderi and Prof. Mahamoud Abo-Shehada for their help for collecting samples in Iran and Jordan. We also thank two anonymous reviewers for constructive and stimulating comments.</p>
<a id="article1.back1.ack1.p2" name="article1.back1.ack1.p2"></a><p><strong>Institution at which research was done:</strong> Laboratoire d'Ecologie Alpine, Université Joseph Fourier, Grenoble, France.</p>
<a id="article1.back1.ack1.p3" name="article1.back1.ack1.p3"></a><p>Members of the Econogene Consortium (<a href="http://www.econogene.eu">www.econogene.eu</a>):</p>
<a id="article1.back1.ack1.p4" name="article1.back1.ack1.p4"></a><p>Abo-Shehada Mahamoud, Parasitology Resesarch Laboratory, Faculty of Veterinary Medicine, Jordan University of Science and Technology, Ibid, Jordan,</p>
<a id="article1.back1.ack1.p5" name="article1.back1.ack1.p5"></a><p>Ajmone Marsan Paolo, Istituto di Zootecnica, Università Cattolica del S. Cuore, Piacenza, Italy,</p>
<a id="article1.back1.ack1.p6" name="article1.back1.ack1.p6"></a><p>Al Tarrayrah Jamil, Aristotle University of Thessaloniki, Fac. of Agriculture, Dept. of Animal Production, Thessaloniki, Greece,</p>
<a id="article1.back1.ack1.p7" name="article1.back1.ack1.p7"></a><p>Angiolillo Antonella, Dipartimento di Scienze Animali Vegetali e dell'Ambiente, Università del Molise, Campobasso, Italy,</p>
<a id="article1.back1.ack1.p8" name="article1.back1.ack1.p8"></a><p>Baret Philip, Department of Animal Genetics, Université cath. Louvain, Louvain-la-Neuve, Belgium,</p>
<a id="article1.back1.ack1.p9" name="article1.back1.ack1.p9"></a><p>Baumung Roswitha, University of Natural Resources and Applied Life Sciences, Vienna, Vienna, Austria,</p>
<a id="article1.back1.ack1.p10" name="article1.back1.ack1.p10"></a><p>Beja-Pereira Albano, Laboratoire de Biologie des Populations d'Altitude, University Joseph Fourier et Centre National de la Recherche Scientifique, Grenoble, France,</p>
<a id="article1.back1.ack1.p11" name="article1.back1.ack1.p11"></a><p>Bertaglia Marco, Department of Food Economics and Consumption Studies, University of Kiel, Kiel, Germany,</p>
<a id="article1.back1.ack1.p12" name="article1.back1.ack1.p12"></a><p>Bordonaro Salvatore, Dipartimento di Scienze Agronomiche e delle Produzioni Animali, Università degli Studi di Catania, Catania, Italy,</p>
<a id="article1.back1.ack1.p13" name="article1.back1.ack1.p13"></a><p>Bruford Mike, School of Biosciences, Cardiff University, Cardiff, UK,</p>
<a id="article1.back1.ack1.p14" name="article1.back1.ack1.p14"></a><p>Caloz Régis, Ecole Polytechnique Fédérale de Lausanne (EPFL), Laboratoire de Systèmes d'Information Géographique (LASIG), Lausanne, Switzerland,</p>
<a id="article1.back1.ack1.p15" name="article1.back1.ack1.p15"></a><p>Canali Gabriele, Istituto di Economia Agro-Alimentare, Università Cattolica del S. Cuore, Piacenza, Italy,</p>
<a id="article1.back1.ack1.p16" name="article1.back1.ack1.p16"></a><p>Canon Javier, Universidad Computense de Madrid, Madrid, Spain,</p>
<a id="article1.back1.ack1.p17" name="article1.back1.ack1.p17"></a><p>Cappuccio Irene, Dipartimento di Produzioni Animali, Università della Tuscia, Viterbo, Italy,</p>
<a id="article1.back1.ack1.p18" name="article1.back1.ack1.p18"></a><p>Carta Antonello, Istituto Zootecnico e Caseario per la Sardegna, Sassari, Italy,</p>
<a id="article1.back1.ack1.p19" name="article1.back1.ack1.p19"></a><p>Cicogna Mario, Istituto di Zootecnia Generale, Università degli Studi di Milano, Milano, Italy,</p>
<a id="article1.back1.ack1.p20" name="article1.back1.ack1.p20"></a><p>Crepaldi Paola, Istituto di Zootecnia Generale, Università degli Studi di Milano, Milano, Italy,</p>
<a id="article1.back1.ack1.p21" name="article1.back1.ack1.p21"></a><p>Dalamitra Stella, Biological Sciences, University of East Anglia, Norwich, UK,</p>
<a id="article1.back1.ack1.p22" name="article1.back1.ack1.p22"></a><p>Daniela Krugmann, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p23" name="article1.back1.ack1.p23"></a><p>Dobi Petrit, Department of Animal Production, Faculty of Agriculture, Tirana, Albania,</p>
<a id="article1.back1.ack1.p24" name="article1.back1.ack1.p24"></a><p>Dominik Popielarczyk, Department of Sheep and Goat Breeding, Warsaw Agricultural University, Warsaw, Poland,</p>
<a id="article1.back1.ack1.p25" name="article1.back1.ack1.p25"></a><p>Dunner Susana, Universidad Computense de Madrid, Madrid, Spain,</p>
<a id="article1.back1.ack1.p26" name="article1.back1.ack1.p26"></a><p>D'Urso Giuseppe, Dipartimento di Scienze Agronomiche e delle Produzioni Animali, Università degli Studi di Catania, Catania, Italy,</p>
<a id="article1.back1.ack1.p27" name="article1.back1.ack1.p27"></a><p>El-Barody M. A. A., Animal Production Department, Faculty of Agriculture, Minia University, Minia, Egypt,</p>
<a id="article1.back1.ack1.p28" name="article1.back1.ack1.p28"></a><p>England Phillip, Laboratoire de Biologie des Populations d'Altitude, University Joseph Fourier et CNRS, Grenoble, France,</p>
<a id="article1.back1.ack1.p29" name="article1.back1.ack1.p29"></a><p>Erhardt Georg, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p30" name="article1.back1.ack1.p30"></a><p>Ertugrul Okan, Veterinary Faculty, Ankara University, Ankara, Turkey,</p>
<a id="article1.back1.ack1.p31" name="article1.back1.ack1.p31"></a><p>Eva-Maria Prinzenberg, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p32" name="article1.back1.ack1.p32"></a><p>Eveline Ibeagha-Awemu, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p33" name="article1.back1.ack1.p33"></a><p>Ewa Strzelec, Department of Sheep and Goat Breeding, Warsaw Agricultural University, Warsaw, Poland,</p>
<a id="article1.back1.ack1.p34" name="article1.back1.ack1.p34"></a><p>Fadlaoui Aziz, Department of Animal Genetics, Cath. University of Louvain, Louvain-la-Neuve, Belgium,</p>
<a id="article1.back1.ack1.p35" name="article1.back1.ack1.p35"></a><p>Fornarelli Francesca, Istituto di Zootecnia Generale, Università degli Studi di Milano, Milano, Italy,</p>
<a id="article1.back1.ack1.p36" name="article1.back1.ack1.p36"></a><p>Garcia David, Universidad Computense de Madrid, Madrid, Spain,</p>
<a id="article1.back1.ack1.p37" name="article1.back1.ack1.p37"></a><p>Georgoudis Andreas, Aristotle University of Thessaloniki, Fac. of Agriculture, Dept. of Animal Production, Thessaloniki, Greece,</p>
<a id="article1.back1.ack1.p38" name="article1.back1.ack1.p38"></a><p>Gesine Lühken, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p39" name="article1.back1.ack1.p39"></a><p>Giovenzana Stefano, Istituto di Zootecnia Generale, Università degli Studi di Milano, Milano, Italy,</p>
<a id="article1.back1.ack1.p40" name="article1.back1.ack1.p40"></a><p>Gutscher Katja, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p41" name="article1.back1.ack1.p41"></a><p>Hewitt Godfrey, Biological Sciences, University of East Anglia, Norwich, UK,</p>
<a id="article1.back1.ack1.p42" name="article1.back1.ack1.p42"></a><p>Hoda Anila, Department of Animal Production, Faculty of Agriculture, Tirana, Albania,</p>
<a id="article1.back1.ack1.p43" name="article1.back1.ack1.p43"></a><p>Horst Brandt, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p44" name="article1.back1.ack1.p44"></a><p>Istvan Anton, Research Institute for Animal Breedeing and Nutrition, Department of Genetics, Herceghalom, Hungary,</p>
<a id="article1.back1.ack1.p45" name="article1.back1.ack1.p45"></a><p>Juma Gabriela, School of Biosciences, Cardiff University, Cardiff, UK,</p>
<a id="article1.back1.ack1.p46" name="article1.back1.ack1.p46"></a><p>Joost Stéphane, Ecole Polytechnique Fédérale de Lausanne (EPFL), Laboratoire de Systèmes d'Information Géographique (LASIG), Lausanne, Switzerland,</p>
<a id="article1.back1.ack1.p47" name="article1.back1.ack1.p47"></a><p>Jones Sam, The Sheep Trust, Biology Area-8, University of York, York, England,</p>
<a id="article1.back1.ack1.p48" name="article1.back1.ack1.p48"></a><p>Karetsou Katerina, Aristotle University of Thessaloniki, Fac. of Agriculture, Dept. of Animal Production, Thessaloniki, Greece,</p>
<a id="article1.back1.ack1.p49" name="article1.back1.ack1.p49"></a><p>Kliambas Georgios, Aristotle University of Thessaloniki, Fac. of Agriculture, Dept. of Animal Production, Thessaloniki, Greece,</p>
<a id="article1.back1.ack1.p50" name="article1.back1.ack1.p50"></a><p>Koban Evren, Department of Biology, Middle East Technical University, Ankara, Turkey,</p>
<a id="article1.back1.ack1.p51" name="article1.back1.ack1.p51"></a><p>Kutita Olga, Aristotle University of Thessaloniki, Fac. of Agriculture, Dept. of Animal Production, Thessaloniki, Greece,</p>
<a id="article1.back1.ack1.p52" name="article1.back1.ack1.p52"></a><p>Lazlo Fesus, Research Institute for Animal Breedeing and Nutrition, Department of Genetics, Herceghalom, Hungary,</p>
<a id="article1.back1.ack1.p53" name="article1.back1.ack1.p53"></a><p>Lenstra Johannes A., Faculty of Veterinary Medicine, Utrecht University, Utrecht, Netherlands,</p>
<a id="article1.back1.ack1.p54" name="article1.back1.ack1.p54"></a><p>Ligda Christina, National Agricultural Research Foundation, Thessaloniki, Greece,</p>
<a id="article1.back1.ack1.p55" name="article1.back1.ack1.p55"></a><p>Lipsky Shirin, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p56" name="article1.back1.ack1.p56"></a><p>Luikart Gordon, Laboratoire de Biologie des Populations d'Altitude, University Joseph Fourier et CNRS, Grenoble, France,</p>
<a id="article1.back1.ack1.p57" name="article1.back1.ack1.p57"></a><p>Marie-Louise Glowatzki, Institut für Genetik Ernährung und Haltung von Haustieren, Universität Bern, Bern, Switzerland,</p>
<a id="article1.back1.ack1.p58" name="article1.back1.ack1.p58"></a><p>Marilli Marta, Istituto di Zootecnia Generale, Università degli Studi di Milano, Milano, Italy,</p>
<a id="article1.back1.ack1.p59" name="article1.back1.ack1.p59"></a><p>Marletta Donata, Dipartimento di Scienze Agronomiche e delle Produzioni Animali, Università degli Studi di Catania, Catania, Italy,</p>
<a id="article1.back1.ack1.p60" name="article1.back1.ack1.p60"></a><p>Milanesi Elisabetta, Istituto di Zootecnica, Università Cattolica del S. Cuore, Piacenza, Italy,</p>
<a id="article1.back1.ack1.p61" name="article1.back1.ack1.p61"></a><p>Negrini Riccardo, Istituto di Zootecnica, Università Cattolica del S. Cuore, Piacenza, Italy,</p>
<a id="article1.back1.ack1.p62" name="article1.back1.ack1.p62"></a><p>Nijman, Isaäc J., Faculty of Veterinary Medicine, Utrecht University, Utrecht, Netherlands,</p>
<a id="article1.back1.ack1.p63" name="article1.back1.ack1.p63"></a><p>Obexer-Ruff Gabriela, Institut für Genetik Ernährung und Haltung von Haustieren, Universität bern, Bern, Switzerland,</p>
<a id="article1.back1.ack1.p64" name="article1.back1.ack1.p64"></a><p>Papachristoforou Christos , Agricultural Research Institute, Nicosia, Cyprus,</p>
<a id="article1.back1.ack1.p65" name="article1.back1.ack1.p65"></a><p>Pariset Lorraine, Dipartimento di Produzioni Animali, Università della Tuscia, Viterbo, Italy,</p>
<a id="article1.back1.ack1.p66" name="article1.back1.ack1.p66"></a><p>Pellecchia Marco, Istituto di Zootecnica, Università Cattolica del S. Cuore, Piacenza, Italy,</p>
<a id="article1.back1.ack1.p67" name="article1.back1.ack1.p67"></a><p>Peter Christina, Institut für Tierzucht und Haustiergenetik, Justus-Liebig University of Giessen, Giessen, Germany,</p>
<a id="article1.back1.ack1.p68" name="article1.back1.ack1.p68"></a><p>Perez Trinidad, School of Biosciences, Cardiff University, Cardiff, UK,</p>
<a id="article1.back1.ack1.p69" name="article1.back1.ack1.p69"></a><p>Pietrolà Emilio, Dipartimento di Scienze Animali Vegetali e dell'Ambiente, Università del Molise, Campobasso, Italy,</p>
<a id="article1.back1.ack1.p70" name="article1.back1.ack1.p70"></a><p>Pilla Fabio, Dipartimento di Scienze Animali Vegetali e dell'Ambiente, Università del Molise, Campobasso, Italy,</p>
<a id="article1.back1.ack1.p71" name="article1.back1.ack1.p71"></a><p>Roman Niznikowski, Department of Sheep and Goat Breeding, Warsaw Agricultural University, Warsaw, Poland,</p>
<a id="article1.back1.ack1.p72" name="article1.back1.ack1.p72"></a><p>Roosen Jutta, Department of Food Economics and Consumption Studies, University of Kiel, Kiel, Germany,</p>
<a id="article1.back1.ack1.p73" name="article1.back1.ack1.p73"></a><p>Scarpa Riccardo, Environment Department, University of York, York, UK,</p>
<a id="article1.back1.ack1.p74" name="article1.back1.ack1.p74"></a><p>Sechi Tiziana, Istituto Zootecnico e Caseario per la Sardegna, Sassari, Italy,</p>
<a id="article1.back1.ack1.p75" name="article1.back1.ack1.p75"></a><p>Taberlet Pierre, Laboratoire d'Ecologie Alpine (LECA), Université Joseph Fourier, Grenoble, France,</p>
<a id="article1.back1.ack1.p76" name="article1.back1.ack1.p76"></a><p>Taylor Martin, Biological Sciences, University of East Anglia, Norwich, UK,</p>
<a id="article1.back1.ack1.p77" name="article1.back1.ack1.p77"></a><p>Togan Inci, Department of Biology, Middle East Technical University, Ankara, Turkey,</p>
<a id="article1.back1.ack1.p78" name="article1.back1.ack1.p78"></a><p>Trommetter Michel, Institut de recherche agronomique, Unité d'Économie et Sociologie rurales, Grenoble, France,</p>
<a id="article1.back1.ack1.p79" name="article1.back1.ack1.p79"></a><p>Valentini Alessio, Dipartimento di Produzioni Animali, Università della Tuscia, Viterbo, Italy,</p>
<a id="article1.back1.ack1.p80" name="article1.back1.ack1.p80"></a><p>Van Cann, Lisette M., Faculty of Veterinary Medicine, Utrecht University, Utrecht, Netherlands,</p>
<a id="article1.back1.ack1.p81" name="article1.back1.ack1.p81"></a><p>Vlaic Augustin, Department of Animal Genetics, Faculty of Zootechnics, University of Cluj-Napoca, Cluj-Napoca, Romania,</p>
<a id="article1.back1.ack1.p82" name="article1.back1.ack1.p82"></a><p>Wiskin Louise, Biological Sciences, University of East Anglia, Norwich, UK,</p>
<a id="article1.back1.ack1.p83" name="article1.back1.ack1.p83"></a><p>Zundel Stéphanie , Laboratoire d'Ecologie Alpine (LECA), Université Joseph Fourier, Grenoble, France,</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PT FP SN HR SR. Performed the experiments: SN HR SZ. Analyzed the data: FP SN HR. Contributed reagents/materials/analysis tools: SN HR SZ SR HN ME OE. Wrote the paper: PT FP SN HR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001012-CluttonBrock1" id="pone.0001012-CluttonBrock1"></a>Clutton-Brock J (1999) Domesticated animals from early times. The Natural History Museum: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001012-Porter1" id="pone.0001012-Porter1"></a>Porter V (1996) Goats of the world. Ipswich, Uk: Farming Press.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001012-Pringle1" id="pone.0001012-Pringle1"></a>Pringle H (1998) Reading the signs of ancient animal domestication. Science  282: 1448–1450.  <ul class="find" data-citedArticleID="1003448" data-doi="10.1126/science.282.5393.1448"><li><a href="http://dx.doi.org/10.1126/science.282.5393.1448" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reading+the+signs+of+ancient+animal+domestication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reading+the+signs+of+ancient+animal+domestication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001012-Zeder1" id="pone.0001012-Zeder1"></a>Zeder M, Emshwiller E, Smith B, Bradley D (2006) Documenting domestication: the intersection of genetics and archaeology. Trends Genet  22: 139–155.  <ul class="find" data-citedArticleID="1003472" data-doi="10.1016/j.tig.2006.01.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.01.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Documenting+domestication%3A+the+intersection+of+genetics+and+archaeology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Documenting+domestication%3A+the+intersection+of+genetics+and+archaeology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001012-MacHugh1" id="pone.0001012-MacHugh1"></a>MacHugh D, Bradley D (2001) Livestock genetic origins: goats buck the trend. Proc Natl Acad Sci USA  98: 5382–5384.  <ul class="find" data-citedArticleID="1003422" data-doi="10.1073/pnas.111163198"><li><a href="http://dx.doi.org/10.1073/pnas.111163198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Livestock+genetic+origins%3A+goats+buck+the+trend." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Livestock+genetic+origins%3A+goats+buck+the+trend.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001012-Meadow1" id="pone.0001012-Meadow1"></a>Meadow RH (1996) The Origins and Spread of Agriculture and Pastoralism in northwestern South Asia. In: Harris DR. The Origins and Spread of Agriculture and Pastoralism in Eurasia. London: UCL Press. pp. 390–412.  <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0001012-Zeder2" id="pone.0001012-Zeder2"></a>Zeder M, Hesse B (2000) The initial domestication of goats (<em>Capra hircus</em>) in the Zagros Mountains 10,000 years ago. Sciences  287: 2254–2257.  <ul class="find" data-citedArticleID="1003474" data-doi="10.1126/science.287.5461.2254"><li><a href="http://dx.doi.org/10.1126/science.287.5461.2254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+initial+domestication+of+goats+%28Capra+hircus%29+in+the+Zagros+Mountains+10%2C000+years+ago." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+initial+domestication+of+goats+%28Capra+hircus%29+in+the+Zagros+Mountains+10%2C000+years+ago.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001012-Zeder3" id="pone.0001012-Zeder3"></a>Zeder M, Vigne JD, Helmer D (2005)  A view from the Zagros: new perspectives on livestock domestication in the Fertile Crescent. In: Vigne JD, Peters J, Helmer D, editors. The first steps of animal domestication: Oxbow Books. pp. 125–146.  <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0001012-Takada1" id="pone.0001012-Takada1"></a>Takada T, Kikkawa Y, Yonekawa H, Kawakami S, Amano T (1997) Bezoar (<em>Capra aegagrus</em>) is a matriarchal candidate for ancestor of domestic goat (<em>Capra hircus</em>): evidence from the mitochondrial DNA diversity. Biochem Genet  35(9–10): 315–326.  <ul class="find" data-citedArticleID="1003464"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bezoar+%28Capra+aegagrus%29+is+a+matriarchal+candidate+for+ancestor+of+domestic+goat+%28Capra+hircus%29%3A+evidence+from+the+mitochondrial+DNA+diversity.&amp;auth=&amp;atitle=Bezoar+%28Capra+aegagrus%29+is+a+matriarchal+candidate+for+ancestor+of+domestic+goat+%28Capra+hircus%29%3A+evidence+from+the+mitochondrial+DNA+diversity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bezoar+%28Capra+aegagrus%29+is+a+matriarchal+candidate+for+ancestor+of+domestic+goat+%28Capra+hircus%29%3A+evidence+from+the+mitochondrial+DNA+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bezoar+%28Capra+aegagrus%29+is+a+matriarchal+candidate+for+ancestor+of+domestic+goat+%28Capra+hircus%29%3A+evidence+from+the+mitochondrial+DNA+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001012-Manceau1" id="pone.0001012-Manceau1"></a>Manceau V, Despres L, Bouvet J, Taberlet P (1999) Systematics of the genus <em>Capra</em> inferred from mitochondrial DNA sequence data. Mol Phylogenet Evol  13(3): 504–510.  <ul class="find" data-citedArticleID="1003424" data-doi="10.1006/mpev.1999.0688"><li><a href="http://dx.doi.org/10.1006/mpev.1999.0688" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematics+of+the+genus+Capra+inferred+from+mitochondrial+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematics+of+the+genus+Capra+inferred+from+mitochondrial+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001012-Pidancier1" id="pone.0001012-Pidancier1"></a>Pidancier N, Jordan S, Luikart G, Taberlet P (2006) Evolutionary history of the genus <em>Capra</em> (Mammalia, Artiodactyla): discordance between mitochondrial DNA and Y-chromosome phylogenies. Mol Phylogenet Evol  40: 739–749.  <ul class="find" data-citedArticleID="1003444" data-doi="10.1016/j.ympev.2006.04.002"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+history+of+the+genus+Capra+%28Mammalia%2C+Artiodactyla%29%3A+discordance+between+mitochondrial+DNA+and+Y-chromosome+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+history+of+the+genus+Capra+%28Mammalia%2C+Artiodactyla%29%3A+discordance+between+mitochondrial+DNA+and+Y-chromosome+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001012-Bruford1" id="pone.0001012-Bruford1"></a>Bruford M, Bradley D, Luikart G (2003) DNA markers reveal the complexity of livestock domestication. Nat Rev Genet  3: 900–910.  <ul class="find" data-citedArticleID="1003388" data-doi="10.1038/nrg1203"><li><a href="http://dx.doi.org/10.1038/nrg1203" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+markers+reveal+the+complexity+of+livestock+domestication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+markers+reveal+the+complexity+of+livestock+domestication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001012-Luikart1" id="pone.0001012-Luikart1"></a>Luikart G, Gielly L, Excoffier L, Vigne JD, Bouvet J, et al.  (2001) Multiple maternal origins and weak phylogeographic structure in domestic goats. Proc Natl Acad Sci USA  98: 5927–5932.  <ul class="find" data-citedArticleID="1003418" data-doi="10.1073/pnas.091591198"><li><a href="http://dx.doi.org/10.1073/pnas.091591198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+maternal+origins+and+weak+phylogeographic+structure+in+domestic+goats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+maternal+origins+and+weak+phylogeographic+structure+in+domestic+goats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001012-Sultana1" id="pone.0001012-Sultana1"></a>Sultana S, Mannen H, Tsuji S (2003) Mitochondrial DNA diversity of Pakistani goats. Anim Genet  34(6): 417–421.  <ul class="find" data-citedArticleID="1003458" data-doi="10.1046/j.0268-9146.2003.01040.x"><li><a href="http://dx.doi.org/10.1046/j.0268-9146.2003.01040.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+diversity+of+Pakistani+goats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+diversity+of+Pakistani+goats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001012-Joshi1" id="pone.0001012-Joshi1"></a>Joshi MB, Rout PK, Mandal AK, Tyler-Smith C, Singh L, et al.  (2004) Phylogeography and origin of Indian domestic goats. Mol Biol Evol  21(3): 454–462.  <ul class="find" data-citedArticleID="1003410"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogeography+and+origin+of+Indian+domestic+goats.&amp;auth=&amp;atitle=Phylogeography+and+origin+of+Indian+domestic+goats." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+and+origin+of+Indian+domestic+goats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+and+origin+of+Indian+domestic+goats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001012-Chen1" id="pone.0001012-Chen1"></a>Chen SY, Su YH, Wu SF, Sha T, Zhang YP (2005) Mitochondrial diversity and phylogeographic structure of Chinese domestic goats. Mol Phylogenet Evol  37(3): 804–814.  <ul class="find" data-citedArticleID="1003392" data-doi="10.1016/j.ympev.2005.06.014"><li><a href="http://dx.doi.org/10.1016/j.ympev.2005.06.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+diversity+and+phylogeographic+structure+of+Chinese+domestic+goats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+diversity+and+phylogeographic+structure+of+Chinese+domestic+goats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001012-Odahara1" id="pone.0001012-Odahara1"></a>Odahara S, Chung H, Choi S, Yu S, Sasazaki S, et al.  (2006) Mitochondrial DNA diversity of Korean native goats. Asian Australas J Anim Sci  19: 482–485.  <ul class="find" data-citedArticleID="1003436"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mitochondrial+DNA+diversity+of+Korean+native+goats.&amp;auth=&amp;atitle=Mitochondrial+DNA+diversity+of+Korean+native+goats." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+DNA+diversity+of+Korean+native+goats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+DNA+diversity+of+Korean+native+goats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001012-Sardina1" id="pone.0001012-Sardina1"></a>Sardina MT, Ballester M, Marmi J, Finocchiaro R, van Kaam JB, et al.  (2006) Phylogenetic analysis of Sicilian goats reveals a new mtDNA lineage. Anim Genet  37(4): 376–378.  <ul class="find" data-citedArticleID="1003452" data-doi="10.1111/j.1365-2052.2006.01451.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2052.2006.01451.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+Sicilian+goats+reveals+a+new+mtDNA+lineage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+Sicilian+goats+reveals+a+new+mtDNA+lineage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001012-Amills1" id="pone.0001012-Amills1"></a>Amills M, Capote J, Tomas A, Kelly L, Obexer-Ruff G, et al.  (2004) Strong phylogeographic relationships among three goat breeds from the Canary Islands. J Dairy Res  71(3): 257–262.  <ul class="find" data-citedArticleID="1003382" data-doi="10.1017/s0022029904000342"><li><a href="http://dx.doi.org/10.1017/s0022029904000342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strong+phylogeographic+relationships+among+three+goat+breeds+from+the+Canary+Islands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strong+phylogeographic+relationships+among+three+goat+breeds+from+the+Canary+Islands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001012-Azor1" id="pone.0001012-Azor1"></a>Azor PJ, Monteagudo LV, Luque M, Tejedor MT, Rodero E, et al.  (2005) Phylogenetic relationships among Spanish goats breeds. Anim Genet  36(5): 423–425.  <ul class="find" data-citedArticleID="1003384" data-doi="10.1111/j.1365-2052.2005.01327.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2052.2005.01327.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+relationships+among+Spanish+goats+breeds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+relationships+among+Spanish+goats+breeds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001012-Pereira1" id="pone.0001012-Pereira1"></a>Pereira F, Pereira L, Van Asch B, Bradley D, Amorim A (2005) The mtDNA catalogue of all Portuguese autochthonous goat (<em>Capra hircus</em>) breeds: high diversity of female lineages at the western fringe of European distribution. Mol Ecol  14: 2313–2318.  <ul class="find" data-citedArticleID="1003442" data-doi="10.1111/j.1365-294x.2005.02594.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2005.02594.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mtDNA+catalogue+of+all+Portuguese+autochthonous+goat+%28Capra+hircus%29+breeds%3A+high+diversity+of+female+lineages+at+the+western+fringe+of+European+distribution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mtDNA+catalogue+of+all+Portuguese+autochthonous+goat+%28Capra+hircus%29+breeds%3A+high+diversity+of+female+lineages+at+the+western+fringe+of+European+distribution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001012-Rogers1" id="pone.0001012-Rogers1"></a>Rogers A, Harpending H (1992) Population growth makes waves in the distribution of pairwise genetic differences. Mol Biol Evol  9: 552–569.  <ul class="find" data-citedArticleID="1003450"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Population+growth+makes+waves+in+the+distribution+of+pairwise+genetic+differences.&amp;auth=&amp;atitle=Population+growth+makes+waves+in+the+distribution+of+pairwise+genetic+differences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+growth+makes+waves+in+the+distribution+of+pairwise+genetic+differences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+growth+makes+waves+in+the+distribution+of+pairwise+genetic+differences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001012-Howell1" id="pone.0001012-Howell1"></a>Howell N, Smejkal CB, Mackey DA, Chinnery PF, Turnbull DM, et al.  (2003) The Pedigree Rate of Sequence Divergence in the Human Mitochondrial Genome: There Is a Difference Between Phylogenetic and Pedigree Rates. Am J Hum Genet  72: 659–670.  <ul class="find" data-citedArticleID="1003408" data-doi="10.1086/368264"><li><a href="http://dx.doi.org/10.1086/368264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pedigree+Rate+of+Sequence+Divergence+in+the+Human+Mitochondrial+Genome%3A+There+Is+a+Difference+Between+Phylogenetic+and+Pedigree+Rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pedigree+Rate+of+Sequence+Divergence+in+the+Human+Mitochondrial+Genome%3A+There+Is+a+Difference+Between+Phylogenetic+and+Pedigree+Rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001012-Denver1" id="pone.0001012-Denver1"></a>Denver DR, Morris K, Lynch M, Vassilieva LL, Thomas WK (2000) High Direct Estimate of the Mutation Rate in the Mitochondrial Genome of <em>Caenorhabditis elegans</em>. Science  289: 2342.  <ul class="find" data-citedArticleID="1003396" data-doi="10.1126/science.289.5488.2342"><li><a href="http://dx.doi.org/10.1126/science.289.5488.2342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+Direct+Estimate+of+the+Mutation+Rate+in+the+Mitochondrial+Genome+of+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+Direct+Estimate+of+the+Mutation+Rate+in+the+Mitochondrial+Genome+of+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001012-Lambert1" id="pone.0001012-Lambert1"></a>Lambert DM, Ritchie PA, Millar CD, Holland B, Drummond AJ, et al.  (2002) Rates of Evolution in Ancient DNA from Adélie Penguins. Science  295: 2270.  <ul class="find" data-citedArticleID="1003416" data-doi="10.1126/science.1068105"><li><a href="http://dx.doi.org/10.1126/science.1068105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+Evolution+in+Ancient+DNA+from+Ad%C3%A9lie+Penguins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+Evolution+in+Ancient+DNA+from+Ad%C3%A9lie+Penguins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001012-Bradley1" id="pone.0001012-Bradley1"></a>Bradley D, MacHugh D, Cunningham P, Loftus R (1996) Mitochondrial diversity and the origins of African and European cattle. Proc Natl Acad Sci USA  93: 5131–5135.  <ul class="find" data-citedArticleID="1003386" data-doi="10.1073/pnas.93.10.5131"><li><a href="http://dx.doi.org/10.1073/pnas.93.10.5131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+diversity+and+the+origins+of+African+and+European+cattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+diversity+and+the+origins+of+African+and+European+cattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001012-Naderi1" id="pone.0001012-Naderi1"></a>Naderi S, Rezaei HR, Pompanon F, Blum MGB, Negrini R, et al.  (In Prep.) Genetic evidence for a large scale domestication in goats  <ul class="find" data-citedArticleID="1003434"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+evidence+for+a+large+scale+domestication+in+goats&amp;auth=&amp;atitle=Genetic+evidence+for+a+large+scale+domestication+in+goats" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+evidence+for+a+large+scale+domestication+in+goats" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+evidence+for+a+large+scale+domestication+in+goats%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001012-Fumagalli1" id="pone.0001012-Fumagalli1"></a>Fumagalli L, Taberlet P, Favre L, Hausser J (1996) Origin and evolution of homologous repeated sequences in the mitochondrial DNA control region of shrews. Mol Biol Evol  13: 31–46.  <ul class="find" data-citedArticleID="1003404" data-doi="10.1093/oxfordjournals.molbev.a025568"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025568" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+and+evolution+of+homologous+repeated+sequences+in+the+mitochondrial+DNA+control+region+of+shrews." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+and+evolution+of+homologous+repeated+sequences+in+the+mitochondrial+DNA+control+region+of+shrews.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001012-Hiendleder1" id="pone.0001012-Hiendleder1"></a>Hiendleder S, Lewalski H, Wassmuth R, Janke A (1998) The complete mitochondrial DNA sequence of the domestic sheep (<em>Ovis aries</em>) and comparison with the other major ovine haplotype. J Mol Evol  47: 441–448.  <ul class="find" data-citedArticleID="1003406" data-doi="10.1007/pl00006401"><li><a href="http://dx.doi.org/10.1007/pl00006401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+mitochondrial+DNA+sequence+of+the+domestic+sheep+%28Ovis+aries%29+and+comparison+with+the+other+major+ovine+haplotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+mitochondrial+DNA+sequence+of+the+domestic+sheep+%28Ovis+aries%29+and+comparison+with+the+other+major+ovine+haplotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001012-Meadows1" id="pone.0001012-Meadows1"></a>Meadows JRS, Li K, Kantanen J, Tapio M, Sipos W, et al.  (2005) Mitochondrial Sequence Reveals High Levels of Gene Flow Between Breeds of Domestic Sheep from Asia and Europe. J Hered  96(5): 494–501.  <ul class="find" data-citedArticleID="1003430" data-doi="10.1093/jhered/esi100"><li><a href="http://dx.doi.org/10.1093/jhered/esi100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondrial+Sequence+Reveals+High+Levels+of+Gene+Flow+Between+Breeds+of+Domestic+Sheep+from+Asia+and+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondrial+Sequence+Reveals+High+Levels+of+Gene+Flow+Between+Breeds+of+Domestic+Sheep+from+Asia+and+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001012-Meadows2" id="pone.0001012-Meadows2"></a>Meadows JRS, Cemal I, Karaca O, Gootwine E, Kijas JW, et al.  (2007) Five Ovine Mitochondrial Lineages Identified From Sheep Breeds of the Near East. Genetics  175: 1371–1379.  <ul class="find" data-citedArticleID="1003432" data-doi="10.1534/genetics.106.068353"><li><a href="http://dx.doi.org/10.1534/genetics.106.068353" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Five+Ovine+Mitochondrial+Lineages+Identified+From+Sheep+Breeds+of+the+Near+East." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Five+Ovine+Mitochondrial+Lineages+Identified+From+Sheep+Breeds+of+the+Near+East.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001012-Mannen1" id="pone.0001012-Mannen1"></a>Mannen H, Kohno M, Nagata Y, Tsuji S, Bradley DG, et al.  (2004) Independent mitochondrial origin and historical genetic differentiation in North Eastern Asian cattle. Mol Phylogenet Evol  32: 539–544.  <ul class="find" data-citedArticleID="1003426" data-doi="10.1016/j.ympev.2004.01.010"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.01.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+mitochondrial+origin+and+historical+genetic+differentiation+in+North+Eastern+Asian+cattle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+mitochondrial+origin+and+historical+genetic+differentiation+in+North+Eastern+Asian+cattle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001012-Freeman1" id="pone.0001012-Freeman1"></a>Freeman AR, Meghan CM, Machugh DE, Loftus RT, Achukwi MD, et al.  (2004) Admixture and diversity in West African cattle populations. Mol Ecol  13: 3477–3487.  <ul class="find" data-citedArticleID="1003402" data-doi="10.1111/j.1365-294x.2004.02311.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2004.02311.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Admixture+and+diversity+in+West+African+cattle+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Admixture+and+diversity+in+West+African+cattle+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001012-Fernndez1" id="pone.0001012-Fernndez1"></a>Fernández H, Hugues S, Vigne JD, Helmer D, Hodgins G, et al.  (2006) Divergent mtDNA lineages of goats in an Early Neolithic site, far from the initial domestication areas. Proc Natl Acad Sci USA  103: 15375–15379.  <ul class="find" data-citedArticleID="1003400" data-doi="10.1073/pnas.0602753103"><li><a href="http://dx.doi.org/10.1073/pnas.0602753103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergent+mtDNA+lineages+of+goats+in+an+Early+Neolithic+site%2C+far+from+the+initial+domestication+areas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergent+mtDNA+lineages+of+goats+in+an+Early+Neolithic+site%2C+far+from+the+initial+domestication+areas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001012-Vigne1" id="pone.0001012-Vigne1"></a>Vigne J.-D, et al. pp. 52–75.  in Archaeozoology of the Near East IV, Proc. 4th int. Symp. Archaeozoology of Southwestern Asia and adjacent areas (ASWA; Paris, June 1998) M. Mashkour, A. M. Choyke, H. Buitenhuis, F. Poplin, Eds. (Archaeological Research and Consultancy, Groningen, 2000), vol. Publicaties 32.  <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001012-Tuon1" id="pone.0001012-Tuon1"></a>Tuñon M, Gonzalez JP, Vallejo M (1989) Genetic relationships between 14 native spanish breeds of goat. Anim Genet  20: 205–212.  <ul class="find" data-citedArticleID="1003466" data-doi="10.1111/j.1365-2052.1989.tb00858.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2052.1989.tb00858.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+relationships+between+14+native+spanish+breeds+of+goat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+relationships+between+14+native+spanish+breeds+of+goat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001012-Caon1" id="pone.0001012-Caon1"></a>Cañon J, García D, García-Atance M, Obexer-Ruff AG, Lenstra J, et al.  (2006) Geographical partitioning of goat diversity in Europe and the Middle East. Anim Genet  37: 327–334.  <ul class="find" data-citedArticleID="1003390" data-doi="10.1111/j.1365-2052.2006.01461.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2052.2006.01461.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geographical+partitioning+of+goat+diversity+in+Europe+and+the+Middle+East." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geographical+partitioning+of+goat+diversity+in+Europe+and+the+Middle+East.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001012-Pariset1" id="pone.0001012-Pariset1"></a>Pariset L, Cappuccio I, Ajmone Marsan P, Dunner S, Luikart G, et al.  (2006) Assessment of population structure by single nucleotide polymorphisms (SNPs) in goat breeds. J Chromatography B.   <ul class="find" data-citedArticleID="1003438" data-doi="10.1016/j.jchromb.2006.01.011"><li><a href="http://dx.doi.org/10.1016/j.jchromb.2006.01.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+population+structure+by+single+nucleotide+polymorphisms+%28SNPs%29+in+goat+breeds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+population+structure+by+single+nucleotide+polymorphisms+%28SNPs%29+in+goat+breeds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001012-Taberlet1" id="pone.0001012-Taberlet1"></a>Taberlet P, Ajmone-Marsan P, Valentini A, Rezaei HR, Naderi S, et al.  (In Press) Are cattle, sheep, and goats endangered species? Mol Ecol.   <ul class="find" data-citedArticleID="1003462" data-doi="10.1111/j.1365-294x.2007.03475.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2007.03475.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Are+cattle%2C+sheep%2C+and+goats+endangered+species%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Are+cattle%2C+sheep%2C+and+goats+endangered+species%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001012-Luikart2" id="pone.0001012-Luikart2"></a>Luikart G, England P, Tallmon D, Jordan S, Taberlet P (2003) The power and promise of population genomics: from genotyping to genome typing. Nat Rev Genet  4: 981–994.  <ul class="find" data-citedArticleID="1003420" data-doi="10.1038/nrg1226"><li><a href="http://dx.doi.org/10.1038/nrg1226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+power+and+promise+of+population+genomics%3A+from+genotyping+to+genome+typing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+power+and+promise+of+population+genomics%3A+from+genotyping+to+genome+typing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001012-Shackleton1" id="pone.0001012-Shackleton1"></a>Shackleton DM (1997) Wild sheep and goats and their relatives: Status survey and conservation action plan for Caprinae. Gland, Switzerland: IUCN.   <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001012-Parma1" id="pone.0001012-Parma1"></a>Parma P, Feligini M, Greeppi G, Enne G (2003) The complete nucleotide sequence of goat (<em>Capra hircus</em>) mitochondrial genome. Goat mitochondrial genome. DNA Seq  14(3): 199–203.  <ul class="find" data-citedArticleID="1003440"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+complete+nucleotide+sequence+of+goat+%28Capra+hircus%29+mitochondrial+genome.+Goat+mitochondrial+genome.&amp;auth=&amp;atitle=The+complete+nucleotide+sequence+of+goat+%28Capra+hircus%29+mitochondrial+genome.+Goat+mitochondrial+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+nucleotide+sequence+of+goat+%28Capra+hircus%29+mitochondrial+genome.+Goat+mitochondrial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+nucleotide+sequence+of+goat+%28Capra+hircus%29+mitochondrial+genome.+Goat+mitochondrial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001012-Kumar1" id="pone.0001012-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignement. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="1003414" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001012-Yang1" id="pone.0001012-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. CABIOS  13: 555–556.  <ul class="find" data-citedArticleID="1003470" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001012-Swofford1" id="pone.0001012-Swofford1"></a>Swofford DL (2003) PAUP*: Phylogenetic Analysis Using Parsimony (*and Other Methods), Version 4. Sunderland, Massachusetts: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">46.
              </span><a name="pone.0001012-Excoffier1" id="pone.0001012-Excoffier1"></a>Excoffier L, Laval G, Schneider S (2005) Arlequin ver. 3.0: An integrated software package for population genetics data analysis. Evolutionary Bioinformatics Online  1: 47–50.  <ul class="find" data-citedArticleID="1003398"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis.&amp;auth=&amp;atitle=Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arlequin+ver.+3.0%3A+An+integrated+software+package+for+population+genetics+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001012-Schneider1" id="pone.0001012-Schneider1"></a>Schneider S, Excoffier L (1999) Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA. Genetics  152: 1079–1089.  <ul class="find" data-citedArticleID="1003454"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA.&amp;auth=&amp;atitle=Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+past+demographic+parameters+from+the+distribution+of+pairwise+differences+when+the+mutation+rates+vary+among+sites%3A+application+to+human+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001012-Kuhner1" id="pone.0001012-Kuhner1"></a>Kuhner M (2006) LAMARC 2.0: maximum likelihood and Bayesian estimation of population parameters. Bioinformatics  22: 768–770.  <ul class="find" data-citedArticleID="1003412" data-doi="10.1093/bioinformatics/btk051"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btk051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LAMARC+2.0%3A+maximum+likelihood+and+Bayesian+estimation+of+population+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LAMARC+2.0%3A+maximum+likelihood+and+Bayesian+estimation+of+population+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.XML" value="137787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.PDF" value="331825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g001.PNG_L" value="194109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g001.PNG_M" value="36092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g001.PNG_S" value="8141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g001.TIF" value="306124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g001.PNG_I" value="26669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g002.PNG_L" value="1620930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g002.PNG_M" value="75278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g002.PNG_S" value="9575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g002.TIF" value="2082500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g002.PNG_I" value="48517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t001.PNG_L" value="68290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t001.PNG_M" value="96949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t001.PNG_S" value="14211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t001.TIF" value="303892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t001.PNG_I" value="32655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t002.PNG_L" value="827653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t002.PNG_M" value="213909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t002.PNG_S" value="19601"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t002.TIF" value="3435898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t002.PNG_I" value="96336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t003.PNG_L" value="61865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t003.PNG_M" value="37222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t003.PNG_S" value="8234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t003.TIF" value="286816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t003.PNG_I" value="11374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g003.PNG_L" value="269501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g003.PNG_M" value="48964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g003.PNG_S" value="8457"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g003.TIF" value="598040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.g003.PNG_I" value="17389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t004.PNG_L" value="103331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t004.PNG_M" value="61719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t004.PNG_S" value="9017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t004.TIF" value="436956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t004.PNG_I" value="19040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t005.PNG_L" value="141686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t005.PNG_M" value="137483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t005.PNG_S" value="19196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t005.TIF" value="677040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.t005.PNG_I" value="75032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001012.s001.XLS" value="1237504"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001012&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001012" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001012&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001012&volume=&issue=&title=Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity&author_name=Saeid%20Naderi%2C%20Hamid-Reza%20Rezaei%2C%20Pierre%20Taberlet%2C%20St%C3%A9phanie%20Zundel%2C%20Seyed-Abbas%20Rafat%2C%20Hamid-Reza%20Naghash%2C%20Mohamed%20A.%20A.%20El-Barody%2C%20Okan%20Ertugrul%2C%20Fran%C3%A7ois%20Pompanon%2C%20for%20the%20Econogene%20Consortium&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001012" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012&amp;t=Large-Scale%20Mitochondrial%20DNA%20Analysis%20of%20the%20Domestic%20Goat%20Reveals%20Six%20Haplogroups%20with%20High%20Diversity" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012&title=Large-Scale%20Mitochondrial%20DNA%20Analysis%20of%20the%20Domestic%20Goat%20Reveals%20Six%20Haplogroups%20with%20High%20Diversity&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012&amp;title=Large-Scale%20Mitochondrial%20DNA%20Analysis%20of%20the%20Domestic%20Goat%20Reveals%20Six%20Haplogroups%20with%20High%20Diversity" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001012&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Large-Scale Mitochondrial DNA Analysis of the Domestic Goat Reveals Six Haplogroups with High Diversity';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001012';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Large-Scale%20Mitochondrial%20DNA%20Analysis%20of%20the%20Domestic%20Goat%20Reveals%20Six%20Haplogroups%20with%20High%20Diversity http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001012" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001012" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Geography%22" title="Search for articles in the subject area:'Geography'"><div class="flagText">Geography</div></a>
              <div data-categoryid="1863" data-articleid="24858"
                   data-categoryname="Geography"
                   class="flagImage" title="Flag 'Geography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Goats%22" title="Search for articles in the subject area:'Goats'"><div class="flagText">Goats</div></a>
              <div data-categoryid="42799" data-articleid="24858"
                   data-categoryname="Goats"
                   class="flagImage" title="Flag 'Goats' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplogroups%22" title="Search for articles in the subject area:'Haplogroups'"><div class="flagText">Haplogroups</div></a>
              <div data-categoryid="43141" data-articleid="24858"
                   data-categoryname="Haplogroups"
                   class="flagImage" title="Flag 'Haplogroups' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="24858"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="24858"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondrial+DNA%22" title="Search for articles in the subject area:'Mitochondrial DNA'"><div class="flagText">Mitochondrial DNA</div></a>
              <div data-categoryid="32451" data-articleid="24858"
                   data-categoryname="Mitochondrial DNA"
                   class="flagImage" title="Flag 'Mitochondrial DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="24858"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+groupings%22" title="Search for articles in the subject area:'Population groupings'"><div class="flagText">Population groupings</div></a>
              <div data-categoryid="40421" data-articleid="24858"
                   data-categoryname="Population groupings"
                   class="flagImage" title="Flag 'Population groupings' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9986'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6766'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8437&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
