

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000667"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000667" />

    <meta name="citation_title" content="A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities"/>
    <meta itemprop="name" content="A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities"/>

      <meta name="citation_author" content="David Emerson"/>
            <meta name="citation_author_institution" content="American Type Culture Collection, Manassas, Virginia, United States of America"/>
      <meta name="citation_author" content="Jeremy A. Rentz"/>
            <meta name="citation_author_institution" content="American Type Culture Collection, Manassas, Virginia, United States of America"/>
      <meta name="citation_author" content="Timothy G. Lilburn"/>
            <meta name="citation_author_institution" content="American Type Culture Collection, Manassas, Virginia, United States of America"/>
      <meta name="citation_author" content="Richard E. Davis"/>
            <meta name="citation_author_institution" content="Department of Biology, Western Washington University, Bellingham, Washington, United States of America"/>
      <meta name="citation_author" content="Henry Aldrich"/>
            <meta name="citation_author_institution" content="Department of Microbiology, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Clara Chan"/>
            <meta name="citation_author_institution" content="Woods Hole Oceanographic Institute, Woods Hole, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Craig L. Moyer"/>
            <meta name="citation_author_institution" content="Department of Biology, Western Washington University, Bellingham, Washington, United States of America"/>

    <meta name="citation_date" content="2007/8/1"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000667.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e667"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Microbial oxidation of Fe(II) at circumneutral pH.; citation_author=D Emerson; citation_number=1; citation_pages=31-52; citation_date=2000; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=The oxygenation of the atmosphere and oceans.; citation_author=HD Holland; citation_journal_title=Phil Trans R Soc B; citation_volume=361; citation_number=2; citation_pages=903-915; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biogenic influences on the geochemistry of certain ferruginous sediments of hydrothermal origin.; citation_author=NG Holm; citation_journal_title=Chem Geol; citation_volume=63; citation_number=3; citation_pages=45-57; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Hydrothermal oxide and nontronite deposits on seamounts in the Eastern Pacific.; citation_author=JC Alt; citation_journal_title=Marine Geology; citation_volume=81; citation_number=4; citation_pages=227-239; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Filamentous iron-silica deposits from modern and ancient hydrothermal sites.; citation_author=SK Juniper; citation_author=Y Fouquet; citation_journal_title=Canadian Mineralogist; citation_volume=26; citation_number=5; citation_pages=859-869; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Microbe-metal interactions and mineral deposition at hydrothermal vents.; citation_author=SK Juniper; citation_author=BM Tebo; citation_number=6; citation_pages=219-253; citation_date=1995; citation_publisher=CRC Press; " />
      <meta name="citation_reference" content="citation_title=Vailulu'u Seamount, Samoa: life and death on an active submarine volcano.; citation_author=H Staudigel; citation_author=SR Hart; citation_author=A Pile; citation_author=BE Bailey; citation_author=ET Baker; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=7; citation_pages=6448-6453; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Diversity of life in ocean floor basalt.; citation_author=IH Thorseth; citation_author=T Torsvik; citation_author=V Torsvik; citation_author=FL Daae; citation_author=R Pedersen; citation_journal_title=Earth Planet Sci Lett; citation_volume=194; citation_number=8; citation_pages=31-37; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Iron and sulfide oxidation within the basaltic ocean crust: implications for chemolithoautotrophic microbial biomass production.; citation_author=W Bach; citation_author=KJ Edwards; citation_journal_title=Geochim Cosmochim Acta; citation_volume=67; citation_number=9; citation_pages=3871-3887; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The quantum event of oceanic crustal accretion: impacts of diking at mid-ocean ridges.; citation_author=JR Delaney; citation_author=DS Kelley; citation_author=MD Lilley; citation_author=DA Butterfield; citation_author=JA Baross; citation_journal_title=Science; citation_volume=281; citation_number=10; citation_pages=222-230; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Formation and occurrence of biogenic iron-rich minerals.; citation_author=D Fortin; citation_author=S Langley; citation_journal_title=Earth-Science Rev; citation_volume=72; citation_number=11; citation_pages=1-19; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Neutrophilic iron-oxidizing bacteria in the ocean: Their habitats, diversity, and roles in mineral deposition, rock alteration, and biomass production in the deep-sea.; citation_author=KJ Edwards; citation_author=W Bach; citation_author=TM McCollom; citation_author=DR Rogers; citation_journal_title=Geomicrobiol J; citation_volume=21; citation_number=12; citation_pages=393-404; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Neutrophilic Fe-oxidizing bacteria are abundant at the Loihi Seamount hydrothermal vents and play a major role in Fe oxide deposition.; citation_author=D Emerson; citation_author=CL Moyer; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=13; citation_pages=3085-3093; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Modern methods for identifying bacteria.; citation_author=DE Stead; citation_author=J Hennessy; citation_author=J Wilson; citation_journal_title=Plant Cell Tissue Organ Cult; citation_volume=52; citation_number=14; citation_pages=17-25; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Application of fatty acid methyl esters for taxonomic analysis of the genus Xanthomonas.; citation_author=L Yang; citation_author=L Vauterin; citation_author=M Vancanneyt; citation_author=J Swings; citation_author=K Kersters; citation_journal_title=Syst Appl Microbiol; citation_volume=16; citation_number=15; citation_pages=47-71; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Thermomonas fusca sp nov and Thermomonas brevis sp nov., two mesophilic species isolated from a denitrification reactor with poly(epsilon-caprolactone) plastic granules as fixed bed, and emended description of the genus Thermomonas.; citation_author=J Mergaert; citation_author=MC Cnockaert; citation_author=J Swings; citation_journal_title=International Journal of Systematic and Evolutionary Microbiology; citation_volume=53; citation_number=16; citation_pages=1961-1966; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Overview: a phylogenetic backbone and taxonomic framework for prokaryotic systematics.; citation_author=W Ludwig; citation_author=H-P Klenk; citation_number=17; citation_pages=49-65; citation_date=2001; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=The road map to the manual.; citation_author=GM Garrity; citation_author=JG Holt; citation_number=18; citation_pages=119-155; citation_date=2001; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=fastDNAmL: a tool for construction of phylogenetic trees of DNA sequences using maximum likelihood.; citation_author=GJ Olsen; citation_author=H Matsuda; citation_author=R Hagstrom; citation_author=R OVerbeek; citation_journal_title=Comput Appl Biosci; citation_volume=10; citation_number=19; citation_pages=41-48; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for molecular evolutionary genetics analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=20; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Iron corrosion by novel anaerobic microorganisms.; citation_author=HT Dinh; citation_author=J Kuever; citation_author=M Mubmann; citation_author=AW Hassel; citation_author=M Stratmann; citation_journal_title=Nature; citation_volume=427; citation_number=21; citation_pages=829-832; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Microbial polysaccharides template assembly of nanocrystal fibers.; citation_author=CS Chan; citation_author=G De Stasio; citation_author=SA Welch; citation_author=M Girasole; citation_author=BH Frazer; citation_journal_title=Science; citation_volume=303; citation_number=22; citation_pages=1656-1658; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ultrastructure and chemical composition of the sheath of Leptothrix discophora SP-6.; citation_author=D Emerson; citation_author=WC Ghiorse; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=23; citation_pages=7808-7818; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Gallionella.; citation_author=EL Hallbeck; citation_author=K Pedersen; citation_number=24; citation_pages=880-886; citation_date=2005; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Rapid microbial production of filamentous sulfur mats at hydrothermal vents.; citation_author=CD Taylor; citation_author=CO Wirsen; citation_author=F Gaill; citation_journal_title=Appl Environ Microbiol; citation_volume=65; citation_number=25; citation_pages=2253-2255; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Growth and mechanism of filamentous-sulfur formation by Candidatus Arcobacter sulfidicus in opposing oxygen-sulfide gradients.; citation_author=SM Sievert; citation_author=EBA Wieringa; citation_author=CO Wirsen; citation_author=CD Taylor; citation_journal_title=Environ Microbiol.; citation_volume=9; citation_number=26; citation_pages=271-276; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of novel psychrophilic, neutrophilic Fe-oxidizing, chemolithotrophic α− and γ-Proteobacteria from the deep sea.; citation_author=KJ Edwards; citation_author=DR Rogers; citation_author=CO Wirsen; citation_author=TM McCollom; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=27; citation_pages=2906-2913; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogeny and phenotypic characterization of the stalk-forming and iron-oxidizing bacterium Gallionella ferruginea.; citation_author=L Hallbeck; citation_author=F Ståhl; citation_author=K Pedersen; citation_journal_title=J Gen Microbiol; citation_volume=139; citation_number=28; citation_pages=1531-1535; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Characterization of a neutrophilic, chemolithoautotrophic Fe(II)-ß-proteobacterium from freshwater wetland sediments.; citation_author=D Sobolev; citation_author=EE Roden; citation_journal_title=Geomicrobiol J; citation_volume=21; citation_number=29; citation_pages=1-10; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of novel iron-oxidizing bacteria that grow at circumneutral pH.; citation_author=D Emerson; citation_author=C Moyer; citation_journal_title=Appl Environ Microbiol; citation_volume=63; citation_number=30; citation_pages=4784-4792; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Sideroxydans lithotrophicus, gen. nov., sp. nov. and Gallionella capsiferriformans sp.nov., oxygen-dependent ferrous iron-oxidizing bacteria that grow at circumneutral pH.; citation_author=D Emerson; citation_author=JA Rentz; citation_author=T Plaia; citation_number=31; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Characterization of neutrophilic Fe(II)-oxidizing bacteria isolated from the rhizosphere of wetland plants and description of Ferritrophicum radicicola gen. nov. sp. nov., and Sideroxydans paludicola sp. nov.; citation_author=JV Weiss; citation_author=JA Rentz; citation_author=T Plaia; citation_author=SC Neubauer; citation_author=MM Floyd; citation_number=32; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Microbial diversity of the brine-seawater interface of the Kebrit Deep, Red Sea, studied via 16S rRNA gene sequences and cultivation methods.; citation_author=W Eder; citation_author=LL Jahnke; citation_author=M Schmidt; citation_author=R Huber; citation_journal_title=Appl Environ Microbiol; citation_volume=67; citation_number=33; citation_pages=3077-3085; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Estimation of diversity and community structure through restriction fragment polymorphism distribution analysis of bacterial 16S rRNA genes from a microbial mat at an active, hydrothermal vent system, Loihi Seamount, Hawaii.; citation_author=CL Moyer; citation_author=FC Dobbs; citation_author=DM Karl; citation_journal_title=Appl Environ Microbiol; citation_volume=60; citation_number=34; citation_pages=871-879; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic diversity of the bacterial community from a microbial mat at an active, hydrothermal vent system, Loihi Seamount, Hawaii.; citation_author=CL Moyer; citation_author=FC Dobbs; citation_author=DM Karl; citation_journal_title=Appl Environ Microbiol; citation_volume=61; citation_number=35; citation_pages=1555-1562; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of sulfate-reducing bacteria in the Guaymas Basin.; citation_author=A Dhillon; citation_author=A Teske; citation_author=J Dillon; citation_author=DA Stahl; citation_author=ML Sogin; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=36; citation_pages=2765-2722; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Spatial and temporal variability in microbial communities from pre- and post-eruption microbial mats collected from Loihi Seamount, Hawaii.; citation_author=RE Davis; citation_author=T Carney; citation_author=K Leal; citation_author=CL Moyer; citation_journal_title=Abstr Fall Mtg Eos Trans AGU; citation_volume=86; citation_number=37; citation_pages=V51C-1508; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expanding frontiers in deep subsurface microbiology.; citation_author=JP Amend; citation_author=A Teske; citation_journal_title=Paleogeogra Paleoclimatol Palaeoecol; citation_volume=219; citation_number=38; citation_pages=131-155; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Geomicrobiology of deep-sea hydrothermal vents.; citation_author=HW Jannasch; citation_author=MJ Mottl; citation_journal_title=Science; citation_volume=229; citation_number=39; citation_pages=717-725; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Geochemical constraints on primary productivity in submarine hydrothermal vent plumes.; citation_author=TM McCollum; citation_journal_title=Deep-Sea Res; citation_volume=47; citation_number=40; citation_pages=85-101; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Characterization of bacteriogenic iron oxide deposits from Axial Volcano, Juan de Fuca Ridge, Northeast Pacific Ocean.; citation_author=CB Kennedy; citation_author=SD Scott; citation_author=FG Ferris; citation_journal_title=Geomicrobiol J; citation_volume=20; citation_number=41; citation_pages=199-214; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Enrichment and isolation of iron-oxidizing bacteria at neutral pH.; citation_author=D Emerson; citation_author=MM Floyd; citation_journal_title=Methods Enzymol; citation_volume=397; citation_number=42; citation_pages=112-124; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Alkanindiges illinoisensis gen. nov., sp. nov., an obligately hydrocarbonoclastic, aerobic squalene-degrading bacterium isolated from oilfield soils.; citation_author=BW Bogan; citation_author=WR Sullivan; citation_author=KJ Kayser; citation_author=KD Derr; citation_author=HC Aldrich; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=53; citation_number=43; citation_pages=1389-1395; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Tindallia californiensis sp. nov., a new anaerobic, haloalkalophilic, spore-forming acetogen isolated from Mono Lake in California.; citation_author=E Pikuta; citation_author=RB Hoover; citation_author=AK Bej; citation_author=D Marsic; citation_author=EN Detkova; citation_journal_title=Extremophiles; citation_volume=7; citation_number=44; citation_pages=327-334; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ARB: a software environment for sequence data.; citation_author=W Ludwig; citation_author=O Strunk; citation_author=R Westram; citation_author=L Richter; citation_author=H Meier; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=45; citation_pages=1363-1371; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BioEdit: A user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT.; citation_author=TA Hall; citation_journal_title=Nucleic Acids Symp Ser; citation_volume=41; citation_number=46; citation_pages=95-98; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the maximum likelihood estimate of the evolutionary tree topologies from DNA sequence data, and the branching order in hominoidea.; citation_author=H Kishino; citation_author=M Hasegawa; citation_journal_title=J Mol Evol; citation_volume=29; citation_number=47; citation_pages=170-179; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=MAFFT version 5: improvements in accuracy of multiple sequence alignment.; citation_author=K Katoh; citation_author=K Kuma; citation_author=H Toh; citation_author=T Miyata; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=48; citation_pages=511-518; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluation of iterative alignment algorithms for multiple alignment.; citation_author=IM Wallace; citation_author=O Orla; citation_author=DG Higgins; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=49; citation_pages=1408-1414; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple comparisons of log-likelihoods with applications ot plylogenetic inference.; citation_author=H Shimodaira; citation_author=M Hasegawa; citation_journal_title=Molec Biol Evol; citation_volume=16; citation_number=50; citation_pages=1114-1116; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Characterization of bacterial community structure in vestimentiferan tubeworm Ridgeia piscesae trophosomes.; citation_author=LSL Chao; citation_author=RE Davis; citation_author=CL Moyer; citation_journal_title=Mar Ecol; citation_volume=28; citation_number=51; citation_pages=1-14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Fidelity of select restriction endonucleases in determining microbial diversity by terminal-restriction fragment length polymorphism.; citation_author=JJ Engebretson; citation_author=CL Moyer; citation_journal_title=Appl Environ Microbiol; citation_volume=69; citation_number=52; citation_pages=4823-4829; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Pearson product-moment correlation coefficient is better suited for identification of DNA fingerprint profiles than band matching algorithms.; citation_author=BG Häne; citation_author=K Jäger; citation_author=HG Drexler; citation_journal_title=Electrophorhesis; citation_volume=14; citation_number=53; citation_pages=967-972; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities"/>
    <meta name="twitter:description" content="BackgroundFor decades it has been recognized that neutrophilic Fe-oxidizing bacteria (FeOB) are associated with hydrothermal venting of Fe(II)-rich fluids associated with seamounts in the world's oceans. The evidence was based almost entirely on the mineralogical remains of the microbes, which themselves had neither been brought into culture or been assigned to a specific phylogenetic clade. We have used both cultivation and cultivation-independent techniques to study Fe-rich microbial mats associated with hydrothermal venting at Loihi Seamount, a submarine volcano.Methodology/Principle FindingsUsing gradient enrichment techniques, two iron-oxidizing bacteria, strains PV-1 and JV-1, were isolated. Chemolithotrophic growth was observed under microaerobic conditions; Fe(II) and Fe0 were the only energy sources that supported growth. Both strains produced filamentous stalk-like structures composed of multiple nanometer sized fibrils of Fe-oxyhydroxide. These were consistent with mineralogical structures found in the iron mats. Phylogenetic analysis of the small subunit (SSU) rRNA gene demonstrated that strains PV-1 and JV-1 were identical and formed a monophyletic group deeply rooted within the Proteobacteria. The most similar sequence (85.3% similarity) from a cultivated isolate came from Methylophaga marina. Phylogenetic analysis of the RecA and GyrB protein sequences confirmed that these strains are distantly related to other members of the Proteobacteria. A cultivation-independent analysis of the SSU rRNA gene by terminal-restriction fragment (T-RF) profiling showed that this phylotype was most common in a variety of microbial mats collected at different times and locations at Loihi.ConclusionsOn the basis of phylogenetic and physiological data, it is proposed that isolate PV-1T (&#8202;=&#8202;ATCC BAA-1019: JCM 14766) represents the type strain of a novel species in a new genus, Mariprofundus ferrooxydans gen. nov., sp. nov. Furthermore, the strain is the first cultured representative of a new candidatus class of the Proteobacteria that is widely distributed in deep-sea environments, Candidatus &zeta; (zeta)-Proteobacteria cl. nov."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000667.g005"/>

  <meta property="og:title" content="A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9635'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4839'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9075&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000667">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000667" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000667&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Emerson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:demerson@gmu.edu">demerson@gmu.edu</a></p>

                <p>Affiliation:
                  American Type Culture Collection, Manassas, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jeremy A. Rentz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  American Type Culture Collection, Manassas, Virginia, United States of America
                </p>

                <p>
                    Current address: Civil and Environmental Engineering, Washington State University, Pullman, Washington, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Timothy G. Lilburn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  American Type Culture Collection, Manassas, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Richard E. Davis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Western Washington University, Bellingham, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Henry Aldrich, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Clara Chan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Woods Hole Oceanographic Institute, Woods Hole, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Craig L. Moyer
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Western Washington University, Bellingham, Washington, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 01, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000667</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000667-g001" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000667-t001" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000667-g002" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000667-g003" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000667-g004" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000667-g005" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000667">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000667" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>For decades it has been recognized that neutrophilic Fe-oxidizing bacteria (FeOB) are associated with hydrothermal venting of Fe(II)-rich fluids associated with seamounts in the world's oceans. The evidence was based almost entirely on the mineralogical remains of the microbes, which themselves had neither been brought into culture or been assigned to a specific phylogenetic clade. We have used both cultivation and cultivation-independent techniques to study Fe-rich microbial mats associated with hydrothermal venting at Loihi Seamount, a submarine volcano.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using gradient enrichment techniques, two iron-oxidizing bacteria, strains PV-1 and JV-1, were isolated. Chemolithotrophic growth was observed under microaerobic conditions; Fe(II) and Fe<sup>0</sup> were the only energy sources that supported growth. Both strains produced filamentous stalk-like structures composed of multiple nanometer sized fibrils of Fe-oxyhydroxide. These were consistent with mineralogical structures found in the iron mats. Phylogenetic analysis of the small subunit (SSU) rRNA gene demonstrated that strains PV-1 and JV-1 were identical and formed a monophyletic group deeply rooted within the <em>Proteobacteria</em>. The most similar sequence (85.3% similarity) from a cultivated isolate came from <em>Methylophaga marina</em>. Phylogenetic analysis of the RecA and GyrB protein sequences confirmed that these strains are distantly related to other members of the <em>Proteobacteria</em>. A cultivation-independent analysis of the SSU rRNA gene by terminal-restriction fragment (T-RF) profiling showed that this phylotype was most common in a variety of microbial mats collected at different times and locations at Loihi.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>On the basis of phylogenetic and physiological data, it is proposed that isolate PV-1<sup>T</sup> ( = ATCC BAA-1019: JCM 14766) represents the type strain of a novel species in a new genus, <em>Mariprofundus ferrooxydans</em> gen. nov., sp. nov. Furthermore, the strain is the first cultured representative of a new candidatus class of the <em>Proteobacteria</em> that is widely distributed in deep-sea environments, Candidatus ζ <em>(zeta)</em>-<em>Proteobacteria</em> cl. nov.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Emerson D, Rentz JA, Lilburn TG, Davis RE, Aldrich H, et al.  (2007) A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities. PLoS ONE 2(8):
          e667.
            doi:10.1371/journal.pone.0000667</p><p><strong>Academic Editor: </strong>Anna-Louise Reysenbach, Portland State University, United States of America</p><p><strong>Received:</strong> March 26, 2007; <strong>Accepted:</strong> June 25, 2007; <strong>Published:</strong> August 1, 2007</p><p><strong>Copyright:</strong> © 2007 Emerson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding was provided to DE and CLM by the National Science Foundation (0348330) and to DE through the NASA Astobiology Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Bacteria are known for the wide variety of reduced inorganic substrates they are able to oxidize and utilize as energy sources for growth. These include sulfur, ammonia, methane, hydrogen, and iron. The oxidation of ferrous (Fe (II)) to ferric (Fe(III)) iron is especially enigmatic because thermodynamically it yields a minimal amount of energy for growth, approximately ΔG°′ = −109 kJ mol<sup>−1</sup>, and at neutral pH chemical oxidation of Fe(II) to Fe(III), or iron oxyhydroxides, i.e. rust, occurs rapidly in the presence of oxygen <a href="#pone.0000667-Emerson1">[1]</a>. To circumvent these problems, bacteria that oxidize Fe(II) to obtain energy at circumneutral pH have two key requirements for growth, low O<sub>2</sub> partial pressures and sustained Fe(II) concentrations. This kind of metabolism has received scant attention in marine systems, first, because practically nothing is known about marine iron-oxidizing bacteria, and second, because the oceans are generally considered to be oxygenated and depleted in Fe(II). In addition, marine sediments tend to be rich in sulfide, which reacts rapidly with Fe(II) to precipitate FeS.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>An important exception to these oceanic iron depleted conditions occurs in areas of hydrothermal venting, either at seamounts or at crustal spreading centers. Here, anoxic vent fluids charged with Fe(II) come in contact with the cold, oxygenated ocean water. It is estimated that the present-day flux of Fe(II) from hydrothermal venting is approximately 3×10<sup>11</sup> mol yr<sup>−1</sup> <a href="#pone.0000667-Holland1">[2]</a>. As a result, there are often substantial deposits of Fe-oxides at these vents, and in nearly every case that has been examined, the morphology of Fe-oxides provide abundant evidence for microbial activity <a href="#pone.0000667-Holm1">[3]</a>, <a href="#pone.0000667-Alt1">[4]</a>, <a href="#pone.0000667-Juniper1">[5]</a>, <a href="#pone.0000667-Juniper2">[6]</a>, <a href="#pone.0000667-Staudigel1">[7]</a>. The tell-tale signs of this activity are the tubular sheath casts of filamentous Fe-oxidizing bacteria (FeOB), or the twisted stalks of <em>Gallionella</em>-like organisms. Another potential habitat for marine FeOB is freshly formed oceanic basalts that result from volcanic and/or tectonic activity. In this case, leaching of Fe(II) directly from the mineral or glass surfaces may occur allowing for growth of biofilms of FeOB on the weathered rocks <a href="#pone.0000667-Thorseth1">[8]</a>, <a href="#pone.0000667-Bach1">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>These types of habitats are more common than is often appreciated, since crustal spreading centers and subduction zones, as well as seamounts with either active volcanism or cold seepage of mineral rich fluid flow are common throughout the world's oceans <a href="#pone.0000667-Delaney1">[10]</a>. Because iron is the fourth most abundant element in the Earth's crust, it is often the most abundant potential energy source at the boundaries of the oxygenated ocean and the reducing subsurface <a href="#pone.0000667-Fortin1">[11]</a>. However, we know very little about the population biology of prokaryotic lithotrophic communities that could potentially live off the Fe(II)-rich waters and vent fluids associated with these kinds of boundary habitats <a href="#pone.0000667-Emerson1">[1]</a>, <a href="#pone.0000667-Edwards1">[12]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We previously reported on the role that lithotrophic FeOB appear to play in the formation of extensive microbial mats that form at the Loihi Seamount <a href="#pone.0000667-Emerson2">[13]</a>. During that study two novel FeOB were isolated, strains JV-1 and PV-1. These organisms were striking because as they grew, they formed filamentous stalk-like structures that appeared to consist primarily of an amorphous iron oxyhydroxide. These filaments bore a strong morphological resemblance to the types of oxides seen in the mats at Loihi<a href="#pone.0000667-Emerson2">[13]</a>. We report here a more complete characterization of these two isolates and show that their metabolism is limited to Fe-oxidation. A cultivation-independent approach confirmed that they are common in iron mats located at Loihi. They are phylogenetically novel and represent the first cultured isolates of a new class of <em>Proteobacteria</em> that until now has only been represented by sequences in environmental SSU rRNA gene libraries associated with the floor of the deep ocean.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Growth characteristics of PV-1 and JV-1</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Both strains grew microaerobically with Fe<sup>2+</sup> as the sole energy source and CO<sub>2</sub> as the only available carbon source indicating the strains were obligate, lithotrophic FeOB. The following inorganic and organic substrates did not support growth: ammonium, thiosulfate, sulfide, Mn(II), tetrathionate, hydrogen, formate, acetate, pyruvate, succinate, glucose, galactose, ribose, glycerol, aspartate, and glycine. In addition no growth either under aerobic or microaerobic conditions was observed for the following complex media: tryptic soy agar, marine agar, and R2A (Difco) supplemented with 2.5% NaCl, nutrient agar supplemented with 2.5% NaCl, blood agar, PYG medium supplemented with 2.5% salt, or thioglycollate medium also supplemented with NaCl. The strains had an absolute requirement for marine salts and did not grow in a freshwater Fe-oxidizing medium that contained &lt;0.2 g/l Mg or Ca-salts. Both strains are motile. They prefer low oxygen concentrations and, based on their behavior in gradient tubes, appear to be aerotactic. Neither strain was able to reduce Fe(III) with acetate anaerobically, nor were they able to oxidize Fe(II) with nitrate as the electron acceptor. The isolates grew at temperatures between 10° and 30°C. Growth appeared to be very slow at 5°C and was not observed at 35°C. The pH range for growth was 5.5–7.2 and appeared optimal between 6.2 and 6.5. A spot test for catalase activity was negative.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>These FeOB were able to grow on a variety of forms of Fe(II) including FeS or on FeCO<sub>3</sub> (siderite) in gradient culture and on FeCl<sub>2</sub>, Fe(NH<sub>4</sub>)<sub>2</sub>(SO<sub>4</sub>)<sub>2</sub>, and FeSO<sub>4</sub> in liquid bottle cultures. They were also able to grow on zero valent iron, Fe(0). A growth curve for strain PV-1 comparing growth on Fe(0) and FeS is shown in <a href="#pone-0000667-g001">Figure 1</a>. Final growth yields tended to be higher on FeS, but initial growth yields on both forms of iron were comparable. The doubling time at 23°C was on the order of 12 h.</p>
<div class="figure" id="pone-0000667-g001"><div class="img"><a name="pone-0000667-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Growth curve of PV-1 on gradient plates with FeS (□) or Fe<sup>0</sup> (○) as the iron source.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A control (σ) with agarose but no FeS was included. Cell numbers were determined by direct counts and errors are based on duplicate samples.</p>
<span>doi:10.1371/journal.pone.0000667.g001</span></div>

<h4>FAME analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The MIDI system was only able to identify approximately 45% of the fatty acids in lipid extracts from late-log phase PV-1 cells. The dominant fatty acid was 11:0 iso 3OH, followed in relative abundance by 17:0 iso 3OH, 16:0, and 18:1 iso H (<a href="#pone-0000667-t001">Table 1</a>). Another dominant set of fatty acids was 16:1 w7c/15 iso 2OH, however these two fatty acids could not be discriminated from one another and are reported as summed in Feature 3 in <a href="#pone-0000667-t001">Table 1</a>. The abundance of the branched chain fatty acid 11:0 iso 3OH in PV-1 is unusual. This is generally considered a rare fatty acid in bacteria <a href="#pone.0000667-Stead1">[14]</a>. It has been shown to be diagnostic of the family <em>Xanthomonadaceae</em> and normally comprises &lt;10% of the total fatty acids in these organisms <a href="#pone.0000667-Yang1">[15]</a>, <a href="#pone.0000667-Mergaert1">[16]</a>.</p>
<div class="figure" id="pone-0000667-t001"><div class="img"><a name="pone-0000667-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000667.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000667.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>FAME Analysis, showing fatty acids ≥2% of total.</span></strong></p><span>doi:10.1371/journal.pone.0000667.t001</span></div>

<h4>Morphology</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Cells of strains PV-1 and JV-1 were curved rods (approximately 0.5×2–5 µm). The most distinguishing characteristic of these strains was the filamentous stalk-like structures of iron oxyhydroxides that they formed during cell growth, <a href="#pone-0000667-g002">Figure 2</a> and <a href="#pone.0000667-Emerson2">[13]</a>. The cells had Gram negative type cell walls when viewed by TEM, <a href="#pone-0000667-g002">Figure 2e</a>, and the fibrillar Fe-oxyhydroxides were observed as well as lesser amounts of fine particulate oxides sometimes associated with the cells. HRTEM showed that the stalks were themselves composed of a set of discrete fibers aligned parallel to the length of the stalk, <a href="#pone-0000667-g002">Figure 2f</a>. The stalk shown in <a href="#pone-0000667-g002">Figure 2f</a> is between 0.6 and 0.7 microns wide, with individual fibers that are each ~70 nm in diameter. The sample was not stained; thus contrast is due to iron minerals on or within the stalk. It was confirmed by energy dispersive X-ray analysis that the major element in these structures was Fe (data not shown). Treatment of the stalks with 0.3 M oxalic acid caused a rapid and nearly complete dissolution, suggesting that they are composed primarily of poorly crystalline iron oxyhydroxides (Supplemental <a href="#pone.0000667.s003">Figure S1</a>). When glass surfaces (microscope slides) were placed in a culture of PV-1 growing in liquid culture, the cells attached to the glass surface and formed stalks, <a href="#pone-0000667-g002">Figure 2a–d</a>. This confirmed that the cells grew at the apical ends of the stalks; presumably these cells are extruding the Fe oxyhydroxide filaments as they grow.</p>
<div class="figure" id="pone-0000667-g002"><div class="img"><a name="pone-0000667-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Micrographs of strain PV-1.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A–D, Light micrographs showing cells attached at the end of the filaments; the cells have been stained with Syto and composite epifluoresence and phase contrast images produced to show the cells stained green. The bar = 5 µm. E TEM image of the cells, note the Gram-negative type cell wall and the fibers of Fe-oxides in the lower left corner. The bar = 0.5 µm. F. A HRTEM image of the Fe-oxide filament showing its composition as a bundle of conjoined fibers of Fe-oxide. The bar = 0.63 µm.</p>
<span>doi:10.1371/journal.pone.0000667.g002</span></div>

<h4>Phylogeny</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>PV-1 and JV-1 SSU rDNA sequences were identical (1488 bases). Their genotypic similarity was confirmed with rep-PCR, where both strains gave nearly identical banding patterns (Supplemental <a href="#pone.0000667.s004">Figure S2</a>), that were distinct from freshwater FeOB. A phylogenetic analysis based on comparison of the SSU rRNA gene with other bacteria revealed that PV-1 and JV-1 were unusual members of the phylum <em>Proteobacteria</em> in that they did not cluster with any of the known classes of α-, β-, γ-, ε-, or δ-<em>Proteobacteria</em> (<a href="#pone-0000667-g003">Figure 3</a>) <a href="#pone.0000667-Ludwig1">[17]</a>, <a href="#pone.0000667-Garrity1">[18]</a>. Instead, the strains branched deeply within the <em>Proteobacteria</em> as shown in a phylogenetic tree created by maximum likelihood <a href="#pone.0000667-Olsen1">[19]</a>, <a href="#pone-0000667-g003">Figure 3</a>. Very similar tree topologies were found by neighbor-joining, maximum parsimony or minimum evolution methods using MEGA3 <a href="#pone.0000667-Kumar1">[20]</a>(data not shown). The novelty of the strain PV-1 within the <em>Proteobacteria</em> was confirmed by phylogenetic analysis of two other highly conserved proteins, gyrase beta-subunit (GyrB), and the recombination protein A (RecA). Phylogenetic trees for the GyrB and RecA proteins translated from the gene sequences are shown in <a href="#pone-0000667-g004">Figure 4a and 4b</a>, respectively. In both cases, PV-1 clustered within the <em>Proteobacteria</em>, but formed a distinct lineage from the other described classes of <em>Proteobacteria</em>.</p>
<div class="figure" id="pone-0000667-g003"><div class="img"><a name="pone-0000667-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Maximum-likelihood phylogenetic tree showing the evolutionary placement of <em>Mariprofundus ferrooxydans</em> (strains PV-1 and JV-1) belonging to the novel class of zeta-Proteobacteria along with representatives from other previously described classes of Proteobacteria.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Scale bar represents 5 nucleotide substitutions per 100 positions.</p>
<span>doi:10.1371/journal.pone.0000667.g003</span></div><div class="figure" id="pone-0000667-g004"><div class="img"><a name="pone-0000667-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Two maximum likelihood trees showing the phylogenetic relationship of <em>Mariprofundus ferrooxydans</em> PV-1 to other <em>Proteobacteria</em> based on conserved protein sequences.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Figures A and B show trees inferred from the amino acid sequences of the widely conserved RecA and GyrB proteins, respectively. The shaded clusters represent the recognized classes of <em>Proteobacteria</em>. The number of sequences representing each class is shown inside the cluster. The numbers at the nodes represent the per cent support for that group. The scale bar represents the estimated number of substitutions along the branches.</p>
<span>doi:10.1371/journal.pone.0000667.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>A BLAST search of GenBank using the SSU rRNA gene sequence from PV-1 revealed a group of environmental clones that clustered within the PV-1/JV-1 lineage, <a href="#pone-0000667-g003">Figure 3</a>. All these clones were from deep-sea marine habitats, primarily associated with hydrothermal activity. A T-RFLP (terminal restriction fragment length polymorphism) analysis of Fe-rich microbial mats collected at several different vent sites around the summit of Loihi in 2004 found that the PV-1 phylotype was consistently present at all the sampled sites, <a href="#pone-0000667-g005">Figure 5</a>. The vent water temperatures at these sites ranged from 10° to 65°C, and there were two main T-RF clusters, Loihi Group I that was associated with cooler vents &lt;40°C and Loihi Group 2 that was associated with vents &gt;40°C. The PV-1/JV-1 phylotype appeared most strongly associated with the Group I populations, <a href="#pone-0000667-g005">Figure 5</a>. Light microscopy indicated the presence of filamentous Fe oxyhydroxides similar to those formed by PV-1 at all these sites as well (data not shown).</p>
<div class="figure" id="pone-0000667-g005"><div class="img"><a name="pone-0000667-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000667" data-uri="info:doi/10.1371/journal.pone.0000667.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000667.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000667.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000667.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>UPGMA/Pearson product-moment cluster analysis of T-RFLP bacterial community fingerprints using eight restriction digest treatments from iron-dominated mats from Loihi Seamount including the isolate <em>Mariprofundus ferrooxydans.</em></span></strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>Two groups of communities have been identified and designated Loihi Group 1 (from lower temperature vents with enhanced ferrous iron) and Loihi Group 2 (from higher temperature vents with enhanced sulphide; (<a href="#pone.0000667-Davis1">[37]</a>). Triangles indicate T-RFs associated with <em>Mariprofundus ferrooxydans</em> from restriction digests with Hinf I and Msp I (176 bp and 449 bp, respectively). T-RF scales range from 50 to 500 base pairs. Numbers at the nodes indicate the cophenetic correlation value of the cluster. Dendrogram scale bar units are r value ×100.</p>
<span>doi:10.1371/journal.pone.0000667.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>A systematic analysis of these two microbes from microbial iron mats at Loihi Seamount indicates that they appear to be the first cultured isolates of a candidatus class of <em>Proteobacteria</em>, the “ζ-<em>Proteobacteria</em>”. The most conspicuous trait of these two strains is their obligate dependence upon Fe(II) as an energy source and microaerobic conditions. They are incapable of heterotrophic growth; neither can they grow on other inorganic energy sources such as reduced S-compounds or H<sub>2</sub>. These strains are also obligate aerobes, they do not oxidize Fe(II) when grown anaerobically with nitrate, nor can they reduce Fe(III) in the presence of acetate. The ability of these strains to grow on Fe<sup>0</sup> metal is of interest, as we are not aware of other reports of Fe-oxidizing bacteria growing on metallic iron, although there is evidence that there can be direct electron flow from Fe<sup>0</sup> to support the growth of methanogens and obligately anaerobic sulfate-reducing bacteria <a href="#pone.0000667-Dinh1">[21]</a>. Under aerobic conditions the reaction of iron metal is <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>Presumably the actual substrate for the FeOB is Fe(II) that is released upon chemical oxidation of the Fe(0); this process would be speeded up due to the ionic strength of seawater. Because these bacteria are growing under suboxic gradient conditions, O<sub>2</sub> may not always be present, in which case the oxidant of of Fe<sup>0</sup> may be water which results in hydrogen production: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000667.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>However, since these bacteria have been shown not to use H<sub>2</sub> as an energy source, their growth substrate will still be Fe(II). We have not yet demonstrated CO<sub>2</sub> fixation by either PV-1 or JV-1, however both grow well without any added organic C source. The draft genome sequence of PV-1 (NZ_AATS00000000) reveals it has a gene for the small subunit ribulose-1,5-bisphosphate carboxylase/oxygenase (Rubisco) protein, as well as genes for both form I and form II of the large subunit Rubisco, indicating that PV-1 has the genetic complement necessary for autotrophic growth.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>As was noted earlier, these marine Fe-oxidizers produce a unique filamentous stalk-like structure composed primarily of Fe-oxyhydroxides <a href="#pone.0000667-Emerson2">[13]</a>. More detailed electron micrographic analysis presented here indicates that the Fe-containing stalks are themselves composed of bundles of iron-rich fibrils. The processes of how these fibrils are excreted and their subcellular localization are currently under investigation. While it is presumed that an organic matrix is excreted by the cells that may serve as a ‘scaffold’ for the formation of Fe-oxides, the composition of the matrix is unknown. Presumably it contains acidic polysaccharides that play a role in metal binding and coordination <a href="#pone.0000667-Chan1">[22]</a>, <a href="#pone.0000667-Emerson3">[23]</a>. At a gross morphological level there appears to be a remarkable congruence between the filamentous Fe oxyhydroxides produced by strains PV-1 and JV-1 and the ferrihydrite-rich stalks produced by <em>Gallionella ferruginea</em> <a href="#pone.0000667-Hallbeck1">[24]</a>. <em>G. ferruginea</em> cells share a similar morphology to PV-1 and JV-1 cells, since they are also curved rods. The ferrihydrite-containing stalk of <em>G. ferruginea</em> is noted for its regular helical structure. Both PV-1 and JV-1 can produce twisted stalks, and at times these can appear helical; however as viewed by light microscopy, and shown in <a href="#pone-0000667-g002">Fig 2</a> (a–d) they do not appear to have the regular helicity that is often observed with <em>G. ferruginea</em>. The overall morphological similarity between <em>G. ferruginea</em> and the PV-1 and JV-1 strains suggests that reports of <em>Gallionella spp.</em> from deep-sea marine habitats may need to be re-evaluated.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It is also interesting to note that certain lithoautotrophic S-oxidizing bacteria that have been found at hydrothermal vents also may produce copious filamentous sulfur structures <a href="#pone.0000667-Taylor1">[25]</a>. The morphology of these sulfur filaments bears a notable resemblance to the iron stalks formed by strains PV-1 and JV-1; however, the organism identified with filamentous sulfur formation is an epsilon ε-<em>Proteobacteria, Candidatus</em> Arcobacter sulfidicus <a href="#pone.0000667-Sievert1">[26]</a>, and, as far as is known, does not oxidize iron. It will be interesting to determine if the mechanisms for deposition of iron-rich filaments and sulfur-rich filaments share any commonalities.</p>

<h4>Comparison to other neutrophilic FeOB</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Edwards <em>et al.</em> <a href="#pone.0000667-Edwards2">[27]</a> described two groups of marine FeOB that were most closely related to <em>Hyphomonas jannaschiana</em> and <em>Marinobacter aquaeolei</em>, which are α- and γ-<em>Proteobacteria</em>, respectively. These organisms can both grow heterotrophically using organic substrates, as well as lithoautotrophically, on Fe(II), although their doubling times are appreciably slower than either PV-1 or JV-1 on Fe(II). Neither of these organism forms a filamentous iron oxyhydroxide. Freshwater FeOB isolates described to date include <em>G. ferruginea</em>, as discussed above <a href="#pone.0000667-Hallbeck2">[28]</a>, FeOB TW2 <a href="#pone.0000667-Sobolev1">[29]</a>, and two strains isolated from groundwater, ES-1, and ES-2 <a href="#pone.0000667-Emerson4">[30]</a>; The latter two organisms have recently been described as ‘<em>Sideroxydans lithotrophicus</em>’ and ‘<em>Gallionella ferricapsiformans</em>’, novel members of the <em>β-Proteobacteria</em> <a href="#pone.0000667-Emerson5">[31]</a>. Another group of obligately lithotrophic FeOB has been isolated from the rhizosphere of wetland plants. These organisms are also members of the <em>β-Proteobacteria</em> and include a new species, ‘<em>Sideroxydans paludicola</em>’ and a novel genus ‘<em>Ferrotrophicum radicicola</em>’ <a href="#pone.0000667-Weiss1">[32]</a>. Like other freshwater isolates they do not grow with elevated salt concentrations, nor do they form morphologically unique Fe-oxides. Another differentiating feature of these marine FeOB is that the dominant fatty acid that was identified was the branched chain hydroxy 11:0 iso 3OH; this fatty acid was not found in the freshwater strains, instead the saturated straight chain fatty acids 14:0 and 16:0 were more prevalent in these isolates.</p>


<h4>Environmental Relevance</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Several clones from deep-sea environmental samples were the closest relatives to PV-1 based upon SSU rRNA gene sequences retrieved through a BLASTN search (<a href="#pone-0000667-g003">Figure 3</a>). For near full-length sequences (&gt;1200 bases), a clone from the Kebrit Deep (depth 1468m), in the Red Sea <a href="#pone.0000667-Eder1">[33]</a> was 95.4% similar (1373 bases) to PV-1 and a Loihi Seamount hydrothermal vent clone, Loihi Seamount PV<em>B</em> OTU4 <a href="#pone.0000667-Moyer1">[34]</a>, <a href="#pone.0000667-Moyer2">[35]</a> was 93.4% similar (1455 bases). In addition to the Loihi clone, other related SSU rRNA gene signatures have been observed from deep-sea hydrothermal systems, including the Guaymas Basin <a href="#pone.0000667-Dhillon1">[36]</a>, the Cleft Segment hydrothermal system off the coast of Oregon <a href="#pone.0000667-Davis1">[37]</a>, and the Mariana Trench in the western Pacific. With the exception of the Moyer <em>et al</em>. <a href="#pone.0000667-Moyer2">[35]</a>study that predated the current work at Loihi, none of the reports yielding related sequences were specifically investigating FeOB. However, the observation of sequences related to PV-1 from numerous hydrothermal systems suggests FeOB may be more widespread than is currently recognized <a href="#pone.0000667-Amend1">[38]</a>, <a href="#pone.0000667-Jannasch1">[39]</a>, <a href="#pone.0000667-McCollum1">[40]</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Observation of iron-oxide filaments similar to those made by PV-1 in deep-sea sediments have also been made at the Axial Volcano of the Juan de Fuca Ridge <a href="#pone.0000667-Kennedy1">[41]</a> and Vailul'u Volcano near Samoa <a href="#pone.0000667-Staudigel1">[7]</a>. Representatives of the <em>Mariprofundus</em> group have also been detected by T-RFLP and clone library analyses of microbial mats from N.W. Eifuku Volcano along the Mariana Island Arc, as well as the Cleft Segment of the Juan de Fuca Ridge and have been shown to co-occur with characteristic Fe-rich filaments <a href="#pone.0000667-Davis1">[37]</a>. Numerous samples of microbial mats collected at different times from different vent sites from Loihi also share predominant populations that correspond with those expected from the PV-1/JV-1 phylotype (data not shown). These findings corroborate the T-RFLP analysis presented here (<a href="#pone-0000667-g005">Figure 5</a>), that show the <em>Mariprofundus</em> phylotype exists at numerous vent sites at Loihi.</p>


<h4>Description of <em>Mariprofundus</em> gen. nov</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p><em>Mariprofundus</em> gen. nov. (Mar.is.pro.fund'us. L. masc. n. <em>maris</em> the sea; L. adj. <em>profundus</em> deep; L. masc. n. <em>Mariprofundus</em> a deep-sea organism).</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Cells are motile curved rods that appear Gram-negative by transmission electron microscopy. Growth is obligately lithotrophic and requires Fe<sup>2+</sup> as the energy source. Growth is oxygen dependent and requires marine salts.</p>


<h4>Description of Mariprofundus ferrooxydans sp. nov</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p><em>Mariprofundus ferrooxydans</em> (ferr.ox'y.dans. L. n. <em>ferrum</em> iron; Gr. adj. <em>oxys</em> sour; N.L. v. <em>oxydo</em> to sour, oxidize; N.L. part. adj. <em>ferrooxydans</em> iron-oxidizing).</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Displays the following properties in addition to those given by the genus description. Growth requires microaerobic conditions. The cells do not grow on other reduced inorganic energy sources besides iron or on organic compounds. As a result of growth filamentous stalk-like structures containing poorly crystalline iron oxyhydroxide are produced. The optimum growth temperature is 30°C and the optimum pH range is 6.0–6.5. The G+C content of the DNA of PV-1 is 54%. The cells are catalase negative. The dominant fatty acids are 11:0 iso 3OH, 17:0 iso 3OH, 16:0, and 18:1 iso H The type strain is PV-1<sup>T</sup>, ATCC™ BAA-1019 ( = JCM 14766) isolated from iron-rich microbial mats associated with regions of hydrothermal venting at Loihi Seamount in the Pacific Ocean. The class, order, and family description for this species are provided in the supplemental information (Supplemental <a href="#pone.0000667.s002">Text S1</a>).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Source and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Both isolates were enriched from Fe-rich mats associated with hydrothermal venting at Loihi Seamount as described previously <a href="#pone.0000667-Emerson2">[13]</a>. PV-1 came from an Fe-mat collected in 1996 associated with a cool (23°C), diffuse vent site, Naha Vents, located below the summit of the Loihi at a depth of 1325 m. JV-1 was isolated from an Fe-mat collected in 1998 that came from the lower jet at the Ikaika Vents (1298 m), a hot (165°C) vent located in a 300 m deep caldera close to the summit of Loihi. PV-1 was isolated directly from an inoculum of undiluted mat material, while JV-1 was enriched from a mat sample initially diluted to 10<sup>–7</sup>. Details of the isolation of JV-1 have been published previously and much the same techniques were used for the isolation of PV-1 <a href="#pone.0000667-Emerson2">[13]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The basal medium for all studies described here was artificial sea water (ASW) amended with nitrogen and phosphate sources according to the following formulation (per L deionized-H<sub>2</sub>O): 27.5 g NaCl, 6.78 g MgCl<sub>2</sub> 7H<sub>2</sub>O, 5.38 g MgCl<sub>2 </sub>· 6H<sub>2</sub>O, 1.4 g CaCl<sub>2</sub> · 2H<sub>2</sub>O, 1.0 g NH<sub>4</sub>Cl, 0.72 g KCl, 0.2 g NaHCO<sub>3</sub>, 0.05 g K<sub>2</sub>HPO<sub>4</sub> · 3H<sub>2</sub>O, 1.0 ml trace mineral supplement, and 1.0 ml vitamin supplement (both from American Type Culture Collection). Sterile aliquots of the vitamins and mineral solutions were added after the mineral salts medium was autoclaved at 121°C for 20 min. Filter sterilized sodium bicarbonate (10 mM final concentration) was added to provide buffering capacity and a carbon source.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Growth of the organisms was done under conditions of limited, but constant, supply of Fe(II) and O<sub>2</sub> using gradient tubes or plates, or bottle cultures. Details of these techniques have been published elsewhere <a href="#pone.0000667-Emerson6">[42]</a>. They all share in common the provision of Fe(II) concentrations in the 50–300 µM range and O<sub>2</sub> concentrations below 15% of ambient. One modification of the gradient plate technique was developed to test the ability of these strains to grow on zero valence iron (Fe<sup>0</sup>). This utilized autoclaved Fe<sup>0</sup> particles as an iron source instead of the agarose stabilized FeS layer. Approximately 100 mg of Fe<sup>0</sup> was added to 15 ml of ASW medium in a standard Petri plate, and the plate was gently swirled to distribute the Fe particles as uniformly as possible. The bacteria were inoculated into the liquid and the plates were incubated in Gaspak jars as previously described <a href="#pone.0000667-Emerson2">[13]</a>.</p>


<h4>Growth and phenotypic studies</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Tests for growth on individual C-sources for both strains were done as described before for JV-1 <a href="#pone.0000667-Emerson2">[13]</a>. Growth rates were determined in gradient plates. Twelve plates were inoculated and incubated in Gaspak jars at room temperature. Each day two plates were removed and the liquid medium (15 ml) from each plate was harvested into a 50 ml conical tube using a 10 ml pipet. The Fe-oxides were vortexed for 30 seconds and aliquots were removed for direct counts as described previously <a href="#pone.0000667-Emerson2">[13]</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The pH range for growth of each strain was tested by varying the buffers used in the top layer of the gradient tubes. Sodium bicarbonate (10 mM, gassed with CO<sub>2</sub>) was used to buffer at pH 5.0, 5.5, and 6.5; 5 mM N-Cyclohexyl-2-aminoethanesulfonic acid (HEPES), adjusted with NaOH, was used at pH 7.0 and 7.5. Gradient tubes were considered positive or negative for growth after one week by the formation of a tight band of growth at the oxic-anoxic interface; growth was confirmed as presence of cells after staining with Syto 13 (Molecular Probes) and viewing by epifluoresence microscopy.</p>


<h4>Electron Microscopy</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For standard transmission electron microscopy (TEM), PV-1 cells were grown on gradient plates with FeS to the late log phase of growth. The cells were harvested by centrifugation and resuspended in 20 mM HEPES buffer (pH 7.0) and fixed with 2.5% glutaraldehyde. The cells were then washed in 0.1 M sodium cacodylate buffer and fixed with 1% osmium tetroxide in the cacodylate buffer for 45 min on ice. Next the cells were washed with distilled water and stained with 1% aqueous uranyl acetate, then dehydrated in an ethanol series and embedded in Spurr resin. Thin sections were cut on an LKB Nova ultramicrotome, post-stained with uranyl acetate and lead citrate, and observed and photographed in a Zeiss EM-10CA TEM <a href="#pone.0000667-Bogan1">[43]</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>For high resolution transmission electron microscopy (HRTEM), a late log phase culture of PV-1 was grown in gradient plates and preserved with 2% gluteraldehyde. A few microliters of preserved sample was deposited onto a formvar-coated copper grid, rinsed with deionized water, and air-dried. The sample was coated with carbon and examined on a Philips CM200 transmission electron microscope, operated at an accelerating voltage of 200 kV.</p>


<h4>Fatty acids analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Fatty acid methyl esters (FAME) were identified using the procedure recommended by Microbial Identification System (MIDI, Sherlock Microbial Identification System Version 4.0, MIS Operating Manual, March 2001, Newark, DE). The cells were grown in gradient plates (100 ml total) until the late log phase of growth and then harvested by centrifugation. Excess Fe(III) was removed from the cell pellet by treating the sample with 0.33 M oxalic acid for one hour at 37°C, and then washing it three times by centrifugation with de-ionized water. The extraction of FAMEs and their analysis by gas chromatography was done as previously described <a href="#pone.0000667-Pikuta1">[44]</a>.</p>


<h4>Phylogenetic Analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>DNA was extracted from phosphate buffered (50 mM) saline (pH 8.0) washed pellets containing cells and iron oxides using the MoBio PowerSoil DNA Isolation kit. This minimal procedure produced DNA of high quantity and quality, which was not matched in previous studies <a href="#pone.0000667-Emerson2">[13]</a>, <a href="#pone.0000667-Emerson4">[30]</a> due to low cell counts (10<sup>7</sup> ml<sup>−1</sup>) and binding of DNA to iron oxide surfaces.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Universal primers 27F and 1492R were used to amplify the nearly full length SSU rRNA gene from the DNA extract. The PCR product was run on a 1% pre-cast SeaKem Gold Agarose gel (1× TBE buffer plus ethidium bromide from Cambrex Bio Science, ME). The specific band was excised from the gel and purified using a QIAquick Gel extraction kit (Qiagen Sciences, MD). The purified DNA was sequenced with a CEQ 8000 genetic analyzer (Beckman Coulter, CA).</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>For phylogenetic analysis, representative sequences were imported into the ARB software environment and aligned to the ARB database using the ARB fast aligner <a href="#pone.0000667-Ludwig2">[45]</a>. Additional sequences were selected in the database and exported to the sequence alignment program Bioedit <a href="#pone.0000667-Hall1">[46]</a> for additional alignment editing. Phylogenetic analyses were restricted to regions of moderately to highly conserved nucleotide positions that were unambiguously aligned for all sequences. Phylogenetic placements were calculated using fastDNAml version 1.2.2 <a href="#pone.0000667-Olsen1">[19]</a> using the general two-parameter model of evolution <a href="#pone.0000667-Kishino1">[47]</a> and allowing for the global swapping of branches. Using these parameters, the search for an optimal tree was repeated until the best log likelihood tree was calculated in at least three independent tree calculations. Each phylogenetic tree was bootstrapped 100 times allowing for the global swapping of branches. The search for each bootstrap was repeated until the best log likelihood score was calculated for at least two independent bootstrap calculations.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>Phylogenetic analysis of the DNA topoisomerase gyrase subunit B (GyrB) protein, and the bacterial recombination (RecA) protein was carried out by extracting the appropriate DNAsequences from the draft genome sequence of PV-1. The genome (NZ_AATS00000000) was sequenced by the J Craig Venter Institute (<a href="http://www.VenterInstitute.org">www.VenterInstitute.org</a>); details will be published elsewhere. The GyrB sequence was aligned with 53 homologous sequences from species in the <em>Proteobacteria</em>, using the L-INS-i option of the MAFFT program <a href="#pone.0000667-Katoh1">[48]</a>, <a href="#pone.0000667-Wallace1">[49]</a>. A tree was inferred from 725 homologous positions using the TREEFINDER algorithm (<a href="http://www.treefinder.de">www.treefinder.de</a>). The Dayhoff model of amino acid substitution was used and the discrete gamma distribution model, with four rate categories and an estimated alpha parameter, was implemented. Branch support was estimated using the LRSH approximation <a href="#pone.0000667-Shimodaira1">[50]</a> in TREEFINDER. The RecA sequence was aligned to 60 homologous sequences from the <em>Proteobacteria</em> and 317 positions were used to build the tree. The same software and parameters were used for aligning and inferring the tree. A list of the taxa used to construct both RecA and GyrB trees is provided in Supplementary <a href="#pone.0000667.s001">Table S1</a>.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>Repetitive element PCR (Rep-PCR) was carried out using the Diversilab system from Bacterial Barcodes (<a href="http://www.bacbarcodes.com">www.bacbarcodes.com</a>). PV-1 and JV-1 cells were grown in gradient plates, harvested, and subjected to DNA extraction as described above. The DNA was amplified using the <em>Stenotrophomonas</em> kit provided by Bacterial Barcodes, which contains a proprietary set of primers for repetitive elements common among <em>Proteobacteria</em>. The PCR products were separated on an Agilent 2100 Bioanalyzer, and a dendrogram comparing the new profiles to profiles from a database of samples was generated using the DiversiLab software from Bacterial Barcodes.</p>


<h4>T-RFLP analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Microbial mat samples were collected in 2004 using a suction sampling apparatus (Emerson and Moyer, 2002) either within the caldera or near the summit of Loihi Seamount from hydrothermal vent sites ranging in depth from 1150 to 1325 mbsl. Metagenomic DNA was extracted from these mat samples using the FastDNA Spin Kit following the manufacturer's protocol (Qbiogene, Irvine, CA). Extracted DNA was pooled, cleaned, and concentrated using Montáge PCR Centrifugal Filter Devices (Millipore, Bedford, MA). The DNA was then quantified using a NanoDrop spectrophotometer (Nanodrop Technologies, Wilmington, DE) and diluted to 10 ng·µl<sup>−1</sup> using filter sterilized 10 mM Tris, pH 8.0. Three replicate SSU rDNA amplifications were performed, each using 50 ng of metagenomic DNA and bacterial Domain specific primers 68F and 1492R with PCR conditions as previously described <a href="#pone.0000667-Emerson2">[13]</a>. The forward primer was labeled with 6-FAM (6-carboxyfluorescein) on the 5′ end. The amplicons were visually assayed for size by 1% agarose gel electrophoresis against a 1-kb ladder DNA size standard. Only reactions where corresponding negative controls yielded no amplification products were used. The remaining fluorescently-labeled PCR products were desalted and treated with eight tetrameric restriction enzymes as previously described in Chao et al. <a href="#pone.0000667-Chao1">[51]</a>. The fragments were separated by capillary electrophoresis using an ABI 3100 genetic analyzer using POP-6 with a 50&nbsp;cm capillary array (Applied Biosystems, Foster City, CA). The fluorescently labeled 5′ terminal-restriction fragments were sized against the Genescan ROX-500 internal size standard using GeneMapper v3.7 (Applied Biosystems). Only fragments between 50 and 500 nucleotides were included in the analysis as this size range has been show to have the highest degree of precision <a href="#pone.0000667-Engebretson1">[52]</a>. Resulting electropherograms were then imported into BioNumerics v4.61 (Applied Maths, Austin, TX). Community fingerprints were compared in BioNumerics using the Pearson product-moment correlation <a href="#pone.0000667-Hne1">[53]</a>and unweighted pair group method with arithmetic mean (UPGMA) cluster analysis. The cophenetic correlation coefficient was calculated to assess the robustness of the assigned clusters.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000667"></div><a name="pone.0000667.s001" id="pone.0000667.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000667.s001">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000667.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000667.s002" id="pone.0000667.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000667.s002">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000667.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000667.s003" id="pone.0000667.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000667.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">A time series of images showing the effect of treating a stalk with oxalic acid. The oxalate reduces the iron oxides in the stalk causing shrinkage and substantial reduction in size of the stalk; however a remnant of material remains. In this stalk there was no visual change after 10 minutes. No cell was present on this stalk. For this experiment, stalks from a fresh culture of PV-1 were viewed at 1,000× by phase contrast microscopy and a drop of 0.3 M oxalic acid was placed at the edge of the coverslip allowing the oxalate to diffuse under the coverslip and reduce the Fe-oxides. Photomicrographs were captured at the indicated times. The bar = 5 μm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000667.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.88 MB TIF)</p>
<a name="pone.0000667.s004" id="pone.0000667.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000667.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Rep-PCR comparison of strains PV-1 and JV-1 with freshwater FeOB strains ES-2 (<em>‘Gallionella capsiferriformans’</em>), ES-1 (<em>‘Sideroxydans lithotrophicus’</em>), and BrT (<em>‘Sideroxydans paludicola’</em>). <em>Stenotrophomonas maltophilia</em> is included as a control. In duplicated runs PV-1 and JV-1 shared &gt;98% similarity in their rep-PCR profiles. Rep-PCR profiles with % similarities &lt;95% usually indicate unrelated strains.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000667.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.08 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Paul Krader for providing FAME analysis, Cynthia Lydell for the Rep-PCR analysis, and Jason Cooper and Jeff Benson for 16S sequencing. Melissa Floyd provided technical assistance and is credited with the name <em>Mariprofundus</em>. We thank Ken Takai and an anonymous reviewer for helpful comments that improved the manuscript. We are especially grateful to the captain and crew of the R/V <em>Kaimikai-o-Kanaloa</em> and the Hawaiian Undersea Research Lab, including Terry Kerby, chief pilot of the DSRV <em>Pisces V</em>, who were all instrumental in sample collection at Loihi. HRTEM analysis was performed at the National Center for Electron Microscopy at the Lawrence Berkeley National Laboratory.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DE CM. Performed the experiments: DE JR RD HA CC. Analyzed the data: DE JR TL RD CM. Wrote the paper: DE JR TL CM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000667-Emerson1" id="pone.0000667-Emerson1"></a>Emerson D (2000)  Microbial oxidation of Fe(II) at circumneutral pH. In: Lovley DR, editor. Environmental Microbe-Metal Interactions. Washington, D.C.: ASM Press. pp. 31–52.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000667-Holland1" id="pone.0000667-Holland1"></a>Holland HD (2006) The oxygenation of the atmosphere and oceans. Phil Trans R Soc B  361: 903–915.  <ul class="find" data-citedArticleID="970828" data-doi="10.1098/rstb.2006.1838"><li><a href="http://dx.doi.org/10.1098/rstb.2006.1838" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+oxygenation+of+the+atmosphere+and+oceans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+oxygenation+of+the+atmosphere+and+oceans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000667-Holm1" id="pone.0000667-Holm1"></a>Holm NG (1987) Biogenic influences on the geochemistry of certain ferruginous sediments of hydrothermal origin. Chem Geol  63: 45–57.  <ul class="find" data-citedArticleID="970830" data-doi="10.1016/0009-2541(87)90073-8"><li><a href="http://dx.doi.org/10.1016/0009-2541(87)90073-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biogenic+influences+on+the+geochemistry+of+certain+ferruginous+sediments+of+hydrothermal+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biogenic+influences+on+the+geochemistry+of+certain+ferruginous+sediments+of+hydrothermal+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000667-Alt1" id="pone.0000667-Alt1"></a>Alt JC (1988) Hydrothermal oxide and nontronite deposits on seamounts in the Eastern Pacific. Marine Geology  81: 227–239.  <ul class="find" data-citedArticleID="970776" data-doi="10.1016/0025-3227(88)90029-1"><li><a href="http://dx.doi.org/10.1016/0025-3227(88)90029-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydrothermal+oxide+and+nontronite+deposits+on+seamounts+in+the+Eastern+Pacific." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydrothermal+oxide+and+nontronite+deposits+on+seamounts+in+the+Eastern+Pacific.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000667-Juniper1" id="pone.0000667-Juniper1"></a>Juniper SK, Fouquet Y (1988) Filamentous iron-silica deposits from modern and ancient hydrothermal sites. Canadian Mineralogist  26: 859–869.  <ul class="find" data-citedArticleID="970834"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Filamentous+iron-silica+deposits+from+modern+and+ancient+hydrothermal+sites.&amp;auth=&amp;atitle=Filamentous+iron-silica+deposits+from+modern+and+ancient+hydrothermal+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filamentous+iron-silica+deposits+from+modern+and+ancient+hydrothermal+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Filamentous+iron-silica+deposits+from+modern+and+ancient+hydrothermal+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000667-Juniper2" id="pone.0000667-Juniper2"></a>Juniper SK, Tebo BM (1995)  Microbe-metal interactions and mineral deposition at hydrothermal vents. In: Karl DM, editor. The Microbiology of Deep-sea Hydrothermal Vents. Boca Raton: CRC Press. pp. 219–253.  <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0000667-Staudigel1" id="pone.0000667-Staudigel1"></a>Staudigel H, Hart SR, Pile A, Bailey BE, Baker ET, et al.  (2006) Vailulu'u Seamount, Samoa: life and death on an active submarine volcano. Proc Natl Acad Sci U S A  103: 6448–6453.  <ul class="find" data-citedArticleID="970868" data-doi="10.1073/pnas.0600830103"><li><a href="http://dx.doi.org/10.1073/pnas.0600830103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vailulu%27u+Seamount%2C+Samoa%3A+life+and+death+on+an+active+submarine+volcano." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vailulu%27u+Seamount%2C+Samoa%3A+life+and+death+on+an+active+submarine+volcano.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000667-Thorseth1" id="pone.0000667-Thorseth1"></a>Thorseth IH, Torsvik T, Torsvik V, Daae FL, Pedersen R, et al.  (2001) Diversity of life in ocean floor basalt. Earth Planet Sci Lett  194: 31–37.  <ul class="find" data-citedArticleID="970874" data-doi="10.1016/s0012-821x(01)00537-4"><li><a href="http://dx.doi.org/10.1016/s0012-821x(01)00537-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+of+life+in+ocean+floor+basalt." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+of+life+in+ocean+floor+basalt.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000667-Bach1" id="pone.0000667-Bach1"></a>Bach W, Edwards KJ (2003) Iron and sulfide oxidation within the basaltic ocean crust: implications for chemolithoautotrophic microbial biomass production. Geochim Cosmochim Acta  67: 3871–3887.  <ul class="find" data-citedArticleID="970780" data-doi="10.1016/s0016-7037(03)00304-1"><li><a href="http://dx.doi.org/10.1016/s0016-7037(03)00304-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron+and+sulfide+oxidation+within+the+basaltic+ocean+crust%3A+implications+for+chemolithoautotrophic+microbial+biomass+production." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron+and+sulfide+oxidation+within+the+basaltic+ocean+crust%3A+implications+for+chemolithoautotrophic+microbial+biomass+production.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000667-Delaney1" id="pone.0000667-Delaney1"></a>Delaney JR, Kelley DS, Lilley MD, Butterfield DA, Baross JA, et al.  (1998) The quantum event of oceanic crustal accretion: impacts of diking at mid-ocean ridges. Science  281: 222–230.  <ul class="find" data-citedArticleID="970790" data-doi="10.1126/science.281.5374.222"><li><a href="http://dx.doi.org/10.1126/science.281.5374.222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+quantum+event+of+oceanic+crustal+accretion%3A+impacts+of+diking+at+mid-ocean+ridges." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+quantum+event+of+oceanic+crustal+accretion%3A+impacts+of+diking+at+mid-ocean+ridges.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000667-Fortin1" id="pone.0000667-Fortin1"></a>Fortin D, Langley S (2005) Formation and occurrence of biogenic iron-rich minerals. Earth-Science Rev  72: 1–19.  <ul class="find" data-citedArticleID="970816" data-doi="10.1016/j.earscirev.2005.03.002"><li><a href="http://dx.doi.org/10.1016/j.earscirev.2005.03.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Formation+and+occurrence+of+biogenic+iron-rich+minerals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Formation+and+occurrence+of+biogenic+iron-rich+minerals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000667-Edwards1" id="pone.0000667-Edwards1"></a>Edwards KJ, Bach W, McCollom TM, Rogers DR (2004) Neutrophilic iron-oxidizing bacteria in the ocean: Their habitats, diversity, and roles in mineral deposition, rock alteration, and biomass production in the deep-sea. Geomicrobiol J  21: 393–404.  <ul class="find" data-citedArticleID="970798" data-doi="10.1080/01490450490485863"><li><a href="http://dx.doi.org/10.1080/01490450490485863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutrophilic+iron-oxidizing+bacteria+in+the+ocean%3A+Their+habitats%2C+diversity%2C+and+roles+in+mineral+deposition%2C+rock+alteration%2C+and+biomass+production+in+the+deep-sea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutrophilic+iron-oxidizing+bacteria+in+the+ocean%3A+Their+habitats%2C+diversity%2C+and+roles+in+mineral+deposition%2C+rock+alteration%2C+and+biomass+production+in+the+deep-sea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000667-Emerson2" id="pone.0000667-Emerson2"></a>Emerson D, Moyer CL (2002) Neutrophilic Fe-oxidizing bacteria are abundant at the Loihi Seamount hydrothermal vents and play a major role in Fe oxide deposition. Appl Environ Microbiol  68: 3085–3093.  <ul class="find" data-citedArticleID="970804" data-doi="10.1128/aem.68.6.3085-3093.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.6.3085-3093.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutrophilic+Fe-oxidizing+bacteria+are+abundant+at+the+Loihi+Seamount+hydrothermal+vents+and+play+a+major+role+in+Fe+oxide+deposition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutrophilic+Fe-oxidizing+bacteria+are+abundant+at+the+Loihi+Seamount+hydrothermal+vents+and+play+a+major+role+in+Fe+oxide+deposition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000667-Stead1" id="pone.0000667-Stead1"></a>Stead DE, Hennessy J, Wilson J (1998) Modern methods for identifying bacteria. Plant Cell Tissue Organ Cult  52: 17–25.  <ul class="find" data-citedArticleID="970870" data-doi="10.1023/a:1005991706343"><li><a href="http://dx.doi.org/10.1023/a:1005991706343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modern+methods+for+identifying+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modern+methods+for+identifying+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000667-Yang1" id="pone.0000667-Yang1"></a>Yang L, Vauterin L, Vancanneyt M, Swings J, Kersters K (1993) Application of fatty acid methyl esters for taxonomic analysis of the genus <em>Xanthomonas</em>. Syst Appl Microbiol  16: 47–71.  <ul class="find" data-citedArticleID="970880" data-doi="10.1016/s0723-2020(11)80250-x"><li><a href="http://dx.doi.org/10.1016/s0723-2020(11)80250-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Application+of+fatty+acid+methyl+esters+for+taxonomic+analysis+of+the+genus+Xanthomonas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Application+of+fatty+acid+methyl+esters+for+taxonomic+analysis+of+the+genus+Xanthomonas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000667-Mergaert1" id="pone.0000667-Mergaert1"></a>Mergaert J, Cnockaert MC, Swings J (2003) <em>Thermomonas fusca</em> sp nov and <em>Thermomonas brevis</em> sp nov., two mesophilic species isolated from a denitrification reactor with poly(epsilon-caprolactone) plastic granules as fixed bed, and emended description of the genus <em>Thermomonas</em>. International Journal of Systematic and Evolutionary Microbiology  53: 1961–1966.  <ul class="find" data-citedArticleID="970852" data-doi="10.1099/ijs.0.02684-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.02684-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermomonas+fusca+sp+nov+and+Thermomonas+brevis+sp+nov.%2C+two+mesophilic+species+isolated+from+a+denitrification+reactor+with+poly%28epsilon-caprolactone%29+plastic+granules+as+fixed+bed%2C+and+emended+description+of+the+genus+Thermomonas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermomonas+fusca+sp+nov+and+Thermomonas+brevis+sp+nov.%2C+two+mesophilic+species+isolated+from+a+denitrification+reactor+with+poly%28epsilon-caprolactone%29+plastic+granules+as+fixed+bed%2C+and+emended+description+of+the+genus+Thermomonas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000667-Ludwig1" id="pone.0000667-Ludwig1"></a>Ludwig W, Klenk H-P (2001)  Overview: a phylogenetic backbone and taxonomic framework for prokaryotic systematics. In: Boone DR, Castenholtz RW, editors. Bergey's Manual of Systematic Bacteriology. Berlin: Springer-Verlag. pp. 49–65.  <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000667-Garrity1" id="pone.0000667-Garrity1"></a>Garrity GM, Holt JG (2001)  The road map to the manual. In: Garrity GM, editor. Bergey's manual of systematic bacteriology, vol 1: the <em>Archaea</em> and the deeply branching phototrophic <em>Bacteria</em>. New York: Springer. pp. 119–155.  <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0000667-Olsen1" id="pone.0000667-Olsen1"></a>Olsen GJ, Matsuda H, Hagstrom R, OVerbeek R (1994) fastDNAmL: a tool for construction of phylogenetic trees of DNA sequences using maximum likelihood. Comput Appl Biosci  10: 41–48.  <ul class="find" data-citedArticleID="970858" data-doi="10.1093/bioinformatics/10.1.41"><li><a href="http://dx.doi.org/10.1093/bioinformatics/10.1.41" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=fastDNAmL%3A+a+tool+for+construction+of+phylogenetic+trees+of+DNA+sequences+using+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22fastDNAmL%3A+a+tool+for+construction+of+phylogenetic+trees+of+DNA+sequences+using+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000667-Kumar1" id="pone.0000667-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for molecular evolutionary genetics analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="970844" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000667-Dinh1" id="pone.0000667-Dinh1"></a>Dinh HT, Kuever J, Mubmann M, Hassel AW, Stratmann M, et al.  (2004) Iron corrosion by novel anaerobic microorganisms. Nature  427: 829–832.  <ul class="find" data-citedArticleID="970794" data-doi="10.1038/nature02321"><li><a href="http://dx.doi.org/10.1038/nature02321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iron+corrosion+by+novel+anaerobic+microorganisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iron+corrosion+by+novel+anaerobic+microorganisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000667-Chan1" id="pone.0000667-Chan1"></a>Chan CS, De Stasio G, Welch SA, Girasole M, Frazer BH, et al.  (2004) Microbial polysaccharides template assembly of nanocrystal fibers. Science  303: 1656–1658.  <ul class="find" data-citedArticleID="970784" data-doi="10.1126/science.1092098"><li><a href="http://dx.doi.org/10.1126/science.1092098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+polysaccharides+template+assembly+of+nanocrystal+fibers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+polysaccharides+template+assembly+of+nanocrystal+fibers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000667-Emerson3" id="pone.0000667-Emerson3"></a>Emerson D, Ghiorse WC (1993) Ultrastructure and chemical composition of the sheath of <em>Leptothrix discophora</em> SP-6. J Bacteriol  175: 7808–7818.  <ul class="find" data-citedArticleID="970806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Ultrastructure+and+chemical+composition+of+the+sheath+of+Leptothrix+discophora+SP-6.&amp;auth=&amp;atitle=Ultrastructure+and+chemical+composition+of+the+sheath+of+Leptothrix+discophora+SP-6." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultrastructure+and+chemical+composition+of+the+sheath+of+Leptothrix+discophora+SP-6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultrastructure+and+chemical+composition+of+the+sheath+of+Leptothrix+discophora+SP-6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000667-Hallbeck1" id="pone.0000667-Hallbeck1"></a>Hallbeck EL, Pedersen K (2005)  Gallionella. In: Brenner DJ, Krieg NR, Staley JT, editors. Bergey's Manual of Systematic Bacteriology. 2nd ed. New York: Springer. pp. 880–886.  <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0000667-Taylor1" id="pone.0000667-Taylor1"></a>Taylor CD, Wirsen CO, Gaill F (1999) Rapid microbial production of filamentous sulfur mats at hydrothermal vents. Appl Environ Microbiol  65: 2253–2255.  <ul class="find" data-citedArticleID="970872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rapid+microbial+production+of+filamentous+sulfur+mats+at+hydrothermal+vents.&amp;auth=&amp;atitle=Rapid+microbial+production+of+filamentous+sulfur+mats+at+hydrothermal+vents." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+microbial+production+of+filamentous+sulfur+mats+at+hydrothermal+vents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+microbial+production+of+filamentous+sulfur+mats+at+hydrothermal+vents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000667-Sievert1" id="pone.0000667-Sievert1"></a>Sievert SM, Wieringa EBA, Wirsen CO, Taylor CD (2006) Growth and mechanism of filamentous-sulfur formation by <em>Candidatus</em> Arcobacter sulfidicus in opposing oxygen-sulfide gradients. Environ Microbiol.  9: 271–276.  <ul class="find" data-citedArticleID="970864" data-doi="10.1111/j.1462-2920.2006.01156.x"><li><a href="http://dx.doi.org/10.1111/j.1462-2920.2006.01156.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Growth+and+mechanism+of+filamentous-sulfur+formation+by+Candidatus+Arcobacter+sulfidicus+in+opposing+oxygen-sulfide+gradients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Growth+and+mechanism+of+filamentous-sulfur+formation+by+Candidatus+Arcobacter+sulfidicus+in+opposing+oxygen-sulfide+gradients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000667-Edwards2" id="pone.0000667-Edwards2"></a>Edwards KJ, Rogers DR, Wirsen CO, McCollom TM (2003) Isolation and characterization of novel psychrophilic, neutrophilic Fe-oxidizing, chemolithotrophic α− and γ-<em>Proteobacteria</em> from the deep sea. Appl Environ Microbiol  69: 2906–2913.  <ul class="find" data-citedArticleID="970800" data-doi="10.1128/aem.69.5.2906-2913.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.5.2906-2913.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+novel+psychrophilic%2C+neutrophilic+Fe-oxidizing%2C+chemolithotrophic+%26%23x3b1%3B%26%23x2212%3B+and+%26%23x3b3%3B-Proteobacteria+from+the+deep+sea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+novel+psychrophilic%2C+neutrophilic+Fe-oxidizing%2C+chemolithotrophic+%26%23x3b1%3B%26%23x2212%3B+and+%26%23x3b3%3B-Proteobacteria+from+the+deep+sea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000667-Hallbeck2" id="pone.0000667-Hallbeck2"></a>Hallbeck L, Ståhl F, Pedersen K (1993) Phylogeny and phenotypic characterization of the stalk-forming and iron-oxidizing bacterium <em>Gallionella ferruginea</em>. J Gen Microbiol  139: 1531–1535.  <ul class="find" data-citedArticleID="970824" data-doi="10.1099/00221287-139-7-1531"><li><a href="http://dx.doi.org/10.1099/00221287-139-7-1531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+and+phenotypic+characterization+of+the+stalk-forming+and+iron-oxidizing+bacterium+Gallionella+ferruginea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+and+phenotypic+characterization+of+the+stalk-forming+and+iron-oxidizing+bacterium+Gallionella+ferruginea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000667-Sobolev1" id="pone.0000667-Sobolev1"></a>Sobolev D, Roden EE (2004) Characterization of a neutrophilic, chemolithoautotrophic Fe(II)-ß-proteobacterium from freshwater wetland sediments. Geomicrobiol J  21: 1–10.  <ul class="find" data-citedArticleID="970866" data-doi="10.1080/01490450490253310"><li><a href="http://dx.doi.org/10.1080/01490450490253310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+neutrophilic%2C+chemolithoautotrophic+Fe%28II%29-%26%23xdf%3B-proteobacterium+from+freshwater+wetland+sediments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+neutrophilic%2C+chemolithoautotrophic+Fe%28II%29-%26%23xdf%3B-proteobacterium+from+freshwater+wetland+sediments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000667-Emerson4" id="pone.0000667-Emerson4"></a>Emerson D, Moyer C (1997) Isolation and characterization of novel iron-oxidizing bacteria that grow at circumneutral pH. Appl Environ Microbiol  63: 4784–4792.  <ul class="find" data-citedArticleID="970808"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+and+characterization+of+novel+iron-oxidizing+bacteria+that+grow+at+circumneutral+pH.&amp;auth=&amp;atitle=Isolation+and+characterization+of+novel+iron-oxidizing+bacteria+that+grow+at+circumneutral+pH." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+novel+iron-oxidizing+bacteria+that+grow+at+circumneutral+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+novel+iron-oxidizing+bacteria+that+grow+at+circumneutral+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000667-Emerson5" id="pone.0000667-Emerson5"></a>Emerson D, Rentz JA, Plaia T (2007) <em>Sideroxydans lithotrophicus</em>, gen. nov., sp. nov. and <em>Gallionella capsiferriformans</em> sp.nov., oxygen-dependent ferrous iron-oxidizing bacteria that grow at circumneutral pH. Int J Syst Evol Microbiol In Press.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0000667-Weiss1" id="pone.0000667-Weiss1"></a>Weiss JV, Rentz JA, Plaia T, Neubauer SC, Floyd MM, et al.  (2007) Characterization of neutrophilic Fe(II)-oxidizing bacteria isolated from the rhizosphere of wetland plants and description of <em>Ferritrophicum radicicola</em> gen. nov. sp. nov., and <em>Sideroxydans paludicola</em> sp. nov. Geomicrobiol J In Press.   <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0000667-Eder1" id="pone.0000667-Eder1"></a>Eder W, Jahnke LL, Schmidt M, Huber R (2001) Microbial diversity of the brine-seawater interface of the Kebrit Deep, Red Sea, studied via 16S rRNA gene sequences and cultivation methods. Appl Environ Microbiol  67: 3077–3085.  <ul class="find" data-citedArticleID="970796" data-doi="10.1128/aem.67.7.3077-3085.2001"><li><a href="http://dx.doi.org/10.1128/aem.67.7.3077-3085.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+diversity+of+the+brine-seawater+interface+of+the+Kebrit+Deep%2C+Red+Sea%2C+studied+via+16S+rRNA+gene+sequences+and+cultivation+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+diversity+of+the+brine-seawater+interface+of+the+Kebrit+Deep%2C+Red+Sea%2C+studied+via+16S+rRNA+gene+sequences+and+cultivation+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000667-Moyer1" id="pone.0000667-Moyer1"></a>Moyer CL, Dobbs FC, Karl DM (1994) Estimation of diversity and community structure through restriction fragment polymorphism distribution analysis of bacterial 16S rRNA genes from a microbial mat at an active, hydrothermal vent system, Loihi Seamount, Hawaii. Appl Environ Microbiol  60: 871–879.  <ul class="find" data-citedArticleID="970854"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Estimation+of+diversity+and+community+structure+through+restriction+fragment+polymorphism+distribution+analysis+of+bacterial+16S+rRNA+genes+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii.&amp;auth=&amp;atitle=Estimation+of+diversity+and+community+structure+through+restriction+fragment+polymorphism+distribution+analysis+of+bacterial+16S+rRNA+genes+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+diversity+and+community+structure+through+restriction+fragment+polymorphism+distribution+analysis+of+bacterial+16S+rRNA+genes+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+diversity+and+community+structure+through+restriction+fragment+polymorphism+distribution+analysis+of+bacterial+16S+rRNA+genes+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000667-Moyer2" id="pone.0000667-Moyer2"></a>Moyer CL, Dobbs FC, Karl DM (1995) Phylogenetic diversity of the bacterial community from a microbial mat at an active, hydrothermal vent system, Loihi Seamount, Hawaii. Appl Environ Microbiol  61: 1555–1562.  <ul class="find" data-citedArticleID="970856"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogenetic+diversity+of+the+bacterial+community+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii.&amp;auth=&amp;atitle=Phylogenetic+diversity+of+the+bacterial+community+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+diversity+of+the+bacterial+community+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+diversity+of+the+bacterial+community+from+a+microbial+mat+at+an+active%2C+hydrothermal+vent+system%2C+Loihi+Seamount%2C+Hawaii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000667-Dhillon1" id="pone.0000667-Dhillon1"></a>Dhillon A, Teske A, Dillon J, Stahl DA, Sogin ML (2003) Molecular characterization of sulfate-reducing bacteria in the Guaymas Basin. Appl Environ Microbiol  69: 2765–2722.  <ul class="find" data-citedArticleID="970792" data-doi="10.1128/aem.69.5.2765-2772.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.5.2765-2772.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+sulfate-reducing+bacteria+in+the+Guaymas+Basin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+sulfate-reducing+bacteria+in+the+Guaymas+Basin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000667-Davis1" id="pone.0000667-Davis1"></a>Davis RE, Carney T, Leal K, Moyer CL (2005) Spatial and temporal variability in microbial communities from pre- and post-eruption microbial mats collected from Loihi Seamount, Hawaii. Abstr Fall Mtg Eos Trans AGU  86: V51C–1508.  <ul class="find" data-citedArticleID="970788"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spatial+and+temporal+variability+in+microbial+communities+from+pre-+and+post-eruption+microbial+mats+collected+from+Loihi+Seamount%2C+Hawaii.&amp;auth=&amp;atitle=Spatial+and+temporal+variability+in+microbial+communities+from+pre-+and+post-eruption+microbial+mats+collected+from+Loihi+Seamount%2C+Hawaii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+and+temporal+variability+in+microbial+communities+from+pre-+and+post-eruption+microbial+mats+collected+from+Loihi+Seamount%2C+Hawaii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+and+temporal+variability+in+microbial+communities+from+pre-+and+post-eruption+microbial+mats+collected+from+Loihi+Seamount%2C+Hawaii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000667-Amend1" id="pone.0000667-Amend1"></a>Amend JP, Teske A (2005) Expanding frontiers in deep subsurface microbiology. Paleogeogra Paleoclimatol Palaeoecol  219: 131–155.  <ul class="find" data-citedArticleID="970778" data-doi="10.1016/j.palaeo.2004.10.018"><li><a href="http://dx.doi.org/10.1016/j.palaeo.2004.10.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expanding+frontiers+in+deep+subsurface+microbiology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expanding+frontiers+in+deep+subsurface+microbiology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000667-Jannasch1" id="pone.0000667-Jannasch1"></a>Jannasch HW, Mottl MJ (1985) Geomicrobiology of deep-sea hydrothermal vents. Science  229: 717–725.  <ul class="find" data-citedArticleID="970832" data-doi="10.1126/science.229.4715.717"><li><a href="http://dx.doi.org/10.1126/science.229.4715.717" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geomicrobiology+of+deep-sea+hydrothermal+vents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geomicrobiology+of+deep-sea+hydrothermal+vents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000667-McCollum1" id="pone.0000667-McCollum1"></a>McCollum TM (2000) Geochemical constraints on primary productivity in submarine hydrothermal vent plumes. Deep-Sea Res  47: 85–101.  <ul class="find" data-citedArticleID="970850" data-doi="10.1016/s0967-0637(99)00048-5"><li><a href="http://dx.doi.org/10.1016/s0967-0637(99)00048-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geochemical+constraints+on+primary+productivity+in+submarine+hydrothermal+vent+plumes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geochemical+constraints+on+primary+productivity+in+submarine+hydrothermal+vent+plumes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000667-Kennedy1" id="pone.0000667-Kennedy1"></a>Kennedy CB, Scott SD, Ferris FG (2003) Characterization of bacteriogenic iron oxide deposits from Axial Volcano, Juan de Fuca Ridge, Northeast Pacific Ocean. Geomicrobiol J  20: 199–214.  <ul class="find" data-citedArticleID="970840" data-doi="10.1080/01490450303873"><li><a href="http://dx.doi.org/10.1080/01490450303873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+bacteriogenic+iron+oxide+deposits+from+Axial+Volcano%2C+Juan+de+Fuca+Ridge%2C+Northeast+Pacific+Ocean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+bacteriogenic+iron+oxide+deposits+from+Axial+Volcano%2C+Juan+de+Fuca+Ridge%2C+Northeast+Pacific+Ocean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000667-Emerson6" id="pone.0000667-Emerson6"></a>Emerson D, Floyd MM (2005) Enrichment and isolation of iron-oxidizing bacteria at neutral pH. Methods Enzymol  397: 112–124.  <ul class="find" data-citedArticleID="970812" data-doi="10.1016/s0076-6879(05)97006-7"><li><a href="http://dx.doi.org/10.1016/s0076-6879(05)97006-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enrichment+and+isolation+of+iron-oxidizing+bacteria+at+neutral+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enrichment+and+isolation+of+iron-oxidizing+bacteria+at+neutral+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000667-Bogan1" id="pone.0000667-Bogan1"></a>Bogan BW, Sullivan WR, Kayser KJ, Derr KD, Aldrich HC, et al.  (2003) <em>Alkanindiges illinoisensis</em> gen. nov., sp. nov., an obligately hydrocarbonoclastic, aerobic squalene-degrading bacterium isolated from oilfield soils. Int J Syst Evol Microbiol  53: 1389–1395.  <ul class="find" data-citedArticleID="970782" data-doi="10.1099/ijs.0.02568-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.02568-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alkanindiges+illinoisensis+gen.+nov.%2C+sp.+nov.%2C+an+obligately+hydrocarbonoclastic%2C+aerobic+squalene-degrading+bacterium+isolated+from+oilfield+soils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alkanindiges+illinoisensis+gen.+nov.%2C+sp.+nov.%2C+an+obligately+hydrocarbonoclastic%2C+aerobic+squalene-degrading+bacterium+isolated+from+oilfield+soils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000667-Pikuta1" id="pone.0000667-Pikuta1"></a>Pikuta E, Hoover RB, Bej AK, Marsic D, Detkova EN, et al.  (2003) <em>Tindallia californiensis</em> sp. nov., a new anaerobic, haloalkalophilic, spore-forming acetogen isolated from Mono Lake in California. Extremophiles  7: 327–334.  <ul class="find" data-citedArticleID="970860" data-doi="10.1007/s00792-003-0326-7"><li><a href="http://dx.doi.org/10.1007/s00792-003-0326-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tindallia+californiensis+sp.+nov.%2C+a+new+anaerobic%2C+haloalkalophilic%2C+spore-forming+acetogen+isolated+from+Mono+Lake+in+California." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tindallia+californiensis+sp.+nov.%2C+a+new+anaerobic%2C+haloalkalophilic%2C+spore-forming+acetogen+isolated+from+Mono+Lake+in+California.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000667-Ludwig2" id="pone.0000667-Ludwig2"></a>Ludwig W, Strunk O, Westram R, Richter L, Meier H, et al.  (2004) ARB: a software environment for sequence data. Nucleic Acids Res  32: 1363–1371.  <ul class="find" data-citedArticleID="970848" data-doi="10.1093/nar/gkh293"><li><a href="http://dx.doi.org/10.1093/nar/gkh293" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ARB%3A+a+software+environment+for+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ARB%3A+a+software+environment+for+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000667-Hall1" id="pone.0000667-Hall1"></a>Hall TA (1999) BioEdit: A user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic Acids Symp Ser  41: 95–98.  <ul class="find" data-citedArticleID="970820"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=BioEdit%3A+A+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.&amp;auth=&amp;atitle=BioEdit%3A+A+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BioEdit%3A+A+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BioEdit%3A+A+user-friendly+biological+sequence+alignment+editor+and+analysis+program+for+Windows+95%2F98%2FNT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000667-Kishino1" id="pone.0000667-Kishino1"></a>Kishino H, Hasegawa M (1989) Evaluation of the maximum likelihood estimate of the evolutionary tree topologies from DNA sequence data, and the branching order in hominoidea. J Mol Evol  29: 170–179.  <ul class="find" data-citedArticleID="970842" data-doi="10.1007/bf02100115"><li><a href="http://dx.doi.org/10.1007/bf02100115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+maximum+likelihood+estimate+of+the+evolutionary+tree+topologies+from+DNA+sequence+data%2C+and+the+branching+order+in+hominoidea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+maximum+likelihood+estimate+of+the+evolutionary+tree+topologies+from+DNA+sequence+data%2C+and+the+branching+order+in+hominoidea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000667-Katoh1" id="pone.0000667-Katoh1"></a>Katoh K, Kuma K, Toh H, Miyata T (2005) MAFFT version 5: improvements in accuracy of multiple sequence alignment. Nucleic Acids Res  33: 511–518.  <ul class="find" data-citedArticleID="970838" data-doi="10.1093/nar/gki198"><li><a href="http://dx.doi.org/10.1093/nar/gki198" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAFFT+version+5%3A+improvements+in+accuracy+of+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAFFT+version+5%3A+improvements+in+accuracy+of+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000667-Wallace1" id="pone.0000667-Wallace1"></a>Wallace IM, Orla O, Higgins DG (2005) Evaluation of iterative alignment algorithms for multiple alignment. Bioinformatics  21: 1408–1414.  <ul class="find" data-citedArticleID="970876" data-doi="10.1093/bioinformatics/bti159"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti159" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+iterative+alignment+algorithms+for+multiple+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+iterative+alignment+algorithms+for+multiple+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000667-Shimodaira1" id="pone.0000667-Shimodaira1"></a>Shimodaira H, Hasegawa M (1999) Multiple comparisons of log-likelihoods with applications ot plylogenetic inference. Molec Biol Evol  16: 1114–1116.  <ul class="find" data-citedArticleID="970862" data-doi="10.1093/oxfordjournals.molbev.a026201"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+comparisons+of+log-likelihoods+with+applications+ot+plylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+comparisons+of+log-likelihoods+with+applications+ot+plylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000667-Chao1" id="pone.0000667-Chao1"></a>Chao LSL, Davis RE, Moyer CL (2007) Characterization of bacterial community structure in vestimentiferan tubeworm <em>Ridgeia piscesae</em> trophosomes. Mar Ecol  28: 1–14.  <ul class="find" data-citedArticleID="970786" data-doi="10.1111/j.1439-0485.2007.00151.x"><li><a href="http://dx.doi.org/10.1111/j.1439-0485.2007.00151.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+bacterial+community+structure+in+vestimentiferan+tubeworm+Ridgeia+piscesae+trophosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+bacterial+community+structure+in+vestimentiferan+tubeworm+Ridgeia+piscesae+trophosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000667-Engebretson1" id="pone.0000667-Engebretson1"></a>Engebretson JJ, Moyer CL (2003) Fidelity of select restriction endonucleases in determining microbial diversity by terminal-restriction fragment length polymorphism. Appl Environ Microbiol  69: 4823–4829.  <ul class="find" data-citedArticleID="970814" data-doi="10.1128/aem.69.8.4823-4829.2003"><li><a href="http://dx.doi.org/10.1128/aem.69.8.4823-4829.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fidelity+of+select+restriction+endonucleases+in+determining+microbial+diversity+by+terminal-restriction+fragment+length+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fidelity+of+select+restriction+endonucleases+in+determining+microbial+diversity+by+terminal-restriction+fragment+length+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000667-Hne1" id="pone.0000667-Hne1"></a>Häne BG, Jäger K, Drexler HG (1993) The Pearson product-moment correlation coefficient is better suited for identification of DNA fingerprint profiles than band matching algorithms. Electrophorhesis  14: 967–972.  <ul class="find" data-citedArticleID="970826" data-doi="10.1002/elps.11501401154"><li><a href="http://dx.doi.org/10.1002/elps.11501401154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pearson+product-moment+correlation+coefficient+is+better+suited+for+identification+of+DNA+fingerprint+profiles+than+band+matching+algorithms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pearson+product-moment+correlation+coefficient+is+better+suited+for+identification+of+DNA+fingerprint+profiles+than+band+matching+algorithms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.XML" value="101800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.PDF" value="292428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g001.PNG_L" value="16532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g001.PNG_M" value="48275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g001.PNG_S" value="8914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g001.TIF" value="21612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g001.PNG_I" value="17648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.t001.PNG_L" value="63771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.t001.PNG_M" value="81667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.t001.PNG_S" value="13603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.t001.TIF" value="316396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.t001.PNG_I" value="27350"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g002.PNG_L" value="539309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g002.PNG_M" value="237003"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g002.PNG_S" value="10017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g002.TIF" value="708074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g002.PNG_I" value="78185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g003.PNG_L" value="331730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g003.PNG_M" value="221957"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g003.PNG_S" value="11801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g003.TIF" value="1331976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g003.PNG_I" value="114736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g004.PNG_L" value="146007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g004.PNG_M" value="35192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g004.PNG_S" value="6098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g004.TIF" value="231494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g004.PNG_I" value="11880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g005.PNG_L" value="88465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g005.PNG_M" value="29282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g005.PNG_S" value="6037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g005.TIF" value="199208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.g005.PNG_I" value="10628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.e001.PNG" value="9275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.e001.TIF" value="25332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.e002.PNG" value="8968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.e002.TIF" value="25092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.s001.DOC" value="53248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.s002.DOC" value="20480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.s003.TIF" value="879856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000667.s004.TIF" value="1083150"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000667&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000667" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000667&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000667&volume=&issue=&title=A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities&author_name=David%20Emerson%2C%20Jeremy%20A.%20Rentz%2C%20Timothy%20G.%20Lilburn%2C%20Richard%20E.%20Davis%2C%20Henry%20Aldrich%2C%20Clara%20Chan%2C%20Craig%20L.%20Moyer&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000667" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667&amp;t=A%20Novel%20Lineage%20of%20Proteobacteria%20Involved%20in%20Formation%20of%20Marine%20Fe-Oxidizing%20Microbial%20Mat%20Communities" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667&title=A%20Novel%20Lineage%20of%20Proteobacteria%20Involved%20in%20Formation%20of%20Marine%20Fe-Oxidizing%20Microbial%20Mat%20Communities&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667&amp;title=A%20Novel%20Lineage%20of%20Proteobacteria%20Involved%20in%20Formation%20of%20Marine%20Fe-Oxidizing%20Microbial%20Mat%20Communities" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000667&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Novel Lineage of Proteobacteria Involved in Formation of Marine Fe-Oxidizing Microbial Mat Communities';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000667';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Novel%20Lineage%20of%20Proteobacteria%20Involved%20in%20Formation%20of%20Marine%20Fe-Oxidizing%20Microbial%20Mat%20Communities http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000667" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000667" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24172"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Fatty+acids%22" title="Search for articles in the subject area:'Fatty acids'"><div class="flagText">Fatty acids</div></a>
              <div data-categoryid="18501" data-articleid="24172"
                   data-categoryname="Fatty acids"
                   class="flagImage" title="Flag 'Fatty acids' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrothermal+vents%22" title="Search for articles in the subject area:'Hydrothermal vents'"><div class="flagText">Hydrothermal vents</div></a>
              <div data-categoryid="21099" data-articleid="24172"
                   data-categoryname="Hydrothermal vents"
                   class="flagImage" title="Flag 'Hydrothermal vents' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Marine+bacteria%22" title="Search for articles in the subject area:'Marine bacteria'"><div class="flagText">Marine bacteria</div></a>
              <div data-categoryid="43067" data-articleid="24172"
                   data-categoryname="Marine bacteria"
                   class="flagImage" title="Flag 'Marine bacteria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24172"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="24172"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Seamounts%22" title="Search for articles in the subject area:'Seamounts'"><div class="flagText">Seamounts</div></a>
              <div data-categoryid="50639" data-articleid="24172"
                   data-categoryname="Seamounts"
                   class="flagImage" title="Flag 'Seamounts' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="24172"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4613'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9619'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=132&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=20351">Publisher's Note: Correction</a><br>Posted by demerson2</p>
      <p><a href="/annotation/listThread.action?root=19321">Publisher's Note: Correction</a><br>Posted by demerson2</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
