

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001937"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001937" />

    <meta name="citation_title" content="Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3"/>
    <meta itemprop="name" content="Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3"/>

      <meta name="citation_author" content="Fengping Wang"/>
            <meta name="citation_author_institution" content="Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China"/>
            <meta name="citation_author_institution" content="Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China"/>
      <meta name="citation_author" content="Jianbin Wang"/>
            <meta name="citation_author_institution" content="Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China"/>
      <meta name="citation_author" content="Huahua Jian"/>
            <meta name="citation_author_institution" content="Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China"/>
            <meta name="citation_author_institution" content="Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China"/>
      <meta name="citation_author" content="Bing Zhang"/>
            <meta name="citation_author_institution" content="Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China"/>
      <meta name="citation_author" content="Shengkang Li"/>
            <meta name="citation_author_institution" content="Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China"/>
            <meta name="citation_author_institution" content="Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China"/>
      <meta name="citation_author" content="Feng Wang"/>
            <meta name="citation_author_institution" content="Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China"/>
            <meta name="citation_author_institution" content="Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China"/>
      <meta name="citation_author" content="Xiaowei Zeng"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China"/>
      <meta name="citation_author" content="Lei Gao"/>
            <meta name="citation_author_institution" content="The T-Life Research Center, Fudan University, Shanghai, People's Republic of China"/>
      <meta name="citation_author" content="Douglas Hoyt Bartlett"/>
            <meta name="citation_author_institution" content="Center for Marine Biotechnology and Biomedicine, Marine Biology Research Division, Scripps Institution of Oceanography, University of California, San Diego, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Jun Yu"/>
            <meta name="citation_author_institution" content="Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China"/>
      <meta name="citation_author" content="Songnian Hu"/>
            <meta name="citation_author_institution" content="Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China"/>
            <meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China"/>
      <meta name="citation_author" content="Xiang Xiao"/>
            <meta name="citation_author_institution" content="Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China"/>
            <meta name="citation_author_institution" content="Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001937.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1937"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Pressure effects on in vivo microbial processes.; citation_author=DH Bartlett; citation_journal_title=Biochim Biophys Acta; citation_volume=1595; citation_number=1; citation_pages=367-381; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The molecular biology of barophilic bacteria.; citation_author=C Kato; citation_author=DH Bartlett; citation_journal_title=Extremophiles; citation_volume=1; citation_number=2; citation_pages=111-116; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Microbial adaptations to the psychrosphere/piezosphere.; citation_author=DH Bartlett; citation_journal_title=J Mol Microbiol Biotechnol; citation_volume=1; citation_number=3; citation_pages=93-100; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Life at depth: Photobacterium profundum genome sequence and expression analysis.; citation_author=A Vezzi; citation_author=S Campanaro; citation_author=M D'Angelo; citation_author=F Simonato; citation_author=N Vitulo; citation_journal_title=Science; citation_volume=307; citation_number=4; citation_pages=1459-1461; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The psychrophilic lifestyle as revealed by the genome sequence of Colwellia psychrerythraea 34H through genomic and proteomic analyses.; citation_author=BA Methe; citation_author=KE Nelson; citation_author=JW Deming; citation_author=B Momen; citation_author=E Melamud; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=5; citation_pages=10913-10918; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the deep-sea gamma-proteobacterium Idiomarina loihiensis reveals amino acid fermentation as a source of carbon and energy.; citation_author=S Hou; citation_author=JH Saw; citation_author=KS Lee; citation_author=TA Freitas; citation_author=C Belisle; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=6; citation_pages=18036-18041; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Coping with cold: The genome of the versatile marine Antarctica bacterium Pseudoalteromonas haloplanktis TAC125.; citation_author=C Medigue; citation_author=E Krin; citation_author=G Pascal; citation_author=V Barbe; citation_author=A Bernsel; citation_journal_title=Genome Res; citation_volume=15; citation_number=7; citation_pages=1325-1335; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Polyphasic taxonomy of the genus Shewanella and description of Shewanella oneidensis sp. nov.; citation_author=K Venkateswaran; citation_author=DP Moser; citation_author=ME Dollhopf; citation_author=DP Lies; citation_author=DA Saffarini; citation_journal_title=Int J Syst Bacteriol; citation_volume=49 Pt 2; citation_number=8; citation_pages=705-724; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Bacterial manganese reduction and growth with manganese oxide as the sole electron acceptor.; citation_author=CR Myers; citation_author=KH Nealson; citation_journal_title=Science; citation_volume=240; citation_number=9; citation_pages=1319-1321; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the dissimilatory metal ion-reducing bacterium Shewanella oneidensis.; citation_author=JF Heidelberg; citation_author=IT Paulsen; citation_author=KE Nelson; citation_author=EJ Gaidos; citation_author=WC Nelson; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=10; citation_pages=1118-1123; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of 42 possible cytochrome C genes in the Shewanella oneidensis genome and characterization of six soluble cytochromes.; citation_author=TE Meyer; citation_author=AI Tsapin; citation_author=I Vandenberghe; citation_author=L de Smet; citation_author=D Frishman; citation_journal_title=Omics; citation_volume=8; citation_number=11; citation_pages=57-77; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global transcriptome analysis of the heat shock response of Shewanella oneidensis.; citation_author=H Gao; citation_author=Y Wang; citation_author=X Liu; citation_author=T Yan; citation_author=L Wu; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=12; citation_pages=7796-7803; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptome profiling of Shewanella oneidensis gene expression following exposure to acidic and alkaline pH.; citation_author=AB Leaphart; citation_author=DK Thompson; citation_author=K Huang; citation_author=E Alm; citation_author=XF Wan; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=13; citation_pages=1633-1642; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=14; citation_pages=D16-20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Correlation between phylogenetic structure and function: examples from deep-sea Shewanella.; citation_author=C Kato; citation_author=Y Nogi; citation_journal_title=FEMS Microbiol Ecol; citation_volume=35; citation_number=15; citation_pages=223-230; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pressure response in deep-sea piezophilic bacteria.; citation_author=C Kato; citation_author=MH Qureshi; citation_journal_title=J Mol Microbiol Biotechnol; citation_volume=1; citation_number=16; citation_pages=87-92; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Microbial diversity in a deep-sea Station of Pacific Nodule Province, Studied via 16S rRNA Gene Sequences and Extremophile Cultivation.; citation_author=MX Xu; citation_author=P Wang; citation_author=FP Wang; citation_author=X Xiao; citation_journal_title=Biodiv and Conserv; citation_volume=14; citation_number=17; citation_pages=3363-3380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Geomicrobiology: How Molecular-Scale Interactions Underpin Biogeochemical Systems.; citation_author=K Dianne; citation_author=JFB Newman; citation_journal_title=Science; citation_volume=296; citation_number=18; citation_pages=1071-1077; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Isolation of extremophiles with the detection and retrieval of Shewanella strains in deep-sea sediments from the west Pacific.; citation_author=F Wang; citation_author=P Wang; citation_author=M Chen; citation_author=X Xiao; citation_journal_title=Extremophiles; citation_volume=8; citation_number=19; citation_pages=165-168; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Shewanella psychrophila sp. nov., Shewanella piezotolerans sp. nov., two new Shewanella species from west Pacific deep-sea sediment.; citation_author=X Xiao; citation_author=P Wang; citation_author=X Zeng; citation_author=MX Chen; citation_author=DH Bartlett; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=57; citation_number=20; citation_pages=60-65; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Metal reduction and iron biomineralization by a psychrotolerant Fe(III)-reducing bacterium, Shewanella sp. strain PV-4.; citation_author=Y Roh; citation_author=H Gao; citation_author=H Vali; citation_author=DW Kennedy; citation_author=ZK Yang; citation_journal_title=Appl Environ Microbiol; citation_volume=72; citation_number=21; citation_pages=3236-3244; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Shewanella loihica sp. nov., isolated from iron-rich microbial mats in the Pacific Ocean.; citation_author=H Gao; citation_author=A Obraztova; citation_author=N Stewart; citation_author=R Popa; citation_author=JK Fredrickson; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=56; citation_number=22; citation_pages=1911-1916; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for symmetric chromosomal inversions around the replication origin in bacteria.; citation_author=JA Eisen; citation_author=JF Heidelberg; citation_author=O White; citation_author=SL Salzberg; citation_journal_title=Genome Biol; citation_volume=1; citation_number=23; citation_pages=RESEARCH0011; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Whole Proteome Prokaryote Phylogeny Without Sequence Alignment: A K-String Composition Approach.; citation_author=J Qi; citation_author=B Wang; citation_author=BL Hao; citation_journal_title=J Mol Evol; citation_volume=58; citation_number=24; citation_pages=1-11; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes.; citation_author=RL Tatusov; citation_author=ND Fedorova; citation_author=JD Jackson; citation_author=AR Jacobs; citation_author=B Kiryutin; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=25; citation_pages=41; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pressure-regulated biosynthesis of cytochrome bd in piezo- and psychrophilic deep-sea bacterium Shewanella violacea DSS12.; citation_author=H Tamegai; citation_author=H Kawano; citation_author=A Ishii; citation_author=S Chikuma; citation_author=K Nakasone; citation_journal_title=Extremophiles; citation_volume=9; citation_number=26; citation_pages=247-253; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of the marine planctomycete Pirellula sp. strain 1.; citation_author=FO Glockner; citation_author=M Kube; citation_author=M Bauer; citation_author=H Teeling; citation_author=T Lombardot; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=27; citation_pages=8298-8303; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Production of polyunsaturated fatty acids by polyketide synthases in both prokaryotes and eukaryotes.; citation_author=JG Metz; citation_author=P Roessler; citation_author=D Facciotti; citation_author=C Levering; citation_author=F Dittrich; citation_journal_title=Science; citation_volume=293; citation_number=28; citation_pages=290-293; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pseudouridines and pseudouridine synthases of the ribosome.; citation_author=J Ofengand; citation_author=A Malhotra; citation_author=J Remme; citation_author=NS Gutgsell; citation_author=M Del Campo; citation_journal_title=Cold Spring Harb Symp Quant Biol; citation_volume=66; citation_number=29; citation_pages=147-159; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pseudouridine in RNA: what, where, how, and why.; citation_author=M Charette; citation_author=MW Gray; citation_journal_title=IUBMB Life; citation_volume=49; citation_number=30; citation_pages=341-351; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Uracil content of 16S rRNA of thermophilic and psychrophilic prokaryotes correlates inversely with their optimal growth temperatures.; citation_author=AN Khachane; citation_author=KN Timmis; citation_author=VA dos Santos; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=31; citation_pages=4016-4022; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of thermal adaptation revealed from the genomes of the Antarctic Archaea Methanogenium frigidum and Methanococcoides burtonii.; citation_author=NF Saunders; citation_author=T Thomas; citation_author=PM Curmi; citation_author=JS Mattick; citation_author=E Kuczek; citation_journal_title=Genome Res; citation_volume=13; citation_number=32; citation_pages=1580-1588; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The ribosome: Structure, function, antibiotics and cellular interactions.; citation_author=J Ofengand; citation_author=KE Rudd; citation_number=33; citation_pages=175-189; citation_date=2000; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=The pseudouridine synthase RluD is required for normal ribosome assembly and function in Escherichia coli.; citation_author=NS Gutgsell; citation_author=MP Deutscher; citation_author=J Ofengand; citation_journal_title=RNA; citation_volume=11; citation_number=34; citation_pages=1141-1152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of lateral flagella gene transcription in Vibrio parahaemolyticus.; citation_author=R Belas; citation_author=M Simon; citation_author=M Silverman; citation_journal_title=J Bacteriol; citation_volume=167; citation_number=35; citation_pages=210-218; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Flagellar dynamometer controls swarmer cell differentiation of V. parahaemolyticus.; citation_author=L McCarter; citation_author=M Hilmen; citation_author=M Silverman; citation_journal_title=Cell; citation_volume=54; citation_number=36; citation_pages=345-351; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Filamentous phage assembly: variation on a protein export theme.; citation_author=M Russel; citation_author=NA Linderoth; citation_author=A Sali; citation_journal_title=Gene; citation_volume=192; citation_number=37; citation_pages=23-32; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A novel filamentous phage from a deep-sea bacterium Shewanella piezotolerans WP3 is induced at low temperature.; citation_author=F Wang; citation_author=FP Wang; citation_author=Q Li; citation_author=X Xiao; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=38; citation_pages=7151-7153; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Identification of genes regulated by changing salinity in the deep-sea bacterium Shewanella sp. WP3 using RNA arbitrarily primed PCR.; citation_author=S Li; citation_author=X Xiao; citation_author=J Li; citation_author=J Luo; citation_author=F Wang; citation_journal_title=Extremophiles; citation_volume=10; citation_number=39; citation_pages=97-104; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A complete sequence of the T. tengcongensis genome.; citation_author=Q Bao; citation_author=Y Tian; citation_author=W Li; citation_author=Z Xu; citation_author=Z Xuan; citation_journal_title=Genome Res; citation_volume=12; citation_number=40; citation_pages=689-700; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genome sequence of Salmonella enterica serovar Choleraesuis, a highly invasive and resistant zoonotic pathogen.; citation_author=CH Chiu; citation_author=P Tang; citation_author=C Chu; citation_author=S Hu; citation_author=Q Bao; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=41; citation_pages=1690-1698; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment.; citation_author=B Ewing; citation_author=L Hillier; citation_author=MC Wendl; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=42; citation_pages=175-185; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities.; citation_author=B Ewing; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=43; citation_pages=186-194; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Consed: a graphical tool for sequence finishing.; citation_author=D Gordon; citation_author=C Abajian; citation_author=P Green; citation_journal_title=Genome Res; citation_volume=8; citation_number=44; citation_pages=195-202; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria.; citation_author=JR Lobry; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=45; citation_pages=660-665; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER.; citation_author=AL Delcher; citation_author=D Harmon; citation_author=S Kasif; citation_author=O White; citation_author=SL Salzberg; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=46; citation_pages=4636-4641; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=47; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.; citation_author=TM Lowe; citation_author=SR Eddy; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=48; citation_pages=955-964; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The InterPro database, an integrated documentation resource for protein families, domains and functional sites.; citation_author=R Apweiler; citation_author=TK Attwood; citation_author=A Bairoch; citation_author=A Bateman; citation_author=E Birney; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=49; citation_pages=37-40; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The KEGG resource for deciphering the genome.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=S Kawashima; citation_author=Y Okuno; citation_author=M Hattori; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=50; citation_pages=D277-280; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=51; citation_pages=406-425; citation_date=1987; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3"/>
    <meta name="twitter:description" content="Shewanella species are widespread in various environments. Here, the genome sequence of Shewanella piezotolerans WP3, a piezotolerant and psychrotolerant iron reducing bacterium from deep-sea sediment was determined with related functional analysis to study its environmental adaptation mechanisms. The genome of WP3 consists of 5,396,476 base pairs (bp) with 4,944 open reading frames (ORFs). It possesses numerous genes or gene clusters which help it to cope with extreme living conditions such as genes for two sets of flagellum systems, structural RNA modification, eicosapentaenoic acid (EPA) biosynthesis and osmolyte transport and synthesis. And WP3 contains 55 open reading frames encoding putative c-type cytochromes which are substantial to its wide environmental adaptation ability. The mtr-omc gene cluster involved in the insoluble metal reduction in the Shewanella genus was identified and compared. The two sets of flagellum systems were found to be differentially regulated under low temperature and high pressure; the lateral flagellum system was found essential for its motility and living at low temperature."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001937.g008"/>

  <meta property="og:title" content="Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5807'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1248'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9263&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001937">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001937" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001937&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium <i>Shewanella piezotolerans</i> WP3
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fengping Wang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Fengping Wang, Jianbin Wang</p>

              
              

                <p>Affiliations:
                  Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China, 
                  Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jianbin Wang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Fengping Wang, Jianbin Wang</p>

              
              

                <p>Affiliations:
                  Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China, 
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huahua Jian, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China, 
                  Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bing Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China, 
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shengkang Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China, 
                  Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Feng Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China, 
                  Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaowei Zeng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lei Gao, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The T-Life Research Center, Fudan University, Shanghai, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Douglas Hoyt Bartlett, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Marine Biotechnology and Biomedicine, Marine Biology Research Division, Scripps Institution of Oceanography, University of California, San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jun Yu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China, 
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Songnian Hu
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:husn@genomics.org.cn">husn@genomics.org.cn</a> (SH); <a href="mailto:xxwfp@public.xm.fj.cn">xxwfp@public.xm.fj.cn</a> (XX)</p>

                <p>Affiliations:
                  Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, People's Republic of China, 
                  James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiang Xiao
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:husn@genomics.org.cn">husn@genomics.org.cn</a> (SH); <a href="mailto:xxwfp@public.xm.fj.cn">xxwfp@public.xm.fj.cn</a> (XX)</p>

                <p>Affiliations:
                  Key Laboratory of Marine Biogenetic Resources, State Oceanic Administration, Xiamen, People's Republic of China, 
                  Third Institute of Oceanography, State Oceanic Administration, Xiamen, People's Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001937</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001937-g001" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-t001" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g002" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g003" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g004" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g005" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g006" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g007" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-g008" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001937-t002" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001937">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001937" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">


    <div id="fch_corr" class="fch_corr">
      <a id="corrections" name="corrections" toc="corrections" title= "Corrections (1)"></a>
      <div class="corr_title">
        <span></span>
        <strong>Corrections</strong>
      </div>
      <div class="citation">
          <p>
            <span class="date"> 8 May 2008:  </span>
   
  <span class="citation_author">Wang
    F,
    </span>
   
  <span class="citation_author">Wang
    J,
    </span>
   
  <span class="citation_author">Jian
    H,
    </span>
   
  <span class="citation_author">Zhang
    B,
    </span>
   
  <span class="citation_author">Li
    S,
    </span>
  <span class="citation_author">et al. </span>
<span class="citation_date">(2008)</span>
<span
    class="citation_article_title">Correction: Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium <i>Shewanella piezotolerans</i> WP3.</span>
<span class="citation_journal_title">PLoS ONE</span><span
    class="citation_issue"> 3(5):</span>
<span class="citation_start_page">10.1371/annotation/744d7c8d-db8a-4ad4-aec5-62549b1dbefe.</span>
<span class="citation_doi">
 doi: 10.1371/annotation/744d7c8d-db8a-4ad4-aec5-62549b1dbefe</span>
              <span>|</span>
              <a href= "/article/info:doi/10.1371%2Fannotation%2F744d7c8d-db8a-4ad4-aec5-62549b1dbefe">View correction</a> </span>
          </p>
      </div>
    </div>





<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p><em>Shewanella</em> species are widespread in various environments. Here, the genome sequence of <em>Shewanella piezotolerans</em> WP3, a piezotolerant and psychrotolerant iron reducing bacterium from deep-sea sediment was determined with related functional analysis to study its environmental adaptation mechanisms. The genome of WP3 consists of 5,396,476 base pairs (bp) with 4,944 open reading frames (ORFs). It possesses numerous genes or gene clusters which help it to cope with extreme living conditions such as genes for two sets of flagellum systems, structural RNA modification, eicosapentaenoic acid (EPA) biosynthesis and osmolyte transport and synthesis. And WP3 contains 55 open reading frames encoding putative <em>c</em>-type cytochromes which are substantial to its wide environmental adaptation ability. The <em>mtr-omc</em> gene cluster involved in the insoluble metal reduction in the <em>Shewanella</em> genus was identified and compared. The two sets of flagellum systems were found to be differentially regulated under low temperature and high pressure; the lateral flagellum system was found essential for its motility and living at low temperature.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang F, Wang J, Jian H, Zhang B, Li S, et al.  (2008) Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium <em>Shewanella piezotolerans</em> WP3. PLoS ONE 3(4):
          e1937.
            doi:10.1371/journal.pone.0001937</p><p><strong>Editor: </strong>Christophe Herman, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> October 31, 2007; <strong>Accepted:</strong> February 28, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Wang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was financially supported by China COMRA foundation DYLY0202-03, NSFC fund (40625016, 40776095).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The deep sea, which is characterized by extremely low temperatures (&lt;5°C) and high pressures (up to 110 MPa), comprises the bulk of the world's oceans. Microorganisms are known to thrive in this extreme environment, however, knowledge of the adaptations of deep-sea microorganisms to the psychrosphere/piezosphere remains fragmentary <a href="#pone.0001937-Bartlett1">[1]</a>, <a href="#pone.0001937-Kato1">[2]</a>, <a href="#pone.0001937-Bartlett2">[3]</a>. Recently genome sequence information of bacteria from high pressure and/or cold environments (including <em>Photobacterium profundum</em> <a href="#pone.0001937-Vezzi1">[4]</a>, <em>Colwellia psychrerythraea</em> <a href="#pone.0001937-Methe1">[5]</a>, <em>Idiomarina loihiensis</em> <a href="#pone.0001937-Hou1">[6]</a> and <em>Pseudoalteromonas haloplanktis</em> <a href="#pone.0001937-Medigue1">[7]</a>) has indicated some of the relevant adaptation strategies. However, much more information is needed.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Members of the genus <em>Shewanella</em> inhabit various environments, and are well known for their versatile respiratory capabilities, coupling the oxidation of diverse substrates to the respiration of many different electron acceptors <a href="#pone.0001937-Venkateswaran1">[8]</a>. As a result of these properties, <em>Shewanella</em> strains have been proposed as candidates for bioremediation of metal and organic contaminants. Since potential clean up sites often include cold subsurface settings, it is essential to understand how these conditions influence <em>Shewanella</em> evolution, adaptation and ecology. One <em>Shewanella</em> species, <em>Shewanella oneidensis</em> MR-1, isolated from Oneida Lake sediments (NY, USA) <a href="#pone.0001937-Myers1">[9]</a> has been the subject of detailed genomic and proteomic investigations. The genome sequence of MR-1 has provided invaluable information about its respiratory capabilities <a href="#pone.0001937-Heidelberg1">[10]</a>, <a href="#pone.0001937-Meyer1">[11]</a>. However, still little is known about environmental acclimation or evolution within this genus <a href="#pone.0001937-Gao1">[12]</a>, <a href="#pone.0001937-Leaphart1">[13]</a>. At present, there are more than 18 <em>Shewanella</em> strains from various environments that have been sequenced by the Joint Genome Institute and other organizations <a href="#pone.0001937-Heidelberg1">[10]</a>, <a href="#pone.0001937-Benson1">[14]</a>. These sequences provide a powerful frame of reference for additional genomic analyses of new <em>Shewanella</em> isolates possessing distinctive phenotypes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p><em>Shewanella</em> strains are frequently the most abundant <em>Proteobacteria</em> in the benthos <a href="#pone.0001937-Kato2">[15]</a>, <a href="#pone.0001937-Kato3">[16]</a>, and in some deep-sea sediments, they dominate the bacterial community <a href="#pone.0001937-Xu1">[17]</a>, reflecting their critical roles in certain biogeochemical cycles of both organic and inorganic compounds <a href="#pone.0001937-Dianne1">[18]</a>. <em>Shewanella piezotolerans</em> WP3 (hereafter called WP3) is a bacterium isolated from a western Pacific Ocean sediment sample located at a water column depth of 1,914 m <a href="#pone.0001937-Wang1">[19]</a>, <a href="#pone.0001937-Xiao1">[20]</a>. It is psychrotolerant and piezotolerant, growing optimally at 15–20°C and with a broad pressure optimum extending from atmospheric pressure to about 20 MPa. Because of its relatively rapid growth and broad range of physical adaptations, it provides a particularly good model for understanding <em>Shewanella</em> adaptation to the deep ocean. Here, we present the results of whole genome sequence and related functional analysis of WP3. In addition, we compared the WP3 genome with other <em>Shewanella</em> genomes, in particular with <em>S. oneidensis</em> MR-1. This is the first detailed comparative genomic analysis of <em>Shewanella</em> species inhabiting dramatically different environments (shallow lake versus deep sea).</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>General genome features</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The WP3 genome consists of a single circular chromosome of 5,396,476 bp with 4,944 predicted genes (<a href="#pone-0001937-g001">Fig. 1</a>). WP3 has the largest genome size among the sequenced <em>Shewanella</em> genomes. General features of the WP3 genomes are presented in <a href="#pone-0001937-t001">Table 1</a>. Of these predicted proteins, 3,326 (67.3%) are similar to known proteins in current databases, 877 (17.7%) are conserved hypothetical proteins, and 741 (15.0%) are hypothetical proteins which have no database match (see Supplementary Table S1, especially the 554 hypothetical proteins of 30–100 amino-acid-in-length). The genome of WP3 is mostly related to <em>S. loihica</em> PV-4 genome (supplementary Tabel S2), a psychrotolerant bacterium isolated from iron-rich microbial mats at an active, deep-sea, hydrothermal Naha vent (1,325 m below sea level) located on the South Rift of Loihi Seamount, Hawaii <a href="#pone.0001937-Roh1">[21]</a>, <a href="#pone.0001937-Gao2">[22]</a>. WP3 and PV-4 have extensive regions of similar gene order (synteny), as revealed by an obvious “X” pattern when the two genomes are aligned (figure not shown); this pattern indicates that <em>Shewanella</em> genomes have undergone extensive inversions around the origin and terminus, as has been seen in other closely related bacteria <a href="#pone.0001937-Eisen1">[23]</a>.</p>
<div class="figure" id="pone-0001937-g001"><div class="img"><a name="pone-0001937-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Circular representation of the <em>Shewanella piezotolerans</em> WP3 genome.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>From the outside inward: the first and second circles show predicted protein-coding regions on the plus and minus strands, by biological role: translation/ribosome structure/biogenesis, pink; transcription, olive drab; DNA replication/recombination/repair, forest green; cell division/chromosome partitioning, light blue; posttranslational modification/protein turnover/chaperones, purple; cell envelop biogenesis, red; cell motility/secretion, plum; inorganic ion transport/metabolism, dark sea green; signal transduction mechanisms, medium purple; energy production/conversion, dark olive green; carbohydrate transport/metabolisms, gold; amino acid transport/metabolism, yellow; nucleotide transport/metabolism, orange; coenzyme transport/metabolism, tan; lipid transport/metabolism, salmon; secondary metabolites biosynthesis/transport/catabolism, light green; defense mechanism, black; general function prediction only, dark blue; function unknown and hypothetical protein, gray. The third circle shows G+C content percentage in relation to the mean G+C% for the chromosome. The fourth circle shows GC skew. The fifth circle shows insertion sequences, retrotransposons (black) and prophage (orange). The sixth and seventh circles show tRNA genes and rRNA operons, with blue or black on the plus strand and red on the minus one.</p>
<span>doi:10.1371/journal.pone.0001937.g001</span></div><div class="figure" id="pone-0001937-t001"><div class="img"><a name="pone-0001937-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>General features of two <em>Shewanella</em> genomes.</span></strong></p><span>doi:10.1371/journal.pone.0001937.t001</span></div>

<h4>Proteome phylogenetic analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We used a whole proteome phylogenetic analysis method which utilizes the oligopeptide content, i.e., frequency of amino acid K strings in the complete proteomes to infer the evolutionary relatedness of <em>Shewanella</em> species <a href="#pone.0001937-Qi1">[24]</a>. Our analysis includes 13 sequenced <em>Shewanella</em>: <em>S. baltica</em> OS185, OS155; <em>S. putrefaciens</em> CN-32; <em>S. sp.</em> W3-18-1; <em>S. amazonensis</em> SB2B; <em>S. oneidensis</em> MR-1; <em>S.</em> sp. ANA-3; MR-4; MR-7; <em>S. loihica</em> PV-4; <em>S. piezotolerans</em> WP3; <em>S. denitrificiens</em> OS217; <em>S. frigidimarina</em> NCIMB400. The whole proteome phylogenetic tree is similar, but not entirely consistent with the 16S rRNA gene phylogenetic tree, which is widely used for species phylogenetic relationship analysis (<a href="#pone-0001937-g002">Fig. 2A, B</a>). Both of the proteome and 16S rRNA gene phylogenetic analysis showed that strain WP3 and PV-4, OS217 and NCIMB400 are closest related, respectively. In the proteome tree, MR-1, ANA-3, MR-4, and MR-7 formed a cluster, while strain OS185, OS155, CN-32, W3-18-1 formed another cluster. On the other hand, there is no clear cluster seperation formed by these eight strains in the 16S rRNA gene tree. Both analyses indicate that <em>S. oneidensis</em> MR-1, which has been subjected to a detail genomic and proteomic investigation, evolved after the species PV4 and WP3 during the <em>Shewanella</em> genus evolution.</p>
<div class="figure" id="pone-0001937-g002"><div class="img"><a name="pone-0001937-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phylogenetic tree based on 16S rRNA (A) and whole proteome analysis (B).</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The phylogenetic tree of the 16S rRNA gene sequences from various <em>Shewnaella</em> species was constructed by neighbor-joining method using the programs of MEGA package. 1000 trial of bootstrap analysis was used to provide confident estimates for phylogenetic tree topologies. The phylogenetic tree based on the whole proteome sequences of the strains was constructed as described in the <a href="#s3">materials and methods</a> with 100 trial of bootstrap.</p>
<span>doi:10.1371/journal.pone.0001937.g002</span></div>

<h4>Functional classification and genome comparison</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As <em>S. oneidensis</em> MR-1 has been used for detailed genomic and proteomic investigations, the COG (Clusters of Orthologous Groups) <a href="#pone.0001937-Tatusov1">[25]</a> functional classification of WP3 was compared with that of MR-1. The data revealed that the percentage of genes involved in most functional categories in the two genomes are quite similar (P = 0.6815). WP3 has slight increase in genes involved in cell wall/membrane/envelope biogenesis (M), energy production and conversion (C), intracellular trafficking , secretion, and vesicular transport (U) and inorganic ion transport and metabolism (P), while <em>S. oneidensis</em> has more genes in replication, recombination and repair (L), cell cycle control, cell division, chromosome partitioning (D), signal transduction (T), posttranslational modification, protein turnover, chaperones (O), nucleotide transport and metabolism (F) and coenzyme transport and metabolism (H). There are many more unclassified genes in WP3 (1823, 36.9%) than <em>S. oneidensis</em> (1365, 30.5%). Comparing the 13 complete <em>Shewanella</em> genome sequences mentioned above, 1,591 common genes were found. These common genes are supposed to constitute the core genome of the <em>Shewanella</em> genus. Of all the shared genes, most belong to nucleic acid binding, nucleotide binding, hydrolase activity, transferase activity and oxidoreductase activity based on Gene Ontology term (<a href="#pone-0001937-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0001937-g003"><div class="img"><a name="pone-0001937-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Percentage of shared genes in <em>Shewanella</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The function classification of shared genes in 13 compared <em>Shewanella genomes</em> was based on Gene Ontology term. Of all common genes, those belong to nucleic acid binding, nucleotide binding, hydrolase activity, transferase activity and oxidoreductase activity are predominant.</p>
<span>doi:10.1371/journal.pone.0001937.g003</span></div>

<h4>Duplicated genes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Compared with other sequenced <em>Shewanella</em> genomes, the WP3 genome is the largest. We compared paralogous gene families (transposases excluded) in MR-1 and WP3 genomes. The results clearly indicated that to each pairs of corresponding gene families (paralog) in both genomes, WP3 usually has much more gene number in these families than MR-1has: WP3 contains 225 paralogous families which have more gene members (432 more genes) than their ortholog families in MR-1, while only 102 families of larger gene content (165 more genes) are present in MR-1. Supplementary Table S3 lists the expanded paralogous families in both genomes (having &gt;2 more genes in one paralogous gene family compared to the other and &gt;20% more genes of the average family member size; 31 versus 5 families are presented). In WP3, these expanded families are primarily involved in transport, secretion, energy metabolism and transcriptional regulation. The existence of additional duplicated genes could provide more capacity for coping with environmental change and less selective pressure because of their redundancy in WP3.</p>


<h4>Electron transport</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p><em>S. oneidensis</em> contains 42 putative cytochrome <em>c</em> genes which encode <em>c</em>-type cytochromes as constituents of its diversified respiratory network <a href="#pone.0001937-Meyer1">[11]</a>. WP3 also can use a variety of electron acceptors such as nitrate, fumarate, trimethylamine N-oxide (TMAO), dimethyl sulfoxide (DMSO), and insoluble metals during anaerobic growth <a href="#pone.0001937-Xiao1">[20]</a>. By careful pattern searching and BLAST analyses <a href="#pone.0001937-Meyer1">[11]</a>, the cytochrome <em>c</em> genes in WP3 genome were identified as listed in Table S4. Its genome contains 55 putative <em>c</em>-type cytochrome genes including 1 with 11 heme-binding sites, 11 with 10 heme-binding sites, 5 with 7 or 8 sites, and 12 with 4 or 5 sites. According to sequence similarities, the cytochromes <em>c</em> genes in WP3 could be divided into 17 groups as shown in Supplementary Table S5. WP3 contains many more decaheme cytochome <em>c</em> and nitrite reductase genes than MR-1 (17 and 5 versus 10 and 1, respectively), while it has less fumarate reductase genes than MR-1. Among the sequenced <em>Shewanella</em>, only the <em>S. denitrificiens</em> OS217 does not appear to be able to reduce metals. There are 10 hypothetical outer membrane lipoproteins in WP3 which are probably involved in the insoluble substance reduction. The <em>mtrABC</em>/<em>omcA</em> gene cluster which is known in MR1 to participate in the metal reduction is conserved in all the <em>Shewanella</em> genomes except OS217. <a href="#pone-0001937-g004">Fig. 4</a> shows the <em>mtrCAB</em> operon and its upstream and downstram genes in the <em>Shewanella</em> genomes. Upstream the <em>mtrBAC</em> operon, an <em>omcA</em> or hypothetical <em>omcA</em> like genes follow. Four types of <em>omcA</em> genes, here named as <em>omcA-1</em> to <em>omcA-4,</em> were identified. Strains CN32 and W3-18-1 contain an 11 heme-binding cytochrome <em>c</em> gene-<em>omcA2</em>, and both of these two strains together with NCIMB400 lack the <em>mtrDEF</em> which are conserved in some of the <em>Shewanella</em> genomes. WP3 also contains an 11 heme-binding <em>cytC</em> gene-swp3899 which is not involved in the <em>mtr</em>-<em>omc</em> gene cluster. The protein OmcA and MtrC are known to form a complex to reduce metal oxides <em>in vitro</em>. Upstream the <em>mtrC</em> gene, strain SB2B contains two hypothetical <em>omcA</em> genes, while strain PV4 and WP3 contain 3 hypothetical <em>omcA</em> like genes. Most of the gene cluster is preceded by a ferrous iron transporter gene cluster <em>feoAB</em> which is only absent in NCIMB400. Downstream the <em>mtrCAB</em> operon, there exists a phage SPO1 DNA polymerase domain protein (named as <em>spoP</em>) in most of the genomes. Reports have suggested that the membrane-based cytochrome respiratory system may not be fully functional at high pressure, and the expression of genes for some respiration components is regulated by hydrostatic pressure <a href="#pone.0001937-Methe1">[5]</a>, <a href="#pone.0001937-Tamegai1">[26]</a>. We are making <em>cytC</em> gene mutations to investigate the functions of the CytCs in WP3 living under different environments. The high number of cytochrome <em>c</em> genes in WP3 could be an adaptation to high pressure deep-sea environments.</p>
<div class="figure" id="pone-0001937-g004"><div class="img"><a name="pone-0001937-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The <em>mtr-omc</em> gene cluster and its surrounding genes.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>The <em>mtr-omc</em> gene clusters in several <em>Shewanella</em> genomes are illustrated. The conserved <em>mtrBAC</em> operon is boxed. <em>hypP</em> represents gene for hypothetical protein, <em>spoP</em> encodes for phage SPO1 DNA polymerase domain protein. 1, <em>Shewanella</em> sp. MR4; <em>S.</em> sp.ANA3; <em>S</em>. sp. MR7; <em>S. baltica</em> OS155; 2, <em>S. oneidensis</em> MR-1; <em>S. baltica</em> OS185; 3, <em>S.frigidimarina</em> NCIMB400; 4, <em>S.putrefaciens</em> CN32; 5, <em>S.</em> sp. W3-18-1; 6, <em>S. amazonensis</em> SB2B; 7, <em>S. Loihica</em> PV4; 8, <em>S.piezotolerans</em> WP3.</p>
<span>doi:10.1371/journal.pone.0001937.g004</span></div>

<h4>Metabolic pathways</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Compared to MR-1, WP3 has a great number of genes involved in various metabolic pathways (Supplementary Table S6). These include genes for the catabolism of starch and sucrose, pentose, fructose and mannose, ascorbate and aldarate, galactose, nucleotide sugars and aminosugars; glycan metabolism (e.g., glycosphingolipid and glycosaminoglycan); amino acid metabolism (e.g., arginine, proline and beta-alanine) amine and urea cycle metabolism; and the breakdown of assorted other compounds such as glycerolipids, porphyrins and chlorophyll. Besides these enzymes, there are also genes encoding specific transporters, transferases and regulators involved in the above pathways (Supplementary Table S7). WP3 inhabits an oligotrophic, cold and high pressure deep-sea environment. The existence of so many carbon and energy utilization pathways and osmolyte transport and synthesis systems is a reflection of the adaptation to this setting. Likewise, the absence of any light-activated photolyase genes indicates an existence free of ultraviolet light (in contrast <em>S. oneidenis</em> contains one deoxyribodipyrimidine photolyase gene). This is consistent with the absence of sunlight in the deep-sea environment WP3 inhabits.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>WP3 contains eight putative sulfatase genes, whereas <em>S. oneidensis</em> contains only one (SO4628, sulfatase). Bacterial sulfatases are known to function primarily in sulfur scavenging. Since marine systems are characterized by high inorganic sulfur concentrations, the limitation of this element would be expected to be rare where WP3 is found. However, sulfatases have been observed in other marine bacteria. The marine bacterium <em>Pirellula</em> sp. strain 1 harbors 110 putative sulfatase genes in its genome, which is the largest number known at the present time <a href="#pone.0001937-Glockner1">[27]</a>. It has been suggested that <em>Pirellula</em> sp. strain 1 uses the sulfatases to efficiently degrade sulfated glycopolymers to gain energy rather than to meet its sulfate requirements. The sulfatase genes in WP3 may have a similar function. In addition, there are three other WP3 specific genes, with four copies each, involved in sulfur metabolism (Supplementary Table S8). These appear to be sodium/sulphate symporters (COG0471), arylsulfatase regulators (Fe-S oxidoreductase, COG0641), and glutathione S- transferases (COG0625).</p>


<h4>WP3 Membrane Fluidity</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>A change in membrane fluidity is a well-established response of poikilothermic organisms to low temperature and high pressure exposure <a href="#pone.0001937-Bartlett2">[3]</a>. WP3 can synthesize eicosapentaenoic acid (EPA; 20:5), and its content in the cell membrane increases as a function of the decreased temperatures <a href="#pone.0001937-Wang1">[19]</a>. WP3 harbors a gene cluster (WP3546–3554) for eicosapentaenoic acid (EPA; 20:5) omega-3 polyunsaturated fatty acid synthesis. Five genes in this gene cluster have been shown essential for EPA synthesis <a href="#pone.0001937-Metz1">[28]</a>. Mutation of the orf2 gene which encodes the putative phosphopantetheinyl transferase (PPTase) diminished the EPA synthesizing capability of WP3, and the Δorf2 WP3 strain showed much slower growth rate under low temperature or high pressure compared with the wild type strain (<a href="#pone-0001937-g005">Fig. 5</a>). On the other hand, in the deep-sea bacterium <em>Photobacterium profundum</em> SS9, it has been observed that EPA is not essential for the growth of SS9 at low temperature and high pressure. We are still not clear why EPA plays different roles in the environmental adaptation of WP3 and SS9. Comparison of the genome of WP3 with those of PV-4, <em>Desulfotalea psychrophila</em> Lsv54, <em>Colwellia psychrerythraea</em> 34H and <em>Photobacterium profundum</em> SS9, all of which are cold-adapted bacteria, only indicated the presence of a few features in common: WP3, PV-4, 34H and SS9 all contain the EPA synthesis gene cluster, and Lsv54 contains two copies of β-keto-acyl-CoA synthase I, which elongates unsaturated fatty acids. This further illustrates that maintaining membrane physical structure is crucial for life in cold and/or high pressure environments, and this can be achieved by increasing the ratio of unsaturated fatty acids.</p>
<div class="figure" id="pone-0001937-g005"><div class="img"><a name="pone-0001937-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Growth curve of WP3 and its mutant.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>The growth of WP3 and the WP3Δorf2 mutant under different temperature and pressure conditions were monitered by measuring the optical density of the cultutes at 0D<sub>600</sub>. For each measurement, triplicate cultures were measured simultaneously, and the average value was recorded. A: Growth of the strains at 0.1 MPa and 20 MPa. Wild type WP3 was grown at 0.1 MPa (□), or at 20 MPa (▪); WP3Δorf2 mutant grown at 0.1 MPa (▴) or 20 MPa(Δ); B: Growth of the strains at 4°C and 20°C. Wild type WP3 was grown at 4°C (□), or at 20°C (▪); WP3Δorf2 mutant grown at 4°C (▴) or 20°C (Δ).</p>
<span>doi:10.1371/journal.pone.0001937.g005</span></div>

<h4>tRNA, rRNA modifications</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>WP3 contains seven pseudouridine synthase genes (COG1187), nine pseudouridylate synthase genes (COG0564) and three genes (COG0101, COG0130, COG4445) for RNA modification <a href="#pone.0001937-Ofengand1">[29]</a>, <a href="#pone.0001937-Charette1">[30]</a> (Supplementary Table S9). In general, bacteria only have 1–4 copies of these genes; this is the first report of so many pseudouridine synthase genes in a single bacteria genome. The high number of pseudouridine synthase genes was also observed in several of the <em>Shewanella</em> genomes including PV4, MR-4, MR-7, CN-32, SB2B, OS155, 0S217, but absent in the MR1 genome. Khachane <em>et al</em> reported that the uracil content of 16S rRNA in thermophilic and psychrophilic prokaryotes increase as their optimal growth temperatures drop <a href="#pone.0001937-Khachane1">[31]</a>. Previous studies have also indicated that the maintenance of flexibility at low temperatures is important for tRNA function, and this is mainly achieved through post-transcriptional incorporation of dihydrouridine in psychrophilic bacteria <a href="#pone.0001937-Saunders1">[32]</a>. Cellular RNA contains a number of post-transcriptionally modified nucleosides, the most common of which is pseudouridine (ψ). Pseudouridine has been identified in tRNAs, rRNAs, snRNAs and/or snoRNAs for which the stability of their tertiary structure is important for function <a href="#pone.0001937-Ofengand1">[29]</a>, <a href="#pone.0001937-Ofengand2">[33]</a>. Pseudouridine synthases are the enzymes responsible for formation of the ψ residues. In <em>E. coli</em>, the pseudouridine synthase RluD is responsible for the formation of ψ1911, ψ1915, and ψ1917 in 23S rRNA. Disruption of the <em>rluD</em> gene and/or loss of the pseudouridine residues for which it is responsible resulted in a severe growth phenotype. RluD and/or the ψ1911, ψ1915, and ψ1917 residues formed by it are necessary for normal ribosome biogenesis, assembly, and function in <em>E. coli</em> <a href="#pone.0001937-Ofengand1">[29]</a>, <a href="#pone.0001937-Gutgsell1">[34]</a>. The large number of genes for structural RNA modification in so many <em>Shewanella</em> genomes may be one of the important mechanisms for the wide environmental adaptation and distribution of the genus.</p>


<h4>Cell motility</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Two sets of flagella genes are present in the WP3 genome (flagella I: swp1493-swp1550, flagella II: swp5082-swp5126, <a href="#pone-0001937-g006">Fig. 6</a>). Flagella I contains the <em>flgM, flgN, fliJ</em> and <em>fliO</em> genes, which are missing in the flagella II gene cluster. Two stator proteins, MotY, MotAB (swp5089, swp5126,5127), responsible for converting proton motive force into torque against the rotor/switch, are found to be encoded in the flagella II gene cluster. No motor protein genes were found near the Fla I gene cluster area. However, two clustered genes, swp3615 and swp3616, which code for putative MotB and MotA proteins are found in the genome far away from both of the flagella gene clusters. These two genes could be responsible for the rotor/switch function of Fla I or both. Sequence comparison results suggest that the flagella I and II in WP3 correspond to the polar and lateral flagella of <em>V. parahaemolyticus</em> <a href="#pone.0001937-Belas1">[35]</a>. The polar flagellum propels the bacterium in a liquid environment (swimming), while the lateral flagella are responsible for movement over surfaces or viscous environments (swarming) <a href="#pone.0001937-McCarter1">[36]</a>. The WP3 cells in the liquid cultures only contains a single polar flagellum as observed by transmission electronic microscope (<a href="#pone-0001937-g007">Fig. 7A</a>), while the lateral flagella was induced when cultivated on agar plates (<a href="#pone-0001937-g007">Fig. 7B</a>). The two flagella systems could facilitate the movement of WP3 cells in sediment pore fluids or in seawater from planktonic to surface-associated lifestyles. Interestingly, the two sets of flagellum systems were found to be inversely regulated by low temperature and high pressure (our unpublished microarray data). The differential regulation of the flagellum by temperature and pressure was verified by quantitative RT-PCR (<a href="#pone-0001937-g007">Fig. 7C, D</a>). As shown in the figure, the genes of lateral flagellum were highly up regulated by low temperature (4°C versus 20°C) and slightly repressed by high pressure (20 MPa versus 0.1 MPa); while the genes of polar flagellum were largely repressed at low temperature and highly activated at high pressure. Mutants which have defects in polar or lateral flagella synthesis (WP3Δ1508 defect in polar flagellum; WP3Δ5118, WP3Δ5125 defect in lateral flagellum) showed decreased motility on 0.3% agar plates at 20°C (<a href="#pone-0001937-g008">Fig. 8A</a>). The mutant lacking lateral flagella completely lost motility at 4°C, while the mutant with a defect in the polar flagellum remained motile (<a href="#pone-0001937-g008">Fig. 8B</a>). This clearly indicated that the lateral flagellum is essential for the motility of the strain at low temperature. Strains WP3Δ5118 and WP3Δ5125 showed decreased growth rate compared to the wild type when grown at 4°C (<a href="#pone-0001937-g008">Fig. 8C</a>), while no growth arrest was observed at 20°C. This is the first demonstration of a requirement for a second flagellum system for growth at low temperature. The deep-sea bacterium <em>Photobacterium profundum</em> strain SS9 also has two sets of flagella genes <a href="#pone.0001937-Vezzi1">[4]</a> and thus the results presented here may extend to multiple genera of deep-sea microbes. The nature of the connection between swimming speed and anabolism and cell division are unknown and will require further investigation.</p>
<div class="figure" id="pone-0001937-g006"><div class="img"><a name="pone-0001937-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Gene organization map of two flagellum systems in WP3.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>A: Flagellum gene cluster I (polar): swp1493—swp1550; B: Flagellum gene cluster II (lateral): swp5082—swp5126. Arrow indicates gene transcription direction.</p>
<span>doi:10.1371/journal.pone.0001937.g006</span></div><div class="figure" id="pone-0001937-g007"><div class="img"><a name="pone-0001937-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>TEM image of polar (A), lateral ( B) flagella and the transcriptional regulation of genes in flagellum synthesis under different pressures (C) and temperatures (D).</span></strong></p><a id="article1.body1.sec2.sec9.fig2.caption1.p1" name="article1.body1.sec2.sec9.fig2.caption1.p1"></a><p>The WP3 cells from liquid culture and from agar plates were visualized by TEM separately. The cells from liquid culture were shown to have polar flagellum (A); while the cells from ager plates also contain lateral flagellum (B). The transcription level of four genes (swp1499, swp1511, swp1508, swp1519) for polar flagellum synthesis and four genes (swp5109, swp5114, swp5118, swp5125) for lateral flagellum synthesis under different temperatures and pressures were checked by Quantitative RT-PCR as described in the <a href="#s3">materials and methods</a>. C: The transcription changes of genes at 20 MPa as compared with that at 0.1 MPa. The transcription levels of the genes at 0.1 MPa are set as control; D: The transcription change of genes at 4°C as compared with that at 20°C. The transcription levels of the genes at 20°C are set as control</p>
<span>doi:10.1371/journal.pone.0001937.g007</span></div><div class="figure" id="pone-0001937-g008"><div class="img"><a name="pone-0001937-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Motility and growth assays of the WP3 strains.</span></strong></p><a id="article1.body1.sec2.sec9.fig3.caption1.p1" name="article1.body1.sec2.sec9.fig3.caption1.p1"></a><p>The motility assays for WP3 and the mutants (WP3Δ1508 has defect in polar flagellum; WP3Δ5118, WP3Δ5125 defect in lateral flagellum) were conducted on 0.3% agar plates at 20°C (A) or 4°C (B). Triplicates of each strain were spotted onto the soft agar, the motile ability of the strains were compared by observing the movements of the strains after 3 days at 20°C, or 7 days at 4°C. The growth of WP3 and the mutants at 4°C was monitered by measuring the optical density of the cultutes at 0D<sub>600</sub>. The mean values with standard deviations (indicated by vertical bars) from triplicate experiments are given. Δ represents the wild type WP3; • mutant WP3Δ1508; ○WP3Δ5118 ▾WP3Δ5125.</p>
<span>doi:10.1371/journal.pone.0001937.g008</span></div>

<h4>WP3 Phage</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>The WP3 genome includes only one prophage, a small 7,718 bp filamentous phage genome (named SW1), while MR-1 contains three prophages including one lambda-like phage (LambdaSo, 51,857 bp) and two phylogenetically distinct phages related to the <em>E. coli</em> Mu (MuSo1, 34,551 bp and MuSo2, 35,666 bp). Phage SW1 is integrated into the terminus of the chromosome and also exits as an extra-chromosomal plasmid. It contains nine putative open reading frames (ORF) as listed in <a href="#pone-0001937-t002">Table 2</a>. This is the first filamentous phage found in a <em>Shewanella</em> species and could be a useful genetic engineering tool. Filamentous phages are unlike most other bacterial viruses in that the infected host remains viable and phages are continuously assembled and extruded from the cell in a concerted process <a href="#pone.0001937-Russel1">[37]</a>. Most of the sequences of the SW1 phage ORFs share low but significant similarities with those of <em>Vibrio</em> filamentous phages. SW1 could be assigned to genus <em>Inovirus</em> of the family <em>Inoviridae</em> of bacteriophages. The SW1 prophage is flanked by 13 base pair (<span class="gene">GAATGCGCACAAT</span>) direct repeats. Viable filamentous phage was successfully isolated from WP3 and it was found significantly induced and released at low temperature <a href="#pone.0001937-Wang2">[38]</a>. WP3 lives in the permanent cold deep-sea environments, at which condition the filamentous phage is highly induced and released into the environments. The significance of this character for the species environmental adaptation and evolution is still unclear. Similarity searches among sequenced <em>Shewanella</em> genomes indicated that two strains, <em>Shewanella baltica</em> OS155 and <em>Shewanella putrifaciens</em> CN-32, contain sequences similar to those of filamentous phage. These two DNA fragments are adjacent to the phage integrase gene, suggesting that they are derived through phage mediated horizontal gene transfer. The roles of filamentous phage in <em>Shewanella</em> evolution await further clarification.</p>
<div class="figure" id="pone-0001937-t002"><div class="img"><a name="pone-0001937-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001937" data-uri="info:doi/10.1371/journal.pone.0001937.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001937.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001937.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001937.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001937.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001937.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Putative ORFs in the SW1 phage sequence and the BLAST search results.</span></strong></p><span>doi:10.1371/journal.pone.0001937.t002</span></div>

<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>The complete genome sequence of the deep-sea bacterium <em>Shewanella piezotolerans</em> WP3 reflects a capacity for coping with diverse energy sources and variable physical conditions. The WP3 genome contains genes or gene clusters which enhance its capacity for nutrient acquisition, energy production, macromolecule synthesis and protein function under deep-sea conditions. These include genes for two sets of flagellum systems, structural RNA modification, functional EPA synthesis, and osmolyte synthesis and transport. WP3 contains 55 putative cytochrome c genes, indicating a versatile respiratory adaptation strategy and bioenergetic modification of electron flow. Understanding how different <em>Shewanella</em> species sense and respond to diverse physiochemical parameters holds the promise of extending the environmental range of their biotechnological applications.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Bacterial cultivation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p><em>Shewanella piezotolerans</em> WP3 was isolated from a shallow sediment sample obtained from the western Pacific Ocean (E142°30′08″, N8°00′11″) at a water column depth of 1914 m. It was grown in modified Marine 2216E broth at 20°C <a href="#pone.0001937-Xiao1">[20]</a>. For high pressure incubation of WP3, overnight log phase atmospheric pressure cultures of WP3 cells were grown and then diluted 1000-fold by the same medium, transferred into sterile injection syringes and placed inside pressure vessels. Cultures were then incubated at 20°C at either atmospheric pressure or at 20 MPa with the same method as described elsewhere <a href="#pone.0001937-Li1">[39]</a>.</p>


<h4>Sequencing and assembly</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>DNA isolation, cloning, sequencing, and assay were performed as described for other genomes sequenced by Beijing Institute of Genomics/Hangzhou Genomics Institute ( BIG/HGI) <a href="#pone.0001937-Bao1">[40]</a>, <a href="#pone.0001937-Chiu1">[41]</a>. We constructed three pUC18 plasmid libraries, with insertion size of 2K, 3K, and 5K respectively. Sequences were generated by using ABI3730 and MegaBase 1000 automatic sequencers and assembled by Phred/Phrap/Consed software <a href="#pone.0001937-Ewing1">[42]</a>, <a href="#pone.0001937-Ewing2">[43]</a>, <a href="#pone.0001937-Gordon1">[44]</a>. Scaffolds were established by contigs, according to linking information from forward and reverse sequence ends of each clone. Sequence gaps were closed by re-sequencing the end sequences of each contig, primer walking on linking clones, and by sequencing PCR products from genomic DNA. Consensus sequence coverage and base ambiguities were improved by re-sequencing existing DNA samples and by sequencing additional PCR products, together with manual editing. All repeated regions were verified with PCR amplification across the repeat and sequencing the product. The final genome is based on 85,529 reads, with a total coverage of 6.52X (Q20) and overall base error probability 0.26/10 Kb.</p>


<h4>Genome analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The replication origin was determined by sequence similarity to the origin of <em>Shewanella oneidensis</em> and GC nucleotide skew analysis <a href="#pone.0001937-Lobry1">[45]</a>. An initial set of open reading frames (ORFs) were predicted by Glimmer software <a href="#pone.0001937-Delcher1">[46]</a>. Both predicted ORFs and intergenic sequences were subjected for further manual inspections. All predicted proteins &gt;30 amino acids were searched against GenBank non-redundant protein database using BLAST program <a href="#pone.0001937-Altschul1">[47]</a>. Frameshifts and point mutations were detected and corrected where appropriate. Remaining frameshifts and point mutations are considered to be authentic and were annotated as “authentic frameshift” or “authentic point mutation”, or, in the case of multiple lesions within a single ORF, “degenerate”. tRNAs were predicted by tRNAscan-SE <a href="#pone.0001937-Lowe1">[48]</a>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>The predicted proteins were further investigated by InterProScan program to InterPro database <a href="#pone.0001937-Apweiler1">[49]</a>, which now included 14 integrated databases of protein families, domains and functional sites. Functional classification of WP3 proteins were done according to COG functional category <a href="#pone.0001937-Tatusov1">[25]</a>. Metabolic pathways were analysis with KEGG database <a href="#pone.0001937-Kanehisa1">[50]</a>.</p>


<h4>Proteome phylogenetic analysis</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The whole proteome phylogenetic analysis of the speices were conducted by the Composition Vector method as described by Qi <em>et al.</em> <a href="#pone.0001937-Qi1">[24]</a>. First, we collected all amino acid sequences of a species, and then calculated the frequency of appearance of overlapping oligopeptides of length K. A random background needed to be subtracted from these frequencies by using a Markov model of order (K−2) in order to diminish the influence of random neutral mutations at the molecular level and to highlight the shaping role of selective evolution. Then, we put the "normalized" frequencies in a fixed order; a composition vector of dimension 20K was obtained for each species. And the correlation C (A, B) between two species A and B was determined by taking projection of one normalized vector on another, i.e., taking the cosine of the angle between them. Finally, the normalized distance between the two species was defined to be D = (1−C)/2. Once a distance matrix has been calculated, we then constructed phylogenetic trees by following the standard neighbor-joining method <a href="#pone.0001937-Saitou1">[51]</a> in the Phylip package.</p>


<h4>Bacterial cultivation for RNA isolation</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>WP3 was cultivated as introduced above. As for different temperature conditions, overnight log phase cultures of WP3 cells diluted 1000-fold by the same medium were grown at 4°C or 25°C for 3 days to reach an equivalent optical density. All samples were in triplicate for convenience of reproductivity. The cells were immediately centrifuged and RNA was extracted using TRI reagent-RNA/DNA/protein isolation kit (Molecular research center, Inc., Cincinnati, USA). The RNA was stored in 75% ethanol at −80°C</p>


<h4>Transmission Electron microscopy and swimming motility assay</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Strains were grown to mid-log phase in 2216E medium, centrifuged(5000 g, 1 min), and resuspended in distill water. Samples were placed to a carbon-coated grid (200 mesh) and stained with 1% phosphotungstic acid (PTA). The grid was air dried and examined in a JEM-1230 microscope (JEOL, Tokyo, Japan). The swimming motility of the strains was assessed by examining their movements in the 0.3% 2216E agar plates. The assays were initiated by spotting 1 µl of an overnight cell culture which has been normalized to an optical density of around 1.5 at OD <sub>600nm</sub> onto the agar plates, then incubated at 20°C or 4°C for 3 days or a week, respectively.</p>


<h4>Real Time RT-PCR</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>Three genes for polar and four genes for lateral flagellum synthesis were used for transcription analysis using the Real time RT-PCR method. The primer pairs of selected genes for real-time PCR were designed using Primer Express software (ABI). The primer sequences are: swp5109 For <span class="gene">5′-CCCGCTCAGGT GCCATAG</span>, swp5109Rev <span class="gene">5′-CAA TAGG GCGGCCATCTAAG</span>; swp5114 For <span class="gene">5′-ACCTGTTTAGCTACGATCCCAGTA</span>, swp5114Rev <span class="gene">5′-AGCCAACATATCA GCGTTGAAG</span>; swp5118 For <span class="gene">5′-GCAGATGGTGCATTAGACGAACT</span>, swp5118Rev <span class="gene">5′-TTCACGCCGTTAGCTGCTT</span>; swp1499 For <span class="gene">5′-GCAGCGCAGGCGAAACT</span>, swp1499Rev <span class="gene">5′-TGCTGTTGTGCTTTG CTCATT</span>; swp1511 For <span class="gene">5′-GCACAAAGC GCGATTGC</span>, swp1511Rev <span class="gene">5′-CGGTTCTGTACCGCACCTAAG</span>; swp1508 For 5′-G<span class="gene"> GGTGGACTTGATGTCAATGTG</span>, swp1508 Rev <span class="gene">5′-TTAGAGGTTGAGTCGCATCAAAAT</span>; swp1519 For <span class="gene">5′-GAAATGTG AGCGAACTTCAGTCA</span>, swp1519Rev <span class="gene">5′-TGTCACTGTGGTATCGCCCATA</span>; swp5125 For <span class="gene">5′-GGCTGCACAACTACGATGTGTT</span>, swp5125 Rev <span class="gene">5′-GGCGGTTTGCCTTTAGCA</span>. PCR cycling was conducted using 7500 System SDS software in reaction mixtures with total volumes of 25 ul containing 1× Sybr Green I Universal PCR Master Mix (ABI), 0.5 uM each primer, 1 ul cDNA template. In this method, the amount of target was normalized to that of the reference gene swp0453 whose expression remains stable under various conditions, relative to the calibrator (The transcription levels of the genes at 0.1 MPa, or 20°C were set as 1). Real time RT-PCR assays were performed in triplicate for each sample, and a mean value and standard deviation were calculated for the relative RNA expression levels.</p>


<h4>Accession number</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>The sequence has been deposited in GenBank with accession number CP000472.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FW XX SH. Performed the experiments: JW HJ SL FW. Analyzed the data: JY FW XX SH BZ JW XZ LG. Contributed reagents/materials/analysis tools: XX SH. Wrote the paper: DB FW XX SH JW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001937-Bartlett1" id="pone.0001937-Bartlett1"></a>Bartlett DH (2002) Pressure effects on in vivo microbial processes. Biochim Biophys Acta  1595: 367–381.  <ul class="find" data-citedArticleID="1086768" data-doi="10.1016/s0167-4838(01)00357-0"><li><a href="http://dx.doi.org/10.1016/s0167-4838(01)00357-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pressure+effects+on+in+vivo+microbial+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pressure+effects+on+in+vivo+microbial+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001937-Kato1" id="pone.0001937-Kato1"></a>Kato C, Bartlett DH (1997) The molecular biology of barophilic bacteria. Extremophiles  1: 111–116.  <ul class="find" data-citedArticleID="1086806" data-doi="10.1007/s007920050023"><li><a href="http://dx.doi.org/10.1007/s007920050023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+biology+of+barophilic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+biology+of+barophilic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001937-Bartlett2" id="pone.0001937-Bartlett2"></a>Bartlett DH (1999) Microbial adaptations to the psychrosphere/piezosphere. J Mol Microbiol Biotechnol  1: 93–100.  <ul class="find" data-citedArticleID="1086770"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Microbial+adaptations+to+the+psychrosphere%2Fpiezosphere.&amp;auth=&amp;atitle=Microbial+adaptations+to+the+psychrosphere%2Fpiezosphere." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+adaptations+to+the+psychrosphere%2Fpiezosphere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+adaptations+to+the+psychrosphere%2Fpiezosphere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001937-Vezzi1" id="pone.0001937-Vezzi1"></a>Vezzi A, Campanaro S, D'Angelo M, Simonato F, Vitulo N, et al.  (2005) Life at depth: <em>Photobacterium profundum</em> genome sequence and expression analysis. Science  307: 1459–1461.  <ul class="find" data-citedArticleID="1086854" data-doi="10.1126/science.1103341"><li><a href="http://dx.doi.org/10.1126/science.1103341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life+at+depth%3A+Photobacterium+profundum+genome+sequence+and+expression+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life+at+depth%3A+Photobacterium+profundum+genome+sequence+and+expression+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001937-Methe1" id="pone.0001937-Methe1"></a>Methe BA, Nelson KE, Deming JW, Momen B, Melamud E, et al.  (2005) The psychrophilic lifestyle as revealed by the genome sequence of <em>Colwellia psychrerythraea</em> 34H through genomic and proteomic analyses. Proc Natl Acad Sci U S A  102: 10913–10918.  <ul class="find" data-citedArticleID="1086826" data-doi="10.1073/pnas.0504766102"><li><a href="http://dx.doi.org/10.1073/pnas.0504766102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+psychrophilic+lifestyle+as+revealed+by+the+genome+sequence+of+Colwellia+psychrerythraea+34H+through+genomic+and+proteomic+analyses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+psychrophilic+lifestyle+as+revealed+by+the+genome+sequence+of+Colwellia+psychrerythraea+34H+through+genomic+and+proteomic+analyses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001937-Hou1" id="pone.0001937-Hou1"></a>Hou S, Saw JH, Lee KS, Freitas TA, Belisle C, et al.  (2004) Genome sequence of the deep-sea gamma-proteobacterium <em>Idiomarina loihiensis</em> reveals amino acid fermentation as a source of carbon and energy. Proc Natl Acad Sci U S A  101: 18036–18041.  <ul class="find" data-citedArticleID="1086802" data-doi="10.1073/pnas.0407638102"><li><a href="http://dx.doi.org/10.1073/pnas.0407638102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+deep-sea+gamma-proteobacterium+Idiomarina+loihiensis+reveals+amino+acid+fermentation+as+a+source+of+carbon+and+energy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+deep-sea+gamma-proteobacterium+Idiomarina+loihiensis+reveals+amino+acid+fermentation+as+a+source+of+carbon+and+energy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001937-Medigue1" id="pone.0001937-Medigue1"></a>Medigue C, Krin E, Pascal G, Barbe V, Bernsel A, et al.  (2005) Coping with cold: The genome of the versatile marine Antarctica bacterium <em>Pseudoalteromonas haloplanktis</em> TAC125. Genome Res  15: 1325–1335.  <ul class="find" data-citedArticleID="1086824" data-doi="10.1101/gr.4126905"><li><a href="http://dx.doi.org/10.1101/gr.4126905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coping+with+cold%3A+The+genome+of+the+versatile+marine+Antarctica+bacterium+Pseudoalteromonas+haloplanktis+TAC125." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coping+with+cold%3A+The+genome+of+the+versatile+marine+Antarctica+bacterium+Pseudoalteromonas+haloplanktis+TAC125.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001937-Venkateswaran1" id="pone.0001937-Venkateswaran1"></a>Venkateswaran K, Moser DP, Dollhopf ME, Lies DP, Saffarini DA, et al.  (1999) Polyphasic taxonomy of the genus Shewanella and description of <em>Shewanella oneidensis</em> sp. nov. Int J Syst Bacteriol  49 Pt 2: 705–724.  <ul class="find" data-citedArticleID="1086852" data-doi="10.1099/00207713-49-2-705"><li><a href="http://dx.doi.org/10.1099/00207713-49-2-705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polyphasic+taxonomy+of+the+genus+Shewanella+and+description+of+Shewanella+oneidensis+sp.+nov." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polyphasic+taxonomy+of+the+genus+Shewanella+and+description+of+Shewanella+oneidensis+sp.+nov.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001937-Myers1" id="pone.0001937-Myers1"></a>Myers CR, Nealson KH (1988) Bacterial manganese reduction and growth with manganese oxide as the sole electron acceptor. Science  240: 1319–1321.  <ul class="find" data-citedArticleID="1086832" data-doi="10.1126/science.240.4857.1319"><li><a href="http://dx.doi.org/10.1126/science.240.4857.1319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+manganese+reduction+and+growth+with+manganese+oxide+as+the+sole+electron+acceptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+manganese+reduction+and+growth+with+manganese+oxide+as+the+sole+electron+acceptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001937-Heidelberg1" id="pone.0001937-Heidelberg1"></a>Heidelberg JF, Paulsen IT, Nelson KE, Gaidos EJ, Nelson WC, et al.  (2002) Genome sequence of the dissimilatory metal ion-reducing bacterium <em>Shewanella oneidensis</em>. Nat Biotechnol  20: 1118–1123.  <ul class="find" data-citedArticleID="1086800" data-doi="10.1038/nbt749"><li><a href="http://dx.doi.org/10.1038/nbt749" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+dissimilatory+metal+ion-reducing+bacterium+Shewanella+oneidensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+dissimilatory+metal+ion-reducing+bacterium+Shewanella+oneidensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001937-Meyer1" id="pone.0001937-Meyer1"></a>Meyer TE, Tsapin AI, Vandenberghe I, de Smet L, Frishman D, et al.  (2004) Identification of 42 possible cytochrome C genes in the <em>Shewanella oneidensis</em> genome and characterization of six soluble cytochromes. Omics  8: 57–77.  <ul class="find" data-citedArticleID="1086830" data-doi="10.1089/153623104773547499"><li><a href="http://dx.doi.org/10.1089/153623104773547499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+42+possible+cytochrome+C+genes+in+the+Shewanella+oneidensis+genome+and+characterization+of+six+soluble+cytochromes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+42+possible+cytochrome+C+genes+in+the+Shewanella+oneidensis+genome+and+characterization+of+six+soluble+cytochromes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001937-Gao1" id="pone.0001937-Gao1"></a>Gao H, Wang Y, Liu X, Yan T, Wu L, et al.  (2004) Global transcriptome analysis of the heat shock response of <em>Shewanella oneidensis</em>. J Bacteriol  186: 7796–7803.  <ul class="find" data-citedArticleID="1086790" data-doi="10.1128/jb.186.22.7796-7803.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.22.7796-7803.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transcriptome+analysis+of+the+heat+shock+response+of+Shewanella+oneidensis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transcriptome+analysis+of+the+heat+shock+response+of+Shewanella+oneidensis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001937-Leaphart1" id="pone.0001937-Leaphart1"></a>Leaphart AB, Thompson DK, Huang K, Alm E, Wan XF, et al.  (2006) Transcriptome profiling of <em>Shewanella oneidensis</em> gene expression following exposure to acidic and alkaline pH. J Bacteriol  188: 1633–1642.  <ul class="find" data-citedArticleID="1086814" data-doi="10.1128/jb.188.4.1633-1642.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.4.1633-1642.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+profiling+of+Shewanella+oneidensis+gene+expression+following+exposure+to+acidic+and+alkaline+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+profiling+of+Shewanella+oneidensis+gene+expression+following+exposure+to+acidic+and+alkaline+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001937-Benson1" id="pone.0001937-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2006) GenBank. Nucleic Acids Res  34: D16–20.  <ul class="find" data-citedArticleID="1086774" data-doi="10.1093/nar/gkm929"><li><a href="http://dx.doi.org/10.1093/nar/gkm929" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001937-Kato2" id="pone.0001937-Kato2"></a>Kato C, Nogi Y (2001) Correlation between phylogenetic structure and function: examples from deep-sea <em>Shewanella</em>. FEMS Microbiol Ecol  35: 223–230.  <ul class="find" data-citedArticleID="1086808" data-doi="10.1111/j.1574-6941.2001.tb00807.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6941.2001.tb00807.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+phylogenetic+structure+and+function%3A+examples+from+deep-sea+Shewanella." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+phylogenetic+structure+and+function%3A+examples+from+deep-sea+Shewanella.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001937-Kato3" id="pone.0001937-Kato3"></a>Kato C, Qureshi MH (1999) Pressure response in deep-sea piezophilic bacteria. J Mol Microbiol Biotechnol  1: 87–92.  <ul class="find" data-citedArticleID="1086810"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pressure+response+in+deep-sea+piezophilic+bacteria.&amp;auth=&amp;atitle=Pressure+response+in+deep-sea+piezophilic+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pressure+response+in+deep-sea+piezophilic+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pressure+response+in+deep-sea+piezophilic+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001937-Xu1" id="pone.0001937-Xu1"></a>Xu MX, Wang P, Wang FP, Xiao X (2005) Microbial diversity in a deep-sea Station of Pacific Nodule Province, Studied via 16S rRNA Gene Sequences and Extremophile Cultivation. Biodiv and Conserv  14: 3363–3380.  <ul class="find" data-citedArticleID="1086862" data-doi="10.1007/s10531-004-0544-z"><li><a href="http://dx.doi.org/10.1007/s10531-004-0544-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+diversity+in+a+deep-sea+Station+of+Pacific+Nodule+Province%2C+Studied+via+16S+rRNA+Gene+Sequences+and+Extremophile+Cultivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+diversity+in+a+deep-sea+Station+of+Pacific+Nodule+Province%2C+Studied+via+16S+rRNA+Gene+Sequences+and+Extremophile+Cultivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001937-Dianne1" id="pone.0001937-Dianne1"></a>Dianne K, Newman JFB (2002) Geomicrobiology: How Molecular-Scale Interactions Underpin Biogeochemical Systems. Science  296: 1071–1077.  <ul class="find" data-citedArticleID="1086782" data-doi="10.1126/science.1010716"><li><a href="http://dx.doi.org/10.1126/science.1010716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geomicrobiology%3A+How+Molecular-Scale+Interactions+Underpin+Biogeochemical+Systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geomicrobiology%3A+How+Molecular-Scale+Interactions+Underpin+Biogeochemical+Systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001937-Wang1" id="pone.0001937-Wang1"></a>Wang F, Wang P, Chen M, Xiao X (2004) Isolation of extremophiles with the detection and retrieval of <em>Shewanella</em> strains in deep-sea sediments from the west Pacific. Extremophiles  8: 165–168.  <ul class="find" data-citedArticleID="1086856" data-doi="10.1007/s00792-003-0365-0"><li><a href="http://dx.doi.org/10.1007/s00792-003-0365-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+extremophiles+with+the+detection+and+retrieval+of+Shewanella+strains+in+deep-sea+sediments+from+the+west+Pacific." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+extremophiles+with+the+detection+and+retrieval+of+Shewanella+strains+in+deep-sea+sediments+from+the+west+Pacific.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001937-Xiao1" id="pone.0001937-Xiao1"></a>Xiao X, Wang P, Zeng X, Chen MX, Bartlett DH, et al.  (2007) <em>Shewanella psychrophila</em> sp. nov., <em>Shewanella piezotolerans</em> sp. nov., two new <em>Shewanella</em> species from west Pacific deep-sea sediment. Int J Syst Evol Microbiol  57: 60–65.  <ul class="find" data-citedArticleID="1086860" data-doi="10.1099/ijs.0.64500-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.64500-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shewanella+psychrophila+sp.+nov.%2C+Shewanella+piezotolerans+sp.+nov.%2C+two+new+Shewanella+species+from+west+Pacific+deep-sea+sediment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shewanella+psychrophila+sp.+nov.%2C+Shewanella+piezotolerans+sp.+nov.%2C+two+new+Shewanella+species+from+west+Pacific+deep-sea+sediment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001937-Roh1" id="pone.0001937-Roh1"></a>Roh Y, Gao H, Vali H, Kennedy DW, Yang ZK, et al.  (2006) Metal reduction and iron biomineralization by a psychrotolerant Fe(III)-reducing bacterium, <em>Shewanella</em> sp. strain PV-4. Appl Environ Microbiol  72: 3236–3244.  <ul class="find" data-citedArticleID="1086840" data-doi="10.1128/aem.72.5.3236-3244.2006"><li><a href="http://dx.doi.org/10.1128/aem.72.5.3236-3244.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metal+reduction+and+iron+biomineralization+by+a+psychrotolerant+Fe%28III%29-reducing+bacterium%2C+Shewanella+sp.+strain+PV-4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metal+reduction+and+iron+biomineralization+by+a+psychrotolerant+Fe%28III%29-reducing+bacterium%2C+Shewanella+sp.+strain+PV-4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001937-Gao2" id="pone.0001937-Gao2"></a>Gao H, Obraztova A, Stewart N, Popa R, Fredrickson JK, et al.  (2006) <em>Shewanella loihica</em> sp. nov., isolated from iron-rich microbial mats in the Pacific Ocean. Int J Syst Evol Microbiol  56: 1911–1916.  <ul class="find" data-citedArticleID="1086792" data-doi="10.1099/ijs.0.64354-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.64354-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shewanella+loihica+sp.+nov.%2C+isolated+from+iron-rich+microbial+mats+in+the+Pacific+Ocean." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shewanella+loihica+sp.+nov.%2C+isolated+from+iron-rich+microbial+mats+in+the+Pacific+Ocean.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001937-Eisen1" id="pone.0001937-Eisen1"></a>Eisen JA, Heidelberg JF, White O, Salzberg SL (2000) Evidence for symmetric chromosomal inversions around the replication origin in bacteria. Genome Biol  1: RESEARCH0011.  <ul class="find" data-citedArticleID="1086784" data-doi="10.1186/gb-2000-1-6-research0011"><li><a href="http://dx.doi.org/10.1186/gb-2000-1-6-research0011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+symmetric+chromosomal+inversions+around+the+replication+origin+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+symmetric+chromosomal+inversions+around+the+replication+origin+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001937-Qi1" id="pone.0001937-Qi1"></a>Qi J, Wang B, Hao BL (2004) Whole Proteome Prokaryote Phylogeny Without Sequence Alignment: A K-String Composition Approach. J Mol Evol  58: 1–11.  <ul class="find" data-citedArticleID="1086838" data-doi="10.1007/s00239-003-2493-7"><li><a href="http://dx.doi.org/10.1007/s00239-003-2493-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole+Proteome+Prokaryote+Phylogeny+Without+Sequence+Alignment%3A+A+K-String+Composition+Approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole+Proteome+Prokaryote+Phylogeny+Without+Sequence+Alignment%3A+A+K-String+Composition+Approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001937-Tatusov1" id="pone.0001937-Tatusov1"></a>Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, et al.  (2003) The COG database: an updated version includes eukaryotes. BMC Bioinformatics  4: 41.  <ul class="find" data-citedArticleID="1086850"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+COG+database%3A+an+updated+version+includes+eukaryotes.&amp;auth=&amp;atitle=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COG+database%3A+an+updated+version+includes+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001937-Tamegai1" id="pone.0001937-Tamegai1"></a>Tamegai H, Kawano H, Ishii A, Chikuma S, Nakasone K, et al.  (2005) Pressure-regulated biosynthesis of cytochrome bd in piezo- and psychrophilic deep-sea bacterium <em>Shewanella violacea</em> DSS12. Extremophiles  9: 247–253.  <ul class="find" data-citedArticleID="1086848" data-doi="10.1007/s00792-005-0439-2"><li><a href="http://dx.doi.org/10.1007/s00792-005-0439-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pressure-regulated+biosynthesis+of+cytochrome+bd+in+piezo-+and+psychrophilic+deep-sea+bacterium+Shewanella+violacea+DSS12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pressure-regulated+biosynthesis+of+cytochrome+bd+in+piezo-+and+psychrophilic+deep-sea+bacterium+Shewanella+violacea+DSS12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001937-Glockner1" id="pone.0001937-Glockner1"></a>Glockner FO, Kube M, Bauer M, Teeling H, Lombardot T, et al.  (2003) Complete genome sequence of the marine planctomycete <em>Pirellula</em> sp. strain 1. Proc Natl Acad Sci U S A  100: 8298–8303.  <ul class="find" data-citedArticleID="1086794" data-doi="10.1073/pnas.1431443100"><li><a href="http://dx.doi.org/10.1073/pnas.1431443100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+the+marine+planctomycete+Pirellula+sp.+strain+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+the+marine+planctomycete+Pirellula+sp.+strain+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001937-Metz1" id="pone.0001937-Metz1"></a>Metz JG, Roessler P, Facciotti D, Levering C, Dittrich F, et al.  (2001) Production of polyunsaturated fatty acids by polyketide synthases in both prokaryotes and eukaryotes. Science  293: 290–293.  <ul class="find" data-citedArticleID="1086828" data-doi="10.1126/science.1059593"><li><a href="http://dx.doi.org/10.1126/science.1059593" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Production+of+polyunsaturated+fatty+acids+by+polyketide+synthases+in+both+prokaryotes+and+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Production+of+polyunsaturated+fatty+acids+by+polyketide+synthases+in+both+prokaryotes+and+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001937-Ofengand1" id="pone.0001937-Ofengand1"></a>Ofengand J, Malhotra A, Remme J, Gutgsell NS, Del Campo M, et al.  (2001) Pseudouridines and pseudouridine synthases of the ribosome. Cold Spring Harb Symp Quant Biol  66: 147–159.  <ul class="find" data-citedArticleID="1086834" data-doi="10.1101/sqb.2001.66.147"><li><a href="http://dx.doi.org/10.1101/sqb.2001.66.147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudouridines+and+pseudouridine+synthases+of+the+ribosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudouridines+and+pseudouridine+synthases+of+the+ribosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001937-Charette1" id="pone.0001937-Charette1"></a>Charette M, Gray MW (2000) Pseudouridine in RNA: what, where, how, and why. IUBMB Life  49: 341–351.  <ul class="find" data-citedArticleID="1086776" data-doi="10.1080/152165400410182"><li><a href="http://dx.doi.org/10.1080/152165400410182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudouridine+in+RNA%3A+what%2C+where%2C+how%2C+and+why." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudouridine+in+RNA%3A+what%2C+where%2C+how%2C+and+why.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001937-Khachane1" id="pone.0001937-Khachane1"></a>Khachane AN, Timmis KN, dos Santos VA (2005) Uracil content of 16S rRNA of thermophilic and psychrophilic prokaryotes correlates inversely with their optimal growth temperatures. Nucleic Acids Res  33: 4016–4022.  <ul class="find" data-citedArticleID="1086812" data-doi="10.1093/nar/gki714"><li><a href="http://dx.doi.org/10.1093/nar/gki714" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uracil+content+of+16S+rRNA+of+thermophilic+and+psychrophilic+prokaryotes+correlates+inversely+with+their+optimal+growth+temperatures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uracil+content+of+16S+rRNA+of+thermophilic+and+psychrophilic+prokaryotes+correlates+inversely+with+their+optimal+growth+temperatures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001937-Saunders1" id="pone.0001937-Saunders1"></a>Saunders NF, Thomas T, Curmi PM, Mattick JS, Kuczek E, et al.  (2003) Mechanisms of thermal adaptation revealed from the genomes of the Antarctic Archaea <em>Methanogenium frigidum</em> and <em>Methanococcoides burtonii</em>. Genome Res  13: 1580–1588.  <ul class="find" data-citedArticleID="1086846" data-doi="10.1101/gr.1180903"><li><a href="http://dx.doi.org/10.1101/gr.1180903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+thermal+adaptation+revealed+from+the+genomes+of+the+Antarctic+Archaea+Methanogenium+frigidum+and+Methanococcoides+burtonii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+thermal+adaptation+revealed+from+the+genomes+of+the+Antarctic+Archaea+Methanogenium+frigidum+and+Methanococcoides+burtonii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001937-Ofengand2" id="pone.0001937-Ofengand2"></a>Ofengand J, Rudd KE (2000)  The ribosome: Structure, function, antibiotics and cellular interactions. In: Garrett RA et al, editor. The ribosome: Structure, function, antibiotics, and cellular interactions. Washington, DC: ASM Press. pp. 175–189.  <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0001937-Gutgsell1" id="pone.0001937-Gutgsell1"></a>Gutgsell NS, Deutscher MP, Ofengand J (2005) The pseudouridine synthase RluD is required for normal ribosome assembly and function in <em>Escherichia coli</em>. RNA  11: 1141–1152.  <ul class="find" data-citedArticleID="1086798" data-doi="10.1261/rna.2550105"><li><a href="http://dx.doi.org/10.1261/rna.2550105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+pseudouridine+synthase+RluD+is+required+for+normal+ribosome+assembly+and+function+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+pseudouridine+synthase+RluD+is+required+for+normal+ribosome+assembly+and+function+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001937-Belas1" id="pone.0001937-Belas1"></a>Belas R, Simon M, Silverman M (1986) Regulation of lateral flagella gene transcription in <em>Vibrio parahaemolyticus</em>. J Bacteriol  167: 210–218.  <ul class="find" data-citedArticleID="1086772"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+lateral+flagella+gene+transcription+in+Vibrio+parahaemolyticus.&amp;auth=&amp;atitle=Regulation+of+lateral+flagella+gene+transcription+in+Vibrio+parahaemolyticus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+lateral+flagella+gene+transcription+in+Vibrio+parahaemolyticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+lateral+flagella+gene+transcription+in+Vibrio+parahaemolyticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001937-McCarter1" id="pone.0001937-McCarter1"></a>McCarter L, Hilmen M, Silverman M (1988) Flagellar dynamometer controls swarmer cell differentiation of <em>V. parahaemolyticus</em>. Cell  54: 345–351.  <ul class="find" data-citedArticleID="1086822" data-doi="10.1016/0092-8674(88)90197-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90197-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flagellar+dynamometer+controls+swarmer+cell+differentiation+of+V.+parahaemolyticus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flagellar+dynamometer+controls+swarmer+cell+differentiation+of+V.+parahaemolyticus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001937-Russel1" id="pone.0001937-Russel1"></a>Russel M, Linderoth NA, Sali A (1997) Filamentous phage assembly: variation on a protein export theme. Gene  192: 23–32.  <ul class="find" data-citedArticleID="1086842" data-doi="10.1016/s0378-1119(96)00801-3"><li><a href="http://dx.doi.org/10.1016/s0378-1119(96)00801-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Filamentous+phage+assembly%3A+variation+on+a+protein+export+theme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Filamentous+phage+assembly%3A+variation+on+a+protein+export+theme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001937-Wang2" id="pone.0001937-Wang2"></a>Wang F, Wang FP, Li Q, Xiao X (2007) A novel filamentous phage from a deep-sea bacterium Shewanella piezotolerans WP3 is induced at low temperature. J Bacteriol  189: 7151–7153.  <ul class="find" data-citedArticleID="1086858" data-doi="10.1128/jb.00569-07"><li><a href="http://dx.doi.org/10.1128/jb.00569-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+filamentous+phage+from+a+deep-sea+bacterium+Shewanella+piezotolerans+WP3+is+induced+at+low+temperature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+filamentous+phage+from+a+deep-sea+bacterium+Shewanella+piezotolerans+WP3+is+induced+at+low+temperature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001937-Li1" id="pone.0001937-Li1"></a>Li S, Xiao X, Li J, Luo J, Wang F (2006) Identification of genes regulated by changing salinity in the deep-sea bacterium <em>Shewanella</em> sp. WP3 using RNA arbitrarily primed PCR. Extremophiles  10: 97–104.  <ul class="find" data-citedArticleID="1086816" data-doi="10.1007/s00792-005-0476-x"><li><a href="http://dx.doi.org/10.1007/s00792-005-0476-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+genes+regulated+by+changing+salinity+in+the+deep-sea+bacterium+Shewanella+sp.+WP3+using+RNA+arbitrarily+primed+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+genes+regulated+by+changing+salinity+in+the+deep-sea+bacterium+Shewanella+sp.+WP3+using+RNA+arbitrarily+primed+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001937-Bao1" id="pone.0001937-Bao1"></a>Bao Q, Tian Y, Li W, Xu Z, Xuan Z, et al.  (2002) A complete sequence of the <em>T. tengcongensis</em> genome. Genome Res  12: 689–700.  <ul class="find" data-citedArticleID="1086766" data-doi="10.1101/gr.219302"><li><a href="http://dx.doi.org/10.1101/gr.219302" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+complete+sequence+of+the+T.+tengcongensis+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+complete+sequence+of+the+T.+tengcongensis+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001937-Chiu1" id="pone.0001937-Chiu1"></a>Chiu CH, Tang P, Chu C, Hu S, Bao Q, et al.  (2005) The genome sequence of Salmonella enterica serovar Choleraesuis, a highly invasive and resistant zoonotic pathogen. Nucleic Acids Res  33: 1690–1698.  <ul class="find" data-citedArticleID="1086778" data-doi="10.1093/nar/gki297"><li><a href="http://dx.doi.org/10.1093/nar/gki297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+of+Salmonella+enterica+serovar+Choleraesuis%2C+a+highly+invasive+and+resistant+zoonotic+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+of+Salmonella+enterica+serovar+Choleraesuis%2C+a+highly+invasive+and+resistant+zoonotic+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001937-Ewing1" id="pone.0001937-Ewing1"></a>Ewing B, Hillier L, Wendl MC, Green P (1998) Base-calling of automated sequencer traces using phred. I. Accuracy assessment. Genome Res  8: 175–185.  <ul class="find" data-citedArticleID="1086786" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+I.+Accuracy+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001937-Ewing2" id="pone.0001937-Ewing2"></a>Ewing B, Green P (1998) Base-calling of automated sequencer traces using phred. II. Error probabilities. Genome Res  8: 186–194.  <ul class="find" data-citedArticleID="1086788" data-doi="10.1101/gr.8.3.175"><li><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base-calling+of+automated+sequencer+traces+using+phred.+II.+Error+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001937-Gordon1" id="pone.0001937-Gordon1"></a>Gordon D, Abajian C, Green P (1998) Consed: a graphical tool for sequence finishing. Genome Res  8: 195–202.  <ul class="find" data-citedArticleID="1086796" data-doi="10.1101/gr.8.3.195"><li><a href="http://dx.doi.org/10.1101/gr.8.3.195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consed%3A+a+graphical+tool+for+sequence+finishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consed%3A+a+graphical+tool+for+sequence+finishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001937-Lobry1" id="pone.0001937-Lobry1"></a>Lobry JR (1996) Asymmetric substitution patterns in the two DNA strands of bacteria. Mol Biol Evol  13: 660–665.  <ul class="find" data-citedArticleID="1086818" data-doi="10.1093/oxfordjournals.molbev.a025626"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025626" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Asymmetric+substitution+patterns+in+the+two+DNA+strands+of+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Asymmetric+substitution+patterns+in+the+two+DNA+strands+of+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001937-Delcher1" id="pone.0001937-Delcher1"></a>Delcher AL, Harmon D, Kasif S, White O, Salzberg SL (1999) Improved microbial gene identification with GLIMMER. Nucleic Acids Res  27: 4636–4641.  <ul class="find" data-citedArticleID="1086780" data-doi="10.1093/nar/27.23.4636"><li><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+microbial+gene+identification+with+GLIMMER." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+microbial+gene+identification+with+GLIMMER.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001937-Altschul1" id="pone.0001937-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="1086762" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001937-Lowe1" id="pone.0001937-Lowe1"></a>Lowe TM, Eddy SR (1997) tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res  25: 955–964.  <ul class="find" data-citedArticleID="1086820" data-doi="10.1093/nar/25.5.955"><li><a href="http://dx.doi.org/10.1093/nar/25.5.955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001937-Apweiler1" id="pone.0001937-Apweiler1"></a>Apweiler R, Attwood TK, Bairoch A, Bateman A, Birney E, et al.  (2001) The InterPro database, an integrated documentation resource for protein families, domains and functional sites. Nucleic Acids Res  29: 37–40.  <ul class="find" data-citedArticleID="1086764" data-doi="10.1093/nar/29.1.37"><li><a href="http://dx.doi.org/10.1093/nar/29.1.37" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+InterPro+database%2C+an+integrated+documentation+resource+for+protein+families%2C+domains+and+functional+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+InterPro+database%2C+an+integrated+documentation+resource+for+protein+families%2C+domains+and+functional+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001937-Kanehisa1" id="pone.0001937-Kanehisa1"></a>Kanehisa M, Goto S, Kawashima S, Okuno Y, Hattori M (2004) The KEGG resource for deciphering the genome. Nucleic Acids Res  32: D277–280.  <ul class="find" data-citedArticleID="1086804" data-doi="10.1093/nar/gkh063"><li><a href="http://dx.doi.org/10.1093/nar/gkh063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+KEGG+resource+for+deciphering+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+KEGG+resource+for+deciphering+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001937-Saitou1" id="pone.0001937-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="1086844"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.XML" value="109076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.PDF" value="1321159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g001.PNG_L" value="1314383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g001.PNG_M" value="316537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g001.PNG_S" value="14086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g001.TIF" value="1756426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g001.PNG_I" value="102138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t001.PNG_L" value="96787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t001.PNG_M" value="52883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t001.PNG_S" value="9453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t001.TIF" value="530642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t001.PNG_I" value="15881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g002.PNG_L" value="660160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g002.PNG_M" value="158617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g002.PNG_S" value="8839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g002.TIF" value="897436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g002.PNG_I" value="73698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g003.PNG_L" value="135160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g003.PNG_M" value="38136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g003.PNG_S" value="5088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g003.TIF" value="239330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g003.PNG_I" value="12548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g004.PNG_L" value="290049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g004.PNG_M" value="75689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g004.PNG_S" value="8560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g004.TIF" value="386100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g004.PNG_I" value="24988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g005.PNG_L" value="307716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g005.PNG_M" value="23635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g005.PNG_S" value="4777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g005.TIF" value="507650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g005.PNG_I" value="8670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g006.PNG_L" value="581698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g006.PNG_M" value="101987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g006.PNG_S" value="10632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g006.TIF" value="970824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g006.PNG_I" value="36007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g007.PNG_L" value="1688950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g007.PNG_M" value="197518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g007.PNG_S" value="10316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g007.TIF" value="2492404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g007.PNG_I" value="60658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g008.PNG_L" value="773853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g008.PNG_M" value="221923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g008.PNG_S" value="12119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g008.TIF" value="1183678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.g008.PNG_I" value="80638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t002.PNG_L" value="181000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t002.PNG_M" value="97479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t002.PNG_S" value="11476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t002.TIF" value="755568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001937.t002.PNG_I" value="29244"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001937&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001937" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001937&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001937&volume=&issue=&title=Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3&author_name=Fengping%20Wang%2C%20Jianbin%20Wang%2C%20Huahua%20Jian%2C%20Bing%20Zhang%2C%20Shengkang%20Li%2C%20Feng%20Wang%2C%20Xiaowei%20Zeng%2C%20Lei%20Gao%2C%20Douglas%20Hoyt%20Bartlett%2C%20Jun%20Yu%2C%20Songnian%20Hu%2C%20Xiang%20Xiao&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001937" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937&amp;t=Environmental%20Adaptation%3A%20Genomic%20Analysis%20of%20the%20Piezotolerant%20and%20Psychrotolerant%20Deep-Sea%20Iron%20Reducing%20Bacterium%20Shewanella%20piezotolerans%20WP3" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937&title=Environmental%20Adaptation%3A%20Genomic%20Analysis%20of%20the%20Piezotolerant%20and%20Psychrotolerant%20Deep-Sea%20Iron%20Reducing%20Bacterium%20Shewanella%20piezotolerans%20WP3&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937&amp;title=Environmental%20Adaptation%3A%20Genomic%20Analysis%20of%20the%20Piezotolerant%20and%20Psychrotolerant%20Deep-Sea%20Iron%20Reducing%20Bacterium%20Shewanella%20piezotolerans%20WP3" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001937&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium Shewanella piezotolerans WP3';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001937';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Environmental%20Adaptation%3A%20Genomic%20Analysis%20of%20the%20Piezotolerant%20and%20Psychrotolerant%20Deep-Sea%20Iron%20Reducing%20Bacterium%20Shewanella%20piezotolerans%20WP3 http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001937" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001937" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->
<div class="block">
    <div class="header">
        <h3>Related PLOS Articles</h3>
    </div>
      <p><a href = "/article/info:doi/10.1371/annotation/744d7c8d-db8a-4ad4-aec5-62549b1dbefe" title="Read Open-Access Article">Correction: Environmental Adaptation: Genomic Analysis of the Piezotolerant and Psychrotolerant Deep-Sea Iron Reducing Bacterium <i>Shewanella piezotolerans</i> WP3</a></p>
</div>


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="26698"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacteriophages%22" title="Search for articles in the subject area:'Bacteriophages'"><div class="flagText">Bacteriophages</div></a>
              <div data-categoryid="43083" data-articleid="26698"
                   data-categoryname="Bacteriophages"
                   class="flagImage" title="Flag 'Bacteriophages' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="26698"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flagella%22" title="Search for articles in the subject area:'Flagella'"><div class="flagText">Flagella</div></a>
              <div data-categoryid="22393" data-articleid="26698"
                   data-categoryname="Flagella"
                   class="flagImage" title="Flag 'Flagella' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26698"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26698"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22High+pressure%22" title="Search for articles in the subject area:'High pressure'"><div class="flagText">High pressure</div></a>
              <div data-categoryid="27813" data-articleid="26698"
                   data-categoryname="High pressure"
                   class="flagImage" title="Flag 'High pressure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="26698"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7994'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7646'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5775&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1297">Referee comments: Referee 2 (Chiaki Kato)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
