

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001662"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001662" />

    <meta name="citation_title" content="High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array"/>
    <meta itemprop="name" content="High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array"/>

      <meta name="citation_author" content="Sandra L. Spurgeon"/>
            <meta name="citation_author_institution" content="Fluidigm Corporation, South San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Robert C. Jones"/>
            <meta name="citation_author_institution" content="Fluidigm Corporation, South San Francisco, California, United States of America"/>
      <meta name="citation_author" content="Ramesh Ramakrishnan"/>
            <meta name="citation_author_institution" content="Fluidigm Corporation, South San Francisco, California, United States of America"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001662.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1662"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Nat Biotechnol; citation_volume=24; citation_number=1; citation_pages=1039; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MAQC papers over the cracks. Nat Biotechnol 25: 27–28.; citation_author=P Liang; citation_journal_title=Comment on Nat Biotechnol (2006); citation_volume=24; citation_number=2; citation_pages=1151-61; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene expression correlates of clinical prostate cancer behavior.; citation_author=D Singh; citation_author=PG Febbo; citation_author=K Ross; citation_author=DG Jackson; citation_author=J Manola; citation_journal_title=Cancer Cell; citation_volume=1; citation_number=3; citation_pages=203-209; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microfluidic Large-Scale Integration: The Evolution of Design Rules for Biological Automation.; citation_author=J Melin; citation_author=SR Quake; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=36; citation_number=4; citation_pages=213-231; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of a hyperactive Escherichia coli glycerol kinase mutant Gly230 –> Asp obtained using microfluidic crystallization devices.; citation_author=MJ Anderson; citation_author=B DeLaBarre; citation_author=A Raghunathan; citation_author=BO Palsson; citation_author=AT Brunger; citation_journal_title=Biochemistry; citation_volume=46; citation_number=5; citation_pages=5722-5731; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Transcription factor profiling in individual hematopoietic progenitors by digital RT-PCR.; citation_author=L Warren; citation_author=D Bryder; citation_author=IL Weissman; citation_author=SR Quake; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=6; citation_pages=17807-17812; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cells on chips.; citation_author=J El-Ali; citation_author=PK Sorger; citation_author=KF Jensen; citation_journal_title=Nature; citation_volume=442; citation_number=7; citation_pages=403-411; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Solving the “world-to-chip” interface problem with a microfluidic matrix.; citation_author=J Liu; citation_author=C Hansen; citation_author=SR Quake; citation_journal_title=Anal Chem; citation_volume=75; citation_number=8; citation_pages=4718-4723; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=9; " />
      <meta name="citation_reference" content="citation_title=Lola regulates Drosophila olfactory projection neuron identity and targeting specificity; citation_author=ML Spletter; citation_author=J Liu; citation_author=J Liu; citation_author=H Su; citation_author=E Giniger; citation_author=T Komiyama; citation_author=S Quake; citation_author=L Luo; citation_journal_title=Neural Development; citation_volume=2; citation_number=10; citation_pages=14; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Solving the “world-to-chip” interface problem with a microfluidic matrix.; citation_author=J Liu; citation_author=C Hansen; citation_author=SR Quake; citation_journal_title=Anal Chem.; citation_volume=75; citation_number=11; citation_pages=4718-4723; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=GeneNote: whole genome expression profiles in normal human tissues.; citation_author=O Shmueli; citation_author=S Horn-Saban; citation_author=V Chalifa-Caspi; citation_author=M Shmoish; citation_author=R Ophir; citation_journal_title=C R Biol; citation_volume=326; citation_number=12; citation_pages=1067-1072; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The mammalian myosin heavy chain gene family.; citation_author=A Weiss; citation_author=LA Leinwand; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=12; citation_number=13; citation_pages=417-439; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A millennial myosin census.; citation_author=JS Berg; citation_author=BC Powell; citation_author=RE Cheney; citation_journal_title=Mol Biol Cell; citation_volume=12; citation_number=14; citation_pages=780-794; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Signaling Pathways in Skeletal Muscle Remodeling.; citation_author=R Bassel-Duby; citation_author=EN Olson; citation_journal_title=Annu Rev Biochem; citation_volume=75; citation_number=15; citation_pages=19-37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MyoD and the transcriptional control of myogenesis.; citation_author=CA Berkes; citation_author=SJ Tapscott; citation_journal_title=Semin Cell Dev Biol; citation_volume=16; citation_number=16; citation_pages=585-595; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Promoter-specific regulation of MyoD binding and signal transduction cooperate to pattern gene expression.; citation_author=DA Bergstrom; citation_author=BH Penn; citation_author=A Strand; citation_author=RLS Perry; citation_author=MA Rudnicki; citation_journal_title=Mol Cell; citation_volume=9; citation_number=17; citation_pages=587-600; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dynamic gene expression during the onset of myoblast differentiation in vitro.; citation_author=I Delgado; citation_author=X Huang; citation_author=S Jones; citation_author=L Zhang; citation_author=R Hatcher; citation_journal_title=Genomics; citation_volume=82; citation_number=18; citation_pages=109-121; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=In vivo filtering of in vitro expression data reveals MyoD targets.; citation_author=P Zhao; citation_author=J Seo; citation_author=Z Wang; citation_author=Y Wang; citation_author=B Shneiderman; citation_journal_title=C R Biol; citation_volume=326; citation_number=19; citation_pages=1049-1065; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Myogenic regulatory protein (MyoD1) expression in childhood solid tumors: diagnostic utility in rhabdomyosarcoma.; citation_author=P Dias; citation_author=DM Parham; citation_author=DN Shapiro; citation_author=BL Webber; citation_author=PJ Houghton; citation_journal_title=Am J Pathol; citation_volume=137; citation_number=20; citation_pages=1283-1291; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of the mammalian transforming growth factor-beta superfamily.; citation_author=H Chang; citation_author=CW Brown; citation_author=MM Matzuk; citation_journal_title=Endocr Rev; citation_volume=23; citation_number=21; citation_pages=787-823; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Regulation of muscle mass by myostatin.; citation_author=SJ Lee; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=20; citation_number=22; citation_pages=61-86; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A mutation in the myostatin gene increases muscle mass and enhances racing performance in heterozygote dogs.; citation_author=DS Mosher; citation_author=P Quignon; citation_author=CD Bustamante; citation_author=NB Sutter; citation_author=CS Mellersh; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=23; citation_pages=e79; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Human adaptive evolution at Myostatin (GDF8), a regulator of muscle growth.; citation_author=MA Saunders; citation_author=JM Good; citation_author=EC Lawrence; citation_author=RE Ferrell; citation_author=WH Li; citation_journal_title=Am J Hum Genet; citation_volume=79; citation_number=24; citation_pages=1089-1097; citation_date=2006; " />
      <meta name="citation_reference" content="citation_number=25; " />
      <meta name="citation_reference" content="citation_title=The family of bone morphogenetic proteins.; citation_author=P Ducy; citation_author=G Karsenty; citation_journal_title=Kidney Int; citation_volume=57; citation_number=26; citation_pages=2207-2214; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Regulation of anterior/posterior patterning of the axial skeleton by growth/differentiation factor 11.; citation_author=AC McPherron; citation_author=AM Lawler; citation_author=SJ Lee; citation_journal_title=Nat Genet; citation_volume=22; citation_number=27; citation_pages=260-264; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Identification of ubiquitin ligases required for skeletal muscle atrophy.; citation_author=SC Bodine; citation_author=E Latres; citation_author=S Baumhueter; citation_author=VK Lai; citation_author=L Nunez; citation_journal_title=Science; citation_volume=294; citation_number=28; citation_pages=1704-1708; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes.; citation_author=J Vandesompele; citation_author=K De Preter; citation_author=F Pattyn; citation_author=B Poppe; citation_author=N Van Roy; citation_journal_title=Genome Biol; citation_volume=3; citation_number=29; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets.; citation_author=CL Andersen; citation_author=JL Jensen; citation_author=TF Ørntoft; citation_journal_title=Cancer Res; citation_volume=64; citation_number=30; citation_pages=5245-5250; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=31; " />
      <meta name="citation_reference" content="citation_title=The Digital Array Response Curve.; citation_author=LA Warren; citation_author=JA Weinstein; citation_author=SR Quake; citation_number=32; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.; citation_author=KJ Livak; citation_author=TD Schmittgen; citation_journal_title=Methods; citation_volume=25; citation_number=33; citation_pages=402-408; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array"/>
    <meta name="twitter:description" content="We describe a high throughput gene expression platform based on microfluidic dynamic arrays. This system allows 2,304 simultaneous real time PCR gene expression measurements in a single chip, while requiring less pipetting than is required to set up a 96 well plate. We show that one can measure the expression of 45 different genes in 18 tissues with replicates in a single chip. The data have excellent concordance with conventional real time PCR and the microfluidic dynamic arrays show better reproducibility than commercial DNA microarrays."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001662.g003"/>

  <meta property="og:title" content="High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6033'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=105'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4735&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001662">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001662" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001662&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sandra L. Spurgeon, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Fluidigm Corporation, South San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Robert C. Jones, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Fluidigm Corporation, South San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ramesh Ramakrishnan
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:ramesh.ramakrishnan@fluidigm.com">ramesh.ramakrishnan@fluidigm.com</a></p>

                <p>Affiliation:
                  Fluidigm Corporation, South San Francisco, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001662</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001662-g001" data-doi="info:doi/10.1371/journal.pone.0001662" data-uri="info:doi/10.1371/journal.pone.0001662.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001662-g002" data-doi="info:doi/10.1371/journal.pone.0001662" data-uri="info:doi/10.1371/journal.pone.0001662.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001662-g003" data-doi="info:doi/10.1371/journal.pone.0001662" data-uri="info:doi/10.1371/journal.pone.0001662.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001662">Reader Comments (4)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001662" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We describe a high throughput gene expression platform based on microfluidic dynamic arrays. This system allows 2,304 simultaneous real time PCR gene expression measurements in a single chip, while requiring less pipetting than is required to set up a 96 well plate. We show that one can measure the expression of 45 different genes in 18 tissues with replicates in a single chip. The data have excellent concordance with conventional real time PCR and the microfluidic dynamic arrays show better reproducibility than commercial DNA microarrays.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Spurgeon SL, Jones RC, Ramakrishnan R (2008) High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array. PLoS ONE 3(2):
          e1662.
            doi:10.1371/journal.pone.0001662</p><p><strong>Academic Editor: </strong>Cathal Seoighe, University of Cape Town, South Africa</p><p><strong>Received:</strong> September 27, 2007; <strong>Accepted:</strong> January 29, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 Spurgeon et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>All the authors are employees of Fluidigm Corp.</p><p><strong>Competing interests:</strong> This is work conducted using a platform which has been commercially released by Fluidigm Corp.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Large scale gene expression analysis has become an essential tool for many biological and medical investigations. DNA microarrays are extremely powerful tools for such studies in that they allow one to probe virtually the entire transcriptome to give an overall picture of gene expression behavior <a href="#pone.0001662-Making1">[1]</a>. However, their results for any given gene are often noisy or ambiguous <a href="#pone.0001662-Liang1">[2]</a>. Therefore it has become common practice to check the results of a genome-wide study with real time PCR (RT-qPCR), which has excellent sensitivity, dynamic range, and reproducibility and is widely regarded as the “gold standard” measurement. Unfortunately RT-qPCR is a low throughput technique, which limits the number genes that can be verified. Furthermore, many microarray studies are concluded by identifying a set of 20–100 genes which are the most important determinants of the phenomenon of interest <a href="#pone.0001662-Singh1">[3]</a>. Validation studies or practical medical application then require measurement of those genes on much larger sample sets than are practical with conventional microarrays. It is therefore of great interest to create new automation tools that increase the parallelism and throughput of RT-qPCR.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Microfluidic technology has found a number of applications in biological automation <a href="#pone.0001662-Melin1">[4]</a>. The ability to make arbitrary fluidic manipulations at the nanoliter scale has led to the development of a number of new tools with applications including protein crystallization <a href="#pone.0001662-Anderson1">[5]</a>, single cell gene expression <a href="#pone.0001662-Warren1">[6]</a>, and cell culture <a href="#pone.0001662-ElAli1">[7]</a>. It has been shown that one particularly useful way to apply such small plumbing is in the creation of microfluidic matrixes, or “dynamic arrays”, that let one perform all possible combinatorial assays on a set of reagents while realizing significant economies of scale in both pipetting, labor and reagent consumption <a href="#pone.0001662-Liu1">[8]</a>. Here we show that microfluidic dynamic arrays can be used to perform high throughput gene expression measurements with real time PCR. Single chips were used to measure the expression of 45 genes in 18 different adult and fetal tissues, and the results were compared both with microarray measurements and with conventional RT-qPCR.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Chip Design and Raw Data</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><a href="#pone-0001662-g001">Figure 1A</a> shows a picture of the microfluidic chip used in this study, a 48.48 dynamic array. The chip is mounted on a plastic carrier with interface and containment accumulators and 48 sample inlets and detector inlets. The dimensions of the inlets and the size of the plate conform to SBS standards <a href="#pone.0001662-MicroplatesHeight1">[9]</a> and are robotically compatible. The integrated fluidic circuit (IFC) is a network of fluid lines, NanoFlex™ valves and chambers. The NanoFlex™ valves are made of an elastomeric material which deflects under pressure to create a tight seal and are used to regulate the flow of liquids in the IFC. Prior to loading, the chip is primed using the NanoFlex™ IFC controller which pressurizes the control lines and closes the interface valves. Individual samples are pipetted into the sample inlets and the gene expression assays are pipetted into the detector inlets. The chip is then placed back on the NanoFlex™ IFC controller for loading and mixing. During this process, pressure is applied to the fluid in the sample inlets and the fluid is pushed into the fluid lines and then into the individual wells. At the same time the fluid in the detector inlets is pushed into the fluid lines. Mixing of the two fluids is prevented by the closed interface valve. The containment valves are then closed and the interface valves opened which pushes the reagents in the detector inlets into the individual reaction chambers to allow for mixing. At the end of the mixing the interface valves are again closed and the chip is ready for cycling. This process takes approximately 55 minutes. While some details of this chip and process have been previously described <a href="#pone.0001662-Spletter1">[10]</a>, <a href="#pone.0001662-Liu2">[11]</a>, more details on the design, function and characterization of the dynamic array chip are in preparation (Unger M, et al. The Dynamic Array: High Throughput Real-Time Quantitative PCR by Microfluidic Large Scale Integration, manuscript in preparation, 2008).</p>
<div class="figure" id="pone-0001662-g001"><div class="img"><a name="pone-0001662-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001662" data-uri="info:doi/10.1371/journal.pone.0001662.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001662.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001662.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The 48.48 dynamic array chip.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A. Photograph of a 48.48 dynamic array chip showing the position of the sample inlets and the detector inlets in which the gene expression assay reagents are added. The check valves allow pressure to be applied and released. The accumulators provide reservoirs to hold the pressure and keep the valves closed during the reaction. The integrated Fluidic Circuit (IFC) is in the center of the chip. This is a network of fluid lines, NanoFlex™ valves and reaction chambers. The insert shows a blow-up of a portion of the IFC with one of the 2304 individual reaction chambers (RC) and the associated containment valves (CV) and interface valve (IV) There are two containment valves and one interface valve associated with each reaction chamber. B. A computer generated image (heat map) of a 48.48 dynamic array chip obtained after thermal cycling of the chip. Each of the squares represents 1 reaction chamber from the chip. The color indicates the C<sub>T</sub> value according to the legend shown on the right. Black chambers indicate a C<sub>T</sub>&gt;40.</p>
<span>doi:10.1371/journal.pone.0001662.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>After loading and mixing is complete, the chip is placed into the instrument for thermal cycling. Once the cycling program has started, the chip is imaged at the end of each cycle. When the cycling is completed, analysis software generates PCR curves for each of the 2304 wells. <a href="#pone-0001662-g001">Figure 1B</a> shows a computer generated image of the data obtained from one of the chips used for the gene expression analysis study. Forty-five gene expression assays and preamplified cDNA from 18 different tissues and 3 control preamplified cDNAs were included in this study. A complete list of the gene expression assays and cDNA samples can be found in <a href="#pone.0001662.s003">Table S1</a>. Each square of the image represents one of the 2304 wells on the chip. The different colors indicate different C<sub>T</sub> values according to the color key shown on the right side of the figure. Squares that are black indicate that the C<sub>T </sub>is greater than 40. For the 48.48 dynamic array chip used here amplification curves are visible over 6 logs from a C<sub>T</sub> value of 7.9 to 25.7 for a gene expression assay with an efficiency of 89% (<a href="#pone.0001662.s001">Fig. S1</a>, <a href="#pone.0001662.s004">Table S2</a>, supplemental data), which shows data across a 10-fold serial dilution across 288 replicates of the same gene. The quality of data is very high; for example, for the highest concentration (Curve set 1, relative concentration 1×10<sup>0</sup>), the mean Ct value was 7.9, and the standard deviation across the 288 Cts was only 0.089. This is extremely high quality data. We routinely test our chips by using a similar concentration of sample across the entire chip (2304 replicates), and typically obtain a standard deviation of about 0.1 across 2304 replicates (data not shown).The last curve has a C<sub>T</sub> of 25.7; the mean copy number for that sample was measured by microfluidic digital PCR and found to be 8.</p>


<h4>Validation of Gene Expression Data</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We compared a subset of the gene expression data obtained here to both microarray data reported for the same tissues and genes and to conventional real time PCR on microliter volume samples. GeneNote is a publicly available data set of gene expression in normal human tissues <a href="#pone.0001662-Shmueli1">[12]</a>. The data can be viewed at <a href="http://bioinfo2.weizmann.ac.il/cgi-bin/genenote/home_page.pl">http://bioinfo2.weizmann.ac.il/cgi-bin/g​enenote/home_page.pl</a> . Data for 8 of the 18 tissues used in the gene expression study were found in the GeneNote data base. These were brain, heart, kidney, liver, lung, prostate, muscle and spleen. Data for all but <em>MYH1</em> was found in GeneNote. For each gene the data from two duplicates is reported.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The plots shown in <a href="#pone-0001662-g002">Fig 2</a> A,B, C and D represent comparison data between and within measurement platforms for two tissues and 43 genes. <a href="#pone-0001662-g002">Fig. 2A</a> shows that the microarray data has moderate internal consistency, with a correlation coefficient between replicates of r = 0.933 and a number of significant outlier data points. In <a href="#pone-0001662-g002">Fig. 2B</a> the mean of the two microarray duplicates is compared to the data from the microfluidic dynamic array obtained with preamplified cDNA (PA cDNA). For these genes and tissues the correlation coefficient is only 0.864, a value that is slightly worse than the internal consistency of the micrarray data. A similar result was obtained when conventional RT-qPCR data from microliter volume samples obtained with cDNA was compared to the microarray data (<a href="#pone-0001662-g002">Fig. 2C</a>). Reasons for this could include a combination of systematic differences between measurement platforms as well as differences in the source tissue or in the cDNA preparation process. However, the internal noise in the microarray data is at least as large a contribution as those factors combined. The microfluidic dynamic array data obtained with PA cDNA has a high correlation with the microliter RT-qPCR data obtained with cDNA, with a correlation coefficient r = 0.989 (<a href="#pone-0001662-g002">Fig 2D</a>). This is similar to the result of a comparison between microfluidic dynamic array data for PA cDNA and microliter RT-qPCR data obtained with PA cDNA (r = 0.989). The reproducibility of the microfluidic dynamic array data for replicates within a chip and between different chips is very good (r&gt;0.99) and is shown in <a href="#pone-0001662-g002">Fig. 2E and F</a>. Therefore microfluidic digital PCR performed on a dynamic array is able to obtain high throughput gene expression data that is essentially identical in quality to conventional microliter RT-qPCR and of superior quality to publicly available microarray data from the same tissue type.</p>
<div class="figure" id="pone-0001662-g002"><div class="img"><a name="pone-0001662-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001662" data-uri="info:doi/10.1371/journal.pone.0001662.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001662.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001662.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Comparisons of data.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The correlation coefficient r is shown above each plot. For details regarding these comparisons see <a href="#s3">Materials and Methods</a>. A. Comparison of Duplicate samples from the GeneNote database<a href="#pone.0001662-Spletter1">[10]</a> for 45 genes and two tissues, heart and liver. B. Comparison of data from the 48.48 dynamic array obtained for preamplifid cDNA (PA cDNA) and data from the GeneNote database for the same genes and tissues. C. Comparison of the data from the 7900HT Sequence Detection System obtained with cDNA and data from the GeneNote database. D. Comparison of data from the 7900HT obtained with cDNA and the 48.48 dynamic array obtained with PA cDNA for the same genes and tissues. E. Comparison of the ΔΔC<sub>T</sub> values for replicates on one of the 48.48 dynamic array chips used for gene expression analysis for all 44 genes and 18 tissues. F. Pair-wise comparison of the mean ΔΔC<sub>T </sub>values from two chips for the same 44 genes and 18 tissues.</p>
<span>doi:10.1371/journal.pone.0001662.g002</span></div>

<h4>Tissue specificity of gene expression</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We measured the expression of both muscle-specific genes and housekeeping genes. <a href="#pone-0001662-g003">Fig. 3A</a> shows the results from the muscle specific genes across all tissues, grouped by gene name. <em>MYH1(MHC IIx/d)</em>, <em>MYH2(MHC IIa)</em>, <em>MYH6(MYHCA)</em> and <em>MYH7(MYHCB)</em> are genes that code for isoforms of myosin heavy chain which together with myosin light chain form the subunits of myosin. Each molecule of myosin consists of two identical subunits of myosin heavy chain and two pairs of non-identical subunits of myosin light chain. Thirteen isoforms of myosin heavy chain have been identified, and their distribution shows both tissue and developmental specificity <a href="#pone.0001662-Weiss1">[13]</a>, <a href="#pone.0001662-Berg1">[14]</a>. The expression of these genes also varies in response to aging, exercise, and disease <a href="#pone.0001662-BasselDuby1">[15]</a>. <em>MYH1</em> and <em>MYH2</em> are expressed primarily in adult skeletal muscle, which is consistent with the gene expression results observed here. The products of <em>MYH6</em> and <em>MYH7</em> are the predominant forms of myosin heavy chain expressed in cardiac muscle. <em>MYH7</em> is also expressed in skeletal muscle in slow twitch (Type I) muscle fibers, but the expression of <em>MYH6</em> shows a high degree of specificity for cardiac tissue. Our observations are consistent with this: the expression of <em>MYH6</em> was more than 100-fold higher in heart than muscle, and <em>MYH7</em> was similarly 4–5 fold higher. Interestingly, the absolute abundance of <em>MYH7</em> transcripts in heart was higher than <em>MYH6</em> (<a href="#pone-0001662-g003">Fig. 3A</a>).</p>
<div class="figure" id="pone-0001662-g003"><div class="img"><a name="pone-0001662-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001662" data-uri="info:doi/10.1371/journal.pone.0001662.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001662.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001662.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001662.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001662.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001662.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Estimated copy numbers.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Using the standard curve shown in <a href="#pone.0001662.s002">Fig. S2</a> the actual number of copies in the reaction chambers was estimated from the mean C<sub>T</sub> values of all the data obtained on the three chips used previously. The data is plotted as the log<sub>10</sub>. Error bars were determined from the mean C<sub>T</sub> values +/−one standard deviation. The raw data for the standard curve in <a href="#pone.0001662.s002">Figure S2</a> is presented in <a href="#pone.0001662.s006">Table S4</a>.</p>
<span>doi:10.1371/journal.pone.0001662.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The other five genes in this group are all related to muscle development and regulation of muscle mass. <em>MYOD1(MYOD)</em> codes for a myogenic regulatory factor (MRF) and is crucial for the differentiation of skeletal muscle <a href="#pone.0001662-Berkes1">[16]</a>. The product of <em>MyoD1</em> is responsible for the regulation of a wide range of genes during myogenesis <a href="#pone.0001662-Bergstrom1">[17]</a>–<a href="#pone.0001662-Zhao1">[19]</a>. In the work done here expression of this gene was detected primarily in muscle and fetal muscle which is consistent with previous reports <a href="#pone.0001662-Dias1">[20]</a>. However, we also detected expression at much lower levels in heart, spleen, testis and fetal testis as shown in <a href="#pone-0001662-g003">Fig. 3A</a>. The proteins encoded by <em>GDF8</em>(<em>MSTN</em>) and <em>GDF11</em>(<em>BMP11</em>) are both members of the TGF-β superfamily of transcription factors <a href="#pone.0001662-Chang1">[21]</a> and are closely related proteins <a href="#pone.0001662-Lee1">[22]</a>. The product of <em>GDF8</em> is a negative regulator of skeletal muscle growth also known as myostatin and is expressed in skeletal muscle<a href="#pone.0001662-Lee1">[22]</a>. Mutations in this gene result in increased muscle mass in mice, cattle, dogs and humans <a href="#pone.0001662-Mosher1">[23]</a>, <a href="#pone.0001662-Saunders1">[24]</a>. In the work reported here (<a href="#pone-0001662-g003">Fig. 3A</a>) we detected expression of <em>GDF8</em> in muscle tissue and at lower levels in other tissues as well. This result is similar to data reported for this gene in the GNF SymAtlas<a href="#pone.0001662-GNF1">[25]</a>. The protein encoded by <em>GDF11</em> is a bone morphogenic protein <a href="#pone.0001662-Ducy1">[26]</a>which is involved in the regulation of anterior/posterior axial patterning in the developing embryo <a href="#pone.0001662-McPherron1">[27]</a>. The expression of this gene is higher in many other tissues than in muscle as shown in <a href="#pone-0001662-g003">Fig. 3A</a> and the GNF SymAtlas <a href="#pone.0001662-GNF1">[25]</a>. The last two genes in this group, <em>FBXO32</em> (<em>ATROGIN1</em> or <em>MAFbx</em>) and <em>TRIM63</em>(<em>MURF1</em>) both code for E3 ubiquitin ligases and their overexpression is associated with muscle atrophy <a href="#pone.0001662-Bodine1">[28]</a>. The expression of both these genes was found in muscle. Lower levels of expression of <em>FBXO32</em> were detected in all the other tissues, similar to the data reported in GeneNote <a href="#pone.0001662-Shmueli1">[12]</a>. The expression of <em>TRIM63</em> was detected more specifically in muscle, heart and fetal muscle than in other tissues.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In comparison to the muscle specific genes, the set of housekeeping genes is expressed across all of the tissues (<a href="#pone-0001662-g003">fig 3B</a>). Of the genes included here, expression of CSNK2B was the most stable across all of the tissues as determined by both geNorm <a href="#pone.0001662-Vandesompele1">[29]</a> and Normfinder <a href="#pone.0001662-Andersen1">[30]</a> and for this reason was used as the reference gene for the relative gene expression data. It is interesting to note that the relative expression levels of pairs of housekeeping genes are not necessarily constant between tissues. This illustrates one of the pitfalls of normalizing gene expression results to another gene – it is extremely challenging, if not impossible, to find genes which are expressed at constant levels across all tissues. Reliable quantitative analysis of gene expression levels requires a procedure to obtain absolute calibration, which can be accomplished either by creating defined concentrations of synthetic cDNA for each gene, or less laboriously by performing digital PCR on the original cDNA sample <a href="#pone.0001662-Warren1">[6]</a>. Digital PCR involves the isolation and amplification of single DNA molecules, which is made possible by the use of the Fluidigm digital array chip <a href="#pone.0001662-Warren1">[6]</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>In conclusion, we have shown that the use of microfluidic dynamic array chips for real-time gene expression analysis is a rapid and reliable method for high throughput gene expression analysis. The 48.48 dynamic array chip enables 2,304 individual reactions to be analyzed with less pipetting than is required to set up of a conventional 96 well plate. The data has excellent concordance with conventional real time PCR and has better reproducibility than DNA microarrays.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Instrumentation and microfluidic chips</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>48.48 dynamic array chips and 12.765 digital array chips were from Fluidigm Corporation. The NanoFlex ™ 4-IFC Controller and the BioMark ™ Real-Time PCR System are manufactured by Fluidigm Corporation. The Νanoflex ™ 4-IFC Controller utilizes pressure to control the valves in the chips and load samples and gene expression assay reagents into the reaction chambers. The BioMark system is a real-time PCR instrument designed to thermal cycle these microfluidic chips and image the data in real time. Pre-amplification reactions were done in a GeneAmp PCR System 9700 from Applied Biosystems. Real-time PCR reactions in 384 well plates utilized an ABI Prism 7900HT Sequence Detection instrument from Applied Biosystems.</p>


<h4>Real-Time PCR Assays</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>First strand cDNA samples from 18 different human tissues were purchased from OriGene, Inc. Universal human cDNA was purchased from BioChain Institute, Inc. Taq Man Universal Master Mix, TaqMan PreAmp Master Mix and gene expression assays were from Applied Biosystems. Prior to use the 20x gene expression assays were diluted to 10x using the DA Assay Loading Reagent (Fluidigm PN 85000735). For preamplification of cDNA samples the TaqMan PreAmp Master Mix was used according to the manufacturer's directions except for the amount of the final dilution <a href="#pone.0001662-TaqMan1">[31]</a>. Reactions used either 6.25 µL of cDNA in a 25 µL reaction or 1.25 µL of cDNA in a 5 µL reaction and were cycled using the recommended program for 14 cycles. At the end of the cycling program the reactions were diluted 1:5. Preamplified cDNA prepared in this way was either utilized immediately or stored at −20°C until needed. Validation of the preamplification reaction with the 45 gene expression assays used in this study was done on the 7900 following the protocol as described by the manufacturer. Universal cDNA was used for this analysis and one of the genes, <em>CSNK2B</em>, was used as the reference gene. Only 3 of the 44 genes expression assays tested gave a value outside of +/−1.5 ΔΔC<sub>T</sub>. The values obtained with the assays for <em>ACTB</em> and <em>ENSA</em> were +3.24 and +2.82, respectively. The ΔΔC<sub>T</sub> value obtained for the <em>GAPDH</em> assay was 1.52 with the universal cDNA but was below that value when tested with cDNA from four other tissues. For the assay for <em>CCND1</em> expression the ΔΔC<sub>T</sub> value was +1.35 but was slightly higher when this analysis was repeated with other cDNA samples. The mean value obtained for 5 different cDNA samples was +1.63 ΔΔC<sub>T </sub>for the <em>CCND1</em> assay. For real-time gene expression assays on the dynamic array chips, the TaqMan Universal Master Mix was modified by the addition of 1/10 volume of DA Sample Loading Reagent (Fluidigm PN 85000735). Preamplified cDNA was added to the modified 2x TaqMan Universal Master Mix to make the final concentration of Master Mix 1.1x in the samples. Prior to loading the samples and assay reagents into the inlets, the chip was primed in the NanoFlex™ 4-IFC Controller. Five µL of sample prepared as described was then loaded into each sample inlet of the dynamic array chip and 5 µL of 10x gene expression assay mix was loaded into each detector inlet. The chip was then placed on the NanoFlex™ 4-IFC Controller for loading and mixing. After approximately 55 minutes the chip was ready for thermal cycling and detection of the reaction products on the BioMark™ Real-Time PCR System. The cycling program used consisted of 10min at 95°C followed by 40 cycles of 95°C for 15 sec and 1 min at 60°C. Data was analyzed using the BioMark Gene Expression Data Analysis software to obtain Ct values and/or ΔΔCt values. The number of replicates and the composition of the samples varied depending on the particular experiment. Details of individual experiments are given in the Results and Discussion Section. Reactions for analysis on the 7900HT Sequence Detection System consisted of 5 µL of TaqMan Universal Master Mix, 1 µL of 10x Gene Expression Assay Mix and either cDNA or preamplified cDNA for a final volume of 10 µL per well of a 384 well plate. The cycling program consisted of a 10 min incubation at 95°C followed by 40 cycles of 95°C for 15sec and 60°c for 1 min.</p>


<h4>12.765 Digital array chips</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>For each sample 13 µL of sample reaction mix was prepared using 10 µL of TaqMan Universal Master Mix, 1 µL DA Sample Loading Reagent (Fluidigm PN 85000735) and 2 µL of the appropriate 10x gene expression assay previously modified by the addition of DA Assay Loading Reagent (Fluidigm PN 85000735). Seven µL of cDNA, preamplified cDNA or water was added to the sample reaction mix. The solution was mixed and 9.5 µL was added to each of two sample inlets. The chip was then placed on the NanoFlex™ IFC Controller and the samples were automatically loaded into the individual reaction chambers. The chip was then placed on the BioMark System for thermal cycling and detection of products. Data was analyzed using the BioMark digital array software and the numbers of positive chambers were corrected to estimate the true number of copies <a href="#pone.0001662-Warren2">[32]</a>. This number was used to determine the number of copies in the original sample.</p>


<h4>Determination of relative gene expression</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Relative gene expression values were determined using the 2<sup>−ΔΔCT</sup> method of Livak and Schmittgen <a href="#pone.0001662-Livak1">[33]</a>. <em>CSNK2B</em> was used as the reference gene and muscle was used as the reference sample. Values for ΔΔC<sub>T</sub> were obtained directly from the software.</p>


<h4>Comparison to Microarray Data and other data comparisons</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The microarray data was obtained from GeneNote <a href="#pone.0001662-Spletter1">[10]</a>. The data used was MAS5.0 Normalized data. Since all of the data from the 48.48 dynamic array chips and the 7900 platform was reported as fold-expression relative to muscle, the data from the microarray for liver and heart was converted to a similar format using muscle as the reference. In cases where microarray data from more than one probe was reported, the data which matched the RT-qPCR data most closely was used. For comparison to the RT-qPCR data the mean of the two duplicates was used. All expression data has been plotted as log<sub>2 </sub>for comparison to RT-qPCR data. For intra- and inter-chip comparisons of 48.48 dynamic array data (<a href="#pone-0001662-g002">Fig. 2</a> E and F) the ΔΔC<sub>T</sub> values obtained from the software were used directly.</p>


<h4>Estimated number of copies in preamplified cDNA samples</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Preamplified cDNA for twelve adult tissues was assayed on a 48.48 dynamic array chip with the gene expression assay for <em>MYH7</em> and a large number of replicates to determine the C<sub>T</sub> values. The number of copies of <em>MYH7</em> transcript per µL of the original solution of preamplified cDNA was then determined for 10 of the samples using 12.765 digital array chips. If necessary the samples were diluted to a concentration suitable for analysis on the digital array chips. From this data the mean number of copies per chamber on the 48.48 dynamic array chip was calculated. For liver and PBL (peripheral blood leukocytes) the expression level for <em>MYH7</em> was too low to measure. The C<sub>T</sub> values determined from the 48.48 dynamic array chip were plotted against the log<sub>10</sub> of the concentration for samples with a mean copy number per chamber &gt; than one. The C<sub>T</sub> for single copy that was determined from the digital array data was included as the zero value on the plot. This plot was used as a standard curve to estimate the number of copies from C<sub>T</sub> values for the samples used in the gene expression study.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001662"></div><a name="pone.0001662.s001" id="pone.0001662.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001662.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">A. Real-Time PCR curves for a ten-fold dilution series of preamplified cDNA generated with a gene expression assay for GAPDH. Each curve represents data from 288 individual reaction chambers. The curves for each dilution in the series agree very well except for four curves for the third dilution in the series. These four curves were localized to four adjacent chambers in on region of the chip. The most likely explanation is the presence of a flaw in this particular chip at that position. B. Plot of CT vs log10 of the dilution. Based on this curve the efficiency in this reaction is 89%. Values for the mean CT value and standard deviations of the curves are shown in <a href="#pone.0001662.s004">Table S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001662.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.92 MB TIF)</p>
<a name="pone.0001662.s002" id="pone.0001662.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001662.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Standard curve constructed from the data in <a href="#pone.0001662.s004">Table S4</a> and the CT value for single copy determined from the digital array chip.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001662.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.18 MB TIF)</p>
<a name="pone.0001662.s003" id="pone.0001662.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001662.s003">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">List of Gene Expression Assays and cDNA samples. All of the assays that were used for this work are listed in the table. Assays in Groups 1 and 2 were used for the gene expression studies. The assays in Group 3 were used for the reproducibility study reported in <a href="#pone.0001662.s005">Table S3</a> along with selected assays from Groups 1 and 2. The efficiency of the assays was measured in 48.48 dynamic array chips using preamplified cDNA for human muscle and with universal cDNA as template. A total of four chips were used. Error values are based on the standard deviation of 2–4 determinations. Only data that included at least five good data points was used to calculate the final number. The cDNAs were purchased from OriGene Technologies (Rockville, MD).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001662.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001662.s004" id="pone.0001662.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001662.s004">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Mean CT values and standard deviation for curves shown in <a href="#pone.0001662.s001">Fig. S1</a>. The data in this table shows the relationship between the relative concentration and CT values for GAPDH. The standard deviation is signifcantly higher for curve 6 which has a CT of 25.7. This CT value represents a mean of 8 copies per chamber as determined by analysis on a 12.765 digital array chip.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001662.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001662.s005" id="pone.0001662.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001662.s005">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Reproducibility of 48.48 dynamic array data. For this study the samples were preamplified cDNA from 12 normal, adult tissues. Four replicates of each sample were loaded into the sample inlets. Forty-eight gene expression assays were used which included fifteen assays for genes related to the immune and inflammatory response(Group3, <a href="#pone.0001662.s003">Table S1</a>) as well as an additional 33 from Groups 1 and 2 in <a href="#pone.0001662.s003">Table S1</a>. The mean of the CT values obtained for each of for four replicates for each of the 12 tissues was determined for all 48 assays on each Chip. The data obtained from each chip was compared in a pairwise fashion to the data from all of the other chips in a scatterplot. A total of 6 chips were run. The values determined for the correlation coefficient r and the slope are shown for all of these comparisons.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001662.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001662.s006" id="pone.0001662.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001662.s006">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Relationship of CT and number of copies for MYH7. Forty-seven detector inlets were loaded with the assay for MYH7 and 4 sample inlets were loaded for each sample on a 48.48 dynamic array chip. The samples contained preamplified cDNA from 12 normal, human tissues. The mean CT value and standard deviation was determined for the data from all of the positive chambers. The copies per µL was measured for each of the samples using the 12.765 digital array chip and the mean copies per 10 nL chamber was calculated from that value.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001662.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Dr. Jian Qin for assistance with the analysis of samples using the 12.765 digital array chips and the processing of data, and Dr Stephen Quake for his assistance in the interpretation of the results. We would also like to acknowledge Aparna Kamath for her help in preparing the figures for this publication.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RR SS RJ. Performed the experiments: SS. Analyzed the data: RR SS. Contributed reagents/materials/analysis tools: RR. Wrote the paper: RR SS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001662-Making1" id="pone.0001662-Making1"></a>Making the most of microarrays (2006) Nat Biotechnol 24: 1039.  <ul class="find" data-citedArticleID="1061700" data-doi="10.1038/nbt0906-1039"><li><a href="http://dx.doi.org/10.1038/nbt0906-1039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nat+Biotechnol" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nat+Biotechnol%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001662-Liang1" id="pone.0001662-Liang1"></a>Liang P (2007) MAQC papers over the cracks. Nat Biotechnol 25: 27–28. Comment on Nat Biotechnol (2006)  24: 1151–61.  <ul class="find" data-citedArticleID="1061692" data-doi="10.1038/nbt0107-27"><li><a href="http://dx.doi.org/10.1038/nbt0107-27" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAQC+papers+over+the+cracks.+Nat+Biotechnol+25%3A+27%E2%80%9328." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAQC+papers+over+the+cracks.+Nat+Biotechnol+25%3A+27%E2%80%9328.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001662-Singh1" id="pone.0001662-Singh1"></a>Singh D, Febbo PG, Ross K, Jackson DG, Manola J, et al.  (2002) Gene expression correlates of clinical prostate cancer behavior. Cancer Cell  1: 203–209.  <ul class="find" data-citedArticleID="1061714" data-doi="10.1016/s1535-6108(02)00030-2"><li><a href="http://dx.doi.org/10.1016/s1535-6108(02)00030-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+correlates+of+clinical+prostate+cancer+behavior." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+correlates+of+clinical+prostate+cancer+behavior.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001662-Melin1" id="pone.0001662-Melin1"></a>Melin J, Quake SR (2007) Microfluidic Large-Scale Integration: The Evolution of Design Rules for Biological Automation. Annu Rev Biophys Biomol Struct  36: 213–231.  <ul class="find" data-citedArticleID="1061704" data-doi="10.1146/annurev.biophys.36.040306.132646"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.36.040306.132646" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microfluidic+Large-Scale+Integration%3A+The+Evolution+of+Design+Rules+for+Biological+Automation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microfluidic+Large-Scale+Integration%3A+The+Evolution+of+Design+Rules+for+Biological+Automation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001662-Anderson1" id="pone.0001662-Anderson1"></a>Anderson MJ, DeLaBarre B, Raghunathan A, Palsson BO, Brunger AT, et al.  (2007) Crystal structure of a hyperactive <em>Escherichia coli</em> glycerol kinase mutant Gly230 –&gt; Asp obtained using microfluidic crystallization devices. Biochemistry  46: 5722–5731.  <ul class="find" data-citedArticleID="1061666" data-doi="10.1021/bi700096p"><li><a href="http://dx.doi.org/10.1021/bi700096p" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+a+hyperactive+Escherichia+coli+glycerol+kinase+mutant+Gly230+%E2%80%93%26gt%3B+Asp+obtained+using+microfluidic+crystallization+devices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+a+hyperactive+Escherichia+coli+glycerol+kinase+mutant+Gly230+%E2%80%93%26gt%3B+Asp+obtained+using+microfluidic+crystallization+devices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001662-Warren1" id="pone.0001662-Warren1"></a>Warren L, Bryder D, Weissman IL, Quake SR (2006) Transcription factor profiling in individual hematopoietic progenitors by digital RT-PCR. Proc Natl Acad Sci U S A  103: 17807–17812.  <ul class="find" data-citedArticleID="1061722" data-doi="10.1073/pnas.0608512103"><li><a href="http://dx.doi.org/10.1073/pnas.0608512103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription+factor+profiling+in+individual+hematopoietic+progenitors+by+digital+RT-PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription+factor+profiling+in+individual+hematopoietic+progenitors+by+digital+RT-PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001662-ElAli1" id="pone.0001662-ElAli1"></a>El-Ali J, Sorger PK, Jensen KF (2006) Cells on chips. Nature  442: 403–411.  <ul class="find" data-citedArticleID="1061686" data-doi="10.1038/nature05063"><li><a href="http://dx.doi.org/10.1038/nature05063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cells+on+chips." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cells+on+chips.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001662-Liu1" id="pone.0001662-Liu1"></a>Liu J, Hansen C, Quake SR (2003) Solving the “world-to-chip” interface problem with a microfluidic matrix. Anal Chem  75: 4718–4723.  <ul class="find" data-citedArticleID="1061694" data-doi="10.1021/ac0346407"><li><a href="http://dx.doi.org/10.1021/ac0346407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solving+the+%E2%80%9Cworld-to-chip%E2%80%9D+interface+problem+with+a+microfluidic+matrix." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solving+the+%E2%80%9Cworld-to-chip%E2%80%9D+interface+problem+with+a+microfluidic+matrix.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001662-MicroplatesHeight1" id="pone.0001662-MicroplatesHeight1"></a>  Microplates-Height Dimensions [ANSI/SBS 2-2004] (<a href="http://www.sbsonline.com/msdc/pdf/ANSI_SBS_2-2004.pdf">http://www.sbsonline.com/msdc/pdf/ANSI_S​BS_2-2004.pdf</a>).  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001662-Spletter1" id="pone.0001662-Spletter1"></a>Spletter ML, Liu J, Liu J, Su H, Giniger E, Komiyama T, Quake S, Luo L (2007) Lola regulates <em>Drosophila</em> olfactory projection neuron identity and targeting specificity Neural Development  2: 14.  (<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgiartid1947980">http://www.pubmedcentral.nih.gov/article​render.fcgiartid1947980</a>).  <ul class="find" data-citedArticleID="1061716" data-doi="10.1186/1749-8104-2-14"><li><a href="http://dx.doi.org/10.1186/1749-8104-2-14" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lola+regulates+Drosophila+olfactory+projection+neuron+identity+and+targeting+specificity" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lola+regulates+Drosophila+olfactory+projection+neuron+identity+and+targeting+specificity%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001662-Liu2" id="pone.0001662-Liu2"></a>Liu J, Hansen C, Quake SR (2003) Solving the “world-to-chip” interface problem with a microfluidic matrix. Anal Chem.  75: 4718–4723.  <ul class="find" data-citedArticleID="1061696" data-doi="10.1021/ac0346407"><li><a href="http://dx.doi.org/10.1021/ac0346407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solving+the+%E2%80%9Cworld-to-chip%E2%80%9D+interface+problem+with+a+microfluidic+matrix." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solving+the+%E2%80%9Cworld-to-chip%E2%80%9D+interface+problem+with+a+microfluidic+matrix.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001662-Shmueli1" id="pone.0001662-Shmueli1"></a>Shmueli O, Horn-Saban S, Chalifa-Caspi V, Shmoish M, Ophir R, et al.  (2003) GeneNote: whole genome expression profiles in normal human tissues. C R Biol  326: 1067–1072.  <ul class="find" data-citedArticleID="1061712" data-doi="10.1016/j.crvi.2003.09.012"><li><a href="http://dx.doi.org/10.1016/j.crvi.2003.09.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneNote%3A+whole+genome+expression+profiles+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneNote%3A+whole+genome+expression+profiles+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001662-Weiss1" id="pone.0001662-Weiss1"></a>Weiss A, Leinwand LA (1996) The mammalian myosin heavy chain gene family. Annu Rev Cell Dev Biol  12: 417–439.  <ul class="find" data-citedArticleID="1061726" data-doi="10.1146/annurev.cellbio.12.1.417"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.12.1.417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mammalian+myosin+heavy+chain+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mammalian+myosin+heavy+chain+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001662-Berg1" id="pone.0001662-Berg1"></a>Berg JS, Powell BC, Cheney RE (2001) A millennial myosin census. Mol Biol Cell  12: 780–794.  <ul class="find" data-citedArticleID="1061670" data-doi="10.1091/mbc.12.4.780"><li><a href="http://dx.doi.org/10.1091/mbc.12.4.780" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+millennial+myosin+census." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+millennial+myosin+census.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001662-BasselDuby1" id="pone.0001662-BasselDuby1"></a>Bassel-Duby R, Olson EN (2006) Signaling Pathways in Skeletal Muscle Remodeling. Annu Rev Biochem  75: 19–37.  <ul class="find" data-citedArticleID="1061668" data-doi="10.1146/annurev.biochem.75.103004.142622"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.75.103004.142622" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling+Pathways+in+Skeletal+Muscle+Remodeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling+Pathways+in+Skeletal+Muscle+Remodeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001662-Berkes1" id="pone.0001662-Berkes1"></a>Berkes CA, Tapscott SJ (2005) MyoD and the transcriptional control of myogenesis. Semin Cell Dev Biol  16: 585–595.  <ul class="find" data-citedArticleID="1061674" data-doi="10.1016/j.semcdb.2005.07.006"><li><a href="http://dx.doi.org/10.1016/j.semcdb.2005.07.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MyoD+and+the+transcriptional+control+of+myogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MyoD+and+the+transcriptional+control+of+myogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001662-Bergstrom1" id="pone.0001662-Bergstrom1"></a>Bergstrom DA, Penn BH, Strand A, Perry RLS, Rudnicki MA, et al.  (2002) Promoter-specific regulation of MyoD binding and signal transduction cooperate to pattern gene expression. Mol Cell  9: 587–600.  <ul class="find" data-citedArticleID="1061672" data-doi="10.1016/s1097-2765(02)00481-1"><li><a href="http://dx.doi.org/10.1016/s1097-2765(02)00481-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Promoter-specific+regulation+of+MyoD+binding+and+signal+transduction+cooperate+to+pattern+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Promoter-specific+regulation+of+MyoD+binding+and+signal+transduction+cooperate+to+pattern+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001662-Delgado1" id="pone.0001662-Delgado1"></a>Delgado I, Huang X, Jones S, Zhang L, Hatcher R, et al.  (2003) Dynamic gene expression during the onset of myoblast differentiation in vitro. Genomics  82: 109–121.  <ul class="find" data-citedArticleID="1061680" data-doi="10.1016/s0888-7543(03)00104-6"><li><a href="http://dx.doi.org/10.1016/s0888-7543(03)00104-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+gene+expression+during+the+onset+of+myoblast+differentiation+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+gene+expression+during+the+onset+of+myoblast+differentiation+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001662-Zhao1" id="pone.0001662-Zhao1"></a>Zhao P, Seo J, Wang Z, Wang Y, Shneiderman B, et al.  (2003) In vivo filtering of in vitro expression data reveals MyoD targets. C R Biol  326: 1049–1065.  <ul class="find" data-citedArticleID="1061728" data-doi="10.1016/j.crvi.2003.09.035"><li><a href="http://dx.doi.org/10.1016/j.crvi.2003.09.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+filtering+of+in+vitro+expression+data+reveals+MyoD+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+filtering+of+in+vitro+expression+data+reveals+MyoD+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001662-Dias1" id="pone.0001662-Dias1"></a>Dias P, Parham DM, Shapiro DN, Webber BL, Houghton PJ (1990) Myogenic regulatory protein (MyoD1) expression in childhood solid tumors: diagnostic utility in rhabdomyosarcoma. Am J Pathol  137: 1283–1291.  <ul class="find" data-citedArticleID="1061682"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Myogenic+regulatory+protein+%28MyoD1%29+expression+in+childhood+solid+tumors%3A+diagnostic+utility+in+rhabdomyosarcoma.&amp;auth=&amp;atitle=Myogenic+regulatory+protein+%28MyoD1%29+expression+in+childhood+solid+tumors%3A+diagnostic+utility+in+rhabdomyosarcoma." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Myogenic+regulatory+protein+%28MyoD1%29+expression+in+childhood+solid+tumors%3A+diagnostic+utility+in+rhabdomyosarcoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Myogenic+regulatory+protein+%28MyoD1%29+expression+in+childhood+solid+tumors%3A+diagnostic+utility+in+rhabdomyosarcoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001662-Chang1" id="pone.0001662-Chang1"></a>Chang H, Brown CW, Matzuk MM (2002) Genetic analysis of the mammalian transforming growth factor-beta superfamily. Endocr Rev  23: 787–823.  <ul class="find" data-citedArticleID="1061678" data-doi="10.1210/er.2002-0003"><li><a href="http://dx.doi.org/10.1210/er.2002-0003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analysis+of+the+mammalian+transforming+growth+factor-beta+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analysis+of+the+mammalian+transforming+growth+factor-beta+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001662-Lee1" id="pone.0001662-Lee1"></a>Lee SJ (2004) Regulation of muscle mass by myostatin. Annu Rev Cell Dev Biol  20: 61–86.  <ul class="find" data-citedArticleID="1061690" data-doi="10.1146/annurev.cellbio.20.012103.135836"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.20.012103.135836" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+muscle+mass+by+myostatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+muscle+mass+by+myostatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001662-Mosher1" id="pone.0001662-Mosher1"></a>Mosher DS, Quignon P, Bustamante CD, Sutter NB, Mellersh CS, et al.  (2007) A mutation in the myostatin gene increases muscle mass and enhances racing performance in heterozygote dogs. PLoS Genet  3(5): e79.  doi:10.1371/journal.pgen.0030079.  <ul class="find" data-citedArticleID="1061708" data-doi="10.1371/journal.pgen.0030079.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030079.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mutation+in+the+myostatin+gene+increases+muscle+mass+and+enhances+racing+performance+in+heterozygote+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mutation+in+the+myostatin+gene+increases+muscle+mass+and+enhances+racing+performance+in+heterozygote+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001662-Saunders1" id="pone.0001662-Saunders1"></a>Saunders MA, Good JM, Lawrence EC, Ferrell RE, Li WH, et al.  (2006) Human adaptive evolution at Myostatin (GDF8), a regulator of muscle growth. Am J Hum Genet  79: 1089–1097.  <ul class="find" data-citedArticleID="1061710" data-doi="10.1086/509707"><li><a href="http://dx.doi.org/10.1086/509707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+adaptive+evolution+at+Myostatin+%28GDF8%29%2C+a+regulator+of+muscle+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+adaptive+evolution+at+Myostatin+%28GDF8%29%2C+a+regulator+of+muscle+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001662-GNF1" id="pone.0001662-GNF1"></a>  GNF SymAtlas. Available at <a href="http://symatlas.gnf.org/SymAtlas/">http://symatlas.gnf.org/SymAtlas/</a> Accessed 18 Sept. 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001662-Ducy1" id="pone.0001662-Ducy1"></a>Ducy P, Karsenty G (2000) The family of bone morphogenetic proteins. Kidney Int  57: 2207–2214.  <ul class="find" data-citedArticleID="1061684" data-doi="10.1046/j.1523-1755.2000.00081.x"><li><a href="http://dx.doi.org/10.1046/j.1523-1755.2000.00081.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+family+of+bone+morphogenetic+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+family+of+bone+morphogenetic+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001662-McPherron1" id="pone.0001662-McPherron1"></a>McPherron AC, Lawler AM, Lee SJ (1999) Regulation of anterior/posterior patterning of the axial skeleton by growth/differentiation factor 11. Nat Genet  22: 260–264.  <ul class="find" data-citedArticleID="1061702"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+anterior%2Fposterior+patterning+of+the+axial+skeleton+by+growth%2Fdifferentiation+factor+11.&amp;auth=&amp;atitle=Regulation+of+anterior%2Fposterior+patterning+of+the+axial+skeleton+by+growth%2Fdifferentiation+factor+11." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+anterior%2Fposterior+patterning+of+the+axial+skeleton+by+growth%2Fdifferentiation+factor+11." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+anterior%2Fposterior+patterning+of+the+axial+skeleton+by+growth%2Fdifferentiation+factor+11.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001662-Bodine1" id="pone.0001662-Bodine1"></a>Bodine SC, Latres E, Baumhueter S, Lai VK, Nunez L, et al.  (2001) Identification of ubiquitin ligases required for skeletal muscle atrophy. Science  294: 1704–1708.  <ul class="find" data-citedArticleID="1061676" data-doi="10.1126/science.1065874"><li><a href="http://dx.doi.org/10.1126/science.1065874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+ubiquitin+ligases+required+for+skeletal+muscle+atrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+ubiquitin+ligases+required+for+skeletal+muscle+atrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001662-Vandesompele1" id="pone.0001662-Vandesompele1"></a>Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, et al.  (2002) Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol  3:   RESEARCH0034.  <ul class="find" data-citedArticleID="1061720"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes.&amp;auth=&amp;atitle=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001662-Andersen1" id="pone.0001662-Andersen1"></a>Andersen CL, Jensen JL, Ørntoft TF (2004) Normalization of real-time quantitative reverse transcription-PCR data: a model-based variance estimation approach to identify genes suited for normalization, applied to bladder and colon cancer data sets. Cancer Res  64: 5245–5250.  <ul class="find" data-citedArticleID="1061664" data-doi="10.1158/0008-5472.can-04-0496"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-04-0496" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Normalization+of+real-time+quantitative+reverse+transcription-PCR+data%3A+a+model-based+variance+estimation+approach+to+identify+genes+suited+for+normalization%2C+applied+to+bladder+and+colon+cancer+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Normalization+of+real-time+quantitative+reverse+transcription-PCR+data%3A+a+model-based+variance+estimation+approach+to+identify+genes+suited+for+normalization%2C+applied+to+bladder+and+colon+cancer+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001662-TaqMan1" id="pone.0001662-TaqMan1"></a>  TaqMan PreAmp Master Mix Kit Protocol. Applied Biosystems Part#: 4366127 Rev D Available: http://www3.appliedbiosystems.com/cms/gr​oups/mcb_support/documents/generaldocume​nts/cms_042472.pdfAccessed 18 Sept 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001662-Warren2" id="pone.0001662-Warren2"></a>Warren LA, Weinstein JA, Quake SR (2007) The Digital Array Response Curve.  Available: <a href="http://thebigone.stanford.edu/quake/publications/DigResCurve.pdf">http://thebigone.stanford.edu/quake/publ​ications/DigResCurve.pdf</a> Accessed 18 Sept 2007.  <ul class="find" data-citedArticleID="1061724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Digital+Array+Response+Curve.&amp;auth=&amp;atitle=The+Digital+Array+Response+Curve." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Digital+Array+Response+Curve." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Digital+Array+Response+Curve.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001662-Livak1" id="pone.0001662-Livak1"></a>Livak KJ, Schmittgen TD (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods  25: 402–408.  <ul class="find" data-citedArticleID="1061698" data-doi="10.1006/meth.2001.1262"><li><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.XML" value="65804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.PDF" value="701318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g001.PNG_L" value="806362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g001.PNG_M" value="226748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g001.PNG_S" value="9980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g001.TIF" value="1447162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g001.PNG_I" value="70947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g002.PNG_L" value="347156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g002.PNG_M" value="113273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g002.PNG_S" value="11964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g002.TIF" value="619242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g002.PNG_I" value="41295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g003.PNG_L" value="310968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g003.PNG_M" value="126567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g003.PNG_S" value="13634"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g003.TIF" value="2106766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.g003.PNG_I" value="44542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.s001.TIF" value="779241"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.s002.TIF" value="204376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.s003.DOC" value="35328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.s004.DOC" value="19968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.s005.DOC" value="19968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001662.s006.DOC" value="20992"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001662&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001662" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001662&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001662&volume=&issue=&title=High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array&author_name=Sandra%20L.%20Spurgeon%2C%20Robert%20C.%20Jones%2C%20Ramesh%20Ramakrishnan&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001662" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662&amp;t=High%20Throughput%20Gene%20Expression%20Measurement%20with%20Real%20Time%20PCR%20in%20a%20Microfluidic%20Dynamic%20Array" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662&title=High%20Throughput%20Gene%20Expression%20Measurement%20with%20Real%20Time%20PCR%20in%20a%20Microfluidic%20Dynamic%20Array&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662&amp;title=High%20Throughput%20Gene%20Expression%20Measurement%20with%20Real%20Time%20PCR%20in%20a%20Microfluidic%20Dynamic%20Array" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001662&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'High Throughput Gene Expression Measurement with Real Time PCR in a Microfluidic Dynamic Array';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001662';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20High%20Throughput%20Gene%20Expression%20Measurement%20with%20Real%20Time%20PCR%20in%20a%20Microfluidic%20Dynamic%20Array http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001662" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001662" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cardiac+muscles%22" title="Search for articles in the subject area:'Cardiac muscles'"><div class="flagText">Cardiac muscles</div></a>
              <div data-categoryid="48893" data-articleid="26150"
                   data-categoryname="Cardiac muscles"
                   class="flagImage" title="Flag 'Cardiac muscles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40169" data-articleid="26150"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="26150"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="26150"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26150"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26150"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="26150"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Skeletal+muscles%22" title="Search for articles in the subject area:'Skeletal muscles'"><div class="flagText">Skeletal muscles</div></a>
              <div data-categoryid="47715" data-articleid="26150"
                   data-categoryname="Skeletal muscles"
                   class="flagImage" title="Flag 'Skeletal muscles' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5298'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7217'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2416&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7665">Charles Auffray's Evaluation at F1000 Biology</a><br>Posted by F1000Medicine</p>
      <p><a href="/annotation/listThread.action?root=18967">Corrected reference</a><br>Posted by RameshR</p>
      <p><a href="/annotation/listThread.action?root=339">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
