

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Biofilm Induced Tolerance towards Antimicrobial Peptides</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001891"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001891" />

    <meta name="citation_title" content="Biofilm Induced Tolerance towards Antimicrobial Peptides"/>
    <meta itemprop="name" content="Biofilm Induced Tolerance towards Antimicrobial Peptides"/>

      <meta name="citation_author" content="Anders Folkesson"/>
            <meta name="citation_author_institution" content="Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark"/>
      <meta name="citation_author" content="Janus A. J. Haagensen"/>
            <meta name="citation_author_institution" content="Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark"/>
      <meta name="citation_author" content="Claudia Zampaloni"/>
            <meta name="citation_author_institution" content="Department of Molecular Cellular Animal Biology, University of Camerino, Camerino, Italy"/>
      <meta name="citation_author" content="Claus Sternberg"/>
            <meta name="citation_author_institution" content="Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark"/>
      <meta name="citation_author" content="Søren Molin"/>
            <meta name="citation_author_institution" content="Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark"/>

    <meta name="citation_date" content="2008/4/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001891.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1891"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Natural conjugative plasmids induce bacterial biofilm development.; citation_author=JM Ghigo; citation_journal_title=Nature; citation_volume=412; citation_number=1; citation_pages=442-445; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Establishment of new genetic traits in a microbial biofilm community.; citation_author=BB Christensen; citation_author=C Sternberg; citation_author=JB Andersen; citation_author=L Eberl; citation_author=S Moller; citation_journal_title=Appl Environ Microbiol; citation_volume=64; citation_number=2; citation_pages=2247-2255; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Gene transfer occurs with enhanced efficiency in biofilms and induces enhanced stabilisation of the biofilm structure.; citation_author=S Molin; citation_author=T Tolker-Nielsen; citation_journal_title=Curr Opin Biotechnol; citation_volume=14; citation_number=3; citation_pages=255-261; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=An IncI1 plasmid contributes to the adherence of the atypical enteroaggregative Escherichia coli strain C1096 to cultured cells and abiotic surfaces.; citation_author=EG Dudley; citation_author=C Abe; citation_author=JM Ghigo; citation_author=P Latour-Lambert; citation_author=JC Hormazabal; citation_journal_title=Infect Immun; citation_volume=74; citation_number=4; citation_pages=2102-2114; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Synergistic effects in mixed Escherichia coli biofilms: conjugative plasmid transfer drives biofilm expansion.; citation_author=A Reisner; citation_author=BM Holler; citation_author=S Molin; citation_author=EL Zechner; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=5; citation_pages=3582-3588; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Development and maturation of Escherichia coli K-12 biofilms.; citation_author=A Reisner; citation_author=JA Haagensen; citation_author=MA Schembri; citation_author=EL Zechner; citation_author=S Molin; citation_journal_title=Mol Microbiol; citation_volume=48; citation_number=6; citation_pages=933-946; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Antimicrobial resistance of Pseudomonas aeruginosa biofilms.; citation_author=E Drenkard; citation_journal_title=Microbes Infect; citation_volume=5; citation_number=7; citation_pages=1213-1219; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Heterogeneity in biofilms.; citation_author=J Wimpenny; citation_author=W Manz; citation_author=U Szewzyk; citation_journal_title=FEMS Microbiol Rev; citation_volume=24; citation_number=8; citation_pages=661-671; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Spatial physiological heterogeneity in Pseudomonas aeruginosa biofilm is determined by oxygen availability.; citation_author=KD Xu; citation_author=PS Stewart; citation_author=F Xia; citation_author=CT Huang; citation_author=GA McFeters; citation_journal_title=Appl Environ Microbiol; citation_volume=64; citation_number=9; citation_pages=4035-4039; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Biofilm resistance to antimicrobial agents.; citation_author=KD Xu; citation_author=GA McFeters; citation_author=PS Stewart; citation_journal_title=Microbiology; citation_volume=146; citation_number=10; citation_pages=547-549; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Microbial biofilms.; citation_author=JW Costerton; citation_author=Z Lewandowski; citation_author=DE Caldwell; citation_author=DR Korber; citation_author=HM Lappin-Scott; citation_journal_title=Annu Rev Microbiol; citation_volume=49; citation_number=11; citation_pages=711-745; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Automated confocal laser scanning microscopy and semiautomated image processing for analysis of biofilms.; citation_author=M Kuehn; citation_author=M Hausner; citation_author=HJ Bungartz; citation_author=M Wagner; citation_author=PA Wilderer; citation_journal_title=Appl Environ Microbiol; citation_volume=64; citation_number=12; citation_pages=4115-4127; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=LPS, TLR4 and infectious disease diversity.; citation_author=SI Miller; citation_author=RK Ernst; citation_author=MW Bader; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=13; citation_pages=36-46; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A ten year review of colomycin.; citation_author=JM Littlewood; citation_author=C Koch; citation_author=PA Lambert; citation_author=N Hoiby; citation_author=JS Elborn; citation_journal_title=Respir Med; citation_volume=94; citation_number=14; citation_pages=632-640; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Antimicrobial susceptibility of pathogenic Escherichia coli isolated from sick cattle and pigs in Japan.; citation_author=K Harada; citation_author=T Asai; citation_author=A Kojima; citation_author=C Oda; citation_author=K Ishihara; citation_journal_title=J Vet Med Sci; citation_volume=67; citation_number=15; citation_pages=999-1003; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluation of colistin as an agent against multi-resistant Gram-negative bacteria.; citation_author=J Li; citation_author=RL Nation; citation_author=RW Milne; citation_author=JD Turnidge; citation_author=K Coulthard; citation_journal_title=Int J Antimicrob Agents; citation_volume=25; citation_number=16; citation_pages=11-25; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Fatty acid alterations and polymyxin B binding by lipopolysaccharides from Pseudomonas aeruginosa adapted to polymyxin B resistance.; citation_author=RS Conrad; citation_author=C Galanos; citation_journal_title=Antimicrob Agents Chemother; citation_volume=33; citation_number=17; citation_pages=1724-1728; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Characterization of the lipopolysaccharide from the polymyxin-resistant pmrA mutants of Salmonella typhimurium.; citation_author=M Vaara; citation_author=T Vaara; citation_author=M Jensen; citation_author=I Helander; citation_author=M Nurminen; citation_journal_title=FEBS Lett; citation_volume=129; citation_number=18; citation_pages=145-149; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Nalidixic acid resistance: a second genetic character involved in DNA gyrase activity.; citation_author=M Gellert; citation_author=K Mizuuchi; citation_author=MH O'Dea; citation_author=T Itoh; citation_author=JI Tomizawa; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=74; citation_number=19; citation_pages=4772-4776; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=DNA gyrase, topoisomerase IV, and the 4-quinolones.; citation_author=K Drlica; citation_author=X Zhao; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=61; citation_number=20; citation_pages=377-392; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Mode of action of fluoroquinolones.; citation_author=DC Hooper; citation_journal_title=Drugs; citation_volume=58; citation_number=21; citation_pages=6-10; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Synthesis of F pilin.; citation_author=K Maneewannakul; citation_author=S Maneewannakul; citation_author=K Ippen-Ihler; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=22; citation_pages=1384-1391; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The Escherichia coli K-12 F plasmid gene traX is required for acetylation of F pilin.; citation_author=D Moore; citation_author=CM Hamilton; citation_author=K Maneewannakul; citation_author=Y Mintz; citation_author=LS Frost; citation_journal_title=J Bacteriol; citation_volume=175; citation_number=23; citation_pages=1375-1383; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Characterization of traX, the F plasmid locus required for acetylation of F-pilin subunits.; citation_author=K Maneewannakul; citation_author=S Maneewannakul; citation_author=K Ippen-Ihler; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=24; citation_pages=2957-2964; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DNA transfer occurs during a cell surface contact stage of F sex factor-mediated bacterial conjugation.; citation_author=MM Panicker; citation_author=EG Minkley; citation_journal_title=J Bacteriol; citation_volume=162; citation_number=25; citation_pages=584-590; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Cointegrate formation mediated by Tn9. II. Activity of IS1 is modulated by external DNA sequences.; citation_author=M Chandler; citation_author=DJ Galas; citation_journal_title=J Mol Biol; citation_volume=170; citation_number=26; citation_pages=61-91; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Polymyxin and related peptide antibiotics.; citation_author=DR Storm; citation_author=KS Rosenthal; citation_author=PE Swanson; citation_journal_title=Annu Rev Biochem; citation_volume=46; citation_number=27; citation_pages=723-763; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of antimicrobial peptide action and resistance.; citation_author=MR Yeaman; citation_author=NY Yount; citation_journal_title=Pharmacol Rev; citation_volume=55; citation_number=28; citation_pages=27-55; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.; citation_author=KA Datsenko; citation_author=BL Wanner; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=29; citation_pages=6640-6645; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Statistical analysis of Pseudomonas aeruginosa biofilm development: impact of mutations in genes involved in twitching motility, cell-to-cell signaling, and stationary-phase sigma factor expression.; citation_author=A Heydorn; citation_author=B Ersboll; citation_author=J Kato; citation_author=M Hentzer; citation_author=MR Parsek; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=30; citation_pages=2008-2017; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Distribution of bacterial growth activity in flow-chamber biofilms.; citation_author=C Sternberg; citation_author=BB Christensen; citation_author=T Johansen; citation_author=A Toftgaard Nielsen; citation_author=JB Andersen; citation_journal_title=Appl Environ Microbiol; citation_volume=65; citation_number=31; citation_pages=4108-4117; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Accumulation of a polyisoprene-linked amino sugar in polymyxin-resistant Salmonella typhimurium and Escherichia coli: structural characterization and transfer to lipid A in the periplasm.; citation_author=MS Trent; citation_author=AA Ribeiro; citation_author=WT Doerrler; citation_author=S Lin; citation_author=RJ Cotter; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=32; citation_pages=43132-43144; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A Genome-Wide View of the Escherichia coli BasS-BasR Two-component System Implicated in Iron-responses.; citation_author=D Hagiwara; citation_author=T Yamashino; citation_author=T Mizuno; citation_journal_title=Biosci Biotechnol Biochem; citation_volume=68; citation_number=33; citation_pages=1758-1767; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phenotypic differences between Salmonella and Escherichia coli resulting from the disparate regulation of homologous genes.; citation_author=MD Winfield; citation_author=EA Groisman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=34; citation_pages=17162-17167; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tobramycin resistance of Pseudomonas aeruginosa cells growing as a biofilm on urinary catheter material.; citation_author=JC Nickel; citation_author=I Ruseska; citation_author=JB Wright; citation_author=JW Costerton; citation_journal_title=Antimicrob Agents Chemother; citation_volume=27; citation_number=35; citation_pages=619-624; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The Calgary Biofilm Device: new technology for rapid determination of antibiotic susceptibilities of bacterial biofilms.; citation_author=H Ceri; citation_author=ME Olson; citation_author=C Stremick; citation_author=RR Read; citation_author=D Morck; citation_journal_title=J Clin Microbiol; citation_volume=37; citation_number=36; citation_pages=1771-1776; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Riddle of biofilm resistance.; citation_author=K Lewis; citation_journal_title=Antimicrob Agents Chemother; citation_volume=45; citation_number=37; citation_pages=999-1007; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Biofilm susceptibility to antimicrobials.; citation_author=P Gilbert; citation_author=J Das; citation_author=I Foley; citation_journal_title=Adv Dent Res; citation_volume=11; citation_number=38; citation_pages=160-167; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A genetic basis for Pseudomonas aeruginosa biofilm antibiotic resistance.; citation_author=TF Mah; citation_author=B Pitts; citation_author=B Pellock; citation_author=GC Walker; citation_author=PS Stewart; citation_journal_title=Nature; citation_volume=426; citation_number=39; citation_pages=306-310; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pseudomonas biofilm formation and antibiotic resistance are linked to phenotypic variation.; citation_author=E Drenkard; citation_author=FM Ausubel; citation_journal_title=Nature; citation_volume=416; citation_number=40; citation_pages=740-743; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Resistance of Streptococcus sanguis biofilms to antimicrobial agents.; citation_author=T Larsen; citation_author=NE Fiehn; citation_journal_title=Apmis; citation_volume=104; citation_number=41; citation_pages=280-284; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Resistance of Pseudomonas pseudomallei growing as a biofilm on silastic discs to ceftazidime and co-trimoxazole.; citation_author=M Vorachit; citation_author=K Lam; citation_author=P Jayanetra; citation_author=JW Costerton; citation_journal_title=Antimicrob Agents Chemother; citation_volume=37; citation_number=42; citation_pages=2000-2002; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The effects of adherence to silicone surfaces on antibiotic susceptibility in Staphylococcus aureus.; citation_author=I Williams; citation_author=WA Venables; citation_author=D Lloyd; citation_author=F Paul; citation_author=I Critchley; citation_journal_title=Microbiology; citation_volume=143; citation_number=43; citation_pages=2407-2413; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Antibiotic susceptibility assay for Staphylococcus aureus in biofilms developed in vitro.; citation_author=B Amorena; citation_author=E Gracia; citation_author=M Monzon; citation_author=J Leiva; citation_author=C Oteiza; citation_journal_title=J Antimicrob Chemother; citation_volume=44; citation_number=44; citation_pages=43-55; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Dynamic interactions of biofilms of mucoid Pseudomonas aeruginosa with tobramycin and piperacillin.; citation_author=H Anwar; citation_author=JL Strap; citation_author=K Chen; citation_author=JW Costerton; citation_journal_title=Antimicrob Agents Chemother; citation_volume=36; citation_number=45; citation_pages=1208-1214; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Increasing resistance of planktonic and biofilm cultures of Burkholderia cepacia to ciprofloxacin and ceftazidime during exponential growth.; citation_author=M Desai; citation_author=T Buhler; citation_author=PH Weller; citation_author=MR Brown; citation_journal_title=J Antimicrob Chemother; citation_volume=42; citation_number=46; citation_pages=153-160; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comparative in vitro antibiotic resistance of surface-colonizing coagulase-negative staphylococci.; citation_author=AG Gristina; citation_author=RA Jennings; citation_author=PT Naylor; citation_author=QN Myrvik; citation_author=LX Webb; citation_journal_title=Antimicrob Agents Chemother; citation_volume=33; citation_number=47; citation_pages=813-816; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Antibiotic susceptabilities of Pseudomonas aeruginosa isolates derived from patients with cystic fibrosis under aerobic, anaerobic, and biofilm conditions.; citation_author=D Hill; citation_author=B Rose; citation_author=A Pajkos; citation_author=M Robinson; citation_author=P Bye; citation_journal_title=J Clin Microbiol; citation_volume=43; citation_number=48; citation_pages=5085-5090; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Combination effect of fosfomycin and ofloxacin against Pseudomonas aeruginosa growing in a biofilm.; citation_author=H Kumon; citation_author=N Ono; citation_author=M Iida; citation_author=JC Nickel; citation_journal_title=Antimicrob Agents Chemother; citation_volume=39; citation_number=49; citation_pages=1038-1044; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Biofilms and planktonic cells of Pseudomonas aeruginosa have similar resistance to killing by antimicrobials.; citation_author=AL Spoering; citation_author=K Lewis; citation_journal_title=J Bacteriol; citation_volume=183; citation_number=50; citation_pages=6746-6751; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Lipid and lipopolysaccharide composition of Escherichia coli surface-altered mutants selected for resistance to levallorphan, tetracaine, and polymyxin.; citation_author=JB Dame; citation_author=BM Shapiro; citation_journal_title=J Bacteriol; citation_volume=137; citation_number=51; citation_pages=1043-1047; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Differentiation and distribution of colistin- and sodium dodecyl sulfate-tolerant cells in Pseudomonas aeruginosa biofilms.; citation_author=JA Haagensen; citation_author=M Klausen; citation_author=RK Ernst; citation_author=SI Miller; citation_author=A Folkesson; citation_journal_title=J Bacteriol; citation_volume=189; citation_number=52; citation_pages=28-37; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Current Protocols in Molecular Biology.; citation_author=FM Ausubel; citation_author=MR Brent; citation_author=RE Kingston; citation_author=DD Moore; citation_author=JG Siedman; citation_number=53; citation_date=1996; citation_publisher=John Wiley; " />
      <meta name="citation_reference" content="citation_title=Studies on lysogenesis. I. The mode of phage liberation by lysogenic Escherichia coli.; citation_author=G Bertani; citation_journal_title=J Bacteriol; citation_volume=62; citation_number=54; citation_pages=293-300; citation_date=1951; " />
      <meta name="citation_reference" content="citation_title=Swarming populations of Salmonella represent a unique physiological state coupled to multiple mechanisms of antibiotic resistance.; citation_author=W Kim; citation_author=MG Surette; citation_journal_title=Biol Proced Online; citation_volume=5; citation_number=55; citation_pages=189-196; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Swarm-cell differentiation in Salmonella enterica serovar typhimurium results in elevated resistance to multiple antibiotics.; citation_author=W Kim; citation_author=T Killam; citation_author=V Sood; citation_author=MG Surette; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=56; citation_pages=3111-3117; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A short course in bacterial genetics: a laboratory manual and handbook for Escherichia coli and related bacteria.; citation_author=JH Miller; citation_number=57; citation_date=1992; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of pathogenic bacteria. A laboratory manual.; citation_author=SR Maloy; citation_author=VJ Stewart; citation_author=RK Taylor; citation_number=58; citation_date=1996; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=New cloning vectors for integration in the lambda attachment site attB of the Escherichia coli chromosome.; citation_author=L Diederich; citation_author=LJ Rasmussen; citation_author=W Messer; citation_journal_title=Plasmid; citation_volume=28; citation_number=59; citation_pages=14-24; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=In vitro biofilm formation of commensal and pathogenic Escherichia coli strains: impact of environmental and genetic factors.; citation_author=A Reisner; citation_author=KA Krogfelt; citation_author=BM Klein; citation_author=EL Zechner; citation_author=S Molin; citation_journal_title=J Bacteriol; citation_volume=188; citation_number=60; citation_pages=3572-3581; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA replication and the division cycle in Escherichia coli.; citation_author=DJ Clark; citation_author=Z Maaløe; citation_journal_title=J Mol Biol; citation_volume=23; citation_number=61; citation_pages=99-112; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Quantification of biofilm structures by the novel computer program COMSTAT.; citation_author=A Heydorn; citation_author=AT Nielsen; citation_author=M Hentzer; citation_author=C Sternberg; citation_author=M Givskov; citation_journal_title=Microbiology; citation_volume=146; citation_number=62; citation_pages=2395-2407; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Isolation of an Escherichia coli K-12 mutant strain able to form biofilms on inert surfaces: involvement of a new ompR allele that increases curli expression.; citation_author=O Vidal; citation_author=R Longin; citation_author=C Prigent-Combaret; citation_author=C Dorel; citation_author=M Hooreman; citation_journal_title=J Bacteriol; citation_volume=180; citation_number=63; citation_pages=2442-2449; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Construction of derivatives of the F plasmid pOX-tra715: characterization of traY and traD mutants that can be complemented in trans.; citation_author=K Maneewannakul; citation_author=P Kathir; citation_author=S Endley; citation_author=D Moore; citation_author=J Manchak; citation_journal_title=Mol Microbiol; citation_volume=22; citation_number=64; citation_pages=197-205; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Construction and characterization of derivatives carrying insertion mutations in F plasmid transfer region genes, trbA, artA, traQ, and trbB.; citation_author=P Kathir; citation_author=K Ippen-Ihler; citation_journal_title=Plasmid; citation_volume=26; citation_number=65; citation_pages=40-54; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=TraG-like proteins of DNA transfer systems and of the Helicobacter pylori type IV secretion system: inner membrane gate for exported substrates?; citation_author=G Schroder; citation_author=S Krause; citation_author=EL Zechner; citation_author=B Traxler; citation_author=HJ Yeo; citation_journal_title=J Bacteriol; citation_volume=184; citation_number=66; citation_pages=2767-2779; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria.; citation_author=JB Andersen; citation_author=C Sternberg; citation_author=LK Poulsen; citation_author=SP Bjorn; citation_author=M Givskov; citation_journal_title=Appl Environ Microbiol; citation_volume=64; citation_number=67; citation_pages=2240-2246; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Biofilm Induced Tolerance towards Antimicrobial Peptides"/>
    <meta name="twitter:description" content="Increased tolerance to antimicrobial agents is thought to be an important feature of microbes growing in biofilms. We address the question of how biofilm organization affects antibiotic susceptibility. We established Escherichia coli biofilms with differential structural organization due to the presence of IncF plasmids expressing altered forms of the transfer pili in two different biofilm model systems. The mature biofilms were subsequently treated with two antibiotics with different molecular targets, the peptide antibiotic colistin and the fluoroquinolone ciprofloxacin. The dynamics of microbial killing were monitored by viable count determination, and confocal laser microscopy. Strains forming structurally organized biofilms show an increased bacterial survival when challenged with colistin, compared to strains forming unstructured biofilms. The increased survival is due to genetically regulated tolerant subpopulation formation and not caused by a general biofilm property. No significant difference in survival was detected when the strains were challenged with ciprofloxacin. Our data show that biofilm formation confers increased colistin tolerance to cells within the biofilm structure, but the protection is conditional being dependent on the structural organization of the biofilm, and the induction of specific tolerance mechanisms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001891.g005"/>

  <meta property="og:title" content="Biofilm Induced Tolerance towards Antimicrobial Peptides" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8837'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4845'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2806&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001891">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001891" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001891&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Biofilm Induced Tolerance towards Antimicrobial Peptides
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anders Folkesson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:anf@bio.dtu.dk">anf@bio.dtu.dk</a></p>

                <p>Affiliation:
                  Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Janus A. J. Haagensen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claudia Zampaloni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cellular Animal Biology, University of Camerino, Camerino, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claus Sternberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Søren Molin
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Infection Microbiology Group, BioCentrum-DTU, Technical University of Denmark, Lyngby, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001891</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001891-g001" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001891-g002" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001891-t001" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001891-g003" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001891-g004" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001891-g005" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001891-t002" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001891">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001891" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Increased tolerance to antimicrobial agents is thought to be an important feature of microbes growing in biofilms. We address the question of how biofilm organization affects antibiotic susceptibility. We established <em>Escherichia coli</em> biofilms with differential structural organization due to the presence of IncF plasmids expressing altered forms of the transfer pili in two different biofilm model systems. The mature biofilms were subsequently treated with two antibiotics with different molecular targets, the peptide antibiotic colistin and the fluoroquinolone ciprofloxacin. The dynamics of microbial killing were monitored by viable count determination, and confocal laser microscopy. Strains forming structurally organized biofilms show an increased bacterial survival when challenged with colistin, compared to strains forming unstructured biofilms. The increased survival is due to genetically regulated tolerant subpopulation formation and not caused by a general biofilm property. No significant difference in survival was detected when the strains were challenged with ciprofloxacin. Our data show that biofilm formation confers increased colistin tolerance to cells within the biofilm structure, but the protection is conditional being dependent on the structural organization of the biofilm, and the induction of specific tolerance mechanisms.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Folkesson A, Haagensen JAJ, Zampaloni C, Sternberg C, Molin S (2008) Biofilm Induced Tolerance towards Antimicrobial Peptides. PLoS ONE 3(4):
          e1891.
            doi:10.1371/journal.pone.0001891</p><p><strong>Editor: </strong>Frederick M. Ausubel, Massachusetts General Hospital, United States of America</p><p><strong>Received:</strong> November 19, 2007; <strong>Accepted:</strong> February 23, 2008; <strong>Published:</strong> April 2, 2008</p><p><strong>Copyright:</strong> © 2008 Folkesson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Danish research councils to Søren Molin and a FEMS Research Fellowship to Claudia Zampaloni.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Natural conjugative plasmids express factors that can stimulate planktonic bacteria to form biofilm communities <a href="#pone.0001891-Ghigo1">[1]</a>. Biofilm formation also favors the infectious transfer of plasmids <a href="#pone.0001891-Christensen1">[2]</a>, <a href="#pone.0001891-Molin1">[3]</a>, and many medically relevant microbial pathogens are often associated with a variety of conjugative plasmids indicating an increased tendency for biofilm formation <a href="#pone.0001891-Ghigo1">[1]</a>, <a href="#pone.0001891-Dudley1">[4]</a>–<a href="#pone.0001891-Reisner2">[6]</a>. Microbial biofilms are notoriously hard to treat with antimicrobial compounds. The molecular nature of this apparent resistance is not well elucidated and a number of mechanisms have been proposed to explain the reduced susceptibility <a href="#pone.0001891-Drenkard1">[7]</a>. The architecture of mature biofilms varies considerably from flat homogenous layers of cells to highly organized cell clusters characterized by mushroom shaped structures interspersed by water-filled channels <a href="#pone.0001891-Wimpenny1">[8]</a>. The organization of the biofilm affects the distribution of chemical and physiological gradients within the microbial assembly, which in turn affects the metabolic state of the bacteria, resulting in a heterogeneous congregation of cells in different physiological states <a href="#pone.0001891-Wimpenny1">[8]</a>–<a href="#pone.0001891-Xu2">[10]</a>. The spatial and physiological heterogeneity of the biofilm architecture has the potential to affect the antimicrobial tolerance profiles of the organisms in the biofilm by forming microhabitats with diverse properties <a href="#pone.0001891-Costerton1">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p><em>Escherichia coli</em> (<em>E.coli</em>) K12 strains usually form unstructured flat biofilms in comparison with the highly organized and substantial biofilms formed by bacteria such as <em>Pseudomonas aeruginosa</em> (<em>P.aeruginosa</em>) <a href="#pone.0001891-Reisner2">[6]</a>, <a href="#pone.0001891-Kuehn1">[12]</a>. Introduction of transfer constitutive IncF plasmids dramatically changes the organization of <em>E.coli</em> biofilms to a structure resembling those reported for <em>P.aeruginosa</em> <a href="#pone.0001891-Ghigo1">[1]</a>, <a href="#pone.0001891-Reisner2">[6]</a>. Biofilm organization is determined by the configuration of the transfer pili since various mutants affected in the processing and activity of the pili exhibit differentially structured biofilms <a href="#pone.0001891-Reisner2">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The ability to survive exposure to cationic antimicrobial peptides is an essential virulence property for many microbial pathogens <a href="#pone.0001891-Miller1">[13]</a>. Antimicrobial peptides are an integral part of the host defense system and believed to kill microorganisms by affecting their membrane integrity. Colistin is an amphipathic polypetide antibiotic utilized in the treatment of multi-resistant <em>P.aeruginosa</em> causing chronic pulmonary infections among cystic fibrosis patients. The peptide antibiotic is also extensively used both as feed additive and in the treatment of severe colibacillosis caused by pathogenic <em>E.coli</em> in livestock <a href="#pone.0001891-Littlewood1">[14]</a>, <a href="#pone.0001891-Harada1">[15]</a>. Resistance towards antimicrobial peptides such as colistin is seldom identified in the clinical setting <a href="#pone.0001891-Li1">[16]</a>. Gram-negative bacteria have evolved mechanisms that remodel the composition of the outer membrane through modification of the lipopolysaccharide (LPS) molecules <a href="#pone.0001891-Miller1">[13]</a>. The modifications lead to an increased tolerance to antimicrobial peptide exposure, and is controlled by a two-component regulatory system encoded by homologs to the <em>Salmonella typhimurium pmrA/B</em> genes in many bacterial species <a href="#pone.0001891-Conrad1">[17]</a>, <a href="#pone.0001891-Vaara1">[18]</a>. LPS alterations are thought to play a significant role during infection by conferring increased bacterial tolerance to host antimicrobial factors and avoidance of immune system recognition <a href="#pone.0001891-Miller1">[13]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In contrast to colistin, the primary <em>E.coli</em> target for fluoroquinolone antibiotics such as ciprofloxacin is the topoisomerase II enzyme, DNA gyrase, which is essential for DNA synthesis <a href="#pone.0001891-Gellert1">[19]</a>. Resistance to fluoroquinolones in <em>E. coli</em> occurs by chromosomal mutation and requires alterations in either of the genes that encode the topoisomerases (<em>gyrA</em> and <em>gyrB</em>, encoding gyrase, and <em>parC</em> and <em>parE,</em> encoding topoisomerase IV) or in genes that influence cell permeability or drug export <a href="#pone.0001891-Drlica1">[20]</a>, <a href="#pone.0001891-Hooper1">[21]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this report, we challenge <em>E.coli</em> biofilms with differential structural organization with colistin or ciprofloxacin, two clinically relevant antibiotics with different molecular targets and mechanisms of resistance. We show that biofilm formation increases the survival of the microbes after challenge by the peptide antibiotic but not by the fluoroquinolone. The increased survival is dependent on the <em>basR/basS</em> (<em>pmrA</em>/B) two-component system, since disruption of their encoding genes abolishes the observed protective effect while not affecting the organization of the biofilm. We demonstrate that the increased survival is due to development of a tolerant subpopulation within the biofilm and that the formation of the subpopulation is influenced by the spatial organization of the biofilm indicating that intra-biofilm physiochemical gradients promote antibiotic tolerance within spatially structured environments.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4><em>E.coli</em> cells in biofilms differing in organization display differential survival after colistin treatment</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The presence of transfer constitutive IncF plasmids fundamentally influences the architecture and organization of <em>E.coli</em> biofilms <a href="#pone.0001891-Reisner2">[6]</a>. The final spatial arrangement of the biofilm is determined by the configuration of the transfer pili, as various mutants in the processing and activity of the organelle display differentially organized biofilms <a href="#pone.0001891-Reisner2">[6]</a>. F pilus synthesis requires 13 essential genes and 4 auxiliary genes of plasmid origin. TraQ is required for insertion of propilin (TraA) in the inner membrane. Absence of the TraQ chaperone results in rapid degradation of TraA in the cytoplasm <a href="#pone.0001891-Maneewannakul1">[22]</a>. TraX is responsible for acetylation of the N-terminus of mature pilin, but the modification is non-essential for pilus elaboration and transfer proficiency <a href="#pone.0001891-Moore1">[23]</a>, <a href="#pone.0001891-Maneewannakul2">[24]</a>. TraD is not involved in pilus assembly but instead has an essential role in DNA transfer at a stage following cell/cell contact <a href="#pone.0001891-Panicker1">[25]</a>. To study the effects of differential biofilm organization in relation to antibiotic tolerance, we established a static biofilm and survival assay of wildtype, <em>traQ</em>, <em>traX</em>, and <em>traD</em> mutant derivatives of pOX38 <a href="#pone.0001891-Chandler1">[26]</a> in <em>E.coli</em> K12 CSH26 derivative SAR18, which were compared with biofilms formed by plasmid-free SAR18 in standard flat-bottomed 96 well microtiter dishes. The strains were inoculated in Luria broth and incubated statically for 24h. Differential biomass accumulation for SAR18 F<sup>+</sup> and the various mutants compared to SAR18 was confirmed by crystal violet staining of the wells (<a href="#pone-0001891-g001">Fig. 1A</a>). The results were consistent with the previously observed biofilm phenotype in the flow-chamber biofilm system. SAR18F<sup>+</sup>, F<sup>+</sup>(<em>traD</em>) and F<sup>+</sup>(<em>traX</em>) accumulate increased amounts of biomass in microtiter wells compared to SAR18F<sup>+</sup>(<em>traQ</em>) and SAR18 lacking F<sup>+</sup> (<a href="#pone-0001891-g001">Fig. 1A</a>) <a href="#pone.0001891-Reisner2">[6]</a>. For the colistin survival assay, the established biofilms were challenged with 10 µg/ml of the cationic antimicrobial peptide colistin instead of washing and crystal violet staining <a href="#pone.0001891-Storm1">[27]</a>. The killing activity and mode of action of antimicrobial peptides is highly influenced by experimental conditions and bacterial cell number <a href="#pone.0001891-Yeaman1">[28]</a>.To keep the experimental conditions as comparable as possible the antibiotic was added directly to the wells without prior washing and removal of putative planktonic cells. After an additional 24h incubation with antibiotic, the total surviving CFU were determined by serial dilution and plating on culture media by dislodging the biofilms from the microtiter wells by vigorous aspiration. We observed a three fold difference in survival between SAR18 and SAR18 F<sup>+</sup>, where the surviving CFU were significantly elevated in SAR18 F<sup>+</sup> biofilms (<a href="#pone-0001891-g001">Fig. 1B</a>) (P&lt;0.0044). Survival was even more enhanced in the biofilm produced by the <em>traD</em> mutant where the difference was five fold compared to SAR18 (<a href="#pone-0001891-g001">Fig. 1B</a>). Survival did not correlate with the amount of biofilm biomass formed because the <em>traX</em> mutant, that forms biofilm in a quantity approximately equal to the <em>traD</em> mutant, did not show any enhanced survival (<a href="#pone-0001891-g001">Fig. 1B</a>). The presence of the F derivatives did not affect colony forming unit (CFU) formation in the wells since no difference in CFU counts could be detected in untreated control wells (<a href="#pone-0001891-g001">Fig. 1B and C</a>) (P&gt;0.05). Addition of the fluoroquinolone ciprofloxacin (0.3 µg/ml), an antibiotic with an intracellular target, showed no significant difference in killing of the different strains (<a href="#pone-0001891-g001">Fig. 1C</a>) (P&gt;0.05). The MIC for colistin and ciprofloxacin were 1 µg/ml and 0.03 µg/ml, respectively, for all of the strains in the study.</p>
<div class="figure" id="pone-0001891-g001"><div class="img"><a name="pone-0001891-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span><em> E.coli</em> cells in biofilms differing in organization display differential survival after colistin treatment.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A). Microtitre plate biofilm assay with the parental <em>E. coli</em> SAR18 in the presence and absence of the de-repressed IncF plasmid and mutants in the IncF plasmid (<em>traD, traX,</em> and <em>traQ</em>,) stained with crystal violet. Values are normalized with respect to the mean of SAR18 F<sup>+</sup> (<em>traD</em>) set as 100% B). A microtitre plate biofilm antibiotic tolerance assay, in which the indicated strains were challenged with 10 µg/ml colistin for 24h and bacterial survival determined by viable count as indicated by white bars. Untreated control is indicted by black bars. C). Microtitre plate biofilm antibiotic tolerance assay, in which the indicated strains were challenged with 0.3 µg/ml ciprofloxacin for 24h and bacterial survival determined by viable count as indicted by white bars. Untreated control is indicated by black bars. D). The survival of planktonically cultured stationary phase <em>E. coli</em> SAR18 in the presence and absence of the de-repressed IncF plasmid and mutants in the IncF plasmid (<em>traD, traX,</em> and <em>traQ</em>,) after a 24 h 10 µg/ml colistin or 0.3 µg/ml ciprofloxacin challenge. Black bars indicate CFU in untreated culture while white bars indicate treated cells as specified in the figure.</p>
<span>doi:10.1371/journal.pone.0001891.g001</span></div>

<h4>Mutations in the transfer operons do not confer protection in plantonic cultures</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Mutations in the transfer operon may have important pleiotropic consequences independent of the biofilm context, we therefore investigated the survival after colistin and ciprofloxacin treatment of planktonic cultures to the of wildtype, <em>traQ</em>, <em>traX</em>, and <em>traD</em> mutant derivatives of pOX38 and plasmid-free SAR18 biofilm cultures. The strains were inoculated in Luria broth using standard polystyrene culture tubes and incubated under shaking (rpm 220) conditions for 24 h. The cultures were then challenged with 10 µg/ml colistin for an additional 24h and surviving colony-forming units (CFU) were determined by serial dilution and plating on culture media. For all strains approximately 3×10<sup>7</sup> CFU had survived the colistin challenged, while the untreated control cultures had reached a CFU of 1×10<sup>9</sup>. There was no significant (P&gt;0.05) difference in survival between wildtype, <em>traQ</em>, <em>traX</em>, and <em>traD</em> mutant derivatives of pOX38 and plasmid-free SAR18 biofilm cultures (<a href="#pone-0001891-g001">Fig. 1D</a>). If the shaken over night cultures instead were challenge with 0.3 µg/ml ciprofloxacin for 24 h, 2×10<sup>6</sup> CFU were detected corresponding to a 3 log difference of survival between the different growth model systems (<a href="#pone-0001891-g001">Fig. 1D</a>). These data indicate that the presence of the F plasmid and its mutant derivatives does not affect antibiotic tolerance during planktonic growth.</p>


<h4>The increased survival of cells in colistin treated biofilms is dependent on a functional <em>basS</em> gene</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The <em>E.coli</em> K12 strain MG1655 is better characterized genetically than the SAR18 strain. Moreover, SAR18 harbors a number of antibiotic resistance markers that make genetic manipulation difficult (<a href="#pone-0001891-t001">Table 1</a>). We included the strain AF504, a nalidixic acid resistant derivative of MG1655, and a curli over-producing <em>ompR234</em> mutant of MG1655 in the investigation. Curli overproduction results in increased biofilm formation in the biofilm flow chamber system in an F-plasmid independent manner <a href="#pone.0001891-Reisner2">[6]</a>. To investigate if the observed increase in colistin tolerance of biofilm cells is genetically determined or due to a generic difference in biofilm permeability, we introduced insertion deletions into the <em>basS</em> gene of AF504, using one-step allelic replacement <a href="#pone.0001891-Datsenko1">[29]</a>. We established biofilms of these mutant strains and compared the survival of the bacteria with or without the F-plasmid and its mutant <em>traD</em> variant after colistin treatment. As expected, introduction of the F-plasmid or its <em>traD</em> mutant derivative into AF504 drastically increased biofilm formation as well as colistin tolerance of the resulting strains (<a href="#pone-0001891-g002">Fig. 2A and B</a>). The protective effect of biofilm formation with F<sup>+</sup> and F<sup>+</sup>(<em>traD</em>) was completely abolished in the <em>basS</em> mutants, and the survival reached a level comparable to the survival of plasmid free AF504 (<a href="#pone-0001891-g002">Fig. 2B</a>). If the <em>basR</em> and <em>basS</em> genes were reintroduced on a multicopy plasmid into the <em>basS traD</em> mutant biofilm protection was restored (<a href="#pone-0001891-g002">Fig. 2B</a>), while if the same plasmid was introduced into AF504 or AF504 F<sup>+</sup>(<em>traX</em>) no effect on survival could be detected (<a href="#pone-0001891-g002">Fig. 2B</a>). If the vector alone was introduced to the strains, no effect on biofilm formation or survival could be detected (data not shown). The nalidixic acid resistance of AF504 does not affect the outcome of the biofilm survival assay since there is no detectable difference in survival between MG1655 and AF504 after colistin challenge (data not shown). These results indicate that a specific genetic mechanism is underlying the observed increased tolerance in biofilm forming <em>E.coli</em>. Over-expression of the curli fimbriae drastically influences the biofilm development in <em>E.coli</em> and such strains form biofilms comparable to the ones formed by F<sup>+</sup>(<em>traD</em>) and F<sup>+</sup>(<em>traX</em>) mutants (<a href="#pone-0001891-g002">Fig. 2A</a> <a href="#pone.0001891-Reisner2">[6]</a>, <a href="#pone.0001891-Heydorn1">[30]</a>. No increase in survival after colistin treatment compared to the plasmid free AF504 was detected, demonstrating that biofilm formation and survival after antibiotic treatment are not directly correlated (<a href="#pone-0001891-g002">Fig. 2B</a>). If the same <em>basS</em> mutation was introduced into Sar18 F<sup>+</sup>, pili mediated protection was abolished, showing that <em>basS</em> dependence is not exclusive for AF504 (<a href="#pone-0001891-g001">Fig. 1B</a>). Increased concentrations of colistin decreased the number of viable cells within the microtiter wells in a concentration dependent manner. Colistin concentrations above 60 µg/ml reduced the number of viable cells to below the detection level of 100 CFU/ml (Data not shown). The MIC for colistin for all of the AF504 derived strains were 1 µg/ml in microtiter assay.</p>
<div class="figure" id="pone-0001891-g002"><div class="img"><a name="pone-0001891-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The increased survival of cells in biofilms is dependent on a functional <em>basS</em> gene.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A). Microtitre plate biofilm assay with the parental <em>E. coli</em> AF504 in the presence and absence of the de-repressed IncF plasmid and mutants in the IncF plasmid (<em>traD,</em>), insertion deletions of the <em>basS</em> and <em>basR</em> genes, and the curli overproducing <em>opmR</em> strain stained with crystal violet. Values are normalized with respect to the mean of AF504 F<sup>+</sup> (<em>traD</em>) set as 100%. B). A microtitre plate biofilm antibiotic tolerance assay in which the indicated strains were challenged with 10 µg/ml colistin for 24 h and bacterial survival determined. The fraction untreated/treated is indicated for each strain.</p>
<span>doi:10.1371/journal.pone.0001891.g002</span></div><div class="figure" id="pone-0001891-t001"><div class="img"><a name="pone-0001891-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Bacterial strains used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001891.t001</span></div>

<h4><em>E.coli</em> flow-chamber biofilms are sensitive to the cationic antimicrobial peptide colistin</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To be able to study biofilm formation <em>in vitro</em> in the hydrodynamic continuous-flow biofilm model system <a href="#pone.0001891-Sternberg1">[31]</a> we tagged AF504 F<sup>+</sup> with the green fluorescent protein (GFP) under the control of a ribosomal promoter forming the strain AF504 <em>gfp</em> F<sup>+</sup>. The flow chambers were seeded with the strain and incubated for 48h at 30°C in glucose minimal medium. The mature biofilms were then exposed to 25 µg/ml colistin for 0h (untreated control), 6.5h and 24h and stained with propidium iodide (PI) to distinguish living GFP expressing cells and dead PI stained cells (<a href="#pone-0001891-g003">Fig. 3 A, B and C</a>). The untreated control biofilm exhibited a few dead (red) cells spread out through the biofilm (<a href="#pone-0001891-g003">Fig. 3A</a>) while the majority of the cells in the treated biofilm were clearly dead after 24 h (<a href="#pone-0001891-g003">Fig. 3 C</a>). The killing activity of the colistin moved from the edge of the microcolonies to the inner part as shown in time series image capturing (<a href="#pone.0001891.s001">Movie S1</a>). The time series also confirmed that detachment was absent or low under these conditions. To determine the proportion of viable cells in the biofilm channels, the biofilm was dislodged from the flow-channel and the CFU/ml were determined and normalized by the OD450 value obtained for each channel. The number of viable cells recovered from the channel decreased from 10<sup>9</sup> to 10<sup>3</sup> CFU/OD<sub>450</sub> in 24 hours of treatment and reached 10<sup>2</sup> CFU/OD<sub>450</sub> after 48 hours of drug-exposure. The number of viable cells in the control channel remained constant during the time of the experiment (10<sup>9</sup> CFU/OD<sub>450</sub>). If 50 µg/ml colistin was used, an exposure for 24 h was enough for complete eradication of the biofilm and indeed no growth was observable after plating. If plasmid free AF504 <em>gfp</em> cells were inoculated in the flow channel system a few scattered loosely attached cells could be detected through the channel after 24 h and after a 24 h, 25 µg/ml colistin challenge very few bacteria could be detected (data not shown).</p>
<div class="figure" id="pone-0001891-g003"><div class="img"><a name="pone-0001891-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>A colistin tolerant subpopulation is formed in mature <em>E .coli</em> flow-chamber biofilms.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>CLSM micrographs of a mature AF504 <em>gfp</em>/F<sup>+</sup> biofilm after A) control, B) 6.5h; and C) 24h 25 µg/ml of colistin challenge. The biofilm was stained with PI at the specified time points before image acquisition. Scale bars represent 50 µm D) CLSM micrograph showing the location of live (green) and dead (red) cells in <em>E. coli</em> AF504 <em>gfp/</em>F<sup>+</sup> biofilm grown for 48 h and then exposed to 10 µg/ml of colistin for 24 h, an overview image taken with a 10× objective. Scale bar represent 300 µm. E) CLSM 3D micrograph image taken with a 40× objective of a single microcolony in the AF504 <em>gfp/</em>F<sup>+</sup> biofilm. F) CLSM micrograph 3D structure showing the spatial distribution of living (green) and dead (red) cells of <em>E. coli</em> AF504 <em>gfp/</em>F<sup>+</sup> in a 6 days old biofilm, that has been grown in antibiotic free media for 2 days and then exposed to 25 µg/ml colistin for 24 h and then switched back to antibiotic free medium.</p>
<span>doi:10.1371/journal.pone.0001891.g003</span></div>

<h4>A colistin tolerant subpopulation is formed in mature <em>E. coli</em> flow-chamber biofilms</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p><em>E. coli</em> mature biofilm cells are killed efficiently by colistin through gradual penetration into the centre of the microcolonies (<a href="#pone-0001891-g003">Fig 3 A, B and C</a>). However, in large microcolonies we observed a surviving population gradually increasing in biomass during the first 48 h after addition of colistin. This phenomenon was observed using 10 µg/ml and 25 µg/ml colistin (<a href="#pone-0001891-g003">Fig. 3D and E</a>). The tolerant subpopulation only formed in fully mature biofilms of bacteria expressing the transfer pili and was able to re-develop into a new biofilm if colistin free media was supplied to the flow chamber system (<a href="#pone-0001891-g003">Fig. 3F</a>). One day after a shift from 25 µg/ml colistin to antibiotic-free medium the tolerant population was growing and after additional two days, microcolony formation and biofilm re-development was observed (<a href="#pone-0001891-g003">Fig. 3F</a>).</p>


<h4>Colistin tolerance is not due to resistance mutations</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Based on these findings we wanted to investigate if the developing colistin tolerant subpopulations were the results of genetically mutated resistant cells, or if they represented phenotypically tolerant variants. We isolated the biofilm material from 2-day-old flow channel biofilms, grown in glucose minimal media, which had subsequently been exposed to 10 µg/ml or 25 µg/ml colistin, for 24–48 h. The isolated biofilm samples were homogenized and stained with PI after which individual cells were sorted with a FACS cell sorter in red and green subpopulations. The red population was not viable as shown by plating on LB plates whereas the green population was growing on LB plates, but not on plates containing colistin (10 µg/ml or 25 µg/ml). Development of a tolerant subpopulation in the biofilm after treatment with colistin thus seems to be linked to adaptation or activation of a surviving phenotype suggesting that a transiently colistin tolerant subpopulation is formed in the biofilm. When the cells were isolated from the flow chamber and re-grown without colistin they lost the tolerance to the peptide and became sensitive to the antibiotic, showing that the surviving tolerant cells were not genetically resistant mutants selected in the flow chamber.</p>


<h4>Colistin tolerance is correlated to biofilm structure and architecture</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To investigate the effect of biofilm architecture on antibiotic susceptibility we established flow chamber biofilms with <em>E.coli</em> in presence and absence of the de-repressed IncF plasmids (wildtype, <em>traD</em>, <em>traQ</em> and <em>traX</em>). It has been previously shown that the <em>tra</em> mutations lead to formation of biofilms with strikingly different spatial distribution <a href="#pone.0001891-Reisner2">[6]</a>. The different biofilms were treated with colistin and the distribution of living and dead cells monitored. Interestingly, strains with the de-repressed IncF WT plasmid and the <em>traD</em> mutation forming biofilms with elaborate, rounded tower structures rising from the confluent cell layer, displayed regions of tolerant cells in the center of the biofilm micro-colonies after 24h challenge with 10 µg/ml colistin. Moreover, this subpopulation formation seemed to be more extensive in the <em>traD</em> mutant biofilm compared to the wild type F<sup>+</sup> (<a href="#pone-0001891-g004">Fig. 4A</a>). The strains lacking the IncF plasmid or carrying the <em>traQ</em> and <em>traX</em> mutant plasmids formed biofilms with a less structured organization, and without any recognizable mushroom shaped microcolony structures. In these biofilms no observable tolerant subpopulation was detected (<a href="#pone-0001891-g004">Fig. 4A</a>, data not shown). To evaluate the differences in bacterial survival in the biofilms quantitatively, we analyzed the CLSM data with the computer program COMSTAT <a href="#pone.0001891-Heydorn1">[30]</a>. The analysis showed that approximately 35% of the cells survived in the biofilm formed by SAR18 F<sup>+</sup>(<em>traD</em>), while survival was significantly lower (P&lt;0.0030) for SAR18 F<sup>+</sup> biofilms, in which around 15% of the cells were still green after 24h colistin treatment. In the biofilm formed by SAR18 F<sup>+</sup>(<em>traX</em>) less than 5% of the cells were viable after colistin challenge which is significantly less than SAR18F<sup>+</sup> (P&lt;0.0061). If the AF504 F<sup>+</sup>(<em>traD</em>) were challenged with colistin in the flow chamber system an extensive tolerant subpopulation was observed while when the AF504 F<sup>+</sup>(<em>traX</em>) strain no subpopulation could be detected (data not shown).</p>
<div class="figure" id="pone-0001891-g004"><div class="img"><a name="pone-0001891-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The colistin tolerance is influenced by biofilm structural organization and tolerance induction is <em>basS</em> dependent.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>A). CLSM micrographs showing the distribution of live and dead cells in biofilms of the SAR18/F<sup>+</sup> (top left panel), SAR18/F<sup>+</sup>(traD) (top right panel), SAR18/F<sup>+</sup>(traX) (bottom left panel), and PHL628 <em>gfp ompR234</em> (bottom right panel) after 24h challenge by 10 µg/ml colistin. Scale bars represent 50 µm. B). CLSM micrographs showing the distribution of live and dead cells in biofilms of AF504 <em>gfp</em>/F<sup>+</sup> (left panel) and AF574 <em>gfp basS</em>::Kan/F<sup>+</sup> (right panel) mutant showing the <em>basS</em> dependence of subpopulation formation after 24h challenge by 10 µg/ml colistin. Overview images taken with a 40× objective. Scale bars represent 100 µm.</p>
<span>doi:10.1371/journal.pone.0001891.g004</span></div>

<h4>Induction of colistin tolerance is <em>basS</em> dependent</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>The development of the tolerant subpopulation can be due to a phenotypic change of the bacteria in the center of the large micro colonies or to a generic difference in biofilm permeability. We established flow chamber biofilms of <em>E.coli</em> AF504 F<sup>+</sup> with additional chromosomal mutations in the <em>basR/basS</em> two-component regulatory system. There was no measurable effect of the <em>basS</em> mutation on biofilm structure, surface coverage and biomass as determined by COMSTAT analysis (data not shown). No tolerant subpopulation could be detected in the AF504 <em>basS</em> F<sup>+</sup> biofilms after 24 h colistin treatment (<a href="#pone-0001891-g004">Fig. 4B</a>). COMSTAT quantification of the acquired images clearly showed a significant difference in the ratio between PI stained cells and GFP expressing cells in the AF504 F<sup>+</sup> and the AF504 <em>basS</em> F<sup>+</sup> (data not shown) (P&lt;0.0093). These results suggest that subpopulation formation is genetically regulated.</p>


<h4>The <em>yfbE</em> gene is induced within structured biofilms prior to colistin challenge</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>The <em>ybf</em> operon consists of seven genes (<em>yfbE, F, G, H, arnT, yfbW</em> and <em>J</em>) that are implicated in the process of addition of the 4-amino-4-deoxy-L-arabinose (L-Ara4N) moiety to the phosphate groups of lipid A leading to increased polymyxin tolerance <a href="#pone.0001891-Trent1">[32]</a>. The induction of the <em>ybf</em> genes in vitro by low Mg<sup>2+</sup> and high Fe<sup>3+</sup> is under the control of the <em>basR/basS</em> two-component system <a href="#pone.0001891-Hagiwara1">[33]</a>, <a href="#pone.0001891-Winfield1">[34]</a>. To determine if tolerant subpopulation formation is correlated with increased <em>yfbE</em> operon expression, we constructed a plasmid harbouring the AF504 <em>yfbE</em> promoter in front of a promoterless <em>gfp</em> gene and introduced it into our collection of AF504 strains with differential biofilm formation ability. Expression of the <em>yfbE::gfp</em> reporter was induced when the strains were grown in N-minimal media with 10 µΜ Mg<sup>+2</sup> and 100 µM Fe<sup>+3</sup> relative to the expression levels when the strains were grown in media with 10 mΜ Mg<sup>+2</sup>(<a href="#pone-0001891-g005">Fig. 5A</a>). No increase in fluorescence could be detected in strains, with a mutated <em>basS</em> gene, even when they were cultivated under inducing conditions (<a href="#pone-0001891-g005">Fig. 5A</a>), and no induction of the <em>yfbE</em> reporter could be detected by the addition of colistin to the N-minimal media under low or high Mg<sup>2+</sup> conditions (data not shown). We then investigated GFP fluorescence of the different strains in microtiter biofilm assays. The bacterial population in the microtiter wells is highly heterogeneous and a direct measurement of the total fluorescence in the wells can be misleading. Instead, we determined the proportion of cells in the wells with elevated GFP fluorescence by using a strain with constitutive GFP expression to determine maximum range fluorescence and a strain with a promoterless vector control to determine minimum fluorescence. The proportion of induced fluorescent cells was significantly elevated in the biofilms formed by the AF504 F<sup>+</sup> and AF504 F<sup>+</sup>(<em>traD</em>) strains compared to the biofilms formed by the wild type AF504 strain and the F<sup>+</sup>(<em>traX</em>) mutant (<a href="#pone-0001891-g005">Fig. 5B</a>). Interestingly, no increased fluorescence was detected in strains harboring a <em>basS</em> mutation indicating that observed increased fluorescence in the F<sup>+</sup> and F<sup>+</sup>(<em>traD</em>) strains is due to elevated <em>yfbE</em> expression, and that the induction occurs prior to colistin challenge (<a href="#pone-0001891-g005">Fig. 5B</a>).</p>
<div class="figure" id="pone-0001891-g005"><div class="img"><a name="pone-0001891-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The <em>yfbE</em> gene is induced within structured biofilms prior to colistin challenge.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>A). Fluorescence of <em>yfbE</em>-<em>gfp</em> transcriptional fusion expressed in bacteria grown in N-minimal media, pH 7.7, containing 38 mM glycerol with either 10 mM Mg<sup>2+</sup> (black bars), or 10 µM Mg<sup>2+</sup> and 100 µM Fe<sup>3+</sup> (white bars) Data shown are the average of three experiments. Mean fluorescence units were detected by using a Becton Dickingson FACS Vantage SE cell sorter. Strains harboring the <em>attB</em>::<em>bla-rrnB</em>P1::<em>gfp</em>mut3b or the pJBA113 vector alone demonstrated constitutive fluorescence and no fluorescence in all growth conditions, respectively. B). Fraction of fluorescent cells in the microtitre plate biofilm assay with the parental <em>E. coli</em> AF504 in the presence and absence of the de-repressed IncF plasmid and mutants in the IncF plasmid (<em>traD,</em>), and the reciprocal insertion deletions of the <em>basS</em> and <em>basR</em> genes, harboring the <em>yfbE</em>-<em>gfp</em> transcriptional fusion. The threshold for GFP fluorescence was cells with a value above 600 arbitrary units. 99% of the constitutively expressing cells were above this threshold while 96% of the negative control was below. C). CLSM micrographs showing the expression of the <em>yfbE</em>-<em>gfp</em> transcriptional fusion in flow chamber biofilms of AF504/F<sup>+</sup> (left panel) and AF574 <em>basS/</em>F<sup>+</sup> (right panel). Green indicates GFP expressing cells all other cells appear red from treatment with SYTO62.</p>
<span>doi:10.1371/journal.pone.0001891.g005</span></div><a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>The <em>yfbE::gfp</em> reporter expression was also investigated in the hydrodynamic flow chamber system. AF504 F<sup>+</sup> and AF504 <em>basS</em> F<sup>+</sup> strains were inoculated in the system and GFP expression monitored after 24h our incubation without colistin treatment. Green fluorescence could be detected in the centre of large micro colonies in AF504 F<sup>+</sup> biofilm while no fluorescence could be detected in <em>basS</em> mutant biofilm indictaing that there is a correlation of <em>yfbE</em> expression and sub population formation in flow chamber system (<a href="#pone-0001891-g005">Fig. 5C</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The formation of bacterial biofilms is often associated with a reduced susceptibility to antimicrobial challenge <a href="#pone.0001891-Nickel1">[35]</a>, <a href="#pone.0001891-Ceri1">[36]</a>. The mechanisms underlying the observed increased tolerance are poorly elucidated and a direct causal relationship between antibiotic tolerance development and biofilm formation is inadequately supported <a href="#pone.0001891-Lewis1">[37]</a>, <a href="#pone.0001891-Gilbert1">[38]</a>. Recently, Mah <em>et. al.</em> discovered that periplasmic glucans interact with the aminoglycoside tobramycin in <em>P. aeruginosa</em> biofilms and were proposed to delay by sequestration the effect of the antibiotic <a href="#pone.0001891-Mah1">[39]</a>. <em>P. aeruginosa</em> also has the ability to convert to an antibiotic-resistant phenotype after antibiotic exposure with enhanced biofilm forming ability <a href="#pone.0001891-Drenkard2">[40]</a>. A large proportion of the studies investigating biofilm tolerance are comparisons between cultures grown under shaking conditions (planktonic growth) and a particular biofilm model system <a href="#pone.0001891-Ceri1">[36]</a>, <a href="#pone.0001891-Larsen1">[41]</a>–<a href="#pone.0001891-Kumon1">[49]</a>. The result of this kind of comparisons can be misleading because there are many other factors affecting antibiotic susceptibility other than biofilm formation in these systems. Cell number, growth phase, and media flow are factors that are varying in the different growth systems and can potentially affect antibiotic tolerance. There is a risk that results obtained under these conditions more reflect differences between the particular growth model systems rather than actually describing particular aspects of biofilm physiology compared to planktonically grown cells. In studies where tolerance has been investigated under physiologically similar conditions the differences in susceptibility has been less dramatic <a href="#pone.0001891-Williams1">[43]</a>, <a href="#pone.0001891-Spoering1">[50]</a>. To better understand the influence of biofilm formation on the development of antibiotic tolerance, we exploited the recent discovery of Reisner <em>et al.</em> <a href="#pone.0001891-Reisner2">[6]</a> that biofilm structure and organization can be manipulated by varying the IncF transfer pilus configuration (<a href="#pone-0001891-g001">Fig. 1 A</a>). Using this set of isogenic <em>E. coli</em> strains enable the investigation of antibiotic tolerance and resistance under nearly identical physiological conditions and the direct study of the influence of biofilm formation on antibiotic tolerance rather than comparing different growth model systems. We demonstrate that biofilm formation can fundamentally influence bacterial survival upon antibiotic challenge, but the increased survival did not correlate directly with biofilm forming ability as such, but rather depended on the organization of the biofilm and on the antibiotic used (<a href="#pone-0001891-g001">Fig. 1</a> and <a href="#pone-0001891-g002">2</a>). When the biofilm forming strains were challenged with the surface-active peptide antibiotic colistin, there was a dramatic difference in survival between strains harboring the de-repressed incF transfer pilus and the plasmid free strains (<a href="#pone-0001891-g001">Figs. 1</a> and <a href="#pone-0001891-g002">2</a>). In contrast, when the biofilms were challenged with the fluoroquinolone ciprofloxacin, an antibiotic with an intracellular target, no significant differences in survival were observed between the investigated strains. Interestingly, resistance to ciprofloxacin is only known to occur through chromosomal mutations in <em>E. coli,</em> while antimicrobial peptide resistance and tolerance can arise through both physiological and mutational mechanisms <a href="#pone.0001891-Drlica1">[20]</a>, <a href="#pone.0001891-Hooper1">[21]</a>, <a href="#pone.0001891-Winfield1">[34]</a>, <a href="#pone.0001891-Dame1">[51]</a>. In fact, our data suggest that physiological tolerance mechanism may be essential for the increased colistin tolerance of bacteria in biofilms.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>When flow chamber biofilms with complex spatial organization were challenged with colistin, subpopulations of tolerant cells were observed, while no such populations were detected in biofilms with flat, undifferentiated structure (<a href="#pone-0001891-g003">Figs. 3</a> and <a href="#pone-0001891-g004">4</a>). The localization of the subpopulation in the center of the micro-colonies indicates that the local environment within the biofilm induces subpopulation formation in our experimental system. Tolerance development is thus structurally dependent, influenced by the conditions created within the biofilm. However, we do not believe our data directly support a correlation of the difference in colistin tolerance to a distinct spatial organization and it is important to make distinction between a phenotype being structurally dependent and being correlated with a specific structure. When mutations in <em>basS</em> were introduced into the strains with wildtype F as well as the <em>traD</em> mutant, survival was comparable to the strain without F (<a href="#pone-0001891-g002">Fig. 2A</a>). Since <em>basS</em> is regulating the LPS modifications leading to augmented antimicrobial peptide tolerance, these data show that the increased survival is directly coupled to a genetically regulated event. We show that the expression of the <em>basR/S</em> regulated <em>yfbE</em> operon is up-regulated in the strains with increased survival and subpopulation formation, when the strains are grown as biofilms (<a href="#pone-0001891-g005">Fig. 5B</a>). Because the enzymes encoded by the operon are involved in the process of LPS modification, it is likely that it is through this process that the tolerant subpopulation is formed <a href="#pone.0001891-Trent1">[32]</a>. The increased expression of the operon occurs prior to colistin addition, indicating that the subpopulation formation process is not directly coupled to the addition of colistin or an adaptation to the presence of antimicrobial peptide (<a href="#pone-0001891-g005">Fig. 5B</a>). The main rational behind including the <em>basRS</em> mutant was to be able to differentiate between the development of a generic biofilm induced tolerance and specific genetic regulation to explain the observed increased survival in the F<sup>+</sup> and F(<em>traD</em>) biofilms. Our data cannot rule out that there are other <em>basRS</em> dependent resistance mechanisms than LPS modification involved in the observed phenomena. This possibility does not change the fact that colistin tolerance is dependent on a specific genetic mechanisms rather than a generic biofilm property.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>When <em>P. aeruginosa</em> was challenged with colistin in the flow chamber system, a tolerant subpopulation was also found <a href="#pone.0001891-Haagensen1">[52]</a>. In contrast to <em>E. coli</em>, the surviving population was not observed in the center of large micro colonies but rather on the surface of the biofilm, and it was associated with a migrating cell population that eventually colonized the top of the microcolonies, resulting in the development of mushroom-shaped multicellular structures characterizing <em>P</em>. <em>aeruginosa</em> flow chamber biofilms <a href="#pone.0001891-Haagensen1">[52]</a>. The <em>P. aeruginosa pmr</em> system is known to be activated by the presence of cationic antimicrobial peptides <a href="#pone.0001891-Ausubel1">[53]</a>, <a href="#pone.0001891-Bertani1">[54]</a>. The <em>yfbE</em> homolog, <em>pmrH</em>, was indeed induced in the migrating tolerant subpopulation when the peptide was added to the system while cells in the stalks of the mushroom-shaped micro colonies seemed unable to induce or sense the presence of the peptide and were subsequently killed <a href="#pone.0001891-Haagensen1">[52]</a>. It is also interesting to note that even though the cells in the deeper layers of the flow chamber biofilm were coated by a thick layer of tolerant cells, this was not able to protect these cells from the action of the antibiotic <a href="#pone.0001891-Haagensen1">[52]</a>. No colistin-mediated induction has been shown for <em>E.coli,</em> indicating a dissimilar regulation of homologous genes responding to disparate environmental stimuli since a migrating population is not present in the IncF mediated biofilm. Even though the tolerant subpopulations form at different locations within the <em>E.coli</em> and <em>P.aeruginosa</em> biofilms, there are notable common features. Both populations require the presence of genetically encoded tolerance mechanisms and seem to be influenced by specific environmental cues that are independent of general biofilm properties (<a href="#pone-0001891-g003">Figs. 3</a> and <a href="#pone-0001891-g004">4</a>) <a href="#pone.0001891-Haagensen1">[52]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Physicochemical gradients have previously been demonstrated in microbial biofilms <a href="#pone.0001891-Xu1">[9]</a>, <a href="#pone.0001891-Xu2">[10]</a>. However, little is known about how these gradients affect the behavior of the cells integrated in the biofilm structure. The specific signals regulating the formation of the colistin tolerant subpopulation in <em>E.coli</em> are currently unknown, but since the development is under control of the <em>basR/basS</em> two-component system, it is probable that local environmental stimuli affect the cells in the biofilms. It is known that media with low Mg2+ in combination with high Fe<sup>3</sup>+ ion concentrations activate this regulatory network <em>in vitro</em> (<a href="#pone-0001891-g005">Fig. 5B</a>) <a href="#pone.0001891-Winfield1">[34]</a>. We find it unlikely that such conditions prevail inside the large micro-colonies under the conditions tested. Luria broth is a medium with relatively low Mg2+ levels, but no extra iron is added in the experiments making it difficult to imagine a local accumulation of this ion in the micro colonies in sufficient amounts to induce <em>basS</em> activation. The magnesium ion concentration in the minimal medium used in flow chamber experiments is sufficiently high to prevent <em>basS</em> dependent induction, even in high iron conditions in a non biofilm setting and it is therefore unlikely that differential magnesium iron concentration is responsible for the observed induction <a href="#pone.0001891-Hagiwara1">[33]</a>. The close proximity of the cells in the larger micro colonies in biofilm communities may create conditions mimicking the <em>in vitro</em> effect of high iron levels. The bacteria may have to undergo complex physiological adaptations in response to the consequences of the high cell density within the micro-colonies. It is interesting to note, that the homologous operon to the <em>yfbE</em>, the <em>pmrH</em> operon in <em>Salmonella typhimurium</em> is induced during swarm-cell differentiation, and an increased tolerance of swarmer cells to peptide antibiotics have been found to be due to an altered phenotypic state of the cells, in the absence of genetic changes <a href="#pone.0001891-Kim1">[55]</a>, <a href="#pone.0001891-Kim2">[56]</a>. It is possible that the conditions within the organized microcolonies mimic those that induce swarm-cell development. In addition, it cannot be ruled out that the <em>basR/basS</em> are components of a general stress response to toxic compounds produced by the metabolism of the cells in the micro-colonies.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The data presented here suggest that biofilm mediated antibiotic tolerance may often be a matter of subpopulation differentiation within the biofilm. The basis for tolerance development is not the biofilm formation <em>per se</em>, but that the spatial and environmental heterogeneity provided by structurally organized biofilms have the capacity to promote tolerant subpopulation formation. In our experimental system, the increased antibiotic tolerance exhibited by biofilm formation is antibiotic specific and conditional, dependent on the actual biofilm structure and on the presence of a specific genetically encoded tolerance mechanism. Our data even suggest that specific modes of biofilm organization may induce antibiotic tolerance, and that the factors determining the properties of heterogeneous aggregated microbial communities may act on the individual cells within the structures rather than on the community. No heritable genetic changes are involved in the phenomenon; we suggest that many other cases of biofilm-induced tolerance to antibiotics could be the result of similar mechanisms. The biofilm mode of growth does not directly predict antibiotic resistance and we suggest that a complete elucidation of bacterial biofilm antibiotic tolerance phenomena merits investigations that are not limited by preconceived assumptions of the consequence of biofilm formation.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Bacterial strains, plasmids and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The bacterial strains relevant to this study are listed in <a href="#pone-0001891-t001">Table 1</a>. Plasmids and PCR primers used in the study are listed in <a href="#pone-0001891-t002">Table 2</a>. Except where indicated, the strains used are derivatives of <em>E. coli</em> K-12 strain MG1655 or CSH26. Luria–Bertani (LB) broth or agar <a href="#pone.0001891-Bertani1">[54]</a> was used for standard cultivation supplemented with 0.2% glucose. Appropriate antibiotics were added at the following final concentrations 100 µg ml<sup>−1</sup> ampicillin (Amp), 10 µg ml<sup>−1</sup> chloramphenicol (Cm), 100 µg ml<sup>−1</sup> nalidixic acid (Nal) and 50 µg ml<sup>−1</sup> kanamycin (Km) when required.</p>
<div class="figure" id="pone-0001891-t002"><div class="img"><a name="pone-0001891-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001891" data-uri="info:doi/10.1371/journal.pone.0001891.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001891.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001891.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001891.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001891.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001891.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Plasmids and primers used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001891.t002</span></div>

<h4>Genetic techniques and mutant construction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Unless otherwise stated all genetic methods followed standard procedures as described by Ausubel, F.M., and others <a href="#pone.0001891-Ausubel1">[53]</a>. <em>E. coli</em> phage P1 transductions were performed by the method of Miller <a href="#pone.0001891-Miller2">[57]</a>. To enable conjugative plasmid transfer a spontaneous nalidixic acid resistant mutant of MG1655 was isolated on LB agar plates containing 10 µg ml<sup>−1</sup> nalidixic acid at 37°C and designated AF504. Conjugations of pAR108 pOX38-<em>traD411</em> and pOX38-<em>traX482</em> into AF504 from were performed by plate mating <a href="#pone.0001891-Maloy1">[58]</a> with SAR18/pAR108, SAR18/pOX38-<em>traD411</em> pSK410 and SAR18/pOX38-<em>traX482</em> as respective donors. The defined insertion mutations in <em>basS</em> and <em>basR</em> were produced by using one-step allelic replacement utilizing the λ Red recombinase system <a href="#pone.0001891-Datsenko1">[29]</a>. In short, transformants of AF504 carrying the Red recombinase plasmid pKD46 were grown in 5 ml SOB cultures with ampicillin and L-arabinose at 30°C. pKD46 shows temperature-sensitive replication and the Red recombinase genes are under the control of a arabinose sensitive promoter. The culture was grown to an OD600 of approximately 0.6 and then made electro-competent by washing four times in ice-cold 15% glycerol. PCR products were polymerase chain reaction (PCR)-amplified via the Expand High Fidelity System (Roche) on a Biometra T3 thermocycler using the plasmids pKD4 and pKD3 as templates for kanamycin resistant and chloramphenicol resistant insertions respectively. The amplified products were gel-purified, digested with <em>DpnI</em>, repurified and suspended in water. Electroporation was performed by using a Gene pulser (Bio-Rad) as recommended by the manufacturer using 50 µl bacterial cells and 10–100 ng PCR product. The cells were then added to 1 ml of SOC medium <a href="#pone.0001891-Ausubel1">[53]</a> and incubated for 1 h at 37°C and then one-half was spread onto selective agar plates. After primary selection, putative mutants were re-streaked on selective media at 37°C and then tested for ampicillin sensitivity. Putative insertion mutations were transduced with phage P1 into fresh AF504 background for further phenotypic characterization. The primers basSmutf and basSmutr were used for allelic replacement. Insertions were verified by PCR. The cloning of the full-length <em>basRS</em> genes and <em>yfbE</em> promoter region, the primers ecoBasR1, xbaBasS1 and yfbEco, yfbExba were used, respectively. The primers contain an EcoRI restriction site or an XbaI for site to facilitate insertion in the corresponding restriction sites on the <em>gfp</em> expression vector pJBA113. The amplification was performed as described above for the mutagenesis PCR. The PCR fragment was inserted into the corresponding sites in the multiple cloning site of plasmid pJBA113 and electroporated into <em>E. coli</em> XL10 (Stratagene), resulting in the transcriptional GFP reporter plasmids pBASrs and pYFBE32. DNA extraction, treatment with modifying enzymes and restriction endonucleases, ligation of DNA fragments with T4 ligase and transformation of <em>E. coli</em> was performed using standard methods <a href="#pone.0001891-Ausubel1">[53]</a>. Plasmid DNA was isolated with the Spin Miniprep Kit (Qiagen). DNA fragments agarose gel purified using GFX DNA Purification system (Amersham Pharmacia Biothech). Insertion of <em>gfpmut3</em>* under the <em>rrnB</em>P1 promoter in the <em>attB</em> site <em>E.coli</em> AF504 was performed using a vector system previously described <a href="#pone.0001891-Diederich1">[59]</a>. The pLDR11 vector was digested with NotI endonuclease, dephosphorylated and the 1.8 kb fragment containing the <em>bla</em> gene and the lambda <em>attP</em> attachment site was isolated. This fragment was ligated with a NotI fragment of pSM1690. Competent <em>E. coli</em> cells carrying transformants were selected at 42°C.</p>


<h4>Microtitre plate biofilm and antibiotic tolerance assays</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Biofilm formation and antibiotic tolerance was quantified using the static microtiter plate biofilm assay. Bacterial cultures of the investigated strains were grown in 5ml LB in 37°C to turbidity. The cultures were diluted 1:300 and inoculated in triplicate in a 96-well microtiter plate (V-bottom; BioSterilin). Test plates were transferred to plastic bags to avoid evaporation and statically incubated for 24 h at 37°C. Biofilm formation was assayed by staining of attached cells with crystal violet as described <a href="#pone.0001891-Reisner3">[60]</a>. In short, the growth medium was carefully removed from the biofilm microtiter wells, washed twice with 0.9% NaCl solution, and covered with 0.1% CV for 15 min. The wells were again washed twice with 0.9% NaCl where after surface bound CV was extracted by addition of 96% ethanol. Absorbance of stained wells was determined at 590 nm with a VICTOR<sup>2</sup> multilabel counter (Perkin Elmer, Inc.). For the antibiotic tolerance microtiter assay, biofilms were grown in microtitre plates without antibiotics as for biofilm quantification. After the 24 h incubation, colistin (H. Lundbeck, Copenhagen) or ciprofloxacin (Bayer) was carefully added to the wells at the indicated concentrations. After 24 h static additional incubation cells were vigorously re-suspended and the total CFUs were determined by serial dilution and plating on LB plates without prior washing. Consistent removal of the bacterial cells was confirmed by crystal-violet staining of the wells. For microtitre biofilm assays and tolerance assay the data is presented as mean±S.E.M. of 3 experiments performed in triplicate. In parallel with each biofilm experiment, a microtiter plate MIC was determined for the planktonic cells in the culture used as the inoculum. That the biofilm-associated cells truly were dispersed to single cells by vigorous aspiration was ascertained by observing the dislodged biofilm under a microscope prior to plating.</p>


<h4>GFP Expression Assay and <em>yfbE</em> expression visualization</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Strains harboring the plasmids pYFBe32 were grown in N-minimal media, pH 7.7, containing 38 mM glycerol with either 10 µM MgCl<sub>2</sub>, or 10 µM MgCl<sub>2</sub> and 100 µM FeSO<sub>4</sub> and supplemented with 50 µg/ml ampicillin. GFP expression was analyzed after 6h of growth at 37°C by using a fluorescence-activated cell sorter (Becton Dickinson FACS Vantage SE). Assays were performed in triplicate. For <em>yfbE</em> expression in biofilms, strains were inoculated in 96 well microtiter plates as described for the antibiotic tolerance assay. After incubation for 24 h at 37°C cells were vigorously re-suspended washed twice in 0.9% NaCl<sub>2</sub> and GFP fluorescent monitored in the FACS. Strains harboring the <em>attB</em>::<em>bla-rrnB</em>P1::<em>gfp</em>mut3b (GFP constitutive) and the pJBA113 vector alone (WT no reporter) demonstrated constitutive fluorescence and no fluorescence in all growth conditions, respectively. The threshold for GFP fluorescence was set as cells with a value above 600 arbitrary units. 99% of the constitutively expressing cells were above this threshold while 96% of the negative control was below. Experiments were done in triplicate. For visualization of <em>yfbE</em> expression in the hydrodynamic flow chamber systems the non-GFP fluorescent cells were counterstained with SYTO62 (Invitrogen) according to the manufacturers instructions.</p>


<h4>Bacterial viability assay and fluorescence-activated cell sorting</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The LIVE/DEAD <em>Bac</em>light bacterial viability kit (Invitrogen) was used to for distinguishing live and dead bacteria after antibiotic treatment according to the manufacturers instructions. For the biofilm experiments fluorescence-tagged bacteria (GFP) and red-fluorescent propidium iodide (PI) staining were used. Red fluorescing and green fluorescing cells from a colistin-treated biofilm were sorted by fluorescence-activated cell sorting (Becton Dickinson FACS Vantage SE) and plated on Luria Bertani plates to verify that red fluorescing cells were not viable and green fluorescing cells were viable.</p>


<h4>Flow chamber biofilms</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Cultivation of biofilms in laminar flow was done at 30°C as previously described <a href="#pone.0001891-Reisner2">[6]</a>. Flow channels irrigated with AB minimal medium <a href="#pone.0001891-Clark1">[61]</a> supplemented with 0.01 mM Fe-EDTA, 1.0 mg/ml thiamine,or 1.0 mg/ml proline and uracil, where appropriate. The channels were inoculated with 250 µl of normalized dilutions (optical density at 600 nm of 0.05) of saturated bacterial cultures. After 24h incubation, biofilms were challenge with 50 µg/ml, 25 µg/ml or 10 µg/ml colistin for an additional 24h where after the bacterial red fluorescent viability stain propidium iodide was injected into the flow channels and incubated for 15 min.</p>


<h4>Images acquisition and analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>All microscopic observations were performed by a Zeiss LSM510 Confocal Laser Scanning Microscope, CLSM (Carl Zeiss, Jena, Germany) equipped with an argon/krypton laser and detectors and filter sets for simultaneous monitoring of GFP (excitation 488 nm, emission 517 nm) and red-fluorescents emitted from the PI (excitation 543nm, emission 565 nm). For the quantification of biofilms and analysis of living and dead cells in treated biofilms the software COMSTAT was used <a href="#pone.0001891-Heydorn2">[62]</a>. Images were obtained using a 40×/1.3 Plan-Neofluar oil objective. Multichannel simulated fluorescence projection (SFP, a shadow projection) images and vertical cross sections through the biofilm were generated by using the IMARIS software package (Bitplane AG, Zürich, Switzerland) running on a PC. Images were further processed for display by using Photoshop software (Adobe, Mountain View, Calif.). Time series experiments were performed on the Zeiss LSM510 microscope and movie sequences were produced using the Jasc software, Animation Shop 3. For the quantification of <em>E. coli</em> F+ wild type and the different mutants three independent biofilm experiments were performed acquiring at least 9 image stacks for each strain and situation/time point randomly down through the flow channel using a 40x/1.3 Plan-Neofluar oil objective. For the quantification of living and dead cells, each strain was grown in three channels in three independent experiments, and from each experiment six image stacks were acquired randomly in the channel. Images were further treated using COMSTAT <a href="#pone.0001891-Heydorn2">[62]</a>.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Statistical significance was determined using unpaired Student's <em>t</em>-test, values of <em>P</em>&lt;0.05 were considered significant.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001891"></div><a name="pone.0001891.s001" id="pone.0001891.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001891.s001">Movie S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI"><em>E.coli</em> flow-chamber biofilms are sensitive to the cationic antimicrobial peptide colistin. Time series showing killing dynamic of a mature AF504 <em>gfp</em>/F<sup>+</sup> biofilm. Staining of the biofilm with PI (red) before/during image acquisition for monitoring of dead cells. Images were acquired every 15 min for 42 h.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001891.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(7.65 MB MPG)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SM AF JH. Performed the experiments: AF JH CZ. Analyzed the data: SM AF JH CZ. Contributed reagents/materials/analysis tools: AF JH CS. Wrote the paper: SM AF JH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001891-Ghigo1" id="pone.0001891-Ghigo1"></a>Ghigo JM (2001) Natural conjugative plasmids induce bacterial biofilm development. Nature  412: 442–445.  <ul class="find" data-citedArticleID="1082844" data-doi="10.1038/35086581"><li><a href="http://dx.doi.org/10.1038/35086581" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+conjugative+plasmids+induce+bacterial+biofilm+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+conjugative+plasmids+induce+bacterial+biofilm+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001891-Christensen1" id="pone.0001891-Christensen1"></a>Christensen BB, Sternberg C, Andersen JB, Eberl L, Moller S, et al.  (1998) Establishment of new genetic traits in a microbial biofilm community. Appl Environ Microbiol  64: 2247–2255.  <ul class="find" data-citedArticleID="1082818"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Establishment+of+new+genetic+traits+in+a+microbial+biofilm+community.&amp;auth=&amp;atitle=Establishment+of+new+genetic+traits+in+a+microbial+biofilm+community." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Establishment+of+new+genetic+traits+in+a+microbial+biofilm+community." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Establishment+of+new+genetic+traits+in+a+microbial+biofilm+community.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001891-Molin1" id="pone.0001891-Molin1"></a>Molin S, Tolker-Nielsen T (2003) Gene transfer occurs with enhanced efficiency in biofilms and induces enhanced stabilisation of the biofilm structure. Curr Opin Biotechnol  14: 255–261.  <ul class="find" data-citedArticleID="1082896" data-doi="10.1016/s0958-1669(03)00036-3"><li><a href="http://dx.doi.org/10.1016/s0958-1669(03)00036-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+transfer+occurs+with+enhanced+efficiency+in+biofilms+and+induces+enhanced+stabilisation+of+the+biofilm+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+transfer+occurs+with+enhanced+efficiency+in+biofilms+and+induces+enhanced+stabilisation+of+the+biofilm+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001891-Dudley1" id="pone.0001891-Dudley1"></a>Dudley EG, Abe C, Ghigo JM, Latour-Lambert P, Hormazabal JC, et al.  (2006) An IncI1 plasmid contributes to the adherence of the atypical enteroaggregative Escherichia coli strain C1096 to cultured cells and abiotic surfaces. Infect Immun  74: 2102–2114.  <ul class="find" data-citedArticleID="1082840" data-doi="10.1128/iai.74.4.2102-2114.2006"><li><a href="http://dx.doi.org/10.1128/iai.74.4.2102-2114.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+IncI1+plasmid+contributes+to+the+adherence+of+the+atypical+enteroaggregative+Escherichia+coli+strain+C1096+to+cultured+cells+and+abiotic+surfaces." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+IncI1+plasmid+contributes+to+the+adherence+of+the+atypical+enteroaggregative+Escherichia+coli+strain+C1096+to+cultured+cells+and+abiotic+surfaces.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001891-Reisner1" id="pone.0001891-Reisner1"></a>Reisner A, Holler BM, Molin S, Zechner EL (2006) Synergistic effects in mixed Escherichia coli biofilms: conjugative plasmid transfer drives biofilm expansion. J Bacteriol  188: 3582–3588.  <ul class="find" data-citedArticleID="1082904" data-doi="10.1128/jb.188.10.3582-3588.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.10.3582-3588.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synergistic+effects+in+mixed+Escherichia+coli+biofilms%3A+conjugative+plasmid+transfer+drives+biofilm+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synergistic+effects+in+mixed+Escherichia+coli+biofilms%3A+conjugative+plasmid+transfer+drives+biofilm+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001891-Reisner2" id="pone.0001891-Reisner2"></a>Reisner A, Haagensen JA, Schembri MA, Zechner EL, Molin S (2003) Development and maturation of Escherichia coli K-12 biofilms. Mol Microbiol  48: 933–946.  <ul class="find" data-citedArticleID="1082906" data-doi="10.1046/j.1365-2958.2003.03490.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03490.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+maturation+of+Escherichia+coli+K-12+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+maturation+of+Escherichia+coli+K-12+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001891-Drenkard1" id="pone.0001891-Drenkard1"></a>Drenkard E (2003) Antimicrobial resistance of Pseudomonas aeruginosa biofilms. Microbes Infect  5: 1213–1219.  <ul class="find" data-citedArticleID="1082834" data-doi="10.1016/j.micinf.2003.08.009"><li><a href="http://dx.doi.org/10.1016/j.micinf.2003.08.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antimicrobial+resistance+of+Pseudomonas+aeruginosa+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antimicrobial+resistance+of+Pseudomonas+aeruginosa+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001891-Wimpenny1" id="pone.0001891-Wimpenny1"></a>Wimpenny J, Manz W, Szewzyk U (2000) Heterogeneity in biofilms. FEMS Microbiol Rev  24: 661–671.  <ul class="find" data-citedArticleID="1082928" data-doi="10.1111/j.1574-6976.2000.tb00565.x"><li><a href="http://dx.doi.org/10.1111/j.1574-6976.2000.tb00565.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneity+in+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneity+in+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001891-Xu1" id="pone.0001891-Xu1"></a>Xu KD, Stewart PS, Xia F, Huang CT, McFeters GA (1998) Spatial physiological heterogeneity in Pseudomonas aeruginosa biofilm is determined by oxygen availability. Appl Environ Microbiol  64: 4035–4039.  <ul class="find" data-citedArticleID="1082932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Spatial+physiological+heterogeneity+in+Pseudomonas+aeruginosa+biofilm+is+determined+by+oxygen+availability.&amp;auth=&amp;atitle=Spatial+physiological+heterogeneity+in+Pseudomonas+aeruginosa+biofilm+is+determined+by+oxygen+availability." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spatial+physiological+heterogeneity+in+Pseudomonas+aeruginosa+biofilm+is+determined+by+oxygen+availability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spatial+physiological+heterogeneity+in+Pseudomonas+aeruginosa+biofilm+is+determined+by+oxygen+availability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001891-Xu2" id="pone.0001891-Xu2"></a>Xu KD, McFeters GA, Stewart PS (2000) Biofilm resistance to antimicrobial agents. Microbiology  146: 547–549.  <ul class="find" data-citedArticleID="1082934"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biofilm+resistance+to+antimicrobial+agents.&amp;auth=&amp;atitle=Biofilm+resistance+to+antimicrobial+agents." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biofilm+resistance+to+antimicrobial+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biofilm+resistance+to+antimicrobial+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001891-Costerton1" id="pone.0001891-Costerton1"></a>Costerton JW, Lewandowski Z, Caldwell DE, Korber DR, Lappin-Scott HM (1995) Microbial biofilms. Annu Rev Microbiol  49: 711–745.  <ul class="find" data-citedArticleID="1082824" data-doi="10.1146/annurev.mi.49.100195.003431"><li><a href="http://dx.doi.org/10.1146/annurev.mi.49.100195.003431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001891-Kuehn1" id="pone.0001891-Kuehn1"></a>Kuehn M, Hausner M, Bungartz HJ, Wagner M, Wilderer PA, et al.  (1998) Automated confocal laser scanning microscopy and semiautomated image processing for analysis of biofilms. Appl Environ Microbiol  64: 4115–4127.  <ul class="find" data-citedArticleID="1082870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Automated+confocal+laser+scanning+microscopy+and+semiautomated+image+processing+for+analysis+of+biofilms.&amp;auth=&amp;atitle=Automated+confocal+laser+scanning+microscopy+and+semiautomated+image+processing+for+analysis+of+biofilms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automated+confocal+laser+scanning+microscopy+and+semiautomated+image+processing+for+analysis+of+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automated+confocal+laser+scanning+microscopy+and+semiautomated+image+processing+for+analysis+of+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001891-Miller1" id="pone.0001891-Miller1"></a>Miller SI, Ernst RK, Bader MW (2005) LPS, TLR4 and infectious disease diversity. Nat Rev Microbiol  3: 36–46.  <ul class="find" data-citedArticleID="1082892" data-doi="10.1038/nrmicro1068"><li><a href="http://dx.doi.org/10.1038/nrmicro1068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LPS%2C+TLR4+and+infectious+disease+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LPS%2C+TLR4+and+infectious+disease+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001891-Littlewood1" id="pone.0001891-Littlewood1"></a>Littlewood JM, Koch C, Lambert PA, Hoiby N, Elborn JS, et al.  (2000) A ten year review of colomycin. Respir Med  94: 632–640.  <ul class="find" data-citedArticleID="1082880" data-doi="10.1053/rmed.2000.0834"><li><a href="http://dx.doi.org/10.1053/rmed.2000.0834" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+ten+year+review+of+colomycin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+ten+year+review+of+colomycin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001891-Harada1" id="pone.0001891-Harada1"></a>Harada K, Asai T, Kojima A, Oda C, Ishihara K, et al.  (2005) Antimicrobial susceptibility of pathogenic Escherichia coli isolated from sick cattle and pigs in Japan. J Vet Med Sci  67: 999–1003.  <ul class="find" data-citedArticleID="1082854" data-doi="10.1292/jvms.67.999"><li><a href="http://dx.doi.org/10.1292/jvms.67.999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antimicrobial+susceptibility+of+pathogenic+Escherichia+coli+isolated+from+sick+cattle+and+pigs+in+Japan." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antimicrobial+susceptibility+of+pathogenic+Escherichia+coli+isolated+from+sick+cattle+and+pigs+in+Japan.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001891-Li1" id="pone.0001891-Li1"></a>Li J, Nation RL, Milne RW, Turnidge JD, Coulthard K (2005) Evaluation of colistin as an agent against multi-resistant Gram-negative bacteria. Int J Antimicrob Agents  25: 11–25.  <ul class="find" data-citedArticleID="1082878" data-doi="10.1016/j.ijantimicag.2004.10.001"><li><a href="http://dx.doi.org/10.1016/j.ijantimicag.2004.10.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+colistin+as+an+agent+against+multi-resistant+Gram-negative+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+colistin+as+an+agent+against+multi-resistant+Gram-negative+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001891-Conrad1" id="pone.0001891-Conrad1"></a>Conrad RS, Galanos C (1989) Fatty acid alterations and polymyxin B binding by lipopolysaccharides from Pseudomonas aeruginosa adapted to polymyxin B resistance. Antimicrob Agents Chemother  33: 1724–1728.  <ul class="find" data-citedArticleID="1082822" data-doi="10.1128/aac.33.10.1724"><li><a href="http://dx.doi.org/10.1128/aac.33.10.1724" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fatty+acid+alterations+and+polymyxin+B+binding+by+lipopolysaccharides+from+Pseudomonas+aeruginosa+adapted+to+polymyxin+B+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fatty+acid+alterations+and+polymyxin+B+binding+by+lipopolysaccharides+from+Pseudomonas+aeruginosa+adapted+to+polymyxin+B+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001891-Vaara1" id="pone.0001891-Vaara1"></a>Vaara M, Vaara T, Jensen M, Helander I, Nurminen M, et al.  (1981) Characterization of the lipopolysaccharide from the polymyxin-resistant pmrA mutants of Salmonella typhimurium. FEBS Lett  129: 145–149.  <ul class="find" data-citedArticleID="1082920" data-doi="10.1016/0014-5793(81)80777-6"><li><a href="http://dx.doi.org/10.1016/0014-5793(81)80777-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+lipopolysaccharide+from+the+polymyxin-resistant+pmrA+mutants+of+Salmonella+typhimurium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+lipopolysaccharide+from+the+polymyxin-resistant+pmrA+mutants+of+Salmonella+typhimurium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001891-Gellert1" id="pone.0001891-Gellert1"></a>Gellert M, Mizuuchi K, O'Dea MH, Itoh T, Tomizawa JI (1977) Nalidixic acid resistance: a second genetic character involved in DNA gyrase activity. Proc Natl Acad Sci U S A  74: 4772–4776.  <ul class="find" data-citedArticleID="1082842" data-doi="10.1073/pnas.74.11.4772"><li><a href="http://dx.doi.org/10.1073/pnas.74.11.4772" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nalidixic+acid+resistance%3A+a+second+genetic+character+involved+in+DNA+gyrase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nalidixic+acid+resistance%3A+a+second+genetic+character+involved+in+DNA+gyrase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001891-Drlica1" id="pone.0001891-Drlica1"></a>Drlica K, Zhao X (1997) DNA gyrase, topoisomerase IV, and the 4-quinolones. Microbiol Mol Biol Rev  61: 377–392.  <ul class="find" data-citedArticleID="1082838"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones.&amp;auth=&amp;atitle=DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+gyrase%2C+topoisomerase+IV%2C+and+the+4-quinolones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001891-Hooper1" id="pone.0001891-Hooper1"></a>Hooper DC (1999) Mode of action of fluoroquinolones. Drugs  58: 6–10.  <ul class="find" data-citedArticleID="1082862" data-doi="10.2165/00003495-199958002-00002"><li><a href="http://dx.doi.org/10.2165/00003495-199958002-00002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mode+of+action+of+fluoroquinolones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mode+of+action+of+fluoroquinolones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001891-Maneewannakul1" id="pone.0001891-Maneewannakul1"></a>Maneewannakul K, Maneewannakul S, Ippen-Ihler K (1993) Synthesis of F pilin. J Bacteriol  175: 1384–1391.  <ul class="find" data-citedArticleID="1082886"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synthesis+of+F+pilin.&amp;auth=&amp;atitle=Synthesis+of+F+pilin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthesis+of+F+pilin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthesis+of+F+pilin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001891-Moore1" id="pone.0001891-Moore1"></a>Moore D, Hamilton CM, Maneewannakul K, Mintz Y, Frost LS, et al.  (1993) The Escherichia coli K-12 F plasmid gene traX is required for acetylation of F pilin. J Bacteriol  175: 1375–1383.  <ul class="find" data-citedArticleID="1082898"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Escherichia+coli+K-12+F+plasmid+gene+traX+is+required+for+acetylation+of+F+pilin.&amp;auth=&amp;atitle=The+Escherichia+coli+K-12+F+plasmid+gene+traX+is+required+for+acetylation+of+F+pilin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Escherichia+coli+K-12+F+plasmid+gene+traX+is+required+for+acetylation+of+F+pilin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Escherichia+coli+K-12+F+plasmid+gene+traX+is+required+for+acetylation+of+F+pilin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001891-Maneewannakul2" id="pone.0001891-Maneewannakul2"></a>Maneewannakul K, Maneewannakul S, Ippen-Ihler K (1995) Characterization of traX, the F plasmid locus required for acetylation of F-pilin subunits. J Bacteriol  177: 2957–2964.  <ul class="find" data-citedArticleID="1082888"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+traX%2C+the+F+plasmid+locus+required+for+acetylation+of+F-pilin+subunits.&amp;auth=&amp;atitle=Characterization+of+traX%2C+the+F+plasmid+locus+required+for+acetylation+of+F-pilin+subunits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+traX%2C+the+F+plasmid+locus+required+for+acetylation+of+F-pilin+subunits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+traX%2C+the+F+plasmid+locus+required+for+acetylation+of+F-pilin+subunits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001891-Panicker1" id="pone.0001891-Panicker1"></a>Panicker MM, Minkley EG Jr (1985) DNA transfer occurs during a cell surface contact stage of F sex factor-mediated bacterial conjugation. J Bacteriol  162: 584–590.  <ul class="find" data-citedArticleID="1082902"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+transfer+occurs+during+a+cell+surface+contact+stage+of+F+sex+factor-mediated+bacterial+conjugation.&amp;auth=&amp;atitle=DNA+transfer+occurs+during+a+cell+surface+contact+stage+of+F+sex+factor-mediated+bacterial+conjugation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+transfer+occurs+during+a+cell+surface+contact+stage+of+F+sex+factor-mediated+bacterial+conjugation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+transfer+occurs+during+a+cell+surface+contact+stage+of+F+sex+factor-mediated+bacterial+conjugation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001891-Chandler1" id="pone.0001891-Chandler1"></a>Chandler M, Galas DJ (1983) Cointegrate formation mediated by Tn9. II. Activity of IS1 is modulated by external DNA sequences. J Mol Biol  170: 61–91.  <ul class="find" data-citedArticleID="1082816" data-doi="10.1016/s0022-2836(83)80227-7"><li><a href="http://dx.doi.org/10.1016/s0022-2836(83)80227-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cointegrate+formation+mediated+by+Tn9.+II.+Activity+of+IS1+is+modulated+by+external+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cointegrate+formation+mediated+by+Tn9.+II.+Activity+of+IS1+is+modulated+by+external+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001891-Storm1" id="pone.0001891-Storm1"></a>Storm DR, Rosenthal KS, Swanson PE (1977) Polymyxin and related peptide antibiotics. Annu Rev Biochem  46: 723–763.  <ul class="find" data-citedArticleID="1082916" data-doi="10.1146/annurev.bi.46.070177.003451"><li><a href="http://dx.doi.org/10.1146/annurev.bi.46.070177.003451" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymyxin+and+related+peptide+antibiotics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymyxin+and+related+peptide+antibiotics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001891-Yeaman1" id="pone.0001891-Yeaman1"></a>Yeaman MR, Yount NY (2003) Mechanisms of antimicrobial peptide action and resistance. Pharmacol Rev  55: 27–55.  <ul class="find" data-citedArticleID="1082936" data-doi="10.1124/pr.55.1.2"><li><a href="http://dx.doi.org/10.1124/pr.55.1.2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+antimicrobial+peptide+action+and+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+antimicrobial+peptide+action+and+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001891-Datsenko1" id="pone.0001891-Datsenko1"></a>Datsenko KA, Wanner BL (2000) One-step inactivation of chromosomal genes in <em>Escherichia coli</em> K-12 using PCR products. Proc Natl Acad Sci U S A  97: 6640–6645.  <ul class="find" data-citedArticleID="1082828" data-doi="10.1073/pnas.120163297"><li><a href="http://dx.doi.org/10.1073/pnas.120163297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22One-step+inactivation+of+chromosomal+genes+in+Escherichia+coli+K-12+using+PCR+products.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001891-Heydorn1" id="pone.0001891-Heydorn1"></a>Heydorn A, Ersboll B, Kato J, Hentzer M, Parsek MR, et al.  (2002) Statistical analysis of Pseudomonas aeruginosa biofilm development: impact of mutations in genes involved in twitching motility, cell-to-cell signaling, and stationary-phase sigma factor expression. Appl Environ Microbiol  68: 2008–2017.  <ul class="find" data-citedArticleID="1082856" data-doi="10.1128/aem.68.4.2008-2017.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.4.2008-2017.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+analysis+of+Pseudomonas+aeruginosa+biofilm+development%3A+impact+of+mutations+in+genes+involved+in+twitching+motility%2C+cell-to-cell+signaling%2C+and+stationary-phase+sigma+factor+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+analysis+of+Pseudomonas+aeruginosa+biofilm+development%3A+impact+of+mutations+in+genes+involved+in+twitching+motility%2C+cell-to-cell+signaling%2C+and+stationary-phase+sigma+factor+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001891-Sternberg1" id="pone.0001891-Sternberg1"></a>Sternberg C, Christensen BB, Johansen T, Toftgaard Nielsen A, Andersen JB, et al.  (1999) Distribution of bacterial growth activity in flow-chamber biofilms. Appl Environ Microbiol  65: 4108–4117.  <ul class="find" data-citedArticleID="1082914"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Distribution+of+bacterial+growth+activity+in+flow-chamber+biofilms.&amp;auth=&amp;atitle=Distribution+of+bacterial+growth+activity+in+flow-chamber+biofilms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+bacterial+growth+activity+in+flow-chamber+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+bacterial+growth+activity+in+flow-chamber+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001891-Trent1" id="pone.0001891-Trent1"></a>Trent MS, Ribeiro AA, Doerrler WT, Lin S, Cotter RJ, et al.  (2001) Accumulation of a polyisoprene-linked amino sugar in polymyxin-resistant Salmonella typhimurium and Escherichia coli: structural characterization and transfer to lipid A in the periplasm. J Biol Chem  276: 43132–43144.  Epub 42001 Sep 43104.  <ul class="find" data-citedArticleID="1082918" data-doi="10.1074/jbc.m106962200"><li><a href="http://dx.doi.org/10.1074/jbc.m106962200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accumulation+of+a+polyisoprene-linked+amino+sugar+in+polymyxin-resistant+Salmonella+typhimurium+and+Escherichia+coli%3A+structural+characterization+and+transfer+to+lipid+A+in+the+periplasm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accumulation+of+a+polyisoprene-linked+amino+sugar+in+polymyxin-resistant+Salmonella+typhimurium+and+Escherichia+coli%3A+structural+characterization+and+transfer+to+lipid+A+in+the+periplasm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001891-Hagiwara1" id="pone.0001891-Hagiwara1"></a>Hagiwara D, Yamashino T, Mizuno T (2004) A Genome-Wide View of the Escherichia coli BasS-BasR Two-component System Implicated in Iron-responses. Biosci Biotechnol Biochem  68: 1758–1767.  <ul class="find" data-citedArticleID="1082852" data-doi="10.1271/bbb.68.1758"><li><a href="http://dx.doi.org/10.1271/bbb.68.1758" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Genome-Wide+View+of+the+Escherichia+coli+BasS-BasR+Two-component+System+Implicated+in+Iron-responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Genome-Wide+View+of+the+Escherichia+coli+BasS-BasR+Two-component+System+Implicated+in+Iron-responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001891-Winfield1" id="pone.0001891-Winfield1"></a>Winfield MD, Groisman EA (2004) Phenotypic differences between Salmonella and Escherichia coli resulting from the disparate regulation of homologous genes. Proc Natl Acad Sci U S A  101: 17162–17167.  Epub.  <ul class="find" data-citedArticleID="1082930" data-doi="10.1073/pnas.0406038101"><li><a href="http://dx.doi.org/10.1073/pnas.0406038101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+differences+between+Salmonella+and+Escherichia+coli+resulting+from+the+disparate+regulation+of+homologous+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+differences+between+Salmonella+and+Escherichia+coli+resulting+from+the+disparate+regulation+of+homologous+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001891-Nickel1" id="pone.0001891-Nickel1"></a>Nickel JC, Ruseska I, Wright JB, Costerton JW (1985) Tobramycin resistance of Pseudomonas aeruginosa cells growing as a biofilm on urinary catheter material. Antimicrob Agents Chemother  27: 619–624.  <ul class="find" data-citedArticleID="1082900" data-doi="10.1128/aac.27.4.619"><li><a href="http://dx.doi.org/10.1128/aac.27.4.619" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tobramycin+resistance+of+Pseudomonas+aeruginosa+cells+growing+as+a+biofilm+on+urinary+catheter+material." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tobramycin+resistance+of+Pseudomonas+aeruginosa+cells+growing+as+a+biofilm+on+urinary+catheter+material.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001891-Ceri1" id="pone.0001891-Ceri1"></a>Ceri H, Olson ME, Stremick C, Read RR, Morck D, et al.  (1999) The Calgary Biofilm Device: new technology for rapid determination of antibiotic susceptibilities of bacterial biofilms. J Clin Microbiol  37: 1771–1776.  <ul class="find" data-citedArticleID="1082814"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Calgary+Biofilm+Device%3A+new+technology+for+rapid+determination+of+antibiotic+susceptibilities+of+bacterial+biofilms.&amp;auth=&amp;atitle=The+Calgary+Biofilm+Device%3A+new+technology+for+rapid+determination+of+antibiotic+susceptibilities+of+bacterial+biofilms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Calgary+Biofilm+Device%3A+new+technology+for+rapid+determination+of+antibiotic+susceptibilities+of+bacterial+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Calgary+Biofilm+Device%3A+new+technology+for+rapid+determination+of+antibiotic+susceptibilities+of+bacterial+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001891-Lewis1" id="pone.0001891-Lewis1"></a>Lewis K (2001) Riddle of biofilm resistance. Antimicrob Agents Chemother  45: 999–1007.  <ul class="find" data-citedArticleID="1082876" data-doi="10.1128/aac.45.4.999-1007.2001"><li><a href="http://dx.doi.org/10.1128/aac.45.4.999-1007.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Riddle+of+biofilm+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Riddle+of+biofilm+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001891-Gilbert1" id="pone.0001891-Gilbert1"></a>Gilbert P, Das J, Foley I (1997) Biofilm susceptibility to antimicrobials. Adv Dent Res  11: 160–167.  <ul class="find" data-citedArticleID="1082846" data-doi="10.1177/08959374970110010701"><li><a href="http://dx.doi.org/10.1177/08959374970110010701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biofilm+susceptibility+to+antimicrobials." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biofilm+susceptibility+to+antimicrobials.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001891-Mah1" id="pone.0001891-Mah1"></a>Mah TF, Pitts B, Pellock B, Walker GC, Stewart PS, et al.  (2003) A genetic basis for Pseudomonas aeruginosa biofilm antibiotic resistance. Nature  426: 306–310.  <ul class="find" data-citedArticleID="1082882" data-doi="10.1038/nature02122"><li><a href="http://dx.doi.org/10.1038/nature02122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genetic+basis+for+Pseudomonas+aeruginosa+biofilm+antibiotic+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genetic+basis+for+Pseudomonas+aeruginosa+biofilm+antibiotic+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001891-Drenkard2" id="pone.0001891-Drenkard2"></a>Drenkard E, Ausubel FM (2002) Pseudomonas biofilm formation and antibiotic resistance are linked to phenotypic variation. Nature  416: 740–743.  <ul class="find" data-citedArticleID="1082836" data-doi="10.1038/416740a"><li><a href="http://dx.doi.org/10.1038/416740a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudomonas+biofilm+formation+and+antibiotic+resistance+are+linked+to+phenotypic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudomonas+biofilm+formation+and+antibiotic+resistance+are+linked+to+phenotypic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001891-Larsen1" id="pone.0001891-Larsen1"></a>Larsen T, Fiehn NE (1996) Resistance of Streptococcus sanguis biofilms to antimicrobial agents. Apmis  104: 280–284.  <ul class="find" data-citedArticleID="1082874" data-doi="10.1111/j.1699-0463.1996.tb00718.x"><li><a href="http://dx.doi.org/10.1111/j.1699-0463.1996.tb00718.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+of+Streptococcus+sanguis+biofilms+to+antimicrobial+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+of+Streptococcus+sanguis+biofilms+to+antimicrobial+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001891-Vorachit1" id="pone.0001891-Vorachit1"></a>Vorachit M, Lam K, Jayanetra P, Costerton JW (1993) Resistance of Pseudomonas pseudomallei growing as a biofilm on silastic discs to ceftazidime and co-trimoxazole. Antimicrob Agents Chemother  37: 2000–2002.  <ul class="find" data-citedArticleID="1082924" data-doi="10.1128/aac.37.9.2000"><li><a href="http://dx.doi.org/10.1128/aac.37.9.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+of+Pseudomonas+pseudomallei+growing+as+a+biofilm+on+silastic+discs+to+ceftazidime+and+co-trimoxazole." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+of+Pseudomonas+pseudomallei+growing+as+a+biofilm+on+silastic+discs+to+ceftazidime+and+co-trimoxazole.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001891-Williams1" id="pone.0001891-Williams1"></a>Williams I, Venables WA, Lloyd D, Paul F, Critchley I (1997) The effects of adherence to silicone surfaces on antibiotic susceptibility in Staphylococcus aureus. Microbiology  143: 2407–2413.  <ul class="find" data-citedArticleID="1082926" data-doi="10.1099/00221287-143-7-2407"><li><a href="http://dx.doi.org/10.1099/00221287-143-7-2407" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+adherence+to+silicone+surfaces+on+antibiotic+susceptibility+in+Staphylococcus+aureus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+adherence+to+silicone+surfaces+on+antibiotic+susceptibility+in+Staphylococcus+aureus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001891-Amorena1" id="pone.0001891-Amorena1"></a>Amorena B, Gracia E, Monzon M, Leiva J, Oteiza C, et al.  (1999) Antibiotic susceptibility assay for Staphylococcus aureus in biofilms developed in vitro. J Antimicrob Chemother  44: 43–55.  <ul class="find" data-citedArticleID="1082804" data-doi="10.1093/jac/44.1.43"><li><a href="http://dx.doi.org/10.1093/jac/44.1.43" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antibiotic+susceptibility+assay+for+Staphylococcus+aureus+in+biofilms+developed+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antibiotic+susceptibility+assay+for+Staphylococcus+aureus+in+biofilms+developed+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001891-Anwar1" id="pone.0001891-Anwar1"></a>Anwar H, Strap JL, Chen K, Costerton JW (1992) Dynamic interactions of biofilms of mucoid Pseudomonas aeruginosa with tobramycin and piperacillin. Antimicrob Agents Chemother  36: 1208–1214.  <ul class="find" data-citedArticleID="1082808" data-doi="10.1128/aac.36.6.1208"><li><a href="http://dx.doi.org/10.1128/aac.36.6.1208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+interactions+of+biofilms+of+mucoid+Pseudomonas+aeruginosa+with+tobramycin+and+piperacillin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+interactions+of+biofilms+of+mucoid+Pseudomonas+aeruginosa+with+tobramycin+and+piperacillin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001891-Desai1" id="pone.0001891-Desai1"></a>Desai M, Buhler T, Weller PH, Brown MR (1998) Increasing resistance of planktonic and biofilm cultures of Burkholderia cepacia to ciprofloxacin and ceftazidime during exponential growth. J Antimicrob Chemother  42: 153–160.  <ul class="find" data-citedArticleID="1082830" data-doi="10.1093/jac/42.2.153"><li><a href="http://dx.doi.org/10.1093/jac/42.2.153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Increasing+resistance+of+planktonic+and+biofilm+cultures+of+Burkholderia+cepacia+to+ciprofloxacin+and+ceftazidime+during+exponential+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Increasing+resistance+of+planktonic+and+biofilm+cultures+of+Burkholderia+cepacia+to+ciprofloxacin+and+ceftazidime+during+exponential+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001891-Gristina1" id="pone.0001891-Gristina1"></a>Gristina AG, Jennings RA, Naylor PT, Myrvik QN, Webb LX (1989) Comparative in vitro antibiotic resistance of surface-colonizing coagulase-negative staphylococci. Antimicrob Agents Chemother  33: 813–816.  <ul class="find" data-citedArticleID="1082848" data-doi="10.1128/aac.33.6.813"><li><a href="http://dx.doi.org/10.1128/aac.33.6.813" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+in+vitro+antibiotic+resistance+of+surface-colonizing+coagulase-negative+staphylococci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+in+vitro+antibiotic+resistance+of+surface-colonizing+coagulase-negative+staphylococci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001891-Hill1" id="pone.0001891-Hill1"></a>Hill D, Rose B, Pajkos A, Robinson M, Bye P, et al.  (2005) Antibiotic susceptabilities of Pseudomonas aeruginosa isolates derived from patients with cystic fibrosis under aerobic, anaerobic, and biofilm conditions. J Clin Microbiol  43: 5085–5090.  <ul class="find" data-citedArticleID="1082860" data-doi="10.1128/jcm.43.10.5085-5090.2005"><li><a href="http://dx.doi.org/10.1128/jcm.43.10.5085-5090.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antibiotic+susceptabilities+of+Pseudomonas+aeruginosa+isolates+derived+from+patients+with+cystic+fibrosis+under+aerobic%2C+anaerobic%2C+and+biofilm+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antibiotic+susceptabilities+of+Pseudomonas+aeruginosa+isolates+derived+from+patients+with+cystic+fibrosis+under+aerobic%2C+anaerobic%2C+and+biofilm+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001891-Kumon1" id="pone.0001891-Kumon1"></a>Kumon H, Ono N, Iida M, Nickel JC (1995) Combination effect of fosfomycin and ofloxacin against Pseudomonas aeruginosa growing in a biofilm. Antimicrob Agents Chemother  39: 1038–1044.  <ul class="find" data-citedArticleID="1082872" data-doi="10.1128/aac.39.5.1038"><li><a href="http://dx.doi.org/10.1128/aac.39.5.1038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combination+effect+of+fosfomycin+and+ofloxacin+against+Pseudomonas+aeruginosa+growing+in+a+biofilm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combination+effect+of+fosfomycin+and+ofloxacin+against+Pseudomonas+aeruginosa+growing+in+a+biofilm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001891-Spoering1" id="pone.0001891-Spoering1"></a>Spoering AL, Lewis K (2001) Biofilms and planktonic cells of Pseudomonas aeruginosa have similar resistance to killing by antimicrobials. J Bacteriol  183: 6746–6751.  <ul class="find" data-citedArticleID="1082912" data-doi="10.1128/jb.183.23.6746-6751.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.23.6746-6751.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biofilms+and+planktonic+cells+of+Pseudomonas+aeruginosa+have+similar+resistance+to+killing+by+antimicrobials." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biofilms+and+planktonic+cells+of+Pseudomonas+aeruginosa+have+similar+resistance+to+killing+by+antimicrobials.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001891-Dame1" id="pone.0001891-Dame1"></a>Dame JB, Shapiro BM (1979) Lipid and lipopolysaccharide composition of Escherichia coli surface-altered mutants selected for resistance to levallorphan, tetracaine, and polymyxin. J Bacteriol  137: 1043–1047.  <ul class="find" data-citedArticleID="1082826"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lipid+and+lipopolysaccharide+composition+of+Escherichia+coli+surface-altered+mutants+selected+for+resistance+to+levallorphan%2C+tetracaine%2C+and+polymyxin.&amp;auth=&amp;atitle=Lipid+and+lipopolysaccharide+composition+of+Escherichia+coli+surface-altered+mutants+selected+for+resistance+to+levallorphan%2C+tetracaine%2C+and+polymyxin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lipid+and+lipopolysaccharide+composition+of+Escherichia+coli+surface-altered+mutants+selected+for+resistance+to+levallorphan%2C+tetracaine%2C+and+polymyxin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lipid+and+lipopolysaccharide+composition+of+Escherichia+coli+surface-altered+mutants+selected+for+resistance+to+levallorphan%2C+tetracaine%2C+and+polymyxin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001891-Haagensen1" id="pone.0001891-Haagensen1"></a>Haagensen JA, Klausen M, Ernst RK, Miller SI, Folkesson A, et al.  (2007) Differentiation and distribution of colistin- and sodium dodecyl sulfate-tolerant cells in Pseudomonas aeruginosa biofilms. J Bacteriol  189: 28–37.  <ul class="find" data-citedArticleID="1082850" data-doi="10.1128/jb.00720-06"><li><a href="http://dx.doi.org/10.1128/jb.00720-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differentiation+and+distribution+of+colistin-+and+sodium+dodecyl+sulfate-tolerant+cells+in+Pseudomonas+aeruginosa+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differentiation+and+distribution+of+colistin-+and+sodium+dodecyl+sulfate-tolerant+cells+in+Pseudomonas+aeruginosa+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001891-Ausubel1" id="pone.0001891-Ausubel1"></a>Ausubel FM, Brent MR, Kingston RE, Moore DD, Siedman JG, et al.  (1996) Current Protocols in Molecular Biology. New York, NY: John Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">54.
              </span><a name="pone.0001891-Bertani1" id="pone.0001891-Bertani1"></a>Bertani G (1951) Studies on lysogenesis. I. The mode of phage liberation by lysogenic Escherichia coli. J Bacteriol  62: 293–300.  <ul class="find" data-citedArticleID="1082812"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Studies+on+lysogenesis.+I.+The+mode+of+phage+liberation+by+lysogenic+Escherichia+coli.&amp;auth=&amp;atitle=Studies+on+lysogenesis.+I.+The+mode+of+phage+liberation+by+lysogenic+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Studies+on+lysogenesis.+I.+The+mode+of+phage+liberation+by+lysogenic+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Studies+on+lysogenesis.+I.+The+mode+of+phage+liberation+by+lysogenic+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001891-Kim1" id="pone.0001891-Kim1"></a>Kim W, Surette MG (2003) Swarming populations of Salmonella represent a unique physiological state coupled to multiple mechanisms of antibiotic resistance. Biol Proced Online  5: 189–196.  Epub 2003 Sep 2026.  <ul class="find" data-citedArticleID="1082866" data-doi="10.1251/bpo61"><li><a href="http://dx.doi.org/10.1251/bpo61" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Swarming+populations+of+Salmonella+represent+a+unique+physiological+state+coupled+to+multiple+mechanisms+of+antibiotic+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Swarming+populations+of+Salmonella+represent+a+unique+physiological+state+coupled+to+multiple+mechanisms+of+antibiotic+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001891-Kim2" id="pone.0001891-Kim2"></a>Kim W, Killam T, Sood V, Surette MG (2003) Swarm-cell differentiation in Salmonella enterica serovar typhimurium results in elevated resistance to multiple antibiotics. J Bacteriol  185: 3111–3117.  <ul class="find" data-citedArticleID="1082868" data-doi="10.1128/jb.185.10.3111-3117.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.10.3111-3117.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Swarm-cell+differentiation+in+Salmonella+enterica+serovar+typhimurium+results+in+elevated+resistance+to+multiple+antibiotics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Swarm-cell+differentiation+in+Salmonella+enterica+serovar+typhimurium+results+in+elevated+resistance+to+multiple+antibiotics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001891-Miller2" id="pone.0001891-Miller2"></a>Miller JH (1992) A short course in bacterial genetics: a laboratory manual and handbook for Escherichia coli and related bacteria. Cold Spring Harbor, N.Y: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">58.
              </span><a name="pone.0001891-Maloy1" id="pone.0001891-Maloy1"></a>Maloy SR, Stewart VJ, Taylor RK (1996) Genetic analysis of pathogenic bacteria. A laboratory manual. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0001891-Diederich1" id="pone.0001891-Diederich1"></a>Diederich L, Rasmussen LJ, Messer W (1992) New cloning vectors for integration in the lambda attachment site attB of the Escherichia coli chromosome. Plasmid  28: 14–24.  <ul class="find" data-citedArticleID="1082832" data-doi="10.1016/0147-619x(92)90032-6"><li><a href="http://dx.doi.org/10.1016/0147-619x(92)90032-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+cloning+vectors+for+integration+in+the+lambda+attachment+site+attB+of+the+Escherichia+coli+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+cloning+vectors+for+integration+in+the+lambda+attachment+site+attB+of+the+Escherichia+coli+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001891-Reisner3" id="pone.0001891-Reisner3"></a>Reisner A, Krogfelt KA, Klein BM, Zechner EL, Molin S (2006) In vitro biofilm formation of commensal and pathogenic Escherichia coli strains: impact of environmental and genetic factors. J Bacteriol  188: 3572–3581.  <ul class="find" data-citedArticleID="1082908" data-doi="10.1128/jb.188.10.3572-3581.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.10.3572-3581.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+biofilm+formation+of+commensal+and+pathogenic+Escherichia+coli+strains%3A+impact+of+environmental+and+genetic+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+biofilm+formation+of+commensal+and+pathogenic+Escherichia+coli+strains%3A+impact+of+environmental+and+genetic+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001891-Clark1" id="pone.0001891-Clark1"></a>Clark DJ, Maaløe Z (1967) DNA replication and the division cycle in Escherichia coli. J Mol Biol  23: 99–112.  <ul class="find" data-citedArticleID="1082820" data-doi="10.1016/s0022-2836(67)80070-6"><li><a href="http://dx.doi.org/10.1016/s0022-2836(67)80070-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+replication+and+the+division+cycle+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+replication+and+the+division+cycle+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001891-Heydorn2" id="pone.0001891-Heydorn2"></a>Heydorn A, Nielsen AT, Hentzer M, Sternberg C, Givskov M, et al.  (2000) Quantification of biofilm structures by the novel computer program COMSTAT. Microbiology  146: 2395–2407.  <ul class="find" data-citedArticleID="1082858"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantification+of+biofilm+structures+by+the+novel+computer+program+COMSTAT.&amp;auth=&amp;atitle=Quantification+of+biofilm+structures+by+the+novel+computer+program+COMSTAT." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantification+of+biofilm+structures+by+the+novel+computer+program+COMSTAT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantification+of+biofilm+structures+by+the+novel+computer+program+COMSTAT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001891-Vidal1" id="pone.0001891-Vidal1"></a>Vidal O, Longin R, Prigent-Combaret C, Dorel C, Hooreman M, et al.  (1998) Isolation of an Escherichia coli K-12 mutant strain able to form biofilms on inert surfaces: involvement of a new ompR allele that increases curli expression. J Bacteriol  180: 2442–2449.  <ul class="find" data-citedArticleID="1082922"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Isolation+of+an+Escherichia+coli+K-12+mutant+strain+able+to+form+biofilms+on+inert+surfaces%3A+involvement+of+a+new+ompR+allele+that+increases+curli+expression.&amp;auth=&amp;atitle=Isolation+of+an+Escherichia+coli+K-12+mutant+strain+able+to+form+biofilms+on+inert+surfaces%3A+involvement+of+a+new+ompR+allele+that+increases+curli+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+an+Escherichia+coli+K-12+mutant+strain+able+to+form+biofilms+on+inert+surfaces%3A+involvement+of+a+new+ompR+allele+that+increases+curli+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+an+Escherichia+coli+K-12+mutant+strain+able+to+form+biofilms+on+inert+surfaces%3A+involvement+of+a+new+ompR+allele+that+increases+curli+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001891-Maneewannakul3" id="pone.0001891-Maneewannakul3"></a>Maneewannakul K, Kathir P, Endley S, Moore D, Manchak J, et al.  (1996) Construction of derivatives of the F plasmid pOX-tra715: characterization of traY and traD mutants that can be complemented in trans. Mol Microbiol  22: 197–205.  <ul class="find" data-citedArticleID="1082890" data-doi="10.1046/j.1365-2958.1996.00087.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1996.00087.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+derivatives+of+the+F+plasmid+pOX-tra715%3A+characterization+of+traY+and+traD+mutants+that+can+be+complemented+in+trans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+derivatives+of+the+F+plasmid+pOX-tra715%3A+characterization+of+traY+and+traD+mutants+that+can+be+complemented+in+trans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001891-Kathir1" id="pone.0001891-Kathir1"></a>Kathir P, Ippen-Ihler K (1991) Construction and characterization of derivatives carrying insertion mutations in F plasmid transfer region genes, trbA, artA, traQ, and trbB. Plasmid  26: 40–54.  <ul class="find" data-citedArticleID="1082864" data-doi="10.1016/0147-619x(91)90035-u"><li><a href="http://dx.doi.org/10.1016/0147-619x(91)90035-u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+and+characterization+of+derivatives+carrying+insertion+mutations+in+F+plasmid+transfer+region+genes%2C+trbA%2C+artA%2C+traQ%2C+and+trbB." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+and+characterization+of+derivatives+carrying+insertion+mutations+in+F+plasmid+transfer+region+genes%2C+trbA%2C+artA%2C+traQ%2C+and+trbB.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001891-Schroder1" id="pone.0001891-Schroder1"></a>Schroder G, Krause S, Zechner EL, Traxler B, Yeo HJ, et al.  (2002) TraG-like proteins of DNA transfer systems and of the Helicobacter pylori type IV secretion system: inner membrane gate for exported substrates? J Bacteriol  184: 2767–2779.  <ul class="find" data-citedArticleID="1082910" data-doi="10.1128/jb.184.10.2767-2779.2002"><li><a href="http://dx.doi.org/10.1128/jb.184.10.2767-2779.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TraG-like+proteins+of+DNA+transfer+systems+and+of+the+Helicobacter+pylori+type+IV+secretion+system%3A+inner+membrane+gate+for+exported+substrates%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TraG-like+proteins+of+DNA+transfer+systems+and+of+the+Helicobacter+pylori+type+IV+secretion+system%3A+inner+membrane+gate+for+exported+substrates%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001891-Andersen1" id="pone.0001891-Andersen1"></a>Andersen JB, Sternberg C, Poulsen LK, Bjorn SP, Givskov M, et al.  (1998) New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria. Appl Environ Microbiol  64: 2240–2246.  <ul class="find" data-citedArticleID="1082806"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=New+unstable+variants+of+green+fluorescent+protein+for+studies+of+transient+gene+expression+in+bacteria.&amp;auth=&amp;atitle=New+unstable+variants+of+green+fluorescent+protein+for+studies+of+transient+gene+expression+in+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+unstable+variants+of+green+fluorescent+protein+for+studies+of+transient+gene+expression+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+unstable+variants+of+green+fluorescent+protein+for+studies+of+transient+gene+expression+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.XML" value="137491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.PDF" value="480070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g001.PNG_L" value="410633"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g001.PNG_M" value="57544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g001.PNG_S" value="10425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g001.TIF" value="741352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g001.PNG_I" value="20062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g002.PNG_L" value="532321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g002.PNG_M" value="29551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g002.PNG_S" value="7027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g002.TIF" value="767780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g002.PNG_I" value="10532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t001.PNG_L" value="109763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t001.PNG_M" value="152742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t001.PNG_S" value="17461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t001.TIF" value="456460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t001.PNG_I" value="49606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g003.PNG_L" value="2177134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g003.PNG_M" value="272723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g003.PNG_S" value="14605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g003.TIF" value="3135706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g003.PNG_I" value="86398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g004.PNG_L" value="2828954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g004.PNG_M" value="435879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g004.PNG_S" value="24915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g004.TIF" value="3828442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g004.PNG_I" value="280769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g005.PNG_L" value="4492039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g005.PNG_M" value="185728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g005.PNG_S" value="11979"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g005.TIF" value="6555526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.g005.PNG_I" value="60980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t002.PNG_L" value="187579"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t002.PNG_M" value="167612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t002.PNG_S" value="21293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t002.TIF" value="907334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.t002.PNG_I" value="115837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001891.s001.MPG" value="7650740"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001891&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001891" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001891&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001891&volume=&issue=&title=Biofilm Induced Tolerance towards Antimicrobial Peptides&author_name=Anders%20Folkesson%2C%20Janus%20A.%20J.%20Haagensen%2C%20Claudia%20Zampaloni%2C%20Claus%20Sternberg%2C%20S%C3%B8ren%20Molin&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001891" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891&amp;t=Biofilm%20Induced%20Tolerance%20towards%20Antimicrobial%20Peptides" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891&title=Biofilm%20Induced%20Tolerance%20towards%20Antimicrobial%20Peptides&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891&amp;title=Biofilm%20Induced%20Tolerance%20towards%20Antimicrobial%20Peptides" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001891&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Biofilm Induced Tolerance towards Antimicrobial Peptides';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001891';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Biofilm%20Induced%20Tolerance%20towards%20Antimicrobial%20Peptides http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001891" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001891" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antibiotics%22" title="Search for articles in the subject area:'Antibiotics'"><div class="flagText">Antibiotics</div></a>
              <div data-categoryid="42767" data-articleid="26606"
                   data-categoryname="Antibiotics"
                   class="flagImage" title="Flag 'Antibiotics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Antimicrobials%22" title="Search for articles in the subject area:'Antimicrobials'"><div class="flagText">Antimicrobials</div></a>
              <div data-categoryid="42703" data-articleid="26606"
                   data-categoryname="Antimicrobials"
                   class="flagImage" title="Flag 'Antimicrobials' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+biofilms%22" title="Search for articles in the subject area:'Bacterial biofilms'"><div class="flagText">Bacterial biofilms</div></a>
              <div data-categoryid="47421" data-articleid="26606"
                   data-categoryname="Bacterial biofilms"
                   class="flagImage" title="Flag 'Bacterial biofilms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biofilms%22" title="Search for articles in the subject area:'Biofilms'"><div class="flagText">Biofilms</div></a>
              <div data-categoryid="47823" data-articleid="26606"
                   data-categoryname="Biofilms"
                   class="flagImage" title="Flag 'Biofilms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Confocal+laser+microscopy%22" title="Search for articles in the subject area:'Confocal laser microscopy'"><div class="flagText">Confocal laser microscopy</div></a>
              <div data-categoryid="17403" data-articleid="26606"
                   data-categoryname="Confocal laser microscopy"
                   class="flagImage" title="Flag 'Confocal laser microscopy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Operons%22" title="Search for articles in the subject area:'Operons'"><div class="flagText">Operons</div></a>
              <div data-categoryid="21291" data-articleid="26606"
                   data-categoryname="Operons"
                   class="flagImage" title="Flag 'Operons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pili+and+fimbriae%22" title="Search for articles in the subject area:'Pili and fimbriae'"><div class="flagText">Pili and fimbriae</div></a>
              <div data-categoryid="34911" data-articleid="26606"
                   data-categoryname="Pili and fimbriae"
                   class="flagImage" title="Flag 'Pili and fimbriae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="26606"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9045'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3765'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7945&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6637">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
