

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000743"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000743" />

    <meta name="citation_title" content="Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis"/>
    <meta itemprop="name" content="Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis"/>

      <meta name="citation_author" content="Garret Suen"/>
            <meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York, United States of America"/>
      <meta name="citation_author" content="Barry S. Goldman"/>
            <meta name="citation_author_institution" content="Monsanto, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Roy D. Welch"/>
            <meta name="citation_author_institution" content="Department of Biology, Syracuse University, Syracuse, New York, United States of America"/>

    <meta name="citation_date" content="2007/8/15"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000743.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e743"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Metagenomics: genomic analysis of microbial communities.; citation_author=CS Riesenfeld; citation_author=PD Schloss; citation_author=J Handelsman; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=1; citation_pages=525-552; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bioinformatics for whole-genome shotgun sequencing of microbial communities.; citation_author=K Chen; citation_author=L Pachter; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=2; citation_pages=106-112; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Concepts of bacterial biodiversity for the age of genomics.; citation_author=FM Cohan; citation_number=3; citation_date=2004; citation_publisher=Humana Press; " />
      <meta name="citation_reference" content="citation_title=Biology's next revolution.; citation_author=N Goldenfeld; citation_author=C Woese; citation_journal_title=Nature; citation_volume=445; citation_number=4; citation_pages=369; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ecological perspectives on the sequenced genome collection.; citation_author=JB Martiny; citation_author=D Field; citation_journal_title=Ecology Letters; citation_volume=8; citation_number=5; citation_pages=1334-1345; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The niche-relationships of the California thrasher.; citation_author=J Grinell; citation_journal_title=Auk; citation_volume=34; citation_number=6; citation_pages=427-433; citation_date=1917; " />
      <meta name="citation_reference" content="citation_title=Animal ecology.; citation_author=C Elton; citation_number=7; citation_date=1927; citation_publisher=Sidgwick&Jackson; " />
      <meta name="citation_reference" content="citation_title=System Theory of the Ecological Niche.; citation_author=BC Patten; citation_author=GT Auble; citation_journal_title=The American Naturalist; citation_volume=117; citation_number=8; citation_pages=893-922; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Microbial biogeography: putting microorganisms on the map.; citation_author=JB Martiny; citation_author=BJ Bohannan; citation_author=JH Brown; citation_author=RK Colwell; citation_author=JA Fuhrman; citation_journal_title=Nat Rev Microbiol; citation_volume=4; citation_number=9; citation_pages=102-112; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bergey's Manual of Systematic Bacteriology.; citation_author=GM Garrity; citation_number=10; citation_date=2001; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Bacterial evolution.; citation_author=CR Woese; citation_journal_title=Microbiol Rev; citation_volume=51; citation_number=11; citation_pages=221-271; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya.; citation_author=CR Woese; citation_author=O Kandler; citation_author=ML Wheelis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=87; citation_number=12; citation_pages=4576-4579; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Towards a genome-based taxonomy for prokaryotes.; citation_author=KT Konstantinidis; citation_author=JM Tiedje; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=13; citation_pages=6258-6264; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Toward automatic reconstruction of a highly resolved tree of life.; citation_author=FD Ciccarelli; citation_author=T Doerks; citation_author=C von Mering; citation_author=CJ Creevey; citation_author=B Snel; citation_journal_title=Science; citation_volume=311; citation_number=14; citation_pages=1283-1287; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome phylogeny based on gene content.; citation_author=B Snel; citation_author=P Bork; citation_author=MA Huynen; citation_journal_title=Nat Genet; citation_volume=21; citation_number=15; citation_pages=108-110; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Whole-genome trees based on the occurrence of folds and orthologs: implications for comparing genomes on different levels.; citation_author=J Lin; citation_author=M Gerstein; citation_journal_title=Genome Res; citation_volume=10; citation_number=16; citation_pages=808-818; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phylogeny determined by protein domain content.; citation_author=S Yang; citation_author=RF Doolittle; citation_author=PE Bourne; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=17; citation_pages=373-378; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome trees from conservation profiles.; citation_author=F Tekaia; citation_author=E Yeramian; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=18; citation_pages=e75; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Critical issues in bacterial phylogeny.; citation_author=RS Gupta; citation_author=E Griffiths; citation_journal_title=Theor Popul Biol; citation_volume=61; citation_number=19; citation_pages=423-434; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Examining bacterial species under the specter of gene transfer and exchange.; citation_author=H Ochman; citation_author=E Lerat; citation_author=V Daubin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=20; citation_pages=6595-6599; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prokaryotic evolution in light of gene transfer.; citation_author=JP Gogarten; citation_author=WF Doolittle; citation_author=JG Lawrence; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=21; citation_pages=2226-2238; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Lateral gene transfer and the nature of bacterial innovation.; citation_author=H Ochman; citation_author=JG Lawrence; citation_author=EA Groisman; citation_journal_title=Nature; citation_volume=405; citation_number=22; citation_pages=299-304; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Lateral gene transfer and the origins of prokaryotic groups.; citation_author=Y Boucher; citation_author=CJ Douady; citation_author=RT Papke; citation_author=DA Walsh; citation_author=ME Boudreau; citation_journal_title=Annu Rev Genet; citation_volume=37; citation_number=23; citation_pages=283-328; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Duplication and divergence: the evolution of new genes and old ideas.; citation_author=JS Taylor; citation_author=J Raes; citation_journal_title=Annu Rev Genet; citation_volume=38; citation_number=24; citation_pages=615-643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution by Gene Duplication.; citation_author=S Ohno; citation_number=25; citation_date=1970; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Microbial minimalism: genome reduction in bacterial pathogens.; citation_author=NA Moran; citation_journal_title=Cell; citation_volume=108; citation_number=26; citation_pages=583-586; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genes lost and genes found: evolution of bacterial pathogenesis and symbiosis.; citation_author=H Ochman; citation_author=NA Moran; citation_journal_title=Science; citation_volume=292; citation_number=27; citation_pages=1096-1099; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Decoding the genomic tree of life.; citation_author=AB Simonson; citation_author=JA Servin; citation_author=RG Skophammer; citation_author=CW Herbold; citation_author=MC Rivera; citation_journal_title=Proc Natl Acad Sci U S A 102 Suppl; citation_volume=1; citation_number=28; citation_pages=6608-6613; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary origins of genomic repertoires in bacteria.; citation_author=E Lerat; citation_author=V Daubin; citation_author=H Ochman; citation_author=NA Moran; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=29; citation_pages=e130; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein families and their evolution-a structural perspective.; citation_author=CA Orengo; citation_author=JM Thornton; citation_journal_title=Annu Rev Biochem; citation_volume=74; citation_number=30; citation_pages=867-900; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Domain combinations in archaeal, eubacterial and eukaryotic proteomes.; citation_author=G Apic; citation_author=J Gough; citation_author=SA Teichmann; citation_journal_title=J Mol Biol; citation_volume=310; citation_number=31; citation_pages=311-325; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolution of the protein repertoire.; citation_author=C Chothia; citation_author=J Gough; citation_author=C Vogel; citation_author=SA Teichmann; citation_journal_title=Science; citation_volume=300; citation_number=32; citation_pages=1701-1703; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cluster stability and the use of noise in interpretation of clustering; 2001.; citation_author=G Davidson; citation_author=B Wylie; citation_author=K Boyack; citation_number=33; citation_pages=23-30; citation_date=2001; citation_publisher=IEEE Computer Society; " />
      <meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services.; citation_author=RD Finn; citation_author=J Mistry; citation_author=B Schuster-Bockler; citation_author=S Griffiths-Jones; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=34; citation_pages=D247-251; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=From gene trees to organismal phylogeny in prokaryotes: the case of the gamma-Proteobacteria.; citation_author=E Lerat; citation_author=V Daubin; citation_author=NA Moran; citation_journal_title=PLoS Biol; citation_volume=1; citation_number=35; citation_pages=E19; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular interactions between bacterial symbionts and their hosts.; citation_author=C Dale; citation_author=NA Moran; citation_journal_title=Cell; citation_volume=126; citation_number=36; citation_pages=453-465; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic sequence of a Lyme disease spirochaete, Borrelia burgdorferi.; citation_author=CM Fraser; citation_author=S Casjens; citation_author=WM Huang; citation_author=GG Sutton; citation_author=R Clayton; citation_journal_title=Nature; citation_volume=390; citation_number=37; citation_pages=580-586; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Proposal of Burkholderia gen. nov. and transfer of seven species of the genus Pseudomonas homology group II to the new genus, with the type species Burkholderia cepacia (Palleroni and Holmes 1981) comb. nov.; citation_author=E Yabuuchi; citation_author=Y Kosako; citation_author=H Oyaizu; citation_author=I Yano; citation_author=H Hotta; citation_journal_title=Microbiol Immunol; citation_volume=36; citation_number=38; citation_pages=1251-1275; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=N-acylhomoserine-lactone-mediated communication between Pseudomonas aeruginosa and Burkholderia cepacia in mixed biofilms.; citation_author=K Riedel; citation_author=M Hentzer; citation_author=O Geisenberger; citation_author=B Huber; citation_author=A Steidle; citation_journal_title=Microbiology; citation_volume=147; citation_number=39; citation_pages=3249-3262; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Microbial ecology of the cystic fibrosis lung.; citation_author=F Harrison; citation_journal_title=Microbiology; citation_volume=153; citation_number=40; citation_pages=917-923; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis of Hyphomonas neptunium contradicts 16S rRNA gene-based phylogenetic analysis: implications for the taxonomy of the orders ‘Rhodobacterales’ and Caulobacterales.; citation_author=JH Badger; citation_author=JA Eisen; citation_author=NL Ward; citation_journal_title=Int J Syst Evol Microbiol; citation_volume=55; citation_number=41; citation_pages=1021-1026; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Guilt-by-association goes global.; citation_author=S Oliver; citation_journal_title=Nature; citation_volume=403; citation_number=42; citation_pages=601-603; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The integrated microbial genomes (IMG) system.; citation_author=VM Markowitz; citation_author=F Korzeniewski; citation_author=K Palaniappan; citation_author=E Szeto; citation_author=G Werner; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=43; citation_pages=D344-348; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The ribosomal database project (RDP-II): introducing myRDP space and quality controlled public data.; citation_author=JR Cole; citation_author=B Chai; citation_author=RJ Farris; citation_author=Q Wang; citation_author=AS Kulam-Syed-Mohideen; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=44; citation_pages=D169-172; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=CDD: a conserved domain database for interactive domain family analysis.; citation_author=A Marchler-Bauer; citation_author=JB Anderson; citation_author=MK Derbyshire; citation_author=C DeWeese-Scott; citation_author=NR Gonzales; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=45; citation_pages=D237-240; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=46; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput.; citation_author=RC Edgar; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=47; citation_pages=1792-1797; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PHYLIP (Phylogeny Inference Package), version 3.6.; citation_author=J Felenstein; citation_number=48; citation_date=2006; citation_publisher=Department of Genetics, university of Washington; " />
      <meta name="citation_reference" content="citation_title=Exploring prokaryotic diversity in the genomic era.; citation_author=P Hugenholtz; citation_journal_title=Genome Biol; citation_volume=3; citation_number=49; citation_pages=REVIEWS0003; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis"/>
    <meta name="twitter:description" content="Automated DNA sequencing technology is so rapid that analysis has become the rate-limiting step. Hundreds of prokaryotic genome sequences are publicly available, with new genomes uploaded at the rate of approximately 20 per month. As a result, this growing body of genome sequences will include microorganisms not previously identified, isolated, or observed. We hypothesize that evolutionary pressure exerted by an ecological niche selects for a similar genetic repertoire in those prokaryotes that occupy the same niche, and that this is due to both vertical and horizontal transmission. To test this, we have developed a novel method to classify prokaryotes, by calculating their Pfam protein domain distributions and clustering them with all other sequenced prokaryotic species. Clusters of organisms are visualized in two dimensions as &lsquo;mountains&rsquo; on a topological map. When compared to a phylogenetic map constructed using 16S rRNA, this map more accurately clusters prokaryotes according to functional and environmental attributes. We demonstrate the ability of this map, which we term a &ldquo;niche map&rdquo;, to cluster according to ecological niche both quantitatively and qualitatively, and propose that this method be used to associate uncharacterized prokaryotes with their ecological niche as a means of predicting their functional role directly from their genome sequence."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000743.g005"/>

  <meta property="og:title" content="Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9517'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9941'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2539&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000743">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000743" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000743&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Garret Suen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Syracuse University, Syracuse, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Barry S. Goldman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Monsanto, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roy D. Welch
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rowelch@syr.edu">rowelch@syr.edu</a></p>

                <p>Affiliation:
                  Department of Biology, Syracuse University, Syracuse, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 15, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000743</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000743-g001" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000743-g002" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000743-g003" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000743-t001" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000743-g004" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000743-g005" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000743">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000743" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Automated DNA sequencing technology is so rapid that analysis has become the rate-limiting step. Hundreds of prokaryotic genome sequences are publicly available, with new genomes uploaded at the rate of approximately 20 per month. As a result, this growing body of genome sequences will include microorganisms not previously identified, isolated, or observed. We hypothesize that evolutionary pressure exerted by an ecological niche selects for a similar genetic repertoire in those prokaryotes that occupy the same niche, and that this is due to both vertical and horizontal transmission. To test this, we have developed a novel method to classify prokaryotes, by calculating their Pfam protein domain distributions and clustering them with all other sequenced prokaryotic species. Clusters of organisms are visualized in two dimensions as ‘mountains’ on a topological map. When compared to a phylogenetic map constructed using 16S rRNA, this map more accurately clusters prokaryotes according to functional and environmental attributes. We demonstrate the ability of this map, which we term a “niche map”, to cluster according to ecological niche both quantitatively and qualitatively, and propose that this method be used to associate uncharacterized prokaryotes with their ecological niche as a means of predicting their functional role directly from their genome sequence.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Suen G, Goldman BS, Welch RD (2007) Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis. PLoS ONE 2(8):
          e743.
            doi:10.1371/journal.pone.0000743</p><p><strong>Academic Editor: </strong>Geraldine Butler, University College Dublin, Ireland</p><p><strong>Received:</strong> June 15, 2007; <strong>Accepted:</strong> July 13, 2007; <strong>Published:</strong> August 15, 2007</p><p><strong>Copyright:</strong> © 2007 Suen et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Publicly available sequenced prokaryote genomes will soon number in the thousands. Genomes of the major microbial model organisms have been sequenced, some multiple times; projects to sequence new genomes will select from a pool of increasingly obscure organisms. Meanwhile, meta-genome sequencing projects <a href="#pone.0000743-Riesenfeld1">[1]</a>, <a href="#pone.0000743-Chen1">[2]</a> are attempting to recapitulate whole genome assemblages, resulting in the genomes of microorganisms that have never even been identified or isolated, let alone observed in a laboratory. A great challenge for microbiologists is to exploit this large and expanding amount of sequence data to provide biological information and ecological context for these new genomes <a href="#pone.0000743-Cohan1">[3]</a>–<a href="#pone.0000743-Martiny1">[5]</a>. If a genome sequence is the only thing known about an organism, what can be learned about its specific role and function in an ecosystem? In other words, can we use a genome sequence to identify an organism's ecological niche?</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Initial attempts to define the term ‘niche’ focused either on the environment an organism inhabits <a href="#pone.0000743-Grinell1">[6]</a> or the function of an organism <a href="#pone.0000743-Elton1">[7]</a>, but a more contemporary definition incorporates both of these aspects <a href="#pone.0000743-Patten1">[8]</a>. Every species occupies a niche, but defining the niche of many prokaryote species is difficult because of their surprisingly wide geographical and ecological ranges <a href="#pone.0000743-Martiny2">[9]</a>. The traditional methods used to define the niche of a novel prokaryote are morphological observation, biochemical characterization, and phylogenetic classification by multiple sequence alignment of its 16S rRNA <a href="#pone.0000743-Garrity1">[10]</a>–<a href="#pone.0000743-Woese2">[12]</a>. If an organism cannot be isolated in the laboratory, however, it cannot be observed morphologically or characterized biochemically. In addition, attempting to identify a niche by phylogenetics alone is proving to be difficult: although there is an apparent connection between phylogeny and niche, phylogenetically distant species sometimes share the same niche, and phylogenetically close species sometimes occupy very different niches. New algorithms that expand phylogenetics to incorporate the entire genome, including those based on average amino acid identity <a href="#pone.0000743-Konstantinidis1">[13]</a>, shared gene orthology <a href="#pone.0000743-Ciccarelli1">[14]</a>, <a href="#pone.0000743-Snel1">[15]</a>, protein structures or domains <a href="#pone.0000743-Lin1">[16]</a>–<a href="#pone.0000743-Tekaia1">[18]</a>, and correlated indel alignments <a href="#pone.0000743-Gupta1">[19]</a>, were developed to either verify existing 16S rRNA phylogeny or to suggest new phylogenetic relationships <a href="#pone.0000743-Tekaia1">[18]</a>. However, these algorithms are not optimized to discern the genomic relationship between organisms in a comprehensive way, since they ignore or minimize the effects of horizontal gene transfer (HGT).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>A genome sequence is the product of adaptation in response to evolutionary pressure. In theory, any set of genes can be transferred between prokaryotes in the same environment <a href="#pone.0000743-Ochman1">[20]</a>, <a href="#pone.0000743-Gogarten1">[21]</a>, and the fixation of new genetic material occurs rapidly relative to eukaryotes, at least in part because of a prokaryote's relatively short generation time. Genetic changes occur through a variety of evolutionary mechanisms, including vertical descent, HGT <a href="#pone.0000743-Ochman1">[20]</a>–<a href="#pone.0000743-Boucher1">[23]</a>, duplication and divergence <a href="#pone.0000743-Taylor1">[24]</a>, <a href="#pone.0000743-Ohno1">[25]</a>, and genome reduction <a href="#pone.0000743-Moran1">[26]</a>, <a href="#pone.0000743-Ochman3">[27]</a>. Although there is disagreement regarding the relative significance of these mechanisms <a href="#pone.0000743-Ochman1">[20]</a>, <a href="#pone.0000743-Simonson1">[28]</a>, there is no doubt that prokaryotes use all of them to acquire new functionality. In this way, a prokaryotic community can be thought of as a single evolving genomic assemblage, with the environment, rather than the species, defining the organisms that inhabit the assemblage <a href="#pone.0000743-Goldenfeld1">[4]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A prokaryote's genetic repertoire is defined as all of the functionality encoded within its genome <a href="#pone.0000743-Lerat1">[29]</a>. To quantify this, an organism's genome must be broken down into fundamental units, and the fundamental unit of evolution is thought to be the protein domain <a href="#pone.0000743-Orengo1">[30]</a>, <a href="#pone.0000743-Apic1">[31]</a>. A prokaryote's genetic complexity is reflected in the expansion and recombination of proteins, and new proteins are created primarily through the duplication and divergence of protein domains <a href="#pone.0000743-Chothia1">[32]</a>. Therefore, the genetic repertoire of a prokaryote is the distribution of protein domains within its genome, and the genetic repertoires of prokaryotes can be clustered according to the similarity of their protein domain distributions. We hypothesize that these clusters will correspond to specific ecological niches.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In this article, we report on the clustering of over 450 sequenced prokaryotes according to their genetic repertoires. Protein domain distributions were ranked using Spearman's correlation, clustered using multidimensional scaling and force-directed placement <a href="#pone.0000743-Davidson1">[33]</a>, and visualized as mountains on a topographical map. Prokaryotes described as occupying the same or similar environment were found within the same mountains on the map, and those sharing similar physiological roles also clustered, thus providing insight as to how these organisms evolved and adapted to their niches. We conclude that this type of DNA sequence metadata analysis can provide useful information about the biology of a prokaryote for which the only available data are its genome sequence and annotation.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>The Protein family (Pfam) annotation was used to measure the protein domain distribution of each genome <a href="#pone.0000743-Finn1">[34]</a>. The Pfam annotation is an extensive collection of manually curated protein multiple sequence alignments that describes each protein family as a set of conserved domains related to a particular function. We aligned the genetic repertoires, as represented by their predicted proteomes, of over 450 sequenced prokaryotes against the Pfam database to construct a distribution profile for each genome, as shown in <a href="#pone-0000743-g001">Figure 1a, b</a>. This profile is represented as an <em>n</em>-component vector of values, with each value corresponding to the total number of instances a particular protein domain occurs within each prokaryote's genome. A map, which we call a “niche map”, was then constructed by clustering each prokaryote's Pfam profile as shown in <a href="#pone-0000743-g001">Figure 1c, d</a>. A phylogenetic map was constructed by the same method using 16S rRNA, so that we could directly compare the niche map to this more traditional analysis, as visualized in the phylogenetic map. A preliminary examination reveals obvious differences with respect to the number of mountains on each map, as shown in <a href="#pone-0000743-g002">Figure 2</a>; the niche and phylogenetic maps contain 18 and 9 mountains, respectively.</p>
<div class="figure" id="pone-0000743-g001"><div class="img"><a name="pone-0000743-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Construction of the niche and phylogenetic maps.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>The niche map is constructed by comparing all predicted proteins within each prokaryote (B<sub>1</sub>–B<sub>n</sub>) against the Pfam database (a). Likewise, construction of a phylogenetic map is done by performing a multiple sequence alignment using the 16S rRNA sequence from each prokaryote. Each metric is then converted into a Pfam profile and 16S distance matrix, respectively (b). The Pfam profile matrix is further converted into a similarity matrix by applying Spearman's rank correlation (c). Each Prokaryote is then assigned an (x, y) coordinate by applying a combination of multi-dimensional scaling and force-directed placement to both similarity and distance matrices as shown in (d). Finally, a topographical map is generated using the computer program <em>VxInsight</em>.</p>
<span>doi:10.1371/journal.pone.0000743.g001</span></div><div class="figure" id="pone-0000743-g002"><div class="img"><a name="pone-0000743-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Topographical representation of the niche (a) and phylogenetic (b) maps.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Mountain numbers and the corresponding taxonomic groups for each prokaryote within each mountain are shown. Taxonomic abbreviations are as follows: ACI = Acidobacteria; ACT = Actinobacteria; APB = Alphaproteobacteria; AQU = Aquificae; BAC = Bacteroidetes/Chlorobi; BPB = Betaproteobacteria; DPB = Deltaproteobacteria; CHV = Chlamydiae/Verrucomicrobia; CHL = Chloroflexi; CRE = Crenarchaeota; CYA = Cyanobacteria; DET = Deinococcus-Thermus; EPB = Epsilonproteobacteria; EUR = Euryarchaeota; FIR = Firmicutes; FUS = Fusobacteria; GPB = Gammaproteobacteria; NAN = Nanoarchaeota; PLA = Planctomycetes; SPI = Spirochaetes; THE = Thermotogae.</p>
<span>doi:10.1371/journal.pone.0000743.g002</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Classification of prokaryotic species is based on the phylogenetic distance between each species' 16S rRNA sequence, and the phylogenetic map clusters according to this metric. As there is clearly some overlap between phylogenetic distance and the concept of niche similarity, some mountains on the phylogenetic map will contain organisms that occupy the same niche. When protein domain distribution also correlates with niche similarity, the niche and phylogenetic maps will overlap. However, when 16S rRNA distance and protein domain distribution do not correlate, the niche and phylogenetic maps will diverge and, if genetic repertoire more closely correlates to niche similarity, then divergent mountains on the niche map will correspond to niche similarity, while divergent mountains on the phylogenetic map will correspond to phylogenetic distance.</p>

<h4>Mountains on the Phylogenetic Map Correlate to Phylogenetic Distance</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To identify the differences between phylogenetic and niche maps, we assigned phylogenies to all prokaryotic species in both maps, as shown in <a href="#pone-0000743-g002">Figure 2</a>; a tabular representation of this annotation is presented in <a href="#pone-0000743-t001">Table 1</a>, and <a href="#pone.0000743.s002">Table S1</a>. Based on these assignments, 11 out of the 15 taxonomic groups (73%) containing more than one sequenced member had all of their members clustering in a single mountain on the phylogenetic map, as compared to only 5 out of 15 (33%) on the niche map. If clusters are based on phylogenetic distance, then the nearest neighbors of any prokaryote should share in the same phylogenetic group designation; to test this, we applied a shared phylogenetic group metric to the nearest neighbor sets of each prokaryote on both maps. Analysis of both maps across the nearest neighbor sets of 5, 10, 15, 20, and 25 (determined by measuring the Euclidean distance between a prokaryote and all other prokaryotes on the map) indicates that prokaryotes on the phylogenetic map have a higher percentage of nearest neighbors that share the same phylogenetic group than those on the niche map, as shown in <a href="#pone-0000743-g003">Figure 3</a>. Both maps exhibit significant correlation to phylogeny when compared to a randomized control, calculated by randomly assigning nearest neighbors to each prokaryote.</p>
<div class="figure" id="pone-0000743-g003"><div class="img"><a name="pone-0000743-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Percentage of comparisons with matches for each prokaryote based on a shared taxonomic group metric.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The top 5, 10, 15, 20, and 25 nearest neighbors for each prokaryotic species were retained and their taxonomic groups compared by computing the average number of matches. This analysis was performed for prokaryotes on the niche map (circles) and the phylogenetic map (squares) as shown. In all cases, sequencing bias was taken into account by disregarding all pairs of prokaryotic species that had a Jaccard coefficient greater than 0.90 based on Pfam profile analysis (see <a href="#s4">Materials and Methods</a>). A control (average of 10 trials) was also performed by randomizing the nearest neighbors for each prokaryotic species (hashes). Error bars for the randomized control are too small to be displayed.</p>
<span>doi:10.1371/journal.pone.0000743.g003</span></div><div class="figure" id="pone-0000743-t001"><div class="img"><a name="pone-0000743-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000743.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000743.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Composition of mountains for the niche map based on 469 sequenced prokaryotic genomes.</span></strong></p><span>doi:10.1371/journal.pone.0000743.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The highest percent of shared phylogenetic grouping on the phylogenetic map occurs when the top 5 nearest neighbors are used, producing a value close to 80%, which then steadily decreases to near 70% as the nearest neighbor set increases to 25. It is important to note that 16 out of the 21 represented phylogenetic groups currently have less than 25 sequenced members, so this decrease toward 70% is almost certainly due to the limited number of prokaryotic genomes currently available (<a href="#pone.0000743.s003">Table S2</a>). These results indicate that, as the number of sequenced genomes approaches the set of all prokaryotic species, correlation between the phylogenetic map and phylogeny will approach 100%. In contrast, for the niche map, the percent of shared phylogenetic grouping remains constant at 60% across all nearest neighbor sets, indicating that it will not approach 100% as the set of all sequenced prokaryotic genomes approaches saturation. These data show that both phylogenetic and niche maps correlate to phylogeny, and that the phylogenetic map exhibits better correlation than the niche map.</p>


<h4>Mountains on the Niche Map Correlate to Environment and Function</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To make an objective comparison of phylogenetic and niche maps based on niche similarity, we applied a metric that, although arguably not ideal, is externally derived and relates to the environmental and functional definition of a niche. Specifically, we selected nine categories from the Organism Information dataset associated with the current collection of sequenced prokaryotic genomes (see <a href="#s4">Materials and Methods</a> for a full list of categories). We then calculated the average percent of categorical matches between each prokaryote and its nearest neighbor sets of 5, 10, 15, 20, and 25 for both phylogenetic and niche maps, as shown in <a href="#pone-0000743-g004">Figure 4</a>. The similarity values for the niche map increase from 65% to 68% as the nearest neighbor sets decrease from 25 to 5, whereas values for the phylogenetic map and randomized control are consistently less than those for the niche map, remaining constant across all nearest neighbor sets, with percentages of 62% and 52%, respectively. This comparison between the phylogenetic and niche maps using these nine Organism Information categories presents the inverse of the phylogeny comparison. These data show that phylogenetic and niche maps both correlate to the Organism Information categories, and that the niche map exhibits better correlation than the phylogenetic map.</p>
<div class="figure" id="pone-0000743-g004"><div class="img"><a name="pone-0000743-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Percentage of comparisons with matches for each prokaryotic species based on an environment and function metric.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The top 5, 10, 15, 20, and 25 nearest neighbors for each prokaryotic species were retained and their corresponding environmental and functional data were compared by computing the average number of matches across nine categories. This analysis was performed for prokaryotes on the niche map (circles) and the phylogenetic map (squares) as shown. In all cases, sequencing bias was taken into account by disregarding all pairs of prokaryotic species that had a Jaccard coefficient greater than 0.90 based on Pfam profile analysis (see <a href="#s4">Materials and Methods</a>). A control (average of 10 trials) was also performed by randomizing the nearest neighbors for each prokaryotic species (hashes). Error bars for all three analyses are too small to be displayed.</p>
<span>doi:10.1371/journal.pone.0000743.g004</span></div>

<h4>Qualitative Analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Phylogeny has a rigorous definition but niche does not. The Organism Information categories used to compare phylogenetic and niche maps relate to niche because they describe aspects of environment and function, but we make no claim that these categories are comprehensive; given the present state of prokaryotic post-genomics, it would be impossible to craft a comprehensive definition of prokaryotic niche that would be universally accepted. It is possible to discuss specific organisms and the niches they inhabit, however. Therefore, to further support the hypothesis that the niche map clusters according to niche similarity, we present a detailed analysis of some mountains on the phylogenetic and niche maps.</p>


<h4>Comparison of Clustering by Phylogeny: the Gammaproteobacteria</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The Gammaproteobacteria are a broad group that includes the largest number of sequenced genomes, currently at more than 100, including many from extensively characterized prokaryotes <a href="#pone.0000743-Lerat2">[35]</a>. Members of this group, such as <em>Escherichia coli, Salmonella typhimurium</em>, and <em>Yersinia pestis</em> are not only prominent model organisms, but also include some of mankind's most pernicious pathogens. An analysis of the phylogenetic map places this diverse group into three mountains (PM02, PM06, and PM07; <a href="#pone.0000743.s002">Table S1</a>). PM02 contains predominantly members of the <em>Pseudomondales, Legionellales</em>, and <em>Thiotrichales</em>. PM06 contains members of the <em>Xanthomondales</em>, including species of <em>Xanthomonas</em> and <em>Xylella</em>, and members of the <em>Chromatiales</em> including <em>Alkaliminicola</em> and <em>Nitrosococcus</em>. PM07 contains the largest grouping of Gammaproteobacteria, including the <em>Enterobacteriales</em>, including <em>Escherichia</em> and <em>Salmonella</em>, and the <em>Vibrionales</em>, including <em>Vibrio</em> and <em>Photobacterium</em>. Also found in this mountain are the <em>Pasteurales</em>, including species of <em>Mannheimia</em> and <em>Haemophilus</em>, and members of the <em>Alteromonadales</em> including <em>Shewanella</em> and <em>Pseudoalteromonas</em>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In contrast, the niche map distributes this same set of organisms across nine mountains: NM01, NM02, NM08, NM10, NM11, NM12, NM14, NM15, and NM16 (<a href="#pone-0000743-t001">Table 1</a>). The <em>Enterobacteriale</em>s form a tight cluster within NM12 that also includes four members of the <em>Pasteurales</em> that are all associated with humans as pathogens. These include the genera <em>Haemophilus, Mannheinia</em>, and <em>Pasteurella</em>. The only member of the <em>Enterobacteriales</em> not found in this mountain is the aphid symbiont <em>Buchnera</em>, which is found in NM16 along with a group of Alphaproteobacteria that are either obligate symbionts or pathogens (see below).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The <em>Xanthomonadales</em> form a distinct group in NM01 (<a href="#pone-0000743-t001">Table 1</a>) of the plant pathogenic species <em>Xylella</em> and <em>Xanthomonas</em>. The set of Pfams that distinguish the <em>Xanthomonadales</em> from the other groups was analyzed as described in <a href="#pone.0000743.s001">Text S1</a> and shown in <a href="#pone.0000743.s004">Table S3</a>; many of these proteins degrade carbohydrates, which is consistent with the biology of plant pathogens. In addition, a number of Pfams that correspond to membrane lipoproteins and proteins of uncharacterized function are also found as part of this set. Interestingly, the Gammaproteobacteria <em>Pseudomonas syringae</em>, another plant pathogen, ends up in NM14 (<a href="#pone-0000743-t001">Table 1</a>) with the other <em>Pseudomonadales</em> and an order of the Betaproteobacteria called the <em>Burkholderiales</em>, which contains the genera <em>Ralstonia, Bordetella</em>, and <em>Burkholderia</em>. What sets this group apart is the predominance of soil and plant associated prokaryotes, many of which are also opportunistic human pathogens (see below).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Perhaps the group that reveals the most about the ability of the niche map to clustering according to niche is in NM11, (<a href="#pone-0000743-t001">Table 1</a> and <a href="#pone-0000743-g005">Figure 5a</a>) which contains several different members of the <em>Alteromonadales, Oceanospirillales</em>, and the <em>Vibrionales</em>. The members of this phylogenetically diverse group within the Gammaproteobacteria are all marine organisms, and the niche map clusters them according to a shared environment. A survey of the Pfams specific to the prokaryotes in NM11 (<a href="#pone.0000743.s005">Table S4</a>) reveal many membrane lipoproteins and proteins of uncharacterized function, similar to those domains observed for the <em>Xanthomondales</em>. Interestingly, the marine Gammaproteobacteria in NM11 (<a href="#pone-0000743-t001">Table 1</a>) are found adjacent to the <em>Xanthomondales</em> in NM01 (<a href="#pone-0000743-t001">Table 1</a>) on the niche map (see <a href="#pone-0000743-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000743-g005"><div class="img"><a name="pone-0000743-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000743" data-uri="info:doi/10.1371/journal.pone.0000743.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000743.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000743.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000743.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000743.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Clustering of prokaryotic species on the niche map.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Three groups of prokaryotic species are shown, including the marine Gammaproteobacteria in NM11 that cluster according to phylogeny (a); the obligate symbionts and pathogens in NM10 and NM16 that cluster according to function (b); and the prokaryotes existing at the soil, plant, and human interface in NM14 that cluster according to environment. A high resolution view of each mountain is shown, in addition to the complete niche map labeled with the corresponding mountain (blue circles). The genus of every prokaryote in each mountain is also shown.</p>
<span>doi:10.1371/journal.pone.0000743.g005</span></div>

<h4>Comparison of Clustering by Function: Obligate Symbionts and Pathogens</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Obligate symbionts and pathogens describe a diverse group that has undergone massive genome reduction <a href="#pone.0000743-Moran1">[26]</a>, <a href="#pone.0000743-Ochman3">[27]</a>, presumably in response to their constrained relationship with a host. Symbionts are characterized as colonizing without necessarily compromising host vigor, whereas pathogens produce a negative impact on the overall health of the host <a href="#pone.0000743-Dale1">[36]</a>. Organisms that exist as obligate symbionts and pathogens form two tight clusters on the niche map, NM10 and NM16, as shown in <a href="#pone-0000743-t001">Table 1</a> and <a href="#pone-0000743-g005">Figure 5b</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>NM10 (<a href="#pone-0000743-t001">Table 1</a>) consists mainly of obligate pathogens that are distributed across three mountains on the phylogenetic map, PM05, PM07, and PM09 (<a href="#pone.0000743.s002">Table S1</a>) that encompass four different orders including members of the Firmicutes, Gammaproteobacteria, Nanoarchaeota, and Spirochaetes. The majority of prokaryotes found in NM10 (<a href="#pone-0000743-t001">Table 1</a>) belong to the Mollicutes, including all sequenced members of the genera <em>Mycoplasma</em>, the human pathogen <em>Ureaplasma urealyticum</em>, and the plant pathogens <em>Aster yellows witches'-broom phytoplasma</em> and <em>Onion yellows phytoplasma</em>. All of these organisms are known to infect a wide variety of hosts, including swine, chickens, plants, and humans. A total of five of the seven sequenced Spirochaetes are also found in this mountain, including all sequenced members of the genera <em>Borrelia</em> and <em>Treponema</em>. All of these organisms are known to cause a number of diseases in humans, including Lyme disease, Syphilis, and Gingivitis. Interestingly, it has been suggested that the close genomic similarities that exist between the distantly related <em>Borrelia</em> and <em>Mycoplasma</em> are an indication of convergent evolution <a href="#pone.0000743-Fraser1">[37]</a>. The only sequenced member of the Nanoarchaeota, <em>Nanoarchaeum equitans</em>, a symbiont of the archaeon <em>Ignicoccus</em>, is also found in this mountain.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The phylogenetic composition of the organisms clustered in NM16 (<a href="#pone-0000743-t001">Table 1</a>) are found distributed across four different mountains on the phylogenetic map, PM02, PM03, PM07, and PM08 (<a href="#pone.0000743.s002">Table S1</a>), and include members of the Alphaproteobacteria, Deltproteobacteria, Epsilonproteobacteria, and Gammaproteobacteria. There is evident sub-clustering within this mountain, with the Epsilonproteobacteria grouping together on the periphery. This sub-cluster includes members of the <em>Campylobacterales</em>, including the genera <em>Helicobacter</em> and <em>Campylobacter,</em> while the rest of this mountain contains the following mixture of Alphaproteobacteria and Gammaproteobacteria: obligate pathogens of the genera <em>Ehrlichia</em> that target horses, dogs, and humans; human pathogens of the genera <em>Rickettsia</em> and <em>Neorickettsia</em>; sheep pathogens of the genera <em>Anaplasma</em>; human pathogens of the genera <em>Francisella</em>; cattle and human pathogens of the genera <em>Haemophilus</em>; human pathogens of the genera <em>Bartonella</em>; and the human pathogen <em>Coxiella brunetii</em>. Interestingly, the only sequenced pathogenic Deltaproteobacteria, <em>Lawsonia intracellularis</em>, a swine pathogen, is also peripherally found in NM16. In addition to these pathogens, three obligate insect symbionts, the Alphaproteobacterium <em>Wolbachia</em> and the Gammaproteobacteria <em>Buchnera</em> and <em>Wigglesworthia,</em> are also found in this mountain. A functional characteristic that distinguishes the organisms in NM16 from those in NM10 (<a href="#pone-0000743-t001">Table 1</a>) is that the majority of prokaryotes in NM10 directly infect humans and plants, while those in NM16 are transmitted to their final host through insect carriers.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Genome reduction in obligate symbionts and parasites might be at least partially responsible for the organisms in NM10 and NM16 (<a href="#pone-0000743-t001">Table 1</a>) being separate from the rest of the niche map, but it does not explain why these organisms form two separate mountains. We compared the Pfam sets between these two mountains, and compiled a list of Pfams specific to each mountain (<a href="#pone.0000743.s006">Table S5</a> and <a href="#pone.0000743.s007">S6</a>). An analysis of the Pfam set specific to NM10 reveals a majority of ribosomal subunit proteins, whereas the same analysis for NM16 reveals a set that is largely involved in DNA replication, transcription and translation, and cell division. Perhaps the set of Pfams that distinguish NM10 are a hallmark of direct human-related pathogenicity, while the set of Pfams that distinguish NM16 reflect the selective pressures that result from being transmitted via insects.</p>


<h4>Comparison of Clustering by Environment: The Soil, Plant, and Human Interface</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Within the niche map, NM14 (<a href="#pone-0000743-t001">Table 1</a>) exemplifies the wide ecological range exhibited by some prokaryotes <a href="#pone.0000743-Martiny2">[9]</a>. The prokaryotes that cluster within NM14 belong to the Betaproteobacteria and the Gammaproteobacteria, and are found distributed across two mountains on the phylogenetic map (PM02 and PM06; <a href="#pone.0000743.s002">Table S1</a>) according to phylogeny. The reason these organisms form a single mountain on the niche map may be due to their common environment, since the organisms in NM14 exist in soils, as plant-associated microbes, as pathogens in humans, or at the interface between these three niches. Soil-specific microbes within this mountain include the nitrogen fixer <em>Azotobacter vinelandii,</em> heavy-metal degraders <em>P. putida</em> and <em>Cupriavidus metallidurans</em>, and members of the genera <em>Polaromonas, Rhodoferax</em>, and <em>Ralstonia</em>. Exclusive plant-associated microbes include the plant pathogen <em>P. syringae</em>, as well as the plant symbiont <em>B. thailandiensis</em>. Human pathogens within NM14 include almost all represented members of the genera <em>Burkholderia</em>, all members of the genera <em>Bordetella</em>, and <em>P. aeruginosa PA01</em>. Some of the prokaryotes that also cluster in this mountain exist in more than one environment, such as the human and plant pathogen <em>P. aeruginosa PA14</em>; the soil-dwelling and plant associated <em>P. fluorescens</em>; the soil-dwelling and human pathogens <em>Acinetobacter sp. ADP1</em> and <em>Chromobacterium violaceum</em>; and <em>B. xenovorans</em>, which is found to exist in all three environments.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Furthermore, NM14 (<a href="#pone-0000743-t001">Table 1</a>) exhibits clustering of prokaryotes that share both niche and morphology. Members of the <em>Pseudomondales</em> and <em>Burkholderiales</em> are known to thrive in the lungs of Cystic Fibrosis patients, and historically, the <em>Burkholderiales</em> were originally classified as belonging to the <em>Pseudomonadales</em> based on their morphological similarity. Subsequent studies using 16S rRNA revealed that these prokaryotes belong to different clades altogether <a href="#pone.0000743-Yabuuchi1">[38]</a>. Interestingly, these prokaryotes form mixed biofilms <a href="#pone.0000743-Riedel1">[39]</a> and <em>P. aeruginosa</em> is known to promote <em>B. cepacia</em> pathogenesis by upregulating certain virulence factors <a href="#pone.0000743-Harrison1">[40]</a>. Therefore, the similarity in morphology shared by both groups and their ability to cooperate in mixed biofilms may suggest that these prokaryotes share a more complex evolutionary history than that proposed by 16S analysis.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>An analysis of the Pfams that distinguish this group from other groups on the map reveal that many of these are membrane lipoproteins and proteins of uncharacterized function as shown in <a href="#pone.0000743.s008">Table S7</a>. Interestingly, these Pfams are also characteristic of the <em>Xanthomondales</em> and the marine Gammaproteobacteria in NM01 and NM11 (<a href="#pone-0000743-t001">Table 1</a>), respectively. Accordingly, the soil, plant, and human interface prokaryotes found in NM14 are also found adjacent to both the <em>Xanthomondales</em> and marine Gammaproteobacteria on the niche map (<a href="#pone-0000743-g002">Figure 2</a>). Differences between Pfam sets could indicate proteins that are specific to a particular niche, such as the carbohydrate degradation enzymes specific to the <em>Xanthomondales</em>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>This is the first demonstration of correlation between an organism's genomic repertoire and its ecological niche. We present a novel computational method for clustering organisms according to their protein domain distribution and demonstrate, both quantitatively and qualitatively, that the resulting niche map correlates to the concept of ecological niche <a href="#pone.0000743-Patten1">[8]</a> better than a phylogenetic map. The accuracy of the niche map will likely improve as new sequenced genomes are added. For example, an earlier iteration of the niche map based on 340 genomes clustered the <em>Xanthomondales</em> (NM01, <a href="#pone-0000743-t001">Table 1</a>) and the marine Gammaproteobacteria (NM11, <a href="#pone-0000743-t001">Table 1</a>) into a single mountain (data not shown). The subsequent division into two mountains on the current map resolves these plant pathogens from the marine-associated Gammaproteobacteria.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Through a comparison of phylogenetic and niche maps, we demonstrate that the phylogenetic map exhibits closer correlation to phylogeny than the niche map. This result is expected because, like a phylogenetic tree, the topology of the phylogenetic map is based on a multiple sequence alignment of different organisms' 16S rRNA. The 16S subunit is chosen because it is considered to be both ubiquitous and essential, and is therefore likely to be evolving predominantly through vertical descent <a href="#pone.0000743-Woese1">[11]</a>. Although there are examples where 16S rRNA is thought to be transmitted through means other than vertical descent <a href="#pone.0000743-Badger1">[41]</a>, the phylogenetic map effectively isolates the outcome of vertical decent from other evolutionary processes, such as HGT, duplication and divergence, and genome reduction. The topology of the phylogenetic map is, therefore, a visualization of one evolutionary mechanism.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We also demonstrate that the niche map exhibits closer correlation than the phylogenetic map to nine Organism Information categories that relate to the definition of niche. The topology of the niche map is based on the clustering of multiple prokaryotes' protein domain distribution, with each distribution representing one prokaryote's genomic repertoire reduced to its fundamental functional elements. In this way, a protein domain distribution is the opposite of a carefully selected sequence; it is the Pfam parts list for a genome and, as such, is a blunt expression of an organism's overall evolutionary outcome at the genome scale. The topology of the niche map is a visualization of all evolutionary mechanisms including, but not limited to vertical decent. If phylogenetic map topology is determined by a subset of the evolutionary mechanisms that determine niche map topology, then those mountains exhibiting the highest degree of overlap between phylogenetic and niche maps represent clusters of organisms whose ecological niche constrains evolution to vertical decent. An example of this phenomenon is the Archaea, the majority of which are extremophiles that have limited interaction with other organisms due to their restrictive environment. The Archaea cluster within a single mountain on both phylogenetic (PM05, <a href="#pone.0000743.s002">Table S1</a>) and niche (NM13, <a href="#pone-0000743-t001">Table 1</a>) maps, except for the symbiont <em>N. equitans,</em> which appropriately clusters with other symbionts and pathogens on the niche map (NM10, <a href="#pone-0000743-t001">Table 1</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We report several other specific examples of similarities and differences between phylogenetic and niche maps, using detailed qualitative analysis to demonstrate a correlation between defining characteristics of niche (i.e. environment and function) and the clustering of organisms on the niche map. Although this analysis requires that the organisms be morphologically or physiologically characterized, construction of the niche map does not. Placement of each organism on the niche map is based on an algorithm that incorporates only genome sequence data, so that entirely novel and uncharacterized organisms can be placed on the map as soon as they are sequenced. The purely computational implementation of the niche map also permits the assignment of putative characteristics to unknown organisms based on the shared characteristics of well-studied organisms within the same mountain. This type of ‘guilt-by-association’ analysis is frequently employed in functional genomics, where it is used to assign putative functions to genes and interaction partners to proteins <a href="#pone.0000743-Oliver1">[42]</a>. Although the immediate value of this analysis lies in the practical application of the genome metadata set to predict the function of unknown prokaryotes, it is not outside the realm of possibility that similar forms of protein domain clustering could be extended to eukaryotes. For example, epigenomic protein domain clustering could be used to identify differences between individuals within the same species, and expression profile protein domain clustering could be used to identify and characterize different cell types within the same organism.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Because of our vast sequencing infrastructure, we are understanding prokaryotic diversity genome-first. To bring these data into biological focus, we must unify genomic and morphological taxonomy. In addition to building better algorithms to support or refute phylogeny based on 16S rRNA, we should apply computational rigor to other theoretical concepts of genomic evolution, and then process the vast post-genomic dataset to determine what ideas can withstand scrutiny. If a protein domain represents the smallest functional unit of evolution, and if the role of an organism within its ecological niche can be represented by the sum of its functional parts, then a comparison of a genome's ‘parts lists’ should group organism by ecological niche. Setting all ambiguities regarding the definition of prokaryotic species and niche aside, the approach presented in this report is effective.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Datasets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>In this study, 469 prokaryotic genome sequences were used in the construction of the maps. A total of 381 completed sequences were obtained from the National Institute of Biotechnological Information (NCBI, <a href="http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi">http://www.ncbi.nlm.nih.gov/genomes/lpro​ks.cgi</a>, accessed: 10/20/2006) and an additional 88 draft genomes were obtained from the Integrated Microbial Genomes database <a href="#pone.0000743-Markowitz1">[43]</a> (<a href="http://img.jgi.doe.gov/cgi-bin/pub/main.cgi">http://img.jgi.doe.gov/cgi-bin/pub/main.​cgi</a>, accessed: 10/20/2006). Sequences of the 16S small ribosomal subunit for the 385 NCBI genomes were obtained from NCBI or the Ribosomal Database Project <a href="#pone.0000743-Cole1">[44]</a> (<a href="http://rdp.cme.msu.edu/">http://rdp.cme.msu.edu/</a>, accessed: 10/20/2006) and used in the construction of the phylogenetic map. The Pfam database used in this study was obtained from NCBI's Conserved Domain Database for protein classification <a href="#pone.0000743-MarchlerBauer1">[45]</a> (<a href="ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/">ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/</a>, accessed: 10/20/2006). Phylogenetic group, environmental, and functional annotations for all prokaryotes used in this study were obtained from the organism information page associated with the microbial genome projects available from NCBI (<a href="http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi">http://www.ncbi.nlm.nih.gov/genomes/lpro​ks.cgi</a>, accessed: 10/20/2006).</p>


<h4>Construction of the niche map</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The predicted proteomes of all 469 prokaryotic genomes was used in the construction of the niche map. Two maps were constructed; one based on the 381 completed prokaryotic genomes from NCBI, and a second map based on all 469 complete and draft genomes. For each map, all proteins were aligned against the Pfam database using RPSBLAST <a href="#pone.0000743-MarchlerBauer1">[45]</a>, <a href="#pone.0000743-Altschul1">[46]</a>, and a Pfam distribution matrix was constructed as shown in <a href="#pone-0000743-g001">Figure 1</a>, with each row representing a different prokaryotic genome and each column specifying a Pfam identification number. Each cell within this matrix was populated with the number of instances each Pfam was found within each genome. A similarity matrix was then constructed by calculating the correlation between each pair of species using Spearman's rank correlation. For each prokaryote, the top 25 prokaryotes that had the highest positive correlation scores were retained, and a combination of multidimensional scaling and force-directed placement was then used to ordinate each prokaryote on a two-dimensional plane. This ordination was done using the <em>VxOrd</em> program included as part of the <em>VxInsight</em> package <a href="#pone.0000743-Davidson1">[33]</a>, with the settings at position 3 on the edge cuts slider and no sub-clustering. These coordinates were then visualized in three-dimensions using the computer program <em>VxInsight</em> as shown in <a href="#pone.0000743.s009">Map S1</a>. The resulting topographical niche map clusters prokaryotes with similar Pfam distribution profiles into mountains, with the height of each mountain corresponding to the density of prokaryotes found in that area. Each mountain on this map was then numbered in ascending order according to the number of prokaryotes present in each mountain, as shown in <a href="#pone-0000743-t001">Table 1</a>.</p>


<h4>Construction of the phylogenetic map</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A map based on the 16S small ribosomal subunit was created for the 381 completed genome sequences obtained from NCBI. Multiple sequence alignment of all 16S rRNA was done using MUSCLE <a href="#pone.0000743-Edgar1">[47]</a>, as shown in <a href="#pone-0000743-g001">Figure 1</a>, and a distance matrix was then computed using the DNADIST program included in the PHYLIP computer package <a href="#pone.0000743-Felenstein1">[48]</a>. The Jukes-Cantor evolutionary model was used in the calculation of this distance matrix. For each prokaryote, the top 50 species with the closest distance scores were retained, and a combination of multidimensional scaling and force-directed placement was used to ordinate each genome on a two-dimensional plane. This ordination was done using the <em>VxOrd</em> program included as part of the <em>VxInsight</em> package <a href="#pone.0000743-Davidson1">[33]</a>, with the settings at position 3 on the edge cuts slider and no sub-clustering. These coordinates were then visualized in three-dimensions as a topographical phylogenetic map using <em>VxInsight</em> as shown in <a href="#pone.0000743.s010">Map S2</a>. Each mountain on this map was then numbered in ascending order according to the number of prokaryotes present in each mountain, as shown in <a href="#pone.0000743.s002">Table S1</a>. All analysis using the Jukes-Cantor phylogenetic map was also performed on a second map constructed using the Kimura 2-paramater evolutionary model, and similar results were obtained in each case.</p>


<h4>Comparison of niche and phylogenetic maps</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To establish if there is a quantitative difference between the niche similarity and vertical descent derived maps, we compared the nearest neighbor sets for each prokaryote. This was done using the niche and phylogenetic maps based on the 381 completed genome sequences from NCBI. The nearest 5, 10, 15, 20, and 25 neighbor sets were obtained by calculating the Euclidean distance between each prokaryote and every other prokaryote on the map, and corrected for sequencing bias.</p>


<h4>Sequencing Bias Correction</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>An analysis of the complete genome sequences from NCBI reveals that they are skewed toward particular taxonomic groups as shown in <a href="#pone.0000743.s003">Table S2</a>. This sequencing bias is due to a number of factors, including the limitation of technology to sequence only those prokaryotes that can be cultured, and the fact that prokaryotes are usually sequenced based on their interest for research purposes <a href="#pone.0000743-Martiny1">[5]</a>, <a href="#pone.0000743-Hugenholtz1">[49]</a>. As a result, there is an overrepresentation of some prokaryotes, and in particular, specific strains of the same species. The clustering of specific strains of the same prokaryote using Pfam may introduce “false positives” because these prokaryotes are likely clustering due to the similar Pfam distributions generated from their nearly identical genomes. To correct for this, we compared the Pfam distributions of all prokaryotes, based on the presence or absence of individual Pfam identification numbers in their respective genomes, and calculated the Jaccard coefficient of similarity (defined as: A∩B/A∪B for two sets A and B). We retained only those pairs of prokaryotes that had a Jaccard coefficient greater than 0.90 (data not shown). All subsequent comparisons between the niche similarity and vertical descent derived maps were done by disregarding any pair of prokaryotes that appeared on this sequencing bias correction list.</p>


<h4>Assessment of clustering using a phylogenetic group metric</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>To determine the extent at which each map clustered prokaryotes according to phylogenetic groups, we calculated the percentage of nearest neighbors that shared the same phylogenetic group for each prokaryote on the niche and phylogenetic maps. We calculated the 5, 10, 15, 20, and 25 nearest neighbor sets for each species (sequencing bias corrected) and calculated the percentage of nearest neighbors that shared the same phylogenetic group. These percentages were then averaged across all prokaryotes for each nearest neighbor set. A control (average of 10 trials) was also done by randomly assigning nearest neighbors to each prokaryote (sequencing bias corrected) and applying this shared phylogenetic group metric.</p>


<h4>Assessment of clustering according to environment and function</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>We utilized an objective and externally derived metric that corresponds to both environment and function. We calculated the 5, 10, 15, 20, and 25 nearest neighbor sets for each species (sequencing bias corrected) and counted the number of matches that occurred for a total of 9 Organism Information categories. These categories include <em>Shape, Arrangement, Endospores, Motility, Oxygen Requirements, Habitat, Temperature,</em> and <em>Pathogenicity</em>. Categories for which each pair of prokaryotes did not both contain data were excluded from this comparison. The percentage of comparisons with matches was averaged across all prokaryotes for each nearest neighbor set. A control (average of 10 trials) was also done by randomly assigning nearest neighbors to each prokaryote (sequencing bias corrected) and applying this metric.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000743"></div><a name="pone.0000743.s001" id="pone.0000743.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s001">Text S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0000743.s002" id="pone.0000743.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s002">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000743.s003" id="pone.0000743.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s003">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000743.s004" id="pone.0000743.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s004">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.14 MB DOC)</p>
<a name="pone.0000743.s005" id="pone.0000743.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s005">Table S4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0000743.s006" id="pone.0000743.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s006">Table S5. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000743.s007" id="pone.0000743.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s007">Table S6. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(0.12 MB DOC)</p>
<a name="pone.0000743.s008" id="pone.0000743.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s008">Table S7. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000743.s009" id="pone.0000743.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s009">Map S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="postSiDOI">(3.90 MB GZ)</p>
<a name="pone.0000743.s010" id="pone.0000743.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000743.s010">Map S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000743.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="postSiDOI">(3.89 MB GZ)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Iain J. Anderson, Shawn L. Stricklin, and Laura G. Welch for insightful discussions and critical reading of the manuscript. We would also like to thank members of the Welch lab for their support and encouragement.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BG. Performed the experiments: GS. Analyzed the data: GS. Wrote the paper: RW BG GS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000743-Riesenfeld1" id="pone.0000743-Riesenfeld1"></a>Riesenfeld CS, Schloss PD, Handelsman J (2004) Metagenomics: genomic analysis of microbial communities. Annu Rev Genet  38: 525–552.  <ul class="find" data-citedArticleID="977902" data-doi="10.1146/annurev.genet.38.072902.091216"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.091216" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metagenomics%3A+genomic+analysis+of+microbial+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metagenomics%3A+genomic+analysis+of+microbial+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000743-Chen1" id="pone.0000743-Chen1"></a>Chen K, Pachter L (2005) Bioinformatics for whole-genome shotgun sequencing of microbial communities. PLoS Comput Biol  1: 106–112.  <ul class="find" data-citedArticleID="977830" data-doi="10.1371/journal.pcbi.0010024"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioinformatics+for+whole-genome+shotgun+sequencing+of+microbial+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioinformatics+for+whole-genome+shotgun+sequencing+of+microbial+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000743-Cohan1" id="pone.0000743-Cohan1"></a>Cohan FM (2004)  Concepts of bacterial biodiversity for the age of genomics. In: Fraser CM, Read T, Nelson KE, editors. Microbial Genomes. Totowa, NJ: Humana Press.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000743-Goldenfeld1" id="pone.0000743-Goldenfeld1"></a>Goldenfeld N, Woese C (2007) Biology's next revolution. Nature  445: 369.  <ul class="find" data-citedArticleID="977858" data-doi="10.1038/445369a"><li><a href="http://dx.doi.org/10.1038/445369a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biology%27s+next+revolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biology%27s+next+revolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000743-Martiny1" id="pone.0000743-Martiny1"></a>Martiny JB, Field D (2005) Ecological perspectives on the sequenced genome collection. Ecology Letters  8: 1334–1345.  <ul class="find" data-citedArticleID="977880" data-doi="10.1111/j.1461-0248.2005.00837.x"><li><a href="http://dx.doi.org/10.1111/j.1461-0248.2005.00837.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+perspectives+on+the+sequenced+genome+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+perspectives+on+the+sequenced+genome+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000743-Grinell1" id="pone.0000743-Grinell1"></a>Grinell J (1917) The niche-relationships of the California thrasher. Auk  34: 427–433.  <ul class="find" data-citedArticleID="977860" data-doi="10.2307/4072271"><li><a href="http://dx.doi.org/10.2307/4072271" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+niche-relationships+of+the+California+thrasher." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+niche-relationships+of+the+California+thrasher.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000743-Elton1" id="pone.0000743-Elton1"></a>Elton C (1927) Animal ecology. London: Sidgwick&amp;Jackson.   <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0000743-Patten1" id="pone.0000743-Patten1"></a>Patten BC, Auble GT (1981) System Theory of the Ecological Niche. The American Naturalist  117: 893–922.  <ul class="find" data-citedArticleID="977898" data-doi="10.1086/283777"><li><a href="http://dx.doi.org/10.1086/283777" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=System+Theory+of+the+Ecological+Niche." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22System+Theory+of+the+Ecological+Niche.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000743-Martiny2" id="pone.0000743-Martiny2"></a>Martiny JB, Bohannan BJ, Brown JH, Colwell RK, Fuhrman JA, et al.  (2006) Microbial biogeography: putting microorganisms on the map. Nat Rev Microbiol  4: 102–112.  <ul class="find" data-citedArticleID="977882" data-doi="10.1038/nrmicro1341"><li><a href="http://dx.doi.org/10.1038/nrmicro1341" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+biogeography%3A+putting+microorganisms+on+the+map." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+biogeography%3A+putting+microorganisms+on+the+map.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000743-Garrity1" id="pone.0000743-Garrity1"></a>Garrity GM (2001) Bergey's Manual of Systematic Bacteriology. New York: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0000743-Woese1" id="pone.0000743-Woese1"></a>Woese CR (1987) Bacterial evolution. Microbiol Rev  51: 221–271.  <ul class="find" data-citedArticleID="977912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacterial+evolution.&amp;auth=&amp;atitle=Bacterial+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000743-Woese2" id="pone.0000743-Woese2"></a>Woese CR, Kandler O, Wheelis ML (1990) Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya. Proc Natl Acad Sci U S A  87: 4576–4579.  <ul class="find" data-citedArticleID="977914" data-doi="10.1073/pnas.87.12.4576"><li><a href="http://dx.doi.org/10.1073/pnas.87.12.4576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+natural+system+of+organisms%3A+proposal+for+the+domains+Archaea%2C+Bacteria%2C+and+Eucarya." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+natural+system+of+organisms%3A+proposal+for+the+domains+Archaea%2C+Bacteria%2C+and+Eucarya.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000743-Konstantinidis1" id="pone.0000743-Konstantinidis1"></a>Konstantinidis KT, Tiedje JM (2005) Towards a genome-based taxonomy for prokaryotes. J Bacteriol  187: 6258–6264.  <ul class="find" data-citedArticleID="977868" data-doi="10.1128/jb.187.18.6258-6264.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.18.6258-6264.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+genome-based+taxonomy+for+prokaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+genome-based+taxonomy+for+prokaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000743-Ciccarelli1" id="pone.0000743-Ciccarelli1"></a>Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, et al.  (2006) Toward automatic reconstruction of a highly resolved tree of life. Science  311: 1283–1287.  <ul class="find" data-citedArticleID="977834" data-doi="10.1126/science.1123061"><li><a href="http://dx.doi.org/10.1126/science.1123061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000743-Snel1" id="pone.0000743-Snel1"></a>Snel B, Bork P, Huynen MA (1999) Genome phylogeny based on gene content. Nat Genet  21: 108–110.  <ul class="find" data-citedArticleID="977906" data-doi="10.1038/5052"><li><a href="http://dx.doi.org/10.1038/5052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+phylogeny+based+on+gene+content." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+phylogeny+based+on+gene+content.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000743-Lin1" id="pone.0000743-Lin1"></a>Lin J, Gerstein M (2000) Whole-genome trees based on the occurrence of folds and orthologs: implications for comparing genomes on different levels. Genome Res  10: 808–818.  <ul class="find" data-citedArticleID="977874" data-doi="10.1101/gr.10.6.808"><li><a href="http://dx.doi.org/10.1101/gr.10.6.808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+trees+based+on+the+occurrence+of+folds+and+orthologs%3A+implications+for+comparing+genomes+on+different+levels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+trees+based+on+the+occurrence+of+folds+and+orthologs%3A+implications+for+comparing+genomes+on+different+levels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000743-Yang1" id="pone.0000743-Yang1"></a>Yang S, Doolittle RF, Bourne PE (2005) Phylogeny determined by protein domain content. Proc Natl Acad Sci U S A  102: 373–378.  <ul class="find" data-citedArticleID="977918" data-doi="10.1073/pnas.0408810102"><li><a href="http://dx.doi.org/10.1073/pnas.0408810102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+determined+by+protein+domain+content." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+determined+by+protein+domain+content.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000743-Tekaia1" id="pone.0000743-Tekaia1"></a>Tekaia F, Yeramian E (2005) Genome trees from conservation profiles. PLoS Comput Biol  1: e75.  <ul class="find" data-citedArticleID="977910" data-doi="10.1371/journal.pcbi.0010075"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+trees+from+conservation+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+trees+from+conservation+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000743-Gupta1" id="pone.0000743-Gupta1"></a>Gupta RS, Griffiths E (2002) Critical issues in bacterial phylogeny. Theor Popul Biol  61: 423–434.  <ul class="find" data-citedArticleID="977862" data-doi="10.1006/tpbi.2002.1589"><li><a href="http://dx.doi.org/10.1006/tpbi.2002.1589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Critical+issues+in+bacterial+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Critical+issues+in+bacterial+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000743-Ochman1" id="pone.0000743-Ochman1"></a>Ochman H, Lerat E, Daubin V (2005) Examining bacterial species under the specter of gene transfer and exchange. Proc Natl Acad Sci U S A  102: Suppl 16595–6599.  <ul class="find" data-citedArticleID="977886" data-doi="10.1073/pnas.0502035102"><li><a href="http://dx.doi.org/10.1073/pnas.0502035102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Examining+bacterial+species+under+the+specter+of+gene+transfer+and+exchange." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Examining+bacterial+species+under+the+specter+of+gene+transfer+and+exchange.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000743-Gogarten1" id="pone.0000743-Gogarten1"></a>Gogarten JP, Doolittle WF, Lawrence JG (2002) Prokaryotic evolution in light of gene transfer. Mol Biol Evol  19: 2226–2238.  <ul class="find" data-citedArticleID="977856" data-doi="10.1093/oxfordjournals.molbev.a004046"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prokaryotic+evolution+in+light+of+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prokaryotic+evolution+in+light+of+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000743-Ochman2" id="pone.0000743-Ochman2"></a>Ochman H, Lawrence JG, Groisman EA (2000) Lateral gene transfer and the nature of bacterial innovation. Nature  405: 299–304.  <ul class="find" data-citedArticleID="977888" data-doi="10.1038/35012500"><li><a href="http://dx.doi.org/10.1038/35012500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lateral+gene+transfer+and+the+nature+of+bacterial+innovation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lateral+gene+transfer+and+the+nature+of+bacterial+innovation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000743-Boucher1" id="pone.0000743-Boucher1"></a>Boucher Y, Douady CJ, Papke RT, Walsh DA, Boudreau ME, et al.  (2003) Lateral gene transfer and the origins of prokaryotic groups. Annu Rev Genet  37: 283–328.  <ul class="find" data-citedArticleID="977828" data-doi="10.1146/annurev.genet.37.050503.084247"><li><a href="http://dx.doi.org/10.1146/annurev.genet.37.050503.084247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lateral+gene+transfer+and+the+origins+of+prokaryotic+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lateral+gene+transfer+and+the+origins+of+prokaryotic+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000743-Taylor1" id="pone.0000743-Taylor1"></a>Taylor JS, Raes J (2004) Duplication and divergence: the evolution of new genes and old ideas. Annu Rev Genet  38: 615–643.  <ul class="find" data-citedArticleID="977908" data-doi="10.1146/annurev.genet.38.072902.092831"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.092831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000743-Ohno1" id="pone.0000743-Ohno1"></a>Ohno S (1970) Evolution by Gene Duplication. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000743-Moran1" id="pone.0000743-Moran1"></a>Moran NA (2002) Microbial minimalism: genome reduction in bacterial pathogens. Cell  108: 583–586.  <ul class="find" data-citedArticleID="977884" data-doi="10.1016/s0092-8674(02)00665-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00665-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+minimalism%3A+genome+reduction+in+bacterial+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+minimalism%3A+genome+reduction+in+bacterial+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000743-Ochman3" id="pone.0000743-Ochman3"></a>Ochman H, Moran NA (2001) Genes lost and genes found: evolution of bacterial pathogenesis and symbiosis. Science  292: 1096–1099.  <ul class="find" data-citedArticleID="977890" data-doi="10.1126/science.1058543"><li><a href="http://dx.doi.org/10.1126/science.1058543" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+lost+and+genes+found%3A+evolution+of+bacterial+pathogenesis+and+symbiosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+lost+and+genes+found%3A+evolution+of+bacterial+pathogenesis+and+symbiosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000743-Simonson1" id="pone.0000743-Simonson1"></a>Simonson AB, Servin JA, Skophammer RG, Herbold CW, Rivera MC, et al.  (2005) Decoding the genomic tree of life. Proc Natl Acad Sci U S A 102 Suppl  1: 6608–6613.  <ul class="find" data-citedArticleID="977904" data-doi="10.1073/pnas.0501996102"><li><a href="http://dx.doi.org/10.1073/pnas.0501996102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Decoding+the+genomic+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Decoding+the+genomic+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000743-Lerat1" id="pone.0000743-Lerat1"></a>Lerat E, Daubin V, Ochman H, Moran NA (2005) Evolutionary origins of genomic repertoires in bacteria. PLoS Biol  3: e130.  <ul class="find" data-citedArticleID="977870" data-doi="10.1371/journal.pbio.0030130"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+origins+of+genomic+repertoires+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+origins+of+genomic+repertoires+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000743-Orengo1" id="pone.0000743-Orengo1"></a>Orengo CA, Thornton JM (2005) Protein families and their evolution-a structural perspective. Annu Rev Biochem  74: 867–900.  <ul class="find" data-citedArticleID="977896" data-doi="10.1146/annurev.biochem.74.082803.133029"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.74.082803.133029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+families+and+their+evolution-a+structural+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+families+and+their+evolution-a+structural+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000743-Apic1" id="pone.0000743-Apic1"></a>Apic G, Gough J, Teichmann SA (2001) Domain combinations in archaeal, eubacterial and eukaryotic proteomes. J Mol Biol  310: 311–325.  <ul class="find" data-citedArticleID="977824" data-doi="10.1006/jmbi.2001.4776"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.4776" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+combinations+in+archaeal%2C+eubacterial+and+eukaryotic+proteomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+combinations+in+archaeal%2C+eubacterial+and+eukaryotic+proteomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000743-Chothia1" id="pone.0000743-Chothia1"></a>Chothia C, Gough J, Vogel C, Teichmann SA (2003) Evolution of the protein repertoire. Science  300: 1701–1703.  <ul class="find" data-citedArticleID="977832" data-doi="10.1126/science.1085371"><li><a href="http://dx.doi.org/10.1126/science.1085371" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+protein+repertoire." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+protein+repertoire.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000743-Davidson1" id="pone.0000743-Davidson1"></a>Davidson G, Wylie B, Boyack K (2001) Cluster stability and the use of noise in interpretation of clustering; 2001. IEEE Computer Society. pp. 23–30.  <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0000743-Finn1" id="pone.0000743-Finn1"></a>Finn RD, Mistry J, Schuster-Bockler B, Griffiths-Jones S, Hollich V, et al.  (2006) Pfam: clans, web tools and services. Nucleic Acids Res  34: D247–251.  <ul class="find" data-citedArticleID="977850" data-doi="10.1093/nar/gkj149"><li><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pfam%3A+clans%2C+web+tools+and+services." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pfam%3A+clans%2C+web+tools+and+services.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000743-Lerat2" id="pone.0000743-Lerat2"></a>Lerat E, Daubin V, Moran NA (2003) From gene trees to organismal phylogeny in prokaryotes: the case of the gamma-Proteobacteria. PLoS Biol  1: E19.  <ul class="find" data-citedArticleID="977872" data-doi="10.1371/journal.pbio.0000019"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0000019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+gene+trees+to+organismal+phylogeny+in+prokaryotes%3A+the+case+of+the+gamma-Proteobacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+gene+trees+to+organismal+phylogeny+in+prokaryotes%3A+the+case+of+the+gamma-Proteobacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000743-Dale1" id="pone.0000743-Dale1"></a>Dale C, Moran NA (2006) Molecular interactions between bacterial symbionts and their hosts. Cell  126: 453–465.  <ul class="find" data-citedArticleID="977840" data-doi="10.1016/j.cell.2006.07.014"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.07.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+interactions+between+bacterial+symbionts+and+their+hosts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+interactions+between+bacterial+symbionts+and+their+hosts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000743-Fraser1" id="pone.0000743-Fraser1"></a>Fraser CM, Casjens S, Huang WM, Sutton GG, Clayton R, et al.  (1997) Genomic sequence of a Lyme disease spirochaete, Borrelia burgdorferi. Nature  390: 580–586.  <ul class="find" data-citedArticleID="977852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genomic+sequence+of+a+Lyme+disease+spirochaete%2C+Borrelia+burgdorferi.&amp;auth=&amp;atitle=Genomic+sequence+of+a+Lyme+disease+spirochaete%2C+Borrelia+burgdorferi." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+sequence+of+a+Lyme+disease+spirochaete%2C+Borrelia+burgdorferi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+sequence+of+a+Lyme+disease+spirochaete%2C+Borrelia+burgdorferi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000743-Yabuuchi1" id="pone.0000743-Yabuuchi1"></a>Yabuuchi E, Kosako Y, Oyaizu H, Yano I, Hotta H, et al.  (1992) Proposal of Burkholderia gen. nov. and transfer of seven species of the genus Pseudomonas homology group II to the new genus, with the type species Burkholderia cepacia (Palleroni and Holmes 1981) comb. nov. Microbiol Immunol  36: 1251–1275.  <ul class="find" data-citedArticleID="977916" data-doi="10.1111/j.1348-0421.1992.tb02129.x"><li><a href="http://dx.doi.org/10.1111/j.1348-0421.1992.tb02129.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proposal+of+Burkholderia+gen.+nov.+and+transfer+of+seven+species+of+the+genus+Pseudomonas+homology+group+II+to+the+new+genus%2C+with+the+type+species+Burkholderia+cepacia+%28Palleroni+and+Holmes+1981%29+comb.+nov." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proposal+of+Burkholderia+gen.+nov.+and+transfer+of+seven+species+of+the+genus+Pseudomonas+homology+group+II+to+the+new+genus%2C+with+the+type+species+Burkholderia+cepacia+%28Palleroni+and+Holmes+1981%29+comb.+nov.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000743-Riedel1" id="pone.0000743-Riedel1"></a>Riedel K, Hentzer M, Geisenberger O, Huber B, Steidle A, et al.  (2001) N-acylhomoserine-lactone-mediated communication between Pseudomonas aeruginosa and Burkholderia cepacia in mixed biofilms. Microbiology  147: 3249–3262.  <ul class="find" data-citedArticleID="977900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=N-acylhomoserine-lactone-mediated+communication+between+Pseudomonas+aeruginosa+and+Burkholderia+cepacia+in+mixed+biofilms.&amp;auth=&amp;atitle=N-acylhomoserine-lactone-mediated+communication+between+Pseudomonas+aeruginosa+and+Burkholderia+cepacia+in+mixed+biofilms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=N-acylhomoserine-lactone-mediated+communication+between+Pseudomonas+aeruginosa+and+Burkholderia+cepacia+in+mixed+biofilms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22N-acylhomoserine-lactone-mediated+communication+between+Pseudomonas+aeruginosa+and+Burkholderia+cepacia+in+mixed+biofilms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000743-Harrison1" id="pone.0000743-Harrison1"></a>Harrison F (2007) Microbial ecology of the cystic fibrosis lung. Microbiology  153: 917–923.  <ul class="find" data-citedArticleID="977864" data-doi="10.1099/mic.0.2006/004077-0"><li><a href="http://dx.doi.org/10.1099/mic.0.2006/004077-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microbial+ecology+of+the+cystic+fibrosis+lung." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microbial+ecology+of+the+cystic+fibrosis+lung.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000743-Badger1" id="pone.0000743-Badger1"></a>Badger JH, Eisen JA, Ward NL (2005) Genomic analysis of Hyphomonas neptunium contradicts 16S rRNA gene-based phylogenetic analysis: implications for the taxonomy of the orders ‘Rhodobacterales’ and Caulobacterales. Int J Syst Evol Microbiol  55: 1021–1026.  <ul class="find" data-citedArticleID="977826" data-doi="10.1099/ijs.0.63510-0"><li><a href="http://dx.doi.org/10.1099/ijs.0.63510-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+of+Hyphomonas+neptunium+contradicts+16S+rRNA+gene-based+phylogenetic+analysis%3A+implications+for+the+taxonomy+of+the+orders+%26%23x2018%3BRhodobacterales%26%23x2019%3B+and+Caulobacterales." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+of+Hyphomonas+neptunium+contradicts+16S+rRNA+gene-based+phylogenetic+analysis%3A+implications+for+the+taxonomy+of+the+orders+%26%23x2018%3BRhodobacterales%26%23x2019%3B+and+Caulobacterales.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000743-Oliver1" id="pone.0000743-Oliver1"></a>Oliver S (2000) Guilt-by-association goes global. Nature  403: 601–603.  <ul class="find" data-citedArticleID="977894" data-doi="10.1038/35001165"><li><a href="http://dx.doi.org/10.1038/35001165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Guilt-by-association+goes+global." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Guilt-by-association+goes+global.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000743-Markowitz1" id="pone.0000743-Markowitz1"></a>Markowitz VM, Korzeniewski F, Palaniappan K, Szeto E, Werner G, et al.  (2006) The integrated microbial genomes (IMG) system. Nucleic Acids Res  34: D344–348.  <ul class="find" data-citedArticleID="977878" data-doi="10.1093/nar/gkj024"><li><a href="http://dx.doi.org/10.1093/nar/gkj024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+integrated+microbial+genomes+%28IMG%29+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+integrated+microbial+genomes+%28IMG%29+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000743-Cole1" id="pone.0000743-Cole1"></a>Cole JR, Chai B, Farris RJ, Wang Q, Kulam-Syed-Mohideen AS, et al.  (2007) The ribosomal database project (RDP-II): introducing myRDP space and quality controlled public data. Nucleic Acids Res  35: D169–172.  <ul class="find" data-citedArticleID="977838" data-doi="10.1093/nar/gkl889"><li><a href="http://dx.doi.org/10.1093/nar/gkl889" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ribosomal+database+project+%28RDP-II%29%3A+introducing+myRDP+space+and+quality+controlled+public+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ribosomal+database+project+%28RDP-II%29%3A+introducing+myRDP+space+and+quality+controlled+public+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000743-MarchlerBauer1" id="pone.0000743-MarchlerBauer1"></a>Marchler-Bauer A, Anderson JB, Derbyshire MK, DeWeese-Scott C, Gonzales NR, et al.  (2007) CDD: a conserved domain database for interactive domain family analysis. Nucleic Acids Res  35: D237–240.  <ul class="find" data-citedArticleID="977876" data-doi="10.1093/nar/gkl951"><li><a href="http://dx.doi.org/10.1093/nar/gkl951" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CDD%3A+a+conserved+domain+database+for+interactive+domain+family+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CDD%3A+a+conserved+domain+database+for+interactive+domain+family+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000743-Altschul1" id="pone.0000743-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="977822" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000743-Edgar1" id="pone.0000743-Edgar1"></a>Edgar RC (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res  32: 1792–1797.  <ul class="find" data-citedArticleID="977844" data-doi="10.1093/nar/gkh340"><li><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000743-Felenstein1" id="pone.0000743-Felenstein1"></a>Felenstein J (2006) PHYLIP (Phylogeny Inference Package), version 3.6. Seattle: Department of Genetics, university of Washington.   <ul class="find-nolinks"></ul></li><li><span class="label">49.
              </span><a name="pone.0000743-Hugenholtz1" id="pone.0000743-Hugenholtz1"></a>Hugenholtz P (2002) Exploring prokaryotic diversity in the genomic era. Genome Biol  3: REVIEWS0003.  <ul class="find" data-citedArticleID="977866" data-doi="10.1186/gb-2002-3-2-reviews0003"><li><a href="http://dx.doi.org/10.1186/gb-2002-3-2-reviews0003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+prokaryotic+diversity+in+the+genomic+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+prokaryotic+diversity+in+the+genomic+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.XML" value="107042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.PDF" value="420528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g001.PNG_L" value="3235155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g001.PNG_M" value="132898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g001.PNG_S" value="14930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g001.TIF" value="4939816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g001.PNG_I" value="69264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g002.PNG_L" value="2399139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g002.PNG_M" value="165870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g002.PNG_S" value="10343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g002.TIF" value="4067654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g002.PNG_I" value="53880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g003.PNG_L" value="29814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g003.PNG_M" value="19611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g003.PNG_S" value="5294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g003.TIF" value="44480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g003.PNG_I" value="7211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.t001.PNG_L" value="727975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.t001.PNG_M" value="308346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.t001.PNG_S" value="20182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.t001.TIF" value="2850496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.t001.PNG_I" value="134007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g004.PNG_L" value="34116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g004.PNG_M" value="22967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g004.PNG_S" value="5846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g004.TIF" value="49356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g004.PNG_I" value="8423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g005.PNG_L" value="6072794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g005.PNG_M" value="391421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g005.PNG_S" value="18116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g005.TIF" value="9598268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.g005.PNG_I" value="130139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s001.DOC" value="22016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s002.DOC" value="40448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s003.DOC" value="35840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s004.DOC" value="139776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s005.DOC" value="55296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s006.DOC" value="43008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s007.DOC" value="121856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s008.DOC" value="42496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s009.GZ" value="3899264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000743.s010.GZ" value="3890440"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000743&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000743" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000743&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000743&volume=&issue=&title=Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis&author_name=Garret%20Suen%2C%20Barry%20S.%20Goldman%2C%20Roy%20D.%20Welch&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000743" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743&amp;t=Predicting%20Prokaryotic%20Ecological%20Niches%20Using%20Genome%20Sequence%20Analysis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743&title=Predicting%20Prokaryotic%20Ecological%20Niches%20Using%20Genome%20Sequence%20Analysis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743&amp;title=Predicting%20Prokaryotic%20Ecological%20Niches%20Using%20Genome%20Sequence%20Analysis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000743&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Predicting Prokaryotic Ecological Niches Using Genome Sequence Analysis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000743';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Predicting%20Prokaryotic%20Ecological%20Niches%20Using%20Genome%20Sequence%20Analysis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000743" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000743" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ecological+niches%22" title="Search for articles in the subject area:'Ecological niches'"><div class="flagText">Ecological niches</div></a>
              <div data-categoryid="48505" data-articleid="24324"
                   data-categoryname="Ecological niches"
                   class="flagImage" title="Flag 'Ecological niches' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24324"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24324"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mountains%22" title="Search for articles in the subject area:'Mountains'"><div class="flagText">Mountains</div></a>
              <div data-categoryid="18603" data-articleid="24324"
                   data-categoryname="Mountains"
                   class="flagImage" title="Flag 'Mountains' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="24324"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+pathogens%22" title="Search for articles in the subject area:'Plant pathogens'"><div class="flagText">Plant pathogens</div></a>
              <div data-categoryid="34897" data-articleid="24324"
                   data-categoryname="Plant pathogens"
                   class="flagImage" title="Flag 'Plant pathogens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Prokaryotic+cells%22" title="Search for articles in the subject area:'Prokaryotic cells'"><div class="flagText">Prokaryotic cells</div></a>
              <div data-categoryid="17761" data-articleid="24324"
                   data-categoryname="Prokaryotic cells"
                   class="flagImage" title="Flag 'Prokaryotic cells' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+domains%22" title="Search for articles in the subject area:'Protein domains'"><div class="flagText">Protein domains</div></a>
              <div data-categoryid="40153" data-articleid="24324"
                   data-categoryname="Protein domains"
                   class="flagImage" title="Flag 'Protein domains' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5105'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5865'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7860&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=45697">A nice method for applying genomics to the niche concept.</a><br>Posted by JAdkins</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
