

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001404"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001404" />

    <meta name="citation_title" content="Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets"/>
    <meta itemprop="name" content="Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets"/>

      <meta name="citation_author" content="Sheena Lee"/>
            <meta name="citation_author_institution" content="OXION Initiative, Oxford Centre for Gene Function, Department of Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Arzu Sayin"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Stuart Grice"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Howard Burdett"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Dilair Baban"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom"/>
      <meta name="citation_author" content="Marcel van den Heuvel"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001404.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1404"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Identification and characterization of a spinal muscular atrophy-determining gene.; citation_author=S Lefebvre; citation_author=L Burglen; citation_author=S Reboullet; citation_author=O Clermont; citation_author=P Burlet; citation_journal_title=Cell; citation_volume=80; citation_number=1; citation_pages=155-165; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Correlation between severity and SMN protein level in spinal muscular atrophy.; citation_author=S Lefebvre; citation_author=P Burlet; citation_author=Q Liu; citation_author=S Bertrandy; citation_author=O Clermont; citation_journal_title=Nature Genetics; citation_volume=16; citation_number=2; citation_pages=265-269; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inactivation of the survival motor neuron gene, a candidate gene for human spinal muscular atrophy, leads to massive cell death in early mouse embryos.; citation_author=B Schrank; citation_author=R Gotz; citation_author=JM Gunnersen; citation_author=JM Ure; citation_author=KV Toyka; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=94; citation_number=3; citation_pages=9920-9925; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of a mouse homologue of the spinal muscular atrophy-determining gene, survival motor neuron.; citation_author=A Bergin; citation_author=G Kim; citation_author=DL Price; citation_author=SS Sisodia; citation_author=MK Lee; citation_journal_title=Gene; citation_volume=204; citation_number=4; citation_pages=47-53; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Why do cells need an assembly machine for RNA-protein complexes?; citation_author=J Yong; citation_author=L Wan; citation_author=G Dreyfuss; citation_journal_title=Trends in cell biology; citation_volume=14; citation_number=5; citation_pages=226-232; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The SMN complex.; citation_author=AK Gubitz; citation_author=W Feng; citation_author=G Dreyfuss; citation_journal_title=Experimental cell research; citation_volume=296; citation_number=6; citation_pages=51-56; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Neuromuscular defects in a Drosophila survival motor neuron gene mutant.; citation_author=YB Chan; citation_author=I Miguel Aliaga; citation_author=C Franks; citation_author=N Thomas; citation_author=B Trulzsch; citation_journal_title=Human molecular genetics; citation_volume=12; citation_number=7; citation_pages=1367-1376; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reduced expression of nicotinic AChRs in myotubes from spinal muscular atrophy 1 patients.; citation_author=AS Arnold; citation_author=M Gueye; citation_author=S Guettier-Sigrist; citation_author=I Courdier-Fruh; citation_author=G Coupin; citation_author=P Poindron; citation_author=JP Gies; citation_journal_title=Laboratory Investigation; citation_number=8; citation_pages=1-8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SMNDelta7, the major product of the centromeric survival motor neuron (SMN2) gene, extends survival in mice with spinal muscular atrophy and associates with full-length SMN.; citation_author=TT Le; citation_author=LT Pham; citation_author=ME Butchbach; citation_author=HL Zhang; citation_author=UR Monani; citation_journal_title=Human molecular genetics; citation_volume=14; citation_number=9; citation_pages=845-857; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparison of seven methods for producing Affymetrix expression scores based on False Discovery Rates in disease profiling data.; citation_author=K Shedden; citation_author=W Chen; citation_author=R Kuick; citation_author=D Ghosh; citation_author=J Macdonald; citation_journal_title=BMC bioinformatics; citation_volume=6; citation_number=10; citation_pages=26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Four different subunits are essential for expressing the synaptic glutamate receptor at neuromuscular junctions of Drosophila.; citation_author=G Qin; citation_author=T Schwarz; citation_author=RJ Kittel; citation_author=A Schmid; citation_author=TM Rasse; citation_journal_title=Journal of neuroscience the official journal of the Society for Neuroscience ISSN (ELECTRONIC VERSION): 1529 2401; citation_volume=25; citation_number=11; citation_pages=3209-3218; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Spinal muscular atrophy: a deficiency in a ubiquitous protein; a motor neuron-specific disease.; citation_author=UR Monani; citation_journal_title=Neuron ISSN (PRINT VERSION): 0896 6273; citation_volume=48; citation_number=12; citation_pages=885-895; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mutant small heat-shock protein 27 causes axonal Charcot-Marie-Tooth disease and distal hereditary motor neuropathy.; citation_author=OV Evgrafov; citation_author=I Mersiyanova; citation_author=J Irobi; citation_author=L Van Den Bosch; citation_author=I Dierick; citation_journal_title=Nature genetics; citation_volume=36; citation_number=13; citation_pages=602-606; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hot-spot residue in small heat-shock protein 22 causes distal motor neuropathy.; citation_author=J Irobi; citation_author=K Van Impe; citation_author=P Seeman; citation_author=A Jordanova; citation_author=I Dierick; citation_journal_title=Nature genetics; citation_volume=36; citation_number=14; citation_pages=597-601; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Active transport of the survival motor neuron protein and the role of exon-7 in cytoplasmic localization.; citation_author=HL Zhang; citation_author=F Pan; citation_author=D Hong; citation_author=SM Shenoy; citation_author=RH Singer; citation_journal_title=Journal of neuroscience the official journal of the Society for Neuroscience; citation_volume=23; citation_number=15; citation_pages=6627-6637; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ca2+, mitochondria and selective motoneuron vulnerability: implications for ALS.; citation_author=F von Lewinski; citation_author=BU Keller; citation_journal_title=Trends in neurosciences; citation_volume=28; citation_number=16; citation_pages=494-500; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Meaningless minis? Mechanisms of neurotransmitter-receptor clustering.; citation_author=P Verstreken; citation_author=HJ Bellen; citation_journal_title=Trends in neurosciences; citation_volume=25; citation_number=17; citation_pages=383-385; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Presynaptic glutamic acid decarboxylase is required for induction of the postsynaptic receptor field at a glutamatergic synapse.; citation_author=DE Featherstone; citation_author=EM Rushton; citation_author=M Hilderbrand-Chae; citation_author=AM Phillips; citation_author=FR Jackson; citation_journal_title=Neuron; citation_volume=27; citation_number=18; citation_pages=71-84; citation_date=2000; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets"/>
    <meta name="twitter:description" content="Spinal Muscular Atrophy is a recessive genetic disease and affects lower motor neurones and muscle tissue. A single gene is disrupted in SMA: SMN1 activity is abolished but a second copy of the gene (SMN2) provides limited activity. While the SMN protein has been shown to function in the assembly of RNA-protein complexes, it is unclear how the overall reduction in SMN activity specifically results in the neuromuscular phenotypes. Similar to humans, reduced smn activity in the fly causes earliest phenotypes in neuromuscular tissues. To uncover the effects of reduced SMN activity, we have studied gene expression in control and diseased fly tissues using whole genome micro-arrays. A number of gene expression changes are recovered and independently validated. Identified genes show trends in their predicted function: several are consistent with the function of SMN, in addition some uncover novel pathways. This and subsequent genetic analysis in the fly indicates some of the identified genes could be taken for further studies as potential drug targets for SMA and other neuromuscular disorders."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001404.g004"/>

  <meta property="og:title" content="Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7466'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=365'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4864&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001404">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001404" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001404&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sheena Lee
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sheena Lee, Arzu Sayin</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:sheena.lee@dpag.ox.ac.uk">sheena.lee@dpag.ox.ac.uk</a></p>

                <p>Affiliation:
                  OXION Initiative, Oxford Centre for Gene Function, Department of Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arzu Sayin
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sheena Lee, Arzu Sayin</p>

              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stuart Grice, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Howard Burdett, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dilair Baban, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marcel van den Heuvel
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Functional Genetics Unit, Department of Physiology, Human Anatomy and Genetics, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001404</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001404-g001" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-t001" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-g002" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-t002" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-g003" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-g004" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-t003" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001404-t004" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001404">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001404" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">


    <div id="fch_corr" class="fch_corr">
      <a id="corrections" name="corrections" toc="corrections" title= "Corrections (1)"></a>
      <div class="corr_title">
        <span></span>
        <strong>Corrections</strong>
      </div>
      <div class="citation">
          <p>
            <span class="date"> 22 Apr 2008:  </span>
   
  <span class="citation_author">Lee
    S,
    </span>
   
  <span class="citation_author">Sayin
    A,
    </span>
   
  <span class="citation_author">Cauchi
    RJ,
    </span>
   
  <span class="citation_author">Grice
    S,
    </span>
   
  <span class="citation_author">Burdett
    H,
    </span>
  <span class="citation_author">et al. </span>
<span class="citation_date">(2008)</span>
<span
    class="citation_article_title">Correction: Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets.</span>
<span class="citation_journal_title">PLoS ONE</span><span
    class="citation_issue"> 3(4):</span>
<span class="citation_start_page">10.1371/annotation/f45a5dc0-e317-4886-8c5b-1569c79a5f08.</span>
<span class="citation_doi">
 doi: 10.1371/annotation/f45a5dc0-e317-4886-8c5b-1569c79a5f08</span>
              <span>|</span>
              <a href= "/article/info:doi/10.1371%2Fannotation%2Ff45a5dc0-e317-4886-8c5b-1569c79a5f08">View correction</a> </span>
          </p>
      </div>
    </div>





<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Spinal Muscular Atrophy is a recessive genetic disease and affects lower motor neurones and muscle tissue. A single gene is disrupted in SMA: SMN1 activity is abolished but a second copy of the gene (SMN2) provides limited activity. While the SMN protein has been shown to function in the assembly of RNA-protein complexes, it is unclear how the overall reduction in SMN activity specifically results in the neuromuscular phenotypes. Similar to humans, reduced <em>smn</em> activity in the fly causes earliest phenotypes in neuromuscular tissues. To uncover the effects of reduced SMN activity, we have studied gene expression in control and diseased fly tissues using whole genome micro-arrays. A number of gene expression changes are recovered and independently validated. Identified genes show trends in their predicted function: several are consistent with the function of SMN, in addition some uncover novel pathways. This and subsequent genetic analysis in the fly indicates some of the identified genes could be taken for further studies as potential drug targets for SMA and other neuromuscular disorders.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Lee S, Sayin A, Grice S, Burdett H, Baban D, et al.  (2008) Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets. PLoS ONE 3(1):
          e1404.
            doi:10.1371/journal.pone.0001404</p><p><strong>Academic Editor: </strong>Ulrich Mueller, University of Giessen, Germany</p><p><strong>Received:</strong> November 1, 2007; <strong>Accepted:</strong> December 3, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Lee et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>We gratefully acknowledge funding by the Motor Neurone Disease Association, Jennifer Trust and Medical Research Council for the Drosophila SMA model and array validation. We thank the Wellcome Trust Integrative Physiology Initiative on Ion Channels (OXION) for funding the microarray work.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Whole genome expression analysis of animals and tissues offers a novel way of penetrating functional changes in diseased versus normal tissues. Such analysis will map direct and indirect effects of the disease state on gene expression and could identify novel gene targets for drug interference.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Genome wide expression arrays have been made possible by the annotation and gene predictions of whole genome sequence data for human and a large number of model organisms. The genome sequencing work and subsequent species comparisons have also indicated that the genetic differences between species are smaller than anticipated. Both the total number of genes identified and, interestingly, the variety of encoded proteins (and domains) are smaller than expected. Combining these findings with functional genetic analysis has led to the conclusion that for most tissues a group of highly conserved genes is required for differentiation and subsequent function. These genes are expressed and functional in the investigated tissues across the evolutionary tree of animal species. Tissue specific analysis can thus be performed in analogous tissues using a wide variety of model organisms, concentrating on these highly conserved genes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We have decided to probe the function of known human disease genes by creating and studying mutants in homologues of such genes, in simple model organisms. Here we present a study of gene expression in a simple model organism to identify novel gene pathways that are differentially regulated in normal versus mutant, i.e. diseased, states. The identification of such pathways could potentially lead to the identification of novel drug targets.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>SMA is a common recessive genetic disease affecting 1:8000-10.000 across the human population and is characterized by a specific loss of lower motor neurones and muscle atrophy. The genetic cause was identified as the Survival Motor Neurone gene (SMN), however, in the disease situation, SMN protein activity is never completely lost, instead is reduced <a href="#pone.0001404-Lefebvre1">[1]</a>. The human genome contains two copies of the SMN gene, the activity of the first copy (SMN1) is genetically affected in patients and they therefore rely totally on the activity contributed by the second copy (SMN2). However SMN2 produces less functional protein to varying degrees. The level of remaining SMN activity has been directly correlated to the severity of the phenotypes <a href="#pone.0001404-Lefebvre2">[2]</a>. In the patient cohort this has lead to distinct groupings, type I being strong, affecting newborns and babies, type II, affecting childhood and type III, affecting late childhood and adults; each group defined on the basis of concrete motor abilities. The generation of an amenable model reflecting human conditions has been complex. The mouse has a single SMN gene and loss of the gene has been shown to lead to early embryonic lethality <a href="#pone.0001404-Schrank1">[3]</a>. Molecular and functional data relating SMN function to SMA have been slow to accumulate and despite the knowledge of the genetic basis of the disease, little is known about the neuromuscular background of the phenotypes.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The SMN gene is expressed ubiquitously in all animals investigated <a href="#pone.0001404-Bergin1">[4]</a>. Extensive biochemical analysis concluded that it is involved in the assembly of large RNA-protein complexes, one of which is the spliceosome (reviewed in <a href="#pone.0001404-Yong1">[5]</a>, <a href="#pone.0001404-Gubitz1">[6]</a>). Although this work was critical in understanding the function of the SMN protein, it doesn't explain the neuronal and/or muscle specificity of SMA. Thus it remains unclear why this protein seemingly required for basal functions in the cell, leads to such specific phenotypes when activity falls below a certain threshold. It is possible that a certain basal function reaches a critical level in the neuromuscular tissues, or SMN might perform a highly specific function in these tissues.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>We have generated a fly model for SMA by knocking out the endogenous <em>smn</em> gene. Unlike in mice, the knock-out animals are born; the embryos survive on the basis of maternal contribution of transcripts coding for normal Smn protein. Maternal contribution, in the homozygous mutant background, runs out during larval stages and leads primarily to neuromuscular phenotypes before the animals die. Because the only genetic aberration in these animals is the reduction of SMN activity, the observed phenotypes must be due to SMN activity reaching a threshold for neuromuscular functionality. Further characterisation showed that these larvae develop clear external motor problems and cell and physiological defects at the motor neuron-muscle junction <a href="#pone.0001404-Chan1">[7]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>To further the molecular understanding of SMA we used our fly model in a genome wide gene expression analysis. We performed micro array analysis of whole animals and SMA affected tissues across normal and <em>smn</em> mutant animals. We used Drosophila whole genome arrays and have produced statistically relevant results. We have validated these using standard methods. In addition we find that, using genetic testing in the fly to investigate functional interactions, novel loci influencing the <em>smn</em> mutant phenotypes can be identified using these data, thus opening the path for novel drug target studies.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Experimental background</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We set out to analyse gene expression in whole animals as well as in individual tissues. The latter potentially poses a problem due to the low quantities of starting material that can be obtained from Drosophila. In order to assess gene expression using micro arrays based on the high density GeneChip system, we needed to establish if it was possible to obtain reliable expression data using small quantities of Drosophila RNA.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We compared the same RNA sample at different dilutions, using the “low starting material” amplification procedures to obtain labelled cRNA for the low concentrations (see <a href="#s4">Material and methods</a>). The overall yield of labelled RNA and the quality of the hybridisation data are presented in Supplemental <a href="#pone.0001404.s001">Figure S1</a>. Using the Affymetrix two-cycle amplification method we are able to produce ≥15 ug cRNA using starting amounts as low as 0.1 ng; below this level the output drops significantly. The product size decreases as smaller amounts of RNA are used, resulting in 5′ truncation of the cRNA and the subsequent loss in detection of genes with probes directed towards the 5′ end of the gene. On the basis of these tests, we used between 10 ng to 100 ng of starting material in our experiments to minimise 5′ truncation of the cRNA transcripts.</p>


<h4>Experimental set up</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Our aim was to compare gene expression changes between normal and <em>smn</em> mutant animals and tissues. The strong genetics of Drosophila make it possible to use a “normal” background that is genetically very close to the mutant, this genetic system is thus highly appropriate for genomic analysis. In this study we used a mutant allele of <em>smn</em> that was recombined onto an isogenised third chromosome background (see <a href="#s4">Materials and methods</a>); the <em>smn</em> mutant animals or isolated tissues will be homozygous for this allele. The control “normal” background was the same isogenised chromosome in homozygous state, but without the <em>smn</em> mutation. The other chromosomes are free to drift genetically in both normal and <em>smn</em> mutant strains.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To obtain information on the time frame of molecular changes in the <em>smn</em> mutant background and thus when to isolate our samples, we have assessed the neuromuscular tissues in second and third instar larval stages. Motility defects in the <em>smn</em> mutant animals are first seen halfway between these stages and are statistically significant by third instar. At this life stage, we have previously reported that clustering of the neurotransmitter receptor is disrupted at the neuromuscular junction (NMJ) <a href="#pone.0001404-Chan1">[7]</a>. Disruption of neurotransmitter receptor clustering has also been reported in patient tissue <a href="#pone.0001404-Arnold1">[8]</a> and in a recently described mouse model <a href="#pone.0001404-Le1">[9]</a>. On our time course, in early second instar larvae the accumulation of clusters of the neurotransmitter receptor, Glutamate receptor IIA (GluR IIA), is similar in mutant animals compared to controls (<a href="#pone-0001404-g001">Figure 1</a>). By late second stage the number and size of clusters in the mutant tissues changes compared to the controls. By the time that the larvae display motility problems (third instar) this difference is exacerbated, leaving very few clusters in the mutant situation. These results argue that the loss of motility in our animals is concomitant with or proceeded by a loss of neurotransmitter receptor clustering at the NMJ after an initially normal pattern. Thus molecular defects are visible from late second instar and increase into third instar stage, note however that even at mid-to-late third instar stage little tissue degeneration is observed (unpublished and see <a href="#pone.0001404-Chan1">[7]</a>. We used animals at the second and third instar stage for whole animal gene expression analysis and concentrated on individual tissues at the later stage only. The loss in neurotransmitter receptor clustering is most clearly seen at the later stage and since this defect has been shown in other model systems and patients, it would be most interesting to explore in our experimental analysis.</p>
<div class="figure" id="pone-0001404-g001"><div class="img"><a name="pone-0001404-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Confocal Scanning Fluorescent images of Drosophila neuromuscular junctions.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A to C, early second instar larvae; D to F, late second instar larvae and G to I, third instar larvae (mid stage). A, D and G, normal (+/+) larvae, B, C, E, F, H and I, <em>smn</em> mutant larvae (<em>smn</em>/<em>smn</em>). A, B, D, E, G and H stained for Glutamate receptor subunit; C, F and I stained for a pre-synaptic marker (syntaxin). Comparing the accumulation of receptor clusters in the normal tissue across time to the mutant tissues, it is clear that the mutant displays a reduced number of glutamate receptor clusters across time.</p>
<span>doi:10.1371/journal.pone.0001404.g001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>For tissue samples, we dissected muscle tissue and motor neuron cell bodies from mobility impaired third instar larvae. The ventral abdominal muscles were obtained by filleting larvae and removing gut, neurons and dorsal muscles, the ventral muscles were dissected from the remaining cuticle. The cell bodies of the motor neurons are found in the ventral ganglion as part of the brain structure in the larvae; this tissue was dissected out. The whole larvae and tissue samples were processed to generate labelled cRNA.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>We used the GC-RMA (Robust Multi-array Analysis) method to normalize the hybridisation results; this method uses the GC content of the probes to reduce variance in the mismatch (control) probe levels as spotted on the Affymetrix arrays <a href="#pone.0001404-Shedden1">[10]</a>.</p>


<h4>Hybridisation results</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We combined the results across the various samples to identify overlapping changes in gene expression. Any overlap patterns would allow trends to be uncovered. Common disruptions might point to an affected function in SMN deprived tissues. We used a 1.5 fold change in expression as cut-off in these comparisons and stringent statistics. We compared the expression changes for brain and muscle tissues to the whole animal data. If the role of SMN in SMA is linked to a general function that SMN performs in all cells then a transcriptional response to reduced activity might be seen in all cells of the animal. Conversely, changes specific to either muscle and/or brain might point to genes that influence the tissue specificity of SMA (see Gene Pathway Changes paragraph). The number of genes that change in these comparisons is shown in <a href="#pone-0001404-t001">Table 1</a>, and the genes listed in supplemental <a href="#pone.0001404.s002">Figure S2</a>, <a href="#pone.0001404.s003">S3</a>, <a href="#pone.0001404.s004">S4</a>, <a href="#pone.0001404.s005">S5</a>, <a href="#pone.0001404.s006">S6</a>, <a href="#pone.0001404.s007">S7</a>, <a href="#pone.0001404.s008">S8</a>, <a href="#pone.0001404.s009">S9</a> and <a href="#pone.0001404.s010">S10</a>.</p>
<div class="figure" id="pone-0001404-t001"><div class="img"><a name="pone-0001404-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Table showing the number of gene expression changes at cut-off of ≥1.5 fold, in common between tissues indicated and larvae of different stages.</span></strong></p><span>doi:10.1371/journal.pone.0001404.t001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The number of genes in common between the tissue and whole animal samples is small; each tissue sample itself shows consistently the largest number of gene expression changes (“No change in larvae” row). This might reflect the fact that the lowered levels of <em>smn</em> activity specifically affect the neuromuscular tissues; gene expression changes specific to these tissues could fall below the cut-off levels in the whole animal samples. We have previously assessed the sensitivity of the muscle and neuronal tissues in a genetic experiment. A transgene encoding Smn was expressed in a homozygous <em>smn</em> mutant background. As expected rescue was almost 100% by ubiquitous expression of the transgene, however, interestingly, significant rescue was also observed when the transgene was expressed solely in muscle and neurons. These results showed clearly that these specific tissues in the fly are more sensitive to the levels of <em>smn</em> compared to the rest of the animal <a href="#pone.0001404-Chan1">[7]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Although it is difficult to concentrate on individual genes at this point, we will highlight some here. Across all samples, three genes encoding cytochrome P450 domains are up-regulated as well as two Glutathione S transferases. Interestingly two genes that encode G-protein-coupled-receptor-like proteins, amenable drug targets, with unknown function are uniformly up-regulated. Amongst the genes that are down regulated, different family members of Glutathione S transferase (CG6776) and cytochrome P450 (or electron transport, CG4511) encoding genes are found. Within the down-regulated genes in the muscle a gene that has predicted RNA cap binding function is found (CG8023).</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The larval samples were taken at two time points and this enables us to investigate the changes in gene expression across time. Many genes that have been annotated but not functionally analysed are identified in this way. Within these, a trend is recognizable towards a down-regulation of metabolic and transporter activities. Some genes associated with immune response are found up-regulated. In both brain and muscle samples (and whole larvae) a gene annotated as N-methyl-D-aspartate receptor associated protein is up-regulated, only at the later stage. This might indicate a link with the reduced accumulation of Glutamate Receptors at the mutant NMJs in the mutant animals. A protein known to function in transcript transport and translation control, <em>spire</em>, is up-regulated at the earlier stage in muscle only.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>When looking only at the tissues samples, it appears that there are more genes showing expression changes in muscle than in brain samples. The overlap between muscle and brain samples will uncover gene expression changes specific to the neuromuscular system. The lower rows in <a href="#pone-0001404-t001">Table 1</a> show the number of genes changing and the genes are listed in supplemental <a href="#pone.0001404.s002">Figure S2</a>, <a href="#pone.0001404.s003">S3</a>, <a href="#pone.0001404.s004">S4</a>, <a href="#pone.0001404.s005">S5</a>, <a href="#pone.0001404.s006">S6</a>, <a href="#pone.0001404.s007">S7</a>, <a href="#pone.0001404.s008">S8</a>, <a href="#pone.0001404.s009">S9</a> and <a href="#pone.0001404.s010">S10</a>. Within the down-regulated genes two syntaxin encoding genes are identified. Interestingly, three genes encoding small Heat Shock Proteins are down-regulated especially in the muscle sample.</p>
<a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>To further understand whether SMN is required specifically in the neuromuscular tissues, we have assessed the expression of the smn gene in these tissues versus the rest of the animal using both our array data and Real-Time Polymerase Chain Reaction (RT-PCR) analysis. This latter method compared the concentration of the smn transcript in larval and adult tissues against non-changing control genes. This analysis clearly shows that expression of smn is higher in larval and adult neuromuscular tissues, with especially high expresssion in the nervous system (<a href="#pone-0001404-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001404-g002"><div class="img"><a name="pone-0001404-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span><em>Smn</em> gene expression analysis using two different methods.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A) Results of micro-array hybridisations. “Sample” column lists the different RNA samples used for cRNA preparation. The control and <em>smn</em> mutant relative concentrations are shown. Expression is low in the RNA samples derived from whole animals but is significantly higher in neuromuscular tissues, especially in brain. The expression is reduced by 1.9 and 1.5 fold in the mutant animals in muscle and brain respectively. Note that these tissues are mutant for the somatic gene (a point mutation so transcripts will be produced) while still containing detectable maternal RNA (data not shown). B) Real Time-PCR analysis of <em>smn</em> expression across different tissues. The Bar figure shows the relative abundance of <em>smn</em> transcript in tissues versus either whole animals (thus containing the isolated tissues; all third instar larval stage derived) or versus isolated gut tissue (all adult derived), all from <em>smn +/+</em>. In both Central Nervous System (CNS) and muscle the expression of <em>smn</em> is higher. C) The significance of this finding increases in adults</p>
<span>doi:10.1371/journal.pone.0001404.g002</span></div><a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7"></a><p>The molecular analysis of the NMJs in the <em>smn</em> mutant clearly showed a deregulation of GluR protein accumulation (GluR IIA subunits). Using our data, we assessed the changes in expression of some of the GluR genes (<a href="#pone-0001404-t002">Table 2</a>). GluR IIA and IIB, for which protein is found at the NMJ, are mildly down regulated in <em>smn</em> mutant muscle. GluR III (also knows as IIC) is up-regulated in <em>smn</em> mutant muscle and no change is seen for GluR IID. All have a p value ≤0.05 of change between controls and <em>smn</em> mutant muscle in a Welch t-test.</p>
<div class="figure" id="pone-0001404-t002"><div class="img"><a name="pone-0001404-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Fold changes of GluR genes in <em>smn</em> mutant muscle compared to wild type muscle (p value ≤0.05 of a difference between the 2 conditions).</span></strong></p><span>doi:10.1371/journal.pone.0001404.t002</span></div><a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8"></a><p>It has been suggested that the formation of functional receptors requires at least GluR III and one of either IIA or IIB (and possibly other sub units) <a href="#pone.0001404-Qin1">[11]</a>. The significant up-regulation we observe for GluR III might be a response of the animal to the reduction of receptor activity at the NMJ. The reduction in transcripts for the IIA subunit cannot explain the degree of loss of clustering we see at the neuromuscular junction, arguing the problem in the SMN depleted tissue is not primarily based on transcript reduction.</p>


<h4>Validation of array results</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To assess the validity of our hybridisation results we examined the expression of a number of genes, randomly picked from our list, of differentially expressed genes using Real Time Polymerase Chain Reaction (RT-PCR) amplification. Strong correlation between PCR expression data and micro array values was found (<a href="#pone-0001404-g003">Figure 3</a>). All changes determined by RT-PCR were statistically significant in a Student's t-test (n = 3 and 4 for whole larvae and muscle respectively) and consistent with the direction of change reported by the micro-array analysis.</p>
<div class="figure" id="pone-0001404-g003"><div class="img"><a name="pone-0001404-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Bar graphs showing the gene expression changes across a set of eight genes comparing Affymetrix micro-array hybridisation (top) and Real-Time Polymerase Chain Reaction (bottom) results in whole 3rd instar larvae and muscle.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Sample size was three for larvae and four for muscle. The eight genes were CG6776, CG8836, Cyp4ac3, mthl3, Gip, CG4778, CG4511 and Tsp42Ed. The changes measured this way showed that mRNA synthesis in whole 3rd instar SMN mutant larvae compared to WT larvae for CG6776 was decreased 3.57 fold (P&lt;0.01), CG8836 was increased 14.6 fold (P&lt;0.05), Cyp4ac3 was increased 2.7 fold (P&lt;0.05), mthl3 was increased 8.9 fold (P&lt;0.05). mRNA synthesis in SMN mutant larval muscle for Gip was decreased 2.7 fold (P&lt;0.01), CG4778 was decreased 2.7 fold (&lt;P0.01), CG4511 was decreased 1.6 fold (P&lt;0.05), Tsp42Ed was increased 21.6 fold (P&lt;0.05) . The RT-PCR analysis validates the micro-array data: the overall directions of change are all identical and the fold changes are within the same range</p>
<span>doi:10.1371/journal.pone.0001404.g003</span></div>

<h4>Gene pathway changes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To explore functional changes within neuromuscular tissues we used Gene Ontology (GO) analysis to identify molecular pathways that were up- or down-regulated in <em>smn</em> mutant tissues versus control. <a href="#pone-0001404-g004">Figure 4A</a> shows a selection of categories in muscle and brain tissues where respectively ≥ 20% and ≥ 10% of genes per category are changed with a P value of ≤0.05 (Supplemental <a href="#pone.0001404.s011">Figures S11</a> and <a href="#pone.0001404.s012">S12</a> show the full list of GO terms in muscle and brain tissues, up and down regulated).</p>
<div class="figure" id="pone-0001404-g004"><div class="img"><a name="pone-0001404-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>A) A selection of Gene Ontology terms associated with up (red)- or down (green) -regulated genes in muscle and brain tissues, comparing normal with <em>smn</em> mutant.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>B) A selection of Gene Ontology terms associated with up- or down-regulated genes in 2<sup>nd</sup> and 3<sup>rd</sup> instar larvae, comparing normal with <em>smn</em> mutant, same colour-coding. For both A) and B) full lists of these terms are shown in Supplemental <a href="#pone.0001404.s003">Figure S3</a>.</p>
<span>doi:10.1371/journal.pone.0001404.g004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We find GO terms associated with NMJ physiology and locomotion down regulated in the brain tissue while terms involving general and rRNA metabolism are up regulated. In the muscle samples, terms associated with cellular transport and degradation are down-regulated. In addition neurotransmitter associated terms are down regulated.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Interestingly within the up-regulated genes in the muscle, very clear clustering of terms associated with energy metabolism is observed. Out of the highly significant terms (more than 50% of genes in a selected class are changing) more than half are associated with energy generation and mitochondrial activities. The other very clear grouping that is found up-regulated in the muscle is associated with RNA metabolism. RNA polymerase activity, RNA helicase as well as tRNA synthesis are found amongst the highly significant changes. The established function of SMN in RNA metabolism makes this an interesting observation. This possibly relates to the following. Comparing the muscle and brain samples, the gene expression changes in the muscle are often more significant and a considerable larger number of genes is found changed in muscle versus brain. These observations indicate that the effect of the reduction of SMN activity leads to a more pronounced response in muscle than in brain.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>To study the temporal response to low SMN activity in the whole organism, we compared the GO terms in the 2<sup>nd</sup> and 3<sup>rd</sup> instar samples (using 20% cut off per category). The results are shown in <a href="#pone-0001404-g004">Figure 4B</a> (the full lists are shown in Supplemental <a href="#pone.0001404.s013">Figures S13</a> and <a href="#pone.0001404.s014">14</a>). Most changes in GO terms replicate across these samples, this strongly indicates that these reflect real responses to a reduction in SMN. In the down-regulated genes many terms fall into the category of metabolic, especially carbohydrate, enzyme activities. This indicates that the animals are possibly slowing their metabolic rate. Consistent with this we have observed that the mutant animals become emaciated.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>In the up-regulated genes, interestingly, moving from the earlier time point to the later, terms associated with lytic activities are identified. This might indicate that the muscle tissues are very sensitive to the continued reduction in SMN activity and become degenerative (although no obvious signs are present on the tissue level).</p>


<h4>Genetic interactions</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Our experiments show that expression of certain genes changes dramatically in <em>smn</em> depleted tissues. The function of SMN has been shown to involve the assembly of RNA-protein complexes <a href="#pone.0001404-Yong1">[5]</a>, <a href="#pone.0001404-Monani1">[12]</a>. Rather than direct influences of SMN on gene transcription, the changes in gene expression we observe are therefore likely to be indirect due to the reduction of SMN activity. Such changes could be associated with an attempt within the tissues to maintain normal activity in the presence of depleted SMN function. Genes that are found up-regulated could perform a rescue function in the tissue. Reversely the down regulation of certain genes could be associated with the tissue reducing the activity of pathways that are disadvantageous at low levels of SMN. Agonists and antagonists to the function of these genes could aid the tissues in their struggle against phenotypes caused by low levels of SMN.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>To further understand our findings and progress them into a functional realm we performed genetic experiments. We set out to probe if we could identify direct genetic interactions of loci corresponding to isolated genes, with <em>smn</em>. We combined an <em>smn</em> heterozygous mutant background with either mutant backgrounds (where available) and/or with over-expression constructs of random genes out of our lists. The selection of loci from our lists was random except for the fact that we were limited by the availability of genetic aberrations for selected genes. <a href="#pone-0001404-t003">Table 3</a> shows the genes selected and the results of these genetic tests.</p>
<div class="figure" id="pone-0001404-t003"><div class="img"><a name="pone-0001404-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Results of genetic experiments where <em>smn</em> mutant allele and alleles named in 2<sup>nd</sup> column were combined in trans-heterozygous combination and welfare and survival were tested by counting the number of individuals of each genotype.</span></strong></p><span>doi:10.1371/journal.pone.0001404.t003</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The test we used relies on subtle changes in the survival of the offspring. This test is relevant to the patient situation but it is only partially probing any functional interaction. On the other hand we examined a significant number of individuals in our test, generating a statistically more relevant outcome. We could also not assess if the aberrations we used here actually disrupt (or enhance) the activity of the indicated gene (note that numbered genes are only annotated as a gene and there are no functional data).</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>Despite these calculated set-backs, we clearly identified some genetic interactions with SMN in these tests and these will be discussed below.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Here we present a study of genome-wide transcription changes in a fly SMA model. We show that the technical difficulties in microarray experiments of working with small amounts of tissue material, inherent to working with Drosophila, are surmountable. This has led to statistically relevant and high quality data, which we independently verified using direct transcript level measurements.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>By comparing the transcriptome from whole animal and individual tissues, samples isolated from virtually identical animals, we show that across the animal metabolic changes are taking place when SMN function is depleted. Across time, concomitant with the advent of SMA-like defects, certain metabolic genes are down-regulated. Interestingly, using Gene Ontology analysis, another set of genes that are associated with metabolic processes is identified in the muscle. These terms are mostly up-regulated and include many mitochondrion localized processes. It appears that the depletion of SMN function leads to metabolic changes in the animal.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Small Hsp genes provide activity to rescue cells under duress and three such genes are isolated in muscle and brain tissues as significantly changing in our experiments. Recently two such genes have been pinpointed as leading to a form of Charcot-Marie-Tooth disease, the most common inherited neuromuscular disease <a href="#pone.0001404-Evgrafov1">[13]</a>, <a href="#pone.0001404-Irobi1">[14]</a>. These findings hint at a close and perhaps general interaction of small HSP genes with neuromuscular disorders.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We have previously shown <a href="#pone.0001404-Chan1">[7]</a> that the accumulation of neurotransmitter receptor at the SMN-depleted fly neuromuscular junction is disrupted. Here we show that this is a temporal process, while at the same time the levels of <em>smn</em> mRNA are slowly depleted. Our current experiments show that the transcription of a rate-limiting receptor subunit is increased, possibly reflecting an adjustment made in gene expression to increase receptor complex levels. In addition, our results also show for the first time, by using array results and direct measurements, that the levels of <em>smn</em> mRNA are higher in neuromuscular tissues, consistent with the higher sensitivity of these tissues to SMN depletion. Interestingly, comparing the reduction in GluR protein clusters with the reduction of subunit transcripts, it appears that the clustering problem is not based at the overall level of transcript. The suggested role of SMN in transcript localisation <a href="#pone.0001404-Zhang1">[15]</a> thus indicates that SMN could act directly on transcripts associated with neurotransmitter function.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The Gene Ontology analysis allows the identification of functions affected in the system under study, rather than individual genes. It is clear the energy metabolism is isolated as affected in both types of analysis. In the muscle tissues metabolic processes associated with the mitochondrion are strongly affected. In amyotropic lateral sclerosis the case for an involvement of mitochondrial disturbances, a fatal neurodegenerative disorder, has been made extensively <a href="#pone.0001404-vonLewinski1">[16]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A second grouping of GO terms becomes obvious in the MAPP Finder results. These are mostly associated with neurotransmitter generation, levels and reception, as such involved in synaptic function. In brain tissue, many of the GO terms identified fall within this class, and they both increase as well as decrease in expression levels. In muscle however, the situation is skewed towards genes that are decreased in expression. These results point towards a function of SMN in the maintenance of synaptic function as driven by neurotransmitter generation, transfer and reception. As mentioned above, we have previously identified the loss of neurotransmitter receptor clusters as a highly consistent and relatively early defect in the <em>smn</em> mutant larvae. Clustering of receptors is at least partially driven by pre-synaptic activity <a href="#pone.0001404-Verstreken1">[17]</a> and maintenance of clustering has been shown to be dependent on glutamate (neurotransmitter at the fly neuromuscular junction) <a href="#pone.0001404-Featherstone1">[18]</a>. These results indicate that neurotransmitter metabolism might be a primary defect in the <em>smn</em> mutant and thus in SMA pathology.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Drosophila is a popular model system because of its traceable and rapid genetics. These qualities allow the discovery and characterisation of hitherto unknown functional relations between genes. We have used fly genetics to find new genes interacting with SMN, by selecting loci from our array results and combining these in a simple assay with the <em>smn</em> mutant locus in the fly. The number of genes (and available mutations) does not allow for an extensive study, however even in our limited analysis we have found evidence for genetic interaction with some genes.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Because of the large number of genes representing metabolic functions, our genetic analysis contains several of these. Interestingly however all loci that appear positive in our test (<em>Bc</em> (<em>Black cells</em>), Lag1, Aldh (Aldehyde dehydrogenase), <em>stc</em> (<em>shuttle craft</em>), <em>und</em> (<em>uninitiated</em>) and <em>bowl</em> (<em>brother of odd with entrails limited</em>) are in some way functionally associated with basic metabolism. Some are transcription oriented some are associated with basic enzymatic functions. We would conclude that although our test sample was small that metabolism is an important factor in determining the overall survival of the <em>smn</em> mutant animals and therefore possibly of SMA patients. On a more general level, we feel that this type of analysis, combining genomic technology with genetics will be useful in the functional analysis of (human disease) genes in simple model organisms. In addition, the continuous and rapid increase in molecularly characterised mutations in random genes in the fly, will make the resources reported here useful for further SMA functional studies.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Fly strain used</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Our lab had obtained a mutant in the <em>smn</em> gene by analyzing candidate recessive-lethal mutants in the region where the gene is located based on genome annotation (73A7-9) <a href="#pone.0001404-Chan1">[7]</a>. Sequence analysis showed that one of the many lethal mutations in this region, the mutant l(3) 73Ao, contained a point mutation in the coding region of <em>smn</em>, resulting in a mis-sense mutation. Our lab recombined this <em>smn<sup>73Ao</sup></em> mutant background with an isogenized stock to remove any unwanted lethal mutations in the genetic background (on the same chromosome). The resulting <em>smn<sup>73Ao</sup></em> mutant was homozygous lethal at third larval stages.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The (normal) control used is the isogenic background on which the <em>smn</em> mutant background was recombined.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Both lines were maintained using standard conditions. For collection of material, flies were housed on apple juice plates with yeast in 25°C incubator with 10 hrs light-14 hrs dark cycle. Times larvae were isolated using genetic markers or Green Fluorescent Protein (expressed on the non-mutant allele, homozygous mutants will not light up green).</p>


<h4>Immunolabelling</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For detection of specific proteins at the neuromuscular junction, we used antibodies against Smn protein (described in Thomas et al., under consideration) and syntaxin (see acknowledgements) on dissected tissues (as above) fixed using 4% paraformaldehyde. Incubations of antibodies (including secondary detection antibodies obtained from Molecular Probes/Invitrogen) were done in 1×PBS+0.1% Tween 20 and 0.3% saponin.</p>


<h4>Sample collection</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>In total we collected four samples for RNA isolation. The first sample is 2<sup>nd</sup> instar larvae, second sample is 3<sup>rd</sup> instar larvae. We selected female larvae (on basis of externally visible markers for 3<sup>rd</sup> instar, for 2<sup>nd</sup> instar we took 10 larvae total) in order to avoid any sex related gene expression changes between samples. The larvae were rinsed twice in 1xPBS (to wash off any bacteria or yeast) before RNA preparation (see below). The third and fourth samples are the 3<sup>rd</sup> instar female larval brains and (abdominal ventral) muscle. Both tissues were dissected from the larvae in 1xPBS, and stored in RNAlater RNA Stabilization Reagent at −80°C. . Three biological replicates of larvae were prepared and four biological replicates of brain and muscle.</p>


<h4>RNA preparation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Total RNA was extracted from 2<sup>nd</sup> and 3<sup>rd</sup> instar larvae using the Qiagen RNeasy Mini Kit. The animals were homogenized in 350 µL RLT buffer (+ß-Mercapto-Ethanol: ME) with a hand held homogenizer (5 mm probe blades) and the manufacturers protocol was followed for further isolation.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Total RNA was extracted from brain and muscle tissues using the Qiagen RNeasy Micro Kit, as above.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>RNA concentration and integrity were assessed by absorbance at 260 and 280 nm and by an Agilent Bioanalyzer RNA 6000 Nano Assay (data not shown).</p>


<h4>Sample labelling and hybridisation to Affymetrix array</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Samples were prepared for Affymetrix analysis from whole 2<sup>nd</sup> and 3<sup>rd</sup> instar larvae using 1 ug starting RNA. The Affymetrix One-Cycle Target Labelling and Control kit (P/N900493) was used according to protocol 701027 Rev. 4 in the Affymetrix Eukaryotic Sample and Array Processing Manual. For Drosophila larval brain and muscle 100 ng and 40 ng starting RNA were used respectively. To successfully generate sufficient labelled cRNA for microarray analysis two rounds of amplification were necessary. For this the Affymetrix Two-Cycle Target Labelling and Control kit was used (P/N 900494) according to the manufacturer's instructions described in the Affymetrix Eukaryotic Sample and Array Processing Manual version 701027 Rev.5. For the first cycle of <em>in vitro</em> transcription the Ambion MEGAscript T7 kit (P/N 1334) was used as detailed in the Affymetrix manual. Labelled cRNA was hybridised to Drosophila 1 chips and arrays were washed and then scanned using the Affymetrix GeneChip Scanner3000.</p>


<h4>Analysis of differential gene expression</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Microarray data were GCRMA normalised independently for each tissue using GeneSpring GX (Agilent). Differentially expressed genes were identified using a Welch t test with a p value cut off of ≤0.05 and a fold change difference between wild type and mutant <em>smn</em> of ≥1.5.</p>


<h4>Real-Time RT-PCR (SYBR green)</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Total RNA from WT and <em>smn</em> mutant whole 3<sup>rd</sup> instar larvae and muscle was used as a template for RT-PCR to independently assess the validity of Affymetrix GeneChip data. 10 ng 3<sup>rd</sup> instar total larval RNA and 1 ng larval muscle RNA per sample was used to synthesise first-strand cDNA, using random hexamers and SuperScript III reverse transcriptase (Invitrogen). The cDNA was used for quantitative real-time polymerase chain reaction (PCR) amplification with Sybr Green chemistry (Applied Biosystems). Primer Express software (Applied Biosystems) was used to design primers from the RefSeq sequences. The target gene names, RefSeq numbers and primer sequences are shown in <a href="#pone-0001404-t004">Table 4</a>. Triplicate cDNA template samples were amplified and analyzed in the Prism 7000 sequence detection system (Applied Biosystems). The thermal cycle conditions were 50°C for 2 min, 95°C for 10 min, 40 cycles of 15 s at 95°C followed by 1 min at 60°C. Immediately after the amplification, melt curve protocols were performed to ensure that primer-dimers and other non-specific products had been minimized or eliminated. A standard curve of cycle thresholds using serial dilutions of cDNA samples was established and used to calculate the relative abundance of the target gene in WT and mutant <em>smn</em> samples. Values were normalized to gamma-Tubulin (at map position 23C), we used this gene as its expression was not changing in our Affymetrix data. Minus RT samples tested simultaneously with experimental samples by quantitative RT-PCR with Sybr Green consistently yielded no amplification below 30 cycles using the above protocol.</p>
<div class="figure" id="pone-0001404-t004"><div class="img"><a name="pone-0001404-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001404" data-uri="info:doi/10.1371/journal.pone.0001404.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001404.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001404.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001404.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001404.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Probe and Primer Sequences for Real-Time PCR</span></strong></p><span>doi:10.1371/journal.pone.0001404.t004</span></div>

<h4>Real-time RT-PCR (Probe)</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Real-time quantitative PCR was performed by using TaqMan TAMRA probe chemistry on the ABI Prism 7000 Sequence Detector System (Applied Biosystems). Primers and probes were designed using Primer Express V2.0 software. Thermal Cycling parameters were 95°C for 10 min, followed by 40 cycles of 95°C for 1 minute, 59°C for 1 minute, and 72°C for 45 s. Aliquots of 50 µl consisting of primers, probes, cDNA and TaqMAN RT PCR master mix were run in triplicate on the Applied Biosystems 7000 sequence detection system. Primers were used at 900 nM concentration and probes at 20 nM concentration for both target and control. A serial dilution for each target gene was run on each plate in triplicate. The dilution series consisted of cDNA at 2 ng 1 ng, 0.5 ng 0.2 ng, 0.1 ng, 0.05 ng, and 0.02 ng concentrations. Data was analysed using the comparative relative quantification method and samples were normalised to both Actin42A and GAPDH. Three plates were run for this experiment.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001404"></div><a name="pone.0001404.s001" id="pone.0001404.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">cRNA yields and quality from decreasing amounts of starting RNA. A) The yield of cRNA falls steeply when the amount of starting material goes below 1 ng. B) The size distribution is smaller when the 2 cycle labelling kit is used for 100 ng RNA compared with cRNA transcripts generated using the 1 cycle kit for 5 ug starting RNA. The smaller the amount of starting RNA the smaller the transcripts that are generated. C) Scatter plots show some genes decrease in signal when smaller amounts starting amounts of RNA are used. This was shown to be due to 5′ truncation of cRNA transcripts, preventing complete hybridisation to the rarer 5′ probe sets (data not shown). However the loss of gene expression signal resulting from reduced starting material may not always translate into changed gene expression changes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.70 MB DOC)</p>
<a name="pone.0001404.s002" id="pone.0001404.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Genes identified in brain tissue, not in 2nd and 3rd instar larvae, corresponding to 1st and 5th rows in <a href="#pone-0001404-t001">Table 1a</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0001404.s003" id="pone.0001404.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Genes identified in 2nd instar larvae and brain tissue, not in 3rd instar larvae, corresponding to 2nd and 6th rows in <a href="#pone-0001404-t001">Table 1a</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001404.s004" id="pone.0001404.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Genes identified in 3rd instar larvae and brain, not in 2nd instar larvae, corresponding to 3rd and 7th rows <a href="#pone-0001404-t001">Table 1a</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001404.s005" id="pone.0001404.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Genes identified in brain, 2nd and 3rd instar larvae, corresponding to 4th and 8th rows in <a href="#pone-0001404-t001">Table 1a</a></p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001404.s006" id="pone.0001404.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Genes identified in muscle tissue, not in 2nd and 3rd instar larvae, corresponding to 1st and 5th rows in <a href="#pone-0001404-t001">Table 1b</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.26 MB XLS)</p>
<a name="pone.0001404.s007" id="pone.0001404.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s007">Figure S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Genes identified in 2nd instar larvae and muscle tissue, not in 3rd instar larvae, corresponding to 2nd and 6th rows in <a href="#pone-0001404-t001">Table 1b</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001404.s008" id="pone.0001404.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s008">Figure S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Genes identified in 3rd instar larvae and muscle, not in 2nd instar larvae, corresponding to 3rd and 7th rows in <a href="#pone-0001404-t001">Table 1b</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001404.s009" id="pone.0001404.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s009">Figure S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Genes identified in muscle, 2nd and 3rd instar larvae, corresponding to 4th and 8th rows in <a href="#pone-0001404-t001">Table 1b</a></p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001404.s010" id="pone.0001404.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s010">Figure S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Genes identified as in common between brain and muscle samples, corresponding to last two rows in <a href="#pone-0001404-t001">Table 1c</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.06 MB XLS)</p>
<a name="pone.0001404.s011" id="pone.0001404.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s011">Figure S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">Gene Ontology (GO) collective terms for genes that are differentially regulated in SMN mutant muscle with a 1.5 fold cut-off. Terms are selected only if 20% or more of genes in a GO term are changed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.08 MB XLS)</p>
<a name="pone.0001404.s012" id="pone.0001404.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s012">Figure S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Gene Ontology (GO) collective terms for genes that are differentially regulated in mutant SMN brain with a 1.5 fold cut-off. GO terms are selected only if 10% or more of genes in a GO term are changed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.03 MB PPT)</p>
<a name="pone.0001404.s013" id="pone.0001404.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s013">Figure S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Gene Ontology (GO) collective terms for genes that are differentially regulated in SMN mutant 2nd instar larvae with a 1.5 fold cut-off. Terms are selected only if 20% or more of genes in a GO term are changed</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001404.s014" id="pone.0001404.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001404.s014">Figure S14. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">Gene Ontology (GO) collective terms for genes that are differentially regulated in SMN mutant 3rd instar larvae with a 1.5 fold cut-off. Terms are selected only if 20% or more of genes in a GO term are changed</p>
<p class="siDoi">doi:10.1371/journal.pone.0001404.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The (monoclonal against syntaxin) developed by dr. Seymour Benzer was obtained from the Developmental Studies Hybridoma Bank developed under the auspices of the NICHD and maintained by The University of Iowa, Department of Biological Sciences, Iowa City, IA 52242. We would like to thank Prof. Kay E. Davies for allowing us to use the OXION facilities.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: Mv SG SL. Performed the experiments: DB AS SG SL HB. Analyzed the data: AS SG SL HB. Contributed reagents/materials/analysis tools: Mv SL. Wrote the paper: Mv.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001404-Lefebvre1" id="pone.0001404-Lefebvre1"></a>Lefebvre S, Burglen L, Reboullet S, Clermont O, Burlet P, et al.  (1995) Identification and characterization of a spinal muscular atrophy-determining gene. Cell  80: 155–165.  <ul class="find" data-citedArticleID="1038800" data-doi="10.1016/0092-8674(95)90460-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90460-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+a+spinal+muscular+atrophy-determining+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+a+spinal+muscular+atrophy-determining+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001404-Lefebvre2" id="pone.0001404-Lefebvre2"></a>Lefebvre S, Burlet P, Liu Q, Bertrandy S, Clermont O, et al.  (1997) Correlation between severity and SMN protein level in spinal muscular atrophy. Nature Genetics  16: 265–269.  <ul class="find" data-citedArticleID="1038802" data-doi="10.1038/ng0797-265"><li><a href="http://dx.doi.org/10.1038/ng0797-265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+between+severity+and+SMN+protein+level+in+spinal+muscular+atrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+between+severity+and+SMN+protein+level+in+spinal+muscular+atrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001404-Schrank1" id="pone.0001404-Schrank1"></a>Schrank B, Gotz R, Gunnersen JM, Ure JM, Toyka KV, et al.  (1997) Inactivation of the survival motor neuron gene, a candidate gene for human spinal muscular atrophy, leads to massive cell death in early mouse embryos. Proceedings of the National Academy of Sciences of the United States of America  94: 9920–9925.  <ul class="find" data-citedArticleID="1038808" data-doi="10.1073/pnas.94.18.9920"><li><a href="http://dx.doi.org/10.1073/pnas.94.18.9920" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+the+survival+motor+neuron+gene%2C+a+candidate+gene+for+human+spinal+muscular+atrophy%2C+leads+to+massive+cell+death+in+early+mouse+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+the+survival+motor+neuron+gene%2C+a+candidate+gene+for+human+spinal+muscular+atrophy%2C+leads+to+massive+cell+death+in+early+mouse+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001404-Bergin1" id="pone.0001404-Bergin1"></a>Bergin A, Kim G, Price DL, Sisodia SS, Lee MK, et al.  (1997) Identification and characterization of a mouse homologue of the spinal muscular atrophy-determining gene, survival motor neuron. Gene  204: 47–53.  <ul class="find" data-citedArticleID="1038786" data-doi="10.1016/s0378-1119(97)00510-6"><li><a href="http://dx.doi.org/10.1016/s0378-1119(97)00510-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+a+mouse+homologue+of+the+spinal+muscular+atrophy-determining+gene%2C+survival+motor+neuron." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+a+mouse+homologue+of+the+spinal+muscular+atrophy-determining+gene%2C+survival+motor+neuron.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001404-Yong1" id="pone.0001404-Yong1"></a>Yong J, Wan L, Dreyfuss G (2004) Why do cells need an assembly machine for RNA-protein complexes? Trends in cell biology  14: 226–232.  <ul class="find" data-citedArticleID="1038814" data-doi="10.1016/j.tcb.2004.03.010"><li><a href="http://dx.doi.org/10.1016/j.tcb.2004.03.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Why+do+cells+need+an+assembly+machine+for+RNA-protein+complexes%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Why+do+cells+need+an+assembly+machine+for+RNA-protein+complexes%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001404-Gubitz1" id="pone.0001404-Gubitz1"></a>Gubitz AK, Feng W, Dreyfuss G (2004) The SMN complex. Experimental cell research  296: 51–56.  <ul class="find" data-citedArticleID="1038794" data-doi="10.1016/j.yexcr.2004.03.022"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2004.03.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SMN+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SMN+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001404-Chan1" id="pone.0001404-Chan1"></a>Chan YB, Miguel Aliaga I, Franks C, Thomas N, Trulzsch B, et al.  (2003) Neuromuscular defects in a Drosophila survival motor neuron gene mutant. Human molecular genetics  12: 1367–1376.  <ul class="find" data-citedArticleID="1038788" data-doi="10.1093/hmg/ddg157"><li><a href="http://dx.doi.org/10.1093/hmg/ddg157" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuromuscular+defects+in+a+Drosophila+survival+motor+neuron+gene+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuromuscular+defects+in+a+Drosophila+survival+motor+neuron+gene+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001404-Arnold1" id="pone.0001404-Arnold1"></a>Arnold AS, Gueye M, Guettier-Sigrist S, Courdier-Fruh I, Coupin G, Poindron P, Gies JP (2004) Reduced expression of nicotinic AChRs in myotubes from spinal muscular atrophy 1 patients. Laboratory Investigation 1–8.  <ul class="find" data-citedArticleID="1038784" data-doi="10.1038/labinvest.3700163"><li><a href="http://dx.doi.org/10.1038/labinvest.3700163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+expression+of+nicotinic+AChRs+in+myotubes+from+spinal+muscular+atrophy+1+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+expression+of+nicotinic+AChRs+in+myotubes+from+spinal+muscular+atrophy+1+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001404-Le1" id="pone.0001404-Le1"></a>Le TT, Pham LT, Butchbach ME, Zhang HL, Monani UR, et al.  (2005) SMNDelta7, the major product of the centromeric survival motor neuron (SMN2) gene, extends survival in mice with spinal muscular atrophy and associates with full-length SMN. Human molecular genetics  14: 845–857.  <ul class="find" data-citedArticleID="1038798" data-doi="10.1093/hmg/ddi078"><li><a href="http://dx.doi.org/10.1093/hmg/ddi078" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMNDelta7%2C+the+major+product+of+the+centromeric+survival+motor+neuron+%28SMN2%29+gene%2C+extends+survival+in+mice+with+spinal+muscular+atrophy+and+associates+with+full-length+SMN." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMNDelta7%2C+the+major+product+of+the+centromeric+survival+motor+neuron+%28SMN2%29+gene%2C+extends+survival+in+mice+with+spinal+muscular+atrophy+and+associates+with+full-length+SMN.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001404-Shedden1" id="pone.0001404-Shedden1"></a>Shedden K, Chen W, Kuick R, Ghosh D, Macdonald J, et al.  (2005) Comparison of seven methods for producing Affymetrix expression scores based on False Discovery Rates in disease profiling data. BMC bioinformatics  6: 26.  <ul class="find" data-citedArticleID="1038810"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+seven+methods+for+producing+Affymetrix+expression+scores+based+on+False+Discovery+Rates+in+disease+profiling+data.&amp;auth=&amp;atitle=Comparison+of+seven+methods+for+producing+Affymetrix+expression+scores+based+on+False+Discovery+Rates+in+disease+profiling+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+seven+methods+for+producing+Affymetrix+expression+scores+based+on+False+Discovery+Rates+in+disease+profiling+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+seven+methods+for+producing+Affymetrix+expression+scores+based+on+False+Discovery+Rates+in+disease+profiling+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001404-Qin1" id="pone.0001404-Qin1"></a>Qin G, Schwarz T, Kittel RJ, Schmid A, Rasse TM, et al.  (2005) Four different subunits are essential for expressing the synaptic glutamate receptor at neuromuscular junctions of Drosophila. Journal of neuroscience the official journal of the Society for Neuroscience ISSN (ELECTRONIC VERSION): 1529 2401  25: 3209–3218.  <ul class="find" data-citedArticleID="1038806" data-doi="10.1523/jneurosci.4194-04.2005"><li><a href="http://dx.doi.org/10.1523/jneurosci.4194-04.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Four+different+subunits+are+essential+for+expressing+the+synaptic+glutamate+receptor+at+neuromuscular+junctions+of+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Four+different+subunits+are+essential+for+expressing+the+synaptic+glutamate+receptor+at+neuromuscular+junctions+of+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001404-Monani1" id="pone.0001404-Monani1"></a>Monani UR (2005) Spinal muscular atrophy: a deficiency in a ubiquitous protein; a motor neuron-specific disease. Neuron ISSN (PRINT VERSION): 0896 6273  48: 885–895.  <ul class="find" data-citedArticleID="1038804" data-doi="10.1016/j.neuron.2005.12.001"><li><a href="http://dx.doi.org/10.1016/j.neuron.2005.12.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spinal+muscular+atrophy%3A+a+deficiency+in+a+ubiquitous+protein%3B+a+motor+neuron-specific+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spinal+muscular+atrophy%3A+a+deficiency+in+a+ubiquitous+protein%3B+a+motor+neuron-specific+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001404-Evgrafov1" id="pone.0001404-Evgrafov1"></a>Evgrafov OV, Mersiyanova I, Irobi J, Van Den Bosch L, Dierick I, et al.  (2004) Mutant small heat-shock protein 27 causes axonal Charcot-Marie-Tooth disease and distal hereditary motor neuropathy. Nature genetics  36: 602–606.  <ul class="find" data-citedArticleID="1038790" data-doi="10.1038/ng1354"><li><a href="http://dx.doi.org/10.1038/ng1354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutant+small+heat-shock+protein+27+causes+axonal+Charcot-Marie-Tooth+disease+and+distal+hereditary+motor+neuropathy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutant+small+heat-shock+protein+27+causes+axonal+Charcot-Marie-Tooth+disease+and+distal+hereditary+motor+neuropathy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001404-Irobi1" id="pone.0001404-Irobi1"></a>Irobi J, Van Impe K, Seeman P, Jordanova A, Dierick I, et al.  (2004) Hot-spot residue in small heat-shock protein 22 causes distal motor neuropathy. Nature genetics  36: 597–601.  <ul class="find" data-citedArticleID="1038796" data-doi="10.1038/ng1328"><li><a href="http://dx.doi.org/10.1038/ng1328" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hot-spot+residue+in+small+heat-shock+protein+22+causes+distal+motor+neuropathy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hot-spot+residue+in+small+heat-shock+protein+22+causes+distal+motor+neuropathy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001404-Zhang1" id="pone.0001404-Zhang1"></a>Zhang HL, Pan F, Hong D, Shenoy SM, Singer RH, et al.  (2003) Active transport of the survival motor neuron protein and the role of exon-7 in cytoplasmic localization. Journal of neuroscience the official journal of the Society for Neuroscience  23: 6627–6637.  <ul class="find" data-citedArticleID="1038816"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Active+transport+of+the+survival+motor+neuron+protein+and+the+role+of+exon-7+in+cytoplasmic+localization.&amp;auth=&amp;atitle=Active+transport+of+the+survival+motor+neuron+protein+and+the+role+of+exon-7+in+cytoplasmic+localization." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Active+transport+of+the+survival+motor+neuron+protein+and+the+role+of+exon-7+in+cytoplasmic+localization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Active+transport+of+the+survival+motor+neuron+protein+and+the+role+of+exon-7+in+cytoplasmic+localization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001404-vonLewinski1" id="pone.0001404-vonLewinski1"></a>von Lewinski F, Keller BU (2005) Ca2+, mitochondria and selective motoneuron vulnerability: implications for ALS. Trends in neurosciences  28: 494–500.  <ul class="find" data-citedArticleID="1038818" data-doi="10.1016/j.tins.2005.07.001"><li><a href="http://dx.doi.org/10.1016/j.tins.2005.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ca2%2B%2C+mitochondria+and+selective+motoneuron+vulnerability%3A+implications+for+ALS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ca2%2B%2C+mitochondria+and+selective+motoneuron+vulnerability%3A+implications+for+ALS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001404-Verstreken1" id="pone.0001404-Verstreken1"></a>Verstreken P, Bellen HJ (2002) Meaningless minis? Mechanisms of neurotransmitter-receptor clustering. Trends in neurosciences  25: 383–385.  <ul class="find" data-citedArticleID="1038812" data-doi="10.1016/s0166-2236(02)02197-5"><li><a href="http://dx.doi.org/10.1016/s0166-2236(02)02197-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Meaningless+minis%3F+Mechanisms+of+neurotransmitter-receptor+clustering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Meaningless+minis%3F+Mechanisms+of+neurotransmitter-receptor+clustering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001404-Featherstone1" id="pone.0001404-Featherstone1"></a>Featherstone DE, Rushton EM, Hilderbrand-Chae M, Phillips AM, Jackson FR, et al.  (2000) Presynaptic glutamic acid decarboxylase is required for induction of the postsynaptic receptor field at a glutamatergic synapse. Neuron  27: 71–84.  <ul class="find" data-citedArticleID="1038792" data-doi="10.1016/s0896-6273(00)00010-6"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)00010-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Presynaptic+glutamic+acid+decarboxylase+is+required+for+induction+of+the+postsynaptic+receptor+field+at+a+glutamatergic+synapse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Presynaptic+glutamic+acid+decarboxylase+is+required+for+induction+of+the+postsynaptic+receptor+field+at+a+glutamatergic+synapse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.XML" value="97895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.PDF" value="437973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g001.PNG_L" value="2135939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g001.PNG_M" value="328815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g001.PNG_S" value="13310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g001.TIF" value="2788534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g001.PNG_I" value="96947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t001.PNG_L" value="162919"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t001.PNG_M" value="133608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t001.PNG_S" value="19574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t001.TIF" value="796382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t001.PNG_I" value="93637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g002.PNG_L" value="623016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g002.PNG_M" value="73227"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g002.PNG_S" value="10844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g002.TIF" value="2868668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g002.PNG_I" value="36341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t002.PNG_L" value="60262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t002.PNG_M" value="77517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t002.PNG_S" value="12329"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t002.TIF" value="251644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t002.PNG_I" value="26665"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g003.PNG_L" value="413089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g003.PNG_M" value="32612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g003.PNG_S" value="7830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g003.TIF" value="727956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g003.PNG_I" value="16415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g004.PNG_L" value="441951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g004.PNG_M" value="171830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g004.PNG_S" value="10815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g004.TIF" value="667376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.g004.PNG_I" value="64188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t003.PNG_L" value="238055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t003.PNG_M" value="107701"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t003.PNG_S" value="20501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t003.TIF" value="1211316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t003.PNG_I" value="151743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t004.PNG_L" value="147962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t004.PNG_M" value="197565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t004.PNG_S" value="21866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t004.TIF" value="651956"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.t004.PNG_I" value="81342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s001.DOC" value="695296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s002.XLS" value="87552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s003.XLS" value="19456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s004.XLS" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s005.XLS" value="18944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s006.XLS" value="256512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s007.XLS" value="24576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s008.XLS" value="30208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s009.XLS" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s010.XLS" value="59904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s011.XLS" value="81408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s012.PPT" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s013.XLS" value="24064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001404.s014.XLS" value="18944"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001404&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001404" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001404&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001404&volume=&issue=&title=Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets&author_name=Sheena%20Lee%2C%20Arzu%20Sayin%2C%20Stuart%20Grice%2C%20Howard%20Burdett%2C%20Dilair%20Baban%2C%20Marcel%20van%20den%20Heuvel&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001404" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404&amp;t=Genome-Wide%20Expression%20Analysis%20of%20a%20Spinal%20Muscular%20Atrophy%20Model%3A%20Towards%20Discovery%20of%20New%20Drug%20Targets" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404&title=Genome-Wide%20Expression%20Analysis%20of%20a%20Spinal%20Muscular%20Atrophy%20Model%3A%20Towards%20Discovery%20of%20New%20Drug%20Targets&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404&amp;title=Genome-Wide%20Expression%20Analysis%20of%20a%20Spinal%20Muscular%20Atrophy%20Model%3A%20Towards%20Discovery%20of%20New%20Drug%20Targets" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001404&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001404';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome-Wide%20Expression%20Analysis%20of%20a%20Spinal%20Muscular%20Atrophy%20Model%3A%20Towards%20Discovery%20of%20New%20Drug%20Targets http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001404" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001404" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->
<div class="block">
    <div class="header">
        <h3>Related PLOS Articles</h3>
    </div>
      <p><a href = "/article/info:doi/10.1371/annotation/f45a5dc0-e317-4886-8c5b-1569c79a5f08" title="Read Open-Access Article">Correction: Genome-Wide Expression Analysis of a Spinal Muscular Atrophy Model: Towards Discovery of New Drug Targets</a></p>
</div>


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+genomics%22" title="Search for articles in the subject area:'Animal genomics'"><div class="flagText">Animal genomics</div></a>
              <div data-categoryid="17455" data-articleid="25638"
                   data-categoryname="Animal genomics"
                   class="flagImage" title="Flag 'Animal genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25638"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25638"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Larvae%22" title="Search for articles in the subject area:'Larvae'"><div class="flagText">Larvae</div></a>
              <div data-categoryid="17821" data-articleid="25638"
                   data-categoryname="Larvae"
                   class="flagImage" title="Flag 'Larvae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Muscle+proteins%22" title="Search for articles in the subject area:'Muscle proteins'"><div class="flagText">Muscle proteins</div></a>
              <div data-categoryid="20221" data-articleid="25638"
                   data-categoryname="Muscle proteins"
                   class="flagImage" title="Flag 'Muscle proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Muscle+tissue%22" title="Search for articles in the subject area:'Muscle tissue'"><div class="flagText">Muscle tissue</div></a>
              <div data-categoryid="49177" data-articleid="25638"
                   data-categoryname="Muscle tissue"
                   class="flagImage" title="Flag 'Muscle tissue' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Neurotransmitters%22" title="Search for articles in the subject area:'Neurotransmitters'"><div class="flagText">Neurotransmitters</div></a>
              <div data-categoryid="19883" data-articleid="25638"
                   data-categoryname="Neurotransmitters"
                   class="flagImage" title="Flag 'Neurotransmitters' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="25638"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9260'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5716'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8582&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=14807">Publisher's Note: Modification to Author Contributions</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
