

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001844"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001844" />

    <meta name="citation_title" content="Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides"/>
    <meta itemprop="name" content="Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides"/>

      <meta name="citation_author" content="Keng Boon Wee"/>
            <meta name="citation_author_institution" content="Bioinformatics Institute, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="Graduate School for Integrative Sciences and Engineering, Centre for Life Sciences, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Zacharias Aloysius Dwi Pramono"/>
            <meta name="citation_author_institution" content="Neuromuscular Research Laboratory, National Neuroscience Institute, Singapore, Singapore"/>
      <meta name="citation_author" content="Jian Li Wang"/>
            <meta name="citation_author_institution" content="Neuromuscular Research Laboratory, National Neuroscience Institute, Singapore, Singapore"/>
      <meta name="citation_author" content="Karl F. MacDorman"/>
            <meta name="citation_author_institution" content="School of Informatics, Indiana University, Indianapolis, Indiana, United States of America"/>
      <meta name="citation_author" content="Poh San Lai"/>
            <meta name="citation_author_institution" content="Department of Paediatrics, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Woon Chee Yee"/>
            <meta name="citation_author_institution" content="Neuromuscular Research Laboratory, National Neuroscience Institute, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="Department of Neurology, National Neuroscience Institute, Singapore, Singapore"/>

    <meta name="citation_date" content="2008/3/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001844.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1844"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Antisense oligonucleotides in the treatment of Duchenne muscular dystrophy: Where are we now?; citation_author=SD Wilton; citation_author=S Fletcher; citation_journal_title=Neuromuscul Disord; citation_volume=15; citation_number=1; citation_pages=399-402; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RNA splicing manipulation: strategies to modify gene expression for a variety of therapeutic outcomes.; citation_author=SD Wilton; citation_author=S Fletcher; citation_journal_title=Curr Gene Ther; citation_volume=5; citation_number=2; citation_pages=467-483; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rescue of dystrophin mRNA of Duchenne muscular dystrophy by inducing exon skipping.; citation_author=M Matsuo; citation_author=Y Takeshima; citation_journal_title=Acta Myol; citation_volume=24; citation_number=3; citation_pages=110-114; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Advances in Duchenne muscular dystrophy gene therapy.; citation_author=JC van Deutekom; citation_author=GJ van Ommen; citation_journal_title=Nat Rev Genet; citation_volume=4; citation_number=4; citation_pages=774-783; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Duchenne and Becker muscular dystrophy: from gene diagnosis to molecular therapy.; citation_author=M Matsuo; citation_journal_title=IUBMB Life; citation_volume=53; citation_number=5; citation_pages=147-152; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Antisense Oligonucleotide-induced Exon Skipping Across the Human Dystrophin Gene Transcript.; citation_author=SD Wilton; citation_author=AM Fall; citation_author=PL Harding; citation_author=G McClorey; citation_author=C Coleman; citation_journal_title=Mol Ther; citation_volume=15; citation_number=6; citation_pages=1288-1296; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Exploring the frontiers of therapeutic exon skipping for Duchenne muscular dystrophy by double targeting within one or multiple exons.; citation_author=A Aartsma-Rus; citation_author=WE Kaman; citation_author=R Weij; citation_author=JT den Dunnen; citation_author=GJ van Ommen; citation_journal_title=Mol Ther; citation_volume=14; citation_number=7; citation_pages=401-407; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Antisense-induced multiexon skipping for Duchenne muscular dystrophy makes more sense.; citation_author=A Aartsma-Rus; citation_author=AA Janson; citation_author=WE Kaman; citation_author=M Bremmer-Bout; citation_author=GJ van Ommen; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=8; citation_pages=83-92; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Targeted exon skipping as a potential gene correction therapy for Duchenne muscular dystrophy.; citation_author=A Aartsma-Rus; citation_author=M Bremmer-Bout; citation_author=AA Janson; citation_author=JT den Dunnen; citation_author=GJ van Ommen; citation_journal_title=Neuromuscul Disord; citation_volume=12; citation_number=9; citation_pages=S71-S77; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of 114 exon-internal AONs for targeted DMD exon skipping: indication for steric hindrance of SR protein binding sites.; citation_author=A Aartsma-Rus; citation_author=CL De Winter; citation_author=AA Janson; citation_author=WE Kaman; citation_author=GJ van Ommen; citation_journal_title=Oligonucleotides; citation_volume=15; citation_number=10; citation_pages=284-297; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Duchenne/Becker muscular dystrophy: from molecular diagnosis to gene therapy.; citation_author=M Matsuo; citation_journal_title=Brain Dev; citation_volume=18; citation_number=11; citation_pages=167-172; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Induction of exon skipping of the dystrophin transcript in lymphoblastoid cells by transfecting an antisense oligodeoxynucleotide complementary to an exon recognition sequence.; citation_author=ZA Pramono; citation_author=Y Takeshima; citation_author=H Alimsardjono; citation_author=A Ishii; citation_author=S Takeda; citation_journal_title=Biochem Biophys Res Commun; citation_volume=226; citation_number=12; citation_pages=445-449; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Chimeric RNA/ethylene-bridged nucleic acids promote dystrophin expression in myocytes of duchenne muscular dystrophy by inducing skipping of the nonsense mutation-encoding exon.; citation_author=A Surono; citation_author=T Van Khanh; citation_author=Y Takeshima; citation_author=H Wada; citation_author=M Yagi; citation_journal_title=Hum Gene Ther; citation_volume=15; citation_number=13; citation_pages=749-757; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Repair of a splicing defect in erythroid cells from patients with beta-thalassemia/HbE disorder.; citation_author=T Suwanmanee; citation_author=H Sierakowska; citation_author=S Fucharoen; citation_author=R Kole; citation_journal_title=Mol Ther; citation_volume=6; citation_number=14; citation_pages=718-726; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Restoration of correct splicing of thalassemic beta-globin pre-mRNA by modified U1 snRNAs.; citation_author=L Gorman; citation_author=DR Mercatante; citation_author=R Kole; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=15; citation_pages=35914-35919; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Therapeutic potential of antisense oligonucleotides as modulators of alternative splicing.; citation_author=P Sazani; citation_author=R Kole; citation_journal_title=J Clin Invest; citation_volume=112; citation_number=16; citation_pages=481-486; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Restoration of correct splicing in thalassemic pre-mRNA by antisense oligonucleotides.; citation_author=Z Dominski; citation_author=R Kole; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=17; citation_pages=8673-8677; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Aberrant splicing in the ocular albinism type 1 gene (OA1/GPR143) is corrected in vitro by morpholino antisense oligonucleotides.; citation_author=F Vetrini; citation_author=R Tammaro; citation_author=S Bondanza; citation_author=EM Surace; citation_author=A Auricchio; citation_journal_title=Hum Mutat; citation_volume=27; citation_number=18; citation_pages=420-426; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modification of alternative splicing by antisense oligonucleotides as a potential chemotherapy for cancer and other diseases.; citation_author=DR Mercatante; citation_author=P Sazani; citation_author=R Kole; citation_journal_title=Curr Cancer Drug Targets; citation_volume=1; citation_number=19; citation_pages=211-230; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Induced dystrophin exon skipping in human muscle explants.; citation_author=G McClorey; citation_author=AM Fall; citation_author=HM Moulton; citation_author=PL Iversen; citation_author=JE Rasko; citation_journal_title=Neuromuscul Disord; citation_volume=16; citation_number=20; citation_pages=583-590; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Antisense oligonucleotide-induced exon skipping restores dystrophin expression in vitro in a canine model of DMD.; citation_author=G McClorey; citation_author=HM Moulton; citation_author=PL Iversen; citation_author=S Fletcher; citation_author=SD Wilton; citation_journal_title=Gene Ther; citation_volume=13; citation_number=21; citation_pages=1373-1381; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Therapeutic restoration of dystrophin expression in Duchenne muscular dystrophy.; citation_author=DJ Wells; citation_journal_title=J Muscle Res Cell Motil; citation_volume=27; citation_number=22; citation_pages=387-398; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Systemic delivery of morpholino oligonucleotide restores dystrophin expression bodywide and improves dystrophic pathology.; citation_author=J Alter; citation_author=F Lou; citation_author=A Rabinowitz; citation_author=H Yin; citation_author=J Rosenfeld; citation_journal_title=Nat Med; citation_volume=12; citation_number=23; citation_pages=175-177; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Induction of revertant fibres in the mdx mouse using antisense oligonucleotides.; citation_author=AM Fall; citation_author=R Johnsen; citation_author=K Honeyman; citation_author=P Iversen; citation_author=S Fletcher; citation_journal_title=Genet Vaccines Ther; citation_volume=4; citation_number=24; citation_pages=3; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Systemic delivery of antisense oligoribonucleotide restores dystrophin expression in body-wide skeletal muscles.; citation_author=QL Lu; citation_author=A Rabinowitz; citation_author=YC Chen; citation_author=T Yokota; citation_author=H Yin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=25; citation_pages=198-203; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Intravenous infusion of an antisense oligonucleotide results in exon skipping in muscle dystrophin mRNA of Duchenne muscular dystrophy.; citation_author=Y Takeshima; citation_author=M Yagi; citation_author=H Wada; citation_author=K Ishibashi; citation_author=A Nishiyama; citation_journal_title=Pediatr Res; citation_volume=59; citation_number=26; citation_pages=690-694; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=128th ENMC International Workshop on ‘Preclinical optimization and Phase I/II Clinical Trials Using Antisense Oligonucleotides in Duchenne Muscular Dystrophy’ 22-24 October 2004, Naarden, The Netherlands.; citation_author=F Muntoni; citation_author=K Bushby; citation_author=G van Ommen; citation_journal_title=Neuromuscul Disord; citation_volume=15; citation_number=27; citation_pages=450-457; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mfold web server for nucleic acid folding and hybridization prediction.; citation_author=M Zuker; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=28; citation_pages=3406-3415; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pfold: RNA secondary structure prediction using stochastic context-free grammars.; citation_author=B Knudsen; citation_author=J Hein; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=29; citation_pages=3423-3428; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A statistical sampling algorithm for RNA secondary structure prediction.; citation_author=Y Ding; citation_author=CE Lawrence; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=30; citation_pages=7280-7301; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=RNA folding at elementary step resolution.; citation_author=C Flamm; citation_author=W Fontana; citation_author=IL Hofacker; citation_author=P Schuster; citation_journal_title=RNA; citation_volume=6; citation_number=31; citation_pages=325-338; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Computational antisense oligo prediction with a neural network model.; citation_author=AM Chalk; citation_author=EL Sonnhammer; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=32; citation_pages=1567-1575; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Antisense-induced exon skipping restores dystrophin expression in DMD patient derived muscle cells.; citation_author=JC van Deutekom; citation_author=M Bremmer-Bout; citation_author=AA Janson; citation_author=IB Ginjaar; citation_author=F Baas; citation_journal_title=Hum Mol Genet; citation_volume=10; citation_number=33; citation_pages=1547-1554; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mechanical devices of the spliceosome: motors, clocks, springs, and things.; citation_author=JP Staley; citation_author=C Guthrie; citation_journal_title=Cell; citation_volume=92; citation_number=34; citation_pages=315-326; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases.; citation_author=BJ Blencowe; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=35; citation_pages=106-110; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A general role for splicing enhancers in exon definition.; citation_author=BJ Lam; citation_author=KJ Hertel; citation_journal_title=RNA; citation_volume=8; citation_number=36; citation_pages=1233-1241; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sorting out the complexity of SR protein functions.; citation_author=BR Graveley; citation_journal_title=RNA; citation_volume=6; citation_number=37; citation_pages=1197-1211; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SR proteins are ‘locators’ of the RNA splicing machinery.; citation_author=BR Graveley; citation_author=KJ Hertel; citation_author=T Maniatis; citation_journal_title=Curr Biol; citation_volume=9; citation_number=38; citation_pages=R6-R7; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Distribution of SR protein exonic splicing enhancer motifs in human protein-coding genes.; citation_author=J Wang; citation_author=PJ Smith; citation_author=AR Krainer; citation_author=MQ Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=39; citation_pages=5053-5062; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Listening to silence and understanding nonsense: exonic mutations that affect splicing.; citation_author=L Cartegni; citation_author=SL Chew; citation_author=AR Krainer; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=40; citation_pages=285-298; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases.; citation_author=BJ Blencowe; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=41; citation_pages=106-110; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Antisense-induced exon skipping and synthesis of dystrophin in the mdx mouse.; citation_author=CJ Mann; citation_author=K Honeyman; citation_author=AJ Cheng; citation_author=T Ly; citation_author=F Lloyd; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=42; citation_pages=42-47; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Specific removal of the nonsense mutation from the mdx dystrophin mRNA using antisense oligonucleotides.; citation_author=SD Wilton; citation_author=F Lloyd; citation_author=K Carville; citation_author=S Fletcher; citation_author=K Honeyman; citation_journal_title=Neuromuscul Disord; citation_volume=9; citation_number=43; citation_pages=330-338; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=On the importance of being co-transcriptional.; citation_author=KM Neugebauer; citation_journal_title=J Cell Sci; citation_volume=115; citation_number=44; citation_pages=3865-3871; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The human dystrophin gene requires 16 hours to be transcribed and is cotranscriptionally spliced.; citation_author=CN Tennyson; citation_author=HJ Klamut; citation_author=RG Worton; citation_journal_title=Nat Genet; citation_volume=9; citation_number=45; citation_pages=184-190; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Physical isolation of nascent RNA chains transcribed by RNA polymerase II: evidence for cotranscriptional splicing.; citation_author=J Wuarin; citation_author=U Schibler; citation_journal_title=Mol Cell Biol; citation_volume=14; citation_number=46; citation_pages=7219-7225; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The mRNA assembly line: transcription and processing machines in the same factory.; citation_author=D Bentley; citation_journal_title=Curr Opin Cell Biol; citation_volume=14; citation_number=47; citation_pages=336-342; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The organization of replication and transcription.; citation_author=PR Cook; citation_journal_title=Science; citation_volume=284; citation_number=48; citation_pages=1790-1795; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An extensive network of coupling among gene expression machines.; citation_author=T Maniatis; citation_author=R Reed; citation_journal_title=Nature; citation_volume=416; citation_number=49; citation_pages=499-506; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Integrating mRNA processing with transcription.; citation_author=NJ Proudfoot; citation_author=A Furger; citation_author=MJ Dye; citation_journal_title=Cell; citation_volume=108; citation_number=50; citation_pages=501-512; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Co-transcriptional splicing of pre-messenger RNAs: considerations for the mechanism of alternative splicing.; citation_author=AC Goldstrohm; citation_author=AL Greenleaf; citation_author=MA Garcia-Blanco; citation_journal_title=Gene; citation_volume=277; citation_number=51; citation_pages=31-47; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effects of RNA secondary structure on alternative splicing of pre-mRNA: is folding limited to a region behind the transcribing RNA polymerase?; citation_author=LP Eperon; citation_author=IR Graham; citation_author=AD Griffiths; citation_author=IC Eperon; citation_journal_title=Cell; citation_volume=54; citation_number=52; citation_pages=393-401; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Splice site selection, rate of splicing, and alternative splicing on nascent transcripts.; citation_author=AL Beyer; citation_author=YN Osheim; citation_journal_title=Genes Dev; citation_volume=2; citation_number=53; citation_pages=754-765; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=RNP particles at splice junction sequences on Drosophila chorion transcripts.; citation_author=YN Osheim; citation_author=OL Miller; citation_author=AL Beyer; citation_journal_title=Cell; citation_volume=43; citation_number=54; citation_pages=143-151; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=RNA structure in solution.; citation_author=J Nowakowski; citation_author=IF Timoco; citation_number=55; citation_pages=567-598; citation_date=1999; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Sequential folding of transfer RNA. A nuclear magnetic resonance study of successively longer tRNA fragments with a common 5′ end.; citation_author=J Boyle; citation_author=GT Robillard; citation_author=SH Kim; citation_journal_title=J Mol Biol; citation_volume=139; citation_number=56; citation_pages=601-625; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Secondary structure formation during RNA synthesis.; citation_author=FR Kramer; citation_author=DR Mills; citation_journal_title=Nucleic Acids Res; citation_volume=9; citation_number=57; citation_pages=5109-5124; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Formation of metastable RNA structures by sequential folding during transcription: time-resolved structural analysis of potato spindle tuber viroid (-)-stranded RNA by temperature-gradient gel electrophoresis.; citation_author=D Repsilber; citation_author=S Wiese; citation_author=M Rachen; citation_author=AW Schroder; citation_author=D Riesner; citation_journal_title=RNA; citation_volume=5; citation_number=58; citation_pages=574-584; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Probing complex RNA structures by mechanical force.; citation_author=S Harlepp; citation_author=T Marchal; citation_author=J Robert; citation_author=JF Leger; citation_author=A Xayaphoummine; citation_journal_title=Eur Phys J E Soft Matter; citation_volume=12; citation_number=59; citation_pages=605-615; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Co-transcriptional folding is encoded within RNA genes.; citation_author=IM Meyer; citation_author=I Miklos; citation_journal_title=BMC Mol Biol; citation_volume=5; citation_number=60; citation_pages=10; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The influence of antisense oligonucleotide length on dystrophin exon skipping.; citation_author=PL Harding; citation_author=AM Fall; citation_author=K Honeyman; citation_author=S Fletcher; citation_author=SD Wilton; citation_journal_title=Mol Ther; citation_volume=15; citation_number=61; citation_pages=157-166; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Effects of RNA secondary structure on cellular antisense activity.; citation_author=TA Vickers; citation_author=JR Wyatt; citation_author=SM Freier; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=62; citation_pages=1340-1347; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Theoretical design of antisense genes with statistically increased efficacy.; citation_author=MJ Lehmann; citation_author=V Patzel; citation_author=G Sczakiel; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=63; citation_pages=2597-2604; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The activity of siRNA in mammalian cells is related to structural target accessibility: a comparison with antisense oligonucleotides.; citation_author=FR Kretschmer-Kazemi; citation_author=G Sczakiel; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=64; citation_pages=4417-4424; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Theoretical and experimental approaches to design effective antisense oligonucleotides.; citation_author=G Sczakiel; citation_journal_title=Front Biosci; citation_volume=5; citation_number=65; citation_pages=D194-D201; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=ESEfinder: A web resource to identify exonic splicing enhancers.; citation_author=L Cartegni; citation_author=J Wang; citation_author=Z Zhu; citation_author=MQ Zhang; citation_author=AR Krainer; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=66; citation_pages=3568-3571; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Predictive identification of exonic splicing enhancers in human genes.; citation_author=WG Fairbrother; citation_author=RF Yeh; citation_author=PA Sharp; citation_author=CB Burge; citation_journal_title=Science; citation_volume=297; citation_number=67; citation_pages=1007-1013; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Target selection for antisense oligonucleotide induced exon skipping in the dystrophin gene.; citation_author=SJ Errington; citation_author=CJ Mann; citation_author=S Fletcher; citation_author=SD Wilton; citation_journal_title=J Gene Med; citation_volume=5; citation_number=68; citation_pages=518-527; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure.; citation_author=DH Mathews; citation_author=J Sabina; citation_author=M Zuker; citation_author=DH Turner; citation_journal_title=J Mol Biol; citation_volume=288; citation_number=69; citation_pages=911-940; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular Biology of the Cell.; citation_author=B Alberts; citation_author=A Johnson; citation_author=J Lewis; citation_author=M Raff; citation_author=K Roberts; citation_number=70; citation_pages=317-326; citation_date=2002; citation_publisher=Garland Science, Taylor & Francis; " />
      <meta name="citation_reference" content="citation_title=Splice site selection, rate of splicing, and alternative splicing on nascent transcripts.; citation_author=AL Beyer; citation_author=YN Osheim; citation_journal_title=Genes Dev; citation_volume=2; citation_number=71; citation_pages=754-765; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing. R Foundation for Statistical Computing, Viena, Austria. ISBN 3-900051-07-0.; citation_number=72; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides"/>
    <meta name="twitter:description" content="Antisense oligonucleotides (AONs) mediated exon skipping offers potential therapy for Duchenne muscular dystrophy. However, the identification of effective AON target sites remains unsatisfactory for lack of a precise method to predict their binding accessibility. This study demonstrates the importance of co-transcriptional pre-mRNA folding in determining the accessibility of AON target sites for AON induction of selective exon skipping in DMD. Because transcription and splicing occur in tandem, AONs must bind to their target sites before splicing factors. Furthermore, co-transcriptional pre-mRNA folding forms transient secondary structures, which redistributes accessible binding sites. In our analysis, to approximate transcription elongation, a &ldquo;window of analysis&rdquo; that included the entire targeted exon was shifted one nucleotide at a time along the pre-mRNA. Possible co-transcriptional secondary structures were predicted using the sequence in each step of transcriptional analysis. A nucleotide was considered &ldquo;engaged&rdquo; if it formed a complementary base pairing in all predicted secondary structures of a particular step. Correlation of frequency and localisation of engaged nucleotides in AON target sites accounted for the performance (efficacy and efficiency) of 94% of 176 previously reported AONs. Four novel insights are inferred: (1) the lowest frequencies of engaged nucleotides are associated with the most efficient AONs; (2) engaged nucleotides at 3&prime; or 5&prime; ends of the target site attenuate AON performance more than at other sites; (3) the performance of longer AONs is less attenuated by engaged nucleotides at 3&prime; or 5&prime; ends of the target site compared to shorter AONs; (4) engaged nucleotides at 3&prime; end of a short target site attenuates AON efficiency more than at 5&prime; end."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001844.g007"/>

  <meta property="og:title" content="Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5909'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2876'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1293&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001844">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001844" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001844&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Dynamics of <i>Co-Transcriptional</i> Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Keng Boon Wee
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Keng Boon Wee, Zacharias Aloysius Dwi Pramono</p>

              
              

                <p>Affiliations:
                  Bioinformatics Institute, Singapore, Singapore, 
                  Graduate School for Integrative Sciences and Engineering, Centre for Life Sciences, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zacharias Aloysius Dwi Pramono
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Keng Boon Wee, Zacharias Aloysius Dwi Pramono</p>

              
              <p><span class="email">* E-mail:</span> <a href="mailto:dwi_pramono@nni.com.sg">dwi_pramono@nni.com.sg</a></p>

                <p>Affiliation:
                  Neuromuscular Research Laboratory, National Neuroscience Institute, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jian Li Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Neuromuscular Research Laboratory, National Neuroscience Institute, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karl F. MacDorman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Informatics, Indiana University, Indianapolis, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Poh San Lai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Paediatrics, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Woon Chee Yee
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Neuromuscular Research Laboratory, National Neuroscience Institute, Singapore, Singapore, 
                  Department of Neurology, National Neuroscience Institute, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 26, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001844</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001844-g001" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-g002" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-t001" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-g003" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-g004" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-g005" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-t002" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-g006" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001844-g007" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001844">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001844" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Antisense oligonucleotides (AONs) mediated exon skipping offers potential therapy for Duchenne muscular dystrophy. However, the identification of effective AON target sites remains unsatisfactory for lack of a precise method to predict their binding accessibility. This study demonstrates the importance of <em>co-transcriptional</em> pre-mRNA folding in determining the accessibility of AON target sites for AON induction of selective exon skipping in DMD. Because transcription and splicing occur in tandem, AONs must bind to their target sites before splicing factors. Furthermore, <em>co-transcriptional</em> pre-mRNA folding forms transient secondary structures, which redistributes accessible binding sites. In our analysis, to approximate transcription elongation, a “window of analysis” that included the entire targeted exon was shifted one nucleotide at a time along the pre-mRNA. Possible c<em>o-transcriptional</em> secondary structures were predicted using the sequence in each step of transcriptional analysis. A nucleotide was considered “<em>engaged</em>” if it formed a complementary base pairing in all predicted secondary structures of a particular step. Correlation of frequency and localisation of <em>engaged</em> nucleotides in AON target sites accounted for the performance (efficacy and efficiency) of 94% of 176 previously reported AONs. Four novel insights are inferred: (1) the lowest frequencies of <em>engaged</em> nucleotides are associated with the most efficient AONs; (2) <em>engaged</em> nucleotides at 3′ or 5′ ends of the target site attenuate AON performance more than at other sites; (3) the performance of longer AONs is less attenuated by <em>engaged</em> nucleotides at 3′ or 5′ ends of the target site compared to shorter AONs; (4) <em>engaged</em> nucleotides at 3′ end of a short target site attenuates AON efficiency more than at 5′ end.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wee KB, Pramono ZAD, Wang JL, MacDorman KF, Lai PS, et al.  (2008) Dynamics of <em>Co-Transcriptional</em> Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides. PLoS ONE 3(3):
          e1844.
            doi:10.1371/journal.pone.0001844</p><p><strong>Editor: </strong>Jean-Nicolas Volff, Ecole Normale Supérieure de Lyon, France</p><p><strong>Received:</strong> November 21, 2007; <strong>Accepted:</strong> February 8, 2008; <strong>Published:</strong> March 26, 2008</p><p><strong>Copyright:</strong> © 2008 Wee et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Grants from the National Medical Research Council of Singapore (NMRC/0938/2005) and SingHealth (NHGA-STP/04022) supported this study. K.B. Wee is a recipient of an A*STAR Graduate Scholarship. There is no financial interest in this manuscript from any of the above funding organizations.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Antisense oligonucleotides (AONs) are synthetic single-stranded molecules, typically consisting of 16 to 30 nucleotides that are complementary to a specific sequence in the target RNA. Apart from their well-documented applications to suppress gene expression, AONs have been used to modulate pre-messenger RNA (pre-mRNA) splicing as potential therapeutic strategy for genetic diseases such as Duchenne muscular dystrophy or DMD <a href="#pone.0001844-Wilton1">[1]</a>–<a href="#pone.0001844-Surono1">[13]</a>, thalassemia <a href="#pone.0001844-Suwanmanee1">[14]</a>–<a href="#pone.0001844-Dominski1">[17]</a>, ocular albinism <a href="#pone.0001844-Vetrini1">[18]</a> and cancer <a href="#pone.0001844-Mercatante1">[19]</a>. Studies of AON in DMD (MIM #310200), a fatal X-linked disorder affecting 1 in 3300 newborn males caused by mutations in the dystrophin gene, have progressed to preliminary human trials <a href="#pone.0001844-AartsmaRus1">[7]</a>, <a href="#pone.0001844-AartsmaRus2">[8]</a>, <a href="#pone.0001844-McClorey1">[20]</a>–<a href="#pone.0001844-Takeshima1">[26]</a>. The strategy involves selective exon skipping, either to remove the mutation carried by the exon, for point mutations, or to restore the mRNA reading frame, for frame-shift mutations. Although the resulting protein will be shorter than the wild type, it is expected to reduce the severe symptoms of DMD to the much milder allelic form of the disease, Becker muscular dystrophy (BMD, MIM #300376) <a href="#pone.0001844-Wilton1">[1]</a>–<a href="#pone.0001844-Matsuo2">[5]</a>. Restoration of widespread dystrophin expression by AONs has been demonstrated in animal models <a href="#pone.0001844-McClorey2">[21]</a>, <a href="#pone.0001844-Alter1">[23]</a>, <a href="#pone.0001844-Lu1">[25]</a>. Currently, the first phase I/II clinical trials of AON for DMD therapy are being initiated <a href="#pone.0001844-Muntoni1">[27]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The design of AONs for exon skipping of dystrophin involves the selection of appropriate AON target sites using mfold <a href="#pone.0001844-Zuker1">[28]</a> and other similar computational tools <a href="#pone.0001844-Knudsen1">[29]</a>–<a href="#pone.0001844-Flamm1">[31]</a> for prediction of pre-mRNA secondary structure. However, the conventional applications of mfold for selecting AON target sites are not satisfactory <a href="#pone.0001844-AartsmaRus3">[9]</a>, <a href="#pone.0001844-Chalk1">[32]</a>, <a href="#pone.0001844-vanDeutekom2">[33]</a>. For example, Aartsma-Rus <em>et al.</em> <a href="#pone.0001844-AartsmaRus4">[10]</a> concluded that, using mfold to predict the secondary structure of targeted pre-mRNA, they still had no clear insight into the accessibility of the targeted sequence within the folded pre-mRNA structure. We hypothesize that this outcome may be due to the omission in considering the dynamic localization of accessible sites during the ‘opportune period’ of pre-mRNA transcription. We propose that this omission may underlie the low success rate in the design of effective and efficient AONs.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>An AON induces exon skipping by competitive binding at its target site against splicing factors during transcription <a href="#pone.0001844-Wilton2">[2]</a>, <a href="#pone.0001844-Pramono1">[12]</a>. Splicing factors form the 60S splicing machinery called the spliceosome that removes the introns while retaining the exons during pre-mRNA processing <a href="#pone.0001844-Staley1">[34]</a>. These splicing factors bind to important sequences in the pre-mRNA, which include donor and acceptor splice sites, branch points (BP), pyrimidine tracts and exon splicing enhancers (ESEs) <a href="#pone.0001844-Blencowe1">[35]</a>. Blocking these sites with AONs prevents the spliceosome from identifying the targeted region as an exon, which will thereby be removed along with the introns. Because of long introns sequences in dystrophin, ESE-dependent exon identification <a href="#pone.0001844-Lam1">[36]</a>–<a href="#pone.0001844-Blencowe2">[41]</a> is particularly important (<a href="#pone.0001844.s001">Figure S1</a> and <a href="#pone.0001844.s002">figure S2</a> of the <em>Online Supporting Information</em>). Indeed, AONs targeted to ESEs showed specific and effective induction of exon skipping in human tissue or cells <a href="#pone.0001844-AartsmaRus1">[7]</a>–<a href="#pone.0001844-Surono1">[13]</a> but resulted in unpredictable skipping of adjacent exons when targeted to the splice sites <a href="#pone.0001844-AartsmaRus3">[9]</a>, <a href="#pone.0001844-Mann1">[42]</a>, <a href="#pone.0001844-Wilton4">[43]</a> in DMD.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The splicing of introns by the spliceosome <a href="#pone.0001844-Staley1">[34]</a> is considered <em>co-transcriptional</em> <a href="#pone.0001844-Neugebauer1">[44]</a>–<a href="#pone.0001844-Eperon1">[52]</a>, as it happens simultaneously during transcription <a href="#pone.0001844-Beyer1">[53]</a>, <a href="#pone.0001844-Osheim1">[54]</a> of the pre-mRNA, at the point when an exon and its flanking introns are defined in the nascent pre-mRNA. <em>Co-transcriptional</em> splicing of dystrophin gene was first reported by Tennyson <em>et al</em> <a href="#pone.0001844-Tennyson1">[45]</a> in which the authors observed that “spliced transcript accumulates first at the 5′ end of the gene and at progressively later times as one moves further downstream from the muscle promoter” over a time period consistent with <em>co-transcriptional</em> splicing. The authors argued that given the exceptionally large size of the gene and large numbers of exons, <em>co-transcriptional</em> splicing is an effective way to limit the number of possible splice sites and thereby decrease the probabilities of incorrect splicings.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Recent experimental results support the notion that the transcription and splicing machineries are intricately coupled (reviewed by Maniatis T. &amp; Reed R. <a href="#pone.0001844-Maniatis1">[49]</a>). Specifically, by being tethered to both the RNA polymerase II and transcription elongation factors, splicing factors are localized directly adjacent to the nascent pre-mRNA emerging from the polymerase. This indicates that <em>co-transcriptional</em> exon recognition occurs at the proximity of the emerging nascent transcript, which seems to be supported by identical observations of both Aartsma-Rus <em>et al.</em> <a href="#pone.0001844-AartsmaRus4">[10]</a> and Wilton <em>et al.</em> <a href="#pone.0001844-Wilton3">[6]</a>. They reported that AONs targeting either acceptor sites or ESE sites in the first half of the exon are generally more efficient in inducing exon skipping than at the other half. This suggests that <em>co-transcriptional</em> exon recognition not only occurs as soon as recognition sites are transcribed, it is efficient as well; this implies that competition for binding to exon recognition sites starts as soon as they are transcribed. Together with the fact that <em>co-transcriptional</em> exon recognition precedes <em>co-transcriptional</em> intron removal (splicing), we conjecture that effective AONs must bind to their target sites during <em>co-transcriptional</em> target exon recognition.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To be efficacious, an AON must bind to an effective target site at the right time. An effective target site is a pre-mRNA sequence containing functional ESEs within the exon to be skipped. The right time or ‘opportune period’ is before splicing factors bind to the AON target site. Thus, two major factors defining AON efficiency are (1) binding to functional ESEs within the target site by the AON and (2) accessibility of the target site to binding during the ‘opportune period’, which in turn depends on the secondary structure of the pre-mRNA. The tendency to form complementary base pairings among the nucleotides within the pre-mRNA may cause a target site to be inaccessible, as a nucleotide that is “paired” is not accessible for binding. However, there are certain regions in the pre-mRNA with secondary structure motifs devoid of base pairing, such as loops, bulges, joint sequences and free 3′ or 5′ ends <a href="#pone.0001844-Nowakowski1">[55]</a>. Hence, for optimal AON binding, the prediction of base pairings and secondary structure motifs of the target site is likely to be crucial. As <em>co-transcriptional</em> pre-mRNA folding will lead to dynamic and transient secondary structures <a href="#pone.0001844-Boyle1">[56]</a>–<a href="#pone.0001844-Meyer1">[60]</a>, which in turn result in dynamic and transient nucleotide base pairing, the <em>co-transcriptional</em> folding of the nascent pre-mRNA during the “opportune period” must also be taken into account in determining optimal AON binding.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>To test this hypothesis, we developed novel scoring methodologies to semi-quantify the <em>co-transcriptional</em> binding accessibility of AON target sites. The AON target sites of 2 sets of published AONS were scored for c<em>o-transcriptional</em> binding accessibility and their scores correlated with the degree of reported AON efficiency and efficacy. The scoring methodologies are based on the application of an established software (i.e. mfold) for secondary structure prediction in conjunction with a method to approximate the dynamics of transcription. A “window of analysis” of pre-determined sequence length of 1500 nucleotides that includes the full length of the targeted exon (see <a href="#s4"><em>Methods and Materials</em></a>) corresponds to a “step of transcriptional analysis”. To approximate the transcription elongation process, the window of analysis is shifted one nucleotide at a time along the pre-mRNA sequence towards the 3′ end (<a href="#pone-0001844-g001">Figure 1</a>). At each step of transcriptional analysis, the possible secondary structures for the window sequence are predicted with mfold. Subsequently, each nucleotide within the AON target site will be scored for binding accessibility based on whether it is paired in the predicted secondary structures.</p>
<div class="figure" id="pone-0001844-g001"><div class="img"><a name="pone-0001844-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Approximation of the transcriptional elongation process of pre-mRNA.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>To approximate the transcription elongation process, a “window of analysis” is shifted one nucleotide at a time along the pre-mRNA sequence towards the 3′ end, beginning with the 3′ end of the exon targeted to be skipped at the window's 3′ end and stopping when the 5′ end of the targeted exon reaches the 5′ end of the window. Each window of analysis corresponds to a step of transcriptional analysis at which the possible secondary structures of its sequence were predicted.</p>
<span>doi:10.1371/journal.pone.0001844.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>A total of 176 AONs, reported by two independent sources <a href="#pone.0001844-AartsmaRus4">[10]</a> and <a href="#pone.0001844-Wilton3">[6]</a>, that target ESEs to induce exon skipping in dystrophin pre-mRNA was analyzed. Although the cell lines and experimental protocols used in these two studies were similar, the AONs from each study were analyzed separately because of the following reasons. The range of AON lengths, which may influence AON performance <a href="#pone.0001844-Harding1">[61]</a>, differed significantly between the studies. The AONs from the two sources <a href="#pone.0001844-AartsmaRus4">[10]</a> and <a href="#pone.0001844-Wilton3">[6]</a> showed median lengths of 19 and 26 nucleotides respectively, and for the purpose of this study, are henceforth denoted as Set A and Set B respectively. Secondly, the respective sources graded their AONs differently. In Set A, AONs were graded as (++), (+) or (−) if their efficiencies were &gt;25%, &lt;25% or 0% (i.e., non-effective) respectively. In Set B, AONs are graded as (++), (+<sup>1</sup>), (+<sup>2</sup>) or (−) if their efficiencies were &gt;30%, 10%–30%, &lt;10% or 0% respectively. For our analysis, grades (+<sup>1</sup>) and (+<sup>2</sup>) of Set B were merged into a single grade (+) while retaining the other grades as used by the respective sources.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Four levels of analysis using scoring methodologies of increasing complexity were used to score the accessibility of AON target sites. Scores at each level of analysis were then correlated with AON efficiency and efficacy for each of the two sets of AONs.</p>

<h4>First level analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>At first level analysis, each nucleotide within the AON target site, a nucleotide accessibility score will be determined by the following ratio:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e001&amp;representation=PNG" class="inline-graphic"></span><br>Note that multiple secondary structures will be predicted in each step of transcriptional analysis, see <a href="#pone-0001844-g002">Figure 2</a>. Hence, for each nucleotide, all secondary structures predicted at every step of transcriptional analysis are included. The accessibility score for the AON target site (<em>L1</em>) will be:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e002&amp;representation=PNG" class="inline-graphic"></span><br>The <em>L1</em> scores for the target sites of the analyzed AONs are tabulated in <a href="#pone.0001844.s004">Table S1</a> of the <em>Online Supporting Information</em>. The K-S tests failed to show any statistical difference between <em>L1</em> scores for the target sites of Set A AONs of different grades (<a href="#pone-0001844-t001">Table 1A</a>), which agrees with the results reported in refs. <a href="#pone.0001844-AartsmaRus4">[10]</a> and <a href="#pone.0001844-Harding1">[61]</a>. On the other hand, the <em>L1</em> scores for target sites of Set B (++) and (+<sup>1</sup>) AONs were statistically higher that the <em>L1</em> scores for target sites of (−) AONs (<a href="#pone-0001844-t001">Table 1B</a>). This result indicates that (++) and (+<sup>1</sup>) AON target sites are more accessible than (−) AON target sites, and therefore, the <em>L1</em> score is able to correlate with AON efficacy for Set B AONs. However, as this is not applicable to Set A AONs, we proceeded to the next level of analysis.</p>
<div class="figure" id="pone-0001844-g002"><div class="img"><a name="pone-0001844-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Definition of an <em>engaged</em> nucleotide in a particular step of transcriptional analysis.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) to (C). This is to illustrate that multiple secondary structures of the targeted exon (drawn in green) are predicted in each step of transcriptional analysis, with some of the possible structural motifs shown here. For illustration purpose, a particular nucleotide (marked in red) within an AON target site (green line) is tracked. When this nucleotide is paired (denoted with *), it is not accessible for AON binding. If this nucleotide is paired in all predicted secondary structures, this nucleotide is defined as an <em>engaged</em> nucleotide at this particular step of transcriptional analysis (B).</p>
<span>doi:10.1371/journal.pone.0001844.g002</span></div><div class="figure" id="pone-0001844-t001"><div class="img"><a name="pone-0001844-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>p-values for K-S tests using the first level score (<em>L1</em>) and third level score (<em>L3</em>) as test variables for differentiating the efficacy and/or efficiency of AONs.</span></strong></p><span>doi:10.1371/journal.pone.0001844.t001</span></div>

<h4>Second level analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>At this level of analysis, the nucleotide accessibility scores of all nucleotides in an AON target site were screened to determine the presence of two or more scores with values below 0.1 occurring consecutively in the nucleotide sequence of the target site (refer to <a href="#pone.0001844.s005">Table S2</a> of <em>Online Supporting Information</em>). Such grouping of below 0.1 nucleotide accessibility scores is termed a “low accessibility cluster”. In Set A, 71% of target sites of (−) AONs had one or more low accessibility cluster(s). While only 17% of target sites of (+) AONs had one or more clusters, they were manifested in 52% of target sites of (++) AONs. Set B also exhibited similar trends: 71%, 70% and 80% of target sites of (−) AONs, (+) AONs and (++) AONs respectively had one or more clusters. Therefore, the presence of these clusters in the AON target sites cannot correlate with AON efficacy and efficiency.</p>


<h4>Third level analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The nucleotide accessibility scores at the first and second levels of analysis are mean scores. As a result, two nucleotides with identical accessibility scores may have markedly different numbers of unpaired predicted secondary structures at each step of transcriptional analysis. In analyzing accessibility for AON binding, it may be important to take into account steps of transcriptional analysis in which a nucleotide is predicted to have total absence of unpaired secondary structures, i.e. the nucleotide is predicted to be completely inaccessible or “<em>engaged</em>” at the particular step of transcriptional analysis (<a href="#pone-0001844-g002">Figure 2B</a>). For the purpose of analysis, at every step of transcriptional analysis, each nucleotide in the AON target site which is <em>engaged</em> may then be depicted in a plot as illustrated in <a href="#pone-0001844-g003">Figure 3</a>. <a href="#pone.0001844.s006">Table S3</a> of the <em>Online Supporting Information</em> tabulates these plots for all the AON target sites analyzed.</p>
<div class="figure" id="pone-0001844-g003"><div class="img"><a name="pone-0001844-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Occurrences of <em>engaged</em> nucleotides at each step of transcriptional analysis.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>In the above illustration, the horizontal axis denotes sequential steps of transcriptional analysis while the vertical axis denotes numbered nucleotides within the AON target site. At each step of transcriptional analysis, nucleotides in the target site that are <em>engaged</em> are depicted as a black dot in the plot. The calculations of the fourth level scores, <em>L4_OR</em> and <em>L4_AND</em>, are illustrated (refer to main text for details).</p>
<span>doi:10.1371/journal.pone.0001844.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>For each nucleotide in an AON target site, a nucleotide <em>engaged</em> score may be derived as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e003&amp;representation=PNG" class="inline-graphic"></span><br>Following this, an AON target site <em>engaged</em> score (<em>L3</em>) may be derived as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e004&amp;representation=PNG" class="inline-graphic"></span><br>(<a href="#pone.0001844.s004">Table S1</a> of the <em>Online Supporting Information</em> tabulates the <em>L3</em> scores for all the AONs analyzed)</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>For Set A AONs, target sites of (++) AONs had statistically lower <em>engaged</em> scores than target sites of both (−) and (+) AONs. Therefore, <em>L3</em> score can statistically differentiate both AON efficacy and efficiency (<a href="#pone-0001844-t001">Table 1A</a>). However, seven outlier AONs (6% of the total) were identified. In this context, these were AONs in which the target site <em>L3</em> scores contradicted their AON grades. For instance, target sites of h52AON2 and h60AON2 graded as (−) could not induce exon skipping even though their <em>L3</em> scores were below the 5<sup>th</sup>-percentile of <em>L3</em> scores of (++) AON target sites. On the other hand, target sites of h45AON5 and h46AON4 graded as (+) and target sites of h51AON29, h55AON5 and h77AON2 graded as (++) all had <em>L3</em> scores higher than the 95<sup>th</sup>-percentile of <em>L3</em> scores of (−) AON target sites but could still induce exon skipping. The omission of these outlier AONs strengthened the correlation of <em>L3</em> scores with AON efficacy and efficiency (<a href="#pone-0001844-t001">Table 1A</a>). For Set B AONs, target sites of (++) AONs had statistically lower <em>engaged</em> scores than target sites of (−) AONs. Therefore, <em>L3</em> scores can statistically differentiate AON efficacy (<a href="#pone-0001844-t001">Table 1B</a>). Similarly, four outlier AONs (6% of the total) were identified, i.e., H30A, H58A, H64A and H34A2. The omission of these outlier AONs enabled <em>L3</em> scores to statistically differentiate efficacy between (+<sup>1</sup>) and (−) AONs, and efficiency between (++) and (+<sup>2</sup>) AONs. Remarkably, <em>L3</em> scores can differentiate between more AON grades than <em>L1</em> scores. Moreover, for all K-S tests in which <em>L1</em> scores showed statistical significance, the corresponding K-S tests of <em>L3</em> score obtained smaller p-values. Taken together, <em>L3</em> scores can differentiate both AON efficacy and efficiency better than <em>L1</em> scores.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>To appreciate the contrast between K-S test results of the first and third level scores, we plotted the quartiles of the normalized <em>L1</em> scores (<em><span class="underline">L1</span></em>) and <em>L3</em> scores (<em><span class="underline">L3</span></em>) of AON target sites for AONs in each grade of Sets A and B for comparison (<a href="#pone-0001844-g004">Figure 4A and 4B</a> respectively). For example, the <em><span class="underline">L1</span></em> score of an AON target site from Set A is the relative percentage difference between its <em>L1</em> score and the average <em>L1</em> score of all AON target sites from Set A. As expected, (++) AON target sites had the lowest <em><span class="underline">L3</span></em> score quartiles in both sets of AONs. Surprisingly, the maximum range for <em><span class="underline">L3</span></em> scores is 7 to 10 times larger than the range for <em><span class="underline">L1</span></em> score. Specifically, for Set A AONs, the range for target site <em><span class="underline">L3</span></em> scores is 140% in contrast to 12% for <em><span class="underline">L1</span></em> scores; for Set B AONs, the range for target site <em><span class="underline">L3</span></em> scores is 280% in contrast to 40% for <em><span class="underline">L1</span></em> score.</p>
<div class="figure" id="pone-0001844-g004"><div class="img"><a name="pone-0001844-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Quartiles of the normalized first and third level scores.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The quartiles (Q1, median and Q3) of the normalized first level scores (<em><span class="underline">L1</span></em>), and normalized third level scores (<em><span class="underline">L3</span></em>) for target sites of AONS in every AON grade of (A) Set A and (B) Set B are plotted. The units for all the vertical axes are in percentages.</p>
<span>doi:10.1371/journal.pone.0001844.g004</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Overall, for Set B AONs, the <em>L3</em> scores can satisfactorily differentiate efficacies and efficiencies of (++), (+) and (+<sup>1</sup>) AONs. On the other hand, for Set A AONs, while <em>L3</em> scores can differentiate (++) AONs from the rest of AON grades, they cannot account for the efficacies of (+) AONs because both (+) and (−) AON target sites had statistically similar <em>L3</em> scores. Intriguingly, the p-value for (++) vs. (+) AONs was even smaller than for (++) vs. (−) AONs. This indicates that although (+) AON target sites had high <em>L3</em> scores, they can still induce exon skipping albeit not efficiently. Hence, a more detail analysis was needed.</p>


<h4>Fourth level analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>While third level analysis primarily involves a general measure of frequency of <em>engaged</em> nucleotides, fourth level analysis includes consideration of localization of consecutive <em>engaged</em> nucleotides in the sequence of steps of transcriptional analysis of an AON target site. Three fourth level scores were developed for this analysis. These scores were applied to groups of 2 to 5 consecutive nucleotides in the AON target site and correlated with AON efficacy and efficiency (see <a href="#s4"><em>Methods and Materials</em></a> for details).</p>


<ol class="order">

<li><em>L4_AVG</em>–<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e005&amp;representation=PNG" class="inline-graphic"></span><br></li>

<li><em>L4_AND</em>–<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e006&amp;representation=PNG" class="inline-graphic"></span><br></li>

<li><em>L4_OR</em>–<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.e007&amp;representation=PNG" class="inline-graphic"></span><br></li>

</ol><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Preliminary analysis showed that the presence of consecutive <em>engaged</em> nucleotides at the ends of an AON target site attenuated AON efficacy and efficiency more than at other sites (data not shown). With this insight and to increase the power of the statistical tests, we analysed the localization of consecutive <em>engaged</em> nucleotides at the ends and away from the ends of the AON target site separately.</p>


<h4>Engaged nucleotides away from the ends of an AON target site</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>For the purpose of this analysis, “away from the ends of an AON target site” refers to nucleotides in the target site that are at least four nucleotides away from both 3′ and 5′ ends, as illustrated in <a href="#pone-0001844-g003">Figure 3</a>. We extracted groups of consecutive nucleotides consisting of two to five nucleotides from every AON target site analyzed. The three fourth level scores were calculated only for those groups of consecutive nucleotides meeting the following criterion for analysis: every nucleotide in the group having an <em>engaged</em> score of at least 0.1. Subsequently, statistical tests as in <a href="#pone-0001844-t001">Table 1</a> were applied to the scores. The analyses were stratified according to the number of consecutive nucleotides in the groups scored, as described below.</p>
<h5>Groups of two consecutive nucleotides.</h5><a id="article1.body1.sec2.sec5.sec1.p1" name="article1.body1.sec2.sec5.sec1.p1"></a><p>For both Set A and Set B AONs, the K-S tests found no statistical differences in all three scores of AON target sites at the different AON grades (data not shown). Note: inadequate AON sample size in Set B restricted the tests to scores of target sites of (++) vs. (+<sup>1</sup>) AONs and (++) vs. (+/−) AONs.</p>

<h5>Groups of three consecutive nucleotides.</h5><a id="article1.body1.sec2.sec5.sec2.p1" name="article1.body1.sec2.sec5.sec2.p1"></a><p>K-S tests could not be performed for both sets, as AON sample sizes of many AON grades were not adequate (&lt;6) to confer statistical confidence. Nevertheless, for Set A AONs, box-plots for each score were constructed in <a href="#pone-0001844-g005">Figure 5A</a>. The <em>L4_AND</em> score can differentiate (++) AONs from the other two AON grades comparatively well. While the <em>L4_AVG</em> score displayed some ability to differentiate (++) AONs, the <em>L4_OR</em> score failed to do so. For Set B AONs, only (++) AONs had adequate sample size to construct the box-plot (<a href="#pone-0001844-g005">Figure 5A</a>). Consistent with the results for Set A AONs, the <em>L4_AND</em> score of (++) Set B AON target sites had the smallest median and inter-quartile range compared to the other two scores.</p>
<div class="figure" id="pone-0001844-g005"><div class="img"><a name="pone-0001844-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Box-plots for each of the fourth level scores <em>L4_AVG, L4_AND</em> and <em>L4_OR.</em></span></strong></p><a id="article1.body1.sec2.sec5.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec5.sec2.fig1.caption1.p1"></a><p>(A) Box-plots for scores of groups of 3 nucleotides meeting criterion of analysis in target sites of (++) AONs, (+) AONs and (−) AONs of Set A, and of (++) AONs of Set B (***). (B) Box-plots for scores of groups of 4 nucleotides meeting criterion of analysis in target sites of both (+) and (−) AONs of Set A.</p>
<span>doi:10.1371/journal.pone.0001844.g005</span></div>
<h5>Groups of four consecutive nucleotides.</h5><a id="article1.body1.sec2.sec5.sec3.p1" name="article1.body1.sec2.sec5.sec3.p1"></a><p>For Set A AONs, only (+) and (−) AONs had adequate sample sizes to construct the box-plots (<a href="#pone-0001844-g005">Figure 5B</a>). Again, the <em>L4_AND</em> score demonstrated the best ability to differentiate (+) AONs from (−) AONs while the <em>L4_OR</em> score failed to do so. The sample sizes of Set B AONs at all grades were too small for analysis.</p>

<h5>Groups of five consecutive nucleotides.</h5><a id="article1.body1.sec2.sec5.sec4.p1" name="article1.body1.sec2.sec5.sec4.p1"></a><p>For Set A AONs, while three target sites of (−) AONs were found to have such a group as defined for this analysis, it was not found in all target sites of both (+) and (++) AONs. For Set B AONs, such a group was found in 33%, 11% and 6% of target sites of (−) AONs, (+) AONs and (++) AONs respectively.</p>
<a id="article1.body1.sec2.sec5.sec4.p2" name="article1.body1.sec2.sec5.sec4.p2"></a><p>Taken together, these results suggested that the presence of at least three consecutive <em>engaged</em> nucleotides at simultaneous steps of transcriptional analysis but not the individual nucleotide <em>engaged</em> score attenuated AON efficacy and efficiency.</p>



<h4>Engaged nucleotides at the ends of an AON target site</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>“At the ends of an AON target site” refers to nucleotides in the target site that are within three bases at 3′ or 5′ ends. For every AON target site analyzed, the three fourth level scores were calculated for every group of three consecutive nucleotides at 3′ and 5′ ends of the target site. Groups with zero L4_AVG score, i.e., all the nucleotides were not <em>engaged</em> at any step of transcriptional analysis, were excluded from the statistical tests. Statistical tests as in <a href="#pone-0001844-t001">Table 1</a> were applied to each of the three fourth level scores. Significant p-values of the K-S tests for target site scores of Sets A and B AONs are tabulated in <a href="#pone-0001844-t002">Table 2A</a>.</p>
<div class="figure" id="pone-0001844-t002"><div class="img"><a name="pone-0001844-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>p-values for K-S tests using the fourth level scores as test variables.</span></strong></p><span>doi:10.1371/journal.pone.0001844.t002</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>For Set A AONs, the <em>L4_OR</em> scores demonstrated the best ability to differentiate AON efficacy and efficiency, followed by the <em>L4_AVG</em> and <em>L4_AND</em> scores. In contrast, the <em>L4_AVG</em> score demonstrated the best ability to differentiate Set B (++) AONs, followed by the <em>L4_AND</em> score but not the <em>L4_OR</em> score. As the <em>L4_AND</em> score did not show the best ability to correlate AON efficacy and efficiency in both sets, AON efficacy and efficiency is more attenuated by presence of <em>engaged</em> nucleotides at the ends of target sites than at other sites. In addition, given that the <em>L4_OR</em> score only counted steps of transcriptional analysis in which at least one nucleotide was <em>engaged</em>, the test results suggested that efficacy and efficiency of shorter AONs (Set A) was more attenuated by <em>engaged</em> nucleotides at the ends of their target sites compared to longer AONs (Set B). To investigate whether <em>engaged</em> nucleotides at either 3′ or 5′ end of target sites affected AON efficacy and efficiency differently, we stratified the groups into 3′ and 5′, and repeated the same K-S tests, as discussed below.</p>
<h5>Engaged nucleotides at 3′ end.</h5><a id="article1.body1.sec2.sec6.sec1.p1" name="article1.body1.sec2.sec6.sec1.p1"></a><p>(<a href="#pone-0001844-t002">Table 2B</a>) For Set A AONs, the K-S test results were consistent with those obtained from both ends of AON target sites (<a href="#pone-0001844-t002">Table 2A</a>) except that, the <em>L4_AND</em> scores now failed to differentiate any AON grade. For Set B AONs, small sample sizes only permitted testing between target sites of (++) vs. (+<sup>1</sup>) AONs and no statistical difference was obtained for all three scores (data not shown).</p>

<h5>Engaged nucleotides at 5′ end.</h5><a id="article1.body1.sec2.sec6.sec2.p1" name="article1.body1.sec2.sec6.sec2.p1"></a><p>(<a href="#pone-0001844-t002">Table 2C</a>) For Set A AONs, only the <em>L4_OR</em> scores can differentiate AON efficacy. For Set B, small sample sizes only permitted testing between target sites of (++) AONs vs. (+<sup>1</sup>), (+) and (+/−) AONs. Although both the <em>L4_AVG</em> and <em>L4_OR</em> scores can differentiate (++) AONs from (+/−) AONs, the <em>L4_AVG</em> scores had a smaller p-value. This plausibly suggests that although <em>engaged</em> nucleotides at 5′ end also attenuated the efficacy and efficiency of Set B AONs, the extent of attenuation was less marked than Set A AONs. Altogether, these test results (<a href="#pone-0001844-t002">Tables 2A–2C</a>) strongly support the conclusion that <em>engaged</em> nucleotides at the ends of a shorter target site attenuated AON efficacy and efficiency more markedly than a longer target site.</p>

<h5>Engaged nucleotides at 3′ end vs. engaged nucleotides at 5′ end.</h5><a id="article1.body1.sec2.sec6.sec3.p1" name="article1.body1.sec2.sec6.sec3.p1"></a><p>Here, we tested whether there was statistical difference in the localization of <em>engaged</em> nucleotides in the sequence of steps of transcriptional analysis between 3′ and 5′ ends of target sites of AONs of the same grade. For instance, in the case of (++) AONs, each of the fourth level scores of the groups of nucleotides at the target site 3′ end was compared with the 5′ end using K-S tests. For Set A AONs, the groups of nucleotides at the 3′ end of (++) AON target sites had statistically smaller <em>L4_AVG</em> scores than at the 5′ end (<a href="#pone-0001844-t002">Table 2D</a>). This implied that <em>engaged</em> nucleotides at 3′ end of a short target site attenuated AON efficiency more than at 5′ end. For Set B AONs, on the other hand, small sample sizes only permitted testing of target sites of (++) AONs and (+) AONs but no statistical difference was obtained (data not shown).</p>


</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Previous studies have supported the general principle that mRNA secondary structures influence AON efficacy and efficiency <a href="#pone.0001844-Vickers1">[62]</a>–<a href="#pone.0001844-KretschmerKazemi1">[64]</a>, although these studies did not consider <em>co-transcriptional</em> folding in the prediction of the secondary structures. Furthermore, laboratories working this field <a href="#pone.0001844-AartsmaRus4">[10]</a>, <a href="#pone.0001844-Harding1">[61]</a> have reported no correlation with secondary mRNA structure in designing AONs to induce exon skipping of the dystrophin gene. In these reports, <em>co-transcriptional</em> dynamic changes in secondary structure were either not considered or were approximated with a relatively unrefined methodology. Our study aims to re-visit this issue by using a more refined method to approximate <em>co-transcriptional</em> dynamic changes in pre-mRNA secondary structures and by developing novel methods to take into account the localization of completely inaccessible nucleotides in the <em>co-transcriptional</em> process. Applying four levels of analysis with scoring methodologies of increasing complexity, we demonstrate that the frequency and localization of consecutive <em>engaged</em> nucleotides in the sequence of steps of transcriptional analysis correlated with efficacy and efficiency of 94% of previously reported AONs.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Four key novel insights pertaining to AON efficacy and efficiency were deduced from this study. Firstly, the lowest frequencies of <em>engaged</em> nucleotides manifested at target sites were associated with the most efficient (++) AONs. Secondly, <em>engaged</em> nucleotides at 3′ or 5′ ends of the target site attenuated AON efficacy and efficiency more than at other sites. Thirdly, the efficacy and efficiency of longer AONs were less attenuated by <em>engaged</em> nucleotides at 3′ or 5′ ends of the target site as compared to shorter AONs. In fact, the frequency and localization of <em>engaged</em> nucleotides of short AON (Set A) and long AON (Set B) target sites were statistically similar (see <a href="#pone.0001844.s007">Table S4</a> of the <em>Online Supporting Information</em> for more details). In agreement with reported studies <a href="#pone.0001844-Harding1">[61]</a>, these results indicate that AONs targeting longer target sites can induce exon skipping more effectively and efficiently by possibly binding to the pre-mRNA more stably as compared to AONs targeting shorter target sites. Fourthly, <em>engaged</em> nucleotides at 3′ end of a short target site attenuated AON efficiency more than at 5′ end. This might explain why AON efficiency is more sensitive to nucleotide changes at the 3′ end than 5′ end of its target site <a href="#pone.0001844-Sczakiel1">[65]</a>. Notably, our results provide quantitative statistical proof for these experimental observations.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>To demonstrate the correlative power of the fourth level scores, three common examples in which only the fourth level scores can differentiate (++) AONs in Set A (<a href="#pone-0001844-g006">Figure 6</a>) are discussed. <a href="#pone-0001844-g006">Figure 6A</a> illustrated an example wherein AON target sites with identical accessibility scores (<em>L1</em>) can have strikingly different <em>engaged</em> scores (<em>L3</em>). Whereas the (−) AON target site high <em>engaged</em> score was expected, the higher <em>engaged</em> score of the (++) AON target site compared to the (+) AON target site was confounding. In contrast to the <em>L3</em> score, fourth level analysis showed more <em>engaged</em> nucleotides at the ends of (+) AON target site than at (++) AON target site, i.e. was able to discriminate between (+) and (++) AON target sites. <a href="#pone-0001844-g006">Figure 6B</a> illustrated an example in which the target site <em>engaged</em> scores (<em>L3</em>) correlate inversely with AON efficacy and efficiency, i.e., AON target sites with higher <em>engaged</em> scores had better ability to induce exon skipping. Again, the fourth level scores resolved this conundrum in a similar manner as the first example. The final example (<a href="#pone-0001844-g006">Figure 6C</a>) illustrated a widespread phenomenon in the data set in which (+) AON target sites had higher <em>engaged</em> scores (<em>L3</em>) than (−) AON target sites. In fact, this phenomenon caused the p-values of K-S tests of (++) vs. (+) AON target site <em>L3</em> scores to be smaller than for (++) vs. (−) target site <em>L3</em> scores. In most instances, most of the <em>engaged</em> nucleotides manifested in (+) AON target sites were localized away from the ends of the sites. Altogether, these examples showed that localization is as important as the frequency of <em>engaged</em> nucleotides.</p>
<div class="figure" id="pone-0001844-g006"><div class="img"><a name="pone-0001844-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Three examples for where only the fourth level scores can correlate AON efficacy and efficiency compared to third level scores.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>(A) to (C) For every example, the incidences of <em>engaged</em> nucleotides at each step of transcriptional analysis for all nucleotide in target sites of (++), (+) and (−) AONs were depicted.</p>
<span>doi:10.1371/journal.pone.0001844.g006</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>As an illustration, an AON (novelAON57) target sequence was selected to skip exon 57. All three reported AONs (h57AON1, h57AON2, h57AON3) designed to induce exon 57 skipping by targeting the intra-exonic sequences failed to skip exon 57 <a href="#pone.0001844-AartsmaRus4">[10]</a>. Interestingly, exon 57 manifests an overwhelming occurrence of <em>engaged</em> nucleotides (<a href="#pone.0001844.s003">Figure S3</a> of the <em>Online Supporting Information</em>); hence, it is relatively difficult to locate a suitably long sequence that has ESE activity as well as <em>co-transcriptional</em> binding accessibility that fulfils the four insights (as described above). For instance, the 3′ ends of the target sites of both h57AON1 and h57AON2 AONs manifest substantial <em>engaged</em> nucleotides whereas the first half of the target site of h57AON3 AON manifests extensive <em>engaged</em> nucleotides (<a href="#pone.0001844.s006">Table S3</a> of the <em>Online Supporting Information</em>). We designed novelAON57 to have a target site with the following characteristics: negligible occurrence of <em>engaged</em> nucleotides, presence of ESE motifs predicted by ESE-Finder <a href="#pone.0001844-Cartegni2">[66]</a> and RESCUE-ESE <a href="#pone.0001844-Fairbrother1">[67]</a>, and location at the first half of the exon. Notably, novelAON57 targets a completely different site from the published AONs, as shown in <a href="#pone-0001844-g007">Figure 7</a>. At all AON concentrations tested, i.e. 100nM, 200nM, and 400nM, novelAON57 demonstrates selective skipping of exon 57 with an efficiency of (++) (<a href="#pone-0001844-g007">Figure 7</a>).</p>
<div class="figure" id="pone-0001844-g007"><div class="img"><a name="pone-0001844-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001844" data-uri="info:doi/10.1371/journal.pone.0001844.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001844.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001844.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001844.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001844.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>The efficiency of a novel AON targeting exon 57.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>The left panel shows the RT-PCR analysis of dystrophin mRNA treated with novelAON57 at concentrations of 100nM, 200nM and 400nM of AONs. For every AON concentration, the relative percentage of total transcripts (average value of duplicate transfections) with skipping of the targeted exon is given below the lanes. The middle panel indicates the relative position of the novel AONs with respect to published AONs. The right panel plots the <em>co-transcriptional</em> binding accessibility of the novel AON target sites, wherein the horizontal axis denotes sequential steps of transcriptional analysis and the vertical axis denotes numbered nucleotides within the AON target site. At each step of transcriptional analysis, nucleotides in the target site that are <em>engaged</em> are depicted as a black dot in the plot.</p>
<span>doi:10.1371/journal.pone.0001844.g007</span></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The number of secondary structures predicted for each exon was tabulated in <a href="#pone.0001844.s008">Table S5</a> of the <em>Online Supporting Information</em>. There was an average of 44,582 predicted secondary structures per exon and 24 to 47 predicted secondary structures per step of transcriptional analysis (<a href="#pone.0001844.s008">Table S5</a>). As a result, the identification of <em>engaged</em> nucleotides at a step of transcriptional analysis had low false-positives, as an <em>engaged</em> nucleotide must be paired in all predicted secondary structures. In addition, the use of numerous windows of analysis had the added advantage of spreading out the prediction error of mfold as vast numbers of secondary structures were used in the analyses. Despite the exceptional length of the dystrophin gene, the measured average elongation rate does not differ significantly from other genes <a href="#pone.0001844-Tennyson1">[45]</a>. While this seems to suggest that transcription of dystrophin gene is similar to other genes, the possibility of other transcription and/or splicing factors being involved cannot be dismissed. In the event that such factors significantly affect the rate of transcript elongation and/or the mechanism of exon recognition, the results of our analyses might differ substantially.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Besides <em>co-transcriptional</em> binding accessibilities of AON targets, AON efficiency depends on other factors such as presence of ESEs, stability of AONs (by chemical modifications), thermodynamic considerations, absolute distance of AON target site from 3′ splice site, etc. For instance, statistical analysis of predicted ESE sites by Aartsma-Rus <em>et al</em> <a href="#pone.0001844-AartsmaRus4">[10]</a> showed that target sites of (++) AONs in Set A had marginally more active ESE sites (p-value≈0.05). Therefore, the prediction of efficient AON targets does not simply involve annealing of an AON to a structurally accessible target as the best <em>co-transcriptionally</em> accessible target site might not contain ESEs. As these factors are often not mutually exclusive, an AON target site that fares very well in one factor but poorly in others might not be efficient.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The development of scores at four levels of analysis to semi-quantify co-transcriptional binding accessibility of AON target sites allows their correlation with AON efficacy and efficiency using statistical tests. These methodologies are potentially applicable to the development of a systematic approach to identify optimal target sites in the design of AONs to induce exon skipping of dystrophin pre-mRNA. Similarly, the methodologies may also be applicable in analyzing the efficiency of AONs applied in other diseases, such as thalassemia <a href="#pone.0001844-Suwanmanee1">[14]</a>–<a href="#pone.0001844-Dominski1">[17]</a>, ocular albinism <a href="#pone.0001844-Vetrini1">[18]</a> and cancer <a href="#pone.0001844-Mercatante1">[19]</a>, in which exon splicing modulation to correct the mRNA reading frame has been proposed as a therapeutic strategy.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data Set</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Set A is extracted from the list of 114 AONs published by Aartsma-Rus <em>et al.</em> <a href="#pone.0001844-AartsmaRus4">[10]</a>. Among them, 41 of them induce exon skipping in &gt;25% of the total dystrophin mRNA transcripts and are graded as (++); 35 of them induce exon skipping in &lt;25% of the total transcripts and are graded as (+); and 38 of them fail to induce exon skipping and are graded as (−). On the other hand, Set B is extracted from the list of AONs published by Wilton <em>et al.</em> <a href="#pone.0001844-Wilton3">[6]</a>. Although they published a total of 82 AONs, only 62 of them are applicable for this study. The remaining ones either target non-ESE sites or result in unspecific exon skipping. Among the relevant AONs, 35 of them induce exon skipping in &gt;30% of the total dystrophin mRNA transcripts and are graded as Type 1; 11 of them induce exon skipping in between 10% to 30% of the total transcripts and are graded as Type 2; 9 of them induce exon skipping in &lt;10% of the total transcripts and are graded as Type 3; and 7 of them fail to induce exon skipping (i.e., non-effective) and are graded as Type 4. For naming consistency, Types 1–4 are renamed as grades (++), (+<sup>1</sup>), (+<sup>2</sup>) and (−) respectively. Altogether, the 176 AONs target 67 exons in the dystrophin gene by blocking ESEs. AON efficacy was determined from RT-PCR analysis while AON efficiency was calculated based on densitograph semi-quantification <a href="#pone.0001844-Wilton3">[6]</a>, <a href="#pone.0001844-AartsmaRus4">[10]</a>.</p>


<h4>Computational prediction of the dynamical pre-mRNA secondary structure</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The methodology for quantifying and analyzing the dynamics of the pre-mRNA structures in the progression of transcription did not depend on the choice of the prediction tool as long as <em>co-transcriptional</em> structures were obtainable. mfold was eventually chosen because of its relative efficiency for computing long RNAs as well as the advantage of being used in most published experimental results on AONs that target dystrophin gene <a href="#pone.0001844-AartsmaRus3">[9]</a>, <a href="#pone.0001844-AartsmaRus4">[10]</a>, <a href="#pone.0001844-Errington1">[68]</a> and therefore, the results in this work can be compared with them on a common basis. mfold version 3.1 <a href="#pone.0001844-Zuker1">[28]</a>, <a href="#pone.0001844-Mathews1">[69]</a> was executed on a Dell PowerEdge SC1420 server running Red Hat Enterprise Linux 4.0 OS. Since it was highly probable that the nascent pre-mRNA may not have the chance to assume optimal structures, we accepted sub-optimal structures whose energies lie within 5% of the optimum.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Since long introns are typical in dystrophin gene, only local secondary structures around the targeted exon need to be considered. This was because abundant hnRNPs (heterogeneous nuclear ribonuclear proteins) package long intron regions into compact and manageable secondary structures for pre-mRNA processing that deterred long-distance or global intra-molecular complementary base pairings, which possibly prevented an exon from being entangled in a complex structure that would obstruct the spliceosome from accessing it <a href="#pone.0001844-Alberts1">[70]</a>. On the other hand, sequence length of the window of analysis was estimated from experimental measurements: elongation rate of dystrophin mRNA ranged from 1.7 to 2.5 kb per minute <a href="#pone.0001844-Tennyson1">[45]</a>; and RNP formation at 3′ splice sites was observed 48 seconds after 3′ splice sites synthesis <a href="#pone.0001844-Beyer2">[71]</a>. During the time-delay from 3′ splice site synthesis to its recognition, about 1360 to 2000 bases were appended to the nascent transcript. The dynamical secondary structures of exons 2 (62 bp), 29 (150 bp) and 59 (269 bp) were predicted based on 1200, 1500 and 2000 sequence length of the window of analysis. For each exon, there was no statistical difference in the nucleotide accessibility and <em>engaged</em> scores computed from secondary structures predicted based on different sequence lengths of window of analysis (data not shown). Therefore, the predicted secondary structures of a target exon were not sensitive to sequence length of the window of analysis.</p>


<h4>Statistical test for differentiating AON efficacy and efficiency</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Two-sample Kolmogorov-Smirnov (K-S) test was used to test for statistical differences and significances of the first, third and fourth level scores in their abilities to differentiate AON efficacy and efficiency between any two AON samples. Both two-tailed (H<sub>o</sub>: the two AON samples have different probability distributions) and one-tailed (H<sub>o</sub>: the first AON sample is larger/smaller than the second AON sample) tests were performed to ensure consistency of test results. All statistical tests were performed on the statistical software, R Version 2.0.0 <a href="#pone.0001844-R1">[72]</a>. Note: Wilcoxon rank-sum test was not used because box-plots of two AON samples showed that they had different distributional shapes (data not shown), which violated a key Wilcoxon test assumption.</p>


<h4>Preliminary analysis in the fourth level analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>In this preliminary analysis, the localization of groups of consecutive <em>engaged</em> nucleotides in the sequence of steps of transcriptional analysis of an AON target site was tested for correlation with AON efficacy and efficiency. For each AON analyzed, all possible groups of consecutive nucleotides in the AON target site were obtained. For instance, groups of two consecutive nucleotides were extracted by walking one nucleotide at a time from one end of an AON target site to the other end. Likewise, groups of three to five consecutive nucleotides are obtained similarly. The three fourth level scores (<em>L4_AVG</em>, <em>L4_AND</em> and <em>L4_OR</em>) were next applied on every group of consecutive nucleotides. Their scores were then correlated with AON efficacy and efficiency by K-S tests among various AON grades in Sets A and B. As the number of nucleotides in the groups may influence AON efficacy and efficiency, the K-S tests were stratified according to the numbers of nucleotides in the groups. Note: as the majority of the groups with more than five consecutive nucleotides have zero <em>L4_AND</em> scores, inadequate sample size constrained the analysis to a maximum of five consecutive nucleotides.</p>


<h4>Illustration of the efficiency of a novel AON targeting exon 57</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>A novel AON (novelAON57) was synthesized by Sigma-Prologo (France) with 2′-O-methyl and full length phosphorothioate backbones according to our specifications. Transfections were done on normal human fibroblast cells (Coriell, USA) cultured in 6-wells plates, with the AON concentrations of 100 nM, 200 nM or 400 nM and LipofectAmine 2000, with ratio of concentrations as suggested by the manufacturer (Invitrogen, Carlsbad, Canada). The transfection was done in duplicate. 24 hours after transfection, the cells were harvested and subjected to mRNA analysis to assess the performance of the AONs in inducing exon skipping. Total RNA was isolated using Trizol (Invitrogen, Carlsbad, Canada). Single step RT-PCR was performed on ~400ng total RNA using a single step RT-PCR analysis kit, Access RT-PCR system (Promega, Madison, USA), according to the manufacturer's instructions for 20 cycles, followed by nested PCR for 22 cycles. Sequences of dystrophin exon-specific primers used for single step RT-PCR and nested PCR are available upon request. Exon skipping efficiency was estimated by densitometry analysis of the gel images comparing the density of amplicons from dystrophin mRNA with exon 57 skipping to the native dystrophin mRNA.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001844"></div><a name="pone.0001844.s001" id="pone.0001844.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Genomic lengths of all exons and introns of dystrophin gene. The sequence lengths for each of the 79 exons in dystrophin are plotted as black bars. For every exon, both of their flanking introns sequence lengths are shown as gray bars. Note that the sequence length is on a logarithmic scale (vertical axis). Total exonic and intronic sequence length are 11,034 bps and 2,209,348 respectively. The exons occupied a mere 0.5% of its full DNA sequence. The lengths of the exons range from 7 to 269 bps; introns range from 107 to 319,058 bps.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001844.s002" id="pone.0001844.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Percentage genomic lengths of each exon relative to the total length of its flanking introns. To underscore the fact that locating an exon in dystrophin is akin to finding a needle in a haystack, the percentage of the length of an exon relative to the total length of its 3′ and 5′ intron sequences is computed and is plotted here. The majority of the exons constitute less than 1% of their intronic lengths and even the highest percentage is less than 7%.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001844.s003" id="pone.0001844.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Co-transcriptional binding accessibilities of exon 57. The horizontal axis denotes sequential steps of transcriptional analysis whereas the vertical axis denotes numbered nucleotides within the AON target site. At each step of transcriptional analysis, nucleotides in the target site that are engaged are depicted as a black dot in the plot.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
<a name="pone.0001844.s004" id="pone.0001844.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">First level score (<em>L1</em>) and third level score (<em>L3</em>) of 176 AON target sites analysed. This table tabulates the first level score (<em>L1</em>) and the third level score (<em>L3</em>). The AONs are sorted in ascending order of their target exon number, where the exon number is indicated in the AON names after the letter ‘h’, for e.g. h2AON1 targets exon 2. The sources of the AONs are indicated as superscripts on their names.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.20 MB DOC)</p>
<a name="pone.0001844.s005" id="pone.0001844.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">The nucleotide accessibility score of all nucleotide in an AON target site is plotted for all the 176 AONs analysed. The horizontal axis represents the nucleotide position in the respective target exon and the nucleotide accessibility score is plotted on the vertical axis.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.48 MB DOC)</p>
<a name="pone.0001844.s006" id="pone.0001844.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Occurrences of engaged nucleotides at each step of transcriptional analysis for all nucleotide in an AON target site are depicted for each of the 176 AON target sites analysed. The horizontal axis denotes sequential steps of transcriptional analysis while the vertical axis denotes numbered nucleotides within the AON target site. At each step of transcriptional analysis, nucleotides in the target site that are engaged are depicted as a black dot in the plot.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(1.93 MB DOC)</p>
<a name="pone.0001844.s007" id="pone.0001844.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s007">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">p-values for K-S tests using the third level score (<em>L3</em>) and fourth level scores (<em>L4_AVG</em>, <em>L4_AND</em> and <em>L4_OR</em>) as test variables between Set A and Set B. (A) (++) and (+) AONs are tested between Set A and Set B for statistical difference using L3 as test variable. (B) to (E) Fourth level scores as test variables for engaged nucleotides localized at (B) both 3′ and 5′ ends, (C) at 3′ end, (D) at 5′ end and (E) away from the ends of the AON target sites Note: (-) AONs between the two sets cannot be tested because the sample size in Set B is too small to confer statistical confidence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001844.s008" id="pone.0001844.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001844.s008">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Number of predicted secondary structures generated in each exon. For every exon in dystrophin gene, the total number of secondary structures predicted as well as the average number of predicted secondary structures per step of transcriptional analysis is tabulated.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001844.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.12 MB DOC)</p>
</div>

<div id="section6" class="section"><a id="s6" name="s6" toc="s6" title="Acknowledgments"></a><h3>Acknowledgments</h3></div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Analyzed the data: KW JW KM. Wrote the paper: ZP KW KM PL WY. Other: Created and implemented new methodologies: KW. Analyzed data: KW. Implemented methodologies: JW. Generated data: KW JW. Statistical analysis: KM. Co-Principle Investigator of the study: PL WY. Provided the laboratory: WY. Principal investigator of the study: ZP. Research grant owner: ZP. Conceptualized the study: KW WY PL ZP. Designed the study: KW ZP. Conceptualized new methodologies: ZP. Provided equipment for the study: WY ZP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001844-Wilton1" id="pone.0001844-Wilton1"></a>Wilton SD, Fletcher S (2005) Antisense oligonucleotides in the treatment of Duchenne muscular dystrophy: Where are we now? Neuromuscul Disord  15: 399–402.  <ul class="find" data-citedArticleID="1078646" data-doi="10.1016/j.nmd.2005.03.003"><li><a href="http://dx.doi.org/10.1016/j.nmd.2005.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+oligonucleotides+in+the+treatment+of+Duchenne+muscular+dystrophy%3A+Where+are+we+now%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+oligonucleotides+in+the+treatment+of+Duchenne+muscular+dystrophy%3A+Where+are+we+now%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001844-Wilton2" id="pone.0001844-Wilton2"></a>Wilton SD, Fletcher S (2005) RNA splicing manipulation: strategies to modify gene expression for a variety of therapeutic outcomes. Curr Gene Ther  5: 467–483.  <ul class="find" data-citedArticleID="1078648" data-doi="10.2174/156652305774329249"><li><a href="http://dx.doi.org/10.2174/156652305774329249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+splicing+manipulation%3A+strategies+to+modify+gene+expression+for+a+variety+of+therapeutic+outcomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+splicing+manipulation%3A+strategies+to+modify+gene+expression+for+a+variety+of+therapeutic+outcomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001844-Matsuo1" id="pone.0001844-Matsuo1"></a>Matsuo M, Takeshima Y (2005) Rescue of dystrophin mRNA of Duchenne muscular dystrophy by inducing exon skipping. Acta Myol  24: 110–114.  <ul class="find" data-citedArticleID="1078594"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rescue+of+dystrophin+mRNA+of+Duchenne+muscular+dystrophy+by+inducing+exon+skipping.&amp;auth=&amp;atitle=Rescue+of+dystrophin+mRNA+of+Duchenne+muscular+dystrophy+by+inducing+exon+skipping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rescue+of+dystrophin+mRNA+of+Duchenne+muscular+dystrophy+by+inducing+exon+skipping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rescue+of+dystrophin+mRNA+of+Duchenne+muscular+dystrophy+by+inducing+exon+skipping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001844-vanDeutekom1" id="pone.0001844-vanDeutekom1"></a>van Deutekom JC, van Ommen GJ (2003) Advances in Duchenne muscular dystrophy gene therapy. Nat Rev Genet  4: 774–783.  <ul class="find" data-citedArticleID="1078658" data-doi="10.1038/nrg1180"><li><a href="http://dx.doi.org/10.1038/nrg1180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Advances+in+Duchenne+muscular+dystrophy+gene+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Advances+in+Duchenne+muscular+dystrophy+gene+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001844-Matsuo2" id="pone.0001844-Matsuo2"></a>Matsuo M (2002) Duchenne and Becker muscular dystrophy: from gene diagnosis to molecular therapy. IUBMB Life  53: 147–152.  <ul class="find" data-citedArticleID="1078596" data-doi="10.1080/15216540212333"><li><a href="http://dx.doi.org/10.1080/15216540212333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duchenne+and+Becker+muscular+dystrophy%3A+from+gene+diagnosis+to+molecular+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duchenne+and+Becker+muscular+dystrophy%3A+from+gene+diagnosis+to+molecular+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001844-Wilton3" id="pone.0001844-Wilton3"></a>Wilton SD, Fall AM, Harding PL, McClorey G, Coleman C, et al.  (2007) Antisense Oligonucleotide-induced Exon Skipping Across the Human Dystrophin Gene Transcript. Mol Ther  15: 1288–1296.  <ul class="find" data-citedArticleID="1078650" data-doi="10.1038/sj.mt.6300095"><li><a href="http://dx.doi.org/10.1038/sj.mt.6300095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+Oligonucleotide-induced+Exon+Skipping+Across+the+Human+Dystrophin+Gene+Transcript." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+Oligonucleotide-induced+Exon+Skipping+Across+the+Human+Dystrophin+Gene+Transcript.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001844-AartsmaRus1" id="pone.0001844-AartsmaRus1"></a>Aartsma-Rus A, Kaman WE, Weij R, den Dunnen JT, van Ommen GJ, et al.  (2006) Exploring the frontiers of therapeutic exon skipping for Duchenne muscular dystrophy by double targeting within one or multiple exons. Mol Ther  14: 401–407.  <ul class="find" data-citedArticleID="1078518" data-doi="10.1016/j.ymthe.2006.02.022"><li><a href="http://dx.doi.org/10.1016/j.ymthe.2006.02.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+frontiers+of+therapeutic+exon+skipping+for+Duchenne+muscular+dystrophy+by+double+targeting+within+one+or+multiple+exons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+frontiers+of+therapeutic+exon+skipping+for+Duchenne+muscular+dystrophy+by+double+targeting+within+one+or+multiple+exons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001844-AartsmaRus2" id="pone.0001844-AartsmaRus2"></a>Aartsma-Rus A, Janson AA, Kaman WE, Bremmer-Bout M, van Ommen GJ, et al.  (2004) Antisense-induced multiexon skipping for Duchenne muscular dystrophy makes more sense. Am J Hum Genet  74: 83–92.  <ul class="find" data-citedArticleID="1078520" data-doi="10.1086/381039"><li><a href="http://dx.doi.org/10.1086/381039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense-induced+multiexon+skipping+for+Duchenne+muscular+dystrophy+makes+more+sense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense-induced+multiexon+skipping+for+Duchenne+muscular+dystrophy+makes+more+sense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001844-AartsmaRus3" id="pone.0001844-AartsmaRus3"></a>Aartsma-Rus A, Bremmer-Bout M, Janson AA, den Dunnen JT, van Ommen GJ, et al.  (2002) Targeted exon skipping as a potential gene correction therapy for Duchenne muscular dystrophy. Neuromuscul Disord  12: Suppl 1S71–S77.  <ul class="find" data-citedArticleID="1078522" data-doi="10.1016/s0960-8966(02)00086-x"><li><a href="http://dx.doi.org/10.1016/s0960-8966(02)00086-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+exon+skipping+as+a+potential+gene+correction+therapy+for+Duchenne+muscular+dystrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+exon+skipping+as+a+potential+gene+correction+therapy+for+Duchenne+muscular+dystrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001844-AartsmaRus4" id="pone.0001844-AartsmaRus4"></a>Aartsma-Rus A, De Winter CL, Janson AA, Kaman WE, van Ommen GJ, et al.  (2005) Functional analysis of 114 exon-internal AONs for targeted DMD exon skipping: indication for steric hindrance of SR protein binding sites. Oligonucleotides  15: 284–297.  <ul class="find" data-citedArticleID="1078524" data-doi="10.1089/oli.2005.15.284"><li><a href="http://dx.doi.org/10.1089/oli.2005.15.284" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+114+exon-internal+AONs+for+targeted+DMD+exon+skipping%3A+indication+for+steric+hindrance+of+SR+protein+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+114+exon-internal+AONs+for+targeted+DMD+exon+skipping%3A+indication+for+steric+hindrance+of+SR+protein+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001844-Matsuo3" id="pone.0001844-Matsuo3"></a>Matsuo M (1996) Duchenne/Becker muscular dystrophy: from molecular diagnosis to gene therapy. Brain Dev  18: 167–172.  <ul class="find" data-citedArticleID="1078598" data-doi="10.1016/0387-7604(96)00007-1"><li><a href="http://dx.doi.org/10.1016/0387-7604(96)00007-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duchenne%2FBecker+muscular+dystrophy%3A+from+molecular+diagnosis+to+gene+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duchenne%2FBecker+muscular+dystrophy%3A+from+molecular+diagnosis+to+gene+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001844-Pramono1" id="pone.0001844-Pramono1"></a>Pramono ZA, Takeshima Y, Alimsardjono H, Ishii A, Takeda S, et al.  (1996) Induction of exon skipping of the dystrophin transcript in lymphoblastoid cells by transfecting an antisense oligodeoxynucleotide complementary to an exon recognition sequence. Biochem Biophys Res Commun  226: 445–449.  <ul class="find" data-citedArticleID="1078616" data-doi="10.1006/bbrc.1996.1375"><li><a href="http://dx.doi.org/10.1006/bbrc.1996.1375" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+exon+skipping+of+the+dystrophin+transcript+in+lymphoblastoid+cells+by+transfecting+an+antisense+oligodeoxynucleotide+complementary+to+an+exon+recognition+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+exon+skipping+of+the+dystrophin+transcript+in+lymphoblastoid+cells+by+transfecting+an+antisense+oligodeoxynucleotide+complementary+to+an+exon+recognition+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001844-Surono1" id="pone.0001844-Surono1"></a>Surono A, Van Khanh T, Takeshima Y, Wada H, Yagi M, et al.  (2004) Chimeric RNA/ethylene-bridged nucleic acids promote dystrophin expression in myocytes of duchenne muscular dystrophy by inducing skipping of the nonsense mutation-encoding exon. Hum Gene Ther  15: 749–757.  <ul class="find" data-citedArticleID="1078630" data-doi="10.1089/1043034041648444"><li><a href="http://dx.doi.org/10.1089/1043034041648444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chimeric+RNA%2Fethylene-bridged+nucleic+acids+promote+dystrophin+expression+in+myocytes+of+duchenne+muscular+dystrophy+by+inducing+skipping+of+the+nonsense+mutation-encoding+exon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chimeric+RNA%2Fethylene-bridged+nucleic+acids+promote+dystrophin+expression+in+myocytes+of+duchenne+muscular+dystrophy+by+inducing+skipping+of+the+nonsense+mutation-encoding+exon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001844-Suwanmanee1" id="pone.0001844-Suwanmanee1"></a>Suwanmanee T, Sierakowska H, Fucharoen S, Kole R (2002) Repair of a splicing defect in erythroid cells from patients with beta-thalassemia/HbE disorder. Mol Ther  6: 718–726.  <ul class="find" data-citedArticleID="1078632" data-doi="10.1006/mthe.2002.0805"><li><a href="http://dx.doi.org/10.1006/mthe.2002.0805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repair+of+a+splicing+defect+in+erythroid+cells+from+patients+with+beta-thalassemia%2FHbE+disorder." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repair+of+a+splicing+defect+in+erythroid+cells+from+patients+with+beta-thalassemia%2FHbE+disorder.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001844-Gorman1" id="pone.0001844-Gorman1"></a>Gorman L, Mercatante DR, Kole R (2000) Restoration of correct splicing of thalassemic beta-globin pre-mRNA by modified U1 snRNAs. J Biol Chem  275: 35914–35919.  <ul class="find" data-citedArticleID="1078566" data-doi="10.1074/jbc.m006259200"><li><a href="http://dx.doi.org/10.1074/jbc.m006259200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Restoration+of+correct+splicing+of+thalassemic+beta-globin+pre-mRNA+by+modified+U1+snRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Restoration+of+correct+splicing+of+thalassemic+beta-globin+pre-mRNA+by+modified+U1+snRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001844-Sazani1" id="pone.0001844-Sazani1"></a>Sazani P, Kole R (2003) Therapeutic potential of antisense oligonucleotides as modulators of alternative splicing. J Clin Invest  112: 481–486.  <ul class="find" data-citedArticleID="1078624" data-doi="10.1172/jci19547"><li><a href="http://dx.doi.org/10.1172/jci19547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Therapeutic+potential+of+antisense+oligonucleotides+as+modulators+of+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Therapeutic+potential+of+antisense+oligonucleotides+as+modulators+of+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001844-Dominski1" id="pone.0001844-Dominski1"></a>Dominski Z, Kole R (1993) Restoration of correct splicing in thalassemic pre-mRNA by antisense oligonucleotides. Proc Natl Acad Sci U S A  90: 8673–8677.  <ul class="find" data-citedArticleID="1078552" data-doi="10.1073/pnas.90.18.8673"><li><a href="http://dx.doi.org/10.1073/pnas.90.18.8673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Restoration+of+correct+splicing+in+thalassemic+pre-mRNA+by+antisense+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Restoration+of+correct+splicing+in+thalassemic+pre-mRNA+by+antisense+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001844-Vetrini1" id="pone.0001844-Vetrini1"></a>Vetrini F, Tammaro R, Bondanza S, Surace EM, Auricchio A, et al.  (2006) Aberrant splicing in the ocular albinism type 1 gene (OA1/GPR143) is corrected in vitro by morpholino antisense oligonucleotides. Hum Mutat  27: 420–426.  <ul class="find" data-citedArticleID="1078638" data-doi="10.1002/humu.20303"><li><a href="http://dx.doi.org/10.1002/humu.20303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aberrant+splicing+in+the+ocular+albinism+type+1+gene+%28OA1%2FGPR143%29+is+corrected+in+vitro+by+morpholino+antisense+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aberrant+splicing+in+the+ocular+albinism+type+1+gene+%28OA1%2FGPR143%29+is+corrected+in+vitro+by+morpholino+antisense+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001844-Mercatante1" id="pone.0001844-Mercatante1"></a>Mercatante DR, Sazani P, Kole R (2001) Modification of alternative splicing by antisense oligonucleotides as a potential chemotherapy for cancer and other diseases. Curr Cancer Drug Targets  1: 211–230.  <ul class="find" data-citedArticleID="1078604" data-doi="10.2174/1568009013334124"><li><a href="http://dx.doi.org/10.2174/1568009013334124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modification+of+alternative+splicing+by+antisense+oligonucleotides+as+a+potential+chemotherapy+for+cancer+and+other+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modification+of+alternative+splicing+by+antisense+oligonucleotides+as+a+potential+chemotherapy+for+cancer+and+other+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001844-McClorey1" id="pone.0001844-McClorey1"></a>McClorey G, Fall AM, Moulton HM, Iversen PL, Rasko JE, et al.  (2006) Induced dystrophin exon skipping in human muscle explants. Neuromuscul Disord  16: 583–590.  <ul class="find" data-citedArticleID="1078600" data-doi="10.1016/j.nmd.2006.05.017"><li><a href="http://dx.doi.org/10.1016/j.nmd.2006.05.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induced+dystrophin+exon+skipping+in+human+muscle+explants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induced+dystrophin+exon+skipping+in+human+muscle+explants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001844-McClorey2" id="pone.0001844-McClorey2"></a>McClorey G, Moulton HM, Iversen PL, Fletcher S, Wilton SD (2006) Antisense oligonucleotide-induced exon skipping restores dystrophin expression in vitro in a canine model of DMD. Gene Ther  13: 1373–1381.  <ul class="find" data-citedArticleID="1078602" data-doi="10.1038/sj.gt.3302800"><li><a href="http://dx.doi.org/10.1038/sj.gt.3302800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+oligonucleotide-induced+exon+skipping+restores+dystrophin+expression+in+vitro+in+a+canine+model+of+DMD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+oligonucleotide-induced+exon+skipping+restores+dystrophin+expression+in+vitro+in+a+canine+model+of+DMD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001844-Wells1" id="pone.0001844-Wells1"></a>Wells DJ (2006) Therapeutic restoration of dystrophin expression in Duchenne muscular dystrophy. J Muscle Res Cell Motil  27: 387–398.  <ul class="find" data-citedArticleID="1078644" data-doi="10.1007/s10974-006-9081-6"><li><a href="http://dx.doi.org/10.1007/s10974-006-9081-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Therapeutic+restoration+of+dystrophin+expression+in+Duchenne+muscular+dystrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Therapeutic+restoration+of+dystrophin+expression+in+Duchenne+muscular+dystrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001844-Alter1" id="pone.0001844-Alter1"></a>Alter J, Lou F, Rabinowitz A, Yin H, Rosenfeld J, et al.  (2006) Systemic delivery of morpholino oligonucleotide restores dystrophin expression bodywide and improves dystrophic pathology. Nat Med  12: 175–177.  <ul class="find" data-citedArticleID="1078528" data-doi="10.1038/nm1345"><li><a href="http://dx.doi.org/10.1038/nm1345" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systemic+delivery+of+morpholino+oligonucleotide+restores+dystrophin+expression+bodywide+and+improves+dystrophic+pathology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systemic+delivery+of+morpholino+oligonucleotide+restores+dystrophin+expression+bodywide+and+improves+dystrophic+pathology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001844-Fall1" id="pone.0001844-Fall1"></a>Fall AM, Johnsen R, Honeyman K, Iversen P, Fletcher S, et al.  (2006) Induction of revertant fibres in the mdx mouse using antisense oligonucleotides. Genet Vaccines Ther  4: 3.  <ul class="find" data-citedArticleID="1078560"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Induction+of+revertant+fibres+in+the+mdx+mouse+using+antisense+oligonucleotides.&amp;auth=&amp;atitle=Induction+of+revertant+fibres+in+the+mdx+mouse+using+antisense+oligonucleotides." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+revertant+fibres+in+the+mdx+mouse+using+antisense+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+revertant+fibres+in+the+mdx+mouse+using+antisense+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001844-Lu1" id="pone.0001844-Lu1"></a>Lu QL, Rabinowitz A, Chen YC, Yokota T, Yin H, et al.  (2005) Systemic delivery of antisense oligoribonucleotide restores dystrophin expression in body-wide skeletal muscles. Proc Natl Acad Sci U S A  102: 198–203.  <ul class="find" data-citedArticleID="1078586" data-doi="10.1073/pnas.0406700102"><li><a href="http://dx.doi.org/10.1073/pnas.0406700102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systemic+delivery+of+antisense+oligoribonucleotide+restores+dystrophin+expression+in+body-wide+skeletal+muscles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systemic+delivery+of+antisense+oligoribonucleotide+restores+dystrophin+expression+in+body-wide+skeletal+muscles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001844-Takeshima1" id="pone.0001844-Takeshima1"></a>Takeshima Y, Yagi M, Wada H, Ishibashi K, Nishiyama A, et al.  (2006) Intravenous infusion of an antisense oligonucleotide results in exon skipping in muscle dystrophin mRNA of Duchenne muscular dystrophy. Pediatr Res  59: 690–694.  <ul class="find" data-citedArticleID="1078634" data-doi="10.1203/01.pdr.0000215047.51278.7c"><li><a href="http://dx.doi.org/10.1203/01.pdr.0000215047.51278.7c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intravenous+infusion+of+an+antisense+oligonucleotide+results+in+exon+skipping+in+muscle+dystrophin+mRNA+of+Duchenne+muscular+dystrophy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intravenous+infusion+of+an+antisense+oligonucleotide+results+in+exon+skipping+in+muscle+dystrophin+mRNA+of+Duchenne+muscular+dystrophy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001844-Muntoni1" id="pone.0001844-Muntoni1"></a>Muntoni F, Bushby K, van Ommen G (2005) 128th ENMC International Workshop on ‘Preclinical optimization and Phase I/II Clinical Trials Using Antisense Oligonucleotides in Duchenne Muscular Dystrophy’ 22-24 October 2004, Naarden, The Netherlands. Neuromuscul Disord  15: 450–457.  <ul class="find" data-citedArticleID="1078608" data-doi="10.1016/j.nmd.2005.02.007"><li><a href="http://dx.doi.org/10.1016/j.nmd.2005.02.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=128th+ENMC+International+Workshop+on+%E2%80%98Preclinical+optimization+and+Phase+I%2FII+Clinical+Trials+Using+Antisense+Oligonucleotides+in+Duchenne+Muscular+Dystrophy%E2%80%99+22-24+October+2004%2C+Naarden%2C+The+Netherlands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22128th+ENMC+International+Workshop+on+%E2%80%98Preclinical+optimization+and+Phase+I%2FII+Clinical+Trials+Using+Antisense+Oligonucleotides+in+Duchenne+Muscular+Dystrophy%E2%80%99+22-24+October+2004%2C+Naarden%2C+The+Netherlands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001844-Zuker1" id="pone.0001844-Zuker1"></a>Zuker M (2003) Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res  31: 3406–3415.  <ul class="find" data-citedArticleID="1078656" data-doi="10.1093/nar/gkg595"><li><a href="http://dx.doi.org/10.1093/nar/gkg595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001844-Knudsen1" id="pone.0001844-Knudsen1"></a>Knudsen B, Hein J (2003) Pfold: RNA secondary structure prediction using stochastic context-free grammars. Nucleic Acids Res  31: 3423–3428.  <ul class="find" data-citedArticleID="1078576" data-doi="10.1093/nar/gkg614"><li><a href="http://dx.doi.org/10.1093/nar/gkg614" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pfold%3A+RNA+secondary+structure+prediction+using+stochastic+context-free+grammars." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pfold%3A+RNA+secondary+structure+prediction+using+stochastic+context-free+grammars.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001844-Ding1" id="pone.0001844-Ding1"></a>Ding Y, Lawrence CE (2003) A statistical sampling algorithm for RNA secondary structure prediction. Nucleic Acids Res  31: 7280–7301.  <ul class="find" data-citedArticleID="1078550" data-doi="10.1093/nar/gkg938"><li><a href="http://dx.doi.org/10.1093/nar/gkg938" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+statistical+sampling+algorithm+for+RNA+secondary+structure+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+statistical+sampling+algorithm+for+RNA+secondary+structure+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001844-Flamm1" id="pone.0001844-Flamm1"></a>Flamm C, Fontana W, Hofacker IL, Schuster P (2000) RNA folding at elementary step resolution. RNA  6: 325–338.  <ul class="find" data-citedArticleID="1078562" data-doi="10.1017/s1355838200992161"><li><a href="http://dx.doi.org/10.1017/s1355838200992161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+folding+at+elementary+step+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+folding+at+elementary+step+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001844-Chalk1" id="pone.0001844-Chalk1"></a>Chalk AM, Sonnhammer EL (2002) Computational antisense oligo prediction with a neural network model. Bioinformatics  18: 1567–1575.  <ul class="find" data-citedArticleID="1078546" data-doi="10.1093/bioinformatics/18.12.1567"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.12.1567" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+antisense+oligo+prediction+with+a+neural+network+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+antisense+oligo+prediction+with+a+neural+network+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001844-vanDeutekom2" id="pone.0001844-vanDeutekom2"></a>van Deutekom JC, Bremmer-Bout M, Janson AA, Ginjaar IB, Baas F, et al.  (2001) Antisense-induced exon skipping restores dystrophin expression in DMD patient derived muscle cells. Hum Mol Genet  10: 1547–1554.  <ul class="find" data-citedArticleID="1078660" data-doi="10.1093/hmg/10.15.1547"><li><a href="http://dx.doi.org/10.1093/hmg/10.15.1547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense-induced+exon+skipping+restores+dystrophin+expression+in+DMD+patient+derived+muscle+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense-induced+exon+skipping+restores+dystrophin+expression+in+DMD+patient+derived+muscle+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001844-Staley1" id="pone.0001844-Staley1"></a>Staley JP, Guthrie C (1998) Mechanical devices of the spliceosome: motors, clocks, springs, and things. Cell  92: 315–326.  <ul class="find" data-citedArticleID="1078628" data-doi="10.1016/s0092-8674(00)80925-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80925-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanical+devices+of+the+spliceosome%3A+motors%2C+clocks%2C+springs%2C+and+things." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanical+devices+of+the+spliceosome%3A+motors%2C+clocks%2C+springs%2C+and+things.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001844-Blencowe1" id="pone.0001844-Blencowe1"></a>Blencowe BJ (2000) Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases. Trends Biochem Sci  25: 106–110.  <ul class="find" data-citedArticleID="1078536" data-doi="10.1016/s0968-0004(00)01549-8"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01549-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001844-Lam1" id="pone.0001844-Lam1"></a>Lam BJ, Hertel KJ (2002) A general role for splicing enhancers in exon definition. RNA  8: 1233–1241.  <ul class="find" data-citedArticleID="1078582" data-doi="10.1017/s1355838202028030"><li><a href="http://dx.doi.org/10.1017/s1355838202028030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+role+for+splicing+enhancers+in+exon+definition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+role+for+splicing+enhancers+in+exon+definition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001844-Graveley1" id="pone.0001844-Graveley1"></a>Graveley BR (2000) Sorting out the complexity of SR protein functions. RNA  6: 1197–1211.  <ul class="find" data-citedArticleID="1078568" data-doi="10.1017/s1355838200000960"><li><a href="http://dx.doi.org/10.1017/s1355838200000960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sorting+out+the+complexity+of+SR+protein+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sorting+out+the+complexity+of+SR+protein+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001844-Graveley2" id="pone.0001844-Graveley2"></a>Graveley BR, Hertel KJ, Maniatis T (1999) SR proteins are ‘locators’ of the RNA splicing machinery. Curr Biol  9: R6–R7.  <ul class="find" data-citedArticleID="1078570" data-doi="10.1016/s0960-9822(99)80032-3"><li><a href="http://dx.doi.org/10.1016/s0960-9822(99)80032-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SR+proteins+are+%E2%80%98locators%E2%80%99+of+the+RNA+splicing+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SR+proteins+are+%E2%80%98locators%E2%80%99+of+the+RNA+splicing+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001844-Wang1" id="pone.0001844-Wang1"></a>Wang J, Smith PJ, Krainer AR, Zhang MQ (2005) Distribution of SR protein exonic splicing enhancer motifs in human protein-coding genes. Nucleic Acids Res  33: 5053–5062.  <ul class="find" data-citedArticleID="1078642" data-doi="10.1093/nar/gki810"><li><a href="http://dx.doi.org/10.1093/nar/gki810" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+SR+protein+exonic+splicing+enhancer+motifs+in+human+protein-coding+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+SR+protein+exonic+splicing+enhancer+motifs+in+human+protein-coding+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001844-Cartegni1" id="pone.0001844-Cartegni1"></a>Cartegni L, Chew SL, Krainer AR (2002) Listening to silence and understanding nonsense: exonic mutations that affect splicing. Nat Rev Genet  3: 285–298.  <ul class="find" data-citedArticleID="1078542" data-doi="10.1038/nrg775"><li><a href="http://dx.doi.org/10.1038/nrg775" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Listening+to+silence+and+understanding+nonsense%3A+exonic+mutations+that+affect+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Listening+to+silence+and+understanding+nonsense%3A+exonic+mutations+that+affect+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001844-Blencowe2" id="pone.0001844-Blencowe2"></a>Blencowe BJ (2000) Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases. Trends Biochem Sci  25: 106–110.  <ul class="find" data-citedArticleID="1078538" data-doi="10.1016/s0968-0004(00)01549-8"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01549-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001844-Mann1" id="pone.0001844-Mann1"></a>Mann CJ, Honeyman K, Cheng AJ, Ly T, Lloyd F, et al.  (2001) Antisense-induced exon skipping and synthesis of dystrophin in the mdx mouse. Proc Natl Acad Sci U S A  98: 42–47.  <ul class="find" data-citedArticleID="1078590" data-doi="10.1073/pnas.98.1.42"><li><a href="http://dx.doi.org/10.1073/pnas.98.1.42" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense-induced+exon+skipping+and+synthesis+of+dystrophin+in+the+mdx+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense-induced+exon+skipping+and+synthesis+of+dystrophin+in+the+mdx+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001844-Wilton4" id="pone.0001844-Wilton4"></a>Wilton SD, Lloyd F, Carville K, Fletcher S, Honeyman K, et al.  (1999) Specific removal of the nonsense mutation from the mdx dystrophin mRNA using antisense oligonucleotides. Neuromuscul Disord  9: 330–338.  <ul class="find" data-citedArticleID="1078652" data-doi="10.1016/s0960-8966(99)00010-3"><li><a href="http://dx.doi.org/10.1016/s0960-8966(99)00010-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+removal+of+the+nonsense+mutation+from+the+mdx+dystrophin+mRNA+using+antisense+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+removal+of+the+nonsense+mutation+from+the+mdx+dystrophin+mRNA+using+antisense+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001844-Neugebauer1" id="pone.0001844-Neugebauer1"></a>Neugebauer KM (2002) On the importance of being co-transcriptional. J Cell Sci  115: 3865–3871.  <ul class="find" data-citedArticleID="1078610" data-doi="10.1242/jcs.00073"><li><a href="http://dx.doi.org/10.1242/jcs.00073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+importance+of+being+co-transcriptional." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+importance+of+being+co-transcriptional.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001844-Tennyson1" id="pone.0001844-Tennyson1"></a>Tennyson CN, Klamut HJ, Worton RG (1995) The human dystrophin gene requires 16 hours to be transcribed and is cotranscriptionally spliced. Nat Genet  9: 184–190.  <ul class="find" data-citedArticleID="1078636" data-doi="10.1038/ng0295-184"><li><a href="http://dx.doi.org/10.1038/ng0295-184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+dystrophin+gene+requires+16+hours+to+be+transcribed+and+is+cotranscriptionally+spliced." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+dystrophin+gene+requires+16+hours+to+be+transcribed+and+is+cotranscriptionally+spliced.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001844-Wuarin1" id="pone.0001844-Wuarin1"></a>Wuarin J, Schibler U (1994) Physical isolation of nascent RNA chains transcribed by RNA polymerase II: evidence for cotranscriptional splicing. Mol Cell Biol  14: 7219–7225.  <ul class="find" data-citedArticleID="1078654"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Physical+isolation+of+nascent+RNA+chains+transcribed+by+RNA+polymerase+II%3A+evidence+for+cotranscriptional+splicing.&amp;auth=&amp;atitle=Physical+isolation+of+nascent+RNA+chains+transcribed+by+RNA+polymerase+II%3A+evidence+for+cotranscriptional+splicing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical+isolation+of+nascent+RNA+chains+transcribed+by+RNA+polymerase+II%3A+evidence+for+cotranscriptional+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical+isolation+of+nascent+RNA+chains+transcribed+by+RNA+polymerase+II%3A+evidence+for+cotranscriptional+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001844-Bentley1" id="pone.0001844-Bentley1"></a>Bentley D (2002) The mRNA assembly line: transcription and processing machines in the same factory. Curr Opin Cell Biol  14: 336–342.  <ul class="find" data-citedArticleID="1078530" data-doi="10.1016/s0955-0674(02)00333-2"><li><a href="http://dx.doi.org/10.1016/s0955-0674(02)00333-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mRNA+assembly+line%3A+transcription+and+processing+machines+in+the+same+factory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mRNA+assembly+line%3A+transcription+and+processing+machines+in+the+same+factory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001844-Cook1" id="pone.0001844-Cook1"></a>Cook PR (1999) The organization of replication and transcription. Science  284: 1790–1795.  <ul class="find" data-citedArticleID="1078548" data-doi="10.1126/science.284.5421.1790"><li><a href="http://dx.doi.org/10.1126/science.284.5421.1790" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+organization+of+replication+and+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+organization+of+replication+and+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001844-Maniatis1" id="pone.0001844-Maniatis1"></a>Maniatis T, Reed R (2002) An extensive network of coupling among gene expression machines. Nature  416: 499–506.  <ul class="find" data-citedArticleID="1078588" data-doi="10.1038/416499a"><li><a href="http://dx.doi.org/10.1038/416499a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+extensive+network+of+coupling+among+gene+expression+machines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+extensive+network+of+coupling+among+gene+expression+machines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001844-Proudfoot1" id="pone.0001844-Proudfoot1"></a>Proudfoot NJ, Furger A, Dye MJ (2002) Integrating mRNA processing with transcription. Cell  108: 501–512.  <ul class="find" data-citedArticleID="1078618" data-doi="10.1016/s0092-8674(02)00617-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00617-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrating+mRNA+processing+with+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrating+mRNA+processing+with+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001844-Goldstrohm1" id="pone.0001844-Goldstrohm1"></a>Goldstrohm AC, Greenleaf AL, Garcia-Blanco MA (2001) Co-transcriptional splicing of pre-messenger RNAs: considerations for the mechanism of alternative splicing. Gene  277: 31–47.  <ul class="find" data-citedArticleID="1078564" data-doi="10.1016/s0378-1119(01)00695-3"><li><a href="http://dx.doi.org/10.1016/s0378-1119(01)00695-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Co-transcriptional+splicing+of+pre-messenger+RNAs%3A+considerations+for+the+mechanism+of+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Co-transcriptional+splicing+of+pre-messenger+RNAs%3A+considerations+for+the+mechanism+of+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001844-Eperon1" id="pone.0001844-Eperon1"></a>Eperon LP, Graham IR, Griffiths AD, Eperon IC (1988) Effects of RNA secondary structure on alternative splicing of pre-mRNA: is folding limited to a region behind the transcribing RNA polymerase? Cell  54: 393–401.  <ul class="find" data-citedArticleID="1078554" data-doi="10.1016/0092-8674(88)90202-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90202-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+RNA+secondary+structure+on+alternative+splicing+of+pre-mRNA%3A+is+folding+limited+to+a+region+behind+the+transcribing+RNA+polymerase%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+RNA+secondary+structure+on+alternative+splicing+of+pre-mRNA%3A+is+folding+limited+to+a+region+behind+the+transcribing+RNA+polymerase%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001844-Beyer1" id="pone.0001844-Beyer1"></a>Beyer AL, Osheim YN (1988) Splice site selection, rate of splicing, and alternative splicing on nascent transcripts. Genes Dev  2: 754–765.  <ul class="find" data-citedArticleID="1078532" data-doi="10.1101/gad.2.6.754"><li><a href="http://dx.doi.org/10.1101/gad.2.6.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splice+site+selection%2C+rate+of+splicing%2C+and+alternative+splicing+on+nascent+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splice+site+selection%2C+rate+of+splicing%2C+and+alternative+splicing+on+nascent+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001844-Osheim1" id="pone.0001844-Osheim1"></a>Osheim YN, Miller OL Jr, Beyer AL (1985) RNP particles at splice junction sequences on Drosophila chorion transcripts. Cell  43: 143–151.  <ul class="find" data-citedArticleID="1078614" data-doi="10.1016/0092-8674(85)90019-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(85)90019-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNP+particles+at+splice+junction+sequences+on+Drosophila+chorion+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNP+particles+at+splice+junction+sequences+on+Drosophila+chorion+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001844-Nowakowski1" id="pone.0001844-Nowakowski1"></a>Nowakowski J, Timoco IF (1999)  RNA structure in solution. In: Neidle S, editor. Oxford Handbook of Nucleic Acid Structures. New York: Oxford University Press. pp. 567–598.  <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0001844-Boyle1" id="pone.0001844-Boyle1"></a>Boyle J, Robillard GT, Kim SH (1980) Sequential folding of transfer RNA. A nuclear magnetic resonance study of successively longer tRNA fragments with a common 5′ end. J Mol Biol  139: 601–625.  <ul class="find" data-citedArticleID="1078540"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequential+folding+of+transfer+RNA.+A+nuclear+magnetic+resonance+study+of+successively+longer+tRNA+fragments+with+a+common+5%3F+end.&amp;auth=&amp;atitle=Sequential+folding+of+transfer+RNA.+A+nuclear+magnetic+resonance+study+of+successively+longer+tRNA+fragments+with+a+common+5%3F+end." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequential+folding+of+transfer+RNA.+A+nuclear+magnetic+resonance+study+of+successively+longer+tRNA+fragments+with+a+common+5%E2%80%B2+end." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequential+folding+of+transfer+RNA.+A+nuclear+magnetic+resonance+study+of+successively+longer+tRNA+fragments+with+a+common+5%E2%80%B2+end.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001844-Kramer1" id="pone.0001844-Kramer1"></a>Kramer FR, Mills DR (1981) Secondary structure formation during RNA synthesis. Nucleic Acids Res  9: 5109–5124.  <ul class="find" data-citedArticleID="1078578" data-doi="10.1093/nar/9.19.5109"><li><a href="http://dx.doi.org/10.1093/nar/9.19.5109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secondary+structure+formation+during+RNA+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secondary+structure+formation+during+RNA+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001844-Repsilber1" id="pone.0001844-Repsilber1"></a>Repsilber D, Wiese S, Rachen M, Schroder AW, Riesner D, et al.  (1999) Formation of metastable RNA structures by sequential folding during transcription: time-resolved structural analysis of potato spindle tuber viroid (-)-stranded RNA by temperature-gradient gel electrophoresis. RNA  5: 574–584.  <ul class="find" data-citedArticleID="1078622" data-doi="10.1017/s1355838299982018"><li><a href="http://dx.doi.org/10.1017/s1355838299982018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Formation+of+metastable+RNA+structures+by+sequential+folding+during+transcription%3A+time-resolved+structural+analysis+of+potato+spindle+tuber+viroid+%28-%29-stranded+RNA+by+temperature-gradient+gel+electrophoresis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Formation+of+metastable+RNA+structures+by+sequential+folding+during+transcription%3A+time-resolved+structural+analysis+of+potato+spindle+tuber+viroid+%28-%29-stranded+RNA+by+temperature-gradient+gel+electrophoresis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001844-Harlepp1" id="pone.0001844-Harlepp1"></a>Harlepp S, Marchal T, Robert J, Leger JF, Xayaphoummine A, et al.  (2003) Probing complex RNA structures by mechanical force. Eur Phys J E Soft Matter  12: 605–615.  <ul class="find" data-citedArticleID="1078574" data-doi="10.1140/epje/e2004-00033-4"><li><a href="http://dx.doi.org/10.1140/epje/e2004-00033-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+complex+RNA+structures+by+mechanical+force." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+complex+RNA+structures+by+mechanical+force.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001844-Meyer1" id="pone.0001844-Meyer1"></a>Meyer IM, Miklos I (2004) Co-transcriptional folding is encoded within RNA genes. BMC Mol Biol  5: 10.  <ul class="find" data-citedArticleID="1078606"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Co-transcriptional+folding+is+encoded+within+RNA+genes.&amp;auth=&amp;atitle=Co-transcriptional+folding+is+encoded+within+RNA+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Co-transcriptional+folding+is+encoded+within+RNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Co-transcriptional+folding+is+encoded+within+RNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001844-Harding1" id="pone.0001844-Harding1"></a>Harding PL, Fall AM, Honeyman K, Fletcher S, Wilton SD (2007) The influence of antisense oligonucleotide length on dystrophin exon skipping. Mol Ther  15: 157–166.  <ul class="find" data-citedArticleID="1078572" data-doi="10.1038/sj.mt.6300006"><li><a href="http://dx.doi.org/10.1038/sj.mt.6300006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+influence+of+antisense+oligonucleotide+length+on+dystrophin+exon+skipping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+influence+of+antisense+oligonucleotide+length+on+dystrophin+exon+skipping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001844-Vickers1" id="pone.0001844-Vickers1"></a>Vickers TA, Wyatt JR, Freier SM (2000) Effects of RNA secondary structure on cellular antisense activity. Nucleic Acids Res  28: 1340–1347.  <ul class="find" data-citedArticleID="1078640" data-doi="10.1093/nar/28.6.1340"><li><a href="http://dx.doi.org/10.1093/nar/28.6.1340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+RNA+secondary+structure+on+cellular+antisense+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+RNA+secondary+structure+on+cellular+antisense+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001844-Lehmann1" id="pone.0001844-Lehmann1"></a>Lehmann MJ, Patzel V, Sczakiel G (2000) Theoretical design of antisense genes with statistically increased efficacy. Nucleic Acids Res  28: 2597–2604.  <ul class="find" data-citedArticleID="1078584" data-doi="10.1093/nar/28.13.2597"><li><a href="http://dx.doi.org/10.1093/nar/28.13.2597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theoretical+design+of+antisense+genes+with+statistically+increased+efficacy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theoretical+design+of+antisense+genes+with+statistically+increased+efficacy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001844-KretschmerKazemi1" id="pone.0001844-KretschmerKazemi1"></a>Kretschmer-Kazemi FR, Sczakiel G (2003) The activity of siRNA in mammalian cells is related to structural target accessibility: a comparison with antisense oligonucleotides. Nucleic Acids Res  31: 4417–4424.  <ul class="find" data-citedArticleID="1078580" data-doi="10.1093/nar/gkg649"><li><a href="http://dx.doi.org/10.1093/nar/gkg649" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+activity+of+siRNA+in+mammalian+cells+is+related+to+structural+target+accessibility%3A+a+comparison+with+antisense+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+activity+of+siRNA+in+mammalian+cells+is+related+to+structural+target+accessibility%3A+a+comparison+with+antisense+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001844-Sczakiel1" id="pone.0001844-Sczakiel1"></a>Sczakiel G (2000) Theoretical and experimental approaches to design effective antisense oligonucleotides. Front Biosci  5: D194–D201.  <ul class="find" data-citedArticleID="1078626" data-doi="10.2741/sczakiel"><li><a href="http://dx.doi.org/10.2741/sczakiel" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theoretical+and+experimental+approaches+to+design+effective+antisense+oligonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theoretical+and+experimental+approaches+to+design+effective+antisense+oligonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001844-Cartegni2" id="pone.0001844-Cartegni2"></a>Cartegni L, Wang J, Zhu Z, Zhang MQ, Krainer AR (2003) ESEfinder: A web resource to identify exonic splicing enhancers. Nucleic Acids Res  31: 3568–3571.  <ul class="find" data-citedArticleID="1078544" data-doi="10.1093/nar/gkg616"><li><a href="http://dx.doi.org/10.1093/nar/gkg616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ESEfinder%3A+A+web+resource+to+identify+exonic+splicing+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ESEfinder%3A+A+web+resource+to+identify+exonic+splicing+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001844-Fairbrother1" id="pone.0001844-Fairbrother1"></a>Fairbrother WG, Yeh RF, Sharp PA, Burge CB (2002) Predictive identification of exonic splicing enhancers in human genes. Science  297: 1007–1013.  <ul class="find" data-citedArticleID="1078558" data-doi="10.1126/science.1073774"><li><a href="http://dx.doi.org/10.1126/science.1073774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predictive+identification+of+exonic+splicing+enhancers+in+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predictive+identification+of+exonic+splicing+enhancers+in+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001844-Errington1" id="pone.0001844-Errington1"></a>Errington SJ, Mann CJ, Fletcher S, Wilton SD (2003) Target selection for antisense oligonucleotide induced exon skipping in the dystrophin gene. J Gene Med  5: 518–527.  <ul class="find" data-citedArticleID="1078556" data-doi="10.1002/jgm.361"><li><a href="http://dx.doi.org/10.1002/jgm.361" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Target+selection+for+antisense+oligonucleotide+induced+exon+skipping+in+the+dystrophin+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Target+selection+for+antisense+oligonucleotide+induced+exon+skipping+in+the+dystrophin+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001844-Mathews1" id="pone.0001844-Mathews1"></a>Mathews DH, Sabina J, Zuker M, Turner DH (1999) Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure. J Mol Biol  288: 911–940.  <ul class="find" data-citedArticleID="1078592" data-doi="10.1006/jmbi.1999.2700"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.2700" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expanded+sequence+dependence+of+thermodynamic+parameters+improves+prediction+of+RNA+secondary+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expanded+sequence+dependence+of+thermodynamic+parameters+improves+prediction+of+RNA+secondary+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001844-Alberts1" id="pone.0001844-Alberts1"></a>Alberts B, Johnson A, Lewis J, Raff M, Roberts K, et al.  (2002) Molecular Biology of the Cell. London: Garland Science, Taylor &amp; Francis. pp. 317–326.  <ul class="find-nolinks"></ul></li><li><span class="label">71.
              </span><a name="pone.0001844-Beyer2" id="pone.0001844-Beyer2"></a>Beyer AL, Osheim YN (1988) Splice site selection, rate of splicing, and alternative splicing on nascent transcripts. Genes Dev  2: 754–765.  <ul class="find" data-citedArticleID="1078534" data-doi="10.1101/gad.2.6.754"><li><a href="http://dx.doi.org/10.1101/gad.2.6.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Splice+site+selection%2C+rate+of+splicing%2C+and+alternative+splicing+on+nascent+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Splice+site+selection%2C+rate+of+splicing%2C+and+alternative+splicing+on+nascent+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001844-R1" id="pone.0001844-R1"></a>R Development Core Team (2004) R: A language and environment for statistical computing. R Foundation for Statistical Computing, Viena, Austria. ISBN 3-900051-07-0.   URL <a href="http://www.R-project org">http://www.R-project org</a>.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.XML" value="148608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.PDF" value="637400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g001.PNG_L" value="36300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g001.PNG_M" value="63256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g001.PNG_S" value="8163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g001.TIF" value="248316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g001.PNG_I" value="22923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e001.PNG" value="19734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e001.TIF" value="36600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e002.PNG" value="20704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e002.TIF" value="37632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g002.PNG_L" value="710755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g002.PNG_M" value="163835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g002.PNG_S" value="18637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g002.TIF" value="1161910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g002.PNG_I" value="93609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t001.PNG_L" value="193469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t001.PNG_M" value="112962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t001.PNG_S" value="13947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t001.TIF" value="884690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t001.PNG_I" value="35073"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g003.PNG_L" value="51432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g003.PNG_M" value="74608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g003.PNG_S" value="6487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g003.TIF" value="350676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g003.PNG_I" value="27421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e003.PNG" value="20521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e003.TIF" value="37532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e004.PNG" value="20136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e004.TIF" value="37012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g004.PNG_L" value="36460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g004.PNG_M" value="49747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g004.PNG_S" value="7341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g004.TIF" value="402294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g004.PNG_I" value="20010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e005.PNG" value="19277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e005.TIF" value="35920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e006.PNG" value="27213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e006.TIF" value="44560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e007.PNG" value="26361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.e007.TIF" value="43808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g005.PNG_L" value="66368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g005.PNG_M" value="65300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g005.PNG_S" value="7435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g005.TIF" value="572664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g005.PNG_I" value="31215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t002.PNG_L" value="150522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t002.PNG_M" value="85995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t002.PNG_S" value="12690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t002.TIF" value="675292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.t002.PNG_I" value="26951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g006.PNG_L" value="83115"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g006.PNG_M" value="107084"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g006.PNG_S" value="7878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g006.TIF" value="626096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g006.PNG_I" value="41661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g007.PNG_L" value="116030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g007.PNG_M" value="50522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g007.PNG_S" value="6437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g007.TIF" value="380090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.g007.PNG_I" value="18710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s001.DOC" value="59904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s002.DOC" value="57856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s003.DOC" value="75264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s004.DOC" value="202752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s005.DOC" value="475136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s006.DOC" value="1927680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s007.DOC" value="73728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001844.s008.DOC" value="117248"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001844&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001844" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001844&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001844&volume=&issue=&title=Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides&author_name=Keng%20Boon%20Wee%2C%20Zacharias%20Aloysius%20Dwi%20Pramono%2C%20Jian%20Li%20Wang%2C%20Karl%20F.%20MacDorman%2C%20Poh%20San%20Lai%2C%20Woon%20Chee%20Yee&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001844" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844&amp;t=Dynamics%20of%20Co-Transcriptional%20Pre-mRNA%20Folding%20Influences%20the%20Induction%20of%20Dystrophin%20Exon%20Skipping%20by%20Antisense%20Oligonucleotides" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844&title=Dynamics%20of%20Co-Transcriptional%20Pre-mRNA%20Folding%20Influences%20the%20Induction%20of%20Dystrophin%20Exon%20Skipping%20by%20Antisense%20Oligonucleotides&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844&amp;title=Dynamics%20of%20Co-Transcriptional%20Pre-mRNA%20Folding%20Influences%20the%20Induction%20of%20Dystrophin%20Exon%20Skipping%20by%20Antisense%20Oligonucleotides" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001844&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Dynamics of Co-Transcriptional Pre-mRNA Folding Influences the Induction of Dystrophin Exon Skipping by Antisense Oligonucleotides';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001844';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Dynamics%20of%20Co-Transcriptional%20Pre-mRNA%20Folding%20Influences%20the%20Induction%20of%20Dystrophin%20Exon%20Skipping%20by%20Antisense%20Oligonucleotides http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001844" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001844" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="26512"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26512"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="26512"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Dystrophin%22" title="Search for articles in the subject area:'Dystrophin'"><div class="flagText">Dystrophin</div></a>
              <div data-categoryid="23541" data-articleid="26512"
                   data-categoryname="Dystrophin"
                   class="flagImage" title="Flag 'Dystrophin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="26512"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="26512"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26512"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26512"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6342'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1415'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9804&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17499">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=11561">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
