

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001182"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001182" />

    <meta name="citation_title" content="RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization"/>
    <meta itemprop="name" content="RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization"/>

      <meta name="citation_author" content="Antonio Rodríguez-Campos"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology of Barcelona (IBMB), CSIC, and Institute for Research in Biomedicine (IRB), Parc Científic de Barcelona, Barcelona, Spain"/>
      <meta name="citation_author" content="Fernando Azorín"/>
            <meta name="citation_author_institution" content="Institute of Molecular Biology of Barcelona (IBMB), CSIC, and Institute for Research in Biomedicine (IRB), Parc Científic de Barcelona, Barcelona, Spain"/>

    <meta name="citation_date" content="2007/11/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001182.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1182"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Structure of chromatin.; citation_author=RD Kornberg; citation_journal_title=Annu Rev Biochem; citation_volume=46; citation_number=1; citation_pages=931-954; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Nucleosome structure.; citation_author=JD McGhee; citation_author=G Felsenfeld; citation_journal_title=Annu Rev Biochem; citation_volume=49; citation_number=2; citation_pages=1115-1156; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Structure of nucleosome core particle.; citation_author=T Finch; citation_author=LC Lutter; citation_author=D Rhodes; citation_author=AS Brown; citation_author=B Rushton; citation_journal_title=Nature (London); citation_volume=269; citation_number=3; citation_pages=29-36; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Structure of nucleosome core particle at 7Å resolution.; citation_author=TJ Richmond; citation_author=JT Finch; citation_author=B Rushton; citation_author=D Rhodes; citation_author=A Klug; citation_journal_title=Nature (London); citation_volume=311; citation_number=4; citation_pages=532-537; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the nucleoosme core particle at 2,8Å resolution.; citation_author=K Luger; citation_author=AW Mäder; citation_author=RK Richmond; citation_author=DF Sargent; citation_author=TJ Richmond; citation_journal_title=Nature; citation_volume=389; citation_number=5; citation_pages=251-260; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Electrostatic mechanism of chromatin folding.; citation_author=DJ Clark; citation_author=T Kimura; citation_journal_title=J Mol Biol; citation_volume=211; citation_number=6; citation_pages=883-896; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Role of histone “tails” in the folding of oligonucleosomes depleted of histone H1.; citation_author=M Garcia-Ramírez; citation_author=F Dong; citation_author=J Ausió; citation_journal_title=J Biol Chem; citation_volume=267; citation_number=7; citation_pages=19587-19595; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Core histone tail domains mediate oligonucleome folding and nucleosomal DNA organization through distinct molecular mechanisms.; citation_author=TM Fletcher; citation_author=JC Hansen; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=8; citation_pages=25359-25362; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Reversible oligonuucleosome self-association: dependence of divalent cations and core histone tail domains.; citation_author=PM Schwarz; citation_author=A Felthauser; citation_author=TM Fletcher; citation_author=JC Hansen; citation_journal_title=Biochemistry; citation_volume=35; citation_number=9; citation_pages=4009-4015; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Toward a unified model of chromatin folding. Annu Rev Bioph.; citation_author=J Widom; citation_journal_title=Chem; citation_volume=18; citation_number=10; citation_pages=365-395; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=In Chromatin (Rich A ed.),; citation_author=KH Van Holde; citation_number=11; citation_date=1988; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=DNA conformation and protein binding.; citation_author=AA Travers; citation_journal_title=Annu Rev Biochem; citation_volume=58; citation_number=12; citation_pages=427-452; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Epigenetic codes for heterochromatin formation and silencing: rounding up the usual suspects.; citation_author=EJ Richards; citation_author=S Elgin; citation_journal_title=Cell; citation_volume=108; citation_number=13; citation_pages=489-500; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Chromatin-based silencing mechanisms.; citation_author=J Bender; citation_journal_title=Curr Opin Plant Biol; citation_volume=7; citation_number=14; citation_pages=521-526; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The role of RNA interference in heterochromatic silencing.; citation_author=Z Lippman; citation_author=R Martienssen; citation_journal_title=Nature; citation_volume=431; citation_number=15; citation_pages=364-370; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=RNAi components are required for nuclear clustering of Polycomb group response elements.; citation_author=C Grimaud; citation_author=F Bantignies; citation_author=M Pal-Bhadra; citation_author=P Ghana; citation_author=U Bhadra; citation_journal_title=Cell; citation_volume=124; citation_number=16; citation_pages=957-971; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Noncoding RNA genes in dosage compensation and imprinting.; citation_author=RL Kelley; citation_author=MI Kuroda; citation_journal_title=Cell; citation_volume=103; citation_number=17; citation_pages=9-12; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=X-chromosome inactivation: counting, choice and initiation.; citation_author=P Avner; citation_author=E Heard; citation_journal_title=Nature Rev Genet; citation_volume=2; citation_number=18; citation_pages=59-67; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Higher-order structure in pericentric heterochromatin involves a distinct pattern of histone modification and an RNA component.; citation_author=C Maison; citation_author=D Bailly; citation_author=AHFM Peters; citation_author=JP Quivy; citation_author=D Roche; citation_journal_title=Nature Genet; citation_volume=30; citation_number=19; citation_pages=329-334; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sucrose gradient techniques and applications to nucleosome structure.; citation_author=H Noll; citation_author=M Noll; citation_journal_title=In Methods in Enzymology (Wassarman PM, Kornberg RD eds.); citation_volume=170; citation_number=20; citation_pages=55-116; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A multiple ribosomal structure in protein synthesis.; citation_author=JR Warner; citation_author=PM Knopf; citation_author=A Rich; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=49; citation_number=21; citation_pages=122-129; citation_date=1963; " />
      <meta name="citation_reference" content="citation_title=Ribosomal aggregate engaged in protein synthesis: characterization of the ergosome.; citation_author=O Wettstein; citation_author=T Staehelin; citation_author=H Noll; citation_journal_title=Nature; citation_volume=197; citation_number=22; citation_pages=430-435; citation_date=1963; " />
      <meta name="citation_reference" content="citation_title=Subunit structure of chromatin.; citation_author=M Noll; citation_journal_title=Nature; citation_volume=251; citation_number=23; citation_pages=249-251; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Action of micrococcal nuclease on chromatin and the location of histone H1.; citation_author=M Noll; citation_author=RD Kornberg; citation_journal_title=J Mol Biol; citation_volume=109; citation_number=24; citation_pages=393-404; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Histone H1 represses transcription from minichromosomes assembled in vitro.; citation_author=A Shimamura; citation_author=M Sapp; citation_author=A Rodríguez-Campos; citation_author=A Worcel; citation_journal_title=Mol Cell Biol; citation_volume=9; citation_number=25; citation_pages=5573-5584; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A chicken homeobox gene related to Drosophila paired is predominantly expressed in the developing limb.; citation_author=T Nohno; citation_author=E Koyama; citation_author=F Myokai; citation_author=S Taniguchi; citation_author=H Ohuchi; citation_journal_title=Dev Biol; citation_volume=158; citation_number=26; citation_pages=254-264; " />
      <meta name="citation_reference" content="citation_title=A 41–42 bp tandemly repeated sequences isolated from nuclear envelopes of chicken erythrocytes is located predominantly on minichromosomes.; citation_author=MA Matzke; citation_author=F Varga; citation_author=H Berger; citation_author=J Schernthaner; citation_author=D Scheizer; citation_journal_title=Chromosoma (Berl); citation_volume=99; citation_number=27; citation_pages=131-137; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=α-amanitin: a specific inhibitor of one of two DNA-dependent RNA polymerase activities from calf thymus.; citation_author=C Kedinger; citation_author=M Gmazdowski; citation_author=JL Madel; citation_author=F Gissinger; citation_author=P Chambon; citation_journal_title=Biochem. Biophys. Res. Comm.; citation_volume=38; citation_number=28; citation_pages=165-171; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Digestion of RNA of chromatin and ribonucleoprotein by staphylococcal nuclease.; citation_author=LH Augenlicht; citation_author=M McCormick; citation_author=M Lipkin; citation_journal_title=Biochemistry; citation_volume=15; citation_number=29; citation_pages=3818-3823; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Bilogical function of unnanotated transcription during the early development of Drosophila melanogaster.; citation_author=JR Manak; citation_author=S Dike; citation_author=V Sementchenko; citation_author=P Kapranov; citation_author=F Biemar; citation_journal_title=Nature Genet; citation_volume=38; citation_number=30; citation_pages=1151-1158; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs: genomics, biogenesis, mechanism and function.; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=116; citation_number=31; citation_pages=281-297; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Small regulatory RNAs in mammals.; citation_author=JS Mattick; citation_author=IV Makunin; citation_journal_title=Hum Mol Genet; citation_volume=14 (Suppl 1); citation_number=32; citation_pages=R121-R132; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=In Noncoding RNAs: Molecular biology and molecular medicine, (Barciszewsky J, Erdmann VA eds.): 49 Georgetown Landes Bioscience.; citation_author=A Wutz; citation_journal_title=Texas; citation_number=33; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The non coding Air RNA is required for silencing autosomal imprinted genes.; citation_author=F Sleutels; citation_author=R Zwart; citation_author=DP Barlow; citation_journal_title=Nature; citation_volume=415; citation_number=34; citation_pages=810-813; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Clusters of internally primed transcripts reveal a novel long noncoding RNAs.; citation_author=M Furuno; citation_author=KC Pang; citation_author=N Ninomiya; citation_author=S Fukuda; citation_author=M Frith; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=35; citation_pages=e37. 0537-0553; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Distinctive higher-order chromatin structure at mammalian centromeres.; citation_author=N Gilbert; citation_author=J Allan; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=36; citation_pages=11949-11954; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Physical properties of a genomic condensed chromatin fragments.; citation_author=R Ghirlando; citation_author=MD Litt; citation_author=MN Prioleau; citation_author=F Recillas-Targa; citation_author=G Felsenfeld; citation_journal_title=J Mol Biol; citation_volume=336; citation_number=37; citation_pages=597-605; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Coordinated methyl and RNA binding is required for heterochromatin localization of mammalian HP1α; citation_author=C Muchardt; citation_author=M Guillemé; citation_author=JS Seeler; citation_author=D Trouche; citation_author=A Dejean; citation_journal_title=EMBO Rep; citation_volume=3; citation_number=38; citation_pages=975-81; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization"/>
    <meta name="twitter:description" content="Chromatin structure is influenced by multiples factors, such as pH, temperature, nature and concentration of counterions, post-translational modifications of histones and binding of structural non-histone proteins. RNA is also known to contribute to the regulation of chromatin structure as chromatin-induced gene silencing was shown to depend on the RNAi machinery in S. pombe, plants and Drosophila. Moreover, both in Drosophila and mammals, dosage compensation requires the contribution of specific non-coding RNAs. However, whether RNA itself plays a direct structural role in chromatin is not known. Here, we report results that indicate a general structural role for RNA in eukaryotic chromatin. RNA is found associated to purified chromatin prepared from chicken liver, or cultured Drosophila S2 cells, and treatment with RNase A alters the structural properties of chromatin. Our results indicate that chromatin-associated RNAs, which account for 2%&ndash;5% of total chromatin-associated nucleic acids, are polyA&minus; and show a size similar to that of the DNA contained in the corresponding chromatin fragments. Chromatin-associated RNA(s) are not likely to correspond to nascent transcripts as they are also found bound to chromatin when cells are treated with &alpha;-amanitin. After treatment with RNase A, chromatin fragments of molecular weight &gt;3.000 bp of DNA showed reduced sedimentation through sucrose gradients and increased sensitivity to micrococcal nuclease digestion. This structural transition, which is observed both at euchromatic and heterochromatic regions, proceeds without loss of histone H1 or any significant change in core-histone composition and integrity."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001182.g008"/>

  <meta property="og:title" content="RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=54'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=79'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7902&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001182">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001182" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001182&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Antonio Rodríguez-Campos
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:arobmc@gmail.com">arobmc@gmail.com</a></p>

                <p>Affiliation:
                  Institute of Molecular Biology of Barcelona (IBMB), CSIC, and Institute for Research in Biomedicine (IRB), Parc Científic de Barcelona, Barcelona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fernando Azorín
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Molecular Biology of Barcelona (IBMB), CSIC, and Institute for Research in Biomedicine (IRB), Parc Científic de Barcelona, Barcelona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001182</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001182-g001" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g002" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g003" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g004" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g005" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g006" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g007" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001182-g008" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001182">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001182" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Chromatin structure is influenced by multiples factors, such as pH, temperature, nature and concentration of counterions, post-translational modifications of histones and binding of structural non-histone proteins. RNA is also known to contribute to the regulation of chromatin structure as chromatin-induced gene silencing was shown to depend on the RNAi machinery in <em>S. pombe</em>, plants and <em>Drosophila</em>. Moreover, both in <em>Drosophila</em> and mammals, dosage compensation requires the contribution of specific non-coding RNAs. However, whether RNA itself plays a direct structural role in chromatin is not known. Here, we report results that indicate a general structural role for RNA in eukaryotic chromatin. RNA is found associated to purified chromatin prepared from chicken liver, or cultured <em>Drosophila</em> S2 cells, and treatment with RNase A alters the structural properties of chromatin. Our results indicate that chromatin-associated RNAs, which account for 2%–5% of total chromatin-associated nucleic acids, are polyA<sup>−</sup> and show a size similar to that of the DNA contained in the corresponding chromatin fragments. Chromatin-associated RNA(s) are not likely to correspond to nascent transcripts as they are also found bound to chromatin when cells are treated with α-amanitin. After treatment with RNase A, chromatin fragments of molecular weight &gt;3.000 bp of DNA showed reduced sedimentation through sucrose gradients and increased sensitivity to micrococcal nuclease digestion. This structural transition, which is observed both at euchromatic and heterochromatic regions, proceeds without loss of histone H1 or any significant change in core-histone composition and integrity.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rodríguez-Campos A, Azorín F (2007) RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization. PLoS ONE 2(11):
          e1182.
            doi:10.1371/journal.pone.0001182</p><p><strong>Academic Editor: </strong>Giacomo Cavalli, Centre National de la Recherche Scientifique, France</p><p><strong>Received:</strong> May 17, 2007; <strong>Accepted:</strong> October 25, 2007; <strong>Published:</strong> November 14, 2007</p><p><strong>Copyright:</strong> © 2007 Rodríguez-Campos, Azorin. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was financed by grants from the MCyT (BMC2003-243), the CIRIT (2001SGR00344 and 2005SGR00678), and the EU (LSHB-CT-2004-511965). A.R.-C. acknowledges receipt of an I3P postdoctoral contract from CSIC.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In eukaryotes, histones pack DNA into chromatin, a periodic structure of regularly spaced nucleosomes <a href="#pone.0001182-Kornberg1">[1]</a>. In the nucleosome, 145–147 bp of DNA are wrapped around a protein core formed by two copies of each histone H2A, H2B, H3 and H4. This evolutionarily conserved nucleoprotein complex is evenly distribute, every 200±40 bp, all throughout the eukaryotic genome <a href="#pone.0001182-McGhee1">[2]</a>. Much is known about nucleosome structure as it was determined at atomic resolution <a href="#pone.0001182-Finch1">[3]</a>–<a href="#pone.0001182-Luger1">[5]</a>. In addition to core histones, higher eukaryotes contain linker histone H1. Histone H1 contributes to folding of the nucleosomes into higher-order chromatin structures, which are stabilized by the interaction of both histone H1 and the core histone N-terminal tails with the linker DNA <a href="#pone.0001182-Clark1">[6]</a>–<a href="#pone.0001182-Schwarz1">[9]</a>. Therefore, histone-DNA interactions determine both the basic structural properties of the nucleosome as well as the formation of higher-order chromatin structures <a href="#pone.0001182-Widom1">[10]</a>. Multiple factors, however, are known to influence both nucleosome structure and higher-order chromatin structures <a href="#pone.0001182-VanHolde1">[11]</a>, <a href="#pone.0001182-Travers1">[12]</a>. These include pH, temperature, ionic-strength, DNA bendability, histone modifications and, in particular, binding of structural non-histone proteins (i.e., HP1, Polycomb), which often depends on specific post-translational modifications of the histones N-terminal tails <a href="#pone.0001182-Richards1">[13]</a>. Several results indicate that RNA also influences chromatin structure as, first, chromatin-induced gene silencing depends on the RNAi machinery <a href="#pone.0001182-Bender1">[14]</a>–<a href="#pone.0001182-Grimaud1">[16]</a> and, second, dosage compensation requires the contribution of specific non-coding RNAs <a href="#pone.0001182-Kelley1">[17]</a>, <a href="#pone.0001182-Avner1">[18]</a>. A role of RNA in the formation of higher-order heterochromatin structures was also proposed as, in mammals, HP1 <em>foci</em> are sensitive to treatment with RNase A being recovered upon the addition of total RNA or hnRNA, but not by the addition of tRNA or bacterial mRNA <a href="#pone.0001182-Maison1">[19]</a>. Altogether, these observations indicate a contribution of RNA to the regulation of chromatin structure and function. Whether RNA itself plays a direct structural role in chromatin is, however, not known.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Here, we report results indicating that RNA plays a general structural role in eukaryotic chromatin. Our results indicate that purified chromatin contains significant amounts of RNA (2%–5% of total nucleic acids). Whether this chromatin-associated RNA(s) contributes to chromatin structure was addressed by analyzing the sedimentation behavior through linear sucrose gradients of native oligonucleosomal fragments, purified from chicken liver or cultured <em>Drosophila</em> S2 cells, before and after treatment with RNase A. Density gradient centrifugation is a powerful method to separate and analyze macromolecules <a href="#pone.0001182-Noll1">[20]</a>. A classical application of sedimentation in sucrose gradients is the analysis of repetitive nucleoprotein structures such as polysomes <a href="#pone.0001182-Warner1">[21]</a>, <a href="#pone.0001182-Wettstein1">[22]</a> and nucleosomes <a href="#pone.0001182-Noll2">[23]</a>. Soluble chromatin fragments, obtained by mild micrococcal nuclease digestion of purified nuclei, are resolved by density gradient centrifugation into fractions of homogeneous composition and precise molecular entity, from mononucleosomes to oligonucleosomes <a href="#pone.0001182-Noll1">[20]</a>. Our results show that, upon treatment with RNase A, chromatin fragments display a clear shift towards the lighter zone of the gradient, which is abolished in the presence of the specific RNase inhibitor, antiRNase. Decreased sedimentation is observed for bulk chromatin as well as for chromatin at specific genomic locations, either euchromatic or heterochromatic, and is accompanied by an increased sensitivity to MNase digestion. Altogether, these results indicate that RNA is an integral component of chromatin that contributes to its structural organization.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Purified chromatin contains RNA</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As shown in <a href="#pone-0001182-g001">Figure 1</a>, purified chicken liver chromatin contains RNA. In these experiments, chicken liver chromatin was prepared by micrococcal nuclease (MNase, Sigma) digestion of isolated nuclei that were purified by centrifugation through a sucrose cushion to avoid as much as possible contamination with cytoplasmic RNA. After MNase digestion, soluble chromatin was fractionated by centrifugation through a linear 5%–30% sucrose gradient and chromatin fractions were subjected to total nucleic acids extraction or to selective RNA extraction using Ultraspec™ RNA Isolation System (Biotecx). Density gradient centrifugation results in the typical distribution of chromatin fragments in which more dense fractions contain longer oligonucleosomal fragments (<a href="#pone-0001182-g001">Figure 1A</a>, left). Selective RNA extraction from these purified chromatin fractions renders significant amounts of product as determined by Northern blotting using total high-weight chicken genomic DNA as a probe (<a href="#pone-0001182-g001">Figure 1A</a>, right). As shown in <a href="#pone-0001182-g001">Figure 1</a>, the Ultraspec-extracted material shows an average size very similar to that of the DNA contained in the same chromatin fraction. The Ultraspec-extracted material is sensitive to digestion with RNase A. In this experiment, fractions 7 and 10 of the gradient shown in <a href="#pone-0001182-g001">Figure 1A</a> were subjected to selective RNA extraction and, then, treated with RNase A, or not, prior to Northern analysis (<a href="#pone-0001182-g001">Figure 1B</a>). After RNase A treatment, no hybridization signal is observed. To further confirm its RNA nature, the material obtained after RNA extraction from a mixture of fractions 7, 8 and 9 of the same gradient (<a href="#pone-0001182-g001">Figure 1A</a>, left) was radioactively labeled by reverse transcription in the absence of any treatment (<a href="#pone-0001182-g001">Figure 1C</a>, lane 2) or after treatment with RNase A (<a href="#pone-0001182-g001">Figure 1C</a>, lane 1). No labeling can be observed upon RNase A treatment. On the contrary, strong labeling is observed when RNase A digestion was carried out in the presence of the RNase inhibitor anti-RNase (Ambion) (<a href="#pone-0001182-g001">Figure 1C</a>, lane 3) or when the material was subjected to digestion with DNase I (<a href="#pone-0001182-g001">Figure 1C</a>, lane 4). Altogether, these results indicate that purified chicken liver chromatin contains significant amounts of RNA, which accounts for around 2% to 5% of total nucleic acids content (see <a href="#s4">Materials and Methods</a> for the quantitative determination of the RNA content of purified chromatin). As shown in <a href="#pone-0001182-g001">Figure 1D</a>, the vast majority of chromatin-associated RNA(s) is not bound by oligo-dT affinity resin, indicating that it is polyA<sup>−</sup>.</p>
<div class="figure" id="pone-0001182-g001"><div class="img"><a name="pone-0001182-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Purified chicken liver chromatin contains RNA.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) Chicken liver chromatin was prepared by micrococcal nuclease digestion of purified nuclei and then subjected to sedimentation through a linear 5%–30% sucrose gradients. After centrifugation, 1ml fractions were collected, subjected to total nucleic acids extraction and analyzed by electrophoresis in 1% agarose-TBE gels (left panel). In parallel, chromatin fractions were subjected to RNA extraction using Ultraspec™ RNA Isolation System (Biotecx) and analyzed by Northern blotting in a glyoxal-1% agarose gel using high molecular weight genomic chicken DNA as a probe (right panel). Fraction numbers are indicated. Lanes M correspond to molecular weight markers. B) Chromatin fractions 7 and 10 of the gradient shown in A) were subjected to RNA extraction as indicated above, treated with RNase A (lanes 3 and 4) or not (lanes 1 and 2), and analyzed by Northern blotting as in A). Lane M, corresponds to molecular weight markers. Lane P, corresponds to the probe used. C) Chromatin from a mixture of fractions 7, 8 and 9 of the gradient shown in A) were subjected to RNA extraction and either untreated (lane 2), treated with RNase A in the absence (lane 1) or in the presence of anti-RNase (Ambion) (lane 3), or treated with DNase I (Roche) (lane 4). After phenol extraction and isopropanol precipitation, samples were <sup>32</sup>P-labeled by reverse transcription with Omniscript® RT Kit (Qiagen) (2 h at 37°C) using a mixture of hexanucleotides of random sequence. Samples were then analyzed in a 1% agarose-TBE gel, blotted and the membrane directly exposed. Lane M, corresponds to molecular weight markers. D) Chromatin-associated RNA was purified and incubated with oligo-dT immobilized resin (Oligotex™ mRNA Purification System, QIAGEN). After elution, bound (lane 2) and unbound material (lane1) were analyzed in a glyoxal-1% agarose, 10 mM sodium phosphate (pH 6,8) gel. Lanes M correspond to molecular weight markers.</p>
<span>doi:10.1371/journal.pone.0001182.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In the experiments described above, sucrose gradient centrifugation was performed at low ionic strength (0,2 mM EDTA). RNA, however, remains associated to chromatin at high ionic strength (<a href="#pone-0001182-g002">Figure 2</a>). In these experiments, chicken liver chromatin was fractionated by centrifugation through a linear 5%–30% sucrose gradient containing 0,65 M NaCl. At this ionic strength, histone H1 is known to be released from chromatin <a href="#pone.0001182-VanHolde1">[11]</a>, <a href="#pone.0001182-Noll1">[20]</a> and, accordingly, no histone H1 was detected when the protein content of chromatin fractions prepared at 0,65 M NaCl was analised by SDS-PAGE electrophoresis (<a href="#pone-0001182-g002">Figure 2B</a>). Under these conditions, however, chromatin fractions contained significant amounts of RNA (<a href="#pone-0001182-g002">Figure 2A</a>), indicating that RNA associates to chromatin tighter than histone H1.</p>
<div class="figure" id="pone-0001182-g002"><div class="img"><a name="pone-0001182-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>RNA remains associated to chromatin at high ionic strength.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A) Chicken liver chromatin was prepared by micrococcal nuclease digestion of purified nuclei and then subjected to sedimentation through a linear 5%–30% sucrose gradients containing 0,65 M NaCl. After centrifugation, fractions were collected and their DNA (top) and RNA (bottom) content determined as in <a href="#pone-0001182-g001">Figure 1</a>. Fraction numbers are indicated. Lane M, corresponds to molecular weight markers. B) The histone content of each fraction was analyzed by SDS-PAGE (lanes 1-P). As controls, H1 from calf thymus and hydroxylapatite-purified core histones from chicken liver are also presented. The gel was stained with silver.</p>
<span>doi:10.1371/journal.pone.0001182.g002</span></div>

<h4>Treatment with RNase A alters the sedimentation behavior of chromatin</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In these experiments, prior to sedimentation through linear 5%–30% sucrose gradients, bulk chicken liver chromatin obtained by mild MNase digestion of purified nuclei was either digested with RNase A (<a href="#pone-0001182-g003">Figure 3A</a>, bottom panel) or not (<a href="#pone-0001182-g003">Figure 3A</a>, top panel). As shown in <a href="#pone-0001182-g003">Figure 3A</a>, equivalent fractions of the same density contain chromatin fragments of higher molecular weight when chromatin is digested with RNase A than when it is not. This effect increases with the size of the nucleosomal fragments (<a href="#pone-0001182-g003">Figure 3A</a>, graph), so that chromatin fragments of up to 1.000 bp of DNA show no significant change in sedimentation after treatment with RNase A. On the contrary, chromatin fragments of molecular weight &gt;3.000 bp of DNA show a strong change in sedimentation after digestion with RNase A. Decreased sedimentation observed after treatment with RNase A requires actual degradation of an RNA component as it is not observed when treatment with RNase A is carried out in the presence of the specific RNase A inhibitor anti-RNase (<a href="#pone-0001182-g003">Figure 3C</a>). Moreover, the shift in the sedimentation rate observed upon treatment with RNase A is not a consequence of the loss or degradation of the histone component (<a href="#pone-0001182-g003">Figure 3B</a>). In these experiments, chromatin from fractions 11 and 12 of the gradients shown in <a href="#pone-0001182-g003">Figure 3A</a> was precipitated by the addition of MgCl<sub>2 </sub>and NaCl, and the pellets were dissolved, loaded and electrophoresed in a PAGE-SDS gel. No significant change in histone composition or integrity is detected after treatment with RNase A (<a href="#pone-0001182-g003">Figure 3B</a>, compare lanes 4 and 5 with lanes 6 and 7). In particular, treatment with RNase A does not result in any significant change in histone H1 content (<a href="#pone-0001182-g003">Figure 3B</a>, graph). This observation is especially relevant as it is known that the degree of compactness of chromatin and, therefore, its sedimentation rate is strongly dependent on the presence of histone H1 <a href="#pone.0001182-Noll3">[24]</a>, <a href="#pone.0001182-Shimamura1">[25]</a>.</p>
<div class="figure" id="pone-0001182-g003"><div class="img"><a name="pone-0001182-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Treatment with RNase A alters the sedimentation behavior of purified chicken liver chromatin.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) Chicken liver chromatin, prepared by micrococcal nuclease digestion of purified nuclei, was subjected to sedimentation through a linear 5%–30% sucrose gradient after treatment with RNase A (bottom panel) or not (top panel). After centrifugation and fractionation, samples were subjected to total nucleic acids extraction and analyzed by electrophoresis in 1% agarose-TBE gels. Fraction numbers are indicated. Lanes M correspond to molecular weight markers. Quantitation of the results is shown on the right where the average molecular weight (M) of the chromatin fragments contained in each fraction, expressed as bp of DNA, is presented as a function of the fraction number: (°) untreated chromatin, (•) chromatin treated with RNase A. B) Histone content of chromatin fractions 11 and 12 of the gradients shown in A) was analyzed by SDS-PAGE: untreated chromatin (lanes 4 and 5), chromatin treated with RNase A (lanes 6 and 7) and, as controls, H1 from calf thymus (lane 1), RNase A (lane 2) and hydroxylapatite-purified core histones from chicken liver (lane 3). The gel was stained with silver. Quantitation of the results is shown on the right where the H1/H2A ratio of fractions 11 and 12 is presented before (white columns) and after (black columns) treatment with RNase A. C) The sedimentation behavior of chicken liver chromatin was determined before (left panel) and after treatment RNase A either in the presence of anti-RNase (Ambion) (central panel) or in the absence of any added inhibitor (right panel). Fraction numbers are indicated. Lanes M, correspond to molecular weight markers. Quantitation of the results is shown on the right: (°) untreated chromatin, (▴) chromatin treated with RNase A in the presence of anti-RNase, (•) chromatin treated with RNase A in the absence of any added inhibitor.</p>
<span>doi:10.1371/journal.pone.0001182.g003</span></div>

<h4>Treatment with RNase A alters chromatin structure at both euchromatic and heterochromatic regions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Results shown above indicate that treatment with RNase A reduces the sedimentation rate of bulk chicken liver chromatin. Similar results were obtained when the effects of treatment with RNase A on the chromatin structure of specific genomic regions were determined by Southern blotting using specific probes (<a href="#pone-0001182-g004">Figure 4</a>). In these experiments, two specific genomic regions were analyzed, the euchromatic <em>Pax3 locus</em> <a href="#pone.0001182-Nohno1">[26]</a> (<a href="#pone-0001182-g004">Figure 4A</a>) and the heterochromatic 41–42 bp centromeric chicken satellite <a href="#pone.0001182-Matzke1">[27]</a> (<a href="#pone-0001182-g004">Figure 4B</a>). In both cases decreased sedimentation is observed upon treatment with RNase A. In fact, changes in sedimentation observed at these specific genomic locations are undistinguishable from those observed with bulk chromatin (<a href="#pone-0001182-g004">Figure 4A and 4B</a>, graphs).</p>
<div class="figure" id="pone-0001182-g004"><div class="img"><a name="pone-0001182-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Treatment with RNase A alters the sedimentation behavior of chicken liver chromatin at specific genomic regions.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The effect of treatment with RNase A on the sedimentation behavior of chromatin was determined for bulk chicken liver chromatin (left panels) and for chromatin at specific genomic locations (right panels) by Northern analysis of the gels on the left using specific probes for the <em>Pax3 locus</em> A) and the chicken 41–42 bp centromeric satellite B). Fraction numbers are indicated. Lanes M, correspond to molecular weight markers. Quantitation of the results is shown on the right of each panel: (°) bulk chromatin, (▴) chromatin of the <em>Pax3 locus</em> A) or the chicken 41–42 bp centromeric satellite B).</p>
<span>doi:10.1371/journal.pone.0001182.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Similar results were obtained when chromatin was prepared from cultured <em>Drosophila</em> S2 cells (<a href="#pone-0001182-g005">Figure 5</a>). Also in this case, purified chromatin contains RNA (<a href="#pone-0001182-g006">Figure 6B</a>) and treatment with RNase A decreases sedimentation of bulk S2-chromatin as well as of chromatin of two specific genomic locations, the euchromatic <em>Trl locus</em> (<a href="#pone-0001182-g005">Figure 5A</a>) and the heterochromatic centromeric dodeca-satellite (<a href="#pone-0001182-g005">Figure 5B</a>). As with chicken liver chromatin, no decreased sedimentation is observed when treatment with RNase A is carried out in the presence of anti-RNase (<a href="#pone-0001182-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0001182-g005"><div class="img"><a name="pone-0001182-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Treatment with RNase A alters the sedimentation behavior of purified <em>Drosophila</em> S2 chromatin.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The effect of treatment with RNase A on the sedimentation behavior of chromatin prepared from <em>Drosophila</em> S2 cells was determined for bulk chromatin (left panels) and for chromatin at specific genomic locations (right panels) by Southern analysis of the gels on the left using specific probes for the <em>Trl locus</em> A) and the centromeric <em>Drosophila</em> dodeca-satellite B). Fraction numbers are indicated. Lanes M, correspond to molecular weight markers. Quantitation of the results is shown on the right of each panel for bulk chromatin and chromatin of the <em>Trl locus</em> A) or the centromeric dodeca-satellite B): untreated chromatin (°); chromatin treated with RNase A in the presence of anti-RNase (Ambion) (▴) or in the absence of any added inhibitor (•).</p>
<span>doi:10.1371/journal.pone.0001182.g005</span></div>

<h4>Treatment with α-amanitin does not affect association of RNA to chromatin</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To address the question of whether chromatin-associated RNA(s) corresponds to nascent RNA transcripts, we analysed the effects of blocking transcription by treatment with α-amanitin <a href="#pone.0001182-Kedinger1">[28]</a> (<a href="#pone-0001182-g006">Figure 6</a>). In these experiments, <em>Drosophila</em> S2 cells were treated for 36 h with 0,2 µg/ml of α-amanitin as described under <a href="#s4">Materials and Methods</a>. To determine the effectiveness of this treatment, the levels of nascent RNAs encoding GAGA (<em>Trl</em>), Actin 5C and RP-49 were determined by RT-PCR (<a href="#pone-0001182-g006">Figure 6C</a>). In all three cases, treatment with α-amanitin results in an approximately 3 to 4-fold reduction on the levels of the corresponding nascent RNAs. After treatment, chromatin was digested with MNase and analysed by sedimentation through 5%–30% linear sucrose gradients. As shown in <a href="#pone-0001182-g006">Figure 6A</a>, treatment with α-amanitin does not significantly affect the sedimentation rate of purified chromatin. Moreover, after treatment with α-amanitin, the RNA content of purified chromatin is similar to that observed in chromatin prepared from untreated cells (<a href="#pone-0001182-g006">Figure 6B</a>). These results indicate that chromatin-associated RNA(s) are not likely to correspond to nascent transcripts.</p>
<div class="figure" id="pone-0001182-g006"><div class="img"><a name="pone-0001182-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Treatment with α-amanitin does not alter the sedimentation behavior of purified <em>Drosophila</em> S2 chromatin.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Prior to nuclei isolation and chromatin purification, S2 cells were either treated for 36 h with 0,2 µg/ml α-amanitin or not. A) S2 chromatin from untreated (top) and treated cells (bottom) was purified by micrococcal nuclease digestion of purified nuclei and subjected to sedimentation through linear 5%–30% sucrose gradients as described above. Quantitation of the results is shown on the right where the average molecular weight (M) of the chromatin fragments contained in each fraction, expressed as bp of DNA, is presented as a function of the fraction number for chromatin prepared from untreated (▴) and treated cells (Δ). B) Chromatin fractions 4, 5 and 6 of the gradients shown in A) obtained from cells treated with α-amanitin (lanes 2 and 4) or not (lanes 1 and 3), were subjected to RNA extraction and analised in a glyoxal-1% agarose-sodium phosphate gel before (lanes 1 and 2) and after treatment with RNase A (lanes 3 and 4). Lane M corresponds to molecular weight markers. C and D) Analysis of the efficiency of treatment with α-amanitin. D) Total RNA was prepared from treated (lane 1) or untreated cells (lanes 2), and 5 µg of each were analysed in a 1% agarose-TBE native gel. Lane M corresponds to molecular weight markers. C) Determination of the levels of nascent transcripts encoding GAGA (<em>Trl</em>), actin 5C and RP-49. Increasing amounts of total RNA (0,1, 0,2 and 0,5 µg, lanes 1-3), prepared from untreated (lanes -) and treated cells (lanes +), were analysed by RT-PCR (Omniscript® RT Kit, QIAGEN) as indicated under <a href="#s4">Materials and Methods</a> using appropriate primers to specifically amplify fragments of the Actin 5C (585 bp), <em>Trl</em> (662 bp) and RP-49 (702 bp) genes. Amplified fragments were analysed in a 1% agarose-TBE gel. Lanes M correspond to molecular weight markers.</p>
<span>doi:10.1371/journal.pone.0001182.g006</span></div>

<h4>RNA-depleted chromatin displays a higher sensitivity to MNase digestion</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Decreased sedimentation observed after treatment with RNase A is likely the consequence of a reduction on the degree of compactness of chromatin. Consistent with this hypothesis, accessibility to digestion by MNase increases after RNA depletion (<a href="#pone-0001182-g007">Figure 7</a>). In these experiments, chromatin prepared by mild MNase digestion of isolated nuclei was purified by centrifugation through a linear 5%–30% sucrose gradient (<a href="#pone-0001182-g007">Figure 7A</a>) and chromatin from fraction 8 that, in average, contains oligomers of about 25 nucleosomes (~5.000 bp of DNA) was either treated with RNase A or not, and, then, subjected to further MNase digestion for increasing times (<a href="#pone-0001182-g007">Figure 7B</a>). Upon treatment with RNase A, sensitivity to cleavage by MNase strongly increases as reflected by the increased production of mononucleosomal fragments obtained after equivalent times of digestion (<a href="#pone-0001182-g007">Figure 7C</a>). Similar results were obtained when chromatin was prepared from cultured <em>Drosophila</em> S2 cells (<a href="#pone-0001182-g008">Figure 8</a>). Also in this case, treatment of S2-chromatin with RNase A increases its accessibility to digestion with MNase.</p>
<div class="figure" id="pone-0001182-g007"><div class="img"><a name="pone-0001182-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Treatment with RNase A increases the sensitivity of chicken liver chromatin to cleavage by micrococcal nuclease.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A) Preparative 5%–30% linear sucrose gradient of chicken liver chromatin. Fraction numbers are indicated. Lanes M, correspond to molecular weight markers. B) Fraction 8 of the gradient shown in A) was treated with RNase A (lanes 7–12) or not (lanes 1–6) and, then, digested at 37°C with 0.4 units of micrococcal nuclease (Sigma) at increasing times as indicated. After micrococcal nuclease digestion, samples were deproteinized and analyzed by electrophoresis in 1% agarose-TBE gels. Lanes M correspond to molecular weight markers. C) Quantitation of the results shown in B). The ratio of mononucleosomal DNA versus total DNA is presented as a function of the digestion time for untreated chromatin (°) and chromatin treated with RNase A (•).</p>
<span>doi:10.1371/journal.pone.0001182.g007</span></div><div class="figure" id="pone-0001182-g008"><div class="img"><a name="pone-0001182-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001182" data-uri="info:doi/10.1371/journal.pone.0001182.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001182.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001182.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001182.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001182.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Treatment with RNase A increases the sensitivity of purified <em>Drosophila</em> S2 chromatin to cleavage by micrococcal nuclease.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>A) Preparative 5%–30% linear sucrose gradient of chromatin from <em>Drosophila</em> S2 cells. Fraction numbers are indicated. Lane M corresponds to molecular weight markers. B) Fraction 13 of the gradient shown in A) was treated RNase A (lanes 8 to 14) or not (lanes 1–7), and, then, digested at 37°C with 0,4 units of micrococcal nuclease (Sigma) at increasing times as indicated. After micrococcal nuclease digestion, samples were deproteinized and analyzed by electrophoresis in 1% agarose-TBE gels. Lane M corresponds to molecular weight markers. C) Quantitation of the results shown in B). The ratio of mononucleosomal DNA versus total DNA is presented as a function of the digestion time for untreated chromatin (°) and chromatin treated with RNase A (•).</p>
<span>doi:10.1371/journal.pone.0001182.g008</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Here, we report evidence supporting the physical association of RNA to chromatin purified by sucrose gradient centrifugation. Sucrose gradient centrifugation is used for the purification of discrete molecular entities from complex mixtures of macromolecules. Purification is achieved on the basis of the differential sedimentation rates of the various components, which depend on their mass, density and shape <a href="#pone.0001182-Noll1">[20]</a>. In particular, sucrose gradient centrifugation is extensively used for the purification of large multimeric complexes such as chromatin. On the appropriate conditions of pH and ionic strength, chromatin contained in a particular fraction is free of soluble components other than those directly associated to the native nucleoprotein complex. In our experiments, chromatin was obtained by mild MNase digestion of isolated nuclei that, to avoid as much as possible contamination with cytoplasmic RNA, were purified by centrifugation through a sucrose cushion. Our results show that chromatin purified by sucrose gradient centrifugation contains significant amounts of RNA. This conclusion arises from the fact that, when purified chromatin is subjected to selective RNA extraction, a nucleic acid residue is obtained, which hybridizes to radioactively labeled total high-weight genomic DNA (<a href="#pone-0001182-g001">Figure 1A</a>) and is sensitive to digestion with RNase A (DNase and protease-free) (<a href="#pone-0001182-g001">Figures 1B and 1C</a>). These chromatin-associated RNAs, which account for 2%–5% of the total nucleic acids content of chromatin, are likely to correspond to long single-stranded molecules as, on one hand, they are digested by RNase A and, second, their size is similar to that of the DNA isolated from the same chromatin fractions (<a href="#pone-0001182-g001">Figures 1A</a> and <a href="#pone-0001182-g002">2A</a>). At this respect, it must be noticed that MNase cleaves both DNA and single-stranded RNA <a href="#pone.0001182-Augenlicht1">[29]</a>. Therefore, the actual size the chromatin-associated RNAs is likely to be higher than that observed after MNase digestion. What is the nature of these chromatin-associated RNAs? Our results indicate that they are not likely to correspond to nascent transcripts as they are also observed when cells are treated with α-amanitin (<a href="#pone-0001182-g006">Figure 6B</a>). The fact that RNA remains associated to chromatin at high ionic strength (<a href="#pone-0001182-g002">Figure 2</a>), and that silent heterochromatic regions appear to contain RNA(s), also argue against this possibility. Our data also show that the vast majority of these chromatin-associated RNAs are polyA<sup>−</sup> (<a href="#pone-0001182-g001">Figure 1D</a>), indicating that they do not correspond either to mature RNA-polII transcripts. It is possible that these chromatin-associated RNAs would correspond to a class of non-coding RNAs. Increasing evidence indicates that a majority of the eukaryotic genome, including non-coding regions, is transcribed to RNA. It was recently shown that over 85% of the <em>Drosophila melanogaster</em> genome is transcribed <a href="#pone.0001182-Manak1">[30]</a>. In the recent years, non-coding RNAs have focused considerable interest due to their regulatory properties. Most of the regulatory non-coding RNAs identified to date are relatively small (micro-RNAs, siRNAs) <a href="#pone.0001182-Bartel1">[31]</a>, <a href="#pone.0001182-Mattick1">[32]</a> but a few long non-coding RNAs have also been identified. These include <em>Xist</em> (18 kb long), which is responsible for silencing of the X-chromosome <a href="#pone.0001182-Wutz1">[33]</a>, and <em>Air</em> (&gt;100 kb long), which silences the paternal <em>Igf2r/Slc22a2/Slc22a3</em> gene <a href="#pone.0001182-Sleutels1">[34]</a>. Most likely, long non-coding RNAs are more frequent than previously anticipated. Actually, a recent analysis in mouse revealed the existence of multiple nuclear long non-coding RNAs (&gt;10 kb) of unknown function <a href="#pone.0001182-Furuno1">[35]</a>. Whether these long non-coding RNAs associate to chromatin remains to be determined but it is possible that some of them would play a structural role in chromatin as such described here.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Treatment with RNase A alters the sedimentation behavior of chromatin through sucrose gradients (<a href="#pone-0001182-g003">Figure 3A</a>). In density gradients, particles sediment according to their sedimentation coefficient (<em>s</em>). As <em>s</em> depends on molecular weight, density and shape, the analysis of the sedimentation behavior could yield ambiguous information. But, when two of these parameters are constant, the change in the sedimentation rate actually reflects the variation in one particular property; for particles of similar mass and density, the value of <em>s</em> depends only on their shape <a href="#pone.0001182-Noll1">[20]</a>. Centrifugation through sucrose gradients has been extensively used to analyze structural changes in chromatin <a href="#pone.0001182-Gilbert1">[36]</a>, <a href="#pone.0001182-Ghirlando1">[37]</a>. Actually, identification of the nucleosome as the repeating unit of chromatin was made possible in part from sedimentation analysis of chromatin prepared by mild MNase digestion of purified nuclei <a href="#pone.0001182-Noll2">[23]</a>, <a href="#pone.0001182-Noll3">[24]</a>. Further analysis demonstrates that the selective removal of the linker histone H1 produces a significant decrease in sedimentation rate due to unfolding of the chromatin fiber <a href="#pone.0001182-Noll3">[24]</a>, <a href="#pone.0001182-Shimamura1">[25]</a>. Our results indicate that treatment of purified chromatin with RNase A decreases its sedimentation rate through sucrose gradients. The magnitude of the change in sedimentation observed is much higher than that expected from the reduction of mass associated to the degradation of the RNA component of chromatin. RNA accounts only for 2%–5% of the total nucleic acids content of purified chromatin. Therefore, full degradation of chromatin associated RNAs would result in 1%–2,5% loss of total chromatin mass, a reduction too small to be detected by sucrose gradient centrifugation. Therefore, as with H1-depleted chromatin, decreased sedimentation likely reflects that treatment with RNase A reduces the degree of compactness of chromatin. Actually, the change in sedimentation observed for RNA-depleted chromatin is similar to that of H1-depleted chromatin (compare <a href="#pone-0001182-g002">Figures 2A</a> and <a href="#pone-0001182-g003">3A</a>). Consistent with this hypothesis, accessibility to MNase cleavage increases after treatment with RNase A (<a href="#pone-0001182-g007">Figures 7</a> and <a href="#pone-0001182-g008">8</a>). This structural transition is not due to a change in histone composition or integrity (<a href="#pone-0001182-g003">Figure 3B</a>). In particular, after treatment with RNase A, histone H1 remains bound to the chromatin fiber. Moreover, decreased sedimentation requires actual degradation of an RNA component as it is abolished in the presence of anti-RNase, a specific RNase inhibitor. Altogether, these results indicate that, similar to depletion of histone H1, degradation of chromatin-associated RNA(s) changes the global compactness of the chromatin fiber. This structural transition is observed both for bulk chromatin as well as for chromatin at specific genomic locations, either euchromatic or heterochromatic, suggesting that the association of RNA is a general characteristic of chromatin. What is the molecular basis of the association of RNA to chromatin and its contribution to chromatin structure? Our results suggest that RNA is tightly associated to chromatin as it is resistant to treatment at high ionic strength that, on the other hand, releases linker histone H1. Moreover, preliminary results indicate that, upon total nucleic acids extraction, RNA is found bound to DNA (not shown). These results suggest that the association of RNA to chromatin occurs <em>via</em> DNA. On the other hand, several indications suggest that RNA might stabilize binding of structural non-histone proteins to chromatin. For instance, binding of HP1 to heterochromatin is sensitive to digestion with RNase A <a href="#pone.0001182-Maison1">[19]</a> and it requires the contribution of the hinge domain, which is known to bind RNA <em>in vitro</em> <a href="#pone.0001182-Muchardt1">[38]</a>. It is possible that RNA also influences binding of other structural non-histone proteins. Binding of RNA to chromatin might also stabilizes high-order chromatin structures by facilitating folding of the chromatin fiber, or long-distance chromatin-chromatin interactions, through its simultaneous binding to multiple sites. Further work is neccesary to understand the precise structural role that RNA plays in chromatin.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Isolation of nuclei</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All plasticware, glassware and solutions were sterilized by autoclaving. Fresh chicken livers were purchased on the public food market. <em>Drosophila</em> S2 cells were grew by conventional methods. Nuclei were isolated by Dounce homogenization of minced material (when liver) or spun cells (when S2) in 0,5 M sucrose, 0,5% Triton X-100, 100 µg/ml Phenyl Methane Sulfonyl Fluoride (PMSF), TES buffer (15 mM Tris-HCl pH 7,4, 60 mM KCl, 0,15 mM spermine, 0,5 mM spermidine, 2 mM EDTA, 1 mM DTT). The homogenate was diluted with three volumes of 2,3 M sucrose in TES and layered over a 5 ml cushion of 2,3 M sucrose in TES. After centrifugation in a Beckman SW28 rotor (27.000 rpm, 4°C, 3 h), pelleted nuclei were washed twice with cold TES, precipitated in a Beckman JA-25.50 rotor (5.000 rpm, 10 min, 4°C) and resuspended in the same buffer. The DNA concentration was measured at 260 nm and the nuclei, usually at 0,5–1 mg/ml, were frozen at −80°C after the addition of Dimethyl sulfoxide (DMSO) to a 10% final concentration.</p>


<h4>Preparation of native chromatin</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Native chromatin was obtained by mild digestion with micrococcal nuclease (MNase) of isolated nuclei in 3 mM CaCl<sub>2 </sub>during different times and the reaction stopped by addition of EDTA (50 mM final concentration). After pelleting in an Eppendorf tabletop centrifuge (8.000 rpm, 2 min, 4°C), the nuclei were disrupted by pipetting in 0,2 mM EDTA and spun again; the soluble chromatin was recovered from the supernatant and DNA concentration was measured at 260 nm. On average, up to 30–40% of the starting material was recovered as soluble chromatin.</p>


<h4>RNase A treatment</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Treatment with RNaseA (DNase and protease-free) (Roche) was performed using 0,5 to 1 µg of enzyme. When indicated, RNaseA activity was inhibited by the addition of 100 to 200 units of anti-RNase (Ambion) following manufacturer instructions.</p>


<h4>RNA isolation and quantitation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>RNA was extracted using Ultraspec™ RNA Isolation System (Biotecx) according to manufacturer instructions. Glyoxal-agarose gels were performed on a NaOH-treated electrophoresis cell. To minimize RNA degradation, agarose solution, electrophoresis and loading buffers were autoclaved. When the amount of RNA associated to purified chromatin was determined, chromatin purified by sucrose gradient sedimentation was deproteinized overnight at 37°C with 0,2% SDS, 0,5 mg/ml proteinase K, followed by phenol extraction and ethanol precipitation. The precipitated sample was resuspended in sterile water and total nucleic acid content determined at 260 nm using a quartz cuvette extensively treated with 2N NaOH and sterile water. The sample was, then, divided into three aliquots, containing 6 µg of total nucleic acid, which were treated at 37°C for 2 h with 20 µg of DNAse I (RNase-free), with 20 µg of RNase A (DNase and protease-free) or not treated. After these treatments, nucleic acid content of serial 10-fold dilutions of each sample was determined with RiboGreen® RNA Quantitation Reagent (Molecular Probes) in a fluorescence microplate reader (excitation at 480 nm, emission at 520 nm) in comparison to both DNA and RNA standards. In parallel, another aliquot of deproteinized chromatin was extracted with Ultraspec™ RNA Isolation System and the purified material was divided in two portions, incubated for 2 h at 37°C with 20 µg of RNase A (DNase and protease-free) or not, and processed as before.</p>


<h4>Sedimentation analysis through linear sucrose gradients</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Linear sucrose gradients were performed with a linear gradient maker. Prior to their use, the mixing chambers and polyvinyl tubing were treated during 1 h with a 2N NaOH solution and then washed with sterile water. To prevent proteases activities, PMSF was added to a 100 µg/ml final concentration to the sucrose solutions. 5%–30% sucrose (dissolved in 5 mM HEPES-NaOH pH 7,4, 0,2 mM EDTA) preparative gradients containing 1 to 5 mg of soluble native chromatin were centrifuged on Beckman SW28 rotor at 22.000 rpm during 14 h at 4°C. Analytical gradients containing 100–200 µg were run in a Beckman SW41 rotor (22.000 rpm, 14 h, 4°C).</p>


<h4>Isolation of RNA poly A-</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The polyadenylated status of purified chromatin-associated RNA(s) was analysed by affinity chromatography using an oligo-dT resin (Oligotex™ mRNA Purification System, QIAGEN). Bound (poly A+) and unbound (poly A-) samples were purified and electrophoresed on glyoxal-agarose denaturant gel.</p>


<h4>Treatment with α-amanitin</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p><em>Drosophila</em> S2 cells were grew until confluence and divided in two equal portions of 300 ml of suspended cells; one of them was incubated with α-amanitin (Sigma) (0,2 µg/ml of final concentration) during 36 h at 24°C. An aliquot (5 ml) of each cellular culture was processed for total RNA isolation and their nucleic acid content determined at 260 nm using a quartz cuvette extensively treated with 2N NaOH and sterile water. Native chromatin isolation and sedimentation analysis of samples incubated or not with α-amanitin was performed as described before. Efficiency of transcriptional inhibition by α-amanitin was measured by retrotranscription using Omniscript® RT Kit (QIAGEN). RT-reactions were performed with different amounts of RNA templates using a mixture of hexanucleotides as primers. After RT, samples were phenolized, precipitated with ethanol at room temperature and, then, subjected to PCR with <em>Taq</em> polymerase (Biotools) using specific primers derived from intron and adjacent exon sequences for actin 5C, <em>Trl</em> and RP-49 genes. Primers used were: for actin 5C, 5′-TATCACTACCGTTTGAGTTC-3′, corresponding to the first exon (positions +2 to +22) and 5′-CGTGACACGCCCACATCAGC-3′, corresponding to the first intron (positions +567 to +587); for <em>Trl,</em> 5′-TGGATCTAAGACTTCGGTCC-3′, corresponding to the second exon (positions +3208 to +3228) and 5′- AGCAACTCATTCCTTCCTTG-3′, corresponding to the second intron (positions +3850 to +3870) and, for RP-49, 5′- CATGTTATCAATGGTGCTGC-3′, corresponding to the first exon (positions +20 to +40) and 5′-GAATTATGCATTAGTGGGAC-3′, corresponding to the second intron (positions +702 to +722). The size of the expected fragments are 585 bp, 662 bp and 702 bp for actin 5C, <em>Trl</em> and RP-49, respectively.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are thankful to Drs M.A. Matzke, A.J.M. Matzke, E. Martí, J. Bernués and D. Huertas, for materials, and to J. Font and Dr. C. Bonet for technical help.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AR. Performed the experiments: AR. Analyzed the data: FA AR. Wrote the paper: FA AR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001182-Kornberg1" id="pone.0001182-Kornberg1"></a>Kornberg RD (1977) Structure of chromatin. Annu Rev Biochem  46: 931–954.  <ul class="find" data-citedArticleID="1018026" data-doi="10.1146/annurev.bi.46.070177.004435"><li><a href="http://dx.doi.org/10.1146/annurev.bi.46.070177.004435" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001182-McGhee1" id="pone.0001182-McGhee1"></a>McGhee JD, Felsenfeld G (1980) Nucleosome structure. Annu Rev Biochem  49: 1115–1156.  <ul class="find" data-citedArticleID="1018040" data-doi="10.1146/annurev.bi.49.070180.005343"><li><a href="http://dx.doi.org/10.1146/annurev.bi.49.070180.005343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleosome+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleosome+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001182-Finch1" id="pone.0001182-Finch1"></a>Finch T, Lutter LC, Rhodes D, Brown AS, Rushton B, et al.  (1977) Structure of nucleosome core particle. Nature (London)  269: 29–36.  <ul class="find" data-citedArticleID="1018008" data-doi="10.1038/269029a0"><li><a href="http://dx.doi.org/10.1038/269029a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+nucleosome+core+particle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+nucleosome+core+particle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001182-Richmond1" id="pone.0001182-Richmond1"></a>Richmond TJ, Finch JT, Rushton B, Rhodes D, Klug A (1984) Structure of nucleosome core particle at 7Å resolution. Nature (London)  311: 532–537.  <ul class="find" data-citedArticleID="1018054" data-doi="10.1038/311532a0"><li><a href="http://dx.doi.org/10.1038/311532a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+nucleosome+core+particle+at+7%C3%85+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+nucleosome+core+particle+at+7%C3%85+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001182-Luger1" id="pone.0001182-Luger1"></a>Luger K, Mäder AW, Richmond RK, Sargent DF, Richmond TJ (1997) Crystal structure of the nucleoosme core particle at 2,8Å resolution. Nature  389: 251–260.  <ul class="find" data-citedArticleID="1018030" data-doi="10.1038/38444"><li><a href="http://dx.doi.org/10.1038/38444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+nucleoosme+core+particle+at+2%2C8%C3%85+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+nucleoosme+core+particle+at+2%2C8%C3%85+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001182-Clark1" id="pone.0001182-Clark1"></a>Clark DJ, Kimura T (1990) Electrostatic mechanism of chromatin folding. J Mol Biol  211: 883–896.  <ul class="find" data-citedArticleID="1018006" data-doi="10.1016/0022-2836(90)90081-v"><li><a href="http://dx.doi.org/10.1016/0022-2836(90)90081-v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Electrostatic+mechanism+of+chromatin+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Electrostatic+mechanism+of+chromatin+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001182-GarciaRamrez1" id="pone.0001182-GarciaRamrez1"></a>Garcia-Ramírez M, Dong F, Ausió J (1992) Role of histone “tails” in the folding of oligonucleosomes depleted of histone H1. J Biol Chem  267: 19587–19595.  <ul class="find" data-citedArticleID="1018014"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+histone+%3Ftails%3F+in+the+folding+of+oligonucleosomes+depleted+of+histone+H1.&amp;auth=&amp;atitle=Role+of+histone+%3Ftails%3F+in+the+folding+of+oligonucleosomes+depleted+of+histone+H1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+histone+%E2%80%9Ctails%E2%80%9D+in+the+folding+of+oligonucleosomes+depleted+of+histone+H1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+histone+%E2%80%9Ctails%E2%80%9D+in+the+folding+of+oligonucleosomes+depleted+of+histone+H1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001182-Fletcher1" id="pone.0001182-Fletcher1"></a>Fletcher TM, Hansen JC (1995) Core histone tail domains mediate oligonucleome folding and nucleosomal DNA organization through distinct molecular mechanisms. J Biol Chem  270: 25359–25362.  <ul class="find" data-citedArticleID="1018010" data-doi="10.1074/jbc.270.43.25359"><li><a href="http://dx.doi.org/10.1074/jbc.270.43.25359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Core+histone+tail+domains+mediate+oligonucleome+folding+and+nucleosomal+DNA+organization+through+distinct+molecular+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Core+histone+tail+domains+mediate+oligonucleome+folding+and+nucleosomal+DNA+organization+through+distinct+molecular+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001182-Schwarz1" id="pone.0001182-Schwarz1"></a>Schwarz PM, Felthauser A, Fletcher TM, Hansen JC (1996) Reversible oligonuucleosome self-association: dependence of divalent cations and core histone tail domains. Biochemistry  35: 4009–4015.  <ul class="find" data-citedArticleID="1018056" data-doi="10.1021/bi9525684"><li><a href="http://dx.doi.org/10.1021/bi9525684" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reversible+oligonuucleosome+self-association%3A+dependence+of+divalent+cations+and+core+histone+tail+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reversible+oligonuucleosome+self-association%3A+dependence+of+divalent+cations+and+core+histone+tail+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001182-Widom1" id="pone.0001182-Widom1"></a>Widom J (1989) Toward a unified model of chromatin folding. Annu Rev Bioph. Chem  18: 365–395.  <ul class="find" data-citedArticleID="1018070" data-doi="10.1146/annurev.bb.18.060189.002053"><li><a href="http://dx.doi.org/10.1146/annurev.bb.18.060189.002053" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+unified+model+of+chromatin+folding.+Annu+Rev+Bioph." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+unified+model+of+chromatin+folding.+Annu+Rev+Bioph.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001182-VanHolde1" id="pone.0001182-VanHolde1"></a>Van Holde KH (1988) In Chromatin (Rich A ed.),. Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001182-Travers1" id="pone.0001182-Travers1"></a>Travers AA (1989) DNA conformation and protein binding. Annu Rev Biochem  58: 427–452.  <ul class="find" data-citedArticleID="1018062" data-doi="10.1146/annurev.bi.58.070189.002235"><li><a href="http://dx.doi.org/10.1146/annurev.bi.58.070189.002235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+conformation+and+protein+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+conformation+and+protein+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001182-Richards1" id="pone.0001182-Richards1"></a>Richards EJ, Elgin S (2002) Epigenetic codes for heterochromatin formation and silencing: rounding up the usual suspects. Cell  108: 489–500.  <ul class="find" data-citedArticleID="1018052" data-doi="10.1016/s0092-8674(02)00644-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00644-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+codes+for+heterochromatin+formation+and+silencing%3A+rounding+up+the+usual+suspects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+codes+for+heterochromatin+formation+and+silencing%3A+rounding+up+the+usual+suspects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001182-Bender1" id="pone.0001182-Bender1"></a>Bender J (2004) Chromatin-based silencing mechanisms. Curr Opin Plant Biol  7: 521–526.  <ul class="find" data-citedArticleID="1018004" data-doi="10.1016/j.pbi.2004.07.003"><li><a href="http://dx.doi.org/10.1016/j.pbi.2004.07.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin-based+silencing+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin-based+silencing+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001182-Lippman1" id="pone.0001182-Lippman1"></a>Lippman Z, Martienssen R (2004) The role of RNA interference in heterochromatic silencing. Nature  431: 364–370.  <ul class="find" data-citedArticleID="1018028" data-doi="10.1038/nature02875"><li><a href="http://dx.doi.org/10.1038/nature02875" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+RNA+interference+in+heterochromatic+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+RNA+interference+in+heterochromatic+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001182-Grimaud1" id="pone.0001182-Grimaud1"></a>Grimaud C, Bantignies F, Pal-Bhadra M, Ghana P, Bhadra U, et al.  (2006) RNAi components are required for nuclear clustering of Polycomb group response elements. Cell  124: 957–971.  <ul class="find" data-citedArticleID="1018020" data-doi="10.1016/j.cell.2006.01.036"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.036" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi+components+are+required+for+nuclear+clustering+of+Polycomb+group+response+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi+components+are+required+for+nuclear+clustering+of+Polycomb+group+response+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001182-Kelley1" id="pone.0001182-Kelley1"></a>Kelley RL, Kuroda MI (2000) Noncoding RNA genes in dosage compensation and imprinting. Cell  103: 9–12.  <ul class="find" data-citedArticleID="1018024" data-doi="10.1016/s0092-8674(00)00099-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)00099-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Noncoding+RNA+genes+in+dosage+compensation+and+imprinting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Noncoding+RNA+genes+in+dosage+compensation+and+imprinting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001182-Avner1" id="pone.0001182-Avner1"></a>Avner P, Heard E (2001) X-chromosome inactivation: counting, choice and initiation. Nature Rev Genet  2: 59–67.  <ul class="find" data-citedArticleID="1018000" data-doi="10.1038/35047580"><li><a href="http://dx.doi.org/10.1038/35047580" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X-chromosome+inactivation%3A+counting%2C+choice+and+initiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X-chromosome+inactivation%3A+counting%2C+choice+and+initiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001182-Maison1" id="pone.0001182-Maison1"></a>Maison C, Bailly D, Peters AHFM, Quivy JP, Roche D, et al.  (2002) Higher-order structure in pericentric heterochromatin involves a distinct pattern of histone modification and an RNA component. Nature Genet  30: 329–334.  <ul class="find" data-citedArticleID="1018032" data-doi="10.1038/ng843"><li><a href="http://dx.doi.org/10.1038/ng843" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-order+structure+in+pericentric+heterochromatin+involves+a+distinct+pattern+of+histone+modification+and+an+RNA+component." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-order+structure+in+pericentric+heterochromatin+involves+a+distinct+pattern+of+histone+modification+and+an+RNA+component.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001182-Noll1" id="pone.0001182-Noll1"></a>Noll H, Noll M (1989) Sucrose gradient techniques and applications to nucleosome structure. In Methods in Enzymology (Wassarman PM, Kornberg RD eds.)  170: 55–116.  <ul class="find" data-citedArticleID="1018046" data-doi="10.1016/0076-6879(89)70043-4"><li><a href="http://dx.doi.org/10.1016/0076-6879(89)70043-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sucrose+gradient+techniques+and+applications+to+nucleosome+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sucrose+gradient+techniques+and+applications+to+nucleosome+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001182-Warner1" id="pone.0001182-Warner1"></a>Warner JR, Knopf PM, Rich A (1963) A multiple ribosomal structure in protein synthesis. Proc Natl Acad Sci USA  49: 122–129.  <ul class="find" data-citedArticleID="1018066" data-doi="10.1073/pnas.49.1.122"><li><a href="http://dx.doi.org/10.1073/pnas.49.1.122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+multiple+ribosomal+structure+in+protein+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+multiple+ribosomal+structure+in+protein+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001182-Wettstein1" id="pone.0001182-Wettstein1"></a>Wettstein O, Staehelin T, Noll H (1963) Ribosomal aggregate engaged in protein synthesis: characterization of the ergosome. Nature  197: 430–435.  <ul class="find" data-citedArticleID="1018068" data-doi="10.1038/197430a0"><li><a href="http://dx.doi.org/10.1038/197430a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+aggregate+engaged+in+protein+synthesis%3A+characterization+of+the+ergosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+aggregate+engaged+in+protein+synthesis%3A+characterization+of+the+ergosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001182-Noll2" id="pone.0001182-Noll2"></a>Noll M (1974) Subunit structure of chromatin. Nature  251: 249–251.  <ul class="find" data-citedArticleID="1018048" data-doi="10.1038/251249a0"><li><a href="http://dx.doi.org/10.1038/251249a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Subunit+structure+of+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Subunit+structure+of+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001182-Noll3" id="pone.0001182-Noll3"></a>Noll M, Kornberg RD (1977) Action of micrococcal nuclease on chromatin and the location of histone H1. J Mol Biol  109: 393–404.  <ul class="find" data-citedArticleID="1018050" data-doi="10.1016/s0022-2836(77)80019-3"><li><a href="http://dx.doi.org/10.1016/s0022-2836(77)80019-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Action+of+micrococcal+nuclease+on+chromatin+and+the+location+of+histone+H1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Action+of+micrococcal+nuclease+on+chromatin+and+the+location+of+histone+H1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001182-Shimamura1" id="pone.0001182-Shimamura1"></a>Shimamura A, Sapp M, Rodríguez-Campos A, Worcel A (1989) Histone H1 represses transcription from minichromosomes assembled in vitro. Mol Cell Biol  9: 5573–5584.  <ul class="find" data-citedArticleID="1018058"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Histone+H1+represses+transcription+from+minichromosomes+assembled+in+vitro.&amp;auth=&amp;atitle=Histone+H1+represses+transcription+from+minichromosomes+assembled+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H1+represses+transcription+from+minichromosomes+assembled+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H1+represses+transcription+from+minichromosomes+assembled+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001182-Nohno1" id="pone.0001182-Nohno1"></a>Nohno T, Koyama E, Myokai F, Taniguchi S, Ohuchi H, et al. A chicken homeobox gene related to Drosophila paired is predominantly expressed in the developing limb. Dev Biol  158: 254–264.  <ul class="find" data-citedArticleID="1018044" data-doi="10.1006/dbio.1993.1184"><li><a href="http://dx.doi.org/10.1006/dbio.1993.1184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+chicken+homeobox+gene+related+to+Drosophila+paired+is+predominantly+expressed+in+the+developing+limb." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+chicken+homeobox+gene+related+to+Drosophila+paired+is+predominantly+expressed+in+the+developing+limb.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001182-Matzke1" id="pone.0001182-Matzke1"></a>Matzke MA, Varga F, Berger H, Schernthaner J, Scheizer D, et al.  (1990) A 41–42 bp tandemly repeated sequences isolated from nuclear envelopes of chicken erythrocytes is located predominantly on minichromosomes. Chromosoma (Berl)  99: 131–137.  <ul class="find" data-citedArticleID="1018038" data-doi="10.1007/bf01735329"><li><a href="http://dx.doi.org/10.1007/bf01735329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+41%E2%80%9342+bp+tandemly+repeated+sequences+isolated+from+nuclear+envelopes+of+chicken+erythrocytes+is+located+predominantly+on+minichromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+41%E2%80%9342+bp+tandemly+repeated+sequences+isolated+from+nuclear+envelopes+of+chicken+erythrocytes+is+located+predominantly+on+minichromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001182-Kedinger1" id="pone.0001182-Kedinger1"></a>Kedinger C, Gmazdowski M, Madel JL, Gissinger F, Chambon P (1970) α-amanitin: a specific inhibitor of one of two DNA-dependent RNA polymerase activities from calf thymus. Biochem. Biophys. Res. Comm.  38: 165–171.  <ul class="find" data-citedArticleID="1018022" data-doi="10.1016/0006-291x(70)91099-5"><li><a href="http://dx.doi.org/10.1016/0006-291x(70)91099-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%CE%B1-amanitin%3A+a+specific+inhibitor+of+one+of+two+DNA-dependent+RNA+polymerase+activities+from+calf+thymus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%CE%B1-amanitin%3A+a+specific+inhibitor+of+one+of+two+DNA-dependent+RNA+polymerase+activities+from+calf+thymus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001182-Augenlicht1" id="pone.0001182-Augenlicht1"></a>Augenlicht LH, McCormick M, Lipkin M (1976) Digestion of RNA of chromatin and ribonucleoprotein by staphylococcal nuclease. Biochemistry  15: 3818–3823.  <ul class="find" data-citedArticleID="1017998" data-doi="10.1021/bi00662a026"><li><a href="http://dx.doi.org/10.1021/bi00662a026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Digestion+of+RNA+of+chromatin+and+ribonucleoprotein+by+staphylococcal+nuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Digestion+of+RNA+of+chromatin+and+ribonucleoprotein+by+staphylococcal+nuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001182-Manak1" id="pone.0001182-Manak1"></a>Manak JR, Dike S, Sementchenko V, Kapranov P, Biemar F, et al.  (2006) Bilogical function of unnanotated transcription during the early development of Drosophila melanogaster. Nature Genet  38: 1151–1158.  <ul class="find" data-citedArticleID="1018034" data-doi="10.1038/ng1875"><li><a href="http://dx.doi.org/10.1038/ng1875" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bilogical+function+of+unnanotated+transcription+during+the+early+development+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bilogical+function+of+unnanotated+transcription+during+the+early+development+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001182-Bartel1" id="pone.0001182-Bartel1"></a>Bartel DP (2004) MicroRNAs: genomics, biogenesis, mechanism and function. Cell  116: 281–297.  <ul class="find" data-citedArticleID="1018002" data-doi="10.1016/s0092-8674(04)00045-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00045-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001182-Mattick1" id="pone.0001182-Mattick1"></a>Mattick JS, Makunin IV (2005) Small regulatory RNAs in mammals. Hum Mol Genet  14 (Suppl 1): R121–R132.  <ul class="find" data-citedArticleID="1018036" data-doi="10.1093/hmg/ddi101"><li><a href="http://dx.doi.org/10.1093/hmg/ddi101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+regulatory+RNAs+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+regulatory+RNAs+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001182-Wutz1" id="pone.0001182-Wutz1"></a>Wutz A (2003) In Noncoding RNAs: Molecular biology and molecular medicine, (Barciszewsky J, Erdmann VA eds.): 49 Georgetown Landes Bioscience. Texas.   <ul class="find" data-citedArticleID="1018072"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+Noncoding+RNAs%3A+Molecular+biology+and+molecular+medicine%2C+%28Barciszewsky+J%2C+Erdmann+VA+eds.%29%3A+49+Georgetown+Landes+Bioscience.&amp;auth=&amp;atitle=In+Noncoding+RNAs%3A+Molecular+biology+and+molecular+medicine%2C+%28Barciszewsky+J%2C+Erdmann+VA+eds.%29%3A+49+Georgetown+Landes+Bioscience." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+Noncoding+RNAs%3A+Molecular+biology+and+molecular+medicine%2C+%28Barciszewsky+J%2C+Erdmann+VA+eds.%29%3A+49+Georgetown+Landes+Bioscience." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+Noncoding+RNAs%3A+Molecular+biology+and+molecular+medicine%2C+%28Barciszewsky+J%2C+Erdmann+VA+eds.%29%3A+49+Georgetown+Landes+Bioscience.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001182-Sleutels1" id="pone.0001182-Sleutels1"></a>Sleutels F, Zwart R, Barlow DP (2002) The non coding Air RNA is required for silencing autosomal imprinted genes. Nature  415: 810–813.  <ul class="find" data-citedArticleID="1018060" data-doi="10.1038/415810a"><li><a href="http://dx.doi.org/10.1038/415810a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+non+coding+Air+RNA+is+required+for+silencing+autosomal+imprinted+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+non+coding+Air+RNA+is+required+for+silencing+autosomal+imprinted+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001182-Furuno1" id="pone.0001182-Furuno1"></a>Furuno M, Pang KC, Ninomiya N, Fukuda S, Frith M, et al.  (2006) Clusters of internally primed transcripts reveal a novel long noncoding RNAs. PLoS Genetics  2: e37. 0537–0553.  <ul class="find" data-citedArticleID="1018012" data-doi="10.1371/journal.pgen.0020037"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clusters+of+internally+primed+transcripts+reveal+a+novel+long+noncoding+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clusters+of+internally+primed+transcripts+reveal+a+novel+long+noncoding+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001182-Gilbert1" id="pone.0001182-Gilbert1"></a>Gilbert N, Allan J (2001) Distinctive higher-order chromatin structure at mammalian centromeres. Proc Natl Acad Sci USA  98: 11949–11954.  <ul class="find" data-citedArticleID="1018018" data-doi="10.1073/pnas.211322798"><li><a href="http://dx.doi.org/10.1073/pnas.211322798" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinctive+higher-order+chromatin+structure+at+mammalian+centromeres." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinctive+higher-order+chromatin+structure+at+mammalian+centromeres.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001182-Ghirlando1" id="pone.0001182-Ghirlando1"></a>Ghirlando R, Litt MD, Prioleau MN, Recillas-Targa F, Felsenfeld G (2004) Physical properties of a genomic condensed chromatin fragments. J Mol Biol  336: 597–605.  <ul class="find" data-citedArticleID="1018016" data-doi="10.1016/j.jmb.2003.12.049"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.12.049" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical+properties+of+a+genomic+condensed+chromatin+fragments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical+properties+of+a+genomic+condensed+chromatin+fragments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001182-Muchardt1" id="pone.0001182-Muchardt1"></a>Muchardt C, Guillemé M, Seeler JS, Trouche D, Dejean A, et al.  (2002) Coordinated methyl and RNA binding is required for heterochromatin localization of mammalian HP1α EMBO Rep  3: 975–81.  <ul class="find" data-citedArticleID="1018042" data-doi="10.1093/embo-reports/kvf194"><li><a href="http://dx.doi.org/10.1093/embo-reports/kvf194" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coordinated+methyl+and+RNA+binding+is+required+for+heterochromatin+localization+of+mammalian+HP1%CE%B1" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coordinated+methyl+and+RNA+binding+is+required+for+heterochromatin+localization+of+mammalian+HP1%CE%B1%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.XML" value="80980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.PDF" value="3090468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g001.PNG_L" value="1466877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g001.PNG_M" value="172624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g001.PNG_S" value="11432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g001.TIF" value="1405370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g001.PNG_I" value="63311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g002.PNG_L" value="1226522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g002.PNG_M" value="135268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g002.PNG_S" value="7831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g002.TIF" value="1181404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g002.PNG_I" value="42742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g003.PNG_L" value="1599296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g003.PNG_M" value="113847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g003.PNG_S" value="9370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g003.TIF" value="1792928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g003.PNG_I" value="35769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g004.PNG_L" value="2086069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g004.PNG_M" value="150613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g004.PNG_S" value="9871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g004.TIF" value="2091862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g004.PNG_I" value="47411"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g005.PNG_L" value="2117989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g005.PNG_M" value="146692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g005.PNG_S" value="10407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g005.TIF" value="2251898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g005.PNG_I" value="46173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g006.PNG_L" value="1278260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g006.PNG_M" value="140418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g006.PNG_S" value="10650"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g006.TIF" value="1306846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g006.PNG_I" value="45660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g007.PNG_L" value="932053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g007.PNG_M" value="113005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g007.PNG_S" value="8974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g007.TIF" value="992400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g007.PNG_I" value="36582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g008.PNG_L" value="1353258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g008.PNG_M" value="132279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g008.PNG_S" value="8932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g008.TIF" value="1403658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001182.g008.PNG_I" value="42193"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001182&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001182" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001182&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001182&volume=&issue=&title=RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization&author_name=Antonio%20Rodr%C3%ADguez-Campos%2C%20Fernando%20Azor%C3%ADn&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001182" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182&amp;t=RNA%20Is%20an%20Integral%20Component%20of%20Chromatin%20that%20Contributes%20to%20Its%20Structural%20Organization" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182&title=RNA%20Is%20an%20Integral%20Component%20of%20Chromatin%20that%20Contributes%20to%20Its%20Structural%20Organization&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182&amp;title=RNA%20Is%20an%20Integral%20Component%20of%20Chromatin%20that%20Contributes%20to%20Its%20Structural%20Organization" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001182&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'RNA Is an Integral Component of Chromatin that Contributes to Its Structural Organization';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001182';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20RNA%20Is%20an%20Integral%20Component%20of%20Chromatin%20that%20Contributes%20to%20Its%20Structural%20Organization http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001182" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001182" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chickens%22" title="Search for articles in the subject area:'Chickens'"><div class="flagText">Chickens</div></a>
              <div data-categoryid="34017" data-articleid="25196"
                   data-categoryname="Chickens"
                   class="flagImage" title="Flag 'Chickens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="25196"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="25196"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="25196"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+structure%22" title="Search for articles in the subject area:'RNA structure'"><div class="flagText">RNA structure</div></a>
              <div data-categoryid="35143" data-articleid="25196"
                   data-categoryname="RNA structure"
                   class="flagImage" title="Flag 'RNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribonucleases%22" title="Search for articles in the subject area:'Ribonucleases'"><div class="flagText">Ribonucleases</div></a>
              <div data-categoryid="46199" data-articleid="25196"
                   data-categoryname="Ribonucleases"
                   class="flagImage" title="Flag 'Ribonucleases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sedimentation%22" title="Search for articles in the subject area:'Sedimentation'"><div class="flagText">Sedimentation</div></a>
              <div data-categoryid="35255" data-articleid="25196"
                   data-categoryname="Sedimentation"
                   class="flagImage" title="Flag 'Sedimentation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sucrose%22" title="Search for articles in the subject area:'Sucrose'"><div class="flagText">Sucrose</div></a>
              <div data-categoryid="32241" data-articleid="25196"
                   data-categoryname="Sucrose"
                   class="flagImage" title="Flag 'Sucrose' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5377'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6976'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2856&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=45477">Interesting treatment.</a><br>Posted by Versteeg</p>
      <p><a href="/annotation/listThread.action?root=44043">More studies should be looking at this</a><br>Posted by Murrell</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
