

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001682"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001682" />

    <meta name="citation_title" content="Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms"/>
    <meta itemprop="name" content="Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms"/>

      <meta name="citation_author" content="Karen E. James"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Harald Schneider"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Stephen W. Ansell"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Margaret Evers"/>
            <meta name="citation_author_institution" content="Diversity Arrays Technology P/L, Canberra, Australia"/>
      <meta name="citation_author" content="Lavinia Robba"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Grzegorz Uszynski"/>
            <meta name="citation_author_institution" content="Diversity Arrays Technology P/L, Canberra, Australia"/>
      <meta name="citation_author" content="Niklas Pedersen"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Angela E. Newton"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Stephen J. Russell"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Johannes C. Vogel"/>
            <meta name="citation_author_institution" content="Department of Botany, The Natural History Museum, London, United Kingdom"/>
      <meta name="citation_author" content="Andrzej Kilian"/>
            <meta name="citation_author_institution" content="Diversity Arrays Technology P/L, Canberra, Australia"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001682.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1682"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular markers, natural history, and evolution.; citation_author=JC Avise; citation_number=1; citation_date=2004; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Uncovering evolutionary patterns of gene expression using microarrays.; citation_author=JM Ranz; citation_author=CA Machado; citation_journal_title=Trends in Ecology & Evolution; citation_volume=21; citation_number=2; citation_pages=29-37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Using DNA microarrays to study natural variation.; citation_author=Y Gilad; citation_author=J Borevitz; citation_journal_title=Current Opinion in Genetics & Development; citation_volume=16; citation_number=3; citation_pages=553-558; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Arbitrarily amplified DNA markers as characters for phylogenetic inference.; citation_author=JD Bussell; citation_author=M Waycott; citation_author=JA Chappill; citation_journal_title=Perspectives in Plant Ecology Evolution and Systematics; citation_volume=7; citation_number=4; citation_pages=3-26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The fast and the cheap: SNP and DArT-based whole genome profiling for crop improvement.; citation_author=A Kilian; citation_author=E Huttner; citation_author=P Wenzl; citation_author=D Jaccoud; citation_author=J Carling; citation_number=5; citation_pages=443-461; citation_date=2003; citation_publisher=Avenue Media; " />
      <meta name="citation_reference" content="citation_title=Validation of the high-throughput marker technology DArT using the model plant Arabidopsis thaliana.; citation_author=AHJ Wittenberg; citation_author=T van der Lee; citation_author=C Cayla; citation_author=A Kilian; citation_author=RGF Visser; citation_journal_title=Molecular Genetics and Genomics; citation_volume=274; citation_number=6; citation_pages=30-39; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Diversity Arrays Technology (DArT) for whole-genome profiling of barley.; citation_author=P Wenzl; citation_author=J Carling; citation_author=D Kudrna; citation_author=D Jaccoud; citation_author=E Huttner; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=101; citation_number=7; citation_pages=9915-9920; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Diversity Arrays: a solid state technology for sequence information independent genotypinG.; citation_author=D Jaccoud; citation_author=K Peng; citation_author=D Feinstein; citation_author=A Kilian; citation_journal_title=Nucleic Acids Research; citation_volume=29; citation_number=8; citation_pages=1-7; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Development and assessment of microarray-based DNA fingerprinting in Eucalyptus grandis.; citation_author=S Lezar; citation_author=AA Myburg; citation_author=DK Berger; citation_author=MJ Wingfield; citation_author=BD Wingfield; citation_journal_title=Theoretical and Applied Genetics; citation_volume=109; citation_number=9; citation_pages=1329-1336; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DArT for high-throughput genotyping of Cassava (Manihot esculenta) and its wild relatives.; citation_author=L Xia; citation_author=KM Peng; citation_author=SY Yang; citation_author=P Wenzl; citation_author=MC de Vicente; citation_journal_title=Theoretical and Applied Genetics; citation_volume=110; citation_number=10; citation_pages=1092-1098; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome.; citation_author=M Akbari; citation_author=P Wenzl; citation_author=V Caig; citation_author=J Carling; citation_author=L Xia; citation_journal_title=Theoretical and Applied Genetics; citation_volume=113; citation_number=11; citation_pages=1409-1420; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A high-density consensus map of barley linking DArT markers to SSR, RFLP and STS loci and agricultural traits.; citation_author=P Wenzl; citation_author=HB Li; citation_author=J Carling; citation_author=MX Zhou; citation_author=H Raman; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=12; citation_pages=206; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Low level of genetic diversity in cultivated pigeonpea compared to its wild relatives is revealed by diversity arrays technology.; citation_author=SY Yang; citation_author=W Pang; citation_author=G Ash; citation_author=J Harper; citation_author=J Carling; citation_journal_title=Theoretical and Applied Genetics; citation_volume=113; citation_number=13; citation_pages=585-595; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Über das spontane Entstehen von Asplenium adulterinum aus einem natürlichen Bastard.; citation_author=JD Lovis; citation_author=T Reichstein; citation_journal_title=Die Natuerwissenschaften; citation_volume=55; citation_number=14; citation_pages=117-120; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue.; citation_author=J Doyle; citation_author=J Doyle; citation_journal_title=Phytochem. Bull.; citation_volume=19; citation_number=15; citation_pages=11-15; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Polyploidy, phylogeography and Pleistocene refugia of the rockfern Asplenium ceterach: evidence from chloroplast DNA.; citation_author=SA Trewick; citation_author=M Morgan-Richards; citation_author=SJ Russell; citation_author=S Henderson; citation_author=FJ Rumsey; citation_journal_title=Molecular Ecology; citation_volume=11; citation_number=16; citation_pages=2003-2012; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Suppression subtractive hybridization: A method for generating differentially regulated or tissue-specific cDNA probes and libraries.; citation_author=L Diatchenko; citation_author=YFC Lau; citation_author=AP Campbell; citation_author=A Chenchik; citation_author=F Moqadam; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=93; citation_number=17; citation_pages=6025-6030; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Optimizing Parental Selection for Genetic-Linkage Maps.; citation_author=JA Anderson; citation_author=GA Churchill; citation_author=JE Autrique; citation_author=SD Tanksley; citation_author=ME Sorrells; citation_journal_title=Genome; citation_volume=36; citation_number=18; citation_pages=181-186; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Universal Primers for Amplification of 3 Noncoding Regions of Chloroplast DNA.; citation_author=P Taberlet; citation_author=L Gielly; citation_author=G Pautou; citation_author=J Bouvet; citation_journal_title=Plant Molecular Biology; citation_volume=17; citation_number=19; citation_pages=1105-1109; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Origin of the endemic fern genus Diellia coincides with the renewal of Hawaiian terrestrial life in the Miocene.; citation_author=H Schneider; citation_author=T Ranker; citation_author=SJ Russell; citation_author=R Cranfill; citation_author=JMO Geiger; citation_journal_title=Proceedings of the Royal Society B-Biological Sciences; citation_volume=272; citation_number=20; citation_pages=455-460; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Development of primer sets for PCR amplification of the PgiC gene in ferns.; citation_author=H Ishikawa; citation_author=Y Watano; citation_author=K Kano; citation_author=M Ito; citation_author=S Kurita; citation_journal_title=Journal of Plant Research; citation_volume=115; citation_number=21; citation_pages=65-70; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic and morphological studies within the Ptychomniales, with emphasis on the evolution of dwarf males.; citation_author=N Pedersen; citation_author=AE Newton; citation_number=22; citation_pages=367-392; citation_date=2007; citation_publisher=CRC Press (Taylor & Francis Group); " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=JH Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Research; citation_volume=25; citation_number=23; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=PAST: Palaeontological Statistics Software package for education and data analysis.; citation_author=O Hammer; citation_author=D Harper; citation_author=P Ryan; citation_journal_title=Palaeontologia Electronica; citation_volume=4; citation_number=24; citation_pages=9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Interactive Analysis of Phylogeny and Character Evolution Using the Computer-Program Macclade.; citation_author=WP Maddison; citation_author=DR Maddison; citation_journal_title=Folia Primatologica; citation_volume=53; citation_number=25; citation_pages=190-202; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=MRBAYES: Bayesian inference of phylogenetic trees.; citation_author=JP Huelsenbeck; citation_author=F Ronquist; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=26; citation_pages=754-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Paup-a Computer-Program for Phylogenetic Inference Using Maximum Parsimony.; citation_author=DL Swofford; citation_journal_title=Journal of General Physiology; citation_volume=102; citation_number=27; citation_pages=A9-A9; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=NeighborNet: An agglomerative method for the construction of planar phylogenetic networks. In: Algorithms in Bioniformatics: Second International Workshop, WABI 2002, Rome, Italy, September 17–21, 2002. Proceedings.; citation_author=D Bryant; citation_author=V Moulton; citation_number=28; citation_pages=375-391; citation_date=2002; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=TRACER, version 1.2.; citation_author=A Rambaut; citation_author=A Drummond; citation_number=29; citation_date=2004; citation_publisher=University of Oxford; " />
      <meta name="citation_reference" content="citation_title=Treecon-a Software Package for the Construction and Drawing of Evolutionary Trees.; citation_author=Y Vandepeer; citation_author=R Dewachter; citation_journal_title=Computer Applications in the Biosciences; citation_volume=9; citation_number=30; citation_pages=177-182; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Multiple origins of polyploids in European Asplenium (Pteridophyta).; citation_author=JC Vogel; citation_number=31; citation_date=1995; citation_publisher=University of Cambridge; " />
      <meta name="citation_reference" content="citation_title=On hybrid formation in the rock fern Asplenium x alternifolium (Aspleniaceae, Pteridophyta).; citation_author=JC Vogel; citation_author=SJ Russell; citation_author=FJ Rumsey; citation_author=JA Barrett; citation_author=M Gibby; citation_journal_title=Botanica Acta; citation_volume=111; citation_number=32; citation_pages=241-246; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Phylogeny reconstruction in the gap between too little and too much divergence: the closest relatives of Senecio jacobaea (Asteraceae) according to DNA sequences and AFLPs.; citation_author=PB Pelser; citation_author=B Gravendeel; citation_author=R van der Meijden; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=29; citation_number=33; citation_pages=613-628; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic signal in AFLP data sets.; citation_author=WJM Koopman; citation_journal_title=Systematic Biology; citation_volume=54; citation_number=34; citation_pages=197-217; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A penalty of using anonymous dominant markers (AFLPs, ISSRs, and RAMS) for phylogenetic inference.; citation_author=MP Simmons; citation_author=LB Zhang; citation_author=CT Webb; citation_author=K Muller; citation_journal_title=Molecular Phylogenetics and Evolution; citation_volume=42; citation_number=35; citation_pages=528-542; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mathematical Model for Studying Genetic Variation in Terms of Restriction Endonucleases.; citation_author=M Nei; citation_author=WH Li; citation_journal_title=PNAS; citation_volume=76; citation_number=36; citation_pages=5269-5273; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Evolutionary relationships within recently radiated taxa: comments on methodology and analysis of inter-simple sequence repeat data and other hypervariable, dominant markers.; citation_author=JK Archibald; citation_author=ME Mort; citation_author=DJ Crawford; citation_author=A Santos-Guerra; citation_journal_title=Taxon; citation_volume=55; citation_number=37; citation_pages=747-756; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolution of the mitochondrial rps3 intron in perennial and annual angiosperms and homology to nad5 intron 1.; citation_author=J Laroche; citation_author=J Bousquet; citation_journal_title=Molecular Biology and Evolution; citation_volume=16; citation_number=38; citation_pages=441-452; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=SNPs, SSRs and inferences on cassava's origin.; citation_author=KM Olsen; citation_journal_title=Plant Molecular Biology; citation_volume=56; citation_number=39; citation_pages=517-526; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Testing the phylogenetic position of a parasitic plant (Cuscuta, Convolvulaceae, Asteridae): Bayesian inference and the parametric bootstrap on data drawn from three genomes.; citation_author=S Stefanovic; citation_author=RG Olmstead; citation_journal_title=Systematic Biology; citation_volume=53; citation_number=40; citation_pages=384-399; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genealogical evidence of homoploid hybrid speciation in an adaptive radiation of Scaevola (Goodeniaceae) in the Hawaiian Islands.; citation_author=DG Howarth; citation_author=DA Baum; citation_journal_title=Evolution; citation_volume=59; citation_number=41; citation_pages=948-961; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Use of DNA barcodes to identify flowering plants.; citation_author=WJ Kress; citation_author=KJ Wurdack; citation_author=EA Zimmer; citation_author=LA Weigt; citation_author=DH Janzen; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=42; citation_pages=8369-8374; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular genetic variation in Passiflora alata (Passifloraceae), an invasive species in southern Brazil.; citation_author=P Koehler-Santos; citation_author=AP Lorenz-Lemke; citation_author=VC Muschner; citation_author=SL Bonatto; citation_author=FM Salzano; citation_journal_title=Biological Journal of the Linnean Society; citation_volume=88; citation_number=43; citation_pages=611-630; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogeography of the bitter apple, Citrullus colocynthis.; citation_author=F Dane; citation_author=JR Liu; citation_author=CK Zhang; citation_journal_title=Genetic Resources and Crop Evolution; citation_volume=54; citation_number=44; citation_pages=327-336; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms"/>
    <meta name="twitter:description" content="BackgroundHigh-throughput tools for pan-genomic study, especially the DNA microarray platform, have sparked a remarkable increase in data production and enabled a shift in the scale at which biological investigation is possible. The use of microarrays to examine evolutionary relationships and processes, however, is predominantly restricted to model or near-model organisms.Methodology/Principal FindingsThis study explores the utility of Diversity Arrays Technology (DArT) in evolutionary studies of non-model organisms. DArT is a hybridization-based genotyping method that uses microarray technology to identify and type DNA polymorphism. Theoretically applicable to any organism (even one for which no prior genetic data are available), DArT has not yet been explored in exclusively wild sample sets, nor extensively examined in a phylogenetic framework. DArT recovered 1349 markers of largely low copy-number loci in two lineages of seed-free land plants: the diploid fern Asplenium viride and the haploid moss Garovaglia elegans. Direct sequencing of 148 of these DArT markers identified 30 putative loci including four routinely sequenced for evolutionary studies in plants. Phylogenetic analyses of DArT genotypes reveal phylogeographic and substrate specificity patterns in A. viride, a lack of phylogeographic pattern in Australian G. elegans, and additive variation in hybrid or mixed samples.Conclusions/SignificanceThese results enable methodological recommendations including procedures for detecting and analysing DArT markers tailored specifically to evolutionary investigations and practical factors informing the decision to use DArT, and raise evolutionary hypotheses concerning substrate specificity and biogeographic patterns. Thus DArT is a demonstrably valuable addition to the set of existing molecular approaches used to infer biological phenomena such as adaptive radiations, population dynamics, hybridization, introgression, ecological differentiation and phylogeography."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001682.g004"/>

  <meta property="og:title" content="Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6831'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4460'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=893&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001682">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001682" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001682&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karen E. James
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:k.james@nhm.ac.uk">k.james@nhm.ac.uk</a></p>

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harald Schneider, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen W. Ansell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Margaret Evers, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Diversity Arrays Technology P/L, Canberra, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lavinia Robba, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Grzegorz Uszynski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Diversity Arrays Technology P/L, Canberra, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Niklas Pedersen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Angela E. Newton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephen J. Russell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Johannes C. Vogel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Botany, The Natural History Museum, London, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrzej Kilian
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Diversity Arrays Technology P/L, Canberra, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001682</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001682-t001" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-t002" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-t003" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-t004" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-t005" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-g001" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-g002" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-g003" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001682-g004" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001682">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001682" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>High-throughput tools for pan-genomic study, especially the DNA microarray platform, have sparked a remarkable increase in data production and enabled a shift in the scale at which biological investigation is possible. The use of microarrays to examine evolutionary relationships and processes, however, is predominantly restricted to model or near-model organisms.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>This study explores the utility of Diversity Arrays Technology (DArT) in evolutionary studies of non-model organisms. DArT is a hybridization-based genotyping method that uses microarray technology to identify and type DNA polymorphism. Theoretically applicable to any organism (even one for which no prior genetic data are available), DArT has not yet been explored in exclusively wild sample sets, nor extensively examined in a phylogenetic framework. DArT recovered 1349 markers of largely low copy-number loci in two lineages of seed-free land plants: the diploid fern <em>Asplenium viride</em> and the haploid moss <em>Garovaglia elegans</em>. Direct sequencing of 148 of these DArT markers identified 30 putative loci including four routinely sequenced for evolutionary studies in plants. Phylogenetic analyses of DArT genotypes reveal phylogeographic and substrate specificity patterns in <em>A. viride</em>, a lack of phylogeographic pattern in Australian <em>G. elegans</em>, and additive variation in hybrid or mixed samples.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These results enable methodological recommendations including procedures for detecting and analysing DArT markers tailored specifically to evolutionary investigations and practical factors informing the decision to use DArT, and raise evolutionary hypotheses concerning substrate specificity and biogeographic patterns. Thus DArT is a demonstrably valuable addition to the set of existing molecular approaches used to infer biological phenomena such as adaptive radiations, population dynamics, hybridization, introgression, ecological differentiation and phylogeography.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>James KE, Schneider H, Ansell SW, Evers M, Robba L, et al.  (2008) Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms. PLoS ONE 3(2):
          e1682.
            doi:10.1371/journal.pone.0001682</p><p><strong>Academic Editor: </strong>Pawel Michalak, University of Texas Arlington, United States of America</p><p><strong>Received:</strong> December 18, 2007; <strong>Accepted:</strong> February 1, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 James et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work has been financially supported by The Natural History Museum Research Fund, NERC grant GR3/12073 in support of specimen collection and Marie Curie Postdoctoral Fellowship MEIF-CT-2003-501682 to Niklas Pedersen.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The development of innovative methods for detecting genetic variation has progressively enhanced the study of evolutionary relationships and processes <a href="#pone.0001682-Avise1">[1]</a>. High-throughput genomic tools such as the DNA microarray platform have sparked a remarkable increase in data production, leading to new evolutionary insights <a href="#pone.0001682-Ranz1">[2]</a>. Their application is however nearly exclusively restricted to model and near-model organisms <a href="#pone.0001682-Ranz1">[2]</a>, <a href="#pone.0001682-Gilad1">[3]</a>. In contrast, the detection of genetic variation in non-model plants and animal—the majority of life on earth—is largely restricted to direct sequencing of previously-identified variable loci or arbitrarily amplified dominant (AAD) markers, especially RAPDs, ISSRs and AFLPs <a href="#pone.0001682-Bussell1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Diversity Arrays Technology (DArT) was developed to overcome these and other limitations that prevent the application of microarray technology to non-model organisms. DArT is a hybridization-based genotyping technology which is currently implemented on the microarray platform to rapidly and simultaneously identify and type DNA polymorphism <a href="#pone.0001682-Kilian1">[5]</a>. DArT detects primarily dominant markers, mostly resulting from single nucleotide polymorphisms at restriction sites <a href="#pone.0001682-Wittenberg1">[6]</a> at hundreds to thousands of arbitrary genomic loci <a href="#pone.0001682-Wittenberg1">[6]</a>, <a href="#pone.0001682-Wenzl1">[7]</a>. First used to infer the genetic diversity in cultivated varieties of rice (<em>Oryza sativa</em> L.) <a href="#pone.0001682-Jaccoud1">[8]</a>, DArT was subsequently applied to barley (<em>Hordeum vulgare</em> L.) <a href="#pone.0001682-Wenzl1">[7]</a>, grand eucalyptus (<em>Eucalyptus grandis</em> Hill ex Maiden) <a href="#pone.0001682-Lezar1">[9]</a>, cassava (<em>Manihot esculenta</em> Crantz) <a href="#pone.0001682-Xia1">[10]</a>, wheat (<em>Triticum aestivum</em> L.) <a href="#pone.0001682-Akbari1">[11]</a> and validated in the model organism thale cress (<em>Arabidopsis thaliana</em> L. Heynh) <a href="#pone.0001682-Wittenberg1">[6]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The application of DArT has so far been restricted to detection and mapping of DNA polymorphism in cultivated varieties of agriculturally important angiosperms. While some wild crop relatives have been included in previous DArT studies <a href="#pone.0001682-Xia1">[10]</a>, <a href="#pone.0001682-Wenzl2">[12]</a>, <a href="#pone.0001682-Yang1">[13]</a>, the potential of DArT has not yet been explored in exclusively wild sample sets, nor extensively examined in a phylogenetic framework. Furthermore, because all existing DArT studies have been applied to flowering plants, its applicability to the rest of the plant kingdom and other organisms requires validation.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>To explore whether DArT data can contribute to characterising evolutionary patterns and processes, we selected two lineages of seed-free land plants that provide two distinct opportunities to infer evolutionary hypotheses and detect hybridization by employing DArT: 1) the evolution of substrate specificity in the diploid fern <em>Asplenium viride</em> L. and 2) the phylogeography of the haploid moss <em>Garovaglia elegans</em> Dozy &amp; Molk. Hampe ex Bosch &amp; Sande Lac. These organisms were selected based on accessibility to well-preserved plant material and prior knowledge obtained by studying these organisms using conventional molecular evidence (allozymes and/or DNA sequencing). Furthermore, evolutionary interpretations have been hampered in both organisms by low DNA sequence diversity of a small number of selected loci.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Preparation and evaluation of DArT Discovery Libraries</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Independent DArT discovery arrays were constructed for two 16-specimen study groups: (1) 14 specimens of the diploid fern <em>Asplenium viride</em>, one specimen of diploid <em>A. trichomanes</em> L. to polarize genetic structuring within <em>A. viride</em> and one specimen of their naturally derived allotetraploid hybrid, <em>A. adulterinum</em> J. Milde <a href="#pone.0001682-Lovis1">[14]</a>, and (2) 15 specimens of the haploid moss <em>Garovaglia elegans,</em> collected from various locations in Australia and New Guinea, and one specimen of <em>G. powellii</em> Mitt. for polarization (<a href="#pone-0001682-t001">Tables 1</a> and <a href="#pone-0001682-t002">2</a>). At least 1 µg of total genomic DNA was extracted from silica gel desiccated (<em>Asplenium</em>) or air dried (<em>Garovaglia</em>) leaf material using a modification of the standard CTAB procedure <a href="#pone.0001682-Doyle1">[15]</a> as specified in Trewick <em>et al.</em> <a href="#pone.0001682-Trewick1">[16]</a> except that extractions were incubated in 500 µL CTAB buffer, 50 µl sarkosyl and 10 µl proteinase-K and purified by phenol-chloroform extraction. DNA quantity, quality and competence for restriction endonuclease digestion were confirmed by visualizing 2 µL of total genomic DNA alongside 2 µL of <em>Mse</em>I restriction digested total genomic DNA by agarose gel electrophoresis and ethidium bromide staining. Genomic representations were produced from a mixture of either <em>Asplenium</em> or <em>Garovaglia</em> DNA samples according to Wenzl <em>et al.</em> <a href="#pone.0001682-Wenzl1">[7]</a>. Libraries were prepared as described <a href="#pone.0001682-Wenzl1">[7]</a>, <a href="#pone.0001682-Jaccoud1">[8]</a> either directly from PCR amplification products or from normalized amplification products subtracted using a modified version of suppression subtractive hybridization (SSH) <a href="#pone.0001682-Diatchenko1">[17]</a> as described below. Polymorphic markers detected on the <em>Asplenium</em> and <em>Garovaglia</em> discovery arrays were re-arrayed onto “genotyping arrays” to enable genotyping of the individual samples. The <em>in vivo</em> genomic copy number of each DArT marker was approximated by hybridizing un-amplified metagenomic DNA to each genotyping array.</p>
<div class="figure" id="pone-0001682-t001"><div class="img"><a name="pone-0001682-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span><em>Asplenium</em> samples used in this study with voucher information, collecting locality, substrate specificity, ploidy level and GenBank accession numbers.</span></strong></p><span>doi:10.1371/journal.pone.0001682.t001</span></div><div class="figure" id="pone-0001682-t002"><div class="img"><a name="pone-0001682-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span><em>Garovaglia</em> samples used in this study with voucher information, collecting locality, substrate specificity, ploidy level and GenBank accession numbers.</span></strong></p><span>doi:10.1371/journal.pone.0001682.t002</span></div>

<h4>Genotyping and analysis of DArT images</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The genotypes of all individual <em>Asplenium</em> and <em>Garovaglia</em> samples were scored as absence/presence for 444 and 905 markers, respectively, and formatted as a binary data matrix (<a href="#pone.0001682.s001">Table S1</a>). The reproducibility of the two DArT genotyping arrays was examined by independent assay using the same DNA. The genotyping of genomic representations of individual samples was performed substantially as described <a href="#pone.0001682-Wenzl1">[7]</a>, with the exception that the polylinker fragment (reference in DArT assay) was labelled with FAM instead of Cy5 dye. Arrays were scanned with 10 µm resolution at 543 nm (Cy3) and 488 nM (FAM) on a LS300 confocal laser scanner (Tecan, Grödig, Austria) as described in Akbari <em>et al.</em> <a href="#pone.0001682-Akbari1">[11]</a>. Array images were analyzed with DArTsoft 7.4 (Diversity Arrays Technology P/L, Canberra, Australia). The program automatically recognized array features using a seeded-region-growth algorithm and reported, for each fluorescent channel, the average and standard deviation (SD) of pixel intensities within and around each array feature, the fraction of saturated pixels within each feature and the number of pixels of each feature, amongst other parameters (C. Cayla, personal communication). Clones with variable relative hybridization intensity across slides were subjected to fuzzy k-means clustering to convert relative hybridization intensities into binary scores (presence vs. absence). The quality of each marker was then determined using several parameters including 1) p-value, the variance of the relative target hybridization intensity between allelic states as a percentage of the total variance, 2) call rate, the percentage of DNA samples with binary (1 or 0) allele calls and 3) reproducibility, the fraction of concordant calls for replicate assays (C. Cayla, personal communication). Samples with cluster membership probability calculated by DArTsoft's clustering algorithm below the threshold of 0.8 were not classified (“X”). The frequencies of <em>Asplenium</em> and <em>Garovaglia</em> DArT markers with at least one X score across the sample set were 178 and 1 respectively. This discrepancy may be explained by differences in ploidy between <em>Asplenium</em> (diploid sporophyte) and <em>Garovaglia</em> (haploid gametophyte) because scoring efficiency would be expected to diminish as a result of intermediate signal intensities corresponding to copy number variation in polyploid samples. Indeed, the majority (56%) of X scores in the <em>Asplenium</em> data set were detected in the genotype of the known allotetraploid F1 hybrid sample of <em>A. adulterinum</em>.</p>


<h4>Suppression subtractive hybridization (SSH)</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>SSH was performed using a modification of the protocol given by Diatchenko <em>et al.</em> <a href="#pone.0001682-Diatchenko1">[17]</a>. <em>Asplenium viride</em> sample Avi284b and <em>G. elegans</em> ssp. <em>dietrichiae</em> sample Gel6504 were selected arbitrarily as subtraction drivers for the <em>Asplenium</em> and <em>Garovaglia</em> experiments respectively. The following different mixtures (“testers”) of <em>Asplenium</em> and <em>Garovaglia</em> samples were used: <em>Asplenium</em> tester 1 (all <em>A. viride</em> amplification products excluding the driver Avi284b), <em>Asplenium</em> tester 2 (all <em>Asplenium</em> amplification products excluding the driver Avi284b) and <em>Garovaglia</em> tester (all <em>Garovaglia</em> amplification products excluding the driver Gel6504). All <em>Asplenium</em> and <em>Garovaglia</em> tester and driver digestion products were phenol chloroform extracted, isopropanol precipitated and resuspended in deionized water. Digested testers were ligated to adapters with either Core 1 or Core 2, corresponding to the relevant digesting enzymes, in separate reactions (<a href="#pone-0001682-t003">Table 3</a>). Subtraction was carried out in either one or two stages. 600 ng driver was mixed with 20 ng tester-Core 1 and 20 ng tester-Core 2, isopropanol precipitated and resuspended in Subtraction Hybridization Buffer (50 mM HEPES pH 8.3, 0.5 M NaCl, 0.02 M EDTA pH 8.0, 10% PEG 8000 w/v). The dissolved pellet was denatured and hybridized for 5 hr at 68°C (for two-stage subtraction, 300ng freshly denatured driver was added and hybridized for a further 5 hr at 68°C). Subtraction products were used as a template in an amplification reaction using Core 1 and 2 primers (Core 1 <span class="gene">5′-GAGTAGTGCCAGAACGGTC-3′</span>, Core 2 <span class="gene">5′-TCGTAGACTGCGTATCCG-3′</span>).</p>
<div class="figure" id="pone-0001682-t003"><div class="img"><a name="pone-0001682-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Subtraction adapters (5′ to 3′).</span></strong></p><span>doi:10.1371/journal.pone.0001682.t003</span></div>

<h4>Quantitative evaluation of DArT data</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Polymorphism information content (PIC) <a href="#pone.0001682-Anderson1">[18]</a>, P and Q, reproducibility and call rate (C. Cayla, personal communication) were examined for both <em>Asplenium</em> and <em>Garovaglia</em> datasets to assess the distribution and reliability of hybridization (<a href="#pone.0001682.s001">Table S1</a>). The distribution of <em>Asplenium</em> DArT polymorphisms was examined on a per species basis according the following four categories: <em>A. trichomanes</em> private, <em>A. viride</em> private, <em>A. trichomanes</em>/<em>A. viride</em> shared, or <em>A. viride</em> “X”s but no “1”s. The <em>A. adulterinum</em> genotype was excluded from this process, as it contained polymorphisms shared by and private to both <em>A. viride</em> and <em>A. trichomanes</em>. Also excluded were 84 subtraction-derived DArT markers in the <em>Asplenium</em> data set that displayed an identical scoring pattern across all samples (this pattern did not match any known geographic patterns or systematic relationships and, when these 84 markers were included in a comparison of the diversity patterns between subtracted and non-subtracted genomic representations, substantial differences in inferred relationships were observed). Samples Avi288 and Avi289 were also excluded due to substrate ambiguity. The distribution of <em>Garovaglia</em> DArT polymorphisms was similarly examined on a per species basis according the following four categories: <em>G. powellii</em> private, <em>G. elegans</em> ssp. <em>dietrichiae</em> (Australia) private, <em>G. elegans</em> ssp. <em>elegans</em> and <em>G. elegans</em> fo. <em>latifolia</em> (Papua New Guinea) shared, or unassignable by taxonomy or geography. The <em>G. elegans</em> sample Gel6434 was excluded from this process, as it was derived from a mixed-taxon cushion, containing individuals of both <em>G. elegans</em> ssp. <em>dietrichiae</em> and <em>G. powellii</em>.</p>


<h4>Direct sequencing of chloroplast and nuclear loci</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>For <em>Asplenium</em>, total genomic DNA was extracted from silica desiccated leaf material as described above. The following portions of the chloroplast genome were amplified by the polymerase chain reaction: the combined <em>trn</em>L(<sub>CAA</sub>) gene and <em>trn</em>L<em>-trn</em>F(<sub>GAA</sub>) intergenic spacer (IGS) region (<em>trn</em>L-F), using primers FERN1 <a href="#pone.0001682-Trewick1">[16]</a> and F <a href="#pone.0001682-Taberlet1">[19]</a> as specified in Trewick <em>et al.</em> <a href="#pone.0001682-Trewick1">[16]</a> and the combined <em>rps4</em> gene and <em>rps4-trn</em>S(<sub>GAA</sub>) IGS region (<em>rps4-trn</em>S) following the primers and conditions specified in Schneider <em>et al.</em> <a href="#pone.0001682-Schneider1">[20]</a>. Exons 14 to 16 of the single-copy nuclear locus <em>pgi</em>C were amplified using primers 14F and16R according to Ishikawa <em>et al.</em> <a href="#pone.0001682-Ishikawa1">[21]</a>. Bidirectional cycle sequencing was carried out on an ABI 3730 capillary DNA sequencer using ABI PRISM BigDye Terminator Cycle Sequencing Ready Reaction Kit (Perkin-Elmer) and the same primers used for amplification. Sequence contigs were assembled and automatically aligned with subsequent manual corrections using SeqMan and MegAlign respectively (v. 6.00 Lasergene, DNAstar, Madison, WI, USA). For <em>Garovaglia</em>, total genomic DNA was extracted using a modification of the standard CTAB procedure <a href="#pone.0001682-Doyle1">[15]</a> as specified in Pedersen and Newton <a href="#pone.0001682-Pedersen1">[22]</a>. The chloroplast encoded <em>trn</em>G(<sub>UCC</sub>) intron (<em>trn</em>G) was amplified, sequenced and aligned using primers and conditions as specified in Pedersen and Newton <a href="#pone.0001682-Pedersen1">[22]</a>. All sequences were deposited in GenBank (<a href="#pone-0001682-t001">Tables 1</a> and <a href="#pone-0001682-t002">2</a>).</p>


<h4>Direct sequencing of selected DArT markers</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Inserts and part of the polylinker region were amplified for a representative selection of 96 DArT marker clones from each study using M13 forward and reverse primers as reported in Jaccoud <em>et al</em> <a href="#pone.0001682-Jaccoud1">[8]</a>. Amplification products were bidirectionally sequenced as described above using the same primers used for amplification. Successful sequence reads were obtained for 74 <em>Asplenium</em> and 74 <em>Garovaglia</em> DArT markers. Sequence contigs were assembled (vectors removed) and aligned as described above. All sequences were deposited in GenBank (<a href="#pone.0001682.s001">Table S1</a>).</p>


<h4>Sequence similarity searches and redundancy estimates</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Each of the 148 DArT marker sequences were queried against 1) all other sequenced DArT markers in the same study and 2) GenBank (<a href="http://www.ncbi.nlm.nih.gov">www.ncbi.nlm.nih.gov</a>) to identify putative sequence identities using the Basic Local Alignment Search Tool <a href="#pone.0001682-Altschul1">[23]</a> for nucleotides (blastn) and translated to proteins (tblastx). For inclusion in <a href="#pone.0001682.s001">Tables S1</a> and 4 we applied an arbitrary maximum expect value of 10<sup>−4</sup> and an arbitrary minimum of 70% identity at the nucleotide sequence level or two or more alignments to the same locus/function in different plant species.</p>


<h4>Phylogenetic analyses of DArT data and DNA sequence data</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Because the standard DArT protocol (DNA not sheared, not subtracted) yielded the most consistent data across plates and also produced the greatest number of DArT markers of any of the treatments, only DArT markers produced using the standard treatment were used in phylogenetic analyses. <em>Asplenium</em> samples Avi288 and Avi289 were excluded due to substrate ambiguity (<a href="#pone-0001682-t001">Table 1</a>). To infer relationships, a binary matrix was generated in which absence (0) or presence (1) of DNA fragments was scored (<a href="#pone.0001682.s001">Table S1</a>). Both DArT data sets (<em>Asplenium</em> and <em>Garovaglia</em>) were analysed using principal component analyses (PCA) and principal coordinate analyses (PCoA) with Euclidean distances using the software packages PAST <a href="#pone.0001682-Hammer1">[24]</a> and MVSP (Kovach Computing, Pentraeth, UK). In the case of <em>Garovaglia</em> data, a hierarchical PCoA procedure was performed in which the most distant specimens were excluded step-by-step to obtain a higher resolution in the scatter plots for the more closely related specimens. Phylogeny reconstructions were carried out using either distance-based approaches with LogDet and Nei-Li distance corrections or character based approaches, for example Bayesian inference of phylogeny with the MK1 model, maximum parsimony with equally weighted and unordered characters. NeighborNet analyses were employed to explore putative alternative relationships. Evolution of substrate specificity was reconstructed using the consensus topology based on all trees sampled from the plateau phase of the Bayesian analyses carried out with MrBayes. Both maximum parsimony and maximum likelihood approaches were applied to reconstruct changes in substrate preferences. These analyses were carried out with the appropriate software MacClade <a href="#pone.0001682-Maddison1">[25]</a>, Mesquite <a href="#pone.0001682-Maddison1">[25]</a>, MrBayes <a href="#pone.0001682-Huelsenbeck1">[26]</a>, PAUP* 4.0 <a href="#pone.0001682-Swofford1">[27]</a>, Splitstree <a href="#pone.0001682-Bryant1">[28]</a>, Tracer <a href="#pone.0001682-Rambaut1">[29]</a>, and Treecon <a href="#pone.0001682-Vandepeer1">[30]</a>. DNA sequence data were assembled and aligned using the Lasergene software package (DNASTAR, Madison, WI, USA) and MacClade. Phylogenetic hypotheses based on sequence data were generated using PAUP and MrBayes.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The frequencies of polymorphic markers detected on the <em>Asplenium</em> and <em>Garovaglia</em> discovery arrays were 6% and 15% respectively, resulting in 444 and 905 polymorphic markers recovered respectively (<a href="#pone-0001682-t005">Table 5</a>). The reproducibility of the two DArT genotyping arrays was successfully validated by independent assays from the same DNA (unpublished data). The polymorphism information content <a href="#pone.0001682-Anderson1">[18]</a> for each marker is shown in <a href="#pone.0001682.s001">Table S1</a>; with average PIC (0.21 for <em>Asplenium</em>, 0.25 for <em>Garovaglia</em>), lower than expected for randomly chosen bi-allelic loci (0.50) and lower than in previous DArT studies in barley (0.38) <a href="#pone.0001682-Wenzl1">[7]</a> and cassava (0.42) <a href="#pone.0001682-Xia1">[10]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To examine taxonomic specificity of DArT markers and to evaluate DArT protocol variables, especially enrichment by suppression subtractive hybridization (SSH), <em>Asplenium</em> and <em>Garovaglia</em> DArT markers (<a href="#pone.0001682.s001">Table S1</a>) were classified according to their distribution among species. <em>Asplenium</em> DArT markers were distributed as follows: <em>A. trichomanes</em> private (65%), <em>A. viride</em> private (16%), <em>A. trichomanes</em>/<em>A. viride</em> shared (13%) and <em>A. trichomanes/A. viride</em> putatively shared (present in <em>A. trichomanes</em>, “X” in <em>A. viride</em>, 5%). <em>Garovaglia</em> DArT marker presence was distributed as follows: <em>G. powellii</em> private (30%), <em>G. elegans</em> ssp. <em>dietrichiae</em> private (33%), <em>G. elegans</em> ssp. <em>elegans</em> and/or <em>G. elegans</em> fo. <em>latifolia</em> private (19%) unassignable by taxonomy (18%).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The fraction of <em>Asplenium</em> DArT markers in each of these categories varied according to treatment: DArT markers derived from the pooled <em>Asplenium</em> metagenome using the standard DArT protocol (no shearing, no SSH) (<em>Asplenium</em> plates 1–6, n = 103) were distributed as follows: <em>A. trichomanes</em> private (58%), <em>A. viride</em> private (12%), <em>A. trichomanes</em>/<em>A. viride</em> shared (20%) and <em>A. trichomanes/A. viride</em> putatively shared (10%). By contrast, DArT markers derived solely from <em>A. viride</em> genomic DNA driver Avi284b (<em>Asplenium</em> plates 7–8, n = 22) were enriched for <em>A. viride</em> private (50%) and <em>A. trichomanes</em>/<em>A. viride</em> shared (50%) polymorphisms. In the reverse treatment, <em>A. viride</em> Avi284b genomic DNA was subtracted from the pooled <em>Asplenium</em> metagenome (<em>Asplenium</em> plates 11–12 &amp; 15–16, n = 193) and the resulting dataset was enriched for <em>A. trichomanes</em> private polymorphisms (86%). Lastly, when <em>A. viride</em> Avi284b genomic DNA was subtracted from a sub-metagenome containing only <em>A. viride</em> genomic DNA samples except Avi284b (<em>Asplenium</em> plates 9–10 &amp; 13–14, n = 35), the proportion of <em>A. trichomanes</em> private polymorphisms was reduced (9%) and <em>A. viride</em> private polymorphisms increased (71%).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The fraction of <em>Garovaglia</em> DArT markers in each category also varied according to treatment: DArT markers derived from the pooled <em>Garovaglia</em> metagenome using the standard DArT protocol (no shearing, no SSH, <em>Garovaglia</em> plates 1–8, n = 262) were distributed as follows: <em>G. powellii</em> private (15%), <em>G. elegans</em> ssp. <em>dietrichiae</em> private (32%), <em>G. elegans</em> ssp. <em>elegans</em> and/or <em>G. elegans</em> fo. <em>latifolia</em> private (31%) or unassignable by taxonomy or geography (23%). By contrast, DArT markers derived solely from <em>G. elegans</em> genomic DNA driver Gel6504 (<em>Garovaglia</em> plates 15–16, n = 55) were enriched for <em>G. elegans</em> ssp. <em>dietrichiae</em> private (35%) and <em>G. elegans</em> ssp. <em>elegans</em> and/or <em>G. elegans</em> fo. <em>latifolia</em> private (42%) polymorphisms. In the reverse treatment, <em>G. elegans</em> Gel6504 genomic DNA was subtracted from the pooled <em>Garovaglia</em> metagenome (<em>Garovaglia</em> plates 9–14, n = 413) and the resulting dataset was enriched for <em>G. powellii</em> private polymorphisms (44%).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>To detect DArT marker specificity by substrate or geography, we examined the distribution of DArT markers in substrate-specific or geographically defined <em>Asplenium</em> and <em>Garovaglia</em> specimens. In the complete unsubtracted <em>Asplenium</em> data set (<em>Asplenium</em> plates 1–6), the <em>A. viride</em> private marker class included 1 global limestone-specific DArT marker (marker ID <em>Asplenium</em>6F4), 4 serpentine-specific DArT markers, and 7 non-substrate-specific DArT markers; the <em>A. trichomanes</em>/<em>A. viride</em> shared marker class included 6 DArT markers absent from global limestone and 1 DArT marker absent from serpentine (marker ID <em>Asplenium</em>3O24). In the complete unsubtracted <em>Garovaglia</em> data set (<em>Garovaglia</em> plates 1–8, <a href="#pone.0001682.s001">Table S1</a>), 33% of DArT markers are specific to Australian <em>G. elegans</em> samples while 19% are specific to Papua New Guinean <em>G. elegans</em> samples.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The genotype of <em>G. elegans</em> sample Gel6434 contains both <em>G. powellii</em>- and <em>G. elegans</em> (Australia)-private DArT markers, potentially consistent with a hybrid origin. Perplexingly, this accession was not a known or suspected hybrid. Re-examination of the cushion from which this accession was dissected revealed that it was a mixed-taxon cushion containing individuals of both <em>G. elegans</em> ssp. <em>dietrichiae</em> and <em>G. powellii</em>. Gel6434 was therefore retroactively excluded from the marker distribution analyses of all of the other markers reported above.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The copy number and genomic origin of any marker system, and the potential for redundancy between markers, is an important consideration for evolutionary applications. Only approximately 15% of <em>Asplenium</em> DArT markers hybridized to their respective un-amplified, labelled metagenomes (unpublished data) indicating that approximately 85% of <em>Asplenium</em> DArT markers are low-copy sequences. The level of redundancy between supposedly independent DArT markers on each genotyping array, as well as between SSH treatments, was assessed by sequencing a selection of 74 <em>Asplenium</em> and 74 <em>Garovaglia</em> DArT markers, (<a href="#pone.0001682.s001">Table S1</a>, summarized in <a href="#pone-0001682-t004">Table 4</a>). This revealed that 10.7% of <em>Asplenium</em> DArT markers and 16.2% of <em>Garovaglia</em> DArT markers shared over 99% identity at the nucleotide sequence level to other DArT markers (<a href="#pone-0001682-t005">Table 5</a>). Redundancy dropped to 3.3% and 5.6% respectively when SSH-derived DArT markers were removed. Similar redundancy estimates, including lower redundancy in un-subtracted libraries, are reported for wheat <a href="#pone.0001682-Akbari1">[11]</a>, tomato, sorghum and sugarcane (DArT P/L, unpublished).</p>
<div class="figure" id="pone-0001682-t004"><div class="img"><a name="pone-0001682-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Predicted locus/function for sequenced DArT markers with &gt;70% identity to one or more GenBank sequences or (<sup>a</sup>and) &gt;1 plant species hit for the same function.</span></strong></p><span>doi:10.1371/journal.pone.0001682.t004</span></div><div class="figure" id="pone-0001682-t005"><div class="img"><a name="pone-0001682-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Summary of quantitative evaluation of <em>Asplenium</em> and <em>Garovaglia</em> DArT markers (for individual markers see <a href="#pone.0001682.s001">Table S1</a>).</span></strong></p><span>doi:10.1371/journal.pone.0001682.t005</span></div><a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>BLAST searches of 74 <em>Asplenium</em> and 74 <em>Garovaglia</em> DArT marker sequences against GenBank recovered 18 (24%) and 14 (19%) significant alignments respectively, and included known mitochondrial (i.e. <em>rpl</em>2 mitochondrial pseudogene), chloroplast (i.e. <em>psb</em>A and <em>16s</em>) and nuclear (i.e. <em>Hcr2</em> and <em>cyc07/rps3</em>) loci as well as proteins (e.g. CYP20, EIF3B and G3PDH) and predicted expressed genes (<a href="#pone-0001682-t004">Table 4</a>).</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>PCO and PCoA analyses of the <em>Asplenium</em> and <em>Garovaglia</em> DArT data sets derived from the standard DArT procedure (not subtracted) recovered a cumulative explanatory percentage for axes 1 and 2 of 72.7% for <em>Asplenium</em> and 73.5% for <em>Garovaglia</em>, the latter increasing to 100% in the exclusively Papua New Guinean sample subset, but not in the exclusively Australian subset. These DArT data enabled reconstruction of intraspecific structure in <em>A. viride</em> revealing phylogeographic and substrate specificity patterns (<a href="#pone-0001682-g001">Figures 1</a> &amp; <a href="#pone-0001682-g002">2</a>). These patterns were not detectable in our analyses of cpDNA or nrDNA sequences, due to insufficient polymorphisms to generate a fully resolved phylogeny (data not shown). The <em>rps</em>4-<em>trn</em>S IGS contained just two polymorphic positions, while the <em>trn</em>L-F region had one unambiguous polymorphic nucleotide position. In both cpDNA regions, one polymorphic site separates samples Avi169 and Avi272 from the remaining samples of <em>A. viride.</em> Variation in the nuclear <em>pgi</em>C data set was similarly low, with seven unambiguous polymorphic sites each of which was specific to only one specimen (e.g. Avi69, Avi245, Avi284a).</p>
<div class="figure" id="pone-0001682-g001"><div class="img"><a name="pone-0001682-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Two dimensional PCO scatter plot of the <em>Asplenium viride</em> DArT marker data set (triangles) plus a single specimen of <em>A. adulterinum</em> (star) and <em>A. trichomanes</em> (hexagon).</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Color of symbol corresponds to the substrate on which the sample was growing: limestone = white, serpentine = black and magnesit = gray. Numbers correspond to the sample number (<a href="#pone-0001682-t001">Table 1</a>).</p>
<span>doi:10.1371/journal.pone.0001682.g001</span></div><div class="figure" id="pone-0001682-g002"><div class="img"><a name="pone-0001682-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Splitgraph obtained by a NeighborNet analysis with LogDet distances for the <em>Asplenium viride</em> DArT marker set.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>Sample numbers, symbols and symbol colors as in <a href="#pone-0001682-g001">Figure 1</a>. The dotted ovals mark putative groups found in phylogenetic analyses (<a href="#pone-0001682-g003">Figure 3</a>).</p>
<span>doi:10.1371/journal.pone.0001682.g002</span></div><a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>All analyses of the <em>Asplenium</em> DArT marker data indicate one group with a widespread limestone genotype and a second group containing a north-African/Iberian limestone genotype with the two magnesit samples from Austria as putative relatives (<a href="#pone-0001682-g001">Figures 1</a>, <a href="#pone-0001682-g002">2</a>, <a href="#pone-0001682-g003">3</a>). In PCO analyses, the three <em>A. viride</em> samples occurring on serpentine are closely associated, and form a putative group in some phylogenetic reconstructions, forming a basal grade at the base of the remaining <em>A. viride</em> groups in others. The three samples of serpentine <em>A. viride</em> were collected in three different regions of Europe: Scandinavia (Avi291), Scotland (Avi169) and Austria (Avi272). The <em>Asplenium</em> DArT marker data set suggests that serpentine was the ancestral substrate of <em>A. viride</em> with two independent colonizations onto limestone (<a href="#pone-0001682-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0001682-g003"><div class="img"><a name="pone-0001682-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Consensus phylogram obtained using Bayesian inference of phylogeny analyses for the <em>A. viride</em> DArT marker data set rooted with <em>A. trichomanes</em>.</span></strong></p><a id="article1.body1.sec3.fig3.caption1.p1" name="article1.body1.sec3.fig3.caption1.p1"></a><p>Posterior probabilities (0.XX), maximum parsimony bootstrap values (M XX), and bootstrap values for a neighbor joining distance analyses with Nei-Li distances (N XX) are given above branches if they are &gt;0.75 for posterior probabilities or &gt;50% for bootstrap values. Sample numbers and symbols as in <a href="#pone-0001682-g001">Figure 1</a>. Color of symbol corresponds to the substrate on which the sample was growing: limestone = yellow, serpentine = dark red and magnesit = green. Branch color corresponds to the character state reconstructed using a maximum parsimony approach. The pie charts represent the likelihood of substrate preference for branches with a putative switch between limestone, magnesit, or serpentine substrates, shown as either limestone versus non-limestone or serpentine versus non-serpentine.</p>
<span>doi:10.1371/journal.pone.0001682.g003</span></div><a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>DNA sequence data from chloroplast (<em>Asplenium trn</em>L-F, <em>rps4</em>-<em>trn</em>S; <em>Garovaglia trn</em>G) and nuclear (<em>Asplenium pgi</em>C) loci largely corroborated the results of the above analyses of <em>Asplenium</em> and <em>Garovaglia</em> DArT data (<a href="#pone-0001682-g002">Figures 2</a> &amp; <a href="#pone-0001682-g004">4</a> and unpublished data). The analysis of <em>Garovaglia</em> chloroplast DNA sequence data resulted in a phylogram with a clear separation of Papua New Guinean and Australian specimens (<a href="#pone-0001682-g004">Figure 4</a>). The same clusters were recovered using <em>Garovaglia</em> DArT markers. Neither cpDNA nor the highly variable DArT markers recovered any phylogeographic patterns within Australia. In a hierarchical PCO analysis procedure (step by step exclusion of more distantly related samples), no increase in the cumulative percentage explained by axes 1 and 2 was observed.</p>
<div class="figure" id="pone-0001682-g004"><div class="img"><a name="pone-0001682-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001682" data-uri="info:doi/10.1371/journal.pone.0001682.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001682.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001682.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001682.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001682.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Two-dimensional PCO scatter plot of the <em>Garovaglia elegans</em> DArT marker data set superimposed with the phylogenetic tree obtained based on cpDNA.</span></strong></p><a id="article1.body1.sec3.fig4.caption1.p1" name="article1.body1.sec3.fig4.caption1.p1"></a><p><em>Garovaglia powelli</em> is included to root the phylogenetic tree. Triangles correspond to single species samples, whereas the star corresponds to a mixed sample that includes individuals of <em>G. elegans</em> and <em>G. powellii</em>.</p>
<span>doi:10.1371/journal.pone.0001682.g004</span></div><a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>In both <em>Asplenium</em> and <em>Garovaglia,</em> DArT markers enabled the detection of additive variation caused by hybridization (<em>A. adulterinum</em>) or a mixed-taxon specimen (<em>Garovaglia</em>). The genotype of <em>A. adulterinum</em>, a known allopolyploid <em>A. viride x trichomanes</em> hybrid, contains DArT markers that are private to both <em>A. trichomanes</em> and to the widespread limestone <em>A. viride</em> (<a href="#pone.0001682.s001">Table S1</a>). The position of <em>A. adulterinum</em> in the PCO analysis is intermediate between that of <em>A. trichomanes</em> and widespread limestone <em>A. viride</em> (<a href="#pone-0001682-g001">Figure 1</a>). Chloroplast DNA sequence data corroborate these DArT-based results: <em>A. adulterinum</em> exhibits an <em>A. viride</em> chloroplast haplotype (100% sequence identity with the common cp haplotype of <em>A. viride</em>), consistent with the known maternal inheritance of chloroplast DNA in <em>Asplenium</em> and that <em>A. viride</em> is always <em>A. adulterinum's</em> ovule parent <a href="#pone.0001682-Vogel1">[31]</a>, <a href="#pone.0001682-Vogel2">[32]</a>. The genotype of the mixed-taxon <em>Garovaglia</em> sample Gel6434 contains DArT markers private to both <em>G. powellii</em> and the Australian <em>G. elegans</em> group.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>This study demonstrates that the DArT protocol can be utilized to reproducibly detect largely low-copy genomic variation in sample sets from two lineages of seed-free land plants, suggesting that DArT may be useful across a broader taxonomic spectrum than that addressed in previous studies focusing on cultivated angiosperm species <a href="#pone.0001682-Wittenberg1">[6]</a>–<a href="#pone.0001682-Akbari1">[11]</a>. <em>Asplenium</em> and <em>Garovaglia</em> DArT data were useful for making biological inferences, for example, phylogeographic structure in plants with a high dispersal capacity, naturally occurring hybridization, exploration of chimeric environmental samples, and the reconstruction of ecological differentiation. Here we discuss procedures for both detecting and analysing DArT data for evolutionary applications, practical considerations for investigators, and the evolutionary hypotheses suggested by these results.</p>

<h4>Procedures for detecting DArT markers for evolutionary studies</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The frequencies of polymorphism detected on the <em>Asplenium</em> (6%) and <em>Garovaglia</em> (15%) discovery arrays are similar to those reported for previous studies in cultivated angiosperms and their wild relatives: barley (2.9–10.4%) <a href="#pone.0001682-Wenzl2">[12]</a>, cassava (9–14%) <a href="#pone.0001682-Xia1">[10]</a>, mouse-ear cress (7.3%) <a href="#pone.0001682-Wittenberg1">[6]</a>. The number of polymorphic DArT markers obtained from <em>Asplenium</em> (444) and <em>Garovaglia</em> (905) is a function of number DArT clones screened in the discovery array step, and the frequency of polymorphism therein. Because marker redundancy was estimated as less than 10% between DArT markers from un-subtracted metagenomic libraries, it is clear that several times as many DArT markers can be developed for the <em>Asplenium</em> or <em>Garovaglia</em> metagenomes through screening additional clones/arrays before saturation of marker libraries is reached.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>That the average PIC values (0.21 for <em>Asplenium</em>, 0.25 for <em>Garovaglia</em>) are lower than previous DArT studies in barley (0.38) <a href="#pone.0001682-Wenzl1">[7]</a> and cassava (0.42) <a href="#pone.0001682-Xia1">[10]</a>, and is likely due to the choice of sample sets with a strong structure, and clearly defined outgroups. All DArT markers private to the outgroups exhibited low PIC values as expected, and thus the high frequency of such markers impacted strongly on average PIC value.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Because the aim of this study was to pilot DArT in new taxonomic contexts and apply it to different classes of questions, a range of methodologies were compared to inform similar future studies. The distributions of DArT markers derived from sample-specific vs. pooled metagenomes exhibited expected shifts in marker distribution. For example, the DArT markers derived from one <em>A. viride</em> sample (Avi284b, <em>Asplenium</em> plates 7 &amp; 8) or the <em>A. viride</em> sub-metagenome (<em>Asplenium</em> plates 9, 10, 13 &amp; 14) were enriched for markers private to and shared with <em>A. viride</em> compared to the baseline marker set derived from the complete pooled <em>Asplenium</em> metagenome. Similarly, the distributions of DArT markers derived from subtracted vs. un-subtracted libraries exhibited expected shifts in marker distribution. For example, when Avi284b genomic DNA was subtracted from the <em>A. viride</em> sub-metagenome (all <em>A. viride</em> samples except Av284b), the number of <em>A. viride</em> private polymorphisms was enriched as expected. Thus restricting the starting metagenome and/or performing SSH allows investigators to intentionally enrich for DArT markers private to any one or more input samples in cases where such an enrichment is desired. However, when the maximum number of random markers is preferred, such as for unbiased evolutionary analyses, SSH is unnecessary for or even detrimental to the detection and typing of DArT markers in randomly derived sample sets.</p>


<h4>Procedures for analysing DArT markers to reconstruct the evolutionary history of plant species</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The phylogenetic analyses of <em>Asplenium</em> and <em>Garovaglia</em> DArT data largely corroborated results generated using direct sequencing, and often improved upon the resolution provided by these methods. Considering the small sample set used in these pilot studies, our result holds much promise for the use of DArT to address evolutionary-genetic hypotheses when large sample sets are employed. The two case studies were designed to demonstrate the utility of DArT markers for studies in which highly variable markers are required for evolutionary interpretation and in which relationships are poorly resolved using DNA sequence data due to a low level of variation. Various other marker systems, isozymes and arbitrarily amplified dominant (AAD) markers for example, have been employed to overcome this problem <a href="#pone.0001682-Bussell1">[4]</a>, <a href="#pone.0001682-Pelser1">[33]</a>, <a href="#pone.0001682-Koopman1">[34]</a>. DArT markers share several features with AAD markers but also differ substantially in other respects. DArT markers and AAD markers have the same binary data structure, that is, presence or absence of markers or bands and a dominant inheritance. Here we discuss some of the relevant distinguishing features of DArT data in comparison with AAD markers, and recommend how DArT data should be handled in phylogenetic or phylogeographic studies.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Non-independence of markers and false homology of bands are common problems associated with AAD markers. These drawbacks restrict the application of AAD markers to very closely related samples (e.g. within species) and often necessitate a dense sampling of the lineages to infer evolutionary history <a href="#pone.0001682-Bussell1">[4]</a>. Because DArT markers are detected by DNA-DNA hybridization rather than fragment size, DArT markers were expected to suffer only a very low level of erroneous homology assessment/assignment. This assumption was validated by Wenzl <em>et al.</em> <a href="#pone.0001682-Wenzl2">[12]</a> through genetic mapping of over 2000 DArT markers in many mapping populations of barley, where fewer than 2% of the markers mapped to more than one locus in the genome. The high percentage (mostly above 70%) accumulation for axis 1 and axis 2 in all PCO and PCoA analyses of DArT markers in both case studies provides additional support that this expectation is justified. DNA sequencing of DArT markers confirmed that redundancy between different DArT markers is low. Some DArT markers, however, are still likely to be erroneously scored as independent if they originated from partially overlapping genomic regions. DArT marker DNA sequences suggest that very few DArT markers are mistakenly treated as independent and further comfort is derived from the observation that all three plant genomes contributed to the DArT data set.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>A related problem is the assumption of homology when DArT markers are absent and data are analysed in a parsimony framework <a href="#pone.0001682-Simmons1">[35]</a>. Similar to other presence/absence data (AFLP, ISSR), homology assessment is restricted to the hypothesis that the state of presence is the result of presence of homologous markers but the state of absence may be the result of alternative processes <a href="#pone.0001682-Simmons1">[35]</a>. Currently, it is not possible to evaluate the putative misleading effects of such a bias. A further restriction of presence/absence data is the limited number of character states (two). However, this problem is likely to be offset by the large number of DArT markers available <a href="#pone.0001682-Simmons1">[35]</a>, which exceeds several hundred for both case studies presented here.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>Distance-based methods are putatively less affected by these problems because some similarity scores take data structure and heritage (which in DArT is likely dominant) into account. Thus, Nei-Li similarity score <a href="#pone.0001682-Nei1">[36]</a> is probably the most appropriate for DArT data, for the same reasons given for AFLP data, i.e. presence/absence data structure and dominant inheritance <a href="#pone.0001682-Archibald1">[37]</a>. This is an advantage of using the DArT technique because existing software tools designed for analyses of AAD markers can be easily adjusted to analyse DArT data. However, users should keep in mind the existence of modified versions of these distance measures <a href="#pone.0001682-Archibald1">[37]</a>. The selection of tree building algorithms, for example UPGMA and Neighbor-Joining (NJ) is also important in the context of distance-based approaches. UPGMA is likely to result in incorrect topology as the result of the imbalanced distribution of variation among putative branches in the tree. Recently developed approaches such as NeighborNet and split decomposition analyses as implemented in Splitstree <a href="#pone.0001682-Bryant1">[28]</a> are likely to be more powerful approaches for phylogenetic reconstruction based on DArT data transformed into appropriate distance matrices. Importantly, DArT procedures hold the potential to generate co-dominant markers by taking into account the strength of the signal for each DArT marker. This improvement could potentially overcome several restrictions limiting the analyses of AAD markers and current (dominant) DArT markers.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>In general, character-based approaches are a more powerful tool to assess phylogenetic patterns and some authors therefore argue for their superiority <a href="#pone.0001682-Simmons1">[35]</a>. Most parsimony analyses are now commonly used in addition to distance based approaches to evaluate AAD data sets, and these analyses should also be employed to analyse DArT data sets. More recently, Bayesian inference of phylogeny was made available by implementing a restriction site (binary) model based on an F81-like model into MrBayes 3.01 <a href="#pone.0001682-Huelsenbeck1">[26]</a>. This model can also be used to analyse DArT marker data sets due to the underlying assumptions for the restriction sites model, non-observable absence data for example, fitting the structure of DArT data.</p>


<h4>Practical advantages of DArT in evolutionary research</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The results reported here provoke the question: what are the practical advantages and disadvantages of using DArT (as opposed to other methods) to detect genomic variation for analysis in an evolutionary or phylogenetic framework? DArT offers seven key advantages over other widely used methods for detecting molecular variation: First, DArT is a sequence-independent discovery tool that requires no preliminary sequence information such as the identification of candidate loci or time-intensive development and optimization of primers. Thus DArT is likely to find variable sequences that other sequence-dependent technologies might miss. Second, DArT recovers a high level of variation, and offers the potential for very high throughput analysis of both markers and specimens. Third, the presence of a DArT marker is determined by DNA-DNA hybridization rather than fragment length and hence DArT suffers less from ambiguous homology assessments than other finger-printing methods utilising AAD markers, especially RAPDs, ISSRs and AFLPs <a href="#pone.0001682-Bussell1">[4]</a>. Indeed, independent segregation of DArT markers in mapping experiments in barley <a href="#pone.0001682-Wenzl2">[12]</a> as well as in other species like wheat and sorghum (unpublished data) provides strong evidence for this point. Fourth, DArT is highly reproducible; DArT marker scoring is consistent among independent replicates (this study and <a href="#pone.0001682-Wenzl1">[7]</a>), indicating that DArT may be a more reliable genotyping method than other AAD markers <a href="#pone.0001682-Bussell1">[4]</a>. Fifth, redundancy between DArT markers is both quantifiable and low (this study and <a href="#pone.0001682-Wenzl1">[7]</a>). Sixth, DArT markers are very easily sequenced (no gel isolation required), allowing similarity searches against sequence databases and internal redundancy estimates. Twenty-four percent of <em>Asplenium</em> and 19% of <em>Garovaglia</em> DArT marker DNA sequences yielded GenBank alignments including known mitochondrial, chloroplast and nuclear loci, proteins and predicted expressed genes (<a href="#pone-0001682-t004">Table 4</a>). Amongst these were matches to <em>rpl2, psbA</em> and <em>rps3</em> genes, and the locus encoding G3PDH, all of which are routinely used in sequence-based evolutionary studies in plants <a href="#pone.0001682-Laroche1">[38]</a>–<a href="#pone.0001682-Dane1">[44]</a>. This demonstrates that DArT is capable of identifying known polymorphic loci, and suggests that the sequences of the hundreds of other unknown polymorphic fragments recovered by DArT may facilitate primer design for the amplification of novel target loci for direct sequencing. Finally, DArT is able to detect both genetic hybrids and chimeric environmental samples, demonstrating its utility for studies exploring the origin of hybrids and their specific genetic structure.</p>


<h4>Evolutionary insights derived from this study</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Analyses of <em>Asplenium</em> and <em>Garovaglia</em> DArT data enabled the reconstruction of intraspecific structure in <em>A. viride</em> and <em>G. elegans</em> and revealed substrate specificity and/or phylogeographic patterns which, for <em>A. viride,</em> were not detectable using chloroplast or nuclear DNA sequences. All analyses of the <em>Asplenium</em> DArT data indicated a widespread limestone genotype group and a second limestone genotype group occurring in northern Africa and the Iberian Peninsula, incorporating two magnesit samples from Austria as putative relatives (<a href="#pone-0001682-g001">Figures 1</a>, <a href="#pone-0001682-g002">2</a>, <a href="#pone-0001682-g003">3</a>). The phylogeographic and substrate specificity patterns detected in analyses of <em>A. viride</em> DArT data (<a href="#pone-0001682-g001">Figures 1</a> and <a href="#pone-0001682-g002">2</a>) provoke at least two alternative scenarios of substrate evolution in <em>A. viride</em>: (1) <em>A. viride</em> originated on serpentine and twice colonized limestone habitats or (2) the extant serpentine populations of <em>A. viride</em> are relicts able to survive on a less favorable substrate. Both scenarios are consistent with the lack of alleles private to serpentine populations of <em>A. viride</em> (unpublished data). For <em>Garovaglia,</em> both DArT and cpDNA data clearly separated the Australian from the Papua New Guinean samples (<a href="#pone-0001682-g004">Figure 4</a>). When contrasted against the geographical distinction between Papua New Guinean and Australian <em>G. elegans</em> (ssp. <em>dietrichiae</em>, endemic to Australia) specimens, the lack of micro-biogeographic structure detected in Australian ssp. dietrichiae clade (<a href="#pone-0001682-g001">Figures 1</a> and <a href="#pone-0001682-g002">2</a>) suggests a high rate of gene exchange among the Australian populations (which may be a reflection of its nanandrous breeding system) or a low rate of gene exchange between Australian and New Guinean populations. Lastly, DArT enabled the detection of additive variation caused by genetic hybridization (<em>A. adulterinum</em>) or a mixed-taxon specimen (<em>G. elegans</em> Gel6434).</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>In conclusion, while DArT has been previously used to complement existing technologies in crop breeding and genomics <a href="#pone.0001682-Xia1">[10]</a>, our studies show that the abundant non <em>a priori</em> comparative molecular data generated by DArT also holds real potential for use in a wide range of high-throughput evolutionary studies. These include the detection of biological correlations and phenomena including similarity, hybridization, mixed environmental samples, ecological differentiation and geographical distribution, as well as other studies of non-model organisms including gene discovery, QTL mapping, population and conservation genetics, speciation and environmental forensics.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001682"></div><a name="pone.0001682.s001" id="pone.0001682.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001682.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">DArT markers, specimen genotypes and sequence-associated data. This supplementary table contains 1) detailed information about each of the 1349 DArT markers recovered for this study including marker-specific library preparation data and statistics, 2) DArT genotyptes of <em>Asplenium</em> and <em>Garovaglia</em> specimens, and 3) sequence-associated data from each of the 148 sequenced DArT markers including GenBank accession numbers, fragment lengths, blastn and tblastx identifications, % identity (closest match), and number different species hit for same function.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001682.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.50 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We acknowledge Heinar Streimann and Jacques de Sloover for collecting <em>Garovaglia</em> specimens in Papua New Guinea and the curators of the New York Botanical Garden (NY) for giving us permission to use these specimens, Drs Adolf and Oluna Ceska, Dr Curtis Bjjoerk, Dr Fred Rumsey, Dr Mark Carine and Mr Stephan Jessen for support collecting <em>Asplenium</em> specimens, and Meredith Murphy Thomas for technical assistance. We also thank many members of Diversity Arrays Technology Pty Ltd team for help in execution of this project.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KJ AK. Performed the experiments: KJ SA ME LR GU NP. Analyzed the data: KJ HS SA ME LR NP AK. Contributed reagents/materials/analysis tools: AN JV AK. Wrote the paper: KJ HS SA AK. Other: Technical assistance: SR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001682-Avise1" id="pone.0001682-Avise1"></a>Avise JC (2004) Molecular markers, natural history, and evolution. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001682-Ranz1" id="pone.0001682-Ranz1"></a>Ranz JM, Machado CA (2006) Uncovering evolutionary patterns of gene expression using microarrays. Trends in Ecology &amp; Evolution  21: 29–37.  <ul class="find" data-citedArticleID="1063328" data-doi="10.1016/j.tree.2005.09.002"><li><a href="http://dx.doi.org/10.1016/j.tree.2005.09.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uncovering+evolutionary+patterns+of+gene+expression+using+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uncovering+evolutionary+patterns+of+gene+expression+using+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001682-Gilad1" id="pone.0001682-Gilad1"></a>Gilad Y, Borevitz J (2006) Using DNA microarrays to study natural variation. Current Opinion in Genetics &amp; Development  16: 553–558.  <ul class="find" data-citedArticleID="1063290" data-doi="10.1016/j.gde.2006.09.005"><li><a href="http://dx.doi.org/10.1016/j.gde.2006.09.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+DNA+microarrays+to+study+natural+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+DNA+microarrays+to+study+natural+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001682-Bussell1" id="pone.0001682-Bussell1"></a>Bussell JD, Waycott M, Chappill JA (2005) Arbitrarily amplified DNA markers as characters for phylogenetic inference. Perspectives in Plant Ecology Evolution and Systematics  7: 3–26.  <ul class="find" data-citedArticleID="1063282" data-doi="10.1016/j.ppees.2004.07.001"><li><a href="http://dx.doi.org/10.1016/j.ppees.2004.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arbitrarily+amplified+DNA+markers+as+characters+for+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arbitrarily+amplified+DNA+markers+as+characters+for+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001682-Kilian1" id="pone.0001682-Kilian1"></a>Kilian A, Huttner E, Wenzl P, Jaccoud D, Carling J, et al.  (2003)  The fast and the cheap: SNP and DArT-based whole genome profiling for crop improvement. In: Tuberosa R, Phillips RL, Gale M, editors. In the Wake of the Double Helix: From the Green Revolution to the Gene Revolution. Bologna, Italy: Avenue Media. pp. 443–461.  <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001682-Wittenberg1" id="pone.0001682-Wittenberg1"></a>Wittenberg AHJ, van der Lee T, Cayla C, Kilian A, Visser RGF, et al.  (2005) Validation of the high-throughput marker technology DArT using the model plant Arabidopsis thaliana. Molecular Genetics and Genomics  274: 30–39.  <ul class="find" data-citedArticleID="1063352" data-doi="10.1007/s00438-005-1145-6"><li><a href="http://dx.doi.org/10.1007/s00438-005-1145-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Validation+of+the+high-throughput+marker+technology+DArT+using+the+model+plant+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Validation+of+the+high-throughput+marker+technology+DArT+using+the+model+plant+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001682-Wenzl1" id="pone.0001682-Wenzl1"></a>Wenzl P, Carling J, Kudrna D, Jaccoud D, Huttner E, et al.  (2004) Diversity Arrays Technology (DArT) for whole-genome profiling of barley. Proceedings of the National Academy of Sciences of the United States of America  101: 9915–9920.  <ul class="find" data-citedArticleID="1063348" data-doi="10.1073/pnas.0401076101"><li><a href="http://dx.doi.org/10.1073/pnas.0401076101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+Arrays+Technology+%28DArT%29+for+whole-genome+profiling+of+barley." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+Arrays+Technology+%28DArT%29+for+whole-genome+profiling+of+barley.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001682-Jaccoud1" id="pone.0001682-Jaccoud1"></a>Jaccoud D, Peng K, Feinstein D, Kilian A (2001) Diversity Arrays: a solid state technology for sequence information independent genotypinG. Nucleic Acids Research  29: 1–7.  <ul class="find" data-citedArticleID="1063300" data-doi="10.1093/nar/29.4.e25"><li><a href="http://dx.doi.org/10.1093/nar/29.4.e25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+Arrays%3A+a+solid+state+technology+for+sequence+information+independent+genotypinG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+Arrays%3A+a+solid+state+technology+for+sequence+information+independent+genotypinG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001682-Lezar1" id="pone.0001682-Lezar1"></a>Lezar S, Myburg AA, Berger DK, Wingfield MJ, Wingfield BD (2004) Development and assessment of microarray-based DNA fingerprinting in Eucalyptus grandis. Theoretical and Applied Genetics  109: 1329–1336.  <ul class="find" data-citedArticleID="1063312" data-doi="10.1007/s00122-004-1759-9"><li><a href="http://dx.doi.org/10.1007/s00122-004-1759-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+and+assessment+of+microarray-based+DNA+fingerprinting+in+Eucalyptus+grandis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+and+assessment+of+microarray-based+DNA+fingerprinting+in+Eucalyptus+grandis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001682-Xia1" id="pone.0001682-Xia1"></a>Xia L, Peng KM, Yang SY, Wenzl P, de Vicente MC, et al.  (2005) DArT for high-throughput genotyping of Cassava (Manihot esculenta) and its wild relatives. Theoretical and Applied Genetics  110: 1092–1098.  <ul class="find" data-citedArticleID="1063354" data-doi="10.1007/s00122-005-1937-4"><li><a href="http://dx.doi.org/10.1007/s00122-005-1937-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DArT+for+high-throughput+genotyping+of+Cassava+%28Manihot+esculenta%29+and+its+wild+relatives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DArT+for+high-throughput+genotyping+of+Cassava+%28Manihot+esculenta%29+and+its+wild+relatives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001682-Akbari1" id="pone.0001682-Akbari1"></a>Akbari M, Wenzl P, Caig V, Carling J, Xia L, et al.  (2006) Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome. Theoretical and Applied Genetics  113: 1409–1420.  <ul class="find" data-citedArticleID="1063270" data-doi="10.1007/s00122-006-0365-4"><li><a href="http://dx.doi.org/10.1007/s00122-006-0365-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+arrays+technology+%28DArT%29+for+high-throughput+profiling+of+the+hexaploid+wheat+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+arrays+technology+%28DArT%29+for+high-throughput+profiling+of+the+hexaploid+wheat+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001682-Wenzl2" id="pone.0001682-Wenzl2"></a>Wenzl P, Li HB, Carling J, Zhou MX, Raman H, et al.  (2006) A high-density consensus map of barley linking DArT markers to SSR, RFLP and STS loci and agricultural traits. BMC Genomics  7: 206.  <ul class="find" data-citedArticleID="1063350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+high-density+consensus+map+of+barley+linking+DArT+markers+to+SSR%2C+RFLP+and+STS+loci+and+agricultural+traits.&amp;auth=&amp;atitle=A+high-density+consensus+map+of+barley+linking+DArT+markers+to+SSR%2C+RFLP+and+STS+loci+and+agricultural+traits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+high-density+consensus+map+of+barley+linking+DArT+markers+to+SSR%2C+RFLP+and+STS+loci+and+agricultural+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+high-density+consensus+map+of+barley+linking+DArT+markers+to+SSR%2C+RFLP+and+STS+loci+and+agricultural+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001682-Yang1" id="pone.0001682-Yang1"></a>Yang SY, Pang W, Ash G, Harper J, Carling J, et al.  (2006) Low level of genetic diversity in cultivated pigeonpea compared to its wild relatives is revealed by diversity arrays technology. Theoretical and Applied Genetics  113: 585–595.  <ul class="find" data-citedArticleID="1063356" data-doi="10.1007/s00122-006-0317-z"><li><a href="http://dx.doi.org/10.1007/s00122-006-0317-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Low+level+of+genetic+diversity+in+cultivated+pigeonpea+compared+to+its+wild+relatives+is+revealed+by+diversity+arrays+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Low+level+of+genetic+diversity+in+cultivated+pigeonpea+compared+to+its+wild+relatives+is+revealed+by+diversity+arrays+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001682-Lovis1" id="pone.0001682-Lovis1"></a>Lovis JD, Reichstein T (1967) Über das spontane Entstehen von <em>Asplenium adulterinum</em> aus einem natürlichen Bastard. Die Natuerwissenschaften  55: 117–120.  <ul class="find" data-citedArticleID="1063314" data-doi="10.1007/bf00624239"><li><a href="http://dx.doi.org/10.1007/bf00624239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%C3%9Cber+das+spontane+Entstehen+von+Asplenium+adulterinum+aus+einem+nat%C3%BCrlichen+Bastard." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%C3%9Cber+das+spontane+Entstehen+von+Asplenium+adulterinum+aus+einem+nat%C3%BCrlichen+Bastard.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001682-Doyle1" id="pone.0001682-Doyle1"></a>Doyle J, Doyle J (1987) A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem. Bull.  19: 11–15.  <ul class="find" data-citedArticleID="1063288"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+rapid+DNA+isolation+procedure+for+small+quantities+of+fresh+leaf+tissue.&amp;auth=&amp;atitle=A+rapid+DNA+isolation+procedure+for+small+quantities+of+fresh+leaf+tissue." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid+DNA+isolation+procedure+for+small+quantities+of+fresh+leaf+tissue." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid+DNA+isolation+procedure+for+small+quantities+of+fresh+leaf+tissue.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001682-Trewick1" id="pone.0001682-Trewick1"></a>Trewick SA, Morgan-Richards M, Russell SJ, Henderson S, Rumsey FJ, et al.  (2002) Polyploidy, phylogeography and Pleistocene refugia of the rockfern <em>Asplenium</em> ceterach: evidence from chloroplast DNA. Molecular Ecology  11: 2003–2012.  <ul class="find" data-citedArticleID="1063340" data-doi="10.1046/j.1365-294x.2002.01583.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01583.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polyploidy%2C+phylogeography+and+Pleistocene+refugia+of+the+rockfern+Asplenium+ceterach%3A+evidence+from+chloroplast+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polyploidy%2C+phylogeography+and+Pleistocene+refugia+of+the+rockfern+Asplenium+ceterach%3A+evidence+from+chloroplast+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001682-Diatchenko1" id="pone.0001682-Diatchenko1"></a>Diatchenko L, Lau YFC, Campbell AP, Chenchik A, Moqadam F, et al.  (1996) Suppression subtractive hybridization: A method for generating differentially regulated or tissue-specific cDNA probes and libraries. Proceedings of the National Academy of Sciences of the United States of America  93: 6025–6030.  <ul class="find" data-citedArticleID="1063286" data-doi="10.1073/pnas.93.12.6025"><li><a href="http://dx.doi.org/10.1073/pnas.93.12.6025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+subtractive+hybridization%3A+A+method+for+generating+differentially+regulated+or+tissue-specific+cDNA+probes+and+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+subtractive+hybridization%3A+A+method+for+generating+differentially+regulated+or+tissue-specific+cDNA+probes+and+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001682-Anderson1" id="pone.0001682-Anderson1"></a>Anderson JA, Churchill GA, Autrique JE, Tanksley SD, Sorrells ME (1993) Optimizing Parental Selection for Genetic-Linkage Maps. Genome  36: 181–186.  <ul class="find" data-citedArticleID="1063274" data-doi="10.1139/g93-024"><li><a href="http://dx.doi.org/10.1139/g93-024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimizing+Parental+Selection+for+Genetic-Linkage+Maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimizing+Parental+Selection+for+Genetic-Linkage+Maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001682-Taberlet1" id="pone.0001682-Taberlet1"></a>Taberlet P, Gielly L, Pautou G, Bouvet J (1991) Universal Primers for Amplification of 3 Noncoding Regions of Chloroplast DNA. Plant Molecular Biology  17: 1105–1109.  <ul class="find" data-citedArticleID="1063338" data-doi="10.1007/bf00037152"><li><a href="http://dx.doi.org/10.1007/bf00037152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Universal+Primers+for+Amplification+of+3+Noncoding+Regions+of+Chloroplast+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Universal+Primers+for+Amplification+of+3+Noncoding+Regions+of+Chloroplast+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001682-Schneider1" id="pone.0001682-Schneider1"></a>Schneider H, Ranker T, Russell SJ, Cranfill R, Geiger JMO, et al.  (2005) Origin of the endemic fern genus Diellia coincides with the renewal of Hawaiian terrestrial life in the Miocene. Proceedings of the Royal Society B-Biological Sciences  272: 455–460.  <ul class="find" data-citedArticleID="1063330" data-doi="10.1098/rspb.2004.2965"><li><a href="http://dx.doi.org/10.1098/rspb.2004.2965" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+the+endemic+fern+genus+Diellia+coincides+with+the+renewal+of+Hawaiian+terrestrial+life+in+the+Miocene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+the+endemic+fern+genus+Diellia+coincides+with+the+renewal+of+Hawaiian+terrestrial+life+in+the+Miocene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001682-Ishikawa1" id="pone.0001682-Ishikawa1"></a>Ishikawa H, Watano Y, Kano K, Ito M, Kurita S (2002) Development of primer sets for PCR amplification of the PgiC gene in ferns. Journal of Plant Research  115: 65–70.  <ul class="find" data-citedArticleID="1063298" data-doi="10.1007/s102650200010"><li><a href="http://dx.doi.org/10.1007/s102650200010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+primer+sets+for+PCR+amplification+of+the+PgiC+gene+in+ferns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+primer+sets+for+PCR+amplification+of+the+PgiC+gene+in+ferns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001682-Pedersen1" id="pone.0001682-Pedersen1"></a>Pedersen N, Newton AE (2007)  Phylogenetic and morphological studies within the Ptychomniales, with emphasis on the evolution of dwarf males. In: Newton AE, Tangney R, DeLuna E, editors. Pleurocarpous Mosses: Systematics and Evolution. Boca Raton, FL: CRC Press (Taylor &amp; Francis Group). pp. 367–392.  <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001682-Altschul1" id="pone.0001682-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang JH, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Research  25: 3389–3402.  <ul class="find" data-citedArticleID="1063272" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001682-Hammer1" id="pone.0001682-Hammer1"></a>Hammer O, Harper D, Ryan P (2001) PAST: Palaeontological Statistics Software package for education and data analysis. Palaeontologia Electronica  4: 9.  <ul class="find" data-citedArticleID="1063292"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PAST%3A+Palaeontological+Statistics+Software+package+for+education+and+data+analysis.&amp;auth=&amp;atitle=PAST%3A+Palaeontological+Statistics+Software+package+for+education+and+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAST%3A+Palaeontological+Statistics+Software+package+for+education+and+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAST%3A+Palaeontological+Statistics+Software+package+for+education+and+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001682-Maddison1" id="pone.0001682-Maddison1"></a>Maddison WP, Maddison DR (1989) Interactive Analysis of Phylogeny and Character Evolution Using the Computer-Program Macclade. Folia Primatologica  53: 190–202.  <ul class="find" data-citedArticleID="1063316" data-doi="10.1159/000156416"><li><a href="http://dx.doi.org/10.1159/000156416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactive+Analysis+of+Phylogeny+and+Character+Evolution+Using+the+Computer-Program+Macclade." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactive+Analysis+of+Phylogeny+and+Character+Evolution+Using+the+Computer-Program+Macclade.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001682-Huelsenbeck1" id="pone.0001682-Huelsenbeck1"></a>Huelsenbeck JP, Ronquist F (2001) MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics  17: 754–755.  <ul class="find" data-citedArticleID="1063296" data-doi="10.1093/bioinformatics/17.8.754"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.8.754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MRBAYES%3A+Bayesian+inference+of+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001682-Swofford1" id="pone.0001682-Swofford1"></a>Swofford DL (1993) Paup-a Computer-Program for Phylogenetic Inference Using Maximum Parsimony. Journal of General Physiology  102: A9–A9.  <ul class="find" data-citedArticleID="1063336"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Paup-a+Computer-Program+for+Phylogenetic+Inference+Using+Maximum+Parsimony.&amp;auth=&amp;atitle=Paup-a+Computer-Program+for+Phylogenetic+Inference+Using+Maximum+Parsimony." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paup-a+Computer-Program+for+Phylogenetic+Inference+Using+Maximum+Parsimony." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paup-a+Computer-Program+for+Phylogenetic+Inference+Using+Maximum+Parsimony.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001682-Bryant1" id="pone.0001682-Bryant1"></a>Bryant D, Moulton V (2002) NeighborNet: An agglomerative method for the construction of planar phylogenetic networks. In: Algorithms in Bioniformatics: Second International Workshop, WABI 2002, Rome, Italy, September 17–21, 2002. Proceedings. Berlin: Springer. pp. 375–391.  <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0001682-Rambaut1" id="pone.0001682-Rambaut1"></a>Rambaut A, Drummond A (2004) TRACER, version 1.2. Oxford, UK: University of Oxford.   <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0001682-Vandepeer1" id="pone.0001682-Vandepeer1"></a>Vandepeer Y, Dewachter R (1993) Treecon-a Software Package for the Construction and Drawing of Evolutionary Trees. Computer Applications in the Biosciences  9: 177–182.  <ul class="find" data-citedArticleID="1063342" data-doi="10.1093/bioinformatics/9.2.177"><li><a href="http://dx.doi.org/10.1093/bioinformatics/9.2.177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Treecon-a+Software+Package+for+the+Construction+and+Drawing+of+Evolutionary+Trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Treecon-a+Software+Package+for+the+Construction+and+Drawing+of+Evolutionary+Trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001682-Vogel1" id="pone.0001682-Vogel1"></a>Vogel JC (1995) Multiple origins of polyploids in European Asplenium (Pteridophyta). University of Cambridge.   Ph.D. thesis. <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001682-Vogel2" id="pone.0001682-Vogel2"></a>Vogel JC, Russell SJ, Rumsey FJ, Barrett JA, Gibby M (1998) On hybrid formation in the rock fern Asplenium x alternifolium (Aspleniaceae, Pteridophyta). Botanica Acta  111: 241–246.  <ul class="find" data-citedArticleID="1063346" data-doi="10.1111/j.1438-8677.1998.tb00702.x"><li><a href="http://dx.doi.org/10.1111/j.1438-8677.1998.tb00702.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+hybrid+formation+in+the+rock+fern+Asplenium+x+alternifolium+%28Aspleniaceae%2C+Pteridophyta%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+hybrid+formation+in+the+rock+fern+Asplenium+x+alternifolium+%28Aspleniaceae%2C+Pteridophyta%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001682-Pelser1" id="pone.0001682-Pelser1"></a>Pelser PB, Gravendeel B, van der Meijden R (2003) Phylogeny reconstruction in the gap between too little and too much divergence: the closest relatives of Senecio jacobaea (Asteraceae) according to DNA sequences and AFLPs. Molecular Phylogenetics and Evolution  29: 613–628.  <ul class="find" data-citedArticleID="1063324" data-doi="10.1016/s1055-7903(03)00139-8"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00139-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeny+reconstruction+in+the+gap+between+too+little+and+too+much+divergence%3A+the+closest+relatives+of+Senecio+jacobaea+%28Asteraceae%29+according+to+DNA+sequences+and+AFLPs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeny+reconstruction+in+the+gap+between+too+little+and+too+much+divergence%3A+the+closest+relatives+of+Senecio+jacobaea+%28Asteraceae%29+according+to+DNA+sequences+and+AFLPs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001682-Koopman1" id="pone.0001682-Koopman1"></a>Koopman WJM (2005) Phylogenetic signal in AFLP data sets. Systematic Biology  54: 197–217.  <ul class="find" data-citedArticleID="1063306"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phylogenetic+signal+in+AFLP+data+sets.&amp;auth=&amp;atitle=Phylogenetic+signal+in+AFLP+data+sets." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+signal+in+AFLP+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+signal+in+AFLP+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001682-Simmons1" id="pone.0001682-Simmons1"></a>Simmons MP, Zhang LB, Webb CT, Muller K (2007) A penalty of using anonymous dominant markers (AFLPs, ISSRs, and RAMS) for phylogenetic inference. Molecular Phylogenetics and Evolution  42: 528–542.  <ul class="find" data-citedArticleID="1063332" data-doi="10.1016/j.ympev.2006.08.008"><li><a href="http://dx.doi.org/10.1016/j.ympev.2006.08.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+penalty+of+using+anonymous+dominant+markers+%28AFLPs%2C+ISSRs%2C+and+RAMS%29+for+phylogenetic+inference." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+penalty+of+using+anonymous+dominant+markers+%28AFLPs%2C+ISSRs%2C+and+RAMS%29+for+phylogenetic+inference.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001682-Nei1" id="pone.0001682-Nei1"></a>Nei M, Li WH (1979) Mathematical Model for Studying Genetic Variation in Terms of Restriction Endonucleases. PNAS  76: 5269–5273.  <ul class="find" data-citedArticleID="1063318" data-doi="10.1073/pnas.76.10.5269"><li><a href="http://dx.doi.org/10.1073/pnas.76.10.5269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mathematical+Model+for+Studying+Genetic+Variation+in+Terms+of+Restriction+Endonucleases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mathematical+Model+for+Studying+Genetic+Variation+in+Terms+of+Restriction+Endonucleases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001682-Archibald1" id="pone.0001682-Archibald1"></a>Archibald JK, Mort ME, Crawford DJ, Santos-Guerra A (2006) Evolutionary relationships within recently radiated taxa: comments on methodology and analysis of inter-simple sequence repeat data and other hypervariable, dominant markers. Taxon  55: 747–756.  <ul class="find" data-citedArticleID="1063276" data-doi="10.2307/25065648"><li><a href="http://dx.doi.org/10.2307/25065648" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+relationships+within+recently+radiated+taxa%3A+comments+on+methodology+and+analysis+of+inter-simple+sequence+repeat+data+and+other+hypervariable%2C+dominant+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+relationships+within+recently+radiated+taxa%3A+comments+on+methodology+and+analysis+of+inter-simple+sequence+repeat+data+and+other+hypervariable%2C+dominant+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001682-Laroche1" id="pone.0001682-Laroche1"></a>Laroche J, Bousquet J (1999) Evolution of the mitochondrial rps3 intron in perennial and annual angiosperms and homology to nad5 intron 1. Molecular Biology and Evolution  16: 441–452.  <ul class="find" data-citedArticleID="1063310" data-doi="10.1093/oxfordjournals.molbev.a026126"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+mitochondrial+rps3+intron+in+perennial+and+annual+angiosperms+and+homology+to+nad5+intron+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+mitochondrial+rps3+intron+in+perennial+and+annual+angiosperms+and+homology+to+nad5+intron+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001682-Olsen1" id="pone.0001682-Olsen1"></a>Olsen KM (2004) SNPs, SSRs and inferences on cassava's origin. Plant Molecular Biology  56: 517–526.  <ul class="find" data-citedArticleID="1063320" data-doi="10.1007/s11103-004-5043-9"><li><a href="http://dx.doi.org/10.1007/s11103-004-5043-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNPs%2C+SSRs+and+inferences+on+cassava%27s+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNPs%2C+SSRs+and+inferences+on+cassava%27s+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001682-Stefanovic1" id="pone.0001682-Stefanovic1"></a>Stefanovic S, Olmstead RG (2004) Testing the phylogenetic position of a parasitic plant (Cuscuta, Convolvulaceae, Asteridae): Bayesian inference and the parametric bootstrap on data drawn from three genomes. Systematic Biology  53: 384–399.  <ul class="find" data-citedArticleID="1063334"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Testing+the+phylogenetic+position+of+a+parasitic+plant+%28Cuscuta%2C+Convolvulaceae%2C+Asteridae%29%3A+Bayesian+inference+and+the+parametric+bootstrap+on+data+drawn+from+three+genomes.&amp;auth=&amp;atitle=Testing+the+phylogenetic+position+of+a+parasitic+plant+%28Cuscuta%2C+Convolvulaceae%2C+Asteridae%29%3A+Bayesian+inference+and+the+parametric+bootstrap+on+data+drawn+from+three+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+the+phylogenetic+position+of+a+parasitic+plant+%28Cuscuta%2C+Convolvulaceae%2C+Asteridae%29%3A+Bayesian+inference+and+the+parametric+bootstrap+on+data+drawn+from+three+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+the+phylogenetic+position+of+a+parasitic+plant+%28Cuscuta%2C+Convolvulaceae%2C+Asteridae%29%3A+Bayesian+inference+and+the+parametric+bootstrap+on+data+drawn+from+three+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001682-Howarth1" id="pone.0001682-Howarth1"></a>Howarth DG, Baum DA (2005) Genealogical evidence of homoploid hybrid speciation in an adaptive radiation of Scaevola (Goodeniaceae) in the Hawaiian Islands. Evolution  59: 948–961.  <ul class="find" data-citedArticleID="1063294" data-doi="10.1111/j.0014-3820.2005.tb01034.x"><li><a href="http://dx.doi.org/10.1111/j.0014-3820.2005.tb01034.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genealogical+evidence+of+homoploid+hybrid+speciation+in+an+adaptive+radiation+of+Scaevola+%28Goodeniaceae%29+in+the+Hawaiian+Islands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genealogical+evidence+of+homoploid+hybrid+speciation+in+an+adaptive+radiation+of+Scaevola+%28Goodeniaceae%29+in+the+Hawaiian+Islands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001682-Kress1" id="pone.0001682-Kress1"></a>Kress WJ, Wurdack KJ, Zimmer EA, Weigt LA, Janzen DH (2005) Use of DNA barcodes to identify flowering plants. Proceedings of the National Academy of Sciences of the United States of America  102: 8369–8374.  <ul class="find" data-citedArticleID="1063308" data-doi="10.1073/pnas.0503123102"><li><a href="http://dx.doi.org/10.1073/pnas.0503123102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+DNA+barcodes+to+identify+flowering+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+DNA+barcodes+to+identify+flowering+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001682-KoehlerSantos1" id="pone.0001682-KoehlerSantos1"></a>Koehler-Santos P, Lorenz-Lemke AP, Muschner VC, Bonatto SL, Salzano FM, et al.  (2006) Molecular genetic variation in Passiflora alata (Passifloraceae), an invasive species in southern Brazil. Biological Journal of the Linnean Society  88: 611–630.  <ul class="find" data-citedArticleID="1063304" data-doi="10.1111/j.1095-8312.2006.00647.x"><li><a href="http://dx.doi.org/10.1111/j.1095-8312.2006.00647.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+genetic+variation+in+Passiflora+alata+%28Passifloraceae%29%2C+an+invasive+species+in+southern+Brazil." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+genetic+variation+in+Passiflora+alata+%28Passifloraceae%29%2C+an+invasive+species+in+southern+Brazil.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001682-Dane1" id="pone.0001682-Dane1"></a>Dane F, Liu JR, Zhang CK (2007) Phylogeography of the bitter apple, Citrullus colocynthis. Genetic Resources and Crop Evolution  54: 327–336.  <ul class="find" data-citedArticleID="1063284" data-doi="10.1007/s10722-005-4897-2"><li><a href="http://dx.doi.org/10.1007/s10722-005-4897-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+of+the+bitter+apple%2C+Citrullus+colocynthis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+of+the+bitter+apple%2C+Citrullus+colocynthis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.XML" value="137622"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.PDF" value="235791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t001.PNG_L" value="259519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t001.PNG_M" value="153554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t001.PNG_S" value="15394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t001.TIF" value="1140500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t001.PNG_I" value="46319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t002.PNG_L" value="225647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t002.PNG_M" value="119426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t002.PNG_S" value="12998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t002.TIF" value="996748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t002.PNG_I" value="35625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t003.PNG_L" value="87393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t003.PNG_M" value="59753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t003.PNG_S" value="8546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t003.TIF" value="435436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t003.PNG_I" value="18855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t004.PNG_L" value="295695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t004.PNG_M" value="120229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t004.PNG_S" value="24585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t004.TIF" value="1456972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t004.PNG_I" value="193715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t005.PNG_L" value="147042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t005.PNG_M" value="85683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t005.PNG_S" value="11225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t005.TIF" value="689274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.t005.PNG_I" value="26174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g001.PNG_L" value="144124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g001.PNG_M" value="19849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g001.PNG_S" value="4695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g001.TIF" value="283898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g001.PNG_I" value="7193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g002.PNG_L" value="134032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g002.PNG_M" value="37259"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g002.PNG_S" value="3205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g002.TIF" value="211640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g002.PNG_I" value="14878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g003.PNG_L" value="344582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g003.PNG_M" value="73057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g003.PNG_S" value="8862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g003.TIF" value="759766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g003.PNG_I" value="32566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g004.PNG_L" value="103197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g004.PNG_M" value="25383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g004.PNG_S" value="2855"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g004.TIF" value="175030"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.g004.PNG_I" value="10199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001682.s001.XLS" value="500736"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001682&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001682" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001682&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001682&volume=&issue=&title=Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms&author_name=Karen%20E.%20James%2C%20Harald%20Schneider%2C%20Stephen%20W.%20Ansell%2C%20Margaret%20Evers%2C%20Lavinia%20Robba%2C%20Grzegorz%20Uszynski%2C%20Niklas%20Pedersen%2C%20Angela%20E.%20Newton%2C%20Stephen%20J.%20Russell%2C%20Johannes%20C.%20Vogel%2C%20Andrzej%20Kilian&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001682" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682&amp;t=Diversity%20Arrays%20Technology%20(DArT)%20for%20Pan-Genomic%20Evolutionary%20Studies%20of%20Non-Model%20Organisms" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682&title=Diversity%20Arrays%20Technology%20(DArT)%20for%20Pan-Genomic%20Evolutionary%20Studies%20of%20Non-Model%20Organisms&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682&amp;title=Diversity%20Arrays%20Technology%20(DArT)%20for%20Pan-Genomic%20Evolutionary%20Studies%20of%20Non-Model%20Organisms" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001682&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Diversity Arrays Technology (DArT) for Pan-Genomic Evolutionary Studies of Non-Model Organisms';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001682';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Diversity%20Arrays%20Technology%20(DArT)%20for%20Pan-Genomic%20Evolutionary%20Studies%20of%20Non-Model%20Organisms http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001682" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001682" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="26190"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="26190"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26190"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="26190"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogeography%22" title="Search for articles in the subject area:'Phylogeography'"><div class="flagText">Phylogeography</div></a>
              <div data-categoryid="32107" data-articleid="26190"
                   data-categoryname="Phylogeography"
                   class="flagImage" title="Flag 'Phylogeography' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+evolution%22" title="Search for articles in the subject area:'Plant evolution'"><div class="flagText">Plant evolution</div></a>
              <div data-categoryid="18531" data-articleid="26190"
                   data-categoryname="Plant evolution"
                   class="flagImage" title="Flag 'Plant evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="26190"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="26190"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5629'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9419'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5966&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=3285">Blog post</a><br>Posted by karenj</p>
      <p><a href="/annotation/listThread.action?root=19877">Referee comments: Referee 3</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=3099">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
