

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001803"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001803" />

    <meta name="citation_title" content="Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells"/>
    <meta itemprop="name" content="Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells"/>

      <meta name="citation_author" content="Takeshi Nagashima"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Takahiro Suzuki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
            <meta name="citation_author_institution" content="Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan"/>
      <meta name="citation_author" content="Shinji Kondo"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Yoko Kuroki"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Kaoru Takahashi"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Kaori Ide"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Noriko Yumoto"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Aki Hasegawa"/>
            <meta name="citation_author_institution" content="Advanced Genome Information Technology Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Tetsuro Toyoda"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Toshio Kojima"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Akihiko Konagaya"/>
            <meta name="citation_author_institution" content="Advanced Genome Information Technology Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Harukazu Suzuki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Yoshihide Hayashizaki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
            <meta name="citation_author_institution" content="Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan"/>
            <meta name="citation_author_institution" content="Genome Science Laboratory, Discovery and Research Institute, RIKEN Wako Main Campus, Saitama, Japan"/>
            <meta name="citation_author_institution" content="Functional RNA Research Program, RIKEN Frontier Research System, Saitama, Japan"/>
      <meta name="citation_author" content="Yoshiyuki Sakaki"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>
      <meta name="citation_author" content="Mariko Hatakeyama"/>
            <meta name="citation_author_institution" content="Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan"/>

    <meta name="citation_date" content="2008/3/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001803.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1803"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Distinct characteristics of heregulin signals mediated by HER3 or HER4.; citation_author=FU Weiss; citation_author=C Wallasch; citation_author=M Campiglio; citation_author=W Issing; citation_author=A Ullrich; citation_journal_title=J Cell Physiol; citation_volume=173; citation_number=1; citation_pages=187-195; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Heregulin stimulates mitogenesis and phosphatidylinositol 3-kinase in mouse fibroblasts transfected with erbB2/neu and erbB3.; citation_author=KL Carraway; citation_author=SP Soltoff; citation_author=AJ Diamonti; citation_author=LC Cantley; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=2; citation_pages=7111-7116; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Heregulin beta1-activated phosphatidylinositol 3-kinase enhances aggregation of MCF-7 breast cancer cells independent of extracellular signal-regulated kinase; citation_author=M Tan; citation_author=R Grijalva; citation_author=D Yu; citation_journal_title=Cancer Res; citation_volume=59; citation_number=3; citation_pages=1620-1625; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Quantitative transcriptional control of ErbB receptor signaling undergoes graded to biphasic response for cell differentiation.; citation_author=T Nagashima; citation_author=H Shimodaira; citation_author=K Ide; citation_author=T Nakakuki; citation_author=Y Tani; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=4; citation_pages=4045-4056; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Untangling the ErbB signalling network.; citation_author=Y Yarden; citation_author=MX Sliwkowski; citation_journal_title=Nat Rev Mol Cell Biol.; citation_volume=2; citation_number=5; citation_pages=127-137; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Activation of the unliganded estrogen receptor by EGF involves the MAP kinase pathway and direct phosphorylation.; citation_author=G Bunone; citation_author=PA Briand; citation_author=RJ Miksicek; citation_author=D Picard; citation_journal_title=EMBO J; citation_volume=15; citation_number=6; citation_pages=2174-2183; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Phosphatidylinositol 3-kinase/AKT-mediated activation of estrogen receptor alpha: a new model for anti-estrogen resistance.; citation_author=RA Campbell; citation_author=P Bhat-Nakshatri; citation_author=NM Patel; citation_author=D Constantinidou; citation_author=S Ali; citation_author=H Nakshatri; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=7; citation_pages=9817-9824; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A role for Akt in mediating the estrogenic functions of epidermal growth factor and insulin-like growth factor I.; citation_author=MB Martin; citation_author=TF Franke; citation_author=GE Stoica; citation_author=P Chambon; citation_author=BS Katzenellenbogen; citation_journal_title=Endocrinology; citation_volume=141; citation_number=8; citation_pages=4503-4511; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Estrogen receptor interaction with estrogen response elements.; citation_author=CM Klinge; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=9; citation_pages=2905-2919; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Transcriptional repression of oestrogen receptor by metastasis-associated protein 1 corepressor.; citation_author=A Mazumdar; citation_author=RA Wang; citation_author=SK Mishra; citation_author=L Adam; citation_author=R Bagheri-Yarmand; citation_journal_title=Nat Cell Biol; citation_volume=3; citation_number=10; citation_pages=30-37; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Selective estrogen receptor modulators: discrimination of agonistic versus antagonistic activities by gene expression profiling in breast cancer cells.; citation_author=J Frasor; citation_author=F Stossi; citation_author=JM Danes; citation_author=B Komm; citation_author=CR Lyttle; citation_journal_title=Cancer Res; citation_volume=64; citation_number=11; citation_pages=1522-1533; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Profiling of estrogen up- and down-regulated gene expression in human breast cancer cells: insights into gene networks and pathways underlying estrogenic control of proliferation and cell phenotype.; citation_author=J Frasor; citation_author=JM Danes; citation_author=B Komm; citation_author=KC Chang; citation_author=CR Lyttle; citation_journal_title=Endocrinology; citation_volume=144; citation_number=12; citation_pages=4562-4574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The estrogen receptor binds tightly to its responsive element as a ligand-induced homodimer.; citation_author=V Kumar; citation_author=P Chambon; citation_journal_title=Cell; citation_volume=55; citation_number=13; citation_pages=145-156; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=pp90rsk1 regulates estrogen receptor-mediated transcription through phosphorylation of Ser-167. Mol Cell Biol.; citation_author=PB Joel; citation_author=J Smith; citation_author=TW Sturgill; citation_author=TL Fisher; citation_author=J Blenis; citation_volume=18; citation_number=14; citation_pages=1978-1984; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=HER-2 tyrosine kinase pathway targets estrogen receptor and promotes hormone-independent growth in human breast cancer cells.; citation_author=RJ Pietras; citation_author=J Arboleda; citation_author=DM Reese; citation_author=N Wongvipat; citation_author=MD Pegram; citation_journal_title=Oncogene; citation_volume=10; citation_number=15; citation_pages=2435-2446; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Functions of Site-Specific Histone Acetylation and Deacetylation.; citation_author=MD Shahbazian; citation_author=M Grunstein; citation_journal_title=Annu Rev Biochem; citation_volume=76; citation_number=16; citation_pages=75-100; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Histone acetylation is associated with differential gene expression in the rapid and robust memory CD8(+) T-cell response.; citation_author=M Fann; citation_author=JM Godlove; citation_author=M Catalfamo; citation_author=WH Wood; citation_author=FJ Chrest; citation_journal_title=Blood; citation_volume=108; citation_number=17; citation_pages=3363-3370; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global identification of human transcribed sequences with genome tiling arrays.; citation_author=P Bertone; citation_author=V Stolc; citation_author=TE Royce; citation_author=JS Rozowsky; citation_author=AE Urban; citation_journal_title=Science; citation_volume=306; citation_number=18; citation_pages=2242-2246; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=OmicBrowse: a browser of multidimensional omics annotations.; citation_author=T Toyoda; citation_author=Y Mochizuki; citation_author=K Player; citation_author=N Heida; citation_author=N Kobayashi; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=19; citation_pages=524-526; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The UCSC Genome Browser Database.; citation_author=D Karolchik; citation_author=R Baertsch; citation_author=M Diekhans; citation_author=TS Furey; citation_author=A Hinrichs; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=20; citation_pages=51-54; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=From genomics to chemical genomics: new developments in KEGG.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=M Hattori; citation_author=KF Aoki-Kinoshita; citation_author=M Itoh; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=21; citation_pages=D354-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modulation of transcriptional activation by ligand-dependent phosphorylation of the human oestrogen receptor A/B region.; citation_author=S Ali; citation_author=D Metzger; citation_author=JM Bornert; citation_author=P Chambon; citation_journal_title=EMBO J; citation_volume=12; citation_number=22; citation_pages=1153-1160; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A functional serine 118 phosphorylation site in estrogen receptor-alpha is required for down-regulation of gene expression by 17beta-estradiol and 4-hydroxytamoxifen.; citation_author=J Cheng; citation_author=C Zhang; citation_author=DJ Shapiro; citation_journal_title=Endocrinology; citation_volume=148; citation_number=23; citation_pages=4634-4641; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Estrogen receptors: orchestrators of pleiotropic cellular responses.; citation_author=JG Moggs; citation_author=G Orphanides; citation_journal_title=EMBO Rep; citation_volume=2; citation_number=24; citation_pages=775-781; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of estrogen action.; citation_author=S Nilsson; citation_author=S Makela; citation_author=E Treuter; citation_author=M Tujague; citation_author=J Thomsen; citation_journal_title=Physiol Rev; citation_volume=81; citation_number=25; citation_pages=1535-1565; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Synergistic coupling of histone H3 phosphorylation and acetylation in response to epidermal growth factor stimulation.; citation_author=P Cheung; citation_author=KG Tanner; citation_author=WL Cheung; citation_author=P Sassone-Corsi; citation_author=JM Denu; citation_journal_title=Mol Cell; citation_volume=5; citation_number=26; citation_pages=905-1005; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=High tumor incidence and activation of the PI3K/AKT pathway in transgenic mice define AIB1 as an oncogene.; citation_author=MI Torres-Arzayus; citation_author=J Font de Mora; citation_author=J Yuan; citation_author=F Vazquez; citation_author=R Bronson; citation_journal_title=Cancer Cell; citation_volume=6; citation_number=27; citation_pages=263-274; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dual roles for the phosphatase PPM1D in regulating progesterone receptor function.; citation_author=DA Proia; citation_author=BW Nannenga; citation_author=LA Donehower; citation_author=NL Weigel; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=28; citation_pages=7089-7101; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The p44S10 locus, encoding a subunit of the proteasome regulatory particle, is amplified during progression of cutaneous malignant melanoma.; citation_author=S Ren; citation_author=MJ Smith; citation_author=ID Louro; citation_author=P McKie-Bell; citation_author=MR Bani; citation_journal_title=Oncogene; citation_volume=19; citation_number=29; citation_pages=1419-1427; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Amplification of the BCAS2 gene at chromosome 1p13.3-21 in human primary breast cancer.; citation_author=N Maass; citation_author=F Rosel; citation_author=C Schem; citation_author=J Hitomi; citation_author=W Jonat; citation_journal_title=Cancer Lett; citation_volume=185; citation_number=30; citation_pages=219-223; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bidirectional interactions between the estrogen receptor and the cerbB-2 signaling pathways: heregulin inhibits estrogenic effects in breast cancer cells.; citation_author=TW Grunt; citation_author=M Saceda; citation_author=MB Martin; citation_author=R Lupu; citation_author=E Dittrich; citation_journal_title=Int J Cancer; citation_volume=63; citation_number=31; citation_pages=560-567; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Whole-genome cartography of estrogen receptor alpha binding sites.; citation_author=CY Lin; citation_author=VB Vega; citation_author=JS Thomsen; citation_author=T Zhang; citation_author=SL Kong; citation_author=M Xie; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=32; citation_pages=e87; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of the human estrogen receptor by mitogen-activated protein kinase and casein kinase II: consequence on DNA binding.; citation_author=SF Arnold; citation_author=JD Obourn; citation_author=H Jaffe; citation_author=AC Notides; citation_journal_title=J Steroid Biochem Mol Biol; citation_volume=55; citation_number=33; citation_pages=163-172; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Interaction of oestrogen receptor with the regulatory subunit of phosphatidylinositol-3-OH kinase.; citation_author=T Simoncini; citation_author=A Hafezi-Moghadam; citation_author=DP Brazil; citation_author=K Ley; citation_author=WW Chin; citation_journal_title=Nature; citation_volume=407; citation_number=34; citation_pages=538-541; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Estrogen action and cytoplasmic signaling pathways. Part II: the role of growth factors and phosphorylation in estrogen signaling.; citation_author=PH Driggers; citation_author=JH Segars; citation_journal_title=Trends Endocrinol Metab; citation_volume=13; citation_number=35; citation_pages=422-427; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Histone H3 acetylated at lysine 9 in promoter is associated with low nucleosome density in the vicinity of transcription start site in human cell.; citation_author=H Nishida; citation_author=T Suzuki; citation_author=S Kondo; citation_author=H Miura; citation_author=F Fujimura; citation_journal_title=Chromosome Res; citation_volume=14; citation_number=36; citation_pages=203-211; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic maps and comparative analysis of histone modifications in human and mouse.; citation_author=BE Bernstein; citation_author=M Kamal; citation_author=K Lindblad-Toh; citation_author=S Bekiranov; citation_author=DK Bailey; citation_journal_title=Cell; citation_volume=120; citation_number=37; citation_pages=169-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A RecA-mediated exon profiling method.; citation_author=Y Hasegawa; citation_author=S Fukuda; citation_author=K Shimokawa; citation_author=S Kondo; citation_author=N Maeda; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=38; citation_pages=e97; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Novel RNAs identified an in-depth analysis of the transcriptome of human chromosomes 21 and 22.; citation_author=D Kampa; citation_author=J Cheng; citation_author=P Kapranov; citation_author=M Yamanaka; citation_author=S Brubaker; citation_journal_title=Genome Res; citation_volume=14; citation_number=39; citation_pages=331-342; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Determination of genomic breakpoints in an epileptic patient using genotyping array.; citation_author=T Kojima; citation_author=W Mukai; citation_author=D Fuma; citation_author=Y Ueda; citation_author=M Okada; citation_journal_title=Biochem Biophys Res Commun; citation_volume=341; citation_number=40; citation_pages=792-796; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes.; citation_author=V Matys; citation_author=OV Kel-Margoulis; citation_author=E Fricke; citation_author=I Liebich; citation_author=S Land; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=41; citation_pages=D108-110; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Epidermal growth factor-induced signaling in breast cancer cells results in selective target gene activation by orphan nuclear receptor estrogen-related receptor alpha.; citation_author=JB Barry; citation_author=V Giguere; citation_journal_title=Cancer Res; citation_volume=65; citation_number=42; citation_pages=6120-6129; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells"/>
    <meta name="twitter:description" content="Heregulin &szlig;-1 (HRG) is an extracellular ligand that activates mitogen-activated protein kinase (MAPK) and phosphatidylinositol-3-OH kinase (PI3K)/Akt signaling pathways through ErbB receptors. MAPK and Akt have been shown to phosphorylate the estrogen receptor (ER) at Ser-118 and Ser-167, respectively, thereby mimicking the effects of estrogenic activity such as estrogen responsive element (ERE)-dependent transcription. In the current study, integrative analysis was performed using two tiling array platforms, comprising histone H3 lysine 9 (H3K9) acetylation and RNA mapping, together with array comparative genomic hybridization (CGH) analysis in an effort to identify HRG-regulated genes in ER-positive MCF-7 breast cancer cells. Through application of various threshold settings, 333 (326 up-regulated and 7 down-regulated) HRG-regulated genes were detected. Prediction of upstream transcription factors (TFs) and pathway analysis indicated that 21% of HRG-induced gene regulation may be controlled by the MAPK cascade, while only 0.6% of the gene expression is controlled by ERE. A comparison with previously reported estrogen (E2)-regulated gene expression data revealed that only 12 common genes were identified between the 333 HRG-regulated (3.6%) and 239 E2-regulated (5.0%) gene groups. However, with respect to enriched upstream TFs, 4 common TFs were identified in the 14 HRG-regulated (28.6%) and 13 E2-regulated (30.8%) gene groups. These results indicated that while E2 and HRG may induce common TFs, the regulatory mechanisms that govern HRG- and E2-induced gene expression differ."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001803.g006"/>

  <meta property="og:title" content="Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6201'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1401'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9060&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001803">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001803" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001803&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takeshi Nagashima, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takahiro Suzuki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan, 
                  Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shinji Kondo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoko Kuroki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kaoru Takahashi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kaori Ide, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Noriko Yumoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Aki Hasegawa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Advanced Genome Information Technology Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tetsuro Toyoda, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Toshio Kojima, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Akihiko Konagaya, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Advanced Genome Information Technology Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harukazu Suzuki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshihide Hayashizaki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genome Exploration Research Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan, 
                  Division of Genomic Information Resources, Supramolecular Biology, International Graduate School of Arts and Sciences, Yokohama City University, Yokohama, Japan, 
                  Genome Science Laboratory, Discovery and Research Institute, RIKEN Wako Main Campus, Saitama, Japan, 
                  Functional RNA Research Program, RIKEN Frontier Research System, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshiyuki Sakaki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mariko Hatakeyama
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:marikoh@gsc.riken.jp">marikoh@gsc.riken.jp</a></p>

                <p>Affiliation:
                  Computational and Experimental Systems Biology Group, RIKEN Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 19, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001803</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001803-g001" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001803-g002" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001803-t001" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001803-g003" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001803-g004" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001803-g005" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001803-g006" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001803">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001803" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Heregulin ß-1 (HRG) is an extracellular ligand that activates mitogen-activated protein kinase (MAPK) and phosphatidylinositol-3-OH kinase (PI3K)/Akt signaling pathways through ErbB receptors. MAPK and Akt have been shown to phosphorylate the estrogen receptor (ER) at Ser-118 and Ser-167, respectively, thereby mimicking the effects of estrogenic activity such as estrogen responsive element (ERE)-dependent transcription. In the current study, integrative analysis was performed using two tiling array platforms, comprising histone H3 lysine 9 (H3K9) acetylation and RNA mapping, together with array comparative genomic hybridization (CGH) analysis in an effort to identify HRG-regulated genes in ER-positive MCF-7 breast cancer cells. Through application of various threshold settings, 333 (326 up-regulated and 7 down-regulated) HRG-regulated genes were detected. Prediction of upstream transcription factors (TFs) and pathway analysis indicated that 21% of HRG-induced gene regulation may be controlled by the MAPK cascade, while only 0.6% of the gene expression is controlled by ERE. A comparison with previously reported estrogen (E2)-regulated gene expression data revealed that only 12 common genes were identified between the 333 HRG-regulated (3.6%) and 239 E2-regulated (5.0%) gene groups. However, with respect to enriched upstream TFs, 4 common TFs were identified in the 14 HRG-regulated (28.6%) and 13 E2-regulated (30.8%) gene groups. These results indicated that while E2 and HRG may induce common TFs, the regulatory mechanisms that govern HRG- and E2-induced gene expression differ.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nagashima T, Suzuki T, Kondo S, Kuroki Y, Takahashi K, et al.  (2008) Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells. PLoS ONE 3(3):
          e1803.
            doi:10.1371/journal.pone.0001803</p><p><strong>Editor: </strong>Dorraya El-Ashry, University of Michigan, United States of America</p><p><strong>Received:</strong> July 23, 2007; <strong>Accepted:</strong> February 13, 2008; <strong>Published:</strong> March 19, 2008</p><p><strong>Copyright:</strong> © 2008 Nagashima et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>All funding for this study was supported by current research expenses in RIKEN.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Heregulin ß-1 (HRG) is an extracellular ligand that binds to and activates ErbB3 and ErbB4 receptors in a process associated with the control of cell proliferation and differentiation <a href="#pone.0001803-Weiss1">[1]</a>–<a href="#pone.0001803-Tan1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Our previous study using MCF-7 breast cancer cells showed that HRG rapidly up-regulated hundreds of transcripts in a ligand dose-dependent manner, as determined by employing expression arrays using the Affymetrix U133A 2.0 chip that mounts protein-coding and non-protein coding regions of ca. 18,400 transcripts <a href="#pone.0001803-Nagashima1">[4]</a>. HRG-stimulated ErbB receptor is known to activate mitogen-activated protein kinase (MAPK) and phosphatidylinositol-3-OH kinase (PI3K)/Akt signaling pathways <a href="#pone.0001803-Yarden1">[5]</a>. Consequently, the transcriptional regulation induced by HRG was thought to be controlled by activation of the aforementioned signaling pathways and associated transcription factors (TFs). On the other hand, MAPK and Akt are also known to phosphorylate the estrogen receptor (ER) at Ser-118 and Ser-167, respectively, thereby mimicking the effects of estrogenic activity in ER-positive cells in the absence of estrogen (E2) <a href="#pone.0001803-Bunone1">[6]</a>–<a href="#pone.0001803-Martin1">[8]</a>. Activated ER is believed to bind directly to the estrogen responsive element (ERE) DNA sequence and subsequently induce ERE-dependent transcription <a href="#pone.0001803-Klinge1">[9]</a>. Given that MCF-7 is an ER-positive breast cancer cell line, HRG-induced transcription may involve both ERE- and non-ERE-dependent regulation.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In our earlier study using the Affymetrix U133A 2.0 chip, however, we found that only 6 transcripts among the 241 HRG-regulated transcripts (2.49%) were derived from genes that possessed the ERE sequence within a 2 kb upstream region of the transcription start site (TSS) <a href="#pone.0001803-Nagashima1">[4]</a>. Notwithstanding the fact that HRG-stimulated MCF-7 cells showed ERE-mediated suppression of histone acetylation and gene expression <a href="#pone.0001803-Mazumdar1">[10]</a>, the population of ERE-regulated transcripts in our analysis seemed to be relatively low. This suggested the possibility that use of the expression array platform may have failed to identify the complete range of transcripts associated with HRG-induced gene regulation. In the current study, integrative analysis of HRG-induced gene expression in MCF-7 cells was performed using two tiling array platforms comprising histone H3 lysine 9 (H3K9) acetylation (ChIP-on-chip) and RNA mapping. Array comparative genomic hybridization (CGH) analysis was also employed in an effort to identify gene expression unrelated to HRG treatment.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Integrated analysis of the expression data using various array platforms facilitated the identification of transcripts that were regulated in cells following HRG treatment. The data showed that 326 genes were up-regulated and 7 genes were down-regulated. Prediction of upstream TFs and mapping onto the biological pathways indicated that 21% (69 out of 333) of the genes associated with HRG-induced gene regulation may be controlled by TFs known to be activated by the MAPK cascade, while only 0.6% (2 out of 333) of the gene expression is controlled by ERE.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Comparison with previously reported data pertaining to E2-regulated genes <a href="#pone.0001803-Frasor1">[11]</a>, <a href="#pone.0001803-Frasor2">[12]</a> revealed that only 12 common genes were identified between the 333 HRG-regulated (3.6%) and 239 E2-regulated (5.0%) gene groups. However, with respect to upstream TFs that are more likely to be enriched, 4 common TFs were identified in the 14 HRG-induced (28.6%) and 13 E2-induced (30.8%) gene groups. These results indicated that while E2 and HRG may induce common TFs, the regulatory mechanisms that govern HRG- and E2-induced gene expression differ.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Stimulation by HRG induces phosphorylation of ER at Ser-118 and Ser-167 in MCF-7 cells</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Modes of ER activation have been categorized largely on the basis of steroid (E2)-dependent and steroid-independent mechanisms. E2 binds directly to ER thereby facilitating ER dimerization and subsequent high affinity binding to ERE (referred to as steroid-dependent ER activation) <a href="#pone.0001803-Kumar1">[13]</a>. Growth hormones such as epidermal growth factor (EGF) and insulin-like growth factor (IGF)-I have been shown to activate ER by phosphorylating ER at Ser-118 through activation of MAPK <a href="#pone.0001803-Bunone1">[6]</a>. On the other hand, Akt and pp90<sup>rsk1</sup> activated by EGF have been shown to phosphorylate ER at Ser-167 <a href="#pone.0001803-Campbell1">[7]</a>, <a href="#pone.0001803-Martin1">[8]</a>, <a href="#pone.0001803-Joel1">[14]</a>. These steroid-independent activations of ER are also capable of leading to ER- and ERE-mediated transcription events. HRG is a growth hormone ligand that strongly activates MAPK and PI3K/Akt through ErbB receptor activation <a href="#pone.0001803-Weiss1">[1]</a>–<a href="#pone.0001803-Nagashima1">[4]</a>. We first assessed an assay to confirm that HRG induces ER phosphorylation at Ser-118 and Ser-167 in MCF-7 cells in our culture condition. The results clearly indicated that HRG treatment immediately induced phosphorylation at both serine residues (<a href="#pone-0001803-g001">Figure 1</a>). These results are consistent with an earlier report showing HRG-dependent ER activation <a href="#pone.0001803-Pietras1">[15]</a>. On the other hand, E2 induced weak phosphorylation at Ser-118 and no phosphorylation at Ser-167. We assumed that ER is strongly activated by HRG under the assay conditions used in this study.</p>
<div class="figure" id="pone-0001803-g001"><div class="img"><a name="pone-0001803-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Estrogen receptor phosphorylation induced by HRG and E2.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Phosphorylation levels of estrogen receptor alpha at Ser-118 and Ser-167 treated with HRG (10 nM) or E2 (10 nM) were measured at the designated time period. Three-independent western blots were performed for each ligand. Representative figures are shown on the left. Levels of phosphorylation were quantified using a densitometer and are shown graphically on the right.</p>
<span>doi:10.1371/journal.pone.0001803.g001</span></div>

<h4>Comparison of histone acetylation and RNA mapping data with expression array data</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>HRG-induced gene expression data obtained from various array platforms including the tiling array data was then investigated. These data were submitted to Gene Expression Omnibus (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) and can be accessed through GEO series accession numbers GSE8579 (tiling arrays, histone H3K9 acetylation and RNA mapping) and GSE8471 (expression arrays) and GSE8508 (array CGH)</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Although chromatin modification by H3K9 acetylation has been linked to the regulation of gene expression, the precise correlation between the extent of histone acetylation and gene expression remains unclear <a href="#pone.0001803-Shahbazian1">[16]</a>, <a href="#pone.0001803-Fann1">[17]</a>. On the other hand, although use of RNA mapping on whole-genome tiling arrays seems to represent an unbiased approach <a href="#pone.0001803-Bertone1">[18]</a>, this method often yields many false-positive and false-negative values. Therefore, the current study employed an analysis that began with an evaluation of acetylation and RNA map data sets.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In an effort to determine whether the tiling array data accurately reflects the gene expression profile, HRG-mediated up- and down-regulated genes were initially identified from the histone acetylation and RNA map data (see <a href="#s4">Materials and Methods</a> section). To this end, 27,354 RefSeq transcripts were selected that correspond to 17,843 Entrez Gene IDs (hereafter, the 17,843 genes are referred to as the baseline set). To assess the validity of the gene selection, the derived gene list was compared with genes that were identified in expression arrays as a reference indicator (see <a href="#s4">Materials and Methods</a> for the selection).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>HRG-induced up- and down-regulation of genes was assessed by identifying signal differences between HRG-treated and untreated cells. The difference in acetylation signal <em>d<sub>A</sub></em> and that of the RNA map <em>d<sub>R</sub></em> were calculated as follows: <em>d<sub>A</sub></em> = <em>A<sub>hrg</sub>–A<sub>control</sub></em> and <em>d<sub>R</sub></em> = <em>R<sub>hrg</sub>–R<sub>control</sub></em>, where <em>A<sub>hrg</sub></em> and <em>R<sub>hrg</sub></em> represent acetylation and RNA map signals in HRG-treated cells, respectively, and <em>A<sub>control</sub></em> and <em>R<sub>control</sub></em> represent acetylation and RNA map signals in untreated cells, respectively. If <em>d<sub>A</sub></em> or <em>d<sub>R</sub></em> was greater than 0, the gene was regarded as being up-regulated. <a href="#pone-0001803-g002">Figure 2</a> shows the distribution of these values. The average value of <em>d<sub>A</sub></em> (2.7, black dotted line in <a href="#pone-0001803-g002">Figure 2A</a>) and <em>d<sub>R</sub></em> (0.67, black dotted line in <a href="#pone-0001803-g002">Figure 2B</a>) and the distribution peak were greater than 0, which implies that HRG treatment induced up-regulation of histone acetylation and RNA mapping. If gene expression changes (up- or down-regulation) can be identified by examining acetylation (or RNA map) signals, it is anticipated that this would be reflected in the form of a distinct spread of <em>d<sub>A</sub></em> (or <em>d<sub>R</sub></em>) values relating to up- and down-regulated genes. However, when independently comparing the signal distribution pertaining to the acetylation (or RNA map) data with the expression array results, no clear difference was observed (red and green dashed lines were not distinct in <a href="#pone-0001803-g002">Figure 2</a>). Thus, our analysis revealed that independent tiling array analysis may have failed to identify the complete range of transcripts associated with HRG-induced gene regulation.</p>
<div class="figure" id="pone-0001803-g002"><div class="img"><a name="pone-0001803-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of signal differences <em>d<sub>A</sub></em> and <em>d<sub>R</sub></em> between HRG-treated and untreated cells.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Upper and lower panel shows the distribution of acetylation and RNA map signal differences, <em>d<sub>A</sub></em> ( = <em>A<sub>hrg</sub>–A<sub>control</sub></em>) and <em>d<sub>R</sub></em> ( = <em>R<sub>hrg</sub>–R<sub>control</sub></em>) (A and B), respectively. Larger <em>d<sub>A</sub></em> (or <em>d<sub>R</sub></em>) values represent a higher level of acetylation (or RNA map) in HRG-treated cells compared with untreated cells. Red and green dashed lines show the distribution of these values in the expression array and represent up- and down-regulated genes, respectively. Colored dotted lines depicted along the vertical axis represent average values of each distribution.</p>
<span>doi:10.1371/journal.pone.0001803.g002</span></div><a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>In an effort to confirm this possibility, the overlapping rate of genes selected from each tiling array was then compared with the gene list derived from the expression array analysis. For the selection of up-regulated genes from the histone acetylation data, the criterion <em>A<sub>hrg</sub></em>−<em>A<sub>control</sub></em>≥<em>α</em> was adopted. If a gene exceeded threshold <em>α</em>, such a gene was regarded as being up-regulated. In the same way, <em>R<sub>hrg</sub></em>−<em>R<sub>control</sub></em>≥<em>β</em> was used for the RNA map data. <a href="#pone-0001803-t001">Table 1</a> shows the relative ratio of up-regulated genes that were found in each tiling array data and expression array-based gene list under various thresholds of <em>α</em> and <em>β</em>. In general, increased thresholds resulted in higher overlapping rates. However, even with such a tight threshold setting, more than 50% of the genes selected from the histone acetylation data were not covered in the gene list obtained from the expression array data. The RNA map data show a higher overlapping rate when the number of genes obtained is quite small (<em>β</em> = 4 gives only 29 overlapping genes).</p>
<div class="figure" id="pone-0001803-t001"><div class="img"><a name="pone-0001803-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Comparison of up-regulated genes obtained from each tiling array data with the expression array results.</span></strong></p><span>doi:10.1371/journal.pone.0001803.t001</span></div><a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>The threshold setting used to identify regulated genes from the above two integrated tiling array data sets was then examined. In this analysis, genes that only satisfied both <em>A<sub>hrg</sub></em>−<em>A<sub>control</sub></em>≥<em>α</em> and <em>R<sub>hrg</sub></em>−<em>R<sub>control</sub></em>≥<em>β</em> were compared with results of the expression arrays. <a href="#pone-0001803-g003">Figure 3</a> represents the percentage (A and B) and number (C and D) of genes that were identified as being up-regulated from the combined tiling array data and which were also found in the expression array results under various threshold (<em>α</em> and <em>β</em>) settings. As shown, increasing <em>α</em> (<a href="#pone-0001803-g003">Figure 3A</a>) and <em>β</em> (<a href="#pone-0001803-g003">Figure 3B</a>) yielded higher overlapping rates from the two tiling array data and expression array data sets. In contrast, increasing the threshold resulted in the identification of fewer genes (<a href="#pone-0001803-g003">Figure 3C and D</a>). <em>β</em> has a higher contribution to the precision rate and number of genes identified compared with <em>α</em>. For specific combinations of <em>α</em> and <em>β</em> (e.g. <em>α</em> = 4 and <em>β</em> = 4), 100% of the tiling array inferred that up-regulated genes were found in the expression array results. However, only 19 genes were identified in that setting. There is a trade-off between precision rate and the number of identified genes. Two parameters, <em>r</em><sub>min</sub> (minimum precision rate) and <em>N</em> (number of extracted genes), need to be determined in order to arrive at a reasonable threshold setting. Here, <em>r</em><sub>min</sub> and <em>N</em> were set to 70 and 300, respectively. Under these conditions, the largest <em>β</em> value was searched for and α was then determined since a larger threshold results in a higher precision rate. Finally, <em>α</em> = 4 and <em>β</em> = 2 was obtained and these values were used as thresholds. As a result, 326 and 7 genes were identified as being up- and down-regulated, respectively, following HRG treatment of cells (<a href="#pone.0001803.s004">Table S1</a>). With this setting, the up- and down-regulation of 72.4% of these transcripts (241 out of 333 genes) was confirmed with the expression array data.</p>
<div class="figure" id="pone-0001803-g003"><div class="img"><a name="pone-0001803-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of tiling array-inferred up-regulated genes and expression array results at various thresholds.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The proportion of genes that were identified as up-regulated and also found in the expression array-based gene list at various thresholds of <em>α</em> and <em>β</em> ((A) and (B)). (C) and (D) represent the number of identified genes.</p>
<span>doi:10.1371/journal.pone.0001803.g003</span></div>

<h4>Identification of HRG-induced genes from acetylation and RNA mapping data and confirmation using expression array data</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As described above, 326 genes up-regulated following HRG treatment were chosen from the tiling array-based H3K9 acetylation and RNA mapping data sets. Three-hundred and eighteen of the 326 genes (97.5%) have corresponding probe sets on the expression array. The remaining 8 transcripts identified by the tiling array were manually inspected using a web-based genome browser <a href="#pone.0001803-Toyoda1">[19]</a>. As a result, only one gene, <em>LOC400214</em>, was identified as a novel candidate (<a href="#pone.0001803.s001">Figure S1</a>), which was not mounted on an expression array chip. Although this gene was derived from RefSeq model mRNA (XM_375081), an elevated level of acetylation in the promoter region (2nd and 3rd lanes in <a href="#pone.0001803.s001">Figure S1</a>) and increased RNA map signals in exons (4th and 5th lanes in <a href="#pone.0001803.s001">Figure S1</a>) indicate that this transcript may also be induced following HRG treatment of MCF-7 cells.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Of the 7 down-regulated genes inferred by the tiling array data, genes known to be down-regulated following HRG treatment, including the ER gene (<em>ESR1</em>), were not detected in our dataset. This was due to the fact that although the <em>ESR1</em> gene was down-regulated as determined by the histone acetylation analysis (56.3 for untreated cells vs. 44.5 for HRG-treated cells), it was slightly up-regulated according to the RNA mapping analysis (8.67 for untreated cells vs. 8.93 for HRG-treated cells). As for the down-regulated genes in our dataset, one gene had no associated probe set in the Affymetrix U133 Plus 2.0 chip and 3 genes were removed due to weak expression intensities. Finally, although the down-regulation of all 3 genes (<em>ATAD4, C6orf139</em> and <em>CRADD</em>) was confirmed using results of the expression arrays, however, these genes do not possess regulatory binding sites for ER or ERE at the promoter region.</p>


<h4>Relationship between transcriptional activity and copy number</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Since MCF-7 is a cancer cell line, there was a concern that high genome copy numbers in the cells may result in strong gene expression in the absence of HRG that may potentially obscure any HRG-related effects. The relationship between gene expression and genome copy number was therefore investigated utilizing the array CGH data. Actively transcribed regions were screened from the whole genome in an unbiased fashion by searching for chromosomal regions with significant acetylation and RNA mapping signals in untreated cells. As a result, 2,163 regions were identified to be actively transcribed in the absence of exogenous stimuli. The average relative copy number in these regions and for the whole genome was 1.12 and 1.03, respectively.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Copy numbers for the 326 HRG-induced up-regulated genes were then examined in an effort to investigate genomic amplifications. The location of genes was obtained from the Entrez Gene database and compared to that of SNP markers where the copy number was calculated. Where a gene locus contained multiple SNP markers, an average copy number was assigned. If no marker was present, copy numbers from the nearest 5′ and 3′ end of a gene were used to calculate the corresponding gene. As a result, an average relative copy number of 1.02 was obtained for the 326 HRG-induced up-regulated genes (1.03 for the whole genome). Seven genes (<em>GEM</em>, <em>KLF10</em>, <em>MRPS23</em>, <em>MYC</em>, <em>NME1</em>, <em>PTRH2</em> and <em>TRIB1</em>) had copy numbers greater than 1.5 (<a href="#pone.0001803.s002">Figure S2</a>). However, this low frequency of high copy number in the selected genes can be considered to have negligible contribution to the overall gene expression. Therefore, it was concluded that HRG-induced up-regulation of genes detected by tiling arrays essentially reflects that activity associated with HRG treatment, rather than chromosomal amplification.</p>


<h4>Functional characteristics of HRG-regulated genes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In an effort to investigate the regulatory mechanism relating to the 326 up-regulated and 7 down-regulated genes associated with HRG treatment, the biological function of these genes was examined using two public data sources comprising the transcription factor binding site (TFBS) information in the UCSC genome browser database <a href="#pone.0001803-Karolchik1">[20]</a> and the KEGG pathway database <a href="#pone.0001803-Kanehisa1">[21]</a>.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>TFBSs for the 333 genes were predicted using the UCSC genome browser database in an effort to provide information on potential upstream regulatory elements that link biological pathways and HRG-regulated gene expression. Genomic coordinate of genes were obtained by mapping the mRNA sequences derived from the UCSC genome browser database. TFBSs located within a 2 kb upstream region from the TSS were then used to find candidate TFs. As a result, 754 potential transcriptional regulatory relationships were found for 62% (208 out of 333) of the HRG-regulated genes. The 208 genes possess predicted TFBSs for 117 TFs (<a href="#pone.0001803.s005">Table S2</a>). Rigorous statistical testing did not result in significant enrichment of the 117 TFs. However, TFs such as <em>CREB1</em>, <em>ATF2</em> and <em>EGR1</em>, which are known to be activated immediately following growth hormone stimulation, showed relatively small <em>p</em>-values (<em>p</em>&lt;0.05, before multiple test correction).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The predicted TFs were then assigned to signaling pathways using a public database to determine the contribution of specific biological pathways to gene transcription. As a result, the MAPK pathway was found to be the most significant pathway associated with 13 TFs (<a href="#pone-0001803-g004">Figure 4</a>; enrichment of this pathway was supported by Fisher's exact test, <em>p</em>&lt;10<sup>−4</sup>). Sixty-nine HRG-regulated genes, which account for 21% of the 333 genes, possess binding sites for 13 MAPK-regulated TFs. Of these, 5 TFs involved in the MAPK pathway possessed small <em>p</em>-values (<a href="#pone.0001803.s006">Table S3</a>).</p>
<div class="figure" id="pone-0001803-g004"><div class="img"><a name="pone-0001803-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>HRG-regulated TFs involved in the KEGG MAPK signaling pathway.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Thirteen TFs included in the 117 HRG-regulated TF list involved in the MAPK signaling pathway and are highlighted in red. Note that since <em>Elk-1</em> appears twice in the figure (downstream of classical MAP kinase pathway and JNK and p38 MAP kinase pathway), 14 boxes are highlighted.</p>
<span>doi:10.1371/journal.pone.0001803.g004</span></div>

<h4>Relationship with ER and signaling pathway</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>As shown in <a href="#pone-0001803-g001">Figure 1</a>, HRG treatment facilitated phosphorylation of ER at Ser-118 and Ser-167 and therefore seems to activate ER under the assay conditions used. Whereas E2 did not show Ser-167 phosphorylation of ER in our assay condition, both E2 and HRG are capable of phosphoylating ER at Ser-118 that is required for ER-dependent transcription <a href="#pone.0001803-Ali1">[22]</a>, <a href="#pone.0001803-Cheng1">[23]</a>. Therefore, if a portion of HRG-regulated transcription was mediated by ER in this study, some genes that are also regulated by E2 should be overlapped since E2 and HRG may utilize common ER or ERE-dependent transcription mechanism through Ser-118-mediated ER activation.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>A comparison of our gene list with two published gene lists obtained by expression array experiments using E2-stimulated MCF-7 cells <a href="#pone.0001803-Frasor1">[11]</a>, <a href="#pone.0001803-Frasor2">[12]</a> was employed to determine whether the 333 HRG-regulated and 239 E2-regulated gene groups contained common genes. As a result, it was found that only 12 genes were common to both gene groups, representing 5.0 and 3.6% of the E2- and HRG-regulated gene groups, respectively. (<a href="#pone.0001803.s004">Table S1</a>).</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>In an effort to identify upstream regulators common to both HRG- and E2-mediated gene expression, candidate TFs predicted from E2-regulated gene sets were analyzed. TFBS prediction yielded 731 potential transcriptional regulatory relationships that were comprised of 112 TFs and 170 genes for E2. Unexpectedly, of the 239 E2-stimulated genes, only 5 genes (<em>BHLHB2, BRCA1, DYRK2, FOS</em> and <em>GREB1</em>) possessed binding sites for ER or ERE at the promoter region (within 2 kb upstream from the TSS). This result indicated the potential involvement of transcriptional regulators other than ERE, such as AP-1 and Sp1, in E2-regulated gene expression <a href="#pone.0001803-Moggs1">[24]</a>, <a href="#pone.0001803-Nilsson1">[25]</a>. A comparison of the 117 HRG TFs and 112 E2 TFs identified a total of 111 TFs common to both TF groups. Furthermore, 12 TFs (10.7% of the 112 TFs) predicted from the E2-regulated genes are involved in the MAPK signaling pathway (<a href="#pone.0001803.s003">Figure S3</a>) and all of which were common to the 13 TFs associated with the HRG-regulated genes. Even when TFs enriched in HRG- and E2-regulated genes were investigated (<em>p</em>&lt;0.05, before multiple test correction), 4 TFs were still identified as being common to the 14 and 13 HRG- and E2-related TF groups, respectively. For example, <em>ARNT</em> was commonly predicted from 9 HRG-regulated and 7 E2-regulated genes. Interestingly, however, the expression level of <em>KLF6,</em> one of the target genes of <em>ARNT</em>, increased with HRG treatment and decreased with E2 treatment. This result indicates that TFs activated following E2 and HRG stimulation may yield opposite effects in terms of gene regulation. Additionally, our analysis revealed that HRG- and E2-regulated gene sets contain few common members. These overall analytical results suggest that E2 and HRG depend considerably on the MAPK pathway for transcriptional control, where each TF may be differentially regulated by other factors, thereby resulting in the induction of different sets of genes. However, as expected, the presence of enriched TFs in the MAPK signaling pathway, 5 (out of 14) TFs for HRG and 1 (out of 13) TFs for E2, suggests that HRG is more dependent on this pathway.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, integrative analysis of HRG-regulated gene expression in MCF-7 cells was performed using two types of tiling array data, comprising histone H3K9 acetylation and RNA mapping, and compared the results with those based on expression array and array CGH. Following the determination of various threshold settings, 333 transcripts that were regulated by HRG were finally identified. The up- and down-regulation of 72.4% of these transcripts (241 of the 333 genes) was confirmed using expression arrays.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>EGF, a growth hormone ligand for EGFR/ErbB1 receptor, is known to induce coupling of phosphorylation and acetylation of histone H3 that results in enhanced gene expression <a href="#pone.0001803-Cheung1">[26]</a>. Our current analysis with HRG also indicated that histone acetylation and gene up-regulation may be closely related.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our current results indicated that the majority of HRG-regulated genes were up-regulated (326 of the 333 genes). However, since MCF-7 is a cancer cell line, there was a concern that high genome copy numbers in the cells may result in strong gene expression in the absence of HRG that may potentially obscure any HGR-related effects. Genes such as <em>NCOA3</em>, <em>PPM1D</em>, <em>PSMD6</em> and <em>BCAS2</em> were detected with high expression and copy numbers in MCF-7 cells (<a href="#pone-0001803-g005">Figures 5</a> and <a href="#pone-0001803-g006">6</a>), and these genes have been associated with breast cancer progression <a href="#pone.0001803-TorresArzayus1">[27]</a>–<a href="#pone.0001803-Maass1">[30]</a>. However, our analysis revealed that the overall correlation between average copy number and the extent of gene expression was surprisingly weak.</p>
<div class="figure" id="pone-0001803-g005"><div class="img"><a name="pone-0001803-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Chromosome-wide plots for copy number distribution.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Four representative chromosomes (1, 3, 17 and 20) are shown in this figure. See <a href="#pone.0001803.s002">Figure S2</a> for the same plots for all chromosomes. Red and blue bars at the top and bottom in each plot designate actively transcribed and high copy number regions, respectively. Green triangles represent location of the <em>BCAS2</em> (chromosome 1), <em>PSMD6</em> (chromosome 3) , <em>PPM1D</em> (chromosome 17) and <em>NCOA3</em> (chromosome 20) gene loci, respectively.</p>
<span>doi:10.1371/journal.pone.0001803.g005</span></div><div class="figure" id="pone-0001803-g006"><div class="img"><a name="pone-0001803-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001803" data-uri="info:doi/10.1371/journal.pone.0001803.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001803.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001803.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001803.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001803.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>OmicBrowse display for the <em>BCAS2</em> gene.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>Experimental results (tiling arrays and array CGH) together with genome annotation are represented along the genomic coordinate using OmicBrowse, a web-based genome browser. The upper panel shows the copy number (1st lane) and <em>BCAS2</em> gene locus (2nd lane) in chromosome 1. The lower panel represents a detailed view of this locus with 5 lanes. Each lane represents 1) H3K9 acetylation signals in the control experiment, 2) RNA mapping signal in the control experiment, 3) Ensembl Gene with exon-intron structure, 4) human CAGE tag counts, and 5) a region of <em>BCAS2</em> derived from the NCBI Entrez Gene database. The height of the vertical bars in the 1st and 2nd lanes corresponds to the strength of the signals. Light green bars in the top 2 lanes represent regions with significant signal intensity. The upper panel shows that <em>BCAS2</em> is located in amplified regions. In the lower panel, a high level of acetylation around the TSS, significant and higher signal intensity of the RNA map at exons, and a high CAGE tag can be seen for <em>BCAS2</em>. These data support the view that <em>BCAS2</em> was expressed in the absence of exogenous stimuli.</p>
<span>doi:10.1371/journal.pone.0001803.g006</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The frequency of occurrence of ER or ERE-mediated transcription was investigated using integrated gene expression analysis. Surprisingly, whereas HRG treatment resulted in the phosphorylation of ER at Ser-118, which is thought to lead to ER activation, ER- or ERE-dependent transcription induced by HRG seemed to be quite minor (0.6%). To account for this observation we initially speculated that it was due to the short exposure time (2 hrs) of the cells to HRG, or perhaps to a lack of depletion of E2-like compounds from the culture medium, which are thought to down-regulate ER levels <a href="#pone.0001803-Pietras1">[15]</a>, <a href="#pone.0001803-Grunt1">[31]</a>. In an effort to address this issue, it was confirmed by western blot analysis that HRG could phosphorylate ER under the culture conditions used (<a href="#pone-0001803-g001">Figure 1</a>). As for the exposure time, given that 4–48 hrs of E2-stimulated gene regulation failed to show ERE at its upstream region, a complex regulatory mechanism for gene expression rather than the exposure time of cells to stimuli seems to be a major factor. In fact, recent analysis of ER-alpha binding on the whole genome revealed that 95% of ERE binding sites of targeted genes are located at intronic or distal locations (&gt;5 kb from 5′ and 3′ ends of the transcript) <a href="#pone.0001803-Lin1">[32]</a>, its regulation seems to be far more complex than that associated with MAPK-regulated transcription. Additionally, our current analytical results suggested the presence of distinct regulatory roles for TFs that yield opposite effects in terms of HRG- and E2-induced gene regulation. ER expression and activation is widely regulated by various signaling pathways and kinases <a href="#pone.0001803-Arnold1">[33]</a>–<a href="#pone.0001803-Driggers1">[35]</a>. The current study has provided some insight into the similarities and differences between complex regulatory mechanisms in growth hormone- and E2-induced gene expression in ER-positive breast cancer cells.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Cell culture</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The MCF-7 human breast cancer cell line was obtained from the American Type Culture Collection and maintained in DMEM medium supplemented with 10% fetal bovine serum. Prior to growth hormone treatment, cells were serum-starved for 16–24 hours. For the expression analysis, 10 nM HRG-β 176–246 (R&amp;D Systems) was added to the cells and incubated for 2 hrs. Cells untreated with growth hormone were used as the control.</p>


<h4>ER phosphorylation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For the ER phosphorylation assay, cells were incubated with HRG (10 nM) or E2 (10 nM) for up to 90 min. Cells were then rinsed with ice-cold PBS and lysed using the Bio-Plex Cell Lysis Kit (Bio-Rad Laboratories). The cell lysate was cleared by centrifugation, and the protein concentration of the supernatant was determined using DC protein assay reagent (Bio-Rad Laboratories). Equal amounts of protein were subjected to Western blot analysis. Electrophoretically resolved proteins were blotted and subsequently probed with anti-phospho-Ser-118 (Upstate Biotechnology, Inc.), anti-phospho-Ser-167 (Cell Signaling Technology, Inc.) and anti-ER (Upstate Biotechnology, Inc.) antibodies. Phosphorylated proteins were quantified using a densitometer.</p>


<h4>Expression array</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Expression array experiments were performed using the Affymetrix Human Genome U133 Plus 2.0 Chip (Affymetrix, Santa Clara, CA, USA) that encodes 54,000 probe sets (with 47,000 transcripts and variants that includes 38,500 well-characterized genes) for integrative expression analyses of RNA mapping and histone acetylation, and for verification of the expression of novel transcripts. Hybridization and washing were performed according to the manufacturer's protocol. Signals were processed according to the GeneChip expression analysis algorithm (GCOS ver. 1.2) (Affymetrix). Probe sets with a low signal (100) were discarded in subsequent analyses. Relative changes in expression levels of HRG-treated cells against untreated cells were transformed using a log2 function. A gene was regarded as being up-regulated if this value was greater than 0.</p>


<h4>ChIP on chip experiment to measure the level of acetylation at lysine 9 of histone H3</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The level of enrichment for acetylation at lysine 9 of histone H3 (H3K9Ac) in the human genome was determined using the Affymetrix whole-genome tiling array (Affymetrix Human tiling array 1.0), which tiles the non-repetitive portion of the human genome at 35-bp intervals with more than 41 million pairs of 25-mer probe sequences. The experimental protocol employed was that previously described <a href="#pone.0001803-Nishida1">[36]</a>. MCF-7 cells were fixed with 1% formaldehyde and sonicated using a Branson 450 Sonifier. The sonicated lysates were pre-cleared using protein G sepharose. Following removal of control aliquots, the pre-cleared lysates were treated with anti-acetylated K9 histone H3 antibody (Upstate Biotechnology, Inc.). The immune complexes were precipitated with protein G dynabeads (Dynal, Invitrogen), washed and eluted from the beads. The eluted immune complexes were reverse cross-linked and treated with RNase and Proteinase K. The precipitated DNA was extracted using a phenol/chloroform method and ethanol precipitation. Both the precipitated (treated) and control DNA (whole-cell extract) was amplified using the ligation mediated (LM)-PCR method. The amplified DNA was purified, fragmented with DNase I and end-labeled with biotin-ddATP. The end-labeled DNA was hybridized to the tiling array for 18 hrs at 45°C. Following hybridization, arrays were washed and scanned using the Affymetrix GeneChip System.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The hybridization intensities (background-subtracted intensity; PM–MM, where PM and MM represent intensities detected by a 25-mer comprising perfectly matching and one-base-mismatching genome sequences, respectively) of the probes were measured in triplicate for each of the treated and control samples. A shift in intensity of the treated relative to control data in a 400-bp window centered at each probe was evaluated using a Wilcoxon Rank Sum test, which assigned a <em>p</em>-value to the probe position <a href="#pone.0001803-Bernstein1">[37]</a>. The TAS Affymetrix software (<a href="http://www.affymetrix.com/support/developer/downloads/TilingArrayTools">http://www.affymetrix.com/support/develo​per/downloads/TilingArrayTools</a>) was used for <em>p</em>-value calculations.</p>


<h4>RNA mapping and computation of signal intensity</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>RNA mapping experiments using the Affymetrix tiling array were performed as previously described <a href="#pone.0001803-Hasegawa1">[38]</a>. Fifteen micrograms of total RNA was used as a template for first-strand synthesis. Double-stranded cDNAs were synthesized using reverse transcribed cDNAs as templates. Double-stranded cDNAs were purified, fragmented, end-labeled and hybridized to the tiling arrays in triplicate. Hybridized arrays were washed and scanned using the Affymetrix GeneChip System. The signal intensity at each probe position was computed using the TAS Affymetrix software. A Wilcoxon signed-rank test was applied to hybridization intensities measured at probes located within ±50 bp of every probe location, and the pseudomedian generated by the Wilcoxon test was assigned as an estimate of the signal intensity to the probe position <a href="#pone.0001803-Kampa1">[39]</a>. The signal intensity is given on a log2 scale.</p>


<h4>Level of acetylation at lysine 9 of histone H3 in the promoter region of protein-coding genes</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Using the 5′ ends of mRNA sequences (RefSeq database: <a href="http://ftp://ftp.ncbi.nih.gov/refseq">ftp://ftp.ncbi.nih.gov/refseq</a>) aligned to the genome as the TSS, we attempted to measure the magnitude of H3K9Ac enrichment in the promoter region (±1.5 kb of TSS) of the genes. We considered the highest, average and median of −10 log (<em>p</em>-value) detected by the probes located in the promoter region.</p>


<h4>Estimation of the expression level of genes from the RNA mapping data</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The signal intensities given by the RNA mapping were used to estimate the expression level of genes in the human genome. The average signal intensity of the probes located in exons of each gene was used as an indicator of the expression level. Only genes that were mapped by 50 (tentative) or more probes were used in the analysis.</p>


<h4>Array CGH analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Genome-wide DNA screening on an Affymetrix GeneChip Mapping 250K Nsp EA array was performed using the GeneChip Instrument system according to the standard protocols of the manufacturer (Affymetrix). Briefly, genomic DNA was digested with restriction endonuclease, ligated to an adaptor, and subjected to PCR amplification with adaptor-specific primers. The PCR products were digested with DNase I and labeled with a biotinylated nucleotide analogue using terminal deoxynucleotidyl transferase. The labeled DNA fragments were hybridized to the microarray, the hybridized DNA probes were captured by streptavidin-phycoerythrin conjugates, and the array was scanned. The signal intensity ratio was calculated as previously described with some modifications <a href="#pone.0001803-Kojima1">[40]</a>. For this analysis, the publicly available data-Gene Expression Omnibus (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) accession number GSE5013 was employed as the reference data. A moving average of the signal intensity ratio across 5 adjacent markers was calculated and mapped onto the human genome according to the physical position of each marker. To calculate signal intensity ratio for X chromosome, gender information of the sample is necessary. Because called gender is not reported in the report file for Affymetrix GeneChip Mapping 250K Nsp EA array, we did not calculate signal intensity ratio for X chromosome in this study.</p>


<h4>Identification of HRG-induced up- and down-regulated genes from the H3K9 acetylation and RNA mapping data</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>In an effort to identify HRG-induced up-regulated genes, signal intensities in the control experiment (without HRG treatment) were subtracted from those obtained following HRG treatment. Genes satisfying both <em>A<sub>hrg</sub></em>−<em>A<sub>control</sub></em>≥<em>α</em> and <em>R<sub>hrg</sub></em>−<em>R<sub>control</sub></em>≥<em>β</em> were regarded as being up-regulated. Here, <em>A<sub>control</sub></em> and <em>A<sub>hrg</sub></em> represent the acetylation signal (average of −10 log (<em>p</em>-value) in the promoter region) before and after HRG treatment, respectively. Similarly, <em>R<sub>control</sub></em> and <em>R<sub>hrg</sub></em> refer to the average RNA map signals in exons in untreated cells and 2 hrs after HRG treatment, respectively. For down-regulated genes, the left-hand member was multiplied by -1. Thresholds <em>α</em> and <em>β</em> were set to 4 and 2, respectively. After obtaining the list of RefSeq accession numbers, these were converted to Entrez Gene IDs using the accession number mapping table provided at the NCBI ftp site (<a href="http://ftp://ftp.ncbi.nih.gov/gene/DATA">ftp://ftp.ncbi.nih.gov/gene/DATA</a>). RefSeq entries without corresponding Entrez Gene IDs were discarded in subsequent analyses.</p>


<h4>Prediction of upstream TF binding site</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>In an effort to determine potential transcriptional regulatory relationships, TFBS predicted from the UCSC genome browser database (<a href="http://hgdownload.cse.ucsc.edu/goldenPath/hg17/database">http://hgdownload.cse.ucsc.edu/goldenPat​h/hg17/database</a>) were utilized. Since the version of the human genome used in the tiling array and array CGH analysis was NCBI build 35, we used a corresponding data set (hg17). The region corresponding to the binding site sequence search was restricted to a 2 kb upstream region from the TSS. The genomic location of mRNA was downloaded from the UCSC genome browser database and the first base was assigned as the TSS. The relation between the mRNA accession numbers and Entrez Gene IDs was determined from the mapping table provided at the NCBI ftp site (<a href="http://ftp://ftp.ncbi.nih.gov/gene/DATA">ftp://ftp.ncbi.nih.gov/gene/DATA</a>). TF Gene IDs were initially searched for using UniProt accession numbers. When no information was found, TF names were matched against gene symbols. When no Gene ID was found, the TF was discarded from the analysis. Enrichment of TFBSs for a specific gene set was tested using Fisher's exact test (implemented using the ‘fisher.test’ function in the R language) followed by Bonferroni's correction. When searching for ER binding sites, in addition to predicted TFBSs as described above, information from TRANSFAC <a href="#pone.0001803-Matys1">[41]</a> and pattern search implementation using 3 consensus sequences, <span class="gene">TCGACGCTTTCAAGGTCATATCCG</span>, <span class="gene">TCGACAAAGTCAGGTCACAGTGACCTGATCAAG</span> and <span class="gene">GGTCANNNTGACC</span>, where ‘N’ represents an arbitrary nucleotide, <a href="#pone.0001803-Klinge1">[9]</a>, <a href="#pone.0001803-Barry1">[42]</a> were also utilized.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001803"></div><a name="pone.0001803.s001" id="pone.0001803.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001803.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">OmicBrowse display for the <em>LOC400214</em> gene. The OmicBrowse display comprises 8 lanes and each lane represents 1) copy numbers in non-treated cells, 2) and 3) H3K9 acetylation signals in the control and HRG-treated cells, respectively, 4) and 5) RNA mapping signals in the control and HRG-treated cells, respectively, 6) location of Ensembl Gene with exon-intron structure, 7) human CAGE tag counts, and 8) location of the gene obtained from NCBI Entrez Gene. The height of the vertical bars in the 2nd, 3rd, 4th and 5th lanes reflects the strength of the signals. Light green vertical bars in these lanes show regions with significant signal intensity.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001803.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(2.86 MB TIF)</p>
<a name="pone.0001803.s002" id="pone.0001803.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001803.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Copy number distribution in MCF-7 cells. Red and blue bars at the top and bottom in each plot designate actively transcribed and high copy number regions, respectively. The location of 7 genes (<em>GEM, KLF10, MRPS23, MYC, NME1, PTRH2</em> and <em>TRIB1</em>) is represented by green triangles.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001803.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.21 MB PDF)</p>
<a name="pone.0001803.s003" id="pone.0001803.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001803.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">E2-regulated TFs involved in the KEGG MAPK signaling pathway. Twelve TFs related to estrogen-induced genes involved in the KEGG MAPK signaling pathway are highlighted by a blue box. TFs are color-coded according to changes in expression of target genes (blue: down, purple: both up and down). Note that since <em>Elk-1</em> appears twice in the figure (downstream of classical MAP kinase pathway and JNK and p38 MAP kinase pathway), 13 boxes are highlighted.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001803.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(2.84 MB TIF)</p>
<a name="pone.0001803.s004" id="pone.0001803.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001803.s004">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">HRG-induced up- and down-regulated genes identified from integrated H3K9 acetylation and RNA mapping data</p>
<p class="siDoi">doi:10.1371/journal.pone.0001803.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.07 MB XLS)</p>
<a name="pone.0001803.s005" id="pone.0001803.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001803.s005">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">The list of predicted transcription factors for HRG-induced up- and down-regulated genes</p>
<p class="siDoi">doi:10.1371/journal.pone.0001803.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001803.s006" id="pone.0001803.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001803.s006">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">The list of predicted transcription factors with relatively small p-values for HRG- and E2-induced genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001803.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Takashi Nakakuki and other GSC members for performing the gene expression analysis.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HS MH. Performed the experiments: TS KI KT NY YK TK. Analyzed the data: SK TN AH. Contributed reagents/materials/analysis tools: AK YH YS TT. Wrote the paper: SK MH TN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001803-Weiss1" id="pone.0001803-Weiss1"></a>Weiss FU, Wallasch C, Campiglio M, Issing W, Ullrich A (1997) Distinct characteristics of heregulin signals mediated by HER3 or HER4. J Cell Physiol  173: 187–195.  <ul class="find" data-citedArticleID="1075320" data-doi="10.1002/(sici)1097-4652(199711)173:2<187::aid-jcp19&gt;3.3.co;2-r"><li><a href="http://dx.doi.org/10.1002/(sici)1097-4652(199711)173:2<187::aid-jcp19&gt;3.3.co;2-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+characteristics+of+heregulin+signals+mediated+by+HER3+or+HER4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+characteristics+of+heregulin+signals+mediated+by+HER3+or+HER4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001803-Carraway1" id="pone.0001803-Carraway1"></a>Carraway KL 3rd, Soltoff SP, Diamonti AJ, Cantley LC (1995) Heregulin stimulates mitogenesis and phosphatidylinositol 3-kinase in mouse fibroblasts transfected with erbB2/neu and erbB3. J Biol Chem  270: 7111–7116.  <ul class="find" data-citedArticleID="1075254" data-doi="10.1074/jbc.270.13.7111"><li><a href="http://dx.doi.org/10.1074/jbc.270.13.7111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heregulin+stimulates+mitogenesis+and+phosphatidylinositol+3-kinase+in+mouse+fibroblasts+transfected+with+erbB2%2Fneu+and+erbB3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heregulin+stimulates+mitogenesis+and+phosphatidylinositol+3-kinase+in+mouse+fibroblasts+transfected+with+erbB2%2Fneu+and+erbB3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001803-Tan1" id="pone.0001803-Tan1"></a>Tan M, Grijalva R, Yu D (1999) Heregulin beta1-activated phosphatidylinositol 3-kinase enhances aggregation of MCF-7 breast cancer cells independent of extracellular signal-regulated kinase Cancer Res  59: 1620–1625.  <ul class="find" data-citedArticleID="1075314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Heregulin+beta1-activated+phosphatidylinositol+3-kinase+enhances+aggregation+of+MCF-7+breast+cancer+cells+independent+of+extracellular+signal-regulated+kinase&amp;auth=&amp;atitle=Heregulin+beta1-activated+phosphatidylinositol+3-kinase+enhances+aggregation+of+MCF-7+breast+cancer+cells+independent+of+extracellular+signal-regulated+kinase" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heregulin+beta1-activated+phosphatidylinositol+3-kinase+enhances+aggregation+of+MCF-7+breast+cancer+cells+independent+of+extracellular+signal-regulated+kinase" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heregulin+beta1-activated+phosphatidylinositol+3-kinase+enhances+aggregation+of+MCF-7+breast+cancer+cells+independent+of+extracellular+signal-regulated+kinase%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001803-Nagashima1" id="pone.0001803-Nagashima1"></a>Nagashima T, Shimodaira H, Ide K, Nakakuki T, Tani Y, et al.  (2007) Quantitative transcriptional control of ErbB receptor signaling undergoes graded to biphasic response for cell differentiation. J Biol Chem  282: 4045–4056.  <ul class="find" data-citedArticleID="1075298" data-doi="10.1074/jbc.m608653200"><li><a href="http://dx.doi.org/10.1074/jbc.m608653200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+transcriptional+control+of+ErbB+receptor+signaling+undergoes+graded+to+biphasic+response+for+cell+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+transcriptional+control+of+ErbB+receptor+signaling+undergoes+graded+to+biphasic+response+for+cell+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001803-Yarden1" id="pone.0001803-Yarden1"></a>Yarden Y, Sliwkowski MX (2001) Untangling the ErbB signalling network. Nat Rev Mol Cell Biol.  2: 127–137.  <ul class="find" data-citedArticleID="1075322" data-doi="10.1038/35052073"><li><a href="http://dx.doi.org/10.1038/35052073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Untangling+the+ErbB+signalling+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Untangling+the+ErbB+signalling+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001803-Bunone1" id="pone.0001803-Bunone1"></a>Bunone G, Briand PA, Miksicek RJ, Picard D (1996) Activation of the unliganded estrogen receptor by EGF involves the MAP kinase pathway and direct phosphorylation. EMBO J  15: 2174–2183.  <ul class="find" data-citedArticleID="1075250"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Activation+of+the+unliganded+estrogen+receptor+by+EGF+involves+the+MAP+kinase+pathway+and+direct+phosphorylation.&amp;auth=&amp;atitle=Activation+of+the+unliganded+estrogen+receptor+by+EGF+involves+the+MAP+kinase+pathway+and+direct+phosphorylation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+the+unliganded+estrogen+receptor+by+EGF+involves+the+MAP+kinase+pathway+and+direct+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+the+unliganded+estrogen+receptor+by+EGF+involves+the+MAP+kinase+pathway+and+direct+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001803-Campbell1" id="pone.0001803-Campbell1"></a>Campbell RA, Bhat-Nakshatri P, Patel NM, Constantinidou D, Ali S, Nakshatri H (2001) Phosphatidylinositol 3-kinase/AKT-mediated activation of estrogen receptor alpha: a new model for anti-estrogen resistance. J Biol Chem  276: 9817–9824.  <ul class="find" data-citedArticleID="1075252" data-doi="10.1074/jbc.m010840200"><li><a href="http://dx.doi.org/10.1074/jbc.m010840200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphatidylinositol+3-kinase%2FAKT-mediated+activation+of+estrogen+receptor+alpha%3A+a+new+model+for+anti-estrogen+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphatidylinositol+3-kinase%2FAKT-mediated+activation+of+estrogen+receptor+alpha%3A+a+new+model+for+anti-estrogen+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001803-Martin1" id="pone.0001803-Martin1"></a>Martin MB, Franke TF, Stoica GE, Chambon P, Katzenellenbogen BS, et al.  (2000) A role for Akt in mediating the estrogenic functions of epidermal growth factor and insulin-like growth factor I. Endocrinology  141: 4503–4511.  <ul class="find" data-citedArticleID="1075290" data-doi="10.1210/endo.141.12.7836"><li><a href="http://dx.doi.org/10.1210/endo.141.12.7836" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+role+for+Akt+in+mediating+the+estrogenic+functions+of+epidermal+growth+factor+and+insulin-like+growth+factor+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+role+for+Akt+in+mediating+the+estrogenic+functions+of+epidermal+growth+factor+and+insulin-like+growth+factor+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001803-Klinge1" id="pone.0001803-Klinge1"></a>Klinge CM (2001) Estrogen receptor interaction with estrogen response elements. Nucleic Acids Res  29: 2905–2919.  <ul class="find" data-citedArticleID="1075280" data-doi="10.1093/nar/29.14.2905"><li><a href="http://dx.doi.org/10.1093/nar/29.14.2905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estrogen+receptor+interaction+with+estrogen+response+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estrogen+receptor+interaction+with+estrogen+response+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001803-Mazumdar1" id="pone.0001803-Mazumdar1"></a>Mazumdar A, Wang RA, Mishra SK, Adam L, Bagheri-Yarmand R, et al.  (2001) Transcriptional repression of oestrogen receptor by metastasis-associated protein 1 corepressor. Nat Cell Biol  3: 30–37.  <ul class="find" data-citedArticleID="1075294" data-doi="10.1038/35050532"><li><a href="http://dx.doi.org/10.1038/35050532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+repression+of+oestrogen+receptor+by+metastasis-associated+protein+1+corepressor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+repression+of+oestrogen+receptor+by+metastasis-associated+protein+1+corepressor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001803-Frasor1" id="pone.0001803-Frasor1"></a>Frasor J, Stossi F, Danes JM, Komm B, Lyttle CR, et al.  (2004) Selective estrogen receptor modulators: discrimination of agonistic versus antagonistic activities by gene expression profiling in breast cancer cells. Cancer Res  64: 1522–1533.  <ul class="find" data-citedArticleID="1075264" data-doi="10.1158/0008-5472.can-03-3326"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-03-3326" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+estrogen+receptor+modulators%3A+discrimination+of+agonistic+versus+antagonistic+activities+by+gene+expression+profiling+in+breast+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+estrogen+receptor+modulators%3A+discrimination+of+agonistic+versus+antagonistic+activities+by+gene+expression+profiling+in+breast+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001803-Frasor2" id="pone.0001803-Frasor2"></a>Frasor J, Danes JM, Komm B, Chang KC, Lyttle CR, et al.  (2003) Profiling of estrogen up- and down-regulated gene expression in human breast cancer cells: insights into gene networks and pathways underlying estrogenic control of proliferation and cell phenotype. Endocrinology  144: 4562–4574.  <ul class="find" data-citedArticleID="1075266" data-doi="10.1210/en.2003-0567"><li><a href="http://dx.doi.org/10.1210/en.2003-0567" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Profiling+of+estrogen+up-+and+down-regulated+gene+expression+in+human+breast+cancer+cells%3A+insights+into+gene+networks+and+pathways+underlying+estrogenic+control+of+proliferation+and+cell+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Profiling+of+estrogen+up-+and+down-regulated+gene+expression+in+human+breast+cancer+cells%3A+insights+into+gene+networks+and+pathways+underlying+estrogenic+control+of+proliferation+and+cell+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001803-Kumar1" id="pone.0001803-Kumar1"></a>Kumar V, Chambon P (1998) The estrogen receptor binds tightly to its responsive element as a ligand-induced homodimer. Cell  55: 145–156.  <ul class="find" data-citedArticleID="1075284" data-doi="10.1016/0092-8674(88)90017-7"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90017-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+estrogen+receptor+binds+tightly+to+its+responsive+element+as+a+ligand-induced+homodimer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+estrogen+receptor+binds+tightly+to+its+responsive+element+as+a+ligand-induced+homodimer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001803-Joel1" id="pone.0001803-Joel1"></a>Joel PB, Smith J, Sturgill TW, Fisher TL, Blenis J, et al.  (1998) pp90rsk1 regulates estrogen receptor-mediated transcription through phosphorylation of Ser-167. Mol Cell Biol. 18: 1978–1984.  <ul class="find" data-citedArticleID="1075272"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=pp90rsk1+regulates+estrogen+receptor-mediated+transcription+through+phosphorylation+of+Ser-167.+Mol+Cell+Biol.&amp;auth=&amp;atitle=pp90rsk1+regulates+estrogen+receptor-mediated+transcription+through+phosphorylation+of+Ser-167.+Mol+Cell+Biol." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=pp90rsk1+regulates+estrogen+receptor-mediated+transcription+through+phosphorylation+of+Ser-167.+Mol+Cell+Biol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22pp90rsk1+regulates+estrogen+receptor-mediated+transcription+through+phosphorylation+of+Ser-167.+Mol+Cell+Biol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001803-Pietras1" id="pone.0001803-Pietras1"></a>Pietras RJ, Arboleda J, Reese DM, Wongvipat N, Pegram MD, et al.  (1995) HER-2 tyrosine kinase pathway targets estrogen receptor and promotes hormone-independent growth in human breast cancer cells. Oncogene  10: 2435–2446.  <ul class="find" data-citedArticleID="1075304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=HER-2+tyrosine+kinase+pathway+targets+estrogen+receptor+and+promotes+hormone-independent+growth+in+human+breast+cancer+cells.&amp;auth=&amp;atitle=HER-2+tyrosine+kinase+pathway+targets+estrogen+receptor+and+promotes+hormone-independent+growth+in+human+breast+cancer+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HER-2+tyrosine+kinase+pathway+targets+estrogen+receptor+and+promotes+hormone-independent+growth+in+human+breast+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HER-2+tyrosine+kinase+pathway+targets+estrogen+receptor+and+promotes+hormone-independent+growth+in+human+breast+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001803-Shahbazian1" id="pone.0001803-Shahbazian1"></a>Shahbazian MD, Grunstein M (2007) Functions of Site-Specific Histone Acetylation and Deacetylation. Annu Rev Biochem  76: 75–100.  <ul class="find" data-citedArticleID="1075310" data-doi="10.1146/annurev.biochem.76.052705.162114"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.76.052705.162114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functions+of+Site-Specific+Histone+Acetylation+and+Deacetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functions+of+Site-Specific+Histone+Acetylation+and+Deacetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001803-Fann1" id="pone.0001803-Fann1"></a>Fann M, Godlove JM, Catalfamo M, Wood WH 3rd, Chrest FJ, et al.  (2006) Histone acetylation is associated with differential gene expression in the rapid and robust memory CD8(+) T-cell response. Blood  108: 3363–3370.  <ul class="find" data-citedArticleID="1075262" data-doi="10.1182/blood-2006-02-005520"><li><a href="http://dx.doi.org/10.1182/blood-2006-02-005520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+acetylation+is+associated+with+differential+gene+expression+in+the+rapid+and+robust+memory+CD8%28%2B%29+T-cell+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+acetylation+is+associated+with+differential+gene+expression+in+the+rapid+and+robust+memory+CD8%28%2B%29+T-cell+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001803-Bertone1" id="pone.0001803-Bertone1"></a>Bertone P, Stolc V, Royce TE, Rozowsky JS, Urban AE, et al.  (2004) Global identification of human transcribed sequences with genome tiling arrays. Science  306: 2242–2246.  <ul class="find" data-citedArticleID="1075248" data-doi="10.1126/science.1103388"><li><a href="http://dx.doi.org/10.1126/science.1103388" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+identification+of+human+transcribed+sequences+with+genome+tiling+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+identification+of+human+transcribed+sequences+with+genome+tiling+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001803-Toyoda1" id="pone.0001803-Toyoda1"></a>Toyoda T, Mochizuki Y, Player K, Heida N, Kobayashi N, et al.  (2007) OmicBrowse: a browser of multidimensional omics annotations. Bioinformatics  23: 524–526.  <ul class="find" data-citedArticleID="1075318" data-doi="10.1093/bioinformatics/btl523"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl523" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OmicBrowse%3A+a+browser+of+multidimensional+omics+annotations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OmicBrowse%3A+a+browser+of+multidimensional+omics+annotations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001803-Karolchik1" id="pone.0001803-Karolchik1"></a>Karolchik D, Baertsch R, Diekhans M, Furey TS, Hinrichs A, et al.  (2003) The UCSC Genome Browser Database. Nucleic Acids Res  31: 51–54.  <ul class="find" data-citedArticleID="1075278" data-doi="10.1093/nar/gkg129"><li><a href="http://dx.doi.org/10.1093/nar/gkg129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Genome+Browser+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Genome+Browser+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001803-Kanehisa1" id="pone.0001803-Kanehisa1"></a>Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, et al.  (2006) From genomics to chemical genomics: new developments in KEGG. Nucleic Acids Res  34: D354–357.  <ul class="find" data-citedArticleID="1075276" data-doi="10.1093/nar/gkj102"><li><a href="http://dx.doi.org/10.1093/nar/gkj102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001803-Ali1" id="pone.0001803-Ali1"></a>Ali S, Metzger D, Bornert JM, Chambon P (1993) Modulation of transcriptional activation by ligand-dependent phosphorylation of the human oestrogen receptor A/B region. EMBO J  12: 1153–1160.  <ul class="find" data-citedArticleID="1075240"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modulation+of+transcriptional+activation+by+ligand-dependent+phosphorylation+of+the+human+oestrogen+receptor+A%2FB+region.&amp;auth=&amp;atitle=Modulation+of+transcriptional+activation+by+ligand-dependent+phosphorylation+of+the+human+oestrogen+receptor+A%2FB+region." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+transcriptional+activation+by+ligand-dependent+phosphorylation+of+the+human+oestrogen+receptor+A%2FB+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+transcriptional+activation+by+ligand-dependent+phosphorylation+of+the+human+oestrogen+receptor+A%2FB+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001803-Cheng1" id="pone.0001803-Cheng1"></a>Cheng J, Zhang C, Shapiro DJ (2007) A functional serine 118 phosphorylation site in estrogen receptor-alpha is required for down-regulation of gene expression by 17beta-estradiol and 4-hydroxytamoxifen. Endocrinology  148: 4634–4641.  <ul class="find" data-citedArticleID="1075256" data-doi="10.1210/en.2007-0148"><li><a href="http://dx.doi.org/10.1210/en.2007-0148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+functional+serine+118+phosphorylation+site+in+estrogen+receptor-alpha+is+required+for+down-regulation+of+gene+expression+by+17beta-estradiol+and+4-hydroxytamoxifen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+functional+serine+118+phosphorylation+site+in+estrogen+receptor-alpha+is+required+for+down-regulation+of+gene+expression+by+17beta-estradiol+and+4-hydroxytamoxifen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001803-Moggs1" id="pone.0001803-Moggs1"></a>Moggs JG, Orphanides G (2001) Estrogen receptors: orchestrators of pleiotropic cellular responses. EMBO Rep  2: 775–781.  <ul class="find" data-citedArticleID="1075296" data-doi="10.1093/embo-reports/kve185"><li><a href="http://dx.doi.org/10.1093/embo-reports/kve185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estrogen+receptors%3A+orchestrators+of+pleiotropic+cellular+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estrogen+receptors%3A+orchestrators+of+pleiotropic+cellular+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001803-Nilsson1" id="pone.0001803-Nilsson1"></a>Nilsson S, Makela S, Treuter E, Tujague M, Thomsen J, et al.  (2001) Mechanisms of estrogen action. Physiol Rev  81: 1535–1565.  <ul class="find" data-citedArticleID="1075300"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanisms+of+estrogen+action.&amp;auth=&amp;atitle=Mechanisms+of+estrogen+action." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+estrogen+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+estrogen+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001803-Cheung1" id="pone.0001803-Cheung1"></a>Cheung P, Tanner KG, Cheung WL, Sassone-Corsi P, Denu JM, et al.  (2000) Synergistic coupling of histone H3 phosphorylation and acetylation in response to epidermal growth factor stimulation. Mol Cell  5: 905–1005.  <ul class="find" data-citedArticleID="1075258" data-doi="10.1016/s1097-2765(00)80256-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80256-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synergistic+coupling+of+histone+H3+phosphorylation+and+acetylation+in+response+to+epidermal+growth+factor+stimulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synergistic+coupling+of+histone+H3+phosphorylation+and+acetylation+in+response+to+epidermal+growth+factor+stimulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001803-TorresArzayus1" id="pone.0001803-TorresArzayus1"></a>Torres-Arzayus MI, Font de Mora J, Yuan J, Vazquez F, Bronson R, et al.  (2004) High tumor incidence and activation of the PI3K/AKT pathway in transgenic mice define AIB1 as an oncogene. Cancer Cell  6: 263–274.  <ul class="find" data-citedArticleID="1075316" data-doi="10.1016/j.ccr.2004.06.027"><li><a href="http://dx.doi.org/10.1016/j.ccr.2004.06.027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+tumor+incidence+and+activation+of+the+PI3K%2FAKT+pathway+in+transgenic+mice+define+AIB1+as+an+oncogene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+tumor+incidence+and+activation+of+the+PI3K%2FAKT+pathway+in+transgenic+mice+define+AIB1+as+an+oncogene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001803-Proia1" id="pone.0001803-Proia1"></a>Proia DA, Nannenga BW, Donehower LA, Weigel NL (2006) Dual roles for the phosphatase PPM1D in regulating progesterone receptor function. J Biol Chem  281: 7089–7101.  <ul class="find" data-citedArticleID="1075306" data-doi="10.1074/jbc.m511839200"><li><a href="http://dx.doi.org/10.1074/jbc.m511839200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dual+roles+for+the+phosphatase+PPM1D+in+regulating+progesterone+receptor+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dual+roles+for+the+phosphatase+PPM1D+in+regulating+progesterone+receptor+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001803-Ren1" id="pone.0001803-Ren1"></a>Ren S, Smith MJ, Louro ID, McKie-Bell P, Bani MR, et al.  (2000) The p44S10 locus, encoding a subunit of the proteasome regulatory particle, is amplified during progression of cutaneous malignant melanoma. Oncogene  19: 1419–1427.  <ul class="find" data-citedArticleID="1075308" data-doi="10.1038/sj.onc.1203462"><li><a href="http://dx.doi.org/10.1038/sj.onc.1203462" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+p44S10+locus%2C+encoding+a+subunit+of+the+proteasome+regulatory+particle%2C+is+amplified+during+progression+of+cutaneous+malignant+melanoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+p44S10+locus%2C+encoding+a+subunit+of+the+proteasome+regulatory+particle%2C+is+amplified+during+progression+of+cutaneous+malignant+melanoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001803-Maass1" id="pone.0001803-Maass1"></a>Maass N, Rosel F, Schem C, Hitomi J, Jonat W, et al.  (2002) Amplification of the BCAS2 gene at chromosome 1p13.3-21 in human primary breast cancer. Cancer Lett  185: 219–223.  <ul class="find" data-citedArticleID="1075288" data-doi="10.1016/s0304-3835(02)00286-0"><li><a href="http://dx.doi.org/10.1016/s0304-3835(02)00286-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amplification+of+the+BCAS2+gene+at+chromosome+1p13.3-21+in+human+primary+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amplification+of+the+BCAS2+gene+at+chromosome+1p13.3-21+in+human+primary+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001803-Grunt1" id="pone.0001803-Grunt1"></a>Grunt TW, Saceda M, Martin MB, Lupu R, Dittrich E, et al.  (1995) Bidirectional interactions between the estrogen receptor and the cerbB-2 signaling pathways: heregulin inhibits estrogenic effects in breast cancer cells. Int J Cancer  63: 560–567.  <ul class="find" data-citedArticleID="1075268" data-doi="10.1002/ijc.2910630417"><li><a href="http://dx.doi.org/10.1002/ijc.2910630417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bidirectional+interactions+between+the+estrogen+receptor+and+the+cerbB-2+signaling+pathways%3A+heregulin+inhibits+estrogenic+effects+in+breast+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bidirectional+interactions+between+the+estrogen+receptor+and+the+cerbB-2+signaling+pathways%3A+heregulin+inhibits+estrogenic+effects+in+breast+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001803-Lin1" id="pone.0001803-Lin1"></a>Lin CY, Vega VB, Thomsen JS, Zhang T, Kong SL, Xie M, et al.  (2007) Whole-genome cartography of estrogen receptor alpha binding sites. PLoS Genet  3: e87.  <ul class="find" data-citedArticleID="1075286" data-doi="10.1371/journal.pgen.0030087"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+cartography+of+estrogen+receptor+alpha+binding+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+cartography+of+estrogen+receptor+alpha+binding+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001803-Arnold1" id="pone.0001803-Arnold1"></a>Arnold SF, Obourn JD, Jaffe H, Notides AC (1995) Phosphorylation of the human estrogen receptor by mitogen-activated protein kinase and casein kinase II: consequence on DNA binding. J Steroid Biochem Mol Biol  55: 163–172.  <ul class="find" data-citedArticleID="1075242" data-doi="10.1016/0960-0760(95)00177-2"><li><a href="http://dx.doi.org/10.1016/0960-0760(95)00177-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+the+human+estrogen+receptor+by+mitogen-activated+protein+kinase+and+casein+kinase+II%3A+consequence+on+DNA+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+the+human+estrogen+receptor+by+mitogen-activated+protein+kinase+and+casein+kinase+II%3A+consequence+on+DNA+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001803-Simoncini1" id="pone.0001803-Simoncini1"></a>Simoncini T, Hafezi-Moghadam A, Brazil DP, Ley K, Chin WW, et al.  (2000) Interaction of oestrogen receptor with the regulatory subunit of phosphatidylinositol-3-OH kinase. Nature  407: 538–541.  <ul class="find" data-citedArticleID="1075312" data-doi="10.1038/35035131"><li><a href="http://dx.doi.org/10.1038/35035131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+oestrogen+receptor+with+the+regulatory+subunit+of+phosphatidylinositol-3-OH+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+oestrogen+receptor+with+the+regulatory+subunit+of+phosphatidylinositol-3-OH+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001803-Driggers1" id="pone.0001803-Driggers1"></a>Driggers PH, Segars JH (2002) Estrogen action and cytoplasmic signaling pathways. Part II: the role of growth factors and phosphorylation in estrogen signaling. Trends Endocrinol Metab  13: 422–427.  <ul class="find" data-citedArticleID="1075260" data-doi="10.1016/s1043-2760(02)00634-3"><li><a href="http://dx.doi.org/10.1016/s1043-2760(02)00634-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estrogen+action+and+cytoplasmic+signaling+pathways.+Part+II%3A+the+role+of+growth+factors+and+phosphorylation+in+estrogen+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estrogen+action+and+cytoplasmic+signaling+pathways.+Part+II%3A+the+role+of+growth+factors+and+phosphorylation+in+estrogen+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001803-Nishida1" id="pone.0001803-Nishida1"></a>Nishida H, Suzuki T, Kondo S, Miura H, Fujimura F, et al.  (2006) Histone H3 acetylated at lysine 9 in promoter is associated with low nucleosome density in the vicinity of transcription start site in human cell. Chromosome Res  14: 203–211.  <ul class="find" data-citedArticleID="1075302" data-doi="10.1007/s10577-006-1036-7"><li><a href="http://dx.doi.org/10.1007/s10577-006-1036-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+H3+acetylated+at+lysine+9+in+promoter+is+associated+with+low+nucleosome+density+in+the+vicinity+of+transcription+start+site+in+human+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+H3+acetylated+at+lysine+9+in+promoter+is+associated+with+low+nucleosome+density+in+the+vicinity+of+transcription+start+site+in+human+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001803-Bernstein1" id="pone.0001803-Bernstein1"></a>Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, et al.  (2005) Genomic maps and comparative analysis of histone modifications in human and mouse. Cell  120: 169–181.  <ul class="find" data-citedArticleID="1075246" data-doi="10.1016/j.cell.2005.01.001"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001803-Hasegawa1" id="pone.0001803-Hasegawa1"></a>Hasegawa Y, Fukuda S, Shimokawa K, Kondo S, Maeda N, et al.  (2006) A RecA-mediated exon profiling method. Nucleic Acids Res  34: e97.  <ul class="find" data-citedArticleID="1075270" data-doi="10.1093/nar/gkl497"><li><a href="http://dx.doi.org/10.1093/nar/gkl497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+RecA-mediated+exon+profiling+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+RecA-mediated+exon+profiling+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001803-Kampa1" id="pone.0001803-Kampa1"></a>Kampa D, Cheng J, Kapranov P, Yamanaka M, Brubaker S, et al.  (2004) Novel RNAs identified an in-depth analysis of the transcriptome of human chromosomes 21 and 22. Genome Res  14: 331–342.  <ul class="find" data-citedArticleID="1075274" data-doi="10.1101/gr.2094104"><li><a href="http://dx.doi.org/10.1101/gr.2094104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+RNAs+identified+an+in-depth+analysis+of+the+transcriptome+of+human+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+RNAs+identified+an+in-depth+analysis+of+the+transcriptome+of+human+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001803-Kojima1" id="pone.0001803-Kojima1"></a>Kojima T, Mukai W, Fuma D, Ueda Y, Okada M, et al.  (2006) Determination of genomic breakpoints in an epileptic patient using genotyping array. Biochem Biophys Res Commun  341: 792–796.  <ul class="find" data-citedArticleID="1075282" data-doi="10.1016/j.bbrc.2006.01.032"><li><a href="http://dx.doi.org/10.1016/j.bbrc.2006.01.032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determination+of+genomic+breakpoints+in+an+epileptic+patient+using+genotyping+array." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determination+of+genomic+breakpoints+in+an+epileptic+patient+using+genotyping+array.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001803-Matys1" id="pone.0001803-Matys1"></a>Matys V, Kel-Margoulis OV, Fricke E, Liebich I, Land S, et al.  (2006) TRANSFAC and its module TRANSCompel: transcriptional gene regulation in eukaryotes. Nucleic Acids Res  34: D108–110.  <ul class="find" data-citedArticleID="1075292" data-doi="10.1093/nar/gkj143"><li><a href="http://dx.doi.org/10.1093/nar/gkj143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRANSFAC+and+its+module+TRANSCompel%3A+transcriptional+gene+regulation+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRANSFAC+and+its+module+TRANSCompel%3A+transcriptional+gene+regulation+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001803-Barry1" id="pone.0001803-Barry1"></a>Barry JB, Giguere V (2005) Epidermal growth factor-induced signaling in breast cancer cells results in selective target gene activation by orphan nuclear receptor estrogen-related receptor alpha. Cancer Res  65: 6120–6129.  <ul class="find" data-citedArticleID="1075244" data-doi="10.1158/0008-5472.can-05-0922"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-0922" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epidermal+growth+factor-induced+signaling+in+breast+cancer+cells+results+in+selective+target+gene+activation+by+orphan+nuclear+receptor+estrogen-related+receptor+alpha." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epidermal+growth+factor-induced+signaling+in+breast+cancer+cells+results+in+selective+target+gene+activation+by+orphan+nuclear+receptor+estrogen-related+receptor+alpha.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.XML" value="102039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.PDF" value="773200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g001.PNG_L" value="929525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g001.PNG_M" value="91573"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g001.PNG_S" value="6382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g001.TIF" value="1192048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g001.PNG_I" value="30028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g002.PNG_L" value="383025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g002.PNG_M" value="71178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g002.PNG_S" value="9237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g002.TIF" value="950846"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g002.PNG_I" value="43936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.t001.PNG_L" value="79993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.t001.PNG_M" value="99995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.t001.PNG_S" value="16642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.t001.TIF" value="477862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.t001.PNG_I" value="39049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g003.PNG_L" value="1413049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g003.PNG_M" value="173417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g003.PNG_S" value="15300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g003.TIF" value="2087416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g003.PNG_I" value="60210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g004.PNG_L" value="1919513"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g004.PNG_M" value="163064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g004.PNG_S" value="13328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g004.TIF" value="3458430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g004.PNG_I" value="54627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g005.PNG_L" value="1947621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g005.PNG_M" value="146190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g005.PNG_S" value="10138"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g005.TIF" value="2579250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g005.PNG_I" value="84703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g006.PNG_L" value="8300769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g006.PNG_M" value="212250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g006.PNG_S" value="13799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g006.TIF" value="9556348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.g006.PNG_I" value="95314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.s001.TIF" value="5604022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.s002.PDF" value="1205322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.s003.TIF" value="3567032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.s004.XLS" value="65536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.s005.XLS" value="17408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001803.s006.XLS" value="20992"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001803&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001803" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001803&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001803&volume=&issue=&title=Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells&author_name=Takeshi%20Nagashima%2C%20Takahiro%20Suzuki%2C%20Shinji%20Kondo%2C%20Yoko%20Kuroki%2C%20Kaoru%20Takahashi%2C%20Kaori%20Ide%2C%20Noriko%20Yumoto%2C%20Aki%20Hasegawa%2C%20Tetsuro%20Toyoda%2C%20Toshio%20Kojima%2C%20Akihiko%20Konagaya%2C%20Harukazu%20Suzuki%2C%20Yoshihide%20Hayashizaki%2C%20Yoshiyuki%20Sakaki%2C%20Mariko%20Hatakeyama&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001803" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803&amp;t=Integrative%20Genome-Wide%20Expression%20Analysis%20Bears%20Evidence%20of%20Estrogen%20Receptor-Independent%20Transcription%20in%20Heregulin-Stimulated%20MCF-7%20Cells" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803&title=Integrative%20Genome-Wide%20Expression%20Analysis%20Bears%20Evidence%20of%20Estrogen%20Receptor-Independent%20Transcription%20in%20Heregulin-Stimulated%20MCF-7%20Cells&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803&amp;title=Integrative%20Genome-Wide%20Expression%20Analysis%20Bears%20Evidence%20of%20Estrogen%20Receptor-Independent%20Transcription%20in%20Heregulin-Stimulated%20MCF-7%20Cells" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001803&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Integrative Genome-Wide Expression Analysis Bears Evidence of Estrogen Receptor-Independent Transcription in Heregulin-Stimulated MCF-7 Cells';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001803';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Integrative%20Genome-Wide%20Expression%20Analysis%20Bears%20Evidence%20of%20Estrogen%20Receptor-Independent%20Transcription%20in%20Heregulin-Stimulated%20MCF-7%20Cells http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001803" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001803" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acetylation%22" title="Search for articles in the subject area:'Acetylation'"><div class="flagText">Acetylation</div></a>
              <div data-categoryid="21455" data-articleid="26430"
                   data-categoryname="Acetylation"
                   class="flagImage" title="Flag 'Acetylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="26430"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26430"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="26430"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="26430"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26430"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="26430"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MAPK+signaling+cascades%22" title="Search for articles in the subject area:'MAPK signaling cascades'"><div class="flagText">MAPK signaling cascades</div></a>
              <div data-categoryid="21105" data-articleid="26430"
                   data-categoryname="MAPK signaling cascades"
                   class="flagImage" title="Flag 'MAPK signaling cascades' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7782'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8273'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=305&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=1611">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
