

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001235"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001235" />

    <meta name="citation_title" content="Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray"/>
    <meta itemprop="name" content="Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray"/>

      <meta name="citation_author" content="Kouji Satoh"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Koji Doi"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Toshifumi Nagata"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Naoki Kishimoto"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Kohji Suzuki"/>
            <meta name="citation_author_institution" content="Hitachi Software Engineering, Shinagawa-ku, Tokyo, Japan"/>
      <meta name="citation_author" content="Yasuhiro Otomo"/>
            <meta name="citation_author_institution" content="Laboratory of Genome Sequencing and Analysis Group, Foundation for Advancement of International Science (FAIS), Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Jun Kawai"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
            <meta name="citation_author_institution" content="Genome Science Laboratory, RIKEN Wako Institute, Wako, Saitama, Japan"/>
      <meta name="citation_author" content="Mari Nakamura"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Tomoko Hirozane-Kishikawa"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Saeko Kanagawa"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Takahiro Arakawa"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Juri Takahashi-Iida"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Mitsuyoshi Murata"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Noriko Ninomiya"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Daisuke Sasaki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Shiro Fukuda"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Michihira Tagami"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
      <meta name="citation_author" content="Harumi Yamagata"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Kanako Kurita"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Kozue Kamiya"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Mayu Yamamoto"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Ari Kikuta"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Takahito Bito"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Nahoko Fujitsuka"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Kazue Ito"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Hiroyuki Kanamori"/>
            <meta name="citation_author_institution" content="Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Il-Ryong Choi"/>
            <meta name="citation_author_institution" content="Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, DAPO, Metro Manila, Philippines"/>
      <meta name="citation_author" content="Yoshiaki Nagamura"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Takashi Matsumoto"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Kazuo Murakami"/>
            <meta name="citation_author_institution" content="Laboratory of Genome Sequencing and Analysis Group, Foundation for Advancement of International Science (FAIS), Tsukuba, Ibaraki, Japan"/>
      <meta name="citation_author" content="Ken-ichi Matsubara"/>
            <meta name="citation_author_institution" content="Laboratory of Genome Sequencing and Analysis Group, Foundation for Advancement of International Science (FAIS), Tsukuba, Ibaraki, Japan"/>
            <meta name="citation_author_institution" content="Nara Institute of Science and Technology (NAIST), Ikoma, Nara, Japan"/>
      <meta name="citation_author" content="Piero Carninci"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
            <meta name="citation_author_institution" content="Genome Science Laboratory, RIKEN Wako Institute, Wako, Saitama, Japan"/>
      <meta name="citation_author" content="Yoshihide Hayashizaki"/>
            <meta name="citation_author_institution" content="Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan"/>
            <meta name="citation_author_institution" content="Genome Science Laboratory, RIKEN Wako Institute, Wako, Saitama, Japan"/>
      <meta name="citation_author" content="Shoshi Kikuchi"/>
            <meta name="citation_author_institution" content="Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001235.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1235"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp. japonica).; citation_author=SA Goff; citation_author=D Ricke; citation_author=TH Lan; citation_author=G Presting; citation_author=R Wang; citation_author=M Dunn; citation_journal_title=Science; citation_volume=296; citation_number=1; citation_pages=92-100; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp. indica).; citation_author=J Yu; citation_author=S Hu; citation_author=J Wang; citation_author=GK Wong; citation_author=S Li; citation_journal_title=Science; citation_volume=296; citation_number=2; citation_pages=79-92; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genomes of Oryza sativa: a history of duplications.; citation_author=J Yu; citation_author=J Wang; citation_author=W Lin; citation_author=S Li; citation_author=H Li; citation_number=3; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome.; citation_journal_title=Nature; citation_volume=436; citation_number=4; citation_pages=793-800; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice.; citation_journal_title=Science; citation_volume=301; citation_number=5; citation_pages=376-379; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A comprehensive rice transcript map containing 6591 expressed sequence tag sites.; citation_author=J Wu; citation_author=T Maehara; citation_author=T Shimokawa; citation_author=S Yamamoto; citation_author=C Harada; citation_journal_title=Plant Cell; citation_volume=14; citation_number=6; citation_pages=525-535; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gene identification and expression analysis of 86,136 Expressed Sequence Tags (EST) from the rice genome.; citation_author=Y Zhou; citation_author=J Tang; citation_author=MG Walker; citation_author=X Zhang; citation_author=J Wang; citation_journal_title=Genomics Proteomics Bioinformatics; citation_volume=1; citation_number=7; citation_pages=26-42; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Features of the expressed sequences revealed by a large-scale analysis of ESTs from a normalized cDNA library of the elite indica rice cultivar Minghui 63.; citation_author=J Zhang; citation_author=Q Feng; citation_author=C Jin; citation_author=D Qiu; citation_author=L Zhang; citation_journal_title=Plant J; citation_volume=42; citation_number=8; citation_pages=772-780; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The TIGR Rice Genome Annotation Resource: improvements and new features.; citation_author=S Ouyang; citation_author=W Zhu; citation_author=J Hamilton; citation_author=H Lin; citation_author=M Campbell; citation_journal_title=Nucleic Acids Res; citation_volume=35(Database issue); citation_number=9; citation_pages=D883-887; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gramene: a bird's eye view of cereal genomes.; citation_author=P Jaiswal; citation_author=J Ni; citation_author=I Yap; citation_author=D Ware; citation_author=W Spooner; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=10; citation_pages=D717-723; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PlnTFDB: an integrative plant transcription factor database.; citation_author=DM Riano-Pachon; citation_author=S Ruzicic; citation_author=I Dreyer; citation_author=B Mueller-Roeber; citation_number=11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=In plants, highly expressed genes are the least compact.; citation_author=XY Ren; citation_author=O Vorst; citation_author=MW Fiers; citation_author=WJ Stiekema; citation_author=JP Nap; citation_journal_title=Trends Genet; citation_volume=22; citation_number=12; citation_pages=528-532; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Intron gain and loss in segmentally duplicated genes in rice.; citation_author=H Lin; citation_author=W Zhu; citation_author=JC Silva; citation_author=X Gu; citation_author=CR Buell; citation_number=13; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Rice Annotation Project Database (RAP-DB): hub for Oryza sativa ssp. japonica genome information.; citation_author=H Ohyanagi; citation_author=T Tanaka; citation_author=H Sakai; citation_author=Y Shigemoto; citation_author=K Yamaguchi; citation_journal_title=Nucleic Acids Res 34(Database issue); citation_number=14; citation_pages=D741-744; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Curated genome annotation of Oryza sativa ssp. japonica and comparative genome analysis with Arabidopsis thaliana.; citation_journal_title=Genome Res; citation_volume=17; citation_number=15; citation_pages=175-183; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=BGI-RIS: an integrated information resource and comparative analysis workbench for rice genomics.; citation_author=W Zhao; citation_author=J Wang; citation_author=X He; citation_author=X Huang; citation_author=Y Jiao; citation_journal_title=Nucleic Acids Res; citation_volume=32(Database issue); citation_number=16; citation_pages=D377-382; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide transcription analyses in rice using tiling microarrays.; citation_author=L Li; citation_author=X Wang; citation_author=V Stolc; citation_author=X Li; citation_author=D Zhang; citation_journal_title=Nat Genet; citation_volume=38; citation_number=17; citation_pages=124-129; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional annotation of a full-length Arabidopsis cDNA collection.; citation_author=M Seki; citation_author=M Narusaka; citation_author=A Kamiya; citation_author=J Ishida; citation_author=M Satou; citation_journal_title=Science; citation_volume=296; citation_number=18; citation_pages=141-145; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Integrative annotation of 21,037 human genes validated by full-length cDNA clones. PLoS Biol 2.; citation_author=T Imanishi; citation_author=T Itoh; citation_author=Y Suzuki; citation_author=C O'Donovan; citation_author=S Fukuchi; citation_number=19; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional annotation of a full-length mouse cDNA collection.; citation_journal_title=Nature; citation_volume=409; citation_number=20; citation_pages=685-690; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Plant MPSS databases: signature-based transcriptional resources for analyses of mRNA and small RNA.; citation_author=M Nakano; citation_author=K Nobuta; citation_author=K Vemaraju; citation_author=SS Tej; citation_author=JW Skogen; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=21; citation_pages=D731-735; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Global transcript analysis of rice leaf and seed using SAGE technology.; citation_author=JG Gibbings; citation_author=BP Cook; citation_author=MR Dufault; citation_author=SL Madden; citation_author=S Khuri; citation_journal_title=Plant Biotechnol J; citation_volume=1; citation_number=22; citation_pages=271-285; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=5′-end SAGE for the analysis of transcriptional start sites.; citation_author=S Hashimoto; citation_author=Y Suzuki; citation_author=Y Kasai; citation_author=K Morohoshi; citation_author=T Yamada; citation_journal_title=Nat Biotechnol.; citation_volume=22; citation_number=23; citation_pages=1146-1149; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Empirical analysis of transcriptional activity in the Arabidopsis genome.; citation_author=K Yamada; citation_author=J Lim; citation_author=JM Dale; citation_author=H Chen; citation_author=P Shinn; citation_journal_title=Science; citation_volume=302; citation_number=24; citation_pages=842-846; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): a model organism database providing a centralized, curated gateway to Arabidopsis biology, research materials and community.; citation_author=SY Rhee; citation_author=W Beavis; citation_author=TZ Berardini; citation_author=G Chen; citation_author=D Dixon; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=25; citation_pages=224-228; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Experimental validation of the regulated expression of large numbers of non-coding RNAs from the mouse genome.; citation_author=T Ravasi; citation_author=H Suzuki; citation_author=KC Pang; citation_author=S Katayama; citation_author=M Furuno; citation_journal_title=Genome Res; citation_volume=16; citation_number=26; citation_pages=11-19; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The human transcriptome map: clustering of highly expressed genes in chromosomal domains.; citation_author=H Caron; citation_author=B van Schaik; citation_author=M van der Mee; citation_author=F Baas; citation_author=G Riggins; citation_journal_title=Science.; citation_volume=291; citation_number=27; citation_pages=1289-1292; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Clustering of housekeeping genes provides a unified model of gene order in the human genome.; citation_author=MJ Lercher; citation_author=AO Urrutia; citation_author=LD Hurst; citation_journal_title=Nat Genet; citation_volume=31; citation_number=28; citation_pages=180-183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Rice as a model for centromere and heterochromatin research.; citation_author=H Yan; citation_author=J Jiang; citation_journal_title=Chromosome Res; citation_volume=15; citation_number=29; citation_pages=77-84; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sequencing of a rice centromere uncovers active genes.; citation_author=K Nagaki; citation_author=Z Cheng; citation_author=S Ouyang; citation_author=PB Talbert; citation_author=M Kim; citation_journal_title=Nat Genet; citation_volume=36; citation_number=30; citation_pages=138-145; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic and genetic characterization of rice Cen3 reveals extensive transcription and evolutionary implications of a complex centromere.; citation_author=H Yan; citation_author=H Ito; citation_author=K Nobuta; citation_author=S Ouyang; citation_author=W Jin; citation_journal_title=Plant Cell; citation_volume=18; citation_number=31; citation_pages=2123-2133; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification and mapping of expressed genes, simple sequence repeats and transposable elements in centromeric regions of rice chromosomes.; citation_author=H Mizuno; citation_author=K Ito; citation_author=J Wu; citation_author=T Tanaka; citation_author=H Kanamori; citation_journal_title=DNA Res; citation_volume=13; citation_number=32; citation_pages=267-274; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis.; citation_author=X Zhang; citation_author=J Yazaki; citation_author=A Sundaresan; citation_author=S Cokus; citation_author=SW Chan; citation_journal_title=Cell; citation_volume=126; citation_number=33; citation_pages=1189-1201; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services.; citation_author=RD Finn; citation_author=J Mistry; citation_author=B Schuster-Bockler; citation_author=S Griffiths-Jones; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=34(Database issue); citation_number=34; citation_pages=D247-251; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=New developments in the InterPro database.; citation_author=NJ Mulder; citation_author=R Apweiler; citation_author=TK Attwood; citation_author=A Bairoch; citation_author=A Bateman; citation_journal_title=Nucleic Acids Res; citation_volume=35(Database issue); citation_number=35; citation_pages=D224-228; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Intron insertion facilitates amplification of cloned virus cDNA in Escherichia coli while biological activity is reestablished after transcription in vivo.; citation_author=IE Johansen; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=93; citation_number=36; citation_pages=12400-12405; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Infectious in vivo and in vitro transcripts from a full-length cDNA clone of PVY-N605, a Swiss necrotic isolate of potato virus Y.; citation_author=G Jakab; citation_author=E Droz; citation_author=G Brigneti; citation_author=D Baulcombe; citation_author=P Malnoe; citation_journal_title=J Gen Virol; citation_volume=78; citation_number=37; citation_pages=3141-3145; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Fully biologically active in vitro transcripts of the eriophyid mite-transmitted wheat streak mosaic tritimovirus.; citation_author=IR Choi; citation_author=R French; citation_author=GL Hein; citation_author=DC Stenger; citation_journal_title=Phytopathology; citation_volume=89; citation_number=38; citation_pages=1182-1185; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Hsp90 chaperonins possess ATPase activity and bind heat shock transcription factors and peptidyl prolyl isomerases.; citation_author=K Nadeau; citation_author=A Das; citation_author=CT Walsh; citation_journal_title=J Biol Chem; citation_volume=268; citation_number=39; citation_pages=1479-1487; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Non-coding RNAs: lost in translation?; citation_author=FF Costa; citation_journal_title=Gene; citation_volume=386; citation_number=40; citation_pages=1-10; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sequencing and characterization of telomere and subtelomere regions on rice chromosomes 1S, 2S, 2L, 6L, 7S, 7L and 8S.; citation_author=H Mizuno; citation_author=J Wu; citation_author=H Kanamori; citation_author=M Fujisawa; citation_author=N Namiki; citation_journal_title=Plant J; citation_volume=46; citation_number=41; citation_pages=206-217; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DDBJ working on evaluation and classification of bacterial genes in INSDC.; citation_author=H Sugawara; citation_author=T Abe; citation_author=T Gojobori; citation_author=Y Tateno; citation_journal_title=Nucleic Acids Res; citation_volume=35(Database issue); citation_number=42; citation_pages=D13-15; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=BLAT: The BLAST-like alignment tool.; citation_author=WJ Kent; citation_journal_title=Genome Res; citation_volume=12; citation_number=43; citation_pages=656-664; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A revised medium for rapid growth and bioassays with tobacco tissue cultures.; citation_author=T Murashige; citation_author=F Skoog; citation_journal_title=Physiol Plant; citation_volume=15; citation_number=44; citation_pages=473-497; citation_date=1962; " />
      <meta name="citation_reference" content="citation_title=NCBI GEO: mining tens of millions of expression profiles–database and tools update.; citation_author=T Barrett; citation_author=DB Troup; citation_author=SE Wilhite; citation_author=P Ledoux; citation_author=D Rudnev; citation_journal_title=Nucleic Acids Res; citation_volume=35(Database issue); citation_number=45; citation_pages=D760-765; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray"/>
    <meta name="twitter:description" content="Rice (Oryza sativa L.) is a model organism for the functional genomics of monocotyledonous plants since the genome size is considerably smaller than those of other monocotyledonous plants. Although highly accurate genome sequences of indica and japonica rice are available, additional resources such as full-length complementary DNA (FL-cDNA) sequences are also indispensable for comprehensive analyses of gene structure and function. We cross-referenced 28.5K individual loci in the rice genome defined by mapping of 578K FL-cDNA clones with the 56K loci predicted in the TIGR genome assembly. Based on the annotation status and the presence of corresponding cDNA clones, genes were classified into 23K annotated expressed (AE) genes, 33K annotated non-expressed (ANE) genes, and 5.5K non-annotated expressed (NAE) genes. We developed a 60mer oligo-array for analysis of gene expression from each locus. Analysis of gene structures and expression levels revealed that the general features of gene structure and expression of NAE and ANE genes were considerably different from those of AE genes. The results also suggested that the cloning efficiency of rice FL-cDNA is associated with the transcription activity of the corresponding genetic locus, although other factors may also have an effect. Comparison of the coverage of FL-cDNA among gene families suggested that FL-cDNA from genes encoding rice- or eukaryote-specific domains, and those involved in regulatory functions were difficult to produce in bacterial cells. Collectively, these results indicate that rice genes can be divided into distinct groups based on transcription activity and gene structure, and that the coverage bias of FL-cDNA clones exists due to the incompatibility of certain eukaryotic genes in bacteria."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001235.g002"/>

  <meta property="og:title" content="Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6309'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1232'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=840&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001235">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001235" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001235&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kouji Satoh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Koji Doi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Toshifumi Nagata, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Naoki Kishimoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kohji Suzuki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Hitachi Software Engineering, Shinagawa-ku, Tokyo, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yasuhiro Otomo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Genome Sequencing and Analysis Group, Foundation for Advancement of International Science (FAIS), Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jun Kawai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan, 
                  Genome Science Laboratory, RIKEN Wako Institute, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mari Nakamura, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tomoko Hirozane-Kishikawa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Saeko Kanagawa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takahiro Arakawa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juri Takahashi-Iida, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mitsuyoshi Murata, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Noriko Ninomiya, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daisuke Sasaki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shiro Fukuda, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michihira Tagami, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Harumi Yamagata, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kanako Kurita, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kozue Kamiya, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mayu Yamamoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ari Kikuta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takahito Bito, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nahoko Fujitsuka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kazue Ito, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiroyuki Kanamori, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of the Society for Techno-innovation of Agriculture, Forestry and Fisheries, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Il-Ryong Choi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Plant Breeding, Genetics and Biotechnology Division, International Rice Research Institute, DAPO, Metro Manila, Philippines
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshiaki Nagamura, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takashi Matsumoto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kazuo Murakami, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Genome Sequencing and Analysis Group, Foundation for Advancement of International Science (FAIS), Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ken-ichi Matsubara, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratory of Genome Sequencing and Analysis Group, Foundation for Advancement of International Science (FAIS), Tsukuba, Ibaraki, Japan, 
                  Nara Institute of Science and Technology (NAIST), Ikoma, Nara, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Piero Carninci, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan, 
                  Genome Science Laboratory, RIKEN Wako Institute, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoshihide Hayashizaki, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Genome Exploration Research Group, Genomic Sciences Center, RIKEN Yokohama Institute, Yokohama, Kanagawa, Japan, 
                  Genome Science Laboratory, RIKEN Wako Institute, Wako, Saitama, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shoshi Kikuchi
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:skikuchi@nias.affrc.go.jp">skikuchi@nias.affrc.go.jp</a></p>

                <p>Affiliation:
                  Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, Tsukuba, Ibaraki, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001235</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001235-t001" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t002" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-g001" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t003" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t004" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t005" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t006" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t007" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t008" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t008" title="Table 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t009" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t009" title="Table 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-t010" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t010" title="Table 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001235-g002" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001235">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001235" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Rice (<em>Oryza sativa</em> L.) is a model organism for the functional genomics of monocotyledonous plants since the genome size is considerably smaller than those of other monocotyledonous plants. Although highly accurate genome sequences of <em>indica</em> and <em>japonica</em> rice are available, additional resources such as full-length complementary DNA (FL-cDNA) sequences are also indispensable for comprehensive analyses of gene structure and function. We cross-referenced 28.5K individual loci in the rice genome defined by mapping of 578K FL-cDNA clones with the 56K loci predicted in the TIGR genome assembly. Based on the annotation status and the presence of corresponding cDNA clones, genes were classified into 23K annotated expressed (AE) genes, 33K annotated non-expressed (ANE) genes, and 5.5K non-annotated expressed (NAE) genes. We developed a 60mer oligo-array for analysis of gene expression from each locus. Analysis of gene structures and expression levels revealed that the general features of gene structure and expression of NAE and ANE genes were considerably different from those of AE genes. The results also suggested that the cloning efficiency of rice FL-cDNA is associated with the transcription activity of the corresponding genetic locus, although other factors may also have an effect. Comparison of the coverage of FL-cDNA among gene families suggested that FL-cDNA from genes encoding rice- or eukaryote-specific domains, and those involved in regulatory functions were difficult to produce in bacterial cells. Collectively, these results indicate that rice genes can be divided into distinct groups based on transcription activity and gene structure, and that the coverage bias of FL-cDNA clones exists due to the incompatibility of certain eukaryotic genes in bacteria.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Satoh K, Doi K, Nagata T, Kishimoto N, Suzuki K, et al.  (2007) Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray. PLoS ONE 2(11):
          e1235.
            doi:10.1371/journal.pone.0001235</p><p><strong>Academic Editor: </strong>Markus Grebe, Umeå Plant Science Centre, Sweden</p><p><strong>Received:</strong> July 17, 2007; <strong>Accepted:</strong> October 30, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Satoh et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by a Rice Genome Full-Length cDNA Library Construction Project grant from BRAIN and by the Rice Genome Simulator Project from the Ministry of Agriculture, Forestry and Fisheries (MAFF), Japan; additional funding was received through a QTL project on Green Techno from MAFF and the Generation Challenge program funded by several international donors of the Consultative Group for International Agricultural Research (CGIAR).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Rice (<em>Oryza sativa</em> L.) is a model organism for functional genomics of monocot plants because the genome size is smaller than those of other monocot plants. Highly accurate genome sequences of rice are available <a href="#pone.0001235-Goff1">[1]</a>–<a href="#pone.0001235-Yu1"></a>,<a href="#pone.0001235-Yu2"></a>,<a href="#pone.0001235-International1">[4]</a>. Genome sequences of rice (<em>O. sativa</em> ssp. <em>japonica</em> cv. Nipponbare) have been assembled independently by The Institute for Genome Research (TIGR), International Rice Genome Sequencing Project (IRGSP), and Beijing Genomics Institute (BGI). In addition, full-length complementary DNA (FL-cDNA) sequences <a href="#pone.0001235-Rice1">[5]</a> and expressed sequence tags (ESTs) <a href="#pone.0001235-Wu1">[6]</a>–<a href="#pone.0001235-Zhou1"></a>,<a href="#pone.0001235-Zhang1">[8]</a> from rice have served as valuable resources for genomic and genetic studies.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The rice genome sequences assembled by TIGR (TIGR Pseudomolecules) have been revised to Pseudomolecules Release 4 (TIGR4) <a href="#pone.0001235-Ouyang1">[9]</a>. TIGR4 was subsequently used to improve the accuracy of predicted gene loci, model genes and gene annotation. (see TIGR OSA1: <a href="http://www.tigr.org/tdb/e2k1/osa1/index.shtml">http://www.tigr.org/tdb/e2k1/osa1/index.​shtml</a>) <a href="#pone.0001235-Ouyang1">[9]</a>. In TIGR OSA1, the transcriptome information compiled from numerous data of FL-cDNA, ESTs, Massively Parallel Signature Sequencing (MPSS), and Serial Analysis of Gene Expression (SAGE) were integrated with proteome information. This comprehensive information eventually led to the confirmation of gene expression at 24 799 loci. The genes in TIGR OSA1 were annotated by semi-automatic and manual methods. The OSA1 information has been used for gene classification and assimilation into physical map data <a href="#pone.0001235-Jaiswal1">[10]</a>, <a href="#pone.0001235-RianoPachon1">[11]</a> and has accelerated genomic analysis of rice <a href="#pone.0001235-Ren1">[12]</a>, <a href="#pone.0001235-Lin1">[13]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Builds 3 and 4 of IRGSP Pseudomolecules (IRGSP3 and IRGSP4) are other versions of the rice genome sequences <a href="#pone.0001235-International1">[4]</a> available at the Rice Annotation Project Database (RAP-DB: <a href="http://rapdb.lab.nig.ac.jp/">http://rapdb.lab.nig.ac.jp/</a>) <a href="#pone.0001235-Ohyanagi1">[14]</a>, <a href="#pone.0001235-Rice2">[15]</a>. Gene loci in IRGSP3 and IRGSP4 were determined using only cDNA sequences (FL-cDNA and ESTs, combined ESTs) derived from rice and other cereals. Gene loci in RAP-DB were verified by cDNA analysis, but other predicted loci lacking cDNA support were excluded from the database. The genes in IRGSP3 and IRGSP4 were annotated manually.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Both TIGR Pseudomolecules and IRGSP Pseudomolecules assembled from the same source of bacterial and P1 artificial chromosome clones were constructed by map-based methods. The rice genome sequences for cultivars 93-11 and Nipponbare assembled by BGI were determined by the whole-genome shotgun method (available at RISe: <a href="http://rise.genomics.org.cn/">http://rise.genomics.org.cn/</a>) <a href="#pone.0001235-Zhao1">[16]</a>. The genome of 93-11 (<em>O. sativa</em> ssp. <em>indica</em>) was sequenced by BGI <a href="#pone.0001235-Yu1">[2]</a>, whereas that of Nipponbare was done by Syngenta <a href="#pone.0001235-Goff1">[1]</a>. The RISe database provides predicted gene loci, expression data from genome tiling arrays, and the mapped positions of FL-cDNA and ESTs <a href="#pone.0001235-Zhao1">[16]</a>, <a href="#pone.0001235-Li1">[17]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>FL-cDNA collections have been established for several organisms <a href="#pone.0001235-Seki1">[18]</a>–<a href="#pone.0001235-The1">[20]</a>. The cDNA information was extensively used to determine gene annotation, structures, and the start and the end sites of transcription. The cDNA information also is indispensable for validation of gene function by reverse genetics. The FL-cDNA collection for rice also has been used to complement the information obtained by genome sequencing. The first report of a rice FL-cDNA collection published in 2003 <a href="#pone.0001235-Rice1">[5]</a> described the characteristics and annotation of 28K FL-cDNA sequences. The collection has eventually expanded to 578K FL-cDNA clones, among which 35K cDNA clones were completely sequenced and annotated (available at Knowledge-based Oryza Molecular biological Encyclopedia (KOME): <a href="http://cdna01.dna.affrc.go.jp/cDNA/">http://cdna01.dna.affrc.go.jp/cDNA/</a>).</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The informative data from MPSS <a href="#pone.0001235-Nakano1">[21]</a>, SAGE <a href="#pone.0001235-Gibbings1">[22]</a>, <a href="#pone.0001235-Hashimoto1">[23]</a>, and genome tiling arrays <a href="#pone.0001235-Li1">[17]</a> can serve as additional resources complementary to the rice genome sequences, although the individual approaches may not be able to provide information entirely sufficient for gene prediction. MPSS and SAGE provide supportive evidence for gene expression. The two approaches may identify the start or end of transcription sites, but cannot reveal gene structures. Genome tilling arrays can define exon-intron structures <a href="#pone.0001235-Li1">[17]</a>, <a href="#pone.0001235-Yamada1">[24]</a>, but not the start and end of individual transcripts or alternative transcripts. In contrast, as already described, the large collection of annotated FL-cDNA sequences of rice available from several databases has been used as a comprehensive resource for various analyses of gene structure and function.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>The main objective of this report is to more completely describe the features of the FL-cDNA sequences mapped in the rice genome. We examined global characteristics such as the coverage of full-length cDNA clones among predicted rice genes, and the structural and expression characteristics of genes which were not identified by the gene prediction programs. The results suggest that the cloning efficiency of the FL-cDNA was greatly influenced by transcriptional activity and the structure of the encoded gene. In addition, the results also revealed that the levels of transcription activity are significantly associated with the position within the genome and the structure of individual genes, and that protein domains encoded in individual genes may act as a significant factor causing bias in cloning of FL-cDNA.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>1. Results of mapping cDNA to five currently available genome assemblies</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Five hundred seventy eight thousand rice (ssp. <em>japonica</em>, cv. Nipponbare) FL-cDNA clones (DDBJ accession number: completely sequenced FL-cDNA : AK058203–068528, AK068530–068912, AK068914–70720, AK070722–074028, AK098843–112119, AK119160–122186, AK240633–243692, One-pass sequences of FL-cDNA, 5′ end: CI285358-311811 and CI563340–778739, 3′end: CI000001-285357 and CI311812–563339) were mapped to five genome assemblies: TIGR4, IRGSP3, IRGSP4, the Nipponbare genome determined by Syngenta, and the 93-11 genome by BGI (<a href="#pone-0001235-t001">Table 1</a>, <a href="#pone.0001235.s003">S1</a>). The mapping criteria were &gt;95% identity and &gt;90% coverage. For proper comparison between the assemblies, the results of mapped cDNA for the respective assemblies were compared with those for TIGR4. The numbers of mapped FL-cDNA clones differed among assemblies, with the highest in TIGR4. For some cDNA clones, the orientation in which the clone was mapped onto a chromosome and the chromosome on which the clones were mapped were not consistent among the assemblies.</p>
<div class="figure" id="pone-0001235-t001"><div class="img"><a name="pone-0001235-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>FL-cDNA clones mapped to five rice genome assemblies.</span></strong></p><span>doi:10.1371/journal.pone.0001235.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Of the 32 775 completely sequenced FL-cDNA clones mapped in TIGR4, 29 925 were also mapped in all of the other assemblies, however, the number of clones commonly mapped in both TIGR4 and a given assembly differed (<a href="#pone-0001235-t001">Table 1</a>). The maximum and minimum numbers of common clones were 32 730 in IRGSP4 and 30 162 in 93-11, respectively. The number of mapped clones was greater in the <em>japonica</em> rice genomes than in the <em>indica</em> genome which might reflect differences in the genome sequences between subspecies. The number of common clones between TIGR4 and IRGSP4 was close to that between TIGR4 and IRGSP3, and both numbers were greater than the number of common clones between TIGR4 and the Syngenta sequence. This suggests that the differences in numbers of common clones may have resulted from differences in sequencing methods adopted in the assemblies (TIGR4 and IRGSP by the map-based method; Syngenta sequence by the whole-genome shotgun method).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Mapping of 578K FL-cDNA clones identified about 28 500 loci in the <em>japonica</em> genome and 27 800 loci in the <em>indica</em> genome. A total of 29 925 completely sequenced FL-cDNAs were mapped in any genome assemblies and more than 90% of the FL-cDNAs were mapped in all five assemblies (<a href="#pone-0001235-t001">Table 1</a>). So, we decided to use only the mapping results of TIGR4 for further analyses and to not use results from the other assemblies. The number of predicted loci was about 56K which was sufficient for our data analysis, but probably not sufficient to reach complete accuracy of gene prediction and annotation of TIGR4.</p>


<h4>2. Classification of loci according to FL-cDNA mapping</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A total of 55 890 gene loci were predicted in the rice genome according to TIGR OSA1 release 4. Mapping of FL-cDNA clones on TIGR4 revealed that 533 667 FL-cDNA clones were derived from 28 564 FL-cDNA loci (<a href="#pone-0001235-t002">Table 2</a>). FL-cDNA loci were cross-referenced with TIGR4 loci to examine the overlaps between the two locus groups. According to the sources of mapped loci and the occurrence of overlap, the loci were classified as follows: 1) when a FL-cDNA locus overlapped with a TIGR4 locus, the FL-cDNA locus was defined as FL-AE (annotated expressed), and the TIGR4 locus was defined as coding-sequence-AE (CDS-AE); 2) a FL-cDNA locus that did not overlap with any TIGR4 locus was defined as FL-NAE (non-annotated expressed); and 3) a TIGR4 locus that did not overlap with any FL-cDNA locus was defined as CDS-ANE (annotated non-expressed). Based on the definitions, the loci were classified into 23 117 FL-AE, 23 193 CDS-AE, 5 447 FL-NAE, and 32 697 CDS-ANE (<a href="#pone-0001235-t002">Tables 2</a>, <a href="#pone.0001235.s004">S2a</a>).</p>
<div class="figure" id="pone-0001235-t002"><div class="img"><a name="pone-0001235-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Comparisons of FL-cDNA loci and TIGR4 loci</span></strong></p><span>doi:10.1371/journal.pone.0001235.t002</span></div>

<h4>3. Characterization of loci based on FL-cDNA mapping</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The classification of loci as defined above raised questions concerning whether any characteristic distinctions exist between FL-AE and FL-NAE, and why FL-NAE loci were not predicted in TIGR4. To answer these questions, we analyzed the structures of genes belonging to the respective groups.</p>


<h4>Open reading frame of FL-cDNA clones mapped on genome</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We mapped 32 775 FL-cDNA at 22 943 FL-cDNA loci (FL-AE, 20 324; FL-NAE, 2619) (<a href="#pone.0001235.s004">Table S2b</a>). The numbers of FL-cDNA mapped to FL-AE and FL-NAE were 29 808 and 2967, respectively (<a href="#pone-0001235-t002">Table 2</a>). The median lengths of the FL-cDNA mapped to FL-AE and FL-NAE were 1540 and 1173 bp, respectively (<a href="#pone-0001235-g001">Figure 1a</a>, <a href="#pone-0001235-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001235-g001"><div class="img"><a name="pone-0001235-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Gene structure analysis in rice.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(a) The length distribution of FL-cDNA for FL-AE (black) and FL-NAE (white). (b) The distribution of open reading frame (ORF) proportions for FL-AE (black) and FL-NAE (white). (c) The distribution of FL-cDNA locus lengths for FL-AE (black) and FL-NAE (white). (d) The distribution of locus lengths for CDS-AE (black) and CDS-ANE (white) in TIGR4. (e) The distribution of the number of exons for FL-AE (black) and FL-NAE (white). (f) The distribution of exon (black) and intron (white) lengths for the respective locus types. (g) The distribution of the number of FL-cDNA clones mapped per single FL-AE (black) and FL-NAE (white) locus.</p>
<span>doi:10.1371/journal.pone.0001235.g001</span></div><div class="figure" id="pone-0001235-t003"><div class="img"><a name="pone-0001235-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Structural characteristics of locus types</span></strong></p><span>doi:10.1371/journal.pone.0001235.t003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The proportions of the length of the longest open reading frames (ORF) to that of FL-cDNA were also considerably different between FL-cDNA mapped to FL-AE and FL-NAE. The median proportion of the longest ORF in FL-AE was 66%, versus only 21% in FL-NAE (<a href="#pone-0001235-g001">Figure 1b</a>, <a href="#pone-0001235-t003">Table 3</a>). The results indicate that FL-NAE clones generally encode shorter peptides, and that the ORF lengths differ considerably between FL-AE and FL-NAE (based on the clone length×ORF proportion).</p>


<h4>Locus length</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The start and end sites for transcription were determined in 24 164 loci (FL-AE, 21 263; FL-NAE, 2901) out of 28 564 FL-cDNA loci. The distance between the start and end sites (i.e., locus length) was calculated based on TIGR4. The median locus lengths were considerably different between FL-AE and FL-NAE, with lengths of 3354 and 1727 bp, respectively (<a href="#pone-0001235-g001">Figure 1c</a>, <a href="#pone-0001235-t003">Table 3</a>). The average proportion of locus length to FL-cDNA clone length was greater than 2 for FL-AE, but less than 1.5 for FL-NAE.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The median locus lengths for CDS-AE and CDS-ANE were 3173 and 1643 bp, respectively (<a href="#pone-0001235-g001">Figure 1d</a>). The locus lengths of CDS-ANE appeared to be more variable than those of CDS-AE. The patterns of locus length variation for CDS-AE and CDS-ANE were similar to those for FL-AE and FL-NAE, respectively (<a href="#pone-0001235-g001">Figure 1c,d</a>, <a href="#pone-0001235-t003">Table 3</a>). We could not compare the length of FL-cDNA loci with the TIGR4 loci because FL-cDNAs were constructed from coding and 5′ and 3′ end untranslated regions, whereas many gene structures in TIGR4 were predicted only with coding regions. However, based on the results above, we expect that FL-AE and FL-NAE would differ from each other in this characteristic, as would CDS-AE and CDS-ANE.</p>


<h4>Exon-intron structure</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The average numbers of exons per FL-AE and FL-NAE were 5.3 and 2.3, respectively (<a href="#pone-0001235-g001">Figure 1e</a>, <a href="#pone-0001235-t003">Table 3</a>). The frequency of loci with a single exon was highest in both FL-AE and FL-NAE when the loci were distributed according to the numbers of exons. However, the proportions of loci with a single exon were significantly different between FL-AE and FL-NAE, with more than 50% in FL-NAE, and about 20% in FL-AE (<a href="#pone-0001235-g001">Figure 1e</a>). When FL-cDNA loci with one exon were excluded, the ratios between the lengths of exons and introns in individual loci were approximately 1 irrespective of the locus type, but exon and intron lengths were significantly different (P&lt;0.01, calculated by student's t-test) between FL-AE and FL-NAE with median lengths of about 150 and 250 bp, respectively (<a href="#pone-0001235-g001">Figure 1f</a>, <a href="#pone-0001235-t003">Table 3</a>).</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We also analyzed the exon-intron structures of CDS-AE and CDS-ANE based on the information at TIGR OSA1. The number of exons was higher in CDS-AE (5.8) than in CDS-ANE (3.9) (<a href="#pone-0001235-t003">Table 3</a>). The exon and intron lengths also significantly differed (P&lt;0.01, calculated by student's t-test) between CDS-AE and CDS-ANE with median exon lengths of 147 and 199 bp, and median intron lengths of 162 and 186 bp, respectively (<a href="#pone-0001235-g001">Figure 1f</a>, <a href="#pone-0001235-t003">Table 3</a>).</p>


<h4>Number of mapped FL-cDNA clones at a locus</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>FL-NAE accounted for about 19% of the entire FL-cDNA loci, while the proportion of FL-cDNA clones mapped as FL-NAE was only 5% (21 850) of all mapped FL-cDNA clones (533 667) (<a href="#pone-0001235-t002">Table 2</a>). The average numbers of FL-cDNA clones mapped per locus (collection efficiency) were significantly different between FL-AE (22.3 clones) and FL-NAE (4.1) (<a href="#pone-0001235-g001">Figure 1g</a>, <a href="#pone-0001235-t003">Table 3</a>). The collection efficiency was 1 for more than half of the FL-NAE loci (<a href="#pone-0001235-g001">Figure 1g</a>, <a href="#pone-0001235-t003">Table 3</a>), suggesting that FL-cDNA clones derived from FL-NAE are more difficult to collect than those from FL-AE.</p>


<h4>Gene annotation</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We analyzed the homology of FL-cDNA mapped on TIGR4 with Arabidopsis CDS in The Arabidopsis Information Resource (TAIR6, <a href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>) <a href="#pone.0001235-Rhee1">[25]</a> using the BlastX software. Based on the significance of similarity, FL-cDNA clones were classified into highly-homologous (E-value&lt;10<sup>−50</sup>), low-homologous (10<sup>−50</sup>&lt;E-value&lt;10<sup>−10</sup>) and non-homologous FL-cDNA (E-value&gt;10<sup>−10</sup>). Under these criteria, the numbers of FL-cDNA clones classified as highly-, low-, and non-homologous FL-cDNA were 17 759, 7103, and 7913, respectively. Of these clones, 99.5% of highly-homologous FL-cDNA were mapped to 59% of FL-AE, and 92% of FL-NAE were coding non-homologous genes (<a href="#pone-0001235-t004">Table 4</a>). Thus, the results indicate that nearly all highly-homologous genes were derived from FL-AE, and that most FL-NAE loci encode genes likely to be specific to rice or other monocots. This is consistent with a previous report that some rice genes which do not have homologs in Arabidopsis are similar to genes in the <em>Sorghum</em> genome <a href="#pone.0001235-Yu2">[3]</a>.</p>
<div class="figure" id="pone-0001235-t004"><div class="img"><a name="pone-0001235-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The frequency of Arabidopsis homologus gene in each FL-locus</span></strong></p><span>doi:10.1371/journal.pone.0001235.t004</span></div>

<h4>Causes of inconsistency between gene prediction and FL-cDNA mapping</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Cross-examination between FL-cDNA and TIGR4 loci revealed the existence of FL-NAE clones. The results from our analyses for the locus structures of FL-AE and FL-NAE suggest possible explanations as to why some expressed genes were not annotated. One possible reason is the characteristic length of ORFs in the different classes of loci. The proportion of the longest ORF in FL-NAE (median ratio of 21%) is significantly lower than in FL-AE (median ratio of 66%) indicating that the transcripts from FL-NAE are more likely to encode either small peptides or no peptide. In the TIGR OSA1, 687 CDS encoding less than 50 amino acids were excluded from the predicted gene model. Thus, even though the number of excluded CDS was less than the number of FL-NAE, FL-cDNA sequences overlapping with the excluded CDS in TIGR4 might have been mapped as FL-AE.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>Another possible reason may be the difference in exon-intron structure between FL-NAE and FL-AE. The locus lengths of FL-NAE were generally shorter than those of FL-AE, and more than half of the FL-NAE loci contain only one exon (<a href="#pone-0001235-g001">Figure 1e</a>). Meanwhile, the lengths of both exon and intron in FL-NAE were generally longer than those in FL-AE, CDS-AE and CDS-ANE (<a href="#pone-0001235-g001">Figure 1f</a>). If we consider the structure of FL-AE, CDS-AE and CDS-ANE as the standard type for rice genes, then the structure of many FL-NAE loci may be recognized as an irregular form. Due to the unique features of ORFs and the exon-intron structures in FL-NAE, it may have been difficult to assign proper annotation to the genes in FL-NAE through gene prediction software.</p>
<a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3"></a><p>The structural difference between FL-AE and NAE is also similar to that between protein-coding mRNA and mRNA-like non-coding RNA (ncRNA) in mouse <a href="#pone.0001235-Ravasi1">[26]</a>. In mouse, the total length of ncRNA is shorter and the exon length is longer than in mRNA. Moreover, more than 70% of ncRNA is constructed from one exon. In our classification, FL-AE overlaps with predicted CDS that encode&gt;50 amino acid sequences, so cDNA mapped on FL-AE originates from protein-encoding mRNA. In addition, the diversity of FL-cDNA lengths between FL-AE and FL-NAE is not large and the proportions of ORF between FL-AE and FL-NAE are reversed (<a href="#pone-0001235-g001">Figure 1a,b</a>, <a href="#pone-0001235-t003">Table 3</a>). So, it seems that the proportion of protein-coding FL-NAE loci is not large and many FL-NAE loci encode mRNA-like ncRNA. Therefore, the structural diversity between FL-AE and FL-NAE may correspond to the difference between protein-coding mRNA and mRNA-like ncRNA in rice. In addition, these results may indicate that the structural difference between protein-coding mRNA and mRNA-like ncRNA are also conserved between plant and mammals.</p>
<a id="article1.body1.sec2.sec9.p4" name="article1.body1.sec2.sec9.p4"></a><p>The gene loci in TIGR4 were categorized into three types by cross-examination between FL-cDNA loci and TIGR4 loci. The collection efficiency varied considerably depending on the locus types (FL-AE = 22.3; FL-NAE = 4.1; CDS-ANE = 0). The difference may be associated with the levels of mRNA or the transcription activity of each locus type. Moreover, the general features of locus structure and the average homology levels with Arabidopsis genes were distinctively different among the locus types. Thus, these findings may indicate an interrelationship among locus structure, transcription activity and the assignability of gene annotation. An association of transcription activity with locus structure has been reported in plants <a href="#pone.0001235-Ren1">[12]</a>: highly expressed genes have longer primary transcript, ORF, exon and intron sequences and have more exons than low expressed genes. The results from the previous report <a href="#pone.0001235-Ren1">[12]</a> are consistent with our hypothesis that locus structure affects transcription activity, except that the results for intron length differed from our results. The locus length of FL-AE was greater than that of FL-NAE and the collection efficiency of FL-AE was also greater than that of FL-NAE. Moreover, the cloning efficiency of mRNA-like ncRNA in mouse is lower than for protein-coding mRNA and half of the ncRNA in mouse has an efficiency of 1 <a href="#pone.0001235-Ravasi1">[26]</a>. Cloning efficiency features in mouse are also similar to those of FL-NAE which may imply that many FL-NAE loci encode mRNA-like ncRNA. Transcription activity of protein-coding mRNA (FL-AE) is presumed to be higher than that of ncRNA (FL-NAE) and the transcriptional diversity has been confirmed by microarray analysis (see below).</p>
<a id="article1.body1.sec2.sec9.p5" name="article1.body1.sec2.sec9.p5"></a><p>We analyzed the diversity between FL-AE and FL-NAE identified from FL-cDNA mapping and found some differences between FL-AE and FL-NAE. The difference between FL-AE and FL-NAE is similar to that between protein-coding mRNA and mRNA-like ncRNA. In our classification, FL-AE implies a protein-coding locus and FL-NAE is an ncRNA-coding locus which might explain why FL-NAE loci are not predicted in TIGR4. Although gene prediction software can identify loci that encode proteins, it does not detect loci that are transcribed into ncRNA. Therefore, prediction software can not find a FL-NAE locus that encodes ncRNA.</p>


<h4>4. Validation of gene expression on loci</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Transcription from 23K TIGR loci was confirmed by the mapping of FL-cDNA, while another 33K TIGR loci were not confirmed. It seemed that the FL-cDNA libraries did not cover all the rice genes in the genome since the transcription of 3K CDS-ANE was confirmed by other transcriptome resources in TIGR OSA1. To obtain a more accurate evaluation of gene expression from FL-cDNA loci and TIGR4 loci, we designed about 82K 60-mer oligo probes based on TIGR Pseudomolecules Release 2. The TIGR4 loci were remapped according to the revised Pseudomolecules and other additional transcriptome resources such as EST, MPSS and SAGE. The 82K designed probes were remapped in TIGR4 according to our criteria (i.e.&gt;95% identity, &gt;90% coverage, non-redundant highest score). As a result, 63 455 probes were mapped at 43 090 loci (FL-AE, 21 885; FL-NAE, 3540; CDS-ANE, 17 665, see <a href="#pone.0001235.s005">Table S3</a>).</p>
<a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2"></a><p>RNA samples were prepared from shoots, roots, panicles, and calluses of Nipponbare (<em>japonica</em> variety) and shoots of IR64 (<em>indica</em> variety), and the signal intensities were compared among the samples (GEO accession number: GSE7366 and GSE7374). The median and average signal intensities for each locus type were similar among the samples from different tissues of Nipponbare, but the intensity values for the samples from IR64 were generally lower than those from Nipponbare (<a href="#pone.0001235.s005">Table S3</a>). The difference in signal intensities between the samples from Nipponbare and IR64 could be due to the variations in the genome sequence. The apparent signal intensities in the samples from IR64 may have been affected by transcription activity and the variations in the genome sequence; therefore, the data from IR64 were not used in further analyses.</p>
<a id="article1.body1.sec2.sec10.p3" name="article1.body1.sec2.sec10.p3"></a><p>A significant signal (see <a href="#s3">Materials and Methods</a>) was detected at 93% of the loci identified by the designed probes in at least one of the samples from Nipponbare. The numbers of loci with a significant signal were 21702, 3357, and 14 892 for FL-AE, FL-NAE, and CDS-ANE, respectively (<a href="#pone.0001235.s005">Table S3</a>). The median signal intensity varied among the locus types, with the highest intensity for AE (about 2403), followed by NAE (839) and ANE (237, only about 10% of the AE intensity) (<a href="#pone.0001235.s005">Table S3</a>). The 25% and 75% percentiles of signal intensities for FL-AE were about 800 and 6900, respectively, and the corresponding values for CDS-ANE were 0 and 700, respectively. Moreover, the observed signal profiles showing that the intensity of FL-NAE is lower than that of FL-AE also supports the hypothesis that many FL-NAE code mRNA-like ncRNA <a href="#pone.0001235-Ravasi1">[26]</a></p>
<a id="article1.body1.sec2.sec10.p4" name="article1.body1.sec2.sec10.p4"></a><p>Examination of the median signal intensity and the collection efficiency for each locus type revealed a positive correlation between the signal intensity and the collection efficiency in both FL-AE (R = 0.99) and FL-NAE (R = 0.86) (<a href="#pone-0001235-t005">Table 5</a>). The median signal intensities in FL-NAE were consistently lower than those in FL-AE when the collection efficiencies for the two locus types were within the same range. This positive correlation between signal intensity and cloning efficiency suggests it was difficult to create FL-cDNA clones from CDS-ANE transcripts, because the median signal intensity of CDS-ANE (238) was lower than those of FL-AE and FL-NAE (FL-AE: 852, FL-NAE: 596 in range of collection efficiency 1-2).</p>
<div class="figure" id="pone-0001235-t005"><div class="img"><a name="pone-0001235-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Signal intensities in microarray analysis in relation to the locus types and the collection efficiencies of FL-cDNA.</span></strong></p><span>doi:10.1371/journal.pone.0001235.t005</span></div><a id="article1.body1.sec2.sec10.p5" name="article1.body1.sec2.sec10.p5"></a><p>The results from microarray analysis indicated that genes identified with the designed probes are expressed at a significant level in at least one of the tissues examined (<a href="#pone-0001235-t005">Table 5</a>), although the possibility of cross-hybridization can not be excluded. To avoid misinterpretation of results due to potential cross hybridization effects, we examined the numbers of loci with significant signal intensity at higher threshold values (significant signal intensity&gt;1000). The number of loci with a significant signal intensity decreased for all locus types as the threshold levels increased. At the threshold, 85% of FL-AE loci (18.6K loci) exhibited a significant signal in at least one of the samples from different tissues, while only 58% (2K loci) of FL-NAE and 27% (4.6K loci) of CDS-ANE exhibited a significant signal. This difference was expected since the median intensities were significantly different among the locus types (<a href="#pone.0001235.s005">Table S3</a>). The difference in the numbers of loci with significant signals among the samples from different tissues was not large (<a href="#pone-0001235-t006">Table 6</a>). The number of loci with a significant signal in respective tissues was also similar to the numbers of loci showing significant signals in all four samples (<a href="#pone-0001235-t006">Table 6</a>). A similar tendency was observed for the number of loci classified as no-significant signal (<a href="#pone-0001235-t006">Table 6</a>). These results suggested that some loci in rice are consistently highly expressed and that some loci are consistently expressed at a low level or perhaps not expressed at all.</p>
<div class="figure" id="pone-0001235-t006"><div class="img"><a name="pone-0001235-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>The number of significant signal-detected loci in each locus type</span></strong></p><span>doi:10.1371/journal.pone.0001235.t006</span></div>

<h4>Relationship between transcription activity and locus position within the genome</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>It was previously reported that human housekeeping genes (e.g., tissue-non-specific genes) were mapped to clusters in the genome <a href="#pone.0001235-Caron1">[27]</a>, <a href="#pone.0001235-Lercher1">[28]</a>. In our data (<a href="#pone-0001235-t006">Table 6</a>), most FL-AE appeared as consistently-highly expressed genes, while expression of a large portion of CDS-ANE was consistently low. Therefore, we hypothesized that the respective locus types may exhibit a certain distribution pattern in the rice genome. To test the hypothesis, we examined the frequency of loci belonging to each locus type (FL-AE, FL-NAE, and CDS-ANE) along the genome using a sliding-window analysis (<a href="#pone.0001235.s004">Table S2a</a>). The results of the analysis for the respective locus types and TE-related loci are presented in <a href="#pone.0001235.s001">Figure S1</a>. FL-AE and CDS-ANE were not equally distributed within the genome, and there appears to be a negative correlation between the frequency of FL-AE and CDS-ANE (R = −0.97). The frequency of FL-AE decreased in regions rich in TE-related loci (R = −0.82); whereas, the frequency of CDS-ANE loci increased in these regions (R = 0.79). FL-NAE loci appeared more evenly distributed along the genome, and thus the frequency of FL-NAE did not show any correlations with those of FL-AE and TE-related loci (R = −0.24 and 0.21, respectively). TE-rich regions are generally heterochromatin or centromeric regions <a href="#pone.0001235-Yan1">[29]</a>. Although transcripts from centromeres and heterochromatin in plants have been reported <a href="#pone.0001235-Yu2">[3]</a>, <a href="#pone.0001235-Yan1">[29]</a>–<a href="#pone.0001235-Mizuno1">[32]</a>, these regions appear to be mostly silent with low transcription activity <a href="#pone.0001235-Yu2">[3]</a>. In addition, centromere and heterochromatic regions of Arabidopsis are highly methylated and pseudogenes which contain homology to coding sequences of transposons also contain many methyleted cytosines. The transcription activity of the gene located on highly methylated region is low <a href="#pone.0001235-Zhang2">[33]</a>. From these findings in Arabidopsis, it can be predicted that transcription activity also is suppressed in the centromere and heterochromatic regions in rice.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>We also analyzed the relationship between TE density and transcription activities in FL-AE, FL-NAE and CDS-ANE (<a href="#pone.0001235.s006">Table S4</a>). A negative association between the density of TE-related loci and transcription activity was observed for FL-AE and CDS-ANE, but not for FL-NAE. This result implies transcription activity of many genes may also be influenced by chromatin structure (as TE-density), but FL-NAE may not be affected.</p>
<a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>Microarray experiments can directly indicate transcription activity that is not detected by FL-cDNA cloning. Microarray experiments revealed the following: 1) Almost all FL-cDNA clones were derived from transcripts mapped to euchromatic regions; 2) Cloning efficiency was positively associated with transcription activity; 3) Transcription activity was affected by chromatin structure, but the activity of FL-NAE was not affected which implies that regulation of transcription activity may be different between FL-AE and FL-NAE; and 4) Although the signal intensity of 25% CDS-ANE exceeded 1000 in at least one tissue, these genes were not obtained as cDNA clones. That is, cloning efficiency is not only dependent on transcription activity, but also is influenced by other factors.</p>


<h4>5. Other bias for full-length cDNA cloning</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>The results presented above suggest the efficiency of FL-cDNA cloning may be dependent on transcription activity to a significant degree. The transcription activity of FL-AE and FL-NAE loci was higher than that of CDS-ANE loci; thus, FL-cDNA from CDS-ANE loci could be difficult to obtain. However, microarray analyses revealed that high signal intensity also was detected from about 15% of CDS-ANE loci (2.6K) in samples from all four tissues (<a href="#pone-0001235-t006">Table 6</a>), suggesting that factors other than transcription activity may affect cloning efficiency. To reveal factors which may be involved, we investigated the cloning efficiency of FL-cDNA from gene groups classified on the basis of their functions.</p>


<h4>Relationship between cloning efficiency and gene function</h4>
<a id="article1.body1.sec2.sec13.p1" name="article1.body1.sec2.sec13.p1"></a><p>We examined the coverage of FL-cDNA clones for genes encoding transcription factors (TF) and enzymes for metabolic pathways to see whether there are considerable differences among gene families. The coverage of FL-cDNA clones was determined as the ratio of CDS-AE loci to all CDS loci (hereafter designated as CDS-AE proportion). Based on the Rice Transcription Factor Database version 2.1 (<a href="http://ricetfdb.bio.uni-potsdam.de/">http://ricetfdb.bio.uni-potsdam.de/</a>) <a href="#pone.0001235-RianoPachon1">[11]</a>, 1903 TIGR4 CDS were classified as TF (TF-CDS) representing 65 gene families. Transcription of 1432 TF-CDS loci was supported by detection of the corresponding FL-cDNA clones (CDS-AE). The CDS-AE proportion of the entire TF-CDS was about 75%, but varied among gene families (<a href="#pone-0001235-t007">Table 7</a>). The proportions of TF-CDS for AP2-EREBP (64%), MADS (58%), and GRAS (61%) were significantly lower than the proportion (75%) for the entire TF-CDS (<em>P</em>&lt;0.05, calculated by chi-squared test), while those for C3H (86%) and PHD (90%) were significantly higher.</p>
<div class="figure" id="pone-0001235-t007"><div class="img"><a name="pone-0001235-t007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t007.TIF"></span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Coverage of genes encoding transcription factors in the rice FL-cDNA libraries</span></strong></p><span>doi:10.1371/journal.pone.0001235.t007</span></div><a id="article1.body1.sec2.sec13.p2" name="article1.body1.sec2.sec13.p2"></a><p>Based on RiceCyc version 1.1 at the GRAMENE Web site (<a href="http://www.gramene.org/pathway/">http://www.gramene.org/pathway/</a>) <a href="#pone.0001235-Jaiswal1">[10]</a>, 2750 TIGR4 CDS were annotated as genes encoding enzymes involved in metabolic pathways. The CDS-AE proportion of the entire CDS involved in metabolic pathways was about 75%, but the proportions varied depending on the pathways (<a href="#pone-0001235-t008">Table 8</a>). The CDS-AE proportion for “removal of superoxide radicals” (DTOX1-PWY, 16.5%) was particularly lower than the proportion for the entire CDS (75%). These results imply that the efficiency of FL-cDNA cloning could be biased depending on the nature of gene functions.</p>
<div class="figure" id="pone-0001235-t008"><div class="img"><a name="pone-0001235-t008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t008.TIF"></span>)
                </a></li></ul></div><p><strong>Table 8.  <span>Coverage of genes associated with metabolic pathways in rice FL-cDNAs libraries</span></strong></p><span>doi:10.1371/journal.pone.0001235.t008</span></div>

<h4>The structures of protein encoded in cDNA influence cloning efficiency</h4>
<a id="article1.body1.sec2.sec14.p1" name="article1.body1.sec2.sec14.p1"></a><p>For broader assessment of the biased cloning of FL-cDNA by gene functions, the CDS-AE proportions were examined for gene families which contain domains categorized at Pfam <a href="#pone.0001235-Finn1">[34]</a> and InterPro <a href="#pone.0001235-Mulder1">[35]</a> in TIGR OSA1 <a href="#pone.0001235-Ouyang1">[9]</a>. A total of 24 247 TIGR4 loci, excluding TE genes, were found containing domains listed in Pfam. About 72% (17 459) of such TIGR4 loci were CDS-AE. The CDS-AE proportions varied depending on the domain types. For example, the proportions of CDS-AE for gene families encoding metabolic enzyme related domains (PF00657, PF00561) are generally higher than those for all loci; whereas, the proportions for some gene families encoding domains for regulatory roles such as protein kinases (PF00069) and receptors (PF00560, PF00931) were lower (<a href="#pone-0001235-t009">Table 9</a>, <a href="#pone.0001235.s007">Table S5a</a>). These results corroborate the observation that cloning efficiencies of FL-cDNA vary depending on the nature of gene functions.</p>
<div class="figure" id="pone-0001235-t009"><div class="img"><a name="pone-0001235-t009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t009.TIF"></span>)
                </a></li></ul></div><p><strong>Table 9.  <span>Coverage of genes with Pfam domains in the rice FL-cDNA libraries</span></strong></p><span>doi:10.1371/journal.pone.0001235.t009</span></div><a id="article1.body1.sec2.sec14.p2" name="article1.body1.sec2.sec14.p2"></a><p>It was especially noticed that the CDS-AE proportions were extremely low for three gene families with rice-specific domains (PF03578, PF06654, and PF07197). However, the median signal intensity for CDS-ANE containing PF03578 was approximately 900 (data not shown), suggesting that the inefficient cloning of FL-cDNA from the genes with rice-specific domains might not be due to the lack of active transcription.</p>
<a id="article1.body1.sec2.sec14.p3" name="article1.body1.sec2.sec14.p3"></a><p>It was reported that manipulation and amplification of FL-cDNA clones from some plant virus genomes in <em>E. coli</em> were rather difficult because of the instability of viral gene sequences <a href="#pone.0001235-Johansen1">[36]</a>–<a href="#pone.0001235-Choi1">[38]</a>. It was suggested that the instability of viral genes in <em>E. coli</em> resulted from the toxicity of virus-specific gene products <a href="#pone.0001235-Johansen1">[36]</a>–<a href="#pone.0001235-Choi1">[38]</a>. From these observations we hypothesized that one cause of inefficient FL-cDNA recovery from some gene families could be toxicity of genes encoding protein domains heterologous to <em>E. coli</em>. To test this hypothesis, we examined whether the protein domains listed in <a href="#pone-0001235-t009">Table 9</a> are heterologous to <em>E. coli</em>. The protein domains encoded in genes with lower CDS-AE proportions were predominantly heterologous to <em>E. coli</em>, while a majority of the domains encoded in genes with higher CDS-AE proportions are also found encoded in genes of <em>E. coli</em> (<a href="#pone-0001235-t009">Table 9</a>, <a href="#pone.0001235.s007">Table S5a</a>). Such inefficient production of FL-cDNA from genes encoding domains heterologous to <em>E. coli</em> was also observed in Arabidopsis by our analysis. Based on the information available at TAIR6 (<a href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>) <a href="#pone.0001235-Rhee1">[25]</a> and SIGnAL database (<a href="http://signal.salk.edu/">http://signal.salk.edu/</a>), TAIR6 loci were classified into CDS-AE or CDS-ANE. CDS-AE proportions varied depending on the domains encoded in genes. Gene families encoding domains that are not found in <em>E. coli</em> had significantly lower CDS-AE proportions (<a href="#pone.0001235.s007">Table S5b</a>). Therefore, these results support the idea that FL-cDNAs encoding rice-specific or eukaryotic-specific domains might be difficult to clone in <em>E. coli</em>. Though we could not analyze the diversity of cloning efficiency between rice and Arabidopsis because the numbers of FL-cDNA clones are drastically different between rice and Arabidopsis (580K and 27K, respectively (<a href="http://signal.salk.edu/">http://signal.salk.edu/</a>)), we could not find the diversity of cloning efficiency by organisms (<a href="#pone.0001235.s007">Table S5a and b</a>).</p>
<a id="article1.body1.sec2.sec14.p4" name="article1.body1.sec2.sec14.p4"></a><p>During amplification of foreign genes cloned in <em>E. coli</em>, spontaneous deletions and rearrangements of genes which are supposed to be unfavorable to <em>E. coli</em> were observed <a href="#pone.0001235-Johansen1">[36]</a>, <a href="#pone.0001235-Choi1">[38]</a>. If a similar event occurs during cloning of rice FL-cDNA, our libraries should contain reassembled FL-cDNA clones. To find out whether such events might have occurred for rice FL-cDNA, we compared the sequences of about 200 short FL-cDNA (&lt;600 bp) with those of the corresponding FL-AE and CDS-AE. The result indicated that some of the short FL-cDNA could be clones which might have been reassembled in <em>E. coli</em> (<a href="#pone-0001235-t010">Table 10</a>, <a href="#pone.0001235.s002">Figure S2</a>). It appeared that part of internal exons encoding a protein domain might have been deleted in each FL-cDNA, and most of the domains encoded in the deleted sequences were either absent in <em>E. coli</em> or were domains functioning as a gene regulator such as Hsp90 <a href="#pone.0001235-Nadeau1">[39]</a>. These results suggested that the cloning of some full-length cDNAs was adversely affected in <em>E. coli</em>, and that sequences encoding heterologous domains and those functioning as gene regulators in <em>E. coli</em> are the most likely to be affected.</p>
<div class="figure" id="pone-0001235-t010"><div class="img"><a name="pone-0001235-t010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.t010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.t010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.t010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.t010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.t010.TIF"></span>)
                </a></li></ul></div><p><strong>Table 10.  <span>The list of FL-cDNA in excluded internal sequences</span></strong></p><span>doi:10.1371/journal.pone.0001235.t010</span></div><a id="article1.body1.sec2.sec14.p5" name="article1.body1.sec2.sec14.p5"></a><p>As the results above suggested, rearranged cDNA sequences might have been included in those used to analyze gene structures (e.g., alternative forms, exon-intron analysis). This may have led artificial mismatches in exon-intron structure between the predicted CDS and the FL-cDNA sequences, resulting in misinterpretations of the alternative forms of the loci. Since the redundancy rate for the entire FL-cDNA was only 1.43 (32 775 FL-cDNA/22 943 FL-cDNA loci), it is particularly difficult to confirm the exon-intron structure for most loci by multiple FL-cDNA sequences. Thus, the effect of the <em>E. coli</em> bias on recovery and rearrangement of FL-cDNA should be considered in order to avoid any <em>in silico</em> artifacts of FL-cDNA.</p>
<a id="article1.body1.sec2.sec14.p6" name="article1.body1.sec2.sec14.p6"></a><p>Our results revealed that the rice FL-cDNA that we collected originated mostly from genes with high transcription activity. In addition, we found significant differences in the recovery of FL-cDNA among gene families presumably due to the bias related to gene function and gene structure in <em>E. coli</em> that results in unequal rice FL-cDNA collections. Because rice FL-cDNA clones are produced under cloning biases as described above, it is important to overcome them for accurate and efficient cloning.</p>
<a id="article1.body1.sec2.sec14.p7" name="article1.body1.sec2.sec14.p7"></a><p>The FL-cDNA library contains both protein-coding and non-protein-coding transcripts. Non-coding transcripts have been reported to function as gene regulators <a href="#pone.0001235-Costa1">[40]</a>; thus, the information on non-coding transcripts is also important to reveal transcription regulation. It has been reported that the number of mRNA-like ncRNA loci was similar to that of protein-coding mRNA loci in mouse <a href="#pone.0001235-Ravasi1">[26]</a>, and it seems that the number of ncRNA in rice is not many number (FL-AE:FL-NAE = 23 117 : 5 447, <a href="#pone-0001235-t002">Table 2</a>). However, the number of mouse FL-cDNA clones is more than 2000K and is more than three times the number of FL-cDNA clones in rice. So, it does not indicate that the number of ncRNA of rice is less than that of mouse.</p>
<a id="article1.body1.sec2.sec14.p8" name="article1.body1.sec2.sec14.p8"></a><p>Approximately 2400 FL-cDNA clones which were sequenced completely in our library were not able to be mapped on TIGR4. It has previously been reported that some FL-cDNA were mapped in telomere or centromere regions, which have not been assembled in the Pseudomolecules database <a href="#pone.0001235-Yan1">[29]</a>–<a href="#pone.0001235-Mizuno1">[32]</a>, <a href="#pone.0001235-Mizuno2">[41]</a>. However, the size of un-assembled regions predicted in IRGSP4 is about 4% of the entire genome (<a href="http://rgp.dna.affrc.go.jp/E/IRGSP/index.html">http://rgp.dna.affrc.go.jp/E/IRGSP/index​.html</a>). This value was lower than the value for unmapped clones (7%) (<a href="#pone-0001235-t001">Table 1</a>). If unmapped clones were mapped on Gap region in genome, we expected the annotation profile of unmapped clones would be similar to that of mapped clones. Therefore, we analyzed unmapped clones for their homology to Arabidopsis CDS using BlastX and classified them according to their similarity (see Gene annotation). Although about 230 of 2400 unmapped clones were confirmed to map on the rice genome by other genome assemblies (<a href="#pone-0001235-t001">Table 1</a>), the proportion of highly-homologous unmapped clones was lower than that of mapped clones and the proportion of no-homologous unmapped clones was higher than that of mapped clones (<a href="#pone.0001235.s008">Table S6</a>). So, we think that about 1000 unmapped clones could be contaminants originating from organisms other than rice.</p>
<a id="article1.body1.sec2.sec14.p9" name="article1.body1.sec2.sec14.p9"></a><p>About 530K out of 578K FL-cDNA clones were derived from about 28.5K loci. Some loci which were not supported by FL-cDNA clones (CDS-ANE) were found to be active in transcription based on the microarray analysis in this study. The signal detection frequency for CDS-ANE loci varied between 26 and 84% depending on the threshold chosen for analysis (<a href="#pone-0001235-t005">Table 5</a>, <a href="#pone.0001235.s005">S3</a>). Thus, we estimate the number of rice genes based on TIGR4 predictions to be between 31.8K (23 193 CDS-AE+[32 697 CDS-ANE×0.265]) and 50.8K (23 193 CDS-AE+[32 697 CDS-ANE×0.843]), excluding non-protein-coding genes such as FL-NAE.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>FL-cDNA library construction, selection of clones, and one-pass sequence analysis</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Starting materials used to construct FL-cDNA libraries are indicated in <a href="#pone.0001235.s009">Table S7</a>. Tissues were collected from <em>Oryza sativa</em> ssp. <em>japonica</em> cv. Nipponbare. Construction of the FL-cDNA libraries, selection of the cDNA clones, and one-pass sequence analysis were described previously <a href="#pone.0001235-Rice1">[5]</a>. The 5′-, the 3′- or both ends of 578 473 FL-cDNA clones derived from more than 90 libraries were sequenced, and 241 854 5′ end one-pass sequences (FL-EST) and 536 885 3′ end FL-EST were obtained. In addition, 35 187 FL-cDNA clones were completely sequenced. All cDNA sequences were deposited in DNA Data bank of Japan (DDBJ, <a href="http://www.ddbj.nig.ac.jp/">http://www.ddbj.nig.ac.jp/</a>) (<a href="#pone.0001235.s010">Table S8</a>) <a href="#pone.0001235-Sugawara1">[42]</a>.</p>


<h4>Mapping of FL-cDNA sequences and FL-EST for rice genomic DNA</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>FL-cDNA and FL-EST sequences were mapped onto several rice genome assemblies (TIGR4, IRGSP3 and IRGSP4, the Nipponbare sequence by Syngenta, and the 93-11 sequence by BGI) using the BLAT software <a href="#pone.0001235-Sugawara1">[42]</a>. The criterion for mapping was &gt;95% identity and &gt;90% coverage. The genomic sequence hit with the highest score was defined as the mapping position. FL-cDNAs and FL-ESTs that mapped to more than one genome sequence with a similarly high score were excluded from analysis.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>After mapping the FL-cDNAs and FL-ESTs, clustering analysis was performed to define the FL-cDNA mapped locus. The FL-cDNA and FL-EST mapping results were clustered independently. The 5′ end FL-EST and 3′ end FL-EST mapping results were also clustered independently, and these clustered regions were then combined (<a href="#pone-0001235-g002">Figure 2</a>). These grouping regions were defined as FL-EST regions. The FL-cDNA regions and FL-EST regions were compared to identify the FL-cDNA locus (<a href="#pone-0001235-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001235-g002"><div class="img"><a name="pone-0001235-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001235" data-uri="info:doi/10.1371/journal.pone.0001235.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001235.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001235.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001235.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001235.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001235.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Strategy for mapping of FL-cDNA clones and FL-ESTs, and for definition of FL-cDNA loci.</span></strong></p><span>doi:10.1371/journal.pone.0001235.g002</span></div>

<h4>FL-cDNA analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The length of the longest ORF was defined as the distance between the nucleotide position (number) of the A in ATG and the last nucleotide before the termination codon. The BlastX algorithm was used for homology search with Arabidopsis genes (TAIR6 (<a href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>) <a href="#pone.0001235-Rhee1">[25]</a>).</p>


<h4>Microarray experiment</h4>
<h5>1. Design of probes and oligo-arrays.</h5><a id="article1.body1.sec3.sec4.sec1.p1" name="article1.body1.sec3.sec4.sec1.p1"></a><p>FL-cDNA clone sequences (FL-cDNA and FL-EST) were mapped in TIGR Pseudomolecules Release 2 using the BLAT program <a href="#pone.0001235-Kent1">[43]</a>. FL-cDNA loci were identified with the same criteria as above (i.e., 95% identity and 90% coverage). After mapping, we selected nucleotide sequences from the FL-cDNA locus and CDS of TIGR Pseudomolecules Release 2 to design probes. About 82K 60-mer probe candidates were designed by Agilent Technologies (<a href="http://www.home.agilent.com/">http://www.home.agilent.com/</a>) considering factors such as the guanine-cytosine (GC) content and melting temperature of the oligonucleotides. The probes were re-mapped in TIGR4 using BLAT <a href="#pone.0001235-Kent1">[43]</a> according to the same criteria used previously (&gt;95% identity, &gt;90% coverage, non-redundant highest score). We found that 63 455 probes met this criteria and they were mapped onto FL-loci or TIGR loci.</p>

<h5>2. Sample preparation.</h5><a id="article1.body1.sec3.sec4.sec2.p1" name="article1.body1.sec3.sec4.sec2.p1"></a><p>For the microarray experiment, we used total RNA prepared from shoots, roots, panicles, and callus. RNA samples from 10 individual plants were pooled for microarray analysis. Shoots were collected from 14-day-old seedlings of varieties Nipponbare and IR64 grown in pot culture in a greenhouse. Roots were derived from 10-day-old seedlings grown in hydroculture in an incubator. Panicles were collected during the late-boot stage. Callus that originated from the scutellum was incubated for 4 weeks on MS medium <a href="#pone.0001235-Murashige1">[44]</a> that contained 2 mg/L of 2,4-dichlorophenoxyacetic acid and 0.1% Kao and Michayluk Vitamin Solution (Sigma-Aldrich, <a href="http://www.sigmaaldrich.com/">http://www.sigmaaldrich.com/</a>). These tissues were frozen with liquid N<sub>2</sub> and stored at −80 °C. Total RNA from the shoots was extracted with Trizol (Invitrogen, <a href="https://www.invitrogen.com/">https://www.invitrogen.com/</a>) according to the manufacturer's instructions, and was extracted from other tissues using the RNeasy Midi Kit (Qiagen, <a href="http://www1.qiagen.com/">http://www1.qiagen.com/</a>). The concentration and quality of RNA were checked using the NanoDrop (ND-1000, NanoDrop Technologies, <a href="http://www.nanodrop.com/">http://www.nanodrop.com/</a>) and the BioAnalyzer (G2938A, Agilent Technologies, <a href="http://www.home.agilent.com/">http://www.home.agilent.com/</a>), respectively.</p>

<h5>3. Labeling, hybridization, and washing.</h5><a id="article1.body1.sec3.sec4.sec3.p1" name="article1.body1.sec3.sec4.sec3.p1"></a><p>Labeled cRNA was synthesized from 500 ng of total RNA using a Low RNA Input Linear Amplification kit (Agilent Technologies, <a href="http://www.home.agilent.com/">http://www.home.agilent.com/</a>), according to the manufacturer's protocols. For two-color hybridization, 1 μg of Cyanine 3 (Cy3)-labeled cRNA and 1 μg of Cyanine 5 (Cy5)-labeled cRNA were both used, and the hybridization solution was prepared with an <em>In Situ</em> Hybridization Kit Plus (Agilent Technologies, <a href="http://www.home.agilent.com/">http://www.home.agilent.com/</a>). Hybridization and washing were performed according to the manufacturer's protocols.</p>

<h5>4. Scanning and data analysis.</h5><a id="article1.body1.sec3.sec4.sec4.p1" name="article1.body1.sec3.sec4.sec4.p1"></a><p>The slide images were scanned with the DNA microarray scanner (G2505B, Agilent Technologies, <a href="http://www.home.agilent.com/">http://www.home.agilent.com/</a>) and the resulting image files were processed to obtain normalized Cy3 and Cy5 signal intensity for each probe using the manufacturer's Feature Extraction version 8.1 software. In this software, normalization of Cy3 and Cy5 signal intensity in each image was performed using locally weighted scatterplot smoothing and rank-consistency methods. In addition, the software also validated whether the signal intensity of each probe was significantly higher than the background signal intensity. The validation was performed in two steps. First, a two-tailed <em>t</em>-test was calculated based on the probe's signal and the local background signal and their corresponding errors. If the result was significant at <em>p</em>&lt;0.01, the signal was considered to be significantly different from the background. Next, if the raw signal intensity was significantly higher than the local background intensity, the signal intensity was recalculated with the background subtracted. If the resulting signal intensity was greater than 2.6 times the background signal's standard deviation, the signal was also defined as significantly higher than the background signal intensity.</p>
<a id="article1.body1.sec3.sec4.sec4.p2" name="article1.body1.sec3.sec4.sec4.p2"></a><p>Each sample was hybridized at least four times, but the normalized signal intensity differed among experiments. The differences were attributed to the variation among slides. To reduce the variation, the median of the normalized signal intensity in each array data was adjusted to 1000. After fixation of median intensity, any normalized Cy3 and Cy5 signal intensities &gt;100 000 were considered as the intensity of 100,000, and the dynamic range of signal intensity was defined empirically to be between 0 to 100 000 based on previous experiments. We then calculated the median of the normalized signal intensity for each probe in each sample, and used the resulting median signal intensity to represent the signal intensity of each probe. If only one probe was designed from a FL-cDNA locus or a CDS, the probe's signal intensity was used to define the signal intensity of the locus or CDS. However, if several probes were designed from a FL-cDNA locus or a CDS, the highest signal intensity among the probes was defined as the signal intensity for the locus or CDS. All microarray data are available at the Gene Expression Omnibus (GEO) <a href="#pone.0001235-Barrett1">[45]</a> repository at the National Center for Biotechnology Information (GSE7366 and GSE7374, <a href="http://ncbi.nlm.nih.gov/geo/">http://ncbi.nlm.nih.gov/geo/</a>).</p>


</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001235"></div><a name="pone.0001235.s001" id="pone.0001235.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The results of gene structure analysis. (a) The length distribution of FL-cDNA for FL-AE and FL-NAE. (b) The distribution of open reading frame (ORF) proportions for FL-AE and FL-NAE. (c) The distribution of FL-cDNA locus lengths for FL-AE and FL-NAE. (d) The distribution of locus lengths for CDS-AE and CDS-ANE in TIGR4. (e) The distribution of the number of exons for FL-AE and FL-NAE. (f) The distribution of exon and intron lengths for the respective locus types. (g) The distribution of the number of FL-cDNA clones mapped per single FL-AE and FL-NAE loci.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(8.78 MB TIF)</p>
<a name="pone.0001235.s002" id="pone.0001235.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Sequences encoding specific domains which may have been excluded from FL-cDNA clones. The figures were modified from those found at TIGR OSA1. (a) AK058349 in which the sequence encoding the AP2 domain (IPR001471) was excluded; (b) AK062487 in which the sequence encoding the MYB domain (IPR001005) was excluded.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(8.85 MB DOC)</p>
<a name="pone.0001235.s003" id="pone.0001235.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s003">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The mapping positions of FL-cDNA in five genome assemblies.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(2.66 MB ZIP)</p>
<a name="pone.0001235.s004" id="pone.0001235.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s004">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Mapping information of FL-cDNA. a: Information for all loci in TIGR4, b: The list of FL-cDNA loci and mapped FL-cDNA</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(3.10 MB ZIP)</p>
<a name="pone.0001235.s005" id="pone.0001235.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s005">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Signal intensity data for the five samples. a): NB: “Nipponbare”, IR: IR64 b): no-signal: number of loci on which signal intensity was not significant higher than backgroud intensity</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001235.s006" id="pone.0001235.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s006">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">The correlation coefficient between the density of TE-related locus and the density of locus based on signal intensity under each locus type</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.01 MB XLS)</p>
<a name="pone.0001235.s007" id="pone.0001235.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s007">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">The relationship between the CDS-AE proportion and domains. a: rice based on TIGR OSA1, b: Arabidopsis based on TAIR6</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.11 MB ZIP)</p>
<a name="pone.0001235.s008" id="pone.0001235.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s008">Table S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">The frequency of Arabidopsis homologous genes between mapped and un-mapped FL-cDNA clones</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001235.s009" id="pone.0001235.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s009">Table S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Library information used to construct the FL-cDNA libraries</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001235.s010" id="pone.0001235.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001235.s010">Table S8. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">DDBJ accession IDs for FL-cDNA. a: sequenced 5′ end FL-ESTs and 3′ end FL-ESTs, b: completely sequenced FL-cDNA</p>
<p class="siDoi">doi:10.1371/journal.pone.0001235.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(3.58 MB ZIP)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. T. Nakamura (National Agricultural Research Center for Tohoku Region, Japan), E. Minami, Y. Nishizawa, Y. Ohashi, and M. Shimono (National Institute of Agrobiological Sciences), and Prof. M. Nishiguchi (Ehime University, Japan) for providing valuable samples used in the construction of the FL-cDNA libraries. We thank N. Sakazume-Mizutani, M. Ohno-Nishijima, K. Nomura (RIKEN Yokohama Institute) for creation and sequencing of FL-cDNA, and S. Kimura and S. Iizumi (National Institute of Agrobiological Sciences) for supporting microarray experiments. We thank Prof. B.Gengenbach for editing this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YH PC JK S Kikuchi  K Matsubara K Murakami. Performed the experiments: K Kamiya K Kurita  MY AK TB NF KI YN SF MT NN HK TM K Satoh  YO IC MN TH S Kanagawa TA JT MM DS HY. Analyzed the data: K Satoh TN K Suzuki  KD. Contributed reagents/materials/analysis tools: K Satoh NK. Wrote the paper: K Satoh IC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001235-Goff1" id="pone.0001235-Goff1"></a>Goff SA, Ricke D, Lan TH, Presting G, Wang R, Dunn M, et al.  (2002) A draft sequence of the rice genome (Oryza sativa L. ssp. japonica). Science  296: 92–100.  <ul class="find" data-citedArticleID="1023192" data-doi="10.1126/science.1068275"><li><a href="http://dx.doi.org/10.1126/science.1068275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp.+japonica%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp.+japonica%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001235-Yu1" id="pone.0001235-Yu1"></a>Yu J, Hu S, Wang J, Wong GK, Li S, et al.  (2002) A draft sequence of the rice genome (<em>Oryza sativa</em> L. ssp. <em>indica</em>). Science  296: 79–92.  <ul class="find" data-citedArticleID="1023258" data-doi="10.1126/science.1068037"><li><a href="http://dx.doi.org/10.1126/science.1068037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp.+indica%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp.+indica%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001235-Yu2" id="pone.0001235-Yu2"></a>Yu J, Wang J, Lin W, Li S, Li H, et al.  (2005) The genomes of <em>Oryza sativa</em>: a history of duplications. PLoS Biol 3.   Available: <a href="http://biology.plosjournals.org/perlserv/requestget-documentdoi10.1371/journal.pbio.0030038">http://biology.plosjournals.org/perlserv​/requestget-documentdoi10.1371/journal.p​bio.0030038</a> Accessed 1 Feb 2005.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001235-International1" id="pone.0001235-International1"></a>International Rice Genome Sequencing Project (2005) The map-based sequence of the rice genome. Nature  436: 793–800.  <ul class="find" data-citedArticleID="1023198" data-doi="10.1038/nature03895"><li><a href="http://dx.doi.org/10.1038/nature03895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+map-based+sequence+of+the+rice+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+map-based+sequence+of+the+rice+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001235-Rice1" id="pone.0001235-Rice1"></a>Rice Full-Length cDNA Consortium (2003) Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice. Science  301: 376–379.  <ul class="find" data-citedArticleID="1023240" data-doi="10.1126/science.1081288"><li><a href="http://dx.doi.org/10.1126/science.1081288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collection%2C+mapping%2C+and+annotation+of+over+28%2C000+cDNA+clones+from+japonica+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collection%2C+mapping%2C+and+annotation+of+over+28%2C000+cDNA+clones+from+japonica+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001235-Wu1" id="pone.0001235-Wu1"></a>Wu J, Maehara T, Shimokawa T, Yamamoto S, Harada C, et al.  (2002) A comprehensive rice transcript map containing 6591 expressed sequence tag sites. Plant Cell  14: 525–535.  <ul class="find" data-citedArticleID="1023250" data-doi="10.1105/tpc.010274"><li><a href="http://dx.doi.org/10.1105/tpc.010274" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+rice+transcript+map+containing+6591+expressed+sequence+tag+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+rice+transcript+map+containing+6591+expressed+sequence+tag+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001235-Zhou1" id="pone.0001235-Zhou1"></a>Zhou Y, Tang J, Walker MG, Zhang X, Wang J, et al.  (2003) Gene identification and expression analysis of 86,136 Expressed Sequence Tags (EST) from the rice genome. Genomics Proteomics Bioinformatics  1: 26–42.  <ul class="find" data-citedArticleID="1023268"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+identification+and+expression+analysis+of+86%2C136+Expressed+Sequence+Tags+%28EST%29+from+the+rice+genome.&amp;auth=&amp;atitle=Gene+identification+and+expression+analysis+of+86%2C136+Expressed+Sequence+Tags+%28EST%29+from+the+rice+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+identification+and+expression+analysis+of+86%2C136+Expressed+Sequence+Tags+%28EST%29+from+the+rice+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+identification+and+expression+analysis+of+86%2C136+Expressed+Sequence+Tags+%28EST%29+from+the+rice+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001235-Zhang1" id="pone.0001235-Zhang1"></a>Zhang J, Feng Q, Jin C, Qiu D, Zhang L, et al.  (2005) Features of the expressed sequences revealed by a large-scale analysis of ESTs from a normalized cDNA library of the elite indica rice cultivar Minghui 63. Plant J  42: 772–780.  <ul class="find" data-citedArticleID="1023262" data-doi="10.1111/j.1365-313x.2005.02408.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02408.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Features+of+the+expressed+sequences+revealed+by+a+large-scale+analysis+of+ESTs+from+a+normalized+cDNA+library+of+the+elite+indica+rice+cultivar+Minghui+63." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Features+of+the+expressed+sequences+revealed+by+a+large-scale+analysis+of+ESTs+from+a+normalized+cDNA+library+of+the+elite+indica+rice+cultivar+Minghui+63.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001235-Ouyang1" id="pone.0001235-Ouyang1"></a>Ouyang S, Zhu W, Hamilton J, Lin H, Campbell M, et al.  (2007) The TIGR Rice Genome Annotation Resource: improvements and new features. Nucleic Acids Res  35(Database issue): D883–887.  <ul class="find" data-citedArticleID="1023230" data-doi="10.1093/nar/gkl976"><li><a href="http://dx.doi.org/10.1093/nar/gkl976" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+TIGR+Rice+Genome+Annotation+Resource%3A+improvements+and+new+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+TIGR+Rice+Genome+Annotation+Resource%3A+improvements+and+new+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001235-Jaiswal1" id="pone.0001235-Jaiswal1"></a>Jaiswal P, Ni J, Yap I, Ware D, Spooner W, et al.  (2006) Gramene: a bird's eye view of cereal genomes. Nucleic Acids Res  34(Database issue): D717–723.  <ul class="find" data-citedArticleID="1023200" data-doi="10.1093/nar/gkj154"><li><a href="http://dx.doi.org/10.1093/nar/gkj154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gramene%3A+a+bird%27s+eye+view+of+cereal+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gramene%3A+a+bird%27s+eye+view+of+cereal+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001235-RianoPachon1" id="pone.0001235-RianoPachon1"></a>Riano-Pachon DM, Ruzicic S, Dreyer I, Mueller-Roeber B (2007) PlnTFDB: an integrative plant transcription factor database. BMC Bioinformatics 8.   Available: <a href="http://www.biomedcentral.com/1471-2105/8/42">http://www.biomedcentral.com/1471-2105/8​/42</a> Accessed 7 Feb 2007.  <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001235-Ren1" id="pone.0001235-Ren1"></a>Ren XY, Vorst O, Fiers MW, Stiekema WJ, Nap JP (2006) In plants, highly expressed genes are the least compact. Trends Genet  22: 528–532.  <ul class="find" data-citedArticleID="1023234" data-doi="10.1016/j.tig.2006.08.008"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.08.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+plants%2C+highly+expressed+genes+are+the+least+compact." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+plants%2C+highly+expressed+genes+are+the+least+compact.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001235-Lin1" id="pone.0001235-Lin1"></a>Lin H, Zhu W, Silva JC, Gu X, Buell CR (2006) Intron gain and loss in segmentally duplicated genes in rice. Genome Biol 7.   Available: <a href="http://genomebiology.com/2006/7/5/R41">http://genomebiology.com/2006/7/5/R41</a> Accessed 23 May 2006.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0001235-Ohyanagi1" id="pone.0001235-Ohyanagi1"></a>Ohyanagi H, Tanaka T, Sakai H, Shigemoto Y, Yamaguchi K, et al.  (2006) The Rice Annotation Project Database (RAP-DB): hub for <em>Oryza sativa</em> ssp. <em>japonica</em> genome information. Nucleic Acids Res 34(Database issue) D741–744.  <ul class="find" data-citedArticleID="1023228" data-doi="10.1093/nar/gkj094"><li><a href="http://dx.doi.org/10.1093/nar/gkj094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Rice+Annotation+Project+Database+%28RAP-DB%29%3A+hub+for+Oryza+sativa+ssp.+japonica+genome+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Rice+Annotation+Project+Database+%28RAP-DB%29%3A+hub+for+Oryza+sativa+ssp.+japonica+genome+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001235-Rice2" id="pone.0001235-Rice2"></a>Rice Annotation Project (2007) Curated genome annotation of <em>Oryza sativa</em> ssp. <em>japonica</em> and comparative genome analysis with <em>Arabidopsis thaliana</em>. Genome Res  17: 175–183.  <ul class="find" data-citedArticleID="1023242" data-doi="10.1101/gr.5509507"><li><a href="http://dx.doi.org/10.1101/gr.5509507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Curated+genome+annotation+of+Oryza+sativa+ssp.+japonica+and+comparative+genome+analysis+with+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Curated+genome+annotation+of+Oryza+sativa+ssp.+japonica+and+comparative+genome+analysis+with+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001235-Zhao1" id="pone.0001235-Zhao1"></a>Zhao W, Wang J, He X, Huang X, Jiao Y, et al.  (2004) BGI-RIS: an integrated information resource and comparative analysis workbench for rice genomics. Nucleic Acids Res  32(Database issue): D377–382.  <ul class="find" data-citedArticleID="1023266" data-doi="10.1093/nar/gkh085"><li><a href="http://dx.doi.org/10.1093/nar/gkh085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BGI-RIS%3A+an+integrated+information+resource+and+comparative+analysis+workbench+for+rice+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BGI-RIS%3A+an+integrated+information+resource+and+comparative+analysis+workbench+for+rice+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001235-Li1" id="pone.0001235-Li1"></a>Li L, Wang X, Stolc V, Li X, Zhang D, et al.  (2006) Genome-wide transcription analyses in rice using tiling microarrays. Nat Genet  38: 124–129.  <ul class="find" data-citedArticleID="1023210" data-doi="10.1038/ng1704"><li><a href="http://dx.doi.org/10.1038/ng1704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+transcription+analyses+in+rice+using+tiling+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+transcription+analyses+in+rice+using+tiling+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001235-Seki1" id="pone.0001235-Seki1"></a>Seki M, Narusaka M, Kamiya A, Ishida J, Satou M, et al.  (2002) Functional annotation of a full-length <em>Arabidopsis</em> cDNA collection. Science  296: 141–145.  <ul class="find" data-citedArticleID="1023244" data-doi="10.1126/science.1071006"><li><a href="http://dx.doi.org/10.1126/science.1071006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+annotation+of+a+full-length+Arabidopsis+cDNA+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+annotation+of+a+full-length+Arabidopsis+cDNA+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001235-Imanishi1" id="pone.0001235-Imanishi1"></a>Imanishi T, Itoh T, Suzuki Y, O'Donovan C, Fukuchi S, et al.  (2004) Integrative annotation of 21,037 human genes validated by full-length cDNA clones. PLoS Biol 2.   Available: <a href="http://biology.plosjournals.org/perlserv/requestget-documentdoi10.13712Fjournal.pbio.0020162">http://biology.plosjournals.org/perlserv​/requestget-documentdoi10.13712Fjournal.​pbio.0020162</a> Accessed 20 Apr 2004.  <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0001235-The1" id="pone.0001235-The1"></a>The RIKEN Genome Exploration Research Group Phase II Team and the FANTOM Consortium (2001) Functional annotation of a full-length mouse cDNA collection. Nature  409: 685–690.  <ul class="find" data-citedArticleID="1023248"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+annotation+of+a+full-length+mouse+cDNA+collection.&amp;auth=&amp;atitle=Functional+annotation+of+a+full-length+mouse+cDNA+collection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+annotation+of+a+full-length+mouse+cDNA+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+annotation+of+a+full-length+mouse+cDNA+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001235-Nakano1" id="pone.0001235-Nakano1"></a>Nakano M, Nobuta K, Vemaraju K, Tej SS, Skogen JW, et al.  (2006) Plant MPSS databases: signature-based transcriptional resources for analyses of mRNA and small RNA. Nucleic Acids Res  34(Database issue): D731–735.  <ul class="find" data-citedArticleID="1023226" data-doi="10.1093/nar/gkj077"><li><a href="http://dx.doi.org/10.1093/nar/gkj077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+MPSS+databases%3A+signature-based+transcriptional+resources+for+analyses+of+mRNA+and+small+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+MPSS+databases%3A+signature-based+transcriptional+resources+for+analyses+of+mRNA+and+small+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001235-Gibbings1" id="pone.0001235-Gibbings1"></a>Gibbings JG, Cook BP, Dufault MR, Madden SL, Khuri S, et al.  (2003) Global transcript analysis of rice leaf and seed using SAGE technology. Plant Biotechnol J  1: 271–285.  <ul class="find" data-citedArticleID="1023190" data-doi="10.1046/j.1467-7652.2003.00026.x"><li><a href="http://dx.doi.org/10.1046/j.1467-7652.2003.00026.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+transcript+analysis+of+rice+leaf+and+seed+using+SAGE+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+transcript+analysis+of+rice+leaf+and+seed+using+SAGE+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001235-Hashimoto1" id="pone.0001235-Hashimoto1"></a>Hashimoto S, Suzuki Y, Kasai Y, Morohoshi K, Yamada T, et al.  (2004) 5′-end SAGE for the analysis of transcriptional start sites. Nat Biotechnol.  22: 1146–1149.  <ul class="find" data-citedArticleID="1023194" data-doi="10.1038/nbt998"><li><a href="http://dx.doi.org/10.1038/nbt998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=5%E2%80%B2-end+SAGE+for+the+analysis+of+transcriptional+start+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%225%E2%80%B2-end+SAGE+for+the+analysis+of+transcriptional+start+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001235-Yamada1" id="pone.0001235-Yamada1"></a>Yamada K, Lim J, Dale JM, Chen H, Shinn P, et al.  (2003) Empirical analysis of transcriptional activity in the <em>Arabidopsis</em> genome. Science  302: 842–846.  <ul class="find" data-citedArticleID="1023252" data-doi="10.1126/science.1088305"><li><a href="http://dx.doi.org/10.1126/science.1088305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+analysis+of+transcriptional+activity+in+the+Arabidopsis+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+analysis+of+transcriptional+activity+in+the+Arabidopsis+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001235-Rhee1" id="pone.0001235-Rhee1"></a>Rhee SY, Beavis W, Berardini TZ, Chen G, Dixon D, et al.  (2003) The Arabidopsis Information Resource (TAIR): a model organism database providing a centralized, curated gateway to <em>Arabidopsis</em> biology, research materials and community. Nucleic Acids Res  31: 224–228.  <ul class="find" data-citedArticleID="1023236" data-doi="10.1093/nar/gkg076"><li><a href="http://dx.doi.org/10.1093/nar/gkg076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+Information+Resource+%28TAIR%29%3A+a+model+organism+database+providing+a+centralized%2C+curated+gateway+to+Arabidopsis+biology%2C+research+materials+and+community." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+Information+Resource+%28TAIR%29%3A+a+model+organism+database+providing+a+centralized%2C+curated+gateway+to+Arabidopsis+biology%2C+research+materials+and+community.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001235-Ravasi1" id="pone.0001235-Ravasi1"></a>Ravasi T, Suzuki H, Pang KC, Katayama S, Furuno M, et al.  (2006) Experimental validation of the regulated expression of large numbers of non-coding RNAs from the mouse genome. Genome Res  16: 11–19.  <ul class="find" data-citedArticleID="1023232" data-doi="10.1101/gr.4200206"><li><a href="http://dx.doi.org/10.1101/gr.4200206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+validation+of+the+regulated+expression+of+large+numbers+of+non-coding+RNAs+from+the+mouse+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+validation+of+the+regulated+expression+of+large+numbers+of+non-coding+RNAs+from+the+mouse+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001235-Caron1" id="pone.0001235-Caron1"></a>Caron H, van Schaik B, van der Mee M, Baas F, Riggins G, et al.  (2001) The human transcriptome map: clustering of highly expressed genes in chromosomal domains. Science.  291: 1289–1292.  <ul class="find" data-citedArticleID="1023182" data-doi="10.1126/science.1056794"><li><a href="http://dx.doi.org/10.1126/science.1056794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+transcriptome+map%3A+clustering+of+highly+expressed+genes+in+chromosomal+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+transcriptome+map%3A+clustering+of+highly+expressed+genes+in+chromosomal+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001235-Lercher1" id="pone.0001235-Lercher1"></a>Lercher MJ, Urrutia AO, Hurst LD (2002) Clustering of housekeeping genes provides a unified model of gene order in the human genome. Nat Genet  31: 180–183.  <ul class="find" data-citedArticleID="1023208" data-doi="10.1038/ng887"><li><a href="http://dx.doi.org/10.1038/ng887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+of+housekeeping+genes+provides+a+unified+model+of+gene+order+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+of+housekeeping+genes+provides+a+unified+model+of+gene+order+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001235-Yan1" id="pone.0001235-Yan1"></a>Yan H, Jiang J (2007) Rice as a model for centromere and heterochromatin research. Chromosome Res  15: 77–84.  <ul class="find" data-citedArticleID="1023254" data-doi="10.1007/s10577-006-1104-z"><li><a href="http://dx.doi.org/10.1007/s10577-006-1104-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rice+as+a+model+for+centromere+and+heterochromatin+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rice+as+a+model+for+centromere+and+heterochromatin+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001235-Nagaki1" id="pone.0001235-Nagaki1"></a>Nagaki K, Cheng Z, Ouyang S, Talbert PB, Kim M, et al.  (2004) Sequencing of a rice centromere uncovers active genes. Nat Genet  36: 138–145.  <ul class="find" data-citedArticleID="1023224" data-doi="10.1038/ng1289"><li><a href="http://dx.doi.org/10.1038/ng1289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+of+a+rice+centromere+uncovers+active+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+of+a+rice+centromere+uncovers+active+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001235-Yan2" id="pone.0001235-Yan2"></a>Yan H, Ito H, Nobuta K, Ouyang S, Jin W, et al.  (2006) Genomic and genetic characterization of rice Cen3 reveals extensive transcription and evolutionary implications of a complex centromere. Plant Cell  18: 2123–2133.  <ul class="find" data-citedArticleID="1023256" data-doi="10.1105/tpc.106.043794"><li><a href="http://dx.doi.org/10.1105/tpc.106.043794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+and+genetic+characterization+of+rice+Cen3+reveals+extensive+transcription+and+evolutionary+implications+of+a+complex+centromere." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+and+genetic+characterization+of+rice+Cen3+reveals+extensive+transcription+and+evolutionary+implications+of+a+complex+centromere.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001235-Mizuno1" id="pone.0001235-Mizuno1"></a>Mizuno H, Ito K, Wu J, Tanaka T, Kanamori H, et al.  (2006) Identification and mapping of expressed genes, simple sequence repeats and transposable elements in centromeric regions of rice chromosomes. DNA Res  13: 267–274.  <ul class="find" data-citedArticleID="1023214" data-doi="10.1093/dnares/dsm001"><li><a href="http://dx.doi.org/10.1093/dnares/dsm001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+mapping+of+expressed+genes%2C+simple+sequence+repeats+and+transposable+elements+in+centromeric+regions+of+rice+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+mapping+of+expressed+genes%2C+simple+sequence+repeats+and+transposable+elements+in+centromeric+regions+of+rice+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001235-Zhang2" id="pone.0001235-Zhang2"></a>Zhang X, Yazaki J, Sundaresan A, Cokus S, Chan SW, et al.  (2006) Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis. Cell  126: 1189–1201.  <ul class="find" data-citedArticleID="1023264" data-doi="10.1016/j.cell.2006.08.003"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.08.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+high-resolution+mapping+and+functional+analysis+of+DNA+methylation+in+arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+high-resolution+mapping+and+functional+analysis+of+DNA+methylation+in+arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001235-Finn1" id="pone.0001235-Finn1"></a>Finn RD, Mistry J, Schuster-Bockler B, Griffiths-Jones S, Hollich V, et al.  (2006) Pfam: clans, web tools and services. Nucleic Acids Res  34(Database issue): D247–251.  <ul class="find" data-citedArticleID="1023188" data-doi="10.1093/nar/gkj149"><li><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pfam%3A+clans%2C+web+tools+and+services." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pfam%3A+clans%2C+web+tools+and+services.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001235-Mulder1" id="pone.0001235-Mulder1"></a>Mulder NJ, Apweiler R, Attwood TK, Bairoch A, Bateman A, et al.  (2007) New developments in the InterPro database. Nucleic Acids Res  35(Database issue): D224–228.  <ul class="find" data-citedArticleID="1023218" data-doi="10.1093/nar/gkl841"><li><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+developments+in+the+InterPro+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+developments+in+the+InterPro+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001235-Johansen1" id="pone.0001235-Johansen1"></a>Johansen IE (1996) Intron insertion facilitates amplification of cloned virus cDNA in <em>Escherichia coli</em> while biological activity is reestablished after transcription <em>in vivo</em>. Proc Natl Acad Sci USA  93: 12400–12405.  <ul class="find" data-citedArticleID="1023204" data-doi="10.1073/pnas.93.22.12400"><li><a href="http://dx.doi.org/10.1073/pnas.93.22.12400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intron+insertion+facilitates+amplification+of+cloned+virus+cDNA+in+Escherichia+coli+while+biological+activity+is+reestablished+after+transcription+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intron+insertion+facilitates+amplification+of+cloned+virus+cDNA+in+Escherichia+coli+while+biological+activity+is+reestablished+after+transcription+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001235-Jakab1" id="pone.0001235-Jakab1"></a>Jakab G, Droz E, Brigneti G, Baulcombe D, Malnoe P (1997) Infectious <em>in vivo</em> and <em>in vitro</em> transcripts from a full-length cDNA clone of PVY-N605, a Swiss necrotic isolate of potato virus Y. J Gen Virol  78: 3141–3145.  <ul class="find" data-citedArticleID="1023202"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Infectious+in+vivo+and+in+vitro+transcripts+from+a+full-length+cDNA+clone+of+PVY-N605%2C+a+Swiss+necrotic+isolate+of+potato+virus+Y.&amp;auth=&amp;atitle=Infectious+in+vivo+and+in+vitro+transcripts+from+a+full-length+cDNA+clone+of+PVY-N605%2C+a+Swiss+necrotic+isolate+of+potato+virus+Y." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Infectious+in+vivo+and+in+vitro+transcripts+from+a+full-length+cDNA+clone+of+PVY-N605%2C+a+Swiss+necrotic+isolate+of+potato+virus+Y." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Infectious+in+vivo+and+in+vitro+transcripts+from+a+full-length+cDNA+clone+of+PVY-N605%2C+a+Swiss+necrotic+isolate+of+potato+virus+Y.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001235-Choi1" id="pone.0001235-Choi1"></a>Choi IR, French R, Hein GL, Stenger DC (1999) Fully biologically active <em>in vitro</em> transcripts of the eriophyid mite-transmitted wheat streak mosaic tritimovirus. Phytopathology  89: 1182–1185.  <ul class="find" data-citedArticleID="1023184" data-doi="10.1094/phyto.1999.89.12.1182"><li><a href="http://dx.doi.org/10.1094/phyto.1999.89.12.1182" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fully+biologically+active+in+vitro+transcripts+of+the+eriophyid+mite-transmitted+wheat+streak+mosaic+tritimovirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fully+biologically+active+in+vitro+transcripts+of+the+eriophyid+mite-transmitted+wheat+streak+mosaic+tritimovirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001235-Nadeau1" id="pone.0001235-Nadeau1"></a>Nadeau K, Das A, Walsh CT (1993) Hsp90 chaperonins possess ATPase activity and bind heat shock transcription factors and peptidyl prolyl isomerases. J Biol Chem  268: 1479–1487.  <ul class="find" data-citedArticleID="1023222"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hsp90+chaperonins+possess+ATPase+activity+and+bind+heat+shock+transcription+factors+and+peptidyl+prolyl+isomerases.&amp;auth=&amp;atitle=Hsp90+chaperonins+possess+ATPase+activity+and+bind+heat+shock+transcription+factors+and+peptidyl+prolyl+isomerases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp90+chaperonins+possess+ATPase+activity+and+bind+heat+shock+transcription+factors+and+peptidyl+prolyl+isomerases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp90+chaperonins+possess+ATPase+activity+and+bind+heat+shock+transcription+factors+and+peptidyl+prolyl+isomerases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001235-Costa1" id="pone.0001235-Costa1"></a>Costa FF (2007) Non-coding RNAs: lost in translation? Gene  386: 1–10.  <ul class="find" data-citedArticleID="1023186" data-doi="10.1016/j.gene.2006.09.028"><li><a href="http://dx.doi.org/10.1016/j.gene.2006.09.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-coding+RNAs%3A+lost+in+translation%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-coding+RNAs%3A+lost+in+translation%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001235-Mizuno2" id="pone.0001235-Mizuno2"></a>Mizuno H, Wu J, Kanamori H, Fujisawa M, Namiki N, et al.  (2006) Sequencing and characterization of telomere and subtelomere regions on rice chromosomes 1S, 2S, 2L, 6L, 7S, 7L and 8S. Plant J  46: 206–217.  <ul class="find" data-citedArticleID="1023216" data-doi="10.1111/j.1365-313x.2006.02684.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02684.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequencing+and+characterization+of+telomere+and+subtelomere+regions+on+rice+chromosomes+1S%2C+2S%2C+2L%2C+6L%2C+7S%2C+7L+and+8S." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequencing+and+characterization+of+telomere+and+subtelomere+regions+on+rice+chromosomes+1S%2C+2S%2C+2L%2C+6L%2C+7S%2C+7L+and+8S.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001235-Sugawara1" id="pone.0001235-Sugawara1"></a>Sugawara H, Abe T, Gojobori T, Tateno Y (2007) DDBJ working on evaluation and classification of bacterial genes in INSDC. Nucleic Acids Res  35(Database issue): D13–15.  <ul class="find" data-citedArticleID="1023246" data-doi="10.1093/nar/gkl908"><li><a href="http://dx.doi.org/10.1093/nar/gkl908" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DDBJ+working+on+evaluation+and+classification+of+bacterial+genes+in+INSDC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DDBJ+working+on+evaluation+and+classification+of+bacterial+genes+in+INSDC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001235-Kent1" id="pone.0001235-Kent1"></a>Kent WJ (2002) BLAT: The BLAST-like alignment tool. Genome Res  12: 656–664.  <ul class="find" data-citedArticleID="1023206" data-doi="10.1101/gr.229202."><li><a href="http://dx.doi.org/10.1101/gr.229202." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLAT%3A+The+BLAST-like+alignment+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLAT%3A+The+BLAST-like+alignment+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001235-Murashige1" id="pone.0001235-Murashige1"></a>Murashige T, Skoog F (1962) A revised medium for rapid growth and bioassays with tobacco tissue cultures. Physiol Plant  15: 473–497.  <ul class="find" data-citedArticleID="1023220" data-doi="10.1111/j.1399-3054.1962.tb08052.x"><li><a href="http://dx.doi.org/10.1111/j.1399-3054.1962.tb08052.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+revised+medium+for+rapid+growth+and+bioassays+with+tobacco+tissue+cultures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+revised+medium+for+rapid+growth+and+bioassays+with+tobacco+tissue+cultures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001235-Barrett1" id="pone.0001235-Barrett1"></a>Barrett T, Troup DB, Wilhite SE, Ledoux P, Rudnev D, et al.  (2007) NCBI GEO: mining tens of millions of expression profiles–database and tools update. Nucleic Acids Res  35(Database issue): D760–765.  <ul class="find" data-citedArticleID="1023180" data-doi="10.1093/nar/gkl887"><li><a href="http://dx.doi.org/10.1093/nar/gkl887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+GEO%3A+mining+tens+of+millions+of+expression+profiles%E2%80%93database+and+tools+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+GEO%3A+mining+tens+of+millions+of+expression+profiles%E2%80%93database+and+tools+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.XML" value="188136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.PDF" value="302483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t001.PNG_L" value="219526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t001.PNG_M" value="120817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t001.PNG_S" value="15090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t001.TIF" value="1121756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t001.PNG_I" value="37305"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t002.PNG_L" value="47199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t002.PNG_M" value="60898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t002.PNG_S" value="11516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t002.TIF" value="234180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t002.PNG_I" value="20308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g001.PNG_L" value="139914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g001.PNG_M" value="31264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g001.PNG_S" value="7063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g001.TIF" value="315136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g001.PNG_I" value="16904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t003.PNG_L" value="108724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t003.PNG_M" value="62933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t003.PNG_S" value="9343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t003.TIF" value="446166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t003.PNG_I" value="19823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t004.PNG_L" value="57075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t004.PNG_M" value="81930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t004.PNG_S" value="13436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t004.TIF" value="247256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t004.PNG_I" value="27319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t005.PNG_L" value="107537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t005.PNG_M" value="58954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t005.PNG_S" value="9759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t005.TIF" value="514804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t005.PNG_I" value="18033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t006.PNG_L" value="114355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t006.PNG_M" value="160982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t006.PNG_S" value="19746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t006.TIF" value="462414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t006.PNG_I" value="54196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t007.PNG_L" value="150425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t007.PNG_M" value="148097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t007.PNG_S" value="19773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t007.TIF" value="691898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t007.PNG_I" value="80810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t008.PNG_L" value="285421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t008.PNG_M" value="155880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t008.PNG_S" value="15042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t008.TIF" value="1152934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t008.PNG_I" value="48117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t009.PNG_L" value="288894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t009.PNG_M" value="162481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t009.PNG_S" value="15987"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t009.TIF" value="1233306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t009.PNG_I" value="50578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t010.PNG_L" value="242794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t010.PNG_M" value="135768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t010.PNG_S" value="13755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t010.TIF" value="998272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.t010.PNG_I" value="41663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g002.PNG_L" value="537750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g002.PNG_M" value="51002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g002.PNG_S" value="9269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g002.TIF" value="735842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.g002.PNG_I" value="31091"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s001.TIF" value="8795460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s002.DOC" value="8849236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s003.ZIP" value="2659277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s004.ZIP" value="3095777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s005.XLS" value="18432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s006.XLS" value="14336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s007.ZIP" value="106278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s008.XLS" value="19456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s009.XLS" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001235.s010.ZIP" value="3575124"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001235&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001235" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001235&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001235&volume=&issue=&title=Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray&author_name=Kouji%20Satoh%2C%20Koji%20Doi%2C%20Toshifumi%20Nagata%2C%20Naoki%20Kishimoto%2C%20Kohji%20Suzuki%2C%20Yasuhiro%20Otomo%2C%20Jun%20Kawai%2C%20Mari%20Nakamura%2C%20Tomoko%20Hirozane-Kishikawa%2C%20Saeko%20Kanagawa%2C%20Takahiro%20Arakawa%2C%20Juri%20Takahashi-Iida%2C%20Mitsuyoshi%20Murata%2C%20Noriko%20Ninomiya%2C%20Daisuke%20Sasaki%2C%20Shiro%20Fukuda%2C%20Michihira%20Tagami%2C%20Harumi%20Yamagata%2C%20Kanako%20Kurita%2C%20Kozue%20Kamiya%2C%20Mayu%20Yamamoto%2C%20Ari%20Kikuta%2C%20Takahito%20Bito%2C%20Nahoko%20Fujitsuka%2C%20Kazue%20Ito%2C%20Hiroyuki%20Kanamori%2C%20Il-Ryong%20Choi%2C%20Yoshiaki%20Nagamura%2C%20Takashi%20Matsumoto%2C%20Kazuo%20Murakami%2C%20Ken-ichi%20Matsubara%2C%20Piero%20Carninci%2C%20Yoshihide%20Hayashizaki%2C%20Shoshi%20Kikuchi&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001235" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235&amp;t=Gene%20Organization%20in%20Rice%20Revealed%20by%20Full-Length%20cDNA%20Mapping%20and%20Gene%20Expression%20Analysis%20through%20Microarray" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235&title=Gene%20Organization%20in%20Rice%20Revealed%20by%20Full-Length%20cDNA%20Mapping%20and%20Gene%20Expression%20Analysis%20through%20Microarray&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235&amp;title=Gene%20Organization%20in%20Rice%20Revealed%20by%20Full-Length%20cDNA%20Mapping%20and%20Gene%20Expression%20Analysis%20through%20Microarray" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001235&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Gene Organization in Rice Revealed by Full-Length cDNA Mapping and Gene Expression Analysis through Microarray';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001235';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Gene%20Organization%20in%20Rice%20Revealed%20by%20Full-Length%20cDNA%20Mapping%20and%20Gene%20Expression%20Analysis%20through%20Microarray http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001235" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001235" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">





















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="25302"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cloning%22" title="Search for articles in the subject area:'Cloning'"><div class="flagText">Cloning</div></a>
              <div data-categoryid="23551" data-articleid="25302"
                   data-categoryname="Cloning"
                   class="flagImage" title="Flag 'Cloning' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="25302"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25302"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25302"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="25302"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rice%22" title="Search for articles in the subject area:'Rice'"><div class="flagText">Rice</div></a>
              <div data-categoryid="31915" data-articleid="25302"
                   data-categoryname="Rice"
                   class="flagImage" title="Flag 'Rice' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="25302"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4480'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8865'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9913&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
