

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000766"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000766" />

    <meta name="citation_title" content="The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists"/>
    <meta itemprop="name" content="The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists"/>

      <meta name="citation_author" content="Sven Heinicke"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Michael S. Livstone"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Charles Lu"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Rose Oughtred"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Fan Kang"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Samuel V. Angiuoli"/>
            <meta name="citation_author_institution" content="The Institute for Genomic Research, Rockville, Maryland, United States of America"/>
            <meta name="citation_author_institution" content="Center for Bioinformatics and Computational Biology, University of Maryland, College Park, Maryland, United States of America"/>
      <meta name="citation_author" content="Owen White"/>
            <meta name="citation_author_institution" content="The Institute for Genomic Research, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="David Botstein"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Kara Dolinski"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>

    <meta name="citation_date" content="2007/8/22"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000766.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e766"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=GenBank.; citation_author=DA Benson; citation_author=I Karsch-Mizrachi; citation_author=DJ Lipman; citation_author=J Ostell; citation_author=DL Wheeler; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=1; citation_pages=D21-25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=EMBL Nucleotide Sequence Database in 2006.; citation_author=T Kulikova; citation_author=R Akhtar; citation_author=P Aldebert; citation_author=N Althorpe; citation_author=M Andersson; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=2; citation_pages=D16-20; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Overview and comparison of ortholog databases.; citation_author=A Alexeyenko; citation_author=J Lindberg; citation_author=A Perez-Bercoff; citation_author=ELL Sonnhammer; citation_journal_title=Drug Discov Today; citation_volume=11; citation_number=3; citation_pages=137-143; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes.; citation_author=L Li; citation_author=CJ Stoeckert; citation_author=DS Roos; citation_journal_title=Genome Res; citation_volume=13; citation_number=4; citation_pages=2178-2189; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=5; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=OMIA (Online Mendelian Inheritance in Animals): an enhanced platform and integration into the Entrez search interface at NCBI.; citation_author=J Lenffer; citation_author=FW Nicholas; citation_author=K Castle; citation_author=A Rao; citation_author=S Gregory; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=6; citation_pages=D599-601; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Assessing performance of orthology detection strategies applied to eukaryotic genomes.; citation_author=F Chen; citation_author=AJ Mackey; citation_author=JK Vermunt; citation_author=DS Roos; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=7; citation_pages=e383; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Saccharomyces Genome Database (SGD) provides secondary gene annotation using the Gene Ontology (GO).; citation_author=SS Dwight; citation_author=MA Harris; citation_author=K Dolinski; citation_author=CA Ball; citation_author=G Binkley; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=8; citation_pages=69-72; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=GO::TermFinder–open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes.; citation_author=EI Boyle; citation_author=S Weng; citation_author=J Gollub; citation_author=H Jin; citation_author=D Botstein; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=9; citation_pages=3710-3715; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Orthologs, paralogs, and evolutionary genomics.; citation_author=EV Koonin; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=10; citation_pages=309-338; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Yeast actin filaments display ATP-dependent sliding movement over surfaces coated with rabbit muscle myosin.; citation_author=SJ Kron; citation_author=DG Drubin; citation_author=D Botstein; citation_author=JA Spudich; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=11; citation_pages=4466-4470; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Alpha-tubulin from early-diverging eukaryotic lineages and the evolution of the tubulin family.; citation_author=PJ Keeling; citation_author=WF Doolittle; citation_journal_title=Mol Biol Evol; citation_volume=13; citation_number=12; citation_pages=1297-1305; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Long-lost relatives reappear: identification of new members of the tubulin superfamily.; citation_author=SK Dutcher; citation_journal_title=Curr Opin Microbiol; citation_volume=6; citation_number=13; citation_pages=634-640; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetics of eukaryotic RNA polymerases I, II, and III.; citation_author=J Archambault; citation_author=JD Friesen; citation_journal_title=Microbiol Rev; citation_volume=57; citation_number=14; citation_pages=703-724; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=RNA polymerase IV directs silencing of endogenous DNA.; citation_author=AJ Herr; citation_author=MB Jensen; citation_author=T Dalmay; citation_author=DC Baulcombe; citation_journal_title=Science; citation_volume=308; citation_number=15; citation_pages=118-120; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Plant nuclear RNA polymerase IV mediates siRNA and DNA methylation-dependent heterochromatin formation.; citation_author=Y Onodera; citation_author=JR Haag; citation_author=T Ream; citation_author=PC Nunes; citation_author=O Pontes; citation_journal_title=Cell; citation_volume=120; citation_number=16; citation_pages=613-622; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Glycosylphosphatidylinositol anchors represent the major carbohydrate modification in proteins of intraerythrocytic stage Plasmodium falciparum.; citation_author=DC Gowda; citation_author=P Gupta; citation_author=EA Davidson; citation_journal_title=J Biol Chem; citation_volume=272; citation_number=17; citation_pages=6428-6439; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Plasmodium falciparum: glycosylation status of Plasmodium falciparum circumsporozoite protein expressed in the baculovirus system.; citation_author=MH Kedees; citation_author=N Azzouz; citation_author=P Gerold; citation_author=H Shams-Eldin; citation_author=J Iqbal; citation_journal_title=Exp Parasitol; citation_volume=101; citation_number=18; citation_pages=64-68; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High-level expression of the malaria blood-stage vaccine candidate Plasmodium falciparum apical membrane antigen 1 and induction of antibodies that inhibit erythrocyte invasion.; citation_author=CH Kocken; citation_author=C Withers-Martinez; citation_author=MA Dubbeld; citation_author=A van der Wel; citation_author=F Hackett; citation_journal_title=Infect Immun; citation_volume=70; citation_number=19; citation_pages=4471-4476; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Automatic clustering of orthologs and inparalogs shared by multiple proteomes.; citation_author=A Alexeyenko; citation_author=I Tamas; citation_author=G Liu; citation_author=EL Sonnhammer; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=20; citation_pages=e9-15; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=COCO-CL: hierarchical clustering of homology relations based on evolutionary correlations.; citation_author=R Jothi; citation_author=E Zotenko; citation_author=A Tasneem; citation_author=TM Przytycka; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=21; citation_pages=779-788; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes.; citation_author=RL Tatusov; citation_author=ND Fedorova; citation_author=JD Jackson; citation_author=AR Jacobs; citation_author=B Kiryutin; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=22; citation_pages=41; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cross-referencing eukaryotic genomes: TIGR Orthologous Gene Alignments (TOGA).; citation_author=Y Lee; citation_author=R Sultana; citation_author=G Pertea; citation_author=J Cho; citation_author=S Karamycheva; citation_journal_title=Genome Res; citation_volume=12; citation_number=23; citation_pages=493-502; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=24; citation_pages=D173-180; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inparanoid: a comprehensive database of eukaryotic orthologs.; citation_author=KP O'Brien; citation_author=M Remm; citation_author=EL Sonnhammer; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=25; citation_pages=D476-480; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=OrthoDisease: a database of human disease orthologs.; citation_author=KP O'Brien; citation_author=I Westerlund; citation_author=EL Sonnhammer; citation_journal_title=Hum Mutat; citation_volume=24; citation_number=26; citation_pages=112-119; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups.; citation_author=F Chen; citation_author=AJ Mackey; citation_author=CJ Stoeckert; citation_author=DS Roos; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=27; citation_pages=D363-368; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=YOGY: a web-based, integrated database to retrieve protein orthologs and associated Gene Ontology terms.; citation_author=CJ Penkett; citation_author=JA Morris; citation_author=V Wood; citation_author=J Bahler; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=28; citation_pages=W330-334; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=GeneMCL in microarray analysis.; citation_author=B Samuel Lattimore; citation_author=S van Dongen; citation_author=MJ Crabbe; citation_journal_title=Comput Biol Chem; citation_volume=29; citation_number=29; citation_pages=354-359; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Saccharomyces cerevisiae oligosaccharyltransferase is a protein complex composed of Wbp1p, Swp1p, and four additional polypeptides.; citation_author=DJ Kelleher; citation_author=R Gilmore; citation_journal_title=J Biol Chem; citation_volume=269; citation_number=30; citation_pages=12908-12917; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Functional analyses of mammalian protein kinase C isozymes in budding yeast and mammalian fibroblasts.; citation_author=S Nomoto; citation_author=Y Watanabe; citation_author=J Ninomiya-Tsuji; citation_author=LX Yang; citation_author=Y Nagai; citation_journal_title=Genes Cells; citation_volume=2; citation_number=31; citation_pages=601-614; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Localization of mouse CLC-6 and CLC-7 mRNA and their functional complementation of yeast CLC gene mutant.; citation_author=Y Kida; citation_author=S Uchida; citation_author=H Miyazaki; citation_author=S Sasaki; citation_author=F Marumo; citation_journal_title=Histochem Cell Biol; citation_volume=115; citation_number=32; citation_pages=189-194; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Bloom's and Werner's syndrome genes suppress hyperrecombination in yeast sgs1 mutant: implication for genomic instability in human diseases.; citation_author=K Yamagata; citation_author=J Kato; citation_author=A Shimamoto; citation_author=M Goto; citation_author=Y Furuichi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=33; citation_pages=8733-8738; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional analysis of human MutSalpha and MutSbeta complexes in yeast.; citation_author=AB Clark; citation_author=ME Cook; citation_author=HT Tran; citation_author=DA Gordenin; citation_author=MA Resnick; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=34; citation_pages=736-742; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A mutation in protein phosphatase 2A regulatory subunit A affects auxin transport in Arabidopsis.; citation_author=C Garbers; citation_author=A DeLong; citation_author=J Deruere; citation_author=P Bernasconi; citation_author=D Soll; citation_journal_title=Embo J; citation_volume=15; citation_number=35; citation_pages=2115-2124; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Identification and functional analysis of a defect in the human ALG9 gene: definition of congenital disorder of glycosylation type IL.; citation_author=CG Frank; citation_author=CE Grubenmann; citation_author=W Eyaid; citation_author=EG Berger; citation_author=M Aebi; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=36; citation_pages=146-150; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Deficiency of GDP-Man:GlcNAc2-PP-dolichol mannosyltransferase causes congenital disorder of glycosylation type Ik.; citation_author=M Schwarz; citation_author=C Thiel; citation_author=J Lubbehusen; citation_author=B Dorland; citation_author=T de Koning; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=37; citation_pages=472-481; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The NF1 locus encodes a protein functionally related to mammalian GAP and yeast IRA proteins.; citation_author=R Ballester; citation_author=D Marchuk; citation_author=M Boguski; citation_author=A Saulino; citation_author=R Letcher; citation_journal_title=Cell; citation_volume=63; citation_number=38; citation_pages=851-859; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Functional significance of lysine 1423 of neurofibromin and characterization of a second site suppressor which rescues mutations at this residue and suppresses RAS2Val-19-activated phenotypes.; citation_author=P Poullet; citation_author=B Lin; citation_author=K Esson; citation_author=F Tamanoi; citation_journal_title=Mol Cell Biol; citation_volume=14; citation_number=39; citation_pages=815-821; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Human wild-type SEDL protein functionally complements yeast Trs20p but some naturally occurring SEDL mutants do not.; citation_author=J Gecz; citation_author=MA Shaw; citation_author=JR Bellon; citation_author=M de Barros Lopes; citation_journal_title=Gene; citation_volume=320; citation_number=40; citation_pages=137-144; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ERS1 encodes a functional homologue of the human lysosomal cystine transporter.; citation_author=XD Gao; citation_author=J Wang; citation_author=S Keppler-Ross; citation_author=N Dean; citation_journal_title=Febs J; citation_volume=272; citation_number=41; citation_pages=2497-2511; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human frataxin maintains mitochondrial iron homeostasis in Saccharomyces cerevisiae.; citation_author=P Cavadini; citation_author=C Gellera; citation_author=PI Patel; citation_author=G Isaya; citation_journal_title=Hum Mol Genet; citation_volume=9; citation_number=42; citation_pages=2523-2530; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Expression of the human ferritin light chain in a frataxin mutant yeast affects ageing and cell death.; citation_author=L Desmyter; citation_author=S Dewaele; citation_author=R Reekmans; citation_author=T Nystrom; citation_author=R Contreras; citation_journal_title=Exp Gerontol; citation_volume=39; citation_number=43; citation_pages=707-715; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The higher plant Arabidopsis thaliana encodes a functional CDC48 homologue which is highly expressed in dividing and expanding cells.; citation_author=HS Feiler; citation_author=T Desprez; citation_author=V Santoni; citation_author=J Kronenberger; citation_author=M Caboche; citation_journal_title=Embo J; citation_volume=14; citation_number=44; citation_pages=5626-5637; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Functional assessment of the carboxy-terminus of the Wilson disease copper-transporting ATPase, ATP7B.; citation_author=G Hsi; citation_author=LM Cullen; citation_author=D Moira Glerum; citation_author=DW Cox; citation_journal_title=Genomics; citation_volume=83; citation_number=45; citation_pages=473-481; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The nucleotide sequence of Saccharomyces cerevisiae chromosome XVI.; citation_author=H Bussey; citation_author=RK Storms; citation_author=A Ahmed; citation_author=K Albermann; citation_author=E Allen; citation_journal_title=Nature; citation_volume=387; citation_number=46; citation_pages=103-105; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Purification and functional reconstitution of the human Wilson copper ATPase, ATP7B.; citation_author=R Portmann; citation_author=M Solioz; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=47; citation_pages=3589-3595; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Caenorhabditis elegans cDNA for a Menkes/Wilson disease gene homologue and its function in a yeast CCC2 gene deletion mutant.; citation_author=Y Sambongi; citation_author=T Wakabayashi; citation_author=T Yoshimizu; citation_author=H Omote; citation_author=T Oka; citation_journal_title=J Biochem (Tokyo); citation_volume=121; citation_number=48; citation_pages=1169-1175; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Copper-induced trafficking of the cU-ATPases: a key mechanism for copper homeostasis.; citation_author=JF Mercer; citation_author=N Barnes; citation_author=J Stevenson; citation_author=D Strausak; citation_author=RM Llanos; citation_journal_title=Biometals; citation_volume=16; citation_number=49; citation_pages=175-184; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Functional expression of the menkes disease protein reveals common biochemical mechanisms among the copper-transporting P-type ATPases.; citation_author=AS Payne; citation_author=JD Gitlin; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=50; citation_pages=3765-3770; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=SEM1, a homologue of the split hand/split foot malformation candidate gene Dss1, regulates exocytosis and pseudohyphal differentiation in yeast.; citation_author=J Jantti; citation_author=J Lahdenranta; citation_author=VM Olkkonen; citation_author=H Soderlund; citation_author=S Keranen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=51; citation_pages=909-914; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Sem1p is a novel subunit of the 26 S proteasome from Saccharomyces cerevisiae.; citation_author=T Sone; citation_author=Y Saeki; citation_author=A Toh-e; citation_author=H Yokosawa; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=52; citation_pages=28807-28816; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A mouse homolog of the Escherichia coli recA and Saccharomyces cerevisiae RAD51 genes.; citation_author=T Morita; citation_author=Y Yoshimura; citation_author=A Yamamoto; citation_author=K Murata; citation_author=M Mori; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=53; citation_pages=6577-6580; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=A highly conserved binding site in vesicle-associated membrane protein-associated protein (VAP) for the FFAT motif of lipid-binding proteins.; citation_author=CJ Loewen; citation_author=TP Levine; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=54; citation_pages=14097-14104; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lethality in yeast of trichothiodystrophy (TTD) mutations in the human xeroderma pigmentosum group D gene. Implications for transcriptional defect in TTD.; citation_author=SN Guzder; citation_author=P Sung; citation_author=S Prakash; citation_author=L Prakash; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=55; citation_pages=17660-17663; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Human xeroderma pigmentosum group D gene encodes a DNA helicase.; citation_author=P Sung; citation_author=V Bailly; citation_author=C Weber; citation_author=LH Thompson; citation_author=L Prakash; citation_journal_title=Nature; citation_volume=365; citation_number=56; citation_pages=852-855; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Functional analysis in Saccharomyces cerevisiae of naturally occurring amino acid substitutions in human dihydrolipoamide dehydrogenase.; citation_author=MM Lanterman; citation_author=JR Dickinson; citation_author=DJ Danner; citation_journal_title=Hum Mol Genet; citation_volume=5; citation_number=57; citation_pages=1643-1648; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Complementation analysis in PtdInsP kinase-deficient yeast mutants demonstrates that Schizosaccharomyces pombe and murine Fab1p homologues are phosphatidylinositol 3-phosphate 5-kinases.; citation_author=RK McEwen; citation_author=SK Dove; citation_author=FT Cooke; citation_author=GF Painter; citation_author=AB Holmes; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=58; citation_pages=33905-33912; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Human pancreatic glucokinase (GlkB) complements the glucose signalling defect of Saccharomyces cerevisiae hxk2 mutants.; citation_author=I Mayordomo; citation_author=P Sanz; citation_journal_title=Yeast; citation_volume=18; citation_number=59; citation_pages=1309-1316; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of two novel mutations in the murine Nsdhl sterol dehydrogenase gene and development of a functional complementation assay in yeast.; citation_author=ME Lucas; citation_author=Q Ma; citation_author=D Cunningham; citation_author=J Peters; citation_author=B Cattanach; citation_journal_title=Mol Genet Metab; citation_volume=80; citation_number=60; citation_pages=227-233; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The ACA4 gene of Arabidopsis encodes a vacuolar membrane calcium pump that improves salt tolerance in yeast.; citation_author=M Geisler; citation_author=N Frangne; citation_author=E Gomes; citation_author=E Martinoia; citation_author=MG Palmgren; citation_journal_title=Plant Physiol; citation_volume=124; citation_number=61; citation_pages=1814-1827; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A plant plasma membrane Ca2+ pump is required for normal pollen tube growth and fertilization.; citation_author=M Schiott; citation_author=SM Romanowsky; citation_author=L Baekgaard; citation_author=MK Jakobsen; citation_author=MG Palmgren; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=62; citation_pages=9502-9507; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Functional identification of an Arabidopsis snf4 ortholog by screening for heterologous multicopy suppressors of snf4 deficiency in yeast.; citation_author=T Kleinow; citation_author=R Bhalerao; citation_author=F Breuer; citation_author=M Umeda; citation_author=K Salchert; citation_journal_title=Plant J; citation_volume=23; citation_number=63; citation_pages=115-122; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Domain fusion between SNF1-related kinase subunits during plant evolution.; citation_author=V Lumbreras; citation_author=MM Alba; citation_author=T Kleinow; citation_author=C Koncz; citation_author=M Pages; citation_journal_title=EMBO Rep; citation_volume=2; citation_number=64; citation_pages=55-60; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Isolation, characterization, and functional expression of cDNAs encoding NADH-dependent methylenetetrahydrofolate reductase from higher plants.; citation_author=S Roje; citation_author=H Wang; citation_author=SD McNeil; citation_author=RK Raymond; citation_author=DR Appling; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=65; citation_pages=36089-36096; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Saccharomyces cerevisiae expresses two genes encoding isozymes of methylenetetrahydrofolate reductase.; citation_author=RK Raymond; citation_author=EK Kastanos; citation_author=DR Appling; citation_journal_title=Arch Biochem Biophys; citation_volume=372; citation_number=66; citation_pages=300-308; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Functional expression in yeast of the human secretory pathway Ca(2+), Mn(2+)-ATPase defective in Hailey-Hailey disease.; citation_author=VK Ton; citation_author=D Mandal; citation_author=C Vahadji; citation_author=R Rao; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=67; citation_pages=6422-6427; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Expression of Hailey-Hailey disease mutations in yeast.; citation_author=VK Ton; citation_author=R Rao; citation_journal_title=J Invest Dermatol; citation_volume=123; citation_number=68; citation_pages=1192-1194; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hailey-Hailey disease as an orthodisease of PMR1 deficiency in Saccharomyces cerevisiae.; citation_author=R Kellermayer; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=69; citation_pages=2021-2025; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression of heterologous phosphofructokinase genes in yeast.; citation_author=JJ Heinisch; citation_journal_title=FEBS Lett; citation_volume=328; citation_number=70; citation_pages=35-40; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Functional expression of human mutant phosphofructokinase in yeast: genetic defects in French Canadian and Swiss patients with phosphofructokinase deficiency.; citation_author=N Raben; citation_author=R Exelbert; citation_author=R Spiegel; citation_author=JB Sherman; citation_author=H Nakajima; citation_journal_title=Am J Hum Genet; citation_volume=56; citation_number=71; citation_pages=131-141; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=GTP-cyclohydrolase I gene mutations in patients with autosomal dominant and recessive GTP-CH1 deficiency: identification and functional characterization of four novel mutations.; citation_author=B Garavaglia; citation_author=F Invernizzi; citation_author=ML Carbone; citation_author=V Viscardi; citation_author=F Saracino; citation_journal_title=J Inherit Metab Dis; citation_volume=27; citation_number=72; citation_pages=455-463; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complementation of the fol2 deletion in Saccharomyces cerevisiae by human and Escherichia coli genes encoding GTP cyclohydrolase I.; citation_author=R Mancini; citation_author=F Saracino; citation_author=G Buscemi; citation_author=M Fischer; citation_author=N Schramek; citation_journal_title=Biochem Biophys Res Commun; citation_volume=255; citation_number=73; citation_pages=521-527; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mutations in the Delta1-pyrroline 5-carboxylate dehydrogenase gene cause type II hyperprolinemia.; citation_author=MT Geraghty; citation_author=D Vaughn; citation_author=AJ Nicholson; citation_author=WW Lin; citation_author=G Jimenez-Sanchez; citation_journal_title=Hum Mol Genet; citation_volume=7; citation_number=74; citation_pages=1411-1415; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cloning, characterization, and expression of cDNAs encoding human delta 1-pyrroline-5-carboxylate dehydrogenase.; citation_author=CA Hu; citation_author=WW Lin; citation_author=D Valle; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=75; citation_pages=9795-9800; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Functional XPB/RAD25 redundancy in Arabidopsis genome: characterization of AtXPB2 and expression analysis.; citation_author=PG Morgante; citation_author=CM Berra; citation_author=M Nakabashi; citation_author=RM Costa; citation_author=CF Menck; citation_journal_title=Gene; citation_volume=344; citation_number=76; citation_pages=93-103; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A yeast model for the study of Batten disease.; citation_author=DA Pearce; citation_author=F Sherman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=77; citation_pages=6915-6918; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of Drosophila gamma-glutamylcysteine synthetase by functional complementation of a yeast mutant.; citation_author=RD Saunders; citation_author=LI McLellan; citation_journal_title=FEBS Lett; citation_volume=467; citation_number=78; citation_pages=337-340; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Yeast lacking superoxide dismutase(s) show elevated levels of “free iron” as measured by whole cell electron paramagnetic resonance.; citation_author=C Srinivasan; citation_author=A Liba; citation_author=JA Imlay; citation_author=JS Valentine; citation_author=EB Gralla; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=79; citation_pages=29187-29192; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Zinc metalloproteinase, ZMPSTE24, is mutated in mandibuloacral dysplasia.; citation_author=AK Agarwal; citation_author=JP Fryns; citation_author=RJ Auchus; citation_author=A Garg; citation_journal_title=Hum Mol Genet; citation_volume=12; citation_number=80; citation_pages=1995-2001; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reconstitution of the Ste24p-dependent N-terminal proteolytic step in yeast a-factor biogenesis.; citation_author=WK Schmidt; citation_author=A Tam; citation_author=S Michaelis; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=81; citation_pages=6227-6233; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The C66W mutation in the deafness dystonia peptide 1 (DDP1) affects the formation of functional DDP1.TIM13 complexes in the mitochondrial intermembrane space.; citation_author=S Hofmann; citation_author=U Rothbauer; citation_author=N Muhlenbein; citation_author=W Neupert; citation_author=KD Gerbitz; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=82; citation_pages=23287-23293; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Role of the deafness dystonia peptide 1 (DDP1) in import of human Tim23 into the inner membrane of mitochondria.; citation_author=U Rothbauer; citation_author=S Hofmann; citation_author=N Muhlenbein; citation_author=SA Paschen; citation_author=KD Gerbitz; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=83; citation_pages=37327-37334; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional complementation of yeast ste6 by a mammalian multidrug resistance mdr gene.; citation_author=M Raymond; citation_author=P Gros; citation_author=M Whiteway; citation_author=DY Thomas; citation_journal_title=Science; citation_volume=256; citation_number=84; citation_pages=232-234; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Effect of ATP binding cassette/multidrug resistance proteins on ATP efflux of Saccharomyces cerevisiae.; citation_author=R Boyum; citation_author=G Guidotti; citation_journal_title=Biochem Biophys Res Commun; citation_volume=230; citation_number=85; citation_pages=22-26; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Characterization of TRZ1, a yeast homolog of the human candidate prostate cancer susceptibility gene ELAC2 encoding tRNase Z.; citation_author=Y Chen; citation_author=A Beck; citation_author=C Davenport; citation_author=Y Chen; citation_author=D Shattuck; citation_journal_title=BMC Mol Biol; citation_volume=6; citation_number=86; citation_pages=12; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reciprocal regulation of delta 1-pyrroline-5-carboxylate synthetase and proline dehydrogenase genes controls proline levels during and after osmotic stress in plants.; citation_author=Z Peng; citation_author=Q Lu; citation_author=DP Verma; citation_journal_title=Mol Gen Genet; citation_volume=253; citation_number=87; citation_pages=334-341; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Uracil-DNA glycosylase-deficient yeast exhibit a mitochondrial mutator phenotype.; citation_author=A Chatterjee; citation_author=KK Singh; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=88; citation_pages=4935-4940; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=BLM helicase complements disrupted type II telomere lengthening in telomerase-negative sgs1 yeast.; citation_author=K Lillard-Wetherell; citation_author=KA Combs; citation_author=J Groden; citation_journal_title=Cancer Res; citation_volume=65; citation_number=89; citation_pages=5520-5522; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The DNA helicase activity of BLM is necessary for the correction of the genomic instability of bloom syndrome cells.; citation_author=NF Neff; citation_author=NA Ellis; citation_author=TZ Ye; citation_author=J Noonan; citation_author=K Huang; citation_journal_title=Mol Biol Cell; citation_volume=10; citation_number=90; citation_pages=665-676; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=ALG12 mannosyltransferase defect in congenital disorder of glycosylation type lg.; citation_author=CE Grubenmann; citation_author=CG Frank; citation_author=S Kjaergaard; citation_author=EG Berger; citation_author=M Aebi; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=91; citation_pages=2331-2339; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Isolation and functional expression of human COQ2, a gene encoding a polyprenyl transferase involved in the synthesis of CoQ.; citation_author=M Forsgren; citation_author=A Attersand; citation_author=S Lake; citation_author=J Grunler; citation_author=E Swiezewska; citation_journal_title=Biochem J; citation_volume=382; citation_number=92; citation_pages=519-526; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The AtPPT1 gene encoding 4-hydroxybenzoate polyprenyl diphosphate transferase in ubiquinone biosynthesis is required for embryo development in Arabidopsis thaliana.; citation_author=K Okada; citation_author=K Ohara; citation_author=K Yazaki; citation_author=K Nozaki; citation_author=N Uchida; citation_journal_title=Plant Mol Biol; citation_volume=55; citation_number=93; citation_pages=567-577; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Yeast genes that enhance the toxicity of a mutant huntingtin fragment or alpha-synuclein.; citation_author=S Willingham; citation_author=TF Outeiro; citation_author=MJ DeVit; citation_author=SL Lindquist; citation_author=PJ Muchowski; citation_journal_title=Science; citation_volume=302; citation_number=94; citation_pages=1769-1772; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The catalytic domain of the neurofibromatosis type 1 gene product stimulates ras GTPase and complements ira mutants of S. cerevisiae.; citation_author=GF Xu; citation_author=B Lin; citation_author=K Tanaka; citation_author=D Dunn; citation_author=D Wood; citation_journal_title=Cell; citation_volume=63; citation_number=95; citation_pages=835-841; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Hepatitis C virus core protein binds to a DEAD box RNA helicase.; citation_author=N Mamiya; citation_author=HJ Worman; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=96; citation_pages=15751-15756; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Belle is a Drosophila DEAD-box protein required for viability and in the germ line.; citation_author=O Johnstone; citation_author=R Deuring; citation_author=R Bock; citation_author=P Linder; citation_author=MT Fuller; citation_journal_title=Dev Biol; citation_volume=277; citation_number=97; citation_pages=92-101; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detection of Arabidopsis thaliana AtRAD1 cDNA variants and assessment of function by expression in a yeast rad1 mutant.; citation_author=EJ Vonarx; citation_author=NG Howlett; citation_author=RH Schiestl; citation_author=BA Kunz; citation_journal_title=Gene; citation_volume=296; citation_number=98; citation_pages=1-9; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional and genomic approaches reveal an ancient CHEK2 allele associated with breast cancer in the Ashkenazi Jewish population.; citation_author=A Shaag; citation_author=T Walsh; citation_author=P Renbaum; citation_author=T Kirchhoff; citation_author=K Nafa; citation_journal_title=Hum Mol Genet; citation_volume=14; citation_number=99; citation_pages=555-563; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Isolation of a tobacco cDNA encoding Sar1 GTPase and analysis of its dominant mutations in vesicular traffic using a yeast complementation system.; citation_author=M Takeuchi; citation_author=M Tada; citation_author=C Saito; citation_author=H Yashiroda; citation_author=A Nakano; citation_journal_title=Plant Cell Physiol; citation_volume=39; citation_number=100; citation_pages=590-599; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A homologue of Saccharomyces cerevisiae Dpm1p is not sufficient for synthesis of dolichol-phosphate-mannose in mammalian cells.; citation_author=S Tomita; citation_author=N Inoue; citation_author=Y Maeda; citation_author=K Ohishi; citation_author=J Takeda; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=101; citation_pages=9249-9254; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Overexpression of human UDP-glucose pyrophosphorylase rescues galactose-1-phosphate uridyltransferase-deficient yeast.; citation_author=K Lai; citation_author=LJ Elsas; citation_journal_title=Biochem Biophys Res Commun; citation_volume=271; citation_number=102; citation_pages=392-400; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification and analysis of the Arabidopsis thaliana BSH gene, a member of the SNF5 gene family.; citation_author=J Brzeski; citation_author=W Podstolski; citation_author=K Olczak; citation_author=A Jerzmanowski; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=103; citation_pages=2393-2399; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A novel family of cys-rich membrane proteins mediates cadmium resistance in Arabidopsis.; citation_author=WY Song; citation_author=E Martinoia; citation_author=J Lee; citation_author=D Kim; citation_author=DY Kim; citation_journal_title=Plant Physiol; citation_volume=135; citation_number=104; citation_pages=1027-1039; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An ABC-transporter of Arabidopsis thaliana has both glutathione-conjugate and chlorophyll catabolite transport activity.; citation_author=R Tommasini; citation_author=E Vogt; citation_author=M Fromenteau; citation_author=S Hortensteiner; citation_author=P Matile; citation_journal_title=Plant J; citation_volume=13; citation_number=105; citation_pages=773-780; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=ECA1 complements yeast mutants defective in Ca2+ pumps and encodes an endoplasmic reticulum-type Ca2+-ATPase in Arabidopsis thaliana.; citation_author=F Liang; citation_author=KW Cunningham; citation_author=JF Harper; citation_author=H Sze; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=106; citation_pages=8579-8584; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis thaliana cDNA isolated by functional complementation shows homology to serine/threonine protein kinases.; citation_author=L Covic; citation_author=RR Lew; citation_journal_title=Biochim Biophys Acta; citation_volume=1305; citation_number=107; citation_pages=125-129; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=A gain of superoxide dismutase (SOD) activity obtained with CCS, the copper metallochaperone for SOD1.; citation_author=PJ Schmidt; citation_author=M Ramos-Gomez; citation_author=VC Culotta; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=108; citation_pages=36952-36956; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Functional homology of mammalian and yeast RAS genes.; citation_author=T Kataoka; citation_author=S Powers; citation_author=S Cameron; citation_author=O Fasano; citation_author=M Goldfarb; citation_journal_title=Cell; citation_volume=40; citation_number=109; citation_pages=19-26; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Expression pattern of a nuclear encoded mitochondrial arginine-ornithine translocator gene from Arabidopsis.; citation_author=E Catoni; citation_author=M Desimone; citation_author=M Hilpert; citation_author=D Wipf; citation_author=R Kunze; citation_journal_title=BMC Plant Biol; citation_volume=3; citation_number=110; citation_pages=1; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Nucleotide sequence of Arabidopsis thaliana arginase expressed in yeast.; citation_author=PM Krumpelman; citation_author=SK Freyermuth; citation_author=JF Cannon; citation_author=GR Fink; citation_author=JC Polacco; citation_journal_title=Plant Physiol; citation_volume=107; citation_number=111; citation_pages=1479-1480; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Functional complementation analysis of yeast bc1 mutants. A study of the mitochondrial import of heterologous and hybrid proteins.; citation_author=S van Wilpe; citation_author=H Boumans; citation_author=G Lobo-Hajdu; citation_author=LA Grivell; citation_author=JA Berden; citation_journal_title=Eur J Biochem; citation_volume=264; citation_number=112; citation_pages=825-832; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cellular toxicity of polyglutamine expansion proteins: mechanism of transcription factor deactivation Functional complementation analysis of yeast bc1 mutants. A study of the mitochondrial import of heterologous and hybrid proteins.; citation_author=G Schaffar; citation_author=P Breuer; citation_author=R Boteva; citation_author=C Behrends; citation_author=N Tzvetkov; citation_journal_title=Mol Cell; citation_volume=15; citation_number=113; citation_pages=95-105; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The lamin B receptor of Drosophila melanogaster.; citation_author=N Wagner; citation_author=D Weber; citation_author=S Seitz; citation_author=G Krohne; citation_journal_title=J Cell Sci; citation_volume=117; citation_number=114; citation_pages=2015-2028; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human and Saccharomyces cerevisiae dolichol phosphate mannose synthases represent two classes of the enzyme, but both function in Schizosaccharomyces pombe.; citation_author=PA Colussi; citation_author=CH Taron; citation_author=JC Mack; citation_author=P Orlean; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=115; citation_pages=7873-7878; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists"/>
    <meta name="twitter:description" content="Many biological databases that provide comparative genomics information and tools are now available on the internet. While certainly quite useful, to our knowledge none of the existing databases combine results from multiple comparative genomics methods with manually curated information from the literature. Here we describe the Princeton Protein Orthology Database (P-POD, http://ortholog.princeton.edu), a user-friendly database system that allows users to find and visualize the phylogenetic relationships among predicted orthologs (based on the OrthoMCL method) to a query gene from any of eight eukaryotic organisms, and to see the orthologs in a wider evolutionary context (based on the Jaccard clustering method). In addition to the phylogenetic information, the database contains experimental results manually collected from the literature that can be compared to the computational analyses, as well as links to relevant human disease and gene information via the OMIM, model organism, and sequence databases. Our aim is for the P-POD resource to be extremely useful to typical experimental biologists wanting to learn more about the evolutionary context of their favorite genes. P-POD is based on the commonly used Generic Model Organism Database (GMOD) schema and can be downloaded in its entirety for installation on one's own system. Thus, bioinformaticians and software developers may also find P-POD useful because they can use the P-POD database infrastructure when developing their own comparative genomics resources and database tools."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000766.g005"/>

  <meta property="og:title" content="The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1590'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6887'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5989&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000766">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000766" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000766&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sven Heinicke
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sven Heinicke, Michael S. Livstone, Charles Lu, Rose Oughtred</p>

              
              

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael S. Livstone
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sven Heinicke, Michael S. Livstone, Charles Lu, Rose Oughtred</p>

              
              

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Charles Lu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sven Heinicke, Michael S. Livstone, Charles Lu, Rose Oughtred</p>

              
              

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rose Oughtred
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Sven Heinicke, Michael S. Livstone, Charles Lu, Rose Oughtred</p>

              
              

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fan Kang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Samuel V. Angiuoli, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  The Institute for Genomic Research, Rockville, Maryland, United States of America, 
                  Center for Bioinformatics and Computational Biology, University of Maryland, College Park, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Owen White, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Institute for Genomic Research, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Botstein, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kara Dolinski
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kara@genomics.princeton.edu">kara@genomics.princeton.edu</a></p>

                <p>Affiliation:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 22, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000766</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000766-t001" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-g001" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-t002" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-g002" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-t003" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-t004" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-t005" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-g003" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-g004" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-g005" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000766-t006" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000766">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000766" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Many biological databases that provide comparative genomics information and tools are now available on the internet. While certainly quite useful, to our knowledge none of the existing databases combine results from multiple comparative genomics methods with manually curated information from the literature. Here we describe the Princeton Protein Orthology Database (P-POD, <a href="http://ortholog.princeton.edu">http://ortholog.princeton.edu</a>), a user-friendly database system that allows users to find and visualize the phylogenetic relationships among predicted orthologs (based on the OrthoMCL method) to a query gene from any of eight eukaryotic organisms, and to see the orthologs in a wider evolutionary context (based on the Jaccard clustering method). In addition to the phylogenetic information, the database contains experimental results manually collected from the literature that can be compared to the computational analyses, as well as links to relevant human disease and gene information via the OMIM, model organism, and sequence databases. Our aim is for the P-POD resource to be extremely useful to typical experimental biologists wanting to learn more about the evolutionary context of their favorite genes. P-POD is based on the commonly used Generic Model Organism Database (GMOD) schema and can be downloaded in its entirety for installation on one's own system. Thus, bioinformaticians and software developers may also find P-POD useful because they can use the P-POD database infrastructure when developing their own comparative genomics resources and database tools.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Heinicke S, Livstone MS, Lu C, Oughtred R, Kang F, et al.  (2007) The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists. PLoS ONE 2(8):
          e766.
            doi:10.1371/journal.pone.0000766</p><p><strong>Academic Editor: </strong>Berend Snel, Utrecht University, Netherlands</p><p><strong>Received:</strong> May 18, 2007; <strong>Accepted:</strong> July 18, 2007; <strong>Published:</strong> August 22, 2007</p><p><strong>Copyright:</strong> © 2007 Heinicke et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded by NIH grant 5R01HG003471 awarded to DB (PI) and KD (co-investigator), by NIH grant P50 GM071508 awarded to DB, and by NIH contract NO1-AI-40038 awarded to OW.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>With the great explosion of biological data in the last decade, biological databases have become an essential part of today's research. The earliest online databases were the sequence repositories, such as Genbank <a href="#pone.0000766-Benson1">[1]</a> and EMBL <a href="#pone.0000766-Kulikova1">[2]</a>, that provided the non-expert public access to the sequence data for genes, chromosomes, and eventually entire genomes, along with highly effective query and comparison tools. Soon after, several model organism databases that store and display the annotated genome sequences of well-studied organisms were developed. These databases now serve as an essential basic information source for all kinds of biological researchers.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>For working biologists, some of the most important information concerns the phylogenetic relationships among proteins, which is not necessarily straightforward to recover from the basic sequence databases. Regardless of which organism one works with, much of the functional annotation of gene and protein functions is transferred, based on sequence similarity, from other organisms where more experimental information is available (for example, see the Gene Ontology annotations at <a href="http://www.geneontology.org/GO.current.annotations.shtml">http://www.geneontology.org/GO.current.a​nnotations.shtml</a>). It is for this reason that sequence similarity searching has become one of the most popular database tools in current use, perhaps second only to searching the published literature. To make good use of sequence similarity information, it would be very useful to have a simple, user-friendly way to visualize relationships in their phylogenetic context, particularly the relationships among the proteins in the model organisms from which most of the functional annotations are derived. It is of particular value to be able to know which proteins are (or might be) orthologous [i.e. similar to each other in sequence because they originated from a common ancestor, having been separated in evolutionary time only by speciation event(s)]. It is also useful to see these orthologous relationships in the context of the larger paralogous gene families ultimately caused by gene duplications during the course of evolution.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this paper, we describe P-POD, which provides the user an easy way to find and visualize the orthologs to a query sequence in the eukaryotes of greatest interest to working biologists (<em>i.e.</em> the experimental model organisms and the human) in their evolutionary context, and to link these relationships with the relevant literature. Several databases that specialize in comparative genomics have recently come online. Each of these databases, including P-POD, has both useful features and problems specific to the methods or species chosen in the analysis (<a href="#pone-0000766-t001">Table 1</a>, reviewed in <a href="#pone.0000766-Alexeyenko1">[3]</a>); none is perfect, but each fulfills the needs of particular database users.</p>
<div class="figure" id="pone-0000766-t001"><div class="img"><a name="pone-0000766-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Comparative genomics web resources.</span></strong></p><span>doi:10.1371/journal.pone.0000766.t001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>P-POD is meant to complement these existing databases by providing a comparative genomics analysis system readily accessible to and readable by experimentalists, containing not just computational comparative analyses of the most common experimental organisms but also literature curation and links to other databases of interest. For example, while the OrthoMCL database contains sequences from over 55 prokaryotic and eukaryotic genomes, we chose to include protein sequences from eight eukaryotic organisms for their medical value or their status as widely-studied model organisms. There are certainly users who would need the more comprehensive species set from OrthoMCL. While P-POD uses the underlying OrthoMCL algorithm, it is meant to complement the OrthoMCL online database by serving another set of users, primarily experimental biologists who wish to query with their gene of interest from a well studied model organism to quickly get the evolutionary context of that gene along with other relevant information about that gene without sorting through a very large list of other sequences.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We designed our comparative genomics analysis system so that different components could be added to and removed from the pipeline in a modular fashion; the initial version of the pipeline described here generates related protein families using two different methods to provide complementary views of phylogenetic relationships. We used OrthoMCL (<a href="#pone.0000766-Li1">[4]</a>) to find the orthologs and a version of Jaccard Clustering [modified to find homologs across multiple genomes (S. Angiuoli and O. White, in preparation)] to provide a larger protein family context. The phylogenetic relationships among family members from each method are determined using CLUSTAL W <a href="#pone.0000766-Thompson1">[5]</a> and PHYLIP and visualized as arbitrarily rooted trees. In addition, we provide relevant gene and disease information from the Online Mendelian Inheritance in Man (OMIM) <a href="#pone.0000766-Lenffer1">[6]</a> database and also provide information culled from the literature that can be used to indicate when functional conservation has been shown experimentally between predicted orthologs. All the data within the database are freely available through the web and by downloading the entire software and database system via the following URL: <a href="http://ortholog.princeton.edu/">http://ortholog.princeton.edu/</a></p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Historically, genomic databases have been developed in isolation, with idiosyncratic database schemata and software. Much duplication of effort can be avoided by developing generic modular databases and software that save, especially in the long run, both time and money spent on development, maintenance, and user training. In constructing P-POD we made use of the database schema, installation and loading tools, and various software components from the Generic Model Organism Database (GMOD) project (<a href="http://www.gmod.org">www.gmod.org</a>), The goal of GMOD is to develop an open and generic genomic database environment, including database schemata and required software tools.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>The P-POD Pipeline</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In the interests of both simplicity and flexibility, the P-POD pipeline employs a modular architecture. The pipeline takes FASTA-formatted protein sequences as input, performs comparative genomic analyses, and stores the results in a database. In addition, we have created web tools that allow searching and browsing of the results in a user-friendly manner. We built the initial pipeline to identify putative orthologous proteins using OrthoMCL <a href="#pone.0000766-Li1">[4]</a>. We chose OrthoMCL over other algorithms mainly because it can be run on multiple species at once and is one of the better-performing algorithms in terms of sensitivity and specificity <a href="#pone.0000766-Chen1">[7]</a> <a href="#pone.0000766-Alexeyenko1">[3]</a>. We generated larger families of related sequences using Jaccard clustering modified to find homologs across multiple genomes; see the <a href="#s4">Materials and Methods</a> section for algorithm details. It is important to note that we built the P-POD system so that we can easily add or remove results from different analysis methods. We acknowledge that the first choice is not always the best choice, and as algorithms improve and/or as users request other methods, we plan to modify and expand the system as appropriate. P-POD generates phylogenetic trees from both analyses using CLUSTAL W <a href="#pone.0000766-Thompson1">[5]</a> and PHYLIP; the trees are graphically displayed on the web. The overall pipeline is illustrated in <a href="#pone-0000766-g001">Figure 1</a>. The sources and versions of the pipeline components are listed in <a href="#pone-0000766-t002">Table 2</a>. The data are stored in a Generic Model Organism Database (GMOD) database schema using the freely available PostgreSQL software to make the entire system accessible to as many users as possible, not only through the web but also via download of the entire system.</p>
<div class="figure" id="pone-0000766-g001"><div class="img"><a name="pone-0000766-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Steps in the analysis pipeline.</span></strong></p><span>doi:10.1371/journal.pone.0000766.g001</span></div><div class="figure" id="pone-0000766-t002"><div class="img"><a name="pone-0000766-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Components of the analysis pipeline.</span></strong></p><span>doi:10.1371/journal.pone.0000766.t002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The P-POD database contains protein sequences from eight eukaryotic organisms with fully sequenced genomes chosen for either their medical value or their status as widely-studied model organisms. They include a yeast (<em>Saccharomyces cerevisiae</em>), a nematode worm (<em>Caenorhabditis elegans</em>), a fruit fly (<em>Drosophila melanogaster</em>), a flowering plant (<em>Arabidopsis thaliana</em>), a fish (<em>Danio rerio</em>), a mouse (<em>Mus musculus</em>), and human (<em>Homo sapiens</em>). These are the leading experimental organisms for modern biologists, and among them span much of the evolutionary tree of the eukaryotes. Also included is the malaria parasite <em>Plasmodium falciparum</em>, an organism that, although it is a eukaryote, has a relatively exotic parasitic lifestyle. Sources for each protein set are listed in <a href="#pone-0000766-t003">Table 3</a>. Also stored in the system are results from each step of the pipeline, gene and disease information from OMIM, and curated information from the literature describing experimental tests of functional conservation (see <a href="#pone-0000766-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000766-g002"><div class="img"><a name="pone-0000766-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Screenshots of the P-POD web interface.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(A) A portion of the results page for the <em>DPM1</em> OrthoMCL family is shown superimposed on the search form. Results from OrthoMCL are provided, and a link to the larger Jaccard family (B) is also available. Disease information from OMIM is displayed, as well as any relevant disease or cross-complementation literature.</p>
<span>doi:10.1371/journal.pone.0000766.g002</span></div><div class="figure" id="pone-0000766-t003"><div class="img"><a name="pone-0000766-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Sources and numbers of sequences analyzed.</span></strong></p><span>doi:10.1371/journal.pone.0000766.t003</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The pipeline generated a total of 25,271 OrthoMCL families and 15,050 Jaccard Clustering families that contain a total of 165,970 proteins (154,736 and 152,799 for each method, respectively) from eight different organisms. There are 984 OrthoMCL families that contain at least one protein from each of the species, with 112 of them containing exactly one protein from each. We used the GO Term Mapper tool available at SGD to determine the distribution of GO annotations for the 112 yeast proteins in these families; we chose the yeast proteins because complete GO annotation is available for the entire yeast genome <a href="#pone.0000766-Dwight1">[8]</a>. Not surprisingly, these proteins are involved in core biological processes that are common across eukaryotes, including translation, transport, cell cycle regulation, and cytoskeleton organization. These genes are also well characterized; only four of the 112 genes were annotated to “biological process unknown.” We also used the GO Term Finder <a href="#pone.0000766-Boyle1">[9]</a> implementation at Princeton (<a href="http://go.princeton.edu/">http://go.princeton.edu/</a>) to look for enrichment of GO terms among the 112 genes. Again unsurprisingly, the most significant shared term is “ribosome biogenesis and assembly” (corrected P-value = 5.85e-18) along with other terms related to translation and basic metabolic processes, all processes common among the eukaryotes.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The complete species distribution of each family is available via the web (<a href="http://ortholog.princeton.edu/organismdist.html">http://ortholog.princeton.edu/organismdi​st.html</a>), and the number of proteins found in families and orphan proteins (those not found in an OrthoMCL or Jaccard family) from all the species is found in <a href="#pone-0000766-t004">Table 4</a>.</p>
<div class="figure" id="pone-0000766-t004"><div class="img"><a name="pone-0000766-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Number of proteins in each organism found in OrthoMCL or Jaccard families.</span></strong></p><span>doi:10.1371/journal.pone.0000766.t004</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>The percentage of orphans is generally strikingly low, with the percent orphaned in a given species 13% or lower, with two exceptions, yeast (32%) and <em>Plasmodium</em> (33%). These numbers confirm the high conservation of proteins across eukaryotes, with the notable exception the <em>Plasmodium</em> outlier. The high percentage of yeast orphans is due to the fact that we did the analysis with the complete protein set, including over 800 ORFs flagged as “Dubious” by SGD; these are not likely to actually encode proteins, and when they are excluded the percentage of orphans in yeast drops to about 20%.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>P-POD includes 1,895 human proteins that are associated with human diseases (based on protein-OMIM disease files downloaded from ENSEMBL), 1,852 of which were found in either an OrthoMCL or Jaccard family; in each of these cases, links to the relevant OMIM records are provided online.</p>


<h4>Manually Curated Information</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>P-POD also includes curated literature that contains information relevant to the yeast proteins in the database. The source of the literature is the <em>Saccharomyces</em> Genome Database (SGD). SGD provides a Literature Guide tool that categorizes yeast literature into different topics, two of which, “Cross-species expression” and “Disease-gene related,” are particularly relevant to the data in P-POD; we believe that this set of papers, which is continually updated and curated, contains most, if not quite all, of the experimental data testing functional conservation between yeast and other organisms. All papers associated with these topics were downloaded from the SGD FTP site and loaded into the database (see <a href="#s4">Materials and Methods</a>). They are then displayed on the web interface, with links to PubMed, so that users can compare experimentally determined functional conservation and computationally predicted orthology. This set of papers does not, of course, address proteins without a yeast ortholog. A way of dealing with this limitation is under study; a likely development will be the inclusion of papers from the literatures of other model organisms. For disease-related genes, we provide OMIM links that at least partially fill this gap for the human.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In addition, we manually curated the “Cross-species expression” papers to indicate explicitly when functional conservation was experimentally determined. These cross-species expression experiments test whether expressing a putative ortholog from one organism will restore wildtype function to the corresponding inactivated gene in another organism (almost always <em>S. cerevisiae</em>). <a href="#pone-0000766-t005">Table 5</a> summarizes this curated information for only the yeast proteins in the disease-related families to illustrate how this information can be compared to computational results, but P-POD contains experimental results for all yeast proteins for which curated information is available. The orthologs predicted by OrthoMCL often exhibit conserved function. Of the 643 curated complementation experiments between yeast genes and their putative orthologous sequences from other organisms, 395 showed functional conservation and were also identified as orthologs by OrthoMCL; 50 did not complement and were also not predicted to be orthologs by OrthoMCL. Thus, in most cases (445/643), the computational determination of orthology was consistent with experimental results of functional conservation. However, in 153 experiments, complementation was observed, but the proteins were not in the same OrthoMCL family, and in 45 experiments, complementation did not occur, but OrthoMCL predicted an orthologous relationship between the two proteins. These experimental results can be used as a rudimentary assessment of the computational predictions but it must be noted that the definition of orthology does not require functional conservation <a href="#pone.0000766-Koonin1">[10]</a>, and there are actual cases (<em>e.g.</em> actin) where <em>in vivo</em> complementation fails for biological reasons, even for true orthologs that can function <em>in vitro</em> <a href="#pone.0000766-Kron1">[11]</a>.</p>
<div class="figure" id="pone-0000766-t005"><div class="img"><a name="pone-0000766-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Functional conservation vs. ortholog prediction: comparing experimental results with the OrthoMCL ortholog predictions for disease-related families.</span></strong></p><span>doi:10.1371/journal.pone.0000766.t005</span></div>

<h4>The P-POD User Interface: Orthologs, Families and Diseases</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We designed a simple web interface that allows users to search and browse the data in several ways (<a href="#pone-0000766-g002">Figure 2</a>). Results can be queried by various peptide identifiers or gene names, choosing from any of eight model organisms for the query protein and a particular analysis method, or they can be searched or browsed by Online Mendelian Inheritance in Man (OMIM) ID.</p>


<h4>Searches generate result pages that contain:</h4>


<ul class="bulletlist">

<li>a hyperlinked phylogenetic tree of predicted orthologs generated by OrthoMCL or of more distantly-related proteins generated by Jaccard clustering,</li>

<li>a list of diseases and genes associated with the human ortholog(s) as documented in OMIM,</li>

<li>a manually curated list of papers with cross-complementation experiments involving the yeast ortholog(s), and</li>

<li>a downloadable ClustalW alignment of family members.</li>

</ul>

<h4>Using P-POD to Compare Methods: Jaccard and OrthoMCL</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To illustrate the usefulness of being able to store multiple analyses in a single database, we further compared the results between the OrthoMCL and Jaccard Clustering methods. A query for yeast <em>TUB1</em> using only OrthoMCL reveals the alpha tubulins from yeast and other organisms (<a href="#pone-0000766-g003">Figure 3</a>), but not the important paralogous relationships to the beta and gamma tubulins <a href="#pone.0000766-Keeling1">[12]</a> <a href="#pone.0000766-Dutcher1">[13]</a>, which are observed in the <em>TUB1</em> Jaccard cluster (not shown). These three main classes of tubulins are related to the bacterial FtsZ protein and diverged prior to the divergence of the eukaryotes <a href="#pone.0000766-Keeling1">[12]</a>. Many such examples are found, especially among the ancient gene families that go back to the common ancestors of all eukaryotes. The Jaccard clustering provides this larger evolutionary context.</p>
<div class="figure" id="pone-0000766-g003"><div class="img"><a name="pone-0000766-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>OrthoMCL family of the alpha tubulins.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>This OrthoMCL family contains only the alpha tubulins, while the tubulin family generated by the Jaccard family (too large to be shown here) contains the alpha, beta, and gamma tubulins.</p>
<span>doi:10.1371/journal.pone.0000766.g003</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>While OrthoMCL identifies predicted orthologs, the Jaccard clustering algorithm should build broader families of more distantly related sequences. Accordingly, one might initially expect that each OrthoMCL family would be a subset of a corresponding Jaccard cluster. Of course, because each algorithm defines homologs quite differently, in practice it would be reasonable to expect a certain degree of disagreement between the OrthoMCL and Jaccard clustering results. Of the 25,271 OrthoMCL families, 17,340 (69%) are subsets of Jaccard clusters. A certain amount of the “loss” of family members is due to stochastic effects; 72% of the 22,216 OrthoMCL families with ten or fewer members remain intact as subsets of Jaccard clusters, compared to only 49% of the 3,055 larger families. Fully 91% of the peptides assigned to OrthoMCL families also lie in Jaccard clusters. 82% of the OrthoMCL families have 80% or more of their peptides in a single Jaccard cluster; 93% have 50% or more.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Another possible source of inconsistency between the OrthoMCL and Jaccard results is that these analyses were run with different parameter settings. In particular, an alignment constraint was used for the Jaccard clustering alone because the default and recommended settings for OrthoMCL do not include an alignment constraint (see <a href="http://orthomcl.cbil.upenn.edu/ORTHOMCL/">http://orthomcl.cbil.upenn.edu/ORTHOMCL/</a>). The Jaccard clustering software was configured to ignore BLAST hits that did not align over 50% of the length of both peptides. For example, yeast <em>MET3</em> and <em>MET14</em> respectively encode ATP sulfurylase and adenylylsulfate kinase, which catalyze the first two steps of a sulfate assimilation pathway. <em>A. thaliana</em> retains this distinction, but <em>C. elegans</em>, <em>D. melanogaster</em>, <em>D. rerio</em>, human, and mouse have bifunctional proteins containing both activities. The OrthoMCL family contains all of these peptides (<a href="#pone-0000766-g004">Figure 4B</a>), but <em>MET14</em> and the four <em>Arabidopsis</em> adenylylsulfate kinases form their own Jaccard cluster (<a href="#pone-0000766-g004">Figure 4A</a>). At 202 amino acids, Met14p is less than half the length of the other OrthoMCL family members and therefore fails to satisfy the 50% alignment constraint used in the Jaccard clustering algorithm.</p>
<div class="figure" id="pone-0000766-g004"><div class="img"><a name="pone-0000766-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The <em>MET3/MET14</em> families.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>(A) <em>MET14</em> Jaccard family, and (B) <em>MET3/MET14</em> OrthoMCL family.</p>
<span>doi:10.1371/journal.pone.0000766.g004</span></div><a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Again, having both sets of results within the same database made comparison of the two methods and detection of possible issues relatively straightforward. We expect that this will be a useful feature for database developers and/or bioinformaticians who may download the entire P-POD system for local installation to use as a development base for their algorithms of choice.</p>


<h4>Other Uses for P-POD</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We provide several examples of how P-POD might be used by experimental biologists, and not necessarily those expert in phylogenomics. In addition, we illustrate how providing results from different analysis methods can help to identify issues characteristic of the different methods.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The P-POD system can be used in a simple way to learn something global about the genes and/or proteins of an organism. As an illustration, we studied the conservation of essential genes, <em>i.e</em>. genes that are required for viability, across yeast and mammals. Among the 929 OrthoMCL families with unambiguous orthologs from yeast, mouse, and human (<em>i.e.</em> exactly one member from each of these species), phenotype data were available for the yeast and mouse genes in 107 cases. In 28 cases, the yeast gene was essential, and in 24 of these families (86%), the mouse gene was also essential. The entire analysis can be found at <a href="http://ortholog.princeton.edu/essential_analysis.html">http://ortholog.princeton.edu/essential_​analysis.html</a>.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>P-POD can be used to estimate whether essential yeast genes are more likely to be conserved and/or related to a human disease gene. There are 1100 essential and 4670 non-essential yeast genes, respectively. 853 essential yeast genes (77.5%) are found in an OrthoMCL family, while 247 (22.5%) are not. Of the non-essential genes, 2968 (63.6%) are found in families, while 1702 (36.4%) are not. These data suggest that essential genes are more conserved than non-essential genes (χ<sup>2</sup> = 78, p = 1.1e-18). When examining essentiality among the 954 yeast genes found in disease-related families, 191 of them are essential (20% of the disease-related genes, 17% of all essential genes), while 691 of them are non-essential (72% of disease-related genes, 14.8% of all non-essential genes); phenotype data are not available for the remaining 72 yeast genes. Thus, there does not appear to be enrichment of essential genes among the disease-related yeast genes (χ<sup>2</sup> = 4.5, p = 0.03). The lack of enrichment of essential genes among disease-related genes is initially surprising; however, this result can be explained if genes required for viability in yeast are also required for viability of human cells, thus making it impossible for the mammal to fully develop into even a diseased organism.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>P-POD simplifies the study of the relationships among families of proteins with related functions. One example is the DNA-dependent RNA polymerase family (<a href="#pone-0000766-g005">Figure 5A, B, C</a>). Transcription of genes in eukaryotes is generally performed by three RNA polymerases (I, II, and III), each of which is composed of more than 10 subunits <a href="#pone.0000766-Archambault1">[14]</a>, Searching on a selection of individual yeast RNA polymerase subunits (<em>RPO21, RPO31, RPA190, RPB2, RPB4, RPB5, RPA135</em>,&nbsp;and <em>RET1</em>) resulted in separate phylogenetic tree displays for each protein, demonstrating that they had been effectively resolved into distinct ortholog clusters. Within each cluster, there were mainly one-to-one orthologous relationships between the proteins from each species, except for <em>RPA135</em>, and <em>RET1,</em> which include orthologs from each species examined except for <em>D. rerio</em> (<a href="#pone-0000766-g005">Figure 5A, B</a>).</p>
<div class="figure" id="pone-0000766-g005"><div class="img"><a name="pone-0000766-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>OrthoMCL and Jaccard clustering results for the second largest RNA polymerase subunit families of <em>S. cerevisiae.</em></span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The second largest subunits of RNA polymerase I, II, and III in yeast are named <em>RPA135, RPB2</em>, and <em>RET1</em>, respectively. (A) Phylogenetic tree display of OrthoMCL results showing individual yeast subunit <em>RPA135</em> and its predicted orthologs resolved into a distinct family. OrthoMCL results showing yeast RNA polymerase subunits <em>RET1</em> (B) and <em>RPB2</em> (C) resolved into separate families of orthologs. (D) Jaccard clustering results showing a “super family” of related RNA polymerase subfamilies. Arrows from each OrthoMCL family on the left point to the separate subfamilies in the Jaccard results. I to IV on the right of each tree indicates RNA polymerase subfamily. The second largest subunits for a fourth RNA polymerase, Pol IV, unique to plants were resolved into their own distinct two-member family by the OrthoMCL program (not shown), and were appropriately clustered with this superfamily by the Jaccard clustering method. (Adapted from <a href="#pone-0000766-g002">figure 2</a> of <a href="#pone.0000766-Herr1">[15]</a>)</p>
<span>doi:10.1371/journal.pone.0000766.g005</span></div><a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>For some subunits, in particular <em>RPO21, RPA190</em>, and <em>RPA135</em>, there appear to be more than one mouse or human paralog; however, upon further investigation, it was determined that the separate peptides were encoded by a single mouse or human gene (<a href="#pone-0000766-g005">Figure 5A</a>). Therefore, for the most part, each protein from each species appeared to be orthologous to the others, as would be expected for proteins functioning in a core biological process <a href="#pone.0000766-Archambault1">[14]</a>.</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>Interestingly, experimental evidence shows that although all eukaryotes have RNA polymerases I, II, and III, plants are unique in that they have subunits for a fourth polymerase, Pol IV. The closely related genes, AT3G18090.1 (NRPD2B) and AT3G23780.1 (NRPD2A), have been found to encode the second largest subunit of plant Pol IV, with most of the NRPD2 transcripts coming from NRPD2A. These atypical second largest subunits occurring only in plants are most similar in sequence to the RNA polymerase II second largest subunits in other eukaryotes such as yeast <em>RPB2</em> <a href="#pone.0000766-Herr1">[15]</a>, <a href="#pone.0000766-Onodera1">[16]</a>. Despite this sequence similarity, they were effectively resolved away from the OrthoMCL-generated ortholog cluster containing yeast <em>RBP2</em> into their own distinct two-member family. The Jaccard clustering method, on the other hand, correctly grouped these unique Pol IV plant subunits with the other second largest RNA polymerase subunit families, as shown in <a href="#pone-0000766-g005">Figure 5D</a>.</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>As another illustration, we examined thirty yeast ER proteins involved in asparagine-linked glycosylation, a pathway which is well-conserved between yeast and humans in its early steps and diverges soon after glycosylated proteins enter the Golgi (<a href="#pone-0000766-t006">Table 6</a>). Of these, 27 are known from the literature to have human homologs. This analysis shows that 26 lie in ortholog families, with the majority having orthologs in <em>Homo sapiens</em> (26), <em>D. melanogaster</em> (24), <em>A. thaliana</em> (24), <em>M. musculus</em> (23), <em>C. elegans</em> (23), and <em>D. rerio</em> (21). The four proteins that do not lie in ortholog families are subunits of the yeast oligosaccharyltransferase complex. Deleterious mutations in ten of the human homologs cause congenital disorders of glycosylation. Interestingly, only nine of the thirty yeast ER proteins have orthologs in <em>P. falciparum</em>. N-linked glycosylation has been detected only at very low levels in <em>P. falciparum</em> <a href="#pone.0000766-Gowda1">[17]</a>, and ensuring appropriate glycosylation in heterologously-expressed <em>P. falciparum</em> proteins has been a technical challenge in the development of malaria vaccines <a href="#pone.0000766-Kedees1">[18]</a>, <a href="#pone.0000766-Kocken1">[19]</a>.</p>
<div class="figure" id="pone-0000766-t006"><div class="img"><a name="pone-0000766-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000766" data-uri="info:doi/10.1371/journal.pone.0000766.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000766.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000766.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000766.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000766.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Conservation of yeast proteins involved in N-linked glycosylation.</span></strong></p><span>doi:10.1371/journal.pone.0000766.t006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The database system (P-POD) we constructed shows users predicted orthologs of query proteins alone (using OrthoMCL) and in their broader evolutionary context (using Jaccard clustering). It consists of a comparative genomics analysis pipeline whose results are stored in a generic, modular database schema (GMOD/chado) using a freely available database system (PostgreSQL). P-POD is meant not to replace but rather to complement the currently available comparative genomics databases. To our knowledge, no other comparative genomics database provides experimental evidence of conservation curated from the primary literature.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We envision at least three sets of users of our database system. First, molecular biologists can query the database over the web to browse orthology data, both computational and experimental, for their favorite proteins. Another set of users consists of model organism database developers, who will quickly be able to provide comparative genomics tools with their species of interest by implementing our system. Finally, we expect that computational biologists who are developing novel comparative genomics algorithms will find the curated information and computational data from other methods extremely useful in assessing their approach. In addition, by using our system, they will save time in implementation and will be able to more readily distribute their algorithms.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It is important to emphasize that while computational methods to identify orthologs are extremely useful, they are by no means perfect. While OrthoMCL does reasonably well in creating putative orthologous groups, like all computational methods, in many cases it fails, either leaving out true orthologs or inappropriately including paralogs <a href="#pone.0000766-Chen1">[7]</a>. If one's main goal is to use such an algorithm solely to identify strict orthologs, then the selection of species is critical, and the inclusion of two mammals along with the distantly related <em>Plasmodium</em> certainly will increase the number of families that contain extraneous paralogs. Our goal, however, is to provide a database that can serve not only computational or evolutionary biologists but also the day-to-day needs of biologists who work on the common model organisms. P-POD provides a way for biologists to query directly for their gene of interest from their species of study, even though in some cases the phylogenetic trees must be manually examined to determine true orthologs because of the occasional inclusion of paralogs. As more refined methods for automatic detection of orthology are developed (for example, <a href="#pone.0000766-Alexeyenko2">[20]</a>, <a href="#pone.0000766-Jothi1">[21]</a>) we plan to incorporate them into the P-POD tool, taking advantage of our modular design scheme.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>We plan to provide regular updates to the data contained within the database. At the time of writing, we are running the analysis pipeline with the latest versions of the genomes. In addition, we will add new features to the web interface and will expand upon the amount of data stored within the database. We will also continue to provide curated literature describing experimental confirmation of orthology. All the data within the database are freely and publicly available through the web and by downloading the entire database system via the URL <a href="http://ortholog.princeton.edu/">http://ortholog.princeton.edu/</a>.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The overall analysis pipeline is illustrated in <a href="#pone-0000766-g001">Figure 1</a>. The sources and versions of the pipeline components are listed in <a href="#pone-0000766-t002">Table 2</a>.</p>

<h4>WU-BLAST</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The same WU-BLAST results were used as input to both OrthoMCL and Jaccard algorithms described below. WU-BLAST (version 2.0MP-WashU) was run with the default BLASTP settings: matrix = BLOSUM62, Expectation Threshold = 10, ctxfactor = 1.0, no filtering.</p>


<h4>OrthoMCL and Jaccard Algorithms</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>OrthoMCL (v. 1.2, 14-March-2005 <a href="#pone.0000766-Li1">[4]</a>) compares the all-against-all BLASTP scores from a set of genomes, first identifying putative orthologs as reciprocal best hits between pairs of genomes, then identifying candidate recent paralogs as proteins within the same species that are more similar to each other than to any sequence in the other species. All orthologs and recent paralogs are then converted into a graph where the nodes represent the proteins and the edges represent their relationships. A normalization step is then used to correct for systematic biases when comparing pairs of genomes. Finally, the ortholog families are resolved by application of the Markov Cluster algorithm (MCL v. 1.005, 05-118). Since this procedure maximally includes in a family only those proteins at least as closely related as between-species reciprocal best hits, the resultant OrthoMCL group can be considered a set of putative orthologs in that every protein in the group is likely orthologous to at least one other group member. Some groups, however, consist solely of&nbsp;proteins from a single species; obviously, such groups only contain recent paralogs, but this information is often of great importance to experimental biologists.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>We used the following OrthoMCL parameters. P-value cutoff: 1e<sup>−5</sup>, percent identity and percent match cutoffs: 0, maximum weight: 100.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>OrthoMCL family size can be adjusted by changing the inflation index (1.5 in this study), but this does not loosen the fundamental restriction that the algorithm begins with a list of putative orthologs and paralogs. To get larger families showing more distant relationships, we wanted to remove this restriction and include proteins that exhibit significant sequence similarity over a large portion of their lengths. We chose to perform Jaccard clustering and to apply a more broadly-defined set of criteria, namely that members of the same family should have significant BLAST scores over at least half of their length. This last point is important to reduce the chance of grouping two sequences together based on the presence of short promiscuous domains.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>In the Jaccard clustering analysis, two proteins are grouped into the same family if they share a significant number of homologs, calculated as follows. First, a list of homologs for each sequence, consisting of those whose relative BLASTP scores are less than 1e<sup>−5</sup> over a total of at least 50% of the length of each, is generated for each protein. Then the Jaccard index for each pair is calculated; this is the ratio of the magnitude of the intersection of their homolog sets vs. the union, or |A∩B| / |A∪B|. Final clusters are generated by linking proteins whose mutual Jaccard index is above a pre-determined cutoff. We evaluated the impact of varying the cutoff over a range of 0.3 to 0.8 for several well-characterized protein families, such as actins, tubulins, RNA polymerases, and several proteins containing RING finger or SH3 domains. We chose a Jaccard index of 0.4 since it most broadly permitted the inclusion of expected members of the families while excluding obvious non-members. For example, at a cutoff of 0.5, the family containing yeast actin (<em>ACT1</em>) inappropriately omitted the human and mouse actin-related proteins <em>ACTR8</em> and <em>Actr8</em>, while a cutoff of 0.3 was clearly too low and yielded many families with hundreds of extraneous members.</p>


<h4>Generation of phylogenetic trees</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>P-POD generates phylogenetic trees of the OrthoMCL and Jaccard families using CLUSTAL W <a href="#pone.0000766-Thompson1">[5]</a> and PHYLIP (Felsenstein, J. 2005. PHYLIP (Phylogeny Inference Package) version 3.6. Distributed by the author. Department of Genome Sciences, University of Washington, Seattle), using ProML with global rearrangements turned on. CLUSTAL W was run with the default settings: matrix = BLOSUM, Gapopen = 10, Gapext = 0.2, Gapdist = 8, Max div. = 40, ENDGAPS, NOPGAPS and NOHGAPS off, PWMATRIX = BLOSUM, PWGAPOPEN = 10, PWGAPEXT = 0.1, Distance = Kimura, TOSSGAPS = ON, Output = PHYLIP.</p>


<h4>Literature</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>During literature curation at SGD for its “Literature Guide” resource, papers may be associated with yeast genes and various topics that describe what the paper addresses. A list of all papers associated with the topics “Cross-species expression” or “Disease-related” was downloaded from the SGD FTP site and loaded into the P-POD database, along with links to the yeast genes as made by the SGD curators. These papers are displayed on the P-POD interface whenever a family that contains the relevant yeast genes is viewed; each paper displayed is hyperlinked to the PubMed database. For the papers associated with the “Cross-species expression” topic, we manually read each paper to extract which gene(s) from which organism(s) were tested, and whether functional complementation was demonstrated. These results are stored in the database and displayed on the P-POD interface.</p>


<h4>Database schema and software</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>P-POD uses the Generic Model Organism Database (GMOD) database package using PostgreSQL software. Information and documentation about the GMOD schema (also known as the “chado” schema) can be found on the GMOD web site (<a href="http://www.gmod.org">www.gmod.org</a>). In addition, Supplemental <a href="#pone-0000766-t001">Table 1</a> (<a href="http://ortholog.princeton.edu/help.html#schema">http://ortholog.princeton.edu/help.html#​schema</a>) provides details about our particular implementation of the GMOD schema, including how data from our analysis (FASTA files, OrthoMCL results, etc.) are mapped to the GMOD database tables.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We acknowledge John Wiggins and Mark Schroeder for excellent technical support and Mike Cherry (SGD), Shuai Weng (SGD), Eurie Hong (SGD), Laurie Kramer (Princeton) and John Matese (Princeton) for valuable discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DB KD SH CL. Performed the experiments: SH CL. Analyzed the data: KD RO SH ML. Contributed reagents/materials/analysis tools: OW SA FK. Wrote the paper: DB KD RO ML.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000766-Benson1" id="pone.0000766-Benson1"></a>Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2007) GenBank. Nucleic Acids Res  35: D21–25.  <ul class="find" data-citedArticleID="979932" data-doi="10.1093/nar/gkl986"><li><a href="http://dx.doi.org/10.1093/nar/gkl986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenBank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenBank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000766-Kulikova1" id="pone.0000766-Kulikova1"></a>Kulikova T, Akhtar R, Aldebert P, Althorpe N, Andersson M, et al.  (2007) EMBL Nucleotide Sequence Database in 2006. Nucleic Acids Res  35: D16–20.  <ul class="find" data-citedArticleID="980028" data-doi="10.1093/nar/gkl913"><li><a href="http://dx.doi.org/10.1093/nar/gkl913" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EMBL+Nucleotide+Sequence+Database+in+2006." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EMBL+Nucleotide+Sequence+Database+in+2006.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000766-Alexeyenko1" id="pone.0000766-Alexeyenko1"></a>Alexeyenko A, Lindberg J, Perez-Bercoff A, Sonnhammer ELL (2006) Overview and comparison of ortholog databases. Drug Discov Today  11: 137–143.  <ul class="find" data-citedArticleID="979924" data-doi="10.1016/j.ddtec.2006.06.002"><li><a href="http://dx.doi.org/10.1016/j.ddtec.2006.06.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overview+and+comparison+of+ortholog+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overview+and+comparison+of+ortholog+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000766-Li1" id="pone.0000766-Li1"></a>Li L, Stoeckert CJ Jr, Roos DS (2003) OrthoMCL: identification of ortholog groups for eukaryotic genomes. Genome Res  13: 2178–2189.  <ul class="find" data-citedArticleID="980038" data-doi="10.1101/gr.1224503"><li><a href="http://dx.doi.org/10.1101/gr.1224503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OrthoMCL%3A+identification+of+ortholog+groups+for+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OrthoMCL%3A+identification+of+ortholog+groups+for+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000766-Thompson1" id="pone.0000766-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="980128" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000766-Lenffer1" id="pone.0000766-Lenffer1"></a>Lenffer J, Nicholas FW, Castle K, Rao A, Gregory S, et al.  (2006) OMIA (Online Mendelian Inheritance in Animals): an enhanced platform and integration into the Entrez search interface at NCBI. Nucleic Acids Res  34: D599–601.  <ul class="find" data-citedArticleID="980036" data-doi="10.1093/nar/gkj152"><li><a href="http://dx.doi.org/10.1093/nar/gkj152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OMIA+%28Online+Mendelian+Inheritance+in+Animals%29%3A+an+enhanced+platform+and+integration+into+the+Entrez+search+interface+at+NCBI." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OMIA+%28Online+Mendelian+Inheritance+in+Animals%29%3A+an+enhanced+platform+and+integration+into+the+Entrez+search+interface+at+NCBI.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000766-Chen1" id="pone.0000766-Chen1"></a>Chen F, Mackey AJ, Vermunt JK, Roos DS (2007) Assessing performance of orthology detection strategies applied to eukaryotic genomes. PLoS ONE  2: e383.  <ul class="find" data-citedArticleID="979948" data-doi="10.1371/journal.pone.0000383"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+performance+of+orthology+detection+strategies+applied+to+eukaryotic+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+performance+of+orthology+detection+strategies+applied+to+eukaryotic+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000766-Dwight1" id="pone.0000766-Dwight1"></a>Dwight SS, Harris MA, Dolinski K, Ball CA, Binkley G, et al.  (2002) Saccharomyces Genome Database (SGD) provides secondary gene annotation using the Gene Ontology (GO). Nucleic Acids Res  30: 69–72.  <ul class="find" data-citedArticleID="979964" data-doi="10.1093/nar/30.1.69"><li><a href="http://dx.doi.org/10.1093/nar/30.1.69" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saccharomyces+Genome+Database+%28SGD%29+provides+secondary+gene+annotation+using+the+Gene+Ontology+%28GO%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Saccharomyces+Genome+Database+%28SGD%29+provides+secondary+gene+annotation+using+the+Gene+Ontology+%28GO%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000766-Boyle1" id="pone.0000766-Boyle1"></a>Boyle EI, Weng S, Gollub J, Jin H, Botstein D, et al.  (2004) GO::TermFinder–open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes. Bioinformatics  20: 3710–3715.  <ul class="find" data-citedArticleID="979934" data-doi="10.1093/bioinformatics/bth456"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GO%3A%3ATermFinder%26%23x2013%3Bopen+source+software+for+accessing+Gene+Ontology+information+and+finding+significantly+enriched+Gene+Ontology+terms+associated+with+a+list+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GO%3A%3ATermFinder%26%23x2013%3Bopen+source+software+for+accessing+Gene+Ontology+information+and+finding+significantly+enriched+Gene+Ontology+terms+associated+with+a+list+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000766-Koonin1" id="pone.0000766-Koonin1"></a>Koonin EV (2005) Orthologs, paralogs, and evolutionary genomics. Annu Rev Genet  39: 309–338.  <ul class="find" data-citedArticleID="980022" data-doi="10.1146/annurev.genet.39.073003.114725"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.114725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orthologs%2C+paralogs%2C+and+evolutionary+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orthologs%2C+paralogs%2C+and+evolutionary+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000766-Kron1" id="pone.0000766-Kron1"></a>Kron SJ, Drubin DG, Botstein D, Spudich JA (1992) Yeast actin filaments display ATP-dependent sliding movement over surfaces coated with rabbit muscle myosin. Proc Natl Acad Sci U S A  89: 4466–4470.  <ul class="find" data-citedArticleID="980024" data-doi="10.1073/pnas.89.10.4466"><li><a href="http://dx.doi.org/10.1073/pnas.89.10.4466" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+actin+filaments+display+ATP-dependent+sliding+movement+over+surfaces+coated+with+rabbit+muscle+myosin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+actin+filaments+display+ATP-dependent+sliding+movement+over+surfaces+coated+with+rabbit+muscle+myosin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000766-Keeling1" id="pone.0000766-Keeling1"></a>Keeling PJ, Doolittle WF (1996) Alpha-tubulin from early-diverging eukaryotic lineages and the evolution of the tubulin family. Mol Biol Evol  13: 1297–1305.  <ul class="find" data-citedArticleID="980010" data-doi="10.1093/oxfordjournals.molbev.a025576"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alpha-tubulin+from+early-diverging+eukaryotic+lineages+and+the+evolution+of+the+tubulin+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alpha-tubulin+from+early-diverging+eukaryotic+lineages+and+the+evolution+of+the+tubulin+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000766-Dutcher1" id="pone.0000766-Dutcher1"></a>Dutcher SK (2003) Long-lost relatives reappear: identification of new members of the tubulin superfamily. Curr Opin Microbiol  6: 634–640.  <ul class="find" data-citedArticleID="979962" data-doi="10.1016/j.mib.2003.10.016"><li><a href="http://dx.doi.org/10.1016/j.mib.2003.10.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long-lost+relatives+reappear%3A+identification+of+new+members+of+the+tubulin+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long-lost+relatives+reappear%3A+identification+of+new+members+of+the+tubulin+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000766-Archambault1" id="pone.0000766-Archambault1"></a>Archambault J, Friesen JD (1993) Genetics of eukaryotic RNA polymerases I, II, and III. Microbiol Rev  57: 703–724.  <ul class="find" data-citedArticleID="979928"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetics+of+eukaryotic+RNA+polymerases+I%2C+II%2C+and+III.&amp;auth=&amp;atitle=Genetics+of+eukaryotic+RNA+polymerases+I%2C+II%2C+and+III." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetics+of+eukaryotic+RNA+polymerases+I%2C+II%2C+and+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetics+of+eukaryotic+RNA+polymerases+I%2C+II%2C+and+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000766-Herr1" id="pone.0000766-Herr1"></a>Herr AJ, Jensen MB, Dalmay T, Baulcombe DC (2005) RNA polymerase IV directs silencing of endogenous DNA. Science  308: 118–120.  <ul class="find" data-citedArticleID="979992" data-doi="10.1126/science.1106910"><li><a href="http://dx.doi.org/10.1126/science.1106910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+polymerase+IV+directs+silencing+of+endogenous+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+polymerase+IV+directs+silencing+of+endogenous+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000766-Onodera1" id="pone.0000766-Onodera1"></a>Onodera Y, Haag JR, Ream T, Nunes PC, Pontes O, et al.  (2005) Plant nuclear RNA polymerase IV mediates siRNA and DNA methylation-dependent heterochromatin formation. Cell  120: 613–622.  <ul class="find" data-citedArticleID="980074" data-doi="10.1016/j.cell.2005.02.007"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.02.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+nuclear+RNA+polymerase+IV+mediates+siRNA+and+DNA+methylation-dependent+heterochromatin+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+nuclear+RNA+polymerase+IV+mediates+siRNA+and+DNA+methylation-dependent+heterochromatin+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000766-Gowda1" id="pone.0000766-Gowda1"></a>Gowda DC, Gupta P, Davidson EA (1997) Glycosylphosphatidylinositol anchors represent the major carbohydrate modification in proteins of intraerythrocytic stage Plasmodium falciparum. J Biol Chem  272: 6428–6439.  <ul class="find" data-citedArticleID="979984" data-doi="10.1074/jbc.272.10.6428"><li><a href="http://dx.doi.org/10.1074/jbc.272.10.6428" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Glycosylphosphatidylinositol+anchors+represent+the+major+carbohydrate+modification+in+proteins+of+intraerythrocytic+stage+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Glycosylphosphatidylinositol+anchors+represent+the+major+carbohydrate+modification+in+proteins+of+intraerythrocytic+stage+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000766-Kedees1" id="pone.0000766-Kedees1"></a>Kedees MH, Azzouz N, Gerold P, Shams-Eldin H, Iqbal J, et al.  (2002) Plasmodium falciparum: glycosylation status of Plasmodium falciparum circumsporozoite protein expressed in the baculovirus system. Exp Parasitol  101: 64–68.  <ul class="find" data-citedArticleID="980008" data-doi="10.1016/s0014-4894(02)00030-9"><li><a href="http://dx.doi.org/10.1016/s0014-4894(02)00030-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plasmodium+falciparum%3A+glycosylation+status+of+Plasmodium+falciparum+circumsporozoite+protein+expressed+in+the+baculovirus+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plasmodium+falciparum%3A+glycosylation+status+of+Plasmodium+falciparum+circumsporozoite+protein+expressed+in+the+baculovirus+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000766-Kocken1" id="pone.0000766-Kocken1"></a>Kocken CH, Withers-Martinez C, Dubbeld MA, van der Wel A, Hackett F, et al.  (2002) High-level expression of the malaria blood-stage vaccine candidate Plasmodium falciparum apical membrane antigen 1 and induction of antibodies that inhibit erythrocyte invasion. Infect Immun  70: 4471–4476.  <ul class="find" data-citedArticleID="980020" data-doi="10.1128/iai.70.8.4471-4476.2002"><li><a href="http://dx.doi.org/10.1128/iai.70.8.4471-4476.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-level+expression+of+the+malaria+blood-stage+vaccine+candidate+Plasmodium+falciparum+apical+membrane+antigen+1+and+induction+of+antibodies+that+inhibit+erythrocyte+invasion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-level+expression+of+the+malaria+blood-stage+vaccine+candidate+Plasmodium+falciparum+apical+membrane+antigen+1+and+induction+of+antibodies+that+inhibit+erythrocyte+invasion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000766-Alexeyenko2" id="pone.0000766-Alexeyenko2"></a>Alexeyenko A, Tamas I, Liu G, Sonnhammer EL (2006) Automatic clustering of orthologs and inparalogs shared by multiple proteomes. Bioinformatics  22: e9–15.  <ul class="find" data-citedArticleID="979926" data-doi="10.1093/bioinformatics/btl213"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+clustering+of+orthologs+and+inparalogs+shared+by+multiple+proteomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+clustering+of+orthologs+and+inparalogs+shared+by+multiple+proteomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000766-Jothi1" id="pone.0000766-Jothi1"></a>Jothi R, Zotenko E, Tasneem A, Przytycka TM (2006) COCO-CL: hierarchical clustering of homology relations based on evolutionary correlations. Bioinformatics  22: 779–788.  <ul class="find" data-citedArticleID="980004" data-doi="10.1093/bioinformatics/btl009"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=COCO-CL%3A+hierarchical+clustering+of+homology+relations+based+on+evolutionary+correlations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22COCO-CL%3A+hierarchical+clustering+of+homology+relations+based+on+evolutionary+correlations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000766-Tatusov1" id="pone.0000766-Tatusov1"></a>Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, et al.  (2003) The COG database: an updated version includes eukaryotes. BMC Bioinformatics  4: 41.  <ul class="find" data-citedArticleID="980126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+COG+database%3A+an+updated+version+includes+eukaryotes.&amp;auth=&amp;atitle=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COG+database%3A+an+updated+version+includes+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000766-Lee1" id="pone.0000766-Lee1"></a>Lee Y, Sultana R, Pertea G, Cho J, Karamycheva S, et al.  (2002) Cross-referencing eukaryotic genomes: TIGR Orthologous Gene Alignments (TOGA). Genome Res  12: 493–502.  <ul class="find" data-citedArticleID="980034" data-doi="10.1101/gr.212002"><li><a href="http://dx.doi.org/10.1101/gr.212002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-referencing+eukaryotic+genomes%3A+TIGR+Orthologous+Gene+Alignments+%28TOGA%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-referencing+eukaryotic+genomes%3A+TIGR+Orthologous+Gene+Alignments+%28TOGA%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000766-Wheeler1" id="pone.0000766-Wheeler1"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2006) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  34: D173–180.  <ul class="find" data-citedArticleID="980142" data-doi="10.1093/nar/gkj158"><li><a href="http://dx.doi.org/10.1093/nar/gkj158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000766-OBrien1" id="pone.0000766-OBrien1"></a>O'Brien KP, Remm M, Sonnhammer EL (2005) Inparanoid: a comprehensive database of eukaryotic orthologs. Nucleic Acids Res  33: D476–480.  <ul class="find" data-citedArticleID="980068" data-doi="10.1093/nar/gki107"><li><a href="http://dx.doi.org/10.1093/nar/gki107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inparanoid%3A+a+comprehensive+database+of+eukaryotic+orthologs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inparanoid%3A+a+comprehensive+database+of+eukaryotic+orthologs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000766-OBrien2" id="pone.0000766-OBrien2"></a>O'Brien KP, Westerlund I, Sonnhammer EL (2004) OrthoDisease: a database of human disease orthologs. Hum Mutat  24: 112–119.  <ul class="find" data-citedArticleID="980070" data-doi="10.1002/humu.20068"><li><a href="http://dx.doi.org/10.1002/humu.20068" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OrthoDisease%3A+a+database+of+human+disease+orthologs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OrthoDisease%3A+a+database+of+human+disease+orthologs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000766-Chen2" id="pone.0000766-Chen2"></a>Chen F, Mackey AJ, Stoeckert CJ Jr, Roos DS (2006) OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups. Nucleic Acids Res  34: D363–368.  <ul class="find" data-citedArticleID="979950" data-doi="10.1093/nar/gkj123"><li><a href="http://dx.doi.org/10.1093/nar/gkj123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OrthoMCL-DB%3A+querying+a+comprehensive+multi-species+collection+of+ortholog+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OrthoMCL-DB%3A+querying+a+comprehensive+multi-species+collection+of+ortholog+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000766-Penkett1" id="pone.0000766-Penkett1"></a>Penkett CJ, Morris JA, Wood V, Bahler J (2006) YOGY: a web-based, integrated database to retrieve protein orthologs and associated Gene Ontology terms. Nucleic Acids Res  34: W330–334.  <ul class="find" data-citedArticleID="980082" data-doi="10.1093/nar/gkl311"><li><a href="http://dx.doi.org/10.1093/nar/gkl311" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=YOGY%3A+a+web-based%2C+integrated+database+to+retrieve+protein+orthologs+and+associated+Gene+Ontology+terms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22YOGY%3A+a+web-based%2C+integrated+database+to+retrieve+protein+orthologs+and+associated+Gene+Ontology+terms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000766-SamuelLattimore1" id="pone.0000766-SamuelLattimore1"></a>Samuel Lattimore B, van Dongen S, Crabbe MJ (2005) GeneMCL in microarray analysis. Comput Biol Chem  29: 354–359.  <ul class="find" data-citedArticleID="980100" data-doi="10.1016/j.compbiolchem.2005.07.002"><li><a href="http://dx.doi.org/10.1016/j.compbiolchem.2005.07.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneMCL+in+microarray+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneMCL+in+microarray+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000766-Kelleher1" id="pone.0000766-Kelleher1"></a>Kelleher DJ, Gilmore R (1994) The Saccharomyces cerevisiae oligosaccharyltransferase is a protein complex composed of Wbp1p, Swp1p, and four additional polypeptides. J Biol Chem  269: 12908–12917.  <ul class="find" data-citedArticleID="980012"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Saccharomyces+cerevisiae+oligosaccharyltransferase+is+a+protein+complex+composed+of+Wbp1p%2C+Swp1p%2C+and+four+additional+polypeptides.&amp;auth=&amp;atitle=The+Saccharomyces+cerevisiae+oligosaccharyltransferase+is+a+protein+complex+composed+of+Wbp1p%2C+Swp1p%2C+and+four+additional+polypeptides." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Saccharomyces+cerevisiae+oligosaccharyltransferase+is+a+protein+complex+composed+of+Wbp1p%2C+Swp1p%2C+and+four+additional+polypeptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Saccharomyces+cerevisiae+oligosaccharyltransferase+is+a+protein+complex+composed+of+Wbp1p%2C+Swp1p%2C+and+four+additional+polypeptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000766-Nomoto1" id="pone.0000766-Nomoto1"></a>Nomoto S, Watanabe Y, Ninomiya-Tsuji J, Yang LX, Nagai Y, et al.  (1997) Functional analyses of mammalian protein kinase C isozymes in budding yeast and mammalian fibroblasts. Genes Cells  2: 601–614.  <ul class="find" data-citedArticleID="980066" data-doi="10.1046/j.1365-2443.1997.1470346.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2443.1997.1470346.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analyses+of+mammalian+protein+kinase+C+isozymes+in+budding+yeast+and+mammalian+fibroblasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analyses+of+mammalian+protein+kinase+C+isozymes+in+budding+yeast+and+mammalian+fibroblasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000766-Kida1" id="pone.0000766-Kida1"></a>Kida Y, Uchida S, Miyazaki H, Sasaki S, Marumo F (2001) Localization of mouse CLC-6 and CLC-7 mRNA and their functional complementation of yeast CLC gene mutant. Histochem Cell Biol  115: 189–194.  <ul class="find" data-citedArticleID="980016"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Localization+of+mouse+CLC-6+and+CLC-7+mRNA+and+their+functional+complementation+of+yeast+CLC+gene+mutant.&amp;auth=&amp;atitle=Localization+of+mouse+CLC-6+and+CLC-7+mRNA+and+their+functional+complementation+of+yeast+CLC+gene+mutant." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localization+of+mouse+CLC-6+and+CLC-7+mRNA+and+their+functional+complementation+of+yeast+CLC+gene+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localization+of+mouse+CLC-6+and+CLC-7+mRNA+and+their+functional+complementation+of+yeast+CLC+gene+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000766-Yamagata1" id="pone.0000766-Yamagata1"></a>Yamagata K, Kato J, Shimamoto A, Goto M, Furuichi Y, et al.  (1998) Bloom's and Werner's syndrome genes suppress hyperrecombination in yeast sgs1 mutant: implication for genomic instability in human diseases. Proc Natl Acad Sci U S A  95: 8733–8738.  <ul class="find" data-citedArticleID="980148" data-doi="10.1073/pnas.95.15.8733"><li><a href="http://dx.doi.org/10.1073/pnas.95.15.8733" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bloom%27s+and+Werner%27s+syndrome+genes+suppress+hyperrecombination+in+yeast+sgs1+mutant%3A+implication+for+genomic+instability+in+human+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bloom%27s+and+Werner%27s+syndrome+genes+suppress+hyperrecombination+in+yeast+sgs1+mutant%3A+implication+for+genomic+instability+in+human+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000766-Clark1" id="pone.0000766-Clark1"></a>Clark AB, Cook ME, Tran HT, Gordenin DA, Resnick MA, et al.  (1999) Functional analysis of human MutSalpha and MutSbeta complexes in yeast. Nucleic Acids Res  27: 736–742.  <ul class="find" data-citedArticleID="979954" data-doi="10.1093/nar/27.3.736"><li><a href="http://dx.doi.org/10.1093/nar/27.3.736" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+of+human+MutSalpha+and+MutSbeta+complexes+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+of+human+MutSalpha+and+MutSbeta+complexes+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000766-Garbers1" id="pone.0000766-Garbers1"></a>Garbers C, DeLong A, Deruere J, Bernasconi P, Soll D (1996) A mutation in protein phosphatase 2A regulatory subunit A affects auxin transport in Arabidopsis. Embo J  15: 2115–2124.  <ul class="find" data-citedArticleID="979976" data-doi="10.1016/s1360-1385(96)86891-0"><li><a href="http://dx.doi.org/10.1016/s1360-1385(96)86891-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mutation+in+protein+phosphatase+2A+regulatory+subunit+A+affects+auxin+transport+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mutation+in+protein+phosphatase+2A+regulatory+subunit+A+affects+auxin+transport+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000766-Frank1" id="pone.0000766-Frank1"></a>Frank CG, Grubenmann CE, Eyaid W, Berger EG, Aebi M, et al.  (2004) Identification and functional analysis of a defect in the human ALG9 gene: definition of congenital disorder of glycosylation type IL. Am J Hum Genet  75: 146–150.  <ul class="find" data-citedArticleID="979970" data-doi="10.1086/422367"><li><a href="http://dx.doi.org/10.1086/422367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+functional+analysis+of+a+defect+in+the+human+ALG9+gene%3A+definition+of+congenital+disorder+of+glycosylation+type+IL." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+functional+analysis+of+a+defect+in+the+human+ALG9+gene%3A+definition+of+congenital+disorder+of+glycosylation+type+IL.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000766-Schwarz1" id="pone.0000766-Schwarz1"></a>Schwarz M, Thiel C, Lubbehusen J, Dorland B, de Koning T, et al.  (2004) Deficiency of GDP-Man:GlcNAc2-PP-dolichol mannosyltransferase causes congenital disorder of glycosylation type Ik. Am J Hum Genet  74: 472–481.  <ul class="find" data-citedArticleID="980112" data-doi="10.1086/382492"><li><a href="http://dx.doi.org/10.1086/382492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deficiency+of+GDP-Man%3AGlcNAc2-PP-dolichol+mannosyltransferase+causes+congenital+disorder+of+glycosylation+type+Ik." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deficiency+of+GDP-Man%3AGlcNAc2-PP-dolichol+mannosyltransferase+causes+congenital+disorder+of+glycosylation+type+Ik.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000766-Ballester1" id="pone.0000766-Ballester1"></a>Ballester R, Marchuk D, Boguski M, Saulino A, Letcher R, et al.  (1990) The NF1 locus encodes a protein functionally related to mammalian GAP and yeast IRA proteins. Cell  63: 851–859.  <ul class="find" data-citedArticleID="979930" data-doi="10.1016/0092-8674(90)90151-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90151-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+NF1+locus+encodes+a+protein+functionally+related+to+mammalian+GAP+and+yeast+IRA+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+NF1+locus+encodes+a+protein+functionally+related+to+mammalian+GAP+and+yeast+IRA+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000766-Poullet1" id="pone.0000766-Poullet1"></a>Poullet P, Lin B, Esson K, Tamanoi F (1994) Functional significance of lysine 1423 of neurofibromin and characterization of a second site suppressor which rescues mutations at this residue and suppresses RAS2Val-19-activated phenotypes. Mol Cell Biol  14: 815–821.  <ul class="find" data-citedArticleID="980086"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+significance+of+lysine+1423+of+neurofibromin+and+characterization+of+a+second+site+suppressor+which+rescues+mutations+at+this+residue+and+suppresses+RAS2Val-19-activated+phenotypes.&amp;auth=&amp;atitle=Functional+significance+of+lysine+1423+of+neurofibromin+and+characterization+of+a+second+site+suppressor+which+rescues+mutations+at+this+residue+and+suppresses+RAS2Val-19-activated+phenotypes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+significance+of+lysine+1423+of+neurofibromin+and+characterization+of+a+second+site+suppressor+which+rescues+mutations+at+this+residue+and+suppresses+RAS2Val-19-activated+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+significance+of+lysine+1423+of+neurofibromin+and+characterization+of+a+second+site+suppressor+which+rescues+mutations+at+this+residue+and+suppresses+RAS2Val-19-activated+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000766-Gecz1" id="pone.0000766-Gecz1"></a>Gecz J, Shaw MA, Bellon JR, de Barros Lopes M (2003) Human wild-type SEDL protein functionally complements yeast Trs20p but some naturally occurring SEDL mutants do not. Gene  320: 137–144.  <ul class="find" data-citedArticleID="979978" data-doi="10.1016/s0378-1119(03)00819-9"><li><a href="http://dx.doi.org/10.1016/s0378-1119(03)00819-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+wild-type+SEDL+protein+functionally+complements+yeast+Trs20p+but+some+naturally+occurring+SEDL+mutants+do+not." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+wild-type+SEDL+protein+functionally+complements+yeast+Trs20p+but+some+naturally+occurring+SEDL+mutants+do+not.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000766-Gao1" id="pone.0000766-Gao1"></a>Gao XD, Wang J, Keppler-Ross S, Dean N (2005) ERS1 encodes a functional homologue of the human lysosomal cystine transporter. Febs J  272: 2497–2511.  <ul class="find" data-citedArticleID="979972" data-doi="10.1111/j.1742-4658.2005.04670.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.04670.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ERS1+encodes+a+functional+homologue+of+the+human+lysosomal+cystine+transporter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ERS1+encodes+a+functional+homologue+of+the+human+lysosomal+cystine+transporter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000766-Cavadini1" id="pone.0000766-Cavadini1"></a>Cavadini P, Gellera C, Patel PI, Isaya G (2000) Human frataxin maintains mitochondrial iron homeostasis in Saccharomyces cerevisiae. Hum Mol Genet  9: 2523–2530.  <ul class="find" data-citedArticleID="979944" data-doi="10.1093/hmg/9.17.2523"><li><a href="http://dx.doi.org/10.1093/hmg/9.17.2523" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+frataxin+maintains+mitochondrial+iron+homeostasis+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+frataxin+maintains+mitochondrial+iron+homeostasis+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000766-Desmyter1" id="pone.0000766-Desmyter1"></a>Desmyter L, Dewaele S, Reekmans R, Nystrom T, Contreras R, et al.  (2004) Expression of the human ferritin light chain in a frataxin mutant yeast affects ageing and cell death. Exp Gerontol  39: 707–715.  <ul class="find" data-citedArticleID="979960" data-doi="10.1016/j.exger.2004.01.008"><li><a href="http://dx.doi.org/10.1016/j.exger.2004.01.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+human+ferritin+light+chain+in+a+frataxin+mutant+yeast+affects+ageing+and+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+human+ferritin+light+chain+in+a+frataxin+mutant+yeast+affects+ageing+and+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000766-Feiler1" id="pone.0000766-Feiler1"></a>Feiler HS, Desprez T, Santoni V, Kronenberger J, Caboche M, et al.  (1995) The higher plant Arabidopsis thaliana encodes a functional CDC48 homologue which is highly expressed in dividing and expanding cells. Embo J  14: 5626–5637.  <ul class="find" data-citedArticleID="979966"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+higher+plant+Arabidopsis+thaliana+encodes+a+functional+CDC48+homologue+which+is+highly+expressed+in+dividing+and+expanding+cells.&amp;auth=&amp;atitle=The+higher+plant+Arabidopsis+thaliana+encodes+a+functional+CDC48+homologue+which+is+highly+expressed+in+dividing+and+expanding+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+higher+plant+Arabidopsis+thaliana+encodes+a+functional+CDC48+homologue+which+is+highly+expressed+in+dividing+and+expanding+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+higher+plant+Arabidopsis+thaliana+encodes+a+functional+CDC48+homologue+which+is+highly+expressed+in+dividing+and+expanding+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000766-Hsi1" id="pone.0000766-Hsi1"></a>Hsi G, Cullen LM, Moira Glerum D, Cox DW (2004) Functional assessment of the carboxy-terminus of the Wilson disease copper-transporting ATPase, ATP7B. Genomics  83: 473–481.  <ul class="find" data-citedArticleID="979996" data-doi="10.1016/j.ygeno.2003.08.022"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2003.08.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+assessment+of+the+carboxy-terminus+of+the+Wilson+disease+copper-transporting+ATPase%2C+ATP7B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+assessment+of+the+carboxy-terminus+of+the+Wilson+disease+copper-transporting+ATPase%2C+ATP7B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000766-Bussey1" id="pone.0000766-Bussey1"></a>Bussey H, Storms RK, Ahmed A, Albermann K, Allen E, et al.  (1997) The nucleotide sequence of Saccharomyces cerevisiae chromosome XVI. Nature  387: 103–105.  <ul class="find" data-citedArticleID="979940"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+nucleotide+sequence+of+Saccharomyces+cerevisiae+chromosome+XVI.&amp;auth=&amp;atitle=The+nucleotide+sequence+of+Saccharomyces+cerevisiae+chromosome+XVI." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nucleotide+sequence+of+Saccharomyces+cerevisiae+chromosome+XVI." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nucleotide+sequence+of+Saccharomyces+cerevisiae+chromosome+XVI.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000766-Portmann1" id="pone.0000766-Portmann1"></a>Portmann R, Solioz M (2005) Purification and functional reconstitution of the human Wilson copper ATPase, ATP7B. FEBS Lett  579: 3589–3595.  <ul class="find" data-citedArticleID="980084" data-doi="10.1016/j.febslet.2005.05.042"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.05.042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification+and+functional+reconstitution+of+the+human+Wilson+copper+ATPase%2C+ATP7B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification+and+functional+reconstitution+of+the+human+Wilson+copper+ATPase%2C+ATP7B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000766-Sambongi1" id="pone.0000766-Sambongi1"></a>Sambongi Y, Wakabayashi T, Yoshimizu T, Omote H, Oka T, et al.  (1997) Caenorhabditis elegans cDNA for a Menkes/Wilson disease gene homologue and its function in a yeast CCC2 gene deletion mutant. J Biochem (Tokyo)  121: 1169–1175.  <ul class="find" data-citedArticleID="980098" data-doi="10.1093/oxfordjournals.jbchem.a021711"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a021711" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caenorhabditis+elegans+cDNA+for+a+Menkes%2FWilson+disease+gene+homologue+and+its+function+in+a+yeast+CCC2+gene+deletion+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caenorhabditis+elegans+cDNA+for+a+Menkes%2FWilson+disease+gene+homologue+and+its+function+in+a+yeast+CCC2+gene+deletion+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000766-Mercer1" id="pone.0000766-Mercer1"></a>Mercer JF, Barnes N, Stevenson J, Strausak D, Llanos RM (2003) Copper-induced trafficking of the cU-ATPases: a key mechanism for copper homeostasis. Biometals  16: 175–184.  <ul class="find" data-citedArticleID="980058" data-doi="10.1023/a:1020719016675"><li><a href="http://dx.doi.org/10.1023/a:1020719016675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Copper-induced+trafficking+of+the+cU-ATPases%3A+a+key+mechanism+for+copper+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Copper-induced+trafficking+of+the+cU-ATPases%3A+a+key+mechanism+for+copper+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000766-Payne1" id="pone.0000766-Payne1"></a>Payne AS, Gitlin JD (1998) Functional expression of the menkes disease protein reveals common biochemical mechanisms among the copper-transporting P-type ATPases. J Biol Chem  273: 3765–3770.  <ul class="find" data-citedArticleID="980076" data-doi="10.1074/jbc.273.6.3765"><li><a href="http://dx.doi.org/10.1074/jbc.273.6.3765" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+expression+of+the+menkes+disease+protein+reveals+common+biochemical+mechanisms+among+the+copper-transporting+P-type+ATPases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+expression+of+the+menkes+disease+protein+reveals+common+biochemical+mechanisms+among+the+copper-transporting+P-type+ATPases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000766-Jantti1" id="pone.0000766-Jantti1"></a>Jantti J, Lahdenranta J, Olkkonen VM, Soderlund H, Keranen S (1999) SEM1, a homologue of the split hand/split foot malformation candidate gene Dss1, regulates exocytosis and pseudohyphal differentiation in yeast. Proc Natl Acad Sci U S A  96: 909–914.  <ul class="find" data-citedArticleID="980000" data-doi="10.1073/pnas.96.3.909"><li><a href="http://dx.doi.org/10.1073/pnas.96.3.909" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SEM1%2C+a+homologue+of+the+split+hand%2Fsplit+foot+malformation+candidate+gene+Dss1%2C+regulates+exocytosis+and+pseudohyphal+differentiation+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SEM1%2C+a+homologue+of+the+split+hand%2Fsplit+foot+malformation+candidate+gene+Dss1%2C+regulates+exocytosis+and+pseudohyphal+differentiation+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000766-Sone1" id="pone.0000766-Sone1"></a>Sone T, Saeki Y, Toh-e A, Yokosawa H (2004) Sem1p is a novel subunit of the 26 S proteasome from Saccharomyces cerevisiae. J Biol Chem  279: 28807–28816.  <ul class="find" data-citedArticleID="980116" data-doi="10.1074/jbc.m403165200"><li><a href="http://dx.doi.org/10.1074/jbc.m403165200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sem1p+is+a+novel+subunit+of+the+26+S+proteasome+from+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sem1p+is+a+novel+subunit+of+the+26+S+proteasome+from+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000766-Morita1" id="pone.0000766-Morita1"></a>Morita T, Yoshimura Y, Yamamoto A, Murata K, Mori M, et al.  (1993) A mouse homolog of the Escherichia coli recA and Saccharomyces cerevisiae RAD51 genes. Proc Natl Acad Sci U S A  90: 6577–6580.  <ul class="find" data-citedArticleID="980062" data-doi="10.1073/pnas.90.14.6577"><li><a href="http://dx.doi.org/10.1073/pnas.90.14.6577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mouse+homolog+of+the+Escherichia+coli+recA+and+Saccharomyces+cerevisiae+RAD51+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mouse+homolog+of+the+Escherichia+coli+recA+and+Saccharomyces+cerevisiae+RAD51+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000766-Loewen1" id="pone.0000766-Loewen1"></a>Loewen CJ, Levine TP (2005) A highly conserved binding site in vesicle-associated membrane protein-associated protein (VAP) for the FFAT motif of lipid-binding proteins. J Biol Chem  280: 14097–14104.  <ul class="find" data-citedArticleID="980044" data-doi="10.1074/jbc.m500147200"><li><a href="http://dx.doi.org/10.1074/jbc.m500147200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+highly+conserved+binding+site+in+vesicle-associated+membrane+protein-associated+protein+%28VAP%29+for+the+FFAT+motif+of+lipid-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+highly+conserved+binding+site+in+vesicle-associated+membrane+protein-associated+protein+%28VAP%29+for+the+FFAT+motif+of+lipid-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000766-Guzder1" id="pone.0000766-Guzder1"></a>Guzder SN, Sung P, Prakash S, Prakash L (1995) Lethality in yeast of trichothiodystrophy (TTD) mutations in the human xeroderma pigmentosum group D gene. Implications for transcriptional defect in TTD. J Biol Chem  270: 17660–17663.  <ul class="find" data-citedArticleID="979988" data-doi="10.1074/jbc.270.30.17660"><li><a href="http://dx.doi.org/10.1074/jbc.270.30.17660" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lethality+in+yeast+of+trichothiodystrophy+%28TTD%29+mutations+in+the+human+xeroderma+pigmentosum+group+D+gene.+Implications+for+transcriptional+defect+in+TTD." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lethality+in+yeast+of+trichothiodystrophy+%28TTD%29+mutations+in+the+human+xeroderma+pigmentosum+group+D+gene.+Implications+for+transcriptional+defect+in+TTD.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000766-Sung1" id="pone.0000766-Sung1"></a>Sung P, Bailly V, Weber C, Thompson LH, Prakash L, et al.  (1993) Human xeroderma pigmentosum group D gene encodes a DNA helicase. Nature  365: 852–855.  <ul class="find" data-citedArticleID="980122" data-doi="10.1038/365852a0"><li><a href="http://dx.doi.org/10.1038/365852a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+xeroderma+pigmentosum+group+D+gene+encodes+a+DNA+helicase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+xeroderma+pigmentosum+group+D+gene+encodes+a+DNA+helicase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000766-Lanterman1" id="pone.0000766-Lanterman1"></a>Lanterman MM, Dickinson JR, Danner DJ (1996) Functional analysis in Saccharomyces cerevisiae of naturally occurring amino acid substitutions in human dihydrolipoamide dehydrogenase. Hum Mol Genet  5: 1643–1648.  <ul class="find" data-citedArticleID="980032" data-doi="10.1093/hmg/5.10.1643"><li><a href="http://dx.doi.org/10.1093/hmg/5.10.1643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+analysis+in+Saccharomyces+cerevisiae+of+naturally+occurring+amino+acid+substitutions+in+human+dihydrolipoamide+dehydrogenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+analysis+in+Saccharomyces+cerevisiae+of+naturally+occurring+amino+acid+substitutions+in+human+dihydrolipoamide+dehydrogenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000766-McEwen1" id="pone.0000766-McEwen1"></a>McEwen RK, Dove SK, Cooke FT, Painter GF, Holmes AB, et al.  (1999) Complementation analysis in PtdInsP kinase-deficient yeast mutants demonstrates that Schizosaccharomyces pombe and murine Fab1p homologues are phosphatidylinositol 3-phosphate 5-kinases. J Biol Chem  274: 33905–33912.  <ul class="find" data-citedArticleID="980056" data-doi="10.1074/jbc.274.48.33905"><li><a href="http://dx.doi.org/10.1074/jbc.274.48.33905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementation+analysis+in+PtdInsP+kinase-deficient+yeast+mutants+demonstrates+that+Schizosaccharomyces+pombe+and+murine+Fab1p+homologues+are+phosphatidylinositol+3-phosphate+5-kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementation+analysis+in+PtdInsP+kinase-deficient+yeast+mutants+demonstrates+that+Schizosaccharomyces+pombe+and+murine+Fab1p+homologues+are+phosphatidylinositol+3-phosphate+5-kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000766-Mayordomo1" id="pone.0000766-Mayordomo1"></a>Mayordomo I, Sanz P (2001) Human pancreatic glucokinase (GlkB) complements the glucose signalling defect of Saccharomyces cerevisiae hxk2 mutants. Yeast  18: 1309–1316.  <ul class="find" data-citedArticleID="980054"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+pancreatic+glucokinase+%28GlkB%29+complements+the+glucose+signalling+defect+of+Saccharomyces+cerevisiae+hxk2+mutants.&amp;auth=&amp;atitle=Human+pancreatic+glucokinase+%28GlkB%29+complements+the+glucose+signalling+defect+of+Saccharomyces+cerevisiae+hxk2+mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+pancreatic+glucokinase+%28GlkB%29+complements+the+glucose+signalling+defect+of+Saccharomyces+cerevisiae+hxk2+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+pancreatic+glucokinase+%28GlkB%29+complements+the+glucose+signalling+defect+of+Saccharomyces+cerevisiae+hxk2+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000766-Lucas1" id="pone.0000766-Lucas1"></a>Lucas ME, Ma Q, Cunningham D, Peters J, Cattanach B, et al.  (2003) Identification of two novel mutations in the murine Nsdhl sterol dehydrogenase gene and development of a functional complementation assay in yeast. Mol Genet Metab  80: 227–233.  <ul class="find" data-citedArticleID="980046" data-doi="10.1016/s1096-7192(03)00137-9"><li><a href="http://dx.doi.org/10.1016/s1096-7192(03)00137-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+two+novel+mutations+in+the+murine+Nsdhl+sterol+dehydrogenase+gene+and+development+of+a+functional+complementation+assay+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+two+novel+mutations+in+the+murine+Nsdhl+sterol+dehydrogenase+gene+and+development+of+a+functional+complementation+assay+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000766-Geisler1" id="pone.0000766-Geisler1"></a>Geisler M, Frangne N, Gomes E, Martinoia E, Palmgren MG (2000) The ACA4 gene of Arabidopsis encodes a vacuolar membrane calcium pump that improves salt tolerance in yeast. Plant Physiol  124: 1814–1827.  <ul class="find" data-citedArticleID="979980" data-doi="10.1104/pp.124.4.1814"><li><a href="http://dx.doi.org/10.1104/pp.124.4.1814" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ACA4+gene+of+Arabidopsis+encodes+a+vacuolar+membrane+calcium+pump+that+improves+salt+tolerance+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ACA4+gene+of+Arabidopsis+encodes+a+vacuolar+membrane+calcium+pump+that+improves+salt+tolerance+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000766-Schiott1" id="pone.0000766-Schiott1"></a>Schiott M, Romanowsky SM, Baekgaard L, Jakobsen MK, Palmgren MG, et al.  (2004) A plant plasma membrane Ca2+ pump is required for normal pollen tube growth and fertilization. Proc Natl Acad Sci U S A  101: 9502–9507.  <ul class="find" data-citedArticleID="980106" data-doi="10.1073/pnas.0401542101"><li><a href="http://dx.doi.org/10.1073/pnas.0401542101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+plant+plasma+membrane+Ca2%2B+pump+is+required+for+normal+pollen+tube+growth+and+fertilization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+plant+plasma+membrane+Ca2%2B+pump+is+required+for+normal+pollen+tube+growth+and+fertilization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000766-Kleinow1" id="pone.0000766-Kleinow1"></a>Kleinow T, Bhalerao R, Breuer F, Umeda M, Salchert K, et al.  (2000) Functional identification of an Arabidopsis snf4 ortholog by screening for heterologous multicopy suppressors of snf4 deficiency in yeast. Plant J  23: 115–122.  <ul class="find" data-citedArticleID="980018" data-doi="10.1046/j.1365-313x.2000.00809.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2000.00809.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+identification+of+an+Arabidopsis+snf4+ortholog+by+screening+for+heterologous+multicopy+suppressors+of+snf4+deficiency+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+identification+of+an+Arabidopsis+snf4+ortholog+by+screening+for+heterologous+multicopy+suppressors+of+snf4+deficiency+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000766-Lumbreras1" id="pone.0000766-Lumbreras1"></a>Lumbreras V, Alba MM, Kleinow T, Koncz C, Pages M (2001) Domain fusion between SNF1-related kinase subunits during plant evolution. EMBO Rep  2: 55–60.  <ul class="find" data-citedArticleID="980048" data-doi="10.1093/embo-reports/kve001"><li><a href="http://dx.doi.org/10.1093/embo-reports/kve001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Domain+fusion+between+SNF1-related+kinase+subunits+during+plant+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Domain+fusion+between+SNF1-related+kinase+subunits+during+plant+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000766-Roje1" id="pone.0000766-Roje1"></a>Roje S, Wang H, McNeil SD, Raymond RK, Appling DR, et al.  (1999) Isolation, characterization, and functional expression of cDNAs encoding NADH-dependent methylenetetrahydrofolate reductase from higher plants. J Biol Chem  274: 36089–36096.  <ul class="find" data-citedArticleID="980094" data-doi="10.1074/jbc.274.51.36089"><li><a href="http://dx.doi.org/10.1074/jbc.274.51.36089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation%2C+characterization%2C+and+functional+expression+of+cDNAs+encoding+NADH-dependent+methylenetetrahydrofolate+reductase+from+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation%2C+characterization%2C+and+functional+expression+of+cDNAs+encoding+NADH-dependent+methylenetetrahydrofolate+reductase+from+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000766-Raymond1" id="pone.0000766-Raymond1"></a>Raymond RK, Kastanos EK, Appling DR (1999) Saccharomyces cerevisiae expresses two genes encoding isozymes of methylenetetrahydrofolate reductase. Arch Biochem Biophys  372: 300–308.  <ul class="find" data-citedArticleID="980090" data-doi="10.1006/abbi.1999.1498"><li><a href="http://dx.doi.org/10.1006/abbi.1999.1498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Saccharomyces+cerevisiae+expresses+two+genes+encoding+isozymes+of+methylenetetrahydrofolate+reductase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Saccharomyces+cerevisiae+expresses+two+genes+encoding+isozymes+of+methylenetetrahydrofolate+reductase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000766-Ton1" id="pone.0000766-Ton1"></a>Ton VK, Mandal D, Vahadji C, Rao R (2002) Functional expression in yeast of the human secretory pathway Ca(2+), Mn(2+)-ATPase defective in Hailey-Hailey disease. J Biol Chem  277: 6422–6427.  <ul class="find" data-citedArticleID="980134" data-doi="10.1074/jbc.m110612200"><li><a href="http://dx.doi.org/10.1074/jbc.m110612200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+expression+in+yeast+of+the+human+secretory+pathway+Ca%282%2B%29%2C+Mn%282%2B%29-ATPase+defective+in+Hailey-Hailey+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+expression+in+yeast+of+the+human+secretory+pathway+Ca%282%2B%29%2C+Mn%282%2B%29-ATPase+defective+in+Hailey-Hailey+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000766-Ton2" id="pone.0000766-Ton2"></a>Ton VK, Rao R (2004) Expression of Hailey-Hailey disease mutations in yeast. J Invest Dermatol  123: 1192–1194.  <ul class="find" data-citedArticleID="980136" data-doi="10.1111/j.0022-202x.2004.23437.x"><li><a href="http://dx.doi.org/10.1111/j.0022-202x.2004.23437.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+Hailey-Hailey+disease+mutations+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+Hailey-Hailey+disease+mutations+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000766-Kellermayer1" id="pone.0000766-Kellermayer1"></a>Kellermayer R (2005) Hailey-Hailey disease as an orthodisease of PMR1 deficiency in Saccharomyces cerevisiae. FEBS Lett  579: 2021–2025.  <ul class="find" data-citedArticleID="980014" data-doi="10.1016/j.febslet.2005.03.003"><li><a href="http://dx.doi.org/10.1016/j.febslet.2005.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hailey-Hailey+disease+as+an+orthodisease+of+PMR1+deficiency+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hailey-Hailey+disease+as+an+orthodisease+of+PMR1+deficiency+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000766-Heinisch1" id="pone.0000766-Heinisch1"></a>Heinisch JJ (1993) Expression of heterologous phosphofructokinase genes in yeast. FEBS Lett  328: 35–40.  <ul class="find" data-citedArticleID="979990" data-doi="10.1016/0014-5793(93)80960-3"><li><a href="http://dx.doi.org/10.1016/0014-5793(93)80960-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+heterologous+phosphofructokinase+genes+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+heterologous+phosphofructokinase+genes+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000766-Raben1" id="pone.0000766-Raben1"></a>Raben N, Exelbert R, Spiegel R, Sherman JB, Nakajima H, et al.  (1995) Functional expression of human mutant phosphofructokinase in yeast: genetic defects in French Canadian and Swiss patients with phosphofructokinase deficiency. Am J Hum Genet  56: 131–141.  <ul class="find" data-citedArticleID="980088"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+expression+of+human+mutant+phosphofructokinase+in+yeast%3A+genetic+defects+in+French+Canadian+and+Swiss+patients+with+phosphofructokinase+deficiency.&amp;auth=&amp;atitle=Functional+expression+of+human+mutant+phosphofructokinase+in+yeast%3A+genetic+defects+in+French+Canadian+and+Swiss+patients+with+phosphofructokinase+deficiency." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+expression+of+human+mutant+phosphofructokinase+in+yeast%3A+genetic+defects+in+French+Canadian+and+Swiss+patients+with+phosphofructokinase+deficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+expression+of+human+mutant+phosphofructokinase+in+yeast%3A+genetic+defects+in+French+Canadian+and+Swiss+patients+with+phosphofructokinase+deficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000766-Garavaglia1" id="pone.0000766-Garavaglia1"></a>Garavaglia B, Invernizzi F, Carbone ML, Viscardi V, Saracino F, et al.  (2004) GTP-cyclohydrolase I gene mutations in patients with autosomal dominant and recessive GTP-CH1 deficiency: identification and functional characterization of four novel mutations. J Inherit Metab Dis  27: 455–463.  <ul class="find" data-citedArticleID="979974" data-doi="10.1023/b:boli.0000037349.08483.96"><li><a href="http://dx.doi.org/10.1023/b:boli.0000037349.08483.96" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GTP-cyclohydrolase+I+gene+mutations+in+patients+with+autosomal+dominant+and+recessive+GTP-CH1+deficiency%3A+identification+and+functional+characterization+of+four+novel+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GTP-cyclohydrolase+I+gene+mutations+in+patients+with+autosomal+dominant+and+recessive+GTP-CH1+deficiency%3A+identification+and+functional+characterization+of+four+novel+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0000766-Mancini1" id="pone.0000766-Mancini1"></a>Mancini R, Saracino F, Buscemi G, Fischer M, Schramek N, et al.  (1999) Complementation of the fol2 deletion in Saccharomyces cerevisiae by human and Escherichia coli genes encoding GTP cyclohydrolase I. Biochem Biophys Res Commun  255: 521–527.  <ul class="find" data-citedArticleID="980052"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Complementation+of+the+fol2+deletion+in+Saccharomyces+cerevisiae+by+human+and+Escherichia+coli+genes+encoding+GTP+cyclohydrolase+I.&amp;auth=&amp;atitle=Complementation+of+the+fol2+deletion+in+Saccharomyces+cerevisiae+by+human+and+Escherichia+coli+genes+encoding+GTP+cyclohydrolase+I." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementation+of+the+fol2+deletion+in+Saccharomyces+cerevisiae+by+human+and+Escherichia+coli+genes+encoding+GTP+cyclohydrolase+I." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementation+of+the+fol2+deletion+in+Saccharomyces+cerevisiae+by+human+and+Escherichia+coli+genes+encoding+GTP+cyclohydrolase+I.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0000766-Geraghty1" id="pone.0000766-Geraghty1"></a>Geraghty MT, Vaughn D, Nicholson AJ, Lin WW, Jimenez-Sanchez G, et al.  (1998) Mutations in the Delta1-pyrroline 5-carboxylate dehydrogenase gene cause type II hyperprolinemia. Hum Mol Genet  7: 1411–1415.  <ul class="find" data-citedArticleID="979982" data-doi="10.1093/hmg/7.9.1411"><li><a href="http://dx.doi.org/10.1093/hmg/7.9.1411" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+Delta1-pyrroline+5-carboxylate+dehydrogenase+gene+cause+type+II+hyperprolinemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+Delta1-pyrroline+5-carboxylate+dehydrogenase+gene+cause+type+II+hyperprolinemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0000766-Hu1" id="pone.0000766-Hu1"></a>Hu CA, Lin WW, Valle D (1996) Cloning, characterization, and expression of cDNAs encoding human delta 1-pyrroline-5-carboxylate dehydrogenase. J Biol Chem  271: 9795–9800.  <ul class="find" data-citedArticleID="979998" data-doi="10.1074/jbc.271.16.9795"><li><a href="http://dx.doi.org/10.1074/jbc.271.16.9795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning%2C+characterization%2C+and+expression+of+cDNAs+encoding+human+delta+1-pyrroline-5-carboxylate+dehydrogenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning%2C+characterization%2C+and+expression+of+cDNAs+encoding+human+delta+1-pyrroline-5-carboxylate+dehydrogenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0000766-Morgante1" id="pone.0000766-Morgante1"></a>Morgante PG, Berra CM, Nakabashi M, Costa RM, Menck CF, et al.  (2005) Functional XPB/RAD25 redundancy in Arabidopsis genome: characterization of AtXPB2 and expression analysis. Gene  344: 93–103.  <ul class="find" data-citedArticleID="980060" data-doi="10.1016/j.gene.2004.10.006"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.10.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+XPB%2FRAD25+redundancy+in+Arabidopsis+genome%3A+characterization+of+AtXPB2+and+expression+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+XPB%2FRAD25+redundancy+in+Arabidopsis+genome%3A+characterization+of+AtXPB2+and+expression+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0000766-Pearce1" id="pone.0000766-Pearce1"></a>Pearce DA, Sherman F (1998) A yeast model for the study of Batten disease. Proc Natl Acad Sci U S A  95: 6915–6918.  <ul class="find" data-citedArticleID="980078" data-doi="10.1073/pnas.95.12.6915"><li><a href="http://dx.doi.org/10.1073/pnas.95.12.6915" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+yeast+model+for+the+study+of+Batten+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+yeast+model+for+the+study+of+Batten+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0000766-Saunders1" id="pone.0000766-Saunders1"></a>Saunders RD, McLellan LI (2000) Molecular cloning of Drosophila gamma-glutamylcysteine synthetase by functional complementation of a yeast mutant. FEBS Lett  467: 337–340.  <ul class="find" data-citedArticleID="980102" data-doi="10.1016/s0014-5793(00)01148-0"><li><a href="http://dx.doi.org/10.1016/s0014-5793(00)01148-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+Drosophila+gamma-glutamylcysteine+synthetase+by+functional+complementation+of+a+yeast+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+Drosophila+gamma-glutamylcysteine+synthetase+by+functional+complementation+of+a+yeast+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0000766-Srinivasan1" id="pone.0000766-Srinivasan1"></a>Srinivasan C, Liba A, Imlay JA, Valentine JS, Gralla EB (2000) Yeast lacking superoxide dismutase(s) show elevated levels of “free iron” as measured by whole cell electron paramagnetic resonance. J Biol Chem  275: 29187–29192.  <ul class="find" data-citedArticleID="980120" data-doi="10.1074/jbc.m004239200"><li><a href="http://dx.doi.org/10.1074/jbc.m004239200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+lacking+superoxide+dismutase%28s%29+show+elevated+levels+of+%26%23x201c%3Bfree+iron%26%23x201d%3B+as+measured+by+whole+cell+electron+paramagnetic+resonance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+lacking+superoxide+dismutase%28s%29+show+elevated+levels+of+%26%23x201c%3Bfree+iron%26%23x201d%3B+as+measured+by+whole+cell+electron+paramagnetic+resonance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0000766-Agarwal1" id="pone.0000766-Agarwal1"></a>Agarwal AK, Fryns JP, Auchus RJ, Garg A (2003) Zinc metalloproteinase, ZMPSTE24, is mutated in mandibuloacral dysplasia. Hum Mol Genet  12: 1995–2001.  <ul class="find" data-citedArticleID="979922" data-doi="10.1093/hmg/ddg213"><li><a href="http://dx.doi.org/10.1093/hmg/ddg213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zinc+metalloproteinase%2C+ZMPSTE24%2C+is+mutated+in+mandibuloacral+dysplasia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Zinc+metalloproteinase%2C+ZMPSTE24%2C+is+mutated+in+mandibuloacral+dysplasia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0000766-Schmidt1" id="pone.0000766-Schmidt1"></a>Schmidt WK, Tam A, Michaelis S (2000) Reconstitution of the Ste24p-dependent N-terminal proteolytic step in yeast a-factor biogenesis. J Biol Chem  275: 6227–6233.  <ul class="find" data-citedArticleID="980108" data-doi="10.1074/jbc.275.9.6227"><li><a href="http://dx.doi.org/10.1074/jbc.275.9.6227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstitution+of+the+Ste24p-dependent+N-terminal+proteolytic+step+in+yeast+a-factor+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstitution+of+the+Ste24p-dependent+N-terminal+proteolytic+step+in+yeast+a-factor+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0000766-Hofmann1" id="pone.0000766-Hofmann1"></a>Hofmann S, Rothbauer U, Muhlenbein N, Neupert W, Gerbitz KD, et al.  (2002) The C66W mutation in the deafness dystonia peptide 1 (DDP1) affects the formation of functional DDP1.TIM13 complexes in the mitochondrial intermembrane space. J Biol Chem  277: 23287–23293.  <ul class="find" data-citedArticleID="979994" data-doi="10.1074/jbc.m201154200"><li><a href="http://dx.doi.org/10.1074/jbc.m201154200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C66W+mutation+in+the+deafness+dystonia+peptide+1+%28DDP1%29+affects+the+formation+of+functional+DDP1.TIM13+complexes+in+the+mitochondrial+intermembrane+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C66W+mutation+in+the+deafness+dystonia+peptide+1+%28DDP1%29+affects+the+formation+of+functional+DDP1.TIM13+complexes+in+the+mitochondrial+intermembrane+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0000766-Rothbauer1" id="pone.0000766-Rothbauer1"></a>Rothbauer U, Hofmann S, Muhlenbein N, Paschen SA, Gerbitz KD, et al.  (2001) Role of the deafness dystonia peptide 1 (DDP1) in import of human Tim23 into the inner membrane of mitochondria. J Biol Chem  276: 37327–37334.  <ul class="find" data-citedArticleID="980096" data-doi="10.1074/jbc.m105313200"><li><a href="http://dx.doi.org/10.1074/jbc.m105313200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+deafness+dystonia+peptide+1+%28DDP1%29+in+import+of+human+Tim23+into+the+inner+membrane+of+mitochondria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+deafness+dystonia+peptide+1+%28DDP1%29+in+import+of+human+Tim23+into+the+inner+membrane+of+mitochondria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0000766-Raymond2" id="pone.0000766-Raymond2"></a>Raymond M, Gros P, Whiteway M, Thomas DY (1992) Functional complementation of yeast ste6 by a mammalian multidrug resistance mdr gene. Science  256: 232–234.  <ul class="find" data-citedArticleID="980092" data-doi="10.1126/science.1348873"><li><a href="http://dx.doi.org/10.1126/science.1348873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+complementation+of+yeast+ste6+by+a+mammalian+multidrug+resistance+mdr+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+complementation+of+yeast+ste6+by+a+mammalian+multidrug+resistance+mdr+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0000766-Boyum1" id="pone.0000766-Boyum1"></a>Boyum R, Guidotti G (1997) Effect of ATP binding cassette/multidrug resistance proteins on ATP efflux of Saccharomyces cerevisiae. Biochem Biophys Res Commun  230: 22–26.  <ul class="find" data-citedArticleID="979936" data-doi="10.1006/bbrc.1996.5913"><li><a href="http://dx.doi.org/10.1006/bbrc.1996.5913" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+ATP+binding+cassette%2Fmultidrug+resistance+proteins+on+ATP+efflux+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+ATP+binding+cassette%2Fmultidrug+resistance+proteins+on+ATP+efflux+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0000766-Chen3" id="pone.0000766-Chen3"></a>Chen Y, Beck A, Davenport C, Chen Y, Shattuck D, et al.  (2005) Characterization of TRZ1, a yeast homolog of the human candidate prostate cancer susceptibility gene ELAC2 encoding tRNase Z. BMC Mol Biol  6: 12.  <ul class="find" data-citedArticleID="979952"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+TRZ1%2C+a+yeast+homolog+of+the+human+candidate+prostate+cancer+susceptibility+gene+ELAC2+encoding+tRNase+Z.&amp;auth=&amp;atitle=Characterization+of+TRZ1%2C+a+yeast+homolog+of+the+human+candidate+prostate+cancer+susceptibility+gene+ELAC2+encoding+tRNase+Z." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+TRZ1%2C+a+yeast+homolog+of+the+human+candidate+prostate+cancer+susceptibility+gene+ELAC2+encoding+tRNase+Z." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+TRZ1%2C+a+yeast+homolog+of+the+human+candidate+prostate+cancer+susceptibility+gene+ELAC2+encoding+tRNase+Z.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0000766-Peng1" id="pone.0000766-Peng1"></a>Peng Z, Lu Q, Verma DP (1996) Reciprocal regulation of delta 1-pyrroline-5-carboxylate synthetase and proline dehydrogenase genes controls proline levels during and after osmotic stress in plants. Mol Gen Genet  253: 334–341.  <ul class="find" data-citedArticleID="980080" data-doi="10.1007/pl00008600"><li><a href="http://dx.doi.org/10.1007/pl00008600" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reciprocal+regulation+of+delta+1-pyrroline-5-carboxylate+synthetase+and+proline+dehydrogenase+genes+controls+proline+levels+during+and+after+osmotic+stress+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reciprocal+regulation+of+delta+1-pyrroline-5-carboxylate+synthetase+and+proline+dehydrogenase+genes+controls+proline+levels+during+and+after+osmotic+stress+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0000766-Chatterjee1" id="pone.0000766-Chatterjee1"></a>Chatterjee A, Singh KK (2001) Uracil-DNA glycosylase-deficient yeast exhibit a mitochondrial mutator phenotype. Nucleic Acids Res  29: 4935–4940.  <ul class="find" data-citedArticleID="979946" data-doi="10.1093/nar/29.24.4935"><li><a href="http://dx.doi.org/10.1093/nar/29.24.4935" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uracil-DNA+glycosylase-deficient+yeast+exhibit+a+mitochondrial+mutator+phenotype." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uracil-DNA+glycosylase-deficient+yeast+exhibit+a+mitochondrial+mutator+phenotype.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0000766-LillardWetherell1" id="pone.0000766-LillardWetherell1"></a>Lillard-Wetherell K, Combs KA, Groden J (2005) BLM helicase complements disrupted type II telomere lengthening in telomerase-negative sgs1 yeast. Cancer Res  65: 5520–5522.  <ul class="find" data-citedArticleID="980042" data-doi="10.1158/0008-5472.can-05-0632"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-0632" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BLM+helicase+complements+disrupted+type+II+telomere+lengthening+in+telomerase-negative+sgs1+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BLM+helicase+complements+disrupted+type+II+telomere+lengthening+in+telomerase-negative+sgs1+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0000766-Neff1" id="pone.0000766-Neff1"></a>Neff NF, Ellis NA, Ye TZ, Noonan J, Huang K, et al.  (1999) The DNA helicase activity of BLM is necessary for the correction of the genomic instability of bloom syndrome cells. Mol Biol Cell  10: 665–676.  <ul class="find" data-citedArticleID="980064" data-doi="10.1091/mbc.10.3.665"><li><a href="http://dx.doi.org/10.1091/mbc.10.3.665" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+DNA+helicase+activity+of+BLM+is+necessary+for+the+correction+of+the+genomic+instability+of+bloom+syndrome+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+DNA+helicase+activity+of+BLM+is+necessary+for+the+correction+of+the+genomic+instability+of+bloom+syndrome+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0000766-Grubenmann1" id="pone.0000766-Grubenmann1"></a>Grubenmann CE, Frank CG, Kjaergaard S, Berger EG, Aebi M, et al.  (2002) ALG12 mannosyltransferase defect in congenital disorder of glycosylation type lg. Hum Mol Genet  11: 2331–2339.  <ul class="find" data-citedArticleID="979986" data-doi="10.1093/hmg/11.19.2331"><li><a href="http://dx.doi.org/10.1093/hmg/11.19.2331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ALG12+mannosyltransferase+defect+in+congenital+disorder+of+glycosylation+type+lg." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ALG12+mannosyltransferase+defect+in+congenital+disorder+of+glycosylation+type+lg.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0000766-Forsgren1" id="pone.0000766-Forsgren1"></a>Forsgren M, Attersand A, Lake S, Grunler J, Swiezewska E, et al.  (2004) Isolation and functional expression of human COQ2, a gene encoding a polyprenyl transferase involved in the synthesis of CoQ. Biochem J  382: 519–526.  <ul class="find" data-citedArticleID="979968" data-doi="10.1042/bj20040261"><li><a href="http://dx.doi.org/10.1042/bj20040261" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+functional+expression+of+human+COQ2%2C+a+gene+encoding+a+polyprenyl+transferase+involved+in+the+synthesis+of+CoQ." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+functional+expression+of+human+COQ2%2C+a+gene+encoding+a+polyprenyl+transferase+involved+in+the+synthesis+of+CoQ.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0000766-Okada1" id="pone.0000766-Okada1"></a>Okada K, Ohara K, Yazaki K, Nozaki K, Uchida N, et al.  (2004) The AtPPT1 gene encoding 4-hydroxybenzoate polyprenyl diphosphate transferase in ubiquinone biosynthesis is required for embryo development in Arabidopsis thaliana. Plant Mol Biol  55: 567–577.  <ul class="find" data-citedArticleID="980072" data-doi="10.1007/s11103-004-1298-4"><li><a href="http://dx.doi.org/10.1007/s11103-004-1298-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+AtPPT1+gene+encoding+4-hydroxybenzoate+polyprenyl+diphosphate+transferase+in+ubiquinone+biosynthesis+is+required+for+embryo+development+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+AtPPT1+gene+encoding+4-hydroxybenzoate+polyprenyl+diphosphate+transferase+in+ubiquinone+biosynthesis+is+required+for+embryo+development+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0000766-Willingham1" id="pone.0000766-Willingham1"></a>Willingham S, Outeiro TF, DeVit MJ, Lindquist SL, Muchowski PJ (2003) Yeast genes that enhance the toxicity of a mutant huntingtin fragment or alpha-synuclein. Science  302: 1769–1772.  <ul class="find" data-citedArticleID="980144" data-doi="10.1126/science.1090389"><li><a href="http://dx.doi.org/10.1126/science.1090389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yeast+genes+that+enhance+the+toxicity+of+a+mutant+huntingtin+fragment+or+alpha-synuclein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yeast+genes+that+enhance+the+toxicity+of+a+mutant+huntingtin+fragment+or+alpha-synuclein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0000766-Xu1" id="pone.0000766-Xu1"></a>Xu GF, Lin B, Tanaka K, Dunn D, Wood D, et al.  (1990) The catalytic domain of the neurofibromatosis type 1 gene product stimulates ras GTPase and complements ira mutants of S. cerevisiae. Cell  63: 835–841.  <ul class="find" data-citedArticleID="980146" data-doi="10.1016/0092-8674(90)90149-9"><li><a href="http://dx.doi.org/10.1016/0092-8674(90)90149-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+catalytic+domain+of+the+neurofibromatosis+type+1+gene+product+stimulates+ras+GTPase+and+complements+ira+mutants+of+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+catalytic+domain+of+the+neurofibromatosis+type+1+gene+product+stimulates+ras+GTPase+and+complements+ira+mutants+of+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0000766-Mamiya1" id="pone.0000766-Mamiya1"></a>Mamiya N, Worman HJ (1999) Hepatitis C virus core protein binds to a DEAD box RNA helicase. J Biol Chem  274: 15751–15756.  <ul class="find" data-citedArticleID="980050" data-doi="10.1074/jbc.274.22.15751"><li><a href="http://dx.doi.org/10.1074/jbc.274.22.15751" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hepatitis+C+virus+core+protein+binds+to+a+DEAD+box+RNA+helicase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hepatitis+C+virus+core+protein+binds+to+a+DEAD+box+RNA+helicase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0000766-Johnstone1" id="pone.0000766-Johnstone1"></a>Johnstone O, Deuring R, Bock R, Linder P, Fuller MT, et al.  (2005) Belle is a Drosophila DEAD-box protein required for viability and in the germ line. Dev Biol  277: 92–101.  <ul class="find" data-citedArticleID="980002" data-doi="10.1016/j.ydbio.2004.09.009"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2004.09.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Belle+is+a+Drosophila+DEAD-box+protein+required+for+viability+and+in+the+germ+line." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Belle+is+a+Drosophila+DEAD-box+protein+required+for+viability+and+in+the+germ+line.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0000766-Vonarx1" id="pone.0000766-Vonarx1"></a>Vonarx EJ, Howlett NG, Schiestl RH, Kunz BA (2002) Detection of Arabidopsis thaliana AtRAD1 cDNA variants and assessment of function by expression in a yeast rad1 mutant. Gene  296: 1–9.  <ul class="find" data-citedArticleID="980138" data-doi="10.1016/s0378-1119(02)00869-7"><li><a href="http://dx.doi.org/10.1016/s0378-1119(02)00869-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+Arabidopsis+thaliana+AtRAD1+cDNA+variants+and+assessment+of+function+by+expression+in+a+yeast+rad1+mutant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+Arabidopsis+thaliana+AtRAD1+cDNA+variants+and+assessment+of+function+by+expression+in+a+yeast+rad1+mutant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">99.
              </span><a name="pone.0000766-Shaag1" id="pone.0000766-Shaag1"></a>Shaag A, Walsh T, Renbaum P, Kirchhoff T, Nafa K, et al.  (2005) Functional and genomic approaches reveal an ancient CHEK2 allele associated with breast cancer in the Ashkenazi Jewish population. Hum Mol Genet  14: 555–563.  <ul class="find" data-citedArticleID="980114" data-doi="10.1093/hmg/ddi052"><li><a href="http://dx.doi.org/10.1093/hmg/ddi052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+and+genomic+approaches+reveal+an+ancient+CHEK2+allele+associated+with+breast+cancer+in+the+Ashkenazi+Jewish+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+and+genomic+approaches+reveal+an+ancient+CHEK2+allele+associated+with+breast+cancer+in+the+Ashkenazi+Jewish+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">100.
              </span><a name="pone.0000766-Takeuchi1" id="pone.0000766-Takeuchi1"></a>Takeuchi M, Tada M, Saito C, Yashiroda H, Nakano A (1998) Isolation of a tobacco cDNA encoding Sar1 GTPase and analysis of its dominant mutations in vesicular traffic using a yeast complementation system. Plant Cell Physiol  39: 590–599.  <ul class="find" data-citedArticleID="980124" data-doi="10.1093/oxfordjournals.pcp.a029409"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.pcp.a029409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+a+tobacco+cDNA+encoding+Sar1+GTPase+and+analysis+of+its+dominant+mutations+in+vesicular+traffic+using+a+yeast+complementation+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+a+tobacco+cDNA+encoding+Sar1+GTPase+and+analysis+of+its+dominant+mutations+in+vesicular+traffic+using+a+yeast+complementation+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">101.
              </span><a name="pone.0000766-Tomita1" id="pone.0000766-Tomita1"></a>Tomita S, Inoue N, Maeda Y, Ohishi K, Takeda J, et al.  (1998) A homologue of Saccharomyces cerevisiae Dpm1p is not sufficient for synthesis of dolichol-phosphate-mannose in mammalian cells. J Biol Chem  273: 9249–9254.  <ul class="find" data-citedArticleID="980130" data-doi="10.1074/jbc.273.15.9249"><li><a href="http://dx.doi.org/10.1074/jbc.273.15.9249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+homologue+of+Saccharomyces+cerevisiae+Dpm1p+is+not+sufficient+for+synthesis+of+dolichol-phosphate-mannose+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+homologue+of+Saccharomyces+cerevisiae+Dpm1p+is+not+sufficient+for+synthesis+of+dolichol-phosphate-mannose+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">102.
              </span><a name="pone.0000766-Lai1" id="pone.0000766-Lai1"></a>Lai K, Elsas LJ (2000) Overexpression of human UDP-glucose pyrophosphorylase rescues galactose-1-phosphate uridyltransferase-deficient yeast. Biochem Biophys Res Commun  271: 392–400.  <ul class="find" data-citedArticleID="980030" data-doi="10.1006/bbrc.2000.2629"><li><a href="http://dx.doi.org/10.1006/bbrc.2000.2629" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+human+UDP-glucose+pyrophosphorylase+rescues+galactose-1-phosphate+uridyltransferase-deficient+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+human+UDP-glucose+pyrophosphorylase+rescues+galactose-1-phosphate+uridyltransferase-deficient+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">103.
              </span><a name="pone.0000766-Brzeski1" id="pone.0000766-Brzeski1"></a>Brzeski J, Podstolski W, Olczak K, Jerzmanowski A (1999) Identification and analysis of the Arabidopsis thaliana BSH gene, a member of the SNF5 gene family. Nucleic Acids Res  27: 2393–2399.  <ul class="find" data-citedArticleID="979938" data-doi="10.1093/nar/27.11.2393"><li><a href="http://dx.doi.org/10.1093/nar/27.11.2393" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+analysis+of+the+Arabidopsis+thaliana+BSH+gene%2C+a+member+of+the+SNF5+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+analysis+of+the+Arabidopsis+thaliana+BSH+gene%2C+a+member+of+the+SNF5+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">104.
              </span><a name="pone.0000766-Song1" id="pone.0000766-Song1"></a>Song WY, Martinoia E, Lee J, Kim D, Kim DY, et al.  (2004) A novel family of cys-rich membrane proteins mediates cadmium resistance in Arabidopsis. Plant Physiol  135: 1027–1039.  <ul class="find" data-citedArticleID="980118" data-doi="10.1104/pp.103.037739"><li><a href="http://dx.doi.org/10.1104/pp.103.037739" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+family+of+cys-rich+membrane+proteins+mediates+cadmium+resistance+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+family+of+cys-rich+membrane+proteins+mediates+cadmium+resistance+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">105.
              </span><a name="pone.0000766-Tommasini1" id="pone.0000766-Tommasini1"></a>Tommasini R, Vogt E, Fromenteau M, Hortensteiner S, Matile P, et al.  (1998) An ABC-transporter of Arabidopsis thaliana has both glutathione-conjugate and chlorophyll catabolite transport activity. Plant J  13: 773–780.  <ul class="find" data-citedArticleID="980132" data-doi="10.1046/j.1365-313x.1998.00076.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1998.00076.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+ABC-transporter+of+Arabidopsis+thaliana+has+both+glutathione-conjugate+and+chlorophyll+catabolite+transport+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+ABC-transporter+of+Arabidopsis+thaliana+has+both+glutathione-conjugate+and+chlorophyll+catabolite+transport+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">106.
              </span><a name="pone.0000766-Liang1" id="pone.0000766-Liang1"></a>Liang F, Cunningham KW, Harper JF, Sze H (1997) ECA1 complements yeast mutants defective in Ca2+ pumps and encodes an endoplasmic reticulum-type Ca2+-ATPase in Arabidopsis thaliana. Proc Natl Acad Sci U S A  94: 8579–8584.  <ul class="find" data-citedArticleID="980040" data-doi="10.1073/pnas.94.16.8579"><li><a href="http://dx.doi.org/10.1073/pnas.94.16.8579" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ECA1+complements+yeast+mutants+defective+in+Ca2%2B+pumps+and+encodes+an+endoplasmic+reticulum-type+Ca2%2B-ATPase+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ECA1+complements+yeast+mutants+defective+in+Ca2%2B+pumps+and+encodes+an+endoplasmic+reticulum-type+Ca2%2B-ATPase+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">107.
              </span><a name="pone.0000766-Covic1" id="pone.0000766-Covic1"></a>Covic L, Lew RR (1996) Arabidopsis thaliana cDNA isolated by functional complementation shows homology to serine/threonine protein kinases. Biochim Biophys Acta  1305: 125–129.  <ul class="find" data-citedArticleID="979958" data-doi="10.1016/0167-4781(95)00233-2"><li><a href="http://dx.doi.org/10.1016/0167-4781(95)00233-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+thaliana+cDNA+isolated+by+functional+complementation+shows+homology+to+serine%2Fthreonine+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+thaliana+cDNA+isolated+by+functional+complementation+shows+homology+to+serine%2Fthreonine+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">108.
              </span><a name="pone.0000766-Schmidt2" id="pone.0000766-Schmidt2"></a>Schmidt PJ, Ramos-Gomez M, Culotta VC (1999) A gain of superoxide dismutase (SOD) activity obtained with CCS, the copper metallochaperone for SOD1. J Biol Chem  274: 36952–36956.  <ul class="find" data-citedArticleID="980110" data-doi="10.1074/jbc.274.52.36952"><li><a href="http://dx.doi.org/10.1074/jbc.274.52.36952" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gain+of+superoxide+dismutase+%28SOD%29+activity+obtained+with+CCS%2C+the+copper+metallochaperone+for+SOD1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gain+of+superoxide+dismutase+%28SOD%29+activity+obtained+with+CCS%2C+the+copper+metallochaperone+for+SOD1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">109.
              </span><a name="pone.0000766-Kataoka1" id="pone.0000766-Kataoka1"></a>Kataoka T, Powers S, Cameron S, Fasano O, Goldfarb M, et al.  (1985) Functional homology of mammalian and yeast RAS genes. Cell  40: 19–26.  <ul class="find" data-citedArticleID="980006" data-doi="10.1016/0092-8674(85)90304-6"><li><a href="http://dx.doi.org/10.1016/0092-8674(85)90304-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+homology+of+mammalian+and+yeast+RAS+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+homology+of+mammalian+and+yeast+RAS+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">110.
              </span><a name="pone.0000766-Catoni1" id="pone.0000766-Catoni1"></a>Catoni E, Desimone M, Hilpert M, Wipf D, Kunze R, et al.  (2003) Expression pattern of a nuclear encoded mitochondrial arginine-ornithine translocator gene from Arabidopsis. BMC Plant Biol  3: 1.  <ul class="find" data-citedArticleID="979942"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+pattern+of+a+nuclear+encoded+mitochondrial+arginine-ornithine+translocator+gene+from+Arabidopsis.&amp;auth=&amp;atitle=Expression+pattern+of+a+nuclear+encoded+mitochondrial+arginine-ornithine+translocator+gene+from+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+pattern+of+a+nuclear+encoded+mitochondrial+arginine-ornithine+translocator+gene+from+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+pattern+of+a+nuclear+encoded+mitochondrial+arginine-ornithine+translocator+gene+from+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">111.
              </span><a name="pone.0000766-Krumpelman1" id="pone.0000766-Krumpelman1"></a>Krumpelman PM, Freyermuth SK, Cannon JF, Fink GR, Polacco JC (1995) Nucleotide sequence of Arabidopsis thaliana arginase expressed in yeast. Plant Physiol  107: 1479–1480.  <ul class="find" data-citedArticleID="980026" data-doi="10.1104/pp.107.4.1479"><li><a href="http://dx.doi.org/10.1104/pp.107.4.1479" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleotide+sequence+of+Arabidopsis+thaliana+arginase+expressed+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleotide+sequence+of+Arabidopsis+thaliana+arginase+expressed+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">112.
              </span><a name="pone.0000766-vanWilpe1" id="pone.0000766-vanWilpe1"></a>van Wilpe S, Boumans H, Lobo-Hajdu G, Grivell LA, Berden JA (1999) Functional complementation analysis of yeast bc1 mutants. A study of the mitochondrial import of heterologous and hybrid proteins. Eur J Biochem  264: 825–832.  <ul class="find" data-citedArticleID="980150" data-doi="10.1046/j.1432-1327.1999.00673.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1327.1999.00673.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+complementation+analysis+of+yeast+bc1+mutants.+A+study+of+the+mitochondrial+import+of+heterologous+and+hybrid+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+complementation+analysis+of+yeast+bc1+mutants.+A+study+of+the+mitochondrial+import+of+heterologous+and+hybrid+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">113.
              </span><a name="pone.0000766-Schaffar1" id="pone.0000766-Schaffar1"></a>Schaffar G, Breuer P, Boteva R, Behrends C, Tzvetkov N, et al.  (2004) Cellular toxicity of polyglutamine expansion proteins: mechanism of transcription factor deactivation Functional complementation analysis of yeast bc1 mutants. A study of the mitochondrial import of heterologous and hybrid proteins. Mol Cell  15: 95–105.  <ul class="find" data-citedArticleID="980104" data-doi="10.1016/j.molcel.2004.06.029"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.06.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+toxicity+of+polyglutamine+expansion+proteins%3A+mechanism+of+transcription+factor+deactivation+Functional+complementation+analysis+of+yeast+bc1+mutants.+A+study+of+the+mitochondrial+import+of+heterologous+and+hybrid+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+toxicity+of+polyglutamine+expansion+proteins%3A+mechanism+of+transcription+factor+deactivation+Functional+complementation+analysis+of+yeast+bc1+mutants.+A+study+of+the+mitochondrial+import+of+heterologous+and+hybrid+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">114.
              </span><a name="pone.0000766-Wagner1" id="pone.0000766-Wagner1"></a>Wagner N, Weber D, Seitz S, Krohne G (2004) The lamin B receptor of Drosophila melanogaster. J Cell Sci  117: 2015–2028.  <ul class="find" data-citedArticleID="980140" data-doi="10.1242/jcs.01052"><li><a href="http://dx.doi.org/10.1242/jcs.01052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+lamin+B+receptor+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+lamin+B+receptor+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">115.
              </span><a name="pone.0000766-Colussi1" id="pone.0000766-Colussi1"></a>Colussi PA, Taron CH, Mack JC, Orlean P (1997) Human and Saccharomyces cerevisiae dolichol phosphate mannose synthases represent two classes of the enzyme, but both function in Schizosaccharomyces pombe. Proc Natl Acad Sci U S A  94: 7873–7878.  <ul class="find" data-citedArticleID="979956" data-doi="10.1073/pnas.94.15.7873"><li><a href="http://dx.doi.org/10.1073/pnas.94.15.7873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+and+Saccharomyces+cerevisiae+dolichol+phosphate+mannose+synthases+represent+two+classes+of+the+enzyme%2C+but+both+function+in+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+and+Saccharomyces+cerevisiae+dolichol+phosphate+mannose+synthases+represent+two+classes+of+the+enzyme%2C+but+both+function+in+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.XML" value="216465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.PDF" value="626023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t001.PNG_L" value="206365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t001.PNG_M" value="121222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t001.PNG_S" value="13104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t001.TIF" value="957062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t001.PNG_I" value="37225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g001.PNG_L" value="64113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g001.PNG_M" value="21330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g001.PNG_S" value="5067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g001.TIF" value="200966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g001.PNG_I" value="7717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t002.PNG_L" value="87735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t002.PNG_M" value="48271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t002.PNG_S" value="8223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t002.TIF" value="409984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t002.PNG_I" value="14282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g002.PNG_L" value="618367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g002.PNG_M" value="202885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g002.PNG_S" value="15845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g002.TIF" value="1504184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g002.PNG_I" value="101226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t003.PNG_L" value="92611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t003.PNG_M" value="50318"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t003.PNG_S" value="8534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t003.TIF" value="414866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t003.PNG_I" value="15004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t004.PNG_L" value="76339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t004.PNG_M" value="98911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t004.PNG_S" value="14162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t004.TIF" value="315406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t004.PNG_I" value="32714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t005.PNG_L" value="975082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t005.PNG_M" value="137313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t005.PNG_S" value="22112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t005.TIF" value="3924186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t005.PNG_I" value="169822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g003.PNG_L" value="193680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g003.PNG_M" value="183024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g003.PNG_S" value="11244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g003.TIF" value="1038202"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g003.PNG_I" value="65106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g004.PNG_L" value="194271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g004.PNG_M" value="150523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g004.PNG_S" value="10397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g004.TIF" value="1106862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g004.PNG_I" value="53336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g005.PNG_L" value="204725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g005.PNG_M" value="131720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g005.PNG_S" value="10951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g005.TIF" value="915644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.g005.PNG_I" value="43330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t006.PNG_L" value="233918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t006.PNG_M" value="125995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t006.PNG_S" value="15978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t006.TIF" value="1198096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000766.t006.PNG_I" value="39513"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000766&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000766" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000766&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000766&volume=&issue=&title=The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists&author_name=Sven%20Heinicke%2C%20Michael%20S.%20Livstone%2C%20Charles%20Lu%2C%20Rose%20Oughtred%2C%20Fan%20Kang%2C%20Samuel%20V.%20Angiuoli%2C%20Owen%20White%2C%20David%20Botstein%2C%20Kara%20Dolinski&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000766" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766&amp;t=The%20Princeton%20Protein%20Orthology%20Database%20(P-POD)%3A%20A%20Comparative%20Genomics%20Analysis%20Tool%20for%20Biologists" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766&title=The%20Princeton%20Protein%20Orthology%20Database%20(P-POD)%3A%20A%20Comparative%20Genomics%20Analysis%20Tool%20for%20Biologists&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766&amp;title=The%20Princeton%20Protein%20Orthology%20Database%20(P-POD)%3A%20A%20Comparative%20Genomics%20Analysis%20Tool%20for%20Biologists" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000766&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Princeton Protein Orthology Database (P-POD): A Comparative Genomics Analysis Tool for Biologists';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000766';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Princeton%20Protein%20Orthology%20Database%20(P-POD)%3A%20A%20Comparative%20Genomics%20Analysis%20Tool%20for%20Biologists http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000766" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000766" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="24370"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24370"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24370"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+families%22" title="Search for articles in the subject area:'Human families'"><div class="flagText">Human families</div></a>
              <div data-categoryid="18723" data-articleid="24370"
                   data-categoryname="Human families"
                   class="flagImage" title="Flag 'Human families' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Model+organisms%22" title="Search for articles in the subject area:'Model organisms'"><div class="flagText">Model organisms</div></a>
              <div data-categoryid="35381" data-articleid="24370"
                   data-categoryname="Model organisms"
                   class="flagImage" title="Flag 'Model organisms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+polymerase%22" title="Search for articles in the subject area:'RNA polymerase'"><div class="flagText">RNA polymerase</div></a>
              <div data-categoryid="40207" data-articleid="24370"
                   data-categoryname="RNA polymerase"
                   class="flagImage" title="Flag 'RNA polymerase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="24370"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="24370"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8538'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=107'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2621&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44487"></a><br>Posted by jstajich</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
