

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000577"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000577" />

    <meta name="citation_title" content="A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis"/>
    <meta itemprop="name" content="A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis"/>

      <meta name="citation_author" content="Tal Murthy"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Andreas Rolfs"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Yanhui Hu"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Zhenwei Shi"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jacob Raphael"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Donna Moreira"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Fontina Kelley"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Seamus McCarron"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Daniel Jepson"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Elena Taycher"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Dongmei Zuo"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Stephanie E. Mohr"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="DF/HCC DNA Resource Core, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Mauricio Fernandez"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Leonardo Brizuela"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Joshua LaBaer"/>
            <meta name="citation_author_institution" content="Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="DF/HCC DNA Resource Core, Harvard Medical School, Cambridge, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/6/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000577.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e577"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome-scale cloning and expression of individual open reading frames using topoisomerase I-mediated ligation.; citation_author=JA Heyman; citation_author=J Cornthwaite; citation_author=L Foncerrada; citation_author=JR Gilmore; citation_author=E Gontang; citation_journal_title=Genome Res; citation_volume=9; citation_number=1; citation_pages=383-392; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The Pseudomonas aeruginosa PA01 gene collection.; citation_author=J LaBaer; citation_author=Q Qiu; citation_author=A Anumanthan; citation_author=W Mar; citation_author=D Zuo; citation_journal_title=Genome Res; citation_volume=14; citation_number=2; citation_pages=2190-2200; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human ORFeome version 1.1: A platform for reverse proteomics.; citation_author=JF Rual; citation_author=T Hirozane-Kishikawa; citation_author=T Hao; citation_author=N Bertin; citation_author=S Li; citation_journal_title=Genome Res; citation_volume=14; citation_number=3; citation_pages=2128-2135; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Building a human kinase gene repository: bioinformatics, molecular cloning, and functional validation.; citation_author=J Park; citation_author=Y Hu; citation_author=TV Murthy; citation_author=F Vannberg; citation_author=B Shen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=4; citation_pages=8114-8119; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Generation of gene targeting constructs for Plasmodium berghei by a PCR-based method amenable to high-throughput applications.; citation_author=A Ecker; citation_author=R Moon; citation_author=RE Sinden; citation_author=O Billker; citation_journal_title=Mol Biochem Parasitol; citation_volume=145; citation_number=5; citation_pages=265-268; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe.; citation_author=A Matsuyama; citation_author=R Arai; citation_author=Y Yashiroda; citation_author=A Shirai; citation_author=A Kamata; citation_journal_title=Nat Biotechnol; citation_volume=24; citation_number=6; citation_pages=841-847; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Approaching a complete repository of sequence-verified protein-encoding clones for Saccharomyces cerevisiae.; citation_author=Y Hu; citation_author=A Rolfs; citation_author=B Bhullar; citation_author=TV Murthy; citation_author=C Zhu; citation_journal_title=Genome Res; citation_volume=17; citation_number=7; citation_pages=536-543; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=High throughput methods for gene cloning and expression.; citation_author=L Dieckman; citation_author=M Gu; citation_author=L Stols; citation_author=M Donnely; citation_author=FR Collart; citation_journal_title=Protein Expr Purif; citation_volume=25; citation_number=8; citation_pages=1-7; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=C. elegans ORFeome version 1.1: experimental verification of the genome annotation and resource for proteome-scale protein expression.; citation_author=J Reboul; citation_author=P Vaglio; citation_author=JF Rual; citation_author=P Lamesch; citation_author=M Martinez; citation_journal_title=Nat Genet; citation_volume=34; citation_number=9; citation_pages=35-41; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-throughput cloning of Campylobacter jejuni ORfs by in vivo recombination in Escherichia coli.; citation_author=JR Parrish; citation_author=T Limjindaporn; citation_author=JA Hines; citation_author=J Liu; citation_author=G Liu; citation_journal_title=J Proteome Res; citation_volume=3; citation_number=10; citation_pages=582-586; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=From genome to proteome: developing expression clone resources for the human genome.; citation_author=G Temple; citation_author=P Lamesch; citation_author=S Milstein; citation_author=DE Hill; citation_author=L Wagner; citation_journal_title=Hum Mol Genet 15 Spec No; citation_volume=1; citation_number=11; citation_pages=R31-43; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional proteomics approach to investigate the biological activities of cDNAs implicated in breast cancer.; citation_author=AE Witt; citation_author=LM Hines; citation_author=NL Collins; citation_author=Y Hu; citation_author=RN Gunawardane; citation_journal_title=J Proteome Res; citation_volume=5; citation_number=12; citation_pages=599-610; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High-throughput screening for soluble recombinant expressed kinases in Escherichia coli and insect cells.; citation_author=SP Chambers; citation_author=DA Austen; citation_author=JR Fulghum; citation_author=WM Kim; citation_journal_title=Protein Expr Purif; citation_volume=36; citation_number=13; citation_pages=40-47; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bacterial cell-free system for high-throughput protein expression and a comparative analysis of Escherichia coli cell-free and whole cell expression systems.; citation_author=TV Murthy; citation_author=W Wu; citation_author=QQ Qiu; citation_author=Z Shi; citation_author=J LaBaer; citation_journal_title=Protein Expr Purif; citation_volume=36; citation_number=14; citation_pages=217-225; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Proteome-scale purification of human proteins from bacteria.; citation_author=P Braun; citation_author=Y Hu; citation_author=B Shen; citation_author=A Halleck; citation_author=M Koundinya; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=15; citation_pages=2654-2659; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High throughput protein production for functional proteomics.; citation_author=P Braun; citation_author=J LaBaer; citation_journal_title=Trends Biotechnol; citation_volume=21; citation_number=16; citation_pages=383-388; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Intracellular survival mechanisms of Francisella tularensis, a stealth pathogen.; citation_author=A Sjostedt; citation_journal_title=Microbes Infect; citation_volume=8; citation_number=17; citation_pages=561-567; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Growth of Francisella tularensis LVS in macrophages: the acidic intracellular compartment provides essential iron required for growth.; citation_author=AH Fortier; citation_author=DA Leiby; citation_author=RB Narayanan; citation_author=E Asafoadjei; citation_author=RM Crawford; citation_journal_title=Infect Immun; citation_volume=63; citation_number=18; citation_pages=1478-1483; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Tularemia as a biological weapon: medical and public health management.; citation_author=DT Dennis; citation_author=TV Inglesby; citation_author=DA Henderson; citation_author=JG Bartlett; citation_author=MS Ascher; citation_journal_title=Jama; citation_volume=285; citation_number=19; citation_pages=2763-2773; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of Francisella tularensis, the causative agent of tularemia.; citation_author=P Larsson; citation_author=PC Oyston; citation_author=P Chain; citation_author=MC Chu; citation_author=M Duffield; citation_journal_title=Nat Genet; citation_volume=37; citation_number=20; citation_pages=153-159; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Francisella tularensis: Taxonomy, Genetics, and Immunopathogenesis of a Potential Agent of Biowarfare.; citation_author=MK McLendon; citation_author=MA Apicella; citation_author=LA Allen; citation_journal_title=Annu Rev Microbiol; citation_volume=60; citation_number=21; citation_pages=167-185; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Post-genomic vaccine development.; citation_author=D Serruto; citation_author=R Rappuoli; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=22; citation_pages=2985-2992; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis"/>
    <meta name="twitter:description" content="The rapid development of new technologies for the high throughput (HT) study of proteins has increased the demand for comprehensive plasmid clone resources that support protein expression. These clones must be full-length, sequence-verified and in a flexible format. The generation of these resources requires automated pipelines supported by software management systems. Although the availability of clone resources is growing, current collections are either not complete or not fully sequence-verified. We report an automated pipeline, supported by several software applications that enabled the construction of the first comprehensive sequence-verified plasmid clone resource for more than 96% of protein coding sequences of the genome of F. tularensis, a highly virulent human pathogen and the causative agent of tularemia. This clone resource was applied to a HT protein purification pipeline successfully producing recombinant proteins for 72% of the genes. These methods and resources represent significant technological steps towards exploiting the genomic information of F. tularensis in discovery applications."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000577.g002"/>

  <meta property="og:title" content="A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1293'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3529'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6927&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000577">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000577" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000577&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for <i>Francisella tularensis</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tal Murthy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Thermo Fisher Scientific, Hudson, New Hampshire, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andreas Rolfs, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yanhui Hu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhenwei Shi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jacob Raphael, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Donna Moreira, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fontina Kelley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Seamus McCarron, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel Jepson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elena Taycher, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dongmei Zuo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stephanie E. Mohr, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America, 
                  DF/HCC DNA Resource Core, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mauricio Fernandez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leonardo Brizuela, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Agilent Technologies, Cambridge, Massachusetts, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joshua LaBaer
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:josh@hms.harvard.edu">josh@hms.harvard.edu</a></p>

                <p>Affiliations:
                  Harvard Institute of Proteomics, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Cambridge, Massachusetts, United States of America, 
                  DF/HCC DNA Resource Core, Harvard Medical School, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 27, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000577</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000577-g001" data-doi="info:doi/10.1371/journal.pone.0000577" data-uri="info:doi/10.1371/journal.pone.0000577.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000577-t001" data-doi="info:doi/10.1371/journal.pone.0000577" data-uri="info:doi/10.1371/journal.pone.0000577.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000577-g002" data-doi="info:doi/10.1371/journal.pone.0000577" data-uri="info:doi/10.1371/journal.pone.0000577.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000577">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000577" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The rapid development of new technologies for the high throughput (HT) study of proteins has increased the demand for comprehensive plasmid clone resources that support protein expression. These clones must be full-length, sequence-verified and in a flexible format. The generation of these resources requires automated pipelines supported by software management systems. Although the availability of clone resources is growing, current collections are either not complete or not fully sequence-verified. We report an automated pipeline, supported by several software applications that enabled the construction of the first comprehensive sequence-verified plasmid clone resource for more than 96% of protein coding sequences of the genome of <em>F. tularensis</em>, a highly virulent human pathogen and the causative agent of tularemia. This clone resource was applied to a HT protein purification pipeline successfully producing recombinant proteins for 72% of the genes. These methods and resources represent significant technological steps towards exploiting the genomic information of <em>F. tularensis</em> in discovery applications.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Murthy T, Rolfs A, Hu Y, Shi Z, Raphael J, et al.  (2007) A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for <em>Francisella tularensis</em>. PLoS ONE 2(6):
          e577.
            doi:10.1371/journal.pone.0000577</p><p><strong>Academic Editor: </strong>Joseph Petrosino, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> April 24, 2007; <strong>Accepted:</strong> May 30, 2007; <strong>Published:</strong> June 27, 2007</p><p><strong>Copyright:</strong> © 2007 Murthy et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported by the NIAID/NIH grant for the New England Regional Center for Excellence #AI057159.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Over the past several years, complete or nearly complete sets of clones representing the open reading frames (ORFs) of various species have been constructed and made available <a href="#pone.0000577-Heyman1">[1]</a>–<a href="#pone.0000577-LaBaer1"></a>,<a href="#pone.0000577-Rual1"></a>,<a href="#pone.0000577-Park1"></a>,<a href="#pone.0000577-Ecker1"></a>,<a href="#pone.0000577-Matsuyama1"></a>,<a href="#pone.0000577-Hu1">[7]</a>. These collections often employ recombinational cloning vectors, enabling the transfer of the ORFs into virtually any protein expression vector in a simple conservative transfer reaction. Once transferred, these expression clones can be employed in a wide variety of assays, including high-throughput (HT) cell-based and proteomic discovery assays <a href="#pone.0000577-LaBaer1">[2]</a>, <a href="#pone.0000577-Park1">[4]</a>, <a href="#pone.0000577-Dieckman1">[8]</a>–<a href="#pone.0000577-Reboul1"></a>,<a href="#pone.0000577-Parrish1"></a>,<a href="#pone.0000577-Temple1"></a>,<a href="#pone.0000577-Witt1">[12]</a> .</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Clone collections have been used successfully to produce proteins using <em>in vitro</em>, bacterial or insect cell expression systems <a href="#pone.0000577-Chambers1">[13]</a>, <a href="#pone.0000577-Murthy1">[14]</a>. Although several heterologous protein expression systems are capable of HT protein expression, simplicity and ease of handling have made the bacterial systems the best starting point to express large numbers of recombinant proteins <a href="#pone.0000577-Braun1">[15]</a>, <a href="#pone.0000577-Braun2">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Among the most important properties that transferable clone collections should embody include comprehensive genomic representation of the ORFs and full length sequence validation, a combination of features that has thus far eluded the collections available today. In part, this is because sequence validation of clones is a tedious process that cannot be easily achieved without a well developed automated pipeline. Nevertheless, the major use of these collections will be to study protein function, emphasizing the critical importance of full length sequence validation.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>There is a pressing need to generate clone and protein resources for highly infectious organisms that could be used in bioterrorism. One such organism is <em>Francisella tularensis</em>, a highly virulent, gram-negative, facultative intracellular pathogen that is the causative agent of tularemia. <em>F. tularensis</em> is capable of infecting many mammalian species and cell types, and has been isolated from more than 250 animal species, including mammals, arthropods and protozoa <a href="#pone.0000577-Sjostedt1">[17]</a>. In mammalian hosts, <em>F. tularensis</em> thrives in the intracellular environment of macrophages <a href="#pone.0000577-Fortier1">[18]</a>. The virulent subspecies <em>F. tularensis tularensis</em> is found in North America. As few as ten cells of this subspecies are sufficient to cause an infection in humans and 30% to 60% of untreated infections are fatal <a href="#pone.0000577-Dennis1">[19]</a>. The high infection capability and transmission of the organism by aerosols pose a significant threat, leading the U.S. Center for Disease Control (CDC) to consider <em>F. tularensis</em> a category A biodefense pathogen <a href="#pone.0000577-Dennis1">[19]</a>. Recent studies on the pathogen have focused on genomic analysis, identification of antigen targets for vaccine development, and on understanding the mechanisms of infection <a href="#pone.0000577-Larsson1">[20]</a>, <a href="#pone.0000577-McLendon1">[21]</a>. The genome sequence of <em>F. tularensis</em> (subsp<em>. tularensis SCHU S4</em>) was published in 2005 <a href="#pone.0000577-Larsson1">[20]</a>. The organism has a genome of approximately 1.9 MB that is AT rich (33% GC content) and is predicted to encode 1,804 genes, of which 302 sequences are unique to <em>Francisella</em> <a href="#pone.0000577-Larsson1">[20]</a>. More than 10% of genes are predicted to be pseudogenes or gene fragments <a href="#pone.0000577-Larsson1">[20]</a>. The current annotation at NCBI includes 1,852 genes, of which 1,603 represent putative protein-coding sequences.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The study of <em>F. tularensis</em> pathogenesis has been hindered by the lack of reliable genetic methods <a href="#pone.0000577-McLendon1">[21]</a>. The availability of clone and protein resources would enable functional proteomics studies directed at the detection, prevention and treatment of this disease agent. Comprehensive studies using recombinant proteins can be used both to determine which proteins stimulate cell-mediated and humoral immune responses in the blood of infected individuals and to test the proteins in the <em>reverse vaccinology</em> approach <a href="#pone.0000577-Serruto1">[22]</a>. Using HT pipelines described here, we constructed the first comprehensive sequence verified gene collection of <em>F. tularensis</em> (subsp<em>. tularensis SCHU S4</em>) and a corresponding <em>E. coli</em> expressed protein collection. The strategies adopted in generating these resources, as well as some of the challenges overcome in the completion of the collection, provide a guideline for HT gene cloning efforts.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Generation of the <em>F. tularensis</em> gene collection</h4>
<h5>Cloning Strategy.</h5><a id="article1.body1.sec2.sec1.sec1.p1" name="article1.body1.sec2.sec1.sec1.p1"></a><p>The assembly of this protein coding clone collection was initiated by acquiring annotated genome sequence information to predict the relevant ORFs and design the PCR amplification primers. We started with a genome annotation kindly provided by H. Tettelin at the Institute for Genomic Research (TIGR) that predicted 2,036 ORFs based on a draft genome sequence provided by H. Svenson and P. Larsson. However, during the course of this project, a revised annotation of the genome sequence was published <a href="#pone.0000577-Larsson1">[20]</a> that reduced the number of ORFs to 1804 with 1603 protein coding genes and adjusted many ORF boundaries from the earlier draft. The mapping and comparison of ORFs from preliminary and published genome annotations can be found in <a href="#pone.0000577.s001">Table S1</a>. Thus, this cloning project was accomplished in two phases corresponding to these two annotations. The annotated ORF information was parsed and imported into our Full-Length Expression Gene (FLEXGene) database <a href="#pone.0000577-LaBaer1">[2]</a>. Relevant features stored in our database for each ORF include CDS sequence; genome location; CDS length; GC content; NCBI protein GI number; and FTT number (an organism-specific identifier). Additionally, a new and unique tracking identifier was assigned to each ORF by FLEXGene.</p>
<a id="article1.body1.sec2.sec1.sec1.p2" name="article1.body1.sec2.sec1.sec1.p2"></a><p>The cloning work-flow, shown in <a href="#pone-0000577-g001">Figure 1</a>, distinguishes between two phases of clone production. All barcoded plates and individual samples were tracked by FLEXGene, which also stores associated results such as culture growth, colony counts and electrophoretic gel images. In addition, each clone is associated with its target (reference) sequence for comparison during sequence validation.</p>
<div class="figure" id="pone-0000577-g001"><div class="img"><a name="pone-0000577-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000577" data-uri="info:doi/10.1371/journal.pone.0000577.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000577.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000577.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic representation of the work flow used in genome cloning of <em>Francisella tularensis.</em></span></strong></p><a id="article1.body1.sec2.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.sec1.fig1.caption1.p1"></a><p>The entire process, from design of primers to production of clonal glycerol stocks, is shown. Most steps are common to both phases; steps specific to phase 2 are shown with a dashed line. The process began with design of primers for each ORF in the genome (Step 1A). The primers were used to amplify ORFs from genomic DNA (Step 1B). Subsequent amplification with universal primers (Step 1C) generated ORF sequences flanked by complete recombinational cloning sites for capture by BP (Step 3). For amplicons captured by In-Fusion in Phase 2, universal primed PCR was not necessary (Step 2B) as the capture reaction completes creation of the recombinational cloning sites. Successful PCR was monitored by agarose gel electrophoresis (Steps 2A and B). In Phase 1 all products were purified from preparative gels (Step 2B) and cloned into a recombinational cloning vector via the BP clonase reaction (Steps 3), whereas in Phase 2, the capture method depended on ORF size as indicated, with only diagnostic gels needed for short amplicons (Step 2A) and preparative gels needed when In-Fusion capture was performed. Competent bacteria were transformed with the reaction mix to yield colonies which were isolated robotically, cultured in liquid media and stored as 15% glycerol stocks (Step 4).</p>
<span>doi:10.1371/journal.pone.0000577.g001</span></div><a id="article1.body1.sec2.sec1.sec1.p3" name="article1.body1.sec2.sec1.sec1.p3"></a><p>In phase 1, all ORFs were amplified using gene specific primers with partial <em>att</em> sites to generate a primary PCR product, which was further amplified using universal primers to complete the <em>attB</em> recombinational cloning sites (<a href="#pone-0000577-g001">Figure 1</a>). The final PCR product was gel purified and subsequently cloned into an Entry vector, pDONR221 (Invitrogen), to generate a master clone. New primers were designed for all targeted ORFs in phase 2, which were channeled into two groups. For ORFs below 2,000 bp we employed the phase 1 strategy. For ORFs above 2,000 bp, we omitted the second PCR reaction and instead gel purified the amplicons with partial <em>att</em> sites for capture reactions using In-Fusion™ enzyme (Clontech) into a purified, linear pDONR221 derivative, pGW-NcoEco109. The full <em>attB</em> sites are regenerated as part of the In-Fusion™ capture step. Capture reactions were transformed into competent bacteria and either four colonies (phase 1) or one colony (phase 2) per gene were isolated robotically, stored as a 15% glycerol stock, and subjected to sequence verification.</p>

<h5>Sequencing Strategy.</h5><a id="article1.body1.sec2.sec1.sec2.p1" name="article1.body1.sec2.sec1.sec2.p1"></a><p>In all cases, sequencing began with universally primed end reads for one isolate per gene and proceeded to internal reads where indicated (i.e., incomplete coverage but end reads acceptable). In cases where the first isolate failed during sequence validation, additional isolates were tested either by selecting another isolate (phase 1) or by re-plating the stored transformation mix for new colonies (phase 2). In cases where these strategies failed, cloning was repeated <em>de novo</em>. In both phases, we used our Automated Clone Evaluation (ACE) software to assemble the end reads and determine if they were sufficient to obtain complete sequence coverage (coding plus linker sequences). If coverage was complete, the assembled sequence was compared with the reference sequence to determine if the clone were acceptable. Discrepancies that occur in regions of low sequence confidence (typically phred&lt;25) likely reflect sequencing errors, whereas discrepancies that occur in regions of high sequence confidence likely reflect true differences between the clone and its reference sequence. Clones were accepted if they had no high confidence discrepancies leading to protein truncations or frame shifts, no discrepancies in the critical linker regions, and no more than two amino acid differences with the reference polypeptide. If the coverage was not complete or if there were regions of unacceptably low confidence sequence, additional sequencing was performed. This process was repeated iteratively until all clones were either accepted or rejected.</p>

<h5>Cloning and Sequencing Results.</h5><a id="article1.body1.sec2.sec1.sec3.p1" name="article1.body1.sec2.sec1.sec3.p1"></a><p>At least one isolate was obtained for more than 99% of the targeted 2,036 ORFs in phase 1. Sequence analysis revealed that 1500 ORFs were acceptable (73.6%; <a href="#pone-0000577-t001">Table 1</a>) based on the draft annotation. Subsequent to this analysis, a revised genome sequence of <em>F. tularensis</em> (subsp<em>. tularensis SCHU S4</em>) was published and further annotated at NCBI to predict 1,603 protein-coding genes <a href="#pone.0000577-Larsson1">[20]</a>. A total of 1,104 ORFs were identical between the two annotations, of which we had successfully cloned 900. The other 499 ORFs were either new or adjusted ORFs, both warranting a new cloning attempt. Thus, the second phase focused on the 499 adjusted ORFs plus 204 ORFs that failed in the first phase (a total of 703 ORFs). Phase 2 was very successful, resulting in 696 acceptable clones for 634 ORFs (&gt;97% for phase 2; <a href="#pone-0000577-t001">Table 1</a>). Overall, a total of 1,534 acceptable protein-coding clones matching the current NCBI annotation were obtained (96% success rate relative to the updated annotation). A complete list of these clones including their GenBank Accession Numbers and sequences can be found in <a href="#pone.0000577.s002">Table S2</a> and at <a href="http://plasmid.hms.harvard.edu">http://plasmid.hms.harvard.edu</a>. Protein expression clones were generated via recombinational sub-cloning into the amino terminal hexa-histidine tag expression vector pDEST17 (Invitrogen), using similar automated pipelines, attaining a 97% first pass efficiency.</p>
<div class="figure" id="pone-0000577-t001"><div class="img"><a name="pone-0000577-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000577" data-uri="info:doi/10.1371/journal.pone.0000577.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000577.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000577.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the cloning process of two annotations of <em>F. tularensis</em></span></strong></p><span>doi:10.1371/journal.pone.0000577.t001</span></div>


<h4>Automated high-throughput production of <em>Francisella tularensis</em> proteins</h4>
<h5>Construction of an automated 96-well protein purification pipeline.</h5><a id="article1.body1.sec2.sec2.sec1.p1" name="article1.body1.sec2.sec2.sec1.p1"></a><p>The availability of isolated proteins for genes in <em>F. tularensis</em> enables proteome scale investigation into the role that each component of this microbe plays in cellular and humoral immunity. We established an automated workstation for HT production, isolation and analysis of microgram quantities of protein, sufficient for use in immunoassays (e.g., ELISPOT). Purification was based on immobilized metal affinity chromatography (IMAC) using magnetic beads. Microfluidic-based protein analysis of aliquots was performed to determine the relative purity and quantity of the each recombinant protein.</p>
<a id="article1.body1.sec2.sec2.sec1.p2" name="article1.body1.sec2.sec2.sec1.p2"></a><p>Automated analysis via Labchip90 provided a computerized, quantitative output of polypeptide size, concentration and purity for each protein. In order to separate information about the protein of interest from the other contaminating host cell proteins, we parsed the digital output files using in-house software that collected information about each protein peak into a tab-delimited format that could be uploaded into a database. This enabled queries to look for protein peaks of the expected molecular weight and, if found, to obtain key parameters (size, concentration and purity).</p>



<h4>Expression and affinity purification of 6xHIS tagged <em>F. tularensis</em> proteins</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>From 1961 sequence-verified full-length clones representing 96% of protein coding <em>F. tularensis</em> ORFs, we successfully expressed and purified 72% of the proteins. Protein isolation was considered successful when at least 120 ng of the putative recombinant protein was present and the apparent molecular weight was within +/− 40% of the theoretical molecular weight. Representative virtual gels of protein analysis generated from the pipeline are shown in <a href="#pone-0000577-g002">Figure 2</a> and detailed information about all successfully purified proteins can be found in <a href="#pone.0000577.s003">Table S3</a>. Amongst the successful proteins, the concentrations varied from under 200 to more than 9,000 ng/µl with an average of 1494 ng/µl (SDEV: 7425 ng/µl). There was no apparent effect of protein size on the purification success rate for this collection, but the overall success rate for proteins predicted to contain trans-membrane domains (<a href="http://www.cbs.dtu.dk/services/TMHMM/">http://www.cbs.dtu.dk/services/TMHMM/</a>) was only 57% compared with the overall success rate of 72%.</p>
<div class="figure" id="pone-0000577-g002"><div class="img"><a name="pone-0000577-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000577" data-uri="info:doi/10.1371/journal.pone.0000577.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000577.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000577.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000577.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000577.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000577.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Representative virtual protein analysis gel of 188 proteins produced via the high-throughput protein production pipeline.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The label NTFT02#### indicates the unique identifier for each ORF of <em>Francisella tularensis</em>. The expected molecular weights (based on predicted protein coding sequences of the ORFs) are shown below each lane. Black arrows (left side) indicate protein bands observed at approximately the expected molecular weight. MW, molecular weight; NS, no sample.</p>
<span>doi:10.1371/journal.pone.0000577.g002</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The genome sequence of <em>F. tularensis</em> was recently published and provides a starting point to explore new avenues of research, which should ultimately aid in the development of more effective vaccines and foster our understanding of tularemia. In this work, we present the production of a high quality clone collection for <em>F. tularensis</em> (subsp<em>. tularensis SCHU S4</em>) with general utility for a broad range of different protein-based assays. To accelerate progress towards that goal, we developed HT protein purification pipelines to produce proteins useful for immunological assays.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>One of the major bottlenecks we encountered during the cloning of the <em>F. tularensis</em> ORFeome was a significant change to the preliminary genome annotation. We began our work using the best available annotation and when the revised annotation of the genome was released, we decided to incorporate this updated information into our production pipelines. Nearly 45% of the ORFs from our initial target set had to be classified as incorrectly annotated ORFs, with the majority being partial ORFs. The new annotation had two distinct effects on our pipeline. First, as our clone and protein productions were carried out simultaneously, downstream pipeline steps were burdened with some partial-length forms of genes along with full-length forms. Second, as the target set was revised, completion of clone production and protein purification took longer than initially expected. Both problems could be tolerated when balanced by an ultimately more complete and accurate clone collection.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Other factors that affected cloning success included the source of amplification primers and the use of polymerases with different fidelities (see <a href="#pone-0000577-t001">Table 1</a>). Changing the source for gene-specific primers in phase 2 decreased significantly the fraction of errors in the linker region (reduced from 6% to 0.6%). In addition, using a polymerase with higher fidelity, combined with improved primer quality, led to a 6 fold decrease in the overall rate of mutation from 1/608 to 1/3,939 base pairs, and decreased the number of attempted clones per final clone from 1.9 to 1.6. The second phase did require an increased number of sequence reads per clone; although it should be noted that this phase had an increased number of long or challenging clones (all ORFs that had failed the first phase were included again in the second phase).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>For protein production, we relied on a widely used heterologous bacterial expression system, optimized experimental procedures, and developed a completely automated platform. The simplicity of the bacterial system facilitated automation in a way that may not have readily been possible using other protein production systems. Moreover, we were able to use the bacterial system to produce proteins of the required concentration and purity to perform immuno assays for over 72% of the proteins in the <em>F. tularensis</em> proteome. Failure in protein production cannot be attributed to nonsense or other mutations in the clone set, as each clone in the collection was fully sequenced and clones with any form of truncation were not included in the collection. What we categorized as ‘failure’ of the remaining 28% of the <em>F. tularensis</em> proteome is attributable at least in part to yields below our cut-off of 120 ng. Low expression may reflect inherent properties of the proteins (such as high hydrophobic content) that could lead to poor expression in or toxicity to the heterologous host cells. Finally, the conditions used for HT protein expression were by necessity optimized for a wide range of proteins. The use of individualized conditions might result in improved results for particular proteins. The protein expression clones generated in this study can readily be used to scale up protein expression to achieve higher amounts of protein if required. The use of the Gateway recombinational cloning system to build the clones further facilitates easy shuttling of ORFs into a variety of protein expression vectors with different affinity tags. As this collection includes a normalized stop codon at the end of the ORF, this collection is restricted to adding tags at only the N-terminus.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Automated protein analysis obviated the need to use gel-based chromatography and resulted in a quantitative digital output of protein concentration and purity. This form of output allowed normalization of protein concentration prior to use in downstream assays. Because our past experience suggested that HT experimentation could be error prone, we took precautions at every step to minimize manual intervention and ensure efficient tracking at the plate and sample levels.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In summary, the first complete full length sequence verified clone set representing the genome of <em>F. tularensis</em> (subsp. <em>tularensis SCHU S4</em>) was created. The clone collection was successfully used to generate a protein expression clone resource, which was subsequently used to produce proteins for over 72% of the <em>F. tularensis</em> proteome. The entire operation was automated and was supported by a LIMS, as well as custom databases and software tools. The clone repository serves as an important resource with which to probe the biology of <em>Francisella</em> and with slight alterations, the automated pipelines we developed will be used for a variety of different high-throughput assays. The clones generated in this study are openly available at <a href="http://plasmid.hms.harvard.edu">http://plasmid.hms.harvard.edu</a>. We expect that the operational methods adopted in this study will serve as an example for the design of similar processes relevant to other experimental systems.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Oligonucleotide design and ORF amplification</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The ORFs were amplified using two consecutive rounds of PCR amplification from the genomic DNA of <em>Francisella tularensis Schu4</em>. In the first round, matched 5′ and 3′ oligonucleotides containing gene-specific sequences with normalized start (ATG) and stop (TAG) codons plus a short segment of the <em>attB1</em> and <em>attB2</em> sequences, respectively, was used to amplify each target ORF. The resulting product was then further amplified in the second round of PCR using “universal” primers that overlapped and completed the <em>attB1</em> and <em>attB2</em> sequences. This two-step PCR improves fidelity and lowers cost. Oligonucleotides were automatically designed using in-house software employing the nearest-neighbor algorithm to generate primer pairs that match the ends of the coding sequences with a specified melting temperature and then appends the partial <em>attB</em> tails as follows: forward primer, 5′<span class="underline">TACAAAAAAGCAGGCT</span>CCACC- atg→<em>gene-specific sequences</em>; reverse primer, 5′<span class="underline">GTACAAGAAAGCTGGGT</span>C -tag → <em>gene-specific sequences</em> (underlining indicates partial <em>attB</em> sequences). Second-step PCR universal primers were synthesized as follows: forward primer, 5′-GGG<span class="underline">GACAAGTTTGTACAAAAAAGCAGGCT</span>CC; reverse primer, <span class="gene">5′-GGG<span class="underline">GACCACTTTGTACAAGAAAGCTGGGT</span>C</span> (underlined are <em>attB</em> sequences).</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>For phase 1, first and second-step PCR amplifications were done with KOD enzyme (Novagen). Conditions were as follows: PCR-1: 0.06 mM each primer, 1× KOD buffer 1, 0.2 mM dNTPs, 1 mM MgSO4, 1× KOD buffer 2, 200 ng genomic DNA, 0.6U KOD polymerase; 94°C 2 min, 15 cycles [94°C 15 s, 59°C 1 min, 68°C 6 min] 68°C 12 min, 4°C hold; PCR-2: 0.125 mM each <em>att</em>-primer, 1× KOD buffer1, 0.3 mM dNTPs, 1 mM MgSO<sub>4</sub>, 1× KOD buffer2, 0.6U KOD polymerase, 40% (v/v) PCR-1; 94°C 2 min, 6 cycles [94°C 15 s, 59°C 1 min, 68°C 6 min] 68°C 12 min, 4°C hold.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>For phase two, both PCRs used Phusion™ enzyme (New England Biolabs) using the following conditions: PCR-1: 0.1 mM each primer, 1× reaction buffer, 0.2 mM dNTPs, 200 ng genomic DNA, 0.8U Phusion polymerase, 6% (v/v) DMSO; 94°C 2 min, 15 cycles [94°C 15 s, 52°C 1 min, 68°C 5 min] 68°C 12 min, 4°C hold; PCR-2: 0.125 mM each <em>att</em>-primer, 1x reaction buffer, 0.2 mM dNTPs, 0.8U Phusion polymerase, 6%(v/v) DMSO, 40% (v/v) PCR-1; 94°C 2 min, 5 cycles [94°C 15 s, 52°C 1 min, 68°C 5 min] 68°C 12 min, 4°C hold.</p>


<h4>Cloning of the F. tularensis FLEXGene collection</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Cloning of the <em>F. tularensis</em> FLEXGene collection was performed as described in <a href="#pone-0000577-g001">Figure 1</a>, and as published previously <a href="#pone.0000577-LaBaer1">[2]</a>, <a href="#pone.0000577-Hu1">[7]</a>. Briefly, the PCR amplified ORFs were recombined to generate ‘entry’ clones, i.e. ORFs captured in an entry or initial cloning vector that facilitates sub-cloning of the ORFs into vectors appropriate for specific experimental approaches. <em>E. coli</em> strain DH5αT1 (Invitrogen) was used for propagation of the clones. Expression clones for protein production were generated from the entry clones in a one-step recombinational sub-cloning reaction into pDEST17 (N-terminal 6xHIS tag; Invitrogen) as described by the manufacturers protocols and elsewhere <a href="#pone.0000577-LaBaer1">[2]</a>. All entry and expression recombinant clones are stored as DNA and as bacterial glycerol stocks (15% v/v glycerol) of single colony-selected <em>E. coli</em> transformants.</p>


<h4>Protein expression</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Cell growth, transformation, and protein purification were performed according to the protocols in our laboratories and are described elsewhere <a href="#pone.0000577-Murthy1">[14]</a>. Briefly, BL21 star (DE3) pLysS (Invitrogen) transformants harboring the recombinant plasmids were grown at 37°C as 1 ml cultures in a 96-well block (Marsh Biomedical Products) to an OD<sub>600</sub> of ~0.7. The cultures were induced with 1 mM IPTG and grown on a 96-well plate shaker (Multitron) at a speed of 900 rpm. After allowing a post-induction growth for a period of 4 hrs, the cells were harvested at 4°C and stored at −80°C for further use.</p>


<h4>Automated 96-well protein purification</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Protein purifications were performed in 96-well plates using a BiomekFx (Beckman Coulter) robotic liquid handler under conditions optimized in our laboratory. Cell lysis, protein binding and washes were optimized by modulating the number of pipetting cycles, the shaking speed and the volumes of various reagents. For 6xHIS denaturing affinity purification, the robotic deck was setup using 15 ml of lysis buffer I (100 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Tris, pH 8.0), 15 ml of lysis buffer II (100 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Tris, 6M Guanidine hydrochloride, 10 mM, 2-Mercaptoethanol, pH 8.0), 25 ml of wash buffer (100 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Tris, 8M Urea) and 15 ml of elution buffer (wash buffer with 500 mM imidazole) for each 96-well block. A 96-well Magnabot (Promega, #V8151) and compatible plates (Greiner Bio-one, #650101) were used for purification steps and the final eluate was collected in a fresh 96-well plate.</p>


<h4>6xHIS affinity purification</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The cell pellets were thawed at room temperature for 15 min, lysed in the presence of protease inhibitors in 100 µl lysis buffer I, robotically resuspended in a 96-well block and agitated at 900 rpm for 10 min (5 min in the clockwise direction and 5 min in the counterclockwise direction). Then, 10 µl of DNase mix (10 mg/ml. DNase from Sigma Aldrich in 900 mM MgCl<sub>2</sub>, 100 mM MnCl<sub>2</sub>) was added to the lysate followed by agitation at 900 rpm for 10 min. Next, 100 µl of lysis buffer II was added to create denaturing conditions. The cell lysate was allowed to bind to 30 µl of MagneHIS (Promega #V8565) with shaking at 900 rpm for 20 min (10 min clockwise, 10 min counterclockwise). The beads were separated using a magnabot (24-pin magnet) and the remaining lysate was robotically pipetted and discarded. The MagneHIS beads with bound protein were washed three times with wash buffer. Bead adherence to the walls during washing was prevented by shaking at 900 rpm for 2.5 min clockwise and then 2.5 min counterclockwise. The bound protein was either directly used in assays or eluted in 50 µl elution buffer (i.e., wash buffer with 500 mM imidazole).</p>


<h4>Automated 96-well protein analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Protein analysis was performed in a 96-well format using a capillary-based instrument, the LabChip90 (Caliper Sciences). Protein samples were resuspended in analysis buffer (Caliper Sciences), heated to 96°C for 5 min., cooled to room temperature and briefly centrifuged to collect the sample. Distilled water (35 µl) was added to each sample prior to analysis. The analysis chip (Caliper Sciences) was primed according to the manufacturer's instructions. The automated protein analysis generated three different forms of output: a chromatogram that shows migration time; a virtual gel that mimics a Coomassie stained gel; and a results table that includes the estimated size, quality and quantity of each peak. The LabChip90 analyzed 96 proteins at a time with analysis time of 40 seconds per sample. The output results were parsed and imported into our protein database. As the error range for the LC90 was +/− 20%, any bands corresponding to +/− 40% of expected size and above the 120 ng cutoff were annotated as the correct band. The computed results were reviewed manually, and in the case of small proteins (&lt;14 kDa), the size criteria were expanded to +/− 60% due to the resolution of the instrument in this range.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000577"></div><a name="pone.0000577.s001" id="pone.0000577.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000577.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Genome annotation</p>
<p class="siDoi">doi:10.1371/journal.pone.0000577.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.19 MB XLS)</p>
<a name="pone.0000577.s002" id="pone.0000577.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000577.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Complete clone list</p>
<p class="siDoi">doi:10.1371/journal.pone.0000577.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.71 MB XLS)</p>
<a name="pone.0000577.s003" id="pone.0000577.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000577.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Protein expression data</p>
<p class="siDoi">doi:10.1371/journal.pone.0000577.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.17 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank all past and present members of the Harvard Institute of Proteomics who contributed ideas and suggestions to this work. We thank Hakan Svenson, Par Larsson and Herve Tettelin for providing the preliminary genome annotation of the <em>Francisella tularensis</em>. The genomic DNA used to generate the collection was kindly provided by Wayne Conlan at the National Research Council, Canada. We thank Drs. Arthur Tzianabos, LeeAnn Blalock, Dennis Kasper and Gerald Beltz and Ms. Janice Williamson for their suggestions during the course of the project. We also thank Michael Collins for computer systems support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JL LB. Performed the experiments: TM AR ZS JR DM FK SM DJ. Analyzed the data: YH AR ET DZ. Contributed reagents/materials/analysis tools: AR ZS JR DM FK SM DJ MF. Wrote the paper: YH TM AR SM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000577-Heyman1" id="pone.0000577-Heyman1"></a>Heyman JA, Cornthwaite J, Foncerrada L, Gilmore JR, Gontang E, et al.  (1999) Genome-scale cloning and expression of individual open reading frames using topoisomerase I-mediated ligation. Genome Res  9: 383–392.  <ul class="find" data-citedArticleID="963012"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome-scale+cloning+and+expression+of+individual+open+reading+frames+using+topoisomerase+I-mediated+ligation.&amp;auth=&amp;atitle=Genome-scale+cloning+and+expression+of+individual+open+reading+frames+using+topoisomerase+I-mediated+ligation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-scale+cloning+and+expression+of+individual+open+reading+frames+using+topoisomerase+I-mediated+ligation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-scale+cloning+and+expression+of+individual+open+reading+frames+using+topoisomerase+I-mediated+ligation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000577-LaBaer1" id="pone.0000577-LaBaer1"></a>LaBaer J, Qiu Q, Anumanthan A, Mar W, Zuo D, et al.  (2004) The Pseudomonas aeruginosa PA01 gene collection. Genome Res  14: 2190–2200.  <ul class="find" data-citedArticleID="963016" data-doi="10.1101/gr.2482804"><li><a href="http://dx.doi.org/10.1101/gr.2482804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pseudomonas+aeruginosa+PA01+gene+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pseudomonas+aeruginosa+PA01+gene+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000577-Rual1" id="pone.0000577-Rual1"></a>Rual JF, Hirozane-Kishikawa T, Hao T, Bertin N, Li S, et al.  (2004) Human ORFeome version 1.1: A platform for reverse proteomics. Genome Res  14: 2128–2135.  <ul class="find" data-citedArticleID="963032" data-doi="10.1101/gr.2973604"><li><a href="http://dx.doi.org/10.1101/gr.2973604" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+ORFeome+version+1.1%3A+A+platform+for+reverse+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+ORFeome+version+1.1%3A+A+platform+for+reverse+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000577-Park1" id="pone.0000577-Park1"></a>Park J, Hu Y, Murthy TV, Vannberg F, Shen B, et al.  (2005) Building a human kinase gene repository: bioinformatics, molecular cloning, and functional validation. Proc Natl Acad Sci U S A  102: 8114–8119.  <ul class="find" data-citedArticleID="963026" data-doi="10.1073/pnas.0503141102"><li><a href="http://dx.doi.org/10.1073/pnas.0503141102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Building+a+human+kinase+gene+repository%3A+bioinformatics%2C+molecular+cloning%2C+and+functional+validation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Building+a+human+kinase+gene+repository%3A+bioinformatics%2C+molecular+cloning%2C+and+functional+validation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000577-Ecker1" id="pone.0000577-Ecker1"></a>Ecker A, Moon R, Sinden RE, Billker O (2006) Generation of gene targeting constructs for Plasmodium berghei by a PCR-based method amenable to high-throughput applications. Mol Biochem Parasitol  145: 265–268.  <ul class="find" data-citedArticleID="963008" data-doi="10.1016/j.molbiopara.2005.10.006"><li><a href="http://dx.doi.org/10.1016/j.molbiopara.2005.10.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+gene+targeting+constructs+for+Plasmodium+berghei+by+a+PCR-based+method+amenable+to+high-throughput+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+gene+targeting+constructs+for+Plasmodium+berghei+by+a+PCR-based+method+amenable+to+high-throughput+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000577-Matsuyama1" id="pone.0000577-Matsuyama1"></a>Matsuyama A, Arai R, Yashiroda Y, Shirai A, Kamata A, et al.  (2006) ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nat Biotechnol  24: 841–847.  <ul class="find" data-citedArticleID="963020" data-doi="10.1038/nbt1222"><li><a href="http://dx.doi.org/10.1038/nbt1222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ORFeome+cloning+and+global+analysis+of+protein+localization+in+the+fission+yeast+Schizosaccharomyces+pombe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ORFeome+cloning+and+global+analysis+of+protein+localization+in+the+fission+yeast+Schizosaccharomyces+pombe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000577-Hu1" id="pone.0000577-Hu1"></a>Hu Y, Rolfs A, Bhullar B, Murthy TV, Zhu C, et al.  (2007) Approaching a complete repository of sequence-verified protein-encoding clones for <em>Saccharomyces cerevisiae</em>. Genome Res  17: 536–543.  <ul class="find" data-citedArticleID="963014" data-doi="10.1101/gr.6037607"><li><a href="http://dx.doi.org/10.1101/gr.6037607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approaching+a+complete+repository+of+sequence-verified+protein-encoding+clones+for+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approaching+a+complete+repository+of+sequence-verified+protein-encoding+clones+for+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000577-Dieckman1" id="pone.0000577-Dieckman1"></a>Dieckman L, Gu M, Stols L, Donnely M, Collart FR (2002) High throughput methods for gene cloning and expression. Protein Expr Purif  25: 1–7.  <ul class="find" data-citedArticleID="963006" data-doi="10.1006/prep.2001.1602"><li><a href="http://dx.doi.org/10.1006/prep.2001.1602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+throughput+methods+for+gene+cloning+and+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+throughput+methods+for+gene+cloning+and+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000577-Reboul1" id="pone.0000577-Reboul1"></a>Reboul J, Vaglio P, Rual JF, Lamesch P, Martinez M, et al.  (2003) C. elegans ORFeome version 1.1: experimental verification of the genome annotation and resource for proteome-scale protein expression. Nat Genet  34: 35–41.  <ul class="find" data-citedArticleID="963030" data-doi="10.1038/ng1140"><li><a href="http://dx.doi.org/10.1038/ng1140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=C.+elegans+ORFeome+version+1.1%3A+experimental+verification+of+the+genome+annotation+and+resource+for+proteome-scale+protein+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22C.+elegans+ORFeome+version+1.1%3A+experimental+verification+of+the+genome+annotation+and+resource+for+proteome-scale+protein+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000577-Parrish1" id="pone.0000577-Parrish1"></a>Parrish JR, Limjindaporn T, Hines JA, Liu J, Liu G, et al.  (2004) High-throughput cloning of Campylobacter jejuni ORfs by in vivo recombination in Escherichia coli. J Proteome Res  3: 582–586.  <ul class="find" data-citedArticleID="963028" data-doi="10.1021/pr0341134"><li><a href="http://dx.doi.org/10.1021/pr0341134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+cloning+of+Campylobacter+jejuni+ORfs+by+in+vivo+recombination+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+cloning+of+Campylobacter+jejuni+ORfs+by+in+vivo+recombination+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000577-Temple1" id="pone.0000577-Temple1"></a>Temple G, Lamesch P, Milstein S, Hill DE, Wagner L, et al.  (2006) From genome to proteome: developing expression clone resources for the human genome. Hum Mol Genet 15 Spec No  1: R31–43.  <ul class="find" data-citedArticleID="963038" data-doi="10.1093/hmg/ddl048"><li><a href="http://dx.doi.org/10.1093/hmg/ddl048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genome+to+proteome%3A+developing+expression+clone+resources+for+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genome+to+proteome%3A+developing+expression+clone+resources+for+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000577-Witt1" id="pone.0000577-Witt1"></a>Witt AE, Hines LM, Collins NL, Hu Y, Gunawardane RN, et al.  (2006) Functional proteomics approach to investigate the biological activities of cDNAs implicated in breast cancer. J Proteome Res  5: 599–610.  <ul class="find" data-citedArticleID="963040" data-doi="10.1021/pr050395r"><li><a href="http://dx.doi.org/10.1021/pr050395r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+proteomics+approach+to+investigate+the+biological+activities+of+cDNAs+implicated+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+proteomics+approach+to+investigate+the+biological+activities+of+cDNAs+implicated+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000577-Chambers1" id="pone.0000577-Chambers1"></a>Chambers SP, Austen DA, Fulghum JR, Kim WM (2004) High-throughput screening for soluble recombinant expressed kinases in Escherichia coli and insect cells. Protein Expr Purif  36: 40–47.  <ul class="find" data-citedArticleID="963002" data-doi="10.1016/j.pep.2004.03.003"><li><a href="http://dx.doi.org/10.1016/j.pep.2004.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+screening+for+soluble+recombinant+expressed+kinases+in+Escherichia+coli+and+insect+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+screening+for+soluble+recombinant+expressed+kinases+in+Escherichia+coli+and+insect+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000577-Murthy1" id="pone.0000577-Murthy1"></a>Murthy TV, Wu W, Qiu QQ, Shi Z, LaBaer J, et al.  (2004) Bacterial cell-free system for high-throughput protein expression and a comparative analysis of Escherichia coli cell-free and whole cell expression systems. Protein Expr Purif  36: 217–225.  <ul class="find" data-citedArticleID="963024" data-doi="10.1016/j.pep.2004.04.002"><li><a href="http://dx.doi.org/10.1016/j.pep.2004.04.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+cell-free+system+for+high-throughput+protein+expression+and+a+comparative+analysis+of+Escherichia+coli+cell-free+and+whole+cell+expression+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+cell-free+system+for+high-throughput+protein+expression+and+a+comparative+analysis+of+Escherichia+coli+cell-free+and+whole+cell+expression+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000577-Braun1" id="pone.0000577-Braun1"></a>Braun P, Hu Y, Shen B, Halleck A, Koundinya M, et al.  (2002) Proteome-scale purification of human proteins from bacteria. Proc Natl Acad Sci U S A  99: 2654–2659.  <ul class="find" data-citedArticleID="962998" data-doi="10.1073/pnas.042684199"><li><a href="http://dx.doi.org/10.1073/pnas.042684199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteome-scale+purification+of+human+proteins+from+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteome-scale+purification+of+human+proteins+from+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000577-Braun2" id="pone.0000577-Braun2"></a>Braun P, LaBaer J (2003) High throughput protein production for functional proteomics. Trends Biotechnol  21: 383–388.  <ul class="find" data-citedArticleID="963000" data-doi="10.1016/s0167-7799(03)00189-6"><li><a href="http://dx.doi.org/10.1016/s0167-7799(03)00189-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+throughput+protein+production+for+functional+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+throughput+protein+production+for+functional+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000577-Sjostedt1" id="pone.0000577-Sjostedt1"></a>Sjostedt A (2006) Intracellular survival mechanisms of Francisella tularensis, a stealth pathogen. Microbes Infect  8: 561–567.  <ul class="find" data-citedArticleID="963036" data-doi="10.1016/j.micinf.2005.08.001"><li><a href="http://dx.doi.org/10.1016/j.micinf.2005.08.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intracellular+survival+mechanisms+of+Francisella+tularensis%2C+a+stealth+pathogen." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intracellular+survival+mechanisms+of+Francisella+tularensis%2C+a+stealth+pathogen.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000577-Fortier1" id="pone.0000577-Fortier1"></a>Fortier AH, Leiby DA, Narayanan RB, Asafoadjei E, Crawford RM, et al.  (1995) Growth of Francisella tularensis LVS in macrophages: the acidic intracellular compartment provides essential iron required for growth. Infect Immun  63: 1478–1483.  <ul class="find" data-citedArticleID="963010"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Growth+of+Francisella+tularensis+LVS+in+macrophages%3A+the+acidic+intracellular+compartment+provides+essential+iron+required+for+growth.&amp;auth=&amp;atitle=Growth+of+Francisella+tularensis+LVS+in+macrophages%3A+the+acidic+intracellular+compartment+provides+essential+iron+required+for+growth." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Growth+of+Francisella+tularensis+LVS+in+macrophages%3A+the+acidic+intracellular+compartment+provides+essential+iron+required+for+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Growth+of+Francisella+tularensis+LVS+in+macrophages%3A+the+acidic+intracellular+compartment+provides+essential+iron+required+for+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000577-Dennis1" id="pone.0000577-Dennis1"></a>Dennis DT, Inglesby TV, Henderson DA, Bartlett JG, Ascher MS, et al.  (2001) Tularemia as a biological weapon: medical and public health management. Jama  285: 2763–2773.  <ul class="find" data-citedArticleID="963004" data-doi="10.1001/jama.285.21.2763"><li><a href="http://dx.doi.org/10.1001/jama.285.21.2763" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tularemia+as+a+biological+weapon%3A+medical+and+public+health+management." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tularemia+as+a+biological+weapon%3A+medical+and+public+health+management.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000577-Larsson1" id="pone.0000577-Larsson1"></a>Larsson P, Oyston PC, Chain P, Chu MC, Duffield M, et al.  (2005) The complete genome sequence of Francisella tularensis, the causative agent of tularemia. Nat Genet  37: 153–159.  <ul class="find" data-citedArticleID="963018" data-doi="10.1038/ng1499"><li><a href="http://dx.doi.org/10.1038/ng1499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+Francisella+tularensis%2C+the+causative+agent+of+tularemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+Francisella+tularensis%2C+the+causative+agent+of+tularemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000577-McLendon1" id="pone.0000577-McLendon1"></a>McLendon MK, Apicella MA, Allen LA (2006) Francisella tularensis: Taxonomy, Genetics, and Immunopathogenesis of a Potential Agent of Biowarfare. Annu Rev Microbiol  60: 167–185.  <ul class="find" data-citedArticleID="963022" data-doi="10.1146/annurev.micro.60.080805.142126"><li><a href="http://dx.doi.org/10.1146/annurev.micro.60.080805.142126" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Francisella+tularensis%3A+Taxonomy%2C+Genetics%2C+and+Immunopathogenesis+of+a+Potential+Agent+of+Biowarfare." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Francisella+tularensis%3A+Taxonomy%2C+Genetics%2C+and+Immunopathogenesis+of+a+Potential+Agent+of+Biowarfare.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000577-Serruto1" id="pone.0000577-Serruto1"></a>Serruto D, Rappuoli R (2006) Post-genomic vaccine development. FEBS Lett  580: 2985–2992.  <ul class="find" data-citedArticleID="963034" data-doi="10.1016/j.febslet.2006.04.084"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.04.084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Post-genomic+vaccine+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Post-genomic+vaccine+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.XML" value="69614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.PDF" value="399240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g001.PNG_L" value="568036"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g001.PNG_M" value="21097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g001.PNG_S" value="2818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g001.TIF" value="1075038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g001.PNG_I" value="17441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.t001.PNG_L" value="237472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.t001.PNG_M" value="81089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.t001.PNG_S" value="4259"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.t001.TIF" value="1202624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.t001.PNG_I" value="32776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g002.PNG_L" value="1572390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g002.PNG_M" value="276268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g002.PNG_S" value="7748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g002.TIF" value="2975614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.g002.PNG_I" value="104099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.s001.XLS" value="193024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.s002.XLS" value="1708032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000577.s003.XLS" value="169984"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000577&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000577" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000577&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000577&volume=&issue=&title=A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis&author_name=Tal%20Murthy%2C%20Andreas%20Rolfs%2C%20Yanhui%20Hu%2C%20Zhenwei%20Shi%2C%20Jacob%20Raphael%2C%20Donna%20Moreira%2C%20Fontina%20Kelley%2C%20Seamus%20McCarron%2C%20Daniel%20Jepson%2C%20Elena%20Taycher%2C%20Dongmei%20Zuo%2C%20Stephanie%20E.%20Mohr%2C%20Mauricio%20Fernandez%2C%20Leonardo%20Brizuela%2C%20Joshua%20LaBaer&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000577" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577&amp;t=A%20Full-Genomic%20Sequence-Verified%20Protein-Coding%20Gene%20Collection%20for%20Francisella%20tularensis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577&title=A%20Full-Genomic%20Sequence-Verified%20Protein-Coding%20Gene%20Collection%20for%20Francisella%20tularensis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577&amp;title=A%20Full-Genomic%20Sequence-Verified%20Protein-Coding%20Gene%20Collection%20for%20Francisella%20tularensis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000577&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Full-Genomic Sequence-Verified Protein-Coding Gene Collection for Francisella tularensis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000577';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Full-Genomic%20Sequence-Verified%20Protein-Coding%20Gene%20Collection%20for%20Francisella%20tularensis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000577" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000577" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">


















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="23992"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cloning%22" title="Search for articles in the subject area:'Cloning'"><div class="flagText">Cloning</div></a>
              <div data-categoryid="23551" data-articleid="23992"
                   data-categoryname="Cloning"
                   class="flagImage" title="Flag 'Cloning' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Francisella+tularensis%22" title="Search for articles in the subject area:'Francisella tularensis'"><div class="flagText">Francisella tularensis</div></a>
              <div data-categoryid="44741" data-articleid="23992"
                   data-categoryname="Francisella tularensis"
                   class="flagImage" title="Flag 'Francisella tularensis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+prediction%22" title="Search for articles in the subject area:'Gene prediction'"><div class="flagText">Gene prediction</div></a>
              <div data-categoryid="40279" data-articleid="23992"
                   data-categoryname="Gene prediction"
                   class="flagImage" title="Flag 'Gene prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23992"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23992"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+expression%22" title="Search for articles in the subject area:'Protein expression'"><div class="flagText">Protein expression</div></a>
              <div data-categoryid="17513" data-articleid="23992"
                   data-categoryname="Protein expression"
                   class="flagImage" title="Flag 'Protein expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="23992"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5365'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3437'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6344&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
